<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/6/457" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/6/457.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/6/457.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/6/457.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/1/6/457/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/6/457.atom" />
<meta name="HW.pisa" content="ggg;1/6/457" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genotype Imputation with Thousands of Genomes" />
<meta name="DC.Identifier" content="10.1534/g3.111.001198" />
<meta name="DC.Date" content="2011-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2011 Howie et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotype imputation is a statistical technique that is often used to increase the power and resolution of genetic association studies. Imputation methods work by using haplotype patterns in a reference panel to predict unobserved genotypes in a study dataset, and a number of approaches have been proposed for choosing subsets of reference haplotypes that will maximize accuracy in a given study population. These panel selection strategies become harder to apply and interpret as sequencing efforts like the 1000 Genomes Project produce larger and more diverse reference sets, which led us to develop an alternative framework. Our approach is built around a new approximation that uses local sequence similarity to choose a custom reference panel for each study haplotype in each region of the genome. This approximation makes it computationally efficient to use all available reference haplotypes, which allows us to bypass the panel selection step and to improve accuracy at low-frequency variants by capturing unexpected allele sharing among populations. Using data from HapMap 3, we show that our framework produces accurate results in a wide range of human populations. We also use data from the Malaria Genetic Epidemiology Network (MalariaGEN) to provide recommendations for imputation-based studies in Africa. We demonstrate that our approximation improves efficiency in large, sequence-based reference panels, and we discuss general computational strategies for modern reference datasets. Genome-wide association studies will soon be able to harness the power of thousands of reference genomes, and our work provides a practical way for investigators to use this rich information. New methodology from this study is implemented in the IMPUTE2 software package." />
<meta name="DC.Contributor" content="Bryan Howie" />
<meta name="DC.Contributor" content="Jonathan Marchini" />
<meta name="DC.Contributor" content="Matthew Stephens" />
<meta name="article:published_time" content="2011-11-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Genotype Imputation with Thousands of Genomes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotype imputation is a statistical technique that is often used to increase the power and resolution of genetic association studies. Imputation methods work by using haplotype patterns in a reference panel to predict unobserved genotypes in a study dataset, and a number of approaches have been proposed for choosing subsets of reference haplotypes that will maximize accuracy in a given study population. These panel selection strategies become harder to apply and interpret as sequencing efforts like the 1000 Genomes Project produce larger and more diverse reference sets, which led us to develop an alternative framework. Our approach is built around a new approximation that uses local sequence similarity to choose a custom reference panel for each study haplotype in each region of the genome. This approximation makes it computationally efficient to use all available reference haplotypes, which allows us to bypass the panel selection step and to improve accuracy at low-frequency variants by capturing unexpected allele sharing among populations. Using data from HapMap 3, we show that our framework produces accurate results in a wide range of human populations. We also use data from the Malaria Genetic Epidemiology Network (MalariaGEN) to provide recommendations for imputation-based studies in Africa. We demonstrate that our approximation improves efficiency in large, sequence-based reference panels, and we discuss general computational strategies for modern reference datasets. Genome-wide association studies will soon be able to harness the power of thousands of reference genomes, and our work provides a practical way for investigators to use this rich information. New methodology from this study is implemented in the IMPUTE2 software package.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/11/01" />
<meta name="citation_mjid" content="ggg;1/6/457" />
<meta name="citation_id" content="1/6/457" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/6/457" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/6/457.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/6/457.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/6/457.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001198" />
<meta name="citation_pmid" content="22384356" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="457" />
<meta name="citation_lastpage" content="470" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bryan Howie" />
<meta name="citation_author_institution" content="Department of Human Genetics and" />
<meta name="citation_author_email" content="bhowie@uchicago.edu" />
<meta name="citation_author_email" content="marchini@stats.ox.ac.uk" />
<meta name="citation_author" content="Jonathan Marchini" />
<meta name="citation_author_institution" content="Department of Statistics, University of Oxford, Oxford OX1 3TG, United Kingdom" />
<meta name="citation_author_email" content="bhowie@uchicago.edu" />
<meta name="citation_author_email" content="marchini@stats.ox.ac.uk" />
<meta name="citation_author" content="Matthew Stephens" />
<meta name="citation_author_institution" content="Department of Human Genetics and" />
<meta name="citation_author_institution" content="Department of Statistics, University of Chicago, Chicago, Illinois 60637, and" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=S. R. Browning;citation_title=Missing data imputation and haplotype phase inference for genome-wide association studies.;citation_pages=439-450;citation_volume=124;citation_year=2008;citation_issue=5;citation_pmid=18850115;citation_doi=10.1007/s00439-008-0568-7" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Bryc;citation_author=A. Auton;citation_author=M. R. Nelson;citation_author=J. R. Oksenberg;citation_author=S. L. Hauser;citation_title=Genome-wide patterns of population structure and admixture in West Africans and African Americans.;citation_volume=107;citation_year=2010;citation_pmid=20080753;citation_doi=10.1073/pnas.0909559107" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=M. C. Campbell;citation_author=S. A. Tishkoff;citation_title=African genetic diversity: implications for human demographic history, modern human origins, and complex disease mapping.;citation_pages=403-433;citation_volume=9;citation_year=2008;citation_pmid=18593304;citation_doi=10.1146/annurev.genom.9.081307.164258" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. C. Campbell;citation_author=S. A. Tishkoff;citation_title=The evolution of human genetic and phenotypic variation in Africa.;citation_pages=R166-R173;citation_volume=20;citation_year=2010;citation_issue=4;citation_pmid=20178763;citation_doi=10.1016/j.cub.2009.11.050" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=P. I. de Bakker;citation_author=M. A. Ferreira;citation_author=X. Jia;citation_author=B. M. Neale;citation_author=S. Raychaudhuri;citation_title=Practical aspects of imputation-driven meta-analysis of genome-wide association studies.;citation_volume=17;citation_year=2008;citation_pmid=18852200;citation_doi=10.1093/hmg/ddn288" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=M. R. Egyud;citation_author=Z. K. Gajdos;citation_author=J. L. Butler;citation_author=S. Tischfield;citation_author=L. Le Marchand;citation_title=Use of weighted reference panels based on empirical estimates of ancestry for capturing untyped variation.;citation_pages=295-303;citation_volume=125;citation_year=2009;citation_issue=3;citation_pmid=19184111;citation_doi=10.1007/s00439-009-0627-8" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Guan;citation_author=M. Stephens;citation_title=Practical issues in imputation-based association mapping.;citation_pages=e1000279-e1000279;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19057666;citation_doi=10.1371/journal.pgen.1000279" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. D. Hernandez;citation_title=A flexible forward simulator for populations subject to selection and demography.;citation_volume=24;citation_year=2008;citation_pmid=18842601;citation_doi=10.1093/bioinformatics/btn522" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. Huang;citation_author=Y. Li;citation_author=A. B. Singleton;citation_author=J. A. Hardy;citation_author=G. Abecasis;citation_title=Genotype-imputation accuracy across worldwide human populations.;citation_pages=235-250;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19215730;citation_doi=10.1016/j.ajhg.2009.01.013" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. Huang;citation_author=C. Wang;citation_author=N. A. Rosenberg;citation_title=The relationship between imputation error and statistical power in genetic association studies in diverse populations.;citation_pages=692-698;citation_volume=85;citation_year=2009;citation_issue=5;citation_pmid=19853241;citation_doi=10.1016/j.ajhg.2009.09.017" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Jallow;citation_author=Y. Y. Teo;citation_author=K. S. Small;citation_author=K. A. Rockett;citation_author=P. Deloukas;citation_title=Genome-wide and fine-resolution association analysis of malaria in West Africa.;citation_pages=657-665;citation_volume=41;citation_year=2009;citation_issue=6;citation_pmid=19465909;citation_doi=10.1038/ng.388" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=L. Jostins;citation_author=K. I. Morley;citation_author=J. C. Barrett;citation_title=Imputation of low-frequency variants using the HapMap3 benefits from large, diverse reference sets.;citation_pages=662-666;citation_volume=19;citation_year=2011;citation_issue=6;citation_pmid=21364697;citation_doi=10.1038/ejhg.2011.10" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=J. Kennedy;citation_author=I. Mandoiu;citation_author=B. Pasaniuc;citation_title=Genotype error detection using Hidden Markov Models of haplotype diversity.;citation_pages=1155-1171;citation_volume=15;citation_year=2008;citation_issue=9;citation_pmid=18973433;citation_doi=10.1089/cmb.2007.0133" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Kimmel;citation_author=R. Shamir;citation_title=GERBIL: genotype resolution and block identification using likelihood.;citation_volume=102;citation_year=2005;citation_pmid=15615859;citation_doi=10.1073/pnas.0404730102" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Kong;citation_author=G. Masson;citation_author=M. L. Frigge;citation_author=A. Gylfason;citation_author=P. Zusmanovich;citation_title=Detection of sharing by descent, long-range phasing and haplotype imputation.;citation_pages=1068-1075;citation_volume=40;citation_year=2008;citation_issue=9;citation_pmid=19165921;citation_doi=10.1038/ng.216" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Li;citation_author=M. Stephens;citation_title=Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data.;citation_pages=2213-2233;citation_volume=165;citation_year=2003;citation_pmid=14704198" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=Y. Li;citation_author=C. Willer;citation_author=S. Sanna;citation_author=G. Abecasis;citation_title=Genotype imputation.;citation_pages=387-406;citation_volume=10;citation_year=2009;citation_pmid=19715440;citation_doi=10.1146/annurev.genom.9.081307.164242" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=Y. Li;citation_author=C. J. Willer;citation_author=J. Ding;citation_author=P. Scheet;citation_author=G. R. Abecasis;citation_title=MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.;citation_pages=816-834;citation_volume=34;citation_year=2010;citation_issue=8;citation_pmid=21058334;citation_doi=10.1002/gepi.20533" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Li;citation_author=C. Sidore;citation_author=H. M. Kang;citation_author=M. Boehnke;citation_author=G. Abecasis;citation_title=Low coverage sequencing: implications for the design of complex trait association studies.;citation_volume=21;citation_year=2011;citation_pmid=21460063;citation_doi=10.1101/gr.117259.110" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Z. Liu;citation_author=F. Tozzi;citation_author=D. M. Waterworth;citation_author=S. G. Pillai;citation_author=P. Muglia;citation_title=Meta-analysis and imputation refines the association of 15q25 with smoking quantity.;citation_pages=436-440;citation_volume=42;citation_year=2010;citation_issue=5;citation_pmid=20418889;citation_doi=10.1038/ng.572" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Malaria Genomic Epidemiology Network;citation_title=A global network for investigating the genomic epidemiology of malaria.;citation_pages=732-737;citation_volume=456;citation_year=2008;citation_issue=7223;citation_pmid=19079050;citation_doi=10.1038/nature07632" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_author=S. Myers;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A new multipoint method for genome-wide association studies by imputation of genotypes.;citation_pages=906-913;citation_volume=39;citation_year=2007;citation_issue=7;citation_pmid=17572673;citation_doi=10.1038/ng2088" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. I. McCarthy;citation_author=G. R. Abecasis;citation_author=L. R. Cardon;citation_author=D. B. Goldstein;citation_author=J. Little;citation_title=Genome-wide association studies for complex traits: consensus, uncertainty and challenges.;citation_pages=356-369;citation_volume=9;citation_year=2008;citation_issue=5;citation_pmid=18398418;citation_doi=10.1038/nrg2344" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=B. Pasaniuc;citation_author=R. Avinery;citation_author=T. Gur;citation_author=C. F. Skibola;citation_author=P. M. Bracci;citation_title=A generic coalescent-based framework for the selection of a reference panel for imputation.;citation_pages=773-782;citation_volume=34;citation_year=2010;citation_issue=8;citation_pmid=21058333;citation_doi=10.1002/gepi.20505" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=T. J. Pemberton;citation_author=C. Wang;citation_author=J. Z. Li;citation_author=N. A. Rosenberg;citation_title=Inference of unexpected genetic relatedness among individuals in HapMap Phase III.;citation_pages=457-464;citation_volume=87;citation_year=2010;citation_issue=4;citation_pmid=20869033;citation_doi=10.1016/j.ajhg.2010.08.014" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=F. A. Reed;citation_author=S. A. Tishkoff;citation_title=African human diversity, origins and migrations.;citation_pages=597-605;citation_volume=16;citation_year=2006;citation_issue=6;citation_pmid=17056248;citation_doi=10.1016/j.gde.2006.10.008" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=N. A. Rosenberg;citation_author=L. Huang;citation_author=E. M. Jewett;citation_author=Z. A. Szpiech;citation_author=I. Jankovic;citation_title=Genome-wide association studies in diverse populations.;citation_pages=356-366;citation_volume=11;citation_year=2010;citation_issue=5;citation_pmid=20395969;citation_doi=10.1038/nrg2760" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. Servin;citation_author=M. Stephens;citation_title=Imputation-based analysis of association studies: candidate regions and quantitative traits.;citation_pages=e114-e114;citation_volume=3;citation_year=2007;citation_issue=7;citation_pmid=17676998;citation_doi=10.1371/journal.pgen.0030114" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Stranger;citation_author=E. A. Stahl;citation_author=T. Raj;citation_title=Progress and promise of genome-wide association studies for human complex trait genetics.;citation_volume=187;citation_year=2011;citation_pmid=21115973;citation_doi=10.1534/genetics.110.120907" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=The 1000 Genomes Project Consortium;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=The International HapMap Consortium;citation_title=A haplotype map of the human genome.;citation_pages=1299-1320;citation_volume=437;citation_year=2005;citation_issue=7063;citation_pmid=16255080;citation_doi=10.1038/nature04226" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=The International HapMap Consortium;citation_title=Integrating common and rare genetic variation in diverse human populations.;citation_pages=52-58;citation_volume=467;citation_year=2010;citation_issue=7311;citation_pmid=20811451;citation_doi=10.1038/nature09298" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=The Wellcome Trust Case Control Consortium;citation_title=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.;citation_pages=661-678;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554300;citation_doi=10.1038/nature05911" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. Tishkoff;citation_author=F. A. Reed;citation_author=A. Ranciaro;citation_author=B. F. Voight;citation_author=C. C. Babbitt;citation_title=Convergent adaptation of human lactase persistence in Africa and Europe.;citation_pages=31-40;citation_volume=39;citation_year=2007;citation_issue=1;citation_pmid=17159977;citation_doi=10.1038/ng1946" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. A. Tishkoff;citation_author=F. A. Reed;citation_author=F. R. Friedlaender;citation_author=C. Ehret;citation_author=A. Ranciaro;citation_title=The genetic structure and history of Africans and African Americans.;citation_volume=324;citation_year=2009;citation_pmid=19407144;citation_doi=10.1126/science.1172257" />
<meta name="citation_reference" content="citation_journal_title=Pharmacogenomics;citation_journal_abbrev=Pharmacogenomics;citation_author=E. Zeggini;citation_author=J. P. Ioannidis;citation_title=Meta-analysis in genome-wide association studies.;citation_pages=191-201;citation_volume=10;citation_year=2009;citation_issue=2;citation_pmid=19207020;citation_doi=10.2217/14622416.10.2.191" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=J. Zheng;citation_author=Y. Li;citation_author=G. R. Abecasis;citation_author=P. Scheet;citation_title=A comparison of approaches to account for uncertainty in analysis of imputed genotypes.;citation_pages=102-110;citation_volume=35;citation_year=2011;citation_issue=2;citation_pmid=21254217;citation_doi=10.1002/gepi.20552" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genotype Imputation with Thousands of Genomes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="twitter:description" content="Genotype imputation is a statistical technique that is often used to increase the power and resolution of genetic association studies. Imputation methods work by using haplotype patterns in a reference panel to predict unobserved genotypes in a study dataset, and a number of approaches have been proposed for choosing subsets of reference haplotypes that will maximize accuracy in a given study population. These panel selection strategies become harder to apply and interpret as sequencing efforts like the 1000 Genomes Project produce larger and more diverse reference sets, which led us to develop an alternative framework. Our approach is built around a new approximation that uses local sequence similarity to choose a custom reference panel for each study haplotype in each region of the genome. This approximation makes it computationally efficient to use all available reference haplotypes, which allows us to bypass the panel selection step and to improve accuracy at low-frequency variants by capturing unexpected allele sharing among populations. Using data from HapMap 3, we show that our framework produces accurate results in a wide range of human populations. We also use data from the Malaria Genetic Epidemiology Network (MalariaGEN) to provide recommendations for imputation-based studies in Africa. We demonstrate that our approximation improves efficiency in large, sequence-based reference panels, and we discuss general computational strategies for modern reference datasets. Genome-wide association studies will soon be able to harness the power of thousands of reference genomes, and our work provides a practical way for investigators to use this rich information. New methodology from this study is implemented in the IMPUTE2 software package." />
<meta name="og-title" property="og:title" content="Genotype Imputation with Thousands of Genomes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/6/457" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotype imputation is a statistical technique that is often used to increase the power and resolution of genetic association studies. Imputation methods work by using haplotype patterns in a reference panel to predict unobserved genotypes in a study dataset, and a number of approaches have been proposed for choosing subsets of reference haplotypes that will maximize accuracy in a given study population. These panel selection strategies become harder to apply and interpret as sequencing efforts like the 1000 Genomes Project produce larger and more diverse reference sets, which led us to develop an alternative framework. Our approach is built around a new approximation that uses local sequence similarity to choose a custom reference panel for each study haplotype in each region of the genome. This approximation makes it computationally efficient to use all available reference haplotypes, which allows us to bypass the panel selection step and to improve accuracy at low-frequency variants by capturing unexpected allele sharing among populations. Using data from HapMap 3, we show that our framework produces accurate results in a wide range of human populations. We also use data from the Malaria Genetic Epidemiology Network (MalariaGEN) to provide recommendations for imputation-based studies in Africa. We demonstrate that our approximation improves efficiency in large, sequence-based reference panels, and we discuss general computational strategies for modern reference datasets. Genome-wide association studies will soon be able to harness the power of thousands of reference genomes, and our work provides a practical way for investigators to use this rich information. New methodology from this study is implemented in the IMPUTE2 software package." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
    <title>Genotype Imputation with Thousands of Genomes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"hvk1hxk-Gwj6nygkxDEPz7eOhYr5IXeIzAqVXPxTf08","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459194","apath":"\/ggg\/1\/6\/457.atom","pisa":"ggg;1\/6\/457","ac":{"\/ggg\/1\/6\/457.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/6\/457.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/6\/457"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/6\/457"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/6\/457":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EDdP8mL70B9rPIDm4lvutA32i8epU1K5Lqk8mww93yvUN2cmi3AJ0z6cvFkVZgs4JFpVgJL0\\\\\\\/dhTKzX5sv40WmNXAUrXsFNF1czbY\\\\\\\/9cswrrHIzqCUWr\\\\\\\/t+i5SKztNAr1bmORIGG2msrTra2IsHXGQqu\\\\\\\/4eJfyS\\\\\\\/k53yT\\\\\\\/u1ndq8X5JnqQToKFzSdp0NwvNAcaH3AuL6Rki6fOF2hvBJ8BeSkxvRfzFmi3FcWf3RNIh0TOEOcpZ8FuRGfqFqCSM6JsEE4LE4WbiTCuyi\\\\\\\/akGgqxV+pUFN0H3oZOxWvzFu45WkZXoEAjrkqenSzGpOJIS1gqTOFdlljgXAi+uWPymOuDQ9g13XsgA\\\\\\\/CsnRgaYLky35k7i000gE+wKj+ah95tCfGQYf+VBnYzag4e0FThoMYYrPb7MeLcrz0b9gHC4C9feWdTuPhTTkOogJaHvYooAM36sZAG0kqeEwoHco0W9mVPwmHqf+oP2T8Zxym7AI35MCj2UY3XKDG6dQw0Cbeet5KH3FN0zSV5zOwIWs9cBaMspsM+XxwdoDqRiAMvp0BcjuBREauzxWEr+mAFuSRVKc44LZu+vdTICxgU0sqijhvLLLupYMjjWDTUkJSUO9KV3PcEnI6ggBWqzxPV+sVL5K\\\\\\\/ksCZBJynYbiR5kBAQWiRGyi2XRuYGuAYT6Lx7cjBaCkSdxPbZ3koaLWOgIDXdU+Z3A6HJYQRtNOOvINzW38bCcYAZjC9rckC5kuZlqLJ35rCqN2TEmyNkoYkoJQWW5QHwDMogqIvds6CagMMF13C\\\\\\\/p1jLRcZQZTjVgWDjse2b\\\\\\\/r356Ofn4pGOHkwaPd7EadaoIFFGmBIlxSFm3rhVhhaCPgGR6IY0ZOxTNsMRZpXF0uQKwqpNSlPAdnbEE+x01EdntGWU1Z6Q51cQAgkyUGaYRQZyaJ9mz7yzshCnUR8iBTPbUDRz90kpZvRrib2LUgpvM2v9\\\\\\\/0zNjuAWGUUagp39oSwtf98zfDhLI\\\\\\\/WdDuPw9zGZ5DwYvjDXnlz6kFP\\\\\\\/LO5JNZfUsIhuMk5IwOaWx51uJaxpKXKMfkNtBEoDxsHohRvzIZvcpFnAFCRN\\\\\\\/Qdb6qCeBUE9aeMiEa5z0iOI0ExgD0UruHb\\\\\\\/H7zWzX4f8evieozRKqoXTc5OXYYNTK8Qc5OHHe1rOWJXhbrdGYrafGSTG2tzoRO4kzjqFMp5iM+caNS2UrWUyXIf+wP6I4g\\\\\\\/\\\\\\\/cvO69vm6xTw8tzdy6W2tnQH\\\\\\\/8cwqsa44r6Vur2xC40okvA6e921bcAdNxVRkXQHw\\\u0022,\\\u0022iv\\\u0022:\\\u0022iYnS56BtwkDLqa9Fd28ofw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002299ed5bd1ad5fa8ee9cfdfada3e9c895ae04c1ae22ba7f76129ba07e326179a17\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mTDWt2k5ngDpWNo4DokjkYSK4Qc8tGhz0bXkLLgQgbhujrPhdsicbTqCvcw7hIkFw8r90VO5CtjgQwj\\\\\\\/vOz7tiQdG1pPSKqEEVYo6EZpXxuxghHAP10evJG7vTEB2NAr7MXtwvC0XlMnFbupgTQDNUTo+4k\\\\\\\/+pmYTV\\\\\\\/vktU\\\\\\\/XRulBYS9l6BFJ\\\\\\\/F2u\\\\\\\/vcudiLmS8yI5wC2L2KgBFzN3UYGrdqXja8u6WRcP6nBIlQ8kkThsyiIFZubFeyXd0mHwo8\\\\\\\/9he\\\\\\\/\\\\\\\/RvNIHatRE+RCMdNydlpQkridvv3kkEWT+\\\\\\\/2YQa7SoB5Gwse6mzTMsSxHXIjmwIRjGe4+HLbSyY0c2jFzWG2yOPBVluW\\\\\\\/k4gU+cq67cNHQ6mQloNCgouNFZW1yBMZI2VaruUAWx3GBosKeh\\\\\\\/cBJmv4NXxoNEbP5otAQPMkmRNqw+livZxAhsAcqb+yYxOHrOfO+JKvwLL3kg2wzsfXC01mJ0ZZIbii7E8lXodiKEPQk5T\\\\\\\/\\\\\\\/HpVgkWn0I3bcs+Tad5abeoJ0gGtzzHpGE6gANgtJdB0KEmdGNutjEPnSAgHcQjjrszZF9zSh0FIzv1DQ+993BEYzizNC2ceOded1nwkX9X+NxNmtw8ibfCIq8zW131MT2VNoBbbivpM6R5tOiQjwAzWsRPGu5brNu\\\\\\\/gvos8K61ApzPxusATHo7SZYIu8ctndLaBy\\\\\\\/zxVFVPbNp4mpmcJcJqyImv5jkzz6Llr7oGE1NJ0UYBK1\\\\\\\/EbxyiOkh57X5ptomaKf\\\\\\\/ezi3DfWjRp8CJd3U++REiHzU+2CaKp717ae\\\\\\\/9dWDOVfBEYN0quwpVNNqUVbg6arlNWOYhrQf2Shu1whONSTxDdOiWxIk+xJpYlrrngqUTu3HAxsbSCkodNAdcxfDuPNx\\\\\\\/PNeqW6JuE9jJI9NZeHoB1gZFoS6\\\\\\\/quhtC+2TElJ08QpiPJva4Gxlqg+5E3nkAb15bmBOXKd\\\\\\\/WDLMCfHnSnTVwRT9IrjdnktHnvnZU2ehxhfvX4eCZNVZ8rFk71EjaBy5LNUVKfQWANKp48WLZKqbConYmF5Y\\\\\\\/VYrdn1yogrgvsW8Uh3t6GdGuVyPoA+65imaMb+ze3Y8xtV2TnlPNzA1kSAbiLYMyKPdOM+HJ0J5TlAbLmQgs+gaUfDrR1Yiq1lodbYoDdy8W4J\\\\\\\/j71lpH9r9EzkTHVkbc2U0KKUPoK97YuoyvoN5Ptxwj8wl1Cc7YOu2ptPWEFq81uE2EVj\\\\\\\/o7EP58HYu4nY+pxy8kDBU85XVRsJ4Jsp1cslnVsQVVezEH8UCzgwThuKwhS6DDch0w1udFMxl9Q\\\\\\\/XsO\\\\\\\/m0FY0Bdp6kYTKq57yBdAhS9dM9H3eTL0sl8onYm6M3NFAZP\\\\\\\/Lpy\\\\\\\/wCUeVbZxbOYNMpO5ZmmKR+igF4A44XVxPjIikUvfA4Dwki0J4cc1o088eZx5v3Ot7w88tvy\\\\\\\/Df4Cy+w1JJxl6GTh\\\\\\\/25OkL7472lsKtta2Do2E3BdELtaft0sqfGVYfOEm8YYOFyyNQb4ZD7lWO5AXj2Dakb9tXxzUz5BE7f4+a8HPKOk7SB03pRM+tbRBskYF9jz7EPw6cDuz0LgqBAt7MYcW29dyPi\\\\\\\/yHPYOON3vjbp6ppcpCzAS9HgLTAXXSzg1FYlN5eVjVc2fX4Z1e5ku1DV2y5wjQ1u9BeBJJayG48oXYWcbz6BgNva968BpbJ0vwpJ2gqyGfNb9yaTMmAZ85m5QRuuFLbERJF4MJWneM9dIijbQCmg1yC6SyunrT7xHfsjhoBGBXbyNFeeqqZNKZkydH2yt8MYsayPNbzvawA9P2CAHjsRchM6s\\\\\\\/XJQW3qONktL8Zse8SWZQIaTdT5NBuPZWM01MgTi9jsjAfq1GqrgK+OKSXchdl5HWcCe20zMY0mzl5eChnEkbvGw3FtfIqGScHdxhLuSXfB+ecGWzU7StjKeVM8GSVNzBKNJ8SacuswH9SQKBO5vXubbT9UXds8wdiZzsGJB957v2yaGTHJGSuXXM2FlgHNRdtiEAvwt4c0Sd35s0cK0ebMR+A0hmiw6GaLB3OYVgR7SCzlhlFCnQkA4mzPxj3WGlvQgUjSvvKstnPgK1noYbiNSGX35\\\\\\\/Yo4c7RMbSZaqyY+yRusfSBZVgHMOjKOjA+8dPfGMsc7Nz\\\\\\\/gBPgX4wo0JI3Cz\\\\\\\/jAY8kqp2EYnANE4Wi2EQKK1ayBpXSkMkZEp6lYPE18QbCIkPiM1nJKRvYd69D2JBvLxfJF0wve3HxEpK9orur7Or6x8RdFK9WIShr4YQYxhfj1l\\\\\\\/hM\\\\\\\/V1bf7aUTt6OlpGJhrbOXFYcWCyFtO\\\\\\\/jcUghwiPL1NLIWXfVLRQQwCEdilxhsINNKrg8FkNE3lAnjA6\\\\\\\/RjPDuKwz9uCZoSOhHCLSKUnDLgPihMwfC4mZbXswPr7Uijr\\\\\\\/teF8bnJhzbeYerv3QKiZC6sRfL+4ovMzODT+wlNgicY4SXy30eDWT5Nf1MVmERCLtlwnop4o7k1a5J1G2Vo89lvVC5cgSR+0XKqhen+L9Tr2ig7HxavjBKRzWgCzAdVkgdX1\\\\\\\/p8F56i1g8\\\\\\\/J9fX7oEEBqqiOK9T4myBWeb\\\\\\\/0LwMKCMtH29CZo7zGU\\\\\\\/pjBC7chT3Ua4O74cOkRn91C6N5ngRE84Jeg+Wf8T5kgdrGu9df7CDRkkFUZfgWomDNokhxc2uRZILuLSRxVbj1YSRXJkA0e4m6YmjrenSk2zUUy\\\\\\\/YopvxzWmAY9uDsQNDZrZlFDj7Bm\\\\\\\/qm7YrMIYhkIclGINfXbdSFeom8dzisWQS7gYxPJ4L6eHdScFjI4\\\\\\\/zSOL0fkASfUEIbpxWl15\\\\\\\/\\\\\\\/d7KRRgdREscQrszRVdfq6qzHsoTZ4dJhMCJG+06A9dVg8\\\\\\\/GH11tvIbeB+YbihY7ckQzjwlGW\\\\\\\/l7JoV18S0eNDPniY0GbyKKFSDJerTmZTx2QI\\\\\\\/cosJuUaaoqczRFndJIh1y4sek5\\\\\\\/z2f4xb7Ke0bKu0O8WEq35jlGTwjKandCc+ehGc2voSB+xBt0P2gOZOwJxASDq0TichbyzSretixibbBMDJq43lKHrzDc\\\\\\\/2u+QCNGC\\\\\\\/X4Pbl9gQucsolOMxla0Q6cnmp1lfIe+14Yl5mEAyKxBYYsZCAXeptQ5Ukb8JcW4Ks\\\\\\\/PrFLieI80UL0rwxJ1L2TW0d\\\\\\\/v5UtiwTA8H+S1MjuuYSS0xxh8HQrJ6qmE+aYMCkRVu203iOtePhGOstg+P6cdKJ\\\\\\\/+ggqA1pMU5qdwsbXY\\\\\\\/aHaxHq3e+adQ4BPio1U4vh+t0C24O96bblHs0yXdGCPC\\\\\\\/K9R5\\\\\\\/NZ522EWVxIBmHXiIIKo8ozFMKm+SCN6AUF4lAml29VZC7e0hBXvi3LhOEIX6a0ClQd6ExJnv73mma+++S1nbNacADHRCz\\\\\\\/0SxwBIaGLvZEnILvfqVF1DruGSiN7IINvs\\\\\\\/bs\\\\\\\/dBxEeDrgXhyDJ6d5L5ugw4Ez6M3iPVXGKtOJogjF6VykFvSRzWirUtUGY7sb0JztNe3BSZTJk8F7NuJzZ4QIG8l5feTXOtts9qnURYEb+UK7PZev8eFBFrWoXBjKcker+rHVXcUgKXF581zNDTHf6AiWpefJgy64beHKzq33GwkNtATbyESeoofZ7bnaO3PN6wuhd\\\\\\\/UaFJXvvN9H9ds5W87qUHGHkkECIT4C8sdlsh3qmgd3OmbCmvm10ruMGFcCOQz9pZr9Q4L5gsHuw3K0yHO17UZ7t05MzkUG\\\\\\\/jJ0wDdpXxSBX6cTmsgk0b5sp4y+aB\\\\\\\/qfavzYHxpJXVLKYHIupf5HskeUzh60TH9WF98\\\\\\\/GC1yaZVBqpEdsQ\\\\\\\/FRk\\\\\\\/ludqZpLsNsfBbZ5aZONIi5Ai91ur+uBGsFvhVYX0ZSbPcQC5Cbj77wAMub9Hjv26uIinOlhIwR16MZ0uwxWMjVQ\\\\\\\/QKsbNo\\\\\\\/3lDKNOTt\\\\\\\/XWgb8uuyG+M0G\\\\\\\/rivm1HdQNIOmrxEGRZrM3DSKux\\\\\\\/s0rPK3IKkOAllquADD8VsI8yNV\\\\\\\/o7bSoqWf1UaxUSxZm4hQg+BvDFLsfAaUbh5TcliTEcQ1kERmcrsvlEpthzYcYbFg3cu2HBodAWhVbV7lGmcE2dj61tiagcVBIEhCVbbJYtHIwhwZ7Ff5TT4bXbyUlEiVX5hNNhEZyv0c3hgTwuNf24r5S85HGGnz\\\\\\\/81J2pVgsgR11W0MB1DhlTiF0Br8nzwTdOQQvKwgp8nYsFTRFl8ctkD7zTpsXFCI20aVoejnr5cWtJXjk2rj3t9zr+PWJlSK1norVimqyEqzF3zkuTr37guV7RGaFJqdceJORklddCOHEHODheRWxSBLWnM+qO\\\\\\\/dgliCgx4D5c8zzJ6ImcFDGtpg8qXq2MwboLXPrJTfbqEE6pgXOh4Z4v8HFqPBt6m9ucPfMIXU6hRVEE9IB7AXLd3BL\\\\\\\/0TZLT0mPT0P+1p8jFuDtygcfgf6qKEYlw32s\\\\\\\/WD6vXzHX4e7QWNBLg5KGaa4MSNgLET3cv5+okmFz0HIImsJkHSeSG5Rw4zcVp46Noyo9CX+UQxxKJytSrnv02kK5iaKvbYVNlFn301OPKH8XIg7v92DcV6WXnsPyVO7mGBahnYSqdsi0Pz\\\\\\\/jA3j5y7vKMzBDlXb+emVtoCDdz3pch6MYqg6MiN+zZiBsmwN51waAw23XS17ocjGOwfi5sx1gBfnx1O4WA5v9G7PzhNgWs52Wus8LvuwDgk7uY\\\\\\\/XjNreiPpoSPmJkKFSB4iatuRFVzbldGvXsS3dqQRjTyowVnFVTGUI2Gb0xkmzLKmoPi9WBEe4sxHVB3I52tUKS+LKQIpFesq8sd1OH6G+ByHa7oarM\\\\\\\/5BdhdThyXMqEvCJLcB7Dz3bubRsRPeE\\\\\\\/WqM2xfjom5Sg7jF5guj\\\\\\\/0h4g2cwNEc+WMpglvfnrKa7xz6SSwZWv7jnv1yo2SOeI0pQmpdIdKWiMweS2qpyM8gvGEWvtvWkdtgTN+Q3nxa0mOaWsSQCdJ4AqSVST9AA2aC20EwrzZojwJ6xA1FkLGgp9Ah\\\\\\\/W6csd5H18g6e3EPRwfIkvMzMaHgc7h5zTjo0wC4LljZB69eYhx5pAEE50XH0nnzsrW5aohpeFbDS7FFkVwzJH0AYaLtFR8tdkOpHCxTTr29VYpVJx8OQpgACxRec0uFPJBNcyQY09G32bYuU0JyDyBmnYgA4iY556tGZEeWv9g+tDMU6RgiXVA54k+x6TNXjIUJXnOLz+fPHLNdUHxR6aqHJe2gOqaStMHsfLVheXZ6sQJFZiTziCO9YGADYO8gfEYJrhOH004IjZPDCp7TTHG3YvJkDETVoOeosx31NYmfZp5vo33TE4Dewdm0wE\\\\\\\/zjBb4pg66yN\\\\\\\/Kel5tM4QQQOJu73UCPQN8j6WubYe\\\\\\\/UKhIt+gQIDTcOl2SzEpK3HKBqs2nETtJkNuhfZt31N5i\\\\\\\/nbJXq2CHtzfTO13CXhK390SUM5FmzYZVA4oPpiEPkhqPh9rUvYRWLTp9W\\\\\\\/rtME2DjcNrBa5aIAGS8qelQegtEP6GyB5F0hFPzPPk24EfiUftPvNFCb1OuEA4ujPXAvblnsllMxgwMlhmy6k6I4jxNXizD\\\\\\\/Lrw8A7KUHOjJ+YVk9gJYxO1eyAbXTLbDmcvD\\\\\\\/LUY3n7Px6CRB0gHyge7OW6qzFKwpk6U8NIvPEJy5U3qlq+VDvP4aP7gh4xcqWeBHKe\\\\\\\/L4Ar2o3aauD+vQXHy1S5B\\\\\\\/wApalv08xG3JXZQofI4u7lw2I9RRYJnvGqpMbWs5p9ry9PC2dSkSQUxFQ+u1iXQeIXj28RP8UJZuzSZEXyPtlNT9+PSs7jp0p4vdCDEwQQoxFc9\\\\\\\/PoTya9g2GYE1ii4XFdBuvOmYt1zPtNXBO5Vz4SCdi6H07uJCqKCYN\\\\\\\/ghjX5loWdPWCxKAwrtTB517uWEYKX49XKdGyag6Cg3dQ5fMm\\\\\\\/tR6F9zTAuCyOtcO6U7egqUc119gblra5jxMNIpGCJOHScJd9FDyPTRuPU9PSrWps5O8L4NDT6+WELfY8+\\\\\\\/29jPBM5wrMMdJW7UwbeljI0zD2NPj5nE8adpo3QAdSPLw0K\\\\\\\/g6ZVK8Pv6rK9em5TkGzo+VP1URXv8vTRCff9bCitbQBeuVgdTS2bFNsW510A8xiDu0Pq5sR0\\\\\\\/fwAlSXLTQ+Wk6T63FANmmwP8UDFh9q7slt2i5+kUXOR9\\\\\\\/DH43CUge4fBo9NJiXv2uIxEI+oOpdBvqzXHs1GvZWW2bNIDMmWMswdTDAoNXKeM7jxavmiM8atZkmJcMgPNWfR9uE92KK2Bt56NmOFistqZ+KDRh+v2mvuLCk+YVT5wHn4+DAuezoLz6R\\\\\\\/R6tp+ExSy8kCyckaLS+yvO7bbtn1nxVPIYIvAzQ4ycCMsC1QwFXUWqaMxu0km9mzC0HEHH7TRwUiyw4J9aw7JTcuyR7uwwcjyaJ9XOgg4x6JXcmuP4BRVbg412I0VB+ocn4BIKpodLazajgAbIogSneLw\\\\\\\/0ztUlS4OeqlyLwwLAxxbBwAI47\\\\\\\/B5zq2lsi32qfWtJYhrDNj7gp6llsWXZEnri+RJI87BCCaFqiS2bV1946cY5kJUnSabzybf+9obSzOannXfBXRHoS0VHGpgVKda2iUSnyNOiWDs6rum9Lxxs0UuzCWUGrdjNUoVXDyJ9bm+pLq\\\\\\\/34VFKxfnnTzzFS744KulNhY+SeE9OIRJWzTy4ehwqdHOaE8yDUxTSvtW1YaVCEv88TVFIK3ikkgzRsyy5UWGg8hgg8ACXLNDG1bVnqpBLY\\\\\\\/Odjrz4\\\\\\\/xXdpb5lOu8RbDOHaUxcEBYh0+Q2ke8zNPRmrrAWlZ16m09BIXBajNk1O20VpQU8W3VJbGa8l\\\\\\\/xXICnM8MT4XcY3\\\\\\\/DI9Yw8RgeYU9nd1NuQNCDPVehCcbo2KpngNRP10PnDKdClrfMS4fpVAqXkQNgmKsqJHRoh1jqWq2bBiXvQRaG51sWusOIOuyE6R98XtkTi0eSBURr3+j5RX8y6mu2Rr8IXC40G66wmpous65dXWFHj7h0NRVaNPKzoqyRr7grQtunAXRw2u\\\\\\\/sWmrRyJYTonKsclQ+7+cUpmcBwGTKbDaiJ17Q73\\\\\\\/hC8+7SZp9zyyqOWm1PevAGya8gznI687nWMqyIXF8VpGBfItIa3\\\\\\\/bVGtC60hbGR4UsIYa9y+MUolnhYoRXnknyC30vl\\\\\\\/uEb9NbNQ81tPo\\\\\\\/iH3+t6T07bRhjy7Q7IXssuJQpNaLetlVGXcuiLTtjU3HM9xRd82BsRqFUhphXBILyuJXkBICeAfY5SiqnapXgmRrhdhoJbWxWoxWJpnT14p6PLIJ42tRk1dJW16x\\\\\\\/ROP0uVR480eS3y2C9p4ekiqzQKVM0O8d4jr4pYuiZHW4SzPV5TGUZNBFi+g6n8COXyL2w5fXhligw3ek\\\\\\\/F1C5GQ7hMlvzdUMYDkPJsQQc5\\\\\\\/LO9zTOotZkcxKTKhQ0e8S7mj+rxGwqd1ISnDNx0asObHH8UyNgTKbI5e51gQ2mOjw5BHnTXyrXNdSA4XNDuEkGCdB0VfsbfBjNQkoe0nPs06PE89NaplR4szb1jlZvcgZcFRGzxRE7kvNz9mWYtMx+vrDWlhDaj6pou25Xl57DUE0w45\\\\\\\/gyJswcWuQjFoE9bcaa1P2tSNrxv2LgSiYCDG\\\\\\\/s5vxsMTPau2rCXWH3+X4mYlCnHEl3oeCcHDmipN4PEgpiNUCZmBFkgZJH2TdQzIFj5SXdF02M6SAvgEYDQMTQy5P9o07b\\\\\\\/4wzHMvT8njlX0B+ig3aq6RfS5Z0Vo3m1nnE4I+NE52g8U8xSYmRLgjvRlk8BUGIruKy+9z6qfW531fU8O6cHgiBc8mDM2DcarwoDgEpT2V7FU1WDL7bmLDHNxJE6cVa3+XgVaNFDXY+ktyp9xDLmvN1a3PsVkSMTF0i9hySncH4lIjoxOABoOmZZVqvpy2CUBgpQLkxSl7FdGcWXnWS5+bXzZm33uVZODxmtGFTjmR495p9JJHHdS4GzW\\\\\\\/siqFABIo8X0BiULysMjIODbObLpNu2MmqN+lqQAqwTEmycCj8oyBJsl3RWuzLNkP9EwnwKKZ9k40lLjdjisnoPzqdn7CUwTTbiRY27FOo+UvaE5wAw7wWolIVkTcJumcKBkemKfsnKcjeICere4D44fuR8Vhgy56N\\\\\\\/eP07AzguQfFeZ6dYCkhJZa3rYFyAiL4AUVTO3I2XHWV5K9s9xMpUg+9j9CBXiuSuyzorpD+cEcwtb\\\\\\\/efY\\\\\\\/sZs4srpTj+NwWUdsESxSWLggb\\\\\\\/Qf8TLJf38bR6hQiU2btS5UtoV0H\\\\\\\/nzkOylWxnJQ1O\\\\\\\/P0FCkRgQHss7tA4yMMTrwyuBTLVhT4UQVGhhmkmO1VBCnCIsQbBCdBOC6as6VH5i0I\\\\\\\/Sw11U8a\\\\\\\/qEuO2O8VBu1UMIx7HGh\\\\\\\/urY4M1DHVDRnO5kxFAYsfAtjhSuzRgfUIARRasBr2AWLfKbolxQEBXrrHknI+r+kMojW0o3UNNi+PDvs9+VE6zP7jypFUQT54U563jtRb3OD8uqqLG5zGJCzEYqqVp6fw8N3UrD1tVo+PUlAWiWqat0poXO\\\\\\\/U3umYQ+7h3AZykfqCBxN77S+vN9d8RH\\\\\\\/JzgbR71WK94+YvLldPwhS3gTtAOU7gxIpZWcmqyiQl3atgKgRaaISisAPBng6zyCXrE4QLNzlx24JAQBm44t+rXclh8vhsnjM4v6UCF6DUWQ+uXrMsAqZglvMgzVjZJ8YoDiwjRYDcAdDbq3WvyYVRntXg8L87Vm36SRBkpX1n1UvSFiU5Cw0GtYOmn9ZIQq+qwqE+NVtNXF4RxuFr7s4IARz53Rnf+2WJe5VwQm5jWWLmXcL2LncHFQZLbMjdmf0mu4fUq0IBBYAL8nndRdxh4fwEGlevlJR1f2yPbyVPvHjvJ7bRWMyfDt7UAaNcFUoKnVx5GN11oQHZStm8UEoDYXMLmQPaNNKHDqXTHn6id+4fLz7J9fqm+1MYJISWtN8F9Cjx+kLBtbuN9E7PkeddFoQLwGnjPLq+6pL3ymsTU5\\\\\\\/vVwCE9WwYtE8\\\\\\\/vLErT9tLxOcgWiNMyxOqQnyLSERC0nrKpThrwvGKfguh0hXEzwfPq83ZsbzzYGS7yoNqSGNBmQuXbdoDqPjDKcg1ao0HgIczCgL\\\\\\\/hulxIL73d8lg1u91G9mNXq2xdxtFJshn+tGW5nODBvRa6n+BqopbE+4PszJlc9w5aKrdm22FrfQ0n4F45uVxnRfTfPFU7XWoYNuB2oNt3lfqZfoCwqJ7L0XRFbZ6CZeKsV6jqChbqjiFIovnjtUuy\\\\\\\/m0XsMsYWS5XsuTSrQZAbOQjjnuWyf7+2DJQZe+3JWvvD5rb0tD+YRHyqC8A19cAdAGHsiRxeZFmATIl88q2TOUdBgUEl5s7tV8j9ewPjadS4uNYNDxIfZOz6FHIqDWGT6Z0vbt06BFl8H1C5SX00kHE7YOFY\\\\\\\/+2QSslU88fmKFlqeShKVNcWN2\\\\\\\/NDdyq84G288AvWH1m\\\\\\\/v4Bf\\\\\\\/XWofCmZ5WOUw8XoiJ6\\\\\\\/k\\\\\\\/LgTPim2VWpLCluvPH+Dww\\\\\\\/8qXCJDSqdPfAgjBQAVff2g5UqsBRnQCvX\\\\\\\/G1mZpam1\\\\\\\/U+UXewn2bwYef2Pv4jeMDf\\\\\\\/tjb\\\\\\\/5qgXm2cU4o\\\\\\\/B2sfPr7P8OCKG42w3gJfJ4L8iv59pXNE63KGLhjOpbX2JqE0EKCz3bfth1h68X2bf5U52N1Rv666QaLlb1m9BcyBqwIHotpD8d99Ksm9eIVSUN\\\\\\\/vN8InOZ1M1279IGnK6ceMJHrZdWPcTNAO\\\\\\\/fnr4TxPCMT3PkJp2Ynm3elMkG4+4uiePsv28YUNDYvFt2NnDXvwZup4y\\\\\\\/JlyzSMjIlJAq3gJaHouJkp8F1ulEiBY\\\\\\\/YS25s\\\\\\\/Ph1kaGJZieG6HcgHP\\\\\\\/ZiwJbiNr8qAIZahR8MCV7zsV+l4AhqpFl3\\\\\\\/IpcXTYICFF3u5j4ruBTds8+nW8UPe7A7zgUV7tzpe69anCIkPmAxxdl0j54IBi02dE1oPU2ZpmiMekdLV9kqJTgSYqQiojVVIHLE3gNX8ugdSGfcm4SM1GZoIXdWJikutPZkdK2u9T660aktLIp7\\\\\\\/iO1W4u2MZXzK4PlXqb4l0OPhTLttb8ptK7VBcVSHnhynwkdFgGq3DUpq8rw2tY716fCFVUvcc4S9nvraFQJN2x6inBu9mCsRwDWeu8sBL0IjMrzR5r9a\\\\\\\/BBkepam1nvyWf0jWd2jSnpBfj4ZJRuffQS8ncokQgZXX\\\\\\\/pPs3GYfNC15HOlChbbJoXePs5wA0y5v5My1fzK81VloB\\\\\\\/R3rIivk4siuTjzMW4iUU+6ihJbpLpupCUCnoE8j7kYC41MsCTMssOzYlxsf10oW7V3Gc+KEDOW9J2MSrreWAgzjpOks9zDIIxwe438CXf7G\\\\\\\/kcxZfcMktU3OptRd0anViTGFTRCDgnNpUNe2mJQvaWS+tuFOYasb0lHQfEuXmsEemidL8fX9IMiiAciLRC1tb3ECW0UhjOhB+lX\\\\\\\/BYa88vNUTWhhk0Oi6W0GS0eay5U+MMYH0RmAc2Cg\\\\\\\/Pj7I\\\\\\\/v9C3YdcP0OAmZ8wIjsVONTyohcnNlPTd03a\\\\\\\/GG4UB7lJJG2lsyHM5NAubZH\\\\\\\/RZTIuk\\\\\\\/eybLejqSUFjajrbqArj1Z9W7rv8t72wH4LZwh8Ld8YtESVuG\\\\\\\/aV1+hUfaljWvuXfnBmvELPeuAUDVJX8\\\\\\\/knRnoQ8YjmLPfYVtJv2KR7bAe\\\\\\\/BGTixKGHk5SYT3BurTp4T96U8noPvEJqztSIZua4qPX0K8iIDYRvs3uWdvAPrjgWVxXFQ5P+evypHNP4vEcArMpk8Q6AeqpU2f7d5rxD\\\\\\\/9J+2R\\\\\\\/w\\\\\\\/wVw7NRuNw1Xz5vgSpF\\\\\\\/dpzIXy5xQL2R2NMZN3oPcuyEn+bMeDTTF9yhU4ecxC5FNNEQHarcsAz8HER+gu7ek2tF95m+5xCriZUzvz+tEZH7KA5C5fFDUQVqqZKAnUghcmP9HXYgbSImegd\\\\\\\/7UvDPtQxQ2isczkSYWRXDEVTrjFdKjlnd\\\\\\\/1V6M\\\\\\\/JUmmt8VLXWb52GnrvNltZs4gfzrMDqfshOjxBXTTmVfJGlMHAaYwCHXsYetJz2Veb9BRh+gyfiPzb5hatfNyw8VAk+iL9LVFQtP2lP6FIujYguOdlKRE28PoeWRKnu3t8GOIsz4Ua3Yz1EYZdXfIH39xEIEVkLqm1Rr5UJGbbG8ekTMx1cdH\\\\\\\/6uHBEPVfjV8XfGbvV55Pjmrfg804cldscj2nqWfqTQVb\\\\\\\/v76aHlajipxdmr5czivHs1pI5G4HhQOIltWra4x6JTIWm86xEAeAhpmJW8lBFdoh9+OHPwkb7U8G5emI+7pHB3DAF\\\\\\\/Zx493DTJAI\\\\\\\/YE6HKfABwHItFEPAg\\\\\\\/+raNfPu6effbybcX4aFHZ7rEzOYocIVoeuWUk2fT2rgYdK5+FfCe1r9eyQbJNHqjWobrTVl\\\\\\\/P3MBfPYvKtX7eRgErsULbp7q2avYRCC2oTB5cb4kmy5DkolYKpBY+t9L71yw9FJWO9JpZorOxZlfEfmvcnm9T9QOVb4KiPNTX99BKdndsx0CcJk\\\\\\\/1Dly6hDskadJqEGsWf+RhxIrI8zgOupSo0ivu5GU7fyPmpXG4Z+FuEKKmyJ+a7OU0MaQ8rVB70WO1YDQh6kh3d\\\\\\\/5RPLFj4Xrtd7BYrS5bY87UnvWHxtGRtz6f+zOSTYo8GWh3UILNArf0vliq9q4R4KhbJT78suN0Yx8r+pBDaSUCCfeIzfE1O5Vs62NhS0ymbah0nkBxb1xO4ALrF6HqM0LIOyzptE9nO1cQ3zGiAgVrLn\\\\\\\/U9TWKhaTt0ZxbRdIdzumKm52ii9y5E\\\\\\\/koi1ygAUA7T\\\\\\\/xrcN5NIKqVc3y\\\\\\\/pJLSdbSzcEcfxa5Q4Nl4lgI4NJhsiNihgl9Cinea5p23bkxQ2Q1\\\\\\\/9aOZyqDl7Dms02YsNInZiIjKl+hXTiEbvnwpvBSw99ZUU27dF+VrmR20S5GhCOFAxoKCH+WlYML1nDHAIVOtLHSBw3lDrkfEVXyTscw96dsGsH9k4+LaoNBPx\\\\\\\/s4zgTdofFZsdG1pJ8hwyTPnr8lD0A5pkZ6Qi5V+ZNxeLgVLPBL4Amqn9VXne8uq5XvCRGGKz31TsrlcFvPaddzmVbmnmpg25zn7Uk1cXp9vFCJUd9eCsNoscs6hmSs3yW8O9UxUOWvvw3Mz3GF5pvpw5K6+\\\\\\\/nErbZSb+y6P63XY7KThCT7vqJVi96COdthYJJc5eP9sjXv3mXadIVa1iItmToeikiH4LQ\\\\\\\/FjPFNfJLdz7\\\\\\\/t1OF6UkFTqHcmoWa35sisKOD5XJrF1mEyUMmipBPTUXcxFfekatFITByZ0qcg6egLjWOCIpE3sQeH3vJnMCCVBygSFigG2eKxI4Ajwzkj0u5iswU26vp+rExFkvZIEwhjv7LZXABx9EBzi44gAniS1nYaR4OUISuIoaeC2vgnBVwS7SUdghVI1Fm\\\\\\\/3KptBjv4nIdY5v4+GFyfsqJLOVEqJhGc9xe+Mj0q4odH7553nQvTSihxumWsYkM1UVC7gy1I0Wib8669bQnJg28453gZB6WBf\\\\\\\/Rxgo5grpSBY5iWyU\\\\\\\/f5A8LxIw+NIn58fLqiIOGMgP2VJ43ET2wutOvkD7mTgf6InPPTD2+eGThS4qPp6U9A\\\\\\\/40uVsw4Xhg6PLw1QYWGY3Hhqf+GSZP+dklXYjl1N0nVIiuw8ldaBLqKL7tsa73r8aHAEQgwHZrGcFRpBRCDWKnYeLaN4O3N4ISRurZH2HNl2I38u0d7FJ3YCwXBq3wy\\\\\\\/4je70D8MTiNztQL6KJe0CDDW+z8sL\\\\\\\/1\\\\\\\/qOdaq\\\\\\\/u9QcTLQdHSAW8UGBZW9BY3tFv6Oel0oPlYmxLFu+hmFoNzs9IG2u8HaezFhe3u+Abu2ibQPCcEpdT1nhhmoUokhVHkwDs4D1AAPOm6y3N2yYzPcTCQgA7XTBgIuhA7P0hEJAMImiXeejYyL3Xm6+euE6TL2LLs2lyL4MFOmUEDv0QK0u03oznWh4OFPlE1rRpTBF3l2LFOBHmB1CFZm4le0XeIQRNHfAUZrEFubgTcpJfHTOi4onjuNp1Z3X8Wr3oq5R0q\\\\\\\/P3AVhU0+7mhneM4Zxg9gFzo4Nf6ZfrKpXBMWHH8QyQzs3rI3c3vLFZRrhnntJzBC83u\\\\\\\/LDyoUywbUEHaoUPeBXvZHfKHrPQdCGg3M7+\\\\\\\/Qrlu1qfSkGjX70lVhomceFDwG0xtZfW6lrouA4HOqKbPP6uwDQyF\\\\\\\/Exe+3zyy9Nu7eG6vmYZdqpZpQiqglkZoXV7lyiGK1ZlaV3A3lRmnjK5OW5yXmNPkJsKiWPFyCKW\\\\\\\/RByIRfAwIxvkTaEyZean+cSBrD8iGr2RGLMURmOyltoPdFynuRQHmbk2Oe6xZ9UHfpBtiuiGoOfnoY1MbWHHlcyaTtSdXf7X8OnkQAZ82kXqcyOpX90R21pM7tU\\\\\\\/RDwM+WyTiXbPtxbb0O+Ivd8YYmL\\\\\\\/5YShLgKEc6GJGn0IsDI5XgMV\\\\\\\/T6my2h91\\\\\\\/nXCQr9Xzrl6JiZewweRyNXIaj1Yg7D7k+yUDz+84imJv09YMCOKepTihkxa0yuMegwF+NSpyN6dfmQDaTZu8HANPv2dEUwixBdMiCJuQjAy+C0yNggAeP2J6jZANjxbN+7R3SDhcbAhbwzBdH8ntJZyvmM7G0H3TIRnRCRN\\\\\\\/WmMnfH+Z9Xus6gJnanN3vjGmb8uTeBwWIzxQgG7mwc3oUpawKNn+2D+gaat2YacM3dNeUy4hupOJLOvQFneywA\\\\\\\/ruIbIkYOSDim1DtYstgiN3vM17tZdGKiqC9sm5mNlwtJikYiKEmT\\\\\\\/vWdM4aPv2YPnETaxB5WUTllqEsrXOKObhX897OZYFPgngjswt0JmqH2zQrsyBQ3tqXu6XkLGMCg0StqWsMtFXy0re915qBi\\\\\\\/cEHke55bSkCHjJVyrqcNkpiZGn9v4\\\\\\\/VJa0an4DwuV6UEMQ5EYR+KwkljRzO6IQd91717nleWqsoa6keUI4S6wI7M4c7lJGHS34k694Gf20R\\\\\\\/XTg8JVTmabPCFNAI5u2EnXkWWv86+I9j41xnEIfQvtnmpxjdeH8lAtxRRfGHFaEvYH4GT00p8HBhuaQZXBdszVA5OF5UM2mYB3sEYz9eJA7TI5pyREAEEexL4BUNXOl5GRLvQ+zP81lGG9i7YBa3L2MCzV6vHneHcOG8bi8e0\\\\\\\/xpftL+SjKE7M4npszLaSqF\\\\\\\/DTe06HzgEmkUmCvIKccur51edQvdHG7eRtulxQTdzMxNuYeXDznNCYamZGw7iGShER+2mMioRjWH4iVHotTNxMwVZV3lHiejbMjDlJhLzK5Eg8fKVv2ha4E6o5qQqCxBNvDMNAlWTEb4VXyU5Xpmu+JBc9CXo\\\\\\\/aMG\\\\\\\/lXtWCIHg7jHiqIjwKhLoIyuubFVOF4uVZeFZJM42\\\\\\\/585hqBkizEQEfBEZ5A+e\\\\\\\/iEhQlBzsrReaimp8h76qlU93W5g7vXtdRyhNnWt25pOdLmtfrsY9roJCUL5ui\\\\\\\/\\\\\\\/8QyLaF7FwmPjwnh7\\\\\\\/BDNM689ecV2k+mnzn7Q5eZF7z4WbIxkkIDLWDexR3KRgWcZXg1kXl4WAgGsGcI72R5foGoqbJ\\\\\\\/L+kMRoIRbHTd4GClolkANvXNrxxMNKOnvOIhErcGBy7cRB6rYQqywapgCBuvXSAaZL326o+0orEej4M3\\\\\\\/Eb1Lel9t7o7LGlolZmFlNlnIkPo\\\\\\\/uH59TtSTOzeQnQPkDFbPxG966zIjb8dvX7+NXZRkT\\\\\\\/vIHSxrPp2413SCd\\\\\\\/6FOVkFaXb+k0IHLZBNHaKX8ekRbDgdNQEW5xbh+zOcvbUQ69rw6dZZ6bUxV+p9ocsQ+BcrMorM3fN5XUZT+m+qC7RatHHmwmskejFRCfw7A13MBLFQkDLxEOKtGGzcYqyGjmzdCg9hyfxOoFT3fjA3GUROl0PdGGLfpkYkV4cppU18jGNlC1uxMbvX9O9a8Xpv1gfsunxZHK1KYxP7EHZwiHjv4LRRZ6goQIEUjP4GqFm3qfVtum5uGRZP2SKini92B8LMxkwowVXNaovnSlBi10JzM6Swgdi1N0eFcQhH7\\\\\\\/1vsmRsbzeQ1vrV\\\\\\\/5O3Z8pHhduxy0Ux1Udz7B7Dw+qbpTRJoijB1GhEPSFP3O97zi0JtJyeEIm6Mx7GY2fFLWrk3++X9u\\\\\\\/\\\\\\\/RWgdqBsaA6J1DwmTqhLaMd4J\\\\\\\/GPY9Iz1ElU1i08ehP0u+n8zmqGabWsiiA+JkkZKqAvinY\\\\\\\/qwnEe3Me9n5FtP19tFwWvfvrkKbZySLVVIl1VrYBDYMf1sw6tv4eq2ALBtYI6Vj3d6lqIraRcFXgbAVlVrjifMQvHa++pYuyHt\\\\\\\/i4V3l8DoTc9QhzB+ilU+ugg0qrXfAqe1bpvF6eAZjHhlYKD+tEbQYl8kAtllvdCZLZndmD1nlaXemAwn3kXs2NvEAjyedGCYovHakO0FK3nTM5T0VWV7Gny\\\\\\\/FPXBUVSnby3jjQiBT+0syFz82vuX11Qiyij\\\\\\\/qzX83dztpzEhn\\\\\\\/ZMHwxECb\\\\\\\/kfzo5Wb3yeNMNN0Sx6p\\\\\\\/H3Kve\\\\\\\/TkU5ZVmsMPYnwqmSxzvqNEdK+0qYnL6UxO54MBTCHcSX34d5wG3dRFfkl+ODwsjJHk9ZF4rwmB1C9Xk5WwrraMAR0QA+ezCFEWvLSuKi8EjsW2FDuQzwpcH7BNFWnjckHvT07k3y38azAyXUTJOo+SbJ8VuDX4kGfCLChina+aFJ6ntuGd6knQS9KSKZjDKF\\\\\\\/bAbpij4TKFUe7Lh8WC0QZFFHJ\\\\\\\/sa2yngYHkRaKGoAmzhbRkjDZ2f2ARitXxhsK9r3TsgyUnQUp1Pk096gQAK26PFxhfFqHTdlZqepiruS8W7ztFJRuh5g5noAcP+d96o4hUNEM+19MMR46bPGFP\\\\\\\/J\\\\\\\/rsF0D+yexWxl1y9fVjRaYX4bF4kRGy\\\\\\\/hV58CWIBHH5DltljYWGwe7dHanzxqdxOe9o6LR\\\\\\\/eA\\\\\\\/owUSJ+hrkWD1VZulpmy0E5UEe5\\\\\\\/JT8bZV\\\\\\\/riRy+qyHc7dTuGkvYNO050BYq3GKDD0RmbPwcCKhmGZOysjty2FWFYsQGdryYLPi1dURlA4OYyfAuQPnjQLdp1E7w2X6luV3DLLOC6X4JGi6hx8mUW+0Ev+jkirmgXhvhv\\\\\\\/RQbufMq9Oc02OBZRPSe+iLCvU8QRffHQ7ydV\\\\\\\/AE7ZuwOHManoFKuPI+deuVFtFUKgWwgKx0ppFnolFuzpj\\\\\\\/d7TeoMt\\\\\\\/qRn5v2Ulnopqii2TZXtxoQfyax\\\\\\\/GQLjeZzfqV8pVUQLHwxF66hCl5P8pzg8T7lZ8wEXMJG0rL2L7FbrH14Sqi+2EwGD6+kCFC9s4UEvT6SsSACH8ZDZ7tUO0d7bHZwhk++Rp5s1ukOEYtysweF+VJndmqTuL9rHJbUvMIIjJPqF\\\\\\\/BoSjHoFJdl1bNEuaOYGLJnt\\\\\\\/6Imgqw6cdIPM0+JWDnFyVEd5Y2qgoVEKwWyraSmH2TkEup7rbXO\\\\\\\/l7AkmXTTPU58AgDGsbK0dJqXf+qIQ9kFD7jhL8fDaM5c+utd3nvXq34DpGQ2Z9+9fQfJieYxYmxWQ8okwl2CyAVjOKXrTFuu21dJ3l3ASRrnxSyoSd3Eec8hacxl88c2AyfqWSIqWj300s\\\\\\\/blBmtdZmC+OsgS9DmiS4WStp1bW8N\\\\\\\/EgPCu0bbw6+gT\\\\\\\/ut0ScHf3a0XdsAf7aS7aZ4iehlyKj9ayqO6fKk47By5z\\\\\\\/wYc4ZKizRuQmFC4PZNH3\\\\\\\/hQKyYHKgEx1lpHjEKlIVVsDCoCo3Tox1la2erDpWYDhNzedmWAPsK6HOOJeSyiEPTum3V+wPiXLdenbTnnzNsyMPg6L99PUl4GE\\\\\\\/QZ+61NeMb+nWQe0DQ6O16RYSrGTA8acn8hZfQspuaspgjDjWq2NqzW3Nd64ISItStl2uIh\\\\\\\/Bdez6s5kEMF+BS2HjUFYVRY\\\\\\\/VFBtzq4fDyShpThkquAGwpkjivYy+sQEudZAJwjE4h2q55Ovv2DfZy+yKKg8adnlXiOR0hMmpenKlL2LJUQLZelWOmCUCJBeBLI21J0MQJCcSejEXg6YntlVCih236V8+Q2IZWJQVyqNROl+VfBKGxyBSj7mzZ9P5rcEu91cg26t74GdfQa9yzAIX9EbmsYLufX8SIAkiEb4VIBo9gJXRLiMwj3+OspfIz3Fis8RzrkE2pMAzq5+8wka1hivfjLOrU0655moltgNH9SUbkwgR8V8h6OZ1dnqzrxchA1Q72K8VyR2wD9Xaj77w8xdRiIYpkd9WVlsWO5C\\\\\\\/ZBztdkb7V2FUlG\\\\\\\/rHpbD4EJ4+XaPW2GsfL91SZ0iOWf7UKyed5pEQm+z1g8MP9AeYbSxPnFn6JJ\\\\\\\/zP+KxABbT14KASRD5oiU8sVRfD2eKTSUdlfWduZ3ivFdyBTxawkZ6Vc2nblgmbkGKmBNQE6qj7yCFZZJgI4GrWau7spvJ0LnRuwGlh1P5IM+gnOrhCB+yT1LN0KhKZAnQ1f6pnTJZZt7doJjpmy3FOpJ4fCdw2ABI+EQDvQGGi33TMXSByAhShkJttD8j02dIvrfywpLo47rtWRZkYCbk9xmDaK43dIaXkhwHBB6kvWrohim7S3og28+ehXU2b8aHFmAXmOOt1SZEeLEv+Z0Lp5dGbOkB6sy2H3c0ETVYVN7uUtATi0wgdL16FHjVt3OeIAHQ\\\\\\\/VZ3h\\\\\\\/oGen+J7ThqNgPYbFDGX7vaeqqf04gYxTX5g4N6+keCIGDa9C0ZrVdSyB671xC2tae4tfZJD9VBzDmVewwQ9WdKJz0web4FtE8PDd+ZKclTG0CxOGfynB1lFV3x+ucSfI8zBN9SG2y7heNV3376pDAdUR9\\\\\\\/+CP4fNsM5jn5X2QoaFHjQEZJ1NJTs2DenycXrbob+SV+aMgfZfBOAg6SGttU5qyrjDG+ixYgQSG03VCrWPVkQSKaoq3SlaiEnvZLCbVWBVelLAa9EehXG+G5E3pQnVlb2bNYlkBEHUPVHJlXfWYRdrqvsXx+2iZ\\\\\\\/naQXt2wcP4l2CcXXD2OH1HyaccQEl3K4Fa+KdVQHBJgO2bEXboGCu5ECvuvVaaFA1salvs\\\\\\\/Y4ntftepG2SMYTwrP43Ka\\\\\\\/PK4AgwUbs+mPhZ5Z8G9Wp9NXI4\\\\\\\/\\\\\\\/bdVFvbo6XVO9\\\\\\\/viYk+MIh0N2tiDX4raj\\\\\\\/P7xzqdyZNjEjcWNu31al5KSx4iJ+1R+swpMbBw4a8vXfQ0TBUJd3x7\\\\\\\/NE2y67N5p9rnz9kO7Y9MUN+RkcuuBZlfohjytBz2cAbZRK8a7jezvZWDkTXtbHDpWHRAONXNzAuYZFCkO3EuLTUcI1S5qkMFlee6TxY2mFdKZFFDhuy3t4Th28X1RPUA\\\\\\\/XJaA2kKAq7tpNHR\\\\\\\/4K85nm1R7juIQuCR+XYQiOMWrkfbCtxlaGyDCXHXstqXxhTooRhprgG8DJ2Iay6+KQFbZIzY6xmIRNS9Ji3DraXSN4e1dov7bIJBNP46I\\\\\\\/89TmEApyYC\\\\\\\/MIO0vl\\\\\\\/cDOlsmgbBxm8EV\\\\\\\/nqNtarBxsCZ6DV6M4mmJrNlnZD0PreCYCqoE9KrLTSx4JwUH1uLzz8vXIvY8Ct+RHuPmO+LZMQwPcs2FNEeil32oSm\\\\\\\/YeyFFTWEJPsjAzMc7+EF+pEqQAE1UYGSrbjUhS5IsVuWkANnaS2vPCHqt7IMrjUlY4VpVGhMBlWXZpSCLy57Ly915yHQDAx5\\\\\\\/Ea9PbJSK6+DO6mGvtLi6NQAqHvcobEVMT+o1pYnGZRKNXQSxCX6o+ZYdLI1dcX5ieClTSAxzx6JMc6SMwzW+bDY1W7ZMMrfZK9BTmgqnkj60VpNzrO3NC4JSrWAYGaU4\\\\\\\/+N6T2j9yy4LeBnP1NENb7CoHb7+NyDSkHRsjnp\\\\\\\/2cnwDPX1jHf56eN0xtcODuP049yo9Npiw6+oKCtA5iXJdYqL\\\\\\\/2KvqpUWPpuqn7iFDNGwHjbWQp4ye+dTL6OI4SGfdbIBm41HhxsX4uPBcPslAi4UlwZVn7bG9fhnLkpduG+15Bz35cLee1YyCZivGU5r9pi5Ol5sB9x2RM7kogURWbcvacMht3B05B\\\\\\\/f4A2RYS5WrqrOPUCXdfFm9UT6\\\\\\\/\\\\\\\/HxHGYyldnOhx\\\\\\\/8TqoWDChlYuLPmJHzFeKpUNXc1b7UhCIQRGBBMZytWwgwwG2Wr63655G\\\\\\\/CGkE2EgstHIHGyAHLG66vS5dMCAibLPi9bR8Kmxj9tnTisrjsjtTPzDYTCx71Pn5gI2y344A7Zt4rvNZLANTO8paXkFxovC2vNBHYiMZXkycgQotY6w5JdNHupPk1zhVX1GlAXPOfyIPRTxl2TQ5xC+lI7cciWF+1uV+60wY19KTX7ro+J2yBtNDdGidW1orqE1q3DVQ+YEJCtvkxPlvDBH3jHW0us93bKe2oxzLJJJKj7cxOsLpxgetTDAtenLYvPhEeCz+WOADTwg5kw4CsvZz13N96lf2HqECen09FOoU16kF8aWv64MqjLHvgAklekXqfxgbphas6Gfpj6dTAs9c5NNzSSojeIy3mNU9mEdJy1V+\\\\\\\/ooERaox5ZMHdK0ZNpLGU9kqkIE6+8oDThK41YYGop5A9tdauWoAlT\\\\\\\/0mgTjvHYSSAB+IDvObyVtwMfWwJhAT1wjR8dnP6+SSHgSXKpmEeAALbZk4All52frPMBxFEVo7BfewwQRb0Bbbjo9F6vDVEf2U5fjID4b2ZWIwnrunX1TAVXeRh+qp4uXIu+ZTt7FIcLk++hKU4a0GZrTfCnGem53jGk8cUbUWYhp+UipoeMfESNpXFjHt6nCan5xAIZZAVAW5v4ty2uOaWvP7rVA2rsmzqJLMOhG4tQBPkunSMjxFvD2R9AsuJHg1TnvjVAsnlgLyx\\\\\\\/s22w8ZOCzWU9Y8mzuv9j0miez4AWDzPFPd9sydd6FqMUQzb2bAq\\\\\\\/RCS4x\\\\\\\/yrTh1EqBu1u7zqn7Mg6R+n9oLD0RbtIoqgxMbVK2mc+rocEZcRJvUowXp98EgtCXEmZ8QbR7\\\\\\\/6MIQ5cQn0AEGMgxWzeGeYpvN3ZFbHrlqE5mYuOnp5YgrOsE2UNAgt19Sywsuc9hCfRdrm1kab3vH43804F37dK\\\\\\\/PFnQ3VHQCi4Qq2MU56VPgOUnOKrngtaKarf9t055DcZRknWHzqpZx+H2vfdlppt34aN+aUxxu0YsDLnRHpDIHEJwihg+i8bF7lU+KDtfNTT+Avz\\\\\\\/GgDG4tTCD2poA526GpgfnGlk7N2a3yYS2btqc8J\\\\\\\/SvS1GkAl7cvQqDRj6pr5lKw1aPn0yzIRczzSacQ1xfVxPWnBSwBTcBYXejp8SJGnAeqUfWRTzxn7Wqr2aAgExOUimiGsJHgE\\\\\\\/l1ti\\\\\\\/4I6PY6sOjKpfD2Bnx\\\\\\\/MQrrZcdp27L4OPn2sxcmyz0pp2QRCrynAfCGYIlh1fBWxvU2kEiUDFRpGk8WiFm25BLDDbR3iXEshHzBFPxCb1b487gAfOh6ESo5JoYwG8558vv\\\\\\\/RwLgkmKPPCWHONDPAxV1XYmaJYWn11Jjiazak3LTLInYyxyvva90t2lbRCtCh+LsovoCatclPY+TJWEVu\\\\\\\/xLUzdmC4e6uZUEeKdgFRv09wkoXRu63PFrlhDb2emeF0lie\\\\\\\/2OZhSf223vWk1xJCCpBS6WKqjlNQuWTrPDvAov1wdgERJh23nnsF6QzprQnVy7bJyiuQjzc7LMWe8yg1Q8t7T9c37pmufuz5lmKjtqIDGCKGofXdZoMyVt1KyydtsxGvtUifJ1bsoonKnfYPo6ZRCOYaV\\\\\\\/aY7okq3WREippWQW7xx1TmWiO+MF7kS3jt7tRLd2QWeJ1m742paUyLZrIxYRu5F7L1YYGCSEzOVXnTHz9Q1Gj\\\\\\\/XheCbCkXWLhnh1n0f6ajOZ3x6HiYixCPf9VoW8UXeDOlKOLnxMh8GPPyIuh11mLmqblgQSyUe186SYBuKe1M9kBD\\\\\\\/Z6FPJ867M6Q9PCIHf+WU4c98VzCQlsi\\\\\\\/Fh8+ev72vgr\\\\\\\/CEERAMw3jPvwtYh+gwcPYzc9QZ483MVhtA1M6t+VzUxu5uVecYsd0u6RFl8i89qwAXOQetg9YgV0euKjz91DFXdpCZXUeNIJ1f+j\\\\\\\/IH+92UHkYsz0JW+55RgqYNAwE2XaEc21gSlooPy5Z0pfSDJyz9VlxrEijzT5bAz8iL7fQUy4hiZs04FB82bs1lNdYGjRsr4aLLnEbdSOHydb0PP8SIuHl4CViVt0L47YVGmcuV\\\\\\\/rl7rXArPDUDiZbluDBqETaWf6WQPvDOOh7lRzvD6piR1F9PIBW3kqN4FDXQhnXorz5JhlT433GltRhcy0poz1YXYNRgSPdSbXi6hh88RvT67rhOf\\\\\\\/52sBb1cvA5oZrs679frvjulMEfLmdtFQpTusGMKa+kuMrx1AKNB3L1KsYgdWskr6VFuegmveKKcNdmc4DLnUUeVnsGpMZ+zREOBjnSlOfhjz1R8dB1yCRpRGvKNrsPKTfEMBg6cq9fKN+twYRwjbJpCB3chKBy9PGLGaG5ec6EZahdpn5SJ6HnBQqac3bVa8GM8LP3Va8EwXdkOTBIZPKC2pTkJwSlEdPtUtQGgNd9oPbz4muFHqYZ9nZXWayJpv6eRMogfJHwEi5JXBIpCANggr04+LUPt\\\\\\\/1L6nZigNbgkvSa41hETRGtfYYapzBY1LEr1yAFwo2RdkBQNskuRqOXlyF1oDeT4CthEDmRdoSyfCFZTZC9KaodqK46LQMF5EWx7KLWM8FkRdUKMaEvnmIup1EEoT67eYw75jOHOaOTVtO5Ap0sLj+d5mXUhOlPHHuPCMlviVomUW4ApsRW9WRVEnk8zR9elJTXtPgUHMsj0NfgVm0gWNsG8L\\\\\\\/IoWDmMAFYKsc\\\\\\\/uizU906h5GcsPvDaZgGpACcG0c1pdpBu1MMyuHmt8UOmh\\\\\\\/rmYeNvp0Xp2LH0kPVLK\\\\\\\/3hJH5D1yOajRQvgNd7rGhapNN5S610RHVIN8IEeKoAHZh25vewFdMYYLGCJOWO9iazunqFPPISCY7zvJ5dRgHsF+ieIaaBtvJ1nOUQm+64G1SdBjjQ5rqQCIAeuApSu9GSOoC+nM0GlWopMddcESqPnGCY3XrGqYHXLkEylrN6SV8t1Wi0ptJSTZue\\\\\\\/ml7cxzNMpmz8pnCI+esiPV56CKCKo4NNXLzutqBuL6SLoHLVGv19AO2pYTa+pLi+zonNQ4tNU\\\\\\\/Jy6IrMycn4SaDwoW15rOY1Ijy4cbZ3TSMGbeQAUZTMJVyVcl9AyfMOrHfue+iQsVWQ3zHqRSMtBnWsQH9Ovgs\\\\\\\/EwIuxkEPLzObmTbhiXN4+jrQHPpQ9hFLEg3FdztesHt9pQVuy1riRQIG0oWjeUQsCnPNakqb1We8l8YOR05AGO6xzebx3cvD64bBxbfd69haaqqEgcBxE6dbNHTjMTx2+xdGIBk6Ha+zZUIf1hcqLPidiHBKVTHYEG9TtvyYLEVB5+pOz1gSc0raz72lLWQSKdwRowtbWLpraWZTDcpUdLK8ZfeJ0GKRPAiyN+wiSXoX\\\\\\\/AsMf+479vcIPo+8u8UKCzDmBvmTgLP2zCE1MWcL3OILjLaa\\\\\\\/VVUkt+BFxOviVA2wDFYYaMqgXSn0kpO\\\\\\\/x1aVY248nSq9XBzWugfImoHpxvWf91DyF1xcuqm0DoDdhOAJSCd2Fr7KJiagjuYajNXCl0+11vADeVWIHVXNCSy4mIaTMNHaSEauGWpOil6p4Yzm01rm1iefI+CdFYq8xShZ+Kf2G2zXa\\\\\\\/Th7qZOMlJF+d9HwAANJdohNyZr4ciVSzDB6KsQBMUDE3irMC+bcJZEYiJhg1RycMjX7CJIO2LNN3pDyQrr7bFuJkaBFQYnwJ20sffHivDWbBIEgXTC5c3lEmgZjvoJwo1xe\\\\\\\/zDlCf5NtgqKdGzrlyUcWopyR8nqKTm1so4FX9N+IvUwSuCQ+rhrex8IRwHbI5Tu6S4OqP9yfQ+ZKMKvuC0mbPMYig60xKMIUtRDq13dvUUyMMerHMVMNJAMIbPvWL5UOqKUGPcmiyhEbF8djz9+TW63oDGvEniSFPutbIMOSw78vrtmEaP0w+7I2aqPsQX22XPQqRkjIY1O3PYOFTBqh5tatdEqBpM8lBX88UGdWLdUGjoJDNWve80nuKGsfU3dgyfWCDJLX9WnCm1vSzSn\\\\\\\/z1CdakM7BQmHrLyMMt+DCphODjJlb0JXfaEK3JIleX10UhG5UlK\\\\\\\/Ne0x2GfDep2vQstM4lu\\\\\\\/\\\\\\\/b57FfzJP\\\\\\\/a4i2ePlXNJcKmxiasKLbTVE9BohbHZNnDMJWIFmatpTm0LshyLZpQpbfz9A\\\\\\\/k2orAM5LNxCdgpFVMmnwdBEdWektyL9zMVDi9g3Tul0CjhQHSnaHFovHRxTqlxit\\\\\\\/neykO\\\\\\\/BFNuGHfZDMXBX\\\\\\\/D0QHCUsx8pfx2Bv0vYDQ5S8wlovqaCw+TNL2WtGwfH0bPtFetxuNBk6Ae0So2wXQAwdPVmDDlRxuNDm1ytB420K7zyCaZ69K82NHX60uJ6TaEFLwHE6+r93N+bZeWoamNl3abirV6CoWh6OK1BmiddZNYlacyxmJzTHgYnmD1bWsUN5ZyPZ9iUK1Mn7XIPqYe2MtQ9N5oB5joaRwOKY6gqe98qInRia5KqjwuUnZ7Yw3M23AQk1mMytBcaHVPrmNVKENwOaNwFCAKsEuQJXERik6inWp7xJJ3Me2Lju1ujy6yimKSh2CIbENTPsjm9mO\\\\\\\/MY6ZL7WFn6G\\\\\\\/l5Qysr5hWJDXnpmyFk1pCyt+tDf+xmX45VwWnClTi\\\\\\\/AhrKGi8tW1Fby0Cz9qqcSTJ8VhUBVfwRoQds2\\\\\\\/xkpaHNwwYyfbqc9lGX6k5XQLPv1ZX9EeJ6jzHnTpN2\\\\\\\/4C05LQb7rH\\\\\\\/c43blkuYecVMORATyLU6aLzVbqRq8LsTRPHgMaJmUmR9W41aI7UYcpH2Xb1KmM5atHVIwR5NOlQOgCrtwdQolkGiDL6ChncDpC80w8MBMHVQF\\\\\\\/vpNvvvmKjASV3GWRFsBQgjWUGUE9\\\\\\\/11Qt71psTEs4ZiEhZeBH+LbOiOfGa3PlIJStOT\\\\\\\/vqDzq8oNlCIyAzujXYiIOIGU3IDnKj8SPCecnal9TPn89jEFEWW7AVantPdSlGi0tv5cc+3F8N+vsqUccxGqGeVcp9sxsIUIfxFdI9pj1YdJRDD9BbmBLtZt3Jh6wL1WXudR5atcOU8mV11GRjjaPgWUfjKriIcCZ1L69gNJKDAXpDlccc0JLhSHFjhJ2uekq+RbP2e9g+RVEh2E3jVomukM9cZTloXZ1SYTkYRV+6ZQp83sIChkJ32B\\\\\\\/Q\\\\\\\/B0l7Mu0DJ8a7h6v1I2kMIYdl\\\\\\\/sNKECDSnOCg5GvTeAbJhInCIfrFU+B7sKYO58ZB7GXsP8gXTWsOyd7UVLJdoENUAbgXbjJPnJykGI9XX\\\\\\\/6BqXSsHwL+ZnA+7tRVnu1rqpvMGvmowAHic9Fy29f4r9qbVv3WSWHqVBx0KOEaEejCkLr37O4MI82FR+aZnw3+dFwYzuES\\\\\\\/0hqM4JwGoSHpR0Psl3SajHWS+dqBvkadzRypWS41Fful4z49GLqnwldXWngDvVQjOI9zK3xw5KGr2MadnA\\\\\\\/LWkaHmLWDCmQcxo0kv9CLn9mBLXZznwenr5hTAcXL5q+qt540DIT4L3MWhTWsiSGefkJMn3ho1SGL1qKbFLaPWs2kRF75AO+Wzt4iKNP6QAjBt4Oumq5jEbSRc5ECrw8Ws+olUt+toDUWwR5izkfv\\\\\\\/wa+jWFLOCNHL1NdgFg1PV34S0lZGw0n1wdd7EaRdl6CancPgEz5bcHhbGLq\\\\\\\/7AgKRTl4BTuXWBL9NIn2UM+lH6cwFHkXvR9VJnxVyuv\\\\\\\/YLKcvrd64oIvaO1XrSvm6L6OEEHFPiJgZ5+o7Y1bauSEM71oRyTNPX+sm0qpxy65\\\\\\\/8E4c6HPRq\\\\\\\/9+hh2CcOs7egomT7uNiJLvab8hXZONPQLowV7gLlU4eDtsSHXBg3WbijLxS1E9aN4zP3Teq3aDXKlLdsjbutKaljXEO5qIqTv0th667RTHU2OmouRxvOXm+rLQxB5cLbMBWjdNU1r2MnifxvS0O+Nd6vN5nucpSJNdC3SsCqKY\\\\\\\/J9S7cZyhEV6BsDzA4x8kU3vDsyW31tp+3Jjl8l3b1X8iPTp7Wh1Zn7X086GHAm0kekC1XOnyVGwG\\\\\\\/vzmUWM+FeszX6qv\\\\\\\/Z+FDf9N69Ce9VIJTaJUp4CQOm3pQSQ613hrcanfesw8inXMUddKRVXFDKlrDdsE2+3QpOkQpQaykSSZCfjaHPHZbMDXLzrwVfgtobe9gP4EtzuoPSF+ka1\\\\\\\/gOrl5f3RUyrwfKEJFDRlUgJp6Y8\\\\\\\/g\\\\\\\/a4Bh0MiaX3tSDq6\\\\\\\/cdajGB60f1N9qZjVEu4QNrkL9m6Szrxw\\\\\\\/gKme8uchcFV08uv\\\\\\\/DFLnze8EcQkLwpYPpQqT6rslzbb4+JyY5DEZPbMzXdGZV4t8rXZJlXdu8LvT85W6c6AoVoCgr\\\\\\\/DJm8Gs6+UbOEbWvdHbcfRRmK5gy55GcryiJn9KWO5I76iTzx\\\\\\\/\\\\\\\/kWiy\\\\\\\/eVydBr8oB8Xc1rXYRI+DKEpFFhwVzqc4jwSWuOdl3LNwiZqpkD6Z5puYYlnVyGeUFKRyfTZyaY3ZslxpjT7oQLFk4P09+uP7W9d8I0pZ8o8Ds\\\\\\\/bEN\\\\\\\/5BlgjosfU\\\\\\\/G9M1DazfEiuhgn6Qdeqnt6QPkoeGcT9F1eetvi8DehRjxbAhtFMPGWvJLy07v22FgRf+RzRZq03C+E\\\\\\\/jcI8fVeyBey2vsqVm5EsZ4Upb1hJpgtu+ZRBEOwfUv1ikqpBjNUZ3DMBKMg\\\\\\\/HdrfDvzNBce7DTx0A6f+1z\\\\\\\/DcHf+IPWbShtVtM2dt3jp4vdqCuLrLWbO8KYifK7aTpQpGyxChOQ9s3X2kK7rowAAlcq8XGu45aSkrTIRIIzU7vtQDnBtt33W5DouWVcicM46mfks0xrmVUvTHCNOGG0rV3zZ7cqrSfcSV0XO5mVtfJ4oyP2pv4kpED3gmSQ7Q\\\\\\\/k3RakJqRRPzodui3w9\\\\\\\/qP91mFNwQOYvpHNozs145xs\\\\\\\/TOnNlBdd0MZkFDgYjXv9HppnQ3nVv8D2FmQn6lImRDqxZYRwjtFHAFaHLeNXRE0Ulmo78QSf\\\\\\\/cnJD6BMy4hbqY0oeEy0kzb39AfJ+xb\\\\\\\/3bGu0QLEKJ7tXMjLwvvdsGXdJcBv756T6\\\\\\\/Xi6tXxb13IB7Kut5bgumDlj3cJI7+QsbH5\\\\\\\/Zlc5mCLScdxWT26iJQpB43p8zPgZCzoNT193fZVBXZuhyGFiHtFFcPxv\\\\\\\/eS3\\\\\\\/6C8dW4+6Y6VCy5MaIxiYK2uy8GBmo9VqUYLnFcDZ8ERsq+z+0tTwZi\\\\\\\/JvxUL0CuLLI4k+5WZ75WVUFNr1N4wz87zLTCWvcR9AEtkEJ+Ct9eEFlTm9dcKuoloSGBP7tXhIQgBPj0UjZTrtYlqeFaR\\\\\\\/l\\\\\\\/Idg4PSwm2Hg6DoyfGDM\\\\\\\/cf3qbr+jZptFvufcvoBsXN\\\\\\\/wrw9i2qK+b9FvEMWD78tihS6fxAhixZrWJ3JjMN9rLZAQ6TaXjITBqc\\\\\\\/ZSJX5hVekPfMtAWMMCHx32nYtw4TyKNNgYY0076hgeY7woI1WPvRJZ1O71kGzprK+E10+oZ+M6XOPNKpQtHclLdxsbjj\\\\\\\/yFn9EEPPT9N8VCF43EfV9ACHu\\\\\\\/u1W1kYuUq71Wmw81QL8klCd\\\\\\\/3wjU0g40n\\\\\\\/dUZidcPKaOYTkNjUAjUjSXqVbOb1\\\\\\\/2EZB84OnyB7vxd3qe2bhm6JOSppzOGRvnP+YcLeJjdgUpse+tgJ5iJe\\\\\\\/OMMT6FWOoRpvn3vCA\\\\\\\/ggh2CXWO7iYyiE0bRzc5ZaGOuOD6YkplI2b+0970FQQvzWejlV\\\\\\\/ZgN79mSlOnsoBn6F\\\\\\\/1qzs1oqGYuuyChu7SeKvvDyOJH54fIdJHxVuFWhdW8VHtPsufTx4H6mqePW3cfeQkEs+Ym0QG1h0qFcgEZebszxGEQzQ5GLNovErzcVEUbOwrVZelUuJwVvVIWRzECxtygE2lxIRthE2E7A1cQYy6BrdKZSg5FYsx2U4Qh6RDXgm97qF30n6o0GvLrj\\\\\\\/iYzlPBFdkAB8r8B075RYxT68kvNEL\\\\\\\/MgmCQhspyPQF9volC5R3LzgnJvchZRKtwPTmc6gXKshLxaoP0Q4dxEFZ\\\\\\\/eJgDHKapNwtoChPsD4b3fmsyut5QY7W8KFqPp9Xo7o5aHOadR4raihXY4BYvyDGIeEkT31SasfmvRHwhvMJEkPTFrTgmg4FRP2jOmuusQ2idW8gLsVgz1JbeiOsWJi27e4RzfJbaWwKmlwjpdDVH2W+KsrOFNc7nlcjIRZG1Srbm3rheKcnblqJ+nnp2E83igTORP+K0hT4VoG8ZcrnDwUZ\\\\\\\/oYAdQ3twfiKCH2UI6tTmKSPOJW+ClmJGE5o6YoZUpOXIgwTQHf3v9G6XBSo8KVi41FmYJo0DIWUu4k4s\\\\\\\/jsuQl4kcz\\\\\\\/OXYRKNo+9uWLSsSJjEOhF7VJzPwrH8QsV\\\\\\\/H3H\\\\\\\/2g1lmWM5k2\\\\\\\/M381QH0\\\\\\\/Q2JXJcCeRwe1gtQQMV5IEWF5SOq6f+b+DwHA11+xeGmorge6jVl5uJG6fJKlsXSZ8UgJ1c6taf5QOHqkky1EiHP8UX1zzUhF\\\\\\\/VXz9JQHoTpYvk1qZeAxkzFkf7kvyQNlXRjOoJi9d8eML\\\\\\\/U9IzoiUwYHZX6U6ErCNvcA32+VIEoRIbGf9Sl4vgT2C9\\\\\\\/+M9DPbuB\\\\\\\/99IMdnd2S1lx8UAlgQ84jTRyYMJg0mW56lYeh+5Vz4lINB4XS2jSq04gf80MNHNb4+3A7TJMrDx5PiDZwZfDpW6c+Q3UX\\\\\\\/e+3tEsOaJuiQNaVFKUMcv3dQ\\\\\\\/moM+Z9CR\\\\\\\/KlD63VxblvPdN11IS6Jzi7PXt8VS+axPjVDvEeYjZ1hEslqxuYyN5bzmCzjAof91HuwGKc6XM\\\\\\\/XuPiQL9ETaVqoRmgRgU1ZJI9nVqfL5Wp58mQULq98DK49vdOchmm9T90IN3n7\\\\\\\/WzOsnBjrnSj\\\\\\\/mUBmgfmIowa5nDR3sV3Uz7DWExz7TNiY+r2QU\\\\\\\/WCnZzvHWr3zGs8ZWcjcwZ4NDZ0voo\\\\\\\/DwbHgfRk1JH5pDv2q\\\\\\\/jGBUd1z7x\\\\\\\/5Wj42YxU2CbcAyqpDo6lrurmTwBpDNRHk0RMw8zznWfW6XZgOmuMzE9BH4tNKSRrKDiTa83qB99\\\\\\\/C02pWGxYpU8nlYXVsJzcP4r5ZwcRCc+IQbNAGq9Zhwice5jDVbDjU3qhGhHvAali0R7m1on1IsQW9JwgrXv7UWKjWrlFfg==\\\u0022,\\\u0022iv\\\u0022:\\\u00225MrNG1Vh+r4PFLA2OBnxpw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d7c2ef6bf7825cad16f66f1f593929e933dc4c011889d1ce6acb295d5c085d37\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lqhGz9FrOfuTMQOEUEtdsuf2+h8tVVgoxBHINKTno7p0gRIueoJTC8EOvRpMaMXFVLeNVKDfffjKhR8qYhQv1LMhWrwMlvW04LGX8dnhJNuqLK3tMXhn9uaC7yrCYRiXNN0GS3s0fjA2qQAEtTj\\\\\\\/z1WVdif0ZdQbY5New7ThWPVoLvHIvxBSVnigVfEc7sJ6\\\\\\\/4BWfhxMIEuhiJDOy\\\\\\\/D4OpXkLYmGYqC8P\\\\\\\/Suzrci7v5mse+z7uaxiFAkF7Q4UMlaHXzsJQptL+13bNvd1qBlWXzWF\\\\\\\/5WREHML8a+2u1Oblm78m1EXqfD3RH56\\\\\\\/LX4qfJUFucx0I7Zy7OzlsW0ZiuWsCVHZfkfOHi4rsDAG7BUucxhTRFwm\\\\\\\/PIz9c75jDJqm+aQm617ypUqZirWyy6YTuvz1dcftNhaJ7rZtpjomvopomLcFj4sKjVSFqeUJAyX25qOxya\\\\\\\/K5rq55mxqrmW7rAdglsKRDAO0IoPB490SQ84TYkiKcJyKNWZpfucorcfLy6+YD6Vlf5Y88krGfBdPHWAmti22xllcitsJhnGagQD3fDL2KCg6Kk34yIPDd858IFGZtXPw2wI9gpv5p5dsyIzs\\\\\\\/CK4C+e3Vl1MPASiWZaaDid8sNAKFSVbUTihpEULYRMR0zuYX0sK7k8acfNhPue1mlGD\\\\\\\/ara60bmCdbPmr0+TlcOszgXM\\\\\\\/SfLyi3WP7fGhfx6YS9JJg2V0yXackSgofEUFJ77Z5TW7rTUp1k\\\\\\\/35bS8aLQ9iK3fO7Dzxg9qFHa15nrCOov3PrxKrB5ax0IVO4tEbhZXHDTk7rm6opYcFm9vuftvL\\\\\\\/XEpAKsgdMqEoLrpiJcluCga5CpvR6BT6xaHQ6RdBqcWVqbZusULdzDfGdu5Pv9O9WDQf1rVDqWJALpz\\\\\\\/iRaB1qinPy9ZqUauClKKwlRglACa77mTzyxGPOGsXtqa5Qj+tL\\\\\\\/zCaFfouHNcYj5fSFsHelOrkpRt6wrtPn4emY+uvSx8NthkcxFDpqTU23h8S5tepTu+cAHNyYtBMh75MFWvaFcEgDCkWJlQqbKNdeR3qcN6VU\\\\\\\/oFTsoR7a3stfNjxNETV1\\\\\\\/I7gA1IR\\\\\\\/+ePS8Q43\\\\\\\/qfHCNNOacY+434gZxo72OccgXSEFUdIDR7kelGZ9liryrX8488fIIvEyj3Yr+Elvw3rb0AmuDqTI\\\\\\\/8m318cBZ4lnVzzCkt0Wjjk0n8m9IFKfvBP0M0Ps5faRIpY6cVK4Bevm0IJLRzGANMFRXzOJJFDHzXPVUGS+7\\\\\\\/kgnvDHyiOpskNloIeE7xAo\\\\\\\/o2exmQzIq5Yc1Wu0vmnUPW8DiNdEWaNKwM9TXxMe7f3fJw1z\\\\\\\/bnyihL84MXuEAKTinBOK7SYAKqmoQ8mRG21pkLiGn4gVjLQgeJAcr+m2H15DMUGdgJBHZ9w0jeA9c8ej+qtkL1+V8DZmVmy7U3RF51LzriDWVv6JTIOd4Pepp7su7sSM3rRkWdHE3VwaxB8\\\\\\\/T15Hk+L13Qaztrsz5HhKlBYgXmCpwM\\\\\\\/WKOWwvXcSnCjQ+aoJvlboiX+sxzvbpX1UozDqFLk6qLoMYYs1EpSmSYnwk4DLVhkjCU2S648z8+0pGAIhOTGePqJ2bYcy2rFNuDThGVMdk7m4CRJgJQBfwZeRhrSjerIbVXGxaR\\\\\\\/dWTmYmx7OQtGWomXw67DR8AcJyYqNrw0wMJ4nzKa5EN2zmfsd\\\\\\\/KbPZzNesNtxIBofO2CRFtogaAjIaEjxM39kxbM9dyYVsUzinnDutge4Aw46YtArIRh8plRFLTSCD2KdSAKNLnXTo62pOGuMIasn6hMkCoWjm50mWV8KMLf1klXiWk7iR1vV0IGgT1MiHbO5PaPdzQpt3IqcExzrccb1I\\\\\\\/bdC5oLpTn1Y+m4C6Zmel3NZY7fn9XqKqj77P\\\\\\\/sXokmLY98oM9p3rHyuu9vPKo\\\\\\\/z3D4W5QrTGYXtciDMbcbBbEYerRcVlwvcV6Mpvk0sVstZ2s6sUjJf+PPH5uUZBTWD5pmHrHWXcDt+Aufzl\\\\\\\/pd3g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YGHAo4ZiLYHgrf1nZG6jSg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b9c8e7f8dbcd2b7589c4e50746f6839ac7e41c4c2ebbc47a3fafd818638105a7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459194 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/457" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NxD1F7guVWVykTO_RbTtUGK4b_6QbSaa-5QUXwbVn1s" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/457" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zVg7NYlOfTojqu07_55oouwmNZ9py2dFdQdHPi6W7wM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/451" title="The SPRED1 Variants Repository for Legius Syndrome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/6/471" title="Modulation of the Activity of a Polycomb-Group Response Element in Drosophila by a Mutation in the Transcriptional Activator Woc" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459194" id="node459194" data-pisa="ggg;1/6/457" data-pisa-master="ggg;g3.111.001198" data-apath="/ggg/1/6/457.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Imputation with Thousands of Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bryan Howie</span>, <span class="highwire-citation-author" data-delta="1">Jonathan Marchini</span> and <span class="highwire-citation-author" data-delta="2">Matthew Stephens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">457-470;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001198 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459194" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bryan Howie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bryan%2BHowie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howie%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABryan%2BHowie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bhowie@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">bhowie@uchicago.edu</a>
<a href="mailto:marchini@stats.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">marchini@stats.ox.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan Marchini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Statistics, University of Oxford, Oxford OX1 3TG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BMarchini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marchini%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BMarchini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bhowie@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">bhowie@uchicago.edu</a>
<a href="mailto:marchini@stats.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">marchini@stats.ox.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthew Stephens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics and</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Statistics, University of Chicago, Chicago, Illinois 60637, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BStephens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephens%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BStephens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/6/457" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459194" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/457.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459194" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/457.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459194" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/457.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459194" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/6/457.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459194" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genotype imputation is a statistical technique that is often used to increase the power and resolution of genetic association studies. Imputation methods work by using haplotype patterns in a reference panel to predict unobserved genotypes in a study dataset, and a number of approaches have been proposed for choosing subsets of reference haplotypes that will maximize accuracy in a given study population. These panel selection strategies become harder to apply and interpret as sequencing efforts like the 1000 Genomes Project produce larger and more diverse reference sets, which led us to develop an alternative framework. Our approach is built around a new approximation that uses local sequence similarity to choose a custom reference panel for each study haplotype in each region of the genome. This approximation makes it computationally efficient to use all available reference haplotypes, which allows us to bypass the panel selection step and to improve accuracy at low-frequency variants by capturing unexpected allele sharing among populations. Using data from HapMap 3, we show that our framework produces accurate results in a wide range of human populations. We also use data from the Malaria Genetic Epidemiology Network (MalariaGEN) to provide recommendations for imputation-based studies in Africa. We demonstrate that our approximation improves efficiency in large, sequence-based reference panels, and we discuss general computational strategies for modern reference datasets. Genome-wide association studies will soon be able to harness the power of thousands of reference genomes, and our work provides a practical way for investigators to use this rich information. New methodology from this study is implemented in the IMPUTE2 software package.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/reference-panel" class="hw-term hw-article-keyword hw-article-keyword-reference-panel" rel="nofollow">reference panel</a></li><li class="kwd"><a href="/keyword/haplotype" class="hw-term hw-article-keyword hw-article-keyword-haplotype" rel="nofollow">haplotype</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/human" class="hw-term hw-article-keyword hw-article-keyword-human" rel="nofollow">human</a></li></ul><p id="p-4">Genotype imputation is a well-established statistical technique for estimating unobserved genotypes in association studies (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Browning 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Marchini and Howie 2010</a>). Imputation works by copying haplotype segments from a densely genotyped reference panel into individuals typed at a subset of the reference variants. In this way, genotypes can be estimated and tested for association at variants that were not assayed in a study. This approach can increase the power of a given study (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Guan and Stephens 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Marchini <em>et al.</em> 2007</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Servin and Stephens 2007</a>), find candidate susceptibility variants to guide fine-mapping (<em>e.g.</em> <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Liu <em>et al.</em> 2010</a>), and facilitate meta-analyses that combine studies genotyped on different sets of variants (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">De Bakker <em>et al.</em> 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zeggini and Ioannidis 2009</a>).</p><p id="p-5">Although the statistical methods for genotype imputation are now highly developed and widely used, there are still open questions in the field. Of these, one of the most pressing is how best to use rapidly accumulating reference datasets from around the world to impute into various human populations. Our goal in this paper is to develop a framework for achieving accurate and efficient imputation with current and future reference panels. We focus specifically on low-frequency variation (defined here as polymorphisms with MAFs from 0.5 to 5%) because interrogating such variants is a major aim of contemporary association studies (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">McCarthy <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Stranger <em>et al.</em> 2011</a>).</p><p id="p-6">In the first generation of genome-wide association studies (GWAS), most imputation analyses used reference panels from Phase 2 of the International HapMap Project (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">The International HapMap Consortium 2005</a>), which contain a total of 210 unrelated individuals with ancestry from West Africa, East Asia, and Europe. More recently, Phase 3 of the HapMap Project (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">The International HapMap Consortium 2010</a>) increased the public reference set to over 1000 unrelated individuals sampled from 11 locations, including parts of the world that were poorly represented by HapMap 2. The 1000 Genomes Project (The 1000 Genomes Project Consortium 2010) is currently extending this resource by applying whole-genome shotgun sequencing to an even larger number of individuals (â¼2500) sampled from a finer geographic grid (â¼25 separate locations). Direct sequencing can identify variants not represented on the genotyping arrays that were used in the HapMap Project, so this reference set will include many more polymorphic sites than did previous imputation reference panels, with most of the new variants occurring at low population frequencies. As high-throughput genotyping and sequencing technologies continue to mature and decrease in cost, the worldwide collection of reference data will continue to grow.</p><p id="p-7">There are numerous approaches for building imputation reference panels from publicly available resources. Many published GWAS simply used the HapMap 2 panel that most closely matched the ancestry of the study population; <em>e.g.</em> CEU for European populations and CHB+JPT for East Asian populations. This âbest matchâ strategy produced useful results in several studies, but it can yield suboptimal accuracy with more diverse reference collections (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">The International HapMap Consortium 2010</a>) or in studies with no clear reference matches (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> 2009a</a>). A simple alternative is to use a âcosmopolitanâ reference set that includes all available haplotypes, each of which is assigned an equal chance of being copied <em>a priori</em>. This approach produces relatively accurate results in a variety of human populations and has therefore been proposed as a good fallback choice when the optimal panel composition is unclear (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Guan and Stephens 2008</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> 2009a</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2010</a>). Another class of methods tries to maximize accuracy by weighting reference panels through cross-validation (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> 2009a</a>) or ancestry estimation (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Egyud <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Pasaniuc <em>et al.</em> 2010</a>); the Pasaniuc <em>et al.</em> approach differs from the others in that it uses <em>local</em> ancestry estimates to provide customized reference weights for each study individual. As an alternative, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jostins <em>et al.</em> (2011)</a> suggested balancing accuracy and computation by using reference panels that âapproximately clusterâ with the study individuals on a plot of principal components (PC) that capture genetic ancestry.</p><p id="p-8">The standard way to impute genotypes in a GWAS is to apply one of these panel selection schemes and then pass the indicated haplotypes to an imputation method. Modern reference datasets pose a couple of challenges to this paradigm. First, many panel selection strategies become harder to execute or interpret as reference data are sampled from more populations. For example, the systematic HapMap 2 cross-validations from <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> (2009a)</a> would be more difficult to implement with additional panels, and it is harder to make clear demarcations on a plot of PCs [as recommended by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Jostins <em>et al.</em> (2011)</a>] as more populations are included. More importantly, larger reference panels increase the computational burden of imputation, which may compel some investigators to use smaller panels at the cost of imputation accuracy and association power. Methodological developments like âpre-phasingâ (which we address in the <em>Discussion</em>) can alleviate this concern by speeding up imputation, but existing methods still need substantial computing power to handle reference sets with thousands of haplotypes.</p><p id="p-9">Our work was motivated by the idea that growing reference datasets need not make panel selection more difficult or force tradeoffs between imputation speed and accuracy; in principle, larger and more diverse reference collections could actually make it easier to identify haplotype sharing with simple models, thereby making imputation faster <em>and</em> more accurate. Kong <em>et al.</em> (2008) demonstrated this point in a large sample from a founder population, where they found that genotypes could be accurately phased and imputed using simple identity-by-state (IBS) calculations to identify shared haplotype segments. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> (2009)</a> introduced a related idea for phasing smaller samples from outbred populations, and we wanted to see whether a similar approach could help impute genotypes from modern reference panels.</p><p id="p-10">On the basis of our findings, we propose an imputation framework with two basic components: (i) a cosmopolitan reference panel and (ii) a new algorithmic approximation that maintains the accuracy of large, diverse panels while controlling computational costs. We use a cosmopolitan panel for its combination of simplicity and accuracy: past studies have shown that such panels produce similar accuracy to those chosen by more sophisticated schemes (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Guan and Stephens 2008</a>; <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> 2009a</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2010</a>) and that they can improve accuracy at low-frequency variants (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Jostins <em>et al.</em> 2011</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Marchini and Howie 2010</a>). Our approximation is based on the idea that, within a limited genomic region, allelic consistency between study and reference individuals can be used to quickly rule out unhelpful reference haplotypes, thereby making imputation faster without sacrificing accuracy. In our framework, this approximation reduces the full reference set to a custom panel for each study haplotype in each part of the genome. The approximation is similar to the one that drives the phasing algorithm of IMPUTE version 2 (IMPUTE2; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> 2009</a>), and we have implemented it in the same software package.</p><p id="p-11">We evaluate our framework by running extensive cross-validations in HapMap 3 and in African data from the MalariaGEN Project (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Malaria Genomic Epidemiology Network 2008</a>). With a more detailed panel selection scheme as a benchmark, we find that our approach produces high imputation accuracy in all populations considered, with the greatest benefits at low-frequency variants. We also use simulated data to show that our approximation substantially reduces the computational cost of adding haplotypes to a reference set. We further demonstrate that an implementation of our framework is faster and more accurate than another leading method (Beagle; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Browning and Browning 2009</a>) when imputing from large, sequence-based reference panels. On the basis of our results, we discuss general computational strategies for balancing efficiency and accuracy, and we explain how our methodology can be combined with other techniques for speeding up imputation.</p><p id="p-12">We have tied together numerous threads from the literature to create a coherent, efficient, and accurate framework for imputing genotypes from modern reference datasets. Imputation-based GWAS are beginning to harness the power of thousands of reference genomes, and we expect that the practical solutions provided here will help investigators make the most of these rich genetic resources.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>IMPUTE2 algorithm</h3><p id="p-13">We begin by describing the basic IMPUTE2 algorithm, which will be retained in this work with some modifications. Full details of the original algorithm are available in <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> (2009)</a>. Although that paper addressed the use of multiple reference panels typed on different SNP sets (âScenario Bâ in their terminology), for simplicity we will focus on the situation where the reference haplotypes are all defined on the same SNPs (âScenario Aâ). Although we do not discuss Scenario B in this paper, the ideas presented here are easily extended to that setting.</p><p>IMPUTE2 uses a Markov chain Monte Carlo (MCMC) algorithm that alternates between phasing typed SNPs and imputing untyped SNPs. Each MCMC iteration includes two steps:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-15"><span class="list-label">1. </span>Sample a new phase configuration for each study individual, using information from other study individuals and reference panel haplotypes at SNPs typed in the study.</p></li><li id="list-item-2"><p id="p-16"><span class="list-label">2. </span>Given the newly sampled haplotypes for all study individuals, treat each haplotype as independent (conditional on the reference panel) and analytically impute the alleles at untyped SNPs.</p></li></ul><p>This MCMC algorithm is run for a number of iterations (typically 30, including 10 burn-in iterations), then the probabilities from Step 2 are averaged across iterations to produce marginal posterior genotype probabilities at each untyped SNP.</p><p id="p-17">The phasing and imputation calculations are driven by the hidden Markov model (HMM) of Li and Stephens (2003). This model can be used to update an individual's haplotypes by constructing them as âimperfect mosaicâ copies of a set of template haplotypes. In the simplest case, the templates would include all reference+study haplotypes (minus the pair being updated) in Step 1 and all reference haplotypes in Step 2.</p><p id="p-18">To reduce the computational burden of Step 1, <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> (2009)</a> introduced an approximation that restricts each phasing update to a set of <em>k</em> template haplotypes, which are chosen separately for each individual at each iteration; the other templates are implicitly assigned copying probabilities of zero. The <em>k</em> templates are chosen by computing Hamming distances between an individual's current sampled haplotypes and each possible template haplotype. We refer to the <em>k</em> templates with the smallest distances as âsurrogate family membersâ because they (ideally) share recent ancestry with the study individual. [These haplotypes were called âinformed conditioning statesâ in the <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> (2009)</a> paper and early versions of the IMPUTE2 documentation. We now prefer the nomenclature used here because of the approximationâs relationship to the âsurrogate parentâ phasing method of Kong <em>et al.</em> (2008).]</p><p id="p-19"><a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> (2009)</a> used the surrogate family approximation to speed up the phasing updates in IMPUTE2 (Step 1 of the MCMC algorithm described above), but they used all available reference haplotypes for the imputation updates (Step 2). To make computation faster in large, ancestrally diverse reference panels, we now extend the approximation to imputation updates.</p><p id="p-20">To decide which reference haplotypes to copy at a particular point in an IMPUTE2 run, we add an extra step between Steps 1 and 2. After individual <em>i</em> has sampled a new haplotype pair in Step 1, we calculate the Hamming distance from each of these haplotypes to each of the reference haplotypes, using only the overlapping SNPs. Then, separately for each of individual <em>i</em> 's haplotypes, we perform Step 2 (haploid imputation of untyped alleles) using only the <em>k<sub>hap</sub></em> nearest reference haplotypes as templates. This procedure is not guaranteed to identify <em>k<sub>hap</sub></em> unique haplotypes as multiple haplotypes near the <em>k<sub>hap</sub></em> cutoff may have the same Hamming distance. In these situations, we select a random subset of the boundary haplotypes to produce a reference panel with <em>k<sub>hap</sub></em> states. Intuitively, our approach corresponds to imputing each study haplotype from a âcustomâ reference panel containing close genealogical neighbors. We generally choose larger values for <em>k<sub>hap</sub></em> than for <em>k</em> because phasing updates require evaluation of <em>k</em><sup>2</sup>/2 HMM states per individual per SNP, whereas imputation updates require evaluation of only <em>k<sub>hap</sub></em> states.</p></div><div id="sec-3" class="subsection"><h3>HapMap 3 cross-validation experiments</h3><p id="p-21">To assess the accuracy of genotype imputation from reference panels of diverse ancestry, we performed leave-one-out cross-validations in data from HapMap Phase 3. The HapMap 3 paper (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">The International HapMap Consortium 2010</a>) includes a number of similar comparisons, which use a series of carefully controlled experiments to show how population ancestry in the reference and study data affects imputation accuracy. By contrast, our main goal is to validate a general strategy for using ancestrally diverse reference data. One way to view this distinction is that their experiments provide information about the kinds of reference data one might want to collect to improve imputation accuracy in a given population, whereas our experiments illustrate a strategy for getting good results from whichever data are available. To the extent that we consider how reference panels of different ancestries affect imputation accuracy, the main point is to guide intuition about why the overall strategy works.</p><p id="p-22">All of our experiments were based on phased haplotypes from HapMap 3, release 2, in NCBI Build 36 coordinates. HapMap 3 includes samples from 11 analysis panels, which are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. After some minor processing (which is described in the <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> legend), there was a total of 1011 unrelated individuals from 10 panels in this dataset. Recent work on the HapMap data has revealed a small number of close relatives among these putative unrelateds (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pemberton <em>et al.</em> 2010</a>). We conducted the analyses for this study before those relationships were revealed, but we do not anticipate that removing the related individuals would have a major effect on our results or conclusions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461013/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461013/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1â</span> <span class="caption-title">HapMap 3 panels used for cross-validation</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">The HapMap 3 samples were genotyped on both the Affymetrix SNP 6.0 platform and the Illumina 1M Human platform. For our cross-validations, we masked the SNPs not typed on the Affymetrix platform in one individual at a time, then imputed the masked genotypes from the haplotypes of other HapMap 3 individuals; details about which individuals were included in the reference panel are provided below. We also masked the phase of the observed genotypes in the individual being imputed. For these experiments, we used all of the HapMap 3 SNPs on chromosome 20, which led to 16,606 non-Affymetrix SNPs being imputed from 19,650 Affymetrix SNPs. We also repeated the experiments after reversing the roles of the SNP platforms, with results shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS3.pdf">File S3</a>.</p><p id="p-26">Once every individual in a panel had been masked and imputed, we assessed accuracy at each SNP as the squared Pearson correlation (<em>R</em><sup>2</sup>) between the masked genotypes, which take values in {0,1,2}, and the imputed allele dosages (also known as posterior mean genotypes), which take values in [0,2]. The allele dosage is defined for each genotype G as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/457/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/457/embed/mml-math-1.gif"/></noscript></span></span>, where <em>Pr</em> (<em>G</em> = <em>x</em>) is a marginal posterior probability generated by an imputation method. Once the correlation <em>R</em><sup>2</sup> had been measured for every masked SNP, we calculated the mean <em>R</em><sup>2</sup> across SNPs and reported this as a scalar summary of imputation accuracy in that cross-validation experiment. In rare situations, the correlation at a SNP was undefined because the imputation produced identical allele dosages for all individuals. In these cases, we set <em>R</em><sup>2</sup> = 0 to capture the intuition that there would be no power to detect an effect at such SNPs. We note that the HapMap 3 samples were phased together in continental groups, which implies that the absolute accuracies in this experiment may be slightly optimistic. However, our main focus is on the <em>relative</em> accuracy of different reference panel configurations, so the non-independence of cross-validation samples is not a meaningful shortcoming.</p><p id="p-27">We repeated this experiment for each of the 10 HapMap 3 panels in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> under various conditions. To understand the benefits and drawbacks of different reference panel compositions, we sequentially added HapMap 3 panels to the reference set and re-imputed after each addition. Adding the panels in all possible orders is combinatorically daunting and of dubious interpretive value, and we preferred to have an objective ordering, so we added panels in order of increasing pairwise <em>F<sub>ST</sub></em> to the cross-validation panel. We calculated <em>F<sub>ST</sub></em> between each pair of panels as the average across all HapMap 3 SNPs on the autosomes. The panel orderings induced by this criterion are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS1.pdf">File S1</a>. We also considered different orderings â <em>e.g.</em> adding panels as dictated by a greedy algorithm based on cross-validation accuracy â but these did not change our qualitative results (data not shown).</p><p id="p-28">Up to this point, we have described 10 separate cross-validation experiments for each HapMap 3 panel: a cross-validation within the panel of interest, followed by 9 additional experiments with successively more inclusive reference panels. We also wanted to assess the sensitivity of the inference to the <em>k<sub>hap</sub></em> parameter in IMPUTE2, so we repeated each of these experiments across a grid of <em>k<sub>hap</sub></em> values: 15, 30, 60, 90, 120, 150, 200, 250, 300, 350, 400, 500, 600, 700, 800, 900, 1000, 1100, 1200, 1300, 1400, 1500, 1600, 1700, 1800, 1900, and 2020 (which is the total number of HapMap 3 haplotypes, 2022, minus the two haplotypes of the masked individual). By definition, <em>k<sub>hap</sub></em> cannot exceed the number of haplotypes in the reference panel, so each reference configuration used only the values that were consistent with its panel size. Each configuration also used a special <em>k<sub>hap</sub></em> value that was set to the total number of reference haplotypes in that experiment, since this value seldom fell on a grid point.</p><p id="p-29">We performed these experiments with IMPUTE version 2.1.2 under the following settings: <em>k</em> = 80 (tuning parameter for phasing updates), <em>iter</em> = 30 (total number of MCMC iterations), <em>burnin</em> = 10 (number of <em>iter</em> to discard as burn-in), <em>hap_spec_fam</em> (flag to make the program choose a custom reference panel for each study haplotype), and <em>Ne</em> = 20000. The <em>Ne</em> parameter represents the effective size of the population being analyzed, and it is used to scale the recombination rates in the imputation HMM. It may seem odd that we use a single <em>Ne</em> value in populations that clearly have different effective sizes, but our pilot experiments showed that IMPUTE2 is largely insensitive to this parameter and that 20000 is a good universal value (data not shown). The approximations underlying the <em>k</em> and <em>k<sub>hap</sub></em> parameters are modeled on local genealogies with limited recombination, so we split chromosome 20 into nonoverlapping 5-Mb chunks for analysis, with a 250-kb buffer region on each side to prevent edge effects (this is a default setting in IMPUTE2).</p><p id="p-30">It is useful to run a separate imputation method as an external benchmark. We chose to compare against Beagle (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Browning and Browning 2009</a>) because past results showed that it could be competitive on a dataset of this scale (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Browning and Browning 2009</a>; <a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> 2009</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Jostins <em>et al.</em> 2011</a>). Beagle has already been compared with IMPUTE2 in a large, well-matched reference panel of European ancestry (<a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> 2009</a>), so to simplify the presentation we applied it only to the cosmopolitan HapMap 3 reference panels. We used Beagle version 3.0.2 with default settings for all experiments presented here. To facilitate parallel computation, we ran Beagle on the same 5-Mb chromosome chunks (with buffers) that were used by IMPUTE2.</p><p id="p-31">We also attempted to use a âcoalescent-basedâ method for choosing custom reference panels (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Pasaniuc <em>et al.</em> 2010</a>), but we could not get it to produce accurate results on our data; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS6.pdf">File S6</a> for details. We omitted another leading imputation method, MaCH (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2010</a>), because it was not computationally feasible with the full HapMap 3 panel at the time of these experiments. Both IMPUTE2 and MaCH have recently been made more efficient through âpre-phasingâ of GWAS genotypes (the MaCH implementation is called âminimacâ). While we did not evaluate these approaches here, we have found that pre-phasing is complementary to our <em>k<sub>hap</sub></em> approximation in preliminary experiments, as we explain in the <em>Discussion</em>. All else being equal, we would expect minimac to achieve similar accuracy to IMPUTE2 since both methods are based on the Li and Stephens (2003) model of DNA sequence variation, although further work may be needed to compare these methods in various contexts.</p></div><div id="sec-4" class="subsection"><h3>MalariaGEN cross-validation experiments</h3><p id="p-32">To evaluate strategies for reference panel construction in African populations, we performed cross-validation experiments in genotypes that were kindly provided by the Malaria Genetic Epidemiology Network (MalariaGEN; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Malaria Genomic Epidemiology Network 2008</a>). The data we used were collected by MalariaGEN investigators for genome-wide association studies of human resistance or susceptibility to severe malaria. The individuals in these datasets were recruited at medical centers in the Gambia and Ghana; we henceforth refer to these samples by the tags GMB and GHN, respectively, with the understanding that they may not represent the full spectrum of genetic diversity in the countries of origin. Further details about the study recruitment are available at <a href="http://www.malariagen.net">www.malariagen.net</a>.</p><p id="p-33">Each dataset consists of trios (658 from GMB and 608 from GHN) that were ascertained via proband children diagnosed with malaria in hospitals. The members of each trio were genotyped on the Illumina 650Y array, with the genotypes subjected to standard quality control procedures and phased by Beagle with trio information. MalariaGEN carried out the genotyping and data processing, then provided us with inferred haplotypes for the trio parents (1316 GMB individuals and 1216 GHN individuals) in NCBI Build 36 coordinates.</p><p id="p-34">As in the HapMap 3 comparisons, our main goal in designing a cross-validation experiment was to provide guidance on how to use existing and future imputation reference panels. We chose to focus on the GMB panel because genotype imputation has previously been evaluated in a Gambian GWAS (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jallow <em>et al.</em> 2009</a>) and because the 1000 Genomes Project is planning to sequence a set of individuals from the Gambia.</p><p id="p-35">To mimic the planned 1000 Genomes Gambian dataset, we randomly allocated 100 GMB individuals to a reference panel. We also formed a reference panel from 100 randomly chosen GHN individuals. Ghana is located between Nigeria (the source of the HapMap YRI panel) and the Gambia on the Atlantic coast of Africa, so this panel contains reference haplotypes sampled nearer to the location of interest than those in the HapMap data.</p><p id="p-36">We used the remaining 1216 GMB individuals as a validation set to model imputation into a Gambian GWAS. We imputed the GMB genotypes from a series of reference panels: all 2022 HapMap 3 (HM3) haplotypes; 200 GMB haplotypes (GMB); 200 GMB haplotypes plus 200 GHN haplotypes (GMB+GHN); and a combined set containing all 2422 GMB, GHN, and HM3 haplotypes (GMB+GHN+HM3).</p><p id="p-37">For each reference panel, we masked and imputed every 25<sup>th</sup> SNP in the GMB validation set, then repeated this analysis in a sliding window so that every genotyped SNP was imputed exactly once. To mimic a GWAS of unrelated individuals, we treated the nonmasked genotypes in the GMB validation set as unphased. We masked and imputed all available Illumina 650Y SNPs on chromosomes 20 and 11 (we added chromosome 11 to raise the counts of low-frequency SNPs, which are underrepresented in this dataset), except for those that were not typed in or had allele conflicts with HapMap 3, yielding a total of 40,300 SNPs for imputation. As in the HapMap 3 cross-validations, we split the chromosomes into non-overlapping 5-Mb regions to speed up the analysis and support IMPUTE2âs computational approximations.</p><p id="p-38">We also wanted to compare against Beagle on reference panels of primarily African ancestry, so we generated another reference panel called âHM3.afrâ that included the HapMap3 haplotypes from the ASW, LWK, MKK, and YRI panels (822 haplotypes). We then used IMPUTE2 and Beagle to impute genotypes in the 1216 Gambians based on two reference panels: GMB (200 haplotypes) and GMB+GHN+HM3.afr (1222 haplotypes). We masked the SNPs as described above, except that this time we masked every 13<sup>th</sup> SNP so that Beagle would treat the data as it would a standard GWAS analysis. (Beagle invokes a different model-fitting strategy when fewer than 7% of the genotypes are missing from the study dataset.)</p></div><div id="sec-5" class="subsection"><h3>Computational benchmarking</h3><p id="p-39">To produce computational benchmarks in a realistic imputation scenario, we simulated data that model the large, ancestrally diverse reference panel that is being generated by the 1000 Genomes Project. Our simulations were based on the sfs_code program (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hernandez 2008</a>), which uses a pre-specified demographic model (typically obtained from unbiased site frequency spectra) and DNA sequence annotations to drive a forward simulation that models the effects of genetic drift and natural selection on a population of chromosomes. Ryan Hernandez kindly provided us with the output of an sfs_code run that used a joint demographic model of three HapMap panels (CEU, CHB, and YRI) on chromosome 17p12 (a 4.7-Mb region). At the end of the forward simulation, the program sampled 10,000 haplotypes from each of the three populations. These haplotypes do not capture the full demographic complexity of the 1000 Genomes sample set, but the simulation does provide realistic DNA sequence data for three major sources of human genetic variation.</p><p id="p-40">Given these simulated sequences, we sought to create imputation reference panels that would capture features of the anticipated 1000 Genomes panels. We mirrored the overall size of the 1000 Genomes reference set by sampling a panel of 1600 chromosomes from each population, which yielded a total of 4800 chromosomes worldwide, just under the 1000 Genomes target of 5000. The genome-wide sequencing module of the 1000 Genomes Project is based on a low-coverage design, so a certain fraction of low-frequency variants will be missed in the real data. To mimic this ascertainment process, we used power calculations from the 1000 Genomes pilot paper (The 1000 Genomes Project Consortium 2010) to determine the chances of discovering SNPs with different numbers of variant allele copies. The discovery probabilities are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/TableS2.pdf">Table S2</a>; we applied them separately in each set of 1600 reference chromosomes, under the assumption that true SNPs are discovered (or not) independently of each other. Conditional on a SNP being discovered in any panel, we assumed it was genotyped perfectly in all three panels. This is a reasonable assumption for a benchmarking experiment because sporadic genotyping errors are unlikely to have a noticeable effect on a programâs computational burden.</p><p id="p-41">Having simulated a reference panel, we then simulated a GWAS dataset as the target for imputation. Of the 8400 CEU-like haplotypes from sfs_code that were not included in the reference set, we selected 2000 and randomly paired them to create a GWAS sample of 1000 individuals. We thinned this dataset to the approximate SNP density and MAF distribution observed in real Affymetrix 500k data on a set of British controls [the 1958 Birth Cohort of the Wellcome Trust Case Control Consortium (2007)].</p><p id="p-42">We imputed our simulated GWAS dataset from two different reference panels: a âCosmopolitanâ panel containing the full set of 4800 haplotypes and a âEuropeanâ panel containing 1000 haplotypes sampled from the 1600 CEU-like reference chromosomes. The first panel models the worldwide 1000 Genomes set, whereas the second panel models a single cluster of related populations (roughly speaking, the 1000 Genomes samples will be divided among five such clusters containing 500 individuals each). By imputing from both reference panels, we can see how reference panel size and diversity affect the computational loads of different imputation methods.</p><p id="p-43">To provide a simple benchmark, we selected an imputation region that contained exactly 10,000 polymorphic sites in the European panel of 1000 haplotypes; this yielded the 1.9-Mb interval [11200000,13096366] on chromosome 17. This region contains a larger number of SNPs in the Cosmopolitan reference panel, but we restricted that panel to the same 10,000 SNPs to simplify the comparison. In practice, GWAS investigators may perform similar filtering on Cosmopolitan reference panels to remove variants that are underpowered in a particular study.</p><p id="p-44">For each 10,000-SNP reference panel, we imputed the 1,000 GWAS individuals using IMPUTE v2.1.2 and Beagle v3.0.2. We used the default settings for both methods (for IMPUTE2, the default <em>k<sub>hap</sub></em> setting is 500), and we also ran IMPUTE2 with <em>k<sub>hap</sub></em> set to include all available reference haplotypes. We recorded the single-processor running times and memory requirements for each run.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-45">We performed a series of experiments to evaluate our proposed imputation framework, which combines a cosmopolitan reference panel with a new approximation for speeding up imputation from large reference datasets.</p><p id="p-46">We implemented our approximation within IMPUTE2, which uses an iterative algorithm to impute untyped variants in GWAS datasets. Whereas the original algorithm imputes genotypes from the full set of reference haplotypes, the new approximation imputes each study haplotype from a custom subset of reference haplotypes. (Study genotypes seldom come with known phase, but the haplotypes can be inferred as part of the algorithm.) Each of these custom reference panels includes the <em>k<sub>hap</sub></em> reference haplotypes that have the fewest allele differences with a study haplotype at overlapping SNPs, where <em>k<sub>hap</sub></em> is a user-defined parameter that controls the computational cost of imputation. If this method is applied over a limited genomic region (<em>e.g.</em> a few million base pairs rather than a whole chromosome), we expect the <em>k<sub>hap</sub></em> reference haplotypes to be enriched for those that share recent common ancestry with the study haplotype of interest. We refer to these haplotypes as âsurrogate family membersâ because, like real family members, they may share segments of nearly identical DNA that can be used for imputation. We explore the relationship between <em>k<sub>hap</sub></em> and accuracy in the results that follow, and we provide practical suggestions for applying this approximation in the <em>Discussion</em>.</p><div id="sec-7" class="subsection"><h3>HapMap 3 cross-validation experiments</h3><p id="p-47">We first tested our proposed imputation framework on a chromosome 20 dataset from HapMap 3. This dataset includes hundreds of haplotypes from each of several locations around the world, making it a good qualitative model for future reference panels like those being generated by the 1000 Genomes Project. The HapMap 3 panels used in this study are described in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>; note that we combined the CHB and JPT panels for this analysis.</p><p id="p-48">Within each of the 10 panels, we masked a set of SNPs in one individual at a time and used IMPUTE2 to infer the hidden genotypes. We repeated this procedure across a range of <em>k<sub>hap</sub></em> values and with various reference panels, which were creating by cumulatively adding HapMap 3 panels in the order dictated by genome-wide average <em>F<sub>ST</sub></em><sub>.</sub> One way to think of this procedure is to imagine building a composite reference panel for a population of interest: we start with a population-specific reference panel, and we successively add more-diverged panels to see if they will help (or possibly hurt) the imputation accuracy. These composite panels capture aspects of the <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> (2009a)</a> cross-validation strategy for choosing reference sets, and they provide population-label-informed benchmarks against which to compare our label-free way of using reference data.</p><div id="sec-8" class="subsection"><h4>Selected results for ASW and TSI:</h4><p id="p-49">In this section, we present results for the ASW (African American) and TSI (Italian) panels, which exhibit general trends and unexpected outcomes from our HapMap 3 cross-validations. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows how imputation accuracy depends on reference panel composition and the number of surrogate family haplotypes chosen from each composite panel (<em>k<sub>hap</sub></em>). We constructed these plots in ways that highlight interesting features, and we restricted the results to SNPs with MAF &lt; 5% in the target panel. We provide analogous plots for all HapMap 3 target panels in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS1.pdf">File S1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/457/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Imputation accuracy at low-frequency SNPs in HapMap 3 cross-validations in ASW and TSI, as a function of reference panel composition and khap value. These plots show the imputation accuracy of IMPUTE2 in (A) the ASW panel and (B) the TSI panel. The accuracy of each experiment is plotted on the y-axis as the mean R2 across all SNPs with MAF &lt; 5% in the cross-validation panel (identified by the gray box in each plot). The x-axis shows the khap parameter, which scales linearly with the computational burden of imputation updates in IMPUTE2. Each curve represents a different reference panel, with panels added cumulatively in the order shown in the legends, reading from bottom to top. Similar plots for other HapMap 3 target panels can be found in File S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1844122719" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Imputation accuracy at low-frequency SNPs in HapMap 3 cross-validations in ASW and TSI, as a function of reference panel composition and &lt;em&gt;k&lt;sub&gt;hap&lt;/sub&gt;&lt;/em&gt; value. These plots show the imputation accuracy of IMPUTE2 in (A) the ASW panel and (B) the TSI panel. The accuracy of each experiment is plotted on the &lt;em&gt;y&lt;/em&gt;-axis as the mean &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; across all SNPs with MAF &lt; 5% in the cross-validation panel (identified by the gray box in each plot). The &lt;em&gt;x&lt;/em&gt;-axis shows the &lt;em&gt;k&lt;sub&gt;hap&lt;/sub&gt;&lt;/em&gt; parameter, which scales linearly with the computational burden of imputation updates in IMPUTE2. Each curve represents a different reference panel, with panels added cumulatively in the order shown in the legends, reading from bottom to top. Similar plots for other HapMap 3 target panels can be found in File S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/457/F1.medium.gif" width="440" height="235"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/457/F1.medium.gif" width="440" height="235"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/457/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/457/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460898" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-50" class="first-child">Imputation accuracy at low-frequency SNPs in HapMap 3 cross-validations in ASW and TSI, as a function of reference panel composition and <em>k<sub>hap</sub></em> value. These plots show the imputation accuracy of IMPUTE2 in (A) the ASW panel and (B) the TSI panel. The accuracy of each experiment is plotted on the <em>y</em>-axis as the mean <em>R<sup>2</sup></em> across all SNPs with MAF &lt; 5% in the cross-validation panel (identified by the gray box in each plot). The <em>x</em>-axis shows the <em>k<sub>hap</sub></em> parameter, which scales linearly with the computational burden of imputation updates in IMPUTE2. Each curve represents a different reference panel, with panels added cumulatively in the order shown in the legends, reading from bottom to top. Similar plots for other HapMap 3 target panels can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS1.pdf">File S1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">The <em>x</em>-axis shows the value of the <em>k<sub>hap</sub></em> parameter, and the <em>y</em>-axis shows the imputation accuracy, which is measured as the mean SNP-wise <em>R</em><sup>2</sup> between true and imputed allele dosages (posterior mean genotypes) for each cross-validation experiment. The computational cost of imputation is roughly proportional to <em>k<sub>hap</sub></em><sub>.</sub> Applying different <em>k<sub>hap</sub></em> settings to a single reference panel generates a curve, and each curve represents a different reference panel. (For further details of the <em>R</em><sup>2</sup> distributions underlying the mean values at <em>k<sub>hap</sub></em> = 500, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS2.pdf">File S2</a>.) Each individual was imputed initially from haplotypes in the same panel, then from reference sets that cumulatively added panels in the order shown in the plot legends, reading from bottom to top. The black curves represent our suggested strategy of using a cosmopolitan reference panel.</p><p id="p-52">One observation from <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> is that the full set of reference haplotypes generated some of the highest accuracy levels in this experiment â the black curves almost always lie above the other curves. As we show in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS3.pdf">File S3</a>, this holds true across HapMap 3 target panels, genotyping arrays, and SNP frequency classes. Another salient feature of <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> is that the curves plateau quickly with increasing values of <em>k<sub>hap</sub></em> (moving from left to right within each plot). This shows that our surrogate family approximation can decrease computing time without losing accuracy. For example, the runs that selected 500 haplotypes from the full HapMap 3 panel (<em>k<sub>hap</sub></em> = 500; black curves) achieved similar accuracy to the runs that didn't use this approximation (<em>k<sub>hap</sub></em> = 2020), but the first set of runs was about four times faster.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/457/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Imputation accuracy at low-frequency SNPs in HapMap 3 cross-validations, as a function of target panel, reference panel composition, khap value, and imputation method. These plots show the imputation accuracy of IMPUTE2 and Beagle in various cross-validation experiments. The accuracy of each experiment is plotted on the y-axis as the mean R2 across all SNPs with MAF &lt; 5% in the cross-validation panel (identified by the gray box in each plot). The x-axis shows the khap parameter, which scales linearly with the computational burden of imputation updates in IMPUTE2. The solid black curves show how R2 varies with khap when using IMPUTE2 with a reference panel containing the full set of 2020 HapMap 3 haplotypes; the dashed black lines show the accuracy of Beagle with this reference panel. IMPUTE2 was also applied to subpanels of the full HapMap 3 panel, with results shown as orange curves. Similar plots for other observed SNP sets and imputed SNP MAFs can be found in File S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1844122719" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Imputation accuracy at low-frequency SNPs in HapMap 3 cross-validations, as a function of target panel, reference panel composition, &lt;em&gt;k&lt;sub&gt;hap&lt;/sub&gt;&lt;/em&gt; value, and imputation method. These plots show the imputation accuracy of IMPUTE2 and Beagle in various cross-validation experiments. The accuracy of each experiment is plotted on the &lt;em&gt;y&lt;/em&gt;-axis as the mean &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; across all SNPs with MAF &lt; 5% in the cross-validation panel (identified by the gray box in each plot). The &lt;em&gt;x&lt;/em&gt;-axis shows the &lt;em&gt;k&lt;sub&gt;hap&lt;/sub&gt;&lt;/em&gt; parameter, which scales linearly with the computational burden of imputation updates in IMPUTE2. The solid black curves show how &lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; varies with &lt;em&gt;k&lt;sub&gt;hap&lt;/sub&gt;&lt;/em&gt; when using IMPUTE2 with a reference panel containing the full set of 2020 HapMap 3 haplotypes; the dashed black lines show the accuracy of Beagle with this reference panel. IMPUTE2 was also applied to subpanels of the full HapMap 3 panel, with results shown as orange curves. Similar plots for other observed SNP sets and imputed SNP MAFs can be found in File S3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/457/F2.medium.gif" width="367" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/457/F2.medium.gif" width="367" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/457/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/457/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460998" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-53" class="first-child">Imputation accuracy at low-frequency SNPs in HapMap 3 cross-validations, as a function of target panel, reference panel composition, <em>k<sub>hap</sub></em> value, and imputation method. These plots show the imputation accuracy of IMPUTE2 and Beagle in various cross-validation experiments. The accuracy of each experiment is plotted on the <em>y</em>-axis as the mean <em>R<sup>2</sup></em> across all SNPs with MAF &lt; 5% in the cross-validation panel (identified by the gray box in each plot). The <em>x</em>-axis shows the <em>k<sub>hap</sub></em> parameter, which scales linearly with the computational burden of imputation updates in IMPUTE2. The solid black curves show how <em>R<sup>2</sup></em> varies with <em>k<sub>hap</sub></em> when using IMPUTE2 with a reference panel containing the full set of 2020 HapMap 3 haplotypes; the dashed black lines show the accuracy of Beagle with this reference panel. IMPUTE2 was also applied to subpanels of the full HapMap 3 panel, with results shown as orange curves. Similar plots for other observed SNP sets and imputed SNP MAFs can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS3.pdf">File S3</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54"><a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> also shows how imputation accuracy improved as particular panels were added to the reference set. The trends for the ASW panel (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>) are basically as expected: each successive African panel (MKK, LWK, and YRI) improved the accuracy incrementally, and the panels that capture components of European ancestry (MXL, GIH, TSI, and CEU) collectively raised the accuracy to its maximum value for this experiment. The east Asian panels (CHB+JPT and CHD) did not increase imputation accuracy any further, but nor did they reduce it; this demonstrates the ability of the method to ignore unhelpful reference haplotypes.</p><p id="p-55">The results for the TSI panel (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>) are more surprising. As we would expect, accuracy improved with the addition of each panel that contains recent European ancestry (CEU and MEX). The GIH panel also increased the accuracy, which is reasonable given the relatively modest genetic divergence between this panel and TSI. Perhaps unexpectedly, the maximum imputation accuracy in TSI was not achieved until a set of African haplotypes (MKK) was added to the reference panel. This improvement was also observed when adding MKK haplotypes to the reference set for CEU imputation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS1.pdf">File S1</a>). Supplementary results from the HapMap 3 paper (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">The International HapMap Consortium 2010</a>) show that the MKK panel has an admixture component that could reflect an ancient migration from Europe or the Middle East into eastern Africa, which might explain our results, although we observed a similar increase in accuracy after replacing the MKK panel with the remaining African panels (data not shown). Regardless of the underlying explanation, these results highlight the complexity of human demographic history, which is one motivation for frameworks like ours that use inclusive reference panels without population labels.</p><p id="p-56">Notably, the common SNPs did not follow all of the patterns seen here for low-frequency SNPs (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS3.pdf">File S3</a>.) A cosmopolitan reference panel produced high imputation accuracy in both frequency classes, but the accuracy at common SNPs came almost entirely from the most closely related panels. For example, the maximum accuracy at common SNPs in TSI was attained with a TSI+CEU reference panel, and the addition of other panels neither increased nor decreased the accuracy. Hence, our results confirm that cosmopolitan reference panels are benign for imputing common variants, while showing that such panels can be positively helpful for imputing low-frequency variants. This difference can be understood in terms of the representation of the minor allele in the reference panel: common alleles are usually well-represented in population-matched panels of nontrivial size, whereas low-frequency alleles may be present in only a few copies or absent entirely, depending on allele frequency, SNP ascertainment scheme, and the number of reference haplotypes. Additional copies of these alleles (and their associated haplotype backgrounds) may sometimes be found in other populations, which explain why a cosmopolitan reference panel can improve imputation accuracy at low-frequency variants. These statements are supported by the results in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS2.pdf">File S2</a>.</p></div><div id="sec-9" class="subsection"><h4>High-level results for all HapMap 3 panels:</h4><p id="p-57">We now extend the results from <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> to the full set of HapMap 3 panels and a competing imputation method. We chose to compare against Beagle (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Browning and Browning 2009</a>) because past studies showed that it could achieve competitive speed and accuracy with large reference panels (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Browning and Browning 2009</a>; <a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> 2009</a>; <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Jostins <em>et al.</em> 2011</a>).</p><p id="p-58">The high-level results of our cross-validation experiment are shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. As before, the solid black curves depict IMPUTE2 results with our suggested strategy of using a cosmopolitan reference panel, and different points on the <em>x</em>-axis correspond to different values of <em>k<sub>hap</sub></em><sub>.</sub> Here, the multicolored curves from <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a> (which represent imputation from subsets of the HapMap 3 haplotypes) are replaced with orange curves. The identities of the orange curves are omitted for plotting clarity, but full details are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS1.pdf">File S1</a>. We imputed and evaluated 1,523-2,364 low-frequency SNPs per HapMap 3 panel, with the exact numbers provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/TableS1.pdf">Table S1</a>. The black dashed lines in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> show the results of using Beagle with the full HapMap 3 reference panel; these lines are flat because Beagle does not have an analog of the <em>k<sub>hap</sub></em> parameter.</p><p id="p-59">As in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>, our proposed framework always produced near-maximal accuracy. Also as before, the solid black curves typically reach their highest accuracy values at small values of <em>k<sub>hap</sub></em><sub>.</sub> <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> shows that IMPUTE2 achieved higher accuracy than Beagle in every panel, except at the lowest <em>k<sub>hap</sub></em> settings. In some target panels, the difference between methods was small; for example, IMPUTE2 was only slightly more accurate than Beagle in the CEU and TSI panels, which is consistent with previous results comparing these methods on a European dataset (<a id="xref-ref-10-10" class="xref-bibr" href="#ref-10">Howie <em>et al.</em> 2009</a>). [We note that <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Jostins <em>et al.</em> (2011)</a> reached the contradictory conclusion that Beagle is more accurate than IMPUTE2 when imputing Europeans from diverse reference panels. We believe that their conclusion was driven by spurious IMPUTE2 results, as we explain in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS4.pdf">File S4</a>.] On the other hand, IMPUTE2 was more accurate by a large margin in the African panels (YRI, LWK, and MKK). These trends cannot be attributed to the fact that we are running Beagle with a stratified reference panel when the method is not designed for that situation: IMPUTE2 also produced higher accuracy when we used reference panels that were well-matched to the target panels, both in the current HapMap 3 framework (data not shown) and in our MalariaGEN analyses (results below). We further note that the Beagle results shown here are better than the ones we obtained with smaller, less diverse HapMap 3 reference sets (data not shown). We tried running Beagle with larger values of its <em>niterations</em> and <em>nsamples</em> parameters, but there was essentially no change in these results (data not shown). We speculate on the mechanistic reasons for the accuracy differences between IMPUTE2 and Beagle in the <em>Discussion</em>.</p><p id="p-60">A subtle feature of <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> is that not all of the IMPUTE2 curves are monotonically increasing with <em>k<sub>hap</sub></em>: some of the black curves peak at intermediate values of this parameter, then steadily decay as <em>k<sub>hap</sub></em> grows to its maximum value of 2,020. This trend is clearest in the YRI panel, but it is also observable in other panels. There should be few problems of statistical computation (<em>e.g.</em> failure of the MCMC algorithm to converge) in our leave-one-out experiments, so we assume that this result reflects a real feature of the method. Our interpretation is that restricting the reference set via <em>k<sub>hap</sub></em> actually imposes a more appropriate prior distribution on the haplotype copying probabilities when there is significant population structure in the panel. Tuning this prior by changing <em>k<sub>hap</sub></em> has only a small effect on mean accuracy, which implies that our imputation method is largely robust to stratified reference data even without the surrogate family approximation. At the same time, this result implies that choosing custom reference panels may have benefits beyond just speeding up the computation, which is consistent with the conclusions of <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Pasaniuc <em>et al.</em> (2010)</a>.</p><p id="p-61">It is not straightforward to compare these results to those of <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> (2009a)</a> because our experimental design is somewhat different. Nonetheless, we believe that our basic conclusions align with theirs. For example, we observed that the imputation accuracy with a worldwide reference panel (<em>k<sub>hap</sub></em> = 2020) was never much lower than the accuracy with the optimal <em>k<sub>hap</sub></em> (peaks of black curves in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>), and the <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> (2009a)</a> results show a similar trend for cosmopolitan <em>vs.</em> optimal mixtures of HapMap 2 panels. In this sense, our <em>k<sub>hap</sub></em> approximation can be viewed as a flexible and automatic way of implementing the <a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> (2009a)</a> panel selection approach with an arbitrarily large number of reference populations.</p></div></div><div id="sec-10" class="subsection"><h3>MalariaGEN cross-validation experiments</h3><p id="p-62">To assess whether our imputation strategy would yield similar benefits in African populations outside the HapMap 3 set, we performed additional cross-validations in a Gambian dataset from the Malaria Genomic Epidemiology Network (MalariaGEN; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Malaria Genomic Epidemiology Network 2008</a>). Previous work on imputing Gambians in a disease study found that the HapMap 2 YRI panel produced weaker association signals than did a Gambia-specific panel at the strongly selected beta-globin gene (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Jallow <em>et al.</em> 2009</a>). Here, we extend that work to examine imputation accuracy in larger reference panels and a broader variety of loci. We also rephrase the question to ask whether haplotypes sampled outside The Gambia can improve accuracy when a dedicated Gambian panel is available.</p><p id="p-63">To answer these questions, we masked and imputed 40,300 SNPs from the Illumina 650Y array in a set of 1216 Gambian individuals from MalariaGEN. We repeated the analysis for each of four reference panels: all 2022 HapMap 3 haplotypes (HM3); 200 Gambian haplotypes (GMB); 200 Gambian haplotypes plus 200 Ghanaian haplotypes (GMB+GHN); and all of the aforementioned panels combined (GMB+GHN+HM3; 2422 haplotypes). For each imputation run, we used IMPUTE2 with <em>k<sub>hap</sub></em> = 500. In reference panels with fewer than 500 haplotypes, we reduced <em>k<sub>hap</sub></em> to the number of available haplotypes. The results are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, which breaks down the imputation accuracy by the minor allele frequencies of the SNPs in the 1216 imputed Gambians.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/457/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Imputation accuracy in Gambian validation set as a function of reference panel composition and minor allele frequency. These plots show the accuracy obtained when imputing masked SNPs in 1216 Gambian individuals from the MalariaGEN dataset using IMPUTE2 with khap = 500. Each reference panel is represented by a different color, and the results are shown for (A) all SNPs and (B) SNPs with MAF &lt; 10% in the Gambian validation set. The results are binned by MAF, with 5% bins in (A) and 1% bins in (B). Each point on a curve is located in the middle of the corresponding MAF bin. The following reference panel codes are used in the legend: GMB (Gambia, 200 haplotypes); GHN (Ghana, 200 haplotypes); and HM3 (HapMap 3, 2022 haplotypes)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1844122719" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Imputation accuracy in Gambian validation set as a function of reference panel composition and minor allele frequency. These plots show the accuracy obtained when imputing masked SNPs in 1216 Gambian individuals from the MalariaGEN dataset using IMPUTE2 with &lt;em&gt;k&lt;sub&gt;hap&lt;/sub&gt;&lt;/em&gt; = 500. Each reference panel is represented by a different color, and the results are shown for (A) all SNPs and (B) SNPs with MAF &lt; 10% in the Gambian validation set. The results are binned by MAF, with 5% bins in (A) and 1% bins in (B). Each point on a curve is located in the middle of the corresponding MAF bin. The following reference panel codes are used in the legend: GMB (Gambia, 200 haplotypes); GHN (Ghana, 200 haplotypes); and HM3 (HapMap 3, 2022 haplotypes).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/457/F3.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/457/F3.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/457/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/457/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460980" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-64" class="first-child">Imputation accuracy in Gambian validation set as a function of reference panel composition and minor allele frequency. These plots show the accuracy obtained when imputing masked SNPs in 1216 Gambian individuals from the MalariaGEN dataset using IMPUTE2 with <em>k<sub>hap</sub></em> = 500. Each reference panel is represented by a different color, and the results are shown for (A) all SNPs and (B) SNPs with MAF &lt; 10% in the Gambian validation set. The results are binned by MAF, with 5% bins in (A) and 1% bins in (B). Each point on a curve is located in the middle of the corresponding MAF bin. The following reference panel codes are used in the legend: GMB (Gambia, 200 haplotypes); GHN (Ghana, 200 haplotypes); and HM3 (HapMap 3, 2022 haplotypes).</p><div class="sb-div caption-clear"></div></div></div><p id="p-65"><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> shows the results for all imputed SNPs, whereas <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a> is restricted to SNPs with MAF &lt; 10%. We omitted SNPs with MAF &lt; 1% from both plots as there were too few of these to provide reliable measurements. Like previous authors (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Jallow <em>et al.</em> 2009</a>), we found that a Gambia-specific reference set outperformed a HapMap set that did not include Gambians: the GMB panel (blue curves) produced higher overall accuracy than did the HM3 panel (red curves), despite the fact that there were â¼800 chromosomes of African ancestry in the HM3 set and only 200 chromosomes in the GMB set. This suggests that historical divergence between the populations in the HapMap 3 and Gambian panels makes HapMap 3 less accurate as an imputation resource.</p><p id="p-66">The HapMap 3 panel is still useful, however. Across the allele frequency spectrum, the difference in mean <em>R</em><sup>2</sup> between the GMB and HM3 panels was never larger than 2.5%, and the difference was smallest at low-frequency SNPs. In fact, the HM3 panel was more accurate than the GMB panel for SNPs in the 1â2% and 2â3% MAF bands (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>); this shows that an ancestrally inclusive, nonspecific reference panel can capture low-frequency alleles that are poorly represented in a Gambia-specific panel. A recent simulation study found that association power and mean imputation <em>R</em><sup>2</sup> have a roughly linear relationship with a slope near 1.0 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zheng <em>et al.</em> 2011</a>), which suggests that using the HM3 panel in place of the GMB panel would cause only a small loss of power in an imputation-based association scan. [We note that the results from <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Zheng <em>et al.</em> (2011)</a> appear to conflict with those of a recent study by <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huang <em>et al.</em> (2009b)</a>, which found that power drops quickly with mean <em>R</em><sup>2</sup>. We believe that the <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Zheng <em>et al.</em> (2011)</a> results more accurately reflect this relationship because the mathematical model of imputation errors used by Huang <em>et al.</em> ignores the correlation of imputation error across individuals, which could be substantial, <em>e.g.</em> at hard-to-impute SNPs.</p><p id="p-67"><a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> also shows that non-Gambian panels can improve imputation accuracy in the presence of a Gambian panel: the orange curve represents the addition of haplotypes from a moderately diverged population (GHN) and the black curve represents the further addition of a worldwide reference panel (HM3). These results support our strategy of using cosmopolitan reference panels: regardless of which reference data are available, the highest accuracy is achieved when using all available haplotypes and letting the imputation method decide which ones to use.</p><p id="p-68">Aside from these questions of reference panel composition, we can also ask whether the choice of imputation method matters in African datasets. Our HapMap 3 comparisons suggest that Beagle has trouble imputing African genotypes, and we wanted to see if this conclusion would hold up in the MalariaGEN data. To address this, we used Beagle to impute the Gambian validation set using two different reference panels: (i) the GMB panel and (ii) a composite reference set containing the GMB panel, the GHN panel, and the HM3 panels with majority African ancestry (ASW, LWK, MKK, and YRI). We decided to reduce the HM3 set to these panels (which we label âHM3.afrâ) in case Beagleâs difficulties in the previous comparisons were caused by the inclusion of non-African haplotypes in the reference panel. For consistency, we also imputed from the GMB+GHN+HM3.afr panel with IMPUTE2.</p><p id="p-69">The results of this comparison are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. Results based on the GMB panel are shown in blue, while results based on the GMB+GHN+HM3.afr panel are shown in gray. Accuracy curves for IMPUTE2 and Beagle are drawn with solid and dashed lines, respectively. In concurrence with our previous results on African populations, we found that Beagle was much less accurate than IMPUTE2 when provided with the same reference panel: for each color, the solid line in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> is consistently above the dashed line. In fact, IMPUTE2 achieved higher accuracy when imputing from 200 GMB haplotypes (solid blue line) than Beagle did when imputing from 1222 African haplotypes (dashed gray line), of which the 200 GMB were a subset. The difference between methods was largest at low-frequency SNPs, but there was a substantial gap across the entire frequency spectrum.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/457/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of imputation accuracy between IMPUTE2 and Beagle in Gambian validation set. This plot shows the accuracy obtained when imputing masked SNPs in 1216 Gambian individuals from the MalariaGEN dataset using either IMPUTE2 with khap = 500 (solid lines) or Beagle on default settings (dashed lines). Imputation was performed with a reference panel of Gambian haplotypes (blue) and a reference panel of Gambian, Ghanaian, and HapMap 3 African ancestry haplotypes (gray). The results are grouped into 5% MAF bins, and each point on a curve is located in the middle of the corresponding MAF bin. The following reference panel codes are used in the legend: GMB (Gambia, 200 haplotypes); GHN (Ghana, 200 haplotypes); and HM3.afr (HapMap 3 African ancestry, 822 haplotypes)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1844122719" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of imputation accuracy between IMPUTE2 and Beagle in Gambian validation set. This plot shows the accuracy obtained when imputing masked SNPs in 1216 Gambian individuals from the MalariaGEN dataset using either IMPUTE2 with &lt;em&gt;k&lt;sub&gt;hap&lt;/sub&gt;&lt;/em&gt; = 500 (solid lines) or Beagle on default settings (dashed lines). Imputation was performed with a reference panel of Gambian haplotypes (blue) and a reference panel of Gambian, Ghanaian, and HapMap 3 African ancestry haplotypes (gray). The results are grouped into 5% MAF bins, and each point on a curve is located in the middle of the corresponding MAF bin. The following reference panel codes are used in the legend: GMB (Gambia, 200 haplotypes); GHN (Ghana, 200 haplotypes); and HM3.afr (HapMap 3 African ancestry, 822 haplotypes).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/457/F4.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/457/F4.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/457/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/457/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460933" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4â</span> <p id="p-70" class="first-child">Comparison of imputation accuracy between IMPUTE2 and Beagle in Gambian validation set. This plot shows the accuracy obtained when imputing masked SNPs in 1216 Gambian individuals from the MalariaGEN dataset using either IMPUTE2 with <em>k<sub>hap</sub></em> = 500 (solid lines) or Beagle on default settings (dashed lines). Imputation was performed with a reference panel of Gambian haplotypes (blue) and a reference panel of Gambian, Ghanaian, and HapMap 3 African ancestry haplotypes (gray). The results are grouped into 5% MAF bins, and each point on a curve is located in the middle of the corresponding MAF bin. The following reference panel codes are used in the legend: GMB (Gambia, 200 haplotypes); GHN (Ghana, 200 haplotypes); and HM3.afr (HapMap 3 African ancestry, 822 haplotypes).</p><div class="sb-div caption-clear"></div></div></div><p id="p-71">We believe that Beagle's difficulties in imputing African datasets arise from properties of its clustering model. It is possible that Beagle could produce better results with different settings, although we tried varying both the model-fitting parameters (<em>niterations</em>, <em>nsamples</em>) and the model-building parameters (<em>scale</em>, <em>shift</em>) without observing a meaningful change in accuracy (data not shown).</p></div><div id="sec-11" class="subsection"><h3>Computational benchmarking</h3><p id="p-72">To provide computational benchmarks for the imputation methods used in this study, we simulated two reference panels: one containing 4800 haplotypes modeled on the ancestrally diverse reference dataset that is being produced by the 1000 Genomes Project, and another containing 1000 haplotypes modeled on the European component of the 1000 Genomes set. We restricted the panels to a shared set of 10,000 SNPs spanning 1.9 Mb of sequence, and we imputed from both panels into a simulated GWAS of 1000 European individuals. The GWAS samples were provided with genotypes at SNPs that mirror the properties of the Affymetrix 500k platform; there were 337 such SNPs in the simulated region.</p><p id="p-73">We imputed the simulated GWAS dataset from each reference panel using IMPUTE2 and Beagle. We ran Beagle on the same (default) settings as in the cross-validations, and we ran IMPUTE2 on two settings for each panel: <em>k<sub>hap</sub></em> = 500 and <em>k<sub>hap</sub></em> = <em>N</em>, where <em>N</em> is the number of haplotypes in the reference panel. As in the cross-validations, we fixed the IMPUTE2 phasing parameter at <em>k</em> = 80. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the single-processor running times and random-access memory (RAM) requirements of each program. We obtained these benchmarks from a single computer with 148 GB of RAM and a 2.4 GHz Intel Xeon processor.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460994/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460994/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2â</span> <span class="caption-title">Computational benchmarks for a simulated GWAS of 1000 European individuals imputed from reference panels with 10,000 SNPs</span><div class="sb-div caption-clear"></div></div></div><p id="p-76"><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> illustrates the computational benefits of our surrogate family approximation. In the Cosmopolitan reference panel with 4800 haplotypes, reducing <em>k<sub>hap</sub></em> from 4800 to 500 decreased IMPUTE2âs running time by a factor of 4.8. Another way of viewing this is to notice that with <em>k<sub>hap</sub></em> fixed at 500, IMPUTE2's running time increased by only a factor of 1.4 when moving from a panel with 1000 haplotypes to a panel with 4800 haplotypes. By comparison, Beagle's running time increased by a factor of 9 with the same panels. In this setting, fixing <em>k<sub>hap</sub></em> fixes the cost of the imputation calculations used by IMPUTE2, so the 1.4-fold increase in running time at <em>k<sub>hap</sub></em> = 500 reflects the additional time needed to evaluate a larger number of haplotypes when choosing which 500 to use for imputation. Preliminary experiments suggest that this evaluation step could be shortened by ignoring divergent haplotypes after the first few iterations of the algorithm (data not shown), which would make the overall running time almost independent of the number of reference haplotypes for fixed <em>k<sub>hap</sub></em><sub>.</sub></p><p id="p-77">On these kinds of imputation datasets, IMPUTE2 shows clear computational advantages over Beagle. Even when using a large, ancestrally diverse reference panel, IMPUTE2 finished in less time on default settings (127 min with <em>k<sub>hap</sub></em> = 500) than it took Beagle to impute from an ancestrally homogeneous panel with almost five times fewer haplotypes (655 min). IMPUTE2 also required much less RAM: for each reference panel, Beagle needed about 20 times more memory. These results, in combination with our cross-validation results, confirm that IMPUTE2 is both more accurate and more efficient than Beagle in the kinds of imputation datasets that are beginning to drive the field. As with any sophisticated inference method, there are ways to tweak Beagle's settings to achieve better speed, but all of them would reduce imputation accuracy. (Beagle's memory footprint can also be reduced, at the cost of even longer running times.) We explore some of the factors underlying these computational differences, and the implications they hold for future methods development, in the <em>Discussion</em>.</p><p id="p-78">For fixed <em>k</em> and <em>k<sub>hap</sub></em><sub>,</sub> IMPUTE2âs computational burden scales linearly with the number of study individuals, the number of reference haplotypes, the number of study SNPs, and the number of reference SNPs. Each of these factors makes a different per-unit contribution to the overall running time, with the number of study individuals and the number of reference SNPs having the biggest effect in modern datasets. Extrapolating the numbers from <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> to the entire genome and assuming the availability of 100 parallel computer processors, we predict that it would take IMPUTE2 about a day to impute 1000 individuals from a reference panel with thousands of sequenced haplotypes. For investigators with limited computational resources or very large GWAS cohorts, the imputation can be made even faster by prephasing the GWAS genotypes, as we explain in the <em>Discussion</em>.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-79">Advances in DNA sequencing technologies have made it feasible to obtain near-complete genome sequences from thousands of individuals. Association mapping studies will immediately benefit from these developments: whole-genome sequencing of large GWAS datasets will not be practical for a while yet; in the meantime, we can impute a wide range of genetic variation from genomes that have already been sequenced. Most of the mutations discovered in these genomes will occur at low population frequencies, so it is important that imputation strategies be tailored to capture low-frequency variants. As sequencing projects produce larger and more diverse reference datasets, imputation-based GWAS will also face practical challenges like choosing appropriate reference panels and keeping computation tractable.</p><p id="p-80">We have developed a coherent and convenient imputation framework that addresses these concerns. To simplify the process of choosing reference haplotypes from a diverse collection, our approach uses a cosmopolitan reference panel. Previous work suggested that cosmopolitan panels could increase imputation accuracy at low-frequency variants, and our results extend these findings to a wide range of human populations. It is computationally intensive to perform imputation with large, ancestrally diverse reference panels, so we have also developed an approximation that decreases the cost of adding haplotypes to a reference set without sacrificing accuracy.</p><p id="p-81">Our framework was motivated by the idea that larger reference datasets should make imputation faster and simpler, rather than slower and more complicated. We believe that our work represents a first step toward bridging current imputation practice with the paradigm suggested by Kong <em>et al.</em> (2008), in which large population samples eliminate the need for complex models and reference panel selection, and investigators do not have to balance efficiency and accuracy. Detailed population models and reference panel weighting schemes may provide modest accuracy improvements in the short term, but we expect that the power gains from such developments will seldom justify the added computational costs.</p><p id="p-82">Conversely, imputation strategies that are agnostic to population labels (as ours is) may become increasingly attractive as sequencing studies fill in the continuum of human genetic diversity. One benefit of such approaches is that they can capture unexpected allele sharing without needing to model the complexities of human demographic history, as we demonstrated by showing that African haplotypes can improve imputation accuracy in Europeans. Approaches like ours are also well-suited for imputation in recently admixed populations: methods that choose custom reference panels for different admixed individuals in different parts of the genome can increase accuracy by adapting to local ancestry changes, as previously suggested by <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Pasaniuc <em>et al.</em> (2010)</a>.</p><p id="p-83">The framework we have proposed can serve as a general approach for using reference data in current and future imputation studies, and we expect that it will spur additional methods development in this area. Below, we discuss the aspects of this framework that make it successful in modern reference panels, along with some practical and theoretical questions that may arise when extending the conclusions of this study to other datasets.</p><div id="sec-13" class="subsection"><h3>Extending our results to future studies</h3><p id="p-84">Our cross-validation experiments have provided a wealth of information about how to use existing imputation resources like HapMap 3, but these datasets do not capture the full range of features that will be present in future reference panels. For example, our results are based on data from commercial SNP arrays, whose composition is biased toward variants that share alleles across populations. Consequently, population-specific accuracy contributions like the ones seen in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a> should not be treated as quantitative predictions for newly discovered variants. While we could have used 1000 Genomes data to address the SNP ascertainment issue, the data available when we were preparing this manuscript contained smaller sample sizes and a narrower sampling of human genetic diversity than found in HapMap 3, so we decided to focus on the latter dataset as a model of future 1000 Genomes reference panels. We have run similar imputation experiments with an interim release of the 1000 Genomes Phase I haplotypes, and we have continued to see benefits from using ancestrally inclusive reference panels (B. Howie; unpublished data).</p><p id="p-85">In this work, we have highlighted the fact that combining reference data from different populations can improve imputation accuracy at low-frequency variants. This finding reflects both the limited sample sizes of existing reference panels and the shared ancestry of human populations: an allele that occurs at low frequency in a study population may be poorly represented in a well-matched reference panel due to sampling effects; however, that same allele may be found in reference sets from other populations due to genetic drift or introduction by recent migrants. While a multipopulation reference panel can improve accuracy at this kind of variant, there are other situations in which accuracy might be harmed by such panels. Possible mechanisms for decreased accuracy include (i) the imputation of variant alleles at sites that do not segregate in a study dataset, (ii) signal dilution from reference haplotypes that are similar to those in a study population but do not carry a variant allele that segregates in that population, and (iii) misleading results from reference haplotypes that carry recurrent mutations. We discuss these issues in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS5.pdf">File S5</a>; we conclude that they will seldom hurt the imputation of low-frequency alleles from HapMap 3 or 1000 Genomes haplotypes, but that reference panel composition may need to be reevaluated when imputing rare alleles (MAF &lt; 0.5%) or using other reference datasets.</p><p id="p-86">Another question to consider when applying our framework is whether the optimal number of surrogate family haplotypes will change with different reference datasets. Judging from our experience in a variety of studies, we suggest the rule of thumb that <em>k<sub>hap</sub></em> should be set to the number of reference haplotypes that have broadly similar ancestry to the study population. For example, the broad ancestral groupings in HapMap 3 (Europe, East Asia, Africa) each include 500â800 haplotypes, and we found that <em>k<sub>hap</sub></em> = 500 worked well with this resource. Imputation accuracy is not highly sensitive to this variable, regardless of other factors like chunk size and local recombination rate, so it should not usually be necessary to optimize <em>k<sub>hap</sub></em> empirically. As reference sets grow and we further develop our approximation, we anticipate that it will be possible to achieve high accuracy with even lower values of <em>k<sub>hap</sub></em><sub>.</sub></p></div><div id="sec-14" class="subsection"><h3>Suggestions for imputation-based GWAS in Africa</h3><p id="p-87">African populations pose a special challenge for imputation because they are among the most genetically diverse in the world (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rosenberg <em>et al.</em> 2010</a>). Genetic relationships among African populations have been shaped by complex demographic histories, deep ancestries, and strong selective pressures, which can cause patterns of haplotype sharing in Africa to look much different than patterns in other parts of the world (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bryc <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Campbell and Tishkoff 2008</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Reed and Tishkoff 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Tishkoff <em>et al.</em> 2007</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">2009</a>). African populations also carry substantial burdens of common disease, yet few large-scale GWAS have been conducted in this setting. Efforts like the 1000 Genomes Project and MalariaGEN are changing this landscape, so it will be important to define effective reference panels for GWAS in a variety of African populations.</p><p id="p-88">We addressed this question by performing cross-validation experiments in Gambian individuals from MalariaGEN. In concurrence with a more limited analysis by <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Jallow <em>et al.</em> (2009)</a>, we found that a population-specific reference panel yielded higher average accuracy than did a larger HapMap 3 panel that lacked Gambian haplotypes. However, we also found that the HapMap 3 panel produced reasonable imputation accuracy across the allele frequency spectrum, and that non-Gambian haplotypes improved accuracy when added to a Gambia-specific reference panel.</p><p id="p-89">Although these findings were obtained from a limited sampling of African genetic diversity, they provide some intuition about general strategies for imputation in people of recent African ancestry. It will always help to collect new reference data through population-specific sequencing or genotyping, but the gains from this approach will often be larger in Africa. Whether or not a well-matched panel is available for a particular study population, imputing from all available haplotypes of African ancestry may often improve the results. Non-specific reference panels can weaken association signals near loci that have recently experienced strong selection, as with the beta-globin region in the Jallow <em>et al.</em> study, so it may be worthwhile to re-impute from just the ancestry-matched reference haplotypes (when such haplotypes are available) in regions showing decisive evidence of selective sweeps.</p><p id="p-90">Imputation methods based on the Li and Stephens (2003) model of DNA sequence variation (like IMPUTE2 and MaCH) are well suited to performing imputation in African GWAS. As shown in our HapMap 3 and MalariaGEN cross-validations, this kind of model can consistently produce higher accuracy than clustering approaches like the one used by Beagle, for reasons we discuss below. When the model is implemented via an efficient algorithm like IMPUTE2, this accuracy can be achieved at a fraction of the computational price.</p><p id="p-91">Looking ahead, we anticipate that many of the initial African GWAS will be conducted in west African populations; for example, such populations constitute a large part of MalariaGEN's Consortial Project 1. The power of these studies could potentially be boosted by using African American haplotypes to augment the reference sets collected in Africa. While there are clear merits to this idea, one might worry that the haplotype segments of non-African ancestry would pose problems for imputation. We can address this question by inspecting the IMPUTE2 results from our HapMap 3 cross-validations. Encouragingly, these results show that adding the HapMap 3 ASW panel to the reference set improved accuracy in every African cross-validation panel (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-15" class="subsection"><h3>Computational strategies for imputation with large, sequence-based reference panels</h3><p id="p-92">Throughout our imputation experiments, we found that IMPUTE2 can attain both higher accuracy and faster computation than Beagle, which is a leading inference method for large datasets. We believe that the success of IMPUTE2 in this context can be attributed to its computational strategies and its model of DNA sequence variation. We discuss these attributes here in hopes that they will inform future methods development, and we address the role that prephasing can play in speeding up imputation.</p><p id="p-93">Beagle's basic modeling approach is to combine haplotypes into clusters. This speeds up computation because it restricts the number of HMM states that need to be considered: rather than perform HMM calculations on every haplotype in a dataset, Beagle can run the calculations on a smaller set of clusters. Similar state-reduction techniques are used by GERBIL (Kimmel and Shamir 2005), fastPHASE (Scheet and Stephens 2006), GEDI (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kennedy <em>et al.</em> 2008</a>), and other related methods. By contrast, the basic HMM used by IMPUTE2 and MaCH includes a state for every haplotype. Using all of the states makes computation intractable, which is why IMPUTE2 restricts the states via its <em>k</em> and <em>k<sub>hap</sub></em> parameters. The intuition is that the âsurrogate family membersâ identified in this way should include the most informative haplotypes for a particular individual in a particular part of the genome.</p><p id="p-94">Both of these state-reduction approaches speed up imputation, but our cross-validations show that IMPUTE2 attains higher accuracy than Beagle in practice, especially at low-frequency variants in datasets that have higher haplotype diversity (<em>e.g.</em> those with recent African ancestry). We suggest that this is because clustering models have inherent difficulties capturing low-frequency variation: by grouping similar haplotypes into clusters, these methods obscure the differences between those haplotypes, which reduces the ability to impute low-frequency variants. This could explain why the accuracy disparity between IMPUTE2 and Beagle was largest in African populations, which have higher genetic diversity than non-African populations and hence a larger fraction of low-frequency haplotypes. Methods like Beagle may be able to make up some of this ground by using more clusters, but this will further increase the computational load.</p><p id="p-95">These trends should persist as imputation datasets continue to grow: clustering models will need to add even more states to their HMMs to remain competitive on accuracy, whereas the closest <em>k</em> (or <em>k<sub>hap</sub></em>) surrogate family haplotypes will become even more informative, thereby enhancing the running time and accuracy advantages of methods like IMPUTE2. The natural endpoint of this process will arrive when so many genomes have been sequenced that imputation requires just a handful of the closest genealogical neighbors, which is where âsurrogate parentâ methods, like the one developed by Kong <em>et al.</em> (2008), will take hold. Until that point is reached, we suggest that our surrogate family approximation will remain an attractive way to balance accuracy and speed.</p><p id="p-96">Another technique for increasing the efficiency of imputation is called âpre-phasing.â The idea is to (pre-)phase the assayed genotypes in a GWAS dataset, then impute directly into the inferred haplotypes; this speeds up imputation by more than an order of magnitude at the cost of a small amount of accuracy (B. Howie and C. Fuchsberger, unpublished data). In principle, most imputation methods could use this approach, and researchers can already download implementations based on the IMPUTE2 and MaCH models (the MaCH implementation is called âminimacâ). We have found that <em>k<sub>hap</sub></em> has similar accuracy characteristics in both unphased and pre-phased GWAS datasets (data not shown), so we view pre-phasing as being complementary to our surrogate family approximation: both approaches speed up imputation, and they can be used together for even greater efficiency.</p></div><div id="sec-16" class="subsection"><h3>Extensions</h3><p id="p-97">One potential extension of the results and methodology seen in this study is to whole-genome sequencing efforts like the 1000 Genomes Project. One study design that has arisen in this context is to sequence many individuals at low coverage; say, 2â4Ã. The data from this kind of experiment are too sparse to directly and confidently determine most genotypes, but they can be called with high accuracy by applying the same kinds of models that are used for genotype imputation in GWAS (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2011</a>; The 1000 Genomes Project Consortium 2010). We expect that the approach of combining information across populations will help call low-frequency alleles in that setting, much as it helped impute low-frequency alleles in this study.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-98">We would like to thank Ryan Hernandez for assistance with the sfs_code simulations, MalariaGEN for providing us with Gambian and Ghanaian haplotypes, the HapMap Project for providing an excellent imputation resource, and two anonymous reviewers for helpful comments on an earlier version of the manuscript. B.H. and M.S. were supported by National Institutes of Health (NIH) grants HL084689 and HGO2585 to M.S.. J.M. was supported by United Kingdom Medical Research Council grant number G0801823.</p><p id="p-99">This study makes use of data generated by MalariaGEN. A full list of the investigators who contributed to the generation of the data is available from www.MalariaGEN.net. Funding for the MalariaGEN project was provided by the Foundation for the National Institutes of Health and the Wellcome Trust, and it comes through the Grand Challenges in Global Health Initiative.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001198/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-100"><em>Communicating editor: A. Chakravarti</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-06-11"><span class="received-label">Received </span>June 11, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-09-19"><span class="accepted-label">Accepted </span>September 19, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2011 Howie <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.1" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>â<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.2" data-doi="10.1007/s00439-008-0568-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Missing data imputation and haplotype phase inference for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">439</span>â<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D124%26rft.issue%253D5%26rft.spage%253D439%26rft.epage%253D450%26rft.atitle%253DMissing%2Bdata%2Bimputation%2Band%2Bhaplotype%2Bphase%2Binference%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-008-0568-7%26rft_id%253Dinfo%253Apmid%252F18850115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-008-0568-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18850115&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261178900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.3" data-doi="10.1073/pnas.0909559107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oksenberg</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genome-wide patterns of population structure and admixture in West Africans and African Americans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">786</span>â<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909559107%26rft_id%253Dinfo%253Apmid%252F20080753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMi83ODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.4" data-doi="10.1146/annurev.genom.9.081307.164258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">African genetic diversity: implications for human demographic history, modern human origins, and complex disease mapping.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">403</span>â<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DCampbell%26rft.auinit1%253DM.%2BC.%26rft.volume%253D9%26rft.spage%253D403%26rft.epage%253D433%26rft.atitle%253DAfrican%2Bgenetic%2Bdiversity%253A%2Bimplications%2Bfor%2Bhuman%2Bdemographic%2Bhistory%252C%2Bmodern%2Bhuman%2Borigins%252C%2Band%2Bcomplex%2Bdisease%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164258%26rft_id%253Dinfo%253Apmid%252F18593304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18593304&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259629000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.5" data-doi="10.1016/j.cub.2009.11.050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The evolution of human genetic and phenotypic variation in Africa.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">R166</span>â<span class="cit-lpage">R173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DCampbell%26rft.auinit1%253DM.%2BC.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253DR166%26rft.epage%253DR173%26rft.atitle%253DThe%2Bevolution%2Bof%2Bhuman%2Bgenetic%2Band%2Bphenotypic%2Bvariation%2Bin%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.11.050%26rft_id%253Dinfo%253Apmid%252F20178763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.11.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178763&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275149200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.6" data-doi="10.1093/hmg/ddn288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Bakker</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raychaudhuri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Practical aspects of imputation-driven meta-analysis of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">R122</span>â<span class="cit-lpage">R128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddn288%26rft_id%253Dinfo%253Apmid%252F18852200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddn288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852200&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262764200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.7" data-doi="10.1007/s00439-009-0627-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Egyud</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gajdos</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tischfield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Marchand</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Use of weighted reference panels based on empirical estimates of ancestry for capturing untyped variation.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">295</span>â<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DEgyud%26rft.auinit1%253DM.%2BR.%26rft.volume%253D125%26rft.issue%253D3%26rft.spage%253D295%26rft.epage%253D303%26rft.atitle%253DUse%2Bof%2Bweighted%2Breference%2Bpanels%2Bbased%2Bon%2Bempirical%2Bestimates%2Bof%2Bancestry%2Bfor%2Bcapturing%2Buntyped%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-009-0627-8%26rft_id%253Dinfo%253Apmid%252F19184111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-009-0627-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19184111&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264480500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.8" data-doi="10.1371/journal.pgen.1000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Practical issues in imputation-based association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGuan%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000279%26rft.epage%253De1000279%26rft.atitle%253DPractical%2Bissues%2Bin%2Bimputation-based%2Bassociation%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000279%26rft_id%253Dinfo%253Apmid%252F19057666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19057666&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.9" data-doi="10.1093/bioinformatics/btn522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">A flexible forward simulator for populations subject to selection and demography.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2786</span>â<span class="cit-lpage">2787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn522%26rft_id%253Dinfo%253Apmid%252F18842601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18842601&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261168600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.10" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.11" data-doi="10.1016/j.ajhg.2009.01.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singleton</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span>â<span class="cit-article-title">Genotype-imputation accuracy across worldwide human populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">235</span>â<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D235%26rft.epage%253D250%26rft.atitle%253DGenotype-imputation%2Baccuracy%2Bacross%2Bworldwide%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.013%26rft_id%253Dinfo%253Apmid%252F19215730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19215730&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.12" data-doi="10.1016/j.ajhg.2009.09.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span>â<span class="cit-article-title">The relationship between imputation error and statistical power in genetic association studies in diverse populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">692</span>â<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DL.%26rft.volume%253D85%26rft.issue%253D5%26rft.spage%253D692%26rft.epage%253D698%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Bimputation%2Berror%2Band%2Bstatistical%2Bpower%2Bin%2Bgenetic%2Bassociation%2Bstudies%2Bin%2Bdiverse%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.09.017%26rft_id%253Dinfo%253Apmid%252F19853241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.09.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19853241&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.13" data-doi="10.1038/ng.388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jallow</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teo</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Small</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockett</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deloukas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genome-wide and fine-resolution association analysis of malaria in West Africa.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">657</span>â<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJallow%26rft.auinit1%253DM.%26rft.volume%253D41%26rft.issue%253D6%26rft.spage%253D657%26rft.epage%253D665%26rft.atitle%253DGenome-wide%2Band%2Bfine-resolution%2Bassociation%2Banalysis%2Bof%2Bmalaria%2Bin%2BWest%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.388%26rft_id%253Dinfo%253Apmid%252F19465909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19465909&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266411700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.14" data-doi="10.1038/ejhg.2011.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jostins</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morley</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Imputation of low-frequency variants using the HapMap3 benefits from large, diverse reference sets.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">662</span>â<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DJostins%26rft.auinit1%253DL.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D662%26rft.epage%253D666%26rft.atitle%253DImputation%2Bof%2Blow-frequency%2Bvariants%2Busing%2Bthe%2BHapMap3%2Bbenefits%2Bfrom%2Blarge%252C%2Bdiverse%2Breference%2Bsets.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2011.10%26rft_id%253Dinfo%253Apmid%252F21364697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2011.10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21364697&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.15" data-doi="10.1089/cmb.2007.0133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandoiu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genotype error detection using Hidden Markov Models of haplotype diversity.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1155</span>â<span class="cit-lpage">1171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DKennedy%26rft.auinit1%253DJ.%26rft.volume%253D15%26rft.issue%253D9%26rft.spage%253D1155%26rft.epage%253D1171%26rft.atitle%253DGenotype%2Berror%2Bdetection%2Busing%2BHidden%2BMarkov%2BModels%2Bof%2Bhaplotype%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2007.0133%26rft_id%253Dinfo%253Apmid%252F18973433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2007.0133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18973433&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260958500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.457.16" data-doi="10.1073/pnas.0404730102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimmel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shamir</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">GERBIL: genotype resolution and block identification using likelihood.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">158</span>â<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404730102%26rft_id%253Dinfo%253Apmid%252F15615859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDIvMS8xNTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.457.17" data-doi="10.1038/ng.216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frigge</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gylfason</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zusmanovich</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Detection of sharing by descent, long-range phasing and haplotype imputation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1068</span>â<span class="cit-lpage">1075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D9%26rft.spage%253D1068%26rft.epage%253D1075%26rft.atitle%253DDetection%2Bof%2Bsharing%2Bby%2Bdescent%252C%2Blong-range%2Bphasing%2Band%2Bhaplotype%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.216%26rft_id%253Dinfo%253Apmid%252F19165921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165921&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258761200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.457.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2213</span>â<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D165%26rft.spage%253D2213%26rft_id%253Dinfo%253Apmid%252F14704198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14704198&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187805300045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.19" data-doi="10.1146/annurev.genom.9.081307.164242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanna</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genotype imputation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">387</span>â<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.spage%253D387%26rft.epage%253D406%26rft.atitle%253DGenotype%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164242%26rft_id%253Dinfo%253Apmid%252F19715440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19715440&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.20" data-doi="10.1002/gepi.20533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">816</span>â<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D816%26rft.epage%253D834%26rft.atitle%253DMaCH%253A%2Busing%2Bsequence%2Band%2Bgenotype%2Bdata%2Bto%2Bestimate%2Bhaplotypes%2Band%2Bunobserved%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20533%26rft_id%253Dinfo%253Apmid%252F21058334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21058334&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284719100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.21" data-doi="10.1101/gr.117259.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidore</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehnke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Low coverage sequencing: implications for the design of complex trait association studies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">940</span>â<span class="cit-lpage">951</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.117259.110%26rft_id%253Dinfo%253Apmid%252F21460063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.22" data-doi="10.1038/ng.572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tozzi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterworth</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillai</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muglia</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Meta-analysis and imputation refines the association of 15q25 with smoking quantity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">436</span>â<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLiu%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D42%26rft.issue%253D5%26rft.spage%253D436%26rft.epage%253D440%26rft.atitle%253DMeta-analysis%2Band%2Bimputation%2Brefines%2Bthe%2Bassociation%2Bof%2B15q25%2Bwith%2Bsmoking%2Bquantity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.572%26rft_id%253Dinfo%253Apmid%252F20418889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.572&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20418889&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277179500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.23" data-doi="10.1038/nature07632"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Malaria Genomic Epidemiology Network</span>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">A global network for investigating the genomic epidemiology of malaria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">732</span>â<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D456%26rft.issue%253D7223%26rft.spage%253D732%26rft.epage%253D737%26rft.atitle%253DA%2Bglobal%2Bnetwork%2Bfor%2Binvestigating%2Bthe%2Bgenomic%2Bepidemiology%2Bof%2Bmalaria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07632%26rft_id%253Dinfo%253Apmid%252F19079050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079050&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261559900037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.24" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>â<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.25" data-doi="10.1038/ng2088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">A new multipoint method for genome-wide association studies by imputation of genotypes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">906</span>â<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D39%26rft.issue%253D7%26rft.spage%253D906%26rft.epage%253D913%26rft.atitle%253DA%2Bnew%2Bmultipoint%2Bmethod%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bby%2Bimputation%2Bof%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2088%26rft_id%253Dinfo%253Apmid%252F17572673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17572673&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247619800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.26" data-doi="10.1038/nrg2344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genome-wide association studies for complex traits: consensus, uncertainty and challenges.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">356</span>â<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMcCarthy%26rft.auinit1%253DM.%2BI.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D356%26rft.epage%253D369%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bcomplex%2Btraits%253A%2Bconsensus%252C%2Buncertainty%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2344%26rft_id%253Dinfo%253Apmid%252F18398418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18398418&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255057300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.27" data-doi="10.1002/gepi.20505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avinery</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skibola</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bracci</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">A generic coalescent-based framework for the selection of a reference panel for imputation.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">773</span>â<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DPasaniuc%26rft.auinit1%253DB.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D773%26rft.epage%253D782%26rft.atitle%253DA%2Bgeneric%2Bcoalescent-based%2Bframework%2Bfor%2Bthe%2Bselection%2Bof%2Ba%2Breference%2Bpanel%2Bfor%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20505%26rft_id%253Dinfo%253Apmid%252F21058333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21058333&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.28" data-doi="10.1016/j.ajhg.2010.08.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Inference of unexpected genetic relatedness among individuals in HapMap Phase III.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">457</span>â<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPemberton%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D87%26rft.issue%253D4%26rft.spage%253D457%26rft.epage%253D464%26rft.atitle%253DInference%2Bof%2Bunexpected%2Bgenetic%2Brelatedness%2Bamong%2Bindividuals%2Bin%2BHapMap%2BPhase%2BIII.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.08.014%26rft_id%253Dinfo%253Apmid%252F20869033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.08.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20869033&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283037600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.29" data-doi="10.1016/j.gde.2006.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">African human diversity, origins and migrations.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">597</span>â<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DReed%26rft.auinit1%253DF.%2BA.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D597%26rft.epage%253D605%26rft.atitle%253DAfrican%2Bhuman%2Bdiversity%252C%2Borigins%2Band%2Bmigrations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2006.10.008%26rft_id%253Dinfo%253Apmid%252F17056248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2006.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17056248&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242647400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.30" data-doi="10.1038/nrg2760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szpiech</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jankovic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genome-wide association studies in diverse populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">356</span>â<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D356%26rft.epage%253D366%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bdiverse%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2760%26rft_id%253Dinfo%253Apmid%252F20395969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20395969&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276771400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.457.31" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>â<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.32" data-doi="10.1371/journal.pgen.0030114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Imputation-based analysis of association studies: candidate regions and quantitative traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DServin%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De114%26rft.epage%253De114%26rft.atitle%253DImputation-based%2Banalysis%2Bof%2Bassociation%2Bstudies%253A%2Bcandidate%2Bregions%2Band%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030114%26rft_id%253Dinfo%253Apmid%252F17676998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676998&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.33" data-doi="10.1534/genetics.110.120907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranger</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Progress and promise of genome-wide association studies for human complex trait genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">367</span>â<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120907%26rft_id%253Dinfo%253Apmid%252F21115973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzIvMzY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.457.34" data-doi="10.1038/nature09534"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The 1000 Genomes Project Consortium</span>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">A map of human genome variation from population-scale sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>â<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.35" data-doi="10.1038/nature04226"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The International HapMap Consortium</span>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">A haplotype map of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1299</span>â<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D437%26rft.issue%253D7063%26rft.spage%253D1299%26rft.epage%253D1320%26rft.atitle%253DA%2Bhaplotype%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04226%26rft_id%253Dinfo%253Apmid%252F16255080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16255080&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232829100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.36" data-doi="10.1038/nature09298"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The International HapMap Consortium</span>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Integrating common and rare genetic variation in diverse human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">52</span>â<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%2BM.%26rft.volume%253D467%26rft.issue%253D7311%26rft.spage%253D52%26rft.epage%253D58%26rft.atitle%253DIntegrating%2Bcommon%2Band%2Brare%2Bgenetic%2Bvariation%2Bin%2Bdiverse%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09298%26rft_id%253Dinfo%253Apmid%252F20811451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20811451&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281461200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.457.37" data-doi="10.1038/nature05911"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The Wellcome Trust Case Control Consortium</span>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">661</span>â<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D661%26rft.epage%253D678%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B14%252C000%2Bcases%2Bof%2Bseven%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05911%26rft_id%253Dinfo%253Apmid%252F17554300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554300&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.38" data-doi="10.1038/ng1946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranciaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babbitt</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Convergent adaptation of human lactase persistence in Africa and Europe.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">31</span>â<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTishkoff%26rft.auinit1%253DS.%2BA.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D40%26rft.atitle%253DConvergent%2Badaptation%2Bof%2Bhuman%2Blactase%2Bpersistence%2Bin%2BAfrica%2Band%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1946%26rft_id%253Dinfo%253Apmid%252F17159977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17159977&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243136500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.39" data-doi="10.1126/science.1172257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedlaender</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehret</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranciaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The genetic structure and history of Africans and African Americans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">1035</span>â<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1172257%26rft_id%253Dinfo%253Apmid%252F19407144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjQvNTkzMC8xMDM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.40" data-doi="10.2217/14622416.10.2.191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeggini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Meta-analysis in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Pharmacogenomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">191</span>â<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacogenomics%26rft.stitle%253DPharmacogenomics%26rft.aulast%253DZeggini%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D191%26rft.epage%253D201%26rft.atitle%253DMeta-analysis%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.2217%252F14622416.10.2.191%26rft_id%253Dinfo%253Apmid%252F19207020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/14622416.10.2.191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19207020&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263811400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.457.41" data-doi="10.1002/gepi.20552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A comparison of approaches to account for uncertainty in analysis of imputed genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">102</span>â<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DZheng%26rft.auinit1%253DJ.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D102%26rft.epage%253D110%26rft.atitle%253DA%2Bcomparison%2Bof%2Bapproaches%2Bto%2Baccount%2Bfor%2Buncertainty%2Bin%2Banalysis%2Bof%2Bimputed%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20552%26rft_id%253Dinfo%253Apmid%252F21254217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21254217&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/6/457.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/451" title="The SPRED1 Variants Repository for Legius Syndrome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/6/471" title="Modulation of the Activity of a Polycomb-Group Response Element in Drosophila by a Mutation in the Transcriptional Activator Woc" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/6"><span>Volume 1 Issue 6, November 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/6.cover.gif?itok=aUjFCEJ8" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/457" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DGWIP4_KaN7QZRt42UIJojA6AHyP9NMYy7VfHeXPGt4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/6/457" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/6/457" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/6/457" class="active" data-icon-position="" data-hide-link-title="0">Genotype Imputation with Thousands of Genomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459194" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UhCpCm84PF71RKK7b-Qp9kYfsGknaWdpsVUKBMtHZgk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/457.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/6/457" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-7oDr_VhxhGvtoiESjAVZwofZKsKsHekFFYOsBwv2CrM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/6/457" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/457.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459194" id="node459194--2" data-pisa="ggg;1/6/457" data-pisa-master="ggg;g3.111.001198" data-apath="/ggg/1/6/457.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Imputation with Thousands of Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bryan Howie</span>, <span class="highwire-citation-author" data-delta="1">Jonathan Marchini</span> and <span class="highwire-citation-author" data-delta="2">Matthew Stephens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">457-470;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001198 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459194--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bryan Howie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bryan%2BHowie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howie%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABryan%2BHowie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bhowie@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">bhowie@uchicago.edu</a>
<a href="mailto:marchini@stats.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">marchini@stats.ox.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan Marchini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Statistics, University of Oxford, Oxford OX1 3TG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BMarchini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marchini%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BMarchini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bhowie@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">bhowie@uchicago.edu</a>
<a href="mailto:marchini@stats.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">marchini@stats.ox.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthew Stephens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics and</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Statistics, University of Chicago, Chicago, Illinois 60637, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BStephens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephens%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BStephens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/6/457" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/6/457&amp;title=Genotype%20Imputation%20with%20Thousands%20of%20Genomes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/6/457&amp;title=Genotype%20Imputation%20with%20Thousands%20of%20Genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/6/457&amp;title=Genotype%20Imputation%20with%20Thousands%20of%20Genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/6/457&amp;text=Genotype%20Imputation%20with%20Thousands%20of%20Genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/6/457&amp;title=Genotype%20Imputation%20with%20Thousands%20of%20Genomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/6/457&amp;t=Genotype%20Imputation%20with%20Thousands%20of%20Genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/6/457&amp;title=Genotype%20Imputation%20with%20Thousands%20of%20Genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/6/457&amp;title=Genotype%20Imputation%20with%20Thousands%20of%20Genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459194/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459194" id="node459194--3" data-pisa="ggg;1/6/457" data-pisa-master="ggg;g3.111.001198" data-apath="/ggg/1/6/457.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotype Imputation with Thousands of Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bryan Howie</span>, <span class="highwire-citation-author" data-delta="1">Jonathan Marchini</span> and <span class="highwire-citation-author" data-delta="2">Matthew Stephens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">457-470;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001198 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459194--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bryan Howie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bryan%2BHowie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howie%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABryan%2BHowie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bhowie@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">bhowie@uchicago.edu</a>
<a href="mailto:marchini@stats.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">marchini@stats.ox.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan Marchini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Statistics, University of Oxford, Oxford OX1 3TG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BMarchini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marchini%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BMarchini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bhowie@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">bhowie@uchicago.edu</a>
<a href="mailto:marchini@stats.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">marchini@stats.ox.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthew Stephens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Human Genetics and</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Statistics, University of Chicago, Chicago, Illinois 60637, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BStephens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephens%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BStephens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=457&amp;rft.epage=470&amp;rft.atitle=Genotype%20Imputation%20with%20Thousands%20of%20Genomes&amp;rft.volume=1&amp;rft.issue=6&amp;rft.date=2011-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Howie%2C+Bryan&amp;rft.au=Marchini%2C+Jonathan&amp;rft.au=Stephens%2C+Matthew"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459194/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459194/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459194/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459194/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459194/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459194/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459194/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459194/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459194/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459194/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459194/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459194/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/6/457" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/6/457.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/6/457.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/6/457.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/6/457.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
