<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/6/479" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/6/479.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/6/479.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/6/479.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/1/6/479/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/6/479.atom" />
<meta name="HW.pisa" content="ggg;1/6/479" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel" />
<meta name="DC.Identifier" content="10.1534/g3.111.001016" />
<meta name="DC.Date" content="2011-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2011 Eriksson, Manica. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Although ascertainment bias in single nucleotide polymorphisms is a well-known problem, it is generally accepted that microsatellites have mutation rates too high for bias to be a concern. Here, we analyze in detail the large set of microsatellites typed for the Human Genetic Diversity Panel (HGDP)-CEPH panel. We develop a novel framework based on rarefaction to compare heterozygosity across markers with different mutation rates. We find that, whereas di- and tri-nucleotides show similar patterns of within- and between-population heterozygosity, tetra-nucleotides are inconsistent with the other two motifs. In addition, di- and tri-nucleotides are consistent with 16 unbiased tetra-nucleotide markers, whereas the HPGP-CEPH tetra-nucleotides are significantly different. This discrepancy is due to the HGDP-CEPH tetra-nucleotides being too homogeneous across Eurasia, even after their slower mutation rate is taken into account by rarefying the other markers. The most likely explanation for this pattern is ascertainment bias. We strongly advocate the exclusion of tetra-nucleotides from future population genetics analysis of this dataset, and we argue that other microsatellite datasets should be investigated for the presence of bias using the approach outlined in this article." />
<meta name="DC.Contributor" content="Anders Eriksson" />
<meta name="DC.Contributor" content="Andrea Manica" />
<meta name="article:published_time" content="2011-11-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Although ascertainment bias in single nucleotide polymorphisms is a well-known problem, it is generally accepted that microsatellites have mutation rates too high for bias to be a concern. Here, we analyze in detail the large set of microsatellites typed for the Human Genetic Diversity Panel (HGDP)-CEPH panel. We develop a novel framework based on rarefaction to compare heterozygosity across markers with different mutation rates. We find that, whereas di- and tri-nucleotides show similar patterns of within- and between-population heterozygosity, tetra-nucleotides are inconsistent with the other two motifs. In addition, di- and tri-nucleotides are consistent with 16 unbiased tetra-nucleotide markers, whereas the HPGP-CEPH tetra-nucleotides are significantly different. This discrepancy is due to the HGDP-CEPH tetra-nucleotides being too homogeneous across Eurasia, even after their slower mutation rate is taken into account by rarefying the other markers. The most likely explanation for this pattern is ascertainment bias. We strongly advocate the exclusion of tetra-nucleotides from future population genetics analysis of this dataset, and we argue that other microsatellite datasets should be investigated for the presence of bias using the approach outlined in this article.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/11/01" />
<meta name="citation_mjid" content="ggg;1/6/479" />
<meta name="citation_id" content="1/6/479" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/6/479" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/6/479.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/6/479.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/6/479.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001016" />
<meta name="citation_pmid" content="22384358" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="479" />
<meta name="citation_lastpage" content="488" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anders Eriksson" />
<meta name="citation_author_institution" content="Evolutionary Ecology Group, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, United Kingdom" />
<meta name="citation_author_email" content="aje44@cam.ac.uk" />
<meta name="citation_author" content="Andrea Manica" />
<meta name="citation_author_institution" content="Evolutionary Ecology Group, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, United Kingdom" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. M. Akey;citation_author=K. Zhang;citation_author=M. M. Xiong;citation_author=L. Jin;citation_title=The effect of single nucleotide polymorphism identification strategies on estimates of linkage disequilibrium.;citation_volume=20;citation_year=2003;citation_pmid=12598690;citation_doi=10.1093/molbev/msg032" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. M. Bowcock;citation_author=A. Ruiz Linares;citation_author=J. Tomfohrde;citation_author=E. Minch;citation_author=J. R. Kidd;citation_title=High resolution of human evolutionary trees with polymorphic microsatellites.;citation_pages=455-457;citation_volume=368;citation_year=1994;citation_issue=6470;citation_pmid=7510853;citation_doi=10.1038/368455a0" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Bustamante;citation_author=A. Fledel-Alon;citation_author=S. Williamson;citation_author=R. Nielsen;citation_author=M. T. Hubisz;citation_title=Natural selection on protein-coding genes in the human genome.;citation_pages=1153-1157;citation_volume=437;citation_year=2005;citation_issue=7062;citation_pmid=16237444;citation_doi=10.1038/nature04240" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. M. Cann;citation_author=C. de Toma;citation_author=L. Cazes;citation_author=M.-F. Legrand;citation_author=V. Morel;citation_title=A human genome diversity cell line panel.;citation_pages=261-262;citation_volume=296;citation_year=2002;citation_issue=5566;citation_pmid=11954565;citation_doi=10.1126/science.296.5566.261b" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. S. Carlson;citation_author=M. A. Eberle;citation_author=M. J. Rieder;citation_author=J. Smith;citation_author=L. Kruglyak;citation_title=Additional SNPs and linkage-disequilibrium analyses are necessary for whole-genome association studies in humans.;citation_pages=518-521;citation_volume=33;citation_year=2003;citation_issue=4;citation_pmid=12652300;citation_doi=10.1038/ng1128" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. Chakraborty;citation_author=M. Kimmel;citation_author=D. N. Stivers;citation_author=L. J. Davison;citation_author=R. Deka;citation_title=Relative mutation rates at di-, tri-, and tetranucleotide microsatellite loci;citation_pages=1041-1046;citation_volume=94;citation_year=1997;citation_issue=3;citation_pmid=9023379;citation_doi=10.1073/pnas.94.3.1041" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. F. Conrad;citation_author=M. Jakobsson;citation_author=G. Coop;citation_author=X. Q. Wen;citation_author=J. D. Wall;citation_title=A worldwide survey of haplotype variation and linkage disequilibrium in the human genome.;citation_pages=1251-1260;citation_volume=38;citation_year=2006;citation_issue=11;citation_pmid=17057719;citation_doi=10.1038/ng1911" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=O. Deshpande;citation_author=S. Batzoglou;citation_author=M. W. Feldman;citation_author=L. L. Cavalli-Sforza;citation_title=A serial founder effect model for human settlement out of Africa;citation_pages=291-300;citation_volume=276;citation_year=2009;citation_issue=1655;citation_pmid=18796400;citation_doi=10.1098/rspb.2008.0750" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Ellegren;citation_title=Microsatellites: simple sequences with complex evolution.;citation_pages=435-445;citation_volume=5;citation_year=2004;citation_issue=6;citation_pmid=15153996;citation_doi=10.1038/nrg1348" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Foll;citation_author=O. Gaggiotti;citation_title=Identifying the environmental factors that determine the genetic structure of populations.;citation_pages=875-891;citation_volume=174;citation_year=2006;citation_pmid=16951078;citation_doi=10.1534/genetics.106.059451" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. S. Friedlaender;citation_author=F. R. Friedlaender;citation_author=F. A. Reed;citation_author=K. K. Kidd;citation_author=J. R. Kidd;citation_title=The genetic structure of Pacific Islanders.;citation_pages=e19-e19;citation_volume=4;citation_year=2008;citation_issue=1;citation_pmid=18208337;citation_doi=10.1371/journal.pgen.0040019" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. J. Haasl;citation_author=B. A. Payseur;citation_title=Multi-locus inference of population structure: a comparison between single nucleotide polymorphisms and microsatellites.;citation_pages=158-171;citation_volume=106;citation_year=2011;citation_issue=1;citation_pmid=20332809;citation_doi=10.1038/hdy.2010.21" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=H. Harpending;citation_author=A. Rogers;citation_title=GENETIC PERSPECTIVES ON HUMAN ORIGINS AND DIFFERENTIATION;citation_pages=361-385;citation_volume=1;citation_year=2000;citation_issue=1;citation_pmid=11701634;citation_doi=10.1146/annurev.genom.1.1.361" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Jakobsson;citation_author=S. W. Scholz;citation_author=P. Scheet;citation_author=J. R. Gibbs;citation_author=J. M. VanLiere;citation_title=Genotype, haplotype and copy-number variation in worldwide human populations.;citation_pages=998-1003;citation_volume=451;citation_year=2008;citation_issue=7181;citation_pmid=18288195;citation_doi=10.1038/nature06742" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Kimura;citation_author=T. Ohta;citation_title=Distribution of allelic frequencies in a finite population under stepwise production of neutral alleles.;citation_pages=2761-2764;citation_volume=72;citation_year=1975;citation_issue=7;citation_pmid=1058491;citation_doi=10.1073/pnas.72.7.2761" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Kimura;citation_author=T. Ohta;citation_title=Stepwise mutation model and distribution of allelic frequencies in a finite population.;citation_pages=2868-2872;citation_volume=75;citation_year=1978;citation_issue=6;citation_pmid=275857;citation_doi=10.1073/pnas.75.6.2868" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. K. Kuhner;citation_author=P. Beerli;citation_author=J. Yamato;citation_author=J. Felsenstein;citation_title=Usefulness of Single Nucleotide Polymorphism Data for Estimating Population Parameters;citation_pages=439-447;citation_volume=156;citation_year=2000;citation_issue=1;citation_pmid=10978306" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Z. Li;citation_author=D. M. Absher;citation_author=H. Tang;citation_author=A. M. Southwick;citation_author=A. M. Casto;citation_title=Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation;citation_pages=1100-1104;citation_volume=319;citation_year=2008;citation_issue=5866;citation_pmid=18292342;citation_doi=10.1126/science.1153717" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=H. Liu;citation_author=F. Prugnolle;citation_author=A. Manica;citation_author=F. Balloux;citation_title=A geographically explicit genetic model of worldwide human-settlement history.;citation_pages=230-237;citation_volume=79;citation_year=2006;citation_issue=2;citation_pmid=16826514;citation_doi=10.1086/505436" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=A. Manica;citation_author=F. Prugnolle;citation_author=F. Balloux;citation_title=Geography is a better determinant of human genetic differentiation than ethnicity.;citation_pages=366-371;citation_volume=118;citation_year=2005;citation_issue=3-4;citation_pmid=16189711;citation_doi=10.1007/s00439-005-0039-3" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Manica;citation_author=W. Amos;citation_author=F. Balloux;citation_author=T. Hanihara;citation_title=The effect of ancient population bottlenecks on human phenotypic variation.;citation_pages=346-348;citation_volume=448;citation_year=2007;citation_issue=7151;citation_pmid=17637668;citation_doi=10.1038/nature05951" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Nei;citation_title=Estimation of average heterozygosity and genetic distance from a small number of individuals.;citation_pages=583-590;citation_volume=89;citation_year=1978;citation_pmid=17248844" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Novembre;citation_author=T. Johnson;citation_author=K. Bryc;citation_author=Z. Kutalik;citation_author=A. R. Boyko;citation_title=Genes mirror geography within Europe.;citation_pages=274;citation_volume=456;citation_year=2008;citation_doi=10.1038/nature07347" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. J. Pemberton;citation_author=C. I. Sandefur;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=Sequence determinants of human microsatellite variability.;citation_pages=612-612;citation_volume=10;citation_year=2009;citation_pmid=20015383;citation_doi=10.1186/1471-2164-10-612" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of population structure using multilocus genotype data.;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_pmid=10835412" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=F. Prugnolle;citation_author=A. Manica;citation_author=F. Balloux;citation_title=Geography predicts neutral genetic diversity of human populations.;citation_pages=R159-R160;citation_volume=15;citation_year=2005;citation_issue=5;citation_pmid=15753023;citation_doi=10.1016/j.cub.2005.02.038" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Ramachandran;citation_author=O. Deshpande;citation_author=C. C. Roseman;citation_author=N. A. Rosenberg;citation_author=M. W. Feldman;citation_title=Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa.;citation_volume=102;citation_year=2005;citation_pmid=16243969;citation_doi=10.1073/pnas.0507611102" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=N. Ray;citation_author=M. Currat;citation_author=P. Berthier;citation_author=L. Excoffier;citation_title=Recovering the geographic origin of early modern humans by realistic and spatially explicit simulations;citation_pages=1161-1167;citation_volume=15;citation_year=2005;citation_issue=8;citation_pmid=16077015;citation_doi=10.1101/gr.3708505" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=A. R. Rogers;citation_author=L. B. Jorde;citation_title=Ascertainment bias in estimates of average heterozygosity.;citation_pages=1033-1041;citation_volume=58;citation_year=1996;citation_issue=5;citation_pmid=8651264" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=I. G. Romero;citation_author=A. Manica;citation_author=J. Goudet;citation_author=L. L. Handley;citation_author=F. Balloux;citation_title=How accurate is the current picture of human genetic variation?;citation_pages=120-126;citation_volume=102;citation_year=2009;citation_issue=2;citation_pmid=18766200;citation_doi=10.1038/hdy.2008.89" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. A. Rosenberg;citation_author=S. Mahajan;citation_author=C. Gonzalez-Quevedo;citation_author=M. Blum;citation_author=L. Nino-Rosales;citation_title=Low levels of genetic divergence across geographically and linguistically diverse populations from India.;citation_pages=2052-2061;citation_volume=2;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=N. A. Rosenberg;citation_title=Standardized subsets of the HGDP-CEPH Human Genome Diversity Cell Line Panel, accounting for atypical and duplicated samples and pairs of close relatives.;citation_pages=841-847;citation_volume=70;citation_year=2006;citation_issue=Pt 6;citation_pmid=17044859;citation_doi=10.1111/j.1469-1809.2006.00285.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. A. Szpiech;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=ADZE: a rarefaction approach for counting alleles private to combinations of populations.;citation_volume=24;citation_year=2008;citation_pmid=18779233;citation_doi=10.1093/bioinformatics/btn478" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. A. Sun;citation_author=J. C. Mullikin;citation_author=N. Patterson;citation_author=D. E. Reich;citation_title=Microsatellites are molecular clocks that support accurate inferences about history.;citation_volume=26;citation_year=2009;citation_pmid=19221007;citation_doi=10.1093/molbev/msp025" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. A. Tishkoff;citation_author=F. Reed;citation_author=F. Friedlaender;citation_author=C. Ehret;citation_author=A. Ranciaro;citation_title=The genetic structure and history of Africans and African Americans.;citation_volume=324;citation_year=2009;citation_pmid=19407144;citation_doi=10.1126/science.1172257" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=S. A. Tishkoff;citation_author=B. C. Verrelli;citation_title=Patterns of human genetic diversity: implications for human evolutionary history and disease.;citation_pages=293-340;citation_volume=4;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Wakeley;citation_author=R. Nielsen;citation_author=S. N. Liu-Cordero;citation_author=K. Ardlie;citation_title=The discovery of single-nucleotide polymorphisms--and inferences about human demographic history.;citation_pages=1332-1347;citation_volume=69;citation_year=2001;citation_issue=6;citation_pmid=11704929;citation_doi=10.1086/324521" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Wang;citation_author=C. M. Lewis;citation_author=M. Jakobsson;citation_author=S. Ramachandran;citation_author=N. Ray;citation_title=Genetic variation and population structure in native Americans.;citation_pages=e185-e185;citation_volume=3;citation_year=2007;citation_issue=11;citation_pmid=18039031;citation_doi=10.1371/journal.pgen.0030185" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=L. A. Zhivotovsky;citation_author=L. Bennett;citation_author=A. M. Bowcock;citation_author=M. W. Feldman;citation_title=Human Population Expansion and Microsatellite Variation;citation_pages=757-767;citation_volume=17;citation_year=2000;citation_issue=5;citation_pmid=10779536;citation_doi=10.1093/oxfordjournals.molbev.a026354" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. A. Zhivotovsky;citation_author=N. A. Rosenberg;citation_author=M. W. Feldman;citation_title=Features of evolution and expansion of modern humans, inferred from genomewide microsatellite markers.;citation_pages=1171-1186;citation_volume=72;citation_year=2003;citation_issue=5;citation_pmid=12690579;citation_doi=10.1086/375120" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="twitter:description" content="Although ascertainment bias in single nucleotide polymorphisms is a well-known problem, it is generally accepted that microsatellites have mutation rates too high for bias to be a concern. Here, we analyze in detail the large set of microsatellites typed for the Human Genetic Diversity Panel (HGDP)-CEPH panel. We develop a novel framework based on rarefaction to compare heterozygosity across markers with different mutation rates. We find that, whereas di- and tri-nucleotides show similar patterns of within- and between-population heterozygosity, tetra-nucleotides are inconsistent with the other two motifs. In addition, di- and tri-nucleotides are consistent with 16 unbiased tetra-nucleotide markers, whereas the HPGP-CEPH tetra-nucleotides are significantly different. This discrepancy is due to the HGDP-CEPH tetra-nucleotides being too homogeneous across Eurasia, even after their slower mutation rate is taken into account by rarefying the other markers. The most likely explanation for this pattern is ascertainment bias. We strongly advocate the exclusion of tetra-nucleotides from future population genetics analysis of this dataset, and we argue that other microsatellite datasets should be investigated for the presence of bias using the approach outlined in this article." />
<meta name="og-title" property="og:title" content="Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/6/479" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Although ascertainment bias in single nucleotide polymorphisms is a well-known problem, it is generally accepted that microsatellites have mutation rates too high for bias to be a concern. Here, we analyze in detail the large set of microsatellites typed for the Human Genetic Diversity Panel (HGDP)-CEPH panel. We develop a novel framework based on rarefaction to compare heterozygosity across markers with different mutation rates. We find that, whereas di- and tri-nucleotides show similar patterns of within- and between-population heterozygosity, tetra-nucleotides are inconsistent with the other two motifs. In addition, di- and tri-nucleotides are consistent with 16 unbiased tetra-nucleotide markers, whereas the HPGP-CEPH tetra-nucleotides are significantly different. This discrepancy is due to the HGDP-CEPH tetra-nucleotides being too homogeneous across Eurasia, even after their slower mutation rate is taken into account by rarefying the other markers. The most likely explanation for this pattern is ascertainment bias. We strongly advocate the exclusion of tetra-nucleotides from future population genetics analysis of this dataset, and we argue that other microsatellite datasets should be investigated for the presence of bias using the approach outlined in this article." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
    <title>Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"dczmairYE1ENh4HGvUx6Qmk2AJTyoBBwZVNqp90VvaM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459198","apath":"\/ggg\/1\/6\/479.atom","pisa":"ggg;1\/6\/479","ac":{"\/ggg\/1\/6\/479.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/6\/479.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/6\/479"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/6\/479"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/6\/479.figures-only":true,"\/content\/1\/6\/479":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VX0bvFOurb15IUgb7juTLRFUrjUj89VDEZy0S7ao5+wrKvQf\\\\\\\/zNDCHbTpekGKqe8J5cTgAZ81gQpZVSRO+fZSyXC0542uYxM3r4c1SziCa2ZJqni+mLfRCvnXD85SlIVe6+1dxVvJzZRBZ1lYy6k5LAMyq90PjvBqkITCsuJIjIkiRi1Uc+Pjns9cEQh7zgSs5\\\\\\\/AvIwEZk40I2ibSV+yg7CfNEwKnPSYp2Docj6SjijYV7HUSD1ITYyZi74pnx1Y06kHXH4ZrvOrxEr+8juVsCifhEVHIDk6+beFKfsI0eg\\\\\\\/Z\\\\\\\/tyXBdhUciQtiST6BsDleoGI01sKwTz8VFIR8Yp9fSqdq1TYh\\\\\\\/aCNZmLHQdrz1zbmZ18u+LetZTER1r3SiKwZCU1Uu\\\\\\\/IcMfMrwmRvoR8GzcGUgp43PUAsP0ZL3+ZeYqIy4bWe4bvZE25RTG+4Sfa+2hN5dLxH6DNsJMdO4MvKior9gYYCodhkMZmtWJ4l2aBb1KasRXyb+uGxxccsC40fsi4k2oMH3BxMOuZr2dg\\\\\\\/o9m\\\\\\\/iY9vrg6cAizxEH5yN+\\\\\\\/dGXJU\\\\\\\/aheWFSRDg5\\\\\\\/njFRf7m\\\\\\\/bBsbtsPIImB8\\\\\\\/EOQKjq3lWbPB35RCoC7\\\\\\\/\\\\\\\/TPhcc3cBkMd5myjBH765634CWvStc4oWVOD2jt76LXNow9iIWjR6wh2tDA1AO2G7CIjRhd8bclT\\\\\\\/5gDDgtqwQhsyxfYMWVjO\\\\\\\/+sTm0FP6VgeE0t7MI8oSaBmWweLe6VWhkO00ucgdYjIbsYbnzp00fwaMSRcykS4FAdZ7ZcBsEFfn+1RVrXygblSCB2+D268nWRMeU0ExKVpHGFgHvepWC0lZ6QaQUqBpHw\\\\\\\/BLokqG91aW5ggnwKYnjW0odkgzJlJ5tyyJiAg51PF\\\\\\\/2eL6p03csI4KpA1YQN\\\\\\\/RVRzhcWGDRaxreziiCfJV3zoDd5JenYhHzWKF3QPWoTTnu+1BPi8+8psDBkEq13SaALTvv7ekfvYzGUxrWKgv0MKpBm8H5Ja5h4E81V99UfksQkSAYNHqE1nCZwiw04dR\\\\\\\/oAJrToa10Whyd7qLHOJOQ9xtqO1CJH920SKespD6k4g37YcyNO9uvNnyetiewUw9a9Um1GmJL5hMjd1klLYRFJDSAR17WinoB3+TKzLINYRoufujPU+piX6VEsr0f7bnFyV6CKq9obhIb7xFXFRa7R4FYdJ1WEsrwAFkJiZ8JftKMDkt8oU4l25vDkJdYS7rJhMEigAsLfrogAzVV94QQAiebjgm+ja\\\\\\\/IGowDtLX0CTNvYTnOo9Fa\\\u0022,\\\u0022iv\\\u0022:\\\u00222R993QyhQcBRiWY+KeUmIw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a03321b50bc057598af68dad530de78d4f0a2cc3e4b047494244105c0e952c02\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oeRbtTftJylOlskfIPUz3SwRsIQKNMjMxHXKg\\\\\\\/rR4tql\\\\\\\/XVr7Put3yfIWv3G91bNtc996ANTTVusE35aXoUh0yN69EjgQIbmQat2lVMu2aIjT39n+egBeaptX0UIaZ2yjoJHuIqjnp\\\\\\\/EOElDswaUlUkZMlARTe2nb+\\\\\\\/g0uyhiS\\\\\\\/dwc7LsxAJlR4CWIwDaRTePtzSucpG3zZGLdUFDFkaHNFBXmKRYgN\\\\\\\/uZI\\\\\\\/dhpRkXAw5wmDWRZjBN+CVK\\\\\\\/+ZPiOplrZo7BQv3JnnA2bu4xAG5DKgXw7R0z2O7bu\\\\\\\/jo0GmKJrNrl\\\\\\\/xDgY6epRGsMcyJ2wF4GuVcMH75t7l0I0W1fSrJjFUJag5\\\\\\\/Q8BPy4q\\\\\\\/yaOqmmEdLTKy7cEwhTqoe4CQZkC8r8KwaBwpgAiQaAk\\\\\\\/sQJqg4BIQn1qpnIOc5ME8XPQ4PfKu\\\\\\\/qujM9kErPhCUG7zmwGxcY6xygr6XylssuGmFhRBxzk4Z8nzIgdHz7X3DqZPEXrpVvJ+N7xqqebTCwcVhu+dKO+Uyl\\\\\\\/NZ6UYivpX8OQPZb7apZSVPACGYjR6uyMjVClliucE3uGbn3ScJWA3TkQo2NgOiDZIRrfx4MGGdBW9JJb9rxyfWWhx3vGL4JhqZC29\\\\\\\/xkPYvLoIbZhxTN7ie53Ug4D7TLr1Md1YiI5svkdIbaQyz+Y9\\\\\\\/cm3oiCSV2AptxLV9lk8himANHcqAPLC\\\\\\\/gwOAdpXihXZAbnAt6IPz1n\\\\\\\/jEfeMBtTsNy3sM+Xm5dvO6IVrx3M8PJpFvrvGXwXYKe0kBPSrHgLcmid4Mn14Nty7F3XrUHnrCYtu320R5IK5bEQtxiE9esylqzlGX8yLRVQN1N3j1iVb78DjyNRjEbAnr2BI7UilLUIcKzDx9shBFE4TxBHdZIYUtizppnT\\\\\\\/fiSFFWs+MSM32jxcWFE7VcmUUsNofZ+B2LdFNhQoyCxVSD3rfwRPWKF1YRYYeM2LTofCYKOO42F\\\\\\\/OLi2dURTjlQpukktiDvsB9+HseJaoOje8hNaZovnz0\\\\\\\/igZkB0ikzNFEnuyspekvhAaCC9WkbREvGUrtMerzW39nWEdLlLitQPQ6JpMx+vMws2abLPMs94YIEAZyyU+uvp2wT6bhKyqUkfzWhWyRDsV7c5BVnhGYX48EPgx5dnkPLLq\\\\\\\/UzOU5EEjqA0iknAtc5wKZ4qZyN4rrT4yYXcO2O2DH5Xi5GhSrA5KhhA9\\\\\\\/jAQ0AbmtxN\\\\\\\/WOlA9RiqXfM69lr8pb\\\\\\\/np7mH4ZTSdIbS6ok4QUDOOpPacFJqkWbKRT7GS5sO2DNMtQr6cy2SlKBBNYOBU6H8F2ZaluPRyFQScqOEFDb3\\\\\\\/wwmQRAgA1N+WJESxRIlX6BB7nruu\\\\\\\/Rdnu1o89JM3xHO0JVoFQmDdz4R4q8RdyjI9CDbTlfBtBm\\\\\\\/+t9dmnxC6kBloBR5cGtLDD1NXydZqENJitUjVbsWxrbz8uatwsFcGGqv4jN9EpBelb8T0E25xCRDGp+si6rR2SSrB1XBMQSooHM7KpXCAUkUA0MxB6A7g3m5RfzqHBE4Lmoav43aAtIVPmFIH3+CNh414FOuK9H1KXOkLH2xSZyp+HGTUkGinJMAy9oaDPdRNKa3ga69YYdFV8O\\\\\\\/sAfgSMYtz4Uqm8k8mqJgecwXqUwWSpVhPO6xU\\\\\\\/H4F4SBr0Er7pxs31NimgFXX+y9IjE4KdTB3K+lp8Og1xOaBtJCkGq4ffXj2v4nyGm4tPkicdAdkJHeUZG3jiudeBCEXgR0GLsMmr\\\\\\\/jF\\\\\\\/pWsiOM2G\\\\\\\/+KBfw88D3YiYrucB40HwpdNxKZ724ELtoIWSKrfs4muJc3HIU0b2hxseqLXQkXyP+9pjoyDO2VO\\\\\\\/tI3+zQ8lddTP9ThCbzsdMFoJKButZkwArmRoYHuCQs0WkEHOSNHvhmK\\\\\\\/Au6ODolOCJSqcD04GjQlkHmrYZmGmNt\\\\\\\/jZRQan2x1Hg4Ps5cfsQw2K+h3JwNDvKxf1EvMI2rkRekiENO9d9B+UuUPZr+dKkSfpCSfBrljXzKgsPg7whPYvAV3QneHgB96p94bE\\\\\\\/OetCOMHXR25w6auaDc0CdE3kyBSAIZhY+b7C2AM94CtdacmiNOFqLr0baXApexzTBJx4Pb828ACeTQlWMlp2Lb9vX9z6pLHJMqY6O5IidYHRpeTGe6HxpkNcbq+Qy5\\\\\\\/cCg7wm2+4vA1SO\\\\\\\/VZzJdl5jdsVYrhs2SeUXD50yurESB6b+1n4frhw+qEEugF3td3xNCWsU+khn0nuqPj5zf34Xsj6W5T73Bj8aoMj0x9\\\\\\\/31mgxDQWi\\\\\\\/YA8wVUzLNufvDt6mTN13evG9gHGPcOE7BjU0Kh4XZ7ShiPyPVYUAffO1taMT75bcjJBx7S+Rbqw7R\\\\\\\/GPe9dxvelV+5N+gsQNhutG8jexIW9G4+55fLdyuMOBvOqvji4IkDKojd+kD8GVHRpmJf1XLvSLwJA4wADZVvJsjk1HQig70mJk9YswfSLG7L3Jq72qPpindI6e1c26QAqnVftoaE54y1Yyd+Yy5lGGkzHuusdAGQ6zQIbNCE9aMuiK7OOatgg9SfEEJHPbw5VEeNzjuXJp4GLDHNw3JxuI7y0HgjePrkzV4LVNvqo6t5dgO71tw5DwnqatiHZo\\\\\\\/JY7T3bUrddsGFW0C3SOlWThGAHnQSUyyLSU2YrMG\\\\\\\/qcdWbvJ9PKTwxvkPldarZkjUg3gFBSKOcOTwEHBB0LEfyYIktb5KXCPpcDo6flzw9Ct5T1Kt3uBGnvHIFSnwR6jmQMXF2\\\\\\\/HVPN+BA5SjEIM1+D9XkqHJ21KrXvwaQKzJ3H1g++R4QKdr1pNXUj2qNFKpMouy0nnxv0llWgCdbdprs1mA3MKlAPT5y\\\\\\\/3JFvE5Yg3mQWyvSCQgp\\\\\\\/FZByjSDaCk2hq4muXe\\\\\\\/82OZ1Jgn5AZ8BmS+hzhXZ\\\\\\\/XbSVOJ+ngcnNaxznzcusWNyc1bw8BbfqgaUOGhWyRAsHTHTG4UoLJZyfeSQyLhvR1J05CYA+79ybfDW2JwJORW229OizdvOwnDewuT0R02ZAcfH3F0xFpingk3sQbPvWXgqMZGCL+1nTYwecU40u7bQFgrblhp3oA+B2kxzv+gpuRzaOehPBxzHIkH7g2G2x7OceM5q4BWjSqy0xi0ZR1ydK7YdA1VGYu1Ou8KCvJecnz\\\\\\\/OmuHFW5G9i9PrM\\\\\\\/UUD0bmSDZD366aOjHHh1eftIXUGkzlmUqQ1kKlQDGNKM\\\\\\\/wy4Ya9R2xk7gDDU\\\\\\\/i48DCLlmkLvFtSiWocLRcmCizddo0OaAdUtHjUqgf1u8EUVGRXymZMf0kERl18zq4NOHAtA3SkjUBEhOKagf77nf2JHdwSlzbpB\\\\\\\/9CWG6JyXjK5xRBokZw2\\\\\\\/4XxQitXxHYLhz9L1ovGskuvl6wcQqZaDxI0XEkrexNLg8CLQmwCNCKMf2i6DFMHLxYkdmP77SalgRlhAbcGzNXcqnhKwdVm\\\\\\\/fHrGVgY3PSfehZG2dzy01j9V7DNzspQEZMavvwZYi2+PsWIYlN+wh+d3UyT5gChTpnD3mwMaX8tsNOpoKMsfyfJEM1HMGogVCwoOMppgnjDOdL+Vb1fMmYsTn7wFlkO5fALwJcxrLYtaEOIY6D+tGc2v09fK\\\\\\\/BFUf4rPiEL26qVPV8VqBWiNV5pGfTErIkVSprb7Is5KLNWh5HE9eEW9VlwFyCmOJbEz1BhOlpEtRXjEk+ciVSMwmctPO\\\\\\\/Pyy5GnXdOF8TjXJlnHw\\\\\\\/PgFvVq7sZ6O0Uf3sn9v3DhDjIs4+ZHiRE9eJqASbevTOr06nXZIJ80yzXAoc6YSePVoSuIuwx8bdRzvyhlTZMIFjaK58KBxorwarCllxyc72UxxWJ7ySbpK6lJ0AzppNdpJrbM3\\\\\\\/+iYkIxG9NtbDepEFdjRpjx3Jy6GcvbG0VypEJs6deIe0FOoFXydSUwhlf\\\\\\\/mHDqy9E1WRIyhtk\\\\\\\/ob16T22kvAu3lETDcx\\\\\\\/2dj5wcsf2K4miR0zdIgzk3BjYFI7h1xXgtG5W91Bv2I7WfCStmptbmRCLcTFwzxSlhYpuB38YIhbZRFzcN485tJZw2J+HZT4xwBqmP97691KoLnSkvEHkUMIhbpCmGNZV7LAWQ8UTPfizXx7MGvI9aVKhvjnC2usMTTj\\\\\\\/bu3YJGjBy1TVt6jRCVdJaASkZezIg4x7RIJEOn26sCVPC23YKZ\\\\\\\/YrHpH9KWTeSLP0aZBwnY4a511SxM26SAN\\\\\\\/K9oWrYIGlk3Dkrm29A15IgNNCqNTE+1N0NToUiAGF4U9r+L2MmvBYJE08fwQzZHIwvo2vPy95MxU5yBpY+9Ng\\\\\\\/BXyveZUIFJePVg4DzIzdy9uNsfGH1K2UqYsTqH+gQgnPjtNAQDTvJD0dSNyOtErBhZqfgEZpMYXJNs0pLq5gcqxJ4HINSqI5XRsRmSfqBZ+1+UesgathJ8RjDgAS6OTdd9Sak1GSh0TLPAiOgVmK3NthVfnA4KcMJMCn4WBdMSzaagrTZqkvoa9apD2RpHRoNIe8nsfA8N267UwWHmwm4c8Yrwh+YWKiZr8ddyQ8wsCDXR7ZDrclsZiCcxObn2TE\\\\\\\/thRdk3YBALpxFE37C4hmpTWhCvH0sEOjjQjdhmuz6gueux4FBd7kR\\\\\\\/Jps1u3KrWwN1d4RADiOVskSsYX3EuqHQXZ+x0RCqfuHepwCx35mjePff4S9mxiN5Lvskbn6WL5t4WjgPpXbwyWwz6trvUy6wxTVoleRdLBsDUNS50MOm72PlLG8bPMwrJVMdMIcpmeszfycWYxnrTx5DDUuLjCDen4zY+Gycx2Fn+e1cr1isOAVoqfcjd8gFhRKpB37yJzMaksRu2A6Eu5rROOAulSqm6PQoG7fxzAbKzGxsS7SjEFxfeeBQJ9yOHz3Mrc0smdQlD9KXbGM8\\\\\\\/M1zTCpjiUyZ2dMppbpT+H5\\\\\\\/ZOQKI2yFbtJ57UqLurcqg2i\\\\\\\/9JqCQ1KzhB6D4JXJDV6d+qZuOYaTi1KLFhxq2Zazf1uTZiK44inF6S+QGNz0U4X4fFt\\\\\\\/d3iaPbR+EW8zS6ol7HX1JLP\\\\\\\/CW1bpDyfr5ZTu+5U6DZERHT2MEWMcC0ECmcFV05NY23uPWRAUhq4YPZZ4gkhJ1Qd1ljz9beDqjyKIAt3n8uQGsKqLQvkacalxiNt214SPIWZT+\\\\\\\/18B4tSLmzVazICyfjtY9R5rNk+Ipaa\\\\\\\/wvf2UTJ7YsA6SUMPe+5nyggdlo2DTOFeKWxs0CEj7+\\\\\\\/QidrpKaL1Yl8ksJp7G6vWBPAdgbzUCy+cHDuAEj84b9J1kNrC6RAScjMy4IOM0cFG5juvUxa0CUUFOhqbJCVgqLmTgjXjol+tVCR6GDAkJjO4YA5t+A8i2lxOS49oyrqzX2fAeWCR3GZL0qgL90dAfXtbZQ84ZalRpyA8PoAC8SdZX\\\\\\\/\\\\\\\/A787sIi0r+8GSFRMr7EY0vwE1qxioRYxfA9UcdwdF9B9b1ePYKwLFJpis3DCrDbaahY9oryajdLFOxI201WXAUX8zTI1SrYmpf53RXEGhlMpanN5t\\\\\\\/gvlGB3rDzRqSiKJs8AR5Wp3ez4esajr+t+zo0szDY58wF2aq89amN31DdjLfvYgu06MM9BzBYETZ\\\\\\\/8PtoACLEuLtR+GtXM7AzQgnztPgleZYH3gY9RsogSWu2MdS4a2XcUlhrbdiPppGU\\\\\\\/KCcmC5UzbHPQ0d22sWRecwYf9BTho8gyw0plAK2BM0ngrnO1kYhj1jlWpjzgnwyBnJm5BzgTR60oDnV+tRLlIvsQ6v6C+eFKOFH4WsXlQFY\\\\\\\/7HTQgMG5RrDWOh2\\\\\\\/kVNhrsjzoO7ZO+Z75Ugt08Ql7eCyQxWaBMJm1pW6ciFZDioMS72spyaQouldoSMIZCJ4vDxD\\\\\\\/MbmuL8ff\\\\\\\/zEBxgraYNWfVEJKCKRXzEaUWoAgvZzM2ngA10DRgcpNVwvBIffmWMrLniKuo\\\\\\\/PzJnKrI58ZGXiQcoqHnd0OPdf9wsn\\\\\\\/FDlzANi96bK9Lac+nT+s7pBEXURtXGV6onWuV9QXU6ANp6DfBRyZNn8vj6LURHlxbdYZV2QJaHrBbxZIvqslu+v08+T42ulXXgzaGNio\\\\\\\/7DYxxlesLsu\\\\\\\/UaA2dyupoLSzg6Y09bE3Zvzf+C\\\\\\\/puXJwLF9CogOquVChAUtQyxJWwyrWlUx5roBjIQ6bP9xTgzQyFTDMa0USV\\\\\\\/WqTtQXOvCvkTkZ3VsvoRQ08tS2VVJswwDPzldXehU4qQBZWx8t6EMvjidG4V9hZ\\\\\\\/kfrUStGFiJZE9wP7qfZDia\\\\\\\/eQ5UISCfv8o5rRSHCziUMVhIShM+4hKr9dt23\\\\\\\/1zpRgeeUAG+fJ59m7bhN0\\\\\\\/6vAHiqif99d3QFSOGJfkmU2xgohELwUIs51BL9syFamkaP\\\\\\\/zVpSHAxYbYr7+B\\\\\\\/zjJTC1d\\\\\\\/B9qslMRH3yW1pYniQZqv12J6iU0rfgXvzNdYQSTh\\\\\\\/QY\\\\\\\/o4qyblPGZLMAe32+MKeUxdIsiJfPrKR92MmkNDX3Nxj+3D6l0JXxYNG+dkYu4UoMW2jPpORNYEiW6rYimfIfW0l8\\\\\\\/n1m8GnMMz6T5CGRyXaUAqppd4\\\\\\\/Oa7Rh2b0oWygwxZsrhaYGRLkm\\\\\\\/xS0Ua3ZhyUfSUbnyl7IRmjxwJIihUbGbI4BxoY5U06SUynPKJQ06qbFRG1MgETvEms6QLMz704rpU\\\\\\\/dGGEaItWF8qAP4pKztKWnb5Sr3XY87xhzGB8bp2JUTI6jdjgwv6AFfB2mFSHyNuYHiR3R1JgAOlR322gskvD4zyOULr2UfOrWjnAcIUYay0SDocUih5LBnltgSrB+iun2HixBrb9otZX6rmDwQFFVrEuwkPc9jc1hgdgG19lcg5V0G1HwDs4odRW1jObHAts5OrQe2PIYWoNpGpXr82NFWN0yRRQWTiNQCZ4hYvONoH1KQ0dXNY0XH75XuUGefNqAzziGjOTxcGViLkJ9K0Gp7cKPXTRYwSjl+o5O+sYkPkTKImzKQHkPPNmYUnijhDbaw4xSGqQIyO21lmDHRZIW3cnw6uM4h9WnmPusXuGKX6Xx3VLPoP8oR0ZYKXt5+qmlpqaZqI4AFzkKvtVTO+TxDFRMmZB+0h0q7iy7sgQXrO6g97zox1wH0HyTkVQIOd843RsfWMAocvAvN7INAdrhu15Qhr5G9yE+y\\\\\\\/55cV+qhCjpq0Y9T8wOhzugbmUUr8ULdTu7U9A5AZRWe8eGhZan9wBOEJx5ITrOadSHTLWNaJmyFUoHChKk9ujAKViedLqkbYpRdu0ri1c2qLJeeCq+Fur6n27TVzJOl7VlErliMcphygwBCzrJ+FBtIsz2gwB4lpJOc7ozMCjnfONmPFVIYIf6ACE+JS2fVr\\\\\\\/p2EnrSgjyCQUBHE8E48YCYjmNEMLW4AGLUT517Q7xBEPU+p9fzPAAvnYZcnMfx+E90wira8s3+5kSVnf0F5STuF0JGQPNgWuLxKkXDoCxil6s8G3kWaqGqYvW3pIB1YE7MoOA9uxx7FG2plNga1kRf\\\\\\\/ydg730R189hJLcG\\\\\\\/ho9EmWk4vwiH7a2bJ4vo5SrJdgCCHZXuT3Ky4nYrC1eBKckZkboczXV3roNYChkZBunkOoPRRbP\\\\\\\/UtW0MlpYDBWP+XPnYL0KbFJqwv1dYrM3cUzo5L3+YBd\\\\\\\/mbcqLYDu2LIggmFuyedPyJmhJgbe37fN9QaPBguKwCTwGzvSQ6hb6G\\\\\\\/oqmoHNIewFsWp9+FzCO4VqvP4IweZEANi1PD63Yt5yRyhQ4L8xpWGPE614vt76dw9Ukyn5FHeEp+ip5liT5d9KtB6QDW9oBQdjGnDYWNsjmvUG8xyZbL\\\\\\\/9i74HsHNJcEOLZzSeiOmrrIO80f3KTyKDa9RygI2\\\\\\\/BmRt\\\\\\\/wgW4pE1ARfSuVwiHh2cf3a+n+9DZcIkMKUOo+mQAQZmbVIgfOFZHuacl5dVR+JrBdGWcewVHJQt7yXuqdtHQbSgU\\\\\\\/Tws\\\\\\\/4r623E5N8omOAli6ONZ0ALZvptddA8tKPRthtwVfgPXZHGTmhJ8Bmo57eujTqGCXOQi72xq7+5W5JvOZnd\\\\\\\/shKZt8bMEoDdekh03n0lYMfH6rFGAumcO7Z3y7Hw8f\\\\\\\/sFB3UfFqjuB\\\\\\\/wkXz6qPJLt2utoMWyvRXwPESdYo7KMpJzxDgLiAwTEFxiErEZBZZa4Z2MvgkjWWo\\\\\\\/tMu+lvuMSNSuxEY\\\\\\\/lqrXVa3n6+xcGorxcHHqTNsdOsSXDjpLoooAaFUfYAXhoGCOvX0t45KiVFlX1QBDPyDjtaJRftlPFE0SCr5TI33A4s\\\\\\\/BmiHprho31KTU+YYepeu5ii0Y6XDDZijHWZH4yg053jMliHcyB7KJoIqcMJwifkeF\\\\\\\/x3ow5KxbhiIq7eXd5q7cG0o8boQbHXCNYAn5+9VcaHpZw+vQCv9g0mGiuPzArbw8S8V8PBu\\\\\\\/Yc4aUVeisZKvGknCUC7LG3Kbaai5j6t0niYdI9xEBzFQ+fotk01pykIo8VchdbfUXej79\\\\\\\/jPCeuuKeqp9mxibwoq3xguEZEdSMrOr9adJqfwv8baCJc+RuhbzcG8Izk\\\\\\\/UZZa0d96CJMXs9mcXC01Q+YWY47zZ\\\\\\\/3oiP3+sDpvNUWwkliiBgy4F73Xq2j4GNEcklobRQ6E+d66A9GRID\\\\\\\/k8alcTT4hFQTfsFvmiaoaCWf5jjkemXWMEsLtcoY9nicLK0HR2QSHH5d0iW5+r7Nlw8znXzeQVYgDXXsthX6bgmBnHljJWsdydFjiLzcDMVGoGz\\\\\\\/KKOOtiYsk3dPJRLBFWFZNffQEHtDowyxKfELtmaLSqXG8s8Ad9jWzqN39wfOoAAdbhwurriRPzCM958M6gPZnAzz9J97CjB+Efij043gfVNWCGxSoLNE7xVyqphgbFXQKKzUdhDG\\\\\\\/E2ITt9QBUjnaHIAiEhXxCcXg\\\\\\\/Pg1QJXzR8MZtY6+Tl7KvRiqJsAX0G5MRgDyVE+WNjtAR9+KFG6FsMmjefzPZg0pSCHV4D05BITk+4Br1mq5MZGdAEyTXNiAVhDvJsljUfZw\\\\\\\/QXqAPbK8gVbeG75JkNp0YecaZ2RSskd07PBMdQCXjDU0jFbIw1FGdGBlLaJHo9SPN\\\\\\\/xi3ubs1JXTqdXFe2SPJHTnwRnXb9LtXD3+i8kByKIg3doTLc\\\\\\\/BgHfl4oY\\\\\\\/HbVL\\\\\\\/zsbD6TY2acSc9KyYD42PwSpCOLIqjBWp51XwgVCeQEJQgk2pw\\\\\\\/eK8bpVmEue+QrLlSeKzWJ5BXgTQnwOZ6peCWvIR7Q0+Qx\\\\\\\/dEtX5XQIYeoULvRdpQz1bYac+9mPtbmVhM5BZ7kqnyAhClddgaobE8VSk3TMHQBdvQdCcNL3DX2i0hWaQqDH8Vx+fFn\\\\\\\/7f2ZLNzBv\\\\\\\/o4k0iWICdioz+qlxl0jJLp7nyCBxnjQ9L\\\\\\\/mDAdlSjQGuLGgG7P6aBDdj10wepboL5ZIE5DUIuO5mxK1fRDHwOiOEyCnahsJlWBqpgWAAbZpys\\\\\\\/O3vhXayQ0RiEVUpIRcjx7CwgrjZLa2CVB0ZRsFNBD9qxqdoMyJXoDW2syl6SJicOPUPl\\\\\\\/tNCVAXigaTSjsAQV1xtqeaOxP3FJbOm+W2egfd49RipzJGH\\\\\\\/LZuQTR9ZYreNzxegOMYTJKHhZtLSYnJQ+GHrghC9T\\\\\\\/1nIEgR9V8w6fAqlghCKG2tQcJY++kguK8u68oIMocRNhCN1SpPiQrxcum31vROzSKgG13twUjct8miIQWL6Nu2lwIY2Qi6aHT0Xo5hUfmkLKNee9b9sLyGW1kFx\\\\\\\/Cuw4YHVcYwggD1Uv07CX3Ac1WNDMS5XhCdGR8MSJPJmvpxTJ3cts0TXmjPdF1lEpE5rV7pzP+S+qwxCdYbMVCyibc6Dz+HHgnWhYwrDEl0ObpzOZc3Qv1XIdeBLRMUZjYK6HybPyk+cT0XCW7IQQrzejt4EPk6R0L2rWGkosieqI4Tdy++awPGcWvT2sk9zVVmGrSYeGvNY9Jb58CoGco2QVrEdGmjAwGBuM3FpOxZut3BK\\\\\\\/F+v1P\\\\\\\/ipRcqBavSZSXXA5TYrMIDL0UVE1fuoDbf5YNuYCCYgjUuysMBHQYbDcIUXaFqJBqBj\\\\\\\/fFuIFaHjriK04zG6NBX\\\\\\\/8SluomxgaxnARS9+rxP05W0pQSGxuVEYHJjUsJBGlzawMRGVwqWLpVIrDix9GDjR8172agmzWkcaXhdDfNespnMzEws7MeDyMvh7Zp4cYHo9nDrWDHXWBozT3WpLNvls4lID0pbbt3IeRSq5h0dKsuA2KGXO0kwQbfSs527ZdbERO3UWVchiBkJLd\\\\\\\/JTCtOUByhHLfQkBJ83WZ80Qdo7cUpGeZ3wSXx0DCOn3fJc8xLdd6gIeuhe1A2+5KXJgad4VGNVxicrlFpCS1G2U6dsSoH+YTdlnuQMjHyXiMsrrZpXvyYJgxcFmGNgQL9r7Kl3nDNoyYtbIQQaLvVLBZGqivJfo+dgoqKOUk\\\\\\\/CYlEHPatRdk6SHB1XdriTdadRdNFHMZdn5EbAirbTL5FB6d0RSDE07szgg5m8uFTOLNdN4FvLmaH\\\\\\\/yKj9vh3r\\\\\\\/zYr2z3woKqqqbmQ8kJY1qb+DWUcg0DDz1SRkztyS2j4psg1HpfvE27AHPWjQp83n4gxqktaLHlRT4VjbA0xaJU0yrfzGugt7zpXnTHRv0GEULJhQJzzxekXKJ\\\\\\\/Xm5LAay01QwUiXI+FnxVub5hRhFQuPERoygQU2tnWQcTZjxTr2\\\\\\\/u1TSXm4Op6ZdwddsD1OoEYW5PQ8ITNR9x79EAqOIMZyLHt9wtw3Q+2IOdOZVczolUK+a1EulgRZWJZxBVn43hYxyhhNQ+m8VqxJLa7ogzykP5N8A6kxch7BMbOStH4B8PgP+vN9TgVjT9mmQ\\\\\\\/WeowAfLbL3z4jXm\\\\\\\/S1pOKPojcGLG3GWSjb4RoI4pWEI2zwua7ovQSh1MaB7pbbWI80z+WUycVr0ZFhkERPHUzpPsCVk9w968ZqhGg8JuifVyS4si2czNGt+HzZxtCIHeE0cHSkvcUXk7nh6WuFgjr+YgFXub3k3OOEC6bXoQr8tus0Fk34eFvQ0pvcIhCgQLcDGeWNhwX3n3TxMuywxENDRudlHCCHrXY9tNfAotcciiOWmKU6uPTGbrOwG0t8qdXLnjiJWiPJ\\\\\\\/3KoPdL\\\\\\\/IFXJQbFJJLq1Bfd+o3yLEZLmOQlng3W8npMfBXOQPc7M7ylk5JvLIMmkd4iqL9I9990v+w+dahAt6ZkkTNcN7oNpap3M8oPhkzjwxE6WMTNqeerBu5bceI1gFK4IBN1qVmCpqlNwbW1s0rjrhPwgWz0+LZ3DyODukpbRqdLL0Jp2nhpWxJjZRW10o+mWnUSEfu6Fsn2koJ22f0qL6gimmgM3s0LzHGQaknDteZoVY1BH9\\\\\\\/2GieZ\\\\\\\/MtttUcEBYIps6eJW302K50vzh3U1KkovOAmKc0OP64TrKwZ3uI6Bbz7ptWIdxhiKgcnXk95crpQvUZdpKmi9BAUmJkhZNJG3Uk4B+xq\\\\\\\/u58if2AmgtvoNQFLAIg07qGBt7rGaFwcmSOYAzeO2PXViEkyVLHZqg74Q1xCu1wRF9i0O7zwjDiHtUZ5\\\\\\\/WZAcjm1DSeGyhG+mU7PotvUcn83CpIsILhhNoVS58Rp5sI0GF+AGeM1kANGmvaKcq3CM\\\\\\\/W+O9ziK0qwwSDDVTkH+W7vOZ0rYWmzJIg21jNatTW3go+Q4Iw4R7xk9mFsncECNm0aml\\\\\\\/7VgAV7TRWZm1zktBBJlsFG9WQ3GLFfV35aki6bBt8hABx4Ai7zqnF55IDaif\\\\\\\/gkLisrLZOZuU30fhy\\\\\\\/4a64k7FJwpoAkoZJTSFfftT6scW\\\\\\\/VMct3y2oC3UmRbP6IXIxQvY+rbBRxMud4jf1GwU5NYqZe9j6GP+\\\\\\\/miOZAyT5+mxzGGeQsCtSeN3t3M\\\\\\\/bvvP3NBa+YY7VTUr2p7gL3uCTcMykKEgPxga\\\\\\\/838BJMULXG18OnDsA1trNffjsQL3\\\\\\\/5vvIl6uwW4mzGkpWTYJWD0r5BIKgyS8rOA7qceBaX+aObJj0Ye9wK7xXxMFccoVw9n1W9MuNlzvAhptWXxXvD\\\\\\\/zQ7wMYTcztEC0MU0iborfSNB4YzCVMkgkEkG3VTGE78XiBbwAPGocBotxdXLMokFjF+F1\\\\\\\/FOBCg8nOh6aHNspoNstmGa9tBMTlajzmPBRRgW31JNs8ZHWsnr4FBIo\\\\\\\/qYpPIpuewj3\\\\\\\/uIdIXnV+ecyKlQ+giR\\\\\\\/6uCviNFxkW0NfSX2c\\\\\\\/l8JhYP\\\\\\\/zBbEvk5s\\\\\\\/UV\\\\\\\/2oGikmLqlvSg0BJYgDltklT5Yz9h4mbDyNHdpRqsP2x7D3VHA274iZLHOKpQwwfg2i2aeVQ9ONfymColczjFZ9iZluiTInNiKVU0vVV2m1S6H3HTFE65u8EK2CsnU71uJMIY9eqiuLDnKifAc9bn+ZdTTDJPsV59DhmHbaoh0X707oxwykAAI044xeUgMNyWxGDo4boPG1ltwQJaxdyic7KLhZym5r\\\\\\\/1BZXzngEwOMyZK3lRtE8Khb5vUcqW3he3ygy058T14hBhG3wLglmg96fPj4inr\\\\\\\/lV7yRxOjg6W50WBTdh7NEIbxTuGeVwfIDQkZ34QIZug7d+C9VxO9u3uFHn6Uul3CM7x49bcyXNveDRtDDXZ2IBCoasz2CsfjeCCZXxyiT9awc9VfnOrJnMf4iZN6GBvcIeQ2S3MjJ32703a6WY4uhjYU4DN9OAn8zMELkI7KDBq0isUdwDUuGAge9hckUpSWYU1i5oZzMbnFU3aJqS\\\\\\\/7PVWUHcvoHiGuWeYP5zTWAskonFbsEW5B5TZVIS7YS9UJB+6bZfa8S0sIs7DYEO\\\\\\\/a2X7Q6hhygGiGFxm9B+I5MK6hXadqTsXmCp8qTOdQEXZCBWZIpte\\\\\\\/FtaPzdeSGimBZcwOJdED6wI5dyk0DPDR0l8q0XVJc2ikst5X8F6hm9AEJr1X8ChX4jFHNM82+pS9YCb8Ygf7vPOFPRLikIve+qZ8tH2CLQwHI7vsyJKsdSVV3pdGXnzAvISeIAFGKO+QUL5QxjcArmA6\\\\\\\/+ZiJu3fgJtDHbdfLd4CaRN9FvIgWM1miv6N1tD5RCSkdnqYqeo6GjgpG5MyZ0QrVD9idnCvJCkqpML+85pde4Jio3Ddf5l0iMnYmieTvMVagpDAI+yZ8iM6MEL4Uv+6SGO4WCzTMwQj6TM+W1wWVgM0\\\\\\\/z6LjaMhSMvBwUiOi2aGM4ekDhkF758rwUbtruER\\\\\\\/TOcXVPYka2gH4CSBmqCNduOFZYE83z6qc2LeMrBgvqShUUcZ6YvXD3mRIIYSXE\\\\\\\/nvakTN6xBcS7SaCLNXBhIeBjbF2U64m6HHuiz\\\\\\\/TFV9MOmDsAKuHWa0Ujw1lPv0k4ZpLhdWaqno3N1+aOXjkIMQbESp3ATKXKLetvS38Z16FuMy4foFvCYuNu0trruRD+WsVVMi5V\\\\\\\/SLDbEId7nhJRIfqWG2A9Bemvy5OwBf6hXiPOZQ8inApBKGLs5quU9E673z0mLwP4bWJWiyRGNYPg5hvbjpOH\\\\\\\/Khp2JhdUQejY04Jlpc\\\\\\\/R3sddS5QAQfrL7Ds54sRBYIJ8VPwJmR0JyGOAzjxm6nngclkd8zd7ogzXomHSNK3VIlAm4WX9A7+dfnz6ZM0ur8jlr8z212lxDu6gjGDbXI47n0Y613gYxoQgZqhwjmLlgDmCOLWWLjiIcKEdVEVlFWY672fH+pfDbnC8aLklKGOFBQyitjFR75+IxoQtS2uFreHZ2BDIvrrQu4svhyvJnvm2iNCFDNGEzZWZfbmCwxSsltnxijn9f7UVTgO\\\\\\\/JY9vnocLRayH68WNBFCLEgOagxgq1GguKTJGWmWlT0C7oKWOxi6CqnNIFJxMkrAGQOL\\\\\\\/Fv3OMDtrAsMLKzV6MOIh9KxKBcZJM3+SagVt\\\\\\\/m8qLOUiTLpR3\\\\\\\/J2pW5xLKnH4g4meAGEoioNAdJ+0qtbexobJHXYT4fgpe\\\\\\\/3\\\\\\\/fxaEMAEMp0wC\\\\\\\/IyMixLkIls7f40jLHvLqoBbMLgQfF3\\\\\\\/MN9MdJIGLkrdPY+kPz0Eqh47b5UXwwTfk8nCCbbbHU1RbPN+G3jbArUTQXw7wX+FDHmm6ut3K018xs4oP8BGFByOhrdTmmQIVMgjt1CZnTbgWphoUvsUj4EFIS5BUIhCw9ymTxJvkuTKGOlnnJSlmuZOzyHcRHahxDisg66eF6R6\\\\\\\/Z0IKrzCwo5sWwGhGM5FTJvBTZY5B3ti2M8UjlvVLRsSGuDjnVrkotS5e9pGWf061ktaZKWlP0xcy8qRHK+pUfoZTRlPRfauShxg\\\\\\\/827uFNN2lWz1s24mxtyLaglFb1QtLSTW0zAhJomw9DoRI+OqiKiN9PMtwIMSzYCoThvcIjaBLNAQUceK+jvKSa1GWejJS92k+Dd+LmUbe71ANjG2\\\\\\\/sFI5W\\\\\\\/v47XOgeUGYH9+44rLgwhvs4DcOnEic+4axLUhK8Qig8JdpCraPPd3FRdK1C03LMA+GB6TLzVKhRwcIXNiFQf5683DCORA\\\\\\\/xayMr5Y+X9n36XoYmVUTddlX5N+jFDhfWlWMligJIg7WNF1G0\\\\\\\/Dm+d0Ch6\\\\\\\/tuzSr6eVV1Dno4xak6bCIJigMP85eWPLlJl+VBhIqu+x\\\\\\\/2k+77puig2t1VnZajAmUlrqpBCulUbivHFUNgEtVUA+zYmLLTv\\\\\\\/I9zGd1YIYMg3GCySpCA7AL9e1QqipeHlBP+hD5b96xu4ksyirHM8Pkd8t5eI\\\\\\\/fpbTGPg7h9zJ84D\\\\\\\/jR6Ptvn0H6C\\\\\\\/VdeRXCqKTIsmV0F7FeKOn5h5ckUlriZJkkjjTi4q1mxUU\\\\\\\/emIpMP2UFSfclV2+bu6Y0o+6Q2uIXJnYeQqAda\\\\\\\/X+8lAeHGaLwCHPnifySDczSAFNKA5HHCzk6YUmdgmX0CGTdzMolnpQQbyB7isZzwLJ732XqFFZxdx4tJV59LuM4fd6FTdqRTZvr12EC92GSgGLmCMwWYV1jw+J7bJysiijOJ1BcdiDiyKL3KqvvHbgYVrIHAmfN7x1fFNpoF5vDp+BYxvTzrQekBRi3qyfS2KGj75CN26vWG1oSM+qhVbxElk21IfhD3WjWTzQQfB25r7Kt2m0W5KsfOeP6kZXZVPq1xM4cRwppO99F+Q+dcq\\\\\\\/VZ1KzTYdlLs7aFgrVULu1sP3AUAmzFDG4ZEmdsEJ7QgiFTOVac\\\\\\\/Ql2Zmk40kAdi5RskAmTMh4KprwheajJrL7BA7EUqqkgrmCOnRV52NHEyhls5PtLu+wdbat1xWiiVSX1ih\\\\\\\/WQoKWQRuRv+Fv6y25AQGWHv1ZxuQwxXCQGgYpZS3qFBBuqt\\\\\\\/ArxpryBoF7SBBrqCaKH7REZCJUMS9i1SgHt\\\\\\\/isAoqChYQnFD1o+3iPTCyZxMFLXMmnbDr7O6XjT2hycTmyw9Oxnb4twDFpKxFDSYtyriIrM8E8SH9ybifjMkxvpK59QqIiaY4lbJ48JTbda+zTU7JCMqVFYpa4bQHwTCmI9aVIlDysyGrqklORDOsII+eoU1Cq5JZeUr37e7RMQG7bO\\\\\\\/bzo+2YWbWTLV0KikKLm816kbwo7VUfv68TiJ7gSaGOpRBCg0\\\\\\\/y3aNwC1OmYn\\\\\\\/ozf9mNb8E+lSgs+XF5kBgeEw3gLsmDlS5v+TuezUFUaFHJRvIFtCaGgaQs8H1LbRtSKXq0I1wzPTTYnYd1HSGNIKD6pJPFJ3aH2PhxkSn6+Fu7jw6h+6\\\\\\\/cL0EUiK\\\\\\\/lpMJDA8rcI5yQRKNIAWSmrPXngIoCssr5iRJa6LP2kvt5kajC+oKVW+2Ay0QpF3Pcv1DpuaaVWu1AyWIPCPi9\\\\\\\/MiNwQOhl9EzTorUJWcRt0yfYNVjxGrvBrf7l1o2yE+Z2+5\\\\\\\/veR\\\\\\\/yODlSjbaZCnJmm4\\\\\\\/JQlzBx9epJwbHw5Hc2TE+2xsOneL5fgsFUAmP74yNKX7yY5gIAjpVTOCZhQmKuiUJyRSnBrBbmIDo8FTnsmQu9rZiGJgmowwyanx1Hp04lv6y6OBU6J\\\\\\\/+l7OUxC7IqG31xq6jSo6Yb0iKFOwTdAUuDC8gcNlvlAujWZAwR93cqfgccxSvVwp9Hws8DoVS4Xwvynl+w8oQM\\\\\\\/vpZ9p\\\\\\\/8eXM8lvs0IRbmIFpJ6qOj2E2igzTPxQ6akFCkzsN5joonG+2qk0poRkCVR7etG0QGeOCrdesxjrflNWTcAyShv4hlUIQZHsSOpZfAGmWJpykt2KM2yxFQ1nGv66xsi1gJFX11M7aUpKey\\\\\\\/X7QdaTbz\\\\\\\/ej44eg9mW0y8o+o49UR0lhL9r6nqqdhOlIJHVQ9dIOq5RxGDBlcBK+RvxHKd2sjWHyTxCb0uYAFZ5Wf9TOQfQQ95zGM6B\\\\\\\/bOy7lQ2L5l+HurUxk94WoVQfdPC3PvvBFI\\\\\\\/kfOp6xmIl31pIjjBq4GEMpc8emcEbhijJsIZEv0j1oYkCMJKchtWs+kI5aLhfcwe+CGq4ONUR8Cv3XaHL9eXNg8fYb7L5i\\\\\\\/S\\\\\\\/FV+NpQ\\\\\\\/nEtaSldGrBfhBX6AxSmyA1VTNInKPjzyIpsEHa7ANEq632nGjxosyJuyf8xmV\\\\\\\/rcZItBCwhI7svb1\\\\\\\/lIUAzaYaVCA20+nvNRLs7+BeAMYHiVfaHB2quCPudsY\\\\\\\/KtjS1ehkM7i5b4Oh+grS6VXdf9c6yuJ1al2BshaGgsFaaMbj0K\\\\\\\/LQg1cb\\\\\\\/e1sJQaPa93NhRnJ+quwBg1l6GDhH7RKgJQw4\\\\\\\/+1ucOOiyt+yZosqUhMxYechu0KSh2uBCNpRMRI3PXuVW9AzmAFfzyeaJYhZiMe1xdwZxRu\\\\\\\/c0bneKyuffYn0btE6mfkzK9YoDT+PJ2MvQKcZVeLhGqI5EB0OnJl1W3bCjOl6vLhVLiZo66+eCX1EcQVMU1vFoAthBcFbUyBWK5EEG+RF2wtCevOS5V151MpkDwE\\\\\\\/J3YocMWKL3ggiRCIvNHfXM2nMXCJYY942VwAPuEaem4msyHxHjL4+gTs\\\\\\\/D18yMD1DZLy71m3qUxLccHddcSsuYAaBPeJiOv3s3jjd9yqp+cICAjxEqoCJfH9IuPFwK2yM0WjcC5e3XUKDGc2Jki0heHG5X2UiI4meRvFjGTKaYWpmjoMnotu7dntSmUNIDZ\\\\\\\/FLq0zB71gprVbjw9IP7YlPOzFmW9rR7vx73fRk5wYjn7VSt\\\\\\\/fRt17AFoOTkd1BbtR0JeYkVvuUwiTCAB1THpJz9LeJ8ZSgKPBFoflQHpKxx6B9z7ULVOemIah01l1zjJTEbOShy0eLZ7ndjEbi0bgfBOXkFIHS6YUGmCtOy50zQ5fCeGyVwHfhJLiJI2igN43cINmG2hC8QrPGWHu+773712LV8RNVcy+4hft55KLSkO9NP6QiQ0EBFF9yBDDjNhZiBR02m\\\\\\\/mmqiMEtqgolxf98IuxvUI+j8Uk0sZHzDEHXfyVp1Ao3w2e+JYa4NmbD3E2NXwcs7jZSMzmcjb3KCuGzbHHHXePjCj0TE8Q3LszOlKpqTwMN6FlM3IXb1+cKWpDpoK+hfF7w7SunB4liHh2iJNMvyxRBX9Ubmy09VnUlpT3gUdfVKNLLBuAQob3Hj8Bm1TVbm\\\\\\\/NquqMheXaYOylBAGgas71HNLWwSazI\\\\\\\/l9NXlK92xX4DCoU56yGbjaxLrAyeFZy2wrAE+bhHKaUswPioRkqT1p07tj0J1TgfM0xWT9Q1619tMXb8qw\\\\\\\/O9n3YlGewpku0aErXT140rEd2bXozI6smNCw\\\\\\\/3l0c+L2UdL+sH8Z8BM0HP16PXmv2Dyi1uh0dIUocy++3CgWsyuGlR85DoQmrfhMcQPnxtAw18IRvgSkHUzI1Vm6X5M+KconioPB64lsXjhVe9+ZTkDF5S3bY2EDBzR0Qkj563am3LBqhLboVXBiPvhkeu\\\\\\\/IaiTY8JgV9dZRfmuyNMfZFmX2PC86RjsJHLQ30BxKlwnB8QIYVn8AKvP6dxrC1u8\\\\\\\/ROn7uUltESUb0HHKmDMsR\\\\\\\/8Bc44vopTuZcVu+S7VpZyC2ezyU\\\\\\\/B+Duxeh+ONTcO+0TSPJAk9zG7lIJnklgn3EpNaMIwVrhZFKtr+qB8JWK0ZCvCNDlnrBcvZvZuHUJCGJByhmqoYunjmkxwANJa+ON3X6b+KNbrNNLqiKMuoi3Kh0fukPBYncyTtRDgwsLz+\\\\\\\/Vl1r7FEPQXLO4HpOfDahYgQtnFVuJKdFBJSSs7\\\\\\\/l4gDcgUme53HFqN0KwXRlsNBfF2VkjbTtRS\\\\\\\/Z4bE18kNw+3oEsy1M6z9Ua6pI3I2gQhJ5l1\\\\\\\/eZDudBL2llVYlNigL8lgfi7VR9hzBiLDbjzwojxQqjrUMZzAw49qdRoE4Wv3COxC7ZbLtkPS8GtxrICxErzTCjwP1M6teef5Ku46YtBqY0e9+XA0mkge9b6iF0nqLZAOWYPS6Eqm1T4KW\\\\\\\/HqzUmPVNvfVFEd15VViJgKRWBO8rX43uztrGJeovjviBRHK2L3IaDvM2RumCkVgWNbkAuQAbH5UsKzH+RXulOToE\\\\\\\/SBJAzzkFRbOPNBfB8+ADYl+M+QsuvpeFe5qG0I8B\\\\\\\/o0g72UXfBD1ZDxKljCQKWImG8Hn9sJAdYhTa0\\\\\\\/yMAaYxWNX+LllIuKEnV1pM8ypbJHUqfOjkV1y+5ydo7b08++jdywqOCf1\\\\\\\/naRQ6ujK9FD63e7Bokwf+TmZH8UBnWQWBR9jIWYqx7aHun5PLW9zUL74AprGfC\\\\\\\/iLTg0PkdmpwNEnelVXANQM1caV9duJAe6v+MtM+OQxB6Wz1Rzr4CS95pVjsJTwnF1ZhnlHIjYxImpcs6hpszSyeSB9qMG2n2+oNJWQ4r7KWYFu7gKo\\\\\\\/ikL+z+jKjCxcj2rnYlxVCuqjPZGGJRBwZNEL2K3rYY4qjg3FX\\\\\\\/z+OIO8CsCrAr7F6yMWUoy9cesg+BcXeeH0KEM3AyIWtsGIr0xiqqzVwi4T6fp9FGFwmUspE\\\\\\\/Dl8sLa0eSs\\\\\\\/Iu5qDR94lMF+u+J1SdxpdBoKA6buwlS5SuzQFPM3t0jBaDpPtR2VJXvUWDZ1nu3tH08wQs1uHUyp5bH0Dn6jZOgnyRq5hZD4L\\\\\\\/F4WaEGunG\\\\\\\/5NKx51cVzQGKxLMAnrIfYgxpdk36IvbrNgtpxVUQGMyGD7sjM8QvcWbNL\\\\\\\/nk08cvY6+HlXA8H6J7tE8bQn6D7wCxuajHiamFyQDYeTfJEsGfG08S6jXv8MO9L1S9\\\\\\\/jLK98vwTBeUugabIXCrKLLtBCYmS8k58+VaBIf77vnL3nVzw41J0JvTZiHirMvcX8HKhmoq+R+zXPpjF7G4yedMGaOunxCoD9LPnoW7nzP9f08ANbqSW+MjlkzbPxKfran4tslQlabMOXZEtXVWHxS\\\\\\\/VSBEZ32S\\\\\\\/0+sMmQFcjTeO7hxc4ZtezfjVkVSRu6WqKtDEieu0r9xPGysCcUhl4PVRPsdiXtn3ixGxECMwGnY5+rWnzJjd1BKM6rMYdE4yKSEvU1JKI72aqlxZH9KkMZAw0lmlbjvmkH4Segd9a61D\\\\\\\/cFGfweACV3p7BsyRTSw58rsok16h00\\\\\\\/1Swen38XOvoJfaLh5FBw2h5mQG8xrf\\\\\\\/cHl1OSbMwRl9EsmjK84HYvaxxwe12Jmdvz0brXTqKyAOPS4QiOcxvGlvQcKCnyGghvTdIjcJNRDWIWSfCXDmsXi0k3PdC71enuwF4SoWiBM7bDCejlZbd4S+DGalq1FfIP2CgLX22OwQoyOlEg20hUwwc1Cj1AccxYnz6VmjOPPvAbhOhUl3eyxZmP8GJ06RdekGe50GHK1RZTrpy4A6ZYVX56iZtpNW5qXSeZVq8TTTj0RP0Xvp52s1qPqGD6FCpjVzTKM44E8HWtj7VlKmmCJAX+XFYMEAxaADSZFvgUazOgszHbnzp\\\\\\\/SBc5\\\\\\\/GN464bDxu2swemfV4HoGWoOqwFuWQLBENvr\\\\\\\/g3UQ+vokLR810+Z1ugmJpX30QFdBufyUX7t1Z5BqUQC8qWqSSF8m7l6JCFECeEFy\\\\\\\/3BS97sLycjZ74JW9O7HxA74T8g2F8KbNZ06KxrtjFohg2MnmIOMtEZ1LSLtl9sRQhUUWiJi4X6LXRvalnNzRrL+oJA\\\\\\\/JcVd+vmtNKBaaUPiyRDO6kV96l5ZTk0yfeV7Fzw1sHa+fqfVZ\\\\\\\/5CorMi4C0xu+gnmac0ILXE5EOE3oQu0NdLg5u8jAehKJvrqmZAmtfZQt+MmyAwf1r6uhy1Uf4g4SXBZjp7hJSFKsctrUWrmhc7U2AYLcmQ+eCDsg+A6hEffyNQ5aYbSQ9GGrVwzZczQPuC+Yo6Rf3pWQgfgT4A+BMZ63HEexWBhBUeuLKIA6ffgJr5A9uCU3g8aY3IIR3l1zXtM+\\\\\\\/KqoUumjM4QNBMiFrXsbMSJfBY8VtvpO6mcCAMOpFzWBurfqVy2jHigFHJPgvJ\\\\\\\/JBf4G09EV\\\\\\\/JrnTBUwbykiriGo9X85bZzrgcbXP1tBHWoZmAzC9dhNgCF1H4xDEtzrO9gILEdfBW6lT4\\\\\\\/i7eruxlbN6Wvju4lVk3OjL9j4J+cC\\\\\\\/udUB3QEPUzDVAVYN4pdxEJJSoM4COH6rbrkuaQ862mo5+1brrlrYKg21hxjruQTr2kmHFhpx0OAdcU7jMObqgw4bbLy3QXDF+zThn1hSzr7\\\\\\\/ic8frMTzoxnXBi7xXg0s+Ne2ii\\\\\\\/asyUXZ1W7DOH49EJPCfL2hQxA+K8KWwuMnPHNGCYPuI7ZIsolcqOuLTSc24TYlphfLJveDO3izHUbwDSEFmNeaCgnbjsXKzSu6nQreSBKHQySBtQRi+x2c0lmdoDREbqHOPiTabEHcqLDZt0kqlQ7QIFgaRs98wPHFD0WCMqoTMUaCSHXPFVNCFZFFhcxkVSyDgYWAvU435njjJEZaeMk38pTMI9jfcJAStd9SwtxiX9XpOktalweGpmRzHV0qYBhblrutJjllWGobUFN35OkN9DJz+KipnMihVFiVWf0TWwVyv7hI2\\\\\\\/TeVirRN3c8NJkPf18F01VqOBSenbFbprtJR7N3TPe9mRh9XwirDFWqwOhTq5mDq4DxFMTI1c4eC+oTF1hZXXdoXrQXyvFjstTLeUfcBkUVIxohCmoCFKf14BCObBNUsIwAVPWa\\\\\\\/+HqlqDtjn\\\\\\\/JhkWOz+u3Y4itM+E+sZUgFatOh1jeE80K\\\\\\\/qp5LNHxZ8Xkns\\\\\\\/s\\\\\\\/0\\\\\\\/g2jGyhV+pY9jsARJOavXOA5XDHjGqBmSzB2B7685wrVqL\\\\\\\/QCYmq4c0US+9VKa0NGAEEtwHWcEJM\\\\\\\/938p0I52rp7MbYo2HufZWKHBSQ2DvHhtVPr3r4tPLMih9z1G4MDlIQDMfbJK93H1TJh2M4QYkW8++8FVbGvXWe0xT18Ucobb+abayNxJ+smqixHudu1qM9Bk\\\\\\\/4XIzfkxtL0LfigiJGrXfU4UvsYZUp\\\\\\\/GLo0\\\\\\\/gnYa5gEMl1uFylMCgotrRphhRduim9pLHJkvh\\\\\\\/xF7Npsi6lTimt652M2x3Myg1oZb7OzujY2QIva0AElQmJsjp9YPHre05Z6qeuoc1ZYpiNIfijvyGPCut3oyYQ8hHgsLXeTkOIBwSwCM9L8SQTbjy+azg8P2LeVxuBpT3ac0pGNa2pGYETXYnmxqd+EiNqYBjnzrjYqzd79QftIMuGU5Tol\\\\\\\/8EvZtblcfqtaxsDh1DPWgfDbfWZQxBWMjYjDmMcPewGswBMQQwhRzWLsIc4MxAxtGwlkx2m9cJ\\\\\\\/cRg+cnYCqaiSIk4NPDttizpU3nGrbmpjcN3+haWhm8TPk3j+SAOM0hSJLVg82uQnfi47LftK1BFE4nmdtdB72SLl5mn77NsNVCLY6HInVmwu1NZjgfDifgigjAph+7pFMZ1fzxtuw0QcgXJ3qNCaA1BH2wqPS9CC10tXDdE7jfAvFVtXiF9KUnpa4mxy08Ej3WO+hmyVK0DXCjEnqcHfgLynQeWn8QXpFzrPubNFR11tdtzEgDjTiCmXUsyblwlfPtqtA+lqtm\\\\\\\/IKSRiwRUysiMVNL0BM\\\\\\\/FVNThiU9SydQ\\\\\\\/JyM+GpklEdZ7ejBUfFNIrsALkFo\\\\\\\/llofx04K11aYf7fHnJv8YI3nE1nWcAbz2n81TKtwoBTvI3V8mcQfQ0O6XvqHa5cjT83lWmc3doMBDj+REyYKUMVXciTISqBIjqnBXmeWpaISjtqui7NS8x8QdSurWdsy0s3OpOtOth1Mg9Bo80d4EjyGr4fQM4M53SZpK0vG1CTrl32iUn6A6zcRB36oUySuqN9ZdI4Wz582\\\\\\\/L\\\\\\\/FVOe0RhByN14Tx0\\\\\\\/rPKgROcZthqil8+\\\\\\\/FuBE+1Sl6RFWdCuSbfkHLgKABtgiXP0DoUtJB72oTvx6S4seBsCK4XhUjPyvIvpP3iUumOC2pkoIT13plDwoUeYVfN2xWqmmwhhC7B5ivU0dCgFdIsgMzT5BN\\\\\\\/1rMiVSYBRRueFEr5EEOjtPzg1wblh1D8FFpOkoiTyWRhuFryLXQ8HL5vJc44iDWpW9dLs99bxmK6LWi3jsES\\\\\\\/JKI2ChFG2HsCYuVOsAFdS1Z+T\\\\\\\/0HN1QEi5xbeio5+R\\\\\\\/OQ9y2rlyfZ1HnaBkXUjKDviGmTilgwcaW4+TxEiNIjV967ZeEhOHy2bNQ7jvfXfR5RMAJrOsSrEM891\\\\\\\/DvxUHU7BsHOCkSgQbMbKAq4nAn9whTBQYml0Icb+1eKrchpCr7brVH+3BYIKl96ihnnqXfFcLy4YrEHZb5y0qvfhRBF1bYBuSgrrLSpfShKcam8J2bJS8OuFydFUPTq54MKhruyD+ib\\\\\\\/MTwylCYPMDyAYmHgcIe2Mg3oMeMPyf7NUVxYAjV2pJJXlXUsUYE8v4xdXK4zim4JcOabiRnjAgb2idIucbq2s1QXzKg5KU3zJSPIvXRcefiJ2MldWojmzr7ADsxOmvDd7n3blQL2o1rpu1hyPyY+AdxfCHr7SqmMhGGaxsFj1Ss7yBjhpbuggv2+YnY4r8\\\\\\\/c6HhSF0Ga7Y2KmTlwMxrBEDC4Vpy3LGoIdzUe+XCicdwUaZoQcfFoVPNYOs+Ik7ibFtdNAZcG0ZMAZKjCQfht4pm8NcHn+232jiY2x359xwlcFIBFn6Ppye7gw+b66t29YOJzo\\\\\\\/ODdyutU4O9Rfy0SNLkK4cuImiNDIxKDfVYci3o5S\\\\\\\/cBB1jf2wn0mWi58K6ZtGxWDEyN8g9I\\\\\\\/dk0+Quz8jOUOdilYOocwYFXXGUr95N8hyfIY1pzi1FCZnEcaEifk88vDmruRCgqwA+5Wl48vI4PxBxAyOjRRe2DVopgwZVsSncCzd9iE3R+n+yLdwYX59Kv1KrYZmiJ5kY+JcujsAFD9gVYY8siWOce5+7aYDVigNBFMouC10FajyAOWKXidVHswYAx6mXKiNkObsdC+Aao72r98aD3NKVcz5IXJ2VqU4XtTXD3bYlD27vil+Tm5FjZ9LUoB0HToS3uHLvp4JL7NI6OmZ303epwoOVm6pwRz3FT3fJtPu1LbQKkMIERVlSIYz6o9\\\\\\\/kbPPsboZIfL4pXuy\\\\\\\/iyuo6JfM9Tr6z3n55VABPLi8iQqJj0XjwkFV4kXeODiXxMczZl6osPX\\\\\\\/PH6fWcSkhWnZP9G0sFFpWLnaU+WDEPG8CGkIwK47afT5752LFof0E8vz1c01tMFliPg145LBuY4DuRarOWSADLQR2h+S70R3V8pW5l7JZnyY5mDQwYEJ61VXRRowXHTWxuzBixud06rguuZQaN0mtA+jp4zFUtzhCVm\\\\\\\/GbkiB8ozwfbdLjHVz7EifAYkHaApb9waGMZMdpSqcsq1tpt05\\\\\\\/L0YRWC9RC8EUS2O\\\\\\\/TdzteSC\\\\\\\/OGeAkmezXkFcwPUhWJTJEEprbNDBX69PuDJV7jTcJs\\\\\\\/3QhJ+ilvsoO6UiiiWgy7pBKGi307XqEqUYK3x58jI3GartUMBLTp01k3UGIsuMBdoNfwu1XesFyk8WaJV+YZmlG6eRwXKn\\\\\\\/lRQVsBAPbTDYVvotRPhYnwqeC6OsRz8tamJHBW9Dxd\\\\\\\/39qPXU3SDgHThIh3xOIiS++4A2KiiiPcyIMad1Roln85kbDPAtodnBGRAJbRURj0wEzf8MwaSojwhBk4cvBoDIqfWf8JNDHHp2yJ80a87xK\\\\\\\/1z1qx3bmUGyiLykQDLo2lpDcJOD\\\\\\\/uqIks2iV5FHrNl0Sg6WBIgBlbX77SknB912kSDRvwZhznZjIF+PA7TUV+8\\\\\\\/Yms4KGpjklfBiohYHwXbwzL02qb20sqwhrsx2uBNSYBMXxHgqOWhZZxpEXRfZp4EfWT1VAq4QJcPB95dQydh6\\\\\\\/A0sHvOho4REHOx5KqndsTl0TWnRrBTZUINz1V03YJdhRzB4vEOslDvnmHM0kRyf34B4BkAnRgBp4T8J8QKtKHsugK0X7VXLTEsoyyW4rsR4x1QaS4gNfC4GVX69\\\\\\\/rc5eaf25oMsXzD7tTyDoAoS6X7+Cr3n2gNNE\\\\\\\/Yah3CLvWf\\\\\\\/0IiyXC01jxSAqxUTXcMWSZxMWkAQmpd+FC2GcDC9CrSkzVeMfBxFkzd\\\\\\\/SglRe\\\\\\\/PBmCEFmJrbLwBExxW2P2iYB96texjNhQQTTvW7VyVeJjsUNvM9dsd7X+CJ4IqNfun\\\\\\\/aNhot1XJp0NFFNnhCXhNXDd7JnvAbj9ThA84R\\\\\\\/wH9fRJsTYBTQIPrk+ji2Ky9JsE5DXyqK6bToHrFISC\\\\\\\/fEdtQad5Jm4m7DCoM4cIWQLAOHoRAII8A9LPAAy0nmr+UcykD4Ub2a04+\\\\\\\/n0wb7NLzBGCn8OfJegtVU5BEQuuEY4EHRVCeOwAqStKPZrLpVCyeSgVzOL6afdNU2qzEy79QPBK3Mt0dZnLNNY0UOW2VRZGfBtaoZAKdDj72djpdSx1ZBvWxlVcJpmP3Xe8oJVO7dr3gW74h5mqsbn2LsuhRQq0GzLNZzVoJPu6xCseZQIL\\\\\\\/I1wuyAqbYMoD78QNgkohFzD13R632AzTLnhj\\\\\\\/wVtBli6MW3pQ8oBax4Aum8wlbYc2vm7AKhxXGuR1DAbCYDucOFtLcxmZso6DIJXY4sijllrjk0BxYMAeWdHdeweeKxVoweIvVGpzjIocNSSHLPKqsP1W6za58KB6RI0B2eIdDpd0MPz6rHwlpkacj\\\\\\\/RRKbwirV+RlpjWaEdQ6WwM4VTxbh+vCOWUjcPjIxWfT90MUFxrtRespDJzWxqlCGZ5YmLbaFLxfX+gSclgb0AbueGCBBny4hwNmU1SaoLNPATZQMii1znGYk3zokVgG9Bjc9Yq\\\\\\\/ZTZukJK9GqcYdMnXxF5IPZ7+39CTj0DlNyA4I+2h643ls3bFSuLnRCiCdL3yysjdM8QjGFVEjk4Rlh38hLZVtgxpymgDJUF9P\\\\\\\/K8hi\\\\\\\/9XQNkMP8l1aieZY08\\\\\\\/w21kSeEztVscuX1Zf91xFje053D6I9rTDKxtyc\\\\\\\/oEzsX7eVRguzKXmnIYMtcLZPd5nSA960vlN22HkF2MI4H\\\\\\\/ng\\\\\\\/s\\\\\\\/pYxGHufZFjsPNVqww330PgzJ63CwadgaqZa\\\\\\\/J+IQjHWsCrfI9ELlh4FqnDN+UYRrfjUjgTVVl+hZJP0RPP6uF8AsASQJaidPU0Q4H17Swz+K48vsOrecNN04eIhtX\\\\\\\/LPzOBEP6Mnxb0sKdhRp6ZPjfyl2ftETvC0C54SxR2DHNcggNiR+UNS1ZMf\\\\\\\/VmBt\\\\\\\/vEcO3Xif5NOzINUOcmK2qAakMelCgDsI8sqJvGSyjRXVX+4OgLOVNYqLIy8ewo2vE6Ge4T4jD19Rb4cfKfkvoBuzkHLhtC8L+Vne2PcPNXYFdXy+KNje6kH5itIQQlKDOF0UvNF1MsCD7fsEgedRnaAkuCUlEXVgZHto\\\\\\\/6DxycJSn6S2afdEwAVjf9zznjl6f2P+46f0t3WGJb8IG3bgInk9dVMeO0EKYUCIiyC3mxMGMxCU2cuaVG1q04NsdOkkyzQoT\\\\\\\/tT7m+k0\\\\\\\/lt1LDVgG\\\\\\\/LFImYgbyuGe7pZif1Igt6fM9oIVdOPa4eQRD40xt5pOuN4BJn\\\\\\\/dqPyIxIO\\\\\\\/V3clb7z\\\\\\\/ZKCzTxj9kyhn0Rtpdx3Tb\\\\\\\/PT\\\\\\\/pHJjuDaiW\\\\\\\/6Y2j+yBgg622tAk9GBt\\\\\\\/J3AFgLCCP+mVdccBsFTlNApZxz\\\\\\\/XMgLp\\\\\\\/UDScVfS+i+eAvE+WSBfPHI0mZmEMBVT7a7ORXrM3ee46gk+7\\\\\\\/QND\\\\\\\/jn3F\\\\\\\/7wWuBh7zeXrrv38sKAO5WPB6iPbLleS11VK69rz54xwwJQI\\\\\\\/O6Pp8cBrk12mvUvX8dFoapkwOlGWrmZyv\\\\\\\/\\\\\\\/jcZc4r8WYaQmJhwTJ1j7DcPDwwHuBw1HpiSHdnHcFkjmbTGc79Cmb52U7VcxEkRG0\\\\\\\/RdosycM9hINWt\\\\\\\/v9HY9mg5yiK614dSxoUdA5P9D0hnXjQ9kTzrC1OKKM8ZJl\\\\\\\/L84kh7eZosS6PZ2+SZmKl8kOMsoZvrKVFdMA7YoiMzmREUtFssrxPvszt6vGvXMIFAKOESdg7yuFXnC4WIF4BeT82v3IY7oFCvK7mrD\\\\\\\/paUSGp0EZ5yb6B1ldwDOG+fvhh3m+7jNxN9eRxe33OIZs7LOwuAYAgnIPV2uC\\\\\\\/c37gWA6vRVSr+MVOcwuSb6q6ei\\\\\\\/Edcc1Gatp7KmiiKDXGXn6VduGOyatvFNT\\\\\\\/VWDY2evdmqMQ2wK4FcucR+K8jz5fQvve\\\\\\\/\\\\\\\/WQ3KcRxvSdS0EwV9fBiMwks\\\\\\\/ywKP4J3Sw\\\\\\\/puiLws48h6ILepvZxSkTIRJM91hnhSnoOzO73kXgL31Qo4NfInj8ZiVKHRPlXVxDD0o1UlzyEgxf1rg79gcNgWT5fXm0CFCdcHKqBuvII8InURYRN68LuzGsgeeahESPjP0Ia0z\\\\\\\/qRTKYxqdeF4oJ+x8NKuplfX6Vs6WraOu\\\\\\\/cmKWhWgJMtaq+ZqgnmwWz1AUT+9Xijv7PhQRkQ+ZQRIHpcm+pz3qWr7Z7gpiYdd0VUAsPLyjnuCQP6js5tKNPDFQfsvJl42jET6arGCmcERs08WjH+xxcB0F\\\\\\\/HPaRV1+IYd0TFMFssoVOJFhQqFWA9DX1S7Vg8LwtrmkTH3bYKipBHdl7IiUYjzaHCd9P\\\\\\\/SC9O\\\\\\\/zyan0rGLc7CAhLyqrP9dPk4TY\\\\\\\/nkXCivOgLsQh+cYLFhfxej\\\\\\\/xD88flz5ajCtooVNbxs2HIQWM+8seW2mllFHjYettZ4Wbdi4dV1olpJ\\\\\\\/owpcVazkg\\\\\\\/fvWD7tWlnpe9LEVwmeVdTYt+W7uaBQ9aBxzTx2MaoiW7W\\\\\\\/wZT\\\\\\\/p+iF3JR0SQHjKHOrizaJJPJBWTHSzSbBFnesR\\\\\\\/e3GpRHHf29A9EkplsRPRI2Gcn3\\\\\\\/87ZE6hnLziH0WI+hG11RfCztpDUzpZPhadOLIPDJmxti\\\\\\\/gIU1PBLJa29zZPvQ3UsQxutMYMsy5J9oI\\\\\\\/Oqm1naMOiRDNRFw5P\\\\\\\/kXZI5ZgjNv\\\\\\\/xgln0u6QGLLyRDwxIaWCbe1LdBk+Pf601b\\\\\\\/Y559k8cYn1tW2KgiueDdp9t2f+HgvMgi+q8nPwpfytICE+7TpazTcs0jbuaoZ0pa03xkktWkZerP7W7NVUoXlr9xh5p0NuasE9cd5OjEB0c2TRwWmR7fUcrflVbhy70iuxKHRRVnJqTFYkQHWBQ9Ztj6mS\\\\\\\/MhwUza2FR0fPCYVbIWzi2qPwyiTvqSzO5LUzIseePD5h4GPVJotLr7R0nmyQY9Ze67x0J0AzztrLBmGTWflg5Sm2W4VoZb0jcS4rKeVYF9TxYIDtjRfPQaOU7Hylxco0cxzpyc92h29zCeecdK1Nr1TDtY4ZuDlUOg7Z4XRehupMau0qEMwoVY12sLsJStyh7h2RaCGovApq3TNyROc9chhmd\\\\\\\/UeAaojFksDGiqmUWiKblcunItSQgA\\\\\\\/QwmxwIsBP2+01S\\\\\\\/hI4LKHDJej4vzPQHrNKhVCW+URZKHI6u1rRCx6TjWpxCp+rZe5g437MvthypFX4Rj10H7esTXF0xV22TuqNjUhkSAv6M7wibm\\\\\\\/+CaRYiQpymp+nqOtTukMNKo\\\\\\\/tKVRY95dhRv7SwRT1Bg9q\\\\\\\/5zbuiJOGU2h5ip3FMFcTmKKFHnA2eF9McgNBU7ZkiJz9IyQYyf7mHT9Seti7+wCgJWi6OH\\\\\\\/1sNrp5JGmXoHTq3eWOUrKygU8aNCAI+RwM1DjVTggWrdWtO948ydpDjBlSU9AM7hPNHmwxIoL\\\\\\\/Q0bgH\\\\\\\/sWJu\\\\\\\/5E+kH2Ince2zjkKGmgZS64FZBqIBd4BKVXurDZwmSR\\\\\\\/xhMx+XootiDvhvTQeiLMFBBJPnw9+LeFuv3tDGRjuiEZItZoQPkgP\\\\\\\/ZnAP7pno6a1bDvZEnhG3xJi04jXEfnB3Oi8GRYnbFoj+LGi9BW\\\\\\\/BjFyoY+e1z+rMkDfi96LHrf+VsiWMUnX1ZZJ5l+xBMPUUX4N4CnRibdIUTpkStUNpGSyhcouWApwGwjN+SYT+hnp\\\\\\\/lZy\\\\\\\/1mQ0FAu0rv4yXS\\\u0022,\\\u0022iv\\\u0022:\\\u0022sNNDV0vZYaFCVU0RalxH+g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002264095fe7689ea5f7d28c0b1041624e83f5f68bc137609a0bd315c965d3e2af9e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SD4j461WtDcp503OyqUhejOviK+hWvQTdghBoGiQd3k39g7B8iw9f7k+OpUoRoPej+k2iEvHCy1AdntvjOgkFzO8kI5VPN\\\\\\\/\\\\\\\/uxw7DTYH4+HgT0Oxrp4hvE+OkWV0e8vk3WYgsSIhE9B7\\\\\\\/jgQFpD7JDYSynNySn\\\\\\\/w2Jaj\\\\\\\/O9JpFujVQpI6iCzytinF9+QjSUNpkyY9UDKa9q7StxPG3MxFeVTP6ZF7Hj1v6tNUZxG0uWBvafcx+8\\\\\\\/zZ2uuSLArLEq4XSH1\\\\\\\/tr0ykXL8BskhCLe5txnnUYZXytoJdRDQewl3caojOz1VAuE\\\\\\\/rQtro4FTl4C7KNYBOsqwhrbeQVur3BpLQb0R849JDiq8XgN6XVVaVXXjDjkGJ6vDcEVPREiInLGbM9lA+XQvRumlWEt0QORNRVu6VI2i3I4JPLYAzyGAw8NgmFDMmLwj6yZ7cZaKL9POJj5TKYcFJyQFn77KyeHRmBPtPlDhZRvXULQSq4xJKw7tcHZAB6m+QlXrdLcbGSKyPouV66fVZRFh3RXcBpUVI0pwxFxuI\\\\\\\/sTCXsD7fInZCS1lkxUCJB5dPGzHka24Zsj6hqOg80OZxHbxaunTfzpdLGQ1\\\\\\\/xqYsnMS4TVKRjEHrHy7tKrBzGLGsimpCTUb9akSfTI04SWJ1UoZh00L511rnjQrOE5OtnCq0ayF+zSoeC1x56ClwzuRjSYB8GrD41htLVCyJ9ZXcYMcRXpUoX7PxvKT3ImZmsIiobzX3q7VDXFYMxKLymvtLSFNwU59Nz297kTkwadKNmZqw+F+w4jCAuLUrSIgGKbU2vlzsJ9gLbTY2tD50jrDa9ClWw9njZvdsxUOcMIO9\\\\\\\/RY2e5R50ytbvbRxhswAIQQ5sb4Vk8kQgNx3DHGAE9D5P+r3TBQSAM6MxcUQde669k6B\\\\\\\/CQnwj3ULysWZJGgTa8gtgNtvzDjqIb7af7983SpZ4pSYUtwcdme1kh\\\\\\\/g0v5aNK3tlkvRSxmczGH1FC5J6j6WlX+0ys4\\\\\\\/nR3gfN01t8aUBlGYewa4j5XtooSjUlBrsbugQGGj+ajSTeiS3q+\\\\\\\/aCmVCrP1qGrx\\\\\\\/bDL6ghox0Kz3We\\\\\\\/gQp6axZvM0p4uTRca7zqe2ch0g3GB9HU0h+hE+XoGydUUEPrY8ITEQyxuuFxP97hYTglQwj\\\\\\\/YN\\\\\\\/GTx4AGWg2Nlb\\\\\\\/Lk7v0oaeXUChhtfjkuD3jAqK3Jj7LWutoSUtA7UsvTv6fMJ+TVEmBP0He7OD2opWBWiXwul15EA50JbMubXkU7Ij4Py6YK72vZv6KAnV1SfuT4MSYASSZa+sAwT9Z6ThEC21HrTohRq2vGfw+PhL3HFvX4db0ie0KV8AuoZjql3llgQPeFZhj+lCht+iO+M7HrZ6GRNViBsUBQg7sRM\\\\\\\/L6QUQBkXKAM6EwWpJ7TcpNeizKf9GxSak20q0MMN\\\\\\\/3dqzHq05+jwavtpUkmgt8aP5bI\\\\\\\/phVPMv47A+fcwOhIHUibwyoplWNbh3\\\\\\\/RGoq11479RxqWxWcsCCrbwBwaxS1APEQ0wLP4iBNm4uI9GljJz14W+zA5mqPUksBy\\\\\\\/AkwAsGasTuBkU21UQ1exIKKWli\\\\\\\/RWZhtGTHWY9urQuzbumFOhNMHbrEQyltGgS8pYVYcs13WBU\\\\\\\/Z\\\\\\\/ZRl7ch+Yj7uLCR4TcgxoJnQO1iYR3Ghx1hmVaVuje9\\\\\\\/7Zm\\\\\\\/d4ojxmfDW8tvTiJa5kJrDNOkWJxAUbq+vsYNpKc0S14pDs5Vc43S+NjORlmI71etD\\\\\\\/fYDQOOWT9YrWFDm1A+q8L9hWPN\\\\\\\/yYZY4UBjaIIwePBHbxfESrzMxBnayqPSvzMxACrHhC2suxpoxkch2RnFiJgNXsgOVFzx\\\\\\\/rNonDERJYz6WZSloBAVCZVKAejzjKdhyAgJBqf+Nzcys5QrK1TuWY9NwdRMnpWssNPd5OVuZyBN00lIYiaiji6+9Sb9N0lJE0EmenGCTv3+wHi9Z111cqK8tCm\\\\\\\/bUgcAYVM2Xt3E4Kgz1UzXoDWfdjFKoi3PRVyo9BMUTnIXjQiJKte5mIRrXwF3NYm8GRj2zE5dhk3Z2dkjKsjEqKXu\\\u0022,\\\u0022iv\\\u0022:\\\u0022Zl5XadEokYrzT2sVXofnAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ff5d02abc6bc7c21eccc7fa193f5d9009b6efd4c94f3e6e77670598ecd6a62f2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459198 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/479" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MnLfEmjOOhPKUziu-FZAuUZW6vi0AYmpDgY7PIo0cnw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/479" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-03NDKjZ-fRQfb3EaZtW0XeBNZSAsoO5BYMYMPQLPKX8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/471" title="Modulation of the Activity of a Polycomb-Group Response Element in Drosophila by a Mutation in the Transcriptional Activator Woc" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/6/489" title="Miniature- and Multiple-Eyespot Loci in Chlamydomonas reinhardtii Define New Modulators of Eyespot Photoreception and Assembly" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459198" id="node459198" data-pisa="ggg;1/6/479" data-pisa-master="ggg;g3.111.001016" data-apath="/ggg/1/6/479.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anders Eriksson</span> and <span class="highwire-citation-author" data-delta="1">Andrea Manica</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">479-488;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001016 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459198" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anders Eriksson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Evolutionary Ecology Group, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BEriksson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eriksson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnders%2BEriksson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aje44@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">aje44@cam.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrea Manica </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Evolutionary Ecology Group, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BManica%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manica%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrea%2BManica%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/6/479" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459198" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459198/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/479.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459198" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459198/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/479.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459198" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459198/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/479.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459198" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459198/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/6/479.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459198" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459198/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Although ascertainment bias in single nucleotide polymorphisms is a well-known problem, it is generally accepted that microsatellites have mutation rates too high for bias to be a concern. Here, we analyze in detail the large set of microsatellites typed for the Human Genetic Diversity Panel (HGDP)-CEPH panel. We develop a novel framework based on rarefaction to compare heterozygosity across markers with different mutation rates. We find that, whereas di- and tri-nucleotides show similar patterns of within- and between-population heterozygosity, tetra-nucleotides are inconsistent with the other two motifs. In addition, di- and tri-nucleotides are consistent with 16 unbiased tetra-nucleotide markers, whereas the HPGP-CEPH tetra-nucleotides are significantly different. This discrepancy is due to the HGDP-CEPH tetra-nucleotides being too homogeneous across Eurasia, even after their slower mutation rate is taken into account by rarefying the other markers. The most likely explanation for this pattern is ascertainment bias. We strongly advocate the exclusion of tetra-nucleotides from future population genetics analysis of this dataset, and we argue that other microsatellite datasets should be investigated for the presence of bias using the approach outlined in this article.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/microsatellites" class="hw-term hw-article-keyword hw-article-keyword-microsatellites" rel="nofollow">microsatellites</a></li><li class="kwd"><a href="/keyword/ascertainment-bias" class="hw-term hw-article-keyword hw-article-keyword-ascertainment-bias" rel="nofollow">ascertainment bias</a></li><li class="kwd"><a href="/keyword/hgdp-ceph" class="hw-term hw-article-keyword hw-article-keyword-hgdp-ceph" rel="nofollow">HGDP-CEPH</a></li><li class="kwd"><a href="/keyword/rarefaction" class="hw-term hw-article-keyword hw-article-keyword-rarefaction" rel="nofollow">rarefaction</a></li></ul><p id="p-4">The rapidly decreasing cost of high-throughput genotyping means that large datasets with both detailed genetic coverage and wide geographic scope are finally a reality (<em>e.g.</em>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Novembre <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jakobsson <em>et al.</em> 2008</a>). These datasets provide fantastic opportunities to investigate human historical demography, and they promise to allow us to unravel the relative role of mutation, drift, and selection in shaping human diversity. However, the sets of single nucleotide polymorphisms (SNP) typed by current technologies were originally chosen to represent diversity within panels of narrow geographic scope (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Tishkoff and Verrelli 2003</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carlson <em>et al.</em> 2003</a>) and thus suffer strongly of ascertainment bias (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rogers and Jorde 1996</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kuhner <em>et al.</em> 2000</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wakeley <em>et al.</em> 2001</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akey <em>et al.</em> 2003</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bustamante <em>et al.</em> 2005</a>), making their use problematic when reconstructing the past of human populations at a worldwide level.</p><p id="p-5">A key step in assessing the importance of ascertainment bias in affecting demographic reconstructions has been the comparison with similar analyses performed on microsatellites (also known as single-tandem repeats, STR). Ascertainment schemes have been argued to have little effect on these highly polymorphic markers (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bowcock <em>et al.</em> 1994</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Rogers and Jorde 1996</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Harpending and Rogers 2000</a>), which are generally thought to provide an unbiased estimate of neutral variation; thus, they are used as a yardstick against which attempts to remove ascertainment bias from SNP are assessed. For example, when analyzing a dataset of 3024 SNPs from the Human Genetic Diversity Panel [HGDP-CEPH (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cann <em>et al.</em> 2002</a>), arguably the most comprehensive dataset on human genetic diversity covering over 1000 individuals from 52 populations], <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Conrad <em>et al.</em> (2006)</a> investigated how using differently sized windows to define haplotypes affected estimates of within-population heterozygosity and concluded that a window of 20 Kb provided the best estimates as it gave the tightest fit with similar estimates obtained from 783 STRs typed in the same populations. Later analyses of a larger number of SNPs from the same panel have kept comparing the STR data, validating a variety of approaches, such as investigations of isolation by distance using patterns of between-population diversity (<em>F</em><sub>ST</sub>) (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Jakobsson <em>et al.</em> 2008</a>), of decrease in within-population variability with increasing distance from Africa (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2008</a>), and of population clustering using STRUCTURE (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2008</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Jakobsson <em>et al.</em> 2008</a>).</p><p id="p-6">The assumption that microsatellites are free of bias has been questioned by two studies looking at the HGDP-CEPH data (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ray <em>et al.</em> 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Foll and Gaggiotti 2006</a>). <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Romero <em>et al.</em> (2009)</a> failed to find a bias when comparing the full set of 783 STRs used in the HGDP-CEPH dataset with a set of 16 unbiased STRs discovered in a multiethnic panel; however, the small number of markers in the unbiased set limits the power of their analysis. Here, we look in detail for signs of ascertainment bias in the 783 STRs used in the HGDP-CEPH panel (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rosenberg 2006</a>). Specifically, we compare results obtained using different statistical approaches applied to di-, tri-, and tetra-nucleotides separately, as we would expect the effect of bias to depend on the different mutation rates found for markers with different motif lengths. As some inconsistencies in results using different motifs might be ascribed to their different sensitivity to demographic processes (such as bottlenecks) rather than ascertainment, we develop a metric that reflects the underlying gene genealogy, effectively providing a description of diversity that is unaffected by mutation rates. More specifically, we use a rarefaction framework to generate estimates of expected heterozygosity (both within and between populations) that are rescaled to a single reference mutation rate and, thus, are directly comparable from a statistical point of view. Using this novel framework, which removes the differential effect of demographic processes on markers with different mutation rates, we demonstrate that tetra-nucleotides are inconsistent with di- and tri-nucleotides, a likely sign of ascertainment bias.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Identifying and removing inconsistencies in STR genotype data</h3><p id="p-7">We first developed a simple approach to determine the motif length of microsatellite markers from their fragment lengths, providing an objective way of removing inconsistencies from the data prior to our analysis. All fragment lengths <em>L</em> at a given locus can be written in the form <em>L</em> = <em>L<sub>m</sub> n</em> + Î³, where <em>L<sub>m</sub></em> is the length of the locusâs repeat unit, <em>n</em> is the number of repeat units (plus an unknown offset from the flanking regions), and Î³ is an integer between zero and <em>L<sub>m</sub></em> â 1 (the remainder of dividing <em>L</em> by <em>L<sub>m</sub></em>). For each locus, we are interested in obtaining a reliable estimate of <em>L<sub>m</sub></em> and Î³.</p><p id="p-8">The analysis, which is repeated for each locus, consists of three steps. First, <em>L</em><sub>m</sub> is determined by finding the shift (by 2, 3, 4, or 5 nucleotides) that maximizes the relative overlap <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-1.gif"/></noscript></span></span>, where <em>f<sub>i</sub></em> is the frequency of allele <em>i</em> in the sample. Second, we calculate the remainder Î³ from dividing each allele length <em>L</em> by <em>L<sub>m</sub></em>. Ideally, there should be a unique value of Î³ obtained from all lengths <em>L</em> for a given locus. In reality, several loci have multiple values. Loci for which the most common value of Î³ exceeded 95% of the total count were cleaned by recoding individuals with at least one allele with an unusual offset as missing data for that locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/001016SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/FigureS1.pdf">Figure S1</a>, A and B); loci with less than 95% estimates of Î³ with same value were excluded from future analysis as they deviate too strongly from the assumed model (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/FigureS1.pdf">Figure S1</a>, C and D). For all clean loci, allele lengths were converted into repeat numbers as <em>n</em> = (<em>L</em> â Î³)/<em>L<sub>m</sub></em>. A Matlab implementation of this procedure is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Statistical analysis of filtered genotype data</h3><p id="p-9">We investigated the results obtained for different motif lengths from two types of statistical approaches: hierarchical clustering with STRUCTURE v. 2.2.3, (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pritchard <em>et al.</em> 2000</a>) and pairwise population differentiation estimated as <em>F</em><sub>ST</sub> [calculated in Matlab v7.11 from the expected within- and between-population heterozygosities weighted according to sample size (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Nei 1978</a>)]. We would expect both clustering and <em>F</em><sub>ST</sub> patterns to be relatively robust to differences in mutation rates between the markers with different motif lengths (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Haasl and Payseur 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Mutation rarefaction</h3><p id="p-10">Demographic processes such as bottlenecks might be sampled differently by markers with different mutation rates. Even the normalization of between- by within-population variability used when computing pairwise <em>F</em><sub>ST</sub> will not fully account for this effect. To be able to statistically compare markers with different mutation rates, we would ideally need a metric that reflects the underlying gene genealogy, effectively providing a description of diversity that is unaffected by mutation rates. To this end, we have developed a rarefaction framework to generate estimates of expected heterozygosity (both within and between populations) that are rescaled to a single reference mutation rate and, thus, are directly comparable from a statistical point of view.</p><p id="p-11">Heterozygosity, defined as the probability that two alleles sampled at random are identical, is a frequently used measure of genetic diversity. It is most commonly computed to estimate within-population diversity, but it can be equally applied to describe between-population differentiation. However, heterozygosity estimates from markers with different mutation rates cannot be compared directly and do not obey a simple scaling rule; how heterozygosity scales with mutation rate depends on the detailed shape of the underlying gene genealogy (and thus on the distribution of time to most recent common ancestor, TMRCA). To solve this problem, we analyze the more general question of how the distribution <em>p</em>(Î) of pairwise differences in repeat count, Î, depends on the mutation rate.</p><p id="p-12">The principle of our method is best understood by considering a single microsatellite locus with mutation rate Î¼. What would <em>p</em>(Î) be if we had a lower mutation rate, Î¼â²? If we knew the underlying gene genealogy of locus in a sample, and the location of mutations on the gene genealogy, we could rarefy the mutations to the new mutation rate by randomly removing mutations such that on average a fraction Î¼â²/Î¼ of the mutations remains, and calculate the new <em>p</em>(Î) from the resulting genetic variation. This principle is illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>, and the effect of mutation rarefaction on the distribution <em>p</em>(Î) is illustrated in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/479/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of the mutation rarefaction principle for a given locus. (A) The gene genealogy of the sample (solid lines) and the location of the mutations since the most recent common ancestor (red circles). (B) Randomly removing a fraction of the mutations (open circles) tends to reduce the heterozygosity observed the sample. However, the extent to which this happens depends on the location of the deleted mutations in the gene genealogy. (C) The effect of rarefaction on the distribution of difference D in repeat count between individuals, p(D), with MRCA 1000 generations ago. The blue and red curves show the distributions for mutation rates m1 = 0.01 and m2 = 0.006, respectively, and the red dots show the result of rarefaction of the blue distribution by a factor m2/m1 = 0.6 (using Equation 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1085988595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of the mutation rarefaction principle for a given locus. (A) The gene genealogy of the sample (solid lines) and the location of the mutations since the most recent common ancestor (red circles). (B) Randomly removing a fraction of the mutations (open circles) tends to reduce the heterozygosity observed the sample. However, the extent to which this happens depends on the location of the deleted mutations in the gene genealogy. (C) The effect of rarefaction on the distribution of difference &lt;em&gt;D&lt;/em&gt; in repeat count between individuals, &lt;em&gt;p&lt;/em&gt;(&lt;em&gt;D&lt;/em&gt;), with MRCA 1000 generations ago. The blue and red curves show the distributions for mutation rates &lt;em&gt;m&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 0.01 and &lt;em&gt;m&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.006, respectively, and the red dots show the result of rarefaction of the blue distribution by a factor &lt;em&gt;m&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;m&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 0.6 (using Equation 2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/479/F1.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/479/F1.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/479/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/479/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460894" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-13" class="first-child">Illustration of the mutation rarefaction principle for a given locus. (A) The gene genealogy of the sample (solid lines) and the location of the mutations since the most recent common ancestor (red circles). (B) Randomly removing a fraction of the mutations (open circles) tends to reduce the heterozygosity observed the sample. However, the extent to which this happens depends on the location of the deleted mutations in the gene genealogy. (C) The effect of rarefaction on the distribution of difference <em>D</em> in repeat count between individuals, <em>p</em>(<em>D</em>), with MRCA 1000 generations ago. The blue and red curves show the distributions for mutation rates <em>m</em><sub>1</sub> = 0.01 and <em>m</em><sub>2</sub> = 0.006, respectively, and the red dots show the result of rarefaction of the blue distribution by a factor <em>m</em><sub>2</sub>/<em>m</em><sub>1</sub> = 0.6 (using <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation 2</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">In reality, we can observe p(Î), but we do not know the underlying gene genealogy, how many mutations occurred, or where they are placed on the gene genealogy. Nevertheless, for large samples, it is possible to estimate the rarefied <em>p</em>(Î) from the observed distribution by implicitly removing a given fraction of the underlying mutations. Consider the characteristic function of the distribution <em>p</em>(Î), defined as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-2.gif"/></noscript></span></span></p><p id="p-15">Using the standard Stepwise Mutation Model [SMM, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kimura and Ohta (1975</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">1978</a>)], we can express this function in terms of the (unknown) distribution of TMRCA for pairs of individuals. In the SMM, each mutation leads to the addition or removal of single repeat units, with equal probability. Assuming a mutation occurs independently with rate Î¼ per generation, the difference Î in repeat count for a given pair of individuals with <em>t</em> generations to the MRCA can be written as the sum of 2<em>t</em> independent random variables, which are â1, 0, or 1 with probability Î¼/2, 1 â Î¼, or Î¼/2, respectively. As the characteristic function of the sum of two independent variables is the product of their characteristic functions, and each variable here has the characteristic function <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-3.gif"/></noscript></span></span>, the characteristic function of Î is <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-4.gif"/></noscript></span></span>. For a sample of more than two individuals, we average the characteristic function over the distribution <em>g<sub>t</sub></em> of pairwise time to MRCA in the sample:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-5.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-6.gif"/></noscript></span></span> is a generating function for <em>g<sub>t</sub></em>. Solving for Ï as a function of <em>z</em>, one obtains<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-7.gif"/></noscript></span></span></p><p id="p-16">As the underlying distribution of time to the MRCA is assumed to be independent of the mutation rate (because of neutrality), this expression holds for any mutation rate. Hence, for mutation rate Î¼â², we have<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-8.gif"/></noscript></span></span></p><p id="p-17">Writing <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-9.gif"/></noscript></span></span> in terms of Î gives an explicit expression for the rarefied distribution of Î in terms of the original distribution:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-10.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span></p><p id="p-18">From the characteristic function, we can obtain the probability of observing difference Î in repeat counts between individuals, <em>p</em><sub>rarefied</sub>(Î), in the rarefied sample by taking an inverse Fourier transform:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-11.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span></p><p id="p-19"><a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a> illustrates the effect of rarefaction on <em>p</em>(Î). Finally, using this relation, we obtain the following scaling rule for heterozygosity:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-12.gif"/></noscript></span></span></p><p id="p-20">Mathematically, this method only works when going from larger to smaller mutation rates. This makes intuitive sense, because for Î¼â² &lt; Î¼, the method is removing information, but going in the opposite direction would correspond to increasing the amount of information in the sample. This type of extrapolation is so unstable that it is useless for all practical purposes.</p><p id="p-21">Finally, as a sanity check, we verify that our scaling method obey the scaling relation <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-13.gif"/></noscript></span></span>, where <em>V</em> is the variance of the counts of repeat units, which is the standard method in SMM theory for calibrating the relative mutation rate of two markers [<em>e.g.</em>, see <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhivotovsky <em>et al.</em> (2003)</a> and references therein]. It is straightforward to show that this relation holds for the rarefaction method, using the Taylor expansion of <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation 1</a> around Ï = 0.</p></div><div id="sec-5" class="subsection"><h3>Estimating mutation rates from mutation rarefaction</h3><p id="p-22">For the markers used in the HGDP-CEPH panel, only the mutation rate for di-nucleotides has been estimated directly from pedigree data [Î¼<sub>2</sub> = 1.52 Ã 10<sup>â3</sup>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhivotovsky <em>et al.</em> (2000)</a>], whereas mutation rates for tri- and tetra-nucleotides have to be estimated indirectly (we do not consider penta-nucleotides, as there are too few of them). As the scaling rule for heterozygosity depends only on the ratio of mutation rates (<em>i.e.</em>, on the fraction of mutations kept in the rarefaction process), we can use it to find the best estimates of mutation rates in tri- and tetra- nucleotides.</p><p id="p-23">For example, if we want to estimate the mutation rate for tri-nucleotides, we can find the optimal rarefaction factor, <em>k</em><sub>23</sub> (<em>i.e.</em>, Î¼<sub>3</sub>/Î¼<sub>2</sub>), that brings the within- and between-population heterozygosities of di-nucleotides closest to those of tri-nucleotides. The mutation rate estimate of tri-nucleotides is then Î¼<sub>3</sub> = <em>k</em><sub>23</sub>Î¼<sub>2</sub>. An additional estimate for tri-nucleotides can be obtained using tetra-nucleotides as a reference for scaling the mutation rate of di- and tri-nucleotides. Given the optimal rarefaction factors <em>k</em><sub>24</sub> (<em>i.e.</em>, Î¼<sub>4</sub>/Î¼<sub>2</sub>) and <em>k</em><sub>34</sub> (<em>i.e.</em>, Î¼<sub>4</sub>/Î¼<sub>3</sub>), we estimate the mutation rate of tri-nucleotides as Î¼<sub>3</sub> = Î¼<sub>2</sub><em>k</em><sub>24</sub>/<em>k</em><sub>34</sub>.</p></div><div id="sec-6" class="subsection"><h3>Trends in within- and between-population heterozygosity</h3><p id="p-24">To test whether the matrix of within- and between population heterozygosities of motifs with different lengths are consistent after scaling, we separately scaled di- and tri-nucleotide markers to match the HGDP-CEPH tetra-nucleotide markers (by minimizing mean-square difference of the heterozygosity matrices). To test whether the one-to-one relation can be significantly rejected, we estimated 95% confidence intervals of the regression lines (di- and tri-nucleotide heterozygosities <em>versus</em> tetra-nucleotide heterozygosities) from 10,000 bootstrap samples of the data using a Matlab script. We then repeated the same analysis comparing di-, tri- and tetra-nucleotide heterozygosities to the heterozygosities of the unbiased tetra-nucleotide markers, after scaling to this set of markers.</p><p id="p-25">Finally, we analyzed the dependence of within-population expected heterozygosity and allelic richness [calculated using ADZE v.1.0 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Szpiech <em>et al.</em> 2008</a>)] on the distance from sub-Saharan Africa [calculated as in (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Manica <em>et al.</em> 2005</a>)] for di-, tri-, and tetra-nucleotide markers. Especially, we analyzed the effect of subsetting the markers and scaling of heterozygosities on the cline in heterozygosity away from sub-Saharan Africa.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Identifying and removing inconsistencies in STR genotype data</h3><p id="p-26">Despite our strict demands for accepting a locus, our approach gave 749 markers (out of 783) that could be used for analysis, subdivided into 54 di-, 166 tri-, 519 tetra-, and 10 penta-nucleotides (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/TableS1.pdf">Table S1</a>). The 10 penta-nucleotide markers were dropped from further analysis because they were deemed too few. The cleaned data from the 749 markers [as well a cleaned version of the 16 unbiased markers from <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Romero <em>et al.</em> (2009)</a>] are available (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/FileS2.zip">File S2</a>). For those markers that had been previously classified based on the reference Human Genome sequence (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pemberton <em>et al.</em> 2009</a>), Pemberton <em>et al.</em> and our classifications were in agreement, with the exception of five markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/TableS2.pdf">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/FileS3.zip">File S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/FileS4.xls">File S4</a>).</p></div><div id="sec-9" class="subsection"><h3>The effect of motif length on STRUCTURE clusters and <em>F</em><sub>ST</sub> patterns</h3><p id="p-27">Analysis of di-, tri- and tetra-nucleotides with STRUCTURE yielded the same clusters irrespective of which motif length was used (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). However, when comparing <em>F</em><sub>ST</sub> patterns for these three categories of markers, we found a strong effect of marker length for <em>F</em><sub>ST</sub> patterns: di- and tri-nucleotides were in very good correspondence (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>), but tetra-nucleotides showed much lower population differentiation than the shorter motifs (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>). To determine whether this discrepancy can be attributed to demographic processes being sampled differently by markers with dissimilar mutation rates, we apply our rarefaction approach to compute statistically comparable, rarefied heterozygosity estimates.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/479/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Five-cluster STRUCTURE analysis of the HGDP-CEPH data, with populations ordered according to increasing distance from sub-Saharan Africa. The top panel shows the clusters of the original HGDP-CEPH dataset. The remaining panels show, from top to bottom, the respective clusters based on the di-, tri-, and tetra-nucleotides in the cleaned HGDP-CEPH dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1085988595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Five-cluster STRUCTURE analysis of the HGDP-CEPH data, with populations ordered according to increasing distance from sub-Saharan Africa. The top panel shows the clusters of the original HGDP-CEPH dataset. The remaining panels show, from top to bottom, the respective clusters based on the di-, tri-, and tetra-nucleotides in the cleaned HGDP-CEPH dataset.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/479/F2.medium.gif" width="440" height="392"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/479/F2.medium.gif" width="440" height="392"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/479/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/479/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460960" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-28" class="first-child">Five-cluster STRUCTURE analysis of the HGDP-CEPH data, with populations ordered according to increasing distance from sub-Saharan Africa. The top panel shows the clusters of the original HGDP-CEPH dataset. The remaining panels show, from top to bottom, the respective clusters based on the di-, tri-, and tetra-nucleotides in the cleaned HGDP-CEPH dataset.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/479/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FST of cleaned HGDP-CEPH markers with different repeat motif lengths. The three panels show pairwise comparisons between FST values: (A) di- versus tri-nucleotides, (B) tri- versus tetra-nucleotides, and (C) di- versus tetra-nucleotides." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1085988595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; of cleaned HGDP-CEPH markers with different repeat motif lengths. The three panels show pairwise comparisons between &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; values: (A) di- &lt;em&gt;versus&lt;/em&gt; tri-nucleotides, (B) tri- &lt;em&gt;versus&lt;/em&gt; tetra-nucleotides, and (C) di- &lt;em&gt;versus&lt;/em&gt; tetra-nucleotides.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/479/F3.medium.gif" width="175" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/479/F3.medium.gif" width="175" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/479/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/479/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460913" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-29" class="first-child"><em>F</em><sub>ST</sub> of cleaned HGDP-CEPH markers with different repeat motif lengths. The three panels show pairwise comparisons between <em>F</em><sub>ST</sub> values: (A) di- <em>versus</em> tri-nucleotides, (B) tri- <em>versus</em> tetra-nucleotides, and (C) di- <em>versus</em> tetra-nucleotides.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Mutation rates from mutation rarefaction</h3><p id="p-30">To rarefy estimates of within- and between-population expected heterozygosity, we first use our scaling procedure to find the relative mutation rates of the different groups of markers (see <em>Materials and Methods</em>). <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the result of estimating the mutation rates of di-, tri-, and tetra-nucleotides, as well as a set of 16 unbiased tetra-nucleotides from <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Romero <em>et al.</em> (2009)</a>. Each row corresponds to using a given reference set of markers, and the columns contain the mutation estimates for different sets. In addition to fitting the within- and between-population heterozygosities, <em>p</em>(0), we show mutation rates calculated by fitting the distribution of count differences <em>p</em>(Î) and by scaling the variances (the approach used in previous articles). We find that the mutation rates obtained by rarefaction analysis are insensitive to which markers are used as reference and whether the fitting was based on the full distribution of differences, <em>p</em>(Î), or on only the heterozygosities, <em>p</em>(0). The scaling of variance, however, yields a slightly lower mutation rate for the tri-nucleotide loci (0.65 Ã 10<sup>â3</sup> <em>versus</em> 0.71 Ã 10<sup>â3</sup> â 0.77 Ã 10<sup>â3</sup>). There is also a mismatch for the mutation rate for the unbiased tetra-nucleotides (0.46 Ã 10<sup>â3</sup> <em>versus</em> 0.23 Ã 10<sup>â3</sup>), but the small number of markers makes these estimates somewhat unreliable. For comparison, we also show the mutation rates found by <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Zhivotovsky <em>et al.</em> (2003)</a>, which are based on a subset of the CEPH-HGDP markers obtained by removing markers with extreme variances.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460872/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460872/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1â</span> <span class="caption-title">The mutation rates of di-, tri- and tetra-nucleotide markers in the cleaned data and the unbiased tetra-nucleotide markers, when scaling down to different reference markers</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Trends in within- and between-population heterozygosity</h3><p id="p-33"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows pairwise comparisons of the rarefied within- and between-population average heterozygosities of HGDP-CEPH markers after fitting to the tetra-nucleotide markers (left panels) or unbiased tetra-nucleotide markers (right panels, together with lines indicating the one-to-one relationships and 95% confidence intervals of the regression lines. The di-, tri- and unbiased tetra-nucleotides are in good agreement (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, A, B, and D</a>), whereas the HGDP-CEPH tetra-nucleotides are significantly different from all other groups (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, C, E, and F</a>), consistent with patterns in <em>F</em><sub>ST</sub> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/479/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise comparisons of the scaled within- and between-population expected heterozygosities (red and blue dots, respectively). HGDP-CEPH markers are scaled to match the tetra-nucleotide markers (A, C, E) or unbiased tetra-nucleotide markers (B, D, F). In all panels, the solid black lines show the one-to-one relations, and the dashed black lines show the 95% confidence intervals of regression lines based on 10,000 bootstrap samples of the data. A, C, E: Comparisons of heterozygosities of di-, tri-, and tetra-nucleotide markers. B, D, F: comparisons of di-, tri- and tetra-nucleotide to the unbiased markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1085988595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pairwise comparisons of the scaled within- and between-population expected heterozygosities (red and blue dots, respectively). HGDP-CEPH markers are scaled to match the tetra-nucleotide markers (A, C, E) or unbiased tetra-nucleotide markers (B, D, F). In all panels, the solid black lines show the one-to-one relations, and the dashed black lines show the 95% confidence intervals of regression lines based on 10,000 bootstrap samples of the data. A, C, E: Comparisons of heterozygosities of di-, tri-, and tetra-nucleotide markers. B, D, F: comparisons of di-, tri- and tetra-nucleotide to the unbiased markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/479/F4.medium.gif" width="345" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/479/F4.medium.gif" width="345" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/479/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/479/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460900" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4â</span> <p id="p-34" class="first-child">Pairwise comparisons of the scaled within- and between-population expected heterozygosities (red and blue dots, respectively). HGDP-CEPH markers are scaled to match the tetra-nucleotide markers (A, C, E) or unbiased tetra-nucleotide markers (B, D, F). In all panels, the solid black lines show the one-to-one relations, and the dashed black lines show the 95% confidence intervals of regression lines based on 10,000 bootstrap samples of the data. A, C, E: Comparisons of heterozygosities of di-, tri-, and tetra-nucleotide markers. B, D, F: comparisons of di-, tri- and tetra-nucleotide to the unbiased markers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">This discrepancy seems to be due to populations in Eurasia being relatively homogeneous for tetra-nucleotides. This effect can be visualized by plotting the decline in within-population genetic diversity with increasing distance from sub-Saharan Africa [a pattern attributed to the effect of sequential founder events during the spread out of Africa by anatomically modern humans (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Prugnolle <em>et al.</em> 2005</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ramachandran <em>et al.</em> 2005</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Manica <em>et al.</em> 2007</a>)]. Irrespective of whether we look at raw estimates of expected within-population heterozygosity (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>) or at rarefied estimates using the approach outlined in this article (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>), it is clear that, whereas di- and tri-nucleotides show a smooth decline of heterozygosity with increasing distance from Africa, the heterozygosity of tetra-nucleotides is much flatter across Eurasia (corresponding to approximately 5,000â15,000 km in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>), with a steep decline evident only once we reach the Americas. A similar pattern holds for allelic richness (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/479/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Within-population expected heterozygosity as a function of distance from sub-Saharan Africa for di-, tri-, and tetra-nucleotides separately (blue, green, red symbols, respectively). Panels A and B show the heterozygosity before and after scaling to match the full tetra-nucleotide H-matrix, respectively (see Table 1 for the relative mutation rates)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1085988595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Within-population expected heterozygosity as a function of distance from sub-Saharan Africa for di-, tri-, and tetra-nucleotides separately (blue, green, red symbols, respectively). Panels A and B show the heterozygosity before and after scaling to match the full tetra-nucleotide &lt;em&gt;H&lt;/em&gt;-matrix, respectively (see Table 1 for the relative mutation rates).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/479/F5.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/479/F5.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/479/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/479/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460880" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5â</span> <p id="p-36" class="first-child">Within-population expected heterozygosity as a function of distance from sub-Saharan Africa for di-, tri-, and tetra-nucleotides separately (blue, green, red symbols, respectively). Panels A and B show the heterozygosity before and after scaling to match the full tetra-nucleotide <em>H</em>-matrix, respectively (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> for the relative mutation rates).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">To what extend do tetra-nucleotides affect previous estimates of relationship between within-population heterozygosity and distance from sub-Saharan Africa? To answer this question, we estimated the relationship for three different averages: (<em>i</em>) of the complete set of markers for the HGDP-CEPH panel (<em>i.e.</em>, pooling all di-, tri- and tetra-nucleotides and ignoring differences in mutation rates, as is commonly done when estimating within-population heterozygosity); (<em>ii</em>) of only the subset of di- and tri-nucleotides (<em>i.e.</em>, pooling only the markers that were consistent with each other, again ignoring mutation rates); and (<em>iii</em>) of an estimate of the latter after rarefying the di-nucleotide to the mutation rate of tri-nucleotides (<em>i.e.</em>, the correct way of combining markers with different mutation rates). As can be seen in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, the slope estimated for the full set of markers (â6.792Â·10<sup>â6</sup> Â± 4.277Â·10<sup>â7</sup>) is significantly shallower than that estimated for the subset of di- and tri-nucleotides (â8.866Â·10<sup>â6</sup> Â± 4.584Â·10<sup>â7</sup>; F<sub>1,98</sub> = 10.94, <em>P</em> = 0.001) and for the rarefied dataset (â8.656Â·10<sup>â6</sup> Â± 4.498Â·10<sup>â7</sup>; F<sub>1,98</sub> = 9.02, <em>P</em> = 0.003). The is no difference in the slope estimates for subsets of di- and tri-nucleotides and for the rarefied dataset (F<sub>1,98</sub> = 0.11, <em>P</em> = 0.744), but as one would expect, the intercept is significantly lower for the latter (7.820Â·10<sup>â01</sup> Â± 5.929Â·10<sup>â3</sup> <em>versus</em> 7.958Â·10<sup>â1</sup> Â± 6.043Â·10<sup>â3</sup>; F<sub>1,99</sub> = 6.92, <em>P</em> = 0.010).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/479/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average within-population heterozygosity for the CEPH-HGDP populations, as a function of distance from sub-Saharan Africa. Orange: Average of all cleaned markers. Red: Average of di- and tri-nucleotides. Black: Average of di- and tri-nucleotides, with di-nucleotides scaled down to the tri-nucleotides. Inset: Regression lines for the three colors. For reference, the scaled data are shown as gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1085988595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average within-population heterozygosity for the CEPH-HGDP populations, as a function of distance from sub-Saharan Africa. Orange: Average of all cleaned markers. Red: Average of di- and tri-nucleotides. Black: Average of di- and tri-nucleotides, with di-nucleotides scaled down to the tri-nucleotides. Inset: Regression lines for the three colors. For reference, the scaled data are shown as gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/479/F6.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/479/F6.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/479/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/479/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460991" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6â</span> <p id="p-38" class="first-child">Average within-population heterozygosity for the CEPH-HGDP populations, as a function of distance from sub-Saharan Africa. Orange: Average of all cleaned markers. Red: Average of di- and tri-nucleotides. Black: Average of di- and tri-nucleotides, with di-nucleotides scaled down to the tri-nucleotides. Inset: Regression lines for the three colors. For reference, the scaled data are shown as gray.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-39">Our analyses clearly illustrates that, whereas di- and tri-nucleotides show patterns that are highly consistent with each other, the tetra-nucleotides originally typed for the HGDP-CEPH panel are inconsistent as they are too homogeneous in their individual distributions across Eurasia. This discrepancy cannot have arisen from demographic processes affecting markers with different mutation rates to different extents: rarefying diversity to the slowest mutation rate should always give equivalent estimates, as the underlying distribution of gene genealogies is the same irrespective of the marker of choice. Although this is true for di- and tri-nucleotides, for which both within and between-population heterozygosity estimates are equivalent after rarefaction, tetra-nucleotides remain inconsistent even when we consider rarefied estimates of heterozygosity.</p><p id="p-40">The most likely explanation for this discrepancy seems to be some form of bias in the choice of markers. We know that tetra-nucleotide markers included in the HGDP-CEPH panel show significantly higher diversity than the unbiased markers developed by <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Romero <em>et al.</em> (2009)</a>. The patterns observed in the data would be consistent with a scenario where the HGDP-CEPH tetra-nucleotides were selected for high diversity from a panel mostly composed by Eurasians. It is difficult to say whether di- and tri-nucleotides were selected from a broader panel (and thus there was little or no bias in their selection process) or whether they escaped the effect of bias because of their higher mutation rates (<em>i.e.</em>, most markers have a diversity well above the threshold for selection). In any case, the discrepancy between different microsatellites of different motif lengths indicates that microsatellites are probably not as immune to ascertainment bias as suggested in the past (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Rogers and Jorde 1996</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bowcock <em>et al.</em> 1994</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Harpending and Rogers 2000</a>).</p><p id="p-41">It is clear that the choice of metrics extracted from the markers is important, as some are more affected by bias than others are. In contrast to the heterozygosity-based measures, we found STRUCTURE clusters to be relatively robust to the bias affecting tetra-nucleotides. This result is consistent with simulation studies of microsatellites and STRs, showing that STRUCTURE is relatively robust against ascertainment bias (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Haasl and Payseur 2011</a>).</p><p id="p-42">Average heterozygosity is a commonly computed quantity, often used to parameterize population genetic models (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Zhivotovsky <em>et al.</em> 2003</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Liu <em>et al.</em> 2006</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Deshpande <em>et al.</em> 2009</a>). However, when the markers have heterogeneous mutation rates, this quantity does not correspond to the heterozygosity of the average mutation rate (or to any other single mutation rate). An important contribution of our article is the development of a rarefaction approach to combine markers with different mutation rates in heterozygosity-based measures corresponding to a single mutation rate. This approach allows all markers to carry equal weight in determining the overall metric, and is a much more accurate approach than simply averaging estimates obtained for different types of markers. Our approach is model free with respect to the underlying demographic processes (<em>i.e.</em>, we do not explicitly reconstruct bottlenecks, migrations, etc.). It is important to note that to combine heterozygosities, we do not need to know the mutation rate of any of the types but only have an <em>a priori</em> classification scheme that allows us to group markers with relatively similar mutation rates. However, if the mutation rate of one of the marker types is known, rarefaction yields the mutation rate of the combined heterozygosities. This way of estimating mutation rates has the advantage to be relatively robust to the inclusion of markers with extreme values, a known weakness of the commonly used approach of scaling the variances (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Zhivotovsky <em>et al.</em> 2003</a>).</p><p id="p-43">Although we have not performed a formal investigation of the number of makers needed for our approach to be stable, we estimate our rarefaction method to require at least 10, and ideally over 20 markers in each group to avoid artifacts while rescaling. In the case of microsatellites, we obviously need markers to be clean enough to comply with the SMM framework. We provide several tools to check such compliance.</p><p id="p-44">The SMM model is the simplest and most widely used model of microsatellite evolution, but it may seem overly simplistic given the heterogeneous nature of human microsatellites (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chakraborty <em>et al.</em> 1997</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ellegren 2004</a>). For example, the fact that the SMM model assumes symmetric mutation rates may seem to limit its validity for our method. However, our method uses only the pairwise difference in number of repeats between individuals and is, therefore, independent of any constant, directional bias in the mutations (toward higher or lower repeat counts). It is also possible to use our method with more complex models. In the appendix, we show how our method can be adapted to use mutation models with arbitrary length-dependent increment and decrement rates and for models in which multiple repeat units can sometimes be added or removed. The main conclusion from analyzing these models is that the scaling relation between rarefied and original distribution of repeat counts depends only on the ratio of the two mutation rates, and they can be used in the same way as the simple SMM model. Although in many cases, it is not possible to write the scaling relation in a closed form, it is straightforward to compute it numerically. For the case of constraining the SMM model to have repeat counts between one and <em>L</em>, we show in the appendix that the scaling relation is the same as in the standard SMM model.</p><p id="p-45">What are the implications of the bias in HGDP-CEPH tetra-nucleotides for studies that have used these markers to study the effect of SNP ascertainment bias? <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Conrad <em>et al.</em> (2006)</a> plotted within-population haplotype heterozygosity against microsatellite heterozygosity for different haplotype lengths. Although SNPs were found to have little relation to microsatellites, they found that points fall on a curve for haplotypes at least 20 kb long (see <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> in <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Conrad <em>et al.</em> 2006</a>). If we were to remove the tetra-nucleotide markers from this plot, the American data points would be shifted strongly left, the Asian data points slightly so, and the remaining populations would be essentially unchanged (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Although the resulting pattern would be slightly less linear, the points would still fall on a curve (because heterozygosity of microsatellites and SNPs depend differently on the underlying gene genealogies, the relation between the two is not linear in general). The conclusion that sufficiently long haplotypes are effectively free of ascertainment bias is not strongly affected by the bias in the tetra-nucleotides. <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sun <em>et al.</em> (2009)</a> found a linear relation between microsatellite average-squared difference (ASD) and sequence divergence for individuals from different populations. The bias observed in the tetra-nucleotides has little effect on ASD (unpublished) and, therefore, does not change any of their conclusions in that article.</p><p id="p-46">For models describing the expansion of anatomically modern humans out of Africa that have been parameterized using the average heterozygosity of all markers (<em>e.g.</em>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Ramachandran <em>et al.</em> 2005</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Liu <em>et al.</em> 2006</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Deshpande <em>et al.</em> 2009</a>), we would not expect the biases discussed in this article to put their main conclusions into doubt. On the other hand, we would expect the actual estimates of most parameters to be affected. This is especially true of the timing of the expansion out of Africa, which is usually the most interesting quantity. In addition, the relatively flat <em>F</em><sub>ST</sub> pattern from tetra-nucleotide markers (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), as well as the weaker cline in within-population expected heterozygosity (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>), would cause such models to overestimate gene flow (migration) and underestimate population bottlenecks during the expansion; the magnitude of these effects depends on the details of the models, but it is approximately proportional to the relative effect of the bias on <em>F</em><sub>ST</sub> and the trend in expected heterozygosity.</p><p id="p-47">The HGDP-CEPH panel has been used as a blueprint for the development of large datasets based on microsatellites, and subsets of the markers used in the HGDP-CEPH have been adopted for investigating genetic diversity in Africa (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tishkoff <em>et al.</em> 2009</a>), India (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rosenberg <em>et al.</em> 2006</a>), Oceania (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Friedlaender <em>et al.</em> 2008</a>), and the Americas (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wang <em>et al.</em> 2007</a>). A simple precaution for future users of these datasets would be to limit their analysis to di- and tri-nucleotides. Although tetra-nucleotides constitute the majority of the markers in the HGDP-CEPH panel, di- and tri-nucleotides together still provide 220 markers, a very large number in terms of population genetics. Furthermore, our analyses above show that these markers are enough to perform all the type of analytical approaches usually employed on this type of data and, indeed, drive the results for measures such as <em>H</em> and <em>F</em><sub>ST</sub> when the data are combined. However, as several of these datasets concentrate on geographic areas at the boundaries of the coverage from the HGDP-CEPH panel, we would recommend following the same steps adopted in this article to investigate the presence of biases and make sure that they are fully removed.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">This work was supported by a Philip Leverhume Award and Biotechnology and Biological Sciences Research Council Grant BB/H005854/1.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section other1" id="sec-13"><h3>The SMM Model with Arbitrary Jump Distributions</h3><p id="p-49">Consider an extension of the standard SMM model to arbitrary jump distributions, such that at each mutation the probability of a change of <em>k</em> repeats is <em>q<sub>k</sub></em> (with <em>q</em><sub>0</sub> = 0). The standard SMM model corresponds to taking <em>q</em><sub>1</sub>= <em>q</em><sub>-1</sub>= 1/2, and <em>q<sub>k</sub></em> = 0 otherwise. Using the characteristic function<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-14.gif"/></noscript></span></span>we can write the characteristic function <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-15.gif"/></noscript></span></span> for the difference Î in repeat count between two alleles as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-16.gif"/></noscript></span></span>where <em>g</em>(<em>z</em>) is the generating function for the distribution of the TMRCA. Expanding to the first order in Âµ, we obtain<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-17.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-18.gif"/></noscript></span></span></p><p id="p-50">Hence, for any small mutation rate Âµ,<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-19.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-20.gif"/></noscript></span></span> denotes the inverse function of <em>G</em>. Thus, for mutation rate Âµâ², we express <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-21.gif"/></noscript></span></span> in terms of <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-22.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-23.gif"/></noscript></span></span></p><p id="p-51">Finally, we can use the inverse Fourier transform to obtain the distribution of Î at mutation rate Âµâ² from <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-24.gif"/></noscript></span></span>. As for the standard SMM model, the scaling relation depends only on ratio of the two mutation rates, which means that our scaling method works equally well for these models.</p><p id="p-52">We also note that only the component of the jump distribution symmetric around <em>k</em> = 0 enters into the final expression. This is a consequence of the fact that we consider only differences in repeat count, and implies that the scaling relation is independent of any systematic increase or decrease in the repeat count.</p><p id="p-53">If the jump distribution is fully symmetric, the scaling relation is exact. Otherwise, there is a small correction to the scaling relation, but it is still possible to express <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-25.gif"/></noscript></span></span> in terms of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-26.gif"/></noscript></span></span>. However, when the mutation rates are small the correction is negligible.</p></div><div class="section other2" id="sec-14"><h3>The <em>L</em>-allele Mutation Model</h3><p id="p-54">In this section, we consider the effect of limiting the SMM models to repeat counts in the range [1, <em>L</em>] (shifting the range with an arbitrary amount makes no difference to the analysis). At the ends of the interval, we introduce reflecting boundary conditions such that the stationary distribution is uniform over the range. For alleles with repeat count in the range [2, <em>L</em>-1], mutations occur independently each generation with probability Âµ, with equal probability of gaining or losing a single repeat unit. At repeat count one, only mutations leading to an increment are accepted, and the repeat count <em>L</em> only decrements are accepted. Starting from a distribution <em>x</em> at time zero, the distribution of <em>p</em> of alleles <em>t</em> generations later can be written on matrix form as<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-27.gif"/></noscript></span></span>where the matrix M has elements given by <em>M<sub>i,i</sub></em> = 1, <em>M<sub>i</sub></em><sub>-1,</sub><em><sub>i</sub></em> = <em>M<sub>i</sub></em><sub>+1,</sub><em><sub>i</sub></em> = â1/2, and <em>M<sub>i</sub></em><sub>,</sub><em><sub>j</sub></em> = 0 otherwise, except at the boundaries where <em>M</em><sub>1,1</sub> = <em>M<sub>L</sub></em><sub>,</sub><em><sub>L</sub></em> = 1/2. Correspondingly, the joint distribution <em>P<sub>ij</sub></em> for two individuals becomes (given that <em>M</em> is symmetric)<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-28.gif"/></noscript></span> <span class="disp-formula-label">(A1)</span></span>where <em>D</em> denotes the diagonal matrix with diagonal elements equal to <em>x</em>.</p><p id="p-55">Because <em>M</em> is symmetric, all eigenvectors and eigenvalues are real-valued, and the eigenvectors form an orthonormal basis. Expanding <em>M</em> in the eigenvectors <em>u</em><sub>Î±</sub> and eigenvalues <em>Î»</em><sub>Î±</sub>, we have<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-29.gif"/></noscript></span></span>where the superscript <em><span class="underline">T</span></em> denotes transpose. Inserting this in the equation for <em>P</em> above, and multiplying from the left and right with <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-31.gif"/></noscript></span></span>, respectively, gives<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-32.gif"/></noscript></span></span></p><p id="p-56">If we now assume that the distribution of alleles in the MRCA of the two individuals is the stationary distribution (<em>i.e.</em>, the locus is old enough), this equation simplifies to<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-33.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-34.gif"/></noscript></span></span> is Kroeneckerâs delta, which is equal to one if <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-35.gif"/></noscript></span></span> and is zero otherwise, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-36.gif"/></noscript></span></span> is a diagonal matrix. If we now consider an arbitrary distribution of TMRCA with generating function <em>g</em>(<em>z</em>), as in the previous section, we obtain<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-37.gif"/></noscript></span></span></p><p id="p-57">Again, the function <em>g</em>(<em>z</em>) is independent of the mutation rate. Thus, this equation suggests a scaling relation <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-38.gif"/></noscript></span></span> between the eigenvalues at mutation rate <em>Âµ</em> and <em>Âµ</em>â for the same underlying distribution of TMRCA. The joint distribution of alleles for mutation rate Âµâ², <em>P</em>(Âµâ²), can then written in terms of the diagonal elements <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-39.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-40.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-41.gif"/></noscript></span></span> denotes the value of Î± corresponding to Î±â² under the scaling relation. A problem with this view is that the scaled values of Î± may not be an integer. In this case we can use that the underlying function <em>g</em>(<em>z</em>) is smooth: hence, if we sort the eigenvalues in increasing order, we can estimate <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-42.gif"/></noscript></span></span> at non-integer Î± using interpolation.</p><p id="p-58">For the <em>L</em>-SMM, the eigenvalues are<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-43.gif"/></noscript></span></span></p><p id="p-59">Solving the scaling relation gives the correspondence<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-44.gif"/></noscript></span></span></p><p id="p-60">Identifying the angular frequency <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-45.gif"/></noscript></span></span>, we see that this relation is identical to the scaling relation for the standard SMM model (see <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Equation 1</a>). Indeed, since the homozygosity <em>F</em> is the sum of the diagonal of <em>P</em>, and this scalar is invariant under unitary matrix transformations, we can write <em>F</em> as <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-46.gif"/></noscript></span></span> which is the direct correspondence to <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">Equation 2</a> with Î = 0.</p></div><div class="section other3" id="sec-15"><h3>SMM Models with Length-Dependent Mutation Rates</h3><p id="p-61">The analyses in the previous section can be extended to single-step mutation models (with finite number of states) where the probability of increment and decrement mutations can both have an arbitrary dependence on the number of repeats in the sequence. As in the preceding section, we will assume that the locus is old enough that the ancestral probability distribution is approximately equal to the stationary distribution (this is an assumption implicit in methods typically used in estimating length-dependent mutation rates). Our starting point is <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">Equation A1</a> in the preceding section, where <em>M</em> is the (tri-diagonal) mutation matrix, and <em>D</em> the diagonal matrix corresponding to the stationary distribution. Here, <em>M</em> is generally not symmetric, but it is straightforward to verify that the matrix <em>MD</em> is (it follows directly from the condition for having a stationary distribution). Hence, we can use an approach similar to above using<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-47.gif"/></noscript></span></span></p><p id="p-62">Inserting <a id="xref-disp-formula-15-2" class="xref-disp-formula" href="#disp-formula-15">Equation A1</a>, we obtain<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-48.gif"/></noscript></span></span></p><p id="p-63">The matrix <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-49.gif"/></noscript></span></span> is symmetric, as<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-50.gif"/></noscript></span></span></p><p id="p-64">Hence, we can use the method from the previous section to relate <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-51.gif"/></noscript></span></span> to the generating function <em>g</em>(<em>z</em>) of the TMRCA of the locus,<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-52.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-53.gif"/></noscript></span></span> are the eigenvalues of <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-54.gif"/></noscript></span></span> (which are identical to those of <em>M</em>), and the value of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-55.gif"/></noscript></span></span>, for 0 &lt; <em>Âµ</em>â² &lt; <em>Âµ</em>, can again be found by interpolation from <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-56.gif"/></noscript></span></span>. Finally, the joint distribution of alleles at the lower mutation rate can be calculated using<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-57.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-58.gif"/></noscript></span></span> are the eigenvectors of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/479/embed/mml-math-59.gif"/></noscript></span></span>.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001016/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-65"><em>Communicating editor: S. W. Scherer</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-08-29"><span class="received-label">Received </span>August 29, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-09-20"><span class="accepted-label">Accepted </span>September 20, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2011 Eriksson, Manica</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.1" data-doi="10.1093/molbev/msg032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">The effect of single nucleotide polymorphism identification strategies on estimates of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">232</span>â<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg032%26rft_id%253Dinfo%253Apmid%252F12598690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12598690&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181036100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.2" data-doi="10.1038/368455a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowcock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz Linares</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomfohrde</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minch</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">High-resolution of human evolutionary trees with polymorphic microsatellites.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">368</span>: <span class="cit-fpage">455</span>â<span class="cit-lpage">457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBowcock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D368%26rft.issue%253D6470%26rft.spage%253D455%26rft.epage%253D457%26rft.atitle%253DHigh%2Bresolution%2Bof%2Bhuman%2Bevolutionary%2Btrees%2Bwith%2Bpolymorphic%2Bmicrosatellites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F368455a0%26rft_id%253Dinfo%253Apmid%252F7510853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/368455a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7510853&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994ND12000063&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.3" data-doi="10.1038/nature04240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Natural selection on protein-coding genes in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1153</span>â<span class="cit-lpage">1157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBustamante%26rft.auinit1%253DC.%2BD.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1153%26rft.epage%253D1157%26rft.atitle%253DNatural%2Bselection%2Bon%2Bprotein-coding%2Bgenes%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04240%26rft_id%253Dinfo%253Apmid%252F16237444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16237444&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.4" data-doi="10.1126/science.296.5566.261b"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Toma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cazes</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legrand</span>  <span class="cit-name-given-names">M.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">A human genome diversity cell line panel.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">261</span>â<span class="cit-lpage">262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCann%26rft.auinit1%253DH.%2BM.%26rft.volume%253D296%26rft.issue%253D5566%26rft.spage%253D261%26rft.epage%253D262%26rft.atitle%253DA%2Bhuman%2Bgenome%2Bdiversity%2Bcell%2Bline%2Bpanel.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.296.5566.261b%26rft_id%253Dinfo%253Apmid%252F11954565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.296.5566.261b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11954565&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175000300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.5" data-doi="10.1038/ng1128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberle</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieder</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Additional SNPs and linkage-disequilibrium analyses are necessary for whole-genome association studies in humans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">518</span>â<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCarlson%26rft.auinit1%253DC.%2BS.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D518%26rft.epage%253D521%26rft.atitle%253DAdditional%2BSNPs%2Band%2Blinkage-disequilibrium%2Banalyses%2Bare%2Bnecessary%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1128%26rft_id%253Dinfo%253Apmid%252F12652300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12652300&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181928600025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.6" data-doi="10.1073/pnas.94.3.1041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimmel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stivers</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deka</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Relative mutation rates at di-, tri-, and tetranucleotide microsatelliteâloci.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1041</span>â<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DChakraborty%26rft.auinit1%253DR.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D1041%26rft.epage%253D1046%26rft.atitle%253DRelative%2Bmutation%2Brates%2Bat%2Bdi-%252C%2Btri-%252C%2Band%2Btetranucleotide%2Bmicrosatellite%2Bloci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.3.1041%26rft_id%253Dinfo%253Apmid%252F9023379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NC8zLzEwNDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.7" data-doi="10.1038/ng1911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A worldwide survey of haplotype variation and linkage disequilibrium in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1251</span>â<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D38%26rft.issue%253D11%26rft.spage%253D1251%26rft.epage%253D1260%26rft.atitle%253DA%2Bworldwide%2Bsurvey%2Bof%2Bhaplotype%2Bvariation%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1911%26rft_id%253Dinfo%253Apmid%252F17057719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17057719&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241592700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.8" data-doi="10.1098/rspb.2008.0750"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzoglou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A serial founder effect model for human settlement out of Africa.</span> <abbr class="cit-jnl-abbrev">P. R. Soc. B</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">291</span>â<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DDeshpande%26rft.auinit1%253DO.%26rft.volume%253D276%26rft.issue%253D1655%26rft.spage%253D291%26rft.epage%253D300%26rft.atitle%253DA%2Bserial%2Bfounder%2Beffect%2Bmodel%2Bfor%2Bhuman%2Bsettlement%2Bout%2Bof%2BAfrica%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2008.0750%26rft_id%253Dinfo%253Apmid%252F18796400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc2LzE2NTUvMjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.9" data-doi="10.1038/nrg1348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Microsatellites: simple sequences with complex evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">435</span>â<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D435%26rft.epage%253D445%26rft.atitle%253DMicrosatellites%253A%2Bsimple%2Bsequences%2Bwith%2Bcomplex%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1348%26rft_id%253Dinfo%253Apmid%252F15153996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15153996&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221759700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.10" data-doi="10.1534/genetics.106.059451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Identifying the environmental factors that determine the genetic structure of populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">875</span>â<span class="cit-lpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D174%26rft.spage%253D875%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.059451%26rft_id%253Dinfo%253Apmid%252F16951078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.059451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16951078&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241798700027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.11" data-doi="10.1371/journal.pgen.0040019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedlaender</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedlaender</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The genetic structure of Pacific islanders.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFriedlaender%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253De19%26rft.epage%253De19%26rft.atitle%253DThe%2Bgenetic%2Bstructure%2Bof%2BPacific%2BIslanders.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0040019%26rft_id%253Dinfo%253Apmid%252F18208337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0040019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18208337&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.12" data-doi="10.1038/hdy.2010.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haasl</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Multi-locus inference of population structure: a comparison between single nucleotide polymorphisms and microsatellites.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">158</span>â<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaasl%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D158%26rft.epage%253D171%26rft.atitle%253DMulti-locus%2Binference%2Bof%2Bpopulation%2Bstructure%253A%2Ba%2Bcomparison%2Bbetween%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Band%2Bmicrosatellites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.21%26rft_id%253Dinfo%253Apmid%252F20332809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20332809&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285336100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.13" data-doi="10.1146/annurev.genom.1.1.361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harpending</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Genetic perspectives on human origins and differentiation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">361</span>â<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DHarpending%26rft.auinit1%253DH.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D361%26rft.epage%253D385%26rft.atitle%253DGENETIC%2BPERSPECTIVES%2BON%2BHUMAN%2BORIGINS%2BAND%2BDIFFERENTIATION%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.1.1.361%26rft_id%253Dinfo%253Apmid%252F11701634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.1.1.361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11701634&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165768900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.14" data-doi="10.1038/nature06742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanLiere</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genotype, haplotype and copy-number variation in worldwide human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>: <span class="cit-fpage">998</span>â<span class="cit-lpage">1003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJakobsson%26rft.auinit1%253DM.%26rft.volume%253D451%26rft.issue%253D7181%26rft.spage%253D998%26rft.epage%253D1003%26rft.atitle%253DGenotype%252C%2Bhaplotype%2Band%2Bcopy-number%2Bvariation%2Bin%2Bworldwide%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06742%26rft_id%253Dinfo%253Apmid%252F18288195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18288195&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253313100050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.15" data-doi="10.1073/pnas.72.7.2761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span>â<span class="cit-article-title">Distribution of allelic frequencies in a finite population under stepwise production of neutral alleles.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">2761</span>â<span class="cit-lpage">2764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D72%26rft.issue%253D7%26rft.spage%253D2761%26rft.epage%253D2764%26rft.atitle%253DDistribution%2Bof%2Ballelic%2Bfrequencies%2Bin%2Ba%2Bfinite%2Bpopulation%2Bunder%2Bstepwise%2Bproduction%2Bof%2Bneutral%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.72.7.2761%26rft_id%253Dinfo%253Apmid%252F1058491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Mi83LzI3NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.16" data-doi="10.1073/pnas.75.6.2868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span>â<span class="cit-article-title">Stepwise mutation model and distribution of allelic frequencies in a finite population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">2868</span>â<span class="cit-lpage">2872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D75%26rft.issue%253D6%26rft.spage%253D2868%26rft.epage%253D2872%26rft.atitle%253DStepwise%2Bmutation%2Bmodel%2Band%2Bdistribution%2Bof%2Ballelic%2Bfrequencies%2Bin%2Ba%2Bfinite%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.75.6.2868%26rft_id%253Dinfo%253Apmid%252F275857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3NS82LzI4NjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhner</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beerli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamato</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Usefulness of single nucleotide polymorphism data for estimating population parameters.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">439</span>â<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKuhner%26rft.auinit1%253DM.%2BK.%26rft.volume%253D156%26rft.issue%253D1%26rft.spage%253D439%26rft.epage%253D447%26rft.atitle%253DUsefulness%2Bof%2BSingle%2BNucleotide%2BPolymorphism%2BData%2Bfor%2BEstimating%2BPopulation%2BParameters%26rft_id%253Dinfo%253Apmid%252F10978306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzEvNDM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.18" data-doi="10.1126/science.1153717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southwick</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casto</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Worldwide human relationships inferred from genome-wide patterns of variation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">1100</span>â<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D319%26rft.issue%253D5866%26rft.spage%253D1100%26rft.epage%253D1104%26rft.atitle%253DWorldwide%2BHuman%2BRelationships%2BInferred%2Bfrom%2BGenome-Wide%2BPatterns%2Bof%2BVariation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153717%26rft_id%253Dinfo%253Apmid%252F18292342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2Ni8xMTAwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.19" data-doi="10.1086/505436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prugnolle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A geographically explicit genetic model of worldwide human-settlement history.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">230</span>â<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLiu%26rft.auinit1%253DH.%26rft.volume%253D79%26rft.issue%253D2%26rft.spage%253D230%26rft.epage%253D237%26rft.atitle%253DA%2Bgeographically%2Bexplicit%2Bgenetic%2Bmodel%2Bof%2Bworldwide%2Bhuman-settlement%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F505436%26rft_id%253Dinfo%253Apmid%252F16826514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/505436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16826514&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239040400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.20" data-doi="10.1007/s00439-005-0039-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prugnolle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Geography is a better determinant of human genetic differentiation than ethnicity.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">366</span>â<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DManica%26rft.auinit1%253DA.%26rft.volume%253D118%26rft.issue%253D3-4%26rft.spage%253D366%26rft.epage%253D371%26rft.atitle%253DGeography%2Bis%2Ba%2Bbetter%2Bdeterminant%2Bof%2Bhuman%2Bgenetic%2Bdifferentiation%2Bthan%2Bethnicity.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-005-0039-3%26rft_id%253Dinfo%253Apmid%252F16189711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-005-0039-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16189711&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234542900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.21" data-doi="10.1038/nature05951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanihara</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The effect of ancient population bottlenecks on human phenotypic variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">448</span>: <span class="cit-fpage">346</span>â<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DManica%26rft.auinit1%253DA.%26rft.volume%253D448%26rft.issue%253D7151%26rft.spage%253D346%26rft.epage%253D348%26rft.atitle%253DThe%2Beffect%2Bof%2Bancient%2Bpopulation%2Bbottlenecks%2Bon%2Bhuman%2Bphenotypic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05951%26rft_id%253Dinfo%253Apmid%252F17637668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05951&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637668&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248118300056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span>â<span class="cit-article-title">Estimation of average heterozygosity and genetic distance from a small number of individuals.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">583</span>â<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D89%26rft.spage%253D583%26rft_id%253Dinfo%253Apmid%252F17248844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17248844&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978FK00600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.23" data-doi="10.1038/nature07347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kutalik</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genes mirror geography within Europe.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D456%26rft.spage%253D274%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.24" data-doi="10.1186/1471-2164-10-612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandefur</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Sequence determinants of human microsatellite variability.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DPemberton%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D10%26rft.spage%253D612%26rft.epage%253D612%26rft.atitle%253DSequence%2Bdeterminants%2Bof%2Bhuman%2Bmicrosatellite%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-612%26rft_id%253Dinfo%253Apmid%252F20015383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20015383&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>â<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D945%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10835412&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087475100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.26" data-doi="10.1016/j.cub.2005.02.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prugnolle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Geography predicts neutral genetic diversity of human populations.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R159</span>â<span class="cit-lpage">R160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPrugnolle%26rft.auinit1%253DF.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253DR159%26rft.epage%253DR160%26rft.atitle%253DGeography%2Bpredicts%2Bneutral%2Bgenetic%2Bdiversity%2Bof%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2005.02.038%26rft_id%253Dinfo%253Apmid%252F15753023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2005.02.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15753023&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227691700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.27" data-doi="10.1073/pnas.0507611102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roseman</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">15942</span>â<span class="cit-lpage">15947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0507611102%26rft_id%253Dinfo%253Apmid%252F16243969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzQ0LzE1OTQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.28" data-doi="10.1101/gr.3708505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berthier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Recovering the geographic origin of early modern humans by realistic and spatially explicit simulations.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1161</span>â<span class="cit-lpage">1167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DRay%26rft.auinit1%253DN.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D1161%26rft.epage%253D1167%26rft.atitle%253DRecovering%2Bthe%2Bgeographic%2Borigin%2Bof%2Bearly%2Bmodern%2Bhumans%2Bby%2Brealistic%2Band%2Bspatially%2Bexplicit%2Bsimulations%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3708505%26rft_id%253Dinfo%253Apmid%252F16077015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTE2MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi80NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Ascertainment bias in estimates of average heterozygosity.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1033</span>â<span class="cit-lpage">1041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DRogers%26rft.auinit1%253DA.%2BR.%26rft.volume%253D58%26rft.issue%253D5%26rft.spage%253D1033%26rft.epage%253D1041%26rft.atitle%253DAscertainment%2Bbias%2Bin%2Bestimates%2Bof%2Baverage%2Bheterozygosity.%26rft_id%253Dinfo%253Apmid%252F8651264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8651264&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UH19200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.30" data-doi="10.1038/hdy.2008.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handley</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">How accurate is the current picture of human genetic variation?</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">120</span>â<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DRomero%26rft.auinit1%253DI.%2BG.%26rft.volume%253D102%26rft.issue%253D2%26rft.spage%253D120%26rft.epage%253D126%26rft.atitle%253DHow%2Baccurate%2Bis%2Bthe%2Bcurrent%2Bpicture%2Bof%2Bhuman%2Bgenetic%2Bvariation%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2008.89%26rft_id%253Dinfo%253Apmid%252F18766200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2008.89&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18766200&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262550500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Quevedo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nino-Rosales</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Low levels of genetic divergence across geographically and linguistically diverse populations from India.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2052</span>â<span class="cit-lpage">2061</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D2%26rft.spage%253D2052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.32" data-doi="10.1111/j.1469-1809.2006.00285.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Standardized subsets of the HGDP-CEPH human genome diversity cell line panel, accounting for atypical and duplicated samples and pairs of close relatives.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">841</span>â<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D70%26rft.issue%253DPt%2B6%26rft.spage%253D841%26rft.epage%253D847%26rft.atitle%253DStandardized%2Bsubsets%2Bof%2Bthe%2BHGDP-CEPH%2BHuman%2BGenome%2BDiversity%2BCell%2BLine%2BPanel%252C%2Baccounting%2Bfor%2Batypical%2Band%2Bduplicated%2Bsamples%2Band%2Bpairs%2Bof%2Bclose%2Brelatives.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.2006.00285.x%26rft_id%253Dinfo%253Apmid%252F17044859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.2006.00285.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17044859&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241191400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.33" data-doi="10.1093/bioinformatics/btn478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szpiech</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">ADZE: a rarefaction approach for counting alleles private to combinations of populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2498</span>â<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn478%26rft_id%253Dinfo%253Apmid%252F18779233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18779233&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260381200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.34" data-doi="10.1093/molbev/msp025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullikin</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Microsatellites are molecular clocks that support accurate inferences about history.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1017</span>â<span class="cit-lpage">1027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp025%26rft_id%253Dinfo%253Apmid%252F19221007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19221007&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265274000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.35" data-doi="10.1126/science.1172257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedlaender</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehret</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranciaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The genetic structure and history of Africans and African Americans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">1035</span>â<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1172257%26rft_id%253Dinfo%253Apmid%252F19407144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjQvNTkzMC8xMDM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verrelli</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Patterns of human genetic diversity: implications for human evolutionary history and disease.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">293</span>â<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D4%26rft.spage%253D293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.37" data-doi="10.1086/324521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu-Cordero</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ardlie</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">The discovery of single-nucleotide polymorphisms - and inferences about human demographic history.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1332</span>â<span class="cit-lpage">1347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWakeley%26rft.auinit1%253DJ.%26rft.volume%253D69%26rft.issue%253D6%26rft.spage%253D1332%26rft.epage%253D1347%26rft.atitle%253DThe%2Bdiscovery%2Bof%2Bsingle-nucleotide%2Bpolymorphisms--and%2Binferences%2Babout%2Bhuman%2Bdemographic%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F324521%26rft_id%253Dinfo%253Apmid%252F11704929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/324521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11704929&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172086900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.38" data-doi="10.1371/journal.pgen.0030185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genetic variation and population structure in native Americans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De185%26rft.epage%253De185%26rft.atitle%253DGenetic%2Bvariation%2Band%2Bpopulation%2Bstructure%2Bin%2Bnative%2BAmericans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030185%26rft_id%253Dinfo%253Apmid%252F18039031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18039031&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.39" data-doi="10.1093/oxfordjournals.molbev.a026354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhivotovsky</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowcock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Human population expansion and microsatellite variation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">757</span>â<span class="cit-lpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DZhivotovsky%26rft.auinit1%253DL.%2BA.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D757%26rft.epage%253D767%26rft.atitle%253DHuman%2BPopulation%2BExpansion%2Band%2BMicrosatellite%2BVariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026354%26rft_id%253Dinfo%253Apmid%252F10779536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10779536&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086875300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.6.479.40" data-doi="10.1086/375120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhivotovsky</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Features of evolution and expansion of modern humans, inferred from genomewide microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">1171</span>â<span class="cit-lpage">1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DZhivotovsky%26rft.auinit1%253DL.%2BA.%26rft.volume%253D72%26rft.issue%253D5%26rft.spage%253D1171%26rft.epage%253D1186%26rft.atitle%253DFeatures%2Bof%2Bevolution%2Band%2Bexpansion%2Bof%2Bmodern%2Bhumans%252C%2Binferred%2Bfrom%2Bgenomewide%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F375120%26rft_id%253Dinfo%253Apmid%252F12690579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/375120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690579&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182474400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/6/479.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/471" title="Modulation of the Activity of a Polycomb-Group Response Element in Drosophila by a Mutation in the Transcriptional Activator Woc" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/6/489" title="Miniature- and Multiple-Eyespot Loci in Chlamydomonas reinhardtii Define New Modulators of Eyespot Photoreception and Assembly" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/6"><span>Volume 1 Issue 6, November 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/6.cover.gif?itok=aUjFCEJ8" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/479.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eS140x3wi-QuSE0i5LZV47E-plfHuGKD8UzWjZA2LwU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/6/479" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/6/479" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/6/479" class="active" data-icon-position="" data-hide-link-title="0">Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459198" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZnLr7XbbJWrr_vFM2-GqPQwNzla9-BEgxo905JYuX8A" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/479.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/6/479" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-bDGVoH6tBEqGWDKZDM6oDGCML7_LDuvflPW-C3YF9AI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/6/479" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/479.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459198" id="node459198--2" data-pisa="ggg;1/6/479" data-pisa-master="ggg;g3.111.001016" data-apath="/ggg/1/6/479.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anders Eriksson</span> and <span class="highwire-citation-author" data-delta="1">Andrea Manica</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">479-488;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001016 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459198--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anders Eriksson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Evolutionary Ecology Group, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BEriksson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eriksson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnders%2BEriksson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aje44@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">aje44@cam.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrea Manica </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Evolutionary Ecology Group, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BManica%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manica%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrea%2BManica%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/6/479" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/6/479&amp;title=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/6/479&amp;title=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/6/479&amp;title=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/6/479&amp;text=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/6/479&amp;title=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/6/479&amp;t=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/6/479&amp;title=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/6/479&amp;title=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459198/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459198" id="node459198--3" data-pisa="ggg;1/6/479" data-pisa-master="ggg;g3.111.001016" data-apath="/ggg/1/6/479.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting and Removing Ascertainment Bias in Microsatellites from the HGDP-CEPH Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anders Eriksson</span> and <span class="highwire-citation-author" data-delta="1">Andrea Manica</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">479-488;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001016 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459198--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anders Eriksson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Evolutionary Ecology Group, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BEriksson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eriksson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnders%2BEriksson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aje44@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">aje44@cam.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrea Manica </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Evolutionary Ecology Group, Department of Zoology, University of Cambridge, Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BManica%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manica%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrea%2BManica%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=479&amp;rft.epage=488&amp;rft.atitle=Detecting%20and%20Removing%20Ascertainment%20Bias%20in%20Microsatellites%20from%20the%20HGDP-CEPH%20Panel&amp;rft.volume=1&amp;rft.issue=6&amp;rft.date=2011-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Eriksson%2C+Anders&amp;rft.au=Manica%2C+Andrea"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459198/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459198/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459198/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459198/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459198/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459198/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459198/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459198/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459198/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459198/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459198/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459198/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/6/479" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/6/479.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/6/479.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/6/479.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/6/479.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
