<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/6/515" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/6/515.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/6/515.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/6/515.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/1/6/515/embed/inline-graphic-3.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/6/515.atom" />
<meta name="HW.pisa" content="ggg;1/6/515" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Scale Analysis of Programmed DNA Elimination Sites in Tetrahymena thermophila" />
<meta name="DC.Identifier" content="10.1534/g3.111.000927" />
<meta name="DC.Date" content="2011-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Fass et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetically programmed DNA rearrangements can regulate mRNA expression at an individual locus or, for some organisms, on a genome-wide scale. Ciliates rely on a remarkable process of whole-genome remodeling by DNA elimination to differentiate an expressed macronucleus (MAC) from a copy of the germline micronucleus (MIC) in each cycle of sexual reproduction. Here we describe results from the first high-throughput sequencing effort to investigate ciliate genome restructuring, comparing Sanger long-read sequences from a Tetrahymena thermophila MIC genome library to the MAC genome assembly. With almost 25% coverage of the unique-sequence MAC genome by MIC genome sequence reads, we created a resource for positional analysis of MIC-specific DNA removal that pinpoints MAC genome sites of DNA elimination at nucleotide resolution. The widespread distribution of internal eliminated sequences (IES) in promoter regions and introns suggests that MAC genome restructuring is essential not only for what it removes (for example, active transposons) but also for what it creates (for example, splicing-competent introns). Consistent with the heterogeneous boundaries and epigenetically modulated efficiency of individual IES deletions studied to date, we find that IES sites are dramatically under-represented in the ∼25% of the MAC genome encoding exons. As an exception to this general rule, we discovered a previously unknown class of small (&lt;500 bp) IES with precise elimination boundaries that can contribute the 3′ exon of an mRNA expressed during genome restructuring, providing a new mechanism for expanding mRNA complexity in a developmentally regulated manner." />
<meta name="DC.Contributor" content="Joseph N. Fass" />
<meta name="DC.Contributor" content="Nikhil A. Joshi" />
<meta name="DC.Contributor" content="Mary T. Couvillion" />
<meta name="DC.Contributor" content="Josephine Bowen" />
<meta name="DC.Contributor" content="Martin A. Gorovsky" />
<meta name="DC.Contributor" content="Eileen P. Hamilton" />
<meta name="DC.Contributor" content="Eduardo Orias" />
<meta name="DC.Contributor" content="Kyungah Hong" />
<meta name="DC.Contributor" content="Robert S. Coyne" />
<meta name="DC.Contributor" content="Jonathan A. Eisen" />
<meta name="DC.Contributor" content="Douglas L. Chalker" />
<meta name="DC.Contributor" content="Dawei Lin" />
<meta name="DC.Contributor" content="Kathleen Collins" />
<meta name="article:published_time" content="2011-11-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Genome-Scale Analysis of Programmed DNA Elimination Sites in Tetrahymena thermophila" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetically programmed DNA rearrangements can regulate mRNA expression at an individual locus or, for some organisms, on a genome-wide scale. Ciliates rely on a remarkable process of whole-genome remodeling by DNA elimination to differentiate an expressed macronucleus (MAC) from a copy of the germline micronucleus (MIC) in each cycle of sexual reproduction. Here we describe results from the first high-throughput sequencing effort to investigate ciliate genome restructuring, comparing Sanger long-read sequences from a &lt;i&gt;Tetrahymena thermophila&lt;/i&gt; MIC genome library to the MAC genome assembly. With almost 25% coverage of the unique-sequence MAC genome by MIC genome sequence reads, we created a resource for positional analysis of MIC-specific DNA removal that pinpoints MAC genome sites of DNA elimination at nucleotide resolution. The widespread distribution of internal eliminated sequences (IES) in promoter regions and introns suggests that MAC genome restructuring is essential not only for what it removes (for example, active transposons) but also for what it creates (for example, splicing-competent introns). Consistent with the heterogeneous boundaries and epigenetically modulated efficiency of individual IES deletions studied to date, we find that IES sites are dramatically under-represented in the ∼25% of the MAC genome encoding exons. As an exception to this general rule, we discovered a previously unknown class of small (&amp;lt;500 bp) IES with precise elimination boundaries that can contribute the 3′ exon of an mRNA expressed during genome restructuring, providing a new mechanism for expanding mRNA complexity in a developmentally regulated manner.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/11/01" />
<meta name="citation_mjid" content="ggg;1/6/515" />
<meta name="citation_id" content="1/6/515" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/6/515" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/6/515.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/6/515.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/6/515.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000927" />
<meta name="citation_pmid" content="22384362" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="515" />
<meta name="citation_lastpage" content="522" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Joseph N. Fass" />
<meta name="citation_author_institution" content="Bioinformatics Core and" />
<meta name="citation_author" content="Nikhil A. Joshi" />
<meta name="citation_author_institution" content="Bioinformatics Core and" />
<meta name="citation_author" content="Mary T. Couvillion" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California, Berkeley, California 94720" />
<meta name="citation_author" content="Josephine Bowen" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York 14627" />
<meta name="citation_author" content="Martin A. Gorovsky" />
<meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York 14627" />
<meta name="citation_author" content="Eileen P. Hamilton" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California 93106" />
<meta name="citation_author" content="Eduardo Orias" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California 93106" />
<meta name="citation_author" content="Kyungah Hong" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California, Berkeley, California 94720" />
<meta name="citation_author" content="Robert S. Coyne" />
<meta name="citation_author_institution" content="J. Craig Venter Institute, Rockville, Maryland 20850" />
<meta name="citation_author" content="Jonathan A. Eisen" />
<meta name="citation_author_institution" content="Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Douglas L. Chalker" />
<meta name="citation_author_institution" content="Department of Biology, Washington University, St. Louis, Missouri 63130" />
<meta name="citation_author" content="Dawei Lin" />
<meta name="citation_author_institution" content="Bioinformatics Core and" />
<meta name="citation_author" content="Kathleen Collins" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California, Berkeley, California 94720" />
<meta name="citation_author_email" content="kcollins@berkeley.edu" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=C. Baudry;citation_author=S. Malinsky;citation_author=M. Restituito;citation_author=A. Kapusta;citation_author=S. Rosa;citation_title=PiggyMac, a domesticated piggyBac transposase involved in programmed genome rearrangements in the ciliate Paramecium tetraurelia;citation_pages=2478-2483;citation_volume=23;citation_year=2009;citation_issue=21;citation_pmid=19884254;citation_doi=10.1101/gad.547309" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=F. J. Calzone;citation_author=R. C. Angerer;citation_author=M. A. Gorovsky;citation_title=Regulation of protein synthesis in Tetrahymena. Quantitative estimates of the parameters determining the rates of protein synthesis in growing, starved, and starved-deciliated cells;citation_pages=6887-6898;citation_volume=258;citation_year=1983a;citation_issue=11;citation_pmid=6853508" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=F. J. Calzone;citation_author=V. A. Stathopoulos;citation_author=D. Grass;citation_author=M. A. Gorovsky;citation_author=R. C. Angerer;citation_title=Regulation of protein synthesis in Tetrahymena. RNA sequence sets of growing and starved cells;citation_pages=6899-6905;citation_volume=258;citation_year=1983b;citation_issue=11;citation_pmid=6189832" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=D. L. Chalker;citation_author=M. C. Yao;citation_title=Non-Mendelian, heritable blocks to DNA rearrangement are induced by loading the somatic nucleus of Tetrahymena thermophila with germ line- limited DNA;citation_pages=3658-3667;citation_volume=16;citation_year=1996;citation_issue=7;citation_pmid=8668182;citation_doi=10.1128/MCB.16.7.3658" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. L. Chalker;citation_author=M. C. Yao;citation_title=Nongenic, bidirectional transcription precedes and may promote developmental DNA deletion in Tetrahymena thermophila;citation_pages=1287-1298;citation_volume=15;citation_year=2001;citation_issue=10;citation_pmid=11358871;citation_doi=10.1101/gad.884601" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=D. L. Chalker;citation_title=Dynamic nuclear reorganization during genome remodeling of Tetrahymena.;citation_pages=2130-2136;citation_volume=1783;citation_year=2008;citation_issue=11;citation_pmid=18706458;citation_doi=10.1016/j.bbamcr.2008.07.012" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=C. Y. Cheng;citation_author=A. Vogt;citation_author=K. Mochizuki;citation_author=M. C. Yao;citation_title=A domesticated piggyBac transposase plays key roles in heterochromatin dynamics and DNA cleavage during programmed DNA deletion in Tetrahymena thermophila.;citation_volume=21;citation_year=2010;citation_pmid=20357003;citation_doi=10.1091/mbc.E09-12-1079" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=K. Collins;citation_author=M. A. Gorovsky;citation_title=Tetrahymena thermophila.;citation_pages=R317-R318;citation_volume=15;citation_year=2005;citation_issue=9;citation_pmid=15886083;citation_doi=10.1016/j.cub.2005.04.039" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. S. Coyne;citation_author=M. A. Nikiforov;citation_author=J. F. Smothers;citation_author=C. D. Allis;citation_author=M. C. Yao;citation_title=Parental expression of the chromodomain protein Pdd1p is required for completion of programmed DNA elimination and nuclear differentiation.;citation_pages=865-872;citation_volume=4;citation_year=1999;citation_issue=5;citation_pmid=10619033;citation_doi=10.1016/S1097-2765(00)80396-2" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. S. Coyne;citation_author=M. Thiagarajan;citation_author=K. M. Jones;citation_author=J. Wortman;citation_author=L. J. Tallon;citation_title=Refined annotation and assembly of the Tetrahymena thermophila genome sequence through EST analysis, comparative genomic hybridization, and targeted gap closure.;citation_pages=562-562;citation_volume=9;citation_year=2008;citation_pmid=19036158;citation_doi=10.1186/1471-2164-9-562" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=B. Cui;citation_author=M. A. Gorovsky;citation_title=Centromeric Histone H3 Is Essential for Vegetative Cell Division and for DNA Elimination during Conjugation in Tetrahymena thermophila;citation_pages=4499-4510;citation_volume=26;citation_year=2006;citation_issue=12;citation_pmid=16738316;citation_doi=10.1128/MCB.00079-06" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=S. Duharcourt;citation_author=G. Lepere;citation_author=E. Meyer;citation_title=Developmental genome rearrangements in ciliates: a natural genomic subtraction mediated by non-coding transcripts.;citation_pages=344-350;citation_volume=25;citation_year=2009;citation_issue=8;citation_pmid=19596481;citation_doi=10.1016/j.tig.2009.05.007" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Duret;citation_author=J. Cohen;citation_author=C. Jubin;citation_author=P. Dessen;citation_author=J. F. Gout;citation_title=Analysis of sequence variability in the macronuclear DNA of Paramecium tetraurelia: a somatic view of the germline.;citation_volume=18;citation_year=2008;citation_pmid=18256234;citation_doi=10.1101/gr.074534.107" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=J. A. Eisen;citation_author=R. S. Coyne;citation_author=M. Wu;citation_author=D. Wu;citation_author=M. Thiagarajan;citation_title=Macronuclear genome sequence of the ciliate Tetrahymena thermophila, a model eukaryote.;citation_pages=e286-e286;citation_volume=4;citation_year=2006;citation_issue=9;citation_pmid=16933976;citation_doi=10.1371/journal.pbio.0040286" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=M. Gorovsky;citation_title=Genome organization and reorganization in Tetrahymena.;citation_pages=203-239;citation_volume=14;citation_year=1980;citation_pmid=6782938;citation_doi=10.1146/annurev.ge.14.120180.001223" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=R. A. Howard-Till;citation_author=M. C. Yao;citation_title=Tudor nuclease genes and programmed DNA rearrangements in Tetrahymena thermophila.;citation_volume=6;citation_year=2007;citation_pmid=17715366;citation_doi=10.1128/EC.00192-07" />
<meta name="citation_reference" content="citation_title=Tetrahymena thermophila;citation_title=Tetrahymena genetics: two nuclei are better than one;citation_author=K. M. Karrer;citation_pages=127-186;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Differentiation; research in biological diversity;citation_journal_abbrev=Differentiation;citation_author=M. Kloc;citation_author=B. Zagrodzinska;citation_title=Chromatin elimination--an oddity or a common mechanism in differentiation and development?;citation_pages=84-91;citation_volume=68;citation_year=2001;citation_issue=2-3;citation_pmid=11686238;citation_doi=10.1046/j.1432-0436.2001.680202.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. D. Malone;citation_author=A. M. Anderson;citation_author=J. A. Motl;citation_author=C. H. Rexer;citation_author=D. L. Chalker;citation_title=Germ Line Transcripts Are Processed by a Dicer-Like Protein That Is Essential for Developmentally Programmed Genome Rearrangements of Tetrahymena thermophila;citation_pages=9151-9164;citation_volume=25;citation_year=2005;citation_issue=20;citation_pmid=16199890;citation_doi=10.1128/MCB.25.20.9151-9164.2005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=W. Miao;citation_author=J. Xiong;citation_author=J. Bowen;citation_author=W. Wang;citation_author=Y. Liu;citation_title=Microarray analyses of gene expression during the Tetrahymena thermophila life cycle.;citation_pages=e4429-e4429;citation_volume=4;citation_year=2009;citation_issue=2;citation_pmid=19204800;citation_doi=10.1371/journal.pone.0004429" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=K. Mochizuki;citation_author=M. A. Gorovsky;citation_title=Small RNAs in genome rearrangement in Tetrahymena.;citation_pages=181-187;citation_volume=14;citation_year=2004;citation_issue=2;citation_pmid=15196465;citation_doi=10.1016/j.gde.2004.01.004" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=K. Mochizuki;citation_author=M. A. Gorovsky;citation_title=A Dicer-like protein in Tetrahymena has distinct functions in genome rearrangement, chromosome segregation, and meiotic prophase.;citation_volume=19;citation_year=2005;citation_pmid=15598983;citation_doi=10.1101/gad.1265105" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=C. H. Rexer;citation_author=D. L. Chalker;citation_title=Lia1p, a novel protein required during nuclear differentiation for genome-wide DNA rearrangements in Tetrahymena thermophila.;citation_volume=6;citation_year=2007;citation_pmid=17586719;citation_doi=10.1128/EC.00157-07" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=L. N. Rusche;citation_author=J. Rine;citation_title=Switching the mechanism of mating type switching: a domesticated transposase supplants a domesticated homing endonuclease;citation_pages=10-14;citation_volume=24;citation_year=2010;citation_issue=1;citation_pmid=20047997;citation_doi=10.1101/gad.1886310" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. M. White;citation_author=D. L. Shapiro;citation_author=C. D. Allis;citation_author=M. A. Gorovsky;citation_title=Sequence and properties of the message encoding Tetrahymena hv1, a highly evolutionarily conserved histone H2A variant that is associated with active genes;citation_pages=179-198;citation_volume=16;citation_year=1988;citation_issue=1;citation_pmid=3340523;citation_doi=10.1093/nar/16.1.179" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=M. C. Yao;citation_author=M. A. Gorovsky;citation_title=Comparison of the sequences of macro- and micronuclear DNA of Tetrahymena pyriformis.;citation_pages=1-18;citation_volume=48;citation_year=1974;citation_issue=1;citation_pmid=4218159;citation_doi=10.1007/BF00284863" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. C. Yao;citation_author=J. Choi;citation_author=S. Yokoyama;citation_author=C. F. Austerberry;citation_author=C. H. Yao;citation_title=DNA elimination in Tetrahymena: a developmental process involving extensive breakage and rejoining of DNA at defined sites.;citation_pages=433-440;citation_volume=36;citation_year=1984;citation_issue=2;citation_pmid=6319023;citation_doi=10.1016/0092-8674(84)90236-8" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. C. Yao;citation_author=P. Fuller;citation_author=X. Xi;citation_title=Programmed DNA Deletion As an RNA-Guided System of Genome Defense;citation_pages=1581-1584;citation_volume=300;citation_year=2003;citation_issue=5625;citation_pmid=12791996;citation_doi=10.1126/science.1084737" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=M. C. Yao;citation_author=J. L. Chao;citation_title=RNA-guided DNA deletion in Tetrahymena: an RNAi-based mechanism for programmed genome rearrangements.;citation_pages=537-559;citation_volume=39;citation_year=2005;citation_pmid=16285871;citation_doi=10.1146/annurev.genet.39.073003.095906" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_author=M. C. Yao;citation_author=C. H. Yao;citation_author=L. M. Halasz;citation_author=P. Fuller;citation_author=C. H. Rexer;citation_title=Identification of novel chromatin-associated proteins involved in programmed genome rearrangements in Tetrahymena.;citation_volume=120;citation_year=2007;citation_pmid=17519286;citation_doi=10.1242/jcs.006502" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Scale Analysis of Programmed DNA Elimination Sites in Tetrahymena thermophila" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="twitter:description" content="Genetically programmed DNA rearrangements can regulate mRNA expression at an individual locus or, for some organisms, on a genome-wide scale. Ciliates rely on a remarkable process of whole-genome remodeling by DNA elimination to differentiate an expressed macronucleus (MAC) from a copy of the germline micronucleus (MIC) in each cycle of sexual reproduction. Here we describe results from the first high-throughput sequencing effort to investigate ciliate genome restructuring, comparing Sanger long-read sequences from a Tetrahymena thermophila MIC genome library to the MAC genome assembly. With almost 25% coverage of the unique-sequence MAC genome by MIC genome sequence reads, we created a resource for positional analysis of MIC-specific DNA removal that pinpoints MAC genome sites of DNA elimination at nucleotide resolution. The widespread distribution of internal eliminated sequences (IES) in promoter regions and introns suggests that MAC genome restructuring is essential not only for what it removes (for example, active transposons) but also for what it creates (for example, splicing-competent introns). Consistent with the heterogeneous boundaries and epigenetically modulated efficiency of individual IES deletions studied to date, we find that IES sites are dramatically under-represented in the ∼25% of the MAC genome encoding exons. As an exception to this general rule, we discovered a previously unknown class of small (&lt;500 bp) IES with precise elimination boundaries that can contribute the 3′ exon of an mRNA expressed during genome restructuring, providing a new mechanism for expanding mRNA complexity in a developmentally regulated manner." />
<meta name="og-title" property="og:title" content="Genome-Scale Analysis of Programmed DNA Elimination Sites in Tetrahymena thermophila" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/6/515" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetically programmed DNA rearrangements can regulate mRNA expression at an individual locus or, for some organisms, on a genome-wide scale. Ciliates rely on a remarkable process of whole-genome remodeling by DNA elimination to differentiate an expressed macronucleus (MAC) from a copy of the germline micronucleus (MIC) in each cycle of sexual reproduction. Here we describe results from the first high-throughput sequencing effort to investigate ciliate genome restructuring, comparing Sanger long-read sequences from a Tetrahymena thermophila MIC genome library to the MAC genome assembly. With almost 25% coverage of the unique-sequence MAC genome by MIC genome sequence reads, we created a resource for positional analysis of MIC-specific DNA removal that pinpoints MAC genome sites of DNA elimination at nucleotide resolution. The widespread distribution of internal eliminated sequences (IES) in promoter regions and introns suggests that MAC genome restructuring is essential not only for what it removes (for example, active transposons) but also for what it creates (for example, splicing-competent introns). Consistent with the heterogeneous boundaries and epigenetically modulated efficiency of individual IES deletions studied to date, we find that IES sites are dramatically under-represented in the ∼25% of the MAC genome encoding exons. As an exception to this general rule, we discovered a previously unknown class of small (&lt;500 bp) IES with precise elimination boundaries that can contribute the 3′ exon of an mRNA expressed during genome restructuring, providing a new mechanism for expanding mRNA complexity in a developmentally regulated manner." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
    <title>Genome-Scale Analysis of Programmed DNA Elimination Sites in Tetrahymena thermophila | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CHcSF6Esk2j5CaZ1MxA_0Y974Kb89Frgeq295Tkr4Yw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459197","apath":"\/ggg\/1\/6\/515.atom","pisa":"ggg;1\/6\/515","ac":{"\/ggg\/1\/6\/515.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/6\/515.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/6\/515"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/6\/515"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/6\/515.figures-only":true,"\/content\/1\/6\/515":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QYYyBP+Gh5ZNk+6N282b4LXqas1esMvcpqYqKAKcw\\\\\\\/EnK+w3PnzcrzHxZ2iw2BXvS0JqpKzfzFgg31eo2HnzAPP8YflJrc0H3eePbt6Zn+kjYEjZZwcTb1Qmf0F3B5vYdyu2tpf\\\\\\\/DltPJip48c3T2wC179jsyFiyG2UYiAIqhhCgdN6r6kpxtrYBKBT2fTiZYGMEGQe1nFQfoUddGH2jVIN6+pNBcAYxUhCE8yS\\\\\\\/S3zc5NEohiE4jGdUmtFjTVrWRt3HewF0y7OgoWrNu8BuhUAgFowh9Pw80GZBS2M6QHzX3GUDiZZlYbjG4d0uhbERhOd3Mnrf8BipyFhcOF2hxoAHeOxeyg+73+xoI5zB9Q\\\\\\\/jXz\\\\\\\/9IUOPD3zsNu5bgdzOqA6yQKCYYLonMhKzcSNnAPkp7LRggZ2Vyc7pshny++N\\\\\\\/f+L1U0\\\\\\\/Ukt+cCqHpKcVxsYY8ur4T\\\\\\\/wOSsJHNzsA72tyQOTJYxPG3GHJK7JUNc4eIlfgh4c0lwPhgQ0ruDJmX1MG\\\\\\\/UjvC9lXM6hFT2KGD3dqIg6s+fMcR62ESYnD5aiPoMgjc23xW\\\\\\\/hTO1E7g65\\\\\\\/G16AbC8hhCN3geqyRUfajzq2\\\\\\\/Or01NmaTM1KcAn9wbYlCHX\\\\\\\/+0O8LCKgQjO83MQGcipmJWgEJVZlWiJL3CYkyao5SDt0itIQWkKHqpv\\\\\\\/rdS4vP+qKCiKunN70asfPvtyrlDsBSPmX91eBd2wtQED5+H+XajbRYSMzI0VUq5Urpo5og8LP68\\\\\\\/UQTia+pxF\\\\\\\/zf9\\\\\\\/y\\\\\\\/yzmAQ3mH9HXi2So3qiieUHn6GpCVWCLnr9fTG7dxocuhmGZ2brRecNyu8N3V1Tv9sii0jSUvkt+36tzNAahyJcDbnkoXMnSOWnIKQzl1TXxneNvE7zEq0o2y5xRy6ICDGN1m5NyE0r1cs9yM76LmdGp4vNu\\\\\\\/sPfi4Wzi0qHpUyCNoF++HJaMeuZJlGKAtw\\\\\\\/jM\\\\\\\/LJ1CaQcFAK97m8\\\\\\\/7ZSLGjDHFkK3ysDABWzpzN8Goxw0TmE7hq4EUIQ09CBgws1T4yKpjXnY5qtoeGeRvjumgbWX\\\\\\\/DylQaxmov0fculDApu\\\\\\\/ETydxJGU8KwDLQ0Pey\\\\\\\/jYI5SrT2nGh1TEcJVd07TrrY1zvH0+AOm\\\\\\\/+T+Ysh7Vge\\\\\\\/uP0GV9QlBzKuWg469GJQSyZ0jik5exSbl82+l7RDbj\\\\\\\/QRrqv73iu0EX0qWk88HhXsJzbkEoe+3rxOR4RyPCCZ+TTLqaMPmmu+igNo1DDHlp4jR3KC5efb44a3UMLnNiBZJPATXEd6XWpr6vQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022RjPKMH8nToXj4hr1bFgWog==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002227de4b4e386adbcae43980a377985eded9b992bc2955b3257b9bcd929ca146ff\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IznkTHrGBRbzgQRbyTleygegecD8eZK\\\\\\\/tpNOa5P2yH9SDiQik2sCD+UtGTnaV+PFlouEtu5z8LJdBpO6yf5G6tIwPhJf5KA6wsN4+GNQvIFXxhSBBuiwAyDpO+lW8LeFGic0TRq6HH2ZM2AraepyIU3rZuVaMHl6QoKa52qlonY8ayG1CfjD5jgyScuasn9UXRhgb080XcmC\\\\\\\/Y0HVmAaFaYAxA9od0cOZvLM1Qe+Lj8WtOVcz\\\\\\\/nqt9wySObm09yDw+cQipNsJyiDQHjIEmlln+s+lME9sD4EvbEnpGGc0JlE6mMz\\\\\\\/wOt7xUB499naCCwrgmAtN3aYhzzhcUBWGADvZh5bcam3Bg3CAmyJfdwQoo2StC1exnSiPQcY4e9DqaEtjc9+SDkihqA2iXrgWYqz9dGVduxO+bMkyqgiy1a29rpeozQs2rl0zh\\\\\\\/DY4jghWwdtpVsEPirY2\\\\\\\/hbXJRQBGhs5pzGtfSMYlY8zIPbYfzCNWKQkPeYjIouRG0l2pjNIebCqj6UCo3\\\\\\\/5\\\\\\\/dre4TQCXzvFXyB+IC8uFw3lx7JGUG4H+rBGZaHO1\\\\\\\/O8wYW4CFwWD6GWyNdsa0NOPEHxwPCBF7Oz05pdGWcHkI3JJ2Bg4uDETMZ2Y5Vr2mnuD8t3ZvmT06PYdYklDnS9xN0uaIrkflZ0xB8xqOhvaS7zT0JgW4uVOz\\\\\\\/XeCo7fyLwMkHMwwhd3naAiSZJB\\\\\\\/bKpqH9Fq7MiC+d+wO+Tkytv\\\\\\\/XMTE0beh4U2aYrGXyexvBw8dYGdF6x8MVaTGBUjOvKtTAYhMi7bI8tN68WZoXWJhKcTRmH8nMQpKPOc8Ud4lzR5UERA1wpubB0wPrVzhj04oLsRXo1AmOk8ezeSOoh58SXaS7ZkOr4J6\\\\\\\/oAO3m5MN4E5WYmxHQ6N4DFdFrZ+U7EBRcs3rVRiJnT7C+AboozfJ4uQhe7y674I1brixOqeHc3e3JHWDqA84JSkKcw\\\\\\\/m\\\\\\\/HciabPSk93Hw+Z+8DuxpaoyvY58MpSGvItjPPI2AIZGmfdP5YMfvcji5E96A26SXh02IUPiUUfrSYlb6UoS+RDG3vMAOH6jQZ\\\\\\\/r9NYSMJx7Qoe80b\\\\\\\/JmoGSNcVyFUXr7CVq98yObpKjT8gTvQpSdGkpm2Eq2YbtATd+6tzBuV\\\\\\\/xFKLfEEGY5QMVtq61vy+h3cTzg7oHdcCvpIkhIKrN8vUmp3W2mUip6LwIhq1xeueEvFk8J75XlauviJEJ3FTDJ18tOZZfWrOIQlTw2TqKj7kVI6GSzucF5KbcP6ztzE+z0PYbD1YD\\\\\\\/msj\\\\\\\/oFHGWp95m1hMkCeAUAwTAyOft9Bp43r4ifrO0itltIzOtznYNNoStlhrGKtykaSO5RKdptaruPdegtXfVFtQ+6WLJ6EfG+FstO0O2KXXenks0Kq6AFCmlf+wIvoBwxywG4tOZH5pvuzHVsH5JQLzAJt4g96+I\\\\\\\/R+1yjcNsI8g20tQbbm6SoHp4VEpfcyavCjIJNb\\\\\\\/XCFVaFs6UGEGbW3NzYq1nsXtOr30xoZwNFd+gqlnrIhGbucNYEZQfRP9ODyOhl5tJAf8Tk3SZcXRLGED5w9\\\\\\\/+G3rLPS+LBIHe2Zb9dMytKI2OsQ3buupt5E49wdIeSt7\\\\\\\/b1XPV+99D+Wau2R4UKQj7ybz1g2csWG7lbor5V80mXar7Li5UL9uuV42EZCPXnQE3RIpH1VDBMWcLQI5gC5nVz25W9Aho\\\\\\\/FDPMWuUTGkOmx6fpIdjM6Hgl78GM33RzuvNMB2612QGKQEW8p\\\\\\\/JTsev9FfNKXUK3JRNQqONGLOfhwbsHXIlc0Rn6+2iKKma6KZHY3GW6w3A0EUcBm8yJsfyUScMuwSxFjDsESoAi8jvnbhzZ5NjvzN8gxUQSw4TyKgemcYOK8PdwVp7x5cyJvy7C4a2+rrYCH9pnyvdSMXbkEgtxxJ7J9mlKIpmiRH7\\\\\\\/6+zasIIk35+FqSndfSz+gpa3Cf4N+IgyYdlnwUyS9huGFxCaY4rxfzT9rySsZ2IWHkg7Sp9XehxByyll55PjbrvGSL+kwZbP4s6xvVO6gQLJUWN5cYj0yAMkCzdu0hj3XwByzf6jg5ee4fSfpRsvUFQmct2Y6PLKLNb53nzyuldGT7meq0Wj1mgZK0crySE9D66OeecfNd7yZ3lB2CGutbcArVUpDbhTgkY2Crgj6U+yI2ikN19kBmCfX6VEMGpjPuZ4OIg5zwrmrkvKlkrnhwwNza0znbsc0ewbQfB2WafhMLOUGVAsD62gO2ciZIdqnkpduz0U2gQZ\\\\\\\/\\\\\\\/pB4+cFX5RKBwY8yb1lgh3rjPHT99FXDCIbdzzp4q0dfsKnnV0NZAULdgElioYghTx2Z0l3ewxMGFUBTUmNapvfLcU3mnLYqI6EJLEWxyW4m3sQtF8r4TTJguv+sL7gslVDmct1GFPzY3rIgnckugq2vXWHht2Eud919IW1bMFXBHYL1It5Qi3RatEEGqLmVZ3uqinNMFSnj+7Mb\\\\\\\/K9lfjxhDa0UsBh4agB09eKAJ4q4fnwOCY09B4b4aTYxdkna5MrKG\\\\\\\/xZepfdH8ENG9cCQew7Bp4pHjNKFTgVebdtagwcc1wZNmsxOVBXWQACsuRk1leWD7mJwpG1Nc1fg\\\\\\\/ZTKZt1XCL6T0oxm5EfjJpfaPdSuSH2lhdeohw9ueBpUXH2ke8MmYpUyNmVg+pOxpsaHkCW3i+yWbiJPHnNXw4tmnnVoB2\\\\\\\/NheS0XrmviFrPLfMNiYcH9QvFbjQNcmE2Se5jA30dOfPYwlvRVXyjBb+KGK17MG7XPqqpc+j1UHJPHqGv9duz6mah4bXmBTwKGi+WWjHLFjho1u7E\\\\\\\/GHGCDp\\\\\\\/Vhe3uVlV3T6SyNj3Vzs9cGyOn64QZKb4BaYFj4Cx3HzIa2P1hb6kKpL7D3xOTmwjzKPCagn2H54xuW3u+L3Fm3W0ZhdMR5\\\\\\\/nibIDc5PBxsbDLCxZlxlPaSmTDyNu0wHCIhmdDBO2MxxPLnnu87EzLjMhcUFTyH7qtGMvIbPtp4i10Prv4BeETY9sAAytQ75Yzte8OqZW5ulSGFo26dmjFx8HNofFrGfC8m1OvLF2v\\\\\\\/v\\\\\\\/T+RGSdEum9sGwdjb3j2hbKGALGE5KCqt44blQ9NTdt\\\\\\\/AApgoVTG9vfrYOu7c5J5fksyNnP2T644OTKWMa0s01g1sVG4ihTTAUkBmumEy+\\\\\\\/lBVnqUp8\\\\\\\/\\\\\\\/amsMRROMEovtUArkdeuEzXN9hZyqmAVbPQEXuuCPmlYJ+jwsMW7dxAkHV3h7nmWqzysmX3ETRKswSd+PBlAQVYnrdUexLTpDIldbT0FROC9xBtMOQTMr1MjZct2GIQVDRdhYczRrE+Rhv+8fl4neoK1xmAkAc8y19fzTDnPPgbXWFUVWUUj2pan0MlRAPqPz8Qgy9yaHcF3Ymb7Cm+1WaFepTN7yubhJPbxfPVLk86Yh0TtA0bDuo3fwaJVh1qLyeG7zLJLY8P573NUB8Ro3Gx5y45wGQWjQPZz51Kj7Mhci\\\\\\\/56szQ1dfC3uysgZ+zZzvxPHkp8fMgQMHDfw4J7Uok93owy2BV9i1MN\\\\\\\/E3hmlBeXrvIWIsAk9oTkQEryazfnMo\\\\\\\/MnQ1LovENHe+rVciwBUVrMmsCRkbwxC0XRkdTL+bOUFsdp9oPJJf17qEH+w30+XSP+WHYZYIJj2cYE5HmU22KtxdoADjjqwWZiWIHyUP8gwEKY5nxEbP1OYBXfEZRXWcJkZnWzvPrJPj20svzOk\\\\\\\/r1dOJFe4bWy3VOFXMJH8OvAYirf8Rps6H1twLjyRUEZfcoDDfg5fWvjs8POE2h\\\\\\\/86nK4Vlzk9IbKIgtEtH1RMEBaV2p4jSxjwCXB3J0bWqhO\\\\\\\/QIa+RgLJ7CyY9OTOobeRCfdOyR7kDfsdWmW+EnI8mlXhDWmIJ1RnrAY8HVjnxV5sFKjdd7FxZhqLxvDQdXr3fdn+xXPpq8lsYINXMRnJQ03kdjyDLsYo576lDvvaaDrFzdzKkj1eiFb6IAkt9X+Tyxb29dJlQMsVvsVq01rnHJ9xzM95HI37O+C82J8MOdjDXzTrDpnkDpJ8srQj2\\\\\\\/jZPfOIi9GchV1kizVueQwdfJeheMQBI9Car3i0M1sH5xRiNVr8VoCPTIlU7Bj3E6yYIO6iGm\\\\\\\/uWODKrPdLcBOy5Ft8Ej3otlemoQa2+7aDYXbXt6tlgL10YzUsZTM5uZM4s2kz1Q2SiUaauQ7\\\\\\\/3ZQiI2U87NWX4maPFFYFSpIlvIwP2djnIwuc23XQJ\\\\\\\/HN\\\\\\\/c7ainm1crKScfOH+pqRT9nt8gE\\\\\\\/cn9HHMVNv99iY\\\\\\\/KypzAdjt9jiHiBPMFmNlgylHFJ9L8\\\\\\\/GqPX0\\\\\\\/1YVopbyOMEJnlzRcTPoVJ1\\\\\\\/kAQOTDPEtw0JZdMZUrYf61rPZwQg6NASzYNoyp3CChe+DU1DFwCLCef6JFp8cxXRF\\\\\\\/iuR0yMJR8krJfAk7J5D4nFbHbaR3BhCDb9pexyER3Qaa3FKoGOQsH57B0RLISwfz+k5BPCK984aJ5U2Uz819UJP52JV6VvFWuosb4vW3JpZ3q9BXTj3P55VBo3xYKtDynTKcj32wCR2BOwcjsvLHjFt5+dTtSFtE8iog0A5t9XWMlxJAGXkAgoku9pjqFuaxhk+9IcYVilAlebu6JSebDmZdnvANRm+gmXdXbtZL9t4Zosrx4IKxgI0ZbuoE6Kh\\\\\\\/kaPypOG3eFq9CnkwlSwlYnnmcmv+ZGfYR4ePIxmDshESyO63GCLD+HZHjaEg7kOu5zeOX+dbXdjjTH4wTHzkBO7hZATMtjndOE0RVNKQL+pa2FPz5rLQfmlYUThEkTVm3o03NKGp1mUe7veFAaT7zLASEQbbk9eEAZfv2m46WHr0i2VSTJ0WA5Jl7BHXGw5pN1WU9yPWzbpNHSXtpIZvy93umjt6u\\\\\\\/7BAazzqEQdma7SwIkHBEXtFKgZ7P2gOx4kEunYM4cfcqsAEOlSkk8MQhYjhsgoBPUbNhNO+\\\\\\\/LIBa2bT3lVvoqBEZ5YjzBlsHNfOhyy6Dz\\\\\\\/5f0RjlGnxxIsrx2M3bgi\\\\\\\/soygO8WMyDeDSLibHIjJlvqf2KZ\\\\\\\/Gy5l1g9r2JyEJiUq+P4IC9Mx13LTjvj6rhPgOO3GDIOVP6ZM3zzHWT9B843Gw3Y+yh4HXR6iOKLtqoD\\\\\\\/GCyuWLJdYXQg9Qs8AiEsPSGDiQzNaIQ31dS+N0naUKhceoYVoIWTDOrpaCfZXMd406K0ARZR3NdKiFIkltvBSPd70UL2N2ePlnxgk1ykKpheC9KPxKJbGJw35Yc6DJ+KrOjkmvxUmW\\\\\\\/MVR\\\\\\\/1PNs7R8XqdkhA5riLxmMV4vAUoYhNJkWj8ge3izx4ZuNz7A7UcN+BIp4adOfBVzAPEwRlTnbrweR8VPNr1ZftmBFE47eKDs++WNdQeCAptYX8NlF7u+OAbMKCD3JQ7JbcQafz2XZQbbanbx5mMQn0wVdu98OEVZCqODx\\\\\\\/xxDCvb205mmFw2bpCxKMDmxttCA36H\\\\\\\/lTtFzakcPZkHVB9BeK3ihVgvYaeNGigUxt6cCSpiTzMI4DX\\\\\\\/mz1nKUx6NxV5irJrgoCriT3iziqK\\\\\\\/WwBFL38Zq17KFpz3lao\\\\\\\/ge7QAvu0Q7XenEt7IBX\\\\\\\/ezKWx9HSuxMEiO7DT2co\\\\\\\/olYLPJDKmsBfLTasMxiDZNyX3kZcVf2mFhbjVvc1B9u5x5kH+XH7ub8Xe1R1LZJuJ7ycoLThlP+W9hMU1wJmX0WRBVM6AxgumhSpkJuQOA9+g3of8GyDeZ1diS\\\\\\\/LPyuFmqmDEtq7DDsWf8muO3EH7qYrs\\\\\\\/k4k19NtcD0OAUZQFUefap1r9djpMvrXZFx7cHngOIutTCfQpMgZQJNWGPm1rIO\\\\\\\/LtaRq1Kpm0x2yKkVwRiV6N6ZNtAoqwvWQqPZts4R\\\\\\\/qAU\\\\\\\/JsWX1gVg6efEQ+Fryi9eSmScnKWmVmryg7xeI7ZXzzuCiOiNCu3JUoo\\\\\\\/RTC1jBez5vUbiUVgSdI5yu11WDeCG+pQHUY77SjzU1nfxfdVlJxlGV873GCCSYP22GWj8g7cE9kdsc37wNRrzHaYlMrqhGZ5q5WEko576K07DKDM4mciFpvxqU6xUz35Hr9w2D28f8hh6zavjzYyVY6OKLoaDtR0in1X5cJNTz2ah+sZdFYKY2COa0U1C0Iq6+NDXBYhCYZtCFM989c0aQNJ74dADjfg1VDGIXWVgGZH1aid8xuOg1UmMFzS5xl49xskzEddtn0ptbFYTSOlmen3bdjfj+jWvysFIPON5D2T2OrfBP2eo54VZ+4xyVvW+8AEHvyXVrRNYKn\\\\\\\/zW2QO+HNvwq+jMEgPFg2Tgc11qa794K7biJ7KRGgdkKbJDwsC93brIwTErPycZKyuoWB\\\\\\\/vikKplZuW6xDB4kHo32912LplMvr5PaRJx5Lejt+PwmQ5QdWcqTo6sgvLW\\\\\\\/Wp2T9yf1GowFbUp90FpGZ2bmYqlc2JTmTZIcyVVyeIZWcyqhQYthxO5iXD67+ynq0IFzyRg5NL4mk+YxSlKr61h\\\\\\\/i7Kr61UqF6A+Ro3d+H+eXywjfxdgFjr4MKUq8i\\\\\\\/\\\\\\\/fg7Mg41XQbQ9P7B\\\\\\\/FujHYJq9zfs4xWntW80POFbyoCrJz\\\\\\\/CPQVaKWtHb2G+BPlFNgoxnTD\\\\\\\/VZRDu4C1\\\\\\\/tuqL4hehMNjfXYtGaAywFUnIsUvJNsCbcC9XgJQGTgWh7LvJ+Am5ghTv42l3a3RoP\\\\\\\/G7sSf2\\\\\\\/NW3HW4UjH2Jv5nDbwXDbgzfpioelNxy6DEcN5QQ5pCO++KAgcFPM9vpWyHqHvDhzHQPq1Z4tDvzFasn\\\\\\\/tF8\\\\\\\/QyDgMRRPl5BoQOr+eyLtGhBLMw93QcEl5xnIJcGjZ4zD9MFd04p1DQEl9ihbniHBb\\\\\\\/bSY79hTPzCTs6FOf3CzIWLNYgzg5OT8mnmicjcxYGv1DtcCL4pX4UYCqc96DGkysZ0xEwv\\\\\\\/4c98tlQxYaAK0iZLhN\\\\\\\/eNuCeaVFO1klmUgGIfCKvWY+WVwtUEjyNY1Crz9jPDvwcRXjfGCocWGnt4ZGoVtb2CBAZXzMJeSfP\\\\\\\/Xk+J0i93opkWAXzoU5mnRZAfvIdmertztxU9fOwBAgm5V0uZD2tYiTHk6+fDZQIx8SLkL\\\\\\\/aJz2ousenG44VcslCLnwTgXBU40UfW\\\\\\\/Bl6bbDsbQUefR6CJmFZaJHXMSZr831enQCQKxY+b5BjjTMqIkZE5Ga9z36crS+NtXRON+GT\\\\\\\/s3\\\\\\\/ujZ2kCbCDrUEfns6dkC76fq7MOL615p0qM42r+b3nCPgnRHfFxEmHJVK69uBpv1DhIGZ4\\\\\\\/8tV3pI9PSXjw7UrTWySKaRhKvbsDapkOg63IXRW8llGWwdgiO3ej2t6XJzuHtLuSLrff3kYpaAMaKo0CgTrAjpyi1c\\\\\\\/w2ns4FWQJvpkHOSrg+06BgtR8kXBPKx2HnriPw4fH1XogqqLGahpqnHcH7CgWE5NDh96\\\\\\\/eWto7zX0bP7xaK87auadVGu\\\\\\\/jA6vhtKdYS+R2DEQFxZFlQgKw+Lqdr1+FlEtRPKktf\\\\\\\/Bd83vhCb83R1BkdXao58nhdJVeDUNv7fLYOeEr3E24JRp5f8FDZmM+WCFw3NUWIurdH3\\\\\\\/rmMgxu0gDYo9NZQxbT1DD93nF\\\\\\\/+3E02iFBqT1z0jKHiX55SGgg5XQagmtsoq5Pogy8zL9Q\\\\\\\/mf5LCaF3P8PXqYYnkazRea+R3ZkP3OS76DfBTV59Z775kariziTya45fgBCS9Jgu1\\\\\\\/f3+JkecJNN3alc6xgt9DBOXlDS9FNklpBbeYRiQTR9ZUiFIH2LtPR3f0sFxUrYioXj6snlfhy1FvV2KqnmcUx4z4qYodL1iTpAhls\\\\\\\/9tLoY2j2WBHtjI33FlKPEBvp7kSuwLOo2TiI0lYzx7zrRICEcTmbWXO6GGUMu0GxX84A0LZoFqWQRrQPSAsfr9WiagJQwqnF6JB+aKpnR0fVfs\\\\\\\/l+ponF71K0C1DyUkdLFvizjtJ63EckTsvZehbQX\\\\\\\/rLWG43kA83wWPIfE75tdq5hEliPKPKeNTBzTCiUNkML6GO1U+lwFQuOo9LFnzm0FO\\\\\\\/fh87OOoVqo03leJUrrJQvb1qsRTBZXI55kYsQj5I36xsBvNCVHJOZKrs3iSwMF\\\\\\\/IVIY7kbuNhTm\\\\\\\/FvCUkw6DvQFIzqlsxGl3HZokSpwSYWPMhtkjnoYodiNxoIyDEwzTHUhgw8qzrrNUZdRnpLLe5BExTBk38Qp0OanJvLGK9tHLZB1jrRbGu5baM\\\\\\\/U9QYASb91+NHdLU7IMLXc7Xa5UP5ib+wDZZ4nJdr5wTZDY+oa8pZUgocOBn30baZsQTsvihU6LbxtDTA0eonhG9b51aP+hC7RxGa6qWURScA5EO7V3Y0WrG268jSrrdJcHfxzsndSsIhOU0RdI320Zl8L6LsGjGEMZrnTzxTbpPYJ2HOx8xKLOJe6aB849BaV8EX+TuK9jrMfRSflzHTjiyFMfUMDKoveV4oETzwVkSoBGL09c0n5RSAWRxcdg5bgm9bNIyshga\\\\\\\/YepRrtxYjlLEx1spxmNbm+8M32swDunmYxnnXu7IIeIhyPvOwogESFsQDe0kSLfVrVjGvOHm+zW57jfhDt2eSTaOk8EshQcjIUNNpbyZJvWVzM77qNDLIPGAyu3q8Ro\\\\\\\/fxnr+JvAQi+YUZB97oYlDLSfG6JLl689NdmJorVmDHklIJQi+DvyQIRWtMGBiwrXuHl4tp+8DvVezqlakecnsyscPWhquKR7OCWYqhen+b4MYM3Y2\\\\\\\/fJ25xKordx3Bb\\\\\\\/K20mhGnBuGcuOrFvumtpV8Aoy3+YxGiLR8VGKU13WdPsFmiE94WIsa+UJtCsWMlHE6B8FDAYJdIlO3bsc80U\\\\\\\/33mW2LQfQrToEViuxqoIXbNX2mHXJWs3CGritZ0YyPylD0RIC9ZRfXH2z95dmuClbO9jMwtgNYZtqbx\\\\\\\/V+Db2zfqDT0BHCot\\\\\\\/jhazsCKx2DWfT1NZ+pGAfiosGfUkLuh+X78L890ud9Plarb7s7ZIIMN95JqlTYaVgL2a2fYtsKEJEk+pMVU7VMV0hLBmdHcjCAjt7Gq9XEz4VN2W0eo7o8Zncg+oGoRntG9SFh\\\\\\\/CJ8\\\\\\\/1MMqLQjKR9rzdSqaW0tf03SKcAbLury2uIO7MU5ud45gM7ObBs60rnWcfCium\\\\\\\/60yJ3AMndw+5dBU9+NNkiET\\\\\\\/FH4CdIeMNPoNEBmxrMyu3v7nEwftYgiaL7a3CQ\\\\\\\/2kgITXsHK+g0nm+Fi1zWRQX\\\\\\\/d+GNXg8D1wFVGCnu5\\\\\\\/EEM49UGF\\\\\\\/GteLO270CBV\\\\\\\/PKOiekVXn67c4WBfgJwwOTjA1fMsM50iSfNaHgUjvqE1wytktooqi52zMkEUup3z9ECPS9vHt\\\\\\\/BWVPKuGmyaiKnKxazUXyZBh1vh2czlWnrwX7nvqAi5rhTrsRdqgCk38Q72pFjue3i5OVSDq\\\\\\\/eUpyWQ+LfsEC5MrAZ2+suRBN2BK2Dcorj92L3+lIUyBZS4z5AaZ2bhESsVfhKcZjLmaDcXNqMh8bb9pd2y2sUDQAlPMuwWLJuhnzCSaPYE60CVO4azYtXcZIisrHalb2eE1x2zBYniWxcacnlXdOCArdpR\\\\\\\/UolW284UBxYJ4cn9Ff4OlMNa843RQsUI6k9HmOhmThlGdWpAIMiWGlT\\\\\\\/OqC+G3hHNDyLPDyqyr9IFvIasxPeI7Y7YeBhZCWy9tg3CYfkmZeICaupvaRhbc3dtAGzWSgKXcfmdBFpVNUulXP9M1xIxwr61doeFUy4PJvzONDcHWyVhxeSmqmqHZ+JpwQwLr0w8w8x+4QRHYF+QmWZ9s3HnGusk6crqeEeBZsiB277O1M4BVzLu6RMcMwYvbUMFJmgIlelbuipwFxseuDIpVsC8alyP8o8Lz+9H53Sp2ky2kRx+FondAZqDRFIN2733uRoZ1iAYp6PJCFt2gjSt16J9CVam7KbG0\\\\\\\/b0gxECgJz8tJZb7SgocEr\\\\\\\/A69b+HUmnyM4k0TzxQmBgo6aGq32KSqDmIgy67MQiH7p9j8\\\\\\\/sB++xeE720JjldQ586uyq29vRjGZFMsHQGlJHkOhDkSLSrvJjM4RyIFSwTJO7ZOeXux45\\\\\\\/xe55YLx2m8\\\\\\\/nS+5gk\\\\\\\/OZ\\\\\\\/D8Tax4HdU5T5EhPTbmSIk4JxloRzxjZcLWoRRvevL9Er77hvETnIo40SjaOEFHHxVr0NC6fZwdi1OqwB2I2prPGoTErtUh7u0j1asCJ2eV15C4cHhspDgrHF7M05MUhmvPCzWFUGScQnlZ7OOh11s2j48Mow2pX0caWwHt79tjcLgA9vOGk0Yr5ATPLq6dtacoFBix5POfkmODE0cs9sn0V8pIvcK1xg1nnaoGgsnZze2kcDJbQO1aiOPvDfmKHKWYhmZsitxDi1gXxVOZFEHVOLHW3i7TWvJjaD1qr8nTKeuOlBuYfC1oc9XeEqHM+dRcQrn2SRc9uK1W+r4jdROo995NyPzq4E0Hjqyqij35GkERZpxHyqCtJIQqO2I9O0Bt\\\\\\\/jVBtoixDxMcUeFdzh3VdRhKep3Ls1K6ZNRn0KScB9PRz2IDJzK17UHwk0ETffPNekWbkBx9cKMlXyIlepWXrs4T9w2JsUF95NF8Hm8CTyHExDyWrBtT0FuUawCbfCV6G8hxnUzRWE9zFeuPhsKoaVH8PRzW5OmohK1X\\\\\\\/5JLm7OdmX1ojUH5L6\\\\\\\/NOi8EuVQ0wlIE3U+4\\\\\\\/nXrsQQDeIWbUCQC9TgMs2gpiI6jQolbgYqa+GMWEqtTrFi5unHfj7gPLpFuXfKag4rBAX2etX1ZfurigPcv5nTf1GofN81TcwCrnsJi4RsNoMYzP289j4xDTgFhn0AeFXt7x6WFdWTgN17BM8FA0ydbrRqwDVw2CMFCGKNR8NVdCFIl9SNBN64kJDG6inDt8noThnEpd\\\\\\\/Yf1gB9BoKAbuU3V2HVB+U90cafumWdleu6dLP5x1fhYgHkQKI9I6iyc3JauzLFr24abk4pitIZAWsN76ekZlSx9JOf7I7HJ0saOZaVECMKG88ntfVupt+5jtA\\\\\\\/hkphpJseuKSw\\\\\\\/J88ZizBIViYfOAoDbL+xzIMr6tNEfAHA8bJjLurRIjbgpsEPV5KocsbMoXFA36qhT4+7ZVtc7WtJ6cReajuW9UAiVeW0K9p2litD8++DCqg6Cw5vQB2C8Yx2ozVBoPizoW7Pdhrlf4IYt9hX9MsLdJD8hAT5I2u22C+bp1a1y1r7+6rqkc34gvGk524SYLlNZjyJ4iOAK8prB5jLtZxdfU5HuZY8DT2UB1g81rONhtlPG8fnkQzNlqyhHwZzpo7bYmiUi2Ch0emqfhU+Hu7L90b8Cbms88KVrJMXkFd6rkWlDOuUhqp7j9AReJePMvndgfmP9XIYvbY0qm7Zm5AqN4I3M52abmQfQwiEJXiRK6d6z4v8RXHdgLjsBQePSOGOwpjmILBwJWrhS8Wxk30QuqM4YjmWCFdwzhfDRABu\\\\\\\/KOmuJvZj5Efx3wc6SGp2ieiKUL7D4wlXAQRWp3u\\\\\\\/c8FenLD29Jr5Keq6y3vzZCJzLs6SmByvf0BIn+PaNs4kPzn6NHZqFqWsEffFLVuRZT\\\\\\\/bK2b4JGBtz2sBOJ4car5bmbuW7YE1HyjsyyiaVYyEXUb1\\\\\\\/YxcDD9RLwPjY3cAwcEOyKpW6FzJSxav7qD7gOYcgYr9yJrMxxeGzr97wUG18dI4HWjmENz0\\\\\\\/T0bI4cUruNxC15KbORSsneeVTFi1Am3hXyPMM1ldQtRy0eP3pWhhAb+qTtC8pnfiFVfgxRn+ibkxZcM4dfF5xqcIbe8sNPqwuj\\\\\\\/0LUA+LDqiy0\\\\\\\/OI9ul5WflkBjty5p3yLysExyli1wSKVkM5HJe0tOCK4NxddJ0dWZDH6yrU1hzDKkcmjWXGJg4RsbO7SU3qXBFo5AHpeX94YrvCQEogNGgwyYdzYG5CXKIoGmVxE1g80zev4LUx7ADc8t34OfFanOy1\\\\\\\/yUu\\\\\\\/TKmUYG1bsXuPpXCtGBQ6sbOX80dXw7Lhn\\\\\\\/zJ9VPDBUtltvm0i\\\\\\\/doMXnVZQfcuy9n72+laB6akov1HViqyJbCwfbTfZ7UEmAu4bN35nbA+N+B7GB9n6gyzX\\\\\\\/6IPNAZJqvGVNN9V35Nqu4BlsMJt5YJaLzkPrg5+t2+bDpTVHwEyFAhHK88wjAYCR7kbdc2\\\\\\\/vgpN1NfCCwLlozCm54jtT6FmzsFlaknt9e51X5Txv7mh5TIB4JV9JfNMqtO2ecNRIift\\\\\\\/lnHbsKhqYPRv8gAClYSij2BIavEDc5Iy5gI4uYTLw34tXxolJ+izS0X4QB1HNxY07fTQcjgQoRajMnIG4CY\\\\\\\/HfRbgWt\\\\\\\/HCiH2mbohXQ8MPS1mtrOX48VUBrwwMUwVUwA3V3itFyllFZx7632v54wiTBB9Xrhg5yP+qzK1+aKQhcAptfcVLhxjl95RvIBI5BYoVYsVcm8W1BVt9GtXarqzmMzRLA2iFAlFLsKH7WhvTHN6ysBhG3iFjLCwbN9rxidv+LcDxRouqhFQF5f25b8BEeOdA72l\\\\\\\/XQWvWf2AplQUzLvf0zTpnB1G2JDWRaE7V0ROrZJTixMl3jed5doF+oFkxpt4\\\\\\\/dOsWFdWtqEXj+EMfDeqEncpzl+wq0oqMH7fjyUgVZ6zb2dgYrwOo\\\\\\\/xXCNKB\\\\\\\/UJaV1z8wIshddnNRezY+vjWSjnDvJO7RNpNA1xt2nfPDu6pbgRqfyZmJaEpb2NgoEKIX34f23vw3YzPjz+Qwhi3kjhoAq7fFb20IBZcUD2NDUZyecLnxc0eaAsztrQGz0xXY4xPowoWIwkqXFUtv4AL5c\\\\\\\/l9LEOfZPxERGvpgQ9qeq2UUtLy2cHzr4Wegfo7h2IRfKwVkEMO2D1wBtwMmgnRSjd5pN5\\\\\\\/Msnmobo3UqF1tuzi8FwjOkn8hxqhx54oE1dMKJAtazgu2VntMXf0\\\\\\\/xyT6kuhliGQp3BrN4nLImtiflbyr8xpxwUNPQRpZ+h96jWuIBATeg3V1FZbhLa9DuNGIkk36TxRinNWwE6s0uur6E\\\\\\\/aYqNLeMFi6ziYHXhrSjC5SM7vNRcooLqcrpWv9A74oGNSAeU4AGr8f\\\\\\\/bfZej9VeoKfxNeoBx7ckRBo\\\\\\\/kFmDn6QTbFaMZlmeZrp4CXRxwLPXnLmoVtk\\\\\\\/SVrJVTdlP95jSfkZRK1+gHAWTYQANKQciMmPsg1+DjEzcqVUTy9IhdOtQRfeavdETjBEDuCZyODLl+7X9pcm8e6RXudJpxPY2mEnzRGKNwLBpFhX+kK6LCXLp8B4EM89qq82PIuxdT4w+4uHy9rjIalAPWPnhsp3J3hYf5Kilk1m7nJZU5Jky\\\\\\\/Jb2Umj2rX0+HWKqU2\\\\\\\/ciSJoXUOoBt\\\\\\\/9ivoKxePEu6cd4ljD2QWvoLllR+Hfr5YOQ20IechOBV+\\\\\\\/mzqSfeyQVBSnr9uIRoUYJ9THLJPSCBwmvI1UXy3F8PeErefYxK0XkynX1U2rLJoq5mtZJkCZeyedPKcogglU6Ii4LaPnSsadrGVz8qbm3gGCLTmtACeT404d84YfZXArs2BiWrfUUImhSMLt+yIQ5zkzd1hxhNpHsynJohgJDW1mh0ilbVl4P7wi7QLr9T7S9sDcKGIysCuSbMrT0XMPeNMlRNkZT098Kb3fSPjoIjkvYpyNtLtAiVQpFv6cpEXorC9qOm1ufnUh\\\\\\\/3g\\\\\\\/GGEiA1OxE07p\\\\\\\/RKAo\\\\\\\/f2PJNBdAwVoGu0csW3QyMVjPCBJJKOarz3VmARscPbto2OHY3ZEvz0iQUsXZJCv4MsbrhH1xsbgLXy6Q47CZIUeJT3IQPILOCS1s1KSOrDC45iWvtJWWWMP7ilLkTzZJygbP5+XI\\\\\\\/tT4DbzVwyO+6tLUDNUuHMg+6fsGPN22wjnDYgf+tMW3r7OYJ36ScS4hgtqy2yFrt\\\\\\\/4GfMwV9yKWKWzJZPSI5FcR2P0eqT6Y+7sjhQYVZQO5x7ebDXIdK9OkBPZ\\\\\\\/lqkkJrGN1+DvcIyOgEN95iwJYt2+Izt9zpXX5Hw1FEuwJbhmL6CrVgOlcNyoG3wrIVwHwQ+3IAQ3mXWrDtcO9wlyi5pLLO2R7hLerpv58BZZK+cD5hqleLNYEPwjCB8WJVku7M64XlHUelIMmhmcDa5AXrtT5KFKVM2JeKMHNVylRQhKCSibgQB7thxe3rdNI\\\\\\\/DBtiLG1YL95rkl7H5N09V\\\\\\\/il5p74uJfsKXRkH0PpMpiJ28IKDIDRnJtZunyM+x71g6KuWbJLbh40z6rhQza6ftjAvkZ+38sVQQ1n5SaP9+Y0lhifbuv43zMiKiZKV39s5u8hVCFkjpB5NEueZJN1Bdy08BXj0EDKVnlQfFbHud7cbsRg6QimNX8PoelYSB8OkNZCYYcQ5US\\\\\\\/Ky12aAJC\\\\\\\/MUeyuxIzDSI\\\\\\\/o2vSSRHNrG64daVmMybK2owehd8elQQPXv0lhKUF1kbNx+qneP44SqWhMlENDnzp38iD04\\\\\\\/P6arm6vC9E5X1ZBrGQ\\\\\\\/3VOMBvRTM\\\\\\\/W2GvY1TpLNUQ0EAGp1W0Nw\\\\\\\/dHl7aUNSWWZ9yrNr9ccXHq5eMjxopVt8GU\\\\\\\/KIyPUqZUvfqRAjfXeVmfD6aWS93ZQdjFuS\\\\\\\/F6YqRx4iYAKkHjIMiPEB6AfEMcdz8KSu6bNXFlNSP4rDWboO3WBnea3lNMUljcBrwawXq37GGsJKJFVz6xhXB9fphk6CpBr9idgU7RHzfknKXI+rHfLzGhOk2rUuQDW6aFWRY1nosGuKkNT7xK8nFM2lc8gX9iTqRRM6TEDHbIFxoBLfOAfqF5DXz6hvxFsQ5GHT7xrNce6CvG6r4kA8LXInch8UrREZlLd76jYy2w+Dl3zqcfrK6XiF2vTx+sT\\\\\\\/LFCZpzGM1DumlB6mBITuRIOemoMob0H2ubIAtCFNRX\\\\\\\/lpXlZP0kNAx5Dlm5lYWrH1I9N5L7\\\\\\\/XskcgZ2AmI7I9pPBvfzhKIHNUyeNFysXQJHzoCcduQM\\\\\\\/A1x+mInHexwiAMUI82pvJJBV1TlElBHz7pG1dFDBOOHfosI0H7T4ojP0uoXQI1u9Un176zwEEW\\\\\\\/tt5EA7WVxm3PjZQD3UakxmmAIJGClUfckeAAGdsvGXb8sJIn+kP8nm4MRRCVGMd9AuZQjZps\\\\\\\/gTAwHW9z+Qscw96wwNv80IDCiH2oTvuCqMHxpUyMuigWQQy9\\\\\\\/EtnRs4GQcH+eQyKjJoJMWYRC24hZupQNVsmK2be\\\\\\\/meRiyXBeKfXqaaEfACKb7yhnYKe9JMdfxaHAbhe6TSnT7QlEFaCUAjZh9Z58Ap+chqYBbXAxKIsjiuOxpbVRPJUA\\\\\\\/b\\\\\\\/GhFCuyiCGEZB3sOLGl\\\\\\\/\\\\\\\/ovVTfHnAWCPDR2M5SU7v\\\\\\\/s\\\\\\\/7wV10U1SLCDvEr657bw9YvdyTYeSzqp4IfA4ELirpNoCJVpQdBolcGsOoQusUIRmWsRh7IIPldFZHCwt3y4lOFddyD3nJxTdKCCWU4leI94zwdHnS8NjWkWxvC25zNX6vBf0ZXLqRfrQqIs7SJoWcpa5IQHw6O9TyvnvAQX4PaLIM0vBXxRNdI3raY52adVVZb3Y6xahIbVJo9SGlrSy4MaWbFbl4pUyO1Pc1HHXyztxW0KsDl5AuAvCfX0R7xu12j58prsd8Ku9OrZ6+BdtWtO7e1LKeTcBciWtrQYcy3Qw5UqYHwdxHtRMEtl7VmhdtWN8kZRb1Qqu3lr9rRIjYko9QV5nxFHqm8QXz1sjOpSd7RRUYcYw4qnROVdBLUxGL\\\\\\\/e+BFnpSwxM8z2wmcnJf65cdsY8bx5+XXcXWm0jtuabxYP6R4MLZE4RaW4NcUphbrSdtTrE5SRX9lR9RVyhF0j+8u2xD6Bie4YoBrt+GD9sHgQ2fywweCVpBqnIBi6fMSgrEHP2SutZKy3ByZKcR5ex7WqfBYVZLrNxfBRIFnR3xcYk8Ie12QNNQljr7YmyecLw7jdP6IcwKXWobAFBgLXlUIKa8x9cvG1lxiwW06NOZ9SbAOrcQJLIBeCmJ7+Avu8ek8NJb3t+QkXMoKaeajiqBplEsv18dMQ\\\\\\\/uxGspb2sfC4TyPp8H6RP+tpicnmFaDW2Ahhrsohg9Qrry3SpkcI4XAyeP\\\\\\\/f7ekyzLYzf5eR8O4QREcPZquYTBDBs+F0xz6y\\\\\\\/bT75oXUwBfJtC01euHH9nFVPapRAkGQxOMvmNVjoT1nr4\\\\\\\/zqYgz\\\\\\\/Nh\\\\\\\/Tsaz8LzhMa38TYu3ThSXSmfzchDDhVlTRzpC3rk\\\\\\\/LdYdx4Z3OW7wnXrQo1hJsYbAt24\\\\\\\/\\\\\\\/FyNDf8gIAgqEsRv5waoE\\\\\\\/nf3rtlAmuOOytLPVG04alFZg\\\\\\\/Hhetx4zMdXeXkSh9IO05gI44YeArE3IT5\\\\\\\/9hzZquynwJ0\\\\\\\/teGVlQ60Tfw8+kZ07GPUe1uHM3xoRXj5ovHq3O\\\\\\\/iIzZyMkFNfM7CFjTHsbAApnO2JnHDWQeADDXfKE\\\\\\\/XhW6fWDkwBV1v5pbeUVQwa+a7iIRA47i9ZKJt5gL8UR0X05AkSxbyo4WA2r3FqRnTpAmKP9ck+PItR281+e7O41kWObcg8n6dOLSi9Bam\\\\\\\/Raa9uOkJBXAd5b8ieJpuIPuc+UcAVKJANPz9ms8yUEBZtRC\\\\\\\/IHjdd2t8FgK+tX3irUHxycK7WLcrSGm2tOyUhloXzJb+3MuBgZehbsChJUd3jSdciMLeCELaOtIaYcJM+FIO9kDul1wFIuZiLgW1ajEgL0mzq+8y\\\\\\\/4NTmZLGi6i1aXMjCmjA2fjh0IUbgwT3Gh9B8v7sQ7kttj\\\\\\\/+0a3QYyZyCl2NH1dppvYh1pls9Z7pNjoPa8w0V6H7rVEH0H7pMDGxOLat2UVOtky+xnA6mdbysu28gjT9290Tn+nnJFosVAAc+wUVvOJZp8oBNjUZ+vJkXxjnEH+YyqE9SjDNSJKDB3pXVp6i2RyUD1H1PvftvtvzyYg8t8YJWgyPV9gcPTuCOpR0gZ0EMIFbhEwFZRwH2Bh62ayWUArF34SfErTROCz3Tqkh69oLgRur2u2kNnB5\\\\\\\/JFehydGXOaJS5KEuBGHRGZmX5evh60NSFBgXqzdc4EdiP3n3fhaZ6TEW2YupZWuC1iuC9c9fdy+OJQp4e7R5F98JV\\\\\\\/c+QCRL4BMlxA6fSPkMFHQ6CFLna2A2i5i289FnWkL22TOCja6WT3sl+UOB55ivfuiJt\\\\\\\/HWNIZMz5+fo6HrwjDdxpPQ7JpGNJdGI4YmbK+mAXgKTszum6pcwOeYYkW\\\\\\\/6s9sqZ3omI+t1yBT45iw63hBlOUoMVUGaakRSBWfCpIAwD8ESH4vwgC10ciWFJ5RhAHVer3fH8ec6rIcKPCFP47i7BOwmVtY0let+igGxAkTIz7folPWBsEZUirjYkY6iF8s8ESJlzqH1qjKZ3lAYodogwYvL2jAd+axl1OicNeHLwbWaH7zEazeR3ejh41foGOHHuJA0hgfi5PGBD9bT4ch32BcqvJHBwqpDOir\\\\\\\/macAku5VmqH09QqEfzOjf0EaASw1ixB+Z+Uc+uhQFmePJWNaCvAA0gTxV0Fra5L+ilQisvjVYBpjDXVkK1Vd2fBC+qFA0YNYfnBMB0XMxCm4WbpWbVUYRPJOhUz\\\\\\\/AFl+NT9Iw6APiB3syfnsg6Y4BBDF84+BNY7eyn4NdcFBOtQB1VKevMUkTdppbQGfKWsPIx2K+ecLoepagmUTBBhLwccnfCpex8+Us8rq4Q\\\\\\\/xB3kM2SwaN\\\\\\\/mn8rAOUo+1YnUdclKO2Q88YyKYXsHaFU\\\\\\\/cbGKcI9aOZlVEFNBUJsPUmgPpsfFfmi47wQrjbp8DP5Z\\\\\\\/xP1aQ9enA5+S2C4V3jVJoGfWUvC93McDETpYmk1FZKcxnKHIYPRYEGV3N\\\\\\\/FoDCNJiNXjlIqEav7TQPuEGaXByZYDvWyFaAAvsRYESzGuYjQnhX0sJn\\\\\\\/M8C7EvhJ0SCr\\\\\\\/pZSqene543MCBmb2dvPm3BJS2f11Eusd336otinJVodZo5b8HELxaQCaDH88yMq5u9OkNrt93OHUbZ+EChqtgcVcOAUS4LvhAzLpdguDuOQ\\\\\\\/BwANSnatDdgArUBOZp5oYvuumNMJKK76E9\\\\\\\/HurNMhmWOzYT7\\\\\\\/l3YU2fZbo6stGpL7cx+HTKkJWz0uwlFHxakHbwIN6fqeWmiCexgryPBK3dz8Shi44iGLKA\\\\\\\/b1T664ezU+acV8PH0fV\\\\\\\/sRvrJglANjFzXwjjva+\\\\\\\/KSK8q\\\\\\\/3fgO8ZvtZ9Dyp9ry07TtMnrFMbNuaMaCAFz1gvmUjpJFaxeoRk8Kbcxmm\\\\\\\/7Y1VQtJ\\\\\\\/o2ChS9bt8R4gZf\\\\\\\/O+b86365m+VNx4qdd5BjgZr5oY8xt4QFIHaRGnD9RVfMbR7H\\\\\\\/0t52LZmNl\\\\\\\/P6W2Iv\\\\\\\/efkGzq1O96i2uC5WxBlEYUR9\\\\\\\/rB2h1F+O3OtgY9BF7dIN\\\\\\\/fs\\\\\\\/7djN\\\\\\\/4XhY8\\\\\\\/q4O8VWzQ9X6gKEDCQ0E0ZI\\\\\\\/sH1NpQvNJAUIpQ6FcwKBmcXhIx8AbTj7x6O86\\\\\\\/P4JnjYikdlyj1cRG1uXDOWGqxfeS8bST9QiJ8PlqfFBhMGJYI9K6+TwIsZDgr8cbwTlx0KulVscEMB6SpRFXcCyopAknTMplcfgbz0UAxXBVEOvraqOOBEomW98cbmldRjE0jXM8ttxKjyawZjiLikIvWJyp0WFIuwwyZAApjjiTqVE9rcE6BW7PlO9BqXuFkDEa3+BZP2vhf3l41xb1UZvoIkU0e5BAWqTEx2VXAjc3eaiDbmXaAXYAy0LCrxzo211jx+FRDlNCDz5gj5zzs1YLK6LWjLZtFB0bfKIy\\\\\\\/lBpao37F\\\\\\\/2MiL7vdt2D2xIgf4rL2yj7fRpchBytF3vSlNSkRSK473rAbAXeee70BqgOQ4YfcHatTf8o\\\\\\\/W6\\\\\\\/5cxIx65LIMf7bWKZktYlKfwC1kf8nMWdfZIx4fotXSeDSGyyaFNgSOFV5i16IoUXltQg1T\\\\\\\/JXVcg7ucu5fotbD0PzQdl27pcvxrUspYdCHFLMQuMpz0n3Sm6rbhwZPZNpO08pqbJNvGtbR4IhirchQAqlVNtZhbKFqECGPLPKYAa0t1Z0eqEZ+JlEo+Cc9BtLMEF\\\\\\\/3mnAnR\\\\\\\/PG0DkqnIYYmUF5kax\\\\\\\/AQ0CXCFDDzp1iigR+6jpoPpjeieMyuEQFuu9fFBk5kwfUQln7blsHeJErCCU4Sj4tsSqxt18uL9hhIRX0w9O0gWD0gPL4aVsgl85ZjIATi3ohjluWqqZn57KtDR1V\\\\\\\/ZATWITzIpSHillaGUbvk44vyYopDUwdCvyBHHyUjMkuJdTef20z5L9OW2ldFOtPA9mVOjugyv7FfT\\\\\\\/yHMNVWA43Brc2aPhDL4fXlOBB2doOxzlK8kgWSgtOWAilR4yEAOYXqHYeZJsbj+0hSUAwGPhMdlQCOQ6Uvge+kBNXxw14XGkiWlPp384mRmTxYefSFFYWeZuBzBpezzcFdNuEOBz7pzPQOtf9MVZand9d6lewFldDKzNFNj+5x6jpab64EqVFF7uelkSk4mWwQfZ8WYztQRGD5Iue+jQjnaLqqpNIsheMJBLrrdR2q\\\\\\\/OalQ3+g5A5b2BWXsdk0FYiTd8NPsKDtAhZcCwTKWZXzrvlQfhha9C\\\\\\\/gVq9LBHX8wEJBfoNITpvowdZNvYFZAklyhZjhP8igLVDOgD\\\\\\\/gNtqlESvo1mYP96sgBRgDfBAcxKSyxiQRzSC58xfDjra0c40C1L8S1JtPsgnbuOMB0aBxHN8RcPIN\\\\\\\/1a+0C6YI8Ec2ZfXgXx8itn+tEpwBPwfsUa1vNDHCBudW6B8Jpy5jz+Fv6Jl6C6qV7tAIV8fcyIHJP32Fv3DJY5Kin\\\\\\\/NX++2PMA1g5YzBLBtLcL7a0TuTX1ngTvdD32Z2VvIIBiNH4uOLgYV5QxD49qTyRAfiY+McJBxCeU4VXhBmK+MoMz6Kx1t1b3HmiLay98fPeLp37z3G0RuzM2yTt1lglF9L+NkKrQTYGtNGua\\\\\\\/O+6AAWnCn9Bl70bwmCFC3b3aaZ36ILwv9Dnw+7+ocuM0rRa+zeJVLvw4Fv20m\\\\\\\/osdd339NfBiUWbq\\\\\\\/FdSc+N2DWHqECgvF7KXlks81WB4Z1slPPpdZ9FD81g2PtgLMIlf22EA6m30NfQr31wVYqUZGNWpVtuOAeivwCmIyUGS2V5MUa8UG1AEjyWZQtzbdGUPsiwb4j5ivwRKyW8CnBbFeOqHMJIItQeT9hQlERWD\\\\\\\/jlCx7DmKrtCz+nS9311dzHuSWd1DdYRc3SPe2eySZBn6pJ4nap0zLx9LlZ\\\\\\\/MMZeKENa0qzTqiZs4zwXgYSkCBYa6kmfwbQ25zVyRekwBnNtAXKiUNczoKwZFJMhDPHjR5u3k6zPgIr3DiIrSqAEFTkdEdDe2GgHV1KTFJuTjbtOx7JmNu85EiqQL5AWVbuvx+Nqvkg+QanERPlVZu9KQSvlOcK6cKclfjUDxlhLYyFy4wICqrR\\\\\\\/s7ouQ5QnojmS\\\\\\\/EPTOcVwKUo+2CXZRvGl8sNz2mt1PlibKkyogDvA8xjgLEyBOJna6WYKGwyp8Qjoys5tH+oxXma6xQYp73KX4CngSGwRNBBn39bA\\\\\\\/SHztDeVY\\\\\\\/6VdYAdh+1It\\\\\\\/RhfqRA+aZTnKoNAV81fjSebrGF85Cfx8pb6DI4VXAwcXZ5tuaJPFjjGNaGeNJx0Qle3sFgGtm9SDjYoLzl7bG9mRZV4vEhbp\\\\\\\/3DenWm9b7oAivmwumgQbPKJJyAx8baK\\\\\\\/+466iuHb0LILiYNySRTAv9oVTMS4Buwi6Bst2IFNuRJEtGjmzD2QiUBqhkdoE+0F\\\\\\\/\\\\\\\/5Y5uoZMxtSd3\\\\\\\/EkAA1JhcmaLfLkzb0rT7zIYqzsRAz\\\\\\\/Ud35fr1XBB2pBZrqNBuRKzbD\\\\\\\/eoD0HFHpfOaT92zavF58wYAodWYQF2fXR\\\\\\\/yDuZbmG+kKysxWU+efx4ZKyQPBLF7QcvqiI9+1zh1PCW+TSfrBgv5rthKd8bF0tJqcEU+j67BhNcG14Vj872lY1IiQEQJh743jlT2lFc3+RuPCCYYDtMb92VTJkirCSf\\\\\\\/SZmPzhaE\\\\\\\/usP+61OdIRit5X+u7GItm8nzh0Kv0R4q\\\\\\\/879h1wAKv8QP0473T4KE5mZSPeFKG17Mw1\\\\\\\/DzNwG1QNhf7qNvwhPz+kK\\\\\\\/qNN98AT\\\\\\\/RxweEWL8Pve56zmKSI6kh1+gdUVDXunekknHjTJIr6OxSEFFLBIpI0+\\\\\\\/Sa5pWozpzafGIyvR0fe8NmkPjiGaVyDGiGU\\\\\\\/xbbId1iInKXV1PJKkLXCkqYXnNY2ZjaZ0uhKq2meQcCVwhLc3COSGnWw8rUrzNq\\\\\\\/5BZFjbY3cqL4EuL\\\\\\\/NL\\\\\\\/ttX5hTwejrxSJz6qBlg2CYzsQf71sW4dmQMT\\\\\\\/Gpjs1dCvPNdCxzxf2LgRR02MSRUKzi\\\\\\\/NgeULxzmcLinGWJM6bRJrT4+vvVZt\\\\\\\/+sxYGMbCJMozjr4K6lv+O4qgBbtY1Nv\\\\\\\/PTWV+SrNmEF2TfUJcWiJv6jCwvocZbZD3ie\\\\\\\/7pnQghEIISRXgyzC8mch77DSssNqOmDayC0gCj\\\\\\\/Pk7wMzoAxeys2ZSuSzzJXkTXptwRfD8vjhLOMOCqDTbYsX4cw\\\\\\\/ImpW58+E86B8OTx1osdW1bNrqCdwWNKrAj8B9ipheyGAReQX7KsjqBBSdl6KSjmEw0g4E3Ew1tYpOuSc+7d0bkV5+flmUIzkupGPTmh1c8c2MrzbF4Vcb6Fs+V6l54r8onYZpiPRiclXhco7S\\\\\\\/VR\\\\\\\/AEpE19oYHweGBx+7t39eHr5QI13SYj0OOUICvGTG+FRPq5Huj94HMv9OAwQI1XLz+OwtAAFzRuQR0lOFc+V+QUb83rASWRByeYZS19fIumdoXrGcn6ul6VsqHy7vmZY9+yu4CmHI+7g8nhj6OPEDFmNBlEwj9xwoiPacGHHKrry\\\\\\\/Wc2RCq9cwWL0ze2+G80xwUb\\\\\\\/X962V3vWxHgZM\\\\\\\/8pSpIPr8ke9Ao++F312h8066qzBeni8ot6GHPke1xY4khsDJI10uoWx1w+g4rhyJfEp9wJNgIud32vwRzooL6wT5KNY49emASXuYYPzHBf7xKdZqtPw1JwSc2dCi7ZiSHfuNFCcWUj7MBpkYB478tjClLjRPp2rO9n2q7YYKGp8F3NF7poPVrTs3wLTZLnx+ViF74cAVe0zWlRsrYV9kLxcyhBxTsYMJbd8pIDVr3wxLRFDtCP00nZetVRtD94mPBi+Z6mOmX8s0jKZ1Q5qduSGR37oUSihtEhDWIi+uaPyZgy5elrLrV4pu+ll\\\\\\\/wq\\\\\\\/GbbH+1fshC7VY9gbYoFmoASa1uSJvYKi6m4eY7jafZcNPQLbPOzA3PlXkDFJvsQwMt2M\\\\\\\/SNJHwClIAwA1D9I5otXgMwFFNI0feXOKvIXP5dk8MqNVD89f7YrHy\\\\\\\/vW1P3nny9MjCp1v5dnFrWz1vKTtMCvJSpnDawQdTwwSkF4NoN98MDDlrovR\\\\\\\/GknDjLiaIoiJjpwPPx\\\\\\\/qkZNZhrPfundnbSn+G3QrRD9TUy6AsHlrPwIwR17KthXtkSapw5elCo0aNKRlfrgEsR0nCu3MuPrxTFznSqOZi6JyHnh55onnzCPpck0DnblEIHNh34DIxBjKU6bhNex4j7dbsMPDzZQCP3V7fG7tboA2J3nMEpz47IeH+jt\\\\\\\/z0jWezS5Aamurp\\\\\\\/HZXdW+0+yxv4MDqjL\\\\\\\/at6wtPnJ3MShCIads22tb08\\\\\\\/HGCgV\\\\\\\/\\\\\\\/TFXx3YCiOSkBjrPWc57\\\\\\\/S6WHXSb4urt+Nm4BkL7\\\\\\\/XLc02PuAB2jXX2rPH7PSxdgikpxlejrueB+tgwtAm7ZDuHaIf3pHGyqI+QO+EAdMK+6ohDAsFNGemsbqORHzt2Tm9zJj3PPISAGnIN3dF9LwzmHZGGEul9++\\\\\\\/H5xmaJsKyfG\\\\\\\/zHSoLpoFQuYZHVhzBFZMl5BWd+vb+Bdj8mUK9K6PKDYgLcM2D4Oh7zHCJMm0ZJ70Pee+udHPci3T6cvNRujh9KCAowZkc2s\\\\\\\/nkJsOkdi+xFI6YbcaxlTv4ov6RFsiAaEG\\\\\\\/vJDJpLZTVxAvOaLIeXFqmAxApP3t0CCXfDAmOkm2ANTExj7C6UnhOJk5nanCA29XvXzXtuHIhSdjus83acMtszt67sSHnU2\\\\\\\/jDMot9RgxCyl45suomQCI40YHW\\\\\\\/p4+6\\\\\\\/wRNhm7vefR5iw86eeR+pkzE5p9u\\\\\\\/0X5ZYS4y76xRLwnSFgYLw9btcLkNLoL8S\\\\\\\/zgIGw7NSdFUKBc+gf3Upsrt\\\\\\\/07O1zLiVk6JZLE3KBrjMnPNGkEuMVrv\\\\\\\/1zulMvtCnNT8tYn4afQ5YnRymauhbSr2\\\\\\\/x+lGhywLEOYg9R25pCTeAnqsJIsfINc5IXb+nH2kD3i7uKS\\\\\\\/LNI9O\\\\\\\/ReB0F9IpNNcrEnOlRik04n7mzgwOGoC3bXQrh0\\\\\\\/fDRI5V81viyblo6o\\\\\\\/C9YJ9FG\\\\\\\/ezP0\\\\\\\/nIxbUkBfmPQA08\\\\\\\/diTuL0URKBDIxvOaRnrie3+2F7GGHEKunUELc0flZvGddjs6yt\\\\\\\/NA45SN9wb0w4gtzSp+X6g9Othbywk+R5eiFO6DWQQxPZmBb6TZydQRMvpNaaYAddpUoQMrZug2wYxpkXwMoos4OdhAjoYAYiypNQtIxmQoWpUyLDrSSQQbeYfwvlNmudPMNCBvBiF5GxFdFSi9idH5o1Ex\\\\\\\/Z5Xu9LOoEJdoQVP35h3MKQ82ceIevXDJEDRTpp7bkF98DfqurA1SplB1zg6mif0DXSPNFh0O+ec7xNC0x5kLptHLLFJIqTR2PDklakGO4J8TNucIKAwnluiFpEO3QGBwx16Es3UfDGX1DjgWG9OInlLOVyoJgQn0oiiFDFCl8WO0twdPalj58YmlbGtCV0FVpeW9G0t5\\\\\\\/NurxiDAx9msaLGtWDj8Qr9k0Fj88jZ9+iBKlO7Y6VvoDxzHg8aCDZsQF\\\\\\\/ZngCZSG4XaHvwW6OmkYzvJAHKa0VOJvqy3XFiV9rTgPZ1Tga1zxzKBUv9W0loJB3nSZ7bCU3kpit\\\\\\\/ImlFtUfIzHHGz6Lb8ZlvfbUP+V1Ik7RoIBtlD2E2MnChMF0eYZG76p9yzXL5PvUdCYY52u6uAJBQNJphf8uX2bMRNPGZYDZEn5DXW+GLa4chL8QWfL6lQOSnwrAiGdCxDdOJjj9GHJ5IvXeEbAnPUbQ37PNa0XjfFVVfbnY9nPKYp+AJ78SIzeA1TiWBAR+6fvJSfhv4Qf6ZC0Pz+tMX3h\\\\\\\/7CxfYg7dVpYiIHzuIu1XpfMKNgcuRO2yRIUFQG7aOekf7vQ35DIhkrahG\\\\\\\/L7hBFWC2Lj4H4Zhe7XZwbLIB6rYytzg\\\\\\\/Kwe16J20irIX0SLIuftvOzyp\\\\\\\/PZd7eG9fUABOVWQYIVMS5tPUsnWztayHsNrK4hbTFyaG6fz3thTCCOtqHN4aLK3CkUvykKgmBoyTvjNso+3aHN50hwOc4Io4XesedmmlDi0jJz3BUw5iqh3ZYMWnxzoiHVBRnhLpfpJCB+pGUYCZfkqWQNnpW6a+JCempanbfB9xJcKk6Dwk0kC\\\\\\\/tKMXJu4Dw7xqq9rLcLZaSTz3+60Ekpk4XqUYLD6uxSlhDJZKb0uAa5YGtcP79d26MP9V2Z1v0d7Plad\\\\\\\/SxlkPanYR\\\\\\\/oEsAqPAP2zwZeyHQrUHjxxhLyJxwU1U4qAqxTbFAAXTo8pnywrXqwnqlbcOBlkEtgXcr2OTmOdfhDY1czVsNPUf0U6pCgUS06\\\\\\\/es5o4rx6YZ7vc+GolMFNuX0bYMbCelswUDPCMTzbR9+DmWwWQlhY3DzSNkZCOAkX0MsF0wIJg\\\\\\\/PxYj7oF5wgCyY\\\\\\\/3grjZ+CAVbcgMHQbbjI3wkGS+ywQA4wvLzMswFqilZCDLC1SC9JNFf2X6XlEUTVBB4bvFQFfNOVzhdFiJZl8KUbqK9qCv436V3proEkUnGgnK2XsvRAbDZY\\\\\\\/QURHUgFJUwk6mSU3CJtnvmungz5y+RQzIY2QfEHfXN\\\\\\\/vnN9Al\\\\\\\/YAc\\\\\\\/ZKbJIJ8FZIT0lFoiky+20McTvm84fisiU5hvpXWOn+j4cRgB8ao8ktFpzYbIU7TE7hbL8xBPk5zl7UOBYUPu2u6mXLUI3m2yejjjovdH7exZ7+EySYYBy2WdjY2xab1+xp3aYX4ciTQAcc0c\\\\\\\/4onxAKc32lE5yRmrmtB0YF4MusYCVYPa+iYulGu4EDgQvr7Lp9ot+vGn7kqcbUAPqWZm5VoMx\\\\\\\/4M3uur8tY5VKeiXVmN3cbVtBAOjFKhR\\\\\\\/32HSI1x6vVNJrSax72CQsx6qaJ2nzezyrY2GaREgB91QoJu2u5duQbI67Tpsg8NHVzVaOWL06y3wuL2BPUbUbXJeWdC9KufHZGaM95Lb61XKp0wQGs468Ug65QjTHTaF5ogZOdkRSHrldoZ106CatclgRuDGqYwqeq8NoaYTTjhMTCXMgJwXoY59nyNjLNV1wwPAzAS3OpiUmIi3bRywEP9RtLlSITFFfryAEMru0REY8yZj4g5cuqOV1VKBmAxOUaooulMqeFnv6Hp2YxJnWwhSQ9SkwbEnEMGb\\\\\\\/\\\\\\\/uZ4+8Mn\\\\\\\/x0Fe2qYVPHXD9JL30Asb9KoG5e\\\\\\\/SqmE+TBCDmdVHdIUmKuNRykJ+BdT\\\\\\\/zOJQvHnVCjU6q2um8JFdbvsZDpZKdZG9V4jitM7qqy22ayySrSA6J6aehKtV+3mKPtDVP2SLuhN5nK5B4an6EqrxLQr\\\\\\\/wp5apjY17qLVIJWKel5kwozvf1\\\\\\\/+G78fNp\\\\\\\/Q5ayTsda+O2YSWUe\\\\\\\/3pWlybJGvLaRovi+Gs\\\\\\\/CLor8NtT6ksctP2AgAiNrxJ6e0lzZ4ms5R40sO46UjJ5pe4LFSTC4KLpSc6Oo1MYVCx1vZSxeT0wQunq+QSn0Psu\\\\\\\/dKWSrbWNsJexG5xodIW4j+lsZCrngFmkEwzl+xZOvwCxhuK4x8pRIfZqgXCc\\\\\\\/GtqeDjlPPMdBuPok9zwykri7oCX0bpQaePntzQeH1PXk1lDbx0W59cvaN3nc8R746Qnk2pinUDTpGks5xaN\\\\\\\/pI4WR0piQ6GwPDQ\\\\\\\/a39CwuShvXiV6dOCXcSXQfDXVTNaaPxa3NYlJkw781kGUF7baFNDs5wM9i2SGWq7XgVrAyxLI\\\\\\\/vA3rwaLNxX3t3waybLjkiAsNZIXRiazu\\\\\\\/4686MzaHHolYTxd5xe\\\\\\\/7FXzFhYzEj99Q3sB\\\\\\\/M\\\\\\\/S2SXWo+wQo8Vr\\\\\\\/dq3NnmJ7MPElsnQOxv3Eq1LRDXEi9qwERmTEvJsSkKZNY2C2SrspKLYmBlOB8j74xIqMehds2e0Pi7jdgLnA6v5\\\\\\\/sp6phssc2BKIUR3WnMLbzMA3vIpyrxvBb4OZX0rTPs5RndDbWvlgNgTAUyhHL4is2NyxGB9IRtHaHNjLnkfhyxSu0cxO\\\\\\\/md2lpPYJ+k1jD4mmI9ouTaBhd6llYR9mokG27Cq2uhhlKMRxAVPb82\\\\\\\/KVLtXbDQz6N5ELjrSCioz5LsIuRxnORlQH\\\\\\\/o6AlsHMp4u28PKuBy+HmR1Y1evHPcM4C6+r6M\\\\\\\/7541PnzhuJKuML9XFB\\\\\\\/isQcrkhuRnGwfScw9tBrdbKyPo61x7+DN+ZJlK6JDvE0FZ\\\\\\\/EVROjHN+Y1Afr7SF0WnAi6BNuiApt4SUv3WU2MVdVynq125qTh9vePdi\\\\\\\/MifzEokrE0VHZ3DMZlvUjZqWOMu8qG6A2BfnUehsZxXmwz1j6JvjtlQtUhz4erxDXt+NyMtB\\\\\\\/0o9JBAkON8P2Yqk2pOrgIzpA1bZUoLYLhj6r7VF7MOKKL48nddRq\\\\\\\/dzAvqSTR2rz5TRd\\\\\\\/ftBe5y\\\\\\\/vJ+mv3f9mRlgzKZpLIDSx76Ozh\\\\\\\/WzQrStNxbRFQwJpDCHQ+mENxWIOD5dIHWvX7\\\\\\\/4bArLHcJDFfGdz6\\\\\\\/rLbAOgF2E\\\\\\\/\\\\\\\/S\\\\\\\/BHcsJ4DjFSc6wutrT36E5b8u1S87CVs2kBh7S4JqjGnNYF4\\\\\\\/4KmVheKCxbS3pS4Vt6qIjX6q25fhdjqmtZrs1Rm4D8caglCQF8uGPXQJMEQBEW\\\\\\\/nZR0eu+EHcN+pgbU4oh00WAP3wwf0H6xXvyxu\\\\\\\/\\\\\\\/+oygHIGNQxBHorkjGhtKdK4xPhnv+\\\\\\\/VciT\\\\\\\/TAaahVT908pUVtbDUZL0LpUqeV4cLmvmg2S\\\\\\\/BPHVLdKaRcRoOpYjvmvKP2bkqwqaEn+hDsLhU\\\\\\\/5lSX5mLaqCAYLnaZoFRgVLO78NZu4bB7RVHiEnNWu8eHQMgu+D4ctrJiDFIgY1Ll8eT+adgVlQEzTwDFlhRrrwWJILsqE\\\\\\\/IGsfIHs3073AbWs1lm8Julz2ZD8qj1\\\\\\\/YKD9BWnU\\\\\\\/qv9DU5xMXn4aQR2j54uNcMPauA8qG4wPRzcW9UT7VA9NL03OgJYEUGDZt1jlifD3AQ0g25t5XSrX\\\\\\\/VtbtzSkK1C77vERrcSWN8BeVItYCQ0dATlQjF6PskwOU\\\\\\\/ZzOk4DToVWZk3ZrAXELlyZDq6uEb9HDghwYbYv9zdBlafI+73g+R8O1VOEW9mK5Iue2jxTU5QDWsjm9BO7dcBMWt0Uso1y7mMih9pXAQSJlTMigB3LD1dtVC7MV4Tl74isJRHhZXuIwlW2mvfRXTcn+rvDcWf6g3X8Eq\\\\\\\/2imY68aGZYA\\\\\\\/bRSD5IdFf4n\\\\\\\/Tdqyi6UxuU4Tm8QJPTPY7MxnfjNTvft3Kl3iecE47MWSGK9YjxUvKkXZ9qh2IdyOUCX3nce9OoFKszWSRLCP+FLVPAkj6o6L0Dc0lHlgdVFa0Q1W96poXwB30W91b9nd2KxTIBxmgv6P+o41Pch+j5Uf0ltiQ5SSvIEfkPSs2yce0FSpyeMF0HfDdaBbdIwdYyEWuNw3wZUztWamlZM6K6yqxPSp8BC6pCxIVJWghWa357NMWuIOZm0ALFLOmugDJJdTzxXNYhyncV7Ye4qVNoLv0qw3DoAYlw2yTWYMQFCKbjSo1WuBuu7p892BcPv7h9avYn6w4kfUCdzXweG6Wnawqt0nFWKYlGEFnBYrW1BP4OgwmaA9mGnE1ZoZNmdPMPRI59UDN\\\\\\\/ISlZ\\\\\\\/gNSj5hv3xhlozty4E1CCz7HClSjce6mWAlFFq6XiCYeXbyTRiapXCBiNZzsXwFjZASdDWlmDbYMOWnF5T25J+GpiYWOQ4Sk6NKY5hhaTfgjKAeumTC4bI8zhbsE8\\\\\\\/4Wybp5Jww+cP7lj7vzosysfg\\\\\\\/4FS38ZLfM9r23qdr0OQXIxHi+iqBps29t0E99sKpvC4Q4bKa\\\\\\\/ZxUY8San0\\\\\\\/gl0OtUCCC8jsfXMP8KhSRKA\\\\\\\/U1s5Y2dprepKRpJJi9tTBPrwMyk8JDmUvx9f5b6pvuTb2SSg6eeIdneN1qwR2YAVnmXJDWxfTZBnAb1Azda\\\\\\\/QstxUIRR0C4+myAUHeli9z09u7RU+A\\\\\\\/BEMV0YCoZp04fKWqSPJJ3\\\\\\\/PgQLpxp823+4hDSLZ\\\u0022,\\\u0022iv\\\u0022:\\\u0022qUK38qOohoDfz9b5oXlhqg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d391b9ca81e6548c1b6bf36652714a8be561725772136c3c69d613554e03e28e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022txFdceEu6nFd0pNHpO12L0oUwvp757MF+PBd1Ld+CjvAER4nHUeaaMit9ptt4dMhu541izTfByltdN5uCCbNZRKiz4rKE8udzEqEaqz9L9ukwn74BrsF2AYffG6Hm4+4qs631T4bj9w+Z\\\\\\\/XTXs0H7Lx5Fpo+Jc8+Aw+Uo\\\\\\\/m\\\\\\\/hMLoyw8\\\\\\\/MkFRBsrWko+sXG8ZfcoG3nOw8oJEurXOrUVOafl58b8WQWw5ySg6uc1ZpHTX9oWZ5DNkHMF1TabABcBJr9EQEtHPcOtPjsUXSyuL1eCXG8LpqIdlL7fyOdSBKp8ZSV3V9pcU8a0UcL\\\\\\\/dbBGUebvfQfoPvNiWSM9Q8HGH1fd2Usmc739779efI7y7ZeBmWqrvQi8EmlNfSzcJPUlNQI81nxr4Ftj5ve1g0niRoKB2wPeVHdRTILElylkh\\\\\\\/LZuEMkHlWH2ojftehfM8CtINAJx0dKhjhYL9cM4L1rFeHs+mlopIhU41Ka20EgF7E2uNEAvRbdW6rGnm8CrbbLD4O766udMj\\\\\\\/fcOnIAiQqdvXJk2yvAVeYCRCphJCUPBfHWUVTKARvM40Smi+iV+4v5cbaqu\\\\\\\/IR7sk0U661xJUXM7xz16nGhGC20Z50p4dgsvHrotx\\\\\\\/IMN6fBY36vI6BmVI8eVsK1cN0OWIHb0y3ZCAOASHFHSgW4GS9QQmHL78s+CdaSBltscEeFyPvqmH1wmYWyOKSiGVYjkZN5snY3oR3+VYeRcXntS5LJRIL9AXvtOQ26XEE24iLl0XAbZrQkubIL3\\\\\\\/BKzXdlIgP0xRJ24HU5hJfC0x5GTjhlZGekJeMVSbVS+TVDF5FGk83v0Zc1m\\\\\\\/gFdRFX71gkt1rUkWChuYfy+uGrEZMLM5eVPjTdbvgWABALCzHZMIV2OFISx1+g614ewU2ijPAkxauFQTLNbJY57KmXgV0CHRf8zKosxsDJDdqtM5TavX77d39wSK+SEQl3csZq8nvxIeeEkrsML614eOM5Bg+xoFGVy0fI6u\\\\\\\/7lSf5BPgsgIwJpoQZ265ezttWe4KScjjTSWLCci8FW1uaeHhV7gkQYmcrPgKPG9r03LVcI+IQU6G\\\\\\\/7bZMOpNIqtD\\\\\\\/8IOaSuy5FrBBK4xYsgLu3Z4cJ2Fj2KgXDdMSWPVfyLHL8Oo6HJaC1tYMeSp7VdQ9gBN7GT2uLrePsI82e4GbMDM9xdDKBiWgklR563bXxdFS9qdcTQzTznoluLY+wYBOmUP8na7JX0tubdRrbuFTJKkuXD+JIb3lyEJJPnJCk7Ch\\\\\\\/XZsU4XfSTZebjOfe8v50QwrpN9fjoS4yszA7suGjkIOSaT5Iaa14P0XE+xD3ShzwwF1ACCJEYy08qUG0cGaPCl6uF+P9qUOqRjRhDwMWyWO6DO6AMVc9Z0bjNKAgvWumUgURzMQiZd6N58\\\\\\\/Wb4D7LMjjXC2VujLV+wTp4MAJvKutM6xdUI4av75+qew2f0I5HUNM42a1LLfJPev8capxYdcIDtOTVXB\\\\\\\/s0AQLcx5H3BoTtglZOTQzX1WBvC7kv6ODJ1TSOYZVbT38nNywmHkYisiKxOudJxtQJB1vWt6j8f2UId7MallC3vTpF1KQQcKcgJnpw2QJFbj53wahqi6olQuznWS5FNHWU2SaxVkBh68bCQMEoVEhpzlbpV1xnwB2kEDrhmUH5yIp3bbRxwUe5yb54rNnQmiLz2WvGtBDH9IbadUFDsI9ai4Lu3WIc7\\\\\\\/vsgcqCge7atBMzmQ8k3203TDRLOLVFSH4yXkJJFH1tAyA3dlXwDUJMayvfGJqPc82hHq2LZ0qC8pvgt\\\\\\\/jHTpagkseb+nX2CEruQnpGRYrEQIbZK4zDHv4BoXsIxbk4k6hY4qe8Vkbdg2aCYmzjbk8HKyVrEckRW\\\\\\\/oeHKNIvAujutH2xx0SuWb14QAMESaonGh89Th6lfmRjfYxlwVcwLSCafRU0F+cNgRwLsP1BVBxL7YmkCB8yoJqJZv8IwyTg6KRSFpOZvfbQh3HvXeW8A\\\\\\\/WhPx2044mqZJkSAyJJwvShj60DOGtGIMlErHRq64gtU3jvvaPKf8voiLYOnl\\\u0022,\\\u0022iv\\\u0022:\\\u0022lm2Q7BScT3m8t1thcfIKjQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022525e2c9abef8a752dd62bdcbd28e58e9b59291346cf68c3adc80038b320df057\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459197 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/515" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OuRazDamOVhBL3mT_a4p7Ed1PlVddUfTVJfbvoUvID4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/515" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VsaQluncSfhDvJUlGWC1HEsjJY2pEhjCeYEEYqBnBkc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/505" title="Genome-Wide Association Analysis of Ischemic Stroke in Young Adults" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/7/523" title="High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used Candida albicans Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459197" id="node459197" data-pisa="ggg;1/6/515" data-pisa-master="ggg;g3.111.000927" data-apath="/ggg/1/6/515.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Scale Analysis of Programmed DNA Elimination Sites in <em>Tetrahymena thermophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joseph N. Fass</span>, <span class="highwire-citation-author" data-delta="1">Nikhil A. Joshi</span>, <span class="highwire-citation-author" data-delta="2">Mary T. Couvillion</span>, <span class="highwire-citation-author" data-delta="3">Josephine Bowen</span>, <span class="highwire-citation-author" data-delta="4">Martin A. Gorovsky</span>, <span class="highwire-citation-author" data-delta="5">Eileen P. Hamilton</span>, <span class="highwire-citation-author" data-delta="6">Eduardo Orias</span>, <span class="highwire-citation-author" data-delta="7">Kyungah Hong</span>, <span class="highwire-citation-author" data-delta="8">Robert S. Coyne</span>, <span class="highwire-citation-author" data-delta="9">Jonathan A. Eisen</span>, <span class="highwire-citation-author" data-delta="10">Douglas L. Chalker</span>, <span class="highwire-citation-author" data-delta="11">Dawei Lin</span> and <span class="highwire-citation-author" data-delta="12">Kathleen Collins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">515-522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000927 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459197" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph N. Fass </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BN.%2BFass%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fass%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BN.%2BFass%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nikhil A. Joshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikhil%2BA.%2BJoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joshi%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANikhil%2BA.%2BJoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mary T. Couvillion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BT.%2BCouvillion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Couvillion%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BT.%2BCouvillion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josephine Bowen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BBowen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosephine%2BBowen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin A. Gorovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BA.%2BGorovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gorovsky%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BA.%2BGorovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Eileen P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eileen%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEileen%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eduardo Orias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BOrias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orias%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEduardo%2BOrias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kyungah Hong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyungah%2BHong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hong%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyungah%2BHong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Robert S. Coyne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>J. Craig Venter Institute, Rockville, Maryland 20850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BS.%2BCoyne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coyne%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BS.%2BCoyne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jonathan A. Eisen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BA.%2BEisen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eisen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BA.%2BEisen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Douglas L. Chalker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biology, Washington University, St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douglas%2BL.%2BChalker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chalker%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADouglas%2BL.%2BChalker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Dawei Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dawei%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADawei%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kathleen Collins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BCollins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collins%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathleen%2BCollins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kcollins@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">kcollins@berkeley.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/6/515" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459197" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/515.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459197" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/515.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459197" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/515.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459197" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/6/515.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459197" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Genetically programmed DNA rearrangements can regulate mRNA expression at an individual locus or, for some organisms, on a genome-wide scale. Ciliates rely on a remarkable process of whole-genome remodeling by DNA elimination to differentiate an expressed macronucleus (MAC) from a copy of the germline micronucleus (MIC) in each cycle of sexual reproduction. Here we describe results from the first high-throughput sequencing effort to investigate ciliate genome restructuring, comparing Sanger long-read sequences from a <em>Tetrahymena thermophila</em> MIC genome library to the MAC genome assembly. With almost 25% coverage of the unique-sequence MAC genome by MIC genome sequence reads, we created a resource for positional analysis of MIC-specific DNA removal that pinpoints MAC genome sites of DNA elimination at nucleotide resolution. The widespread distribution of internal eliminated sequences (IES) in promoter regions and introns suggests that MAC genome restructuring is essential not only for what it removes (for example, active transposons) but also for what it creates (for example, splicing-competent introns). Consistent with the heterogeneous boundaries and epigenetically modulated efficiency of individual IES deletions studied to date, we find that IES sites are dramatically under-represented in the ∼25% of the MAC genome encoding exons. As an exception to this general rule, we discovered a previously unknown class of small (&lt;500 bp) IES with precise elimination boundaries that can contribute the 3′ exon of an mRNA expressed during genome restructuring, providing a new mechanism for expanding mRNA complexity in a developmentally regulated manner.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/tetrahymena" class="hw-term hw-article-keyword hw-article-keyword-tetrahymena" rel="nofollow">Tetrahymena</a></li><li class="kwd"><a href="/keyword/ciliate-nuclear-dualism" class="hw-term hw-article-keyword hw-article-keyword-ciliate-nuclear-dualism" rel="nofollow">ciliate nuclear dualism</a></li><li class="kwd"><a href="/keyword/community-sequencing-project" class="hw-term hw-article-keyword hw-article-keyword-community-sequencing-project" rel="nofollow">community sequencing project</a></li><li class="kwd"><a href="/keyword/genome-rearrangement" class="hw-term hw-article-keyword hw-article-keyword-genome-rearrangement" rel="nofollow">genome rearrangement</a></li><li class="kwd"><a href="/keyword/dna-breakage-and-joining" class="hw-term hw-article-keyword hw-article-keyword-dna-breakage-and-joining" rel="nofollow">DNA breakage and joining</a></li></ul><p id="p-5">Regulated genome rearrangements are an evolutionarily widespread mechanism for affecting changes in gene expression, for example, switching cell mating type, alternating surface protein presentation, or expanding the repertoire of antibody production (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rusche and Rine 2010</a>). More extreme cases of genome remodeling by large-scale chromatin diminution have been demonstrated in diverse eukaryotes, including ciliated protozoa, parasitic nematodes, and hagfish (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kloc and Zagrodzinska 2001</a>). In the entire phylum Ciliata, a transcriptionally active MAC is differentiated from a copy of the germline MIC by elimination of repetitive DNA (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Karrer 2000</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Duharcourt <em>et al.</em> 2009</a>). This process is proposed to accomplish a defense of the phenotypically expressed genome from the influence of foreign DNA. Consistent with this hypothesis, repetitive DNA elimination in ciliates involves the same process of RNA-guided heterochromatin formation required for transposon silencing in other eukaryotes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mochizuki and Gorovsky 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Yao and Chao 2005</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chalker 2008</a>). Knowledge of how the MAC and MIC differ is fundamental to understanding the evolutionarily success of ciliates as well as for enabling studies of the chromosome structures that support meiosis and mitosis (MIC chromosomes) or chromosome segregation without classic heterochromatin (MAC chromosomes).</p><p id="p-6">Among the ciliates, <em>Tetrahymena thermophila</em> has been a favorable model organism for discoveries of fundamental eukaryotic biology (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Collins and Gorovsky 2005</a>). The ∼104 Mbp MAC genome of <em>T. thermophila</em> has been sequenced and annotated, revealing a complexity of gene families comparable to that in multicellular organisms (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Eisen <em>et al.</em> 2006</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>). Genome-scale analysis of a ciliate MIC has not yet been described. Reassociation kinetics and quantitative DNA staining methods estimate <em>T. thermophila</em> MIC genome complexity as 10–20% greater than that of the MAC (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Yao and Gorovsky 1974</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gorovsky 1980</a>), but only a handful of MIC-specific elements, known as IES, have been characterized. IES are removed from the genome of the developing MAC <em>en masse</em> in a period of only a few hours during the sexual process of conjugation (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Yao and Chao 2005</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chalker 2008</a>). Extrapolation from the frequency of IES detection by differential restriction fragment mobility of MIC <em>vs.</em> MAC DNA, based on Southern blots of a few randomly selected genome regions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Yao <em>et al.</em> 1984</a>), suggests a number of ∼6,000 MAC genome sites of IES removal. Many sequenced IES appear to be noncoding, whereas others carry ORFs related to transposon-encoded genes (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Yao and Chao 2005</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chalker 2008</a>). No known <em>T. thermophila</em> IES interrupts a protein-coding open reading frame (ORF), although the much shorter IES of <em>Paramecium tetraurelia</em> that lack epigenetic modulation of excision frequently do (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Duret <em>et al.</em> 2008</a>).</p><p id="p-7">Enabled by a Joint Genome Institute (JGI) Community Sequencing Project, we used high-throughput <em>T. thermophila</em> MIC genome sequencing to initiate the genome-scale investigation of nuclear differentiation from MIC to MAC. By aligning MIC genome Sanger sequence reads to the set of assembled MAC contigs in a manner refined for pinpointing positions of DNA elimination, we created a community resource for IES investigation. Although IES are dramatically depleted in the ∼25% of the MAC genome predicted to contain exons with ORF, as an exception, we show that one member of a new class of short IES provides an exon that changes the mRNA 3′ end of a protein expressed during genome restructuring. The demonstration that an IES can provide an exon cassette establishes a new mechanism for increasing ciliate mRNA complexity in a developmentally regulated manner.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Nucleic acid purification and analysis</h3><p id="p-8">Nuclei were purified from the inbred <em>T. thermophila</em> strain SB210 used previously in the MAC genome project, with homozygous MIC allele content. MIC isolation from MAC was performed by differential centrifugation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">White <em>et al.</em> 1988</a>). MAC contamination of the MIC preparation was estimated by nuclei counts as ∼0.1%, which adjusting for differential DNA content corresponds to a mass contamination of 1–2%. Total cellular DNA was used for PCR assays and total cellular RNA was used for Northern blots. Primers are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/000927SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/-/DC1/TableS1.pdf">Table S1</a>. Northern blots used hexamer-primed radiolabeled probes synthesized from double-stranded DNA templates.</p></div><div id="sec-3" class="subsection"><h3>Library sequencing and read alignments</h3><p id="p-9">Paired-end (b1/g1) reads from an ∼8 kbp MIC DNA fragment library in pMCL are available from the NCBI trace archives (Project ID 32845). After vector sequence removal, reads were retrimmed for average qualities at least Q20 and aligned using BWA-SW default settings (bio-bwa.sourceforge.net). Read segments were designated by alignment order as aln1, aln2, etc. If more than 10 bp did not align from one side of the read only, the alignment was segregated to a separate browser track as a candidate indicator of IES position. Fully mapping and one-sided-mapping reads that could align at more than one MAC genome location were segregated to a separate set of multimapper browser tracks.</p></div><div id="sec-4" class="subsection"><h3>Browser reference genome design and annotations</h3><p id="p-10">MIC library reads were aligned to a concatenation of MAC genome contigs ordered by decreasing size. This reference genome contains 103,002,206 bp of <em>T. thermophila</em> MAC genome project sequence with 10 kbp blocks of N added between contigs, for a total of 114,482,206 bp. Browser track annotation of candidate IES sites allowed no more than 9 bp of overlap between the MAC-matching portions of convergent L and R reads (described below). The mapped sequence reads, IES annotations, previously predicted <em>T. thermophila</em> genes (indicated by TTHERM number), and their individual predicted exons (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>) are aligned in an open-access community resource with the UCSC genome browser format (gb.genomecenter.ucdavis.edu, clade Alveolata, genome <em>T. thermophila</em>, Sep 2009 assembly).</p></div></div><div class="section results-discussion" id="sec-5"><h2 class="">Results and Discussion</h2><div id="sec-6" class="subsection"><h3>Library construction and read alignments</h3><p id="p-11">Previous shotgun sequencing and assembly of the <em>T. thermophila</em> MAC used paired-end Sanger reads from 2–6 kbp DNA fragment libraries (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Eisen <em>et al.</em> 2006</a>). In pilot Sanger sequencing trials of MIC genome libraries, an ∼8 kbp insert DNA library was the maximal insert size to give a high yield of validated reads. However, high-throughput sequencing of this MIC genome library did not yield the expected genome coverage of high-quality sequence reads, suggesting some destabilization of the 8 kbp insert library from the ∼75% AT bias of the <em>T. thermophila</em> genomes (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Yao and Gorovsky 1974</a>).</p><p id="p-12">Extensively quality-filtered sequence reads were aligned to a concatenated assembly of MAC genome contigs ordered from the largest to smallest in size, termed the conMAC. The smallest MAC chromosome, encoding only ribosomal RNA, was omitted, as was the mitochondrial chromosome. The conMAC arrangement of the reference genome has the benefit of segregating the expected MIC contamination of the MAC genome assemblies, present in small scaffolds of generally low-sequence coverage (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>), to the right edge of the reference genome representation (gb.genomecenter.ucdavis.edu, clade Alveolata, genome <em>T. thermophila</em>, Sep 2009 assembly; see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for genome browser track displays and annotations). The MAC contains some repetitive sequences, including RNA-coding, protein-coding, and noncoding genome segments (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Eisen <em>et al.</em> 2006</a>). Therefore, we segregated MIC genome sequence reads that aligned at multiple MAC locations to browser tracks separate from the uniquely aligning reads. The uniquely mapping MIC genome reads aligned to 22,339,139 bp of the MAC genome sequence, approaching 25% coverage of the unique-sequence fraction of the MAC genome.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/515/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Annotation of MAC positions of IES excision. (A) A representative high-confidence win1 IES site prediction in the MAC genome. The candidate IES site (IES D) falls within an internal exon of the gene model for TTHERM_00198180, which is displayed in its entirety (see scale bar at top). The genome browser track &#x201C;Putative IES sites using uniquely mapped Sanger reads&#x201D; indicates positions of the win1, win2, and win3 IES predictions described in the text. The track &#x201C;Sanger Reads Left (unique)&#x201D; shows the extent of MAC-matching sequence within a MIC genome read that maps to the MAC with its left end but has a nonmapping sequence on its right end; the read matched a unique MAC sequence, in contrast to reads segregated to the track &#x201C;Sanger Reads Left (multi).&#x201D; The track &#x201C;Sanger Reads Right (unique)&#x201D; shows the extent of MAC-matching sequence within a MIC genome read that maps to the MAC with its right end but has a nonmapping sequence on its left end; the read matched a unique MAC sequence, in contrast to reads segregated to the track &#x201C;Sanger Reads Right (multi).&#x201D; The tracks &#x201C;Sanger Reads Fully Mapped&#x201D; show read alignments that matched the MAC without a left-end-only or right-end-only extension of nonmapping sequence. (B) Widespread MAC chromosome distribution of IES sites predicted from Sanger L and/or R read alignments. Note that the conMAC region containing the smallest MAC genome contigs at right is also overrepresented in the 10 kbp spacer blocks of N that were added between contigs; therefore, the bp amount of MAC assembly in this region is exaggerated as a proportion of conMAC length. The upper track shows the entire conMAC assembly of MAC genome contigs joined in order of decreasing length; in the browser, this track is designated as the fakeasome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-956301985" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Annotation of MAC positions of IES excision. (A) A representative high-confidence win1 IES site prediction in the MAC genome. The candidate IES site (IES D) falls within an internal exon of the gene model for TTHERM_00198180, which is displayed in its entirety (see scale bar at top). The genome browser track &#x201C;Putative IES sites using uniquely mapped Sanger reads&#x201D; indicates positions of the win1, win2, and win3 IES predictions described in the text. The track &#x201C;Sanger Reads Left (unique)&#x201D; shows the extent of MAC-matching sequence within a MIC genome read that maps to the MAC with its left end but has a nonmapping sequence on its right end; the read matched a unique MAC sequence, in contrast to reads segregated to the track &#x201C;Sanger Reads Left (multi).&#x201D; The track &#x201C;Sanger Reads Right (unique)&#x201D; shows the extent of MAC-matching sequence within a MIC genome read that maps to the MAC with its right end but has a nonmapping sequence on its left end; the read matched a unique MAC sequence, in contrast to reads segregated to the track &#x201C;Sanger Reads Right (multi).&#x201D; The tracks &#x201C;Sanger Reads Fully Mapped&#x201D; show read alignments that matched the MAC without a left-end-only or right-end-only extension of nonmapping sequence. (B) Widespread MAC chromosome distribution of IES sites predicted from Sanger L and/or R read alignments. Note that the conMAC region containing the smallest MAC genome contigs at right is also overrepresented in the 10 kbp spacer blocks of N that were added between contigs; therefore, the bp amount of MAC assembly in this region is exaggerated as a proportion of conMAC length. The upper track shows the entire conMAC assembly of MAC genome contigs joined in order of decreasing length; in the browser, this track is designated as the fakeasome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/515/F1.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/515/F1.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/515/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/515/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-13" class="first-child">Annotation of MAC positions of IES excision. (A) A representative high-confidence win1 IES site prediction in the MAC genome. The candidate IES site (IES D) falls within an internal exon of the gene model for TTHERM_00198180, which is displayed in its entirety (see scale bar at top). The genome browser track “Putative IES sites using uniquely mapped Sanger reads” indicates positions of the win1, win2, and win3 IES predictions described in the text. The track “Sanger Reads Left (unique)” shows the extent of MAC-matching sequence within a MIC genome read that maps to the MAC with its left end but has a nonmapping sequence on its right end; the read matched a unique MAC sequence, in contrast to reads segregated to the track “Sanger Reads Left (multi).” The track “Sanger Reads Right (unique)” shows the extent of MAC-matching sequence within a MIC genome read that maps to the MAC with its right end but has a nonmapping sequence on its left end; the read matched a unique MAC sequence, in contrast to reads segregated to the track “Sanger Reads Right (multi).” The tracks “Sanger Reads Fully Mapped” show read alignments that matched the MAC without a left-end-only or right-end-only extension of nonmapping sequence. (B) Widespread MAC chromosome distribution of IES sites predicted from Sanger L and/or R read alignments. Note that the conMAC region containing the smallest MAC genome contigs at right is also overrepresented in the 10 kbp spacer blocks of N that were added between contigs; therefore, the bp amount of MAC assembly in this region is exaggerated as a proportion of conMAC length. The upper track shows the entire conMAC assembly of MAC genome contigs joined in order of decreasing length; in the browser, this track is designated as the fakeasome.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Defining MAC locations of IES</h3><p id="p-14">We exploited the relatively long Sanger read lengths for an alignment-based strategy of defining MAC genome positions of IES excision. Because known <em>T. thermophila</em> IES are longer than a Sanger read length, we expected some MIC library reads to align to the MAC genome at one end only, with the remaining nonmapping read extension continuing into an IES. Therefore, reads that aligned to the MAC were sorted into three categories of mapping specificity: reads fully aligned to the MAC (50,057), reads with some internal discontinuity or mismatch that could have resulted from cloned DNA rearrangement (27,002), and reads aligned to the MAC from one end only with a nonmapping &gt;10 bp terminal extension at the other end (12,037; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/-/DC1/FileS1.html">File S1</a>). Reads that mapped with one end only but were at scaffold edges (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/-/DC1/FileS2.html">File S2</a>) were excluded from the one-end-mapper reads used for IES annotation. Nonmapping reads that did not align over any of the read length were also obtained, potentially representing MIC-specific DNA content (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/-/DC1/FileS3.fasta">File S3</a>).</p><p id="p-15">We used the Sanger reads that aligned to MAC genome sequence with one end only to define candidate MAC genome positions of IES. Of these reads, 5736 aligned on the browser left side of a putative IES (L alignments), and 6301 aligned on the browser right side of a putative IES (R alignments). Because reads capable of alignment with repetitive MAC sequences can falsely suggest IES by aligning elsewhere from their locus of origin, we segregated the reads that did not map uniquely to separate browser tracks. Considering only the uniquely mapping L and R reads, we identified 3251 MAC genome locations as candidate IES positions based on the criteria of (1) departure of an L and/or R read from MAC-mapping sequence, (2) no more than 9 bp of overlap between the MAC-mapping portions of convergent L and R read sequences (allowing for up to 9 bp of IES-flanking microhomology), and (3) no read mapping to the MAC across the putative IES location (which would be evidence against IES interruption of the MAC genome). We binned these putative IES positions as one of three window (win) designations: (1) IES positions implicated by converging L and R alignments with terminal MAC-matching positions separated by no more than 9 bp (win1 category, 404 sites), (2) IES positions implicated by L alignment(s) only (win2 category, 1378 sites), or (3) R alignment(s) only (win3 category, 1469 sites). <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a> shows a win1 category IES site prediction (shown for IES candidate D, described below).</p><p id="p-16">Some win2 and win3 category IES site predictions could be artifacts of sequence rearrangement within the library of cloned MIC genomic DNA fragments, but the convergent L and R alignments of the win1 category should pinpoint IES positions with high confidence and at nucleotide resolution. Assuming that the ∼25% coverage of the MAC genome by MIC genome sequence reads is randomly distributed, any given MAC genome position of IES excision would have a slightly less than 25% probability of coverage by an L read and the same probability of coverage by an R read. Therefore, as a rough calculation, we would expect to have defined somewhat less than 1/16<sup>th</sup> of the total number of IES positions with the high-confidence win1 predictions. Given the total of 404 win1 category annotations, our extrapolation for a genome-wide tally of IES excision sites modestly exceeds the ∼6,000 sites predicted from a small sampling of loci for differential MAC and MIC restriction fragment sizes (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Yao <em>et al.</em> 1984</a>).</p></div><div id="sec-8" class="subsection"><h3>Genome-wide distribution of IES positions</h3><p id="p-17">We first examined the distribution of predicted IES positions across MAC chromosomes. On a genome-wide scale (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>), IES positions are widely interspersed across the MAC genome. We next considered whether there was a bias to IES positions at a more local level, for example, with respect to protein-coding regions of the genome. Considering the inventory of ∼25,000 predicted MAC protein-coding genes and an average of ∼1000 nt per mRNA (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Calzone <em>et al.</em> 1983a</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>), segments of protein-coding ORF would account for almost 25% of the MAC genome sequence. <em>T. thermophila</em> mRNAs include an average of 3.6 introns per gene (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>), with typically short intron and untranslated region lengths of ∼150 nt, bringing the total sequence transcribed by RNA polymerase II to ∼50% of the MAC genome. Hybridization measurements of transcript complexity support the conclusion that more than half of the MAC genome is transcribed and that more than half of the nuclear RNA complexity is represented in translated mRNA (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Calzone <em>et al.</em> 1983b</a>). Therefore, if IES positions are random with respect to MAC genome context, almost 25% of IES should interrupt a protein-coding ORF.</p><p id="p-18">We sought to identify candidate exon-interrupting IES using two approaches. First, all of the high-confidence win1 category predictions of IES positions that were informatically determined to overlap predicted exons were examined manually for the validity of the gene and exon models. Candidates were excluded from further evaluation if they fell within exons of improbable gene models; for example, genes composed of a tiny exon and/or with the atypically long introns that were initially overpredicted by automated gene annotation (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>). Second, within the first 16,026,800 bp (&gt;10%) of the conMAC containing the largest, telomere-to-telomere assembled MAC chromosomes, we manually surveyed candidate IES positions implicated by L-only or R-only alignments for overlap with an exon of a predicted gene. The overwhelming majority of candidate IES positions were intergenic or within an intron, consistent with the MAC genome context of the <em>T. thermophila</em> IES characterized to date. For predicted exon-interrupting IES positions, close examination usually revealed that the putative exon was part of an improbable gene model. The limited number of exceptions was investigated as described below.</p></div><div id="sec-9" class="subsection"><h3>Validation of IES located in predicted exons</h3><p id="p-19">Following the analysis above, we retained 10 potentially reasonable predictions for putative IES that would excise from the MAC genome context of an exon (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The top three candidates (IES 1–3) met all of the selection criteria: high-confidence IES prediction by convergent L and R reads, expressed sequence tag (<a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>) and microarray-based (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Miao <em>et al.</em> 2009</a>) evidence for locus transcription (either indicated in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> as “e” for Gene “expression”), and Tetrahymena Genome Database annotation of at least one functional domain in the predicted gene product (indicated in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> as “a” for Gene “annotation”). Seven additional candidates (IES A–G) met a subset of these criteria (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Importantly, these IES were also supported by the presence of the same sequence of nonmapping extensions adjacent to the MAC-aligned read segment of each L or R read; this extension provided some or all of the IES sequence.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460984/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460984/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Exon-interrupting IES candidates</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Remarkably, for four of the candidate IES (1-2 and A-B), a complete IES sequence was possible to determine by comparison of the full-length Sanger sequence reads with each other and the MAC genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/-/DC1/FileS4.pdf">File S4</a>). Each of these IES was shorter than all previously characterized <em>T. thermophila</em> IES (&lt;500 bp; <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, under “IES Length”), with 3–4 bp of microhomology flanking the IES that was retained in single copy in the MAC genome (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>, under “MAC Junction”). All four IES include a TA dinucleotide within the microhomology, and three of the four have the complete TTAA motif that is efficiently cleaved by a domesticated <em>piggyBac</em> transposase-like protein essential for conjugation in <em>T. thermophila</em> and <em>P. tetraurelia</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baudry <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheng <em>et al.</em> 2010</a>). For the other candidate IES, a comparison of the MIC genome sequence reads with the assembled MAC revealed no or variable sequences of IES-flanking microhomology (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). Lack of a specific IES-flanking sequence motif is thought to be typical in <em>T. thermophila</em>, as is the heterogeneity of MAC junctions created by IES excision (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Yao <em>et al.</em> 2003</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Howard-Till and Yao 2007</a>). We suggest that the excision of different IES types could proceed by the action of different enzymes or involve different enzyme-associated factors, accounting for conservation of the TTAA motif at the boundaries of the new class of short IES but not at the boundaries of other, potentially more epigenetically regulated IES.</p><p id="p-28">To confirm the presence of the predicted IES with an independent approach, we used IES-flanking primers for PCR. As PCR templates, we prepared total DNA (MAC+MIC) from the sequenced inbred <em>T. thermophila</em> strain SB210 and two other MIC-homozygous inbred strains, CU428 and B2086. Initial PCR reactions were performed with primers containing MAC-destined sequences flanking the site of IES removal, which would amplify both the MAC genome junction following IES removal and, if suitably limited in length, the MIC genome IES-containing DNA as well (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, main panels and schematic at right). Additional PCR reactions were performed using one primer that spanned from MAC-destined to MIC-specific DNA and/or was entirely within the IES, designed using the nonmapping sequence extensions of the reads giving L and R alignments (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, small panels and schematics below the main panels). Because many IES are repetitive elements within the MIC genome, PCR using two entirely IES-internal primers was avoided; such reactions have the potential to amplify products from IES other than the intended MIC genome locus.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/515/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="IES validation by PCR. Genomic DNA isolated from strain SB210, CU428, or B2086 was amplified by PCR using primers flanking the putative IES site in MAC-destined DNA, as schematized at right. PCR products are visualized here as the negative image of an agarose gel stained with ethidium bromide. The smaller panels below the main panels for IES 2, A, and E show IES-specific PCR amplification using primer(s) that overlap or are internal to the IES, as also schematized. Relevant DNA standards are indicated. Expected MAC genome amplification products are labeled with &#x201C;&#x201D;; IES-containing PCR products are labeled with &#x201C;&#x201D;; note that IES size could be underestimated for IES C, D, and E if the IES contains internal repeat(s)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-956301985" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;IES validation by PCR. Genomic DNA isolated from strain SB210, CU428, or B2086 was amplified by PCR using primers flanking the putative IES site in MAC-destined DNA, as schematized at right. PCR products are visualized here as the negative image of an agarose gel stained with ethidium bromide. The smaller panels below the main panels for IES 2, A, and E show IES-specific PCR amplification using primer(s) that overlap or are internal to the IES, as also schematized. Relevant DNA standards are indicated. Expected MAC genome amplification products are labeled with &#x201C;&#x201D;; IES-containing PCR products are labeled with &#x201C;&#x201D;; note that IES size could be underestimated for IES C, D, and E if the IES contains internal repeat(s).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/515/F2.medium.gif" width="440" height="172"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/515/F2.medium.gif" width="440" height="172"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/515/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/515/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460930" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-29" class="first-child">IES validation by PCR. Genomic DNA isolated from strain SB210, CU428, or B2086 was amplified by PCR using primers flanking the putative IES site in MAC-destined DNA, as schematized at right. PCR products are visualized here as the negative image of an agarose gel stained with ethidium bromide. The smaller panels below the main panels for IES 2, A, and E show IES-specific PCR amplification using primer(s) that overlap or are internal to the IES, as also schematized. Relevant DNA standards are indicated. Expected MAC genome amplification products are labeled with “<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F2/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F2/embed/inline-graphic-1.gif"/></noscript></span></span>”; IES-containing PCR products are labeled with “<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F2/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F2/embed/inline-graphic-2.gif"/></noscript></span></span>”; note that IES size could be underestimated for IES C, D, and E if the IES contains internal repeat(s).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The inbred strains SB210, CU428, and B2086 were generated from independent sexual progeny and thus could differ in the MAC genome junctions created by IES removal. For most of the candidate IES, including all four of the entirely sequenced small IES (1-2 and A-B), PCR using MAC-destined sequence primers flanking the IES produced the same size of predominant amplification product from the MAC genomes of all three strains (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, designated by “<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/embed/inline-graphic-3.gif"/></noscript></span></span>”). However, the MAC junction product for IES F was shorter in length in CU428 than in SB210 and not detectable in B2086, whereas the MAC junction product for IES G was longer in length in CU428 than in SB210 or B2086. The MAC junction product for IES C also appeared heterogeneous. Importantly, for some amplification reactions, a unique lower-abundance product of substantially longer length than the expected MAC genome amplification product could also be detected (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, designated by “<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/embed/inline-graphic-4.gif"/></noscript></span></span>”). For cases in which a longer product was amplified with questionable specificity, we used one primer that included the MAC-destined sequence and a second primer that included the IES sequence to amplify a MIC genome product without competition from the MAC genome product (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, lower panels). Many PCR primer sequences and combinations were used to confirm a reliable IES length for the new class of &lt;500 bp IES (IES 1-2 and A-B) and for other IES of modest length up to ∼2 kbp (IES C–E); however, repeated attempts to determine the length of IES 3, F, and G were unsuccessful (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-31">Next, for the IES that could be amplified by PCR, we tested the genetic requirements for IES excision. Homokaryon gene knockout strains were mated that lacked the conjugation-essential enzyme Dicer-like 1 (<em>DCL1</em>), an IES-associated chromodomain protein (<em>PDD1</em>), or an additional factor (<em>LIA1</em>) required for IES excision (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Coyne <em>et al.</em> 1999</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Malone <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mochizuki and Gorovsky 2005</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Rexer and Chalker 2007</a>). Total DNA was purified from polyclonal populations of cells arrested without completing conjugation and compared with total DNA from strain SB210 in asexual vegetative growth. All of the IES appeared more abundant in conjugation-arrested cells, detected as an increase in the amplification of the IES-containing PCR product (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, designated by “<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/embed/inline-graphic-5.gif"/></noscript></span></span>”). We conclude that removal of even the smallest known <em>T. thermophila</em> IES is directly or indirectly dependent on RNA-guided heterochromatin formation. Heterogeneous boundaries for removal of IES C were evident by PCR (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">3</a>) and verified by sequencing of cloned junction amplification products (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Thus, sequencing of MAC genomes from inbred strains other than SB210 could allow the discovery of IES sites through sequence heterogeneity at the MAC junctions of IES removal.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/515/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="IES excision requirements. For each IES, PCR was done with the schematized primer pairs using genomic DNA isolated from the strain SB210 in vegetative growth or from the polyclonal pool of cells arrested 28 hr after initiation of conjugation for gene knockout strains lacking DCL1, PDD1, or LIA1. PCR products are visualized as the negative image of an agarose gel stained with ethidium bromide. DNA standard lanes have the 500 bp marker denoted with &#x201C;o&#x201D;; expected MAC genome amplification products are labeled with &#x201C;&#x201D;; and IES-containing PCR products are labeled with &#x201C;&#x201D; (these labels are included in a subset of adjacent gel lanes for clarity)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-956301985" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;IES excision requirements. For each IES, PCR was done with the schematized primer pairs using genomic DNA isolated from the strain SB210 in vegetative growth or from the polyclonal pool of cells arrested 28 hr after initiation of conjugation for gene knockout strains lacking &lt;em&gt;DCL1&lt;/em&gt;, &lt;em&gt;PDD1&lt;/em&gt;, or &lt;em&gt;LIA1&lt;/em&gt;. PCR products are visualized as the negative image of an agarose gel stained with ethidium bromide. DNA standard lanes have the 500 bp marker denoted with &#x201C;o&#x201D;; expected MAC genome amplification products are labeled with &#x201C;&#x201D;; and IES-containing PCR products are labeled with &#x201C;&#x201D; (these labels are included in a subset of adjacent gel lanes for clarity).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/515/F3.medium.gif" width="440" height="137"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/515/F3.medium.gif" width="440" height="137"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/515/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/515/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460942" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-32" class="first-child">IES excision requirements. For each IES, PCR was done with the schematized primer pairs using genomic DNA isolated from the strain SB210 in vegetative growth or from the polyclonal pool of cells arrested 28 hr after initiation of conjugation for gene knockout strains lacking <em>DCL1</em>, <em>PDD1</em>, or <em>LIA1</em>. PCR products are visualized as the negative image of an agarose gel stained with ethidium bromide. DNA standard lanes have the 500 bp marker denoted with “o”; expected MAC genome amplification products are labeled with “<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F3/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F3/embed/inline-graphic-6.gif"/></noscript></span></span>”; and IES-containing PCR products are labeled with “<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F3/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F3/embed/inline-graphic-7.gif"/></noscript></span></span>” (these labels are included in a subset of adjacent gel lanes for clarity).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/515/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence alignment of MAC junctions resulting from removal of IES C. A multiple sequence alignment is shown for cloned MAC junctions resulting from the removal of IES C, which together have three lengths and four distinct sequences. Clones were sequenced for junctions amplified from DNA of the inbred strains B2086, CU428, and SB210 (lines 1&#x2013;3) or amplified from the DNA of noninbred strain crosses used in Figure 3 (lines 4&#x2013;5; conj1 and conj2 indicate the two distinct sequences obtained from many cloned DNA fragment sequences). Below the alignment, &#x201C;&#x201D; indicates the region of consensus sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-956301985" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sequence alignment of MAC junctions resulting from removal of IES C. A multiple sequence alignment is shown for cloned MAC junctions resulting from the removal of IES C, which together have three lengths and four distinct sequences. Clones were sequenced for junctions amplified from DNA of the inbred strains B2086, CU428, and SB210 (lines 1&#x2013;3) or amplified from the DNA of noninbred strain crosses used in Figure 3 (lines 4&#x2013;5; conj1 and conj2 indicate the two distinct sequences obtained from many cloned DNA fragment sequences). Below the alignment, &#x201C;&#x201D; indicates the region of consensus sequence.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/515/F4.medium.gif" width="440" height="67"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/515/F4.medium.gif" width="440" height="67"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/515/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/515/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460974" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-33" class="first-child">Sequence alignment of MAC junctions resulting from removal of IES C. A multiple sequence alignment is shown for cloned MAC junctions resulting from the removal of IES C, which together have three lengths and four distinct sequences. Clones were sequenced for junctions amplified from DNA of the inbred strains B2086, CU428, and SB210 (lines 1–3) or amplified from the DNA of noninbred strain crosses used in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> (lines 4–5; conj1 and conj2 indicate the two distinct sequences obtained from many cloned DNA fragment sequences). Below the alignment, “<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F4/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/515/F4/embed/inline-graphic-8.gif"/></noscript></span></span>” indicates the region of consensus sequence.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>IES inclusion in mRNA</h3><p id="p-34"><em>T. thermophila</em> gene predictions include divergent, nonfunctional paralogs of productive genes, and as a likely consequence, some predicted genes are not detectably expressed in any lifecycle stage evaluated by EST identification or microarray hybridization (<a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Miao <em>et al.</em> 2009</a>). Conversely, microarray or even EST evidence for locus expression is not certain evidence of a productive mRNA, particularly for data collected from conjugating cells with nongenic transcription. Therefore, we used Northern blot hybridization of total RNA to investigate whether putative exons flanking the six shortest, confirmed IES (IES 1-2 and A-D) were incorporated into discrete, biologically stable mRNAs. We analyzed total RNA from vegetatively growing cells and a conjugation time course extending through the interval of IES excision in the newly differentiating MAC genome (0–12 hr). No specific Northern blot hybridization signal was detected using probes containing the predicted mRNA region 5′ of IES 1, 2, or D, but probes for the regions flanking IES A, B, and C detected at least a heterogeneous smear of conjugation-specific nongenic RNA, predominantly less than 500 nucleotides (nt) in length (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). This nongenic transcription promotes IES assembly into heterochromatin (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chalker and Yao 2001</a>). At least for IES A, B, and C, nongenic transcription must extend far enough into the IES-flanking region to be detectable by hybridization with a probe containing an entirely MAC-destined sequence. No discrete mRNA was detected for the putative host gene of IES A.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/515/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="IES host gene expression. (A) Total RNA was isolated from strain SB210 in asexual vegetative growth (V) or at the indicated times after initiation of conjugation using strains SB210 and CU428 (in hours). Northern blot hybridization was performed to detect the putative mRNA region adjacent to the indicated IES. Blots are shown in size register; nonspecific signal left from a previous probing of the same membrane is indicated by &#x201C;NS.&#x201D; (B) Transcripts from the region of the LIA2 locus hosting IES B were isolated by RT-PCR and sequenced. Boxes denote exons, and thick lines denote introns or untranslated regions. Gray boxes and solid lines indicate MAC-destined sequence, and the open box and dashed line indicate IES. Translation termination codons (TGA) and positions of mRNA polyadenylation (pA) are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-956301985" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;IES host gene expression. (A) Total RNA was isolated from strain SB210 in asexual vegetative growth (V) or at the indicated times after initiation of conjugation using strains SB210 and CU428 (in hours). Northern blot hybridization was performed to detect the putative mRNA region adjacent to the indicated IES. Blots are shown in size register; nonspecific signal left from a previous probing of the same membrane is indicated by &#x201C;NS.&#x201D; (B) Transcripts from the region of the &lt;em&gt;LIA2&lt;/em&gt; locus hosting IES B were isolated by RT-PCR and sequenced. Boxes denote exons, and thick lines denote introns or untranslated regions. Gray boxes and solid lines indicate MAC-destined sequence, and the open box and dashed line indicate IES. Translation termination codons (TGA) and positions of mRNA polyadenylation (pA) are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/515/F5.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/515/F5.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/515/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/515/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460877" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-35" class="first-child">IES host gene expression. (A) Total RNA was isolated from strain SB210 in asexual vegetative growth (V) or at the indicated times after initiation of conjugation using strains SB210 and CU428 (in hours). Northern blot hybridization was performed to detect the putative mRNA region adjacent to the indicated IES. Blots are shown in size register; nonspecific signal left from a previous probing of the same membrane is indicated by “NS.” (B) Transcripts from the region of the <em>LIA2</em> locus hosting IES B were isolated by RT-PCR and sequenced. Boxes denote exons, and thick lines denote introns or untranslated regions. Gray boxes and solid lines indicate MAC-destined sequence, and the open box and dashed line indicate IES. Translation termination codons (TGA) and positions of mRNA polyadenylation (pA) are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The host gene of IES C differs from the other nine putative IES host genes in having a microarray expression profile peak in vegetatively growing cells, albeit with low signal intensity, and no detectable expression in conjugating cells. Correspondingly, Northern blot hybridization revealed a discrete &gt;2,000 nt transcript that was most abundant in vegetatively growing cells (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>, IES C locus). Some of this transcript was also detectable in the starved cells that were mixed to initiate conjugation (time zero of the conjugation time course) and that remained as a fraction of the cells collected at all conjugation time points. Despite readily detectable mRNA production from the IES C host gene, RT-PCR attempts failed to detect the inclusion of the IES C sequence in a polyadenylated mRNA transcript.</p><p id="p-37">At least two discrete forms of mRNA were detected for the host gene of IES B amid the extensive background of nongenic transcript hybridization (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>, IES B locus). The host gene of IES B encodes the helicase-domain protein Lia2, which a previous study (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Yao <em>et al.</em> 2007</a>) and our Northern blot analysis (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>) concur in detecting with conjugation-specific expression. Curiously, the peaks of mRNA and nongenic RNA expression were largely coincident, and the expression timing of nongenic transcription was shifted slightly later compared with nongenic transcription from the loci of IES A and IES C (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5A</a>). RT-PCR product sequences revealed that IES B was incorporated into polyadenylated mRNA (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5B</a>; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/-/DC1/FileS5.pdf">File S5</a> and GenBank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=EF219412.1&amp;atom=%2Fggg%2F1%2F6%2F515.atom">EF219412.1</a>). Remarkably, the primary transcript containing the IES was spliced to remove a portion of the predicted 3′ exon of the macronuclear <em>LIA2</em> gene and part of the IES, creating an ORF that directed both translation termination and polyadenylation within the IES (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5B</a>). As a consequence, at least one form of <em>LIA2</em> mRNA encodes a protein C-terminus translated from within IES B. Whether this mRNA is translated to produce a stably accumulated protein product remains to be established by direct analysis of Lia2 polypeptide(s).</p><p id="p-38">The IES-containing <em>LIA2</em> mRNA would be transcribed in the developing MAC. Thus, it is surprising that nongenic RNA expression from the developing MAC is also readily detected, contemporary with <em>LIA2</em> mRNA. We suggest that the location of the IES in the 3′ terminal exon of the host gene (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>) may not be coincidental: this location provides maximum physical spacing between the initiation sites of mRNA transcription and nongenic IES transcription, which may reduce their competition. In addition, as a requisite for or result of mRNA expression, nongenic IES transcription may be delayed for the <em>LIA2</em> locus relative to loci that are not producing an mRNA product in conjugating cells. In conclusion, although IES are clearly depleted from exons on a genome-wide scale, expression of an IES-containing mRNA provides a previously unrecognized mechanism for increasing mRNA complexity in a developmentally regulated manner.</p></div><div id="sec-11" class="subsection"><h3>Concluding perspectives</h3><p id="p-39">By positional identification of MAC genome sites of IES removal, we have created a community resource for IES discovery and characterization. Based on the <em>T. thermophila</em> content of MIC-specific <em>vs.</em> MAC-destined DNA and the estimated number of IES in the MAC genome, an average IES should be on the order of ∼2 kbp. However, we suggest that there is not an “average” IES. Instead, a true complexity of IES types seems likely to exist varying in size, sequence complexity, excision boundary precision, sensitivity to epigenetic modulation, and mechanism of elimination. Although the bulk of the MIC-specific sequence should be repetitive elements (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gorovsky 1980</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Yao and Chao 2005</a>), the new category of &lt;500 bp IES characterized here could contribute a large number of limited-length unique sequences. IES containing repetitive DNA have functions in germline genome maintenance, for example, providing the specialized chromatin regions necessary for chromosome segregation during mitosis and meiosis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cui and Gorovsky 2006</a>). We suggest that other types of IES, for example, the &lt;500 bp IES with precise elimination boundaries established by TTAA motifs, may at least infrequently gain regulatory function by altering the level or sequence of a specific mRNA. This form of regulation would be restricted to the interval of conjugation subsequent to transcriptional onset and prior to the completion of IES excision in a differentiating MAC.</p><p id="p-40">On an individual level, experimentally induced failures of IES excision are not necessarily deleterious for cell survival (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chalker and Yao 1996</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Yao and Chao 2005</a>). In contrast, on genomic scale, halting the process of DNA elimination at any of a large number of intermediate steps has precluded the generation of viable sexual progeny (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Chalker 2008</a>). Our IES positional analysis suggests that few if any growth-expressed <em>T. thermophila</em> genes have exons interrupted by an IES, but the widespread distribution of IES in promoter regions and introns can account for why global failures of IES excision are incompatible with MAC function. Introns in <em>T. thermophila</em> are of limited length (even less than the length of the new class of short IES characterized in this work), and splice sites and polyadenylation signals have low sequence complexity (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Eisen <em>et al.</em> 2006</a>; <a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>). As demonstrated by the example of the <em>LIA2</em> IES above, inclusion of IES sequence in an mRNA primary transcript would therefore likely trigger additional mRNA splicing and/or premature mRNA polyadenylation. Also, due to high gene density in the <em>T. thermophila</em> MAC (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Eisen <em>et al.</em> 2006</a>; <a id="xref-ref-10-10" class="xref-bibr" href="#ref-10">Coyne <em>et al.</em> 2008</a>), many intergenic IES are likely to interrupt coordination between the regulatory elements of a transcriptional promoter. Thus, although IES occupy safe-haven locations in the MAC genome in the sense of tolerance for imprecise deletion boundaries, the bulk of IES length would need to be removed to produce a functional organization of many <em>T. thermophila</em> genes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">Sanger reads were generated from a JGI Community Sequencing Project; JGI is supported by the Office of Science of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231. Support for this research was also provided by National Institutes of Health (NIH) Grant RR-009231 and National Science Foundation (NSF) Grant MCB-1025069 (E.O.); NIH Grant GM-072752 (M.A.G.); NSF Grant EF-0240361 and NIH Grant GM067012 (J.A.E.); NSF Grant MCB-0642162 (D.L.C.); and NIH Grant GM-054198 and a Miller Institute Professorship (K.C.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000927/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Paired-end Sanger reads from the JGI Community Sequencing Project for the <em>Tetrahymena thermophila</em> micronuclear genome: NCBI trace archives Project ID 32845. Open-access UCSC-format genome browser display of annotated micronuclear genome read alignments to a size-ranked concatenation of macronuclear genome scaffolds and gene predictions: gb.genomecenter.ucdavis.edu, clade Alveolata, genome <em>T. thermophila</em>.</p></li><li class="fn" id="fn-10"><p id="p-42"><em>Communicating editor: M. J. Cherry</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-08-21"><span class="received-label">Received </span>August 21, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-09-30"><span class="accepted-label">Accepted </span>September 30, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Fass <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.1" data-doi="10.1101/gad.547309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudry</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malinsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Restituito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapusta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">PiggyMac, a domesticated piggyBac transposase involved in programmed genome rearrangements in the ciliate <em>Paramecium tetraurelia</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2478</span>–<span class="cit-lpage">2483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBaudry%26rft.auinit1%253DC.%26rft.volume%253D23%26rft.issue%253D21%26rft.spage%253D2478%26rft.epage%253D2483%26rft.atitle%253DPiggyMac%252C%2Ba%2Bdomesticated%2BpiggyBac%2Btransposase%2Binvolved%2Bin%2Bprogrammed%2Bgenome%2Brearrangements%2Bin%2Bthe%2Bciliate%2BParamecium%2Btetraurelia%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.547309%26rft_id%253Dinfo%253Apmid%252F19884254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIzLzIxLzI0NzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calzone</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angerer</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983a</span> <span class="cit-article-title">Regulation of protein synthesis in <em>Tetrahymena</em>. Quantitative estimates of the parameters determining the rates of protein synthesis in growing, starved, and starved-deciliated cells.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">258</span>: <span class="cit-fpage">6887</span>–<span class="cit-lpage">6898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DCalzone%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D258%26rft.issue%253D11%26rft.spage%253D6887%26rft.epage%253D6898%26rft.atitle%253DRegulation%2Bof%2Bprotein%2Bsynthesis%2Bin%2BTetrahymena.%2BQuantitative%2Bestimates%2Bof%2Bthe%2Bparameters%2Bdetermining%2Bthe%2Brates%2Bof%2Bprotein%2Bsynthesis%2Bin%2Bgrowing%252C%2Bstarved%252C%2Band%2Bstarved-deciliated%2Bcells%26rft_id%253Dinfo%253Apmid%252F6853508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNTgvMTEvNjg4NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi81MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calzone</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stathopoulos</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grass</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angerer</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1983b</span> <span class="cit-article-title">Regulation of protein synthesis in <em>Tetrahymena</em>. RNA sequence sets of growing and starved cells.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">258</span>: <span class="cit-fpage">6899</span>–<span class="cit-lpage">6905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DCalzone%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D258%26rft.issue%253D11%26rft.spage%253D6899%26rft.epage%253D6905%26rft.atitle%253DRegulation%2Bof%2Bprotein%2Bsynthesis%2Bin%2BTetrahymena.%2BRNA%2Bsequence%2Bsets%2Bof%2Bgrowing%2Band%2Bstarved%2Bcells%26rft_id%253Dinfo%253Apmid%252F6189832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNTgvMTEvNjg5OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi81MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.4" data-doi="10.1128/MCB.16.7.3658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalker</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Non-Mendelian, heritable blocks to DNA rearrangement are induced by loading the somatic nucleus of <em>Tetrahymena thermophila</em> with germ line-limited DNA.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3658</span>–<span class="cit-lpage">3667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DChalker%26rft.auinit1%253DD.%2BL.%26rft.volume%253D16%26rft.issue%253D7%26rft.spage%253D3658%26rft.epage%253D3667%26rft.atitle%253DNon-Mendelian%252C%2Bheritable%2Bblocks%2Bto%2BDNA%2Brearrangement%2Bare%2Binduced%2Bby%2Bloading%2Bthe%2Bsomatic%2Bnucleus%2Bof%2BTetrahymena%2Bthermophila%2Bwith%2Bgerm%2Bline-%2Blimited%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.7.3658%26rft_id%253Dinfo%253Apmid%252F8668182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE2LzcvMzY1OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi81MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.5" data-doi="10.1101/gad.884601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalker</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Nongenic, bidirectional transcription precedes and may promote developmental DNA deletion in <em>Tetrahymena thermophila</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1287</span>–<span class="cit-lpage">1298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DChalker%26rft.auinit1%253DD.%2BL.%26rft.volume%253D15%26rft.issue%253D10%26rft.spage%253D1287%26rft.epage%253D1298%26rft.atitle%253DNongenic%252C%2Bbidirectional%2Btranscription%2Bprecedes%2Band%2Bmay%2Bpromote%2Bdevelopmental%2BDNA%2Bdeletion%2Bin%2BTetrahymena%2Bthermophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.884601%26rft_id%253Dinfo%253Apmid%252F11358871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzEwLzEyODciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.6" data-doi="10.1016/j.bbamcr.2008.07.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalker</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dynamic nuclear reorganization during genome remodeling of <em>Tetrahymena</em>.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1783</span>: <span class="cit-fpage">2130</span>–<span class="cit-lpage">2136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DChalker%26rft.auinit1%253DD.%2BL.%26rft.volume%253D1783%26rft.issue%253D11%26rft.spage%253D2130%26rft.epage%253D2136%26rft.atitle%253DDynamic%2Bnuclear%2Breorganization%2Bduring%2Bgenome%2Bremodeling%2Bof%2BTetrahymena.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbamcr.2008.07.012%26rft_id%253Dinfo%253Apmid%252F18706458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbamcr.2008.07.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18706458&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.7" data-doi="10.1091/mbc.E09-12-1079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mochizuki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A domesticated piggyBac transposase plays key roles in heterochromatin dynamics and DNA cleavage during programmed DNA deletion in <em>Tetrahymena thermophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E09-12-1079%26rft_id%253Dinfo%253Apmid%252F20357003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMC8xNzUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzUxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.8" data-doi="10.1016/j.cub.2005.04.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Tetrahymena thermophila.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R317</span>–<span class="cit-lpage">R318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DCollins%26rft.auinit1%253DK.%26rft.volume%253D15%26rft.issue%253D9%26rft.spage%253DR317%26rft.epage%253DR318%26rft.atitle%253DTetrahymena%2Bthermophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2005.04.039%26rft_id%253Dinfo%253Apmid%252F15886083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2005.04.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15886083&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.9" data-doi="10.1016/S1097-2765(00)80396-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikiforov</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smothers</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allis</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Parental expression of the chromodomain protein Pdd1p is required for completion of programmed DNA elimination and nuclear differentiation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DCoyne%26rft.auinit1%253DR.%2BS.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D865%26rft.epage%253D872%26rft.atitle%253DParental%2Bexpression%2Bof%2Bthe%2Bchromodomain%2Bprotein%2BPdd1p%2Bis%2Brequired%2Bfor%2Bcompletion%2Bof%2Bprogrammed%2BDNA%2Belimination%2Band%2Bnuclear%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252980396-2%26rft_id%253Dinfo%253Apmid%252F10619033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)80396-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10619033&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083885400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.10" data-doi="10.1186/1471-2164-9-562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiagarajan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tallon</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Refined annotation and assembly of the <em>Tetrahymena thermophila</em> genome sequence through EST analysis, comparative genomic hybridization, and targeted gap closure.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DCoyne%26rft.auinit1%253DR.%2BS.%26rft.volume%253D9%26rft.spage%253D562%26rft.epage%253D562%26rft.atitle%253DRefined%2Bannotation%2Band%2Bassembly%2Bof%2Bthe%2BTetrahymena%2Bthermophila%2Bgenome%2Bsequence%2Bthrough%2BEST%2Banalysis%252C%2Bcomparative%2Bgenomic%2Bhybridization%252C%2Band%2Btargeted%2Bgap%2Bclosure.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-562%26rft_id%253Dinfo%253Apmid%252F19036158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19036158&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.11" data-doi="10.1128/MCB.00079-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Centromeric histone H3 is essential for vegetative cell division and for DNA elimination during conjugation in <em>Tetrahymena thermophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4499</span>–<span class="cit-lpage">4510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DCui%26rft.auinit1%253DB.%26rft.volume%253D26%26rft.issue%253D12%26rft.spage%253D4499%26rft.epage%253D4510%26rft.atitle%253DCentromeric%2BHistone%2BH3%2BIs%2BEssential%2Bfor%2BVegetative%2BCell%2BDivision%2Band%2Bfor%2BDNA%2BElimination%2Bduring%2BConjugation%2Bin%2BTetrahymena%2Bthermophila%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00079-06%26rft_id%253Dinfo%253Apmid%252F16738316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi80NDk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzUxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.12" data-doi="10.1016/j.tig.2009.05.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duharcourt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepere</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Developmental genome rearrangements in ciliates: a natural genomic subtraction mediated by non-coding transcripts.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDuharcourt%26rft.auinit1%253DS.%26rft.volume%253D25%26rft.issue%253D8%26rft.spage%253D344%26rft.epage%253D350%26rft.atitle%253DDevelopmental%2Bgenome%2Brearrangements%2Bin%2Bciliates%253A%2Ba%2Bnatural%2Bgenomic%2Bsubtraction%2Bmediated%2Bby%2Bnon-coding%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.05.007%26rft_id%253Dinfo%253Apmid%252F19596481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19596481&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269967600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.13" data-doi="10.1101/gr.074534.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jubin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dessen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gout</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analysis of sequence variability in the macronuclear DNA of <em>Paramecium tetraurelia</em>: a somatic view of the germline.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074534.107%26rft_id%253Dinfo%253Apmid%252F18256234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzQvNTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzUxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.14" data-doi="10.1371/journal.pbio.0040286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiagarajan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Macronuclear genome sequence of the ciliate <em>Tetrahymena thermophila</em>, a model eukaryote.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DEisen%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De286%26rft.epage%253De286%26rft.atitle%253DMacronuclear%2Bgenome%2Bsequence%2Bof%2Bthe%2Bciliate%2BTetrahymena%2Bthermophila%252C%2Ba%2Bmodel%2Beukaryote.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040286%26rft_id%253Dinfo%253Apmid%252F16933976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16933976&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.15" data-doi="10.1146/annurev.ge.14.120180.001223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Genome organization and reorganization in <em>Tetrahymena</em>.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DGorovsky%26rft.auinit1%253DM.%2BA.%26rft.volume%253D14%26rft.spage%253D203%26rft.epage%253D239%26rft.atitle%253DGenome%2Borganization%2Band%2Breorganization%2Bin%2BTetrahymena.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.14.120180.001223%26rft_id%253Dinfo%253Apmid%252F6782938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.14.120180.001223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6782938&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980KU80900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.16" data-doi="10.1128/EC.00192-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard-Till</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Tudor nuclease genes and programmed DNA rearrangements in <em>Tetrahymena thermophila</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1795</span>–<span class="cit-lpage">1804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00192-07%26rft_id%253Dinfo%253Apmid%252F17715366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI2LzEwLzE3OTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNTE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-1.6.515.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karrer</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title"><em>Tetrahymena</em> genetics: two nuclei are better than one</span>, pp. <span class="cit-fpage">127</span>–<span class="cit-lpage">186</span> <span class="cit-comment">in</span> <span class="cit-source">Tetrahymena thermophila</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Asai</span>  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Forney</span>  <span class="cit-name-given-names">J. D.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.18" data-doi="10.1046/j.1432-0436.2001.680202.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kloc</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zagrodzinska</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromatin elimination—an oddity or a common mechanism in differentiation and development?</span> <abbr class="cit-jnl-abbrev">Differentiation</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDifferentiation%253B%2Bresearch%2Bin%2Bbiological%2Bdiversity%26rft.stitle%253DDifferentiation%26rft.aulast%253DKloc%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D2-3%26rft.spage%253D84%26rft.epage%253D91%26rft.atitle%253DChromatin%2Belimination--an%2Boddity%2Bor%2Ba%2Bcommon%2Bmechanism%2Bin%2Bdifferentiation%2Band%2Bdevelopment%253F%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1432-0436.2001.680202.x%26rft_id%253Dinfo%253Apmid%252F11686238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1432-0436.2001.680202.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11686238&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172865000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.19" data-doi="10.1128/MCB.25.20.9151-9164.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motl</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rexer</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalker</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Germ line transcripts are processed by a Dicer-like protein that is essential for developmentally programmed genome rearrangements of <em>Tetrahymena thermophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">9151</span>–<span class="cit-lpage">9164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMalone%26rft.auinit1%253DC.%2BD.%26rft.volume%253D25%26rft.issue%253D20%26rft.spage%253D9151%26rft.epage%253D9164%26rft.atitle%253DGerm%2BLine%2BTranscripts%2BAre%2BProcessed%2Bby%2Ba%2BDicer-Like%2BProtein%2BThat%2BIs%2BEssential%2Bfor%2BDevelopmentally%2BProgrammed%2BGenome%2BRearrangements%2Bof%2BTetrahymena%2Bthermophila%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.20.9151-9164.2005%26rft_id%253Dinfo%253Apmid%252F16199890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8yMC85MTUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzUxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.20" data-doi="10.1371/journal.pone.0004429"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Microarray analyses of gene expression during the <em>Tetrahymena thermophila</em> life cycle.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e4429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMiao%26rft.auinit1%253DW.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De4429%26rft.epage%253De4429%26rft.atitle%253DMicroarray%2Banalyses%2Bof%2Bgene%2Bexpression%2Bduring%2Bthe%2BTetrahymena%2Bthermophila%2Blife%2Bcycle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0004429%26rft_id%253Dinfo%253Apmid%252F19204800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0004429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19204800&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.21" data-doi="10.1016/j.gde.2004.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mochizuki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Small RNAs in genome rearrangement in <em>Tetrahymena</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DMochizuki%26rft.auinit1%253DK.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D181%26rft.epage%253D187%26rft.atitle%253DSmall%2BRNAs%2Bin%2Bgenome%2Brearrangement%2Bin%2BTetrahymena.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2004.01.004%26rft_id%253Dinfo%253Apmid%252F15196465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2004.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15196465&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220832800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.22" data-doi="10.1101/gad.1265105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mochizuki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A Dicer-like protein in <em>Tetrahymena</em> has distinct functions in genome rearrangement, chromosome segregation, and meiotic prophase.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1265105%26rft_id%253Dinfo%253Apmid%252F15598983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMTkvMS83NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi81MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.23" data-doi="10.1128/EC.00157-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rexer</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalker</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Lia1p, a novel protein required during nuclear differentiation for genome-wide DNA rearrangements in <em>Tetrahymena thermophila</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1320</span>–<span class="cit-lpage">1329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00157-07%26rft_id%253Dinfo%253Apmid%252F17586719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI2LzgvMTMyMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi81MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.24" data-doi="10.1101/gad.1886310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rusche</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Switching the mechanism of mating type switching: a domesticated transposase supplants a domesticated homing endonuclease.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRusche%26rft.auinit1%253DL.%2BN.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D10%26rft.epage%253D14%26rft.atitle%253DSwitching%2Bthe%2Bmechanism%2Bof%2Bmating%2Btype%2Bswitching%253A%2Ba%2Bdomesticated%2Btransposase%2Bsupplants%2Ba%2Bdomesticated%2Bhoming%2Bendonuclease%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1886310%26rft_id%253Dinfo%253Apmid%252F20047997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMjQvMS8xMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi81MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.25" data-doi="10.1093/nar/16.1.179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allis</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Sequence and properties of the message encoding <em>Tetrahymena</em> hv1, a highly evolutionarily conserved histone H2A variant that is associated with active genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWhite%26rft.auinit1%253DE.%2BM.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D179%26rft.epage%253D198%26rft.atitle%253DSequence%2Band%2Bproperties%2Bof%2Bthe%2Bmessage%2Bencoding%2BTetrahymena%2Bhv1%252C%2Ba%2Bhighly%2Bevolutionarily%2Bconserved%2Bhistone%2BH2A%2Bvariant%2Bthat%2Bis%2Bassociated%2Bwith%2Bactive%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F16.1.179%26rft_id%253Dinfo%253Apmid%252F3340523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/16.1.179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3340523&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988L683300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.26" data-doi="10.1007/BF00284863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Comparison of the sequences of macro- and micronuclear DNA of <em>Tetrahymena pyriformis</em>.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DYao%26rft.auinit1%253DM.%2BC.%26rft.volume%253D48%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DComparison%2Bof%2Bthe%2Bsequences%2Bof%2Bmacro-%2Band%2Bmicronuclear%2BDNA%2Bof%2BTetrahymena%2Bpyriformis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00284863%26rft_id%253Dinfo%253Apmid%252F4218159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00284863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4218159&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974U551000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.27" data-doi="10.1016/0092-8674(84)90236-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yokoyama</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austerberry</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">DNA elimination in <em>Tetrahymena</em>: a developmental process involving extensive breakage and rejoining of DNA at defined sites.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DYao%26rft.auinit1%253DM.%2BC.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D440%26rft.atitle%253DDNA%2Belimination%2Bin%2BTetrahymena%253A%2Ba%2Bdevelopmental%2Bprocess%2Binvolving%2Bextensive%2Bbreakage%2Band%2Brejoining%2Bof%2BDNA%2Bat%2Bdefined%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252884%252990236-8%26rft_id%253Dinfo%253Apmid%252F6319023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(84)90236-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6319023&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984SD94400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.28" data-doi="10.1126/science.1084737"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xi</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Programmed DNA deletion as an RNA-guided system of genome defense.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYao%26rft.auinit1%253DM.-C.%26rft.volume%253D300%26rft.issue%253D5625%26rft.spage%253D1581%26rft.epage%253D1584%26rft.atitle%253DProgrammed%2BDNA%2BDeletion%2BAs%2Ban%2BRNA-Guided%2BSystem%2Bof%2BGenome%2BDefense%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1084737%26rft_id%253Dinfo%253Apmid%252F12791996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNS8xNTgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzUxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.29" data-doi="10.1146/annurev.genet.39.073003.095906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">RNA-guided DNA deletion in <em>Tetrahymena</em>: an RNAi-based mechanism for programmed genome rearrangements.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DYao%26rft.auinit1%253DM.%2BC.%26rft.volume%253D39%26rft.spage%253D537%26rft.epage%253D559%26rft.atitle%253DRNA-guided%2BDNA%2Bdeletion%2Bin%2BTetrahymena%253A%2Ban%2BRNAi-based%2Bmechanism%2Bfor%2Bprogrammed%2Bgenome%2Brearrangements.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.39.073003.095906%26rft_id%253Dinfo%253Apmid%252F16285871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.39.073003.095906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285871&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F515.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.515.30" data-doi="10.1242/jcs.006502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halasz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rexer</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identification of novel chromatin-associated proteins involved in programmed genome rearrangements in <em>Tetrahymena</em>.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1978</span>–<span class="cit-lpage">1989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.006502%26rft_id%253Dinfo%253Apmid%252F17519286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjEyMC8xMi8xOTc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzUxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/6/515.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/505" title="Genome-Wide Association Analysis of Ischemic Stroke in Young Adults" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/7/523" title="High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used Candida albicans Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/6"><span>Volume 1 Issue 6, November 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/6.cover.gif?itok=aUjFCEJ8" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/515.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8hneQzoPATOd5FeK0uzMa5sMc9ccusgdnRI11QUkqGM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/6/515" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/6/515" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/6/515" class="active" data-icon-position="" data-hide-link-title="0">Genome-Scale Analysis of Programmed DNA Elimination Sites in Tetrahymena thermophila</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459197" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-VbvRZyt1cR6RlzCk_ZdQvakbW6f9IGBM8F5U2pd4rAA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/515.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/6/515" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-n0TIkrB94iQhBErOVG2rvb20DCzjCjAHdE8Rf2MniSY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/6/515" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/515.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459197" id="node459197--2" data-pisa="ggg;1/6/515" data-pisa-master="ggg;g3.111.000927" data-apath="/ggg/1/6/515.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Scale Analysis of Programmed DNA Elimination Sites in <em>Tetrahymena thermophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joseph N. Fass</span>, <span class="highwire-citation-author" data-delta="1">Nikhil A. Joshi</span>, <span class="highwire-citation-author" data-delta="2">Mary T. Couvillion</span>, <span class="highwire-citation-author" data-delta="3">Josephine Bowen</span>, <span class="highwire-citation-author" data-delta="4">Martin A. Gorovsky</span>, <span class="highwire-citation-author" data-delta="5">Eileen P. Hamilton</span>, <span class="highwire-citation-author" data-delta="6">Eduardo Orias</span>, <span class="highwire-citation-author" data-delta="7">Kyungah Hong</span>, <span class="highwire-citation-author" data-delta="8">Robert S. Coyne</span>, <span class="highwire-citation-author" data-delta="9">Jonathan A. Eisen</span>, <span class="highwire-citation-author" data-delta="10">Douglas L. Chalker</span>, <span class="highwire-citation-author" data-delta="11">Dawei Lin</span> and <span class="highwire-citation-author" data-delta="12">Kathleen Collins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">515-522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000927 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459197--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph N. Fass </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BN.%2BFass%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fass%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BN.%2BFass%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nikhil A. Joshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikhil%2BA.%2BJoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joshi%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANikhil%2BA.%2BJoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mary T. Couvillion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BT.%2BCouvillion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Couvillion%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BT.%2BCouvillion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josephine Bowen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BBowen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosephine%2BBowen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin A. Gorovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BA.%2BGorovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gorovsky%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BA.%2BGorovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Eileen P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eileen%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEileen%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eduardo Orias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BOrias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orias%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEduardo%2BOrias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kyungah Hong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyungah%2BHong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hong%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyungah%2BHong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Robert S. Coyne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>J. Craig Venter Institute, Rockville, Maryland 20850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BS.%2BCoyne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coyne%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BS.%2BCoyne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jonathan A. Eisen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BA.%2BEisen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eisen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BA.%2BEisen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Douglas L. Chalker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biology, Washington University, St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douglas%2BL.%2BChalker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chalker%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADouglas%2BL.%2BChalker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Dawei Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dawei%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADawei%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kathleen Collins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BCollins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collins%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathleen%2BCollins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kcollins@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">kcollins@berkeley.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/6/515" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/6/515&amp;title=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/6/515&amp;title=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/6/515&amp;title=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/6/515&amp;text=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/6/515&amp;title=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/6/515&amp;t=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/6/515&amp;title=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/6/515&amp;title=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459197/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459197" id="node459197--3" data-pisa="ggg;1/6/515" data-pisa-master="ggg;g3.111.000927" data-apath="/ggg/1/6/515.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Scale Analysis of Programmed DNA Elimination Sites in <em>Tetrahymena thermophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joseph N. Fass</span>, <span class="highwire-citation-author" data-delta="1">Nikhil A. Joshi</span>, <span class="highwire-citation-author" data-delta="2">Mary T. Couvillion</span>, <span class="highwire-citation-author" data-delta="3">Josephine Bowen</span>, <span class="highwire-citation-author" data-delta="4">Martin A. Gorovsky</span>, <span class="highwire-citation-author" data-delta="5">Eileen P. Hamilton</span>, <span class="highwire-citation-author" data-delta="6">Eduardo Orias</span>, <span class="highwire-citation-author" data-delta="7">Kyungah Hong</span>, <span class="highwire-citation-author" data-delta="8">Robert S. Coyne</span>, <span class="highwire-citation-author" data-delta="9">Jonathan A. Eisen</span>, <span class="highwire-citation-author" data-delta="10">Douglas L. Chalker</span>, <span class="highwire-citation-author" data-delta="11">Dawei Lin</span> and <span class="highwire-citation-author" data-delta="12">Kathleen Collins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">515-522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000927 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459197--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph N. Fass </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BN.%2BFass%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fass%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BN.%2BFass%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nikhil A. Joshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikhil%2BA.%2BJoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joshi%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANikhil%2BA.%2BJoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mary T. Couvillion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BT.%2BCouvillion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Couvillion%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BT.%2BCouvillion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josephine Bowen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BBowen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosephine%2BBowen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin A. Gorovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Rochester, Rochester, New York 14627</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BA.%2BGorovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gorovsky%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BA.%2BGorovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Eileen P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eileen%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEileen%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eduardo Orias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BOrias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Orias%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEduardo%2BOrias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kyungah Hong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyungah%2BHong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hong%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyungah%2BHong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Robert S. Coyne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>J. Craig Venter Institute, Rockville, Maryland 20850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BS.%2BCoyne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coyne%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BS.%2BCoyne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jonathan A. Eisen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BA.%2BEisen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eisen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BA.%2BEisen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Douglas L. Chalker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Biology, Washington University, St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douglas%2BL.%2BChalker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chalker%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADouglas%2BL.%2BChalker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Dawei Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Core and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dawei%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADawei%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kathleen Collins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cell Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BCollins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collins%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathleen%2BCollins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kcollins@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">kcollins@berkeley.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=515&amp;rft.epage=522&amp;rft.atitle=Genome-Scale%20Analysis%20of%20Programmed%20DNA%20Elimination%20Sites%20in%20Tetrahymena%20thermophila&amp;rft.volume=1&amp;rft.issue=6&amp;rft.date=2011-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fass%2C+Joseph+N.&amp;rft.au=Joshi%2C+Nikhil+A.&amp;rft.au=Couvillion%2C+Mary+T.&amp;rft.au=Bowen%2C+Josephine&amp;rft.au=Gorovsky%2C+Martin+A.&amp;rft.au=Hamilton%2C+Eileen+P.&amp;rft.au=Orias%2C+Eduardo&amp;rft.au=Hong%2C+Kyungah&amp;rft.au=Coyne%2C+Robert+S.&amp;rft.au=Eisen%2C+Jonathan+A.&amp;rft.au=Chalker%2C+Douglas+L.&amp;rft.au=Lin%2C+Dawei&amp;rft.au=Collins%2C+Kathleen"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459197/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459197/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459197/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459197/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459197/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459197/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459197/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459197/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459197/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459197/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459197/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459197/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/6/515" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/6/515.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/6/515.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/6/515.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/6/515.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
