<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1449" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1449.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1449.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1449.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1449/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1449.atom" />
<meta name="HW.pisa" content="ggg;9/5/1449" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Gaussian-Distributed Codon Frequencies of Genomes" />
<meta name="DC.Identifier" content="10.1534/g3.118.200939" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Khomtchouk, Nonner. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="DNA encodes protein primary structure using 64 different codons to specify 20 different amino acids and a stop signal. Frequencies of codon occurrence when ordered in descending sequence provide a global characterization of a genome’s preference (bias) for using the different codons of the redundant genetic code. Whereas frequency/rank relations have been described by empirical expressions, here we propose a statistical model in which two different forms of codon usage co-exist in a genome. We investigate whether such a model can account for the range of codon usages observed in a large set of genomes from different taxa. The differences in frequency/rank relations across these genomes can be expressed in a single parameter, the proportion of the two codon compartments. One compartment uses different codons with weak bias according to a Gaussian distribution of frequency, the other uses different codons with strong bias. In prokaryotic genomes both compartments appear to be present in a wide range of proportions, whereas in eukaryotic genomes the compartment with Gaussian distribution tends to dominate. Codon frequencies that are Gaussian-distributed suggest that many evolutionary conditions are involved in shaping weakly-biased codon usage, whereas strong bias in codon usage suggests dominance of few evolutionary conditions." />
<meta name="DC.Contributor" content="Bohdan B. Khomtchouk" />
<meta name="DC.Contributor" content="Wolfgang Nonner" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Gaussian-Distributed Codon Frequencies of Genomes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;DNA encodes protein primary structure using 64 different codons to specify 20 different amino acids and a stop signal. Frequencies of codon occurrence when ordered in descending sequence provide a global characterization of a genome’s preference (bias) for using the different codons of the redundant genetic code. Whereas frequency/rank relations have been described by empirical expressions, here we propose a statistical model in which two different forms of codon usage co-exist in a genome. We investigate whether such a model can account for the range of codon usages observed in a large set of genomes from different taxa. The differences in frequency/rank relations across these genomes can be expressed in a single parameter, the proportion of the two codon compartments. One compartment uses different codons with weak bias according to a Gaussian distribution of frequency, the other uses different codons with strong bias. In prokaryotic genomes both compartments appear to be present in a wide range of proportions, whereas in eukaryotic genomes the compartment with Gaussian distribution tends to dominate. Codon frequencies that are Gaussian-distributed suggest that many evolutionary conditions are involved in shaping weakly-biased codon usage, whereas strong bias in codon usage suggests dominance of few evolutionary conditions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1449" />
<meta name="citation_id" content="9/5/1449" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1449" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1449.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1449.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1449.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200939" />
<meta name="citation_pmid" content="30808691" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1449" />
<meta name="citation_lastpage" content="1456" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bohdan B. Khomtchouk" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, CA 94305" />
<meta name="citation_author_email" content="bohdan@stanford.edu" />
<meta name="citation_author" content="Wolfgang Nonner" />
<meta name="citation_author_institution" content="Department of Physiology and Biophysics, University of Miami Miller School of Medicine, Miami, FL 33136" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=S. G. Andersson;citation_author=C. Kurland;citation_title=Codon preferences in free-living microorganisms.;citation_pages=198-210;citation_volume=54;citation_year=1990;citation_issue=2;citation_pmid=2194095" />
<meta name="citation_reference" content="citation_journal_title=Curr. Anthropol.;citation_author=M. Bender;citation_author=P. Gill;citation_title=The genetic code and Zipf’s law.;citation_pages=280-283;citation_volume=27;citation_year=1986;citation_doi=10.1086/203436" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G. Boël;citation_author=R. Letso;citation_author=H. Neely;citation_author=W. N. Price;citation_author=K.-H. Wong;citation_title=Codon influence on protein expression in E. coli correlates with mRNA levels.;citation_pages=358-363;citation_volume=529;citation_year=2016;citation_pmid=26760206;citation_doi=10.1038/nature16509" />
<meta name="citation_reference" content="citation_journal_title=Journal of biomolecular structure &amp; dynamics;citation_journal_abbrev=J Biomol Struct Dyn;citation_author=M. Y. Borodovsky;citation_author=S. Gusein-Zade;citation_title=A general rule for ranged series of codon frequencies in different genomes.;citation_pages=1001-1012;citation_volume=6;citation_year=1989;citation_issue=5;citation_pmid=2556159;citation_doi=10.1080/07391102.1989.10506527" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Chen;citation_author=W. Lee;citation_author=A. K. Hottes;citation_author=L. Shapiro;citation_author=H. H. McAdams;citation_title=Codon usage between genomes is constrained by genome-wide mutational processes.;citation_volume=101;citation_year=2004;citation_pmid=14990797;citation_doi=10.1073/pnas.0307827100" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=F. Crick;citation_title=The origin of the genetic code.;citation_pages=367-379;citation_volume=38;citation_year=1968;citation_issue=3;citation_pmid=4887876;citation_doi=10.1016/0022-2836(68)90392-6" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=L. Duret;citation_title=Evolution of synonymous codon usage in metazoans.;citation_pages=640-649;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433576;citation_doi=10.1016/S0959-437X(02)00353-2" />
<meta name="citation_reference" content="citation_journal_title=Physical review. E, Statistical, nonlinear, and soft matter physics;citation_journal_abbrev=Phys Rev E Stat Nonlin Soft Matter Phys;citation_author=L. Frappat;citation_author=C. Minichini;citation_author=A. Sciarrino;citation_author=P. Sorba;citation_title=Universality and Shannon entropy of codon usage.;citation_pages=061910-061910;citation_volume=68;citation_year=2003;citation_issue=6 Pt 1;citation_pmid=14754237;citation_doi=10.1103/PhysRevE.68.061910" />
<meta name="citation_reference" content="citation_journal_title=Journal of biomolecular structure &amp; dynamics;citation_journal_abbrev=J Biomol Struct Dyn;citation_author=S. M. Gusein-Zade;citation_author=M. Y. Borodovsky;citation_title=An improved distribution of codon frequencies allowing for inhomogeneity of DNA's primary-structure evolution.;citation_pages=1185-1197;citation_volume=7;citation_year=1990;citation_issue=5;citation_pmid=2361006;citation_doi=10.1080/07391102.1990.10508555" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G. Hanson;citation_author=J. Coller;citation_title=Codon optimality, bias and usage in translation and mRNA decay.;citation_pages=20-30;citation_volume=19;citation_year=2018;citation_pmid=29018283;citation_doi=10.1038/nrm.2017.91" />
<meta name="citation_reference" content="citation_author=R. D. Knight;citation_author=S. J. Freeland;citation_author=L. F. Landweber;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=A. A. Komar;citation_title=The yin and yang of codon usage.;citation_volume=25;citation_year=2016;citation_pmid=27354349;citation_doi=10.1093/hmg/ddw207" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=Y. Nakamura;citation_author=T. Gojobori;citation_author=T. Ikemura;citation_title=Codon usage tabulated from international DNA sequence databases: status for the year 2000;citation_pages=292-292;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592250;citation_doi=10.1093/nar/28.1.292" />
<meta name="citation_reference" content="citation_journal_title=New J. Phys.;citation_author=G. Naumis;citation_author=G. Cocho;citation_title=The tails of rank-size distributions due to multiplicative processes: from power laws tp stretched exponentials and beta-like functions.;citation_pages=286;citation_volume=9;citation_year=2007;citation_doi=10.1088/1367-2630/9/8/286" />
<meta name="citation_reference" content="citation_journal_title=Physica A;citation_author=G. Naumis;citation_author=G. Cocho;citation_title=Tail universalities in rank distributions as an algebraic problem: the beta-like function.;citation_pages=84-96;citation_volume=387;citation_year=2008;citation_doi=10.1016/j.physa.2007.08.002" />
<meta name="citation_reference" content="citation_journal_title=Lect. Notes Comput. Sci.;citation_author=O. Obst;citation_author=D. Polani;citation_author=M. Prokopenko;citation_title=Origins of scaling in genetic code. Advances in Artificial Life. Darwin Meets von Neumann;citation_pages=85-93;citation_volume=5778;citation_year=2011;citation_doi=10.1007/978-3-642-21314-4_11" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Shah;citation_author=M. A. Gilchrist;citation_title=Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift.;citation_volume=108;citation_year=2011;citation_pmid=21646514;citation_doi=10.1073/pnas.1016719108" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=P. M. Sharp;citation_author=L. R. Emery;citation_author=K. Zeng;citation_title=Forces that influence the evolution of codon bias;citation_pages=1203-1212;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308095;citation_doi=10.1098/rstb.2009.0305" />
<meta name="citation_reference" content="citation_journal_title=Physical review. E, Statistical, nonlinear, and soft matter physics;citation_journal_abbrev=Phys Rev E Stat Nonlin Soft Matter Phys;citation_author=A. Som;citation_author=S. Chattapadhyay;citation_author=J. Chakrabarti;citation_author=D. Bandyopadhyay;citation_title=Codon distributions in DNA.;citation_pages=051908-051908;citation_volume=63;citation_year=2001;citation_issue=5 Pt 1;citation_pmid=11414934;citation_doi=10.1103/PhysRevE.63.051908" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=A. Tsonis;citation_author=J. Elsner;citation_author=P. Tsonis;citation_title=Is DNA a language?;citation_pages=25-29;citation_volume=184;citation_year=1997;citation_issue=1;citation_pmid=9039397;citation_doi=10.1006/jtbi.1996.0239" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=E. Zuckerkandl;citation_author=L. Pauling;citation_title=Molecules as documents of evolutionary history.;citation_pages=357-366;citation_volume=8;citation_year=1965;citation_issue=2;citation_pmid=5876245;citation_doi=10.1016/0022-5193(65)90083-4" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Gaussian-Distributed Codon Frequencies of Genomes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="DNA encodes protein primary structure using 64 different codons to specify 20 different amino acids and a stop signal. Frequencies of codon occurrence when ordered in descending sequence provide a global characterization of a genome’s preference (bias) for using the different codons of the redundant genetic code. Whereas frequency/rank relations have been described by empirical expressions, here we propose a statistical model in which two different forms of codon usage co-exist in a genome. We investigate whether such a model can account for the range of codon usages observed in a large set of genomes from different taxa. The differences in frequency/rank relations across these genomes can be expressed in a single parameter, the proportion of the two codon compartments. One compartment uses different codons with weak bias according to a Gaussian distribution of frequency, the other uses different codons with strong bias. In prokaryotic genomes both compartments appear to be present in a wide range of proportions, whereas in eukaryotic genomes the compartment with Gaussian distribution tends to dominate. Codon frequencies that are Gaussian-distributed suggest that many evolutionary conditions are involved in shaping weakly-biased codon usage, whereas strong bias in codon usage suggests dominance of few evolutionary conditions." />
<meta name="og-title" property="og:title" content="Gaussian-Distributed Codon Frequencies of Genomes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1449" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="DNA encodes protein primary structure using 64 different codons to specify 20 different amino acids and a stop signal. Frequencies of codon occurrence when ordered in descending sequence provide a global characterization of a genome’s preference (bias) for using the different codons of the redundant genetic code. Whereas frequency/rank relations have been described by empirical expressions, here we propose a statistical model in which two different forms of codon usage co-exist in a genome. We investigate whether such a model can account for the range of codon usages observed in a large set of genomes from different taxa. The differences in frequency/rank relations across these genomes can be expressed in a single parameter, the proportion of the two codon compartments. One compartment uses different codons with weak bias according to a Gaussian distribution of frequency, the other uses different codons with strong bias. In prokaryotic genomes both compartments appear to be present in a wide range of proportions, whereas in eukaryotic genomes the compartment with Gaussian distribution tends to dominate. Codon frequencies that are Gaussian-distributed suggest that many evolutionary conditions are involved in shaping weakly-biased codon usage, whereas strong bias in codon usage suggests dominance of few evolutionary conditions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Gaussian-Distributed Codon Frequencies of Genomes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7fyui3VNuDXi8QunTUgFIdGuA_FtBQ7TjS1I44tzPFg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492878","apath":"\/ggg\/9\/5\/1449.atom","pisa":"ggg;9\/5\/1449","ac":{"\/ggg\/9\/5\/1449.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1449.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1449"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1449"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1449":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ph41bChOiq3kno8o+ieNL0xE\\\\\\\/Vk8Hpi6TLXZSCTpYSJ8YWZpXlKdTjht8xUKRRmCjcsv7v\\\\\\\/SNzZdkq7Bw1yrKnWPf7tDv3rk3WkDpHC258EpJ5PsGi4c71QyLD3jmAcccsonKpxun9GI6l3Y3S8i4WdndwRdQLLLH2lFVzaadvRSuUQl2myICKkvFY7fYTT1w\\\\\\\/utJcijw8ACWs7YAP8RLl3cnYQEkqAgYJvVBK9+AfPyjvVU37TCvOWNXD7DSl2oQOa0xq7VGJqSKLk2GxBwqpcuHOuGHb0ybdZjuMUqwT+FaA62F0KvZmXgwOf5gz8sQMbE\\\\\\\/+w0wuE8+Bg0EQiuGCg3ALTjfVHAfLWp\\\\\\\/7HqtmnRFEF\\\\\\\/txJadgQTl8slzzRTrGpK\\\\\\\/JKwy7thiPmXWqj67sqiEac4t+oMuqkEpJJGhcLkkzChbapu6fbf6FKDmCRjufECduK3kL9\\\\\\\/Pq2nVBpOgiXfR0u5e7hAL+tOChc3GrvmUOyjnb4zOrlA85XoTQEYIhaEK\\\\\\\/dG374ATIGWAqYRs5fA9cjmfnwHSYIxDy4h1nzIWs3eoBZz7+rnPnilR9hh\\\\\\\/obx4tU00i1OhEkqRd\\\\\\\/09D40WetFGieeyeTOgk+EoxDDAWONannNc2D3ykFwaYKInx6fhxSat2TiwL6lINH1QHYk8aSvEc+PVGyACodA8v\\\\\\\/FGTUN+vYjY0RedPijiYl9dmIj7ws2cdL1UKzD4PF0c+jQla0CfxQSVwh1td\\\\\\\/tbHaPS5mjZeUP8HsPmbEwV8EbMLDXxGvIeQ+wW2bezpz6kEwjyxYTTKJ\\\\\\\/u0GbVandj+kK\\\\\\\/H801Dh2MA+akvSvr9iImoC5xLkbtYUm4lZYjT1YprvXz2p31Bp1Fdnb3MdVhERzgC7d2Wo3Z8vL1tVlfYKDYj3EsAzBO2tVWK81qUZ8upJgz05s\\\\\\\/WNFi3AebcO554LGEazIzOQqFZ5MrXtL77NFSmNIOISdIJm+QXWDjrA0qqaSM5Y1EMkDUYBN\\\\\\\/l5GToEsEP5gpJ9Zs\\\\\\\/df4Dt+G\\\\\\\/eRUFfzX9g7MpMmEsmEf2WdMNVyrVf947nJILSRgKbLkw6BbjK7\\\\\\\/BNEiil6wXvkalo\\\\\\\/S4qZ11yfxOzzAMe8RffonvtA1xgntJIhlvvgP7iP+FvTH9C6esNUKtg6zRh9G16Quy3gW56rH1yv248GbfFVRXQlLKTOD4k5o0PkHIlO3uv60leRJ00iQS7sR1CFW1wBa6PAlsmHgqIw3almcJishIWLXqlHOo6szNEmnFgWNt0mwflITQMxP6fwG3emBT8y\\\u0022,\\\u0022iv\\\u0022:\\\u0022RTuEqMTvwrmsVvG74ufF9w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223041519d13d90b30bdf5c82b95259c4751edd0e5da7d18db67de71d68bcb3851\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bjcpsYw6Bp4OEnFGr8e1q01KJRgv49HsmxjH0rFzHr7pqvc9yedZH7hFjOUXVMt4ZEip3fOOkKzqixG2b0+lau2Amaxf92l9wTT9a3OaQIsrSYf8Lzx7xDG\\\\\\\/OvNaITaooEEwQTBm07VitT0t0VdOTt\\\\\\\/KrsWS46tIiHzfQ0cejiFxsJweFjhi638lQUqJkFv9\\\\\\\/yvhpwuwCPLbZbDmBAiPZJ4dU4XbhFg\\\\\\\/bprPABKP+8T+bJ5yAE088f1UnRTn1kKzznDj\\\\\\\/ymMCYE+r3oLJxywF01X+D5mRGOQzqLBz33lKs563XN1V1gwsqVIc6nshvqpxuTtamcistQHVAQBhCAaKSlusPUoxUTc1ziSgLKYeoCP8Kg9TDH2E0LuFdcIYZ0hUE8VOCRKR9I0tqXGhpoIiAKmW7NZL8Yk8fDatKHq18KdbOFV5HVV2BqN5z8H8fbG4IQmpj6X5XhdNpBQxqAojIl47XlK46Dx97hht4M9yuTYGnE5\\\\\\\/\\\\\\\/e5IdN8njsLJ\\\\\\\/sdoYfKCdYwCiTyXSrmOtBT89YV7CTDN0clPh1YSbmD5qqhtejsPsKQWpCddhKjrloAA+lX82tzupwm1HHrIkxovopFyPG8wpIv4SwYeNqiBVoApGDmufBsLY\\\\\\\/m2zQfX8oCIZHfd8Gycq24ApJI0rYXUTtd81D1a6y\\\\\\\/An6vygoKn9d5UmdeWQsx4ioqoTpY3iAHKOh6XGus69RHi7P8lTKTRGwkOVDr0mpZGGc1V9kutkNu03C0fDL9s4z6OfuYAht+uYIPlqcipE3MAGNn7zQwHEFs\\\\\\\/argUpS3lnkZHZ1KuZqdr9ikj1HOZZtYWp0mcOuwEulB\\\\\\\/f+ka87nXBQovAchF3welrc8YmDmPLRbGDhXb\\\\\\\/Dk8+owtSRHhSJklDGIzhBHrF0DpOtr\\\\\\\/X5WLdRfhKllOsTdJRBW4EMMIJrkYiR+j\\\\\\\/i6V80daQsG\\\\\\\/+A1Rgj6dw40BhtOoRN4cbpDzTKTLzSlVsXCzWfmqgCqxDlazzGwbwMfW\\\\\\\/pXWIKMGJZX773nr+6fjNvxkXPGJ+Cefn3eY93k0GuPiIehojfjtJBpE14ZvUfJmB2vBbn2JSXQGBilOWIdKyjLZRO622KJP5BxYMWXHzOYYnGDv9K8yr+E\\\\\\\/1BW5uXd+cB+k7yQa698J7wVIA0fjLxmZFOFnhZ4cALcdCeT91IgzTVOeAvOAkdszHm5\\\\\\\/dzxAGVUulXrN3SoFEFFcBM035dfXV7DXZz+FHZp3nt9\\\\\\\/TcAkBSK8UuhDH4nq7FPnsoU2kOZy2FQxlJh36qArnuZUa8T2J32Tp2\\\\\\\/vD0hEVwFC8iGCVVZha\\\\\\\/QvV3VPnVMgPcHXuskdu8Arh2boYYDTkOz5fIlyIlxjVgoGaNU6NuPrEZ1pCvsdJR8JPilqoFlF+IVYzprKxu5BMJxXmx1YpTj4UvOFAHDXxQa5elD0ATLyFNJy3maAkt9Fvwun4daEf+3cdoNCZrhjgzyyi7xz1fWTExGcAY+R3ZsEPj0NHZ9KSz2m+hgbXShbXOfk243hVaA+YfIneRaiFRzXbqcRdocnn4RoSQ4pMTpedeI9+F+3UJmlcNHO\\\\\\\/SnEqeZT0ha09MwypL4rV+2205ss1YIRSPBkoRUSs07YTCdYq3hmDFezQ2BwBa2hv4d50MSpETP7ngScP4mIzgnqwm56Y4McgSg6OuAq+707rZaClDjEVDufUWc\\\\\\\/e8NybE9tg\\\\\\\/WJscQpfw0XLw\\\\\\\/dd2CXB+KD\\\\\\\/y1+lOx20zE1xH1\\\\\\\/RR6HHyVgHbFzkPpMzLW9oOLR9Otnw5cklgRb+bH8CjUXT4F2ldjpEH9H5hnBIDlypVZTO2rY4o3EqMihnKm6JhwzFRlXszRJFLwbfy\\\\\\\/zDFMDX4xfDPA8Ky0Kk\\\\\\\/7L2TQndnMkuFoLj3bWKyP0f0TpmxNrd3xN8rICmTwXrPxYLvLKfX+AoaMMzFa6enEd6aFgqXfjOyheJu8gaTT\\\\\\\/AGpVyCJRS+dbK4utuXpkyNe29zHyCoZNYkTixrcojf6Wghgpd9yYkggsvRdM3XBTpDKS4BL2m45cKhz5yhmircow+Oc44AnNsyeoy86N1K\\\\\\\/bDBCzY1Zlc4wzHNh+ZBbFCYimff78c37BY\\\\\\\/oncKgk0PY40iVwX4TXDRBED3CJLK8GX6x83ju5ydacXF1s\\\\\\\/k9JZ\\\\\\\/8j2WObhC265WF+B9wFOHIudkibuSTXYkE\\\\\\\/wRK7NKXtNr6G8s6dX3FN+o4Q7tcaLeE6cyv4+5B7QbXtTkmT8xhMEz9e3EGG8nGM6caM3Tj2qyTT10LhjMJwU0wWaV\\\\\\\/IStEffC+61F94Czjo1aSTPw\\\\\\\/ktyrexW4yeaBsdPNTr9cnnRYzpY1hmSE3II0XFNywdTt4t7ynJtZMEqf2pXpDntfH+iNsP\\\\\\\/rwiVdWPoOCsis03s3VhmwmfVJ8N\\\\\\\/RJnSUXfFoRLFzGlgn6p\\\\\\\/\\\\\\\/tzunh9xR1hg0uY6DFlBwcmIAVEZV1TgdFhMSpJ9Rk\\\\\\\/RubU2yGh+aerokx58nkIQNuvtnSE9My0wvk8GkW9sGu0TftwZRqi8nI73UcUppAJaQ4EiS5HcfHQ12X7jYmHPbogvpN8LrfvpZ+kL5XyooO4ZUicchIgyYAp0AXhSmVvlZmFaD3U2prv\\\\\\\/KtfW80y0J3EnvHDRKy4cz0lknfP02QRZ8eHMcIdajfvUxPdBKs7V3cXdZ6IvdiQZL9bl0vQy7zO9+Ss\\\\\\\/Ppp\\\\\\\/MtnSnnTi2n8\\\\\\\/fDl352KEr8bi9QH3iiwsfz3xJZ7RltmL6Eb48wczfKEdEkSvlY7AR0l003zvvIRQFIrs5KZ8r5mpj7xlOpLUzrYnJN+SPCilnwpRgMx3eYvae41OAckolQL0KppMqRBmppoGfOu1V3NE1+Hjsheqa3\\\\\\\/byXwTS8JHu6MqDWhJeVNYmQr3ky4Va2WrB9vuQ+5p8I8en7FO23CWS3nIgBG27zdUfeYDwJKFcIhahc3lluMLv+UouaMw1CKe5uyHfcSs64xoSWI52XrxYXjQ81VrXiHWvlytbCJDovkqEhr5qO3lYrULg7QvSErvsMDaunFURxGJXP6TlFI2X3\\\\\\\/C+R\\\\\\\/yemN5R+XQzFmRPbmb7S2v09TV\\\\\\\/cVpWmwTcRicszvYBfVJSlweOd\\\\\\\/1oMO+DLvNdY38FAKvSnbFgHmpKuj8pwVDgsLellx\\\\\\\/sM1Max0mpzohwwCr5OzlmdSat3xpQ0pvuayAwm7CBgM6uBT9LFF\\\\\\\/SAm3SsopxjTuVpji2feIzYshk3IQhe\\\\\\\/AsAMvJum91gmVKMzMasBPDFS+yUlqFX3EYW6c5MnBV3UM9SgBTPtyOOJ9m8nOH4hnhluMrIxtqcf8jKceyZ00XMcIcnf+WJR9dCiw508OKcQ2p0q3vyaHtotBqKLzfpN4XbndMjQmr9CSPO\\\\\\\/NB2ww1W8767D17XI3APKSwYcUWaxIVsMNdspukttf2zTuHNQ++YBC9ZWVqsFJxovc1TUggTHkUhVC\\\\\\\/8o\\\\\\\/C1n+vrW0jo6wrzQZIndql7alIpiegqM1KzOHikNkGB4ZjuHtuJ6oUlpXu0r9UIdJn4nwsclvuSnLzSgcQMg5KyiPA8SEy+HAEGiXx+x88qQBDrxcTeIYwqxDvuLeEgZpY8Rumtn5juYBUw95zTBpLnUZaI1RMWlzNu0\\\\\\\/oeLX+\\\\\\\/BClqXNKUF8Y\\\\\\\/3IL64Nr4QomkkuSNmJX0qzAQIW2qckbzVgGzpB8ceL72GWyf7+eP3Y8fCQqzdJPUjhCevFETKyONVyRb5cN7+30+81\\\\\\\/5rBM+Is7gMKRJ0d\\\\\\\/iwoUUKO3Mmi2G2G\\\\\\\/xY2c2aSst4kFmI065wtyIXcXF\\\\\\\/8zLIIEdiLqWRyD4rjSuJ1+S3VDghGoQd2a2OTZP9usxqSKTC41clbdFPHoyiXuvOfS\\\\\\\/dH0A3dlJCsQ1jKg2NObXMcbE2ZKXDQZIX2MqgGvGiBV\\\\\\\/vYZ1Fm1txNdS+4HAD5BSs8g1rJCn085ggJo49wD32DwHq+YNA\\\\\\\/LnqH6v25B6k93Fgj7\\\\\\\/9hu1XtIM29q5kqlt4nbJ5baVrG2zd27T4vKDmlEKMSHbaxCV6QDuPGb31sxGI54V19AZ0QqKCF74Z+GdEgRCwsxb7WBtlguklGXLbHDycrehaCSIVy\\\\\\\/+I508v09w8jQtn4CZ1fxQR8ZeSicQC4aZ8\\\\\\\/r4Nqi8UZ+IbXPw8fOWG49nivDFI52ti86YMWS0raYLwxAOytV+ay89A6DyNL09LdWxAQzQgxrYUEM548vW+GgKVphy9mbyx8uUvWmvKy3IL0w4aG0OfXdSJZWeYxUDYEscv5AVDBtUH+Y6gat\\\\\\\/WWJWlTGQXCvXhTyqBE+77Lq001Gy2zA5PpnPuRPieA5SBCTkzUGAk1MTkI0S+rkdeSd1sbTP7V8RTThm87b4oKyj4JwpB2P\\\\\\\/YSFvADM15Ar2ryuMwo7s\\\\\\\/qH+cUwrYvM2I+\\\\\\\/dNHDQTSeEwx6k3SgOlYsPwgyo5iSqnrxWjaCisbMKVjMl3Q6S6UfkY3kxr53gpjSFKlIAk\\\\\\\/h1Qu0U4oO13L3Hdr079hj6mZn1DWiIGDE8YpQREs\\\\\\\/PmAILNjZqx+h4zJzXxo2pDSBWnjBQu8rdfG\\\\\\\/wFaSfqFpfMwpvm\\\\\\\/W6b1egFAd1Zsjkz3N+YQvbzz4IHGiZPWveTuBW\\\\\\\/YDN2Wdiuk1VOaLL0OYYggP1ZCxmFLrPyDI4Z9GMC0ZdwdaPw7HMCzB8IiSKyB4769eHys3anKx\\\\\\\/nqxQHQ7tl7O0lO2BoKTJav4zxYS7JgX8N6gYIef3Q1OMRzkypeSLW7Vo0F6P9VSaU0kCPrI4TI6zw0C3\\\\\\\/SgmFt5Ol+3LU2AAo0CSKMjx28M4kiNFqQKr3EeCEsfS8BljkgMw2ziYt9hEt2xDheWmjVFNJb3hQnG7hXYkuEPvAblA8WHVt1vb8yCa\\\\\\\/KW3alHWy\\\\\\\/TCH35jUD3qrFXg3\\\\\\\/CdMUyJ9P8iSb60zdOKBkAUdk4u8pUg210RBf4yuED81CRXR8HTJYo\\\\\\\/p+6tXnhfuHvRC3JB1ZX6mKTV6nlbZUjVPy2LZGLKxiVxQ8rf1RKGhA9Tks5hk6ZI5PAflWCpWzornokdslsjXLV7W\\\\\\\/RjrjqNlTmzMq95wXda\\\\\\\/DgFYry2ajJES9WRTiWt8AyuOLA6+3KOcs7WjavTsOKjKAhIXXHbUthn5c0ypdy8ulxPK1hK6kVEKfnWE9iqEoZePQtThvGat0TzkSeqf8\\\\\\\/6hJ5EeTdinCWfjWAO\\\\\\\/SSPeqPByjMWX8hY3PQjXevEu\\\\\\\/cslmR6vs61XORuNh0pjpQhnj4t4Syj4d0SH\\\\\\\/2qPZIUeChLq4S9Js6tRkb7y8BytZ8IazbjhQs0SVdVxwSUn3ReMur7TChpH6GXSpQIFQcwR612IgV9ZhRb1tjsI7KfUOqS9ag9uWVK8v8QFoXyG+C4wbN+zv6yf\\\\\\\/8TMY\\\\\\\/MTsV2naiunnDiOLKMjMAjeg6THVuYlGN9RIOAfIUfvW0M+VHz8juicKN+RFBH\\\\\\\/ZEz8aNE+iRxCZSRtgmy\\\\\\\/kYppn79\\\\\\\/tfvnJnweKr2KMO3PIwU5lryCZzg8TKIWKDZPCBM2ekr\\\\\\\/\\\\\\\/7iw6RtkrKWddKBfQk\\\\\\\/39gVy8RynjWtLLLV+CN+aqV8+r6\\\\\\\/RLnRYD4Lf71FfEyTNJabxoYwBdxGCNl+DjPGJLtZHd0TvSfNw\\\\\\\/6kjV2\\\\\\\/lq03AFiqR\\\\\\\/GrGfSowdPhRXXgVQnko6\\\\\\\/H8FcP+FBzIX52ZRVlEW8ttBD\\\\\\\/Z+rcFQ6erhMZ76fOE1wj\\\\\\\/NUKvTv7pmR4lWbLQaDP2r7MlvWdHFrUuC6M9nBlWCaCh1uAuDdeSFlpBaDsRUFe5HhQGgn5IuEAom3FZI1npMOzN5YoKG1JRCzOLDnMAOefol+QA0u7jilK+bWqEKBreoLo2DTWTAFsRlsaBOIjlzg4okvisI5UbAn9Xyg3dIj9gor8uacCPNFsinKd8yXADtWSmGBAMdqE\\\\\\\/70j\\\\\\\/0pQqwBeOQ9ipm1l1XaxUxSb1oGwHsyPqqwFSosZhFRWWQq4A5Z8GH7PHuL6CgknjDFpz5WiymSiFH\\\\\\\/jVRpnTjv7HPXuEIlJvhukbyDXTLKvxtZQSwuZ1mSRB7FDpJ0N8X0FX+yKcpaEYTyIVh2MQGAm+yInlRkxmz\\\\\\\/31yGXSkddS\\\\\\\/+ofSi+HCCOskF63+CTrsJTAxCbiTn1mfjGphdGtDvMpOJkb+FzaAV6xc7E3ldmrRIxLIAtLd0pjHtOWPW0E6PpfLSMwTgr10kiH\\\\\\\/sODSHQZHEHRddjzTxgEtvfyEvsOqSGmxtBhiQTx7a8jsA0UOdDKIEsPOy2F5mHxa8GhgFtLAoAhdLZmpk6NPYKvOX4J9MBw72f5oEyif2C3QlW29Nh\\\\\\\/biLeZzwWVIJn87tqqwYJ3PUdfX3VcRiMhVBsRsKJz1YVtJanww5mbVUin5\\\\\\\/VZhXAy3rPiMFzq0ljXHwK05\\\\\\\/mpAmorS\\\\\\\/XqUws9sk+Yx4sV0vzBUYNIkmqh1Cx5f830X6DYbW1Y\\\\\\\/32iAmHODY+EW7f8aGmwXCcfDHbK5Q4B3vEB4z3+l8f7H7YidzcU0BE65\\\\\\\/1NKhCTx\\\\\\\/RtVfnEgTwquI\\\\\\\/FSk8yzIZreiMCp8p4IaMEp1KBzGdMi6U6RgZ\\\\\\\/anXuqElJfzOaVTRbEdLFNu\\\\\\\/Ft\\\\\\\/JbVyLpjCEf1SM2\\\\\\\/wH8Hkfmv6Za8ZPhIIOYwt6IM1aWbA99CGwJufeHtsegUq3+6r84\\\\\\\/DfvMtEkJ4mOvcP+2QP8RjJ4Oc05uj48w7vF2vVfSN6p5VQc3U1eoagdE6BfGFmhYxAoM5Il\\\\\\\/iwafLXqeIY5H0K3C5b6iqTNla20Zs4gfSyNjYoLZdbreSAX4Cy+x2x3agLoqtbdewWl2lEPSTotHCZWgGIQOMhJt+K6eCtG2QPBu8nQZR23sUdllgrYwotXvWjhrfhFXnG6N\\\\\\\/EfyDmOXrPcr0cbuD9BOwKOH7wBine2kBBZ2dAd4LXlNmxIoMy04j5fhPSdEa+T2VhME7tx\\\\\\\/RbAT4Yy7lxySIaav54q2U7zWAlD3SriShkyMzULF\\\\\\\/JNEBWxJ+mzttGieeuphgawpQZninnASkjeNMEf6TVTD7YrUPu0SRBRSBsRKmLWSkslDZfpz\\\\\\\/38SxsVV29krjHBk8wwTSiK1Y+SQlH4ukn96+HtEmvFEos9BCdNghDQFqgQW+QYerU4hFrV1bWNwhIQoumjfnM84ZYGB40N+Q8nphiX68Vj2igw9kyoaeIF392I2erDlL+CYKAB5nyZKsmjErG\\\\\\\/xnSiv3x69zuvecGXBCwwUVw0QVMAzim3cGrXZKRQjPiY5hXjtc+Id8GFRiy0IbnP4WItowOYpAzhSi32izNHkyiy1V8bqxO+y+gEuGaP1J5UJ2RzF8\\\\\\\/o40NxSkO+FAAaabTumwdzc+m9Bu9t5QYuWDaXyp0D+ZECdayCD44lvLeWndSLmkYzs8wXGCEzC39lRacIQ\\\\\\\/lYieum1E6KBvRtwT1I18JFH\\\\\\\/FoGlq2vvQoRBw4ydmxJsvk3hM1m5EAou4bQOjPfncv7m1qEm1xxvrq4X+S\\\\\\\/HXzavMc\\\\\\\/haNeQwhrr2Wxo4rL7wUfQxVbBD7KddmhKZv5ZMOkg6pDxpmnqhB0zIJ3kqNEulU1yWW6IApEn1odR0+klFhiwW2DbokYYJCz4QoFzvLXp0vQYZZIecX82WWewVoa9+B1AB6TC9OJAatoU7HtHKvEj2+OE9nBqAEf89WCtL18o1OyQJjTons7BQhrmFYHL7yG0XGPigBUiiTDQ2ACnVundJNIJp53tiEGzXVNmkRbkMq4x0dljkj60kph7lio90XuG6BKzs7oO05y7\\\\\\\/wknalenbPgaYZdssRufaQM3oeRkv9Cf1nXvQH+ZudZ1WGeUA3G8HyElm\\\\\\\/evzj+wKc5QigpBTJRMqFfrDljw4AkbeHHik2Ktttt09sW8k7Au5lXELG\\\\\\\/pKnh9amOUwNdO0Ff15peU8xXvZhzlKPPpYZBD5W0K769qG\\\\\\\/3XqIzcqeNsoV++DOe4TEPlsBmpW1Y6FEN4F8YG8skyzR2Xyou7NAIBUraC9a13M\\\\\\\/73RemqfmZA4UMSqH\\\\\\\/hjxR8aeIAc5fsn3B5xgVPGncXQgfHkIP5kyJ5TrsF0rplVaItZN+\\\\\\\/A28IC5wxhzMdc+kpnOtoBgfH9vhbEB3YqYP9RHJThhCpCOUbBXDFKjc9IVfVfT0+0FtqhZZQczORyRAnBf2U37UxBbeG3Dbj73aKFodzwMWnoZHHliidJ\\\\\\\/serrr+QpsZMFgH1PUinKFKcRZ7H1VFCkDdV8dT1YPBK21wOV0OHHM4RqbtghJ48dEow1k4ZTQyWsC+yp\\\\\\\/Vkw8Wjn3HWDzHgCl5WDMiVp2\\\\\\\/22Vg4W0fhFp44pfjx7eaVCf\\\\\\\/Hmsth+GBOLbac4LYIpsAVID02Hd02iFz8SgU8DSUnUgYteJ4fter2VxnHzEe9yRf0W\\\\\\\/vAhemansCnvH71DLTW9xXnRCs\\\\\\\/Cri0VoSgkjbHsr+s23iN5gG7CUuuJcU803ZBaqaj2ZZT6ZdpTtnG+S66rk3+Uf1ZN7a2taqwjDbl91Q4g3ji7aG+aF4znmaJkIo4sLusR7jXgCOxkm7IRlw5X1SpKfjFQiwoKW0v7C8FPp9XRK1tnJuj9Z77FrK99OyDgpNeSH4+uTur2JMzsKtB9QuNsCMEoOQ74srM6MCQXN2VDmk1iuQ2p4d4n1TpGT83BXCSyZ6fjwq2Lcti1YWNsiFeIAw+dLqh9P0MH+jMrWreAXBb\\\\\\\/uxIKGIRML\\\\\\\/ZUA2z6cF2TqQC7dpf89fEN\\\\\\\/ZJ4GDtppA83SZ3VAAzF47XKNaVEd04M0SvbVPaUZncmlS5S8QOy9y4MpcvVPzxRWW7r0AXRhc++kKpIkgRNjZDqXtxirMP02hw9lLzFCbvLhfH5hOqdo+PVWfLmd9L3X7jP43plNn2EKp8R12NwiZVnYab54D23zKBSyiMJ1sWPednJ5X1Ug6WfmlrtMZdZX26FprLlJMTT8ZTgsfjmRGsj13bg5MyBP7CPSBOCu5n+FbfQh3e5lrUb+LKhQITO71I2IzwdAfN6a03r6AWoRlO2QyWMPW5SRN8Xi3LrUs9MB6BM8lpXkVZzO6onivd3nsrj308In7mV1mEnWtj1aBZoWcVOOFdQSS\\\\\\\/b52iyPajcE4GZNTfjUWAEWn6tiCIYQe0fIwbpldBTMXp3SdGay54hSxFUvA\\\\\\\/SkQF+bRVczIV36M8MBmkFnrBjfi8Of\\\\\\\/8qrArthREwHParfKpfGKF9SrnE6r8XMcnVWUsHM0k4yIdavPTBxS9zvok0UFJSPxKxCuXVCxx2jucW6npOVISad0FwbU8W0ZX21QpxAu8PBqT\\\\\\\/4nQak5hVDWdx4DBfifVzk1H9ItjodL9mP57mYbu6Uqwg3wI5D7mbKIecURlv24pXWAgffx2PhHhHVeA4BFN1VkN1lZDT5rwHRoL+shBk4Gy6FlGTcUVO+UzlbVMsj3LIBpYx6bNoib9jDR30CvCF0VecxrgzXKvy54trK4VaKiH\\\\\\\/Dg+tOmfYUNPBmwHYZ0g5Rlrnf8aCr+oCXy60WaOggKiUiS1soWAu54cbkrkdXb26ApQ7L7akidZbyy024\\\\\\\/Y3RXXg5zvKkOx+sysKYF1JzceNH+aSh9DtyNy2MMwJfYEG18iCe2OoVh8mfe84Hz2MQZlNXsPTK1MEJBxk0P2S6N4AVvLA2XDVZfES01diPbuRQ6ADa0t7Mf132Ui8d+57BYIgcu9ObEat+f+PzAkqKpKFGuCJfBVE9f2dTG3MqkZXsEq7PALRLKooW83j64hEcVOwKXw0s2lmiXPswwKNfGQUZnESnIBehOLqiktKa3Yi6xZD8oIco3qeKv3VsecXttMTSCfx8HGBSdTvz7xEp9X16trImUUdBPUOdzChYm277P0Dw2VptuWd\\\\\\\/J0Vq9Fyxh+UX\\\\\\\/ncoCt4LyPIx89hl3uwuGGsEJo5Du6Z6B\\\\\\\/2MTS5SqD72TOtxRGV8HQM5O2B2QeZ6Ro\\\\\\\/Xv\\\\\\\/cZQLMVUpUUVThEd2iBs1UYOPop628+F95bEVZWPxV7TwaR2Y9Xee+9DvnKrMQY9JY5tbwkk10982O22EPoEqiFBZOd2tSLdJqGweho83bAPD8tYRd9e0tETx4F7NocaVOODLKbcW8CP3upLF0dGFz4b76Yrgi3pzDDPkYxjlouGVDNmrTL+BMxkK1yjQL6KfXzu\\\\\\\/cXO+eiKrxMsOg31o5nIokRmfKRp5xeJXKcKMbMkVCbKbraM4zrJdOFM78rcPvb5OwetiJ+WkmWcRtmNQUhJkneQ3yaA1ZXT6\\\\\\\/+uinLLvwY0hQLklVe+sh2Hv\\\\\\\/0qSwTXf1oGpDVcxJQ8ABPRHBiLSEsX06DzDGdJt4a6PggCvMLEvZnJL7WlQipFoj\\\\\\\/Ro7phu3Z6jI4TdYrJ6sjZw3sj++7k5uGjql2LqzPiL0qjQ2ayh8qhG135e+81l+ASylLt1HK1W1qBQD8g32S5lvSpxlnmpS5oChGtvAjM1FiidAVEWlx7r\\\\\\\/ZnF3yxI2O6K\\\\\\\/nsotRce\\\\\\\/7GB6NSNZISviSjQY8aMvQCcCukW0DBULLagjQ3s4ZxfsbdfbyZxQzkzx\\\\\\\/7BDWWjThrhLSxVY7FyUJlYp1TEy5laiaS+ZOLQyqss238ErBfqxVt2BfnV4buGxllGbt4PNJH3hOkSBoGYV+1KgZDEBGoUxXQteZwEL24lcnPK3vt7mCYUHWAerveiJFZ4Gl6Ba7ywdO2y9SMbeNhBbZrX+WTesdKb9Bna0w6DZsnWdyyhH\\\\\\\/AlO1jiQKmVXLLAr25wVlLQKtme7KtC76wosthP1fNje6H1B2YdjlYLFi2tjizOyFaAGggOA5RPlF6XN2MfrNKYiAuj6+fAnTuMJnGBL70cIYhjKUOtHqT7R86CSfC0VIBtZqQ8KzTf8mVUrDpdxrdptbJZmXUnIGs2qhyx8qI7yO1wGGRpNcZWVSc0VMsIxmTE7s+0MKRYIBbPXSHy29RA8CXeCLRf5+rQbaO5lyxxNwiASudY1D2Z84M+N9qaniquXY5hrb5a6HwPq+DMkFboPbpx01IqsaEretq\\\\\\\/imr+d\\\\\\\/iXbjSKtA+j80LcQ4P6hPMpHWQcrzai98MHIQT5UirB3q8L9As5Ku+WT5T0la71ZOijg3BNFHXJleSrDTUl8hyV5NSiRxBh92kbvRIynuR5g2FTv6i1XV0uiX7RJUaimHq+MMPvYz7qVdBHYMg9Nx8j7dPU7yKFqlX2gERnECtYFMj9J2jSoCMKT9ahrzOYFSwSMAnp86xdkAljNY87orvKo5G4ZCbu1vhBVn08oZztKqB3UBi3qUYP6sIHJ4SuvUfwQzKafmCrSc8Fa9pXkS7c8dLrqgH2ASsRxw2peqe6f9qLM4xxHzEIRVqH0gw\\\\\\\/VYz65WF+lEfS31kviMskCFHVMxmBmqQQ\\\\\\\/+aJD1tR9pYJnR7Tetmd25iQgNozyNn7n0zdGMiDZ1X4uCA7sCvrHR1Q4y3eFOLUEILLZotq2s5wcpjI0X7JXW2n\\\\\\\/SpL3P7B4EvNN2OgTRHvspD8GvyLA8Oe2\\\\\\\/a0YFAVpXj8B02aiIgsjLYgjzZyZiwznPFMy3c4bAToAiJHu1lpVxO+mbu4h\\\\\\\/IccvR35Bs5FUikCvabKeBP4bGhDva8nnUitJMUO0VTPYszVboycQOuecnhm\\\\\\\/z5wG3mtUVGAdBtvbu8yXZozbmW7QuUTmYFzOXVwK2LfC9YDecKRpsTzu4Tf0u4ej+CgfwUdXPrA7xmVzgTHfRfh8XzNdiM4X5wzon4cfFQPExlTwhgfsFa6IxHYyu3hacfvPuwxufbU+9LwQRYlua3m4TUbgBHcdTKEW2ggnP+laFtS9FoqUJ4QIehr80GCFxtQShojlH0gk94j7X1LUOxxZbumqK5ip7kRaS997YSYX\\\\\\\/yktQPxqFXnEdXqzz88UA3gpJrQ1lnN4yroEN5nAxYBipgMmWsLwjAV0z+wMpOdoy25oOYUYzuTlu7Cu360nMyzOc\\\\\\\/OEsKEpWW2qKyKXM8JYqsFlBlg6W3LaXGsuOy2S\\\\\\\/w0pvjGxm1VHdzWKePeBYXTh2QWKPsyGYkdKEg0wX1YWh81En6j3kwq7Jx\\\\\\\/haj2N8o8rOwkwdT+TzlQ2sQGVVwzQEWa8+rCcWTk2BRdICxcMKuFs8Q6JPqCEFd+g0fPEuKy2UyEnqdmwG2ZcbDIgIK4KQOrMhZtITbTwrrYJAJLFzzoQ6nKBLnJXLA7aSRUMz93zUwAZ\\\\\\\/wKcReHhnar2WyCObzx0ZzQXMKtWRqsN8oUWwY5u0ZHPgrvzb8aJ8IFZofu4UtZX0tA61\\\\\\\/iGzNuU30D6O6jLk2NjvuOdu05dsV2\\\\\\\/VZ+iypZhf7EP4zYej1U+8ez9ax6eGIWnQBJQumIyjlMigE5uSjd+wak0iXD3n0x1VK+3e0Y1\\\\\\\/xKtTOUCkCVLWLIJ+O+jeBeHrd8oWJ4t5jSM0vf+C36zzzqHiwyNOp5lL0hsF5Cg9A1G36FIFa3Nt64urw08uQmCYzBqwFVVG7MJdNbOTWHYey2v5vbe7KPc6FrGLkfznj9jOIuLBqzXOmV0R9WPYgic8dio8HiDiXNM+WBl6mx3meCV9mEaRCZQfypgxl7H7PhgP4054XPpSH7TN9DSeHFRCP0w0otAnk75VrmqmUuM+vdZHxLdLTH0PfAiyEJENecBzHRB0otAKkJ9fNoWjdHrMOTLd\\\\\\\/vTVecq7oHo+kI434fp9JuHxBGOwVgjbTn3V0h5CyqRkTUMtjlcaVtH\\\\\\\/ehi5AOhTJ+y9jomnxmhjEnVwKphZd0KnX8I2S\\\\\\\/ZAYIKznWSh32EVd9bMbor8bfMw9vhBMLZk+D++g6GdfeBfUGrQO\\\\\\\/R7QQM5zPHohjDZmZSWsxIAZ2s98p18dWxJUeX6+xYa1sb5EGdwIZeZFU6ZTA2PyCJItvcgx8KlwS9SDjgzDADIaPbcoTh9KmOijsPkPSTCv5QIiUoAQFE7hD1qoLj2KbC0UX7Q4CBAXzMmkFVybvXGcX\\\\\\\/wUWmctvXS7+mD1IaAqhaURhCnKVO7cbYqdknRp2GKez1\\\\\\\/FQb+XR1ZJ2iYxfl+rnMnTa48UnBBv92671xWCwYz\\\\\\\/\\\\\\\/CJgobZnWVgnx8UrsFC4TX2DamMidZzESyabxt1oa\\\\\\\/y4c\\\\\\\/mee8LiqJ76fSziD8ArXzP5yJO48tshobd9HcATL+1iyi3dxAGKwd7WJfDJOCRbMAHNYa5fAVflWrgttGsq73gLzMI0p\\\\\\\/jWhGgXLS3dgjqZork2C22bY5mLfISUTwhTVoFpOgrD27RzaLzJJY6xclz3xZedcYFuAs1GPULX7vjGhQBWOBAkfOPmsFEAQtq3DRGjw7XqmGBKHZ4SW1URwSiEo5ALB4NJLePn1vo5VDOLLNnWGOeLGYtOxcPVZBx8eelX5VH5PVoRTxibSEO2VINC29V14SDLZr8G6MXif8R0SZRbFLu19lv0n9\\\\\\\/zjJwmIrO\\\\\\\/db2fUigHY1BRwvxzQhyFQQGlxz149hZlW0M+RDvim53H8LX6vq\\\\\\\/T9Z6KDYnm8G5vEngDZxLUUBevT6+pmwRvSc3DwleAHolDFJUmyhjPpqZokMv9LZ8uPw5im6HQYb03RGPZmhecA2h3R04xRMhXVAtm4QhBarUZWX2rD1TUFYuHTzGttHITVr9yTWVY\\\\\\\/uvuRSi1hbXRCUmYYkYDKPlm3SgsiiKpr\\\\\\\/wlgcwSsmmMeAdriy5EekXfSqTTXRTHfv0QX7TNcf33rmlxAvklzcP5XeN+qnGEoyW7qXkMfSJpz7kLQaRJ1Pei\\\\\\\/Tj4rI8X10e1OSI371W28ZjYO9mabBeYBr907u7L+SKbrGZwq6Nzd90z7QIaeUqgCZpqlbPpGfvZk3PDRtXzc6peW\\\\\\\/G21CWvDFn5KFSQr3EbDGJ+r+baoi+Wy5bpc5UgfmJnRIytuJdmsAqbTXQ8n3IXoWtNNVpETeWtd+ja0zrNU+PPwZOb+ChviBtdGkHHD8CS5o2EFl7KU9ver6iSB5s0zd8VZ\\\\\\\/5GvmyBkdCHakr4JK37hCjm1RPX1OdpeV9W5zmXhTRWFZlHG2z4YhwGBLnZ1D5X\\\\\\\/AUfPzQCNBMCBVWLdHH9SzNz+rbBFZkMq8cS0yroFxA2ai+ZeelKjjpzK3d\\\\\\\/seOhYjItH9g6znefktn8tZz0+DCMsgxlbozxS0FwDUAGEBSq+XkdUtgsV7XOU5eyOUxqKhnFgew3w0LrD7vQV7OIOb1eNmJ8xE3qCERGLIEMfWgDhJWOml0vuYT0mf1ltfvq2or2MtKMeqoILLx9dfcdUqAkDLaiUfwTr6oxLEaL3OLa4nShrzfSA\\\\\\\/jyOkWNWsY6KtBX1Lc0TvjKAZhLKnTXDZTcN3f9LuYzDvKitqj3f9QsrO6I3ZK76nL0WdJmn1sdw9\\\\\\\/oyjN9I1pkvTBawbzV\\\\\\\/EfcfhnetDd8w5PD9YjTGovGIzqWVvrBYBCA4mNtOLl5Pd6LG9Q\\\\\\\/cDSE4ZhtpTzRSoVUywsUBW1vSo8aITTrNqjzNZDC57VOpUXtx0ljX+FGexNKqpGYbV63koPXIcRhrfV6gPcHxUnlfzHzQADDZqWNtcDyJGFFxj7K3hTViYxtoHE8N1xjDJ5UCPOj0HyxRMrcvnAYz2XsgZqoOI3DHwzRgYY4Qlj2ahALwT\\\\\\\/O6qokWHlRVhwSpsgGGPMa0ehPDiYngazb9\\\\\\\/WZXCSRLROhH7aTNKYLabU0aleu26pGyNm\\\\\\\/FK4tN9B7ERMTQ436JGiWx9LiAmtN2aqHoiImtlYRpCe0HdxaYCYib3YdxTXw9nEKT2t78vNp3xGYXborooKYyim+mGczUUfvyKq95056KOt5O4gRwgCDzLolemQvgzUvTAvjQcY+04gv3RNB66RGLvLnx6biJZDXcvskaWweYIz9kfj6Ad72mPwrBFW4LIHDb7qXCOoBGA\\\\\\\/i8eGzE8NI7QNBeBLjX1vZAR\\\\\\\/D3sLBSOIeTxwix+76qHd7zZfUK+5ThpbQJk24pW0F7Hg\\\\\\\/gQGNeOrQFXbSq6Yka8YyPM6z3zWYJQLp6zdsQmJszlrhT0kC3bI9Z2IpOnSKl0qtQkinOB5iDBO7WJklboxNbpgwcPL3zpXs7Wss1IprEbZdS5FIaDK9iwEUfcRTk0kxWycw8eJCqH65cJrh7rxpWPiyw43eCTzCpuF24XF7YmXsU6J+SDX2xid\\\\\\\/KuPg4GUfL5CAXb3JBoX+FYzTr+1CFm4UsWRfeoGsfTH\\\\\\\/PqBpcC3QUyCwP6YmlE1HSJ88nfc6HTLA2KLXNkTANWG5ZuQamlqAsOWdYRY5kV3Yia06ybe1WcF5Ta64NwG91zG+jlJNrgfhdf5ch1Gh\\\\\\\/5gl4Ge5zB7SJD9KsOnEOYjVUHFpIZKB\\\\\\\/92Ei0jwNkwV8s0mO9MYjUmL3mdWgdSjo2Wc\\\\\\\/UklxTzyPMDnD+rQ0ufNIwTIm992xc58p7VoqcyRalNXCERSKH1FHAGmOQOliqXxgXWuuSVALFobIv0RbZrcEdmCg5AC5xLI1M7YOYGCmaZxC9gXCJXWParHJt7PkLr8mjgHfsysa8ubcNHkV7aMO5Bo5On0No9BrykWC5wGWZK\\\\\\\/UtguRsgWSMaBtzn+3xd3QDY2D2d\\\\\\\/APXkmzICxs5qr1JtHv3zHtnoAiqE0JtElNkCbpZ0JoX0\\\\\\\/uLB13OESg8znj1HrsPM33QTkCSllTgTQR16HAyfb+Tqb9R7siBNMhJJmQ9oLTcGl70hULuSEpyoVZQ311GfmMPaAkn+UBC4+BQzWBoOPXwXfwEBGfWUYAlj+v2v5BingKPYE8nuEbkvtstzrz3WXWc+nBeqS3zzLjIdgTUYyrxC2udYKvOEekBlw9C8OAmB1TtCeAdyEFP3pQQqa2XhNOH+F3CSXTRHxX4fVdvo9cF\\\\\\\/UJar4R42aFOz9AGirHbrigjpPr\\\\\\\/XCd5l3PVR+u50j0C5BcO2fDXUn\\\\\\\/C3N\\\\\\\/FO2qUXWcq8Mt+QLgsnZfheZHxjcPF2QktJXYO+w\\\\\\\/WWNT1j+GVOK5YF6LuNwrMnRhsT3RlJeQUmdLqSgvnneDpyJRzmhlkUnmRJAnnpdszJRBdsX2udmAMhP7JZAJ2B9zDZQ6uwBq5OHWJNj93HaRcGUket8NaGqzukA+s8SXRBv7Y75t4hnDGdsohN\\\\\\\/OjQWPQu48u8UbdyUJnf\\\\\\\/uxG+bgR\\\\\\\/+AUajdGpVxxu2t6nbbj8VUvAHqlLK8NuYc0s9fOt+yXKPqQjWhZAQF9kGAvM+54BeQWqUqS5l2kJS\\\\\\\/ZQyBpBnDxhPpAR16xG6Jr0Ajo4wdnXNoUR4hH3ciWT0U128QZUlCezQiG3AupWZpww5M6QKq+XHu+To9ql80pZmqaj1dvsBS+uc81erJS804qN1LcjnskxG5ZFD+pI2GuZ8fuGtP229cJlzr5EmRhd7JmSMEBSMZZzfLLtCLEPHZdeCWywbZ3wrJh8oMdzOn50UVRCyaU036v1KHhcJ4vXes4ZE5GH+0VrSHDJKw19qE8LbrJa+MEl0EKC2p51aunndxL\\\\\\\/jQHlSymJtatXWQNrnb0wWthopuJmbr+3fhj8Ylc7wbVEUnAPaO++5lw6QYH6qToLJTsncJqqmJ99Gow5oIjiquqqaRJX4c5D2LWESUy0k7x0kSmaSzXtX8uoYvplkMylO3hqfWaYQYm9UJ7J9qWKtEy1Ebg38D4oIsrdQnnHs73GU4mOWKhKVWVXDFu5ia8NF6VTEQLPMcnQMUU9kU54UnsuQsz5g0KL3ArtBgZ1xTeFU\\\\\\\/FbrE2rwGfi8AQbNj\\\\\\\/+WRHwdiQFPgdmAvT5lOmI+jm4FNBJImdZVf9him6h\\\\\\\/xj2AIJSWXt\\\\\\\/a\\\\\\\/Q\\\\\\\/hnopIVxqZPnxmMRVr2M3Cq2xTrya7T13FXYuJLW\\\\\\\/JUUBlENfA\\\\\\\/fHMIre22WGFQIloIdbP+HT3k1iYne0GLYUJi\\\\\\\/OAEgrk4SRzqBAQFAPz+0mjkLNQMZB0jC\\\\\\\/swCx8VqyUrmgeC7SZcuMkRClHbYNbD9sWvy0q\\\\\\\/AqsqvEnKxPDmJ3VM\\\\\\\/L91cFMWjAhJB0CW2Yp3IoP9zEdKInr2UJx\\\\\\\/CJawRcPaGniZbXiAAQHkf5vjwfkHFwhpGxbr\\\\\\\/AxeXoRMMhcUMjmC0ks1ZYCsGS225hXH4XCw6Q9K6H4M1zA9H6kc2dpdERgq2Id++XH0Oid6iwJQIRQ+AEHi1Yi1\\\\\\\/Hgvfpd8FotXY9i3tirca8c1aYZCEoiSIDZY\\\\\\\/xLeYGVh3tHCjGtl8LrF9riRrQLtcQ5qMySfbqvJcSvFMGdQ9HRsG1YD72Iq0r3ggueTFKIaYFYOzsdtCRbt0Yn5hWpFAmwk4KJoH21JMyanfVSZoqa6\\\\\\\/r+NpH9oJvaUm5JlWbQasaVcKsZ1sGcrYF4ETvNHq5tXmxlxj6GwPTkQOvBohGG9lOYn2ydRA\\\\\\\/xRGQYje4UJqS30bycxxnoNciDqGDHYhR1YvNICPzHljctesc2Eyjgmo\\\\\\\/oIM7u\\\\\\\/otbHNZHOxZnF1Lt8zeezN4gLbIJHZTeao7bpUlcpnFfA4TJ5nbWlA4Pz6Bvt04ct66YsopEdYe4QDl8P8UrJ325FokSZHcTPZPwCZ4KKyuawg+Cmap+9qXn++fa9fjK0BBNbq6CutYuu\\\\\\\/BPKl9kThL+wgLusVF4U4i61HITFDrG4eo8XC6KhRnyzTaDc+OqoDbFDpXaIhbim6Xx6ff\\\\\\\/vOe5XYWKDEk0cWoe\\\\\\\/gw3ffZAx1HYrkpN6ev8mjaBXhfCJsK45Hf23mx\\\\\\\/Y\\\\\\\/GJQqkzB4Bpifzu0DKz+Pse6C4EPv9dpNp\\\\\\\/TVenSguFF6Ox0GrVVH11F0B0fdUOpZVqKKT\\\\\\\/Le4UDcsjzysLIDkz\\\\\\\/Gd0kelYVjPrQ1\\\\\\\/z1g\\\\\\\/Kf2urI5zsTjY6HMXGod7\\\\\\\/53bO07OoxULhG02ST8AcaMaMbAV+MkRVPmDmMh\\\\\\\/TztzHC4neQM\\\\\\\/1+81u4AO0ybVj0s87yL8bTMXUz5F+LsJmqiPtqhOTw68NlEEuhEPKlssMOezmhvYYojlDrurzLewxdWVWNVMEafgRkEA4bLHmuoCy\\\\\\\/IFLvgCzcPi2uhvWb\\\\\\\/i2SpJWA1Au\\\\\\\/AUg1MqLVbgjK6yg4O3xqlJ0+7LKAUZkCpNyxraEndquXqb\\\\\\\/NGHq43i+tObdA8i60nTzRLyV4FTdGahA\\\\\\\/F5pEj7FKAfTjFsT1IIiNrH\\\\\\\/Ey7HeWp7GCl\\\\\\\/l\\\\\\\/F\\\\\\\/eeepowC2g3SSECf1T+oNmGv\\\\\\\/utxD5iQBJoZkdbFMDjspucyTX1OsEpbduvkl47zXIR6mE9KKPyLOUp42n5gry2vXt4DpskgjLtKCd1wclninFL19WdcolIH\\\\\\\/G6c0xm6LxvnLs1AdQDjeSSK9sjvMvkrNbOvOO27ekWh39QVqJraQ8H6iaveC4GrsrhStEkmSpCOpGCnpGtB4xye1jdheOwvi0brlxRm6b0iwPelhdx8YI1oNcrxCiLrimxtHJK4vU+tzv\\\\\\\/TDLNnOhrbB+GkmrtDFg3iXcfFeyrT7ObHplwvb7yoE9vrWsf98NuNZF4YvEy3VSrRzc4t7lKCR2BN+JP9meFWgEUB8g69gCfLS0pNIvq9hEzsntC0epbvCyD69fmafE9WjDS1nOvSspyK8g6skPJpYMvK1z2s7q1GSF3heLpOY6xy005aaAWV\\\\\\\/84TxKl2xJUUlGweD9l+OmJ6zwaiZd2eoR0gqwEtLccUKnluZhVr6JbPDgekSa2aRnPdaD1L0z+2CVOc5MUGZNYTeq9J634\\\\\\\/PCeaDLzuZugNTGNsv3uexhxkKMgTplHALSxHmSjtfBF3m+aiVmcqAVHpRIQdbjdHzZrmxlo06e7XN0bgj74NKR1hmf6xTGXYeNngHwBcY+qZLrNOXiNLFGMrYVlTwdmfLfl9Q0SSjKnuTP5ByljKubbJFQAGx45\\\\\\\/x2OiGfGq8FEoJ3T2PTYAnMiQAYzPLj0JBwaBDB+z5rs\\\\\\\/a\\\\\\\/HR5e9jdsLJzVGLxt8WUVg7J2O4FqIc0+PZCQu4SK62KNJpAUz1\\\\\\\/YZsBU48DVCtNSUL\\\\\\\/eMMmLpE01S4\\\\\\\/F7lMn98rv0fHbAlrWL\\\\\\\/sqV9I3rZdwPsKtGqKRZU\\\\\\\/gT7YegM61T2mw66AvIIqQ4EUB+ayUGmfS8X1clGjRDJXJVAa7\\\\\\\/oIgbfg2A0angSGOTvmnP9kUwgiLv95aMeU3NBR\\\\\\\/RfnVYOP9XB9Cc5CJIPup18Ff58OpGnkFeTKxmDnlROYNNJhBeCaMftUnN5uCDdRO90BTZG6SEqVS5YleMM3LBG\\\\\\\/1hvzqCCjsJ8bBiE8zki6vfc2SE+\\\\\\\/zeURfz18B1zLioVGo\\\\\\\/4XSuZqyVCFtvZ6I0DLBV7XyaNGBR7FaajUVn2RkQzCID\\\\\\\/nQKNIQ+HvDHpeLjIxh0TF+yAhfuRDT8SNumaMQedG7rhk22MbObAz5sxe0oqXNCi3A3VT8w4Dtccw+6pLYJXAD5FYPkXxR7iXD+p\\\\\\\/ByHIVYNgWfHkhLwFq6uYSXwezeCJeDCXJZNa\\\\\\\/isKJDLJ3GKtgeOz1\\\\\\\/M9iRjS8+9kwpbrtPW+ewnRmm6xcK9MKz5uZsAmJvTJBF\\\\\\\/yddxin7H+NLm80wPT1ReU22Lb7+Pu4Jo9mqZ\\\\\\\/dAUE+RyXutfigPyWEwFtW2C7jL0udsV8CgGwf+aYhkC3a5zq0\\\\\\\/aWrs2UNuFaswn3k3K94huuRG2VY\\\\\\\/n7upAOiQJyQuyaAJaozlFwZrKMa7WwlMLjXtzOktf3ny5DH12WyYSurI\\\\\\\/QrHW2QOaTnF2YOiNqH+Qd1kcQfNqizsiedPwrMDyXGEvGOMe6Oq1NtvgJsKI1xSHWTizuEJx7tjoQNvOMveYjIy8xkH4GDjalnLvkA90VmydNImBT2C3+FsWhrynqmSeXn4BouLSf8gz3CgNyyLiYMvb138GA\\\\\\\/5kycqoLHI59v1IrJ+wN+IsjCkFuwNyLQUAELBMwfJLEq1GMfASAqF\\\\\\\/16IujrBIYTzaVn9XdCLh1sSuHBywAxooo9pOxNMMGhqJFiWaVeqXYadChB3zoW0uECscmAcprvcLKh3mjAn2CGYaTNZwqhM3r78zbq6CEg35gDVfah3Wad3zZNU+EckS8S3x9hHGFvvjgal\\\\\\\/Ksz8lEswuor+UZNmKrLpfkVW6wP3Tz6GgA6PvpQ9q+iXcgjx3Pg\\\\\\\/cEusCfcpsMzl12VbcjxMgZuYoPycsoIoFF2\\\\\\\/JE9T\\\\\\\/luvUSoIKLsG+Xz4DOhjRdOmbbkKt0siGJ0px5QG1PXdAlYaB998GpEGPqP5tFHXYyr49+ZqRaDOf2Euqlhu2wunKJEjT7FKeRbpZT+bxlfog55O\\\\\\\/Prl0p9kiPZfzMLVWSr6L9zRWlNpwFanZZJHrWTtECC5l75B1omCCGqKDvm8Hhc1aC4hxWpNMeypF9g8TXOsZIFrJeHLEJ3vE63pV45+KSD4dciwOwIta1NDl7aty5++nDWSDbVZP5+V4RuEX\\\\\\\/jcicV14C3j1hpKRPYuyrQADqIgwUmWDcj9qamq3SvAwfR76hJCmRcLTgCuTieubWUGAwGrOIHIU7iFxrxk\\\\\\\/uS4wLy1eZLACjuLKvQWfO1GOCTvINhoBrWYYYxM1xGr4PUKmAiw4Fb9ypTazBv4fh7Y\\\\\\\/kTTLlIz00GjZdsZnSlxa1dC7H\\\\\\\/8rwheh\\\\\\\/WrdMqQbSHHxVz0DV3xQi4uTQzo1dWIiHVZFCZ3NFK2I5tPjXstVC5ZPcX+9Uqly83r2bYQyMrJIAsGxYhHPeUvgpm39CFv6FmyPBEMeNbI6XPUkyD6O8+56I6ftb52VhUT7QbmPD+MUCyZfWTL+PE476rCjaQpKWgxZVRYT21qDvGTeP3etWoZkQZ1BcFGtAToSDlvWpmAIIEHWdiKuY0WP45oxACi1uTf2olBU6UYvFjaPj5ModEoZ5FP9t0e73BmS8wYSYSkrsfCnwu9zQ33C6fYrmIAhM63mTlOo44vxFpc\\\\\\\/B9uem4G0MxtqN1Lwji87xjmFgh6FWPuyT2kjxoEH6WuXb5D2p\\\\\\\/amBhQlBJfE56GeepuqeZ0RNtfma7ltMj4WbbtMB7W9mEJdrBtzop5fPSv0FvXOiqRto7EwEYACvaN1OttqJ7YSd9HptsZXfwqrRmZ7nTUGwiYdS+LNuTrUHvemi29lH4wcVdrR\\\\\\\/Q\\\\\\\/KsyTCIC82QBSs8ssS3ETThxcGR3Bh27UMCwimJ2BUjlGmOXRwisFJajYFPMx3PC6ZXaSj0tW8B4B\\\\\\\/MiJgSxJQ6HPiRP5MMnB3EGFRDqwiaMmhQRIkYJ40yRPa7llX6r8IW1YpiDPOQCS\\\\\\\/kvG8wR5n6leVNcv8N1u3MMZ7n\\\\\\\/KkkTuuW5B\\\\\\\/K5S9wH+TYabk0hbvC+Lrdn9HLAoAPQGhChWOgEPV5yRtEbqNjCkW268p5vUAEPvxGAtu9ypuhMshWC0MtMlOxSH1ilknhcFZH0V+24+XyQJ4oi3ffciorpvq94Ei534xFZHpke\\\\\\\/zg2GM7InGWoCrwUPAG85GIHqGRx6LWOADqm4G\\\\\\\/eDBhxux56CmTQ0BvcjP4Y1s82rAFwWFP841Z0heIspSJz+sfoqu3MK94ItnqRFlgIWAShrU++m8moW0D1Cw9tGs\\\\\\\/lmSisp+6K7XqABmnwV0mge1Fso2DvFdFvfh6PNf29zZZm1lrULtrQTSAkLnAbZNf4OuuxifGWYNzVSfSTA7jEcLJ15pnUsolwbgxmb4v4wCQyW8YF6WMn4aMkR7ASo21rM4S21DtXqnaPLSy2hgCHTwlile\\\\\\\/pQvaJ74FIsacjHc6hv+0vUnsrINX7EQ0TX+losNcxI6bS\\\\\\\/Cb40f7f9z3trcA54gRUmlcN62Jp\\\\\\\/GrSHA4NWsIApoOf\\\\\\\/kVHbZ4Sfhr5xlucgQoFWp4oCI9w6eoEm8\\\\\\\/5123nxg36pZnSFF+D1zaKqG8S2bh7MvsGW89pTKViow45rAoo7vbfVey6ymKFP1+I8J6yS+edJFTssTKFDkTX0iQtrL8qW5Rx0sF505ZYxyZBJlwBtIbv38XgZQUZfZ39pz6l7CyN0GQEWgAu7L5SHmvMWzxXSFtKzcMAw6mtP9C8jSxa9atuWmktV0pJbWyzGF3jIiuRBVFA1XPIA9sff7qh8IneJUjgb0xe3bpjbZAJNOSVrwtm27eISo1JPt51ZuTArtQiBjSRfQNbv4fc4anpE7N1dz9c6UhHGugUY4frY9a1VUIBcdqv2hMCkNnZ9Z7iYxAFP8U87\\\\\\\/oKxS0hlg8gdKzFpvpIdpbzwWWGGjy1ghe0TsQ9ZVRfiOgnpT5tpO5dooZrHhVkBu04qH1yWC70P4Pt1bknMrAXnSVLnTd4UiKW52uf4tpZkajznuXvRS6RTTHDmE+9HSOVskFJAuUOhwbPgq5UgJ3uFBQM0tsoewIY6C7ophqtopzcaBIXQf7zvuac6cqUWRqsBm83wirXtNpiuIvC13XKQNwfC30M4huwpuLrwNSzHfQQlqAKwsUcS87mRxqk+rZCfRxyeG5Gv99mkmmux3JfTOK7it1rwywcnqx80gluy4nWkHOPN9UR\\\\\\\/VWYIrojQgH9fqVEFfUluHszi\\\\\\\/IB+GVYIJ6nD+XmC\\\\\\\/S7PCEAgWBOwS7eu38ZUadDsvgp1pwinHHudwlPW081Li56zm9ZBTnqTmJGVifuIfXVSiPcmN3e0psL+szxoAuTP3UDB0wylWZPKEgEdjXynZGerX4\\\\\\\/xf7np3RWdQWnFeV2HO\\\\\\\/jhyTENRTjAjTRgGtbwn31QqKBFBL9CHj+pw2AiSYQOiCN6QeHrWwQHA5wi9JSgGFQ+w8TD2c7foLi3nP8JJjNLWOjKFcdhD6DMuVVz3MhitLtW51Ex0VunBj3NY304IZrUtVva\\\\\\\/2TOMeZF3aTiTZ7oFhoC\\\\\\\/nV4LFisU\\\\\\\/q\\\\\\\/e\\\\\\\/y9lOWCM6uhCylMkxEdJuvtOSy6bZHy\\\\\\\/1bEeekVhDFWkQ5MqmEp\\\\\\\/8Z6QP16zx\\\\\\\/2GAzQegkgmtE\\\\\\\/29Zu6C0p89oPLvjXnnF8uhpuLLjTXPD\\\\\\\/VeKQthbtuli3G\\\\\\\/cGMghAmjBXLzMBh11pB4eY5BAYCkGwGEeb93KnIXX2Fw8\\\\\\\/kl8lsG4xc\\\\\\\/f1enONrMfjKGbj\\\\\\\/zvTco1783SuP7ItbuTrOfXiio5aQVbzjrtYCRZozuystMhriDNfK2QBWPio9GX6HU9wNThaeLtmKeoy3Mc+E0wHCKoOmSmVxo33BGZqd\\\\\\\/5FiQhy4z\\\\\\\/fV5voxf+YIyvgGgeOPeSXk5jn6kSU5F3n\\\\\\\/Nm7Cx2fm1noG290eUmkqEyrvOHHJoHp69gZz2T9uekNlxr8KoiuVd9v61+G9jGwqD\\\\\\\/rp9tOh8HQZyat6TYolgZYsFXcDL6bIqY3sa6+GqftCQca7hKnDB+TKny606dI6AZ\\\\\\\/UVGhCvBumAlfnqaYD06OcLdYvd0YMv\\\\\\\/VhepC3g9TMKxHrv2G95tGhpTCeLdjlTlBSD6LnJoX7YepV\\\\\\\/BjkCKokeSEWKK\\\\\\\/3QCAfxxbVqaIziBNuClyFS0o+00ResGClG47WxiDf428cxa2FthHMvSHTb1R7opVAiB4x84K2Imj\\\\\\\/o7MC4mSa9PCmI\\\\\\\/OABLw6VZqd4emqcbCPTAuiGAzKXjb1ZjpIDAw2M5\\\\\\\/BsK\\\\\\\/I3bvnQ857PGqlegsaYofiAhmb+r5Ng0fRkKg\\\\\\\/5ScUARDmqS3F+gTofWKr\\\\\\\/nOLfhMpvshzkGZU11sz9adR2U3eM0qS8+JwF+Wo5kVcXe7BvEPf2wp\\\\\\\/GSaryu0yJAndLyiYN8U3fGbPD2sOEKZCmYYCuuGKyOJo6XJZQ\\\\\\\/DX\\\\\\\/aRgT5Gsc30WhUCOY5eFUgIsAYPXEv58VZfYV4FMtMmQ42wrovoPoUjIAjmiub99lH7rCWN7LKq2HsQf9rh+AdK\\\\\\\/qVO0tma5Mb8ZcW1UKmC3E+SVwkOKj0Uyiidn0AIGF0fpclOK3F\\\\\\\/v0VxFCHYbBDFQmsVe7FytDuq+Ocl+P3rurp\\\\\\\/6peAiTTsxIaDk13+piQlLCiuVHCnHE5djARMOvS28MchVjGaXw684mzXFBIzUPzFnzj+lShMAxRPDWwc61EI5ANy3YQp3JV4JhZH\\\\\\\/NbREGPgXgLiGGClGrs7MpGv0Om2Fz\\\\\\\/EPuGNO2QtcktK7ZPxPBfOGUb2GNFculjRCvlbI5IQNqXtx0HqTQ9sOEBwwBtdNKkQdD7oN54PuoxsjCphSOMfOzpyUop6wJ08KSR7fZZrS6PDggvRt8iHHZSX\\\\\\\/uSwHH9BKElCervakDwSxz+Bj8QnioTWIjDi\\\\\\\/ZpbR9DKpJMe4kldc0qHVoh8X7xtRfcSpaLNPaX2crK5UOXTwWIXxWsqP\\\\\\\/JoFGZ7xctk2\\\\\\\/JBQB5hwu2sREH28uHe9FVleuYE6AmpZelA0IGNAzzEpVLL7nUe94tomvmpe7UD5lkCeOMj2tsWz0TeEMtV79FohAjIodcQs0cX3nNe2ooM7CUIhkZRhCY8+FbH7tAVWD4pSJHIRC8daDg\\\\\\\/76jpkkpwzIOP4caXU17KDnmr3vWNkNlGyXF2wBw5PNb5mgK\\\\\\\/GLStUZPiBfph+MoR2Dy3y5l4lvUmxv5KI8YEi+ggOIsBvpGsep7Xr9NWIhadbFnOwFiofFTHFPAax6Rl7VftiUL1nIw9sCygeGZWJVLTK3MPK9DDHIobtpLg7wm1ssRmbd6MV4bUeP+oEFZ6gI3g9pZGvzzf9d6OAIB89CtUQsyQDvt+2gq0Y\\\\\\\/6yW02Z9np8c2kVKFjFc3P3nwi2mIlFD3z5ckZfvYIBsRuQ5O0me2vr44uufbPcIfjIjGUFp2kkOY0HcObX0pnHsyCqvmLS+fpktMspHfMEzb2EYlyxJzrFh2aJU146gyUurbZwoXAEsE6j0P0OrAcgNoP3MJsp66BRtPTntSVgfFo3Fov0xro5Vj37vWQlyEOPFK72KDH3+Tz8LIdkcltvRiSJOq2XtKEmNZGc+f1o98UNW2g2Vn5vfIbx9arW2geQJ4uQAhCgy36UxeGfKHxCzUd9QvAEbpbOikq1BWla2GmP\\\\\\\/yhBEylN7lGacq9r37Y61fD67Ic09XOO86zs5IVR2PCaHDSvrLRu5VFyJ78VEAfzm8JmQuslS7UVey7Ar98mUVsWuIVkyXRTJlM5JXG4JC83vEcJ9cRuJsqqBG\\\\\\\/nfBv4g7C7UKrmQrYM0Uc1h00bmA4iXFC06c+szV5gjAseImDUK0ldW3dKGXAvvV9NFhgr\\\\\\\/z9sdWAUtBLQvfxWDA+jq\\\\\\\/g4q9BVcKkFKY4WwzFXVPnVwo0gKJ+gpGmsBzYIauAg5QP7JhZ7fSdHKjIe99v6Sjoeip\\\\\\\/0saKSlfxFhy5mWgRnIF5VgnsKy+l3CdJuaGAs96keOXjMUZuyn14RyOMfxiq6mdY\\\\\\\/lHgKZEK6v8UMd5kLCnnKekfrVd+wt\\\\\\\/wIrRLPZw8ati8LseW9oH9Q828gbxm\\\\\\\/lzyYc4JYZ1n0lZVbcfSvPbuz0KH67FAcNnqqP5ORws4NehFbDdN9\\\\\\\/MGs+Si0bcIrPnU9L0TJ7rxGQayl4h7BS3g8+tHlcosBdTGkzR3b596328pA8o6THvpkuQrOFWloyKQ\\\\\\\/mlKVt40tSTnJHVF63J2nWYFSJJ32ifoP7rbRxtCEXFV9BbcBwnINnYoHyZN0F+t2uSrm6crme2FGjvf+qHASaHT8bJydOcm4+zTQUqA\\\\\\\/pz4VXpTaIQJoFFSbxEXI0KEA7SkGIOHYlzkugmMcwqBI03AUcVcrK6DCk6ZgB+UGqsGwBNHjQE\\\\\\\/eZT1PDDoU2VMht1d4dFrY3hPpuyFlXyGjn4QZWgZrIorMoJCRfmhRc8L4KY8\\\\\\\/k8m+pRLXarAfVGHiqPv++tZ8pKqD+YKxOC0dHkxOiLopyqzQZd16cYtjBivfY80VOo6zOQuUxf9owCnZbhGD\\\\\\\/A35xJjP7P\\\\\\\/nNN+gEtMatHGHMSXN\\\\\\\/dbIvRdGdCo3H+U9x7SKjghBiS1\\\\\\\/a5wc6QLT8ruMLAmv2326UMK2E+ulmITU+rnAYugfRjkRsm6MRUNDE4BKoBSie59Ussj0+xurBz6AHPrEPzXaVwQl4P9j0OSc\\\\\\\/q6h9Tir2joH\\\\\\\/7QkPJqLKAy0PM5S2e2onFaUUXGoQwUO0vZgVc\\\\\\\/0N8w1cL\\\\\\\/IKwilrJdlmQUWZyoI6OownqUNG+jWzu5J5ARf\\\\\\\/6k3n3s+rwZ\\\\\\\/A6UYxCAsxObawzZHwNkUvuYZ7mPOjY9txYFecanFgnTsAgZW7wLZz5H1Q0usDGeE7v+UXazlZJRrjxf8Irv\\\\\\\/BsHRaVwUCNJZvp7p8pcAGymYSv57w\\\\\\\/HSM6ABIwMamGXutLIXcZScY4drxtUqSv0xPpo5lW77wsOL8v\\\\\\\/Pb5+sy985EH2YrdbH8dseGpvS2Vo8ONIz6v9rlTdHaan3UKdrUq0iHOp+qVden9pT0JijxY4hqv82lt9iU5FnP5e0vTdb4CgFbLYn7CdHWKrbEetR9sSpXGdenQX0o1kkQKTo7IEx1rci6rvBrgHPgzsLh5sJJmv3tCrSuYnyL6YqREyVXy0\\\\\\\/Ko744pF9u0n39tC4E6QePidvaOSgmMmOJeTR2JkhdC\\\\\\\/6KOjd6MCXE\\\\\\\/NfYpDelo8mozxLTyusmpYB3vosGsMisNHF\\\\\\\/cyJB20hQeR2dX3ZtECmII2parMpVdPdVnJM4oiGgAuEsO1wM+dX5+lsxlI318mPn4dBYUVPkmTnzOpip0j2ch6QjRQtnwIhJByznVinU2O1zmSU\\\\\\\/hp7kYj9BTHqtOcfZvgEwoKY08sLAfJDmRVSmDjOqkkkLN5OIP09w\\\\\\\/NgrNYWm4LnQmyAWiddkh39oqCOfyz9SIOjNMRaN9xBnmgFcKZUs9qVzeVYjWPDtigd3NlelyFbbJbRP2h\\\\\\\/mZUFg0Jfp8P0UtZdBce\\\\\\\/iaLRrLQRYJGDHF8+ZZfYSEqcRWzeKPp\\\\\\\/pT18vZS0ApTjEcSyWCIFVOFIN\\\\\\\/fBpoD1ihW7s8TIx803bUFXcnTYpd+u5NeZYDjC5LhwQxhFeEaCe+ajQ96KKzNxakOIELEW+OYP0j21AHmT0LQosLT64Av0C0YqrOiH+Dbnhmm4pfh\\\\\\\/p3w8hR5qSLb5ui+Tasm90QDtjOrtOaOaAl8P45VeWxZch+V7vNhtxsELP7yPcBO1oRyVo7uzok0yG5uemEias\\\\\\\/e8U3KEhXt7ZaN7XNJkgC8gKDAyQEhFPWjC5myx4CZY9+O3ihslNpbA\\\\\\\/95DRvEzPWQgsVJsvb1YHhfBlZsfRFil1AClm89iRFt5W9moM\\\\\\\/+b7NlnFI9Q8SZkxGe44t0SjzJ+iaCByRdgFPLzRndRlujHf19YkZCUAAROq1kjVfrK0HCp5dGeWzi1dU\\\\\\\/ETaxg84W8XTv15mQlvdZrEZQj8RZzI83oy0v+0PVC3WsZ8rMs7x7GbWL0hbPuzpbtbzC9c3m28FPw0gsObPPllRv+s4zYjawvcxNooZhw65BN37bLjjca2zJ80YDjbo198vvawcDfFuLD70cDol1bsoFXWUlsYzdn+cR9tSqXsj8mgFgbl\\\\\\\/peqV5XC4hMfdFoMNwQQI+d\\\\\\\/Kgo0igHpYtQ5zYmCzIC7Ow35HSN5X48wutE74T2gXKqhFoFfXeoMZ1C2ZQ+aYBM9WpTrNAlng8V95kN\\\\\\\/wbydDacA4gtU1vyWHEXTJefvnwSO1ehSpTJIqGMI8\\\\\\\/wXv1k8bGzsPmk3Q0zyQPxSVSzrbTs0QNAqZhFW8nBgq+E50GF1D48Q9W7t9nwODJwH4wO\\\\\\\/djqnBpaeZjlseHrY542NgSOlCgZYaW\\\\\\\/PomPxmDTh9odVsA1AKMXuepob84qu4\\\\\\\/+bSBbsIeAodyiMsr0WMWZG\\\\\\\/t9a5dEt9gkhTbOsJlUKX+MbYdxy7kZ+ATjFlUe0YK\\\\\\\/PEpAzqZX0Op\\\\\\\/FKhf+B8gSeO+\\\\\\\/LLGS3BAGCemfy+kpZbpfSqTe1e3cKrfMGyH\\\\\\\/qXA2fZ+YLp3zjlOTXUhLCkH7Ca2Mx8FPjdh1lvPbJkmp15b9DnTnmrAKpLM3dyz7MV9wz44LqU1VkwsFIME2a+KJZNXcIIk+P+7rV9unw+AnBH4+8BM0cZI63wbhKT6W3xihUmAsq9K3EF4ivUTnxAG3Tyji+eF6z4FPYXffjz6Z6bPrDM\\\\\\\/bUmoTorOu52LjtUOOrEvdKvCS0f5Qt3D7H7sndzZ99ps94b\\\\\\\/ZG5FG5mKHx9JwR0EIwMrq0uN9z2+MlkhUCU6w013mJegGSkxNMhSi9NZ6oJSOtugXJThyOveTYWRyM\\\\\\\/51Zb12eZi4IzhHYVa2y68GgJ89jTQiVUu4vjaCLtKYp12+3sYiMKVztodCg8mPHXGKVa221HW0qC3emzUGKI6dedoOXzUg+hUaCHx1XZUkls8rod9OZkXnlUX3FIDGSjvh6I+DzBU5+4kjLowE5sY4T0x62ZcpBll9aMoxVRxLeg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QJTkm4DmlEKV7nSzNsfS1g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bcd4f38e5966e8b0f63455ffcacfcfaa5a1fbaa92b39b17700eaf19c3455b759\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wpGU7ts7oESHFSfoyH1keBJBpxgQM5Qx6TEf5fUqdQMNQ4t3M3eYdulsK58ryXbgvapb0iMiuCuiSMRD98Qo8kZvqZ1qaO4nxrXHMCI3rpJ5QtZokoLvS6XnYVrCh9lw2u+5u4MQp\\\\\\\/D0OvTFa\\\\\\\/X0EJaLXMi\\\\\\\/sFxqz1naKoue0fGw45Gi0Xs0s3Z0n3grj0qBZeXTRv7mRg+n\\\\\\\/npkZ2mnG5LwSWNwgtvMVG4OPBhH9NvgUpH+nsCXoVChgSrs4lgyKMINFFhp9l+z1sumAPJd0G2vfpZweDODFIpGUzpvp0HN7fpmCuhhxml7Yy0CMn6uOA0hMn3EWXGsRC+sbqKVOaQwzNANKX8J+vHBzmwxv1kDkLNzU+kBjVyrZguRx00Ztkjij+GQh1HoHvqQ0EuCFoLMDRpShGG+Y4fTx0H6zQUedWgVisdVCF1TX+jEAETKceNjeTIcc9BPHqPxDq6mjfwGbSbJ0H1pKwIQstTbJSyks7grwsZLlsmZBv+Eq7vN2KHG4t5UnY3LZbsKnMv7YnbAYVpVNAZ+PoQYFRFGSMRURDuC+X6T6NSO+O0CAGvBae99teoO3bgZxKzybNJn5hxFV54DhOdu2cWpD7YianRp+oOu\\\\\\\/ss8qXzfKn0sotgBum6cM7awifAPrvJ4t9ia+EWDXekoBXYdRT5K+8ggUD8GQXeNyq9JjFkrUIEYMRVn48WRsHF9d3qLq+yIJgGO8XAza9DFtVrm1NW7oE0YOoh7Osgfwccy0bWfw6f8mpVyN2h9V8Y1mAotX4SwaZ\\\\\\\/Tgi7GGqsCOW6FD3cvdiF1VwP4JmWqANLcLCR64hWb1Lvu4A+Td\\\\\\\/6d1+R4fB45S3MyzS6iJZd0nSdIbTFTde90yc8zzZD2\\\\\\\/61Z\\\\\\\/uzRoMo6eLwB3HJOVfbDBrYBLH8QyTfPsdcRk6kKeJhCXSePTuo\\\\\\\/FGrseQK+xenurOEcxN1Bie3rgP4ZROuRuv4TEHUNPLshnIHbSVrSO63kUoXT5SB0mv4kgmSlnW0xMNtCxYSOGUOjuCOmMYmVT7O1Ofl3L9gXOPuU9WpY\\\\\\\/lc7V8pWp5ne3cCOJ4bN1MD1vS9q31eA1UOaF\\\\\\\/+eCRF12P0NU5VvnaIHv17dMCtapavkkeURe7NNJzXwQAp2XWaNg+hSClnO3QSVdX9HYZluHIaz166a1nkA6uYWn2A8oi1qcCQXeG9cA55qrTeiUr+uxU\\\\\\\/p51lb+BBedHQ+Qk7AWn01shxt33L+CJAtTng5lxBHomjMaw740Bpr\\\\\\\/j0LLsmSggWhXeSnDXx\\\\\\\/lZDG4PDOLQ0b\\\\\\\/SIBqMpiNgLjvUOi0jtZkZXfK9VqhjZIwKwYwPABvblV\\\\\\\/rWotSEi7mYvnA1lg1+onsywe6CQQJ29xav0qWaP4Jd+lCsKIg7x0+dO0Qgt\\\\\\\/5qO1au5ue8KOY01noPGhhrYXJtU149Gux\\\\\\\/lqiORhGM7FdUnAEVu5+PI0JXJ\\\\\\\/dOLPYBlF3sBESV8ODbfdzYn5Dots0Pvy\\\\\\\/hMetCHMN58+JiMNmV0IH6fxLACfWLtsaoBxs39wzuupyyfktKZTv+6WNZg+FTQF6OXPI+prfiHs9fOdz0kltByZFhReqAH77riE5H0SIeMH3QNcpwW7sKYG9oOIiNFlE4jv\\\\\\\/3OY78KMprnM0X+EiyQ49hbjD0XyjqzYnIG4XOUCPTSLG7ckm1v8doquCftR+K9yxKQpSrFz3Q7u2UQi+6BkdTsqx91YxusYB+o4xkg+PDfnqO\\\\\\\/NViEm0PJPHKKRQY8b+BCuzP65RS0J28zpe6PzT0ceC1qIKlKujvHAobJVrEsc7H5womKrJV2s4i+rUkL6muybwmcidoTBAmDuL183DCINtoVbnVz7iZQKbz5EkrRXWujvsNJpHplTvb6QMpK6Dr2s3+fG6ge7PWiiYKAHB+9MPvFDP6gokcKr2lZrL90sYCq0dnQTLvvyc0vFApi9mxdpG2Cgm6BKmrUMbeGv93+WUfLb3a4qQVF1so4ETWCNL3CyFV2bm9cf4I2gfepDA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CBXjnTrMo1P9ZIphfN4Sow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022946aeba53f1c103fffdab374be01bf091fcd40d46ea9fb0931af7f473c163342\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492878 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1449" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-p5b7J4wAuEDDJW0nfrE19bvqVJI9Yo3lN5j9WyjM4zA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1449" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-f4izY7U94v2rwvml7nzxNFvaUFllMgrjbtVyIU2gT5E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1437" title="Unravelling the Complex Genetics of Karnal Bunt (Tilletia indica) Resistance in Common Wheat (Triticum aestivum) by Genetic Linkage and Genome-Wide Association Analyses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1457" title="Isolation by Distance, Source-Sink Population Dynamics and Dispersal Facilitation by Trade Routes: Impact on Population Genetic Structure of a Stored Grain Pest" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492878" id="node492878" data-pisa="ggg;9/5/1449" data-pisa-master="ggg;g3.118.200939" data-apath="/ggg/9/5/1449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gaussian-Distributed Codon Frequencies of Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bohdan B. Khomtchouk</span> and <span class="highwire-citation-author" data-delta="1">Wolfgang Nonner</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1449-1456;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200939 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492878" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bohdan B. Khomtchouk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bohdan%2BB.%2BKhomtchouk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khomtchouk%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABohdan%2BB.%2BKhomtchouk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bohdan@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">bohdan@stanford.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wolfgang Nonner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiology and Biophysics, University of Miami Miller School of Medicine, Miami, FL 33136</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BNonner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nonner%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BNonner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1449" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492878" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492878/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1449.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492878" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492878/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1449.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492878" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492878/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1449.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492878" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492878/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">DNA encodes protein primary structure using 64 different codons to specify 20 different amino acids and a stop signal. Frequencies of codon occurrence when ordered in descending sequence provide a global characterization of a genome’s preference (bias) for using the different codons of the redundant genetic code. Whereas frequency/rank relations have been described by empirical expressions, here we propose a statistical model in which two different forms of codon usage co-exist in a genome. We investigate whether such a model can account for the range of codon usages observed in a large set of genomes from different taxa. The differences in frequency/rank relations across these genomes can be expressed in a single parameter, the proportion of the two codon compartments. One compartment uses different codons with weak bias according to a Gaussian distribution of frequency, the other uses different codons with strong bias. In prokaryotic genomes both compartments appear to be present in a wide range of proportions, whereas in eukaryotic genomes the compartment with Gaussian distribution tends to dominate. Codon frequencies that are Gaussian-distributed suggest that many evolutionary conditions are involved in shaping weakly-biased codon usage, whereas strong bias in codon usage suggests dominance of few evolutionary conditions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/codon-usage" class="hw-term hw-article-keyword hw-article-keyword-codon-usage" rel="nofollow">codon usage</a></li><li class="kwd"><a href="/keyword/genetic-code" class="hw-term hw-article-keyword hw-article-keyword-genetic-code" rel="nofollow">genetic code</a></li></ul><p id="p-3">Genome sequencing has provided a large amount of information on how specific genomes use the 64 different codons of the redundant genetic code. This information on codon usage by genomes has raised many questions ranging from the fundamental to the technical: Can codon usage help elucidate the origin of the universal genetic code <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crick (1968)</a>? Do genes carry information beyond amino acid sequence in their usage of synonymous codons <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Zuckerkandl and Pauling (1965)</a>? What are the evolutionary mechanisms and possible (dis)advantages that make genomes use certain different codons more (or less) often than others <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Zuckerkandl and Pauling (1965)</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersson and Kurland (1990)</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Duret (2002)</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Plotkin and Kudla (2011)</a>? How do variations of codon usage arise in evolution <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Knight <em>et al.</em> (2001)</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> (2004)</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Sharp <em>et al.</em> (2010)</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Shah and Gilchrist (2011)</a>? What needs to be understood and controlled to optimize translation in heterologous expression systems <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boël <em>et al.</em> (2016)</a>?</p><p id="p-4">The detailed information compiled on diversity of codon usages within and across specific genomes has naturally overshadowed a search for common patterns in codon usage that overarch specific codon usages in specific organisms. One such pattern is the descending sequence of genomic codon frequencies. It characterizes bias inherent to the usages of different codons made within a given genome. Intra-genomic bias is necessary for the existence of inter-genomic diversity in the usage of the different codons.</p><p id="p-5">Interest into frequency/rank relations of genomes first concerned linguistic analogies of the ‘language of genes’ and spoken languages <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bender and Gill (1986)</a>. Empirical formal descriptions from this line of work include a power of rank (Zipf’s law) <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bender and Gill (1986)</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Obst <em>et al.</em> (2011)</a>, an exponential of rank <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Tsonis <em>et al.</em> (1997)</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Som <em>et al.</em> (2001)</a>, and a combination of exponential and linear relations <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Frappat <em>et al.</em> (2003)</a>. Codon frequency/rank curves were described by statistical relations with either no external parameter <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Borodovsky and Gusein-Zade (1989)</a>, or two external parameters <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Naumis and Cocho (2007</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2008</a>). With exception of the statistical model of <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Borodovsky and Gusein-Zade (1989)</a>, these descriptions have not been interpreted with regard to characteristics of evolutionary conditions that shape codon usage. In contrast, to account for observations of inhomogeneous codon usages among genes of the same organism, Gusein-Zade and Borodovsky <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gusein-Zade and Borodovsky (1990)</a> have proposed that there exist two compartments with exponentially-distributed codon frequencies.</p><p id="p-6">In this study we investigate intra-genomic bias in codon usage and test the applicability of a generalized two-compartment model in a large set of different genomes. We find that superposition of two compartments of codons with different usages describes genomic frequency/rank plots consistently. Variation of the proportion of the two compartments captures the variations observed among frequency/rank plots of genomes from many taxa. In one compartment, prevailing in eukaryotes, codon usage follows a random pattern as described by a Gaussian distribution of frequencies. Usage bias in that compartment is weak. In the other compartment, usage bias is strong to the extent of using essentially a subset of the available different codons. Prokaryote genomes reveal both compartments in widely varying proportions. The existence of a genomic compartment using codons with Gaussian-distributed frequencies likely implies the existence of many evolutionary conditions, which together could underlie codon usage of eukaryotic genomes.</p><div class="section other1" id="sec-1"><h2 class="">Biased Usage Of Codons: Observations</h2><p id="p-7">Different codons occur with a range of frequencies in a given genome, and any specific codon tends to occur with generally different frequencies across different genomes. These two aspects of bias in codon usage are illustrated in the cases of four organisms sampled from diverse taxa (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A,B</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Biased usage of different codons in genomes. Codon frequencies of Homo sapiens (A, red color), Arabidopsis thaliana (A, blue color), Streptomyces griseus (B, red color), and Clostridium tetani E88 (B, blue color). Frequencies are ordered as described in the text, with the smaller frequency of each two compared genomes shown in the foreground. C: histograms of bias B for 1840 genomes from the CUTG database (gray shaded) and of the eubacterial (black line), archaeal (red, scaled by 5), and eukaryotic subsets (blue) of these genomes. The total set comprised 1855 genomes, of which 1840 located to the binned range; the latter are analyzed in this study. Asterisks mark the bins to which the genomes of the panels A,B locate. D: frequency/rank curves averaged over the genomes of each histogram bin in panel C (with bias increasing from purple to red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968941336" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Biased usage of different codons in genomes. Codon frequencies of &lt;em&gt;Homo sapiens&lt;/em&gt; (A, red color), &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; (A, blue color), &lt;em&gt;Streptomyces griseus&lt;/em&gt; (B, red color), and &lt;em&gt;Clostridium tetani E88&lt;/em&gt; (B, blue color). Frequencies are ordered as described in the text, with the smaller frequency of each two compared genomes shown in the foreground. C: histograms of bias &lt;em&gt;B&lt;/em&gt; for 1840 genomes from the CUTG database (gray shaded) and of the eubacterial (&lt;em&gt;black line&lt;/em&gt;), archaeal (&lt;em&gt;red&lt;/em&gt;, scaled by 5), and eukaryotic subsets (&lt;em&gt;blue&lt;/em&gt;) of these genomes. The total set comprised 1855 genomes, of which 1840 located to the binned range; the latter are analyzed in this study. Asterisks mark the bins to which the genomes of the panels A,B locate. D: frequency/rank curves averaged over the genomes of each histogram bin in panel C (with bias increasing from purple to red).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1449/F1.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1449/F1.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Biased usage of different codons in genomes. Codon frequencies of <em>Homo sapiens</em> (A, red color), <em>Arabidopsis thaliana</em> (A, blue color), <em>Streptomyces griseus</em> (B, red color), and <em>Clostridium tetani E88</em> (B, blue color). Frequencies are ordered as described in the text, with the smaller frequency of each two compared genomes shown in the foreground. C: histograms of bias <em>B</em> for 1840 genomes from the CUTG database (gray shaded) and of the eubacterial (<em>black line</em>), archaeal (<em>red</em>, scaled by 5), and eukaryotic subsets (<em>blue</em>) of these genomes. The total set comprised 1855 genomes, of which 1840 located to the binned range; the latter are analyzed in this study. Asterisks mark the bins to which the genomes of the panels A,B locate. D: frequency/rank curves averaged over the genomes of each histogram bin in panel C (with bias increasing from purple to red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">In <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, the frequencies of the 64 different codons in human genomic DNA have been put in descending order (<em>red color</em>). When the frequencies of the different codons of the plant <em>Arabidopsis thaliana</em> are plotted in the order of the ordered human codons, they do not form a descending sequence (<em>blue color</em>). The codon frequencies of the plant, however, when ordered among themselves, form a descending sequence (<em>line</em>) that is quite similar to that of the ordered human frequencies. Codon usages of the bacteria <em>Clostridium tetani</em> and <em>Streptomyces griseus</em> also reveal similar intra-genomic bias, but diverse patterns of inter-genomic bias (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Both forms of bias are stronger in the bacteria than in human and plant. In each of the pairs of genomes of <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A,B</a>, a shared pattern of intra-genomic bias is associated with two distinct patterns of inter-genomic bias.</p><p id="p-10">Here we study the intra-genomic aspect of codon usage bias in the genomic coding DNA of a set of 1840 organisms of different taxa tabulated in the CUTG database <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Nakamura <em>et al.</em> (2000)</a> (files ‘gbxxx.spsum.txt’ where xxx = bct, inv, mam, pln, pri, rod, vrt). We restrict analysis to genomes represented with total codon counts <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-1.gif"/></noscript></span></span>, and define frequencies of occurrence, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-2.gif"/></noscript></span></span>, of different codons <em>i</em> (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-3.gif"/></noscript></span></span>) by:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>For a non-parametric assessment we measure intra-genomic ‘bias’ as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>For a hypothetical genome in which all different codons occur with the uniform frequency 1/64, bias <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-6.gif"/></noscript></span></span>. A real genome will have bias <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-7.gif"/></noscript></span></span>.</p><p id="p-11">A histogram of codon usage bias, <em>B</em>, constructed over the entire set of 1840 analyzed genomes is shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a> (<em>gray-shaded</em>). Histograms drawn as <em>lines</em> represent the subsets eubacteria (<em>black</em>), archaea (<em>red</em>, counts scaled by a factor of 5), and eukaryotes (<em>blue</em>). The genomes presented as examples in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A,B</a> locate to the bins marked by <em>asterisks</em>. Weak bias of codon usage is characteristic of eukaryotic genomes, whereas prokaryotic genomes reveal a wide range of bias. Overall, bias varies about threefold over the sampled set of genomes.</p><p id="p-12">Frequency/rank curves averaged over the genomes in each bin of the bias histogram in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a> are shown in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1D</a>. Codon usage bias with respect to average codon frequency (<em>dashed line</em>) is observed in all genomes, but a characteristic of the frequency/rank curve changes in the succession of curves from weak (<em>purple lines</em>) to strong bias (<em>red lines</em>): in curves associated with weak bias, curvature changes direction near the mean frequency, whereas curves associated with strong bias decline without inflection. Moreover, frequency of genomes with strong bias decays in a cascade-like fashion about the lower and upper halves of the ranks.</p><p id="p-13">Frequency/rank relations of a genome assign to a given frequency the number of different codons that have frequencies larger than or equal to that frequency. In this way they define the cumulative distribution of the random variable, codon frequency. For this statistical view of the descending frequency sequence, rotate <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a> clockwise by 90 degrees. Then the horizontal axis in panels A,B, and D is the random variable, codon frequency. A position on the vertical axis, read as offset with respect to rank 64, indicates how many different codon values have frequencies less than that at that frequency. The distributions in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1A</a> reveal a sigmoid cumulative distribution, whereas those in <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1B</a> resemble an exponential distribution.</p><p id="p-14">In an earlier study of codon usage, Gusein-Zade and Borodovsky <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gusein-Zade and Borodovsky (1990)</a> have investigated the possibility that genomes are inhomogeneous due to the existence of gene compartments that make distinct usages of different codons. Specifically, they developed a model of two compartments, each characterized by an exponential distribution. When only one compartment was present in a genome, the distribution was exponential. When both compartments co-existed, the overall frequency distribution was described by the convolution of the two scaled exponential distributions. This model was compared to a small dataset then available to the authors and found to account for characteristics such as the inflection observed in certain frequency/rank relations.</p><p id="p-15">In view of the larger dataset provided by the CUTG database, we noticed that the Gusein-Zade and Borodovsky (GZB) model with two exponentially distributed components falls short regarding the range of bias that it can describe. Here, bias can vary between <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-9.gif"/></noscript></span></span> depending on the proportion of the two compartments. This range is substantially less than the observed bias range (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1C</a>). In this study, we find that a two-compartment model of codon usage describes also the larger dataset of the CUTG database, where variations among genomes are accounted for by a single parameter, the proportion of the two compartments. A generalization of the GZB model, however, is necessary to describe the larger dataset: the distributions that describe codon usage in each compartment need to be chosen differently.</p><p id="p-16">In our generalized model, one compartment uses different codons according to a Gaussian distribution that produces minimal bias. We suppose this compartment to dominate codon usages like those by the genomes in <a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1A</a>. A Gaussian distribution would be expected to arise due to the Central Limit Theorem of statistics if many different evolutionary conditions shape codon usage in this compartment. Alternatively, the Gaussian compartment could represent many smaller compartments with not necessarily Gaussian distributions of codon frequencies.</p><p id="p-17">The other compartment of our model is governed by a distribution that produces even stronger bias than the exponential distributions proposed by GZB. The exponential distribution, as pointed out by GZB <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Gusein-Zade and Borodovsky (1990)</a>, ‘obeys the principle of maximum diversity of frequencies’, but this does not exclude that more biased distributions are possible (though less likely to arise by chance). For instance, genomes might restrict their machinery of translation to using a subset of different codons within the degenerate genetic code, or even use certain amino acids with preference. Such a codon usage is manifest in the genomes of <a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1B</a>. In our model, we derive an empirical description of codon frequencies in the second compartment from a subset of the data. We suppose this compartment to dominate in genomes that use different codons with strong bias.</p></div><div class="section other2" id="sec-2"><h2 class="">Construction Of The Model</h2><p id="p-18">We describe the model together with a Monte-Carlo numerical approach used to construct it (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Our approach reveals (and allows one to assess) a limitation inherent to modeling multiple compartments of codon usage.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Codon usage in a two-compartment genome model. Frequency sequences computed with Gaussian (A) or empirical frequency distribution (B) (lines). Colored: frequencies scaled by 1/2 for constructing a joint sequence representing two codon compartments of equal sizes. The colors illustrate different codons associated with the frequency ranks: the order of the second compartment is randomized with regard to the first. (C) Pairwise algebraic sum of the compartmental frequencies of each different codon (matched by color). (D) Frequency sequence (C) after re-ordering for a descending joint sequence. (E) Three samples of frequency sequence like that in (D) computed for different randomizations of the order of different codons in panel B (colored lines), and the mean of 100 samples (black line). Colored lines along the bottom of the graph: differences between individual samples and the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968941336" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Codon usage in a two-compartment genome model. Frequency sequences computed with Gaussian (A) or empirical frequency distribution (B) (&lt;em&gt;lines&lt;/em&gt;). &lt;em&gt;Colored&lt;/em&gt;: frequencies scaled by 1/2 for constructing a joint sequence representing two codon compartments of equal sizes. The colors illustrate different codons associated with the frequency ranks: the order of the second compartment is randomized with regard to the first. (C) Pairwise algebraic sum of the compartmental frequencies of each different codon (matched by color). (D) Frequency sequence (C) after re-ordering for a descending joint sequence. (E) Three samples of frequency sequence like that in (D) computed for different randomizations of the order of different codons in panel B (&lt;em&gt;colored lines&lt;/em&gt;), and the mean of 100 samples (&lt;em&gt;black line&lt;/em&gt;). Colored lines along the bottom of the graph: differences between individual samples and the mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1449/F2.medium.gif" width="401" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1449/F2.medium.gif" width="401" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493134" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Codon usage in a two-compartment genome model. Frequency sequences computed with Gaussian (A) or empirical frequency distribution (B) (<em>lines</em>). <em>Colored</em>: frequencies scaled by 1/2 for constructing a joint sequence representing two codon compartments of equal sizes. The colors illustrate different codons associated with the frequency ranks: the order of the second compartment is randomized with regard to the first. (C) Pairwise algebraic sum of the compartmental frequencies of each different codon (matched by color). (D) Frequency sequence (C) after re-ordering for a descending joint sequence. (E) Three samples of frequency sequence like that in (D) computed for different randomizations of the order of different codons in panel B (<em>colored lines</em>), and the mean of 100 samples (<em>black line</em>). Colored lines along the bottom of the graph: differences between individual samples and the mean.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Assume a genome comprises two compartments in which, generally, each different codon is used according to two distinct and mutually independent statistical distributions: a Gaussian distribution in compartment 1, and an empirically described distribution in compartment 2. Both distributions are naturally truncated to the range of codon frequency, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-10.gif"/></noscript></span></span>. Overall frequency of a different codon is given by the algebraic sum of the scaled frequencies in the two compartments. If <em>α</em> is the proportion of compartment 1,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>The cumulative probability of frequency <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-12.gif"/></noscript></span></span> in compartment 1 is that of a truncated Gaussian distribution,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where Φ denotes the cumulative normal distribution function with position <em>μ</em> and scale <em>σ</em>. These two parameters are restricted by the requirement that the mean of the truncated distribution be equal to <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-14.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-15.gif"/></noscript></span></span> being the number of different codons). This restriction follows from the normalization inherent to codon frequency. It leads to the relation:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-17.gif"/></noscript></span></span>. Thereby, the Gaussian distribution has only one external parameter. We will choose <em>σ</em> and determine <em>μ</em> by solving <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">equation 5</a> for <em>μ</em>.</p><p id="p-21">We generate a frequency sequence for compartment 1 using Monte-Carlo sampling of frequency into <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-18.gif"/></noscript></span></span> discrete bins that are associated with uniform increments of cumulative probability over the range <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-19.gif"/></noscript></span></span>. In each of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-20.gif"/></noscript></span></span> sampling cycles, a cumulative probability value is drawn from a generator of uniform random values in the range <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-21.gif"/></noscript></span></span>. The associated frequency <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-22.gif"/></noscript></span></span> is computed by solving <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">equation 4</a> for <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-23.gif"/></noscript></span></span> using a root finder. The frequency <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-24.gif"/></noscript></span></span> is included into the frequency average accumulated in the bin associated with the probability interval. A Monte-Carlo sampled frequency sequence is shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> (<em>solid line</em>). This calculation needs to be done only once for a chosen Gaussian distribution.</p><p id="p-22">We compute the frequency sequence of compartment 2 empirically, as an average over a group of genomes that reveal the strongest bias of codon usage in our dataset. We average the frequency sequences of the genomes locating to the right-most bin of the bias histogram (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1C</a>). In this study we will not further analyze the statistical basis of this empirical distribution. We will show here that the characteristics of that distribution are universally detected in genomes across taxa, as are the characteristics of a particular Gaussian distribution.</p><p id="p-23">To construct a frequency sequence for a genome comprising, <em>e.g.</em>, two equally large codon compartments, we scale the compartmental distribution frequencies by <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-25.gif"/></noscript></span></span>. We also randomize the association of different codon and frequency rank among the two scaled frequency sequences. This is done by assuming some order of different codons in the ranks of both compartments. Compartment 1 retains that order (represented by a rainbow sequence of colors in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>), whereas the order is scrambled in compartment 2 by swapping the colors of randomly chosen pairs of ranks in <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-26.gif"/></noscript></span></span> cycles (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-24">We merge the scaled and ordered sequences by algebraically adding to each frequency from compartment 1 the frequency from compartment 2 that is associated with the same different codon (<em>i.e.</em>, color). This produces the frequency sequence in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>, which happens to be no longer in descending order. Re-ordering of that sequence produces the joint frequency sequence of the two codon compartments of the genome (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>).</p><p id="p-25">The frequency sequence of the compartmentalized genome is intermediate between the compartmental sequences. It also has stochastic roughness that does not exist in the contributing sequences. Both compartmental sequences are thoroughly sampled in the procedure of merging frequencies but each joint frequency is sampled from exactly one pair of frequencies that is formed by a particular different codon in the two compartmental sequences. We cannot expect to model this pseudo-stochastic aspect of joint frequencies because an observed genomic frequency sequence does not reveal how a different codon is associated with rank in each compartment. We can, however, assess the potential consequences of this uncertainty by simulations with the model.</p><p id="p-26">We re-compute the joint frequency sequence using different associations of different codons and frequency ranks in the model compartments (by randomizing the ‘colors’ in compartment 2 for each trial). <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2E</a> shows examples of frequency sequence computed in three different trials (<em>colored lines</em>) and an average curve computed over 100 trials (<em>black line</em>) after re-ordering the sequence from each trial. All individual trials produce frequency sequences that fluctuate with respect to their mean to an extent small enough (rms of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-27.gif"/></noscript></span></span>) to preserve systematic features of the frequency sequence. These simulations quantify how much the mean theoretical curve, which does not capture consequences of specific ranks of different codons, is expected to diverge from the curve observed in a genome in whose compartments different codons have specific, but unknown ranks.</p><p id="p-27">The computer code for all computations was written from scratch in a PostScript-like language and is available in Zenodo (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Khomtchouk and Nonner 2019</a>). Code was executed using a virtual machine (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Peyser and Nonner 2011</a>).</p><div id="sec-3" class="subsection"><h3>Data availability</h3><p id="p-28">The authors affirm that all data necessary for confirming the conclusions of the article are present within the article, figures, source code, and tables.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><p id="p-29">We test the generalized two-compartment model on 1840 genomes from the CUTG database. Frequency sequences of the entire genome set are modeled by adjusting only the proportion <em>α</em> of the compartment with the Gaussian distribution while maintaining the Gaussian or empirical distributions of each compartment invariant. The scale of the Gaussian distribution is chosen to be <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-28.gif"/></noscript></span></span>, which implies the position <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-29.gif"/></noscript></span></span> by <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">equation 5</a>. The standard deviation of the truncated Gaussian distribution is 0.0081, and that of the empirical distribution 0.0207. These compartmental distributions (and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-30.gif"/></noscript></span></span>) have been used in the computations for <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>. In optimizing <em>α</em> we quantify the closeness of fit by the rms residual between observed and mean theoretical frequencies (the latter determined from 100 trials as described above). This residual is expected to approach <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-31.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-32.gif"/></noscript></span></span> because the theoretical frequency sequence is an average over many different rankings of a different codon’s frequency in the two compartments (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2E</a>).</p><p id="p-30"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A,B,D,E</a> shows observed and theoretical frequency/rank relations for the genomes introduced in <a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1A,B</a>. The model describes the data quite well, as is indicated by the residuals of the fit (<em>dotted lines</em>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed frequencies of codon occurrence compared to theory. Gray shaded: ordered sequences of the frequencies of four genomes (A, B, D, E) and ordered frequencies averaged across the subsets of genomes in the two marked histogram bins in Fig. 1C (C, F). Theoretical frequencies (black lines) are computed with the indicated proportion, &#x3B1;, of the compartment with Gaussian distribution; dotted lines: residuals between observed and theoretical frequencies. The rms values of the residuals, scaled by , are: 1 (A), 0.8 (B), 0.4 (C), 2.2 (D), 1.9 (E), and 1.1 (F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968941336" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Observed frequencies of codon occurrence compared to theory. &lt;em&gt;Gray shaded:&lt;/em&gt; ordered sequences of the frequencies of four genomes (A, B, D, E) and ordered frequencies averaged across the subsets of genomes in the two marked histogram bins in Fig. 1C (C, F). Theoretical frequencies (&lt;em&gt;black lines&lt;/em&gt;) are computed with the indicated proportion, &lt;em&gt;&#x3B1;&lt;/em&gt;, of the compartment with Gaussian distribution; &lt;em&gt;dotted lines&lt;/em&gt;: residuals between observed and theoretical frequencies. The rms values of the residuals, scaled by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, are: 1 (A), 0.8 (B), 0.4 (C), 2.2 (D), 1.9 (E), and 1.1 (F).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1449/F3.medium.gif" width="407" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1449/F3.medium.gif" width="407" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493170" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Observed frequencies of codon occurrence compared to theory. <em>Gray shaded:</em> ordered sequences of the frequencies of four genomes (A, B, D, E) and ordered frequencies averaged across the subsets of genomes in the two marked histogram bins in <a id="xref-fig-1-17" class="xref-fig" href="#F1">Fig. 1C</a> (C, F). Theoretical frequencies (<em>black lines</em>) are computed with the indicated proportion, <em>α</em>, of the compartment with Gaussian distribution; <em>dotted lines</em>: residuals between observed and theoretical frequencies. The rms values of the residuals, scaled by <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/F3/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/F3/embed/mml-math-33.gif"/></noscript></span></span>, are: 1 (A), 0.8 (B), 0.4 (C), 2.2 (D), 1.9 (E), and 1.1 (F).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The specific order of codons in the hypothesized compartments of the individual genomes, which is unknown, might actually limit our model’s capacity to account for the frequencies in an individual genome. On the assumption that different genomes of similar codon usage bias have different specific orders of codons in their compartments (compare <a id="xref-fig-1-18" class="xref-fig" href="#F1">Figure 1A,B</a>), we expect that averaging the ordered sequences of frequencies of such genomes produces a frequency sequence that is more accurately predicted by the (averaged) sequence of the model than are sequences of individual genomes. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C,F</a> compare averaged observed frequency sequence with fitted theoretical sequence. The residuals are substantially smaller than in the case of the individual genomes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A,B,D,E</a>; see legend for rms values of residuals). The model describes frequency/rank relations averaged over genomes more accurately than those of individual genomes of similar usage bias.</p><p id="p-33"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> extends the comparison of observed and theoretical frequency sequences to twelve genomes that are commonly studied. These genomes belong to diverse taxa, and the bias of their codon usage is in the range most frequently observed in the genomes of the CUTG database. In all cases, adjustment of the proportion of two compartments in the model within the range <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-34.gif"/></noscript></span></span> allows the model to reproduce these frequency sequences quite well. Although the theoretical frequency sequences are determined by varying substantial contributions from both hypothesized compartments in these cases, the observed sequences are reproduced without any change made to the distributions that are assumed to underlie codon usage in each compartment.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed frequencies of codon occurrence compared to theory. Sequences of frequencies of 12 genomes (gray-shaded). Lines are computed from theory with the indicated proportion, &#x3B1;, of the compartment with Gaussian distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968941336" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Observed frequencies of codon occurrence compared to theory. Sequences of frequencies of 12 genomes (&lt;em&gt;gray-shaded&lt;/em&gt;). &lt;em&gt;Lines&lt;/em&gt; are computed from theory with the indicated proportion, &lt;em&gt;&#x3B1;&lt;/em&gt;, of the compartment with Gaussian distribution.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1449/F4.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1449/F4.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492888" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Observed frequencies of codon occurrence compared to theory. Sequences of frequencies of 12 genomes (<em>gray-shaded</em>). <em>Lines</em> are computed from theory with the indicated proportion, <em>α</em>, of the compartment with Gaussian distribution.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A,B</a> summarize the application of the model to the entire set of genomes, divided between eubacteria and archaea (A) and plants, invertebrates, and vertebrates (B). The proportion of the Gaussian compartment, <em>α</em>, which is obtained by the fits of the model to the data, is plotted <em>vs.</em> the observed bias (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation 1</a>) for each individual genome (<em>symbols</em>). The <em>lines</em> show the theoretical relation:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-35.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-36.gif"/></noscript></span></span> is the standard error of the truncated Gaussian distribution, and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1449/embed/mml-math-37.gif"/></noscript></span></span> the standard error of the empirical distribution.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The two-compartment model describes biased codon usage of 1840 genomes. (A) The single varied model parameter &#x3B1; describing a genome plotted vs. the observed codon usage bias, equation 1: black symbols eubacteria, red symbols archaea. The line represents the theoretical relation, equation 6. (B) Like panel A, representing eukaryotes: green plants, blue invertebrates, red vertebrates. (C) rms of residuals between the observed and theoretical frequency/rank relations (symbols) and theoretical rms expected to arise due to unknown ranks of a different codon in the model compartments (line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968941336" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The two-compartment model describes biased codon usage of 1840 genomes. (A) The single varied model parameter &lt;em&gt;&#x3B1;&lt;/em&gt; describing a genome plotted &lt;em&gt;vs.&lt;/em&gt; the observed codon usage bias, equation 1: &lt;em&gt;black symbols&lt;/em&gt; eubacteria, &lt;em&gt;red symbols&lt;/em&gt; archaea. The &lt;em&gt;line&lt;/em&gt; represents the theoretical relation, equation 6. (B) Like panel A, representing eukaryotes: &lt;em&gt;green&lt;/em&gt; plants, &lt;em&gt;blue&lt;/em&gt; invertebrates, &lt;em&gt;red&lt;/em&gt; vertebrates. (C) rms of residuals between the observed and theoretical frequency/rank relations (&lt;em&gt;symbols&lt;/em&gt;) and theoretical rms expected to arise due to unknown ranks of a different codon in the model compartments (&lt;em&gt;line&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1449/F5.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/5/1449/F5.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1449/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493057" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">The two-compartment model describes biased codon usage of 1840 genomes. (A) The single varied model parameter <em>α</em> describing a genome plotted <em>vs.</em> the observed codon usage bias, <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">equation 1</a>: <em>black symbols</em> eubacteria, <em>red symbols</em> archaea. The <em>line</em> represents the theoretical relation, <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">equation 6</a>. (B) Like panel A, representing eukaryotes: <em>green</em> plants, <em>blue</em> invertebrates, <em>red</em> vertebrates. (C) rms of residuals between the observed and theoretical frequency/rank relations (<em>symbols</em>) and theoretical rms expected to arise due to unknown ranks of a different codon in the model compartments (<em>line</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Genomes follow the theoretical relation very closely for all taxa represented here, showing that the model produces a consistent relation between compartment proportions and codon usage bias over the full range of bias and of shapes found in the frequency/rank relations of these 1840 genomes.</p><p id="p-38">The accuracy of the model in describing these data are quantified in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5C</a> showing the rms residual between observed and theoretical frequency/rank relations for each genome (<em>symbols</em>). The <em>line</em> gives the rms residual expected to arise from the uncertain frequency ranks of a different codon in the two model compartments (see Construction of the model and <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2E</a>). For the majority of genomes, the actual residual of the fit is similar to the residual expected from the uncertainty of ranks.</p></div><div class="section discussion" id="sec-5"><h2 class="">Discussion</h2><p id="p-39">We have analyzed aspects of biased usage of different codons at a scale ranging from the individual full genome to many genomes of different taxa. Building on early work of Gusein-Zade and Borodovsky <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Gusein-Zade and Borodovsky (1990)</a>, we have constructed a generalized statistical model to describe the ordered sequence of different-codon frequency in the large genome set of the CUTG database.</p><p id="p-40">Our model posits that codon usage is generally inhomogeneous within a genome, albeit to varying extents. Two compartments of codons provide a good description if each compartment has a distinct distribution of codon frequency. Adjusting a single external parameter, the proportion of the two compartments, while keeping the compartmental frequency distributions the same then suffices to describe codon usage throughout the database.</p><p id="p-41">The two compartmental distributions are ‘antipodal’ in that one is Gaussian with a scale that results in weak bias of codon usage, and the other, described empirically, results in very strong bias. The model is not concerned with a specific map relating the two compartments to the genome’s codons (at whatever scale), nor with a specific association of different codons (or their nucleotide composition) with rank in the ordered frequency sequence. Nevertheless, the model reveals a pattern of codon usages that is apparently universal among genomes.</p><p id="p-42">Existence, within one and the same genome, of two forms of codon usage indicates the existence of ‘inhomogeneous conditions of molecular evolution within a genome’ <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Gusein-Zade and Borodovsky (1990)</a>. Inhomogeneity must be strong enough to drive codon usage into two antipodal compartmental patterns. Of the conditions that support either antipodal distribution of codon frequency, the ones that result in the strongly biased distribution might intuitively appear as the stronger determinants of codon usage. Then, the compartment with Gaussian distribution might be determined more by the absence of strong determinants than by distinguished conditions that favor the more balanced codon usage. Such a simple view of conditions determining codon usage is diffracted by the fact that even the postulated ‘strong’ conditions must generally result in much different ranks of particular different codons in similar ordered sequences of frequency (<a id="xref-fig-1-19" class="xref-fig" href="#F1">Figure 1B</a>), a tendency recognizable even in genomes that use codons with less bias (<a id="xref-fig-1-20" class="xref-fig" href="#F1">Figure 1A</a>). It is therefore clear that a further discussion of conditions that shape codon usages needs to consider the fate of the specific different codons (a study of this kind will be presented in a separate paper).</p><p id="p-43">In the codon compartment that we model by a Gaussian distribution of frequency, that distribution likely summarizes consequences of many evolutionary conditions as well as outcomes in many subpopulations of the compartment. The Central Limit Theorem of statistics would then imply that the joint distribution approaches a Gaussian, even if individual contributions have non-Gaussian frequency distributions with generally different means and variances. Inhomogeneity of codon usage among genes, or within genes, of a given genome is a common phenomenon, and in many instances has been associated with functional aspects of translation (for recent reviews see <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Komar (2016)</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hanson and Coller (2018)</a>).</p><p id="p-44">The wide range of taxa revealing a Gaussian compartment in genomic codon usage (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>) suggests that conditions contributing to this compartment exist universally across genomes, but does not require that conditions be the same across taxa. On the other hand, our observation that one distribution (of fixed scale and position) describes the Gaussian codon compartment of different genomes suggests the possibility that important evolutionary conditions shaping Gaussian-distributed codon usage are widely shared by genomes.</p><p id="p-45">The strongly biased distribution of the second codon compartment, by virtue of being non-Gaussian, suggests a comparably simple structure informed by few but strong evolutionary conditions. This raises the question of how consequences of both weak and strong evolutionary conditions might co-exist in one genome. An answer might have to include not only the short-term evolutionary status but also the long-term evolutionary history of an organism – the evolutionary conditions informing codon compartments need not co-exist within one and the same span of time.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">Research reported in this publication was supported by the National Institute on Aging of the National Institutes of Health under Award Number T32 AG0047126 (to BK). The content is solely the responsibility of the author(s) and does not necessarily represent the official views of the National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-47"><em>Communicating editor: J. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-01"><span class="received-label">Received </span>December 1, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-25"><span class="accepted-label">Accepted </span>February 25, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Khomtchouk, Nonner</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurland</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Codon preferences in free-living microorganisms.</span> <abbr class="cit-jnl-abbrev">Microbiol. Rev.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DAndersson%26rft.auinit1%253DS%2BG%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D198%26rft.epage%253D210%26rft.atitle%253DCodon%2Bpreferences%2Bin%2Bfree-living%2Bmicroorganisms.%26rft_id%253Dinfo%253Apmid%252F2194095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI1NC8yLzE5OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.2" data-doi="10.1086/203436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">The genetic code and Zipf’s law.</span> <abbr class="cit-jnl-abbrev">Curr. Anthropol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/203436</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BAnthropol.%26rft.volume%253D27%26rft.spage%253D280%26rft_id%253Dinfo%253Adoi%252F10.1086%252F203436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/203436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.3" data-doi="10.1038/nature16509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boël</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Letso</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neely</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Codon influence on protein expression in <em>E. coli</em> correlates with mRNA levels.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">529</span>: <span class="cit-fpage">358</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature16509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D529%26rft.spage%253D358%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16509%26rft_id%253Dinfo%253Apmid%252F26760206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26760206&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.4" data-doi="10.1080/07391102.1989.10506527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gusein-Zade</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A general rule for ranged series of codon frequencies in different genomes.</span> <abbr class="cit-jnl-abbrev">J. Biomol. Struct. Dyn.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07391102.1989.10506527</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiomolecular%2Bstructure%2B%2526%2Bdynamics%26rft.stitle%253DJ%2BBiomol%2BStruct%2BDyn%26rft.aulast%253DBorodovsky%2BMYu%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D1001%26rft.epage%253D1012%26rft.atitle%253DA%2Bgeneral%2Brule%2Bfor%2Branged%2Bseries%2Bof%2Bcodon%2Bfrequencies%2Bin%2Bdifferent%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07391102.1989.10506527%26rft_id%253Dinfo%253Apmid%252F2556159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07391102.1989.10506527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2556159&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.5" data-doi="10.1073/pnas.0307827100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hottes</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAdams</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon usage between genomes is constrained by genome-wide mutational processes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">3480</span>–<span class="cit-lpage">3485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0307827100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307827100%26rft_id%253Dinfo%253Apmid%252F14990797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEwLzM0ODAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.6" data-doi="10.1016/0022-2836(68)90392-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crick</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">The origin of the genetic code.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2836(68)90392-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCrick%26rft.auinit1%253DF.%2BH.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D367%26rft.epage%253D379%26rft.atitle%253DThe%2Borigin%2Bof%2Bthe%2Bgenetic%2Bcode.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252868%252990392-6%26rft_id%253Dinfo%253Apmid%252F4887876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(68)90392-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4887876&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1968C697500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.7" data-doi="10.1016/S0959-437X(02)00353-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evolution of synonymous codon usage in metazoans.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0959-437X(02)00353-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D640%26rft.epage%253D649%26rft.atitle%253DEvolution%2Bof%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bmetazoans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900353-2%26rft_id%253Dinfo%253Apmid%252F12433576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00353-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433576&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.8" data-doi="10.1103/PhysRevE.68.061910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frappat</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minichini</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sciarrino</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorba</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Universality and Shannon entropy of codon usage.</span> <abbr class="cit-jnl-abbrev">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">061910</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1103/PhysRevE.68.061910</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2Breview.%2BE%252C%2BStatistical%252C%2Bnonlinear%252C%2Band%2Bsoft%2Bmatter%2Bphysics%26rft.stitle%253DPhys%2BRev%2BE%2BStat%2BNonlin%2BSoft%2BMatter%2BPhys%26rft.aulast%253DFrappat%26rft.auinit1%253DL.%26rft.volume%253D68%26rft.issue%253D6%2BPt%2B1%26rft.spage%253D061910%26rft.epage%253D061910%26rft.atitle%253DUniversality%2Band%2BShannon%2Bentropy%2Bof%2Bcodon%2Busage.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevE.68.061910%26rft_id%253Dinfo%253Apmid%252F14754237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevE.68.061910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14754237&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.9" data-doi="10.1080/07391102.1990.10508555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusein-Zade</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">An improved distribution of codon frequencies allowing for inhomogeneity of DNA’s primary-structure evolution.</span> <abbr class="cit-jnl-abbrev">J. Biomol. Struct. Dyn.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07391102.1990.10508555</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiomolecular%2Bstructure%2B%2526%2Bdynamics%26rft.stitle%253DJ%2BBiomol%2BStruct%2BDyn%26rft.aulast%253DGusein-Zade%26rft.auinit1%253DS.%2BM.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D1185%26rft.epage%253D1197%26rft.atitle%253DAn%2Bimproved%2Bdistribution%2Bof%2Bcodon%2Bfrequencies%2Ballowing%2Bfor%2Binhomogeneity%2Bof%2BDNA%2527s%2Bprimary-structure%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07391102.1990.10508555%26rft_id%253Dinfo%253Apmid%252F2361006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07391102.1990.10508555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2361006&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.10" data-doi="10.1038/nrm.2017.91"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Codon optimality, bias and usage in translation and mRNA decay.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrm.2017.91</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D19%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm.2017.91%26rft_id%253Dinfo%253Apmid%252F29018283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm.2017.91&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29018283&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1449.11"><div class="cit-metadata"><cite>Khomtchouk, B., and W. Nonner, 2019 Source code release for: Gaussian distributed codon frequencies of genomes (version 1.0.0), Zenodo<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5281/zenodo.2556048</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeland</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landweber</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes. Genome Biology 2: 10.1–10.13.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.13" data-doi="10.1093/hmg/ddw207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komar</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The yin and yang of codon usage.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">R77</span>–<span class="cit-lpage">R85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/hmg/ddw207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddw207%26rft_id%253Dinfo%253Apmid%252F27354349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddw207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27354349&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.14" data-doi="10.1093/nar/28.1.292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gojobori</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Codon usage tabulated from international DNA sequence databases: status for the year 2000.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/28.1.292</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNakamura%26rft.auinit1%253DY.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D292%26rft.epage%253D292%26rft.atitle%253DCodon%2Busage%2Btabulated%2Bfrom%2Binternational%2BDNA%2Bsequence%2Bdatabases%253A%2Bstatus%2Bfor%2Bthe%2Byear%2B2000%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.292%26rft_id%253Dinfo%253Apmid%252F10592250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592250&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300084&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.15" data-doi="10.1088/1367-2630/9/8/286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naumis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cocho</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The tails of rank-size distributions due to multiplicative processes: from power laws tp stretched exponentials and beta-like functions.</span> <abbr class="cit-jnl-abbrev">New J. Phys.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1088/1367-2630/9/8/286</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BJ.%2BPhys.%26rft.volume%253D9%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1088%252F1367-2630%252F9%252F8%252F286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1088/1367-2630/9/8/286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.16" data-doi="10.1016/j.physa.2007.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naumis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cocho</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Tail universalities in rank distributions as an algebraic problem: the beta-like function.</span> <abbr class="cit-jnl-abbrev">Physica A</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">96</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.physa.2007.08.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysica%2BA%26rft.volume%253D387%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.physa.2007.08.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.physa.2007.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.17" data-doi="10.1007/978-3-642-21314-4_11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Obst</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polani</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prokopenko</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Origins of scaling in genetic code. Advances in Artificial Life. Darwin Meets von Neumann</span>. <abbr class="cit-jnl-abbrev">Lect. Notes Comput. Sci.</abbr> <span class="cit-vol">5778</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-21314-4_11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLect.%2BNotes%2BComput.%2BSci.%26rft.volume%253D5778%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-21314-4_11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-21314-4_11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1449.18"><div class="cit-metadata"><cite>Peyser, A., and W. Nonner, 2011 Deuterostome: 4.8.10, Zenodo<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5281/zenodo.1168556</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.19" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2899</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.20" data-doi="10.1073/pnas.1016719108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilchrist</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">10231</span>–<span class="cit-lpage">10236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1016719108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1016719108%26rft_id%253Dinfo%253Apmid%252F21646514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI1LzEwMjMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.21" data-doi="10.1098/rstb.2009.0305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Forces that influence the evolution of codon bias.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1203</span>–<span class="cit-lpage">1212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2009.0305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1203%26rft.epage%253D1212%26rft.atitle%253DForces%2Bthat%2Binfluence%2Bthe%2Bevolution%2Bof%2Bcodon%2Bbias%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0305%26rft_id%253Dinfo%253Apmid%252F20308095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2009.0305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20308095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.22" data-doi="10.1103/PhysRevE.63.051908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Som</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chattapadhyay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakrabarti</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Codon distributions in DNA.</span> <abbr class="cit-jnl-abbrev">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">051908</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1103/PhysRevE.63.051908</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2Breview.%2BE%252C%2BStatistical%252C%2Bnonlinear%252C%2Band%2Bsoft%2Bmatter%2Bphysics%26rft.stitle%253DPhys%2BRev%2BE%2BStat%2BNonlin%2BSoft%2BMatter%2BPhys%26rft.aulast%253DSom%26rft.auinit1%253DA.%26rft.volume%253D63%26rft.issue%253D5%2BPt%2B1%26rft.spage%253D051908%26rft.epage%253D051908%26rft.atitle%253DCodon%2Bdistributions%2Bin%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevE.63.051908%26rft_id%253Dinfo%253Apmid%252F11414934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevE.63.051908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11414934&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.23" data-doi="10.1006/jtbi.1996.0239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsonis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsonis</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Is DNA a language?</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jtbi.1996.0239</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DTsonis%26rft.auinit1%253DA.%2BA.%26rft.volume%253D184%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DIs%2BDNA%2Ba%2Blanguage%253F%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.1996.0239%26rft_id%253Dinfo%253Apmid%252F9039397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.1996.0239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9039397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1449.24" data-doi="10.1016/0022-5193(65)90083-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuckerkandl</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauling</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Molecules as documents of evolutionary history.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-5193(65)90083-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DZuckerkandl%26rft.auinit1%253DE.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D357%26rft.epage%253D366%26rft.atitle%253DMolecules%2Bas%2Bdocuments%2Bof%2Bevolutionary%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-5193%252865%252990083-4%26rft_id%253Dinfo%253Apmid%252F5876245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-5193(65)90083-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5876245&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19656369100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1449.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1437" title="Unravelling the Complex Genetics of Karnal Bunt (Tilletia indica) Resistance in Common Wheat (Triticum aestivum) by Genetic Linkage and Genome-Wide Association Analyses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1457" title="Isolation by Distance, Source-Sink Population Dynamics and Dispersal Facilitation by Trade Routes: Impact on Population Genetic Structure of a Stored Grain Pest" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1449" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zwzru6igr_E_wstIH0k4MujVs-1O32jLQSMQyAy57CE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1449" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1449" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1449" class="active" data-icon-position="" data-hide-link-title="0">Gaussian-Distributed Codon Frequencies of Genomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492878" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UvXN7wKjfdcGOQI0Zf10i3xrjoqAvGsASdZkyFMD_Lc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1449.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1449" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-NhK2afU1Vpr7ibtKVNeQAp94m1NUKNM9X_gZZJG4ma4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1449" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1449.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492878" id="node492878--2" data-pisa="ggg;9/5/1449" data-pisa-master="ggg;g3.118.200939" data-apath="/ggg/9/5/1449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gaussian-Distributed Codon Frequencies of Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bohdan B. Khomtchouk</span> and <span class="highwire-citation-author" data-delta="1">Wolfgang Nonner</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1449-1456;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200939 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492878--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bohdan B. Khomtchouk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bohdan%2BB.%2BKhomtchouk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khomtchouk%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABohdan%2BB.%2BKhomtchouk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bohdan@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">bohdan@stanford.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wolfgang Nonner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiology and Biophysics, University of Miami Miller School of Medicine, Miami, FL 33136</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BNonner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nonner%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BNonner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1449" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1449&amp;title=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1449&amp;title=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1449&amp;title=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1449&amp;text=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1449&amp;title=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1449&amp;t=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1449&amp;title=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1449&amp;title=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492878/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492878" id="node492878--3" data-pisa="ggg;9/5/1449" data-pisa-master="ggg;g3.118.200939" data-apath="/ggg/9/5/1449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gaussian-Distributed Codon Frequencies of Genomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bohdan B. Khomtchouk</span> and <span class="highwire-citation-author" data-delta="1">Wolfgang Nonner</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1449-1456;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200939 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492878--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bohdan B. Khomtchouk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bohdan%2BB.%2BKhomtchouk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khomtchouk%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABohdan%2BB.%2BKhomtchouk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bohdan@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">bohdan@stanford.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wolfgang Nonner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiology and Biophysics, University of Miami Miller School of Medicine, Miami, FL 33136</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BNonner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nonner%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BNonner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1449&amp;rft.epage=1456&amp;rft.atitle=Gaussian-Distributed%20Codon%20Frequencies%20of%20Genomes&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Khomtchouk%2C+Bohdan+B.&amp;rft.au=Nonner%2C+Wolfgang"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492878/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492878/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492878/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492878/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492878/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492878/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492878/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492878/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492878/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492878/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492878/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492878/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1449" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Biased Usage Of Codons: Observations</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Construction Of The Model</a></li><li class="even"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1449.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1449.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1449.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
