<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1663" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1663.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1663.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1663.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1663/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1663.atom" />
<meta name="HW.pisa" content="ggg;9/5/1663" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach" />
<meta name="DC.Identifier" content="10.1534/g3.119.400167" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Shen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Apple fruit ring rot (FRR), caused by Botryosphaeria dothidea , is a worldwide disease that impacts Asian apple production regions. However, no substantial progress has thus far been made toward the mapping of candidate genes or the development of effective genetic makers. In this five-year study, the resistance of 1,733 F1 hybrids from the cross ‘Jonathan’ × ‘Golden Delicious’ was phenotyped by non-wounding inoculation with four B. dothidea isolates. We first conducted systematic comparison of different analytic strategies for bulk segregant analysis by re-sequencing (BSA-Seq) and obtained suitable one for outbreeding species such as Malus . Forty-six quantitative trait loci (QTL) for resistance/susceptibility to the four isolates, including one QTL ‘hotspot’ on chromosome 14, were identified via BSA-Seq. Using integrated multi-omics strategies including RNA-sequencing, parental re-sequencing, BSA-Seq and meta-analysis of RNA-sequencing, fifty-seven candidate genes and corresponding functional mutations from the QTL were predicted. Functional mutations located on the candidate genes were validated using kompetitive allele-specific PCR in hybrids and Malus germplasm accessions with extremely resistant/susceptible phenotypes. Ten effective markers for apple ring rot were developed. The results provide an example of rapid candidate gene mapping for complex traits in outbreeding species." />
<meta name="DC.Contributor" content="Fei Shen" />
<meta name="DC.Contributor" content="Zhenyu Huang" />
<meta name="DC.Contributor" content="Baoguo Zhang" />
<meta name="DC.Contributor" content="Yi Wang" />
<meta name="DC.Contributor" content="Xi Zhang" />
<meta name="DC.Contributor" content="Ting Wu" />
<meta name="DC.Contributor" content="Xuefeng Xu" />
<meta name="DC.Contributor" content="Xinzhong Zhang" />
<meta name="DC.Contributor" content="Zhenhai Han" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Apple fruit ring rot (FRR), caused by &lt;i&gt;Botryosphaeria dothidea&lt;/i&gt;, is a worldwide disease that impacts Asian apple production regions. However, no substantial progress has thus far been made toward the mapping of candidate genes or the development of effective genetic makers. In this five-year study, the resistance of 1,733 F1 hybrids from the cross ‘Jonathan’ × ‘Golden Delicious’ was phenotyped by non-wounding inoculation with four &lt;i&gt;B. dothidea&lt;/i&gt; isolates. We first conducted systematic comparison of different analytic strategies for bulk segregant analysis by re-sequencing (BSA-Seq) and obtained suitable one for outbreeding species such as &lt;i&gt;Malus&lt;/i&gt;. Forty-six quantitative trait loci (QTL) for resistance/susceptibility to the four isolates, including one QTL ‘hotspot’ on chromosome 14, were identified via BSA-Seq. Using integrated multi-omics strategies including RNA-sequencing, parental re-sequencing, BSA-Seq and meta-analysis of RNA-sequencing, fifty-seven candidate genes and corresponding functional mutations from the QTL were predicted. Functional mutations located on the candidate genes were validated using kompetitive allele-specific PCR in hybrids and &lt;i&gt;Malus&lt;/i&gt; germplasm accessions with extremely resistant/susceptible phenotypes. Ten effective markers for apple ring rot were developed. The results provide an example of rapid candidate gene mapping for complex traits in outbreeding species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1663" />
<meta name="citation_id" content="9/5/1663" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1663" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1663.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1663.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1663.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400167" />
<meta name="citation_pmid" content="30910819" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1663" />
<meta name="citation_lastpage" content="1678" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Fei Shen" />
<meta name="citation_author_institution" content="College of Horticulture, China Agricultural University, Beijing 100193, P. R. China" />
<meta name="citation_author" content="Zhenyu Huang" />
<meta name="citation_author_institution" content="College of Horticulture, China Agricultural University, Beijing 100193, P. R. China" />
<meta name="citation_author" content="Baoguo Zhang" />
<meta name="citation_author_institution" content="Zhongbaolvdu Agricultural Research Centre, Beidaihe, Hebei, 066100, China" />
<meta name="citation_author" content="Yi Wang" />
<meta name="citation_author_institution" content="College of Horticulture, China Agricultural University, Beijing 100193, P. R. China" />
<meta name="citation_author" content="Xi Zhang" />
<meta name="citation_author_institution" content="College of Horticulture, China Agricultural University, Beijing 100193, P. R. China" />
<meta name="citation_author" content="Ting Wu" />
<meta name="citation_author_institution" content="College of Horticulture, China Agricultural University, Beijing 100193, P. R. China" />
<meta name="citation_author" content="Xuefeng Xu" />
<meta name="citation_author_institution" content="College of Horticulture, China Agricultural University, Beijing 100193, P. R. China" />
<meta name="citation_author" content="Xinzhong Zhang" />
<meta name="citation_author_institution" content="College of Horticulture, China Agricultural University, Beijing 100193, P. R. China" />
<meta name="citation_author_email" content="zhangxinzhong999@126.com" />
<meta name="citation_author_email" content="rschan@cau.edu.cn" />
<meta name="citation_author" content="Zhenhai Han" />
<meta name="citation_author_institution" content="College of Horticulture, China Agricultural University, Beijing 100193, P. R. China" />
<meta name="citation_author_email" content="zhangxinzhong999@126.com" />
<meta name="citation_author_email" content="rschan@cau.edu.cn" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=S. H. Bai;citation_author=C. H. Dong;citation_author=J. Zhu;citation_author=Y. G. Zhang;citation_author=H. Y. Dai;citation_title=Identification of a xyloglucan-specific endo-(1–4)-beta-D-glucanase inhibitor protein from apple (Malus × domestica Borkh.) as a potential defense gene against Botryosphaeria dothidea.;citation_pages=11-19;citation_volume=231;citation_year=2015;citation_doi=10.1016/j.plantsci.2014.11.003" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=T. Bai;citation_author=Y. Zhu;citation_author=F. Fernandez-Fernandez;citation_author=J. Keulemans;citation_author=S. Brown;citation_title=Fine genetic mapping of the Co locus controlling columnar growth habit in apple.;citation_pages=437-450;citation_volume=287;citation_year=2012;citation_issue=5;citation_pmid=22526430;citation_doi=10.1007/s00438-012-0689-5" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=Y. Bai;citation_author=L. Dougherty;citation_author=M. Li;citation_author=G. Fazio;citation_author=L. Cheng;citation_title=A natural mutation-led truncation in one of the two aluminum-activated malate transporter-like genes at the Ma locus is associated with low fruit acidity in apple.;citation_pages=663-678;citation_volume=287;citation_year=2012;citation_issue=8;citation_pmid=22806345;citation_doi=10.1007/s00438-012-0707-7" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=B. Balan;citation_author=T. Caruso;citation_author=F. Martinelli;citation_title=Gaining insight into exclusive and common transcriptomic features linked with biotic stress responses in Malus.;citation_pages=1569;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01569" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=B. Balan;citation_author=F. P. Marra;citation_author=T. Caruso;citation_author=F. Martinelli;citation_title=Transcriptomic responses to biotic stresses in Malus × domestica: a meta-analysis study.;citation_pages=1970;citation_volume=8;citation_year=2018;citation_doi=10.1038/s41598-018-19348-4" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=L. Bianco;citation_author=A. Cestaro;citation_author=G. Linsmith;citation_author=H. Muranty;citation_author=C. Denance;citation_title=Development and validation of the Axiom() Apple480K SNP genotyping array.;citation_pages=62-74;citation_volume=86;citation_year=2016;citation_doi=10.1111/tpj.13145" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Bianco;citation_author=A. Cestaro;citation_author=D. J. Sargent;citation_author=E. Banchi;citation_author=S. Derdak;citation_title=Development and validation of a 20K single nucleotide polymorphism (SNP) whole genome genotyping array for apple (Malus x domestica Borkh).;citation_pages=e110377;citation_volume=9;citation_year=2014;citation_pmid=25303088;citation_doi=10.1371/journal.pone.0110377" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=G. Bénaouf;citation_author=L. Parisi;citation_title=Genetics of Host-Pathogen Relationships Between Venturia inaequalis Races 6 and 7 and Malus Species.;citation_pages=236-242;citation_volume=90;citation_year=2000;citation_issue=3;citation_pmid=18944614;citation_doi=10.1094/PHYTO.2000.90.3.236" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=C. Bonnet;citation_author=S. Lassueur;citation_author=C. Ponzio;citation_author=R. Gols;citation_author=M. Dicke;citation_title=Combined biotic stresses trigger similar transcriptomic responses but contrasting resistance against a chewing herbivore in Brassica nigra.;citation_pages=127;citation_volume=17;citation_year=2017;citation_doi=10.1186/s12870-017-1074-7" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. Chiang;citation_author=R. M. Layer;citation_author=G. G. Faust;citation_author=M. R. Lindberg;citation_author=D. B. Rose;citation_title=SpeedSeq: ultra-fast personal genome analysis and interpretation.;citation_pages=966-968;citation_volume=12;citation_year=2015;citation_pmid=26258291;citation_doi=10.1038/nmeth.3505" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Gotz;citation_author=J. M. Garcia-Gomez;citation_author=J. Terol;citation_author=M. Talon;citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_journal_title=J. Phytopathol.;citation_author=M. S. Cui;citation_author=L. L. Yang;citation_author=Y. Y. Han;citation_author=Q. Zhang;citation_author=Y. B. Zhao;citation_title=Genetic mapping reveals sophisticated responses of Malus domestica to Botryosphaeria dothidea isolates.;citation_pages=42-53;citation_volume=163;citation_year=2014;citation_doi=10.1111/jph.12279" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=N. Daccord;citation_author=J. M. Celton;citation_author=G. Linsmith;citation_author=C. Becker;citation_author=N. Choisne;citation_title=High-quality de novo assembly of the apple genome and methylome dynamics of early fruit development.;citation_pages=1099-1106;citation_volume=49;citation_year=2017;citation_doi=doi: 10.1038/ng.3886" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=E. A. Di Pierro;citation_author=L. Gianfranceschi;citation_author=M. Di Guardo;citation_author=H. J. Koehorst-van Putten;citation_author=J. W. Kruisselbrink;citation_title=A high-density, multi-parental SNP genetic map on apple validates a new mapping approach for outcrossing species.;citation_pages=16057;citation_volume=3;citation_year=2016;citation_doi=10.1038/hortres.2016.57" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. Eide;citation_author=M. Broderius;citation_author=J. Fett;citation_author=M. L. Guerinot;citation_title=A novel iron-regulated metal transporter from plants identified by functional expression in yeast.;citation_pages=5624-5628;citation_volume=93;citation_year=1996;citation_issue=11;citation_pmid=8643627;citation_doi=10.1073/pnas.93.11.5624" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=R. V. Espley;citation_author=C. Brendolise;citation_author=D. Chagne;citation_author=S. Kutty-Amma;citation_author=S. Green;citation_title=Multiple repeats of a promoter segment causes transcription factor autoregulation in red apples.;citation_volume=21;citation_year=2009;citation_pmid=19151225;citation_doi=10.1105/tpc.108.059329" />
<meta name="citation_reference" content="citation_journal_title=Microbiol. Res.;citation_author=H. Y. Fan;citation_author=J. J. Ru;citation_author=Y. Y. Zhang;citation_author=Q. Wang;citation_author=Y. Li;citation_title=Fengycin produced by Bacillus subtilis 9407 plays a major role in the biocontrol of apple ring rot disease.;citation_pages=89-97;citation_volume=199;citation_year=2017;citation_doi=10.1016/j.micres.2017.03.004" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomes Genetics;citation_author=K. M. Gardner;citation_author=P. Brown;citation_author=T. F. Cooke;citation_author=S. Cann;citation_author=F. Costa;citation_title=Fast and cost-effective genetic mapping in apple using next-generation sequencing.;citation_volume=4;citation_year=2014;citation_pmid=25031181;citation_doi=10.1534/g3.114.011023" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. Geng;citation_author=C. Jiang;citation_author=J. Yang;citation_author=L. Wang;citation_author=X. Wu;citation_title=Rapid identification of candidate genes for seed weight using the SLAF-Seq method in Brassica napus.;citation_pages=e0147580;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0147580" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Grattapaglia;citation_author=R. Sederoff;citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers.;citation_pages=1121-1137;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982566" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=C. Gu;citation_author=L. Wang;citation_author=W. Wang;citation_author=H. Zhou;citation_author=B. Ma;citation_title=Copy number variation of a gene cluster encoding endopolygalacturonase mediates flesh texture and stone adhesion in peach.;citation_volume=67;citation_year=2016;citation_pmid=26850878;citation_doi=10.1093/jxb/erw021" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Plant Pathol.;citation_author=Y. Guan;citation_author=R. Chang;citation_author=G. Liu;citation_author=Y. Wang;citation_author=T. Wu;citation_title=Role of lenticels and microcracks on susceptibility of apple fruit to Botryosphaeria dothidea.;citation_pages=317-330;citation_volume=143;citation_year=2015;citation_doi=10.1007/s10658-015-0682-z" />
<meta name="citation_reference" content="citation_journal_title=Plant Protection;citation_author=L. Guo;citation_author=J. Li;citation_author=B. H. Li;citation_author=X. Zhang;citation_author=Z. Zhou;citation_title=Investigations on the occurrence and chemical control of Botryosphaeria canker of apple in China.;citation_pages=120-123;citation_volume=35;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Food Funct.;citation_author=X. F. Guo;citation_author=B. Yang;citation_author=J. Tang;citation_author=J. J. Jiang;citation_author=D. Li;citation_title=Apple and pear consumption and type 2 diabetes mellitus risk: a meta-analysis of prospective cohort studies.;citation_pages=927-934;citation_volume=8;citation_year=2017;citation_doi=10.1039/C6FO01378C" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Z. Y. Han;citation_author=Y. A. Hu;citation_author=Y. D. Lv;citation_author=J. K. C. Rose;citation_author=Y. Q. Sun;citation_title=Natural variation underlies differences in ETHYLENE RESPONSE FACTOR17 activity in fruit peel degreening.;citation_volume=176;citation_year=2018;citation_pmid=29431631;citation_doi=10.1104/pp.17.01320" />
<meta name="citation_reference" content="Harris, R. S., 2007 Improved pairwise alignment of genomic DNA. Ph.D. Thesis, The Pennsylvania State University." />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. T. Hill;citation_author=B. L. Demarest;citation_author=B. W. Bisgrove;citation_author=B. Gorsi;citation_author=Y. C. Su;citation_title=MMAPPR: mutation mapping analysis pipeline for pooled RNA-seq.;citation_volume=23;citation_year=2013;citation_pmid=23299975;citation_doi=10.1101/gr.146936.112" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Illa-Berenguer;citation_author=J. Van Houten;citation_author=Z. Huang;citation_author=E. van der Knaap;citation_title=Rapid and reliable identification of tomato fruit weight and locule number loci by QTL-seq.;citation_pages=1329-1342;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-015-2509-x" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=N. T. Keen;citation_title=Gene-For-Gene Complementarity In Plant-Pathogen Interactions;citation_pages=447-463;citation_volume=24;citation_year=1990;citation_issue=1;citation_pmid=2088175;citation_doi=10.1146/annurev.ge.24.120190.002311" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. Kim;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=HISAT: a fast spliced aligner with low memory requirements.;citation_pages=357-360;citation_volume=12;citation_year=2015;citation_pmid=25751142;citation_doi=10.1038/nmeth.3317" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kobayashi;citation_author=N. Goto-Yamamoto;citation_author=H. Hirochika;citation_title=Retrotransposon-Induced Mutations in Grape Skin Color;citation_pages=982-982;citation_volume=304;citation_year=2004;citation_issue=5673;citation_pmid=15143274;citation_doi=10.1126/science.1095011" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=M. Kruijt;citation_author=M. J. DE Kock;citation_author=P. J. de Wit;citation_title=Receptor-like proteins involved in plant disease resistance.;citation_pages=85-97;citation_volume=6;citation_year= 2005;citation_issue=1;citation_pmid=20565641;citation_doi=10.1111/j.1364-3703.2004.00264.x" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=M. Kunihisa;citation_author=S. Moriya;citation_author=K. Abe;citation_author=K. Okada;citation_author=T. Haji;citation_title=Genomic dissection of a ‘Fuji’ apple cultivar: re-sequencing, SNP marker development, definition of haplotypes, and QTL detection.;citation_pages=499-515;citation_volume=66;citation_year=2016;citation_doi=10.1270/jsbbs.16018" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=F. Laurens;citation_author=M. J. Aranzana;citation_author=P. Arus;citation_author=D. Bassi;citation_author=M. Bink;citation_title=An integrated approach for increasing breeding efficiency in apple and peach in Europe.;citation_pages=11;citation_volume=5;citation_year=2018;citation_doi=10.1038/s41438-018-0016-3" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. H. Lee;citation_author=H. W. Choi;citation_author=B. K. Hwang;citation_title=The pepper E3 ubiquitin ligase RING1 gene, CaRING1, is required for cell death and the salicylic acid-dependent defense response.;citation_volume=156;citation_year=2011;citation_pmid=21628629;citation_doi=10.1104/pp.111.177568" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Res.;citation_author=H. S. Lee;citation_author=G. H. Kim;citation_author=S. I. Kwon;citation_author=J. H. Kim;citation_author=Y. S. Kwon;citation_title=Analysis of ‘Fuji’ apple somatic variants from next-generation sequencing.;citation_volume=15;citation_year=2016;citation_doi=10.4238/gmr.15038185" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol. J.;citation_author=Y. Li;citation_author=L. R. Han;citation_author=Y. Y. Zhang;citation_author=X. C. Fu;citation_author=X. Y. Chen;citation_title=Biological control of apple ring rot on fruit by Bacillus amyloliquefaciens 9001.;citation_pages=168-173;citation_volume=29;citation_year=2013;citation_doi=10.5423/PPJ.SI.08.2012.0125" />
<meta name="citation_reference" content="citation_journal_title=J. Phytopathol.;citation_author=H. T. Liu;citation_author=Y. J. Zhang;citation_author=C. M. Li;citation_author=Y. Wang;citation_author=G. J. Liu;citation_title=Evaluation of the resistance of Malus germplasm to bot canker caused by Botryosphaeria dothidea.;citation_pages=511-515;citation_volume=159;citation_year=2011;citation_doi=10.1111/j.1439-0434.2011.01794.x" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=K. J. Livak;citation_author=T. D. Schmittgen;citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.;citation_pages=402-408;citation_volume=25;citation_year=2001;citation_issue=4;citation_pmid=11846609;citation_doi=10.1006/meth.2001.1262" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. I. Love;citation_author=W. Huber;citation_author=S. Anders;citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.;citation_pages=550;citation_volume=15;citation_year=2014;citation_pmid=25516281;citation_doi=10.1186/s13059-014-0550-8" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. Lu;citation_author=T. Lin;citation_author=J. Klein;citation_author=S. Wang;citation_author=J. Qi;citation_title=QTL-seq identifies an early flowering QTL located near flowering locus T in cucumber.;citation_pages=1491-1499;citation_volume=127;citation_year=2014;citation_pmid=24845123;citation_doi=10.1007/s00122-014-2313-z" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genom. Biol.;citation_author=D. Lv;citation_author=J. Y. Zhang;citation_author=Z. Zhang;citation_author=Z. Q. Zhou;citation_author=X. K. Chen;citation_title=The relationship between rDNA-ITS sequences and biological characteristics of the apple ring rot pathogen Botryosphaeria berengeriana de Not f. sp. piricola (Nose).;citation_pages=104;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=P. M. Magwene;citation_author=J. H. Willis;citation_author=J. K. Kelly;citation_title=The statistics of bulk segregant analysis using next generation sequencing.;citation_pages=e1002255-e1002255;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072954;citation_doi=10.1371/journal.pcbi.1002255" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. W. Michelmore;citation_author=I. Paran;citation_author=R. V. Kesseli;citation_title=Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.;citation_pages=9828-9832;citation_volume=88;citation_year=1991;citation_issue=21;citation_pmid=1682921;citation_doi=10.1073/pnas.88.21.9828" />
<meta name="citation_reference" content="citation_journal_title=Molecular biotechnology;citation_journal_abbrev=Mol Biotechnol;citation_author=C. Moser;citation_author=P. Gatto;citation_author=M. Moser;citation_author=M. Pindo;citation_author=R. Velasco;citation_title=Isolation of functional RNA from small amounts of different grape and apple tissues.;citation_pages=95-100;citation_volume=26;citation_year=2004;citation_issue=2;citation_pmid=14764934;citation_doi=10.1385/MB:26:2:95" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Mou;citation_author=Z. Liu;citation_author=D. Guan;citation_author=A. Qiu;citation_author=Y. Lai;citation_title=Functional analysis and expressional characterization of rice ankyrin repeat-containing protein, OsPIANK1, in basal defense against Magnaporthe oryzae attack.;citation_pages=e59699;citation_volume=8;citation_year=2013;citation_pmid=23555750;citation_doi=10.1371/journal.pone.0059699" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. N. Ngaki;citation_author=B. Wang;citation_author=B. B. Sahu;citation_author=S. K. Srivastava;citation_author=M. S. Farooqi;citation_title=Tanscriptomic study of the soybean-Fusarium virguliforme interaction revealed a novel ankyrin-repeat containing defense gene, expression of whose during infection led to enhanced resistance to the fungal pathogen in transgenic soybean plants.;citation_pages=e0163106;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0163106" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Plant Sci;citation_author=C. J. Park;citation_author=D. F. Caddell;citation_author=P. C. Ronald;citation_title=Protein phosphorylation in plant immunity: insights into the regulation of pattern recognition receptor-mediated signaling.;citation_pages=177-177;citation_volume=3;citation_year=2012;citation_pmid=22876255;citation_doi=10.3389/fpls.2012.00177" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=M. K. Pandey;citation_author=A. W. Khan;citation_author=V. K. Singh;citation_author=M. K. Vishwakarma;citation_author=Y. Shasidhar;citation_title=QTL-seq approach identified genomic regions and diagnostic markers for rust and late leaf spot resistance in groundnut (Arachis hypogaea L.).;citation_pages=927-941;citation_volume=15;citation_year=2017;citation_doi=10.1111/pbi.12686" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. Pertea;citation_author=G. M. Pertea;citation_author=C. M. Antonescu;citation_author=T. C. Chang;citation_author=J. T. Mendell;citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads.;citation_pages=290-295;citation_volume=33;citation_year=2015;citation_pmid=25690850;citation_doi=10.1038/nbt.3122" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=R. Piskol;citation_author=G. Ramaswami;citation_author=J. B. Li;citation_title=Reliable identification of genomic variants from RNA-Seq data.;citation_pages=641-651;citation_volume=93;citation_year=2013;citation_pmid=24075185;citation_doi=10.1016/j.ajhg.2013.08.008" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. Ries;citation_author=D. Holtgrawe;citation_author=P. Viehover;citation_author=B. Weisshaar;citation_title=Rapid gene identification in sugar beet using deep sequencing of DNA from phenotypic pools selected from breeding panels.;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2566-9" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Rombauts;citation_author=P. Dehais;citation_author=M. Van Montagu;citation_author=P. Rouze;citation_title=PlantCARE, a plant cis-acting regulatory element database.;citation_volume=27;citation_year=1999;citation_pmid=9847207;citation_doi=10.1093/nar/27.1.295" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Y. Shang;citation_author=Y. Ma;citation_author=Y. Zhou;citation_author=H. Zhang;citation_author=L. Duan;citation_title=Biosynthesis, regulation, and domestication of bitterness in cucumber;citation_pages=1084-1088;citation_volume=346;citation_year=2014;citation_issue=6213;citation_pmid=25430763;citation_doi=10.1126/science.1259215" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=V. K. Singh;citation_author=A. W. Khan;citation_author=D. Jaganathan;citation_author=M. Thudi;citation_author=M. Roorkiwal;citation_title=QTL-seq for rapid identification of candidate genes for 100-seed weight and root/total plant dry weight ratio under rainfed conditions in chickpea.;citation_pages=2110-2119;citation_volume=14;citation_year=2016;citation_doi=10.1111/pbi.12567" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Sun;citation_author=Y. Chang;citation_author=F. Yang;citation_author=Y. Wang;citation_author=H. Li;citation_title=A dense SNP genetic map constructed using restriction site-associated DNA sequencing enables detection of QTLs controlling apple fruit quality.;citation_pages=747;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1946-x" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=H. Takagi;citation_author=A. Abe;citation_author=K. Yoshida;citation_author=S. Kosugi;citation_author=S. Natsume;citation_title=QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations.;citation_pages=174-183;citation_volume=74;citation_year=2013;citation_pmid=23289725;citation_doi=10.1111/tpj.12105" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. M. Takos;citation_author=F. W. Jaffe;citation_author=S. R. Jacob;citation_author=J. Bogs;citation_author=S. P. Robinson;citation_title=Light-induced expression of a MYB gene regulates anthocyanin biosynthesis in red apples.;citation_volume=142;citation_year=2006;citation_pmid=17012405;citation_doi=10.1104/pp.106.088104" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=S. Terakami;citation_author=S. Moriya;citation_author=Y. Adachi;citation_author=M. Kunihisa;citation_author=C. Nishitani;citation_title=Fine mapping of the gene for susceptibility to black spot disease in Japanese pear (Pyrus pyrifolia Nakai).;citation_pages=271-280;citation_volume=66;citation_year=2016;citation_doi=10.1270/jsbbs.66.271" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. Varet;citation_author=B. Hause;citation_author=G. Hause;citation_author=D. Scheel;citation_author=J. Lee;citation_title=The Arabidopsis NHL3 gene encodes a plasma membrane protein and its overexpression correlates with increased resistance to Pseudomonas syringae pv. tomato DC3000.;citation_volume=132;citation_year=2003;citation_pmid=12913158;citation_doi=10.1104/pp.103.020438" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=A. Varet;citation_author=J. Parker;citation_author=P. Tornero;citation_author=N. Nass;citation_author=T. Nurnberger;citation_title=NHL25 and NHL3, two NDR1/HIN1-1ike genes in Arabidopsis thaliana with potential role(s) in plant defense.;citation_pages=608-616;citation_volume=15;citation_year=2002;citation_issue=6;citation_pmid=12059109;citation_doi=10.1094/MPMI.2002.15.6.608" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=E. A. Van der Biezen;citation_author=J. D. Jones;citation_title=Plant disease-resistance proteins and the gene-for-gene concept.;citation_pages=454-456;citation_volume=23;citation_year=1998;citation_issue=12;citation_pmid=9868361;citation_doi=10.1016/S0968-0004(98)01311-5" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. Velasco;citation_author=A. Zharkikh;citation_author=J. Affourtit;citation_author=A. Dhingra;citation_author=A. Cestaro;citation_title=The genome of the domesticated apple (Malus Ã domestica Borkh.).;citation_pages=833-839;citation_volume=42;citation_year=2010;citation_issue=10;citation_pmid=20802477;citation_doi=10.1038/ng.654" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Wang;citation_author=M. Li;citation_author=H. Hakonarson;citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.;citation_volume=38;citation_year=2010;citation_pmid=20601685;citation_doi=10.1093/nar/gkq603" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=X. Wang;citation_author=Y. Xu;citation_author=S. Zhang;citation_author=L. Cao;citation_author=Y. Huang;citation_title=Genomic analyses of primitive, wild and cultivated citrus provide insights into asexual reproduction.;citation_pages=765-772;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3839" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. T. Win;citation_author=J. Vegas;citation_author=C. Zhang;citation_author=K. Song;citation_author=S. Lee;citation_title=QTL mapping for downy mildew resistance in cucumber via bulked segregant analysis using next-generation sequencing and conventional methods.;citation_pages=199-211;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2806-z" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Xie;citation_author=X. Mao;citation_author=J. Huang;citation_author=Y. Ding;citation_author=J. Wu;citation_title=KOBAS 2.0: a web server for annotation and identification of enriched pathways and diseases.;citation_volume=39;citation_year=2011;citation_pmid=21715386;citation_doi=10.1093/nar/gkr483" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=L. B. Xing;citation_author=D. Zhang;citation_author=X. M. Song;citation_author=K. Weng;citation_author=Y. W. Shen;citation_title=Genome-wide sequence variation identification and floral-associated trait comparisons based on the re-sequencing of the ‘Nagafu No. 2’ and ‘Qinguan’ varieties of apple (Malus domestica Borkh.).;citation_pages=708;citation_volume=7;citation_year=2016;citation_doi=10.3389/fpls.2016.00908" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Yang;citation_author=D. Huang;citation_author=W. Tang;citation_author=Y. Zheng;citation_author=K. Liang;citation_title=Mapping of quantitative trait loci underlying cold tolerance in rice seedlings via high-throughput sequencing of pooled extremes.;citation_pages=e68433;citation_volume=8;citation_year=2013;citation_pmid=23935868;citation_doi=10.1371/journal.pone.0068433" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Rep.;citation_author=X. Y. Yu;citation_author=B. B. Du;citation_author=Z. H. Gao;citation_author=S. J. Zhang;citation_author=X. T. Tu;citation_title=Apple ring rot-responsive putative microRNAs revealed by high-throughput sequencing in Malus × domestica Borkh.;citation_pages=5273-5286;citation_volume=41;citation_year=2014;citation_doi=10.1007/s11033-014-3399-8" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. M. Zdobnov;citation_author=R. Apweiler;citation_title=InterProScan - an integration platform for the signature-recognition methods in InterPro;citation_pages=847-848;citation_volume=17;citation_year=2001;citation_issue=9;citation_pmid=11590104;citation_doi=10.1093/bioinformatics/17.9.847" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. Zhang;citation_author=Y. Lv;citation_author=Y. Wang;citation_author=J. K. Rose;citation_author=F. Shen;citation_title=TATA box insertion provides a selection mechanism underpinning adaptations to Fe deficiency.;citation_volume=173;citation_year=2017;citation_pmid=27881725;citation_doi=10.1104/pp.16.01504" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=S. Zhang;citation_author=W. Chen;citation_author=L. Xin;citation_author=Z. Gao;citation_author=Y. Hou;citation_title=Genomic variants of genes associated with three horticultural traits in apple revealed by genome re-sequencing.;citation_pages=14045;citation_volume=1;citation_year=2014;citation_doi=10.1038/hortres.2014.45" />
<meta name="citation_reference" content="citation_journal_title=J. Phytopathol.;citation_author=Y. Zhuang;citation_author=H. T. Liu;citation_author=C. M. Li;citation_author=Y. Wang;citation_author=Y. B. Zhao;citation_title=INHERITANCE OF AND MOLECULAR MARKERS FOR SUSCEPTIBILITY OF MALUS DOMESTICA TO FRUIT RING ROT (BOTRYOSPHAERIA DOTHIDEA);citation_pages=782-788;citation_volume=159;citation_year=2011;citation_doi=10.1111/j.1439-0434.2011.01839.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Apple fruit ring rot (FRR), caused by Botryosphaeria dothidea , is a worldwide disease that impacts Asian apple production regions. However, no substantial progress has thus far been made toward the mapping of candidate genes or the development of effective genetic makers. In this five-year study, the resistance of 1,733 F1 hybrids from the cross ‘Jonathan’ × ‘Golden Delicious’ was phenotyped by non-wounding inoculation with four B. dothidea isolates. We first conducted systematic comparison of different analytic strategies for bulk segregant analysis by re-sequencing (BSA-Seq) and obtained suitable one for outbreeding species such as Malus . Forty-six quantitative trait loci (QTL) for resistance/susceptibility to the four isolates, including one QTL ‘hotspot’ on chromosome 14, were identified via BSA-Seq. Using integrated multi-omics strategies including RNA-sequencing, parental re-sequencing, BSA-Seq and meta-analysis of RNA-sequencing, fifty-seven candidate genes and corresponding functional mutations from the QTL were predicted. Functional mutations located on the candidate genes were validated using kompetitive allele-specific PCR in hybrids and Malus germplasm accessions with extremely resistant/susceptible phenotypes. Ten effective markers for apple ring rot were developed. The results provide an example of rapid candidate gene mapping for complex traits in outbreeding species." />
<meta name="og-title" property="og:title" content="Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1663" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Apple fruit ring rot (FRR), caused by Botryosphaeria dothidea , is a worldwide disease that impacts Asian apple production regions. However, no substantial progress has thus far been made toward the mapping of candidate genes or the development of effective genetic makers. In this five-year study, the resistance of 1,733 F1 hybrids from the cross ‘Jonathan’ × ‘Golden Delicious’ was phenotyped by non-wounding inoculation with four B. dothidea isolates. We first conducted systematic comparison of different analytic strategies for bulk segregant analysis by re-sequencing (BSA-Seq) and obtained suitable one for outbreeding species such as Malus . Forty-six quantitative trait loci (QTL) for resistance/susceptibility to the four isolates, including one QTL ‘hotspot’ on chromosome 14, were identified via BSA-Seq. Using integrated multi-omics strategies including RNA-sequencing, parental re-sequencing, BSA-Seq and meta-analysis of RNA-sequencing, fifty-seven candidate genes and corresponding functional mutations from the QTL were predicted. Functional mutations located on the candidate genes were validated using kompetitive allele-specific PCR in hybrids and Malus germplasm accessions with extremely resistant/susceptible phenotypes. Ten effective markers for apple ring rot were developed. The results provide an example of rapid candidate gene mapping for complex traits in outbreeding species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RTkbF3eKb-Jy1ReSMdxav5eTz9cSQm-jkO4tIk-w0Yg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492883","apath":"\/ggg\/9\/5\/1663.atom","pisa":"ggg;9\/5\/1663","ac":{"\/ggg\/9\/5\/1663.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1663.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1663"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1663"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1663":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022neM7z5JxRDaMaD1rfL\\\\\\\/NMe1wUBDMunBDY9V19Wk\\\\\\\/CgvhOG8DKPIuK3NWyozT\\\\\\\/EpbuduCJIhzP5Fb57+FHL0dKOH7w\\\\\\\/8I8k\\\\\\\/MDKPW3ol725JX+vrt3iQJ+F\\\\\\\/VXw2Y3FiBJdPXgauZ\\\\\\\/qa9BzchuCTandPoFEIIJqgKJybrYJ\\\\\\\/7q\\\\\\\/VQRiNGTGLIWYnjh7qVN6VqxZbA9J+ULt3ALFsYkXGzYrFtnZMKV0HLEPhmd5YFNT3zUhzTdOlrD4sU92GF74SeTSc3IQTpjJlbzFzV4eS7+CgDfUe62Q3NL1FE8ol0WAjNVxbc04eijAJEeIrs1OTKNk59nRyrVvhuBiKItf9k8p50mtS56Q8OjA8hgLBWkjHdgC7hyJhg4LvzQw5qneYYR7GUkIpIcuiezbDM04blR3fyhLm1u3I\\\\\\\/KgsTnCrri2y5TfOvLFW2YWYJQg\\\\\\\/7pZx6G3saKyK453BaZpUtAXSD48dzAwDqPM9z2AsZNa\\\\\\\/kdcjuCCEBJK6ykT4yCsO\\\\\\\/VrSccynmmfAqJqkV7fFYiSzvaDpiBlYGiCD4crvVeSCq\\\\\\\/sApl2YGWgxWjkA0Dm6zDw+i7pxouabH+sgkekWWf5ji6bDGhFUaom7idyQ\\\\\\\/Ls0JhaIXvUcTbU\\\\\\\/sBomGxfZFYURu4AHFeai7N5eZlynBtv2tyBQq+w7RvuaResWaGJTkufFlnhLXOR8qBBDFAHzrINjRsQ3mlrbaE1JTo73LK29vT9LGN1GM+wTlTUFzgAIe4aIhaacpRsof0cl4M\\\\\\\/q7aTwF7WF\\\\\\\/x3BQc2q7NU2KdbkkbLhCOsNJ3tnwaIQybdGRHmffIE0kIEYV8LsMAvHOTfonxkEH95aGiR4t4sYN\\\\\\\/c1OGxd7EOa8wHe+Dpef7QgYumuRixYadsesbV3\\\\\\\/KO93u8mQqgOTyNAbks7z4UvHz\\\\\\\/bMo6TkRQnq6a\\\\\\\/auopn4SDa1Ytsm07ytY4k28ESgG4YOcMKSGCPERX1bTxeppJ1bwGu5HDLpwk2d7tS+dWO7UdJeFMuzToECiBZmv9eHblr360DUmv7YS4dMqm99vUN0Mm2DI4Fi6S\\\\\\\/A\\\\\\\/nWL\\\\\\\/\\\\\\\/ezAsrFZo3xsgBUoliDlAASYXGAABhmvT11Iu\\\\\\\/rJpOF+IB9pBPwDr4bms8A8f0MBGyBHZ4fRnO2Uj5s24dEMl8h6MUoAEK4L3o1lV+txScX2E8IMyCeLTWUl8xJ8\\\\\\\/t4xFUp5+C4bAqCYRwWvhxbw26R291poxeFQ0TRvfSB5CNtQ9iT36KRRq56hE0ylub423yLdWihH+CX8e0joCwDT4QMWrKNtXf\\\u0022,\\\u0022iv\\\u0022:\\\u0022oLCPNWMTkI1TEWSDD1yRAQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229eb070b90285de95c4e8aeb824455c225465b912234ba6c6b4d97c62f553bf41\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R+nBOH9zn5LWhNuaeRXQUeB6916OgNxSn7VYI+eYuJniO7DxMdFt2pLvEaXBD98EJyfqmDGlgiN25gBLgCMQb4Xj1KeCOAwCbGosfw4Gb8tCEAXa2a0Q6yrJ6UgWvspq9XGU6ySSy1qDjwHjIsCjxcPlvR5fanT+IRJObW+kpOKGafwSdo1mL+52z0MNrz+3FvqxY7\\\\\\\/yZE8YNafKkRMIahAYNnAOrqw8cLPPKTxFZrwR+Lx5u71emwCRg7pG185flVg3ZtdA2wPEblUT7d\\\\\\\/ZWq\\\\\\\/2zl8gIh\\\\\\\/K28DcNUnqUGK3GE+W2WK79C5bN5pqZ11hGbfeeUd0mXP1JjzCejyAtA1s9Ih5nFMB23+c8o4g3nBcNsPMtUhs\\\\\\\/c2qB5TayLKkhMpVX7WtjsBqfGDdRMRMgpfCFpOAQAGJWx0W0DRrXVYSah5h8\\\\\\\/GSO0y\\\\\\\/Bbmf26XcfIFjHZg\\\\\\\/kDeI0SSqv5CJ93uFJZGiiEhOEwXH+8AYut5x9tf8B\\\\\\\/+CNStFLOKrxO3zS8Wfb9vambFBDgbDJTmooQLUrdGKH+tj3XC2LH5yuvpMCOMeZKfHO0BjzVdAsLuhepk+4rvKwJ4QD+ribKE8tde+UmtX5tq6UjZx7ChPGDjVc3iezU9L\\\\\\\/TAgV4kCDETTZwnyEE09beHWLCqIWbXpbRWOkztJa370LYuO96iypCxXFwW6YdYpYOmZt8zDBUzraE3OPPEQNEDwrIC487okqkHbV3nCuHRSencpznYzhL\\\\\\\/1RB7BVcxHJdYfnI0Ag0edrEWOcE09zB\\\\\\\/3ARqwnxwDrKbuD3dRW2H+IMEItjKFm5wcDPy56Xsagk963vOLnymkqJYChm7dKwKlNov9qDEDUKAn6CLm\\\\\\\/EYxtK0sdTJHgR68Fism\\\\\\\/GjsMxYySSvE4Ij\\\\\\\/Q50s+AmcpG42GNkHtBsFksWnXAnt7TVb8f4NzB5CSuzEXSe3QKXtgHWLUZBotYCdJJxxZjbXAHlGW0FxJVK7im7J2aHagXXq5L80+ldjIqzqPtMuvO4qHn0Czi3shCaZrflaP77wwOUlLRT91GjQtcVMa6QrkWiQtZzRd\\\\\\\/QgnL0XR\\\\\\\/RSwz4pUGjo5RQHOdGc0VZ4f7ZyQxtZw+e43wtiHs\\\\\\\/W95jn+pO\\\\\\\/ZCjw6tRJkNGW073WigDkxRoRI8f5E9\\\\\\\/jR47Tlhko8FxW4bJgiqDqXYshyMFvWjuz3KjOGMpTt6wscnVyPwxUY7\\\\\\\/ooOt80JzW+EXrJUlwcmk2zojP8XFpAH0G59qsZiifccjy93VlEK8jkx9lfK6SIVUTHl2\\\\\\\/omM3YAF2yy1wSp0FULyWOljLArS2B2\\\\\\\/fr2\\\\\\\/PbVUDHBz+JMD52xLaK5FnQi06jcIptIZ5TrQjEAj+ROqhrM7lIdEnLTcxlWFwORxkWtwrexUB5KRGlMSFExeRLs3Gsm00Q99Wp27hRIRRT+B+Wygoukh97O7HEWvu4UtG8ZTLYzTGPxgcJ0vaZ0K287Zl\\\\\\\/RplKnCHiiJIi\\\\\\\/KzeljKBcsDoc+oY1cH7LD9bDCIYYFG8TsGXZM2gXVf5C6fmv4bDDBJp049TtruEO4IO5\\\\\\\/eBil8x+9bteOrkRKUHjQKZM1+XXmUuQudst3bUz9AjseTszvNsqeyme3wt9Y\\\\\\\/xEc92ZpZ1N7kLtPEX7h7FZ6foXvwRsy8+OABL6thXDcK3YM8CMuWwyGZTFEACX5YxJ99zgWUaMgKTkIUTl2SSKSB\\\\\\\/Vby2INRS5CJOmU75Tg4CDLLoZIew90j+5rXEAEeRgyXrI0voezC7H2VIL5gUsLr8LCsTGiMQIaVc1aMho5QaH3yD3lvpiLDohHCoaUEXZGCIfBh\\\\\\\/Tq83DopWrFxwqQtkm6Ig3xHHJwE0NQNas61fJ3hqU9ArspMFlo6GjhGX6JF9iaAB+6GyVx5mTpZjU8kPf\\\\\\\/Js8e61Nqzv50ZuhKQC8M5uXb+Nv3Sb+NGXhlcsrM8iB6aGtOE2yxBu3MF1FR2yfoeRR8x79\\\\\\\/4A9UJsp8v8kJ9oyATHanf2FzCzUmHQUnkVTpUQJrVCOjQC\\\\\\\/gvmxYIoBYiddBl9RBk9qazX1WVp3qUksRb0sxKEpZC\\\\\\\/5mng+wn1ShGiyRVSZKf7pxDHTuvCxeiy7zEjSKIF\\\\\\\/wrEn3E\\\\\\\/38vVVlGjLhJy9L0+Om+2oBKlQrf5SiKw\\\\\\\/ary1+fbO5X5iYT\\\\\\\/mF13\\\\\\\/7+KruLC8axwpeim+YgNUfwG7CRJPyWwxEMYSSypJ\\\\\\\/qTDbi186MqOzWzRPM+i6Ao0+lUyC\\\\\\\/\\\\\\\/\\\\\\\/STWCEvLHWiISXwtP3nBN2Bj4wQPTxaT4KVrpyjVQO49cEtmrZQWKsmIxGI8suIve9pLvglfWdMD8iBuUgY0SWlql6yKX68O1JLkU1LGPQnpdiYNx52Q9kgZZ8fuVznseEkKjWSddTiJDgyWrGSjlsGhSNS3u\\\\\\\/lNpouYb875pB9m\\\\\\\/oFiLnZOnqqbA0RfM86pD3LZ81srIljuNOlsrkRkA7y4f0oYgPIF5TrbEimpHc2QuHUaXwADzsZwei0w0\\\\\\\/OFeufPFGCeTBI37Now8cIXZOHJBOJKTrbZhEAZ2MXwqvm4LcXwgoSlVEEBSmglPJook2shtB0+YgwZiZOhUeQ7cfFL9OXjn3wsCGfT3mt2AkbT\\\\\\\/mjVg+To+vbMyG3ZUPFUEVuDYKNwx2SV7sEi7YGHaaVuxdJT5r\\\\\\\/yYIplQ6E7HfpjH1TokMo9VpA3mjwg6Xt1JkDNxn8SIv21e7G4eThJMWMl07WYpfSgoT1cZmv\\\\\\\/FH8j2KRpmwLDSBGU85BR2KBrHQGMqFKyuFz7I8qtfku\\\\\\\/mXRUobVfFMKgY3aMrobs0S+8IM+eZPbjBGueiCOW7IuslZwEJXsDFpe1SIZDQC++p9GKDd2rv9ieztR9Bn2VSdwKuefINRbeDuKcJeI\\\\\\\/8ENtjFrtvo0chkkvRur\\\\\\\/1Bg1PVq7v2dvXADplrx472sJHgBU7fVXWvPCTzLPD1615jm4JPByg\\\\\\\/6HP\\\\\\\/aotNMPH3tPFd2SQPrerDlMHSSuCpneCIfIPN5frNPKfGFuBQiKL6vUCaAEUxwpPUHxgwvEk2w1cQHub7ye02HImKG5fw+2U9ulwBtxl7uZ8B9qEb5VF7aepZ2cvgaT1Jh8zFXlY9Iw3z2ZUd2VCefE7vqZaf24ko3cYwWRDrg1nTk7OsHZLU2sDCGOgekuK1r5ipJdFpva9otW0lRCsGqyiYIVhcyebRG2GkZStdGRcaeNrrqwvexEuRcKwUH2wWFnp1CyUQZ7AJHAbTxtIamDW87dq9FrMyHtO2D8DolsbGEKSd64qGE7LBs3eSC4W\\\\\\\/dsNQwiUzWw8jjZifjuoqGi8mnV+cJ3vAZ1gH0NL\\\\\\\/\\\\\\\/xTX5bOQ+mZTa4xst\\\\\\\/PpFOEy9Oneo0H5kqODP+ehZ71hWI\\\\\\\/Pu910hFeHWsxhmBoaedZFRhwZZyVCTaN25CP\\\\\\\/4G3i1\\\\\\\/izo0G0\\\\\\\/IUlfaTpB+r5ljKNKWn9hXHNnu9bGrY2118o1qeHWJiqfzBMJV6+Lu9+9rJ9B4ynSmNvM2pDoMWsosQCobrD8R4VAt4+LW7NtO1sV\\\\\\\/n5oKHPoIJtoUTdPJ8g6PxDe75418IowrWxYY\\\\\\\/kSvrYaMke5vRcqNc5Db2OmqEiKkCqVuPmKwoXkSDJC8maI2HONii\\\\\\\/wMVGSgXf71gEt564QOAdOCWFGSvMfNTQt6oJZaZ+6f0VVFemwpe1Q3IwSt2ffXl50lrrcus369OnbGOhZRM4LmaVBuZpgq+fXcKSYIS8hsYagP6EG689SWfgJ7boipKYzWkAskyKBGkDDWK6ANr\\\\\\\/N6SAu1yrTJ0GRo8wGqi6QV30rT1wWiiDV7VmEQld712DSyrLmh1VXn4ZZY7l2KVynP9+\\\\\\\/YdCOJfPf2Hj48m+YIrixhqUo2xJZBEd0uaAwiWtvgCHfCanBcz7JQFuvyCSU8udDXx8T0uD7n3Fxn3ncsvMWkHlwuEG3eFad0QqXurJUE06NvgU4QUiEVXeN52SF8FC\\\\\\\/wJy3H1FjQPXZys8dk+ornsM1RJjGT0FV3QGxE0VMFFuEo5WVOjJCRznEjRY4F0yE68oZRygNIDdgFhZ0d7fEtpOEGUjoAxXoUM0zwx8D7YF9ikKqEqcVac5nazF+ZDi\\\\\\\/amwnPy2Ml88pRqA0EqqVCwq8\\\\\\\/G8ad05weuwcCuZ0mTVsF3b7yPiODCkedv5ud4BBPLX3OLYbbJP5huHCJzTUtQ1vBnkvEMnmaIe+pFf5fjqfAVYcgGEIIUJP5Gy4QAia5JO7Wn9qkbleXkIHEZ25\\\\\\\/SRer\\\\\\\/F6QD5VLnFor5h+hElicd\\\\\\\/wFvaL1pyMDtmvCZvffffRBfoMRZvoJvm7PBTA7pvrwdGHGbyzEDHyMjvEISFO0Nw8SBXcyEqk5\\\\\\\/GnbY+3dDxwqGBRwiseF\\\\\\\/kmJ4h1NxXh0p3NExUO3WKNcOmuuK+z4PhtQI0O1AIhIvLfRhiI\\\\\\\/JGMtbFX7NZfOYMQrvOGgH0E5RcfDBgOLsFi8zk2n6XSDhTYv9WuoiVH6Llla6sYE5Ocnu+CUYrG\\\\\\\/y\\\\\\\/SjxYllmbbJQYKuuNg1GEXdzAcHZoOqEmephq3ebFkP1JGz5w8xkKlX8bh79drpOJ9ASxAlCA9CitLWi\\\\\\\/nsoV\\\\\\\/ovhgFPsn1mAzVStcdyoeH+qEKQOI3\\\\\\\/WgRhRWaHIaXLmfLlw2MYjw43tkvv21LrjUTVUcQucROHWwW4Amdrnewdcvduo8hKHeSAYZka8xZIos9o89vbbq\\\\\\\/A7DvOmdWTpEcMK5pNFr1JWGZA8UUIJcK9dTcq7gQDEntXWNrBxEsETQCQ\\\\\\\/4dt7fBaDBEW\\\\\\\/RwVUVogbM3t3wh1h+kQowkrO\\\\\\\/vLgLN2TyvxlEIylVs2beY0dvOpdFJaHupQJw0hR1Sj0eb5Y80+ZMvJoSL\\\\\\\/Df2XgB0wmyl1r5AD8thsYAC54sShOGhKDB9MNpzCXYhdFzSmGRi1efSQc\\\\\\\/ufb3VYAFA47uEsCm6uoZGH1YU4AvkTI8KX53LTyweoZghwEKY8u1lm3Az7auolC4c1\\\\\\\/jap0u+U4GgxOPvUDZ7QrgL0Z1DzeypB9t8W3ompuiEUzaI2OT\\\\\\\/CUABMBZXgLwbH9miuqTlbLDVRqtTBcFav70FgClF4tzyAaU1Vja2advbvsw4Xl8247JfNX4Kjv3TfQrmyfB1popETavBHhORdhJTRTqT4AG0i6CYUM7c8Y4nHbG\\\\\\\/YYZR1\\\\\\\/YyJIh862HjcLB0gyZp3FOWJwcul1FtAf5h5mLlrnh6D69WnuBRrZFITO4uITSS6j2cg8rwk9JXlJ6cSbp5dYK5VooZNI7vRfdHM32ABiAQBwpqaZAx+I+2\\\\\\\/BtuYGokeDheV+cxtEw4rqPT6EMzYtk9Hr+ks9hQzeYt8m+1mNM+v80SOamWmbsy5Va3p8mRYSsZMBZe7KMpeb95V0LxHPPVwLglMh5UFw1LQgwOhs1AzT+Xzp0Nt6Ju90k8bC3xT0gODslAJbb5vG+QHpvTKSiGG7PhTciTSfnDLUyZEaohf1WgVChZMkmh\\\\\\\/rEEUWcQ4Geiot712o\\\\\\\/WSukK8lL95SRNZYCIDlRZ25wIzH1ZxgormQe2aztV2wiCHFYhXfFZwCuMFz0jpqnblQJ3cJuqRAceGAihfnAjlr9NbW7iB+O35bfEnAc8HEaU2l4BCNb1Oy099q3AKVRhdlztbh7BZlTppVC5OtS7uCHSWnz0sYd5L3bomrGFCaj4mtHnkq+teePvt4fT3r\\\\\\\/c2W0Um4Co3UkOoia\\\\\\\/zq50Q7t+Ceg7pp0aQJJ5Ps7qgLQDe\\\\\\\/0hhLpxepre9+78u8aUkNrD09PawGFAK4E6jAWw3AQL0wM+j5dfN8DgWICu4CeJjdIzK96fZ5wkK6qH4cRue1KZUTqAWglEN4DQqrU3doodNE79SQVynosHBPxrrXLgytg3YJ6nWfR5Yvtsjf5q7uQPxOD\\\\\\\/DUZ72V5BQGS\\\\\\\/IifNj5zm21WzSweswLKahiRkhxcdQTMs149lwZFEV04VuV087INq9YxWZ1yZWNRgwUMz7esfq+n7nxKTHZtjcX4To4+LnDp8FoNHGRquQx1P9VSY\\\\\\\/IbGc0EBJJinOIApkbRT04gdoVP5wTwJSGAsP711IbBFqP5KlBG1zoOvM7zf7alcbrt0WfOjT+TYtVJu7RWRrGaCEtCgiKo3XJavH81znOB4k7sqess\\\\\\\/AyAekGNq7VAyeLPy\\\\\\\/A0ohacnKztbudk6qeyPCAwUeczT7iwn7nc6mfuBHc2L7eilSsVJ8uFLZCChlgiZFndnR27ow18sZ9qQuyKD9Xv1\\\\\\\/2ZWWbXlB3SjsszwYrcmddrscFZU6iJsWSFnsS6b8CwVX13UKgTH82nRHa0PcVuMBX0SYtAORaAwv0fOXsqnjY3Jabq5BU11Xzk5a2ojcXxjaQ1qLucCFkDUwp1fpae9pXq2QNmX3UP086ZcJKoQ4ROyTO+obFsRIwT5y0MNeYPmwqumaB\\\\\\\/lgZENoZYLiCMktKtHoyZ6FthpSjAD9seFuWbf1wnLxpBFGvEs5ddGxibZNOXLI7aQRZpNjgqX3ublVC4JTqfCQwVHZ+gQHdPgW\\\\\\\/Y2\\\\\\\/Aor9ChtIvtTegHVinMun6CUwkkqpYvJhFoalQPFYfti1nEBUzRYyAMAyG2afELO5vkfAgCecV9zdyN187Rwn97jR62FBXYvAPZqn8x7x4dVmwydMmggiXqf\\\\\\\/uZe5USgacJiFAYab6BZ16B6yILWfYQMVf9WDX4AIA2UQoAxnmPB\\\\\\\/y1C2UUCOfg4LdCIe\\\\\\\/2cPyNBn5d5h9kC1Tcv5pY3skUioLFVldz1eJSp4a54XHr\\\\\\\/+SfH9txb3AhSIfF1IMc4ylzlT+XeFluWvNLA72astrdmLUAoREgTlXS0T006uackLv41z\\\\\\\/cfhhwtCY5qt1TVWYAf3NKCR7DLaOayHyhRD46TbpecRt\\\\\\\/wkm6YywGKb2Aoyy16NvqSBXr5dCvsoe5g6lObB7p+v2Z8Sjer0rgtLj9o69ejxKkf3OfvHduhR81geH8o6zMsheouq8Z5vI6oq+ATF2XSS9xO2qazHIxfnwY6s3AK0L8soBiuI1oCZzQEoX+t2\\\\\\\/7AtR1L4wV6RPE4rYnpDlOSAyK0KhUBFgWkPiIGwCi9eTaJR5w7OX\\\\\\\/GRQx6pRKLigjOtPzYQrzVBvpo5U9TU3HLvS9lD1n6jRcfowWkDyI\\\\\\\/eWwkObNDltTi7f1y9Z1gynZpX35dwOmg5l\\\\\\\/FPKV2rvdKg6pWL8O4b2uVR8V1LMJx+eOce2HvTeNalNlGRYbz1srTdXwzG6ARovPObJSrmWDbMKw\\\\\\\/v3qPoWT7eDe\\\\\\\/JbNVV7Jv0i3KwVNT+h6jInBRg\\\\\\\/BJh+iMb0klNw484qDTR\\\\\\\/rEsqHXxLCF3J1f\\\\\\\/DkDkR\\\\\\\/7E9rqz3YqORpam9XIOpco2biAb45O6v1jNETXevkTyH+4Z2QM0iNQgEJpa+lNFNn9oUklDijJk53fCl\\\\\\\/Upjz\\\\\\\/BPPJIzn1BZH6U9q5gp0Alj6jxfAVHoFtVhEz1hw3X3bp3BTmhd6zrtBnwoz3lAAs7ICOc0I22jP739ElkAPRgHI\\\\\\\/ZCuwBkIxL1AEyOm7AUQ947g9QcoHgTtfH\\\\\\\/a7CZMP7k+aWstlJEQirrKJjsQC0Xv71FukGLy\\\\\\\/GIeZAtPGy800K83fsMef4PW\\\\\\\/frbd3yN4LgOdaw0vVfdArt7TG5TzR6LvqBr3mlA0RAAESHyElFC1DulVOShrbDSWQf\\\\\\\/EOrF2mhrB4Otwgvxtra2FY+ah11G7DnmggD\\\\\\\/TNr+C80oUJO5QFyRW15MQZqTq+8Hozg6G5Ej5v2pW\\\\\\\/98AFlKnO+K2VWBq14nZEcfjU5f\\\\\\\/vaA70CFLrkT9JdeXlL4IX1CyksM207M2NoWvVjvJmw2CM2TfgumVtuYFTVLRBiEpGtr5uZU3Y6aqxGMljLMVE76B+N989nsctoF3jj\\\\\\\/hkFsuuQAj5Ob1QOMpbnrtSFnuWR+8RPS2znXdQC96gopdEulS5LT3IWwqxOsysNeatq6AlfGDKstlGBFaLDLw11kO9muhrdpfMJnArDeyDCwqs0Pf91Z6cYVS2rgF\\\\\\\/nBpm1vcl9R7SWrw8KKG0xEAHZmpXgNUtB\\\\\\\/Jfe\\\\\\\/HBxzcbIYRGdqHjjz1KHxOrCzHmQuO9uDxgkVbd0oLw336v9eSac7i4fYbwIVCWmcaaFRU9ZGOSWccmLfHR3\\\\\\\/pvyBIbe0NiZy2fhkVMNl0TLpXG\\\\\\\/\\\\\\\/LlKTUFcCQN+kZaV42Yj9wK+cV8ZRcTdv5VZeAQv60rmYBjEzb6EYTrCyqwsLjMLg76\\\\\\\/AJ3c\\\\\\\/Wjy\\\\\\\/vzs6JMCMHurK89riPrlT6EHrKXfDb4lPGk4wc+Mr82IplnjnK\\\\\\\/dXr7XJykQ+agxdM\\\\\\\/4KJFmkXSaJuln6vKt1t4\\\\\\\/uL0ydpjisLfqvIKUDiZiHBo7NSUmlkNzMS+IyKQEtHtOLSk+pzAlc\\\\\\\/ydWCf5GfYJ7dWSC5+z6+GLE7mIXQMPSYZsgmmZQb7IJhzc610+v\\\\\\\/RVWBYoriS86bELMUcZavVlffm5\\\\\\\/ugTasNyIwsZnqDt5PuUjrSL\\\\\\\/9npzHr\\\\\\\/AA1ttwom98xGoH1IcDcVodA7+UoStGdp5\\\\\\\/Vw5mkq\\\\\\\/n1xik5d6Ru1OZtVL9jNMS7GsQrZzZ2\\\\\\\/bwhg8rAxgwBwTslDaE6Kww2klm7RcPLkvw4PLjlW5fkhWoTFCDzRihsENy3RJkelevFE4rE2MGfmvfWtU8wfKkQoF\\\\\\\/1RxsVKmRvfUVjCoAcQuBHsC2qEF5pqbnXBfBydcrnWZO7Xzl7CPFDtBpx0mPHeT1o9kBmkC0G0qIqsckEcEkZG4JVG5\\\\\\\/QPmBc6SIHgZTP+tt\\\\\\\/PHbm+5Fm\\\\\\\/9argAMxOrw1DEGj\\\\\\\/2sVL3XrXJJcOXn5H8Zq21fMRnb\\\\\\\/9kCHqel1ve06G4AIjHV5jqIYvUGd89rd43rBGiyCoCORsqxOTBO1gys8Lc55jlq3yWMYGAQqGE1BZ1WLc38RjIFxcLpYiNFxhpHE+vWfR2LGf7N+LHH6NWd2yMa\\\\\\\/JLb79IrD9QivejN0PfrZD3ttk2R6YQtuYjpbGFkvL0e3dZOdyAG5XgkUF+b7vBBZuMEYQTeRw1gIbX7CnxTGPEk5j6v+UrvaOT1xrEU+AbaEs74SCOtToxPpMefRuZf1S\\\\\\\/9CeWhNc56z9sNiVeRrcxO7fTfH4QzzBiSAShaD+XUWRlBNoCCmrY1foZ814L8PK2dgwfd72FoRQfe9qRpFhr5RWzB\\\\\\\/pKwXfOUN+afCua+YvBpPx5kg8U1PsyoheXAaLXdY\\\\\\\/iDw3OOvzN8EcD9d24jA6onS\\\\\\\/Wl5Jw4d5jWUiLW4rRC61u5mgtrV55X8r8ZN\\\\\\\/81LE2LLLvltyo0wnTabAAlh9d2rWcB8vyWllBHSMKyatHNuAqBpq5xWu87e3gJz6pUyoiCzeKKamjzdq4njGR2ddhGVvCck0z2VHU8ZGG5qVxSZ9AA6dEtBpCekr+MVr4IN7xAiJ\\\\\\\/p6j+\\\\\\\/S0Bjk0eHPpHwSgqZYv5HBDU76At+F4aw8Z3FCant5fYlXajG4KFcAwSJWQ204WTdxbL1Iq3OHka4HZ7W4GW88CxI\\\\\\\/FoqGpSjCzK7vAL3jb9DRiI7SxO8eART7HHc7R\\\\\\\/aKGCCzcOqO\\\\\\\/\\\\\\\/sgR7mKDLuMpTeZUVZBpIiWHC4IA27D3rll0EgqUBnKwV3B9pHpnzmYFRmCo2z29zw9y27zNJKDFDdQwD2lo8fkNvsVmRVSDk8QHYZfiwr0QcqkJLDcDz8leuqUZx4wfhCbQiV5keGS1nkwpvijv6saFW+ognepgih9KZO3+9wiBQef3e6FikdiF1XmB37YjaEGjGlIrXxnnq7JaI3Pq6+YvvSpFl4s6cysHjDby3foh5Im9ek1TTZtGjQntFtEl0dgOMESx88cLmSTtS87BcpDsBAJ\\\\\\\/fd1JsCXY2vC0gpTab5Bv\\\\\\\/ywrLROlHKqGbd3wv9SQQiGFoqi3JKyEgmC3SyHHF5DL1aZMtGJHK9dYVc5r34UEvbGnyStRK0iiGexoowmEItOLgk35EAiN+kttRaqTWBbHqv1\\\\\\\/b7BpUfif5vhTBu9THkmnHxzavDKtdq0sq47qDVaaBsmh4jpPEYlxy\\\\\\\/zdAmQX43FFgrwHKFqFPZmuyqo1JsIVUVT0WeQiC6\\\\\\\/3tfCGzwXLyqOmAJaJsn4p\\\\\\\/6r4xiD+ngacFKykXOt+u2L\\\\\\\/KZb+kkrszcABmnvNcjbVEll1\\\\\\\/EBcLOivYdLsOQ9Q8VUBCa51mA2BQVE5nNhZk59EmYGqIWkqwkAW6DPr8OFabyTWCcCjKXLBdRslZPNK5Gem5B4PZs\\\\\\\/eTX9uc2YzbU9gGQueUGXQvJu0rfsPcxBSdjmIMb5GZVDhxymjFco+Ndr9FtJLcd+XAjvHZYh5JnziAg7cWzPR3MjYCFdflPDvU1\\\\\\\/8KTy2FG3pfs3cPrL5EjWbry8F81VKQTZdHpLgZmeYmebrbOEgtmcSENcK0TAx3NE+yuaBMibYiXFaCcVsC7ohY15cCutwiJNGW3B6TZCX2PVleSl7xM0lY9xg03+rsnGYONdEFkEFHn78dlZB6I\\\\\\\/DaqiIihtXatQIYFpA4VHPqOCmLPLzOJWddFAZ+fvOc3JK34aidYZ7RlU5FhcOVfFLhyq43N+bCCxr4YaBUeyjYPtdeq6KFD\\\\\\\/PtahOxQ3OzC24xRD+rli5Y458RdOo7kgjT2yrwrlH5HN1uIivDI3zueGxeEoXCY3xXjXmdHC1FTzzMRnlzeT5RBmDDyHQNiJSWnSlv8\\\\\\\/DruEdkHG3bQjHx8m50FkI\\\\\\\/8FgOWLMeNSIdE2Pcy9LCiHslcKMDHMzuQBtsGhMViAaGS60AeD+vC7uViP8NQe0n8Etp7bH3LpdlVzZwd5HnXuYgPiMVxhaePfq\\\\\\\/mLfXAW\\\\\\\/hCLf7PBD5qMi01xakXlk5XFpo+RM6xne9s0SjwHqHggXAa\\\\\\\/lk4wQ0plUhpN3nNxVoie3Zqw65MyI4PhFlftbL1Upt0MpWRGhTLAIMxzFm2jDBsk3WM20QttPSeJgwGjRdcFukUdX3gw9QdnmXgtKvPMALKsGp9NA9Mxcv3fBB\\\\\\\/1NTyjNXlbTSBJnCsAl\\\\\\\/08idbEOJrBBpL424gbJ252InDc5KWA10aT6hy4sMwbk+154\\\\\\\/HNSOCzZ4IwQ8AiD0g6g7HMPb3aceAFaYoaJTE284MGGAtTuE7mgINm1CakJnCw1a6wGdgpWSX8+0NCM1vDZy2HknIQFLt\\\\\\\/u2Jozntttn4GRG34wFE\\\\\\\/I2cO3OY9duYjjk4kBITUGok0nvS4go0CPNLIL34UnsIBlsJcivk538DKXZNS6UmVI58rDyb8Hn5oIcY0rDpze7mg5\\\\\\\/7VygsaBymySc\\\\\\\/9OJcu+g0dWunYsTWkF+w58GzNIukSX35+X+xbBNKzCylTjDlUzfGWUS0zsv4fB5CQelHDUMvk\\\\\\\/XiKfy1Xm26GlQf5cPs9HCnVXKhMztWm08EVz2CFeDqDNKlt2wY\\\\\\\/KoDYYU6Cs5P+wrA0T3TMBB03wM6z6D\\\\\\\/IAC6zIxBOAjQ53nExSMjnh+PN3B\\\\\\\/VCQ8pSHQg3RUGa26Ldu24bIoXHNM0lcb28rWZBiDGqXN57i7n4t6p2jRuNfsqGtFSb36+ajRPOIrt5bo6zOx5cS35h6sfbRKkTsGEH3uuH0Yx6MBzBEb1tChBjOE5ajP\\\\\\\/s\\\\\\\/aP6FKwzvdtZkb8JhdyL5S2R+PQNSs1DVUsXarbB554qy+weDeztWy0XMCundjZ\\\\\\\/dA8XXYWYVCvYA9zFFcHXU9Qk34+Mf9C1V3X3N30rxbveFWrX2V6h6Cxh0xKvqjbKVzTtfogu3aU9OA4GcwdMbgWlgzRC4Nt3waKIcO7ACKVLf27+dnosxGvjh0SjyKMQ2RfYnT4LSNI6qnj1D97IR9GyuH7UF7R6SM3d3GmTRV\\\\\\\/N\\\\\\\/q\\\\\\\/PoAMD6YXwSvQVyUpkCVCD2g8MRU6Zgz8aHzNxRHc3EXZTtOKA1lCTfRSQi5LcohGJb+UswjhFQkJShNZWkmBvWw4kYsFQzIPcghjsf4BqgJ5v4dwRKtqK8IphxccgkVVqU4knjY3bUwGzErRciIrl4SMSjENykKWV+ogMrdciFKArXpVXoK8mI\\\\\\\/RK8R9ysIJT+7YZtdrK7Quo+K8c+HLtvQhEBWL0q7xGtN4dXb\\\\\\\/18jvjD4rvCXezgMV1uRZShoFg2kSjSL3PH9PZrShfOGODWRbgVNmhhkecqQPHHo2UPQRKgfm7Gnw9wSFCwoShKxt1U1r0qCJdv6LzhycA5e\\\\\\\/6P4W1LTc7QV8CJUFwbbhNbFThd+erh7zFZNc+00f6lj9RtgHyATwHGLVT9Si88294Zce\\\\\\\/vbypV9S1z5O9JhKL\\\\\\\/qBSQGNaQozX6JcwjwJtphpcmB7xecnyL8GZnMJU2f8mXTRusQ6iAq8SaGsEf4Sh49yG\\\\\\\/1Da3Ggb3IrMwQuCbRPTeNFYqztLMH9Kp2OkzMe1FaOHkvCEr\\\\\\\/E05NeYfkSSRQpXK\\\\\\\/vpTxH41Yo2MAfKXApwmnlTGA16qYzOBB8Da+c9d0r9OxRBuGG8NUzpHik7ZYV3zg9rTTsIjgCMxs0FAT8aKVvY0SS3Pu0uAoKZQ7yJyL+vUE0sD9uTU2JynUPtJrH6JRNTx93MLNxmxH6yXuvd4OCCvHS1xfiq7Y6AYBhm2r\\\\\\\/Nmo1EiDJrm\\\\\\\/alsPoJZ\\\\\\\/t1zg6MzQ1TKu+N3C2KQabkTnndZ64ntBIXMslibrig5Mp9YnWY50cLlxb7nVdBhMqYlwDuK\\\\\\\/V77\\\\\\\/pjEtJJNlH+G66x\\\\\\\/sG3vWnJ+Pb2bY0nCkuLM0vnQQUExviyJ1Xv0CGPGmWo81pS6n99lPNcmnGqWQwbZDGGk329iZ1ZWepj1Nbc8uSw1kO98IiZ6w+ROpU3NVXR3pK4UtBEgi215ev+lpluAIDW3ZAxd2hgFYd\\\\\\\/68+Em4JuxWIP+D9nJFaeGJcTVuqy1oOIHfik0i4ciMTj2cYZLI8Jn4Y6xZ7VimbAi\\\\\\\/ReySk5rwziY1KIfSRvUgy2sCqbuBINk9eLWd4KwDlefH45F5Ux3TTtfvD2VaaUrVmxBfn6jMQIMHxspH1eE9ChUAteLve67lmcZ4aZ7CKOtiCYDrUn0GvKIRmjzZDg86nwlufaOAL9XiIZ0oK6lE45+jOW8S+NcPl+RBmbdC6+HJt3P2G52uX9rWxDAs17Ea9eXYrx6WXGIaO4pP08UicMiYgq6luFbTE7fC+\\\\\\\/8+37cUmWtWk86yGZwn0NrGOGdkh\\\\\\\/C1P9Lu7qpY5lQQjivRqLUPPdW0uZy1NEjEogvpzzMh2WhorCV3QWN8o6Gqz4TXyNunSGBZxKDHxiVzuLRSgWOCB2Y+8AnMf8WdzL2jJUzJMRWZEbIjTr\\\\\\\/1VYQa15KOddB7YQm0TLS1\\\\\\\/+npY3z7dcNlo8HWZcqul6mY9SvTgHntNGGkQ1MJfqDnsWkRUlbuz3UE4+iFdCptT2N9MLT2MtcGN66Pv8mgPRc1vUOTZlv8CS39eZyhBu+GJfMpIb2UwCPApi\\\\\\\/CI4XyAS+0xiKDulBiPunOCaI+XnCORJxlz6hy0gDyYNR+SpHDWg+\\\\\\\/JvVdDz73nA1S7bM9cL0rcw\\\\\\\/9mHyKFqDh6buqZYjMQ\\\\\\\/vIE0t\\\\\\\/iV4L1Gq9LOSLFecR6GADvgk\\\\\\\/H1VW2KXBEzF7KBaHL+AOGBk9jGrZZo8cNLdkqNo0gdad96zE1GxhlsVJ4tbtk+GFP+ov\\\\\\\/P4iqXMFRJRpLzQZxIhKHy3FHOkEI71oQYvBtd4K6iwSp3v2byVr3xSpCwnkRRIQRgPO3sGC2H3s2BukIV5MsyJbqJP7C8oddPO0E7ARUejnJH6\\\\\\\/9vfF3ph3VGHGJWKa5eLRsmJm5pTkITQE2UI18mXfydx1H77SZzgayWbYkpdUsTY7KOufG6Ltm80qEP5iYbQYqjx2ekPoSyMZJoB8q3xflK0v1TyNiZUHZ0xrNXyRqAL6C2nS6oS4gJRKmuUsss6vrX9\\\\\\\/v60wwiI9PMepzMo\\\\\\\/d4pxho9G4qFGrSm3CsBZo7gCAjnYWsRsl3GwOSMKKiliDPzBAEZRJl5KGnwy7Wc1WkbUNAOC0lgbItOM6yDUuPjkSfIexPT1AiZdgyrDh0K08h0SEiRl4AeeR0bUKJs1l6KcJ6g+UcHYc04RUcWRga1p+N+UcXFzkdkMxFDVb+PqauCrOg8kn\\\\\\\/eueb3z2LFWVzAoEgI\\\\\\\/y5NXAePBSITs0IwHk1r0iVFjFexKMuVpk4CC8cyO6yQU5pV\\\\\\\/Ds1SP2XyLLNimHHPTzk1qZ9im0bBtNjVV4O85v0TTJ25\\\\\\\/enVpYMqjUFskLO2zUr6oKeRYzkUKgJPGvhY3voqqY1Yxl\\\\\\\/EI9094Iryo7o4qCfqzKXhpQZGIsMG+hGBWkuzl8W7MdpCwQwbdDBlNWOrGTwLALSRpp4zzXTxehOWC1fD23Y1ZcUi2UHe502G6omWs+rQv9NxRm0VNJNQgxxjOm7kK7RUI4mH3xFkmEa1nQyCXRNCpybNwjv7kRrE3xnd78rVHZIoSv0CMX0csqvUcUeGcCDFfip92DJpOkZj60aBdeCW0b\\\\\\\/gS5oR7wviQcxXxkyM5f\\\\\\\/Eo8JSmLQVzksTlXf0NnHa9kp4CwF1ECi3CpgdjAfeJ6ebFzF64tqjiYGFvUEM0WbOUk+X4w1s5nfkwfNLjcOhC3fBoeOFVe8kW\\\\\\\/bH6kRTUsETa8fTecdQ6J6m5k3IUvgKuoRiqA7yIIjF9Z8Zis4JhQUOEbigpGoBCPxm0PBEbDWFXXTHyA5C8bnLascFOmruVK7DmUgId1Ewer+rGjsxDHbFt4hY\\\\\\\/USZTicuOBYTLdSPzq+f8csHjxUJh5LIaTs\\\\\\\/3CU0lFYX9t8KkYKJ8hrOnY\\\\\\\/\\\\\\\/FCZr5MzZO8fj8eMGPQAwKahI3\\\\\\\/wtBrg3EGcSFmVKd+E+GM0Ron43p6+yAMc31hWhhpB0aM+DN721oaMsw55G+l6kELcKZ+cRepLDQIs6TE1kYETdlfPRQTHyL6fTJSIKjBrgOe49Fvvn5tkrgoLijOAlvLCK95tF9Alu+pyxvi78hgoRyohRmviBaH6iNSPQGoDvwv2pJ6Hg+\\\\\\\/Ow0pwPifxMYhotK1+KHiFWQGLtcQt\\\\\\\/EOBmlELMCp\\\\\\\/CmUGV6QHD42w3xdw2jI6Kpq0af7hBrGZxldOLT05dsraUj\\\\\\\/cB1\\\\\\\/qBmpIOa9YDkMIaapvRDtPFVNIkEzTvCKBRLEymMlrN5GYsBNSvVlqCGsDt\\\\\\\/JpnwS1ZylRUEGSzx2qEVRNW4Or4nffEGJ+YoiFFDk\\\\\\\/D\\\\\\\/a5coi78Qh56lswfeO4k9KauvaqePXcXoQa\\\\\\\/tZqd\\\\\\\/hi3moWt3GdCrJfovMQXiuREF7rC+FIQ\\\\\\\/kUq1AYjnsVhvZk7USaEeDYNPL6m0me+nxbxrEZsxkZraaisR8sbUxytkL6ii4l0AfktFu1gYexHlbr0wdORdlgBuxaxyq0V+sHXh3QoKu9jawcADCHi1ZDdDJ3DbDMJPi6ij+Zi8xKbc6aOMwbCC20niPzWpeX5gdcOJTQ\\\\\\\/Q3AVZkGbosrqW3c5IKtxUW7hEYSsNQo7doq4cIGJ4jTefuhWDRKTItNoDG7SKEbRSnK6NwVMGGX9S5e4WbrmGEUnsJW7EqHW6yECa1CvFIG+Mus0fYNx3jInx6IGtOTBqW+DfmdI6V0i5YDvhCMl2n+Rcgk8KfSjsHPuwXJI2guQE5GBUDCvXYQ9niOqy3yLNuuS+dAniMjEAKO1\\\\\\\/56Ju64KW4lL8+fF9nlmbxR7S3C4SGOoJuxajLbD8\\\\\\\/xuHGspgVmNJjPQ1CqHSAh4e3cCIcFG2Vd\\\\\\\/zigpAaj99OCdFVkljvsk7pCUbvlav7a45sOJjMxi1n4ljQjPYnDXVFZdSy3Ht2Y9MYSICd8qTvKMGCo67f0EhzrddBHfbDSTDAqFfLg5beDx0AIPUatzXBeggLXEiNuoBKUhP6txvkYyL1XMu2tTOnAzMQR7Bg7J6vMJFxdIW2FewOaZaLVPAsrXzotXQvIen37wqaxuL7jWO4D4iOC0VkxLpE4onV7kLLq9wt+kNk8BhHLcJqcAUTgdQkngaMXsMbnRf3muY44VvJQlIwwAc1vuJqed3\\\\\\\/d+n9xxfksL0zmAJMuAMcydGloIJlsbo5cSclIBWRnDW0SV7yNideXQWaLBIwy2T1i1aLM0uTaGuuFGseRbIaVkJ4tZUP08O1D6\\\\\\\/DlfVmwvoik4Mk4gQpucBHy0OtnNlNk8fMjAChVk3xQY2Dusbsq3acqiPOjAk+IFBIspry5LdssjtF6j6Zzd1lKQQLfeFIXh+jnAGdAQfMmEr5R6qAvc67kxIS8+4j694RXc7GsucBeiIvtzlIrhsHZ5GYK5Al2PIuasSeBM3sUwVgmpsDDwmnPYFUzYQJKxz1\\\\\\\/F00+QwbVlQCD\\\\\\\/VNKW90rqifqHrAD2bRx5757ukhm7Ves9izWudNhcynick7bLM0uEO4qImNQuhYv8ze4P2Rmgu3uPldONe35KkorwTp73LcYZUpzNss2enpklfvsAHvd6PBPJkctagabAWjyOfjO97Q9nabAiXSQuDTih4wJvxiy+LC3LH9nm8pkTrO8ouEbclW1rpajuaPwPAHulU1O9kgU6gvnVe+UR4Rw4OWSndaBO\\\\\\\/gvXNoNYWhkUlk8dEaNz1ClikH6U5rqWy9ivUdVEjY1le6\\\\\\\/XQyn2+Gomz3GJOOfdomnMLht\\\\\\\/HqPhsh4kQy3GC0io9TIwRsPTgWHS4v7iA0kLfFvXnuJhDyVFGJhoC1ff1zOR+Ajcur6A\\\\\\\/BEYJ+jB8EUgUI6RqV6fyNoucCE4jW3l+dDdxflmHjhKHjfQM2DdLNNGJytAYqIW6iCmG\\\\\\\/PrWLtXIfkP+lC0hQylWlBdzKsy8+ZrA9sjffhdmWPRgLu2uwNq0W8mtpoRE2YbESz3bArh\\\\\\\/rRaMP79FOWW5\\\\\\\/rfZFY3ilrNBByNkuJcQp8R\\\\\\\/y8gn7Ze8a928iAIZpFWWRfHEknQTuyqzJqp8D5L\\\\\\\/B\\\\\\\/L1Q71bSb3x6O4KRzVkGYVgth30FA+SB+kvtQWPo2TY2lcEr0HqjxpYcUIEXhHliLNlygppfLtQOcNeBSl3XwbhG21EcTDeiFWNsgStPhmEiy\\\\\\\/moek9P837gFZnFpgVQph6kTtRH+WZUFntlvwXrngZgehCs+oWUJsGveMxw3\\\\\\\/XRNzwlEkyO06w\\\\\\\/1DbGxqYPMyvZS\\\\\\\/MG81oObEJtv+xALvr7ipFrZeLFpNF2RqmNEHzmUGA1YBYQMULZtd3QqEBinOJCWzQ\\\\\\\/W27jH0TLICUatqoY5llTpbrjC2SWKO2dHXMgX1GUwYKHBuBVdqs5imvCZlFzy7voz78xvGTfX8CbEihLd9VFf97jUKOxV9Ioxy13TxsN+n40q+WyThmUbTH616tCYubDWzSp+KxykWBgK7IqzKsOG6jUVE+67l5K7MCTlq1\\\\\\\/IKkLlnVPfpz0uhIyRuKnUIViU\\\\\\\/IVscsxe2rdc+GowUhr41ZZQ8Zrt\\\\\\\/kkDnWmr\\\\\\\/F8sT4VftvSdTEZHWB4mmUBFzjo21WBDVAaZcWRcyCpQj59FfLqb3la+cCRFcs6YfvChMdeN8suIRVxmPZUjS5J\\\\\\\/dRhpVjPkmouLcXRVRsRh+X6zsIUtVlNon3rrja7BoCg8FmeIcxB6YZn\\\\\\\/C4sZIkgVO+2x73G8fYPFoJ33WPdGgitUwf5dzAfwdpDa\\\\\\\/LTKlQMm2hAn49PifQ8GYaat9pKfg83j1GTD31pHtyOTRDUTpDLnJA2ZlFoCVl74JYGy9zbv\\\\\\\/3NGhPtCn62swpQH5C9MhsJZfhfgROk4etD5aFmbqufpmPTh\\\\\\\/xdocg6ZL53cntXIkuiFvXdUncpuInqsP0H2iuWchDVBuETwtGKqMAFZ34\\\\\\\/RWrFmE+c25JvHLUuKKFrI\\\\\\\/vm\\\\\\\/vI9+N\\\\\\\/+0lOF+94ewxBvmdnA2WM+IseA9Nmdy9h1w5GY3tLKGN\\\\\\\/qi\\\\\\\/5V1Hf1oi\\\\\\\/QZ42gIBvilA4xgViwl\\\\\\\/Oz4q6POtbczWyCIktNoBSvZKztPa8Kw\\\\\\\/7bmf3GuE03++t9hN5HoGtJWm4Gq08zkyy2YZaTkO7V\\\\\\\/IowTdaFO4p4SrE5aNtNZARRsGwe31DUZJByTWrMlmKWuaPDIEplBa3eI9ET9f3KOic7Zyk5ZHnvdZmICBL6iIYqPVi3p8Vx5w4EJ8odpq3734fv6RMRtBicyv53BA5n2E\\\\\\\/P1fAwF3QuMJ3f489s+uVOt8wWiFJQDEeP\\\\\\\/GnI8xWQoHeHsdbJTMq9\\\\\\\/dsth+qNLOpBojzcXaYPpMUCcHXhXH12BABt64Ge57MxLQA2KEhIV\\\\\\\/bnPfRrjSdIGUzoq8Y7JGlxUUTEAuPRgCAjXo2Op7q1gGFBt7Hj6wymr0qKR4jKX1\\\\\\\/5ooEhDH+bTtylwnaJaQvoDHBonLw2mRDjqr9+dh95iGdyXDK7pugyT3cwciwMTZB5npi\\\\\\\/bLFfWKb2KvlvlkFkbTMtlIDMa4xRq6elWIuuRG2HO4MCn1jqwkQbaKKbr9Ov0VufBZcKsreC5jb0X1ZCiMygvewchuZJpiksdghDjN+u4XLPeSrccqfPPjJA3UfAHZVKg8KWiki7oMqVPct\\\\\\\/tVxtKOGNIljd8pimz5uL+0dn3MvZL8ZmCKFanuKt9wICCSLZNXwuNyT7PTptc58DYE3wby4jNdnLx33Beinz30FsvbA9pzxsaE8kpjl5OTQOiSrImUcdvBm2eFzCYeyNcl9zeUvGL73o1UDvLJQrqhecxhxQys+Oxccs5R6\\\\\\\/+ADHT6dOBnK+h69JiZWQ6Xp4EC0Xe5PZsLLAXbjcF9gPwl5RlJLR3Yd37HXxRPglM0x92gYkrJ8aPCLxDcShpjZdNJ2ZdJghymkrebkXymjDGdka7DY4CxhXfrZ0c2\\\\\\\/8U5TZssCFp67S4DDv5O1jGey5OiNVvu\\\\\\\/EW2SsdILW\\\\\\\/ilOEOAj3NxgMbm9FDgHdV32KyfkSlQQoqBHTkfJUBWd10IIY\\\\\\\/WSMcRQRMa+OqQCx5w1MjE8s3SiJl9ChhS5PBYCUxwnIyxcRiORGNgSVAaWTR3WGjJ30qTyQbpWRWvx\\\\\\\/AqdqLzxVI0FvMb\\\\\\\/1G+a4YHbTQv9MNhEK1ZO+kfjidk6EJZMGCNTVLpS5w6jIAWaNsg03PWfVlvYIeSCL\\\\\\\/P+uWfTtmuz5jVNk7JI5xRFBRX5BtsgvTp1alX01Lo7apHWBq55WSgSuQQOggjSnstbD9l5\\\\\\\/ZivxVOcKPRdAHvnDJE\\\\\\\/\\\\\\\/9CcDra5SncCJX8XA2Shq+tYoybhKgI5zTR+phxGGIo+0WTs9acY6Zk0NqK8RzAuH72LLZgqwIUBLb6+W9NZiRtSkXmdnoxd+zO9PoKLUO7tPgGjW5wrpoosdnl9RIPZ5KePMLcG1Xj3d5tqAiF+j5StyOdXd09S9Ykpy4o3lJNXpMrdnWvynnsiOWHcO43OL1MOpIbkLO19ScVzau1C7ZYpTL1+h4+zN\\\\\\\/CyKkBvWl80Q7TQiysVPN9jWBdAAtfPM0qwOHlhjxes860fB9wmOApJRnBAcEgwAQd22YP7n+yW9HhEDQ4gBKWeliV+DPTXXGO7INGicrURG7lKsX6YBXYc2CUsCWtOjlv\\\\\\\/7hSTnlGkPJx8wExb+vZUleZ6F1zve4P0qgBq\\\\\\\/qgi\\\\\\\/GVSj123E8UFEhg6kGpEkraCPkoGUCRPfSLKiJX2W07RmPP+5vBuJ5ObZPt2yCgwu+tLs33mswX4zKTbgtM89t0dJnPnS2n6+lcXy\\\\\\\/O7qIfCgidUt8jOjQLPx8sVCYyXmMZt65X0KkTKUJJrnBaFrnV3hEYSCJF8ksbXWdastXC6eEC6MTtzt4H9vv3LeN0f7eRx\\\\\\\/gJF64p11guQpA72XoyMEqoCsZyat1D5WBNIfFLgeJjUkuXPK+kiF4B75i52NC6RYNKHq+MokhHA4NGALwYnEZWFXyUbRWP\\\\\\\/o2awyxCsQzVgcgB\\\\\\\/vIzzMwUkzEPfB\\\\\\\/jsdjt0hGJDnWtEJWv1hMFGpdPCo9\\\\\\\/zLfr\\\\\\\/tK0abFpX9AX8U3C0WCtYATtglsKPxZicAf08g3eIX2uC76xCPDLy8vsCwWzEmkAVpDVeIIp8icuYNP3NEDsEwKmWvguc0BPQtei9sbfhITQP6iEtVtQeDWSxv8WctSSL43pyCqy1xGImklvBL6fe7XTbirXC3Fv7nZVWnnXWHsZzPlMGSlnlenkqgi8GQkHk\\\\\\\/j0+YErPTzCS4KhekiWmD4T\\\\\\\/I0Y6J2Alqe+lh2dwolsgPo9LZmjFjQCsubk+myTrjisFrjxq8L8R\\\\\\\/VqLgQGvFdACqjyZlPqeikMbBQCfKQJSA+Ayn\\\\\\\/hAY5QW++iBeB4wfk6ba0GXZOgNdhqMWXNACuxmdoKyQlGL5i2uJCAxqUqxLzDAEcmVUMwwudZqW8s9FdJJRgMaGrs9p1aUpWZvvwQqYeus1yG0LXiQMf\\\\\\\/8E5SVJ51LSokGQ\\\\\\\/MQHvjxcISvg7HglMSvd+wxuknNAqQ10lBWgvRPAFfLQ99Wwc2aupXx5ujgg8AHwAtgjZWRr5SbTse7SFyo7onXKJl2PZaLC7qTscMfLZk9QwEoe3Cqe3fVitalahCavLZ7ZpyaV2dTqx34fxOfQSsaEAcvsHPN62JWpURlD8i79Xaw+uGw7gkLcRhk2gXxkHwIMRn78NAd2RgoU+k6nbG4QyLDnjGu2r2ps1t07XmXk3G2dMCFIZHhguPF3pGnCBr8x8gQc9l4hwUIzQLW2w+wb2lUeviMKghSr9MUNtaMMWsLYti8LGgyF8xjY6eIHx9rDg5epyQ7HuizHq5xquQU+vDzx9Q\\\\\\\/rhLFuQl1k\\\\\\\/Suj9rYcuse2JnjYgo\\\\\\\/FnSP2dABu9feyKBY\\\\\\\/RyebIcNsZifQXq9mWyWvBOvfohH+sBvq9xfsbLMyJwThS1J77\\\\\\\/TudfI3r3I0TivTRVYQp4fErJBbXY8PClTT6F3\\\\\\\/lPlZP1i6R1hZgoNNOX\\\\\\\/NsIr0PrXfNXiXthWZa\\\\\\\/vC5quiX+7eXd+PDzgeIwWvA0bVL9Pmapdntek11K3ZPI0+01zswsD\\\\\\\/qKvH9bRFMfIux7106pzNy2bsH78fOBkWSshrznx6Dcyq4QzsOljTjGjCrkcERYW4Hrj6EgpKAVGM43tD9v\\\\\\\/J0yddA\\\\\\\/N+oeoVlqq9QosgUPR+TT\\\\\\\/+79e9TSwCYkJZaygrbqIPvfqXdtaxcRFXJ8jvl+WCkmHB\\\\\\\/SQPyUWkUP+nbhfFbJJPunSYGQuBGBlYe2J2MrkGWhQqe3zo1d9nQm2EkFoGiqP7iWw9AX9I7qJ2fu52kneeHk+P+gmlsgv240z6iiy6sN6ME5iguYmiEK9maEekmn9XugESyQmxd0UrW55EbK+g40ITTkhp6n7g\\\\\\\/2z84S9tXdpKUdyNm21gfU9zXYNzgtLoklYP8YD++nsCMFvyZcAzi5tv5vo6rGI3G4g9BD5LNEmeYy+dqMV75FVzIKsxgMgKWa\\\\\\\/TpthHcgY1u+4xgh8iSrNkf8Cs7Iltcvedh7A9x3TZ3i3YWEPjiApqF2ADWplWGKKamLBDTu\\\\\\\/+SwbDTWgIsQ\\\\\\\/aj8weqmg7O01Tqe0+QaB53e8BrCiBFUuGSrWFRh3C7WpHqGWQ1ZQXtnS3e\\\\\\\/Eq6jEqwDYemvBxQLnO9x1e3VFxVBLy5NdaNqApV2JEEaIbwgCo\\\\\\\/ofpQMVi01kYHBk+7Yl7AJ8Mb2koveOSyMIl+IkKhItGrBecVC6u2lROubOfj5B\\\\\\\/uh5ZnMr47WMZWbnqeVglrSTZxudt6\\\\\\\/AUDE+QFkBYOXPEPmgwwNK6yIlMT0kstJcPjUXgwz0MLwXq7HYqm\\\\\\\/giKZeBRIGkOhTHchfhcqoBo4odOUZVkuDc9Sd1u6mkewttKiuwU7ZeW8tI73ZsCs6S+dBozlLdP+\\\\\\\/713LxNBAXTf1RRPGFAhCnAIBYvD7smlmEl8vW\\\\\\\/eO0d9XggK0\\\\\\\/Bme2YHnHjRK16HzOXgQWyRP5QHyjyvuIN9ldqfeKZSihhgGynjllPuwGuu\\\\\\\/Y3BNtqu5Dpzi3JYqPZWGNnR0tC7sUaTwWVXJ3Fw27sYkzqfUiksym\\\\\\\/0isioDgm4yahm3Bz1Ugj+++3ge3MeLifAdmTmvKMYItdwYXob95b9uxDn+dFqjtBT4uqPlfQq8z72czYjjtAf21Py60q6zIjUdYSpKXJd40faOviFnprG8gqJ9fB6nJZn\\\\\\\/EDEoIvdoHMGlAGjlYg5Nn0HPraMhOcaBMQxAVRSZ9c3qS+uGwcXXMoWKpcyQeykCimYloI0xxZabN3qLNtYKRZovf1cn5BI\\\\\\\/W4W2yfTx3CDLOtNpWli6pYuGPdYThO1ERpm4eILJhfOYm\\\\\\\/jZPaIoJse32FQnA5HuNYC8NZaAjbCslnpF4RK\\\\\\\/PX3wux6LP7IA0x6DaASEpDRCikTIhhleiQWw67bFvz8kpyX6ATJK2wZCRoxZ1cMk8XJJN2xiWmSnAdq2pkq7Aes8\\\\\\\/LVswso1orr9GVtxsBwm4rNjJp4\\\\\\\/s7DF19jmdvzmHizpu1f6KLNMeTUPcuxFdqBDZfNTOPPUXoaW2mvsGd4lYBf\\\\\\\/INm1asD13oPoyPGxGXWzhve3w45nZfR0tIQs63mokbJTU3PjRWufgOwuCeC8sLhOmal\\\\\\\/yyafEF8f6qq79n983ldPf1og1n81d+gCj3BTR4ZEwM7659yriknlEVQPp3olEOJX5upLhLvLg\\\\\\\/dFe\\\\\\\/FCbffcr2UdeI3AHOS+v87hgcYttz1dxL5Ps0LLQexfYMMp0HAPHPcykGNgsMvYz3N1ckTsLC3bFOXkMPRL44zOVPC8O+wnPkJKScLQAQ0xbhvBnifnPthwM84Ub2IpdpYGnBLZScm0r6Prz9oXtV9vdxPpxjwR\\\\\\\/0EYw+k4ImvGJAsXk\\\\\\\/M197Of5kvvRXQZY+PycjPkjRQtQVJ3tW6GC0BVIvoy+l\\\\\\\/tRp6iZ8pLCqbH+Y7YS0+Ub3IKZjJhrOkVSnA5NxSyDpKhpDWKWeOVzyIWVpc9+AYeTUlCUHXR8EzpaJd4RHR9K9R9YJufoNwcozEEDbde6aq4KjuK46mPG6bR0NwMfXwYRaZalAzLKIq\\\\\\\/lHCZE\\\\\\\/GZKxOA6OyXXF+vV5tNDkGuDzEotY1iKgccyVy1vxQmh6QSVSNUpXQ13EUa8lTEuJdbr+ZbWK14KekT9PusuosymWyx\\\\\\\/2b7Osa0aLNDAFCqvik7zC6CFCcmHgZ2HT6SG\\\\\\\/BLaPN7O649VbnuT6fu74eBaxSjoQllt\\\\\\\/DinfXcitspxp+BwVWrRdseMBgA1xXH\\\\\\\/PDbY1oHTK+imv2HBV\\\\\\\/zdK2zss1sIJ8J9lhwzZSL4W5cDaEYeP8zUhzpLgQ7axu4zi4vFGLXdjCrNOfzr2mFaFEM+nJdeVJanqHPgtAsyPVTnm26u66vfBC0qWlNeEenAlUqPHnONvOkYski8JCwAsNeC4D9C6l8XZSDoVsTPV8TpDTeXXJYle\\\\\\\/HgB6y6zG\\\\\\\/5kRIo+pTN8HwQGFzISFpn6HJ3qcIng1j9z7sAuBcYS\\\\\\\/APwyIeNeRt0M5fzmh0kQPA8xLwAsP3Kv6gu9Fc9h6uaSmFBbxAf9z7L1NkbNuAaZ5reIVbdmQD7LmXl5\\\\\\\/27QEUHK1dJj\\\\\\\/kD5j7Hah08CvvoATW\\\\\\\/WGqIZVrsBsk1IE\\\\\\\/B5X+WDgyJK7gR1fGbblJR4G4NovDzzp0lcHEqcdD+CXo61kQkVN56gHZmvKixzTaf4rALWh2Pnx9bcM9VrpDiMyPHqXFgB2ZjZ5AVg+ep0FMWRKUtJMwrirGRp4gSNu80AJq2bhzh6LuD1Iryr2q+wa+1p17BhlGo6dCtQnstiblLUjMoWzwoihla+zFahuNitx2iZLBgq5HnuF0gkf8itIHghf74h0VYrCaUhYfW7q7YTmcgQ+HhaYZ4iJGI900CSUhcH9batikSUEh8Exqi3v\\\\\\\/bpFQt8GARZ9bbsbeumOknFOwBc5\\\\\\\/Ng9Zy7YoZsnLstaiNjvwDH5HJi\\\\\\\/WH5jax5O6Y1q89vWqz0GmnYEk4ZHz78ghboCX4kzOuTy8vIwHCiwt3L\\\\\\\/X83ev6WaGbUw8I6y23sIvlemZG4GmM2LNIj6WuGhDJVyroKMJ2beaqH9Jv0bDikU1OSoZFOZRMHpS2Fec+VJkBNHsA+7Fn7xFRnnwOzGtuuh+q9xDFfdgk6iANChP9L36ay7SlPHliBi+0aAFMR0E1vC0dxIZid7ou5+coiFaskzZXauNeWjpkWluYoaC9pcJ9SPSMfNaUpiTHcCZGfO9cC+Uz3XcKm9Phu+3XvqOviWLLAwkMIYH\\\\\\\/jUtsuIW7ci0jMvKMIaK6kzMw2NyJ5fpvfBPHLsvz47k8OcAhaYxWueYs5\\\\\\\/FVfaLDOpYsFGcl42LCuo4gH\\\\\\\/QtswtJEDR5MWem7xIrJhxv3i7H4w2PMAeLy1rRzEgqiqAksFtFbhG7uFdI6VkE3GcDRK+LPAD5iJ8t98S7xeQD+8dS9u9urGA6rz+yZQH23g3she+hH3rr8tO69QDBLFVFPG2o4PP769KmQ\\\\\\\/z9j0D0D6ozmjLMEFZWDM1zUz6YBqtybfOzvhikTTkR4fAxVapOWuu1UnTOyLbApMZ9NpOa4rodAz6nQdTh+bQdw+ZqeHocH88WdFaDo1pqvq1H15mH2Z4Nob0KWBNQECdWpBvtT8BEnE7EYpcxiX8RW\\\\\\\/RBY0VorVdlezFqqsQdib\\\\\\\/brOjVyCdBV8cCDYp0EBc329CS+GEfu3pwzjFZO5br1VA+dVz5uPOSlo5OZIv4d\\\\\\\/lLGmm9NUcGWYa7kN\\\\\\\/qdl8kwFPruAkbCvTwyhFtilpQ7OoivY71CKrkbg7f\\\\\\\/OrLWAJIYtm4CZtP55LnMkCW5K\\\\\\\/Gh9ZVr0H8KVbbslAunZYtMvM4FHRrjELzPhGqKKMY\\\\\\\/EbCTX9nZMivQZXFA3t\\\\\\\/pljHa2fdM5oVu9G0jLd4yFosZIHpr5bCDmtSM5rmmg6tYO68iozRCV9+E0FizSpWQ9FCI2kE3LS0LwlZTGsm5AQrC7QTsTlor7BEWoWtNlu8cFvxTaPwYSmrxQqVGziEl0wO3yRkJiQLf\\\\\\\/HvE30TPGLzWbK3Imk02bR2AgHDlJD6FkbzzWi+B07+bI+i6fUgayUrRMn3wqkv8RjVu7ZchAZ9GntJIK2uRbdKfESMF7SYAjMFeNW1sPlfkahOOu6wXehdQHo8K+icSd\\\\\\\/omSbH5heSX97+3zCYlMGHwfrKQMTaG2nKOiRlAknDHd6sjpXVrcsfYx9DX4kBdY5MYG4vcv++F5PhSM2piQrw4fEcTxAQKf6G1q5ZT2+8cHsbx+UqMs5dnrMwDTheD0Hq\\\\\\\/gfX3d5itRLatwtD3pdJ\\\\\\\/IpcIOvF6BYrqH2ZkZLxUtDF2mbyJ1Zi2FYZtibcvhVMyDN9mTmL6fxh7UShfpuWFMfhk3ZcwR6XDQQjRDzXjKHJrtbT60b\\\\\\\/GweJq+vXxLGao37vbrc+T2saBA8TZ7TolwQgXwVCxCRI3L9mI+q2sLEAKQA4eRfxQJJ3\\\\\\\/cTAm9SgqcG2rUlnRaXRL3\\\\\\\/x45j\\\\\\\/qIPKCGhSAirTH\\\\\\\/OSm\\\\\\\/PZIvE7Lbo6TVsBpHvARfpZ\\\\\\\/rmd+\\\\\\\/R0PSVLBOm3s8T8NxYUHnwVH8NhvUEX42O33UdX\\\\\\\/ULdolwlVDcOxQm53CI0LeflLuxv5+GLnKoMtk496RjSZR5\\\\\\\/j4DM6HUKXPJeG16rnL4ND8R4a6UzR1AkTOOnzDXuXfmrMxj+yFZi4jSsLOilkNO0aL3RDsdj9CVNlrqNk0VUcCSyBSLeHWkNLpV5Gn1yGSsu\\\\\\\/ch6Ej7+P0sPdIlTMTNeQl763EwJOd3IysO7v5lBc5TzYr2pEH7o9S4Fl9KE8nQjRHQMSurKzdRjuiimCnevCvGz02806jnHXDzvXoCWZ1lQt7P3jKXnjLNqgzXkdtTd6LOYoqQ2NU9s4RkNO0sLm7sW5ZMdzBuh\\\\\\\/47vArRYuqpp4\\\\\\\/09vIMXc9h1jaldtYWGXoxdQQfxnDm3gNTKJM0AaPtU8CfSNRvcIP3clAxF5jOkHnTOOMFqP0bclq\\\\\\\/F7gX60FtwvPxcs\\\\\\\/L4js4lxErtrD7uwj2sZj\\\\\\\/jiS4BKCE4ycasuZOEmajHhGRm38fy50xypd1OjpzNNnK1iiH3JXjdVKaaX8rKlcNy3rk0\\\\\\\/O+aTPmS3RaJ89c4XQMj5LcB7RjbzRbPM3gch4vvuBWwhcLylUywdBOLQLyrX12cm6Lw+n1Zi4fmMqpBWld7mgazfFX9BA2SyDR+1G5uGi6N4TBP0mTIRomwae9hsyCZ2d45QlTP8sgtsjDligitjlg2UP\\\\\\\/aqn4kV4KsyN40OsiOU0LeUm09lLWofhnlHxx4ZTXiZ7x64+JjfHkOkVIj6aEUrnUOqT2pYYssfRsRykb5BtQD2ohNe1ejZiUcFWC9edFJ70Hmb7iDC68sB7T334eZ51vMBCnrwO8RGb3wCK7O0Z7DC2cGLMKX32ZWl94uiCU5s8XL7VZBf\\\\\\\/5+8ETkiHUCk5KKAcf6scJG6EYpH5Lx2ztMl43EC17AQ6rlgZ+HdWdXoKnCQ88jPTvGxENwQAIklOvkrHmQ3AdVHmpTpNamOJt2aiERpBlXKuncDHjdC\\\\\\\/bYj30TDL5hYeRYIfW9mIEpLVgw2PvsofNuvn33tt0W4Jxia694Q896yPzwGFjK1uGNh3xBb3Wy3fqxc3\\\\\\\/CHn4SAPOh9W9dYk\\\\\\\/Bw0MthNHNglKh5a39Z3Eefxw9kxVjdgNbQFvft8NBBluUrd2UY\\\\\\\/iwWO7R\\\\\\\/kaDw9\\\\\\\/nogNPuO73wgKkkovjuf+xUy5eT30unfjImLUQtwmesKVfgl1cGS13ik\\\\\\\/EhnXbvNxrXA4aaLVwdMEatE89Xci6SAKwobkrusjT6EG41qY+JbEtO5qD8bsvW2uF0VKV1CUaDBs4oIPfy5VXxV22\\\\\\\/qYP\\\\\\\/JmdJk3LTJpxYZWZi9lE84qmbzLpzclP5cvfaUAEaD9x8FfbCkU+7DnqTbgexXa6uaLfb6VmStBe+raOMtH13kC5h6fLjcJjwLEI+VuPC7Y+MgNl4o+pcyZ7wpHFxLv6Qd03VTphfFuvGPK0PHT3xTNsRSVgzph8e+6ctOQZzepYMeLI\\\\\\\/Ft3ncCC4gnQMXtqDqc481H0gGyHGcLhITlSRltI2TtcOEdMZUsTnNBHyySiR1WnMdlluOZmw1caSmfySFicfomerZ0\\\\\\\/X4luYBaCwJda4WLspUWG\\\\\\\/WT2x+px2XHKJm\\\\\\\/uoaia1Q\\\\\\\/0oFjZOaKs\\\\\\\/sX64W9Qj\\\\\\\/8YX0C858OOnXMK8ydamjqhmSOavjSKU4VANLa\\\\\\\/kl68LIhnd0aJkLDN04hPRe\\\\\\\/zoxKoTBVhfpZPpd++KvYSMh9t3ILrtYLI9g5faTax2LsYMOzlCjvFWBbvJ7CB6o3QwLe4jksPw05uBT0RH60YThrtGg4UWwmI+WcDzjy4Q2SXnfPG6CaCNnJo\\\\\\\/qucq\\\\\\\/wx5Fo56W00LrJOU3maxlQzFyiTh3CqWAKXpbHXfEU4BKOByaXUUd8sWknz\\\\\\\/N5ExB6NtLRYc\\\\\\\/EFlYY88LYkQNSodjipBOb23aicC2t+pcwfR0br354X3DGxjwvRhZ6LRKKNSt39766rP\\\\\\\/bDF4AQCKtmP3iRed9rvv709Z2qrc18K2qd\\\\\\\/S5BNNKlwNttdnR+NSVgZAjc8SaIbOLfSt5\\\\\\\/qJ9t0DGMvMeKwtCiZ71UJxc\\\\\\\/Z6ap0BmIzcoWFdteMl7WLLTP2wjV7lyq0xz2fGckx3TFnn32lkrhYoOWAIJqrLSvhroAvCxN1ZUmr5zg4aXcOWbODKdLBQiHxvKJ\\\\\\\/WfQXRH4Gr4Mk1Ug5CRuXteQucz6bwAkQQgVOwSSN6P837iBZpoJvB3GNsp\\\\\\\/9NP7UOEpH4dszb+a+z0kMrFIzI6hEUjI5eQmPdnKdxiXbwDYEvrS4OddphPE8ZffvI8Fu6EUCEBD2x+v697H\\\\\\\/ns7ZNZ4y1Asl5A0CXUZLHx5MfZHfbSQudj5fV6c3XgXcxIQOV7sIi3T7yfYStW2O3xhzBw0ythmyqtyqakt9Ha6H96j9IHrZCSWluN00J4Pouz1Kg2TfDwA2XW1W4fErwE6eTZviU7V0svhwBSzB5i1QbUKdRzHaPGcCnyfbiYUphGhyT7nA7ZyQ1fBFFfZXtCNQU4+JEoAB+J9aRa\\\\\\\/Cr\\\\\\\/rQSGJ5jY29gZkk8l7tNDg\\\\\\\/UmM9MNjO04dZ4\\\\\\\/AA5rh9MGPjsVCstlRxE9\\\\\\\/p1Lfh3BbRWqLJF0M3nl7H2e1\\\\\\\/+WDSsujxP0OUYDRZI3m4DhqZfkslChj3zOihF9hbRiWiKsGOissCSSA+OYE7z12nv9AxAGydAHUvhwr27Lw4oqY5mfCEdsga8LEleRA096fAAqMbRr+zGHldqG8K+61m2QJc4AAopnCR5WGqu4BDfMNQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zmAiTYFFUUHDf1zh8MkY5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227fc3ff244a2c4266208e306615db44906bb6183b0188113d820cd07f6fe05368\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022b8I3SRblgPYo74j0uJX1NvY9DR9MdvogpgQ8DNrS5aayiKj4Kzmq+Gsuv63V5c9avyQRXnF8eUTK6w\\\\\\\/Jry50vXkw5t73zkUePqkyeokacmRBna8pI1EkeXw9JT4oUfB9L4sXhHjiNMgTYp1FuUT+EkSCba8ZN5yrjRCToKZn2weodOFdu9jnVut+mjSoZKgzs8MkX8w5uJ8BgLdpSmvLqWspH85hy8WExhe4VxCAJ+EBw6pOaLmOTBb+VzDiy2ABxjAZjidBdOZWJgRs9R9diP8uXomuVswQRVeLHoedG1T1\\\\\\\/Jrs\\\\\\\/kXkvrACPSDlE5noCtNIdIDWtIsjtJRLmTE2gbXGAjzEfEXet5iagCinHpsko1S\\\\\\\/DUS2LnWfiqa6qTIJWelk0aGwlWXk4vdALHHUsTexJtYAWfreLAHM9nN9CewxIXAZ8y54ipKLPJ4qRHXP5J9uBKwp\\\\\\\/KFpu6SbQEN4+mZ7ynPE8WfYx9m+yTTCSlQRPpHblvLKVIdBI7bYU464B1z1HpHxImHAKb1PLkFB\\\\\\\/d2IYXOLMuxl8ZqQ\\\\\\\/lydFrsJomDtJujX8o1pMZm+\\\\\\\/U8aKraHs5peO7JbdAfIvo\\\\\\\/G1ux869KyL8ffF0lnD6deKW9kHZootTzwvTSJDIOaETkH2h6881Q53jRgycvYyDRwWa6tj1c1gtJlaReS6jXsrvvv1\\\\\\\/7xEpsra\\\\\\\/6q44d2wDDAEF9yx0RQL9e0jtFD9HfykBI+rlHtZ9aVBPyfHJYLJVPbsN8aFKcc\\\\\\\/38RCzxT0VUJzdZB4bVxzLtBIqFspDOoYZNyOmZawZUMS81kCHYjhDIXuthWgrsFHu\\\\\\\/e660TpQZaNlsgPr3A8wkBtAVNnAGfGc7nBtpF1i6Gy0\\\\\\\/uSusqBWKZ5pg720Ky7BGh6ZV3TSGutvp0D6LkBTcBmPNfgKQaVebIlGC0dBs1BkJumV\\\\\\\/eLxQz6zSkg9aEp0RbuGEZZ3LUIS1LE6jV55JvNzp7HAwS5\\\\\\\/0DiqVxyeVWvCqa0drYSwrnircf3SOvniRNGuUP97Jc7RE3OZtrjLCTbqmITZD7oMRI5uPQWU+uIsKJB6aCV1GuKEWbuzgCw\\\\\\\/DPBUu6RYkBUKQlhEHrSRww6O+R4vJYDGGbHYMe\\\\\\\/PSSqeViAzhfzgkyj3TjyaLLpnDUMXL9XGsUAVUB+C90L1yqrczH7dZbm+yQJ4AIfOTw9M15squcpEX1pOEAiGewbt3Ax8r20kCq+EAe8rTt5dzGelizf9wTtgYB1n9Rrb0CWONXWKm3wCFwjXs0Q5bohZKWB\\\\\\\/iKmZZns4vw1hAQkTfIWlYm\\\\\\\/rQJUugmXPNsGwvvBiH6m9HN7nhP321f5ZrZcySJhME0+GJm9b7yT8OQf6dBrcCe0E00rFPOyofU3akZIyk9OeMMqFButaW024XcxCEMm\\\\\\\/wkn4UGFNPWf3b5CK8eEhVqRCS5DZBkvnG7081Kv7yqcCJAqCSKlifuPGyV\\\\\\\/KHywtJSu1vbNWMhm4xHEZI8ljJTjUZ34QU8l08oDrDfzaEZgLjrCWRcgw\\\\\\\/OBBkBIome+9tPB\\\\\\\/HwfKIb0Kxb\\\\\\\/Bm+3Ge5o\\\\\\\/5pF65fVnnOVFo1xEr5B3Gimg2iT3k94nZLxZs9am39uG03FAfJLEFDrBjGaHY\\\\\\\/z4mP0q9UVtLzvlEmvN7oGEaRtEL4OsL3RAiARX+mbZSiiiMeMkKICIQs8RALzMfXWTKOidGwn+PXZvVwrWFBhJdCM\\\\\\\/oqQ2rL\\\\\\\/R2foLll7L7kycw30SRLVjiKABawjn9v0Cu32FxsiWaqwlrRSyE6ruhjosoyrcLdAPjXmZ71wuqi6mEr4kEXhTdVyif84MageQqQsxbCMaB6pqYmmrzc1gIG14K+IfPD26\\\\\\\/vK2mGtyrrWRm+xlsw4yBTvJ1KD\\\\\\\/4oYigxSvvXi+uiiY2NemqJolCOUPEWck1MuMuZ6CmLFga0dEOTSqv8BCiR+0XJJ8HxLJoYm41ErwAq2LrzCdVPW85vr4\\\\\\\/rQCzyGmQDqZ4ROuSTam4mbt\\\\\\\/koFkoeBu9w\\\\\\\/tzcECS+Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022TdyM4Rm6TwfIAGpIFxGC\\\\\\\/Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227db15b58603ba4f7ad02ad7426e14e4575ba81f42ccea5f29acb0ba2a02fa9b8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492883 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1663" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-U5nNMURvyrCzdrUHaIIYXsXW6huMrsDoGKtXG6hb69E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1663" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e4LcUizz4xQ09HKgs4-eDdXd6Ox7lzZsjrc1nG2pF-M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1655" title="Genetic Properties Responsible for the Transgressive Segregation of Days to Heading in Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1679" title="HSB-1 Inhibition and HSF-1 Overexpression Trigger Overlapping Transcriptional Changes To Promote Longevity in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492883" id="node492883" data-pisa="ggg;9/5/1663" data-pisa-master="ggg;g3.119.400167" data-apath="/ggg/9/5/1663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fei Shen</span>, <span class="highwire-citation-author" data-delta="1">Zhenyu Huang</span>, <span class="highwire-citation-author" data-delta="2">Baoguo Zhang</span>, <span class="highwire-citation-author" data-delta="3">Yi Wang</span>, <span class="highwire-citation-author" data-delta="4">Xi Zhang</span>, <span class="highwire-citation-author" data-delta="5">Ting Wu</span>, <span class="highwire-citation-author" data-delta="6">Xuefeng Xu</span>, <span class="highwire-citation-author" data-delta="7">Xinzhong Zhang</span> and <span class="highwire-citation-author" data-delta="8">Zhenhai Han</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1663-1678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400167 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492883" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fei Shen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BShen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shen%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BShen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhenyu Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenyu%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhenyu%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Baoguo Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Zhongbaolvdu Agricultural Research Centre, Beidaihe, Hebei, 066100, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baoguo%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaoguo%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ting Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATing%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xuefeng Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuefeng%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuefeng%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xinzhong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinzhong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinzhong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangxinzhong999@126.com" class="" data-icon-position="" data-hide-link-title="0">zhangxinzhong999@126.com</a>
<a href="mailto:rschan@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">rschan@cau.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zhenhai Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenhai%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhenhai%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangxinzhong999@126.com" class="" data-icon-position="" data-hide-link-title="0">zhangxinzhong999@126.com</a>
<a href="mailto:rschan@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">rschan@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1663" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492883" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492883/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1663.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492883" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492883/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1663.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492883" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492883/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1663.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492883" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492883/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Apple fruit ring rot (FRR), caused by <em>Botryosphaeria dothidea</em>, is a worldwide disease that impacts Asian apple production regions. However, no substantial progress has thus far been made toward the mapping of candidate genes or the development of effective genetic makers. In this five-year study, the resistance of 1,733 F1 hybrids from the cross ‘Jonathan’ × ‘Golden Delicious’ was phenotyped by non-wounding inoculation with four <em>B. dothidea</em> isolates. We first conducted systematic comparison of different analytic strategies for bulk segregant analysis by re-sequencing (BSA-Seq) and obtained suitable one for outbreeding species such as <em>Malus</em>. Forty-six quantitative trait loci (QTL) for resistance/susceptibility to the four isolates, including one QTL ‘hotspot’ on chromosome 14, were identified via BSA-Seq. Using integrated multi-omics strategies including RNA-sequencing, parental re-sequencing, BSA-Seq and meta-analysis of RNA-sequencing, fifty-seven candidate genes and corresponding functional mutations from the QTL were predicted. Functional mutations located on the candidate genes were validated using kompetitive allele-specific PCR in hybrids and <em>Malus</em> germplasm accessions with extremely resistant/susceptible phenotypes. Ten effective markers for apple ring rot were developed. The results provide an example of rapid candidate gene mapping for complex traits in outbreeding species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-Seq</a></li><li class="kwd"><a href="/keyword/bsa-seq" class="hw-term hw-article-keyword hw-article-keyword-bsa-seq" rel="nofollow">BSA-Seq</a></li><li class="kwd"><a href="/keyword/multi-omics" class="hw-term hw-article-keyword hw-article-keyword-multi-omics" rel="nofollow">multi-omics</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/malus-domestica-borkh" class="hw-term hw-article-keyword hw-article-keyword-malus-domestica-borkh" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Malus domestica</em> Borkh.</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/botryosphaeria-dothidea-moug-ex-fr-ces-de-not-0" class="hw-term hw-article-keyword hw-article-keyword-botryosphaeria-dothidea-moug-ex-fr-ces--de-not" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Botryosphaeria dothidea</em> (Moug. ex Fr.) Ces. &amp; De Not</a></li></ul><p id="p-3">Apple fruit ring rot (FRR), caused by <em>Botryosphaeria dothidea</em>, is a worldwide disease that severely impacts Asian apple production regions (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guo <em>et al.</em> 2009</a>). The phenotype of FRR incidence is qualitatively dominant against non-incidence, and the variation in susceptibility to FRR is attributed to the segregation of three major gene loci (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zhuang <em>et al.</em> 2011</a>). Thirty-four major gene loci and six quantitative trait loci (QTL) for bot canker and FRR resistance/susceptibility have been genetically mapped on the apple genome (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cui <em>et al.</em> 2014</a>). However, these QTL regions are too large for identifying candidate genes or to be effectively used in marker-assisted selection.</p><p id="p-4">Bulk segregant analysis (BSA) is a powerful method for identifying DNA markers that are tightly linked to the causal genes in inbreeding plant species (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Michelmore <em>et al.</em> 1991</a>). Markers for both qualitative and quantitative trait loci can be identified by BSA, and BSA has been extensively used in cereal crops, vegetables, ornamentals, and some tree species (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lu <em>et al.</em> 2014</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pandey <em>et al.</em> 2017</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Ries <em>et al.</em> 2016</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Singh <em>et al.</em> 2016</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Takagi <em>et al.</em> 2013</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wang <em>et al.</em> 2017</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Win <em>et al.</em> 2017</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2013</a>). BSA has been greatly improved in recent years by next-generation sequencing (NGS) technology, defined as BSA-Seq or QTL-Seq, and has been effectively used for QTL mapping in many plant species (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Geng <em>et al.</em> 2016</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Illa-Berenguer <em>et al.</em> 2015</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lu <em>et al.</em> 2014</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Pandey <em>et al.</em> 2017</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Ries <em>et al.</em> 2016</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Singh <em>et al.</em> 2016</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Takagi <em>et al.</em> 2013</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Win <em>et al.</em> 2017</a>).</p><p id="p-5">Integrated multi-omics analysis can significantly enhance the efficient identification of candidate genes from QTL regions. Deep re-sequencing is an effective and reliable technology for detecting genome-wide genetic variations among cultivars (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kunihisa <em>et al.</em> 2016</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Xing <em>et al.</em> 2016</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zhang <em>et al.</em> 2014</a>). The transcriptome can reveal both the differentially expressed genes associated with biological processes and the genetic variations in coding sequences (CDS) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Balan <em>et al.</em> 2018</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bonnet <em>et al.</em> 2017</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hill <em>et al.</em> 2013</a>). Large online transcriptome datasets are now available, and the meta-analysis of RNA-sequencing (RNA-Seq) datasets can provide an important reference for characterizing gene functions (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Balan <em>et al.</em> 2017</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Balan <em>et al.</em> 2018</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Guo <em>et al.</em> 2017</a>). For instance, copy number variation in a gene cluster encoding endopolygalacturonase, which mediates flesh texture and stone adhesion, was identified in peach using re-sequencing and RNA-Seq technology following traditional QTL mapping analysis (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gu <em>et al.</em> 2016</a>). The biosynthesis, regulation, and domestication of bitterness in cucumber were determined using integrated approaches, including genome-wide association studies (GWAS), re-sequencing, and transcriptome analysis (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Shang <em>et al.</em> 2014</a>).</p><p id="p-6">In this study, we integrated whole genome parental re-sequencing, BSA-Seq, and RNA-Seq meta-analysis to identify QTL and screen candidate genes for FRR resistance/susceptibility. Markers in the regions of the candidate genes were validated by kompetitive allele-specific PCR (KASP).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-7"><em>Malus</em> germplasms (60) and biparental cross hybrids (1,773) (‘Jonathan’ × ‘Golden Delicious’) were used as segregating populations. The hybrid cross was performed in 2002 and the seedlings were planted in 2003. All plant materials were subjected to conventional cultivation management practices.</p></div><div id="sec-3" class="subsection"><h3>Phenotyping and sampling</h3><p id="p-8">In the years of 2008, 2009, 2014, 2015, and 2016, phenotyping for FRR resistance was undertaken under room temperature conditions using the ripened fruit and mycelia of four <em>B. dothidea</em> isolates (Zz026, Ls1, Lw023, and Lw048) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Guan <em>et al.</em> 2015</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Zhuang <em>et al.</em> 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Re-sequencing of ‘Golden Delicious’ and ‘Jonathan’ and data processing</h3><p id="p-9">Genomic DNA was extracted from the leaves of ‘Golden Delicious’ and ‘Jonathan’ using a Genomic DNA Isolation Kit (TianGen, Beijing, China). The Illumina sequencing libraries were constructed using NEBNext DNA Library Prep Master Mix. Paired-end sequencing was performed using the Illumina HiSeq2500 sequencer (Illumina, San Diego, CA). Burrows-Wheeler Aligner (BWA) was used to map clean reads to the apple double haploid (DH) genome (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li and Durbin 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Daccord <em>et al.</em> 2017</a>). SNP and InDel calling were conducted using SAMtools (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li 2011</a>). SNPs and InDels with a lowest phred-scaled quality of 30 and a minimum read depth of 10 were kept. Structure variation calling was conducted using the SpeedSeq pipeline (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chiang <em>et al.</em> 2015</a>). InterproScan5.21was used to conduct functional domain analysis of all the genes in the apple genome (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zdobnov and Apweiler 2001</a>). <em>Cis</em>-acting regulatory elements in upstream of the genes were analyzed using PlantCARE web tools (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rombauts <em>et al.</em> 1999</a>). Finally, all the variations were annotated with ANNOVAR based on the functional domain, <em>cis</em>-acting regulatory elements, and gene model information (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wang <em>et al.</em> 2010</a>).</p></div><div id="sec-5" class="subsection"><h3>DNA pooling and BSA data processing</h3><p id="p-10">The resistant/susceptible bulks were constructed using 1,773 hybrids of ‘Jonathan’ × ‘Golden Delicious’ (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Guan <em>et al.</em> 2015</a>; <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Zhuang <em>et al.</em> 2011</a>). Based on the lesion length data after fruit non-wounding inoculation with four isolates for five years, hybrids exhibiting the non-incidence phenotype in at least two years were included in the resistance (R) bulk, while incident hybrids with random lesion lengths in at least two years were chosen as the susceptible (S) bulk. Genomic DNA was extracted from the leaves of the selected hybrids using a Genomic DNA Isolation Kit (TianGen). Eight DNA libraries were constructed after DNA pooling based on the group information. The method of Genomic DNA extraction and the DNA library construction were the same as above. Paired-end sequencing was performed using the Illumina HiSeq X Ten platform (Illumina). The clean reads were mapped to the apple DH genome using BWA, and only uniquely mapped reads with a minimum phred-scaled map quality score of 20 were kept.</p><p id="p-11">The SNPs showing polymorphisms between parents or which were double-heterozygous in the two parents were subdivided into three subsets: J-type, whereby the genotype of the markers is homozygous in ‘Golden Delicious’ and heterozygous in ‘Jonathan’; G-type, whereby the genotype of the markers is heterozygous in ‘Golden Delicious’ and homozygous in ‘Jonathan’; and H-type, whereby the genotype of the markers is heterozygous in both parents.</p><p id="p-12">Read counts with different genotypes from extremely resistant and susceptible pools were extracted and separated based on the G-, J-, and H-type subsets, respectively. Thus, for each isolate, three files (G, J, and H) with read counts of the coupled pools were generated.</p><p id="p-13">The allele frequency difference (AFD) between two coupled pools and G-value in each site were calculated based on the read counts. Nadaraya-Watson kernel regression was used as a smoothing function, and the G’-value at each site within the 1 Mbp sliding window size was calculated (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Magwene <em>et al.</em>, 2011</a>). The significance threshold of G’ was estimated using an empirical approach proposed by <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Magwene <em>et al.</em> (2011)</a>. Significant regions with false discovery rate (FDR) &lt;0.01 were selected as candidate QTL.</p><p id="p-14">The QTL analysis was performed three times using each of the three subsets of data. The QTL could be mapped to the pollen, maternal, or both parents using the marker data subset G, J, or H, respectively. QTL with peak G’ value exceeding 30% of the significant threshold were defined as major QTL.</p></div><div id="sec-6" class="subsection"><h3>Estimating the impact of the reference genome on QTL detection</h3><p id="p-15">To investigate the impact of the reference genome on QTL detection, the clean sequencing reads from the two bulked DNA pools (resistant/susceptible to isolate Zz26) were processed to detect QTL using two versions of the apple reference genome (diploid version 1.0p and DH) (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Velasco <em>et al.</em> 2010</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Daccord <em>et al.</em> 2017</a>). The DNA sequences without gaps (Ns) of the QTL were obtained from the corresponding genome (diploid version 1.0p or DH) using SAMtools software (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Li 2011</a>). The LASTZ sequence alignment program was used to align the sequences against the genome, and dot plots were visualized in the R statistical environment (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Harris 2007</a>). To verify the reliability of the QTL, 3-5 markers were selected from each QTL region, and all individuals in the two bulks were genotyped using the KASP assay. Chi-square tests of the genotypic frequency of a certain marker indicating significant differences (<em>P</em>-value &lt;0.05) between the two bulks were considered reliable.</p></div><div id="sec-7" class="subsection"><h3>Comparison of different statistical methods for QTL identification</h3><p id="p-16">To determine the best statistical method, we compared the SNP index (SI), Euclidean distance (ED), and the G-value (GV) methods for QTL identification using the phenotype data of resistance/susceptibility to the Zz26 isolate. The absolute value of the △SNP index (|△SI|) was used for SI (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Wang <em>et al.</em> 2017</a>). Where large numbers of SNPs exist, fitting the ED using a Loess curve with a polynomial exponent of one in the MMAPPR may be hugely time-consuming and computationally intensive (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hill <em>et al.</em> 2013</a>). Instead, we used Nadaraya-Watson kernel regression as a smoothing function in ED (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Magwene <em>et al.</em> 2011</a>). To get a fair decision of the three statistical methods, we adopted the same threshold selection method. The median of all the scores plus three-times the standard deviation (SD) were used as thresholds for SI, ED, and GV.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1663/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1663/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1663/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1663/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1663/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1663/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1663/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1663/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>In the GV method, the G-value was calculated at each SNP using the standard G–statistic followed by smoothing using equation 1, where the sum includes all SNPs within the window W bracketing the SNP, and equation 2, where <em>D<sub>j</sub></em> is the standardized distance, with value 0 at the focal position and value 1 at the edge of the window. Similarly, in the ED method, the Euclidean distance is first calculated at each SNP location using equation 3, where the letters represent the corresponding allele frequency of the bases in each pool. This distance is then raised to four-times the power, following which the data are fit using Nadaraya-Watson kernel regression, <em>i.e.</em>, equation 4. Also, for the SI method in this study, |△SI| was calculated at each maker and then smoothed using the same smoothing function.</p></div><div id="sec-8" class="subsection"><h3>RNA-Seq library construction, sequencing, and data processing</h3><p id="p-17">The fruit of three hybrids randomly chosen from each of the extremely resistant/susceptible bulks were used for the RNA-Seq analysis (as three independent biological replicates). Three ripened apples from each hybrid were inoculated without wounding with the mycelia of the <em>B. dothidea</em> Zz26 isolate, and healthy flesh from the very edge of the lesion was cut and sampled at 0 h, 24 h, 48 h, and 96 h after inoculation (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Guan <em>et al.</em> 2015</a>). Three apples of each non-inoculated hybrid were also sampled as controls.</p><p id="p-18">Total RNA was extracted using the modified CTAB method (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Moser <em>et al.</em> 2004</a>). For each sample, 5 μg total RNA was used to isolate mRNA for the preparation of an RNA-Seq library using NEBNext Poly(A) mRNA Magnetic Isolation Module and NEBNext Ultra Directional RNA Library Prep Kit for Illumina (New England Biolabs) following the manufacturer’s protocols. The cDNA library was sequenced (paired-end 150) from both the 5′ and 3′ ends on the Illumina HiSeq X Ten platform (Illumina) according to the manufacturer’s instructions. The RNA-Seq reads were mapped to the apple DH genome with HISAT2 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kim <em>et al.</em> 2015</a>). StringTie was used to conduct transcript assembly and quantification (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pertea <em>et al.</em> 2015</a>).</p><p id="p-19">DESeq2 software was used to detect differentially expressed genes (DEGs) between the resistant and susceptible hybrids (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Love <em>et al.</em> 2014</a>). Blast2GO was used for gene ontology (GO) classification and GO enrichment analysis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Conesa <em>et al.</em> 2005</a>). KOBAS2.0 was used for Kyoto Encyclopedia of Genes and Genomes (KEGG) metabolism annotation and KEGG enrichment analysis (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Xie <em>et al.</em> 2011</a>). The .bam files from one individual hybrid were merged, followed by genotyping by SAMtools (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Li 2011</a>). SNPs and InDels were called and clustered between resistant and susceptible hybrids.</p></div><div id="sec-9" class="subsection"><h3>Candidate gene mining From the QTL regions based on multi-omics data</h3><div id="sec-10" class="subsection"><h4>Narrowing down the QTL regions with varied sliding windows sizes:</h4><p id="p-20">By using the BSA-Seq data, all markers located on the gene body or up-stream regions of genes were selected from ± 0.5 Mb regions of the QTL main peak and sub-peaks, if present. The markers were then filtered, and those makers with a G-value above the G’-value were selected for subsequent analysis. Sliding window analysis with 0.125 Mbp, 0.25 Mbp, 0.5 Mbp, 0.75 Mbp, and 1 Mbp window sizes was used. The standard deviation and mean of those peaks were calculated to measure the stability of the peak position. Overlapped peak regions with different window sizes were considered to be the defined region</p></div><div id="sec-11" class="subsection"><h4>Excluding genes from QTL intervals using the parental re-sequencing data:</h4><p id="p-21">Too many genes were present in the narrowed-down QTL intervals. To further reduce the gene number, another criterion for candidate functional single nucleotide variants (SNVs) and structural variations (SVs) screening is that all the variations must be consistent with Mendelian inheritance; <em>i.e.</em>, if the QTL is located only in the pollen parent, then the genotype of the screened variant in the pollen parent must be heterozygous, the genotype in the maternal parent must be homozygous, and vice versa. By using the parental SNV and SV database, many genes within the narrowed-down QTL intervals were culled, except where upstream (2,000 bp) variations between the two parents may affect <em>cis</em>-element functioning, or where exonic variations between parents may cause stop-gain, frameshift, or domain non-synonymous altering.</p></div><div id="sec-12" class="subsection"><h4>Excluding genes from candidates using transcriptome data:</h4><p id="p-22">Gene transcription is often affected by variations in up-stream regions, especially on the <em>cis</em>-elements (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Espley <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kobayashi <em>et al.</em> 2004</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zhang <em>et al.</em> 2017</a>). The exonic variation can be also detected in the corresponding cDNA samples if the gene is transcribed (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Han <em>et al.</em> 2018</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Piskol <em>et al.</em> 2013</a>). By using transcriptomic data, the unassociated genes can be further culled from the above-mentioned candidate genes, including those absent in the unigene dataset and thus un-transcribable with those variations in upstream <em>cis</em>-elements, but absent in the DEG dataset; and those with exonic variations absent in the unigene mRNA SNP/InDel-calling dataset.</p></div><div id="sec-13" class="subsection"><h4>Excluding genes from candidates with functional alterations in the incorrect bulk pool:</h4><p id="p-23">We attempted to assemble haplotype blocks using paired-reads. The Hidden Markov Model-based algorithm integrated in SAMtools was adopted to assemble haplotype blocks from the DNA sequence reads (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Li 2011</a>). The haplotype blocks from the resistant/susceptible bulks were considered as main haplotype blocks representing the haplotype shared by most of the samples in the bulks. BSA markers were used as ‘anchor markers’ to assign the allele frequency to the haplotype. Thus, the haplotype blocks could be determined as enriched in resistant or susceptible pools based on the BSA makers.</p></div></div><div id="sec-14" class="subsection"><h3>Marker validation and gene expression analysis</h3><p id="p-24">To examine the RNA-Seq data, randomly selected genes were validated using real-time quantitative (RT-q)PCR, and the gene-specific primers were designed with primer premier software (version 5.0) (Premier Biosoft Interpairs, Palo Alto, CA) (Table S1). One microgram of total RNA was used in the reverse transcription in a total volume of 20 μL in the presence of 6-mer random primer and oligo primer according to the protocol of the TaKaRa kit (TaKaRa Biotechnology Co., Ltd., Japan). The PCR reactions were run in three replicates in a Bio-Rad Sequence Detection System (Bio-Rad Life Science Research, Hercules, CA, USA). <em>Actin</em> was chosen as an internal control gene for normalization. Quantifying the relative expression of the genes at the three sampling time points was performed using the delta-delta Ct method (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Livak and Schmittgen 2001</a>). All data were expressed as the mean ± SD after normalization.</p></div><div id="sec-15" class="subsection"><h3>Validation of SNPs and InDels</h3><p id="p-25">Primers were designed using Primer-blast software (NCBI, Maryland, USA) based on the 200-bp sequences flanking the SNPs and InDels to be validated. Then, monoclonal PCR products comprising SNP loci of interest were amplified in both parents and sequenced using Sanger sequencing (BGI, Beijing, China) to verify the SNPs obtained from the re-sequencing data between the parents.</p></div><div id="sec-16" class="subsection"><h3>KASP genotyping assay and statistical analysis</h3><p id="p-26">A total of 110 DNA samples, including 20 and 30 individuals in the bulks resistant and susceptible to isolate Zz26, respectively, and also 60 <em>Malus</em> germplasm accessions were genotyped using the KASP assay (LGC Genomics, Beverly, MA, USA). The primers were designed based on the 200-bp sequence flanking the SNPs (Table S2). The DNA of ‘Jonathan’ and ‘Golden Delicious’ was used as control. Then the genotypes were validated by Sanger sequencing. Data were analyzed using the “Endpoint Genotyping” method of the Light Cycler 480 Software (release 1.5.0).</p><p id="p-27">The segregation biases of markers between resistant and susceptible bulks were tested by Chi-square test. Chi-square test was run twice, the first, allele A was supposed to be completely/partially dominant on B, and the next, allele B completely/partially dominant on A. The scenario which Chi-square value was statistically significant was accepted.</p></div><div id="sec-17" class="subsection"><h3>Data Availability</h3><p id="p-28">The RNA-sequencing data and the whole genome re-sequencing data of the pooled DNA from the bulks and un-pooled DNA from the parental cultivars have been deposited in the NCBI Sequence Read Archive (SRA) with the accession number PRJNA392908. Gene expression data and other additional datasets and codes are also available at Figshare and File ‘Readme.txt’ contains detailed description of all the additional datasets. File S1 contains all the detailed descriptions of supplemental materials. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7819688">https://doi.org/10.25387/g3.7819688</a>.</p></div></div><div class="section results" id="sec-18"><h2 class="">Results</h2><div id="sec-19" class="subsection"><h3>Bulk construction and whole genome re-sequencing</h3><p id="p-29">Based on the phenotypes over five years (2008, 2009, 2014, 2015, and 2016), eight bulks were developed, each including 20∼37 hybrids with extremely resistant or susceptible phenotypes to each of the four <em>B. dothidea</em> isolates (Zz26, Ls1, Lw023, and Lw048). A total of 199 hybrids were included in the eight bulks, of which 17 and 21 exhibited extreme resistance or susceptibility to at least two pathogen isolates, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and Table S3).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams of the hybrid numbers (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) included in the bulks with phenotypes of extremely resistant (R) and susceptible (S) to apple fruit ring rot (Botryosphaeria dothidea) isolates Ls1 (red), Lw023 (orange), Lw048 (green) and Zz26 (blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-811376878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Venn diagrams of the hybrid numbers (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) included in the bulks with phenotypes of extremely resistant (R) and susceptible (S) to apple fruit ring rot (&lt;em&gt;Botryosphaeria dothidea&lt;/em&gt;) isolates Ls1 (red), Lw023 (orange), Lw048 (green) and Zz26 (blue).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1663/F1.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1663/F1.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493184" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">Venn diagrams of the hybrid numbers (‘Jonathan’ × ‘Golden Delicious’) included in the bulks with phenotypes of extremely resistant (R) and susceptible (S) to apple fruit ring rot (<em>Botryosphaeria dothidea</em>) isolates Ls1 (red), Lw023 (orange), Lw048 (green) and Zz26 (blue).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The whole genome re-sequencing data of the pooled DNA from the bulks and un-pooled DNA from the parental cultivars, ‘Jonathan’ and ‘Golden Delicious’, have been deposited in the NCBI Sequence Read Archive (SRA) with the accession number PRJNA392908. A total of 163,934,426 and 172,856,412 clean reads were obtained by parental re-sequencing for ‘Golden Delicious’ and ‘Jonathan’, respectively (Table S3). About 329.4 M reads were mapped to the apple DH genome (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Daccord <em>et al.</em> 2017</a>), of which about 207.5 M reads were uniquely mapped (Table S3). SNVs and SVs called and annotated using the parental re-sequencing data are included in Table S4–S7. The overall consistency between the NGS and Sanger sequencing platforms was 99.9% and 85% for SNVs and SVs, respectively (Table S8 and Table S9). Based on the re-sequencing data of the two parents, a total of 1,126,610 J-type SNPs, 500,397 G-type SNPs, and 3,098,152 H-type SNPs were identified (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493125/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493125/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Numbers of different types of markers used in bulk segregant analysis</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">Sequencing of the eight bulks generated a total of 917,367,968 reads. The sequence reads of the bulked samples provided on average 39.6–60 X coverage for the DH apple genome (Table S3). For all the reads (937,097,528) from the eight bulks, 97.89% (917,367,968) were mapped to the reference genome and 72.8% (682,727,645) were uniquely mapped (Table S3).</p><p id="p-34">After filtering, the sum of the remaining J-, G-, and H-type SNPs ranged from 1,693,771 to 3,507,347, and the corresponding SNP density across the genome ranged from 2601/Mb to 5387/Mb in the extreme bulks of the four isolates (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The distribution of SNPs in the genome, shown as the number of SNPs per Mb with a 10 Kbp sliding window, was not even, and almost all regions in the genome excluding the gaps were covered by the sliding window analysis (Figure S1–S4).</p></div><div id="sec-20" class="subsection"><h3>Comparison of statistical methods for QTL identification</h3><p id="p-35">The profiles of the corresponding major QTL (J06.1, J08.1, J10.1, J10.2, J10.3, G11.1, G15.1, H03.1, H14.1, H14.2, and H16.1) identified through SI, ED, and GV coincided or overlapped well, indicating that the effects of the major QTL were relatively robust. For all statistical methods, the effects of noise were effectively reduced and AFD was measured between the two extreme pools (Figure S5, Figure S6 and Figure S7). However, the QTL by SI covered broader regions and were less significant than by ED or GV (Figure S5, Figure S6, Figure S7 and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), implying that the QTL sensibility of SI was comparatively lower. As a consequence, one reliable minor QTL (G02.1-GV) was missed in SI (Figure S7 and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Distinct unbiasedness and low false positives in the QTL were achieved using GV, especially for H-type QTL. The QTL intervals by GV (2,551,554 bp) were much narrower than that by ED (2,618,442 bp) (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Additionally, some false positive QTL were found in ED (Figure S6 and Table S11). Collectively, GV was the best statistical method and was thus explored subsequently.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493130/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493130/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of overlapped QTL for apple (Jonathan × Golden Delicious) resistance/susceptibility to Botryosphaeria dothidea isolate Zz26, detected by three statistic methods</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>The impact of the reference genome on QTL detection</h3><p id="p-37">By using the diploid (version 1.0p) and DH genomes, 14 and 12 significant QTL were identified, respectively (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Table S10). The seven reliable QTL in the DH genome coincided or overlapped precisely with nine reliable QTL from the diploid genome (Table S13). However, five reliable QTL (G02.1, J08.1, H16.1, H14.1, and H14.2) in the DH genome did not match any QTL from the diploid genome (Table S13). Conversely, two reliable QTL (G09.1 and G09.2) identified by using the diploid genome were missing from the results of the DH genome (Table S13). Based on the sequence alignment, gaps were clearly detected in two QTL regions between the two genome versions (Table S13 and Figure S9). One QTL (J05.1) in the diploid genome was aligned to chromosome 10 of the DH genome and overlapped with one QTL (J.10.2) (Table S13, Figure S9, and Figure S10). These data confirmed the reliability of the QTL detection and also suggested that using DH genome should generally produce better results, but a few QTL may be potentially missed due to the loss of segments/regions during DH process and recombination.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G&#x2019; value profile of the QTL for apple (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) resistance/susceptibility to Botryosphaeria dothidea isolate Zz26. Y-axis represents G&#x2019; value. X-axis represents chromosome position. Red lines: &#x2018;Jonathan&#x2019;, blue lines: &#x2018;Golden Delicious&#x2019;, black lines: &#x2018;Golden Delicious&#x2019; &amp; &#x2018;Jonathan.&#x2019;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-811376878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;G&#x2019; value profile of the QTL for apple (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) resistance/susceptibility to &lt;em&gt;Botryosphaeria dothidea&lt;/em&gt; isolate Zz26. Y-axis represents G&#x2019; value. X-axis represents chromosome position. Red lines: &#x2018;Jonathan&#x2019;, blue lines: &#x2018;Golden Delicious&#x2019;, black lines: &#x2018;Golden Delicious&#x2019; &amp; &#x2018;Jonathan.&#x2019;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1663/F2.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1663/F2.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493080" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">G’ value profile of the QTL for apple (‘Jonathan’ × ‘Golden Delicious’) resistance/susceptibility to <em>Botryosphaeria dothidea</em> isolate Zz26. Y-axis represents G’ value. X-axis represents chromosome position. Red lines: ‘Jonathan’, blue lines: ‘Golden Delicious’, black lines: ‘Golden Delicious’ &amp; ‘Jonathan.’</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Identification of QTL for FRR resistance</h3><p id="p-39">For resistance/susceptibility to isolate Zz26, 12 significant QTL were obtained, six of which (J06.1-Zz26, J10.1-Zz26, J10.2-Zz26, J10.3-Zz26, H14.1-Zz26, and H14.2-Zz26) were considered as major QTL (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and Table S10). For resistance/susceptibility to <em>B. dothidea</em> isolate Ls1, Lw023, and Lw048, 14, 8, and 12 significant QTL were detected, respectively. Of these QTL, seven (H14.1-Ls1, G15.1-Ls1; G02.1-Lw023, G13.1-Lw023, G14.2-Lw023, H14.1-Lw048, and J15.2-Lw048) constituted major QTL (Table S14 and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a>, and <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G&#x2019; value profile of the QTL for apple (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) resistant/susceptible to Botryosphaeria dothidea isolate Ls1. Y-axis represents G&#x2019; value. X-axis represents chromosome position. Red lines: &#x2018;Jonathan&#x2019;, blue lines: &#x2018;Golden Delicious&#x2019;, black lines: &#x2018;Golden Delicious&#x2019; &amp; &#x2018;Jonathan.&#x2019;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-811376878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;G&#x2019; value profile of the QTL for apple (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) resistant/susceptible to &lt;em&gt;Botryosphaeria dothidea&lt;/em&gt; isolate Ls1. Y-axis represents G&#x2019; value. X-axis represents chromosome position. Red lines: &#x2018;Jonathan&#x2019;, blue lines: &#x2018;Golden Delicious&#x2019;, black lines: &#x2018;Golden Delicious&#x2019; &amp; &#x2018;Jonathan.&#x2019;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1663/F3.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1663/F3.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493014" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">G’ value profile of the QTL for apple (‘Jonathan’ × ‘Golden Delicious’) resistant/susceptible to <em>Botryosphaeria dothidea</em> isolate Ls1. Y-axis represents G’ value. X-axis represents chromosome position. Red lines: ‘Jonathan’, blue lines: ‘Golden Delicious’, black lines: ‘Golden Delicious’ &amp; ‘Jonathan.’</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G&#x2019; value profile of the QTL for apple (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) resistant/susceptible to Botryosphaeria dothidea isolate Lw023. Y-axis represents G&#x2019; value. X-axis represents chromosome position. Red lines: &#x2018;Jonathan&#x2019;, blue lines: &#x2018;Golden Delicious&#x2019;, black lines: &#x2018;Golden Delicious&#x2019; &amp; &#x2018;Jonathan.&#x2019;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-811376878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;G&#x2019; value profile of the QTL for apple (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) resistant/susceptible to &lt;em&gt;Botryosphaeria dothidea&lt;/em&gt; isolate Lw023. Y-axis represents G&#x2019; value. X-axis represents chromosome position. Red lines: &#x2018;Jonathan&#x2019;, blue lines: &#x2018;Golden Delicious&#x2019;, black lines: &#x2018;Golden Delicious&#x2019; &amp; &#x2018;Jonathan.&#x2019;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1663/F4.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1663/F4.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493012" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">G’ value profile of the QTL for apple (‘Jonathan’ × ‘Golden Delicious’) resistant/susceptible to <em>Botryosphaeria dothidea</em> isolate Lw023. Y-axis represents G’ value. X-axis represents chromosome position. Red lines: ‘Jonathan’, blue lines: ‘Golden Delicious’, black lines: ‘Golden Delicious’ &amp; ‘Jonathan.’</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G&#x2019; value profile of the QTL for apple (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) resistant/susceptible to Botryosphaeria dothidea isolate Lw048. Y-axis represents G&#x2019; value. X-axis represents chromosome position. Red lines: &#x2018;Jonathan&#x2019;, blue lines: &#x2018;Golden Delicious&#x2019;, black lines: &#x2018;Golden Delicious&#x2019; &amp; &#x2018;Jonathan.&#x2019;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-811376878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;G&#x2019; value profile of the QTL for apple (&#x2018;Jonathan&#x2019; &#xD7; &#x2018;Golden Delicious&#x2019;) resistant/susceptible to &lt;em&gt;Botryosphaeria dothidea&lt;/em&gt; isolate Lw048. Y-axis represents G&#x2019; value. X-axis represents chromosome position. Red lines: &#x2018;Jonathan&#x2019;, blue lines: &#x2018;Golden Delicious&#x2019;, black lines: &#x2018;Golden Delicious&#x2019; &amp; &#x2018;Jonathan.&#x2019;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1663/F5.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/5/1663/F5.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493115" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">G’ value profile of the QTL for apple (‘Jonathan’ × ‘Golden Delicious’) resistant/susceptible to <em>Botryosphaeria dothidea</em> isolate Lw048. Y-axis represents G’ value. X-axis represents chromosome position. Red lines: ‘Jonathan’, blue lines: ‘Golden Delicious’, black lines: ‘Golden Delicious’ &amp; ‘Jonathan.’</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Of the significant QTL, five QTL regions coincided or overlapped with resistance/susceptibility to more than two isolates. The QTL G02.1-Zz26/G02.1-Ls1, G15.1-Ls1 / H15.1-Lw023 and G06.1-Ls1/G06.1-Lw048 coincided, while the QTL regions of J10.2-Zz26/G10.2-Lw023 and H14.2-Zz26/H14.1-Ls1/H14.1-Lw048 overlapped (Table S14 and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). H14.2-Zz26/H14.1-Ls1/H14.1-Lw048 were consistently mapped to the same region along with four previously reported major gene loci (sf-g14-ls1, sf-g14-lw048, Rb-g14-lw048, and Rb-g14-zz26) (Figure S11) (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cui <em>et al.</em> 2014</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Circular overview of the QTL for apple resistance/susceptibility to fruit ring rot (Botryosphaeria dothidea) isolates Ls1 (red), Lw023 (orange), Lw048 (green), and Zz26 (blue) across the genome. The outer circle: QTL from &#x2018;Golden Delicious&#x2019; (G); intermediate circle: QTL from &#x2018;Jonathan&#x2019; (J); the inner circle: QTL from both &#x2018;Golden Delicious&#x2019; and &#x2018;Jonathan&#x2019;. The colored dots in the central map indicate the geographical origin of the pathogen isolates" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-811376878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Circular overview of the QTL for apple resistance/susceptibility to fruit ring rot (&lt;em&gt;Botryosphaeria dothidea&lt;/em&gt;) isolates Ls1 (red), Lw023 (orange), Lw048 (green), and Zz26 (blue) across the genome. The outer circle: QTL from &#x2018;Golden Delicious&#x2019; (G); intermediate circle: QTL from &#x2018;Jonathan&#x2019; (J); the inner circle: QTL from both &#x2018;Golden Delicious&#x2019; and &#x2018;Jonathan&#x2019;. The colored dots in the central map indicate the geographical origin of the pathogen isolates&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1663/F6.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/5/1663/F6.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1663/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493050" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Circular overview of the QTL for apple resistance/susceptibility to fruit ring rot (<em>Botryosphaeria dothidea</em>) isolates Ls1 (red), Lw023 (orange), Lw048 (green), and Zz26 (blue) across the genome. The outer circle: QTL from ‘Golden Delicious’ (G); intermediate circle: QTL from ‘Jonathan’ (J); the inner circle: QTL from both ‘Golden Delicious’ and ‘Jonathan’. The colored dots in the central map indicate the geographical origin of the pathogen isolates</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Narrowing down the QTL intervals with varied sliding window sizes</h3><p id="p-45">By using varied sliding window sizes, the 46 QTL for FRR resistance to the four pathogen isolates were significantly reduced (Table S15). The spanning regions of the 13 major QTL were narrowed down from 1,685,139 bp to 743,971 bp on average, and thus the significance of the average GV scores was also increased from 5.80 to 6.25 (Table S15). Several broader QTL intervals were split into two or more narrower sub-peaks; <em>e.g.</em>, the QTL J06.1-Zz26 was spliced into two sub peaks, J06.1_sub1-Zz26 and J06.2_sub2-Zz26, respectively (Table S15). As a result of this splicing, the QTL interval was narrowed down from 4,800,068 bp to 1,000,000 bp and 500,000 bp, and the average GV scores increased from 8.93 to 8.34 and 9.71, respectively (Table S15).</p></div><div id="sec-24" class="subsection"><h3>Candidate gene mining from QTL regions based on multi-omics data</h3><div id="sec-25" class="subsection"><h4>Excluding genes from the QTL intervals using parental re-sequencing data:</h4><p id="p-46">Of the 3,058 genes scattered on the 49 narrowed-down QTL regions including sub peaks (Table S15), 1,346 genes were by excluded using the parental SNV and SV databases, and 24,977 SNVs and 253 SVs were distributed in the 1,712 remaining candidate genes (Table S16).</p></div><div id="sec-26" class="subsection"><h4>Excluding genes from candidates using transcriptome data:</h4><p id="p-47">The RNA-Seq data were deposited in the NCBI Sequence Read Archive (SRA) with the accession number PRJNA392908. After removing low-quality reads and those derived from rRNA, the total reads input for mapping was 285.86 M, of which 207.9 M (80.63%) were aligned to the apple DH genome (Table S17). Overall, 136,983 transcripts from 86,808 genes (63,538 referenced and 23,270 non-referenced) were assembled. The Fragments Per Kilobase of transcript per Million mapped reads (FPKM) values of three representative genes, MD05G1236300, MD05G1261100, and MD08G1128800, were closely correlated with their relative expressions by RT-qPCR (R<sup>2</sup> = 0.921-0.962, <em>P</em> &lt; 0.01) (Figure S12). The correlation analysis and hierarchical cluster analysis revealed a close grouping of the same stage in both the extremely susceptible and resistant hybrids (Figure S13).</p><p id="p-48">In total, 1,570 DEGs were detected between the extremely susceptible and resistant hybrids across the sampling time points (Figure S14). SNVs, including 503,067 SNPs and 37,402 InDels, were identified in 36,796 genes. Following functional annotation, 103,510 non-synonymous SNVs were identified and were distributed across 24,533 genes.</p><p id="p-49">In addition to the transcriptomic data in this study, the RNA-Seq data of 25 transcriptome bio-projects and 432 samples from the SRA database were downloaded (Table S18). All the reads were mapped to the DH reference genome to quantify the expression of each gene. The expression data of the genes were also used to assist in the screening of organ-specific or non-expressive genes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eide <em>et al.</em> 1996</a>).</p><p id="p-50">Using the transcriptomic data, 56 non-expressed genes and 46 non-DEGs without variations in their CDS were culled from the 503 candidates in the QTL intervals for resistance/susceptibility to isolate Zz26 (Table S16). In the QTL regions for resistance/susceptibility to isolate Ls1, Lw023, and Lw048, 221 of the 1,209 genes were excluded owing to non-expression in any apple fruit samples (Table S16).</p></div><div id="sec-27" class="subsection"><h4>Excluding genes from candidates by functional annotation:</h4><p id="p-51">One thousand and one of the 1,389 candidate genes were successfully assigned with Gene Ontology (GO) terms. Based on the detailed gene annotations, 386 genes were excluded due to the inconsistency of their organ/tissue/sub-cellular localization, developmental dynamics, and physiological pathway annotations, after which 1,003 genes remained (Table S16).</p></div><div id="sec-28" class="subsection"><h4>Excluding genes from candidates based on their functional alteration appearing in the incorrect bulk:</h4><p id="p-52">The allele frequency of the screened functional variations in the extremity pools can aid candidate gene prediction. However, only small parts of the SNPs used in the BSA analysis can be directly assigned with allele frequency. Here, we phased the variations and constructed 740,329 haplotype blocks in the two parents. BSA markers were used as ‘anchor markers’ to assign the AFD between two bulks to the haplotype. For the pollen parent ‘Golden Delicious’, 5,505,264 SNPs were phased into 364,768 haplotype blocks, and after anchoring, 162,184 (44.46%) haplotype blocks were assigned an AFD between bulks. For the maternal parent ‘Jonathan’, 6,122,520 SNPs were phased into 375,561 haplotype blocks, and 196,220 (52.25%) haplotype blocks were assigned an AFD to the bulk pools. Considering meiotic crossover, the haplotypes of the male and female parent were referenced to each other. As for resistance/susceptibility to isolate Zz26, in this study, 2,051 of the 3,211 variations in 278 candidate genes were assigned an AFD, and the 42 genes with unexpected AFD values between the two bulk pools were excluded. For example, a stop-gain variation was screened in the CDS of MD15G1419300 in QTL G15.1, which is annotated as ‘mitogen-activated protein kinase kinase kinase’ gene; however, the stop-gain variation was enriched in the R pool, which is inconsistent with its function. On the contrary, a stop-gain variation in the CDS of MD10G1288400 in J10.3, a serine/threonine-protein kinase-like protein, was enriched in the S pool, which was expected.</p><p id="p-53">Ultimately, 57 candidate genes from 46 significant QTL associated with resistance/susceptibility to four <em>B. dothidea</em> isolates were selected for further experimental validation (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and Table S14).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493073/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493073/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of the candidate genes mined from the QTL for apple (‘Jonathan’ × ‘Golden Delicious’) resistance/susceptibility to <em>Botryosphaeria dothidea</em> isolates</span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-29" class="subsection"><h3>Experimental validation of the candidate genes</h3><p id="p-55">The presence/absence of variations in the 14 candidate genes from 11 major QTL linked to resistance to FRR isolate Zz26 was confirmed by Sanger sequencing (Table S19 and Table S20). The segregation of the variants was then validated by KASP genotyping in hybrids with extreme phenotypes, and all the variants exhibited significant associations between genotypes and phenotypes (<em>P</em> &lt; 0.05) (Table S21). The presence/absence of the markers was then verified in each 30 <em>Malus</em> germplasm accessions highly resistant/susceptible to FRR isolate Zz26 (Table S22). Ten KASP markers from nine QTL exhibited significant associations with the phenotypes (Table S23).</p><p id="p-56">Of the 14 candidate genes, five genes (MD03G1259600, MD16G1236700, MD10G1169200, MD10G1288400, and MD10G1288500), located in four QTL (H03.1, H16.1, J10.1, and J10.3), constituted protein kinases or kinase-like proteins (Table S19). The variations were validated by Sanger sequencing in the <em>cis</em>-element or transcription factor binding site of the promotors of MD03G1259600, MD16G1236700, and MD10G1169200 (Table S20), which is highly consistent with the finding that the expressions of these genes in fruit from the resistant bulk 48 h after inoculation were significantly higher than that in the susceptible bulk (Figure S15). Frame-shift deletions and stop-gain mutants were located in the CDS of the other two protein kinase genes, MD10G1288400 and MD10G1288500, respectively, which may cause functional loss of the genes (Table S20).</p><p id="p-57">From the major QTL J06.1, MD06G1037600, annotated as ‘NDR1/HIN1-like 8’ was ultimately screened as a candidate gene (Table S19). Re-sequencing and Sanger sequencing confirmed the variations in the <em>cis</em>-element and transcription factor (TF) binding site of this gene between parents (Table S20). The KASP analysis of the maker (KASP100) located in the promoter of MD06G1037600 exhibited significant association with the phenotype in the bulks (χ2= 7.4108 <em>P</em> = 0.006480 &lt;0.05) and the 60 accessions with extreme phenotypes (χ2 = 5.93, <em>P</em> = 0.0146 &lt;0.05) (Table S21, Table S23, and Table S20). The expression of this gene in the fruit from the resistant bulk 48 h after inoculation was consistently significantly higher than that in the susceptible bulk (Figure S15).</p><p id="p-58">Three pentatricopeptide repeat-containing genes, MD02G1213800, MD08G1120500, and MD08G1120700, were identified as candidate genes from QTL G02.1 and J08.1 (Table S19). Nonsynonymous SNVs were distributed in the functional domain of MD02G1213800, and the KASP analysis of one functional variation marker (KASP146) exhibited significant association with the phenotype of the two extreme pools (χ2 = 9.736, <em>P</em> = 0.001807 &lt;0.05) (Table S18 and Table S21). Stop-gain and frame-shift variations were detected and validated in both MD08G1120500 and MD08G1120700 (Table S19 and Table S20). Furthermore, KASP analysis confirmed that the stop-gain mutation A (KASP107), enriched in the susceptible bulk, exhibited significant AFD values between the two pools (χ2= 5.333, <em>P</em> = 0.02090 &lt;0.05) and the 60 extreme accessions (χ2= 4.593, <em>P</em> = 0.0321 &lt;0.05) (Table S20, Table S21, and Table S23). At the same time, frame-shift variations (KASP117) were also significantly enriched in the susceptible bulk (χ2 = 3.8595, <em>P</em> = 0.04950 &lt; 0.05) and extremely susceptible accessions (χ2 = 4.0074, <em>P</em> = 0.0453 &lt;0.05) (Table S20, Table S21, and Table S23).</p><p id="p-59">In the region of QTL G15.1, a disease-resistance gene (MD15G1416500) was detected with the largest G’ value (Table S19). A functional SNP (C to A) causing stop-gain was present in its CDS, and allele A was enriched in the susceptible bulk (Table S19). Furthermore, the corresponding KASP maker (KASP66) was significantly associated with the phenotype in both the resistant/susceptible pools (χ2 =9.145, <em>P</em> = 0.0025 &lt;0.05) and 60 extreme accessions (χ2= 5.079, <em>P</em> = 0.0242 &lt;0.05) (Table S20, Table S21, and Table S23).</p><p id="p-60">Two F-box/kelch-repeat proteins (MD06G1017400 and MD10G1255900) were detected in the QTL J06.1_sub1 and J10.2, respectively (Table S19). This protein is a component of SCF (ASK-cullin-F-box) E3 ubiquitin ligase complexes, which may mediate the ubiquitination and subsequent proteasomal degradation of target proteins (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Zhu <em>et al.</em> 2017</a>). Protein ubiquitination processes play an important role in plant–pathogen interactions (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lee <em>et al.</em> 2011</a>). Re-sequencing and Sanger sequencing confirmed the SNPs and InDels in the <em>cis</em>-element and TF binding site of MD06G1017400 between the parents (Table S18). One SNP (KASP140) located in the TF binding site was significantly associated with the phenotype in the two pools (χ2= 5.718, <em>P</em> = 0.0168 &lt;0.05) and 60 extreme accessions (χ2= 5.880, <em>P</em> = 0.0153 &lt;0.05) (Table S20, TableS21, and Table S23). The expression of this gene in the fruit of hybrids from the resistant bulk 48 h after inoculation was consistently significantly higher than that of the susceptible bulk (Figure S15). While for another gene, MD10G1255900, a functional SNP (G-to-A) (KASP87) causing stop-gain was present in its CDS, and an enriched allele A was also present in the susceptible bulk and exhibited significant AFD between the two pools (χ2 = 5.203, <em>P</em> = 0.02255 &lt;0.05) and the 60 extremes accessions (χ2 = 9.72, <em>P</em> = 0.001823 &lt;0.05) (Table S19, Table S20, Table S21, and Table S23).</p><p id="p-61">Ankyrin repeat-containing genes participate in the processes of signal transduction and protein kinase activity and thus may play an important role in the plant immune system (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mou <em>et al.</em> 2013</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ngaki <em>et al.</em> 2016</a>). In the QTL of G11.1, an ankyrin repeat-containing gene (MD11G1116700) was identified in the peak (Table S19). The expression of this gene in the fruit of hybrids from the resistant bulk 48 h after inoculation was significantly higher than that of the susceptible bulk (Figure S15). The significant difference in expression was possibly caused by the validated mutants in the promoter region of this gene between parents (Figure S15 and Table S20). Furthermore, the KASP marker KASP126, developed in the candidate functional InDels, was significantly associated with the phenotype in the two pools (χ2= 5.966, <em>P</em> = 0.01459 &lt; 0.05) and the 60 extreme accessions (χ2= 5.253, <em>P</em> = 0.02191 &lt; 0.05) (Table S20, Table S21, and Table S23).</p></div></div><div class="section discussion" id="sec-30"><h2 class="">Discussion</h2><div id="sec-31" class="subsection"><h3>The efficiency of the BAS-Seq strategy for QTL mapping in an outbreeding population</h3><p id="p-62">The double pseudo-testcross hypothesis was successfully applied to BSA mapping by separating the three types of markers. Data from either BSA-Seq or parental re-sequencing revealed three types of marker polymorphisms. The J- and G-type markers exhibit allelic heterozygosity and resemble the test cross for the maternal and paternal parent, respectively (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Grattapaglia and Sederoff 1994</a>). H-type markers exhibiting dual allelic heterozygosity are quite similar to the F<sub>2</sub> progeny in inbreeding species (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Ries <em>et al.</em> 2016</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Takagi <em>et al.</em> 2013</a>). The functional variations in the candidate genes controlling the target traits are also expected to possess the three types of polymorphisms (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bai <em>et al.</em> 2012b</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Takos <em>et al.</em> 2006</a>). Here, 13 major QTL were detected, four of which were from G, five of which were from J, and four of which were from the H polymorphism types (Table S14 and <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>-<a id="xref-fig-5-2" class="xref-fig" href="#F5">5</a>). The predicted and validated functional mutations in the candidate genes also exhibited J-, G-, and H-type polymorphisms, which facilitates further gene exploration (Table S19 and Table S20). The QTL detected by using 3-type method exhibited the narrowest QTL intervals or the highest G’ value (File S2). Hence, BSA mapping by separating the three marker types allowed for the full utilization of polymorphic markers as well as the direct mapping of QTL on certain parent(s).</p><p id="p-63">The GV method with a sliding window can realize noise reduction and unbiased statistical analysis for the three marker types. Three methods, namely SI, ED, and GV, are commonly used in BSA-Seq to reduce the noise and to measure the AFD between two extreme pools (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Hill <em>et al.</em> 2013</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Takagi <em>et al.</em> 2013</a>). In this study, 11 of the 12 significant QTL detected using the three methods coincided perfectly, indicating comparable effectiveness (Figure S5, Figure S6, Figure S7, and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). However, the QTL profiles detected by GV were both narrower and more significant than those detected by SI (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>) and exhibited better unbiasedness and a lower false-positive ratio than that of ED (Table S11).</p><p id="p-64">Sliding window analysis using different window sizes significantly narrowed down the QTL intervals. Fine-mapping based on large natural or pedigree segregation populations has been widely used but is both costly and labor-intensive/time-consuming (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bai <em>et al.</em> 2012a</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Laurens <em>et al.</em> 2018</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Terakami <em>et al.</em> 2016</a>). It is very difficult to create recombinant inbred lines or near isogenic lines in outbreeding woody perennials, such as apple, pear, and peach, which greatly hinders fine mapping in these species (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Di Pierro <em>et al.</em> 2016</a>). Here we showed that a large amount of markers derived from the deeper re-sequencing of the two DNA pools made it possible to narrow down the QTL intervals with varied sliding window sizes analysis, <em>e.g.</em>, for resistance/susceptibility to isolate Zz26, the marker density is close to 1059/Mbp on average (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and Figure S1), which is almost 200 times more than conventional MapQTL methods (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gardner <em>et al.</em> 2014</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sun <em>et al.</em> 2015</a>). By using varied sliding windows analysis, the spanning region of the QTL was narrowed down significantly from 1,639,342 bp to 707,246 bp, the averaged GV score increased from 5.55 to 6.03, and the stability of the QTL could also be evaluated (Figure S16 and Table S15).</p><p id="p-65">Compared with the previous studies using the same segregating population, more QTL with much narrow interval were detected and a large number of previously reported QTL or major loci were reproduced in this study (Figure S11) (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Cui <em>et al.</em>, 2014</a>; <a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Zhuang <em>et al.</em>, 2011</a>). H14.1-Ls1, H14.1-Lw048 and H14.2-Zz26 coincided with Sf-g14-ls1, Sf-g14-lw048 (Table S14 and Figure S11) (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Cui <em>et al.</em>, 2014</a>). Sf-j10-zz26 was split into three narrower QTL, J10.1-Zz26, J10.2-Zz26 and J10.3-Zz26, showing that the accuracy on QTL identification is higher by BSA-seq than MapQTL based on low density linkage maps (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Cui <em>et al.</em>, 2014</a>). The pathogen isolate Mx1 was not used in this study because its weak pathogenicity to apple fruit (<a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Zhuang <em>et al.</em>, 2011</a>). Therefore, a few QTL or major loci reported by <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Cui <em>et al.</em> (2014)</a> and <a id="xref-ref-78-6" class="xref-bibr" href="#ref-78">Zhuang <em>et al.</em> (2011)</a> were not presented.</p></div><div id="sec-32" class="subsection"><h3>Multi-omics enables gene culling from candidates in QTL regions</h3><p id="p-66">Deep re-sequencing indicated a lot of genetic variation between the two parental cultivars, which is consistent with previous reports (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lee <em>et al.</em> 2016</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Xing <em>et al.</em> 2016</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Zhang <em>et al.</em> 2014</a>). In the QTL intervals, genes lacking potential function-altering variations can be excluded. Transcriptomes can simultaneously indicate DEGs and genetic variations in CDS (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Balan <em>et al.</em> 2018</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bonnet <em>et al.</em> 2017</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Hill <em>et al.</em> 2013</a>). In this study, 1,570 DEGs and 103,510 SNVs in the CDS of 24,533 genes were identified between the resistant and susceptible hybrids (Figure S14). These data, combined with large online transcriptome datasets, assisted the efficient excluding of many genes from the QTL intervals (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Balan <em>et al.</em> 2017</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Balan <em>et al.</em> 2018</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Guo <em>et al.</em> 2017</a>).</p></div><div id="sec-33" class="subsection"><h3>The polygenetic signature of apple resistance to FRR provides evidence for the pathogenic variation among isolates</h3><p id="p-67">In this study, phenotyping was performed for five years, and hybrids with year-long robust resistance/susceptibility were selected for bulk creation. A total of 46 QTL, including 13 major QTL, were ultimately detected for resistance/susceptibility to the four <em>B. dothidea</em> isolates (Table S14). At least three major QTL for each pathogen isolate were found. These QTL demonstrated the apparent polygenetic signature of apple resistance to FRR, which corroborates previous reports (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Cui <em>et al.</em> 2014</a>; <a id="xref-ref-78-7" class="xref-bibr" href="#ref-78">Zhuang <em>et al.</em> 2011</a>). A variety of factors influence apple resistance to FRR, such as the size and density of the lenticels, the thickness of the wax layer, and the polyphenol content (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bai <em>et al.</em> 2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bénaouf and Parisi 2000</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fan <em>et al.</em> 2017</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2013</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yu <em>et al.</em> 2014</a>; <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Guan <em>et al.</em> 2015</a>). Protein kinase plays an important role in signal transduction and pathogen recognition in the plant immune system (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Park <em>et al.</em> 2012</a>). Receptor-like protein kinases (RLPs) are widely reported to participate in plant–pathogen interactions (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kruijt <em>et al.</em> 2005</a>). The NDR1 gene was once reported to confer resistance to <em>Pseudomonas syringae</em> pv. tomato DC3000 and plays an important role in defense against bacteria and viruses, and in the response to salicylic acid (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Varet <em>et al.</em> 2003</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Varet <em>et al.</em> 2002</a>). MD06G1037600 (<em>MdNDR1</em>), MD10G1288500 (RLP), MD10G1288400 (Serine/threonine-protein kinase-like protein), and MD10G1169200 (Kinase-like protein TMKL), which are associated with the activation of disease resistance, pathogen recognition, and disease resistance signaling, were predicted as the most likely candidates determining resistance/susceptibility to FRR (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and Table S19).</p><p id="p-68">Only four of the 46 QTL for resistance/susceptibility to the four isolates coincided or overlapped with others (Table S14,). The pathogenicity of <em>B. dothidea</em> differs significantly among isolates (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Liu <em>et al.</em> 2011</a>; <a id="xref-ref-78-8" class="xref-bibr" href="#ref-78">Zhuang <em>et al.</em> 2011</a>). The biological characteristics, pathogenicity, and internal transcribed spacer sequences differ significantly between pathogen strains of FRR (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lv <em>et al.</em> 2012</a>). The diversity in resistance/susceptibility of apple to varied pathogen isolates supports the genetic differentiation of the pathogen according to the gene-for-gene hypothesis (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Keen 1990</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Van der Biezen and Jones 1998</a>).</p><p id="p-69">For a complex trait like apple ring rot disease resistance, large number of QTL each capture only a small proportion of the total genetic variance, and the pathogenicity varies dramatically among isolates (<a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Cui <em>et al.</em>, 2014</a>). The 46 QTL identified in this study can be developed into QTL based genomics-assisted breeding or can be used as diagnostic markers among high density SNP arrays, which enables empirical genomic selection (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bianco <em>et al.</em>, 2014</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bianco <em>et al.</em>, 2016</a>). In addition, the candidate genes can help further research on the molecular mechanism of apple FRR resistance.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">We thank Dr. Luca Bianco and Dr. Paolo Fontana (Research and Innovation Center, Edmund Mach Foundation, Italy) for helpful discussion. This work was supported by the earmarked fund for the China Agriculture Research System (CARS-27), the Key Laboratory of Biology and Genetic Improvement of Horticultural Crops (Nutrition and Physiology), and the Ministry of Agriculture, China. The work was also partially supported by the Beijing Huiyuankangmin Organic Agriculture Co. Ltd.</p></div><div class="section ack" id="ack-2"><h2 class="">Ethical standards</h2><p id="p-71">The experiments were performed in accordance with all relevant Chinese laws.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-72">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7819688">https://doi.org/10.25387/g3.7819688</a>.</p></li><li class="fn" id="fn-5"><p id="p-73"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-10"><span class="received-label">Received </span>December 10, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-18"><span class="accepted-label">Accepted </span>March 18, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Shen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.1" data-doi="10.1016/j.plantsci.2014.11.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">H. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of a xyloglucan-specific endo-(1–4)-beta-D-glucanase inhibitor protein from apple (<em>Malus</em> × <em>domestica</em> Borkh.) as a potential defense gene against <em>Botryosphaeria dothidea</em>.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">231</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plantsci.2014.11.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D231%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2014.11.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2014.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.2" data-doi="10.1007/s00438-012-0689-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Fernandez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keulemans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Fine genetic mapping of the Co locus controlling columnar growth habit in apple.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-012-0689-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DBai%26rft.auinit1%253DT.%26rft.volume%253D287%26rft.issue%253D5%26rft.spage%253D437%26rft.epage%253D450%26rft.atitle%253DFine%2Bgenetic%2Bmapping%2Bof%2Bthe%2BCo%2Blocus%2Bcontrolling%2Bcolumnar%2Bgrowth%2Bhabit%2Bin%2Bapple.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-012-0689-5%26rft_id%253Dinfo%253Apmid%252F22526430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-012-0689-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22526430&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.3" data-doi="10.1007/s00438-012-0707-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dougherty</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fazio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">A natural mutation-led truncation in one of the two aluminum-activated malate transporter-like genes at the Ma locus is associated with low fruit acidity in apple.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-012-0707-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DBai%26rft.auinit1%253DY.%26rft.volume%253D287%26rft.issue%253D8%26rft.spage%253D663%26rft.epage%253D678%26rft.atitle%253DA%2Bnatural%2Bmutation-led%2Btruncation%2Bin%2Bone%2Bof%2Bthe%2Btwo%2Baluminum-activated%2Bmalate%2Btransporter-like%2Bgenes%2Bat%2Bthe%2BMa%2Blocus%2Bis%2Bassociated%2Bwith%2Blow%2Bfruit%2Bacidity%2Bin%2Bapple.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-012-0707-7%26rft_id%253Dinfo%253Apmid%252F22806345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-012-0707-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22806345&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.4" data-doi="10.3389/fpls.2017.01569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caruso</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinelli</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Gaining insight into exclusive and common transcriptomic features linked with biotic stress responses in <em>Malus</em>.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01569</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1569%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01569&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.5" data-doi="10.1038/s41598-018-19348-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marra</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caruso</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinelli</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Transcriptomic responses to biotic stresses in <em>Malus</em> × <em>domestica</em>: a meta-analysis study.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1970</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-19348-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D8%26rft.spage%253D1970%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-19348-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-19348-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.6" data-doi="10.1111/tpj.13145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bianco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cestaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linsmith</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muranty</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denance</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development and validation of the Axiom() Apple480K SNP genotyping array.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13145</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D86%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.7" data-doi="10.1371/journal.pone.0110377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bianco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cestaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargent</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derdak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and validation of a 20K single nucleotide polymorphism (SNP) whole genome genotyping array for apple (<em>Malus x domestica</em> Borkh).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e110377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0110377</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De110377%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0110377%26rft_id%253Dinfo%253Apmid%252F25303088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0110377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25303088&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.8" data-doi="10.1094/PHYTO.2000.90.3.236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bénaouf</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parisi</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetics of host-pathogen relationships between <em>Venturia inaequalis</em> races 6 and 7 and <em>Malus</em> species.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">236</span>–<span class="cit-lpage">242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHYTO.2000.90.3.236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DBenaouf%26rft.auinit1%253DG.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D236%26rft.epage%253D242%26rft.atitle%253DGenetics%2Bof%2BHost-Pathogen%2BRelationships%2BBetween%2BVenturia%2Binaequalis%2BRaces%2B6%2Band%2B7%2Band%2BMalus%2BSpecies.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO.2000.90.3.236%26rft_id%253Dinfo%253Apmid%252F18944614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO.2000.90.3.236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18944614&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.9" data-doi="10.1186/s12870-017-1074-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lassueur</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponzio</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gols</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dicke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Combined biotic stresses trigger similar transcriptomic responses but contrasting resistance against a chewing herbivore in <em>Brassica nigra</em>.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-017-1074-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D17%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-017-1074-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-017-1074-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.10" data-doi="10.1038/nmeth.3505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Layer</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faust</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindberg</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">SpeedSeq: ultra-fast personal genome analysis and interpretation.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">966</span>–<span class="cit-lpage">968</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3505</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D966%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3505%26rft_id%253Dinfo%253Apmid%252F26258291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26258291&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.11" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Gomez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti610</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16081474&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231694600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.12" data-doi="10.1111/jph.12279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic mapping reveals sophisticated responses of <em>Malus domestica</em> to <em>Botryosphaeria dothidea</em> isolates.</span> <abbr class="cit-jnl-abbrev">J. Phytopathol.</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jph.12279</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhytopathol.%26rft.volume%253D163%26rft.spage%253D42%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjph.12279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jph.12279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.13" data-doi="doi: 10.1038/ng.3886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daccord</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Celton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linsmith</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choisne</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High-quality de novo assembly of the apple genome and methylome dynamics of early fruit development.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3886</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1099%26rft_id%253Dinfo%253Adoi%252Fdoi%253A%2B10.1038%252Fng.3886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi: 10.1038/ng.3886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.14" data-doi="10.1038/hortres.2016.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Pierro</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianfranceschi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Guardo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koehorst-van Putten</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruisselbrink</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A high-density, multi-parental SNP genetic map on apple validates a new mapping approach for outcrossing species.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">16057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hortres.2016.57</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D3%26rft.spage%253D16057%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhortres.2016.57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hortres.2016.57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.15" data-doi="10.1073/pnas.93.11.5624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eide</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broderius</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fett</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerinot</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A novel iron-regulated metal transporter from plants identified by functional expression in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5624</span>–<span class="cit-lpage">5628</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.93.11.5624</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DEide%26rft.auinit1%253DD.%26rft.volume%253D93%26rft.issue%253D11%26rft.spage%253D5624%26rft.epage%253D5628%26rft.atitle%253DA%2Bnovel%2Biron-regulated%2Bmetal%2Btransporter%2Bfrom%2Bplants%2Bidentified%2Bby%2Bfunctional%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.11.5624%26rft_id%253Dinfo%253Apmid%252F8643627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTMvMTEvNTYyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.16" data-doi="10.1105/tpc.108.059329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Espley</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brendolise</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kutty-Amma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Multiple repeats of a promoter segment causes transcription factor autoregulation in red apples.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.108.059329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.108.059329%26rft_id%253Dinfo%253Apmid%252F19151225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIxLzEvMTY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.17" data-doi="10.1016/j.micres.2017.03.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ru</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Fengycin produced by <em>Bacillus subtilis</em> 9407 plays a major role in the biocontrol of apple ring rot disease.</span> <abbr class="cit-jnl-abbrev">Microbiol. Res.</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.micres.2017.03.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BRes.%26rft.volume%253D199%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.micres.2017.03.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.micres.2017.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.18" data-doi="10.1534/g3.114.011023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast and cost-effective genetic mapping in apple using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Genes Genomes Genetics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.011023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomes%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.011023%26rft_id%253Dinfo%253Apmid%252F25031181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNjgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.19" data-doi="10.1371/journal.pone.0147580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Rapid identification of candidate genes for seed weight using the SLAF-Seq method in Brassica napus.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0147580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0147580</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0147580%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0147580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0147580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1121%26rft.epage%253D1137%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bof%2BEucalyptus%2Bgrandis%2Band%2BEucalyptus%2Burophylla%2Busing%2Ba%2Bpseudo-testcross%253A%2Bmapping%2Bstrategy%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7982566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTY2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.21" data-doi="10.1093/jxb/erw021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Copy number variation of a gene cluster encoding endopolygalacturonase mediates flesh texture and stone adhesion in peach.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1993</span>–<span class="cit-lpage">2005</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erw021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferw021%26rft_id%253Dinfo%253Apmid%252F26850878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erw021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26850878&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.22" data-doi="10.1007/s10658-015-0682-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Role of lenticels and microcracks on susceptibility of apple fruit to <em>Botryosphaeria dothidea</em>.</span> <abbr class="cit-jnl-abbrev">Eur. J. Plant Pathol.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10658-015-0682-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D143%26rft.spage%253D317%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10658-015-0682-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10658-015-0682-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Investigations on the occurrence and chemical control of <em>Botryosphaeria</em> canker of apple in China.</span> <abbr class="cit-jnl-abbrev">Plant Protection</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BProtection%26rft.volume%253D35%26rft.spage%253D120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.24" data-doi="10.1039/C6FO01378C"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Apple and pear consumption and type 2 diabetes mellitus risk: a meta-analysis of prospective cohort studies.</span> <abbr class="cit-jnl-abbrev">Food Funct.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">934</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1039/C6FO01378C</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BFunct.%26rft.volume%253D8%26rft.spage%253D927%26rft_id%253Dinfo%253Adoi%252F10.1039%252FC6FO01378C%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/C6FO01378C&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.25" data-doi="10.1104/pp.17.01320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">J. K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y. Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Natural variation underlies differences in ETHYLENE RESPONSE FACTOR17 activity in fruit peel degreening.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2292</span>–<span class="cit-lpage">2304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.17.01320</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.17.01320%26rft_id%253Dinfo%253Apmid%252F29431631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTc2LzMvMjI5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.5.1663.26"><div class="cit-metadata unstructured">Harris, R. S., 2007 Improved pairwise alignment of genomic DNA. Ph.D. Thesis, The Pennsylvania State University.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.27" data-doi="10.1101/gr.146936.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demarest</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bisgrove</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorsi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MMAPPR: mutation mapping analysis pipeline for pooled RNA-seq.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">687</span>–<span class="cit-lpage">697</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.146936.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.146936.112%26rft_id%253Dinfo%253Apmid%252F23299975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNjg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.28" data-doi="10.1007/s00122-015-2509-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Illa-Berenguer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Houten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Knaap</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rapid and reliable identification of tomato fruit weight and locule number loci by QTL-seq.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1329</span>–<span class="cit-lpage">1342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2509-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D1329%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2509-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2509-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.29" data-doi="10.1146/annurev.ge.24.120190.002311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keen</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Gene-for-gene complementarity in plant-pathogen interactions.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">463</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ge.24.120190.002311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DKeen%26rft.auinit1%253DN.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D447%26rft.epage%253D463%26rft.atitle%253DGene-For-Gene%2BComplementarity%2BIn%2BPlant-Pathogen%2BInteractions%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.24.120190.002311%26rft_id%253Dinfo%253Apmid%252F2088175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.24.120190.002311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2088175&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EN96900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.30" data-doi="10.1038/nmeth.3317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HISAT: a fast spliced aligner with low memory requirements.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3317</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3317%26rft_id%253Dinfo%253Apmid%252F25751142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751142&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.31" data-doi="10.1126/science.1095011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto-Yamamoto</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirochika</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Retrotransposon-induced mutations in grape skin color.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1095011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKobayashi%26rft.auinit1%253DS.%26rft.volume%253D304%26rft.issue%253D5673%26rft.spage%253D982%26rft.epage%253D982%26rft.atitle%253DRetrotransposon-Induced%2BMutations%2Bin%2BGrape%2BSkin%2BColor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1095011%26rft_id%253Dinfo%253Apmid%252F15143274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3My85ODIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTY2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.32" data-doi="10.1111/j.1364-3703.2004.00264.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruijt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DE Kock</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>,<span class="cit-pub-date"> 2005</span> <span class="cit-article-title">Receptor-like proteins involved in plant disease resistance.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1364-3703.2004.00264.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DKruijt%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D97%26rft.atitle%253DReceptor-like%2Bproteins%2Binvolved%2Bin%2Bplant%2Bdisease%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2004.00264.x%26rft_id%253Dinfo%253Apmid%252F20565641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2004.00264.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20565641&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227009400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.33" data-doi="10.1270/jsbbs.16018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunihisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haji</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic dissection of a ‘Fuji’ apple cultivar: re-sequencing, SNP marker development, definition of haplotypes, and QTL detection.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1270/jsbbs.16018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D66%26rft.spage%253D499%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.16018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.16018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.34" data-doi="10.1038/s41438-018-0016-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurens</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranzana</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arus</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">An integrated approach for increasing breeding efficiency in apple and peach in Europe.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41438-018-0016-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D5%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41438-018-0016-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41438-018-0016-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.35" data-doi="10.1104/pp.111.177568"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The pepper E3 ubiquitin ligase RING1 gene, CaRING1, is required for cell death and the salicylic acid-dependent defense response.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">2011</span>–<span class="cit-lpage">2025</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.111.177568</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.177568%26rft_id%253Dinfo%253Apmid%252F21628629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU2LzQvMjAxMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.36" data-doi="10.4238/gmr.15038185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Analysis of ‘Fuji’ apple somatic variants from next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Res.</abbr> <span class="cit-vol">15</span>: gmr.15038185.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4238/gmr.15038185</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.4238%252Fgmr.15038185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4238/gmr.15038185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.37" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.38" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.39" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.40" data-doi="10.5423/PPJ.SI.08.2012.0125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">X. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Biological control of apple ring rot on fruit by <em>Bacillus amyloliquefaciens</em> 9001.</span> <abbr class="cit-jnl-abbrev">Plant Pathol. J.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">173</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5423/PPJ.SI.08.2012.0125</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%2BJ.%26rft.volume%253D29%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.5423%252FPPJ.SI.08.2012.0125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5423/PPJ.SI.08.2012.0125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.41" data-doi="10.1111/j.1439-0434.2011.01794.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evaluation of the resistance of <em>Malus</em> germplasm to bot canker caused by <em>Botryosphaeria dothidea</em>.</span> <abbr class="cit-jnl-abbrev">J. Phytopathol.</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0434.2011.01794.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhytopathol.%26rft.volume%253D159%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0434.2011.01794.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0434.2011.01794.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.42" data-doi="10.1006/meth.2001.1262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livak</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmittgen</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T))</span>. <abbr class="cit-jnl-abbrev">Method. Methods</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">402</span>–<span class="cit-lpage">408</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/meth.2001.1262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DLivak%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D402%26rft.epage%253D408%26rft.atitle%253DAnalysis%2Bof%2Brelative%2Bgene%2Bexpression%2Bdata%2Busing%2Breal-time%2Bquantitative%2BPCR%2Band%2Bthe%2B2%2528-Delta%2BDelta%2BC%2528T%2529%2529%2BMethod.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmeth.2001.1262%26rft_id%253Dinfo%253Apmid%252F11846609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/meth.2001.1262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11846609&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173949500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.43" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">550</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0550-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.44" data-doi="10.1007/s00122-014-2313-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL-seq identifies an early flowering QTL located near flowering locus T in cucumber.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1499</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2313-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1491%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2313-z%26rft_id%253Dinfo%253Apmid%252F24845123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2313-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24845123&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The relationship between rDNA-ITS sequences and biological characteristics of the apple ring rot pathogen <em>Botryosphaeria berengeriana</em> de Not f. sp. <em>piricola</em> (Nose).</span> <abbr class="cit-jnl-abbrev">Fungal Genom. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenom.%2BBiol.%26rft.volume%253D2%26rft.spage%253D104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.46" data-doi="10.1371/journal.pcbi.1002255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The statistics of bulk segregant analysis using next generation sequencing.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1002255</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DMagwene%26rft.auinit1%253DP.%2BM.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002255%26rft.epage%253De1002255%26rft.atitle%253DThe%2Bstatistics%2Bof%2Bbulk%2Bsegregant%2Banalysis%2Busing%2Bnext%2Bgeneration%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002255%26rft_id%253Dinfo%253Apmid%252F22072954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072954&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.47" data-doi="10.1073/pnas.88.21.9828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9828</span>–<span class="cit-lpage">9832</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.88.21.9828</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D88%26rft.issue%253D21%26rft.spage%253D9828%26rft.epage%253D9832%26rft.atitle%253DIdentification%2Bof%2Bmarkers%2Blinked%2Bto%2Bdisease-resistance%2Bgenes%2Bby%2Bbulked%2Bsegregant%2Banalysis%253A%2Ba%2Brapid%2Bmethod%2Bto%2Bdetect%2Bmarkers%2Bin%2Bspecific%2Bgenomic%2Bregions%2Bby%2Busing%2Bsegregating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.21.9828%26rft_id%253Dinfo%253Apmid%252F1682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMjEvOTgyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.48" data-doi="10.1385/MB:26:2:95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pindo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Isolation of functional RNA from small amounts of different grape and apple tissues.</span> <abbr class="cit-jnl-abbrev">Mol. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1385/MB:26:2:95</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiotechnology%26rft.stitle%253DMol%2BBiotechnol%26rft.aulast%253DMoser%26rft.auinit1%253DC.%26rft.volume%253D26%26rft.issue%253D2%26rft.spage%253D95%26rft.epage%253D100%26rft.atitle%253DIsolation%2Bof%2Bfunctional%2BRNA%2Bfrom%2Bsmall%2Bamounts%2Bof%2Bdifferent%2Bgrape%2Band%2Bapple%2Btissues.%26rft_id%253Dinfo%253Adoi%252F10.1385%252FMB%253A26%253A2%253A95%26rft_id%253Dinfo%253Apmid%252F14764934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1385/MB:26:2:95&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14764934&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189294400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.49" data-doi="10.1371/journal.pone.0059699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Functional analysis and expressional characterization of rice ankyrin repeat-containing protein, OsPIANK1, in basal defense against <em>Magnaporthe oryzae</em> attack.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e59699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0059699</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De59699%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0059699%26rft_id%253Dinfo%253Apmid%252F23555750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0059699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555750&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.50" data-doi="10.1371/journal.pone.0163106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ngaki</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahu</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farooqi</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Tanscriptomic study of the soybean-<em>Fusarium virguliform</em>e interaction revealed a novel ankyrin-repeat containing defense gene, expression of whose during infection led to enhanced resistance to the fungal pathogen in transgenic soybean plants.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0163106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0163106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0163106%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0163106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0163106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.51" data-doi="10.3389/fpls.2012.00177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caddell</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronald</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Protein phosphorylation in plant immunity: insights into the regulation of pattern recognition receptor-mediated signaling.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2012.00177</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BPlant%2BSci%26rft.aulast%253DPark%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D3%26rft.spage%253D177%26rft.epage%253D177%26rft.atitle%253DProtein%2Bphosphorylation%2Bin%2Bplant%2Bimmunity%253A%2Binsights%2Binto%2Bthe%2Bregulation%2Bof%2Bpattern%2Brecognition%2Breceptor-mediated%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2012.00177%26rft_id%253Dinfo%253Apmid%252F22876255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2012.00177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22876255&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.52" data-doi="10.1111/pbi.12686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vishwakarma</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shasidhar</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">QTL-seq approach identified genomic regions and diagnostic markers for rust and late leaf spot resistance in groundnut (<em>Arachis hypogaea L</em>.).</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">941</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12686</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D927%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.53" data-doi="10.1038/nbt.3122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonescu</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D290%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3122%26rft_id%253Dinfo%253Apmid%252F25690850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25690850&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.54" data-doi="10.1016/j.ajhg.2013.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piskol</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramaswami</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reliable identification of genomic variants from RNA-Seq data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">651</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2013.08.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.08.008%26rft_id%253Dinfo%253Apmid%252F24075185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24075185&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.55" data-doi="10.1186/s12864-016-2566-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ries</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtgrawe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viehover</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weisshaar</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Rapid gene identification in sugar beet using deep sequencing of DNA from phenotypic pools selected from breeding panels.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2566-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2566-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2566-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.56" data-doi="10.1093/nar/27.1.295"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rombauts</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehais</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Montagu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouze</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">PlantCARE, a plant cis-acting regulatory element database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/27.1.295</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.1.295%26rft_id%253Dinfo%253Apmid%252F9847207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.1.295&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9847207&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077983000078&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.57" data-doi="10.1126/science.1259215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Biosynthesis, regulation, and domestication of bitterness in cucumber.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1088</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1259215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DShang%26rft.auinit1%253DY.%26rft.volume%253D346%26rft.issue%253D6213%26rft.spage%253D1084%26rft.epage%253D1088%26rft.atitle%253DBiosynthesis%252C%2Bregulation%252C%2Band%2Bdomestication%2Bof%2Bbitterness%2Bin%2Bcucumber%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259215%26rft_id%253Dinfo%253Apmid%252F25430763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDYvNjIxMy8xMDg0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.58" data-doi="10.1111/pbi.12567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaganathan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thudi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roorkiwal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">QTL-seq for rapid identification of candidate genes for 100-seed weight and root/total plant dry weight ratio under rainfed conditions in chickpea.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2110</span>–<span class="cit-lpage">2119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12567</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D14%26rft.spage%253D2110%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12567&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.59" data-doi="10.1186/s12864-015-1946-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A dense SNP genetic map constructed using restriction site-associated DNA sequencing enables detection of QTLs controlling apple fruit quality.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">747</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1946-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D747%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1946-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1946-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.60" data-doi="10.1111/tpj.12105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosugi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natsume</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D74%26rft.spage%253D174%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12105%26rft_id%253Dinfo%253Apmid%252F23289725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289725&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316618500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.61" data-doi="10.1104/pp.106.088104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaffe</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacob</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Light-induced expression of a MYB gene regulates anthocyanin biosynthesis in red apples.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">1216</span>–<span class="cit-lpage">1232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.106.088104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.106.088104%26rft_id%253Dinfo%253Apmid%252F17012405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQyLzMvMTIxNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.62" data-doi="10.1270/jsbbs.66.271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terakami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adachi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunihisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishitani</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Fine mapping of the gene for susceptibility to black spot disease in Japanese pear (<em>Pyrus pyrifolia</em> Nakai).</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">280</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1270/jsbbs.66.271</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D66%26rft.spage%253D271%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.66.271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.66.271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.63" data-doi="10.1104/pp.103.020438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hause</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hause</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The <em>Arabidopsis</em> NHL3 gene encodes a plasma membrane protein and its overexpression correlates with increased resistance to <em>Pseudomonas syringae</em> pv. tomato DC3000.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">2023</span>–<span class="cit-lpage">2033</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.103.020438</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.020438%26rft_id%253Dinfo%253Apmid%252F12913158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTMyLzQvMjAyMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.64" data-doi="10.1094/MPMI.2002.15.6.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornero</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nass</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurnberger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">NHL25 and NHL3, two NDR1/HIN1–1ike genes in <em>Arabidopsis thaliana</em> with potential role(s) in plant defense.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">608</span>–<span class="cit-lpage">616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/MPMI.2002.15.6.608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DVaret%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D608%26rft.epage%253D616%26rft.atitle%253DNHL25%2Band%2BNHL3%252C%2Btwo%2BNDR1%252FHIN1-1ike%2Bgenes%2Bin%2BArabidopsis%2Bthaliana%2Bwith%2Bpotential%2Brole%2528s%2529%2Bin%2Bplant%2Bdefense.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI.2002.15.6.608%26rft_id%253Dinfo%253Apmid%252F12059109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI.2002.15.6.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12059109&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175938200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.65" data-doi="10.1016/S0968-0004(98)01311-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Biezen</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Plant disease-resistance proteins and the gene-for-gene concept.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">456</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0968-0004(98)01311-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DVan%2Bder%2BBiezen%26rft.auinit1%253DE.%2BA.%26rft.volume%253D23%26rft.issue%253D12%26rft.spage%253D454%26rft.epage%253D456%26rft.atitle%253DPlant%2Bdisease-resistance%2Bproteins%2Band%2Bthe%2Bgene-for-gene%2Bconcept.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0968-0004%252898%252901311-5%26rft_id%253Dinfo%253Apmid%252F9868361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0968-0004(98)01311-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9868361&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077507300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.66" data-doi="10.1038/ng.654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zharkikh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affourtit</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhingra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cestaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome of the domesticated apple (<em>Malus</em> × <em>domestica</em> Borkh.).</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.654</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DVelasco%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D839%26rft.atitle%253DThe%2Bgenome%2Bof%2Bthe%2Bdomesticated%2Bapple%2B%2528Malus%2B%25C3%2597%2Bdomestica%2BBorkh.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.654%26rft_id%253Dinfo%253Apmid%252F20802477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20802477&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282276600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.67" data-doi="10.1093/nar/gkq603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq603%26rft_id%253Dinfo%253Apmid%252F20601685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20601685&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.68" data-doi="10.1038/ng.3839"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic analyses of primitive, wild and cultivated citrus provide insights into asexual reproduction.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">772</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3839</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D765%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.69" data-doi="10.1007/s00122-016-2806-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Win</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vegas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">QTL mapping for downy mildew resistance in cucumber via bulked segregant analysis using next-generation sequencing and conventional methods.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2806-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D199%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2806-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2806-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.70" data-doi="10.1093/nar/gkr483"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">KOBAS 2.0: a web server for annotation and identification of enriched pathways and diseases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">W316</span>–<span class="cit-lpage">W322</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr483</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr483%26rft_id%253Dinfo%253Apmid%252F21715386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21715386&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292325300051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.71" data-doi="10.3389/fpls.2016.00908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide sequence variation identification and floral-associated trait comparisons based on the re-sequencing of the ‘Nagafu No. 2’ and ‘Qinguan’ varieties of apple (<em>Malus domestica</em> Borkh.).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.00908</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D708%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.00908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.00908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.72" data-doi="10.1371/journal.pone.0068433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping of quantitative trait loci underlying cold tolerance in rice seedlings via high-throughput sequencing of pooled extremes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0068433</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De68433%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068433%26rft_id%253Dinfo%253Apmid%252F23935868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23935868&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.73" data-doi="10.1007/s11033-014-3399-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">X. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Apple ring rot-responsive putative microRNAs revealed by high-throughput sequencing in <em>Malus</em> × <em>domestica</em> Borkh.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">5273</span>–<span class="cit-lpage">5286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11033-014-3399-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BRep.%26rft.volume%253D41%26rft.spage%253D5273%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-014-3399-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-014-3399-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.74" data-doi="10.1093/bioinformatics/17.9.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">InterProScan–an integration platform for the signature-recognition methods in InterPro.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">848</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/17.9.847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DZdobnov%26rft.auinit1%253DE.%2BM.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D847%26rft.epage%253D848%26rft.atitle%253DInterProScan%2B-%2Ban%2Bintegration%2Bplatform%2Bfor%2Bthe%2Bsignature-recognition%2Bmethods%2Bin%2BInterPro%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.9.847%26rft_id%253Dinfo%253Apmid%252F11590104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.9.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11590104&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171566400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.75" data-doi="10.1104/pp.16.01504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">TATA box insertion provides a selection mechanism underpinning adaptations to Fe deficiency.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">727</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.16.01504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.16.01504%26rft_id%253Dinfo%253Apmid%252F27881725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNzMvMS83MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTY2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.76" data-doi="10.1038/hortres.2014.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic variants of genes associated with three horticultural traits in apple revealed by genome re-sequencing.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">14045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hortres.2014.45</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D1%26rft.spage%253D14045%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhortres.2014.45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hortres.2014.45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> The F-box protein KIB1 mediates brassinosteroid-induced inactivation and degradation of GSK3-like kinases in <em>Arabidopsis</em>. Molecular Cell 66: 648–657 e644.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2017.05.012</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1663.78" data-doi="10.1111/j.1439-0434.2011.01839.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inheritance of and molecular markers for susceptibility of <em>Malus domestica</em> to fruit ring rot (<em>Botryosphaeria dothidea</em>).</span> <abbr class="cit-jnl-abbrev">J. Phytopathol.</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">782</span>–<span class="cit-lpage">788</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0434.2011.01839.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhytopathol.%26rft.volume%253D159%26rft.spage%253D782%26rft.atitle%253DINHERITANCE%2BOF%2BAND%2BMOLECULAR%2BMARKERS%2BFOR%2BSUSCEPTIBILITY%2BOF%2BMALUS%2BDOMESTICA%2BTO%2BFRUIT%2BRING%2BROT%2B%2528BOTRYOSPHAERIA%2BDOTHIDEA%2529%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0434.2011.01839.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0434.2011.01839.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1663.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1655" title="Genetic Properties Responsible for the Transgressive Segregation of Days to Heading in Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1679" title="HSB-1 Inhibition and HSF-1 Overexpression Trigger Overlapping Transcriptional Changes To Promote Longevity in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1663" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-A7C8X08K15SWEL-MNcSjre-pzbK6OUYfDN5N1eDzdbs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1663" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1663" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1663" class="active" data-icon-position="" data-hide-link-title="0">Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492883" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zdKETE8sWUNJi-80dj1oOVADLHzyLN_EPI2VbT5QqtI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1663.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1663" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DcNYZD-QSlWKS3QZTaMbEN_LX_w9kmesNBEY1e_Q-Js" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1663" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1663.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492883" id="node492883--2" data-pisa="ggg;9/5/1663" data-pisa-master="ggg;g3.119.400167" data-apath="/ggg/9/5/1663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fei Shen</span>, <span class="highwire-citation-author" data-delta="1">Zhenyu Huang</span>, <span class="highwire-citation-author" data-delta="2">Baoguo Zhang</span>, <span class="highwire-citation-author" data-delta="3">Yi Wang</span>, <span class="highwire-citation-author" data-delta="4">Xi Zhang</span>, <span class="highwire-citation-author" data-delta="5">Ting Wu</span>, <span class="highwire-citation-author" data-delta="6">Xuefeng Xu</span>, <span class="highwire-citation-author" data-delta="7">Xinzhong Zhang</span> and <span class="highwire-citation-author" data-delta="8">Zhenhai Han</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1663-1678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400167 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492883--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fei Shen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BShen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shen%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BShen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhenyu Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenyu%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhenyu%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Baoguo Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Zhongbaolvdu Agricultural Research Centre, Beidaihe, Hebei, 066100, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baoguo%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaoguo%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ting Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATing%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xuefeng Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuefeng%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuefeng%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xinzhong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinzhong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinzhong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangxinzhong999@126.com" class="" data-icon-position="" data-hide-link-title="0">zhangxinzhong999@126.com</a>
<a href="mailto:rschan@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">rschan@cau.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zhenhai Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenhai%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhenhai%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangxinzhong999@126.com" class="" data-icon-position="" data-hide-link-title="0">zhangxinzhong999@126.com</a>
<a href="mailto:rschan@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">rschan@cau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1663" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1663&amp;title=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1663&amp;title=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1663&amp;title=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1663&amp;text=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1663&amp;title=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1663&amp;t=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1663&amp;title=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1663&amp;title=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492883/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492883" id="node492883--3" data-pisa="ggg;9/5/1663" data-pisa-master="ggg;g3.119.400167" data-apath="/ggg/9/5/1663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Gene Markers for Apple Fruit Ring Rot Disease Resistance Using a Multi-omics Approach</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Fei Shen</span>, <span class="highwire-citation-author" data-delta="1">Zhenyu Huang</span>, <span class="highwire-citation-author" data-delta="2">Baoguo Zhang</span>, <span class="highwire-citation-author" data-delta="3">Yi Wang</span>, <span class="highwire-citation-author" data-delta="4">Xi Zhang</span>, <span class="highwire-citation-author" data-delta="5">Ting Wu</span>, <span class="highwire-citation-author" data-delta="6">Xuefeng Xu</span>, <span class="highwire-citation-author" data-delta="7">Xinzhong Zhang</span> and <span class="highwire-citation-author" data-delta="8">Zhenhai Han</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1663-1678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400167 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492883--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fei Shen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BShen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shen%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BShen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhenyu Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenyu%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhenyu%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Baoguo Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Zhongbaolvdu Agricultural Research Centre, Beidaihe, Hebei, 066100, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baoguo%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaoguo%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ting Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATing%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xuefeng Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuefeng%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuefeng%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xinzhong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinzhong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinzhong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangxinzhong999@126.com" class="" data-icon-position="" data-hide-link-title="0">zhangxinzhong999@126.com</a>
<a href="mailto:rschan@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">rschan@cau.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zhenhai Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Horticulture, China Agricultural University, Beijing 100193, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenhai%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhenhai%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangxinzhong999@126.com" class="" data-icon-position="" data-hide-link-title="0">zhangxinzhong999@126.com</a>
<a href="mailto:rschan@cau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">rschan@cau.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1663&amp;rft.epage=1678&amp;rft.atitle=Mapping%20Gene%20Markers%20for%20Apple%20Fruit%20Ring%20Rot%20Disease%20Resistance%20Using%20a%20Multi-omics%20Approach&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shen%2C+Fei&amp;rft.au=Huang%2C+Zhenyu&amp;rft.au=Zhang%2C+Baoguo&amp;rft.au=Wang%2C+Yi&amp;rft.au=Zhang%2C+Xi&amp;rft.au=Wu%2C+Ting&amp;rft.au=Xu%2C+Xuefeng&amp;rft.au=Zhang%2C+Xinzhong&amp;rft.au=Han%2C+Zhenhai"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492883/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492883/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492883/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492883/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492883/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492883/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492883/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492883/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492883/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492883/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492883/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492883/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1663" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-30" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#ack-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Ethical standards</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1663.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1663.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1663.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
