<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1487" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1487.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1487.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1487.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1487.atom" />
<meta name="HW.pisa" content="ggg;9/5/1487" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Neurospora crassa Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA" />
<meta name="DC.Identifier" content="10.1534/g3.119.400006" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Giri et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Meiotic silencing by unpaired DNA (MSUD), an RNAi-mediated gene silencing process, is efficient in crosses made in the Neurospora crassa standard Oak Ridge (OR) genetic background. However, MSUD was decidedly less efficient when the OR-derived MSUD testers were crossed with many wild-isolated strains (W), suggesting that either sequence heterozygosity in tester x W crosses suppresses MSUD, or that OR represents the MSUD-conducive extreme in the range of genetic variation in MSUD efficiency. Our results support the latter model. MSUD was less efficient in near-isogenic crosses made in the novel N. crassa B/S1 genetic background, and in N. tetrasperma strain 85. Possibly, in B/S1 and 85, additional regulatory cues, absent from OR, calibrate the MSUD response. A locus in distal chromosome 1R appears to underlie the OR vs. B/S1 difference. Repeat-induced point mutation (RIP) destroys duplicated genes by G:C to A:T mutation of duplicated DNA sequences. Chromosome segment duplications ( Dp s) dominantly suppress RIP, possibly by titrating out the RIP machinery. In Dp x N crosses, the Dp –borne genes cannot pair properly, hence efficient MSUD, as in OR, silences them and renders the crosses barren. We speculate that the increased productivity engendered by inefficient MSUD enables small duplications to escape RIP." />
<meta name="DC.Contributor" content="Dev Ashish Giri" />
<meta name="DC.Contributor" content="Ajith V. Pankajam" />
<meta name="DC.Contributor" content="Koodali T. Nishant" />
<meta name="DC.Contributor" content="Durgadas P. Kasbekar" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Neurospora crassa Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Meiotic silencing by unpaired DNA (MSUD), an RNAi-mediated gene silencing process, is efficient in crosses made in the &lt;i&gt;Neurospora crassa&lt;/i&gt; standard Oak Ridge (OR) genetic background. However, MSUD was decidedly less efficient when the OR-derived MSUD testers were crossed with many wild-isolated strains (W), suggesting that either sequence heterozygosity in &lt;i&gt;tester&lt;/i&gt; x W crosses suppresses MSUD, or that OR represents the MSUD-conducive extreme in the range of genetic variation in MSUD efficiency. Our results support the latter model. MSUD was less efficient in near-isogenic crosses made in the novel &lt;i&gt;N. crassa&lt;/i&gt; B/S1 genetic background, and in &lt;i&gt;N. tetrasperma&lt;/i&gt; strain 85. Possibly, in B/S1 and 85, additional regulatory cues, absent from OR, calibrate the MSUD response. A locus in distal chromosome 1R appears to underlie the OR &lt;i&gt;vs.&lt;/i&gt; B/S1 difference. Repeat-induced point mutation (RIP) destroys duplicated genes by G:C to A:T mutation of duplicated DNA sequences. Chromosome segment duplications (&lt;i&gt;Dp&lt;/i&gt;s) dominantly suppress RIP, possibly by titrating out the RIP machinery. In &lt;i&gt;Dp&lt;/i&gt; x &lt;i&gt;N&lt;/i&gt; crosses, the &lt;i&gt;Dp&lt;/i&gt;–borne genes cannot pair properly, hence efficient MSUD, as in OR, silences them and renders the crosses barren. We speculate that the increased productivity engendered by inefficient MSUD enables small duplications to escape RIP. &lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1487" />
<meta name="citation_id" content="9/5/1487" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1487" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1487.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1487.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1487.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400006" />
<meta name="citation_pmid" content="30842206" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1487" />
<meta name="citation_lastpage" content="1496" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dev Ashish Giri" />
<meta name="citation_author_institution" content="Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500039, India" />
<meta name="citation_author_institution" content="Graduate Studies, Manipal University, Manipal 576104, and" />
<meta name="citation_author" content="Ajith V. Pankajam" />
<meta name="citation_author_institution" content="School of Biology" />
<meta name="citation_author" content="Koodali T. Nishant" />
<meta name="citation_author_institution" content="School of Biology" />
<meta name="citation_author_institution" content="Centre for Computation Modelling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram 695551, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4342-7501" />
<meta name="citation_author" content="Durgadas P. Kasbekar" />
<meta name="citation_author_institution" content="Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500039, India" />
<meta name="citation_author_email" content="kas@cdfd.org.in" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5269-8020" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Aramayo;citation_author=R. L. Metzenberg;citation_title=Meiotic transvection in fungi.;citation_pages=103-113;citation_volume=86;citation_year=1996;citation_issue=1;citation_pmid=8689677;citation_doi=10.1016/S0092-8674(00)80081-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Bhat;citation_author=D. P. Kasbekar;citation_title=Escape From Repeat-Induced Point Mutation of a Gene-Sized Duplication in Neurospora crassa Crosses That Are Heterozygous for a Larger Chromosome Segment Duplication;citation_pages=1581-1590;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290714" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Bhat;citation_author=F. K. Noubissi;citation_author=M. Vyas;citation_author=D. P. Kasbekar;citation_title=Genetic Analysis of Wild-Isolated Neurospora crassa Strains Identified as Dominant Suppressors of Repeat-Induced Point Mutation;citation_pages=947-961;citation_volume=164;citation_year=2003;citation_issue=3;citation_pmid=12871906" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Bhat;citation_author=R. Tamuli;citation_author=D. P. Kasbekar;citation_title=Genetic Transformation of Neurospora tetrasperma, Demonstration of Repeat-Induced Point Mutation (RIP) in Self-Crosses and a Screen for Recessive RIP-Defective Mutants;citation_pages=1155-1164;citation_volume=167;citation_year=2004;citation_issue=3;citation_pmid=15280231;citation_doi=10.1534/genetics.103.025171" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: A flexible trimmer for Illumina Sequence Data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Calhoun;citation_author=H. B. Howe;citation_title=GENETIC ANALYSIS OF EIGHT-SPORED ASCI PRODUCED BY GENE E IN NEUROSPORA TETRASPERMA;citation_pages=449-459;citation_volume=60;citation_year=1968;citation_issue=3;citation_pmid=5728739" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=A. M. Carroll;citation_author=J. A. Sweigard;citation_author=B. Valent;citation_title=Improved vectors for selecting resistance to hygromycin.;citation_pages=22;citation_volume=41;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=R. H. Davis;citation_author=F. J. De Serres;citation_title=Genetic and microbiological research techniques for Neurospora crassa.;citation_pages=79-143;citation_volume=17;citation_year=1970;citation_doi=10.1016/0076-6879(71)17168-6" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=L. M. Decker;citation_author=H. Xiao;citation_author=E. C. Boone;citation_author=M. M. Vierling;citation_author=B. S. Shanker;citation_title=The nuclear cap-binding complex mediates meiotic silencing by unpaired DNA.;citation_volume=7;citation_year=2017;citation_pmid=28179391;citation_doi=10.1534/g3.116.038679" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. Garimella;citation_author=J. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=B. O. Dodge;citation_title=A new dominant lethal in Neurospora. The E locus in N. tetrasperma.;citation_pages=467-474;citation_volume=30;citation_year=1939;citation_doi=10.1093/oxfordjournals.jhered.a104632" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=M. Fehmer;citation_author=A. Bhat;citation_author=F. K. Noubissi;citation_author=D. P. Kasbekar;citation_title=Wild-isolated Neurospora crassa strains that increase fertility of crosses with segmental aneuploids used to establish that a large duplication suppresses RIP in a smaller duplication.;citation_pages=13-14;citation_volume=48;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. A. Giri;citation_author=S. Rekha;citation_author=D. P. Kasbekar;citation_title=Neurospora heterokaryons with complementary duplications and deficiencies in their constituent nuclei provide an approach to identify nucleus-limited genes.;citation_volume=5;citation_year=2015;citation_pmid=25897010;citation_doi=10.1534/g3.115.017616" />
<meta name="citation_reference" content="citation_journal_title=Adv. Genet.;citation_author=T. M. Hammond;citation_title=Sixteen years of meiotic silencing by unpaired DNA.;citation_volume=97;citation_year=2017;citation_doi=10.1016/bs.adgen.2016.11.001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Hammond;citation_author=H. Xiao;citation_author=E. C. Boone;citation_author=L. M. Decker;citation_author=S. A. Lee;citation_title=Novel proteins required for meiotic silencing by unpaired DNA and siRNA generation in Neurospora crassa.;citation_volume=194;citation_year=2013;citation_pmid=23502675;citation_doi=10.1534/genetics.112.148999" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Hammond;citation_author=W. G. Spollen;citation_author=L. M. Decker;citation_author=S. M. Blake;citation_author=G. K. Springer;citation_title=Identification of small RNAs associated with meiotic silencing by unpaired DNA.;citation_volume=194;citation_year=2013;citation_pmid=23502674;citation_doi=10.1534/genetics.112.149138" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_author=D. J. Jacobson;citation_title=Sexual dysfunction associated with outcrossing in Neurospora tetrasperma, a pseudohomothallic ascomycete.;citation_pages=604-617;citation_volume=87;citation_year=1995;citation_doi=10.2307/3760805" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=D. J. Jacobson;citation_author=N. B. Raju;citation_author=M. Freitag;citation_title=Evidence for the absence of meiotic silencing by unpaired DNA in Neurospora tetrasperma.;citation_pages=351-362;citation_volume=45;citation_year=2008;citation_issue=3;citation_pmid=18023372;citation_doi=10.1016/j.fgb.2007.09.014" />
<meta name="citation_reference" content="citation_journal_title=Journal of biosciences;citation_journal_abbrev=J Biosci;citation_author=D. P. Kasbekar;citation_title=Magic with moulds: meiotic and mitotic crossing over in Neurospora inversions and duplications.;citation_pages=3-4;citation_volume=31;citation_year=2006;citation_issue=1;citation_pmid=16595868;citation_doi=10.1007/BF02705229" />
<meta name="citation_reference" content="citation_title=Neurospora: Genomics and Molecular Biology;citation_title=Neurospora duplications, and genome defense by RIP and meiotic silencing;citation_author=D. P. Kasbekar;citation_pages=109-127;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Biosci.;citation_author=D. P. Kasbekar;citation_author=S. Rekha;citation_title=Neurospora tetrasperma crosses heterozygous for hybrid translocation strains produce rare eight spored asci bearing heterokaryotic ascospores.;citation_pages=15-21;citation_volume=42;citation_year=2017;citation_doi=10.1007/s12038-017-9671-z" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Liu;citation_author=Q. Wang;citation_author=Y. He;citation_author=L. Chen;citation_author=C. Hao;citation_title=Genome-wide A-to-I RNA editing in fungi independent of ADAR enzymes.;citation_volume=26;citation_year=2016;citation_pmid=26934920;citation_doi=10.1101/gr.199877.115" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Liu;citation_author=Y. Li;citation_author=D. Chen;citation_author=Z. Qi;citation_author=Q. Wang;citation_title=A-to-I RNA editing is developmentally regulated and generally adaptive for sexual reproduction in Neurospora crassa.;citation_volume=114;citation_year=2017;citation_pmid=28847945;citation_doi=10.1073/pnas.1702591114" />
<meta name="citation_reference" content="citation_journal_title=Journal of biosciences;citation_journal_abbrev=J Biosci;citation_author=K. McCluskey;citation_author=A. Wiest;citation_author=M. Plamann;citation_title=The Fungal Genetics Stock Center: a repository for 50 years of fungal genetics research.;citation_pages=119-126;citation_volume=35;citation_year=2010;citation_issue=1;citation_pmid=20413916;citation_doi=10.1007/s12038-010-0014-6" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=R. L. Metzenberg;citation_title=Vogel’s Medium N salts: avoiding the need for ammonium nitrate.;citation_pages=14;citation_volume=50;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Neurospora Newslett.;citation_author=R. L. Metzenberg;citation_author=S. K. Ahlgren;citation_title=Hybrid strains useful in transferring genes from one species of Neurospora to another.;citation_pages=9-10;citation_volume=15;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=T. Nagasowjanya;citation_author=B. Kranthi Raj;citation_author=K. Sreethi Reddy;citation_author=D. P. Kasbekar;citation_title=An apparent increase in meiotic silencing strength in crosses involving inbred Neurospora crassa strains.;citation_pages=158-162;citation_volume=56;citation_year=2013;citation_doi=10.1016/j.fgb.2013.05.002" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Ninomiya;citation_author=K. Suzuki;citation_author=C. Ishii;citation_author=H. Inoue;citation_title=Highly efficient gene replacements in Neurospora strains deficient for nonhomologous end-joining.;citation_volume=101;citation_year=2004;citation_pmid=15299145;citation_doi=10.1073/pnas.0402780101" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=F. K. Noubissi;citation_author=K. McCluskey;citation_author=D. P. Kasbekar;citation_title=Repeat-induced point mutation (RIP) in crosses with wild-isolated strains of Neurospora crassa: Evidence for dominant reduction of RIP.;citation_pages=91-97;citation_volume=31;citation_year=2001;citation_doi=10.1006/fgbi.2000.1235" />
<meta name="citation_reference" content="citation_journal_title=Advances in genetics;citation_journal_abbrev=Adv Genet;citation_author=D. D. Perkins;citation_title=Chromosome rearrangements in Neurospora and other filamentous fungi.;citation_pages=239-398;citation_volume=36;citation_year=1997;citation_pmid=9348657;citation_doi=10.1016/S0065-2660(08)60311-9" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=D. D. Perkins;citation_title=Wild type Neurospora crassa strains preferred for use as standards.;citation_pages=7-8;citation_volume=51;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. D. Perkins;citation_author=M. Freitag;citation_author=V. C. Pollard;citation_author=L. A. Bailey-Shrode;citation_author=E. U. Selker;citation_title=Recurrent Locus-Specific Mutation Resulting From a Cryptic Ectopic Insertion in Neurospora;citation_pages=527-544;citation_volume=175;citation_year=2007;citation_issue=2;citation_pmid=17322355;citation_doi=10.1534/genetics.106.065714" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. J. Pratt;citation_author=D. W. Lee;citation_author=R. Aramayo;citation_title=DNA Methylation Affects Meiotic trans-sensing, Not Meiotic Silencing, in Neurospora;citation_pages=1925-1935;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611165;citation_doi=10.1534/genetics.104.031526" />
<meta name="citation_reference" content="citation_journal_title=Mycol. Res.;citation_author=N. B. Raju;citation_title=Functional heterothallism resulting from homokaryotic conidia and ascospores in Neurospora tetrasperma.;citation_pages=103-116;citation_volume=96;citation_year=1992;citation_doi=10.1016/S0953-7562(09)80923-4" />
<meta name="citation_reference" content="citation_journal_title=Developmental genetics;citation_journal_abbrev=Dev Genet;citation_author=N. B. Raju;citation_author=D. D. Perkins;citation_title=Diverse programs of ascus development in pseudohomothallic species of Neurospora, Gelasinospora, and Podospora.;citation_pages=104-118;citation_volume=15;citation_year=1994;citation_issue=1;citation_pmid=8187347;citation_doi=10.1002/dvg.1020150111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. B. Raju;citation_author=R. L. Metzenberg;citation_author=P. K. T. Shiu;citation_title=Neurospora Spore killers Sk-2 and Sk-3 suppress meiotic silencing by unpaired DNA.;citation_volume=176;citation_year=2007;citation_pmid=17339226;citation_doi=10.1534/genetics.106.069161" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=M. Ramakrishnan;citation_author=T. N. Sowjanya;citation_author=K. B. Raj;citation_author=D. P. Kasbekar;citation_title=Meiotic silencing by unpaired DNA is expressed more strongly in the early than the late perithecia of crosses involving most wild-isolated Neurospora crassa strains and in self-crosses of N. tetrasperma.;citation_pages=1146-1152;citation_volume=48;citation_year=2011;citation_issue=12;citation_pmid=22056520;citation_doi=10.1016/j.fgb.2011.10.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. A. Samarajeewa;citation_author=P. A. Sauls;citation_author=K. J. Sharp;citation_author=Z. J. Smith;citation_author=H. Xiao;citation_title=Efficient detection of unpaired DNA requires a homolog from the Rad54-like family of homologous recombination proteins.;citation_volume=198;citation_year=2014;citation_pmid=25146971;citation_doi=10.1534/genetics.114.168187" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. A. Samarajeewa;citation_author=P. Manitchotpisit;citation_author=M. Henderson;citation_author=H. Xiao;citation_author=D. G. Rehard;citation_title=An RNA recognition motif-containing protein functions in meiotic silencing by unpaired DNA.;citation_volume=7;citation_year=2017;citation_pmid=28667016;citation_doi=10.1534/g3.117.041848" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. U. Selker;citation_title=Premeiotic Instability Of Repeated Sequences In Neurospora Crassa;citation_pages=579-613;citation_volume=24;citation_year=1990;citation_issue=1;citation_pmid=2150906;citation_doi=10.1146/annurev.ge.24.120190.003051" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. K. T. Shiu;citation_author=R. L. Metzenberg;citation_title=Meiotic Silencing by Unpaired DNA: Properties, Regulation and Suppression;citation_pages=1483-1495;citation_volume=161;citation_year=2002;citation_issue=4;citation_pmid=12196394" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. K. T. Shiu;citation_author=N. B. Raju;citation_author=D. Zickler;citation_author=R. L. Metzenberg;citation_title=Meiotic silencing by unpaired DNA.;citation_pages=905-916;citation_volume=107;citation_year=2001;citation_issue=7;citation_pmid=11779466;citation_doi=10.1016/S0092-8674(01)00609-2" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. K. T. Shiu;citation_author=D. Zickler;citation_author=N. B. Raju;citation_author=G. Ruprich-Robert;citation_author=R. L. Metzenberg;citation_title=SAD-2 is required for meiotic silencing by unpaired DNA and perinuclear localization of SAD-1 RNA-directed RNA polymerase.;citation_volume=103;citation_year=2006;citation_pmid=16461906;citation_doi=10.1073/pnas.0508896103" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=P. K. Singh;citation_author=D. P. Kasbekar;citation_title=Titration of repeat-induced point mutation (RIP) by chromosome segment duplications in Neurospora crassa.;citation_pages=267-275;citation_volume=134;citation_year=2008;citation_issue=3;citation_pmid=18046508;citation_doi=10.1007/s10709-007-9234-6" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=P. K. Singh;citation_author=S. V. Iyer;citation_author=M. Ramakrishnan;citation_author=D. P. Kasbekar;citation_title=Chromosome segment duplications in Neurospora crassa: barren crosses beget fertile science.;citation_pages=209-219;citation_volume=31;citation_year=2009;citation_issue=2;citation_pmid=19204993;citation_doi=10.1002/bies.200800098" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. C. Turner;citation_author=D. D. Perkins;citation_title=Conventional and unconventional analysis of an inversion in Neurospora.;citation_pages=175-190;citation_volume=40;citation_year=1982;citation_issue=2;citation_pmid=7152257;citation_doi=10.1017/S0016672300019042" />
<meta name="citation_reference" content="Van der Auwera G. A., M. Carneiro, C. Hartl, R. Poplin, G. del Angel et al., 2013 From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline. Current Protocols in Bioinformatics 43: 11.10. 1–33. 10.1002/0471250953.bi1110s43" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Vyas;citation_author=C. Ravindran;citation_author=D. P. Kasbekar;citation_title=Chromosome segment duplications in Neurospora crassa and their effects on repeat-induced point mutation (RIP) and meiotic silencing by unpaired DNA.;citation_volume=172;citation_year=2006;citation_pmid=16361235;citation_doi=10.1534/genetics.105.050468" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=J. H. Yu;citation_author=Z. Hamari;citation_author=K. H. Han;citation_author=J. A. Seo;citation_author=Y. Reyes-Domínguez;citation_title=Double-joint PCR: a PCR-based molecular tool for gene manipulations in filamentous fungi.;citation_pages=973-981;citation_volume=41;citation_year=2004;citation_issue=11;citation_pmid=15465386;citation_doi=10.1016/j.fgb.2004.08.001" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Neurospora crassa Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Meiotic silencing by unpaired DNA (MSUD), an RNAi-mediated gene silencing process, is efficient in crosses made in the Neurospora crassa standard Oak Ridge (OR) genetic background. However, MSUD was decidedly less efficient when the OR-derived MSUD testers were crossed with many wild-isolated strains (W), suggesting that either sequence heterozygosity in tester x W crosses suppresses MSUD, or that OR represents the MSUD-conducive extreme in the range of genetic variation in MSUD efficiency. Our results support the latter model. MSUD was less efficient in near-isogenic crosses made in the novel N. crassa B/S1 genetic background, and in N. tetrasperma strain 85. Possibly, in B/S1 and 85, additional regulatory cues, absent from OR, calibrate the MSUD response. A locus in distal chromosome 1R appears to underlie the OR vs. B/S1 difference. Repeat-induced point mutation (RIP) destroys duplicated genes by G:C to A:T mutation of duplicated DNA sequences. Chromosome segment duplications ( Dp s) dominantly suppress RIP, possibly by titrating out the RIP machinery. In Dp x N crosses, the Dp –borne genes cannot pair properly, hence efficient MSUD, as in OR, silences them and renders the crosses barren. We speculate that the increased productivity engendered by inefficient MSUD enables small duplications to escape RIP." />
<meta name="og-title" property="og:title" content="The Neurospora crassa Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1487" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Meiotic silencing by unpaired DNA (MSUD), an RNAi-mediated gene silencing process, is efficient in crosses made in the Neurospora crassa standard Oak Ridge (OR) genetic background. However, MSUD was decidedly less efficient when the OR-derived MSUD testers were crossed with many wild-isolated strains (W), suggesting that either sequence heterozygosity in tester x W crosses suppresses MSUD, or that OR represents the MSUD-conducive extreme in the range of genetic variation in MSUD efficiency. Our results support the latter model. MSUD was less efficient in near-isogenic crosses made in the novel N. crassa B/S1 genetic background, and in N. tetrasperma strain 85. Possibly, in B/S1 and 85, additional regulatory cues, absent from OR, calibrate the MSUD response. A locus in distal chromosome 1R appears to underlie the OR vs. B/S1 difference. Repeat-induced point mutation (RIP) destroys duplicated genes by G:C to A:T mutation of duplicated DNA sequences. Chromosome segment duplications ( Dp s) dominantly suppress RIP, possibly by titrating out the RIP machinery. In Dp x N crosses, the Dp –borne genes cannot pair properly, hence efficient MSUD, as in OR, silences them and renders the crosses barren. We speculate that the increased productivity engendered by inefficient MSUD enables small duplications to escape RIP." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>The Neurospora crassa Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"xrxQy5aAZSFoOTueQ7NLs9jnTjQZMrrf2ylWvVyAilA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492877","apath":"\/ggg\/9\/5\/1487.atom","pisa":"ggg;9\/5\/1487","ac":{"\/ggg\/9\/5\/1487.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1487.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1487"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1487"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1487":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022l6nukMzdbI52VF48Bq3kruJHjqEV0fNnObvhpGYT54ZiwiGbx6U3fCGf7hjDvvWAXI+4Bc8fzECxZhU4o2ZPF08bGii4wdI1opL1pS9sM46IO97OvH\\\\\\\/\\\\\\\/3mhc0t8IvJAZ4YZ7AuMMfnSEQ0URvELpOa87IK5BeTzvEPlw4uI50P3qTeMpo7zn8V15ZlB\\\\\\\/DL6k3HKignm94dxxzWQjlN\\\\\\\/NpmXVlNe53OWSUyciGBfvWGWSjiKRVxQ+lKsbi\\\\\\\/iSBZqhSujEmzB3UA8V2SRCT\\\\\\\/V4ROw6DQPXB5ZDO5xmwv78ghJX\\\\\\\/kQ9qTW29sL52fjYtQS9msQI5AjIhJKZzEl2bZRFrVA7NqKnl0ptRjAbzzV6oodYy4MhEeH29ZOXefnB\\\\\\\/Y1CjORYnRMVr76o5LvQYBuAOLeLP+G5shSLvnA9oIklnagQquGuehaydzxZQ6sUwfklzSCCgyaEgyiB0uY5ktBXT2H\\\\\\\/kbfmg4hEAi9ilUItCjfW3440GfqThYsWoqteg6v5apObFfTogc\\\\\\\/bw1x0+53P0dRk0WHnMdS\\\\\\\/Z8UUswXSxrzAsvMGxMGBRKSjPM72kmeaMFYy5VvEPNrqIbxJOzGRaOBjzewBAGH8a1kB1uZ\\\\\\\/OC09+3ODDfE0oSekLpC0rdykNV3QsIlnIeE2PStz0YtX\\\\\\\/Iq0Y5R9djpHHX1wEKyTtPrtiF\\\\\\\/KkYJ8M7c5mcrpv0FkRH2m5wXEM\\\\\\\/XNEhT0Y5j9seBEScmv2D0\\\\\\\/\\\\\\\/dmyr3Liyg7z\\\\\\\/kbI3iosgoiZXmIdtdNnyVvwARYUjODE6B8hLF76dmC3eySevEzJ+SM+pCMa68mYO9Pu0iOPdcfw3YiC3VFRUzE+oiSZkKG4rTcAbvzhhTfLPaMb2yG6YA3OaFBh2SDXi8tGFCL8uRXZdDEIUAbkO4ytyhRnogKQbJIkTXn9nPZXNrGyOzeUlHpNeh3GLJbaiA90eU4r\\\\\\\/tzRCR4hcHTKVgIG5nQNbCtJyoGCBFIl09BPx+1HrnvSi0wBdtMXrhwkihyBGIS\\\\\\\/lDqCcsj0Wnf02DenJXC1xvKTAnTc8lNTC7dNfFLbUAGAFYcDEy6hTmkcUzRvsf7NiPHF8YmZnccuo\\\\\\\/+TLrzjWk6Mgz4ryEg71o+Vff4Xho1rZeVxcqWdA2wZPH3fvPrGtKBzrYSPch+mUeo1lqsj8UtQsbZldMXom+JCRZcSInEparea0hxpg6PI+yoik6vap225XEOUyM0PrvM0qy+GxbMqsUzUbYKDA8ItzmopxmwrS6V\\\\\\\/Y6KRMp58Y9bXr9OrKKTpGBAhR0ie\\\u0022,\\\u0022iv\\\u0022:\\\u0022Io42K4OZgAkCYlboplG7PQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022424832b97052cca51dc40e4c62508b31fb2a4dd63c37255830ddbc37d6ca4d91\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ETLLBiqu63iusgYLFOsNs4rISuzNWt4Mu7P5HOzQu\\\\\\\/mEzrZv0n1dVp9zyiG72E\\\\\\\/02OWu3uh95B4DEPOIhoa3cGMJk5i+PGYekt5eYwLXG7XuoLW6YmTJlVhmJ8uP49H\\\\\\\/xfzfB7K40MS6\\\\\\\/BpcYzkJpdEgMM+OKW+J1ffAYLSN+wNelR6sC96FzmZER9hN1UyvS9crrX0ZKz0w37JNFGKqr+KR\\\\\\\/sbvC38WYgghGqeMFymeireNZ0tRm8Y8ny598yu328HGQTFA5D0e3r39juW3E1mlFa10LQPEOI1idMLgZz6YcMhF0s7tled9hYY2RDNV6L0LztW1DRKaM4vTNYrsXcrx5i2o33slCpcHEmkwvPHjvu6suI19MlEL+hiWJgvB\\\\\\\/xiLPpjePxEIVkYG+o1MZuuK2FJ8v7raWj3k1PDUxb3aaRVFPcEEwGm8OhFHRRDQPfLkZqS36dmpBsZZD6sbWiVprQZRAsim5pjG1bcd7Jk+5+aywlqPtEwgnDxYHUdVxmVa8ny8YSQ9ln0zD\\\\\\\/P2Dzn9Xp83DuXevZLxeaHFRLlrrWwel4tqsmLa2MaV3oQ1l7cL951EsrT76B4zRje+lyGl9LYPRU84+btnq14TYbhLckHHmPG9Rpi8elxKPGIUCaB57bUX8h2dIQlHq7JVaLXFlWYB\\\\\\\/u0T46g2ZyDTp4\\\\\\\/C2TkEtV6vO3QLTbCiG8IOtEvZnpDGoFNwCxSlBdnOrZRNJVjYjfyBN0nkBOUjhRLJBTqTCmRsdDonPFkcQ4\\\\\\\/HWthP1vwSyCxV56FHMMCwLTc6G\\\\\\\/7jkdgU9hEc9dFE1mkFoJlY4Msz08Kg2jkbvDMkU5l9RsRFQigCoTX4URF9mgtyOuvglxc87hmfSxfUoQv3UWXfqR2D0Iqr0S92g6hgcMbbV3WsV3Oa5yYB1HJ4bygV29D6uQyKalIdKnIDQa0ONZKCCS6aBxMQNQOjwheiPxmrb9hjh5jRFs3QUvlYcbqg2SvjL3w2KpgIa0mgNl26MNs7cHLBPzJJin\\\\\\\/M7zRjfypyVqB20xgAq3dz19nkDgzzEvR0LBFAgF5fYOahgOzuU0AMKrpM70O1RgoFinl3jllnT8enqrlsOC24EW5D3Yc5EKV0QgHGGZBTW8f+Y\\\\\\\/tmKElIDnPLZmon6KSMOs361oRjMAMt9eUQLLTJ9gDLyRfvU6fx24y21Zr6SYK4uGeMCDfLtkxYEg\\\\\\\/qq9hOGWR43APcyDDeYaYH+ixyG9nXfNSqubRgVepBhj6dQFpdne9zJFFI1FIP7x7VnlSPVOMNZf2zCu+BSwgmvISqh+s4VQBjOaVDkvLazCECGqV3YlC9uoKr8RFXL2mK1M7j\\\\\\\/xNtxHesstnH4pkx99D4WP59Uc+Unw7Ra1SSc5MK8kbPzIp2wbFiHTO2RgkWj9eVK+CrWHycx8ev9VZz9WiN7RPkO1n9gmwBfV5fIwFWqkZ89SPT8oLlylumPR2irT9MhPb2qroPe0UqfV4cqsYmQ2OAqHUZXwypUNbIuLYo8IT84Q9xtT7EmbdGTcmkH7J+4gk4OMHbSBO3nL3IVP3FqpYMbbh5Qvx5MKvrmX+RLLmrYodeLUr4K1BiZFoYTKMaTiZ8ZBR67WjzIC1lpyLCcK0l21359CouWjsoJyYygdAnsZXNa6x1i7uLBZOuMkupgJkbDtHe\\\\\\\/89a7IbvZdsn3BHhxBk5qNVSyqdvaiYs0pBQeQJtzp1GW+raGUGTjA5Gt1xb7yshLxeE+oYiAFfkn0MKsB4KItcrnFnEbCmxdnD6YLFc7LxWgSamKVgg9gVrvCaV14d3SJhV1JRMHuffBEZCLQLGpwSngsJARmHETVpFLNkkezVoWjh11pNjcjjKtHOxSzFqBNx1UtFGde6Kd\\\\\\\/2kr2b67oPAdssK+tYwkviUzW8JFqEUX93xxN\\\\\\\/b7ly4xUS5xgcrouKnHjRiVmlico42quwmRdbKILf9TJSD\\\\\\\/Mp67i2u5hk0cgI\\\\\\\/p5BaXID9xAJSyHfa7epozMe7GpPAwjxxDs2VmA9WQCMOLrqXR8Lqbdmb6opGT8F37Aj5QFKGWPWleNjOyArqYawYfJ\\\\\\\/8xJ9OQhkLhXQC7W2pwogOfx8uhQyukYc8rIVtdwbmqnUT4gULETBYR8SaCLCoI3HjVNSNGUCGVee0bOZCIMpHloUKRFt3brWqmP+G7pV6Wr0Zrd0XuAAqxmK3BM+ZRE9Vuht0gA6t8Bj2YPKT5ZA0rNlK71A\\\\\\\/vv+Mn71Vln03QzbIUqQytVlaQQ1Fhb7tWRY7II23hZLt675heAT0Tlo9CLdEZd0qhA6KNpV8XbgwxPrr4Bh0rh5YxZh3mr53g1gkbSJSJh3SD165A8itXuqsnWDLdoB6bHe+c+UUS3FBSKSwtdnS6KQs+W+w81JYha0uMI8MX2cP7GRug1y4nU5La8ELb\\\\\\\/b8yTWQORDgFETqU6v5FoRw2h7hP2KYBgZNja27StkRjRXnG8K4ez7QYJiPWSIczy+r9horDwbb1E2ede8o4ruWt\\\\\\\/KRa6EYeG77IgdqvjWLaP1vAPpXjMsy4wt93kIaxWOnHlVupPF60vFJfATE1qwzjMpq871Xk56KrqPf2D+PWfM\\\\\\\/aIrrbKENbQ+mSaRvf767Nuc2KHP7vXHLG+LJ8FnTG\\\\\\\/bCWC+pmcYclvZzs1tDiNfIojmhJ0WuaR55oB8tADDB0FZyD1t4uxITEed1QEmBii5LbvbFzjyNOZUrD3xeqVQTPoGJbE0qD0SB8avplaMInVq+NhaV+bE1Zjgr\\\\\\\/4vcHqoVBYVvBWj+vNOWkzmlShx32FqF\\\\\\\/Fxc8DX2OF7yX7314j3bpoTLSsSMV9fF5e9c623JGJnjhbWmeVgPqHSyjhuHt0QnaIdGGZj3Q6YJQUOVenPhOmvSShxKm4QQHMEEDoPASFa8XLfJu4OqWMUoYGKxbQScJM3uyZjs3Lcr\\\\\\\/x4NZepz8jZhVIdK\\\\\\\/srUxf1qlX1O1wZ75IqvxuVd11lBFG6bUs49vzOIv5WXj96dV38vYGBaRVhABRPJmjDZGdDg6Re0w\\\\\\\/yqWbUkpp4Zzbz8yyX9TUB8l7M7WAnSK+9hSoA8yRGejOXL\\\\\\\/lYSZl4na+p3pTQTE68uhrjGqFt9aR\\\\\\\/gHNInEmTEAwD7qLu5clr5ouANg6UuAz34zENfRiR80wGE5DUCW\\\\\\\/3LPu1UJCVWN+WZNZIWygpSvTpER5IZ3zthb3Dmd6ZdadIDmLDVIS+Me+BghvRlCcCdKtyzeR5EOUKZ02jCNQNR6JpScPJC9lpFI8NX184L6G7dXb1CsIVe59z7+Rus6E5gRU8qUVOLxOwNNv1+zQQaJceUP4Id8WTcFPyvlHh6u+mcgtYGVjsaz6MXxCp7u1uX1OdEL3cPEmCj1YWupiR8vu4IIVdrpBo\\\\\\\/JP3oIVf0lzI2cfPUXcApzT8M\\\\\\\/YIZecdqn6EWFoQtTzRR9THT4L2ZyaHjXjyJrjsGWUyl4pAzHaLoVY1czg\\\\\\\/IPNLqCGL5TTzUPdQY1f9Aha4A2CavA7cvQU9s2TfgLGCCpycHPefV6YuO5TM66Tjwc1ITNOlELFwq599Hc+JHqMQk6\\\\\\\/BT4qiOdTVllh4zPlrQ2qBHp4YlEM6J9gbHmeyvPO\\\\\\\/g\\\\\\\/I1XCX9xWxyGSrEph8NijKXxlzySpwIAUvXCMJne1rxmm4rrGzrfMrZ0pJ5NoZEjf6LgVGb6Un9Ts72Xr8YjfNZ97V2JsAg11fjyIAWSCCAcD5JDlTrPcUfNqe6JYkthnVHhuVW\\\\\\\/0gs4LcCOM143WN2i68jApOQoVeLIh2wAV+BtQlvQ7v+tYh+po4bHHo2LruhK\\\\\\\/fBW8bMYjDHI\\\\\\\/LE7by3xT4TyQrjNjAJgKxvFUVmRMRkWRWZSpL\\\\\\\/64Mz\\\\\\\/hSakY6bvQjvWpcq0gjKPPdLz+1WvjzaMa8W\\\\\\\/uUuiXLN2xjUd2sh1aG4\\\\\\\/5E4qCMNC880KurO7BGiIUNruOAHqMyY8Kh77Mj2MPZkuPS8HofqXjuXRyId+Ya0emOydaPij8RDWXB4bpeMD1jgEOH7lV+knKKwmn65JNT6j\\\\\\\/g4qv78RPs5xBiuJ1zczNVYxe+gvl7tSLtcLsAwA1Sto7AGORGhpaCVHZ+TmVW86aOC\\\\\\\/4nmlxGWa9zVYilMojqbbLbcYk9MrRNDeo7F4MwrFdSchXlb2AuxNKy57FfvTpxMWKdWEzgJznsOVLAzFtOkYppFZqxUsxB0bPEdgRU8fewbCHn9IxIGlS0PPW6bAUFeWTu+miHW2TtZJSlGKcpiZO7m1j38\\\\\\\/rGIAw5QZn1ojJIk2xaNV2v7GRTvy8Zmd+opW\\\\\\\/ZThnPXvzs8enHke0EBAKIZ2QWA5FYefLty1zBwkaDK6E7elFzZzVvKXYZt6DDpGH7MFJnxEoCspRdf7\\\\\\\/FKGDq6+RLJjO8il1L7757QlGNvamSTvwBVJeH75ISeb6\\\\\\\/WNzIsFIDl4o\\\\\\\/V\\\\\\\/4jDzYK54Opyu44MQB3+MI9o0bfy4U7ZajoqBWMgXIWBk7ScGthh1X1XKQpl3JTCYWypakehdvue2C1PWkIzA7v9mzQsHkJZAE9838wfEkT67SHHXlQjebP31YkHt\\\\\\\/XBY75BJBfTYTpT7TMB2gEb14Gb7Ab5gZ\\\\\\\/9i1fRwffhat4IUGp1n2Z2N9IGbIw0LS1qLoQOGcbntELECk+WxKocfK5GUc+OP4ldU3ybxP04fziOQrHe5b++JWbMC1ErK2nUHuUGMNWBoiu5oxAipA6j0ZY\\\\\\\/ECWQzVuwsv3p0X8Xsx9DDOpEXEmoGnT5+8e5sdnrHS4Z\\\\\\\/WTKw55iX0Lid1AlzuOF38AVBO5CgbUSKR63sRqYGhLPoEfk2Du238+AfN8S\\\\\\\/Ykz6iswMWctusqM7SzaeqFPqroETy05d6RAS4WG4ubTmnf2eDw6GsRNPxVtrXEXQk64K4vMIFZ76xRjw8gt0ZJSDnzCgSTVpIY8x3UqzZK0DjK\\\\\\\/8up+QGg3MQc25RAR5iDdjsbESO0FVlLXtGiWm65RuHDsUjGQjxGpWtsBOc7hh69t3favliyKAYcYICs8yhilLpUaE0pKkdNZm4XMb+45jmSqqRSaGiIiXZ8yHWTG0IXeQz41SWIlZaUduaBj8VRYhSgZG9AnCnuG65U6H3yHBRLxTKcSPppscOnQxdFUQHLLjhhvIP219dBAJMh1IWjn9Ipa1wSZuC01rOUUD8\\\\\\\/BcAQgfLud0C\\\\\\\/2UC1FtzDFWOYRrZ2sB1YBs6mnYzBNNm662qngxhmKWcoXlCbcRCc946V6wBOsSpNaV5ydykNcqYwsh7jkTtq6UC\\\\\\\/6mzZl\\\\\\\/X4cvHoto8RTA\\\\\\\/SZhOBgqdrH0co+q9dV9e4ygVfUrr7nUXjGjqjC5\\\\\\\/\\\\\\\/fQosoYDpiYqbs0WAQa8dUnVJrRkpzcsZk8TlEUSZBNJIlOaKXR1gi5G4Q+vl50R1lA9vOvtUjTFa5qIxbd\\\\\\\/nLpBlCiUCmf9u6JnWP2ChSCGG6g+JBdE9XSIBxxvJgzTbwAS3OStxsEyuLzl0XWpulhnAj2Z0EZFUYxPr72FV7Caph6DFmevnYHK5Qc0qxDfIEkGx5x4PEoLo22Kn1lfpR7WWV6+78ELAUleZCxJ7tkOMBI\\\\\\\/eCUcuByTf0TrIdzW6CFJORp9ErHsH5kxjZCMExcJ5hwuKG8cOtRPNuwijji34FwMOv5RwGVwTzxCSiqjUhMlhOvVsnwJU6hZMNeo7cj138PRq0\\\\\\\/v1snLfgLx3h7UXCzMIpRDumrhoori\\\\\\\/lpR6+PF5R\\\\\\\/ztpKQd1R2o8rzS+Y6RUMD09R\\\\\\\/Me5YHgNOXswsk5wV19nR6EvRMAtf7JWa8SnXiVD0njj94h1kK9JHw3XwyTzuGrwOjzNQfvSGeEvtLVi0TS5U7l01c7khj\\\\\\\/9UTvgg3qYiizwXlbOL8KXTWrQNZtZfBV6nkq8VubWM1siQcYm0bc+KlXGBV07rhTvJC0HdXLb+iAQQViLFbAZpHDNAA\\\\\\\/Z\\\\\\\/Q\\\\\\\/Pklgd0Xz601g9MqvvUIXuQSGfJmSGn17CjjJEj37r5eqXKXsYFol885blD6fhPpl41\\\\\\\/LdIX2JfsOLOd6P6ySWoU+nc6yNNbpxNElMihZlUwY20seZjTVJbmPP1qDeXiZrFnCzmzGFOI1lYcLD3uqp\\\\\\\/XkrwvVmTedEWmkmQoAS+jreEmviPIAKH2duch6RpkMHpZ\\\\\\\/8o3uZ4w1qOrbnKg\\\\\\\/x3pPS58ydnOtRP9wuxCHVHJmcPyikAUhWtG6FIoNhEpiNWxi1TLZLFoXs9VEM+u23Xw3nta\\\\\\\/oSVuSEb++hbN6bUHUksHCBquQEruijpGfGS1R1WkOM6Pywa68plGerqfkPcHPdAJA4Evl5naOZqHk7R4f9val28XtsGwPk5EyjVA6TmoxUs6sAGzcgb2\\\\\\\/ju6SU0m70QWuH3Ltz4E5ffz3z0IfMgw7ze81+r6xD+dLgGiuHdk\\\\\\\/tbm5SAT01tmNcyFZC4l8lHiXRFRwt7m2g65BPk2RoRhms0n+mRkSRQb3R3G4EVgONVBq70w5XQhmSvRF3ufYy3IfzNiGxB+55KoPXUZNXIzqDOuKwwh0szpoLR4OdE+FhocXCU6dYzauAqizfnIk\\\\\\\/aejgSfLmrtlXzNCeIzSr7mMLFBoDYrVPH6fUzUC5rSykfITigvI2fKSEjkYJcLfoBWl8O3CkN0Fm2xTE8saDPhJC80LIR5xlPoITLs7EAyqAAExn0k8KUxjaoaP6fy+mduiOUUrz19nl+3Q8hXrWAi6sI9ZO+K4plUu5dheJ9UmAQxAQ87NU9+Ov6qu+Ae6899j+bCLuZ743\\\\\\\/421guX8N1ZwIabc4IG8QeQhmiuVAMxSqgy3rVhNKW3o5KNeHEprbrXw\\\\\\\/GEZSLCED8BsMS+Sj7unTh27u+S\\\\\\\/nRqG3VYBV38Vcaqxjcmc5Ed9yaXwlRJ1ysbgydqeNhOa8jSaopDK7DnFH0JYevUKvlpBL\\\\\\\/1g0ECJt00hq9LA4PxB6x\\\\\\\/EW6jPmoRpDniH7JteTRje6ay09\\\\\\\/7GlptAk0FlrcAbghP0aPNI9rZjT5ODpJhd\\\\\\\/dD733bHxVS7u3bytAnE6Gx1VyeHdUdC86Q4ntasqQF+5NdRe0fPiNgWE4dUC+R\\\\\\\/nZnfHQg6RhcDBG68PIheXNv4s5nK1d+ii84Wf5IQpHndV7q5d7nBeYtUnOIWmN7Ugm0MROuxaeGu9VJmhfJKKH+XHxDD\\\\\\\/sYUaWMqYeDdnpmWDZ2Fk\\\\\\\/jfh6DYcCpwBXHy2NFo5clunx3uyRRLGTI9rW9ESDh7Dl\\\\\\\/hKQQgoE6EAuanQMZgyJf+J5limoZq0uev\\\\\\\/DWtVCCBcDtGcK7gi1fNVq+Dt0ksdmabdvi\\\\\\\/0UFYMBv6ZXdQtmGwWnTMtIOB2Nb+w28OO9PoXHAtGpzGei1TqZuo9467AenqARtjF72Tm2cdv0g+eEPXrLTTLAL8ahkfUKxlzc7NgNHF\\\\\\\/iKN9UUtT5Ye5cuAWJgrtmj8NNqRgb0r+LUoA70fBVDY73M2eIq57PQrvFZ77YOXafMGflYYLYAjyvutA1e\\\\\\\/zF4n0OWxAu8glkb1avIruEv5DUgDS9082LNHHlAMz9KmaYnfPUM9Jv67RvSkOSIxGJ60shTwI1pH3EN90H9JmolWhZGQE2sKrdHdTyytNWn3AUTvUeIwq8iQ\\\\\\\/RjRubQ79\\\\\\\/ki5g8E3lS05qu5sC\\\\\\\/rwopf2DDaAWeIeWlfNX9589XWXjCkPysOiSmTfxjiE6m5+y+R6cp472e5TBfxfGKO2aukEtzyIfZyhUJmLyZfAgII1pcdWllodaE913y9nmIHRxwN2h0X5a2Zh6N\\\\\\\/CJ9+whCf7NZY\\\\\\\/fTVEot2Iu8NhLegdpG4QRWujdMOZCFONcByzT+9ibMCqPkNZv2NbZy+Wcy2D4jMRpMfNw4cUV3qRVsHT7vT88Yin2UGuz5+R8a7oxUVvzF2rebTNiYjwbnOC+KOPMJFaTNMjvdy0kKpfvetUb8i4skZ8QjaF9RteMAVtXaP\\\\\\\/6+apXHAHStWhFQwRYfnyrxAERrm7mG9k7a86PQcg53\\\\\\\/ZaEjKguwYO4BlyA1+ot15Q2sZuwKTxaFm2A2xlzWLOvFOSgwjWvVBEd3+o1bMzlQPuJgh2scGm6D1DDTmb6S5ofssDb4A6R2bGD7MaIJnhHoRQkXIeEWWdqSAIlLni0SKaaX2ghse0fO\\\\\\\/AubUSOts6AB3g962RMWAJazbNFdXjan+Uxp1KYoN+TT7enpNwcc0TCLIkCJkkg5KC1SP3kVMS9BV6QywY6oG8+7jpqnTETD5wbM7KxeOtoYMH9Ff6hq26Oho4eZM+OGnQ1TRjwRTnphSs10PToGRFEbFm0W\\\\\\\/\\\\\\\/IcfQNrsWlkFGuaGM7SaTKbcJIUxAeQhld6ZDUHfytxPuKB0qnADtUvWT0Kj6KZB1+NEfCs1mhoo11u8b7nVvKQ+iMSb1cwr4V4B72rSQC8rhalJgDeGrGRuzvium8OOynY3xiLeiBgnmY\\\\\\\/fQSCYAjebnCIix6gOLpGJOq6fbAaAb\\\\\\\/SEM1lOJPk9LECuLHbbZ780qV4Xboij4e+1EVqJCm\\\\\\\/0mwN9OudKpPb29vMC8Dt1\\\\\\\/nMEx1w88veBGVU373N5bL2L9rdwy3MehhA753buV3wuhaJCKxCAdXNT5u8VOndvcZewFBS3e\\\\\\\/gwGxtKnCXU7CrvHy91lgReePdpJMAXXotwze3bKWx\\\\\\\/Qc9ijt7laJBmk+tneeYMjuOYMh3jg5KBqTa72OGeLTh6Ft74KvYk6H5ziEJw91+B2K56qJ1V+7clP5FhnVa\\\\\\\/VGgvr2dVxenIJhfIJJxJ52yFUX6hKbgDcvTNa7LC6VSxtSmqK5prE2g69Goal02Lu1pQrN2710O2pyZ5rGkRqliyAvo1ilA7IoED5IUNHTJYqXXTYBg23Y4KRRINNIYn+duUaxAqiXF05Zsp0hh9cJthvsTGu2UN08sJ4kYwYiHK1rGZO728tu9r+iCKFAIQnYk5kEnFcwfcrZrRv9lIFwnoV9dn2JxxSiLTGVgkUGnETgwGH\\\\\\\/9pxyz7tfAb1VNb\\\\\\\/YGnSGMnuyymUYggCG2XKGKvj+\\\\\\\/XyYuT2T8zojowBRScxPTrHxJrAwub4uIcNWye9CumG9DNjmCXvwX5iwzVEPI0JUlm6sUHiD0wLuEZvBEZOxNn72CVHHRj2Hrv2LshlWnEAoEwdfIKtWSMVYn1i+cvLt7T0lO8d0GuE6GBzQM63Jq8S03y9sKnK3\\\\\\\/VwObxGx38B9ZQXuzgSei4an2RxSErUMjhh1zDKz5bARkPKbja9zHdaSpatmOsRSm6mG1DAVu1HtzIZ7ka2zy6Xo74h95qTn0lQEcr7TbPUNMdpxfkC5fYxlVEW72EzTMoHwkqs2xWyoH7\\\\\\\/6P6unlL0UF+NCscR41xOKqhO6veVcWL081YiCxkWdGRkzuzTfrn1iWdrtYeQUufmUeZRpajX3hgmyVLqQIxErZ0QIj+Oq6pnVlA\\\\\\\/Ny96MSguMiykz0t38wfttl8sDhqh4F9+krdsmiHff5tISn0FdulQE6dDZ\\\\\\\/bfa5FOGFjBxIPlYOyvcoHECuzRAPHGL0gIb7VIAf7dzVGmD\\\\\\\/Pv4a+gbxVRiHG6zKEG9Q3DhxH95L3MtWFfcqwm4sA+6bpuCG3H6kbYnek7FLRKm9OmHiKVh+vIODgw9Wz\\\\\\\/MJe8iuFJnmI3jFTG+Xml6qrkyGcYtKxVUchck5ZlqYAKyQyiaak7ImRBWy95twPVB56vjuKnDg3fVzZQsEKXZ64u5vwCnwyL4lLLZyah\\\\\\\/dDEVS9HaxX7DZ7YLHQLDeeGdYK4+z9S8fv5SHPI1tEcr3dI1uLKB2v5wrgBxlwTOF7KEQhzenZjIl5rY+\\\\\\\/EyQZ69oEsPJmm6\\\\\\\/JuPBRWr6LHmVbVMSeL2pLZh\\\\\\\/oMg+QsyekbxIZDbD4Jg\\\\\\\/rpts4I7fLAhPOMaEj9MwMnDifvKdiLmGrCOFXXmv6a5vvO3crjE7mMlR1XHltAcXORKQadku3vXZP6Y9WqrO5GLkGTeCNEj3Xmc45q6ZVgP46Amn7pEEQde8VbJjQAxk+E0mCTvaL5tGE6PTz35rBJQBGRwW+FNyFOdUuvK95b1mqpCpbC\\\\\\\/F2stKSwfnUS0luzH5Gt2wrTSkCqE4cxFr0PbNrQwT9zbHLsm6MmJFXNDj\\\\\\\/LG2oJCDG0RFyfwnFSVsN6zcQKEkDKFv3X+ksyMGPqbLKFXZy2AyFiVAtz\\\\\\\/4cSSFA\\\\\\\/KkwqtagL3KhfiEHAmDM+QXjDnkmAlAAzg7ieqEw6CDsACTUUB+n\\\\\\\/SQtLH0+Rk+OwiDHf6IilrZ9aWq5mh6MaLHRJfrEJ3HJWH+RhmhVkgS+5yL0v\\\\\\\/g9\\\\\\\/TY8nwicWLQkHZZkbVCmmSoEQUYC8aJNrbqSKvWicp7HQoIPGiDlI7qnySQx\\\\\\\/Bsljw+Scfbjk8MDfg1sPa94DSGOxFgh1+nR7FAKKxCgYrNW6fhQCJVPFiFdpC2yspuTbrmzb5XZhxwPiQduuwrlAX4V+MqnD0GORNXmW5DbDyQQNFEz+QSB+tph64yZjj\\\\\\\/OgX\\\\\\\/lX3kn9P2rRurjRkHfOoiWEA045Q6GzUGZeK6+BXI4aKtyn2q8rIX8bJ8ssGn00LCkb9t8isqDUIoNZ1fngWZnybCszV8tCsKYelgQcZc+7gsywZLOYpB2Hy\\\\\\\/WKeEIf53lAxrcHNXxO83wpG67CfVqbH8\\\\\\\/w8eltFaJ07+nCXDS5TWC6M25W\\\\\\\/QnFrN0Opms6DyKOj6PQHu0Q4hLE688dvV8tNUjql0bCMH9qDAO9hrqc3x71+CWkZn4ONWV7fYyeHMfLAYegxo7lwN+r54xxubLbuh4mrOH8YF8Yp4mhJ01vp5rSdcWEcQH\\\\\\\/kEa8lThYJJ4+QGtRUZM6fVxwxVBFRmrX+WeqYay52q7JmAUBD\\\\\\\/RybxH4wwsPL6hxI\\\\\\\/gVPtatmJBdW+4b3\\\\\\\/eal\\\\\\\/8n6MwO7YkMIMYYZY7vgGq83lbxXfUyHnHsQzWqbuBlR9NRkzaLmwXQ546UyRtKDoUXMTNRjQap7rAOBkFXQ2lMqU03k4vKz+JBD2YPGlFtUrypbgCrj7B11jDW73sdznReI8T5HjChylpOtNLSHq\\\\\\\/YeBFLp1l8OEbnNz9eMk3a+GyWIfjk29\\\\\\\/TC28XSWuIgiqPUN1Q5yvDIZBXtXbfMycvpruoxoshuh2RT14IUB8bnx0F5xgUjzlpgKtyjAHfFvDjUTZNMEgIdyon+OiuETThHd18oDe2W3RXhFDIGjEbTUbXrYghlp+96oFGwps979ecEWaZXKKiZyETrQsEYfoOo1JkPkO2sVdzjYv3y2VuMU6oDalosNBNZ7uNXtZ4vtQxWL66t2+7a2fgOF92qk07gipCPgSbMFGwl47NPAa6iteZkKVklC2Dh1FI3iUmrRbltqHeiMN73ZTv5JTRtB4i8OeU6wE6eiMrGtqg20CKdlVNRxYErvTu\\\\\\\/Dzcw9D35a1GgcYKPLy4Us7aO4jqlTmdxu56PqPH4eqc9S1z\\\\\\\/o4JslTRXZOMQaAFpKTk1FR1+l5ltvdtiEGKrv1H1leUXgo7xVIRsvp3QXhoHHbdGYWNMXPrNLeOvnWvnyw\\\\\\\/72dOy9KG\\\\\\\/UoImjS784c9HssP+Xvq79B006JZleUGRQSlCPPnYgXwz3JOYSkC44b3u5wj4SCaVMP+t1\\\\\\\/8xJTumX6LHXtABwg2RKYoe64YxrUUzBNlkOOj6EUKbQjWwLe2fy4LWzaUvXggaaXqtJYiHlexzZRMY3ptmoeMXYyDHUsifNXhiKYUyfxZXd6H2FRZ6fFCuVE6BmDwBLzLbjW75sbavkCQVsstJtfYLJgij2ux\\\\\\\/tklUn3GJgUkiJfg+J\\\\\\\/xECZwdvBdDOaw2ph7VIIQ+zyugv4MKQ\\\\\\\/yRhlb\\\\\\\/jCBB3KOnBHPPP6\\\\\\\/Bvf1OBF1LZC3u1qY+\\\\\\\/I9Mpfg0gl6quLRtCNY1sAOOnc7aBsnmhojs1uwJIGdRF9y1I6J\\\\\\\/2EJPTzt00VVGNKmsgGZsNiiD+\\\\\\\/IOXCOKsX0jsYmZIy97hQ1f3sWS1t31+7UyVE3VxVN7WKYGvn07UttblkOQBtvrSxb0t94KwjGYt9ddjY5zvLB6LxI1d8STIBQi7LordHqIMA5ExpTdI1HfsgcWZVmkkcD4zZfApF9fVwq73cNu+tFQG2P00Z5IPL1MwTwd7D8Z74a\\\\\\\/ideOlujQHNodQZxw2fhclBml6+IpWkb5bT0zZn1p47ckIQ\\\\\\\/R7pIH1vT4oEOIxDNiZsTGCOOmtLsiZ9KEc0TcqqLTrWVK+ep8bc0z2uGgNsIO4A97cHyi+4jUTjecquKfe3oFVKS4U3a3vBw8tWx6A37KZm2+SAc6ZzijU6msXVV3cNyzHQtdxpakRD7jlEtF8tp4Tv7aejXSJYh966Mi2+mGOj6m9IetY8y1lbb3Cz0KWUDUiAptq2I9U1eRcIihDvic6lPpHa+Jgyp0mEMsn5r+L0Pevo4O7ql2dqGHhPSaja31IgJqY\\\\\\\/CNVEgfKDiuhh4yeeFQwi5xOam82FixLchjpkFQK5mCqf\\\\\\\/cPVarr\\\\\\\/HGgiKg22zi9ISJ4\\\\\\\/0NHryJ3sgP0BN\\\\\\\/Gd38mWUETfgyaLo4nqoyyiG7f5oCsLdpMO+SxbKEWuDecqRWRAeP4tylwD5u1N9HStP1zd75B8d12NPN3GJHWSKXQEEDbLqQ\\\\\\\/TL819n9bCV+CrABOZ\\\\\\\/\\\\\\\/10l4TTcZTKa9iOugyuLzy23hpDMIJCmBlC4teasC1u340TLnzP6uejDT8uFsPxAJ3ep7qNnZOC8pc7D\\\\\\\/bsLsxMs4OarReHzQrfzb7GtnXAkMFYnM+SLHzRTPQHRftUBU0xC62KalMAMhwaKig6byePF\\\\\\\/\\\\\\\/mohLu2EL23dzXxz6JuZS0esL8DCZgOjWX\\\\\\\/dTwIksixduTG5fZ8\\\\\\\/kL\\\\\\\/6pq3FQ0ueCdcRsHdoEiW4LuPn5NO8RUD0juKGzATONkFLs6pBk6ZIPiVrDVpn+xbTsjr13R0EccEmuNDVErT\\\\\\\/9nc4pO249xirKuOx6Rkg5i3LesQPXkl48st8pjVN588LifVdTMTjnATeJo2cmdHio3EoXn34QzKLi7GHdKtvKof\\\\\\\/nGBcpzCRXgSly\\\\\\\/6rXg7SfzoU6zAvM80jXnteks54uEcQ3RY0Nj6Xw6FxHRcXI8EtPvLubr4AA2U7f3ZZTWlK9+jIbrznbqXmuyqzkanDAUFeDY4wUj41Lw9lFGc5q0\\\\\\\/2McLXZ1MuUk3u5\\\\\\\/HuuB0A5CDRK4ltwggtNpPgCwUZU+vJOaLXyGClpKG6jIXOkCQdvIN98FfgUN1jm0wRgyUVmcyVqWZ0mlKpQI+PuqwTlOUfdyZzKy9ACb2ErF6H\\\\\\\/aS2YeIDCD7Rz0aQ0cLSrVoirELOUDaTJrCLXmFncr\\\\\\\/vjP45XIDUikHMSXeOEp42hlYvRzU77gn926Now62IrfyYa7Foxyasx5hEqKuMxHHtkPEJVwBvMq2Asz0CRvCJ2HIvijHhtsg67EybN4UwJBpiIqt2LjZNFRIUPhWakUR0Z2ZO14dcNTyzliSFHSERUnpJ+D6xzRfzzRJYEto2j00UQAgliTybqCk5Q\\\\\\\/yQRGWAan3FXK9Od0mdauXZExZw3olWuGD\\\\\\\/uO+7bDgqZUu0xXi6GUxvdx5a+xJH4qX1jocm604bPMkEmqEBqhF95qDI3Qys1HI2e30BMEbLodSH4oEmBCPkjAQrYaoGKxCJT\\\\\\\/VSPBX7FT2hIqWVePkUZANxodOQ+O2MT9c+WxawaqrPbCnju4AB\\\\\\\/Usutl80GtXFoiwZe9hIBGJaNI82Q7CTJlDNvBIVaElwIjk1MEEvZ19wLyZOGjy8sPAOzO5ggyzsvCXHyXktNffGATwnaAzvc9w8VRnlV8wIugJc78A1GirjTfaNPsr3iq7gFg7CG8P2LfH3xnsgRZ4i00iLd84HRDfKAPMVeRtnPKrV\\\\\\\/pPPs+\\\\\\\/QGUetMOjBe3xxjg3ZkOWua\\\\\\\/5XmUx6Yzu1eQ762kGakgE3NkHBX2YQWf48mJXmnKhGqk+aWGEXGDhrQEtV76RZGjA7kqWpZ\\\\\\\/Tso0hvVu1M0jDvhF08YkHCyRcIYC2Y045sdFW\\\\\\\/ApWfI3zmCSAY1dKXl59t65QdDrmbxfEVqMWil8gYQpSoPEc50dHdKocRH+g4Hdet9V2plw6\\\\\\\/pAhHh20uPOJBgZ4Pl0bcMTqEcRCxYhA\\\\\\\/ASd+Gvn\\\\\\\/xlhPgO9jMDnkwp0tr2B3OZzh4fRbOrLAOsofTBIJ34adsthq\\\\\\\/TVoCVjZy4DA9mv4MtR1kUZsQjhFfXw2L7bmB0MbyYfa5Z2CzcoU3xzJVyZ\\\\\\\/3OQB84xlgEY\\\\\\\/LL3dIGe91aofoAlMtxlZ\\\\\\\/W3UKpobpNtcFoj1rh7JOinynyLrw6GrlwhrhJg0QlNh8CSd5S5YDXEv1ib9OtniFISaN2n5AWFAMp7U8bmEWhmXFT0QtDdjY0AYZbpdJpWXyXeJJFaYdRp+hB+COCDlgITJpdckP1UCPrhOOe2Rsi+gdoYvnKAPdZQn6AvM8A1PHvlzHzJYvEdaTKz+HXNzDQ37JcYIvMnMUhrI0e0b3sNndO0zj3vR8tODR0oKePa9PysQb3Am9qVAK79ATvYYFTtmi5U\\\\\\\/AcfIrqdys597xmI6tt2IsU8PBmO2RsNNsV9x0yCTNbNwkTwy6OMiQ1eT\\\\\\\/TSO07104zoCHimf\\\\\\\/fV77JMSqOHxHv6rbWAfBln6k5u5QIAkd8ULru501mFBkQCmD5Ll5S0W7LTOaVzv8nWkytinEfW7dQISXQejxMbxcYWcJ7YwTqWEksLSwgfWFgQS02OXG53S7hxurJGdi1orQ9nmP27az1IrnKFKAV6npcTy1l7dvIvJCdgrjvpDZab6g0V1AbHW9JVJGye6\\\\\\\/\\\\\\\/ETYloUEeKhvGHQ2P+8rESvPnvJ8fBqmBMUWE9Nxb9VDyAcbh1QlEu5baIxrTdSuhtTq0ei8Gr0+18Y45JZyAoUEC4qveQdYAW6+1tZn+kPzBjrj1O6Vs0ttka6RAqvHErf4KYh\\\\\\\/95ADbchMSkR8gzw1h6JJCIWZ+dHRFG1oSxPT\\\\\\\/bdE7DWOOPu0JdsgLFxs8YSXDw5ouw56zIjQG8ZT3fJOIMi6WENAEeg+FCqI3hl9JftDaQ93Qg9vLQgxcoTY8L2kRpkXulJa74nekkAPL6cU4sAYe30DeonPmIMfYENOiUasBj2SGZOwtgmYoNEd0MMPsMhhQ7JbZbI5ntjXCIOsZwx9Rn8PNv+TRPAoHBr08elxOmehdbcuv3cv1ABdc+ZTaw74F8in0cFo\\\\\\\/nTqmZpzFdTwaAvEGY8fTOzv9oK7ng7O0YkyBTeKu+ubE26Isy32dI7enUG8DMoobKOCF8O7PelnwAgK9MuGuM02xFVcgZljBUj\\\\\\\/Afk39yW8D2qfYZVQiodolcfNMuaVxwOtFWyBuknx9VeqGqYaoRlDZciD0KB6MCJdtKvNIvanmGvCIf8Da0kKA6SyU9zGdUr++A3kCNP6Vpt1VJAQFNhYB0Cjsu38WIINnTlisCDr8tM1PhdCx5e2iGuvyD5g0mfi5PRfqDHQLEZ2p\\\\\\\/h78G6qTK9mm+FeCgjfDfWxDC+wEmRLGV6bWAHU0lWIyWBJwxOmtEPNiOfxkcjsIru0UAcH7\\\\\\\/en2BTa40s\\\\\\\/igd35r4DuQ0RgqVRydwCN7MlS\\\\\\\/iObx5cMVfkqfSW7RKVGan+ndCMu127lMFcv5o6xjssEKxsuBc3c5kfeMV8SbpSU\\\\\\\/dEpaHAq1kdNQQQJUKGsC9+VzmMu4kunJkI8FRJqdAP9TWulc7TBIRD\\\\\\\/K29MkJ6dJCVkw+BtV76G8A+WhFB2NbYl8GOPKOIGhuvJ0alzyzMurJeWLdJZvsMCEqC7pPUfV61QACQBo27Kxp1ZbWjCB0SlekozMjwyVKPL6Ar7R38Qpz3Bs3iXf\\\\\\\/U9jpBXpWhwbNF5iivfe6WP6k6iqefKHnC7aCuTODXZmQrZYMihz\\\\\\\/gTLg+obrYwSEDJVY0dTFw3UsnITVd18OL9wtEcnWhZs5Nw0J1feVDrlttsRCguEO+1NkoAYydldiFVKsi4u8MuOlVE+49xTzagVYUn0Sj9X09Q10bKFFU7X8XCSD+ujSFFCfgYXXgF3v9OI8w543FCmKzy1AS3OyG3p5a0EDfv7K6PWM9+Z10Q7nrTEArufHCt0fS+7QrLcxoX0pXBgpV4UziqmEK5Vk9Nx3vmYePmvX1tIl2RxDnOiTw\\\\\\\/1JDV9Nf8N1SaedKRqCSbevYM1MQ7aPIWtFSZoFKWZc3pHgqjKmQXD+Fgm9x+xsYprlVCscK9Z3EuYXw3zWuhV\\\\\\\/T2zuQC4LKqccN3XqXU+nvPsYzJzzRmjJ3gh3hopmIS35Jj2wkIhZ8ku9U\\\\\\\/F0lhYmKK2VgLIVLaUZbbN4T8k7WsK3cVMlnwVQA7JvhJeLY37hC9Goq6M0OWUMvGmrvbbGqaFNV35P6xRBquBs0fqvyoIfMf0lReEg51Jmpfq464bogZlsSCuL2Fgns2DHZ0naCU42atk0Q6NdFNVc1QV6L7fhaR16cMdQYlHy2v0DgVhFOqjyaAw6OmyztSdJHi8LWtClakIL8vs5v8NVna9yKpw59Kj2zdk+1iN8ajfx8KzOzKEausQ0Hj66zpUengVLUeF7HxPq6eqab+YK6aL5X7X3DwwSSxTxz2xEIzIuuBfVy3Ue9iurU\\\\\\\/ykUsS\\\\\\\/Wac+QFws4rtUGJqGeo3xNWNephLF7qiTv4itwhb\\\\\\\/JhmHGKsumCo65Zs6a1ufmbrH5AFF23+ZslunlK0AqUnAGgIP9Ek4mFhizwdGq88GunVHwv4Y2FslyEsGPN9t1aEaZ4slMh4MtnYWeZkp+brZPRvo6wqPrOf7InVrC2OkzhmxGFQtFogMTiTK9DhB0unLpa3+KVEDBzsXhlLXx2+RQ928l9ierV1m6YWIq5\\\\\\\/Hc7G3IP94fbdNFQ1r5tpWldSLu\\\\\\\/djy+wliTNm7CMybonNksHvcahfnVsbT0qQlIqDCU8JsSZ\\\\\\\/b6zMESo9giy0SHnhGoan8gaqjgqs1KnLo7Wm2i+wjFFxiLGbkQuSx7f25f6S0o45H4WLdftktNBpuO2MkIQ0nXzkaOzozvrq7vrw35FSzfABz5KAOHvcF2abq88Pu5oDEL18QfI+gXNIlV7s\\\\\\\/3R7zqUlw20ZQKxHD9973okuaGsqMYVy13Op0WewP+eCah8vHK30\\\\\\\/4HZ1u\\\\\\\/4BLf7sgvKHgF30mKywUxJnf\\\\\\\/QLZVYYANUl3luCJbszDoIy4yyjxl5SfPQRzWMwUvDgb6r27ek+dBt657wuDtRcufLKPGLphyKRQAXZmUbdcqfox0I3rz3mybs6pSV7ILPY509pOsBKTtDnkAlIAxsoDqw+K3\\\\\\\/Bs0B94vi\\\\\\\/d\\\\\\\/M2A2wwWEhiYw5fzx6tF+Cx\\\\\\\/FxtPdIr+e9vZmFm\\\\\\\/J6uVoZJoAJsHeYLVvnLOjGsKlefUVgUwPEKEBBMYzvuzhcdxTD9gdgXlioCY\\\\\\\/N1xyl36QEH1CwSSmmax0o7jWA3tvB4+qFsGPYBMzvZl+Qq6giakqtleVnwG+xX6bA4avnKQ4qDbmpZR\\\\\\\/kOUH03StPgt3y1FFgUvH7BcFG04oSggLzHvT4q38xo3XlTlYz0\\\\\\\/k0oYPI0z32zo3sYR8cbxtCJDhQoBKvTJz7ZZG5N4gX8iy3WliQ0y+l\\\\\\\/+7sBZkB6lvzKWQg9VdAVLBDdcepatx4ehgo+DEAloCARBIvx6C6CHh4H9mJVvUIhOFc7N6N45MLxMOvLJIwv7Ds46b050NTqXqYlLDDbSQIWToD7whxjgUOOTrWjRjy0j6JLYaeqHhsGYIjCiI9KrLSAkns\\\\\\\/zFGm6LaP1itTi3JivXwGvwBjmhzrgnYsqNTw3W2XEYDZcnkV71WQrZHvA2QpUX86XHHNMdRj6NgVA5VCXalXgQnnrTvCrOi\\\\\\\/U6sUxfg8KgWg4EPQaK20s3+Dd8YolIBXyPAY\\\\\\\/PSq\\\\\\\/LQ0sLZJHQvDHuO8\\\\\\\/Khjjcn0mz7MgaHegGq0ZiHdyZioEioJGs\\\\\\\/OKZZIw8QVhTr3DTzXZxJs1HLys6LT1HG763zxuYpsqbrQqHzhpxe7UaDOqWBK0sYG0rFHwgEdZLUIyZ6NQ3vzOdwalJIfIapkMi4X2Z8d7vb6ExTplt1bJ8SRClvyIEpXKeau+H55\\\\\\\/KpybHUljFo8kaQpbyt+ERMm1sQVFfjCVqX7QXn5z\\\\\\\/Ux\\\\\\\/0S656J7RVySkuKPEYKbIPN5taNLRNYBFg7xIZgDZtbI3WRV32knXrysU3rry6m0MCr1dp\\\\\\\/3KMP2NP4VCfVp1acerzAF1OhWyWSezASH96TzV9CGyYgWovJ21k997qi5F5hz9D7kHdbGqFFpiQk+KmaIHPVPMw0pE5t2gyYe3FPIv7GMh\\\\\\\/NU2sC62ArPDqI1ZhEJx3xTwD8T8qSUCtnYS+RPaAz8CsGe+peP2LIaV4CY56E8hUeWkB59nJsMr4M+2rcsA8hEMAeZEgG6zi1Da\\\\\\\/3HjQqMYwAjnykZNfpFVNMAdT\\\\\\\/UclWrXhpI4W29tZaVCwe8EBxXo57IWlHmfgFx8rapbg8IMqFhcQaB3FSrGgnerQK\\\\\\\/tRRJktkQO\\\\\\\/9NthfL\\\\\\\/952WUX+ZndFi5Fjsj0Ecr9\\\\\\\/nZ7cbH1W\\\\\\\/pZMXOVtV5Gv30yXri21r\\\\\\\/TY1+ve1GAweWKge0m0L5aVKrBl2WzUDkowJVLBT0WjtwPJJt7KBPfLGORDnByrXI0ACjhYj9ykZQXz7CfXENtnikGbMVcGP7KTC6vYEsCNP5rHMbRrBxmgmy3p3tUQdjaOcDEsKe9r9BzxpN5IgSNjwjVxBxA6LP6GoM8LKHetDhuvlDwPmyoSUNtzU0OFOURP3bZbj498bPSJ6y0hdnPPhZt\\\\\\\/hwi3WflYOhKQ6lSGz4kuhQ8QZfW9rRJ1im5nzuNjJoUummubgxUFYCN4+VwuAdmIXP3ysprgPsmEmcaspbV6VZA6ZiLrZfL+YSLu98iAZJ6rRO1X5tDvDx0Dms8bArZ9YVvzGh6gm8F\\\\\\\/SP\\\\\\\/Ue54oJv5x3vb8jnjFG9Wi5aLLHDglM5LlQu1Mu74XibSBwf0l7eDB5fnK8wRYCtAbucuqfj2ZaNMZrvNtL6TXDH+l2619pO0QTXbNY8wE5HZcc0SONcKsX3gYs5SKqTs0aR28iFHvABcCsLn+t3ESy4yX4Hc5xSOq6HIYNcP7I3ak3ey0TiikT+b+sNuTHk\\\\\\\/jfJ2\\\\\\\/snu3fpCf46JD\\\\\\\/EKT4CNOtebWcd2oOUYGd2A24tdQO3qSgeerPeGFi6k1RMBJxfo3WSUg\\\\\\\/6\\\\\\\/BzGHyhtO3g021fz9mehqa77yxfif79xAI3nrozsOEdieD5\\\\\\\/VPNVS70Muj25D2n3WWjHh6JNg8DdCb3jcNNQHso+cI+q\\\\\\\/jLtfrpvgoaJCtKu63Jtr+kGqf21V91mrbtXcl\\\\\\\/Na3iDbi3tI\\\\\\\/Osw\\\\\\\/\\\\\\\/WZ8DHQ0cJoysV0t6CeWQUykbR+36nAQr6fyWe5joJpybpGr6Vt+vtj3SeT\\\\\\\/IjpwRgT6kxkEqWl76sibLYXD9+Nxd0ISBFihkjO+EL90E\\\\\\\/qTGdHr94XpS5fAvpsolEHBSPrOshNwm3UIKMHbdWVKmehJNBAFvh3n3et89RU2XmztEIbQ4jE4MLDYLGtlRLgpTjWMg5prBsPZXKh9L+DLasxktig90UQzWM61K7UKdO00yzr+mCJD+RNwUc7mmzSa8ASc4RSwAyV3bcw2\\\\\\\/htycnabU1TkzBPhk3Ur7J6XLhf\\\\\\\/VnyetyALZhUhUvY1qBs1xJJsVRs8\\\\\\\/nUucd20t0jMwvuy0xY6tQxC96pLJBU2VRkalfGqS+71zPL4beBH\\\\\\\/pRzBK5CexkbQ12jUOs02\\\\\\\/ooHCVtVlDKoKctpGGQJlVbz8ChxUA3vrYtGc\\\\\\\/upA65QaMElJmewtaHXMMR5b1oeLmV8CB3d0MC7b+c43BLrSBL7ZJiQf\\\\\\\/4aRqqBCq5GInnppfbQuUTZXCKh2ad8QlLCoeamfPtj+cua2cDl3MkcC2S\\\\\\\/pzKWdWzUNLFT5Bpo9pg8on7XWlHov2BHYj90tpnKsNF1ncAsOMjWWFR+fb1S7MJB6poGOKswJapMmCwe5T9T1MOvaXLWX21bT5HWuM1myeM90NivFUdDfsfzb0a1vOKY7tVpvbnSFlmIAMBt4DOmlmM91wMQnOIFE7\\\\\\\/qqyEX4BF1NZ4xZqTGCDD9yz\\\\\\\/qn\\\\\\\/wbLCmICMQJ52SJplsylgHsg2n6V+vAbTVBVbx3Me4\\\\\\\/M10sRXGBrI4vendfI37HCtzA8bwYGGHe6KWt+uCYAT6ZOcn8Kl2Uu0+dEKMs7Uc15uLI2b3IXu7sDCr7iUoF6gaUYZBDhWDfbxWpC8\\\\\\\/B5dIE\\\\\\\/cpkOQ6NrlTVLwSzVJ3pa\\\\\\\/8prz1sP5WKj7mrfCLsvxv\\\\\\\/LXnNoegjHpXokPBTGtgmXAkcVNUjhDN04CjCYpvdxNzJh5xdqd15r6wj0QtfqE\\\\\\\/ov+J7LZAjOeo6xLtrKjnJhUd9T3Kxi\\\\\\\/TCaw2nQ9XGLe+ZC1GXGB\\\\\\\/olmubDSlB+Ei+SKro7+yLPKFGUWSPkWXYRxMoJeRBwn8JolRPB2l7\\\\\\\/bOOeV\\\\\\\/E2RXWs4Q2yf49PvfZO0RPOry5UDxr17ueX4ieZ9O3wcxuD05+nJe9dohGhH7XmQeTPeEBKD6Lm2T46V85EPrX02lQiLLt1h6XyiseO+qWj6VcD683xdNLAq1FuKwkVuDicPosLWojDuDEJYLMgEvirvZXVouV1vGmWMSPEMddPhXhhPYpLB\\\\\\\/w2m9ypCbl+XL5\\\\\\\/isEtU2QTTiQuT6RferNXCYTzeHIlWJs2Xf1CyaCAg6Be3gJghcmZJ1Dla75GDx4ROOiAma\\\\\\\/ymqLZZSQkvAnCMOvUinBIyRrUdc0uYRwpUD8KEY6u8fwGvdMkJ7VEqFjC1cJctByibkMpv6azLNc+zDEGm1I7E+TGr01YdXrGgGArSZYSdV4VHwtef3YCHU8N+DXUCFjgavnGEZt28rMBAABCNCJeWyIoZlPU6FaGjVxLAPGKEBlPI0X7han8QHcbfrdMmdgLUFUEBdpO9SXUTUagdlNAh6mh0R6tIhqZTacZ7Gbs7fGA8H\\\\\\\/Zu2D0lH519B3tGSUX42EhjhvnB7wNX\\\\\\\/4\\\\\\\/sHJETCzlGpRxEn82NDtRaMR50KyrIHkFIfRiZE0\\\\\\\/YZ9+ZoLcEQOkVuJZXACS\\\\\\\/PTlisdOGYKCdBPEwvKJSYo11UIR9TNyFfbKNxU\\\\\\\/Dcay\\\\\\\/uyCaaRNuCKGtOyffnLSzOfs1Fxgyq3TAZFEptK8S7b9+3BXnyEBZOtNfailuR5SpSUsKsDYCqGFrIveH11b8ty5evM1mjxWKOm5bZC5yYV0Oa+rjS7pX+8\\\\\\\/zCkeGg90XuJ48cVQorKRiirx9g8sHqHY71coLcLSNObx6Br1pOmFpIrAOqe6b67Qznzt0RVpdOvdm9H\\\\\\\/0F2T\\\\\\\/2iKROJXHdwt6qoY0olW0ukg8\\\\\\\/RS6JjhfChIzN5xfRuhJGm3gCohBMzxIw6MHXT88Mdb5HZ0NjVhuMthb9pRgl3Tck3gFOQGylw54bjjga2gfK8Y18DTRFEjbTnzR16V5WElMNxMs0SGd6ufrWS45LUfi\\\\\\\/DV1KokVW81215xF\\\\\\\/8hYP2vVem7N5R0Xp\\\\\\\/Yn0Fxabmstzne+8BWvXn94wG6fw3nsQISkBM75QWJiRzdSohyjPkvcEBS\\\\\\\/mFbmlrZ7r8yyRbzl1Yv7BHsklVc+R4HGKNc3vmS54\\\\\\\/ZYSmzoJlMe01gd4KyICHya1ziWH1OqqyGcTQf6dkBFNMEr6V+QwVuHej92yzKs\\\\\\\/uDV1ddRBPCANXPV2nB8Tbo7ES0RbBSOnmzxAWFcATTrj2AIdYsVYppPJe2y6mmOG8JQyXXl9zBOXwlkieJpsrzPXhDgGLCx\\\\\\\/7LtB2lfyxQxXnQOLDNZ2PscMn7QwZ3RXDj1SaZNrwJxsSuG8h9WTvTPo3u81Nhec7qYxoL34Z3dUioiXZWhDpJk9qZX1RBL6wPy5bQpbnW\\\\\\\/9TqfsNHT3CgtQgQmQpAdSONfOQ4G2VT5PfqL7XvWjJh6R\\\\\\\/AMweXWWVsOmm0TwBNYdltM+L1ggB8FzYLVFQVqWdTNiMO40NE9xspFTBl1QvHfxTROLaDZxETACAec0VpqfaVPMx0M4s13VGdbS4fm8gvpVbw1RJxRBD2vEBB+Xh6hgRXGHdF0C7ihJvGncoV30bwer+vjrG7MXOPWYfOqVVvaaKAjSNd9nxI+EtzFy7Z+bhwJDGylAgVxLLRRYAUy6Je4\\\\\\\/FDD3QrGyGtGZnLMT4rPJLXkyl7kP3f9dnNEB\\\\\\\/2\\\\\\\/ADhlOz8ewo81ZD8gKRlfMmt58JkOVRH6rVqghx0iHS7714JPt0ntGMN\\\\\\\/qw59G05tbjbYakivVxUAxR+eiLdpgGfAbpzsfbFRtJr1qtwEAlaPdWpAa1HAD9D5QNopAOyBf0Zhiviwlv0TjnmRax5xRRq0XQfo5ziLSu51WhGzYpZpEpWd4PX5J7iWYAwgEkHiGEsfqmfMPBOV3oG0SebQnIZP84tFtcQJZ0DYGGW+9AcvYLLbhA9zrOfX6aOB+KAI2VFmeFmVblK3n+NK8nyp0pA1JDxieIxd2M8IKGYfZ2QL4hVPc84zddLiWXPrKGrMiUpjntgMxw9q7AyZrcxycAAh53OkJkWWGgm\\\\\\\/tc\\\\\\\/sNcij7iV5hrrCuQU8xAfkNiecmbQ4h8MSKSTaQBXQ0dr0kNZjHY9f1STJJm3yL57Nvi1w9AQrocAUaZ6Z7s1iZYMUrwINPsekyYO0NWU6JgwTn2Dug+FNG8\\\\\\\/F1kCv4XNCfWSK3GAyuJ2vO94P69MWctKcGja3gsX1D5NSM76xwsO8kfnIbod\\\\\\\/xv0jmHrWifYZZMQeyLhhdXL5E+Ja7eW1TWZ1JC8phq\\\\\\\/gon++LpCqGQ25pjDo4Jcq5dRBLjnKtgObmP0d04LiofcplvRdbmhTmTLArq1RlHLvDOxQPu6sC3LibK1KoLjjrnRGE355gxrjC96iyIx0wc\\\\\\\/XckgputgJmx4OeBCgBoQLl5eXoGY7nTy5HCvZAuRVFVWPeyFpZcak8mYQLs7DQBMuG4R4aa+RNlVgAgZB0YiIxMnWGmTLCXz\\\\\\\/yqf4DY5JeHPFpd3BTgg94BqMevbugY8KtM8HNslb0au3g3apJH2XDM8KOQ6sjE8QGi7xJQWeB++k\\\\\\\/5ASQWiC3ai9OVKcRQ6fILprwvKb4QyQAKwb0+WIm6v2Nh0A1BWOlmAAuC4IVlGsXEJzLbJOtm2Kikxm662DeCsTCQsoAOwdxSLwOUxTUDqeOL1K0UfKJfCYvLuc0ye8K2Vw9fQSXgWUACKqZGbMnoGn2BrVwaCp\\\\\\\/Bmv3Eztq9opE2hPK279Q6fu0Csj8Hl5J5ENN+0fmUGGutdH\\\\\\\/ShLsqbJm6O+XcBCkdrPyH1L2Bkekkao+GR71DoUHVJjc8F42SAc0KENu\\\\\\\/eCazxGylzSopZlJK+Uv4SQLiSD86h47d1jPQ0ubv9B\\\\\\\/1vgitkfnnFUTm8zsKkrUUvYiFMx5RJjd6lBLsQOalBLs1lkCx9DK6RDM5FoCN5EEubDV0nnfYgjhpiY18eZLmSbDaxLn1w1id2GkkcJYoLpCP6rCRnJ39H9sBFWPkM7xCvVNN2WBZch3e6xNJj8vhgbIb0FkanuuVglAgKFl6zURavLU0eMI8\\\\\\\/H6btr0Qsq0sTg\\\\\\\/x2U0\\\\\\\/2PvmJmraIsZZR3mhxQ4Hlrfaye5KFWhzuLZRPcF8i7v6eDD2\\\\\\\/QqNPSVohkU6Xr9WsAAScrmaXQYe1WChCQQkQ0en9WhHv5FasDsWDi8xzemmtt31C4N4na\\\\\\\/E+s1gP93jsiaN4i9LMp9MJ\\\\\\\/LMPkOob8aMvgHhduCWYyYbQIVZZw6cRc5G1kZexDa\\\\\\\/0XHkOqqJndOftZDAjAjY5oik9lWh1J5AI7ZnNqHUls2tGybtgdRV2vvqf0JPFoW4Ct3SzdQ7L2XNtvqabZpdQgLtmlrEUQU7JfDL2I0EMpq\\\\\\\/q7pxRoQdeDTBO70fZlmGfy0BHQDcM\\\\\\\/4aSeiJDZtOiAHwQt8s9AAFrHlSuXRLtVBQAwahLlpJTlb7gEPNiCI0CxULp3p+wB5PzbA0XbDAh1InAWywAnMnng+MYHPFBPbMSecYvddKadG5UkpUr+ghWeLI1qPaYCCaFudswGYEWuM2g\\\\\\\/R7PZUwXF+T\\\\\\\/AfS5tU1pFxlhaIqT9Hs4G63Bp8vOyslem9ym8FSrXzN7ctkJsy2JzGAnvVkeFgt\\\\\\\/c7UZ7sShUBxAZpO4HDRcnQJLzVQ9IVudW5Hxv38anGbxTLu7XT52AMYyky2qgzNPR4vm70YPKjlONtwHcXLjINH3akAMm0jyb2tO7rEpcj2J3r9Hu6taVyvbMW5TXdl\\\\\\\/cpwvs38RgCuN\\\\\\\/+hkHP+kYK56sMt\\\\\\\/C+KH8tUK7PWPCmZcCIihDPgoK2iUq37dZvrywt6QVoB4lEfWA8plE1VRTmhQxyZ\\\\\\\/f\\\\\\\/01BaRGtvjejoHV\\\\\\\/uddKik9DLkOtuV2FTYFxbft5pbACS0vBakDLPDa06GQ5r7hOz3aAeYvQZDhBRxGaWhrC2\\\\\\\/ZlRuhf8j3YK1tOTxOr67o2TDqKalyiUtB7XBS1gWLAFXG6DfUQXW3JpqEgKul9AwQHexrzoVLeGKY1QhFseR0gy0kUofhomziIwTzkE8qepKdozSV3uESY4y5V+GtrcOpIGcm0eWBrk6tUh9jvdNf70ABYThXsvKKaogGhIyk\\\\\\\/qApob4tc0mZLCxepusftKfWr8ncphubRDRyWe0k\\\\\\\/GUuuNxBUKsgzPMy8oG31KcZYddruFYslDyQT++Hbx\\\\\\\/mqhLzx1DdVl\\\\\\\/QNH8vsA8cJzDfGLiJccOUbNnYfWqiusG70eUSoKObS3iaNOX4MPZFehnf5KKZBdbjsa6iPAOHxTKrGeCo58ewoO4qKLa6WIAKcG9gBseeucrFimKlz5twjhuv2PUvQU1VsVBMi4WAGQBtFTNcUPBFi7H6NwKY\\\\\\\/GKdw1Y4bLrjbOzFdY0qsribi2+2HLQ0x3KYq29JAwuN8ZlAfOwfEL2JedBBg439DvHGJTEMBAcGsolYxpp2YrKa0msVZ3ASoB36OTiqskBsHD9fDmV8okKRlOf2dIYs0vCktN\\\\\\\/KArCbS43gRcnKAsEatkZ6MBPfIQLQw2uRVokLdwGu0yIrt1VwrNTtE16AQoTGqQ716+HXjUtiYxlJRznlK70wkCuIfs6W+V+Dr5UGAJl2GVUFXczI62zu9MrXD1iJKidDU2bxyYOQVNQ3PvG2cHl3UdaslYx1IwyL1rINGXQKrukeQHyTYK5L8AiRJBOM43zyRYFi44wWIcAL3ZjUDwgYbxgQd3SjwIKKqkiE5R7HlBETWWo2HQJ6\\\\\\\/BbmzQ5YNnMZ6mvD1YEVuERtM1WRdXc1e2sYFZtk5oWKLsoeeY2VHrz+uiUIJwtDUe07GWdpua71sOXaFE\\\\\\\/QWLDTMwti2SPyl\\\\\\\/u6EBTzCVUWX2sAIL2yxSbNXHfU2BUptmd8M82p961DxPJi5cWeHeTXil2uOTf+5SD9A9mzv6vQ8oDInhXXsej334EuSFCFAKZnZgyotdMCT0c8RETEGjI712xXEtzwXKo60ORcrSxWe+oIywreREPlWnZi0XgTuLM0p2hMoz5uYVzOxKndNBvrelx223RP6NWMhkw7sybhXoqOBNe1LnYib\\\\\\\/ZHqItwb2HkvuUFJxrtoVHEMO6nXbimkj8c72LQa\\\\\\\/iEZ\\\\\\\/WCSV7ApA\\\\\\\/3HdiW3WbzqDQYbN+Q7ZVHlEwVUq93L5hO9s1z1ekX0f\\\\\\\/UT7gt5JomkZ4I\\\\\\\/JqywypfajwBvRayMVjL9PDYpb4CHv1SCpU3cGAMhExc8hsFvp+qRGLlSoVvY1IgmbRg01Accn3TBqVqvOMGRIT35iv8ncBFPzSiaeUaRlCVkw9QVQ+TxYqWLWEv4GF0Kp8ckTi+YmZChXaxGebvMRD4G9+8cIskS0Dsb\\\\\\\/C8WXSGWUBpfpJ0c5wrO9c8PSkMDPkWfqI25qDzESID9kfN06mJLHKD9g3ORf+nC3hnuS6cEIoiU4Z5OkH9vzVgBujJtGSgLvyC00YDleZwAC+yS0ZXl\\\\\\\/\\\\\\\/aXCTuO83Xg+DDUskmrUG36+6cm1zaTzef65oWuszfFGNix4AcjQhDyxR7C99Zkj3JQ7rQI2qxWmdw15KwENLyUuHaJ8uzoudFZlE+i7bl+wCErYFCJyPN+WBSWQwzc5Ac16V+UazrIFRSLGoMvlE31X7q3BUY2JP5+mszfVmHuHuthSbvhD\\\\\\\/OZt7qwqb7R6r52+VFGdsa0VE8XpBTk9Lg5WU1fdp8rqFvkR5GhF2Hcn+xG4tD0I225up70OVSlWht9SR2h0U3BrsHz78LZTYR6yc+lKd3fDKOs6T18yiwR2V3ScDvHKRD\\\\\\\/r3wQsMIc5FGvkh1ZJw+Vv\\\\\\\/vTgjkD5IVCOKY9656cBz48TSrMsYwQHxM3dtPjLn\\\\\\\/1j7txBbqNEpFTHIiesz2Hz54Nk7tE4XSW3v\\\\\\\/raTbFIVZF98pYbLXm5qplDZwvHaS\\\\\\\/m1P7blFBGkcLFt4W\\\\\\\/voHYLQvYnPmpSCLZLmyF+Eqy1GWdrWM81GgeSu4FSndMwV7n9HqTNir63LBUpbgo99\\\\\\\/2bjC4dWtL83NwY7cUEU8xF44B5dXkNWi8LSTHmZnMNPxLEDHFS\\\\\\\/5mqHJzQJvCCOiizUxAwvrIUcAtZdiR5sdocRkKFddCMO1deTseWvUSo0bQXjEd2byKqtLAgoYj7xreUbHOiTPQ\\\\\\\/xfCzrm\\\\\\\/EjXi5HPWfPdJw4UpPi7Rk9bETlXRIARbO943IIgMAqKJMrfcbL0oYdr\\\\\\\/G75GBABVmIdMozAzkUchYuISN5\\\\\\\/KgqOcN9KvMu7CmyKkMVwOE8C+Odd3m2ZBBZfQY0B+2JyQZt+HsLL1nM7CDmvTY8q8QTuwbQP89r+vhsitjyXuXYSqyD9cwKxJp6GNBoQWd+\\\\\\\/o3nyqzhlK\\\\\\\/jjGflyR4ajIRBxbBLJGmP5ndAL1Ia2wD6C+FSQn\\\\\\\/0nO9ebZUKNNqpsYojjLN0MXPCsW8EOmnqySbI3PnaAa0+sGTu4DTWuitp81+1OOTN1WqxBgsxSx00c9WS93M\\\\\\\/LGi\\\\\\\/1BwygJzbd8K\\\\\\\/0\\\\\\\/p6MbNUieXpHV78TuN9z1OXabh3TTubuBGGstTP1IL3ZOCgvEksYt3dw5u4H89bFidscLESNsz4OooPg4i4lrhkwS4SyHq1VnJDP9Fj9lDz1+n0M9zC5sh6MKoTHLIzqsFmkAv15nHhkTJeHzeFWYt6WdOiMKNIiv1y3bL5Bfm7znPanClda2tXVvAR\\\\\\\/hDRLJ2V3ft1jFbyMK7ObZO9TvFSOrnDxj22yUUCtrpoqIm0bvNCkB9\\\\\\\/R6mjGiIc1pCC6+GzZxy1tdxsgczIrxhJBKaRb23hYIah5MoGKAZpYHaaxWjYpaubZoOMe5ljk3yzfUPfUgDGWUgp6b2FIwQrgmTOMFs5M70kWOkS7AD5LRlMdqfl47CLatdQs+4YkuMHzLyggj8oC+o2qRJYfPvWD8NN0rfto4ysnNRGXW4GIKTiYxsGuIT\\\\\\\/56SUyZ1d6JnVcbfJdsaGbW85movAyqmQVX6qm1tyaOYeJjQYEXq\\\\\\\/mGj6yZ+5bLGtgsS4M6Esf8PXC8GETB3IwMD6F3b8B5R1lqs9Dxu1s9WtwLGywZgHsr1BoQwQaRWN1nt8FtercXA+Xiavl4qm9hf\\\\\\\/Hdvz\\\\\\\/4OxXnhy5MkkbXDWs0SQsYPpYOoROjQa53A9NhIqZMsGPiVQZ7dIdNnWu1YBsANemsuh1ub9qVuzqruz4r+ouSiZRI07WzyXyVrce4vbhhYhLsNnOYpILtur8f\\\\\\\/DNrtaGTVy2u2fgYL1D4g+TL5A5ldSCB\\\\\\\/3s7v7HmwtLlWTsSXdlBgHyRn2NqVhDQoeLbVDG\\\\\\\/cGMlYTQR\\\\\\\/oGAh9dotbw2iSVvXonOX5gecBujhcQmSPFrPkofcAKnNg\\\\\\\/GYHZXGG1uITcJty2I6UXoa3l+3H7xpKxjoNUqnBmzfWZ+i9Dv+mAfMGv+Hc59t\\\\\\\/oJgRPOQ87dmi2cIG87\\\\\\\/SeGea2nS9X\\\\\\\/YvwzLggW84nYg\\\\\\\/kG49PUblI10sTSW\\\\\\\/wvyWGRqR+ZHHeF6lAjN3LmUSjmZ0DddfjU5nKyszKIG8rA8Qg3wjXs8p9OFuNxecreKcHIhuB1eJpW84EnzVAoIBnsD0AQdvRAO03yI2zkVtMFCiLJUmtATnSuO+Swrplywl70dxoA\\\\\\\/P6nl9D\\\\\\\/vYlA1KiMZJy6RxF1Ywa1VM8prK23rs40L\\\\\\\/atZQylRImcDKYwq\\\\\\\/QwEydcMr1D6XFT9qv4a5UDLvrmSJA5A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/0ru5FZk\\\\\\\/CkcFyz+dTsnVA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a0c09c3c5c42f5d25b5fd00c9b576661aacee894dc512e1d68a46fe960d9a181\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pUZFL3HHqHYaqeUJBKdXlkC7pTJfT1XPuzdDyzN937X9lTE1AkduEGWRd1rQ0\\\\\\\/hKsm1L2Wz9vznrGnCCNItxqiWCHf61\\\\\\\/yrDgcntJWlgufJeHDEDp3RFYaRvRSWXAXhV5fhs5pouvzl3cpvPXE+MRzwza0Rh53rFXnQSnMVL3wPduS3Bj5+jWj0VTaJKnj40uIQw70sQ1uwQ3Y8lMuhZJou+uIFB2rTnsAirxfjXRvM2xiersunphflbHTWC6CQop2kEsToYYxvQfnFaTKHnaxz9Uzb0gaaHGsxaYSowJK3SCqL5ngFDfduecdvIPczcxmrn0m0U\\\\\\\/6R90\\\\\\\/\\\\\\\/6T23+POU6fAZ5vh10Nt0mMt0ptHnZrIN7lRvByfUpg62g6xxNzQYVWsOjpZ1GAI7UJSvA\\\\\\\/pvERVjhrciy+QarA1+HOah0U\\\\\\\/RrGIsJMir3Zv4FrViXQCQ60OuZ3mx4xU5PWxofalqypazXH1dm0O86I1ZkGIA9vZ\\\\\\\/IMNFBKc\\\\\\\/C+oTlfeS7JtunvV6GNKX0YH9zeE\\\\\\\/Xf6AET2A8KV2\\\\\\\/rX9JckaKd1Jwge7S\\\\\\\/vGerw2YtbHhVi+I+Y2xpJthjg2GCZ0Nd9sTJLEHo7UdnE3Aft4fJfkgeQFUTfxR9eKNOY2+MG5QeDOGnwPUJULn24TpGI2RLz3Wqr1CVxn7GxqogbDjmrcgwya8jkvItVjUndSu3xMS0fy5yG6eFjh02pK2YB6RYxFandZ2oktqCJVDbQ6aHnAlDIVGJ5OG69XfHt71wo04EocdPnxI5Nxp20pNfwVgO\\\\\\\/uriMJYnIdyUA37AXElIlpLmOk82h2Gwyez4l\\\\\\\/Q9gS52TxSjuqWwwYFLQGQY2uqfCH1VDOfDRtw4zJzm7Qxk\\\\\\\/czJN+0jeR3kDNS9F0vGAc06eYM07wxbpyQizV4EwxutUl60HvdaPEEKd8ekknC8+PclT6lKpY8wZ6tybwH3gcI\\\\\\\/sDi+HlxXJlA0zo\\\\\\\/mFDzWLGjtKYuY41+1Ur4iYs\\\\\\\/Jv0\\\\\\\/a\\\\\\\/4Vfj+MEwpjb6s9l2820poMEt56c8dG7clare9pIj0eK3bTi1X9i88Wg3c79fICrkDanoHketx+B5f148nALvGMyrVdCm1e+WFZLqnb7Yz9RazAEpJ\\\\\\\/9Rn9hySZqUAbyWm6fdhw5dVq7UKN3IyhXGjeUsF9aQcXC22LfXZKDZ+YMWYyEg\\\\\\\/afaHHuTiMJPIk+PCn4xid1XWCJ4n7bcoEORTcSKepl5nD3b2\\\\\\\/f2XkiPHglGpiZxbTqIBjrrtIg5BdfZIP69d798Rb27bKUYyYnXYU2bJqyNvTqbKiFIWuaAmk1+kmbzyhwcFHd9l9uNaTxXepyFYsWnQkExUzDzm10OiyOs1Is08ounFqc6+2zQE9001ZhYuWkabxUdjQzAWEXvRLxsnbj6uVbGjpf3bvcwXYr7gMxMluaXH4yNv3Y4S01PlFrSwLsKuhwVhKv5jtes3xbI+cWsKuMRAnETScShkZtXm\\\\\\\/LLqQjHSFmUwbAxpM\\\\\\\/A7c1FV\\\\\\\/i1QBKqQQzzC7Kw9zpEeRU9Zp6eTLTz5GJoCMvjGQ+2th83dKdXTiZeYBuMRmugRXrDOe+MQEWtDCOCgvQeq1kZ8Cy4FBjCHyv7BGKxNbG2Y5+\\\\\\\/1szezGcxMttkXYA0MmvLGqjo2la2BrMGdFgtRPAd76rtnksKPmjaP11wzT4zMlaMdLVEKZAOMlcAvHxi5aonlDR9GdY588ZRq4EKnDdjoFzVg42UG7m\\\\\\\/2A6JqcZ8+A3LyZZLwrOVoSLyfGorLgSGWaTZmXAAy8IgCJtDddQ8aYoRPPp2pxCZjM7UEapKv5\\\\\\\/o4TsUICqLkKrcN3QutHq\\\\\\\/16PilgmttXtbw+6Fe7v6T\\\\\\\/2+I+1UDn8QXSJ\\\\\\\/DjyYpc2Sn1fGEoWx4n6IPWLp+qYQ0ygLQ5pH4xCaihQTij6EmbAAWfDFDiiu8\\\\\\\/ymlKbbYLEzThJ9q9TfwMFVdcKc8RBKNtsN9EY4UA6QvBVja5OTcIw5S9ep61b+hGYj5Im+Jc4m5+P0jXBU9i5w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OtdrtHDGA2TqgXM638XBnw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221494e8f87335c1c175866c99fec3a33d8780a95214e6a4bbbc425d4ba9a9e25e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492877 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1487" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wnB2JWR9ZS0PV2l4Nh3Wh8vR7e79y4rRjXQQb_V2fPY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1487" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Xasby8USntM54A-Kdi7FueAwibG9eyOjLyF73CNTXK4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1481" title="Locating and Characterizing a Transgene Integration Site by Nanopore Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1497" title="Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492877" id="node492877" data-pisa="ggg;9/5/1487" data-pisa-master="ggg;g3.119.400006" data-apath="/ggg/9/5/1487.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The <em>Neurospora crassa</em> Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dev Ashish Giri</span>, <span class="highwire-citation-author" data-delta="1">Ajith V. Pankajam</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4342-7501" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Koodali T. Nishant</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Durgadas P. Kasbekar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1487-1496;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400006 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492877" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dev Ashish Giri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500039, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Studies, Manipal University, Manipal 576104, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BAshish%2BGiri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giri%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BAshish%2BGiri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biology</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Computation Modelling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram 695551, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4342-7501" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Koodali T. Nishant</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Durgadas P. Kasbekar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500039, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Durgadas%2BP.%2BKasbekar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasbekar%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADurgadas%2BP.%2BKasbekar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Durgadas P. Kasbekar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kas@cdfd.org.in" class="" data-icon-position="" data-hide-link-title="0">kas@cdfd.org.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1487" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492877" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492877/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1487.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492877" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492877/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1487.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492877" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492877/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1487.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492877" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492877/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Meiotic silencing by unpaired DNA (MSUD), an RNAi-mediated gene silencing process, is efficient in crosses made in the <em>Neurospora crassa</em> standard Oak Ridge (OR) genetic background. However, MSUD was decidedly less efficient when the OR-derived MSUD testers were crossed with many wild-isolated strains (W), suggesting that either sequence heterozygosity in <em>tester</em> x W crosses suppresses MSUD, or that OR represents the MSUD-conducive extreme in the range of genetic variation in MSUD efficiency. Our results support the latter model. MSUD was less efficient in near-isogenic crosses made in the novel <em>N. crassa</em> B/S1 genetic background, and in <em>N. tetrasperma</em> strain 85. Possibly, in B/S1 and 85, additional regulatory cues, absent from OR, calibrate the MSUD response. A locus in distal chromosome 1R appears to underlie the OR <em>vs.</em> B/S1 difference. Repeat-induced point mutation (RIP) destroys duplicated genes by G:C to A:T mutation of duplicated DNA sequences. Chromosome segment duplications (<em>Dp</em>s) dominantly suppress RIP, possibly by titrating out the RIP machinery. In <em>Dp</em> x <em>N</em> crosses, the <em>Dp</em>–borne genes cannot pair properly, hence efficient MSUD, as in OR, silences them and renders the crosses barren. We speculate that the increased productivity engendered by inefficient MSUD enables small duplications to escape RIP. </p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromosome-segment-duplication" class="hw-term hw-article-keyword hw-article-keyword-chromosome-segment-duplication" rel="nofollow">chromosome segment duplication</a></li><li class="kwd"><a href="/keyword/genome-heterozygosity" class="hw-term hw-article-keyword hw-article-keyword-genome-heterozygosity" rel="nofollow">genome heterozygosity</a></li><li class="kwd"><a href="/keyword/introgression" class="hw-term hw-article-keyword hw-article-keyword-introgression" rel="nofollow">introgression</a></li><li class="kwd"><a href="/keyword/rip-suppression" class="hw-term hw-article-keyword hw-article-keyword-rip-suppression" rel="nofollow">RIP suppression</a></li></ul><div class="section intro" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Meiotic silencing by unpaired DNA (MSUD) was discovered in crosses made in the standard Oak Ridge (OR) genetic background of <em>Neurospora crassa</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aramayo and Metzenberg 1996</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shiu <em>et al.</em> 2001</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Perkins 2004</a>). Any gene not properly paired with its homologous sequence during meiosis is transcribed into ‘aberrant RNA’ that is then processed into single-stranded “MSUD-associated small interfering RNA” (masiRNA), which directs a silencing complex to degrade complementary mRNA, and thus silences the unpaired gene and any other genes homologous to it (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hammond <em>et al.</em> 2013b</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hammond 2017</a>). The ::<em>act</em>, ::<em>asm-1</em>, ::<em>Bml<sup>r</sup></em>, ::<em>mei-3</em>, and ::<em>r<sup>+</sup></em> MSUD tester strains contain an additional copy of the <em>act</em> (<em>actin</em>), <em>asm-1<sup>+</sup></em> (<em>ascospore maturation-1</em>), <em>Bml</em> (<em>β−tubulin</em>), <em>mei-3</em>, or <em>r<sup>+</sup></em> (<em>round ascospores</em>) gene inserted at an ectopic location. The ectopic copy is unpaired in a <em>tester</em> x OR cross and induces the production of masiRNA which degrades its complementary mRNA, and the resulting deficit of actin, ASM-1, β-tubulin, MEI-3, or R protein manifests as striking ascus or ascospore phenotypes, whereas in a homozygous <em>tester A</em> x <em>tester a</em> cross the ectopic copy is paired, therefore there is no MSUD, and ascus and ascospore development is normal (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Raju <em>et al.</em> 2007</a>).</p><p id="p-4">Surprisingly, MSUD was not as efficient when the OR-derived tester strains (<em>tester<sup>OR</sup></em>) were crossed with several wild-isolated <em>N. crassa</em> strains (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ramakrishnan <em>et al.</em> 2011</a>). Of 80 wild-isolated strains tested in crosses with the ::<em>Bml<sup>r</sup></em> and ::<em>mei-3</em> testers, only eight, designated as the “OR” type, showed silencing phenotypes comparable to the <em>tester<sup>OR</sup></em> x OR crosses. Crosses with four wild strains designated as the “Sad” type did not show MSUD, and the remaining 68 strains showed an intermediate phenotype, in that, the crosses silenced <em>bml</em> but not <em>mei-3</em><sup>+</sup>, and they were designated the “Esm” type. One hypothesis (model 1) to explain these results posits that sequence heterozygosity between the <em>tester<sup>OR</sup></em> and wild genomes either overwhelms the MSUD machinery or causes asynapses and self-silencing of one or more MSUD gene. This model extends the observations that deletion alleles of several MSUD genes can act as semi-dominant suppressors of MSUD (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Samarajeewa <em>et al.</em> 2017</a> and references therein), presumably because they cause their wild-type homolog to be unpaired, induce its autogenous silencing, and thus “silence the silencer” (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hammond 2017</a>); and that a 6% divergence over 4.5 kb between the <em>r<sup>+</sup></em> and <em>r<sup>RIP93</sup></em> alleles could induce silencing of the <em>r<sup>+</sup></em> allele (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pratt <em>et al.</em> 2004</a>). The <em>sad-1Δ</em> and <em>sad-2Δ</em> deletions (<em>i.e.</em>, <em>Sad-1</em> and <em>Sad-2</em> (<em>Suppressor of ascus dominance-1</em> and <em>-2</em>)) were strong dominant suppressors whereas the other MSUD gene deletions were less effective, possibly because of high expression or long protein half-life (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hammond <em>et al.</em> 2013a</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Decker <em>et al.</em> 2017</a>). <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shiu and Metzenberg (2002)</a> also generated <em>Sad-1</em> strong alleles by the RIP (repeat-induced point mutation) process, which showed that if the <em>sad-1</em> gene sequence is sufficiently altered then the <em>sad-1<sup>+</sup></em> allele can become unpaired and induce its autogenous silencing, thus supporting the idea that sequence heterozygosity can promote gene silencing. The <em>Sad-1</em> and <em>Sad-2</em> dominant MSUD suppressors suppressed the classical ascus-dominant mutants <em>Ban</em> (<em>Banana</em>), <em>Dip-1</em> (<em>Diploid ascospores-1</em>), <em>Pk</em><sup>D</sup> (<em>Peak-Dominant</em>), and <em>R</em> (<em>Round ascospores</em>), in which known or suspected deletion alleles induce MSUD in their wild-type counterparts (<em>ban<sup>+</sup></em>, <em>dip-1<sup>+</sup></em>, <em>pk<sup>+</sup></em>, and <em>r<sup>+</sup></em>). The <em>Sad-1</em> and <em>Sad-2</em> suppressors also suppressed the barren phenotype of crosses heterozygous for chromosome segment duplications (<em>i.e.</em>, <em>Dp</em> x <em>N</em>), suggesting that the barrenness is caused by MSUD (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kasbekar 2013</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Shiu <em>et al.</em> 2001</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">2006</a>). <em>Dp(EB4)</em> and <em>Dp(IBj5)</em> strains contain duplications of, respectively, 35- and 115-gene segments (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kasbekar 2013</a>). Crosses of <em>Dp(EB4)</em> and <em>Dp(IBj5)</em> strains with the OR type wild strains were barren, with the Sad type were fertile, and with the Esm type were, respectively, fertile and barren (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Ramakrishnan <em>et al.</em> 2011</a>). Reduced MSUD in the related species <em>N. tetrasperma</em> also was attributed to asynapsis and silencing of the <em>sad-1</em> gene because of structural difference between the mating-type chromosomes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jacobson <em>et al.</em> 2008</a>).</p><p id="p-5">An alternative hypothesis (model 2) posits that natural populations harbor a wide variation in MSUD strength, and the OR strains represent the MSUD-conducive extreme. Model 1 predicts that crosses isogenic for a wild-isolated Sad type genetic background would show an OR-like MSUD phenotype. Here, we examined MSUD in tester-heterozygous crosses made in a novel isogenic background generated from the Sad type <em>N. crassa</em> wild strains Bichpuri-1 <em>a</em> and Spurger-3 <em>A</em>.</p><p id="p-6">MSUD is also much less efficient in <em>N. tetrasperma</em> strain 85 (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Jacobson <em>et al.</em> 2008</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Ramakrishnan <em>et al.</em> 2011</a>). Specifically, self-cross of an 85-derived [::<em>asm-1</em> + WT] dikaryotic strain produced &lt; 76% MSUD-induced white-spored asci whereas comparable crosses in OR produced &gt;99% white-spored asci (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Ramakrishnan <em>et al.</em> 2011</a>), thus 85 can be thought of as a Sad- or Esm-type background. We have now introgressed an ::<em>r</em> transgene from <em>N. crassa</em> OR into <em>N. tetrasperma</em> 85, and used the resulting ::<em>r<sup>Nt</sup></em> testers to make ::<em>r<sup>Nt</sup></em> x 85 crosses. We had previously introgressed the <em>N. crassa T(EB4)</em> and <em>T(IBj5)</em> translocations into <em>N. tetrasperma</em> and obtained <em>T(EB4)<sup>Nt</sup></em>, <em>T(IBj5)<sup>Nt</sup></em>, <em>Dp(EB4)<sup>Nt</sup></em> and <em>Dp(IBj5)<sup>Nt</sup></em> strains in which nominally only the rearrangement breakpoints were from <em>N. crassa</em> while the rest of the genome was from <em>N. tetrasperma</em> strain 85 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giri <em>et al.</em> 2015</a>). Since MSUD causes <em>Dp</em>-heterozygous crosses to become barren, and MSUD is less efficient in <em>N. tetrasperma</em> strain 85 than in <em>N. crassa</em> OR, we would expect <em>Dp</em>-heterozygous crosses in <em>N. tetrasperma</em> 85 to be more productive than their <em>N. crassa</em> OR counterparts. We used the <em>T<sup>Nt</sup></em> and <em>Dp<sup>Nt</sup></em> strains to test whether <em>Dp</em>- and <em>T</em>-heterozygous crosses were comparably productive in <em>N. tetrasperma</em>.</p><p id="p-7">During an <em>N. crassa</em> or <em>N. tetrasperma</em> sexual cross, the parental <em>mat A</em> and <em>mat a</em> haploid nuclei fuse to produce a diploid zygote nucleus that undergoes meiosis and a post-meiotic mitosis to generate eight haploid progeny nuclei (4 <em>mat A</em> plus 4 <em>mat a</em>). In <em>N. crassa</em> these nuclei are partitioned into the eight initially uninucleate ascospores that form in an ascus, whereas <em>N. tetrasperma</em> asci form four larger initially binucleate ascospores, each receiving a <em>mat A</em> and <em>mat a</em> pair (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Raju 1992</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Raju and Perkins 1994</a>). Consequently, the mycelium generated from an <em>N. crassa</em> ascospore is homokaryotic for mating type and requires mycelium from another ascospore of opposite mating type to complete the sexual cycle (heterothallic lifecycle), whereas the mycelium generated from a dikaryotic [<em>mat A</em> + <em>mat a</em>] <em>N. tetrasperma</em> ascospore contains both <em>mat A</em> and <em>mat a</em> nuclei and can undergo a self-cross (pseudohomothallic lifecycle). Dikaryotic mycelia can make some homokaryotic conidia (vegetative spores) by chance. Also, during <em>N. tetrasperma</em> ascogenesis, a pair of smaller homokaryotic ascospores can occasionally replace a dikaryotic ascospore to form a minor fraction of 5-8 spored asci. Mycelia from homokaryotic ascospores and conidia enable <em>N. tetrasperma</em> to out-cross with like mycelia of the opposite mating type. The <em>Eight spore</em> (<em>E</em>) mutant increases the replacement of dikaryotic ascospores by homokaryotic pairs, and <em>E</em> x <em>WT</em> crosses produce many 8-spored asci, although <em>E</em> x <em>E</em> crosses are infertile (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dodge 1939</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Calhoun and Howe 1968</a>). Does MSUD underlie the ascus-dominant <em>E</em> mutant phenotype? An earlier attempt to generate <em>N. tetrasperma Sad-1</em> mutants did not yield an allele that could induce silencing of its <em>sad-1<sup>+</sup></em> homolog (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bhat <em>et al.</em> 2004</a>), therefore here we have introgressed a strong <em>N. crassa Sad-2</em> MSUD suppressor into <em>N. tetrasperma</em> 85 and compared the <em>Sad-2</em> x <em>E</em> and <em>Sad-2<sup>+</sup></em> x <em>E</em> crosses.</p></div><div class="section materials-methods" id="sec-2"><h2 class="">Materials and methods</h2><div id="sec-3" class="subsection"><h3>Neurospora strains, culture methods, and crosses</h3><p id="p-8">Unless indicated otherwise, all Neurospora strains (Table S1) were obtained from the Fungal Genetics Stock Center (FGSC; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">McCluskey <em>et al.</em> 2010</a>), Department of Plant Pathology, 4024 Throckmorton Plant Sciences Center, Kansas State University, Manhattan, KS 66506, USA. Neurospora was cultured essentially as described by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Davis and De Serres (1970)</a>, using <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Metzenberg’s (2003)</a> alternative recipe to make Vogel’s medium N. Crosses were made at 25° by simultaneously inoculating mycelial plugs of the parental strains on synthetic cross medium supplemented with 1% sucrose and 2% agar.</p><p id="p-9"><em>N. crassa</em>: The standard laboratory Oak Ridge strains 74-OR23-1VA (FGSC 2489) and 74-ORS-6a (FGSC 4200) will be referred to henceforth as OR <em>A</em> and OR <em>a</em>. The wild-isolated strains Bichpuri-1 <em>a</em> (P750) and Spurger-3 <em>A</em> (FGSC 3201) were used to generate the novel B/S1 strains (see below). The RLM 30-12 strain (genotype <em>Sad-2</em> (<em>RIP32</em>) <em>A</em>) containing a RIP-mutated <em>Sad-2</em> allele was a gift from Robert L. Metzenberg and is described by <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Shiu <em>et al.</em> (2006)</a>. The strains ISU 3118 (genotype: <em>rid his3</em>; <em>mus-52</em>Δ::<em>bar</em>; <em>VIIL</em>::<em>r<sup>ef1</sup>-hph A</em>) and ISU 3119 (genotype <em>rid his-3</em>; <em>mus-52</em>Δ::<em>bar</em>; <em>VIIL</em>::<em>r<sup>ef3</sup>-hph A</em>) are OR-derived MSUD testers bearing the transgenes ::<em>r1</em> and ::<em>r3</em> and were a gift from Dr. Tom Hammond, Illinois State University, Normal, IL 61790, USA. They are described by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Samarajeewa <em>et al.</em> (2014)</a>. The 5.5 kb long <em>r<sup>+</sup></em> gene (ncu02764, nucleotides 9,280,963 to 9,286,536) on chromosome 1 encodes a 3.3 kb ORF that shows robust MSUD that is suppressible only by the strong <em>Sad-1</em> and <em>Sad-2</em> suppressors, unlike other genes (<em>act</em>, <em>asm-1</em>, <em>mei-3</em>) in which MSUD is suppressed by even the weaker MSUD suppressors (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hammond <em>et al.</em> 2013a</a>). These testers are henceforth referred to as ::<em>r1<sup>OR</sup></em> and ::<em>r3<sup>OR</sup></em> to distinguish them from the ::<em>r1<sup>B/S1</sup></em> and ::<em>r3<sup>B/S1</sup></em> testers that we constructed in the B/S1 genetic background (described below). The insertion sites of the ::<em>r1<sup>B/S1</sup></em> and ::<em>r3<sup>B/S1</sup></em> transgenes are identical to those of ::<em>r1<sup>OR</sup></em> and ::<em>r3<sup>OR</sup></em>.</p><p id="p-10">Strains #1 (genotype ::<em>r1<sup>OR</sup>a</em>) and #6 and #15 (genotype ::<em>r1<sup>OR</sup>A</em>) were obtained as segregants from the cross ISU 3118 x OR <em>a</em>, and strains #1 and #10 (genotype ::<em>r3<sup>OR</sup>a</em>), and #11 (genotype ::<em>r3<sup>OR</sup>A</em>) were obtained from ISU 3119 x OR <em>a</em>. Strains #4 (genotype ::<em>r1<sup>B/S1</sup> a</em>) and #1 (genotype ::<em>r1<sup>B/S1</sup> A</em>) were obtained from ::<em>r1<sup>B/S1</sup> A</em> x B/S1 <em>a</em>, and strains #4 and #16 (genotype ::<em>r3<sup>B/S1</sup>A</em>) and #15 and #24 (genotype ::<em>r3<sup>B/S1</sup> a</em>) were obtained from ::<em>r3<sup>B/S1</sup> A</em> x B/S1 <em>a</em> (described below). The results of crosses made with the ::<em>r1<sup>OR</sup></em> and ::<em>r1<sup>B/S1</sup></em> strains are presented in Table S3, and those with ::<em>r3<sup>OR</sup></em> and ::<em>r3<sup>B/S1</sup></em> are presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492961/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492961/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Round ascospore fractions from crosses heterozygous or homozygous for an ::<em>r3</em> transgene</span><div class="sb-div caption-clear"></div></div></div><p id="p-12"><em>N. tetrasperma</em>: The reference strains 85 <em>A</em> (FGSC 1270) and 85 <em>a</em> (FGSC 1271); the <em>E</em> mutants <em>lwn</em>; <em>al(102)</em>, <em>E A</em> (FGSC 2783) and <em>lwn</em>; <em>al(102)</em>, <em>E a</em> (FGSC 2784) (hereafter <em>E A</em> and <em>E a</em>). Previously, we had introgressed the <em>N. crassa T(EB4)</em> and <em>T(IBj5)</em> translocations into <em>N. tetrasperma</em> to construct the strains <em>T(EB4)<sup>Nt</sup></em> and <em>T(IBj5)<sup>Nt</sup></em>, and from the <em>T<sup>Nt</sup></em> x 85 crosses we obtained the self-fertile dikaryotic strains [<em>T<sup>Nt</sup></em> + <em>N</em>] and [<em>Dp<sup>Nt</sup></em> + <em>Df<sup>Nt</sup></em>] (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Giri <em>et al.</em> 2015</a>). Homokaryotic conidial derivatives of genotype <em>T(EB4)<sup>Nt</sup> a</em>, <em>T(IBj5)<sup>Nt</sup> a</em>, <em>Dp(EB4)<sup>Nt</sup> A</em> and <em>Dp(IBj5)<sup>Nt</sup> a</em> were obtained from these dikaryons.</p><p id="p-13">The <em>Sad-2<sup>Nt</sup></em> and ::<em>r3<sup>Nt</sup></em> strains were made by introgressing the relevant <em>N. crassa</em> gene into the strain 85 genetic background (see below). They were used to measure MSUD efficiency in ::<em>r3<sup>Nt</sup></em>-heterozygous and ::<em>r3<sup>Nt</sup></em>-homozygous crosses and in ::<em>r3<sup>Nt</sup></em> x <em>Sad-2<sup>Nt</sup></em>. The superscript “<em>Nt</em>“ in the strain designations (eg., ::<em>r3<sup>Nt</sup> A</em>, ::<em>r3<sup>Nt</sup> a</em>, <em>Sad-2<sup>Nt</sup> A</em>, and <em>WT<sup>Nt</sup>A</em>) indicates that the strain has in its ancestry one or more self-fertile dikaryotic strain (<em>i.e.</em>, <em>N. tetrasperma</em>-like).</p><p id="p-14"><em>N. crassa</em> / <em>N. tetrasperma</em> hybrid strain: <em>C4,T4 a</em> (FGSC 1778; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Metzenberg and Ahlgren 1969</a>) was used as a bridging strain to initiate the introgression of <em>N. crassa</em> genes into <em>N. tetrasperma</em>. Crosses of <em>C4,T4a</em> with opposite mating type strains of <em>N. crassa</em> and <em>N. tetrasperma</em> strains can produce small numbers of viable progeny.</p></div><div id="sec-4" class="subsection"><h3>Ascospore harvests</h3><p id="p-15">Ordinarily ascospores can be harvested by simply washing the lids of the Petri plates in which the crosses are made with 1.5 ml water, concentrating the ascospore suspension to 0.1-0.5 ml, and then counting an aliquot in a hemocytometer. This is simple enough to do and one can obtain adequate ascospore numbers. However, we noticed that many round as well as some spindle-shaped ascospores remained adhered to the outside of perithecia, or were deposited close by on the agar, especially in the crosses involving the B/S1 strains and in <em>N. tetrasperma</em>. This suggested that round ascospores might suffer impaired dispersal relative to the spindle-shaped ones, and the former might also interfere with dispersal of the latter, possibly by clogging the perithecial exit pore (ostiole). In which case, scoring only the lid-harvested ascospores might underestimate MSUD strength. The results in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tables 1</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>, and S4 represent “total harvests”. They were obtained by combining the “lid harvests” with ascospores harvested by scraping the perithecia, mycelia, and ascospores from the agar surface of the cross plate with a rubber policeman or spatula into 2 ml water, transferring the suspension to a 2 ml tube, spinning for 60 sec in a table top centrifuge to pellet the ascospores, and removing the supernatant with debris. The ascospore pellet was re-suspended in 0.1-0.5 ml water and an aliquot counted in a hemocytometer. <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a> and S4 present the results from the total harvests in crosses heterozygous or homozygous for the ::<em>r3</em> and ::<em>r1</em> transgenes, whereas Tables S5 and S6 represent the corresponding lid harvests. It is apparent that the difference between the two harvests was not as pronounced in the OR background. <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Liu <em>et al.</em> (2017)</a> have suggested that A-to-I mRNA editing “corrects” conserved premature stop codons in genes related to ascosporogenesis and ascospore discharge, therefore the altered ascospore dispersal might reflect editing differences in B/S1 and <em>N. tetrasperma</em> relative to OR (see Discussion section).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493189/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493189/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">MSUD-induced round ascospore fractions in <em>N. tetrasperma</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Construction of MSUD testers in the B/S1 background</h3><p id="p-17">A cross was made between the wild-isolated strains Bichpuri-1 <em>a</em> and Spurger-3 <em>A</em> (B <em>a</em> x S <em>A</em>) and the f<sub>1</sub> progeny were used to make four f<sub>1</sub> <em>a</em> x f<sub>1</sub> <em>A</em> sib-pair crosses that initiated four recombinant inbred lines (Figure S2A). Within a line, in each generation sibling progeny of opposite mating types were crossed to produce the next generation (<em>i.e.</em>, sibling f<sub>1</sub> <em>a</em> x f<sub>1</sub> <em>A</em> to produce the f<sub>2</sub>, then sibling f<sub>2</sub> <em>a</em> x f<sub>2</sub> <em>A</em> to produce the f<sub>3</sub>, etc). A recessive sterility-causing mutation became homozygous in the f<sub>4</sub> and f<sub>3</sub> generation of, respectively, lines 2 and 4 (C. Usha and D. P. Kasbekar, unpublished results), but we were able to reach the f<sub>10</sub> generation in lines 1 and 3. A <em>mat A</em> and <em>mat a</em> strain pair of the f<sub>10</sub> generation of line 1 (specifically, segregants #1 and #3) are referred to henceforth as B/S1 <em>A</em> and B/S1 <em>a</em> (Figure S2A). A similar schematic diagram illustrating the early crosses (upto the f<sub>8</sub> generation) was presented in <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nagasowjanya <em>et al.</em> (2013)</a>.</p><p id="p-18">The <em>mus-51</em> gene in the B/S1 background was mutated by RIP (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Selker 1990</a>). Strains mutant in <em>mus-51</em> are defective for non-homologous end joining, consequently, any transforming DNA can integrate only via homologous recombination (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ninomiya <em>et al.</em> 2004</a>). A DNA construct bearing a 1683 bp <em>mus-51</em> segment (-205 to 1478 bp of the 2046 bp MUS-51 ORF) and the hygromycin-resistance (<em>hph</em>) cassette (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carroll <em>et al.</em> 1994</a>) was transformed by electroporation into B/S1 <em>A</em> conidia, and ectopic integration of the transforming DNA created the transgenic <em>Dp(mus-51)</em> duplication. The <em>Dp(mus-51) A</em> primary transformant was crossed to B/S1 <em>a</em> and the progeny were used to perform a <em>Dp(mus-51)</em>-homozygous cross. Of 40 progeny examined from late harvested ascospores, one <em>mat a</em> progeny (#24) was found to contain several RIP-induced mutations in the endogenous <em>mus-51</em> gene, including in-frame stop codons (Genbank accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KM025239&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">KM025239</a>), and it was crossed with the B/S1 <em>A</em> strain to segregate out the <em>Dp(mus-51)</em> transgene and obtain the B/S1 <em>mus-51 A</em> #8, B/S1 <em>mus-51 a</em> #3 strains in the progeny.</p><p id="p-19">Transformation of B/S1 <em>mus-51A</em> conidia was used to make the <em>tester<sup>B/S1</sup></em> strains, namely, ::<em>r1<sup>B/S1</sup></em> and ::<em>r3<sup>B/S1</sup></em>, which are analogous to the OR-derived ::<em>r1<sup>OR</sup></em> and ::<em>r3<sup>OR</sup></em> testers of <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Samarajeewa <em>et al.</em> (2014)</a>. A 2532 bp DNA fragment (called <em>r<sup>ef</sup></em>) was amplified by PCR from the 3′ end of the OR-derived <em>r<sup>+</sup></em> gene and joined to the <em>hph</em> cassette by double-joint PCR (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Yu <em>et al.</em> 2004</a>) to create a 4.1 kb <em>r<sup>ef</sup>-hph</em> fusion construct. The fusion construct’s flanking sequences were derived from the B/S1 background to enable it to be precisely inserted by homologous recombination into the same genomic sites as the ::<em>r1<sup>OR</sup></em> and ::<em>r3<sup>OR</sup></em> testers (Figure S2B). The DNA constructs were transformed by electroporation into the B/S1 <em>mus-51 A</em> strain, and transformants were selected on hygromycin-medium. The potentially heterokaryotic primary transformants were crossed with B/S1 <em>a</em> to segregate out the <em>mus-51</em> mutation and obtain a ::<em>r3<sup>B/S1</sup> A</em> homokaryon from whose cross with B/S1 <em>a</em> we obtained the segregants #15 and #24 (of genotype ::<em>r3<sup>B/S1</sup> a</em>) and #4 and #16 (of genotype ::<em>r3<sup>B/S1</sup>A</em>). In a like manner we obtained an ::<em>r1<sup>B/S1</sup> A</em> homokaryon that was crossed with B/S1 <em>a</em> to obtain the segregant #4 (of genotype ::<em>r1<sup>B/S1</sup> a</em>) and segregant #1 (of genotype ::<em>r1<sup>B/S1</sup> A</em>). Between derivation of the B/S1 <em>A</em> and <em>a</em> strains and the ::<em>r<sup>B/S1</sup> A</em> and <em>a</em> strains five additional backcrosses were done to B/S1, thus increasing the chance that a ::<em>r<sup>B/S1</sup> A</em> x ::<em>r<sup>B/S1</sup> a</em> cross is further reduced in heterozygosity compared to a B/S1 <em>A</em> x B/S1 <em>a</em> cross.</p><p id="p-20">Strains B/S1 <em>A</em> #1 (FGSC 26446), B/S1 <em>a</em> #3 (FGSC 26445), B/S1 <em>mus-51 A</em> #8 (FGSC 26444), B/S1 <em>mus-51 a</em> #3 (FGSC 26443), VIIL ::<em>r1<sup>B/S1</sup>-hph A</em> #1 (FGSC 26442), VIIL ::<em>r1<sup>B/S1</sup>-hph a</em> #4 (FGSC 26441), VIIL ::<em>r3<sup>B/S1</sup>-hph A</em> #16 (FGSC 26440), and VIIL ::<em>r3<sup>B/S1</sup>-hph a</em> #15 (FGSC 26439) have been deposited in the FGSC, with the accession numbers indicated in parenthesis.</p></div><div id="sec-6" class="subsection"><h3>Derivation of the ::r3<sup>Nt</sup> and WT<sup>Nt</sup>A strains</h3><p id="p-21">Strain ISU 3119 (genotype <em>VIIL</em>::<em>r3 A</em>) was crossed with the <em>N. crassa</em> / <em>N. tetrasperma</em> hybrid bridging strain C4T4 <em>a</em> and an ::<em>r3<sup>1C</sup>A</em> progeny was back-crossed with C4T4 <em>a</em> to produce an ::<em>r3<sup>2C</sup> a</em> progeny, that was crossed with 85<em>A</em>, and an ::<em>r3<sup>185</sup>a</em> progeny was back-crossed with 85 <em>A</em>. (Superscript “1C<sup>”</sup> indicates progeny from the first cross with C4T4 <em>a</em>, “2C<sup>”</sup> indicates progeny from the second cross with C4T4 <em>a</em>, and “n85” indicates progeny from the nth backcrosses with strain 85.) From the cross ::<em>r3<sup>185</sup>a</em> x 85 <em>A</em> we obtained several self-fertile progeny bearing the ::<em>r3</em> transgene, <em>viz</em>., 1R, 2R, 6R, 8R, 10R, 13R, 15R, 17R, and 20R. Self-cross of these self-fertile strains produced mostly four-spored asci, including several with round ascospores (% round ascospores, respectively, 90, 5, 5, 45, 90, 1, 2, 55, and 1). The crosses whose results are summarized in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> were made with self-sterile conidial derivatives (CD) obtained from these self-fertile strains. The <em>VIIL</em>::<em>r3<sup>Nt</sup> a</em> strain used was CD #4 derived from 10R, and <em>VIIL</em>::<em>r3<sup>Nt</sup> A</em> was CD #5 derived from 1R. The ::<em>r3<sup>Nt</sup> a</em> strain was deposited in the FGSC with the accession number FGSC 26489.</p><p id="p-22">The <em>WT<sup>Nt</sup>A</em> strain in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> was CD #5 derived from 6R, and it contains the non-transgenic nucleus of the 6R dikaryon. It was used as an additional “wild type” control.</p><p id="p-23">CD #1 derived from 13R turned out to be a self-sterile [::<em>r3<sup>Nt</sup>A</em> + <em>WT<sup>Nt</sup>A</em>] dikaryon whose identification and possible provenance are described in the Results and Discussion sections.</p><p id="p-24">The oligonucleotide primers dhphF and dhphR (Table S2) were used to PCR amplify a 1.5 kb segment from genomic DNA of <em>N. crassa</em> and <em>N. tetrasperma</em> strains containing the chromosome 7 transgene ::<em>r3</em>. Strains bearing a transgene-free chromosome 7 were detected by PCR amplification of a 2.5 kb amplicon with the primers (Table S2) r<sup>Nt</sup>WT-F and either r<sup>Nt</sup>WT-OR-R (for OR-derived DNA) or r<sup>Nt</sup>WT-85-R (for strain 85-derived DNA).</p></div><div id="sec-7" class="subsection"><h3>Derivation of Sad-2<sup>Nt</sup> strains</h3><p id="p-25">The construction of the <em>Sad-2<sup>Nt</sup>A</em> and <em>Sad-2<sup>Nt</sup>a</em> strains is schematically outlined in Figure S1. The bent arrows in the figure indicate identification of the <em>Sad-2</em> progeny by PCR. The primers sad-2spc-F and sad-2spc-R (Table S2) can amplify a 1 kb segment in PCR using genomic DNA of strains bearing the <em>Sad-2</em> (<em>RIP32</em>) allele as template, whereas primers sad-2+spc-F and sad-2+spc-R (Table S2) can amplify a 619 bp amplicon in PCR with genomic DNA of <em>N. crassa</em> and <em>N. tetrasperma</em> strains containing the wild type <em>sad-2<sup>+</sup></em> allele. Briefly, from the cross of the <em>N. crassa</em> RLM 30-12 strain (genotype <em>Sad-2</em> (<em>RIP32</em>) <em>A</em>) with <em>C4,T4 a</em> we identified a <em>Sad-2<sup>1C</sup> a</em> progeny by PCR, and from the cross <em>Sad-2<sup>1C</sup> a</em> x 85<em>A</em> we obtained a <em>Sad-2<sup>185</sup> a</em> segregant and used it to initiate a series of backcrosses with the 85<em>A</em> or 85 <em>a</em> strains. After six successive backcrosses we obtained a self-sterile [<em>Sad-2<sup>685</sup> A</em> + <em>Sad-2<sup>685</sup> a</em>] dikaryon that was productive in crosses with both 85 <em>A</em> and 85 <em>a</em>. From the dikaryon’s cross with 85 <em>a</em> we obtained S12, a self-fertile dikaryon of presumed genotype [<em>Sad-2<sup>785</sup>a</em> + <em>sad-2<sup>+</sup> A</em>]. From the self-cross of S12 we obtained the [<em>Sad-2</em> + <em>sad-2</em><sup>+</sup>] dikaryons 1S12 and 3S12, and, from the self-cross of 1S12 we obtained the [<em>Sad-2</em> + <em>sad-2</em><sup>+</sup>] self-fertile dikaryons 1(1S12), 3(1S12), 4(1S12), 9(1S12). The <em>Sad-2<sup>Nt</sup>A</em> strain used in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> is a <em>mat A</em> conidial derivative from 3(1S12). To derive the <em>Sad-2<sup>Nt</sup>a</em> strain we obtained one <em>Sad-2 a</em> homokaryotic progeny (#20) from the cross of the self-sterile [<em>Sad-2<sup>685</sup> A</em> + <em>Sad-2<sup>685</sup> a</em>] dikaryon with 85 <em>A</em>. This cross yielded mostly self-fertile progeny, including one called S1, from which we obtained the <em>Sad-2<sup>Nt</sup> a</em> strain as a conidial derivative, and it is deposited in the FGSC with accession number FGSC 26490.</p></div><div id="sec-8" class="subsection"><h3>Whole genome sequence analysis of Neurospora strains</h3><p id="p-26">DNA samples were sequenced at Fasteris, Switzerland, on Illumina HiSeq 4500 platform with paired end reads and read length of 150 bp. Quality and statistics of the raw reads were analyzed using FastQC (Version 0.11.5). Illumina adapters from the paired reads were removed using trimmomatics (version 0.36) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>). Preprocessed reads were aligned to the <em>N. crassa</em> OR74A reference genome (version NC12, release 38, ensembl) using bowtie2 (version 2.3.4) with end-to-end parameter (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Langmead and Salzberg 2012</a>). All samples were observed to have more than 80% alignment rate and at least ∼8 million aligned reads (Table S3). Duplicated reads in the alignment file were removed using picardtools (version 2.17.4). Variant calling or genotyping of the thirteen samples (NC1 to NC13) was performed using Gatk HaplotypeCaller (version 4.0.0.0) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">DePristo <em>et al.</em> 2011</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Van der Auwera <em>et al.</em> 2013</a>). We filtered out SNPs with QD (Quality by depth) &lt; 20. Genotype comparison among the NC1 to NC13 samples were performed using custom R script (version 3.4.3). The whole genome sequence data for the NC1 to NC13 strains are available from the National Centre for Biotechnology Information Sequence Read Archive under the accession number SRP149022.</p></div><div id="sec-9" class="subsection"><h3>Data availability statement</h3><p id="p-27">Strains and plasmids are available upon request. We affirm that all data necessary for confirming the conclusions of the article are present within the article, figures, and tables. Nucleotide sequence data are available at GenBank under accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KM025239&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">KM025239</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG009253&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG009253</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG009254&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG009254</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG009255&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG009255</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG009256&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG009256</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG017489&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG017489</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG017490&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG017490</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG017491&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG017491</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG017492&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG017492</a>; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK392332&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MK392332</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK392333.&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MK392333.</a> Whole genome sequence data for 15 <em>N. crassa</em> strains are available at the National Centre for Biotechnology Information Sequence Read Archive under accession number SRP149022. Supplementary Tables S1-S6 and Figures S1, S2 are placed in figshare.com with doi: <a href="https://doi.org/10.6084/m9.figshare.7570934">10.6084/m9.figshare.7570934</a>. Table S1 lists the Neurospora strains used, Table S2 lists the oligonucleotide primers, Table S3 summarizes the Illumina whole genome sequencing statistics, Table S4 presents the round ascospore fractions from crosses involving the ::r1 strains, Table S5 presents the round ascospore fractions in the lid harvests from crosses involving the ::r3 strains, Table S6 presents the round ascospore fractions in the lid harvests from crosses involving the ::r1 strains, Figure S1 is a schematic depiction of the derivation of the <em>Sad-2<sup>Nt</sup></em> strains, and Figure S2 is a schematic depiction of the construction and use of the ::<em>r<sup>B/S1</sup></em> tester strains.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Genome heterozygosity in the near-isogenic ::r3<sup>B/S1</sup> x B/S1 crosses</h3><p id="p-28">Previous work had revealed that only ∼10% of wild-isolated <em>N. crassa</em> strains (W) examined in crosses with the OR-derived MSUD testers showed an OR-type “efficient MSUD” phenotype (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Ramakrishnan <em>et al.</em> 2011</a>). If the hypothesis that MSUD is suppressed by genome sequence heterozygosity in <em>tester<sup>OR</sup></em> x W crosses is correct, then crosses isogenic for a wild-derived non-OR type genome (<em>i.e.</em>, <em>tester<sup>W</sup></em> x W) should exhibit efficient MSUD as in the OR background. To test this, we generated the novel near-isogenic strains B/S1 <em>A</em> and B/S1 <em>a</em> (see Materials and methods) from the Sad-type wild strains Bichpuri-1 <em>a</em> and Spurger-3 <em>A</em> and tested MSUD efficiency in this novel background.</p><p id="p-29">In the series of 10 successive sib-pair cross performed to create the B/S1 strains, the residual heterozygosity in each cross is half the value in the previous cross (100%, 50%, 25%, …, 0.4%, 0.2%, 0.1%). Hence the B/S1 <em>A</em> x B/S1 <em>a</em> cross is expected to be heterozygous in only ∼0.1% (<em>i.e.</em>, ∼44 kbp) of the <em>mat</em>-unlinked genome fraction. To verify this, we used Illumina whole genome sequencing to identify 8784 SNPs between the B/S1 <em>A</em> and B/S1 <em>a</em> genomes. This number excluded the SNPs within the <em>mat</em> locus, in which the <em>mat A</em> and <em>mat a</em> idiomorph sequences are very dissimilar and seemingly unrelated. The Bichpuri-1 <em>a</em> and Spurger-3 <em>A</em> genome sequences are not available, therefore we could not determine the percentage loss of heterozygosity (LOH) or map the LOH boundaries. Instead, of the 8784 SNPs, 7961 (90.6%) were found to be in a 570 kb <em>mat</em>-linked genome segment (see below), and in it the SNP density was 14 SNPs /kb (7961 SNPs/ 570 kb). Of the remaining SNPs, 91 were elsewhere on chromosome I, and 732 were distributed on the other chromosomes (chromosome 2 - 387; 3 - 49; 4 - 104; 5 - 113; 6 - 20; and 7 - 59). Thus, the SNP density in the <em>mat</em>-unlinked genome fraction was 0.02 SNPs/ kb (823 SNPs/ 39894 kb). The ratio 0.02/14 (0.14%) of SNP density in the <em>mat</em>-unlinked and <em>mat</em>-linked genome fractions agrees with the 0.1% estimate made above.</p><p id="p-30">As described in the Materials and Methods, construction of the ::<em>r<sup>B/S1</sup> A</em> and <em>a</em> strains entailed doing five additional backcrosses to the B/S1 background, that potentially could further reduce heterozygosity in ::<em>r1<sup>B/S1</sup> A</em> x ::<em>r1<sup>B/S1</sup> a</em> crosses relative to that in B/S1 <em>A</em> x B/S1 <em>a</em>. We identified a series of PCR-based Bichpuri-1 <em>vs.</em> Spurger-3 RFLP markers in the chromosome 1 <em>mat</em>-distal (d) and <em>mat</em>-proximal (p) regions and determined whether they were homozygous or heterozygous (designated, respectively, with subscripts i and j) in a ::<em>r3<sup>B/S1</sup> A</em> x ::<em>r3<sup>B/S1</sup> a</em>, B/S1 <em>a</em> x B/S1 <em>A</em>, or B/S1 <em>a</em> x ::<em>r3<sup>B/S1</sup>A</em> cross, until the d<sub>i</sub> - d<sub>j</sub> and p<sub>i</sub> - p<sub>j</sub> intervals were small enough to be sequenced from the ::<em>r3<sup>B/S1</sup> A</em>, ::<em>r3<sup>B/S1</sup> a</em>, B/S1 <em>A</em>, and B/S1 <em>a</em> genomic DNA. Alignment of the sequences revealed that the ::<em>r3<sup>B/S1</sup> a</em> x ::<em>r3<sup>B/S1</sup> A</em> and ::<em>r3<sup>B/S1</sup> a</em> x B/S1 <em>A</em> crosses were heterozygous for an ∼258,731 bp <em>mat</em>-linked genome segment, whereas the B/S1<em>a</em> x ::<em>r3<sup>B/S1</sup>A</em> and B/S1<em>a</em> x B/S1 <em>A</em> crosses were heterozygous for an ∼570,329 bp <em>mat</em>-linked segment (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, the “∼” indicates the presumed differences in nucleotide sequence between the OR and Bichpuri-1 or Spurger-3 genomes). These segments contain, respectively, 80 and 151 genes. To these values if we add the ∼44 kbp corresponding to heterozygosity in the <em>mat</em>-unlinked genome fraction, then the first two crosses are heterozygous for ∼303 kbp (0.7%), and the latter two are heterozygous for ∼614 kbp (1.4%), of the genome. The B/S1 x ::<em>r1<sup>B/S1</sup></em> crosses also are heterozygous for no more than 1.4% of the genome. The ∼570 kb <em>mat</em>-linked segment does not include the chromosome 1 gene <em>sad-1</em> (NCU 02178, nucleotides 969499 - 974473).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1487/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="B/S1 crosses are heterozygous in mat-linked region. Chromosome 1 is 9.8 Mb long and its nucleotides are numbered as per the OR genome sequence (ID- CM002236). In the B/S1 a and ::r3B/S1 a strains the mat a locus and flanking sequences are derived from the Bichpuri-1 a strain and shown in gray, whereas in the B/S1 A and ::r3B/S1 A strains the corresponding sequences, shown in black, are derived from Spurger-3 A. In B/S1 a and ::r3B/S1 a sequences distal (leftward) to, respectively, nucleotides corresponding to positions 1317180 and 1629710 in the OR genome sequence, and proximal (rightward) to the nucleotide corresponding to position 1888441, are identical to those in the B/S1 A sequence. Thus, the crosses ::r3B/S1 a x B/S1 A and ::r3B/S1 a x ::r3B/S1 A are heterozygous for a &#x223C;258,731 bp mat-linked segment, whereas the crosses B/S1 a x B/S1 A and B/S1 a x ::r3B/S1A are heterozygous for a &#x223C;570,329 bp mat-linked segment. Overall, the ::r3B/S1a x B/S1 A and B/S1 a x ::r3B/S1A crosses are heterozygous for, respectively, &#x223C;303 kbp (0.7%) and &#x223C;614 kbp (1.4%) of the genome (see text). Sequence accession numbers are MG009253 and MG017489 (Spurger), MG009254, MG017490 and MK392333 (B/S1 A); MK392332 (B/S1 a); MG009255 and MG017491 (::r3B/S1a), and MG009256 and MG017492 (Bichpuri)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-763544152" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;B/S1 crosses are heterozygous in &lt;em&gt;mat&lt;/em&gt;-linked region. Chromosome 1 is 9.8 Mb long and its nucleotides are numbered as per the OR genome sequence (ID- CM002236). In the B/S1 &lt;em&gt;a&lt;/em&gt; and ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt; a&lt;/em&gt; strains the &lt;em&gt;mat a&lt;/em&gt; locus and flanking sequences are derived from the Bichpuri-1 &lt;em&gt;a&lt;/em&gt; strain and shown in gray, whereas in the B/S1 &lt;em&gt;A&lt;/em&gt; and ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt; A&lt;/em&gt; strains the corresponding sequences, shown in black, are derived from Spurger-3 &lt;em&gt;A&lt;/em&gt;. In B/S1 &lt;em&gt;a&lt;/em&gt; and ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt; a&lt;/em&gt; sequences distal (leftward) to, respectively, nucleotides corresponding to positions 1317180 and 1629710 in the OR genome sequence, and proximal (rightward) to the nucleotide corresponding to position 1888441, are identical to those in the B/S1 &lt;em&gt;A&lt;/em&gt; sequence. Thus, the crosses ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt; a&lt;/em&gt; x B/S1 &lt;em&gt;A&lt;/em&gt; and ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt; a&lt;/em&gt; x ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt; A&lt;/em&gt; are heterozygous for a &#x223C;258,731 bp &lt;em&gt;mat&lt;/em&gt;-linked segment, whereas the crosses B/S1 &lt;em&gt;a&lt;/em&gt; x B/S1 &lt;em&gt;A&lt;/em&gt; and B/S1 &lt;em&gt;a&lt;/em&gt; x ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt;A&lt;/em&gt; are heterozygous for a &#x223C;570,329 bp &lt;em&gt;mat&lt;/em&gt;-linked segment. Overall, the ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt;a&lt;/em&gt; x B/S1 &lt;em&gt;A&lt;/em&gt; and B/S1 &lt;em&gt;a&lt;/em&gt; x ::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt;A&lt;/em&gt; crosses are heterozygous for, respectively, &#x223C;303 kbp (0.7%) and &#x223C;614 kbp (1.4%) of the genome (see text). Sequence accession numbers are MG009253 and MG017489 (Spurger), MG009254, MG017490 and MK392333 (B/S1 &lt;em&gt;A&lt;/em&gt;); MK392332 (B/S1 &lt;em&gt;a&lt;/em&gt;); MG009255 and MG017491 (::&lt;em&gt;r3&lt;sup&gt;B/S1&lt;/sup&gt;a&lt;/em&gt;), and MG009256 and MG017492 (Bichpuri).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1487/F1.medium.gif" width="440" height="144"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1487/F1.medium.gif" width="440" height="144"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1487/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1487/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492919" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">B/S1 crosses are heterozygous in <em>mat</em>-linked region. Chromosome 1 is 9.8 Mb long and its nucleotides are numbered as per the OR genome sequence (ID- CM002236). In the B/S1 <em>a</em> and ::<em>r3<sup>B/S1</sup> a</em> strains the <em>mat a</em> locus and flanking sequences are derived from the Bichpuri-1 <em>a</em> strain and shown in gray, whereas in the B/S1 <em>A</em> and ::<em>r3<sup>B/S1</sup> A</em> strains the corresponding sequences, shown in black, are derived from Spurger-3 <em>A</em>. In B/S1 <em>a</em> and ::<em>r3<sup>B/S1</sup> a</em> sequences distal (leftward) to, respectively, nucleotides corresponding to positions 1317180 and 1629710 in the OR genome sequence, and proximal (rightward) to the nucleotide corresponding to position 1888441, are identical to those in the B/S1 <em>A</em> sequence. Thus, the crosses ::<em>r3<sup>B/S1</sup> a</em> x B/S1 <em>A</em> and ::<em>r3<sup>B/S1</sup> a</em> x ::<em>r3<sup>B/S1</sup> A</em> are heterozygous for a ∼258,731 bp <em>mat</em>-linked segment, whereas the crosses B/S1 <em>a</em> x B/S1 <em>A</em> and B/S1 <em>a</em> x ::<em>r3<sup>B/S1</sup>A</em> are heterozygous for a ∼570,329 bp <em>mat</em>-linked segment. Overall, the ::<em>r3<sup>B/S1</sup>a</em> x B/S1 <em>A</em> and B/S1 <em>a</em> x ::<em>r3<sup>B/S1</sup>A</em> crosses are heterozygous for, respectively, ∼303 kbp (0.7%) and ∼614 kbp (1.4%) of the genome (see text). Sequence accession numbers are <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG009253&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG009253</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG017489&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG017489</a> (Spurger), <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG009254&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG009254</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG017490&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MG017490</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK392333&amp;atom=%2Fggg%2F9%2F5%2F1487.atom">MK392333</a> (B/S1 <em>A</em>); MK392332 (B/S1 <em>a</em>); MG009255 and MG017491 (::<em>r3<sup>B/S1</sup>a</em>), and MG009256 and MG017492 (Bichpuri).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>MSUD efficiency differs in the N. crassa OR and B/S1 backgrounds</h3><p id="p-32">Model 1 (see Introduction) predicts that MSUD would be more efficient in the B/S1 x ::<em>r<sup>B/S1</sup></em> crosses than in OR x ::<em>r<sup>B/S1</sup></em> or B/S1 x ::<em>r<sup>OR</sup></em>, since the latter crosses are surely more heterozygous. The results summarized in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Tables 1</a> and S4 allow us to reject this model. As can be seen in the Tables, four control crosses that did not contain any ::<em>r</em> transgene (OR <em>A</em> x OR <em>a</em>, B/S1 <em>A</em> x B/S1 <em>a</em>, OR <em>A</em> x B/S1 <em>a</em>, and OR <em>a</em> x B/S1 <em>A</em>) did not produce any round ascospores, although the B/S1 <em>A</em> x B/S1 <em>a</em> cross was much less productive than OR <em>A</em> x OR <em>a</em> and the two OR x B/S1 crosses. The ::<em>r3<sup>OR</sup></em> x OR crosses produced &gt; 96% round ascospores (Figure S2C, panel i), while the homozygous ::<em>r3<sup>OR</sup> a</em> x ::<em>r3<sup>OR</sup> A</em> cross yielded 1% round ascospores (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, Figure S2C, panel ii). This confirmed the efficient MSUD characteristic of the OR background. The two ::<em>r3<sup>B/S1</sup></em> x ::<em>r3<sup>OR</sup></em> crosses produced &lt; 4% round ascospores, showing that the MSUD machinery recognizes the ::<em>r3<sup>B/S1</sup></em> and ::<em>r3<sup>OR</sup></em> transgenes as alleles (paired). The ::<em>r3<sup>OR</sup></em> x B/S1 and ::<em>r3<sup>B/S1</sup></em> x OR crosses produced 25- 64% round ascospores (see Figure S2, panel iii for an example), consistent with the results of <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Ramakrishnan <em>et al.</em> (2011)</a> showing inefficient MSUD in crosses of the OR-derived testers with the progenitors of the B/S1 strains, namely, Bichpuri-1 and Spurger-3. Significantly, the ::<em>r3<sup>B/S1</sup></em> x B/S1 crosses produced &lt; 24% round ascospores, which was inconsistent with model 1. Additionally, the homozygous ::<em>r3<sup>B/S1</sup> a</em> x ::<em>r3<sup>B/S1</sup> A</em> cross produced ∼1.5 fold more round ascospores than the ::<em>r3<sup>B/S1</sup></em> x ::<em>r3<sup>OR</sup></em> crosses. Crosses with the ::<em>r1<sup>OR</sup></em> and ::<em>r1<sup>B/S1</sup></em> testers gave similar results (Table S4). Overall, the results show MSUD is less efficient in the B/S1 than the OR background, and there may be an increase in inappropriate silencing of paired genes in the B/S1 background. The ::<em>r<sup>OR</sup></em> x B/S1 and ::<em>r<sup>B/S1</sup></em> x OR crosses in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Tables 1</a> and S4 showed that the “efficient MSUD” phenotype of OR was recessive to the “inefficient MSUD” phenotype of B/S1.</p><p id="p-33">Next, we crossed 101 f<sub>1</sub> progeny from B/S1 x OR with ::<em>r3<sup>OR</sup></em> strains of the opposite mating type. Eleven crosses produced &gt; 90% round ascospores (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), suggesting that only ∼11% of the f<sub>1</sub> progeny had inherited the OR-type phenotype. Illumina whole genome sequences of the 11 OR-type strains, and of two control non-OR-types (whose crosses produced 40% and 29% round ascospores), revealed that all the OR-type progeny shared a ∼300 kb OR-derived chromosome 1 segment (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>), which was absent from the two non-OR-type control progeny. Further, a majority (8 and 9, respectively,) of the OR-type progeny also contained OR-derived segments of chromosomes 2 and 7. The segments might contain genes that contribute to the OR-type phenotype.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1487/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of f1 progeny from B/S1 A x OR a. (A) MSUD efficiency was measured as the fraction of round ascospores produced in crosses of 101 f1 progeny from B/S1 A x OR a with ::r3OR strains of opposite mating type, and 11 f1 progeny whose crosses produced &gt;90 round ascospores were deemed to have inherited the efficient MSUD phenotype of the OR a parent. (B) Illumina whole genome sequencing revealed an OR-derived chromosome 1 segment (coordinates 9.5-9.8 Mb) was conserved in all the 11 OR-type f1 progeny. Additionally, a majority (8 and 9, respectively,) of the OR-type f1 progeny also contained OR-derived segments of chromosome 2 (0.3-2.8 Mb) and 7 (0.2-3.1 Mb). The box brackets indicate the OR-type progeny from which the corresponding segments were absent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-763544152" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of f&lt;sub&gt;1&lt;/sub&gt; progeny from B/S1 &lt;em&gt;A&lt;/em&gt; x OR &lt;em&gt;a&lt;/em&gt;. (A) MSUD efficiency was measured as the fraction of round ascospores produced in crosses of 101 f&lt;sub&gt;1&lt;/sub&gt; progeny from B/S1 &lt;em&gt;A&lt;/em&gt; x OR &lt;em&gt;a&lt;/em&gt; with ::&lt;em&gt;r3&lt;sup&gt;OR&lt;/sup&gt;&lt;/em&gt; strains of opposite mating type, and 11 f&lt;sub&gt;1&lt;/sub&gt; progeny whose crosses produced &gt;90 round ascospores were deemed to have inherited the efficient MSUD phenotype of the OR &lt;em&gt;a&lt;/em&gt; parent. (B) Illumina whole genome sequencing revealed an OR-derived chromosome 1 segment (coordinates 9.5-9.8 Mb) was conserved in all the 11 OR-type f&lt;sub&gt;1&lt;/sub&gt; progeny. Additionally, a majority (8 and 9, respectively,) of the OR-type f&lt;sub&gt;1&lt;/sub&gt; progeny also contained OR-derived segments of chromosome 2 (0.3-2.8 Mb) and 7 (0.2-3.1 Mb). The box brackets indicate the OR-type progeny from which the corresponding segments were absent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1487/F2.medium.gif" width="440" height="299"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1487/F2.medium.gif" width="440" height="299"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1487/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1487/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493020" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Analysis of f<sub>1</sub> progeny from B/S1 <em>A</em> x OR <em>a</em>. (A) MSUD efficiency was measured as the fraction of round ascospores produced in crosses of 101 f<sub>1</sub> progeny from B/S1 <em>A</em> x OR <em>a</em> with ::<em>r3<sup>OR</sup></em> strains of opposite mating type, and 11 f<sub>1</sub> progeny whose crosses produced &gt;90 round ascospores were deemed to have inherited the efficient MSUD phenotype of the OR <em>a</em> parent. (B) Illumina whole genome sequencing revealed an OR-derived chromosome 1 segment (coordinates 9.5-9.8 Mb) was conserved in all the 11 OR-type f<sub>1</sub> progeny. Additionally, a majority (8 and 9, respectively,) of the OR-type f<sub>1</sub> progeny also contained OR-derived segments of chromosome 2 (0.3-2.8 Mb) and 7 (0.2-3.1 Mb). The box brackets indicate the OR-type progeny from which the corresponding segments were absent.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>MSUD is inefficient in N. tetrasperma 85</h3><p id="p-35">As a second test of the hypothesis that non-OR strains commonly exhibit inefficient MSUD we examined comparable crosses made in <em>N. tetrasperma</em> strain 85 (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Again, control crosses not containing the ::<em>r3<sup>Nt</sup></em> transgene (85 <em>A</em> x 85 <em>a</em>, 85 <em>A</em> x <em>Sad-2<sup>Nt</sup> a</em>, and <em>Sad-2<sup>Nt</sup> A</em> x 85 <em>a</em>) did not produce any round ascospores, and the <em>Sad-2<sup>Nt</sup> A</em> x <em>Sad-2<sup>Nt</sup> a</em> cross was infertile like its <em>N. crassa</em> counterpart. The ::<em>r3<sup>Nt</sup></em> x 85 crosses produced ∼5–7% round ascospores, which was far less than the &gt;96% seen in the cross ::<em>r3<sup>OR</sup></em> x OR. The <em>WT<sup>Nt</sup>A</em> strain, which contains a transgene-free chromosome 7, was used as an additional “wild-type” control and the ::<em>r3<sup>Nt</sup>a</em> x <em>WT<sup>Nt</sup>A</em> cross produced 21.5% round ascospores. The <em>WT<sup>Nt</sup>A</em> strain was a conidial derivative from the 6R dikaryon obtained during the introgression of the ::<em>r3<sup>Nt</sup></em> transgene from <em>N. crassa</em> OR into <em>N. tetrasperma</em> 85 (see Materials and methods).</p><p id="p-36">One ::<em>r3<sup>Nt</sup>A</em> x ::<em>r3<sup>Nt</sup> a</em> cross and the cross <em>Sad-2<sup>Nt</sup> A</em> x ::<em>r3<sup>Nt</sup> a</em> yielded &lt; 2% round ascospores, which suggested, respectively, that MSUD does not occur in a transgene-homozygous cross, and that it is suppressed by <em>Sad-2<sup>Nt</sup></em>. A second putative ::<em>r3<sup>Nt</sup>A</em> x ::<em>r3<sup>Nt</sup> a</em> cross, made using the strain CD #1 as the ::<em>r3<sup>Nt</sup>A</em> parent, yielded 11.3% round ascospores, which was unexpectedly high for a presumed ::<em>r3<sup>Nt</sup></em>–homozygous cross, and it was explored further as described in the next section.</p><p id="p-37">The crosses <em>Dp(EB4)<sup>Nt</sup>A</em> x 85 <em>a</em> and <em>Dp(IBj5)<sup>Nt</sup>a</em> x 85 <em>A</em> represent <em>Dp</em>-heterozygous crosses in the <em>N. tetrasperma</em> 85 background. Significantly, neither cross showed an obvious barren phenotype, and they were only quantitatively less productive than the control <em>T(EB4)<sup>Nt</sup>a</em> x 85 <em>A</em> and <em>T(IBj5)<sup>Nt</sup>a</em> x 85 <em>A</em> crosses. <em>Dp(EB4)<sup>Nt</sup>A</em> x 85 <em>a</em> and <em>T(EB4)<sup>Nt</sup>a</em> x 85 <em>A</em> produced, respectively, 8.2 × 10<sup>5</sup> and 12.4 × 10<sup>5</sup> ascospores, and <em>Dp(IBj5)<sup>Nt</sup>a</em> x 85 <em>A</em> and <em>T(IBj5)<sup>Nt</sup>a</em> x 85 <em>A</em> produced 1.3 × 10<sup>5</sup> and 7.1 × 10<sup>5</sup> ascospores. The <em>Sad-2<sup>Nt</sup></em> x <em>E</em> and <em>E</em> x <em>Sad-2<sup>+</sup></em> crosses did not appear to differ in ascus development (data not shown).</p></div><div id="sec-14" class="subsection"><h3>A self-sterile dikaryon recovered from a self-fertile strain</h3><p id="p-38">The CD #1 strain was used as the ::<em>r3<sup>Nt</sup>A</em> parent in the second of the two putative ::<em>r3<sup>Nt</sup>A</em> x ::<em>r3<sup>Nt</sup> a</em> crosses, but yielded an unexpectedly high fraction of round ascospores (see the previous section). CD #1 was a self-sterile conidial derivative obtained from the self-fertile strain 13R (see Materials and Methods), and so we considered the possibility that it was in fact an [(::<em>r3<sup>Nt</sup>A</em>) + (<em>+ A</em>)] dikaryon. In which case, its cross with the ::<em>r3<sup>Nt</sup> a</em> strain would represent two crosses, one of genotype (::<em>r3<sup>Nt</sup>A</em>) x ::<em>r3<sup>Nt</sup> a</em>, and the other of genotype (<em>+ A</em>) x ::<em>r3<sup>Nt</sup> a</em>, and the latter would produce most of the round ascospores. PCR with CD #1 genomic DNA and primers specific for either a transgene-bearing or transgene-free chromosome 7 revealed that such indeed was the case. We discuss below the likely provenance of a dikaryotic [(::<em>r3<sup>Nt</sup>A</em>) + (<em>+ A</em>)] conidial derivative from a self-fertile strain.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><div id="sec-16" class="subsection"><h3>Efficient MSUD is not necessarily the norm in Neurospora</h3><p id="p-39">MSUD in the <em>N. crassa</em> B/S1 and <em>N. tetrasperma</em> 85 backgrounds was less efficient than in the <em>N. crassa</em> OR background. In OR, MSUD in crosses heterozygous or homozygous for an ::<em>r</em> transgene was all-or-none (<em>i.e.</em>, &gt; 97% <em>vs.</em> &lt; 1% round ascospores), whereas in B/S1 and 85 the results were not as clear-cut, and were, respectively, 24% <em>vs.</em> 6%, and 7% <em>vs.</em> 1.4%. MSUD efficiency in B/S1 was comparable to that in OR crosses deficient for <em>sad-6<sup>+</sup></em>, which encodes a Rad54-like SNF2 helicase-related protein (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Samarajeewa <em>et al.</em> 2014</a>), or for <em>cbp-20<sup>+</sup></em> and <em>cbp-80<sup>+</sup></em>, that encode the cap-binding complex (CBC) proteins associated with the 5′ cap of eukaryotic mRNA transcripts, which interacts with the Argonaute protein SMS-2 in the meiotic silencing complex (MSC) (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Decker <em>et al.</em> 2017</a>). <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Decker <em>et al.</em> (2017)</a> have speculated that the absence of the CBC makes it harder for the MSC to recognize target mRNAs, allowing some to reach the translational machinery and thus reduces MSUD efficiency. Additionally, the inappropriate silencing of paired genes was apparently increased in B/S1. Thus, efficient MSUD, though characteristic of OR, is not necessarily the norm in <em>N. crassa</em>. The OR phenotype was recessive to the B/S1 phenotype in OR x B/S1 crosses. Possibly, extrinsic or intrinsic cues that modulate the MSUD response in B/S1 and <em>N. tetrasperma</em> 85 are absent from OR, thus making the response all-or-none. Alternatively, the B/S1 background might be null or hypomorphic for a locus encoding a protein, non-coding RNA, or small RNA that enhances the efficiency or half-life of masiRNA in the OR background. The presence of both OR- and B/S1-derived high and low efficiency alleles in an OR x B/S1 cross might result in MSUD of intermediate efficiency. The use of OR strains for genetic studies probably fortuitously facilitated (1) the discovery of MSUD and its semi-dominant <em>Sad</em> suppressors; (2) the discovery of ascus-dominant mutants such as <em>Ban</em>, <em>Dip-1</em>, <em>Pk</em><sup>D</sup>, and <em>R</em>; and (3) the identification of <em>Dp</em> strains via their barrenness in crosses. Efficient MSUD in OR enabled <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Turner and Perkins (1982)</a> to recover normal-sequence progeny from a cross between two chromosome 1 inversions (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kasbekar 2006</a>).</p><p id="p-40">The 2532 bp OR-derived <em>r</em> gene segment used to construct the ::<em>r</em> transgenes in the B/S1 strains differed by 16 SNPs from the corresponding B/S1 sequence, thus creating a ∼0.63% heterozygosity in ::<em>r3<sup>B/S1</sup></em> x B/S1 crosses between the source of the masiRNA and their target mRNAs. We think this is too little to account for the reduced MSUD seen in the B/S1 background for the following reason. In <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>, crosses 10, 11, 14, and 15, one <em>r<sup>+</sup></em> allele targeted for MSUD is from OR and the other is from B/S1, and the round ascospore fractions ranged between 25–56%. Additionally, one OR-type f1 progeny from OR <em>a</em> x B/S1 <em>A</em> contained the B/S1 <em>r<sup>+</sup></em> allele, and its cross with the ::<em>r3<sup>OR</sup></em> tester gave 91% round ascospores. The 25–91% range in these crosses must have causes extrinsic to the sequence difference between the masiRNA and its target mRNAs. Similar results were obtained with the ::<em>r1</em> transgene (Table S4). Hence the difference in round ascospore fraction in ::<em>r<sup>OR</sup></em> x OR <em>vs.</em> ::<em>r<sup>B/S1</sup></em> x B/S1 (&gt;95% <em>vs.</em> &lt;25%) is unlikely to be wholly intrinsic to the sequence difference in the latter cross. Moreover, the <em>r</em> locus was outside the ∼300 kb OR-derived chromosome 1 segment found conserved in all 11 f<sub>1</sub> progeny that showed the OR phenotype in crosses with the ::<em>r<sup>OR</sup></em> testers. However, a conclusive demonstration would require re-doing the ::<em>r<sup>B/S1</sup></em> x B/S1 crosses with a B/S1-derived ::<em>r</em> transgene.</p></div><div id="sec-17" class="subsection"><h3>Genetic difference between OR and B/S1 strains</h3><p id="p-41">Of 101 f<sub>1</sub> progeny examined from B/S1 x OR, 11 had inherited the OR-type phenotype and all of them contained an OR-derived ∼300 kb chromosome 1 segment (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The segment was absent from two control non-OR-type f<sub>1</sub> progeny. Two non-mutually exclusive interpretations are compatible with these results. One, since fewer than 50% of the f<sub>1</sub> progeny showed the OR-type phenotype, the OR-derived chromosome 1 segment might be necessary but not sufficient for efficient MSUD. In other words, the phenotype is complex. Also since a majority (8 or 9) of the OR-type progeny also contained OR-derived segments of chromosomes 2 and 7 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>), the latter segments might contain additional genes that contribute to the OR phenotype. The other interpretation posits that the OR-derived chromosome 1 segment is both necessary and (at least largely) sufficient for the OR phenotype, <em>i.e.</em>, the phenotype is not complex, but a transmission ratio distortion (TRD) skews its inheritance into fewer than 50% f<sub>1</sub> progeny. A preliminary experiment comparing the fraction of f<sub>1</sub> progeny containing the OR <em>vs.</em> B/S1 “allele” revealed a 14:39 ratio, suggesting a TRD disfavoring the OR-derived segment (D. A. Giri and D. P. Kasbekar, unpublished results). We do not yet know whether the TRD and the OR-type phenotype are mechanistically related or are merely genetically linked.</p><p id="p-42">The chromosome 7 candidate segment includes the <em>sad-5</em> gene (1,640,838-1,642,206) Crosses null for <em>sad-5</em> are completely deficient in MSUD (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hammond <em>et al.</em> 2013a</a>). However, it is premature at this time to focus on individual genes, since each candidate segment contains hundreds of other loci. It will be necessary to first narrow down the candidate segments, say by using OR and B/S1 parents with flanking auxotrophic markers, select for prototrophic crossover progeny, and examine the OR-type among them to localize the relevant genes.</p><p id="p-43">Some of the MSUD efficiency difference between the OR, B/S1, and <em>N. tetrasperma</em> 85 backgrounds might arise from differences in sexual-stage-specific A-to-I mRNA editing of MSUD gene transcripts. The editing converts specific adenosine residues (A) in mRNA to inosine (I), and since I is recognized as guanosine (G), therefore the effect is similar to an A-to-G substitution. Of 24,001 editing sites found in <em>N. crassa</em> genes, 51.6% were conserved in their <em>N. tetrasperma</em> orthologs (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Liu <em>et al.</em> 2016</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">2017</a>). In OR, the MSUD genes <em>sad-1</em>, <em>-2</em>, <em>-4</em>, <em>-5</em>, and <em>-6</em>, <em>sms-2</em>, <em>dcl-1</em>/<em>sms-3</em>, and <em>qip</em> contain multiple non-synonymous editing sites, although <em>sad-3</em> has only one.</p></div><div id="sec-18" class="subsection"><h3>Efficient MSUD suppresses Dp-mediated RIP-suppression</h3><p id="p-44">In general, <em>Dp</em>-heterozygous crosses in the OR background display an MSUD-dependent barren phenotype (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Perkins 1997</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Shiu <em>et al.</em> 2001</a>). Most genetic studies on <em>Dp</em>s have used the OR background, hence it was widely assumed that <em>all</em> Neurospora <em>Dp</em>-heterozygous crosses are barren, notwithstanding the fact that before the advent of the <em>Sad</em> suppressors we had used wild-isolated strains (<em>N</em>) to increase progeny numbers in <em>Dp</em> x <em>N</em> crosses (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fehmer <em>et al.</em> 2001</a>). Now, our results revealed that <em>T</em>- and <em>Dp</em>-heterozygous crosses are comparably productive in <em>N. tetrasperma</em> 85, which suggests that inefficient MSUD can increase the productivity of <em>Dp</em>-heterozygous crosses.</p><p id="p-45">Further, large <em>Dp</em>s (&gt; 300 kbp) also can act as dominant suppressors of RIP, possibly because the <em>Dp</em>s titrate out the RIP machinery (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bhat and Kasbekar 2001</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vyas <em>et al.</em> 2006</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Perkins <em>et al.</em> 2007</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Singh and Kasbekar 2008</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Singh <em>et al.</em> 2009</a>). <em>Dp</em>-mediated RIP suppression might be significant when <em>Dp</em>-heterozygous crosses are productive. Previous work in our laboratory had identified seven wild-isolated dominant RIP suppressor strains (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Noubissi <em>et al.</em> 2001</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bhat <em>et al.</em> 2003</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Vyas <em>et al.</em> 2006</a>). It is possible that some of the RIP suppressors were <em>Dp</em> strains in inefficient MSUD backgrounds.</p></div><div id="sec-19" class="subsection"><h3>Does MSUD underlie the ascus-dominant E phenotype?</h3><p id="p-46">We did not find any difference in ascus development in the <em>Sad-2<sup>Nt</sup></em> x <em>E</em> and <em>E</em> x <em>Sad-2<sup>+</sup></em> crosses, which suggested that the ascus-dominant <em>E</em> phenotype does not involve MSUD. This presupposes that the <em>Sad-2</em> mutation effectively silences <em>sad-2</em><sup>+</sup> in the 85 background. This supposition is supported by our results that ::<em>r3<sup>Nt</sup></em> x 85 and <em>Sad-2<sup>Nt</sup> A</em> x ::<em>r3<sup>Nt</sup> a</em> produced, respectively, 5–7% and &lt; 2% round ascospores. A more definitive experiment would be to test the <em>E</em> mutant phenotype in <em>N. tetrasperma</em> crosses homozygous for <em>sad-4Δ</em> or <em>sad-5Δ</em>. The SAD-4 and SAD-5 proteins are essential for MSUD in <em>N. crassa</em>, but unlike SAD-2, they are dispensable for ascus development (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Hammond <em>et al.</em> 2013a</a>). <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jacobson (1995)</a> had found that certain combinations of intercrosses between single mating-type conidial derivatives derived from wild-collected <em>N. tetrasperma</em> heterokaryons produced mostly eight-spored asci when one derivative was used as the female (protoperithecial) parent and mostly four-spored asci when the derivatives were reversed in the reciprocal cross, which suggests that eight-spored asci can form independently of MSUD in <em>N. tetrasperma</em>.</p></div><div id="sec-20" class="subsection"><h3>Origin of the CD #1 conidial derivative</h3><p id="p-47">The self-sterile conidial derivative CD #1 was derived from the <em>N. tetrasperma</em> self-fertile strain 13R. Initially its genotype was assumed to be ::<em>r3<sup>Nt</sup>A</em>, but it turned out to be a [(::<em>r3<sup>Nt</sup>A</em>) + (<em>+ A</em>)] dikaryon. Therefore the 13R strain was likely a trikaryon of genotype [(::<em>r3<sup>Nt</sup>A</em>) + (<em>+ A</em>) + (<em>+ a</em>)] or [(::<em>r3<sup>Nt</sup>A</em>) + (<em>+ A</em>) + (::<em>r3<sup>Nt</sup> a</em>)]. When self-crossed, these trikaryons can yield MSUD-induced round-spored asci. The trikaryotic ascospore can form if the ::<em>r3<sup>Nt</sup></em> locus underwent second-division segregation, and one or more nucleus underwent an additional mitosis subsequent to the post-meiotic mitosis but prior to ascospore partitioning. Crosses involving hybrid strains obtained by introgressing <em>N. crassa</em> translocations into <em>N. tetrasperma</em> occasionally underwent such additional mitoses, and the pre-partition ascus can contain more than eight nuclei (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kasbekar and Rekha 2017</a>), whose partitioning into four ascospores results in some containing more than two nuclei. The CD #1 strain’s genotype cautioned us to re-confirm the genotype of all strains thought to be homokaryotic in this work.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank Ms. S. Rekha for assistance with some of the crosses, Ms. Parijat for processing DNA samples for whole genome sequencing, and Dr. Shrish Tiwari for useful discussions. Two anonymous reviewers gave excellent suggestions to improve the manuscript. DAG was supported by CSIR-UGC Junior and Senior Research Fellowships. AVP was supported by a University Grants Commission Fellowship. KTN was supported by a Wellcome Trust-DBT India Alliance Intermediate Fellowship (IA/I/11/2500268) and by Indian Institute of Science Education and Research Thiruvananthapuram intramural funds. This work was supported in part by a grant from the SERB, Department of Science and Technology, Government of India, and by CDFD Core Funds to DPK during his tenure as Haldane Chair in the Centre for DNA Fingerprinting and Diagnostics (CDFD, July 2, 2012-July 1, 2017). DPK is an Indian National Science Academy (INSA) Senior Scientist in the CDFD, and Honorary Visiting Scientist in the Centre for Cellular and Molecular Biology (CCMB), and thanks the CDFD and CCMB for their hospitality. This paper is dedicated to the memory of Avinash N. Bhisey (January 5, 1938 – July 9, 2018), an excellent cancer cell biologist and institution builder, who made us proud as both family and friend.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-49">Supplemental material available at Figshare: <a href="https://doi.org/10.6084/m9.figshare.7570934">https://doi.org/10.6084/m9.figshare.7570934</a>.</p></li><li class="fn" id="fn-4"><p id="p-50"><em>Communicating editor: M. Sachs</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-14"><span class="received-label">Received </span>January 14, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-04"><span class="accepted-label">Accepted </span>March 4, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Giri <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.1" data-doi="10.1016/S0092-8674(00)80081-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aramayo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Meiotic transvection in fungi.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)80081-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAramayo%26rft.auinit1%253DR.%26rft.volume%253D86%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D113%26rft.atitle%253DMeiotic%2Btransvection%2Bin%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980081-1%26rft_id%253Dinfo%253Apmid%252F8689677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80081-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8689677&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UX93400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Escape from repeat-induced point mutation of a gene-sized duplication in <em>Neurospora crassa</em> crosses that are heterozygous for a larger chromosome segment duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBhat%26rft.auinit1%253DA.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1581%26rft.epage%253D1590%26rft.atitle%253DEscape%2BFrom%2BRepeat-Induced%2BPoint%2BMutation%2Bof%2Ba%2BGene-Sized%2BDuplication%2Bin%2BNeurospora%2Bcrassa%2BCrosses%2BThat%2BAre%2BHeterozygous%2Bfor%2Ba%2BLarger%2BChromosome%2BSegment%2BDuplication%26rft_id%253Dinfo%253Apmid%252F11290714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE1ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noubissi</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic analysis of wild-isolated <em>Neurospora crassa</em> strains identified as dominant suppressors of repeat-induced point mutation (RIP).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">947</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBhat%26rft.auinit1%253DA.%26rft.volume%253D164%26rft.issue%253D3%26rft.spage%253D947%26rft.epage%253D961%26rft.atitle%253DGenetic%2BAnalysis%2Bof%2BWild-Isolated%2BNeurospora%2Bcrassa%2BStrains%2BIdentified%2Bas%2BDominant%2BSuppressors%2Bof%2BRepeat-Induced%2BPoint%2BMutation%26rft_id%253Dinfo%253Apmid%252F12871906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzMvOTQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.4" data-doi="10.1534/genetics.103.025171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamuli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic transformation of <em>Neurospora tetrasperma</em>, demonstration of repeat-induced point mutation (RIP) in self-crosses, and a screen for recessive RIP-defective mutants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1155</span>–<span class="cit-lpage">1164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.103.025171</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBhat%26rft.auinit1%253DA.%26rft.volume%253D167%26rft.issue%253D3%26rft.spage%253D1155%26rft.epage%253D1164%26rft.atitle%253DGenetic%2BTransformation%2Bof%2BNeurospora%2Btetrasperma%252C%2BDemonstration%2Bof%2BRepeat-Induced%2BPoint%2BMutation%2B%2528RIP%2529%2Bin%2BSelf-Crosses%2Band%2Ba%2BScreen%2Bfor%2BRecessive%2BRIP-Defective%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.025171%26rft_id%253Dinfo%253Apmid%252F15280231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny8zLzExNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.5" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina Sequence Data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calhoun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">H. B.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1968</span> <span class="cit-article-title">Genetic analysis of eight-spored asci produced by gene <em>E</em> in <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCalhoun%26rft.auinit1%253DF.%26rft.volume%253D60%26rft.issue%253D3%26rft.spage%253D449%26rft.epage%253D459%26rft.atitle%253DGENETIC%2BANALYSIS%2BOF%2BEIGHT-SPORED%2BASCI%2BPRODUCED%2BBY%2BGENE%2BE%2BIN%2BNEUROSPORA%2BTETRASPERMA%26rft_id%253Dinfo%253Apmid%252F5728739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2MC8zLzQ0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweigard</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valent</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Improved vectors for selecting resistance to hygromycin.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D41%26rft.spage%253D22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.8" data-doi="10.1016/0076-6879(71)17168-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Serres</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Genetic and microbiological research techniques for <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0076-6879(71)17168-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D17%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252871%252917168-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(71)17168-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.9" data-doi="10.1534/g3.116.038679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vierling</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shanker</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The nuclear cap-binding complex mediates meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.038679</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.038679%26rft_id%253Dinfo%253Apmid%252F28179391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNC8xMTQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.10" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.11" data-doi="10.1093/oxfordjournals.jhered.a104632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodge</span>  <span class="cit-name-given-names">B. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1939</span> <span class="cit-article-title">A new dominant lethal in Neurospora. The <em>E</em> locus in <em>N. tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">474</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.jhered.a104632</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D30%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a104632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.jhered.a104632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fehmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noubissi</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Wild-isolated <em>Neurospora crassa</em> strains that increase fertility of crosses with segmental aneuploids used to establish that a large duplication suppresses RIP in a smaller duplication.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D48%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.13" data-doi="10.1534/g3.115.017616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giri</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rekha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Neurospora heterokaryons with complementary duplications and deficiencies in their constituent nuclei provide an approach to identify nucleus-limited genes.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.017616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.017616%26rft_id%253Dinfo%253Apmid%252F25897010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvNi8xMjYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.14" data-doi="10.1016/bs.adgen.2016.11.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Sixteen years of meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/bs.adgen.2016.11.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.adgen.2016.11.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.adgen.2016.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.15" data-doi="10.1534/genetics.112.148999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Novel proteins required for meiotic silencing by unpaired DNA and siRNA generation in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.148999</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.148999%26rft_id%253Dinfo%253Apmid%252F23502675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk0LzEvOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.16" data-doi="10.1534/genetics.112.149138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spollen</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Identification of small RNAs associated with meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.149138</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.149138%26rft_id%253Dinfo%253Apmid%252F23502674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzEvMjc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.17" data-doi="10.2307/3760805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Sexual dysfunction associated with outcrossing in <em>Neurospora tetrasperma</em>, a pseudohomothallic ascomycete.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">604</span>–<span class="cit-lpage">617</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00275514.1995.12026577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.volume%253D87%26rft.spage%253D604%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3760805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3760805&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TD67900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.18" data-doi="10.1016/j.fgb.2007.09.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evidence for the absence of meiotic silencing by unpaired DNA in <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">362</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2007.09.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DJacobson%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D351%26rft.epage%253D362%26rft.atitle%253DEvidence%2Bfor%2Bthe%2Babsence%2Bof%2Bmeiotic%2Bsilencing%2Bby%2Bunpaired%2BDNA%2Bin%2BNeurospora%2Btetrasperma.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2007.09.014%26rft_id%253Dinfo%253Apmid%252F18023372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2007.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18023372&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.19" data-doi="10.1007/BF02705229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Magic with moulds: Meiotic and mitotic crossing over in Neurospora inversions and duplications.</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02705229</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiosciences%26rft.stitle%253DJ%2BBiosci%26rft.aulast%253DKasbekar%26rft.auinit1%253DD.%2BP.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D4%26rft.atitle%253DMagic%2Bwith%2Bmoulds%253A%2Bmeiotic%2Band%2Bmitotic%2Bcrossing%2Bover%2Bin%2BNeurospora%2Binversions%2Band%2Bduplications.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02705229%26rft_id%253Dinfo%253Apmid%252F16595868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02705229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16595868&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1487.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Neurospora duplications, and genome defense by RIP and meiotic silencing</span>, pp. <span class="cit-fpage">109</span>–<span class="cit-lpage">127</span> in <span class="cit-source">Neurospora: Genomics and Molecular Biology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span>, <span class="cit-ed"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">Caister Academic Press</span>, <span class="cit-publ-loc">Norfolk, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.21" data-doi="10.1007/s12038-017-9671-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rekha</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title"><em>Neurospora tetrasperma</em> crosses heterozygous for hybrid translocation strains produce rare eight spored asci bearing heterokaryotic ascospores.</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12038-017-9671-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiosci.%26rft.volume%253D42%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12038-017-9671-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12038-017-9671-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.22" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.23" data-doi="10.1101/gr.199877.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide A-to-I RNA editing in fungi independent of ADAR enzymes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">509</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.199877.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.199877.115%26rft_id%253Dinfo%253Apmid%252F26934920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI2LzQvNDk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.24" data-doi="10.1073/pnas.1702591114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A-to-I RNA editing is developmentally regulated and generally adaptive for sexual reproduction in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E7756</span>–<span class="cit-lpage">E7765</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1702591114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1702591114%26rft_id%253Dinfo%253Apmid%252F28847945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzM3L0U3NzU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.25" data-doi="10.1007/s12038-010-0014-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiest</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plamann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Fungal Genetics Stock Center: a repository for 50 years of fungal genetics research.</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12038-010-0014-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiosciences%26rft.stitle%253DJ%2BBiosci%26rft.aulast%253DMcCluskey%26rft.auinit1%253DK.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D126%26rft.atitle%253DThe%2BFungal%2BGenetics%2BStock%2BCenter%253A%2Ba%2Brepository%2Bfor%2B50%2Byears%2Bof%2Bfungal%2Bgenetics%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12038-010-0014-6%26rft_id%253Dinfo%253Apmid%252F20413916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12038-010-0014-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20413916&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277022500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Vogel’s Medium N salts: avoiding the need for ammonium nitrate.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D50%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahlgren</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Hybrid strains useful in transferring genes from one species of Neurospora to another.</span> <abbr class="cit-jnl-abbrev">Neurospora Newslett.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurospora%2BNewslett.%26rft.volume%253D15%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.28" data-doi="10.1016/j.fgb.2013.05.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagasowjanya</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kranthi Raj</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sreethi Reddy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An apparent increase in meiotic silencing strength in crosses involving inbred <em>Neurospora crassa</em> strains.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2013.05.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D56%26rft.spage%253D158%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2013.05.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2013.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.29" data-doi="10.1073/pnas.0402780101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ninomiya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Highly efficient gene replacements in Neurospora strains deficient for nonhomologous end-joining.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">12248</span>–<span class="cit-lpage">12253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0402780101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0402780101%26rft_id%253Dinfo%253Apmid%252F15299145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzMzLzEyMjQ4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.30" data-doi="10.1006/fgbi.2000.1235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noubissi</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Repeat-induced point mutation (RIP) in crosses with wild-isolated strains of <em>Neurospora crassa</em>: Evidence for dominant reduction of RIP.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/fgbi.2000.1235</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D31%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ffgbi.2000.1235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/fgbi.2000.1235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.31" data-doi="10.1016/S0065-2660(08)60311-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Chromosome rearrangements in Neurospora and other filamentous fungi.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bgenetics%26rft.stitle%253DAdv%2BGenet%26rft.aulast%253DPerkins%26rft.auinit1%253DD.%2BD.%26rft.volume%253D36%26rft.spage%253D239%26rft.epage%253D398%26rft.atitle%253DChromosome%2Brearrangements%2Bin%2BNeurospora%2Band%2Bother%2Bfilamentous%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960311-9%26rft_id%253Dinfo%253Apmid%252F9348657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60311-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9348657&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Wild type <em>Neurospora crassa</em> strains preferred for use as standards.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D51%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.33" data-doi="10.1534/genetics.106.065714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Shrode</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recurrent locus-specific mutation resulting from a cryptic ectopic insertion in Neurospora.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">527</span>–<span class="cit-lpage">544</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.065714</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPerkins%26rft.auinit1%253DD.%2BD.%26rft.volume%253D175%26rft.issue%253D2%26rft.spage%253D527%26rft.epage%253D544%26rft.atitle%253DRecurrent%2BLocus-Specific%2BMutation%2BResulting%2BFrom%2Ba%2BCryptic%2BEctopic%2BInsertion%2Bin%2BNeurospora%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.065714%26rft_id%253Dinfo%253Apmid%252F17322355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzIvNTI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.34" data-doi="10.1534/genetics.104.031526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aramayo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">DNA methylation affects meiotic trans-sensing, not meiotic silencing, in Neurospora.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1925</span>–<span class="cit-lpage">1935</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.031526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPratt%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D1925%26rft.epage%253D1935%26rft.atitle%253DDNA%2BMethylation%2BAffects%2BMeiotic%2Btrans-sensing%252C%2BNot%2BMeiotic%2BSilencing%252C%2Bin%2BNeurospora%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.031526%26rft_id%253Dinfo%253Apmid%252F15611165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzE5MjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.35" data-doi="10.1016/S0953-7562(09)80923-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Functional heterothallism resulting from homokaryotic conidia and ascospores in <em>Neurospora tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Mycol. Res.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0953-7562(09)80923-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycol.%2BRes.%26rft.volume%253D96%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0953-7562%252809%252980923-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0953-7562(09)80923-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.36" data-doi="10.1002/dvg.1020150111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Diverse programs of ascus development in pseudohomothallic species of Neurospora, Gelasinospora, and Podospora.</span> <abbr class="cit-jnl-abbrev">Dev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/dvg.1020150111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bgenetics%26rft.stitle%253DDev%2BGenet%26rft.aulast%253DRaju%26rft.auinit1%253DN.%2BB.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D104%26rft.epage%253D118%26rft.atitle%253DDiverse%2Bprograms%2Bof%2Bascus%2Bdevelopment%2Bin%2Bpseudohomothallic%2Bspecies%2Bof%2BNeurospora%252C%2BGelasinospora%252C%2Band%2BPodospora.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.1020150111%26rft_id%253Dinfo%253Apmid%252F8187347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.1020150111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8187347&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MZ20400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.37" data-doi="10.1534/genetics.106.069161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">P. K. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Neurospora Spore killers <em>Sk-2</em> and <em>Sk-3</em> suppress meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.069161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.069161%26rft_id%253Dinfo%253Apmid%252F17339226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc2LzEvNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.38" data-doi="10.1016/j.fgb.2011.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sowjanya</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Meiotic silencing by unpaired DNA is expressed more strongly in the early than the late perithecia of crosses involving most wild-isolated <em>Neurospora crassa</em> strains and in self-crosses of <em>N. tetrasperma</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1146</span>–<span class="cit-lpage">1152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2011.10.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DRamakrishnan%26rft.auinit1%253DM.%26rft.volume%253D48%26rft.issue%253D12%26rft.spage%253D1146%26rft.epage%253D1152%26rft.atitle%253DMeiotic%2Bsilencing%2Bby%2Bunpaired%2BDNA%2Bis%2Bexpressed%2Bmore%2Bstrongly%2Bin%2Bthe%2Bearly%2Bthan%2Bthe%2Blate%2Bperithecia%2Bof%2Bcrosses%2Binvolving%2Bmost%2Bwild-isolated%2BNeurospora%2Bcrassa%2Bstrains%2Band%2Bin%2Bself-crosses%2Bof%2BN.%2Btetrasperma.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2011.10.002%26rft_id%253Dinfo%253Apmid%252F22056520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2011.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22056520&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.39" data-doi="10.1534/genetics.114.168187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samarajeewa</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauls</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient detection of unpaired DNA requires a homolog from the Rad54-like family of homologous recombination proteins.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.168187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.168187%26rft_id%253Dinfo%253Apmid%252F25146971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvODk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.40" data-doi="10.1534/g3.117.041848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samarajeewa</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manitchotpisit</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rehard</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An RNA recognition motif-containing protein functions in meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2871</span>–<span class="cit-lpage">2882</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.041848</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041848%26rft_id%253Dinfo%253Apmid%252F28667016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOC8yODcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.41" data-doi="10.1146/annurev.ge.24.120190.003051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Premeiotic instability of repeated sequences in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">613</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ge.24.120190.003051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DSelker%26rft.auinit1%253DE.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D579%26rft.epage%253D613%26rft.atitle%253DPremeiotic%2BInstability%2BOf%2BRepeated%2BSequences%2BIn%2BNeurospora%2BCrassa%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.24.120190.003051%26rft_id%253Dinfo%253Apmid%252F2150906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.24.120190.003051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2150906&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EN96900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">P. K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Meiotic silencing by unpaired DNA: properties, regulation, and suppression.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1483</span>–<span class="cit-lpage">1495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShiu%26rft.auinit1%253DP.%2BK.%2BT.%26rft.volume%253D161%26rft.issue%253D4%26rft.spage%253D1483%26rft.epage%253D1495%26rft.atitle%253DMeiotic%2BSilencing%2Bby%2BUnpaired%2BDNA%253A%2BProperties%252C%2BRegulation%2Band%2BSuppression%26rft_id%253Dinfo%253Apmid%252F12196394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MS80LzE0ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.43" data-doi="10.1016/S0092-8674(01)00609-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">P. K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">916</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(01)00609-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DShiu%26rft.auinit1%253DP.%2BK.%26rft.volume%253D107%26rft.issue%253D7%26rft.spage%253D905%26rft.epage%253D916%26rft.atitle%253DMeiotic%2Bsilencing%2Bby%2Bunpaired%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900609-2%26rft_id%253Dinfo%253Apmid%252F11779466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00609-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11779466&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173024200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.44" data-doi="10.1073/pnas.0508896103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">P. K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruprich-Robert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">SAD-2 is required for meiotic silencing by unpaired DNA and perinuclear localization of SAD-1 RNA-directed RNA polymerase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">2243</span>–<span class="cit-lpage">2248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0508896103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508896103%26rft_id%253Dinfo%253Apmid%252F16461906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAzLzcvMjI0MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNDg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.45" data-doi="10.1007/s10709-007-9234-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Titration of repeat-induced point mutation (RIP) by chromosome segment duplications in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">275</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-007-9234-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DSingh%26rft.auinit1%253DP.%2BK.%26rft.volume%253D134%26rft.issue%253D3%26rft.spage%253D267%26rft.epage%253D275%26rft.atitle%253DTitration%2Bof%2Brepeat-induced%2Bpoint%2Bmutation%2B%2528RIP%2529%2Bby%2Bchromosome%2Bsegment%2Bduplications%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-007-9234-6%26rft_id%253Dinfo%253Apmid%252F18046508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-007-9234-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18046508&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.46" data-doi="10.1002/bies.200800098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Chromosome segment duplications in <em>Neurospora crassa</em>: barren crosses beget fertile science.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">219</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.200800098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DSingh%26rft.auinit1%253DP.%2BK.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D209%26rft.epage%253D219%26rft.atitle%253DChromosome%2Bsegment%2Bduplications%2Bin%2BNeurospora%2Bcrassa%253A%2Bbarren%2Bcrosses%2Bbeget%2Bfertile%2Bscience.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.200800098%26rft_id%253Dinfo%253Apmid%252F19204993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.200800098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19204993&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.47" data-doi="10.1017/S0016672300019042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Conventional and unconventional analysis of an inversion in Neurospora.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300019042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DTurner%26rft.auinit1%253DB.%2BC.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D175%26rft.epage%253D190%26rft.atitle%253DConventional%2Band%2Bunconventional%2Banalysis%2Bof%2Ban%2Binversion%2Bin%2BNeurospora.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300019042%26rft_id%253Dinfo%253Apmid%252F7152257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300019042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7152257&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1487.48" data-doi="10.1002/0471250953.bi1110s43"><div class="cit-metadata"><cite>Van der Auwera G. A., M. Carneiro, C. Hartl, R. Poplin, G. del Angel <em>et al.</em>, 2013 From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline. Current Protocols in Bioinformatics 43: 11.10. <span class="cit-fpage">1</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471250953.bi1110s43</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi1110s43%26rft_id%253Dinfo%253Apmid%252F25431634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi1110s43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25431634&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.49" data-doi="10.1534/genetics.105.050468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vyas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravindran</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasbekar</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome segment duplications in <em>Neurospora crassa</em> and their effects on repeat-induced point mutation (RIP) and meiotic silencing by unpaired DNA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1511</span>–<span class="cit-lpage">1519</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.050468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.050468%26rft_id%253Dinfo%253Apmid%252F16361235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE1MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1487.50" data-doi="10.1016/j.fgb.2004.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamari</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Domínguez</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Double-joint PCR: a PCR-based molecular tool for gene manipulations in filamentous fungi.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">981</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2004.08.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DYu%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D41%26rft.issue%253D11%26rft.spage%253D973%26rft.epage%253D981%26rft.atitle%253DDouble-joint%2BPCR%253A%2Ba%2BPCR-based%2Bmolecular%2Btool%2Bfor%2Bgene%2Bmanipulations%2Bin%2Bfilamentous%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2004.08.001%26rft_id%253Dinfo%253Apmid%252F15465386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2004.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15465386&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1487.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226263600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1487.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1481" title="Locating and Characterizing a Transgene Integration Site by Nanopore Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1497" title="Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1487" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-q4hUzUEUIHgnf2P3MeChSupgkXh4hG3tGy_2zCPTefI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1487" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1487" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1487" class="active" data-icon-position="" data-hide-link-title="0">The Neurospora crassa Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492877" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-eeIlm7y93lPH8hmjdDkTqrnsrUJAt74yRpZVQG3WQE0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1487.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1487" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1a52rCp4bEMTMz6VdWJVCUGO6KuEXYK71iTBIwg8AAQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1487" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1487.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492877" id="node492877--2" data-pisa="ggg;9/5/1487" data-pisa-master="ggg;g3.119.400006" data-apath="/ggg/9/5/1487.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The <em>Neurospora crassa</em> Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dev Ashish Giri</span>, <span class="highwire-citation-author" data-delta="1">Ajith V. Pankajam</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4342-7501" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Koodali T. Nishant</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Durgadas P. Kasbekar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1487-1496;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400006 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492877--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dev Ashish Giri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500039, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Studies, Manipal University, Manipal 576104, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BAshish%2BGiri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giri%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BAshish%2BGiri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biology</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Computation Modelling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram 695551, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4342-7501" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Koodali T. Nishant</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Durgadas P. Kasbekar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500039, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Durgadas%2BP.%2BKasbekar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasbekar%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADurgadas%2BP.%2BKasbekar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Durgadas P. Kasbekar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kas@cdfd.org.in" class="" data-icon-position="" data-hide-link-title="0">kas@cdfd.org.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1487" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1487&amp;title=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1487&amp;title=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1487&amp;title=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1487&amp;text=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1487&amp;title=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1487&amp;t=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1487&amp;title=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1487&amp;title=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492877/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492877" id="node492877--3" data-pisa="ggg;9/5/1487" data-pisa-master="ggg;g3.119.400006" data-apath="/ggg/9/5/1487.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The <em>Neurospora crassa</em> Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dev Ashish Giri</span>, <span class="highwire-citation-author" data-delta="1">Ajith V. Pankajam</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4342-7501" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Koodali T. Nishant</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Durgadas P. Kasbekar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1487-1496;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400006 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492877--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dev Ashish Giri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500039, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Graduate Studies, Manipal University, Manipal 576104, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BAshish%2BGiri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giri%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BAshish%2BGiri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ajith V. Pankajam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ajith%2BV.%2BPankajam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pankajam%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAjith%2BV.%2BPankajam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biology</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Computation Modelling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram 695551, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4342-7501" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Koodali T. Nishant</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Durgadas P. Kasbekar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for DNA Fingerprinting and Diagnostics, Hyderabad 500039, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Durgadas%2BP.%2BKasbekar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasbekar%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADurgadas%2BP.%2BKasbekar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5269-8020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Durgadas P. Kasbekar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kas@cdfd.org.in" class="" data-icon-position="" data-hide-link-title="0">kas@cdfd.org.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1487&amp;rft.epage=1496&amp;rft.atitle=The%20Neurospora%20crassa%20Standard%20Oak%20Ridge%20Background%20Exhibits%20Atypically%20Efficient%20Meiotic%20Silencing%20by%20Unpaired%20DNA&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Giri%2C+Dev+Ashish&amp;rft.au=Pankajam%2C+Ajith+V.&amp;rft.au=Nishant%2C+Koodali+T.&amp;rft.au=Kasbekar%2C+Durgadas+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492877/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492877/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492877/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492877/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492877/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492877/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492877/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492877/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492877/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492877/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492877/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492877/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1487" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Introduction</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and methods</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1487.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1487.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1487.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
