<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1283" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1283.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1283.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1283.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1283.atom" />
<meta name="HW.pisa" content="ggg;9/5/1283" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids" />
<meta name="DC.Identifier" content="10.1534/g3.118.200919" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Savilammi et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Salmonids represent an intriguing taxonomical group for investigating genome evolution in vertebrates due to their relatively recent last common whole genome duplication event, which occurred between 80 and 100 million years ago. Here, we report on the chromosome-level genome assembly of European grayling ( Thymallus thymallus ), which represents one of the earliest diverged salmonid subfamilies. To achieve this, we first generated relatively long genomic scaffolds by using a previously published draft genome assembly along with long-read sequencing data and a linkage map. We then merged those scaffolds by applying synteny evidence from the Atlantic salmon ( Salmo salar ) genome. Comparisons of the European grayling genome assembly to the genomes of Atlantic salmon and Northern pike ( Esox lucius ), the latter used as a nonduplicated outgroup, detailed aspects of the characteristic chromosome evolution process that has taken place in European grayling. While Atlantic salmon and other salmonid genomes are portrayed by the typical occurrence of numerous chromosomal fusions, European grayling chromosomes were confirmed to be fusion-free and were characterized by a relatively large proportion of paracentric and pericentric inversions. We further reported on transposable elements specific to either the European grayling or Atlantic salmon genome, on the male-specific sdY gene in the European grayling chromosome 11A, and on regions under residual tetrasomy in the homeologous European grayling chromosome pairs 9A-9B and 25A-25B. The same chromosome pairs have been observed under residual tetrasomy in Atlantic salmon and in other salmonids, suggesting that this feature has been conserved since the subfamily split." />
<meta name="DC.Contributor" content="Tiina Sävilammi" />
<meta name="DC.Contributor" content="Craig R. Primmer" />
<meta name="DC.Contributor" content="Srinidhi Varadharajan" />
<meta name="DC.Contributor" content="René Guyomard" />
<meta name="DC.Contributor" content="Yann Guiguen" />
<meta name="DC.Contributor" content="Simen R. Sandve" />
<meta name="DC.Contributor" content="L. Asbjørn Vøllestad" />
<meta name="DC.Contributor" content="Spiros Papakostas" />
<meta name="DC.Contributor" content="Sigbjørn Lien" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Salmonids represent an intriguing taxonomical group for investigating genome evolution in vertebrates due to their relatively recent last common whole genome duplication event, which occurred between 80 and 100 million years ago. Here, we report on the chromosome-level genome assembly of European grayling (&lt;i&gt;Thymallus thymallus&lt;/i&gt;), which represents one of the earliest diverged salmonid subfamilies. To achieve this, we first generated relatively long genomic scaffolds by using a previously published draft genome assembly along with long-read sequencing data and a linkage map. We then merged those scaffolds by applying synteny evidence from the Atlantic salmon (&lt;i&gt;Salmo salar&lt;/i&gt;) genome. Comparisons of the European grayling genome assembly to the genomes of Atlantic salmon and Northern pike (&lt;i&gt;Esox lucius&lt;/i&gt;), the latter used as a nonduplicated outgroup, detailed aspects of the characteristic chromosome evolution process that has taken place in European grayling. While Atlantic salmon and other salmonid genomes are portrayed by the typical occurrence of numerous chromosomal fusions, European grayling chromosomes were confirmed to be fusion-free and were characterized by a relatively large proportion of paracentric and pericentric inversions. We further reported on transposable elements specific to either the European grayling or Atlantic salmon genome, on the male-specific &lt;i&gt;sdY&lt;/i&gt; gene in the European grayling chromosome 11A, and on regions under residual tetrasomy in the homeologous European grayling chromosome pairs 9A-9B and 25A-25B. The same chromosome pairs have been observed under residual tetrasomy in Atlantic salmon and in other salmonids, suggesting that this feature has been conserved since the subfamily split.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1283" />
<meta name="citation_id" content="9/5/1283" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1283" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1283.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1283.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1283.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200919" />
<meta name="citation_pmid" content="30833292" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="1283" />
<meta name="citation_lastpage" content="1294" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Tiina Sävilammi" />
<meta name="citation_author_institution" content="Department of Biology, University of Turku, 20014 Turku, Finland" />
<meta name="citation_author" content="Craig R. Primmer" />
<meta name="citation_author_institution" content="Organismal &amp; Evolutionary Biology Research Program, Faculty of Biological &amp; Environmental Sciences" />
<meta name="citation_author_institution" content="Institute of Biotechnology, University of Helsinki, 00014 Helsinki, Finland" />
<meta name="citation_author" content="Srinidhi Varadharajan" />
<meta name="citation_author_institution" content="Department of Biosciences, University of Oslo, 0316 Oslo, Norway" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2033-0758" />
<meta name="citation_author" content="René Guyomard" />
<meta name="citation_author_institution" content="INRA, UMR1313 GABI Génétique Animale et Biologie Intégrative, Domaine de Vilvert, 78352, Jouy-en-Josas Cedex, France" />
<meta name="citation_author" content="Yann Guiguen" />
<meta name="citation_author_institution" content="INRA, UR1037 Fish Physiology and Genomics, F-35000, Rennes, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5464-6219" />
<meta name="citation_author" content="Simen R. Sandve" />
<meta name="citation_author_institution" content="Centre for Integrative Genetics (CIGENE), Department of Animal and Aquacultural Sciences, Faculty of Biosciences, Norwegian University of Life Sciences, 1430 Ås, Norway" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4989-5311" />
<meta name="citation_author" content="L. Asbjørn Vøllestad" />
<meta name="citation_author_institution" content="Department of Biosciences, University of Oslo, 0316 Oslo, Norway" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9389-7982" />
<meta name="citation_author" content="Spiros Papakostas" />
<meta name="citation_author_institution" content="Department of Biology, University of Turku, 20014 Turku, Finland" />
<meta name="citation_author_email" content="sigbjorn.lien@nmbu.no" />
<meta name="citation_author_email" content="spiros.papakostas@utu.fi" />
<meta name="citation_author_email" content="spiros.papakostas@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5563-0048" />
<meta name="citation_author" content="Sigbjørn Lien" />
<meta name="citation_author_institution" content="Centre for Integrative Genetics (CIGENE), Department of Animal and Aquacultural Sciences, Faculty of Biosciences, Norwegian University of Life Sciences, 1430 Ås, Norway" />
<meta name="citation_author_email" content="sigbjorn.lien@nmbu.no" />
<meta name="citation_author_email" content="spiros.papakostas@utu.fi" />
<meta name="citation_author_email" content="spiros.papakostas@gmail.com" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Abrusán;citation_author=N. Grundmann;citation_author=L. DeMester;citation_author=W. Makalowski;citation_title=TEclass-a tool for automated classification of unknown eukaryotic transposable elements.;citation_volume=25;citation_year=2009;citation_pmid=19349283;citation_doi=10.1093/bioinformatics/btp084" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=A. Ferrara;citation_author=Q. Fontenot;citation_author=J. H. Postlethwait;citation_title=Genome Evolution and Meiotic Maps by Massively Parallel DNA Sequencing: Spotted Gar, an Outgroup for the Teleost Genome Duplication.;citation_volume=188;citation_year=2011;citation_pmid=21828280;citation_doi=10.1534/genetics.111.127324" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Auvinet;citation_author=P. Graça;citation_author=L. Belkadi;citation_author=L. Petit;citation_author=E. Bonnivard;citation_title=Mobilization of retrotransposons as a cause of chromosomal diversification and rapid speciation: the case for the Antarctic teleost genus Trematomus.;citation_pages=339;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4714-x" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the False Discovery Rate - a Practical and Powerful Approach to Multiple Testing.;citation_pages=289-300;citation_volume=57;citation_year=1995;citation_pmid=16632515;citation_doi=10.1093/biostatistics/kxj037" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=P. R. Berg;citation_author=B. Star;citation_author=C. Pampoulie;citation_author=M. Sodeland;citation_author=J. M. I. Barth;citation_title=Three chromosomal rearrangements promote genomic divergence between migratory and stationary ecotypes of Atlantic cod.;citation_pages=23246;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep23246" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=C. Berthelot;citation_author=F. Brunet;citation_author=D. Chalopin;citation_author=A. Juanchich;citation_author=M. Bernard;citation_title=The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.;citation_pages=3657;citation_volume=5;citation_year=2014;citation_pmid=24755649;citation_doi=10.1038/ncomms4657" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=T. Blomme;citation_author=K. Vandepoele;citation_author=S. De Bodt;citation_author=C. Simillion;citation_author=S. Maere;citation_title=The gain and loss of genes during 600 million years of vertebrate evolution.;citation_pages=R43-R43;citation_volume=7;citation_year=2006;citation_issue=5;citation_pmid=16723033;citation_doi=10.1186/gb-2006-7-5-r43" />
<meta name="citation_reference" content="citation_journal_title=Cytogenet. Genome Res.;citation_author=A. Canapa;citation_author=M. Barucca;citation_author=M. A. Biscotti;citation_author=M. Forconi;citation_author=E. Olmo;citation_title=Transposons, Genome Size, and Evolutionary Insights in Animals.;citation_volume=147;citation_year=2015;citation_doi=10.1159/000444429" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. L. Cantarel;citation_author=I. Korf;citation_author=S. M. Robb;citation_author=G. Parra;citation_author=E. Ross;citation_title=MAKER: An easy-to-use annotation pipeline designed for emerging model organism genomes.;citation_volume=18;citation_year=2008;citation_pmid=18025269;citation_doi=10.1101/gr.6743907" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=D. Charlesworth;citation_title=The status of supergenes in the 21st century: recombination suppression in Batesian mimicry and sex chromosomes and other complex adaptations.;citation_pages=74-90;citation_volume=9;citation_year=2016;citation_pmid=27087840;citation_doi=10.1111/eva.12291" />
<meta name="citation_reference" content="citation_author=N. Chen;citation_year=2004;citation_pmid=22955616;citation_doi=10.1002/0471250953.bi0410s05" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. Daron;citation_author=N. Glover;citation_author=L. Pingault;citation_author=S. Theil;citation_author=V. Jamilloux;citation_title=Organization and evolution of transposable elements along the bread wheat chromosome 3B.;citation_pages=546;citation_volume=15;citation_year=2014;citation_pmid=25476263;citation_doi=10.1186/s13059-014-0546-4" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. P. J. de Koning;citation_author=W. Gu;citation_author=T. A. Castoe;citation_author=M. A. Batzer;citation_author=D. D. Pollock;citation_title=Repetitive elements may comprise over two-thirds of the human genome.;citation_pages=e1002384-e1002384;citation_volume=7;citation_year=2011;citation_issue=12;citation_pmid=22144907;citation_doi=10.1371/journal.pgen.1002384" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=P. P. Edger;citation_author=J. C. Pires;citation_title=Gene and genome duplications: the impact of dosage-sensitivity on the fate of nuclear genes.;citation_pages=699-717;citation_volume=17;citation_year=2009;citation_issue=5;citation_pmid=19802709;citation_doi=10.1007/s10577-009-9055-9" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Ellinghaus;citation_author=S. Kurtz;citation_author=U. Willhoeft;citation_title=LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons.;citation_pages=18-18;citation_volume=9;citation_year=2008;citation_pmid=18194517;citation_doi=10.1186/1471-2105-9-18" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. C. English;citation_author=S. Richards;citation_author=Y. Han;citation_author=M. Wang;citation_author=V. Vee;citation_title=Mind the Gap: Upgrading Genomes with Pacific Biosciences RS Long-Read Sequencing Technology.;citation_pages=e47768-e47768;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185243;citation_doi=10.1371/journal.pone.0047768" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. L. Feder;citation_author=P. Nosil;citation_author=S. M. Flaxman;citation_title=Assessing when chromosomal rearrangements affect the dynamics of speciation: implications from computer simulations.;citation_pages=295;citation_volume=5;citation_year=2014;citation_doi=10.3389/fgene.2014.00295" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=S. Garagna;citation_author=J. Page;citation_author=R. Fernandez-Donoso;citation_author=M. Zuccotti;citation_author=J. B. Searle;citation_title=The Robertsonian phenomenon in the house mouse: mutation, meiosis and speciation.;citation_pages=529-544;citation_volume=123;citation_year=2014;citation_pmid=25053180;citation_doi=10.1007/s00412-014-0477-6" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Gish;citation_author=D. J. States;citation_title=Identification of protein coding regions by database similarity search.;citation_pages=266-272;citation_volume=3;citation_year=1993;citation_issue=3;citation_pmid=8485583;citation_doi=10.1038/ng0393-266" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. J. Haas;citation_author=A. L. Delcher;citation_author=S. M. Mount;citation_author=J. R. Wortman;citation_author=R. K. Smith;citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;citation_pages=5654-5666;citation_volume=31;citation_year=2003;citation_issue=19;citation_pmid=14500829;citation_doi=10.1093/nar/gkg770" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. J. Hoff;citation_author=S. Lange;citation_author=A. Lomsadze;citation_author=M. Borodovsky;citation_author=M. Stanke;citation_title=BRAKER1: Unsupervised RNA-Seq-Based Genome Annotation with GeneMark-ET and AUGUSTUS.;citation_volume=32;citation_year=2016;citation_pmid=26559507;citation_doi=10.1093/bioinformatics/btv661" />
<meta name="citation_reference" content="citation_journal_abbrev=Annu Rev Ecol Evol Syst;citation_author=A. A. Hoffmann;citation_author=L. H. Rieseberg;citation_title=Revisiting the Impact of Inversions in Evolution: From Population Genetic Markers to Drivers of Adaptive Shifts and Speciation?;citation_pages=21-42;citation_volume=39;citation_year=2008;citation_pmid=20419035;citation_doi=10.1146/annurev.ecolsys.39.110707.173532" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=A. L. Hufton;citation_author=G. Panopoulou;citation_title=Polyploidy and genome restructuring: a variety of outcomes.;citation_pages=600-606;citation_volume=19;citation_year=2009;citation_issue=6;citation_pmid=19900800;citation_doi=10.1016/j.gde.2009.10.005" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. Jurka;citation_author=V. V. Kapitonov;citation_author=A. Pavlicek;citation_author=P. Klonowski;citation_author=O. Kohany;citation_title=Repbase Update, a database of eukaryotic repetitive elements.;citation_pages=462-467;citation_volume=110;citation_year=2005;citation_issue=1-4;citation_pmid=16093699;citation_doi=10.1159/000084979" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. H. Kazazian;citation_title=Mobile Elements: Drivers of Genome Evolution;citation_pages=1626-1632;citation_volume=303;citation_year=2004;citation_issue=5664;citation_pmid=15016989;citation_doi=10.1126/science.1089670" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. G. Kirubakaran;citation_author=H. Grove;citation_author=M. P. Kent;citation_author=S. R. Sandve;citation_author=M. Baranski;citation_title=Two adjacent inversions maintain genomic differentiation between migratory and stationary ecotypes of Atlantic cod.;citation_pages=2130-2143;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13592" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_pages=59-59;citation_volume=5;citation_year=2004;citation_pmid=15144565;citation_doi=10.1186/1471-2105-5-59" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Kurtz;citation_author=A. Phillippy;citation_author=A. L. Delcher;citation_author=M. Smoot;citation_author=M. Shumway;citation_title=Versatile and open software for comparing large genomes.;citation_pages=R12-R12;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14759262;citation_doi=10.1186/gb-2004-5-2-r12" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Lee;citation_author=M. Lee;citation_author=W. Mohammed Ismail;citation_author=M. Rho;citation_author=G. C. Fox;citation_title=MGEScan: a Galaxy-based system for identifying retrotransposons in genomes.;citation_volume=32;citation_year=2016;citation_pmid=27153595;citation_doi=10.1093/bioinformatics/btw157" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_author=A. Levan;citation_author=K. Fredga;citation_author=A. A. Sandberg;citation_title=Nomenclature for Centromeric Position on Chromosomes.;citation_pages=201-220;citation_volume=52;citation_year=1964;citation_doi=10.1111/j.1601-5223.1964.tb01953.x" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=O. Levy;citation_author=B. A. Knisbacher;citation_author=E. Y. Levanon;citation_author=S. Havlin;citation_title=Integrating networks and comparative genomics reveals retroelement proliferation dynamics in hominid genomes;citation_pages=e1701256-e1701256;citation_volume=3;citation_year=2017;citation_issue=10;citation_doi=10.1126/sciadv.1701256" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lien;citation_author=B. F. Koop;citation_author=S. R. Sandve;citation_author=J. R. Miller;citation_author=M. P. Kent;citation_title=The Atlantic salmon genome provides insights into rediploidization.;citation_volume=533;citation_year=2016;citation_doi=10.1038/nature17164" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Lomsadze;citation_author=V. Ter-Hovhannisyan;citation_author=Y. O. Chernoff;citation_author=M. Borodovsky;citation_title=Gene identification in novel eukaryotic genomes by self-training algorithm;citation_pages=6494-6506;citation_volume=33;citation_year=2005;citation_issue=20;citation_pmid=16314312;citation_doi=10.1093/nar/gki937" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Origins of Genome Complexity;citation_pages=1401-1404;citation_volume=302;citation_year=2003;citation_issue=5649;citation_pmid=14631042;citation_doi=10.1126/science.1089370" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_title=The frailty of adaptive hypotheses for the origins of organismal complexity.;citation_volume=104;citation_year=2007;citation_pmid=17494740;citation_doi=10.1073/pnas.0702207104;citation_georef=2010067888" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=D. J. Macqueen;citation_author=I. A. Johnston;citation_title=A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.;citation_volume=281;citation_year=2014;citation_pmid=24452024;citation_doi=10.1098/rspb.2013.2881" />
<meta name="citation_reference" content="citation_journal_title=Genes (Basel);citation_author=A. Makhrov;citation_title=A Narrowing of the Phenotypic Diversity Range after Large Rearrangements of the Karyotype in Salmonidae: The Relationship between Saltational Genome Rearrangements and Gradual Adaptive Evolution.;citation_pages=297;citation_volume=8;citation_year=2017;citation_doi=10.3390/genes8110297" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Marçais;citation_author=C. Kingsford;citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.;citation_volume=27;citation_year=2011;citation_pmid=21217122;citation_doi=10.1093/bioinformatics/btr011" />
<meta name="citation_reference" content="citation_journal_title=Evol Lett.;citation_author=T. C. Nelson;citation_author=W. A. Cresko;citation_title=Ancient genomic variation underlies repeated ecological adaptation in young stickleback populations.;citation_pages=9-21;citation_volume=2;citation_year=2018;citation_doi=10.1002/evl3.37" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=K. Ocalewicz;citation_author=G. Furgala-Selezniow;citation_author=M. Szmyt;citation_author=R. Lisboa;citation_author=M. Kucinski;citation_title=Pericentromeric location of the telomeric DNA sequences on the European grayling chromosomes.;citation_pages=409-416;citation_volume=141;citation_year=2013;citation_doi=10.1007/s10709-013-9740-7" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication;citation_author=S. Ohno;citation_year=1970;citation_doi=10.1007/978-3-642-86659-3" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=R. Phillips;citation_author=P. Ráb;citation_title=Chromosome evolution in the Salmonidae (Pisces): an update.;citation_pages=1-25;citation_volume=76;citation_year=2001;citation_issue=1;citation_pmid=11325050;citation_doi=10.1017/S1464793100005613" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. B. Qumsiyeh;citation_title=Evolution of Number and Morphology of Mammalian Chromosomes;citation_pages=455-465;citation_volume=85;citation_year=1994;citation_issue=6;citation_pmid=7995926;citation_doi=10.1093/oxfordjournals.jhered.a111501" />
<meta name="citation_reference" content="R Core Team, 2017 R: a language and environment for statistical computing. R foundation for statistical computing, Vienna, Austria: http://www.r-project.org/" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Rastas;citation_author=F. C. F. Calboli;citation_author=B. Guo;citation_author=T. Shikano;citation_author=J. Merilä;citation_title=Construction of Ultradense Linkage Maps with Lep-MAP2: Stickleback F-2 Recombinant Crosses as an Example.;citation_volume=8;citation_year=2016;citation_pmid=26668116;citation_doi=10.1093/gbe/evv250" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=L. H. Rieseberg;citation_title=Chromosomal rearrangements and speciation.;citation_pages=351-358;citation_volume=16;citation_year=2001;citation_issue=7;citation_pmid=11403867;citation_doi=10.1016/S0169-5347(01)02187-5" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=F. M. Robertson;citation_author=M. K. Gundappa;citation_author=F. Grammes;citation_author=T. R. Hvidsten;citation_author=A. K. Redmond;citation_title=Lineage-specific rediploidization is a mechanism to explain time-lags between genome duplication and evolutionary diversification.;citation_pages=111;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-017-1241-z" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. B. Rondeau;citation_author=D. R. Minkley;citation_author=J. S. Leong;citation_author=A. M. Messmer;citation_author=J. R. Jantzen;citation_title=The Genome and Linkage Map of the Northern Pike (Esox lucius): Conserved Synteny Revealed between the Salmonid Sister Group and the Neoteleostei.;citation_pages=e102089;citation_volume=9;citation_year=2014;citation_pmid=25069045;citation_doi=10.1371/journal.pone.0102089" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Semon;citation_author=K. H. Wolfe;citation_title=Rearrangement rate following the whole-genome duplication in teleosts.;citation_volume=24;citation_year=2007;citation_pmid=17218642;citation_doi=10.1093/molbev/msm003" />
<meta name="citation_reference" content="citation_journal_title=Russ. J. Genet.;citation_author=S. V. Shedko;citation_author=I. L. Miroshnichenko;citation_author=G. A. Nemkova;citation_title=Phylogeny of Salmonids (Salmoniformes: Salmonidae) and its Molecular Dating: Analysis of mtDNA Data.;citation_pages=623-637;citation_volume=49;citation_year=2013;citation_doi=10.1134/S1022795413060112" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. Smeds;citation_author=A. Kunstner;citation_title=ConDeTri--a content dependent read trimmer for Illumina data.;citation_pages=e26314-e26314;citation_volume=6;citation_year=2011;citation_issue=10;citation_pmid=22039460;citation_doi=10.1371/journal.pone.0026314" />
<meta name="citation_reference" content="citation_author=A. F. A. Smit;citation_author=R. Hubley;citation_author=P. Green;citation_author=S. Kurtz;citation_year=2013–2015" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Steinbiss;citation_author=U. Willhoeft;citation_author=G. Gremme;citation_author=S. Kurtz;citation_title=Fine-grained annotation and classification of de novo predicted LTR retrotransposons.;citation_volume=37;citation_year=2009;citation_pmid=19786494;citation_doi=10.1093/nar/gkp759" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=B. J. G. Sutherland;citation_author=T. Gosselin;citation_author=E. Normandeau;citation_author=M. Lamothe;citation_author=N. Isabel;citation_title=Salmonid Chromosome Evolution as Revealed by a Novel Method for Comparing RADseq Linkage Maps.;citation_volume=8;citation_year=2016;citation_doi=10.1093/gbe/evw262" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Y. Van de Peer;citation_author=S. Maere;citation_author=A. Meyer;citation_title=The evolutionary significance of ancient genome duplications.;citation_pages=725-732;citation_volume=10;citation_year=2009;citation_issue=10;citation_pmid=19652647;citation_doi=10.1038/nrg2600" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=S. Varadharajan;citation_author=S. R. Sandve;citation_author=G. Gillard;citation_author=O. K. Tørresen;citation_author=T. Mulugeta;citation_title=The grayling genome reveals selection on gene expression regulation after whole genome duplication.;citation_pages=2785-2800;citation_volume=10;citation_year=2018;citation_doi=10.1093/gbe/evy201" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. C. Wacholder;citation_author=C. Cox;citation_author=T. J. Meyer;citation_author=R. P. Ruggiero;citation_author=V. Vemulapalli;citation_title=Inference of Transposable Element Ancestry.;citation_pages=e1004482;citation_volume=10;citation_year=2014;citation_doi=10.1371/journal.pgen.1004482" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=M. Wellenreuther;citation_author=L. Bernatchez;citation_title=Eco-Evolutionary Genomics of Chromosomal Inversions.;citation_pages=427-440;citation_volume=33;citation_year=2018;citation_doi=10.1016/j.tree.2018.04.002" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Wicker;citation_author=F. Sabot;citation_author=A. Hua-Van;citation_author=J. L. Bennetzen;citation_author=P. Capy;citation_title=A unified classification system for eukaryotic transposable elements.;citation_pages=973-982;citation_volume=8;citation_year=2007;citation_issue=12;citation_pmid=17984973;citation_doi=10.1038/nrg2165" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=A. Yano;citation_author=B. Nicol;citation_author=E. Jouanno;citation_author=E. Quillet;citation_author=A. Fostier;citation_title=The sexually dimorphic on the Y-chromosome gene (sdY) is a conserved male-specific Y-chromosome sequence in many salmonids.;citation_pages=486-496;citation_volume=6;citation_year=2013;citation_pmid=23745140;citation_doi=10.1111/eva.12032" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. M. Zdobnov;citation_author=R. Apweiler;citation_title=InterProScan - an integration platform for the signature-recognition methods in InterPro;citation_pages=847-848;citation_volume=17;citation_year=2001;citation_issue=9;citation_pmid=11590104;citation_doi=10.1093/bioinformatics/17.9.847" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Salmonids represent an intriguing taxonomical group for investigating genome evolution in vertebrates due to their relatively recent last common whole genome duplication event, which occurred between 80 and 100 million years ago. Here, we report on the chromosome-level genome assembly of European grayling ( Thymallus thymallus ), which represents one of the earliest diverged salmonid subfamilies. To achieve this, we first generated relatively long genomic scaffolds by using a previously published draft genome assembly along with long-read sequencing data and a linkage map. We then merged those scaffolds by applying synteny evidence from the Atlantic salmon ( Salmo salar ) genome. Comparisons of the European grayling genome assembly to the genomes of Atlantic salmon and Northern pike ( Esox lucius ), the latter used as a nonduplicated outgroup, detailed aspects of the characteristic chromosome evolution process that has taken place in European grayling. While Atlantic salmon and other salmonid genomes are portrayed by the typical occurrence of numerous chromosomal fusions, European grayling chromosomes were confirmed to be fusion-free and were characterized by a relatively large proportion of paracentric and pericentric inversions. We further reported on transposable elements specific to either the European grayling or Atlantic salmon genome, on the male-specific sdY gene in the European grayling chromosome 11A, and on regions under residual tetrasomy in the homeologous European grayling chromosome pairs 9A-9B and 25A-25B. The same chromosome pairs have been observed under residual tetrasomy in Atlantic salmon and in other salmonids, suggesting that this feature has been conserved since the subfamily split." />
<meta name="og-title" property="og:title" content="The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1283" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Salmonids represent an intriguing taxonomical group for investigating genome evolution in vertebrates due to their relatively recent last common whole genome duplication event, which occurred between 80 and 100 million years ago. Here, we report on the chromosome-level genome assembly of European grayling ( Thymallus thymallus ), which represents one of the earliest diverged salmonid subfamilies. To achieve this, we first generated relatively long genomic scaffolds by using a previously published draft genome assembly along with long-read sequencing data and a linkage map. We then merged those scaffolds by applying synteny evidence from the Atlantic salmon ( Salmo salar ) genome. Comparisons of the European grayling genome assembly to the genomes of Atlantic salmon and Northern pike ( Esox lucius ), the latter used as a nonduplicated outgroup, detailed aspects of the characteristic chromosome evolution process that has taken place in European grayling. While Atlantic salmon and other salmonid genomes are portrayed by the typical occurrence of numerous chromosomal fusions, European grayling chromosomes were confirmed to be fusion-free and were characterized by a relatively large proportion of paracentric and pericentric inversions. We further reported on transposable elements specific to either the European grayling or Atlantic salmon genome, on the male-specific sdY gene in the European grayling chromosome 11A, and on regions under residual tetrasomy in the homeologous European grayling chromosome pairs 9A-9B and 25A-25B. The same chromosome pairs have been observed under residual tetrasomy in Atlantic salmon and in other salmonids, suggesting that this feature has been conserved since the subfamily split." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0yaMSFwMar0Rn1dtTsKTWKOMlhJWxSImE6nHio6tVNs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493108","apath":"\/ggg\/9\/5\/1283.atom","pisa":"ggg;9\/5\/1283","ac":{"\/ggg\/9\/5\/1283.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1283.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1283"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1283"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1283":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220JGkAkb3nMh94eiCe4WKR7xB3GCZ9Ig+f2COvlrDqiJGkhTu8JxDaYnZFJb6cbn54IvzWbwRWPAevwL5UuWj8sMagx5xI\\\\\\\/LtmxWhRnSgFuEj7wfn91NV5sZBbobyaU0mf5WA5oNyZHYfSne47NyVBMAaZvz7odREUCZcO+BO9pVTCTuJuJq1x6DJC6DgO6tg8FHZyqAJHqzJOGd5OqF0EJDksXhKgfoHBOWRgIvr62JKLkGr0V\\\\\\\/xR+gAiOFUgR+n8xyJblrMGJiJ9u9BRqLb5U1IRS1bNt2Eg0bT5MCMAqAmtxV1fF984nSyLOudgFxoZktwS4zcN9nKnlA9nl63FKSx\\\\\\\/a\\\\\\\/uthec18pumjyI2p95pDChglePFsct1YpY2EFgLLmNpOlGzvQvIZlTZasClD85CrfnzkqQ1jxWgQ+FZsSQlXZm7aagcjx5SJ1NH+OIDAlMBp4c8J6Sa1yqRFMLRfrnqR0reRm7vwoB3gB739i1dKFDxus05XVl5NbRR2d1Rh7BmUF6X9vIIuFrHHauRBnCoWWHJDEfXHzlEur+CfuItS991O4e2FrYE5BCLQp5gp40H28TI6sYljC0mn6+y1uL\\\\\\\/aqdIvy8MCF0lXElKxAMyKKYrcI7kPP1qe2l+o5Ouu\\\\\\\/3HUSrGxDrU2+Xs35fca3o6qortn8nmG3JKbPHEFPheaU0ows2CjV82Baz4I\\\\\\\/dagl8ujCYMLo8TvEGsYroMFlijqtpz+xy6H4SrwCd2uBx2OWG\\\\\\\/\\\\\\\/RilWr7FUFB\\\\\\\/JcnATDS4wPRShMEnZIdNCrUbvtwzmykhA8jL2FVGLKWxCN19MHEH6Z92HTHDWzG+Wya+lxEuyg4G9N\\\\\\\/lrdxqaGjH\\\\\\\/uONkw3Do3poIL4HvEphAAImZdxMY\\\\\\\/EfOxI4CySxSZeg6gguFHPyoO06abHRFMSzoA3KYfweayw6sPzGgR\\\\\\\/kUO9cQHBrmUv41vt1Fc9NvmlE8jTbcBM9Q1bK+rTfkru4Upn0yq3aG03DSKhMsMHB3uFtwHyjYjVZagA2lQLdHYN4kEQ\\\\\\\/T59HnFkFKLGDVcNcI5RXd3jhrcDOSKamTDh00EnEXe29En8DRW3h+sStVK3vGcyN7uOOgWNk4Z+0QnHoY6EKIqZgBR+J2uH2yOFBqsjv\\\\\\\/46U8ATcdy01kJNL1Rac2tq+2GNVURshWaARXpVNDvsEXUiUthuFSaBINNicoW8rSp8XulrPVvY2uPth37JAm0SsHL7ytEfqyKCDC0AlhmQa1a4Qobx+CFvigKUFQpSNgp+PiC6YSgsbVmgSboj\\\u0022,\\\u0022iv\\\u0022:\\\u0022CBHn+diNDygzQuCFUQ8Nuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002254e64b685e660fd8d080b59e96a16d92a06f974e380cc21d1c095f617878da86\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002241Cl8IaZ8IIARyXE9FN\\\\\\\/qekZeZ5FDQgFsIm4MthI5MMh3OooDiOKGuF97rTAe0iG1fgq9KfYHZmnj+KW9u865BsF\\\\\\\/tFCWDipFGwv\\\\\\\/5RNuohVLe07uGW\\\\\\\/hN7RF6T5O+38xrlSYUce\\\\\\\/d7KlYtz2ZY25myIbgi1mB1KbOJjHlu7roj9AsFRcYYI1RlilUVmQEkFz1sjR8YkTtRXhypR0IkLzvtHzwwIbecDjgGWC5iCU0QoXV+iJ93\\\\\\\/FLEh6sz8LVhmS9P\\\\\\\/Sjd70GwzXgpv+sg+So5i9nDHXJNSgf+cmXU7\\\\\\\/3+Gnl392xnVVvg8Ju9HZVW3oe6r6DIAwiBNLVt9QBjGK3DGLOThXWBwpvANYcnrpw6PSSVR3qgZOXx5SbbzvOmtwKeQXrYalTB6bJP7M3IIxnwmBvhYCcmi1gds4ogOt15eQYgp+N\\\\\\\/rI3LHEOFKiVW46ua97VY2sm0SyaNqeEwE0xc3XH18xlPsirBpTQPWU7QZnTcpb4FssH9LUH9yowjdEU4SsKXCSvzCLSTk5nsviJtKkMQvtO5rM9cU4HHrokaA5456w6A+l5Ad5mftOaIh0EXq+GNPwNbDsXmbbFG6QJIEQU+jG+wGl\\\\\\\/eKRecS1rodo96CjDWCMIzhnOsH5aZejUvs3CjhW18IaDaprgjFLNkUkYCDl4ctj6FBwWHnz6Q1zHoG8Jvb6g7D5vtxUhrzL\\\\\\\/Ye9wIyNKKHQK0Hvc8oU\\\\\\\/1d1ZSc8pQdzcyH5ZSByXTWPA\\\\\\\/v4zJokq6pkWtkinZbnAltKvhYlOy01igZtKosTACGNHt6XMdWFoqF7j8F+tqYD9v+CXvV\\\\\\\/6wA9UhDAOLpfuE9dUM93QTNWx6aANcGV3VF0M0+s6n6qRMtHQDyuj8A4gLswUXbhDSK0lc5l8fncXIEqCisTHR5eyQobtXePsiiMCfrvi6j4onWGc1\\\\\\\/vuH9pGfkz3j7JFYArMpBAJvK2IYCA0BkTEGwgZ3ui3Kc81j5UabULenSO0n1Vo4cXOc3QIR3hyWNRXXlW3N0WGg3G3OGfyFmnOxH7P651lYZAduiDcCkbSq5WLTOrmNeIs04KyXXTxlLfn8pT8Nux9tdSIyfC0BxjhwkfBc2fnQPoDdRfpGZcyDWVFmLYXwuC9OitELIXgzJ1yvwYqzIJlAiS\\\\\\\/qtecQF\\\\\\\/2NQM7ZBbbcQTAnzAxz8vG6nEEVhPcvjvXc5qt5896qAbJQhYM0mCcYg3OlDreD\\\\\\\/VLOZzFh+NLY91X\\\\\\\/pHGcXBu2o23gKn0pRw7FpBIp7a3LS4AMz8A2CdQrMzBrfHTHIBrDLuv03LB4XM+xwJmQKoBxJvvYz7jOxn+rfiRbea9n3Fi3iiwbKZQ45QeGFp\\\\\\\/WqoHMeTDZP7Y3J60kDpGg1Q0PMTISZBhnnnmmaE4lmAFFZIEYEGNtaW4spM2uUZzF+QWeHYccahmS4NTcPLSruC\\\\\\\/4m1TIL36Ndd60o9lZePia6mUH\\\\\\\/M30NnbLF9EkhVmYkliMxSLFBUNGACkKLZi4nlFGhs8Pgaj6jsh+wKpqDMKFpy9rz7Ci7+LctxTAG9mD8SHS5J\\\\\\\/B9eELIvyZOrWT8oeDDc29Ae+cILiNYOpU4OIrR4+mFtCzq+9ThlRhccaJqxamXykatdgS\\\\\\\/CCKUuUnpwXB2UV01i2+kAgH8voghaMfTxuVjN9bUuq5ANM8eAUkNW\\\\\\\/9M4OZKQkcgGKHAwkT4KTPD6aeBCqDuIAUjJ7qH1FJVA4ARzPC0vj1AszLxnb8YpBiZhg7PA9\\\\\\\/zp8rGqQ2QCxdu5wFo+65sjWYIm5Ri7a3ZsAZmxFNZsIwPvMG\\\\\\\/0BnOdulSsak6HYQ5ec\\\\\\\/erQXm8la15hwMEnAG1VfG1fiWy8NHvFrdh+zX19HGcilBzY7TCgxDiA4\\\\\\\/zrgfYPqwPKDujKTMmAwfADpzi1TLBSaZH\\\\\\\/76G1s3wvOzZHGjwbWEew2lmJ1CsWBJcBXWm8w1RWHQaDyhbzsGyROqdgYV+GzOGl4C+TqOMfBNIzB88BkVgA2ikt4Sz0oLdD6vGN2L5+tz7XhgoZrAecK09rowgWDqPU4pAYuKEFueY+\\\\\\\/mVq4TsWrZgGUIbB0bTC92gCdM+OmJu9bT2+eJuRh462unvSlp4Q4N+BmxL1+Xoc4QpUf\\\\\\\/xUDns\\\\\\\/gYbeNEwQPG+SFwkNBt4GugvtZJAQM58O6kueoXNdUN5kZdWtSy\\\\\\\/vTv7uN+XKeCUNkgjrygsIqKl0hi0iKz4HGIjXKgMgj2pMpRCka3Vznjv6SEfMRABb18RLV55CGhTyFn3z+y7bTO5O7gZg9vDtm\\\\\\\/TVQ3b06LGNQXHyyBjNK0gFEFgSccMmkLjonxrBPPr3u6KvJikqqtTAyCpSuKURwzWNODhzMHsTNj211gMHjC9Zspq5i4D8Z+2KSJodsckscij8HOJRJPpFxiE6\\\\\\\/ZEQG2WPvDCBqh9VCqBTRgfuqpFVwlRpTZ1\\\\\\\/LMgUyL7dRwPOxpc+dY638V\\\\\\\/pUA6XfY+trxjr8DTRSqUHM33S5VB\\\\\\\/DwGzxSAdTwQ3vIeyGQSQf9\\\\\\\/RwteUU3FpB6WmKqQ5ej8ou3bwotAcOn0xhArCHJwcvwUOduEd0Z9PRkDi03zB68WrA7tw5hOYoa3Mq+8oxIshnLIGS\\\\\\\/rb8jpVzzBwPl\\\\\\\/G9+qFikrBPF0lqNpcCyIX3LzINkSaSPmGI2WdrTafhD5b\\\\\\\/HWsWvE0yOVH6zEP5b7287o1HXjPRVTQcAp2hmV8Cghm0lCHAvNrIr1AigmbgLmTifnbJWKjK0CdTPv7O3keYsqS0h6tyo8xbOITindHbxPrvSedkgCIDoAw1j+lzajKUw8s0pw2OvS1kMozSQtyA47RU6jKsGlP4witLO3rJczbwOu6FmULakMFT1lHTTKRuEX2C3GxVd2e8gMizEtUwzrvQrQey56Zwr26QQhi8Qo32qVDbTx\\\\\\\/Hvcuy7h1FCcLWhOsBcnjpPUAT6htSKX6\\\\\\\/xjNaplDXqeJbWeWGZGYmIJ61B6JPIcEZnGlBcGKvoAoZDaWS0EVQTes4P2ui7P6HwP06ScdeMnANCmE6EQI2YMlEW2mDhWOKGqjOqdULfM9I5nl79tXDdK9BaqMHVZ6HYgIL24OjGElDWsqTPzJR4p12Pah3\\\\\\\/72zmEb4tNVydvGceRwgaD9He9AyTzO6voR7FBmA42KNiMZkOzudmE8cO6ZO1+jLUIzwKTUscxZ4urTV0ISVDWV7NZ+J3Q1D5Sd89dD0GmX95DzLOVvbJQR59ycBEYz4HmAKKyeHPookWjMQfAB6TMlN3ZS+HW0GNvqMsVlYOF0j53\\\\\\\/RyXCqppedunQrazI79PXTqghg4ifJ1skqP83HlA4QoA2xeD7e\\\\\\\/zdSdLWX85ULTmUiT82DcY1Dx\\\\\\\/POSVTXVL\\\\\\\/xZp2qqm8YLliyfNIDo020IJcAVgeDLvove36sbLFFcIfHqsp8MvpFfJlXCkjR4SjprDejxsMdp4YIvytiffhLn8bIvlBEfAhE63pQp6uE\\\\\\\/+hINQsBv4y6jlVhM4cxtdETr1t6d1huMiZY8vrbG3w7EArJfjcsC0F40EgCikDBbWHVcKBDdoSa0P8l\\\\\\\/PuEn1g9n6i3k5iC470O7sTQRxjRaUKloGKak+M6HQJCWE8RzeylxXEajrT8olDDjjuVxC+lF70BUqokc4+MZy4\\\\\\\/7GeA+hutLFyChp3OVZ9b9mOyf4yEonxiwHOcqvsm\\\\\\\/MgYq2+C3jqta8k\\\\\\\/3mn32nqtfWCJQyWSRMJdW8Fv4NFm0v17KUuqBau2d2KVQ+2UFIFFu1QnFywb0cbcKxp\\\\\\\/JBouiIlCPgeBB57UkTt9TBPfCu7YidnkS+XVdA1\\\\\\\/uqxJ+NA30BPttk4OgDm44chOsMV0uTArb3tvcw3O3sxjhd\\\\\\\/1KGVGNetol7ydiE7cUs7zYeB3dH5dzhExe8Ng8Tq1zfr1m3f+tT\\\\\\\/uLQoWFLCL+cJhCmDFfPphbOnuyWNOaiSJUD5GkERxrwIaxRMKBJ9i8PfowRx9j5tuLgOcTTtUtobw1y\\\\\\\/gg6FFyHrKl2Ha8beZQW9nTXQiO6oXZS28Zs0huVRBAHsMn28gqI0EZDkHhwTDw+4GIRIsZuC7fNVJ8KWeXLUW6unhWzuWc7\\\\\\\/4mFBfh86gOpZxS9m6c4wk6mADhpDeKTIHfbAXRbei+8R2RqScHqzXHXQMHebMCBYREU5yldz6KLKf7rAgt\\\\\\\/WicUeTkFjGU5AhvgDCCkn6TmUj2RjNqiNWfGvYqvWzIR\\\\\\\/8QDuhum2GUIwwphCJXFzFjmNVlcZ9kwujQAaW6QijduwQxA4iKEe4ablmrwOY4Dy4eit5kBoI8UOj4lB3mAN1bVGCqGuHJxJ\\\\\\\/Dkg0mBp6Z4JV1lEKyUXLRpRBEuE\\\\\\\/iuDq2xVon6P3HNF2kARbZXoYVKFqJSiZnJ1XSDt5LNwH9WjEB2XgxkVYj+KCfi9hDd9jzjp2KN9OlmzV6f+W6j+PHaduK0BnK4o6nOKYiMKw\\\\\\\/zx5XzdQUX\\\\\\\/JzCEuN9wJFjdkQEl82rPSxkqzzyPN6nw8aLfBGB4GiwtB3a52Nrpr3pqKdqYifIphykkWaTKffkfjNYBAzFqYWQSoBOceuFlnC1pLrvIadmw04JUM+nhMFTKE0Df9Rqp0b0HlE9t92xsI2Fcjxgfpo69J\\\\\\\/CghIHlFA2ZjsR1qNYgXfsEVRw7rinS+ubtlBXNJRLYAFUpR5yOCUOvCKZ3yqdlGVuBWs8oyoGoog7xMx8glpcGIIGupAh5+noGoFasHPh\\\\\\\/1G1O8Ufj1Prb5xOsBNNzh2W96Q5Na2sLKTPgS1Kv4313euugBTn4q3X9Bem8SE5VnXAe9FIYVCAnQo2KwAckwBBw07JIjJ+2RtwVnFdTV9c2dFDg13jNq\\\\\\\/KXbrE4ud55HmfxUpk86UWlDMN0WAcl43LJjIejp8cBTia6ROSXMd6bpFoKmmERixe07Uq1Dh\\\\\\\/mtvLOidIEw9xMxWGknM6iNNRPApDp0Wq0YzLRV9SOAky9yupVPb7nL3Nu3xrg1p4LV2qkLOv+Z0wYQWxxLpQmW6JP8K5vxeNttSXiL+Vdux1sk\\\\\\\/4xwmlnz6C0d1gvNC3leSN6WnJczH7UY4Cj5IFREj0iRmqoBKFjhIkuFJpdpFEFabQqY1jHdRkidqmKaH0pog+yh7qU43JHYFJwRKqCh6ZdoyGJwgd0G899RTIsu6WmMJKsnFHDVia04ESrORb3A+p12KSvTndDhWPPRzTEERRmIyvxPeGuIHFCk2j4T\\\\\\\/3PIhCwcQIOqbQoNg48x3mEUh64I0VaGWsZdK4ZkX9k5B3YkZL\\\\\\\/mT3gT3wEVv6KOLBq1XzvfcwDLUNuR1Bu4ChYKGRmexxEUEShmo1u6r6r2J4RVvDLe32\\\\\\\/a8ngX9lEuutGiq9Pm\\\\\\\/aiHIHu2bWw0Brc6wd4uUdDlde79B5fjIL7\\\\\\\/A3ByP4QErNaoB2cncZ9rzjD+1yAnkCoJ0jgvyeHH6DzWuQDBdRtfo5yDMspbywm1bQMyXc\\\\\\\/h80b6t96tJ3VRX0FA1lBFx5NAWKoPp2kwCQhlJheirDngZr\\\\\\\/1ID5RC2Ceu0fdJsXnpnfr6ULPHns3ML2YyXz9iZWHrSWG9yYKEQr0Unk5e4XlsPsmK4UtMb1\\\\\\\/PBvl92mRQkp8K8DcdTEqPbtXcvbsIkj5vFQhYkfuTBqvlgt1\\\\\\\/PjREg6i6eMwNxGygUx1zzAvd2K08KJfyqqbxZU064zwtr8jK+I5tDGi9IQWiYyUS2YEVrFkQAvu5CRp+ggxVtTUpeq7ZnG2HfAxda3tbVKddnbfpHKoYNVdE+lXNg233S7JgieYvYt2kzhw5Qsui2xRuJfb0hyQS7yqZD\\\\\\\/5XopeihnmaVyVGs3BRULMO0q7gzANJJ9CbNxzI6Ml4eX+qdjgvOKX803k013d74EYp4X9GAuGdsNZftyj+AWDt98Ne7\\\\\\\/m\\\\\\\/1E21oW+SANM2PK21XZ\\\\\\\/mRykPnY9xzA48pznEKU5qf29lLlFRT41UgVEOZDl2G1cfNqmY+QYDgl\\\\\\\/TP8Y2brfKAlQsS9B\\\\\\\/qvunIUCs8zWLCOlatJ0hvYUCnNGJHqTuEXLT+Z\\\\\\\/bil4D00wAOJVUU0oac0HZJ\\\\\\\/UHf0tXdBJy8OvPDmq7D5P3iE7jZA6ksqfu7jIrG\\\\\\\/u\\\\\\\/BJgd5g\\\\\\\/cLLv6Iq2p\\\\\\\/IIwdCOLoAg2FICQmFQawmw1Tqjaz9ZjN3NndmHBffvyijWWHAXynHFT5rlxR0R+qy0XmznZBgiYAr64VHqZThD0TLokAcvIstyKJQ90MLUfAjaDboN+5\\\\\\\/p2whgaF+QXpIe6rQqvbclM3KC0xTD842Jcv1oLqfBoK8OtyCLz2RUI55INGdWb4YRqUjcBPSb6qJvJhOzozamAKDkY1WJZLQ5qbbIWHXPMA2m0dO6MbxC3Qp1uyPX+baf6Yf+hof643pGCUYyE9ecWjkg7gKImXpx9mkH3zQswbVfxVwiNW+CXjPJEvigyPrOfhP6ay6Nn+g2rzyVdTBQL48ZamujNLGe1UVFfsZ4GFGWj9rEK4tbTedaMCmc8wghY4Umyx\\\\\\\/V+nDDJqrvTv4quzj+XeHNxt4Dw0rwwDtIE2q6XR5F2cAZ9ZLODD1KmkBWuJ9LtBGOk23uyjtmvCKDhG0iZrJoo3L0+PCcL2lXEOy+UGkL8zCRWOC4xDyGqw2SmNHL5Awk\\\\\\\/cYZvRtESjh+J394xCbTRjr1iHI10DzO6Ygmk1D8g3ktPiQeMo2IORYZTxi1yZA+zegGpXZntU69zPdZkKVtcM3ELNTyuTyNmxbrz2xF1d7EQmfCaIdkYv98l0G6a1\\\\\\\/m9EK55mfd1SHJmm8BgHt0CKrm+XL9B31VSBeLo8O9J\\\\\\\/e9Dkpw35oeV5GN3zELKIDL7CfdeSFU+VsVDqF6hEBT1Cy9VvBtKOdU7ZSnm9Xa9icAc+t6yIkkIb2aN4ibXlujIQUuTaJSjmFSS9O9hIkdBodCF97OdLijrmlhRU71ujiAq7joaEgQpcEfoHgCdZcZR\\\\\\\/UMDVNfUMXBfoVYEy8U4QRzcgTPyHSr56Bnb\\\\\\\/JxQHUKqqrEVbDHS4sk\\\\\\\/uZITlLpRUBoGnAFj7zowvcN6YSX0W9eVqRum1RyG2wxqi\\\\\\\/H0RCKEee8Sb9NrK3B7kvEFtnsawIQDv4epO4NwpEMnPUN6Fo1JpHcyYJMu9I3flc8gSp6xfQSZVFGgK3YSXBxhL9AY16q8dMDcmqooUaLCVOzAOtFYwWUHeh8sZMJUqbPxlJFvCOo2rNVKMGwfCl5sJiUyE+Rznrpp\\\\\\\/i3ljaxrcSVRkNo+uoefSZMpU4rXM4ps6lwgZMfchjSpdpSADaqh21T3FcpA0hVbZC6\\\\\\\/kmSwbRdEG1n9HmEfGjhWnz3yJacnTAvcWVFxxFyQWBujJxB1ZRovD3K8eUMcheyFWQq2KZPEeBpSD\\\\\\\/kPCWD1VRtDereHw2hPvL0JjvpuPFokyt06lsNzrASnzVAboQpnxrCm43+2yXHn8JEewG1b7db1BRlxAePgPCTw\\\\\\\/QBm+j0f\\\\\\\/KYgGpTIibjPNsS7B0jfEaSMOpuFF2xZuMTf1i6hwqB9oOMurlhK7mSE5WmuSn3ehvbYlpV5Fb6OrM5KKltwgZtAbWxXwdaRPDK6ga\\\\\\\/F5+HHkp7E8No+Y1nnc85VgP07tQ6HCnDdjtoy6Cd5VzTd+QLp7mWl5Wr9VsAeneviOmzX1G\\\\\\\/Wnmovo8\\\\\\\/x\\\\\\\/zp1RcC5+0bk9O5vEMn8Ayuj1kmv\\\\\\\/u9DT67ai6DIbyAq8ofVXXzLdWVOx7bkf6lJp3KoYdlI\\\\\\\/X1aGSdKKCq2KryFDScHttnteVQiPfnxzfGie5INVF9foIaoH2p\\\\\\\/APgXrfRdn0W5Z+7FN+mP\\\\\\\/WKu9PF3e8+o3X\\\\\\\/VsZ0RbiTyli78HUXl+qoV9lu2JDE0pUO7wwseXqHEYm5Ybr2qlTUQy1eq0Ua1jOGEpG+GXNObGg6q3Tq5wkTOjP1F7DYeg9oSw7xK7SA7jRl8LoK8GgzvaRkdGifuYL2tmX9MU8G96FmA2KtB15XTxegRK9zfMjjQvxknCE2xUUeysPycG10F9PsjMr9WABiPsqwG0Dcbue1Z6KhbBaC5sH8W2rmD7h8R\\\\\\\/wlNBVWfBrktYX8\\\\\\\/R6RVJIBEHb0lkO6BMEj4IfDYELngV08ycWHeCnxQkwoVQbUC5RVBiRhLQyww5LymkL38CQV5YUatXNLndpl9izLoNhytdIp1WQ0QthaHj8TAuwyU51GiPFZuwjeQsRm2OmxMcHvLT21CjWdIH\\\\\\\/XdZRzA6RSRaSs+biv+GVBjb3DQ48cVQlhJ+8Q\\\\\\\/YJpq5QEBAO5d+Mo7+M8mo+uHB79EyMvd\\\\\\\/oRhHhx+91ahVb2WINePV9gRd95HCz9b0V\\\\\\\/pgfSdDMz4H\\\\\\\/at5Y8cjhb1E1TMMcpzNEhycDQwHkZRkDdQ\\\\\\\/gHaeWIo5+yD1Q110DL4vQfd2654DO7p99XvhflzElP2iLDNqFo7U1p8K3ARKkfHFL\\\\\\\/l67IihKkwZMhTyUefpJ\\\\\\\/kiJhI4dgNOnlhhwHGxUWCU28laL2g\\\\\\\/ajgA8+GZUeK1CSUoRr9U6mYqrZyYVI4zU15bfYdbFNtP5L8iSuz7MLgnhuTisEbfXjDvxOmd5q+yLc1q\\\\\\\/fDgSFg0fy3UEVtx3UoQX90cng3d\\\\\\\/bJ3RnS6ty2JyJL+oR5DA\\\\\\\/Zj9czkmHXlaU8doS7jsm8n\\\\\\\/CidGX9fkDczH06cDN8+G6RfgARRCxM1xhnn2Ptj6eoUE04f\\\\\\\/LIqfjvYgDWkgZDcxe5Urp12yYusQwfVa1KfpREscOqJGAqnBobjhDiS+\\\\\\\/g9bCx44QkfBfAHY6PrCCHHQMkw+7VxcGsDeHmwrTxXDN8t2ymzf01c5AVzaKjoQ9w9+eyLScUIBQ2zxQQUcqi5v5nwY6nBfLreeUFP+saculhsNteuztKrrONGacxzU0LNoHd+k\\\\\\\/EUN7Vu8IIwfLqoM9CxUqiW9qAVR10XkvMxfqFG9M+pqqg8XDhA2gBDhFLaN5wh\\\\\\\/ne6lKnEp0K758z87HJ+nG5GspczfUgvDVQH+c5KwRKx78n0E3GVeDwJ+AvMEgIdecgMbgN8sxdCC5YdSlqoe86HZDX\\\\\\\/3VgUtOLHy+I8ttxF9JkPuSmgbRWNYlJ1\\\\\\\/EEYeIL9nJwXSjK8iW1STPTKOft\\\\\\\/Al\\\\\\\/inszxvmpohGuUxa56Sn0y87KAz1m7CAYgCROv54llZYVY9wqwMVnXPRxXeLsF7PIIPkzjkwZdWgmlktW3s3cVYa2t\\\\\\\/bEljFUCgnxOtVknG\\\\\\\/SFYVsxkTAwipfUp0Y5d3rTo8rY5AkTrOH4Rp76\\\\\\\/qChcR\\\\\\\/2LduoG7DIodcbBORBqSNRILWh5PABy051UM2SoyoXlCdmw5vokrAwzFbGtvyK8ri9q+gUg4NsRCOwdCBuxCpLvE0OT8L8M3ghN1kUKh3Qc7gfDQUOxvrCDcojdaDng9DqkaorGQc1mesXI8iZgYHnpUULWuet9tiI4UP\\\\\\\/0Uym7XFRAIyq3d\\\\\\\/8wJR7Xd60gpn7IHHj+IUtPH8+l5u+tmjldz88ccD73olOaW4dvyknrbV9HgGkccbwILuJEVJ5OXCeqsDPkaBImU+ODzY+gxlOER9xprtbaK0a4MPZYchi9\\\\\\\/ywSPK\\\\\\\/NabsTRPHozq4L1KwVC6V8BFieFYva0MUDkuoOoFBsKnKpE8enHQbtF8mgfIYbJ4FkDB7pBFMO5+FS+dMDmBP5v9iJT6Qdl3eN\\\\\\\/CWyLFkSihtWIOVw\\\\\\\/UlHgHRVrAMkVB97NAp19WzcYRUhTSkR2U2zkJpSwH35eci1l2TbkEkvRlS7yiwOS8R0Hciz1FNv4Gx4BvsoVIgtD8oomFceQxk\\\\\\\/4UMzDqzxHIoWjY\\\\\\\/H8mXwQDODeG9EiilYlJby+zYVTu6\\\\\\\/TWDVkOM3ohiBqAs4NItKBvlwCpXizdJR44zU8ff+6NAAJxFgPocWEH44XHPbOrBBqK+iaaeMB7PMXC3duOHfSw7g7J0PnDyFYgT3dhWCosNy+490Pw1wCvU4kDIHhsxM2ELPM0pC6yCF8DbUI4NTlI9pVJICTnTa2HePprUreL9v0NUUrReB4EdxF1M8huD4woTbevVVaYdYREpqJ2vDd0e6jMSB4QBZ0S\\\\\\\/vtpIoZGrD86YXizY5kS6\\\\\\\/20t+xN+cMeXVpLZy8BdOyyULcV3juHMOB17QsnuM36+AOSF2k\\\\\\\/3BDm5q\\\\\\\/Ramzo\\\\\\\/i7XprotEMt113tqE7dFjrBDSS2eB9fxlfdalOyM8vBFE5IgQ2WyUTN7P5SJkENv9IHrwy\\\\\\\/bUY+YuZR8k02ALV54SBy34rexSkTC9Wx5GJFt0BOF90V5jWBr9bpUb+fjuyUtijHMuvlZ74BT5NBNxEMs6GefVR0D6N2L6yXeeMifma7kNI6sw\\\\\\\/zd1YRjxCdyu5fxy8SrqXA0FzCSR5Pj\\\\\\\/TkiPkYieao0nQGuyK2nf7ft+PVkyZleLco8gSzzEH+y\\\\\\\/H44Ip4\\\\\\\/mzGtrmTFpW3W0plGGtOZxr21QxJil0EC+I6crO2eu\\\\\\\/Cyj2SlCkUPY9Cwy\\\\\\\/Mdvp9wOaNa3NmO+fUbVeGzxjn1X3dpogbNi7JgW1hjOx+CVr0UwVcRF6CvVdDeROlDOQOy6k12byEYTEpxeD\\\\\\\/ByYetujQKUpRINV8GnksMciP7j5r+jjl+M44ugTw+FqsCOROkp643IIiEPZ1rTnLFpdxkWyGCajIC1mSmUQ9wwR6dIO0Iot67cTFicM60Fx\\\\\\\/ExSTy3N6Q603SH7KBxpyzAb851XeRzG1sAh8Yy5B12d9sJBo7y5o85SK1EssQjRyJ3R8\\\\\\\/BHZzBB2O3uuoEY5tcZyUceMBmJsnlUXj5sMRnB7d58rskQ+CN7SPJUax8p5WaMmi2YNMv4m2FpKSITqJ8aQ5LTqJ0qPHp\\\\\\\/Dj2dMoBRzG6GS7l9EZFVdHbuA9HAHcK35SvdARPcrZ9xFziJ+yy6ikH45Tv\\\\\\\/USzNlEUyw0iehtV9DNqQaxsli5NO455l8yqdVVSGGR7yn89xYojgcPOchMSs5NLn+eYVZ22PmOMclMEAKUFVK2wYYGsFZG4\\\\\\\/FfbMntnNAU7nshx4KJAt2uc7CHDYuIYpa2wtwWNk3TDnrPWH\\\\\\\/\\\\\\\/89\\\\\\\/kMlWBFLrUqBCkxLVZG\\\\\\\/jwPrRsQgYU64wxQvvXo03pt1soqDncmGkCmSXRT3ApVUyKyoKA2aq9J0aGuS2G77687MUdvpUb4cplG0m7s+iyq5HAv4WikAxU62Vm1jGgfpEwc1DiyE0IeLI0KUwfqfyml3EWcg683hQDD69kU5VsgZa0oC+aRptsVPuud+bwPDNGGJY3WzPIOGHXKCyp3+dGfWRN\\\\\\\/35uJQjK0v1Jv3uhlCohCH5Gtlqr0woixG0ogs1d8+Prp0XPY8sja8+SOr\\\\\\\/s0cDeIPu\\\\\\\/4lcU0E+FhNSwgVwnQnj5xqNy2V5dVGJhlRF812vRqF6VYNxeCKuyE5FvAC4DMn9muuH1QQJATYwgkEPGE+uOTu7VmkrGVYhNfXOrS4sDP0bhpIgKp7xyAeJr7YdarrUjkJD85ymuU46tKSNCtqvRdw0F2rnrqlJkgWjF\\\\\\\/kiS6ni8HHt1Hl\\\\\\\/C4Z3MbVCyf2DUMG0sJ2ncyuokKqy\\\\\\\/JL1OU2IJF0NoPZ3Rt9DMXKsVTTm6iidlFX3nJnTpP65Iaj8FhOmBNb+qAhbf2dBF\\\\\\\/cZ+REAo\\\\\\\/R73WqMgJ\\\\\\\/S0DQKliLs7gfhc97uYfrSiyxWNMOJFoVpNLXE+C07R6\\\\\\\/6njtnRFNClZrHx70w7OY6BrZKjqwKQMsPdzKepY4tv6QC9Mq2jZIx9GrhZSX3SiXIBR+SN6YQSUeJNNk1TgDk5OcdHYBi2OpNkRTVDvv8cBUxX1DRuSLAwsKu1L0KXqdgKRezcGlkKTCQYzGhA9HPS+yG95rxBBq1szHHVli9\\\\\\\/eF697tfv4D5nmijgQDJskfnEIVjwI9XjEELZPyIbFl+Q5yWq0PrlpIE5HL1YJLgkzBjU9CoMY9qRSvS9UuyhEnXrkfgQTvIwHyLPYk\\\\\\\/MrMU72ZQgKN8AsEerIEIFYz7LJ9RyOKTg1C7qVOlcgyMz\\\\\\\/rUakpKEXdxmt7caChIq5KmztzQWlQ7X3EzezXBWenUQZ05R8fQq6iobbzuPAkYOceVE5h7kFjm8Rg7Yb1MnjMP5GdcO0nz10T3wLU2AKs\\\\\\\/qbKychdv2baC6BVBZPNLHOjIGX88pDiYcwwi05UNqMlpfv5H1qV4t9Hdz4wrYW0pzyVdfFviimQsbWvL7sGrsFc0urV9GdUCxt+9HyxTYv7yuYBVSA1bYs8s1v8KFd9eCgaR2nIyPtWvo8oB5q4pI9g7czlji3Noo4t5cjc\\\\\\\/G14R9XUMz9+hpk0TI1wMzj1fULsU1Q\\\\\\\/SBl4ZM5ZwgB9TlvpVvZpwfFJShAGDEP+3Jp5OlXi6lLyBkLkYv+O426tPM6Yy\\\\\\\/ELxGnoS0i4aybbtgCIENkQNM1+Ip6yYoECjL0UBihixGXBQrYv2Cal89YMWZB8K5CBnBx3TaeYANIRdSt66K4OM9NaxQJpoPZzukYYG\\\\\\\/W+Z6JCxP0Xa4Ll+gRQGEcx2rHMgLiCWDN9QjsElrZDNOwrOEEEo2g\\\\\\\/9qfWbS0MNg4acqqo2AIpLy9J3pgdV5ZA5zJQMxHGO7AtmTbIjvk\\\\\\\/pBcefB5ycl7F8XlY9McWs2CmGuHylUBdLZHFwl6ydhZ4cif5Ne+bxLr0ISm5vsNUVx+HXvPxbdllnqdRtPTNOSIVM3MBb4t\\\\\\\/J9++S1D40yXEVbW+M1HPIqo5qw5z3fxq0JvliR4KCTGOq29wU1GhaHQ\\\\\\\/B\\\\\\\/1bkoo1LLMeNNadrw7A\\\\\\\/bhnUDmbc2l9wG3vOsH7hGugME9+dZQRHPyFNrn8fpSdy9lHQw8AL\\\\\\\/qgBh\\\\\\\/37ozP7o7scVVVC5XMfLxPjjI0oDBV\\\\\\\/8R2hUfgxX9usdO568HiF6elNEpQiGKIyVgR+DttBbBu+MT2sV+g2MGhf\\\\\\\/zKGeadfODa7sEhDMnzAFGUMBzSqDGh\\\\\\\/yoaWLQw4LlpuFy+NoX2XmoD7512e\\\\\\\/mqzSBj7V\\\\\\\/Biz85j+PoqsLlkDXhe+lI80xySfv5qV4qEhPBl8reXgzzQbkHU5TRVkiwY80pOEr\\\\\\\/ppK93h9+xiGAenvM\\\\\\\/4n+AQUZikbscS7HdpWDrK1x+O6coXt8tbakktss0xj+NwMkDzDVxRg\\\\\\\/4ECvlseDKFJyq6wUbvMz8lzuPmYHduG2SwbqUHvZMiaHChaMDYJJwjSNIwbjVjAeO1Krtnej8OhGGz6wdMW4xkV\\\\\\\/RxvL3mQAJzB\\\\\\\/vkMJ+urVCJbAgDJSQZujs+OZ68yYCKNO7rmoIL05L39vsbXqsEBNT1vmqvbuveOeurV1Y7R099x+aJYmEa8ZDDvgMsUYhQjiAbneWHml+TRZEy6keZ14fcyqP36kAO6nQqSELqdQQoRy0Z06Yu7YAL1F\\\\\\\/n612oBngKdgXF5M3Dvm1fGsly+uVSF6pWEKSJOk4lOn47k4WJoVTmdmVbK0pk5xkTaCYgnONA9kinY0YLSqyl89Pb7fuU7S8nTjR8tuze7majqE5ZAJljT0h+HAOo\\\\\\\/6RPBrtLgSeDLpUS83P8KGSqlHvttxg3F3ejH2FhujpXEYoj7dy6YDEToShnuOuQnq1uUHPESYRYW6X+IsIr1D7p1mTshgRAQ\\\\\\\/xW8DdhPlINThMdFy1vqmd6LKxCqnrkUexHS8\\\\\\\/8tdr\\\\\\\/OAcxqyJaP0v4DMLRgRxLC4FC3Q\\\\\\\/0IyZJC+wFRn0YgeYhzuLIPTsIhJjWBySckG+VdVd4\\\\\\\/lX+26efHFqThf30y4XcaJzNLOphvaO8p1D+jsNiyekwn\\\\\\\/Dd+zUubq0THuFdaS7WQBzcMRbc2qXfbeha1dtZr4H0cYp7qHxX9YmaBhgkqdLhXrHpTYrqVCoSqvXeHvaCYFTB6orKk62DLmJIFxCwscJQl\\\\\\\/7PMU4zBhId0rkyrDnXVAXTXykZS7udqDV+XRkFCLbdPWOeAkOzKbBaolcrUofmT1j8NyBYG2U14RjVtl8saRmub8NnGmI7SBJI65ytgFnXF9j4Gf6+vXpKGm6Zvhclcv5ZpshLaq0NWfQ\\\\\\\/Tv19dpWyuZnkijAHp1clT+dqmubcWV6CtLQHg0LvcRD979SvvNhcvrA9ejHBcELZNV8tKWyRLO80c02Z9sJYY6P1Zv8qLjRFw8v8QxNgfoUTfyz3yJeWHC78VtPByakE1GBTf3zpU3DnNKrKPvtOOR6KFi2\\\\\\\/PR+qYK8DSBdzOtRTZcKRDVEkBI\\\\\\\/yy84veSls+ckSvcc5B5coBB5jy7YdAP4lXgIXgym20aNF9v8VPozn8BdojFHrfDitInqQwVWG0EnR6nmvRFkZpwalI97PmCqGsm\\\\\\\/J2LAeM2f7EfkuogvwlEMcAqtAi0UPs3bHYPG9RW\\\\\\\/UoHGtHLowB98mU2UdnF4BBcpPUsU0IGmC9OUNyEMRksR+fVpn1iS0IoSoOrAvEqpsuo6VT0u7qOJmJaXxuZcx73KQT4RXRb4bagrjzkfYqqt9gZRZIVhqO2vaIBX51LFbwtLD0Kd1A3JoiCHcdOdbIAyIbS\\\\\\\/GPjFuFXLugxE2nw8VjWFLkBDudAb4kZ5NYd6JoehEu6yiQf28wyC8tv1gQRE4h4YVhmvKqWvs+1CZrp3Fe\\\\\\\/\\\\\\\/z1oCh0enFUX1r+XCJWVKIWAP9MkmKL3B539zRfWJWKVAlySrB9LNLWXcvKS0GrUfiJOuJp9wwx31P4y4bpBJFuiPDn1\\\\\\\/wYwmBdPfdUBXe9K1BTjBYTkSn4ABsyZSu\\\\\\\/qqWWAlDUojUBjc\\\\\\\/gXZ312p3o1sfKIoe+r7I3i67zPUJH00QgwGnOWYgbCrlr6V+pEh8L7awVC1Vt+uSepSyZcUdPwGPP4UtmAwL0s03HLl69jofFg60hM3tlz0txpoVebqodoDFJlpvdkS5T\\\\\\\/vvpCr91Qn\\\\\\\/184eavkxgtxv17U7IpQ7042ehYVS6cOm0mt\\\\\\\/bd4fyNLoBLId5kSgJYOX7HYfJZkAZ\\\\\\\/8BR\\\\\\\/lDGZGCvHtt682Y3cRejLT0rvzKitUNw3NK2qbOC4WU4OGb\\\\\\\/AZVUZOfigTCk9vBX+ndOZrZYqRJ5jxEQp0znxhNe5RoiO5ysN71ftMXJjp8qN5h18QghejiOgu0Ws1ma6Zm+xNgYeZv5L9VaGjZ6VEl1sw8RhET9XR0NXlXNE3d+1O8\\\\\\\/NFsEihDGBWtrv601lwZt53acW7sNfnxNFgK9Fs4FX+3yNr55Neo\\\\\\\/JkiIw6WMvTSRrWzUS9YiK51zppJX2hObXkm3I2SCkYR3bVtv06ESsgKPYDcL5tZh1eIZdcj9siGLYi3bkkV6DmTQ3LdLajnVUEUOlvF0HwUeHgsIDlmzqzaKbBT9R1JoBe310BsMkJtVxEhkBlVO4TTtnMvx2b+sYv8Bkie\\\\\\\/kBSYe5SYJlOMxjFtqJZPAxzWjDzCBhet0EjQyayiGEvY1FLA60Nf5CoX8\\\\\\\/u60jIVeXD\\\\\\\/E4fqofLV7iBjc4+onKbs28AqSNfTPtXAg3eDmxVOnmbKA8ftTTpNSdFuthDmwjI2pvtz5odbnEI2u9g5IReKPHSPYChKDuzMNeGn0rS45Tyx3PJ5QFi4Qy5At5DXrQ\\\\\\\/djXLc1IX2btRSDcHQzBxZZt3fjEGIxNPUOPD28FajQ4++js5UysObPEnvUOgw+pZrNuWjSHWL3FTlFnpyzlG3mS3J6CyIXDNLiBxKQefK8EU4igLeTJ3cEedbc8vKoxjgtMfL2HswQO45UpqWuFGXXlMBkpeiB3HHuGillMns8do8pCQVjnUTnvOlvhHeQKvvu91LmXWPyyXzxg9PR0kg2ujKXOZ8feKTeVR7uO6QAH5oQ06Gs7zvF1h6MwF4QNtioEFjN46BOgA5jnDa7ghEe3Q1hd7ZuSMXdlxAGlV8dvE0Iv5ICHvKDGCni5UXn7fZsIGxyl9iD1kOH1e14NV8VT14Sf1tKMrmuhhSftppEEjDIEUpOqNnwnf038dRGla\\\\\\\/CRAf4+MLMUw+PgZjTE0y88Uik7BXo1JlONnBVmzKSHsAy\\\\\\\/c94sAILvbevWGyY\\\\\\\/yWzdMdcE78xu2VrW8QwAHKxIOalVmLQQCVnQu6dF0lx57gCirbLtqDKrnIbxXkPyPreEfZgTyeL\\\\\\\/NwOLTMBtHk1GTkR4yVSepmy7\\\\\\\/7HJBg1F0N9rMBzSz90bsDORY8uwpk3BZEiqofNI4tTQlB8QcIN8xsSYJSsZcXCNw0dl7tkXcdVJxeUqtDBm\\\\\\\/6wOTVMPeuf1uiR0F+OtfcLQEsFyGNRVvgHr6Hr4fyVaws\\\\\\\/\\\\\\\/\\\\\\\/tAuANuh2oyDIThWDCXm050hhl9FA54fyTHPVXjBOZ3GLpUPR8uD7cLftjzaLEYOiLCbfl+Y0xRKyWZeJYw4JC0jSH9949dhoZY6hizzADvEgOIqha8QGcvD+ZwfcHOXtjmeAsUYYo3GLw8swNISHG78s2TSKOlGegeDWovEa7jRvkyGWdXXWZi9apsHHt0jI4zJ4UHbZ1WmV7FHvjmTOZVPDxIPxsdueoRGMfH1LiJJuE+K3tkori7UjVrz4WKyA5FA2FfUHflQV2CnHQDwYm1wvZJfu1FE6M5+cHnMSkuy\\\\\\\/sInx9vYhTDMukPCEjVFh2ufe51H3751C\\\\\\\/qtPnNXV8U2cBSr3VNoanRo5W+P1miQIX9Ukl+fSCVBwUBq1f0MvMxMF6W0yen6ZIj74FFA4SU8abD2EU29tH\\\\\\\/CkWjrM5u4Tac5t8u6ImhhVNbicj\\\\\\\/eQhvldQI+I\\\\\\\/zui3iF9H2V1aS2ofV1pHCuMmLm2u0PcN8m24DQaPzSqtOpcBnWXuw2cTm9BLgsIfJ6EWKIPyCtI2\\\\\\\/e4rIUc1fHuucsNy6SYLLR9G0WBbPBh2gY2H+kTWAdy036LgQO0D0zCgsUlNXxNOincA4fdmO5Pxk\\\\\\\/wLRtEzqS0j0rQ+h8J5kRYKOe54apEMPt24a15u6mWsWYWIcJvv75opKbs4LBmoaoCvgsNW9\\\\\\\/Zfgb94TN2P4wyOYRJiXKocTJba15DNj3G7hKnYTnKzcDrA3+d2GHCPoGzQlcoObrfAoNThIGznIGFh\\\\\\\/r\\\\\\\/nJ81vv8Z4yGKUQUOoABVXFYdi9AYfv3vIbEZYt7+e1NuijqwtSgZZWcGqfBZ2PykMFZohcAhOPv6TpGMWImayYaZBUdEAA5oMWWmpm6LwGFkGrcp9lEu2h7FXT7sBJZV2EpZNmlQqsOooJ7VGgcbBWJctmaBCZ3YI0h\\\\\\\/vrfD7g+I2zByyy4TdvcxZwpyTFlOfg+MScHGy\\\\\\\/KQNL2pDthdSAWqHa6twY05iZkgcNOlGTmDWiGFDyYeqnptcr0iKxhn9xS2sQddXz0sAOzkVgrEUDle95JvQaI68i40rjXQcH6jPF2jOj6lvdCVzaj1oEIjHAFHyLMgdi71rO3WSNbxdJTA9ASncXQFwaTzI8JiPozTYCXXWFpDBn\\\\\\\/R+Itrvxkvvnw1ZJK26zcAIoH49D0Hy+lI9EqRy6zwiMJ2WpAtBHD60eIfZf6vnBEjwJd+qtnaxBuzqJg8Jd4lu5x3iN9Zf4RVQnLjSHlLltniz7VqgzJC8O1Zd3CWOplbEfL3BJGg59MYVjjTcIuQv44RJ0DoAEj0qNroLBkpLgTc00LK\\\\\\\/qUxWB5WNLRZfzfAgC5OfNDvpe1wCuiohc3g3miD4iujUYMXjsSrZM5Df5UZeVeH1WkYZa8QDRrcNSmh9mtPsPmqVhiyzqZn\\\\\\\/keCaL1sJVn4xrX02uo+tmR8uVvCDOAzqagr8niGX4BXyo2egI\\\\\\\/lHxTW94bAn7p9H\\\\\\\/\\\\\\\/W3vO\\\\\\\/OVK9DBh\\\\\\\/DKs6nFsfWp20kHs7omjHEXWX1spbdNgMY\\\\\\\/3m+w6qdC8KDWnIx1NgL3IFPoHiaPPAJAjNzB9Rwm6\\\\\\\/j+4wT9ubTsiesvRjjtyjk3dQZDHu2vqPyU1QRneF\\\\\\\/xleAxmeDXYr\\\\\\\/gOTd0kXvjmmFEI9\\\\\\\/g3Ro9aX1vlenrnqp\\\\\\\/OjQTIMNy9V9jDsCWCQq5J9XEe06G3TROl00W+a10YBXRWV+RBpbComzvmvCyWMVKak46qbv+VoBkkx3wVzvoPlNOne0yWFdLB1MJDKhboNu084qWEKx8mfcHnHDEKmqfwBtI38PIlfGXY1Mkno2VuIpQmy6RMRKRg6CkEFeNq2UTIM97VWyj6CnzlfG9+ooyoJcDFcATxqCINQ0zNWCnenPJjTTGSY57j2BeG7r1XHWH6luGNZQUtxacXqHRVQVcMKsbybMz76gDq\\\\\\\/A+IcVhiz62BHUbRpWl3FatBfU9slg3tHPKRLBoTOzBbUO3bcpvkYQBpjNCXU\\\\\\\/SASQr6tZzcvXlG\\\\\\\/T0dooPukKlMZTJ1TbZLahYEbanrxBNmNlRG31yWF3siQfYf02VkHNPpX1Nglqri3VBDL5c1gnC1MurULScrKOnrwaJaygQqUvV6qtJWsZhYcUEo0\\\\\\\/+zoQPvzN33U5kmERkGU0kk3Zdv48eoSTDcsfQblhZFS44VDmDxiPekkWP7zOLsQsVUJ3ehWmNka7Fp8re5rcF2tom0WPZBpFzq9c9EGlXYOUd+ergFaPGeDPIjX7bqBLys5XWiecvGWaaWUrUNSEVaQjCDj06sh4xSMOrG8cLpGjHW11G0dBc6FEBKu44zAxJMcTpqeb9dgCECgKkGYl6zpn7UQ9C57qnPFAeBdtMDUp95RflVm6qOL6jZ9ygkd5UxqT7nqNb\\\\\\\/vswWtyYoBCxhpwyZj4ZNcBRkzPZFHrMtWoWMoHQ6JfUjl8J4IoAKdVxwo6Nwdk6lo1ByVf9fCOBTi\\\\\\\/Y9+sld7G42a9pUBw5wVeltC+Tg1GsvnvwqJKMsjnldK6IqvtGYlJqfErNHXYXucYsuHB69y8mavV+EVVoCoTdJONEzSgZujr500jQvogZHkqxOhVAHO1eW09U5NL\\\\\\\/r7RnLQfVTsQHaGZvZ4Gvk1SCNQJEmBVK247xL5cD4ACnGta2QPGdMD\\\\\\\/hwayXL\\\\\\\/TvOrv9GmYal5lsO2pQtdPuTY5yvhYHYPZ5KdxP9HKZdR3EyGqH\\\\\\\/4YuwNKctgA9GlJIJQxqG84yaRg6cCwAm5d6GnCwTqZYGG99M3N\\\\\\\/dfFuKAF2PRBzgR7CMayfOezuseDk2hEQLymcKvDl7zNtR4LFiVsRrLsczykrKtQBIXg8iYE2ajiA1IuUGBityUec2+V2KwKTbOEeydLDdEWFi4gCdTffBC6YjiD3AZOxG7K+EB\\\\\\\/8ug1wxdy+oviFouTmG8nk8DzeDBz\\\\\\\/FwL\\\\\\\/Hh58OE0upeJUbKk1XmES8hNfR3PB\\\\\\\/PGtUHjcdZX6WsKbNwhoxZlptzTM4Q9GPwfljWYV0T2ZRRyN3OEGDDQYfgonSWm3314IGiTRRDIr8DG1bUUSwIRZY3uYiNan245BowuL1+1CdDyAt347+vyNJpXi0DJK0DrImQ3k+7OddX9FamXsgeZvkVUuqj0OD97wYW9oXH5robe9pb3p9YpZ3RtKfA2DHgN3J5xLysA3yPMTczvDyemTe14ggmGK\\\\\\\/abemK+nH5EtuCCKGpsATiIDcc1pSpDcgsYLs8Lyjq8FtUMdxPYnlU0aL2dnsIQfWR6nJUNnp6SCjMiu\\\\\\\/Qa7rX5\\\\\\\/EcCtrZ\\\\\\\/HHxrSRhuEXVhaZ1PURDO3iOcqSUXoZsuLdtouGifrtzCprOEqjbeVCLjr3QtqDxBPqkPjWm1FvQSvJuyZskfVtDIsMmZ7XmU4T6g6ZQLohnwLhtkYpt7hHFERJtydbQQ\\\\\\\/2q2BPbWL36eWS4DPf7ZObIPi1RTy3qddkRclH0PWOf7AFv7zxLxXOyTRyANYJS\\\\\\\/0KE\\\\\\\/Jf8B2tOHb8xaTq309hfe6hL7Vt4zpwTKZrL3EmKCHa7rkh0iDOMdyZRt2yy8OmGZGxNA2Y1KXCTXMXqTe6wPT+tdfjHGbHSeo6aRIzUFR3CLYc93SLS0c0r18ZPijmn0kCbgJ2TjDVl9RdLfafw7g1fu971JWKzZVknTpilBZ89OAtcP7geVmNmfCgF7IJihDwHaD8lZcDu6uKEJl73LU7J3s7FZvCjrvbdBokkEu871pW23Ux6AS6kzihIlds\\\\\\\/oQpPzgz\\\\\\\/GpjXPfLGK+j+KnecCet9BJlcenOtyCv9a5WUx2eWaAXh6VdhiiGjZM0SGp9WvIebp1OaKkb8bsMueFffoC\\\\\\\/7c4WnHTV7mqrBroaIq\\\\\\\/d9mLhXDrq5Qoq\\\\\\\/wgqmEWVvjQny2L8xU8+3Utrfs6IE4gqgxEi\\\\\\\/Edj0ZyCiZ4p5ssTRDOZirvMR3ZED4J0Lt72s1sD\\\\\\\/5IPpRgSughN+FJs6ydJhE0yJsq2Zo0UPkrjkGEDa66JlJAH8ydcErgG8vDLKOs\\\\\\\/3PNJH4Z7KVhusVllljGG1jB\\\\\\\/y7STTZULEowFPh421ZwOwyp19TUgEM6h9vEuWFaG9nMECq0nV907ULw3ezVmHGSLrLrAIbNfvvnyz4WqPMmwiMIgFcckSyCxbJJoFkPU1A8igLHbNx3RlCRmlXzunjTbWaeUJA3097tp72kvs8NJVterSp+B2YIi4\\\\\\\/udWftgVF5QRQzaETHY8vOwJ92DR0rGJxgAx4Ntb7vovP7U\\\\\\\/m4XlS2AW2XNeSbRn2NvNQV7LWZU\\\\\\\/Qa58neXZv8jPvnfjtqmOG+F8iIx5eSIOnniI93jljDd6PSyhLDUU66gKv\\\\\\\/mBMOz9\\\\\\\/Zl3usnBtJ01HsitaeJwTVk2ERkJGz6YSaXoSAYxbMa6\\\\\\\/NUGf1XsOsEV7ch3lTxFpxPKVaCEDKzbEJa6L+WVRUm3mVsrBjZZyPH4nj10tnY9XoQsbVq3UIcKBhqbQBxcrXOTExrvjeesfe+hcG\\\\\\\/be\\\\\\\/lsesFITFlciBEp6D7QIZdoSCpdyeN7QxYq+kG\\\\\\\/sqsH5gGzA+qvj19kPavSpnUXzDD9p+8tJNOLfnBvtAAWkLOGkygS9NubNaqooJqFe95ks0lUai9AdHNCr5Daucd\\\\\\\/EDLp\\\\\\\/0+WZrqfY6JxRGb91d+lecjkKYnUpWJS8PZ9eqtFoqu6D3\\\\\\\/3Muri3I\\\\\\\/0uS6oTiu6T2fXhVzNMf3Y71JyiTz\\\\\\\/AbBtQvd5r4IkgNRzwgL\\\\\\\/SLD6DXHeufQQRoCZMAZioUdmISkqgGngo0Z1NuMcKLtaaQdAXc7xDcFI0O+nVzdXrkNr7MmSQuvPdPCMvgumYAnXH5KpeaqFTl+iRtz5kVnU4CJx3UJ+M8dw8O6tXVb8ekneGvcySZozBsvxwvopxf10ybU16mK7XAie6Aueum3qmcIqoTwsO+jmdQgNAK37kinlHudOknZ\\\\\\\/bEMULEeYd3bHwbPckS+yTMb\\\\\\\/IsgdYsh20PLUBA1i9j5NvShFr36gfuefwEryyBnE0QOJ\\\\\\\/tEKKhFfeb096VJcJeLbGeMmfPJu7565Kua2\\\\\\\/xHIs+xN3vn1AlUaztYS82GnLIhIcixLao\\\\\\\/ctfyeObAMLH1VoEh2LeE1gqO4OV4NH06RZd1wGrXw5x9kdZCI0RZ8cHH8o3OltQAxg6PiqGqCfb3KeG6zPaYf3K\\\\\\\/tR1aoyIp\\\\\\\/EHSdz9rV7xWWtKX62tW6rFG0d9VSatGUxhC30TlOXqqTEl3n2dpPRXDLRi8pBBJr4lm653CAIhn4NkFpbad0aH4TWlZvHPEklmb5G\\\\\\\/QKnvHPZ07PKYJxcxA0esqBMn59eTEf+O+etSxpf2Zmhnzvn3PIYNV+h3UXHlt0dq8Sh2vKmJbMd0U5rPleBi5XqkhEN0\\\\\\\/C5PhowoH2fs5Cs9ya9qJlYzkf6kMRti6Ad6s2d8+pwSgcscgizLRwNEbcQQ6BP71McYwQ39It6g6CtmWtMSNvbmzdK8UoUv9t0tEtSf+cP4O7FmmRv6r08r2k29z\\\\\\\/DazWuT7xnLCn08511Y0zCSlLXMWaw0rTo6TJvoRCtmQQB9ODQ3uOopD0xpL\\\\\\\/BHF2jXUpkRjzS1YuLNbkIkArXYqaujmcJJhZY2Vho1jAsd0zQkkf9lJzJmwEqkouVAJlh0aRBykF3yHsahR4ZSTfmq1wsT1Fp3B4RwH5YLBQVi\\\\\\\/PgiOiQBYvCbnRZcUxkGGGXfINK9jCHOJuuj9o9AcTLxTviUqd\\\\\\\/cXJhGuar8pemm4WIfCcLVPHrweIoFedST3EB9VzxLzcVMbSBlLGyYyw23djCu1vvgb1L9MDec3PQK1DdnPZ8NV7030s4KPsyFn6qHHG\\\\\\\/R+pLYnKioXAW6SNDQLVeLwI+JxQiYEA\\\\\\\/ENaqnZ1FdTtm8eqj5JbjaEdzHnrLWNbMWqbRhSHaQ5dMujb0FjId2A5A7UJrD1r1vCHsYrC5NXMAnvvPe\\\\\\\/16cNwJSt6JJG7+jGvDcMNChF23zuZO920Jqlne+x6bAxR5AnnXRmUKt3+T5OUgpkpVg8dgsOUyf1W5XxCFQL0uqkbXRPEW7Y+f+qI70Jt08gnYX1hX401THe9rNtnczncwPqvPzkKMkREkbzU\\\\\\\/1xZQFUpB+jjcQlgNkvqClJZKPakUGUb5cXOEbJaaipfgtrk5E7JmY4JIa5scz\\\\\\\/jswYHTBJBwqZeI\\\\\\\/3nkn1WSj3831nRA2mR4ruWQ0YFeGKnXpNH3WOLIf9j7B3vVBPjmtkhcEUoS6aCkib5HnLGqzm3oF5YAOwP5pXlGxsqFgrcj8zpI+2VwFjVqtoecQ7d3v3NTITcfhLB2iViRx2Ul2nvOanuL8fTw1qQN60TsOjCmGHGUjy\\\\\\\/NTvUx5xMkCXIk4aGprYRNkvjFmeMXBgR+NiCTDyEpZU\\\\\\\/Ft\\\\\\\/37VTa8EdZXWbM6jTyz7S0FTYHlS1h5jZWyaftE0JWoSDFBMGC+CEqzIlLUDYdOodW0DUuUYg40+CES2676K6kF0f+VlQcaoOSuoQt9HBa7M5M2iIo1iHeXjhKi4o0pg\\\\\\\/D33m24+s9eW6yMv6dY8aa54xzUshkY4hxk9efMurginSfXfFlrYULFPCdxXYRF4WZNKKNZ9hNPMCqJ\\\\\\\/QqL8KrEUB5bzTxQCHm1iCy+2nv0kM2GljjKDf52GE21pgq8Gs6140LH9okFvjPv42ax\\\\\\\/Sl1iMkqVEsiur9Rc4nwj9aRNIUPeiDY825tC3QC\\\\\\\/IAcCIeKvSwbyLbAzjvrnxIxH3T8RzzJngKi1otwLCLxk+6PeekNorzgW4jROko04zgMxIFY45Yvjc8xSQOLaUHoXdV\\\\\\\/fFYS9UNWaUUOHZ2Ilw204Fsvur3rBeUDRYI2OZylKplO\\\\\\\/W7tSZhBMDIEWKFKwY8CcZzcD4XY8fQUeDmyNm1fMEd\\\\\\\/TBZY2YPOKnR+SNnCzElIIMiGbHL+3lunZI2y5oi0PXel5zZcdA2yfVlroiDhUEd4uej6l8Ja7w3Al7soIB0\\\\\\\/o\\\\\\\/ErjcQvDCs\\\\\\\/K0IBwbwc982ZwV\\\\\\\/mrJENFBfNMBPd0EBnsVwkD7pDRCyNR4+aO+cxb8MOvye6lBgCU9R2bzmaPTvNBbwMOSYQTqx5X1r7SE63ij244zDdbIrqfm4hxItjCkMEKbTva4C0bKMpb4Hw+A0w9ih2VYaRgORJ7mqfwT2ziM0JzlZCI7O\\\\\\\/2z2OMRTH55PLHTqIspUn20UR75Ithdr0uf3+XqLUudEAdJm+5S+m7DeICTRNdOT4U6FrsAy7KmQ6o7yrN8cGu90FFTVx\\\\\\\/cHT29A+fCrmc1BSwMFyv6kjBfHG\\\\\\\/aEvTYQzlcyo3rNaeBEN9GVLR2I2hv\\\\\\\/f5qqpyjTs2z+24NH\\\\\\\/F1aLumR11F3akn7vro660tgiwG\\\\\\\/v9IElRyZmcWVSWXYUO2bfUnYrIFMZ06TRWTPk5RQ01LvoUzNa15p3tu4kmwt\\\\\\\/uc59fO2+hYYLpbklIFZEI+AlxfAEgQ91VCRPL4Y8qtErRHpaDRhv6iwX1\\\\\\\/gWtoC92\\\\\\\/hHOvmMCZQLHb38rTonYXE+0PTbd8cBVkJnqkqeVRGVRs57fZhyB6S0+oQFthPexM6jk29hLFmNadmwPhQUb7cwdYH0EzeY\\\\\\\/uOHltknzd9d3FElGQaUmCL+66cOTofD2QHZeDcpDutSXbtPku3Go9XjrhPeW4r8iX5VrF5YHD5HLedoTpgW5UQxjPJvnliIjlrb64O0DkAYMEMJQg\\\\\\\/czbuBPonxcdNxc57TjVhBfDX27yz\\\\\\\/BJA4XHOYa3IVZVdgZcR8aLeeRfqKSXl\\\\\\\/eK4QcdRNVLN5kGWrXe8N7KbtI9EzqlwubnXKD6QUtEgS8ZT8qIYn3ZbSfXKJgCigt+A8PbLEnFRICpgCw\\\\\\\/t9YdgmRYzCfzXCVd\\\\\\\/h\\\\\\\/NTKVjIJTmuYKjeCjGr5Na8LuWgbwyr6OMemAoRtMcZAywulsSSYs4L1n33sIX2Ez42jUhcbWopur2Fwxlx\\\\\\\/heQLda3INMPzKCgSshXJ\\\\\\\/+X1++dMH0ipFXDx5r8vmdeH+igDlAM+07vyfBzJWgSB1uHRPZNtNl8w9Z51n78n\\\\\\\/3gOhS4XTiItW7W\\\\\\\/VFcZKUnxUb\\\\\\\/8jULEdY8WPDpO30rSw\\\\\\\/eWikmGUInbatJfa9AsVh\\\\\\\/qH\\\\\\\/+ZUVqU47zUNIo3G48jOmyNOzOl3TW30fkgYtU0vzild\\\\\\\/a53\\\\\\\/b3KXbpj5IWvLmX+ovAloIx1BuO8hOlWn8lex6fH5+GOFgwNgFC9SeF8ZUmhzRqub9WIaDFKCuq1hCRDKTHj4sqUbqhlbDhZIF0uWsXPaejMq89nw8gewL5+I3pyJUfXWEeduEmGmZTkSDacsd2oLsr7MehBcqvm+rfnitUBmKN3N92b+izfbdcossEmAprXHoh\\\\\\\/1YeB\\\\\\\/w7IjcfBwoql5N2VDt09Wze\\\\\\\/K+zmnMn3MFVbwVc9Sn7UBrRPZFcJJLt\\\\\\\/Tg0MXJvgT4GnyKX9FrILmZ8BkUhJD9JaWeoEZiVmqta0Ea5SiVOTZav+13XwHWLRZG\\\\\\\/On4aOkU5H5z2uX0dlBNARExp0cmqq+FuH1xOZ3qEvLbTkboG\\\\\\\/QC4aHZRx7AJLr0qNeFXBXo\\\\\\\/BglspFyU\\\\\\\/xvAxvQ1PWVTFKjIttxK4VJnTYzBwVsfpBaMO7Xi1heeWgaydRFXhwBEt2hvul13\\\\\\\/rDklM+7ICuLD\\\\\\\/WYMHP6dP3TKyX7gNdGXxIEt3VZXKQ682aXEKQ0djQBkWUH4r6XT8Iqv4DASykyNGi40YhfHB\\\\\\\/ZvQHEUNSrfoZsByQmiiUwL32N1SSDdLt71JQPLvLk8Zf8aPCWnLfHTjf\\\\\\\/Iu9c6El+AZYFIpX47tiGIh2A1Al1zla+qc58SJAUWBHGztXJ2bL2Uj9MEJmfcs28EChhduUrIMR1DSHNtI2tr\\\\\\\/sA8US90HsiK20nNPrn\\\\\\\/Da5OqxIjYnl+tif4MRQ7egbbWSEToehU9wdBQJyv\\\\\\\/rM+zePm0SAJfprFyVm1otFJ5MXQH8DICgNgw7IAXFpwYt1B55bJaAuGVXCvti0Dhn1Piui26EnELIr1fThXBqlPQYl0dZlvqkadaqJ3KF0EukzNQlySYEzmAkEYHS9Hg1lJDfRh5PEbR3BtRQWzi\\\\\\\/2v08o7nozVqhKB1A\\\\\\\/4S5WD9rAvckRGgqqqvZrjA8X2fA1zfSrGaHtGkQxJHhYwExw+1uYUJiBd8ooccVLzOhogu\\\\\\\/VNoGW0rt\\\\\\\/R7yD5IQ2BshxzASvJO8A9ABBVpu7FU8KGpDLr6\\\\\\\/AUgdQCiltihPT3MlDiz9u+m0UkHUG4L96XIY113Dxk78PbHy0UaLP39V6BYKQ59MwiXaGCw6+5Ohq1BY6xLdZlgavhzI6TMnEf9mW6tz02qgxDkpxT3BVYCVNIDQp2tcmlpouszhW5MZW5x+RxEUrv+t\\\\\\\/yAJxr5HpVON7R9M4GLTmFM5cNxn+c0McfDLG7UdpANjgZbPW0ud5Au+EuST5miTAe8o2bqBSQXhDiX6xhky6g5SY7zECXBGYQlnVP1dxHTe+ahbem3Lig3uSBXT0DZDY3QGgvFa9y58RSGCnlv8TN4P9xqygZf9loHFa6\\\\\\\/fPd6NDk+KYqWAVo99c2sLbg+cmwgaHrXiTuBk\\\\\\\/v6H2ZESrnY3FCTxwxrqr+d0YfYGlVqWRqfrlxTRlw4wRlmxXgj6drEInSspiDDa9ghfXP48ZE8nc51HYP\\\\\\\/ISAfZmqrOXOn5WKeUEbZF\\\\\\\/l7\\\\\\\/9C9+x4RYTCn4X8pY6pOuxf+caoI9zeXx2qv2rb1TRTo6yVqjEzKkHZgbSD7tjgmyZ0+vnqeWiei1ITCa1Ydz8xxhl8WUIi2XjJWy6PuEGpjv2YXTJg6jjCzybHVwWPXnH6neHUA+AcyhTG9PnXzPdWaqBUhkVcEUyKEn0MdOsavqLCd65s8KIN1rsuamDQ8gRwmnkzCvZh0rDS25Y7IGiXN3OdCXJdDKCR6pfEX6uuIYxyVmdP8S567CZPTz6fVYUkpnrGxHVfVcSan4MXOJeNdsRVCob3aOkNWpBbDbPlUTi3oyakCmn7cIWGJaDP4OvaixxcxWn3pAjCpeLuFz7wgVqL1dfuFia3KsthkRClP0xzCkKbnzlvlnMfqj+j7P+S9AtDlDM5f7Me\\\\\\\/+eoBxgfXVSMk5tAttqgUtoiX9TPsuDajIWTaoGpThHIizj2icZba0qWdHK\\\\\\\/glF2K2jucvDQsmkDq6xZGbGbG6uwHhEy3fFiDZsA7Hxl1QGjDBnAus3XFk1yWhzZzwwm07DVomakBSTCEyxi7TzxVLWG8CLH7iz3L1vQBSgrgSt\\\\\\\/ZDSkGjQAjJZznYnmxRDNdMAT1NVYWZb95836JP\\\\\\\/FBcxXVG9xewbTmNek\\\\\\\/bY74SmfS8RiLvi7wLsaeG6hBgjMlgoFYDrMFblzTrsNx1ip6nHve8FpXiC1CYdGcdbWxWPZNV0R3ftFv41AatDnQQF+fKT7A1\\\\\\\/eXL0pWDQ1fpbOE4Z8pq8bVskg0gJUUi\\\\\\\/omwgY+hHKws5yPfc51ChsxPJFFrNtc1LFQq78mwXvfLnPeXlCrFtUnrWKj2orNSaGefFmwKL9oPYA7iJgsap6BUYyRpHGmfRzoe5jd6wh73QOrZoaBrzExKOIVa5dOzzPBM0r9rK\\\\\\\/Pt4e7TNals92yPqWy\\\\\\\/4rBtZLRzzGuMXHTx\\\\\\\/q8bXH9nC5wRGOXnBAEIDyiBLor94zzX7hW5DGkNf34cIFKXa4DqSS63yOfu8UadGH3pAE8Zxt8NBD34dv4llUjVrPy0ZSwdFRW7t4olqpdc6ij6Z99ghJVa+f4Ay2nYnRuORFWxX7ycmFARgJgB28HrdFQwJj6d18WCHxCF34JIfWbfqdT\\\\\\\/2WXiMOde1PzRyL2sYW+7P+cyZL0bhdT003IR8SSOd5WaMYwS\\\\\\\/XUf82mKQXxEwNFKNVfhctUyNcXrWkQFO9Bh0bYh\\\\\\\/AJMtFy7NHxkm89sNVfWMOYBNVurgeqkcVKe4Yb9gR21lb\\\\\\\/xmEHL8HMnEUtIEftPCA7bfwwPitU+1e8bsBqEhlaYmvxbUcNnr\\\\\\\/Mv7W4zQWYCceQYNOiQyxu5SxpPWomhRSJOugH2tFrPerS5s0yeZGRAXhCoHrfEsI+G+UtA5ys8NicXbQ0Z5gN1MFFp4LHaKHpGpmwh9jW9uTZObB7O3+oGw5tqpWfXMPjnKoTQ22Mb\\\\\\\/\\\\\\\/7A0zVnwq98k4X5mV0yhkwlYvg9FlJRDi1Q2xawGoNQ3m1jN2nRsRAZkVl6y8uVLzA6HxLkoFV4nQm0gz7slKXxIxdxbYQSeQlAZbYwHfRYZB0t7UeZh3iywq0ofE\\\\\\\/Ool388N9nz7TiE2BpcQmY2ZVz5uwAt9NUt0kRx9jo9\\\\\\\/b6n\\\\\\\/RG3v7SlCs5QK1NosO5QK5J9vtlQbncLF+f\\\\\\\/sNH78xrTDvsNJuqllTm6kFTElUgIiYqKDdwc0OyBjvOH5LlWP+9LjylY4dn6Lk2HvBZSgpYnUIt\\\\\\\/ezIqhPidVF86EHE9tZKtSkFHJAjSde5aeLW+Mc9q99qXQ8qRIwYCjA\\\\\\\/1mWnJdg0Gpj5EIeMBxs\\\\\\\/Gp8+2XDhIuFIhSdht8eRU0dlNM4\\\\\\\/\\\\\\\/FDyRfzsDnLVdHXcnKB\\\\\\\/Dyu\\\\\\\/eWHWJs5o\\\\\\\/VnAD22aicYximYusyWk37GSFj0scY3RBiZEqL0nm3eX9ZiLirIdYFyu848DmDEGJMGP7qNx3P7dxT6+wV9RpEfJQ6wfyILHZj0UAY0gQxpe0qlC2zZVbI87s6T1vm4j8Xr6NidVhBhfyZ2726xbE4WH9B8tu0yUdq8NC0aAiL5xhcXecqcEJrm46XRXUNTSJU0lLpTaxNZKdmoclE86mTSnmM5QwaSLYayLWB9lKxdxofuh3ezoiqTUBO2f6jIhOQmmwGT4TD6QAC3qgoy5LzxRmzayGNBukWDim7DrabKf+2OnthDhI42WaRRLvkIdpIg120QP1SR1VHdYxHThyL5uLXOrXHVrGhi1lgpJ9psXcQ\\\\\\\/lqzb2lMFiaEf0toTMpuiTk7M4SU071VmkpDv2eOSDEoqUNtYJWyHRhg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Anv7jlKWHq711zYdnu8ajg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022469ed757bad9003c4fd09ca5c48c5f32608f654705df33d660b1a7979585823b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PZo\\\\\\\/iytONFLvmJ7wdvkr5wp4DtnO6hPsAxr+sFAQGmEeMrBRJOPI21Eb00jk3XdscgLOjEoNKGziUD8fYQYdjhsvLEMPPaZYkgrKdlOKF4Bu8sYhWppZr2kLIPF9h4flGW++cfI+dnDVkJo9Ekpbk1Ba8VyhdTX+jwUJHkO3yFPI4dr96+APlVA1WxrKkC6LoPqihkKANVwbUqu6tRUU1hpywGO+P2olHbjaDixA+z0ia4wt6TqKtdls0P6MhD8Zq0KztczNEHxfbAgO0yTtB39t7pYeYIi2gTkPXDKuHBVhWiFLxhmJT3IgKoYDBXzOCUGjTbmDowlMxCHCo17E+\\\\\\\/U1AXHB5V1BHOIQqrmsPDhNGTUZNblo1GsyLjPwa0lvPdr6PZuIFaDrKReJ5ESHzC+ZvAn6JrM\\\\\\\/noLqmy+pOEW690mhpwN6kRfRCm5tZyxQMXMeZ7Je+E0eGKB6F3SPmwwmd4uFN0LZtqgjh2WauQ1YDG1tKUDe\\\\\\\/92tno+2VDSKFhjw6wno6RMRY1RPsWqlyAex1GDdhURPE19CbnS13MQRAt6KLC2WRryhw\\\\\\\/l1Gj+Nz65PgmbYXRVNg5uVDAHo7ElyeSLAfYd2RZcIidMdEuyx4G55WS\\\\\\\/zlcrR5PbJ809MSGQYl6Cf+n3jKC\\\\\\\/vKRJQFnTwb38pArYxH9GcED9e43Vu5+KAB+KFkh+jZH3uF6Lz4PvLnnKdafnaBekuHTX97DmVxBvhU4PsrdIf+oNcv6+PKRFZJwyee2bum\\\\\\\/2xCmOGkAUd8dVl8qN+bZ2kktoAQ\\\\\\\/k9KaNIA9matXZlXhqpf7D7VUk0umDFpfrvJn0Rx+FmS6x5htyIrEE88nBEMgEWfDYRyMGaryhQxJ5I0oTyVBTq9Q\\\\\\\/HHTTX1O3eZm8hu2q1M1wPfplLRzFfDVnW07z8kUJD1DShwctE0njc7d8rIFwKrxTIyhdUAHBgzh7KpUmMdXquB51sG8GCCUtYxwAOSG6Qfl8BeFvwA\\\\\\\/6S8XpoM8jKa+DsmQgYaJbQAGZAtODUt9mWhNtu+sF2rXXKTnl+hNUQ+MuPkDKygq64XnH\\\\\\\/qyL6\\\\\\\/+ZLKqn8Xbz0Kk3eXK1O+o8VnPu0wXAVQtHoLYPgJxlVCqID9GCpSHgU2z6VslaE8xy21zRcbTaRKgATph2bEug04ROHq32ytmyCaJe7otnbVIJnbLk\\\\\\\/G+jKFy0wyFcRBTW2GP\\\\\\\/PbzwhYcvGCONHgwbhSYemEywUGAboA0ASzuPpEQ\\\\\\\/GMztI7a2VywIJv\\\\\\\/Hlf\\\\\\\/0zxFsnnmLl8jZ6+I8osOb9SXDt95T4hBsv0SDUthAtJDCoNeu\\\\\\\/MRr9VmCfCFItqMnL5GnuC2Jp764tTEHmXb5bVye8d59W3+Vgzat6TmjyoHy+gp9sylCNkd1\\\\\\\/aBxnTW6EaogYOD9gaBiVOw5\\\\\\\/hpjfu49koNOShFa8XzL9z4jfqk4p23aEavS3AXF7zZ+ASzONfxMOv2TD\\\\\\\/tBe7YbLLuLfSuZccvbbOw4r2ZowfOEdkPqPfwVZTTXq\\\\\\\/DVm582z0byAvZG4jX+mF6BaNIvldq0EeKMKCDmMysJAdwwjDc\\\\\\\/\\\\\\\/z6bXQRvPMvOav9\\\\\\\/c7GpxzcGUnr0rXF0N+3hUm\\\\\\\/K07Wh\\\\\\\/k9dHhVR4aHC\\\\\\\/CSq4OCPA3NIU6Z5QkVj56BE3KKcoLq45lk1QqtDewr8BUUomNsYah+fjo+LPd2ITkFYOGZM5Oeoc15ofTktw2G1EfvvvDzgOuwDxMYKKYso9tlGH1hPUIkwlYAxOAIo7St0oMl33CkWfANG77SYW9JGZ9G3ST3T\\\\\\\/KV86NXpKQjMx48+5m82v3xXXLx2kQUfnUky7O31eGGBUYTwlvCKW0Jg1SlKh35+xwg+TMdbRZaHOJlacegUdXvRUgd5lX8hApC5ytTjuIqo851kpyiFx+WCATDndlqjh5UGioJO03kcEoTg+l\\\\\\\/esghI03ommoeJv\\\\\\\/kJyPav+KsDYiakgkAJSCPw8O\\\\\\\/5bFD8NOlee52kbEjfpng83VZP3hncy4KMqIZA97tMpyA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DWAVtQosz\\\\\\\/LOjW59LEzwSw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002285bf4db13ebbe39f316b4f6f447649165f52f1e962690182fb9552197cfb5b3e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493108 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1283" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YZl7Q7j_hPIekknY9rif582JfG6dzFSHnrz_MXNdg1M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1283" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cOgMs1zaTynzZCh2x7SuL5A64lppOnHdURQbchz26Ps" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1281" title="Corrigendum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1295" title="The Whole Genome Sequence and mRNA Transcriptome of the Tropical Cyclopoid Copepod Apocyclops royi" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493108" id="node493108" data-pisa="ggg;9/5/1283" data-pisa-master="ggg;g3.118.200919" data-apath="/ggg/9/5/1283.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tiina Sävilammi</span>, <span class="highwire-citation-author" data-delta="1">Craig R. Primmer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2033-0758" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Srinidhi Varadharajan</span>, <span class="highwire-citation-author" data-delta="3">René Guyomard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yann Guiguen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-4989-5311" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simen R. Sandve</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9389-7982" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>L. Asbjørn Vøllestad</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-5563-0048" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Spiros Papakostas</span> and <span class="highwire-citation-author" data-delta="8">Sigbjørn Lien</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1283-1294;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200919 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493108" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tiina Sävilammi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Turku, 20014 Turku, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiina%2BS%C3%A4vilammi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=S%C3%A4vilammi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiina%2BS%25C3%25A4vilammi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Craig R. Primmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Organismal & Evolutionary Biology Research Program, Faculty of Biological & Environmental Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Biotechnology, University of Helsinki, 00014 Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BR.%2BPrimmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Primmer%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACraig%2BR.%2BPrimmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Srinidhi Varadharajan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biosciences, University of Oslo, 0316 Oslo, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srinidhi%2BVaradharajan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varadharajan%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASrinidhi%2BVaradharajan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2033-0758" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Srinidhi Varadharajan</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">René Guyomard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>INRA, UMR1313 GABI Génétique Animale et Biologie Intégrative, Domaine de Vilvert, 78352, Jouy-en-Josas Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ren%C3%A9%2BGuyomard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guyomard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARen%25C3%25A9%2BGuyomard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yann Guiguen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>INRA, UR1037 Fish Physiology and Genomics, F-35000, Rennes, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BGuiguen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guiguen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYann%2BGuiguen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yann Guiguen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Simen R. Sandve </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centre for Integrative Genetics (CIGENE), Department of Animal and Aquacultural Sciences, Faculty of Biosciences, Norwegian University of Life Sciences, 1430 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simen%2BR.%2BSandve%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sandve%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimen%2BR.%2BSandve%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4989-5311" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simen R. Sandve</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">L. Asbjørn Vøllestad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biosciences, University of Oslo, 0316 Oslo, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BAsbj%C3%B8rn%2BV%C3%B8llestad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=V%C3%B8llestad%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AL.%2BAsbj%25C3%25B8rn%2BV%25C3%25B8llestad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9389-7982" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for L. Asbjørn Vøllestad</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Spiros Papakostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Turku, 20014 Turku, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Spiros%2BPapakostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Papakostas%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASpiros%2BPapakostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5563-0048" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Spiros Papakostas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sigbjorn.lien@nmbu.no" class="" data-icon-position="" data-hide-link-title="0">sigbjorn.lien@nmbu.no</a>
<a href="mailto:spiros.papakostas@utu.fi" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@utu.fi</a>
<a href="mailto:spiros.papakostas@gmail.com" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@gmail.com</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sigbjørn Lien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centre for Integrative Genetics (CIGENE), Department of Animal and Aquacultural Sciences, Faculty of Biosciences, Norwegian University of Life Sciences, 1430 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sigbj%C3%B8rn%2BLien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lien%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASigbj%25C3%25B8rn%2BLien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sigbjorn.lien@nmbu.no" class="" data-icon-position="" data-hide-link-title="0">sigbjorn.lien@nmbu.no</a>
<a href="mailto:spiros.papakostas@utu.fi" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@utu.fi</a>
<a href="mailto:spiros.papakostas@gmail.com" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1283" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493108" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1283.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493108" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1283.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493108" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1283.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493108" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Salmonids represent an intriguing taxonomical group for investigating genome evolution in vertebrates due to their relatively recent last common whole genome duplication event, which occurred between 80 and 100 million years ago. Here, we report on the chromosome-level genome assembly of European grayling (<em>Thymallus thymallus</em>), which represents one of the earliest diverged salmonid subfamilies. To achieve this, we first generated relatively long genomic scaffolds by using a previously published draft genome assembly along with long-read sequencing data and a linkage map. We then merged those scaffolds by applying synteny evidence from the Atlantic salmon (<em>Salmo salar</em>) genome. Comparisons of the European grayling genome assembly to the genomes of Atlantic salmon and Northern pike (<em>Esox lucius</em>), the latter used as a nonduplicated outgroup, detailed aspects of the characteristic chromosome evolution process that has taken place in European grayling. While Atlantic salmon and other salmonid genomes are portrayed by the typical occurrence of numerous chromosomal fusions, European grayling chromosomes were confirmed to be fusion-free and were characterized by a relatively large proportion of paracentric and pericentric inversions. We further reported on transposable elements specific to either the European grayling or Atlantic salmon genome, on the male-specific <em>sdY</em> gene in the European grayling chromosome 11A, and on regions under residual tetrasomy in the homeologous European grayling chromosome pairs 9A-9B and 25A-25B. The same chromosome pairs have been observed under residual tetrasomy in Atlantic salmon and in other salmonids, suggesting that this feature has been conserved since the subfamily split.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromosome-evolution" class="hw-term hw-article-keyword hw-article-keyword-chromosome-evolution" rel="nofollow">chromosome evolution</a></li><li class="kwd"><a href="/keyword/chromosomal-structure" class="hw-term hw-article-keyword hw-article-keyword-chromosomal-structure" rel="nofollow">chromosomal structure</a></li><li class="kwd"><a href="/keyword/genomic-rearrangements" class="hw-term hw-article-keyword hw-article-keyword-genomic-rearrangements" rel="nofollow">genomic rearrangements</a></li><li class="kwd"><a href="/keyword/karyotype-evolution" class="hw-term hw-article-keyword hw-article-keyword-karyotype-evolution" rel="nofollow">karyotype evolution</a></li><li class="kwd"><a href="/keyword/retrotransposons" class="hw-term hw-article-keyword hw-article-keyword-retrotransposons" rel="nofollow">retrotransposons</a></li></ul><p id="p-3">Whole genome duplication is known to be an important driver of evolutionary novelty and speciation (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Blomme <em>et al.</em> 2006</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Van de Peer <em>et al.</em> 2009</a>). Whole genome duplication is also regarded as a trigger of certain dramatic consequences in genome evolution (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). A tetraploid genome, for instance, is expected to be very unstable due to a variety of reasons including multivalent pairing during meiosis, unequal separation of sister chromosomes during mitosis, and gene dosage imbalances (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Comai 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Edger and Pires 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hufton and Panopoulou 2009</a>). Chromosomal rearrangements, which are often associated with increased transposable element activity, are expected to be frequent during this period of genomic instability to restore a disomic inheritance of chromosomes, which is also called a rediploidization process (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ohno 1970</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Semon and Wolfe 2007</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hufton and Panopoulou 2009</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). Also possibly driven by transposable element activity, chromosomal fusions, fissions, inversions and indels can suppress, for example, multivalent pairing, and they are also expected to lead to sequence divergence and genome evolution causing genomic incompatibilities among populations thus raising species barriers (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rieseberg 2001</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hoffmann and Rieseberg 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Makhrov 2017</a>). To this end, sequencing and comparing the genomes of taxa with a recent common genome duplication event that have evolved radically different karyotypes holds the promise to illuminate questions regarding the evolutionary consequences of various types of chromosomal rearrangements (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Charlesworth 2016</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wellenreuther and Bernatchez 2018</a>).</p><p id="p-4">The Salmonidae family, also termed salmonid fish, represents an intriguing model system to study genome evolution following whole genome duplication. Salmonid fish have in common a whole genome duplication event that has occurred relatively recently, approximately 80-100 million years ago (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Berthelot <em>et al.</em> 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Macqueen and Johnston 2014</a>). It is considered that the hypothetical ancestor of salmonids had a typical diploid teleost genome with approximately 50 possibly acrocentric chromosomes, and thus the duplication event resulted in approximately 100 possibly acrocentric chromosomes with tetrasomic inheritance (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>). Recent evidence suggests that most of the diploid inheritance has been restored prior to lineage diversification, albeit some regions may still be under residual tetrasomy and thus recombining (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Berthelot <em>et al.</em> 2014</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Robertson <em>et al.</em> 2017</a>).</p><p id="p-5">Present-day salmonids have evolved drastically different karyotypes, which suggests the occurrence of very different genome evolution processes. Lineage diversification has resulted in three salmonid subfamilies: Thymallinae, which includes the European grayling (<em>Thymallus thymallus</em>); Coregoninae, which includes round whitefish (<em>Prosopium</em> spp.), whitefish and cisco (<em>Coregonus</em> spp. and <em>Stenodus</em> spp.); and Salmoninae, which is the subfamily of the well-studied Atlantic salmon (<em>Salmo salar</em>) as well as Pacific salmon and trout (<em>Oncorhynchus</em> spp.) (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Macqueen and Johnston 2014</a>). These subfamilies are known to radically differ in the number of chromosomes and chromosomal arms (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>). The European grayling represents an extreme case as it has an exceptionally high number of chromosomes compared to other salmonids, between 2n = 98 and 2n = 102 depending on the subspecies. The number of European grayling chromosomes has thus remained approximately the same as the number of chromosomes from the ancestral salmonid genome straight after the salmonid-specific whole genome duplication (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>). European graylings also have an exceptionally high number of chromosomal arms, up to 170, which is considered to represent a marked increase over the assumed 100 arms of the hypothetical ancestral duplicated genome of salmonids (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ocalewicz <em>et al.</em> 2013</a>). This is assumed to be a consequence of pericentric inversions, that is, inversions containing the centromere of the ancestral acrocentric chromosomes (<a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Ocalewicz <em>et al.</em> 2013</a>). The rest of the salmonid species have at least a third fewer chromosomes, with Atlantic salmon at the lower end of the distribution with a karyotype of n = 27 and n = 29 chromosomes in the North American and European clade, respectively, and the number of chromosomal arms as low as 72 (<a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>). Many Atlantic salmon chromosomes are also large and metacentric (from ssa01 to ssa07) or large and acrocentric (from ssa09 to ssa20) and are thought to have resulted from Robertsonian fusions of ancestral chromosomes, that is, a fusion of two acrocentric chromosomes at their centromeres (<a id="xref-ref-50-8" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). As such, the Atlantic salmon and European grayling genomes represent clearly distinct genome evolutionary processes that have occurred within salmonids, which demands further investigation.</p><p id="p-6">In this study, we report the first chromosomal-level genome assembly for the European grayling and its in-depth comparison with the Atlantic salmon genome. The assembly builds on the recently published scaffold-level assembly of European grayling that was assembled purely from short-read sequences (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Varadharajan <em>et al.</em> 2018</a>). Scaffold-level genome assemblies can provide excellent source materials for chromosomal-level assembly by employing additional data sources such as long reads, linkage mapping, and synteny with closely related species. Annotating and studying this new European grayling genome assembly further revealed novel insights into the genome evolution differences between the European grayling and Atlantic salmon.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Assembling the European grayling genome at the chromosome level</h3><div id="sec-3" class="subsection"><h4>Assembly of genomic scaffolds using long-read sequence data:</h4><p id="p-7">Using the PacBio RS2 platform, we sequenced the same DNA sample used in the recently published European grayling genome assembly (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Varadharajan <em>et al.</em> 2018</a>) at approximately 19x depth. The sample belonged to a single male adult fish caught from the River Glomma at Evenstad, Norway (61.42 N 11.09 E) that was killed in October 2012. The sequencing effort resulted in a total of 40 gigabase pairs of sequence information. PacBio reads with length &gt;5 kilobase pairs were then processed to consensus sequences using the Canu assembler (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Koren <em>et al.</em> 2017</a>). The resulting reads with length &gt;10 kilobase pairs, amounting to approximately 5x depth, were used in a hybrid assembly. The PacBio reads and the previously published Illumina-based assembly (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Varadharajan <em>et al.</em> 2018</a>) were merged together using the PBJelly2 suite (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">English <em>et al.</em> 2012</a>) using the noSplitSubreads, minMatch 8, minPctIdentity 70, bestn 1 and maxScore 11 parameters. Basic statistics, such as N<sub>50</sub>, L<sub>50</sub> and the length range of the assembled sequences were calculated for each assembly using an in-house developed script (contig_statistics.pl; available in GitHub). After initial mapping of the assembled scaffolds to the Atlantic salmon genome assembly (described in more detail in the linkage mapping section) and manual curation, some of the assembled scaffolds were split in cases of potential sequencing or assembly errors.</p></div><div id="sec-4" class="subsection"><h4>Linkage mapping:</h4><p id="p-8">Male- and female-based linkage maps were built using markers from a single European grayling family originating from the Rhine River (Obenheim, France) that included both parents, 69 female offspring and 44 male offspring that were sequenced using a restriction site associated DNA (RAD) methodology according to previously described protocol (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Amores <em>et al.</em> 2011</a>). The RAD fragments were produced by using the <em>Sbf</em>I restriction enzyme and were sequenced using 100 base pair single-end sequencing using the Illumina HiSeq 2500 platform. Quality trimming of the sequence reads was performed with ConDeTri v. 2.3 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Smeds and Kunstner 2011</a>). The RAD data consisted of a total of 4,167,787 and 7,056,371 reads for the male and female parents, respectively, and an average of 4,041,607 reads for each offspring. Scaffolds containing at least one marker covered 54% of the total length of the hybrid assembly.</p><p id="p-9">To identify the RAD markers, we sorted the trimmed reads to separate files according to barcode, removed the barcode sequence, and verified the restriction site sequence using an in-house Perl script named barcodesplitter5.3.pl (available in GitHub). The reads were then mapped to the hybrid assembly using the Bowtie2 tool (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Langmead and Salzberg 2012</a>). Polymorphic sites were filtered using the following criteria: (a) polymorphisms in parental fish were considered valid only if they were found present in fragments between 182 and 186 base pairs long (fragment extending to both sides from a restriction site) and had read coverage between 9 and 300 per base in both parents; (b) polymorphic sites were retained for linkage mapping when at least one of the parents was heterozygous, the polymorphism was biallelic in the offspring, and the offspring genotype distribution followed a Mendelian segregation pattern as tested by chi-square tests at 5% significance level following correction by false discovery rate according to the Benjamini and Hochberg approach (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benjamini and Hochberg 1995</a>). Additionally, offspring were retained in the analysis if they had at least 1,000 markers genotyped with &gt;8 read coverage, a criterion which resulted in removal of four offspring from the analysis. Polymorphic site filtering was completed using R (script “RADstats_final.R”; available in GitHub). The filtered markers were mapped to linkage groups and ordered using the Lep-MAP2 software (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rastas <em>et al.</em> 2016</a>). Linkage between markers was accepted at LOD ≥ 9, upon which additional individual markers were added at LOD ≥ 7.</p><p id="p-10">The linkage groups were initially constructed based on recombination frequencies and thereafter improved by testing the alternative ordering of markers using comparative mapping information from Atlantic salmon, a procedure hereafter referred to as salmonization. The latter was performed by first mapping the scaffolds from the European grayling hybrid assembly to the Atlantic salmon genome assembly (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>), downloaded from the NCBI Genome database (RefSeq assembly GCF_000233375.1), using the nucmer tool in MUMmer 3.0 aligner (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kurtz <em>et al.</em> 2004</a>). Prior to the alignment, Atlantic salmon chromosome sequences were repeat-masked using a salmon repeat database (ssal_repeats_v2.0) and RepeatMasker v4.0.3 (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Smit <em>et al.</em> 2013–2015</a>). The best matching position for each European grayling scaffold in the Atlantic salmon genome was determined by adding up the number of base pairs in each hit and the number of hits within a scaffold. Second, for each linkage group, markers mapping to the most frequently associated Atlantic salmon chromosome were included in the further salmonization procedure. At each step, the correct map was assumed to be the one with the shortest female map length calculated using Lep-MAP2 (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Rastas <em>et al.</em> 2016</a>). During the first step, the markers in each linkage group were initially reordered according to their locations in the Atlantic salmon assembly, and the resulting map lengths were calculated. In the second step, we investigated the salmonized European grayling linkage maps where breaks in the progression of map length increase indicated possible genomic rearrangements between the European grayling and Atlantic salmon genomes (Fig. S1). We then considered a portion of the largest breaks by applying either of the two following criteria: (a) the absolute map length difference of the break is &gt; 10 map units for any of the markers or (b) the break length is at least eight times the standard deviation of that of all pairwise differences in adjacent markers in that linkage group. This step led to the identification of one to eight blocks of orderly progressed markers per linkage group. In the third step, we investigated the possibility of translocations from these blocks explaining our observations. To do this, we reconstructed each linkage group by permuting the order of the corresponding blocks and selecting the solution with the minimal length as the most parsimonious block order. To test the possibility of inversions in the most parsimonious block order, we then inverted each block and tested whether it further reduced the map length. At the end of this step we validated the final combination of rearrangements by testing if implementing all accepted changes indeed resulted in the minimal map length. The salmonization script “salmonize_final.R” is available in GitHub. As the final step, all maps were further manually curated with special attention given to regions that are known to have &gt;90% sequence similarity in Atlantic salmon, namely, the pairs of Atlantic salmon chromosome arms including 2p-5q, 2q-12qa, 3q-6p, 4p- 8q, 7q-17qb, 11qa-26 and 16qb-17qa (<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). European grayling linkage groups corresponding to salmon chromosome arm pairs 3q-6p, 7q-17qb and 11qa-26 mapped equally well to both of their Atlantic salmon homeolog counterparts. To identify the true homologs in these linkage groups, markers were aligned separately and ordered based on each of the Atlantic salmon homeologs using nucmer and LepMap2, and the best ordering homeolog was chosen as the linkage group identity. Linkage groups corresponding to Atlantic salmon chromosome arms 2q-12qa and 4p-8q had fused linkage maps that could not be separated.</p></div><div id="sec-5" class="subsection"><h4>Synteny-assisted genome scaffolding:</h4><p id="p-11">The European grayling scaffolds that contained markers in the final linkage groups and the scaffolds that had a MUMmer-alignment-based position in the Atlantic salmon genome were arranged into the final European grayling chromosomal order based on synteny with Atlantic salmon chromosomes unless there was strong evidence of a rearrangement based on the European grayling linkage map position. The alignment with Atlantic salmon was also used to orientate the scaffolds. Scaffolds were then concatenated into chromosome-level sequence assemblies by adding 100 base pair gaps between each adjacent scaffold.</p></div></div><div id="sec-6" class="subsection"><h3>Repeat library construction and genome annotation</h3><p id="p-12">A comprehensive repeat library was built by combining <em>de novo</em> identified European grayling-specific repeats as well as repeat elements identified in the Atlantic salmon genome (available at: <a href="http://web.uvic.ca/grasp/salmon_v1.6">http://web.uvic.ca/grasp/salmon_v1.6</a>). We initially ran the RepeatModeler software v. 1.0.11 (available at: <a href="http://www.repeatmasker.org/RepeatModeler">http://www.repeatmasker.org/RepeatModeler</a>; last accessed June 8, 2018) with default parameters. To compile a set of LTRs, we used the LTRharvest (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ellinghaus <em>et al.</em> 2008</a>) and LTRdigest (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Steinbiss <em>et al.</em> 2009</a>) software as described in (<a href="http://weatherby.genetics.utah.edu/MAKER/wiki/index.php/Repeat_Library_Construction-Advanced">http://weatherby.genetics.utah.edu/MAKER/wiki/index.php/Repeat_Library_Construction-Advanced</a>; last accessed: June 8, 2018) and combined the results with the sequences identified by MGEscan-LTR (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lee <em>et al.</em> 2016</a>). All the identified sequences were combined and filtered to remove redundancy. The resulting <em>de novo</em> set of sequences was combine-queried against the Universal Protein Resource database (UniProt proteins release 2017_08, Consortium 2017) to filter out any known proteins sequences. The remaining unclassified set of sequences was then annotated using RepeatClassifier, the Dfam database and TEclass (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abrusán <em>et al.</em> 2009</a>).</p><p id="p-13">An updated reference-based set of transcripts was constructed by first aligning the RNAseq reads to the improved assembly using STAR v. 2.6 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dobin <em>et al.</em> 2013</a>) followed by Cufflinks (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Trapnell <em>et al.</em> 2010</a>) for the prediction of transcript sequences. This along with the <em>de novo</em> assembled transcriptome described in <a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Varadharajan <em>et al.</em> (2018)</a> was used as an input to the PASA pipeline (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Haas <em>et al.</em> 2003</a>) to build a comprehensive transcript database.</p><p id="p-14">Further, predictions from <em>ab initio</em> gene finders like SNAP (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Korf 2004</a>) and GeneMark-ES (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lomsadze <em>et al.</em> 2005</a>) were also used as input into MAKER v. 2.31.9 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cantarel <em>et al.</em> 2008</a>). MAKER pipeline was run for two iterations with transcript evidence from PASA transcriptome assembly and protein coding sequences from the Atlantic salmon, GTF outputs from AUGUSTUS and GeneMark-ET resulting from BRAKER (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hoff <em>et al.</em> 2016</a>), the UniProt database (UniProt proteins release 2017_08, Consortium 2017) as the protein evidence and the above described repeat library. MAKER was run with default options.</p><p id="p-15">Functional annotation was added to the MAKER-predicted gene models using BLAST against UniProt database and domain information was added using InterProScan (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zdobnov and Apweiler 2001</a>). MAKER-predicted gene models were then filtered based on Annotation Edit Distance (AED) and the presence of known PFAM domains to retain high confidence set of genes.</p></div><div id="sec-7" class="subsection"><h3>Genome repetitiveness and repeat element assessment</h3><p id="p-16">Kmer repetitiveness of the previously published and current genome assemblies of European grayling, Atlantic salmon and rainbow trout (<em>Oncorhynchus mykiss</em>) were calculated using Jellyfish software v. 1.1.11 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Marçais and Kingsford 2011</a>) using kmer size of 31. Repetitiveness was calculated by dividing counts of non-unique kmers by total kmers in the assembly. To investigate the European grayling genome in terms of repeat elements, the European grayling repeat library, containing 1,743 <em>de novo</em> repeats, was employed along with the repeats from RepBase v. 20.05 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jurka <em>et al.</em> 2005</a>). Transposable element sequences were curated by first detecting the host genes that were potentially of non-transposable element origin and then classifying the remaining transposable element sequences according to the classification system of <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wicker <em>et al.</em> (2007)</a>. Transposable element abundances were estimated for both European grayling and Atlantic salmon. To remove from the final repeat analysis any repeats that potentially originated from host genes instead of transposable elements, the transposable element sequences were compared to two different repeat databases. These databases were the REPET-formatted RepBase v. 20.05 (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Jurka <em>et al.</em> 2005</a>) and the Swiss-Prot database available in UniProt (as of June 1, 2018). Comparisons were conducted by using the blastn (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>) and blastx (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gish and States 1993</a>) algorithms with parameters set to <em>word_size = 7</em>, and to <em>e-value &gt; 1×10<sup>−10</sup></em>. A custom script named best_multi_blast_score_parser.pl (available in GitHub) was used to select the highest scoring hits for each potential transposable element sequence. Based on the best-scoring hits, each transposable element sequence was categorized as non-transposable element derived host gene and removed if it had a best-scoring hit to a Swiss-Prot sequence. The rest of the library hits were kept for further analysis.</p><p id="p-17">To classify the transposable element sequences, they were compared to the RepBase repeats using both nucleotide sequence and protein similarity. To categorize transposable element sequences to class, order, and superfamily levels, the relevant information from RepBase was used in case a sequence had an acceptable alignment hit with this database. An alignment was accepted if it suggested high similarity between query and reference repeat, defined by <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Wicker <em>et al.</em> (2007)</a>. A high similarity alignment was at least 80 base pair long with at least 80% sequence similarity between query and reference repeat sequence, occupying at least 80% of the query repeat length (which we calculated after removing unknown nucleotides from the query sequence length). These thresholds concerned the blastn search. In case of a non-acceptable nucleotide alignment for a transposable element sequence, then this sequence was searched against the RepBase database using the blastx approach, with an alignment considered valid if the hit had <em>e-value &lt; 1×10<sup>−10</sup></em> (following <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). The repeat element abundance in the European grayling and Atlantic salmon genomes was assessed for each chromosome separately using the RepeatMasker v. 4.0.7 tool (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chen 2004</a>) by using the parameter -qq. The RepeatMasker-based locations of transposable element sequences in each of the two genomes were annotated with a script named “classifyGoodTEHits.R” (available in GitHub). The elements with marked difference in their abundance between the two genomes were sought out by using a linear model <em>log2(salmon abundancy+1) ∼ log2(grayling element abundancy+1)</em> using R (v. 3.4.0, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">R Core Team 2017</a>) and elements that had residuals larger than 1.96 standard deviations from zero were considered outliers, that is, outside the 95% confidence interval limits.</p><div id="sec-8" class="subsection"><h4>Predicting centromere locations using the location of repeats:</h4><p id="p-18">Repetitive element content can reveal information about the chromosomal landscapes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kaminker <em>et al.</em> 2002</a>; <a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). To estimate the repeat content, copies of the generated repeat library were sought from the European grayling chromosomes using RepeatMasker. The abundances of different element classes were quantified using local regression for element abundancy over each chromosome with the R function lowess with parameter f = 0.2, and the maximum position for each element class in each of the European grayling chromosomes was extracted. These maximal density locations in each chromosome were analyzed using principal component analysis. Although centromeres are generally epigenetic structures that cannot be observed from the nucleotide order, some transposable elements have a tendency to accumulate in certain region of the genome (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Daron <em>et al.</em> 2014</a>). This has been previously observed in the Atlantic salmon genome (<a id="xref-ref-39-9" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>) where Tc1-Mariner type elements were shown to accumulate in the centromeric regions. The chromosomal positions with the maximal abundancy of the two element classes, the centromere-related Tc1-mariner, and the most contrasting element class RTE-X were more closely inspected using the occurrences of each of the two element classes in 100 kilobase pair bins and local regression. Hypothetical centromere loci for each chromosome were predicted using the maximal estimates of the Tc1-Mariner-richness from the local regression curves. To predict the karyotype, the long:short arm ratio was estimated for each chromosome (following <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Levan, Fredga and Sandberg 1964</a>). This was performed using the peak position of the Tc1-mariner element to calculate the length of the chromosomal fragments on both sides of the peak and dividing the longer length by the shorter one. Finally, the effect of chromosome size and karyotype on recombination frequencies was estimated using the linear model <em>map length ∼ chromosome size + long:short arm ratio</em>. To validate the effect of the long:short arm ratio in the full model, the chi-square test was performed to compare the full model to a reduced model with chromosome size as the only independent variable.</p></div></div><div id="sec-9" class="subsection"><h3>Identification of the European grayling sex chromosome</h3><p id="p-19">The gene named <em>sdY</em> for sexually dimorphic on the Y-chromosome was searched using a tblastn (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>) homology search against the European grayling chromosome assembly. The rainbow trout <em>sdY</em> protein sequence (GenBank: BAM24747.1) was used as bait in this search. The RAD sequences were then remapped to the chromosome-level assembly and sex-biased loci were detected from the chromosomes.</p></div><div id="sec-10" class="subsection"><h3>Comparison to the Northern pike genome</h3><p id="p-20">We compared the obtained European grayling chromosomes to the chromosomes of Northern pike (<em>Esox lucius</em>), a species that represents the closest sister group to Salmonids prior to the salmonid-specific whole genome duplication with an available genome assembly. The genome assembly was downloaded from the NCBI Genome database (RefSeq assembly GCA_000721915.1). Conserved synteny between European grayling and Northern pike was determined by aligning European grayling and Northern pike chromosome sequences using the nucmer tool in MUMmer 3.0 aligner (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Kurtz <em>et al.</em> 2004</a>) and keeping hits with identity ≥80.0 and matchcount ≥100. Homeologous European grayling chromosome pairs were named according to the Northern pike chromosome naming convention (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rondeau <em>et al.</em> 2014</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sutherland <em>et al.</em> 2016</a>).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-21">The PacBio reads, chromosome-level sequences and unmapped scaffolds over than 2000 base pairs have been deposited at NCBI SRA and Genomes under BioProject ID PRJNA464295. Scripts have been deposited to GitHub under the link <a href="https://github.com/tiinasa/graylinggenome">https://github.com/tiinasa/graylinggenome</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7728512">https://doi.org/10.25387/g3.7728512</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Chromosome-level European grayling genome assembly</h3><div id="sec-14" class="subsection"><h4>Assembly of genomic scaffolds using long-read data:</h4><p id="p-22">By adding the PacBio long-read sequences to the published draft assembly of <a id="xref-ref-65-5" class="xref-bibr" href="#ref-65">Varadharajan <em>et al.</em> (2018)</a> and splitting 23 contigs that were determined as chimeric by initial comparison to the Atlantic salmon genome assembly, we obtained a 38% increase in N<sub>50</sub> with a 24% decrease in L<sub>50,</sub> a 62% increase in the maximum scaffold length, and a 16% increase in the total assembly length (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">TABLE 1</a>). Altogether, the total number of scaffolds decreased by 25%, while the overall GC content remained almost identical at approximately 43% (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">TABLE 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492918/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492918/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Assembly properties of different stages of the European grayling genome assembly process. Stage 1 represents the assembly built using only short-read DNA sequencing data (from <a id="xref-ref-65-6" class="xref-bibr" href="#ref-65">Varadharajan <em>et al.</em> 2018</a>). Stage 2 involves the outcome of the hybrid assembly process, which combined short- and long-read DNA sequencing data. Finally, stage 3 corresponds to the complete genome assembly that was produced using the linkage mapping data and synteny information with the Atlantic salmon genome. Numbers in brackets represent the percent increase/decrease over the previous stage for given statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>Linkage mapping:</h4><p id="p-23">RAD sequencing resulted in 7,684 informative SNP markers with a median female: male ratio of map distances at 1.75:1 (sd= 3.05). Postfiltering, 6,076 markers were assigned to the final linkage groups (TABLE S1). The final female-based total map length was 3,044 centi-Morgan (cM) (Fig. S2 and TABLE S2). The average female map length per million chromosomal base pairs was 2.0 map units (TABLE S1).</p></div><div id="sec-16" class="subsection"><h4>Synteny-assisted genome chromosome building:</h4><p id="p-24">The synteny-assisted chromosome building step represented the biggest improvement in the genome assembly process (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">TABLE 1</a>). Together with the linkage mapping information, we managed to reconstruct all 51 European grayling chromosomes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Nevertheless, of the total of 18,265 scaffolds from the hybrid stage of the assembly, a large number of relatively small-sized scaffolds were left unassigned (N = 8,938 scaffolds ranging in length from 984 to 1,162,211 base pairs of which N = 3,780 scaffolds with &gt; 2,000 base pairs length are available at NCBI), which corresponded to a total length of 91,704,787 unassigned base pairs (or 5.8% of the total genome assembly length).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The European grayling genome. The tracks indicate the following information: (A) Chromosome number according to Northern pike, used as a non-WGD outgroup, and predicted chromosomal type (M = metacentric; SM = submetacentric; and T = acrocentric); (B) Predicted centromere (in purple) and telomere (in yellow) locations; (C) Proportions of the Tc1-mariner (in purple) and RTE-X elements (in yellow), which are used to predict centromere and telomere positions, respectively; (D) Gene density; (E) Female linkage map units; and (F) Female- (in red) and male-biased (in blue) sex-biased loci with the sdY gene region indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1472579586" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The European grayling genome. The tracks indicate the following information: (A) Chromosome number according to Northern pike, used as a non-WGD outgroup, and predicted chromosomal type (M = metacentric; SM = submetacentric; and T = acrocentric); (B) Predicted centromere (in purple) and telomere (in yellow) locations; (C) Proportions of the Tc1-mariner (in purple) and RTE-X elements (in yellow), which are used to predict centromere and telomere positions, respectively; (D) Gene density; (E) Female linkage map units; and (F) Female- (in red) and male-biased (in blue) sex-biased loci with the &lt;em&gt;sdY&lt;/em&gt; gene region indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1283/F1.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1283/F1.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493185" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">The European grayling genome. The tracks indicate the following information: (A) Chromosome number according to Northern pike, used as a non-WGD outgroup, and predicted chromosomal type (M = metacentric; SM = submetacentric; and T = acrocentric); (B) Predicted centromere (in purple) and telomere (in yellow) locations; (C) Proportions of the Tc1-mariner (in purple) and RTE-X elements (in yellow), which are used to predict centromere and telomere positions, respectively; (D) Gene density; (E) Female linkage map units; and (F) Female- (in red) and male-biased (in blue) sex-biased loci with the <em>sdY</em> gene region indicated.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-17" class="subsection"><h3>Genome repetitiveness and repeat element assessment</h3><p id="p-26">The repetitiveness of identical sequences in the chromosome-level European grayling assembly was estimated at 11.9%. By comparison, we estimated repetitiveness at 29.3% in the Atlantic salmon genome and 9.8% in the reported genome assembly of the rainbow trout using the same software. The RepBase and Swiss-Prot searches resulted in a best-scoring hits list including 1,090 transposable element sequences, of which 32 potential host genes were removed from further transposable element analysis. Of the remaining transposable element sequences, 287 had nucleotide-based matches and 434 had protein-based matches to RepBase after filtering. After the matches were combined, they contained 586 unique transposable element sequences. These classified transposable element sequences covered 83% of the European grayling repeat sequences. Altogether, 47.4% of the European grayling genome assembly and 52.3% of the Atlantic salmon genome assembly were covered by these repeats, which could be distinguished into 24 superfamilies of transposable elements (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">TABLE 2</a>). Of the repeat elements identified, 14 were only found in the Atlantic salmon genome assembly, while only three were more abundant in the European grayling genome assembly (TABLE S3 and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492914/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492914/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Transposable element classification and abundances in the European grayling and Atlantic salmon genomes</span><div class="sb-div caption-clear"></div></div></div><div id="sec-18" class="subsection"><h4>Predicting centromere locations using the location of the repeats:</h4><p id="p-27">We considered the Tc1-Mariner abundance to peak around the centromeric regions (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). A LINE-class RTE-X retrotransposon was also found located the furthest from Tc1-Mariner excluding some unknown and simple repeat categories, suggesting a potential subtelomeric enrichment (Fig. S3). Using the Tc1-Mariner abundances we predicted 29 telocentric and 22 sub(metacentric) karyotypes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). After correcting for the chromosome size in megabases, the long:short arm ratio had a negative correlation with the female map length (estimate -1.83, <em>P</em> = 0.0335, F(2,45)=26.94, adjusted R-squared = 0.5246 for the whole model). The chi-square test confirmed that the long:short arm ratio was indeed a significant variable (<em>P</em> = 0.0283) when predicting the map length of a chromosome. This result suggests that the metacentric chromosomes have a relatively higher recombination rate than comparably sized telocentric chromosomes.</p></div><div id="sec-19" class="subsection"><h4>Genome rearrangements between European grayling and Atlantic salmon:</h4><p id="p-28">Rearrangements between European grayling and Atlantic salmon chromosomes suggested conservation of the synteny within blocks of chromosome arms in both of the species, but with frequent chromosomal inversions observed between the two genomes. In particular, we detected 119 blocks from which we could confidently interpret 18 as pericentric and 24 as paracentric inversions in European grayling (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, TABLE S4, and Fig. S4). Compared to Atlantic salmon, in which many chromosomal fusions have occurred after the tetraploid salmonid ancestor, the ancestral chromosome identities were conserved in European grayling with the exception of one chromosomal fission that was noticed (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosomal rearrangements between the European grayling and Atlantic salmon genomes. Each box represents an Atlantic salmon chromosome depicted on the right side of the box. The corresponding European grayling chromosomes are shown in different colors on the left side. Horizontal lines represent corresponding positions between the two genomes, with sequence identity depicted in a bluered scale on the right side of the box. Purple and yellow circles depict predicted centromere and telomere positions in the European grayling chromosomes, respectively. The axes scales represent the log2-transformed abundance in base pairs +1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1472579586" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosomal rearrangements between the European grayling and Atlantic salmon genomes. Each box represents an Atlantic salmon chromosome depicted on the right side of the box. The corresponding European grayling chromosomes are shown in different colors on the left side. Horizontal lines represent corresponding positions between the two genomes, with sequence identity depicted in a bluered scale on the right side of the box. Purple and yellow circles depict predicted centromere and telomere positions in the European grayling chromosomes, respectively. The axes scales represent the log&lt;sub&gt;2&lt;/sub&gt;-transformed abundance in base pairs +1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1283/F2.medium.gif" width="220" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1283/F2.medium.gif" width="220" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492907" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Chromosomal rearrangements between the European grayling and Atlantic salmon genomes. Each box represents an Atlantic salmon chromosome depicted on the right side of the box. The corresponding European grayling chromosomes are shown in different colors on the left side. Horizontal lines represent corresponding positions between the two genomes, with sequence identity depicted in a bluered scale on the right side of the box. Purple and yellow circles depict predicted centromere and telomere positions in the European grayling chromosomes, respectively. The axes scales represent the log<sub>2</sub>-transformed abundance in base pairs +1.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-20" class="subsection"><h3>Identification of the European grayling sex chromosome</h3><p id="p-30">The <em>sdY</em> locus was mapped to 11A [2,137,039-2,136,679]; (<em>e-value = 1.74E-54</em>, score = 190). After remapping the RAD reads, we found the sex-biased loci to be clearly enriched at European grayling 11A close to the <em>sdY</em> locus and in the 11B homeolog, with four sex-biased loci found on 11A and four on 11B. Additionally, three sex-biased loci were more randomly distributed in the genome, more specifically on chromosomes 3A and 18A and in the scaffold Contig7739 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-21" class="subsection"><h3>Comparisons with the genome of Northern pike</h3><p id="p-31">The European grayling chromosomes could be matched to the Northern pike counterparts in a 2:1 ratio except for one ancestral-duplicated chromosome that had been subsequently split in two in European grayling (13A and 13C) (TABLE S1). European grayling chromosomes were named according to the corresponding pike orthologs (TABLE S1).</p></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-32">By assembling the European grayling genome up to the level of chromosomes and comparing it with that of Atlantic salmon, we provided some novel insights about the very distinct genome evolution processes that have been ongoing in European grayling (<a id="xref-ref-50-9" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>) and, in particular, we reported on the complete absence of chromosomal fusions and the somewhat frequent occurrence of chromosomal inversions (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). The absence of a fusion event in European grayling could be parsimoniously hypothesized by observing that the European grayling homeologue chromosomes always mapped to single Northern pike chromosomes (TABLE S1). This finding suggests the absence of chromosomal fusions in either of these two genomes since the last whole genome duplication event. Notably, a single case of chromosomal fission was observed behind the generation of European grayling chromosomes 13A and 13C (TABLE S1). The absence of fusions agrees with previous hypotheses based on karyotype information (<a id="xref-ref-50-10" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>) and is confirmed for the first time at the sequence level. The relatively frequent occurrence of chromosomal inversions in the European grayling lineage was also confirmed by identifying at least 51 inversions between the European grayling and Atlantic salmon genomes, which covered as much as 45% of the total length of the European grayling genome assembly (TABLE S4). Detailing the origin of these inversions, whether specific to European grayling or to Atlantic salmon, proved to be a challenging task. Comparisons of the order of available markers between the genomes of these two species and that of Northern pike are often problematic due to the loss of synteny within chromosome arms and smaller number of European grayling scaffolds reliably positioned in the Northern pike genome (Fig. S4). Nevertheless, for nine of the inversions, it was clear that six were specific to the European grayling genome and three were specific to the Atlantic salmon genome (TABLE S4). Thus, chromosomal inversions appear to have played a role in the genome evolution process in both of these species, albeit more frequently identified in the European grayling genome. A relatively large number of the recognized inversions, 18 out of the 42 resolved cases, were also found to be pericentric, that is, inversions that included the centromere (TABLE S4). Taken together, these findings may also explain the relatively large number of chromosomal arms observed in European grayling, as a pericentric inversion of the assumedly acrocentric ancestral chromosome would double the number of chromosomal arms (<a id="xref-ref-50-11" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>).</p><p id="p-33">Transposable elements may have played a key role in genome evolution processes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Auvinet <em>et al.</em> 2018</a>). Additionally, these elements may be important in the rediploidization process by generating sequence divergence that would separate the homeologs. In particular, the comparison of transposable element classes between organisms with very different genomic rearrangements, such as between European grayling and Atlantic salmon, may be of interest. We found that retrotransposons (class I transposable elements) are more abundant compared to DNA transposons (class II transposable elements) with 1.7 times and 1.3 times higher abundance in the European grayling and Atlantic salmon genomes, respectively (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">TABLE 2</a>). This is similar to what has been observed in the genomes of a wide variety of other eukaryotes, such as many plants, insects, amphibians, reptiles, and mammals, which were found to have a relatively higher proportion of retrotransposons than DNA transposons (reviewed in <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Canapa <em>et al.</em> 2015</a>). Nevertheless, this is different from what has been found in many non-salmonid fish, which were found to have DNA transposons as the most abundant class (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Canapa <em>et al.</em> 2015</a>). The differential accumulation of transposable elements between lineages may be playing a significant role in genome evolution processes, but unraveling the complexity of underlying reasons behind such differences could not be investigated in this study.</p><p id="p-34">Additionally, the comparison of abundances of the transposable elements between European grayling and Atlantic salmon resulted in the recognition of 14 Atlantic salmon-specific and three European grayling-specific transposable elements (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and TABLE S3). The Atlantic salmon-specific elements include the DNA transposons DNA4-1, DNA4-2, DNA4-2B, DNA4-2C and DNA4-8, which altogether covered 80 megabase pairs (3.57%) in the Atlantic salmon genome, but were found to be completely absent in European grayling (TABLE S3). The DNA transposons Mariner-16, Mariner-20, and Mariner-28 were also found to be Atlantic salmon-specific (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and TABLE S3). These elements belong to the same Tc1-Mariner superfamily, which represents one of the most abundant categories of transposable elements in salmonids, with a major suspected role in the Atlantic salmon rediploidization process (<a id="xref-ref-39-10" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). Another case of an Atlantic salmon-specific element is the Copia-12 retrotransposon (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> and TABLE S3), which belongs to the Copia superfamily of retrotransposons that was recently suggested to have a role in chromosomal diversification and speciation in other teleost fishes (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Auvinet <em>et al.</em> 2018</a>). Among the European grayling-specific elements, hAT-10 from the hAT DNA transposon superfamily covered 123,702 base pairs (0.01%) of the assembly and was completely absent in the Atlantic salmon genome assembly (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> and TABLE S3). The hAT DNA transposons, such as the Tc1-Mariner ones, are so-called cut-and-paste elements that have transposition mechanisms with the potential to actively induce genomic rearrangements in addition to indirect ways to generate homologous recombination of element copies. The accumulation of a particular transposable element in one of the two species may be considered as an indication of lineage-specific transposable element activity. These findings may provide unique insights to stimulate further research aimed at better understanding the molecular drivers of these distinct genome evolution processes. While highly accurate in repeat identification and suitable for our purpose to simply compare the element abundances between European grayling and Atlantic salmon, conventional software such as RepeatMasker that we applied has been reported to under-estimate the abundances of transposable elements (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">de Koning <em>et al.</em> 2011</a>). Future studies could benefit from using more sensitive approaches such as repetitive sequence clustering (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">de Koning, <em>et al.</em> 2011</a>) as they may enable improved estimation of repeat element abundances. Moreover, they may allow further insight into repeat community structure and key element identification using network approaches (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wacholder <em>et al.</em>, 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Levy <em>et al.</em> 2017</a>) thus enabling more detailed investigations of the repeat element proliferation dynamics among salmonids.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of repeat element abundances in the European grayling and Atlantic salmon genomes. The RepBase names of the elements that were more abundant in either of the two species and outside the 95% confidence limit are also given. Residuals are colored to indicate element class and order categories." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1472579586" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of repeat element abundances in the European grayling and Atlantic salmon genomes. The RepBase names of the elements that were more abundant in either of the two species and outside the 95% confidence limit are also given. Residuals are colored to indicate element class and order categories.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1283/F3.medium.gif" width="440" height="382"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1283/F3.medium.gif" width="440" height="382"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492993" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Comparison of repeat element abundances in the European grayling and Atlantic salmon genomes. The RepBase names of the elements that were more abundant in either of the two species and outside the 95% confidence limit are also given. Residuals are colored to indicate element class and order categories.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Based on current knowledge we can only speculate what may be the role of the distinct European grayling chromosome architecture in the evolution of the species. <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Qumsiyeh (1994)</a> hypothesizes that high diploid chromosome number leads to increased recombination rates, which in the case of the freshwater European graylings, may be associated with low differentiation, an advantageous trait in variable freshwater environments (<a id="xref-ref-50-12" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In contrast, reduction of chromosome numbers in the other salmonid lineages may be associated with anadromous life history strategy (<a id="xref-ref-50-13" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>). It has been suggested that periods of relaxed purifying selection, as in bottlenecked populations, may be necessary for the deleterious effects of chromosomal rearrangements to become fixed (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lynch and Conery 2003</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lynch 2007</a>). While possibly initially stochastic in nature, the resulting effects of chromosome evolution on mutation and recombination rates can result in directed evolution (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lynch 2007</a>) and phenotypic change. Also, following gene duplication, lineage-specific loss of duplicated gene copies (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lynch and Conery 2000</a>) or possibly divergent expression evolution such as observed between European grayling and Atlantic salmon (<a id="xref-ref-65-7" class="xref-bibr" href="#ref-65">Varadharajan <em>et al.</em> 2018</a>), may contribute to speciation. Evidence of distinct genome evolution processes may provide avenues for further research aimed at exploring links between life history differences in salmonids and the evolution of distinct genome architectures. Transposable element activity, with lineage-specific differences such as those observed between European grayling and Atlantic salmon, is a major driver of genome evolution (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kazazian 2004</a>) and may have been also involved in the distinct genome evolution processes observed here. Furthermore, chromosomal inversions, such as those found frequently in the European grayling genome, have been suggested to have profound effects in the adaptation and speciation processes (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Wellenreuther and Bernatchez 2018</a>). For instance, they have been reported to increase genome sequence divergence between marine and freshwater ecotypes of a stickleback species <em>Pungitius pungitius</em> (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nelson and Cresko 2018</a>) as well as between nonmigratory and migratory ecotypes of Atlantic cod (<em>Gadus morhua</em>) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berg <em>et al.</em> 2016</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kirubakaran <em>et al.</em> 2016</a>). Computer simulations supported these observations and showed that chromosomal inversions may accelerate speciation particularly in certain conditions, such as when adaptation involves multiple genes with small individual fitness effects (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Feder <em>et al.</em> 2014</a>). Experimentation in house mouse (<em>Mus musculus domesticus</em>) has additionally demonstrated the possibility of rapid divergence mediated by Robertsonian fusions (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Garagna <em>et al.</em> 2014</a>). We anticipate that further salmonid-centric research in this direction aided by help from the chromosomal-level European grayling assembly that we provide will illuminate several open questions that stem from these observations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Karyotypic changes among salmonid taxa. A Bayesian chronogram tree based on mitochondrial sequence (tree obtained and edited from Shedko et al. (2013), doi: 10.5061/dryad.r42qf) of those salmonids that have diploid chromosome number (2n) and the number of chromosome arms (NF) available in Phillips and R&#xE1;b (2001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1472579586" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Karyotypic changes among salmonid taxa. A Bayesian chronogram tree based on mitochondrial sequence (tree obtained and edited from Shedko &lt;em&gt;et al.&lt;/em&gt; (2013), doi: 10.5061/dryad.r42qf) of those salmonids that have diploid chromosome number (2n) and the number of chromosome arms (NF) available in Phillips and R&#xE1;b (2001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1283/F4.medium.gif" width="311" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1283/F4.medium.gif" width="311" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1283/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493167" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Karyotypic changes among salmonid taxa. A Bayesian chronogram tree based on mitochondrial sequence (tree obtained and edited from <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Shedko <em>et al.</em> (2013)</a>, doi: 10.5061/dryad.r42qf) of those salmonids that have diploid chromosome number (2n) and the number of chromosome arms (NF) available in <a id="xref-ref-50-14" class="xref-bibr" href="#ref-50">Phillips and Ráb (2001)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Cases of residual tetrasomy and elevated sequence similarity between homeologous chromosomes have been reported in many salmonids, suggesting that some rediploidization in these salmonid species may be ongoing (<a id="xref-ref-39-11" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). Although some species-specific differences in the residually tetrasomic regions have been reported, the tetraploid state appears to be conserved among salmonid species in seven to eight homeologous chromosome pairs (as summarized in <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Sutherland <em>et al.</em> 2016</a>). Although otherwise distinctive, the karyotype evolution of European grayling was comparable to that of most salmonids in the case of residually tetrasomic regions (in chromosomes 9A &amp; 9B homologous to ssa02q &amp; ssa12qa, respectively and 25A &amp; 25B homologous to ssa04p &amp; ssa08q, respectively) also being observed in the European grayling genome assembly based on shared linkage maps. Similarly, other regions (in chromosomes 2A &amp; 2B homologous to ssa26 &amp; ssa11a, respectively; 11A &amp; 11B homologous to ssa6a &amp; ssa3b, respectively; 20A homologous to ssa5b; and 23A &amp; 23 B homologous to ssa7b and 17b, respectively) with reoccurring residual tetrasomy reported among salmonids (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Sutherland <em>et al.</em> 2016</a>) had elevated sequence similarity, which has also be used as a predictor for recent or ongoing tetrasomy (<a id="xref-ref-39-12" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>) (TABLE S1). Residual tetrasomy appears to have persisted in both Salmoninae and Thymallinae since the two lineages split, though the pace of the remaining rediploidization has been very slow since the lineage diversification (<a id="xref-ref-39-13" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>). The evolutionary significance of persistent residual tetrasomy remains unknown, but the existence of residual tetrasomy in the ancestral-like European grayling genome suggests that tetrasomy would be independent of chromosomal fusions typical of other salmonids (<a id="xref-ref-50-15" class="xref-bibr" href="#ref-50">Phillips and Ráb 2001</a>; <a id="xref-ref-39-14" class="xref-bibr" href="#ref-39">Lien <em>et al.</em> 2016</a>) and instead be favored by some other factor. The majority of the sex-linked loci detected were found in the homeologous European grayling chromosome pair 11. Additionally, we located the <em>sdY</em> gene, reported as male-specific among many other salmonids (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yano <em>et al.</em> 2013</a>), in chromosome 11A; thus, we concluded that chromosome 11A is the European grayling sex chromosome.</p><p id="p-39">In conclusion, by utilizing the novel resource of a chromosome-level genome assembly for European grayling, we were able to make some intriguing observations about the genome evolution processes in salmonids that confirmed previous hypotheses and generated new questions. We confirmed the absence of chromosomal fusions and the somewhat high abundance of pericentric inversions in European grayling and highlighted the differences compared to the large number of chromosomal fusions that have taken place in the Atlantic salmon. We also highlighted novel and already described instances of transposable elements with a role in driving these different genome evolution processes. We further identified similar homeologous regions under residual tetrasomy in European grayling as in the genomes of many other salmonid species and discussed the potential underlying evolutionary causes behind the distinctive karyotype evolution of Thymallinae among salmonids as well as the role of genomic rearrangements in speciation. We anticipate that as more salmonid genomes are sequenced, many of these questions will be further investigated and advance our understanding of the major molecular mechanisms that have shaped the salmonid genomes since their last common whole genome duplication event.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">This work was supported by the Academy of Finland (project numbers 287342 and 302873). We thank the Finnish Centre for Scientific Computing for providing computational resources. SV and LAV are supported by Center for Computational Inference in Evolutionary Life Science (CELS) and the Department of Biosciences, University of Oslo. YG and RG contributions were supported by the Agence Nationale de la Recherche (ANR Blanc SVSE 7 2011, project SDS).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-41">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7728512">https://doi.org/10.25387/g3.7728512</a>.</p></li><li class="fn" id="fn-2"><p id="p-42"><em>Communicating editor: J. Yáñez</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-13"><span class="received-label">Received </span>December 13, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-04"><span class="accepted-label">Accepted </span>March 4, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Savilammi <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.1" data-doi="10.1093/bioinformatics/btp084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abrusán</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grundmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeMester</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makalowski</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TEclass-a tool for automated classification of unknown eukaryotic transposable elements.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1329</span>–<span class="cit-lpage">1330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp084</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp084%26rft_id%253Dinfo%253Apmid%252F19349283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19349283&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265950600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1283.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> Tetraploidy and the Evolution of Salmonid Fishes, pp 1–53 in <em>Evolutionary Genetics of Fishes</em>. Springer US, Boston, MA.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978–1-4684–4652–4_1</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.3" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.4" data-doi="10.1534/genetics.111.127324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontenot</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome Evolution and Meiotic Maps by Massively Parallel DNA Sequencing: Spotted Gar, an Outgroup for the Teleost Genome Duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">808</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.127324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127324%26rft_id%253Dinfo%253Apmid%252F21828280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzQvNzk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEyODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.5" data-doi="10.1186/s12864-018-4714-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auvinet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graça</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belkadi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnivard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Mobilization of retrotransposons as a cause of chromosomal diversification and rapid speciation: the case for the Antarctic teleost genus Trematomus.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4714-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D339%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4714-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4714-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.6" data-doi="10.1093/biostatistics/kxj037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the False Discovery Rate - a Practical and Powerful Approach to Multiple Testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxj037%26rft_id%253Dinfo%253Apmid%252F16632515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxj037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16632515&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.7" data-doi="10.1038/srep23246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Star</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pampoulie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sodeland</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barth</span>  <span class="cit-name-given-names">J. M. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Three chromosomal rearrangements promote genomic divergence between migratory and stationary ecotypes of Atlantic cod.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">23246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep23246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D23246%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep23246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep23246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.8" data-doi="10.1038/ncomms4657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berthelot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalopin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juanchich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3657</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms4657</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3657%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4657%26rft_id%253Dinfo%253Apmid%252F24755649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24755649&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.9" data-doi="10.1186/gb-2006-7-5-r43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blomme</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandepoele</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Bodt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simillion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The gain and loss of genes during 600 million years of vertebrate evolution.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2006-7-5-r43</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBlomme%26rft.auinit1%253DT.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253DR43%26rft.epage%253DR43%26rft.atitle%253DThe%2Bgain%2Band%2Bloss%2Bof%2Bgenes%2Bduring%2B600%2Bmillion%2Byears%2Bof%2Bvertebrate%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-5-r43%26rft_id%253Dinfo%253Apmid%252F16723033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-5-r43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16723033&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.10" data-doi="10.1159/000444429"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Canapa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barucca</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biscotti</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forconi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olmo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transposons, Genome Size, and Evolutionary Insights in Animals.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000444429</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenet.%2BGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000444429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000444429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.11" data-doi="10.1101/gr.6743907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cantarel</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robb</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MAKER: An easy-to-use annotation pipeline designed for emerging model organism genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.6743907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6743907%26rft_id%253Dinfo%253Apmid%252F18025269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEyODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.12" data-doi="10.1111/eva.12291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The status of supergenes in the 21st century: recombination suppression in Batesian mimicry and sex chromosomes and other complex adaptations.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/eva.12291</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D9%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12291%26rft_id%253Dinfo%253Apmid%252F27087840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27087840&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.13" data-doi="10.1002/0471250953.bi0410s05"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> Using RepeatMasker to Identify Repetitive Elements in Genomic Sequences. Current Protocols in Bioinformatics 5: 4.10.1–4.10.14.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471250953.bi0410s05</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi0410s05%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi0410s05&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955616&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.14" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.15" data-doi="10.1186/s13059-014-0546-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daron</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pingault</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theil</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamilloux</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Organization and evolution of transposable elements along the bread wheat chromosome 3B.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">546</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0546-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D546%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0546-4%26rft_id%253Dinfo%253Apmid%252F25476263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0546-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25476263&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.16" data-doi="10.1371/journal.pgen.1002384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">A. P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castoe</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzer</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollock</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Repetitive Elements May Comprise Over Two-Thirds of the Human Genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002384</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002384</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253Dde%2BKoning%26rft.auinit1%253DA.%2BP.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De1002384%26rft.epage%253De1002384%26rft.atitle%253DRepetitive%2Belements%2Bmay%2Bcomprise%2Bover%2Btwo-thirds%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002384%26rft_id%253Dinfo%253Apmid%252F22144907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002384&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22144907&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.17" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.18" data-doi="10.1007/s10577-009-9055-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene and genome duplications: the impact of dosage-sensitivity on the fate of nuclear genes.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">717</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-009-9055-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DEdger%26rft.auinit1%253DP.%2BP.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D699%26rft.epage%253D717%26rft.atitle%253DGene%2Band%2Bgenome%2Bduplications%253A%2Bthe%2Bimpact%2Bof%2Bdosage-sensitivity%2Bon%2Bthe%2Bfate%2Bof%2Bnuclear%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-009-9055-9%26rft_id%253Dinfo%253Apmid%252F19802709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-009-9055-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19802709&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270448800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.19" data-doi="10.1186/1471-2105-9-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellinghaus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willhoeft</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-9-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEllinghaus%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DLTRharvest%252C%2Ban%2Befficient%2Band%2Bflexible%2Bsoftware%2Bfor%2Bde%2Bnovo%2Bdetection%2Bof%2BLTR%2Bretrotransposons.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-18%26rft_id%253Dinfo%253Apmid%252F18194517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18194517&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.20" data-doi="10.1371/journal.pone.0047768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">English</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vee</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e47768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0047768</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEnglish%26rft.auinit1%253DA.%2BC.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De47768%26rft.epage%253De47768%26rft.atitle%253DMind%2Bthe%2BGap%253A%2BUpgrading%2BGenomes%2Bwith%2BPacific%2BBiosciences%2BRS%2BLong-Read%2BSequencing%2BTechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047768%26rft_id%253Dinfo%253Apmid%252F23185243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185243&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.21" data-doi="10.3389/fgene.2014.00295"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flaxman</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Assessing when chromosomal rearrangements affect the dynamics of speciation: implications from computer simulations.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00295</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D295%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00295&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.22" data-doi="10.1007/s00412-014-0477-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garagna</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Donoso</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuccotti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Robertsonian phenomenon in the house mouse: mutation, meiosis and speciation.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">544</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00412-014-0477-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D123%26rft.spage%253D529%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-014-0477-6%26rft_id%253Dinfo%253Apmid%252F25053180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-014-0477-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25053180&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.23" data-doi="10.1038/ng0393-266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">States</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Identification of protein coding regions by database similarity search.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">266</span>–<span class="cit-lpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng0393-266</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGish%26rft.auinit1%253DW.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D266%26rft.epage%253D272%26rft.atitle%253DIdentification%2Bof%2Bprotein%2Bcoding%2Bregions%2Bby%2Bdatabase%2Bsimilarity%2Bsearch.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0393-266%26rft_id%253Dinfo%253Apmid%252F8485583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0393-266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8485583&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KP79000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.24" data-doi="10.1093/nar/gkg770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">5654</span>–<span class="cit-lpage">5666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg770</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D31%26rft.issue%253D19%26rft.spage%253D5654%26rft.epage%253D5666%26rft.atitle%253DImproving%2Bthe%2BArabidopsis%2Bgenome%2Bannotation%2Busing%2Bmaximal%2Btranscript%2Balignment%2Bassemblies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg770%26rft_id%253Dinfo%253Apmid%252F14500829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14500829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185600400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.25" data-doi="10.1093/bioinformatics/btv661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lomsadze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">BRAKER1: Unsupervised RNA-Seq-Based Genome Annotation with GeneMark-ET and AUGUSTUS.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv661%26rft_id%253Dinfo%253Apmid%252F26559507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26559507&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.26" data-doi="10.1146/annurev.ecolsys.39.110707.173532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Revisiting the Impact of Inversions in Evolution: From Population Genetic Markers to Drivers of Adaptive Shifts and Speciation?</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.39.110707.173532</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnnu%2BRev%2BEcol%2BEvol%2BSyst%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D39%26rft.spage%253D21%26rft.epage%253D42%26rft.atitle%253DRevisiting%2Bthe%2BImpact%2Bof%2BInversions%2Bin%2BEvolution%253A%2BFrom%2BPopulation%2BGenetic%2BMarkers%2Bto%2BDrivers%2Bof%2BAdaptive%2BShifts%2Band%2BSpeciation%253F%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173532%26rft_id%253Dinfo%253Apmid%252F20419035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20419035&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261725500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.27" data-doi="10.1016/j.gde.2009.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufton</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panopoulou</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Polyploidy and genome restructuring: a variety of outcomes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gde.2009.10.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DHufton%26rft.auinit1%253DA.%2BL.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D600%26rft.epage%253D606%26rft.atitle%253DPolyploidy%2Band%2Bgenome%2Brestructuring%253A%2Ba%2Bvariety%2Bof%2Boutcomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2009.10.005%26rft_id%253Dinfo%253Apmid%252F19900800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2009.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19900800&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273117900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.28" data-doi="10.1159/000084979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapitonov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlicek</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klonowski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohany</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Repbase Update, a database of eukaryotic repetitive elements.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000084979</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DJurka%26rft.auinit1%253DJ.%26rft.volume%253D110%26rft.issue%253D1-4%26rft.spage%253D462%26rft.epage%253D467%26rft.atitle%253DRepbase%2BUpdate%252C%2Ba%2Bdatabase%2Bof%2Beukaryotic%2Brepetitive%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000084979%26rft_id%253Dinfo%253Apmid%252F16093699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000084979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16093699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231064600047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaminker</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronmiller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svirskas</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> The transposable elements of the <em>Drosophila melanogaster</em> euchromatin: a genomics perspective. Genome Biol. 3: RESEARCH0084.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2002–3-12-research0084</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.30" data-doi="10.1126/science.1089670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kazazian</span>  <span class="cit-name-given-names">H. H.</span>  <span class="cit-name-suffix">Jr.</span>.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mobile elements: drivers of genome evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">1626</span>–<span class="cit-lpage">1632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1089670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKazazian%26rft.auinit1%253DH.%2BH.%26rft.volume%253D303%26rft.issue%253D5664%26rft.spage%253D1626%26rft.epage%253D1632%26rft.atitle%253DMobile%2BElements%253A%2BDrivers%2Bof%2BGenome%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1089670%26rft_id%253Dinfo%253Apmid%252F15016989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2NC8xNjI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEyODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.31" data-doi="10.1111/mec.13592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirubakaran</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grove</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Two adjacent inversions maintain genomic differentiation between migratory and stationary ecotypes of Atlantic cod.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2130</span>–<span class="cit-lpage">2143</span> (erratum: Mol. Ecol. 27: 1520).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13592</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D2130%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.32" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walenz</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEyODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.33" data-doi="10.1186/1471-2105-5-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-5-59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DGene%2Bfinding%2Bin%2Bnovel%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-59%26rft_id%253Dinfo%253Apmid%252F15144565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144565&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.34" data-doi="10.1186/gb-2004-5-2-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Versatile and open software for comparing large genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2004-5-2-r12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DVersatile%2Band%2Bopen%2Bsoftware%2Bfor%2Bcomparing%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-2-r12%26rft_id%253Dinfo%253Apmid%252F14759262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-2-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14759262&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.35" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.36" data-doi="10.1093/bioinformatics/btw157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohammed Ismail</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rho</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MGEScan: a Galaxy-based system for identifying retrotransposons in genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2502</span>–<span class="cit-lpage">2504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw157%26rft_id%253Dinfo%253Apmid%252F27153595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153595&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.37" data-doi="10.1111/j.1601-5223.1964.tb01953.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredga</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandberg</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Nomenclature for Centromeric Position on Chromosomes.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-5223.1964.tb01953.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.volume%253D52%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.1964.tb01953.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-5223.1964.tb01953.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19646182600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.38" data-doi="10.1126/sciadv.1701256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knisbacher</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levanon</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Havlin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Integrating networks and comparative genomics reveals retroelement proliferation dynamics in hominid genomes.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e1701256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/sciadv.1701256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DLevy%26rft.auinit1%253DO.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De1701256%26rft.epage%253De1701256%26rft.atitle%253DIntegrating%2Bnetworks%2Band%2Bcomparative%2Bgenomics%2Breveals%2Bretroelement%2Bproliferation%2Bdynamics%2Bin%2Bhominid%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1701256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiMy8xMC9lMTcwMTI1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xMjgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.39" data-doi="10.1038/nature17164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Atlantic salmon genome provides insights into rediploidization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">533</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature17164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.40" data-doi="10.1093/nar/gki937"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lomsadze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ter-Hovhannisyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernoff</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene identification in novel eukaryotic genomes by self-training algorithm.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">6494</span>–<span class="cit-lpage">6506</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gki937</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLomsadze%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253D20%26rft.spage%253D6494%26rft.epage%253D6506%26rft.atitle%253DGene%2Bidentification%2Bin%2Bnovel%2Beukaryotic%2Bgenomes%2Bby%2Bself-training%2Balgorithm%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki937%26rft_id%253Dinfo%253Apmid%252F16314312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki937&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16314312&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233843600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.41" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Evolutionary Fate and Consequences of Duplicate Genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.290.5494.1151</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEyODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.42" data-doi="10.1126/science.1089370"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Origins of Genome Complexity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1089370</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D302%26rft.issue%253D5649%26rft.spage%253D1401%26rft.epage%253D1404%26rft.atitle%253DThe%2BOrigins%2Bof%2BGenome%2BComplexity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1089370%26rft_id%253Dinfo%253Apmid%252F14631042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDIvNTY0OS8xNDAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEyODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.43" data-doi="10.1073/pnas.0702207104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The frailty of adaptive hypotheses for the origins of organismal complexity.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8597</span>–<span class="cit-lpage">8604</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0702207104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0702207104%26rft_id%253Dinfo%253Apmid%252F17494740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNjoiMTA0L3N1cHBsXzEvODU5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xMjgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.44" data-doi="10.1098/rspb.2013.2881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macqueen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20132881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2013.2881</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2881%26rft_id%253Dinfo%253Apmid%252F24452024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2013.2881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24452024&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.45" data-doi="10.3390/genes8110297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makhrov</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Narrowing of the Phenotypic Diversity Range after Large Rearrangements of the Karyotype in Salmonidae: The Relationship between Saltational Genome Rearrangements and Gradual Adaptive Evolution.</span> <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">297</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/genes8110297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D8%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes8110297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes8110297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.46" data-doi="10.1093/bioinformatics/btr011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marçais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsford</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">764</span>–<span class="cit-lpage">770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr011%26rft_id%253Dinfo%253Apmid%252F21217122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217122&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288277300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.47" data-doi="10.1002/evl3.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Ancient genomic variation underlies repeated ecological adaptation in young stickleback populations.</span> <abbr class="cit-jnl-abbrev">Evol Lett.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/evl3.37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BLett.%26rft.volume%253D2%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fevl3.37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/evl3.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.48" data-doi="10.1007/s10709-013-9740-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ocalewicz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furgala-Selezniow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szmyt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisboa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kucinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pericentromeric location of the telomeric DNA sequences on the European grayling chromosomes.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">416</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-013-9740-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D141%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-013-9740-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-013-9740-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1283.49" data-doi="10.1007/978-3-642-86659-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">Evolution by Gene Duplication</span>, <span class="cit-publ-name">Springer Science &amp; Business Media</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-86659-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2Bby%2BGene%2BDuplication%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-86659-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-86659-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.50" data-doi="10.1017/S1464793100005613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ráb</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome evolution in the Salmonidae (Pisces): an update.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S1464793100005613</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D25%26rft.atitle%253DChromosome%2Bevolution%2Bin%2Bthe%2BSalmonidae%2B%2528Pisces%2529%253A%2Ban%2Bupdate.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1464793100005613%26rft_id%253Dinfo%253Apmid%252F11325050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1464793100005613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11325050&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.51" data-doi="10.1093/oxfordjournals.jhered.a111501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qumsiyeh</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Evolution of Number and Morphology of Mammalian Chromosomes.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">465</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.jhered.a111501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DQumsiyeh%26rft.auinit1%253DM.%2BB.%26rft.volume%253D85%26rft.issue%253D6%26rft.spage%253D455%26rft.epage%253D465%26rft.atitle%253DEvolution%2Bof%2BNumber%2Band%2BMorphology%2Bof%2BMammalian%2BChromosomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a111501%26rft_id%253Dinfo%253Apmid%252F7995926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.jhered.a111501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7995926&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PW57600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1283.52"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>, <span class="cit-pub-date">2017</span> R: a language and environment for statistical computing. R foundation for statistical computing, Vienna, Austria: <a href="http://www.r-project.org/">http://www.r-project.org/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.53" data-doi="10.1093/gbe/evv250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calboli</span>  <span class="cit-name-given-names">F. C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction of Ultradense Linkage Maps with Lep-MAP2: Stickleback F-2 Recombinant Crosses as an Example.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv250%26rft_id%253Dinfo%253Apmid%252F26668116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26668116&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.54" data-doi="10.1016/S0169-5347(01)02187-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosomal rearrangements and speciation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0169-5347(01)02187-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DRieseberg%26rft.auinit1%253DL.%2BH.%26rft.volume%253D16%26rft.issue%253D7%26rft.spage%253D351%26rft.epage%253D358%26rft.atitle%253DChromosomal%2Brearrangements%2Band%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252801%252902187-5%26rft_id%253Dinfo%253Apmid%252F11403867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(01)02187-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11403867&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169461000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.55" data-doi="10.1186/s13059-017-1241-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundappa</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grammes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hvidsten</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redmond</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Lineage-specific rediploidization is a mechanism to explain time-lags between genome duplication and evolutionary diversification.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1241-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1241-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1241-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.56" data-doi="10.1371/journal.pone.0102089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rondeau</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minkley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messmer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jantzen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Genome and Linkage Map of the Northern Pike (<em>Esox lucius</em>): Conserved Synteny Revealed between the Salmonid Sister Group and the Neoteleostei.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e102089</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0102089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De102089%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0102089%26rft_id%253Dinfo%253Apmid%252F25069045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0102089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25069045&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.57" data-doi="10.1093/molbev/msm003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Semon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rearrangement rate following the whole-genome duplication in teleosts.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">860</span>–<span class="cit-lpage">867</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm003%26rft_id%253Dinfo%253Apmid%252F17218642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17218642&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244662000024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.58" data-doi="10.1134/S1022795413060112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shedko</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miroshnichenko</span>  <span class="cit-name-given-names">I. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemkova</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phylogeny of Salmonids (Salmoniformes: Salmonidae) and its Molecular Dating: Analysis of mtDNA Data.</span> <abbr class="cit-jnl-abbrev">Russ. J. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1134/S1022795413060112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRuss.%2BJ.%2BGenet.%26rft.volume%253D49%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1134%252FS1022795413060112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1134/S1022795413060112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.59" data-doi="10.1371/journal.pone.0026314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunstner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ConDeTri--a - A Content Dependent Read Trimmer for Illumina Data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e26314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0026314</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSmeds%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De26314%26rft.epage%253De26314%26rft.atitle%253DConDeTri--a%2Bcontent%2Bdependent%2Bread%2Btrimmer%2Bfor%2BIllumina%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0026314%26rft_id%253Dinfo%253Apmid%252F22039460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0026314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039460&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1283.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smit</span>  <span class="cit-name-given-names">A. F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013–2015</span> RepeatMasker Open-4.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.61" data-doi="10.1093/nar/gkp759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinbiss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willhoeft</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gremme</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fine-grained annotation and classification of de novo predicted LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">7002</span>–<span class="cit-lpage">7013</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkp759</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp759%26rft_id%253Dinfo%253Apmid%252F19786494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19786494&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272688400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.62" data-doi="10.1093/gbe/evw262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">B. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosselin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normandeau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamothe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Salmonid Chromosome Evolution as Revealed by a Novel Method for Comparing RADseq Linkage Maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3600</span>–<span class="cit-lpage">3617</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.63" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1621</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.64" data-doi="10.1038/nrg2600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The evolutionary significance of ancient genome duplications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">732</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2600</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DVan%2Bde%2BPeer%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D725%26rft.epage%253D732%26rft.atitle%253DThe%2Bevolutionary%2Bsignificance%2Bof%2Bancient%2Bgenome%2Bduplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2600%26rft_id%253Dinfo%253Apmid%252F19652647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2600&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19652647&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269965100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.65" data-doi="10.1093/gbe/evy201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varadharajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillard</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tørresen</span>  <span class="cit-name-given-names">O. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulugeta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The grayling genome reveals selection on gene expression regulation after whole genome duplication.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2785</span>–<span class="cit-lpage">2800</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evy201</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D10%26rft.spage%253D2785%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevy201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evy201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.66" data-doi="10.1371/journal.pgen.1004482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wacholder</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruggiero</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vemulapalli</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Inference of Transposable Element Ancestry.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004482</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004482</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004482%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.67" data-doi="10.1016/j.tree.2018.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellenreuther</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Eco-Evolutionary Genomics of Chromosomal Inversions.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2018.04.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D33%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2018.04.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2018.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.68" data-doi="10.1038/nrg2165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua-Van</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A unified classification system for eukaryotic transposable elements.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMathew%26rft.auinit1%253DC.%2BG.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D973%26rft.epage%253D982%26rft.atitle%253DA%2Bunified%2Bclassification%2Bsystem%2Bfor%2Beukaryotic%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2165%26rft_id%253Dinfo%253Apmid%252F17984973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17984973&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.69" data-doi="10.1111/eva.12032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jouanno</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quillet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fostier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The sexually dimorphic on the Y-chromosome gene (sdY) is a conserved male-specific Y-chromosome sequence in many salmonids.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/eva.12032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D6%26rft.spage%253D486%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12032%26rft_id%253Dinfo%253Apmid%252F23745140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23745140&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1283.70" data-doi="10.1093/bioinformatics/17.9.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">InterProScan - an integration platform for the signature-recognition methods in InterPro.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">848</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/17.9.847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DZdobnov%26rft.auinit1%253DE.%2BM.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D847%26rft.epage%253D848%26rft.atitle%253DInterProScan%2B-%2Ban%2Bintegration%2Bplatform%2Bfor%2Bthe%2Bsignature-recognition%2Bmethods%2Bin%2BInterPro%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.9.847%26rft_id%253Dinfo%253Apmid%252F11590104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.9.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11590104&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1283.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171566400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1283.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1281" title="Corrigendum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1295" title="The Whole Genome Sequence and mRNA Transcriptome of the Tropical Cyclopoid Copepod Apocyclops royi" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1283" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CuXrcVkz1hJzw7BpIS7C149wE3ZdtHuuxo9joH0b77M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1283" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1283" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1283" class="active" data-icon-position="" data-hide-link-title="0">The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493108" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-GDUacnVP7GJC7oGkdwKkRc8MUgQHV8ejKbPLc3Ea7b8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1283.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1283" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-I4ir5sA4mRjTEXEdZsvS_xxWFr7S8BltctX2WNR03To" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1283" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1283.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493108" id="node493108--2" data-pisa="ggg;9/5/1283" data-pisa-master="ggg;g3.118.200919" data-apath="/ggg/9/5/1283.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tiina Sävilammi</span>, <span class="highwire-citation-author" data-delta="1">Craig R. Primmer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2033-0758" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Srinidhi Varadharajan</span>, <span class="highwire-citation-author" data-delta="3">René Guyomard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yann Guiguen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-4989-5311" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simen R. Sandve</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9389-7982" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>L. Asbjørn Vøllestad</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-5563-0048" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Spiros Papakostas</span> and <span class="highwire-citation-author" data-delta="8">Sigbjørn Lien</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1283-1294;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200919 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493108--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tiina Sävilammi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Turku, 20014 Turku, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiina%2BS%C3%A4vilammi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=S%C3%A4vilammi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiina%2BS%25C3%25A4vilammi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Craig R. Primmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Organismal & Evolutionary Biology Research Program, Faculty of Biological & Environmental Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Biotechnology, University of Helsinki, 00014 Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BR.%2BPrimmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Primmer%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACraig%2BR.%2BPrimmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Srinidhi Varadharajan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biosciences, University of Oslo, 0316 Oslo, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srinidhi%2BVaradharajan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varadharajan%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASrinidhi%2BVaradharajan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2033-0758" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Srinidhi Varadharajan</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">René Guyomard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>INRA, UMR1313 GABI Génétique Animale et Biologie Intégrative, Domaine de Vilvert, 78352, Jouy-en-Josas Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ren%C3%A9%2BGuyomard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guyomard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARen%25C3%25A9%2BGuyomard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yann Guiguen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>INRA, UR1037 Fish Physiology and Genomics, F-35000, Rennes, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BGuiguen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guiguen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYann%2BGuiguen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yann Guiguen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Simen R. Sandve </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centre for Integrative Genetics (CIGENE), Department of Animal and Aquacultural Sciences, Faculty of Biosciences, Norwegian University of Life Sciences, 1430 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simen%2BR.%2BSandve%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sandve%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimen%2BR.%2BSandve%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4989-5311" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simen R. Sandve</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">L. Asbjørn Vøllestad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biosciences, University of Oslo, 0316 Oslo, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BAsbj%C3%B8rn%2BV%C3%B8llestad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=V%C3%B8llestad%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AL.%2BAsbj%25C3%25B8rn%2BV%25C3%25B8llestad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9389-7982" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for L. Asbjørn Vøllestad</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Spiros Papakostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Turku, 20014 Turku, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Spiros%2BPapakostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Papakostas%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASpiros%2BPapakostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5563-0048" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Spiros Papakostas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sigbjorn.lien@nmbu.no" class="" data-icon-position="" data-hide-link-title="0">sigbjorn.lien@nmbu.no</a>
<a href="mailto:spiros.papakostas@utu.fi" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@utu.fi</a>
<a href="mailto:spiros.papakostas@gmail.com" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@gmail.com</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sigbjørn Lien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centre for Integrative Genetics (CIGENE), Department of Animal and Aquacultural Sciences, Faculty of Biosciences, Norwegian University of Life Sciences, 1430 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sigbj%C3%B8rn%2BLien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lien%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASigbj%25C3%25B8rn%2BLien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sigbjorn.lien@nmbu.no" class="" data-icon-position="" data-hide-link-title="0">sigbjorn.lien@nmbu.no</a>
<a href="mailto:spiros.papakostas@utu.fi" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@utu.fi</a>
<a href="mailto:spiros.papakostas@gmail.com" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1283" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1283&amp;title=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1283&amp;title=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1283&amp;title=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1283&amp;text=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1283&amp;title=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1283&amp;t=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1283&amp;title=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1283&amp;title=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493108/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493108" id="node493108--3" data-pisa="ggg;9/5/1283" data-pisa-master="ggg;g3.118.200919" data-apath="/ggg/9/5/1283.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Chromosome-Level Genome Assembly of European Grayling Reveals Aspects of a Unique Genome Evolution Process Within Salmonids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tiina Sävilammi</span>, <span class="highwire-citation-author" data-delta="1">Craig R. Primmer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2033-0758" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Srinidhi Varadharajan</span>, <span class="highwire-citation-author" data-delta="3">René Guyomard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yann Guiguen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-4989-5311" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simen R. Sandve</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9389-7982" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>L. Asbjørn Vøllestad</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-5563-0048" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Spiros Papakostas</span> and <span class="highwire-citation-author" data-delta="8">Sigbjørn Lien</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1283-1294;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200919 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493108--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tiina Sävilammi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Turku, 20014 Turku, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiina%2BS%C3%A4vilammi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=S%C3%A4vilammi%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiina%2BS%25C3%25A4vilammi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Craig R. Primmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Organismal & Evolutionary Biology Research Program, Faculty of Biological & Environmental Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Biotechnology, University of Helsinki, 00014 Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BR.%2BPrimmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Primmer%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACraig%2BR.%2BPrimmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Srinidhi Varadharajan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biosciences, University of Oslo, 0316 Oslo, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srinidhi%2BVaradharajan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varadharajan%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASrinidhi%2BVaradharajan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2033-0758" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Srinidhi Varadharajan</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">René Guyomard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>INRA, UMR1313 GABI Génétique Animale et Biologie Intégrative, Domaine de Vilvert, 78352, Jouy-en-Josas Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ren%C3%A9%2BGuyomard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guyomard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARen%25C3%25A9%2BGuyomard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yann Guiguen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>INRA, UR1037 Fish Physiology and Genomics, F-35000, Rennes, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BGuiguen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guiguen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYann%2BGuiguen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yann Guiguen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Simen R. Sandve </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centre for Integrative Genetics (CIGENE), Department of Animal and Aquacultural Sciences, Faculty of Biosciences, Norwegian University of Life Sciences, 1430 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simen%2BR.%2BSandve%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sandve%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimen%2BR.%2BSandve%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4989-5311" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simen R. Sandve</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">L. Asbjørn Vøllestad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biosciences, University of Oslo, 0316 Oslo, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BAsbj%C3%B8rn%2BV%C3%B8llestad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=V%C3%B8llestad%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AL.%2BAsbj%25C3%25B8rn%2BV%25C3%25B8llestad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9389-7982" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for L. Asbjørn Vøllestad</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Spiros Papakostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Turku, 20014 Turku, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Spiros%2BPapakostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Papakostas%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASpiros%2BPapakostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5563-0048" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Spiros Papakostas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sigbjorn.lien@nmbu.no" class="" data-icon-position="" data-hide-link-title="0">sigbjorn.lien@nmbu.no</a>
<a href="mailto:spiros.papakostas@utu.fi" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@utu.fi</a>
<a href="mailto:spiros.papakostas@gmail.com" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@gmail.com</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sigbjørn Lien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centre for Integrative Genetics (CIGENE), Department of Animal and Aquacultural Sciences, Faculty of Biosciences, Norwegian University of Life Sciences, 1430 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sigbj%C3%B8rn%2BLien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lien%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASigbj%25C3%25B8rn%2BLien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sigbjorn.lien@nmbu.no" class="" data-icon-position="" data-hide-link-title="0">sigbjorn.lien@nmbu.no</a>
<a href="mailto:spiros.papakostas@utu.fi" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@utu.fi</a>
<a href="mailto:spiros.papakostas@gmail.com" class="" data-icon-position="" data-hide-link-title="0">spiros.papakostas@gmail.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1283&amp;rft.epage=1294&amp;rft.atitle=The%20Chromosome-Level%20Genome%20Assembly%20of%20European%20Grayling%20Reveals%20Aspects%20of%20a%20Unique%20Genome%20Evolution%20Process%20Within%20Salmonids&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=S%C3%A4vilammi%2C+Tiina&amp;rft.au=Primmer%2C+Craig+R.&amp;rft.au=Varadharajan%2C+Srinidhi&amp;rft.au=Guyomard%2C+Ren%C3%A9&amp;rft.au=Guiguen%2C+Yann&amp;rft.au=Sandve%2C+Simen+R.&amp;rft.au=V%C3%B8llestad%2C+L.+Asbj%C3%B8rn&amp;rft.au=Papakostas%2C+Spiros&amp;rft.au=Lien%2C+Sigbj%C3%B8rn"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493108/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493108/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493108/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493108/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493108/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493108/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493108/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493108/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493108/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493108/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493108/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493108/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1283" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1283.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1283.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1283.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
