<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1469" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1469.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1469.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1469.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1469/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1469.atom" />
<meta name="HW.pisa" content="ggg;9/5/1469" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression" />
<meta name="DC.Identifier" content="10.1534/g3.119.400129" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Allier et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Predicting the usefulness of crosses in terms of expected genetic gain and genetic diversity is of interest to secure performance in the progeny and to maintain long-term genetic gain in plant breeding. A wide range of crossing schemes are possible including large biparental crosses, backcrosses, four-way crosses, and synthetic populations. In silico progeny simulations together with genome-based prediction of quantitative traits can be used to guide mating decisions. However, the large number of multi-parental combinations can hinder the use of simulations in practice. Analytical solutions have been proposed recently to predict the distribution of a quantitative trait in the progeny of biparental crosses using information of recombination frequency and linkage disequilibrium between loci. Here, we extend this approach to obtain the progeny distribution of more complex crosses including two to four parents. Considering agronomic traits and parental genome contribution as jointly multivariate normally distributed traits, the usefulness criterion parental contribution (UCPC) enables to (i) evaluate the expected genetic gain for agronomic traits, and at the same time (ii) evaluate parental genome contributions to the selected fraction of progeny. We validate and illustrate UCPC in the context of multiple allele introgression from a donor into one or several elite recipients in maize ( Zea mays L.). Recommendations regarding the interest of two-way, three-way, and backcrosses were derived depending on the donor performance. We believe that the computationally efficient UCPC approach can be useful for mate selection and allocation in many plant and animal breeding contexts." />
<meta name="DC.Contributor" content="Antoine Allier" />
<meta name="DC.Contributor" content="Laurence Moreau" />
<meta name="DC.Contributor" content="Alain Charcosset" />
<meta name="DC.Contributor" content="Simon Teyssèdre" />
<meta name="DC.Contributor" content="Christina Lehermeier" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Predicting the usefulness of crosses in terms of expected genetic gain and genetic diversity is of interest to secure performance in the progeny and to maintain long-term genetic gain in plant breeding. A wide range of crossing schemes are possible including large biparental crosses, backcrosses, four-way crosses, and synthetic populations. &lt;i&gt;In silico&lt;/i&gt; progeny simulations together with genome-based prediction of quantitative traits can be used to guide mating decisions. However, the large number of multi-parental combinations can hinder the use of simulations in practice. Analytical solutions have been proposed recently to predict the distribution of a quantitative trait in the progeny of biparental crosses using information of recombination frequency and linkage disequilibrium between loci. Here, we extend this approach to obtain the progeny distribution of more complex crosses including two to four parents. Considering agronomic traits and parental genome contribution as jointly multivariate normally distributed traits, the usefulness criterion parental contribution (UCPC) enables to (i) evaluate the expected genetic gain for agronomic traits, and at the same time (ii) evaluate parental genome contributions to the selected fraction of progeny. We validate and illustrate UCPC in the context of multiple allele introgression from a donor into one or several elite recipients in maize (&lt;i&gt;Zea mays&lt;/i&gt; L.). Recommendations regarding the interest of two-way, three-way, and backcrosses were derived depending on the donor performance. We believe that the computationally efficient UCPC approach can be useful for mate selection and allocation in many plant and animal breeding contexts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1469" />
<meta name="citation_id" content="9/5/1469" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1469" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1469.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1469.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1469.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400129" />
<meta name="citation_pmid" content="30819823" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="1469" />
<meta name="citation_lastpage" content="1479" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Antoine Allier" />
<meta name="citation_author_institution" content="GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France" />
<meta name="citation_author_institution" content="RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6578-1715" />
<meta name="citation_author" content="Laurence Moreau" />
<meta name="citation_author_institution" content="GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7195-1327" />
<meta name="citation_author" content="Alain Charcosset" />
<meta name="citation_author_institution" content="GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6125-503X" />
<meta name="citation_author" content="Simon Teyssèdre" />
<meta name="citation_author_institution" content="RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France" />
<meta name="citation_author_email" content="clehermeier@ragt.fr" />
<meta name="citation_author_email" content="steyssedre@ragt.fr" />
<meta name="citation_author" content="Christina Lehermeier" />
<meta name="citation_author_institution" content="RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France" />
<meta name="citation_author_email" content="clehermeier@ragt.fr" />
<meta name="citation_author_email" content="steyssedre@ragt.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7724-0887" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=D. Akdemir;citation_author=J. I. Sánchez;citation_title=Efficient Breeding by Genomic Mating.;citation_volume=7;citation_year=2016;citation_doi=10.3389/fgene.2016.00210" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=D. Akdemir;citation_author=W. Beavis;citation_author=R. Fritsche-Neto;citation_author=A. K. Singh;citation_author=J. Isidro-Sánchez;citation_title=Multi-objective optimized genomic breeding strategies for sustainable food improvement.;citation_year=2018;citation_doi=10.1038/s41437-018-0147-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=E. Bauer;citation_author=M. Falque;citation_author=H. Walter;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Intraspecific variation of recombination rate in maize.;citation_pages=R103;citation_volume=14;citation_year=2013;citation_pmid=24050704;citation_doi=10.1186/gb-2013-14-9-r103" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=L. Moreau;citation_author=A. Charcosset;citation_title=Number and Fitness of Selected Individuals in Marker-Assisted and Phenotypic Recurrent Selection;citation_pages=1972-1980;citation_volume=46;citation_year=2006;citation_issue=5;citation_doi=10.2135/cropsci2006.01-0057" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Genomewide Selection for Rapid Introgression of Exotic Germplasm in Maize;citation_pages=419-425;citation_volume=49;citation_year=2009;citation_issue=2;citation_doi=10.2135/cropsci2008.08.0452" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. Bernardo;citation_title=Genomewide Selection of Parental Inbreds: Classes of Loci and Virtual Biparental Populations.;citation_pages=2586-2595;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2014.01.0088" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. De Beukelaer;citation_author=Y. Badke;citation_author=V. Fack;citation_author=G. D. Meyer;citation_title=Moving beyond managing realized genomic relationship in long-term genomic selection.;citation_volume=206;citation_year=2017;citation_pmid=28381589;citation_doi=10.1534/genetics.116.194449" />
<meta name="citation_reference" content="Bijma, P., 2000 Long-term genetic contributions: prediction of rates of inbreeding and genetic gain in selected populations (Doctoral dissertation). Veenendaal, The Netherlands." />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Bonk;citation_author=M. Reichelt;citation_author=F. Teuscher;citation_author=D. Segelke;citation_author=N. Reinsch;citation_title=Mendelian sampling covariability of marker effects and genetic values.;citation_pages=36;citation_volume=48;citation_year=2016;citation_doi=10.1186/s12711-016-0214-0" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Charmet;citation_author=N. Robert;citation_author=M. R. Perretant;citation_author=G. Gay;citation_author=P. Sourdille;citation_title=Marker-assisted recurrent selection for cumulating additive and interactive QTLs in recombinant inbred lines.;citation_pages=1143-1148;citation_volume=99;citation_year=1999;citation_doi=10.1007/s001220051318" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. W. Dudley;citation_title=A Method of Identifying Lines for Use in Improving Parents of a Single Cross;citation_pages=355-357;citation_volume=24;citation_year=1984;citation_issue=2;citation_doi=10.2135/cropsci1984.0011183X002400020034x" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics. Ed. 4th.;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Frisch;citation_author=M. Bohn;citation_author=A. E. Melchinger;citation_title=Comparison of Selection Strategies for Marker-Assisted Backcrossing of a Gene;citation_pages=1295-1301;citation_volume=39;citation_year=1999;citation_issue=5;citation_doi=10.2135/cropsci1999.3951295x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Frisch;citation_author=A. E. Melchinger;citation_title=Variance of the Parental Genome Contribution to Inbred Lines Derived From Biparental Crosses.;citation_volume=176;citation_year=2007;citation_pmid=17409089;citation_doi=10.1534/genetics.106.065433" />
<meta name="citation_reference" content="citation_title=Théorie de la sélection en amélioration des plantes;citation_author=A. Gallais;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Bérard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Giraud;citation_author=C. Lehermeier;citation_author=E. Bauer;citation_author=M. Falque;citation_author=V. Segura;citation_title=Linkage Disequilibrium with Linkage Analysis of Multiline Crosses Reveals Different Multiallelic QTL for Hybrid Performance in the Flint and Dent Heterotic Groups of Maize.;citation_volume=198;citation_year=2014;citation_pmid=25271305;citation_doi=10.1534/genetics.114.169367" />
<meta name="citation_reference" content="Goodman M. M., 2000 Incorporation of exotic germplasm into elite maize lines: Maximizing favorable effects of the exotic source. Theor. Pop. Biol." />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=G. Gorjanc;citation_author=J. Jenko;citation_author=S. J. Hearne;citation_author=J. M. Hickey;citation_title=Initiating maize pre-breeding programs using genomic selection to harness polygenic variation from landrace populations.;citation_pages=30;citation_volume=17;citation_year=2016;citation_pmid=26732811;citation_doi=10.1186/s12864-015-2345-z" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Gorjanc;citation_author=R. C. Gaynor;citation_author=J. M. Hickey;citation_title=Optimal cross selection for long-term genetic gain in two-part programs with rapid recurrent genomic selection.;citation_pages=1953-1966;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-018-3125-3" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. Haldane;citation_title=The combination of linkage values, and the calculation of distances between the loci of linked factors.;citation_pages=299-309;citation_volume=8;citation_year=1919" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Han;citation_author=J. N. Cameron;citation_author=L. Wang;citation_author=W. D. Beavis;citation_title=The Predicted Cross Value for Genetic Introgression of Multiple Alleles.;citation_volume=205;citation_year=2017;citation_pmid=28122824;citation_doi=10.1534/genetics.116.197095" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=W. G. Hill;citation_title=Variation in Genetic Composition in Backcrossing Programs;citation_pages=212-213;citation_volume=84;citation_year=1993;citation_issue=3;citation_doi=10.1093/oxfordjournals.jhered.a111319" />
<meta name="citation_reference" content="citation_journal_title=J. Multivariate Anal.;citation_author=W. C. Horrace;citation_title=Some results on the multivariate truncated normal distribution.;citation_pages=209-221;citation_volume=94;citation_year=2005;citation_doi=10.1016/j.jmva.2004.10.007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Hospital;citation_author=A. Charcosset;citation_title=Marker-Assisted Introgression of Quantitative Trait Loci;citation_pages=1469-1485;citation_volume=147;citation_year=1997;citation_issue=3;citation_pmid=9383086" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=H. Iwata;citation_author=T. Hayashi;citation_author=S. Terakami;citation_author=N. Takada;citation_author=T. Saito;citation_title=Genomic prediction of trait segregation in a progeny population: a case study of Japanese pear (Pyrus pyrifolia).;citation_pages=81;citation_volume=14;citation_year=2013;citation_doi=10.1186/1471-2156-14-81" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. Jiao;citation_author=P. Peluso;citation_author=J. Shi;citation_author=T. Liang;citation_author=M. C. Stitzer;citation_title=Improved maize reference genome with single-molecule technologies.;citation_pages=524-527;citation_volume=546;citation_year=2017;citation_pmid=28605751;citation_doi=10.1038/nature22971" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Lehermeier;citation_author=C.-C. Schön;citation_author=G. de Los Campos;citation_title=Assessment of Genetic Heterogeneity in Structured Plant Populations Using Multivariate Whole-Genome Regression Models.;citation_volume=201;citation_year=2015;citation_pmid=26122758;citation_doi=10.1534/genetics.115.177394" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=C. Lehermeier;citation_author=G. de los Campos;citation_author=V. Wimmer;citation_author=C.-C. Schön;citation_title=Genomic variance estimates: With or without disequilibrium covariances?;citation_pages=232-241;citation_volume=134;citation_year=2017;citation_doi=10.1111/jbg.12268" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Lehermeier;citation_author=S. Teyssèdre;citation_author=C.-C. Schön;citation_title=Genetic Gain Increases by Applying the Usefulness Criterion with Improved Variance Prediction in Selection of Crosses.;citation_volume=207;citation_year=2017;citation_pmid=29038144;citation_doi=10.1534/genetics.117.300403" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=L. Lian;citation_author=A. Jacobson;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=Prediction of genetic variance in biparental maize populations: Genomewide marker effects vs. mean genetic variance in prior populations.;citation_pages=1181-1188;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.10.0729" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. Mohammadi;citation_author=T. Tiede;citation_author=K. Smith;citation_title=PopVar: A Genome-Wide Procedure for Predicting Genetic Variance and Correlated Response in Biparental Breeding Populations.;citation_pages=2068-2077;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2015.01.0030" />
<meta name="citation_reference" content="citation_author=D. Müller;citation_author=P. Schopp;citation_author=A. E. Melchinger;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. Osthushenrich;citation_author=M. Frisch;citation_author=E. Herzog;citation_title=Genomic selection of crossing partners on basis of the expected mean and variance of their derived lines.;citation_pages=e0188839;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0188839" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Rio;citation_author=T. Mary-Huard;citation_author=L. Moreau;citation_author=A. Charcosset;citation_title=Genomic selection efficiency and a priori estimation of accuracy in a structured dent maize panel.;citation_pages=81-96;citation_volume=132;citation_year=2019;citation_doi=10.1007/s00122-018-3196-1" />
<meta name="citation_reference" content="citation_title=Bericht über die Arbeitstagung der Vereinigung österreichischer Pflanzenzüchter;citation_title=F1-Leistung und Elternwahl in der Züchtung von Selbstbefruchtern.;citation_author=F. Schnell;citation_author=H. Utz;citation_pages=243-248;citation_year=1975" />
<meta name="citation_reference" content="citation_author=P. Schopp;citation_author=D. Müller;citation_author=Y. C. J. Wientjes;citation_author=A. E. Melchinger;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Servin;citation_author=O. C. Martin;citation_author=M. Mézard;citation_author=F. Hospital;citation_title=Toward a Theory of Marker-Assisted Gene Pyramiding;citation_pages=513-523;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454561;citation_doi=10.1534/genetics.103.023358" />
<meta name="citation_reference" content="citation_title=A tactical approach to the design of crossbreeding programs. In Proceedings of the sixth world congress on genetics applied to livestock production, Armidale;citation_author=R. K. Shepherd;citation_author=B. P. Kinghorn;citation_pages=431-438;citation_volume=11–16;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. F. Troyer;citation_title=Background of U.S. Hybrid Corn;citation_pages=601-626;citation_volume=39;citation_year=1999;citation_issue=3;citation_doi=10.2135/cropsci1999.0011183X003900020001x" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=D. Van Inghelandt;citation_author=A. E. Melchinger;citation_author=J.-P. Martinant;citation_author=B. Stich;citation_title=Genome-wide association mapping of flowering time and northern corn leaf blight (Setosphaeria turcica) resistance in a vast commercial maize germplasm set.;citation_pages=56-56;citation_volume=12;citation_year=2012;citation_pmid=22545925;citation_doi=10.1186/1471-2229-12-56" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. M. Visscher;citation_author=C. S. Haley;citation_author=R. Thompson;citation_title=Marker-Assisted Introgression in Backcross Breeding Programs;citation_pages=1923-1932;citation_volume=144;citation_year=1996;citation_issue=4;citation_pmid=8978075" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. M. Visscher;citation_author=S. E. Medland;citation_author=M. A. R. Ferreira;citation_author=K. I. Morley;citation_author=G. Zhu;citation_title=Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings.;citation_pages=e41-e41;citation_volume=2;citation_year=2006;citation_issue=3;citation_pmid=16565746;citation_doi=10.1371/journal.pgen.0020041" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=P. M. Visscher;citation_title=Whole genome approaches to quantitative genetics.;citation_pages=351-358;citation_volume=136;citation_year=2009;citation_pmid=18668208;citation_doi=10.1007/s10709-008-9301-7" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Wang;citation_author=R. Bernardo;citation_title=Variance of Marker Estimates of Parental Contribution to F2 and BC1-Derived Inbreds;citation_pages=659-665;citation_volume=40;citation_year=2000;citation_issue=3;citation_doi=10.2135/cropsci2000.403659x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=J. A. Woolliams;citation_author=P. Berg;citation_author=B. S. Dagnachew;citation_author=T. H. E. Meuwissen;citation_title=Genetic contributions and their optimization.;citation_pages=89-99;citation_volume=132;citation_year=2015;citation_doi=10.1111/jbg.12148" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=X. Yu;citation_author=X. Li;citation_author=T. Guo;citation_author=C. Zhu;citation_author=Y. Wu;citation_title=Genomic prediction contributing to a promising global strategy to turbocharge gene banks.;citation_pages=16150;citation_volume=2;citation_year=2016;citation_doi=10.1038/nplants.2016.150" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J.-L. Jannink;citation_title=Using Quantitative Trait Loci Results to Discriminate Among Crosses on the Basis of Their Progeny Mean and Variance.;citation_volume=177;citation_year=2007;citation_pmid=17660556;citation_doi=10.1534/genetics.107.075358" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Predicting the usefulness of crosses in terms of expected genetic gain and genetic diversity is of interest to secure performance in the progeny and to maintain long-term genetic gain in plant breeding. A wide range of crossing schemes are possible including large biparental crosses, backcrosses, four-way crosses, and synthetic populations. In silico progeny simulations together with genome-based prediction of quantitative traits can be used to guide mating decisions. However, the large number of multi-parental combinations can hinder the use of simulations in practice. Analytical solutions have been proposed recently to predict the distribution of a quantitative trait in the progeny of biparental crosses using information of recombination frequency and linkage disequilibrium between loci. Here, we extend this approach to obtain the progeny distribution of more complex crosses including two to four parents. Considering agronomic traits and parental genome contribution as jointly multivariate normally distributed traits, the usefulness criterion parental contribution (UCPC) enables to (i) evaluate the expected genetic gain for agronomic traits, and at the same time (ii) evaluate parental genome contributions to the selected fraction of progeny. We validate and illustrate UCPC in the context of multiple allele introgression from a donor into one or several elite recipients in maize ( Zea mays L.). Recommendations regarding the interest of two-way, three-way, and backcrosses were derived depending on the donor performance. We believe that the computationally efficient UCPC approach can be useful for mate selection and allocation in many plant and animal breeding contexts." />
<meta name="og-title" property="og:title" content="Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1469" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Predicting the usefulness of crosses in terms of expected genetic gain and genetic diversity is of interest to secure performance in the progeny and to maintain long-term genetic gain in plant breeding. A wide range of crossing schemes are possible including large biparental crosses, backcrosses, four-way crosses, and synthetic populations. In silico progeny simulations together with genome-based prediction of quantitative traits can be used to guide mating decisions. However, the large number of multi-parental combinations can hinder the use of simulations in practice. Analytical solutions have been proposed recently to predict the distribution of a quantitative trait in the progeny of biparental crosses using information of recombination frequency and linkage disequilibrium between loci. Here, we extend this approach to obtain the progeny distribution of more complex crosses including two to four parents. Considering agronomic traits and parental genome contribution as jointly multivariate normally distributed traits, the usefulness criterion parental contribution (UCPC) enables to (i) evaluate the expected genetic gain for agronomic traits, and at the same time (ii) evaluate parental genome contributions to the selected fraction of progeny. We validate and illustrate UCPC in the context of multiple allele introgression from a donor into one or several elite recipients in maize ( Zea mays L.). Recommendations regarding the interest of two-way, three-way, and backcrosses were derived depending on the donor performance. We believe that the computationally efficient UCPC approach can be useful for mate selection and allocation in many plant and animal breeding contexts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ok5C-6S8QSQqjXoepxUAXxQCC_odufGAzbtKx9P190o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493109","apath":"\/ggg\/9\/5\/1469.atom","pisa":"ggg;9\/5\/1469","ac":{"\/ggg\/9\/5\/1469.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1469.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1469"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1469"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1469":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022u0yk7yIUBeXMjugG4AFpTSllBi1Cxz3AB1kXYQqBq2IYJ2Gae6StGIIke+Ysbe4kOApXybWngz\\\\\\\/TgtNfx+GgSjkyZYcfsw80T4gJHCdD3ySes9I736B+fymFYKMC\\\\\\\/KFx5aLojwg0iVrfV\\\\\\\/\\\\\\\/Oljlv22m6zpw\\\\\\\/7OomNhVwHTKsRy08jzRSFOS6Wnz4tcpW\\\\\\\/nRtslfggBifoJmRTeWLE3WIVw3Stq1A\\\\\\\/zHkkabTThfdYl+Y+nEyGyjpVcH4krP6xUk6IXQQSLGOgKl497fhtAbkZezlH3+b7XnAJigukPrT\\\\\\\/n+\\\\\\\/zQe52D2U\\\\\\\/suB0EqnWAYomgAUZqCBCYxDzva4Pog7FFhV9eQzprosC40RotoQkghF7JvA3aZDZSn485fZM+kBV6UAK54rbbRytxuXZNE5iDjBdnwYeuqozFcXwyTq4invCPThKwbVUDyls88uYIDfGqGpeuHdcxuqtFgxob6l1Wskc4RwhRb6d3f3KRNnckllov+DjQAazhxOoxn83r3zBPVxbHTFdjFHTwcf2dzHLHRZ2KAjN2lizwdBKEfNzYcAiLD33IRpQDghJw2+wI\\\\\\\/PbQgIQkML8LN+KCh7ogA1p5RmGsPfxczKVKTlfQhET0L2p\\\\\\\/zHPr9uTa85jxfEZaMJo04r0Y3Aj8g\\\\\\\/Z1Kxr7CFx4yCl\\\\\\\/SOyOsFx56UCrYOXV21FltImmzpG+u8ME2WcxPbV7zeofse9eDR3kdI3oCXY9Jlet7ykhp0F043N9HO+x5DNIn3oCA1wJlPXj3+NdfgC4IVWmSm5sVqzP0nMjf2oTxfh+flRaE42FCuXXwVCmxMyRnXU035\\\\\\\/ZBxqVwszzrW1gP6vk7ylsPl9+gQ8neN9H5LC8rEvMMtBP6kpUbReUkntNxwi5ygzFteVBcWjoQQLz2FsGxicadZRqeECKK70F77pqZIKs\\\\\\\/5Cw1FxXVV8zUSB6B4sblUUo1SO6ylGeYGMjU\\\\\\\/iSOCG4K0bQOFc2LWXpntghNfxIfbnaJggmjjC6qBYgea2KhZVM4wNQl7hTye5rF+B6cR1h70bxt3Ix5UH6XQKM303oCx\\\\\\\/Ve2yjDuWhUqz8yV4kgwbADyi7gmnQM2Pw+dmA\\\\\\\/G5Dfo36GFy\\\\\\\/PYUiw9YEIrzMQTRuWk4sMRCEkVpfi5y6Ryu9goL9YuEHhlHrga1qr6HMgcbWY1JJ2IjVBiEvVaw4yCZ1JN3AhL+m\\\\\\\/NtUr\\\\\\\/dXo8rr9DYhqHDizHfOHL5E54fbWg6gKLTQkD0G4YiF9QVrtZkVzQNdN4LV2qiy2k\\\\\\\/x1Azrnl8yCzJ3T0\\\u0022,\\\u0022iv\\\u0022:\\\u0022jECElGOWFzaN2YWZ4LxmLQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002216338c92279beec7ee27ebfad259039d025cc4731e1069c9ec5bf0c225b82cfe\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002249sATDEjsaiFMXdSQf4DX7Awzr3vwVkteyuz7uTumw3N+1L+Z7uc+Jh1MvrEHLdQdP5w3e9Sm+wFe8Q5rYPoIH4BnCCo3zHmj1RD2YM\\\\\\\/xYBi0Z55LL8SwlYa\\\\\\\/+CO4IgiUr8ue+liqtYM\\\\\\\/wHoGy+XuPksYKhIh2Z1TnoABiHlIQPHEZqZlilsHvwpbglzlltfuPFyUzq2qFE5Fh8pk7qArThAnExIjODd0FdLzIgCsKbrf7aHr6TZ4EDQUjeyDPLkM\\\\\\\/xC6r4OQOB7GGCISeQO0TQzfr2rjY8De9n4J4f8RDRrZbgCOrugCSpt3BRKR5jwVA4Z3PMB0KoCMydnyLUtS08G4xnDPtNf23MPGcQIjd+FAG4InXU0hePDCgxVz8aGBDc1dCQpA7J5VZdlrnpQw+wx40Bz4zjK6G\\\\\\\/ffCwWzkOibEAdsGi3YhtvuClXhy3AUNsmPul\\\\\\\/H0bL6NNKR4xQLRHbhKRzO81x\\\\\\\/6wKZ0dNCfX2f9cod3ZlaGFtNxpddfuNShr3WdibVgcbHN\\\\\\\/EXdWMyAnVm1V5c57vLQl4XU10skIE1dbMQ8m2DpMDZi+SNybTYBntjh\\\\\\\/c5LskmAy4AdfGKQfDx2WAMKHCBnxxTvzr+xGU1O78zhjfqiqEmCW+QxwlZPpc7VUXBiWf\\\\\\\/4OrPYkbHzqqwOwp1\\\\\\\/vUtqznt8ipwQDdYSQnf\\\\\\\/J1nyddIA\\\\\\\/DmJUE2oDM3ar2zbxgOOXLnNFWb\\\\\\\/54kPxnQKbGF\\\\\\\/8F7JtTmoHnAyIeGq0Fz0DbgaKv+hPg3iU\\\\\\\/OVss7U0nu36QDWuLeNiMw3lnpZ02USDQSM5LaW63hVGEMaoQmciEGB2BF\\\\\\\/D0ZFrLLqdlINZl66RK6UmAyuWWoQo+Cr9k6hTPsbC1qLSXX0ECmYxEfMvnBjDHMvjUxjf9RXZw4mDoHRvPem9h7a5eA6m18Aw2qdbqukh6IifF2oOOBSpyJg0TXu6stRQTWqUwPR6BKccxp9M0kv2KoRnMnunQ1K9ro8cw6BJ4UXlb3UpIcrRH91YfgvzxLa3RryyKT3yQy45mV2wZX6pf1qH7uPIqrhmqZ0mgfluyikKYWEt72rxqGVokIIQXDS5s51ynmg6GdNY6wie6cPsJ\\\\\\\/G4x7rIeay6g7KS+RSVuKQIqjra1oRvLB9ItiG0Pc1G50eEjLCcRxPhC9golZ+b2qNqkqeZeT2TYLF9VkuEgsMtFTQuQAZdDe\\\\\\\/lPOiG8vKbxeW8rt7HbQXCbzJ5HosLxQkbbIlr9QVmsSBLzPUxBMB2pFnbntZHXUr4I2kudea00D\\\\\\\/C91GxRAVyvqOCyVM87z44uIzzLNmEFMrsVD+nK6fApAPc2hZle64BcHQuxOy\\\\\\\/Q4MG+54UGXKazsv2Kql\\\\\\\/615Hu4b8j9cAdb6xqelou2qFMVjNKZYSGsLxDWf0GhsJxYf94GiuqmwVuZq2z1yNYiT0VyxjZUaHagakRGQHio7D\\\\\\\/zGf1a7sbpXXlFsaWZB3DwXFt0CG\\\\\\\/7FbNIR4IGuqWUoZ\\\\\\\/9C9Jw1Iah7+5GCj4DOUrKGbcjGlTVoAfCG17rez5NqtrT4t5VYt113+3p6lNc9zQCvKy6I9ZhlH7K3tKJISoEwnl35XdA6dpzdcyoXUhw5O8YrY7KvdpvBq\\\\\\\/hTkrRCCdJ1I\\\\\\\/p8q0BAHHe3LWC6LeUs3wrZBYL5mJNACnwtpYGcGgoHkyuVGP\\\\\\\/1kD7+eRhpjidDysnS5qPeP7hj+XPZqwVNLyo80yn2LFkvXZ2YZ7umwxh6hMnlvz+oektBWPLoldWZWD+EW7fYM3EonZFfr\\\\\\\/ESCvuxKWRwl+piy5EmEMXnBUX6raSStyTLijFtDJrAi8rr0U+m+J9Yt2fm4\\\\\\\/IpDeQHHlZl34RH3p0fQLiH+kfNNfiiqiVp7r+DzMdGG6jp7BS5NmnUE8SLt\\\\\\\/Xf8hZyR9MJvVAGYV5Ne+\\\\\\\/2b6YJ\\\\\\\/enwyo5wf3\\\\\\\/o6HV5\\\\\\\/n+hnvW0LxG0kOZoqjSHse+tmptYUTy7ct7kd5niUCiU+jevs3U\\\\\\\/JLhzL6sbnGCQ1UNvEj8pMee1pjwwTyF88ey47JFCmvTo01jhE5UIQrUFHZz9LTqHFQVKEDQ6m3AAuTMcOLZekeKi\\\\\\\/MTk8PNwYtJTF8zF6xxNceG68U4KtNw3YVxauuzRp4FM67JkE7dEnMNbgA8skxQmTTcL13JR\\\\\\\/GJzByVw\\\\\\\/8ndGVLXcDCwC4PIbRtjcKu+YuIv+fuAxwMOmxvp9\\\\\\\/a\\\\\\\/xvuI8IR2JAaDjA4PJgzG0tzDgIZmWB+IUC\\\\\\\/R+3PXHk0g3coisbiADtpyuJpGGcq6EhxetF81NlPyibHtMrtx9lcLiwSvJGaJJz010VcVGcx6zqdeN\\\\\\\/mmaflHh9cJ41usemFSIXR5Say4cquB8cI1ImMJRWgNeh89cc0Hjauvg1FKBwSHEYJyu\\\\\\\/Z4kbhQqjMAt9Yi9N7eC57H1c+LscM\\\\\\\/j4owYpyEKSkiKsWVZZHeB529hLA94BgMaWQfIn47+If0nOKeKU7Jjyjdg77O53YMqOi07mS4R5OU9VdIep5EOPN5jwtgG+1z09yrGuMoAFohUQc1Fwokv1X0925Ggrmz8UbAW1E0evBpbxiQm2li0REJc4rUmTXNeHlC6ryhDDsVPqqsLHyeHPnDUZFYVugTycfYZxTkNPDn08Sqq2M2OeAzgb73sYlCXRN3QLmwXiX1eWKGimfQv6GxkFuALoV7UtKc2cVSIhT0WugfZ51XcTrPfY3TLy7J8lqSF2Pcmq\\\\\\\/5xaoxR4ZbeOfjy\\\\\\\/NauOJXIxcH7gc4WyDShVEcLshs45EE+8Oro6KsnRRITHDARHeMwEjwOo8XJCvnSsqNJIfvO\\\\\\\/aLap3I7UOSTFgICZ0DIGv3eQFZguhuZq1+BF\\\\\\\/\\\\\\\/o6jOzQ1P\\\\\\\/7LXAY6vZoeYRLEAiF0aKbF6AzhXlTn5rb5FqQhxzi9EFjbEpzZ\\\\\\\/CXfwcVgmB5x7sLMjRE8zFMa2LkWEjLhEMUBuCXuRGucXNaX7S70Jg6mq9mwIEw1rxz7JpSfrC9\\\\\\\/ON8ZLnp1C7f1lR46TC7rVsH02bujJhz8DFjCJzNni\\\\\\\/Y2XeuE8\\\\\\\/GIDvTXJDnAQEGt3Njfvx7YVL9mzfX7doncv3qClP7vAopcQHdvdhzG3shS0pUYAQV6MG+8KMq4x6yXYGYvn9XXsFAA6L+1wg2VaAO72yIWtN\\\\\\\/HRFXQs5XGrkdNbwLd4npboUUM74XKMoepKtcuwRlfeV6PnGRB1lLbR+7NGgtKwga0afG8AzwjDvTFtnYhtDlASNhOPP1j78yf+he3VusMR6pl0zHsmWEsgPWs3yQumCRzOKFad6YyUvaqO5ID2S7uP6gQoBUSnJd3GLbBfVdySsL12lhIIVz2uAOnbP6n+gU0tktbjMVicu45UYK1ifkvmzzUmqddW8U+qXt2g5e2EjdFADZ2FHCIUIxsBoY2wX5rEHAGoxKx4+Nty7Qn7meTxEzWDFYzS\\\\\\\/yH30ErClSnQ85ZGnzmMC3z9r9AAPLk5gMDb6hRjAMkCZFwq9STY7ZW63dRfxWtqlhIF6W43wZy2Iu0t95YCAjIzsijY05CgZ0h4g4hhi6SGcLdedMdZ4WWtO8+2CzthVmC3wzvjUaLHdrIGLELVtS8xVOB3OzutrDJW6lxOvdTTccelYSqJ8fMmX6IsOT4Zp5+cbupeprK3OAtvBfwI9WFoKZhFSDTrJTmx0qCL4\\\\\\\/TTGWQnKT+aW0TVBM3f86ZZRPnhOypHidnjC\\\\\\\/2nAeSuVkhBuRWencNTCs07S8azjsdnyvo9mdTpx9+1DzXtrOEwmTzGgC1GQpQ+jTejHFCT5H+DF5farReJ44AHxOo9\\\\\\\/wZXmn+yUydgjOD4E1q9gu3hkyDwMvlr8CBNgIt6n8ovu2OnigVCUCPWzHGGUhqwYSVrz+\\\\\\\/+sf6JDDi4dGKLOmfI9AoVICHMxLrs8+myZUbZ\\\\\\\/S0Q1uWSq8M+6e\\\\\\\/Z5gU0JhrHFbXDVS9B+97AqTpPr6Se8CMmtfmuqDFNQNRcZ1pbCR9g2aKOVycS9XuG6EJ+0dPI3OyUo6PRY86hYUiCFx01NX6BZvXd5TGwD7as+SZ0GGQDcHtAI5Mkef4VIHDvzNi3AYkvZ9i3Izm7JbafPZNwruyem79hI+iGotdQ5LZgs8\\\\\\\/sBPvnX5BkZkL8qvXVBgX9I1fQdBSc5hcKZ4ja0Wxt8OrkPV6zrk4vpt6oIXuVCecowYpQcJAbFvKL1YVz8Ia97t1Wcip5WUniFQ9G14jKfQY6UuaqOiKdho6d5JORLFcxGE3wXOqpjfEBosGv2V6u3oWXiJJCzoQ2OOr9hgY4WG2hWkqjSk8a\\\\\\\/0c5Gc5E+aO00Zd7eoax2PiDv+1iQFHe7TUi\\\\\\\/92ly9UrcLdc8iBmtxRKiRy43SjMAYVss6LbWgtTFzvzla8TEGtmh4g3Ie02u85e4CN90\\\\\\\/knSWg9mS14A\\\\\\\/Ksy\\\\\\\/JjD3cPc+sK8qetFBzyzkC2SZaiXXn32KSld97PGCJcc4B6GH\\\\\\\/UStPCZwtUPaxb8uKO8oFVn83WI7Y9w+bc3x+INozkuUqB478pH1LMdsrWxuIUdtz2wSfMQMLqXWrPh2ti4ywMQCO4M8Kt+R3dDmk4AZzcoLVyx3TEaPRZEEsoJTxeDoWIdiy1gqCOO3dxaXlMaj9jCZyr9oIICcHeVxjvChTtg0kBiyqu4tqsoL2xDTxD9JTRYdRtqR2RaMl\\\\\\\/cQ9ffjxp6DntiAzh7VmufYDHTuPJnFBSlwi+HdYk8dl0bK4DdYlxE5DOS50vfbV1+rssggz53jaPmhXdgJiMN0+dgQM8K5kEru9hd\\\\\\\/So1RI3oRTVXwhIB5pTWAvEILZ\\\\\\\/y3FkO0R7UZ6hPQBMjZF6mbg2EnYiSEEgKCkXEB41oRsgRx9nvgKa9rReKt8f2fpwdalN1\\\\\\\/crgqaD3PNqam5En1x2bwewVHPqrgac4j4RIDUWaZrmYZbgwgBDvRjPtcsL\\\\\\\/FC2fy1QKLBrW97Enw0rVTezfQI5frTBixITKOdJjKPQShOC+WD0cdN4xke7snGHalk55ChWXs0pWys5unT+qp3qRvTZIZ3c4WT47aCrvJQNGsfuk0R+zRBuW3wDmWECwr8hK8KVEs9CGBEuajo\\\\\\\/tENvZnYj87dkpLfA4WCP6xvTV6kbChCcnoN8PwKyV\\\\\\\/nly0VvpI5Bv2YvEtpIqyunhKsAqbRHjGSGRTvSkHstm7ppfyJe32h37+7uR9tCj19\\\\\\\/J0puhQiKZvBvjOr9XgAE9GISEPdKbIhcYqFYtGXAEXreZCL3gioPISMSWjVf10uA3HjgYp4KSWklfaU+uRXFcfVPMFDBspNc8PQgsuXjbF9\\\\\\\/esBfBuOfsT2R2vbSmtCZSS+RVEH+wd39vhFZMzvd9hDwElUUm2Vtm65y\\\\\\\/v1XijyW5t92pl6neo7gVrBObOhefwLFbrxU25k9ah5+6piQigdLiv\\\\\\\/aXRbY841uLPCeaaY9W\\\\\\\/T1rY7DvrkddCJLHIsvoOP5dMzYZjtQyQLmdIkFkvDaw3FnbP6SIrRJejomhxuVokdylVHw9HEgjOaLozd42Vzl2kINZe4VxDmIabfdxmgTKqAAfMjUNmquuopgP3O3FD8m3R+Fd0VBr25vklM2888wopRDpbfAhmbdnwjg+z41mZkvu2xZuA8S6L1FNoLSwyjEfj+5S5yKsbJxcW\\\\\\\/EUInG\\\\\\\/q4gMvg5YLdqI8r7AFkBK6dGoR6eHsEQ+XgvtHmZ8DC\\\\\\\/1QAvMei7Z0alqKHmQB8tfjA7e9y4I1tyJnfbwxC0\\\\\\\/i1tGI3wKYzRBK5LsQbMpQFl5WO1Jyc6RKUpoWQJGad1kycQprRHf9kqvAM7h7qWTr5gxWw9S39XO\\\\\\\/rYseTNrwurm8B5TV1ro57adykfDas83d5hfY4HAXMawXntjAn9OcpK72O0Sz4OR5MuFr4t9vgohtvRZJnwZgdB26oVJpxMPIYKyp4iJCXLxTd24bWYCIIzXVhz1dSppTFjKsNbckQ9OG2e6FIX0XvLaBXivYMPgZR023TlJGHn5DM+54R9WBhxnjA83yJ6OkyttYp+MB\\\\\\\/TfEOxvFsXjoi6lXY0dHLNdVtDaYE\\\\\\\/pnroR0XJMyXm1lQprGnMCXIG9vGLuysI2gaIQkC2WJp+djO4\\\\\\\/q0by0gKE5vEQiG2O4absZVjwl2V+K6elgU63QPSt2hECLR6alMZphaXBl4s8+0KmLI0x9EBLKJWrLYbbqQN7Yccgm0XtUbU9bm6zPxP13JQqJtrYjQlC63OrCaZtHKbI0j0r59sj6tgdGLHU0ufknCh1vvxzkH4FRFbgBnRkBVtvCIzOQEFZEsJODAgudEH1FVYxCuBe86yyBrKsqs7e8jGDdOjecGAesf2h4a9hlTL+wPXRtuI7BVqWZw7+JAhMSXtrGSp1aTk3IYWjptXUQrVbM\\\\\\\/FIISj4yyxGpAA8OtHZuhx9jiucgrdDFtotcLWcAUM9olLM83NPa9RS1xF+o9URMRR0AKBfIk7sgpYmweJt\\\\\\\/7ERmNSYdHGc0HMtf1eSG8Ab+5i4nuNfK0\\\\\\\/YN1LKYA2mDEQ1WoM2p2F+P6V3ggA1eI8qs664LBhEI11SSLxPWSPrd8dBpyGTIM6aZXIdahREgM9GrVptG9kvJSOppy615ev7eSIyYnwfZsjPiS\\\\\\\/dxF+W9rgFnqRkW+3Mw5\\\\\\\/yXUczwX6H80cTgojBTpcxFPM2dKT3yae52eAtvM0FFFiW4l8yZD1zoWh03uqkiyUUop3w280wBnHHa8HVR2413b1nRMhlZ5chYFvHfc+34sf6uUF5Qbw34tzA6akBpn3RYkhngpKeItq4srNE7vAYPA3+zuJDPuWSlhvagpNyKH5ZhHb\\\\\\\/1Ls5WKSG7H\\\\\\\/QjhzKXjKGu8S6boGHWovwQ+BW3acW1toDg3Oxpp0sokSdn4PAszPFG8ttwuuMg50vVaiTu+hCmFpyYnrYaZGaHul1rqQCi8cNuI6nXVJRwLAikHvDEido4s+7QEAJ2Ky8XvUR9q5idAqJ+vt9VtSCkl8VI3MUoTj+kcSPn0DmDdeg0EvIQZr2fyAU0zCWYBYr0zhVT76htJw+SMobmNtu3\\\\\\\/OA7axGYqTR1irE1Maz1opCDkLOvsHozanef5jVq1yBSzHdAOHGN8gSIAdGmDJZTbH4OFfJSs2CyLbJEnoGIBJdcbPth7m3khOHGKaLFFprkWLQJqU5VDJjeHyO6y7mXx1Yjt1ickk9xBCZL6BxhUuzN\\\\\\\/uim934ML3YxBqzOawSblkXHKumUDdZaapDo3zT+1oB3l0octdac0s4IeNtBWeGPnNGRsvzPblQVQ1IuqOIkjX4Y5HGWz2snNoH8VcScdwQd5qVa6DVuhy4nlRQX3g4jm6hojEb7b3t1C3kbxLPfpVyOhzLWQIBPQvIPm9pKJdQXr3lJsqSMHDSijpOO2s1Y5XzB\\\\\\\/qx5ngb1vUf9J1YKQl0sfR3hGwRuuY1B6gMfN0a2UDLtRfNSgH3wNJwfcg3BsLZakncTFErJZ6DXAZzUjQq\\\\\\\/n962xu9V5Nl2niVKfEAHWSi4hGuDKHk43hgPTxcorcGUyF0Bvqqfbkpd9hs9AijTIxnmRXFdYQTBRLkWhctSmWkqMxY8y6+nxTDImBNtBea5Tvz6g6ngabnVvERqEixGyRyR9i8gKa4eswrVrgHkN3+6wcRzxiWST93oQxl4JdEA64l\\\\\\\/Etb19CWJsgrf4rCdbz8lKb3+l5EwqdhbtuE6h7i3xLl2MAbtyzWgDla7y81ScGxysTG2SCmcZSEAX8A+ZgRCVjpSzFLK9aLLtHy5tn5UiV+7L7r3eraLRNJr2HrSgBykXQlDvNTmYPqDRgXXZTPs8HwX40uj+tRGXgPHdnctAQd2RQlAjPkLtoP9ki7s\\\\\\\/nTPNoNighvkctH9fQbsXh7tpmzb9d8UebjBGb7gmiHG7QToxIKTmJoUWN0Ce5OoN53p1+gxc9WglvJVxCFwuRyT9lla0ny3KsVVsvbMUZCFn\\\\\\\/twohyVqAv2vpmKCYSoLZ8HLZr9jK0m4ki40V0bglS3pXdFuH+DKQFgodC24LAY43\\\\\\\/ZyAIopAmuSS+VRWYkuplnPFDvEIJfSSM7ffrPtP1B6HFNBjPLR2NEzZMjAxyLAsjbzM1W0SSUJAIbFl4MgwsW\\\\\\\/QRTET\\\\\\\/gLy7KnogNFC8mucLfsDLFk+iu65rsyzLRws2UX\\\\\\\/HrhN\\\\\\\/XFXom1NUUP6W8ewMbxh+5wb44y6Q1t+wKFA96YE8qx8U085uXRNrhRouMbbI4TjB+OC+IEX8ncflbFhgj+rgxUpqD2bwpwrdI91T67lKEnHpAMADkAPG+kUh94z838xMbqPGiJ3m+CG0\\\\\\\/wQEmGpAOS+krhd5Wcg0cNkw2cLmder\\\\\\\/uoNVo37zX7q31aAlx91bJ5G8homuBreVitpddp8F+Fc4\\\\\\\/1Rb1vaE45GyZnqBqaAjE5VrmsudKUOfCnjb5gfTgyoexo+Iuv2ZXk62jwhj2quorW2VsPAaOxzjasdwk1eLsU8\\\\\\\/NNKntTd7Zm1hIFwcyzeAMmDlMQ3LBL4IJexxSzuSL5wKarqGCMSCxSDYbi4IR4LpEbhM7EzMyNipcV6SMB1ZOKOyKH\\\\\\\/DihydTTLUBDi2BtmewiamGiI3ALrUTBLN31Yhh0XhtRhyk5iHT5S1Q9jL\\\\\\\/nRwyUxBzB4NB8UGgNiGPbK8reGbp9eAM+lfHVpVq156mFtSZs5x67eVeHs2p0jk2QUGgEBtdJHyyc39ddG2fjxh3JFUGdlZdQ\\\\\\\/dnoEwy4D9RM7l7VEFypkdJveXSvGPVac3mUu0rlQKRmMh27D62gtII0KPkMyYNAzwxuPOuum6OR8EEFvhiluFCLgpldV30UNDQT4RNwP9ANIrPq\\\\\\\/GTZ3REVqGsN17IV\\\\\\\/LtNI8fVFfplaDmsDSbKRVOQVpXAvD4K7DEhHKuMzVRL2Mk81caaSrKR\\\\\\\/bnTVtb4\\\\\\\/TmO13w6Heri0lPebcd3Ws0bPGza601ox+YjDc6PXYcYjUsajkgJxdHSgsE+4SHLan2UYT2lnifuqGpQExCFIu0msyW2rkgaMp+f\\\\\\\/xCjFKubqlzkPj7L8wsglf21e4gF98Bgo4drTu0RPzcd3DKX5KQw60Jb3R29gAH5lhcpc4wxVTwKxf1kffdhRY2Rm4P+uXH0F3YPSU2p6Kee18kT6mc40w+FhkmbueNK8RF\\\\\\\/7RhyT3hwf0XWe46mk5EqLQ7T5Xb1NbVNVsM7ny92lqPMC1jGC+R3KjnCk5TFyXjluqsY7d236mw6R1qWRE3xe+jDNycJjzFhIRB6kmlP6lf29Iu5dpvq5TSFtuC2+nHRe2ehYMulYWguMRh7nd5mcJDhCyyLg7J5KinejbazoQ+PpIjOMatd0xs2jVA4fzCNgcvJdqW97iDW1FUU78rd3BJ8diteOxQYzBO7C7X+Oz\\\\\\\/P4BMsBGO0HsfB2KuDK3gjQpX98F\\\\\\\/7YpyeHB\\\\\\\/1NstxT\\\\\\\/9yL6L8pvS+aMjbCXbBh6TXCRMnDIUqBtQqcZy5qZV63xnWH\\\\\\\/QCOuz8pwuLVFrqIR\\\\\\\/13rTvqajvaK4DU47GwdhylSDnpT5MAysfPfxeeK8gvE3948plLuUWJR8T6AQPmGEMlwOkd2SYUOfmGCADc8hbFC\\\\\\\/sKffjqMUOdNs6VUM5QWhwmxPZiceuLaqXq5L0X7lMbKgWToFA+Xe7aNlMjvBU+f5iVrcQkfQzbViYIHYk1VOlMn9AAygs6Mb1\\\\\\\/raYLY4f+9jI7SATgNyfx4mBU5iQNUdRCmjFFdRAmIMvlUMwhJ78Q4FW6KrzfTIJWFcIjj7wo4tctai7kPdlGLIMTmjQw0KcFx9RIejMS6\\\\\\\/ezvQb6IR1sVa7dfwRFANtuNHCbCPCtfLh8CkiCRtAWiDwwNFdueOSO3LsVtrjNgEEoRfi7ecV6Uv6HY4zdLO1xCiRRcw0QJ87vcehG4Bvagw6fEUhzSTcBeiE4e3v\\\\\\\/f+DY7Y9EFxOOS6tkCgqhwO\\\\\\\/yVPV3cu004juZFByQEcek3mgJDpgUV9M7ghQmQQJ8IwIKzOLJ60sf9m5qHZcpIXKOnFRx9mr0g2YDV\\\\\\\/r0qbb5ZEz5kDsuRitfJaugJZmXx1\\\\\\\/G97HHmtvNVLelLX642kRMZDYbRh8hX2VTQU\\\\\\\/s3yGFGK5w\\\\\\\/ZK1tLczcH9pB\\\\\\\/Vj4exm2a3aHwAuvBF1IBDGGJvb2AP61Eh8dBK2X1MJNonwWYTP0a6g9cSdLx9MUoRgKMs83PlbsDmdU5vdx8Fg1D2TBOMnVTyWqS48cv7bX6ssu8PBZV2p5VOQsgY4XhO8l5hsJPjbDShhO9srvxKm1nL+Ze5W3\\\\\\\/W\\\\\\\/uBeX3ecgTPAHK3HvpBLoy1gn7ny7wMLJGToQ0T\\\\\\\/rzlKLEeOQWX+qlqgQLqYcTRtvonQ\\\\\\\/Xbx4NB1AZnB0RgWVoAciid37ohOzRI0mwTlf6xE7h6kNVOFiHlO0Y7PA7+MtPAcNEsjclSyxX0mdfuz5MGEPaoNdXy49PqBio7qVlLDc7VDCBkn49bADKrBK9K4xEYlMwi\\\\\\\/yW3aMGk+Hd6CNeB5dP+y1negXHw0qT6JUc9dVMriANlJcDpVYSzKaKDNE+nvI9L\\\\\\\/EX8RIrxPAMvNPRH+T9JDAvKwKDr05EkrstCW2smK\\\\\\\/phJVHVf3mmzruN2qUoIaiEj0r7W8bJ0QEkQWbyebmObi67g8TzQhIuqLEkpalvI9wN9LK0Uvrmsvc0GTLsXlaXZAvmGoX+fUukVnnpFaKkmF1YjMXPk9rx7xUtdq0PE3p7k0pQ\\\\\\\/xXYXXyNnv1kVO8SGRIei+CMse8g0mCMHJ2RuI0QrqWBJMHEHjqxiUKxCb7oxKuT5tVA+2Xyo13Lg2+nkq0p9YZfle9D27Tr97B2JUT\\\\\\\/GS5oMnUWGWzAAeZgy6INDO0bYgZTY0VRsBegq8nb7kSuOrZVbR6gRQL1kYbeohO1mzm0STT0FeNIjw868VN1yL6YJIIWYuEorhXjZA7S9egVKiFPDGC8K94i5bGm8qgjNLpq3BVyTGMliHFJ4MhpKOCE2ppi+Ktqv\\\\\\\/SRk14l9ysVx34AmA9NolsOlILhAa0si0Hi7jRonbo2hB7RfOkE\\\\\\\/hg5X38TXduBwNAKUqlKkKhfEbfRdFFo6AM98\\\\\\\/3KUlfUoAvLGnrAIzNjABiWtVEUbCqGHlJegks1+mCh\\\\\\\/6aSuxxUeevXXQzpjgMwZNu02Oy4lF51PTvuk7To1fKBvND1vorNOAI94dQencP6EIRhn75WG0D8aS9NiKoDQZtYSAcGePBT3nKEfMX\\\\\\\/noGUAPWga31hgidjySGtzq7C9PSmdTZd8xYEuj+14cCmaAbofWw32kG8ziSDvX2PsoZm2DFSGfrH+ww8Qnd3s1prb\\\\\\\/rmucD3lxlB3Zjvj9\\\\\\\/oQmu5WfDJuMzZXr9NOTIZ\\\\\\\/j\\\\\\\/kaLYDl2UITFKoxO8e5O1dmBLeFmHZXo4rx97k5deI4cSnBuhZjbldOAfkPPb6H2l+OrfKC01bh6PyhAtXFrgendbcEIDRPqkpOgyCS5v6y86zKxf0xtkRIiIRgqBdxAY72XaTC+YfdA+srS7AtcqmWBHTSYz8aYBEMH9j\\\\\\\/KeC2FZazda1aziITXAtsGAjB4bUFtYT+G+gA8qjqo+NUXYidSlMUxHH0kleWoSPdugHzcslUkamhAFKsW\\\\\\\/3eBWbGJgk\\\\\\\/MtzZkZjdAQEx8X7pIGvHK+77y5fwm2RphMKF06iotJ9nm3fjqkY\\\\\\\/mcSSe+i6Mn6DJkthCZePmOSR61IeaBbsBbZqvwHCtqqg3L6ogW+Ixx\\\\\\\/zzVK6hD+dLIib61cCqdCNeO6h6gj+QCiOE8r4GMHhw38TifTVKs6YSYdYRWUKkbgWpcMXJKarrIUHPzrJReg0X6OjFve5c4tCzw+lyRZ+u742OfBeFmefmB7GsWtOfXo39M2nM8uzLxM8q2Bm3Cy7WQUwlxMwJDlPahmt5PFi0UJYYRHcfqdIzjtksQ8eeZQ64aFw\\\\\\\/v9BiqU1AueoeA4SwWgcSdQHLuY4PVtmN387VMyfwA2GcoNlA+EjJQQzUm8lEGJx1B7QvfTmDi\\\\\\\/gF7AbIfvhsCE\\\\\\\/Eqi2WkfoBaMnxFAn8oX\\\\\\\/bbxK24+DpzmNiOeAXcSVb1OAlnWBDEvGMtALwWwKE\\\\\\\/plZnd5tfFDVXYWvlH\\\\\\\/LDNfc9HoxjMMhGGjLRGrsS1Umfm5n7lzBcQgMKNiAO554LNdJNMAcsL41bMppWwaYj2kjnWWXF0h92YVaszUAakTilJLTrUude2hOVF4YWs0bDICPrC9s2nUfEAfNYWoFH7erLloea+aqGcd4ATnjlp2sNJm0wyGNAFcA7n3qsDz9JkxxuEKfPmcCLHZik6g53ZzzvZb81mB+4KkKGTIapt7pxxTxae\\\\\\\/0R5igOlFdiTPgudu4M0CmoORXNrztfeHPhOy1uf\\\\\\\/dyAcuFwJwPRYSVZe7RphaHghr5JpHWJK4oWYXzIxrXElRRzdz9299L6ZE3NJ2UZFB\\\\\\\/VsqzfPKmHxURqHhlAhrMNaa5Xddppboc5cqoPU2+rT0h3VsZc2ujNo4UuYo\\\\\\\/mg+LEluiN2y\\\\\\\/qxKAYF6xG5SHteDIz1qN1PZcpjvkLMotgKNzR7lz40AfREAas5EADgq8ShOeFtlsIvyxySJzRpSLc3mCI7e1lMrBUvvcPV4SbljljcqrKZuSgSt1dW3U8NQQkg7KnQ9m3IIUn8J9F2XoNRQ3OD8t8z3LBdvTwbNWt2RY1yYjiA\\\\\\\/ooL5PLYTi1roUprAFd8l0m3YmSsJ8xKFEuqZyCGcaH06tltd1GI2Z76vgvXjS6bFHPXpWAK4Aplwlqslu82oUoZV+GzMT+UMruQuonHNNXUf9SQ\\\\\\\/fm\\\\\\\/XHkaJ5MhjEYO4xoIVKS4ka3Y6PgmnmnSrH4oZYQx4zHUnoweUjwLzYPWTxAWHv8kxAeSRxYxFJdtChBfR1JQ91DU78kckgiY9ROUCjWH2fhdSmhxtn5Hrq88pAqiU5pMFhsxm5qC1nSP9bu0vzEKtOUStDuDXdhfN6HvyOJC1\\\\\\\/MeOZ2P28Ell40WRvxrRYr1YPKuEkebcyFYvMcS8a7VFUmG12aUoewXyfRFUebITh+n0d3vjy4JPhHd\\\\\\\/vIAZWMytzUs1uqoYMnbiOZ8jzYevSv2BvQmu39xWAlKP+IyRtuLIuOaEDRYxSDHrSvTYWw3r51QEWzN24ZUbeLQBG7j1TY9a2ypRNm2VWj7GpJRMn2azSay1VXT1I6ypKl9eukCRtBsv7uYWrqbVoqamOIJS5YwfvMJZbeJICNA6ZU9LhOI0gtWbrw1ZiELzaOgitnAHbhMZEG8zdThChcE7GMUv\\\\\\\/Ggx39dU7UUziqNJtWdqBE1Tk5OCsUf1yZdS4VTeCkjMK5N6LKYa35BU9fSPGUxGFLzRxaeOCJfAtk869in+3bdaQw9Eel8JXjIN1MpLwMqQZnAkOtZAu+7dTv775kz0sqN4twk5y+Pe6GZ8FgsOLtElKreSpGLXv6fWIO2ZRGjQNI\\\\\\\/lMvYiKC2LNIIwnFo1QAqUIFyZ3j6tGZNvHvYrfkAHTW3hILJFTpFrVkN8yOP6S7c+sMcBZqiuOOFaykNLSJBxOnJJtfVJtunROew0uIrJAr6\\\\\\\/VgqtDm9gv+FK\\\\\\\/tvvmcjmRscAGyoXn49xM5jGLG7FRpH7DjlA9DCGHODWjJaCXgjt5Dppbt2+VI5mO3SOR4U9eFt+dUoKP1EEc0dfsxqSVPVZs1okClhzB\\\\\\\/nRx2yTana3FvyNkRbLVH+DBze5vE5vfCxgI3IUXBMGvClmc1oFwW+zpIHxl37FqFGSAG518iE58wwsAoH+cwaU+zR\\\\\\\/c8ytS8JkVQgQTVsDhzxgt2hpaE6nVf5r2TvYOFsWJ5Si0URfBwP13mKd+Wq77gSRweHEMnvHI8cyEguncWRxspOZ8+bLNQxSfWMZ2e7teTB11eaPrnhFbJLTz4\\\\\\\/rjRx63hkIL7vCU6cKrrHAwuol6n3Ac8YfGaXY53Xa1fiPKAafKnA3W1P4ArABUbkX5+BWDsN\\\\\\\/V7P8pbXeldRX+DTgw44tq+wPvN3MAUPQ0DNpSIksPUn4HLujQy4kXOh6gzxueZyWFZJzYMSgxOUg+9AjVbRPzPuA+h74IUd0YTN+aaGdV2D+nGPB5nBcwbue73uNzfSgVPydT9X1k689UV9IweR1nWoBj5+8miHz1yLFd7qXtNurFvg7b908hFC9pp53KlBH0zEiRKxtxvl1gcB7hfKB5qubIBGcuzCiC4liX18kmBNye1EeGp7QvHxX7sUf9T4wZLPi5Swd8Nfo7GdKqYZ2LbmhempwHiL11T\\\\\\\/QkQ\\\\\\\/2H2UmVzkRwvEIFmP92fHZok4mjJaawzVsmTtv08kWFO3cWYSWYzU1TV3a3\\\\\\\/rcDy5Iebeu9Bqe0o0X0SSVDRt\\\\\\\/yqD2AX5606jdOjb\\\\\\\/N59Zl88QzmXZUzwyMrJzKmCrhQHmPNVG1RLhJyK3Od9hbeCR9Pd64vTE3V\\\\\\\/ho4aYeS+9KT7M1MVgoO\\\\\\\/X2WbePy6MXIxDc5pxx9QkbBcj7ujX\\\\\\\/20I12MJcafLU922b9qFY3ZJa7mjfvnGrOt2QIEKFhMwYHrJG+thISikSa3zLnJ6Sd33s2SDJYuzy473KyWtfSaLY4ZiktGszzOsOwPmL2HdonUSILbwB14nNC4Z+dunyExAZWczsYF6CRHTq5Cryr3doPEhOp2UTaqfIJyn2+i2PYiyGOtfaVLZXgn8lEv4uzrhvjoAD\\\\\\\/DWD03tPZx64XhSJ2cRaJpYF3hjHJMJPkouatI20dUwMErBrnwkdMxnOlM7m95ilePaN5dGfYy0vrOZruF+J063lPwOIu0CjFwFICd9qs\\\\\\\/pL8phqJMYqDbEeP0c\\\\\\\/hInw7UIvrZEQD2lx80\\\\\\\/GUnFiL+b3XPU9DRmxsX9r7x1jCYscKOVqNDvV9FD+4MsqblfxGjfV\\\\\\\/OkzyC6JcVRqRETnN0cwVcTOpphKg6nqKVQMXQ3BoyUiJpQ\\\\\\\/HvbxcKPNEUFM7r9htjc3P+v8hibQU3fhn8k5n1477SopUal9L45V48Y7M9T1iG+n1CxJLlMxHtP8s9WUwwTmdj7JSrEJyaGCmbzImhKU1plVxWv6l1EHMj4RSt5mwiPpaHfLc4x18C3l9j\\\\\\\/fhTZRs602XzRxfzYVZP4y\\\\\\\/fUXztP1rL7oEOa7NU+HVX4BJM4kLR7+9JLxfcrQp69u47\\\\\\\/lzYx7hqBBAdngH7Km84cFS1g1V6PPeTHLuNhQs4Yb3G0tVu8N\\\\\\\/uzGcL5oKf7L+bIcyxehKG4ITVtb+CZvOst0I\\\\\\\/d9JjO4Ea27rVo6NDGlCNptTpUfWiit1ZfSvy+T+FrB5GCOqeWe6d4T3+F\\\\\\\/cm0FYsA7lcuVipcidjP2Vh5kANhoH7ICrrHxXP1CiccxSePuWKxBIErgoqfbMtEdqhA7VWc3SVdIJM+lr1QMbuw9eoO4awORqmy23SlurfadqnNMvwB+7LWWUC\\\\\\\/qLzWgCg48F4G5lbsrYrncvT7DjQUaXm8VNxPp9ed7OgRftwS2N02S4yZEZXX4pT2yGgjfqBCKKPxapqssCVBkD4JwbqVonMFhM4LucqHki7PbleL+rhxHXNyuRhSH7dARE7QOnO49DuRcCJd5KSQRltJ+4LbjRWQbLPfBYi5ys7v6LwiZjKdIEtPwsVxaRU9hZJCNfBFf7zanp\\\\\\\/bT5G8MNplRnMLBNyM9ySPNdNtErLUxw88Q+7rYn6o8YMdAW1F8g0WlYjG\\\\\\\/5BJy28KwfVhRKZDEx9oxppKW7myKKJFGLYnXx+7T0YeVAUizpjM3m6u\\\\\\\/yGNWRZlDoaiRRtH96qQ4J6mp\\\\\\\/4L5OzLCR7Q5o6vJQnFkNgYEhTH9I5SvW+XG9Gs7+MTl2be6ZuVnEvUBFtFDxuBk5BROWF5cXirOp5v70jt2kE3wHAoF3R\\\\\\\/2bawUFVIUDKABYW1bmAnLnTQKVHuQaE\\\\\\\/fklLhJN40mc8wZsyk4CzJudACDjjW0Hf5ibaaDVxeJoQ2NS+7N6MP4ltvKvjyqWV1RC5NZ2BfvNqJsleBiTUm8Y\\\\\\\/7t+4P0BliXKgSYbrqe0KFN3QS49rPng5lM4qY23TpoPoJwKttmEDffW9n80xQ8+P\\\\\\\/CBdY1HdaUhia1UdSTq8wIMjFyxdN\\\\\\\/Pfb6qd9hQU7mDp7b4tOBio35YvnUpskBjf4llwJ+sQEBDGzUcKeIPcWUjn7ez0yyfnZWJuKSUgAtrcSRhwwoKDxA9dkgm+dCdz6ahNpqsYZkPvDsnciQtPgMksfOyv8v\\\\\\\/4\\\\\\\/jQc5z\\\\\\\/ArjMdNLF+ckxqLsGgzUp8KV8KpV5DcGiyHfob1OY78HaxcEEuXwBO3Tch524Y6HxkRblYJDgIjQ8nIRSH8quMtOiVTlt70hJ6OoyKI9KMNSSVKVlmhOmAXpqYg76HDbzYThT2KVkvmN4wYtPAY0GZSLFJFcMfqlTt8+uIayTCEdKsvKf+9lcUQdSDDRoAQqdOd7dwSMAY2sFjrqVRfH6ZPReFK68JMKK7rpQlnxSi+aX0j01IRaAHN0c85cjQd0eIv3uGIF40a+pYn0yy8ExAN9thP0RDLiAV\\\\\\\/nCZCNYknEHuIc0K2ifBpGrqWvxykiNUNWFX6EjjggOeA17yTgVJ9AAYy8ndCxvjy\\\\\\\/LlY+Qu70oZI8vfOC+hj5NzxxetUqqUvUz5VErSZurQ7T4PWJqalIn1cDETCt87Sxo7kQTrCU1oSzeh+2aVs6cSEszLdTHZ3MGFEwt2pM3q8XhUIGnyhmyIV29mtJT8muG9m5aEqc4Z67niI3oRLDKHdNk9QUJomNVm14ZWeATWq13xCUstwWZY9BMX7MvUHIoMmfKKpLUZYIOBXbNAyx6VfCZE+lvDOWe5JIWZ9bl0K8uFEJGD9JmokxIFwGdBptwuNQqRtbmBwn31Hw8IdBe6GnWMGpE4eNDirg5z1MNCheBictQEysy6Fl\\\\\\\/IEHSI9K6yBrx2v9KKNEo8gY5+nHnVX80t68s7HAfumNp7evBKx0sFjSz6UhbV5atq\\\\\\\/7NAOoEsPtbd5DlzCQx1j8le9YTh0zVAeLS27jtsh2xiF8EocURFMWbWjWcQPuTMi2nNJKG8sw7mk9HXo7jNf+gkR9DNn6EW9oxd82ca7dw0z0R4\\\\\\\/orCGdxHkJ9lyCn2opq\\\\\\\/xJ6TJvGaMW0ysU1Ftd0cQEfJu0jqyOqy2U4EcpaNhbCT\\\\\\\/oaHJBS13+4mfFtxze3LdGGxvjb1zXuWXveirdtm6Oc3xkc0MqiJrSKXK+fWCxhCz6ncS60fLqfd9wR0DTmujoJW0oXeNRpKqlDCmH7p7LwMV397gbkx6Fmygn3V4HTncWRCR3EYaxTX\\\\\\\/VrUbc9qVwdPfM+4ItGSGFc1GZTu2ovgucvw3Yvoz2tRLTL\\\\\\\/peSH9\\\\\\\/QG\\\\\\\/CtOAV864faIeGnhUCyjUMjzcld1beV1WL79sX+apjwSmU6oAXX2a7zNbRbvVjBisWMJTa3GF49zq5lCsixTtJ1tOtgB45B6H+PD\\\\\\\/5YyN7ssLDhwbNPsZ7VO0D05pnDGpu4gXD+93Un9jRs3qo\\\\\\\/39JW8gpmznSKfnJDXng8gbIEJxJl71OHEqRtnpdziXhtPU+6P1w5Bd2h59wp3TYszlQTTKbENvqNyY06miZCt7goRrBGQd468V1fxV92o0pdVcV664FFImaebkEi2NtPZJE9PXUGjM7CTptDt9yaXDAlNVzsGiXvDfB3UR1cbW81jfDyf4JWXlZ3tfxin0yi+ZN8C6xMyQp8b0+GHUg09XAEYTaDTWVJ24jxOPDIGUQDo22hUGuxG5\\\\\\\/v7HIo9GlrdyCc8lBjUu97WyBsuTJeRdMNHr2KIchlUb7K70RMG\\\\\\\/pxRNxyz6FqAJ6Y0HmrHp7E0ORx98XnGIkS3Tdn5xdPHbX2rmlV4A0LGjIFE8Om4pmd1qgNYBICt3sgi4Pq0VY\\\\\\\/4OZnA4dRaLNpH+R7oWLj0729lygyFMDxESn6473rjy0rAUEfOe8Zfq8vwpETX\\\\\\\/gt8WuXL+pCUTXg1DWC3T363QKGcqp2LwYErHMGGKskcAdlqZMFCRzf3MITBsKtN9zugEmDMfp\\\\\\\/CwFfvlObsBOuA+DbGss6jWVo+pDKlaE3A\\\\\\\/yanbbpr7titaMQwIec8bh0fY\\\\\\\/cvzZnJvBhwUxSYsuu1IRdgUmihKpHIFWozsGvJWJaSuOx1KsFDwoZUG\\\\\\\/tzejmu+YjI8+ythIKg\\\\\\\/bjKXRSG9ELcCM\\\\\\\/LvTTAgmARgEpgT3ec+S2tKyy6KOikieLNpM39lNdm2pQ2ytQuXJSkN83ZZ6YdOHWF8+sUh3DTZvGa8haaipy8d657Iyemt6wadVcP9E5xxrp9m6tnasRVz9ql8yQXtw3tsnrBWaKFgllwzAujopeVuLIYuxzasgOC+181hMp5jwG5suwW+2cSjPK15RIEA\\\\\\\/LWM20pICL9LjOOlGLRvbytcRB6JlMhJ8xw4KvmM9VPVDYOiE4+MSPYydighefWk7udA9Ulq8nSb3nlbK1oNvC8vsDJpx8FVVhkrXovd0H9Hg8SPGnMsF1SemOgbIGtzlRhfuoDWBsMGSJcekwky\\\\\\\/wPTmAJQhCya0nDfT\\\\\\\/Y\\\\\\\/X9nQSCAMp6g6lav8Z1W00rnC61qaze3TVd+6w\\\\\\\/tdX1cY2CaxHTeBUb8MU1U6jmk7Luhg3OZna7yJk9ctOKIosrecO4fABxRpqr8zfuZoGzH\\\\\\\/RYTmpLvsFP7QXfcEmtAoDcozCKzfuAqD9q8eHcxTth1\\\\\\\/8ETSEXI1MO8wo8OZZOMutoQDmHZtDGc9STiTT2bS6H5ErNvrTMb7g9cGVBCZK8byCvsNJH4PALx6ONj6nTHtBaeZWTnwv71sZlBNcma+LUn\\\\\\\/iCqhJrq2Yokm2+vjMwIFGkV\\\\\\\/5nJBcv6\\\\\\\/h9oT9e7Xlax9y9gsSP5shoQS4aVLvIdtu6IeEsCMrG\\\\\\\/PAQjz5Is5psLl19Nj\\\\\\\/Pe3j8pQBtFdGBSeE7NLInt\\\\\\\/OaIr2yk5vtQ0eJSFO+8R3bx24JbqbFfI05mQYg2yDmKj4AwQHqDb3jKukOqSWy8e59H6ulDRexQhV9ag2lvT4rgjI5XxaQmp15CbNu1M9LpO8iEdTrQiA\\\\\\\/mI4bzY8fBwgrUTOX9F8NCMuhP9CgQktkF+g6MKuIcwFG76wm6o1VqKF9yDq6MJ3+JRhQHebqlcqFERb6CarmphhdxI2lL5otO10xJyhH+pEU\\\\\\\/k9LHPLbLa4ZMUJ2xEK3KXIDXICpRO4Rffr7srX3o\\\\\\\/qhYERr6ht6k\\\\\\\/voA5jSib+bE7nyfxdKANDZiWqpwwAybDZwEs+t0geoWoKE7p+KOSYsEBhjrfmfyzRNfzNh5cDkyaaspfCDsRCAreMVY7AZgz2WZ5PXL7Wkhw7G8fOOI7it2K\\\\\\\/sEBePyACs+AKgQm6Mqp\\\\\\\/HPbUegy3ufd0NlCNkXCfjPUXJfy3WA1wF\\\\\\\/0VktJ828mjUY\\\\\\\/FZ2lHqc9d\\\\\\\/1PiYgxhSNKlN8INAD01hGg3OUZqX5eST\\\\\\\/oY+s4exZoyPkNBbxo1e\\\\\\\/1XyXHRrpoUMTk1xNsNkHsaeyjNd5fCmq3YeY6K2hHHO7N+CW8DPv7vcPYym3xa3lSQlAAQStUuL9428xhF\\\\\\\/jm3vhqbiWtS+vmUUIOhoDmk+1\\\\\\\/Lpm6bXqJhi40Eu02P7aOzLwD62XkS9vb0aYOT0KeLH5vbUlUbqbmEqBUahnY3BSjdjAf9eUiocf3ScuLqBzV9q5tmxA2BZEumXQdn84bRroitgwsi4KfKK7+lD6GOIRyXFGndbPXel0\\\\\\\/lrpc0dXSAOLXL4iGSk7irHZwHRBDHvHvJe1mAuQ6k6yf82BRLVnrlDI1+wDYiCv4SrGB9x9QjI6DKKK49tqfoZpoQ70BqGoa\\\\\\\/ee8MzFeb6BIUfyLu+jDVzPtnwo6EvkjGDS2SHCoNLH8jcQHRkwDE\\\\\\\/69WIpLbNimB5PgtBMZWjUQVaUM0Ken4yiLZbNSnlU8zCLbzy2sw+jlWSg8r2\\\\\\\/K+oCQb5W7\\\\\\\/hqX13ttCKAEDdtKby5u9iE4+9HQcLCDgQK6\\\\\\\/cp\\\\\\\/gXWc5ExSr9bL9lcvQjW1ud3dTQ0eVplT9RvCBMPvCP49urs6givaKcfPDNr1pdE5OSC3VkVOcsyWEFhX6DDVJEEWeF\\\\\\\/qlRy6bFNd0rYLP2mboOdoy1YYWed6NiH4tErhebOQ+BEHJsT09vfw3NqqpIugiGxGRp\\\\\\\/wRH6XMyRQJRK6ZEFALQR9+S3WU9MmtrK7QNl63Wm8Sgi10hE+qrJyGTZLfi2V+0X1gxxZaWgAFIxWkpG\\\\\\\/K6Y9EhFksqjFMieRQys2DfVNdWxsFN5gBBY9NK+VHZslH2BflGjofk0aVzAyoCxeRbyw1y1S7aY\\\\\\\/KW2DrFhENuHf5H\\\\\\\/0HceseffDxsjCFk1W9gyVESSbnNEoAr\\\\\\\/+DtmsvRvl64GS9OZumEcgHPU5Xb7soCFOjdCuuLc1dRglxnWAhq+RTqWN5UeIx9vtrlz\\\\\\\/wun9shlQte0TnJL+kTXnQpq1iihzHql695FZ\\\\\\\/vLVk2zEm3Phx48HoSgsObePp4NAFqlAUS1QFoHCDYPSy8tG9JliL9KpEY21\\\\\\\/kDsgijSXpsm5o\\\\\\\/QQ0\\\\\\\/gt1cYIC7Mem4opBa3DDo9OLTjwdFzulcBvyExUG4YjTLWKvVeMSRqg1gbKudBnP15FWVtEueMxVHEVaPyth24IHGS0b6yT1eLzpzOv2gBi6vBBKNHVQ6n7YFge1dVM7JheoN+G694cR\\\\\\\/eGbRpNes9cEC7xWAJXYmygDw7ueZ88BdY386QmLfcKwjoRBKOta\\\\\\\/pR20c064bDA\\\\\\\/\\\\\\\/pD\\\\\\\/NUe2H4bTSBkxZh\\\\\\\/zs\\\\\\\/YgJGavy8D9WKxW+omtJRtZVtjAPqiUwm6hvytu7KwTfGJU7D8WhLNVoRWyOqI7YtUF+k79iLlw7Ae95C54tdb14kGFHVeJu1yFQWbbxkoeWkMNB60USuIWerOhqNLHjtqG\\\\\\\/qIBqcvXYjj2qMs3j67eK4xxpUwWeqC5tBzJrue0Fg+crlwhn+G5gBuYe+\\\\\\\/HSeYVb5\\\\\\\/EhxRXIKyM8ittqzJ++GVOmo5WodvUEjMjoe+EkHy\\\\\\\/SHY\\\\\\\/T+LnF7OjcLBRDCEOn65hqY\\\\\\\/7yej9VdpLI1h2NnyiveAuaBELaege5wsLKDDRmacInFuIGdtbXizC+KeNcONLEK\\\\\\\/L\\\\\\\/Ete+QgdqajvLBlo8ZVnU60TIlig33VtAAJeJg54NFlYrD3VfL3EpG1oUR3aLJeGC7ydk7fsoD475XRSSlFHCYuUjWHPYt6dj+XWIJsp3OWoIiFk7vOaf7SQ0eC1a6kVjgtOKRTodqTC3aw7eccToVHS67vsMApvy133A4aMArupMY9PIVUKs0LJvQtAPfwLBtDi+QGLyRdjRxP9BoHQMv0jcF5adwXZMf+wIRCwrGp0DD\\\\\\\/boe59NjSwIlUwT7100UQBzs9U+yptxpScshSUjPuT8lcVllqOqagMBRQ6K1IfZjdERw7Ev90wha9dhoCj2s4IRyR1f+mvGxJHhBy1E4s\\\\\\\/Zwq7wJS8xAzShDQZj2ReNsxHjaONelgcmc1eCUBBsmMhmA9YTxPtv9vid80U8YQ4ToU7U4bhEjnE31ThJ9V7J7TEhM5O91\\\\\\\/GRtQk2ZeE1YHUyEXmhDozK9Yt+pQLyYpUXoXmScNXGgvNF3ePx5WnZ5svknz9ilZZf6kqhTGhVTnBEhJOPXY4YrA+ESzSMry08sp+XiWRoPVtZqLhTpseAy1qRWw1SUf8MyG5pnXv9rrNhq6Bc1unF\\\\\\\/UGyp\\\\\\\/d\\\\\\\/D8TZzRVHBW3P\\\\\\\/jQMfkrkxwEJCpl1grp0EQ+jFPPHhy2bLt2rHw1aUUaABZVuT5z+wbFU9SCPCEww1DmjF7M1mqeA7+7O0XIIWEb4eqM0nDrictj4g0RfU98i7nlYhGLtCTyqOOuDcL2DS6yaBGNfSA81vkcx71vk0CFhq5hsvX5HYNuK\\\\\\\/8\\\\\\\/l3W1jKiPFzAFLHpcpZgXqKfS493ccnePBelwVy+Y2XV2MjFNt+NepYHPJajymlbL5R8MWTfx1CqFQWK0k+RmNmCJ\\\\\\\/HOMy3a+Uy9obXFF1VTZhL\\\\\\\/FTLb0H0LzeZ7auPSnEBjYB01isS7TWDVdwICC5DTN+fC\\\\\\\/DqBPywjyPji+YreKIFC98yPR8MvZKl06wMfHWJfIFOj6obPCeoLFuMc7+is8dlY0\\\\\\\/QkpSz5WmfddXQo89lUIorZUGLmTZkEUvkNalSIND2ZCp2RdBJjznqMPe3mXj8d8HvKseDUo6FawBCblKbYeZk00NMCk43DqMcC6+vY9mX7cQlkto1CIu8G3pcxMdtdfdvjbcW+tCsTDG8GYY3HKvM5YGhS\\\\\\\/T\\\\\\\/YYxKJDnSJ5ZNKW5Rw\\\\\\\/EBLP05GXIiE4mHkUhHXMSoTj8fDABv5qxbCEw+Mu4ufBMItgbsA0V03Sq7SaT9JDP8wUnUUpzk5arRpm8iWxRUH9yyzUoYbUadyi5Az6eFe8z8pO4jEwj8Cwp53MvW51j0x9vTJ0u0R7ERLjSq\\\\\\\/4YKTI1uqze5r0xkN5jPrJHbAoGC+NsB8gCmzt+ekVfTnJRF6NDe59zi3GJa9LCIRJ30IJtlfq72JQ41Ur8y8GfBKVUjLSLDasvL2VR4lmCQIA18u4VNTOZD\\\\\\\/\\\\\\\/94c904093VKgZTFv0I7Sd4BodVWVNgT3CSYx3pfyFhax+beU1ecLKeveKHIoMFf9JrNRzOLWuvU4\\\\\\\/FP7z3CoxYmtewn6Imw0hFBPXl\\\\\\\/GQnGMPimy6OXQsIuSNiWvBw\\\\\\\/j54nTrA2VUi5zJRBAiwQ94IpVCqY534hTpQb3A\\\\\\\/lEugJWnQ\\\\\\\/o4wrGqLTr+MeXowmqqNS4fKHRXbxycY3x0MQ+Laa7zkvYffLcAqjgiasbdr7vPTgDPfikfHY2jjzuZljL3WchkvADBDo9SoEuOzqtg7WjSmozvij77NOyVJgzTuArclmwSNV5FNgp3cX1IeyQxoFllJIXRV+xyaFbouyi\\\\\\\/QFDxzaYvd7XmJBGgadE246euarZVfreH\\\\\\\/pkXHfDqdOaU32f0WsRIbl1\\\\\\\/w1XjAipk4Sq4iXBZrKudTCGuSa\\\\\\\/\\\\\\\/wQyPeVJPxxbiiOaNszr0\\\\\\\/XLVbliO53ANHiFk5cX21tBmGgJLOxZ4nrh4kiR6pmtg1+vCv04DGg25pSci2SAvN11nrfIrglzokHoG1rCpRXSjVtnsxuWLt5C7tUR3ih5J2O4tWlZdBu+dRNSCsPCuyBnCNZgdZhiqFIxyxBGCA0gnAX9MI+lyx7FgN\\\\\\\/7Ad4iXvWsw1KisNA6u2JoP2Qf0kLpjZT48kAs80UgmHGPP0MnuOdqVuawOZDlIOPtkj4LZqWenoDwiGGX8MPGFDQ0dL+nJ6tZrDHiFv+CGUD7P4SjLMtXvzQnIdQoVaFRV2hrWq4m1Gzw+jv1Xyu0q5diry1UQjANYzH5SSjJDuvcl556rWtA5XpR1X1NpHiOjy00bo\\\\\\\/RwL4MDUJH2xl5EwfRHv2jtl+P2x58xRqkUyCYa\\\\\\\/Tn0\\\\\\\/qtOTlUArciFkW3mV8KHcVQE7vdIb6O6MkMMgv3wPkxvY577TeVjzbKe3MR\\\\\\\/bwCaQzy\\\\\\\/QnCy\\\\\\\/hQgTBmizjoLq4bJYkQq1p1WK5sRyrd038WV4LPKU0T2Ty8\\\\\\\/hnxx5jzNFDrJCglGlAA0Bke1+A2fWxy6Rh2UjeJNiFtdMfAepHkrIjSmHbUIB6s7YptEhj+9RnJI5BFg1cRGAYwSNUJqPp5I1MJdy9v7\\\\\\\/e45EZM2MyTmlWBVsSvTmq2PisrnzGMtjGenVROYHh1NhU15AfM7cikHoA8xsNbpTYudig0CjMF4Fz7t\\\\\\\/41I7oeRM3OAv26dUcYn7gCCP7Be+NQqVmHaWyriqrOTE0MjeZJINo9sRd+UPiVZnSyht9vxhGmHNDlxm4vkVywnsaaeap7BVDNQbfqQw\\\\\\\/2gVwFaqPgaOnDNZcHmGQd7qm3tKoFqslhfQ8fhE\\\\\\\/6hNARZCO5+BZUY78dW+QCEiqVmKPGV9oNWtmcb3YLgfP7YnhoejeVqT4O1PE\\\\\\\/R6VLdnJw0HQuFQNov1FDaG3GmNMTwaGNHCAOc6rxXqLuJJFaqlTKrVY799oW\\\\\\\/uZuMxXgvt6wMAgMtJK\\\\\\\/TAXDezSkzswgG9yI8OHeYhPe4lfKID+RuyilIOfVExebo1jorh+TsL86kKcFFu1J60CyB3I6\\\\\\\/tV55c3AptHWiRIEs3+rcIFG1SNGwgTyYnYpWr4Ztf+fdoomIY5VtRWKoTASPHfP6UYuYmJYWgiIjx1LAa2QCDRYjpnZuWUzNbketAHzuHT1in3LszTpMsZEj0ns4a+8mKRlS7ckD\\\\\\\/pX\\\\\\\/SXNA\\\\\\\/uiMvuBfx\\\\\\\/tYs7x4dF6lI3YeoOJfZnox88x7bAlT90xIMYrcIKhAYhVRITZhNXH7r8lhvv6I2+PZYGrwEhzIb9MeWZiGZs7EcNru\\\\\\\/rvPEcm91BOn+4orMM6xFMUxIGB6Ne2juNeyncUPivOFzrOtd6DsUSSiJSzN8WY7w9djd+GroUEdFtRKEupZF5HqEua4\\\\\\\/v\\\\\\\/q5bjrfhlMW4pSvbnPbRxp41o00+JIFg4NBUl7gj9HqzGIRFwlM99KvZQrf+RVuill\\\\\\\/yoctgUzdl6dYE2xujWkC\\\\\\\/GYOnH4hdpF0x5N5UTxEnn5NntLT9dMroUGGMKOE9PlPHW4+4r\\\\\\\/NtJtWq9cgheePVv7MKWwEL6CX6fRuvIWja+dg4R3COpb7Hxh\\\\\\\/kZJBjKr7yprqvrkfqkWytaEI6vwTr0QkuExP0LUjs7X\\\\\\\/4GkahPSRFnlN371z3Wkr1dpPb3mpVKKx332X8JxeX+CyF\\\\\\\/VYkIstLfAfYlblUlYSLqiMh3V8whUxO\\\\\\\/k5goN95f2qRPsVd38kOfOFcumQZ1Jo7m2lByN2EHKH7wiq0RCH+jiu9WMwarxbPVzKVgjg1KGU3zDOU8BRT3Xjky+2DxZ1lxLNM8gEUijSMdpX\\\\\\\/O2zcL0fiU4ow6WRSgNLAPXWIcSaQq8WeB\\\\\\\/jD5rzwygGyU\\\\\\\/GD744rIP0fr+91adV6F0bhUaSKNZOqtkqWEkOWxreQ8HOjVwlEM6WOwav7DVu2p9pRcRzNXzZNjy+YcQ7zv7q\\\\\\\/T\\\\\\\/GHzF\\\\\\\/gDcX28HelocLeeyYFu6pNtmML7sOg9UXLrSYJlYUZj0C22Y+hWwiI1JgrCnQnY2BWBpP1iEuPfRwt3mB8xe57MxEURZakKc067VNhAsYkXD4fJLuc1RAYCmxSrnA0EWBZH1hZwLDEHIwPczHH\\\\\\\/Q2jUnYjNA01gODlirTUWw9AIEmQl6f3KXccNlBReJcCh4uBjPEaJeXE+3CWiTC8DADDRfamra09tBLaI\\\\\\\/mJMvmCfTlKN4hmImZkUKBw7wfN90U20aFfT65YnfuQipGz42GS37Jc5AH8BHdXQ1AFtTdBXadIIHsbhlfdPHWTiXSam7oZ7ks+KcXiaNA7x+yQP1rg9weBvl8AJlojLRYrcLwXWrNcbpF2ZMgWnGnTnirqLYOHs4kACAtucW5OoJQhMjoVSbeI6dqewL+uWDxlE1\\\\\\\/l6oYWoCUr4kbCSwVlgOkfs3ue3nuWwzth23E+NFuPUtwYFQSUZTnJgpwgkU5LTDK3ny1brpdh+SMwTOXth0sYBDhrYl79TIPdGJAaKYj+6thwc\\\\\\\/NKl9R5xYQ2QvW5ec1vsb6sUPqSJ2euJaOxlacPkUFJOhLDrTqkdvC32WCgOEXJCRlj26Y3Fw66zCYwTniiPj8IJzVzXfYwttWv3WKfwfMHpR9N8S9a9WKAPJALdn1YouKZKbyrtf7kKP4aNxFoa+wt2reC8BthUoyUgMghMNKPGYXs9r9EjHk3+zIvUXNroFx8bvo3BbMhgy6G1qV0VfCEO47PLVxN0KQjlVnQT4uBpBxaY0GmR2NIVrMTw8jXw3suJJ8GZriWM2LxfU9Gh+EJx8rELggFu9uFXe5Off3TX3byO8aV0w9K65DrFnKyC08xJ6hCvlV6PeIL98JyrSPSI01aGzpIABBeo6yFyYXEbmeg77PRNeiNtgcrLd\\\\\\\/QjavglVtyN8ZKxmz1ym\\\\\\\/eIhdlmjNocUnL\\\\\\\/1W\\\\\\\/TzuMlU8jjeqaoDILco\\\\\\\/\\\\\\\/ilRAtlXCWXBw9hb8UQnWRjb5T3EzrYIYtuBuYHvuZN8xqJav1PHUHDVrAgEgg9Vk8fnciMzoswPQtHhlTUvE0fVr0Fr\\\\\\\/GHkDsx5bxqtOgFOft0MUztjQV4Dcwuz3t6JIkIQKxm\\\\\\\/kGuvc\\\\\\\/\\\\\\\/XLeGTbIB84lbMPriJk9EFvkl7ud8efoiYQ2flIgG2Nb8zQmFO705Ri9USeoavcaeF0labXkKhwdPShegugpTc\\\\\\\/8sFXv3Y6dRfpZvjDyMqIHnlxyG2sxkJsZePNcOVFEdiUcpCpzSNq6x6wzJi8FPkZmF2qZNC7NIsyB6ZeLIRAcSs5j8Bs6UfFxF19TnaCntP8Rzt0ovQhNtyqx81QWfb0\\\\\\\/UwZQgFPywn2FWnknXUiTSwJWHcfFXf2OHDUENiUxoR1+5WtwwkgIjKq8p59dWavuA4seCdXfsUZQgt+saIWIs1EnJozYzcsDt6Nggr+wi\\\\\\\/CIoEVgV8nN5q4xuHSiJIJnCcj5dIr1rTRndNOmPpUvavgHZLQdWlsoE2tAuWlt6WG5x1h49Mb5iywUQCLP1An+da\\\\\\\/psl6nhyVNvvxXUWH+UhU1ubPLxG8YM4Us5h2JeChKMJvJu4\\\\\\\/DD5xg2KkEaYbhVid0M9pd0mgq\\\\\\\/QBECDUlKs4v6XZLNmttCGorqujC0DSpK+5ze3Lgw4JYydVZuIhEx375a7J\\\\\\\/82QiJyq20O2sJXFCTSOcxyGjC9fo0P\\\\\\\/TDjdY3RvmWdx7iuFqdj5vE+paP6bIJf+F5E+C4guux6ox\\\\\\\/p6zbvrUi\\\\\\\/KVfSj+vq+JojrKrMOfDVJX5WnZ9249HX9k4qHBIGhMAjsHZSdRZiNmD9l6Un2GfvCX6NLBY5gC7xIdQwwwR+CBFeLU457z3zB6i34+O3sKh4ReCKY1XEntlkuKbywBM3WSncVnMZQzGqzPG3nk7p0mZdur1E15wOMeZK1QN2mexG03ERMZyKEawNJUrlikbNHczeA3i1gvkdWNhCVbtDtb+m0wIVFQnAX0L+RhAx+hsBKRpXDHHNqyYy97+Eu3\\\\\\\/Ebm+1jZUSsku1UANyfEt\\\\\\\/5msTfMcJrOtb97WkoxBF52OV7TYZ3S3L7bQrPvQEjEWeHS0eYYNoy\\\\\\\/B5uMiF+lNrqZen+vmmQcGbLu8YIuQQtK0YC5vMuCbZYK3yovc+fruJW8jdJRcnM\\\\\\\/PKzWWDV0MoM1umgZ5PXSGSlROcYQUZbOV+1VtV+gN\\\\\\\/AaE35lsGywpmu5hwt9NdrljVj24V3AgUtfxi8sBtQEfhhFld2eFYDTf7XgE+mXThR3xD9DskqQSDYTC4Ih5FMsgL\\\\\\\/C+4zWLz\\\\\\\/vrdPXOftyDNn\\\\\\\/Zf9MfkyYWhHVJI0+80oNgrOWfw5kZpvdcckBJkusx\\\\\\\/1bZgnjBFY1DcFCCU1giXr0ARTqQ+V9Qa7Rp+tcYWGzPltl1DifRKlVJHGGkTIdlD1vkWJB29GqBSgoBQoDuZB5umzUEnWGQVBQMrD5NmAALAubEvKjAdSOeJ\\\\\\\/hDfWuGJ9dKRlU+dj4gQ\\\\\\\/vWXlg\\\\\\\/iOOor8Br4qQi\\\\\\\/XsZPLoc\\\\\\\/HY984eU17h\\\\\\\/OrCEhKqvboPq0DZNWt8IxYT3ZzX784ZzK52+QvEIyCQJC7L2SHh\\\\\\\/xOwin6Z4VsrxirOogc4xYybPF0bo7oYDUfMIRmwvjZMQoZzdwZ76JVfFDlQM2NkLTkNKD0f1drGb+kcxOUnXEnsvXmy+PWh4ZZrQstPCKFOJn2OjBU\\\\\\\/jPIG3GpJS2pmyCnJqEL0oxjq8nyv\\\\\\\/K1B3MA4Kmd80GjpmWnsMKapn3WzASA\\\\\\\/nZq73dWHtunzuetMLfCmGVMWHLojtb6L94uLc8dKu7spH2N0xkHwORTv7gkKyv30zOOsLV3JPCsDAUtXsQTrKlwHMT\\\\\\\/CWNjXTcvM70VYn6uRCIvwBFqKK9DlKp7I+iB3iAjtJT4+iPKI6cuXhBTVS7\\\\\\\/4AUvf79tHkKBChhCHEOXHXGhEf4vYY3g9sSANUclIVBzDq5g6iSgtFM0LRoljxBe0RJ4YXzcHLyAqpusUoqlU\\\\\\\/LCQZ83BkbJARbGYsca\\\\\\\/OKLSXMcaL4fP8PSzkLYWyUd\\\\\\\/jORnyIg0cSVlb7nQH1XAIb+UXFrGFACuqh3OR6p15Ss36jbUL4gRQ3caYIMMEBBEy76lqrG8bN7CgfE\\\\\\\/DhpaEHhDWmz+7Z14qJxUT2oLnfruqJKAL108IOAzz\\\\\\\/kTBi1LsQQy4W92z5GPIQ\\\\\\\/RrzX712BV\\\\\\\/4CrOMKnmAcuM9Glq8u\\\\\\\/iM5YddRm85WCFRAgowQ4MNrLoHCp26xhx3\\\u0022,\\\u0022iv\\\u0022:\\\u0022GEcPfQA9zHULij0V\\\\\\\/EVDdQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c61cef15d2e8acd6bf17b7334df87600c38e4a6602622dbfeff22ca52fd43c8f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hFkWwjPqRBl6noxf7GQcrm\\\\\\\/0DKSxGSaYHcZMX\\\\\\\/Aq868QWB1\\\\\\\/NHek4d+S9ILQD8D37hE\\\\\\\/o3\\\\\\\/sxMHVkaO2K+Zf2T8hSCMHQfUyGXTKSA+XEgmVCxeILgtGD0Tvd7MzLdlhvZcPBKegvXpphcdYcYBhtm2bJShadfM1JtY1gthIWJX4JSApquLhojUFxS7wP3CUilmbYXWgPEWjqp50IoHJm7c7DgC1O2U25mbUQzzMjxEZGg3Fw2IZUzmSuJpHjSo9EDerh0duTOPOHUl0nj+5qh3uUIxWOJsQ+hLdVlkdvbmkZBpF3HPItVnbGjcEJVHsgaTP0QyT1kFsfF\\\\\\\/t6QCsdlrJsiIo9MnGKvQTk9CRMbyYZ\\\\\\\/BDujSn2xzBBX\\\\\\\/1wAHk9jVwDvXiNsVTC7MyK6IRgJJdboaEFZx6YvlBKnp9igp6KOxjXP\\\\\\\/tFsx59tIKMfByRYAXwtlTPvK3Q28Na0UcR0fdVrazdq2dftdNqE9CMLmjCtjKx0b+ZNTVtsXJugvCuYVG8CGq\\\\\\\/WdKkJD1YL9b3dqlGwr3vuu5gQRe05\\\\\\\/BgZWLm2RurmnI3EOb5ID6QqM0x5P1mJ5kiNUbaKahj3h+b3qVmUy7e13BemQD17O0CH0CuyGOPsWJlAfHw+IdIV+SAM9hD2Fk+3wvoJzBSNFusSYh3DoIqlD0wun7nDoLnHCAUKYpKo5TH9O\\\\\\\/2GJh8xMvICpf8LDG9eL+4q2luRlbOk+5X2DPoJVGOQ8\\\\\\\/ZTunqxFLh0yDakt+1olKC6SvSZcQum0mGkIVhik2Qk4Tnun+6328J8aHyOuo7CHCWrb7D\\\\\\\/dTQH\\\\\\\/dhFlKi+2POafWmJ0VrncALwwd+RuArIRMsq1RuPgWXqpr9lw0znyxtEaa3ZopVAyBZuv+iLu68li4juRUq+Vyph\\\\\\\/6bySPlJcJRnw6jWxTpIyXHiFCl7NfRsDJH6oQnWVv7xSc+nCvusj6f2sa+bGUp\\\\\\\/Wzjc5DUco+2ewuFSL6mrDjOxt4ZMtzhgEtAA9Omwxf4H7juul70WH3cXuWYkTbjRSMdGiaEsK8jFjJrZt8owKgtm8EWD6UNiNBWB2k81FsHDpvAbEjXcosiaCKi5QZNcHUSXWN23o322JM+iDrBsYDk96iEMAYOentGUrrg9dNWp1CXwSr2EY9md9r1FgdNI0ZI1SO58ZU81NZM4QRffxCdWRcPOheJCT5gWvw2Qxi98gvLSO4oQ8NTOR7x2QbRv9XMB779khqQa6+on8cy9kjHg6ZpQQtzOp7jral1yFEng5cxse2h8UGWXOR3E\\\\\\\/HUVgjFFcSqtGDi8Cuce4wXwoBdtYS1qfcBjhkP2PbopA5qDMmC7eOtfQb+5VMtkAeU9vzA0ZkQSoYuQZmrnWVgmuUTduTCDi86SVcixNok0Dl\\\\\\\/U2zouE4EN3bpoYH1qIXvgG\\\\\\\/1wicSmv1QEQPGuKTF539128s7P1kISe1nW2MkiIt5rpHzSRjmFfHfUPI1RxpM0NnMC+dnbbuCrpaek39A4JDpN\\\\\\\/yNmKcVhMWpQ5EiakGHVAZSW8Hvg2vLwtcxYcHvxeDn00W58ZO7sA8gLVn9B\\\\\\\/kJzxPpGZMw8dJsSVXfadYV87isKcZ9qfUf0zT7\\\\\\\/SKT40QksEQPl3sMx1CyGsbQao6ekrRDPXOklLArIcGHIZ9spajpg7+9mYb+Vm4NYCuokj5FgIYk8YcZAAIgysUwVc700fAbeHa3W9A064100k2mLyl5s5jrLZJz6qTanMBkLlUE6qGC4Ho1Rg\\\\\\\/FYtEq41R10NNMdJUOGVnD68RJAA39tATUV\\\\\\\/vTRTgSkZXTG\\\\\\\/cTJ5wI8ZrhS2Sgy4LTF\\\\\\\/GSDwi0UOk+Fwo3EUMeXF21c\\\\\\\/iDtORr8EWrH5UFHUbVhmUVHcA9lqqQqAVW1QjbBw\\\\\\\/kVKj0xBCGIwsRN84mJ2ru5uznGjrNgSFE333DnfpFEDpc6R42\\\\\\\/iHsHNUgcVPYV\\\\\\\/iaYQ0KAHOeKFxArKqfxMyeOWlF5gbObvthcpquxe6LVLUmjChfxY9Iz1k4k1OL6gL9WjPibqTXesqtgEz5VA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022HjEDOBtDgNH4O0cEMdQLUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229e8f00bf214d34d5d8abaf0f96e5bc723e800d8d92ce2d154e71291039a2a170\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493109 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1469" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PCAnBYqXHgra-5kUzt_XakkXeW7j4JDSmZCwIan-154" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1469" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Exkqezs95QsF3l9UeseJjw52ZHVU4xIpUJh3U-0mBD8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1457" title="Isolation by Distance, Source-Sink Population Dynamics and Dispersal Facilitation by Trade Routes: Impact on Population Genetic Structure of a Stored Grain Pest" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1481" title="Locating and Characterizing a Transgene Integration Site by Nanopore Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493109" id="node493109" data-pisa="ggg;9/5/1469" data-pisa-master="ggg;g3.119.400129" data-apath="/ggg/9/5/1469.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6578-1715" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antoine Allier</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7195-1327" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laurence Moreau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6125-503X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alain Charcosset</span>, <span class="highwire-citation-author" data-delta="3">Simon Teyssèdre</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-7724-0887" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina Lehermeier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1469-1479;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400129 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493109" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Antoine Allier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BAllier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntoine%2BAllier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6578-1715" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antoine Allier</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Laurence Moreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BMoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreau%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurence%2BMoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7195-1327" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laurence Moreau</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alain Charcosset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BCharcosset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charcosset%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BCharcosset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6125-503X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alain Charcosset</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Simon Teyssèdre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BTeyss%C3%A8dre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teyss%C3%A8dre%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BTeyss%25C3%25A8dre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clehermeier@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">clehermeier@ragt.fr</a>
<a href="mailto:steyssedre@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">steyssedre@ragt.fr</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christina Lehermeier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BLehermeier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehermeier%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BLehermeier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7724-0887" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina Lehermeier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clehermeier@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">clehermeier@ragt.fr</a>
<a href="mailto:steyssedre@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">steyssedre@ragt.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1469" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493109" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1469.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493109" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1469.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493109" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1469.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493109" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Predicting the usefulness of crosses in terms of expected genetic gain and genetic diversity is of interest to secure performance in the progeny and to maintain long-term genetic gain in plant breeding. A wide range of crossing schemes are possible including large biparental crosses, backcrosses, four-way crosses, and synthetic populations. <em>In silico</em> progeny simulations together with genome-based prediction of quantitative traits can be used to guide mating decisions. However, the large number of multi-parental combinations can hinder the use of simulations in practice. Analytical solutions have been proposed recently to predict the distribution of a quantitative trait in the progeny of biparental crosses using information of recombination frequency and linkage disequilibrium between loci. Here, we extend this approach to obtain the progeny distribution of more complex crosses including two to four parents. Considering agronomic traits and parental genome contribution as jointly multivariate normally distributed traits, the usefulness criterion parental contribution (UCPC) enables to (i) evaluate the expected genetic gain for agronomic traits, and at the same time (ii) evaluate parental genome contributions to the selected fraction of progeny. We validate and illustrate UCPC in the context of multiple allele introgression from a donor into one or several elite recipients in maize (<em>Zea mays</em> L.). Recommendations regarding the interest of two-way, three-way, and backcrosses were derived depending on the donor performance. We believe that the computationally efficient UCPC approach can be useful for mate selection and allocation in many plant and animal breeding contexts.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/progeny-variance" class="hw-term hw-article-keyword hw-article-keyword-progeny-variance" rel="nofollow">progeny variance</a></li><li class="kwd"><a href="/keyword/parental-genome-contribution" class="hw-term hw-article-keyword hw-article-keyword-parental-genome-contribution" rel="nofollow">parental genome contribution</a></li><li class="kwd"><a href="/keyword/genome-wide-prediction" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-prediction" rel="nofollow">genome-wide prediction</a></li><li class="kwd"><a href="/keyword/multi-parental-crosses" class="hw-term hw-article-keyword hw-article-keyword-multi-parental-crosses" rel="nofollow">multi-parental crosses</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Allocation of resources is a key factor of success in plant and animal breeding. At each selection cycle, breeders are facing the choice of crosses to generate the genetic variation on which selection will act at the next generation. In case of limited genetic variation for targeted traits, the introduction of favorable alleles from donors to elite material is necessary to ensure long term genetic gain. Several approaches have been proposed to introgress superior quantitative trait locus (QTL) alleles from a donor into a recipient. In case of a single desirable allele, it can be accomplished using molecular assisted introgression (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Visscher <em>et al.</em> 1996</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Frisch <em>et al.</em> 1999</a>). In case of multiple desirable alleles, gene pyramiding strategies have been proposed (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hospital and Charcosset 1997</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Charmet <em>et al.</em> 1999</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Servin <em>et al.</em> 2004</a>). More recently, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Han <em>et al.</em> (2017)</a> proposed the predicted cross value (PCV) to select at each generation crosses that maximize the likelihood of pyramiding desirable alleles in their progeny. For quantitative traits implying numerous QTL with small individual effects, genomic selection has been proposed to fasten the introgression of exotic alleles into elite germplasm (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardo 2009</a>) and to harness polygenic variation from genetic resources (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gorjanc <em>et al.</em> 2016</a>) using two-way crosses or backcrosses. However, plant breeders are not only considering biparental crosses such as two-way crosses or backcrosses but also multi-parental crosses including three-way crosses, four-way crosses or synthetic populations (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gallais 1990</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schopp <em>et al.</em> 2017</a>). Crosses implying several parental lines are highly interesting for breeders to exploit at best the genetic diversity underlying one or several traits. Beyond fastening the introgression of genetic resources into elite germplasm, genomic selection could be used to predict the interest of a multi-parental cross involving one or several donors and recipients. Among possible crosses, the identification of those that secure the performance in progeny and maximize the genome contribution of donors to the selected progeny is essential for increasing or maintaining genetic gain and diversity of an elite population.</p><p id="p-4">The interest of a cross for a given quantitative trait can be defined using the usefulness criterion (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schnell and Utz 1975</a>) that is determined by its expected genetic mean (μ) and genetic gain (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-1.gif"/></noscript></span></span>): <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-2.gif"/></noscript></span></span>, where σ is the progeny genetic standard deviation. The selection intensity (i) depends on the selection pressure and the selection accuracy (h) can be assumed to be one when selecting on genotypic effects (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhong and Jannink 2007</a>). While μ can be easily predicted for different crossing schemes by the weighted average of parental values, the difficulty to have a good prediction of progeny variance (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-3.gif"/></noscript></span></span>) hindered the use of UC in favor of simpler criteria (for a recent review on different criteria, see <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mohammadi <em>et al.</em> 2015</a>). <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo <em>et al.</em> (2006)</a> suggested to predict the progeny variance of a given population using genotypic data of its progenitors and quantitative trait loci (QTL) effect estimates, assuming unlinked QTL. <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zhong and Jannink (2007)</a> extended this concept to linked loci. With the availability of high-density genotyping, it has been proposed to predict the progeny variance using <em>in silico</em> simulations of progeny and genome-wide marker effects (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Iwata <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo 2014</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lian <em>et al.</em> 2015</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Mohammadi <em>et al.</em> 2015</a>). However, the geometrically increasing number of cross combinations possible for n parents makes the testing of all crosses computationally intensive. For instance, with only <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-4.gif"/></noscript></span></span> potential parents, a total of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-5.gif"/></noscript></span></span> genetically different two-way crosses can be formed. This number increases by a factor of n when crossing all the possible two-way crosses to the n different parents, so that <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-6.gif"/></noscript></span></span> three-way crosses and backcrosses are possible. Recently, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> (2017b)</a> derived algebraic formulas to predict for a single trait the genetic variance of doubled haploid (DH) or recombinant inbred line (RIL) progeny derived from two-way crosses, using information of recombination frequency and linkage disequilibrium in parental lines. These algebraic formulas have not been extended so far to multi-parental crosses, hindering the prediction of the interest of such crosses.</p><p id="p-5">While the expected genetic gain (UC) is a meaningful measure of the interest of a cross for breeding, it does not account for the parental genome contributions to the selected fraction of progeny that determine the genetic diversity in the next generation. Parental genome contribution to unselected progeny has been studied for several years and is of specific interest in breeding for donor introduction and to manage long term genetic gain and inbreeding rate (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hill 1993</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bijma 2000</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Woolliams <em>et al.</em> 2015</a>). <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hill (1993)</a> derived the variance of the non-recurrent parent genome contribution to heterozygous backcross individuals in cattle. <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wang and Bernardo (2000)</a> formulated the variance of parental genome contribution to F2 and backcross plant progeny considering a finite number of loci. <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Frisch and Melchinger (2007)</a> extended this approach to a continuous integration over loci and showed that a normal distribution approximated well parental genome contribution obtained from computer simulations. Also empirical data on pairs of human full-sibs confirmed that parental genome contributions, <em>i.e.</em>, additive relationship, can be considered as normally distributed around the expected value of 0.5 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Visscher <em>et al.</em> 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Visscher 2009</a>). All these studies considered the parental genome contribution distribution in unselected progeny. However, to control parental contribution during polygenic traits introgression, it is of interest to predict parental genome contribution after selection for quantitative traits.</p><p id="p-6">In this study, we develop a multivariate approach called usefulness criterion parental contribution (UCPC) to evaluate the interest of a multi-parental cross implying a donor line and one or several elite recipients based on the expected genetic gain (UC) and the diversity (parental contributions, PC) in the selected progeny. We extend here the rational given by <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> (2017b)</a> for two important aspects. We address the prediction of progeny variance for multi-parental crosses implying two to four parents and we consider the parental contribution as an additional quantitative trait. The originality of this approach is that it uses derivations of the prediction of progeny variance in multi-parental crosses implying up to four parents to jointly predict (i) the performance of the next generation using the usefulness criterion and (ii) the parental contributions to the selected fraction of progeny, which to our knowledge has not been investigated so far. We illustrate the use of UCPC in the context of external genetic resources introgression into elite material considering the specific case of a unique donor that is crossed to one or several elite recipients. We address the type of multi-parental cross that should be preferred among two-way crosses, three-way crosses or backcrosses in order to maximize genetic gain while introgressing donor alleles in the elite population within one selection cycle.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Application example: breeding context</h3><p id="p-7">We assumed a generic plant breeding population of fully homozygote inbred lines genotyped for biallelic single nucleotide polymorphism (SNP) markers with known positions. We considered a quantitative agronomic trait (<em>e.g.</em>, grain yield) implying p QTL with known additive effects and with positions sampled among the SNP marker positions. Further, we considered that the breeding population is an elite population that should be enriched with several alleles from a donor without <em>a priori</em> knowledge on major QTL to be introgressed. We assumed a donor line (<em>D</em>) has been identified and should be crossed with lines from the elite population (<em>e.g.</em>, <em>E</em><sub>1</sub> and <em>E</em><sub>2</sub>) in order to obtain high-performing progeny that combine donor favorable alleles in a performing elite background. This donor line can vary in its performance level and its diversity relative to the elite population.</p><p id="p-8">In this context, we aimed at evaluating the interest of two-way crosses (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-7.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-8.gif"/></noscript></span></span>), backcrosses (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-10.gif"/></noscript></span></span>) or three-way crosses (<em>i.e.</em> <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-12.gif"/></noscript></span></span>) based on (i) the mean performance of the selected progeny and (ii) the average genome contribution of the donor to the selected progeny. Considering different donor characteristics, <em>i.e.</em>, originality and performance level, we compared the interest of the multi-parental crosses listed above in order to derive guidelines for the use of the donor D. As a benchmark, we also evaluated the interest of different elite multi-parental crosses.</p></div><div id="sec-3" class="subsection"><h3>Usefulness Criterion Parental Contribution</h3><p id="p-9">In order to predict the progeny distribution of a given cross in terms of expected genetic gain and genetic diversity, we considered the agronomic trait and the parental genome contribution as jointly multivariate normally distributed traits. This enabled us to (i) evaluate the genetic gain of the selected progeny for the agronomic trait, and to (ii) evaluate the contribution of each parental line to this selected progeny. An illustration of the concept of UCPC is given in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. In the following sections we present in more detail the theory underlying UCPC in the general case of a four-way cross.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of Usefulness Criterion Parental Contribution (UCPC) for a two-way cross between  and . UCPC combines (A) the concept of usefulness criterion for an agronomic trait normally distributed () and (B)  genome contribution considered as a normally distributed quantitative trait () in a multivariate approach (C). UCPC enables to predict the expected progeny performance for the trait () and  genome contribution to the selected fraction of progeny () that depends on the covariance  mainly driven by the difference between  and  performances." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315272732" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illustration of Usefulness Criterion Parental Contribution (UCPC) for a two-way cross between &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-13&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-14&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. UCPC combines (A) the concept of usefulness criterion for an agronomic trait normally distributed (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-15&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and (B) &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-16&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; genome contribution considered as a normally distributed quantitative trait (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-17&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) in a multivariate approach (C). UCPC enables to predict the expected progeny performance for the trait (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-18&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-19&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; genome contribution to the selected fraction of progeny (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-20&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) that depends on the covariance &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-21&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; mainly driven by the difference between &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-22&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-23&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; performances.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1469/F1.medium.gif" width="440" height="289"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1469/F1.medium.gif" width="440" height="289"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Illustration of Usefulness Criterion Parental Contribution (UCPC) for a two-way cross between <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-14.gif"/></noscript></span></span>. UCPC combines (A) the concept of usefulness criterion for an agronomic trait normally distributed (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-15.gif"/></noscript></span></span>) and (B) <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-16.gif"/></noscript></span></span> genome contribution considered as a normally distributed quantitative trait (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-17.gif"/></noscript></span></span>) in a multivariate approach (C). UCPC enables to predict the expected progeny performance for the trait (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-18.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-19.gif"/></noscript></span></span> genome contribution to the selected fraction of progeny (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-20.gif"/></noscript></span></span>) that depends on the covariance <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-21.gif"/></noscript></span></span> mainly driven by the difference between <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-22.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F1/embed/mml-math-23.gif"/></noscript></span></span> performances.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-4" class="subsection"><h4>Multi-parental crosses and genetic model:</h4><p id="p-11">To cover diverse types of crosses, we consider a general multi-parental cross implying four fully homozygous parents (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-24.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-25.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-26.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-27.gif"/></noscript></span></span>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Note that for this general presentation of the theory, parents can be lines from the elite population and/or considered as external donors. This four-way cross implies two initial crosses giving generations <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-28.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-29.gif"/></noscript></span></span>, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). A second cross between <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-30.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-31.gif"/></noscript></span></span> yields the generation <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-32.gif"/></noscript></span></span> standing for pseudo F1. Two-way crosses, three-way crosses and backcrosses can be seen as specific cases of four-way crosses depending on the number of parents considered as visualized in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of four-way crosses (left) and derived crossing schemes (right). In the general case of four-way crosses, nomenclature is defined for recombinant inbred lines (RILs) after k generations of selfing (RIL-k) from pseudo F1 generation (F1&#x2019;) and doubled haploid lines (DH) derived from the RIL generation k-1 (DH-k, for k &gt; 1). RIL-1 corresponds to the pseudo F2 generation and RIL &#x221E; = DH &#x221E;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315272732" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illustration of four-way crosses (left) and derived crossing schemes (right). In the general case of four-way crosses, nomenclature is defined for recombinant inbred lines (RILs) after k generations of selfing (RIL-k) from pseudo F1 generation (F1&#x2019;) and doubled haploid lines (DH) derived from the RIL generation k-1 (DH-k, for k &gt; 1). RIL-1 corresponds to the pseudo F2 generation and RIL &#x221E; = DH &#x221E;.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1469/F2.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1469/F2.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493163" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-12" class="first-child">Illustration of four-way crosses (left) and derived crossing schemes (right). In the general case of four-way crosses, nomenclature is defined for recombinant inbred lines (RILs) after k generations of selfing (RIL-k) from pseudo F1 generation (F1’) and doubled haploid lines (DH) derived from the RIL generation k-1 (DH-k, for k &gt; 1). RIL-1 corresponds to the pseudo F2 generation and RIL ∞ = DH ∞.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">Assuming known genotypes at p QTL underlying the quantitative trait considered and biallelic markers at QTL positions, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-33.gif"/></noscript></span></span> denotes the p-dimensional genotype vector of parent i, with the j<sup>th</sup> element coded as 1 or -1 for the genotypes AA or aa at locus j. Assuming biallelic QTL effects, a classical way to define the parental genotypes matrix would be a <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-34.gif"/></noscript></span></span>-dimensional matrix <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-35.gif"/></noscript></span></span>. Addressing parental specific effects and following the identical by descent (IBD) genome contribution of parents to progeny requires to consider parental specific alleles. Thus, we extend the definition of parental genotypes to a multi-allelic coding:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-36.gif"/></noscript></span></span>with <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-37.gif"/></noscript></span></span> a <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-38.gif"/></noscript></span></span> dimensional matrix defining the genotype of the four parents at the <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-39.gif"/></noscript></span></span> parental alleles at QTL, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-40.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-41.gif"/></noscript></span></span>-dimensional vector defining the genotype of parent i and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-42.gif"/></noscript></span></span> a <em>p</em>-dimensional vector of zeros.</p><p id="p-14">We first concentrate on doubled haploid (DH) lines derived from the <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-43.gif"/></noscript></span></span> generation (DH-1), and then extend our work to DH lines generated after more selfing generations from the <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-44.gif"/></noscript></span></span> and to recombinant inbred lines (RILs) at different selfing generations, <em>i.e.</em>, partially heterozygous progeny. Absence of selection is assumed while deriving the progeny from generation <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-45.gif"/></noscript></span></span>. In case of DH-1, we denote the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-46.gif"/></noscript></span></span>-dimensional genotyping matrix of N progeny derived from a four-way cross (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) in a multi-allelic context as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-47.gif"/></noscript></span></span>where for instance <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-48.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-49.gif"/></noscript></span></span>-dimensional matrix of progeny genotypes at QTL coded -1 or 1 for alleles inherited from parent <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-50.gif"/></noscript></span></span> and 0 otherwise.</p><p id="p-15">The multi-parental coding enables to consider <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-51.gif"/></noscript></span></span> a <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-52.gif"/></noscript></span></span>-dimensional vector of known parental specific additive effects for the agronomic trait. Thus, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-53.gif"/></noscript></span></span> is the vector of progeny breeding values of the agronomic trait. As we assumed additive effects, the breeding value equals the genetic value. Assuming no parental specific effects for the agronomic trait, as in the application example considered, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-54.gif"/></noscript></span></span> reduces to <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-55.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-56.gif"/></noscript></span></span> is the vector of known QTL effects in the elite and donor populations. Furthermore, the multi-parental coding considered enables to define the effects to follow IBD parental contributions either genome-wide (namely C, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-57.gif"/></noscript></span></span>) or considering only the favorable alleles (namely <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-58.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-59.gif"/></noscript></span></span>). In this study, we focused on the first parent (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-60.gif"/></noscript></span></span>) genome IBD contributions, but a generalization to every parent is straightforward. In the following, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-61.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-62.gif"/></noscript></span></span>-dimensional vector defined to follow <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-63.gif"/></noscript></span></span> genome-wide contribution and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-64.gif"/></noscript></span></span> a <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-65.gif"/></noscript></span></span>-dimensional vector defined to follow <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-66.gif"/></noscript></span></span> genome contribution at favorable alleles. In the general case of four-way crosses <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-68.gif"/></noscript></span></span> is identical to <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-69.gif"/></noscript></span></span> except that if <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-70.gif"/></noscript></span></span> has the unfavorable allele at QTL <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-71.gif"/></noscript></span></span>, the corresponding element of <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-72.gif"/></noscript></span></span> is null. Thus, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-73.gif"/></noscript></span></span> represents the proportion of alleles in the progeny that are inherited from <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-74.gif"/></noscript></span></span> independently of the allele effect and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-75.gif"/></noscript></span></span> represents the proportion of alleles in the progeny that are inherited from <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-76.gif"/></noscript></span></span> and favorable. In the specific case of two-way crosses (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-77.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-78.gif"/></noscript></span></span> so <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-79.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-80.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-81.gif"/></noscript></span></span> genome-wide contribution is defined by <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-82.gif"/></noscript></span></span>.</p></div><div id="sec-5" class="subsection"><h4>Prediction of progeny mean and progeny variance:</h4><p id="p-16">In this section we consider a generic quantitative trait defined by the <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-83.gif"/></noscript></span></span>-dimensional vector of parent specific additive effects <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-84.gif"/></noscript></span></span>. The vector β can be replaced by <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-85.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-86.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-87.gif"/></noscript></span></span> without loss of generality. In order to evaluate the performance of a four-way cross, we derive its expected progeny mean and variance. The expected progeny mean can be derived as the mean of all four parents’ breeding values:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-88.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>The progeny variance can be derived as:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-89.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where Σ is the <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-90.gif"/></noscript></span></span>-dimensional covariance matrix between parental alleles at QTL in progeny. The diagonal elements <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-91.gif"/></noscript></span></span> are equal to the variance of parental alleles in progeny. Note that off-diagonal elements <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-92.gif"/></noscript></span></span> correspond to the disequilibrium covariance between two parental alleles j and l at different QTL (<em>i.e.</em>, different physical positions) or at the same QTL. The linkage disequilibrium parameter in the progeny between parental alleles <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-93.gif"/></noscript></span></span> can be derived from the linkage disequilibrium parameter among the four parental lines and the recombination frequency between parental alleles in progeny (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, see File S1 for derivation). In the specific case considered, <em>i.e.</em>, doubled haploid lines derived from generation <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-94.gif"/></noscript></span></span> (DH-1), this leads to the covariance entry:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-103.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-104.gif"/></noscript></span></span> is the sum of the disequilibrium parameter between parental alleles j and l in pairs of parents implied in the first crosses and <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-105.gif"/></noscript></span></span> is the sum of disequilibrium parameter between parental alleles j and l in pairs of parents indirectly implied in the second cross. <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-106.gif"/></noscript></span></span> denotes the linkage disequilibrium between parental alleles j and l in the pair of parental lines <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-107.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-108.gif"/></noscript></span></span> which can be computed as <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-109.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-110.gif"/></noscript></span></span> is the recombination frequency between parental alleles j and l in the parental lines obtained from the absolute genetic distance <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-111.gif"/></noscript></span></span> in Morgan as <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-112.gif"/></noscript></span></span> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Haldane 1919</a>). When j and l refer to parental alleles at the same QTL, it holds <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-113.gif"/></noscript></span></span>. This formula given in [Equation 3] can be applied analogously in every case presented in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>: three-way crosses, backcrosses and two-way crosses. See File S1 for a detailed derivation of the covariance in DH-1 progeny [Equation 3] and File S2 for an extension to DH progeny derived after selfing generations and to recombinant inbred lines at different selfing generations.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493008/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493008/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of genotypic covariance between loci j and l for different populations derived from the F1’ generation based on the disequilibrium parameter in pairs of parental lines</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h4>Indirect response to selection for parental contributions:</h4><p id="p-21">We aim at predicting the full multivariate progeny distribution (mean, variance and pairwise covariances) for the agronomic trait, <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-114.gif"/></noscript></span></span> genome-wide contribution (C) and <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-115.gif"/></noscript></span></span> contribution at favorable alleles (<span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-116.gif"/></noscript></span></span>). Therefore, we consider all three traits in the <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-117.gif"/></noscript></span></span>-dimensional multi-trait effect matrix <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-118.gif"/></noscript></span></span>. Similarly as for one trait, the mean performance (<span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-119.gif"/></noscript></span></span>) and mean genome-wide contribution of <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-120.gif"/></noscript></span></span> in progeny before selection (<span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-121.gif"/></noscript></span></span>) are derived as the mean of all four parents’ breeding values for each trait [Equation 1]. As expected, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-122.gif"/></noscript></span></span> for four-way, three-way and backcrosses and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-123.gif"/></noscript></span></span> for two-way crosses. Progeny variances for all three traits are estimated using Equation 2 and pairwise covariances in progeny are estimated as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-124.gif"/></noscript></span><span class="disp-formula-label">(4a)</span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-125.gif"/></noscript></span><span class="disp-formula-label">(4b)</span></span>Progeny means and (co)-variances before selection can be used to estimate the expected response to selection on multiple traits. For this purpose, we used the Usefulness Criterion (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schnell and Utz 1975</a>) in a multi-trait approach as illustrated in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Assuming an intra-family selection of the progeny with the highest values for the agronomic trait with a selection intensity<span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-126.gif"/></noscript></span></span> and a selection accuracy of one (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>), the expected mean performance after selection <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-127.gif"/></noscript></span></span> is defined as the usefulness criterion of the cross:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-128.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>The correlated response to selection on <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-129.gif"/></noscript></span></span> genome-wide contribution (<span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-130.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-131.gif"/></noscript></span></span> contribution at favorable alleles (<span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-132.gif"/></noscript></span></span>) are (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Falconer and Mackay 1996</a>):<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-133.gif"/></noscript></span><span class="disp-formula-label">(6a)</span></span>and <span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-134.gif"/></noscript></span><span class="disp-formula-label">(6b)</span></span>The contribution of <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-135.gif"/></noscript></span></span> at unfavorable alleles after selection can be derived as:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-136.gif"/></noscript></span><span class="disp-formula-label">(6c)</span></span><a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 1C</a> illustrates, in the case of a two-way cross (<em>P</em>1 x <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-137.gif"/></noscript></span></span>), the indirect response to selection on <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-138.gif"/></noscript></span></span> genome-wide contribution (<span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-139.gif"/></noscript></span></span>) depending on the covariance <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-140.gif"/></noscript></span></span> that is mainly driven by the difference of performance between <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-141.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-142.gif"/></noscript></span></span>.</p></div></div><div id="sec-7" class="subsection"><h3>Simulation experiments</h3><p id="p-22">We performed two simulation experiments. The aim of the simulation experiment 1 was the validation of the presented formulas for the moments of the distribution of progeny from four-way crosses. In simulation experiment 2, we investigated different crossing schemes (two-way, three-way and backcrosses) in terms of genetic gain and donor contribution.</p><div id="sec-8" class="subsection"><h4>Genetic material:</h4><p id="p-23">We considered 57 Iodent inbred lines from the Amaizing Dent panel (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rio <em>et al.</em> 2019</a>). Iodent defines a heterotic group that has been derived 50 to 70 years ago and that is commonly used in maize breeding (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Troyer 1999</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Van Inghelandt <em>et al.</em> 2012</a>). In the following we refer to these lines as elite lines. Elite lines were genotyped with the Illumina MaizeSNP50 BeadChip (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ganal <em>et al.</em> 2011</a>). After quality control and imputation, 40,478 high quality biallelic SNPs were retained. The genetic map was obtained by predicting genetic positions from physical positions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jiao <em>et al.</em> 2017</a>) using a spline-smoothing interpolating procedure described in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bauer <em>et al.</em> (2013)</a> and the consensus dent genetic map in <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Giraud <em>et al.</em> (2014)</a>. We considered a quantitative agronomic trait (<em>e.g.</em>, grain yield) implying <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-143.gif"/></noscript></span></span> QTL with known biallelic effects <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-144.gif"/></noscript></span></span> sampled from <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-145.gif"/></noscript></span></span></p></div><div id="sec-9" class="subsection"><h4>Simulation experiment 1: validation of UCPC:</h4><p id="p-24">In order to validate the derivations for progeny (co)-variances and UCPC method in case of four-way crosses for DH and RIL progeny for selfing generations <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-146.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), we randomly generated 100 four-way crosses out of the 57 elite lines. For each cross, a set of 500 QTL was randomly sampled among the 40,478 SNP markers across the genome to generate the agronomic trait. We also considered the first parent (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-147.gif"/></noscript></span></span>) contributions: genome-wide (C) and at favorable alleles (<span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-148.gif"/></noscript></span></span>). On one hand, we used algebraic formulas to predict the mean and (co)-variances for trait and contributions before selection within each cross (derivation). On the other hand, 50,000 DH or RIL progeny genotypes were simulated per cross at every selfing generation and the empirical mean and (co)-variances before selection were estimated (<em>in silico</em>). For <em>in silico</em> simulations, crossover positions were determined using recombination rates obtained with Haldane’s function (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Haldane 1919</a>). The correlated response to selection on <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-149.gif"/></noscript></span></span> contributions after selecting the 5% upper fraction of progeny for the agronomic trait were either predicted using UCPC (derivation) or estimated after a threshold selection (<em>in silico</em>). The correspondence between predictors was assessed by the squared linear correlation and the mean squared difference between predicted (derivation) and empirical (<em>in silico</em>) values.</p></div><div id="sec-10" class="subsection"><h4>Simulation experiment 2: evaluation of different multi-parental crossing schemes between donor and elite lines:</h4><p id="p-25">We used UCPC to address the question of the best crossing scheme between a given genetic resource (donor <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-150.gif"/></noscript></span></span>, <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>), and elite lines. We identified the crossing scheme that maximized the short term expected genetic gain and evaluated donor genome contributions to the selected fraction of progeny. For this, we set up a simulation study where, at each iteration, an elite population of 25 lines was randomly sampled out of the 57 elite lines. Further, 500 QTL were sampled among monomorphic and polymorphic markers in the elite population in order to conserve the frequency of monomorphic loci observed on 40,478 SNPs in the entire elite population. At each iteration, 100 intra-elite two-way crosses, backcrosses, and three-way crosses were randomly sampled as benchmark. Their progeny mean (<span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-151.gif"/></noscript></span></span>) and progeny standard deviation (<span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-152.gif"/></noscript></span></span>) for the agronomic trait were predicted by Equation 1 and 2, respectively.</p><p id="p-26">Within each iteration, 216 donor genotypes were constructed to cover a wide spectrum of donors in terms of performance and originality compared to the elite population. We defined three tuning parameters that reflect the proportions of six classes of QTL (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dudley 1984</a>) defined by the polymorphism between the donor and the elite population (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). All possible combinations of the three tuning parameters varying from 0 to 1 with steps of 0.2 were considered. For instance, among the favorable QTL in the elite population (classes I and J, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), in the donor genome these QTL were randomly assigned to be favorable or unfavorable with probability I/(I+J) or J/(I+J), respectively. This was done similarly for all classes in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>. For each donor, we considered the simulated agronomic trait together with the donor genome contributions genome-wide (C) and at favorable alleles (<span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-153.gif"/></noscript></span></span>). We defined the genetic gap with the elite population as the difference between donor and mean elite genetic values. The originality of the donor was defined as its mean pairwise modified Rogers distance (MRD) with elite lines.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493002/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493002/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Classes of quantitative trait loci (QTL) and tuning parameters considered for simulating the donors. The favorable allele at QTL is denoted (+) and the unfavorable is denoted (-). A polymorphic QTL in the elite population is denoted (+/−)</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">For all possible 25 two-way crosses, 600 three-way crosses and 25 backcrosses between every donor and the elite population we predicted the progeny mean (μ) and the progeny standard deviation (σ) of each trait (Equation 1 and Equation 2) and the covariances between agronomic trait and contributions (<span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-154.gif"/></noscript></span></span>, Equation 4a and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-155.gif"/></noscript></span></span>, Equation 4b). We defined the post-selection mean for the agronomic trait using Equation 5 with selection intensity i corresponding to a selection pressure of 5%. For comparison between iterations, we subsequently standardized the UC for the agronomic trait based on the elite population by <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-156.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-157.gif"/></noscript></span></span> is the mean and <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-158.gif"/></noscript></span></span> the genetic standard deviation of the elite population. After selection on the agronomic trait, the correlated response on donor contributions was estimated using Equation 6 a-c. Finally, for each type of cross (two-way, three-way and backcrosses) and each donor, we identified the cross that maximized the expected genetic gain for the agronomic trait (<span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-159.gif"/></noscript></span></span>).</p></div></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-31">Simulations were based on genotypic maize data and genetic map deposited in File S4 at figshare. All simulations have been realized using R coding language (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">R Core Team 2017</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7405892">https://doi.org/10.25387/g3.7405892</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Simulation experiment 1: validation of UCPC</h3><p id="p-32">Predictions from the analytical derivations (Equation 1, 2, 4a, 4b, 5, 6a, 6b) showed a high correspondence with empirical results from <em>in silico</em> simulations for the 100 DH-1 families (DH lines after <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-160.gif"/></noscript></span></span>, <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). The predicted progeny variance from derivations and from <em>in silico</em> simulations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A-C</a>) as well as the covariances between the agronomic trait and parent contributions (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3D-E</a>) showed squared correlations above 0.96. Predicted and simulated post-selection mean of the agronomic trait as well as predicted and simulated post-selection parental genome contributions showed correlations above 0.9 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3F-H</a>) (R<sup>2</sup> = 1.000 for Trait, R<sup>2</sup> = 0.900 for C and R<sup>2</sup> = 0.946 for <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-161.gif"/></noscript></span></span>). Validations for RIL and DH progeny derived from more selfing generations are presented in File S2.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between predicted (derivation) and empirical (in silico) moments of the progeny distributions from 100 four-way crosses consisting of 50,000 DH-1 simulated progeny. Moments shown are (A) variance for the agronomic trait , (B) variance for the genome-wide contribution , (C) variance of the contribution at favorable alleles , (D) covariance between agronomic trait and genome-wide contribution , (E) covariance between agronomic trait and contribution at favorable alleles , (F) post-selection mean for the agronomic trait , (G) post-selection mean for the genome-wide contribution , and (H) post-selection mean for the contribution at favorable alleles . Squared correlations between predicted and empirical values are given within each plot." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315272732" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Comparison between predicted (derivation) and empirical (&lt;em&gt;in silico&lt;/em&gt;) moments of the progeny distributions from 100 four-way crosses consisting of 50,000 DH-1 simulated progeny. Moments shown are (A) variance for the agronomic trait &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-162&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (B) variance for the genome-wide contribution &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-163&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (C) variance of the contribution at favorable alleles &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-164&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (D) covariance between agronomic trait and genome-wide contribution &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-165&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (E) covariance between agronomic trait and contribution at favorable alleles &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-166&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (F) post-selection mean for the agronomic trait &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-167&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (G) post-selection mean for the genome-wide contribution &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-168&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and (H) post-selection mean for the contribution at favorable alleles &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-169&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Squared correlations between predicted and empirical values are given within each plot.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1469/F3.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1469/F3.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493121" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Comparison between predicted (derivation) and empirical (<em>in silico</em>) moments of the progeny distributions from 100 four-way crosses consisting of 50,000 DH-1 simulated progeny. Moments shown are (A) variance for the agronomic trait <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-162.gif"/></noscript></span></span>, (B) variance for the genome-wide contribution <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-163.gif"/></noscript></span></span>, (C) variance of the contribution at favorable alleles <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-164.gif"/></noscript></span></span>, (D) covariance between agronomic trait and genome-wide contribution <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-165.gif"/></noscript></span></span>, (E) covariance between agronomic trait and contribution at favorable alleles <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-166.gif"/></noscript></span></span>, (F) post-selection mean for the agronomic trait <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-167.gif"/></noscript></span></span>, (G) post-selection mean for the genome-wide contribution <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-168.gif"/></noscript></span></span>, and (H) post-selection mean for the contribution at favorable alleles <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F3/embed/mml-math-169.gif"/></noscript></span></span>. Squared correlations between predicted and empirical values are given within each plot.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Simulation experiment 2</h3><div id="sec-15" class="subsection"><h4>Intra-elite multi-parental crosses: a benchmark:</h4><p id="p-34">Considering only the elite population generated at each iteration, the mean average performance over 20 iterations was <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-170.gif"/></noscript></span></span>0.067 ± 1.009 and the mean elite standard deviation was <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-171.gif"/></noscript></span></span>0.748 ± 0.107. We observed (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) that intra-elite three-way crosses generated more progeny standard deviation (<span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-172.gif"/></noscript></span></span>) (0.576 ± 0.034) than two-way crosses (0.510 ± 0.026) and backcrosses (0.442 ± 0.022). In terms of progeny mean (<span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-173.gif"/></noscript></span></span>), differences were not significant between types of crosses. The gain in <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-174.gif"/></noscript></span></span> yielded a higher usefulness criterion (<span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-175.gif"/></noscript></span></span>) with three-way crosses (1.599 <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-176.gif"/></noscript></span></span> ± 0.317) than two-way crosses (1.461 <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-177.gif"/></noscript></span></span> ± 0.268). On the contrary, when only considering the best cross in terms of gain for the agronomic trait (<span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-178.gif"/></noscript></span></span>), two-way crosses led to a higher UC (3.115 <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-179.gif"/></noscript></span></span> ± 0.362) than three-way crosses (2.876 <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-180.gif"/></noscript></span></span> ± 0.420) or backcrosses (2.804 <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-181.gif"/></noscript></span></span> ± 0.377).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493155/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493155/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Intra-Elite crosses predicted progeny mean (<span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-182.gif"/></noscript></span></span>), progeny standard deviation (<span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-183.gif"/></noscript></span></span>) and resulting expected genetic gain <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-184.gif"/></noscript></span></span> with a selection pressure of 5%, once averaged over all crosses (<span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-185.gif"/></noscript></span></span>) and for the best cross identified (<span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/T3/embed/mml-math-186.gif"/></noscript></span></span>). For all parameters the mean (± SD) over 20 iterations is given</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Donor genome contribution in multi-parental crosses:</h4><p id="p-35">For each simulated donor, we identified the two-way cross, three-way cross and backcross that maximized the UC for the agronomic trait (<span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-191.gif"/></noscript></span></span>). Those crosses are denoted as best crosses in the following. We analyzed the relationship between donor contributions to the selected progeny of the best crosses and the genetic gap between the donor and the mean elite population (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The genome-wide contribution, the contribution at favorable alleles, and the contribution at unfavorable alleles are shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figures 4A</a>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">4B</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">4C</a>, respectively. For a given donor, the genome-wide donor contribution after selection was higher in the best two-way crosses than in the best three-way crosses or backcrosses. For illustrative purposes, we differentiated five cases from the worst donor carrying only unfavorable alleles at QTL (case 0) to the best donor carrying favorable alleles at all QTL (case 4). Starting from case 0, the selection tended to eliminate most of the donor genome in progeny until a lower bound (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>, 27.1% for the best two-way cross, 6.7% for the best three-way cross and 6.3% for the best backcross). Very badly performing donors (case 1; genetic gap <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-192.gif"/></noscript></span></span>), <em>i.e.</em>, carrying favorable alleles at maximum 180 QTL, had little chance to pass their favorable alleles to the selected progeny (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>, <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-193.gif"/></noscript></span></span> ≤ 4.5% in the best two-way cross, <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-194.gif"/></noscript></span></span> ≤ 1.9% in the best three-way cross and <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-195.gif"/></noscript></span></span> ≤ 1.7% in the best backcross). When the performance of the donor increased (case 2; <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-196.gif"/></noscript></span></span> genetic gap <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-197.gif"/></noscript></span></span>), a higher portion of the donor genome was retained in the selected progeny (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>). With an increased number of favorable alleles (case 2), genome-wide donor contribution increased linearly with the genetic gap due to both, the selection of favorable alleles from the donor (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4B</a>) and the linkage drag with unfavorable alleles (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4C</a>). This linear trend continued until the donor had mainly favorable alleles (case 3; <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-198.gif"/></noscript></span></span> genetic gap). In case 3, we observed a linear increase of donor contribution at favorable alleles (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4B</a>). A correlated decrease of donor contribution at unfavorable alleles was observed at a nearly constant genome-wide contribution. Finally, in case 4, the genome-wide contribution was equal to an upper bound limit (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4A</a>, 72.6% for the best two-way cross, 42.9% for the best three-way cross and 43.5% for the best backcross).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Donor contribution to the selected progeny of the best two-way cross (Donor*Elite), the best three-way cross ((Donor*Elite1)*Elite2) and the best backcross ((Donor*Elite1)*Elite1), depending on the genetic gap between donor line and the elite population. Each data point corresponds to the progeny of the best cross and is colored depending on the type of cross. (A) Donor genome-wide contribution after selection , (B) donor genome contribution at favorable alleles after selection  and (C) donor genome contribution at unfavorable alleles after selection . Numbers (0, 1, 2, 3, 4) correspond to illustrative cases based on genetic gap referred in the text. Illustrative cases 0 and 4 correspond to the worst and best donor respectively. Illustrative cases 1, 2, 3 are delimited by genetic gap values -5, 5 as represented by the vertical dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315272732" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Donor contribution to the selected progeny of the best two-way cross (Donor*Elite), the best three-way cross ((Donor*Elite1)*Elite2) and the best backcross ((Donor*Elite1)*Elite1), depending on the genetic gap between donor line and the elite population. Each data point corresponds to the progeny of the best cross and is colored depending on the type of cross. (A) Donor genome-wide contribution after selection &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-199&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (B) donor genome contribution at favorable alleles after selection &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-200&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and (C) donor genome contribution at unfavorable alleles after selection &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-201&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Numbers (0, 1, 2, 3, 4) correspond to illustrative cases based on genetic gap referred in the text. Illustrative cases 0 and 4 correspond to the worst and best donor respectively. Illustrative cases 1, 2, 3 are delimited by genetic gap values -5, 5 as represented by the vertical dashed lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1469/F4.medium.gif" width="440" height="113"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1469/F4.medium.gif" width="440" height="113"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493158" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Donor contribution to the selected progeny of the best two-way cross (Donor*Elite), the best three-way cross ((Donor*Elite1)*Elite2) and the best backcross ((Donor*Elite1)*Elite1), depending on the genetic gap between donor line and the elite population. Each data point corresponds to the progeny of the best cross and is colored depending on the type of cross. (A) Donor genome-wide contribution after selection <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F4/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F4/embed/mml-math-199.gif"/></noscript></span></span>, (B) donor genome contribution at favorable alleles after selection <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F4/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F4/embed/mml-math-200.gif"/></noscript></span></span> and (C) donor genome contribution at unfavorable alleles after selection <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F4/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F4/embed/mml-math-201.gif"/></noscript></span></span>. Numbers (0, 1, 2, 3, 4) correspond to illustrative cases based on genetic gap referred in the text. Illustrative cases 0 and 4 correspond to the worst and best donor respectively. Illustrative cases 1, 2, 3 are delimited by genetic gap values -5, 5 as represented by the vertical dashed lines.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h4>Comparison of genetic gain among multi-parental crossing schemes:</h4><p id="p-37">When the donor outperformed the elite population, the best two-way cross was more likely yielding a higher genetic gain than the best three-way cross or backcross (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). On the contrary, when the donor underperformed the elite population, the best three-way cross and backcross yielded a higher genetic gain than the best two-way cross. The higher progeny standard deviation (<span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-202.gif"/></noscript></span></span>) in the best two-way cross compared to the best three-way cross or backcross (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>) did not compensate the loss in progeny mean (<span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-203.gif"/></noscript></span></span>) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>) in the best two-way cross. We observed that the type of cross maximizing the <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-204.gif"/></noscript></span></span> (<em>i.e.</em>, two-way cross, three-way cross or backcross) depended only on the performance of the donor, whatever the mean genetic distance with the elite population (<em>results not shown</em>). A similar comparison between three-way crosses and backcrosses showed that the best backcross yielded similar <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-205.gif"/></noscript></span></span> (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>) but lower <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-206.gif"/></noscript></span></span> than the best three-way cross (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>), especially when the donor had a genetic value close to the best elite lines. This resulted in a slightly higher expected genetic gain in three-way crosses compared to backcrosses (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the best two-way cross (Donor*Elite), the best three-way cross ((Donor*Elite1)*Elite2) and the best backcross ((Donor*Elite1)*Elite1), depending on the genetic gap (x-axis) with the elite population. Each data point corresponds to the progeny of the best cross and is colored depending of the type of cross. Comparison for the (A) expected genetic gain , (B) progeny mean (), and (C) progeny standard deviation ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315272732" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the best two-way cross (Donor*Elite), the best three-way cross ((Donor*Elite1)*Elite2) and the best backcross ((Donor*Elite1)*Elite1), depending on the genetic gap (x-axis) with the elite population. Each data point corresponds to the progeny of the best cross and is colored depending of the type of cross. Comparison for the (A) expected genetic gain &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-207&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (B) progeny mean (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-208&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), and (C) progeny standard deviation (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-209&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1469/F5.medium.gif" width="440" height="116"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/5/1469/F5.medium.gif" width="440" height="116"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1469/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493053" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">Comparison of the best two-way cross (Donor*Elite), the best three-way cross ((Donor*Elite1)*Elite2) and the best backcross ((Donor*Elite1)*Elite1), depending on the genetic gap (x-axis) with the elite population. Each data point corresponds to the progeny of the best cross and is colored depending of the type of cross. Comparison for the (A) expected genetic gain <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F5/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F5/embed/mml-math-207.gif"/></noscript></span></span>, (B) progeny mean (<span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F5/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F5/embed/mml-math-208.gif"/></noscript></span></span>), and (C) progeny standard deviation (<span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F5/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/F5/embed/mml-math-209.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>Usefulness criterion for quantitative traits in multi-parental crosses</h3><p id="p-39">Accurate predictors of progeny variance accounting for the map position of loci and linkage phase of alleles in parents have been recently derived for biparental crosses (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> 2017b</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Osthushenrich <em>et al.</em> 2017</a>). Nonetheless, breeders might use multi-parental crosses implying more than two parents to combine best alleles segregating in the breeding population. Therefore, we extended derivations given by <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> (2017b)</a> for two-way crosses to four-way crosses by accounting for linkage disequilibrium between pairs of parental lines. We validated the derived genetic variance of RIL and DH progeny of four-way crosses by simulations (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, File S2). As expected, the formula for four-way crosses reduces to the one given by <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Lehermeier <em>et al.</em> (2017b)</a> in case of two-way crosses (File S1). The results from our simulations showed that, considering elite material only, three-way crosses generate on average more variance than two-way crosses or backcrosses, resulting in higher genetic gain (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Nevertheless, the best possible cross (<em>i.e.</em>, maximizing the expected genetic gain) was a two-way cross for most iterations (90%). This can be explained by the fact that crossing the two best elite lines generates more genetic gain than crossing them to a third less performant elite line, despite a potential gain in progeny variance. Notice that we considered only one polygenic agronomic trait but three-way crosses can be more advantageous for bringing complementary alleles for several traits. Under the formulated assumptions and with available marker effects (see discussion below), the general formula to predict mean and variance of four-way cross progeny makes it possible to identify the multi-parental cross that maximizes a given multi-trait selection objective (see discussion below) without requiring computationally intensive <em>in silico</em> simulations of progeny. The generalization to several generations of selfing for RIL progeny enables in addition to differentiate crosses releasing differently the variance in time (File S2). The presented formula for four-way crosses can also be applied to crosses of two heterozygous parents by considering its phased genotypes as four separate parents. Doing so, our approach can be adapted for heterozygous plant varieties that are common in perennial species and for crosses with hybrids, as well as for animal breeding where the prediction of Mendelian sampling variance can be very useful for mating decisions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bonk <em>et al.</em> 2016</a>).</p></div><div id="sec-20" class="subsection"><h3>Parental contributions in multi-parental crosses under selection</h3><p id="p-40"><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Frisch and Melchinger (2007)</a> derived the expected variance of parental contribution before selection in fully homozygote progeny accounting for linkage disequilibrium between loci assuming a biparental cross and considering only polymorphic loci. In this study, we proposed an original way to follow parental genome contribution to the selected fraction of progeny in multi-parental crosses, namely UCPC. It is grounded in a normal approximation of the probability mass function of parental contribution (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Hill 1993</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Frisch and Melchinger 2007</a>) and progeny variance derivations. In the specific case of DH lines derived from two-way crosses or backcrosses and considering one chromosome of 100cM, our prediction of parental genome contribution variance converged to the one of <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Frisch and Melchinger (2007)</a> when increasing the number of loci (File S3). However, the previous literature did not combine parental contributions with quantitative traits. Our original multivariate UCPC approach enables to predict the covariance between parental genome contributions and traits of economic interest. Based on multivariate selection theory, UCPC predicts the expected realized parental genome contribution after selection on traits of interest. It allows to follow parental genome contribution inheritance over generations and provides the likelihood of reaching a specific level of parental contribution while prescreening the most performing lines. Such information can guide breeders and researchers to determine the minimal number of progeny to derive from a cross between a donor and one or several elite lines so that the expected donor contribution after selection can reach a targeted value.</p><p id="p-41">Predicted genome-wide donor contribution to progeny after selection was bounded to a minimum in case of the worst donor and a maximum in case of the best donor. In line with the predicted distribution of parental genome contribution before selection obtained in maize by <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Frisch and Melchinger (2007)</a>, these results show that in one selection cycle with a reasonable selection intensity (<em>e.g.</em>, 5%) it is unlikely to get completely rid of unfavorable parental alleles. Parental genome contribution was bounded in selected progeny due to the low probability of combining all alleles from a single parent. Note that UCPC also allows to follow the contribution of parents to progeny performance by defining a vector of effects based on parental performance marker effects. For instance, considering <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-210.gif"/></noscript></span></span> enables to follow the first parent contribution to progeny performance.</p></div><div id="sec-21" class="subsection"><h3>Recommendations for donor by elites crosses</h3><p id="p-42">Using UCPC, we addressed the question of polygenic trait introgression from an inbred donor to inbred elite recipients with a focus on common plant breeding crossing schemes: two-way, three-way and backcrosses. We assumed that the objective was to derive in one selection cycle an inbred progeny that combined donor favorable alleles in a performing elite background. Such progeny can be used as parental lines for new crosses in order to quickly introgress new favorable alleles in a breeding program. Such a short term vision of genetic resource integration can be complementary to a longer term pre-breeding approach using exotic material (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bernardo 2009</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gorjanc <em>et al.</em> 2016</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yu <em>et al.</em> 2016</a>). As expected, donors underperforming the elite population (inferior donor) yielded a higher genetic gain when complemented by two elite lines in three-way crosses or by twice an elite line in backcrosses rather than by a single elite line in two-way crosses. In this case, there is an advantage of crossing schemes involving, on average before selection, only one fourth of the donor genome instead of half of the donor genome as it would be the case for a two-way cross. On the contrary, two-way crosses were more adapted to donors outperforming the elite population. If the donor showed a similar performance level as the elite lines, no general rule could be drawn. In such a case, we recommend to identify the best crossing scheme by predicting every potential cross using the UCPC approach. As expected under a lower dilution of donor alleles into elite alleles in two-way crosses compared to three-way crosses or backcrosses, the predicted genome-wide donor contribution to selected progeny was higher in the best two-way cross than in the best three-way cross or the best backcross (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4A</a>).</p><p id="p-43">We observed for a polygenic trait that, despite a lower competition between donor and elite favorable alleles, backcrosses were not significantly superior to three-way crosses for maintaining higher donor contribution at favorable alleles (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4B</a>). In addition, backcrosses generated less progeny variance (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5C</a>) but similar progeny mean than three-way crosses, resulting in a lower genetic gain (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5A</a>). This observation depends on the elite population considered. For instance, it might not hold if one unique elite line highly outperforms all other lines. More generally, while backcrosses only combine donor alleles with alleles of one elite parent, three-way crosses combine donor alleles with alleles of two complementary elite lines and are thus closer to material generated at the same time using two-way crosses in routine breeding. For these reasons, we suggest that three-way crosses should be preferred over backcrosses for polygenic trait introgression in elite germplasm. Our results support <em>a posteriori</em> the crossing strategy adopted in the Germplasm Enhancement of Maize project (GEM, <em>e.g.</em>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Goodman 2000</a>). In GEM, maize exotic material has been introgressed into maize elite private lines using three-way crosses implying two different private partners. With the possibility to efficiently predict the progeny distribution of three-way crosses (UCPC), the best crossing partners can be identified to meet the targeted outcome in short time which allows to fully profit of the advantages of three-way crosses.</p></div><div id="sec-22" class="subsection"><h3>Multivariate selection for agronomic traits and parental contributions</h3><p id="p-44">We observed that badly performing donors had little chance to pass their favorable alleles to progeny selected for their agronomic trait performance. This is a consequence of the negative covariance between the performance for the trait and donor contribution in case of an inferior donor (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). To prevent this loss of original alleles, we could account for such tension in the multivariate context, for instance by applying a truncation on donor contribution before selecting for the trait using the truncated multivariate normal theory (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Horrace 2005</a>) or vice versa. Otherwise, selection on donor contribution and the agronomic trait can be applied jointly by building a selection index, which is promising to balance short term genetic gain and long term genetic diversity (<em>i.e.</em>, selection on donor contribution) according to specific pre-breeding strategies.</p><p id="p-45">More generally, the multivariate context provides the opportunity to deal with several quantitative traits on which selection is directly or indirectly applied. Further traits for which genome-wide estimated marker effects or QTL effects are available can be considered. For external genetic resource utilization, it enables to introgress secondary traits such as polygenic tolerances to biotic or abiotic stresses (<em>e.g.</em>, drought tolerance), while agronomic flaws (<em>e.g.</em>, plant lodging) can be counter-selected using threshold selection. Recently it has been shown by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akdemir <em>et al.</em> (2018)</a> how the improvement of multiple traits can be addressed with multi-objective optimized breeding strategies.</p></div><div id="sec-23" class="subsection"><h3>Practical implementation of UCPC in breeding</h3><p id="p-46">In practice, marker effects estimated with whole-genome regression models can be used in lieu of QTL effects that are unknown. Such effects should be estimated on a proper training population mixing both elite lines and original genetic resources. Marker effects can be estimated using Bayesian Ridge Regression as suggested in <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lehermeier <em>et al.</em> (2017a</a>; <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">b</a>) to derive an unbiased estimator of progeny variance (PMV: posterior mean variance). In our simulation study, we considered only biallelic QTL effects. As we formulated a multi-allelic model, population-specific additive effects could be considered straightforwardly. Considering that the donor might have a different origin than the elite lines (<em>e.g.</em>, other heterotic group in hybrid crops), it might be of interest to use parental specific effects estimated by <em>e.g.</em>, multivariate QTL mapping (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Giraud <em>et al.</em> 2014</a>) or genome-wide prediction models (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lehermeier <em>et al.</em> 2015</a>). UCPC relies on individual marker effects but the computation of the variance in the progeny accounts for collinearity among markers, <em>i.e.</em>, considers haplotype transmission. We therefore expect that inaccuracies in marker effects estimates will affect UCPC to a limited extent, but this warrant specific investigations as suggested by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Müller <em>et al.</em> (2018)</a>.</p><p id="p-47">Our approach is totally generic and can deal with any information on the position and the effect of QTL. However, main assumptions should be discussed at this point. We assumed known true genetic positions of QTL and no interference during crossover formation to derive recombination frequencies (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Haldane 1919</a>). In practice, the precision of recombination frequency estimates is a function of the available mapping information and the frequency of interference. Furthermore, recombination frequency might vary among the same species (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bauer <em>et al.</em> 2013</a>) impairing the accuracy of variance prediction. To limit this risk we suggest to use a multi-parental consensus map (<em>e.g.</em>, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Giraud <em>et al.</em> 2014</a>). The effect of genetic map inaccuracies on progeny co-variances prediction requires further investigations. Furthermore, derivations assumed no selection before developing progeny. However, selecting progeny from which to derive DH lines is likely in practice. This can involve voluntary molecular prescreening for disease resistance (<em>e.g.</em>, during selfing generations) or practical limitations (<em>e.g.</em>, originating from low DH induction rates). If the genetic correlation between those traits and the traits considered within UCPC is null, the derived progeny distribution and UC for the four-way crosses will still hold.</p><p id="p-48">The derived formula for progeny mean and variance holds for mono- and oligo-genic traits, whereas the usefulness criterion underlying UCPC uses normal distribution properties. When considering traits involving a sufficient number of underlying QTL, as it is the case for most agronomic traits and parental genome contributions, this assumption of normality is likely guaranteed by the central limit theorem. If only a limited number of known major QTL should be introgressed from a donor, an allele pyramiding strategy will be more suitable (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hospital and Charcosset 1997</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Charmet <em>et al.</em> 1999</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Servin <em>et al.</em> 2004</a>). Furthermore, the predicted cross value (PCV) as recently suggested by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Han <em>et al.</em> (2017)</a> can be applied in this context and could be extended to multi-parental crosses considering our derivation of progeny variance.</p><p id="p-49">We presented an IBD definition of parental genome contributions using a multi-allelic approach. The multi-allelic coding yields covariance matrices that are four times larger compared to using a biallelic coding. In practice, to obtain a less computationally intensive solution, the genotyping matrix can be reduced to a bi-allelic coding which yields an identity by state (IBS) parental genome contribution that informs on the sequence similarity between one parent and progeny (see File S3). However, in such a case parental contributions do not sum up to one and it cannot be accounted for multi-allelic (<em>i.e.</em>, haplotypic) effects. For biparental crosses (<em>i.e.</em>, two-way and backcrosses), an IBS approach (File S3) considering only polymorphic markers homogeneously covering the genome can be used as an approximation of the IBD contribution.</p></div><div id="sec-24" class="subsection"><h3>Future research directions</h3><p id="p-50">UCPC is opening several future research directions. We illustrated the use of UCPC for a simple donor introgression problem but it can be extended to more complex problematics commonplace in breeding. For instance, UCPC can be applied to evaluate the interest of introgressing several donors, <em>e.g.</em>, evaluate the interest of combining alleles from two donors (<span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-211.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-212.gif"/></noscript></span></span>) with elites (<span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-213.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-214.gif"/></noscript></span></span>) in <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-215.gif"/></noscript></span></span> or (<span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1469/embed/mml-math-216.gif"/></noscript></span></span>.</p><p id="p-51">Mating design optimizations, <em>i.e.</em>, finding an optimized list of crosses to realize each year, accounting for a compromise between short and long term genetic gain have been investigated using two-way crosses and parental means as predictor of the expected gain and the inbreeding rate in the next generation (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">De Beukelaer <em>et al.</em> 2017</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gorjanc <em>et al.</em> 2018</a>). Applying UCPC within the context of mating design optimization would enable to account for parental complementarity through the use of progeny variation, <em>i.e.</em>, within cross variance, as proposed by <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Shepherd and Kinghorn (1998)</a>, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir and Sánchez (2016)</a> and <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Müller <em>et al.</em> (2018)</a>. Furthermore, UCPC would enable to use parental contribution to the selected fraction of progeny to predict the realized inbreeding in the next generation. We conjecture that considering the realized parental genome contribution together with the usefulness criterion in UCPC is promising for mating design optimization to manage short and long term genetic gain in breeding programs. Future research will also be needed to investigate the use of multi-parental crosses in mating design optimizations. Hereby, UCPC that efficiently predicts the progeny distribution of crosses with up to four parents will represent a good starting point for further research.</p></div><div id="sec-25" class="subsection"><h3>Conclusions</h3><p id="p-52">We developed, validated and illustrated the usefulness criterion parental contribution (UCPC) that evaluates the interest of multi-parental crosses based on the expected genetic gain (UC) and the parental contributions (PC) in the next generation. UCPC allows to (i) predict the progeny variance of four-way crosses accounting for linkage disequilibrium and to (ii) follow all parental genome contributions to the selected progeny to evaluate the interest of a cross regarding an objective that is a function of the expected performance and the diversity in the selected progeny. Illustration of the use of UCPC in the context of polygenic trait introgression from a donor to elite recipients enabled to draw some major recommendations. As expected, three-way crosses and backcrosses were more adapted to donors underperforming the elite population (inferior donor) while two-way crosses were more adapted to donors outperforming the elite population. We also suggested that three-way crosses should be preferred over backcrosses for polygenic traits introgression. Furthermore, we highlighted the importance of a compromise between UC and PC in case of an inferior donor.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">The authors thank the Amaizing program for genotypes used in simulations. This research was funded by RAGT 2n and the ANRT CIFRE Grant n° 2016/1281 for AA.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-54">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7405892">https://doi.org/10.25387/g3.7405892</a>.</p></li><li class="fn" id="fn-9"><p id="p-55"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-30"><span class="received-label">Received </span>November 30, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-27"><span class="accepted-label">Accepted </span>February 27, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Allier <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.1" data-doi="10.3389/fgene.2016.00210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient Breeding by Genomic Mating.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2016.00210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2016.00210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2016.00210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.2" data-doi="10.1038/s41437-018-0147-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsche-Neto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isidro-Sánchez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Multi-objective optimized genomic breeding strategies for sustainable food improvement.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-018-0147-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-018-0147-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-018-0147-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.3" data-doi="10.1186/gb-2013-14-9-r103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Intraspecific variation of recombination rate in maize.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-9-r103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR103%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r103%26rft_id%253Dinfo%253Apmid%252F24050704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24050704&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1469.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.4" data-doi="10.2135/cropsci2006.01-0057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Number and Fitness of Selected Individuals in Marker-Assisted and Phenotypic Recurrent Selection.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1972</span>–<span class="cit-lpage">1980</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2006.01-0057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D46%26rft.issue%253D5%26rft.spage%253D1972%26rft.epage%253D1980%26rft.atitle%253DNumber%2Band%2BFitness%2Bof%2BSelected%2BIndividuals%2Bin%2BMarker-Assisted%2Band%2BPhenotypic%2BRecurrent%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.01-0057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.01-0057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000240821800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.5" data-doi="10.2135/cropsci2008.08.0452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomewide Selection for Rapid Introgression of Exotic Germplasm in Maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.08.0452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D49%26rft.issue%253D2%26rft.spage%253D419%26rft.epage%253D425%26rft.atitle%253DGenomewide%2BSelection%2Bfor%2BRapid%2BIntrogression%2Bof%2BExotic%2BGermplasm%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264654000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.6" data-doi="10.2135/cropsci2014.01.0088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomewide Selection of Parental Inbreds: Classes of Loci and Virtual Biparental Populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">2586</span>–<span class="cit-lpage">2595</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.01.0088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D2586%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.01.0088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.01.0088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.7" data-doi="10.1534/genetics.116.194449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Beukelaer</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badke</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fack</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Moving beyond managing realized genomic relationship in long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1127</span>–<span class="cit-lpage">1138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.194449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.194449%26rft_id%253Dinfo%253Apmid%252F28381589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8yLzExMjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.5.1469.8"><div class="cit-metadata unstructured">Bijma, P., 2000 Long-term genetic contributions: prediction of rates of inbreeding and genetic gain in selected populations (Doctoral dissertation). Veenendaal, The Netherlands.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.9" data-doi="10.1186/s12711-016-0214-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reichelt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teuscher</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segelke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinsch</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mendelian sampling covariability of marker effects and genetic values.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0214-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0214-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0214-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.10" data-doi="10.1007/s001220051318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charmet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perretant</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sourdille</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Marker-assisted recurrent selection for cumulating additive and interactive QTLs in recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220051318</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D99%26rft.spage%253D1143%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084014900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.11" data-doi="10.2135/cropsci1984.0011183X002400020034x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">A Method of Identifying Lines for Use in Improving Parents of a Single Cross.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">357</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1984.0011183X002400020034x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DDudley%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D355%26rft.epage%253D357%26rft.atitle%253DA%2BMethod%2Bof%2BIdentifying%2BLines%2Bfor%2BUse%2Bin%2BImproving%2BParents%2Bof%2Ba%2BSingle%2BCross%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1984.0011183X002400020034x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1984.0011183X002400020034x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984SJ72400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1469.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics. Ed. 4th.</span> <span class="cit-publ-name">Pearson</span>. <span class="cit-publ-loc">Harlow, England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.13" data-doi="10.2135/cropsci1999.3951295x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Comparison of Selection Strategies for Marker-Assisted Backcrossing of a Gene.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1295</span>–<span class="cit-lpage">1301</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1999.3951295x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DFrisch%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D1295%26rft.epage%253D1301%26rft.atitle%253DComparison%2Bof%2BSelection%2BStrategies%2Bfor%2BMarker-Assisted%2BBackcrossing%2Bof%2Ba%2BGene%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.3951295x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.3951295x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000082813800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.14" data-doi="10.1534/genetics.106.065433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Variance of the Parental Genome Contribution to Inbred Lines Derived From Biparental Crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.065433</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.065433%26rft_id%253Dinfo%253Apmid%252F17409089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzEvNDc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1469.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Théorie de la sélection en amélioration des plantes</span>, <span class="cit-publ-name">Masson</span>, <span class="cit-publ-loc">Paris</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.16" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Large Maize (Zea mays L.) SNP Genotyping Array: Development and Germplasm Genotyping, and Genetic Mapping to Compare with the B73 Reference Genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0028334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1469.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.17" data-doi="10.1534/genetics.114.169367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraud</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage Disequilibrium with Linkage Analysis of Multiline Crosses Reveals Different Multiallelic QTL for Hybrid Performance in the Flint and Dent Heterotic Groups of Maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.169367</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169367%26rft_id%253Dinfo%253Apmid%252F25271305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1469.18"><div class="cit-metadata unstructured">Goodman M. M., 2000 Incorporation of exotic germplasm into elite maize lines: Maximizing favorable effects of the exotic source. Theor. Pop. Biol.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.19" data-doi="10.1186/s12864-015-2345-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenko</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Initiating maize pre-breeding programs using genomic selection to harness polygenic variation from landrace populations.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2345-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2345-z%26rft_id%253Dinfo%253Apmid%252F26732811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2345-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26732811&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1469.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.20" data-doi="10.1007/s00122-018-3125-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Optimal cross selection for long-term genetic gain in two-part programs with rapid recurrent genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1953</span>–<span class="cit-lpage">1966</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3125-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D1953%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3125-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3125-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">The combination of linkage values, and the calculation of distances between the loci of linked factors.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D8%26rft.spage%253D299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000200614300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.22" data-doi="10.1534/genetics.116.197095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The Predicted Cross Value for Genetic Introgression of Multiple Alleles.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197095%26rft_id%253Dinfo%253Apmid%252F28122824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS80LzE0MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.23" data-doi="10.1093/oxfordjournals.jhered.a111319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Variation in Genetic Composition in Backcrossing Programs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">213</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.jhered.a111319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D84%26rft.issue%253D3%26rft.spage%253D212%26rft.epage%253D213%26rft.atitle%253DVariation%2Bin%2BGenetic%2BComposition%2Bin%2BBackcrossing%2BPrograms%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a111319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.jhered.a111319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LG87900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.24" data-doi="10.1016/j.jmva.2004.10.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horrace</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Some results on the multivariate truncated normal distribution.</span> <abbr class="cit-jnl-abbrev">J. Multivariate Anal.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmva.2004.10.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMultivariate%2BAnal.%26rft.volume%253D94%26rft.spage%253D209%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmva.2004.10.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmva.2004.10.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Marker-Assisted Introgression of Quantitative Trait Loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">1469</span>–<span class="cit-lpage">1485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHospital%26rft.auinit1%253DF.%26rft.volume%253D147%26rft.issue%253D3%26rft.spage%253D1469%26rft.epage%253D1485%26rft.atitle%253DMarker-Assisted%2BIntrogression%2Bof%2BQuantitative%2BTrait%2BLoci%26rft_id%253Dinfo%253Apmid%252F9383086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Ny8zLzE0NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.26" data-doi="10.1186/1471-2156-14-81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terakami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takada</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction of trait segregation in a progeny population: a case study of Japanese pear (Pyrus pyrifolia).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">81</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-14-81</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-14-81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-14-81&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.27" data-doi="10.1038/nature22971"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peluso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitzer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improved maize reference genome with single-molecule technologies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22971%26rft_id%253Dinfo%253Apmid%252F28605751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22971&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28605751&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1469.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.28" data-doi="10.1534/genetics.115.177394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessment of Genetic Heterogeneity in Structured Plant Populations Using Multivariate Whole-Genome Regression Models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177394%26rft_id%253Dinfo%253Apmid%252F26122758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzEvMzIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.29" data-doi="10.1111/jbg.12268"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic variance estimates: With or without disequilibrium covariances?</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12268</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D134%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12268&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.30" data-doi="10.1534/genetics.117.300403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teyssèdre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetic Gain Increases by Applying the Usefulness Criterion with Improved Variance Prediction in Selection of Crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">1651</span>–<span class="cit-lpage">1661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300403%26rft_id%253Dinfo%253Apmid%252F29038144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy80LzE2NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.31" data-doi="10.2135/cropsci2014.10.0729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prediction of genetic variance in biparental maize populations: Genomewide marker effects <em>vs.</em> mean genetic variance in prior populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1188</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.10.0729</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1181%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.10.0729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.10.0729&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.32" data-doi="10.2135/cropsci2015.01.0030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiede</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PopVar: A Genome-Wide Procedure for Predicting Genetic Variance and Correlated Response in Biparental Breeding Populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2068</span>–<span class="cit-lpage">2077</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.01.0030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2068%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.01.0030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.01.0030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, 2018 Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection. G3 (Bethesda)g3.200091.<span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.34" data-doi="10.1371/journal.pone.0188839"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osthushenrich</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herzog</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic selection of crossing partners on basis of the expected mean and variance of their derived lines.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0188839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0188839</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0188839%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0188839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0188839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1469.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A language and environment for statistical computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.36" data-doi="10.1007/s00122-018-3196-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mary-Huard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genomic selection efficiency and a priori estimation of accuracy in a structured dent maize panel.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">96</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3196-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D132%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3196-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3196-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1469.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnell</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">F1-Leistung und Elternwahl in der Züchtung von Selbstbefruchtern.</span> pp. <span class="cit-fpage">243</span>–<span class="cit-lpage">248</span> in <span class="cit-source">Bericht über die Arbeitstagung der Vereinigung österreichischer Pflanzenzüchter</span>, <span class="cit-publ-name">BAL Gumpenstein</span>, <span class="cit-publ-loc">Gumpenstein, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Genomic Prediction Within and Across Biparental Families: Means and Variances of Prediction Accuracy and Usefulness of Deterministic Equations. G3 (Bethesda)7: 3571–3586.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.39" data-doi="10.1534/genetics.103.023358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mézard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Toward a Theory of Marker-Assisted Gene Pyramiding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.103.023358</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DServin%26rft.auinit1%253DB.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D513%26rft.epage%253D523%26rft.atitle%253DToward%2Ba%2BTheory%2Bof%2BMarker-Assisted%2BGene%2BPyramiding%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.023358%26rft_id%253Dinfo%253Apmid%252F15454561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvNTEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1469.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A tactical approach to the design of crossbreeding programs. In Proceedings of the sixth world congress on genetics applied to livestock production, Armidale</span>, <span class="cit-vol">11–16</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.41" data-doi="10.2135/cropsci1999.0011183X003900020001x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Troyer</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Background of U.S. Hybrid Corn.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">601</span>–<span class="cit-lpage">626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1999.0011183X003900020001x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DTroyer%26rft.auinit1%253DA.%2BF.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D601%26rft.epage%253D626%26rft.atitle%253DBackground%2Bof%2BU.S.%2BHybrid%2BCorn%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.0011183X003900020001x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.0011183X003900020001x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000080943700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.42" data-doi="10.1186/1471-2229-12-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Inghelandt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinant</span>  <span class="cit-name-given-names">J.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping of flowering time and northern corn leaf blight (Setosphaeria turcica) resistance in a vast commercial maize germplasm set.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-12-56</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BInghelandt%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bof%2Bflowering%2Btime%2Band%2Bnorthern%2Bcorn%2Bleaf%2Bblight%2B%2528Setosphaeria%2Bturcica%2529%2Bresistance%2Bin%2Ba%2Bvast%2Bcommercial%2Bmaize%2Bgermplasm%2Bset.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-56%26rft_id%253Dinfo%253Apmid%252F22545925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22545925&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1469.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Marker-Assisted Introgression in Backcross Breeding Programs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1923</span>–<span class="cit-lpage">1932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D144%26rft.issue%253D4%26rft.spage%253D1923%26rft.epage%253D1932%26rft.atitle%253DMarker-Assisted%2BIntrogression%2Bin%2BBackcross%2BBreeding%2BPrograms%26rft_id%253Dinfo%253Apmid%252F8978075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC80LzE5MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.44" data-doi="10.1371/journal.pgen.0020041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medland</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morley</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253De41%26rft.epage%253De41%26rft.atitle%253DAssumption-free%2Bestimation%2Bof%2Bheritability%2Bfrom%2Bgenome-wide%2Bidentity-by-descent%2Bsharing%2Bbetween%2Bfull%2Bsiblings.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020041%26rft_id%253Dinfo%253Apmid%252F16565746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16565746&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1469.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.45" data-doi="10.1007/s10709-008-9301-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Whole genome approaches to quantitative genetics.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-008-9301-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9301-7%26rft_id%253Dinfo%253Apmid%252F18668208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9301-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18668208&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1469.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.46" data-doi="10.2135/cropsci2000.403659x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Variance of Marker Estimates of Parental Contribution to F 2 and BC 1 -Derived Inbreds.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">665</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2000.403659x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D659%26rft.epage%253D665%26rft.atitle%253DVariance%2Bof%2BMarker%2BEstimates%2Bof%2BParental%2BContribution%2Bto%2BF2%2Band%2BBC1-Derived%2BInbreds%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2000.403659x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2000.403659x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000089410400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.47" data-doi="10.1111/jbg.12148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dagnachew</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic contributions and their optimization.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">99</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D132%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.48" data-doi="10.1038/nplants.2016.150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction contributing to a promising global strategy to turbocharge gene banks.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">16150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nplants.2016.150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D16150%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnplants.2016.150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nplants.2016.150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1469.49" data-doi="10.1534/genetics.107.075358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using Quantitative Trait Loci Results to Discriminate Among Crosses on the Basis of Their Progeny Mean and Variance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.075358</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075358%26rft_id%253Dinfo%253Apmid%252F17660556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvNTY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1469.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1457" title="Isolation by Distance, Source-Sink Population Dynamics and Dispersal Facilitation by Trade Routes: Impact on Population Genetic Structure of a Stored Grain Pest" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1481" title="Locating and Characterizing a Transgene Integration Site by Nanopore Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1469" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AXzPg8Ssvay76j6zcSpdC5GlBqQwI5wg2c5wtAVVnxU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1469" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1469" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1469" class="active" data-icon-position="" data-hide-link-title="0">Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493109" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-A-rxFtVAZurEh12S1T2yShxf3237aRtmQ-yQT3-Gy1c" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1469.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1469" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-UutvdH9YHO4mg_NY1a-54RAv8cgpSoRTfDp5U-xEhJQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1469" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1469.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493109" id="node493109--2" data-pisa="ggg;9/5/1469" data-pisa-master="ggg;g3.119.400129" data-apath="/ggg/9/5/1469.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6578-1715" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antoine Allier</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7195-1327" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laurence Moreau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6125-503X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alain Charcosset</span>, <span class="highwire-citation-author" data-delta="3">Simon Teyssèdre</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-7724-0887" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina Lehermeier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1469-1479;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400129 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493109--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Antoine Allier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BAllier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntoine%2BAllier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6578-1715" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antoine Allier</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Laurence Moreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BMoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreau%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurence%2BMoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7195-1327" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laurence Moreau</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alain Charcosset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BCharcosset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charcosset%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BCharcosset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6125-503X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alain Charcosset</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Simon Teyssèdre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BTeyss%C3%A8dre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teyss%C3%A8dre%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BTeyss%25C3%25A8dre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clehermeier@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">clehermeier@ragt.fr</a>
<a href="mailto:steyssedre@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">steyssedre@ragt.fr</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christina Lehermeier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BLehermeier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehermeier%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BLehermeier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7724-0887" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina Lehermeier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clehermeier@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">clehermeier@ragt.fr</a>
<a href="mailto:steyssedre@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">steyssedre@ragt.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1469" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1469&amp;title=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1469&amp;title=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1469&amp;title=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1469&amp;text=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1469&amp;title=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1469&amp;t=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1469&amp;title=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1469&amp;title=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493109/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493109" id="node493109--3" data-pisa="ggg;9/5/1469" data-pisa-master="ggg;g3.119.400129" data-apath="/ggg/9/5/1469.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6578-1715" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antoine Allier</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7195-1327" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laurence Moreau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6125-503X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alain Charcosset</span>, <span class="highwire-citation-author" data-delta="3">Simon Teyssèdre</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-7724-0887" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina Lehermeier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1469-1479;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400129 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493109--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Antoine Allier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BAllier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntoine%2BAllier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6578-1715" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antoine Allier</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Laurence Moreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BMoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreau%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurence%2BMoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7195-1327" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laurence Moreau</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alain Charcosset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Univ. Paris-Saclay, 91190 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BCharcosset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charcosset%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BCharcosset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6125-503X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alain Charcosset</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Simon Teyssèdre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BTeyss%C3%A8dre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teyss%C3%A8dre%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BTeyss%25C3%25A8dre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clehermeier@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">clehermeier@ragt.fr</a>
<a href="mailto:steyssedre@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">steyssedre@ragt.fr</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christina Lehermeier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>RAGT 2n, Genetics and Analytics Unit, 12510 Druelle, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BLehermeier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehermeier%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BLehermeier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7724-0887" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina Lehermeier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:clehermeier@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">clehermeier@ragt.fr</a>
<a href="mailto:steyssedre@ragt.fr" class="" data-icon-position="" data-hide-link-title="0">steyssedre@ragt.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1469&amp;rft.epage=1479&amp;rft.atitle=Usefulness%20Criterion%20and%20Post-selection%20Parental%20Contributions%20in%20Multi-parental%20Crosses%3A%20Application%20to%20Polygenic%20Trait%20Introgression&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Allier%2C+Antoine&amp;rft.au=Moreau%2C+Laurence&amp;rft.au=Charcosset%2C+Alain&amp;rft.au=Teyss%C3%A8dre%2C+Simon&amp;rft.au=Lehermeier%2C+Christina"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493109/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493109/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493109/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493109/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493109/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493109/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493109/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493109/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493109/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493109/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493109/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493109/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1469" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1469.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1469.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1469.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
