<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1385" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1385.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1385.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1385.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1385/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1385.atom" />
<meta name="HW.pisa" content="ggg;9/5/1385" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Realized Genome Sharing in Heritability Estimation Using Random Effects Models" />
<meta name="DC.Identifier" content="10.1534/g3.119.0005" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Wang, Thompson. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="For heritability estimation using a two-component random effects model, we provided formulas for the limiting distribution of the maximum likelihood estimate. These formulas are applicable even when the wrong measure of kinship is used to capture additive genetic correlation. When the model is correctly specified, we showed that the asymptotic sampling variance of heritability estimate is determined by both the study design and the extent of variation in the kinship measure that constitutes the additive genetic correlation matrix. When the correlation matrix is mis-specified, the extent of asymptotic bias depends additionally on how the fitted correlation matrix differs from the truth. In particular, we showed in a simulation study that estimating heritability using a population-based design and the classic GRM as the fitted correlation matrix can potentially contribute to the ”missing heritability” problem." />
<meta name="DC.Contributor" content="Bowen Wang" />
<meta name="DC.Contributor" content="Elizabeth Thompson" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Realized Genome Sharing in Heritability Estimation Using Random Effects Models" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;For heritability estimation using a two-component random effects model, we provided formulas for the limiting distribution of the maximum likelihood estimate. These formulas are applicable even when the wrong measure of kinship is used to capture additive genetic correlation. When the model is correctly specified, we showed that the asymptotic sampling variance of heritability estimate is determined by both the study design and the extent of variation in the kinship measure that constitutes the additive genetic correlation matrix. When the correlation matrix is mis-specified, the extent of asymptotic bias depends additionally on how the fitted correlation matrix differs from the truth. In particular, we showed in a simulation study that estimating heritability using a population-based design and the classic GRM as the fitted correlation matrix can potentially contribute to the ”missing heritability” problem.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1385" />
<meta name="citation_id" content="9/5/1385" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1385" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1385.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1385.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1385.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.0005" />
<meta name="citation_pmid" content="30902892" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1385" />
<meta name="citation_lastpage" content="1391" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bowen Wang" />
<meta name="citation_author_institution" content="Department of Statistics, University of Washington, Seattle, Washington 98195-4322" />
<meta name="citation_author_institution" content="Adobe Inc., San Jose, California 95110-2704" />
<meta name="citation_author" content="Elizabeth Thompson" />
<meta name="citation_author_institution" content="Department of Statistics, University of Washington, Seattle, Washington 98195-4322" />
<meta name="citation_author_email" content="eathomp@uw.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0198-7129" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variation in actual relationship as a consequence of Mendelian sampling and linkage.;citation_pages=47-64;citation_volume=93;citation_year=2011;citation_issue=1;citation_pmid=21226974;citation_doi=10.1017/S0016672310000480" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=J. Jiang;citation_author=C. Li;citation_author=D. Paul;citation_author=C. Yang;citation_author=H. Zhao;citation_title=On high-dimensional misspecified mixed model analysis in genome-wide association study.;citation_pages=2127-2160;citation_volume=44;citation_year=2016;citation_doi=10.1214/15-AOS1421" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=S. Kullback;citation_author=R. A. Leibler;citation_title=On information and sufficiency.;citation_pages=79-86;citation_volume=22;citation_year=1951;citation_doi=10.1214/aoms/1177729694" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Maher;citation_title=Personal genomes: The case of the missing heritability.;citation_pages=18-21;citation_volume=456;citation_year=2008;citation_issue=7218;citation_pmid=18987709;citation_doi=10.1038/456018a" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. A. Manolio;citation_author=F. S. Collins;citation_author=N. J. Cox;citation_author=D. B. Goldstein;citation_author=L. A. Hindorf;citation_title=Finding the missing heritability of complex diseases.;citation_pages=747-753;citation_volume=461;citation_year=2009;citation_issue=7265;citation_pmid=19812666;citation_doi=10.1038/nature08494" />
<meta name="citation_reference" content="citation_journal_title=Stat. Biosci.;citation_author=J. D. Raffa;citation_author=E. A. Thompson;citation_title=Power and effective study size in heritability studies.;citation_pages=264-283;citation_volume=8;citation_year=2016;citation_doi=10.1007/s12561-016-9143-2" />
<meta name="citation_reference" content="citation_title=Likelihood Methods in Statistics;citation_author=T. A. Severini;citation_year=2000" />
<meta name="citation_reference" content="The 1000 Genomes Project Consortium, 2015 A global reference for human genetic variation. Nature 526: 68–74. 10.1038/nature15393" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. A. Thompson;citation_title=Identity by Descent: Variation in Meiosis, Across Genomes, and in Populations;citation_pages=301-326;citation_volume=194;citation_year=2013;citation_issue=2;citation_pmid=23733848;citation_doi=10.1534/genetics.112.148825" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=E. A. Thompson;citation_author=R. G. Shaw;citation_title=Pedigree analysis for quantitative traits: variance components without matrix inversion.;citation_pages=399-413;citation_volume=46;citation_year=1990;citation_issue=2;citation_pmid=2364130;citation_doi=10.2307/2531445" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=A general unified framework to assess the sampling variance of heritability estimates using pedigree or marker-based relationships.;citation_volume=199;citation_year=2015;citation_pmid=25361897;citation_doi=10.1534/genetics.114.171017" />
<meta name="citation_reference" content="Wang, B., 2018 rres: Realized Relatedness Estimation and Simulation. R package version 1.1. https//cran.r-project.org/web/packages/rres" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Wang;citation_author=S. Sverdlov;citation_author=E. A. Thompson;citation_title=Efficient estimation of realized kinship from single nucleotide polymorphism genotypes.;citation_volume=205;citation_year=2017;citation_pmid=28100587;citation_doi=10.1534/genetics.116.197004" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Realized Genome Sharing in Heritability Estimation Using Random Effects Models" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="For heritability estimation using a two-component random effects model, we provided formulas for the limiting distribution of the maximum likelihood estimate. These formulas are applicable even when the wrong measure of kinship is used to capture additive genetic correlation. When the model is correctly specified, we showed that the asymptotic sampling variance of heritability estimate is determined by both the study design and the extent of variation in the kinship measure that constitutes the additive genetic correlation matrix. When the correlation matrix is mis-specified, the extent of asymptotic bias depends additionally on how the fitted correlation matrix differs from the truth. In particular, we showed in a simulation study that estimating heritability using a population-based design and the classic GRM as the fitted correlation matrix can potentially contribute to the ”missing heritability” problem." />
<meta name="og-title" property="og:title" content="Realized Genome Sharing in Heritability Estimation Using Random Effects Models" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1385" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="For heritability estimation using a two-component random effects model, we provided formulas for the limiting distribution of the maximum likelihood estimate. These formulas are applicable even when the wrong measure of kinship is used to capture additive genetic correlation. When the model is correctly specified, we showed that the asymptotic sampling variance of heritability estimate is determined by both the study design and the extent of variation in the kinship measure that constitutes the additive genetic correlation matrix. When the correlation matrix is mis-specified, the extent of asymptotic bias depends additionally on how the fitted correlation matrix differs from the truth. In particular, we showed in a simulation study that estimating heritability using a population-based design and the classic GRM as the fitted correlation matrix can potentially contribute to the ”missing heritability” problem." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Realized Genome Sharing in Heritability Estimation Using Random Effects Models | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FK71Ztx9cbJ7kYsAFR55Klx688gNtZVJxPhc5fJpV0Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492884","apath":"\/ggg\/9\/5\/1385.atom","pisa":"ggg;9\/5\/1385","ac":{"\/ggg\/9\/5\/1385.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1385.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1385"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1385"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1385":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226fTZOJBS4VTLSCeRsDyOx2O8axmEWGjIlX57Vip2mxV96NxVrEmjQBSQFI8jtxwY9Kn73nAzqUYrmPhYqnk6vGLptPCMw8GgBRmLKZZNdJUILmzT8pNSnlLgyh\\\\\\\/zXyMWfVlIvW4lMtB2+R7KBMuqQv3haQpEhFIFyH5XL+lSjDPL8NgPwlG+qoEVBitK2hSZvHYCgAHcYkdjsUFjZcp\\\\\\\/5U2eRfjaoyQZIqqyopOSSNshFXPYsmj8GFXN+aXCcjLqOKGVAKu9O+Y4RHOSxDZhszMCOj0BmfRIdet8PKJHC4ktGnu\\\\\\\/RrGFu4ky6fROKjAcTdx86L2sw1r6JPW9wpNhTIvRseF6eqEPH6qTH9pa3SKrJvypH9GRmtNoBoGG4kBrGSIbwBqCGl7LROi2wsUwdehDXUQpfPIaEeaGjn77JjRg96mHiP3U\\\\\\\/dM0RRZsAC53BKzfE7wYETK+nC2IngBaBuWKLTiUMJVUEL37UUeFAu\\\\\\\/QvrWbCt4OG+vY2NjVRSsEoJRQyqALKuZBPZ8sqi6Yfl55j6HX3p1kVmfxRGQ1uflrC6PANWvShYiUC8EP7RvROTmVlP8PUqzZQnbuXeWLULp7Su8MtEKM5Uzs5fBbOI6NiNZYTy0M2n51a7NtONdw152MbjG7C6G0m3TBcX+omn9p1shfG5Q4AnWcL3GFeN31vHoI6GJF6ItL60o45bhPnMViAXNJdEZk7rGV0egEkRgeTebRI2YzlYp5U3bZaAEvp1j4myU6IpMbbAXtsMH69x5mSzPpG12LkCr6YURRgnMktEJH57UR+b3+cAGiUegqKYQTfv81wF55DTFlltWu4gS7wuYVePRcXm1Ij+Oi8own0tQmDNR8PYPC1Jc7q32fYOyk+rfQu3y6bbAcs8AuLOMIuY9\\\\\\\/ngGoCKL8sS0bNQCmX07+a0aZZVCe9AHZnFcl+Unr8EXETv16YspkCSuv+Pvmm0l\\\\\\\/KOF40N+Yocy5Rk8wlr091Iil9GEvp\\\\\\\/OBRSZAZB\\\\\\\/N6S4wyN1nGiltaY1cNG611+ZxbfWvLnjo6TgtDb1Grv4WVacQ5lF4M4LGMFarPXyGACe5YNZEn7GqmvuF5PfavJFLHPLPfT8A2VN8gozRIBy1Nad3VsSGJQBoaRUbzNXPIvTiRQZm3TMe+g0gqB3rsWZB\\\\\\\/fwPPfO9PNPSyR0YzIIxFvnEm8R5RHCV3Y23rk04JxN9aDTCVua1Lr3ikC6+9LvhREhrwecqGJLYkoSBaLVh9iSdgZzhhpeR0msOGvof6oKXrzpaIekGtxn6\\\u0022,\\\u0022iv\\\u0022:\\\u0022LIf6qFjaiZuO0FTTeyOFag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dd717786e27fbfb05957e73a12c4fe6707e2c24d114e5d0549293fec9cb07764\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LuNez7m8BMOPrCsnqZ\\\\\\\/0wScnez216dTBFDsMkYsQMyIzZPnjGF8TWOrHFTa9sEN60\\\\\\\/Jpq1tWqEFRpezsIx59sGTHrr+0OBd8KQDUHEtSsckmqbH734+qH+vjHOQsAA9JAq5c5ERemcWDKrG1ECwbvOYwkf\\\\\\\/UzuwO5e1ldXzBnHF1C5okZFQ9vmvBbRUe9XIqKyn33AjZqRHA9O0r8NyW\\\\\\\/hQhvCBAbIM+VxXJt7iCG8+SSjBmdb3c4jww3SoJVOjAs0fMMnXL46DvRMlYodOOeYzfrSPoGeOgIBlBdrZkAnsKzbvbCBm35n9TB\\\\\\\/rlY4OlGI82Rd+nD\\\\\\\/plWb8KgnL5zEHbuPe8MUFSbO\\\\\\\/ZXFzV0KeQEq4lTSg\\\\\\\/X72aNIITpvIutkaJzoFQ7RFzXcVqQvo5MRv3upVA+VDoPPN2gUiN7DyXtQpjR5UphKteSeGovpgK20wV7xn8eMcraeL1AVLO\\\\\\\/EOkVNBUh55U8CMYQzM4QLkJkvPh0Bd7xd\\\\\\\/jPt0NS8WOo8djipIXeLdkBdbz6XyPrukR\\\\\\\/VMWUuYoYi\\\\\\\/xzdsDZyCmr+rjeSf4E3AZ3bkfJfzZuee+N9XvVgimEHME9rSgyZ6ISQgRx1aRcBrf\\\\\\\/r62w01Z3itCAlRsfQjU8CUdqhf2X2rJaZ5QwbKFyVXdIRUUM9XilKkXTLXo9Gxee02z7RJdUPuolxeCTwVEwJ9QMbzg6E3UavclKwPeA0TuMGUzRad\\\\\\\/7eg1VoSFq13WFmqBCmjwnk6zIy0y4mYFoBZv4LoU3ABmB\\\\\\\/iB9q3tGEz1EcjaTr35QwV7q9AjYGIblS15jkSDDZ5YhnXhKxh3YjxEEk8WKml9\\\\\\\/ZE5l4JrxyxW0ubOcGnlfcuU0p607tctXpuA63WfO5eq8qgKqEdFEUTO+zwdd7CrxxwQ2BZzAJqkVSpM5CBiBGcDcoA+RFuSqEuhEM015N6k88gGPfxVI4fBZ0Q8CKVgIwkr4936I24lzlBzevOKrzdwhlgo+TLNwTLVDGpzqQh8WQBK5ntVmG2GGkDwsr8b7e6n6TkXH8r\\\\\\\/+ApzEnhKeyCuIFX+bvzdv1ZBThbTBuOzhs\\\\\\\/NArnKJb2cydgsBCzDaNt0BdzufCic1bMcNKdSwhhd3OdPEn5wOhZhyLwAOugMI4X02sp+PirlCR0gBNUdbWApY3SSvYh3fkMYNY\\\\\\\/SLb7TTdy+MaLZ0AS8qiGNfnKjdKwrKmBFjtaTAL73CjGalrkjqbJ9irGHOFEqMnonwX5tO+zHYxU39UpYsD47lEFfp1uZZh24vMNCNA5EfHzrvJsUY6yTAAcuHAYteTqUEt+Yzsu5a2KtNwNwX7ea35dm\\\\\\\/Kp+Uh3XEWz8fmCr1UzSJcBIM7Zw9kWmLEGkfkU8N\\\\\\\/wEU3OmnJroigvHQ\\\\\\\/jBa51qMCE6vo0178mLTocrLFzJHFQVSDRsboyhTJVJ3NyTrfgo4J6oPTPqFAMGLxSi\\\\\\\/lApECX8t0NsK2YAc2Z467fi6BwYGzZHg+hudXoU\\\\\\\/gKEQI3XxvNJI8zHeUaLveOgFNDO7JP+Y0zZWWS0YsfBy5Aw\\\\\\\/vUk\\\\\\\/mJZIhj2\\\\\\\/Es0OwvQc9XnkcshtQ8BsdLsCrI5Rm6Af4jJ\\\\\\\/DXNmPuicc+psAGmjkZMFAl7dBo15UcjhKlsQjDJI70k4TgeAlfgvat9VrVfiR2uTJ2iHwfNaxYV6L+4+Qi\\\\\\\/MUx9sSL8KnyFrpitY\\\\\\\/ry2GxIn0NSDHzaALrMS5wKNSIqpp2iwAM2GvD7CJmuqKX678cT9wIJ4RgdIfXz94+PsRBsSnsOXx6B+qwYhNXwFZH3HIlrtLj+4LfBW3VCJQkkCcx2+xaXPHWqMPHS7tuJZWix\\\\\\\/H+H3qv6E2GJELPI29+xzq1Mj8xZ+v0a0xELKyFm63bYolS\\\\\\\/E26S7qP+6ki7e1euGTWLbtz571oiGIPP408HBoF2qBjQfyEKz0xhW+9zun\\\\\\\/fOASYO1dk\\\\\\\/k+XUWkDEB7KlUzKrgmaYgOE4N1US2XB1+ZoHrde6+CJHkO6BBvV6D1f1p9JaGWcNLveG7tvbPuBiV6h5ic8UQwZPRGtJcwJ+cdfh8RkMO450hwvYjSJXhIk5Md527pp+d0F0P\\\\\\\/WleNWF6TxEXeTAfmfVT3LTlc6cAN3UHXBs9BuvzStp0GFKYFA2oN2T9zzfc3xY6h0ByRuhVaBjaHG9CtpJvgURpGxEcjCf024KAnhrDQ3Y7zpCz3updRsv1o1HuYMWEsHvBVPYpQmvwSuTjDl0SOymnXT7A3n7nMaw3hOEG6PbOoWPWk8Pn0aVtECtxW\\\\\\\/6Nwml0iuoYQsKTY1n3zN+iY6VW\\\\\\\/KIE2eZZ6F6UaRhH1eNKwbymQGP0q7tPc7yLFFBcgkZFNDfw\\\\\\\/EWzQPQy3QRUvkhWnqv6MoY8+muEeEcnfT9i4ImmJ4\\\\\\\/Csm12ZmtGrrTefDn9X3bzSR1RDVjrsNqJbb5dHDSQSC5H8ImpS8YwXffpY0gs4mri0sFjTM\\\\\\\/gjWz6zt2lGAgwJv0nxaW+cKc5ZpbAUY70XiXQSWNq1qA2X\\\\\\\/0ICVXDE2lYB8hT\\\\\\\/OTNXAta\\\\\\\/gqcewxHMp2rprD+f8ZRcMojWSkZM7lJTYUsTLKFD2bxIplb06trszKmZHoLb\\\\\\\/JDmmQ\\\\\\\/ZSUxaS1NIZX3Xbjoe8krnHbP7EpAxXvbwdLlCFXaLew4KqwNPLg7AO8md6t+Lmw+2L0Fe87par8UJ4QjttEAa0LbbK+475i0qWuO2yEGZDLp3t\\\\\\\/kY7EAmS924OKmV9K0\\\\\\\/iCRD68Vvj1zHet6aAI8MO8o+MevKGTv+QKP2bqIXta0W6Y9MKg51ZoPZb2iPPHv3vq8B443tQtiB28LHkezlIfYAFNw09QMSzsSQQNnYVYTIqw6CvYv3ulwgwOViKtnJm3cafKyqtW4ycoLQjyAmxYlcQPWQUOO5PlHVnodP0\\\\\\\/l1UmpCgNTINaA+9wT3i2PGoQLJl6hP5Shhc3NrWbh+MrYpOvFWtIRbjqMY8KYHMCBrki8ESRKTtOFT72lranXg3OCJ0IZ0jLenIe5vYApJFsFhmRd3AMknxuAJzj4Qqgv9tLOx8up\\\\\\\/8q0L1cXgPsbsXLoVU83I1MxnhRdlCjQccnXpbnKORyB6ydoQwixyxotX0vqMFim2vJIXBnIIj0h6dLWBrKiYiI6tJP6abbo\\\\\\\/UgDJ1E6cfEbi+hvahN+vMbDqcO3ecXUvb8CXIGXczCxOT6cJDa424vIdO\\\\\\\/QFLQ3ALT9kPU5WU1HGD2cObDyDSlE2aih8Ye7OhVXWEo0w4UGh1OmLqRYB592RSIzdHQtHjVD5u\\\\\\\/KABMkzm5HHd0eTd+7\\\\\\\/EexdjwPOFbGzercjzwJHv+ZL4HXaQQ+HYz5j+7EktoN5pijp2EVfAAALxE7rFWeQcNC26H\\\\\\\/VAQqcItZYp0eoEOKB7VK8LNOYRE4V0dUoMhkTcgYW9bhsgxNYvJTzH4V8F3tYWmhqj\\\\\\\/zWMi0JZVUCWGQR1ta\\\\\\\/CRnws5pFEbG1zCXQNkJJn65XXjUGtXvmp7KilSZAGJjZ8Aks5XzNA\\\\\\\/IO2dryDulm0ZUOCqA4FKcM+iHK0Osx2FOsho6gwUH1ea5jDCZJ3CDc2\\\\\\\/4WYTwZyx428Q9I59Qn8I0MHg6lHL9tai4pz14R6UBr1qp1kxw0q3V9Jd+ed\\\\\\\/u41rhRnoVo2yV+FF5q300AI5GBYNPKrbbeB4xcwe+gGK+pTmf1XbY5qBRyWlh4wkj0XOao81SDwVheFxpFApHBwmZIOgQPQBYW9RgG6sj\\\\\\\/5ONxbqeUqBSr36h0+Rbgyz2iSs9bhAOvVLJ4rJM\\\\\\\/c0yl6YvjulwcJ9vYIUdXtDMrAY\\\\\\\/BzkyJSjvWRtlyxhKL89NsP7cb4XZE5UxUyQXtqoMItuKej74NXw\\\\\\\/gyUPVNvedOXZyc92IBLl7ydVLBbMMzOxcNAnwqSmNRfKTwJFwLR0fM6LDNcq10IN5tfjebsSvAIWQKAqLFTv+QNvBEzsnpOS+\\\\\\\/Clf745lmeW\\\\\\\/1ZWymMNPbY8uTI0UJpoO6u6\\\\\\\/Bbi\\\\\\\/VZLIbJoFdY\\\\\\\/oCHBk7ITIq5RJpWfkaNgpKeiUUQ7yofcy0YlLDnGZhnVLf802+Vw1Qg8rjhV8ZuLXcGDJMF8Wk+mAjuBFz9NfqZcecayJhD23yER4wMANr4PRwwrC+zh2a3Fq6nGmYEJUd6iGxCQOedQ\\\\\\\/1uekV37+6TqSrrt+9B6tJtVnhO21jxFmBtA3b7Nn+fv9udBF0lV62Erilozd5pyNKZl3ys0smPyZ5yF+CnT4MscU+Aygn84DWObwWrqdALDef08xvD2Mo5YXtRygepYGr8\\\\\\\/4Ud0UbMPqj+8ZihZXHm+hDbeiru2KYG8Dq6Vf9XyvP5PmyWBerq0CWB0RS3MTFI+EPCA6i02ajNpqtKC7YDmc2cYpo9i7+zbCowBWVVOHAHSMWBUO7KHWyU7Tr3Wf3xqQ9eCBTKM40ePDQAdt86YOz08oyoVaVoY2Ppa6\\\\\\\/YKOYffJA6x4CfWahWrJrIMalnWGMR83k9ULFS2XJ0LhvIp\\\\\\\/MQKXGIIxUMtT+5NmF\\\\\\\/yDLbr3FHQ2EC644oCboNg72JfVnmlt0PyZETMc7zhVSxVNhS\\\\\\\/djZNcxyStwZCTT+CQsMUk2Cj0+J5QD7vdifV9bjuMTUuchtUCU+WpFVPU4GpkIUl\\\\\\\/LSLJuO3G12ZSqdVTI7AU8qTbpjibPhk7jZsxRLjsqvpP1zLQ0Yt1fm+sqv3LVzOxUaS+yxZ4NFaFubJNYTEdcppeZ+h5uZZtm8MShZeFHOiaoMOyp0YiRRBNIJJ3HIp5jFSwpD0uSscQBw0CuCJYjWG08YF7FAYxYgZ6OHgKqRuuints2EH2m7wGFyx9NPfcwKg9FmVJnX0BSzfsKKkXh0VRlo5a1HQ6oLbAsdIqqitJSc+X39nXihY6bQk+fpxv+DVHzzCK2j04Ivd2O3J3DecJgJMopWq5cs3l3XCTkv6\\\\\\\/LvDJI3ylXHLkro\\\\\\\/3y\\\\\\\/TvP1AOh27pSuGooQgq2EI9404LF52sxHZ9\\\\\\\/CxXGZGPxKGOWDGj64BcFhzfJI9BAS1yLKgmeTE0QeU0L2rqg79cPDC38ouamIhhSmmFsAHG37ZcGLaejtLnzWmQf3WpOL1Hw6Dyy66giQLnvtGXU0P3KQR\\\\\\\/tSAik2pLyF+eToYA10631an7K8gHSUgJYlZXJzMLofgnI7OVUvUpN709YQ7rKk2cSRjNagD1g+WZ2L4v2OrImgh\\\\\\\/j3z5mLuPHjho\\\\\\\/kRu8lnuGSW5dnUlLE7wtY4EAR8BYZKtPJylqH9PwSXQr7y81vQENGi7nCgyVR6IGdotDOBRdtPhmkvEzT3bhq+A37M3SamHXULPX1hCXchKavB\\\\\\\/qh8w7lHo7VMQDaSMksXE6PDy72g9xoDGPHrSVlXAYIcSy7Cf8sC9ynH2fWbjGHJrOwlNtZ4++REX6Xe1mvCe5nsptuAhtCpXgTTvEhEpRnt39JDTYKiT2LbcVSU5n\\\\\\\/Xx\\\\\\\/CyPuKRn+ChJsOFf6fHYdkWlrtuCclzwWEKtuief+gr4LX4AJCuxWP5sHhHBcee\\\\\\\/L4KUZnZQF2SLZMwVrGgdgrucZlZtoJnN8H5893xHbrltNh8nHP9Rjs29K1EimChwUUW+VQdvGmC7zqpKCQg6F6YUtOOyzHHXqGfp8qX3tLjTl+7nFg6niyZnCLk3yOsqi2us5c0dFfycNGtIsGexlpSt+MyqO4kwBpOO1j55\\\\\\\/774Lfy+nwW6HSAY\\\\\\\/b6H8extkLgyhl1gM9TEFhvVlz5yF8XwpYXVYnsXO236UKHBi7u9sZrEIY8xnBK+mwy12l3iPU8pdQxNSTDcVOvzrE9aHv68LiU3k+58NAqMzeYDZV2ww4OyU3iSVF3MYTLiVrRuI3KS+W075UTnEwM9yEoKyQ9u3wTRb0utpot9IR24lHvMZx3ZinUhbWkGgUy60CDiQ5H9vCG3ZeR7fpZBtYoIBiWPuj1YedzdLP4SXX8hdvyGR50hkQ0s\\\\\\\/eYjDe9B\\\\\\\/S60qBJVmCKxisf2alaGI5lgeNOfX4NmTB+n9FLHQe6TTZ4r4nDyHZvd+F8h5CvfVbrUAUsU7sI\\\\\\\/RsTEuUB5Dll4W5lXef+tGQv3yRxirY00Ns+43VHW3cSvUzH2E1XZhaihKxwhyqReUjNe2k9kAvUSgu7HNkD+srqlFmoFzfhOuNk2JHp8fG6OOBQ9v7rLvyzvDGKayvoEU2TujRDY7MvVyhot5VXvVoZCCCdaCaPfJkV5dbuGC85QJb+PPBtYcP+Od9kZY7RrJyKibpPpYuthaxpHzmZ0BOsIJKC2OwEEMRqFoT3ohD43DDO7RRFEDcOKPeQvAYY0vI+KPlnU0FmMtxRZsBJh4WzZrkDG31H\\\\\\\/GlcpPVZJSAAEzlk10IcMjkmQHrRKIT2HZRKcq2g4s151zLPHlKpR++v81W5yS1ItoHJFh2uJaisfERpbpWj85JeU3Yjw3MR2z9271dnuBU8JwCD0n6RzjDztO+y7s9zGpWFROEHk4bOkjyl1BV4k96ybEwXmNw84WuZcH0iL+IgjWA\\\\\\\/xI7KuJeCxlxO5pjjf8zFpGeuqPWk0P8T6YkOFbFtn5J8KSAj40\\\\\\\/a7vdl6AcMWalP78GJQLzgn1s6pZHnJIMosAC90bACHKt+M9epEp810GUoQs9\\\\\\\/2HI98SIfY3aEr+iLR2K6VzPFw3GA7FoTBCY1Yewm9vBB8k3dLz6sdpjhvMwRppvweRqr6dAe+wZDfB6w2ORge7JwYkck1BonJBT\\\\\\\/lHZWMYbrxMxMRS2qXJ+idJ\\\\\\\/pSTwmUWdfRjBs\\\\\\\/ekYttjPxKF2\\\\\\\/5v+a0k2mG4ipgfCpgr2m12G2SAK0uptvE4p\\\\\\\/MmHYvyEjfS0uX3j\\\\\\\/XOFW0t2HRJT+WG+80yocoFeJM3j+MudbFNyQRXBSvVzYiadCPJE1TPAQV8a2TO63MwYqYm0AxpJh\\\\\\\/hC3A8WZ5OiqiV39q2lWQxsVet\\\\\\\/1OI05AB0hOJcq5H+DfmIFn4XQgW\\\\\\\/Ab1+pvYx9mDofogJc3f2C+TW2tBN1rinZqeQKTyindWHnaHCsky\\\\\\\/ncFi38WNfRDh5899OglwBy08ov79U2hN+UsDwWIkcmzF4F51H3mv0KyZCeVuyMqaCdlTiKReKPKT0PjRGKdJnblo85Oh4WTXnPhdhdHDCMvetpLgPrKXTyr+JBhHfrzsXx4eqXVC1lMTSLNqqMMukQluQNkdBJiY17EH4+9wWfE313jeFwvf4irlLFp97MuQycREQ+WtdYW6W2DlwflLqTEi2rOTT2duk4Rqf5JrBlBBeYzDm9IGM3KF1O07NaT56CdmDiUMZpI6uBmz5T37duSr6CXgvgyr7KGyjbWhMj7D7M46YJj7657yUZZmtrCxOes7mD4YlX2NDt6o+5Z7VqkzSq2soackH8wCFWYoaM4nJRP8Fv7k1m5gAakC9LocAgubL2TovV9I8h4azYB8P2bfSJ5PAseHu8vgCQCVPndGW3ahY094Ehmta\\\\\\\/8IHJDYq9IZ\\\\\\\/xF3q51XthpBk2CQkG2ofiGeuSHCh4W4shHVjMhm\\\\\\\/fSi1e3G1NrK778tRmJiV3+5aKufr51A4IYksZjflNmYI8YjmfdRnzvaajBwlG1qE\\\\\\\/u8Ke5SHcaCbaaCfSbj0M5tHCvrXLS1y04P4XXfYY+BTdSrqkkhLQHbOmzk2jsVB6goeA9Cke7ljMLRjzXSBWcacbO\\\\\\\/ZiHA1JmiS12r6pJafLbg++Rb\\\\\\\/FOOQl4DXGqZhPFV3mILZ6+E7tKimfEcg5CyiOAdUeQH5yNB5K7mieHWYK4PtmX7MD8e7s1fp4ZaMHRtWVjv6bYX2U7n3Tsdy2rPmpDKb4hpRnJCcYw01oGUWtyIKyJoySJvQYRz7q8QoxgLLlPKwJH5Eo5g10GpfndH21tVW4BZRShFq55W819FrxZ6+QU03wIi9C0vVubYK+ybeQaaRCOieSuhMEXxp7daVA5kGPggg349H1b42e6pke+Bi0NvFJfzV48yEkJHWcAvw7uZ26\\\\\\\/tyxtUd\\\\\\\/fB2ZRU2J+wefi\\\\\\\/EKd7V\\\\\\\/Y8vSBGZdIMNmb2VuBVdJ1sKIcDsD4oHe\\\\\\\/isZ9WItdZTq9uwyzrVR+xpBQhBbjnc2TdXBXkvaerJIC9F9oyIwshK3jjNkugNlBN52GPsp0222Fbj6cstZlknN+U5tNTjlM8VQ\\\\\\\/Xz81c0ZweJIorbi6yw1CHIIbzyyogUgY5J8can1rMe+H6+i\\\\\\\/khhVOoyNmEPUNQ11+\\\\\\\/6g9bCO+KTWGyiQrHI+c3OJshzg1L9vgfBW6k3\\\\\\\/Pkux6gofYU5GIKhmyuzS0ycYDanrJpBU4I0qsC1jx5nlsD9G34XeXw25xj0sWw2bm9jzoF5hdb2OyCYg71ZwPcBkJTSICNb\\\\\\\/mhleyU6q6NZh78+wjrkDO9tvxV5RT1ofPLHkuhIJRfQ4CmphXxFpzfOeQWmrHqwpDO0BqWV7TJIL7uP6D2YEZLIagjzil2tUlL9h5V9HdDhFz9cqAiIb+zj2jX9v5qj0qIBm9F9mGLuhTO\\\\\\\/yaNv4siF9Rmt7w1rCWIkR\\\\\\\/eP0cbI069AXP648BiUYI0DDzWJzAepQliCjkIVrJ\\\\\\\/k56tkdnJUIjNlJDof9Fl+IEK6t33yf7ix2jowfxiC+QGTmbdLoYw68Yxt7MkiFB8Q1UzJXUFwL8YuVh787jtck\\\\\\\/hZFYXm14v3TJY0N9oOcB2R98h6kpxszr8MtfvEbYXlicdG79+tVMgG5Jw\\\\\\\/mea+Jl74hQQQHD7pxMfA\\\\\\\/oWrDp9eB0AKQJv1B\\\\\\\/VLlB1VBB1bFOFBsuDdzpwhpUo7P89yRV0TUYuaiqQm0KpCLLjh\\\\\\\/OwrhP5N9t3tmX9k9rb1OCbLj7umg7WzTENYhlRhw6aQpIjbFPLQgq2iynUO\\\\\\\/kuIRmenRNm3oTLpp0teYBIYctinrm46Pr3Btq7XhvZRqHCjc571dVSi7f26XVgL1h4IDH\\\\\\\/SbXf409VFQAtiDf0kKu5u3pt58n6ZZuZ3zcUodoC4AUzyagS3XHJXYZYuLJaq9lgl\\\\\\\/v3BZU7c\\\\\\\/MTrZpmHIBy4OI\\\\\\\/3EciFNnJ3Z0l5\\\\\\\/5spMFFRTQzSt6noZKnwhk+r7ioCaVc0JvGwzo7OzTAPbu57SgL4\\\\\\\/H0jHdxh8uOp6+E6svZXRzDb6MYoTTjik+8v7aEH6Efizb\\\\\\\/ZjUkvGTUgFksHN9Pe47fBzKvhVwvfwqAIDdB064L5DYQ7K1DdJcE40B3aZfUKKPcoThNdIGUe\\\\\\\/i0bCZi5OSejpeUeHQMLqXsYMWQuao8w7RuRfLuR1+orAD3ih8rrlxtGDn6s9uK7mC6XLYMhFicOZXgOxK0\\\\\\\/RNbUSQGMkBTHDOGRJ4hy5nxC23OGkyatPZe4UOrmZOgHllR4xyj4gKwlsC8OadpeK8pzX1iNAovLEe1zjwM5eeB7QLYktRhIDWnHxKYu4ggANSoZR4DzQYjgub97x4TOniXukiIjDHPrzqsUHv2NENIh4QVZs+qApJqh\\\\\\\/zE650pfD332r626BKM\\\\\\\/tbioqioqXaGJXZuB3KEYXA19bR4LuZT7vDAmqbcB8E3LpPCPyZMeDY2+Fbr2m8UOIKNJr4+eISG+uqKJ1JQUJgueVofkiLwo\\\\\\\/xdwClE7OVixiMjB18x1\\\\\\\/Sz5xHzvoId+KNqW9MF7X50PWPe3jiTSgniw7A2tlF0s6EycGafs+ONGWnWwdXCzITTRRMYpwJju6bLumRRU7EofXjKFmxDmHtZR6mRQrbmZWSLrJT8htsvNavpDrDamr7G9VxulkzR7Zp3ahqCj7bnQa0AWHKMDBlI5cmAtxRAhzXUiP26UFt6pQ7AP8r4GaUTmYAH7bRYEN9baEejqor4woaGIjSN8LLCI0Hma9XvRJQPbBv+TJA+he2FaG+XzpE67R+gBwMEr9xOcy0fO5PI95UNK73fkmd2\\\\\\\/LbHQ2MY\\\\\\\/4rVs4nCE17+GydCgIsrIPNnwk7zuWLIm5iV\\\\\\\/2pMXRnuhcm+17zPbmjIvPrCRiEBjL51iuundBlsU5kZePCy0R8wro3BlioE2upSskoJYoHkisUEsuPYcYOFCTuIPFMSgyQnJpe+2c+qUc4gunz6\\\\\\\/eqeCBlSz1OlwfX0v8dSFWj5+ICN4VC6Vzw2hJuvot7Zw3k1HDpdyhi\\\\\\\/n03U2kSpAvMzl+sWGjzdvtxunpVDLKJADojD\\\\\\\/sRr09upYfYWk9aVIKAyGy1tC2FudIKdqwRoVnnejY4G3\\\\\\\/Uk9QrK63P1bmy8wDdqs3jvwUPfhrrZaspaRzQmx3Mo\\\\\\\/StEJ3w1ll922kZaYo985XYpOQxU+NHe9VGWAi3f8WgaLkcLya3Wl+LfrUIa2OOjXqmGDMgolccBXEXr6AuULDxqxy8L0YxvWPa0pdXuJYmlSkJI0FX9knQQBkkzYMZrjA5bj0j\\\\\\\/bgEXvlS6OM2pGK5WNpK83fkL\\\\\\\/ZKkatnlmlAPHMYdtFA6otZvl4PDLaIa5aItnMhhuF4Br5EzuGmoU0wqBoFkWlNG8nqtRXaDecFQsDpLMgS3fx2ydFpvZ3QGwiGhl0u7lfU0trEZrlHTofB\\\\\\\/91DVPxWrLrHGzORRZA4MZ1C1ocYTamQqOKAVUU1ru1hXQ5teLl9M3Jmx6fNvtT8D7L4VERPwjKa8wSAHBw6dSz2WvbezTs\\\\\\\/lpo\\\\\\\/qdIFSJSDek\\\\\\\/BURyp9TqFJ0Tit2kAPXxlabVeaxA+Z3GoMnmdsQFwK64VUTAB+XphwRv6QflOxPTyInlPqze7xVDywDwrpmokkI56axxnbGc2vpe0c7o0lgQxfV0FEbreLkCLzgae7kVC4l7KOFMXOszgZOEGuUhl0\\\\\\\/tGwdYgufNS7NENrDp1hbVSqx\\\\\\\/DZsjcd2g0kWE9cdLk\\\\\\\/MzeK4YjPGk7TVy1Gwryl9foQw22useulMTd4y7GKznxW1KBpmSK2LbhBxocd3LA9O8AbTUnGgy6GolHCFzuMcqAKyJaKObZQpg531QPrM23CCioSbZhbAtm8AWdMnngfSenX9hglXplV0k651JaLECKTvNi+zw5JnQRp60aRNr8ZzWI7RTVODXCGAgU5Sil3ypgK\\\\\\\/6nZyoJ0iH9ZKvuJoGL9qStTctdGApRy5X1GYNCC8DMNqOdBw7CW8Zy4sq0bj1MS3M7MV5yZgd6HbbknVlhpS9FSqKv905rnCLp3qk5R9M\\\\\\\/isY58hYcXUplitIozb5HOhFzUSifNiSQ8BJ\\\\\\\/lobXiayjAXjV83NpoOCfzjk5F6ihjC9SYKgVXf4n2Q1Uj1AgsiZkkfMlKB79jFFbZhpKHHWChETe4hhr5eUpnPQadYvGzbQeIAvyCKtQeWEZ4n7IHRzDCvh+7ofYolA3ZeY\\\\\\\/a+\\\\\\\/5LAiAU9oVXwzkkUKwvENOMwrfYFgoJhg5vNF3pCXsh3LUKXXHdVDl5IJIwmQiui0s5PU9oV0FMYavXltHm29ir4ko6\\\\\\\/RcgM1gZCcCpaIyUqarMJMLTkqwCLIz4ECICAeGBcN\\\\\\\/hjqt11xV23d8x4ibctpmN2nunABdMfjzYWqGpcZPBbuk1zYik6UYi7UG3I6YN6ZN8d8SKUKfX8PL8bMF9NjDbPaXuRwIAPpAkVZ8fBQLXXciZF\\\\\\\/u8H8+Kr4uXtk6T9gh8L66o6sfZcipaIaZxlZX4qGbewzgbTY1GYw5clnzOHC5VNs1W0N3rtRAsmEsMGzTAioWOceX0bVHtNzuMsRPQMruILoa0GlkxfjaNDfKHoUdEUuDEjmEDa+LYKsdNFNvg9Q58JADlxaRfPHeFVHP6PcP+mxGdGD7k889p7rbIq6wKcFffw0RZrL\\\\\\\/RShap8p69D6xereGPMEc90q3F+B949Ux+CnZoa\\\\\\\/iQcun661YekdHi9f5bKvOumMmNRAftA6XlDcG0QletvzKueoRc+aRxotjONUEiJuUf+nUbNMNA7jcMwb4gsO2FEYoWmlLT375B5Ywk07g9g2FNhHWoHoBdZoYKXhtwlZCzlwGYtqYHzkUqdoGVuA52sQ0RMWhCQ\\\\\\\/UchQyJguNoY37LdJLiD1fUxna9lB7Z5uw4uUZpeofqphJBcHNWSiTW6DwQb\\\\\\\/eElvTJJ2S\\\\\\\/07M3AO2jQTN2iKWkuv830G8LnV3eRrvL4L\\\\\\\/k0hmKp5\\\\\\\/u+EkOfRVuUwKKDGUOe5neDsQ+1yiylW8EDEz0bw5QrRkFeEArK+VLY\\\\\\\/ILAGr6hev0LQ3Yc8l0K4EQbw4sgHCHv6FcV7Hds0Wuyzd7GxZr9Q5U3RohCbn5jT5eeSWc2uQJxo3M9urJVwebmjv6YPG+Hxjz+MohAMcFfbabk5z1jHSFQHKISS+Pl8Ep93BjKoNPheZ6zLpTTeVdXBWyUwu\\\\\\\/I8cmfllBfXTCwb1E970xm3U27\\\\\\\/ryz6qtuUYmVU\\\\\\\/PnsNteksbuqOmfiYyL8m\\\\\\\/41w2d688\\\\\\\/T2DqpZp8HwAayOr2o5oLYM7Ydaogkbnl\\\\\\\/STUp1SUcAQKMwyCdgfw3oZAcPviyRsky2Q8oZX7agsq\\\\\\\/gCoY8qSaq8OX8cx9qe6HtH019Z\\\\\\\/6H3tHQqMn556PsVsOqnQbyUrfd3dre2EaRsIy7ly9wj2eAg4dXmBfzR1rMUb3xoG957Dogn\\\\\\\/T5I5zPBbwzJhxu1zyw4HdSixYVRC6GWTf6+GDRM1Ht7Z0on3gQ6DDMAt5SsaXMCM7guSUK2I6QwQAJhh2\\\\\\\/ZcfqL6NVyttrIb8WnC8ZJOhvkaCc1K2y6HVnO7xQUZr0WOSbAkTnGU2Xnq9a1HXTOWv8n60aOBUhQslqBnWuWC53C9G7M0CllBBGyif+CUHvBc4zZFj+VBjE4a4aYo+w4ABI1NNFBKCVDp\\\\\\\/WIPXg8sT51KRK\\\\\\\/kVK0bgZOuFZjq\\\\\\\/uq28FUV8\\\\\\\/jyehECNqdx9TbC91YZ9QqsYg3tsfehUED9i6lMVjcw\\\\\\\/kEG94Kmq1cy5oRo6IM8aE4rUcYwWoKd8\\\\\\\/2MMvlVS5J3WIix9Ds3X2LLwtMOfnpZLM+eLbGvEGcE\\\\\\\/gM0s2U71Z47JP1Zrb7odlpfC+iXamgMRX8AVQcEKd8XKUaHa2754WBcBoGQRc6fYuODmiV9y19N9imyEO09GjDnvM9myfZyziixHKKtGgxtN4Y4h6tI014bF5fckAuLfCzWtct2moaC+wiMfvwKE+ybTcO849jwh2QBfLDBVDuIw4ReA3TAG\\\\\\\/Zn9A3+4yrCCVZBgfIWeBendyIixvSzKfrv9Mr0rliBxkC8MGRwSOIdOzAdNpTiLDhZFC0eVRR29YRzIfsdg0v1WCJlLWsEwo4RYMitDoPjZIl4jgkI9LEMddaoG\\\\\\\/0J3nis\\\\\\\/qcH9eEkvQEMXcE1slDD0Ras1\\\\\\\/fE2tAzkzVAzZNpSUW7dDu6dpLAnZ5CLrMPVmPMJcAehgJBFlAn\\\\\\\/ed\\\\\\\/b\\\\\\\/g8DMJ5nGfG1tZ45Ort4oXpSOgc2F4942vjbrOQzJYQxjd8P9KABgcv\\\\\\\/tJTXwIvHczzkDpS9qWkmWFtIFB8LNsKv97m+5VnWn5xb+7EWvvVGpbbVQ7cgCift3T1AYBSegidjcDru4FJqBMzWRvQnkbw7nr9U8w4z69HVMTA7VkZfsmYmXWex3mc8kIy6sBF7GuSkOID\\\\\\\/frVBx9i0mPK15jgPAgB9PU\\\\\\\/KhnJfTQL\\\\\\\/e6+6e82sGKENyDxnB6qFJV6UxNS0BIC+YaE5Kx3sVqQFoGWFTygXHr09aefbPBaJ0NGvrbiOEi+MY+faKBXtXzkaStZ1Ozhjrs+A7qYzJK3Y6kC2xRJlYvf5yk4OlJYGiR55I5kpUd7k2ZHkTWWh70NAnq\\\\\\\/6ltkHJTdqbx2Kv5lPvlD1SctmyqYZ8d2SyM5rA8EMN4ZXRy5eEPldQZk+8WjEeAeL0ohJUo7nabUjk7lWamSpl3KZoXhgG\\\\\\\/0G1n8yMk9s\\\\\\\/1JfIKlHSMEv8QAXIPLJnOxU\\\\\\\/0SBufceYVrbl715vyi0ws\\\\\\\/k96TF92CSXroAWY3z3bxl\\\\\\\/c7OX6Zng5Zus1IbQxDrEIAZu7SyHmMIUR8xjzOol+6S\\\\\\\/FtV9T3nZIfhuU8dOSnIvINI+yqq4+O0UOD26vHzudkg+wcbbRuvkD1+h3gQvRVbd\\\\\\\/eQ6iSvsQ6J4yX0dhHAlW5VCxZcbt7BwDNtRh46YQ+7GOirvY3syvSuWYWOH28yEEpmpOzOUUSEunBBFHlCduOS1NKP1477qatNAdGtwBQqHjbX+OjAoDbDhZAS37S9rj79puy4UcA+EFU6NzZKlBjbpcSGZKCOqZ4PPIh2uch39BG12E+EPdmhu2sTdmLYPlm09J+kQ6S7t+6W7aNV7L964zTK3KGDGRqESQEg+5VMgbdP3Uif9pZfZx94DIqt9kjso2E8PcK9Y6uj5W4rOooCvSrE9i3kz9Fql7nyi0Yz8bM\\\\\\\/Mj\\\\\\\/fcn7kQh8EBgW9DDYIiIBlqQYejJgg2vkVkxIlfahCXDMz6+NCseSrbXRZQZbVpJaxkhWyCkzFSJp33U9pPU7tBoXb2KUQ\\\\\\\/bPPpXmtm6YCg5F0KrC\\\\\\\/qQH46DCVLGmuCOYR2YbL0addhqF7JPO18Tnqfr13j730S7ocSuiCFE8jz86xO96aAc+jCUpPydLEHL5tnwtWBZepXpeVUTOJRA5kNmX3yKSHQRn97wBccJI5DeR83aszgn4gLU\\\\\\\/hRFE0cZVbODdJaWJMWz5XOyiwklltGNgseJMJT9CJXE5P7h3qDjnrztZ\\\\\\\/aASWUJK2b6ceUBEyS8t0K7hITNWg1lC1N\\\\\\\/O6I6pBbMUXRHWV\\\\\\\/RiuWUMaG0RrxJh\\\\\\\/zILCRp0k3cMlflscce42nrA2GoX+3HEH+2tfsFqXlXE\\\\\\\/og2qIZmsWXD8zxx3maL3nQ\\\\\\\/OPX0D7uZGBOFSSftnYMDHm+Rq1VTkgPzpfhdPA1ss7yGnK\\\\\\\/oP1QEmp0jX4nWcxB+eMTe+TO+pMlpWKicgj9TotvDZVXWqrYvgtXNYsZG2f+GbeRWbo+kIeZ7s7hTj+MRb1\\\\\\\/FATWOJPKg6uXs38pV2P6DdJqNo4fDLJ0T2ReGMiz1DecXTsy6bz3ibezK2heFUQPu2KZa3sNW1O0ZPhJvxDL1xNHNI6oejIadBQZxQhsOMRFSZzWafdHM\\\\\\\/bzTToWlH7QWRYYzCwtBq6yZEl3YiBnVE0xCTUN+Z33WeBJ5r5R9tz9lB7jnoD2jmvXIQa\\\\\\\/JMiM6YjPAhK1KeAGcohaX1dNjapGxKsMyfh126W38YBy\\\\\\\/C43Gcp2mjDr5U++DRLj8mUrfugH567T3RgdFAJG5X8UrafD9F7urx85XFC6Szzyady1n9wnzDFK\\\\\\\/SYcKruFTQ2qwM9QirK0qBTCTtnZopnmWfId2NKKStqnn15t1YgxyyoF0t8px5NMyjDnl\\\\\\\/ojLGE+TQlp3AtUqrhwqALWl\\\\\\\/AQsNvCZHRCY+0QEQTV5MTMxcIzFtwDb2bOaRv12GFNwz5fgH3UmwUF1sR8CCmKFtMKIyma+DKM79eaC2XOINtDXgRk6GHor6fT4wRk8UY1BrANPIvtbhoJB3AY1LW+\\\\\\\/FtcZkdsT+PdvcpIPuZckasuKEIC3ZNHljVniWB7USHMY2LTOS1gKXncTyfJ9cpdnCR270lz\\\\\\\/9PaYGGt\\\\\\\/p031oUE3mKcO6MSkK6RTFE\\\\\\\/IRbCiEX6XbcS7wR6haq3hj+961Pgvn16Hh9O5fIn86gq35VHuL4Fbg5kg2+6QdXHyNj0eeiqmHD8KQ2\\\\\\\/r1cE2BbaJ62Gy6qb+9qBI2t\\\\\\\/y3PsvK3\\\\\\\/g0UK2XSTtmS6nb7nHUF7RUZRnMSqZfKlH+5k7k+B20LeGTDZjfGBMnI6z4oGvTcgEmOvlUi5SY5xUtG0a0jJl+Ls0nv0gLZeoxqWJr9v3miNXneGYgSXwsKJNmnt0gfF5rF1jDUs3AzR2yIQa8zUXb3ZT6T6WjyLSUSkydTQzqJPCKPoRpFibh09AUYsLBBDy9Mo5CXFqnNV\\\\\\\/xuadfWGM5swNR944gsoHQPaoPfeP6RqTx3P0x0J+WjqpdZ+rOqjge+ZQCco+3Dy\\\\\\\/jGf66vmdH3FjMmbDmAAk6CoqghIvyKQ2lY7QbMA6cyM4Nt7OqxKm+AAKjQg86ZSrAuWasSVHDZzMvKpBWcMu2ChPwf+pHOpwz84I1megMYzCIQeRUyyVG7elb\\\\\\\/c5klFu\\\\\\\/HRuFZ3UvQYvGs8H7FJXu\\\\\\\/XyluBEgCf8zgyvpjaGf\\\\\\\/qyxkX+w+ODTCJoXun29KyAWLGyUGBJ4+7LgQ6PFju8q9jcYFWQn8b7s2Y\\\\\\\/h1mYpVE0OvNeSYGimy\\\\\\\/dPahQ5ltuEP8Ox4MFMcwCHsWgzd2pJNghlL\\\\\\\/gHHAoSc3OW8AQJQfxvzW0vPhU2frWiUJyQx4cxN2uViP3QBJe9hQGgcjTZhAphGIEdZG2tBuAuogQU7DizPyNWxFptSDoeeSB1LsUAsq1\\\\\\\/M8T6qInqTfSPj+Iv2PsrdfDsCpG1Zz4P6pClprMPwgb6EJ4NgAR5tbzK74yT\\\\\\\/lGskXX4f0KabejIsHq0F4i1ROZ79Z0R7mzQnOUKlkLJvFs\\\\\\\/cCliitrYBrAAbTEBazbMuVQ0pEHTtgN\\\\\\\/pvVEQ0n\\\\\\\/Ch+drNsfEH96w\\\\\\\/gG35H7nHokZCCgnqDH0YmYX0L2fNAFirCmJryME6e27QX09cb\\\\\\\/IwwJOwLkju7nYePE+lnZg\\\\\\\/zlkAfjV0ZmXsZaBKTFkEb4JkbiQFsZ2N\\\\\\\/ls4dCMsqink4urPLOmaTuzQDv2c2oklrgiD0i8Dzli2IcDEy0DldEfPW1w8pS0+bx9AeTCBMVDKF47YGVqPId7t5bZpXIyW2OzvkUUzTUvTiA4vS2Z7pemwxvmWzWVBhLOZxW0YmLcy8+dHbzMlmklGY19QgFgjKox98Rf76U09xdMheaqSpctJZuHWOWYboitawK2Wd9YitzYPvb7rRhaaZHapVGB+nWCZREFLbgnxmEDrCXygNDKpz6h\\\\\\\/TehOcoqYKZjs6ZShMHYmLR\\\\\\\/pIm7jYJ6oOt3TrOLlk\\\\\\\/HTq9GlGZttJvooKfDTj87W7+Dce3Nl\\\\\\\/AGxkIXu3XpocvjXK9dAV6gW1XFq15xm7NMQe\\\\\\\/1O\\\\\\\/maM202rQljqX8++QxBzKTVffFaEUk1ooB56C8AVdxYy0QA8UTcP5lpeHpnjKGM7SGiWCiVV776tE4ZxU15bYmQ3N7KIxtgKojiNUmPdsapJRL+EEnPplqlBn33va3SW5PARGu1t5NiKVFOQDnbjjr4JSv7TWBLAzOGI3kSS8YAu3bNR4dGIbs9RIn8HJvdwoPXDYxZazWmZ4PdeP232dCfOQD3j14H2maH3ELj0WzeNIh5Moa22azNBod9m7+MQiBtPk4TKv2ettXTxukzzb\\\\\\\/rV1CU2PmRNIOIE+WZXztkZgvrfYIKLuUKmc+MXYr9IFeUbgc4BO8YKS5KxGZk9sQLG05QuKXEj4tDEFHLp\\\\\\\/uUo\\\\\\\/AEme9NQRRHKuedCs95LB+RotlyzTpxPCkPNnrEbN3UMMhoUou1MeCvGns6ILYlOxESLOsYFwJ0jgB00679bCd9IwTVviMkhfEiKd0PWLSTAyV3z+VzwrqMV1YUN4Yj0ybVy0E6LgWiatNPQd0Fo46knXIHNTyKV9HzvBzmegd1MZxrVMxWDnysxmobMaQXwcR93LAxT0TSzGCPohiIKtPNJzjAUYcUB48EXiGHl3fuzyHq5MkHDbaJm1BwVWWbFpO+QodVDbKxq+Zj8TCSVBuUThv9U8xgCf9Wr1Pae1olwV0m1z9a2xG7mE3VskzsgUhfvXDpTSKzH5ioZ0Oi6OfPc7P+6vB4N3U6dFMxvK7RCFZvwylbYHzmMc1\\\\\\\/qtp387upEkQ6UmYzaghzFBnQi2tGprbR6RwHN94uYdX8dnpC4dVTcpVb5bHqSu1dD+PRPWl2wr1hg5g8wP77UBgVvQ2gMACV04iZxG36Y+ANdMOrYOMgGbDGmQvu\\\\\\\/NmpembRW2DVod\\\\\\\/fvyEJ462Qc0YtWeslL+mubC7Dp+PBiSPmUm0yqsHMx8jDy6rSukAapXpBS8ian062RADYRenT59jqKe+cz\\\\\\\/fpLNC0en+kc2zoFpsMgNLob+QQ2jIJ1Yf9muRnI6tDbTgNWEqSHv3UJfoL2kzx9VcPIFcWNvw7nRa1XLX6wmH3MNQmUxKSLPjKtf8Qp\\\\\\\/erz9Z\\\\\\\/5NRmLo0fJHU+Hm+H1TPAGaailqNoSd\\\\\\\/1HNYh201+o98UWVvyLASPUoq\\\\\\\/JjF4X4Eost5pKaKOepCRiB4WdSwlov4K325jtnpre365NJr5zK+K8H2zsjAtIjjf\\\\\\\/1wj+BDVFckieGi\\\\\\\/3tVcYPcRDTNYbMUlEhuL0oAYhGAmKYTGrm+YOjYvFy6PlUnw0A7tYuGThHdWkOt2FYfNr6EkXOiqZcMtLpziNC+7EelFXFIdro2PUq9sn9rq1dpf27a+UaEGPPmS3NglZEM\\\\\\\/im+JGQfuYRductQnbX83UsDYbLmxtNBi\\\\\\\/g2Em1iU9s+s5KCDnysBRCBjgeePnMmbhLXQ7T7SX74n2UVMcQDy2sNkD5pEI9hEMUD9yaUf7cJTmszomM+ZsAk8ANxWia32wvYQos2d7sUjtyqimo3gGIUEXurNGoPHg+zB17eDYVpO8UGXF01f3239Qx\\\\\\\/aO2qFw0K666ukYuKl4S9bDR8THWBSI1Ww9B+GdMVAQ0NjW4kHmLOFmMTZfgyyAAd6byMwcjQahENOAArbIVBxAPpEcQnKHRDFQg7I8mDsx4Br8m0Xt5bOpnWTcK2BjWaOdG132TAP0iMulgNhaH\\\\\\\/7pqgz\\\\\\\/E+hQpwKpabFYjK6TxS\\\\\\\/aIOdwW82oLB6DvamYtjUdHorqwNRpHtD+UY5jeitZqpMfTGeaZuUot+d+9v4qwCcwWVel8EvvuxmMXQCIRK4JOfcze9FEi\\\\\\\/0z7eh1c9rESy+xqWKj4r3ZF8K8dywNNJD+ihSheqthP\\\\\\\/2TnkIGJV\\\\\\\/TQAHNE9c62evJlGlIRnfAFiIikJ+LbeUYmnO0QIFoU\\\\\\\/MAhnZ25csQo6fu+EDroexpVflefQyZKe55n61eUuTIz8JoyNGQgQDAegIkTotFrh7Sgz55AnPyhB882nlZ\\\\\\\/Ac+vJtIWZyDQ6wYq20QxyKNXF9m2iOvkW+CLvMxvn3ySeM6ECND\\\\\\\/MbPmo7eijxQKqCxccvqL6H7Lh9sKTjYV16W\\\\\\\/zz\\\\\\\/Cck22os7flO0U+U7drwkKLHjzVTHOMWA7C4nEeGbp7QEwYSi\\\\\\\/DZBxzVYkEgO5x\\\\\\\/5LAogc15ZEnoUm9VDSVjbG9igtevYInDGHEjK+aXGxz26tkfoGRP1DY9ZXDgPeQ1OaXqsHslnGEFqFresbL0D22qd89X2WBfcoJGj\\\\\\\/nb9BICuGI\\\\\\\/4D8Higb9dr3DBn2HoLWkKEJSQItUz7Lw5u0\\\\\\\/lPkbhX4HD5ItugTFWyMUAxelB8laLVAcYehLEp+IdcA5iUINB+tmTd6V5RbIK4JeL3n2Mn1wmHzMqVqKB+rALcowtoZa3Zutu7x4BLoUekGSfb7qujT6ZLHTsTUfl8VtFhSh1RYLwbQLyXjK2JavnjvAw2zJexaWByZS372irpXcNPBAGEY8Pf6ApNdd9rh3GnHXrSIdlSNYZSt8JyW8SkR7SUV0EQgrLnt6umoR+PPb8oxNH+fJCu1LYLkyuMHzw3Hn1jgeARL9ZV1BnKadYeVggpgYL8TFTLAAMv513E9diuc+z0L6hE9xRQeOkGSp8x8PfDhtqTQPalFqD8vYwJPyI5nQZRoQjC\\\\\\\/puAXKMskdbB7BHLiyWjy8uQHjNtUeyccE6R92OO66twHBKNWtKaiPm8oT4lARpm8s9ZM+3PQ+dwXPNVEEapIHRK3fZ9hQELvIaHsmH4scTT9Rax5U+zOtdbCmbrPTao2BFxiCedKVjeTg\\\\\\\/UXlldGM\\\\\\\/W\\\\\\\/+p0iepCw\\\\\\\/CuFF05dXzyQvvKytrHszP0oulLQBwgyYab3Co53EbAAxN+ei4ohL3vb2euZ2MaVWV42i\\\\\\\/Fd9bRNf+x57y+K3HKKrsLrmgUd1F3IJcX4McDU2p5rYaBPl0OlgBe8vyg4a9eyWxugPyhuHJCgZM5TGj+NahP7VhG4J7BnGggltMG0iQTe+Rh1PSQOLe740OCMJ51ENF\\\\\\\/c2hbjW2aEGbHsErCoYJXeKhJJTuNetqkBzgvqsqH93YZ\\\\\\\/dBTM3\\\\\\\/Cye9oxtuSpdNbozFSzATY8Ml7qAjkSnlNBkG4EwwYCwSWNSP4yU8oKLgyapCeOQ75zlHDKCLHPyP9ywiS72H4qp8Pqd9H7+qbKCU2xOkxlqM2a2iM05WkOmpXiJe4CuwMFOSBPiYxqBSEFybH2\\\\\\\/hQk7VJ5Jxkn0H0cyAuc\\\\\\\/yF2CdwOMfSMAhp8XnXea3Bkvf2NWUim44A1VTo2CM3b+fqwu0zFEu9RVa6wj9aepr07IKFk5iHZCUxgr0sruAsNHrqfgz35f2Sd3hXstut46MsU10o8oxLg7CS9Ar5EdXUxZQ\\\\\\\/uVD3cXzlwUxa8BFhZY\\\\\\\/sUy1TPySeJMn\\\\\\\/2l7HKqrCF+C1N1alG0SQdtO1uT9VoAXHn2dovPsguGImWWGrGlaePTcmS1xrN+LtRc4r14L0Ll4PP7HxcOemX0TVI3WGGzC44+QhXQOCTBBELoqldQKDroMLt4NxrOZX+I1AIMmx2c9HwZwPlGz1o7R7SAfHsq7PIeH9KdwfRvCYaUV44CnOZ7UwLO2K6Pj2ni\\\\\\\/YtTYQPCjeb91evBbV07N7wtoh\\\\\\\/mmbEy62kb4vJZRN7A+b9\\\\\\\/kC0lFIKXC9aXcaI2lO1Jov1hz69SPsndKtfCpYZIS+bL401u5apio8N+aM3q2qzdo2UQ16UuHqMc+kY2Wfp\\\\\\\/MIysfTYG0niyYDgTt4Y6J0Cp5ZU21jm21u9FYrEvlr2XMyZ38sSKmQQC+ruGOhXilCydWDXiv4DvfEVNm9dMrV8XYrFipw+JagkitHFJhNHhv524KMwYZGFS1dK7Uhh5O2xZNVVTYFbl80ao81RoadpRprEZKVDd8hREVwCg2pPlrOzID2GQRY6anQjoNt9vb1V4d+iBBLnXUl2nJSuxwOEmlhAAhvADegk\\\\\\\/YVLzL4rxIgv0KnlxPjgnxHsGGgqwKX15+L1PXuUqn2\\\\\\\/+Zz8zNPdhvuPhSHLcccP4Dr\\\\\\\/3P1yC7gTeKgqilhjbDPTnHz\\\\\\\/pizS\\\\\\\/Ase2OsyjIPKOBCP7S0H1+ubO2QfKxV5sDGE9HgPmewBbGwue2Z08OY6lZcKZYquNLg3g2Wz\\\\\\\/tQDd+MtbeymogTirKschmdeC6rVxMzgH\\\\\\\/FLGNZp\\\\\\\/6sXSUIxi6aTxrfSL8ifuJ\\\\\\\/YQShv1iD7r8Mwh3JfkRz2F04mA3o6Sa11NuUOfbZ0v6iKLkC1btxpCZpQ2Cs0OBI2VrCTORNBh9eVYXugrek\\\\\\\/xJuqp5Bhi8X9M4EKcuM0YLuhehbAzllKnHabeMjXrW\\\\\\\/u9R0XseYSDOOrel5P5Z+qIuR1nRIBMRPXoj3FNyI3lTdp3ecIoQsVCbUacGQ3uDFIy7pdo+sDryEU10W\\\\\\\/\\\\\\\/38u51cUaAAyPJEqoTaSd\\\\\\\/tFgXV7J7b8elz+om5H9niq7\\\\\\\/fTzJoVlRAdC6fcpkM5nmVVmbN00kSfVl6EQmPsWERHO9PawoIhOpCkkMNGrUdjdfeMSeXtVAfiBsv0RC99YIqS92KybURB9toGz5t43INUs9OM47VqLQiX00IneWrQPvyDWRii4WswpyiB7VoQIjM0RqwPEzc02IbZ6raLKbG0VFvcX1wAYCLo1dA8CNttnlA3lzVshVdC+aALskV47EIWS33LnBkZB5jv8Jj6TOUwF7wdRua8nGvX9kAGLKp+MdWgzNE2vJFq3foW\\\\\\\/JrV+y9utDMJ+NiZ4YR\\\\\\\/WGBUoqZlYeagYbwMqqg8jp9TFEEauez5sNXYx+YiWpbVuFuwtmFod9PorKT+y7ViLuycGxbU27lMM+FG2TgjE02K3cQ+yAXYI34Abg50vVrl4nw+CMzjES8txRB+q4\\\\\\\/WeQ+DsNRWWw3FGyg9enYrXxlipRM\\\\\\\/AQrtKnEiC5kyCcBz\\\\\\\/FRfsrOBW3i+OSr4yu7Qph\\\\\\\/ge6LceT6o7M+CjeZlhxsi\\\\\\\/wulfhCOj11jxtzXooXv2QxXqXN6i43dzipF\\\\\\\/W8vPz+uHDuYCabfLsVCFfpmrpHaFnMsnmyI0q0MJiKO21vCTCF8LPxKFDRti2H3kcfrWpNLWGKO5CZHqUbqiOGWeITn6vzukINW3EH0eo4bKaFJX8u921U5zP08Fz9uD3Mmzld9HvhxKeA\\\\\\\/2nYdGeHq\\\\\\\/KDAwqTugnexOrjlLE7HBYn9idLYX7weWkPz9Xf\\\\\\\/5zE7eFZ8N+UKJCMvgVk2JuLzS7Kywda911\\\\\\\/BhaMDdXnCwU4fHjintt4yGefNYsq5s4PiJBVHLdhBiOlUP6maHpzzSzlhkQb1JcYh9OMJ4bjQ7pYfy+\\\\\\\/SQ2T0LHgmO6bJ4UIS6tsp8ycSd1z4sW5z4MFC8mHEVTEGGm0GC1UlYQc6AMDdPQQwSKFuGn7WezbCVwKddmzwtG0syWR8FcrvofNRQhwFOTQrxhwkF8yLLz0H7K+bBKNiQjsGs+sysbJATfXt0gaWY1QpQHNBxUkdHh6Lxmf++iZlemu4vETf7qHK2xJ6F2tLSkEB\\\\\\\/dearmQLAI2Jg0+yeYewXHKSmOhKiO68gGB0Yc8Xg+ig6iwUZlYPGoKbf79RT4SYj8wkGfsC\\\\\\\/+Flh5I8fHN7dZTbigw0B\\\\\\\/2rd4gAcDLfaU0AFCOkHSOijbkNuJ42Kf01bbckuffVTfRJNWS21wo2R80hZt\\\\\\\/3AohuDgX\\\\\\\/88OFphIX\\\\\\\/lTbJqAE7sjE2a2uuoOinC3mS+qxFLpKU6ldoNLbkFDPVBvvnlx5lZIzpewRhfvXZG4QQlv+aawTPUo8Y+6D8z91rZyWnFmiGs7Qpu1wyFcDPADQYhrRg76HbFqWlzmxUVpkq1f\\\\\\\/c2BlvWb8Ra\\\\\\\/s0xVbjO\\\\\\\/VPXtG6\\\\\\\/rKQyiQo8h4uF4\\\\\\\/5h+GcsljVk4aZIHdcpfN6EGD7BgCDyDxTs7DkVMNqRTxuNPWiiR\\\\\\\/hyMa5WRBCPRKAjUmhlk59HgLgzFuzSjZsScURkuXWvNCqpmIyxk+VUx7mgE9lOxqMau3QaZhevwpgEY2JogimF5Eixr4IWRLAJOfdKgLInOZgnt5pLuXtdhyOQF1VQ8j0YzorT8Y7XvNidanCE+OXvcsgZzmHKfsgrNfOmN3BHfHAOxv9nwdD9Nw8Scmxl4pfTt\\\\\\\/KuRlc2Ov4Cf9PTsBHEEISwKpQjPyG1zvUPzciJbK9AkyJE2oiEBVprRq\\\\\\\/bq3pQXsVv1ZPE+l6ZfrevkZa3ykkM7NXM1131QozOJcNG20aQENPtDzYOvc6LEUZoHaeWkl+AnGS7vGIJHcsOX\\\\\\\/C+xvgFnlnuE74uQVXEtcsc3SrErhe\\\\\\\/kExfkv12l\\\\\\\/mxS9jIzf3kMk9oE5BtOxzW4GAISHeVkb94E8I0byUSFcYJxUztTLTcfKtrKwQ\\\\\\\/jN26mibEMGKdVr2o2DA478NtpHBZxvTNNmG+8pPuZygR9Z\\\\\\\/Al0d+6Jsr+vzjGzO2Xf9bhhGb\\\\\\\/Su\\\\\\\/cK5en\\\\\\\/FLgW1D3XCxDUVeL6S1zvZhkFuKmJvl79T\\\\\\\/3qQePYJJUtuVDIBNLnwPL6hO++EKtGZ+VWRrtWwzpF8526Is1zC\\\\\\\/5mcEBgaU71jW\\\\\\\/+LpCm\\\\\\\/b2nKU4PIh0PxgTeLf6LHzims2\\\\\\\/aYL1APXd0rcUSnoDYFLGmDkUPIqmRFsQSJQhHcpqpQ7WGgMoc6qBsLpc4crW9IpVphsCPbemt4MFTZS1zjPS2ZOcgF9BaMMsebqUcv2A9vQ77FGItD27JiN6\\\\\\\/sIYVFROilK7QSuVJFG8AMU9kFYJf4HWyLCq9RFmqPCe9Nv7\\\\\\\/4d2b\\\\\\\/Yguy9Ed6QrGIVt9p6BxMAfL8yIKfbuNA1OJ1tT+tCp9JDX47Jhj+w7D7Bxm2Yxd+G2dJ3pCbPJGBNuM1ae7C+psNq9TqQhDxhHP5mTxfGplFyDvXZj4BCwxd6BLzclylS1HdobwWNHUKIhmDGj8GhHqAomRqRgc9gpefFoXud\\\\\\\/7KOYg2IJUopRbXs0365uJNRaVYIVuSPwk0jwHsnMLskHlNnQ\\\\\\\/WQN4mVKfP37vNLBdz6q76XnpkhnujwWlbneCxByf\\\\\\\/vz57LBHldGxh3cgTbMRKcl+HL9Ij9SyGtERGy5n7Sux3+NQpMMWb7tDj0mksDKzzLn2kJlO26EijwKfP07Q+ec9TBHIMgOzLOOdyLxhvgWWVL+DN0axaI+zhRlT9XM6SIsz1YTV3\\\\\\\/oee3fnPMcauNlmaspLl+fRmD90SZvO6\\\\\\\/GbNG14LiuSpCcAHsmHJTOlnUa1FSp\\\\\\\/uRQTdw++LxPlvsLaMD7USuCS\\\\\\\/nxKwotgd8SC8qK7n7ati2TCibTSwKnLlnuuB3kHRGpiYcKN5L4dv7dfXV6xJxs9OGUouwNHWmjMNUshLhJ4mRwbK\\\\\\\/GWmBf\\\\\\\/BnYLgDfeEhLrdj\\\\\\\/AKoMUM+artz7oAPvUlhy5F\\\\\\\/8JadEfHVmKj1kQ4TctfmxFyob1AGUDvlUdokgAxoHjGMB7w06s6UwIN4c6Yt6HoRYvszccgKLpxQfr+BdaetMJBUnQhNWqU5NaKA0igWTpPUaWkZHMkC9zbN6LxeeH20t4R+l3pr71XxHVPohMlBXnxGpcIniCCcP0BjmZBU0+0nDZ2XjtwH+A84OSVQlPRF2Jgj+g3btppiy6eyNw4wLFOVhxmSd8qYXR2vYBf\\\\\\\/Wj3ju17XdpB5Z9PZZAJNHI9W4\\\\\\\/+wWHkRYDkqjwGoASSILLSzO1Bb5jLwq3ilwQxEclBTR\\\\\\\/jvrJNGhZPkb4PIuhCtWce0DAALD4mYQJkPYLNjoMKIRg\\\\\\\/PRBf+vXyGCeLuoAPeA9F5jTfTF5CDIhO6whmTLgUfG\\\\\\\/q+jA9GMBUTBYNVgDr6XLEkc51jL1XSD2zpB5CFuDhYNNCy6y69Py0O69lAl41PySvUH71UnaRbL7IHjWXlPY\\\\\\\/HqnJ\\\\\\\/YOxfFT8DRc24zZL0vnukC11L6Itv4D29PSdKNi0dAmIeWNxPdTRw7HRA3NN1Wzmry9hq9Q5nlpXnq5sQQF0Mc2E8Vloqk5Rf2pb\\\\\\\/rt+keKFzSdp3NqXkUZELDFLJrYLMangqibrIdS8C\\\\\\\/6lCsn6EyOzbI2RR7xaS2VhuwjqQp68XDtt3CFV4x\\\\\\\/VKWvXBqKgik2pQro3Iu0h\\\\\\\/CnDJXbPXor7yOt0ISiXvbRWLTSYeykwLaxsSL1nXGYYc9hlVr2BBCWoSlOYYSd7ny+ITmGZi8ikA37DfzOo\\\\\\\/WyrqrrBwuuv7EJD2MLQcFXtnjkvTIoq79S1coyrkB\\\\\\\/ebM1dss\\\\\\\/NVn34QyOcOBCtuY764Cn6ATYcjEalbXxR64yWeL7Fzn0FCIDZYZGK3sG8aRvbP\\\\\\\/n36xcQ\\\\\\\/KVqB1JmrzoQiU6IZrsMDSjND7zPUFymy6J0HUxJXBaisEI9a3gDeuqzuTY7V9rmLsFGk50HIXdpne+AlPq+AFWCcfTvP8rNZ163DfjLzWzAnVGMzgNR3dUaF1LYF0wSXw8wfalXuqGoPR1GxQO\\\\\\\/da8VF\\\\\\\/qz\\\\\\\/whKX+ysvaNBeKzAimVKqKMi+RhH5pWeFjOo1kUvSDfoYWdmYLfVKMPVLRvayP+hlwcsfwBAg+nz++2h69DRsVrk+PsDyVPp0egtyNd7KziJDPvXjDWSF341KRer\\\\\\\/V7Snh0vbsbQWXm0dHt7PzN83bZPmQfflN9HIgsop6aCsPLSMOFhfCuezDSgs5YkbjUHW+kl5Seeo+7DvN6Li6LoI6J03c1qsA4IVU+kaxjQFhp7ki4m9DMTdoA\\\\\\\/\\\\\\\/3rGjBB3Ogdv+jeRvrfFq47AWKG3Aqq7Ji+rvCvlCNero0VTxG\\\\\\\/jyNJBXxpkdZKT1Dc9Rs1H\\\\\\\/Assbd4KaGiIbPVHwnVXCFhvssar2DaLRA9d326Shf2BSOvM2HbngdpMhl3j8\\\\\\\/61q\\\\\\\/DmqNyY23fNrDt8JrSH5zEjFYR33qbbrh2QoqNWE1XUHi1W9KaXCFpb9D971q9\\\\\\\/vd2OrAS+D3Zq+V8rYg4ak+rohBNt9zDU4zGzSLvpjOn9N2nZS7Kimp60oZBg\\\\\\\/X1cfjmuOngSJLR9foPLPg6tXee32Mu3NiqLGmBzpt03Dti48Joo5oUKmtqK6aGBk1kes5JMzB7E046KbynAbVe4vft+puJ3Gyf9+4AoEfdKtTWiMJqzL8cKOX\\\\\\\/kcaBstgHHLbaIIQ+kTQ9gz84qQAd1aDUuMWlVq0RdtWa+QagVutp5lp9XzH6GNI4Gb8ZGv4MlwHD1RVEmUhPucrxbcdvBmI\\\\\\\/v+Q8GcFvbVKl5YR4knUCWkGAKSNu51rcZoVx3L7Rd5RCYUPiWa0PRY9M1BUDoJhGi4py3dCJNfLu+kagkpcikLyVYYVVlqk\\\\\\\/pP3ocEr1HVeB2zH6NAeZtM6+GuOZ2pWt16zcCBd0Gd7+81EJlyodI1LozigZo7SsIPhpxy3WPlzGZ9boX9FIdq\\\\\\\/2kta5vqfCCUG9JgUYF6pTQLANsZ5WaX4OPbnmN\\\\\\\/2Kz6+s88Dks+xLPz+i4EWy0e6XVTaam\\\\\\\/yxen4oBzfyqKXpWgF0UMMYViUgHd+546McFG8l3N\\\\\\\/i532m38YBO5G7saVISnJ14by9e4N9sh2H3Id6DQRXmcrseyf4KH0goobArwOdqT8KbDalYVzoPKxG8A12MIMYtq1cOzSVdE0V7PHDEGMqmmgmziW0xrTqk4I8CrLjmOyVovHHa97ukem7jeiXfv6ytRfXTRtxdbflIVKAqmj2MGb7+EdAcbhO7djA6fOIYE4oSdNtPW7pP5223nCHEWfUTAn9GEO4wwtIX8EjL0jBUbLk9VkzTm7NHiw1zcvT3xKpLQhcBHZM2aK9E1uKwFsKuaS4r\\\\\\\/ilmDpBBnIsvMIxCHQgvBXraw16yCmhmgQP+mDuj1WYR+ugVmIqGX\\\\\\\/tDoq+fbmLLK+5o8v85lNbHxQhXuK+cFEa6CW9HFNQi9FCRn1nLGhOK4bQ19UB5mmWA3ubwOljOVjAwAPIWiLHRqZxaGugD3AwOgsyditavXavv3\\\\\\\/bywDuS7mej80O1Kq7CFKxGFlhhUHav4ruutpPxX\\\\\\\/W+I\\\\\\\/fA6mpaLU40mLi8ED9\\\\\\\/JxIM5OSDBNnQBfGE+1Cz+1+nR5wwrD8d7aBQ8k00goxOgljrIQ3FP+AoOmLig3uWfGNzBrkHWf4NdEYXSEVIj52T\\\\\\\/tPHvXMcSnHIJFhlYV0aG8FqPXe+fbUmbzeSHANI0gWd2ZkAFI8oDgipBbahalTyDJc5iGiMrYM3QnFWLQLqzHIqbYC3pIj0bAhLxFzgf2rV+e1W5W\\\\\\\/KhEdbnDHAkapX7ktYPsjX+rCbN5tocQpIX+bNVRBtzRN47UsD\\\\\\\/G1QUDHtxPHuttNekRpVgYHcNAYsCXBcjzCez6xrQvkGk81SI+oIuea+3mFfCgJv2ylkpUw70o1XkZFyuDEMszODKtIzTYOSRxPXti0s78FvGgi8nHYGTBVcpMumk3kqxKmBvC+EzebxHE\\\\\\\/uWiZsZCN5adH5D2DwiJZ1Nuw39kCaVa0q1EnM9xDrXUWpWdqfznNmnZoUWhPAaaz7DBolHnZCDlQJ+LQJK1VqfEmDaJm5DiLptka5FQkkEwvwfcl\\\\\\\/FSCy4ki5wmetSbpy8PSVBSiuksM90q8yHkriWkhcrzwIu8CnSneHkbsWrewdTcqH90k2RIocC1lSvBEbYKqZovKVhiK8WzWuxIpDb6yLHt8SizAImgQD\\\\\\\/ge36YlNZZXq3LAlmATa51ikETKRiwNnelqyx5OirfiXWOezVo5FLA8LzWIgh1FiyL\\\\\\\/pStjA90hR20\\\\\\\/wtCv5Mg\\\\\\\/wmDGyMK6kDX6bd91dMTu70I\\\\\\\/huRU\\\\\\\/TivE65dKSF7sxb3dcouRse0P10ZlqKYgLNVN7KSgFq0o2hy8F0LfJzUKh3YnN1wXU+LQ5HbvYtVyj75+XL6I1P4lHm4fdGMye1n\\\\\\\/k0JO2Odw97qQ\\\\\\\/YdtFhvQfpc8KHapOEejoNda8Ao+QaxFaJTicsTiHTCJbzaUYMmwtNF92Ji61Y+3q4z7\\\\\\\/VPp22NFOZWD6gLsY+E7imVYtSokAHNfEEPc6S0JjoSGji\\\\\\\/amVIP65KsU5T9u5V7Z9E4fpuYJV5c2EOvn8EblqSxgBgfPuT7rnrbAkRiZjRfXvpLIMTj\\\\\\\/BsXoXRnKL6xUu1\\\\\\\/cgY3+6jyWTMP49f0F+pGBYCuj1dFhuW0DDKqE9ms7EQu4x+z8z\\\\\\\/1w2\\\\\\\/mhEFaiGxjlWgbjRcGb62Ce0\\\\\\\/j4tprwFwaBCmqqG56cxMb2uDMlJmdeaOXHv7fbcm6tx0WxzKhF5Im1VAUBtgCYp88AIMlViTM1D\\\\\\\/6N6YSlnEcOIVlE6HF7bzUMdtyPot+O1\\\\\\\/1WLlq+JTvRba4NLfBCNsj4lsi10edFbAwlUousTd3Ihe\\\\\\\/YkwPgl\\\u0022,\\\u0022iv\\\u0022:\\\u0022tMuWf24+ovkBuYAbvt678A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225d00b6737a58a47c8536ecae61e6901d8d470075294b2ada9b64c406e9b8065f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ya8\\\\\\\/9uKP7S1+bvvt8ZprxalZhJvrkNEIvrjl0zG00xTpErJbLef9Fj\\\\\\\/9G9h7gj1WHpECxmGQLy2oR3iPPjJPlCLKO7cw7hIR9vvz4FoTepHm2FRgs\\\\\\\/yZwW3Yo61NU7ZJA7qg4u8I+HKucUv9g0ptBW3mAq8zgRfd7RKmMtAy+8FoyPW3h71Aiit8TN5ZecMNXZ1Rwo6286PZThXzsmdmmhrLOEVew1V7D4ViTTxHUTVsplVf4UVTAV3rxmiQYuPp6GMpcKu9Ah910P8gObKMlKpAA7\\\\\\\/6\\\\\\\/mpRKQAxT1NsJkn+1xEjhQnaUgkUR1oot8Kwq9XyhmjSpxuGoTM644c8Vrbn5D7gAF6HwK9Rxp\\\\\\\/PvSmx5mViTaF43t9ScW57Sb42oxV3UCxgxwc0s4vdylTXYhMYt6ZKVnhtY9dMOVXnhg5x+T5q2JKSQaRPKdrVkYyAVtXM+iDs6296gX1xkDq\\\\\\\/ZbJxC3n4v8ntZIBcfwPLGTsTePjky2gULOQRtjTBjXyLWL4QzRc9qHgU3MwGWi862Cq1pbYQzgeVyBrU8UmWDX\\\\\\\/HMd87tFvWtApzU6SKHe4xv9iXWp8fWB4J0Ec\\\\\\\/iqX6lUyFEicAhz3z5ycKUN3cjIunwrJqP0tBJCVcJ4xbz4u7PQ6i3yavPaD63h7JaJdVxcNFBslyTHDgqUx57L0llqhQV\\\\\\\/Olg66Cq+\\\\\\\/iXnERw1ABZhrSazawA63kPRptnVKoVPCTP30J+tUbURmLPqKXKw2jrTqyqsj2H7DOk3Xz6rYdJAS1DLcgKM+rpPeYU\\\\\\\/nM0yRiFwiJtsG2KBKvyMhUyPin8vYr87i+ncud\\\\\\\/L4lKG8CJE1iXcRA9rcppc+6jwGdUQHVUTKm9L5ijKLyKoBuUSKbSOqSH3IdU7ChzWMHkbfzSPJ3dIQxn9BQKM+r2SPS1zode4gqU3s\\\\\\\/MMkhon9qtY4CCV8CiPEM8WZb9e92aIAKNMi0N\\\\\\\/EYcEtHD1lQlDsAL6NdLedsyUr2iJaGh9di6JygZTIFbZFJ1vBTLpwduWhFFkHmfjfLgBBZx31E5aMqXSOpm1BzBTG5gXDNvdqj0dcMcwpzgqYOCxpV6jwyEpUDgbfVh+KtHCzNsc3ayXnyTgnNR14u2bC5pYLToh47ARx7SizMRSrI2dEHTMLUBBYMJ8PNsqBiPJOLFOvo9N7NotO4JBWxE2Qg\\\\\\\/AuZzAWWxaPy3iSuscxRC8CZBnMAlXCg3kXiDcSx4Nc2\\\\\\\/tyLWNr67WemO5Drc6YR0FALDnZ6+0ryGfPj41ZV4iaZItABnmhqEqssoNm9WuynSBLBK0J20zgnurY+jDaCPFo+3KH5wEshFyq0wbkurQyfb0BS6rIXT68FRljjlOpkCa7ztj+0Pd6BGLHw\\\\\\\/ZgQhn0buzmILIuzp4UvDVwCs\\\\\\\/GV\\\\\\\/mq0y0bbb84F5BtHZQlBUgqUE0y\\\\\\\/yhb68zPmLsSfdvF3T\\\\\\\/SCOhNqCYwUSgget87qPJfMTdNjXydQg2c2ZsrRYpi+Oclc3CSUVXK4oBjPUmN2xz2L\\\\\\\/bmswtDX++no9D8Q43ZhZHEdo5ySXoSXzseEGXuNnVqcrpI\\\\\\\/\\\\\\\/MPh7L9t\\\\\\\/1fdBcsqWnZFbpoEEdPtZP++CiYQpjP7BnYqth422rI2M9i1VF\\\\\\\/uwxxTfAmNuPZAUoXzAp8qMamh5oVdzaP6wIqBOC3wpCSgAChA8Ea1vZSGBokiTpiymfecKNnQKt477xPSuCYr25JSDz696hqa6viqFOi6I8InuFQbSR9y6kIFSyfAwo+zOmotbyRDPq+MQYpUBXMZyK0rafEfLzTHTfpITJ5xR8KjH4h+CsSbulrpo55hDULAXAOwj4c0wJJD63k\\\\\\\/m6qY9ZIoZEkByICW1Z2+i8QvyjA0Y7wFNMJNGttnVelNaY05vm\\\\\\\/y2MRqUUWo9a2eYuogSzCWtSw1rieCdCd8BflrITC2iXw6h5qW51ju4yXG+5UlX39b9td+6xYjcMnK8x2+Lf0D6KMsWt5Kjb9A8f3cpWNVZ6mIfiXB+5KS5PuYjrlVruI8B7NMwsj9EA88c0SDbSs=\\\u0022,\\\u0022iv\\\u0022:\\\u0022eBgDHS\\\\\\\/oifjjvQqGQeqpGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022efaf2cc41a450a410bdf47c5a7f32ec1f0eaa819887b88c9ff1b45d01bc57762\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492884 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1385" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qAwNQJ5frQVBqne1c_KjOH5AfjBuSMVbC8JJffYyl0M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bjjhEval7rAfNjbdmqZ_fN8ETrhH9t21D0B73Z4fkg0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1377" title="BGData - A Suite of R Packages for Genomic Analysis with Big Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1393" title="The Genetics and Genome-Wide Screening of Regrowth Loci, a Key Component of Perennialism in Zea diploperennis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492884" id="node492884" data-pisa="ggg;9/5/1385" data-pisa-master="ggg;g3.119.0005" data-apath="/ggg/9/5/1385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Realized Genome Sharing in Heritability Estimation Using Random Effects Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bowen Wang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0198-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth Thompson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1385-1391;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.0005 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492884" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bowen Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Washington, Seattle, Washington 98195-4322</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Adobe Inc., San Jose, California 95110-2704</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bowen%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABowen%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Washington, Seattle, Washington 98195-4322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0198-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth Thompson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:eathomp@uw.edu" class="" data-icon-position="" data-hide-link-title="0">eathomp@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1385" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492884" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1385.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492884" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1385.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492884" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1385.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492884" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">For heritability estimation using a two-component random effects model, we provided formulas for the limiting distribution of the maximum likelihood estimate. These formulas are applicable even when the wrong measure of kinship is used to capture additive genetic correlation. When the model is correctly specified, we showed that the asymptotic sampling variance of heritability estimate is determined by both the study design and the extent of variation in the kinship measure that constitutes the additive genetic correlation matrix. When the correlation matrix is mis-specified, the extent of asymptotic bias depends additionally on how the fitted correlation matrix differs from the truth. In particular, we showed in a simulation study that estimating heritability using a population-based design and the classic GRM as the fitted correlation matrix can potentially contribute to the ”missing heritability” problem.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/kinship" class="hw-term hw-article-keyword hw-article-keyword-kinship" rel="nofollow">kinship</a></li><li class="kwd"><a href="/keyword/random-effects" class="hw-term hw-article-keyword hw-article-keyword-random-effects" rel="nofollow">random effects</a></li><li class="kwd"><a href="/keyword/asymptotic-bias" class="hw-term hw-article-keyword hw-article-keyword-asymptotic-bias" rel="nofollow">asymptotic bias</a></li><li class="kwd"><a href="/keyword/missing-heritability" class="hw-term hw-article-keyword hw-article-keyword-missing-heritability" rel="nofollow">missing heritability</a></li><li class="kwd"><a href="/keyword/model-mis-specification" class="hw-term hw-article-keyword hw-article-keyword-model-mis-specification" rel="nofollow">model mis-specification</a></li></ul><p id="p-3">DNA inherited from the same ancestral copy by related individuals is said to be identical by descent (IBD). In the case of defined pedigrees, we measure IBD relative to pedigree founders. In quantitative genetics, correlation of trait values between relatives has often been modeled as a function of genome shared IBD. A useful parameter for pairwise IBD is the kinship coefficient, the probability that DNA randomly chosen from each individual at the same locus is IBD. Pedigree kinship (Ψ), the expectation of kinship coefficient over realizations of descent in the pedigree, is a deterministic function of the pedigree relationship. However, genome-wide realized kinship (Φ), the average of kinship coefficients over all loci in the genome varies widely around this expected value (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Hill and Weir 2011</a>). Advances in genetic marker data technology and statistical methodology have enabled us to estimate both local and genome-wide IBD sharing very accurately (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Thompson 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> 2017</a>, and references therein). It is important to understand how differences in modeling genetic correlation between individuals affect outcome of statistical analyses.</p><p id="p-4">We investigate the problem of heritability estimation using a random effects model of two components, where the trait values are sums of normally distributed additive genetic random effects and unique environmental random effects. The correlation structure of the additive genetic random effects is twice the matrix of some kinship measure. This model was used by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Thompson and Shaw (1990)</a> to show how eigen-transformation can significantly improve efficiency of EM algorithm to find the maximum likelihood estimates (MLEs) of the variance parameters. More recently, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Visscher and Goddard (2015)</a> used the same model to study asymptotic sampling variance (ASV) of heritability estimates. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Raffa and Thompson (2016)</a> considered hypothesis testing and construction of confidence intervals for heritability. These authors focused on the use of pedigree kinship (Ψ) under the assumption of correct model specification.</p><p id="p-5">On the applied side, it has become popular to estimate heritability from population samples, where the genetic correlation structure takes the form of the classic Genomic Relationship Matrix (GRM). A population-based design avoids confounding due to shared environmental effects in close relatives. The classic GRM is constructed from identity by state (IBS) matching at genotyped markers, and is often referred to as the relatedness matrix (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Yang <em>et al.</em> 2010</a>). <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> (2017)</a> showed that the classic GRM (at the element level) is an unbiased (under mild assumptions), but not very accurate estimator of twice the genome-wide realized kinship, especially for remote relatives. It is of interest to know how choices of kinship measures (<em>e.g.</em>, realized or pedigree) impact outcome of heritability estimation.</p><p id="p-6">In this paper we study the asymptotic distributions of heritability estimates under the random effects model, when the genetic correlation structure is potentially mis-specified. We investigate how impact of mis-specifying genetic correlation structure on heritability estimation varies with respect to study design and the extent of variation in realized kinship. Finally, we discuss the use of estimated kinship to capture genetic correlation in a population-based design, and how it can contribute to the “missing heritability problem” (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Maher 2008</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Manolio <em>et al.</em> 2009</a>).</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>The polygenic model</h3><p id="p-7">In quantitative genetics, phenotype values are often modeled using a linear mixed model, where fixed effects may include age, sex, or principle components derived from the matrix of SNP genotypes to account for population sub-structure. In this paper we assume the fixed effects have been correctly adjusted for, so that<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where y are trait values after adjustment of fixed effects, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-2.gif"/></noscript></span></span> are additive genetic effects and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-3.gif"/></noscript></span></span> are residuals. <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-5.gif"/></noscript></span></span> are unknown variance parameters, and G is the appropriate genetic correlation matrix. Total phenotypic variance is <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-6.gif"/></noscript></span></span>. The goal is to estimate heritability, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-7.gif"/></noscript></span></span>. We can parametrize the trait distribution in terms of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-8.gif"/></noscript></span></span> as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>We assume throughout the paper that the model described in (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">1</a>) and (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">2</a>) is true, and that <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-11.gif"/></noscript></span></span> are both positive. However, the fitted correlation matrix <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-12.gif"/></noscript></span></span> may differ from the true correlation matrix <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-13.gif"/></noscript></span></span>. To investigate the asymptotic distributions of the MLEs and the effect of pedigree structure on the MLEs, we assume there are <em>m</em> mutually independent pedigrees of the same structure with finite pedigree size <em>n</em>.</p></div><div id="sec-3" class="subsection"><h3>Correct model specification</h3><p id="p-8">When <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-14.gif"/></noscript></span></span>, it follows from likelihood theory that the MLEs are consistent. Let the eigen-decomposition of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-15.gif"/></noscript></span></span> be <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-16.gif"/></noscript></span></span>, where T is the orthogonal matrix of eigenvectors and D is the diagonal matrix of eigenvalues of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-17.gif"/></noscript></span></span>. The transformed trait<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-18.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>The covariance matrix <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-19.gif"/></noscript></span></span> is diagonal, so that the log-likelihood function without the constant term is<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-21.gif"/></noscript></span></span> is the total sample size, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-22.gif"/></noscript></span></span> is the <em>i</em>th eigenvalue of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-23.gif"/></noscript></span></span> (and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-24.gif"/></noscript></span></span> in the case). ASV(<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-25.gif"/></noscript></span></span>) is given by, among others, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Visscher and Goddard (2015)</a> as</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span></div><div id="sec-4" class="subsection"><h3>Model mis-specification</h3><p id="p-9">When <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-27.gif"/></noscript></span></span>, let the matrix of differences be <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-28.gif"/></noscript></span></span>. The same eigen-decomposition and transformation based on the fitted correlation matrix <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-29.gif"/></noscript></span></span> leads to the same log-likelihood function (<a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">4</a>), but the true distribution of the transformed trait is now<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-30.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-31.gif"/></noscript></span></span> are the true parameter values.</p><p id="p-10">Let <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-32.gif"/></noscript></span></span> denote the true trait distribution in (<a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">6</a>). The model space for fitting with <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-33.gif"/></noscript></span></span> is denoted by <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-34.gif"/></noscript></span></span>. If we let <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-35.gif"/></noscript></span></span>, the MLEs from fitting the wrong model, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-36.gif"/></noscript></span></span>, will converge in probability to <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-37.gif"/></noscript></span></span> that minimizes the Kullback-Leibler divergence (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Kullback and Leibler 1951</a>) between <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-39.gif"/></noscript></span></span> over the parameter space Θ (<em>e.g.</em>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Severini 2000</a>). In our case<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-40.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-41.gif"/></noscript></span></span> is the <em>i</em>th eigenvector of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-42.gif"/></noscript></span></span>. It follows from likelihood theory that<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-43.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>with<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-44.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-46.gif"/></noscript></span></span> is the log-likelihood over all data. For instance, the first element of J before taking limit can be shown to be<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-47.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>Given <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-48.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-49.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-50.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-51.gif"/></noscript></span></span> for any finite <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-52.gif"/></noscript></span></span>, (<a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">7</a>) can be solved numerically for <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-53.gif"/></noscript></span></span>. J and K in (<a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">9</a>) and (<a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">10</a>) can be computed subsequently to obtain the asymptotic variance covariance matrix of the MLEs.</p><p id="p-11">For the special case where <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-54.gif"/></noscript></span></span> is twice the pedigree kinship matrix (<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-55.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-56.gif"/></noscript></span></span> corresponds to realized kinship over the causal genome, elements of Δ represent (twice the) deviations of realized kinship from its pedigree expectation. Such deviations have expectation 0 when samples are not ascertained by trait or by IBD sharing. Since the pedigree kinship matrix is fixed given the pedigree structure, we have <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-57.gif"/></noscript></span></span> for all <em>i</em> (pedigree index). It follows from (<a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">6</a>) that Δ enters (<a id="xref-disp-formula-7-2" class="xref-disp-formula" href="#disp-formula-7">7</a>), (<a id="xref-disp-formula-9-2" class="xref-disp-formula" href="#disp-formula-9">9</a>) and (<a id="xref-disp-formula-10-2" class="xref-disp-formula" href="#disp-formula-10">10</a>) only in the form of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-58.gif"/></noscript></span></span>, which converges to 0 as <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-59.gif"/></noscript></span></span>. This implies fitting model (1) with <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-60.gif"/></noscript></span></span>, regardless what is the true causal genome from which realized kinship is measured and captured in <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-61.gif"/></noscript></span></span>, produces consistent MLEs. In addition, the ASV of the MLEs will be the same as in the case <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-62.gif"/></noscript></span></span>, which can be easily computed using (<a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">5</a>).</p></div><div id="sec-5" class="subsection"><h3>Method of analysis</h3><p id="p-12">We conduct two separate simulation studies for this paper. The first simulation study serves to verify results presented in the previous sections, and to investigate the impact of study design and the extent of variation in kinship measure on heritability estimation. We consider four different types of designs: sibpairs, sibships of 14, three-generation pedigree of 14 members (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), and the Cleopatra pedigree of 14 members (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Three types of kinship measures are used for <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-63.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-64.gif"/></noscript></span></span>: twice the pedigree kinship matrix (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-65.gif"/></noscript></span></span>), twice the genome-wide realized kinship matrix (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-66.gif"/></noscript></span></span>), and twice the realized kinship matrix on chromosome 22 (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-67.gif"/></noscript></span></span>). These choices represent increasing amount of variation around the pedigree expectation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Three-generation pedigree. (B) Cleopatra pedigree (Wikipedia 2018)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1140530501" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Three-generation pedigree. (B) Cleopatra pedigree (Wikipedia 2018).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1385/F1.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1385/F1.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493111" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">(A) Three-generation pedigree. (B) Cleopatra pedigree (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Wikipedia 2018</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">We set total phenotypic variance <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-68.gif"/></noscript></span></span> throughout, and use three heritability values, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-69.gif"/></noscript></span></span>. In total, there are 108 simulation scenarios (4× designs, 9× <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-70.gif"/></noscript></span></span> combinations, and 3× heritability values). The total sample size of the study is kept at <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-71.gif"/></noscript></span></span> (<em>e.g.</em>, 700 sibpairs, or 100 sibships of 14). 500 simulation replicates are used to capture the empirical distributions of the MLEs in each of the 108 scenarios. Within each simulation replicate, we obtain <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-72.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-73.gif"/></noscript></span></span> by simulating genome-wide descent in the pedigrees. Then we simulate trait data given <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-74.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-75.gif"/></noscript></span></span> under model (1), and fit model (1) with <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-76.gif"/></noscript></span></span> to obtain the MLEs. Means and standard deviations of the empirical distribution of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-77.gif"/></noscript></span></span> are compared to those computed from (<a id="xref-disp-formula-7-3" class="xref-disp-formula" href="#disp-formula-7">7</a>) and (<a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">8</a>) based on (<span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-78.gif"/></noscript></span></span>).</p><p id="p-15">The second simulation study investigates the potential impact of mis-specifying the genetic correlation structure in a population-based design. For ease of simulation, we use multiple independent pedigrees of second or third cousinships of various sizes. This block-diagonal structure might approximate that of a population sample that included clusters of more closely related individuals. The total sample size is kept at 2000 across different simulation conditions, so that in one of the conditions, for example, the sample consists of 200 independent third cousinships of 10. We consider three different kinship measures, each computed within cousinship. These are twice the genome-wide realized kinship matrix (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-79.gif"/></noscript></span></span>), the classic GRM (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-80.gif"/></noscript></span></span>), and the LD (linkage disequilibrium) weighted GRM (<span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-81.gif"/></noscript></span></span>) introduced in <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> (2017)</a>. Both the classic GRM and the LD weighted GRM are computed from simulated marker data. The LD weighted GRM represents a more accurate estimator of twice the realized kinship matrix than the classic GRM. For the two GRMs, we look at the cases both with and without constraining the diagonal terms (twice the self-kinship) to 1. True parameter values are set to <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-83.gif"/></noscript></span></span>.</p><p id="p-16">Simulations were performed using the R package <em>rres</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Wang 2018</a>). Genome-wide realized kinship between each pair of individuals was measured as the proportion of shared genome (in genetic distances) over the 22 autosomes. To simulate marker genotypes, real haplotypes from the 1000 Genomes Project Phase 3 data (The 1000 Genomes Project Consortium 2015) were assigned to founders and populated down the pedigree given the simulated joint IBD pattern. A total of 169,751 markers were selected from all 22 autosomes based on minor allele frequency, marker spacing, and availability of genetic position. Details of the marker data can be found in <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> (2017)</a>. These SNP markers were used to obtain the classic GRM and the LD weighted GRM using the <em>rres</em> package, with allele frequencies assumed known.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-17">The 1000 Genomes Project Phase 3 data are available at <a href="http://www.1000genomes.org/data">http://www.1000genomes.org/data</a>. Information on the set of 169,751 SNP markers used in the simulation studies can be found in the supplement of <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> (2017)</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-18">In the first simulation study, there is generally very good agreement between the empirical and analytical results. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> displays the results from the 9 combinations of (<span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-84.gif"/></noscript></span></span>), for the sibship design and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-85.gif"/></noscript></span></span>. We see that <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-86.gif"/></noscript></span></span> appears to be unbiased when <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-87.gif"/></noscript></span></span>, and when <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-88.gif"/></noscript></span></span>. In addition, <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-89.gif"/></noscript></span></span> resulted in very similar <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-90.gif"/></noscript></span></span> regardless of <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-91.gif"/></noscript></span></span>, as predicted in the previous section. When <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-92.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-93.gif"/></noscript></span></span> uses a more variable kinship measure than <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-94.gif"/></noscript></span></span> (in this case, realized kinship on a shorter causal genome), both the point estimates and sampling errors tend to be smaller than expected under correct model specification.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493036/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493036/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Comparison of limits and asymptotic sampling standard errors of heritability estimates obtained by fitting model (1) and by using (<a id="xref-disp-formula-7-4" class="xref-disp-formula" href="#disp-formula-7"><strong>7</strong></a>) and (<a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8"><strong>8</strong></a>), with data on 100 independent sibships of 14. True parameters are <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/T1/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/T1/embed/mml-math-95.gif"/></noscript></span></span>. Empirical distribution of <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/T1/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/T1/embed/mml-math-96.gif"/></noscript></span></span> is constructed from 500 simulation replicates</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">When <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-113.gif"/></noscript></span></span>, <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">equation (5)</a> suggests that pedigree structure affects ASV(<span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-114.gif"/></noscript></span></span>) only through eigenvalues of <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-115.gif"/></noscript></span></span>. For large <em>N</em>, the first term in the denominator of (<a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">5</a>),<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-116.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>dominates the second term. Since the summand in (<a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">12</a>) is non-negative, eigenvalues that lead to bigger summand values have a higher tendency to reduce ASV(<span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-117.gif"/></noscript></span></span>). <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> suggests that extreme eigenvalues have biggest impact in reducing sampling variance. <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> shows that twice the realized kinship matrices on shorter genomic segments tend to produce more extreme eigenvalues. This provides intuition on why <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-118.gif"/></noscript></span></span> is much smaller when <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-119.gif"/></noscript></span></span>, as compared to the other cases of correct model specification shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) evaluated at a range of &#x3BB; values and . (B) Cumulative distributions of eigenvalues of  and  from 100 simulated sibships of 14. Vertical dashed lines in both plots represent the two distinct eigenvalues of : 0.5 and 7.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1140530501" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A)&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-120&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; evaluated at a range of &#x3BB; values and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-121&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. (B) Cumulative distributions of eigenvalues of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-122&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-123&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; from 100 simulated sibships of 14. Vertical dashed lines in both plots represent the two distinct eigenvalues of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-124&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: 0.5 and 7.5.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1385/F2.medium.gif" width="440" height="134"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1385/F2.medium.gif" width="440" height="134"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493060" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">(A)<span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-120.gif"/></noscript></span></span> evaluated at a range of λ values and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-121.gif"/></noscript></span></span>. (B) Cumulative distributions of eigenvalues of <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-122.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-123.gif"/></noscript></span></span> from 100 simulated sibships of 14. Vertical dashed lines in both plots represent the two distinct eigenvalues of <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F2/embed/mml-math-124.gif"/></noscript></span></span>: 0.5 and 7.5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows simulation results from use of other designs and heritability values. We omit scenarios where <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-125.gif"/></noscript></span></span> since it cannot be true biologically. Most of the observations from earlier discussion of the sibship design also hold true for other designs. The one clear exception is that when <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-126.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-127.gif"/></noscript></span></span>, the impact of this specific model mis-specification on <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-128.gif"/></noscript></span></span> is very small when using other designs (compare blue circles to black bars in each scenario in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>). This is likely because the distribution of eigenvalues of the correlation matrix depends on the length of the causal genome as well as pedigree structure. For sibship design, a good proportion of eigenvalues associated with <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-129.gif"/></noscript></span></span> are close to 0 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>), which is not the case for other designs (results not shown). As a contrast, when <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-130.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-131.gif"/></noscript></span></span>, the impact of model mis-specification on <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-132.gif"/></noscript></span></span> is clear for all designs (compare blue circles to black bars in each scenario in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3D</a>). This is because the greater variation in <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-133.gif"/></noscript></span></span> due to shorter length of causal genome leads to the possibility of more extreme eigenvalues for each study design.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation results for all four designs, three values of  and various combinations of . (A) and (B) show the average bias of heritability estimates from 500 simulation replicates, when  or  respectively. (C) and (D) show the standard deviation of heritability estimates from 500 simulation replicates, when  or  respectively. In both (C) and (D), results obtained under correct model specification are shown in black bars as references." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1140530501" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation results for all four designs, three values of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-134&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and various combinations of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-135&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. (A) and (B) show the average bias of heritability estimates from 500 simulation replicates, when &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-136&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; or &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-137&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; respectively. (C) and (D) show the standard deviation of heritability estimates from 500 simulation replicates, when &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; or &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-139&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; respectively. In both (C) and (D), results obtained under correct model specification are shown in black bars as references.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1385/F3.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1385/F3.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492906" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Simulation results for all four designs, three values of <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-134.gif"/></noscript></span></span> and various combinations of <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-135.gif"/></noscript></span></span>. (A) and (B) show the average bias of heritability estimates from 500 simulation replicates, when <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-136.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-137.gif"/></noscript></span></span> respectively. (C) and (D) show the standard deviation of heritability estimates from 500 simulation replicates, when <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-138.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F3/embed/mml-math-139.gif"/></noscript></span></span> respectively. In both (C) and (D), results obtained under correct model specification are shown in black bars as references.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Lastly, we note that for any given combination of (<span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-140.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-141.gif"/></noscript></span></span> varies with <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-142.gif"/></noscript></span></span> quite differently across designs. For example, when <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-143.gif"/></noscript></span></span> the sibship design has slightly smaller <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-144.gif"/></noscript></span></span> than the three-generation design at <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-145.gif"/></noscript></span></span>, but it is the opposite when <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-146.gif"/></noscript></span></span>.</p><p id="p-24">In the second simulation study, there is generally more bias in heritability estimates when the sample consists of third cousinships as opposed to second cousinships (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). This is in good agreement with the findings of <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> (2017)</a>, where the authors showed it is relatively more difficult to estimate realized relatedness between more remote relatives. This observed association between level of bias and remoteness of relationship is further verified by additional simulations using full sibships, half sibships and first cousinships (results not shown).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Limits of  when  and . Each sample contains multiple independent second (C2) or third (C3) cousinships of specific sizes. Three kinship measures considered are: twice the genome-wide realized kinship matrix (), the classic GRM () and the LD weighted GRM (). In (A) and (B), ,  respectively. In (C) and (D), ,  respectively. The estimated matrices () are used with and without constraining the diagonal terms to 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1140530501" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Limits of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-147&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; when &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-148&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-149&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Each sample contains multiple independent second (C2) or third (C3) cousinships of specific sizes. Three kinship measures considered are: twice the genome-wide realized kinship matrix (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-150&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), the classic GRM (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-151&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and the LD weighted GRM (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-152&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). In (A) and (B), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-153&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-154&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; respectively. In (C) and (D), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-155&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-156&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; respectively. The estimated matrices (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-157&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) are used with and without constraining the diagonal terms to 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1385/F4.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1385/F4.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1385/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493066" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Limits of <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-147.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-148.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-149.gif"/></noscript></span></span>. Each sample contains multiple independent second (C2) or third (C3) cousinships of specific sizes. Three kinship measures considered are: twice the genome-wide realized kinship matrix (<span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-150.gif"/></noscript></span></span>), the classic GRM (<span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-151.gif"/></noscript></span></span>) and the LD weighted GRM (<span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-152.gif"/></noscript></span></span>). In (A) and (B), <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-153.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-154.gif"/></noscript></span></span> respectively. In (C) and (D), <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-155.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-156.gif"/></noscript></span></span> respectively. The estimated matrices (<span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/F4/embed/mml-math-157.gif"/></noscript></span></span>) are used with and without constraining the diagonal terms to 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Another observation from <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> is that the decision to constrain the diagonal terms of <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-158.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-159.gif"/></noscript></span></span> to 1 makes a big difference when pedigree sizes are small, but that difference erodes when pedigree sizes increase. A possible explanation is that apart from its effect on eigenvalues and eigenvectors in (<a id="xref-disp-formula-7-5" class="xref-disp-formula" href="#disp-formula-7">7</a>), <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-160.gif"/></noscript></span></span> induces bias in heritability estimates most directly from the matrix of difference, Δ. When we constrain the diagonal terms of <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-161.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-162.gif"/></noscript></span></span> to 1, Δ will have 0’s on the diagonal. This is expected to have a bigger impact when pedigree sizes are small and the diagonal terms make up a bigger proportion of non-zero terms of Δ.</p><p id="p-27"><a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">4B</a> shows that when <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-163.gif"/></noscript></span></span>, setting <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-164.gif"/></noscript></span></span> produces less bias than <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-165.gif"/></noscript></span></span> in all cases. This is expected since the LD weighted GRM is a more accurate estimator of twice the genome-wide realized kinship matrix. When constraining the diagonal terms of the estimated matrices to 1, bias increases with pedigree size for both second and third cousinships. Again, this is likely the result of the composition of non-zero terms in Δ: larger pedigree sizes imply higher proportion of non-zero terms. When not constraining the diagonal terms of the estimated matrices to 1, bias first decreases and then increases with pedigree size. A possible explanation is that in smaller cousinships, the diagonal terms of Δ take up a bigger proportion of all non-zero terms. While self-kinship can be estimated relatively accurately compare to realized kinship between remote cousins, the magnitude of deviations on the original scale, as captured in Δ, is greater for self-kinship (<a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> 2017</a>). This means the diagonal terms of Δ could be more influential than off-diagonal terms on a per-entry basis, which lead to the bigger bias observed for smaller pedigrees.</p><p id="p-28"><a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">4D</a> show limits of heritability estimates when <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-166.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-167.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-168.gif"/></noscript></span></span>. In both figures, the main observation is that the scale of bias is very small compare to that in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a> and <a id="xref-fig-4-8" class="xref-fig" href="#F4">4B</a>. This matches our findings from the first simulation study: when <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-169.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-170.gif"/></noscript></span></span> are associated with two kinship measures with zero expected differences (expectation over realization of descent on pedigree), we expect downward bias in heritability estimate if <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-171.gif"/></noscript></span></span> uses a more variable kinship measure than <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-172.gif"/></noscript></span></span>, but not much bias the other way round (<em>e.g.</em>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">3B</a>). <a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> (2017)</a> showed that both the classic GRM and the LD weighted GRM are unbiased estimators of twice the genome-wide realized kinship under certain assumptions, and the unbiasedness property held very well in simulation studies. Since the classic GRM is a less efficient estimator, it is a more variable measure of kinship compare to LD weighted GRM.</p></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-29">For heritability estimation using a two-component random effects model, we have provided formulas for the limits and the asymptotic sampling variances of the MLEs. These formulas are applicable even when the wrong kinship measure is used to capture additive genetic correlation. To study the impact of pedigree structure on asymptotic distribution of the MLEs, we have assumed that the sample consists of multiple independent pedigrees of the same structure. The assumption of a fixed pedigree structure is not required to derive <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">equations (4)</a> through (<a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">11</a>) in <strong>METHODS</strong>, since log-likelihood sums over independent pedigrees of any structure. When different pedigree structures are present in the study sample, one only needs to replace the fixed pedigree size <em>n</em> with <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-173.gif"/></noscript></span></span> for pedigree <em>i</em> in those formulas and the asymptotic results are still valid.</p><p id="p-30">Under correct specification of model (1), ASV of heritability estimates is a function of eigenvalues of the appropriate genetic correlation matrix, which in turn can depend on pedigree structure and variation in the corresponding kinship measure. For any pedigree structure and choice of kinship measure, the resulting kinship matrix can be easily obtained by simulation and subsequently used in the formulas we presented to assess effectiveness of the pedigree in heritability estimation under the assumed model. This is beneficial to pedigree selection in study design.</p><p id="p-31">There can be bias in heritability estimate if the genetic correlation matrix has been mis-specified. The extent of the bias depends on both the true and the fitted correlation matrices, as well as the study design. When the true genetic correlation between individuals is captured by genome-wide realized kinship, one should use an accurate estimator of realized kinship to compute the fitted correlation matrix in order to reduce downward bias in heritability estimate. This is especially important when the study sample contains many remotely related individuals.</p><p id="p-32">The popular classic GRM is not an accurate estimator of twice the realized kinship. When used in a population-based design, it can lead to substantial downward bias in heritability estimate if the truth (<span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-174.gif"/></noscript></span></span>) is that of twice the genome-wide realized kinship. The downward bias is expected to persist even if denser SNP panels are used to estimate realized kinship, as additional markers do not provide information without limit. In a follow up study of <a id="xref-ref-13-9" class="xref-bibr" href="#ref-13">Wang <em>et al.</em> (2017)</a>, we found that increasing marker density by 4 times made little improvement in accuracy of the classic GRM estimator (results not shown). This choice of kinship estimator and study design can contribute to the “missing heritability” problem, among many other factors. On the other hand, using a more accurate kinship estimator to compute the fitted correlation matrix (<span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-175.gif"/></noscript></span></span>) is more robust to mis-specification of kinship measure.</p><p id="p-33">When using a population-based design, it is common to remove an individual in each pair that has a kinship estimate exceeding certain threshold (<em>e.g.</em>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Yang <em>et al.</em> 2010</a>). A threshold of 0.025 roughly corresponds to excluding closer relatives than second cousins (with pedigree kinship 0.0156). While more remote relatives often exist in a population sample, we only presented results on second and third cousinships in the second simulation study because they demonstrate all the important points from this simulation setup. The main reason for using a population-based design is not to eliminate IBD, but to avoid confounding due to shared environmental effects of closer relatives. Since our simulation models do not include shared environmental effects, our findings should generalize to studies that involve more remote relatives than third cousins. Additional results on full sibships, half sibships and first cousinships (not shown) have all confirmed the findings we have discussed. In particular, asymptotic bias in heritability increases with remoteness of relationship when the additive genetic correlation matrix is mis-specified.</p><p id="p-34">Formulas in <strong>METHODS</strong> do not associate <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-176.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-177.gif"/></noscript></span></span> with specific kinship measures. They are applicable to all choices of genetic correlation matrices. The decision to assume realized <em>vs.</em> estimated kinship as the appropriate kinship measure for <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-178.gif"/></noscript></span></span> depends on one’s belief on whether genetic correlation is best captured by IBD sharing or by IBS matching at genotyped markers. Taking the IBS perspective, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Jiang <em>et al.</em> (2016)</a> provided a theoretical justification for consistency of heritability estimates in genome-wide association studies (GWAS) under mis-specified linear mixed model. The authors assumed that <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1385/embed/mml-math-179.gif"/></noscript></span></span> takes the form of the classic GRM, but the fitted model is mis-specified in the sense that only a subset of the GWAS SNPs are actually causal. This setup falls within the more general discussions in this paper. We have shown that fitting a covariance matrix that is broader than truth causes little asymptotic bias (<em>e.g.</em>, <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-35">A model with only two random effects is attractive for ease of computation and interpretability. In particular, an eigen-transformation is possible so that variance covariance matrix of the transformed trait is diagonal, and it depends on the appropriate genetic correlation matrix only through the eigenvalues. This simplification is not possible with more than two random effects. The asymptotic properties of heritability estimates under more complicated random effects models will be a topic of future investigation.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">We thank the two reviewers for their constructive comments. This research was supported in part by National Institutes of Health grants R37 GM-046255.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-37"><em>Communicating editor: R. W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-04"><span class="received-label">Received </span>January 4, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-15"><span class="accepted-label">Accepted </span>March 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Wang, Thompson</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.1" data-doi="10.1017/S0016672310000480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in actual relationship as a consequence of mendelian sampling and linkage.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000480</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D64%26rft.atitle%253DVariation%2Bin%2Bactual%2Brelationship%2Bas%2Ba%2Bconsequence%2Bof%2BMendelian%2Bsampling%2Band%2Blinkage.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000480%26rft_id%253Dinfo%253Apmid%252F21226974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21226974&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.2" data-doi="10.1214/15-AOS1421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">On high-dimensional misspecified mixed model analysis in genome-wide association study.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">2127</span>–<span class="cit-lpage">2160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/15-AOS1421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D44%26rft.spage%253D2127%26rft_id%253Dinfo%253Adoi%252F10.1214%252F15-AOS1421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/15-AOS1421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.3" data-doi="10.1214/aoms/1177729694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kullback</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibler</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">On information and sufficiency.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aoms/1177729694</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D22%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177729694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177729694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.4" data-doi="10.1038/456018a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maher</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Personal genomes: The case of the missing heritability.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/456018a</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMaher%26rft.auinit1%253DB.%26rft.volume%253D456%26rft.issue%253D7218%26rft.spage%253D18%26rft.epage%253D21%26rft.atitle%253DPersonal%2Bgenomes%253A%2BThe%2Bcase%2Bof%2Bthe%2Bmissing%2Bheritability.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F456018a%26rft_id%253Dinfo%253Apmid%252F18987709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/456018a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18987709&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260674000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.5" data-doi="10.1038/nature08494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindorf</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Finding the missing heritability of complex diseases.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">753</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08494</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D747%26rft.epage%253D753%26rft.atitle%253DFinding%2Bthe%2Bmissing%2Bheritability%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08494%26rft_id%253Dinfo%253Apmid%252F19812666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812666&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270547500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.6" data-doi="10.1007/s12561-016-9143-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raffa</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Power and effective study size in heritability studies.</span> <abbr class="cit-jnl-abbrev">Stat. Biosci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12561-016-9143-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BBiosci.%26rft.volume%253D8%26rft.spage%253D264%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12561-016-9143-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12561-016-9143-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1385.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Severini</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Likelihood Methods in Statistics</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.5.1385.8" data-doi="10.1038/nature15393"><div class="cit-metadata"><cite>The 1000 Genomes Project Consortium, <span class="cit-pub-date">2015</span> A global reference for human genetic variation. Nature 526: 68–74.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature15393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D526%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15393%26rft_id%253Dinfo%253Apmid%252F26432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432245&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.9" data-doi="10.1534/genetics.112.148825"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identity by descent: variation in meiosis, across genomes, and in populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.148825</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThompson%26rft.auinit1%253DE.%2BA.%26rft.volume%253D194%26rft.issue%253D2%26rft.spage%253D301%26rft.epage%253D326%26rft.atitle%253DIdentity%2Bby%2BDescent%253A%2BVariation%2Bin%2BMeiosis%252C%2BAcross%2BGenomes%252C%2Band%2Bin%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.148825%26rft_id%253Dinfo%253Apmid%252F23733848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvMzAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.10" data-doi="10.2307/2531445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Pedigree analysis for quantitative traits: variance components without matrix inversion.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">413</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2531445</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DThompson%26rft.auinit1%253DE.%2BA.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D399%26rft.epage%253D413%26rft.atitle%253DPedigree%2Banalysis%2Bfor%2Bquantitative%2Btraits%253A%2Bvariance%2Bcomponents%2Bwithout%2Bmatrix%2Binversion.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2531445%26rft_id%253Dinfo%253Apmid%252F2364130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2531445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2364130&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.11" data-doi="10.1534/genetics.114.171017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A general unified framework to assess the sampling variance of heritability estimates using pedigree or marker-based relationships.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.171017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171017%26rft_id%253Dinfo%253Apmid%252F25361897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzEvMjIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1385.12"><div class="cit-metadata unstructured">Wang, B., 2018 <em>rres</em>: <em>Realized Relatedness Estimation and Simulation</em>. R package version 1.1. <a href="http://https//cran.r-project.org/web/packages/rres">https//cran.r-project.org/web/packages/rres</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.13" data-doi="10.1534/genetics.116.197004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sverdlov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Efficient estimation of realized kinship from single nucleotide polymorphism genotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1063</span>–<span class="cit-lpage">1078</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197004%26rft_id%253Dinfo%253Apmid%252F28100587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS8zLzEwNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1385.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Wikipedia</span></li></ol><cite>, <span class="cit-pub-date">2018</span> Cleopatra pedigree. <a href="http://en.wikipedia.org/wiki/Cleopatra#Ancestry">http://en.wikipedia.org/wiki/Cleopatra#Ancestry</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1385.15" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common snps explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1385.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1377" title="BGData - A Suite of R Packages for Genomic Analysis with Big Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1393" title="The Genetics and Genome-Wide Screening of Regrowth Loci, a Key Component of Perennialism in Zea diploperennis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dol92r8K8gNlU8gK0E1Lv0NEYqe_PaT9tpyGrZdlnb4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1385" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1385" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1385" class="active" data-icon-position="" data-hide-link-title="0">Realized Genome Sharing in Heritability Estimation Using Random Effects Models</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492884" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-MRm_hb9kTpzkUz5aiQZy5NKs7X_gKjrQFNcyrZNfEXw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1385.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1385" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-n57A4PNvueil5oTKFD38wCCUuc0neijczkqQMNHOPyE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1385" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1385.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492884" id="node492884--2" data-pisa="ggg;9/5/1385" data-pisa-master="ggg;g3.119.0005" data-apath="/ggg/9/5/1385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Realized Genome Sharing in Heritability Estimation Using Random Effects Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bowen Wang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0198-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth Thompson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1385-1391;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.0005 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492884--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bowen Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Washington, Seattle, Washington 98195-4322</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Adobe Inc., San Jose, California 95110-2704</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bowen%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABowen%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Washington, Seattle, Washington 98195-4322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0198-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth Thompson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:eathomp@uw.edu" class="" data-icon-position="" data-hide-link-title="0">eathomp@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1385" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1385&amp;title=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1385&amp;title=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1385&amp;title=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1385&amp;text=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1385&amp;title=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1385&amp;t=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1385&amp;title=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1385&amp;title=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492884/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492884" id="node492884--3" data-pisa="ggg;9/5/1385" data-pisa-master="ggg;g3.119.0005" data-apath="/ggg/9/5/1385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Realized Genome Sharing in Heritability Estimation Using Random Effects Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bowen Wang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0198-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth Thompson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1385-1391;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.0005 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492884--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bowen Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Washington, Seattle, Washington 98195-4322</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Adobe Inc., San Jose, California 95110-2704</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bowen%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABowen%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Washington, Seattle, Washington 98195-4322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0198-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth Thompson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:eathomp@uw.edu" class="" data-icon-position="" data-hide-link-title="0">eathomp@uw.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1385&amp;rft.epage=1391&amp;rft.atitle=Realized%20Genome%20Sharing%20in%20Heritability%20Estimation%20Using%20Random%20Effects%20Models&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wang%2C+Bowen&amp;rft.au=Thompson%2C+Elizabeth"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492884/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492884/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492884/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492884/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492884/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492884/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492884/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492884/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492884/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492884/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492884/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492884/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1385" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1385.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1385.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1385.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
