<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1429" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1429.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1429.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1429.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1429/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1429.atom" />
<meta name="HW.pisa" content="ggg;9/5/1429" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp;amp; Perils of Big Data)" />
<meta name="DC.Identifier" content="10.1534/g3.119.400101" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 de los Campos et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The genetic architecture of complex human traits and diseases is affected by large number of possibly interacting genes, but detecting epistatic interactions can be challenging. In the last decade, several studies have alluded to problems that linkage disequilibrium can create when testing for epistatic interactions between DNA markers. However, these problems have not been formalized nor have their consequences been quantified in a precise manner. Here we use a conceptually simple three locus model involving a causal locus and two markers to show that imperfect LD can generate the illusion of epistasis, even when the underlying genetic architecture is purely additive. We describe necessary conditions for such “ phantom epistasis ” to emerge and quantify its relevance using simulations. Our empirical results demonstrate that phantom epistasis can be a very serious problem in GWAS studies (with rejection rates against the additive model greater than 0.28 for nominal p-values of 0.05, even when the model is purely additive). Some studies have sought to avoid this problem by only testing interactions between SNPs with R-sq. &lt;0.1. We show that this threshold is not appropriate and demonstrate that the magnitude of the problem is even greater with large sample size, intermediate allele frequencies, and when the causal locus explains a large amount of phenotypic variance. We conclude that caution must be exercised when interpreting GWAS results derived from very large data sets showing strong evidence in support of epistatic interactions between markers." />
<meta name="DC.Contributor" content="Gustavo de los Campos" />
<meta name="DC.Contributor" content="Daniel Alberto Sorensen" />
<meta name="DC.Contributor" content="Miguel Angel Toro" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp;amp; Perils of Big Data)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The genetic architecture of complex human traits and diseases is affected by large number of possibly interacting genes, but detecting epistatic interactions can be challenging. In the last decade, several studies have alluded to problems that linkage disequilibrium can create when testing for epistatic interactions between DNA markers. However, these problems have not been formalized nor have their consequences been quantified in a precise manner. Here we use a conceptually simple three locus model involving a causal locus and two markers to show that imperfect LD can generate the illusion of epistasis, even when the underlying genetic architecture is purely additive. We describe necessary conditions for such “&lt;i&gt;phantom epistasis&lt;/i&gt;” to emerge and quantify its relevance using simulations. Our empirical results demonstrate that phantom epistasis can be a very serious problem in GWAS studies (with rejection rates against the additive model greater than 0.28 for nominal p-values of 0.05, even when the model is purely additive). Some studies have sought to avoid this problem by only testing interactions between SNPs with R-sq. &amp;lt;0.1. We show that this threshold is not appropriate and demonstrate that the magnitude of the problem is even greater with large sample size, intermediate allele frequencies, and when the causal locus explains a large amount of phenotypic variance. We conclude that caution must be exercised when interpreting GWAS results derived from very large data sets showing strong evidence in support of epistatic interactions between markers.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1429" />
<meta name="citation_id" content="9/5/1429" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1429" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1429.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1429.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1429.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400101" />
<meta name="citation_pmid" content="30877081" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1429" />
<meta name="citation_lastpage" content="1436" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gustavo de los Campos" />
<meta name="citation_author_institution" content="Epidemiology &amp; Biostatistics, Statistics &amp; Probability departments, IQ-Institute for Quantitative Health Science and Engineering, Michigan State University, East Lansing, US" />
<meta name="citation_author_email" content="gustavoc@msu.edu" />
<meta name="citation_author" content="Daniel Alberto Sorensen" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Faculty of Science and Technology, Aarhus University, Aarhus, Denmark" />
<meta name="citation_author" content="Miguel Angel Toro" />
<meta name="citation_author_institution" content="Producción Animal, Universidad Politécnica de Madrid, Madrid, Spain" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=H. Aschard;citation_title=A Perspective on Interaction Effects in Genetic Association Studies.;citation_pages=678-688;citation_volume=40;citation_year=2016;citation_doi=10.1002/gepi.21989" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. G. Bulmer;citation_title=The Effect of Selection on Genetic Variability.;citation_pages=201-211;citation_volume=105;citation_year=1971;citation_doi=10.1086/282718" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=H. J. Cordell;citation_title=Epistasis: what it means, what it doesn't mean, and statistical methods to detect it in humans;citation_pages=2463-2468;citation_volume=11;citation_year=2002;citation_issue=20;citation_pmid=12351582;citation_doi=10.1093/hmg/11.20.2463" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. J. Cordell;citation_title=Detecting gene-gene interactions that underlie human diseases.;citation_pages=392-404;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19434077;citation_doi=10.1038/nrg2579" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic Heritability: What Is It?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of Complex Human Traits Using the Genomic Best Linear Unbiased Predictor.;citation_pages=e1003608;citation_volume=9;citation_year=2013;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=M. A. Toro;citation_author=H. Naya;citation_author=C.-C. Schon;citation_title=Do Molecular Markers Inform About Pleiotropy?;citation_volume=201;citation_year=2015;citation_pmid=26205989;citation_doi=10.1534/genetics.115.179978" />
<meta name="citation_reference" content="Grueneberg, A., and G. de los Campos, 2019 BGData: A Suite of R—Packages for Analysis of Big Genomic Data G3: (Bethesda) 9: @@. 10.1534/g3.119.400018" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. E. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245;citation_volume=136;citation_year=2009;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=D. Haig;citation_title=Does heritability hide in epistasis between linked SNPs?;citation_pages=123-123;citation_volume=19;citation_year=2011;citation_issue=2;citation_pmid=20924408;citation_doi=10.1038/ejhg.2010.161" />
<meta name="citation_reference" content="He, S., J. C. Reif, V. Korzun, R. Bothe, E. Ebmeyer, and Y. Jiang, 2017. “Genome-Wide Mapping and Prediction Suggests Presence of Local Epistasis in a Vast Elite Winter Wheat Populations Adapted to Central Europe.” Theoretical and Applied Genetics 130 (4). Springer Berlin Heidelberg:635–47. 10.1007/s00122-016-2840-x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G. Hemani;citation_author=K. Shakhbazov;citation_author=H. J. Westra;citation_author=T. Esko;citation_author=A. K. Henders;citation_title=Detection and Replication of Epistasis Influencing Transcription in Humans.;citation_pages=249;citation_volume=508;citation_year=2014;citation_pmid=24572353;citation_doi=10.1038/nature13005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_author=W. Huang;citation_author=T. F. C. Mackay;citation_pages=e1006421;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006421" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Huang;citation_author=S. Xu;citation_author=X. Cai;citation_title=Whole-Genome Quantitative Trait Locus Mapping Reveals Major Role of Epistasis on Yield of Rice.;citation_pages=e87330;citation_volume=9;citation_year=2014;citation_pmid=24489897;citation_doi=10.1371/journal.pone.0087330" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. F. C. Mackay;citation_title=Epistasis and Quantitative Traits: Using Model Organisms to Study Gene-Gene Interactions.;citation_pages=22-33;citation_volume=15;citation_year=2014;citation_pmid=24296533;citation_doi=10.1038/nrg3627" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. A. Manolio;citation_author=F. S. Collins;citation_author=N. J. Cox;citation_author=D. B. Goldstein;citation_author=L. A. Hindorff;citation_title=Finding the missing heritability of complex diseases.;citation_pages=747-753;citation_volume=461;citation_year=2009;citation_issue=7265;citation_pmid=19812666;citation_doi=10.1038/nature08494" />
<meta name="citation_reference" content="R Development Core Team, 2012. “R: A Language and Environment for Statistical Computing.” Vienna, Austria. http://www.r-project.org/." />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=T. Strange;citation_author=B. Ask;citation_author=B. Nielsen;citation_title=Genetic Parameters of the Piglet Mortality Traits Stillborn, Weak at Birth, Starvation, Crushing, and Miscellaneous in Crossbred Pigs.;citation_volume=91;citation_year=2013;citation_pmid=23408809;citation_doi=10.2527/jas.2012-5584" />
<meta name="citation_reference" content="citation_journal_title=Human heredity;citation_journal_abbrev=Hum Hered;citation_author=X. Wang;citation_author=R. C. Elston;citation_author=X. Zhu;citation_title=The meaning of interaction.;citation_pages=269-277;citation_volume=70;citation_year=2010;citation_issue=4;citation_pmid=21150212;citation_doi=10.1159/000321967" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=W. H. Wei;citation_author=G. Hemani;citation_author=C. S. Haley;citation_title=Detecting Epistasis in Human Complex Traits.;citation_pages=722-733;citation_volume=15;citation_year=2014;citation_pmid=25200660;citation_doi=10.1038/nrg3747" />
<meta name="citation_reference" content="citation_author=W. Wei;citation_author=A. Gyenesei;citation_author=C. A. Semple;citation_author=C. S. Haley;citation_title=Properties of Local Interactions and Their Potential Value in Complementing Genome-Wide Association Studies.;citation_pages=e71203;citation_volume=8;citation_year=2013;citation_pmid=23940718;citation_doi=10.1371/journal.pone.0071203" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. W. Wood;citation_author=M. A. Tuke;citation_author=M. A. Nalls;citation_author=D. G. Hernandez;citation_author=S. Bandinelli;citation_title=Another Explanation for Apparent Epistasis.;citation_pages=E3-E5;citation_volume=514;citation_year=2014;citation_pmid=25279928;citation_doi=10.1038/nature13691" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp;amp; Perils of Big Data)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="The genetic architecture of complex human traits and diseases is affected by large number of possibly interacting genes, but detecting epistatic interactions can be challenging. In the last decade, several studies have alluded to problems that linkage disequilibrium can create when testing for epistatic interactions between DNA markers. However, these problems have not been formalized nor have their consequences been quantified in a precise manner. Here we use a conceptually simple three locus model involving a causal locus and two markers to show that imperfect LD can generate the illusion of epistasis, even when the underlying genetic architecture is purely additive. We describe necessary conditions for such “ phantom epistasis ” to emerge and quantify its relevance using simulations. Our empirical results demonstrate that phantom epistasis can be a very serious problem in GWAS studies (with rejection rates against the additive model greater than 0.28 for nominal p-values of 0.05, even when the model is purely additive). Some studies have sought to avoid this problem by only testing interactions between SNPs with R-sq. &lt;0.1. We show that this threshold is not appropriate and demonstrate that the magnitude of the problem is even greater with large sample size, intermediate allele frequencies, and when the causal locus explains a large amount of phenotypic variance. We conclude that caution must be exercised when interpreting GWAS results derived from very large data sets showing strong evidence in support of epistatic interactions between markers." />
<meta name="og-title" property="og:title" content="Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp;amp; Perils of Big Data)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1429" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The genetic architecture of complex human traits and diseases is affected by large number of possibly interacting genes, but detecting epistatic interactions can be challenging. In the last decade, several studies have alluded to problems that linkage disequilibrium can create when testing for epistatic interactions between DNA markers. However, these problems have not been formalized nor have their consequences been quantified in a precise manner. Here we use a conceptually simple three locus model involving a causal locus and two markers to show that imperfect LD can generate the illusion of epistasis, even when the underlying genetic architecture is purely additive. We describe necessary conditions for such “ phantom epistasis ” to emerge and quantify its relevance using simulations. Our empirical results demonstrate that phantom epistasis can be a very serious problem in GWAS studies (with rejection rates against the additive model greater than 0.28 for nominal p-values of 0.05, even when the model is purely additive). Some studies have sought to avoid this problem by only testing interactions between SNPs with R-sq. &lt;0.1. We show that this threshold is not appropriate and demonstrate that the magnitude of the problem is even greater with large sample size, intermediate allele frequencies, and when the causal locus explains a large amount of phenotypic variance. We conclude that caution must be exercised when interpreting GWAS results derived from very large data sets showing strong evidence in support of epistatic interactions between markers." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp; Perils of Big Data) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3YZsl7fG09eYIUzJycwgudUjwgLy52sqHsCO5VCDWW0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492855","apath":"\/ggg\/9\/5\/1429.atom","pisa":"ggg;9\/5\/1429","ac":{"\/ggg\/9\/5\/1429.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1429.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1429"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1429"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1429.figures-only":true,"\/content\/9\/5\/1429":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CcaIH1egrJKXI0gu5hRdfeQnF+iuyCGZYMPcGwf95OoQv01H7vsDBf0FKzhWDUT6Zq9PKIne8C5h+P5tgVTB2HRRmjdJPyCubVprPu1E3OOwx0HA+50plYAJWOKilA2kna2MtnvQTX2d8kbfJY3CZ1SGWnTxYox1jrX9Owps7tskQlJD5Vs+d4tetyzz+zHbfMjPbTpuYX9RuxopFCEczW4BLOuqo6cbWCpZMYjbZi0wiE3NfeO3jS+W2lnXFOlTad7fCnwB++KmFki088f7vTWEFZwf9BgZB44Q+6Ldohej6hTIVMuI7x38IzhzpnL+r7GR74mS8eRp3+51ukRacirTPyiJfwv7zcizuLMI7tVJngnTAJhGxodhlMaiGNpmCyOgFiucsy8\\\\\\\/zpzWqPS53n63LmskLZV9nglmcySDe2jLwMUvri1hpkXWkZk2U6\\\\\\\/DleNJZt+NuAg7FN3DVwTpCzzs0+5qA1TMm3tO6E8zWDmJ6gF9ojsR7+Sp3x8gyflVoYJ66qdV3942w7NlYclEXMGfxCdyN7sZS4dMs4gOUNOsYhc4aF2tXYYUGatnodRL5nhuzvpqg4o+I041uYI1ExZCyyVobQA5KJ+e4O8lqttOyg8Unu7yx0TCr+PxU5z3d4jWk2r97Cl1gBasIrgRyhI+4eBFItpblfgh4O1vmW2JZMFBB9KF8PD3B0A0AQ+aY5yhuHDuIz3aewJOWN2TqcAFk5f0F5ybiVb1ToIl4mN2QhIvdlOd4d8wGpA+k0u\\\\\\\/tsFWdiuU+DEa822lvSOUBtM7I1ntogvoKgU+hlC0HnU+52xH98yQoz6wHVPPHHIxqjZjd5v7WOBeswGV2y39M8idKM+hG9srdifWn42XZdYMkvT37D4vm3uQ1v6BBLd0KyGiiWgUprG2aNVjcA\\\\\\\/esNqDkghldJELJhBkb2BtF+POSxRrYZSKY+1kcSxMqR7tQASvlZqeDMtFzO9kLrbUZnQ1utPXtZ+3vmqZ\\\\\\\/wd1Ck5Od3DCzeSg1sGCh6ylj8NjoqXp+pqr9DxTtLusNkMzbyqmfz5OK1XJkZxaw023KTko5GmMD74LKFQl4YHT\\\\\\\/ZZHr+72oeqkQGoGJ0EMWX2\\\\\\\/qJ6WSkGOo8Vi6\\\\\\\/gQIu2kBH2k9Nt1t5LA0RthiE2vXCrmOjr2Qm\\\\\\\/qkhpMflkVZkQIFP2+k7V4gp3j2OJsRARfZGavPNisOAnILeQJRPL3PDMG2bjCk0B4EiVkWf+7CSGDqkN72dSxSjhmtD0\\\\\\\/4PLkOPzGqRMvBjO\\\\\\\/lxnJLMIaZYGO\\\u0022,\\\u0022iv\\\u0022:\\\u0022IQzRsbLkrahh18omq6XKNA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228460f055fb8b5104c906522525e9159296025e8c3e354f40706d190f99d76027\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WFw1kGJ7W3wR38cFujSsv8Blepdw030+pbzhKeMZDDZd6noZlY2qoOnqFZPB9\\\\\\\/Gx7JqQ8wEUzEg3mtoXbNp\\\\\\\/KFD0pqfEwSfxReOY9NgGuv83oHZux7IAQp4Nmdmkp5imG\\\\\\\/rcUEVdj8Pl1djB7gBq5iy9Z3DlvYb2Q53msJJR5H3SyFM1yPNPbEhTED8VixameErLNuRFsM6yO4fkqEIYTewUCeziF6Mg5Yfi9f1lMzhTRhkmQMshr91WfwFxyrd3W8TxGmSbfF5k7SRKq9SPrq\\\\\\\/PkJ9mFBbcjNuoD37M22M+I7avon+fx6NrXmCln9vR3LAEF\\\\\\\/G6o8k6ORQZ3AakHpRslyUXDHSTgeFuH6Owox4swI7oJNZtfMeAJAKDZsjx9BoAqOCYyLhwkFoN\\\\\\\/imZki5ViOb2PsJkvVSyQk\\\\\\\/Z4pHJov5a0UgcSCuQznS1KfXnyOQuOlc9bRG7+B7iZkpySd3UvaiMFXiCb0hVlfpNkDD2qfdYb3gKmQKfJTn4JWCLBHk3Ds3Bd4S\\\\\\\/ATPzwZjsaReP3jEX+ysoGTWn\\\\\\\/Pw9atBhbmA+4v7DhnHAxlhqULpVFvFJLKq8TjrQ4QTInLyYiFI9uK9dKoZv2zTqY2qz6HTU40ytCxEK81dM4vKxoeR1NYFVxVaLJC24hxTgdDKtpnDxjy7UGzIhYZk3LSs4c6G6hxCc+6wdRh9LgyGVxT+0JDftM8kftaXCLLuSOJ4UAjsLFGyzSC9ap\\\\\\\/1cHPhrj\\\\\\\/V0QkwNaPmQkbc4f94V0D6Btuu04cjsLw7b2O1WaWaSn4e3W+EAkL+iqGQ88QZ5XA3JgCRYeCycLUGOsNG4ZYFKjWwIiY9c\\\\\\\/5EZAReAnWjTE3QMP6QhYxb4hM+LLLf9DyLhChwGGTBX1Kiu\\\\\\\/ph1Xy9tXOPqW1XAAGVy5WmQaNKG\\\\\\\/9QEr6r6pxLkJPtkg66lA4\\\\\\\/TwKfHKWmCYXgaDncPt7GawebgcdVVPrF8TY10k32b8jyVo7DNzE1KnWciNb7WM4KedBo4maT7KFtT+Jf7wNZ61Mpyw6Q5T\\\\\\\/Y+a4ctg7Z67xtT4f+ejNnNQX4UMTMnPHknBti92CrxewG5VbJYdClFhDky+NSLJlRIISPXmdnVhwJ0SUCtiq+e5DkmTXrIaM0My6lgjLpBxp4tyGJVbQXGeKwKIqobN5u3gbfS4F49u8yB0e\\\\\\\/QEnB4kZHc1rAVb+Bza73++hm6LVeiGOkHIXGLXXVVSySCwkstCQC3XuNOvaRurI22AnOeVYFKNaGHdyaGQDk8EhvQt+GBru1naUXXOL2mtwAYzb86mqPClMnlYH9lDjmBbjURqNCgql9n7uwK3w1ulc7QAEuEGaib1\\\\\\\/QNVoQlO3VV01iFhFQLcbl67PGRW+J7RcL5Zrk\\\\\\\/5yK4FZ2YjRDCtFyLhrTKcMPCNWswUwWV74KAb6OiItD2kG\\\\\\\/Fj2QhfLL2r53I26QbB3osptUzGGkHHmb+CN9sJf3ysfyBQaSV6TJkPdHLj\\\\\\\/813FD2qkvqij394YLgeDgGfOfCyth3UWm0oaS90aHNCBaKwZwfkN9ZEcY1Fw95PSqp2ZXZvLk6vdPx8uqondwrdaTwGeeWUAkYw2e+Z7Z4icuNazls5HNSUKyliW\\\\\\\/Duz9c+TjLFJZZIjSV40CIlkL6gGE7j8MrLJINgwJ0MqJ4V7LbMe8iK7Y8vPByBJLW+EgzaDlrJTy6fxLHSE5veF2iXscoK1FUz972HTBppnO4mpo0LMq11lGMz+XLipY2zP\\\\\\\/KKH+0CW7++Z8U8AdBUlnAr7OTIVt9vZx5N4ztW4cBabw31+EVZOMk+4Wd3H92btlB2s41j8g+wIu6WRZTHUbTC\\\\\\\/fNtuZOIovuSK7Fzg2SN1RAiUDdxd3TLDuQovDQF\\\\\\\/SpKxZ1d8TakuWOvTDUntOiRrO3VcFJ4bx\\\\\\\/5lAEe0g1oXQF7bW71EwDqjG2lGp9IJb9iDe2bL7yUbTTA0jPnWRyqdFzP0olyVZ2apuDDS2Joft2Eht10DMHATScYuzQYIl7RBPKsooeC6PvAy+ObVZoNWa2Aaa8kDVgojs5v0nOZu4coZNQ0hhWlyMswJjzMo8FfupKJHG\\\\\\\/EvgLuQf8Vq+rYQ0jb67TOgMWkLitKTXHGj\\\\\\\/MG4QzBezlddhEmEUyAH4WDVe9iGYGPLtDwA1tAr93Dc9KQ90\\\\\\\/9pYy14m583PJ42x4SJVqkOieVBG4qdf0EWlmZArj2qqu9zZ2GdhpuM\\\\\\\/DHlTTWEA0LaairJNalXMUTsvygFWqOKttNqLuFbTxnk7aBu0df\\\\\\\/ATNPndyBkVu2LHkbLa1d\\\\\\\/\\\\\\\/FVJEuxNHkpADHCZT+iELIFyNMVjwZ+l6k+ImEAUYHY7dGRMMMvuPPKEj7Ta\\\\\\\/bzINhjL5N1dCFfJVvfTzUX0eWaghLHu1sy1GAykMXE2sITREHpbvrBUO5Sy++ec3i2fovDP7fhpy1TviFVSNCB4\\\\\\\/5KhKOyto\\\\\\\/aBURd0ITePQwMUilZDBPsp0agY9Ln1ylePl+ljAW798naEcT6EjdHVUl2dred\\\\\\\/FvZeTdEJgEFWMLPn+zwLqs2JqOHnm1ntA31ru9eihtdtgRFzKg\\\\\\\/x4hW\\\\\\\/NR8iskZJ60VrhOIHM5wIzsfIRn5Ng8s789\\\\\\\/QbPH1NVdg015fCLwfqLsu7g65GHmSETMJSa8EcwmCYy7546RMPFU0I+dFJo7JiqjxjWgiW7rNplD8B7LjC0bs1bHSekKyMzE2\\\\\\\/Llps1spVEq9lx+0P63iuh6aYUyF9+PNjiBHdWvrZouibajqK51kHWIcoCUr3pFAESmpackYhw8lWKThsIr48UzX4Y7qTp0O9jk\\\\\\\/MCrKSzr9Sxl4GeyNIHUN9tJmCFCKYpy+hrhVBUF+GjWPGzhB2HyWnrXcKqEAgjUkLF7XBOqO4jgAcJKP4jyoQyK9ri0O8Szv1rGowTlUKql16pBz1DxRQahiD+3oTF8MxF6glKu\\\\\\\/s5aaCTmR6ri1kt4VTe3KQLR46L0blseCcepoqQ54BMW8qO1X+7imBSVln+hvtoucTACCVzRhFtUiHf33vktAnEF7HWiI5+6GyRMM2DVTkfhoDkkAy40oUgEiXEny8ERYHK5ha\\\\\\\/LqTG9Y\\\\\\\/BPI6SWaQmLHB7kqxjXNTyqkZWSXWE3rBOLT1OUh5H9kKdfpQAUh06DssUrhtTFH1hxDRG+v4QHErJg1WlUyEkA8XY5qOncwZVeir4RQpRWgLpVeInGqLEyBgE2k1jZBcfAGINkncqzgPN8iKA\\\\\\\/F48dFuMar0GwpamFUz0yjfETvSFGX3po2XJxvrYbOeRYYIg3xQoxl92kZW0mOUt7rIzV\\\\\\\/xRSKM5BEY8Mm3+nZpYM2+yoCyUN\\\\\\\/qQS4iqkt5FC3eYpklP6UpuwqnidC+T4Rfikc7UXHw3Ia7SYK9Rv6u08jT8DwZ7oTHljDIt+93HqZYEGc1SsJRgp3O9CTHS\\\\\\\/h3LzKRK4Vlyb5xjTQZf7d7JmeQnHGHbCREOIWqdGx0ildA4z1cTz3GUF3K7C1IOUXjuZNO9\\\\\\\/ejTAPmGztQfiP\\\\\\\/l8BRapArxFKJL8UKAxnSGqh3pfzBtXu+KPBGEHeL2djRk+ExyVSA77bHlZpyD4KhYIPpwgsHVpftbDcvm7CYEMCrG3WuCX0UDib0V3hNJntutEDLvMvxRyWT31qMG77wlCUiMSQAOkream3UouLiiYw\\\\\\\/7A2i3SPd1fi1W9if1N2aT9bko+cI2N4sPFM7rmu7AkWaJn+5Iv7l5dE\\\\\\\/nwJ+Tvyd5GIqZ4agenREviyvWOVBzoQ1FKMZQhgKzLcfCJp4reJztSdxCWgcjy0aY\\\\\\\/v5nPVU2e3JKuKxlOOx5Mn+jr7AKhkr\\\\\\\/lMQ0zNzllI3xZXrJfeX7MW5e9pPdKBv\\\\\\\/OkfO2pyQz7RmBTZBqhM1PIhL4TjDlR0kY5VWkoJnpGEZrHHN2ztLVp69OtOpH9Nqynp+GHJ2IzwFsIPweUqp4r5e+v5CqScZzvYSISqEgDlcKAwdpGTEcGEhsPjjrDC3JYUisZJS4vp4JkDnXHiPKFD08MZq79qxxNoBDHCwq3vC\\\\\\\/ZcgIzQ69begpogWGi2xIhzZZKjRnGO\\\\\\\/lXavt1FT4j1zJIxqLVoCyau5TjgGuR7qfYB3UVv0dUpT02EnbuU3U6UZnk8GOzTyzHLFbo3mFgluCodMXJXXXnzGT8cyVF1nF2kZ\\\\\\\/EU6Ns5uHnFl0d4ISd8uM7K478s\\\\\\\/g4mCu8Sk\\\\\\\/LlcADX4Ipc+b5bL5Z3RIAs+gt4\\\\\\\/7LfCvsvCM2bqQKkFpES8sq8\\\\\\\/Fpd3DqGc8N17NgJUd43Duo4toIlenNOpvZtxwAMy0cIfYNjqQySwUZriKaTqv0C7DPxRJP1eXoOhNnxcIERlgyBV0KKqWenZbQwnHt8cQEihotDZCGeq97BTHmwJId6mZ\\\\\\\/J3RBTx00NdL28xrE0ivLOdk8JQmaHSpdly07qb9i0KL0iSxUj3Yn0op7p7Vo7Ule4YlgMUS3G3hooDbPpRTp\\\\\\\/9YRYlH4n3dX9LqhwoJtzhrBScSazVx69L8XhpMPcizO0I47Lri090F8PWA9cKB\\\\\\\/1tTUxsTUy22E1kQOVE3+Ygm+nTEeCnrwcfTMOq6GdViVO7mOvRrzmkEmhFlyVvOr5I4JXe6ALE1zv3kOKXRGTDHaSqvPe+Tv\\\\\\\/7RHCNTnQxDbXk49HKX0\\\\\\\/L+eIPx9vSsK8d+vvGdvIMc1my4vgomSWsA+kyWkRomdL1Bt16q8spHu400cfbXBaNsxrRFCxUZaUy8d52O32xURVLHbg+aXPmvmFv\\\\\\\/H\\\\\\\/08btKscKPxjibm6bbRRWwQA5T0ZlacfRmINR9+Hpn10in7Qzs\\\\\\\/QHZTAM8zaMimOv0YcxItB3S4sXwi2ZUyNwf7eoSDmp7Nreqp2ePczObp0Gwt2xNuetl651xW4es4UwF7zFUe6MQ2uW1sAtFiRG2N6u0J5\\\\\\\/zcB5jLg1REPIwNw\\\\\\\/Kt3zkvx+OsK6reZ0pvH2O+ZxegDt\\\\\\\/whiSmUiktQ2xNiPybXVdmjcAgxK+DibOnGdOd8pmf6i5SZxUP3sFGcM7Ohvgup30NjElMsY6G0wH9ykHXeIf5HINiTafO4TI2DY4Cs8wNKsm\\\\\\\/S4CHZ+TBLi4AOBI\\\\\\\/spccByZ7oKr9HYwfwoG+EL07pk85U+pbTLK136K77tdNPlg8VO42IaJebI3+YnwDBO1pOlny5wEpTCqQYN+pWS9T60o+4SdkdmMfXzumvdECsuQcoLtsfmPIY4UDdgtO5cBY+cZfTOQevdeThdDhxwx+4KQOe1ziKljFSajcmXQZQ92obcTT6UDk\\\\\\\/maqeCy83Bh+LrP3F+TPg1efxLR4VXAhdueqIj5RgQ5B5tfn0D2ZWgS42OXuxta6Richwohcm2l1T1Imj8GURHmn1Xft7ID3LXr5E2zujsA6oq8xmk+k9XJrzDgMdkt\\\\\\\/J3oa6u4HVI9Ml\\\\\\\/yY9kwQ1O4qhjNjPwu93KMe30Ovk0vk\\\\\\\/1DzbEjTaF9ldqzJS7PI3g41QEHCrWbBYdp2LHsmf1TXs9uNvSmKRWrhl6ie8ASYEpq4pUuBnyfPy+Q81N2a+wEyNoXnXtriTv8YZLX8SaGA+3lqfkQh8liOuZHwhQ+7dXCFucnVsnqwb+ddKTWXKoWWu6FgCWJOvcVxDtzD3seSErhhUD3hA0\\\\\\\/3AHtwuUvlNz72IM6rHlB+9+S+YvY4eO13wT\\\\\\\/EteMW1JaNY91EP16OslP9k5oyd6aC7yZVMRRPOOcVx9JJlSUI2cBigxm\\\\\\\/QW2zGG41wSvAGNo7CfKU0nwpqo8\\\\\\\/caDbSRKy4wON8btTi4o9Ti2hCrbr1g8TaAEYcrQPV9KyY5FmBV8euyMqKvfgHsB6vnyL5krrS3aJEHdaE9HeVIaYeec9ma9gBaZtCkYGyAURbIuP4tH6rjBWdtt5+mX9kGl6h2c2aztJxr92Von006hHffYV33en6yo6FkYdbUMawDQMQWjFchO6FX\\\\\\\/TP6kjOZ86gHGkl\\\\\\\/2uFwAZnqk\\\\\\\/4C6\\\\\\\/OuAZakd69AHiEaOK4Vs\\\\\\\/lENnUsE6bAI8QbDRwn9gqrzSXRdA6qw0RfjE2O1aBaWiPV67sez4Prx7z3461nICM6j9ATiC1pCUIJpa6qvb53ezS2TLc737r8\\\\\\\/AaYkHVIyE\\\\\\\/AMn5nLJ\\\\\\\/gyOmP99I4UtTtGrYG65DILvRt2Byf6MnI+N\\\\\\\/xyDf+yPpVJnvk4Y7eVAF08hI+JD12gj+l4wIEuJOPembruxmZGsInI65t+iri0iwwaASWBOCQ1k3oMfaaQf\\\\\\\/o\\\\\\\/pvKPu54NfQy0ebnqlu0devcvnINMCLAlf7b6nKDFwOTrB1N+Q4peDDn26WE9JO9SUIe3a15BSZeOPDcu1e2xJ8efBDT\\\\\\\/KquZNowhTxk7Vbs3tvsR6B15hteFn8NuRXx1RgcbhuXIHZPIhIf\\\\\\\/oMbkRSnok1RV5rlbBv34ZVdrlrNth4vtOQmkv4cKVh7cyhUFiiGSCzWnmyBZavT5OZcP\\\\\\\/sqexYRFX9bXLZbPKXnD+yFIO9oUxW6gq1rUyicegNfxaJdxjCLLukvK0NuSC0uhLHF15It+8yxFbGZvtZgwPCGii7Fu6bq4wlZhNXCV1KFmgSiXGXGR\\\\\\\/qEO6SiZUhnwat3rKU9Lk517wHfJsPmw7bjDL0dFdVwBx8PU\\\\\\\/JrPH1dUTqkxnQ4Z1HQZKUCUsTIcc7VyB5wPP7feQwLAkRVg1wMeP5s7Dzqxujz8INShHfZVrT\\\\\\\/H3\\\\\\\/qrFeRjkVPjr8NPwCkD3sSSIYXdPddqRze4Ik18Fkyn+crieFnjnlWCd7kyPjR1bpDlE7V36jO6tRZggnl8o\\\\\\\/1zmOoCjpaj0UC+VhZPJ81vVefWbmt9NsLYe5l9bSg6i2WrAEpTF3n8fs\\\\\\\/CtZm73jZfCGMbv3EUUjjxvyqCjo2ekn1Sw8u8RigDDMhCTiU1Jbuwi4gRnTsd30JwfPqw1UU2wrkSEdMCwQaOMrKtx08uh9p0N0fivxHxyVeLEsrDT9Qt88srT2oCzS5vs2E5CMln5873Pc3ftbBM28O6exneUV7FG+T6hsVg1Pjx6AJUic9BXRPcA0\\\\\\\/KFsu1iebQ9H\\\\\\\/6dTx1CVW735amMYpKT1yLOg7EG0TCiNJ+ln2Bu6pVB6niFsDUkSEE\\\\\\\/n9pY21WgyAPqU9HVFCDbWCAdWHGjmUsRVuxFy1qmGvP7979ZI544yD1bx8Fp0vAXpiZ8X3gVYpGu3oVcfKl3hfZDhvC7kieRXITGhqKTQsoYzrcOWCpAGJf11vgW\\\\\\\/d46yFx24aZjxZpv1kj7E0UA0QgiwPruDUNIZH+7qzycdyJu7xaz6MU0WYz6zHjl2RoUYQ93tV0CBMhVE26qYT0wP3JNQ0APebHQGio87E\\\\\\\/RG2dpTM1xTSsY+K+YFNfhRgXZDfacJ38DjzUazRahNf2iPqTyFH\\\\\\\/B8KzzUGWZSC1pqYJ880E7YVPGyCoXhTzbKeg+SH4Xa4nuLPvm5uNHfUJZcHZ6U61zlqcoWbaJJbIlvH16MCyuKYrWMfkYz7fmmF1n6wtSkUry+YxLymBnwfZyPBPb3+I3qnlyE40Mnx+lFqsq95lfi0kNHS\\\\\\\/R5ynlI0NtHu\\\\\\\/8Twy9Jx3J6C20h7tMP+vhT0SDCTGzWCSgVA089uRCQE0TDzmX64qQ\\\\\\\/OXfxGOM36amaBk\\\\\\\/bvekuYNNwO5\\\\\\\/7IoIo7WtWBdl7\\\\\\\/ZOsuGWfSpTnPGLUEwOJYAU4JSO2z5OzqYn8r\\\\\\\/Rcz5KFH2rLIl3wkXH3tRn\\\\\\\/Yi3\\\\\\\/1QSk6sdmUvb8Ddg0cItZ8eDxiV\\\\\\\/aPXvCDgfPPeT6wacYjxGdxHN\\\\\\\/s58PZSDEkhl\\\\\\\/hNkzO7iycq3M+KBPTBIW2DUChibVZy+tkRT2w\\\\\\\/laKEpoGvu6QkX3\\\\\\\/Xv0EXWQbzSRPzSeyAnxHYTHs2BymEYnOqO24XRchY6T2EFnCiEBR7SKwJUpOq3WUcPu1v+tQrOhATqO3Bh9JI4EXTeS9flNTrPRkDZzY+STxZSRNWBhNlXYg6l9VKIfIL5VDYTfb1Xbv0nWWM7IRKka9Z8bd6HSPx1XeGG4BrOKMprgEVr3fLclozXW\\\\\\\/mA5tiaLA3pPK4EjMJ33S0bIOnMjKYdp5hZk9QAyzwfuk00vz49ukou1IKvYpG3SxqilePbwgykVTlSi27wFNcFTSJaHGyeusCEj9n+R6nvfZEmMS6wzuUKtMpsKRTAAps7OrWgWvIsHr5j86oTfNZQFVspy+cDgAuxtiQUlLmWRqTIVOwhdKCD1M+sXxDEuX8yhgX079AZzfzrS9PyNnDdufuXQ\\\\\\\/66MrjU+qSldEn7nwyDw4XV7lmEzslv8UsMemsAmWyzWXbR7sCHOUD4UqQVIwvvkqp9+uUsDYvyLH+yb5xi4frTcXYnGbldWiYah19cVoOtzzIy\\\\\\\/ubKiwxlbZH+uH5UCc5qNDsPdqSvtBxV1OvAuasdYHTd8kd4eyb1MqTadxvBgzVd\\\\\\\/15wXpX2RzX70gccjfOKNCaxwFAkinwCVYTC0ZZYnBKnwRldQUM6QAb+ApRDsl2wdShFondoPKddgAZQ4YsvM3IDuxtqLgKgzz4xdbul4AgOonDYizarAJBY5U53LJbDxivlLDxPWls+qOokSfK+LsGFHJuAEd5Q33ggKXCBbssaG5KO0ElHIE8r0y0NjLWOA1M0HRsENCme\\\\\\\/iGFz+ghg0P9+NH\\\\\\\/yvfkHmk3BR2ILUAvF\\\\\\\/Bv3y+PggrfPxKSsdQ19NoiVOL\\\\\\\/jAPwWVa+NnvS\\\\\\\/f5CuEJstfoalfzvF43G14uMWp+Ox++2Wccbo\\\\\\\/C8lnpAXYjzg1TOFufAx1FfH+iYq2yH+QVAesJU7Xe0SKOtD3jwGtOtFWFph7b52b5SPnx451h3uCyQ+s0GN\\\\\\\/QKmU2RgF84Bp7P9iTRWfRS2\\\\\\\/2sgxx+aqOaC9PMZn6WsqYCJAGMv9h\\\\\\\/RrfRrlSFoNyGaF3JSJoKWMTlCPZZKFe7rxPgUHkB\\\\\\\/tB5T4PWu2n\\\\\\\/uMTCQ80ILs0xYZLiX2x7521sXBmvGYgLaETiZQROWp4vAGWitkdvqJe8Vzn2jAq2ckxTpGrLJ5nUNyw5smm7KRbLihjbEJ2TaMXhW02pe\\\\\\\/PcaYH1YujridhpJr2p\\\\\\\/cmAGFhBv9wHJFve+b9Z\\\\\\\/FltY6fVlLSPOCmVEhEJLIncQ8z79pzrzi5TiOIyYIFoXW9vQgpDlZZm9KU9oxq7OMfnESon1gKjHtv6Zs0TkJ1Hx2SOi7fh+9KqStVFNKUEi6RcFDcgyhB6AmOR4RqU9rV\\\\\\\/ysTNaT0FWq4\\\\\\\/gwmA\\\\\\\/JGaKsgK1dpSNLZ\\\\\\\/gt6Sxok6a0Brx9SHj1X6D1FIvM0NYS8tYytu0jDf0VL4FbnlzZz7JPmhnoQkhOQQ7R8\\\\\\\/UOi5Qw7q82wx3T8lXSj9T60f8pij9iuJLQCQspGUx4EhlhHt4pf2ppSk+PMBtkbnWRlxBe9FpuzZ1ffj6XSpJeUoiRvus86OUIuncb3ITJaOJRoPGJI2YgmqG8n9mYFErSe+jp5RenZLveu13FMsNwZerIjJ0tnT7OleJ8vyveUxmvLRQLCP22Jmln1MIDV8aLQs0X85sMvEft2RoeYOJyjGYE2EpaatdS07M5MCMDts5VI1i0kzcmlQnu3GnHIQ6aQX7GxaTUYTR+s86DNkRtpE\\\\\\\/je0deUI7Bcme\\\\\\\/1kCFyVIBFybQM8wzerSZBwL8kN9qdVUJwPQqOTz8446l5QpKUSabipf0QoZlJ7i81gJFZq\\\\\\\/f1crPr7xB2VeVsKOgk+auksvnQbB8YqH\\\\\\\/V4bhu0+niY\\\\\\\/xa6O48CyZHJAxfsLlicFIPMr5mq5CbD9L7s2EI1mcfOD4X5co8OtyrEIj8\\\\\\\/FFt6PvS5C3LIQJs+\\\\\\\/B0wlVE4P9bVwx4AW637JoqPO32KG+7gnxEouBhr5muhM9o4qXLk9GPjnKhTAg7JhgvoI58M\\\\\\\/iyKRlVVX5yHS+zYwc4zJAnzqBABEBNQtrxDgoLFkuLa1MQ\\\\\\\/MuLiqcWXESjlzlXZyZzIQNE7MmP3lIP\\\\\\\/VKyqLDs1goycDYMXcN\\\\\\\/F48BUGyld6S+JJeFeog3mIopITOvrxVJjeHr6Ffc+tVzh9zmoYuzF6XdGOX7hFNnim1aRYTfyuG6TarcXQfYwRWyPknS\\\\\\\/grNSefCxLzazVdaPmkBKzlVvZ3\\\\\\\/BDHoDUsAIjXCoqzQshD7TVM37iWFWAGnKahF13nktZWSDKmsnb1K1n7LPI9dxlvNBGwfEfae1rLgL3w4n6\\\\\\\/CzZ2foO5Dgt2ZpAhvQCFcsvTP\\\\\\\/v94YDtFmqUcZSX+6er\\\\\\\/BTDVqsxLGSGUx6aei7MxU6O9MXHP6eaqfD3ZDDPqJpH6ZdlpJbMYwhfvhiVp6eNbnM8NV4X6ZBg9kpukUIJfgT5rVO2F2rJ2KQY61XrWTPoo7n9ezX558SGJuCUgwW8jnnaDUNfPM9Up5J3p86YMOIed0diKB1lo+POBTmIjUI18BsCg7uIvfaFVT4MzQxyBTVxqE9qW02cOEJRlzsoiAPXY9g306bOUHsvctbEIAf\\\\\\\/Tos3k8QmRyPTchQmdvCLnuVPp+ukCZegbKMR6UuovP6hIEWHiaRmYiZibZFxy+PY+0KB17beG7kzJL\\\\\\\/jWtAlcw3jMOb7NctcGAG6VFWI8JxmZhTtci2\\\\\\\/vJ9+HOq+wdFDydlNk45xie4giDMF3TMM2J46rpRNJPu+VYehnEJKEYseLmRx805Ty2RhbMUp9PQ+NNrHdM\\\\\\\/j\\\\\\\/AptYKx2XwyiY105pg1p7tnCMl3BkclWsvTXclSMGCgxu1fJKqi0+XYvwUH5GtjOTEy4lnnKzfvwyTjHyi6w9s2UzIc99QDUrBcKpHy8iXuxyiZPP2NWWkM23B6bXs50h4Ibaa8aH0X7ry4xjr9HIrjtwlS1GxiuJne44BAYv+AJDshVgkYXZEGaeu+HZZPhGsq1FnLvWVrmsspDhVVNH4gBMNYAGad6VVGcxeEP8bQMqKHgnSQw8couZHvVuUSjgpAE+nPWonoXGavS93Aeo2PsM\\\\\\\/gkHCFUJxA2l97rMtwYc42F9eX\\\\\\\/XuzHLZiHvwGTS+zoX175cqI9K6n2XBvhn7lu9yeTfhz1u2qNx\\\\\\\/i3pR7dnwAhFUJ8FF7vO+XbZ8NqvBDX1FU72Bl+SSL+rt1e8dHNJiCYc5LtfhYEOdzIVE\\\\\\\/qbgGtfNyVG\\\\\\\/nc68qBwcwlCru7EB6zW5AjzouHqOcx\\\\\\\/pF2KgNQLC8CtomPF0T96jg\\\\\\\/oHm164lTh38K3AK3I44yvpcN96M3G5LY\\\\\\\/ECaTSNxf9cKemKhE3ID9GbUdAVkq6pcxuetW70wLWxV8jWHN8La0KxbMOnjkWfN9JiVCz\\\\\\\/bjM1ZLMbZBidt5O+5INH82arURtO45CyFBbW7QL+\\\\\\\/902h6NTpYoEgWSMnQ4I4z5awoLI0\\\\\\\/+pfT8ckvnaTIA0dAH3DDnPpQs5dFqmgmeQHQSubvS6JpMVr+z9ND3e8GecYPpsyIij3QBCKJZNXTnGpT4XRHxnrRMyldR+i\\\\\\\/ri2GwT4T3p0VE8GekCuc2qiWsRup0ddPe2Qnh3u+pkAJRWAyZLEpvTe7iCj8asM0OnmWlmPr4jX1xMloyVHSQ0YXzB04ZTbOCqwwaE2y255N8ZfkP1R1LG+u1mbZ1geg5DLt2oAkjUY8whxabDoAMPW7n5+6TMVi17l6x7qew7WDTU9fEoYNyXUq0n8Ws1jYkBCcMHpGGhH7G6VRDIwFFwedNb\\\\\\\/4hUHnw4UmwScBpkXvc53\\\\\\\/SWrFAL2XeTHjeChsES7YkhEnr56i0njznXNvAMk8V0YVw13JDXITRKnt9iBSnRt4YsysWipg1TdC7jces+c5DEhBHAscs75ArFUO+Q7aIKu+KCEi0GecSvoBFPnLFb+PRrp1WoNejlB\\\\\\\/yEyiysyFsU3sj69T5IjeamtR1hot+NF3b\\\\\\\/bJrEAJJRLeqquRctXXSxUngSw2y6lHAdgF2PIaYO\\\\\\\/WxMOnhwCuix3V0Mu0Vv0Zk8qyGp5aqOHg0tSgJH8ceV1KrTHtFgDai7Bnz0O\\\\\\\/KdMyfpYGUFpNwuzAmldRgkcHe6SgN5LkzL06EL4ChVBHPANqrlT2kAgi1tU8RbQUwZxa8xcZz9o89ak77Qpfalrb8MjMeIDrRLMV4JkZ6\\\\\\\/7dcmA3ZzspxF0xMvdXEMDeRNnRP4oAWOoId4ZJfMn8Oyso8YP64OudwzMj+YX12HRFFNTFPYKZRcXoeIDnVZoRAP3qokw6TyGBN2IF1strm9fsIaMdBSmm6+f3OBX\\\\\\\/FOPzJHPZKrrbg\\\\\\\/YKY7eJjiofNqUWv9hNgufzZU+E2t\\\\\\\/k74kS7iN6UQq+O7ZfT6xLNdMCwcQyIjnJCl+IQ09JsDA+fX+Y4q8CvYnqMNR52fO8SfsnyxJH3QZxPHOfMzypR\\\\\\\/G7qdyKDLwDrSEz0l8\\\\\\\/FErvO3Yif5kGHA4uU6XlBGx0lnXnm99QXvwmcN4iEBwgA+vn3JfjjlCJfLzA7mjrjw296UEVW2+qCEsF55o+EotF0AkDTyP+n\\\\\\\/Mb6\\\\\\\/N2hyous+vC86OeqcaPoQv4hHis\\\\\\\/pf7evphUyCj+NIW1T2SfdS7JkTAHKsTW4ZvPeO1oHzp\\\\\\\/2pKz+19qp7f2pUNx2td5SA\\\\\\\/bB\\\\\\\/VvJySrvlWMc4M3wU2VlS1FK1cabRcZ5B85zhcDkz46k57DhDMTyHA6RJ4OUVD+zb3Np9QeeVbt0PME76GKQWuuGLEYTTHZ2tPmRRi7q1POT++IgB0eOcrfsHwzurNpYJc6J3nBlBn\\\\\\\/miUt0NcmJQeturEEOLNiR66wjn9yGCgNWwN4NXtkaDaKHNPMYyz8p5vT9kLqjXCO6EYzujT3DaoSDBINYsOcNw2fP5oe9SqT1q71MKWH7miPk\\\\\\\/A7hhTnzzeUEDqXRdDCGfe\\\\\\\/xaS6z5twmfiBNgOliy+4jZ1pQL1bd4aQ+y85e3Pk93KNK5wz6mn0GF0dEF4g8DX5M6CkGmxEwrUbeZ5Kw09iPNEl\\\\\\\/A+LP6i2T8zGmsGjImSJ7pSOTKMoSb++1NaAjaYPL5Su5kIys1yio6bL8rT8ZPSCqeRIt4sA9wSbl2tjpsL83WxzZNO0DjpKKAwTbyuNf1GHoWR4mMfAR+wJ+j4+QknAoAt\\\\\\\/tz6f49hNjtRyLDMPVMyLTVgsBE4MZMIHSEv7lWtstJUbUK7MN35q8sKJip8WwaXLtF6HWgDuPlLy4CYkqgWkBG5nJTbH6iPT2Zeg3H2OYNgupwBzo9MHni7RFErhMW4PVdwdo7WRTrg1NnjLITgJLD\\\\\\\/dAPgVL5JFhrhcclOQgQfiCvu+tXWG7+lVgG9z0WefgBDGA\\\\\\\/hTkyzBnA\\\\\\\/2Kr6bRo9ZTv5dEaudI6fMeTRtHjvJ8868qgV8cEtc\\\\\\\/SbWBXivZwiSgdnrwhzv+QjoVquSiMl8xkzPjYA44GaVP2Ifkp3KSHV1qhVzIHMzZXT8e8mWUEMIuwTfzrI+oGenhZTrcrQd86AhB7H0dFWicRMrX9xVoBXQmRslEKcbMNYzJJwUaWwwuvMApnF5VjXW4FrlMy5XGtYY4nFRiv8uRCM1HBPUGK2oFTTdrY5EpdXlORNzHU6RwkMzJA1bWFFCAwAVXejVqi0akKooEVgrbYtsupQiAIO3vMeD+hQssoZ9Lux9vCbuZMycN775VM4GQBhQXqWRDxDcVVBNqigS5vet3+AAIF32geFVmoRUAi\\\\\\\/FI4EQ+ZtiFdWfHBbaokoqQFDSsxKe9Tqxk0ZTH9RIQE4RLc5AtbDgwVrM9ecWd9A5TPVswUFYL6WybMySQlMF\\\\\\\/t8x1zlAFR9lv44C6JMmp8OOQ1A4S+BCnDKH5pWtETPvMNhOh0n9BxcAJwB+U9q3s9KEhs7hSSh\\\\\\\/8J7IGqFU1+8CrT6BXRGZksGXVK1d3Nulg3Su\\\\\\\/EaZMRz5vAHqNB8bg5VF0+IzydAyEr7ufa556vEAWVqzKueYbulIKj9BcM9CjS62zVJWszytg6\\\\\\\/AekhrUCIM\\\\\\\/MAet1+GgjvKIt\\\\\\\/5wQJ6zFCuGr\\\\\\\/hNAfT2oAtJ4g6q9Na37071J65X8gpJ7QdFEIciRE5lWLV8l8UFuFFFXuNJxVgrhChPHq\\\\\\\/pe91sq2\\\\\\\/CrulmLWFOkYU1roZ\\\\\\\/Y9+0NBNVfKrjkqRY+Qnpq8UG841fihZgUdVneWvk7hmC4ff6dzGLiF9kTqBqiin9kDQtDcbqvHzsqPEdxC\\\\\\\/ooGd5L1fAjjzOJjTQElnOqnvDPLetR\\\\\\\/DzpooCMDD9Vy3YwKLG+YRlmFQmoi0sK9nESYdhYxvD5h0ZpcCAmmsosrlcPJOOGogCkcM+rX44Kj+hMZPrQbV0q5T4GT3omAHIIIQHWnbQjpO+2tolkNsGPeGQXD2UunCh4npBSE5Y5ndJJ\\\\\\\/5KiNtDKyU6aqtH0xt1KtGp107mt\\\\\\\/5v2XVznW9RFRrffRvKFk2WXatAHiXLPFM5mYdbS7jbls05BL5teazx+qzrwTdxi3HXt1QG25IS6\\\\\\\/GWLUdQkrCFRIkoSHxFFJmbNXr2cRo\\\\\\\/\\\\\\\/Io3ldhp7ISER3++Y97\\\\\\\/+Y4pQ4lznaeaz4MbGXp1SQ40ktMkV\\\\\\\/4zOR5fcBTQUUlLJjBradBwbbVVCYc5lrwGwJ+7uMLcGe5TUEea4N6SmjIcG6DDzDbfudkP0UodjCjj\\\\\\\/kLVAYIHXSVEMUq94aRVPExlcS\\\\\\\/eCczcHkolrosQxtKN2AyEAmOz5\\\\\\\/U7BQI7o3gZqYFfOm8cIEK6rVeRpg2FxDun7fVEEkfH6d3cPnWsujSh7QYhreThTY32LqAmWyeLM6xmAlW3BPjCmbF6rHytXczIlXqT7Uj0u9II8z8GIMEpR0AmiLd42sTJ10uqWLT21KVq\\\\\\\/VlHkh6T0W9juM\\\\\\\/SaqB8Qv16PR7DbpdUjoYP\\\\\\\/FdhaBKkdh2zpdgNPWd0WW+M553NSxbMccL16od5bJDL8voVw3F5e5AdoKKp1zU0FyvLPrA5bK7PpM+HpHtjTIrqsu\\\\\\\/DZkE47KzDaPO6uO+d3JGuN\\\\\\\/ZUh3u7Xw+QOuXwR7XZTnhxRdpkfKw5Xajfr8rBMAuPuICy4vnk8pT86+oc4VLvSrkPtl\\\\\\\/oqH1BBX\\\\\\\/F7eyk14Dj29lb01WFAhTVCAGIVQRoRklyevdXjuz8VXCL1QjkZKoWFLuNlLPb0fe60JbmTOwF3k3KDhrgBf\\\\\\\/IA0pFhxblx4Wtvu4esxfgAjkoMCF9ApOKo4fhz2BPTOTDIn5+b8LJXnZcV\\\\\\\/MQagrQSxrgsshhIJ8pHJGF6\\\\\\\/vBEBLLMTcSwjO+M\\\\\\\/vHKqTzbhuXxWc8j48DoWJTjLuOUFxo2cRXijxy\\\\\\\/rP6txvYYPj6ERs8qRKL3x\\\\\\\/Pfr4XhDYsa4vUjEdX\\\\\\\/hFxK0HMR8pc4Gh\\\\\\\/DtZ28tmxLOaPyfdg86T9cNt8JjVTK9GpIBLYbcQlyDjJv\\\\\\\/gUq0JnTpcbLlu4MQ4cA5Bx95cjhijjuDcx2aPuPmWTpFRLQRkx3brIyghoUesgehWrTCf9IEGuDM98ddfPZwmN4GdIC9VzCSKhzjMfgRLopb9zlrVSCs2kEczTK8ixit82ujFDy\\\\\\\/A6eG2IpA5fnLhvtz7VaSNK1EZGAQSb5N1O7fPsRim5GElpEz\\\\\\\/Bymb0qob5WQKAFQdgskrrMAjnMHEp\\\\\\\/\\\\\\\/W35IvDq0SZ0beyph+LnyfmHEuFeVKu\\\\\\\/KpacZW8ofYDbh7HG7QOKn+FoDScJ7SRTFFL0qc0NTzav0+FT8pX3g6uy5xh2tgPFrW9ACEfoDj7vG+lGoy4ZTsRrtO5MqfCfbYDXEdtc2PPfNVyREUedWlqTF9JY3QpLesKEt6Fi0CTYEbLod4+SP1HFL2M46z9ie6zolKPeYWuey7Bz\\\\\\\/ljnP6dZ2DgZW3aX8JXP1A8F7J+159zBey8PH\\\\\\\/SWg9P7nr\\\\\\\/vRyt1AhosfPNxwCYnW2IzAucAZTz51gxI2JesIB9kcgJXu0l\\\\\\\/NRxBwPwXbqFJm6OC+7CQQBSGaUw7h2FJpULD4BoyF\\\\\\\/4hWONu2ug7k9ejj2cgj5xO5ERalAVUh\\\\\\\/nF3ifyDvO+MJdQHdhTJpiCOX6Sd9nmJVojaQYCQMqI16BYv1DvJJfxm+m9eFPJk8u\\\\\\\/RBUok2umv8ePSa81HiK5YvbHDjMWQLVTWpinQdcrb2JEe4gSqul9E+1QH94lJhx6OyI9LcEKq1lm03MyqagGbswXcCyi4Vl\\\\\\\/oygvRppNCUV\\\\\\\/tx7bHHTEkx0oQ18EVSug1KDGDjLDsuiGDcWwMYrGTqnCd02P1v\\\\\\\/yV9zkzKk58arr5T1S2Am8Sa8SIUSlz3jTaMvXFj2qTg6iQGyeZz+hWo4bsZiwW01rdNte+ljqEyagvuNRPinttxkJIYPdhMuA+0xr7eNqUmmnlC01p5pVJ48EVVMUuT0nR0JPSAEx42WG04PBx+tB6szF2JPDsZ+YbEJqErpZWRsXOJaU8FabuGxP0JmVtMfs\\\\\\\/nGe3LAsQC98F78BJ32TMXr0eQCC7QI15uYDpJ1MatvO3TwzXFMVZr5S9rV0cT5vpTpZKIjFvk1UaQdtgYJ5z\\\\\\\/6MKhMw+7ev1YH5RlgTJsNdbUbUdImtBCzE90EQnkc4f392QtOj3\\\\\\\/p0KxneKEtLG5CvbcKC1Ov9uG5blhM6WSjpzYYb8JeNNd\\\\\\\/mXv2PNkqqm5NVfsP7NuXpIdQpO2zQzqCziiMiUMYyjo83l9BHuCGY0+3TgW+r1Y4fdrcTjhGk57b\\\\\\\/\\\\\\\/gepMvSNzsAzesLcbhNqZt7x7iiwcbhL0hkKh+EM53cHWA8mSBP7bywwAjV\\\\\\\/ZC5o4GpjMXPmCmCbsrokeM4P239V0ORWgwX0n6SBWnA2X0fJr5Scr1V5FvAjU+kcqfie4DmGwgS60MmCpGVjDaIF1I5o1x7mjxubPlI3r9UgTfMxYAZdKkNrr5q1MT4xN8X2cVnBuPaP+shvi\\\\\\\/aqr0oP8AUNjPxJD7EJ12dJlVdo3WWdXnSElC8owXGPQcQqOU7uUZyODwJhkLXJX99yWueUU0CQvwgjXYzcIMMCqN0NJBOemG41GHFZcTl\\\\\\\/hClnT5UM44zq5eDpwlI0Otk9dC6oW+qFIxohFq5OzYN1R2c3wsJUw0PbYP+UNIVKRHPO1KvRVh4WvGTsJmuHL4O5qPyIEH7dBVXHOhBCQW\\\\\\\/OIEGjBxW4A2FnQD6fMD++ylM+SAZ3A56NL7ErwWO8Lbg31fwIMXkJhMQZFtLdQvlcfvWLqxyF6ZNyDIeOvBcCD75gZi+ogz+LXjREGjARaM3X7cADN7i+SK4dq6\\\\\\\/Bp3NlKGlMuTXXWuirBuJ9VvItcg5EDU+YKZzDsHKTXk7EwfDM66Zvco3kYBfdsYMJ3iJX8YfSbWFVzjK4XU23BUg9TeDJauN4x8QBGypQAXJM7CG8wgqDFY2HSheK3mDVbHyBXTJ723CsSf7qZ\\\\\\\/24VX3Lv2yIpIJSt777gnKBLGM5dlVZ6c7FWUTrpNN9ivQnLA\\\\\\\/J+m+jKW0cuDNQLIByJj0ZHb9HsD6PFDVkphAshzCrRYkz9iugUQ6dgCW0\\\\\\\/BBaSnEDt9\\\\\\\/\\\\\\\/bWd4a2EHMgGzXpsn\\\\\\\/CZ\\\\\\\/R1tmxoPHtx86CmMIGshRNmCabbITrIxZRtRS2OhtbXXebnqyih8CrupL9MlAYzGAO0yqcR8ubqccAJK5+0\\\\\\\/m0ECtCfOdeB5D5grzj78B0gBNxrNCaMf7mfHQEdevyA5H8BiT6Axh640GEnZ3l\\\\\\\/C9R6di2+N8fhdeu3u2tWZ0EQgfKr0yKmlr+Wz6T3DyUyuOunocmUtox73wNe5Ep9M6TI+mlbT+6\\\\\\\/Jsnd52vRvUyW2pXFUXi5848ZmW5XWYbC3hXwmme5BqtYlv2f\\\\\\\/qLSKFH3unhe5g\\\\\\\/STuLNf3JBZv3oWVR+Jj9QTGUToOqckKC9r18UEn56Tjpsd+BHibhAJEnaSQmk6Kb2LMYk7EE8v5YuFLyDl4gkh26MxTKP\\\\\\\/cFp8p5LGhMv50CShLfZLdj3lIgnvfEm4orH5Z5jUyqoOehK8jp4\\\\\\\/nni3nh1X4rVU4G+Xafp1z2cMyT\\\\\\\/cWn\\\\\\\/mXQV2ElQ24sVCeeb2A9h+rXWPWf7+2QNRdtSe8g5hLZ6oyIYQlvlN4M\\\\\\\/NpbYzivWNBarunq6Hvjux5uCsnJopfDdQ8I4yLr9TYnVM\\\\\\\/066GYB5DjchT8gQ3i\\\\\\\/7nRZ4DMbQaEAba\\\\\\\/hfa8TO3MF40RLI5\\\\\\\/ldz86JmPrvZQrpEyZZJfcXidjDaANKguzfNVqNzX+chvaZrJK2X2HGca1sB8ixn+\\\\\\\/X0tGK+Cj9Y1EDSgN2qzqm1eDnkXwOea263xEZklZiBP2K6G\\\\\\\/0qIDV8EiWGFUQc0MqduSc6TVtHxgPgXVnMAQZH2nyWq+NuLJ+K2AeHP86D+DSP9kFeQt8iZj0Gbb684dquQCMHvqDM4WuRMsuBw6Qb1WeFkc3R7C3D6pGHxcAHvO4ZoqGHSGDGq985AUJNB6s0sf36F41C9cl2CcEW+FKl9yn2VxAQyJFyPyS24jNPi5a20xHsB1fZvwFK2XZGa5Ld\\\\\\\/nFrUTJmSVbOFWGW2bjebs8UNPg4jXeOKUdiGsvJLm72BU82cQZGR2jgBJB1cX5nCU7RS\\\\\\\/DMD6A2m7um+Q7\\\\\\\/K2W3Q6C2feWG6wuSJw5nh6K2K41acT\\\\\\\/D8WLIMOMmnBy6yTdBSYGVmVMlgVbqhSDF+v6eXbaYr\\\\\\\/CHGeWUs\\\\\\\/TfVi8qZvcPu8WtqctAIHwdKviI06C+B3fTG7ClWnuXmaZ0OdWLT5ygwbqME7uKsNjn9XNbgj2ltPYXTrV\\\\\\\/5J0omxM\\\\\\\/kGIBAJRJ6V856v3Jd72jW3dNgvD9J2wcjx9xckyJDr0A7\\\\\\\/r6J4NBsjEyQe6LtoM\\\\\\\/pvuldHHGOFqNksS2uiO4sfPtyZRn1Sf2lA+TnapH7pjfq\\\\\\\/gTXTyKH1nnuCr175YunnU4LHMljMPiFrl3emuPgnefwPZjm7OWrf8iyGDwZ63AvW4oVibiO1bETSsz0lq68SkVDFxFyB\\\\\\\/nzp9CqvCQMLlwuh2RQBWCQtwTJ3Knx558nq3y9C1cXjX2NRCQmNlYNrY3GBEPV\\\\\\\/JFKZ21TDnOR7y3vbtgBbRptMPvOr4ljHx\\\\\\\/s8TVyuyBA4StlDXTLR0rgprE14X7FKaeK9uq4IS45I0U2o7ECrMDfWPTo6fbrO1ggci+AGIFguI56Plob6JSashX1Gm2e\\\\\\\/wRox6rKTAzKGQcunelGiAb\\\\\\\/z8Jax3g1v3E0hN4utrzc90P73uzgwIvVvsgiKo5G2kyNCp3lZa\\\\\\\/xfrENrZU1YcqLU1U9nJVmpEmuwyw\\\\\\\/oSbTDJLYAAZkuX0ATMEoIFieayg23NhsCsiGtWmRmsBbbxmSI3jsVBk8fJLoVufjNom3Pw4ebm4LjKKZK49VRkpxxyFhRNfFxIkEl21FTPyeLNkOL3Kqaies4HxAw+eDdRT0Zkgsfvd+tPvx72eFL7dCVLkyiDepPTGBI935PZVB71iTRlewTNBjZCmb20Yeu1chT2PlGocZAtkKYQaeKVjbpMTAfUCKEXseSgEldeXPSetztKXWUWFzjrsmRxTl8VwmjPuP68WteQ5NxCcW82dHr8Z+DPxyee5dDSEUSoZN1lO4AzuG+JxVa4JTtX5Z6z+BscWeI4iM1lB5lwuB+T0j11e1zcVpvJQe0qtTelL8ljUj37ybNjYBg+hnV7D6SGTHytqNKDNf32q3CVBcDBcd2oLAzm6yHjWvDcm+H80bYPIw3jc2bHEwM7CyAfnDoM9A8MZ+Ua4I0rHa+e+POJxodUbq8aby4s2HW07kwkr8nHRtT939ErV0FjT5tCGe4kdwA04TPY63DWZKArf49xQiCiZz7NUuZPIiwlIDtSC46twIu5mGe6ZJwm2ISon8bBcWVpzNaSg5l\\\\\\\/SBWKKMdvI\\\\\\\/JpBjrYKJdtXwzX6oUxsFuLviEQ\\\\\\\/mJseTJt5krd+g7nxDq7KXUCmZrbo4puEUZLXnQGBML71GiWYhufM1Bp+Aagralv9MhyEqXcpQVz5mlDij3i8onSs15846y+oHoGbN70kDFEvqOUzY7tQCFoPgZ4GOnaCwCj2MFvROxI3kkJmFyd5BP9vlWhfWkJfC8Qk9\\\\\\\/xRYBciYInjjjpX\\\\\\\/7hdAOW0h6GtXnocXIXhqQos+1nt2AIAxyDMpPNx\\\\\\\/RADtbMdOyRj1kblm624uYc4vEy+1kazULdbwvMtwUHaF5+fT1lsXcPZpNtfUyvrDFRAi5eHuIRWpAd8aoyXmEwG4K5nTDWWemloWNm5fIyrOVqbq7L3m564JF8NtKDfq5\\\\\\\/P2XkqWtyW0nomwJdml8uYaMYNZI1jCoCSMsKc2iQfWAbU9v9gsDMuQOEvEcbSlm00muvIRNMb\\\\\\\/CDOp8LdH4KSUmnCvjLDjEwkdPIUazDI4wrKfOIPhRh698yQbcz+Hcxcc\\\\\\\/UVSbwnlJ28QPSTUnw+Xt78slsLJpIMd6LIUbfvGeAIe0rDj2BkjgL3tHV3VJJqKt1W4tCbAhfgdBGWF9tW1W4vmltcPnRwuGuZUe9nTEwn8oRLG\\\\\\\/xzvW\\\\\\\/RvkLQUUMCQATZIR7Y4dPlu3\\\\\\\/+CdUEeR83xX8lv3DWSiBsLanHmNSYDUYDrsTQGXScCAV3e3+fnTFDT\\\\\\\/PNn+bVVsHtoWsaIu3MaesMEZeQJpv2Fc8\\\\\\\/7L+i9AlaMzmmx+Ddn6oF2TEIrIaOi9esSB\\\\\\\/dM7ofe\\\\\\\/QeocF+n+bZZLFQgavbxjPwjyDD30ENJsiOH3yU1iwtOLdIoAIr3d4v56RYF7dGp+m0dTgDQiGJRSG4GNiSEB786LqYdoaR6l\\\\\\\/NOqzmZQN8R4QHk4M6um2fm\\\\\\\/OOAS0VE27KI4alhqaJ8KHow27SgSgfRzJi4ka0lhuVRgywYyEtNv9FpQOOQjF3nXO3jA901E74qW6ODmzhlEzGBmVsSaZnRG59RdlzBAcxXrCQzji84wpDuJaAgl8rHolCgAUnWXqifnQSh\\\\\\\/sHFcdoW5hyj1GaHMb0Ry4X9dgdBTKuyoyPt\\\\\\\/3EzFJFvUhzzMY2MfWeemP1ESu9UKXmaUi90zdYe6Rt4XNXg5dUTXNtz615GQ+CwaWikWZsfzppIEulixR1hTyI1yul4Rz+ga8j7bKSFV+9FKXmHh0dg1BY5EYLquavj0vOzTJn73L4j37Jj+AcO2PA+aR5Hjmob+CusFOYXSsmlQXNhqdBOC\\\\\\\/uwooq\\\\\\\/D3qrWkxbd3yWTMHI58yUXI3hSqJw6ZbTw7sBvMxaTfa5lCYBOxm8ptGobw\\\\\\\/BGaCtSzgPgs60GP0SXCMwlCg18mZ5yy2yvT\\\\\\\/B6JzBgIZg8TQdv96XTmt8LQ+wK5Q\\\\\\\/d\\\\\\\/AmS+v9WDCn0gD0rWT\\\\\\\/fPrqK7ktbfXesj21eUHJNhio4Ood5SWSoGRWFaWq9a1TyOdE9fpVfxgmIFKqe1G7DnPVAqpbEtnDc4yjnKOSlKrhirCqfH5voIO\\\\\\\/jSpZajKgnRurU029YO6OSGodd\\\\\\\/DXnOW4lA85Z7i\\\\\\\/3NxX3\\\\\\\/WRcPn1FZiyoeELgk7Tmo4TfttQjf76KjBuGIjtxtOqyym87P+\\\\\\\/Q6Q2JT79jJ83KzPC2KfSucqslDOdn3MmJgU1oab5tyL0k2TYRRU9Cc2zA4z1mM5jGJOWDgpo\\\\\\\/2C02U9te5\\\\\\\/ZcGKzSZYX3vTZFdIl6GYYd2w5jLL8aW5aPKpZficlNQA+GcWVSdv8KKPpzdD9GlEU8IEBwkY+is5JlikIgp\\\\\\\/OVoUiXMftZwvOXZTC02aWk3IUhWRdct+\\\\\\\/dssVHCvL\\\\\\\/sDdVK\\\\\\\/6OVeH53MxUE3HRT\\\\\\\/EdnbsepynjVr8Hvs5oCJhwLK2WX1em+r2BGBMNyeD8SiZLKzNiaZe1\\\\\\\/zZ6lx0sI0PVIiKbs9UAVfX8TT20UTYu5+UBVtDnmf38tiVwZ\\\\\\\/L0bTRvjrEykQwCmr+hOeiNMafIeBv6V\\\\\\\/9\\\\\\\/WP5kUo42V5oYhsT+Klq8Qa2XUYVvZS1GDHcKfpRsCJtIFCeWEDALU3sX\\\\\\\/UqHxYBSbwiwyz1HANE1c4KD\\\\\\\/5b55U65uFvAvOGK8VP3N7H\\\\\\\/NfKjAai\\\\\\\/ejiwcO6zn44hb7Qhmm4n5k7ykMwaQa6gcXICQRonsAlHg9n9jBdyOPDUvNEiXdaVLiLwGICcvwbf7eh8CB0xw\\\\\\\/CZVAEoul14G2Iemw6sQovuJQemJlIzZhtmkFuzw\\\\\\\/lI7wsMKVtRvfZkEIoSrqSBzYW34rqF3nrer5WZw2EbJ4V+adEM1dciEqe+jT0rfkpFzsnZPHXoJe+ckMDAKKiVnVr19XDnowPw9s8HLGkpL8YaBsa9nNJSLGyJYyxjQWGq4YwFcuxjkarMKpWx8xiBfzI3vUffYXFgrym7AmQ7484+zi5JykNwV2FtjK3Qg4wfQe8QQ1kUx9wI5qXO6YYcWHVa30GiHJ8FdYsMngyeXh+e0LEclGQvILc+AqwlwezdWHYodw6GHozCewIoWoGnTNWc8\\\\\\\/c1Dg4KrLqx3oWDeNzgtBdIDN\\\\\\\/l70I9JlAnuCuOiYCHqzwA8aHDCB59HUds9e0punP7Tm3Iv3glotBnSvRJ4kwbeggnFQpusmc+bxkGGS8PGITXWzYKW9jZROTkYjRp4dtI4fffT3b1OK521Cl1mGk6kGhuck4sjMlPP9MDxiGQauP03N7XuzZuqIv1z1caCVIIT4h+8vhjAl2C5\\\\\\\/YvyGWyeNOl3KmUJj2pAOVzWd2CWKnyTVKnAgdyEImi8ENEIlK0wwOJQ64OcECzEAd1uYBDpqLtSp4yGmg0kehmrUcDxKgMN6vQdul2m9leZUQ49b5qVK3ShY+x+QAuTh3dK7RoRURVOKMe68JVjkYwQ5xcA7RgyN5XTNpfD\\\\\\\/Z6nuf697NuBDE8ZOEATtX4Ts92aT438KaaQ6ZgkmbutJO2SrfZOSH4H3iGXyNRWbJct0L3kbkNZ8IHz+e8+ifWZnp7webSr+ZlZ9OAajnlfprGIVpuatFhDXHcaLcwwNqAOGp7tuPCz4fFKlWIcF6nQ0Dw1mQDwJ2m+IqnUBlq0hPtQ5JY5f5x8FVI1zRiJrtMUxOYMgFrBiMyydd0SfCtYyFCW1eJwKqpLCuekkznJUITPx29EWHOhVnTbLe9VKzrMQGcMmhQkvqbj3lZ5nk5l\\\\\\\/6Iwu3ZEiFpv0\\\\\\\/a5BXMwqwU\\\\\\\/wTpKoaR7vfDdEnrxVGZ90YT3p1RMjuCpVogxuUnZt+wICh\\\\\\\/o9wrx7BQ6RB1tuRfSg+0MLG12GNPg7NUlLg+0xj0+i5S+jkSdGM0mo608ikfjimoJC\\\\\\\/E+gqxOTkUfAwkElfY+y+bKbpKttUScUUXMlcCrsDaXonDmhByr5TmK+ez1bs0cO6lVehFVH4tiU7AIOCZE1slUL2CPRYiPqO7X3ifRbj\\\\\\\/b\\\\\\\/0QmDGagaEhioX195ZJRFD+3+plmsc\\\\\\\/gk+UaqGNEeGJlfnBJw+KXKUBHXs5mJuVeHiRXa08MPWRwtchvmNrfDhVm5udnPZ4Gb8hU9CTIYSAS\\\\\\\/mOjypU9Uo0j7x1xV9xFBqQEaI2P6HaNNMOKhqJvylvfoji1np7gJP45Ep9ANjQ0fzHdFN13uhm9Y+UOjfFAkhaYhGFxC9g13w2BQL+XQzYAd8laEpCkX8PndM4PeLBXS8xL9k3mxJqtV+Ey+uTxP0VJf7kMBlmkvx+nBxqon9JR15Idr99cjZxvgAqn9PfwconG3P9I+PrlDqqBJTSpkan8qqKUfw9qVjwrsNsFzHdhuHnAcVHYQM0bJ46KnPcpwo8pOz3L4ws0Aatv+NU1q4oWk5IbLrkoDwb3mxXgSsrqp9n8WaPN26CzxRNj6jnWyNJrIah4zZ25WpeOfUO5pq4bpVqxy3QjD0FQTZTP6z10iWtUhMik4nAlWtfLJt8LL\\\\\\\/sjCxOdjSJY8jvyVsQ8eVsHsdM43eiqt2svpYNuAdqONxgVDsV5tKe2f61gQ\\\\\\\/kYAeAWe\\\\\\\/eHo1pPfj3GNNHtTnm4rfJhSOV2+HS6q2ojKTPXOZ3FDkzyBegUdFhEQMDVROrxlrGOhXXW1Cq4+60RK+MTnY4qEO5CKIycFF\\\\\\\/YEqjqvHbdBqHWXGLuxL3XdoeFuLIkVnhgp6Y9HAPP8CxuTZE05mSxQrJfSl7pKh1\\\\\\\/Ur5ipkeQd8b8hamipos4pRqwcThe0vzgRKPES66uJbpUCyV+t8rWn3uTyO1eYfGQfz9UaHiES+78apVNT8o48fVWBTrknDrytIDRCvJofHoATDsPCTHTWnQnny9QtjIfU6okyaozYxuvDMpfgwVDR6aLQJWOuxN1r9lYZK7+IE1WfWb2nqAok0EL2i8uMKKXywAI6xDJtM8UJxS0na+ZQw2ctHpCtxgOA1bk40fQQoKL4wno0WtfoBRAEw7zLkSGQZ2wv\\\\\\\/6Vu70p9JPGE+Zns7cUCEF1zS\\\\\\\/MdwjWLSsHhLQmV38jSXu7DPJeup\\\\\\\/TiAZjS7f7WoonLkp96NnlXvE+1LP\\\\\\\/DGRvJeaHnTNn1mhM53AmCR3Ll8uVo2zhFmFGhnhOTGKIRy2VEarG9je7cuUjpUAlzD\\\\\\\/elObFQGuaWJBg9ShRgyJ20CshQcDcgGcDHvjDpoUf0YZKVedD1NvlWDeaO6vF+Y+5d\\\\\\\/gIOuSO\\\\\\\/+ZcXyNs+vKUTz78XcfitQwZaG+zYvg9gTt5HxMlIJlf2WCLHj6YQWoQ7CTOCEe79jlzHOXLX6tCyZ11wL+P+rbPilZ+edKx7VsM9AXmFjnC22H2k\\\\\\\/FEtRrxU11Dj91aDEGmeyW9g0HAIkZexFRjGknJdr0DFHSWQ+Ninpdf5F1lajShh22pF3LUQxYbxH2uN836LG7HMeBGrCVH+FODgr3HHPa\\\\\\\/UMX9G6dOz+P5Bgp8mG\\\\\\\/CT6UJroX8GewqcTOqNx+w+O0BdJvGeONTnMy7Ght4rCJTOqpoWM+CQ0RAch5Z6QnH5RXoBEOJpXB84kiFuc6qT2NiBvcPlhXSKLjbGl8zUuNn65rlsPbNZjPW5hqc7u5Wnj5D627m6PeiR4tCQE6aoWqf3BBCBhyksSZoBkM8PjG08DhZWMmbO0u5ZocyjTLAlKFAkzjIfbCKx\\\\\\\/u87MLCm7AwjR2fP3F3XZd7LpH00akINectglsiGTDUYB3skN2J9ILPKK\\\\\\\/0pL9hNZnPDvaeNFALBDShKt5QD0UnEoYPlx8Ae0UR3MABrnvMjGXHYirbWJhT+FcbNX41sjDnR4OiAHeog7Gghc3CV2nHR5WXuaoQEFMQVaC8xnktk+ZmrTQMsbXXB0RigaToNAVY2TqkBcEfaWBGJxhWY+YHBNQpbNCUCI5HQX7Ga1845NWkf3jT7d5Um5IZhuTDzqmGp\\\\\\\/QyThY1FGrwV94JxoLsT1UusPd9h4R58Im+hoMD\\\\\\\/ojal3P9JOtgAjt1bJvw9vG14N0lh2nxMnvlWjKgQI93MlvcH3cWq7jgGdAfPoH63K6cO4tI509YXBT3xD075civ6s696fqGkjHrdkoSVQ5km0bzx2dMvexxpDytagNn2Qv2wd4gFRasqvIQQ\\\\\\\/FWCoNlzF\\\\\\\/oTRwnSNuX6GV0HrDpmb6hSP7BwqSnxxQOOvHlBDSvpyeDm\\\\\\\/zej4pNsPTaWsRg2ATUQxKPQa2lSrKPbJOn4\\\\\\\/CPYFFJmtYkIcR0r6m8oWzO\\\\\\\/UpnDAyj8qWaBHiUp4YkP74juSNj0\\\\\\\/9RiOpz7gxEtMlz0phIpQdQ2a0TLmV5fD91y+rTxxLw1Mctr4bso\\\\\\\/Q+bsv6vmPbt6pDngPvkqBFMKT1HTtMwvXFuhTr6XARfaWeny1fZJgO8ZVq5kXlJPNnrpmvdNW7RaxloiHuqpjTgUTBSOsBnGe+3BqHMouQ2L0gRDpEJf7Uq5DWubt4+90MlKuI6hq+spSDSI+C7KGxGyb49DBwSRG5L6HHAm5kgJLt8PKO0YMvlsEQM2NGqwj3Vw8hG8geN0mpgkcK8jqfK9OKII62xAF7vmYTyLeqwDltk8F\\\\\\\/FpHUgiuGADZhj0vb\\\\\\\/KC82O7PomDaP0uQCtobN72ULQopgQUVC00aDnb4ADEoL9CRdJq8vQYlyjFQnKsL8DCiSkNh3q+9+U+2HT40rLePBBiT2KCyX6sqmOwtnWT4QnYSGu65ib5vB0Rby73dZ8k6IbIeSINssyLXQU4peopcWrv+mouRFt8kaokuI+hOEddEte2C1Y2Pe0pU8RNq9ZkbV6XxFYwDZRNeFoKX4Ly2rh07i1Xa\\\\\\\/\\\\\\\/X\\\\\\\/NDg0vpmJCykWn5ZJumw7DVs0M6NXNBcPp6lwRS9ktaowp2uo5QjsCewHZuU6ZsGd7MKqkGqgCOLsHjgsq9wvc2lYVs1eEq0HYvtxC4EZbAUxcPfZlXbtLCZfP+XJ2OrKKt7D16zmB1QH7NGSR4W4sMvY1QmlWQ\\\\\\\/ovX4fP5oNnlAthP304cqwBehykMs8rUvxofs\\\\\\\/55gtSPW80O8+2n\\\\\\\/c3Oc0BQIjJHl+3CLQyBsyQ3OrfuSyHhP24TczEIDOWFtAyTgb96WwCvkR6hzouodEnkNMfsRUeC94474NdlYJRIPCNhBtjfO9gv2WZDnMmpbp43iyWMANSDvx3Eo7jlTOC6Tu8glSQ2PHXGwns3sgZPEJBFo9ZLPvIT2vgODkd1ddcFl6Bn1hTDWHKFbJtgeOEjzLBTae1BHZuI3MPKxlAr6WTJ9D4+J+DGEItLmPKbDg97PtUPNKKXNMVqFGprj2taj9MKk3Z7AAr4kwrNDk2AUDWDtIwUCIT0NKsARxF6CnYF9imHT\\\\\\\/GjJLknWUzxPdC+VDmCJLyK6JEbY03O8IQOn8HHuQEu6PJLzMq2io38+oGY27wtxr3SIkW2WLkvQQAviZlfuknR3485xCJz8exlKXhIOPYUV848j9IFGe9BlzHp\\\\\\\/1bNFa1a0t\\\\\\\/5LN0ZzxBE6zBsFkhj2G9BRnLfWFSNlSmVqKKPtO+UkpUi12PgBalwx1USrrzZ7Inka0CqxEr14RcY9DE0iqAQ1y\\\\\\\/+OMOVimAPRJzMT+Voyy7s8gdPLJqqQoWD622+W8HHU6GX2z+43aEa7jTW2AXZtf0cYRTXR3V0x9DJ\\\\\\\/gjWmU7bKsSAZqwAJcw6F\\\\\\\/\\\\\\\/plbRy9I61u6SkAJuSByhIn5vijsA4EsEq73\\\\\\\/y5ilu7zfifle33\\\\\\\/zWdzy27BCfJF43Mmmf6KMtpsjVEv73EFm1yf23NxVAG\\\\\\\/EaSQTFmtYpob0wcVie40DtD5I+ZFSuUV8BL47UD4Lzi9Z3v0acjffOU\\\\\\\/bbSH5TqBqC4fVl\\\\\\\/mqMYRpifbDuuS6506jbxDAwQRi9NsWQ1fGXESt1nz8Gi8YxiMzLitzD\\\\\\\/fSsXRckqm5x9Opkq2dTZhsFwVM+6poKYfwtKrIZVrRLexXvdRuHXbvpxSx7MHaRswj0lqJ8gxG83isDhxn7GOSTaDXob9W\\\\\\\/6NPggJKWf5WPTLjvZbs14L+LjBoIPYaOoBBbgDEBPnV6LIbtQgqAg3O\\\\\\\/aY6vEGnsOT+pekVEGDeaGT9GkPRgt6WdlRLTvQzadRFrudbCT24\\\\\\\/TJtq1P7Tv3fhZDINDaXR8NUlxCf0+QgLEVaKihBu0EptLPLQ6nDQklMXo8EIV9+b8hcd9Q9gy579sgkH3AeVSDOMaESEGy28+NIsW1sIRJ1hHa0KSzEl8J3OPfjYFwI5F2m7W\\\\\\\/VDhjXMfJ1FzDxRM4aIa6\\\\\\\/bLW0lg1YzdyxmdCfV0Mzudlf0nWFzIXgsV48\\\\\\\/xzrD2N+HBEZZ6Oa64swsZ2OPVV\\\\\\\/2ADSutPDvHHSTuCKl7rJfP2gX5D+0CJPYS0G89DxTkWrBTOfA7Ul1\\\\\\\/gt9Trfiqobe+zz2G8oZhK90hseQ\\\\\\\/RXITMC9S\\\\\\\/54L9RuO3mYXfI4cB4n9Jgvg0aXnDPsTLDmWUzpMLEZomB8qFgBDYrNbyX32Cbes+rdhtQ2H4s7WsQzjtud5CqKlixKqYQbQYI2dOdabcsRCENNpJ26mrWY6TlXW1JxGzFoc+0FtSalunlQ86+PXGSzBlVuzCRO43WX63n18xzBHiMkBZtOTjMYQN8XIpSeTj3N3kpRzR7JpMPwC5JGG7Dhtp85OFtwGrerinfkC3kC2p+OUytiTkVws8PK8nNtPSyfFb7Ymr+mTHyTnULU58eaYzgEj2B16o1H1u8cuY8p4TkwnvzTitZW5xlB4v5DgOVM+CZSlTvywz7RjFwPFg0uup3o5rRrXgu0ac+712rzJ++\\\\\\\/8raXwprsIOt4x6WYBxLbVpAvrfZrcFRHHetYVH\\\\\\\/tsOXf+5V9lLfUpr\\\\\\\/+jarC6C8+ro6O+alOQpCTGZ1warpXjPlmcrh5EzDESKf85JVRQMnhxW8n1\\\\\\\/T0vDGb6WXXNUgxA84AeiVMfIjYz\\\\\\\/v1VyPikNZHBDzw0ySlPycj+H\\\\\\\/l2dhqOCnVYEGFdqyIOtaKCOMARgRmH39Gs0jAVYC1m6gqP3Obq+j4FkXiVFxwOLbIC8LAff3DsSrNu\\\\\\\/1mQGWAHLm\\\u0022,\\\u0022iv\\\u0022:\\\u0022rFHhjD\\\\\\\/CIpUv9csYjHzVgA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002223e0ac4eaac6592087fdd9aafd9a34948659c58fe1c6ec6dafa6bbcc27c75f9c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223kOGZHkTRvZlr9y3gufx2ebJP7GRJY7A0HKaugqD1xhxbZ1fFEXdltVX2sHKbObLaWQpAHSWtHBKzuR6k9RS1HgOpcdfTU\\\\\\\/f4f9vR37qUjNHDj3W9hExx0ij6x0juPi3YSLC7bbYnWlMhJR2AwxN7nVdP5iKHrPMqi1BRsOCh+Xw8bWxnr27UiUg7KEiA1mUp+Gjw4yrMA4kbn4s3K5AMAUHT48nmoHHi8fPv+R53Ps47TpWhwjAqlZEEyQGXxmnm+MO+k2XFaKgLCgEJkvrtcFAMZ2oDc+39l09jCVvke\\\\\\\/mwYcmuZNIDOqN94Isf0qQG4H8TRSL0BYk4vkTlVxxO9LuQEFNA\\\\\\\/9JOVVSPgh4a+uvpQtl7nSPNCBHegE5x05R5OStVZp6ycQmmcPm0jPcG2ze\\\\\\\/GWu9Z84WIP36jq\\\\\\\/VsGiarCaYMjrU+pJ2lFoqT\\\\\\\/wpo2G8blMGj9zFodQfsVT7zO30QObGGmUpEqWh2wdrxz8dxF6o7Nh6QdFUPDnhpO6elCnzOMwDyU1nmgwyHNSFNbSlqe5LibAJlYXIQDdACdjYCTrGorElq+BXapdIDaW7dJidpnuS0J\\\\\\\/RVetziPePC8mQmAuOOqT6+8U\\\\\\\/iZPPFwvFAedFcg4KKcIt3eJxCakuWUBFZEHTqHoIkpORShj\\\\\\\/90mpyCXc1eC7H955EqpVfqixiI3cj5Ucjzd8gfLIa4j4gD6trTb\\\\\\\/xcAFMhG2yWRRF10Dt26dIq78EeQn\\\\\\\/+5E+oJWGbUaAa+wLcolUFybOPvYXFqq3F5tNL67VAPivevtp\\\\\\\/QWZXMEXPz2cKzdeZEWSl32b4m37vgxz0xmNXwBcuypFsm7mcmr2ATNvMenOJ23e9vJOJ49A3YNciBOH4TWvvZ7vkrtwGeeLkfvpKUUERLtgEVZD8rZn2MOpWD5+pbXaiuipJpld84e8ALhHf8vo8DqDcW9Oz9RP4DQJOe8LEdAI+R73yovajLTDwP1Y3LJq+lmvO9GWVZ5I9sn3ljlOiV9OPd9E\\\\\\\/yfmVcLRP09sqWDrrWsRBnJIqm3svQVd+p1PSEpuG3TwZ78Tgjj4DW7PX5MaHyCQmeJHwM0VX0cJRADqLK0Lskp7jG9QNR+ckNi+tyigElLRGj4xxexVDi7M9AKhVPx53z6Jja1GQZeGLMqrJfzbeYe5sq6DroEVAvodcHvh\\\\\\\/ZwZ9SuI+07X1QGbFYpLxuSgjDDTtGkg9hNfyDxrZWhUYdXesK2pXmaqdEhaWx4whtgXEm9s+e1zcE2toJaCJtnFpvWo3GQRI0Mphd8A+bvFWVyOXbeDfjO8VSFhVBe0PxwRf5RaMNPjbRkW7r1G0+vQycTOzrjpkPhjdPkQofWMBZeqKh92yW0T3DptC2mP7deeaFYBvEcdlijJVGz8VQp7Fm3EO0oOca7AI3uJd1qVaAOFzhsaFGvSaSaILzhK66SdS1hiqyAmzFoJS1rOU5OuCytWGwLO6TMR8iu+2veiEIfyjG18HrEeNsg7JVjKLEs3nqpe13MxlYZ6CO5rGl5+S5kZvxF7qUAvTNKjs5dFDqnifObUKu62JHYvWjrJUQ2mLShya3bWOHLWdkdk1glPBVePIKzZ1quj4w\\\\\\\/E8SZ9UV0CVuzU2HA92qnYnQzozpCg1PKpP3aaa1rylR7MFmoorcJclMm2RE9J6xBKEtjMcNLtib1HQq8sBX3w1qsYmYtcoBg+xN9V3tfMuM3iwKh32mqB+ol5c2HH4FKS4XNIT1TWT1T2vfgSUIKlzpOEG9OFtgWbtlorXp090dFiTNPMzMQCoFfVR5YLALTLu3YCB\\\\\\\/kHT34zFQdR\\\\\\\/DzJhjTNu4KmMMUqS6t6og\\\\\\\/L\\\\\\\/LkhkicE+Uza91sNZKa2zd6HRjOQrv7qrt+NO3c5NDVNKUxZl3MvBXg+muOgeky59ty\\\\\\\/zsHhjMtaZfrFy3je+1Ki+Qc4b8165mbU92rTEheEOVyCCSNWO31eGejlhVnv5F2RNA9Y+Rw7v5zoDb4XcynBzP+nK6YGNAg1Visx\\\\\\\/rZdBDpMn8WDsqKXPJJMjjYnkXY++IsO5WRgZWgRhdTj9n\\\u0022,\\\u0022iv\\\u0022:\\\u0022oZyw4lZGW+48ply8e9GmGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d59f3872c7aa5bbb36ffc158cefd94428229254153a8432c233e7e20f90a7490\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492855 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1429" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dyUDc7B2qDHktqRsZCweMv0e3SaHg9aBL4Om_MoBo9E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ndXaqCDZZqtMV5jaw_QUJvRV6iAyHuId_qUnFW8dXII" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1405" title="Deciphering the Genetics of Major End-Use Quality Traits in Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1437" title="Unravelling the Complex Genetics of Karnal Bunt (Tilletia indica) Resistance in Common Wheat (Triticum aestivum) by Genetic Linkage and Genome-Wide Association Analyses" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492855" id="node492855" data-pisa="ggg;9/5/1429" data-pisa-master="ggg;g3.119.400101" data-apath="/ggg/9/5/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp; Perils of Big Data)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="1">Daniel Alberto Sorensen</span> and <span class="highwire-citation-author" data-delta="2">Miguel Angel Toro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400101 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492855" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Epidemiology & Biostatistics, Statistics & Probability departments, IQ-Institute for Quantitative Health Science and Engineering, Michigan State University, East Lansing, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Alberto Sorensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology and Genetics, Faculty of Science and Technology, Aarhus University, Aarhus, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BAlberto%2BSorensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorensen%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BAlberto%2BSorensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Miguel Angel Toro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Producción Animal, Universidad Politécnica de Madrid, Madrid, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BAngel%2BToro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toro%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiguel%2BAngel%2BToro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1429" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492855" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1429.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492855" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1429.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492855" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1429.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492855" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The genetic architecture of complex human traits and diseases is affected by large number of possibly interacting genes, but detecting epistatic interactions can be challenging. In the last decade, several studies have alluded to problems that linkage disequilibrium can create when testing for epistatic interactions between DNA markers. However, these problems have not been formalized nor have their consequences been quantified in a precise manner. Here we use a conceptually simple three locus model involving a causal locus and two markers to show that imperfect LD can generate the illusion of epistasis, even when the underlying genetic architecture is purely additive. We describe necessary conditions for such “<em>phantom epistasis</em>” to emerge and quantify its relevance using simulations. Our empirical results demonstrate that phantom epistasis can be a very serious problem in GWAS studies (with rejection rates against the additive model greater than 0.28 for nominal p-values of 0.05, even when the model is purely additive). Some studies have sought to avoid this problem by only testing interactions between SNPs with R-sq. &lt;0.1. We show that this threshold is not appropriate and demonstrate that the magnitude of the problem is even greater with large sample size, intermediate allele frequencies, and when the causal locus explains a large amount of phenotypic variance. We conclude that caution must be exercised when interpreting GWAS results derived from very large data sets showing strong evidence in support of epistatic interactions between markers.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/apparent-epistasis" class="hw-term hw-article-keyword hw-article-keyword-apparent-epistasis" rel="nofollow">apparent epistasis</a></li><li class="kwd"><a href="/keyword/phantom-epistasis" class="hw-term hw-article-keyword hw-article-keyword-phantom-epistasis" rel="nofollow">phantom epistasis</a></li><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/imperfect-ld" class="hw-term hw-article-keyword hw-article-keyword-imperfect-ld" rel="nofollow">imperfect LD</a></li><li class="kwd"><a href="/keyword/missing-heritability" class="hw-term hw-article-keyword hw-article-keyword-missing-heritability" rel="nofollow">missing heritability</a></li><li class="kwd"><a href="/keyword/big-data" class="hw-term hw-article-keyword hw-article-keyword-big-data" rel="nofollow">Big Data</a></li></ul><p id="p-3">A big challenge in genetics is to understand how variation at the DNA sequences translates into phenotypic variation. Genome-wide-association (GWA) studies address part of this challenge by testing for the association between phenotype (or a disease indicator) with genotype, one locus at a time. In the last decade, many GWA studies were conducted; these studies have reported thousands of SNPs (single nucleotide polymorphism) associated to complex traits and diseases (<a href="http://www.ebi.ac.uk/gwas">http://www.ebi.ac.uk/gwas</a>).</p><p id="p-4">Recently, several studies in model organisms (<em>e.g.</em>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Mackay 2014</a>), humans (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Strange, Ask, and Nielsen 2013</a>) and agricultural species (<em>e.g.</em>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Huang, Xu, and Cai 2014</a>), have used genotype data linked to phenotypes to investigate the presence of epistatic interactions between loci. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cordell (2002</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2009)</a> and <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Wei, Hemani, and Haley (2014)</a> provide comprehensive reviews of the methods commonly used to detect epistatic interactions.</p><p id="p-5">There are several issues associated with studies aimed at detecting interactions, including matters of scale, the importance of the contribution of epistasis at the level of the genotype effects or at the level of the genotypic variance (<em>e.g.</em>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hill, Goddard, and Visscher 2008</a>) and how an interaction detected in a linear statistical model may be associated to biological pathways that underlie a complex trait (<em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Wang, Elston, and Zhu 2010</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aschard 2016</a>). The latter becomes particularly problematic when the markers used to assess associations between SNPs and phenotypes (or a disease indicator) are in imperfect linkage disequilibrium (LD) with the alleles at the causal loci (<em>i.e.</em>, those responsible for inter-individual genetic differences in a trait or disease phenotype). Under those conditions, evidence supporting the existence of a non-null interaction between markers does not necessarily provide definite evidence of epistasis at causal loci. Specifically, when the SNPs used in association analyses are in imperfect LD with the alleles at causal loci, linear regression on SNPs may lead to unaccounted variance, or <em>missing heritability</em> (<em>e.g.</em>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Manolio <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">de Los Campos <em>et al.</em> 2015</a>). When this unaccounted additive signal is correlated with interaction contrasts, the “illusion” of epistasis is created even for traits that are purely additive.</p><p id="p-6">Several authors have expressed concerns about the role that LD can have on the detection of epistasis (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Wood <em>et al.</em> 2014</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">W.-H. Wei, Hemani, and Haley 2014</a>). However, these problems have not been quantified nor have they been given a precise mathematical treatment. In this study, we present a simple three locus model involving a causal (unobserved) locus and two markers that makes explicit how <em>phantom epistasis</em> may emerge even in systems that are strictly additive. We use this model to derive a set of conditions that are necessary for the occurrence of phantom epistasis, and quantify the magnitude of the problem using simulations based on real human genotypes from the UK-Biobank. The existence of phantom epistasis is also studied using extensions of the model that include dominance and multiple loci. Our results indicate that imperfect LD can lead to seriously inflated type-I error rates. We also show that the rate of detection of phantom epistatic interactions increases with sample size; this should be considered when testing for epistatic interactions using big data sets such as the ones that are becoming available.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-7">We begin by considering a simple model with three biallelic loci. One of them, denoted as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-1.gif"/></noscript></span></span>, represents a causal locus (also referred as to the ‘quantitative trait locus’, QTL) on observation <em>i</em> and has a direct effect on the expression of the phenotype <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-2.gif"/></noscript></span></span>. The other two loci, denoted as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-4.gif"/></noscript></span></span>, are markers that are possibly in LD with the QTL but have no causal effect on the phenotype. For SNPs, a standard practice is to code genotypes by counting at each of the loci the number of copies of a reference allele carried by the <em>i<sup>th</sup></em> individual. Here, to facilitate the presentation we assume that genotypic codes and phenotypes are expressed as deviations from their corresponding means; therefore <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-5.gif"/></noscript></span></span>. In this setting, a single locus strictly additive model takes the form<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">[1]</span></span>where b is the additive effect of an allele substitution at locus z, and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-7.gif"/></noscript></span></span> is an error term. Evidently, with only one causal locus there is no epistasis. Next, suppose that an instrumental regression of the form<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">[2]</span></span>is used to investigate the presence of epistasis. Here, the <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-9.gif"/></noscript></span></span> are regression coefficients that are functions of the QTL effect (<em>b</em>) and of the (multilocus) LD involving the two markers and the QTL genotypes. In the population, given the centered genotype codes, the regression coefficients entering in the right-hand-side of [2] are<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-10.gif"/></noscript></span></span>If the random residual <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-11.gif"/></noscript></span></span> in expression [1] is orthogonal to the genotypes, then <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-12.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-14.gif"/></noscript></span></span>. Thus, the population regression coefficients are defined by<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">[3]</span></span>This indicates that the regression coefficients of the instrumental model [2] are not only functions of the QTL effect (<em>b</em>) and of pair-wise (1<sup>st</sup> order) LD but also of higher order LD, <em>e.g.</em>, joint disequilibrium at three loci, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-16.gif"/></noscript></span></span>. The moments involved in the right hand-side of [3] are diploid genotypic measurements of LD. Under random mating these genotypic measures of LD are equal to twice the standard haploid measures of LD (the <em>D</em>-coefficients for two and three loci linkage disequilibrium; see section 1 of the Supplementary Methods for further details).</p><p id="p-8">In the population, the interaction effect <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-17.gif"/></noscript></span></span> is given by a linear combination involving two-loci LD between one of the markers and the QTL, and three-loci LD involving the two markers and the QTL: <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-18.gif"/></noscript></span></span>. Here, the <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-19.gif"/></noscript></span></span> are the entries of the third row of the inverse of the coefficient matrix</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-20.gif"/></noscript></span></span><div id="sec-2" class="subsection"><h3>Required conditions for phantom epistasis</h3><p id="p-9">If the QTL is in LE with the two markers, then <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-21.gif"/></noscript></span></span>. Consequently, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-22.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-23.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-24.gif"/></noscript></span></span>. Therefore, all elements of the right-hand-side of [3] are equal to zero and, thus <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-25.gif"/></noscript></span></span>. Therefore, for phantom epistasis to emerge <em>a <span class="underline">first necessary condition</span></em> is that <em><span class="underline">the QTL must be in LD with at least one of the SNPs</span></em>. (This condition is a special case of a more general condition that we discuss below.)</p><p id="p-10">On the other extreme, if there is perfect LD between the QTL and the marker pair <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-26.gif"/></noscript></span></span>, then the QTL genotype can be expressed as a linear function of the two marker genotypes <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-27.gif"/></noscript></span></span>. In this case, a linear regression on the two markers captures fully the QTL variance and therefore the interaction term will be equal to zero. (A derivation of this intuitive result is presented section 2 of the Supplementary Methods.) Therefore, perfect LD is a sufficient condition for <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-28.gif"/></noscript></span></span>. Consequently, a <em><span class="underline">second necessary condition</span></em> for phantom epistasis to emerge is <em><span class="underline">imperfect LD between the QTL and the marker pair</span></em>. This guarantees that some fraction of the QTL variance is not captured by linear regression on the two marker genotypes. Furthermore, if the left-out QTL signal is not orthogonal to the interaction contrast <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-29.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-30.gif"/></noscript></span></span></p><p id="p-11">Consider now an intermediate case where one of the markers (say <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-31.gif"/></noscript></span></span>) is independent of the pair formed by the QTL and the other marker <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-32.gif"/></noscript></span></span>. This implies that <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-33.gif"/></noscript></span></span>. Under this condition, because the two markers are in LE, the coefficient matrix and its inverse (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-34.gif"/></noscript></span></span>) is diagonal; therefore, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-35.gif"/></noscript></span></span>. Moreover, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-36.gif"/></noscript></span></span>, implying that <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-37.gif"/></noscript></span></span>. Therefore, a <em><span class="underline">third necessary condition</span></em> for phantom epistasis to emerge is that <em><span class="underline">the three loci must be jointly in LD</span></em><span class="underline">.</span></p><p id="p-12"><strong><em>In conclusion</em>,</strong> in the system discussed above, <em><span class="underline">phantom epistasis can emerge if the three loci are in mutual but imperfect LD</span></em>. Unfortunately, this condition cannot be assessed when the QTL genotype is unknown. Empirically only LD between the two markers can be assessed.</p></div><div id="sec-3" class="subsection"><h3>Phantom epistasis in multi-locus models</h3><p id="p-13">In an additive multi-locus model expression [1] becomes <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-38.gif"/></noscript></span></span> When testing pairwise interactions between markers the empirical model [2] remains unchanged; therefore, if the two markers involved in the instrumental model are in LE the matrix representing left-hand side of the OLS systems of equations is diagonal. In this setting, phantom epistasis emerges when the third right-hand side term, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-39.gif"/></noscript></span></span>, is nonzero. Since the expectation is a linear operator, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-40.gif"/></noscript></span></span>. If one of the markers is in LE with the other marker and with all the QTL then <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-41.gif"/></noscript></span></span>, thus there will not be phantom epistasis.</p><p id="p-14"><strong><em>In the presence of dominance</em>,</strong> the causal (single locus) model becomes <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-42.gif"/></noscript></span></span> where a and d are additive and dominance values, respectively. If the empirical model of expression [2] is used to test for epistatic interactions then the left-hand-side of expression [3] remains unchanged, but the right-hand-side becomes<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-43.gif"/></noscript></span></span>indicating that both dominance and additive effects can indeed contribute to phantom epistasis. In this system, phantom epistasis will emerge whenever <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-44.gif"/></noscript></span></span> The conditions needed for phantom epistasis to emerge in a system involving dominance are the same as the ones described for the additive model. These include, <span class="underline">first</span>, imperfect LD between <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-45.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-46.gif"/></noscript></span></span> with the marker pair <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-47.gif"/></noscript></span></span> such that either <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-48.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-49.gif"/></noscript></span></span> or both cannot be fully explained by a linear combination of the two markers. <span class="underline">Second</span>, phantom epistasis requires mutual LD at the three loci. If one of the markers (say <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-50.gif"/></noscript></span></span>) is independent of the other-marker-QTL pair, then, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-51.gif"/></noscript></span></span>.</p></div><div id="sec-4" class="subsection"><h3>Simulation studies</h3><p id="p-15">The analytical results presented in the previous section indicate that multi-locus LD plays an important role in determining whether phantom epistasis may emerge. To shed light on the nature and the magnitude of the problem we conducted Monte Carlo simulations using real human genotypes of distantly related white Caucasian individuals from the UK-Biobank.</p><p id="p-16">In a <strong><em>first set of simulation scenarios</em></strong>, we generated data according to an additive model with a single causal locus (as in [1]) that explained either 0.5 or 1% of the phenotypic variance. The position of the QTL genotype <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-52.gif"/></noscript></span></span> was determined by randomly choosing a marker position on human chromosome 1; marker <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-53.gif"/></noscript></span></span> was always adjacent (“to the left”) to the QTL whereas the other marker (<span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-54.gif"/></noscript></span></span>) was placed at increasing base pair distance (“to the right”) of the QTL.</p><p id="p-17">We analyzed the simulated phenotypes using an instrumental model such as the one in [2] extended with inclusion of an intercept and the top 5 SNP-derived PCs to avoid confounding due to any substructure that may be present (see section 3 of Supplementary Methods for details). The null hypothesis <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-55.gif"/></noscript></span></span> was rejected at a 0.05 significance level; therefore, rejection rates over 0.05 were interpreted as indicative of phantom epistasis. All analyses were done with R (Rcore development team (2012) using the BGData R-package (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Grueneberg and de los Campos, 2019</a>).</p><p id="p-18">Since the power to detect a non-null interaction effect depends on sample size we analyzed data using a sample sizes of n = 10K, 50K, 100K and 250K (K = 1,000).</p></div><div id="sec-5" class="subsection"><h3>Alternative simulation scenarios</h3><div id="sec-6" class="subsection"><h4>SNPs in different chromosomes:</h4><p id="p-19">To assess the potential impact of long-range LD, in a new analysis setting we maintained the QTL-proximal-marker pair (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-56.gif"/></noscript></span></span>) in chromosome 1 as in our main simulation scenario but now positioned the distal marker (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-57.gif"/></noscript></span></span>) in a randomly chosen position on chromosome 2. In this data set, we do not expect high levels of LD between SNPs in different chromosomes; however, at least in theory, this could happen if selection favors combinations of alleles, thus inducing LD at physically unlinked loci (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bulmer 1971</a>).</p></div><div id="sec-7" class="subsection"><h4>Multi-locus models:</h4><p id="p-20">To consider the problem of phantom epistasis in multi-locus models we <span class="underline">first</span> assumed that the genetic architecture of the trait was determined by a major QTL plus an infinitesimal effect that was strictly additive. The trait heritability was 0.5, the main QTL explained 1% of the variance and the infinitesimal component the remaining 49% (see Section 3 of the Supplementary Methods for further details).</p><p id="p-21"><span class="underline">Second</span>, we simulated data using a model that contains three marker-QTL pairs: <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-58.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-59.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-60.gif"/></noscript></span></span>. As before, the trait was strictly additive with the genetic effect, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-61.gif"/></noscript></span></span>, explaining 1% of the phenotypic variance (each QTL explained 1/3 of the total genetic variance). The three pairs were in chromosome 1. Within a pair, the marker was the SNP immediately adjacent to the QTL in the array. To study the effects of LD we moved pairs 1 and 3 further apart and always maintained the 2<sup>nd</sup> pair in the middle point (based on base-pair distance). The empirical model used to test for interaction was as that in expression [2] with <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-62.gif"/></noscript></span></span> in place of <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-63.gif"/></noscript></span></span>; that is we tested for phantom epistasis between SNPs in the first and third pair. This scenario aimed at describing problems that may emerge due to an unaccounted QTL (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-64.gif"/></noscript></span></span> in this case) which may be in simultaneous LD between the two SNPs involved in the interaction.</p></div></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-22">The genotypes used in the simulation were from the UK Biobank. Data were acquired under project identification number 15326. The data are available for all bonafide researchers and can be obtained by applying at <a href="http://www.ukbiobank.ac.uk/register-apply/">http://www.ukbiobank.ac.uk/register-apply/</a>. The Institutional Review Board (IRB) of Michigan State University has approved this research with the IRB number 15-745. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7848233">https://doi.org/10.25387/g3.7848233</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><p id="p-23">We begin presenting results from our main simulation scenario which involves a single-locus with two SNPs, one proximal (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-65.gif"/></noscript></span></span> and one distal (<span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-66.gif"/></noscript></span></span> to the QTL.</p><p id="p-24"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows measures of linkage disequilibrium between the three loci <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-67.gif"/></noscript></span></span>involved in the system. The average (across Monte Carlo replicates) proportion of variance of the QTL (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-68.gif"/></noscript></span></span>) explained by the most adjacent marker (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-69.gif"/></noscript></span></span>) was about 0.085; however, the distribution of this statistic is highly skewed. When <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-70.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-71.gif"/></noscript></span></span> were the two flanking markers of the QTL, on average they jointly explained 15% of the QTL variance. Therefore, on average there was a sizable rate of “missing” heritability. The R-sq. between <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-72.gif"/></noscript></span></span> and either the other marker or the QTL, falls very quickly for lags between 0-0.5Mb and reached near zero values at approximately 1 Mb (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average R-squared between pairs of loci and proportion of variance of the QTL genotype explained by the two markers, , vs. distance between the QTL () and the distal marker (). Marker  was always adjacent to the QTL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-996921391" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Average R-squared between pairs of loci and proportion of variance of the QTL genotype explained by the two markers, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;em&gt;vs.&lt;/em&gt; distance between the QTL (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-74&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and the distal marker (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-75&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Marker &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-76&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; was always adjacent to the QTL.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1429/F1.medium.gif" width="440" height="423"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1429/F1.medium.gif" width="440" height="423"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492979" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Average R-squared between pairs of loci and proportion of variance of the QTL genotype explained by the two markers, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F1/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F1/embed/mml-math-73.gif"/></noscript></span></span>, <em>vs.</em> distance between the QTL (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F1/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F1/embed/mml-math-74.gif"/></noscript></span></span>) and the distal marker (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F1/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F1/embed/mml-math-75.gif"/></noscript></span></span>). Marker <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F1/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F1/embed/mml-math-76.gif"/></noscript></span></span> was always adjacent to the QTL.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> displays empirical rates of rejection by BP-distance between the QTL and the distal marker (<span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-77.gif"/></noscript></span></span>), by sample size and by proportion of variance explained by the QTL. In absence of phantom epistasis, the empirical rejection rate should be very close to the significance level (0.05). For the largest sample size, the curve relating empirical rejection rates with BP distance was clearly above 0.05 for distances of up to 2MB. In all the scenarios, the highest rejection rates were observed when <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-78.gif"/></noscript></span></span> and the QTL were at a distance of about 0.15 MB; here the empirical rejection rate was ∼0.13 when the QTL explained 0.5% of the phenotypic variance and as high as 0.17 when the QTL explained 1% of the variance. The latter (0.17) is more than three times the nominal rate of rejection expected under the absence of phantom epistasis (0.05). The curves relating empirical rejection rates with physical distance reach the nominal rejection rate of 0.05 at ∼1Mb for n = 10,000; however, for larger sample size the curves stayed above 0.05 even for distances longer than 1Mb.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical rejection rates vs. distance between the QTL and the distal marker, by proportion of variance explained by the QTL (left and right panels) and sample size (curves). In the simulations, a single QTL () had an additive effect that explained either 1% (left) or 0.5% (right) of the phenotypic variance. The empirical model considered two SNPs with no causal effect. One of them () was adjacent to the QTL and the other one () was placed at increasing distance from the pair (). Rejection of the null hypothesis (no interaction between  and was conducted at a 0.05 significance level. Empirical rejection rates above 0.05 are indicative of phantom epistasis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-996921391" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Empirical rejection rates &lt;em&gt;vs.&lt;/em&gt; distance between the QTL and the distal marker, by proportion of variance explained by the QTL (left and right panels) and sample size (curves). In the simulations, a single QTL (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-79&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) had an additive effect that explained either 1% (left) or 0.5% (right) of the phenotypic variance. The empirical model considered two SNPs with no causal effect. One of them (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-80&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) was adjacent to the QTL and the other one (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-81&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) was placed at increasing distance from the pair (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-82&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Rejection of the null hypothesis (no interaction between &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-83&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-84&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;was conducted at a 0.05 significance level. Empirical rejection rates above 0.05 are indicative of phantom epistasis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1429/F2.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1429/F2.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493188" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Empirical rejection rates <em>vs.</em> distance between the QTL and the distal marker, by proportion of variance explained by the QTL (left and right panels) and sample size (curves). In the simulations, a single QTL (<span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-79.gif"/></noscript></span></span>) had an additive effect that explained either 1% (left) or 0.5% (right) of the phenotypic variance. The empirical model considered two SNPs with no causal effect. One of them (<span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-80.gif"/></noscript></span></span>) was adjacent to the QTL and the other one (<span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-81.gif"/></noscript></span></span>) was placed at increasing distance from the pair (<span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-82.gif"/></noscript></span></span>). Rejection of the null hypothesis (no interaction between <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-83.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F2/embed/mml-math-84.gif"/></noscript></span></span>was conducted at a 0.05 significance level. Empirical rejection rates above 0.05 are indicative of phantom epistasis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> displays another way of viewing the simulation results of <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> where the average rejection rate is calculated within bins of R-sq. between the two markers. When the two markers were uncorrelated, rejection rates were very close to 0.05 indicating absence of phantom epistasis. However very small LD between the two markers generates considerably higher rejection rates: an <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-85.gif"/></noscript></span></span> leads to rejection rates as high as 0.28 with the largest sample size when the proportion of variance explained by the QTL was 1%. The maximum rejection rates occur when the R-sq. between markers is between 0.1 to 0.2. Beyond this value in the range (0.2-0.9) rejection rates follow a linear decline.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical rejection rates vs. R-squared between the proximal and distal marker, by proportion of variance explained by the QTL (left and right panels) and sample size (curves). The simulation setting here was the same as that in Figure 2: a single QTL () had an additive effect that explained either 1% (left) or 0.5% (right) of the phenotypic variance. The empirical model considered two SNPs with no causal effect. One of them () was adjacent to the QTL and the other one () was placed at increasing distance from the pair (). Rejection of the null hypothesis (no interaction between  and was conducted at a 0.05 significance level. Empirical rejection rates above 0.05 are indicative of phantom epistasis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-996921391" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Empirical rejection rates &lt;em&gt;vs.&lt;/em&gt; R-squared between the proximal and distal marker, by proportion of variance explained by the QTL (left and right panels) and sample size (curves). The simulation setting here was the same as that in Figure 2: a single QTL (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-86&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) had an additive effect that explained either 1% (left) or 0.5% (right) of the phenotypic variance. The empirical model considered two SNPs with no causal effect. One of them (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-87&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) was adjacent to the QTL and the other one (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-88&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) was placed at increasing distance from the pair (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-89&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Rejection of the null hypothesis (no interaction between &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-90&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-91&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;was conducted at a 0.05 significance level. Empirical rejection rates above 0.05 are indicative of phantom epistasis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1429/F3.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1429/F3.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492962" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Empirical rejection rates <em>vs.</em> R-squared between the proximal and distal marker, by proportion of variance explained by the QTL (left and right panels) and sample size (curves). The simulation setting here was the same as that in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>: a single QTL (<span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-86.gif"/></noscript></span></span>) had an additive effect that explained either 1% (left) or 0.5% (right) of the phenotypic variance. The empirical model considered two SNPs with no causal effect. One of them (<span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-87.gif"/></noscript></span></span>) was adjacent to the QTL and the other one (<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-88.gif"/></noscript></span></span>) was placed at increasing distance from the pair (<span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-89.gif"/></noscript></span></span>). Rejection of the null hypothesis (no interaction between <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-90.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/F3/embed/mml-math-91.gif"/></noscript></span></span>was conducted at a 0.05 significance level. Empirical rejection rates above 0.05 are indicative of phantom epistasis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Finally, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the empirical rejection (for the scenario where the QTL explained 1% of the phenotypic variance) rates by minor-allele frequency and the R-squared between the two markers involved in the interaction. For any given bin of minor-allele frequency, the rejection rate was close to 0.05 when R-squared was close to zero, it increases reaching a maximum for R-squared values of 0.1-0.3, and then shows a decline reaching again values close to 0.05 when R-squared is larger than 0.9. These patterns are the same as the ones displayed in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. Interestingly, within a bin of R-squared, when phantom epistasis existed (<em>i.e.</em>, when R-squared was neither null nor perfect) rejection rates were maximum for intermediate allele frequencies and decreased with extreme allele frequencies. This is likely a consequence of the fact that intermediate allele frequencies confer higher power to the test.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap of empirical rejection rates by sample size (left and right panels), minor allele frequency (average of the two SNPs) and R-squared between the two SNPs involved in the interaction. The simulation setting here was the same as the one used to produce the results of Figures 2 and 3. The results in the figure correspond to an additive QTL that explained 1% of the variance and a sample size of either 250K (left) or 50K (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-996921391" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heatmap of empirical rejection rates by sample size (left and right panels), minor allele frequency (average of the two SNPs) and R-squared between the two SNPs involved in the interaction. The simulation setting here was the same as the one used to produce the results of Figures 2 and 3. The results in the figure correspond to an additive QTL that explained 1% of the variance and a sample size of either 250K (left) or 50K (right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1429/F4.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1429/F4.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1429/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493027" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Heatmap of empirical rejection rates by sample size (left and right panels), minor allele frequency (average of the two SNPs) and R-squared between the two SNPs involved in the interaction. The simulation setting here was the same as the one used to produce the results of <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">3</a>. The results in the figure correspond to an additive QTL that explained 1% of the variance and a sample size of either 250K (left) or 50K (right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The results from the model with <strong><em>SNPs in different chromosomes</em></strong> are provided in Table S1. In all cases, the rejection rates were very close to 0.05 suggesting that phantom epistasis between SNPs on different chromosomes, if it exists, it occurs at a rate that does not induce an inflation of rejection rates above the significance level.</p><div id="sec-10" class="subsection"><h3>Multi-locus models</h3><p id="p-33">Figure S1 summarizes the results from the simulation scenarios that included an <strong><em>infinitesimal effect</em></strong>. The empirical rejection rates were almost identical to those obtained, for the same sample size, with the single-QTL model (compare Figure S1 with the curves for N = 50K and 250K in the left panel of <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). This suggests that the interaction contrasts were quasi orthogonal to the infinitesimal effect, which is probably a consequence of the short span of LD in this population.</p><p id="p-34">The results from a scenario involving <strong><em>three marker-QTL pairs</em></strong> are summarized in Figure S2. The rates of rejection in this simulation scenario were very similar to the ones observed in the single QTL model (compare Figure S2 with curves corresponding to N-50K and N = 250K in the left panel of <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>) with mildly elevated rejection rates when the distance between pairs 1 and 3 was between 0.1-1Mb, due to the non-zero term <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-92.gif"/></noscript></span></span> of the right-hand-side of [3], that causes <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-93.gif"/></noscript></span></span>. However, as the distance between the markers involved in the interaction (<em>i.e.</em>, the 1<sup>st</sup> and 3<sup>rd</sup> marker) increased, the estimated rejection rates approached the nominal rejection rate. Specifically, when the distance between the 1<sup>st</sup> and the 2<sup>nd</sup> pair is of 2Mb or larger the three pairs become independent and phantom epistasis vanishes.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-35">There is a substantial amount of literature reporting the presence of epistasis affecting complex traits but results, when scrutinized, have been controversial. Sometimes the controversy spawns from the suspicion that epistatic interactions may be capturing additive signals that were missed by the baseline additive model used to test interactions. For instance, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hemani <em>et al.</em> (2014)</a> identified 30 pairs of SNPs that interact influencing gene expression and that were replicated across two independent studies. In a subsequent study (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Wood <em>et al.</em> 2014</a>) replicated many of the interactions reported by Hemani <em>et al.</em>; however, in each case, using sequence data, a single third variant could explain all the apparent epistasis. This happened even after removal of all pairs of SNPs with <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-94.gif"/></noscript></span></span> which was suggested by <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Wei, Hemani, and Haley (2014)</a> as a means to minimize confounding due to haplotype effects.</p><p id="p-36">However, the problem of why and under what conditions additive effects may generate “epistatic signals” has not be formalized. In this work, we considered a simple three locus model to reveal conditions that lead to phantom epistasis. We show that phantom epistasis emerges in the presence of <span class="underline">simultaneous but imperfect mutual LD between the three loci</span> (the QTL and the two markers involved in the interaction). This conceptually simple three loci model can be extended to more complex settings (<em>e.g.</em>, multiple QTL-marker pairs) without affecting the underlying source of the principle: if additive QTL variance is imperfectly captured by linear regression on markers and the unexplained variation is not orthogonal to interaction contrasts, then phantom epistasis emerges.</p><p id="p-37">We stress that simultaneous LD between the triplet <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-95.gif"/></noscript></span></span>is required for phantom epistasis to emerge. At least in theory, it is possible to have cases where the two SNPs involved in an interaction are in LE yet, they are jointly in LD with an unobserved QTL. An example featuring such patterns is discussed by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Wei <em>et al.</em> (2013)</a> who present a model with dominance at a causal locus that generates phantom epistasis between two flanking markers that are marginally independent. In Wei’s example the two SNPs are jointly in LD with the QTL and therefore, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-96.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-97.gif"/></noscript></span></span> are nonzero, explaining why the model generates phantom epistasis. On the other hand, if one of the markers, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-98.gif"/></noscript></span></span>, is independent of the remaining pair <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-99.gif"/></noscript></span></span> then <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-100.gif"/></noscript></span></span> and phantom epistasis does not not emerge.</p><div id="sec-12" class="subsection"><h3>Inferences Under imperfect LD</h3><p id="p-38">Several authors (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">M E Goddard 2009</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">de Los Campos, Sorensen, and Gianola 2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gianola <em>et al.</em> 2015</a>) have studied the role of imperfect LD on related inferential problems, including missing heritability and whether imperfect LD can lead to estimates of genomic correlations between traits that are different than the underlying genetic correlations (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Gianola <em>et al.</em> 2015</a>). In all these cases, imperfect LD generates inferential difficulties; phantom epistasis is another inferential problem arising when the markers used for inferences are in imperfect LD with causal variants.</p><p id="p-39">Testing interactions among weakly correlated SNPs only (<em>e.g.</em>, considering only SNP-pairs with <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-101.gif"/></noscript></span></span> is not a solution: indeed, our simulation results show that weak LD between markers (<em>e.g.</em>, R-sq. between 0.05 and 0.1) can lead to large numbers of false discoveries especially when sample size is large. However, our simulations also show that near independence between the two SNPs (<em>e.g.</em>, R-sq. &lt;0.01), a condition that in the data set used in this study was achieved at distances of 1.5Mb-2Mb or longer, was enough in the scenarios tested to guarantee rejection rates very close to the significance level.</p></div><div id="sec-13" class="subsection"><h3>Perils of Big Data</h3><p id="p-40">If phantom epistasis exists (<em>i.e.</em>, if the population coefficient <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1429/embed/mml-math-102.gif"/></noscript></span></span>) whether it is detected or not depends on the power of the study which increases with sample size. Our simulation results demonstrate this clearly: pairwise R-sq. of 0.1 between markers and large sample size (<em>e.g.</em>, n &gt; 100K) generates clear signs of phantom epistasis. However, rejection rates are not highly elevated over the significance level when sample size was smaller (n = 10k) because at that R-sq. the size of the interaction effect is small and therefore the power to detect such small interaction effect with small sample size is low. Big Data are a blessing for genomic analysis of complex traits; however, in some cases, large sample size can make an inferential problem even more problematic.</p></div><div id="sec-14" class="subsection"><h3>The proportion of variance explained by the QTL plays a role similar to that of sample size</h3><p id="p-41">The larger the amount of variance explained by the QTL, the higher the power to detect phantom epistasis due to imperfect LD. To assess this, we repeated our simulation with a QTL that explained 50% of the phenotypic variance. The results (see Figure S3) showed, as one would expect, higher rejection rates than the one observed when the QTL explained a small fraction of the phenotypic variance (compare <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a> with Figure S3). Importantly, and in agreement with what our model predicts, the rejection rate reaches the significance level when the two SNPs become independent.</p></div><div id="sec-15" class="subsection"><h3>Phantom Epistasis in multi-locus models</h3><p id="p-42">The presence of <strong><em>a polygenic effect</em></strong> in our simulations did not lead to notoriously inflated rejection rates compared with the ones detected with the single locus model suggesting that the interaction contrasts were quasi orthogonal to the infinitesimal effect. This is probably because LD in this population spans over very short distances and therefore it is highly unlikely to find a pair of markers that are strongly correlated with infinitesimal effects emerging from large numbers of loci distributed over the genome. On the other hand, local polygenicity (<em>i.e.</em>, the accumulation of many small-effect loci in a region with moderate or high LD) may lead to more serious apparent epistasis.</p><div id="sec-16" class="subsection"><h4>Local epistasis?</h4><p id="p-43">Several studies have reported results highlighting the importance of ‘local’ epistatic interactions (<em>e.g.</em>, <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Wei, Hemani, and Haley 2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">He <em>et al.</em> 2017</a>). From a biological perspective, it is plausible that multiple mutations in a gene may have collectively a larger impact than the simple sum of the effects of each mutation individually. And this could manifest itself as “haplotype effects” (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Haig 2011</a>). However, phantom epistasis provides an alternative explanation for why epistatic interactions detected in GWAS occur between loci that are physically close. Indeed, we show analytically and empirically that LD between SNPs is required for phantom epistasis to appear, thus, phantom epistasis is expected to be predominantly a ‘local’ phenomena.</p></div><div id="sec-17" class="subsection"><h4>Inflation of rejection rates due to incorrect statistical assumptions?</h4><p id="p-44">Based on an earlier version of this manuscript Dr. P. Visscher (personal communication) commented that under imperfect LD the distribution of the error terms of the empirical model is not normal. Indeed, because the QTL genotype has three levels, under imperfect LD the distribution of the error terms would be a mixture of three normal distributions with different means. This could certainly explain the inflation of rejection rates in small samples because the conditions for the test statistic to follow a <em>t</em> distribution are not met. However, in large samples, the test statistic follows a normal distribution even if errors are not Gaussian (this is simply an application of the Central Limit Theorem). Therefore, with the size of samples used in standard GWA studies, lack of normality of the error term should not be a cause for inflated rejection rates.</p><p id="p-45">A second possible cause for inflated rejection rate of the null hypothesis can be underestimation of the error variance of the empirical model. However, it can be shown that the estimator of the error variance of the empirical model used in the simulations is unbiased with respect to the marginal distribution of the data. Therefore, phantom epistasis cannot be attributed to underestimation of the error variance.</p></div></div><div id="sec-18" class="subsection"><h3>The additive-non-additive conundrum</h3><p id="p-46">Quantitative genetics studies properties of complex traits using regression analysis. In the field a careful distinction is made between observable and causal features of complex traits. For instance, it is well established that the linear regression of a phenotype on allele content yields estimates of the average effect of allele substitution and that both truly additive as well as dominance and epistatic effects can contribute to allele substitution effects. Furthermore, theoretical and empirical research has demonstrated that highly non-linear systems can generate signals that can often be explained almost completely with a linear model (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hill, Goddard, and Visscher 2008</a>). For this reason, in general, one cannot make causal statements about gene action from observational variance component analyses (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">W. Huang and T. F. C. Mackay 2016</a>). Complicating matters even further we show in this study that the opposite can happen: under a purely additive model, imperfect LD can generate non-additive signals!</p><p id="p-47">The recognition that phantom epistasis may be an important phenomenon does not negate the relevance of gene-gene interactions at the causal level. It simply stresses the difficulties that one faces when trying to learn about causal features of a system using observational data and inputs (markers) which are proxies for the underlying variants that may have causal effects on traits.</p><div id="sec-19" class="subsection"><h4>Phantom epistasis: an opportunity to improve predictive performance?</h4><p id="p-48">In this work, we have stressed that imperfect LD can limit the possibility to learn about causal effects. However, linear and non-linear genomic regressions can be very powerful predictive machines, and it is well established that the model that is best for inferences is not necessarily the best predictive tool. Phantom epistasis creates inferential problems but also opens opportunities for improving prediction models. Indeed, by capturing signals that are missed by an additive model, non-linear models using interactions between markers may increase the amount of genetic variance captured and improve prediction accuracy. This may explain, for instance why some non-linear models such as kernel regressions have shown better predictive performance than additive models, especially in breeding populations with long-span LD and low marker density (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">de los Campos <em>et al.</em> 2010</a>).</p></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">The authors thank the participants and the personnel in charge of generating, curating and maintaining the data of the UK Biobank. GDLC and DS received financial support from NIH Grants R01GM099992 and R01GM101219. MAT received financial support from Spain’s Ministry of <em>Economía y Competitividad</em> (grant CGL2016-75904-C2-2-P, MT). This work was supported in part by Michigan State University through computational resources provided by the Institute for Cyber-Enabled Research. We presented early versions of this work in a genetics seminar in George August Universitat and at the 2017 EAAP (European Federation of Animal Science) meetings. We are grateful for the comments received and would like to particularly thank the feedback offered by Johanes Martini. We also would like to thank Dr. P. Visscher for comments provided to an earlier version of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-50">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7848233">https://doi.org/10.25387/g3.7848233</a>.</p></li><li class="fn" id="fn-2"><p id="p-51"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-03-26"><span class="received-label">Received </span>March 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-23"><span class="accepted-label">Accepted </span>February 23, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 de los Campos <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.1" data-doi="10.1002/gepi.21989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aschard</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Perspective on Interaction Effects in Genetic Association Studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">678</span>–<span class="cit-lpage">688</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21989</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D40%26rft.spage%253D678%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.2" data-doi="10.1086/282718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">The Effect of Selection on Genetic Variability.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/282718</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D105%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971J801400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.3" data-doi="10.1093/hmg/11.20.2463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cordell</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Epistasis: What It Means, What It Doesn’t Mean, and Statistical Methods to Detect It in Humans.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2463</span>–<span class="cit-lpage">2468</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/hmg/11.20.2463</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DCordell%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D11%26rft.issue%253D20%26rft.spage%253D2463%26rft.epage%253D2468%26rft.atitle%253DEpistasis%253A%2Bwhat%2Bit%2Bmeans%252C%2Bwhat%2Bit%2Bdoesn%2527t%2Bmean%252C%2Band%2Bstatistical%2Bmethods%2Bto%2Bdetect%2Bit%2Bin%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252F11.20.2463%26rft_id%253Dinfo%253Apmid%252F12351582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/11.20.2463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12351582&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178515900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.4" data-doi="10.1038/nrg2579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cordell</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detecting Gene-Gene Interactions That Underlie Human Diseases.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2579</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCordell%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D392%26rft.epage%253D404%26rft.atitle%253DDetecting%2Bgene-gene%2Binteractions%2Bthat%2Bunderlie%2Bhuman%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2579%26rft_id%253Dinfo%253Apmid%252F19434077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19434077&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.5" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-Parametric Genomic-Enabled Prediction of Genetic Values Using Reproducing Kernel Hilbert Spaces Methods.</span> Genet. Res. (Camb) <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.6" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic Heritability: What Is It?</span> PLoS Genet. <span class="cit-vol">11</span>: <span class="cit-fpage">e1005048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.7" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prediction of Complex Human Traits Using the Genomic Best Linear Unbiased Predictor.</span> PLoS Genet. <span class="cit-vol">9</span>: <span class="cit-fpage">e1003608</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D9%26rft.spage%253De1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.8" data-doi="10.1534/genetics.115.179978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Do Molecular Markers Inform About Pleiotropy?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.179978</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179978%26rft_id%253Dinfo%253Apmid%252F26205989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAxLzEvMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1429.9" data-doi="10.1534/g3.119.400018"><div class="cit-metadata"><cite>Grueneberg, A., and G. de los Campos, 2019 BGData: A Suite of R—Packages for Analysis of Big Genomic Data G3: (Bethesda) <span class="cit-vol">9</span>: @@.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.119.400018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.119.400018%26rft_id%253Dinfo%253Apmid%252F30894453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjkvNS8xMzc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.10" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-fpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.11" data-doi="10.1038/ejhg.2010.161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haig</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Does Heritability Hide in Epistasis between Linked SNPs?</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ejhg.2010.161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DHaig%26rft.auinit1%253DD.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D123%26rft.epage%253D123%26rft.atitle%253DDoes%2Bheritability%2Bhide%2Bin%2Bepistasis%2Bbetween%2Blinked%2BSNPs%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2010.161%26rft_id%253Dinfo%253Apmid%252F20924408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2010.161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20924408&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1429.12" data-doi="10.1007/s00122-016-2840-x"><div class="cit-metadata"><cite>He, S., J. C. Reif, V. Korzun, R. Bothe, E. Ebmeyer, and Y. Jiang, 2017. “Genome-Wide Mapping and Prediction Suggests Presence of Local Epistasis in a Vast Elite Winter Wheat Populations Adapted to Central Europe.” <em>Theoretical and Applied Genetics</em> 130 (4). Springer Berlin Heidelberg:635–47.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2840-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D130%26rft.spage%253D635%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2840-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2840-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.13" data-doi="10.1038/nature13005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hemani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shakhbazov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westra</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Detection and Replication of Epistasis Influencing Transcription in Humans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">508</span>: <span class="cit-fpage">249</span>–<span class="cit-fpage">253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D508%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13005%26rft_id%253Dinfo%253Apmid%252F24572353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24572353&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333979900045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.14" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and Theory Point to Mainly Additive Genetic Variance for Complex Traits.</span> PLoS Genet. <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.15" data-doi="10.1371/journal.pgen.1006421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-comment">The Genetic Architecture of Quantitative Traits Cannot Be Inferred from Variance Component Analysis. Zhu, X., Editor. PLOS Genet. Public Library of Science, 2016;</span> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D12%26rft.spage%253De1006421%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.16" data-doi="10.1371/journal.pone.0087330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole-Genome Quantitative Trait Locus Mapping Reveals Major Role of Epistasis on Yield of Rice.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e87330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0087330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De87330%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0087330%26rft_id%253Dinfo%253Apmid%252F24489897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0087330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24489897&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.17" data-doi="10.1038/nrg3627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epistasis and Quantitative Traits: Using Model Organisms to Study Gene-Gene Interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3627%26rft_id%253Dinfo%253Apmid%252F24296533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.18" data-doi="10.1038/nature08494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Finding the Missing Heritability of Complex Diseases.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">753</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08494</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D747%26rft.epage%253D753%26rft.atitle%253DFinding%2Bthe%2Bmissing%2Bheritability%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08494%26rft_id%253Dinfo%253Apmid%252F19812666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812666&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270547500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-9.5.1429.19"><div class="cit-metadata unstructured">R Development Core Team, 2012. “R: A Language and Environment for Statistical Computing.” Vienna, Austria. <a href="http://www.r-project.org/">http://www.r-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.20" data-doi="10.2527/jas.2012-5584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strange</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ask</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic Parameters of the Piglet Mortality Traits Stillborn, Weak at Birth, Starvation, Crushing, and Miscellaneous in Crossbred Pigs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">1562</span>–<span class="cit-lpage">1569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2012-5584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2012-5584%26rft_id%253Dinfo%253Apmid%252F23408809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2012-5584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23408809&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.21" data-doi="10.1159/000321967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elston</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Meaning of Interaction.</span> <abbr class="cit-jnl-abbrev">Hum. Hered.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000321967</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bheredity%26rft.stitle%253DHum%2BHered%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D70%26rft.issue%253D4%26rft.spage%253D269%26rft.epage%253D277%26rft.atitle%253DThe%2Bmeaning%2Bof%2Binteraction.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000321967%26rft_id%253Dinfo%253Apmid%252F21150212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000321967&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21150212&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287682600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.22" data-doi="10.1038/nrg3747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Detecting Epistasis in Human Complex Traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">733</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3747</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D722%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3747%26rft_id%253Dinfo%253Apmid%252F25200660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25200660&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.23" data-doi="10.1371/journal.pone.0071203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyenesei</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semple</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Properties of Local Interactions and Their Potential Value in Complementing Genome-Wide Association Studies.</span> PLoS One <span class="cit-vol">8</span>: <span class="cit-fpage">e71203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0071203</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D8%26rft.spage%253De71203%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0071203%26rft_id%253Dinfo%253Apmid%252F23940718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0071203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23940718&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1429.24" data-doi="10.1038/nature13691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuke</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nalls</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandinelli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Another Explanation for Apparent Epistasis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">514</span>: <span class="cit-fpage">E3</span>–<span class="cit-lpage">E5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13691</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D514%26rft.spage%253DE3%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13691%26rft_id%253Dinfo%253Apmid%252F25279928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13691&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25279928&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1429.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1405" title="Deciphering the Genetics of Major End-Use Quality Traits in Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1437" title="Unravelling the Complex Genetics of Karnal Bunt (Tilletia indica) Resistance in Common Wheat (Triticum aestivum) by Genetic Linkage and Genome-Wide Association Analyses" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1429.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gQu0cJKhGcJzTqf3N90GVpzB6rquoV2GZ5nDIYGcBhM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1429" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1429" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1429" class="active" data-icon-position="" data-hide-link-title="0">Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp; Perils of Big Data)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492855" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-i34J3fzmZW9vytCVCGlPBde5-uPnb_kGevH2KPD_9gs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1429.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1429" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-waXxfKmx7n4xdQ4WfQIOelgx3QL5ddKUvI9vAcgdBhw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1429" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1429.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492855" id="node492855--2" data-pisa="ggg;9/5/1429" data-pisa-master="ggg;g3.119.400101" data-apath="/ggg/9/5/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp; Perils of Big Data)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="1">Daniel Alberto Sorensen</span> and <span class="highwire-citation-author" data-delta="2">Miguel Angel Toro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400101 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492855--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Epidemiology & Biostatistics, Statistics & Probability departments, IQ-Institute for Quantitative Health Science and Engineering, Michigan State University, East Lansing, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Alberto Sorensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology and Genetics, Faculty of Science and Technology, Aarhus University, Aarhus, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BAlberto%2BSorensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorensen%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BAlberto%2BSorensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Miguel Angel Toro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Producción Animal, Universidad Politécnica de Madrid, Madrid, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BAngel%2BToro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toro%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiguel%2BAngel%2BToro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1429" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1429&amp;title=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1429&amp;title=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1429&amp;title=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1429&amp;text=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1429&amp;title=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1429&amp;t=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1429&amp;title=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1429&amp;title=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492855/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492855" id="node492855--3" data-pisa="ggg;9/5/1429" data-pisa-master="ggg;g3.119.400101" data-apath="/ggg/9/5/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imperfect Linkage Disequilibrium Generates Phantom Epistasis (&amp; Perils of Big Data)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="1">Daniel Alberto Sorensen</span> and <span class="highwire-citation-author" data-delta="2">Miguel Angel Toro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400101 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492855--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Epidemiology & Biostatistics, Statistics & Probability departments, IQ-Institute for Quantitative Health Science and Engineering, Michigan State University, East Lansing, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Alberto Sorensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology and Genetics, Faculty of Science and Technology, Aarhus University, Aarhus, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BAlberto%2BSorensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorensen%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BAlberto%2BSorensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Miguel Angel Toro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Producción Animal, Universidad Politécnica de Madrid, Madrid, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BAngel%2BToro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toro%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiguel%2BAngel%2BToro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1429&amp;rft.epage=1436&amp;rft.atitle=Imperfect%20Linkage%20Disequilibrium%20Generates%20Phantom%20Epistasis%20%28%26%20Perils%20of%20Big%20Data%29&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=de+los+Campos%2C+Gustavo&amp;rft.au=Sorensen%2C+Daniel+Alberto&amp;rft.au=Toro%2C+Miguel+Angel"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492855/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492855/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492855/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492855/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492855/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492855/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492855/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492855/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492855/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492855/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492855/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492855/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1429" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1429.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1429.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1429.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
