<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1497" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1497.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1497.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1497.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1497.atom" />
<meta name="HW.pisa" content="ggg;9/5/1497" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers" />
<meta name="DC.Identifier" content="10.1534/g3.118.200862" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Lin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="To mine new favorable alleles for tomato breeding, we investigated the feasibility of utilizing Solanum pimpinellifolium as a diverse panel of genome-wide association study through the restriction site-associated DNA sequencing technique. Previous attempts to conduct genome-wide association studies using S. pimpinellifolium were impeded by an inability to correct for population stratification and by lack of high-density markers to address the issue of rapid linkage disequilibrium decay. In the current study, a set of 24,330 SNPs was identified using 99 S. pimpinellifolium accessions from the Tomato Genetic Resource Center. Approximately 84% of Pst I site-associated DNA sequencing regions were located in the euchromatic regions, resulting in the tagging of most SNPs on or near genes. Our genotypic data suggested that S. pimpinellifolium were divided into three single-ancestry subpopulations and four mixed-ancestry subpopulations. Additionally, our SNP genotypic data consistently confirmed the genetic differentiation, achieving a relatively reliable correction of population stratification. Previous studies utilized the 8K tomato SNP array, SolCAP, to investigate the genetic variation of S. pimpinellifolium and we performed a meta-analysis of these genotypes. The result suggested SolCAP array was less appropriate to profile the genetic differentiation of S. pimpinellifolium when more accessions were involved because the samples belonging to the same accession demonstrated different genome patterns. Moreover, as expected, rapid linkage disequilibrium decay was observed in S. pimpinellifolium , especially in euchromatic regions. Approximately two-thirds of the flanking SNP markers did not display linkage disequilibrium based on r2 = 0.1. However, the 18-Kb linkage disequilibrium decay indeed reveals the potential of single-gene resolution in GWAS when markers are saturated." />
<meta name="DC.Contributor" content="Ya-Ping Lin" />
<meta name="DC.Contributor" content="Chu-Yin Liu" />
<meta name="DC.Contributor" content="Kai-Yi Chen" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;To mine new favorable alleles for tomato breeding, we investigated the feasibility of utilizing &lt;i&gt;Solanum pimpinellifolium&lt;/i&gt; as a diverse panel of genome-wide association study through the restriction site-associated DNA sequencing technique. Previous attempts to conduct genome-wide association studies using &lt;i&gt;S. pimpinellifolium&lt;/i&gt; were impeded by an inability to correct for population stratification and by lack of high-density markers to address the issue of rapid linkage disequilibrium decay. In the current study, a set of 24,330 SNPs was identified using 99 &lt;i&gt;S. pimpinellifolium&lt;/i&gt; accessions from the Tomato Genetic Resource Center. Approximately 84% of &lt;i&gt;Pst&lt;/i&gt;I site-associated DNA sequencing regions were located in the euchromatic regions, resulting in the tagging of most SNPs on or near genes. Our genotypic data suggested that &lt;i&gt;S. pimpinellifolium&lt;/i&gt; were divided into three single-ancestry subpopulations and four mixed-ancestry subpopulations. Additionally, our SNP genotypic data consistently confirmed the genetic differentiation, achieving a relatively reliable correction of population stratification. Previous studies utilized the 8K tomato SNP array, SolCAP, to investigate the genetic variation of &lt;i&gt;S. pimpinellifolium&lt;/i&gt; and we performed a meta-analysis of these genotypes. The result suggested SolCAP array was less appropriate to profile the genetic differentiation of &lt;i&gt;S. pimpinellifolium&lt;/i&gt; when more accessions were involved because the samples belonging to the same accession demonstrated different genome patterns. Moreover, as expected, rapid linkage disequilibrium decay was observed in &lt;i&gt;S. pimpinellifolium&lt;/i&gt;, especially in euchromatic regions. Approximately two-thirds of the flanking SNP markers did not display linkage disequilibrium based on &lt;i&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/i&gt; = 0.1. However, the 18-Kb linkage disequilibrium decay indeed reveals the potential of single-gene resolution in GWAS when markers are saturated.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1497" />
<meta name="citation_id" content="9/5/1497" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1497" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1497.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1497.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1497.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200862" />
<meta name="citation_pmid" content="30858236" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1497" />
<meta name="citation_lastpage" content="1505" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ya-Ping Lin" />
<meta name="citation_author_institution" content="Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617" />
<meta name="citation_author" content="Chu-Yin Liu" />
<meta name="citation_author_institution" content="Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617" />
<meta name="citation_author" content="Kai-Yi Chen" />
<meta name="citation_author_institution" content="Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617" />
<meta name="citation_author_email" content="kaiychen@ntu.edu.tw" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6211-0198" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Bauchet;citation_author=S. Grenier;citation_author=N. Samson;citation_author=J. Bonnet;citation_author=L. Grivet;citation_title=Use of modern tomato breeding germplasm for deciphering the genetic control of agronomical traits by Genome Wide Association study.;citation_pages=875-889;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2857-9" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. S. Bhakta;citation_author=V. A. Jones;citation_author=C. E. Vallejos;citation_title=Punctuated distribution of recombination hotspots and demarcation of pericentromeric regions in Phaseolus vulgaris L.;citation_pages=e0116822;citation_volume=10;citation_year=2015;citation_pmid=25629314;citation_doi=10.1371/journal.pone.0116822" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Blanca;citation_author=J. Cañizares;citation_author=L. Cordero;citation_author=L. Pascual;citation_author=M. J. Diez;citation_title=Variation revealed by SNP genotyping and morphology provides insight into the origin of the tomato.;citation_pages=e48198-e48198;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23118951;citation_doi=10.1371/journal.pone.0048198" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Blanca;citation_author=J. Montero-Pau;citation_author=C. Sauvage;citation_author=G. Bauchet;citation_author=E. Illa;citation_title=Genomic variation in tomato, from wild ancestors to contemporary breeding accessions.;citation_pages=257;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1444-1" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Brachi;citation_author=G. P. Morris;citation_author=J. O. Borevitz;citation_title=Genome-wide association studies in plants: the missing heritability is in the field.;citation_pages=232-232;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=22035733;citation_doi=10.1186/gb-2011-12-10-232" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: Software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. L. Caicedo;citation_author=B. A. Schaal;citation_title=Population structure and phylogeography of Solanum pimpinellifolium inferred from a nuclear gene.;citation_pages=1871-1882;citation_volume=13;citation_year=2004;citation_issue=7;citation_pmid=15189210;citation_doi=10.1111/j.1365-294X.2004.02191.x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Capel;citation_author=A. Fernández;citation_author=J. Manuel;citation_author=V. Lima;citation_author=S. Francesc;citation_title=Wide-genome QTL mapping of fruit quality traits in a tomato RIL population derived from the wild-relative species Solanum pimpinellifolium L.;citation_pages=2019-2035;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-015-2563-4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: An analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. L. Chen;citation_author=C. Y. Liu;citation_author=C. H. Chen;citation_author=J. F. Wang;citation_author=Y. C. Liao;citation_title=Reassessment of QTLs for late blight resistance in the tomato accession L3708 using a restriction site associated DNA (RAD) linkage map and highly aggressive isolates of Phytophthora infestans.;citation_pages=e96417;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0096417" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J. W. Davey;citation_author=M. L. Blaxter;citation_title=RADseq: Next-generation population genetics.;citation_volume=9;citation_year=2010;citation_pmid=21266344;citation_doi=10.1093/bfgp/elq031" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=A. P. Dobritsa;citation_author=S. V. Dobritsa;citation_title=DNA protection with the DNA methylase M . BbvI from Bacillus brevis var. GB against cleavage by the restriction endonucleases PstI and PvuII.;citation_pages=105-112;citation_volume=10;citation_year=1980;citation_issue=2;citation_pmid=6248417;citation_doi=10.1016/0378-1119(80)90128-6" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=P. D. Etter;citation_author=S. Bassham;citation_author=P. A. Hohenlohe;citation_author=E. A. Johnson;citation_author=W. A. Cresko;citation_title=SNP discovery and genotyping for evolutionary genetics using RAD sequencing.;citation_pages=157-178;citation_volume=772;citation_year=2011;citation_pmid=22065437;citation_doi=10.1007/978-1-61779-228-1_9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Excoffier;citation_author=P. E. Smouse;citation_author=J. M. Quattro;citation_title=Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data.;citation_pages=479-491;citation_volume=131;citation_year=1992;citation_issue=2;citation_pmid=1644282;citation_doi=10.1007/s00424-009-0730-7" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. Fernandez-Pozo;citation_author=N. Menda;citation_author=J. D. Edwards;citation_author=S. Saha;citation_author=I. Y. Tecle;citation_title=The Sol Genomics Network (SGN)-from genotype to phenotype to breeding.;citation_volume=43;citation_year=2015;citation_pmid=25428362;citation_doi=10.1093/nar/gku1195" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=T. M. Fulton;citation_author=J. Chunwongse;citation_author=S. D. Tanksley;citation_title=Microprep protocol for extraction of DNA from tomato and other herbaceous plants.;citation_pages=207-209;citation_volume=13;citation_year=1995;citation_doi=10.1007/BF02670897" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=T. R. Gaunt;citation_author=S. Rodríguez;citation_author=I. N. Day;citation_title=Cubic exact solutions for the estimation of pairwise haplotype frequencies: implications for linkage disequilibrium analyses and a web tool 'CubeX'.;citation_pages=428-428;citation_volume=8;citation_year=2007;citation_pmid=17980034;citation_doi=10.1186/1471-2105-8-428" />
<meta name="citation_reference" content="citation_author=J. Goudet;citation_author=T. Jombart;citation_author=I. N. Day;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Wild Crop Relatives: Genomic and Breeding Resources Vegetables;citation_author=S. Grandillo;citation_author=R. Chetelat;citation_author=S. Knapp;citation_author=D. Spooner;citation_author=I. Peralta;citation_author=C. Kole;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=P. Kaur;citation_author=K. Gaikwad;citation_title=From genomes to gene-omes: exome sequencing concept and applications in crop improvement.;citation_pages=2164;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.02164" />
<meta name="citation_reference" content="citation_journal_title=Meteorol. Z. (Berl.);citation_author=M. Kottek;citation_author=J. Grieser;citation_author=C. Beck;citation_author=B. Rudolf;citation_author=F. Rubel;citation_title=World map of the Köppen-Geiger climate classification updated.;citation_pages=259-263;citation_volume=15;citation_year=2006;citation_doi=10.1127/0941-2948/2006/0130" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. G. Meirmans;citation_title=The trouble with isolation by distance.;citation_pages=2839-2846;citation_volume=21;citation_year=2012;citation_issue=12;citation_pmid=22574758;citation_doi=10.1111/j.1365-294X.2012.05578.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=L. C. Moyle;citation_title=Ecological and evolutionary genomics in the wild tomatoes (Solanum sect. Lycopersicon).;citation_pages=2995-3013;citation_volume=62;citation_year=2008;citation_issue=12;citation_pmid=18752600;citation_doi=10.1111/j.1558-5646.2008.00487.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=L. W. Pembleton;citation_author=N. O. I. Cogan;citation_author=J. W. Forster;citation_title=StAMPP: An R package for calculation of genetic differentiation and structure of mixed-ploidy level populations.;citation_pages=946-952;citation_volume=13;citation_year=2013;citation_pmid=23738873;citation_doi=10.1111/1755-0998.12129" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. Ranc;citation_author=S. Munos;citation_author=J. Xu;citation_author=M. C. Le Paslier;citation_author=A. Chauveau;citation_title=Genome-wide association mapping in tomato (Solanum lycopersicum) is possible using genome admixture of Solanum lycopersicum var. cerasiforme.;citation_pages=853-864;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.112.002667" />
<meta name="citation_reference" content="citation_journal_title=Plant Genet. Resour.;citation_author=E. S. Rao;citation_author=P. Kadirvel;citation_author=R. C. Symonds;citation_author=S. Geethanjali;citation_author=A. W. Ebert;citation_title=Using SSR markers to map genetic diversity and population structure of Solanum pimpinellifolium for development of a core collection.;citation_pages=38-48;citation_volume=10;citation_year=2012;citation_doi=10.1017/S1479262111000955" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. S. Rao;citation_author=P. Kadirvel;citation_author=R. C. Symonds;citation_author=S. Geethanjali;citation_author=R. N. Thontadarya;citation_title=Variations in DREB1A and VP1.1 genes show association with salt tolerance traits in wild tomato (Solanum pimpinellifolium).;citation_pages=e0132535;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0132535" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. L. Remington;citation_author=J. M. Thornsberry;citation_author=Y. Matsuoka;citation_author=L. M. Wilson;citation_author=S. R. Whitt;citation_title=Structure of linkage disequilibrium and phenotypic associations in the maize genome.;citation_volume=98;citation_year=2001;citation_pmid=11562485;citation_doi=10.1073/pnas.201394398" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=C. M. Rick;citation_author=J. F. Fobes;citation_author=M. Holle;citation_title=Genetic variation in Lycopersicon pimpinellifolium: Evidence of evolutionary change in mating systems.;citation_pages=139-170;citation_volume=127;citation_year=1977;citation_doi=10.1007/BF00984147" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=V. Ruggieri;citation_author=I. Anzar;citation_author=A. Paytuvi;citation_author=R. Calafiore;citation_author=R. A. Cigliano;citation_title=Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP.;citation_pages=81-91;citation_volume=24;citation_year=2017;citation_doi=10.1093/dnares/dsw050" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=K. Shirasawa;citation_author=H. Hirakawa;citation_author=S. Isobe;citation_title=Analytical work flow of double-digest restriction site-associated DNA sequencing based on empirical and in silico optimization in tomato.;citation_volume=23;citation_year=2016;citation_pmid=26932983;citation_doi=10.1093/dnares/dsw004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. C. Sim;citation_author=A. van Deynze;citation_author=K. Stoffel;citation_author=D. S. Douches;citation_author=D. Zarka;citation_title=High-density SNP genotyping of tomato (Solanum lycopersicum L.) reveals patterns of genetic variation due to breeding.;citation_pages=e45520-e45520;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23029069;citation_doi=10.1371/journal.pone.0045520" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. C. Sim;citation_author=G. Durstewitz;citation_author=J. Plieske;citation_author=R. Wieseke;citation_author=M. W. Ganal;citation_title=Development of a large SNP genotyping array and generation of high-density genetic maps in tomato.;citation_pages=e40563-e40563;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22802968;citation_doi=10.1371/journal.pone.0040563" />
<meta name="citation_reference" content="citation_title=Genetic Diversity in Plants;citation_title=Association mapping in plant genomes;citation_author=B. J. Soto-Cerda;citation_author=S. Cloutier;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Taxon;citation_author=D. M. Spooner;citation_author=I. E. Peralta;citation_author=S. Knapp;citation_title=Comparison of AFLPs with other markers for phylogenetic inference in wild tomatoes [Solanum L. section Lycopersicon (Mill.) Wettst.].;citation_pages=43-61;citation_volume=54;citation_year=2005;citation_doi=10.2307/25065301" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. D. Tanksley;citation_author=S. R. Mccouch;citation_title=Seed Banks and Molecular Maps: Unlocking Genetic Potential from the Wild;citation_pages=1063-1066;citation_volume=277;citation_year=1997;citation_issue=5329;citation_pmid=9262467;citation_doi=10.1126/science.277.5329.1063" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Víquez-Zamora;citation_author=M. Caro;citation_author=R. Finkers;citation_author=Y. Tikunov;citation_author=A. Bovy;citation_title=Mapping in the era of sequencing : high density genotyping and its application for mapping TYLCV resistance in Solanum pimpinellifolium.;citation_pages=1152;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2164-15-1152" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.1111/j.1558-5646.1984.tb05657.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Wright;citation_title=ISOLATION BY DISTANCE;citation_pages=114-138;citation_volume=28;citation_year=1943;citation_issue=2;citation_pmid=17247074" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=S. Zhong;citation_author=Z. Fei;citation_author=Y. R. Chen;citation_author=Y. Zheng;citation_author=M. Huang;citation_title=Single-base resolution methylomes of tomato fruit development reveal epigenome modifications associated with ripening.;citation_pages=154-159;citation_volume=31;citation_year=2013;citation_pmid=23354102;citation_doi=10.1038/nbt.2462" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=E. Zuriaga;citation_author=J. M. Blanca;citation_author=L. Cordero;citation_author=A. Sifres;citation_author=W. G. Blas-Cerdán;citation_title=Genetic and bioclimatic variation in Solanum pimpinellifolium.;citation_pages=39-51;citation_volume=56;citation_year=2009;citation_doi=10.1007/s10722-008-9340-z" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="To mine new favorable alleles for tomato breeding, we investigated the feasibility of utilizing Solanum pimpinellifolium as a diverse panel of genome-wide association study through the restriction site-associated DNA sequencing technique. Previous attempts to conduct genome-wide association studies using S. pimpinellifolium were impeded by an inability to correct for population stratification and by lack of high-density markers to address the issue of rapid linkage disequilibrium decay. In the current study, a set of 24,330 SNPs was identified using 99 S. pimpinellifolium accessions from the Tomato Genetic Resource Center. Approximately 84% of Pst I site-associated DNA sequencing regions were located in the euchromatic regions, resulting in the tagging of most SNPs on or near genes. Our genotypic data suggested that S. pimpinellifolium were divided into three single-ancestry subpopulations and four mixed-ancestry subpopulations. Additionally, our SNP genotypic data consistently confirmed the genetic differentiation, achieving a relatively reliable correction of population stratification. Previous studies utilized the 8K tomato SNP array, SolCAP, to investigate the genetic variation of S. pimpinellifolium and we performed a meta-analysis of these genotypes. The result suggested SolCAP array was less appropriate to profile the genetic differentiation of S. pimpinellifolium when more accessions were involved because the samples belonging to the same accession demonstrated different genome patterns. Moreover, as expected, rapid linkage disequilibrium decay was observed in S. pimpinellifolium , especially in euchromatic regions. Approximately two-thirds of the flanking SNP markers did not display linkage disequilibrium based on r2 = 0.1. However, the 18-Kb linkage disequilibrium decay indeed reveals the potential of single-gene resolution in GWAS when markers are saturated." />
<meta name="og-title" property="og:title" content="Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1497" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="To mine new favorable alleles for tomato breeding, we investigated the feasibility of utilizing Solanum pimpinellifolium as a diverse panel of genome-wide association study through the restriction site-associated DNA sequencing technique. Previous attempts to conduct genome-wide association studies using S. pimpinellifolium were impeded by an inability to correct for population stratification and by lack of high-density markers to address the issue of rapid linkage disequilibrium decay. In the current study, a set of 24,330 SNPs was identified using 99 S. pimpinellifolium accessions from the Tomato Genetic Resource Center. Approximately 84% of Pst I site-associated DNA sequencing regions were located in the euchromatic regions, resulting in the tagging of most SNPs on or near genes. Our genotypic data suggested that S. pimpinellifolium were divided into three single-ancestry subpopulations and four mixed-ancestry subpopulations. Additionally, our SNP genotypic data consistently confirmed the genetic differentiation, achieving a relatively reliable correction of population stratification. Previous studies utilized the 8K tomato SNP array, SolCAP, to investigate the genetic variation of S. pimpinellifolium and we performed a meta-analysis of these genotypes. The result suggested SolCAP array was less appropriate to profile the genetic differentiation of S. pimpinellifolium when more accessions were involved because the samples belonging to the same accession demonstrated different genome patterns. Moreover, as expected, rapid linkage disequilibrium decay was observed in S. pimpinellifolium , especially in euchromatic regions. Approximately two-thirds of the flanking SNP markers did not display linkage disequilibrium based on r2 = 0.1. However, the 18-Kb linkage disequilibrium decay indeed reveals the potential of single-gene resolution in GWAS when markers are saturated." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"JKlSxbnXsgKdLYhZEAd7ox3PRDffrmURGOb6hKFkO3I","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493102","apath":"\/ggg\/9\/5\/1497.atom","pisa":"ggg;9\/5\/1497","ac":{"\/ggg\/9\/5\/1497.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1497.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1497"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1497"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1497":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Dr7xXJNscyAqRsWzDuxLfknvZWJbHqXCWg1MND8VbT26K3VL7Z2QItt9UoC+H2YmZpLPV7S0wavPOLYGiPg2oYdEvNbwXJuwrS5+9zmqxbYQyBXlBrnSSCKkCV0KRZ11moTM9+3WFVX20TKIajndeTmZnrp2hfDtYWY9tIwd89IjI7FRNHig0gsZu9YtsZRc+\\\\\\\/YGQ\\\\\\\/MnbeiYSh\\\\\\\/udOs6SksXNx7SZ39AAbJqh2tM0Jqqw7ar4NI\\\\\\\/DMp6CAy7o2KUfifMtjxaUcaJ8MnP4jixFD102fPnupSdDtxj6hxBQLUpQ2\\\\\\\/YGZUw6YRDZyiTphdIW+sNWzU34w\\\\\\\/CkklssBuW5Yndyg36CGPmeBkOG3YAfMImpx7qeipSX1uc9qM4gSixabDJd7jgQ5T3STyurr9NR9xdryt4RVWo1H6hWlODtMEXzk7K5\\\\\\\/+CYsDZ9euPnHPOra78\\\\\\\/RKpyAbuBTn5\\\\\\\/9vxVXgdyqUCkVhwQl\\\\\\\/zsdb7H1YK2XnbPsJewiMU\\\\\\\/F0IHZ\\\\\\\/WyblBqOie8qSkQ+hrpz5STw0SRXjhFxQTUBKHSku8QE4pNRrwO4BvnNvoWbYO\\\\\\\/gMyqPCyfSIrY5H9r0Rq6L3hX2Y1sw1zw6x+iFYitGJeoHXcAFbqjqTIuwmRW6n+0gZwdE0NviMkvuxK7CXxfEa19xVo4zbpYURNm3ZBtTVMHU2n\\\\\\\/mJ03mdZuAFgXBPu8k8ys4Pcbb5m+h3NmyJtHpnIOVz6k1Oo6zL2KuupADUIbQ9ASMgOcW2Xu8LTnqMMCp5hPv4UFSjgUuHDeFzCwHGM\\\\\\\/aMoD1zBq\\\\\\\/H4P3thK1zK9W0uyTVg8dRwk+LfFGWa0QD1MfK\\\\\\\/fMeOoOLUwYB\\\\\\\/3pexy+S5W2JSKVq9USzQLQLTdHk14Ptf6m7gBi+5\\\\\\\/tFf2VobL1qUvjfxmqGgIXnx65qGZ\\\\\\\/g5OnthA5yKER\\\\\\\/zRaXkQNKAPVDLz68EGJt3MSYJ9J4aDERS8ko6ZOkb8SDk7\\\\\\\/e6eve3kr+9kDn6i2mSR+h6v02Oz4HusadeBARuESY98rbsXYYmieEBjqAssaa0olTm5CCGaFSd0NBSr3fL2unIVvzEKWBeF+Uc1YCS7W9F19hhj3eJl8G8Xh\\\\\\\/oBDx9AgFiAFEe6tAPn5N8l+VoGtXU6vILA5dl8flkdDiXnPXTQtFE8zEHMRmn9znK1R68UYDc+BCB3ePEP3ur5j6JxaqOpsWspkSQoR2XZDBEmwFEOuc+7Ymu8EQxgjBkmeNcLDPF22r5NvLkaZqDt\\\\\\\/04nGyHo9ZD13Kgao762ZnNk1OSlF8Z\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ozs13P3ykcPh1fJLTwrVTA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002251d3a55a63188c369b4aa2d2242fa8e9b7a374afcfe336294cf93edf83d37a18\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Wa2t0Fbv7fujUlqmARtgukJHbRJzYeFemmF\\\\\\\/lWyoMvvk4T9dwFVdfdwHm0sWTxOSENhaf9YHKbP7iNKwuuTrTYRwTHD+0P9Qy+P5\\\\\\\/mfzVZcrCmB2vqryOnDNHTeAwTbc2PzCf+r5vkTp5p+SxoalA0edM5J222Fz0MwDVGf2E0\\\\\\\/xUstd6FSXWe5elt7K5HktwHuoJB8PxUcto1AWv2mDOgCgWFwlyFBLk+bxy3nb+rM47SiuGE++J1NlWHW\\\\\\\/2uAqscpclmCB5+tdzKJmDekaBwDa9JgndFF8qGbdvsF+qxsVGG6YqZa6vNuHoEmoJRHGEqBXt+CPwUvKIHEe4G62dTIiT1pEnbe4g5LnfWMZZ5hehg+S3pViFoQ9U2M2iQiML4lq8WEWPr\\\\\\\/tvESytr4Nv1GiMEwX3spEokhSOm9+MqTzjjXAs2yv8nNxJxXn4DwAgWERFc3jn8S5hxcs9L\\\\\\\/kdXuIiG6uMVGjIjHHjR5vBFvQ++Qdm6z03Na3b6vCiPuNLxKlV9ag3vmE6WSeAKhUitg7\\\\\\\/\\\\\\\/+kHjPe\\\\\\\/pQnEUcQLhNLr\\\\\\\/4MYNTMoSS8CvX8g1NgZ\\\\\\\/9mG3H\\\\\\\/SJubp1gY0UW6smdfScJ9\\\\\\\/8eNKU4Wv0flUOVFTX3Y3Up9g0rHEF8CHyB3kGqUGmNIj0h51fKwnb11oAYT8arzUbCMDk1NxQIu+2c0\\\\\\\/n\\\\\\\/pdmyO2d3ddJD1igalZ0xJbngeiJjYqnp+TZcCo\\\\\\\/F4STkwl7yHNl+VGu\\\\\\\/wq9k3Pv5WQXzvX8V0lZvqxpCFljGcuuGU4Vk7qaETvBvGsQ36ujrz4orW4SUTX41x2885OZk+OUiZ3yhzdtVEn9gBOyFFb3rq1cMQWPRdoc0K\\\\\\\/GECUR4xU9Pb7+DDtuxDDuO+4NWaVAv+WdLo+nv16IuMMzKjUnuA5YgOxE0DmxCeuLGqpx1xnIjLMaQzoCAOVmfT5GYsEzXZUvacHuF3qLmgb337vJAKM\\\\\\\/jT37tGDYkMwOJYEafeE6vdQcmkp8IhloqnvBsmoXmFVM1KZUVPhYSKSV41LQKbcPIBQr\\\\\\\/5Zdp5NcH0hmnTVWyT7jEIrqvzGBZHCLZL9AGFiUzVgSuxgquYSc0NP372otZjVgs6b7W4j2TkKPwWVYZkBNlwmbZlYHZWREPpZFtsMILG6XqdirDtgO2i5aB7gjao+Y1hco\\\\\\\/dfn\\\\\\\/1JhWjxrzNzF3dcV0PsAMhxEjeO3ykkUiXBC5aTt3+\\\\\\\/OZ5j5ZFNwigHzMP70TmTlWI6scgMxewk7LpnFn3oeYAZlY1DNdwYlGzTdkDJUboeDVdrxyg+1jwgApxsz7zypb7frNzLby7p+vr\\\\\\\/x4c0yUzeprOljs3T4XsVrnChmM+pPZIDk0fVdoGAY3us7rHCM9dxVgxpLXkbpihDi4R55vtkGUDrmDxdght7dK7xnVDkX9wOcCTE2Pw0jWibWSmBqfRLvCckB8m4NEqMRrgdBCCnNtFqBb2i+24R\\\\\\\/nks8zoAKPingL2Aq3e\\\\\\\/0k3Bp3Z83QCNvEuJK+OnOLNMQWzK2UAGPEqzad3H7XDEPkidQP7cVzxca4mzpcd0D0zkwWSlxbPORPkEwZzu5OEejU1WsUeGSAqzFEJhEgW4op3Kg4p58Ns6SmEtt6za4LwTDCzUJwHtuQ4G9oOzWX5qioTVTnOwANhN8oGBvX6k7eYOr5l1G1PNo8B2lavjRQ05EzU0j2zwaPWJZga7FN605K9xFxUri3O\\\\\\\/nkoNSitrbZ1qE8u7fSAwnXf+odG1HkTMAxvGJmx1cZ+\\\\\\\/9Rtp6Dw3safzLCnrTG7Z49i+Jmq0PNZ0t40TV7b+\\\\\\\/6sSkGxoBoDC7bAjqam9AbUW3QACQfsGVe4XFrnFtqSp5zLA9UeK2\\\\\\\/TP2tQDdMynVtdn+QoWsKQfw1MV96k6dCZXNeAK4zCzI9+0zTfAbb5SIKTnPwmDkc5v3+WZfsjMxatIF8++othTY6c8bgm3K0kmGlhykVghfuv53rrmABsyFW4xZs+3SZe15TALv10KAg\\\\\\\/lXfttgzoKNHCK3mQnkWK8QPK73EY1ARe9eukuemxfEJJ2a0qiJWSziHe3P487RKVMtHG+aPe7ZsYB\\\\\\\/f2lDxpAWGG0X4L0TObSOn4JcALvmgWSez9C8ChA8fIsZwnO0VtHy9J5A2+BxnfnGhpyO2q2+eY3wB6HOdcyZ6SQkzIFrlxQzLrx7YzyMxAAx92aAEGjJz9L5qYLCWAQ1o+VTt8ogg\\\\\\\/eGK9QUMBzV7MYYxlPEKWXO569pJi69riaBldjxRcpT\\\\\\\/CNKiTANRv2wuB3jXvyBHpskmUkke2ocpnH3w9d0GIBCALOoCifk\\\\\\\/ZSCgPDyDsJEB25EsRPDwQI5ULIKP\\\\\\\/V8STRUkNsK0mv9+kmR1yJN5We5POx\\\\\\\/LPwdeMPlrd8EFOyzwksjMGAdIdEnpUEGVuvXw2ZnAfvEhtTM20ij+MU6Sa2+07qhu6D1qXgw4o8Uy3+NVknxOcMx4XQI9w3HNVrtbrdhIIX1StoLy3Ss07xL0sbUzc7P3M2+ZpviUglvr9h569Y3xnPqJWEAFx4b6saee7vHnyi3iUXI0jOFobLGAGYZ5bDJ3Rlf+QIIxALNJuQYdeM6MqX4tEcpiJinc05PubY+2aK6tTUF0Uo1LCn8jOjlzxi7qCkRKr9blLOJQHnvjV9hj2SmMpw2eA1\\\\\\\/z62Gi8I94Qqr0\\\\\\\/BsDeFMpje2mJ3DdXTq8IWTMuf93+cnKGWrwxoqf4BwQ+j10aCszrBBjk4EK38oPOE8dkfYotwIeX\\\\\\\/lLj20r6XNaq+otJZ5zvVeB0ghwqH+YtFjFk3A4dgT7EI\\\\\\\/8DKoqvAOsIe0w8PNXctDyENFB925QXbP3GrQfaMTITBRe\\\\\\\/QNjjv6pRYHRodUYQ972fxbdmjPCwZStZa5wskOpRk+f9bu6xSARjZQKL0tnopm3m6CnS8knUbAV7cM0\\\\\\\/ziRPjag6BJNE82CyTO+WAl4mifJpPBrkKP8ibZT4PRcfr930OzTCUhSmIJ0qSgGTfiPfvVoGP8o6\\\\\\\/fAXk3k1QiUaES2WP0c6BJsTWn+CGYKyrjDCNRWWPLkYy3X47Y\\\\\\\/FvzHDYIJEo3Wv0l1H86\\\\\\\/FtM9GyA+N4NkfrO3cE8spxQFqjZN2JAFK7FHsCohG9k6RzMI68Xe3J9KbDfwzm0zSiXwXUVb2As5YGzVp8mWBa4Mh3VLu8PPxFDoWat9xYgW16U\\\\\\\/J\\\\\\\/QRCC7bGxHDV7IzDsc1qYvoDVwnmQxHSGB2UmhulNwI1UQrO2ddBZBW5pV3Nxk3LGI2Yc8G82yUC9xDpGla5X\\\\\\\/eLKgLJ0p7rRHbAm0T\\\\\\\/VKCdnDwREb1rBnvJpwnw1kgf7na1BgQBcGQuasvHbVIAWUwrB6gTPQYw7IcNUdYEfMxrRvu1shC00O1W4\\\\\\\/cwJ9\\\\\\\/rOIhdWDny0XA9MYZnJwVBVw1aFkco7SB5bii84GeDyzixZKr7LHY\\\\\\\/hcY\\\\\\\/mkAa389d79b7OEEMDudTWB8Q2oGue8DIFlXmM74FjfvfKFBhW9Rru+zWN1Mi4FtEdP+Eneo0r1iVljlZ30n5nexwsLSVzdoAIgCTdPxHqgCLCjH97UAfxAPqCCNzqQB2azF3AIUSxRylThgTOX\\\\\\\/9jN5kMondUsinvxI3kksVDGL1ztPbdEESek2x5bc324\\\\\\\/+eicp4WbC1X\\\\\\\/nG9UjW+5BwaVKZggq8jbe4YRSUruM5RN9FsKbDPcc\\\\\\\/VUXb9Q3o\\\\\\\/69pGLTT+OuWjdjyesyYs5TL8JU7SvATPmGCQ+9w5zwauZWrIX6+1sbLD3tDcvB7vou12Olk91kB4HBWvXDSJOpswnNrTVWIrb408wbQQZbrSIQHyZ7T+aXGRHzj8iQ\\\\\\\/VpsIlddHLwvn8RIp7L5eojDwgKsipkf6jZHEIuljSyRrKFuCBpPqybEuCg6QTGlctqPHMhzpPsudq\\\\\\\/RuzygP\\\\\\\/EcDZciSm\\\\\\\/uumTiUF4RGpL6eV21faIc5NGNT7e\\\\\\\/HGMXlJgi4Kt0El6slDbcnAoy3KrWvWE2n\\\\\\\/Zz0qA+u7c7e0V5SM65wyTsX3cgZsBnSCcNba6guQQDt4KAOQQBjQrR4TaDpHH524evM4bSI+9zrdO0ilQOKf6HMaTolBStb4fhcoYc08GFlLslQmxYg6ThctzJXi3bMDvdc6MtUIk6bzgTHl4yRVpR90nwYL\\\\\\\/N6M38d8KsCAdwBqaw4f3AFGcQm3bKaCYRSwEGQc1SV+bHX5SosCpdijSQuYoxwyLhtzgylqWbTRfN9gLkmCmGUzp8XCpNDKMez92b5or1dr0KHKDqiTI4L3de5Ql9VlGdQ2sDAx1cQlo96yXg2ljr7c7PqJBurvQLUIKoukpN1xZF9a59gWfoZSJaPwAvqoNAEmtn4muGWKkLOZQRFoiD2zhlIccmWv6zPpa25tw6TUAU8058\\\\\\\/gX2CW9PWl1aGGpf0lhjDPNcKKQ\\\\\\\/kyAbJOhHjllW\\\\\\\/j20qUkRjjLyIXY4\\\\\\\/XB7RIQZZcZ8n7\\\\\\\/OZTFU8f4B+j4r9Ro6w5uSaJIYd20OahSXq3qnThtVQdRQRGaEqMeMZItzekq8UprwV1nXS8gX2Jhcw\\\\\\\/tmMJxxsSnBlAOkdOyH3M3oy7GNH7WuWyvN2mgEnTNu7xkOuckLcmOThv7k2FSpof8sd4zE3m5\\\\\\\/SX5Nec47u1hFxu094j3ofjUvjgsgv8cCQybZajJulc7dWhQHiQbBtbBM+7Qd5wrRYf1XiUjMcPFDxFd1FxTUEOY1kr91S776da8QND3xdYA\\\\\\\/GMIDh63ESP8CYi6j1B9aPtCXlBpC0Jd9VDOfzqzUOaHvJWgsTG+pjwyqQ3rRHTIfmEfMOxxqpPjgMn20YcNfyuohEnSyUzONgaRGM57mOuAiPkpTJeHa6VJESRc6\\\\\\\/okdmrHUO52ceIdIdiWmCFpicr5G0tA1CBhjKwWlSTnRt+B3OjLO9MPnGDvxLjJMPwpQyO2oKQqoDW398Vxx4b7Lq4FfH4D5sErLiJJvSZ4A44h2w19hv0ioJhsBZ\\\\\\\/+J6I4c+jjEh6Mv8ydXfsiPbtAHaF58edSkzwPp7OH1acDxqBaTHr6Nk1gGSzUFZeGlSQmwdFylwLWbzrH9+1jRH3GONYzNnbDU164br2mLZO472KiX4VJVIiotHjvUNDZnu4hdB801meX2D17V8pWc1oRjpqOOxlZ+7S66Fs\\\\\\\/U3O1SKu031zTB3GxVA9S9cqkRQKI1EvJSn0UcMeHml\\\\\\\/6DoXuk4dNKui0gYJ61t8QMd60CHNd3oWbQHjYc13OCXkNoWOqBl5yxjON8fBqOqY5Z2FdN\\\\\\\/+QdD4S7Ibqh+02JdxHt\\\\\\\/1QKc7rojY0c4A3QkQfnmlpeonyPMNRc5lSS6lAu7zVwdrZNyenBf73pSUzY6fkFlOD9ISqHTODAguGIjUoqL1SmFL1wrMgVHntsgCq0Ki1pQeN5VqXV295iPYibAftBPmNLz8aMQMZytUffhD7MCVNXI55aszCWMXZNsse3vSWn\\\\\\\/QTUkVx7TbghHFAIaa6b7EvQ6Wshx3hDwFBNWMyrcbFW6LqcE+QJ6I5QOL1fT+3WoFKBXnlfzZhznZta7x8JXH4IEwGADXt1N3ztBlPJvO0RBfFZSigOZuJtXOJqXZzylrFrq+61Qbi4XTBCnJEy710Ph9ptqne+QrChrnAETVXSHsWN2roCNjZt99LuaOPuF6DHkh4++Mqk9ZgiNMbE7cX4PMZA31izXq3OmShBS4XGSpDgI3SiS4G0G21sW1cdvRmfM046QTHk2mu13LZzYhzOIvvEdkACCYcRJvIzF9gl14l+xqShVFdUwfs5VO3tzCqVvmVjBMfAx4uvce\\\\\\\/+LL3l355voHHc5VN4Va2ERF60jUFgDfg7hKleBiQuJbCKwdFo74QkJfZ0vZ5V4VQDeYRsx1k3+fnR3NHOjHk\\\\\\\/CfFL+8rN5\\\\\\\/mlKcbnqUBrc3FUJCwqyAIVqN6UGSVlUDOsYl3hPoGsBPgmjZbsFv5ifD1zpdAlkCuiEDSPmMCXYvRYEFtxC2lZRVGwxi7j2rcxArxbP0b1X+NmPPcrQ+uhWacuQzyQYIuI+MPO5aWLDDAxMpu7j7OZlotbGX8KLOo932ezz01LDVpJYoIVMOFlrdBv+Oxfv7HTXHkoC+xvnZqg9s7eI8WIyheh6ovqvhfemZYV6VyLdctiY5ZzDe5OHJi0KV7C160QCVFSm+fq41R02vzQ3wqBl5wH8EgqESwL4y5AZvFeVxp\\\\\\\/ELEAB7ZU1cK+izgrBljZKcyGQjn3tEY1RNAnXujhqeknZt45\\\\\\\/GnFYDmpqVHemYXDF56ecma4Ju34b2uTHg2ABGAAJrQ\\\\\\\/kBWAXQ91JLst2fggRwx69F8tg1B4MfC11Pz0+gVXhbad2FKTLou2IUQA28nO8GKwtZy\\\\\\\/skoufpK8Gz1e4ha5TyxKhGcvLkOHVvKqiBPI3I92gVfQJ54jO5L32wBeXT2wptiz6aNiOSH00Qq\\\\\\\/Lhz7JpgJA67L3YqvYwvkxq1tYvBLZ3jVazAAqkaS0xx6DSgI5vBS1wrwAOZNein4WI6dqhx+QpGT6L+4qN1rw+cIOjQa3nhp3j8Tkn4RBOQySzZQdXgab4DbdofTfoagVUSC0Pk9aAjOXpI5PX8PMu8ecPY251P6VfK3bmfUL1kh5vhOAA0eXLJEjI2UdM7A+Usl6OjyLmxBmF7B8ZWCd4q1EfOSMf0WI3ZuWTzVcpAGnqK2gzqofk\\\\\\\/kVBq9YIclLT5m+b59StLceOA09hXh1bavbkP9fQOXNES\\\\\\\/DmKM04yZbT38DtRheX4RZZ9AwOHPBqy5T45sicVC\\\\\\\/zFUQPGvDLbeUvCMy8dA94PnwmtNJF1l3tRFmCYoVpZu5r915rgUrZ6EGHdhzaDnQI4eDBKVAI4VIellwCRLa\\\\\\\/kpISTVL\\\\\\\/eVVl3R8Mgkh4oltD7PaSgFH7FeTVUhxz9qZMaYCZ9bGscG3OnPKfTdYAL\\\\\\\/pb1SIBXIE1uq+JtFBaLar\\\\\\\/\\\\\\\/pGUFFXx\\\\\\\/qAr4012Sc3jZWNrJMZO59IgK8WYM2+0Qqm+WkMgpxWKjo9jcB3g0HZSm4At3Iv8eqEGA3vFHWSsKMxs8laS1Is2A6YU3V5eBWde3\\\\\\\/H3xVKZgUk8QqrTHqiozI5wyeqx0uECI4k6VIVtootI6VOh0ln3U0BS57AIWm6ytshZGJVQAXypuptkcRi7\\\\\\\/k1NwcPPbcKzVjrsN2gORisEDfb27wTzGr6TrgTU\\\\\\\/kolyYaJjDp+o9YoiLRYrym42ZVBVP5HADkhhK3DGU4uRE0nQbFXXl8fqumW3JtzXCMIHEidsXt2yhz75y3YB8QXMNKjNcotRaBoii6i\\\\\\\/V0B\\\\\\\/32KHftmHzzEnhOAzkGfG+DSqFRzIE007kT1xnishr0n7U2FU4wOKKx2uk6fVB0Pnn3s0v4kMT2yZbSwugjMO4m\\\\\\\/RGdKv7ahzFzIsCZZy3tzkkILwzEmsaRgqUXEfSQCpWaF1\\\\\\\/MScfvltPIvlAu7acdaLBrhwwTNrmfsFk1P6siUbWPoiTAnlufyJ7zCJRMnpoxh7J\\\\\\\/SEOVA9LTGPAuqos6FEUhfxkU38U5lFgU7p4mn2Fyu2JPAo+W8sRxsTkQudUd\\\\\\\/OJwoQ5WlH3bfp16K\\\\\\\/3VaTfyT4tdFM0iuAELfy4UotMw7nIOl+xfDiORDXtOPWdoPOe3+NL9kJ8Vqi3zbyIRcj9QO315SL5AAl9VMWLmB3qIhp142s+iEWjYun85OXAyQlXXWGLEh4swgGDH4tgQyDN9hZBB2NZWyyHpp+tc\\\\\\\/UZ56fLxXGTzFYUGfO5BdWRgRqzDM28tuZ0p4ju7D5XaZtPvSl9O8jVnJFra2i\\\\\\\/LGlO7PVpSPpwxvdDJoBQ675oVVbr6NQo7DamWG5gcYTaJEo4eB0H35IBMazFlccHigxE2rJHjmbjNchDqxSFWTxLP9qJ3DD6cf9c+kaHFBJtsJ11fAM3+8VmlDVxLluSAP6V4iTHhbV5HPw4GLmzAxErba8xqYZT0tHSq0dLjbhMZO12Nlb6bZAaIv4BmJWeO0i8XuNa6XqnNaaZ5E6+qkR8Ygvl2yA6Gx5YL91rPYycMT3E0PUruRs38KAYKJcxHo6gOKumBV1OjN1fuVFXbxwJQeB1jHcUlNrqSe\\\\\\\/W0ivsvVKpT7JnbUsorGlFvgBsX3S9HzCiWxpy0z0Z8FhvteyMWp+C1OucBeSVtJUsMnak9eArv22dos7efyfm1XlwqPgxv3B8JptI9JD2ZzObcp7qTrF3ufN2KorblE3nsM5wnHRJDfY\\\\\\\/FjP4RPhDRW7gWWYhrEo48cJjigXEbCRi8AS3Vsr0iXacIs1WgWpUG1Q5h1\\\\\\\/xu1403if86aVdLrTDDXVI2O5nZlIyEhyf4Fw2PtXvUy9VFGkrwAzi5YkplbDNcAosWDlwNxnWbRL7fN\\\\\\\/BK76F\\\\\\\/jMe2VFuMBGv+6XIt5Klzt7kC4xQeM\\\\\\\/Vh5apF56bUvPCUE\\\\\\\/f82wwaED98jWPBO\\\\\\\/6SOFNDdtKPI3hKIFSlC2A86Mf0wXI6RxCjLXIML47lNOgJjg873kzZqNWt17Z+D5YEopFiQyRa1CDGk+U0uMDx+XW42azvGWhsTNBNu9vItHrE\\\\\\\/yy9GQFPAbd9gBae9vXmfSER1LQcLsYK1Ow\\\\\\\/yh9FY\\\\\\\/oiK2nQoJ95bOZnv2pgHkEBYQev2lqBnoJRCzpoJIQft\\\\\\\/e3SjUJ1DdXaSXIQ6AHIgPPJ3Kts41uppg08ZOQVCvyr62Gct1U28+WREUEbgHsZrqzc9OEhC5snvi6\\\\\\\/GxLxSWeem8U8u\\\\\\\/\\\\\\\/sYb8TNF7XjgODl21vtgl30x45gOtpB1I04neIgU5iW+ZENXJXJDNvP8chgwAQve4I1OKFYpREsFA4SdaC78gm+CYtWCaJeXwtAMOceCFyNZmm1fOVxk8ubBoKUJHVgvIeJTs19CpCtjnkGwEXj3ef1HL5PRuN9+YP4OXXTKO9B\\\\\\\/wDcd8nyYLEtsTXfiC+TZ+YQ2\\\\\\\/gpI9EU8Dc+4vqEwl+4Z6qvOrGcJuZsBqkl0Kp47fOPFtfwYDgE+WOEhW432hiDehQBqfz8MEiGUrTCRp10oedPhSmBi7ir5VheUqDKC7XE8BEE3FbXf0ZZELGJxNl4RuVv3Y6YKjBRwpigQEhDy1W2Zi5MIT409RfVVHAYJ2333fZKPK4Q0uJrWX9NBIEY6\\\\\\\/BBGo1PmMUEB6QvMBGORawErrpmofK8DRCSoNI7RbTsNN0P\\\\\\\/EiLOQJXlk3ct62I5iJC2uxa01F1hMjBNbfSMw3BRfHSkBrqiZZ+AE0jkceJOaDGf5U5\\\\\\\/yqzwSkdTbJO\\\\\\\/hQaGsGe+ov9rSvmsMhHY+DzEV4I9f3aPSf83y7A4SYDAoa3pwnkj3CrP0UoJ0O7BDa0zD+LS\\\\\\\/wo\\\\\\\/Z7rYsxwLnI2vSnO8S3xC2D36jNxxWiwL\\\\\\\/lHFYj7IrV5JYSX33g+YjZLNOxXAFk8F5\\\\\\\/f\\\\\\\/Zq\\\\\\\/1LC0CL6xucJlSo6mJZJ+3VxqBC4vyb5l\\\\\\\/2ukuE+bWhyRIQmz4lY7Emx2eu91n\\\\\\\/9XjqnQ2Xi4rXbKVed\\\\\\\/pc0X5uQ1iXC35hq9qkcwv3Jus7Ra+o3wMe2TbspW\\\\\\\/sK8eL2SCPb6rtWymAU8PIDqQROmgzQFxDZPzyp3cOBNcCvredQ66qJ86vxlBGIWyUO4njSvJhdM9o2+PmnsOGrm6C7oLKEgbusLevfWDJiTRv0cDy6FN3+fkRKSX9wl6P\\\\\\\/lqncfU1HOYAKPcS7qovtLetcyVSJ8DRJegl9Fr41JZUHr8FecsRgj4QwbYW4XJYoN0umro0SQbNarxj0v38QJ0UKTuTip4yP3+\\\\\\\/uBYQtVxsDE\\\\\\\/M3EXBxY2ca0sXNFZ5KCUuQxkdldNGQozhc8HvYdlzL7oL6pgr\\\\\\\/ZyFUAcfyUTCrn+OZU9eMNIweLSy0I0lyF6X0E2cldKlnym+Ty3OO6WUFB4Vrv6O\\\\\\\/wcP2LJyfEcfuAhM8Gzv\\\\\\\/\\\\\\\/bTj5tV2+BkS4jj6FTDsAjMVINrLcgZtXUZJDfkq76tIjcobLMz40fRUOUnAkM1tQR11Zsqwe9SmMrZvf\\\\\\\/d2R+G8sS7wXmArmVW9Ta92pmICu5Yc391euMANtcagHj2or3KmT+HFlSeZP8XZ8Ef0lxHSHCzRF2ka2N\\\\\\\/wgUoITDKoF6aF8bmxPW3EvF6A9llSzL4QdV\\\\\\\/YL7L4aVuCfSTuF\\\\\\\/4Um0fIpzcQhvA6dmayDFuIbYbTjY\\\\\\\/jMRXh7wjgdEIPkGLGKkxN2b2RdvIxDuptANtvemwBG4sCEne5Myj1Z6djGvBKGIu4PCPOa20SAb01RuH+J0e2nrbYv2j6rZkgZuKDq0vpiUxU0m58xw6m6ZRbLNaVJZS16gJKDqBJFyXUINPqHgrun541d0tkfW1L6xGhUuMpg+Ol2oljyHqMAgr\\\\\\\/yAxJmHCCIaPnAxnDt7HDbS05rzur9DBiJgHoRXXhLKiymHtfePbSAOxdxcpxkqwKbU9rqhXYqN+SWxUC58WAZmuy398AgYd86rJiFfEHmYB\\\\\\\/EX+9+2JIGj6KL3sB9c07Pbn3Ex2hTgvL2C2gKGlhyENEapfCDapGnB5l6yTlnGDXuNRPjS+TO\\\\\\\/fqqUo7KAYPxYrb+5kH5sSs\\\\\\\/RcJVdU61CwhWuna9clmMoGP+twDQl+OW08gZvNZN5+fciPUzfLvKEEd4dTlNgUAHqf6ehRt+UXDn64InOXiAQTsyeGJ9KlGpVrjyi0VynYl9MMHascAN+1cvtxdseMrk1X29CAKzpYdXqYOUbWW13tJMepmec6JzKUFOwoo0t65p1fNmDfA0lOXqsBEeO1NZsOvMpney+WQgtwDafBMp9n7wBZT3b2DYBOI1p84pPt3yXhHBZwHzmhqfWyRGC4Z11EOgA8\\\\\\\/4+G++H4wIJ5cM0dr0oCbUMEzLVPlQ1rx3Vr46PoyKA0NFx9mQ6fYCb+xRm+mfm5gHgIKaFzUlBISlpHZPAfse4\\\\\\\/KOL7CWnITstJmrHEE9Svv10pxmLHPTsbmik88zO56fAHYKax1yRaUudDw2XfplyscKwg\\\\\\\/6jDV4bjz4kFBQC522nhjjbasRBhjbT0ffzZ7zVAThwTDNYkET2vM5DG\\\\\\\/JvxlPu8C+7mpqGT9OALy3cPZNGP5Oaiyvbzw2E20iRWI3h8ioy7tzOqOQBcTfGbXLpYekVwny5YkM2ecNH\\\\\\\/qOgb2xHtsVSVkNYY73eLZPk5M8SncpmmyfzyBvNEK3MIpD7cmWYlrlpQreM8f38ZYzMt8Ygo6YPxn+8FjPTguMEiok6ddVU\\\\\\\/yc9+8s0PwYCkAKNPwgFP0DfeRawE4vK4FFbUXHnUnHS8+EHkexuaV0hhz1BDONZdb2bz4whOYnG9LuGjGlGzflzaRB5ghmOBef6cv5JuoHFhoohNTZ+B5b28bSH5DnfCoKjEGM8BN7ISBk5T+7\\\\\\\/ueIQT0kI5tBV2nBhX4sK5U5\\\\\\\/SG3U9CRnVNp9MSFTVyIKukFSSV\\\\\\\/DtIHwROHSJ\\\\\\\/pcYZostDJNqDLsRv6xFwXho+LKzk5XrjPNlE+ry44GmmVAE5wqCAjU7t\\\\\\\/63KjYDhw0Ns7tJWPIiBA75UewzLRq7pLX8xFuscZiaQOX2abdUj2RXV0vQqJDGV0XVfkZGmssQuX2X52apWS0eTZKkGwe0Tz8yqOE0b2m9px6t\\\\\\\/9RgpHi5ZtcGsf8V9V20\\\\\\\/372OHW1vyaNorjFOwB3bAfSmDzawbnqaplAo3TbX9wk\\\\\\\/xyq9JtnsQ\\\\\\\/6NmbtRLJ5Kl\\\\\\\/e84a4cFSUWmg\\\\\\\/s4vd8vUIRpVVp2wrOWox9YYWyZaliKGRu9DDc93L+8E4cJChfq8dC6j97RunBF+wSCHWRXcnwfidfevguB\\\\\\\/R3ce6BPp\\\\\\\/4cUDO\\\\\\\/cOIb6Amve8360V7RdnPq4+XSC8c0e\\\\\\\/6\\\\\\\/vdJklFaaQ3XGADoAa+fWGttAmQINWauWojVVhgc+XgXFm3QlDjtS+EQ1y5CkEEFQ\\\\\\\/LNNgsz0iU05k6RXGqBvwX05b\\\\\\\/tWz76SDQyik5ppdBVRJUPwsx5n1x2sbS62N90t9T1MxsLYszySkl3mc4R2gSuFxyo\\\\\\\/EC64S9EYz+ep6P52M1Py5wvA6ABErr7V5sS33bSYHesDl556kEUH7fm9PcoHz00HYz7WG9jlbIDOnlHWjAr2YgHrZ\\\\\\\/yw1yUKlnicofKr0LEqNAynXY9EtlG0RWGG\\\\\\\/lQisnRU5z7Op3W7Ol72Gw7ytx6xFCLV0yGaRUBYvik1WzHXtnigojK5e5G1xuuFryura+rSduV4FN9bvbf4M6AP9eYA8B5F5l7yvGl1RJO+eSLNvLFailTpO1JBLqQLQV7BK2WfSl3uhPz5wTUS3kjmedXsTVg9mObK\\\\\\\/e+aBGJMMJTarVxMIUT6\\\\\\\/3uz6fWZFuNEb9rcwBpQ\\\\\\\/PzrCR9FOaI1ojM\\\\\\\/\\\\\\\/orw\\\\\\\/DyLCAC5GVGEy3p2MzMwpKjm0kefiUgSEtCh4\\\\\\\/otr0BQHTupEnXDyg5ltkce1klsFNoEibHBTwOpx1IXVE51nDY3vGGNrTpIRuvT9r9d6nJN3vGVUY6kNLWhKx2ze9PwmsZXydSdOpf5by0W3rt0GDIVdjUDU+KQRUPciZL8WVupmo8RxUQRchF0+qXhYad6AupIC5oPaRu\\\\\\\/qBsLLa6l7DFtfYWstgHHSn\\\\\\\/Wm0fH5lC0JnZkHoqLw\\\\\\\/KJN5HnKsD0GdcCh5sVSF5+ccsN2Z0gXpgtIosSct1aZ5Qs1nFxjuRV+nkJT9+eXnMTUW4JQ3xqpyudC\\\\\\\/aC5Cyr+Swic2o5ueW6EGe7jlHA5thdibq5NhmCztrV8Sa57gyJG25lMaELwOAgdGxVADknVjE4optDEWIDoVKcOvyWha\\\\\\\/kNGkyJIs46pQk8upB277xV5SJPyZqs76t68SDWXqIrDwPAciPwV189e70TmTQATq8PAWFW5iifHqB9dXw51a40WFqY6cwB70F7nj3NmZfKaOkZq\\\\\\\/EmgtSkqQbSA914n10jF\\\\\\\/\\\\\\\/hbRRbGBAh\\\\\\\/aVaM4Y5uPtSA0kyQ6Zng+R77jVDiYm+6vqcDL+s94NUrky4o6w\\\\\\\/qAdc6FMtjKx2cdVATfYjF5Jst8kuKv5OWsGM0Z1vimjoeYAfVggfw5CH0gqRy\\\\\\\/tGCSfzWvc6wGynwBBNALfIuwXgXe2FtYiYT869G0b\\\\\\\/E1TFkA3J5\\\\\\\/E4SAErjFojhr3Llmfv1jI3gNkjWGjEGonm6LqVBjD6YkyrtzlEpIszbiNEaSRYHSh3IWIzOQRT89Kdi6RbGNA7jaeBJm2vSUeyoeRKsqzokYlt1DjvJNwEdAa3BocCJ7AAa9CO73mVHuSYFVgH2epqIGpG\\\\\\\/ryTKj4XqfgBVBeDDm8wmkmTYdrMUstvwfdtWRCt9kfgZf3N9NbIAsWLUt6lwjGTo40i8b1EJKF4B4AILLkUbUJT1RAj+cEioPr57PP2LxcNj+\\\\\\\/dCHHK7qNvww\\\\\\\/aT2736gR1RkxElupXUoRfRPAHF1cs9wYVyCO6Y892\\\\\\\/lJ2dtZEIkPAqpheMmPmn7qf1Cj8Ri29Jtd8RAv3ROTDdD8vaPphJWoZaToZt2vMJGI1rTeszQvLpu3dws+bIjI+HOif3h0R2FrEzjm6pp0wn1ua\\\\\\\/HyJ3fkdT7y7m0jyZ+J5JhBzHNV9Wcjzr\\\\\\\/qN6xg+3cZ1T89ZIDZe8AE6ETNQhMW9a8Va1Qko5\\\\\\\/dEFNE3xoTYM6JDtoMT6Huxuds97\\\\\\\/46MPNBQTSAykXCd+VKzTecyW7JXXrnK3VJTDfLXw9+V3DnEwfRQ55dBa4u8WHCAp07WHy\\\\\\\/dJGtVpNK0qIKI6h3NpLWYhkMGVbpuRU\\\\\\\/4nbzoC0uzRjp8APzVVWaq4Zjspvp7krJYLVTceXe+LvGN6HL3o15MNya6TIM41cdEBssFIhBqq9Lvti576X5XkDkITahxhg7zmwUVC6oiN9YuEYrebrKe0lXpI6nwlPcXH6OrJO5qx+s+5PAdd+N9p8z3+ubMSbOT029zgs10eItOz0JsLX2nEqRVL9w8+UjJz0AUQJKiEI\\\\\\\/opL7WsYDXX198\\\\\\\/Lmf991Q9mY8MSVLFDj0pHzI1BYvvUSqD1cBNkxKRpuFisfuL259GVM8XysnuSToNDURtvmlgecavVwjFyofdQS7r1eh8m1eN4E0M3ZGcJ562tHX0saIy9gyOXWq3e+Ka4kgUxLuux8YzFWGn97NRdaKLsohkGPRIU20Z2Ds\\\\\\\/1YbXBR7AbnNtlH78ojrnXvZa3FhlIWTC\\\\\\\/7klcLRAFqOXGZwC9z0UUFA6gvhPqR2uwrvCfyEWvnJNVL7sB7eIux4+VwGeixr0\\\\\\\/qjjXApDzOB62SylCxsnUeQfmLjmwAJLdKNjz245byZXUUXqOyFBvpYbMIAI+8V3cBk8wXt++rhj2QhYekQaP1IeYuSCzRWvKWm0NHCP6BV1V3bGbonASs00N\\\\\\\/0hbzG6B+k18DMoD0i1TZMp9RZR78AVGsqFFG4ziTICWM5UGzAYGJNSK5IsaA+1RzLzZjFdVEv\\\\\\\/N4EBp74DD5PF1VH15Sob+KjHrn3vocO24qV0X+GyqRuTeohL\\\\\\\/UbU2hG+o7BkY7XuGwTpT\\\\\\\/XX9y\\\\\\\/HxxeEqIPOZRGD452\\\\\\\/YWLvzlnbB2+IF69bUF5cGXfwdmRAfs6GYgFHq9bt85nGtSZbT8A5WmPIOe4Zi8g38cxkAfTWWaAvFHBAl7tY8Jr21PB2GRYykWsXAu82gJ1jggG5YAXUjrsSuW8azJbDZL8lATyFGht5KwsQDpuT\\\\\\\/ywLFiaBwR+WaS7qxgJnB10kq6oWJMBGVJS5imf0hLGsQXD3j4C9MiB89N36qxcSyUAFGWcSLHrzj30eypaLpHZaC5psazbvZScjGoMB4vGdCBJjhqgg2zkq9zDiR7AEELu\\\\\\\/\\\\\\\/NGVRhs5wcGg7vRZTr7FBx7IHKW4R0z6AhVUbY68HB4n56l1bwvhlnSymtbKgjHaOQ3QBPF8\\\\\\\/QefjKGJDZJegeqbQ2ydpSpbaol9ObB44uLYhAm2+\\\\\\\/5rV4gkDVtb84LfJneW2cUmS1LNAd1lSqN4pPwXIDabb4zZ8Y338PNtL5u38WDnx1o0O9OSqW5JPvIgPjpPBQvIw5QVYn4ka\\\\\\\/Ds3vS1\\\\\\\/kFymEy+TiO7mhKdBPleri6Wdt+jpm0xBe7yzOIvx35V+jucWxCH7jmGUPRNj8CDetv+C\\\\\\\/9u+6Wu+0RxzKLrN+N4xtk0UgKxEp8GNxa8Em3LoPFFKhLie0KqCs0Dp9RO77WksLAkJWSGsgf6ytRtcZLMxlCiTDQOaiTLpvn0rj5AaLpnwRVCAXpEhGuE7QoyoFrZX1eAKTGKIuNwbaXWrGt1hBO1KKG6v+DL0xXvlHcxN969aPdgcUUu5sB8rRjsQM1ltZrxOWHqpPWc\\\\\\\/Sq1iE6TCO3CP+u9zhRaM8pmvRfIVodwlLBKhFJAwE8Lwc3UDAIbsFSfHtPmjX0BVA5cAEPjo5xaQdBQqrPGenfqy1Nx7\\\\\\\/FvTol5\\\\\\\/dfSR3AsuPtyTi3GoqvE7TsVsZv+W8VKdQ+pZ5Gm5BiNbtVLOlBIuqNAZf\\\\\\\/FR5zpUY8\\\\\\\/WVqB84XFlqBPF1B50XD04maI\\\\\\\/kIGHIFxdpU+IZW7x+WldcOgZOg7IRAVNqkgG0JbUbJQqU6qI2j5wqfrbKus7Qi4GJvOVJVQ2JU0S5syNJwDu1epY+4I7lA0M8ZD1cM+FnQMIVgRylEAjvzv3r6srIT50+V6e2HRaX1esjYAr2\\\\\\\/NN85pjhR09Ge0erZKdTkZ+CHtwuqOVH94+32Kj2asXib1WAGwe3TodQGbRF\\\\\\\/zeBbQ4uk7yziS8aamVatlr5761K8LI+K5IhhSrya0eR0NlFsYFiNsV0Lb38\\\\\\\/kGgdtQMCUL3KEpfG\\\\\\\/v72zyCGHbLXJCy5sIHBnG2KuC1LGHVdSp2aObR31TeAAPjrG7VNXhMp71SuP0RnEiFeJOaaecfDB\\\\\\\/1lv1d2E2Dk0Ivz8\\\\\\\/ElAW7tZljsF\\\\\\\/lHC6VlNca2UJ47yUhS209o2VDvQ1mzmmVXHjR3KHKF3DkNlomkr3KBCOVj0DhOT8hZ0GP+FXkDL7A6oXZSWVrq9roKbg2GzWNfFrKls0oYUEp5Bh+yPR2tCEIvQMuzzOKjO0MWij5jY8hRPq9rsi0wROH4ddsZfGmIKM5kB0exU32o3NOU2id2lQQFJe1Xus5uFsS6S0rI2cqt3FOjYl8LFOcmKcqjs7yhaexRox8j4WcHIXeCR29NoDxG+DSLj0fWu8FZrawjvWfzrg3pgQtmRd7Rq7vO+XjJJMj+Tyj6HvZJZ8h+Xf2Ti9rleQdnzhaaHlzJC3WGrCu+Tk+wxNnQRGEI\\\\\\\/5B2t\\\\\\\/aSrJi0SCAojb7Bmx5gmE66y5YNDTmqXE4txiOHQinlJRljZ96RNBtrlNCzR3pO0Wr0Vzxi4sF7IAi3j8KMk1kpUP8lBqkXCR9Xi+GOPHHdtLrkZP3TILM7UkXJBe0liubfL9sno05ufPVoTKGQiKlfhwQ06RWz63g+nw2e+w0e\\\\\\\/1c+AzsmyNETypCs75\\\\\\\/3SrjMmmObk3aLYePC5ccDzxhjU0GuP6EXdoZN+zhETzgNoJW11HOlo+m+i3FsyHYQb4lg4ZoKQV4L6M3p34UNioW95dwZ8VCM4LqCmx4jCoOGzH6mxZCyKxUw2\\\\\\\/Yq5awMkUf7VjUwAzv+KLgF1btR4kU7opBL9ZNzkSYWpEZiVU2L5V8ci4mO8vFrHeXShYidiDdh5REGX1Npc4cDAWQpSoo\\\\\\\/CPufkpvXmD1DR5mXpS4epYPcq0n+4LEo+WPO76FQrp0R9OxLqcMzMUh5yWL7SFIhFiDtsh5er6RVf6reHi95kAKW3Mm61p41LB\\\\\\\/xIz9d6djHkYk7Qty61Rigfrf4umF5Q9L7oXxpz9OEXvratS9v9Wl2UEF6XW1zudS1lTvnYfE6A26GMLUwW8QauDNHC0qnkK+VGXmUQMU2iW12\\\\\\\/+VffNjb+sW5v2tcYCm\\\\\\\/8\\\\\\\/IYVUS88H6isaIZiurG5HFh5O4d9PRdV2YvhelMSKwvvpFeV\\\\\\\/jLKo6BsWbznyVHqC4BZRlaco1ZBenqVNa92XYUAPECQWbMDIm2yBUfKU1k9Z9zSiFBgZeE6fSgHKIBX00v65VLTkcHW8dUliurQUqkv\\\\\\\/05OAaJRTrQxBwQSy6feM0PLAn\\\\\\\/OzCJzQ2FwnPQL3m6hW6ENcZY5ZI4d7Nwx2SoIGaLmyUxiJ72Q5xoKGbvbHbGqJZpA5lRGVGQ+YTiqlVci3zX4maMY+6J153oggFM8Gzw5aUW9ff2PujCCJ6xLhVgmJRLMGdxjWZlnLbqeC6e2fZkrRV\\\\\\\/rD+t+gXZKwpE7eCujTl0IghOPpiBjOAUpzXqV1sJcsjlv3U19RybgfdFHPNpMIViEDfwhnOdMeTvnHNfJL\\\\\\\/gigyGxUWYUib+E33x4WhrmNa5C6Vvy7RQhtSjqZZDiXthnGv93PLW4ehNbDwi7JyYt+DjgziEPFZf4yDw3n+ChlwOvYTlgrEvYMrcmG9hb9yrkSy5y7JDkD7uPE\\\\\\\/rRO0V2kQOLgcBhxvrI2yqQry8skQ7k5nbuFk07NdGE41ykRkyX4LWgvI6DnjONUP4kbnGRYelsnA82i4dz23JKCdDbHwvVLFu2UGAyUfzcC4Y\\\\\\\/nOdH6maIw+2HJzJKDZ8312G\\\\\\\/na7KXS5Xl2GBFS2lJ5oSu9S+1bsT2hqdpdnMG561p2\\\\\\\/JjVuaDhGEITZkoEARWo\\\\\\\/CRzllkbEsgUSzjhm8asGYFFUs8b\\\\\\\/9E7qqkFvxiGMfJloeNsegfRp6oDybQ6vy7QLHKsONXEAi0boeNgiRdDoaGFAf8GZz3ctyKqQ5RTtuJsPXm3hXQnBJfmqAG5lPgRqSlIcySodoyZxxrgWGbuy6STiHR4XE9l2JIAoqLYzr8\\\\\\\/z\\\\\\\/IgG0CgRIANKLzeagAXUzA68Tw6iq2km99FtK0SjD21T58077ZezS0ptcDqGGfhFWqI\\\\\\\/Q41ZpIe6PLCJTnZIIHeLFSM\\\\\\\/Hc5yFSTrDYzEDqahGflswYlc0F4dwp\\\\\\\/GeG4\\\\\\\/qO2K6oIZE6wodrfK9YUQOQyo42zbzO8UPextDhm8NgJyV7\\\\\\\/iJGfTonoH8bN8SeY9KZJnvLNFKYXgBxVjvuuQ0Ufe732uiKW8q3N43WS8xIMoPLutEFeOHnN5GaD5JyUEhYRML25T3J5eCFMW53TQrVH9S0G491OBm1sfsZroJPuDuTPJgEuyCIphuiRXclMsv\\\\\\\/Wc+0BRHE9fHbrBYBa2JJaKyc3zJS7viLdezQmD3QDmhCpmGe\\\\\\\/u8N2cpxyLy1sJCKUFU4eqW7IJa5bZvmujMaHKSDy\\\\\\\/\\\\\\\/Y4nKclR8hjLgH5h92pRM3pyGt+\\\\\\\/aIGgnPXf2OGyjC98sv3+Mj1d6YLJC83xuxSskoAzXmIazBISRhu+OVIlAMTDmZiWAJjWo7yZvHLyaVuYE44ID0po6Xm556WTxDS4fj1vbj\\\\\\\/GX+bfeIhwMi+XajEFWcl5CIc6rNeZdnwre67z7pLDWD\\\\\\\/ICRbVxCN7d\\\\\\\/uH5eR\\\\\\\/xQxy3rKxV++NmlaKeYCpyECUaEP\\\\\\\/eIZaBZOAOJLV5U56uCELXUl4zU5Xd3xrt3K+l5BiwGVrJkrQkqThxwd7okFpn+AOiygujZIqS8rFdeXEAS5BPgB\\\\\\\/AInIBopsLL2\\\\\\\/m8\\\\\\\/TUzU9IgIcjNzWPhhrG27GcGSv1FwENapQ8pvqWAEdpa2O9BEQYzgj\\\\\\\/hm8Pt3t+6pDmxLN9u6oqPDXpWJ4Rul0KB6XRUuKQe\\\\\\\/jGgrbc99Fa037st3GSrAzVKVsJWLQrIsZYNTR7Xqv82JLem6XiRgdnWl0OESwcwGQk9LN72qw0EXo8YlRaBBmDqgaDQFRWzlnvmTfNkKc8qyqfw75pY\\\\\\\/NjYpJXISFR568Fs3Uw7b88zGlmXFe4yl97aAxpK+k+shdeCzwKJh4rSZxbcbWayn7cBJU8HfNTE2aT1R9fsN7N8jAlZRxSKBfxkRx\\\\\\\/eEIwPR5jYpkDhhwfHEirPhNx27KmUWNo5MRaQls7quba6PiO9yNOFusiGdRYtM\\\\\\\/wzEO9GbfS\\\\\\\/IoWzTAzjtsoWscOtEDLfygVzgEhSxy4zjpo3bIquAx7fEoN+7+Ss2e6brkPT4CIxH1eKZ04GD8XT6AUqWUMCAQCDvcMRBUg3x1bc3xrSequvWwglW2zJSWN6dkDB8f6YPeUhqIRHie62+96edBTbVBfrtSfVR7jSSM86eCP5ZKb1DDLiXzgO5OhU7W4yr9Vns+50EtLo4Oh9UXiyUjsUx1+HhozHDRkS6MQKpLYnPFv5fQQI\\\\\\\/mSZ+7\\\\\\\/HiGsofRi2a3X8Kag99i6oJKvUYepSX0Zf+APRcfHCNvP3bB4e3XCHcO5lgPI76GPNcWImhyZr2T5TPSq2k4xEr\\\\\\\/Te6SBKXYYO5wc+NYCiHW50g8QOGW7Rp5EzbJ5CLtV2ojna5jA6lLzizw\\\\\\\/\\\\\\\/nV7RsSBHUPt\\\\\\\/5Pvq6uaqrxD\\\\\\\/PH+3k8vbGs84V+yuflggE81OMnRmuDnAH6wNuoOO1IXSCPpOWnkSdNQgw2BuZRKqvlL3dhfk961W3oE14at3Tmzre6d0kq21LWnFME0NyEWZokZGz0HXrri2WNT\\\\\\\/R2IVvWPUKzhNQIVRhj1S\\\\\\\/Iflqw3gIMu58WjgWlreHhUjzPcSbcAnryIgMab\\\\\\\/fUYRXUI1mB0PbUCLhZNcxgJLSDYcQMo8V3P6gnppxrrrXBJA3glWKpDzPwXyjQDpMXzvj\\\\\\\/kKUKULAxLwoIvVl4IArg1jAHq5JjwrvqmdbftnK6HxMZ2KDqE6o+pAeq5rNhnREwvYojc5OAqdUthoTJCTkperkUsHAr4hBe56YAeqhH1ymk8xNU\\\\\\\/5WZon\\\\\\\/difPwd809Tq4ADibjvJqfyXHg7ozBwL3lm8\\\\\\\/Iz08QVO1KA5lIuhvK6zruvYwBlZQVX4cKpEm9xsMVDVTgTIZo981EvYMbb1wWKxEh\\\\\\\/ut8nzmFJHiQ\\\\\\\/P+lg63IOlETJd15GakMcOWRiSG\\\\\\\/0IABacadiMKqvb1\\\\\\\/TyjUyxXeDJgQYvV24j7GotR96Oo6+j2dUyriMLhmEV8AqQTdE4mX2plryI02ewNavBkyJJLFG80Yr1KM04ITfPW8BqT76YwQ\\\\\\\/Q3el5g7RR8MfzWxgcTiXTNNpIDrPeKBqSD9lwTe3Bwj9z0gf0ZNV8YYlwt0lMCiJHUpo0Yg4PuFd9yBI1k8U0hp5G7f7s7lHqyxg6AWbOfcQEDyBRJjvHVtSFdxoxDwR1AmqXZeb3ZZfarXUIF+g1S+utpF+JfCSylfWTSyC04YGpVHcaj4iifFmw2ajrubD7NaWaDqTVY1nGd019s6UCFJuhaMPGlbrEGlSbQw\\\\\\\/\\\\\\\/Hf1lFCGeFV4+o1Ycbfos8mj0rDd0CasqLPNK6ZJretEhRQQAGyPoz811DQ1WJ2VZ6l\\\\\\\/dykfYp0gkuI\\\\\\\/CfTmUfeF99kWs1ypeN\\\\\\\/bDUyEjvFEaDb6gdfXISnmlaqxVrMOgG0+VmN3HvorEsLTmx+ZFkOHMWQE\\\\\\\/qcSt\\\\\\\/2YR0vGUNr7MM2DbdNI7qwwb8OHj+l5PNFdMhdKOHbBysCq1zBmRy8gfrKw48gQKZi4vwTG3KsD4fURwYI98NWQNmpMPYL28O8yF7th7WQ+2pqDp9sYLe\\\\\\\/okgGoUzvrJGtKkDrqK4LXUcZp3+\\\\\\\/F86OEM844iW4nuTFEN1nX3\\\\\\\/Q4puP1xcmnHWwU0LxP2JXN21jlNQ38oQIBO4CCH4fTFKhqlfiO61yVPNSUpOJzmnZXilZAWt2KxubAO4mdTDeN\\\\\\\/4exNHluvG0aAfoJdfSLnwMOdwc8kPT22Yw8sPB8T7FW5+1u8e+SnVg\\\\\\\/mwSHUwUDil\\\\\\\/4q4lk1uCl1kWgn6PfVw0GoZWSv7Tc2v1uWghEGcsA\\\\\\\/4nwIiTMKStCtOyPENrIaE7re7o3xLsvBiyoR8PpLKSZMjJAh2BGGWPyngtYYrQA4mQ2XBuqq0jAyhKgyXDA9azYcDTgnTt1UqhF5H04Jurja\\\\\\\/FcF5dgsPv1eyEiXOGwmM5+XHtgwKUomaEmgShtdOsyKv2wmnK0Jz1Wl8WgHRTLTc\\\\\\\/vKfcgR6TmeMqrGuFYSYugRIpz36ASZ8ZFysNoeFVh4QEcm4NNvu4K04nF0\\\\\\\/lj47Hwzo6RzDKEcQWdhGpo2cSCtD\\\\\\\/Wzz1swj0+6OdRdDmyJMRNDz+0kpA9WseZJdbHtCCDBQawV3e5hR1Avo+WFVPrhQFxYTUwlB8Uef6AWYoUG7jurdee0DXu7\\\\\\\/tjeJbNBccjZE0j8y590lJLWCxPSvW937dIlgPZKIjnsE07CeN9vPyz7eSo6zr3yKGSjBVy4iHm5BeY4nAdY8szumWR3v+eTLn50uZsyAQ02sNDn07uQg5NN7Y\\\\\\\/DkF\\\\\\\/SWFXDgLTBVMfBiNajWNNIX1982zJ7SY07nTV+f0Zf6KDi8uI25tafXqDGnV9uE96NS6KVUDibRI7PnsaWy9BVrVMRgGER12TRVv5gDceGSXJsZ8VN\\\\\\\/Z8EKNBBKzSENvDg77Wj39qccWYlC7NVRuyc4Fm1USw3BsZug6vAjAIGbIPmgT5SaxNjl9gcJtg5y0xToc\\\\\\\/BliE7TVHfEKK0ejL+4G9tdf8nfk0P2t1fX6f6iBG8QnOIiuhH8r4vEA6z++cyWZf4+EZfBQZej4HHgU13\\\\\\\/SKLNm+O\\\\\\\/Tz43NZdAPc0DVZEW7jTaFtLTzFvGhBp6gJq5bRnjM7T3ee0T0gSM54tdQIuDVCDpi4RZme+lijgih+2DGzikb5fGSt9dQEbtQpnVxehaLGSSdMUS+1Q+oScfL7pdjfF+F\\\\\\\/3ioom12giwJVcmwON351Sj4z1Gzl\\\\\\\/e6eqPHTCZrFWNCDO2KqnATpPHjiMUPOhXTtp+WMuJBVgxIrCi3vUQu2TxAk1sco810q1EWWRYR5PbO0MwoeBxFkpA+E5Ha5cwLGUanf3VIN50ZeIuoZ+KdisfNyYzwk9aLaeXLmMr\\\\\\\/wYQjBEjBDIziat+Rtr\\\\\\\/sSF7zNblIGFYrULWxW5\\\\\\\/g4Ajv2VYY93b1ANHviOMjPnW6jgfNVMVY+ic\\\\\\\/oc8b\\\\\\\/UNQOOebcGfGtDQrvF9XLFCBPS1qy5CV4A83kmETR1tSrfpCAAz0UzWPjUDi3MnIpdNAEGXv2SSov8oAwmSylHuaNQ9KJVy+GeVf+OLZlL6ZRn1zSayr007ffV5ttlEkhcXYUBjvDboQxzdILY72eJsOFlRmFRD8IrsZxaoCvi8gqEM5YVLQeLmJQl\\\\\\\/JsG2CwUI34n1VSumvco7ZBzq0lPsnEJmMZWJgSEj5ol8Xo7YuejA\\\\\\\/y\\\\\\\/+t0PyVH8wUhu8B6mHxxLzkFM8F4JqHx5wQYsjw25JJ8oHphva73Mkmpc4TCm8BfbCmfm6\\\\\\\/113sqfsSiZayOrBwJUUOUaizel30sFcloSko756YfXUkId59RHxMAnjZ29pVmDgvg9lZG4W6sFZ3FknuVixPVMZeSqsDFo093nbpXt0c7xt7wSOocv90DpphnAk98Z\\\\\\\/xWfibQKlKwO5rUFu0EFCI\\\\\\\/37F2glKzMMVZzruVwStUARgfC6VDC6S3tS+6i16dCR6+Y2PQLUpsK2uzcb6w28xzJ0ll5IlOYAC8sL77dffEYyq3GCAP1uo+aK1Recj8oaS1zhRamE7VtCGq95qf6eHWmzB0mftg7Gk+XaEzOOCMUh4mxZka6KYvjuHM11KclXG2UGH7b6O4Qny5aZS76s6cVCKZHYoJhCFDlx0YF1cAQzhBrS3sAgfDZaSuJB\\\\\\\/4Xi\\\\\\\/Qczf+9odyuna4iKLXAkaLEqwPH25g3ewuNg4cyA2v6PR19erc5M5HuithktFzfsnOA6+1RHBK0w+cGc35ti+qAqiHVYzxhBxll\\\\\\\/OdfT34xigSL7aLY4UzjLbLQABCj9x\\\\\\\/xNEUn2GW0OIa+Wu\\\\\\\/\\\\\\\/o\\\\\\\/Ht6ohHWreC9Iyw9DtIjZD6mJewZ2tllPIWqGosivV+CpVABoyCbLcpmYAhi9h\\\\\\\/uLn6MUH+stGDu30ZBWNVis8Yxyb3neF6qgS5eFKOpbrygBDxqZ31kYndmWMqKf0LwgjuqCaAZ8oZSe9LAYzFEeYDDJ2QHwdHZsNwtk2p1DDIVCDuv8HBWxHOKTCPGYgC9O97QngkGfJ78gSxXDW4HC4CSb+W2Xw\\\\\\\/rr9eRame8POfe1KSkWC0GESUMAoVeZBf20wZQkihVVtOT\\\\\\\/xRAqBUJV7FSAYl0mDNOw9mmRf48wo524\\\\\\\/X+g3BNpxsl54eqMyr8CFDiiwo4vG32YI8\\\\\\\/ad7cHIhn+UeL9Ex1xTIu9CR7G\\\\\\\/aJexyqeEi88mlIYPuGs9SxvadpxKmRMpAkMBpdtz69HAienXXGR3UUM5c9+L8AdLQ5LMGSXbU49EGQ3FF8c4fsmVQYZUKxqWiFaUo1brifFUX+fEfqktshzk+xb56qhpbgx2jlct7DE6QZcVyemw9e6LIbHkvW16tFfP\\\\\\\/Y18HGDtP5IqSiCmT+GK7Y+I0sQ6j586jPuh1HPLRWslubNv5eHx7Ahxv4NunRIOnBme3auGcbJbtW\\\\\\\/DnA1vcXNP62rvlJhfA9xEJOvy6nT\\\\\\\/8M+sTNXARsAG\\\\\\\/oO9DhEeY\\\\\\\/BQULFVYMc7XdOOuTc07brOjeqfNsEKDUjd6ObdoOwII\\\\\\\/Rh++hS2YbGX3doI3CwBZGGqViE8SyE4x0lYyXKb8IZC30h6B\\\\\\\/8i3cN7EzYvO53PsnbmNOuZUhbhuPqvhgL+NlHnL623oTSPBC5afnybfm4nASZE8HQlVDaof\\\\\\\/2C9D5\\\\\\\/8SYFiXpKG8PeXmPvyAKw\\\\\\\/+jHHKdnaPDaepnnn1q53csWHV8hAPgGhd0HpRXZhAtPKKS07dPJ6WZNTJLCdLxCw2pT3SJD4Yr07jk\\\\\\\/UExjDS4WIE9\\\\\\\/OiQk\\\\\\\/EM2b1oFpA2fhEOYDCCygn8yvnugCazb7zvDCcvoy8Wz6Y9qAh649PH1BD0HTZHHyYv90SPWXoG2LQfBUStsRaL3HnfDvgSndqvWH\\\\\\\/DFOsC3I3sGY+ajASiuyZNhPDxnH6utcaotOZAaOz3L2RD\\\\\\\/\\\\\\\/o7f20XNTYEzIIq5C7oK5s7Kdsv3VSUNDhQTwRX2CGhhmEpMybczGd1hSUPEpMiHMqUG3Vb4AxJSwZTeT40SHADnzIET\\\\\\\/POCrUEwFtNAOaiRhHYZfYCXOWYzFt+lK174J\\\\\\\/jvssZ1z0hcku6hNPL7fe+Vy1znpFZbYwzpuoz\\\\\\\/v9jD5yeb8Pb98puFJg2uvUMVIbVMcuHcUr3xNMbqpRuVxdHZtuvwhMizbIRojJoBixGosUX7XD5gEJ46EBr93PwEkUmyK4ayBV17RFXYkUoeAL2q1vEK7qdOj5kvxCH\\\\\\\/OvM8Kk9pi9HHfzqoVfUmxnKWgavhRYSJNscECVaKoutZWIfCUCHBAQ+kt6ksLynGPKZY+dobtqj+DRP9YqY9bCOqpwCRYcZ655EGtTJvMR2AQUjA4uWpaKeMrE9C+vAggfldthjKCQc830x4zJvZ8QKE8F3Bm\\\\\\\/U58iSeiQ1wpapqGmjDDsXGmffl5VHP57E17XwSBCWBN0cfffRUQOhAVsl6UY\\\\\\\/Wgc6GS\\\\\\\/mgVfuh8hMUUJJdCajeUqNBKE7zq0AFXtKt+wufEBUo1KXItHyx092iQwjyICoxPZUOEgQ8V\\\\\\\/Jy0sCs\\\\\\\/aO1ZODo6zIrEul\\\\\\\/CRZStO7LBi5g9ih2Izg0L7EUnutgrL6ehjN\\\\\\\/8DNhrAQnlMb25ooBAYwh+BMT6HeCaH3ReLdBQ2fxyXc3V1d8A1tTAeP86Lx+u5Eb+49REeXo8GChip5RfGaoQAjMfio79Nn2ZI3JkyWQFXIwlfbeBX9qD1Zg7fneelOyOOPOcwdV3not29nTVM9d+SJTxXfLUnUOcgM7X3Tl\\\\\\\/O7w3n1kql5v+YuFW48FdMICDTQJP4mehYu3KDXhss1OBen7SMs0T3pTDl0ETWusabm4kR96HOefzv41oaBqp4BcDw+kJ60yS1gNw59Hqo8SA9cTJ2bXw0DpAZG0gY9LCvHXtMdn26WOK13FircJEDio4vMjAX0z5JAYoPLrHDBg8nMW6pX9SaCkm0CSv0sfips\\\\\\\/9YrHC+x1M1Z\\\\\\\/0Ska29wkPPQYnFo7Dn+XO4Join5ewu7kc+imLm6hQjIP6oTM53LbIsjsLEscdusDLPUI1xaISUJM6pPOzemYvczQ2bMEdqnd9O6eC2qFmSSMt+QcI1dlBx2TjdBDOnBL4kf1H7C6nROupuyA6E9Dx0TPhVj2tFuefJit8MCgX8v+gghXNlynX\\\\\\\/N4nAhcc7qWut5Awm7B60qUrAVJeddBQt0DO6PJlIRXFIam1Lte4QccsFWFgNT8bjQHLPqJ3Ma5VBH7emlyFNuTH1dzlG4P6v6oyVBoPWeJfuJmCXcR3iXT+vvhDlIMcYFerxBq6h0pWr87LTo0aVWqRHo09WxpLnsX0uIpmdrXbxwNhjwd+h9LtNX3y\\\\\\\/UoNum0YsInB8D+lXoRouJUpxI0upR6CWp8l\\\\\\\/LDrpyvLT\\\\\\\/xrwrO99iEL0Nb\\\\\\\/DRv5N78i5rcIxKDOz96mxnTLPJFHMjm7U0MP44NJp51SQVcpcSv6ug+5hZxGUtCfSL22XyTjJsdw2XRc8BOzMz1J6yOL\\\\\\\/9cOxyg51w6yDvrEHYPAqAWvXjcm3x4i2XRqYcJ4Gx7ae5lTlY+BYHnaucDmnXgEpBXY00AKHozEQ9TVuuI+2BqEHw9p4AhgFqYbSdGFAwqAetLsmhSlpK80HOcl\\\\\\\/pNmZkAY\\\\\\\/4IsHEjZt6tzz7ZQ+M4CiRnSDtw6ZwbNyan155csl84YqMgD5ySBc1R8q7Z7CccF+LV\\\\\\\/jU0f98FjuzHlr2Po6rCeiyubPGbmtWofxtqkH7XJddIU81KS9uAVAdI5H9OqcMhcRkumb8WMhIqlrrILOMuMyGg8rTzGnVDlxXXRt4KuqRYnwmutRNlTHF9Ewl2hhHllFP3L9stSiMzjWLS394qFFZB10ZmrfUWdqcrNLNyZvV9Ngz+AsLsEyVQs6TOqo3PpYbFFfQaMhQafDhFeGKbCGeSYU1usmnEWxCgZixN865Lb3U3Prpz7EHuv7wrqp9MOpof1g0E8MtEwlim7D5S1nwU1TO1a4oAx3STaqfLFWf64lFwiUivJ51tPGoCfRhJcMw\\\\\\\/yuPDZb0qCQ17eGNT3LBd8RFThHmAgPF5sdp+fmJvC6ZtpDBZv8tTTI89TENhar7xEQGBlxzybTVlmIdnWUnXQNuWkvaBzUn1EJhkzUnOJUn\\\\\\\/AbJjTx5n0zmPcXDSyobARD\\\\\\\/uzEM7aZeLxet8OlQAPQ3skRZSR9EWuSfwr6HpXuvWNDvcXYgVep56lE4w5hGsGcmLgsEPcnS4UNXB5XbKskEyxZQlSd3ENxzQvunWKh8+zhhpBezVTbaERWlXQyKo2JfBYFLai8BKQl50NjVr+9vtLHvxa3GbIiwfC4mAIoBJ2WKHtjNjWNCbHP0GcKEmJCuED491qF8JIKfDtBxYbBBSb0nYgdlGte1zixWKo8UAHXoHCdppNGQ2C\\\\\\\/6lHCsufjjrRvXb8qEPZTRd5mR5ZMKssfGxv4Rbhw6SPvMO30qYz1rx3dhMceL+UPjcwEgfjxYM5o\\\\\\\/DiMvnDPgZAg8igNN6AbrxaRNWi5WAAy54XzHewrLrL\\\\\\\/uaYAzM7U4LISrPO0AzhKZA4b2cxUSgkxae+G2ipoik3cjhru9uVLLDODVPgdYEUYIeWrjLneOeRH1g4m2rtietC+kpfASlaRF4qfzhC\\\\\\\/ATk46h6l10HWE8ggdsYbCQyZ\\\\\\\/Cs18wH1e1d6h3Xvix08D1ioQ9Jyh4BxiTWeIaMJ9rGd+zX7mNB2KiJsjswnWxBWOX67Hsb8RSWLXKw5MdHmDGczcb12G0msDtmR+fgvCPqvP6oiLrKuOmNNe\\\\\\\/Qckc7x02vR+IvHuGgfGCcq05Mcwg21nnTpZ2NGh\\\\\\\/xML\\\\\\\/bombmf4\\\\\\\/TGimvmQO0kdY\\\\\\\/3mH2MzZvi4Mudm86IKXyDRKqQkH2\\\\\\\/k+n4TGKB6R+QsGPy\\\\\\\/OzSxFyWCyXV7gHVq1BTTi5cwWrpKJ+4DMNurDuVMh\\\\\\\/aXaAtHh72\\\\\\\/7w\\\\\\\/yoWmoJZ\\\\\\\/MAzFm8qaxuqkbvn+sT4zJZ+M8YYqXfQ9D5bK9w4wKxmCXo4AdEJnHDDAKztHvcL9U1WGt3jrKCmtC73Dmmooc\\\\\\\/Wdyxfv6YfHHG8EcV0QnkoAhA+w1pKncIijcoCJsa2Ws1v\\\\\\\/7QgLgG0G396pAsvV1G\\\\\\\/POkaapbunJed092ET5PKK4wknSIQdPyHzfWOWEvRIIlWRJkLjztZ8cR81BaKupf81brLWnfE9kkGHIu6T2qqt0SR9HDiREXStYiuHRcNvVMwWAfjVbDKqo3PglfKFUUaI+xYn8J7YVMXmPLUBuybKWedbpvhlFYFxuCakqQLs7DF1QlsruMSOZY5nuvzYiDdIm4CRKTXq9pWMNuXun5yqdR8QVKyVxBppTrtbPRzownBuPufy+gfWnzGU7qBDV0GvrYCbWPjxd8wfRxTePoaRwxlz8+smKYQTzUVSZQdHusjDcU9SWvoxdgUZBRFWbTe\\\\\\\/kiwvOWFL2iEj0pKMztrjzQCcIm+lGmAqVXZ+N7vCHr40a05tG8ni533\\\\\\\/Kb\\\\\\\/RR251cz3KkXv1VZaFeLduKcoC0MhNDmlJfrYl0q1bsY6n6POHNGD9Ce2hYLUcIzw49W0xyQW6PYiQvkeznOsfXEEDzt3vpKjcH73MnT0dEu\\\\\\\/Tfbd\\\\\\\/77t1A51trFJ7Uuu9jN1INL\\\\\\\/PM6pAhxGfIi4csqbQqM2vFdb\\\\\\\/x8TnOXR+e7rcm8quhsGo3xFo\\\\\\\/YtCAwbY4P4UQ2setdGhNzk5pn0+KcDrdQ9g8AOw8KaRdA2BX5z7NUO2Uuf17zxHtZ1ZmrCpM1w621JHgf6W2aCqiaj+veF+JvzHw6WkUJvI0LLCB6W7zNMmc\\\\\\\/c4\\\\\\\/J31p9DIxRbamzwcIgp9Sxih790Kwx0WuPjOD8c3qGzaem2DZZ+KRGQrxqT+8vEi72ne3ncGB8rOS+v\\\\\\\/aJZuPtiO7p0I7RrgXhoTrEIcP6qwtuKXk5eBmWyFUhbFXH8o0U7Yfim22HLz+ncOWnmEaK50TqDyR9hqy9eURukHyu3evRqzwGj3Hnk2UFDD32KHd85HajkRneSjAz3YKEyXSax1znqovrHG1CUEKBsRksYou+vDAxnDWdpYPjW8uwZTjbnULo49FcEOedOYjqSE31zXzngnZN0eAtg7brcTK1C4EjpVMOwaWeGHF3HbPUX6tkns7m+YxVFR\\\\\\\/aX2wmmCoN1JGGUJfBcHCYdCKOFQnst33POJODLhHqrWJBHypc\\\\\\\/icWTVKOPrjEwclZgnt+0s7mo0vLeBVpKuClEJNKky3O3B4mcCwVwatFdYQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022m+10iKIYcbYkj69NdKReJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221099d08c29007eb8e7f278eef60c3cf64d30ea14ce17523266c5988eb89c0d45\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ejvtphisIbX4cR+TRiVoU7\\\\\\\/2Tv\\\\\\\/EhXGqamrU\\\\\\\/4CDbdDhYgIlpHTj1GxBcvvYZffuPoVfO6XyRVlRtB5gSmY4yf2ZSkNz6b1r+Xey0kbMT4pkmnO6AcrDRBw8UtT32ZdRmbW\\\\\\\/niCKfC45Y2MQuqgr4+VG81\\\\\\\/7Fv\\\\\\\/Pmd1vrEzFrJfB95Lo73Cr6k+fBzi6x3aY8GqCnPm6NHVb8jLmvFJ7G0\\\\\\\/6CQSBYiT+nS7A0G9UGZ8KV2BeUKESvNOifUpHM8SLfCSaIY7pD5MVw8hrvgNU4ryhUAcyxs7Pz+4gQvUyIY+mcKYCdC8UUHCYvbKeov50kBn83CkMvaiYa67Y79tHBgxu78waVzKurmPSHwz+xEjNjiP9RhLz3Fmtxr+SvfFp4eENA1\\\\\\\/gFF6RmlqWtCRDBd9M\\\\\\\/hCHhmde1Yr1jC6rbG9qJOqGma7iKuLak7H58wRUjor22gg0h74mNhuowcLf9XAkDiae+iO+\\\\\\\/xEXuPFGKUPBDURk93PGY31vWC5jT1igO1O+F3lGkOFl3pDdK4yzwdZhcI6iplZhAL0hKtKaV2vao0Mg9VQhxEJ6JebjKsAjglCkGvffP7MO7lURi+VIhbMraWOMtC0IvcvOYhDUXB+yygbnqpTHbGXOKR+Jy4ZKPBB758reIQL3HFjV5B+xwwF0PkoE8Ijeaq7uwjBKwL8+sNMaczF6sq0EeWyzWomH8j0fXDL3CeFZkHmiwRthe8ofJBkHsWauh8xzjH+962fZeFPEQg3fPx29t6dmCw+GV4LzWcxNxZiy\\\\\\\/gtMZRjXgtdyhqdWt\\\\\\\/mSFO6+yDhyf9aQB+RYjdzdaaCuFywGCGLoEqz18KZAhGJ87uCf3EusbAi72u+AjYRYHKOhcWvgXN7stardZ6AjjUpVHsjplqsfhsJ9l2M7YuBuudUc\\\\\\\/EfSgWj4FYOQtOPOwV4Gc2bKdj5nFjNmbi\\\\\\\/DHBCDHOgf9\\\\\\\/Ifh0Udw7vZeu6IH8HOu813FG2LRVmw6S5j5o+bxNBM5TNpm5+RQ3Tm7kzA2bQ+kvR6cGgPchtElpN\\\\\\\/L\\\\\\\/fYhc4iaYy36+59Np9d1SeKqVZG0sfklOgbcZBnDeY3bZssoly\\\\\\\/Loju6QHqDjwcFulkp+oatq0G0wV+2NDIQZH4frZWlyC9L8ixp4+63FZrJUy\\\\\\\/pYtNy4worTbW7ODjvmTAlXKOzPH5\\\\\\\/Amql7bpvFCmCisdF8lqJ63YfLTdA\\\\\\\/gHW67rXS\\\\\\\/FPh2zdB3PKXGflxNdGIet1k31XQEn8iF2ppW0OSH3XRiIwGWcHwAoQ6fGiHDsmoRvLtNos3ju6523lW7AkPkzGp6ywPNnkRDwZMUlC44FVrKqqbiuzkS8L6mIxigH5BsSx5\\\\\\\/jQLRa4MW13plEG5A5J44PMA3z0KYDTDHkVrZ+eE4QuDe2hlBauacMsJefdgC3kLfpLmTcKsnfYsgXFKXn9uAs4TQIehp\\\\\\\/3hNwgRPdgEuJODS85ehMhGDg2ZBDD+jEC\\\\\\\/qFqvJkeMyJC1Bj9jRSBA1UkuseR1Rhv4zS94ZhgZUBMzac9K\\\\\\\/tVDMy27N8igiuEJGMx7Cmeh4kT293u0v3ai2lOl+EibE\\\\\\\/jViWE7ihh8mfBzHLFIFCEf9d7hOMFGWNnd0x2ctp4pn44mkt5tcgquZQzwrZIl6umclz3a4sIYBgGAW\\\\\\\/SFQd\\\\\\\/lQhEKPUD80OpZEnvr9iebSVTjZ\\\\\\\/oChXwaJcpahsGXGEpCoeWJuMAUkTF2rkd86cuJa7iPxqVDe1XCzqjJNcSoI+mk1iMK5rKH41uubP8RX0wsWSbbF+UuJCAhPE2HKkpH3MpabXvHcCbJMc0S\\\\\\\/qZ9kS63zivCgr9LWa1FLtLOMPYjPwH58gm\\\\\\\/akbQDNtB1RfQsWs0X2EKB+amhPj6PgCjr4o6017UMHqIALpLCPqBxJhUiMh81T2\\\\\\\/LG6IUFAsZ1uiSPCa\\\\\\\/l2KNg32rrTLZVvdDhgnLRpS4s6vjIcKS5TFOh49554KkC5fXjB78p3s0Ysv4Yc4H\\\\\\\/5knTof5PHsryf7sX+x453U39rQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022aBUVlJSJFjHcFfP2eydGgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022887f1ea22ac2a7c9b48422e1e2f40b9e06f239b27826ec626663d52e09b5bffe\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493102 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1497" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XbMLQZpwNdaOF4ZMzYfo4NcEbmIGzaSA32Fa1Z63MWM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1497" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SWgrp-05Dgb6JHULv3A8izcpQ5R7p3yrz3kVXcN692M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1487" title="The Neurospora crassa Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1507" title="Small RNA-Seq Analysis Reveals miRNA Expression Dynamics Across Tissues in the Malaria Vector, Anopheles gambiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493102" id="node493102" data-pisa="ggg;9/5/1497" data-pisa-master="ggg;g3.118.200862" data-apath="/ggg/9/5/1497.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessment of Genetic Differentiation and Linkage Disequilibrium in <em>Solanum pimpinellifolium</em> Using Genome-Wide High-Density SNP Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ya-Ping Lin</span>, <span class="highwire-citation-author" data-delta="1">Chu-Yin Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6211-0198" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kai-Yi Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1497-1505;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200862 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493102" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ya-Ping Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ya-Ping%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYa-Ping%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chu-Yin Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chu-Yin%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChu-Yin%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kai-Yi Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai-Yi%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKai-Yi%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6211-0198" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kai-Yi Chen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kaiychen@ntu.edu.tw" class="" data-icon-position="" data-hide-link-title="0">kaiychen@ntu.edu.tw</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1497" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493102" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493102/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1497.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493102" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493102/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1497.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493102" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493102/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1497.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493102" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493102/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">To mine new favorable alleles for tomato breeding, we investigated the feasibility of utilizing <em>Solanum pimpinellifolium</em> as a diverse panel of genome-wide association study through the restriction site-associated DNA sequencing technique. Previous attempts to conduct genome-wide association studies using <em>S. pimpinellifolium</em> were impeded by an inability to correct for population stratification and by lack of high-density markers to address the issue of rapid linkage disequilibrium decay. In the current study, a set of 24,330 SNPs was identified using 99 <em>S. pimpinellifolium</em> accessions from the Tomato Genetic Resource Center. Approximately 84% of <em>Pst</em>I site-associated DNA sequencing regions were located in the euchromatic regions, resulting in the tagging of most SNPs on or near genes. Our genotypic data suggested that <em>S. pimpinellifolium</em> were divided into three single-ancestry subpopulations and four mixed-ancestry subpopulations. Additionally, our SNP genotypic data consistently confirmed the genetic differentiation, achieving a relatively reliable correction of population stratification. Previous studies utilized the 8K tomato SNP array, SolCAP, to investigate the genetic variation of <em>S. pimpinellifolium</em> and we performed a meta-analysis of these genotypes. The result suggested SolCAP array was less appropriate to profile the genetic differentiation of <em>S. pimpinellifolium</em> when more accessions were involved because the samples belonging to the same accession demonstrated different genome patterns. Moreover, as expected, rapid linkage disequilibrium decay was observed in <em>S. pimpinellifolium</em>, especially in euchromatic regions. Approximately two-thirds of the flanking SNP markers did not display linkage disequilibrium based on <em>r<sup>2</sup></em> = 0.1. However, the 18-Kb linkage disequilibrium decay indeed reveals the potential of single-gene resolution in GWAS when markers are saturated.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/solanum-pimpinellifolium" class="hw-term hw-article-keyword hw-article-keyword-solanum-pimpinellifolium" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Solanum pimpinellifolium</em></a></li><li class="kwd"><a href="/keyword/genetic-differentiation" class="hw-term hw-article-keyword hw-article-keyword-genetic-differentiation" rel="nofollow">genetic differentiation</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/radseq" class="hw-term hw-article-keyword hw-article-keyword-radseq" rel="nofollow">RADseq</a></li></ul><p id="p-3">The wild tomato species <em>Solanum pimpinellifolium</em> is a native perennial shrub in Ecuador and Peru, ranging along the western Andean slopes to the coastal regions. It is believed that <em>S. pimpinellifolium</em> originated in northern Peru and then diversified into several subpopulations after it migrated to Ecuador and southern Peru (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rick <em>et al.</em> 1977</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zuriaga <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blanca <em>et al.</em> 2012</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2015</a>). These regions present gradient temperature and precipitation changes from Ecuador toward southern Peru: western Ecuador is equatorial winter dry; northern Peru is hot, arid desert; southern Peru is cold, barren desert (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kottek <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Moyle 2008</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Zuriaga <em>et al.</em> 2009</a>). Previous studies showed high genetic variation and high outcrossing rate of the accessions in northern Peru and revealed genetic differentiation between the accessions collected in Ecuador and those in Peru (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Rick <em>et al.</em> 1977</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Caicedo and Schaal 2004</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Zuriaga <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Rao <em>et al.</em> 2012</a>). Recently, with the aid of SolCAP genotyping array, <em>S. pimpinellifolium</em> was divided into three subpopulations: one in northern Ecuador, one in the mountains of Ecuador extending to the north of Peru, and one in Peru (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Blanca <em>et al.</em> 2012</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">2015</a>). Because geographic distributions of distinct <em>S. pimpinellifolium</em> subpopulations also aligned from north to south, the genetic distances between subpopulations were thought to correlate with climatic differences (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Zuriaga <em>et al.</em> 2009</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Blanca <em>et al.</em> 2012</a>, <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">2015</a>).</p><p id="p-4"><em>S. pimpinellifolium</em> is an attractive resource for tomato breeding because it can freely cross with cultivated tomatoes and introduces novel alleles into the limited gene pool of cultivated tomatoes (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tanksley and Mccouch 1997</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Spooner <em>et al.</em> 2005</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Moyle 2008</a>). <em>S. pimpinellifolium</em> has been used as a genetic resource for disease resistance and fruit quality traits in tomato breeding (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Grandillo <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Víquez-Zamora <em>et al.</em> 2014</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Capel <em>et al.</em> 2015</a>). A core collection of <em>S. pimpinellifolium</em> was developed at World Vegetable Center for preservation and utilization (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Rao <em>et al.</em> 2012</a>). This core collection has been used to mine novel alleles of salt tolerance via the candidate gene approach (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rao <em>et al.</em> 2015</a>).</p><p id="p-5">A comprehensive way to mine new favorable alleles in wild species is to conduct a genome-wide association study (GWAS). GWAS utilizes linkage disequilibrium (LD), the non-random association between marker alleles and alleles conferring targeted phenotypes in a given collection of germplasm, to map quantitative trait loci (QTL) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Soto-Cerda and Cloutier 2012</a>). In comparison with the traditional genetic mapping method using progenies derived from a bi-parental cross, GWAS usually brings higher mapping resolution because of more detectable recombinant events in a collection of germplasm. Previous studies revealed the average ranges of LD decay in different collections of cultivated tomatoes varied from 6.1 to 12.5 cM based on <em>r<sup>2</sup></em> = 0.2 and 13.4 cM based on <em>r<sup>2</sup></em> = 0.1 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sim <em>et al.</em> 2012a</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Blanca <em>et al.</em> 2015</a>). Because <em>S. pimpinellifolium</em> presents greater genetic variation than cultivated tomatoes, the range of LD decay is expectedly smaller in <em>S. pimpinellifolium</em> populations (1.7 cM based on <em>r<sup>2</sup></em> = 0.1) (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Blanca <em>et al.</em> 2012</a>, <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ranc <em>et al.</em> 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bauchet <em>et al.</em> 2017</a>). However, the 8K SolCAP genotyping array and additional 6K CBSG genotyping array did not achieve full LD coverage across all chromosomal regions for <em>S. pimpinellifolium</em> accessions (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sim <em>et al.</em> 2012a</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2012b</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bauchet <em>et al.</em> 2017</a>). The restriction site-associated DNA sequencing (RADseq) technique may provide an inexpensive solution to address this challenge (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davey and Blaxter 2010</a>).</p><p id="p-6">The RADseq technique limits sequencing resources at the vicinity of restriction enzyme cutting sites and therefore provides flexibility of experimental design regarding the trade-off between cost-effectiveness and marker densities (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bhakta <em>et al.</em> 2015</a>). <em>Pst</em>I is a methylation-sensitive restriction enzyme and recognizes the sequences “CTGCAG” (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dobritsa and Dobritsa 1980</a>). A study of the genome-wide methylation pattern in tomato leaves and immature fruits revealed that the gene-rich euchromatic regions at the distal ends of chromosomes were characterized as the regions with low levels of cytosine methylation at the “CG”, “CTG”, and “CAG” sequences and the pericentromeric heterochromatin regions were the regions with high levels of cytosine methylation (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhong <em>et al.</em> 2013</a>). DNA markers would be found mainly in the gene-rich euchromatic regions and sparsely in the heavily methylated heterochromatic regions when extracted genomic DNAs come from young tomato leaves and are digested with <em>Pst</em>I following the RADseq protocol. Because the euchromatin usually has a higher frequency of genetic recombination than the heterochromatin, this RADseq experimental design based on <em>Pst</em>I digestion could increase marker density in the chromosomal region with higher frequencies of genetic recombination and decrease marker density in the chromosomal region with lower frequencies of genetic recombination. This RADseq design may fulfill the demand of the high-density markers in GWAS using a <em>S. pimpinellifolium</em> collection.</p><p id="p-7">The objective of the current study was first to develop genome-wide high-density SNP markers for a subset of <em>S. pimpinellifolium</em> collections from the Tomato Genetic Resource Center (TGRC) through the RADseq approach. Second, the population differentiation was examined by different methods to ensure a stable estimation. Moreover, a meta-analysis of SolCAP array was performed to infer the population differentiation in a scenario involving in more accessions. Third, the LD decay was assessed to estimate the required marker number and potential resolution in GWAS.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-8">All plant materials and their information were obtained from TGRC (Table S1; <a href="http://tgrc.ucdavis.edu/">http://tgrc.ucdavis.edu/</a>). A total of 12 accessions from Ecuador and 87 accessions from Peru were utilized in this study. According to their mating types, 43 accessions were facultative self-compatible (FSC), and 56 accessions were autogamous self-compatible (ASC). Seeds were propagated by self-pollination for two generations using the method of single-seed descent in a greenhouse. Young leaves collected from plants of these single-seed descendent seeds were used for DNA extraction.</p></div><div id="sec-3" class="subsection"><h3>RAD sequencing</h3><p id="p-9">Total genomic DNA was extracted from young leaves using a modified CTAB method (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fulton <em>et al.</em> 1995</a>) and purified with a DNeasy Blood &amp; Tissue Kit (QIAGEN, Venlo, Netherland) following the manufacturer’s instructions. We chose <em>Pst</em>I to select the sequencing regions because <em>Pst</em>I is a methylation-sensitive restriction enzyme and it may cut more frequently in euchromatin regions than heterochromatin regions (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Dobritsa and Dobritsa 1980</a>). <em>Pst</em>I-digested DNA libraries were prepared following the protocol of Etter <em>et al.</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Etter <em>et al.</em> 2011</a>). Four RADseq libraries were constructed, and each was sequenced in one lane of an Illumina HiSeq2000 flow cell (100 bp single-end reads) (Illumina Inc., San Diego, CA, USA). All the sequences of RADseq were submitted to the NCBI SRA database, and the BioProject Number is PRJNA358110.</p></div><div id="sec-4" class="subsection"><h3>SNP calling</h3><p id="p-10">Reads were analyzed with Stacks version 1.37 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Catchen <em>et al.</em> 2013</a>) and with CLC Genomics Workbench software version 6.5.1 (QIAGEN, Venlo, Netherlands). First, the <em>process_radtags</em> command in Stacks filtered out low-quality reads with Q scores less than 20. The remaining reads were mapped to the tomato reference genome SL2.50 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fernandez-Pozo <em>et al.</em> 2015</a>) using the “Map Reads to Reference” tool in the CLC Genomics Workbench software. Considering that genetic variation between the tomato reference genome <em>S. lycopersicum</em> and <em>S. pimpinellifolium</em> is larger than genetic variation within <em>S. lycopersicum</em>, mapping parameters were set as 0.5 for the length fraction and 0.9 for the similarity fraction. The reads of the same individual in different lanes were merged. In the subsequent analyses using Stacks, the <em>ref_map.pl</em> command set the parameter –m (minimum read depth to create a stack) as 10, and the <em>populations</em> command set the parameter –p (minimum number of populations a locus must be present) as 75. SNPs with a minor allele frequency of less than 0.05 were further excluded, and a set of 24,330 SNP markers was obtained. This set of 24,330 SNP markers was utilized for the analyses of genetic variation, LD, <em>F<sub>st</sub></em> and AMOVA. Another SNP set without ‘redundant SNP markers’ was used to conduct the principal component analysis (PCA) and ADMIXTURE because these two matrices are expected to correct the structure in GWAS. To remove ‘redundant SNP markers’, we defined a sequencing unit as a sequencing region surrounding a <em>Pst</em>I site, usually 186 bp long, which has at least one SNP with a minor allele frequency greater than 0.05 in the <em>S. pimpinellifolium</em> population. If more than one SNPs are located in a sequencing unit and they are in complete LD (<em>r<sup>2</sup></em> = 1), only the first SNP is kept. This process resulted in a total of 19,993 SNP markers. ITAG2.4 gene model from SGN was used as the reference gene annotation.</p></div><div id="sec-5" class="subsection"><h3>Population differentiation</h3><p id="p-11">PCA was performed in TASSEL5.0 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bradbury <em>et al.</em> 2007</a>). ADMIXTURE was completed following by the manual; the best K was determined following the procedure of cross-validation in the manual (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>). Pairwise <em>F<sub>st</sub></em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Weir and Cockerham 1984</a>) and analysis of molecular variance (AMOVA) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Excoffier <em>et al.</em> 1992</a>) were conducted in the R package StAMPP (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pembleton <em>et al.</em> 2013</a>). Details of the analysis of isolation by distance and the meta-analysis of <em>S. pimpinellifolium</em> SolCAP genotyping data are described in the supplements, File S1, and File S2, respectively.</p></div><div id="sec-6" class="subsection"><h3>Estimation of genetic variation and LD</h3><p id="p-12">Genetic variation within overall accessions and within each of the seven groups was assessed based on observed heterozygosity and the within-population gene diversity (expected heterozygosity) using the R package hierfstat (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Goudet and Jombart 2015</a>). Pairwise <em>r<sup>2</sup></em> values between SNP markers were calculated to assess overall extent of LD via plink1.9 within a 1-Mb window (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gaunt <em>et al.</em> 2007</a>) and fit by non-linear regression (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Remington <em>et al.</em> 2001</a>). The baseline of the <em>r<sup>2</sup></em> value was set at 0.1 (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bauchet <em>et al.</em> 2017</a>). The local LD along each chromosome was assessed as follows: for each pair of consecutive sequencing units (defined in the section of SNP calling), the average <em>r<sup>2</sup></em> was calculated between two SNPs in different sequencing units and plotted along the left <em>Pst</em>I cutting site based on the physical position. The heterochromatin regions were marked according to the genetic map of EXPIM 2012 and the physical map of the tomato reference genome (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Sim <em>et al.</em> 2012b</a>).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-13">Data available: NCBI SRA BioProject Number PRJNA358110. The authors affirm that all data necessary for confirming the conclusions of this article are represented fully within the article and its tables and figures. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7730369">https://doi.org/10.25387/g3.7730369</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Identification of 24,330 SNPs From PstI-digested DNA libraries</h3><p id="p-14">A total of 655,973,270 short DNA reads were obtained from four lanes of the Illumina HiSeq2000 flow cell and were divided into 99 parts according to barcode sequences. Each part was derived from the DNA of a <em>S. pimpinellifolium</em> accession and contained at least 3.7 million DNA reads, except for LA2647 (Table S1). Among the 82,814 <em>Pst</em>I sites in the tomato reference sequence SL2.50, only 23,988 <em>Pst</em>I sites were covered by the sequenced DNA reads (Table S2). The sequenced regions included 0.54% of the SL2.50 reference sequences and 12,790 annotated genes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Interestingly, approximately 84% of the sequenced <em>Pst</em>I sites were located in the euchromatic regions (Table S2). Besides, the proportion of sequenced genes in euchromatin (43.13%) were about twice as that in heterochromatin (19.75%) (Table S2).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493088/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493088/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the markers developed with the RAD sequencing strategy and the sequenced genes as well</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">Two criteria were set to ensure the accuracy of SNP calling and genotype calling: one was that the read depth aligning to the reference sequence was equal to or greater than 10, and the other was that at least 75% of the accessions showed genotypes associated with a defined SNP marker. A total of 67,804 SNPs were identified in the sequenced regions of 99 <em>S. pimpinellifolium</em> accessions, and 24,330 of them had the minor allele frequency higher than 0.05. In the genotypic dataset of the 24,330 SNP markers (Table S3), the missing proportion of each accession ranged from 0.72 to 15.92%, except for LA2647 of which the value was 65.68% due to a low number of sequencing reads (Table S1). Regarding the location of these 24,330 SNPs, 16,365 SNPs were found in 7,383 annotated genes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), and the remaining SNPs were in the intergenic regions. Concerning the proportion of sequenced <em>Pst</em>I sites that contained SNPs, there is no significant difference between those sites in euchromatin (68.85%) and those in heterochromatin (60.59%) (Table S2). Meanwhile, the genotypic data of the LA0411 accession was dropped because the observed heterozygosity of LA0411 was inconsistent with its mating type (Table S1).</p></div><div id="sec-10" class="subsection"><h3>A similar distribution between genes and SNPs was identified in the vicinity of PstI cutting site throughout the genome</h3><p id="p-16">The observation that 67.26% (16,365 to 24,330) of the SNPs were located in the annotated gene regions (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) implied a correlation between the distribution of the identified SNPs in the current study and the distribution of the annotated genes. Additional observations in the current study indicated a preference for genomic DNA digestion by the <em>Pst</em>I restriction enzyme in the euchromatic regions: only 28.97% (23,988 to 82,814) of <em>Pst</em>I sites were found in the deep sequencing regions, and 83.55% (20,043 to 23,988) of the deep sequencing regions were located in the euchromatic region (Table S2). It is worth noting that the current RADseq protocol did produce low coverage of sequencing reads in some <em>Pst</em>I sites (with a read depth less than 10), and these <em>Pst</em>I sites were filtered by the criteria of SNP and genotype calling; therefore, the deep sequencing regions indicated that their read depths were no less than 10. Incidentally, because SNPs can be identified only in the sequenced regions, it is a reasonable deduction that most SNPs found in the current study are located in the euchromatic regions. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> confirms clearly that the annotated tomato genes (A layer), the <em>Pst</em>I sites in the deep sequencing regions (C layer), and identified SNPs (D layer) are mainly located in the euchromatic regions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distributions of ITAG2.4 gene model, PstI cutting sites and SNPs throughout the genome. Each section indicates one chromosome, with labeling on the circumference. Circles A, B, C, and D indicate the distribution of ITAG2.4 genes, expected PstI cutting sites, PstI cutting sites in the deep sequencing regions and RADseq SNPs, respectively. The black lines in the inner D layer indicate the heterochromatic regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-716852792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distributions of ITAG2.4 gene model, &lt;em&gt;Pst&lt;/em&gt;I cutting sites and SNPs throughout the genome. Each section indicates one chromosome, with labeling on the circumference. Circles A, B, C, and D indicate the distribution of ITAG2.4 genes, expected &lt;em&gt;Pst&lt;/em&gt;I cutting sites, &lt;em&gt;Pst&lt;/em&gt;I cutting sites in the deep sequencing regions and RADseq SNPs, respectively. The black lines in the inner D layer indicate the heterochromatic regions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1497/F1.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1497/F1.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493042" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">The distributions of ITAG2.4 gene model, <em>Pst</em>I cutting sites and SNPs throughout the genome. Each section indicates one chromosome, with labeling on the circumference. Circles A, B, C, and D indicate the distribution of ITAG2.4 genes, expected <em>Pst</em>I cutting sites, <em>Pst</em>I cutting sites in the deep sequencing regions and RADseq SNPs, respectively. The black lines in the inner D layer indicate the heterochromatic regions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Genetic differentiation of S. pimpinellifolium corresponded to the geographic area</h3><p id="p-18">The collection of 98 <em>S. pimpinellifolium</em> accessions was divided into three single-ancestry subpopulations and four mixed-ancestry subpopulations by the ADMIXTURE software (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and Figure S1). We named the red, blue, and green single-ancestry subpopulations POP S1, POP S2, and POP S3, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Meanwhile, the red-blue, blue-green, red-green, and red-blue-green mixed-ancestry subpopulations were named as POP M1, POP M2, POP M3, and POP M4, respectively (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). POP S1, POP S2, and POP S3 were clustered separately in the PCA plot, in which the first and the second principal components counted for 16.04% and 8.00% of the variance, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Moreover, pairwise <em>F<sub>st</sub></em> confirmed the genetic differentiation (Table S4), and AMOVA revealed that the variance between subpopulations was 41.96% (p-value &lt; 0.001).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ancestry and geographic distribution of 98 Solanum pimpinellifolium accessions from the Tomato Genetics Resource Center. A) Model-based ancestry for each accession. B) Principle component analysis of the S. pimpinellifolium population. C) Geographical distribution of the 98 S. pimpinellifolium accessions. Symbol and color codes are as follows: square symbols with red, blue and green colors indicate three single-ancestry subpopulations corresponding to the same colors in the ancestry plot (POP S1, POP S2 and POP S3, respectively); triangle symbols with purple, aquamarine and goldenrod colors present the POP M1, POP M2 and POP M3, respectively; black circle symbols were the POP M4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-716852792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ancestry and geographic distribution of 98 &lt;em&gt;Solanum pimpinellifolium&lt;/em&gt; accessions from the Tomato Genetics Resource Center. A) Model-based ancestry for each accession. B) Principle component analysis of the &lt;em&gt;S. pimpinellifolium&lt;/em&gt; population. C) Geographical distribution of the 98 &lt;em&gt;S. pimpinellifolium&lt;/em&gt; accessions. Symbol and color codes are as follows: square symbols with red, blue and green colors indicate three single-ancestry subpopulations corresponding to the same colors in the ancestry plot (POP S1, POP S2 and POP S3, respectively); triangle symbols with purple, aquamarine and goldenrod colors present the POP M1, POP M2 and POP M3, respectively; black circle symbols were the POP M4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1497/F2.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1497/F2.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492893" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Ancestry and geographic distribution of 98 <em>Solanum pimpinellifolium</em> accessions from the Tomato Genetics Resource Center. A) Model-based ancestry for each accession. B) Principle component analysis of the <em>S. pimpinellifolium</em> population. C) Geographical distribution of the 98 <em>S. pimpinellifolium</em> accessions. Symbol and color codes are as follows: square symbols with red, blue and green colors indicate three single-ancestry subpopulations corresponding to the same colors in the ancestry plot (POP S1, POP S2 and POP S3, respectively); triangle symbols with purple, aquamarine and goldenrod colors present the POP M1, POP M2 and POP M3, respectively; black circle symbols were the POP M4.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493146/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493146/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genetic variation of each subpopulation</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">The within-population gene diversity was calculated to compare genetic variation within each subpopulation. POP S2 and POP M1 showed the highest genetic variation among the single-ancestry subpopulations and the mixed-ancestry subpopulations, respectively (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Both subpopulations were in northern Peru, which indicated that northern Peru is the origin of <em>S. pimpinellifolium</em>.</p><p id="p-24">Interestingly, most accessions in the same subpopulation were in the same vicinity of their collection sites (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). Also, POP S1, POP S2, and POP S3 spread in somewhat distinct geographic areas along the coastline from Ecuador to southern Peru (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). The geographic distribution of these subpopulations appeared in the following order from north to south: POP S1, POP M1, POP S2, POP M2, and POP S3 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). This geographic distribution showed a trend in which the mixed-ancestry subpopulations were located between their corresponding single-ancestry subpopulations.</p></div><div id="sec-12" class="subsection"><h3>Rapid LD decay</h3><p id="p-25">LD decay was estimated for the mapping resolution in GWAS. In this population, the non-linear regression curve dropped very quickly (Figure S2). Following the non-linear regression curve, the overall LD decay was within 18 Kb when the baseline of the <em>r<sup>2</sup></em> value was set at 0.1 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The fastest LD decay was within 10 Kb on chromosome 9 while the slowest LD decay was within 30 Kb on chromosome 4 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and Figure S3).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493039/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493039/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The local LD profiles of individual chromosomes</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visualization for LD. A) The 50 Kb interval of overall LD decay. The red curve indicates non-linear regression, and the black dotted line indicates the baseline of r2 at 0.1. B) The local LD of chromosome 1. The red dotted line indicates the baseline of r2, and the orange line indicated the heterochromatic region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-716852792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Visualization for LD. A) The 50 Kb interval of overall LD decay. The red curve indicates non-linear regression, and the black dotted line indicates the baseline of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; at 0.1. B) The local LD of chromosome 1. The red dotted line indicates the baseline of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;, and the orange line indicated the heterochromatic region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1497/F3.medium.gif" width="440" height="371"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1497/F3.medium.gif" width="440" height="371"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1497/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492973" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Visualization for LD. A) The 50 Kb interval of overall LD decay. The red curve indicates non-linear regression, and the black dotted line indicates the baseline of <em>r<sup>2</sup></em> at 0.1. B) The local LD of chromosome 1. The red dotted line indicates the baseline of <em>r<sup>2</sup></em>, and the orange line indicated the heterochromatic region.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Heterogeneity of genetic recombination within each chromosome</h3><p id="p-27">LD decay of individual chromosomes was insufficient to capture the local variations of historically accumulated recombination events because the tomato genome comprises more than 75% heterochromatin which usually suppresses recombination events (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Sim <em>et al.</em> 2012a</a>). We assessed the local LD profile of individual chromosomes based on the average <em>r<sup>2</sup></em> value of flanking sequencing units that contained at least one SNP marker. We observed two main trends: marker density in the heterochromatic regions was lower than that in the euchromatic regions (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> and Figure S4), and approximately two-thirds of the <em>r<sup>2</sup></em> values were less than 0.1 (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). The latter observation indicated that these flanking SNP markers were not in a state of linkage disequilibrium.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><div id="sec-15" class="subsection"><h3>Subpopulations clustering from north to south are expected due to the high correlation between genetic distance and geographic distance</h3><p id="p-28">The genetic differentiation revealed in this study should be similar to previous findings because the collection sites of this collection cover most of recorded habitats of <em>S. pimpinellifolium</em>. One previous study for the genetic diversity of <em>S. pimpinellifolium</em> assessed 213 accessions with the genotypes of 10 SSR markers. It suggested the existence of Peruvian and Ecuadorian subpopulations (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Zuriaga <em>et al.</em> 2009</a>). Another study investigated a collection of 190 <em>S. pimpinellifolium</em> accessions using 48 SSR markers (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Rao <em>et al.</em> 2012</a>). It evaluated 120 accessions collected from Peru and 31 accessions from Ecuador, and divided these accessions into two single-ancestry subpopulations and one mixed-ancestry subpopulation. One of the single-ancestry subpopulation contained 93 accessions from Peru and 3 Ecuadorian accessions. These three Ecuadorian accessions were the only Ecuadorian accessions that were grouped into this single-ancestry subpopulation that contained mainly the Peruvian accessions, and the duplicated entries with the same names of these Ecuadorian accessions (LA0411, LA1246, LA1261) in the same study were grouped into the other two subpopulations. Despite of these three confounded Ecuadorian accessions, this study still inferred strong correlation between genetic diversity and geographic distance between Peruvian and Ecuadorian subpopulations (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Rao <em>et al.</em> 2012</a>). With the aid of SolCAP array, two consecutive studies, one with 63 <em>S. pimpinellifolium</em> accessions and the other with 112 <em>S. pimpinellifolium</em> accessions, sorted <em>S. pimpinellifolium</em> into three subpopulations: one in northern Ecuador, another in the mountainous area from southern Ecuador extending to northern Peru and the third in the low-altitude areas of Peru (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Blanca <em>et al.</em> 2012</a>, <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">2015</a>). Our study also supports three single-ancestry subpopulations: one in Ecuador, one in northern Peru, and another in southern Peru. Among all the aforementioned studies, two ancestry subpopulations are confident: one includes the accessions in Ecuador; the other includes the accessions in southern Peru. The different grouping among these studies for those accessions from southern Ecuador to northern Peru may result from different markers and different genetic diversity in each study.</p><p id="p-29">Previous studies suggested that genetic differentiation of <em>S. pimpinellifolium</em> correlated to the climatic variation (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Rick <em>et al.</em> 1977</a>; <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Zuriaga <em>et al.</em> 2009</a>; <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Blanca <em>et al.</em> 2012</a>, <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">2015</a>). The analysis of genetic differentiation based on the RADseq data in the current study supported the same conclusion: most POP S1 accessions are in hot and humid Ecuador; most POP M1 scatter in northern Peru, along the western Andean slopes, in which is a warm desert; most POP S2 are located in the Andean Mountains; most POP M2 are in a warm semi-arid region; most POP S3 spread along the coastal region from central to southern Peru, in which is a relatively cold desert (Table S1 and <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>). Since these subpopulations are located in the environments with different climates, and <em>F<sub>st</sub></em> as well as AMOVA support these subpopulations (Table S4), the genetic differentiation of <em>S. pimpinellifolium</em> is observed evidently with the aid of RADseq SNP markers.</p><p id="p-30">Isolation by distance (IBD) is a common tool to access genetic differentiation that expect a positive correlation between genetic variation and geographic distance (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wright 1943</a>). We conducted this analysis for two datasets, the RADseq data and the SolCAP meta-data (File S1 and File S2), and made comparisons. The former data had the correlation coefficient equal to 0.34, and the latter one was 0.55 (Figure S5 and Figure S6). It seems that the RADseq data showed less genetic differentiation than the SolCAP meta-data. However, it has been argued that IBD test can be severely biased in two situations: unequal migration among all populations in a system, and the detection of loci under selection (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Meirmans 2012</a>). We do not know whether the investigated accessions were equally migrated, but we do know that the SolCAP array was designed mainly on the SNP sites of coding sequences within cultivated tomatoes or between cultivated tomato and wild tomatoes (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Sim <em>et al.</em> 2012b</a>). Therefore, the SNPs on the SolCAP array had higher chances under selection in domestication. Under this premise, the comparisons of the IBD test between the RADseq data and the SolCAP array data could be confounded by the differences in selection strength.</p></div><div id="sec-16" class="subsection"><h3>Discrepancy of genetic clustering in SolCAP meta-analysis</h3><p id="p-31">Our meta-analysis concluded that the genetic compositions of 214 samples came from 15 ancestry populations. This conclusion is different from the conclusion of <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Blanca <em>et al.</em> (2012)</a> and our RADseq data, both of which suggested that there were three ancestry populations of <em>S. pimpinellifolium</em>. This meta-analysis implied an unclear structure; especially the cross validation error has an ambiguous minimal value (Figure S7). It is possible that genetic diversity between wild tomatoes are underestimated because the polymorphisms of SolCAP array are selected between cultivars and wild tomatoes (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Sim <em>et al.</em> 2012b</a>). We noticed that two samples of LA0373 with 76% identity display different genome patterns in ADMIXTURE, while two samples of LA1478 with 71% identity present different patterns as well (Table S7 and Figure S8). Since two samples of the same accession demonstrate dissimilar genome patterns, the SolCAP may be less appropriate to quantize the population structure of <em>S. pimpinellifolium</em>, especially when more samples are involved. Also, for the same reason, we cannot validate the genetic differentiation in the SolCAP meta-analysis by <em>F<sub>st</sub></em> or AMOVA nor achieve a stable estimation of genetic differentiation in a scenario of more accessions via the SolCAP meta-analysis.</p></div><div id="sec-17" class="subsection"><h3>More markers are required to cover through the genome of S. pimpinellifolium</h3><p id="p-32">The observed and expected heterozygosity of this population were 0.0761 and 0.2786, respectively, slightly higher than those in previous researches (<a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Blanca <em>et al.</em> 2012</a>, <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">2015</a>). Since <em>S. pimpinellifolium</em> was detected with up to a 40% outcrossing rate (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Rick <em>et al.</em> 1977</a>) and demonstrated high genetic variation, it is expected to cause rapid LD decay. In this study, LD decay was within 18 Kb throughout the genome, which was much shorter than cultivated tomatoes (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Sim <em>et al.</em> 2012a</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bauchet <em>et al.</em> 2017</a>). However, to put at least one SNP marker within each of 18 Kb intervals in this genome, the 900-Mb tomato genome would require at least 50,000 markers to fulfill QTL detection in GWAS. Therefore, acquiring many SNPs using different methods is essential to conduct a GWAS in the <em>S. pimpinellifolium</em> population. Here, we proposed three possible approaches to increase markers. One is to increase the sample size evenly for each subpopulation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brachi <em>et al.</em> 2011</a>). Since approximately 64% of alleles were rare in this population, the augmentation of the subpopulation size may adjust rare alleles to common alleles, potentially increasing the SNPs without extending coverage. One is to construct DNA libraries with a frequently cutting restriction enzyme. This approach can be simulated and optimized <em>in silico</em> to balance sequencing resource between sample sizes and sequencing coverage (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Shirasawa <em>et al.</em> 2016</a>). Another is exome sequencing, a selective genome sequencing technology that selects desired sequencing regions by the hybridization of designed probes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kaur and Gaikwad 2017</a>). Based on tomato genome sequence information, such as the gene model or EST database, one could design different sets of probes to limit sequencing regions (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ruggieri <em>et al.</em> 2017</a>). Given the approximately 110 Mb total gene length in the ITAG2.4 gene model, the potential coverage could reach 12% and all target the gene region. This exome sequencing strategy may be able to increase SNPs without increasing population size.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">We thank the C.M. Rick Tomato Genetics Resource Center for generously providing plant materials to conduct this experiment and Dr. Tze-Tze Liu in the Genome Research Center at the National Yang-Ming University, Taiwan, for providing help and advice of the massive parallel DNA sequencing task. This work was supported by grants from the Ministry of Science and Technology, Taiwan R.O.C. (Project Nos. NSC 96-2313-B-002-034-MY3 and NSC 101-2313-B-002-006-MY3) to KYC and from National Taiwan University (Project Nos. 103R7858 and 104R7858) to KYC. This work represents the partial fulfillment of requirements for YPL’s Ph.D. degree.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-34">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7730369">https://doi.org/10.25387/g3.7730369</a>.</p></li><li class="fn" id="fn-5"><p id="p-35"><em>Communicating editor: T. Matis</em>e</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-30"><span class="received-label">Received </span>December 30, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-04"><span class="accepted-label">Accepted </span>March 4, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Lin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094052.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNDk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.2" data-doi="10.1007/s00122-017-2857-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauchet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grivet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Use of modern tomato breeding germplasm for deciphering the genetic control of agronomical traits by Genome Wide Association study.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2857-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D875%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2857-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2857-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.3" data-doi="10.1371/journal.pone.0116822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhakta</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallejos</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Punctuated distribution of recombination hotspots and demarcation of pericentromeric regions in <em>Phaseolus vulgaris</em> L.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0116822</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0116822</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0116822%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0116822%26rft_id%253Dinfo%253Apmid%252F25629314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0116822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25629314&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.4" data-doi="10.1371/journal.pone.0048198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanca</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cañizares</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordero</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascual</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diez</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Variation Revealed by SNP Genotyping and Morphology Provides Insight into the Origin of the Tomato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e48198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0048198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBlanca%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De48198%26rft.epage%253De48198%26rft.atitle%253DVariation%2Brevealed%2Bby%2BSNP%2Bgenotyping%2Band%2Bmorphology%2Bprovides%2Binsight%2Binto%2Bthe%2Borigin%2Bof%2Bthe%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048198%26rft_id%253Dinfo%253Apmid%252F23118951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23118951&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.5" data-doi="10.1186/s12864-015-1444-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanca</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montero-Pau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauchet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Illa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic variation in tomato, from wild ancestors to contemporary breeding accessions.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1444-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1444-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1444-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.6" data-doi="10.1186/gb-2011-12-10-232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association studies in plants: the missing heritability is in the field.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2011-12-10-232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBrachi%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D232%26rft.epage%253D232%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bplants%253A%2Bthe%2Bmissing%2Bheritability%2Bis%2Bin%2Bthe%2Bfield.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-10-232%26rft_id%253Dinfo%253Apmid%252F22035733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-10-232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22035733&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.7" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: Software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.8" data-doi="10.1111/j.1365-294X.2004.02191.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caicedo</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaal</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Population structure and phylogeography of <em>Solanum pimpinellifolium</em> inferred from a nuclear gene.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1871</span>–<span class="cit-lpage">1882</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2004.02191.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCaicedo%26rft.auinit1%253DA.%2BL.%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253D1871%26rft.epage%253D1882%26rft.atitle%253DPopulation%2Bstructure%2Band%2Bphylogeography%2Bof%2BSolanum%2Bpimpinellifolium%2Binferred%2Bfrom%2Ba%2Bnuclear%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02191.x%26rft_id%253Dinfo%253Apmid%252F15189210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02191.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15189210&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221933000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.9" data-doi="10.1007/s00122-015-2563-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manuel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francesc</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Wide-genome QTL mapping of fruit quality traits in a tomato RIL population derived from the wild-relative species <em>Solanum pimpinellifolium</em> L.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">2019</span>–<span class="cit-lpage">2035</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2563-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D2019%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2563-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2563-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.10" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: An analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.11" data-doi="10.1371/journal.pone.0096417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reassessment of QTLs for late blight resistance in the tomato accession L3708 using a restriction site associated DNA (RAD) linkage map and highly aggressive isolates of <em>Phytophthora infestans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e96417</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0096417</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De96417%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0096417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0096417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.12" data-doi="10.1093/bfgp/elq031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaxter</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RADseq: Next-generation population genetics.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">416</span>–<span class="cit-lpage">423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq031%26rft_id%253Dinfo%253Apmid%252F21266344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21266344&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286675100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.13" data-doi="10.1016/0378-1119(80)90128-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobritsa</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobritsa</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">DNA protection with the DNA methylase M · BbvI from <em>Bacillus brevis</em> var. GB against cleavage by the restriction endonucleases <em>Pst</em>I and <em>Pvu</em>II.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0378-1119(80)90128-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DDobritsa%26rft.auinit1%253DA.%2BP.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D112%26rft.atitle%253DDNA%2Bprotection%2Bwith%2Bthe%2BDNA%2Bmethylase%2BM%2B.%2BBbvI%2Bfrom%2BBacillus%2Bbrevis%2Bvar.%2BGB%2Bagainst%2Bcleavage%2Bby%2Bthe%2Brestriction%2Bendonucleases%2BPstI%2Band%2BPvuII.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252880%252990128-6%26rft_id%253Dinfo%253Apmid%252F6248417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(80)90128-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6248417&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.14" data-doi="10.1007/978-1-61779-228-1_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SNP discovery and genotyping for evolutionary genetics using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">772</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">178</span>. <a href="https://dx.doi.org/10.1007/978-1-61779-228-1_9">https://dx.doi.org/10.1007/978-1-61779-228-1_9</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DEtter%26rft.auinit1%253DP.%2BD.%26rft.volume%253D772%26rft.spage%253D157%26rft.epage%253D178%26rft.atitle%253DSNP%2Bdiscovery%2Band%2Bgenotyping%2Bfor%2Bevolutionary%2Bgenetics%2Busing%2BRAD%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-228-1_9%26rft_id%253Dinfo%253Apmid%252F22065437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-228-1_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22065437&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321876000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.15" data-doi="10.1007/s00424-009-0730-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smouse</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quattro</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Analysis of molecular variance inferred from metric distances among DNA haplotypes: Application to human mitochondrial DNA restriction data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DExcoffier%26rft.auinit1%253DL.%26rft.volume%253D131%26rft.issue%253D2%26rft.spage%253D479%26rft.epage%253D491%26rft.atitle%253DAnalysis%2Bof%2Bmolecular%2Bvariance%2Binferred%2Bfrom%2Bmetric%2Bdistances%2Bamong%2BDNA%2Bhaplotypes%253A%2Bapplication%2Bto%2Bhuman%2Bmitochondrial%2BDNA%2Brestriction%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00424-009-0730-7%26rft_id%253Dinfo%253Apmid%252F1644282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMxLzIvNDc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.16" data-doi="10.1093/nar/gku1195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Pozo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menda</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tecle</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Sol Genomics Network (SGN)-from genotype to phenotype to breeding.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D1036</span>–<span class="cit-lpage">D1041</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1195%26rft_id%253Dinfo%253Apmid%252F25428362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25428362&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.17" data-doi="10.1007/BF02670897"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chunwongse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Microprep protocol for extraction of DNA from tomato and other herbaceous plants.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02670897</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D13%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02670897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02670897&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RR49200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.18" data-doi="10.1186/1471-2105-8-428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaunt</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">I. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cubic exact solutions for the estimation of pairwise haplotype frequencies: implications for linkage disequilibrium analyses and a web tool “CubeX”.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-8-428</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DGaunt%26rft.auinit1%253DT.%2BR.%26rft.volume%253D8%26rft.spage%253D428%26rft.epage%253D428%26rft.atitle%253DCubic%2Bexact%2Bsolutions%2Bfor%2Bthe%2Bestimation%2Bof%2Bpairwise%2Bhaplotype%2Bfrequencies%253A%2Bimplications%2Bfor%2Blinkage%2Bdisequilibrium%2Banalyses%2Band%2Ba%2Bweb%2Btool%2B%2527CubeX%2527.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-428%26rft_id%253Dinfo%253Apmid%252F17980034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17980034&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1497.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">I. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment"> hierfstat: Estimation and Tests of Hierarchical F-Statistics.</span> <a href="https://cran.r-project.org/web/packages/hierfstat/index.html">https://cran.r-project.org/web/packages/hierfstat/index.html</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1497.20"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Chetelat</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Spooner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Peralta</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <em>Solanum</em> sect. <em>Lycopersicon</em>, pp. 129-215 in <span class="cit-source">Wild Crop Relatives: Genomic and Breeding Resources Vegetables</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kole</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Heidelberg</span>. <a href="https://link.springer.com/chapter/10.1007/978-3-642-20450-0_9">https://link.springer.com/chapter/10.1007/978-3-642-20450-0_9</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.21" data-doi="10.3389/fpls.2017.02164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaikwad</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">From genomes to gene-omes: exome sequencing concept and applications in crop improvement.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.02164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D2164%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.02164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.02164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.22" data-doi="10.1127/0941-2948/2006/0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kottek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieser</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudolf</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubel</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">World map of the Köppen-Geiger climate classification updated.</span> <abbr class="cit-jnl-abbrev">Meteorol. Z. (Berl.)</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">263</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1127/0941-2948/2006/0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMeteorol.%2BZ.%2B%2528Berl.%2529%26rft.volume%253D15%26rft.spage%253D259%26rft_id%253Dinfo%253Adoi%252F10.1127%252F0941-2948%252F2006%252F0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1127/0941-2948/2006/0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.23" data-doi="10.1111/j.1365-294X.2012.05578.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meirmans</span>  <span class="cit-name-given-names">P. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The trouble with isolation by distance.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2839</span>–<span class="cit-lpage">2846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2012.05578.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMeirmans%26rft.auinit1%253DP.%2BG.%26rft.volume%253D21%26rft.issue%253D12%26rft.spage%253D2839%26rft.epage%253D2846%26rft.atitle%253DThe%2Btrouble%2Bwith%2Bisolation%2Bby%2Bdistance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05578.x%26rft_id%253Dinfo%253Apmid%252F22574758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05578.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22574758&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304903100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.24" data-doi="10.1111/j.1558-5646.2008.00487.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moyle</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Ecological and evolutionary genomics in the wild tomatoes (Solanum Sect. Lycopersicon).</span> <abbr class="cit-jnl-abbrev">Evolution.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">2995</span>–<span class="cit-lpage">3013</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2008.00487.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DMoyle%26rft.auinit1%253DL.%2BC.%26rft.volume%253D62%26rft.issue%253D12%26rft.spage%253D2995%26rft.epage%253D3013%26rft.atitle%253DEcological%2Band%2Bevolutionary%2Bgenomics%2Bin%2Bthe%2Bwild%2Btomatoes%2B%2528Solanum%2Bsect.%2BLycopersicon%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00487.x%26rft_id%253Dinfo%253Apmid%252F18752600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00487.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18752600&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261442900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.25" data-doi="10.1111/1755-0998.12129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pembleton</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">StAMPP: An R package for calculation of genetic differentiation and structure of mixed-ploidy level populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">952</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12129</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D13%26rft.spage%253D946%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12129%26rft_id%253Dinfo%253Apmid%252F23738873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23738873&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.26" data-doi="10.1534/g3.112.002667"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ranc</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munos</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Paslier</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauveau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping in tomato (<em>Solanum lycopersicum</em>) is possible using genome admixture of <em>Solanum lycopersicum</em> var. <em>cerasiforme</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">864</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.002667</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D853%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.002667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.002667&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.27" data-doi="10.1017/S1479262111000955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadirvel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symonds</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geethanjali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Using SSR markers to map genetic diversity and population structure of <em>Solanum pimpinellifolium</em> for development of a core collection.</span> <abbr class="cit-jnl-abbrev">Plant Genet. Resour.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S1479262111000955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenet.%2BResour.%26rft.volume%253D10%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1479262111000955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1479262111000955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.28" data-doi="10.1371/journal.pone.0132535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadirvel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symonds</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geethanjali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thontadarya</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Variations in <em>DREB1A</em> and <em>VP1.1</em> genes show association with salt tolerance traits in wild tomato (<em>Solanum pimpinellifolium</em>).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0132535</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0132535</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0132535%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0132535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0132535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.29" data-doi="10.1073/pnas.201394398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuoka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitt</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structure of linkage disequilibrium and phenotypic associations in the maize genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">11479</span>–<span class="cit-lpage">11484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.201394398</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.201394398%26rft_id%253Dinfo%253Apmid%252F11562485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjAvMTE0NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTQ5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.30" data-doi="10.1007/BF00984147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fobes</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holle</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Genetic variation in <em>Lycopersicon pimpinellifolium</em>: Evidence of evolutionary change in mating systems.</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00984147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D127%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00984147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00984147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977DL82000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.31" data-doi="10.1093/dnares/dsw050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruggieri</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anzar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paytuvi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calafiore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cigliano</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsw050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft.volume%253D24%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsw050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsw050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.32" data-doi="10.1093/dnares/dsw004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Analytical work flow of double-digest restriction site-associated DNA sequencing based on empirical and in silico optimization in tomato.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsw004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsw004%26rft_id%253Dinfo%253Apmid%252F26932983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsw004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26932983&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.33" data-doi="10.1371/journal.pone.0045520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">High-density SNP genotyping of tomato (<em>Solanum lycopersicum</em> L.) reveals patterns of genetic variation due to breeding.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0045520</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSim%26rft.auinit1%253DS.%2BC.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45520%26rft.epage%253De45520%26rft.atitle%253DHigh-density%2BSNP%2Bgenotyping%2Bof%2Btomato%2B%2528Solanum%2Blycopersicum%2BL.%2529%2Breveals%2Bpatterns%2Bof%2Bgenetic%2Bvariation%2Bdue%2Bto%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045520%26rft_id%253Dinfo%253Apmid%252F23029069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23029069&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.34" data-doi="10.1371/journal.pone.0040563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wieseke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Development of a large SNP genotyping array and generation of high-density genetic maps in tomato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e40563</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0040563</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSim%26rft.auinit1%253DS.%2BC.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40563%26rft.epage%253De40563%26rft.atitle%253DDevelopment%2Bof%2Ba%2Blarge%2BSNP%2Bgenotyping%2Barray%2Band%2Bgeneration%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040563%26rft_id%253Dinfo%253Apmid%252F22802968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22802968&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1497.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soto-Cerda</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cloutier</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Association mapping in plant genomes</span>, <span class="cit-source">Genetic Diversity in Plants</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Mahmut</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-publ-name">InTech</span>, <span class="cit-publ-loc">Rijeka</span>. <a href="https://doi.org/10.5772/33005">https://doi.org/10.5772/33005</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.36" data-doi="10.2307/25065301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spooner</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peralta</span>  <span class="cit-name-given-names">I. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparison of AFLPs with other markers for phylogenetic inference in wild tomatoes [Solanum L. section Lycopersicon (Mill.) Wettst.].</span> <abbr class="cit-jnl-abbrev">Taxon</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/25065301</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTaxon%26rft.volume%253D54%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.2307%252F25065301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/25065301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227715000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.37" data-doi="10.1126/science.277.5329.1063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mccouch</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Seed banks and molecular maps: unlocking genetic potential from the wild.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">1063</span>–<span class="cit-lpage">1066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.277.5329.1063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTanksley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D277%26rft.issue%253D5329%26rft.spage%253D1063%26rft.epage%253D1066%26rft.atitle%253DSeed%2BBanks%2Band%2BMolecular%2BMaps%253A%2BUnlocking%2BGenetic%2BPotential%2Bfrom%2Bthe%2BWild%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.277.5329.1063%26rft_id%253Dinfo%253Apmid%252F9262467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzcvNTMyOS8xMDYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE0OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.38" data-doi="10.1186/1471-2164-15-1152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Víquez-Zamora</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caro</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finkers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tikunov</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping in the era of sequencing : high density genotyping and its application for mapping TYLCV resistance in <em>Solanum pimpinellifolium</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-1152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1152%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.39" data-doi="10.1111/j.1558-5646.1984.tb05657.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating <em>F</em>-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1984.tb05657.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.1984.tb05657.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.1984.tb05657.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">Isolation by distance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D114%26rft.epage%253D138%26rft.atitle%253DISOLATION%2BBY%2BDISTANCE%26rft_id%253Dinfo%253Apmid%252F17247074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyOC8yLzExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNDk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.41" data-doi="10.1038/nbt.2462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fei</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Single-base resolution methylomes of tomato fruit development reveal epigenome modifications associated with ripening.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2462</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2462%26rft_id%253Dinfo%253Apmid%252F23354102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23354102&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1497.42" data-doi="10.1007/s10722-008-9340-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuriaga</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanca</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordero</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sifres</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blas-Cerdán</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic and bioclimatic variation in <em>Solanum pimpinellifolium</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10722-008-9340-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D56%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-008-9340-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-008-9340-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1497.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1487" title="The Neurospora crassa Standard Oak Ridge Background Exhibits Atypically Efficient Meiotic Silencing by Unpaired DNA" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1507" title="Small RNA-Seq Analysis Reveals miRNA Expression Dynamics Across Tissues in the Malaria Vector, Anopheles gambiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1497" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-w2KKI2b_Rj3PTIdQs9SKUvfJIepkg8pGd-NAaNi0AHY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1497" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1497" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1497" class="active" data-icon-position="" data-hide-link-title="0">Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493102" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zoaf734il7yt-pO3DOfn2FH5bJsM_YZxkDVxNXt436k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1497.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1497" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nX7SN8lVXQ1B3Vv1ra0d4zqGQA6Mfnz3StR46m911j0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1497" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1497.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493102" id="node493102--2" data-pisa="ggg;9/5/1497" data-pisa-master="ggg;g3.118.200862" data-apath="/ggg/9/5/1497.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessment of Genetic Differentiation and Linkage Disequilibrium in <em>Solanum pimpinellifolium</em> Using Genome-Wide High-Density SNP Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ya-Ping Lin</span>, <span class="highwire-citation-author" data-delta="1">Chu-Yin Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6211-0198" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kai-Yi Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1497-1505;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200862 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493102--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ya-Ping Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ya-Ping%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYa-Ping%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chu-Yin Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chu-Yin%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChu-Yin%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kai-Yi Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai-Yi%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKai-Yi%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6211-0198" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kai-Yi Chen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kaiychen@ntu.edu.tw" class="" data-icon-position="" data-hide-link-title="0">kaiychen@ntu.edu.tw</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1497" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1497&amp;title=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1497&amp;title=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1497&amp;title=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1497&amp;text=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1497&amp;title=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1497&amp;t=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1497&amp;title=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1497&amp;title=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493102/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493102" id="node493102--3" data-pisa="ggg;9/5/1497" data-pisa-master="ggg;g3.118.200862" data-apath="/ggg/9/5/1497.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessment of Genetic Differentiation and Linkage Disequilibrium in <em>Solanum pimpinellifolium</em> Using Genome-Wide High-Density SNP Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ya-Ping Lin</span>, <span class="highwire-citation-author" data-delta="1">Chu-Yin Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6211-0198" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kai-Yi Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1497-1505;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200862 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493102--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ya-Ping Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ya-Ping%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYa-Ping%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chu-Yin Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chu-Yin%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChu-Yin%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kai-Yi Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy, National Taiwan University, Taipei, Taiwan, 10617</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai-Yi%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKai-Yi%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6211-0198" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kai-Yi Chen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kaiychen@ntu.edu.tw" class="" data-icon-position="" data-hide-link-title="0">kaiychen@ntu.edu.tw</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1497&amp;rft.epage=1505&amp;rft.atitle=Assessment%20of%20Genetic%20Differentiation%20and%20Linkage%20Disequilibrium%20in%20Solanum%20pimpinellifolium%20Using%20Genome-Wide%20High-Density%20SNP%20Markers&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lin%2C+Ya-Ping&amp;rft.au=Liu%2C+Chu-Yin&amp;rft.au=Chen%2C+Kai-Yi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493102/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493102/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493102/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493102/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493102/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493102/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493102/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493102/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493102/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493102/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493102/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493102/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1497" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1497.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1497.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1497.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
