<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1623" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1623.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1623.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1623.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1623.atom" />
<meta name="HW.pisa" content="ggg;9/5/1623" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce (Picea abies)" />
<meta name="DC.Identifier" content="10.1534/g3.118.200840" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Bernhardsson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Norway spruce ( Picea abies (L.) Karst.) is a conifer species of substanital economic and ecological importance. In common with most conifers, the P. abies genome is very large (∼20 Gbp) and contains a high fraction of repetitive DNA. The current P. abies genome assembly (v1.0) covers approximately 60% of the total genome size but is highly fragmented, consisting of &gt;10 million scaffolds. The genome annotation contains 66,632 gene models that are at least partially validated ([www.congenie.org][1]), however, the fragmented nature of the assembly means that there is currently little information available on how these genes are physically distributed over the 12 P. abies chromosomes. By creating an ultra-dense genetic linkage map, we anchored and ordered scaffolds into linkage groups, which complements the fine-scale information available in assembly contigs. Our ultra-dense haploid consensus genetic map consists of 21,056 markers derived from 14,336 scaffolds that contain 17,079 gene models (25.6% of the validated gene models) that we have anchored to the 12 linkage groups. We used data from three independent component maps, as well as comparisons with previously published Picea maps to evaluate the accuracy and marker ordering of the linkage groups. We demonstrate that approximately 3.8% of the anchored scaffolds and 1.6% of the gene models covered by the consensus map have likely assembly errors as they contain genetic markers that map to different regions within or between linkage groups. We further evaluate the utility of the genetic map for the conifer research community by using an independent data set of unrelated individuals to assess genome-wide variation in genetic diversity using the genomic regions anchored to linkage groups. The results show that our map is sufficiently dense to enable detailed evolutionary analyses across the P. abies genome.

 [1]: http://www.congenie.org" />
<meta name="DC.Contributor" content="Carolina Bernhardsson" />
<meta name="DC.Contributor" content="Amaryllis Vidalis" />
<meta name="DC.Contributor" content="Xi Wang" />
<meta name="DC.Contributor" content="Douglas G. Scofield" />
<meta name="DC.Contributor" content="Bastian Schiffthaler" />
<meta name="DC.Contributor" content="John Baison" />
<meta name="DC.Contributor" content="Nathaniel R. Street" />
<meta name="DC.Contributor" content="M. Rosario García-Gil" />
<meta name="DC.Contributor" content="Pär K. Ingvarsson" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce (Picea abies)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Norway spruce (&lt;i&gt;Picea abies&lt;/i&gt; (L.) Karst.) is a conifer species of substanital economic and ecological importance. In common with most conifers, the &lt;i&gt;P. abies&lt;/i&gt; genome is very large (∼20 Gbp) and contains a high fraction of repetitive DNA. The current &lt;i&gt;P. abies&lt;/i&gt; genome assembly (v1.0) covers approximately 60% of the total genome size but is highly fragmented, consisting of &amp;gt;10 million scaffolds. The genome annotation contains 66,632 gene models that are at least partially validated (www.congenie.org), however, the fragmented nature of the assembly means that there is currently little information available on how these genes are physically distributed over the 12 &lt;i&gt;P. abies&lt;/i&gt; chromosomes. By creating an ultra-dense genetic linkage map, we anchored and ordered scaffolds into linkage groups, which complements the fine-scale information available in assembly contigs. Our ultra-dense haploid consensus genetic map consists of 21,056 markers derived from 14,336 scaffolds that contain 17,079 gene models (25.6% of the validated gene models) that we have anchored to the 12 linkage groups. We used data from three independent component maps, as well as comparisons with previously published &lt;i&gt;Picea&lt;/i&gt; maps to evaluate the accuracy and marker ordering of the linkage groups. We demonstrate that approximately 3.8% of the anchored scaffolds and 1.6% of the gene models covered by the consensus map have likely assembly errors as they contain genetic markers that map to different regions within or between linkage groups. We further evaluate the utility of the genetic map for the conifer research community by using an independent data set of unrelated individuals to assess genome-wide variation in genetic diversity using the genomic regions anchored to linkage groups. The results show that our map is sufficiently dense to enable detailed evolutionary analyses across the &lt;i&gt;P. abies&lt;/i&gt; genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1623" />
<meta name="citation_id" content="9/5/1623" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1623" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1623.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1623.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1623.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200840" />
<meta name="citation_pmid" content="30898899" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1623" />
<meta name="citation_lastpage" content="1632" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Carolina Bernhardsson" />
<meta name="citation_author_institution" content="Department of Ecology and Environmental Science and" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden" />
<meta name="citation_author_institution" content="Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden" />
<meta name="citation_author_email" content="carolina.bernhardsson@gmail.com" />
<meta name="citation_author_email" content="par.ingvarsson@slu.se" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3258-275X" />
<meta name="citation_author" content="Amaryllis Vidalis" />
<meta name="citation_author_institution" content="Department of Ecology and Environmental Science and" />
<meta name="citation_author_institution" content="Department of Population Genetics, Center of Life and Food Sciences Weihenstephan, Technische Universität München, 85354 Freising, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7111-9530" />
<meta name="citation_author" content="Xi Wang" />
<meta name="citation_author_institution" content="Department of Ecology and Environmental Science and" />
<meta name="citation_author_institution" content="Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden" />
<meta name="citation_author" content="Douglas G. Scofield" />
<meta name="citation_author_institution" content="Department of Ecology and Environmental Science and" />
<meta name="citation_author_institution" content="Uppsala Multidisciplinary Center for Advanced Computational Science and" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics: Evolutionary Biology, Uppsala University, Uppsala, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5235-6461" />
<meta name="citation_author" content="Bastian Schiffthaler" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9771-467X" />
<meta name="citation_author" content="John Baison" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9919-907X" />
<meta name="citation_author" content="Nathaniel R. Street" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6031-005X" />
<meta name="citation_author" content="M. Rosario García-Gil" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6834-6708" />
<meta name="citation_author" content="Pär K. Ingvarsson" />
<meta name="citation_author_institution" content="Department of Ecology and Environmental Science and" />
<meta name="citation_author_institution" content="Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden" />
<meta name="citation_author_email" content="carolina.bernhardsson@gmail.com" />
<meta name="citation_author_email" content="par.ingvarsson@slu.se" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9225-7521" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=J. Baison;citation_author=A. Vidalis;citation_author=L. Zhou;citation_author=Z.-Q. Chen;citation_author=Z. Li;citation_title=Association mapping identified novel candidate loci affecting wood formation in Norway spruce.;citation_year=2018;citation_doi=10.1101/292847" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. Bartholomé;citation_author=E. Mandrou;citation_author=A. Mabiala;citation_author=J. Jenkins;citation_author=I. Nabihoudine;citation_title=High-Resolution Genetic Maps of Eucalyptus Improve Eucalyptus Grandis Genome Assembly.;citation_pages=1283-1296;citation_volume=206;citation_year=2015;citation_pmid=25385325;citation_doi=10.1111/nph.13150" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. A. Cartwright;citation_author=M. Troggio;citation_author=R. Velasco;citation_author=A. Gutin;citation_title=Genetic Mapping in the Presence of Genotyping Errors.;citation_volume=176;citation_year=2007;citation_pmid=17277374;citation_doi=10.1534/genetics.106.063982" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The Variant Call Format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. R. De La Torre;citation_author=I. Birol;citation_author=J. Bousquet;citation_author=P. K. Ingvarsson;citation_author=S. Jansson;citation_title=Insights into Conifer Giga-Genomes.;citation_volume=166;citation_year=2014;citation_pmid=25349325;citation_doi=10.1104/pp.114.248708" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. de Miguel;citation_author=J. Bartholomé;citation_author=F. Ehrenmann;citation_author=F. Murat;citation_author=Y. Moriguchi;citation_title=Evidence of Intense Chromosomal Shuffling during Conifer Evolution.;citation_volume=7;citation_year=2015;citation_pmid=26400405;citation_doi=10.1093/gbe/evv185" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=D. R. Drost;citation_author=E. Novaes;citation_author=C. Boaventura-Novaes;citation_author=C. I. Benedict;citation_author=R. S. Brown;citation_title=A microarray-based genotyping and genetic mapping approach for highly heterozygous outcrossing species enables localization of a large fraction of the unassembled Populus trichocarpa genome sequence.;citation_pages=1054-1067;citation_volume=58;citation_year=2009;citation_issue=6;citation_pmid=19220791;citation_doi=10.1111/j.1365-313X.2009.03828.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. B. Endelman;citation_author=C. Plomion;citation_title=LPmerge: An R Package for Merging Genetic Maps by Linear Programming.;citation_volume=30;citation_year=2014;citation_pmid=24532720;citation_doi=10.1093/bioinformatics/btu091" />
<meta name="citation_reference" content="Farjon, A., 1990 Pinaceae. Drawings and Descriptions of the Genera Abies, Cedrus, Pseudolarix, Keteleeria, Nothotsuga, Tsuga, Cathaya, Pseudotsuga, Larix and Picea. Pinaceae. Drawings and Descriptions of the Genera Abies, Cedrus, Pseudolarix, Keteleeria, Nothotsuga, Tsuga, Cathaya, Pseudotsuga, Larix and Picea. Koeltz Scientific Books. https://www.cabdirect.org/cabdirect/abstract/19920656698." />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. L. Fierst;citation_title=Using Linkage Maps to Correct and Scaffold de Novo Genome Assemblies: Methods, Challenges, and Computational Tools.;citation_pages=220;citation_volume=6;citation_year=2015;citation_pmid=26150829;citation_doi=10.3389/fgene.2015.00220" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. S. Gaut;citation_author=S. I. Wright;citation_author=C. Rizzon;citation_author=J. Dvorak;citation_author=L. K. Anderson;citation_title=Recombination: an underappreciated factor in the evolution of plant genomes.;citation_pages=77-84;citation_volume=8;citation_year=2007;citation_issue=1;citation_pmid=17173059;citation_doi=10.1038/nrg1970" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Hedgecock;citation_author=G. Shin;citation_author=A. Y. Gracey;citation_author=D. Van Den Berg;citation_author=M. P. Samanta;citation_title=Second-Generation Linkage Maps for the Pacific Oyster Crassostrea Gigas Reveal Errors in Assembly of Genome Scaffolds. G3: Genes, Genomes;citation_pages=2007-2019;citation_volume=5;citation_year=2015;citation_doi=10.1534/g3.115.019570" />
<meta name="citation_reference" content="citation_journal_title=Cancer Inform.;citation_author=Y. Hu;citation_author=C. Yan;citation_author=C.-H. Hsu;citation_author=Q.-R. Chen;citation_author=K. Niu;citation_title=OmicCircos: A Simple-to-Use R Package for the Circular Visualization of Multidimensional Omics Data.;citation_pages=13-20;citation_volume=13;citation_year=2014;citation_pmid=24526832;citation_doi=10.4137/CIN.S13495" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Kelly;citation_title=A Test of Neutrality Based on Interlocus Associations;citation_pages=1197-1206;citation_volume=146;citation_year=1997;citation_issue=3;citation_pmid=9215920" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=B. J. Knaus;citation_author=N. J. Grünwald;citation_title=vcfR : A Package to Manipulate and Visualize Variant Call Format Data in R.;citation_pages=44-53;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12549" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and Accurate Short Read Alignment with Burrows-Wheeler Transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map Format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Lind;citation_author=T. Källman;citation_author=J. Chen;citation_author=X.-F. Ma;citation_author=J. Bousquet;citation_title=A Picea Abies Linkage Map Based on SNP Markers Identifies QTLs for Four Aspects of Resistance to Heterobasidion Parviporum Infection.;citation_pages=e101049;citation_volume=9;citation_year=2014;citation_pmid=25036209;citation_doi=10.1371/journal.pone.0101049" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=G. R. A. Margarido;citation_author=A. P. Souza;citation_author=A. A. F. Garcia;citation_title=OneMap: software for genetic mapping in outcrossing species.;citation_pages=78-79;citation_volume=144;citation_year=2007;citation_issue=3;citation_pmid=17663699;citation_doi=10.1111/j.2007.0018-0661.02000.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: A MapReduce Framework for Analyzing next-Generation DNA Sequencing Data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. Mollinari;citation_author=G. R. A. Margarido;citation_author=R. Vencovsky;citation_author=A. A. F. Garcia;citation_title=Evaluation of algorithms used to order markers on genetic maps.;citation_pages=494-502;citation_volume=103;citation_year=2009;citation_issue=6;citation_pmid=19639011;citation_doi=10.1038/hdy.2009.96" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Nystedt;citation_author=N. R. Street;citation_author=A. Wetterbom;citation_author=A. Zuccolo;citation_author=Y.-C. Lin;citation_title=The Norway Spruce Genome Sequence and Conifer Genome Evolution.;citation_pages=579-584;citation_volume=497;citation_year=2013;citation_pmid=23698360;citation_doi=10.1038/nature12211" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=N. Pavy;citation_author=A. Deschênes;citation_author=S. Blais;citation_author=P. Lavigne;citation_author=J. Beaulieu;citation_title=The Landscape of Nucleotide Polymorphism among 13,500 Genes of the Conifer Picea Glauca, Relationships with Functions, and Comparison with Medicago Truncatula.;citation_volume=5;citation_year=2013;citation_pmid=24065735;citation_doi=10.1093/gbe/evt143" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=N. Pavy;citation_author=M. Lamothe;citation_author=B. Pelgas;citation_author=F. Gagnon;citation_author=I. Birol;citation_title=A High-Resolution Reference Genetic Map Positioning 8.8 K Genes for the Conifer White Spruce: Structural Genomics Implications and Correspondence with Physical Distance.;citation_pages=189-203;citation_volume=90;citation_year=2017;citation_doi=10.1111/tpj.13478" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=N. Pavy;citation_author=B. Pelgas;citation_author=J. Laroche;citation_author=P. Rigault;citation_author=N. Isabel;citation_title=A spruce gene map infers ancient plant genome reshuffling and subsequent slow evolution in the gymnosperm lineage leading to extant conifers.;citation_pages=84-84;citation_volume=10;citation_year=2012;citation_issue=1;citation_pmid=23102090;citation_doi=10.1186/1741-7007-10-84" />
<meta name="citation_reference" content="citation_journal_title=J. Arnold Arbor.;citation_author=K. Sax;citation_author=H. J. Sax;citation_title=J ARNOLD ARBOR;citation_pages=356-375;citation_volume=14;citation_year=1933;citation_doi=10.5962/bhl.part.9959" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. Schiffthaler;citation_author=C. Bernhardsson;citation_author=P. K. Ingvarsson;citation_author=N. R. Street;citation_title=BatchMap: A parallel implementation of the OneMap R package for fast computation of F1 linkage maps in outcrossing species.;citation_pages=e0189256;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0189256" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=A. H. Sturtevant;citation_title=The Linear Arrangement of Six Sex-Linked Factors in Drosophila, as Shown by Their Mode of Association.;citation_pages=43-59;citation_volume=14;citation_year=1913;citation_doi=10.1002/jez.1400140104" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Sturtevant;citation_title=A THIRD GROUP OF LINKED GENES IN DROSOPHILA AMPELOPHILA;citation_pages=990-992;citation_volume=37;citation_year=1913;citation_issue=965;citation_pmid=17833164;citation_doi=10.1126/science.37.965.990" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinformatics;citation_author=G. A. Van der Auwera;citation_author=M. O. Carneiro;citation_author=C. Hartl;citation_author=R. Poplin;citation_author=G. del Angel;citation_title=From FastQ Data to High Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline.;citation_volume=43;citation_year=2013;citation_pmid=25431634;citation_doi=10.1002/0471250953.bi1110s43" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Van Os;citation_author=P. Stam;citation_author=R. G. F. Visser;citation_author=H. J. Van Eck;citation_title=RECORD: a novel method for ordering loci on a genetic linkage map.;citation_pages=30-40;citation_volume=112;citation_year=2005;citation_issue=1;citation_pmid=16228189;citation_doi=10.1007/s00122-005-0097-x" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=A. Vidalis;citation_author=D. G. Scofield;citation_author=L.-G. Neves;citation_author=C. Bernhardsson;citation_author=M. R. García-Gil;citation_title=Design and evaluation of a large sequence-capture probe set and associated SNPs for diploid and haploid samples of Norway spruce (Picea abies);citation_year=2018;citation_doi=10.1101/291716" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. Wu;citation_author=C.-X. Ma;citation_author=I. Painter;citation_author=Z.-B. Zeng;citation_title=Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species.;citation_pages=349-363;citation_volume=61;citation_year=2002;citation_issue=3;citation_pmid=12027621;citation_doi=10.1006/tpbi.2002.1577" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce (Picea abies)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Norway spruce ( Picea abies (L.) Karst.) is a conifer species of substanital economic and ecological importance. In common with most conifers, the P. abies genome is very large (∼20 Gbp) and contains a high fraction of repetitive DNA. The current P. abies genome assembly (v1.0) covers approximately 60% of the total genome size but is highly fragmented, consisting of &gt;10 million scaffolds. The genome annotation contains 66,632 gene models that are at least partially validated ([www.congenie.org][1]), however, the fragmented nature of the assembly means that there is currently little information available on how these genes are physically distributed over the 12 P. abies chromosomes. By creating an ultra-dense genetic linkage map, we anchored and ordered scaffolds into linkage groups, which complements the fine-scale information available in assembly contigs. Our ultra-dense haploid consensus genetic map consists of 21,056 markers derived from 14,336 scaffolds that contain 17,079 gene models (25.6% of the validated gene models) that we have anchored to the 12 linkage groups. We used data from three independent component maps, as well as comparisons with previously published Picea maps to evaluate the accuracy and marker ordering of the linkage groups. We demonstrate that approximately 3.8% of the anchored scaffolds and 1.6% of the gene models covered by the consensus map have likely assembly errors as they contain genetic markers that map to different regions within or between linkage groups. We further evaluate the utility of the genetic map for the conifer research community by using an independent data set of unrelated individuals to assess genome-wide variation in genetic diversity using the genomic regions anchored to linkage groups. The results show that our map is sufficiently dense to enable detailed evolutionary analyses across the P. abies genome.

 [1]: http://www.congenie.org" />
<meta name="og-title" property="og:title" content="An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce (Picea abies)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1623" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Norway spruce ( Picea abies (L.) Karst.) is a conifer species of substanital economic and ecological importance. In common with most conifers, the P. abies genome is very large (∼20 Gbp) and contains a high fraction of repetitive DNA. The current P. abies genome assembly (v1.0) covers approximately 60% of the total genome size but is highly fragmented, consisting of &gt;10 million scaffolds. The genome annotation contains 66,632 gene models that are at least partially validated ([www.congenie.org][1]), however, the fragmented nature of the assembly means that there is currently little information available on how these genes are physically distributed over the 12 P. abies chromosomes. By creating an ultra-dense genetic linkage map, we anchored and ordered scaffolds into linkage groups, which complements the fine-scale information available in assembly contigs. Our ultra-dense haploid consensus genetic map consists of 21,056 markers derived from 14,336 scaffolds that contain 17,079 gene models (25.6% of the validated gene models) that we have anchored to the 12 linkage groups. We used data from three independent component maps, as well as comparisons with previously published Picea maps to evaluate the accuracy and marker ordering of the linkage groups. We demonstrate that approximately 3.8% of the anchored scaffolds and 1.6% of the gene models covered by the consensus map have likely assembly errors as they contain genetic markers that map to different regions within or between linkage groups. We further evaluate the utility of the genetic map for the conifer research community by using an independent data set of unrelated individuals to assess genome-wide variation in genetic diversity using the genomic regions anchored to linkage groups. The results show that our map is sufficiently dense to enable detailed evolutionary analyses across the P. abies genome.

 [1]: http://www.congenie.org" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce (Picea abies) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3fOzhiEJgrnPuxTONYVdw9xTbHBjtG5jkuNdnNKRoVE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493103","apath":"\/ggg\/9\/5\/1623.atom","pisa":"ggg;9\/5\/1623","ac":{"\/ggg\/9\/5\/1623.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1623.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1623"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1623"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1623":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uzi645ADRWM9E7+fZC\\\\\\\/RZ+I0OoucblMSXMBY5IvdG1VKOIxKMHXBRQEyTuj9lp7ZwKUkVqVFxCCp0DnlI2JlDIM9x5vT137d54gdAWbrqoRVXV1I8nOEuZB0rpMmZX3N63I4WK5P\\\\\\\/U8NqkiNUGL450sY7zqX3ouM9h6s9PNBSbS\\\\\\\/gXblCPCH2NNNNplJ2UdVuT6oQIkKfjkGWPmBxBvKy0R21wfE6RAG39HjLdxn8i4fUvmnOrzyH\\\\\\\/mXY1M3tIqWzRTlYF\\\\\\\/pqkwwIc9E53Gc5vrQ2sV7YR0406mCtHaFWKcbgb4oGB4X8r8TcSf0fkmnuZQMnnfOpTm9NSLbLysYxPvJPsQnV2I+mZBRiZ4xtSnTJqCTeskuFyHX2j8qHzW5OxPntvInaYks5nvnHwtLda52y3U\\\\\\\/TBSNd6MLLi7OWSjYM6Xzl+IarQ\\\\\\\/piIeykfDh8J9bYBVIpDtH1A46EXGG0fmkDVLGL9EkylT1lKkaJXRA3WflVPvgMODfzWw9skDUUBh68IgKse5oCZ0zpl1tvAeMcV+qSZHe1sspRh+mTJS6pGmTOOWJPQpjeRGvKBEl1yYNHThoL5PM5BwrccsY1ur1TqqHf3Du8xKHfPpLTu2G0x2w92cFBPdKtd5rxWtgxfIGh82pOxfcSmyaQqgwbErKx8d+y9\\\\\\\/kd5uiSR9ojXYnfED5gxumYlmXRuXmUGBs0i9wMjMPIHHgmF3pOTdlIu50rKqJ0ERKRM8r5a+akc75GaCCygDudDttxY+jipIieE\\\\\\\/nNU3gaHeSg\\\\\\\/JGf1ATrDCnPREeKvdg8EH0Gksch9lmpM22\\\\\\\/U2BVV43oa+4wdBjizxax0vgPK2b1bDN+0U8KIrzwVrfhramz5mNukkoIg0YvWBb2COIiDR46m4YLwLijEzYRdPKMuOYCOEJGkjocEeqMbIOhORpUj9C7OB8k+RGFSsVF7W9\\\\\\\/xA\\\\\\\/1AB6AjSzeAN\\\\\\\/AWhpksWJbKbY++nrec9yy\\\\\\\/18+l7a\\\\\\\/T6c0mXTbvJLldAjH33CdbUjbSJY2U8jaLEFmyba2wgwfRscgMXDUqIVzJEykpMLY8LqPjkLb7j8fXBjTttf3E3InGLEg8+5fkc1r1cqiSIA4xZO5aokyp\\\\\\\/avE0pL6GfgKhw8PI5D1SryH7I6mbpSx+ETD6qlHF+esztwjKFxhxNye1bYPzRYnrT0tHIjRO73WuADbrrFuL9YjYb\\\\\\\/24ZWCHB\\\\\\\/heYQZwTxBx08F7qrdtj5fMvjHmRGTyVmgMf527loBHefyVFi3Iw7xr53km69RbO+gxL7gPE\\\u0022,\\\u0022iv\\\u0022:\\\u0022GRYZXxwWXfxXgIXdUQL6dQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221e1fc8a6603d2e86e538b5d5fda419d9b9768ac2b6e4080bc1a85deede97d92f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022V9cRhEpyOLe8t16Ui92LRN92xhJTPH3ixO0zV\\\\\\\/uOGRqz54ymRAUBeTXGnV+FnEQsV+mpwFrkkkqL9i708WWbaeSNwK6FarZLQfNjSBaREBMn5lpBqHHyDqHd\\\\\\\/x8R0hN2IDYz9owF83tgCMxGG\\\\\\\/U+brHJ7s0BwofGbTAiognd3zWTvqoIEi+EbDcOhXOd2+1TYfHP\\\\\\\/\\\\\\\/vVJO6pMHu6+CPsoaAhko5isOzmG9XTgV11FpyfEiBfeagSgocHK\\\\\\\/ELgcd2o9FzgTnipBdb8La39Z1lgkNqCAUwLpQ\\\\\\\/5on+QmVxeHiYxENXoJXI1LPRghvY6kz5zlAu1J0XXMQGt9izjH1TahTjIf70G986wzjF43A8Y383IaKUPR48QgXqTZ3i4fnVn3SEhsM5UbcJv4sWDdJdCN5o5ngmLbFj\\\\\\\/DB+BMpWzvQN\\\\\\\/KUzJm2ynEzChG7D1O49Kb87Lv9iNNozRcThFdBOBq56syM9MXFX5L+sc4effemxmRG\\\\\\\/JdlfSRRBl\\\\\\\/ZIB1DpVcRni6SgReypnk+IJyenqkFuPMvGBALZyG96qfwNKe+0jNKRT+6Rvfy4jKiBpNvlaLxqOzLRI2FhjUGMpcsoQrBMG1U1hGUzZP3dc8rQYG4oOvz5zK8jzLnaafpmxUgtVVWwMCR5salP6cFskKOqI5qoD8mYs6clqxqW4V8+ElDoK4DSlDA756t\\\\\\\/m7i2l\\\\\\\/hNOLVh9Gw4DL5RHA+bJh6oIq9RAv4Iyx13WHS9dCTl23dutwJn7bSsjC\\\\\\\/sXhXZJoZG5Vs\\\\\\\/Ze3T1glx6\\\\\\\/poIpoKgl5\\\\\\\/MalErgJP8ywuzDaLg4USug483l7+D9CUYrJ\\\\\\\/wgw+H1nGvahNdXNeKa\\\\\\\/V\\\\\\\/qK2S2LdbNWt6fPrKRKshxxLAZvvqdzs6jySDrcnYkcidgy8kxqfLNMjUyzdwQC1v3gZFz+b8IzJgkSkNrDOkEdp0TruvYGCqQYMXjhsOcAFZW7\\\\\\\/YRr2daLlOiB9R4su3DzNgYDkEFXAftPVss\\\\\\\/0sl420qSyqMD7iBX2oV2hAgYpleeyQ3Nlc0a1vhribE9aG\\\\\\\/juF2MwOmdTqTFI\\\\\\\/CBWupZlUuPT0NmID33WIgpFDboiVseht6o1i\\\\\\\/r1mxyzqgFVRL63Sw5oSsgCiBB85cu0YH5zubP9MbQg+yQqCQ4c0\\\\\\\/VNSr5p6q7xqKw528ZY8QhJf64Lz8D99J9lbEYo5BNiAkiSiKZM0NCYGgBuNxo1GKGV5R\\\\\\\/T5GvWeAJxj+rfgwmH3v9aQddikbEV99x6DeKenBn4b8TyXbHrwT7VjbEFKrVxepNVc6oQzCIeuCcqYCmz2XFqs29w53DRUaP2ypQ0NQ96wtK6GaRg6tOluKD0dYz7NRYsc8857Lb03uUlhH6Ff2yY3YpAtNnyqoYTg7LaSx25zK1FqDacrmH0xMsSqjMgd95uisBS69o3sSvxT6PZ13GpYr8CbWFNSAp2HsY3UtJYHy0csnqqZqqo9CplurfXdm96ZE09Q+xrYSBtbc+uV1EMULqXHucsoUKFMRJK\\\\\\\/s2Swev1\\\\\\\/aKPkSG4\\\\\\\/QHODIhyzj\\\\\\\/LGVo3NW8k82Anu\\\\\\\/KcrLMwpgFX51ySI24wuAAQg0Fjzy0psV0BlNPF6kzw8OShYwaa4S7\\\\\\\/MDEcVVC6hBbwqm+G2UHoFpZwe7TDLDJbhfj8cqK5\\\\\\\/BCkl6G2AOwdO3J5oAegxIP6Ww2vpZeApLvRVGfKkcETGPTl4hi585xphKSZX0Kexp1HLQ05CWeTomCtwrS14ZTnHjtrR6hZ7yLdDbyd8hxmNpOnUD1pm0iV18l+ydPXWkrC8Ijtp8ULEA4Hc2dMDm7F9lfKjvMZ00H0pkFn7zBeHwGfnq1B7FM\\\\\\\/9FsZAhg6bp0nFLS03NNkvt1IOaCmiJBl2UG6Cq\\\\\\\/vgbqLAcfwnHWWiiic7GOx22p9Ge+ZSI5a5ChV8Q7I5+lIMtiCv7aWYOkmCUQrYmgSwA1B9NFozW0p0RssEdD1G42byOdvr2pN8Vc4ljc380T3xIaDdaEYPAOn1wFnVVxWwvvCI0XfmOBdUxfjxgi5fJxbjDpTQSNWLWb5r5Bp41zbXg+vXuXz9rxSkjSlEOhc4\\\\\\\/jDmnJHH07QrQKv8jG16FCLvB1xnAQ0upbNoshdKYWhr+LyETrGAoa0H+zyIYUIwHIzxpXMqFv3S2NhQigO2bKbXcCeYVcftUcpgJ9qtEZ3Qf8W15C4qWopcegKUSI8Pcz2aM5dRm7981pNcs2IOiqx\\\\\\\/FxbbQ849cBoUJIS6bSsLuauuhRhlZxn3scysMxdwHlyAqiZcBvMkTjx7LRcl3j+6mk6aNUB3OQQUxykFRBZGQVoz1NDHYoGOOTirZR+fu0KMDr2xhrqTwIWivelcILGqZ5v4qsPFu1E2BuqCyGGRzzG45v17+lMiK9xSxpKvJujqmLnyT4p+9Of1+phY3qUa3iiGfHDjA2bAHVbFQfB33bbPX6jcMn5Gsg2+HiDTW\\\\\\\/O8HEgkwrYAZj5PJXP6mE+dsPQ6P95KqXUgfIXJBsd25NvhRdVpmC\\\\\\\/6k4+1J8lvmBfwDUXehDY1r+B5ndB0XFTeYwpRmHXbXDW6DDu3XgL5CJV98cb+O0roHakTjG0TrDe4fWszBywzMYGhFHT\\\\\\\/lu2qfOdmoVievaC4nTGw8ZW5q1WrwbLx5hh+8+kH9Zd5vnMvMFz9ES60Dl2dWeljxZlHZXVflo0ckmYSMgpPLIz3MWRzqhpKF0rrLkJMZajJCotOIu\\\\\\\/oQGf+l3I9qgbsmtpnjXVEaKmhTt5tXSD75BXkpQLpGgxV8jXvFBdsNRYc4bt7TImw2tF207JinNNnryaLbipeKvxrXuS0yAE2dNBHrVVGiLigoODqEXYSNPdiDNmy7s7sYhpj8hNtoSUFKQjkiJjfHKGEMFqeiorl0O0ldlsdjM78HOUB1\\\\\\\/wkZmt2fkx\\\\\\\/u\\\\\\\/TaB0Lec\\\\\\\/uKgqbEN9geiX7noASxp4TE4cqRlRn81SEitIf6hEkS797BqbmvWIYqg\\\\\\\/y8dtBWUUm4HpuMpMkt9aBFzcZtZz4tFQcbMnXfIR9kNfOu7Sg3Wan7EGTebHPRhTEJNr6JZpjse7zgQpyFFL\\\\\\\/zWYttJz445mgOiz5GesoYJ2fiWCkeDxW0OnBArH5Izq7N63e2rOqjGtdeZ1O1dfWArHvVOEuv8lqIBNgpIXP\\\\\\\/65V+7QSrc8BaMIthHvdZ4Sd4sqxss0c3eVujYVxypxBKjjSqPqyRUFB+KAtVcgCTM7ibjAhXSNw8aozcbTa+T3\\\\\\\/w7ybeFu7BR5OJFhUwcoY\\\\\\\/BE6TVoDQSCxU9FGJW5s7TJW5gV5H7J9\\\\\\\/YT\\\\\\\/y6TPLbC9wESCeNoRzpxn\\\\\\\/ECOWafYH2eKbQp2m\\\\\\\/+2uVS+33e74O1As+Wmyff\\\\\\\/QOCYbPEcsaJLmgrKOCx51GuQ1DlByyTOn2Ald23ZzZuIffJyL\\\\\\\/p40WnoJ3agrqN1RKgz70GvBuw0Bkzr8UkZg\\\\\\\/RDOJLgSH3YNQOxpGAvMi+BPBb4mLcvIHpk\\\\\\\/A42mTGOkD\\\\\\\/pSJVeb4m+PMxMl\\\\\\\/IkuagKuh11bBkOZCTOXX+iKgbpohmUvBeq5qIb63kmIDJWD0yYoAdlSjkW\\\\\\\/R3+wEysDv5SGKp5X2mO1TnRc8lkazNmtJmF\\\\\\\/Rh4ZzsjknRoaM2KGpzcAmcYaeWcXn0VqIw9LtIRj0ifL203heaIlyeUkuEMigPCsBQ+xyeNX37dA5ykodn5m0oHhzXimnX44t6JmwPR2lhXflJ8wazuDAvi6mXDJPXGRAQp0TTZA\\\\\\\/\\\\\\\/2xnPs6donNyo7kD7EDwp2XWwYx9Y0SevGpE7aWy+m48Xdk+W6DJORyLaThBIIqCw4q1RhCEIkBIr0KE0GBi3+EST2ajPsnSwqiVkT\\\\\\\/dnBVywyksBJZpYm2UjmIIBE0fG4qHjsJu+cIO7NJUHzsGESSJrGqHNM1bRNybPJPYm0iyyz6XFRIiplmL3F0\\\\\\\/oZsVnHIqcgBF49\\\\\\\/6gZrCP5dXqSFVA2UC6yl1LGhRor7TGtSaISAOd\\\\\\\/f7PJ7U2vNoorzBdICwM9vGR0k7U6\\\\\\\/pb3c4aeTTVoH9MZBNIXvvLM3cumE+KDODCFJP5tpIA4tcsOErmHSMqBX15ckRTRynKrGDX6DfOyuVelz3JpHTSDVdhHzdX5q03T4lQq4M5l6s8VMCfqu1ImFsfvGcx+tXPMIIx8IulfFA+GSgKJlnQDLQ13iLIPieVI8mWoATT0ad0oy7fSbLUvuRA16ANLJTNGyLZ62XPt+jPAOMaCzSxq\\\\\\\/OzrAy1bXEgWVzBXJ91nJ3HqLruXDbbCv9y8u0yvPyDpucx4Daw4kv+gs8wnsR1BVc6J\\\\\\\/uugB+GedTnEsMLJK5\\\\\\\/RDAouyNzHInGGNjuYk6LKYd0mr7\\\\\\\/tcpuy+lS09rUJq05JXg3mps\\\\\\\/DbjE8upbX5peei6duawWcU9glSCd5mQIO2vSlEbLYAE3gmxMyKdlMYiJ8ch93UyLeLBV\\\\\\\/rC1Rgl5HUay8LAueIE6NRbSt8XEG5gcz4xdwPPhCSLzG3gL\\\\\\\/ZgLwZJ979Ui27kTVveBUtMLI6RVkMUUq2cP6I5XQvd9DIQjzUWZG1NwBBjEUh+xsDU23qOXeGQEdyM\\\\\\\/euOzTEM8GH6pMeuM1ud3outjARxRUNc55sS908xig8yx4sK+2am5nzfxBjhyoBC6KL2ix3T22behO\\\\\\\/uZqf75zanMYhVtHfd6B\\\\\\\/awn3um0xG3m7KrHG1qLZ+3p9S6PzVIf7ChnvCGcgYJLCTGAfJ\\\\\\\/gR+2SYAs2lYwMSG26OtAHAx7snGFxyoQpXFntwQNvBLgtViG29hns+PB8PjCDTgcGngxkbftB2Prq8\\\\\\\/tV586suN9q9IK7e9joCJ41PxA\\\\\\\/KVEkHXAXmXuOUBvQCt66ucpMzTmrJ4yvNKLj840sibww9AtzyDb+yX9t+nMXS6PkFJwZP3Ky5YFBJZR41Qx+1VAVbBafdOC4odXRswzi\\\\\\\/l5fLRriZRgfwW73NrxkqmfX8dTFb9fU8\\\\\\\/WCW8nJqKDiPDDjQHDx2QSUondryZoyB6moqQSts5YqzcIT1VrE98CL7cVuyO4HgbD5Yns\\\\\\\/AUtdMvLc5GE5jg0p1wJejMqCAlHf5BCiKrIVJBi0IFKE\\\\\\\/MZetFdd8inL4XfBcN2hAEp+kJhjS35gnHc5qZkpPYQWCKLjlpox408LjkYq1eCo2jDabuAuGAij9ts\\\\\\\/yoiBXUQv2Wi\\\\\\\/TVzO9p\\\\\\\/xelN5BxGwwLgLtCAlbtWgKMoCMxiZZwk4ZlgTEUsQzyFcRE4HxGhmgkjqIioq7VDe5V27Tu4OBKD+LQcy\\\\\\\/kan3lZPGAEWdCzgwGcxEzfcMYaYD6p\\\\\\\/u5RzvtoSDNwk44HE1rJFHQc+rvAfAOzSSkk2p9yGKe4ASaPofggoh9x8JDk+xbYQUpgqok8d5N2jRk4Q3KiqyI5y0DpR6cgOAIDrXeaYOzvkWE5ogyMKpO98aSrQvq0OTbUOqH7HR6M+BkSj4j7qvj9iy3yz9UvYQMqwY1krH2Ezh\\\\\\\/0o18HyuqmDCFG6zAkC7YQ7UT7cq9z30yOokCqMOD3lQ2qCcO\\\\\\\/NVG3jYFxxXtIXsO9iU1OYcwakODC+H4knk0w25IyGnBSJRiUXOczRujYitB8s+nx0Sn7f1eiIqb25lw9NShl54sT1ECrxdxEh6T4bp+OAZrpeF+CArBptHUGewmgvLN1UOTZLvWUsgJ5HUhoGKDbsnxd3ERnNHB9EiPxjXnulb5kfjCKV4q1Eoafnxq7TQIIyHUQj9fa6jqL\\\\\\\/Fl8gx5YIwjG43brL09jrwGp0qYjkYcMXg6I4b8M1DYEW9Xbftt8\\\\\\\/1OkE1g5IR65jobF1mjc8mcoGPipunQNRrXUU8Gb3sWV\\\\\\\/N52hW0sVXF+Tm5CJztbl89r3up3gPKDkWhkSsPYcimlVCfdn7zlHo8jNHKK9gPifMpEpPNKB+6QPWn0viFvA2WKllwVJnPs8PZUjH45CYZbbgGfNNv7GhnhjHse40DXOROL3kV7843QPy2le+MDSqs9f0Q0kKAy9EyheS+ADGbxdZpsn1U9a0YEwW8Hmx5C4ry3N4d4crm21fGLdZ\\\\\\\/0lgzHBP3MhANwdCXSjK6x99NWhZvcW9IO\\\\\\\/HfhlJrurMbzlojoxXsvhlUz7sLQcGnkmMGC2Iw7cNIArQHGSgjWp0tiyRsM5nyGnQxwW+0FWWDfTLQxIQ2z\\\\\\\/bbQs8DiJoC0qKRJpCbNDRSXvTR1uuk1INqKIGLir5+LBDpluV5pYiYFsREGOPUvbrzDpbfostLO\\\\\\\/Phkw4SU8V7YubkphEtrKhQ26IQARbkvX9E+7OayX0n6RRXFHc9fDkWbDJ5e81uhDiqWBs9UTszH+O\\\\\\\/o8BQGw0QrqzHXRGP6rEtb3vLJhuz54lJQFP\\\\\\\/a7JXQTHfVlxtQzj+he7d7vkYB6w0p8cYaYs\\\\\\\/4XyvHX5m8Xo+4b8FVf1RscPEek5oQPkAcUrse2VYOHhjwPE2y59x0fI6o4h3+lapoh4Y1P+70kwY14s23aMvYEpwFv69BeZwURtlxOXGnKWWfimQaQnGZIL3FHoFKUH7ch4FuAvlbcOg6WfiWGYD9JWdr804Mgx5IcLDT7KTFs+O21zk+J7NnhiunTCQiaoUuwviKYlxqxbFH2ENnq0X3HWzsE20Hz6\\\\\\\/h225c\\\\\\\/UJfwcnngBNLVIKT9Ee8ufWX9yGzkvo2jPpiaterIPUPA8FknLqRNsLXBgdElRIBBM5ypSV9rSnObbR16Ul8urwINEn\\\\\\\/W9O7yUxnIltkD847i1UBT7ShTgNQ0lUyD1KmUlUV6Yv8m8o91A8b4yiPFm304gdRD6cNPZnzFKFvYOQyrzB46jJ3PcYPBd3qd62FJlz4iYHDufLd\\\\\\\/n+JpcyJpxb8RGR0mi5eWrlv7KZR3tl9OmTyLlhH+zpBSila\\\\\\\/76wtQ1muZQsb6FtMU8HklZD8mSkqkqP5hny5TqiM3ixjkARv44Hml4QY6iWhziy0ESGoigp3BycMa9iranGf5gyu0EnwPUVzmtITtx2cQn1KjDGXdcGC52Ouj5aNfernMnmD\\\\\\\/uIgBc3lT1+5nEINCCZCko1JrqtQ9ldRGmLpUINsoPEa+CAuQ7JltoEcJJ6HzJTfiBJ9\\\\\\\/mQG84hNvUd9qcaL1v29rPwGRFWVmFwmh8+AZO0zxMhkmVpdPpn7xVJ+mwovXmjBO3LidoZoaLXEr\\\\\\\/P\\\\\\\/t1RHAGKOg3inpeIGmiPMDJLsAZBilCXcQREW7CMxj2UnWvB7g3\\\\\\\/maq42p0A3DYOn9lnES7+UkAGfJWtjk+dkStg4XFil7NOyL9HdT0IaaJDnnHe6xW8I6F4dUCY1e21ifQWSO90RKH93elDR6REfDrFFFFdgAX0b1PRt7hIk77MYRU\\\\\\\/+tIDrmYE7196zJuBtI2sKCei1b\\\\\\\/QMmkP7Vw\\\\\\\/uzrolYckRh4tpMmxk7dNxuehsQ2MpYXxhztOtw\\\\\\\/Mc\\\\\\\/p5YUlIwZtsBlJQ1cR3IUteOxxK01s9vjuCNuhPRFT+mVcQWY3+XivTymrceib7G\\\\\\\/mfwmOOy\\\\\\\/L3xQ1kaEfyBa1lPOGBf48KAha2mdX2eTanLpxIWwmUWdYXJYyLlSoYqeXB+spg6\\\\\\\/hH6jwmHJEPtnTAaeuK3Y4ux\\\\\\\/3tyiq0Z4ffp9w72ob0Q2CQpdz5Msfcpw8KK\\\\\\\/aVmeoGpGmfhzpZBULx2wiNiv\\\\\\\/Xdf8HuWop7aVdcJViiUuJNndCfxap9c2wkQHdYV+YTAJoA3fKtfrPwWCA+1uBzMZN5\\\\\\\/bRDQOV2S8tljYh5UbWSwVXrIoQQgYmnRvcjTVP3NSz6lyRTGPAR0Bn5Dyo33nQ6VhB3Ph40AHzYoJSAI292djrTLyftUBrX44uS\\\\\\\/onmFUVrDqMyhHrS1A19AM+NFuZwaLj0z7n+Skcl3Uj1eNcRkewo2nsKglGPgw0NudGLD\\\\\\\/Fr35pr6CiHoYGxMI+8En3QCkbi0U5lyvbLF+R8o6PwqtEpbNymkIic+o0N0TtyJrVXeVdbm4MWxXUITSU7EoaG1MXpGNdd5mCIQlQWfdfNmPP8nMfSf4py8Jt56zwkzh8M6+oJUxmQonOFCqhqJGyRvyZb5\\\\\\\/07JudZOCJ8BIuGwTIK5H5yd+zayR\\\\\\\/6cm7FkTUet4QHkajoptwbVkocRe98zW5OaQUpAXORgf9D5r72FX3wS+LEnNgk9vLAvzCcmfoHHOzvj1FSTo0gqwPrWuV3+8OHMbg6arJLYooQvDHhZ2+XZOg6nF6Q+D1Mp8pxAec35S9ECjZdoaT3I1vJkGqkq1zAGi43Tcbmugwb7WQXlDtURKDkdGBZYYvOacs3co3VUnkk\\\\\\\/G5sg6voMNiZb57XjO+HvxmvfHP0jFRScS81gF9rYxhFjqNl7MFMc1wqwAvshvjcMQz31X6nNd0n0gbaDc4P3Zg4qrgI8zz9z8EGQzhf4E8+t2PSLNAE9VjbL9tu7ogddDQ4yCFAh1LkoAzfeHgVgVq8HU7ts83f8ZNu\\\\\\\/bkEz\\\\\\\/3a6ApqIu2ahtF6R0yhxotd6FOnrloJg20Z2B04Lzo1FOVWDhu7eTkZls0Kh0RF0rHE5higMl0+Y18RaOzfMfDxdrFZr93YdnaaBuuuz6vYe\\\\\\\/+31qW\\\\\\\/HYodNyimu3OHi2L0mwSZeDnPeoAIcD6oi4CmUpDBOI7NZwmhY4Exm7mj35vu9sr4L0f20fbVuoV2mNIKzUf5N09Yyzsmgq+gBAyaBHiykAjpMRY0erzGJxlCq0OfhJuroj4NDHUnA9mN0DhUC9DfwkjgQv9D9SFa3KMtSGb2DDS8doHOdckDPTauPx+Pgxt1DyX4HlRLRbaOmr2LSNNzos4bDy3g8d25nht7sivmCffPVWjqlk0Y9LjljR3Sptcx2P1nNsneVXF9ROCJ\\\\\\\/ptv+uerglg7PYi169Unn7MMlecN\\\\\\\/3gT32TSqpvmCMdRlAGajbOhtawyfL2xqA2mC+wOoiE+uFytbv6F1CnLTCKrUgLGZpXMtKsvAniYuGnyfJt8ERP3sMKCqd8LR4XOFFJJU+sjqahTz0AzN3OlctEpfuKti5uFVVFjX3Ut8YNaU0RrHJVwNxkdedgZtwXJzIFPPfoiGI649Uo2z39q3OIgRXk0czRyqO3QMfKspXlQhUfAKm2X28qTkZ7CdH3TcxG+2QbUW0qWWYMhwhRcajkicGl1WNWFDWM36T90F8jpzocGKvypYwZ4z7f1rugGw+Hy\\\\\\\/AlZ3NaUxeDHuMg12gLKpEs1aMFMgWSke8X9+hFpk4HBVLkJA\\\\\\\/Hw2vRxjNOMNwRi9SAb2Dl7\\\\\\\/+wcDlqyJiyjVfUe91pUSvFW2tYCW325\\\\\\\/umg\\\\\\\/kuFQ\\\\\\\/0YgWzkW\\\\\\\/PlU1wIlLfmFv1nftJPFc3mzWebhxrd6umCsXmNrkQjeb42SKqujR\\\\\\\/kYnxgYltsjN2zsReZufFFeBR0oG3iOIaXYCSfe2RG7m8h6I3eYjUIUm8uf7EY9vrrClHIg4pfO\\\\\\\/GWd06wAjJHGjYWxsKBwiKwTucwdX4JYftAlG+sTMq6rXg5G1zmLpqYUZeJX\\\\\\\/9Cu2Bw0eTtBiYvguVmamVzE64aSCYRALf\\\\\\\/0SJy4ppO8onGNeqsILsKVqOxTQVONvAegya\\\\\\\/bIKXn40cpzzkim+u\\\\\\\/nKvRey\\\\\\\/90rhVommtTsl6XdQ4W0+O\\\\\\\/AGqg760zgGciDrYu8n+v6Hkf\\\\\\\/QcaznaGlyMWvd7a5Avmv854MKKrpEj8eXa0JT6ohtaqMc7GPATUEmBSFjGmzvbiCuAlZMya9hUBhJTHY5L1\\\\\\\/k8YS3aKefemAe2I0KUVaE5z2LPd61okvxFOAOKsZU8ESQlCw4x5oC66G6YVAX3pmzBORXKE0vJdOcgep9TvwrStlo8hodCVclezE5lDYqxR5o8ih2YD3EIM8cJrDT4658KsKweBvO6IQOHh0rEpWGucFEp2\\\\\\\/UnDY2x\\\\\\\/adVCZI44VSBx0mZE4f+NFiDFqLE00l9xFV30nwKhDMt8OSzHitBbP0uK6DSMhAzBTrgoyofV0cG4XIs71eVq2onp0j92jnMXgKcm2J8gR0XWuiZbiEjw\\\\\\\/BD91kugZTsCNZXLILT\\\\\\\/USEvdW1Cn7I6bWgNsV++iFIT+N7Sndy+5kz+YGUpdEg4r0FVu4JuBydqdY5ydpicd0FbvVJ6vqaMu2zoTlk0F7mH3KZjQGj2Fjv0tj1hANgYk1x2avjw5Wbw5\\\\\\\/siSoeH2YoeAOA9aYV2GMHdrg8w569r8VOOXpOeYMyCVDZiHFC286Dvc9kvXm7fpzdgqAaUAl+L9N4HZRdUpqlr9Q6rsNuTR4peDlH1g2MDZZPjJdSRRJbkZasA3pBLMrtqFRni1tp2dFF\\\\\\\/cTMgQMTsLfoTMx9SN5HaSLYpizZuFJY+dle1UCIsdZ+F6tcx06o176Pe5rDrrksPMWmNURG1BH+EnkKMM2D6r1BBdppTwJwFKkgwIexU\\\\\\\/OMIRJ1xnbEXL6E\\\\\\\/YuD6TOFjIuxt5sDUXq7lB\\\\\\\/dmFg1bq0njcmN+riAJmKwN3nC3r0eEbCq8OqrqQZUdJZ+VpZwiCijpiKmCEW\\\\\\\/Y6bl\\\\\\\/mQUNgqD1N5QRhPev6ixAPghhESHyEtuEsUmmKpHfDxLWE17tsVgYurcHjPJiipbOCL4Ci3MAXJA3YC+kQneQVV9dFNqBq\\\\\\\/TiKIMBk1wC+9EkK0oZ5y7j6SG7YCpo2bKZKMB\\\\\\\/i8qwXZUyHOYUUwl65K\\\\\\\/xOMccJcurpiXh366GOJ6kEL5XP0HkRtkcphwl8S9la5RBiJbJjS1UStOr0+OgYtPB9PGg8Vyq\\\\\\\/XKw1Y1COrbXTNfiX0ykOWVqORXYvbO2ba0fem5gK4Ltybbqsk9fcX5K7q0aCMPZzpe6Zcv+zazVVUjqQ\\\\\\\/VjlLZql4yzYjkye7+8eBzYcajSJwEKKImaLKawEGcccp8v8PTxBYGvYC0UY+lwbWd7AzNB0f3iOY8KoYQ8Cjs3XgQrBCpihzyMrgI7yHYubP77ZDNaG2U3Z9vuva8+iFz910OqubwYu14i7\\\\\\\/WUW+I64cYrU9ZmjAKLXsAVsfKEjs\\\\\\\/aqHclP4l9NdZ6G4UyEJPBd6aPcdM59vMyjlUb66MEozBRcX9m5ejHGgNfxvjWNQJHpTY8ZAu0YDC0cRBSxl4a2348r0cuz6rn7Qg\\\\\\\/uAX9XJJJ3Xj2O+8uxvR3NI2U6uhoymFwHnmpoQH+aiPutNKP4vvb3mdtkiIndp+0bPk3DTd6ppeC4sjHMY+3r8c8z\\\\\\\/prIYLreA2o2XBOs0m+arAMJ8vR8hOfj4mg5OncNhMdTGjkX3\\\\\\\/qHPVdb5meAjwGfS83VR7jjuywVfrD\\\\\\\/DVvPmDTRMCQQmRDiKfqKRAif12cPwb+glSqlRX\\\\\\\/sxgcOntx+JzTisFIgiP\\\\\\\/EBojmeChIamsL39dCvBod8KAv+kOc+uyIRVaEaYgH8dxlTmR0BfBK\\\\\\\/dT1X9Hp2YsLGJB4e1Stlhhm0u\\\\\\\/pUKXtyz4ETrAZw3\\\\\\\/nTSCDM4oHPHBTf1j07BmI8PpeBgJcT3kTJ43c3GdEdq3US3L52uCFVBEolgaLvTWBRY53OXmp7uDJuo\\\\\\\/KT+NoZCLv2YJy4jQmNl\\\\\\\/xtxu3FYnmNdlUcomaPtLy5voJ\\\\\\\/Eccjp0cDVl22x7v83hKo7q6T5tuFQLlWfOOuTdwqVDCqk5G6z4E1wUB8ZsPHcS0wgN5XmNzflNTdi2JUBvf9J8Ya0NC9qHf+z70pYBhvF6bX7mLOVpOnhsWQ3lew9c9OVNr6IbTYuecLaESX9iJcA257xPw6ghKT5Q9gz0RPrFxXYz0wl9LqZs2svUZ5a1xWHCx4xKaYZf2eUnAUMCrgz+FmLJQRrKX0fGiW6CTe+3BR4bLaE3fK7dcbm\\\\\\\/vDvCEZwfs26pVs0zUsmymwA\\\\\\\/FcA4XTHHIiFIJC\\\\\\\/cac1zD4cfcxcHrWEf4oOuUAwhRmZFyMeGclzkELhXPlFcbJ+3mluHaMtT4lw7C5QLas9TEz\\\\\\\/CVv4eotZoqGv3Ws7w0wIPYTntwlpNjDFrXPk7BCRXYPnYmaSFyCfIqg01hew+eTQNnvZcPRk7SoZuvFmYQYYHK5WMU06gD49eUAZ4ivC\\\\\\\/4npJ4Zi\\\\\\\/Qd6y\\\\\\\/SO0GcS1tr3x3vHRTZLt+t2cEduoq4Rk2yL\\\\\\\/nCNF\\\\\\\/WnyMMMs9pqC7bucnXCdvAQyGTXpTPWY37c9xd5Qz1M2jgLeVvQNcxQmMeTtUNsCGBJ3w\\\\\\\/z7cxPw1ddZHKOugHX2jaKDM7vnohGw9EoWrBXchu1\\\\\\\/7\\\\\\\/6n6IB+QAVVFWHB2cX4lpkjiaf2Tqt6Rs+urK1yC01700tcpghNvk8vYk3g7w2O14hAxaVdl7H4oInuF\\\\\\\/XBeGRl6VTiZu6q1+h8PaYlSOvw1dK7l4EkkeelTTPqgznUI9lMryW1J\\\\\\\/uQjf\\\\\\\/61VUCPOrT92xKMpvBYoKPGrrlajyBFLe\\\\\\\/bPTI3kv1xy9i3LQq0dqKigu\\\\\\\/kcIvyxpPFtg9ujJqOwzrNpbBKT2BtJ5UufNMX5h78Hgn86lXURcvGdaSdEfpZBPV5h5QpPEUpdQp2H3aOIGr8od6CmOh4i7ENP2lCypDBpw5ROe8gZFb\\\\\\\/67UOQDEphNlLOvbdvVcQD1GLgdOMi5CohSJPymnCklEltQJbQhS36McBgUPDOvjrlyvRiTB8wf19Va+yzHJD14jvkaelxECWxWrg3k97b6pZCNzfjhIzAuUNAvlOBm0eJm25\\\\\\\/qbZ+JLHkzolK2O0ZLAnjiXQiexAY0nLg2sA6dH6L+s5DGxVaojrv+JKmYy57gq5QiUauQSaBfOfLwswQ8RZLHcDqm4btrLU1bXLm3aUVMmRmvIBU3ixch4t+WfBf6R88+dSk2AisT3eqq0qWbp5ZUtBc3KjewJ6cW\\\\\\\/x6fszopsfIdrmo\\\\\\\/Y4qJPTHMLDMtglh1E1gbKKkejTYSxWTHJHlqwimfJBQ9rSvosLwJPiPXjIL3cvmuei\\\\\\\/R5XWBnTWy8mroLrPwXfHMIbNWvTNriMYb3kbOmJ8hLsQ7MSrPASW+sJPDuM1slVj51AkcBE3SFhTbZolf3PT+riO96uZAT2e2MmiB5o1LslEFeDlggO4AAADGKmvBlnIk\\\\\\\/7oCqkuaMPIZSjqTAdS2YwMRsUI3iCmy\\\\\\\/TARxmZqeNlJBIIPyqSMHCWnRiCKD9kRMrPJTMBRFLoY2lc1tluffkMQYH5BctwSEiJJdW7ol3dZPHqUMPeEcvLAGqreIQo9AkKhuU3KiF\\\\\\\/VBtChUSLp\\\\\\\/dhSgI4Mm9I6IrRCLqlQtJ6kMFxqx5ivvUKhQx7t+OYpnzg428YEfxE45xu7Vu10Y5NchNI9gCFAD6\\\\\\\/nBcKCfo+0T5qeUzS22eK0s5z9i8zXlnepsSQVaATUkkMHSH2VhVm2jPBtsSRtCsLyamtSFc2LzLUlZux+Vv1tOznzz7lLzhVN0RGbgRMCvStZEjdipBb9BdtIO5J6k4MaLB65hbZWc4Z2fXPPPXVwUTs\\\\\\\/v++QBWHfpmWzp1vTw\\\\\\\/V4Nk7ZnTnRR4vR6z96UqRN1Cyg5mlqKn9SFK0xzxXzdExPidi7P7pFZPZOBOeW7XWqz+lQ3RANwOo4yR8fS28crV\\\\\\\/bemDh2ZTofWdJbhpJLtsvdb7HfxgT1fZnIyCNN5iLXoja2u4Ukmpwkof9oLxLbp64iNSxCOgWoPK8+dbjweAjYTqw0Cs6HBW25ZfgAhR4N1QqWA59eVH2TdnA\\\\\\\/ZW6kgWmHCEl5eyj3SBA78w26vUvNP1QTwaFHBz7z8+DpKb5doPfA4+V2LAz4u2Xt0leNi4ZhxHZ4v5pdW0SJVxtFfsQJMK8KIfsvxI4NrEW0yw4bniqfCJNtb0N60l3UHcYAw62+O63eb8l4znACP+0sLqqhBh2VrH5fNAWJjwZDNopescqusSB\\\\\\\/LN1ULFr9E6LNI+y72V\\\\\\\/sJM4u+wdlLB2EoOYYID+gAndqZRaY4GnG3UrI+37MKwWYQOYJ1F+I0af3rBPRmlnMekwSogDa\\\\\\\/NS5G+y11qtcDw+MhE7r28HZKHRQB5qFES6xTegefQZhRC6nJwKHmUWqQohBTuIMtLAeSPB2qZsmsPz\\\\\\\/kFtjwQtyhefZSteGfnOD\\\\\\\/k\\\\\\\/rbWqwGSVc\\\\\\\/knBEd5GmMCdiBXHBEb19hM3NJpqkPf4T88K4wvCYi1yWfEO1mNISSjVrj7bpIXvlD8Vq3uKdIwiE\\\\\\\/DGtEaoMp\\\\\\\/g4giBa78MQyHS+eJu6C5fCCjj\\\\\\\/oycZpI35hCDYDF+oDXf0J584exxASYHRRhqkh+kDooLhl9tn6nISVsa4u03ZaT2RCqhx9MkMJHKB8m4hw47+byzGvu\\\\\\\/tUum48vBtXKqsg2I01Nqbybwp1LV5LfgNG3IHuyVazmVLcZ9MNL3pdWHsSCnbypGE\\\\\\\/xyB6vdzyj9yZ+nLF61mHU10FZOiqjE\\\\\\\/kvL4I6ENwVgIko2nlZ1eLGqYeEIYzm0RB7BTI65shIj7TwZzkoMKUCPJt4nHAzNpo0+CN9wvlEfq\\\\\\\/dEQyxd9YtBAkF0ifTUFvVB43NeoQcjoZgn30Wy9MxJOGzsnCIL9Z1LatnYUwwIq3uVPVpwTEBgfN4i2e8PY\\\\\\\/UL0TQbDX\\\\\\\/OwIICKAPmI1O0WXhPfV0ayUiBWh4CnZ5ylofmeoO0bjVHC9UeE15Kg6zA1JHJP\\\\\\\/pCYFqCdlDh44NfMJgB9\\\\\\\/0aMx3GvUaHIfvWYghV8NLcTC+VxFgZbreBB9w7dNlTTFUw\\\\\\\/7AYTqzeXIJ6Q6dUzV9t9D+wyS5I\\\\\\\/6SIlgQxxvubnwoKytFS3\\\\\\\/zSNfzwOTEIXv56KmQLvEy1bwiUxGGqW5oE2MR9PrJDc2Belk+GODGImr1\\\\\\\/Jykoiq1mIB8O8RpySCaT9eL48g6uDZSJ8\\\\\\\/UVXS06nMq9Goy1pjJnh0tuuU\\\\\\\/ACCv5pkJfnJ2Fikj0RB3DaoTWwd1jOwN2FycVFO0E5y1IpOY6jPsvcrxyWmTN2WzftMwVGmRKTAIEHP6XpACmx4GBMa92Gznrcuc9y+kl\\\\\\\/8vwutyLyhz20zGX6z7v1BZz1a745rlrTR2g+luvpdtg1raOgFUJwj7qHZo2msJbO3kKPh+6hZXXTZcUX8fn1mlyNsj6OFqJRyiFs8i5Xnfs\\\\\\\/Dg+YPfrRGpU2UvFK4UsEcxqkK8wHr\\\\\\\/tXglK2wEUS9MgkdrUVqxSci7ZuStEZsw9MIbFf8CRp5XZhOnuYBi4sMN50FZ\\\\\\\/n4TyNZr+5LUO6lEUsQHakrR85b+URTYVSUlFlonDzRBfkawUMieTCxVVcwXuLjmF8\\\\\\\/eZ02WGqL2guyQP1QFTYsQkXAAQQM22WBJWovqdS4hwArboalbLPXiUtj1YndWk7ibtPN2PcIePBYZjBpsyoHjbagRquJwQtasvf2T3PBa1Jn2MW8RKMD9dD3\\\\\\\/vM8j09RC3Hv1V4o1dEbfz5M1yOK1eK0LgfWGJNXdletpaMGhDuejTfgToPd2cyB\\\\\\\/L4IhMXRv5lPWjTC9rjzgyv2AYM4m75LMw7VtmXYnDWV8y+Bf\\\\\\\/Po7xnfMbIGt1ta0TyNU3kAY+HoZOYiz\\\\\\\/S+KfOCLBecD1u6gwaQ4Uzq8\\\\\\\/GT6pgBgjBMBAEjTIVdEUmEoJQVmOn40UHm\\\\\\\/uDLpXo1CnK+Sdv7Gs2V9wxQfwqp48\\\\\\\/GrFON2dV2hrgMsi9O6JHMhdM9ITeNSdFpDFUA1FFLHyUQ5xFTG3nGj\\\\\\\/ukGvnST03tKqT9TtFwnuheePtIAG7ndzalVK\\\\\\\/OPArDcIdKU3YAG48Ef5q5xC2OEYatTK6Oun1fuhctie93SpExUbCje69kY65j7gSXM6uAkDZz40\\\\\\\/yoB+uc8l6l0TOW8jb6VMOUk85E3tnmiyijr3bLFNUDO19OGSSmixfjz2klUh5SM4we6VEIOzV\\\\\\\/jRxN+EHIcfuf9Z860j\\\\\\\/YZIv+Lsv\\\\\\\/3TUN2SoAOWhppj5bVTgPLe3vlLbyuFWKulliWXutGVYx3vkjaNeS71ks1IAoYC28Ev9UmR5YYRsHns7qDq3AeLjtwyyAm2gkcFsxRNVoZUtKWz8DDzkaLlbB8BlLVOzQoMwmrkERT7fkkhcqjHIHdYCAOkETw42NstTthUhM8T8jj24er5LtfrC\\\\\\\/YCfL1ufoS980m\\\\\\\/8LP5jJems18dHnoSpzo\\\\\\\/k22v6AVGRiVjE8x9XImOqxBUXcCDOfE+bBJUBm+jMdSJeVKXWqeSrnXBqOqlum1tTuNwXjTKDeTJHb9ZeVJPl\\\\\\\/zRis+WqXO+GCuFymMuBgrBO4DGlbRjV14UFkOQ+FPHmD6nBj85sMFq7nM\\\\\\\/kDi7RDU0+W7WxieO2hSM8BpMPIcwnczw8K9HERc6ZkSSpbBYTu9Sm2f7P9MPj1g5YSr0VZsLNN8ttz24r6zushOX4tss6Q+RKanrq72o+KD8cedSO8MNSm1Fam2K93sdQjCdlO5GyrLYeBiaLHTCRLUZ6Uz68fesQv22Mwd1G7hCrosNNdDTwVwF\\\\\\\/JpHpQrBKStg\\\\\\\/gWfvuisiIhIbwwld00CknwlgTV0nKhB4wjiG87or1GY\\\\\\\/fian8pKgsLJbdwtiUcd83qTCsvYyZN9a6ieCE61ogyaA6n0djrY04o8ug00nJ3wKrJrUJf1x\\\\\\\/yE+v7YX24K9QWpL731n6aTGyQ9yZ+1zn\\\\\\\/OlHoducaMGG9dmCg6Sx5GbUz5Xi7c7v0soiNCDn39K\\\\\\\/i6nZKoePp\\\\\\\/xfkjeZLkLgJeyKK\\\\\\\/XiUlVq9LT9uWbNFTtRJJftCaw1HuDX84KZk1zfD+FLJ3VrNOeISYfM6XvX9bs94SThxNxN5XMXsbe84uE6x92zv0Sfn0sGVOk7VnOSMR+lduckDdEs8G2b\\\\\\\/kN+eB\\\\\\\/7Quc7C7TRwBgovK1yCD6kSSOPHb+Qqyhfpf9WjaWG1mKVz7tv5g7AmhgzqxhsOB8gLippJg1weiYhZ+DgxSrmE2i\\\\\\\/vrI1IuWAoYCPR4EefhOupJwQJ7cyBjalyQbQNthAdnrBaC4WIe53rSA6NyvBr8p646XQyBTJ1AYBiqopovml6Owo2M49JdD8FXgWsFEAQ0VYCyZFI7onWlWPuUUtFYSr2SPs0G+CL9ua69qNWxLVfTYqULjgnEmq3bdGRFw5bl2RTCaJTfQZuRheTax4ysgTw3P3t56D4PdDtOXTT+o1UpK0QptHyE+8OpgfCBfl+hMGA0\\\\\\\/i77dEiUNiKLFv3V9EOg6Qj2z2I0eiSTOOXo+6tJ7Uqy\\\\\\\/0cwwoZsEOxTptm8imnXH99geCeeR3wfsA\\\\\\\/bnJcB\\\\\\\/Wi1VLMZl+9zg2vTp5NVU1GHo6vtuRUfZmIn8XDtKXTR4E4XldIo4nBMoSdCrXbsK0TCAIQu2c9vz4H3kKWzul0lgUBDBq6ETN\\\\\\\/71I1jEBg810TfD8FX+T2+RJTFaKIDMxkm3tSASYoJMCfgai7xOkZWrjAEvh2PeofP6GMqnT4Zs\\\\\\\/zDXPdEgkDV6GvCITKmUPBSF6diJ5G\\\\\\\/82dNV\\\\\\\/Z95yB7iHMnS\\\\\\\/77oBPQGclHTGX+K5Gf54CHRacQ65UEs8v0I1G2GBBgwzESsehLn6lP19VECD6cGXVVeUGfuK\\\\\\\/45UUR+OwX0PvBfbMhDhDPiPUia1JYUWTST\\\\\\\/fY\\\\\\\/suJrlIrclE7Yp6HIgkoLarvgVx4no\\\\\\\/nP1zAVX1ALVr1waZtvgl+il2Bhq3KiVHEnFHB9oEPmY2GOy\\\\\\\/aznQuvOCdSFv\\\\\\\/qbc\\\\\\\/XYtDa3CIPN22KTiPd4fpYKX4nTaPe3qn6V681jeOQ55VZvuGYZO81cUqfIH3yVoZJHV4UwtB5RUmbh+LS9n7gNhZLSwThMnS8xAeY8qrQIvtZMZY56wQ1vp6TXHunetcIAX4bFrlI2w3F2mPbTflN\\\\\\\/u6ajtiloS7jiCodiFoQ1YqZrfSGNCuzEstepc6eMUyMZDwa2bH+Qgroomh9KxJvU5rmTWET\\\\\\\/yZESorOqX3UXiQMj7glwumhTa2k5wD1eY\\\\\\\/PfS20BpCV70KveBRvTW6LSgaR8SJfcc9SRw5uEzTjY+IcUlOn1NFCkvJ6ks6F3P4RugbVwqQiVtnElEYA8xtzM79Zier+i4sFRCXULfXr0YWSW8TaWgF+Uq2mf3vkhByQx2buv3w3Hd32AP\\\\\\\/rdEuFlIb54MbszENVURHdb3tNvfO9XUBp6aRX5+NjXFhJV25P8VukaxsU0fU1lsm0VgSo2RqtWlSFtRQZ31JbHMpUWU9HJfCrjV5iivIehYk6VvB48j7rgFuR00TlquzD\\\\\\\/sJOgvfWfPbtSf3drctTYjwOv6zAdtsyRFG+8eGzQpcr7ujBFYNRXdV55tvrmIDCFAShEbZ+fg6xP8p8cC3t4re2KQsEmIXPiV0+O18Zn4lGTYofm\\\\\\\/cNn4Di079j3yFWBOojRWZ2h7GUuW1dl+pCMysFMZiR5COnyWkz6qH3Y9ZvS\\\\\\\/hbJzdJ0OWIsXhbNDRDGQg038HCC44BYwKWitYxK\\\\\\\/5YSJmdp7tPnq2ZruyeZ7y3bnWPmuAPN+V\\\\\\\/iDzBu+MOOck6p2g5Ofb3XOXlRN+iYHAd\\\\\\\/nTkMXfA06Brt7YsHbDhYk59lnLEHkOEhf5rLKODa2dkPJeLg229qu5V0qmtGKf5AsQ99jAGhw0aQ9uVgZ5CYcW+exW2shffQWVS3qkjY94PeQto71tmOJdKvNCpquzxjF0wFHNjVBHaGgoP6Z98SZXJkcHB18JjNdwoubrI9OgwPHG+Z\\\\\\\/gydiuegyhxHX+C52K0btBSzgi0cXKTqMgMMlQCeDOpRkAWWifMrf2ujJiZQyc\\\\\\\/HW0L2y3GCecBvHRW8taVMTYMzQIjvQbGSpmfEhp5Q9PG26GD\\\\\\\/z7gukK\\\\\\\/O64q6GpL3CDahlhYcWKhib+\\\\\\\/hX4BMHAeXB5KJC3Rahrw4eU5D45GqMbuvOslHqIHy+knfkgBfFK9YwkjORad4aAiO6u19vXNZOxBwf69qRiNErgSALxjAo3Sbl1GsRiU29vn0Ggfo\\\\\\\/TPmagvM75BIsHD8otLQ8GpKjrp0hAp5AvpU7HTUJkPL633UfAx3JFAYh0nNCfQ9YU+S8YWjCoZQqh10Jd90DssT5Br39bnKXwudtxon\\\\\\\/+7EvYEzKCeOR2EMl2cOU1B0wE2dJEns0uhzOStzOD8PZw2IOagSY2Us+X\\\\\\\/hZaXemMrSKzVYfFzI+y08LOgKl6y+oIFVB\\\\\\\/vmncMz+TKXHG\\\\\\\/FxqX6NNYy26rhjAm8jv6YKabKMhGHNLJRzUn8UYPx74wpVLY\\\\\\\/kOSUUMHJRDF9Vvgatz+EdBY8sI\\\\\\\/2tBsdXSQMbfPqrGiSoOrw+1KJNh2lPn2gcx+RRCRXst9ShDByze2MKB1MKyOD+Veh+yBWlOOC53ljOGIHSNhXFQnKIr7m3JXUuK9q34pg0qkb1YBki50VI3S8DMI54+grKjNEf+dMPonVqTEKvU4PVqjgMVbCIgq3b3KU1b75mtA52HeaVCX0wwgtj1MafBZadtwxCl9erdxL+iA41SIJZDaihirS4WBOFQxjt9MiwtlEO5Yfg4Q6cgKWWZAQBc79KRnKIHHv44TAdWYpzEvj0exoMn9aZhzp\\\\\\\/x7HEWXc8MSAr\\\\\\\/x9+dyfk\\\\\\\/DIj2w89ysDes14R36tdxcT9BAH2J7w7to2dGoCSbH6\\\\\\\/VAJ\\\\\\\/7RCOkDZ\\\\\\\/0L7\\\\\\\/9BumamYMQD4\\\\\\\/YIRgW8stSwlPzSkF4\\\\\\\/TJL8yKmYeTzbS9k99BplpUV+FBQZReF+AmiRCrDpUNxWm6QOo1o6VAaRqycQJLv5SVrwtDmERZDid8qMjB38lAHWqIkkUApFrmJYd03kMXwKbzEKGVDE7x+TbP9fkM23C2wFX1k1YMk3jm2MktZfGx1JDUNa6W\\\\\\\/bwifBtYbFKy7eab8d7MxaYaUB+fsmDyY6NdUVZfD+H8g6KGGyoIaqC535mXGLZr\\\\\\\/x2eRORQDVPzPy1iPB+kRmPIU31fQbvYUZPm7Q2rimOQJGY7Rnv5Ye2w0DiugekiQmvHnq5CTboge1AWWTw+lER9Pg8l4yKv\\\\\\\/beiHsfQvOdv3zhOL8e6tCO960zZJWyN\\\\\\\/+FaFxaYbP6pf0vARLgnSbnnatUSZKFIenMT5+92PZd2FCQPHoxCB1hyI9M5bRacDVxjm952AGa4G0ejTs6TK8L6wf33ONJyVoUpP8zBE3DmNTAsij2Az2CsDRXS38UEQZqzTwIQnRAbOqi4L1qbQVOe\\\\\\\/LPVUykr2rqO5ATVSzkO370bexdgI5caJ+I6TMmlpkdregFS+ywykHzqF\\\\\\\/MD9hBWOVX2L3AvXLqT2ZkU3xVDbbqSP7ZkYrYeIGkZqm\\\\\\\/ufksxEzf2aX9GcV3kGxfSPd5OOHiUhUbAHy5xRy23G6esDXIl86PI9DKdRbwpymXXStLVemPJgoSVsICituLwztU6Z6xKwXkDksU68TMT2gpQ3gJ\\\\\\\/piSFQl4jDCVJmX5V5riehvd9uw+kBzd7DgkpFfTDRQ0WMUraNlQLdsKoOJvM0FNKA9C+qS3fJh\\\\\\\/CYZLTNegVHdLkk2g92XG5wp3dFSeSUV4dgbG1R6i2bPvj1cTYbywbBp0+weqjXxlNmXJh\\\\\\\/o3NYufn1KMCQOhkubSrDxoTsOSW\\\\\\\/pArVOTds8rRlLjgE2i3eXUoUfbkJf69wNkkWREI0+zPZndf9lrFyM1RIRyvueKD7qesrFn5kXn9YNKyNI7u\\\\\\\/A1jZ60kYFOXCesWBq79u1Xd2HnZQvjydXjl5l8qNyMJf+GnKm6XoWK\\\\\\\/jb466N3rf4kSnpmqGqFSWwOpq7LZr\\\\\\\/3ErwE4VGF7bJKe7W9u59oieII8c7i6X5Sz1wZQNS95xnUI3HMnXDfTxEHwzqgYJneF5\\\\\\\/3ZQ0kDZdtJEqrDKmRavoxwFDM1\\\\\\\/jfu95IeYG0hgv4vlBZc1op4tJ5V289olUCo6a+4rJ93Gm8HLjquiK0PDC4gOmgkWD\\\\\\\/LIY55WCyTdmAtGbQl\\\\\\\/+3kImLDm1JfiKt4zRRIQp1GYk1mlGjIXKSxxyzgWHCkdp6NasXa9VDw\\\\\\\/HXvpBxJvaC4GRkk2s45mpJNo8VL7D2loYcFgNVUl8VIpIAo5tMeEa1nkpq5zWNldycwi+HcKYzPg57q4lQYtIWUtpDUZOA\\\\\\\/odKu0515BVf3cYmBZqGQMU7XBAMmUHHiCVg3dwKRQ2kggol0dUBe3Y3MlD1rnieG8oCF04s6q1I7MrPqGS6+C4VPdVk7Bb25\\\\\\\/inqerZtnI59B7D5Yt9XzQR0W2QVgwO9cLU5nBhiVGKRHXoP18znJJ9TnGCXjRfElUN3+FZ\\\\\\\/f4hKWyrkVMHsM9Hz+pbCsqOiZV6t5ehx5QEEiK\\\\\\\/+U83Fa+8pjv1Cl0LAgTBCi8tB8xDtXcytqvGUwzMogH1FHWOwKTletrL2W07+rhPS7Zk47CieN54L4Bq\\\\\\\/BNTC3OjB7auEEQEpflfW5W2jQ\\\\\\\/HNacLs3MByPtIqUbZU4OsPfVf4PsmPguC6cbUOSYjEL3Nsh8Xl7fOhrfaM5EB6tbikCDOZHiSRZYbCHQ7J8V\\\\\\\/wUt5POZ80buxg\\\\\\\/bF8vTseicO9LrwIplq\\\\\\\/Yn83GJ8fbd7JBL\\\\\\\/RBS1HfMvWOEnafq+u4G1dwVzTBhWvO0o7UNQTVQ3GT9WMPXSJqA2dd8BWSSvzhT2dBOSDPBfuOrMvCeyeUGpUYk1JuKJZsuUZk+XYa5hRyffm5qHM6uA0Lp\\\\\\\/R5zGkmz8ak3oxbQshqErGvgRZwC8KA6jYezNu0tpvOX1BocZLx6HcvaarUnWDKjrJfNsHUmeHTapwDxhjNT3CA9D2\\\\\\\/sO9Unty8TonLGUDqJGw7A7aWgVYSXz4\\\\\\\/Ou+nBpFPgePGYuPe8+zm4cmgQBi7P1Wq2b4t3yTh95Z19fvEo90NAQf5EpgDbXdGlOZv9WnUUzGJtMJlKHD6IH3UDL3y6+aP5dg+FHuU\\\\\\\/rjN1shwVORY6EW9n87JhVoneUf1WpnxsZt2t7y4OoSHJ9ETA1gtL5PpnYsL4J4E7uUpjowyp4DK4L8OeSA+6fak6GhhfwAwfLeEpfvSHg6Evm4aoOn05USLqg47xJ40MJ5hUynYRj1odlQKatTLPxcCsG8Llse+GOsJ+fHLzuuT006BtVHtFSVNqvS2LbALFIRJ7+GiFVMTfclZQ7dnXOm3CUP07qMRlEJTllJq43GsYjZ6G2+S5ALHMHF+5y3\\\\\\\/KjV+tcGqUQVUy2yELu44kAvTvLKP8Az8m2D0P53sb9Q+JbHd876P1Y8xOjnCFUreDYXFSbQsLWdvHiuhCiym7HPgPhf9QN2N9q7W4DccbCIiMuSnCJ1Y\\\\\\\/zOzCXVWGsVlyAnKBcEA\\\\\\\/SQw4iMz5h0NB00GUdEE+BpHSTZk8mzv0jbI8HdCJqFHqoZIqd752dvzStIOu7ZlWwKY+HmsHAq7matMSRr6zPdIfD6CMuoSTml\\\\\\\/22\\\\\\\/MSwfbVlA\\\\\\\/g+Uwy70ZOQck1ccOhTbI02kncnOLl4E\\\\\\\/H0UB2sML\\\\\\\/MtVG86H+z+dr7qmtdPLWmuxymW22FH91MvBpIp1OU6Qj28xVslTWef73fzp0MxwRYBiu\\\\\\\/aGE8jiwZNBHhI5qIS0SwdSFp6\\\\\\\/nMAkdFYoawfj0SJXzwzfzIqPtIg+1G77YlPh+KPLpLxc0h3ehBjedegqk4zcCoDpFU8J1CW+peKNZ+\\\\\\\/8GuKpx2g3M30R9M\\\\\\\/H4g0wbeTzyat7FwFF8Lqy8jFeuPppIfzKtz4Ia4mGEv\\\\\\\/xnKEpevI+VYTYLyZNKlHWnH0719R4kUDrZ7uP+t5Ne9iSfrCgKBByDO1MRh+gjzAXqFjbIxckMqYaU9ZagABovWpGozjCkllzvSJeBYsCxUqHgl4G1lSawxB6OhHShI8LHU9IYf1FQOXqOMLzs93755smGBEuSfzBiv\\\\\\\/fHVWL0Fw5zyHtJs3DYwZ3NSkmXnaiwb2W499YHNr9489bhoWafZBhQEPPaBIyFzpESmw6iSnJ7hN3F769s1k9O8WzRA7JdUsFh9mMit4hfyAGjz5m8hl+U7SaxKjTdrWdAMOvkmtuVACPYWf6GqI+w3xXIQevJ80VHNjpw3v6S8yY8kBb3r0l358Fy5odXrVLy8Dgx29Oo4+O1YoNggO\\\\\\\/Oafjem+oFLlz9TDJDETAvpmdIW0peo42xMugBwLdwGs5IEo7+OdD+cVlewZB2ffed4VpuzbTmS1hsuu9Kn5fFN18\\\\\\\/1IFDZF3VVBcVAoRgJycqbdaFA5W03jrxNKq7mrYeAdA1O2B9ChaBiWdaqWQ\\\\\\\/1ZSA43bIAoEiZCafE2+tIXGT0i1M\\\\\\\/gJ+cz1MlZkwuUSVahTak1aeBuUgZ30UIQWrRFZxgM57kHkLug6B1l8NZpg9EUarLFmg3awQx225c1eJbsSBJAaf+Wp9NKAc8skjHGQWtnbVnpE6IM8U5iGOwducJPbMFW729MuZuweKIj0\\\\\\\/EpzLc7dAjdSbRwaHyDwdnEyC\\\\\\\/J\\\\\\\/Ycb3s\\\\\\\/H5L9nPSmRN63E+QSpiUAF2FfVN6gfOgW2hdJUhU+uJTESLSnQ1X9UK5FgcfhW7FYBb\\\\\\\/KheG2uoQP2BVn8ID4j6BEwkgxcggLuvr3wx1ULoBn6VSS9yYaNnLr4sQbfqkHSw2BBPBTiAwmNJ2THQskr9TsoA89VA2ITz9qoTHelluT5QxG5EcEa1zREjPiE+ZtclpWLK9w4jt98Vv85ljAnTNhxk+x1JrTSu7zZcpDXXIodDovmA1hzwdIpy2XtmRmQsmAN7B3UQrl5Z6pzhA0UZdUBcNxmYyzprwj38R\\\\\\\/KjoufHaxzu32MmaUZuNek7czQQpmJrZU4lQGohpktPXCVoKGWZQS9aFiWU3h9oqWS9a3TAI1u7HyEkLPb6K8fRvawKp4BHn6cNtWS0mHsd6HxSJunjYDyV6hia9IFy+qXtvPKBhJIpXBzd7NZFDGdcKAse8ZD7plqHwbw\\\\\\\/X4OfKktB1IFoJ6IJkIzsuL7BF5hW3y67DBZAg1oNX7ZEA18hcL9pyYSq8Pcr+gnwRm2o8CiAb9+8Hm0+ziwVo4bLvcunDey244k7BdELP0k7nY9lGUHSfGUfqUKvKeKd8UBIYndTxzgjY9vmpc8WCpgqLtOXbKAH7DC391eXysUy1N6TTxvEVURrud1E0tWbpBMInE3Q5o9C09uWs1S4CbCi+1YCrTf9btF6GfVJNVc\\\\\\\/ABOBiTFeS6joLWyH3G9p\\\\\\\/OaQDUAYECWGGsL3ScDBa9HMGo\\\\\\\/gB+wIb+POAQKVopMhj8LCLSIn1KTPA4tR5Dh0a5k4R7qE9fnFoBgO\\\\\\\/4\\\\\\\/+QDp+dxA5s5Ux3kSSpMOW999s9PU+2ioDitm83l0gHRm7Nq7FmgseOAvNG09RFLA95X2qXCKjIM+EXU65w09k2NZE2f6uuLNVIc5Qfrx9\\\\\\\/WTHUK6XjkUWD6KeNbYaDXVfhu01ALYXTXyEfqaO0F22ffjvGWTx85WG3wfbsxF0LrQjlMhNVRx1UvKBZNDcOgvVfHNgO4n06mNdoutBonCV70wBPuir2ZwRGZrgFbTsEKe3r6ekfcMUt7NQKuY419vsfY3EOVmxk2j6ep5Lph395AKjedUR4+lRXVV40s1Tx6cvd\\\\\\\/1CJxX4pF3MkjzbGH0dygpaXnbS1nZTCrfWAswHD6rNfEJyQBDzWAJuPwjXc22aXTMOyfZm8mDrqjb5QrD7z5wGWspXnk7F\\\\\\\/et1Ogfdr3kMGli4mNsOc003FnZbAxrpwro4LAXM8sooa8RkoNoqDpMEhVrnkTaiANeLi7NBYAl1\\\\\\\/s01upy3yvveHjBbPGi5KJPUDf2U1nIGClGhClyN16imCO3ji2O32z\\\\\\\/j\\\\\\\/oqruHIaSaCZ3SmpXDLNIBBYCm\\\\\\\/GLV9vZ+jPij2+VOZ\\\\\\\/421\\\\\\\/0kyOxw1MqX6i2ORncwHvFN959i9l8gSoeW6RHx4TqUxTnB65mdRY+rJZB+ss66Tf5fP99FQRFWcBxQqAuQlSQwqVfx9llL23LJp0rzEDgM\\\\\\\/ExsE0Gs6jNb+3k7fWzWKQyfIdtbvxhbLGDXBre\\\\\\\/bLUvIafJuMa8KKokuwiYtyUfEBbU4UFXvSNQl+DGXhIT7nW\\\\\\\/qkXr9TDiTQOHEgwvTduV5SpPlcW8soPV\\\\\\\/Ku98PsavHQkopYKOBKARXStDLpa8ipdBAkmYoDZudjiYmXvR8jLEGZCSfYlX86BPFyitno0DQTeeqe3Noh0N2QQzhMioRUenTePauhZdhc+TNrTXiWi\\\\\\\/k6W1QVTox3+i7iBgvg59CVFGuKq++V1JL2cU4j41inAQOXEYft9HX23J8UXi5ojLhbE1iIon6Pc1ogkpJ88PN6JYnMEo0IJQM7nfdETVb\\\\\\\/zMchVIjHbask9WQ69ouVPYQn8v3JT4J5ee7l3XsKmCOsK6kIUtXDBzEgvp7QA0Mikvt+3Aps0A++zrSJwjXi4iKXYQ3F8B8VuWZsj59LBKRDVutExDQ5mBiDIIAKUqZdG+SAH0ztgwyiRbxLrJZ20VM2DSPlTzgeUOfokwgpbFM4f9KB0BngeeFEPt+zSh1vcKpwX\\\\\\\/VqMLxZFFYrR6wFnKaNB\\\\\\\/vigG1SBOWHNImbCNIpbstykNhGzYkgd3R6KwAhChwOv0qxCuURoc1+iJTyI+6FqI\\\\\\\/FwH4GdigF2zY2ursBWj3H2DG1w37zmHiqH73u8erDjX0s9oKurgVrmDhhq\\\\\\\/xkw3KrqtEPS7W5B8uRXIimjIKNVwmwnJTzGovI9ZkK2MerYcaTpwBioh6LkIxreGiapYIxlSHIu3UHJfGmLd37cIGeuGyl9I5kg5qQl1vbntMRbFrqvAY8++i\\\\\\\/eCyLA1cfo8oAzcFNd0gn7r6Jx6ovYKglc6JMBGLb4QZQHU5PbL9FsoKPDz1+G1zN6Svm5CNj3qDq2ckru8joo9gkLsgCL9cZqrokiReGuNM7TDboZgsuU+cIIZI\\\\\\\/aF8+FqVfgAmc280hcyO\\\\\\\/izcVs2bKhlG+k9lnVh2v4YTs7Wyj662c2AhR5CUkqYWyZeFAJqvyoR8FNT68w+B0M7tEYWkQUp+NtjoLfKAbWspFVWKJz9xHJDRZGa8uodxLn08PnMtoJfjQRDiJuozwy8AUrfjPFw3G0s2hy4dX3+X1iRz0mxlVFjBG5h\\\\\\\/3WqhfP6CKf2oiQomwS8JnG+cBzjhh2EbZtZYzaUFuCcKbeJSkKHjtvmQTiARgsNjnMy1hakMkVfL4d0HaVqvLJGYhHlPwM6GW4\\\\\\\/OxW2MlWG\\\\\\\/B1CCnawQAj1yWQvMXFzF8nHmjtPrX+MwAbvOcap1ly4arQDTrNfd4rBO+9VPhC9FduCihZztQR0IfnVocg0UX0w1SWBGDvK3Qj4BU\\\\\\\/p6Lcs1SKdA+eylpuikLgHcebQREXD+C1AvNm9ayL7qEuYTAOJ+6nwqlM6lIkQLUJX6vd87bkWLT2i\\\\\\\/vQuzhPxqIZ45HiYZAr7EWmRTiaJrlxfnOVCwqRpzH74K77F\\\\\\\/P7yY5Yd7srZBcSzh1AjJi2YIchAXBsSvFU9fpSqkRFT0Gc3VUuxuZ3rk72bNBE1PZ8tscvw5Lqa4F2tha975lX+4YMrWwOh13CkaImeUySf6E7NIEcRHxLUJIUp2tPaHp9bUpEIJGFGCm+cCuU3Vweh4e\\\\\\\/YXewyKV2rRO86JRMv+2\\\\\\\/+D60wc0RwxO5rsA68rjIAEm+Hwk9ReZlmEqKqVtyIrpPgjjohq2+eP6mSR7+6w0PMMpyqVPdh5gSmI91sqld2bgnTNWmkwxJqoHLH1CWIKXL1l\\\\\\\/e0qKeQ2zPBDz2inQe8I6DCfUe+wWECcOimSHEsyqHZU+c+lIxhLE9Xk3zRGRiZbPOB4hgmLWIXdILRtEZ71NDkyvyoRv91HCcC938XVfBtYwqODGf3+AbiuYC8Q4+5DBSIE6ecp\\\\\\\/g4QKrLxOUK15NPX+6WRyDA9UpdKJ28JUPX3+oYiLPP0PxsQWGa+rm5O7c0LWqCBPdqtxS+1gIyVp87BIoDMWp5ChDAbiGcM3+HnuYuh4nDC\\\\\\\/OHOf3TyUAx7Vd+UD8wPypJTGq9Ue2Ksb\\\\\\\/BekyfDX2n\\\\\\\/Hv6azuMOkddUdoFzg1Vhhzy54tDKqMfq8CrZmpsmzv6EOLBVY9S8mFH+bPsGmxHkgTEnQC\\\\\\\/VLbKcDC9Hi8rRQOmoDdv2R81+mXVZf3OfccHNxdRwQOEVZB0VzQ8ywuX+EfVMK4aHbUVw5n6XxvXK7zaxgaiQ3sNmZNyZnHFUt8pzvuKN4h62uNPJlFATO1ns2nG3S4vJOFD3FuD+HVFGlDZel4k\\\\\\\/pCaj7SbTid+rq2QgvKPMaOYKlqLuyWiMB0JAaMccQb+AVRhyqZ1onobgEDEZbnV6sNKXdbm5FwYOQ7E2EylGgvbWIXTbWW8XiQApLrhDytXHbBtUAXsLsMYh9mopApz8w602D+tm5wwBeTghzKl4tKRvtICg0MPs6Ws1IRF55DkXQDfpXjEmagf8TrOtv+si+sek2YB\\\\\\\/Li7vTX2i3ZTxRFzTH99nmIBFabz1b08uuRm3t7EIKb6wdsS+GzESclq2VQe8OaYPX9EBkkDUUGK6wVXIMu+2uETjXs0rqmC2o7t9\\\\\\\/7EhKiKhaGkfLu+IwxfxRKwqlDLAj81CDWSyLA5l1IIdrSxIGDPG7e5Yi+CmSXaJ+gj\\\\\\\/RbOrBQBTIfqxKhTL\\\\\\\/lIqw3sWX5Ak\\\\\\\/XUD8MZktRl7oEqO1v1FpXbnTdU0iYj9b4OOhjDb9qELX+vTVKVd+vpJG1f5KQBpsnw4SomC4SrWzwIG2QxnBv+zINBzVzX5h2IidS\\\\\\\/4LYd6QFXrAoUIxcyj7mfw1\\\\\\\/iZQo1I9K1Hb2U9UbrQaUyGv9hOdwAvqvjRINnbvYN+UZ01\\\\\\\/+v2NzkaZKVsoa689RgJuvVUz+E6iOOhsz2teymdzKoap8o46V58QmISfXiBYX7tk7Jy2zubXBXJnIwvYUENyBRe+RX\\\\\\\/Cz\\\\\\\/XLYhCImYIOVyKvU4LxK9t\\\\\\\/lu9baDTRnQCgDpXvyNMPyV\\\\\\\/kMWNJ7mBHflCF72MY+K7FD+yeEcfvzRRgFVs4JrAPhqMUpfLZRfaav3rrPUlThMcFu0wH\\\\\\\/MiavUB66QvBnbG45lMIVDXTFncLMM7RlWfuaxsI8Uao9VWa2EUy9cO4vJcNIG4raM5o+HD1pljSLWwJPBupUmR220ykmcVzziXd8Wk3raMRvPgsQ\\\\\\\/S2BXCRE75UeORb5YqBBTaPG3eERB9cvcj2xjWtzamnDq3lJUUdGp+\\\\\\\/MqXXdzpMM+PIOa4Z8dDfosJFv0f4rOOV2NF6UY6VQC5r7ya7aiY60Kxz4lgN8US4iUhal5SlYIRK22I1dJPN8oa2HhwJ8OChacU\\\\\\\/mnqwUlqxg5JtnqdSUqyJbHc5BYc4I6Ry7HsRp4K3xMLH7rCqh2BHGcaGoGyEyXrdN1uKy7gbsp8dIT5NH2AXKeJnHlCVJPENIL5eRw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022n7S3WQwrtk+JUdX0Q\\\\\\\/51EA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002262da00e05d545591de889371d5cc7f3416e5a33c02a051739a2112b6d6f7d6fd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022coVP9z4KdQ87N6BZjtrJfVhxvQBgbQX1kTXo87z\\\\\\\/v06epx\\\\\\\/QhlIHu5GS40z6xULa4wjRRaa+mG8fiS2bjTiLzZsw\\\\\\\/J0+j68raf7T80BK7vWGY54TlJdaqk3QaR7STmSVBwkm7C9KdJq5nIWF0EmDsOs0bMmbZNokCC6FZiDfBEVEZk9M\\\\\\\/z5gFztpgj5fzfegw0f58xRl6rrjKrI0wdRESRE2fjS33ESsx0GHPUwAfw0Y83cVVXigQMpHEHvh+zzMjS+k3gn1iM7fCVafHMhAzxWa0vqRFTR+31A0pQ8VI0PkYT2\\\\\\\/WmxzJDjNzOhy8\\\\\\\/MkD8BIIdwKXxfAXuy5WFjjcFsy02Wo67c5J93\\\\\\\/RLLvhi8SStVdiMNnmj8hgclLkpSjlE+UdUvblpSOiHPH3R1ST2CUmVs0QfyWAQcmWnkRqASo28pP5xs0w\\\\\\\/PWp3BnYyqEX+gIwIZZ4j2PVEZ34RETJjaRZv3seYQAsH3ve3DbjvsSPKhPG5tuHfgGPdqyraBnYGIZAQKdXYRDa7ZHyokNTuZsxCNldHFSWi0+gVrcIAbEF6DINrvXcKg2XLUCSMC5bI16YL+v90xajt1LTv9KcALIUxO6hBgN6pxH7CcuCl6rBoyJ8CEypUvNqJXmUidAsycUtn9jnVsFZrPZSoj1QyGaNjGgg5XmzyTKkK0DBzyCCvD06UOM9JfhFr3ZclPWNEOoOvzsc1YdwQiBqcT7+3Vk7nGkxjhFfar2iKDcEoy0rNdRywTrcZFrH7cxj2gMw6YYCfJOIaJmqz03uRfV3VG9wxn2JFbJlAv29UurMVU3pCVHkDUO9uzHRoZ279DCKnYmvzfBCbjhntctMIvWCYv3iGdu2liZsv41k0aXOrPEwBZmmN\\\\\\\/uwBeQGhTVb95IDgNdd8QX5IzJypbixwm6skaPooiHjoZ68OEjQWCLm39z1hyf+8rMCfvF8x2YEOBzry1U35JtVVUMcTDq3STm\\\\\\\/86f3\\\\\\\/Ublw72Jr\\\\\\\/WKFSf+OifV7oOwceaRNVBiyGxxaaAtC0Hube6g\\\\\\\/WGOSYJm0KJrbTTveyR6Hmmxx\\\\\\\/KgVceg+vNGNIJX+99r51vLJVIIDzGqq\\\\\\\/I6HouRu1+6IDFOosZB6O\\\\\\\/\\\\\\\/p1xmkBWqtyqTP9g89+eiwGwCkXjx8T+VPfkmB6nnQNDd9wlXYlZTeXSFIxOc40OnIXm1WqHMP+hjirK9dxfyGetesDBgols8vSWdVjhiHVMzcm1CpoQ496V8R1lbr35xOsR7vrlwDtW5HKgkonU4xo1mNxsIaSYyFfVm0qpxTLAGZoP+WBFdXjsmY8xJdwOVqb\\\\\\\/jTn3ZTXUmFY60yzne5krxqZCEEsA0srpc21STdAskux0VDIyeuIswWz958ebQdHvPmGW5xU3hf20p5BPlVIo9tpXrmH9n0DTu9MaFi6to4sjeSxwXoMjxpzl3D7M54jKOqqnp61H63eHIfuYy\\\\\\\/2T2ama7phxCuCvpZLdv1z0awV+5yvBaA73g2iTbEB52RUQr7fIt8zRvWKUKQH\\\\\\\/0xwA3vkdcQG844\\\\\\\/dX3n\\\\\\\/XwASUMfGBEtFOOeEzCWMJ+iUrSjjUrenZdj4ybMmbt7s7eTtPyxJlrnkp7l6zgZTS4Uw0fFkDE7f7RNTNdfQkRnEo67wYP\\\\\\\/9GZEqvSCuo3gTgEqauz77kmT3oZ3MbRnyfLQidZsjo\\\\\\\/cqnwogNYgefISfYTBKKKCxZDauphI\\\\\\\/IqFUq9\\\\\\\/\\\\\\\/Ba7FtVmLJEGnJq9QAuMXxypmwsOIXU9KZKmLWxV8gyxC3fnBAdELA4Ys\\\\\\\/m4rIhUKIgU4fAVEAzpfZBpY9OFIHCSRevO7aQtpuaF803wgaFyjN\\\\\\\/fBm3UT2JzU5BXjrD9PDGU1jXwV5eiN0e46SkCmclDZk8+TFIPKVjNBIdkwxLdQAZkvhOrtMZWYIjXxIw7j\\\\\\\/I3zZQfl7YMnpjTGBmXCIqRpv6Ki5FlUOwXDcQqoU32azR5N2b1a75vSrBtQeFrh+ft15fvyqCRMz1I3o3yPVLzaEOKnbb4dMq\\\\\\\/fMA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zzrdM6ytPBHv1yGX5OULJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b94ad956e21bae6e8a4ec62f9f6739ff0482da87c3139f2b541d327f39a28f2e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493103 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1623" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QWNipFnOuO4fFKbrYb3pf8bqbQ6pp-tTx_mhxRNohYg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1623" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1zcwZ_gu0aaE7qPiUpMQSIl_0fM6x_prC-i0YiFW04s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1613" title="A Diallel of the Mouse Collaborative Cross Founders Reveals Strong Strain-Specific Maternal Effects on Litter Size" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1633" title="Genomic Variants Among Threatened Acropora Corals" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493103" id="node493103" data-pisa="ggg;9/5/1623" data-pisa-master="ggg;g3.118.200840" data-apath="/ggg/9/5/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce <em>(Picea abies</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Carolina Bernhardsson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7111-9530" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amaryllis Vidalis</span>, <span class="highwire-citation-author" data-delta="2">Xi Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5235-6461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douglas G. Scofield</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bastian Schiffthaler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9919-907X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John Baison</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathaniel R. Street</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6834-6708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Rosario García-Gil</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pär K. Ingvarsson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1632;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200840 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493103" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Carolina Bernhardsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBernhardsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernhardsson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarolina%2BBernhardsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Carolina Bernhardsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:carolina.bernhardsson@gmail.com" class="" data-icon-position="" data-hide-link-title="0">carolina.bernhardsson@gmail.com</a>
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amaryllis Vidalis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Population Genetics, Center of Life and Food Sciences Weihenstephan, Technische Universität München, 85354 Freising, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amaryllis%2BVidalis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vidalis%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmaryllis%2BVidalis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7111-9530" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amaryllis Vidalis</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Douglas G. Scofield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Uppsala Multidisciplinary Center for Advanced Computational Science and</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Ecology and Genetics: Evolutionary Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douglas%2BG.%2BScofield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scofield%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouglas%2BG.%2BScofield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5235-6461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douglas G. Scofield</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bastian Schiffthaler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bastian%2BSchiffthaler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schiffthaler%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABastian%2BSchiffthaler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bastian Schiffthaler</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John Baison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BBaison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baison%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BBaison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9919-907X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John Baison</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nathaniel R. Street </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathaniel%2BR.%2BStreet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Street%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathaniel%2BR.%2BStreet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathaniel R. Street</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">M. Rosario García-Gil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BRosario%2BGarc%C3%ADa-Gil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa-Gil%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BRosario%2BGarc%25C3%25ADa-Gil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6834-6708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Rosario García-Gil</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Pär K. Ingvarsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P%C3%A4r%2BK.%2BIngvarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingvarsson%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP%25C3%25A4r%2BK.%2BIngvarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pär K. Ingvarsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:carolina.bernhardsson@gmail.com" class="" data-icon-position="" data-hide-link-title="0">carolina.bernhardsson@gmail.com</a>
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1623" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493103" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493103/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1623.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493103" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493103/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1623.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493103" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493103/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1623.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493103" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493103/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Norway spruce (<em>Picea abies</em> (L.) Karst.) is a conifer species of substanital economic and ecological importance. In common with most conifers, the <em>P. abies</em> genome is very large (∼20 Gbp) and contains a high fraction of repetitive DNA. The current <em>P. abies</em> genome assembly (v1.0) covers approximately 60% of the total genome size but is highly fragmented, consisting of &gt;10 million scaffolds. The genome annotation contains 66,632 gene models that are at least partially validated (<a href="http://www.congenie.org">www.congenie.org</a>), however, the fragmented nature of the assembly means that there is currently little information available on how these genes are physically distributed over the 12 <em>P. abies</em> chromosomes. By creating an ultra-dense genetic linkage map, we anchored and ordered scaffolds into linkage groups, which complements the fine-scale information available in assembly contigs. Our ultra-dense haploid consensus genetic map consists of 21,056 markers derived from 14,336 scaffolds that contain 17,079 gene models (25.6% of the validated gene models) that we have anchored to the 12 linkage groups. We used data from three independent component maps, as well as comparisons with previously published <em>Picea</em> maps to evaluate the accuracy and marker ordering of the linkage groups. We demonstrate that approximately 3.8% of the anchored scaffolds and 1.6% of the gene models covered by the consensus map have likely assembly errors as they contain genetic markers that map to different regions within or between linkage groups. We further evaluate the utility of the genetic map for the conifer research community by using an independent data set of unrelated individuals to assess genome-wide variation in genetic diversity using the genomic regions anchored to linkage groups. The results show that our map is sufficiently dense to enable detailed evolutionary analyses across the <em>P. abies</em> genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-map" rel="nofollow">genetic map</a></li><li class="kwd"><a href="/keyword/norway-spruce" class="hw-term hw-article-keyword hw-article-keyword-norway-spruce" rel="nofollow">Norway spruce</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/picea-abies" class="hw-term hw-article-keyword hw-article-keyword-picea-abies" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Picea abies</em></a></li><li class="kwd"><a href="/keyword/sequence-capture" class="hw-term hw-article-keyword hw-article-keyword-sequence-capture" rel="nofollow">sequence capture</a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li></ul><p id="p-3">For over a century genetic linkage maps have been used to order genetic markers and link phenotypic traits to genomic regions and chromosomes by calculating recombination events in crosses (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sturtevant 1913a</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sturtevant 1913b</a>). With the advent of Next Generation Sequencing technologies (NGS), large numbers of markers can now be scored at a relatively low cost and within a reasonable time, which has enabled generation of high-density genetic maps consisting of thousands of markers that, in combination with a sufficiently large mapping population, can achieve unprecedented mapping resolution even in non-model systems and in species with large genomes. Genetic maps represent a complementary approach to the local, fine-scale genomic information that is available in scaffolds from a genome assembly, with a genetic map providing information on genome organization over larger scales (up to whole-chromosome level) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fierst 2015</a>). By grouping markers into linkage groups and subsequently ordering them within each linkage group, it is possible to anchor underlying scaffolds containing those markers to putative chromosomes with high precision (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fierst 2015</a>). If several genetic markers, derived from a single genomic scaffold, are placed on the map, information on their relative placement in the genetic map can be used to orient the scaffold and to evaluate scaffolding decisions made in the genome assembly and hence to locate and resolve possible assembly errors (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Drost <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bartholomé <em>et al.</em> 2015</a>). For instance, when two markers originating from a single scaffold map to different linkage groups or to different regions within a linkage group, the contigs comprising the scaffold are candidates for having been wrongly joined during the assembly process. On the other hand, if markers from the same scaffold map close to each other this increases the likelihood that the scaffolding decisions were correct.</p><p id="p-4">Norway Spruce (<em>Picea abies</em>) is one of the most important conifer species in Europe, both from an ecological and economic perspective. The natural distribution range of <em>P. abies</em> extends from the west coast of Norway to the Ural mountains and across the Alps, Carpathians and the Balkans in central Europe. <em>P. abies</em> composes, together with <em>Pinus sylvestris</em>, the majority of the continuous boreal forests of the Northern hemisphere where it is considered a keystone species (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Farjon 1990</a>). <em>P. abies</em> has a genome size of ∼20 Gbp that is characterized by a very high fraction of repetitive sequences. Like most conifers, <em>P. abies</em> has a karyotype consisting of 2<em>n</em> = 24 and with chromosomes that are all uniformly sized (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Sax and Sax 1933</a>). Due to the large and complex genome of conifers, this important group of plants was, until recently, lacking species with available reference genomes. In 2013 the first draft assembly of the <em>P. abies</em> genome was published (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nystedt <em>et al.</em> 2013</a>). Despite extensive whole-genome shotgun sequencing derived from both haploid and diploid tissues, the <em>P. abies</em> genome assembly is still highly fragmented due to the complex nature and size of the genome. The current <em>P. abies</em> genome assembly (v1.0) consists of 10.3 million scaffolds &gt;500 bp and contains 70,736 annotated gene models of which 66,632 are at least partially validated by supporting evidence (ESTs or UniProt proteins) (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">De La Torre <em>et al.</em> 2014</a>). Although the current genome assembly only covers about two thirds of the total genome size (12 Gbp out of the 20 Gbp <em>P. abies</em> genome), it is expected to contain the majority of expressed genes.</p><p id="p-5">In this paper, we used sequence capture to identify segregating SNP markers in megagametophytes from three open-pollinated mother trees. These markers were used to create an ultra-dense haploid genetic map consisting of 21,056 probe-markers derived from 14,336 gene-bearing scaffolds in the <em>P. abies</em> genome assembly. Our aim with creating the genetic map was to 1) anchor, and where possible, order scaffolds to assign as many gene models as possible to linkage groups, and 2) to evaluate the accuracy of the <em>P. abies</em> genome assembly v1.0 on the basis of anchored scaffolds. To evaluate the accuracy of the map itself, we compared scaffold order to previously published genetic maps for <em>P. abies</em> and the closely related <em>Picea glauca</em>. Finally, we evaluated utility of the genetic map for population genomic studies by performing genome-wide analyses of genetic diversity for the genomic regions anchored in the map using a sample of c. 500 unrelated <em>P. abies</em> trees.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>DNA extraction and sequence capture</h3><p id="p-6">In the autumn of 2013, seeds were collected for linkage map construction from five of 30 putative ramets of Z4006, the genotype used to generate the reference genome for <em>Picea abies</em> (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Nystedt <em>et al.</em> 2013</a>). Megagametophytes were dissected from 2,000 seeds by removing the diploid seed coat surrounding the haploid megagametophyte tissue. DNA extraction from megagametophytes was performed using a Qiagen Plant Mini Kit. Each extracted sample was measured for DNA quality using a Qubit ds DNA Broad Range (BR) Assay Kit, and all samples with a total amount of DNA &gt;354 ng were kept. The remaining 1,997 samples were sent to RAPiD Genomics<strong>©</strong> (Gainesville, Florida, USA) in September 2014 for sequence capture using 31,277 capture probes that had been specifically designed to target 19,268 partially-validated gene models from the <em>P. abies</em> genome assembly. Where possible, probes were designed to flank regions of known contig joins in the v1.0 genome assembly (for further detail of the probe design, see <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Vidalis <em>et al.</em> 2018</a>).</p><p id="p-7">The capture data were sequenced by RAPiD Genomics<strong>©</strong> on an Illumina HiSeq 2000 using 1x75 bp sequencing and was delivered in October 2015. The raw reads were mapped against the complete <em>P. abies</em> reference genome v.1.0 using BWA-MEM v.0.7.12 and default settings (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Li and Durbin 2009</a>). Following read mapping, the genome was subset to only contain the probe-bearing scaffolds (a total of 18,461 scaffolds) using Samtools v.1.2 (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Li and Durbin 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2009</a>). Duplicates were marked and local realignment around insertion/deletions (indels) was performed using Picard (<a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>) and GATK (<a href="https://software.broadinstitute.org/gatk/">https://software.broadinstitute.org/gatk/</a>) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">DePristo <em>et al.</em> 2011</a>). Genotyping was performed using GATK Haplotypecaller (version 3.4-46, (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">DePristo <em>et al.</em> 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Van der Auwera <em>et al.</em> 2013</a>) with a diploid ploidy setting and gVCF output format. We used a diploid ploidy setting to increase the likelihood of detecting possible sample contamination from diploid tissue for the haploid megagametophyte samples. CombineGVCFs was then run on batches of ∼200 gVCFs to hierarchically merge them into a single gVCF and a final SNP call was performed using GenotypeGVCFs jointly on the 10 combined gVCF files, using default read mapping filters, a standard minimum confidence threshold for emitting (stand-emit-conf) of 10, and a standard minimum confidence threshold for calling (stand_call_conf) of 20. See <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Vidalis <em>et al.</em> (2018)</a> and the script “per_sample_gv<em>cf</em>.sh” (available at <a href="https://github.com/parkingvarsson/HaploidSpruceMap">https://github.com/parkingvarsson/HaploidSpruceMap</a>) for a full description of the pipeline used for calling variants.</p></div><div id="sec-3" class="subsection"><h3>SNP filtration and megagametophyte relationships</h3><p id="p-8">After SNP filtering, we performed a principle component analysis (PCA) to evaluate the relationship among samples (see Supplementary file for details on the PCA analysis and subsequent filtering steps). Based on the PCA and a hierarchical clustering approach, we divided samples into three clusters representing putative maternal families (Supplementary, Figure S1-3) that were then analyzed independently. In the end we obtained 9,073 probe-markers from 7,101 scaffolds for Cluster 1 (314 samples), 11,648 probe-markers from 8,738 scaffolds for Cluster 2 (270 samples) and 19,006 probe-markers from 13,301 scaffolds for Cluster 3 (842 samples) with a total of 21,056 probe-markers from 14,336 scaffolds across all three clusters (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In total, these scaffolds cover 0.34 Gbp of the <em>P. abies</em> genome and contain 17,079 partially validated gene models.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493001/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493001/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of the three component maps and the total number of probe-markers available in the consensus map. Cluster: Name of each putative maternal family that was identified in the principal component analysis. Samples: Number of megagametophytes in each cluster. Markers: Number of probe-markers in each component map with number of unique segregating bins within brackets (one marker for each bin was used to anchor the bin markers to the genetic map). Scaffolds: Number of scaffolds represented in each component map</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Component and consensus maps</h3><p id="p-9">We created genetic linkage maps using the R-package BatchMap (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Schiffthaler <em>et al.</em> 2017</a>), a parallel implementation of the R-package Onemap (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Margarido, Souza, and Garcia 2007</a>). All probe-markers were recoded using the D1.11 cross-type (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wu <em>et al.</em> 2002</a>), tested for segregation distortion (<em>P</em> &lt; 0.05 after Bonferroni correction) (Supplementary, Figure S4) and grouped into marker bins. The probe-marker with lowest amount of missing data in each bin was then used to represent the bin when constructing the genetic map. Bin markers were grouped into LGs using LOD = 8 and a maximum recombination fraction = 0.35. LGs were then ordered using the RECORD algorithm (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Van Os <em>et al.</em> 2005</a>) with 16 times counting, parallelized over 16 cores, reordered in a 10 marker sliding window with 1 marker incremental steps using the command ‘ripple’ and finally mapped using the Kosambi mapping function and the ‘map batches’ approach (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Schiffthaler <em>et al.</em> 2017</a>) over four parallel cores. Finally, heat maps with pairwise recombination fraction (lower triangular) and phase LOD score (upper triangular) for the ordered markers were created to evaluate the ordering accuracy of independent linkage groups (Supplementary, Figure S5 and S6A-L). We observed 183 probe-marker bins showing signs of segregation distortion. These bins were, however, randomly distributed over the linkage groups and did not appear to affect marker ordering and map distance and were therefore retained in subsequent analyses.</p><p id="p-10">To evaluate correspondence between LGs in maps derived from the three PCA clusters, the number of unique scaffolds shared between cluster LGs were counted (Supplementary, Figure S5). We then created a consensus map for each linkage group from the three independent component maps using the R-package LPmerge (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Endelman and Plomion 2014</a>) with component maps ranked according to marker numbers (Cluster 3, Cluster 2, Cluster 1), a maximum interval setting ranging from one to 10 and map weights proportional to the size of the mapping population (Cluster 3= 0.5, Cluster 2 and Cluster 1 = 0.25). From all possible consensus maps generated by LP merge, for each linkage group we selected the map with the lowest mean root mean square error (RMSE) to serve as the consensus map (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Endelman and Plomion 2014</a>). Order correlations between individual component maps and the consensus maps (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and Supplementary, Figure S7A-L) as well as between the three component maps (Supplementary, Figure S8A-L) were estimated using Kendall’s τ. For visual representation of the consensus map we created a Circos plot using the R-package omicCircos (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hu <em>et al.</em> 2014</a>), available from Bioconductor (<a href="https://bioconductor.org/biocLite.R">https://bioconductor.org/biocLite.R</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492885/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492885/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Marker density and size of each component genetic map created from the three clusters as well as for the consensus map. LG: Linkage group. Cluster 1-3: Component maps for cluster 1-3 with number of probe-markers (marker-bins) assigned, map size (in cM) and maximum gap in map (in cM) for each of the LGs. Consensus: Number of markers and map size of the LGs in the consensus map</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">To evaluate the inflation of map distances due to possible genotyping errors, we performed 100 rounds of random subsampling of 100 probe-marker bins per LG and component map. The following marker ordering and genetic distance calculation were performed with 10 rounds of RECORD and the Kosambi mapping function.</p></div><div id="sec-5" class="subsection"><h3>Accuracy of the reference Picea abies genome assembly</h3><p id="p-12">To evaluate the accuracy of scaffolds from the v1.0 <em>P. abies</em> reference genome containing at least two probe-markers (here after called multi-marker scaffolds) we determine whether probe-markers from the same genomic scaffold mapped to the same region of an LG, on different regions within a single LG or on different LGs. In the consensus map, we considered markers to be positioned in the same region on an LG if all probe-markers from a scaffold mapped within a 5 cM interval of each other. If any marker from the scaffold was positioned further apart, the scaffold was tagged as containing a putative assembly error. The same considerations were made for scaffolds with probe-markers positioned on different LGs.</p></div><div id="sec-6" class="subsection"><h3>Comparative analyses of Picea linkage maps</h3><p id="p-13">To evaluate the consistency of our genetic map with earlier maps from <em>P. abies</em> we compared our haploid consensus map to the <em>P. abies</em> linkage map from <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lind <em>et al.</em> (2014)</a>. The Lind <em>et al.</em> map was created using genetic markers generated using an Illumina 3072 SNP Golden Gate Assay. We performed using tblastn sequence homology searches against the <em>P. abies</em> v1.0 genome assembly for the SNP array sequences of the makers mapped in the Lind <em>et al.</em> map and extracted reciprocal best hits with &gt;95% identity, which were then assigned to the corresponding scaffold in the <em>P. abies</em> genome. We performed similar analyses to compare the synteny between our consensus map and the <em>P. glauca</em> composite map from <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pavy <em>et al.</em> (2017)</a>. Again, we used tblastn sequence homology search comparisons of array sequences from the <em>P. glauca</em> SNP array (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pavy <em>et al.</em> 2013</a>) with scaffolds from the <em>P. abies</em> v1.0 genome assembly to assign corresponding map positions between <em>P. abies</em> and <em>P. glauca</em>. In order to evaluate correspondence between LGs from the different genetic maps, we assessed the number of shared scaffolds between our consensus map, the Lind <em>et al.</em> and Pavy <em>et al.</em> maps. Consistency of scaffold ordering was then evaluated using visual comparisons (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a>) and by calculating correlations of marker orders using Kendall’s τ.</p></div><div id="sec-7" class="subsection"><h3>Population genetic analysis of the consensus genetic map</h3><p id="p-14">In order to independently evaluate the utility of the consensus map for downstream research, we used a subset of the data from <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baison <em>et al.</em> (2018)</a> to estimate patterns of nucleotide diversity across the Norway spruce genome. The data from Baison <em>et al.</em> originally contained 517 individuals sequenced with 40,018 probes designed for diploid spruce samples (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Vidalis <em>et al.</em> 2018</a>). We extracted data for all probes that we had anchored in our genetic map from the VCF file containing the data from Baison <em>et al.</em> We further hard-filtered the resulting VCF file by only considering bi-allelic SNPs within the extended probe regions (120 bp probes ±100 bp) with a QD &gt;5, MQ &gt;50 and a overall DP between 3000 and 16000. Samples containing &gt;25% missing data were removed from further analysis. We used the data to calculate nucleotide diversity (π), the number of segregating sites and Tajima’s D (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Tajima 1989</a>). We used the R package vcfR (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Knaus and Grünwald 2017</a>) to read the VCF-file into R and then used in-house developed scripts to perform all calculations (available at <a href="https://github.com/parkingvarsson/HaploidSpruceMap">https://github.com/parkingvarsson/HaploidSpruceMap</a>). We assigned probes to LGs and map positions by assigning them the coordinates of the physically closest (in bp) probe. We also calculated pairwise linkage disequilibrium (LD) between markers within probes using vcftools (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Danecek <em>et al.</em> 2011</a>) and imported the results into R where they were used to calculate <em>Z<sub>nS</sub></em> scores (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kelly 1997</a>) per probe using an in-house developed script (available at <a href="https://github.com/parkingvarsson/HaploidSpruceMap">https://github.com/parkingvarsson/HaploidSpruceMap</a>). Finally, we ran sliding window analyses along the linkage groups for the different summary statistics using 10 cM windows that were moved in 1 cM incremental steps.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-15">BatchMap input files for the three clusters, component maps and consensus map files are available from zenodo.org at <a href="https://doi.org/10.5281/zenodo.1209841">https://doi.org/10.5281/zenodo.1209841</a>. All scripts needed to recreate the analyses described in the paper are publically available at <a href="https://github.com/parkingvarsson/HaploidSpruceMap">https://github.com/parkingvarsson/HaploidSpruceMap</a>. Raw sequence data for all samples included in this study are available through the European Nucleotide Archive under accession number PRJEB25757. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7258448">https://doi.org/10.25387/g3.7258448</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><p id="p-16">We generated a <em>P. abies</em> consensus linkage map from three haploid component maps containing a total of 21,056 unique probe-markers from 14,336 scaffolds in the <em>P. abies</em> genome assembly v1.0. The consensus map anchored 0.34 Gbp of the <em>P. abies</em> 1.0 assembly, corresponding to 1.7% of the complete <em>P. abies</em> genome or 2.8% of the genome assembly. However, these scaffolds anchor 25.6% of all validated gene models with these anchored scaffolds containing 31.7%, 20.6% and 25.8% of the High-, Medium- and Low confidence gene models from <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Nystedt <em>et al.</em> (2013)</a>, respectively. The consensus map had a total length of 3,556 centiMorgan (cM), distributed over 12 linkage groups (LGs), corresponding to the haploid chromosome number (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Sax and Sax 1933</a>), and with an average distance of 0.17 cM between probe-markers (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Circos plot of the consensus map. A) Marker distribution over the 12 linkage groups (LG I-LG XII). Each black vertical line represents a marker (21,056 in total) in the map and is displayed according to the marker positions in cM. Track B-C visualizes multi marker scaffolds, where each line is a pairwise position comparison of probe-markers from the same scaffold. B) Position comparisons of probe-markers from the same scaffold that are located on the same LG. Light gray lines indicate probe-markers that are located &lt; 5cM from each other, dark gray lines indicate probe-markers located 5-10 cM apart and red lines indicate probe-markers &gt;10 cM apart. C) Position comparisons of probe-markers from the same scaffold that are mapping to different LGs. Orange lines indicated probe-markers from the same scaffold split over 2 LGs, while dark blue lines indicated probe-markers split over 3 LGs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15614445" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Circos plot of the consensus map. A) Marker distribution over the 12 linkage groups (LG I-LG XII). Each black vertical line represents a marker (21,056 in total) in the map and is displayed according to the marker positions in cM. Track B-C visualizes multi marker scaffolds, where each line is a pairwise position comparison of probe-markers from the same scaffold. B) Position comparisons of probe-markers from the same scaffold that are located on the same LG. Light gray lines indicate probe-markers that are located &lt; 5cM from each other, dark gray lines indicate probe-markers located 5-10 cM apart and red lines indicate probe-markers &gt;10 cM apart. C) Position comparisons of probe-markers from the same scaffold that are mapping to different LGs. Orange lines indicated probe-markers from the same scaffold split over 2 LGs, while dark blue lines indicated probe-markers split over 3 LGs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1623/F1.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1623/F1.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493118" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Circos plot of the consensus map. A) Marker distribution over the 12 linkage groups (LG I-LG XII). Each black vertical line represents a marker (21,056 in total) in the map and is displayed according to the marker positions in cM. Track B-C visualizes multi marker scaffolds, where each line is a pairwise position comparison of probe-markers from the same scaffold. B) Position comparisons of probe-markers from the same scaffold that are located on the same LG. Light gray lines indicate probe-markers that are located &lt; 5cM from each other, dark gray lines indicate probe-markers located 5-10 cM apart and red lines indicate probe-markers &gt;10 cM apart. C) Position comparisons of probe-markers from the same scaffold that are mapping to different LGs. Orange lines indicated probe-markers from the same scaffold split over 2 LGs, while dark blue lines indicated probe-markers split over 3 LGs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Correlations of probe-marker order between the three component maps and the consensus map ranged from 0.96 to 0.998, while the correlations between marker orders between individual component maps ranged from 0.943 to 0.993 (Supplementary, Figure S7 and S8). 183 probe-marker bins showed evidence of segregation distortion in Cluster 3, but these were randomly distributed over all linkage groups and we did not observe regions showing clusters of markers with segregation distortion or with conflicting marker orders between clusters (Supplementary, Figure S8). LG XI, which displayed the largest discrepancy in marker order between component maps, has a region at the distal end of the LG, covering 252 probe-markers, where the resolution was too low to identify the correct marker order and where the entire region was positioned at 36.115 cM (Supplementary, Figure S7K and S8K), explaining the lower correlations in marker order between individual maps for this LG.</p><p id="p-19">We used a random subsampling approach to evaluate potential inflation of map distances due to possible genotyping errors. From these analyses, total map size for Cluster 1 ranged between 2,166.8 and 2,450.0 cM with an average size of 2,294.2 cM and a standard deviation (SD) of 3.6- 5.8 cM per LG. Cluster 2 ranged between 2,304.2 and 2,663.6 cM with an average of 2,478.3 cM and a SD of 4.4 – 9.1 cM per LG, while Cluster 3 ranged between 1,855.4 and 2,093.2 cM with an average of 1,971.0 cM and a SD of 2.7 – 7.3 cM per LG. The estimated inflation was therefore predicted to be 0.15 – 0.31 cM per probe-marker bin across the three component maps (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). This inflation per probe-marker bin roughly corresponded to the map resolution of the clusters (Cluster 1- 0.32 cM: Cluster 2 - 0.37 cM: Cluster 3 – 0.12 cM) and yielded an error estimate of ∼1 genotype error per marker-bin or 11-17 genotype errors per sample.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492926/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492926/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimated genetic length of each Linkage Group (LG) in the three component maps. LG: linkage group in the consensus map; Observed genetic length (cM): The genetic length of the LG calculated from all probe-marker bins (same as in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>); Mean estimated genetic length (cM): the average length of the LG when using 100 random probe-marker bins in 100 map calculations; SD (cM): Standard deviation of the estimated length; Inflation/Marker bin: The difference between observed genetic length and the estimated length divided by the number of probe-marker bins in the linkage group</span><div class="sb-div caption-clear"></div></div></div><div id="sec-10" class="subsection"><h3>Evaluation of the Picea abies genome assembly v1.0</h3><p id="p-20">4,859 scaffolds (33.9%) contained more than one unique probe-marker combined over all three component maps. 185 of these multi-marker scaffolds contained markers that were located in more than one LG (<em>inter-split scaffolds</em>) or over different parts of the same LG (<em>intra-split scaffolds</em>). 26 scaffolds (0.18% of mapped scaffolds and 0.54% of multi-marker scaffolds) contained markers that were positioned on the same LG but at distances exceeding 5 cM in the consensus map. When exploring the individual component maps, it was apparent that for two of these scaffolds (MA_281725 on LG X and MA_10431182 on LG I) the probe-markers in the consensus map all came from different component maps. The consensus map thus contains a gap that we cannot verify using any of the individual component maps (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and Supplementary, Figure S9). Three other scaffolds (MA_9458 on LG IX, MA_10431315 on LG II and MA_10432328 on LG III) all have multiple probe-markers present in at least one component map and were these component maps do not support the split we observe in the consensus map (Supplementary, Figure S9). It thus appears that these splits are artifacts arising from the construction of the consensus map.</p><p id="p-21">There were 164 scaffolds (1.14% of mapped scaffolds and 3.38% of multi-marker scaffolds) containing markers that were mapped to two or three different LGs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Supplementary, Figure S10). All LGs contained inter-split scaffolds, while 10 LGs (LGII and LGXI are the exceptions) contained intra-split scaffolds supported by the component maps (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B-C</a> and Supplementary, Figure S9).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fraction of scaffolds that are being represented by 1-11 unique markers in the consensus map. Insert: Fraction of scaffolds that have multiple probe-markers (2-11) that are distributed over 1-3 linkage groups (inter-split scaffolds). Red dot indicate the fraction of scaffolds with multiple probe-markers which are positioned &gt; 5cM apart on the same linkage group (intra-split scaffolds)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15614445" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fraction of scaffolds that are being represented by 1-11 unique markers in the consensus map. Insert: Fraction of scaffolds that have multiple probe-markers (2-11) that are distributed over 1-3 linkage groups (inter-split scaffolds). Red dot indicate the fraction of scaffolds with multiple probe-markers which are positioned &gt; 5cM apart on the same linkage group (intra-split scaffolds).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1623/F2.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1623/F2.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493091" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Fraction of scaffolds that are being represented by 1-11 unique markers in the consensus map. Insert: Fraction of scaffolds that have multiple probe-markers (2-11) that are distributed over 1-3 linkage groups (inter-split scaffolds). Red dot indicate the fraction of scaffolds with multiple probe-markers which are positioned &gt; 5cM apart on the same linkage group (intra-split scaffolds).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The scaffolds covered by the consensus map ranged in length from 0.22 to 208.1 Kbp with a median of 17.1 Kbp, while multi-marker scaffolds ranged from 0.39 to 161.5 Kbp (median of 21 Kbp). The 185 scaffolds that are split within or across LGs ranged in size from 2.5 to 121.6 Kbp, with a median length of 36.9 Kbp. Split scaffolds were significantly longer than multi-marker scaffolds in general (t = -7.7, df = 193.4, p-value = 7.0e-13; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), suggesting that longer scaffolds are more likely to contain assembly errors compared to shorter scaffolds. Split scaffolds mostly contained high- and medium confidence gene models (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). A visual inspection of the split scaffolds revealed that for 75 and 10 of the inter-split and intra-split scaffolds, respectively, the predicted position of the split(s) occurred between different gene models on the same scaffold. Of greater concern, for 88 of the inter-split scaffolds and 11 of the intra-split scaffolds the predicted position of the split was located within a single gene model (Supplementary, Figure S9 and S10). In addition, 21 inter-split scaffolds showed an even more complicated picture, where an interior region of the gene model (most often containing an intron &gt; 5kb) mapped to another chromosome whereas the 5′ and 3′ regions of the gene model mapped to the same chromosome location (Supplementary, Figure S10). However, 84% (184 out of a total of 219 splits) appear to occur between contig joins (where a sequence of N’s appear in the assembly) of the scaffold. Of the 17,079 gene models that were anchored to the consensus genetic map, 330 were positioned on inter- or intra-split scaffolds (5.4% of gene models that were positioned on multi-marker scaffolds) and 100 showed a split within gene models (1.6% of gene models from multi-marker scaffolds) (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box plot of scaffold lengths for all multi-marker scaffolds (dark gray box) and for scaffolds showing a split within or across LGs (light gray box). The split scaffolds are significantly longer than the multi-marker scaffolds in general (t = -7.70, df = 193.39, p-value = 7.00e-13)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15614445" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box plot of scaffold lengths for all multi-marker scaffolds (dark gray box) and for scaffolds showing a split within or across LGs (light gray box). The split scaffolds are significantly longer than the multi-marker scaffolds in general (t = -7.70, df = 193.39, p-value = 7.00e-13).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1623/F3.medium.gif" width="440" height="375"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1623/F3.medium.gif" width="440" height="375"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493117" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Box plot of scaffold lengths for all multi-marker scaffolds (dark gray box) and for scaffolds showing a split within or across LGs (light gray box). The split scaffolds are significantly longer than the multi-marker scaffolds in general (t = -7.70, df = 193.39, p-value = 7.00e-13).</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493127/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493127/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Overview of annotated gene models anchored to the genetic map. Gene models: Annotated protein coding gene models with High-, Medium- and Low confidence level (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Nystedt <em>et al.</em> 2013</a>). Mapped scaffolds: Number of gene models positioned on scaffolds that are anchored to the genetic map (Percentage of total number of gene models for each confidence level). Multi-marker scaffolds: Number of gene models positioned on scaffolds with multiple markers in the genetic map (Percentage of gene models on mapped scaffolds). Inter-split scaffolds: Number of gene models positioned on the 164 scaffolds that are split between LGs in the genetic map (Percentage of gene models on mapped scaffolds / Percentage of gene models on multi-marker scaffolds). Intra-split scaffolds: Number of gene models positioned on the 22 scaffolds that are split between different regions of the same LG (Percentage of gene models on mapped scaffolds / Percentage of gene models on multi-marker scaffolds). Split within gene models: Number of gene models that have an internal split (Percentage of gene models on mapped scaffolds / Percentage of gene models on multi-marker scaffolds)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Comparative analyses to other Picea linkage maps</h3><p id="p-25">In order to assess the accuracy and repeatability of the <em>P. abies</em> genetic maps we compared our consensus map to the <em>P. abies</em> map presented in <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Lind <em>et al.</em> (2014)</a>. 353 comparisons between 298 markers from Lind <em>et al.</em> and 288 scaffolds contained in our consensus map were identified at a &gt;95% identity threshold. Of these markers, 96.7% grouped to the same LG in the two maps while the remaining 3.3% (11 out of 353) were distributed across several LGs (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Correlations of marker order between the two <em>P. abies</em> maps ranged from 0.53 to 0.99 across the 12 LGs. The comparison between the haploid consensus map for LG I and LG 7 from Lind et.al, which had the lowest correlation of marker order, showed inconsistencies of marker order where a contiguous subset of markers were arranged in the opposite order from the rest of the markers for that LG. The remaining LGs showed high synteny, with consistent marker ordering between the two genetic maps.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker order comparison between Linkage Groups (LGs) from the haploid consensus map presented here and the Picea abies map from Lind et al. (2014). Consensus LG I - LG XII are located on the x-axis from left to right. Lind et al. LG 1 - LG 12 are located on the y-axis from top to bottom. Each dot represents a marker comparison from the same scaffold, where black coloration represents the LG where the majority of marker comparisons are mapped. Gray coloration represents markers mapping to a different LG compared to the majority of markers. Turquoise coloration represents markers located on split scaffolds, which are indicative of assembly errors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15614445" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker order comparison between Linkage Groups (LGs) from the haploid consensus map presented here and the &lt;em&gt;Picea abies&lt;/em&gt; map from Lind &lt;em&gt;et al.&lt;/em&gt; (2014). Consensus LG I - LG XII are located on the x-axis from left to right. Lind &lt;em&gt;et al.&lt;/em&gt; LG 1 - LG 12 are located on the y-axis from top to bottom. Each dot represents a marker comparison from the same scaffold, where black coloration represents the LG where the majority of marker comparisons are mapped. Gray coloration represents markers mapping to a different LG compared to the majority of markers. Turquoise coloration represents markers located on split scaffolds, which are indicative of assembly errors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1623/F4.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1623/F4.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492896" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Marker order comparison between Linkage Groups (LGs) from the haploid consensus map presented here and the <em>Picea abies</em> map from <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Lind <em>et al.</em> (2014)</a>. Consensus LG I - LG XII are located on the x-axis from left to right. Lind <em>et al.</em> LG 1 - LG 12 are located on the y-axis from top to bottom. Each dot represents a marker comparison from the same scaffold, where black coloration represents the LG where the majority of marker comparisons are mapped. Gray coloration represents markers mapping to a different LG compared to the majority of markers. Turquoise coloration represents markers located on split scaffolds, which are indicative of assembly errors.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Synteny between <em>P. abies</em> and <em>P. glauca</em> species was assessed by comparing LG location and marker order between our <em>P. abies</em> consensus map and the composite map of <em>P. glauca</em> from <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Pavy <em>et al.</em> (2017)</a>. 14,112 comparisons of 4,053 gene models in the composite map in <em>P. glauca</em> (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Pavy <em>et al.</em> 2017</a>) and 4,310 scaffolds in the <em>P. abies</em> consensus map were identified at a &gt;95% identity threshold. 92.7% (13,084 out of 14,112 comparisons) of these were located on homologous LGs while the remaining 7.3% (1,028 comparisons from 388 <em>P.abies</em> scaffolds) were distributed across the 12 LGs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). 8.2% of all comparisons from multi-probe scaffolds were between non-homologous LGs while 44.3% of all comparisons from split scaffolds were between non-homologous LGs. 31.9% of all non-homologous LG comparisons involved split scaffolds. The correlations of marker order between the two maps were comparable to the correlations we observed between individual component maps in <em>P. abies</em> (0.96-0.99), showing that synteny is highly conserved between <em>P. abies</em> and <em>P glauca</em>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker order comparison of Linkage Groups (LGs) between the Picea abies haploid consensus map presented here and the Picea glauca map from Pavy et al. (2017). Consensus LG I - LG XII are located on the x-axis from left to right. Pavy et al. LG 1 - LG 12 are located on the y-axis from top to bottom. Each dot represents a marker comparison from the same scaffold, where black coloration represents markers mapping to the same LG in the two species, gray coloration represents markers mapping to different LGs. Turquoise coloration represents markers located on split scaffolds, indicating an assembly error." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15614445" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker order comparison of Linkage Groups (LGs) between the &lt;em&gt;Picea abies&lt;/em&gt; haploid consensus map presented here and the &lt;em&gt;Picea glauca&lt;/em&gt; map from Pavy &lt;em&gt;et al.&lt;/em&gt; (2017). Consensus LG I - LG XII are located on the x-axis from left to right. Pavy &lt;em&gt;et al.&lt;/em&gt; LG 1 - LG 12 are located on the y-axis from top to bottom. Each dot represents a marker comparison from the same scaffold, where black coloration represents markers mapping to the same LG in the two species, gray coloration represents markers mapping to different LGs. Turquoise coloration represents markers located on split scaffolds, indicating an assembly error.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1623/F5.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/5/1623/F5.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493175" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Marker order comparison of Linkage Groups (LGs) between the <em>Picea abies</em> haploid consensus map presented here and the <em>Picea glauca</em> map from <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Pavy <em>et al.</em> (2017)</a>. Consensus LG I - LG XII are located on the x-axis from left to right. Pavy <em>et al.</em> LG 1 - LG 12 are located on the y-axis from top to bottom. Each dot represents a marker comparison from the same scaffold, where black coloration represents markers mapping to the same LG in the two species, gray coloration represents markers mapping to different LGs. Turquoise coloration represents markers located on split scaffolds, indicating an assembly error.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Population genetic analyses based on the consensus map</h3><p id="p-29">22,413 probes, covering 12,908 scaffolds, were used in the population genetic analyses based on the consensus genetic map. On a per-probe basis, we observed substantial variation in all neutrality statistics, with the number of segregating sites ranging from 0 - 77 (mean 15.9), nucleotide diversity (π) from 0 - 0.4 (0.005), Zns from 0 - 1 (mean 0.04) and Tajima’s D from -2.4 – 3.5 (mean -0.85). To study large-scale trends and possible chromosomal differences we performed sliding window analyses across the LGs for the different summaries (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). One interesting large-scale feature we observed was that SNP densities were often highest at the distal or central regions of LGs, indicating the possible location of centromeres and telomeres, for which recombination rates are expected to be reduced (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gaut <em>et al.</em> 2007</a>) and where we hence would expect higher densities of probes per cM (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6a</a>). The large-scale analyses also revealed several instances where entire chromosomal arms might be experiencing different evolutionary patterns (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6b-c</a>). Finally, we identified regions that appear to be evolving under the influence of natural selection. For instance, several regions showed higher than average levels of nucleotide diversity and positive Tajima’s D (<em>e.g.</em>, on LG IV, V and XII), suggesting that they might harbor genes under balancing selection. Similarly, regions with low nucleotide diversity, an excess of rare alleles and strong linkage disequilibrium (<em>i.e.</em>, negative Tajima’s D and high <em>Z<sub>ns</sub></em> scores, <em>e.g.</em>, on LG III) could indicate regions harboring possible selective sweeps (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6c-d</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sliding window analysis of neutrality statistics. Analyses were performed using 10 cM windows with 1 cM incremental steps along the consensus map linkage groups and visualized using coloring alternates between adjacent LGs. A) Number of segregating sites. Dashed horizontal line indicates the overall average of 1017. B) Pairwise nucleotide diversity (&#x3C0;). Dashed horizontal line indicates the overall average of 0.005. C) Tajima&#x2019;s D. Dashed horizontal line indicates the overall average of -0.852. D) Linkage disequilibrium Zn scores. Dashed horizontal line indicates the overall average of 0.040." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-15614445" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sliding window analysis of neutrality statistics. Analyses were performed using 10 cM windows with 1 cM incremental steps along the consensus map linkage groups and visualized using coloring alternates between adjacent LGs. A) Number of segregating sites. Dashed horizontal line indicates the overall average of 1017. B) Pairwise nucleotide diversity (&#x3C0;). Dashed horizontal line indicates the overall average of 0.005. C) Tajima&#x2019;s D. Dashed horizontal line indicates the overall average of -0.852. D) Linkage disequilibrium Zn scores. Dashed horizontal line indicates the overall average of 0.040.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1623/F6.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/5/1623/F6.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1623/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492976" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-30" class="first-child">Sliding window analysis of neutrality statistics. Analyses were performed using 10 cM windows with 1 cM incremental steps along the consensus map linkage groups and visualized using coloring alternates between adjacent LGs. A) Number of segregating sites. Dashed horizontal line indicates the overall average of 1017. B) Pairwise nucleotide diversity (π). Dashed horizontal line indicates the overall average of 0.005. C) Tajima’s D. Dashed horizontal line indicates the overall average of -0.852. D) Linkage disequilibrium Zn scores. Dashed horizontal line indicates the overall average of 0.040.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-31">This is, to our knowledge, the densest genetic linkage map ever created for a conifer species and possible for any tree species. We successfully used this genetic map to anchor 1.7% of the 20 Gbp <em>P. abies</em> genome, corresponding to 2.8% of the v1.0 genome assembly (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Nystedt <em>et al.</em> 2013</a>), to 12 LGs, constituting the haploid chromosome number (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Sax and Sax 1933</a>). The <em>P. abies</em> genome has a very large proportion of gene-poor heterochromatin, so while the fraction of the genome that we successfully anchored to the assembly is relatively small, those anchored scaffolds cover 24% of all gene-containing assembly scaffolds and 25% of all partially validated gene models from <a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Nystedt <em>et al.</em> (2013)</a>.</p><p id="p-32">The individual LGs from the three component maps (36 LGs from three independent maps) consisted of 648-1,967 probe-markers and 305-1,185 probe-marker bins and, as such, it was not feasible to analyze the maps using an exhaustive ordering algorithm (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mollinari <em>et al.</em> 2009</a>). Instead, we used RECORD (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Van Os <em>et al.</em> 2005</a>) with 16 times counting, parallelized over 16 cores and with reordering of markers within 10 marker windows, for each LG to determine the most likely marker order. An heuristic approach, such as RECORD, will undoubtedly introduce some errors in marker ordering (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Mollinari <em>et al.</em> 2009</a>), but analyses from simulated data suggested that the average distance between estimated and true marker position is small (&lt;5 markers) for data sets of similar size to ours (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Schiffthaler <em>et al.</em> 2017</a>). However, reliable marker ordering requires robust data and the more genotyping errors and missing data that are present, the harder it will be to determine the true order. This in turn will impact the final size of the map, where both errors in marker order and genotyping results in inflation in the size of the map (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cartwright <em>et al.</em> 2007</a>).</p><p id="p-33">By collecting our 2,000 megagametophytes from what we initially thought were five different ramets of Z4006, we accidentally sampled material from at least three unrelated families. This error stemmed from a mix-up of genotypes due to wrong assignment of ramet ID to the different ramets in the seed orchard. Unfortunately, we were not able to assess which megagametophytes were collected from the different putative ramets since the seed bags were pooled prior to DNA extraction and the sampling errors were not detected until after all sequencing was completed. We used a PCA and hierarchical clustering approach to assign samples into three independent clusters, representing three putative maternal families. We also used PCAs of the putative individual families to verify that these clusters were consistent with offspring derived from a single mother tree (Supplementary, Figure S3). However, we nevertheless cannot completely rule out that a small fraction of samples have been incorrectly assigned to the three families and this would lead to inflated map sizes by introducing an excess of recombination events. Another potential confounding issue is tissue contamination. <em>P. abies</em> megagametophytes are very small and are surrounded by a diploid seed coat that needs to be removed prior to DNA extraction. If traces of the diploid seed coat remain in the material used for DNA extractions, the haploid samples will be contaminated with diploid material. To identify and eliminate this possibility, we called sequence variants using a diploid model and any heterozygous SNP calls were subsequently treated as missing data. Samples with a high proportion of heterozygous (&gt;10%) or missing calls (&gt;20%) were excluded from further analyses to reduce the possibilities of genotyping error due to tissue contamination influencing downstream analyses. We estimated map lengths from 100 rounds of subsampling of 100 random probe-marker bins per component LG and used this to demonstrate that individual maps showed size inflations of 0.15-0.31 cM per probe-marker bin. This inflation is on the same order as the map resolutions for the different clusters and, therefore, indicated an average of ∼1 genotyping error per probe-marker bin or 11-17 genotyping errors per sample.</p><p id="p-34">Both sample- and tissue contaminations can influence the accuracy of the genetic map, both with regards to marker order and map size. The smaller family sizes resulting from dividing our original 2,000 samples into three independent families yielded lower resolution of the three component maps. Fortuitously enough, however, this also enabled us to incorporate more markers into the consensus map since different markers were segregating in the different mother trees from which the three families were derived. Furthermore, it also allowed us to evaluate marker ordering across three independently derived maps. Although our consensus map was 70–90% (60–120% for the individual component maps) larger than previously estimated <em>Picea</em> maps (3,556 cM <em>vs.</em> 1,889-2,083 cM), it also contained 2-31 times more markers than earlier maps (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pavy <em>et al.</em> 2012</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Lind <em>et al.</em> 2014</a>; <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Pavy <em>et al.</em> 2017</a>). When comparing marker order between our three independent component maps (Cluster 1-3), we found overall high correlations of marker order (0.94-0.99, Supplementary, Figure S8), which is similar to what has previously been observed between estimated and true positions in maps derived from simulated data without genotyping errors but with 20% missing data (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Mollinari <em>et al.</em> 2009</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Schiffthaler <em>et al.</em> 2017</a>). Also, earlier <em>Picea</em> maps were all based on diploid F<sub>1</sub> crosses with even the densest composite map containing only 2,300-2,800 markers per framework map (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> - <a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Pavy <em>et al.</em> 2017</a>), compared to our haploid component maps that contained between 3,924 and 11,479 probe-marker bins each (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-35">The comparisons between our haploid consensus map and earlier maps in <em>Picea</em> showed an overall high correlation of marker order, which is in line with previous studies suggesting highly conserved synteny within <em>Picea</em> and in conifers in general (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">de Miguel <em>et al.</em> 2015</a>; <a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Pavy <em>et al.</em> 2017</a>). LG I from our haploid consensus map and LG 7 from <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Lind <em>et al.</em> (2014)</a> showed an inverted order for approximately half of the markers compared (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Whether this inversion is due to ordering errors in one of the maps or represents true biological differences between the parents used for the respective maps is, however, not currently known and further investigations are needed to resolve this issue.</p><p id="p-36">A small percentage of the marker comparisons in both the intra- and inter-specific maps did not co-align to homologous LGs. Some of these errors likely arose from the repetitive nature of the <em>P. abies</em> genome (and conifer genomes in general), where regions with high sequence similarity can often be found interspersed throughout the genome. If the true homologous region between different maps is missing or has been collapsed in the genome assembly due to high sequence similarity, pairwise sequence comparisons may end up assigning homology to regions that are located on different chromosomes. However, it might also be that these errors represent scaffold assembly errors for scaffolds containing only a single probe-marker or where one region of the scaffold is not captured by the probes, therefore negating evaluation. Approximately 72% of all non-homologs LG comparisons between <em>P. abies</em> and <em>P. glauca</em> were from multi-markers scaffolds (of which 45% were from probe-markers on split scaffolds in the consensus map (turquoise points in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). The remaining 28% were comparisons with scaffolds that were only represented by a single probe in the consensus map.</p><p id="p-37">Four percent of the scaffolds containing multiple makers showed a pattern where different markers mapped to different regions, either within or between LGs in the consensus map. This indicates possible errors in scaffolding during the assembly of the v1.0 <em>P. abies</em> genome (<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Nystedt <em>et al.</em> 2013</a>). If this estimate represents the overall picture for the entire assembly, as many as 400,000 of the ∼10 million total scaffolds, and 2,400 of the ∼60,000 gene-containing scaffolds, may suffer from assembly errors. Most worryingly, 2% of the multi-marker scaffolds (100/4,859) contained splits that occurred within a single gene model. It is likely that many of these problematic scaffolds stem from incorrect scaffolding of exons from paralogous genes with a high sequence similarity. Since the <em>P. abies</em> genome contains a high proportion of repetitive content, that also includes a large number of pseudo-genes, this is perhaps not surprising. Additional work is needed to disentangle these issues and to resolve any assembly errors. False scaffold joins in a genome assembly are not a unique feature for <em>P. abies</em>, rather it appears to be a frequent problem in the assembly process. For instance, dense genetic maps in both <em>Eucalyptus</em> and <em>Crassostrea</em> have identified and resolved false scaffold joins, thereby improving the genome assemblies in these species (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bartholomé <em>et al.</em> 2015</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hedgecock <em>et al.</em> 2015</a>). Our goal for the <em>P. abies</em> genetic map was not only to identify incorrect scaffolding decisions in the v1.0 genome assembly, but to also help improve future iterations of the genome. Long-read sequencing technologies (<em>e.g.</em>, Pacific Bioscience or Oxford Nanopore) could be used to resolve these problematic scaffolds and help disentangle the reasons for their ambiguous localization in the genetic map. A future reference genome for <em>P. a</em>bies, based on long read technologies will also be able to utilize this genetic map in a much more efficient way since the resulting assembled scaffolds will be substantially longer and would hence enable anchoring a greater fraction of the genome to LGs, ultimately to the point that chromosome-scale assemblies may be achieved.</p><p id="p-38">Our population genetic analyses based on the scaffolds anchored to the consensus map demonstrates the utility of having a dense, accurate genetic map and suggests that the map will facilitate further analyses of genome-wide patterns of variation and selection in <em>P. abies</em> in addition to facilitating comparative analyses among spruce species. Assigning even a small fraction of the genome to LGs enabled us to analyze patterns of genetic diversity in approximately a quarter of all predicted genes. This allowed for analyses of broad-scale patterns of variation across the genome and, as the genome assembly is further improved and an even greater proportion of the assembly if physically anchored to the genetic map, will allow for even more fine-scaled analyses of how different evolutionary forces have interacted in shaping patterns of genetic diversity across the <em>P. abies</em> genome.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">This study was supported by Knut and Alice Wallenberg’s foundation through funding to the Norway spruce genome project. AV was partially supported by a grant from the Stiftelsen Gunnar och Birgitta Nordins fond through the Kungl. Skogs- och Lantbruksakademien (KSLA). NRS was supported by the Trees and Crops for the Future (TC4F) project. All computations were performed on resources provided by SciLifeLab and SNIC at the Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under project b2010042.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-40">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7258448">https://doi.org/10.25387/g3.7258448</a>.</p></li><li class="fn" id="fn-2"><p id="p-41"><em>Communicating editor: J. Udall</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-10-26"><span class="received-label">Received </span>October 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-15"><span class="accepted-label">Accepted </span>March 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Bernhardsson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.1" data-doi="10.1101/292847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baison</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidalis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Association mapping identified novel candidate loci affecting wood formation in Norway spruce.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/292847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F292847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyOTI4NDd2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.2" data-doi="10.1111/nph.13150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartholomé</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandrou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mabiala</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nabihoudine</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-Resolution Genetic Maps of Eucalyptus Improve Eucalyptus Grandis Genome Assembly.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">1296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D206%26rft.spage%253D1283%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13150%26rft_id%253Dinfo%253Apmid%252F25385325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25385325&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.3" data-doi="10.1534/genetics.106.063982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cartwright</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic Mapping in the Presence of Genotyping Errors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2521</span>–<span class="cit-lpage">2527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.063982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.063982%26rft_id%253Dinfo%253Apmid%252F17277374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzI1MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.4" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Variant Call Format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.5" data-doi="10.1104/pp.114.248708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De La Torre</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Insights into Conifer Giga-Genomes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1724</span>–<span class="cit-lpage">1732</span>. <a href="http://www.plantphysiol.org/content/166/4/1724.short">http://www.plantphysiol.org/content/166/4/1724.short</a>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.114.248708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.114.248708%26rft_id%253Dinfo%253Apmid%252F25349325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTY2LzQvMTcyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.6" data-doi="10.1093/gbe/evv185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Miguel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartholomé</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriguchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evidence of Intense Chromosomal Shuffling during Conifer Evolution.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2799</span>–<span class="cit-lpage">2809</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv185</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv185%26rft_id%253Dinfo%253Apmid%252F26400405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26400405&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.7" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Framework for Variation Discovery and Genotyping Using next-Generation DNA Sequencing Data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.8" data-doi="10.1111/j.1365-313X.2009.03828.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drost</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novaes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boaventura-Novaes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benedict</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A Microarray-Based Genotyping and Genetic Mapping Approach for Highly Heterozygous Outcrossing Species Enables Localization of a Large Fraction of the Unassembled <em>Populus Trichocarpa</em> Genome Sequence.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1067</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2009.03828.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DDrost%26rft.auinit1%253DD.%2BR.%26rft.volume%253D58%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1067%26rft.atitle%253DA%2Bmicroarray-based%2Bgenotyping%2Band%2Bgenetic%2Bmapping%2Bapproach%2Bfor%2Bhighly%2Bheterozygous%2Boutcrossing%2Bspecies%2Benables%2Blocalization%2Bof%2Ba%2Blarge%2Bfraction%2Bof%2Bthe%2Bunassembled%2BPopulus%2Btrichocarpa%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03828.x%26rft_id%253Dinfo%253Apmid%252F19220791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03828.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220791&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266925800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.9" data-doi="10.1093/bioinformatics/btu091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LPmerge: An R Package for Merging Genetic Maps by Linear Programming.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu091</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu091%26rft_id%253Dinfo%253Apmid%252F24532720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24532720&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1623.10"><div class="cit-metadata unstructured">Farjon, A., 1990 Pinaceae. Drawings and Descriptions of the Genera Abies, Cedrus, Pseudolarix, Keteleeria, Nothotsuga, Tsuga, Cathaya, Pseudotsuga, Larix and Picea. <em>Pinaceae. Drawings and Descriptions of the Genera Abies</em>, <em>Cedrus</em>, <em>Pseudolarix</em>, <em>Keteleeria</em>, <em>Nothotsuga</em>, <em>Tsuga</em>, <em>Cathaya</em>, <em>Pseudotsuga</em>, <em>Larix and Picea</em>. Koeltz Scientific Books. <a href="https://www.cabdirect.org/cabdirect/abstract/19920656698">https://www.cabdirect.org/cabdirect/abstract/19920656698</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.11" data-doi="10.3389/fgene.2015.00220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fierst</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Using Linkage Maps to Correct and Scaffold de Novo Genome Assemblies: Methods, Challenges, and Computational Tools.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2015.00220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D220%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00220%26rft_id%253Dinfo%253Apmid%252F26150829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26150829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.12" data-doi="10.1038/nrg1970"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination: An Underappreciated Factor in the Evolution of Plant Genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1970</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGaut%26rft.auinit1%253DB.%2BS.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D84%26rft.atitle%253DRecombination%253A%2Ban%2Bunderappreciated%2Bfactor%2Bin%2Bthe%2Bevolution%2Bof%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1970%26rft_id%253Dinfo%253Apmid%252F17173059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1970&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17173059&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242934300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.13" data-doi="10.1534/g3.115.019570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gracey</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Den Berg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samanta</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-Generation Linkage Maps for the Pacific Oyster Crassostrea Gigas Reveal Errors in Assembly of Genome Scaffolds. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2007</span>–<span class="cit-lpage">2019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.019570</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D5%26rft.spage%253D2007%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.115.019570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.14" data-doi="10.4137/CIN.S13495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Q.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">OmicCircos: A Simple-to-Use R Package for the Circular Visualization of Multidimensional Omics Data.</span> <abbr class="cit-jnl-abbrev">Cancer Inform.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4137/CIN.S13495</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BInform.%26rft.volume%253D13%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.4137%252FCIN.S13495%26rft_id%253Dinfo%253Apmid%252F24526832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4137/CIN.S13495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24526832&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A Test of Neutrality Based on Interlocus Associations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1197</span>–<span class="cit-lpage">1206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKelly%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D146%26rft.issue%253D3%26rft.spage%253D1197%26rft.epage%253D1206%26rft.atitle%253DA%2BTest%2Bof%2BNeutrality%2BBased%2Bon%2BInterlocus%2BAssociations%26rft_id%253Dinfo%253Apmid%252F9215920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Ni8zLzExOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.16" data-doi="10.1111/1755-0998.12549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knaus</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grünwald</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">vcfR : A Package to Manipulate and Visualize Variant Call Format Data in R.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12549</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.17" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and Accurate Short Read Alignment with Burrows-Wheeler Transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.18" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map Format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.19" data-doi="10.1371/journal.pone.0101049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lind</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Picea Abies Linkage Map Based on SNP Markers Identifies QTLs for Four Aspects of Resistance to Heterobasidion Parviporum Infection.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e101049</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0101049</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101049%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0101049%26rft_id%253Dinfo%253Apmid%252F25036209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0101049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25036209&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.20" data-doi="10.1111/j.2007.0018-0661.02000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">OneMap: Software for Genetic Mapping in Outcrossing Species.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.2007.0018-0661.02000.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DMargarido%26rft.auinit1%253DG.%2BR.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D78%26rft.epage%253D79%26rft.atitle%253DOneMap%253A%2Bsoftware%2Bfor%2Bgenetic%2Bmapping%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2007.0018-0661.02000.x%26rft_id%253Dinfo%253Apmid%252F17663699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2007.0018-0661.02000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17663699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248257800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.21" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce Framework for Analyzing next-Generation DNA Sequencing Data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.22" data-doi="10.1038/hdy.2009.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vencovsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evaluation of Algorithms Used to Order Markers on Genetic Maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2009.96</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMollinari%26rft.auinit1%253DM.%26rft.volume%253D103%26rft.issue%253D6%26rft.spage%253D494%26rft.epage%253D502%26rft.atitle%253DEvaluation%2Bof%2Balgorithms%2Bused%2Bto%2Border%2Bmarkers%2Bon%2Bgenetic%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.96%26rft_id%253Dinfo%253Apmid%252F19639011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.96&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19639011&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.23" data-doi="10.1038/nature12211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nystedt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wetterbom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuccolo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Norway Spruce Genome Sequence and Conifer Genome Evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12211%26rft_id%253Dinfo%253Apmid%252F23698360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698360&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319556100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.24" data-doi="10.1093/gbe/evt143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deschênes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blais</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavigne</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Landscape of Nucleotide Polymorphism among 13,500 Genes of the Conifer Picea Glauca, Relationships with Functions, and Comparison with Medicago Truncatula.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1910</span>–<span class="cit-lpage">1925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evt143</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt143%26rft_id%253Dinfo%253Apmid%252F24065735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24065735&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.25" data-doi="10.1111/tpj.13478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamothe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelgas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A High-Resolution Reference Genetic Map Positioning 8.8 K Genes for the Conifer White Spruce: Structural Genomics Implications and Correspondence with Physical Distance.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13478</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D90%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.26" data-doi="10.1186/1741-7007-10-84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelgas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laroche</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigault</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Spruce Gene Map Infers Ancient Plant Genome Reshuffling and Subsequent Slow Evolution in the Gymnosperm Lineage Leading to Extant Conifers.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1741-7007-10-84</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DPavy%26rft.auinit1%253DN.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D84%26rft.epage%253D84%26rft.atitle%253DA%2Bspruce%2Bgene%2Bmap%2Binfers%2Bancient%2Bplant%2Bgenome%2Breshuffling%2Band%2Bsubsequent%2Bslow%2Bevolution%2Bin%2Bthe%2Bgymnosperm%2Blineage%2Bleading%2Bto%2Bextant%2Bconifers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-10-84%26rft_id%253Dinfo%253Apmid%252F23102090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-10-84&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23102090&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.27" data-doi="10.5962/bhl.part.9959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sax</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sax</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1933</span> <span class="cit-article-title">Chromosome Number and Morphology in the Conifers.</span> <abbr class="cit-jnl-abbrev">J. Arnold Arbor.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">375</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5962/bhl.part.9959</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BArnold%2BArbor.%26rft.volume%253D14%26rft.spage%253D356%26rft.atitle%253DJ%2BARNOLD%2BARBOR%26rft_id%253Dinfo%253Adoi%252F10.5962%252Fbhl.part.9959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5962/bhl.part.9959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.28" data-doi="10.1371/journal.pone.0189256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiffthaler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernhardsson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">BatchMap: A parallel implementation of the OneMap R package for fast computation of F<sub>1</sub> linkage maps in outcrossing species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0189256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0189256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0189256%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0189256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0189256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.29" data-doi="10.1002/jez.1400140104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span><span class="cit-comment">a</span> <span class="cit-article-title">The Linear Arrangement of Six Sex-Linked Factors in Drosophila, as Shown by Their Mode of Association.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jez.1400140104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400140104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187376000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.30" data-doi="10.1126/science.37.965.990"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span><span class="cit-comment">b</span> <span class="cit-article-title">A Third Group of Linked Genes in Drosophila Ampelophila.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">990</span>–<span class="cit-lpage">992</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.37.965.990</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSturtevant%26rft.auinit1%253DA.%2BH.%26rft.volume%253D37%26rft.issue%253D965%26rft.spage%253D990%26rft.epage%253D992%26rft.atitle%253DA%2BTHIRD%2BGROUP%2BOF%2BLINKED%2BGENES%2BIN%2BDROSOPHILA%2BAMPELOPHILA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.37.965.990%26rft_id%253Dinfo%253Apmid%252F17833164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTA6IjM3Lzk2NS85OTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.31" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.32" data-doi="10.1002/0471250953.bi1110s43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Auwera</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">del Angel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">From FastQ Data to High Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Bioinformatics</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">11.10.1</span>–<span class="cit-lpage">11.10.33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471250953.bi1110s43</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi1110s43%26rft_id%253Dinfo%253Apmid%252F25431634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi1110s43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25431634&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.33" data-doi="10.1007/s00122-005-0097-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Os</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Eck</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">RECORD: A Novel Method for Ordering Loci on a Genetic Linkage Map.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-005-0097-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVan%2BOs%26rft.auinit1%253DH.%26rft.volume%253D112%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D40%26rft.atitle%253DRECORD%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Bordering%2Bloci%2Bon%2Ba%2Bgenetic%2Blinkage%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0097-x%26rft_id%253Dinfo%253Apmid%252F16228189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0097-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16228189&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.34" data-doi="10.1101/291716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vidalis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scofield</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neves</span>  <span class="cit-name-given-names">L.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernhardsson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Gil</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Design and evaluation of a large sequence-capture probe set and associated SNPs for diploid and haploid samples of Norway spruce (<em>Picea abies</em>)</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/291716</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F291716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyOTE3MTZ2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1623.35" data-doi="10.1006/tpbi.2002.1577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Painter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Simultaneous Maximum Likelihood Estimation of Linkage and Linkage Phases in Outcrossing Species.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/tpbi.2002.1577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D363%26rft.atitle%253DSimultaneous%2Bmaximum%2Blikelihood%2Bestimation%2Bof%2Blinkage%2Band%2Blinkage%2Bphases%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1577%26rft_id%253Dinfo%253Apmid%252F12027621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12027621&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176540100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1623.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1613" title="A Diallel of the Mouse Collaborative Cross Founders Reveals Strong Strain-Specific Maternal Effects on Litter Size" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1633" title="Genomic Variants Among Threatened Acropora Corals" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1623" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tirEh24wQWC8c8ergiLykvRKw2n5jdxXx0qBpo8COlA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1623" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1623" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1623" class="active" data-icon-position="" data-hide-link-title="0">An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce (Picea abies)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493103" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xeiREJi1JigZHNfPhv8mZFyhy6dJOpaJDWrVMVMMASQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1623.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1623" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-NUXKDS6t2Qwb8JPAWe-DpkPvcAyli66xKEtoqOIWQE0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1623" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1623.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493103" id="node493103--2" data-pisa="ggg;9/5/1623" data-pisa-master="ggg;g3.118.200840" data-apath="/ggg/9/5/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce <em>(Picea abies</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Carolina Bernhardsson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7111-9530" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amaryllis Vidalis</span>, <span class="highwire-citation-author" data-delta="2">Xi Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5235-6461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douglas G. Scofield</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bastian Schiffthaler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9919-907X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John Baison</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathaniel R. Street</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6834-6708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Rosario García-Gil</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pär K. Ingvarsson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1632;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200840 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493103--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Carolina Bernhardsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBernhardsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernhardsson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarolina%2BBernhardsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Carolina Bernhardsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:carolina.bernhardsson@gmail.com" class="" data-icon-position="" data-hide-link-title="0">carolina.bernhardsson@gmail.com</a>
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amaryllis Vidalis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Population Genetics, Center of Life and Food Sciences Weihenstephan, Technische Universität München, 85354 Freising, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amaryllis%2BVidalis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vidalis%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmaryllis%2BVidalis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7111-9530" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amaryllis Vidalis</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Douglas G. Scofield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Uppsala Multidisciplinary Center for Advanced Computational Science and</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Ecology and Genetics: Evolutionary Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douglas%2BG.%2BScofield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scofield%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouglas%2BG.%2BScofield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5235-6461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douglas G. Scofield</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bastian Schiffthaler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bastian%2BSchiffthaler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schiffthaler%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABastian%2BSchiffthaler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bastian Schiffthaler</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John Baison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BBaison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baison%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BBaison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9919-907X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John Baison</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nathaniel R. Street </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathaniel%2BR.%2BStreet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Street%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathaniel%2BR.%2BStreet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathaniel R. Street</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">M. Rosario García-Gil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BRosario%2BGarc%C3%ADa-Gil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa-Gil%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BRosario%2BGarc%25C3%25ADa-Gil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6834-6708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Rosario García-Gil</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Pär K. Ingvarsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P%C3%A4r%2BK.%2BIngvarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingvarsson%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP%25C3%25A4r%2BK.%2BIngvarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pär K. Ingvarsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:carolina.bernhardsson@gmail.com" class="" data-icon-position="" data-hide-link-title="0">carolina.bernhardsson@gmail.com</a>
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1623" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1623&amp;title=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1623&amp;title=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1623&amp;title=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1623&amp;text=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1623&amp;title=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1623&amp;t=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1623&amp;title=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1623&amp;title=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493103/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493103" id="node493103--3" data-pisa="ggg;9/5/1623" data-pisa-master="ggg;g3.118.200840" data-apath="/ggg/9/5/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Ultra-Dense Haploid Genetic Map for Evaluating the Highly Fragmented Genome Assembly of Norway Spruce <em>(Picea abies</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Carolina Bernhardsson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7111-9530" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amaryllis Vidalis</span>, <span class="highwire-citation-author" data-delta="2">Xi Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5235-6461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douglas G. Scofield</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bastian Schiffthaler</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9919-907X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John Baison</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathaniel R. Street</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6834-6708" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Rosario García-Gil</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pär K. Ingvarsson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1632;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200840 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493103--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Carolina Bernhardsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBernhardsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernhardsson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarolina%2BBernhardsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Carolina Bernhardsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:carolina.bernhardsson@gmail.com" class="" data-icon-position="" data-hide-link-title="0">carolina.bernhardsson@gmail.com</a>
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amaryllis Vidalis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Population Genetics, Center of Life and Food Sciences Weihenstephan, Technische Universität München, 85354 Freising, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amaryllis%2BVidalis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vidalis%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmaryllis%2BVidalis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7111-9530" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amaryllis Vidalis</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Douglas G. Scofield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Uppsala Multidisciplinary Center for Advanced Computational Science and</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Ecology and Genetics: Evolutionary Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douglas%2BG.%2BScofield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scofield%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouglas%2BG.%2BScofield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5235-6461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douglas G. Scofield</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bastian Schiffthaler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bastian%2BSchiffthaler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schiffthaler%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABastian%2BSchiffthaler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bastian Schiffthaler</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John Baison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BBaison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baison%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BBaison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9919-907X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John Baison</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nathaniel R. Street </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathaniel%2BR.%2BStreet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Street%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathaniel%2BR.%2BStreet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathaniel R. Street</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">M. Rosario García-Gil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Science, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BRosario%2BGarc%C3%ADa-Gil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa-Gil%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BRosario%2BGarc%25C3%25ADa-Gil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6834-6708" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Rosario García-Gil</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Pär K. Ingvarsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Environmental Science and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P%C3%A4r%2BK.%2BIngvarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingvarsson%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP%25C3%25A4r%2BK.%2BIngvarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pär K. Ingvarsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:carolina.bernhardsson@gmail.com" class="" data-icon-position="" data-hide-link-title="0">carolina.bernhardsson@gmail.com</a>
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1623&amp;rft.epage=1632&amp;rft.atitle=An%20Ultra-Dense%20Haploid%20Genetic%20Map%20for%20Evaluating%20the%20Highly%20Fragmented%20Genome%20Assembly%20of%20Norway%20Spruce%20%28Picea%20abies%29&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bernhardsson%2C+Carolina&amp;rft.au=Vidalis%2C+Amaryllis&amp;rft.au=Wang%2C+Xi&amp;rft.au=Scofield%2C+Douglas+G.&amp;rft.au=Schiffthaler%2C+Bastian&amp;rft.au=Baison%2C+John&amp;rft.au=Street%2C+Nathaniel+R.&amp;rft.au=Garc%C3%ADa-Gil%2C+M.+Rosario&amp;rft.au=Ingvarsson%2C+P%C3%A4r+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493103/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493103/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493103/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493103/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493103/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493103/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493103/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493103/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493103/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493103/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493103/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493103/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1623" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1623.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1623.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1623.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
