<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1355" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1355.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1355.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1355.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1355/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1355.atom" />
<meta name="HW.pisa" content="ggg;9/5/1355" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction" />
<meta name="DC.Identifier" content="10.1534/g3.119.400126" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Montesinos-López et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Evidence that genomic selection (GS) is a technology that is revolutionizing plant breeding continues to grow. However, it is very well documented that its success strongly depends on statistical models, which are used by GS to perform predictions of candidate genotypes that were not phenotyped. Because there is no universally better model for prediction and models for each type of response variable are needed (continuous, binary, ordinal, count, etc.), an active area of research aims to develop statistical models for the prediction of univariate and multivariate traits in GS. However, most of the models developed so far are for univariate and continuous (Gaussian) traits. Therefore, to overcome the lack of multivariate statistical models for genome-based prediction by improving the original version of the BMTME, we propose an improved Bayesian multi-trait and multi-environment (BMTME) R package for analyzing breeding data with multiple traits and multiple environments. We also introduce Bayesian multi-output regressor stacking (BMORS) functions that are considerably efficient in terms of computational resources. The package allows parameter estimation and evaluates the prediction performance of multi-trait and multi-environment data in a reliable, efficient and user-friendly way. We illustrate the use of the BMTME with real toy datasets to show all the facilities that the software offers the user. However, for large datasets, the BME() and BMTME() functions of the BMTME R package are very intense in terms of computing time; on the other hand, less intensive computing is required with BMORS functions BMORS() and BMORS_Env() that are also included in the BMTME package." />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="Francisco Javier Luna-Vázquez" />
<meta name="DC.Contributor" content="Fernando H. Toledo" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Morten Lillemo" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Evidence that genomic selection (GS) is a technology that is revolutionizing plant breeding continues to grow. However, it is very well documented that its success strongly depends on statistical models, which are used by GS to perform predictions of candidate genotypes that were not phenotyped. Because there is no universally better model for prediction and models for each type of response variable are needed (continuous, binary, ordinal, count, etc.), an active area of research aims to develop statistical models for the prediction of univariate and multivariate traits in GS. However, most of the models developed so far are for univariate and continuous (Gaussian) traits. Therefore, to overcome the lack of multivariate statistical models for genome-based prediction by improving the original version of the BMTME, we propose an improved Bayesian multi-trait and multi-environment (BMTME) R package for analyzing breeding data with multiple traits and multiple environments. We also introduce Bayesian multi-output regressor stacking (BMORS) functions that are considerably efficient in terms of computational resources. The package allows parameter estimation and evaluates the prediction performance of multi-trait and multi-environment data in a reliable, efficient and user-friendly way. We illustrate the use of the BMTME with real toy datasets to show all the facilities that the software offers the user. However, for large datasets, the BME() and BMTME() functions of the BMTME R package are very intense in terms of computing time; on the other hand, less intensive computing is required with BMORS functions BMORS() and BMORS_Env() that are also included in the BMTME package.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1355" />
<meta name="citation_id" content="9/5/1355" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1355" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1355.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1355.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1355.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400126" />
<meta name="citation_pmid" content="30819822" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="1355" />
<meta name="citation_lastpage" content="1369" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías, (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="aml_uach@hotmail.com" />
<meta name="citation_author" content="Francisco Javier Luna-Vázquez" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México" />
<meta name="citation_author" content="Fernando H. Toledo" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center, (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, CP 56230, Montecillos, Edo. de México, México" />
<meta name="citation_author" content="Morten Lillemo" />
<meta name="citation_author_institution" content="Department of Plant Sciences, Norwegian University of Life Sciences, IHA/CIGENE, P.O. Box 5003, NO-1432 Ås, Norway" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center, (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="aml_uach@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_reference" content="citation_author=M. Ben Hassen;citation_author=J. Bartholomé;citation_author=G. Valè;citation_author=T.-V. Cao;citation_author=N. Ahmadi;citation_year=2018;citation_pmid=29743189;citation_doi=10.1534/g3.118.200098" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. D. L. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype x environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_author=J. Burstin;citation_author=P. Salloignon;citation_author=M. Martinello;citation_author=J. B. Magnin-Robert;citation_author=M. Siol;citation_author=F. Jacquin;citation_title=Genetic diversity and trait genomic prediction in a pea diversity panel.;citation_year=2015;citation_doi=10.1186/s12864-015-1266-1" />
<meta name="citation_reference" content="citation_journal_title=Evolution : GSE;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods. Genetics, Selection;citation_pages=26;citation_volume=43;citation_year=2011;citation_doi=10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26" />
<meta name="citation_reference" content="citation_journal_title=Pesqui. Agropecu. Bras.;citation_author=A. F. N. M. Castro;citation_author=R. V. Castro;citation_author=C. A. O. Oliveira;citation_author=J. E. Lima;citation_author=R. C. Santos;citation_title=Multivariate analysis for the selection of eucalyptus clones destined for charcoal production.;citation_pages=627-635;citation_volume=48;citation_year=2013;citation_doi=10.1590/S0100-204X2013000600008" />
<meta name="citation_reference" content="citation_journal_title=Rencontres Recherches Ruminants;citation_author=J. J. Colleau;citation_author=S. Fritz;citation_author=F. Guillaume;citation_author=A. Baur;citation_author=D. Dupassieux;citation_title=Simulating the potential of genomic selection in dairy cattle breeding.;citation_pages=419;citation_volume=16;citation_year=2009" />
<meta name="citation_reference" content="citation_author=G. Covarrubias-Pazaran;citation_pages=e0156744;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0156744" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=G. Covarrubias-Pazaran;citation_author=B. Schlautman;citation_author=L. Diaz-Garcia;citation_author=E. Grygleski;citation_author=J. Polashock;citation_title=Multivariate GBLUP Improves Accuracy of Genomic Selection for Yield and Fruit Weight in Biparental Populations of Vaccinium macrocarpon Ait.;citation_pages=1310;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.01310" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="de los Campos, G., and A. Grüneberg, 2016 MTM (Multiple-Trait Model) package [WWW Document]. URL http://quantgen.github.io/MTM/vignette.html (accessed 10.25.17)." />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=E. J. de Oliveira;citation_author=M. D. Vilela de Resende;citation_author=V. Silva Santos;citation_author=C. Fortes Ferreira;citation_author=G. A. Fachardo Oliveira;citation_title=Genome-wide selection in cassava.;citation_pages=263-276;citation_volume=187;citation_year=2012;citation_doi=10.1007/s10681-012-0722-0" />
<meta name="citation_reference" content="citation_journal_title=Genomes and Genetics;citation_author=I. Granato;citation_author=J. Cuevas;citation_author=F. Luna-Vázquez;citation_author=J. Crossa;citation_author=O. A. Montesinos-López;citation_title=BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models. Genes;citation_volume=8;citation_year=2018;citation_pmid=30049744;citation_doi=10.1534/g3.118.200435" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. Gilmour;citation_author=R. Thompson;citation_author=B. Cullis;citation_title=Average Information REML: An Efficient Algorithm for Variance Parameter Estimation in Linear Mixed Models.;citation_pages=1440-1450;citation_volume=51;citation_year=1995;citation_doi=10.2307/2533274" />
<meta name="citation_reference" content="citation_journal_title=J STAT SOFT;citation_author=J. D. Hadfield;citation_pages=1;citation_volume=33;citation_year=2010;citation_doi=10.18637/jss.v033.i02" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Genome-wide association and genomic selection in animal breeding.;citation_pages=876-883;citation_volume=53;citation_year=2010;citation_issue=11;citation_pmid=21076503;citation_doi=10.1139/G10-076" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. He;citation_author=D. Kuhn;citation_author=L. Parida;citation_title=Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction.;citation_volume=32;citation_year=2016;citation_pmid=27307640;citation_doi=10.1093/bioinformatics/btw249" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Huang;citation_author=L. Chen;citation_author=Z. Chen;citation_title=Diallel analysis of combining ability and heterosis for yield and yield components in rice by using positive loci.;citation_pages=37-50;citation_volume=205;citation_year=2015;citation_doi=10.1007/s10681-015-1381-8" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. Heuer;citation_author=C. Scheel;citation_author=J. Tetens;citation_author=C. Kühn;citation_author=G. Thaller;citation_title=Genomic prediction of unordered categorical traits: an application to subpopulation assignment in German Warmblood horses.;citation_pages=13;citation_volume=48;citation_year=2016;citation_doi=10.1186/s12711-016-0192-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=D. Jarquín;citation_author=C. Lemes da Silva;citation_author=R. C. Gaynor;citation_author=J. Poland;citation_author=A. Fritz;citation_title=Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.12.0130" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Jiang;citation_author=Q. Zhang;citation_author=L. Ma;citation_author=J. Li;citation_author=Z. Wang;citation_title=Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.;citation_pages=29-36;citation_volume=115;citation_year=2015;citation_pmid=25873147;citation_doi=10.1038/hdy.2015.9" />
<meta name="citation_reference" content="citation_author=S. Kumar;citation_author=D. Chagné;citation_author=M. C. Bink;citation_author=R. K. Volz;citation_author=C. Whitworth;citation_author=C. Carlisle;citation_year=2012" />
<meta name="citation_reference" content="citation_author=M. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_author=J.-L. Jannink;citation_author=R. P. Singh;citation_author=E. Autrique;citation_author=G. de los Campos;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_author=K. M. Eskridge;citation_author=J. Crossa;citation_year=2015" />
<meta name="citation_reference" content="citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=J. Burgueño;citation_author=K. Eskridge;citation_year=2015;citation_pmid=26290569;citation_doi=10.1534/g3.115.021154" />
<meta name="citation_reference" content="citation_author=O. A. Montesinos-Lopez;citation_author=A. Montesinos-Lopez;citation_author=J. Crossa;citation_author=F. H. Toledo;citation_author=O. Perez-Hernandez;citation_author=K. M. Eskridge;citation_author=J. Rutkoski;citation_year=2016;citation_pmid=27342738;citation_doi=10.1534/g3.116.032359" />
<meta name="citation_reference" content="citation_journal_title=Agron. J.;citation_author=A. Montesinos-López;citation_author=O. A. Montesinos-López;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=C. M. Hernández-Suárez;citation_title=Multivariate Bayesian analysis of on-farm trials with multiple-trait and multiple-environment data.;citation_pages=1-12;citation_volume=111;citation_year=2018;citation_doi=10.2134/agronj2018.06.0362" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Montesinos-López;citation_author=O. A. Montesinos-López;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=C. M. Hernández-Suárez;citation_title=Multi-trait, multi-environment deep learning modeling for genomic-enabled prediction of plant traits. G3: Genes, Genomes;citation_pages=3829-3840;citation_volume=8;citation_year=2018;citation_doi=10.1534/g3.118.200728" />
<meta name="citation_reference" content="citation_title=Physical Methods for Stimulation of Plant and Mushroom Development;citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. C. Montesinos-López;citation_author=J. Crossa;citation_author=F. J. Luna-Vázquez;citation_title=A Bayesian Multiple-Trait and Multiple-Environment Model Using the Matrix Normal Distribution;citation_year=2018;citation_doi=10.5772/intechopen.71521" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=K. M. Olson;citation_author=P. M. VanRaden;citation_author=M. E. Tooker;citation_title=Multibreed genomic evaluations using purebred Holsteins, Jerseys, and Brown Swiss.;citation_pages=5378-5383;citation_volume=95;citation_year=2012;citation_issue=9;citation_pmid=22916944;citation_doi=10.3168/jds.2011-5006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-Wide Regression and Prediction with the BGLR Statistical Package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="Plummer, M., 2018 rjags: Bayesian Graphical Models using MCMC. URL https://CRAN.R-project.org/package=rjags. R package version 4–6." />
<meta name="citation_reference" content="R Core Team, 2018 R: A Language and Environment for Statistical Computing. Vienna, Austria. Retrieved from https://www.r-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. E. Rutkoski;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_title=Genomic selection for durable stem rust resistance in wheat.;citation_pages=161-173;citation_volume=179;citation_year=2011;citation_doi=10.1007/s10681-010-0301-1" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=L. R. Schaeffer;citation_title=Multiple trait international bull comparisons.;citation_pages=145-153;citation_volume=69;citation_year=2001;citation_pmid=11337117;citation_doi=10.1016/S0301-6226(00)00255-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. W. Schulthess;citation_author=Y. Zhao;citation_author=C. F. H. Longin;citation_author=J. C. Reif;citation_title=Advantages and limitations of multiple-trait genomic prediction for Fusarium head blight severity in hybrid wheat (Triticum aestivum L.).;citation_year=2017;citation_doi=10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=P. Virk;citation_author=B. Collard;citation_title=Genomic selection and association mapping in rice (Oryza sativa): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.;citation_pages=e1004982;citation_volume=11;citation_year=2015;citation_pmid=25689273;citation_doi=10.1371/journal.pgen.1004982" />
<meta name="citation_reference" content="Spyromitros-Xioufis, E., W. Groves, G. Tsoumakas, and I. Vlahavas, 2012 Multi-label classification methods for multi-target regression. arXiv preprint arXiv:1211.6581, 2012, 1159–1168. Cornell University Library." />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=E. Spyromitros-Xioufis;citation_author=G. Tsoumakas;citation_author=W. Groves;citation_author=I. Vlahavas;citation_title=Multi-target regression via input space expansion: treating targets as inputs.;citation_pages=55-98;citation_volume=104;citation_year=2016;citation_doi=10.1007/s10994-016-5546-z" />
<meta name="citation_reference" content="citation_title=An Introduction to Multivariate Statistics.;citation_author=M. S. Srivastava;citation_author=C. G. Khatri;citation_year=1979" />
<meta name="citation_reference" content="Stan Development Team, (2018). Stan Modeling Language Users Guide and Reference Manual, Version 2.18.0. http://mc-stan.org." />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=F. Technow;citation_author=A. E. Melchinger;citation_title=Genomic prediction of dichotomous traits with Bayesian logistic models.;citation_pages=1133-1143;citation_volume=126;citation_year=2013;citation_pmid=23385660;citation_doi=10.1007/s00122-013-2041-9" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=R. Thompson;citation_author=K. Meyer;citation_title=A review of theoretical aspects in the estimation of breeding values for multi-trait selection.;citation_pages=299-313;citation_volume=15;citation_year=1986;citation_doi=10.1016/0301-6226(86)90071-0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=T. Würschum;citation_author=J. C. Reif;citation_author=T. Kraft;citation_author=G. Janssen;citation_author=Y. Zhao;citation_title=Genomic selection in sugar beet breeding populations.;citation_pages=85-93;citation_volume=14;citation_year=2013;citation_pmid=24047500;citation_doi=10.1186/1471-2156-14-85" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=E. Yamamoto;citation_author=A. Matsunaga;citation_author=A. Onogi;citation_author=H. Kajiya-Kanegae;citation_author=M. Minamikawa;citation_title=A simulation-based breeding design that uses whole-genome prediction in tomato.;citation_pages=19454;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep19454" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Evidence that genomic selection (GS) is a technology that is revolutionizing plant breeding continues to grow. However, it is very well documented that its success strongly depends on statistical models, which are used by GS to perform predictions of candidate genotypes that were not phenotyped. Because there is no universally better model for prediction and models for each type of response variable are needed (continuous, binary, ordinal, count, etc.), an active area of research aims to develop statistical models for the prediction of univariate and multivariate traits in GS. However, most of the models developed so far are for univariate and continuous (Gaussian) traits. Therefore, to overcome the lack of multivariate statistical models for genome-based prediction by improving the original version of the BMTME, we propose an improved Bayesian multi-trait and multi-environment (BMTME) R package for analyzing breeding data with multiple traits and multiple environments. We also introduce Bayesian multi-output regressor stacking (BMORS) functions that are considerably efficient in terms of computational resources. The package allows parameter estimation and evaluates the prediction performance of multi-trait and multi-environment data in a reliable, efficient and user-friendly way. We illustrate the use of the BMTME with real toy datasets to show all the facilities that the software offers the user. However, for large datasets, the BME() and BMTME() functions of the BMTME R package are very intense in terms of computing time; on the other hand, less intensive computing is required with BMORS functions BMORS() and BMORS_Env() that are also included in the BMTME package." />
<meta name="og-title" property="og:title" content="An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1355" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Evidence that genomic selection (GS) is a technology that is revolutionizing plant breeding continues to grow. However, it is very well documented that its success strongly depends on statistical models, which are used by GS to perform predictions of candidate genotypes that were not phenotyped. Because there is no universally better model for prediction and models for each type of response variable are needed (continuous, binary, ordinal, count, etc.), an active area of research aims to develop statistical models for the prediction of univariate and multivariate traits in GS. However, most of the models developed so far are for univariate and continuous (Gaussian) traits. Therefore, to overcome the lack of multivariate statistical models for genome-based prediction by improving the original version of the BMTME, we propose an improved Bayesian multi-trait and multi-environment (BMTME) R package for analyzing breeding data with multiple traits and multiple environments. We also introduce Bayesian multi-output regressor stacking (BMORS) functions that are considerably efficient in terms of computational resources. The package allows parameter estimation and evaluates the prediction performance of multi-trait and multi-environment data in a reliable, efficient and user-friendly way. We illustrate the use of the BMTME with real toy datasets to show all the facilities that the software offers the user. However, for large datasets, the BME() and BMTME() functions of the BMTME R package are very intense in terms of computing time; on the other hand, less intensive computing is required with BMORS functions BMORS() and BMORS_Env() that are also included in the BMTME package." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"f4z-Rf69oSR8bf-Ud3bHoOG5ZBadVMnDqba4o_opZxQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492870","apath":"\/ggg\/9\/5\/1355.atom","pisa":"ggg;9\/5\/1355","ac":{"\/ggg\/9\/5\/1355.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1355.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1355"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1355"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1355.abstract":true,"\/content\/9\/5\/1355":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oRfPjEIL1o2vq\\\\\\\/ZBYpzt7TbUlYiuo+RQhhdMj6kNB2XwctkJ+hasByfpHQiiRr8m1uEPORxUpfc1L8elGmaB79+\\\\\\\/6anNMc41LOQTxOnb6NS6IjI2nnzoYJvtURdVtwsVoOMpsYyxXo3sn1urYmnV1kaD0keLsrYWG4YT56M8nliW+mMd2u3IJv1\\\\\\\/UmPR9ooPKJ\\\\\\\/7nnDLn5pb0FeW7oJmdb7YxncnlbRHN4a8bgrCD0Q8ZkZkQu5SiwRv9Ev56PntX4U\\\\\\\/vq86QEBTHUrniS9Jma8lxxSR0GWRStDCe63+bqg5NtnB3uZ\\\\\\\/2P4ErS8BNihuDx+nIhHEEfZgxMVQ\\\\\\\/wLgh4ojJP2i46M3I+7uF4corVu1lhoCs2SZaYoycrK+z\\\\\\\/JFfdHw9x0pFkv3fmPOs\\\\\\\/HUkyN8TaU9cnwceU4qsyAMoVif7Egu\\\\\\\/uZDd7Ar0floiF3YFT9O1gUxXVZBands9phTQncse1Hoj63WPbks0\\\\\\\/PHzNll2Kabg4dOiZbCaPEIOV0pWnPeEA90BdbHxm30r3lTfbpLcIaEVtjKoBpcKUvHxHJDWxQVjjV3hE0IkVCeevI3QaF100bCrMzqn\\\\\\\/pk5La3zJOvRlztZgtzfRnBywvyWfk7SsH0ajvDi+YbyUXPHaJS+kpm8\\\\\\\/73byJt2N501THa5hXQhH+a7WMvT+ao4Fd4rmOoM8Nwwavqv8YCXN1je5ILpHrVwY1SwgcxTxI1IjYw9MTXMEvO+uKMsrP8Bla0DNgktlsFIQhP65runT3kGofXJtw31i3zbuKjl+yFIsOgGLmXcv3tbOy4K8bWkJkeHjjj66umdgIy4qO7ZEEkfA3MRPcb2bx\\\\\\\/61Yzc9X+mSbtqI3ctG1IZK3\\\\\\\/hPHfYsT3\\\\\\\/iJtrcNPaRX9aIOy\\\\\\\/jTmBHiyU7lcTo4iQK1dB5Ybq58BT9oxcsWCVH7SXiyrs2V3do92O7AlUo13adKac3j6eBr8FBGPgrbi\\\\\\\/+yQIJvq58vvlBQ416LaA9DydQghDfc1EJgJrcl\\\\\\\/DuzLdbP9BtAmkC\\\\\\\/BQsBhaHV07vXoIpSKILL14SN+tY6\\\\\\\/Jbxa9bzTy5JAW5WxwsGKSKQ9HjCbKuP1w5dpBIFiryhPWQyr3QScKVbl6LwPRE+GqnZn8GSkUm6MOPhGBE5RAi30fdskbAEIIm+i3x2qLIK9bLfXGvUeNMjaZu63pYLxtSVjryphD0pDZK1h5OJ9yGrPtkBbuOWuKfx0bnko5g515dccLvKoJIJ2AYkat+FPfKX7LwaQqnqoqG7wLzrfWLN7uTjshCGgARfI\\\u0022,\\\u0022iv\\\u0022:\\\u0022AkMS7TDffCBZ072v7M6OGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002239311a079b0740dc67dd1885a7978455b658060b3727e3d77f3b9b69135dc442\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NGo5zoQtkZceojdnjQWuJSFW3cKf\\\\\\\/+Aog0xUqkpG92EumB7Cjx0+psyfmxyGJIhaSK62kceRAWMLRZDyV4KvcaVHgm8manBSVKFNMm\\\\\\\/4UcL+cy\\\\\\\/E+5XGGOkiFsfy+IeH5O8WSu9dKn1BeowaGYJ4pv693c1WWwiqbhtpIUqwPvfdNI+Wc5mmDwMAZqzWZRWYQOUd9tetiMUiTecpF\\\\\\\/vRLI\\\\\\\/KDtpjNgaEi91yV6sAYYkzH6LPC8UarkFNKvM1UMWId\\\\\\\/ZfCTUQmGHdtfz2esbV\\\\\\\/sDQcg8hJyStOQvxSgY3p1iGg8\\\\\\\/mYTFX7PZuAMVV14WDb+fvXK75evobilkJnXNtIFD27ccFX3li3wcZCXa5Vuv523YMJxhCLfQ+zog1S6gjniE\\\\\\\/m\\\\\\\/lT9BKLGOaKbwnezy2tFo5JKqLhDmR39BBJznQptkhwm09ZnVQmepLYKDbAqCB+GiyS\\\\\\\/e927FgW7BHU\\\\\\\/3AWEMiuKsLaFT3jfCdZx953dZ9IPp8Rr+aMVxz\\\\\\\/A1YgZF7jMqDSnUlBPALrabr+Tia3VleF+RcEEIAxEPfdKhNasF2zeaPuLYNC7bDt0SgXrgFv+d3V1RHhpUQds1\\\\\\\/BhmAPlUlHAoTP8TOAp8Lt\\\\\\\/Q5PyXgveTBrcD8TFJFKZKej5AEZw7SIFDIL3tl5SEK\\\\\\\/tqiSFQznr55oUEVE5wVsOSFXoNwer1AREwG1ksTffpzl++gcAmeV\\\\\\\/UIHKgYVGfcIQj8Tykp4vtlBetCcC9ynQ3ftwQiVlSFo4N+xKCNGUr7H3B1EgU3Iv\\\\\\\/jE9yZ+PsSjqU7bHsuCQUQlFu+XvrDY+FamFsyXeO7SV0mhiD\\\\\\\/AuM2vfy2Q0cn\\\\\\\/f7N+mfnwUZ8SGlQOGsuLQh92Ec5tQGFWru\\\\\\\/8mXtcBIMZwpQEnoAJ81Z1YRErDoa5ft7tvdcxsE1CdIQkmgfBY1b9vOiTOLu95+dR5CBdr+PlM4gacsXWLGSQqJmtGvNlRo7ivTMMMUn1Hwl2khwreexU9ijLRM2CLxy0BPGXYkjjVXUQjkWubDMxG2ewVmn2lq86YhoQmPbX37rvSYFPuFmywvogbyxo6tkF2cmuWLXfxbusNDzmwE\\\\\\\/+YRzEgcujGT7L\\\\\\\/ug+SDM2FpeZJQfWeXaUxgFjO4GlhnUIadhdpwIlnf5fbNqCxNCB\\\\\\\/Ix25jfVYZxGd\\\\\\\/uVMNqD5\\\\\\\/Ob2cdacawgcB8O+eo0gEDMFEBG5blcGOp\\\\\\\/z9Uy4NKGaOGaSr1D9UOCrH7jh5tcW8SCkX3gOGZaiyZicr+OxG9vH\\\\\\\/6RCQq4ltMWIYhSCMGlPDR+ifgucuw9rohphMsSaOuDrtAcc8EKQ9ZmcmUhWWB+lOCGQrZRrrufQ6qicSN2QRoet+lfhyKgBVD4C4m6LCI0axniDJPBMi00aQUNKBpTZw4qLYVqaRKFnxUs\\\\\\\/epo+8RmXnI7GkVjBIp1m8WGMCPl5sVCVGPuZCEDatgjrjlMG5iZMDU\\\\\\\/08JXolJEeVSP9Wk\\\\\\\/kLOX2iLZ8QxdkPVF4oCVEOtw2SjLFmhb0zCJgDzu5Csb+yGhNUMDB4U7OjYcvl9CFr402L7n0fCn0+uC62uSPVf51UvhMLTgifJhKn+6w7kLEqG7f9pBoa0yna67ANzpxSCxCnFlPpRNoC4T4LwV3V7fUOb5CkC343kdnoEqJBFriQ1JZwrw2A2WI03in2xCzJuWvJeEVsGP1KmqsSbZ8COX0oVEr5q\\\\\\\/yZzjGhC59t4x283bsGnZTNprO3vkcXcgqQ8vOR1c7wRrjlobHJ+5EeOaWK5VQz42AoBKkb8pJnrRnxd8x5LzrMs\\\\\\\/faPMfEwC7rykjPUYJT8dUKdEpK6aDCPhK7RuDgWKDidWb\\\\\\\/jJzEa2ZWYlLowhjWMTFrAi7Gx\\\\\\\/XIHJLvAYjw6Vp7A0iL1l8YKESq6sOa81Cjbvxop74vfZZFuy0bcqvUl3dDXW521DrtJo4znx3PyF2sth7aDUrPAn5D3QEgftTldZzddbh2b++UGRZOcxJd\\\\\\\/2O5TUApnHDrRAFBTHZb0IRpEkw1mTBX0y7mS7NGHbOwDN3Atb5HhsNY1lkFU\\\\\\\/Q\\\\\\\/zbDU8lpt4PlZ7Pjt4+S5G7lRVIIT8IdN\\\\\\\/NF1jZ+lIO31U9xXYJpNq3XOP3HjnDE2Sdt4wYAwulu+FpVDwqPSkO8ltYdFSrFaeZF+0afazdrIeAXpYhbcEpf+kUCa5dQs84ZXJOmC50qv9q8HbFpqDWUd3jSFtIK54yGZrhthlBkKLgrhS5loo\\\\\\\/jlHeY\\\\\\\/JDFEVKQKZDYua6R4xjkR6FTm2si\\\\\\\/jzLJVSAkvLY6vR75M9vmX1hQ0Upd0wQSjp7CVPJLVtuGDYfcM3ezRtGwZJLtseyg0IJIpJ7JwAKYVD71trHVBENdcEGdCgDWV8wZzLlHFLLbNHsaeO\\\\\\\/gNLoQXuaOHBnMig9v1V\\\\\\\/o9zbB+vxd8Pp6oB2moxGZNtaL9NSAE0YsSdL0RJZwwXZ978JrjkAVsVDFTruFDpTh74pGN2PL5haFFJ5pxQk61oZUZ6QHNTGxj\\\\\\\/gCYX7AYFMu0eDTaqO59PfD+S+MmEz4meEsp+xQ0g3Hvme+WukBbv9dkFGl\\\\\\\/+zRJJu6BMNfiSWzlfQ+dxgLjlV1hOw3gIh7GMXQgZsZXnlPtqzQrchvUwSgNRDZP8x5AMe6SOyc6f9if8p5TdFUloXUi+rFjqZMr1lQYDCFIy3mhj3FcsVh3FihaGIAu\\\\\\\/lz8nci+COc9q4a5d4AdvQ4zrtFRU8FFbG4OG4UtzD7LXqb1\\\\\\\/C2ISIY4yPYnIoE0CEJRsKIXsEJHrxQ7kh8mNwtGmZyC88bDJKZsHY8XQBvSJAdGeh+yfi\\\\\\\/CQm2svMGuxm64svnoD2+2pa6K5xtSkVQpTbk3ciLWKswtw83gw0SwEZX7WebVddQ7vB9QTNiNvLZY\\\\\\\/ES\\\\\\\/4v5m4AartlpoC9ox3UTdkravN8QoGWrW3mu8eOlTa3Vve4zigpjI423OWq+XkUd9dDUn+vOdB8qmE70NxVdXOaCFGvoYhK\\\\\\\/sK5RXOlzv8fCuqKUba9arbRRdSDB0wMKT+NMARGGBj\\\\\\\/H9JmXjOcHddykwgiI9WDvaxASbszjvUSJcjSet7\\\\\\\/gJ6232Yrvm56nmiThUM3FrYzSgdVC84a24VPMl+XYwE+mf\\\\\\\/yk\\\\\\\/zYXw7CtvnSuvbXxWY3F0\\\\\\\/awfPf\\\\\\\/qKd2zyJTqEhvlAOWa7URMjureMX5\\\\\\\/03TEOO4\\\\\\\/yTgPaM7dGOxKuv0MI9gWRzf\\\\\\\/stc++5YeFa3zT0iP2ov+4CpZ+D4jFQx4UXf34zg9DXe5Kj85MBJmlXdTsUhdihpihrYoH3XqZ3NMt1Zh1TP9QTM\\\\\\\/G2ZbRip5ePCZAhniQ3hVN4ECzM4Ji9G2Gy6emKdtG4oWfQ8wR9vCR5Vpae\\\\\\\/GQ\\\\\\\/mC7C+mGDw6hiX2sftgqbwQvwUvb0edzS+19P\\\\\\\/HrzhWzgHjB2VBhtroxf2bZDKRA04WI20TB6LN4v9+VHEZmU+M2Gct7uU4+5rUFaH2U5d9mvGZzeA4Nz3CZLTzKVaoh5RyZsbKl0h57k6oGhzTOd71VsvviZ+EPT\\\\\\\/1a4L4blQFuOUJ5XK5ORfIRbSKcZYQ7rTpLzvLurGijr5GmSFM76tWBg7K5y3GfqWIiaJNrZOBZ+ijg0VNuA4wiKoz1icmuU6y81vnWATqw4v8zPYxku4PNq37osXS+6JBPNYM8qwsXpjcO2nKuZGGhNHn6vZ9GRhgJkxrbxAX9YEeIOG+nd8ptYSHKdixjbE4m7jMMX572ZSxH+dws\\\\\\\/TGBbZL5uU3PhqMLO\\\\\\\/4H6FRx0yVo29U9CjWesmhfbhzkrE98+voJ8Po9Ii79OqWaE2KWOKRcRgmvP\\\\\\\/hEXROeqrpYyDIWVo9IAO9zYiSeH4mUt2BIprI8Yrzy3DW2OF9+BOotUUQB4XBcFKTwsefPG2cb9jbt014\\\\\\\/ZHKVpoZ1sOrNCHPAqkZcEkMyzVDRCEhcQMY9uYaLajisWGRhqBOdKRh0h3ipVfpNwX6KVuYonl8rOwBKJYKU564b211zlUU4OVHqbjv40PzxQcXbAdmhGYSZhGMllI+xY4PaLYPuWo+tovi1LkjOPP9akPts7Pf4VXbTSG+21giGZa8+vLjnDU0f5cGoOVECUXoNrquFZzu9uGhaJv\\\\\\\/DrGRbcBgZLMMVm9uItGawUnEU6AzlBFlTHzHzRbdB0mtWFl8qc88CPhmzAFWjw4FOAazoxNZtne8ZnlL7BhGgNMG+k3xE8UtJ3mvTjj2ouZyuTa14jP3RKENazw6vskV2o4TkjV0iqeXp42YB372U5ljYbVMgemUkyQDMYIRuOK50O0rmXB1TrQhS8GI6didZ\\\\\\\/WdGNOWbyACQDdCpXDidlPyPis8qeZQJlJkWO5R4XKNWzd129pYsbe0qBKJtNdmFEOauNirSbcxvvNjZ3fzzdbplG0IE3pK7ZLexiVAO92XqzKncIJrqfqizg3rUVxb0big4paPA09ZFgxp9VvckV9g\\\\\\\/vCCL7k3b+wfZHvAnfVexHhLnBxhmx4zTKPypVOIBC5uEyygn0YHOdEecAk9muP4jH6NnwC\\\\\\\/YtWjUKzA42BrCkIzhQvTY5xL1IXuPBiAVZjuThh99CehbisAh4Q2y8B4Vcrs0GTS25KaQAv2+XzGI7oE2UeMOcYet2kykIduWn7OQJVIoqdMohtfFTOZqUXaBjwbeEkb\\\\\\\/B0vI7RUB+PtcoALevEXY4lhxuLBvOVMBSylGIQlSr0AOTQrNrGipPqsg2ri8CyvGM5xgB01vcKnyu0rrMgXvpj6y7VTtEMkkCNljtOjW84\\\\\\\/25k1S7g0g\\\\\\\/5BPxpAVTorRkWMilZRI54HK6wsvYST92Vn+U4szvoZYkR1WbmhpSn+2Tlk1qRy3TXjvRb88D56oTEOFwxZhukS54Epj4Sk6Bc3AZY9jOzuE7Ia\\\\\\\/rRE4Ahk+C3Kdpx7+RZW958BhVv6X1drdUTNE53pKaWVftTOKuhD9fP4sEr9fAQ5XsjoHA+Em\\\\\\\/jPTDMTiL0HrJMhJVo0cV81YjRZWzZr\\\\\\\/MA\\\\\\\/Otf1gw9aMGGTepPir2viIlpUFISpGJ6+ic2PACi1YrN04OZKpB9GxnLrhlJCoSzRXxIFj8kO8x5dkSXzHPMsSNwuUeJixE5T0r0q0oK545ThBwD4oNxKWCvEQHkgGoJiNcWTu5QXoMZg5FIBdoQkFGZdeKi4f59u1ln6JgjaU\\\\\\\/HLRAKHAPrLp2\\\\\\\/fTZdjwU3\\\\\\\/WIWXk5rtWwNyEWDMNmDgFKeEfQ8D7CfoknXjfzlyWEKv1FnHjiUbivGK0mY4cHPYMQZTl4JGkL+mhaubh2uxDDrOHgpvNKmFAlGnFJQgWAlIUCbO4njzQ6+u7s9HaFiolzEa5Xk0Ovmv17V3jdBo\\\\\\\/eoRys7KMKD7CsWwPFSxBfjR1JQ\\\\\\\/rI9KeXTxXx1OP0Sa6oAZMxIThvPE6sTE0PWX+SpFDHW8zP92bkcJrpeO6JdO+uPD8BDD97bUNBu8Cng1pd3nURzIJYl6Z7f+iv44T0ecuIS39nVBmQp+f9kvXMAGYWxQOiHoTmqXrgDNuMmboftbUPWSNMUF0BBTExhDBLqBjJNJCHZ7lNWfihwC6yiRkc1N55GPlPwfrgz21jGGF0hzxM2fO5WNPwJR0TUGtUEcy9h7HqayeSctAAeAK4rXKiGWm6OHVBLpsL8T\\\\\\\/SIQVVoPmP2Ma8aFbrq+z8QnoZ1e7hJX8hOGDqob894pGiO6BaV3BH8y2xXk8JsxFJihNuz82bJSDRLnXBPlpaHmeVpA0XXYyYlV3FeGax6KCMeaFZY9O8fT5MH5NECzwwFEo5I8Cmb1Ev8yrCrxJDd+TQLyNux4YLw52IfLvDzX6JBAkuToY3h+4AesaE70lsBBv7yo0WpYokSDoFTmj53Sfx6ZhiG2AIIfp9XL2TUwqfXx1sZUOyMgaX\\\\\\\/tNjLn7C4QG33yvY8hG7bupFSKRQ93+VOQiw7Xh8ZGFrdJ1H77xyWyCbjOntA90NpUB+QghYFzfV3WkeJXmQLIU3VYg2JssoKlba5wXZnLWvUx05aS\\\\\\\/Nj4TKGVGqWgxDY+Qez69tHbLQ9u5aXM3tj5njkTRlFpgx7t+\\\\\\\/Jzgd17WukhI149zekCoURbkgH5qDeMgr0+7DzYjbfhN3gjkZGHraRfrv4IaPcfMaFhToojdO+E+c2uaFprjO5U\\\\\\\/MXye3q7DldjMySPqIDqmMKm7GiNn++G8iNtupVpdNR3Xr\\\\\\\/eiDk0S3grPHv\\\\\\\/cN\\\\\\\/PBaTnYCIg1TS86vyBlR\\\\\\\/OExWE8TX60L9r0o9LJZsSffmLnpYGAM\\\\\\\/DuNLGWJTByd2qK5hpHBTUiwxrkqP4vDwZX+P5cXU9K71srmWTCOsja23LdkoCMJkSdyMwViq70ju0VIKtP49YMzWZ2dFs3VXbpCXbswnDg3WDW6vW9JW\\\\\\\/CJgu8D9sLgxCeuxRAbuR3tXLy6MHtv\\\\\\\/8bwcbKPd5P\\\\\\\/pq9jKrHZw8AX959DVYLCy5T8GrwbAvaTbRRsrw6FnjwoGHq4nNbYNveDllVlIS7rHoNZVjcCf2DiKT0ybhQ\\\\\\\/\\\\\\\/ha\\\\\\\/A9\\\\\\\/Ajt5Q7gpcmpL+BBHbboLrnPwPFpI8gq2T0AbK1QFE3lUWi3\\\\\\\/yW\\\\\\\/hNfsahgitIvbHQ+SqtjCSFOSil2A\\\\\\\/Y2mhWorhn+HiOoAVCQDVbd6e0SHTuL04\\\\\\\/4N+h0FvVK3LIwkzf+koUdqpcz1X9GGJe8j12asbkwtMZoK68xxxpo\\\\\\\/mJK2AQbX9MJ9LsvnbBWslSs837P2cQhTl7jbB8ZPgi5m2qk2hI3xnecQXwd2TaH7Jl2u06wTfum1Eiy+L0IOXsvOJmPUFq0DzX3MnsisK79Y+mC3\\\\\\\/C4Tml680Y0Sb6YBQwttZDwzgT03apBCkUwL3YGUqxTPvU8Wyy4EoLCqiosLGuVwLJ6SO0NltUPYuVPexLNeHXmIpNVXiwx4rVEUD296TG9Zdd8QVPy9oNnRBcJo5G00cygeweLn5U93BD3hCgkoMMkY0pWPqUFZqnGIVtrXKoh+UZU+FRaRgm0u\\\\\\\/E81c+Lq7q+xpIwHm3kfO7JK\\\\\\\/hC07O1Yhr7yaYG699oscq59LuTGSxRXmFspaw2vT920ierskdzi4mQQ2\\\\\\\/YDn611VRoaQFS4Ld+LOigzUq2KpHmBw6EwpnRyqfgT2zC0pw5NggjOi6f9cVIEvEVW\\\\\\\/63gzj3+SthnutRSovKk9I6DOD0dQYjazz\\\\\\\/JG6a23Yh1s78bDeI9NhbIzt+X52kk5PLUupvYXV7UbSX55BIcLFdA9B7apK+lBhWobwhO07q73vp7drUZsok1MfYK1x3XWuZhYsQbEou8uUyxUcRi1cPCwX4bWlvJe3iNqxGePvw2yzh2u9CE8lGc8aNVWvWxq9e\\\\\\\/vVKr4NfhFj7FZBg01LLp1Has0yU4PlwfDlltsrAyhvhTX2QU2qTXjZKz4MGrP2Z7+xbPBMMj8YJn2FGgbshCa4Esgi4+TeBjyYiN4VU865kNzSGX+UjwnYL8FZongRgftoKO53h8DOS9MOhC77JRRpaVVuSTHhKbJOBGmJEvo+kehlfRXeTBxdZhNbO7SfVynosMiCCkx\\\\\\\/gUYJfCHZ1hvbpQzyAQzxnLrAdpmfm0DCKlyYdI7UdZ6xKC8AjJm+YmSKBJcPAfP9MSRv94yxpZfloqJxXbz7Hi5EAqdVQ\\\\\\\/ba8Gbg30QKnND3vsE13D1uSGIiEZox6\\\\\\\/EeVWBIQp+IjQSKIboxHG6rVxbZLTfqGpPAL7VfiAbErz839jN3+v9oQFMMp+C5fmG98OIit5Xi3NiHaDm3bOT8wrV0QSsLBFnrAV6p3D5\\\\\\\/IIpKF5e5\\\\\\\/CiuMiycfRhXsjkTKrzqDArJa0RlekFZdhRjy0y30nVmWGcjq3CN\\\\\\\/ZwaJRdbk3FCUSwXXZVmUstYrjAFgXqehNFV3onGqOizGaq49xYxdv+zK73CLZ76\\\\\\\/wrPLx58PinRhMHPGlkf6vw\\\\\\\/Rhz8uW\\\\\\\/OlS9Xm5+amPxc1MZXriGUSMdiPnNiPMmC1Fb2tcDKZy+FpReTQz4+J7Iksf1YZkTbpmErvSgRFlVcefB4GxB2lpImejZ6XIfAUQj+N1el3jQyrqpgT2+f3fcsmfgfX3CpK1m+r2XENFh8ZJqQzONyOP+ep\\\\\\\/9tOuAHk8MKWY4NMus568j5vz8irSPntIXMCjIEv\\\\\\\/VwG5M9RQ6FrMvLO0osYMoOVBZV+tOfqGOiSeHfMXZqiLW+KZ0Bu01kfpnIiaTB5QObzW7ORuW4yHFKZkUwbIMJt60buLQNeQJD5dGG23qUYfIjWWrxNoKebu0r+iv0\\\\\\\/7\\\\\\\/je3phBDMyx1Au7CSmwRBpIzU6kLPSusMKMq9tt0u74PrZr0tTjbWWg0TdWoVVC66dgEJny3aUJg7pJ7ePgI+59R2dwwq0yynwke6FDCVvV0lgk8gDM3fP9VxItprhRlwuMjzkphUjOeQvhaTDWEpPyng2sv9HxNedBecuKuth2oBZZbUUWwf\\\\\\\/RHkunQ6Vgli41WOn7e0xknYoiEF5dO0yZjFHrcrsLkmKzfaYqxvf6LpyEtk6u1GOuKvTS2f4WYig86iQNL++XZEg0B0ng2G4Fy7lOb6I7ySNqv0KurIZAueCWivDxtAmfM94DcGSVQ7cOYd8BMAZggygCg+e2R6iX+pHmEpnp78WnJ4XmfYOLm9NMHyYlCB9Q7UqL80WfzuJ9mwvgteVFZbw6yCOGSg60rHrptNjSNwPF7ZI+GI5LKlvGAwvynGYV1YqZa5A\\\\\\\/9VIiRwIXj6ENCxgJmBC+J4h0OZOiMVBNVvL2609Y8d4KM\\\\\\\/gby3i\\\\\\\/DCxsvhzBJm1\\\\\\\/yFy\\\\\\\/fea\\\\\\\/MEOfs4DW+dR3uE6wHhtAOhRRRaUAKbhqDCUdielf5vTwegrGOWStIC9htmUQAerIEUo3czKpiAPps3roeG4FEOJolHJPHinp911AH0UlykiJrHDgFQnq0VKsUfJadMZ98yc3TYdnJnTKdEpWXsL4cD2rt0KpTxCPI7\\\\\\\/ZXcApgHfERWadHUkG3X2Mj1FAWmwBeiEw9NNos3DZOs1wPITa\\\\\\\/tZOfhi1+TvLsoJP5K\\\\\\\/KISUsTiseAMfUDEkF\\\\\\\/d1nJzFC8vd0IcuAdPjy9k291zueQZ9sKgdrySCOHH8h7ERqVhhDBiNpLsQbpyORUqKdMTLbtQEIJ+5YWgsgusoicothZxGyuP0gYG2ok45rjsOG+wwzslnW3qHXyvbnWyoag6DoKAfcInfwa8tY1CP7M2HA34JCSQUN8ZciyHu0BL+NEcSxqw2KD6A53RD4dzUPN3e9XFKc8o44pjE1LM7mlAhvVsbPNbqd023HOJnNIrPZhiAI\\\\\\\/Nucg+iy0gJf\\\\\\\/0I5MRDLJLyFRBD3fnrCR9+0kZUkjM2D5dpQDF86P7JACJq2sLLNhl5\\\\\\\/FNaySEopeK1aw\\\\\\\/CHN9lePF64OrLcIFfJRDuRcSL\\\\\\\/7fUeoC9IMUxDPM+8XudHabJw67X2MkOI2a4Inp4xNJvn0gPFv9o\\\\\\\/8RyeQzLiP3b3B7u6rMIqoTR0wjlaEJHkBQMVpMiWNuUo3hi\\\\\\\/0cX1i2EVIqJQpREJ+proB7EtEL1o9Bdnx0SuRCXap359skJGHehgX4sknVwu4iVFnMgD8QEGa8JVl+3ghSG4a1i2KotFt51M0TT67XABJnsNqv75Kbj2\\\\\\\/m0aB56r6Dvu1ZHnAMPEB8uN0RqugKLRJ6fOsWTqIeVDIUAHMmiUfkde5EvmT6yyp2ipxLPdXNp6YLXzStGW\\\\\\\/EjJcAYE4GTw\\\\\\\/1Oeqy8\\\\\\\/qmE2mydaDsAgyCg2RJA4mToVo2+pRa5fG+RUR97WvEO1cufVHDvIymIa78K\\\\\\\/8FiBrqakgKYg47GAjPN1gdwLImN\\\\\\\/FjTJ66j2x\\\\\\\/8DbimI91TyXNgBczl5ZPwDKCc0j7y+bTjXTj1rmCl+r3WrjNpXxw2i6qpyxDxY+YvCAWKvxb\\\\\\\/4Uo+vWV9ZwIqfR+Ice4A72h8ChX76rw5Tx9Tmqy9to0Tv0FvQtAwiiZJio7Oj2iPlXKWlOsjpq\\\\\\\/PHidjsmAcGRN2Ml3IpvJNk3dJsYmbTkecV33EWQDgB6fUYtx6jYKtu59lgDmaKbtcnb8kujU1jQuXPHlhZd5iwFEIxuF1\\\\\\\/Ccq\\\\\\\/r77G+ZpdxxInJBE\\\\\\\/Y1X8Vhnpwew\\\\\\\/QgF8Obc4lu1F+tvsKaGULWegdlA0neMsJACtf6kjycYgKVwALJNvIo45ecRE0y84ct1qTRrbo83AgmRLOQnemeP4iCws4Gznr7b1mVjjUiHHxSQ0Gvo7f46imjgNlu8zlqKjy\\\\\\\/hsEK18Y7tXmaxS676Z8r2qmt1vSBnQ1zSGjSNOizas1X8hQCH0OaD6dhYCbx4wmzd1AxWwnMznCUHkcOCJmfOds1XQo3GuC3umN+O4WL9\\\\\\\/et3VY8EkWgSt7czLtyMhMj11TbwVYDWArD4GFp\\\\\\\/dLbarG6eJpbRu7pR\\\\\\\/xG3gqiC80J77Qu2H66uWbhCwxOyjxud0vFXkqMTdSy+2HngfjjmDPin0j5HogkQRr7F+bAs\\\\\\\/OlpgslN8NwiqwoyFYNCMUG9AyslHwbogYXy0Awk8Rz0upISZFmtzTGp78Ehrd4XiPKB8p0b7+iwPrzreZeZQCEjkTUubSmul+nWN4NsCnU+P3y2clAxWIxgvxmCyBgr+74g88hO\\\\\\\/48Yfk3yQ137zHdT7JSwjQowfn4ivEi5BdGtYsdxNKv4PHn4FJZF4cZlFe1F7loztwzmG6UE5kVxEEvGIYB0q2Lz4aZI4LXyxnbLM1Gdl2A5uqzH00+u1ZlxDeZ0YK6gCMef2t\\\\\\\/5Wf7aAEpEmAsvVwJEbrZ6754Md+5jXuRfhxEjRHuNi5jqKrtVRVjYvcfe0lmy+l6dBRolqnJ5mYwqtGJc\\\\\\\/Jvgpl5X+lrT7xbVxXIshLxeqcT7oFIUNnrR7zQ9OCu2ndRSvAXnlOyeYJwgg\\\\\\\/1V0aBZaW0VuZUfjNHSHSFbHyRYwRC31TlYSuM0YGiWuYh2b\\\\\\\/9J5CoNZBvEkd9+jOSeONLeUaK07JZIBEcmwQPExBsJZczfjzifNRO\\\\\\\/M7+emj5ep89k+KU8tx2gSg\\\\\\\/LP31YgZ4JAeFIDx8ieu8uaxoftNlOI6woAqbHkyqcRzqW7KXxpx7eNbjf+jW9UGb9Lz5YvfsFYOl7JupO6IcsWPdkSnihsOiJeEgXq0N\\\\\\\/6W7aLnFOgBuU1F7nVrD47dEpwukbQFYodihdKepQje9WY5zEJSjuLA2YVk77\\\\\\\/bPvZLkwaAonV42AHr48wO8VHBD8zi5\\\\\\\/v\\\\\\\/OHYzGtaRNtzsb+rt5P39PRWdufk3mkElWuvUBXMjM\\\\\\\/RdqRLLMn60k1SUxh3IrE13PRqdopUkavz7N6xmY+smHtEuoQn8VMbZ8qPrfXRNuGciNq0DFP9d6tp0jB8HcCQYkkHa0eQQOWn3kFFdHVmPUq7dHoBl+fSDohR24zB\\\\\\\/fNBkHAgvjw6T8T3jKc6qifT4PLFj9Q2FvdqrI3H0P+X6frlP\\\\\\\/nB\\\\\\\/i18d8E9zpSqr+hhPm6hxeUzNxHeCOwAGyv8st1DA2FGTqmsKJGhPrDeKZef7Ys2MykoKntTav218jgurUa97G0z1h4+zIQ+2XS+tIGHhjpSBhUi0kbRf35TXJXy2VOdT31\\\\\\\/OrIOs6Tskm1gCLBoQsluX\\\\\\\/oy2fcDrvViCB9L0upe4HUBRaN1VREQC6KnNBMWtkdi5knJ1UVqoVYEdGV6uUCxnQcyuPNrYQcx2S+YsYwmO+NuhaQeKqM4KYYc6xJAuYvXRGqKWDgvTeo5L1fXCkWu1yTE8RbyJXgIhAaojLXMt\\\\\\\/ZVFdJySiL0OcdZKKw1Wu5e0OH4e3l28FcUTzDwoBmy3wDY3rK4OGXZc4FKVLdpqUyjRle0XQ02sRSBCsSzFqYXN4Tr4PnMpL8bcCL7gzxJnjrTuqFa9I27TrDrBwF3Mtl5tNKHm+FVK2mchlHNXiZrgVY4cnWvnMryNkJ0HO2UAUP+n+Yh4nwTIJlqjXqOtTiom2OhRysGSgov5n\\\\\\\/avYkx3EjaBh4Xf2JgHubxenrOauwF8Kq1WeR7fB1D8skJIqm5CY3OBumeYMTob6KDBNQmrH+EqVlWKyWfjEgNWHKbSM0HI3nGOVYE6K6wy2RoagCW9ip8uOBESfTtmDZt2sQEIUFtg2DhWK+xvJjdPZz9YKaTQcPlecJLeQqEzY7wOKZQkVg2vgN2kGaTeENUjKvf+ZU49HX68DkwV+GpKBDdVwa+\\\\\\\/UXNi7kSlsM6gOLPoTuGnaASpa+aIhg5UmnMtLh6RCvzglbW4JcziqHl19mribxQ\\\\\\\/\\\\\\\/o7X2erlcLOBSsYxJ7rEhr0M8ZYn39cLPFb8+F8AJBl8OnPmC2BKf0L+uoS28mys5AkElbxWcS1vRhUbGLn91QpqsHmnGKE0s5zWiQ4LXMzk0v4E6e\\\\\\\/LosF7zaO5oDDzqnfQNtAJGSYp5HWCQaUo+BypkNJuIVkG50EBMgCB15BJr19kN2US23BPj1hP1fl9ZcZSND1\\\\\\\/qLHIuofuFgX2F0Cu3IVYawpSht0bHr\\\\\\\/c\\\\\\\/J+Vrl5fzW0Dqg8eBdNFXOsPzYe9Wo\\\\\\\/hLfT5sxgJR\\\\\\\/w+osnzEbHcwq2eEaZxTdnYEaJ\\\\\\\/sV+U\\\\\\\/YkUm1BB1uF+\\\\\\\/cYNSanLwJgJ59FZzINfY16eQ7KGcIVyFAUUHdyhAigt\\\\\\\/s2B\\\\\\\/21XF8Et6vetOXQF44TzOaoLf1mlVbkbpNG4wmmoU1p5FoInC\\\\\\\/ekucEzPLxZJ6v21dsPNdi89KDpWq9iybULfu\\\\\\\/lrdHp9fZph9YstpaJl3LAsiYF3LQOo1XD5xDA3TdE4tEssRMdmE9kMeWsTdmYKZjjNyESQhBs8JDivCfrUGfK0+flkJZGokqvj3kNvCEgOv7obHgSBjiwGwaMRRsk\\\\\\\/2KsR+dGPMSzSNE3jRuGbQXojNdJkQairjmcPR61YhfXa4OvIS4GCaFmp3DALXA6OB7vwIM+oCct4sdjod99oHSiy77KdOJo4VAzOcQxAkdOUDsgmv1qSKIs+cMn8cD2mQii5tgs8AEYJDa3sXjaZ+vQ+sqW1YJ0T3ZJVIzbs5++iXGOMrzn1azpWxF1yq5\\\\\\\/i2kA9os\\\\\\\/3eL3pqLRFCuo45l5uV\\\\\\\/EqfIIv5el1f2DeGUzND+eMqR+iKuCDMacM\\\\\\\/825prANy0pW3t0tiyLffaGhlRNDMKAkxBaLYyyTKzNu41t59MrCcrU6CCi3drvp8GTiqs9xuTGiZzFUIkmH+4Q9379vXZFG37YCiorGf3FysOYpKe02kaBuovYKPiZuwAK2ZR8hlnVg2\\\\\\\/5ImiSPke4pkfbf9C7+TmjnMGakop9azDLyPuT4syYm6ehE+5zd93jXbxkaNHFBI+njOz8Vq2g4HlVtg8J8UWZ5wKg5ouVTJ0RXa0zgxxIsCORtVpKMW0YxlW8ojlEHz6PJRYtbU+YmTDlV7nao\\\\\\\/J+qOuGqXMSgdCw6F9b6aNFa\\\\\\\/zz9i7XQ0WEQ6Wr1x\\\\\\\/DZY\\\\\\\/taDz98Bsi82rgheayhqVSrWnJUECy5+nVgj7rcHYBCYcxcSacy6oxbxa5Bjb3KoqoxMUnEFbK7NOsr5y72DmhwOIrM31ykjBaI+yhqkfk5KeasgGHddW5YQme8FYOhC3zdaifLjGMEti3dauUJ+4MbqvQq5gd8GnqfSYGYwF7Q9mT+xfU9ovOPSrXQwB42StazkEJ+i6dTos0s2mBFk5DdFxBzh+M\\\\\\\/h0DnQEb0nNiIYDCU6+rySDZhZHZfv5pqUw\\\\\\\/9\\\\\\\/shHxRX19cObzbKdhDsSwhIw2sLIgNqx34Gu3spuKNFsivRw6\\\\\\\/xTLI\\\\\\\/ISMdJBibuIAFj290PRE\\\\\\\/BD8\\\\\\\/0HXyVLBjUvcCMiwNFj65JUS4N\\\\\\\/eZCBdeIm\\\\\\\/qr4wBSfLo8BLx3YCQOiMPD0SqcH3EE6bSHc4PLb+OydE31MHbTxzLqU5Qpdqfsitkk+4ymMBdZVJ1Ppn+1Xm+ivOZcHXbVqrhfGk7b8tI7Oaq43D37OiYcO925xbQDB2oylHr+wzXXgq\\\\\\\/0GObD4FRsyAf0fb5BJlxNop\\\\\\\/m7VM\\\\\\\/kVv0YX9tyawGBCBvi\\\\\\\/GeqFDa36XBMFjt0Rajc9z6A1j\\\\\\\/UJ+mxJ3j4O8dDpOKSUNLhpq0wc7EjVRjColu3jMzFUmQe3o7Anh7ij5WRF+fAprySoF6tkhMfA8mGZYFdlq9TPcj5TOqKzQ7MHjWmI5m3VJIk+mb3wrt1VdIWUfGBrYKR2+ZOE7FLWBjrT9+eRxN9NQfLNEbLN\\\\\\\/KVv3d27+K8fuc5FQmTCBT93nKwI05RvyspFwGIvM2fQtlVlJhuN7hgkmRqfUtae5a1fuxL1Z9rFXjl+D8cYhzTh1KN7EglF1Ul6Pkr\\\\\\\/D1hAZH5ZvP+YrbQkPwx4Tlyr8AjjTgUL7tpvXfLVA1s3vwxPgNc8eFXWPNDH55kwIM8kkaJKSHylWVU2Qc7kA1YdhOn9JBjCfqPZOQlxNGfFDLD1qRsILEy2ZOOEghXlmXftkJRU5ijGiBuLosCi9k5AP\\\\\\\/cK7wvLVkLU3\\\\\\\/4SjhVhIunL20lRAkGcja2c+nKJUl55L\\\\\\\/km7wODXnC+3hrpgWhiIBFLdhZQY1CALyEjW5ntijwkaCM6xbu8dnt9\\\\\\\/bbXAKpuvykdi5\\\\\\\/8vqoH2EgQt1SF2WlKtyk9RUe5Qol4GMhoxwQGnM3Tr1oCtjtdNSI7IRMqoQuzRm7ygvyGqidjzfwYRV3oCVW\\\\\\\/GAmVsAuCWS9XGs6BUSQ1cYIvkReVBXttTAdcHxdL0ZTjN13pbkU\\\\\\\/7tsSGUsyJk8zBFtql3wKGkApmXiiEzvYSHHqHfxC9grzfCzgrE3az5piuQWLAG4zMebVOhI7hXNKRqK1DfvxbliWGC6i3O4C9ig4njLWk4T6Ee\\\\\\\/RARlKceUeTgTQ2E5R16v1fRv39GKaDsI9tJ9WOKfvhhhYud+8vg\\\\\\\/US1\\\\\\\/BP6APTtUc5SFTJY1UDQ6A2YYcXoWx2VxWU3CZHT1a1lLtjUIzd\\\\\\\/y3h4O3QMUFbwVG643ujKSuH\\\\\\\/Sx7TkIzXv3of9yAowq5qLJ7aFy91zEJHSWuqE5Dis6OK2l\\\\\\\/RZrx\\\\\\\/QbseGta9s4sa4kHzq83cgRhzTC14UEcQjxlcvpCikpiYYxhWBEBitJZLkZz7cVlEnP+eUEZ5y2xLmyoPL4wj2ls0mPhG3wWZg0oJwrpmXynnyWHgoQkxUk6oxSXTcZfy3wKDROquhIjAYCNgiF06MWdUgiw3X55Sn8H2o6nglkZVsFBFUYNJumG0gC8io9AwK\\\\\\\/+4UHH83vrOkPl+JpbgS8XdhG4frZwkp4MkgLSl6RpB717ZGsAxrv4lbJR1pISW9U62Sy1XdYqnCk5iLdeNSO5yexzH4zykZRPAERHch26BNnhaCL6JUPOTYinKKe7pHnku\\\\\\\/+KPhWgN84frXYy4jg1VYfIAuqHpj2Lt1oVOFbQHy+1z36a73Bge4BAXz70QnbbCM8BSfJY\\\\\\\/V7N7SpE+07MawI+zaAoyqy96kt+LBMz6y8JONDu5AtYHlko7qaFyeAvxUUNRwpNfF6xEWl9Ec9B1wBHAvaLY\\\\\\\/uPUMEg9vXvN\\\\\\\/3ox3F2asB\\\\\\\/XsXs8NR8986Z0cDdW32o8cnxlLTilPkp2mPMBXW+axJ0iGdw2HA+Wch0BpKydsT8dJL9mKoT6pyPjzYA15tVfR6JPsKF1t+u1o\\\\\\\/Gvm\\\\\\\/xTH2Y2Fexs\\\\\\\/OQvuNftThXkFFOMEkIpc+pSLg1EJLpWzpEersSrYS4NfAztsPOhD8Qk5htMmumaU+ASMKPrO11KOQQZZ7tZGWMhtdHMTBQ+mql3BJELMVtYXIXx2Dao7PMA6cHN8S+77qvJjZqO2jNuCts+ekdVxKUx+4l7E\\\\\\\/lzFH91DGpFtJLh5x0gQjmDcJpFoPLmOS9iCdb\\\\\\\/xCJlZpmKs9Nhs4h7LapKhYDsc51qdTL+2ozqyrs0kU0CoCjVAwF1Hvb80cfymCfke87LoS3Oz02th5sK8cnmORfpsT6FIcu+dcpenqn4LRB9GfYWNeaLgx0xoxIQxnj+Xj+berf5GGRno2Qw8pcSh1ceoO2dzyW4DgStbSoOwyks7k3dtB+cpjTOqZww2WvKD\\\\\\\/LkEsR4q4c1UVbxuLbnwV9ocUSKYkrNttcszQeFN25qseGos2+Wtx9BGT+gqoURbQybTEvC0sXkjPvNJsWiO10krzfcnkHYRknArgl\\\\\\\/5WP3\\\\\\\/253kIqpdjOpmVgHvyxOnq28WrV2J39\\\\\\\/M0vg1GzUEFfJOwxPS46Txgiopf4H05axRIq7WlARG5zsJbCVCfYT+YJIdgNgEjMjvv\\\\\\\/xltpiSNdqDhWCTL861dUSsNi71+sjJetR1lmFvHolPB9nC\\\\\\\/pthB2ZbPXdIhJSYdmIG3pdBQqg7qAQIg5YZApkXfsGX3knSN4mcTpIjakuzimHdDby5u3IzWgdT4jJsdowLZAXicE41j\\\\\\\/ciKWDJMs13z6Ss8nQXczXnWDEUoTx2sQ9rNVK8cDgdNilIBvs5R9I8B8ZeGjl9AqmBA65iwUokGWUqQiCzF8dgfgU29kbhUiVFyghsY9tDzJTvXxGD9D8fH9Rnk0K3HyIB8Vm7AfHNC8moILLRrG981akU24zBvWBePxK7qG59iywf7GiktzMq+S+kmVMS3U\\\\\\\/644hCOnitfL7xiYieZA4TYhENKkhs\\\\\\\/UF8tWAYSNuzTyWQJNVEiiTowJmSkT7\\\\\\\/fOCm0PL9sL3IcF\\\\\\\/up9LzojCpzw9bMo5wRXhBEj1KsNOPTSnoYBq+fWpwyHDW74UbkhI7FLlgGdTe7nRRrnpZ\\\\\\\/5ngspg+29XjV430nt4LTtc5c0i6IjP5I+iFY21YcCYyOEDlsewqhs0B5oEYRZyLoJCeNAuRUH54RpBPRM+iWskLsCkUdygWv\\\\\\\/1hakN2pnGgk8KKe7iT6wLNMlmitHH8npxmM0ybHwc5+DpSNSoFCafCPq9zzcL4IHVo81MdyopA2Ruh7lao4UeJzBkEgrHTj1YV4SUQMxky1Ro0JWK\\\\\\\/Rn9FUyaPV8Qki\\\\\\\/O4ObykPy27IUDLuiNFo8glvmCeOGL92RvN9VJqnhlyoxiDZPylRterL5oElBWv21XhI3wUJrt2fVAxnsShUtfBaOjzHZnJSWHF3b2Q2l\\\\\\\/3VZfIWJ+q2bpep7VjSBv+Oe++AkOBaodnEVHP\\\\\\\/YM5BBViXxu1qcPmAsHHtsfqMPnO1fmQl2jP3z9xFavZ5avTZvsPO1uE8E6fjmocsHLLG774YMu+WqAC\\\\\\\/kXX09XkQlHmX8RZTXscW9wJjOkR5O+drxGUXaxE9r9of6aBxWWG5BcR4vnlmgEcxCHM7mmjLiBoDuW3k1SLD03Fs5AOiolIElUUrfipxv8W8ccyBhVs\\\\\\\/2cYW0RnniarQoMvjOIz3iYm1tDRnzmX\\\\\\\/l6JM9qj57n8DDMUk5v\\\\\\\/GZic5lcQRHzbsm4BKHrQU5j+WT6X20wWyUn4w7gHld\\\\\\\/ltK1+S8DlqDdkSGFyiBjjKrDNhdgI9jsozVfyX+A8iyrYwN4ZhPUHlEADyT5nBzRqej46Dz5oYkLFWE7xC2M\\\\\\\/KzgG6b8lKL\\\\\\\/GAQlSSC2ue+prNz8bn4r5ydYufYHm71DcQ5PTjnig++HNKWU8hAzLGbsZBXHLsp98mkhoe8l8ycU34Wzwftkf3ToaQkkBFYTDQeU25bgTRucuaxDa69jPeZ6aUZV9kpYPltPJ5bvunAvlSR3kuREZ1AC5Xv0y3jGAhUA3HFaEvJsP+RvNNnGUK9NCHUD01Z+mFjguD29Om6C1Cc1UN\\\\\\\/+5zsj4rRjhF44qkJ6x3K4m3icLIKk1makiQ9kYT7K7VKtFnSL3pNOtrZVrz3XJIeyCCRAXTA2E5vmQjhiYL4NSU+Kf+oOU4NWlq5RYCpN5\\\\\\\/jrNXh3JOirl+4uOdROrde\\\\\\\/LGMPET4PIP0mVzWws8LvMjeWkY6m9BCbMeBerLTR\\\\\\\/GCyOUzUPJw\\\\\\\/9aVaBP\\\\\\\/xjJSS19qwPd43SxdVkTsNLkFd8oyUo0Nsu\\\\\\\/PVdbG1gecewH12rac5RxJFTKdnnotQ1ieKo13UaBw66IeKSnBRLi521V76D5l9ZhEomYU3Sy+GvnkUOtBQ+uk73r5rVf14dewddx3HDH6xj4u5Cd9GK4a1HPGMF5T0MJKqTtnb3ByzzrLo3iOefcAIBs1jfBCuwGmcmkUVRv80aGoRvS9mmtxYQ+ucUUI3Eh75PiYaD4j\\\\\\\/OU89Q19dYMSuD16xbzMihRLw5nb0bBXKQzALwrmxuC3QJSVkEcKTAkjmKp\\\\\\\/K29OdDnKvYifT2bhor8ImO3iKTVhQGj0UGe8\\\\\\\/X+rIoyHIb4A3Z3qAgh7stXAqgt+EAxvDFeYeFcBMJTZ\\\\\\\/d+XkMmZjcP+U2qHxHXMZKkJIwivonKP5uvDMNQdRe95AeiwJpTYCIaIUxRTVIMi2QMf3y03F8avaCJ6698sVDhtDqHRI46SFDAGwzrACJj7MbPurAsN2cZgYaTt1gjbNFgD3i3yDsyOS3EF4gsBrrrVX8kWWkSt8mVO9qgfnjzWB4Kmb2Ee0xwvaaIv2s9XPcETmo3uebqTWXlG1BtIXzq0FdGl6vXlwf+nNh0+Vl3orkQzHA44VHrrtP8MqWSgvPZubRrIoHT6aTRKfO70QZoBAwlg2j6T7vBS86zYqOtuTcvw3hvDcPHMdJLy66krq6PlbMWTduV\\\\\\\/EUAhl4fvSzJhLFrl7T+E291KbaTBZPeOyfr0tfItxlntYtGmgTnN+0MdEpsb534hWln4yzMvks1ghGfKggHi2Ynh9MC2iBC69oiPJ1EbktRIWmFCPzsQYNjWWvQJtnCPZCL8c66e2M+UsXYVncmhVXFyYQR2lWzEP0ny1gtBzjObTbLhOE\\\\\\\/ThrcZwiNdDYCLt\\\\\\\/xR2zk7U5whfjhjSh68V500Mg8NOku\\\\\\\/eJFEmfckHX9QFMXpoPaYXGkexA3d\\\\\\\/LNkD0FWlbLuNXSCzsWICY8Lk192Tv8\\\\\\\/sEZnJgZHihCn6hSceFA\\\\\\\/PMBzd\\\\\\\/+bcrGw1pzviudl+D9LNYNK5lC260CYfNQ5k3uy\\\\\\\/cZz\\\\\\\/aldOmz+ff5y7AhwTJH\\\\\\\/ICZzFJjyf0djrc9vXE+MlOcDCT9x7CRSf0lIh0kcyzZMozL+U5yH9xvHinwXIiR2vTRFbI6W\\\\\\\/BKkqIIj6Vu6JDi+H8xiJ32Yk+dj9RZHQlhObPJVFhc7zjDerE2KejQe2K5CqPAD7qy0ap1fWgmypMJQzIVffW7u0jHjeptYAlyO\\\\\\\/Dz+6dvMgoXdGnKe3oTMjRXT90ur\\\\\\\/rcIWNCzqwFHHX6iHlq9OTGWdrhvgjomUwMq\\\\\\\/+CWsI1zBeWWL+BOscF6VfKK5oGcs2C0N90VjILW35wL82eohJozieGUkV9OlH+MBIvLzj8IvO463VHthNeqkFHVLiVd5IZDS+7OnhCIVUF9Lej4iCOcFC55QUIMziUDwR4YR4b0B+joZ3zTnzc59N5k3G04Z0GiGxjAh\\\\\\\/F8+HuwNWJNQb96IQ0Pf8u+p8eyfVbF2vsLKZr9\\\\\\\/lI+2ihcP5\\\\\\\/xV8GoVWfmz7i5yjwPA++mi3LB09kyE+VEdGqA3Jj0f1HxwiPUkkDJiAG5px1Mbd8KcyngkKB730AktrAokw8mC1Xr0OdOJhMqM4D0t0pEi1ePKc4DORfpM6JGgtqQa3O\\\\\\\/4le1kX9RNysTZaZaQOO5ebQxu\\\\\\\/A5y4AfHDvO0d0S+48Souw3gJ8ziHC42CLKwmK0UzHh4AnCmXLMvoQMQUsUsZ9HtV50HYTlclko7X6aHWrWYrobHmqa3P2v8Aq866OdgtsSc691eZwsJWmwGSgv3Keg8nzG+UaWGbzhA1aTseez9rkxscOJuy+Z9GmcvcBvvFwj6LBadH2OiOnpBLD+NRpgNoh1KzseGf\\\\\\\/LPKuw+rCRf2InuyQqfkyDoZjQogU7MK4CyQifUkEcQQvJ\\\\\\\/w6Xipof3gXQ0aP\\\\\\\/tfToQAILMuFi25bpv8jKnTBnS7pHk9Hw+7OdIW0dvpW+dqbZcS0ijGJbp7iN8BNlyBS0kPbTj0N5AjenSDpOWibjyfc3QmZ0XJpn8UP6vUCGPBlVh0DnZs4BYzVTkdj4MzXZrfi+MNPng225wfVRlYagxyU0dw6C3IO1zQm88X4OfM0SY8pcz87BqgJz1tvWA2oqv2jV5tLlWV5echi1IWsr0wcWhifc9ave6iSpHROTK\\\\\\\/q3Aq638YHXX7B5ICS\\\\\\\/\\\\\\\/ciHtbRfMJV14N5wi\\\\\\\/3OwhC7QYczn1m653hdXuFWFZJ+dMKSnXt+ZmKVem8w2pUKXF\\\\\\\/vZXJ8zAsvqCPvmggiDM68SFhZoBpzON4+eQp6m+MJESSy\\\\\\\/QkNJFHnoAXJi4rxIQJsGgtcpT4uoUMbY6gPDR8jBAmN5a9s3+Wd5r1zuowth0dwFsTDfmLcvs\\\\\\\/M23huqGu\\\\\\\/+cURVOZ6Z27hm75WARScxeh2sVMWHyaTMzpCQFzGwG34ezufTg2AGps5h\\\\\\\/sxMeFpgEcoFwboZdcFsRHXOdoWPUSPdyfnkgJIOwKZc0lrkR2HGm02VRY7gQWsnVaW\\\\\\\/JApgoo8mvyeydFLqqxJNloN5FY5DhNI4U3DfdCIXXrEmxeE\\\\\\\/HURz7zXh+T8YXj6Rpq1jmqMzx6Ng2yr0yWyIsdVIgETSw1YeOjVbHoFQAChdTl1Lv+9eEweJQkeKzbzrYRx9ky1lWY+PZvij2Ga97VP5ydwI9T3YiKmSRqS7lTsAo4vI1OZXWX\\\\\\\/ebANXbVTqYBzHlVamQlMVlG0HbbxE+Orwrm2si+Vju6ZVZgNNKjdES4WKfiFwvCORY3EQvw\\\\\\\/mRBqq7ZceOIdz4mMocnGr6c3OcutsIRHd0kY9AR4a0kVpd13yXxLJ\\\\\\\/jvRaq2JgOkwqWid4cXyrPI2tY+EZiuwfYZeWPFruHJaWvY6x0BGVxcxFiQK+2hYGjg+QrmPm57\\\\\\\/\\\\\\\/XWzMV+Kg5OEv7\\\\\\\/RXITE9w5kRRQiA\\\\\\\/n5rAq4fd97weEgz5tbsnVm8RLpmbt0n4jbt09XjfUWx8Gf22q65HK56zQtHRAOnPh1USFPWDBf66zGx3RorGgvbraIEjw2AphufzE8rR0i0X30p8Qr64SBjfbGQUZ46EJiec0ZoBfLEF5EYjAWMXZDaRZgQpgMys7WlWgqyC1ig4vTFFV1fd2dM7zney0KvSJOsQueU2VC2\\\\\\\/IKMwgLAiWqylzSnSMNj1zmyadtBIU4JrD0\\\\\\\/thIkzujnzoJl3ipiddAarchqqIhZhdZvXWtp47aX8evc6KNU1t4eXBuMT4gtdHmR2eqRVb81KSbKNvKwWQdw1bVdAbNKJQnDGWFGDrsZ+BOT7B2QButj\\\\\\\/zJ5xS+qZtTEcN87wz9dpL3cmEiFdLumKmuJslFnpdgVAOxk6Esljin7\\\\\\\/sxRvxBhge+kds463ak2O2xxjTVgbD6sIqtf6FWKH3UqpcwGpjXlBjXMl6K80CgOa7OOTfSXFVtefKdzggqUMM1pYNhN8\\\\\\\/uLjk7yvQ\\\\\\\/AyulbIXXHrMTAzv5dee4QNbQqqJHUVUkrBkGR04WYkMo7NLdGpSjfuvE0S+Pj4HchVQJQXIsbvODggkCR12\\\\\\\/D0ZcQO9HGi3f63jWM1Q+lrHFiYNSxVlzpRZVJJzhYbgBbH4Ct9VKhT9e7VrdqGl8ItcUQUXLCwV6JvIKOBsVeNRirFmfArg5\\\\\\\/6lhcXY8xu54PbLYMasJi+NreZEkbrxqw99tLZsW278GVsgEMVSeTkwVq\\\\\\\/egpRq3OmdJFTaX6xEjEvwda1YYASlCiTXKMzBj6MKNqtQ9mDWzNvbGkftQKezj5JyvZ2vYsiQ4Fq+0AeZ9Lcy7GQhhBqJoSYbyFEeJ\\\\\\\/\\\\\\\/TfYYv6HtFMDIIVyfJpbiO5GsH68hXWQj+14asQTxP69MoZn3nNNjcnEpsYgov8Q+GdvAZaUwtm3X5IfBO8fuBpUQLumKqFH+YlPO2QN+yYVPvfHvS50+oWVxUF9hQSqKJoeT+4ENrV9ZImS0F3Xt8YdI0Ir66wo4wIXhoxY0R\\\\\\\/f1GYocJMIL5LB2lajJTE0waE53irx5c4rwh6WHWYI2bYgqnDfdkt3Fce3l0XcrKAYK62UhMOuR0T7Z+GpT55U69d5zehbm7o3kBWhZLr5mPdklHN+WOelfUt+WKjRy1Gczw7PtgV3ZT99huoJbPY109I7\\\\\\\/EBdjpNUaDv0lQUkdDFFeJGWyH45RmPYwQNrs6PG19pwh7lZDquHR3Kx1I4dltQHufau4nNsoVeENu+NjxMb9LcnKyDzZWkGoBPv828PtyC7FGd6mN+JRj1g0bI7IjJjOLMC3\\\\\\\/nbNx0MJ0fP04XedWf20\\\\\\\/RHpKk1ShfG8A+JLgxMy1fbzPI\\\\\\\/Z5F5UkO58RhUcCcnoGovQZRtlr\\\\\\\/dBkJoVEJeJaS3lywGJOYKGYVCN4AK5aITTsJvZozFWY0js6rRfB4yO0NelG0fkUb74Ou\\\\\\\/tUS9coPa00y1WlIKy8kjf06Lg50o4wLCD6ola51F8fsV940lzO04F846hYg4+CBnb7Q6NwzK0QP0sx8z9IJq6Lctsy3wCOrO1yMXgs6RJIpolVhOnPrz+eebaI5RYNJFV+KOWKjxVCuIqUAeQW4mOPHwZNnda7kzx6jkdTaSX7nlJ+7L+2mPV4ge27WQb2OMHWT8tr1pWEvT+QmA14wmDGVqP6dtK32Y7WtXxo6diHY8j+byxjl7ArknqyhMbZ6XubyRH7CAe9XGghj\\\\\\\/Ii\\\\\\\/RHdoFDbFvGFFSOHnRN6J+acHnEDtDFXODSxhRHa5RTKuYCyG\\\\\\\/RAU43H+k7ISX8k6AaoMQVAHFXEAvvVAHU2j1KR6eXBEOmSMTfjGnuTEYRrkeX2gms2+w33i\\\\\\\/SWJGTPi5NUAcrtXScsMGqI4Vl7FGYvExb7+eKxOvsRid+OROBLqr8bchlI3Afignswy9XNG18l9xKJFToZ0bpV665dRi5jeyZBUX10y\\\\\\\/y0eoVF95tU23sG4kQjCiuv+d6sAM8wwp6YeA1iG6FxsG8HXrg5fnkFQA9sDluzwTzyYVyXOyB479lmjG4CvGW14iRQtnrIBulrLR+a1NLeg+qfbNSPOEOLhbVVd1Dimn3Wpq7W7mWRWVVa4+TZr3+dmJe0SMyAXICcsCwpBtLTWM9WqcLXCj8WTN2GrGEYuawL4coduy7g\\\\\\\/yDzEWDUAyMWVfl1BlXjSSuqOT3\\\\\\\/jfEQVUopceXwnwlifYPnznHsXu+t27fqVLv1iTfxRtxdWpTML91bZXynjGTzmVWwokvCvaAOSCVxpRR\\\\\\\/D4ErOlYzUKOImur39ocCQd5UjrurJMjA6mAp6qVsJRyCAbTWjI20FJEPwaZ0Z6OrxpDoAXJjIZs+f6YFOKtdrjklHCViOkEe9dSmUhAJOh+7xJQLYqYUbMFaXUfnTF0lvBBsAR4g70q8AzsKO\\\\\\\/0Jmb7ZMpNiYFzEWk1sBDmd9aL0jpim0CHrWYtxFOZca+5OKzCqz31oh6RRvytlSaCmiCOhMNtL17Wg27JKNa7JBjoKLiAgp9gyGNOTR4CauMcWd8IHkNPjAm+PACO720ZfJtRQ5XYSdSxQN52XcxorrGbD+QYTn9lKtS0TBzBRzodme2rxBMUIMLRXG6OPXU8S6QVlX7BJUtfRepdu7ZTKl2jb7KdxXo5hmvIc4mufw6SAS48LKbqmZTapEdcyZsT4+RJNKEqq3Qwab6fAI1UFcqAu4yOtoMWAEzgHg0n0QVBLyq5jk24kjE6Tk5XLmidkZ3hbVZ+qkyhT2EjREuySSr6pUiWWIi65D3YxISkZ58NwIrD\\\\\\\/u9s\\\\\\\/NSVcSEBdZlyz\\\\\\\/j0J2vcdLQOR3\\\\\\\/U8cyYOOo41fiaMfFEDqLMdeRgzdvDjnry0uxxhQhPvGbN6zbOju0+UUh7IwBe141V3lnrv+P1gHhcdw0X+WtXNAhdIOMkxekgUgFUzooQ\\\\\\\/H3vqV5NYTI1DrTym5SdGIsJsky\\\\\\\/hlkWdLLQOU\\\\\\\/J2IhX7AWH76F0SyMrs4\\\\\\\/63H9i6g9pWyM\\\\\\\/Rk4hnsDhXCluMgKGvpWoNizuTqaD5yhO5+pzVyszD6vc\\\\\\\/l5SOstbeXIu\\\\\\\/UzDM2mo1MMh9cm4hXohpfRnU3gvzC+mGSQAKGurzPsMJduGEV+EYdXaEnAQ7vroSePXkMK2HF9KGenRjChMODKIvSTJx9L2NT1pSPz\\\\\\\/iv4FiP9mEFdQ509XVLF\\\\\\\/ER283lTTNsmvlOFh1ycpCX\\\\\\\/a1ex6uII\\\\\\\/k2NMIzqIlm58yZlkY6DEUufSdFIGxNU9OLDsgEAQPI6gOxa2m1M+g1o8K671Bp19lLzoHMsJa3gGjbG5eiCX+59Nk\\\\\\\/1j6ZfcB5fVgmSa0+bCAQP3XQfj2brl82E\\\\\\\/TXDyWLyj+hSaeg5HbcU4HD9SCZ08XRum+33gv+aN9uUOKJA5ayCzSYeOGRTJxO1Q6MxloGbM1vJ60MnXtdK1dYZopVJwxfzJiYi1w8hRWCKJIJaTK0iLijbd5dsxNtS\\\\\\\/Ulaz7Ulk\\\\\\\/TTUf+59zTsOnhR+RTVUq251QgGBolE9Ro4y6HOjK32VEgmoExhAF5O1dha\\\\\\\/dBptLt0fFytE9FGuj82CBrEl54xUqNrDZ2HCLyhLwEMuDVmf2dLlPiOk3NHe1eFhVV8XO8bgJ6qSma2B+OCibhuxbx8pMHUf28Z\\\\\\\/bTO5TRJv5pUjwTd9b\\\\\\\/z+9nRFCYmd8Unvbwe4KApYHb1JvcLe7ke3Q87att6WIih6DSwrvbFDLK9R9eABdfHyG3reNgQJIiHjtZMdroI\\\\\\\/M1yE59OkCTw7Ii1MIqsooFaf1CumIsGKpiYKU1kw\\\\\\\/9\\\\\\\/BVrA7gonua2bkZqRnRGfMKL\\\\\\\/Z2LtY\\\\\\\/+m21BGVuMlYek2ooWrmYoab7IFS5H2EKyyNXH4ecrEB\\\\\\\/PvKMh01wV20\\\\\\\/utTmAiOLp7SgMHOYWsNRvivOuL7b3MMiELa7cEDPFuXefONL\\\\\\\/5r89RzxTvBHSHkF7MNTSHza\\\\\\\/fFoVAMrFS9KHmCWQ5QCZ\\\\\\\/WPzyLBhJbItyaGj8QR9EU5LDugNLz77LuvgfyR\\\\\\\/qZ6WOtRAfVp4Zf6FrU8gwmLO59p2yiXH6QM48FCnP8os\\\\\\\/qE2dO2683H1P\\\\\\\/X8VfuJmKOa3B03jq5zjoOS2amhaMHfwvNULZA6psNbqmDHFDs0pjcNB1Jq38nOy8JKPy\\\\\\\/xEXiEaeW47gZI\\\\\\\/axRc1bN75yJ1oh7YGBnc4+SLEt9DGor2p53KdS9sAffHQi2VJkd0JtHeAXWdjMWPKyFpRnk7sZ6qvTmSVNe9Yg9OtbH9ihzRYG0A8s7BNM\\\\\\\/HAqebTBozdIRkjBNl7SOwXXIHAzQSyejp\\\\\\\/H\\\\\\\/XMlJ7mUp3axjqjKhJu8iHV3PUqwIN0nbtpQ5usV89Lk14urr4SWj7hnStY6Dc1PfCJN91PMXuR5zMIHtC5yXTkSJ5LroIPWEvkksQEO2jKRG7N4uQtlrEjV\\\\\\\/Q2FNz3JtEjoutH0aj4L81hcXlJT+L7hMa\\\\\\\/TJBzvvUnyYxiJ8i0yhaT7VU5cOs0awWuXPlBNopgt6Plu5pv6oAOIlJV4WrHpK2Hxggc7bqFAgjNFpgXNCPYphdOcwzgZLxXhoFCGsMq1GjEo3yZR\\\\\\\/\\\\\\\/tPaQjHZ9C6ladZMvW6bTtU9NymOzD\\\\\\\/DJlEJSuUKFGMZeKiY6FhUb2WRp7qG+9JXMHPzcj9xwDytFDfma6q\\\\\\\/9dAij8AZbsPEQNSOcj7H9T4xrBQxPLXO+PrqjpsyfRUli2bLk8vDTsSlbrr+BMNqVyYNH6CACrlYKlUOqb5QkVFHJ1fT6fcOKRz7U6kolmLv0NaI\\\\\\\/o6r77ehJzHY3uv7p\\\\\\\/hSzYe4WdjfXVA7eOeQUddyeeLkVwYoLZPxKzNPiKxrQ0IpjOkXU98pmFUf9uZf97Ru3V1X37E07CoVeht0YKkKpprQUSXsmys7SCUchd\\\\\\\/E4w2BizNZsiXHti8iIFXneKM+ZQSv7ti7\\\\\\\/\\\\\\\/8HtpY261YVqAGZ396YwyBwMB9VW5tYkbJe9xUBd3x+HXzxkRjkN7sqiROUdCrpHyUfd3bzQdMMdLal3Z7\\\\\\\/SXUCbw0nH7EIQkfLT4kwZMkpYSvyKI0\\\\\\\/XIagMXs8CqQPJ4zqLb\\\\\\\/sVSDZOCP0ITHaPo1ALyDs6tpGSV9xmLB3pTrgBufYx71MpF5aNnll9HfO31nZh\\\\\\\/95sRkY5vMHHdhIk\\\\\\\/D0ScFnXsU\\\\\\\/K8jB5qiOBO4syHHlIZdbfSZ5hvM8Of+TQVfXQn+CK2auDDu15ATyXxSarSSBt60njbWyaOq2hjMp3Hiixad9tFL7lXDgXLD1Cigywm6KPGjaHDiJcPZRKizY78kMB5mR1wDJsXKfWpOMh9jKx13C76sPVQO4glT0RIR4qF8f\\\\\\\/KeYYi4Hsip5wfkwVcUv7QnSN+as64S8JTOcFLd2bLdUmdzqcAlUd76mGeIuS3OZeviZ6ukQv6+AUnp2DpLQwo4D3UvbfYmmTiYZ4eXI224pczl1rnViTxUwNsSXDqAG6UklWSckeCUOW++1ATw4Mgvus54p5joeHTcsiokffJ3pc7n1B0ZLww\\\\\\\/R8tJaHa+04Ioys8Owex7Uin7ypoCBgnGS43qDOKYc3OAAYPsU5RiGFgPrPELr7bQrVtV8kpbJU1mtlRlLaOTUdkQ8KCk4JVfZC\\\\\\\/ELvyNs56rsKrUbGa1Kg+jtuCnsMuW5kwir8z0OK4Pupf+0UTRu0cgbEf9eveVyHAcRyH+MBK\\\\\\\/BhFm6yicnwuxAnlnvx+hJ7+Hx0oFBDzu9y386Rs4gJPm1LJwTZ0x3qVxefQv\\\\\\\/a9HgYS2i+RI8zMU1jOZ39N\\\\\\\/Y5C7Q9SyggJ\\\\\\\/6xvfLYH69e1CaKv1wni2jJL7fM6o5qWwfwyR8LnI3W1+J6M6V5FKBvr0cjb6BlWa\\\\\\\/the4zSBTscFZ50h6MZkOXx4KXcL30bHCabumuAOhrhsU482s3qT6an2rFBOQbOpk3K89mVDt9e8UzFPhe0wTvxRHVyrVOeZlHN4S5vBlm1nGTixspR1tY0caaXkJythhpBxhFIWjkW1eD1DlRJJHQbGGldscQuZn5BtR6CTRLVop5qeeDP\\\\\\\/qFYWTG+twfZbXjs2nembRtk9y3YUgAOqM1klppj09GVbTKmv2YSUdOSF9vTxsciKMrCnENRuPmv1pKTv88YIxTRUnlKo95wthiVazZeOem9AK0lXhfV72xA6+r6flzSfL9jtW37G2r5H0TWWzyFZa4bf46ZX4Px33JpRSKa3QdRPvAm+ZfPMNbnauCyG8G4h3UCW3n8\\\\\\\/FOsAPuwP6GdPVStQVn+of2ionHYxEhpEtKM3P6lmA+5MYQNYcLIkYmo3A5+5E0EcyvANhp1xRPElBSX10CvYsymxZsFuHgUGDGVaM+qeXCfHtmT+p\\\\\\\/hWT6P45wjfcsh6AA4WetiSjoBXmR8lZJZfMYXTiYsqFk7waA88mLFTjcSbmuYS6pWZMals1WXGfTPOJ\\\\\\\/gM5rgaEdXqA9RtLV5vGp3zSAIcAKZceKAcVl6NvXg7S9SsxLvnYjOpaFYuM8+RcjXthLhPZyvOwzjM570NWObSWNBfg\\\\\\\/Bn52jG1n\\\\\\\/+S73wJU1zxvi22pHU9TzUGRwKWg\\\\\\\/TD\\\\\\\/kD99FkMUTATQeogjxUmruUjrU7uKGMHKMd35auSzWyeM5utUUeOamlaPez92PnCKKlgcvt2KjWimyY9xpBFpey7+44vUYbkf2cW5kmVmnkWGkQvTwFhMlAPdEbZeh+XEjLTUKAPaQ+56xlw8QNTnuZjWefiVs\\\\\\\/3+XxFibkHbRjITsKxafw80OREKtJjlMNhsfbZvEEBwpqEtw6sd3Yb6qR8gadn2F6gEkCE\\\\\\\/tLoVQ2zqpOnONtQUzRVFC9aPaWQ9awApVskTI+U9WgCa7bmYa44tsHKpycp0+Q7\\\\\\\/x+MbsW6cpH8t+WZKvxZv4WbiuMHlXcQiRXh+6NM\\\\\\\/tu1vGNgmhynaABnv4LS8rjIajrUZzreS8x3VWIatzMjxKle6DXj9gqVOBa\\\\\\\/DOrApT+GwlluWfVTBLcM\\\\\\\/bbeW3wLBGwMBuy+WaYvQ3NoJYAd1voU4VtHJoam0et7W2nBP0GY7fOJwdb50QQLhDqiuWbvsMaOhA9gTf+QppVb4cr784QAYZxNZo7s+7r8PmX1PpAI2KwLylsHjdgICRANZZMG7Iju3sm1fKqXAgLBA1BCFNv9x2ZJm6jP2nyAXeTlOfsAz2bKPmhHyTE8l+MO1vtRxMdtv6Bww4L7Y1unkiCoAXixyBaQkZKOLyYteVoWtkukyK4bkP7KQPGOMSLa\\\\\\\/AN3a5nfHhgMmamnFK6Haenc5RXM0yrgIoUUjvQ3iF7uC8YxvUOIdeRDwMTFAL4tRQeKpRPNWbqk5Ap4b9IN8Zz0jX+wT4Wwtsv4XQAno00mnQU+347xm1Tl\\\u0022,\\\u0022iv\\\u0022:\\\u0022JOHZztBQx3UGBXj9qOKW7A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002249755a6838330fd2ad926443ef6dfe1d2b0593b269fd9265df569f14ba17a579\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OCcG1XcgQ9g8ujeCPIVDqe10CbrD72eB1d4kubZW5R3xXXcf1PBvXNHoJZbu1elYoeWxQ5U7YVXYddPmuht5qBHlSBpIun5OENE5VhoItKrdcQ\\\\\\\/F08ebE\\\\\\\/0bxbRNhSsjRT+lXiJqGDJgLoJknLw7fbec90OJHdCEX\\\\\\\/BXANpTkOX+v4bHfN6oeo052CbxLVaILjHqIPly2Zz0KJ\\\\\\\/pca0tB3YsinUIKKdtQSzSVf8QfB3jgXs42tCA6BWpeyoy\\\\\\\/HmVVTO4viT+LqwvL6DXbcwANHK9MiX\\\\\\\/tc8iTfMyT+fJ2xVL1Qsq6VfmzsWBJ0HWmPHAXUEifIgmAsHnpNJ6elsgK\\\\\\\/BVfQ6dVVI1TIq\\\\\\\/upNjeBRQCdtzB5mdchQ2rqdZLYXCQjcANr1Fa8LslVUNrKWTvnBk2QaHRL5LLN8HGnnE5zV+JkCPF1MPE6n+d8ucMo7EvQkyRux2kmYpunYUuOEqNAIxI6yoGqndBQQ6qtFY\\\\\\\/GYXaXOinukNvmx+EhXu5bnTot0+4\\\\\\\/EkswvQ4e6pIkpCf8tSwxHKkDSe4e7ndIUw7ZwOKH92x1o3aOxV3ZZPmEHRTKxSVJRwhijoEGZEI4DJuPtYCLhjaq3rfMmdUcn1\\\\\\\/Zbg3O4YYIvBQ1WFQ3QcQSjTDfoIR9TWv3oVxg0fg+1Q5CxGM05ED4BX8+SBxt6f2CMAl1A5\\\\\\\/Qf8jTnS+2Qt1q8YcH5W+jtDJz+DCNacJVVdPBWAUaO9DYKTgXLtXDqc5yCv+rTOTL+xai8qBdjQyifX\\\\\\\/Ssu9fWgJvDVmN44la9KYV+bzUN0DYjkLXI17AfPRrPfPI7nSRibc+HHKGUatGSDr7Efg7ip+AC+UpHZkHPaJcPzg3ou5mBpyovSFSZ6DdoXttLEg+sZpsAVWy3MAnt\\\\\\\/Eu2d4CmUNjzGEVxY11\\\\\\\/xWhEsKI3UG8T5SOxLHtoPT5p8QM2L82UA3d6\\\\\\\/dJa\\\\\\\/akv7y2wh9\\\\\\\/JuyXxoGlpWE5HoZHZUlbASeqdnNE+IeKblzeB4MwZSlmKFV45OX11EsBqtCLZIJE\\\\\\\/o20MFXN7n5orit+EF9QhrQtgRgAlN5CsXzcYTdGpNL1+nVzbrbWsiAU0Uv9Z+lNJ1cd4PITuRet7GPZ+qinZ1GK8vg0JIL5Z\\\\\\\/Rcp9p1sbRq+d2IzTlO5M5vun63k8hDFXVFRrIhcuymJad82Of1HXuxtZgIM1VOAF6mV18PbHmfk8MpGl0alSSpPVLgaIveleMY\\\\\\\/S483lzuhi1LFayaycxePCw\\\\\\\/1dU0UyZnB2Q2lID3NqM2TYLIP6lujbaH3b3+S\\\\\\\/hq\\\\\\\/9k5ri3TOW4kPpx0A9YOaPe2c+rx\\\\\\\/QgfWnCc7PlkGdZF5432doyAD03aKaBv+g+HCw\\\\\\\/EqDKeyPNCS2PyxgEbmth2VSJ3ZHuwxrGSc4Ty4S1AlcElPDz2ZZeEdtqktZP2iplXptK2dpO+fVq6fTm2nq3NuYP6y2gc\\\\\\\/G82JnyNcepraEoqJIc3r4obB9hIpNyvl9t2RuNtARLTM8dpamrDa\\\\\\\/17hj8bAsK1VP5C\\\\\\\/I2lsI+V4ORw8md9HfUEAyXpZrLKrp1qTdDxWi4MMypoi5gFNl\\\\\\\/LO2otzug4wPvZfFxy9To2MptEKmwkC6Xy5KcppGqDotvKHOrkYmeYaHJHSw1pgke9Slt5CcLCuogLYbWWBbZ09pREnGYLM1zxYPOEKMKgQZDxc\\\\\\\/Sh7\\\\\\\/QK0or1\\\\\\\/GL9U25kMTamrHCw4y8V2OjsgBAWA9QOcotZljfwCNMVo5Nc99M5Hb9Dq5m9ucZ2Ny01U3IrpWbzdeql0Qohfuw+LG430xto+aWjcD\\\\\\\/i\\\\\\\/hzktI3LAYhycaQieGjG5VDVUnuIzz0hidSTE1HuYQxW+JSOgbH1AuclbT2iLDoOvlnYsHTcXij8HFYjYt4ctrS9SiDMjW2UaZwuXBwUPcjAbtNC6+5P2LL49EjtetGz+lBD48kolIlhGtTuGbwfD8oXVdJFH4twrPLn+jlB2xOerAp0oF+XWzpsCQKtAyMdMHuG1Eo0bLJlV09P7to4Y1JsqNkt4wO0v3kbDKjA0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Nb175kwa+kmVXLmB5KROJQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f416b222e558164704acf6a22973156f9a873244169a2cccc918dafc681dd9aa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492870 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1355" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7_-V4yh8XuLcY8yWauZACQsnXQvNtjYh0RqdkbZcbHE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1355" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MV2upGiFBbcGgHhc-_3jCBiEAqCIqwMI4DnOqrinV5s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1339" title="Novel exc Genes Involved in Formation of the Tubular Excretory Canals of Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1371" title="TSD: A Computational Tool To Study the Complex Structural Variants Using PacBio Targeted Sequencing Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492870" id="node492870" data-pisa="ggg;9/5/1355" data-pisa-master="ggg;g3.119.400126" data-apath="/ggg/9/5/1355.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Francisco Javier Luna-Vázquez</span>, <span class="highwire-citation-author" data-delta="3">Fernando H. Toledo</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="5">Morten Lillemo</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1355-1369;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400126 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492870" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías, (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Francisco Javier Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BJavier%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BJavier%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center, (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, CP 56230, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Morten Lillemo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Sciences, Norwegian University of Life Sciences, IHA/CIGENE, P.O. Box 5003, NO-1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BLillemo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lillemo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorten%2BLillemo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center, (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1355" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492870" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492870/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1355.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492870" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492870/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1355.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492870" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492870/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1355.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492870" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492870/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Evidence that genomic selection (GS) is a technology that is revolutionizing plant breeding continues to grow. However, it is very well documented that its success strongly depends on statistical models, which are used by GS to perform predictions of candidate genotypes that were not phenotyped. Because there is no universally better model for prediction and models for each type of response variable are needed (continuous, binary, ordinal, count, etc.), an active area of research aims to develop statistical models for the prediction of univariate and multivariate traits in GS. However, most of the models developed so far are for univariate and continuous (Gaussian) traits. Therefore, to overcome the lack of multivariate statistical models for genome-based prediction by improving the original version of the BMTME, we propose an improved Bayesian multi-trait and multi-environment (BMTME) R package for analyzing breeding data with multiple traits and multiple environments. We also introduce Bayesian multi-output regressor stacking (BMORS) functions that are considerably efficient in terms of computational resources. The package allows parameter estimation and evaluates the prediction performance of multi-trait and multi-environment data in a reliable, efficient and user-friendly way. We illustrate the use of the BMTME with real toy datasets to show all the facilities that the software offers the user. However, for large datasets, the BME() and BMTME() functions of the BMTME R package are very intense in terms of computing time; on the other hand, less intensive computing is required with BMORS functions BMORS() and BMORS_Env() that are also included in the BMTME package.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/multi-environment" class="hw-term hw-article-keyword hw-article-keyword-multi-environment" rel="nofollow">multi-environment</a></li><li class="kwd"><a href="/keyword/multi-trait" class="hw-term hw-article-keyword hw-article-keyword-multi-trait" rel="nofollow">multi-trait</a></li><li class="kwd"><a href="/keyword/genome-based-prediction-and-selection" class="hw-term hw-article-keyword hw-article-keyword-genome-based-prediction-and-selection" rel="nofollow">genome-based prediction and selection</a></li><li class="kwd"><a href="/keyword/r-software" class="hw-term hw-article-keyword hw-article-keyword-r-software" rel="nofollow">R-software</a></li><li class="kwd"><a href="/keyword/multivariate-analysis" class="hw-term hw-article-keyword hw-article-keyword-multivariate-analysis" rel="nofollow">multivariate analysis</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li></ul><p id="p-3">Genomic selection (GS) is a methodology used in plant breeding that was proposed by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meuwissen <em>et al.</em> (2001)</a>. It is a type of marker-assisted selection that consists of genotyping and phenotyping a training sample (reference population); with the help of statistical models, predictions of genomic estimated breeding values (GEBV) or phenotypic values of the testing sample (validation population) are obtained for which only genome-wide dense genetic marker data were available. GS does not depend on prior knowledge about a few, large-effect genes or QTL, since all markers are used simultaneously in the training of the statistical models. GS was first used in animal breeding (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hayes and Goddard 2010</a>), but nowadays is being implemented in many crops, for example, maize (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em>, 2014</a>), cassava (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de Oliveira <em>et al.</em>, 2012</a>), wheat (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rutkoski <em>et al.</em>, 2011</a>), sugar beet (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Würschum <em>et al.</em>, 2013</a>), tomato (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yamamoto <em>et al.</em>, 2016</a>), rice (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Spindel <em>et al.</em>, 2015</a>), apple (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kumar <em>et al.</em>, 2012</a>), pea (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burstin <em>et al.</em>, 2015</a>), cranberry (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Covarrubias-Pazaran <em>et al.</em>, 2018</a>) and many others.</p><p id="p-4">In recent years, an active area of research has begun to develop and improve existing statistical models for genomic selection (GS) due to the fact that successful GS implementation is strongly related to the accuracy of the predictions performed by statistical models. However, because there are no universally superior machines for prediction, many models have been proposed. For example, most of the proposed models are univariate and few are multivariate. Most of the univariate models are appropriate for continuous and Gaussian phenotypes, but there are several appropriate models for binary, ordinal and count traits. Some examples of implementations of models for non-Gaussian, non-continuous traits are unordered categorical (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Heuer <em>et al.</em>, 2016</a>), binomial (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Technow and Melchinger 2013</a>) and ordinal categorical (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Montesinos-López <em>et al.</em>, 2015a</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">b</a>). While multivariate models are used almost routinely nowadays, for the joint analysis of multiple-traits (<em>e.g.</em>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jia and Jannink 2012</a>) as well as multiple-environments (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em>, 2012</a>) and even multiple populations (<em>e.g.</em>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Olson <em>et al.</em>, 2012</a>), there are few multivariate practical software programs for continuous and Gaussian phenotypes and there are scarcely any models and software for other types of response variables. To the best of our knowledge, almost none of the currently reported models consider mixed phenotypes including continuous, binary, ordinal, count, etc. traits. For this reason, it is clear that to increase the power of GS technology, it is of paramount importance to develop more models and improve the existing ones.</p><p id="p-5">Multi-trait models in GS have been applied by many scientists. For example, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Calus and Veerkamp (2011)</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jia and Jannink (2012)</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jiang <em>et al.</em> (2015)</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">He <em>et al.</em> (2016)</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schulthess <em>et al.</em> (2017)</a>, and <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Covarrubias-Pazaran <em>et al.</em> (2018)</a> reported that multi-trait analysis outperforms uni-trait analysis in terms of prediction accuracy and that the larger the correlation between traits, the larger the benefit of multi-trait analysis. The Multi-Trait Model (MTM) of <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos and Grüneberg (2016)</a> is a mixed multi-trait Gaussian model under the Bayesian framework that uses a Gibbs sampler for inferences. Furthermore, Bayesian multi-output regressor stacking (BMORS) is a Bayesian version of the multi-trait regressor stacking method proposed by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Spyromitros-Xioufis <em>et al.</em> (2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2016</a>). The training of BMORS has two stages: (1) a single univariate model is implemented using the GBLUP model, and (2) the resulting predictions are directly included by BMORS in an additional training stage. Thus, the concept of BMORS is that a second-stage model will correct the predictions of the first-stage model [using the predictions of the first-stage univariate GBLUP model (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Spyromitros-Xioufis <em>et al.</em>, 2012</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">2016</a>)].</p><p id="p-6"><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Montesinos-Lopez <em>et al.</em> (2016)</a> were the first to develop a comprehensive theory for a Bayesian multi-trait multi-environment (BMTME) model for genome-based prediction. An improved version of BMTME allows general covariance matrices by using the matrix normal distribution that facilitates easy derivation of all full conditional distributions and permits a more efficient model in terms of time of implementation <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> (2018a</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">b</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">c</a>). In general, the matrix normal distribution model considerably improved in terms of implementation time over the time required by the original BMTME. Also, the Gibbs sampler for implementing the new BMTME model can be found in <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> (2018a)</a>, and the priors of the model are given in detail in <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Montesinos-López <em>et al.</em> (2018b)</a>. <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> (2018a)</a> provide the appropriate notations used for the matrix-variate normal distribution that is a generalization of the multivariate normal distributions to matrices. This plays a key role in building the BMTME model. The original software used by <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Montesinos-Lopez <em>et al.</em> (2016)</a> to fit the BMTME was the first attempt to implement the multi-trait multi-environment theory when analyzing real data; however, the lack of the necessary optimization algorithms for efficiently applying the software made the original BMTME difficult to apply to real data.</p><p id="p-7">It is also important to point out that even though the existing R statistical software for Bayesian analysis like ’stan’ (<a href="https://mc-stan.org/">https://mc-stan.org/</a>) and ‘JAGS’ (<a href="https://en.wikipedia.org/wiki/Just_another_Gibbs_sampler">https://en.wikipedia.org/wiki/Just_another_Gibbs_sampler</a>) are very flexible for implementing Bayesian analysis, they are not user-friendly because the user needs a certain level of programming skills to correctly implement them (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Stan Development Team 2018</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Plummer 2018</a>). These two software programs (stan and JAGS) also require more computational resources for their implementation since they are built not with conjugate priors. It is documented that multivariate analysis improves parameter estimation (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schulthess <em>et al.</em>, 2017</a>). For this reason, we agree with <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Castro <em>et al.</em> (2013)</a> and <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Huang <em>et al.</em> (2015)</a>, who stated that multi-trait analysis is a powerful tool for clarifying the relationship and the effect of each studied variable and for building more efficient prediction models.</p><p id="p-8">Due to the background of plant breeders, not only are new models needed, but the existing ones need to be improved. We also need reliable, efficient, user-friendly software in which breeders can implement the existing GS models. One popular R package in the context of genomic selection for continuous and ordinal data are the BGLR package of <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pérez and de los Campos (2014)</a> that was built under the Bayesian framework and is very flexible because it allows the use of a genomic relationship matrix (derived from marker or pedigree), and also allows implementing various methods like BayesA, BayesB, Bayes Lasso, Bayes Ridge and GBLUP and can deal with moderate datasets; however, it only allows the implementation of univaritate models. Therefore, to contribute to this requirement, we developed a Bayesian multi-trait and multi-environment (BMTME) R software that allows the implementation of multi-trait and multi-environment data for performing parameter estimates and evaluating the prediction performance of multiple traits that are studied in many environments. This BMTME package is different from existing ones [sommer (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Covarrubias-Pazaran 2016</a>), BGGE (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Granato <em>et al.</em>, 2018</a>), ASREML (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gilmour <em>et al.</em>, 1995</a>) and MCMCglmm (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hadfield <em>et al.</em>, 2010</a>)] because it takes into account the genetic correlation between traits and between environments. The main difference of BMTME with sommer and ASREML is that our package was built under a Bayesian framework, while sommer and ASREML were based on a classical approach using restricted maximum likelihood. The difference between BGGE and our model is that our model is not only for multi-environment data but rather for multi-environment and multi-trait data simultaneously. On the other hand, the MCMCglmm package only allows a general covariance matrix for traits but not for environments, like the proposed BMTME package; however, it is important to point out that the MCMCglmm package allows modeling not only continuous responses but also binary, ordinal and counts.</p><p id="p-9">The main objective of this research was to illustrate the application of the new BMTME with two real toy datasets; with these we show how to use the functions available in the BMTME package for implementing multi-environment (BME function), multi-trait and multi-environment data (BMTME function), as well as the Bayesian multi-output regressor stacking functions BMORS () and BMORS_ENV (). These two functions are very different to what the existing software [sommer (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Covarrubias-Pazaran 2016</a>), BGGE (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Granato <em>et al.</em>, 2018</a>), ASREML (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gilmour <em>et al.</em>, 1995</a>) and MCMCglmm (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hadfield <em>et al.</em>, 2010</a>)] implements, since the theory behind this function is that of stacking methods. Stacking methods consist of training multiple learning algorithms for the same dataset and then combining the predictions to obtain the final predictions. In this study we used the initial BMTME of <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Montesinos-Lopez <em>et al.</em> (2016)</a> but improved it by using the matrix variate normal distribution (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em>, 2018c</a>) and the appropriate priors given by <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> (2018a)</a> and <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Montesinos-López <em>et al.</em> (2018b)</a>.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Statistical models</h3><div id="sec-3" class="subsection"><h4>Multiple-environment Genomic Best Linear Unbiased Predictor (GBLUP) model:</h4><p id="p-10">Since genotype × environment interaction is of paramount importance in plant breeding, the following univariate linear mixed model is often used for each trait:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-2.gif"/></noscript></span></span> represents the response of the jth line in the ith environment (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-3.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-4.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-5.gif"/></noscript></span></span>represents the effect of the ith environment, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-6.gif"/></noscript></span></span> represents the random genomic effect of the jth line, with <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-7.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-8.gif"/></noscript></span></span> is a genomic variance, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-9.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-10.gif"/></noscript></span></span> and represents the genomic relationship matrix (GRM) and is calculated (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">VanRaden 2008</a>) as <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-11.gif"/></noscript></span></span>, where p denotes the number of markers and W is the matrix of markers of order <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-12.gif"/></noscript></span></span>. The <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-13.gif"/></noscript></span></span>matrix is constructed using the observed similarity at the genomic level between lines, rather than the expected similarity based on pedigree. Further, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-14.gif"/></noscript></span></span> is the random interaction term between the genomic effect of the jth line and the ith environment with <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-15.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-16.gif"/></noscript></span></span> is an interaction variance, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-17.gif"/></noscript></span></span> is a random residual associated with the jth line in the ith environment distributed as <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-18.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-19.gif"/></noscript></span></span> is the residual variance.</p></div><div id="sec-4" class="subsection"><h4>Bayesian multiple-trait multiple-environment (BMTME) model:</h4><p id="p-11">The current BMTME model was implemented by <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> (2018a</a>,<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">b</a>,<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">c</a>). For a complete understanding of its description, first we provide the notations used for the matrix-variate normal distribution that plays a key role in building the BMTME model. Matrix-variate normal distribution is a generalization of the multivariate normal distribution to matrices. The (<em>n</em>×<em>p</em>) random matrix, M, has a matrix-variate normal distribution denoted as <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-20.gif"/></noscript></span></span>, if and only if, the (<em>np</em>×1) random vector <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-21.gif"/></noscript></span></span> is distributed as multivariate normal as <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-22.gif"/></noscript></span></span>; therefore, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-23.gif"/></noscript></span></span> denotes the (<span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-24.gif"/></noscript></span></span>) dimensional matrix-variate normal distribution, H is a (<em>n</em> × <em>p</em>) location matrix, Σ is a (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-25.gif"/></noscript></span></span>) first covariance matrix, and Ω is a (<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-26.gif"/></noscript></span></span>) second covariance matrix (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Srivastava and Khatri 1979</a>). <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-27.gif"/></noscript></span></span> and ⊗ are the standard vector operator and Kronecker product, respectively. Unlike in a multivariate normal model where the data are concatenated into a single vector of length <em>np</em>, in a matrix-variate normal model, the data <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-28.gif"/></noscript></span></span> are in an <em>n</em>×<em>p</em> matrix where each column is a trait (<a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em>, 2018a</a>). Therefore, the proposed BMTME model is defined as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where Y is of order <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-30.gif"/></noscript></span></span>, with L the number of traits and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-31.gif"/></noscript></span></span>, where J denotes the number of lines and I the number of environments, X is of order <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-32.gif"/></noscript></span></span>, β is of order <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-33.gif"/></noscript></span></span>, since <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-34.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-35.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-36.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-37.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-38.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-39.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-40.gif"/></noscript></span></span> and contains the genotype × trait interaction term since <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-41.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-42.gif"/></noscript></span></span> is the effect of the genotype × trait interaction term for <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-43.gif"/></noscript></span></span> and for <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-44.gif"/></noscript></span></span>.<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-45.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-46.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-47.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-48.gif"/></noscript></span></span> and contains the genotype × environment × trait interaction, since <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-49.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-50.gif"/></noscript></span></span> is the effect of genotype × environment × trait interaction for <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-51.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-52.gif"/></noscript></span></span> and for <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-53.gif"/></noscript></span></span>. Vector <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-54.gif"/></noscript></span></span>is distributed under a matrix-variate normal distribution as <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-55.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-56.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-57.gif"/></noscript></span></span> and represents the Genomic Relationship Matrix (GRM) and is calculated using the <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">VanRaden (2008)</a> method as <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-58.gif"/></noscript></span></span>, where p denotes the number of markers and W the matrix of markers of order <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-59.gif"/></noscript></span></span>; and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-60.gif"/></noscript></span></span> is the unstructured genetic (co)variance matrix of traits of order <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-61.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-62.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-63.gif"/></noscript></span></span>is an unstructured (co)variance matrix of order <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-64.gif"/></noscript></span></span>and E is the matrix of residuals of order <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-65.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-66.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-67.gif"/></noscript></span></span> is the unstructured residual (co)variance matrix of traits of order <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-68.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-69.gif"/></noscript></span></span> is the genomic relationship matrix described above (<a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em>, 2018a</a>). The BMTME model resulting from equation (2) was implemented by <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Montesinos-Lopez <em>et al.</em> (2016)</a>.</p><p id="p-12">Next, we used the modified version of the Gibbs sampler of the original BMTME model proposed by <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Montesinos-Lopez <em>et al.</em> (2016)</a> that was implemented in <a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> (2018a)</a>. It is important to point out that model (2) takes into account the genotype × environment terms in the (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-70.gif"/></noscript></span></span> term and, for comparison purposes, we also ran the model in equation (2) but without the (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-71.gif"/></noscript></span></span> term to study the effect on prediction performance with and without the genotype × environment term. The Gibbs sampler for implementing the BMTME model is found in <a id="xref-ref-28-9" class="xref-bibr" href="#ref-28">Montesinos-López <em>et al.</em> (2018a)</a>, and the priors of this model are given in detail in <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Montesinos-López <em>et al.</em> (2018b)</a>. The concept of the matrix variate normal distribution is given in <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em> (2018c)</a>.</p></div><div id="sec-5" class="subsection"><h4>Bayesian multi-output regressor stacking (BMORS):</h4><p id="p-13">The proposed BMORS is a Bayesian version of the multi-trait regressor stacking method proposed by <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Spyromitros-Xioufis <em>et al.</em> (2012</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">2016</a>). The training of BMORS consists of two stages. In the first stage, L single univariate models are implemented using the GBLUP model given in equation (1), but instead of using the resulting predictions directly as the final output, the BMORS includes an additional training stage where a second set of L meta-models are implemented for each of the L traits under study. Each meta-model is implemented with the following model:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-72.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where the covariates <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-73.gif"/></noscript></span></span> represent the scaled predictions of each trait obtained with the GBLUP model in the first-stage analysis, and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-74.gif"/></noscript></span></span> are the beta coefficients for each covariate. The scaling of each prediction was performed by subtracting its mean (<span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-75.gif"/></noscript></span></span>) and dividing by its corresponding standard deviation (<span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-76.gif"/></noscript></span></span>), that is, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-77.gif"/></noscript></span></span>=<span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-78.gif"/></noscript></span></span>, for each <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-79.gif"/></noscript></span></span>. Therefore, the BMORS model contains as predictor information the scaled predictions of its response variables yielded by the first-stage models. In other words, the BMORS model is based on the idea that a second-stage model is able to correct the predictions of a first-stage model using information about the predictions of other first-stage models (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Spyromitros-Xioufis <em>et al.</em>, 2012</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">2016</a>).</p></div></div><div id="sec-6" class="subsection"><h3>Real toy datasets</h3><div id="sec-7" class="subsection"><h4>Mada dataset:</h4><p id="p-14">This dataset was obtained from the study by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ben Hassen <em>et al.</em> (2018)</a>. The dataset is composed of a sample of 188 wheat lines evaluated for six traits. Each of the lines was evaluated in one environment. The lines were genotyped and 32,066 single nucleotide polymorphisms (SNPs) were obtained with a heterozygosity rate &lt; 5% and a minor allele frequency (MAF) &gt; 5%. A subset of the data were included in the package that includes 30 lines, and we named this dataset Mada. For more details, see the study by <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ben Hassen <em>et al.</em> (2018)</a>. Raw markers are not included, and we provide the genomic relationship matrix (GRM) calculated according to the method of <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">VanRaden (2008)</a>.</p></div><div id="sec-8" class="subsection"><h4>Maize dataset:</h4><p id="p-15">This dataset was obtained from the study by <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Montesinos-Lopez <em>et al.</em> (2016)</a>. It consists of a sample of 309 maize lines evaluated for three traits: anthesis-silking interval (ASI), plant height (PH), and grain yield (GY). Each trait was evaluated in three optimal environments (Env1, Env2 and Env3). The lines were genotyped, 681,257 single nucleotide polymorphisms (SNPs) were obtained, and markers with more than 20% missing values were removed. After that, markers were imputed using observed allelic frequencies, and markers with MAF &lt; 0.05 were removed, so that at the end of the quality control and imputation, 158,281 SNPs were still available for further analyses. To load this dataset in the package, we used only 30 lines, and we named this dataset Maize. For more details, see the study by <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Montesinos-Lopez <em>et al.</em> (2016)</a>.</p></div></div><div id="sec-9" class="subsection"><h3>Evaluation of prediction performance</h3><p id="p-16">We implemented cross-validation (CV) to evaluate the prediction performance. Two types of CV were implemented: K-fold cross-validation and random cross-validation.</p><div id="sec-10" class="subsection"><h4>K-fold cross-validation:</h4><p id="p-17">Under this CV, the dataset was partitioned into K subsamples of equal size; each time K-1 of them were used for training (TRN) and the remaining one for testing (TST). In this CV, one observation cannot appear in more than one fold. In the design, some lines can be evaluated in some, but not all, target environments, which mimics a prediction problem faced by breeders in incomplete field trials. This CV strategy is exactly the same as the strategy denoted as CV2 that was proposed and implemented by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jarquín <em>et al.</em> (2017)</a>, where a certain portion of test lines (genotypes) in a certain portion of test environments is predicted, since some test lines that were evaluated in some test environments are assumed to be missing in others.</p></div><div id="sec-11" class="subsection"><h4>Random cross-validation:</h4><p id="p-18">This CV strategy randomly splits the dataset into training (TRN) and testing data (TST). For each such split, the model is fitted to the TRN data, and predictive accuracy is assessed using the TST data. Since we used sampling with replacement, one observation may appear in more than one partition. The implemented CV mimics a prediction problem faced by breeders in incomplete field trials, where some lines may be evaluated in some, but not all, target environments. Since <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-80.gif"/></noscript></span></span> denotes the total number of records per each available trait, then to select lines in the TST dataset, we fixed the percentage of data to be used for TST [PTesting]. Then PTesting<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-81.gif"/></noscript></span></span> (lines) were chosen at random, and subsequently for each of these lines, one environment was randomly picked from I environments. The cells selected through this algorithm were allocated to the TST dataset, while the cells (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-82.gif"/></noscript></span></span> that were not selected were assigned to the TRN dataset. Lines were sampled with replacement if <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1355/embed/mml-math-83.gif"/></noscript></span></span>, and without replacement otherwise (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">López-Cruz <em>et al.</em>, 2015</a>). The metrics used to measure the prediction accuracy under both CV strategies were Pearson’s correlation and the mean arctan absolute percentage error (MAAPE), which has the advantage that no zero estimates are produced when the response variable contains many zeros. They were calculated from each trait-environment combination for each of the testing sets and the average of all random partitions (folds) is reported as a measure of prediction performance.</p></div></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-19">The data used in this study are included in the BMTME package, so once that package is installed, the datasets can be loaded into the R environment.</p></div><div id="sec-13" class="subsection"><h3>Installation of the BMTME package</h3><p id="p-20">The aim of this section is to illustrate the use of the R BMTME package for analyzing multi-environment and multi-trait and multi-environment data from plant breeding programs. The BMTME package was built following the paper by <a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">Montesinos-Lopez <em>et al.</em> (2016)</a> and implemented in the R statistical software (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">R Core Team 2018</a>).</p><p id="p-21">The development version of the BMTME package can be installed directly from the GitHub repository (<a href="https://github.com/frahik/BMTME">https://github.com/frahik/BMTME</a>). In order to install the package, it is necessary to install the appropriate compilers; the installation process and the required tools depend heavily on the operating system. For example, in Windows it is necessary to install Rtools (<a href="https://cran.r-project.org/bin/windows/Rtools/">https://cran.r-project.org/bin/windows/Rtools/</a>), and in modern versions of macOS, it is necessary to install XCode from App Store or the development tools for R from CRAN (<a href="https://cran.r-project.org/bin/macosx/tools/">https://cran.r-project.org/bin/macosx/tools/</a>). In the case of Linux, it is necessary to install the C++ compilers included in your distribution, for example, g++ from GNU (<a href="https://www.gnu.org">https://www.gnu.org</a>). Once the tools have been installed, use the following command to install the package within your R session:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-22">install.packages(′devtools′)</p></li><li id="list-item-2"><p id="p-23">devtools::install_github(′frahik/BMTME′)</p></li></ul><p id="p-24">You can also find the package in the CRAN repository, and you can use the following command (see below) to install a version of the package from CRAN. This will avoid the need to install some dependencies manually and install the Rtools software using the following command:</p><ul class="list-simple " id="list-2"><li id="list-item-3"><p id="p-25">install.packages(′BMTME′)</p></li></ul><p id="p-26">The R package BMTME is available at the following link: <a href="https://cran.r-project.org/web/packages/BMTME/index.html">https://cran.r-project.org/web/packages/BMTME/index.html</a>.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><p id="p-27">The results are given in three main sections. The first section illustrates the use of the BME function for implementing multi-environment analysis, while the second and the third sections illustrate the use of the BMTME and BMORs functions for implementing multi-trait and multi-environment analyses.</p><div id="sec-15" class="subsection"><h3>The BME Function</h3><p id="p-28">This example illustrates how to fit a model when there is only one environment and several dependent variables. First, we load the library:</p><ul class="list-simple " id="list-3"><li id="list-item-4"><p id="p-29">library(BMTME)</p></li></ul><p id="p-30">Then we load the Mada dataset:</p><ul class="list-simple " id="list-4"><li id="list-item-5"><p id="p-31">data(“WheatMadaToy”)</p></li></ul><p id="p-32">Then we define the model to be adjusted; since the dataset only includes an environment where several dependent variables were evaluated, the BME model is used. To implement it, first we need to order the dataset as follows:</p><ul class="list-simple " id="list-5"><li id="list-item-6"><p id="p-33">phenoMada &lt;- (phenoMada[order(phenoMada$GID),])</p></li><li id="list-item-7"><p id="p-34">rownames(phenoMada)=1:nrow(phenoMada)</p></li><li id="list-item-8"><p id="p-35">head(phenoMada)</p></li><li id="list-item-9"><p id="p-36">GID PH FL FE NS SY NP</p></li><li id="list-item-10"><p id="p-37">1 9 29.7776 -8.8882 -4.93900 1.04100 169.06 28.8025</p></li><li id="list-item-11"><p id="p-38">2 11 3.2210 -7.1111 -0.36940 -3.88940 -107.19 58.2516</p></li><li id="list-item-12"><p id="p-39">3 12 6.1670 -9.5337 -12.43680 2.58250 -160.54 17.1278</p></li><li id="list-item-13"><p id="p-40">4 15 6.8117 4.6377 11.78860 -0.03378 235.70 -19.6571</p></li><li id="list-item-14"><p id="p-41">5 20 -14.4480 3.2525 6.40780 -14.23460 131.87 42.2962</p></li><li id="list-item-15"><p id="p-42">6 21 -13.2185 3.8902 0.09722 5.35680 164.06 36.8239</p></li></ul><p id="p-43">This is a very important step in the analysis, because if the dataset is not ordered correctly, this may cause conflicts and produce incorrect estimations. Also, with the head() function we printed the phenotypic dataset, where the required format of the dataset requires a first column with the identifiers of the lines and then the names of all the traits. It is important to respect this format to be able to successfully implement the multi-environment (trait) datasets.</p><p id="p-44">Then, the design matrix for the genetic effects should be generated, as shown below.</p><ul class="list-simple " id="list-6"><li id="list-item-16"><p id="p-45">LG &lt;- cholesky(genoMada)</p></li><li id="list-item-17"><p id="p-46">ZG &lt;- model.matrix(∼0 + as.factor(phenoMada$GID))</p></li><li id="list-item-18"><p id="p-47">Z.G &lt;- ZG %*% LG</p></li></ul><p id="p-48">Then, we can extract the phenotypic responses that were converted to matrix object as shown in the following command:</p><ul class="list-simple " id="list-7"><li id="list-item-19"><p id="p-49">Y &lt;- as.matrix(phenoMada[, -c(1)])</p></li></ul><p id="p-50">Finally, the model was adjusted, and 30,000 iterations were used to adjust the model.</p><ul class="list-simple " id="list-8"><li id="list-item-20"><p id="p-51">fm &lt;- BME(<span class="underline">Y =</span> Y, <span class="underline">Z1 =</span> Z.G, <span class="underline">nIter =</span> 30000, <span class="underline">burnIn =</span> 20000, <span class="underline">thin =</span> 2, <span class="underline">bs =</span> 50)</p></li></ul><p id="p-52">It is important to point out that bs is the block size for sampling from posterior distributions; we suggest using a value of at least 50 but less than 1000.</p><p id="p-53">Next we used the names() function to identify all the available outputs of the fitted model.</p><ul class="list-simple " id="list-9"><li id="list-item-21"><p id="p-54">names(fm)</p></li><li id="list-item-22"><p id="p-55">[1] “Y” “nIter” “burnIn” “thin” “dfe”</p></li><li id="list-item-23"><p id="p-56">[6] “Se” “yHat” “SD.yHat” “beta” “SD.beta”</p></li><li id="list-item-24"><p id="p-57">[11] “b1” “SD.b1” “vare” “SD.vare” “varTrait”</p></li><li id="list-item-25"><p id="p-58">[16] “SD.varTrait” “NAvalues”</p></li></ul><p id="p-59">Here we extracted the observed values ($Y), the predicted values ($yHat), the parameters provided for the model fit ($nIter, $burnIn, $thin, etc.), the estimates of the beta coefficients, random effects of lines and the genetic and residual covariances ($beta, $SD.beta, $b1, $SD.b1, $varTrait, $vare, etc.). Next we show how to extract the predicted values:</p><ul class="list-simple " id="list-10"><li id="list-item-26"><p id="p-60">head(fm$yHat)</p></li><li id="list-item-27"><p id="p-61">PH FL FE NS SY NP</p></li><li id="list-item-28"><p id="p-62">[1,] 13.4602 -4.6073 2.2247 -5.2839 -161.7740 28.9334</p></li><li id="list-item-29"><p id="p-63">[2,] 4.6792 -2.7261 2.3857 -3.3243 -89.1425 29.2848</p></li><li id="list-item-30"><p id="p-64">[3,] 3.0779 -2.0297 0.1369 -3.2161 -91.4642 11.7453</p></li><li id="list-item-31"><p id="p-65">[4,] 5.2408 1.6600 5.6141 1.6136 60.2424 -23.3492</p></li><li id="list-item-32"><p id="p-66">[5,] -5.1899 1.0555 1.8783 -3.0015 49.6383 11.7256</p></li><li id="list-item-33"><p id="p-67">[6,] -9.8591 1.2399 0.5886 3.4539 88.8181 26.9665</p></li></ul><p id="p-68">We also extracted the genetic covariance between traits, as shown below:</p><ul class="list-simple " id="list-11"><li id="list-item-34"><p id="p-69">COV_TraitGenetic &lt;- fm$varTrait</p></li><li id="list-item-35"><p id="p-70">COV_TraitGenetic</p></li><li id="list-item-36"><p id="p-71">PH FL FE NS SY NP</p></li><li id="list-item-37"><p id="p-72">[1,] 64.4266 -4.2148 6.3730 -0.0049 -103.8781 -114.4456</p></li><li id="list-item-38"><p id="p-73">[2,] -4.2148 5.9607 3.2257 -0.1355 175.7770 -21.0909</p></li><li id="list-item-39"><p id="p-74">[3,] 6.3730 3.2257 23.8617 1.3721 -133.4962 -43.6292</p></li><li id="list-item-40"><p id="p-75">[4,] -0.0049 -0.1355 1.3721 46.2903 370.1770 -59.0136</p></li><li id="list-item-41"><p id="p-76">[5,] -103.8781 175.7770 -133.4962 370.1770 27963.1634 604.6203</p></li><li id="list-item-42"><p id="p-77">[6,] -114.4456 -21.0909 -43.6292 -59.0136 604.6203 872.3525</p></li></ul><p id="p-78">To convert this covariance matrix into a correlation matrix, we suggest using the following command:</p><ul class="list-simple " id="list-12"><li id="list-item-43"><p id="p-79">COR_TraitGenetic &lt;- cov2cor(COV_TraitGenetic)</p></li><li id="list-item-44"><p id="p-80">COR_TraitGenetic</p></li><li id="list-item-45"><p id="p-81">PH FL FE NS SY NP</p></li><li id="list-item-46"><p id="p-82">[1,] 1.000000e+00 -0.215077873 0.16254014 -8.972603e-05 -0.07739239 -0.4827480</p></li><li id="list-item-47"><p id="p-83">[2,] -2.150779e-01 1.000000000 0.27047346 -8.157286e-03 0.43054670 -0.2924829</p></li><li id="list-item-48"><p id="p-84">[3,] 1.625401e-01 0.270473457 1.00000000 4.128479e-02 -0.16342747 -0.3023991</p></li><li id="list-item-49"><p id="p-85">[4,] -8.972603e-05 -0.008157286 0.04128479 1.000000e+00 0.32536534 -0.2936710</p></li><li id="list-item-50"><p id="p-86">[5,] -7.739239e-02 0.430546701 -0.16342747 3.253653e-01 1.00000000 0.1224175</p></li><li id="list-item-51"><p id="p-87">[6,] -4.827480e-01 -0.292482897 -0.30239914 -2.936710e-01 0.12241752 1.0000000</p></li></ul><p id="p-88">Here there are no high correlations (<em>i.e.</em>, greater than 0.5). In a similar way, we obtained the residual covariance matrix:</p><ul class="list-simple " id="list-13"><li id="list-item-52"><p id="p-89">COV_ResGenetic &lt;- fm$vare</p></li><li id="list-item-53"><p id="p-90">COV_ResGenetic</p></li><li id="list-item-54"><p id="p-91">PH FL FE NS SY NP</p></li><li id="list-item-55"><p id="p-92">[1,] 65.2173 -2.7253 -12.3196 20.9669 880.3571 -71.9332</p></li><li id="list-item-56"><p id="p-93">[2,] -2.7253 11.6654 10.6847 -7.8144 182.0011 -10.3907</p></li><li id="list-item-57"><p id="p-94">[3,] -12.3196 10.6847 40.1432 -20.4498 -436.0834 -31.4401</p></li><li id="list-item-58"><p id="p-95">[4,] 20.9669 -7.8144 -20.4498 54.0458 375.0032 -70.7700</p></li><li id="list-item-59"><p id="p-96">[5,] 880.3571 182.0011 -436.0834 375.0032 60960.1184 2058.4876</p></li><li id="list-item-60"><p id="p-97">[6,] -71.9332 -10.3907 -31.4401 -70.7700 2058.4876 670.1587</p></li></ul><p id="p-98">For demonstration purposes, we will only extract the first 6 predictions for the 6 evaluated traits. We also plotted the observed values against the predicted values for each trait, as follows (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>):</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed and predicted values for trait FL resulting from fitting the model with the BME function to the Mada dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174096351" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Observed and predicted values for trait FL resulting from fitting the model with the BME function to the Mada dataset.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1355/F1.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1355/F1.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493206" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-99" class="first-child">Observed and predicted values for trait FL resulting from fitting the model with the BME function to the Mada dataset.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-14"><li id="list-item-61"><p id="p-100">plot(fm, trait = ′FL′)</p></li></ul><p id="p-101">Since the code provided above is only appropriate for parameter estimation, now we provide the code required to evaluate the prediction accuracy using the BME() function. For this reason, first we built the random CV strategy with 10 random partitions, each with TRN = 80% and TST = 20%, using the following code:</p><ul class="list-simple " id="list-15"><li id="list-item-62"><p id="p-102">pheno &lt;- data.frame(GID = phenoMada[, 1], Response = phenoMada[, 2])</p></li><li id="list-item-63"><p id="p-103">CrossV &lt;- CV.RandomPart(pheno, NPartitions = 10, PTesting = 0.2, set_seed = 123)</p></li><li id="list-item-64"><p id="p-104">Finally, we implemented the CV strategy as:</p></li><li id="list-item-65"><p id="p-105">pm &lt;- BME(Y = Y, Z1 = Z.G, nIter = 1250, burnIn = 500, thin = 2, bs = 50, testingSet = CrossV)</p></li></ul><p id="p-106">In the summary we show the average predictions of the 10 random partitions implemented under average Pearson's correlation (APC) and MAAPE:</p><ul class="list-simple " id="list-16"><li id="list-item-66"><p id="p-107">summary(pm)</p></li><li id="list-item-67"><p id="p-108">Environment Trait Pearson SE_Pearson MAAPE SE_MAAPE</p></li><li id="list-item-68"><p id="p-109">1 FE 0.3895 0.0847 0.7264 0.0313</p></li><li id="list-item-69"><p id="p-110">2 FL 0.1758 0.0767 0.7751 0.0248</p></li><li id="list-item-70"><p id="p-111">3 NP 0.3920 0.1192 0.7070 0.0347</p></li><li id="list-item-71"><p id="p-112">4 NS 0.4281 0.0879 0.7604 0.0364</p></li><li id="list-item-72"><p id="p-113">5 PH 0.5612 0.0767 0.7285 0.0542</p></li><li id="list-item-73"><p id="p-114">6 SY 0.0242 0.0623 0.7436 0.0255</p></li></ul><p id="p-115">Here we see that the best prediction in terms of APC was found in trait PH (0.5612), while the worst was in trait SY (0.0242). However, in terms of MAAPE, the best prediction was observed in trait NP (0.7070), while the worst was found in trait FL (0.7751). With the boxplot(pm) function, we created a plot summarizing the predictions in terms of Pearson’s correlation, but if users want this plot in MAAPE terms, they need to use the following code: boxplot(pm, select=“MAAPE”) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplot of Pearson&#x2019;s correlation of the testing sets for each trait of the Mada data-set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174096351" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplot of Pearson&#x2019;s correlation of the testing sets for each trait of the Mada data-set.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1355/F2.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1355/F2.medium.gif" width="440" height="206"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493178" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-116" class="first-child">Boxplot of Pearson’s correlation of the testing sets for each trait of the Mada data-set.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-17"><li id="list-item-74"><p id="p-117">boxplot(pm)</p></li></ul><p id="p-118">It is important to point out that the BME function can be used with only 1 testing set that can be defined by the user, as shown in the following example:</p><ul class="list-simple " id="list-18"><li id="list-item-75"><p id="p-119">CrossV1 &lt;-sample(nrow(Y),15)</p></li><li id="list-item-76"><p id="p-120">pm &lt;- BME(Y = Y, Z1 = Z.G, nIter = 1250, burnIn = 500, thin = 2, bs = 50,testingSet = CrossV1)</p></li></ul><p id="p-121">Next we summarize the prediction accuracy:</p><ul class="list-simple " id="list-19"><li id="list-item-77"><p id="p-122">summary(pm)</p></li><li id="list-item-78"><p id="p-123">Environment Trait Pearson SE_Pearson MAAPE SE_MAAPE</p></li><li id="list-item-79"><p id="p-124">1 NA FE 0.1181 NaN 0.7765 NaN</p></li><li id="list-item-80"><p id="p-125">2 NA FL -0.1222 NaN 0.8207 NaN</p></li><li id="list-item-81"><p id="p-126">3 NA NP 0.6937 NaN 0.6813 NaN</p></li><li id="list-item-82"><p id="p-127">4 NA NS 0.4636 NaN 0.7098 NaN</p></li><li id="list-item-83"><p id="p-128">5 NA PH 0.7362 NaN 0.7949 NaN</p></li><li id="list-item-84"><p id="p-129">6 NA SY -0.0079 NaN 0.8251 NaN</p></li></ul><p id="p-130">Since only one training set and one testing set were used, the standard errors for both metrics appear with NaN, given that it is not possible to calculate the standard error because only one testing set is available.</p></div><div id="sec-16" class="subsection"><h3>The Bayesian Multi-Trait and Multi-Environment (BMTME) function</h3><p id="p-131">This example illustrates how to fit a model with multiple traits and multiple environments. To do this, use the Maize dataset; first, load the data using the following function:</p><ul class="list-simple " id="list-20"><li id="list-item-85"><p id="p-132">data(“MaizeToy”)</p></li></ul><p id="p-133">Next, order the dataset, rename the rows of the phenotypic dataset and print the first six observations of the data in order to see the structure required of the data, which consists of a first column that includes the lines, a second column that includes the environments and third, fourth and fifth columns that correspond to the traits under study.</p><ul class="list-simple " id="list-21"><li id="list-item-86"><p id="p-134">phenoMaizeToy&lt;-(phenoMaizeToy[order(phenoMaizeToy$Env,phenoMaizeToy$Line),])</p></li><li id="list-item-87"><p id="p-135">rownames(phenoMaizeToy)=1:nrow(phenoMaizeToy)</p></li><li id="list-item-88"><p id="p-136">head(phenoMaizeToy)</p></li><li id="list-item-89"><p id="p-137">Line Env Yield ASI PH</p></li><li id="list-item-90"><p id="p-138">1 CKDHL0008 EBU 6.88 2.7 226</p></li><li id="list-item-91"><p id="p-139">2 CKDHL0039 EBU 6.85 1.3 239</p></li><li id="list-item-92"><p id="p-140">3 CKDHL0042 EBU 6.37 2.3 238</p></li><li id="list-item-93"><p id="p-141">4 CKDHL0050 EBU 4.98 3.1 239</p></li><li id="list-item-94"><p id="p-142">5 CKDHL0060 EBU 7.07 1.4 242</p></li><li id="list-item-95"><p id="p-143">6 CKDHL0063 EBU 8.62 2.3 250</p></li><li id="list-item-96"><p id="p-144">7 CKDHL0069 EBU 5.16 1.0 248</p></li><li id="list-item-97"><p id="p-145">8 CKDHL0072 EBU 5.77 1.7 227</p></li></ul><p id="p-146">This step is very important for avoiding an incorrect estimation process. Then the design matrices for the line effects, the environment and the genotype×environment interaction are generated:</p><ul class="list-simple " id="list-22"><li id="list-item-98"><p id="p-147">LG &lt;- cholesky(genoMaizeToy)</p></li><li id="list-item-99"><p id="p-148">ZG &lt;- model.matrix(∼0 + as.factor(phenoMaizeToy$Line))</p></li><li id="list-item-100"><p id="p-149">Z.G &lt;- ZG %*% LG</p></li><li id="list-item-101"><p id="p-150">Z.E &lt;- model.matrix(∼0 + as.factor(phenoMaizeToy$Env))</p></li><li id="list-item-102"><p id="p-151">ZEG &lt;- model.matrix(∼0 + as.factor(phenoMaizeToy$Line):as.factor(phenoMaizeToy$Env))</p></li><li id="list-item-103"><p id="p-152">G2 &lt;- kronecker(diag(length(unique(phenoMaizeToy$Env))), data.matrix(genoMaizeToy))</p></li><li id="list-item-104"><p id="p-153">LG2 &lt;- cholesky(G2)</p></li><li id="list-item-105"><p id="p-154">Z.EG &lt;- ZEG %*% LG2</p></li><li id="list-item-106"><p id="p-155">Y &lt;- as.matrix(phenoMaizeToy[, -c(1, 2)])</p></li></ul><p id="p-156">Finally, the following command is used to fit the model:</p><ul class="list-simple " id="list-23"><li id="list-item-107"><p id="p-157">fm &lt;- BMTME(Y = Y, X = Z.E, Z1 = Z.G, Z2 = Z.EG, nIter =15000, burnIn =10000, thin = 2,bs = 50)</p></li></ul><p id="p-158">We used the names() function to see all the things that can be extracted after fitting a model with the BMTME function.</p><ul class="list-simple " id="list-24"><li id="list-item-108"><p id="p-159">names(fm)</p></li><li id="list-item-109"><p id="p-160">[1] “Y” “nIter” “burnIn” “thin” “dfe” “Se” “yHat” [8] “SD.yHat” “beta” “SD.beta” “b1” “b2” “vare” “SD.vare”</p></li><li id="list-item-110"><p id="p-161">[15] “varEnv” “SD.varEnv” “varTrait” “SD.varTrait” “NAvalues”</p></li></ul><p id="p-162">We can extract the predicted and observed values, the random effects of lines, of lines×trait, lines×environment×trait, as well as the genetic covariances between traits and environments, and the residual covariance between traits. To extract the matrix of covariances between traits, we used the following:</p><ul class="list-simple " id="list-25"><li id="list-item-111"><p id="p-163">COV_TraitGenetic &lt;- fm$varTrait</p></li><li id="list-item-112"><p id="p-164">COV_TraitGenetic</p></li><li id="list-item-113"><p id="p-165">Yield ASI PH</p></li><li id="list-item-114"><p id="p-166">[1,] 0.0956 -0.0027 0.9642</p></li><li id="list-item-115"><p id="p-167">[2,] -0.0027 0.0654 0.1893</p></li><li id="list-item-116"><p id="p-168">[3,] 0.9642 0.1893 23.0647</p></li></ul><p id="p-169">To convert this covariance matrix between traits into a correlation matrix, we used the following command:</p><ul class="list-simple " id="list-26"><li id="list-item-117"><p id="p-170">COR_TraitGenetic &lt;- cov2cor(COV_TraitGenetic)</p></li><li id="list-item-118"><p id="p-171">COR_TraitGenetic</p></li><li id="list-item-119"><p id="p-172">Yield ASI PH</p></li><li id="list-item-120"><p id="p-173">[1,] 1.00000000 -0.03414648 0.6493282</p></li><li id="list-item-121"><p id="p-174">[2,] -0.03414648 1.00000000 0.1541302</p></li><li id="list-item-122"><p id="p-175">[3,] 0.64932822 0.15413023 1.0000000</p></li></ul><p id="p-176">Next we show how to extract the matrix of genetic covariance between the environments.</p><ul class="list-simple " id="list-27"><li id="list-item-123"><p id="p-177">COV_EnvGenetic &lt;- fm$varEnv</p></li><li id="list-item-124"><p id="p-178">COV_EnvGenetic</p></li><li id="list-item-125"><p id="p-179">EBU KAK KTI</p></li><li id="list-item-126"><p id="p-180">[1,] 308.6270 258.9984 285.3723</p></li><li id="list-item-127"><p id="p-181">[2,] 258.9984 237.9283 264.3881</p></li><li id="list-item-128"><p id="p-182">[3,] 285.3723 264.3881 324.4536</p></li></ul><p id="p-183">Finally, we show how to extract the residual covariance matrix between traits.</p><ul class="list-simple " id="list-28"><li id="list-item-129"><p id="p-184">COV_ResGenetic &lt;- fm$vare</p></li><li id="list-item-130"><p id="p-185">COV_ResGenetic</p></li><li id="list-item-131"><p id="p-186">Yield ASI PH</p></li><li id="list-item-132"><p id="p-187">[1,] 0.5282 -0.0955 3.3701</p></li><li id="list-item-133"><p id="p-188">[2,] -0.0955 0.4837 -1.5896</p></li><li id="list-item-134"><p id="p-189">[3,] 3.3701 -1.5896 106.5525</p></li></ul><p id="p-190">The observed and predicted values of each trait can be plotted using the following plot(), function (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>):</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed and predicted values for trait Yield resulting from fitting the model with the BMTME function to the Maize dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174096351" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Observed and predicted values for trait Yield resulting from fitting the model with the BMTME function to the Maize dataset.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1355/F3.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1355/F3.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493119" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-191" class="first-child">Observed and predicted values for trait Yield resulting from fitting the model with the BMTME function to the Maize dataset.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-29"><li id="list-item-135"><p id="p-192">plot(fm, trait=′Yield′)</p></li></ul><p id="p-193">We provide the corresponding code for the fivefold CV strategy and its implementation with the BMTME function:</p><ul class="list-simple " id="list-30"><li id="list-item-136"><p id="p-194">pheno &lt;- data.frame(GID = phenoMaizeToy[, 1], Env = phenoMaizeToy[, 2], Response = phenoMaizeToy[, 3])</p></li><li id="list-item-137"><p id="p-195">CrossV &lt;- CV.KFold(pheno, DataSetID = ′GID′, K = 5, set_seed = 123)</p></li><li id="list-item-138"><p id="p-196">pm &lt;- BMTME(Y = Y, X = Z.E, Z1 = Z.G, Z2 = Z.EG, nIter = 1250, burnIn = 500, thin = 2,bs = 50, testingSet = CrossV)</p></li></ul><p id="p-197">With the summary we obtained the average predictions in terms of Pearson’s correlation and MAAPE for each trait × environment combination.</p><ul class="list-simple " id="list-31"><li id="list-item-139"><p id="p-198">summary(pm)</p></li><li id="list-item-140"><p id="p-199">Environment Trait Pearson SE_Pearson MAAPE SE_MAAPE</p></li><li id="list-item-141"><p id="p-200">1 EBU ASI 0.4136 0.1997 0.3128 0.0369</p></li><li id="list-item-142"><p id="p-201">2 EBU PH 0.2141 0.1227 0.0553 0.0054</p></li><li id="list-item-143"><p id="p-202">3 EBU Yield 0.1064 0.2358 0.1554 0.0153</p></li><li id="list-item-144"><p id="p-203">4 KAK ASI 0.1711 0.3141 0.5771 0.0375</p></li><li id="list-item-145"><p id="p-204">5 KAK PH 0.5918 0.1907 0.0462 0.0097</p></li><li id="list-item-146"><p id="p-205">6 KAK Yield 0.6233 0.1642 0.1161 0.0162</p></li><li id="list-item-147"><p id="p-206">7 KTI ASI 0.0064 0.1436 0.3421 0.0300</p></li><li id="list-item-148"><p id="p-207">8 KTI PH 0.2599 0.2212 0.0631 0.0103</p></li><li id="list-item-149"><p id="p-208">9 KTI Yield 0.6077 0.1241 0.1390 0.0116</p></li></ul><p id="p-209">The summary information is given with the following code using the boxplot(pm, select=”MAAPE”, las = 2), where we added the parameter las = 2, to show the labels vertically and be able to distinguish the complete names (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MAAPE of the testing sets for each trait-environment combination of the Maize dataset using the BMTME function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174096351" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;MAAPE of the testing sets for each trait-environment combination of the Maize dataset using the BMTME function.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1355/F4.medium.gif" width="440" height="227"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1355/F4.medium.gif" width="440" height="227"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492959" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-210" class="first-child">MAAPE of the testing sets for each trait-environment combination of the Maize dataset using the BMTME function.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-32"><li id="list-item-150"><p id="p-211">boxplot(pm, select=”MAAPE”, las = 2)</p></li></ul></div><div id="sec-17" class="subsection"><h3>The BMORS Function</h3><p id="p-212">Since the BMORS function is only appropriate for evaluating prediction performance (but not for parameter estimation), we provide the required R script for evaluating the prediction performance of multiple trait and multiple environment data with a random CV strategy. Since we use the same dataset that was used to illustrate the BMTME function, we only provide the R code for building the predictor (ETA), the CV strategy and for implementing the BMORS function. To build the linear predictor, we used the following R code:</p><ul class="list-simple " id="list-33"><li id="list-item-151"><p id="p-213">ETA &lt;- list(Env = list(X = Z.E, model = “BRR”), Gen = list(X = Z.G, model = “BRR”), EnvGen = list(X = Z.EG, model = “BRR”))</p></li></ul><p id="p-214">Next, we provide the R code for implementing the random CV strategy:</p><ul class="list-simple " id="list-34"><li id="list-item-152"><p id="p-215">CrossValidation &lt;-CV.RandomPart(pheno, NPartitions = 10, PTesting = 0.2, set_seed = 123)</p></li></ul><p id="p-216">Finally, the model will be implemented with 15,000 iterations, of which 10,000 will be used as burn-in to fit the model. Below we show the resulting predictive performance using the summary() function. The information = ′complete′ command inside the summary() function shows the prediction performance (in terms of Pearson’s correlation and MAAPE) for all partitions implemented, but we only show the first 20 observations.</p><ul class="list-simple " id="list-35"><li id="list-item-153"><p id="p-217">pm &lt;- BMORS(Y, ETA = ETA, nIter = 15000, burnIn = 10000, thin = 2, progressBar = TRUE, testingSet = CrossValidation, digits = 4)</p></li><li id="list-item-154"><p id="p-218">head(summary(pm, information = ′complete′), 20)</p></li><li id="list-item-155"><p id="p-219">Environment Trait Partition Pearson MAAPE</p></li><li id="list-item-156"><p id="p-220">1 EBU Yield 1 0.4815 0.1693</p></li><li id="list-item-157"><p id="p-221">2 EBU Yield 2 0.1231 0.1170</p></li><li id="list-item-158"><p id="p-222">3 EBU Yield 3 0.2922 0.2016</p></li><li id="list-item-159"><p id="p-223">4 EBU Yield 4 0.5016 0.2224</p></li><li id="list-item-160"><p id="p-224">5 EBU Yield 5 -0.1674 0.2701</p></li><li id="list-item-161"><p id="p-225">6 EBU Yield 6 0.1033 0.1774</p></li><li id="list-item-162"><p id="p-226">7 EBU Yield 7 0.6086 0.0919</p></li><li id="list-item-163"><p id="p-227">8 EBU Yield 8 0.6689 0.1878</p></li><li id="list-item-164"><p id="p-228">9 EBU Yield 9 0.3322 0.1068</p></li><li id="list-item-165"><p id="p-229">10 EBU Yield 10 0.1161 0.1966</p></li><li id="list-item-166"><p id="p-230">11 EBU ASI 1 0.3506 0.3080</p></li><li id="list-item-167"><p id="p-231">12 EBU ASI 2 0.2944 0.2764</p></li><li id="list-item-168"><p id="p-232">13 EBU ASI 3 0.0694 0.4862</p></li><li id="list-item-169"><p id="p-233">14 EBU ASI 4 0.9157 0.3902</p></li><li id="list-item-170"><p id="p-234">15 EBU ASI 5 0.7654 0.2694</p></li><li id="list-item-171"><p id="p-235">16 EBU ASI 6 0.7956 0.2843</p></li><li id="list-item-172"><p id="p-236">17 EBU ASI 7 0.8630 0.3009</p></li><li id="list-item-173"><p id="p-237">18 EBU ASI 8 -0.1160 0.3421</p></li><li id="list-item-174"><p id="p-238">19 EBU ASI 9 0.1343 0.2799</p></li><li id="list-item-175"><p id="p-239">20 EBU ASI 10 0.8346 0.2041</p></li></ul><p id="p-240">The summary of the predictions is obtained with the following code:</p><ul class="list-simple " id="list-36"><li id="list-item-176"><p id="p-241">summary(pm)</p></li><li id="list-item-177"><p id="p-242">Environment Trait Pearson SE_Pearson MAAPE SE_MAAPE</p></li><li id="list-item-178"><p id="p-243">1 EBU Yield 0.3060 0.0835 0.1741 0.0175</p></li><li id="list-item-179"><p id="p-244">2 EBU ASI 0.4907 0.1219 0.3142 0.0245</p></li><li id="list-item-180"><p id="p-245">3 EBU PH 0.3486 0.1612 0.0482 0.0050</p></li><li id="list-item-181"><p id="p-246">4 KAK Yield 0.3075 0.1019 0.0965 0.0079</p></li><li id="list-item-182"><p id="p-247">5 KAK ASI 0.0064 0.1296 0.5733 0.0449</p></li><li id="list-item-183"><p id="p-248">6 KAK PH 0.6010 0.0650 0.0396 0.0037</p></li><li id="list-item-184"><p id="p-249">7 KTI Yield 0.3816 0.0992 0.1639 0.0104</p></li><li id="list-item-185"><p id="p-250">8 KTI ASI 0.0278 0.0973 0.2878 0.0198</p></li><li id="list-item-186"><p id="p-251">9 KTI PH 0.3835 0.0753 0.0591 0.0063</p></li></ul><p id="p-252">To create a graph with a summary of the predictions in terms of Pearson’s correlation and in terms of MAAPE, we used the plot() function (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Because the names are composed of the evaluated traits and environments, we added the parameter las = 2 to show the labels in a vertical way and to distinguish the complete names of the trait-environment combinations. In addition, we used the par() function and the mar parameter to modify the margins of the graph.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average Pearson&#x2019;s correlation of the testing sets for each trait-environment combination of the Maize dataset using the BMORS function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174096351" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average Pearson&#x2019;s correlation of the testing sets for each trait-environment combination of the Maize dataset using the BMORS function.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1355/F5.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/5/1355/F5.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492915" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-253" class="first-child">Average Pearson’s correlation of the testing sets for each trait-environment combination of the Maize dataset using the BMORS function.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-37"><li id="list-item-187"><p id="p-254">par(mar = c(6,4,2,1))</p></li><li id="list-item-188"><p id="p-255">plot(pm, las = 2)</p></li></ul><p id="p-256"><a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> shows that the lowest average Pearson’s correlation obtained was observed in the ASI_KAK and ASI_KTI trait-environment combinations, while the highest average Pearson’s correlation was obtained in the PH_KAK trait-environment combination. It is possible to create a boxplot with the results of the MAAPE, using the following command (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>):</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplot of MAAPE of the testing sets for each trait-environment combination of the Maize dataset using the BMORS function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174096351" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplot of MAAPE of the testing sets for each trait-environment combination of the Maize dataset using the BMORS function.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1355/F6.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/5/1355/F6.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493120" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-257" class="first-child">Boxplot of MAAPE of the testing sets for each trait-environment combination of the Maize dataset using the BMORS function.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-38"><li id="list-item-189"><p id="p-258">boxplot(pm, select =”MAAPE”, las = 2)</p></li></ul><p id="p-259"><a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> shows that the lowest MAAPE was for PH_KAK (best prediction), while the highest MAAPE was for the ASI_KAK trait-environment combination (worst prediction).</p></div><div id="sec-18" class="subsection"><h3>The BMORS_ENV Function</h3><p id="p-260">This function is useful for predicting whole environments using the remaining environments as training. Next we provide the R code for evaluating the prediction performance of the same maize dataset, but using the KAK environment as training and the KTI and EBU environments as testing. Two important things to point out for using this function are: (a) that we provided not only the matrix of response variables, but also a data.frame that contains, in the first column, the names of the environments followed by information on all response variables, and (b) we did not create a separate file for specifying the training and testing individuals; we only specified in testingEnv which environments are used as testing; the remaining environments are used by default as training, as shown below.</p><ul class="list-simple " id="list-39"><li id="list-item-190"><p id="p-261">dataset &lt;- phenoMaizeToy[, 2:5]</p></li><li id="list-item-191"><p id="p-262">pm &lt;- BMORS_Env(dataset, testingEnv = <strong>c</strong>(′KTI′, ′EBU′), ETA = ETA, covModel = ′BayesB′, nIter = 15000, burnIn = 10000, thin = 2, progressBar = TRUE, digits = 3)</p></li><li id="list-item-192"><p id="p-263">summary(pm)</p></li><li id="list-item-193"><p id="p-264">Environment Trait Pearson MAAPE</p></li><li id="list-item-194"><p id="p-265">1 KTI Yield 0.4114 0.1344</p></li><li id="list-item-195"><p id="p-266">2 KTI ASI 0.0334 0.4304</p></li><li id="list-item-196"><p id="p-267">3 KTI PH 0.2091 0.0760</p></li><li id="list-item-197"><p id="p-268">4 EBU Yield 0.2775 0.1722</p></li><li id="list-item-198"><p id="p-269">5 EBU ASI -0.0586 0.3825</p></li><li id="list-item-199"><p id="p-270">6 EBU PH 0.0533 0.0933</p></li></ul><p id="p-271">For this example, we specified that covModel = ′BayesB′, which means that the Bayesian BayesB model will be implemented for the second stage of the model where it is implemented (equation 3). In covModel, in addition to Bayesian Ridge regression (BRR) and BayesB, we can also implement BayesA, BayesC and Bayesian Lasso (BL); however, the BRR model is implemented by default.To create a graph with Pearson’s correlation or the MAAPE index, we used the barplot() function, as shown below (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>):</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average Pearson&#x2019;s correlations of the testing sets for each trait-environment combination of the Maize dataset using the BMORS_Env function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174096351" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average Pearson&#x2019;s correlations of the testing sets for each trait-environment combination of the Maize dataset using the BMORS_Env function.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1355/F7.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/5/1355/F7.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492952" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-272" class="first-child">Average Pearson’s correlations of the testing sets for each trait-environment combination of the Maize dataset using the BMORS_Env function.</p><div class="sb-div caption-clear"></div></div></div><ul class="list-simple " id="list-40"><li id="list-item-200"><p id="p-273">barplot(pm)</p></li></ul><p id="p-274"><a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a> shows that the lowest Pearson’s correlation obtained was in the ASI_EBU trait-environment combination, while the highest Pearson’s correlation was obtained in the Yield_KTI trait-environment combination.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-275">As mentioned in the introduction, we propose a Bayesian R package for implementing multi-environment and multi-trait and multi-environment analysis for parameter estimation and for evaluating prediction accuracy. We illustrate the four main functions [BME(), BMTME(), BMORS() and BMORS_Env()] of the BMTME package with real toy datasets starting from the type and preprocessing required to make correct use of each of these datasets for parameter estimation and for evaluating prediction performance. It is important to point out that one advantage of the BME and BMTME functions is that, in addition to being used to evaluate the prediction accuracy, they can also be used for parameter estimation, which allows estimating the random effects (lines, lines×environments for each trait) and variance-covariance matrices of genetic (for traits and environments) and residual (for traits) effects. The BMORS() and BMORS_Env() functions are not useful for obtaining parameter estimates of covariances between traits and environments because they implement univariate analysis at both stages. However, they have two important advantages: (a) they allow implementing even more complex predictors than the one specified in equation (1), which modifies the ETA list used to create the predictor, and (b) the computational resources required to implement it are much less than those needed by the BMTME() function for implementing multi-trait and multi-environment data. This last point is observed in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> where the implementation time for the Mada and Maize datasets is reported. The figure shows that in the Mada dataset, the BMORS model was more than 15 times faster than the BMTME model (25.246/ 1.621= 15.572), while in the Maize dataset, the BMORS model was more than 37 times faster than the BMTME model (25.668/ 0.692= 37.093); these results were obtained with 10000 iterations of the Gibbs sampler. The BMTME R package provides very synthetic summaries (tables and plots) of the prediction accuracies, which are ready to be interpreted and used to write the results in a manuscript. Additionally, we provide three types of cross-validations useful for breeders that are implemented in this R package, which is very simple to use and implement.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Implementation time in minutes between the BMTME and BMORS models under the Mada and Maize datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174096351" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Implementation time in minutes between the BMTME and BMORS models under the Mada and Maize datasets.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1355/F8.medium.gif" width="440" height="254"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/5/1355/F8.medium.gif" width="440" height="254"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1355/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493132" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-276" class="first-child">Implementation time in minutes between the BMTME and BMORS models under the Mada and Maize datasets.</p><div class="sb-div caption-clear"></div></div></div><p id="p-277">The main disadvantage of the BME() and BMTME() functions of the BMTME R package is that the computational resources required for their implementation are very demanding; fortunately, the parameter estimates involved are stabilized very quickly even with few iterations. The toy examples used in this article are for illustration purposes and to help users follow, as easily as possible, the necessary steps for running the different processes. Comparing them with other software of similar type is not possible, as no similar software for simultaneously fitting multi-trait multi-environment is currently available. For example, the MTM (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">de los Campos and Grüneberg 2016</a>) is an efficient Bayesian multi-trait software but is not multi-trait and multi-environment. Future research on benchmarking the BME() and BMTME() functions of the BMTME package with other potential software to be developed in terms of computing time for processing large datasets should be performed. However, the BMORS() and BMORS_Env() functions that also belong to the BMTME R package are very efficient in terms of computational resources, which gives the user an alternative option for performing this type of analyses.</p><p id="p-278">It is important to point out that the proposed BMTME package is different from existing multi-trait analysis software such as ASREML (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gilmour <em>et al.</em>, 1995</a>), sommer (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Covarrubias-Pazaran 2016</a>), BGGE (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Granato <em>et al.</em>, 2018</a>) and MCMCglmm (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Hadfield <em>et al.</em>, 2010</a>). In addition to taking into account variance-covariance matrices of traits (genetic and residual), it also takes into account the genetic covariance (correlation) between environments, which is estimated from the data. This can help improve parameter estimates and prediction accuracy when the degree of correlation between traits is moderate or high.</p><p id="p-279">Multi-trait models are preferred over univariate-trait models because they have the following advantages: (a) they produce higher prediction accuracy because they have more information (direct or indirect) and better data connectedness (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Colleau <em>et al.</em>, 2009</a>); (b) they improve index selection because optimal weight factors can be obtained for the total merit index (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Colleau <em>et al.</em>, 2009</a>); and (c) they allow obtaining more precise genetic and residual covariances and incorporating them into expected breeding value (EBV) estimates for across-location, across-country or across-region evaluations (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Thompson and Meyer 1986</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schaeffer 2001</a>).</p><p id="p-280">Note that the two datasets used for illustrating the main functions of the BMTME R package are datasets with few lines (toy datasets) with the main intention that users interested in using the package can obtain results very quickly and practice using the software. However, the structure of the data are exactly the same as the structure of the data produced in plant breeding programs. The two toy datasets are included in the BMTME package to facilitate its implementation and allow users to practice using the R software.</p><p id="p-281">To conclude, this paper presents the R package BMTME which allows the implementation of multi-trait, multi-trait and multi-environment analysis for estimating parameters (genetic correlation between traits and environments, residual correlation between traits, random effects of lines and lines×environments) and evaluating the prediction accuracies of many traits simultaneously. We illustrate the implementation of the main functions (BME, BMTE and BMORS) of the R package with two toy real datasets that are very common in plant breeding programs. We provide details of the characteristics that each of the datasets must have, and show how to build the CV strategies available in the package, how to prepare the data to implement the main functions of the BMTME package, how to extract the parameter estimates and how to obtain the summary and plots of prediction accuracies resulting from the implemented CV strategy. The computing time of the BME() and BMTE() functions of the BMTME R package for large datasets is significantly more demanding (in terms of time) than for the toy examples used in this study.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-378">We acknowledge the financial support provided by the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) in Norway through NFR grant 267806. Financial support was also provided by the Bill &amp; Melinda Gates Foundation, as well as the CIMMYT CRP Global Maize Breeding Program that generated the data analyzed in this study.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX A – R CODE</h2><div class="section" id="sec-20"><h3>R code for example 1.</h3><p id="p-282">rm(list=ls())</p><p id="p-283">library(BMTME)</p><p id="p-284">data(“WheatMadaToy”)</p><p id="p-285">ls()</p><p id="p-286">phenoMada &lt;- (phenoMada[order(phenoMada$GID),])</p><p id="p-287">rownames(phenoMada)=1:nrow(phenoMada)</p><p id="p-288">head(phenoMada)</p><p id="p-289">LG &lt;- cholesky(genoMada)</p><p id="p-290">ZG &lt;- model.matrix(∼0 + as.factor(phenoMada$GID))</p><p id="p-291">Z.G &lt;- ZG %*% LG</p><p id="p-292">Y &lt;- as.matrix(phenoMada[, -c(1)])</p><p id="p-293">head(Y)</p><p id="p-294">##############Parameter estimation##############################</p><p id="p-295">fm &lt;- BME(Y = Y, Z1 = Z.G, nIter =15000, burnIn =10000, thin = 2, bs = 50)</p><p id="p-296">names(fm)</p><p id="p-297">head(fm$yHat)</p><p id="p-298">COV_TraitGenetic &lt;- fm$varTrait</p><p id="p-299">COV_TraitGenetic</p><p id="p-300">COR_TraitGenetic &lt;- cov2cor(COV_TraitGenetic)</p><p id="p-301">COR_TraitGenetic</p><p id="p-302">COV_ResGenetic &lt;- fm$vare</p><p id="p-303">COV_ResGenetic</p><p id="p-304">COR_ResGenetic &lt;- cov2cor(COV_ResGenetic)</p><p id="p-305">COR_ResGenetic</p><p id="p-306">head(fm$yHat)</p><p id="p-307">plot(fm,trait=”FL”)</p><p id="p-308">plot(fm,trait=”FE”)</p><p id="p-309">#############Evaluation of prediction performance#################</p><p id="p-310">pheno &lt;- data.frame(GID=phenoMada[, 1],</p><p id="p-311">Response = phenoMada[, 2])</p><p id="p-312">CrossV &lt;- CV.RandomPart(pheno, NPartitions = 10, PTesting = 0.2, set_seed = 123)</p><p id="p-313">pm &lt;- BME(Y = Y, Z1 = Z.G, nIter = 1250, burnIn = 500, thin = 2, bs = 50,</p><p id="p-314">testingSet = CrossV)</p><p id="p-315">summary(pm)</p><p id="p-316">boxplot(pm)</p><p id="p-317">boxplot(pm, select=”MAAPE”)</p><p id="p-318">CrossV1 &lt;-sample(nrow(Y),15)</p><p id="p-319">CrossV1</p><p id="p-320">pm &lt;- BME(Y = Y, Z1 = Z.G, nIter = 1250, burnIn = 500, thin = 2, bs = 50,</p><p id="p-321">testingSet = CrossV1)</p><p id="p-322">summary(pm)</p><p id="p-323">boxplot(pm)</p><p id="p-324">boxplot(pm, select=”MAAPE”)</p></div><div class="section" id="sec-21"><h3>R code for example 2.</h3><p id="p-325">rm(list=ls())</p><p id="p-326">library(BMTME)</p><p id="p-327">data(“MaizeToy”)</p><p id="p-328">phenoMaizeToy&lt;-(phenoMaizeToy[order(phenoMaizeToy$Env,phenoMaizeToy$Line),])</p><p id="p-329">rownames(phenoMaizeToy)=1:nrow(phenoMaizeToy)</p><p id="p-330">head(phenoMaizeToy,8)</p><p id="p-331">LG &lt;- cholesky(genoMaizeToy)</p><p id="p-332">ZG &lt;- model.matrix(∼0 + as.factor(phenoMaizeToy$Line))</p><p id="p-333">Z.G &lt;- ZG %*% LG</p><p id="p-334">Z.E &lt;- model.matrix(∼0 + as.factor(phenoMaizeToy$Env))</p><p id="p-335">ZEG &lt;- model.matrix(∼0 + as.factor(phenoMaizeToy$Line):as.factor(phenoMaizeToy$Env))</p><p id="p-336">G2 &lt;- kronecker(diag(length(unique(phenoMaizeToy$Env))), data.matrix(genoMaizeToy))</p><p id="p-337">LG2 &lt;- cholesky(G2)</p><p id="p-338">Z.EG &lt;- ZEG %*% LG2</p><p id="p-339">Y &lt;- as.matrix(phenoMaizeToy[, -c(1, 2)])</p><p id="p-340">fm &lt;- BMTME(Y = Y, X = Z.E, Z1 = Z.G, Z2 = Z.EG, nIter =1500, burnIn =1000, thin = 2,bs = 50)</p><p id="p-341">str(fm)</p><p id="p-342">COV_TraitGenetic &lt;- fm$varTrait</p><p id="p-343">COV_TraitGenetic</p><p id="p-344">COR_TraitGenetic &lt;- cov2cor(COV_TraitGenetic)</p><p id="p-345">COR_TraitGenetic</p><p id="p-346">COV_EnvGenetic &lt;- fm$varEnv</p><p id="p-347">COV_EnvGenetic</p><p id="p-348">COR_EnvGenetic &lt;- cov2cor(COV_EnvGenetic)</p><p id="p-349">COR_EnvGenetic</p><p id="p-350">COV_ResGenetic &lt;- fm$vare</p><p id="p-351">COV_ResGenetic</p><p id="p-352">COR_ResGenetic &lt;- cov2cor(COV_ResGenetic)</p><p id="p-353">COR_ResGenetic</p><p id="p-354">plot(fm, trait=′Yield′)</p><p id="p-355">plot(fm,trait=”ASI”)</p><p id="p-356">####Evaluation of prediction performance with BMTME###############</p><p id="p-357">pheno &lt;- data.frame(GID = phenoMaizeToy[, 1], Env = phenoMaizeToy[, 2],</p><p id="p-358">Response = phenoMaizeToy[, 3])</p><p id="p-359">CrossV &lt;- CV.KFold(pheno, DataSetID = ′GID′, K = 5, set_seed = 123)</p><p id="p-360">pm &lt;- BMTME(Y = Y, X = Z.E, Z1 = Z.G, Z2 = Z.EG, nIter = 1250, burnIn = 500, thin = 2,bs = 50, testingSet = CrossV)</p><p id="p-361">summary(pm)</p><p id="p-362">boxplot(pm, select =”MAAPE”, las = 2)</p><p id="p-363">####Evaluation of prediction performance with BMORS####################</p><p id="p-364">ETA &lt;- list(Env = list(X = Z.E, model = “BRR”), Gen = list(X = Z.G, model = ’BRR’), EnvGen = list(X = Z.EG, model = “BRR”))</p><p id="p-365">CrossValidation &lt;-CV.RandomPart(pheno, NPartitions = 10, PTesting = 0.2,set_seed = 123)</p><p id="p-366">pm &lt;- BMORS(Y, ETA = ETA, nIter = 15000, burnIn = 10000, thin = 2, progressBar = TRUE, testingSet = CrossValidation, digits = 4)</p><p id="p-367">head(summary(pm, information = ′complete′), 20)</p><p id="p-368">par(mar=c(6,4,2,1))</p><p id="p-369">plot(pm, las = 2)</p><p id="p-370">boxplot(pm, select =”MAAPE”, las = 2)</p><p id="p-371">###########Evaluation of prediction performance with BMORS_Env################################</p><p id="p-372">dataset &lt;- phenoMaizeToy[, 2:5]</p><p id="p-373">pm1 &lt;- BMORS_Env(dataset, testingEnv = c(’KTI’, ’EBU’), ETA = ETA, covModel = ’BayesB’, nIter = 1500, burnIn = 1000, thin = 2, </p><p id="p-374">progressBar = TRUE, digits = 3)</p><p id="p-375">summary(pm1)</p><p id="p-376">barplot(pm1)</p><p id="p-377">barplot(pm1,select=”MAAPE”)</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-379"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-14"><span class="received-label">Received </span>December 14, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-27"><span class="accepted-label">Accepted </span>February 27, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Montesinos-López <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.1" data-doi="10.1534/g3.118.200098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben Hassen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartholomé</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valè</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">T.-V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmadi</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite> <span class="cit-pub-date">2018</span> Genomic Prediction Accounting for Genotype by Environment Interaction Offers an Effective Framework for Breeding Simultaneously for Adaptation to an Abiotic Stress and Performance Under Normal Cropping Conditions in Rice. <em>G3: Genes|Genomes|Genetics</em>, <em>8</em>(July), 2319–2332.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200098%26rft_id%253Dinfo%253Apmid%252F29743189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNy8yMzE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.2" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G. D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype x environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.3" data-doi="10.1186/s12864-015-1266-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burstin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salloignon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinello</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnin-Robert</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic diversity and trait genomic prediction in a pea diversity panel.</span> BMC Genomics, 2015;16(1):105. pmid:25765216.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1266-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1266-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1266-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.4" data-doi="10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods. Genetics, Selection</span>. <abbr class="cit-jnl-abbrev">Evolution : GSE</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297–9686–43–26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%253F%253A%2BGSE%26rft.volume%253D43%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297%2526ndash%253B9686%2526ndash%253B43%2526ndash%253B26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.5" data-doi="10.1590/S0100-204X2013000600008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">A. F. N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">C. A. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multivariate analysis for the selection of eucalyptus clones destined for charcoal production.</span> <abbr class="cit-jnl-abbrev">Pesqui. Agropecu. Bras.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/S0100-204X2013000600008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPesqui.%2BAgropecu.%2BBras.%26rft.volume%253D48%26rft.spage%253D627%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS0100-204X2013000600008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S0100-204X2013000600008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colleau</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupassieux</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Simulating the potential of genomic selection in dairy cattle breeding.</span> <abbr class="cit-jnl-abbrev">Rencontres Recherches Ruminants</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRencontres%2BRecherches%2BRuminants%26rft.volume%253D16%26rft.spage%253D419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.7" data-doi="10.1371/journal.pone.0156744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Covarrubias-Pazaran</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite> <span class="cit-pub-date">2016</span> Genome-assisted prediction of quantitative traits using the R package sommer. PLoS ONE. 11(6): 1–15.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0156744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D11%26rft.spage%253De0156744%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.8" data-doi="10.3389/fpls.2018.01310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Covarrubias-Pazaran</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlautman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diaz-Garcia</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grygleski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polashock</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Multivariate GBLUP Improves Accuracy of Genomic Selection for Yield and Fruit Weight in Biparental Populations of <em>Vaccinium macrocarpon</em> Ait.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.01310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D1310%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.01310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.01310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.9" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1355.10"><div class="cit-metadata unstructured">de los Campos, G., and A. Grüneberg, 2016 MTM (Multiple-Trait Model) package [WWW Document]. URL <a href="http://quantgen.github.io/MTM/vignette.html">http://quantgen.github.io/MTM/vignette.html</a> (accessed 10.25.17).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.11" data-doi="10.1007/s10681-012-0722-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Oliveira</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilela de Resende</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva Santos</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortes Ferreira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fachardo Oliveira</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide selection in cassava.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">276</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-012-0722-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D187%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-012-0722-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-012-0722-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.12" data-doi="10.1534/g3.118.200435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Granato</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luna-Vázquez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models. Genes</span>. <abbr class="cit-jnl-abbrev">Genomes and Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3039</span>–<span class="cit-lpage">3047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200435</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomes%2Band%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200435%26rft_id%253Dinfo%253Apmid%252F30049744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvOS8zMDM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.13" data-doi="10.2307/2533274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Average Information REML: An Efficient Algorithm for Variance Parameter Estimation in Linear Mixed Models.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1440</span>–<span class="cit-lpage">1450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2533274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D51%26rft.spage%253D1440%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TQ15600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.14" data-doi="10.18637/jss.v033.i02"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2010</span> MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package. Journal of Statistical Software, 33(2): 1–22.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v033.i02</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BSTAT%2BSOFT%26rft.volume%253D33%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v033.i02%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v033.i02&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.15" data-doi="10.1139/G10-076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association and genomic selection in animal breeding.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">876</span>–<span class="cit-lpage">883</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/G10-076</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DHayes%26rft.auinit1%253DB.%26rft.volume%253D53%26rft.issue%253D11%26rft.spage%253D876%26rft.epage%253D883%26rft.atitle%253DGenome-wide%2Bassociation%2Band%2Bgenomic%2Bselection%2Bin%2Banimal%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1139%252FG10-076%26rft_id%253Dinfo%253Apmid%252F21076503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/G10-076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076503&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.16" data-doi="10.1093/bioinformatics/btw249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">i37</span>–<span class="cit-lpage">i43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw249%26rft_id%253Dinfo%253Apmid%252F27307640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27307640&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.17" data-doi="10.1007/s10681-015-1381-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diallel analysis of combining ability and heterosis for yield and yield components in rice by using positive loci.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-015-1381-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D205%26rft.spage%253D37%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-015-1381-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-015-1381-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.18" data-doi="10.1186/s12711-016-0192-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heuer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kühn</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of unordered categorical traits: an application to subpopulation assignment in German Warmblood horses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0192-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0192-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0192-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.19" data-doi="10.3835/plantgenome2016.12.0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemes da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.12.0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.12.0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.12.0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.20" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTM1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.21" data-doi="10.1038/hdy.2015.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.9%26rft_id%253Dinfo%253Apmid%252F25873147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25873147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlisle</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite> <span class="cit-pub-date">2012</span> Genomic selection for fruit quality traits in apple (<em>Malus × domestica</em> Borkh.). PloS One. 2012;7(5):e36674. pmid:22574211.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model. G3: Genes|Genomes|Genetics, 5(4): 569–82.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.24" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTM1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> Threshold models for genome-enabled prediction of ordinal categorical traits in plant breeding. G3: Genes| Genomes| Genetics, 5(1): 291–300.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.26" data-doi="10.1534/g3.115.021154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression. G3: Genes|Genomes|Genetics, 5(10): 2113–2126.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021154%26rft_id%253Dinfo%253Apmid%252F26290569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMjExMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xMzU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.27" data-doi="10.1534/g3.116.032359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-Lopez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-Lopez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toledo</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Hernandez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> A Genomic Bayesian Multi-trait and Multi-environment Model. G3: Genes|Genomes|Genetics, 6(9), 2725–2744.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032359%26rft_id%253Dinfo%253Apmid%252F27342738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yNzI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.28" data-doi="10.2134/agronj2018.06.0362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Suárez</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Multivariate Bayesian analysis of on-farm trials with multiple-trait and multiple-environment data.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2018.06.0362</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgron.%2BJ.%26rft.volume%253D111%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2018.06.0362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2018.06.0362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.29" data-doi="10.1534/g3.118.200728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Suárez</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Multi-trait, multi-environment deep learning modeling for genomic-enabled prediction of plant traits. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3829</span>–<span class="cit-lpage">3840</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D3829%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.118.200728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1355.30" data-doi="10.5772/intechopen.71521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luna-Vázquez</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite> <em>et al.</em>, <span class="cit-pub-date">2018</span><span class="cit-comment">c</span> <span class="cit-article-title">A Bayesian Multiple-Trait and Multiple-Environment Model Using the Matrix Normal Distribution</span>, <span class="cit-source">Physical Methods for Stimulation of Plant and Mushroom Development</span>, chap 13., edited by <span class="cit-ed"><span class="cit-name-surname">El-Esawi</span>  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-publ-name">Tanta University</span>, London Bridge Street London, United Kingdom.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5772/intechopen.71521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2BMethods%2Bfor%2BStimulation%2Bof%2BPlant%2Band%2BMushroom%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.5772%252Fintechopen.71521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5772/intechopen.71521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.31" data-doi="10.3168/jds.2011-5006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tooker</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multibreed genomic evaluations using purebred Holsteins, Jerseys, and Brown Swiss.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">5378</span>–<span class="cit-lpage">5383</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2011-5006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DOlson%26rft.auinit1%253DK.%2BM.%26rft.volume%253D95%26rft.issue%253D9%26rft.spage%253D5378%26rft.epage%253D5383%26rft.atitle%253DMultibreed%2Bgenomic%2Bevaluations%2Busing%2Bpurebred%2BHolsteins%252C%2BJerseys%252C%2Band%2BBrown%2BSwiss.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-5006%26rft_id%253Dinfo%253Apmid%252F22916944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-5006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22916944&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.32" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-Wide Regression and Prediction with the BGLR Statistical Package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1355.33"><div class="cit-metadata unstructured">Plummer, M., 2018 rjags: Bayesian Graphical Models using MCMC. URL <a href="https://CRAN.R-project.org/package=rjags">https://CRAN.R-project.org/package=rjags</a>. R package version 4–6.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1355.34"><div class="cit-metadata unstructured">R Core Team, 2018 R: A Language and Environment for Statistical Computing. Vienna, Austria. Retrieved from <a href="https://www.r-project.org/">https://www.r-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.35" data-doi="10.1007/s10681-010-0301-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection for durable stem rust resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">173</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-010-0301-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D179%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-010-0301-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-010-0301-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000289305300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.36" data-doi="10.1016/S0301-6226(00)00255-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Multiple trait international bull comparisons.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0301-6226(00)00255-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D69%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252800%252900255-4%26rft_id%253Dinfo%253Apmid%252F11337117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(00)00255-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11337117&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.37" data-doi="10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Advantages and limitations of multiple-trait genomic prediction for Fusarium head blight severity in hybrid wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122–017–3029–7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B017%2526ndash%253B3029%2526ndash%253B7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.38" data-doi="10.1371/journal.pgen.1004982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection and association mapping in rice (<em>Oryza sativa</em>): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004982</span> (erratum: PLoS Genet. 11: e1005350).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004982%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004982%26rft_id%253Dinfo%253Apmid%252F25689273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004982&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25689273&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1355.39"><div class="cit-metadata unstructured">Spyromitros-Xioufis, E., W. Groves, G. Tsoumakas, and I. Vlahavas, 2012 Multi-label classification methods for multi-target regression. arXiv preprint arXiv:1211.6581, 2012, 1159–1168. Cornell University Library.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.40" data-doi="10.1007/s10994-016-5546-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spyromitros-Xioufis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsoumakas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groves</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vlahavas</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite> <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multi-target regression via input space expansion: treating targets as inputs.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10994-016-5546-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D104%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10994-016-5546-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10994-016-5546-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1355.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatri</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-source"><em>An Introduction to Multivariate Statistics</em>.</span> <span class="cit-publ-name">Elsevier North Holland</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1355.42"><div class="cit-metadata unstructured">Stan Development Team, (2018). Stan Modeling Language Users Guide and Reference Manual, Version 2.18.0. <a href="http://mc-stan.org">http://mc-stan.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.43" data-doi="10.1007/s00122-013-2041-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction of dichotomous traits with Bayesian logistic models.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2041-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D1133%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2041-9%26rft_id%253Dinfo%253Apmid%252F23385660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2041-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23385660&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.44" data-doi="10.1016/0301-6226(86)90071-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">A review of theoretical aspects in the estimation of breeding values for multi-trait selection.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0301-6226(86)90071-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D15%26rft.spage%253D299%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0301-6226%252886%252990071-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0301-6226(86)90071-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.45" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.46" data-doi="10.1186/1471-2156-14-85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraft</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection in sugar beet breeding populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-14-85</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-14-85%26rft_id%253Dinfo%253Apmid%252F24047500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-14-85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24047500&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1355.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1355.47" data-doi="10.1038/srep19454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsunaga</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onogi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kajiya-Kanegae</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minamikawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A simulation-based breeding design that uses whole-genome prediction in tomato.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">19454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep19454</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D19454%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep19454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep19454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1355.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1339" title="Novel exc Genes Involved in Formation of the Tubular Excretory Canals of Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1371" title="TSD: A Computational Tool To Study the Complex Structural Variants Using PacBio Targeted Sequencing Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1355.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UW11eLwUUdqKD0kSpJw_vlFrCD2wTa9du8AQ0SHqs2w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1355" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1355" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1355" class="active" data-icon-position="" data-hide-link-title="0">An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492870" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-t9k6KDYzY-WSwenvqcQLGl3WaYqb-52N6-HSabqHvS8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1355.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1355" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Ub17jEMmDxm9t3gWDmhu6WYuJvVqSLYFx7ARzO3LGXU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1355" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1355.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492870" id="node492870--2" data-pisa="ggg;9/5/1355" data-pisa-master="ggg;g3.119.400126" data-apath="/ggg/9/5/1355.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Francisco Javier Luna-Vázquez</span>, <span class="highwire-citation-author" data-delta="3">Fernando H. Toledo</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="5">Morten Lillemo</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1355-1369;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400126 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492870--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías, (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Francisco Javier Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BJavier%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BJavier%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center, (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, CP 56230, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Morten Lillemo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Sciences, Norwegian University of Life Sciences, IHA/CIGENE, P.O. Box 5003, NO-1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BLillemo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lillemo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorten%2BLillemo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center, (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1355" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1355&amp;title=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1355&amp;title=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1355&amp;title=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1355&amp;text=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1355&amp;title=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1355&amp;t=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1355&amp;title=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1355&amp;title=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492870/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492870" id="node492870--3" data-pisa="ggg;9/5/1355" data-pisa-master="ggg;g3.119.400126" data-apath="/ggg/9/5/1355.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Francisco Javier Luna-Vázquez</span>, <span class="highwire-citation-author" data-delta="3">Fernando H. Toledo</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="5">Morten Lillemo</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1355-1369;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400126 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492870--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías, (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Francisco Javier Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BJavier%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BJavier%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center, (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, CP 56230, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Morten Lillemo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Sciences, Norwegian University of Life Sciences, IHA/CIGENE, P.O. Box 5003, NO-1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BLillemo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lillemo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorten%2BLillemo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center, (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1355&amp;rft.epage=1369&amp;rft.atitle=An%20R%20Package%20for%20Bayesian%20Analysis%20of%20Multi-environment%20and%20Multi-trait%20Multi-environment%20Data%20for%20Genome-Based%20Prediction&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Luna-V%C3%A1zquez%2C+Francisco+Javier&amp;rft.au=Toledo%2C+Fernando+H.&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Lillemo%2C+Morten&amp;rft.au=Crossa%2C+Jos%C3%A9"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492870/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492870/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492870/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492870/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492870/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492870/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492870/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492870/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492870/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492870/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492870/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492870/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1355" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX A &#x2013; R CODE</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1355.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1355.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1355.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
