<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1571" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1571.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1571.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1571.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1571/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1571.atom" />
<meta name="HW.pisa" content="ggg;9/5/1571" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Cleaning Genotype Data from Diversity Outbred Mice" />
<meta name="DC.Identifier" content="10.1534/g3.119.400165" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Broman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Data cleaning is an important first step in most statistical analyses, including efforts to map the genetic loci that contribute to variation in quantitative traits. Here we illustrate approaches to quality control and cleaning of array-based genotyping data for multiparent populations (experimental crosses derived from more than two founder strains), using MegaMUGA array data from a set of 291 Diversity Outbred (DO) mice. Our approach employs data visualizations that can reveal problems at the level of individual mice or with individual SNP markers. We find that the proportion of missing genotypes for each mouse is an effective indicator of sample quality. We use microarray probe intensities for SNPs on the X and Y chromosomes to confirm the sex of each mouse, and we use the proportion of matching SNP genotypes between pairs of mice to detect sample duplicates. We use a hidden Markov model (HMM) reconstruction of the founder haplotype mosaic across each mouse genome to estimate the number of crossovers and to identify potential genotyping errors. To evaluate marker quality, we find that missing data and genotyping error rates are the most effective diagnostics. We also examine the SNP genotype frequencies with markers grouped according to their minor allele frequency in the founder strains. For markers with high apparent error rates, a scatterplot of the allele-specific probe intensities can reveal the underlying cause of incorrect genotype calls. The decision to include or exclude low-quality samples can have a significant impact on the mapping results for a given study. We find that the impact of low-quality markers on a given study is often minimal, but reporting problematic markers can improve the utility of the genotyping array across many studies." />
<meta name="DC.Contributor" content="Karl W. Broman" />
<meta name="DC.Contributor" content="Daniel M. Gatti" />
<meta name="DC.Contributor" content="Karen L. Svenson" />
<meta name="DC.Contributor" content="Śaunak Sen" />
<meta name="DC.Contributor" content="Gary A. Churchill" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="MULTIPARENTAL POPULATIONS" />
<meta name="citation_title" content="Cleaning Genotype Data from Diversity Outbred Mice" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Data cleaning is an important first step in most statistical analyses, including efforts to map the genetic loci that contribute to variation in quantitative traits. Here we illustrate approaches to quality control and cleaning of array-based genotyping data for multiparent populations (experimental crosses derived from more than two founder strains), using MegaMUGA array data from a set of 291 Diversity Outbred (DO) mice. Our approach employs data visualizations that can reveal problems at the level of individual mice or with individual SNP markers. We find that the proportion of missing genotypes for each mouse is an effective indicator of sample quality. We use microarray probe intensities for SNPs on the X and Y chromosomes to confirm the sex of each mouse, and we use the proportion of matching SNP genotypes between pairs of mice to detect sample duplicates. We use a hidden Markov model (HMM) reconstruction of the founder haplotype mosaic across each mouse genome to estimate the number of crossovers and to identify potential genotyping errors. To evaluate marker quality, we find that missing data and genotyping error rates are the most effective diagnostics. We also examine the SNP genotype frequencies with markers grouped according to their minor allele frequency in the founder strains. For markers with high apparent error rates, a scatterplot of the allele-specific probe intensities can reveal the underlying cause of incorrect genotype calls. The decision to include or exclude low-quality samples can have a significant impact on the mapping results for a given study. We find that the impact of low-quality markers on a given study is often minimal, but reporting problematic markers can improve the utility of the genotyping array across many studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1571" />
<meta name="citation_id" content="9/5/1571" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1571" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1571.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1571.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1571.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400165" />
<meta name="citation_pmid" content="30877082" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="MULTIPARENTAL POPULATIONS" />
<meta name="citation_firstpage" content="1571" />
<meta name="citation_lastpage" content="1579" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Karl W. Broman" />
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin–Madison, Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="broman@wisc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4914-6671" />
<meta name="citation_author" content="Daniel M. Gatti" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0667-9926" />
<meta name="citation_author" content="Karen L. Svenson" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7928-1911" />
<meta name="citation_author" content="Śaunak Sen" />
<meta name="citation_author_institution" content="Department of Preventive Medicine, University of Tennessee Health Sciences Center, Memphis, Tennessee 38163" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4519-6361" />
<meta name="citation_author" content="Gary A. Churchill" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9190-9284" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_title=Genotype probabilities at intermediate generations in the construction of recombinant inbred lines.;citation_volume=190;citation_year=2012;citation_pmid=22345609;citation_doi=10.1534/genetics.111.132647" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=K. W. Broman;citation_title=Haplotype probabilities in advanced intercross populations.;citation_pages=199-202;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.111.001818" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_title=R/qtlcharts: interactive graphics for quantitative trait locus mapping.;citation_volume=199;citation_year=2015;citation_pmid=25527287;citation_doi=10.1534/genetics.114.172742" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_author=D. M. Gatti;citation_author=P. Simecek;citation_author=N. A. Furlotte;citation_author=P. Prins;citation_title=R/qtl2: Software for mapping quantitative trait loci with high-dimensional data and multi-parent populations.;citation_volume=211;citation_year=2019;citation_pmid=30591514;citation_doi=10.1534/genetics.118.301595" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=K. W. Broman;citation_author=M. P. Keller;citation_author=A. T. Broman;citation_author=C. Kendziorski;citation_author=B. S. Yandell;citation_title=Identification and correction of sample mix-ups in expression genetic data: A case study.;citation_volume=5;citation_year=2015;citation_pmid=26290572;citation_doi=10.1534/g3.115.019778" />
<meta name="citation_reference" content="citation_title=A guide to QTL mapping with R/qtl;citation_author=K. W. Broman;citation_author=S. Sen;citation_year=2009;citation_doi=10.1007/978-0-387-92125-9" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=T. C. Carter;citation_author=D. S. Falconer;citation_title=Stocks for detecting linkage in the mouse and the theory of their design.;citation_pages=307-323;citation_volume=50;citation_year=1951;citation_pmid=24539711;citation_doi=10.1007/BF02996226" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=C. Cavanagh;citation_author=M. Morell;citation_author=I. Mackay;citation_author=W. Powell;citation_title=From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.;citation_pages=215-221;citation_volume=11;citation_year=2008;citation_issue=2;citation_pmid=18295532;citation_doi=10.1016/j.pbi.2008.01.002" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=G. A. Churchill;citation_author=D. M. Gatti;citation_author=S. C. Munger;citation_author=K. L. Svenson;citation_title=The diversity outbred mouse population.;citation_pages=713-718;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892839;citation_doi=10.1007/s00335-012-9414-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Cox;citation_author=C. Ackert-Bicknell;citation_author=B. Dumont;citation_author=Y. Ding;citation_author=J. Bell;citation_title=A new standard genetic map for the laboratory mouse.;citation_volume=182;citation_year=2009;citation_pmid=19535546;citation_doi=10.1534/genetics.109.105486" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. P. Didion;citation_author=A. P. Morgan;citation_author=L. Yadgary;citation_author=T. A. Bell;citation_author=R. C. McMullan;citation_title=R2d2 drives selfish sweeps in the house mouse.;citation_volume=33;citation_year=2016;citation_pmid=26882987;citation_doi=10.1093/molbev/msw036" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Didion;citation_author=H. Yang;citation_author=K. Sheppard;citation_author=C.-P. Fu;citation_author=L. McMillan;citation_title=Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.;citation_pages=34-34;citation_volume=13;citation_year=2012;citation_pmid=22260749;citation_doi=10.1186/1471-2164-13-34" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=D. M. Gatti;citation_author=P. Simecek;citation_author=L. Somes;citation_author=C. T. Jeffrey;citation_author=M. J. Vincent;citation_title=The effects of sex and diet on physiology and liver gene expression in Diversity Outbred mice.;citation_year=2017;citation_doi=10.1101/098657" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. V. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=S. E. Lincoln;citation_author=E. S. Lander;citation_title=Systematic detection of errors in genetic linkage data.;citation_pages=604-610;citation_volume=14;citation_year=1992;citation_issue=3;citation_pmid=1427888;citation_doi=10.1016/S0888-7543(05)80158-2" />
<meta name="citation_reference" content="citation_author=A. P. Morgan;citation_year=2016;citation_pmid=26684930;citation_doi=10.1534/g3.115.023739" />
<meta name="citation_reference" content="citation_author=A. P. Morgan;citation_author=C.-P. Fu;citation_author=C.-Y. Kao;citation_author=C. E. Welsh;citation_author=J. P. Didion;citation_year=2016;citation_pmid=26684931;citation_doi=10.1534/g3.115.022087" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Mott;citation_author=J. Flint;citation_title=Simultaneous Detection and Fine Mapping of Quantitative Trait Loci in Mice Using Heterogeneous Stocks;citation_pages=1609-1618;citation_volume=160;citation_year=2002;citation_issue=4;citation_pmid=11973314" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mott;citation_author=C. J. Talbot;citation_author=M. G. Turri;citation_author=A. C. Collins;citation_author=J. Flint;citation_title=A method for fine mapping quantitative trait loci in outbred animal stocks.;citation_volume=97;citation_year=2000;citation_pmid=11050180;citation_doi=10.1073/pnas.230304397" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-resolution genetic mapping using the mouse Diversity Outbred population.;citation_volume=190;citation_year=2012;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H.-J. Westra;citation_author=R. C. Jansen;citation_author=R. S. N. Fehrmann;citation_author=G. J. te Meerman;citation_author=D. Van Heel;citation_title=MixupMapper: correcting sample mix-ups in genome-wide datasets increases power to detect small genetic effects.;citation_volume=27;citation_year=2011;citation_pmid=21653519;citation_doi=10.1093/bioinformatics/btr323" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Cleaning Genotype Data from Diversity Outbred Mice" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Data cleaning is an important first step in most statistical analyses, including efforts to map the genetic loci that contribute to variation in quantitative traits. Here we illustrate approaches to quality control and cleaning of array-based genotyping data for multiparent populations (experimental crosses derived from more than two founder strains), using MegaMUGA array data from a set of 291 Diversity Outbred (DO) mice. Our approach employs data visualizations that can reveal problems at the level of individual mice or with individual SNP markers. We find that the proportion of missing genotypes for each mouse is an effective indicator of sample quality. We use microarray probe intensities for SNPs on the X and Y chromosomes to confirm the sex of each mouse, and we use the proportion of matching SNP genotypes between pairs of mice to detect sample duplicates. We use a hidden Markov model (HMM) reconstruction of the founder haplotype mosaic across each mouse genome to estimate the number of crossovers and to identify potential genotyping errors. To evaluate marker quality, we find that missing data and genotyping error rates are the most effective diagnostics. We also examine the SNP genotype frequencies with markers grouped according to their minor allele frequency in the founder strains. For markers with high apparent error rates, a scatterplot of the allele-specific probe intensities can reveal the underlying cause of incorrect genotype calls. The decision to include or exclude low-quality samples can have a significant impact on the mapping results for a given study. We find that the impact of low-quality markers on a given study is often minimal, but reporting problematic markers can improve the utility of the genotyping array across many studies." />
<meta name="og-title" property="og:title" content="Cleaning Genotype Data from Diversity Outbred Mice" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1571" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Data cleaning is an important first step in most statistical analyses, including efforts to map the genetic loci that contribute to variation in quantitative traits. Here we illustrate approaches to quality control and cleaning of array-based genotyping data for multiparent populations (experimental crosses derived from more than two founder strains), using MegaMUGA array data from a set of 291 Diversity Outbred (DO) mice. Our approach employs data visualizations that can reveal problems at the level of individual mice or with individual SNP markers. We find that the proportion of missing genotypes for each mouse is an effective indicator of sample quality. We use microarray probe intensities for SNPs on the X and Y chromosomes to confirm the sex of each mouse, and we use the proportion of matching SNP genotypes between pairs of mice to detect sample duplicates. We use a hidden Markov model (HMM) reconstruction of the founder haplotype mosaic across each mouse genome to estimate the number of crossovers and to identify potential genotyping errors. To evaluate marker quality, we find that missing data and genotyping error rates are the most effective diagnostics. We also examine the SNP genotype frequencies with markers grouped according to their minor allele frequency in the founder strains. For markers with high apparent error rates, a scatterplot of the allele-specific probe intensities can reveal the underlying cause of incorrect genotype calls. The decision to include or exclude low-quality samples can have a significant impact on the mapping results for a given study. We find that the impact of low-quality markers on a given study is often minimal, but reporting problematic markers can improve the utility of the genotyping array across many studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Cleaning Genotype Data from Diversity Outbred Mice | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"c10xZilo_l50iOCIfah-WIzBsVz-tsfcjD7lO5825E0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492866","apath":"\/ggg\/9\/5\/1571.atom","pisa":"ggg;9\/5\/1571","ac":{"\/ggg\/9\/5\/1571.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1571.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1571"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1571"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1571":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xzW9kJCif2rduY8eMqRz+r2jxxfpmB3L2plqyxSmmg10noIcn+UjdJ\\\\\\\/0ihKuHhrhEgsqxql+XGBPJhUHIU4rzU85F2prBuptwZJnHFFGBF3NnaYrBRmL3fewxFGV8TecSTJWaY7W9SM+2YoS3i1yADfMPi0dVtKV0cXp+3HJkWUR3xVJ\\\\\\\/RM2ftoxBn3aPAWV02WIgZoNTVYN2pmxmGSafveVfydkuxAnKLDkqD35owFKKRZw9aJdIntUQHOze9nOSu1RpFFVOD13wroy+4lnQn0M8Z10haqu5xkuwN5lS9s\\\\\\\/QPV8YR6lwNydnk8JMWBIHIUDCq35KZUzQOLElttEU7+v\\\\\\\/tUeB8YmBEKpHiOTgUhQ\\\\\\\/W6I+K0yhIRDNcsdcWmehvFtDcMCYV3cxrTR6kzRwmxdYldap1acN60shFrvkgA0mHBzJykihREBv41DnjchihgnNOVFIJAYCGsNi\\\\\\\/RQNclqcdSOLcnk9RJkbgqXwfSGJdAhwZKF4KdsrAmCMXA8wAHUyxk+tnVIaC90G0QkUbu3OR9Le5ovxvUjuAYh6RX2qUeJQI9weP0efvkd4GgtavatQYIAhQFNnyK9MgK4xQbn\\\\\\\/ruu1pb2hXfid6fsV7SgvWw0OjSEH0vsv0Z23Y4kkvXPNMe83DLfICGFcSTXi40yqTmYoi3gqrqHB2Vh78FwbSLBWkdxNr6f0CU7Jig0wxPrqtz1H4v29ZX0biFPyCcbcy6tQKWUcDvUTJ3jd41PmYm\\\\\\\/YlnyHQJPTc1DKktX76Ae\\\\\\\/zew6mPmvmC5O5b2c4l0O96Gh5U0T7Kbo7FB\\\\\\\/v8pLT35OXdO1Cezk0wvsUBTKSjyLUrUo\\\\\\\/zrsnDXZ+6YAXr+lumpSgxA3HCTYxr2a4rTnL4OjnrqHG8ajDDHf7f7keZG6A7SipMBv2W6hJyQWB2L78iDBli06UaOc5FsgIcJS2qGyopnTaECxAZYGoeoCDHmHlQIlbGUUXeUrHManrde3lLOkdr19vJzvOreNwON3EyUbgkdqNwKKMfM+yNValGWO6oZY\\\\\\\/+YOCk7Pj+w3DT\\\\\\\/xio8kAt1LyF5dLdzO1pgEaaAn7TvIoCkAZd5B4nnzSbyFCvyL8Jee\\\\\\\/Ojr\\\\\\\/XgKRxJDD3vRJvvMoL7g\\\\\\\/nc\\\\\\\/gKjhNPY+Fbq3rmBafGXNrxF0DDZLxtMmMwnFm+mciPaxHtNZLfx83C7\\\\\\\/zLfidzknTbYG5KsrgRMJ+5zstxpS1ofgBtd8JN3ZoTYMnhVsijRFf1CCFCKvl1qifX32\\\\\\\/UWi4VqcifNxBiv9vibSLfDXVEx\\\u0022,\\\u0022iv\\\u0022:\\\u0022qbwap9P+rPPlqT2Tc4D6JQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bc48ca1223f34b79a464d18a4e516980b41ac6f5d2bf1b070c387cf1078d8172\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dFrQoN1JU\\\\\\\/zfzDziq9u39ZbCkXVPWTZCMeYaF1n4A+QJ4hHrxpfz5+5MRGSQWNsQnjjH\\\\\\\/9AjzszZ5D7OLO1QRcTDix0tVp+kn+hpX4va\\\\\\\/17dZdOO9I+A\\\\\\\/s64+XFytGmrjXbyijd8wRBOYlmp3j2McxTrPGoMUMFyjlUXzlpx\\\\\\\/LFv3wJdriYLLzWWHUNJVZMIMDPLzLrW9LQ7xc0WqEQYGzBXoi9YJ8E8EXVPm+DvFZaMHyreBlQyVyxXCy\\\\\\\/TAuwlyicMoijoApNxGd4K5GSmCVQaKAp2pf0XWHXDfX3DHfg5oFl2ZONDcPpR7YQe+WYk640qUpuVl+3SVeilY6fI4GQc0OI72vfNWmD2dA3T6EswxE5NCUHW9iadO93OMsyLGefnmLnS+Dx2b1JuuVWLhrC2bUWwdWi0M0lNlJMy0EUYIazh6dd0DwjYWeLaocKcjfoir5OEwjMUMRXQEjTtvNjmTj+3To2O2DL0+pw3Ykl\\\\\\\/mPexq0QOrK7WP8cYsAIwu2eBC3vhGPLrJOQwXuszX9ngDCzd9suoD\\\\\\\/n01NBVbZs3XcmY7YKmKxnFOoO8f0MtRXGnOkfcIH\\\\\\\/n7vARfC0uMsb5rVCCWGCO2VeXd7c\\\\\\\/vTcB3VIwteaG4yi1Edac+hDgsovscn2\\\\\\\/5rAqxlwAF3tspy59Jirm3TfapgY\\\\\\\/tjAiUBoHJpeJsbKNIpa85WQup9vS10nTJsMMQImvLYniaXmY5E82UxQZd1r7DJDv1tyQpHsYAv6X4URwzwjx1W9eNSo8QDmFMZOKE7wPmzHSz\\\\\\\/BsYrRyUUqhHw44B6hX2fB+oX0F3sFd8LtNgj3YNmB6zJHIuF2C5vY6SIvpxF74W\\\\\\\/b80ny3a\\\\\\\/7Cj6yFVRgZbp9vr7UnNYUjeh68DnL34aaCsIACTHSVKX9QWFEung61vZY7J47IqZU7Y1czRlGjXyJ\\\\\\\/lsUQP6LbD3JHZa2dL+lx32P5Se50xZbntMNIRb2YVJd3eyJVxINcj2lNh5khvTYoFKsttZZ8OvBwA0V0UarU1qHcjDAPOCtR4SVL\\\\\\\/MsGc6PgAFkkpdpc7QqKbidPGQ0K4XmetOUUsjjK+RDVy2Se6yqOqWCZVc8VbdlAEUUiZ13swwFv6cVQXrMoBgpY66CXRPExF6\\\\\\\/xpoq3PLGclKw7JO5+kvAfTCla9r2QCMSioFqSSQeyYPIihYIkvIVu076agNvDmpq6CLWjgXOYEty2qFAKhFljOmbZDyVfqImrUWbjk1HMVUlyJmHqpuJOkOwbQFOsIxznowAXrygLdwAqcVzxoOOFgHpw5tBFrQz+jBU5mNpDFL6JMYleqQ3nNUBb\\\\\\\/XtkXX1skQhr6yA8bcy9WeJh2cFt8jZJLJXdxYXeT1zvGNRcQbwFQJC51IU4QZFqJDPd2vFw7mHYrUqnOn1SujypOGQSqzEDz4mP6asq+Zr5nACpvSKRLb9SuYalLOJZlnmqtqao+\\\\\\\/R7YhyNtCaSRHxh5dTskTltzIKVHNKYHTle2mgJwFgeN9HARIdiEIB1dXSrPKSyGp6kMNc2cQM5drxRxhTqef\\\\\\\/YIEvvc7yk5aIqoohZGQWyE2N3fp4D379d8t3R87R5ZUsji1noPdxC3RgrIyMFS2kDYOxmPRH043901M6Utgi\\\\\\\/PuO2rj1qPFvRqPPmyO6MFs507kuJH9+Ur1iasFLTl5YqCNRBQmX8pwMGV8IVkWly61HZRWqN00QRmTnfMNepmxqQmtiRiu9ikLQABnIQqOApe6C4TvSFGJYwPBBfeVy0Kage70joB0GYby+86LttzyiqVxrRNl1Gq7\\\\\\\/eZ0E5Nflssd5nFjbvh92IXVyuJirqZMm41y\\\\\\\/MzPp7tP\\\\\\\/KfNAMNKq5x69G6GOxOIqqFvpqf24GUw4XFTtFSL6YhzLi2ew5OL4dv3qlfFgutTF\\\\\\\/0brfDMZEeJVOOW5amNKA+DGN\\\\\\\/LLWS6fD6fnyBFW79CzF1M1hp7QgxUAnMg7luqWPPgRlD17s3\\\\\\\/d0ErKSTKqQ0Ls4pyZngDFPU4hbpAtHUiDGm51x+mrT3v+4bEZ\\\\\\\/kbaxnVi8t\\\\\\\/ZYApA2Fy1mAPWzPJcXhc65JkMMK949ntusE3gxnZypOfHHIlggbaAVrzO6UTY1hdN4wk4vWB\\\\\\\/OOmFhJ6vjDqwih1Y4kDcZEOKhmfUq8q18ATM98nFteIZCHsqv1O9JDAnBdtawbEYm5a2fCqyHuDo6TNmjJ7Xyq\\\\\\\/MQDGxvlGpdv0ofcbwR85vP7WxJgtKvDo+MUsEV9wdi+kd80bHhTuQhJww3+SeCuwJDzl\\\\\\\/BViJ3X2qXLCLkdCANM86+HlK2W5LhzPFLEv48V\\\\\\\/DUJHj1PiJ0ZMlC6nASINmirwtY9JYZwmKcpXKOf0ZvULoMvClLeM2BUaskMELX6wFjSqlKbTLSJodCzSE7FeA80hO5tByHksV9C2VKByT15w7Ey+WFgRWjRmrntI\\\\\\\/LEuJpb0A8scrn7VoSvUUysdouvUoumnndVDhF174kEK\\\\\\\/8x5oMM0qom3Bms+uVmlfQfUEydQH3Cv4P9sJ51y4zz2AeUJy8DNcbkRUrpZh5MhZhpQ1y1ne3lPWZjSOYQZlSpRsgnTHjYnW9DHvKmlwXrO3QeDduJgaKVZu\\\\\\\/azgNCFssP0PgHe\\\\\\\/Ac8Ts39BaIM1TpH2n7zijtY5Un1E\\\\\\\/TeaTh2KA+4ij503HvIFgfY\\\\\\\/hIzh4OATN8qXoKyTcMSVptkk7ytL2PwbnL+gcLVjxo4xekmwrALPfi\\\\\\\/HyjgZgc8Y4ZdNmqbrWkSBOURyETC3giBaTZi4dyCxXJFBOlGsYpH1tjDRTVqQh5nq89nrgbpKDhnMc3DCu8SxjGqQDcpyLVun7kQ12zLlxfFSSJ4gDgyz9uRLllPB8ZFlsWJQFehMu6iqB7\\\\\\\/oolTzpphmb+bT7HgccwjUP3i4piCHnMaaNF1x9Af7pJ8QFZ9G914dXfKO2ZqYJsjV7PlEtBIPX9Uy7aDzvPOQG2ETvdXD7sKhRE6+lcZEKRw\\\\\\\/xSGjt3tut07FTmpJ1A2M6F+unbNNJ8M95q3GCtrhts9QnEZA6cmPfDd9NpPYfuJE+9G9\\\\\\\/ZKg1eEwrdJhvgIMF5jEVWV0sExuI6Glu73zkpgQX7xfI8jriAQh+DG0c\\\\\\\/EiF7yg1uqOoZamTsvSiZ7ho5xuQJ98DrGjAIoF2GF\\\\\\\/3CrEBohuXTmGOqjY\\\\\\\/SZbRuickhwJaomU461TbQ6pIR6lb+YTs6pQOSbvlhkPLUt1JzPNTGD+6s9ywB4sAezqHGziuWcQQSoS1MSZiLpMG+9q0d4zNPMiCKDGV+pLUumW6c9jwY1fcenwwIOP1Dge6+bo0lFMRgoIS4dwGxQGiOjfqxlGYSHM53S\\\\\\\/h+Mw+bQMI9LeOGjJDcKDCDPP6TKS3hSbjmGbhEiE1o0smWx5A56MjUYUlK9oq2LXsVXS8+pqDyCLTb4tmw55Oy5+fx2\\\\\\\/+hS98NX\\\\\\\/5ig1piId3pkdJQx+Tb2DMR1l4phn5n8aXi5znEoXrWt7WP9a6hbrWBfcF3+66GI6A1ZGj4hF6IsjFM8fFQFhPbKKFgpz4XU\\\\\\\/LIavdopZbg\\\\\\\/o+3KmUfZhFjTcC88b91a\\\\\\\/BxLXzHu6CHsGJLR89B4ZySXLVwHVHj+pC4dw4eQmvgm5q48bp6rKkY+crgwY0mbOILllKIS1U7CFDxRgF1SZloAnUxPW2ocdXvv5vOWDg2HtZn+akruHFsfa0ukzpoeRF8\\\\\\\/SIyUubNRTGMP4M\\\\\\\/4xLyAHCKbSN0M3M4ZuwhuGcfruV3VYkuo6pLXj\\\\\\\/mnI1rHJpFEYKe9nw4zqSdyx8EFbpOrLpiOc55PdZJza02\\\\\\\/PBu4avz+Dle3aqa53P2rr59u9ErWqtgdKJEV7ciMpLMsJt\\\\\\\/mnMhOG0psG+gbAXW6ql\\\\\\\/6\\\\\\\/Ph6toVgZXVyv\\\\\\\/mLE7oU6AezuHkqALUafqBKlwVlvJ2MewW07gRK3dR59pSXIpdzaqll0YwbmKNVDFie0uQkSRwiqpM0nISJkStr\\\\\\\/\\\\\\\/l04Uc1tZCNl04Ut6IjZBk38ON+HTRu6N2Ig3neFomR3JqUSIO4DNRYvX+sldNFlJ1r4j1mesnkCEWb+K05if0N8gVzWQGYOH\\\\\\\/qIkb7GvF4eiQUydNAA1mONyCDAVyF9L3LRlkAzhhfzYarLhu0YgR4O\\\\\\\/l6ztd+1rdL9EmMVv1tFx\\\\\\\/99qpMTHUy9AYpU7hGNbbqqDYFL7+zzPSjTpLw\\\\\\\/Buy5GnjzPB4wRLEjxTRnAdU3HPpk0Iq8fnsBsFgL9htDb7GS18XRMbdHIO6Km4gyuOVPh00m+U2QgjEdWR3VzIXZ5CMP59Lk3uUPEZQy4BA3ijPxj3rkqHNoYPoCEAqafsLxcQMMSdfQkm7jr8gtoCbMAyyo5asWIB+YTBag5c7Sp6exVBNrAz9HAWDBIXbr6IFrFd8qrM9mQAtN6zaRlkchT8HkUOSI0ihlmpnThK6WT8X5dFeQc\\\\\\\/42+hoy6zvLO2ef8IgZrpN7Waoe7\\\\\\\/vgDXUC8OuMttVduxh1KQNXxQEPuQvvsLledve5itE2dHCYIuyjSUXwfPWhPiJziOV2YTaY43kOFDV3oYRgSsqXCBJSBEM1TdYMyFe1+NeWX6JVL2OdiJopcOrhuwBVoYXvDQu3xRwazGhovN1t4aU3tH0uQB6VfwsCB5nNiJ3XYF32Q++uW8RUBEfDabhmPwSf\\\\\\\/prjWNUoX8gLBnSUdTib7OSNkiXQbKjJIEmMAIBd+RzEYNdEGZCicw+M9LTQFV43roziYNL0rfWwF3LUFUPZO5ZxhuRmXvJzH5+HpRCECtF6a1Tk6ud1GpoO2mMu0hJDpS8lkJCLa+dlij4ebGEEJzodX1UhVPpBWcnBBG51vbVbFD\\\\\\\/1Jybpz5xh0JHPwU+INo+Lud8ryYZji0PW\\\\\\\/QWP5Sbop\\\\\\\/w2vX85VfjzeKNLxn6yfKhRbci+kyGLSX9BqUKEt3k4VYXeh+184VOrX23sHr9kpv2toYIG+CAuoIcuVWdHjRXiKeXIJrNdx+vIuqAWEL7QFxL9P0HTJK4GBbudznxxKD3KzTQj746JKNFn\\\\\\\/5OhRWAkPdFb7g5k6A10XUGuBxdFDNiferu9jg6e6AznAufXfWY5EslVN7DZoF4ySkD\\\\\\\/T8CdxrlwAXxbv3ec5tjX73rf5P4tsae4As7NE2PEJv5aOSpAIkv9jOIUGwJiQKQSRPq0Tj4k4SAULBumRl5fKqgwPxS+4CG4JSA6\\\\\\\/7eO7bRgJ0qfuj15G7cEnT38JYgIsvtlBUDeB8zdNjMVRbBRs5BvpLpML0bt5Sd41ACVUjALTp9qhyFDUmq5WHIQxZfGbdB6XLaiPhLsezF4Zl2\\\\\\\/7eERZc4paEt4vGxdSCz4xqN+fGCncnvxcWhKJ+XIVeTE4p3aIKzE512vpvx5G6zoYrhrtNOJztWZ0hVTQbeVP7iol3cEAl\\\\\\\/6pJx1V0lpaMEYy+x+b6pWfxckhm\\\\\\\/gIOntrIC5F6huf2TphP\\\\\\\/MNFg\\\\\\\/GqKcYkiReXRI5tPMBGs5dPwnK5l5o+9ntNPEKazAl7dWDGUFu+kyeMxL50MxY8B2bCBotXH3Nkyc3fujzwrDCuF+dEIQx6qpSG6aXpRt+IFssZwVcJa5+vtmBdHZcZVJiUY3AZUiqM14PQmMrIMMBE0lqnkZ0vKtJg+aSj5UZOma9\\\\\\\/I+G6ooFZSOtFZdngxhuYByaE0wlgT+uEXUp6BQxyaUYa47qqmTN8OOHI\\\\\\\/6LOdZWtf+MBrp4RkX76CFYYKJvuB4WJgQT8U6zmUILbtV+bnVyuNFNqezpYw5OIwciYIoKf6Wcr60T97R53P5voYPGKM7mR2YT5vncdRfgzrjJXGSIAFSGYRUk7n30bJwlsPv7U48i1dZUZO0qy+sQekVN\\\\\\\/2T5mjFylTqIAo1r2YalgmChrxTmNHuayktPZN4txavFdmnUbSOxt94ZcgWqbHN24YPBzljt8D6B7wP9nFPvdf7JRkjukbAN1QsayazvKASKcPglUdbOr9mY1\\\\\\\/ZNT+wPILHNZPinfl7pIiFp9DouWN6Fb0mrCjNzFjZRVxLoJkaXl0rDPJmVrO6bEa\\\\\\\/1QCoRIArFGDyDDUS8aZQlL0zWmA30PajMIfnzidEKkeyrpzr5mKTdmBtjaILomPKaXhz2xovQr\\\\\\\/\\\\\\\/ZAwfLu4ajMMsgWHQSAFawFYd0WAQ30OvAmNdJFvNe9bTYg3fap4D9JatG4rXeNk+5P5HBtQn8cJ4JS29zZWCisfhcBQpeip0BWcqbnFOrVcEp6rjT4lO6Yur8L23XfsacWGTLZ\\\\\\\/IzTnQKWeCRvGnnNDfEDNjWiIXyGy7nNY6GyS3rDXk8kdYjpiYYYyxt4+0229p5NkMRA6HDZj8Vx0a01E3Art\\\\\\\/l+GU2XdsieA6S9LbFPyEXCk\\\\\\\/xfCxp8oC\\\\\\\/v0oDHufPajto4aNjlUPOjwTyzLc0XpoUQ7DWAbIdDt76r\\\\\\\/Tev6Flbp+boBFB+vjG9uYoHN5xgGmlUEWYvUtkJcBpsOxSJf7UEPe5DIgGeLKShIVlMzh6i7+qaDMlwEV7v5G+RbuaUZWVJ9gwR9v5SlZajZ0fmEGLClbGPTR19y6hiF4OKnj5do1hkrdpjTxm2MntGkltuJjaRvi9aYNEMONuPgPA6zNZaMBSOQq63JaACkLmH20UpZvgS157Ra3PRfK6HrI0anzWx3jKD+SAAX\\\\\\\/AMs2LVkdxQPRAOh8IG\\\\\\\/XCn9HXM7yjnE5tE\\\\\\\/NGGcAh1VIYrrVwJuD6imxGETAm+6eHdXUjxMXTipeaFd+SPU4\\\\\\\/Ip9q8qwnMvfjURQAwqxwc9z5LCglT35\\\\\\\/tcAyiL\\\\\\\/x9NfG1F3XS3g4s+XpHRtFSzxpv5Qa\\\\\\\/hmKD3yUcEXYBSFONynuFm3fY31ddyY97wKVs2NpE8QCNGoUp2SutkTtAssWTLj9etsrQXJErV3iNA1Mzfh\\\\\\\/I7NT4WxJqZJPdqnfiy5BsNDxNxhzj9WTn86zNuNE6WCx0RusHyLHJkQAisW1QblwisT85lLSicHUcJ31KkyIu4ZpIeXXGWhpJGSOJ7MuGE\\\\\\\/fGomoGS\\\\\\\/M9UzoK9FXU3qwUs1otufU7I1cuieT\\\\\\\/KEovrKPevD6Txn+Ox\\\\\\\/Aas+LOo3WP6xVpXD4FSBl9Xgtb7tk1lVdj0R\\\\\\\/T7SsQbu4c7SbmT2c5\\\\\\\/kn\\\\\\\/Mfo2\\\\\\\/xTWleKRtIZDuhrT38H9VHLpKwQRhPjzWovge9xmKFo8G5S\\\\\\\/VlkVibrd5Qq5vSfjc00nv2Tm5PbqFOb\\\\\\\/fkN2dwvKVYYD+yYQCsyItiAMRLi+ZzO\\\\\\\/gqhAWA4YZkJMViM99g\\\\\\\/UvUXl0EpN995+\\\\\\\/kSd3yukLuWJvjUg\\\\\\\/KAoFtt3iQ\\\\\\\/2l5cwaFwB1FeS61UaXwonaDivmIrfPxge+3TczLEN4gdzR7w9r7k7mkO\\\\\\\/Sn7NEUO0JoAoI8Z2ILU\\\\\\\/xLM+69+RQW+EmRWG0nY7eEunljJjQR441n50rpYzu3iUTL9CHk6n6dI54F9QlHV3n265Qfszj1J9HoXJ772nCKPEAV1sC2dlcZYNQc7W4BFg7cHFrAt9Yh0XSbCmGAsNhrSgxjwfnRYp2Bv3WhSz\\\\\\\/jVVmZ3SAX5zsw45KPyyQZiiPleS+L37fcDoODfnFLeOLYZkRNMllJRBXCawqkDOmTa77IyYlOp1f0wBwefKg5ej5BZHmIuGEIZfQeBqJuWuIFyY5tqitbu2aiiFm\\\\\\\/ItwnBjw0uvlquwb7hdEOIlFUI5kzfddtKaGi7B+1EfSbSsLgsPmq7YF6b2Qb2SNegTStuhoqYY6Gok+ZLX7gp7tfSq8idn9o+eLTE\\\\\\\/lQxrxh4CnBcx826tSu7ozbGYbji7K86ECVHOwA7wA6qEoF8QmTasfcfbY+UNgZ4+ZUbVpzTvPcxRKsRNP3qgedtnknJB4OBHN5ZVG58xzd6UgpgqgXUJPYBhxbm13QYeWZKotlbr4crR2nKUoHEENWZj2fCQz2EKK+RhIPWg3tfuRkE7e8q0VSarHIt7Bi9LwEsA\\\\\\\/uPWhcb9blmUhGIb8taNCzaiEk2kuwvAYIf1a9cxfECtprHrXBFfjDrhvE3z9gmU16svrY4QxbgIL05W2xzpyEpbew7PZaqv10ZKypzEwVTB\\\\\\\/YiBs5Ivl1GbJepTIWl6vXRFZf3LtvV9nIayAB6DDA8bFYVJ1p0ufCDpZZJ7UTwLx65DcH5QbOc4467Nm77in5XV+pFLhCn\\\\\\\/gWggujtIvTUnGOptR2EGoc6PXGNTyGhqmksFKQUBrjeR3yqba0XsunKZxlSzIHGx7S6dMJ5inzoy9PMQmflQayGznTlR2ICMy5lEj899N63MFiB\\\\\\\/UAzqC3jH5DXcrYHfb6DJ\\\\\\\/+MYb4X2WQx6COA5lTdN1zNMi3FS0sbL0JooaAQ7K0wlXyfJW6BH8EFLaH68mpkgN0ljVg8P3j1P3UTD1ysoa3CiFrdzRCFJUPc2qA\\\\\\\/jdnhsJdKtkqspdLIpapXo7kPRN4OWaMLPWRVJjOxS45+MwZZwb3wVxQ0r6WYtnY8J1N0NdG68Fax3ze1EVx5DqpKxnDm9xFBUbucn1LZNR0LeN\\\\\\\/y0Jdp2ib\\\\\\\/8EBzfg+JgDKacnEDdn2icfnWRKm0LHYXj6+P+tAoWK\\\\\\\/Hb0eJFePE6XCbvoua85WfwVuhWMNB1MX1uuHFONZmrjJVnVNpScIX0rrzR3EJEITrd+he5c6Q5IcaooEZ5MFkJg6dE6GmQ6JmgjeK17MxVnAK1rd+ZLPFA0S6OPX79ZOXtlSHQBiqiPGYUPSYMU+M9GdRoljNw1EHgzVz\\\\\\\/zMA70iTTxcTVOPJm2hlE0YkjC85M60H+KN+jw3h3lBsuQWXbxE7bmjNnoB6nU4LJLsBZVrKfvRs29yHgZEnGvG5vYq373RU8maoa8ry9yg1Zwjiwa33F\\\\\\\/13eW5YB1fvpZnegyTmWwVTBlsnsO9DOKgnZvQKR+wCdLfboCClqVYRg3VW3N3L83VBkRTRRUBVtqXetiT7z\\\\\\\/dqq1LgRBnGjayToqf+1Sk22rZ2axVt93JpEIjiAK6QwtheN72CI+UMl3rLS1Fkcpa6nrl\\\\\\\/NtwI4slS6donKL5dyusOjdFaV\\\\\\\/3uDZ6qZvqwxeohJ\\\\\\\/ZD7M46Eqvec4MHOxZEFdORKhGhOdQaND1Sp5EU693xFicLQq2IqoTyBkpF0V1bqr2qvpiiTElqyd875ugSvhi+CojrIrtV7PmeNTO7H2mlLZHcEUiaZZUL+Ir5K6zpyGMwIt8WvezBB8IE\\\\\\\/4aDpBkiKYhSHk2oPEDDLbFsFOXw+u3coBZ26pJtQtYOvlkWhlBDodFzxgDHUiaIwKz0YIOCUfroUUgYERoLW\\\\\\\/ODG+hYq6Bia0vYAXblFz6\\\\\\\/cEizNl97mwc7ZGn3qTBNjPj3ZbHHrcSxfqKrf\\\\\\\/FQIL2rEYTxgKn0Zo5E9sVuKhWP9w\\\\\\\/Sy1cdnqbs6FE+0KLP1+\\\\\\\/yrah\\\\\\\/NC0fAsWKC+9PBO7aSrHjX0wCS5wrEI42op7MPvIbPMvb\\\\\\\/0S9YM3KjzcDwLUXJPyspYnlDgo7E30dSuqjnwQaMSyL9uaZLODS\\\\\\\/\\\\\\\/jOXTaompvoqRe6yT\\\\\\\/YeSwGMAQ5a8D1Kp41e5r4md6I6b7NzABDe2aYQpgo7oiMSACWAzjzPAZtsu7MzSaQSWN0Pu4mz\\\\\\\/7UshU20zcv6DrcH1fRr2sk1vApyTDikxj1nGcSz5TguQA7A2qj\\\\\\\/3dSPYFCIqW5BoZF+x\\\\\\\/hAfR5OziWoGvWLf+8zpMrVbQaMLiLTYP4M9ybupPR2MivTP7OJmjirvhY0SB0dPQBCiRs\\\\\\\/Hj\\\\\\\/ODmqNoVfMTS8aIEmJK\\\\\\\/E6S+6IrLJ0UqJllETvdorC3PURHlBqulWqfjnof05fB+l\\\\\\\/K6IyXty4S+oIPomVW7hZVtCg7NvnVYMatdFEZ9roK5XWlgyZN5uIEtJwF4qAt+kYIfJ6uSskV2tVd203iu2Cw7hG9++AVZn0kJv96JiI4nLxoMMTXKzJsGrS5FGTuaqH5lI1Cc58UHtJRUF1o\\\\\\\/+GbrlC1UqroCDFyXBv65Kv7ZDLzVBNck6nAKh0B0erhrzAaeivBiHEvefh+BtqMtB8HJ2mbT5ceIQQHEZdK0jjeN86WrBb7o7fHDWFTxq5JxK0OI0N4cHtgFlGdNSdXl7LT0RvRSWh3nTB\\\\\\\/O\\\\\\\/xVBZkZaoenhhpjNMWnmVJmghE9RPsuPGJFlpHfJ+b0+lQQNH4meJf4zhHlidosSC6MLLbR3K2PQdKiprTQ7N5x6JjgTfVmll4CvYZnNW8ybAX8QebZZgwBGkvmB2BIiMIxRlxAokIYAVjrd5vjGstjAKreaMZoEwEnlMhzfsSyzFOfHDSld7Kj9XuJpEHveT47t0NwJ5yaRVuF731xzwf\\\\\\\/ts7MpgRvR6i8E6IbVVLiXV77BiXFFGmuoTpXVDMCNIqBsqHVt4vAVSdmI9M6siKe7ZRJMgHtAURmaQLx9QjsikEN+LbXcOenD2uOHxvbXL1e8Y1iPEpon4NpMU3ZNM0F\\\\\\\/TrpMWiCMcBsJbyuthZcQLgXMGjy+26O\\\\\\\/v71SHLu0lTKnBaAgwFicXTBo71TiIH4EBxTWUhkp4ZOlpAmZKj6wJXUsZeKTWAQ2B54oLeukCYZkb3bZU7ri\\\\\\\/oEwqrov1cdnDraE53HdB5jPtb1vM0QdZDuj6GT60oHLZ+yDDYL9RRdd74LdW91ydM\\\\\\\/CCgxVAuWsNpWTmsMuhFg5UtQ\\\\\\\/+vNaD2ti+bh1VROKY7qIfkusU9aqkgJHx9ggUupJuY7zm3fVgDi96tjQkDENG7zfyXe6rQW6tAB9bo6FzcVCKchYI78nFM+fuW2EyjAwi4oeBFatzySXtLpgFImjb3EHDQ4m6Ui\\\\\\\/SUKTwGB4l8Dob1P5zs\\\\\\\/Kq4rwEo9+3erxP8Vx7hARNkDHYua34IqbNFKHa5kwnTBP6vK8IBd1+eefTaJu\\\\\\\/dKBAiReOnxuOEkWWEWWlqpwQh4w2tjMtNm8Kh\\\\\\\/lxbT0QAv8iub7aJLkdKELWlqg+CtQc1OcdtITIlE8kcVV4+WMUqgqIFelAEcLd9OCc+BGcR8nkZkKLsepgHWfMss4vwyDinQY5g3r30ZmtHV8tZR\\\\\\\/1ei6dEwW1stp5O9M6oYJzV5N06Qnef47VE9NrY6KTBJe3u2zr0+aLCf1uZfWnSRDZ+EDgKV6zqtFNZ4CawT9WHCmZRzuZ71UHlo65M7MEN9RmScQVoGR+BfRe8qya0sZXAx9rajIQM26hKv5ZTrAb+\\\\\\\/Iqw7Sjpbs7xkynruvOJGwyM\\\\\\\/uNSIPk6oVaG8kjt6LfYB7\\\\\\\/ozIO\\\\\\\/jBtOuWSe9T+\\\\\\\/9q+1DEodTrc87RNFKDkPkUQedXTVRCVk8SLOvkhBN+TLWkS7+Xm0qZwbzv8GOWnjljhVMZ9Pa22hvSW+yaJEoQd2LA8uS5eRR3EKDjOysD1wF6\\\\\\\/IDtH7qP6ta2ebreqW3AWto3fs2NX2MMa+RDPYGqNpu61Eu9MEY50vSaOGv9GuHNLQm7hTY07e665ggvQEuDeth1ipoweb\\\\\\\/\\\\\\\/0une5ADK5PFCe0nkWAFMdWj4yQmQt\\\\\\\/leN\\\\\\\/7BN+DQnFiNp2MFss8rWYLqcsVr57iNKGtUOuJHbev7i9m6Q03zY2j4a\\\\\\\/a+fbd9AL3gVuUX2S90rdZvVPghNou1IISUqXeqf\\\\\\\/pT9N0+P\\\\\\\/RNIw\\\\\\\/b\\\\\\\/RMVb4NBs+l3zvIhtD5u9CooH0lIbwYNvtHeCO3N18c92uRUw0O87Nyqf4qM8MLeFohND0zrHeXnYMBGX8z5iDOX\\\\\\\/AKP55DPkY37f5euwvLng6JZvgMs1QugXcIReHwb\\\\\\\/EVDVd4hBQ9F94oPmtDj2NAMzNXhzcd5GqTAmnhfc3CIHf\\\\\\\/OiJ\\\\\\\/VJxFOHQCiIsruakxZryMarfbQ4oN4Q7YUKuU8PfvMsn4Q7zRwe3CrJtuCuGQ0rR2iXsxCxGyg9Onkfj6EP0dMXmNSkBt\\\\\\\/9ciQ2yy8SFshfn6otqdPkkg+uG\\\\\\\/C60bMngrfvsNnDath0UkWsoqdkk8IZVoG81CmEnjMYmDFOCXJswf2FmO594xlaiH8bvoL+vL\\\\\\\/5x\\\\\\\/WRHlHf7KiR70U2G2EHT8LSLRsVHz3F2a0YtIDOvukLHQZAuyhG1wT6XO1xiWnM5ltU6irONmj9mt+NgNNpwr\\\\\\\/bh0jL6k5DaCbWQRa7\\\\\\\/YJDCA4XOEvdpDIwVsW5\\\\\\\/HtoenJ36YAhOlhTLcPcKtoEEH069pnzrYHYRLpxFkxijXrvK7aJRJHNZuRdA3cRbH7OTS0\\\\\\\/ib1ddrUUrvzPFmuSlIPyZlZBcv9REs7pyAOE4guThq0ntBwsNl5vY9RE0o\\\\\\\/3jU8qeR1gWGZsJpQt1zvpEodlq0TrG3tanmLCOabK2LfdQLWIQgzYhhB0aOS67iEtJtGUBqwYnOWSaE7QPJQChS1SGhfc1NSTbsXu4u\\\\\\\/rXfgv86ju12pUVCmIMPVw1RGXhtfYLOHpJZy1ikxx8MkPCqK9U4Ep0bloU5kls8I86at1vXXNK1J4mueJyho5Y5M8ms4oViRir\\\\\\\/WwBiYjAZbHcjeIO\\\\\\\/gz+5gVPx1WS4jpq0DasEry+7NrNouRKU4g\\\\\\\/S1RHiMgYlkWU2fTx+leEcUUFTHCoH7Nfq3eYEBdWODTKEeRjUrrdft\\\\\\\/V2W3cMSZkPjAKaYLFR7qlQE1uNQGlsMfrDkAADRU6KCNznTtlu4\\\\\\\/AWdkLgiIXI5k\\\\\\\/UeHLhzyH6QkSEVriu2Z1CRu9l6KpVnL7p6pBrkJITKX5lbR2KP8kDEJYmLt9FJ+IjqzRuwJVa+r\\\\\\\/vd9GhmTSjyK6XQSg9ci+3wg5UrDOTTdTFk\\\\\\\/\\\\\\\/gaMggbUrMkq9tpdUTtAQmKialClLdk2jUvCVyqm50326NoJY37MexRfM\\\\\\\/0YZBVI7\\\\\\\/Syl3IRQoysMjEZqctEw8gljC87oGHXtUbxAG8HpwbQm+i\\\\\\\/hYrJ5NQq+LpQ0q2KW9LEk3rec2x0+EmNng2od+gXzwrI0+UrQPYjcXNdveFMYnf6EQtI9\\\\\\\/bgaMjGjdyyPAtlPBw05rG2h66ENiDao87IJ1\\\\\\\/R7FEbBfLBlMtmMHtQe0dtnExRf80\\\\\\\/8Cm4qmTD3IqCfjPD03BOHMiFKI1ceYcaMGMA73Zz99PPADlghYe9sjFxgFJooAN34UTnuuvIuDySAVOxLziM\\\\\\\/K7F1Turj18vfX9QBFVJiC5M0dqkz2oDfYBJDjFnzYWr16wxjmm8fgmbSqgseW8oaBRojVrRSGexYKi4JenoSj8Mkfv6JU45f1Zdjr2jcYcRjZk62WRhJgJ3Sy4y77cJiTN7YryAWbiH1qbklVh++LtpGzdiO422acD5gD\\\\\\\/FD0tNH+3k0C2dX4YLpTVORiOJhTNbILgZxGztalzxrUEkMNr9QAH1cauPBfP8Y2M+PJNHX8t9hpmOqCu1Ro8ytz7UXFEX\\\\\\\/+QvKcc2k1p0LvHpfVhT9tCOHTz5HBka5jKlOS5IjKvJ+ujW7an4B9EprGWps9AnpENbUBwTeBecsT79ftCNKw6Rbv7uXa+Z1X6oDW0\\\\\\\/yexj0YjKVx9\\\\\\\/90w4MXUsxJPMYjuKstxSwx0CWWo+jBvmBKoEOiNfJfntZJxhYPdUgoANBle1Xi4+whWt8+yijcnvG5h5iU0UyXDNJHJ22ML5HzpGOqw19QKdFLJmdXO2sFJWpOmzPUPWgWaO7Rinda2JwbtG5WZonj3uSnesAVsiwH4D+rzfci08ELaco00H9cNfCRxFd688DwU1MfchltCVu4FsYnK9hSNhb067rG5nREFOP7UrNdMWmmyUtO0K2G4Ij2IsBtLRL9f2ogrCwFO97c1W+cPWIXwCE9S52rstemNwX8UVUIGctiGXpIBy7uAs5dJ\\\\\\\/9kX3hdM3ow44NHgx8o3x5oA0zdBvH5BoK0DeRbce\\\\\\\/uyzLG3hwuy8yklL1HmI9qR+lX5RTKM7zSyRji5Chy5A34VIpk+c3\\\\\\\/IuHK7PyJK4Zmb3bNkVYpFWdhGC+Jv9iaOfx\\\\\\\/0OAZK3XTBF3M+3exjvNCm8IWq7F1wEjsEBeSB5umCYpPh6BUvDZNqJCQn3EDoOjJJHOy16HRfNXtrojFeHv9KcV6JvihKfpnsNtbG5putfBZ76ngqu+TuQ2FBf8lqSwEIUnjyrzqa+n3jM4heVj\\\\\\\/6Ald64bN6QCQDhYN08xAWfCE3Xce0+CNhwGPTeoGfjFcc\\\\\\\/0FPlxVvT17DlMOl193p2NXqMH15ugq7wz\\\\\\\/rOd334+Oj5Ca4Dd6RHKQzGQX\\\\\\\/qnlMJPGq6wiT+kW6S24Xm\\\\\\\/Abuj95l8Hu+kizT\\\\\\\/58pWejXImwJHfWDAQbadgJwZJfHVp4VauShJI0B8AepTD0bEGL01eI\\\\\\\/8AzHZpR4yxixWH2xGHNQPVcOH\\\\\\\/8gBx5QKasNTt9IpCgTBWc8uCuk5GqbwNBdrKScBVKBVb\\\\\\\/mbyOrr8NteM9xUg5C7wdohP7Ryj1n\\\\\\\/vawG6udoZjAtmAGWilFSdw0WiQ7GKACCgtb0Pe8P5Gm93pBOmLWzywFLHOG4NjhWHN9\\\\\\\/BsO051G7ARu+D+dVRgJcRXEVF99ykPywIyYXxffhN9YZLitQfMADfxySJuoix3IwFWtHT4bvBSlEmcHn6hmxv4c64NFwosq3+7JTaS3trMS129Qo2Pdp1l3\\\\\\\/10ywoKjG4eOehiGj4XapudX2a+AxQL0jkilhoAqQ5xb3GtsqPSHLlObAg9\\\\\\\/UJtm16aakqgs8\\\\\\\/Ydld3WaDqwqz2TqR6YrDRprXaO0aQ5jfZRWtiQtchJVwNyG72iPuHovX0\\\\\\\/OFmuY1xydMNmfjYOMXeeKkyApSkR0aIKsAKskTXPnvyDEizVXdQqkWZwelWXnBncp2iXkoR6zzDPJy28IOzaCVP36hgWpviJ0Pw4ykbGedO3WbA9UASouFUK07cgQb9TYAW8NrGOLYrf9d2hdlO7NZkLbfsKQCCNTK08GfqZgeXvb3Pook5VRCbEgb5mDPfErvWEiPNXZHi9gyZ\\\\\\\/Nt3A3f+mfVTqjw02AC+ngQ06yQC7SBbMT6\\\\\\\/fcZ3v8rjsVs8TepSssip6RfZCRMZJj1NyUPE9SjOSStfcvaQ81QcwqD8hO7hpUYNFEoDLzWBT26J\\\\\\\/q6v9S8ZpLtvWVlGnYej\\\\\\\/q3onO1UvHlubTCys84H+Dr77d7pfxXc7ugLdX6cPZYb6NuRF05DYglqGCdz8eI+FlNMfBs00V\\\\\\\/VjCDRd89nD\\\\\\\/uiu1DPDzybyv+PaCz3fyWhVFad85x\\\\\\\/vHpM5a0j0QqU4Eau6WkpKbp3TvJBFvoFYUVHvGPDlrD6GpwIgtLHW0J1x1KFvTRNkGaGUFlNXw5JYUKIYynS9yGUal5vHK8z\\\\\\\/fybba645tMj92eY0IpL24r+U03nxgdj6BwgQ1hStGnDi+x6mktdO\\\\\\\/zQB6C+3tH+ZLxube7vqOGDdVPI9AJnOqqFN3E7mRELDBDViHzet7uEI33UNHeTOAKOsi6odsLgW7L\\\\\\\/J4CyxYLklHdodTjbp36vcAk8Vok4tbBZ7pM5gEqDDFvuJ7\\\\\\\/BLtoftfp3T+ry+ktJjOfV9NExm+lTNul3adA+oTAKWG3+Spve\\\\\\\/dcvgWVZdDTDuicDZHt4H62PZ7gu7DIe+SQ5PJQBpxzplIyf8SPxf1JoXhEhf58MWuV1mMh+m5KY45qn6Kbo8rpRuKPBYUYqYekqCm5WiQE25HLtufIlNAFAS1kYxK1gaUulH01EB9Sk00IVWmPTwx3vu5IQqug9VXmu0uKWlF8zm2z89A5gsHih4lxO5CsY83DNhwwgRu2ZkfDMXMUpmqhxPreMvbM3WMCxF6QvQ2kBNzRojkGg9di67q+S1vaDjPiq5NYh+xVa5Vp9oTpACZ+TFaadIqBAU+D\\\\\\\/bwPw4npG6byu0Mc1vOGyWpiIEIQ4rsVoMgD498bTOlIT3HzvuXvMgY99hoArAYptyihX88nhUanMUTfMXIWy3tRsb9FHJfBThV5wtGDec7CmQEF\\\\\\\/8wi8wNSfCDuz9iUU8f8lAkBCx\\\\\\\/ji00wABgHKH03oCW4M2S6JptYQBOnxs4fe4VLFt\\\\\\\/srg5tEX+tT1zfW48vlT1Is7R\\\\\\\/3prvRMyYbHi63RxqK6mdgafWDNqKKDYZI0ga+Qn1ToVm\\\\\\\/e4OJtJekiUIWGAZqY0orAQo666vP5jrDrKBKjts6WhhHJI41Pb3Ok3b1oEHtQWT4Kit7+QCESXuBtpCppmZk9FlBlBdQhsGzSjIupOYfrgElB5e8JtKSeeyQBih+\\\\\\\/OnKaal2mL7mEXvXXxDhxPW+YAUcQKcMstdLRxLQ8Vg5kUBK06rmMRp3v3BTG7dS7gJb2GbT24O2m5r7fYB+rTZSxy9786Hb1\\\\\\\/QqqhmBjKoP1WW5OWacCqvht14RS9WUNTKRp0pkLB35kDQQkJu9+jcMAJruI7NVYwfFMt7ezdRjCHDuDxMAVarYo+yqNjrQ9woUvyNfMhgmdI24sr1bldtKrEJNO2NsMEjLHFWAHcVX\\\\\\\/fEZOrrGOTZo6lNDXWto7wqEPbgACqZPBxZ6MTQrVlNht1riFE6Xg6h3LE2UYplih\\\\\\\/MVqWoqo8OX2xdYge73Jaq2PvR9wQy\\\\\\\/tqweq++jMcVnFL6OnEzufNxqLo4WeWriquziMX55QtZZEEDxRisxdOlMjeTU9vySday9YBYBKFSkjVh0QI1JNOLFDAQMAGpemlnZ9BHq\\\\\\\/v4a\\\\\\\/m7FckpEFzkYFHjiEGVkHQsXV\\\\\\\/e9CqufVHuBpnosSShEPr0Zuz6\\\\\\\/I8HpHvlZYXe2xAMbWOBXn3LeszmjQpQVIj7FI6pyaywTJu6SkfQl+gFqS\\\\\\\/LJtY0BQ5LYW54noyrzVikfGkwjXio8AOwRsifiBvWrHOAyYni2RACMvA5FsHlW0TqO6XMuIbmifJkjwcFgnRZLX\\\\\\\/3OvVeacwci93YWSx1l7U1BkVIMH15hiZCWKtm09jOOB0DwnbPBx3Hfldd9GbDbG8IhhXIKR2l9QZ6+maIhnEyBvY\\\\\\\/PjSoPSm4eQDTuYUMuwg\\\\\\\/ceJXrsL3CbqSuvNoDpcmWevkmoXLmlCcVJzkKWUnoY7Jyplk39dNMvvwQBcfRYuh9\\\\\\\/NDD7PINlC22vxS12EZOAzf7IBHHthSWLAfU5ZZBcddNbj9HZvaq9CMQGzUbmGlM1X4CujbMlGwQjWQq8szvBfapojn7RuHkUpLkxJFTGnkuDlsomq6pUIiC5UDL8Hd48y0ccpugVc\\\\\\\/+p4lR7L1Hij7g8+K8fMgU+ZdoX1G4ccvfDNqNmOw7XO6ZKIuoZWXI6hCY6otjSXrXJ9uYFcHdLV5GcFBWkxfTraxpEs6xSCZOGQdIkGUsNmF8jwhg5dPNvGtAgQVuSCKie+iFHQYeuHotlmD4W6+Ix3NJRSn035sfhb0HyyF61k1Z+usiR2Ta2F0e6mDDc2P2bP7qfCoeaM9TNKiBpXF6r9rCZGa6BwwiohSbMBV\\\\\\\/lvdYG8BKnTg+QewtUpa223LUdZgzhdgMs6lFTLAUQB+ML0Fmap20y1\\\\\\\/dj8DJy4okxRmWDWvPc2tHXJHndMz90BOfv9o5ZkZBrTd4y3C2KxsG4cagILqKaTMfJ80Lx8XjZsoT15UwqkC7c+6cob4e5OJAtDd1j4+\\\\\\\/1KbLxWXtTZKyJtX+BQOCWgcaWUPTgrDsaPFseic8jiAQRWvMQBwkVs6l7YE\\\\\\\/6CG9GIII7ig8ODfdT9VUBOSQT8RJFrr9Txu4aPBxy+qy1OQaUCXs7L+NpSw0AxZZkpfShQR3f9BEWkO+Bb1R2dneyskCMW3jhlDF5Sr4xmXRBPnrJG1dd+bu03vpzoVw1Sh0GN3i0fbKT5XiRdo0xhu8kb1DYPa8Z2NGgsJVRyD+fekRS339nPIdA1mg76\\\\\\\/O5t8fAGqzhKEfDVb5MwsMhtnSYgeZ8iUac7hXa2F6lO9Rx0AoEwXT0vszswdhpIvRFxxxuQuiU7Y7kNZtaHYEW6pORQQ6MNR3QVda10GtFrCW0jt7FUtFESaUuw7RqyPVSeTgH5vo7nB68RpBHd2yVd\\\\\\\/eOswAXJbK9f69EVK5MRjBwt7BDPHk792DWsZfSIyCNwQRhzEjSfoBdVgBFNmTzRmRQUwUMY3+2QHuBaf8WBw57Lg1bWl6QsRj4eLKyHrFzWnwmCGPB7FGE2lIc+qtpYGtcJuRVNikAUibq32UDhYgejKXxbMlF6YylQV+9mlLpLi3VuQcK35zqJjNBVSnV9c3c\\\\\\\/b8Jdd9W\\\\\\\/egaLjOKL4tTZLUiz4AosEA7a2\\\\\\\/IrwQWjyQN6GGIgEBLJDyK1JH9HhXr\\\\\\\/GIadGi2p0aschbQNR\\\\\\\/GFAnNQjhY04CuiTQLf\\\\\\\/gu6jqzLE1qT1nZvK2K+DWHkvlRprmtPYbdlYDAOx0xG49I9wJvnBN0RP809VCCKbrDqRgtK3vBpnMQQ+wZBkWiDljgrlA0Jn0iqyOG2XQCXBx4SrajBSYrvfJUk6hCrTTwQQr7YZ2zBatMvn3gh5kqy7SIxMeK1lETFDp7cSHQqQeLWiDtOLmkLHuJ1ShiMOEIqyefX1GA9juznn7f3thaEsZdTOs7b63KdHM\\\\\\\/6mxAZuWTwO2Ff\\\\\\\/mnFIqPUaoYZgq\\\\\\\/7B8oW3R6ASGO4GxKA8UfKVkQ4c0cq08OM34eF6tRhbrc8o8AhDv5Fy\\\\\\\/3zXLAsvpW+NqtMXR+B4bmuM2v4AY2aKLkdnZ0FBIkT9hU5tW8TA+jUCGZ3+SELJgEgvZOsAI56kXHqBGjcTM2MnV\\\\\\\/eYZRk2iz9cCco2kZ+E48AXHj4zt2XRja1mXRHKT+UHjFjCylmwszG3pltOMPXfsD8TmzeD40Qr\\\\\\\/Vi4Br7vdd6IQJuNNiXlwYoeH1RPFcwITAxpDhPD5xRav+EgtwynECKU5eeJgE8ldIuOfma49wEo3y7aJQk8o282YQ4V8mTDJIVIvHY69qpG5PN5cwHq+JzXNyLjk4hjG6W23hzs+eYL\\\\\\\/pyCokTF+q1d5Sc3h4isIJNTu5aZS7Md8rrjLA\\\\\\\/aXsIXo\\\\\\\/uJj2kWeQ7L6Ltl\\\\\\\/V0KD3qfJ6srRSrCKZss\\\\\\\/DuHPuj\\\\\\\/ks+UwcEJd7yRGdjJelYBa45yc6mvJg8+3uZRMHrZ9l2nHUMV+9bIMDj7D0WuITAjUyPbiAZEFSj7ManMvgoNqrEu\\\\\\\/z85MiRJoAwqHGLAIepW9W6gBOABYi2zUFFcpA\\\\\\\/B7Fu2mvM3O\\\\\\\/mJPR9kDGAUF3xTYelY\\\\\\\/HXZFh27SvIfUta8tbe4eGUiYIq5w8+UsqK3rjoDhaQouDK9QGZQYsKtsSiWjIXlgLrXEEiKpyuGcS6PYIiqSaB7jnZYn\\\\\\\/y7kMaOC3zzzmX1hvKTmBWPKsujpw5f83Of1E26u+E5Gl6BG7JiNX1fYCajtLkn8rJlm8dGdJvkYAIk6ojI8mSFmAca66EPwhn8D+XVLasVIQXmWqmgNXrljau\\\\\\\/StDrEyHymsO1GtoGzsLCT5LifEYoOswqlRlEmxQNAXPUZNPaX+NQBqUUU1IF4wa6gllX1eIuLM6EP9J+PrRlJkJCiUwgyHMOV0LCwpVx\\\\\\\/uP1Svp5+oHKfbGfcva7FsPQ\\\\\\\/4lgWYqs9IjaYd+fnBd1tzBI1w\\\\\\\/JReWK5V2WpXHZp4BXXoHnVNSsBb2o+M3\\\\\\\/j3YrCebEB5oc9P61AvqsG31nKkbHIkOUoy3DIiFbUVME3bd7LclmYs8McCFMKnLbZgDF7qWVAC5EmfMvqZEhipZmmXVgpdpoL0RWwLwNlOBLO\\\\\\\/xd4fL3e9E65qt0o\\\\\\\/sV21MG9mV3JoklDUc6nhYLyF8W0FkV8amyNHhFn1UkbG732qWuYln9wge6e8ui12RKZZ5C5lAiK8txmCldcWNk9jkjRERdA9azssQY+z3FhZngIMG9NHGJm33Im4WArFi4XKpeBY5Z3n1zqRGjmb6RwAoQBgFwbNKodtqveHyqM7Bvl9s0fBesKKH5WX7\\\\\\\/bJExO8LEt5dfsjKqoSG\\\\\\\/2DiWzBbCs1RxNlJZXnghW2yafhicya839khW+60hGd3he4GDvAkBQ0u84RhLWF2fmBm4uMepghKJJM2Dlt4U4aYdijJt5D+r2wfX2z6mBAb4U2c\\\\\\\/xI1BKoM1M1\\\\\\\/i9IOvhSe2dZFkHDEf5MPO8Jxs8pDfpllshRzFRjTGm7HKepX43wRgz4b4RJDuBhKxbVP6znpUoQTWW5LX0rnIBruZRenJ11RjDnNkpZVxbOjHNn\\\\\\\/PLY7LFj6\\\\\\\/yzq9AAVRBMtWq06ZJFl11yljPGOBRwezXvfOcevBhtbT21xYGUruh2PJNDREqbvhBKevSLh1T+L\\\\\\\/UV15tLLoqmEsysuLehZI0xQHq4yq4qcHFPDnEcgaeOFhlG6SltSSoR4bWqrE35a0igcep8DKejQBU5wgkC1MHECFamZcamJW3VKkzeIZ58PcDsXuNOLxqRCSRSiJWuANjbvaHTrTMl9ce7xDr6H47In9mhOneuEHafs2M7Acz\\\\\\\/qLInnxFGaf3hvPmZesQzWdt5K9k1TTTPuFcfeTONtx7X8pgSwhGz1t1JBkRnQ4Z3uJX7GA3Lbcam1D8VoTQRatXJWuzFMKwcDfVjrN1HxUW4APs7bjcZcd4iV5C1yCgxnXDoDQ0WJF5HmJh9tgsNixV6eztnBDo6UyWB8pEzoSspWKfyNjWK9kkA4VtgcNwGCUZMVvH275kKVveedB0RcIVUd9fuUo7ZmVglyiOdQyNd5Aon9X+HfD3MQOfWeeDitIlSTbuJKyA+\\\\\\\/lu2FXQyeeykQMZm9v4AH16iTlGKOuTljnxRowiTk3qF0BeOobRajbbG8hVTPJ8+xFCSPHUD5odpfGFXdZfVSJMX7PLR0G03kE2mYh+bmmmk5IF\\\\\\\/mk+RS\\\\\\\/sSwEDQmxeklAWmL7s1CX8EpDV7o3Q8Cxu9SNG7UKvzK\\\\\\\/a5bQyP3yeIRJjRLCwPA9d1kP7rS5WLG2mpeGcCDuVlFzBAeOpX8O5vHeE4ZDiSTze3\\\\\\\/jmB8H2Rn5ZRRZh22UIfqjJUmzHDuuzD86npPohW4y8nFUCSqaFwVL3CAhe9vBDn3YJTpmxGRQeUdVFOgRxqFEqckqJNFt7SUMF6edbL3O+T4a0utW1KXgdLJNDfkeAVQz3fjAj7Mngn6f1sSuHiLiaX2asTA04SUzdepHkmYeb6+Djr3SoznGbulo06hYrVwYwOelDoEbXwk5CwPoMGse8kfRbLPAWcH6VGBBEGfdwpjpYhJ8\\\\\\\/1C63J6fx8I0hRsJKsAdzUcQX+hB9kstHb1MFVSyCEBefisOo9tKmHc0QXbzanwSRCpq44o0GwuT2yUz99QT2Izr2PGTnzBv8gW2JhdLTkqyrInX7JC3Cf0MaQUP6bNTgEsieyKPWqCSf\\\\\\\/iNV754feB1AFYethZgcvF7yxHz1uEs7v\\\\\\\/2hnmNFNY5ZaWk76QUoWPgAMbw5YXqqVesOGKy2BNjzFFCDOTuqnTn1zalJB5LjX3UmlvMLsTb8qWMGiVlYWcPd4DANJKMVT2USEg+wCHOPiAzRzOkbx9ihu3n99QYAs6Pjye\\\\\\\/LT+HTYG5KLn\\\\\\\/Qq5C3Dc978kZz41QAGkroXeATgsuXvmlDDjNOD2wTrAbXKsjhoVdNPsD2wZKqKU\\\\\\\/UVHW2vwLM\\\\\\\/3LRJzkrbtBwtQ6+4\\\\\\\/dNrJwcJGa+NR8lRUHb85qyuiYxjmiGXUUtdEClUKDLM3FOab\\\\\\\/AITW81mPPI8Z5CYfBaMu7hNdL\\\\\\\/51JfKVW4tnpa5tKj6GGLskK94w0HOMqJHPCWOrWgeNUzyaeNx80dcBw0P8mOLyW8ZYOclhMrPwMk\\\\\\\/\\\\\\\/CFPfezig69yTIpny+SkrwijbBS3vuHqV0IM3fxrgb7ila5NEcHrMBbjcHNqzQkRdt\\\\\\\/H4eOvcPtxruloPkJvFnHt1Q39VpS+oTYpLpT7h5c1eogQDQUR4oG+kOLVQI1wJrKsCxAGHU82e6jMDGVqgZbF+uKP7Cm4W5yrHdH5m2h4R0bxOF1ybjp8lLYh6ZorzHY\\\\\\\/ELplQrXPGLk8ufdRpp9rO3ewW9eKyP26DQxfJtdNGPO9It99QzuwWhS7vSUELWHkYXCUEMP84UqbfL7AU7RqJm40c6Zyipt2r4xXoWvl\\\\\\\/Q8GpPuso7r9\\\\\\\/keqNvFBI0fsiSLqOnaTec1ySctbMVU7y\\\\\\\/OxrKgK6tjR28Pl1adzYtCfGnF\\\\\\\/zkDDZtowImkOuHIUkS885FZj8vetuIKeNZjbyi3LbfKi+Bjmrt681azbXeSP+8\\\\\\\/pr9O1P6i11w+0dDLQoZeHM3NyshYt91ltxV2S+dEnafpZS0K2dlA+UZQL7akrILnOMtgnTi7n+IuT51TdWpdncvFrobdifoqBk1oHlNzoHnA1CXIG0vMKRVy\\\\\\\/\\\\\\\/AzV+SHDLmWmyEPfCHPsRrl3siOEmbTUpUuShom6aBeqzIsweZrnl9o2hdERBhw4ZFFCrtQMXPZmUYOAhmU1lt3LtrJFsMiopP9e2RpII1L9ZpsSxWv2eBuLKQ36ECIk790DrgyppoFtQYXkJRlE+dBt7PdNmyU7p3UvrfeZyoUc1U6mgz7M+my4nH2R0tv2oF4\\\\\\\/995beP0UiUw1cuGRUsS0WJ9x8c2pGJHgQy9BUH0S6wnnyCvdnhkMQ7PQfWCxgOJ+Rgvu4MmYFpIJHBlA0eO9tfp\\\\\\\/5aVYKvN3pdypLbBEtzlPJ3mgB+16BKZwjy8qxrzdZe6eUhe1ap7OkfpR\\\\\\\/3r3nUYMcGkiK7OdZ+vCY9IuESww4FCi02e9C1fV5Z\\\\\\\/Js6bam7RvXULcQ42CwHy8Sl\\\\\\\/9T\\\\\\\/NpXcavYDmyj8ulRqPNDbtji8dUKDXIMd6Rw\\\\\\\/iQsItiqtmLIekrbDsCcEKR24\\\\\\\/PMJh2GxZcwsC2laIiktTaRyMUhrL\\\\\\\/uUDDTaTVwtWPaibTCpHXR3WB\\\\\\\/LZeqGAq94pG\\\\\\\/sbNEWrlh5QOLzNQqYe1cy35uBqS50niIAYwDOGqZECDCCACheYxVjpxdzzUPGkHKJixut3UQNiyZMguj1CegLtrnnQVWak4viqICm5LgNtD1geM5inqfSw\\\\\\\/s8n85u61qc+IZ+aKpSKbDV1s7oHVqMvLMd+MW6m9dyV7P0mX7hM77Q\\\\\\\/1kmS2dIk2e1\\\\\\\/MQ6J8XqGgmdpOEapWfRZPKO6j8vpMEcT8iUZ\\\\\\\/x4cUVNJORkSJmVETYruAY1vgS6wNbabkocfpT9iwEWu4rmUm0UJq4VSpq83ilqIeuyvBhzeAgiLvhboQGPsdKMnatp8g34fWZKuiL6uG8jJfjExDvzB6\\\\\\\/N1ZQ9T7TRSPO5mzYfd4FzhpUGpOW1C9IDtGIeRtMAwqfZ3pbsX4KqPCwF5fP4w\\\\\\\/mw6teLVUBq2xWZOOYxr0Webo5eJcU81LEwntGE2Lt94ySahae5d52iCzqzf04NjX261eVO5QlyCfm1QNI48\\\\\\\/zl3FGhowhE8M3HZr0456kbbCwukU5V9wkMsjYt6qMC9Z8kbvbgcNgemFYejUjkxVL4Qnqy\\\\\\\/cYqFiFlkbvZzXVuwJK5c8IiUY2rHiO4gm0XPa453+iQnfXQv7UTDFHkKyGSLAR+Q\\\\\\\/A4q4UdKYyFF5OljPt74\\\\\\\/94uTevXXgrAle60vziCcwsCK2ebj+2CwHlLOP5BtEi3mrRdOAAv30TtlX7xcYLsblVqlylVHx3sCxHVFvQOaTgAD9yz6fOgeiFmT3TFGqIKCnj64MBbbKWldYhiAw0O5Bhduqkz4WFx4PT1Grmm+n0ZoyO9GSCEHkXk5xkFmbjnWqEcCUZb3g1Z3O+41bwKS1NXTLOjfVbtBHRRy\\\\\\\/odc\\\\\\\/Bd5fib9M0Jq+yPJU1qswG9wqYWFBpFYkd4OT2YnMFH24ex2R6yri4KDwMK\\\\\\\/KMGAMy3Yqm0wkncdGzJMrJ\\\\\\\/bLPJ6Tb5f2w5OW2DkPHwSCHhQLPlnK6hENKdg7v1q6uE48tG4BPr3Hy6LBSDy+nP16Zeslc7glM1d8t2EHeNjFU0MjqSVGzBJpsbZncsK9xY+zqExZmJySRSGxg8AcDH\\\\\\\/45F+XlWffs8JCFrjJ0kvu1I4ZAz1uTVWPjdfh+b5dubKqDxAbVVzMeBajaTxWM0BmHgQgjt0PM7LrYRbT\\\\\\\/Kzy27FrlZlz4ztGoR\\\\\\\/J+9LgXTVnQIxQjEFTihACVO0V4ArSuok1iLYcOMSy05qatzfzhO7ivARuj71CavEav2P+igA44cJXRB8YRGpXYSiAK5xP40QM+JlpIZdvFj0rLufRMHkN3Da3xBBz25eiof0Nc+Q2\\\\\\\/xUQMGBIyyU21+S7hU94ED4\\\\\\\/\\\\\\\/vpqA\\\\\\\/X503Ht660nXpBtWRuz0yNy1JwuuS\\\\\\\/tv3AIM6ecUmoOX62uMT+HtRHBVfiUALhxyBzd4ibihRcDUNIxjowC0UrKBFDezhi17OxsQcq\\\\\\\/xzAk4mgQ4ED4jdW8\\\\\\\/CuO9Ol94uy5i5JzEbiEXnWNnJbrD59118xx9q2T3iG31xbPZmebF3DDaZb0PXc8P1BGFIwo5OGUF+GeH\\\\\\\/XnuXYAcrWyS5\\\\\\\/TOKv0rlJUQck7AfpbLOx5sPxuoaumk20jeKS4VMe+jNciWANzgVwAMt636ywDdaWZSxajW+5fPwTKhYtCQWAgFiT97MLPgJ19KWz1u1vkDM\\\\\\\/q\\\\\\\/BeuL3GjMbY6xFoSvFWjq3cu7GVjJ+bDQ1ls2orFpPplhmZ+XhH7+UC2z11jdkplqXoLPuJJx+fSLJNhqe0Ati\\\\\\\/iK3GfY4GspP+EGNyGuXOWmF25IuBmbAs2MKQG35\\\\\\\/b87Z7XLINbZlSH5jGPS98eWKsQnImegAE6POM+JdKzHB+m4lriyYMLPx5nLaUuIZQLbhVdBB8I7vxsFXb4nrgs9EKLnJs8FNxFcdRTnMUE5+AJPTzVelRLgecuJwPSFd0IlKsoyN8M47yqKD3KNAUuy5sfoKlDqVbq00BQzmKuJE62tbLNbXXRNY0EGLy+JsN14+ayDWy0Ch2JPRziUUB89vONauTjH5ApFMyOpKaK6v8bRNW0LOUzZpN3x1j5gbnRXEdplX+lTk5U\\\\\\\/VlLZ+U6Oao8gzWLBBBaBDyFvJAFg\\\\\\\/E6E4fJQjX1Tyr3Y62pOYOYc0jUWEMDW75wzwmG954nT++QPEOtlD34\\\\\\\/VBd4cW74f9tC\\\\\\\/\\\\\\\/fRR1r0F0M+Xg21aCKz8\\\\\\\/BSf\\\\\\\/M9QsgIs+T78cdBjq\\\\\\\/epo0YOI\\\\\\\/7yFG96nUADh\\\\\\\/MUk7yIgZhJMQtb+mOdKdUbgKcLeMi4GsWi\\\\\\\/sPwAAtO\\\\\\\/tl3I2i3LC3Z5QI\\\\\\\/Zyk9qXGVNF5MOwp1qTytPYyUhPIS9J5xbuV8iFdLZwBXEWbc5iSMyofShsnpdCMQ0EImLNs48QHzE8udtRZJPU9TluM7Mjx6gttM06mDZ+olLxI6a6Gy\\\\\\\/jXBmgiq8hf0mUSyUtiIAdHs7AsGKVS1+pNdtdSBTqv8jypyz7m5SbXFH5PX5MPhJShqNuvjLUrBhOGdEyJD1ywDgQpfuo4mHO++\\\\\\\/ATmkvkqjeLPwczOO6y9DXhmTtMBo6NU0Lnu3b\\\\\\\/FxTEWA0xhGM7cz4S7V2k656e6A1POD53KJqj0BBk+lkMjr59ns4obfBF85hWW\\\\\\\/pt8XfpDuCGg1SZtW8JTRTlQQQO+b9bhHUUg8oOQTbJ6SPe7Xpk1NUpQfWaMKE4pWIfrjzfo0hh0BoEx5LS4Y1vSkF5AbpkMXmBnU+2V2Yw5zMSpXGoj8eKrMtUDCH18kDPNZwDZVbUK7A28fBrKfTQeqCl90wOT9E2Yks0kNkpU2Qj3UDTLZ4rMbUvrN94a8iDMlpo96AkjaWv5UYbRh\\\\\\\/gtenIsQqyxAhzsBMyz0TI3o+t94ScYNbimDUucFTc90+00NZVNNahEyYzs3Z5AyM4h4CHsgVYEhIVL2e2\\\\\\\/mm2tHfp9RkzbsTj9nZYRG5oDXHUda2O5vpDeP8aGqVNCScVr2M+Dwm08VdadkZMhPJ3dJu08nmfWOKoWKsc5ZXKZN3rqnxyG0ddHWNume1JD58cz80nDpwg1RigTG3v5o4BNvvyi3IiXmjuZrtJIOjFLxykCLoPp13\\\\\\\/R0pnrOq3wyUz39mhpimkoefpg1Jl3qENFgm9VgXEBSSuWxPhcWLojjHP+qHTrorlm+0qwLnNXktx3JIPLZidRs1Q+y3VL15IJzJ1Qx+w2\\\\\\\/qJNfL2KCq9ysvxmMJ6gDX\\\\\\\/pZdHfwOIVkQvUjgkcSD3O5e1lFm9cc3V9wI8UhZQ5+zM5Bwgi255YnGrsvUMrFWjZQ2JB9kQ\\\\\\\/\\\\\\\/t0AK9FSnI0YXpiKEYzz9N10kwnsjYuqmEu8b8PmjcFkRSC7H47yqYGB808ceIoEL41FKUOMDguDhMg1gnmadL3aqvkq0Eg9c81j2cmtl3gvCq3+IWtoV7svrsSA84XnkaUSKtWh3eXbhgZiXgbwHZpeSnFmR7ftwOQ7Ehqf5Vf3GJmSQfsDWR4wdmS6RvhobQlhUBrrZtEcLPDY2Zse+wPgfp3EAlS5BTW9UJy\\\\\\\/K1eE4UL\\\\\\\/G3VW9CTRwUbbPQ\\\\\\\/caaN\\\\\\\/vHFCGFsPxOY12aZLAXlxJWbEWxhSmL9s2gEiM+LjXshhYEle8AO\\\\\\\/ATCeBEcl4ymG9ewCLiJ32F1UB0gcHaT7W5H\\\\\\\/zMLwpkzkGBq+agL3tjM3dOBsdnjD5d3kse8SGJn6g6dyBMP9XjIA5sOPkXfQ5FBdmZDRBPz9QhHEIY4kD53Gx3mhtZJSolSBkB\\\\\\\/+dtFYbHwZAUWM9jm8YEnjgi1BClLajxMXxIVr4DvhrNEG5hh\\\\\\\/KzDTxfhK579GCdsQyAzYQXHH9j\\\\\\\/vtZ\\\\\\\/pyyy+3GeTDoOIdgzv9VvUP0fAIx5VI10S4fhUn88MUf88mFspHpdY8wOERzZMooGiXCIUhX33UVzwXLs+WDEfPnKP8PACZ0X27Zl4hFvQrl4qzw81OfUHSwuqy3Tj3CoW4GE2XhB6eRpYBp0uHbf2MDiKI18ZBa9QeZcumOxkxCS0LIRhRMr4zi\\\\\\\/+p9KGlo4S6UTqgUmQhsUJkEIC6On0szMZA3TOQE\\\\\\\/vIFt6pUcb\\\\\\\/Qb7qB9tyaimSCMMMPKF8m1ozS4qhvhVQDzqJpR8O160HYZWU6ATvpVcl2oK5ftPOdhiGSTkQEAasz9reWjUulBJJRuO8P0DlIXdP2iGLWjKYmDJcuJU4t0MtaLP1TcIR2aqfzF1SWqyzanoCVrtjPoELXM6CBfZxWgez4LPCFNAPHA\\\\\\\/s5JNQHzr0D+TgR8k2yn3bBqAY0NLwz8Hkd7zPXk4HTZMtSFsjzsbZNEpg5HrSXltfjovkIRplgE2cYgnKXLA8\\\\\\\/AaviCkiEZkPi+nHqCqHB4cLHgZM2R7Eg3yr11hqXtDdzcRxQDsLDD0RJ2cDrQbQ1WNTqy4VW01tAGG8HeDbxeO36K4HmbdsFEnJVbjwsmyoKIDaLTymOUWG2APoLVwSabpyHuzHXqt1cCVgqPk9k5nXKyAai+qN7BrYRssrkoPBKeZfa\\\\\\\/IvMZOnSGrPUHmLyfRPpDzmmN7C+kwL9KOpUfHgpv92278w8x6vCDlDp7y1zC14xHYTSOvyLUqnr09lSK8k1rS5Bu3qiy+FH6cn7N4w60Yttj1TyIhGyKD55Lp3zsYQl0jnQMhsKQ27QDWXr5lDX2ymGNshXYgBEnvyIRZHM5vS0yFeSYsrvWIQpGVt8ZZZLi3tOO\\\\\\\/2uLRinqDrKWn95V6fhKSBFIE6nOm7Dy2LQCc1ZEp7wHc62nA6LDWQIUDg7rMcn2swoDyUImZV7dUk0k9zI68fvF1drQbWFQe838MTBiS6rqVUcz5MUCSsMWc8TtXN1DK6LYtCVkyNO66o2GpfxKMFnHOOdkhIXso+6b+1WPIG5JCu8a03qvejkv1aI8vzZtYR+ZXqEm\\\\\\\/KwhX1yzmIUsx5Qt2oPG4t01cEDABWXJ7faq9AkT+hQi+gbPkESvw3zDd\\\\\\\/ALNg6Mp7xJZWDAAt2dmPmWktuHjiV0aDOqa0LIPXPlomeV1Di3jRsC71WOiMU8\\\\\\\/Ddl9CzZ+lcBGcAP3fjql\\\\\\\/5VaoXTYsLr6WbWYWXhImSnTk0kDZxKtsTpDCs3greMHpLhfWW4NZ3XSMwHdVtrMti0ybpIdnUqo+p0bKmkqZRPWsexqKBSlQyB1xsHgOsonK3Gp\\\\\\\/0B1sGo0EjfnLyBxGIM6I49SaDqW8Qh5lbWcS\\\\\\\/YaevIJlarql0qkr5ICEZXpwGbvoV8O2cRYOZ8mI7yuACyvLJHXwaTMnlnX4xah7TCYCo5OnLDLRUPz1D9a3mhIzTjmO6QzaHXsI7q8E\\\\\\\/nXlZkJeRWYqIONQvqznb40bC74nkmkOvgkInkRjQ2skKg5Aq3W5UkoOXSGEuKKvqxKd3FwUV1FZqp9q16E6M5WBmwXfJ1MepYVDQvLxBO8GSyPctf8yF9T8MlPb5Tr0htHqP2CZY1vuFYhz8hEuiUag==\\\u0022,\\\u0022iv\\\u0022:\\\u0022inDU8TAhvGy9nw4I5SMHrg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223ee6dc5536369b9796ac561db63b590b258c53e330a5e32f851edfcea3bd6c3b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022x\\\\\\\/BTDBZ49yayDFUPQ1trQKPLaKMTJHmE0jYCVSHTvB+V9p5bOIiPf9rSvdIj4VNjlhn9IBwQsXbYfzSYsZTHWHM6uF78wnFpPmrbVrXr6EQIwuJnHQDaPXlzDm0rN8QoddVCVVRA+hJ7yIz+2ygjW6jCItchXDbeqMhanTSOVW0iGLdtHwT9\\\\\\\/FZ2bgTQI5MQDyE7SWUASiIfYCu5fkT3yijF\\\\\\\/EirUmmhkYwrbT8Aoz4NhHo60Heu9HvuOE4qtWN2S2uWz+G2XybN0mBE3roNBo6QKUgMWZpouAPVBlO5rSGZdxKn3YbiQsmaIek1ZASYnCpR7V+nNv0SnSRqVWA3Ok0pMpBq+KNxJcmFefAQVtAgx7DfHvuLJSM42desHbEmoi7AeiOy5W9Rh5CMBbzLqYUks6KMAprzOCtcHJk2j7bwSAM+Bg5NwszmaAI3qnz80aYEer1hqKDP2iCWhLo4PKh4tYVG13nIN9FkOhSlqRNWqdByPy7gN\\\\\\\/tyfqGRCV\\\\\\\/5t\\\\\\\/FAn3IHEQwni5OOofWQii+PJ4CGo5YzK\\\\\\\/gIHDHalWy3VrHmbwAn0oNWFOpAWbmop9re0jKDlwWWURaM+\\\\\\\/BKnKgTYfLRH2AagpowxnTn5\\\\\\\/HN0QM72Qu8Jl6npa3iPiybREfsl\\\\\\\/nAs8Q5QH8UvkLMtQt3maDUssAmGDS10f9tcCCN2wIoybqBb4H+EXfPVeia9FcNfZ94xbJn+MKqxAzwxb7pa3VIBIjbfOVHT79V5PP6N1fD\\\\\\\/uQKPm0W7ui9GJ99pufubUt48UC481TycKuOSt1RzPmTTLmNucFtYDexI8TR9UH99a4\\\\\\\/Qge8foMk6xeGbxAYmtWa2FUcnHtAMaTJhAXwD73JMjxB019GvcC\\\\\\\/UE8wIDt7jwEsX+p5yvI6kKG8bwCgA9aLAs1pgAyH62IdUTIeQEjwH3A0FPRv01\\\\\\\/u39tPnBQZAfiJ5Vd83fvO4vAKmYIZi7YBJycD6pzfZkHBu75wGbkHOmZoTgWF5gi0ocHohqRnvrjm5jtCBfK5yIQS8fZDFQaJA00eGK25UI8TxUpX6d44Wda9OCp0j4na1kh+1gDVdQXWJ\\\\\\\/76x9w4X6rIDbBCaj6gZcnZDJDWcvEYwGhVflHDPcyP+BcodutnYqgeh6C4RPPQEvDDzM0mjw\\\\\\\/MLi7VCpg3HgFPAtX+Cr39ZOk+iOBu+i9OIlm\\\\\\\/xTLo7TYZ4SIfVRVFnyPRD3lJxg2DQposT1qMUMoBj96ZnyKagk\\\\\\\/5aATdOBOWnu5OU7eukb\\\\\\\/cHqBPdvCNkFxzZ5lj2ZtSVOl2b9Alg\\\\\\\/vbY80anR55miEDg\\\\\\\/Cmh3ydSsaek+ud9UQfFQajrD+1dq38WhPQJPy88amj\\\\\\\/awWoxCeIl3HEXzbGvQ7rOJ\\\\\\\/04XPquMCnz68PR12wW2PuDXslEQDoeIYsNFoz\\\\\\\/k\\\\\\\/tA\\\\\\\/jWcXi3k9aWkaYcQiU05cLAtvyf9wEZpJDM3+0vljKHpNi+u1VqSyrPvC8sZiX5\\\\\\\/kH5fb3VRFdM8pJMaB\\\\\\\/oCjyLjknfG95zWQpSVguabLuxNKmAQbzjBvQzcvB4lDQI4JB39woVWrXjjw4Ul6sgWmTErrsWfl2XoHmBzJEjoTiIMFkLkhtvR15KV8jleVnR98o9t6QoMrMEKfJeVbYLu2yDKt1Csu1crW88VI2JVmM3UyXysegg4+CsjYj5wpfU37NUsN+G2wJsXndaFde+tSvUCVLfm3F17254x0hnvRKsToaKSBzC6YaEa3rSUly4zV5Ormsm4sbkLq8Xhl8BF5eGyvAa4OvkSCGCYqhq0Y10foo5GYJd3Fj\\\\\\\/P51qmHUD5m5ytv5PjHZeQE4T7fZTj09\\\\\\\/Trapi7lG1DC7UMc1KEqVwUGhn8fX7xOlZVry5AL7vsqxh3OwtJSlIpMZZ59VJtEfROT2O4Z7L25noe\\\\\\\/rCe3eiLSVFIfnN0dboYAErQek\\\\\\\/Y+8TQUSg5RYuFh45vGy+xMaiipw\\\\\\\/Rmo5beIYDLNautz4l7LtSvKUQEuAtU6EsVnbgngg7uFUvQHe0Ndg==\\\u0022,\\\u0022iv\\\u0022:\\\u00222w3WHdRC0tKwhnHtYVfEgQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b19d2c452754ea3cdca696330249d68d96044015ff22c5ebb61c0b19111e2904\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492866 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1571" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iLwSZzYi1_wZ8PflspIB4OySx3ZPsa1oc7OU7zvngDE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1571" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-l3tWlLfimTdMh6em8tAkjAwJQXBiQ0Lr67kWuGg42OE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1557" title="Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1581" title="Genetic and Molecular Analysis of Essential Genes in Centromeric Heterochromatin of the Left Arm of Chromosome 3 in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492866" id="node492866" data-pisa="ggg;9/5/1571" data-pisa-master="ggg;g3.119.400165" data-apath="/ggg/9/5/1571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Cleaning Genotype Data from Diversity Outbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0667-9926" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Gatti</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7928-1911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karen L. Svenson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Śaunak Sen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-9190-9284" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1571-1579;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400165 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492866" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin–Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:broman@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">broman@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0667-9926" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Gatti</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen L. Svenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BL.%2BSvenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Svenson%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaren%2BL.%2BSvenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7928-1911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karen L. Svenson</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Śaunak Sen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Preventive Medicine, University of Tennessee Health Sciences Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C5%9Aaunak%2BSen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sen%20%C5%9A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C5%259Aaunak%2BSen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Śaunak Sen</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9190-9284" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gary A. Churchill</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1571" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492866" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492866/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1571.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492866" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492866/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1571.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492866" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492866/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1571.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492866" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492866/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Data cleaning is an important first step in most statistical analyses, including efforts to map the genetic loci that contribute to variation in quantitative traits. Here we illustrate approaches to quality control and cleaning of array-based genotyping data for multiparent populations (experimental crosses derived from more than two founder strains), using MegaMUGA array data from a set of 291 Diversity Outbred (DO) mice. Our approach employs data visualizations that can reveal problems at the level of individual mice or with individual SNP markers. We find that the proportion of missing genotypes for each mouse is an effective indicator of sample quality. We use microarray probe intensities for SNPs on the X and Y chromosomes to confirm the sex of each mouse, and we use the proportion of matching SNP genotypes between pairs of mice to detect sample duplicates. We use a hidden Markov model (HMM) reconstruction of the founder haplotype mosaic across each mouse genome to estimate the number of crossovers and to identify potential genotyping errors. To evaluate marker quality, we find that missing data and genotyping error rates are the most effective diagnostics. We also examine the SNP genotype frequencies with markers grouped according to their minor allele frequency in the founder strains. For markers with high apparent error rates, a scatterplot of the allele-specific probe intensities can reveal the underlying cause of incorrect genotype calls. The decision to include or exclude low-quality samples can have a significant impact on the mapping results for a given study. We find that the impact of low-quality markers on a given study is often minimal, but reporting problematic markers can improve the utility of the genotyping array across many studies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/data-cleaning" class="hw-term hw-article-keyword hw-article-keyword-data-cleaning" rel="nofollow">data cleaning</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/quantitative-trait-loci" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci" rel="nofollow">quantitative trait loci</a></li><li class="kwd"><a href="/keyword/data-diagnostics" class="hw-term hw-article-keyword hw-article-keyword-data-diagnostics" rel="nofollow">data diagnostics</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-inter-cross-magic" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-inter-cross-magic" rel="nofollow">Multiparent Advanced Generation Inter-Cross (MAGIC)</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Data cleaning is a critical first step in analyses to map quantitative trait loci (QTL). Genotyping errors and especially the inclusion of poor quality or erroneously labeled samples can reduce the power to detect QTL. Despite its importance, relatively little has been written about the data cleaning process. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lincoln and Lander (1992)</a> discussed the detection of genotyping errors in genetic map construction. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Broman and Sen (2009</a>, Ch. 3) discussed data cleaning more broadly, in biparental crosses. <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Morgan (2016)</a>, in presenting the R package argyle for quality control for SNP genotyping arrays, emphasized consideration of the amount of missing genotypes, frequency of heterozygotes, and the distribution of array intensities. Here we address data cleaning with an emphasis on multiparent populations and array-based genotyping.</p><p id="p-4">Multiparent populations are experimental crosses derived from more than two founder strains. They have become a popular tool for complex trait genetics in experimental organisms. Examples include heterogeneous stock (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mott <em>et al.</em> 2000</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mott and Flint 2002</a>), MAGIC lines (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cavanagh <em>et al.</em> 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kover <em>et al.</em> 2009</a>), the Collaborative Cross (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Churchill <em>et al.</em> 2004</a>), and Diversity Outbred mice (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Churchill <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Svenson <em>et al.</em> 2012</a>). Genotype data cleaning is more difficult in multiparent populations as individual SNP markers are generally limited to two alleles and thus multiple marker genotypes are required to uniquely identify the founder strain origins at any locus.</p><p id="p-5">We illustrate our process for cleaning genotype data using MegaMUGA SNP array data (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Morgan <em>et al.</em> 2016</a>) on 291 Diversity Outbred (DO) mice. The SNP probe selection for this and other MUGA platforms has been optimized to distinguish among the founder haplotypes of the DO. A key principle that guides our approach to data cleaning is to think about what might have gone wrong, and how it might be revealed in the data. We visualize the data in many ways, and when we see something unexpected, we try to determine the underlying cause.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Mice and genotypes</h3><p id="p-6">We consider genotype data for 291 Diversity Outbred (DO) mice, including 99 mice from generation 8 and 192 mice from generation 11. These are a subset of the mice considered in <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gatti <em>et al.</em> (2017)</a>.</p><p id="p-7">The mice were genotyped using the MegaMUGA SNP array (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Morgan <em>et al.</em> 2016</a>), which includes 77,808 markers. The genotyping was performed at Neogen (Lincoln, NE). Genotype calls using pairs of nucleotides A, C, G, and T were converted to genotypes AA, AB, BB, with A denoting the allele that was most frequent among the eight founder strains (assigned arbitrarily when the two alleles were equally frequent).</p></div><div id="sec-3" class="subsection"><h3>Statistical methods</h3><p id="p-8">Analyses were conducted in R (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">R Core Team 2018</a>) and with R/qtl2 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2019</a>). Most of the statistical analyses involves visualization of summary statistics.</p><p id="p-9">To reconstruct the genomes of the DO mice, we use a hidden Markov model (see <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Broman and Sen 2009</a>, App. D). The transition probabilities for DO mice were taken from <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broman (2012b)</a>, which uses the results of <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Broman (2012a)</a>. We used the genetic map from <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cox <em>et al.</em> (2009)</a>, assumed a genotyping error rate of 0.2%, and used the Carter-Falconer map function (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carter and Falconer 1951</a>). The HMM provides a probability for each possible 36-state diplotype at each marker for each mouse. The term diplotype refers to a pair of founder haplotypes. The 36 diplotypes consist of 8 homozygotes and 28 heterozygotes.</p><p id="p-10">To estimate the number of crossovers in each individual, we inferred the 36-state diplotype at each locus as the state with maximum marginal probability, provided that it was &gt;0.5. In cases where all diplotypes had probability &lt; 0.5, the diplotype state was treated as missing. We then calculated the minimum number of crossovers consistent with these inferred diplotypes.</p><p id="p-11">To identify potential genotyping errors, we calculated the genotyping error LOD scores of <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Lincoln and Lander (1992)</a>, by first converting the 36-state diplotype probabilities to 3-state SNP genotype probabilities, using the SNP genotypes in the eight founder strains. We then used the observed SNP genotype to calculate the error LOD score as in equation 1b of <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Lincoln and Lander (1992)</a>.</p><p id="p-12">To obtain predicted SNP genotypes for each DO mouse, we collapsed the predicted 36-state diplotypes to 3-state SNP genotypes using the SNP genotypes in the founders. The predicted genotypes can differ from the observed genotypes due to the smoothing effects of the HMM haplotype reconstruction.</p></div><div id="sec-4" class="subsection"><h3>Data and software availability</h3><p id="p-13">The raw genotype data are available at FigShare (<a href="https://doi.org/10.6084/m9.figshare.7359542.v1">https://doi.org/10.6084/m9.figshare.7359542.v1</a>). They are also available in the R/qtl2 input format at <a href="https://github.com/rqtl/qtl2data">https://github.com/rqtl/qtl2data</a>. The SNP genotypes for the founder strains are at FigShare, <a href="https://doi.org/10.6084/m9.figshare.5404750.v2">https://doi.org/10.6084/m9.figshare.5404750.v2</a>. We used annotations for the MegaMUGA genotyping array from <a href="https://github.com/kbroman/MUGAarrays">https://github.com/kbroman/MUGAarrays</a>. R/qtl2 is available at <a href="https://kbroman.org/qtl2">https://kbroman.org/qtl2</a> and at GitHub, <a href="https://github.com/rqtl/qtl2">https://github.com/rqtl/qtl2</a>. The custom R scripts used for our analyses and to create the figures are at GitHub (<a href="https://github.com/kbroman/Paper_MPPdiag">https://github.com/kbroman/Paper_MPPdiag</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7848395">https://doi.org/10.25387/g3.7848395</a>.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><p id="p-14">We consider data from a population of 291 Diversity Outbred (DO) mice (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gatti <em>et al.</em> 2017</a>), including 99 mice from generation 8 and 192 mice from generation 11. There are 150 females and 141 males. The mice were genotyped with the MegaMUGA array, which includes 77,808 markers, but we focus on the 69,339 markers that are polymorphic among the eight founder strains.</p><div id="sec-6" class="subsection"><h3>Sample diagnostics</h3><div id="sec-7" class="subsection"><h4>Missing data:</h4><p id="p-15">Our first step in genotype data diagnostics is to look at the proportion of missing genotypes in each mouse, as this is a key indicator of sample quality. Samples with a high proportion of missing genotypes are likely low quality.</p><p id="p-16">The percent missing genotypes by mouse are displayed in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The bulk of mice are missing very little data, but there are 24 mice with &gt;2% missing genotypes, including nine mice with ≥20% missing. The mice with high rates of missing genotypes will show up as outliers for many of the diagnostics below. The problem samples are clustered within a common batch, which may indicate a problem with sample processing.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percent missing genotypes by mouse. The nine mice with &#x2265;20% missing genotypes are labeled with their sample identifiers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percent missing genotypes by mouse. The nine mice with &#x2265;20% missing genotypes are labeled with their sample identifiers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F1.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1571/F1.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493136" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Percent missing genotypes by mouse. The nine mice with ≥20% missing genotypes are labeled with their sample identifiers.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h4>Verify sex:</h4><p id="p-18">To verify the sexes of the mice, we could look at the proportion of heterozygous genotype calls on the X chromosome. However, with array-based genotypes, it is most informative to look at the probe intensities for SNPs on the X and Y chromosomes.</p><p id="p-19">In <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, we display a scatterplot of the average intensity for the 30 Y chromosome markers <em>vs.</em> the average intensity for 2,058 X chromosome markers. Each point is a mouse, with the males in purple and the females in green. We omitted eighteen markers on the X chromosome that did not show a clear sex difference in allele intensities.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average SNP microarray intensity for markers on the Y chromosome vs. that for markers on the X chromosome, for each mouse. Mice that were nominally male are in purple, while females are in green. Samples with &#x2265; 20% missing genotypes are labeled in orange. Two other samples of interest are labeled in black: F386 which appears to be XO, and M377 which was nominally male but appears to be XX." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Average SNP microarray intensity for markers on the Y chromosome &lt;em&gt;vs.&lt;/em&gt; that for markers on the X chromosome, for each mouse. Mice that were nominally male are in purple, while females are in green. Samples with &#x2265; 20% missing genotypes are labeled in orange. Two other samples of interest are labeled in black: F386 which appears to be XO, and M377 which was nominally male but appears to be XX.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F2.medium.gif" width="440" height="267"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1571/F2.medium.gif" width="440" height="267"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493029" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Average SNP microarray intensity for markers on the Y chromosome <em>vs.</em> that for markers on the X chromosome, for each mouse. Mice that were nominally male are in purple, while females are in green. Samples with ≥ 20% missing genotypes are labeled in orange. Two other samples of interest are labeled in black: F386 which appears to be XO, and M377 which was nominally male but appears to be XX.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">There is a distinct cluster of male mice in the upper-left (low X chromosome intensity and high Y chromosome intensity) and a cluster of female mice in the lower-right (high X chromosome intensity and low Y chromosome intensity). However, M377 was labeled male but appears within the female cluster in the lower right, and so is likely female. Also, F386 is labeled female but appears in the lower-left, with reduced X chromosome intensity. This is likely an XO female. Other outliers in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> are mice with high rates of missing genotypes, including the nine mice with ≥20% missing genotypes, which are labeled in orange.</p><p id="p-22">For sex inference, the SNP probe intensities give better separation of the two sexes than heterozygosity on the X chromosome (Figure S1), and they enable us to distinguish between males and XO females. We could also, potentially, identify XXY males, who would have high average intensities on both sex chromosomes.</p></div><div id="sec-9" class="subsection"><h4>Sample duplicates:</h4><p id="p-23">We look for potential sample duplicates by calculating the proportion of matching SNP genotypes for each pair of samples. The nine samples with ≥ 20% missing genotypes look quite different from others, and also show some chance similarities (Supplementary Figure S2A). If we omit those samples (Supplementary Figure S2B), we find that the bulk of pairs share around 50% of genotypes (shifted slightly below 50%; the median is 46.6%). The probability of unrelated individuals sharing the same genotypes at a marker is determined by the SNP minor allele frequency. The average proportion of sharing is a property of both the population structure and the array probe selection strategy. A small proportion of pairs (179 pairs, or 0.4%) share a bit more, at around 67.1%. These likely represent siblings.</p><p id="p-24">Two pairs have almost perfectly matching SNP genotypes. Mice M283 and M292 have matching genotypes at all except one of the 69,025 markers at which they were both genotyped. Mice M377 and F409 have matching genotypes at all except 36 of the 68,291 markers at which they were both genotyped. Note that the second pair includes M377 which was seen in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> to have mislabeled sex.</p><p id="p-25">These two pairs are clear duplicates. We are looking for a separation between the normal sharing between mice and the duplicates, and (having excluded samples with ≥ 20% missing genotypes), these are the only pairs with &gt;76% matching genotypes. These unintended sample duplicates provide an estimate of the genotyping error rate, which looks to be well under 1/1000.</p><p id="p-26">We will omit one mouse from each pair, for the purpose of illustrating our genotype quality control analyses. For the M377/F409 pair, it seems clear that the sample corresponds to F409, since it is a female. But for M283/M292, we can not tell which is the correct label. For later QTL analyses we would likely wish to omit both samples, but for this illustration we will just omit the second one, M292.</p><p id="p-27">If the data included genome-scale phenotypes with strong genetic signals, such as gene expression data, we would at this point look further for possible sample mix-ups (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Westra <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2015</a>), but we will not do so here.</p></div><div id="sec-10" class="subsection"><h4>SNP probe intensities:</h4><p id="p-28">The distribution of probe intensities on the genotyping array can be a useful indicator of problem samples. In <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, we display density estimates of the array intensities for each of the 289 samples, after a <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1571/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1571/embed/mml-math-1.gif"/></noscript></span></span> transformation. We highlight the arrays corresponding to samples with appreciable missing genotypes. The nine samples with ≥ 20% missing data are highlighted in orange; their array intensities are shifted to the left and have a long right tail. There are three samples with ∼9% missing data (in pink); they show a much broader distribution of array intensities. There are twelve samples with 2–5% missing genotypes (in blue); many of these have a spike of intensities near 0.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of array intensities after a  transformation. A: Kernel density estimates of the array intensity distribution. Samples with &gt;20% missing genotype data are in orange; samples with 9&#x2013;20% missing genotype data are in pink; samples with 2&#x2013;5% missing genotype data are in blue; the remaining samples are in gray. B: Scatterplot of the 1st percentile vs. the 99th percentile." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Distribution of array intensities after a &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-2&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; transformation. A: Kernel density estimates of the array intensity distribution. Samples with &gt;20% missing genotype data are in orange; samples with 9&#x2013;20% missing genotype data are in pink; samples with 2&#x2013;5% missing genotype data are in blue; the remaining samples are in gray. B: Scatterplot of the 1&lt;sup&gt;st&lt;/sup&gt; percentile &lt;em&gt;vs.&lt;/em&gt; the 99&lt;sup&gt;th&lt;/sup&gt; percentile.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F3.medium.gif" width="440" height="179"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1571/F3.medium.gif" width="440" height="179"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493011" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Distribution of array intensities after a <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1571/F3/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1571/F3/embed/mml-math-2.gif"/></noscript></span></span> transformation. A: Kernel density estimates of the array intensity distribution. Samples with &gt;20% missing genotype data are in orange; samples with 9–20% missing genotype data are in pink; samples with 2–5% missing genotype data are in blue; the remaining samples are in gray. B: Scatterplot of the 1<sup>st</sup> percentile <em>vs.</em> the 99<sup>th</sup> percentile.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">In <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, we display a scatterplot of the 1<sup>st</sup> and 99<sup>th</sup> percentiles of the log array intensities, in an attempt to summarize the pattern seen in the densities in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>. Most of the samples with high rates of missing genotypes are outliers. We label one sample (F326) which has ≥20% missing genotypes but has an array intensity distribution that is not as extreme as the other samples with ≥20% missing genotypes.</p></div><div id="sec-11" class="subsection"><h4>SNP genotype frequencies:</h4><p id="p-31">The SNP genotype frequency distribution across all markers within an animal can be a useful diagnostic. For example, sample contamination can lead to excess heterozygotes. In DO mice, we split the SNP markers into four groups, based on the minor allele frequency (MAF) in the eight founder strains. To do so, we consider only the 68,357 with complete founder genotypes. (For 982 of the polymorphic markers, at least one of the founders has missing genotype.)</p><p id="p-32">To display the genotype frequencies, we use a ternary plot, which makes use of the fact that for any point within an equilateral triangle, the sum of the distances to the three sides is constant. And so the trinomial SNP genotype frequencies for an individual may be represented by a point in the triangle, with the distance to the lower edge being the frequency of heterozygotes, and the distances to the left and right edges being the frequencies of the two homozygotes.</p><p id="p-33">As seen in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, the DO mice form tight clusters with very similar SNP genotype frequencies, except for a few samples that show high heterozygosity, which are among the samples with high rates of missing genotype data. Another potential outlier is F313, which shows somewhat reduced heterozygosity in SNPs with MAF = 3/8 or 1/2.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP genotype frequencies by mouse, for SNPs split by their minor allele frequency (MAF) in the eight founder strains. Trinomial probabilities are represented by points in an equilateral triangle using the distances to the three sides. Pink points indicate the expected distributions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SNP genotype frequencies by mouse, for SNPs split by their minor allele frequency (MAF) in the eight founder strains. Trinomial probabilities are represented by points in an equilateral triangle using the distances to the three sides. Pink points indicate the expected distributions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F4.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1571/F4.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492931" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">SNP genotype frequencies by mouse, for SNPs split by their minor allele frequency (MAF) in the eight founder strains. Trinomial probabilities are represented by points in an equilateral triangle using the distances to the three sides. Pink points indicate the expected distributions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h4>Counts of crossovers:</h4><p id="p-35">Another important diagnostic is to estimate the number of crossovers, genome-wide, in each mouse. Problem samples may exhibit excessive crossovers. In some cases, a sample may exhibit fewer crossovers than expected.</p><p id="p-36">To estimate crossovers, we first reconstruct the diplotypes across the genome for each of the DO mice. We use a hidden Markov model (HMM) to calculate the probability of each of the 36 possible diplotypes (8 homozygotes plus 28 heterozygotes) at each genotyped SNP, given the multipoint SNP genotype data, with allowance for genotyping errors (Supplementary Figure S3). Figure S3A shows the inferred diplotypes across the genome for a single DO mouse, and Figure S3B shows the detailed diplotype probabilities for that mouse, along one chromosome.</p><p id="p-37">There are a number of different methods for estimating the number of crossovers in a DO genome. We are using the simplest: at each genotyped SNP, pick the most probable diplotype (provided that it has probability at least 50%) and then calculate the minimal number of crossovers that are consistent with that set of predicted diplotypes.</p><p id="p-38">The estimated numbers of crossovers for each DO mouse is shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, with points colored according to their generation. The mice in generation 8 have an average of 304 crossovers, while those in generation 11 have an average of 357 crossovers.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated number of crossovers in each mouse. Colors of the points indicate the two groups of DO mice (generations 8 and 11). Mice with &#x2265;20% missing genotypes are excluded." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated number of crossovers in each mouse. Colors of the points indicate the two groups of DO mice (generations 8 and 11). Mice with &#x2265;20% missing genotypes are excluded.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F5.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/5/1571/F5.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493173" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Estimated number of crossovers in each mouse. Colors of the points indicate the two groups of DO mice (generations 8 and 11). Mice with ≥20% missing genotypes are excluded.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">We have excluded the nine mice with ≥20% missing genotypes. They all show &gt;500 crossovers, and the mice with &gt;50% missing genotypes show &gt;2000 crossovers. This is further evidence that these samples should be omitted.</p><p id="p-41">There are no other apparent outliers, in the numbers of crossovers. But note the importance of taking account of generation number.</p></div><div id="sec-13" class="subsection"><h4>Genotyping error rates:</h4><p id="p-42">The diplotype reconstructions offer the opportunity to identify likely SNP genotyping errors. To identify likely errors, we calculate the genotyping error LOD scores described by <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Lincoln and Lander (1992)</a>. For each SNP marker in each DO mouse, we use the founder strains’ SNP genotypes to collapse our 36-state diplotype probabilities to three-state SNP genotype probabilities. We then compare the predicted SNP genotypes based on the haplotype reconstruction with the observed SNP genotypes, and we calculate a LOD score statistic that measures the evidence for an individual SNP genotype being in error.</p><p id="p-43">We generally focus on error LOD scores &gt;2, which is a reasonably conservative threshold on potential errors. The estimated error rate in each mouse, with this criterion, is shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The nine mice with ≥20% missing genotypes all have estimated genotyping error rates &gt;1%. Three mice have error rates in the range 0.5–1.0%, and these all showed ∼9% missing genotypes. The next-highest estimated error rate is 0.4% for mouse M398, which had shown about 5% missing genotypes. The error rates for the other mice are extremely small. The median rate is just 7.8 in 10,000.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated percent genotyping errors for each mouse. The rates are very small; the median is just 7.8 in 10,000." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated percent genotyping errors for each mouse. The rates are very small; the median is just 7.8 in 10,000.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F6.medium.gif" width="440" height="195"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/5/1571/F6.medium.gif" width="440" height="195"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493156" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Estimated percent genotyping errors for each mouse. The rates are very small; the median is just 7.8 in 10,000.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">In summary, the nine mice with ≥20% missing genotypes also showed excessive crossovers and high rates of apparent genotyping errors. We will omit these from further analyses. Three mice with 5–10% missing genotypes showed slightly elevated genotyping error rates but no excess of crossovers. We chose to not omit them.</p><p id="p-46">The factors that impact the decision to omit samples will vary from study to study and may also depend on the aims of the analysis. Samples that are clearly outliers will likely have a negative impact on mapping power and precision. It is not uncommon for some samples to fall into a gray area where the quality is less than ideal but, based on the number of predicted crossovers, the diplotype reconstruction appears to be reliable. The impact on power of omitting these samples may be greater if the sample size is already small.</p></div></div><div id="sec-14" class="subsection"><h3>Marker diagnostics</h3><p id="p-47">We now turn to the markers, to identify poorly behaved ones. As with the samples, we can look at the percent missing data, estimated rates of genotyping errors, and the genotype frequencies. Ultimately, we want to look at scatterplots of the allele-specific probe intensities for the SNPs, which are most informative of problems, but with 77,808 markers, we cannot inspect all of them, and so we use the other measures to help narrow our search. Throughout these analyses, we will focus on the 280 mice with &lt;20% missing genotypes.</p><div id="sec-15" class="subsection"><h4>Missing data and genotyping errors:</h4><p id="p-48">We start by studying the amount of missing data at each marker, as well as estimated genotyping error rates (based on genotyping error LOD scores &gt; 2). A scatterplot of estimated error rate <em>vs.</em> percent missing data is shown in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated percent genotyping errors vs. percent missing genotypes by marker. Errors defined by genotyping error LOD score &gt; 2. The vast majority of markers showed no apparent errors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Estimated percent genotyping errors &lt;em&gt;vs.&lt;/em&gt; percent missing genotypes by marker. Errors defined by genotyping error LOD score &gt; 2. The vast majority of markers showed no apparent errors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F7.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/5/1571/F7.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492935" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-49" class="first-child">Estimated percent genotyping errors <em>vs.</em> percent missing genotypes by marker. Errors defined by genotyping error LOD score &gt; 2. The vast majority of markers showed no apparent errors.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">The vast majority of markers have virtually no missing data: Of the 69,339 informative markers, 41,931 have no missing data, and 64,003 are missing &lt;2%. However, as seen in <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, there are a number of markers with appreciable missing data: 1,216 are missing &gt;10%.</p><p id="p-51">Similarly, the vast majority of markers have virtually no genotypes with error LOD score &gt; 2, including 66,623 markers with no apparent errors. But 624 markers have estimated error rates &gt;2%, including 190 markers with estimated error rates &gt;10%. The thirteen markers with error rates &gt;50% are highlighted in <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>.</p><p id="p-52">While it may not be apparent in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>, there is a reasonably strong relationship between missing data and error rate: Markers with larger amounts of missing data tend to have higher genotyping error rates. For example, the mean error rate for markers with &lt;2% missing data is 3 per 10,000, while the mean error rate for markers with ≥20% missing data is 5%, about 190 times higher.</p></div><div id="sec-16" class="subsection"><h4>SNP genotype frequencies:</h4><p id="p-53">The genotype frequencies at the markers are displayed in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>, with markers split according to their minor allele frequency (MAF) among the 8 founder strains. The majority of markers conform reasonably well to our expectation. The most striking departure is that there are 22 markers with MAF = 1/8 in the founders but where the MAF in the DO mice is &gt; 40%. The majority of these markers (19/22) are from a region on chromosome 2, 70 – 105 Mbp, where the WSB allele exhibits meiotic drive (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Didion <em>et al.</em> 2016</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP genotype frequencies by marker, with SNPs split by their minor allele frequency (MAF) in the eight founder strains. Trinomial probabilities are represented by points in an equilateral triangle using the distances to the three sides. Pink points indicate the expected distributions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SNP genotype frequencies by marker, with SNPs split by their minor allele frequency (MAF) in the eight founder strains. Trinomial probabilities are represented by points in an equilateral triangle using the distances to the three sides. Pink points indicate the expected distributions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F8.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/5/1571/F8.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492924" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-54" class="first-child">SNP genotype frequencies by marker, with SNPs split by their minor allele frequency (MAF) in the eight founder strains. Trinomial probabilities are represented by points in an equilateral triangle using the distances to the three sides. Pink points indicate the expected distributions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">There are also a number of SNPs with MAF = 3/8 or 4/8 in the founder strains that have reduced frequency of heterozygotes. For example, there are 46 SNPs with MAF = 4/8 in the founders but heterozygosity &lt;0.25. Most of these SNPs (37) have ≥10% missing data or ≥10% genotyping errors, or both. Of the remaining nine markers, all but one are on chromosome 2, in the region with high WSB allele frequency.</p></div><div id="sec-17" class="subsection"><h4>SNP allele intensities:</h4><p id="p-56">The most important diagnostic for SNP quality is a scatterplot of the allele-specific probe intensities. This is particularly informative when colored by both the observed genotype and by the predicted genotype giving the multipoint SNP information. The SNP allele intensities for a set of four SNPs are displayed in <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>. Each point is a single DO mouse; in the left panels, the points are colored by the observed genotype, with yellow and blue corresponding to the two homozygotes, green the heterozygote, and gray being missing. In the right panels, the points are colored by the predicted genotypes given the multipoint SNP information. <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figures 9A</a> and <a id="xref-fig-9-3" class="xref-fig" href="#F9">9B</a> correspond to a well-behaved SNP. The three genotype groups form tight, well-separated clusters, and the observed and predicted genotypes match. (Additional examples of well-behaved markers are shown in Supplementary Figure S4.)</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele intensity plots for four SNPs. In the left panels, points are colored according to the genotype calls, with yellow and blue being the two homozygotes and green being the heterozygote; gray points were not called. In the right panels, points are colored by the inferred SNP genotypes, given the multipoint marker data and the founders&#x2019; genotypes; gray points could not be inferred." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685160164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Allele intensity plots for four SNPs. In the left panels, points are colored according to the genotype calls, with yellow and blue being the two homozygotes and green being the heterozygote; gray points were not called. In the right panels, points are colored by the inferred SNP genotypes, given the multipoint marker data and the founders&#x2019; genotypes; gray points could not be inferred.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1571/F9.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/9/5/1571/F9.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1571/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493203" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-57" class="first-child">Allele intensity plots for four SNPs. In the left panels, points are colored according to the genotype calls, with yellow and blue being the two homozygotes and green being the heterozygote; gray points were not called. In the right panels, points are colored by the inferred SNP genotypes, given the multipoint marker data and the founders’ genotypes; gray points could not be inferred.</p><div class="sb-div caption-clear"></div></div></div><p id="p-58"><a id="xref-fig-9-4" class="xref-fig" href="#F9">Figures 9C</a> and <a id="xref-fig-9-5" class="xref-fig" href="#F9">9D</a> correspond to a SNP where there is an additional cluster of genotypes, and where the genotype calling algorithm assigned it to the wrong genotype. At this particular SNP, there is a cluster of genotypes that were called homozygotes for the major allele but that are really heterozygotes. This is an example of a variable intensity oligonucleotide (VINO; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Didion <em>et al.</em> 2012</a>), where there is an additional SNP within the probe that leads to reduced intensity of one allele. For this particular SNP, the founder strains 129Sv/ImJ and PWK/PhJ appear to have null alleles for the array probe (data not shown). Additional examples of this type of SNP are shown in Supplementary Figure S5.</p><p id="p-59"><a id="xref-fig-9-6" class="xref-fig" href="#F9">Figures 9E</a> and <a id="xref-fig-9-7" class="xref-fig" href="#F9">9F</a> correspond to a SNP where the genotype calling algorithm made a clear mistake. There are three well-defined clusters of genotypes, but one of the homozygotes got called as a heterozygote: the cluster is green in <a id="xref-fig-9-8" class="xref-fig" href="#F9">Figure 9E</a> but yellow (and gray, for missing) in <a id="xref-fig-9-9" class="xref-fig" href="#F9">Figure 9F</a>. <a id="xref-fig-9-10" class="xref-fig" href="#F9">Figures 9G</a> and <a id="xref-fig-9-11" class="xref-fig" href="#F9">9H</a> correspond to another example of a poorly called SNP, where two of the genotype clusters are not well separated, and the genotype calling algorithm made a mistake in identifying the clusters. Supplementary Figures S6 and S7 show additional examples of mistakes in the genotype calling, either because the genotype clusters are arranged horizontally or are not well separated. Figure S8 contains additional examples of particularly ugly SNPs.</p></div></div><div id="sec-18" class="subsection"><h3>Effects of data cleaning</h3><p id="p-60">High-density SNP data are sufficiently redundant that the presence of a small number of poorly behaved SNPs should have little influence on the results. The hidden Markov model used to reconstruct the DO genomes allows for the presence of genotyping errors and so should smooth over the problem markers. But it is worth checking: if we omit the most poorly behaved markers, how much will the diplotype probabilities change?</p><p id="p-61">Of the 69,339 informative markers, we omitted 325 markers with estimated genotyping error rate &gt;5%, using a threshold of 2 on the genotyping error LOD score to define a presumed error. We then re-calculated the 36-state diplotype probabilities for all mice.</p><p id="p-62">For each mouse at each marker, we take the sum of the absolute differences between the diplotype probabilities, before and after data cleaning, as a measure of change. This takes values between 0 and 2, with 2 indicating that a complete different set of diplotypes have positive probabilities, after the data cleaning.</p><p id="p-63">There are few changes in the diplotype probabilities. Of the 69,339 markers × 280 individuals, there are just 6,048 sites where the sum of the absolute differences was &gt;1, and just 1,798 where it was &gt;1.5. To illustrate the types of changes that are seen, Supplementary Figure S9 shows the diplotype probabilities, before and after omitting 325 bad markers, for three individuals on chromosome 9. In each case, an apparent recombinant segment gets removed.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-64">Our approach to cleaning genotype data is built around a series of diagnostic visualizations that can help us to identify problematic samples and markers. Identifying problem samples (mix-ups and low quality samples) is arguably the most important outcome of data cleaning. A small number of incorrect samples can impact the power of QTL mapping whereas poorly performing markers generally have little impact on diplotype reconstructions.</p><p id="p-65">The simplest diagnostic for sample quality, the amount of missing data by individual, is also highly effective. In addition to high rates of missing data, low quality samples will display higher rates of genotype calling errors, unexpectedly high numbers of predicted crossover events, and unusual allele frequencies. It is important to verify that samples are not duplicated and that the correct sex is assigned to individuals because these problem will not be captured in other quality control diagnostics.</p><p id="p-66">The proportion of missing data is also a good diagnostic for SNP marker quality. Additional diagnostics include genotyping frequencies, and the estimated proportion of genotyping errors.</p><p id="p-67">In the data we used as an illustration, with 291 DO mice from generations 8 and 11, there were nine samples with ≥20% missing genotypes. These also showed excessive crossovers and high genotyping error rates and should be omitted from any analyses. There were also two apparent sample duplicates (one being a male/female pair), and one apparent XO female. Finally, there were four samples with higher-than-normal amounts of missing data and genotyping errors, but these samples looked okay otherwise and probably do not need to be omitted. Decisions about which samples to include or omit should be based on the likely impact on mapping analysis and may depend on factors such as the sample size and the extent and quality of phenotyping data.</p><p id="p-68">Verifying sex and identifying sample duplicates are two steps toward identifying sample mix-ups. If there are phenotypes with strong genetic effects, such as coat color, they may be useful to identify further mix-ups. Particularly useful in this regard are genome-scale phenotype data such as gene expression data, whether by microarrays or RNA-seq, which can perfectly identify individuals (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Westra <em>et al.</em> 2011</a>). With gene expression data on multiple tissues, identified mix-ups can potentially be corrected (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2015</a>).</p><p id="p-69">The vast majority of markers appeared well behaved, but we also found a number of markers with a high proportion of apparent genotyping errors. Omitting these markers had small and relatively isolated effects on the diplotype probabilities. Our approach for identifying problem markers relied on our diplotype reconstructions, and so we may be missing some badly behaved SNPs, and the SNPs we miss may be the ones with the greatest influence on results. But if the overall SNP density is high, and the proportion of badly behaved SNPs is low, our approach should provide reasonable results. While poor quality markers may have little impact on any given study, identifying and annotating these markers may impact other studies that use the same array platform.</p><p id="p-70">We have not discussed the problem of cleaning phenotype data, but this is also important. We would focus on data visualizations, including histograms, plots of traits by time of measurement and/or by mouse identifiers, and scatterplots of traits against one another. These plots may reveal typographical errors in the data or inconsistencies in measurement units. They may also indicate the need for phenotype transformations (such as logarithm or square-root), or they may reveal important batch effects or other covariates that should be taken into account in analyses.</p><p id="p-71">We used R (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">R Core Team 2018</a>) and R/qtl2 (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2019</a>) throughout this work. Another important R package for genotype diagnostics is argyle (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Morgan 2016</a>), which provides a variety of diagnostics for SNP genotyping arrays.</p><p id="p-72">We focused on data visualizations to diagnose potential problems, and that is the central tool for data cleaning. Make lots of graphs, focusing on graphs that will reveal anticipated problems but also following up on anything unexpected: Is it a problem with the data, a problem with the sample, or a quirk of biology? Is it ignorable or fixable? What effect might it have on later results? We find interactive data visualizations, such as with R/qtlcharts (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broman 2015</a>), useful in these efforts, particularly for identifying outlier samples in scatterplots.</p><p id="p-73">While we have focused on DO mice, our approach could be applied more generally, to other multiparent populations. The key summary statistics are the proportion of missing genotypes, the average probe intensities for SNPs on the X and Y chromosomes, the proportion of heterozygous SNPs, the estimated number of crossovers, and the estimated genotyping error rate.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-74">This work was supported in part by National Institutes of Health grants R01GM070683 (to K.W.B. and G.A.C.) and R01GM123489 (to Ś.S.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-75">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7848395">https://doi.org/10.25387/g3.7848395</a>.</p></li><li class="fn" id="fn-2"><p id="p-76"><em>Communicating editor: A. Dudley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-11"><span class="received-label">Received </span>January 11, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-14"><span class="accepted-label">Accepted </span>March 14, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Broman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.1" data-doi="10.1534/genetics.111.132647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genotype probabilities at intermediate generations in the construction of recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132647</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132647%26rft_id%253Dinfo%253Apmid%252F22345609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.2" data-doi="10.1534/g3.111.001818"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Haplotype probabilities in advanced intercross populations.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.001818</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D199%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.001818&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.3" data-doi="10.1534/genetics.114.172742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">R/qtlcharts: interactive graphics for quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">361</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.172742</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.172742%26rft_id%253Dinfo%253Apmid%252F25527287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzIvMzU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.4" data-doi="10.1534/genetics.118.301595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furlotte</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">R/qtl2: Software for mapping quantitative trait loci with high-dimensional data and multi-parent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301595%26rft_id%253Dinfo%253Apmid%252F30591514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjExLzIvNDk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.5" data-doi="10.1534/g3.115.019778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kendziorski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification and correction of sample mix-ups in expression genetic data: A case study.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2177</span>–<span class="cit-lpage">2186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.019778</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019778%26rft_id%253Dinfo%253Apmid%252F26290572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMjE3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1571.6" data-doi="10.1007/978-0-387-92125-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">A guide to QTL mapping with R/qtl</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-92125-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2Bguide%2Bto%2BQTL%2Bmapping%2Bwith%2BR%252Fqtl%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-92125-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-92125-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.7" data-doi="10.1007/BF02996226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">Stocks for detecting linkage in the mouse and the theory of their design.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02996226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D50%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02996226%26rft_id%253Dinfo%253Apmid%252F24539711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02996226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24539711&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1951XX17400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.8" data-doi="10.1016/j.pbi.2008.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2008.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBeyer%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D215%26rft.epage%253D221%26rft.atitle%253DFrom%2Bmutations%2Bto%2BMAGIC%253A%2Bresources%2Bfor%2Bgene%2Bdiscovery%252C%2Bvalidation%2Band%2Bdelivery%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2008.01.002%26rft_id%253Dinfo%253Apmid%252F18295532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2008.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18295532&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255623300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.9" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Collaborative Cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1104-1133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.10" data-doi="10.1007/s00335-012-9414-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Diversity Outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-012-9414-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D713%26rft.epage%253D718%26rft.atitle%253DThe%2Bdiversity%2Boutbred%2Bmouse%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9414-2%26rft_id%253Dinfo%253Apmid%252F22892839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9414-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892839&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.11" data-doi="10.1534/genetics.109.105486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackert-Bicknell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A new standard genetic map for the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1335</span>–<span class="cit-lpage">1344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.105486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105486%26rft_id%253Dinfo%253Apmid%252F19535546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEzMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.12" data-doi="10.1093/molbev/msw036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadgary</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>R2d2</em> drives selfish sweeps in the house mouse.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1381</span>–<span class="cit-lpage">1395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw036%26rft_id%253Dinfo%253Apmid%252F26882987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26882987&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.13" data-doi="10.1186/1471-2164-13-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheppard</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-34</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDidion%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D13%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DDiscovery%2Bof%2Bnovel%2Bvariants%2Bin%2Bgenotyping%2Barrays%2Bimproves%2Bgenotype%2Bretention%2Band%2Breduces%2Bascertainment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-34%26rft_id%253Dinfo%253Apmid%252F22260749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22260749&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.14" data-doi="10.1101/098657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somes</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffrey</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The effects of sex and diet on physiology and liver gene expression in Diversity Outbred mice.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/098657</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F098657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwOTg2NTd2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.15" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in <em>arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000551</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.16" data-doi="10.1016/S0888-7543(05)80158-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lincoln</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Systematic detection of errors in genetic linkage data.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">604</span>–<span class="cit-lpage">610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0888-7543(05)80158-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLincoln%26rft.auinit1%253DS.%2BE.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D604%26rft.epage%253D610%26rft.atitle%253DSystematic%2Bdetection%2Bof%2Berrors%2Bin%2Bgenetic%2Blinkage%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0888-7543%252805%252980158-2%26rft_id%253Dinfo%253Apmid%252F1427888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0888-7543(05)80158-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1427888&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JW35200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.17" data-doi="10.1534/g3.115.023739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> argyle: an R package for analysis of Illumina genotyping arrays. G3 (Bethesda) 6: 281–286.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.023739</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.023739%26rft_id%253Dinfo%253Apmid%252F26684930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMi8yODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.18" data-doi="10.1534/g3.115.022087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> The Mouse Universal Genotyping Array: from substrains to subspecies. G3 (Bethesda) 6: 263–279.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.022087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.022087%26rft_id%253Dinfo%253Apmid%252F26684931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMi8yNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Simultaneous detection and fine mapping of quantitative trait loci in mice using heterogeneous stocks.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMott%26rft.auinit1%253DR.%26rft.volume%253D160%26rft.issue%253D4%26rft.spage%253D1609%26rft.epage%253D1618%26rft.atitle%253DSimultaneous%2BDetection%2Band%2BFine%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2Bin%2BMice%2BUsing%2BHeterogeneous%2BStocks%26rft_id%253Dinfo%253Apmid%252F11973314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC80LzE2MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.20" data-doi="10.1073/pnas.230304397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turri</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A method for fine mapping quantitative trait loci in outbred animal stocks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12649</span>–<span class="cit-lpage">12654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.230304397</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.230304397%26rft_id%253Dinfo%253Apmid%252F11050180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjMvMTI2NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1571.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R: A language and environment for statistical computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.22" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the mouse Diversity Outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132597</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1571.23" data-doi="10.1093/bioinformatics/btr323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westra</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fehrmann</span>  <span class="cit-name-given-names">R. S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">te Meerman</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Heel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MixupMapper: correcting sample mix-ups in genome-wide datasets increases power to detect small genetic effects.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2104</span>–<span class="cit-lpage">2111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr323</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr323%26rft_id%253Dinfo%253Apmid%252F21653519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653519&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1571.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1557" title="Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1581" title="Genetic and Molecular Analysis of Essential Genes in Centromeric Heterochromatin of the Left Arm of Chromosome 3 in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1571" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lh8EDAuftn8GFa7LvqgY3gDCYM-UdurZAq8vQ9DIozM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1571" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1571" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1571" class="active" data-icon-position="" data-hide-link-title="0">Cleaning Genotype Data from Diversity Outbred Mice</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492866" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form--yugovneIcOLN-EqVSzn8gRFXhhBw-ZRVdLZnodLYsc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1571.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1571" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-bJL3hG9OnhQY0HUohn52EdcsumR21So64j4HWmqsfdQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1571" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1571.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492866" id="node492866--2" data-pisa="ggg;9/5/1571" data-pisa-master="ggg;g3.119.400165" data-apath="/ggg/9/5/1571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Cleaning Genotype Data from Diversity Outbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0667-9926" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Gatti</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7928-1911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karen L. Svenson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Śaunak Sen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-9190-9284" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1571-1579;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400165 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492866--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin–Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:broman@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">broman@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0667-9926" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Gatti</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen L. Svenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BL.%2BSvenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Svenson%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaren%2BL.%2BSvenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7928-1911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karen L. Svenson</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Śaunak Sen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Preventive Medicine, University of Tennessee Health Sciences Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C5%9Aaunak%2BSen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sen%20%C5%9A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C5%259Aaunak%2BSen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Śaunak Sen</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9190-9284" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gary A. Churchill</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1571" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1571&amp;title=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1571&amp;title=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1571&amp;title=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1571&amp;text=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1571&amp;title=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1571&amp;t=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1571&amp;title=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1571&amp;title=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492866/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492866" id="node492866--3" data-pisa="ggg;9/5/1571" data-pisa-master="ggg;g3.119.400165" data-apath="/ggg/9/5/1571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Cleaning Genotype Data from Diversity Outbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0667-9926" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Gatti</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7928-1911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karen L. Svenson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Śaunak Sen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-9190-9284" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1571-1579;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400165 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492866--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin–Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:broman@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">broman@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0667-9926" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Gatti</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen L. Svenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BL.%2BSvenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Svenson%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaren%2BL.%2BSvenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7928-1911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karen L. Svenson</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Śaunak Sen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Preventive Medicine, University of Tennessee Health Sciences Center, Memphis, Tennessee 38163</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C5%9Aaunak%2BSen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sen%20%C5%9A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C5%259Aaunak%2BSen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Śaunak Sen</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9190-9284" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gary A. Churchill</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1571&amp;rft.epage=1579&amp;rft.atitle=Cleaning%20Genotype%20Data%20from%20Diversity%20Outbred%20Mice&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Broman%2C+Karl+W.&amp;rft.au=Gatti%2C+Daniel+M.&amp;rft.au=Svenson%2C+Karen+L.&amp;rft.au=Sen%2C+%C5%9Aaunak&amp;rft.au=Churchill%2C+Gary+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492866/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492866/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492866/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492866/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492866/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492866/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492866/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492866/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492866/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492866/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492866/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492866/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/MULTIPARENTAL%20POPULATIONS" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1571" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1571.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1571.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1571.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
