<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1557" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1557.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1557.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1557.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1557/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1557.atom" />
<meta name="HW.pisa" content="ggg;9/5/1557" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance" />
<meta name="DC.Identifier" content="10.1534/g3.118.200994" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Technow. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Developing training sets for genomic prediction in hybrid crops requires producing hybrid seed for a large number of entries. In autogamous crop species ( e.g. , wheat, rice, rapeseed, cotton) this requires elaborate hybridization systems to prevent self-pollination and presents a significant impediment to the implementation of hybrid breeding in general and genomic selection in particular. An alternative to F1 hybrids are bulks of F2 seed from selfed F1 plants (F1:2). Seed production for F1:2 bulks requires no hybridization system because the number of F1 plants needed for producing enough F1:2 seed for multi-environment testing can be generated by hand-pollination. This study evaluated the suitability of F1:2 bulks for use in training sets for genomic prediction of F1 level general combining ability and hybrid performance, under different degrees of divergence between heterotic groups and modes of gene action, using quantitative genetic theory and simulation of a genomic prediction experiment. The simulation, backed by theory, showed that F1:2 training sets are expected to have a lower prediction accuracy relative to F1 training sets, particularly when heterotic groups have strongly diverged. The accuracy penalty, however, was only modest and mostly because of a lower heritability, rather than because of a difference in F1 and F1:2 genetic values. It is concluded that resorting to F1:2 bulks is, in theory at least, a promising approach to remove the significant complication of a hybridization system from the breeding process." />
<meta name="DC.Contributor" content="Frank Technow" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Developing training sets for genomic prediction in hybrid crops requires producing hybrid seed for a large number of entries. In autogamous crop species (&lt;i&gt;e.g.&lt;/i&gt;, wheat, rice, rapeseed, cotton) this requires elaborate hybridization systems to prevent self-pollination and presents a significant impediment to the implementation of hybrid breeding in general and genomic selection in particular. An alternative to F1 hybrids are bulks of F2 seed from selfed F1 plants (F1:2). Seed production for F1:2 bulks requires no hybridization system because the number of F1 plants needed for producing enough F1:2 seed for multi-environment testing can be generated by hand-pollination. This study evaluated the suitability of F1:2 bulks for use in training sets for genomic prediction of F1 level general combining ability and hybrid performance, under different degrees of divergence between heterotic groups and modes of gene action, using quantitative genetic theory and simulation of a genomic prediction experiment. The simulation, backed by theory, showed that F1:2 training sets are expected to have a lower prediction accuracy relative to F1 training sets, particularly when heterotic groups have strongly diverged. The accuracy penalty, however, was only modest and mostly because of a lower heritability, rather than because of a difference in F1 and F1:2 genetic values. It is concluded that resorting to F1:2 bulks is, in theory at least, a promising approach to remove the significant complication of a hybridization system from the breeding process.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1557" />
<meta name="citation_id" content="9/5/1557" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1557" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1557.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1557.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1557.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200994" />
<meta name="citation_pmid" content="30862623" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="1557" />
<meta name="citation_lastpage" content="1569" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Frank Technow" />
<meta name="citation_author_institution" content="Maize Product Development/Systems and Innovation for Breeding and Seed Products, DuPont Pioneer, Tavistock/Ontario, Canada" />
<meta name="citation_author_email" content="frank.technow@pioneer.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2497-3136" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Adugna;citation_author=G. Nanda;citation_author=K. Singh;citation_author=N. Bains;citation_title=A comparison of cytoplasmic and chemically-induced male sterility systems for hybrid seed production in wheat (Triticum aestivum L.).;citation_pages=297-304;citation_volume=135;citation_year=2004;citation_doi=10.1023/B:EUPH.0000013320.28114.c6" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Marker-assisted backcrossing to develop an elite cytoplasmic male sterility line in rice.;citation_author=A. Ahmadikhah;citation_author=M. Mirarab;citation_author=M. H. Pahlevani;citation_author=L. Nayyeripasand;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H.-J. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize.;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. L. Araus;citation_author=J. E. Cairns;citation_title=Field high-throughput phenotyping: the new crop breeding frontier.;citation_pages=52-61;citation_volume=19;citation_year=2014;citation_pmid=24139902;citation_doi=10.1016/j.tplants.2013.09.008" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=J. Bennewitz;citation_author=T. H. E. Meuwissen;citation_title=The distribution of QTL additive and dominance effects in porcine F2 crosses.;citation_pages=171-179;citation_volume=127;citation_year=2010;citation_issue=3;citation_pmid=20536634;citation_doi=10.1111/j.1439-0388.2009.00847.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Best Linear Unbiased Prediction of the Performance of Crosses between Untested Maize Inbreds;citation_pages=872-876;citation_volume=36;citation_year=1996;citation_issue=4;citation_doi=10.2135/cropsci1996.0011183X003600040009x" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=U. Beukert;citation_author=Z. Li;citation_author=G. Liu;citation_author=Y. Zhao;citation_author=N. Ramachandra;citation_title=Genome-based identification of heterotic patterns in rice.;citation_pages=22;citation_volume=10;citation_year=2017;citation_doi=10.1186/s12284-017-0163-4" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Rep.;citation_author=A. Bohra;citation_author=U. C. Jha;citation_author=P. Adhimoolam;citation_author=D. Bisht;citation_author=N. P. Singh;citation_title=Cytoplasmic male sterility (CMS) in hybrid breeding in field crops.;citation_pages=967-993;citation_volume=35;citation_year=2016;citation_doi=10.1007/s00299-016-1949-3" />
<meta name="citation_reference" content="citation_journal_title=HortScience;citation_journal_abbrev=hortsci;citation_author=P. W. Bosland;citation_title=Second Generation (F2) Hybrid Cultivars for Jalapeno Production;citation_pages=1679-1681;citation_volume=40;citation_year=2005;citation_issue=6;citation_doi=10.21273/HORTSCI.40.6.1679" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=R. E. Comstock;citation_author=H. F. Robinson;citation_title=The components of genetic variance in populations of biparental progenies and their use in estimating the average degree of dominance.;citation_pages=254-266;citation_volume=4;citation_year=1948;citation_issue=4;citation_pmid=18108899;citation_doi=10.2307/3001412" />
<meta name="citation_reference" content="citation_title=Heterosis;citation_title=Estimation of average dominance of genes.;citation_author=R. E. Comstock;citation_author=H. F. Robinson;citation_pages=494-516;citation_year=1952" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=M. Cooper;citation_author=C. D. Messina;citation_author=D. Podlich;citation_author=L. R. Totir;citation_author=A. Baumgarten;citation_title=Predicting the future of plant breeding: complementing empirical evaluation with genetic prediction.;citation_pages=311;citation_volume=65;citation_year=2014;citation_doi=10.1071/CP14007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_title=The genetics and exploitation of heterosis in crops;citation_title=Heterosis: feeding people and protecting natural resources;citation_author=D. Duvick;citation_pages=19-29;citation_year=1999" />
<meta name="citation_reference" content="citation_title=Plant Breeding Reviews;citation_title=Long-term selection in a commercial hybrid maize breeding program;citation_author=D. Duvick;citation_author=J. Smith;citation_author=M. Cooper;citation_pages=109-152;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Biotechnol. Adv.;citation_author=S. L. Dwivedi;citation_author=A. B. Britt;citation_author=L. Tripathi;citation_author=S. Sharma;citation_author=H. D. Upadhyaya;citation_title=Haploids: constraints and opportunities in plant breeding.;citation_pages=812-829;citation_volume=33;citation_year=2015;citation_doi=10.1016/j.biotechadv.2015.07.001" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. B. Endelman;citation_author=G. N. Atlin;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=X. Zhang;citation_title=Optimal design of preliminary yield trials with genome-wide markers.;citation_pages=48-59;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.03.0154" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_title=Shrinkage estimation of the realized relationship matrix.;citation_pages=1405-1413;citation_volume=2;citation_year=2012;citation_pmid=23173092;citation_doi=10.1534/g3.112.004259" />
<meta name="citation_reference" content="citation_title=Introduction to quantitative genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. Fischer;citation_author=J. Möhring;citation_author=H. P. Maurer;citation_author=H.-P. Piepho;citation_author=E.-M. Thiemt;citation_title=Impact of Genetic Divergence on the Ratio of Variance Due to Specific vs. General Combining Ability in Winter Triticale;citation_pages=2119-2122;citation_volume=49;citation_year=2009;citation_issue=6;citation_doi=10.2135/cropsci2008.12.0689" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Fritsche-Neto;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_title=Accuracy of genomic selection to predict maize single-crosses obtained through different mating designs.;citation_pages=1153-1162;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-018-3068-8" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Fritsche-Neto;citation_author=D. Akdemir;citation_author=J. L. Jannink;citation_title=Correction to: accuracy of genomic selection to predict maize single-crosses obtained through different mating designs.;citation_pages=1603;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-018-3118-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Fu;citation_author=T.-J. Wen;citation_author=Y. I. Ronin;citation_author=H. D. Chen;citation_author=L. Guo;citation_title=Genetic dissection of intermated recombinant inbred lines using a new genetic map of maize.;citation_volume=174;citation_year=2006;citation_pmid=16951074;citation_doi=10.1534/genetics.106.060376" />
<meta name="citation_reference" content="citation_title=Statistical Genetics and Plant Breeding;citation_author=C. Gardner;citation_title=Estimates of genetic parameters in cross-fertilizing plants and their implications in plant breeding;citation_pages=225-251;citation_volume=Vol. 982;citation_year=1963" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. C. Gaynor;citation_author=G. Gorjanc;citation_author=A. R. Bentley;citation_author=E. S. Ober;citation_author=P. Howell;citation_title=A two-part strategy for using genomic selection to develop inbred lines.;citation_pages=2372-2386;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2016.09.0742" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Gerke;citation_author=J. W. Edwards;citation_author=K. E. Guill;citation_author=J. Ross-Ibarra;citation_author=M. D. McMullen;citation_title=The genomic impacts of drift and selection for hybrid performance in maize.;citation_volume=201;citation_year=2015;citation_pmid=26385980;citation_doi=10.1534/genetics.115.182410" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Giraud;citation_author=C. Bauland;citation_author=M. Falque;citation_author=D. Madur;citation_author=V. Combes;citation_title=Reciprocal genetics: identifying QTLs for general and specific combining abilities in hybrids between multiparental populations from two maize (Zea mays L.) heterotic groups.;citation_volume=207;citation_year=2017;citation_pmid=28971957;citation_doi=10.1534/genetics.117.300305" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=G. Gorjanc;citation_author=M. Battagin;citation_author=J.-F. Dumasy;citation_author=R. Antolin;citation_author=R. C. Gaynor;citation_title=Prospects for cost-effective genomic selection via accurate within-family imputation.;citation_pages=216-228;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2016.06.0526" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. R. Hallauer;citation_author=S. A. Eberhart;citation_title=Reciprocal Full-Sib Selection;citation_pages=315-316;citation_volume=10;citation_year=1970;citation_issue=3;citation_doi=10.2135/cropsci1970.0011183X001000030033x" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=P. Hedden;citation_title=The genes of the Green Revolution.;citation_pages=5-9;citation_volume=19;citation_year=2003;citation_issue=1;citation_pmid=12493241;citation_doi=10.1016/S0168-9525(02)00009-4" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost.;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. R. Henderson;citation_title=SIRE EVALUATION AND GENETIC TRENDS;citation_pages=10-41;citation_volume=1973;citation_year=1973;citation_issue=Symposium;citation_doi=10.1093/ansci/1973.Symposium.10" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=S. Dreisigacker;citation_author=J. Crossa;citation_author=S. Hearne;citation_author=R. Babu;citation_title=Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.;citation_pages=1476-1488;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.03.0195" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=G. Gorjanc;citation_author=R. K. Varshney;citation_author=C. Nettelblad;citation_title=Imputation of single nucleotide polymorphism genotypes in biparental, backcross, and topcross populations with a hidden Markov model.;citation_pages=1934-1946;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.09.0648" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_title=Plant Breeding Reviews;citation_title=Epistasis and plant breeding;citation_author=J. B. Holland;citation_pages=27-92;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=X. Huang;citation_author=S. Yang;citation_author=J. Gong;citation_author=Y. Zhao;citation_author=Q. Feng;citation_title=Genomic analysis of hybrid rice varieties reveals numerous superior alleles that contribute to heterosis.;citation_pages=6258;citation_volume=6;citation_year=2015;citation_pmid=25651972;citation_doi=10.1038/ncomms7258" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. U. Jan;citation_author=A. Abbadi;citation_author=S. Lücke;citation_author=R. A. Nichols;citation_author=R. J. Snowdon;citation_title=Genomic prediction of testcross performance in canola (Brassica napus).;citation_pages=e0147769;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0147769" />
<meta name="citation_reference" content="citation_journal_title=CSSA Special Publication;citation_author=J. Janick;citation_title=Hybrids in horticultural crops.;citation_pages=45-56;citation_volume=25;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics Proteomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=M. T. Jenkins;citation_author=A. M. Brunson;citation_title=Methods of Testing Inbred Lines of Maize in Crossbred Combinations;citation_pages=523-530;citation_volume=24;citation_year=1932;citation_issue=7;citation_doi=10.2134/agronj1932.00021962002400070004x" />
<meta name="citation_reference" content="citation_author=D. C. Kadam;citation_author=S. M. Potts;citation_author=M. O. Bohn;citation_author=A. E. Lipka;citation_author=A. J. Lorenz;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=K. Kempe;citation_author=M. Gils;citation_title=Pollination control technologies for hybrid breeding.;citation_pages=417-437;citation_volume=27;citation_year=2011;citation_doi=10.1007/s11032-011-9555-0" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Y.-J. Kim;citation_author=D. Zhang;citation_title=Molecular control of male fertility for crop hybrid breeding.;citation_pages=53-65;citation_volume=23;citation_year=2018;citation_doi=10.1016/j.tplants.2017.10.001" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Labate;citation_author=K. R. Lamkey;citation_author=M. Lee;citation_author=W. L. Woodman;citation_title=Temporal changes in allele frequencies in two reciprocally selected maize populations.;citation_pages=1166-1178;citation_volume=99;citation_year=1999;citation_doi=10.1007/s001220051321" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Larièpe;citation_author=B. Mangin;citation_author=S. Jasson;citation_author=V. Combes;citation_author=F. Dumas;citation_title=The genetic basis of heterosis: multiparental quantitative trait loci mapping reveals contrasted levels of apparent overdominance among traits of agronomical interest in maize (Zea mays L.).;citation_volume=190;citation_year=2012;citation_pmid=22135356;citation_doi=10.1534/genetics.111.133447" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Larièpe;citation_author=L. Moreau;citation_author=J. Laborde;citation_author=C. Bauland;citation_author=S. Mezmouk;citation_title=General and specific combining abilities in a maize (Zea mays L.) test-cross hybrid panel: relative importance of population structure and genetic divergence between parents.;citation_pages=403-417;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2822-z" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. F. H. Longin;citation_author=X. Mi;citation_author=T. Würschum;citation_title=Genomic selection in wheat: optimum allocation of test resources and comparison of breeding strategies for line and hybrid breeding.;citation_pages=1297-1306;citation_volume=128;citation_year=2015;citation_pmid=25877519;citation_doi=10.1007/s00122-015-2505-1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. F. H. Longin;citation_author=J. Mühleisen;citation_author=H. P. Maurer;citation_author=H. Zhang;citation_author=M. Gowda;citation_title=Hybrid breeding in autogamous cereals.;citation_pages=1087-1096;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22918662;citation_doi=10.1007/s00122-012-1967-7" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. J. Lorenz;citation_title=Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment.;citation_pages=481-491;citation_volume=3;citation_year=2013;citation_doi=10.1534/g3.112.004911" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. M. Massman;citation_author=A. Gordillo;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Genomewide predictions from maize single-cross data.;citation_pages=13-22;citation_volume=126;citation_year=2013;citation_pmid=22886355;citation_doi=10.1007/s00122-012-1955-y" />
<meta name="citation_reference" content="citation_title=The genetics and exploitation of heterosis in crops;citation_title=Genetic diversity and heterosis;citation_author=A. E. Melchinger;citation_pages=99-118;citation_year=1999" />
<meta name="citation_reference" content="citation_title=Concepts and Breeding of Heterosis in Crop Plant;citation_title=Overview of heterosis and heterotic groups in agronomic crops;citation_author=A. E. Melchinger;citation_author=R. K. Gumber;citation_pages=29-44;citation_year=1998" />
<meta name="citation_reference" content="Melchinger, A. E., C. F. H. Longin, H. F. Utz, and J. C. Reif, 2005 Hybrid maize breeding with doubled haploid lines: quantitative genetic and selection theory for optimum allocation of resources. In Proceedings of the forty first annual Illinois corn breeders’ School 2005, pp. 8–21, Urbana-Champaign, Illinois, USA." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. A. Mikel;citation_author=J. W. Dudley;citation_title=Evolution of North American Dent Corn from Public to Proprietary Germplasm;citation_pages=1193-1205;citation_volume=46;citation_year=2006;citation_issue=3;citation_doi=10.2135/cropsci2005.10-0371" />
<meta name="citation_reference" content="citation_title=Introduction to the theory of statistics;citation_author=A. M. Mood;citation_author=F. A. Graybill;citation_author=C. B. Duane;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Müller;citation_author=F. Technow;citation_author=A. E. Melchinger;citation_title=Shrinkage estimation of the genomic relationship matrix can improve genomic estimated breeding values in the training set.;citation_pages=693-703;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-015-2464-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. Campos;citation_title=Genome-wide regression &amp; prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_title=R: a language and environment for statistical computing;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Radoev;citation_author=H. C. Becker;citation_author=W. Ecke;citation_title=Genetic analysis of heterosis for yield and yield components in rapeseed (Brassica napus L.) by quantitative trait locus mapping.;citation_volume=179;citation_year=2008;citation_pmid=18562665;citation_doi=10.1534/genetics.108.089680" />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=G. J. Rebetzke;citation_author=R. T. A. Fischer;citation_author=A. F. Herwaarden;citation_author=D. G. Bonnett;citation_author=K. Chenu;citation_title=Plot size matters: interference from intergenotypic competition in plant phenotyping studies.;citation_pages=107-118;citation_volume=41;citation_year=2014;citation_doi=10.1071/FP13177" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. C. Reif;citation_author=F.-M. Gumpert;citation_author=S. Fischer;citation_author=A. E. Melchinger;citation_title=Impact of interpopulation divergence on additive and dominance variance in hybrid populations.;citation_volume=176;citation_year=2007;citation_pmid=17507673;citation_doi=10.1534/genetics.107.074146" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Riedelsheimer;citation_author=J. B. Endelman;citation_author=M. Stange;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic predictability of interconnected biparental maize populations.;citation_volume=194;citation_year=2013;citation_pmid=23535384;citation_doi=10.1534/genetics.113.150227" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloe;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Biometrische Zeitung;citation_author=F. Schnell;citation_title=Die Covarianz zwischen Verwandten in einer gen-orthogonalen Population.;citation_pages=1-49;citation_volume=7;citation_year=1965;citation_doi=10.1002/bimj.19650070102" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. C. Schön;citation_author=B. S. Dhillon;citation_author=H. F. Utz;citation_author=A. E. Melchinger;citation_title=High congruency of QTL positions for heterosis of grain yield in three crosses of maize.;citation_pages=321-332;citation_volume=120;citation_year=2010;citation_issue=2;citation_pmid=19911156;citation_doi=10.1007/s00122-009-1209-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Schopp;citation_author=D. Müller;citation_author=F. Technow;citation_author=A. E. Melchinger;citation_title=Accuracy of genomic prediction in synthetic populations depending on the number of parents, relatedness, and ancestral linkage disequilibrium.;citation_volume=205;citation_year=2017;citation_pmid=28049710;citation_doi=10.1534/genetics.116.193243" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Schopp;citation_author=C. Riedelsheimer;citation_author=H. F. Utz;citation_author=C.-C. Schön;citation_author=A. E. Melchinger;citation_title=Forecasting the accuracy of genomic prediction with different selection targets in the training and prediction set as well as truncation selection.;citation_pages=2189-2201;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-015-2577-y" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Semel;citation_author=J. Nissenbaum;citation_author=N. Menda;citation_author=M. Zinder;citation_author=U. Krieger;citation_title=Overdominant quantitative trait loci for yield and fitness in tomato.;citation_volume=103;citation_year=2006;citation_pmid=16938842;citation_doi=10.1073/pnas.0604635103" />
<meta name="citation_reference" content="Shull, G. H., 1908 The composition of a field of maize. J Hered os-4: 296–301." />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. C. da Silva Dias;citation_title=Impact of improved vegetable cultivars in overcoming food insecurity.;citation_pages=125-136;citation_volume=176;citation_year=2010;citation_doi=10.1007/s10681-010-0237-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=O. S. Smith;citation_title=Covariance between Line per se and Testcross Performance;citation_pages=540-543;citation_volume=26;citation_year=1986;citation_issue=3;citation_doi=10.2135/cropsci1986.0011183X002600030023x" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=G. F. Sprague;citation_author=L. A. Tatum;citation_title=General vs. Specific Combining Ability in Single Crosses of Corn;citation_pages=923-932;citation_volume=34;citation_year=1942;citation_issue=10;citation_doi=10.2134/agronj1942.00021962003400100008x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. W. Stuber;citation_title=Estimation of Genetic Variances Using Inbred Relatives;citation_pages=129-135;citation_volume=10;citation_year=1970;citation_issue=2;citation_doi=10.2135/cropsci1970.0011183X001000020002x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. W. Stuber;citation_author=C. C. Cockerham;citation_title=GENE EFFECTS AND VARIANCES IN HYBRID POPULATIONS;citation_pages=1279-1286;citation_volume=54;citation_year=1966;citation_issue=6;citation_pmid=17248353;citation_doi=10.3168/jds.2014-8210" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=F. Tardieu;citation_author=L. Cabrera-Bosquet;citation_author=T. Pridmore;citation_author=M. Bennett;citation_title=Plant phenomics, from sensors to knowledge.;citation_pages=R770-R783;citation_volume=27;citation_year=2017;citation_doi=10.1016/j.cub.2017.05.055" />
<meta name="citation_reference" content="Technow, F., 2013 hypred: Simulation of genomic data in applied genetics. R package version 0.4." />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=F. Technow;citation_author=J. P. Gerke;citation_title=Parent-progeny imputation from pooled samples for cost-efficient genotyping in plant breeding.;citation_pages=e0190271;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0190271" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Technow;citation_author=C. Riedelsheimer;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.;citation_pages=1181-1194;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733443;citation_doi=10.1007/s00122-012-1905-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Technow;citation_author=T. A. Schrag;citation_author=W. Schipprack;citation_author=E. Bauer;citation_author=H. Simianer;citation_title=Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize.;citation_volume=197;citation_year=2014;citation_pmid=24850820;citation_doi=10.1534/genetics.114.165860" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=F. Technow;citation_author=T. A. Schrag;citation_author=W. Schipprack;citation_author=A. E. Melchinger;citation_title=Identification of key ancestors of modern germplasm in a breeding program of maize.;citation_pages=2545-2553;citation_volume=127;citation_year=2014;citation_pmid=25208647;citation_doi=10.1007/s00122-014-2396-6" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=R. Whitford;citation_author=D. Fleury;citation_author=J. C. Reif;citation_author=M. Garcia;citation_author=T. Okada;citation_title=Hybrid breeding in wheat: technologies to improve hybrid wheat seed production.;citation_volume=64;citation_year=2013;citation_pmid=24179097;citation_doi=10.1093/jxb/ert333" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.;citation_pages=1427-1436;citation_volume=2;citation_year=2013;citation_doi=10.1534/g3.112.003699" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=Y. T. Wu;citation_author=J. M. Yin;citation_author=W. Z. Guo;citation_author=X. F. Zhu;citation_author=T. Z. Zhang;citation_title=Heterosis performance of yield and fibre quality in F1 and F2 hybrids in upland cotton.;citation_pages=285-289;citation_volume=123;citation_year=2004;citation_doi=10.1111/j.1439-0523.2004.00990.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=T. Würschum;citation_author=J. C. Reif;citation_author=T. Kraft;citation_author=G. Janssen;citation_author=Y. Zhao;citation_title=Genomic selection in sugar beet breeding populations.;citation_pages=85;citation_volume=14;citation_year=2013;citation_pmid=24047500;citation_doi=10.1186/1471-2156-14-85" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Mapping quantitative trait loci by controlling polygenic background effects.;citation_volume=195;citation_year=2013;citation_pmid=24077303;citation_doi=10.1534/genetics.113.157032" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Zhao;citation_author=Z. Li;citation_author=G. Liu;citation_author=Y. Jiang;citation_author=H. P. Maurer;citation_title=Genome-based establishment of a high-yielding heterotic pattern for hybrid wheat breeding.;citation_volume=112;citation_year=2015;citation_pmid=26663911;citation_doi=10.1073/pnas.1514547112" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Developing training sets for genomic prediction in hybrid crops requires producing hybrid seed for a large number of entries. In autogamous crop species ( e.g. , wheat, rice, rapeseed, cotton) this requires elaborate hybridization systems to prevent self-pollination and presents a significant impediment to the implementation of hybrid breeding in general and genomic selection in particular. An alternative to F1 hybrids are bulks of F2 seed from selfed F1 plants (F1:2). Seed production for F1:2 bulks requires no hybridization system because the number of F1 plants needed for producing enough F1:2 seed for multi-environment testing can be generated by hand-pollination. This study evaluated the suitability of F1:2 bulks for use in training sets for genomic prediction of F1 level general combining ability and hybrid performance, under different degrees of divergence between heterotic groups and modes of gene action, using quantitative genetic theory and simulation of a genomic prediction experiment. The simulation, backed by theory, showed that F1:2 training sets are expected to have a lower prediction accuracy relative to F1 training sets, particularly when heterotic groups have strongly diverged. The accuracy penalty, however, was only modest and mostly because of a lower heritability, rather than because of a difference in F1 and F1:2 genetic values. It is concluded that resorting to F1:2 bulks is, in theory at least, a promising approach to remove the significant complication of a hybridization system from the breeding process." />
<meta name="og-title" property="og:title" content="Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1557" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Developing training sets for genomic prediction in hybrid crops requires producing hybrid seed for a large number of entries. In autogamous crop species ( e.g. , wheat, rice, rapeseed, cotton) this requires elaborate hybridization systems to prevent self-pollination and presents a significant impediment to the implementation of hybrid breeding in general and genomic selection in particular. An alternative to F1 hybrids are bulks of F2 seed from selfed F1 plants (F1:2). Seed production for F1:2 bulks requires no hybridization system because the number of F1 plants needed for producing enough F1:2 seed for multi-environment testing can be generated by hand-pollination. This study evaluated the suitability of F1:2 bulks for use in training sets for genomic prediction of F1 level general combining ability and hybrid performance, under different degrees of divergence between heterotic groups and modes of gene action, using quantitative genetic theory and simulation of a genomic prediction experiment. The simulation, backed by theory, showed that F1:2 training sets are expected to have a lower prediction accuracy relative to F1 training sets, particularly when heterotic groups have strongly diverged. The accuracy penalty, however, was only modest and mostly because of a lower heritability, rather than because of a difference in F1 and F1:2 genetic values. It is concluded that resorting to F1:2 bulks is, in theory at least, a promising approach to remove the significant complication of a hybridization system from the breeding process." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ZtPI6yltzkTaloU_9lwKK0o-wSRaDrvI72aIpAYv01U__yCSGhNVuCbGf6KLL54AUS64uP6kQfk1f_X5fJJp6-FY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__36Yiqb2U0AhDmqVjDsvW59h3D9k7d0px_jM-NJ9exZg__uqIT76aUscmJwriA5WgUcByCR2EI-ARNdlG2HGF7Oec__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"NzZU8tegY7uuoNaF15n-g6iWI5hlhHF8uHQ440MBHIo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493110","apath":"\/ggg\/9\/5\/1557.atom","pisa":"ggg;9\/5\/1557","ac":{"\/ggg\/9\/5\/1557.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1557.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1557"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1557"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1557":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZmcJK1H6ka5JtpiDo02r+T+18jCz\\\\\\\/PsGx8eX86I9DjZ8sD6Jd3Dk\\\\\\\/3GJSEgkGJ91DPTfCkwj6uvxwCEp2D6H6etDamnm19I94lZ6TMkJduYlSmr1luzdTKw+vX\\\\\\\/Zxy+nHvNOgevjRPpGyNLprW8\\\\\\\/o3mEqGOWYQ\\\\\\\/CCqGDZ6S6ay7jQGDjfIw9Sr7P+8wr7s0Deh+Ls0JJAztwVy8+KKL17ExUMm62l1wXZGftWZ3YIA3vCy5hCeoD\\\\\\\/liSQlYsDI\\\\\\\/rto2vCtH6Iz4bQYyGoHwCSsSDEbfKtGY99lh37Q6ipKG5K954kHfR7uVkr+i40ophCCZSCmAdPVSBl7YjNo2wz+hfHy2JxVl8MPFP1WM+TtYMkx9YjQqUJrdlb2Sw+RXGPdBBxIuKyEMhvXFVRNVb52bv4SsvTvAxk+FvWbAoxkhyWfk18mUEmewoPZ+6EHl\\\\\\\/WdZdTgjTysPLvdfPWEh2qm\\\\\\\/bMG+XbChYsCh955py7EiuFL0j6fT0Sp+bSyrzxQw3QCuI0eTDsEzQndv5t+GzEMRIE4jkvdbYDEYKw4J0SiLTdpg8kYm3ylu1DTgJSXwqFAc2ptY1E3cH3yCElJgcjWBqtATg6Aqy82IjAdbIKgteDjB68n4lcIjcInUcoRZKMCgFEjv1vBLQqkcWNv+vraUeQ4H6dtzO+lamqppP4oIBDDxeLLvu3eECIqNqsjNO7shiXpan1QuARaif0WscXWDOm0XDjZkrDmY7neqFlLjGl5OkPRGqkpJ2sEUP26rtYDzFEezCcA1aco224usT6+mXul1XeFM\\\\\\\/UpxbR00fOm1+1nCRbNheAqO8Wq2HiUa2gu0cIQg1nd6EzvxRDl2+cE60HwUConWoUhClrpIFFAheL8Pz3pI\\\\\\\/CQAiai2Ds49y7IrJCYBZ+0FsQ0i6NZPFygvR0QJmPUH50Bp73zarQb0GYzAMTmnmh2rEvNHqNXScJiBSJ7FPb5NOG3RSEPIHRby6imKyK+u\\\\\\\/mkHvsmAsiH9S2lSaeLLO3RjLir9Oj5u\\\\\\\/x63My1WXXRaTMoQqZRakIkmI1VtKY97H7SNonLHwijIc9+wCbsoJ\\\\\\\/vgs09XPRYbfmNRKZuHMRcK8ewwp1ON0lyWLGSr0gKfOuF5aJCQ3T4CevI\\\\\\\/E9nA9c3Z6UlgtXzYJeqwyPDCErhJK+2X76M9yiB8hFAU8sRsrji5K1Q5NPa3KS69uD0KsY+BIDfPbQAIuLtbWyrStG0CeC4dBob4eT3SojygD6ew+b0q1R8KgtBytWdprjTGOJh5SXGfgD3Hx\\\u0022,\\\u0022iv\\\u0022:\\\u0022WqTWlb9zBbHIYjFboRHYQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022632e5e5ece939ce8bdf589b6e8801fb11c4688f7d76fb6408117f3907ed4a16c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tKNIjDIk0ss8OjUL7YfBTBPEhE6uvLl+NjnqUblst8IG+Q9o8Gtt\\\\\\\/wFEl8CzAIWaPD\\\\\\\/NIae4hwcO3UDpNzZ+o7h7nzI7vNs7r03WjzkIPXvKP2a7pY1rMI\\\\\\\/YJ+K7slmbqFFvkDSs+Hgoaf2B1d9WYzs6poTzUulyp0nSnQ\\\\\\\/7m8mw07fWF1jMA79fC0RegUpnwb1kk4UJPcrt6GYUnA+aqAGqNeZ261PJHOBMEJ9StkiqvcS19jZhUUiPo6BrMVCT2PXTn5vY7A7Ika4teqD\\\\\\\/GDAI+BR\\\\\\\/2OQ9BIQENmdsDExFci8qciyfzVvXggExylcEQUkq9accNgX9ntEZdOhTTn4b\\\\\\\/uYk+r6FonFHZXIMgDh8HpLqjqooeh48RfcIxSwAZldW+2TkeIVr9WPTWoTWCXdtUJ2RURrFfuhZADNDCjcrOo8WMDX7lxaBOuDcSsOuuYMOtrHJRSQiDuQQkrwBQbRHIaao9XlTfASt0nTkqVo0mmHMpiT8ofyj+BXHZthgk5ov37PTUtlGdJoRNduwoBycz\\\\\\\/S74CXNG1D9GQN2akM3piChiqXdt1HPC0ZhESaj8f6cdeFj+BJjTKhgLvlBmsXPp3REGwmCS09TaBLx7NWbWUhE4UJZDNoLsbmPpHOiRijxFB1gxZoyHMC87FNU+lYI2uyFDtSsfFXvdnV5RUdJLnMFLyON+h8\\\\\\\/7ZPeVGwlLytk2RjOAtmbffKQ8VUlA3S7dJwRV0LS3EDEuMmY7V+UY7tMq5yjavxuAZVqX7gkE23mxhi+U7oA5RcYdP4MzohDzvU2slaAP+fBnCrIA+FupPkP77M7UWfTRFeLKAsk+BTRxDCgdthO4rQSGwbYaM7Kh9nL2MtU\\\\\\\/ocVb9HgD4qe1MZoqzANrKhJDEkYuv+G27JklXe6qRKUTgKxn91xSZFSARGuADJC1lWEhYR+iDkmpyTD\\\\\\\/\\\\\\\/nITmuqJtMxhY\\\\\\\/BEFfg3lNeHDIURHZy+jS7vJ+gJUgPQBmha7jPFLV1IVd61G9dz5l4Txs23jxyK246CQsCFjXMf76MyqZlzjSWkIFG3R8PRgWxj5Q5oZbIiBWOZi7H7EJHlaG3YRgIaxrX9z+nYqed\\\\\\\/n5LiSQ8cNcU1QPNePkcAtJQzN1TrfM\\\\\\\/daoD8A2DUyirW0qoK2F40+bZOThv1uZI1+y6vJugte+jVlVxE0g1Yb2o82X9HiHe8p1aPzeWi\\\\\\\/VdO4nSunRYGYeSdL7i2wbFaPQIJskvXR9sASqVOGSfDqkXyBsuwr7WqUe3zmdpYWu\\\\\\\/gZSH2p24w5vtj2INKIrau10ANsjTVm1pvISKHfHPz31sRZgy9F+KMLrp7TjGImQtkR5Mix28xGRKSpI5i\\\\\\\/d1xcTtAjAMF7lg070\\\\\\\/HlWej4JgdZkrANYqa2pxqmz+3Dqb1IT96MFpXb+I8zunUua4wqc8bvr7+RETUIu4sYW1PG5PdeRuoljTJjjGYhUemsdo9NYhnUajLAsCbCiAkGluVNXZCGlpiBVQ\\\\\\\/5qCo3oiFLj8KgH+vF+8Tc5wNKH\\\\\\\/PeEUujCVmtHTEzea3YlhX613UuMiAWqywF9UaaUFcQveyGqk8M\\\\\\\/oI9\\\\\\\/4WQ0C3qtZUjBvmgr+XD1iyfJiLC\\\\\\\/p7lM38YVtlP0xj9DzwwPgrxjUshg\\\\\\\/dUxf6lGRNIzXEuotOnKJfO2XxPfenGDM6X29hYdrIhEgiJe2HUqg7\\\\\\\/I\\\\\\\/eQxKjkpnGJ\\\\\\\/XDMQOeW1esXln1Zy9VJEwsJQ5xI4ZSMxzdomj9R8Y8\\\\\\\/q6Y0x5h8+RCpvApd9EdJC8bYwvrkBDksbP6VGDUDeXO2C9FWtMaldO5kcUlV\\\\\\\/Dj\\\\\\\/BfH\\\\\\\/tsDyuZaIV88j63RzQi\\\\\\\/oFMLzR3YDO95Qco3k\\\\\\\/UvhjmozF+8D3yJTsD1\\\\\\\/QYVo6cLYH0zNLiRCwtTx98LM66WBOZuQyG94LcZQiGCuS8oG2thUJgXCu+bMyFl0WbNjr1UkxFsyDAnN3APZxz+5TOn0T0+w2uH+X2JYdvJsRuQBIggx1AZffHGCkPDZkP4eIHtTMOJpIrIqKP3rT6gq2HqOmMSlNQQ5aeL1yPCnnZSgyZOKxCkYxPNV2dN+YV53qo73wc7R+TVw6zFSi06HahuDpfmePDDl88xju6MorTM1ACN7dWwjZzSWv9NniRS6UXSlS3JYYO\\\\\\\/qoZ2X5vu3QDvGKR83xJO0onQs0w+n\\\\\\\/bdS3MGqCIUvPkW1hFOZ7ILi3LAyaZ65Od96eSbF23vkL\\\\\\\/4PIc\\\\\\\/XybiRXyKSB7Y3WFsGsYlywbgsCiXBqoOq36oGvJsSJUJCZnQZq63s7fQ3ZFHwr+FXMfoNYJEn5iJZp\\\\\\\/ZZ7UjXiHHuv5PNXc0mzKX+siwqf1balsxWNgJI9uuPDCi4k6ZhW4DNa20sQppW7zxq7SdRXdQ\\\\\\\/7hDRSQMdZI97ToMWiEBAYbzqLzGS0YG94tKvhDRy\\\\\\\/wgOletN34fmufU1S5lLX8DPS7PN1iLJZvkoRC32HNu4xzJ5p04dT8jqpUWMCeXzrtSCUHb8+QWSQfbQrZuB9b2mVBK6kjvZixNu0F8u\\\\\\\/fazr6aaQ7J2cXHgJYo6NH7BbNV4ELk1o8BFEUtY6ToVUE\\\\\\\/DWspifAKUIps7kYBLqRd0GytUD7IqYfc8fq5X+T2hq8SCtagQw5xpr\\\\\\\/aQFib4BLj+HkjBo0+5vPeeYoXLQ5Msx6Bnw9xcScMXuyVS3zEiu+kPmpZ3cTmN8YaUjycNRkK6PsrDg\\\\\\\/2Pk1xQEZByGEy\\\\\\\/j\\\\\\\/AXCkRVraYp0Ccc0ocJNqEsZCcIVX3TQSsgl7jBU0TIbYAsyMHWX6MwS2BoLGPEthhxb5NLc8AVLAJJdz\\\\\\\/5yXDTzLhsXPzDjo1uqfXXE3C4n0CBOumTjAi2JspIiOuPE97GvFPCpMaGsDzOriy0e0oKDVGxc5nOkA8JznOHIeMIoYlr6YgID8pdjJdjeIYFxJ\\\\\\\/xHwcId0VZbPmRshkC8MHinRZLyMYHRgPqIFwvb4ZFOPbuIlcjwsj1\\\\\\\/WtXhLnQIPHtk9ouKvGOs06pZniVUIND1Gy3siVPCs7nmrm8V1hYeteQTOm7HIOpoYqOZs1pp9rpbfQ4sfZc8BhQvGJKnFwtvCkyvY+OAdKwCzGvlB\\\\\\\/QOqWvROkuMkc0b+RW6zHYvGIxVOQ1DvlkIRO5YeRjUUpbo\\\\\\\/eClgP+q60NZplSI\\\\\\\/NZQPP1yzPPkLF8y8VmAlZ8XerSfaXI84qwfeWhm6FmkIb9F7vd5H5CK+EclXaZc6BKQ\\\\\\\/M997kkFX5Qxb9HwSq6gkk1Xyb6Yy8Tf7WsUOud79G\\\\\\\/61SXYe\\\\\\\/6X2TgTrx61LlgsyJc+D0EtDi+Ljoq\\\\\\\/rQVvUA8Or6Anj0JxiYCyqbCfj2mBYCVyubnfgSyMgOq2K1zMD+cxX7ccfMGqy96WOofxrjpJAUEVD48WyewsAXwr8pvbgIToYxfb\\\\\\\/E8qrS5MjNuu9EssTmx1YPel8SHqcnEDl0G1EpwJN6qYTq00DOf5MA3EUruOdYEGmnTQX7AQbomv65\\\\\\\/Io1nJnToF6iQBRx1vASxn7KAnRPiXN6NhAWGjHnGkKWntnTxCbjY+OW0rEyNpXlhHV7kBUdDEvzGZbbIQFXq0iIGIbLGIMro9cVsXA2AKLkhNu+6qgG1YZpNfEAZwMREdxlJ10vpn6yu53Bv201rX6kNtyk8lKwBta\\\\\\\/Fdu\\\\\\\/lLjAdQLjS7Cod59FOYoYQWqmDtEV8D3JA6K19NvdCrO3Ck9+TJNH3bZMwAAKIxD4xWFrLgjZ5tpuRtITO5h523WzvQ3+jXFo81NkQy+5pnRT7QrOz1wIk6rIYP66klaUvLa1bbGnTO0USvLY4flteNl7M2KiM54u6H6nmoqQ9\\\\\\\/6Tmoqg8nvWMVDhrsggN63PrqPtY3JYLZKmR6q1PJl6kNe\\\\\\\/odqPbPSWjZXvP5pXJI2w\\\\\\\/i5m4+4ERNK8NZ1aOTE8Df3J696TZVbrqZgXOptOCbjkRmYhEiLt5bL7LoOM+uCBtqShUrIzzIZWAQWx9GGwItB\\\\\\\/fjPkESU8Xmml+Oosh3rmkZgfWKx5vJU769FkcxU4q2+qz9rZvGtfRrcwFYVT2kk7hTHcqpKIHjo1iZPSK+MnBLwAdOZlAiAhmF7q\\\\\\\/U0yjD8EbDTKZSb63G6vHHM4ecWqzSSvlR0CvAyJBLWr3tTutBX50QW50l\\\\\\\/rch9LQk2gA96gs10R0\\\\\\\/IQv1VwmjpDK1OZTDd7np6FDx3LlHyT2H285JCnW90pNOr8nq7w7IxX\\\\\\\/bs5pB73o6S8+PIvd\\\\\\\/JhGDHHrSm+DnDo8HIAkFWipBqzm08HuK+LKkQk20y8QqO55slZEt+YWjrpl2qHfn8HFPnNLN3OXjSh3p9I0tYHkidf6f8fq2CjgyhsDV\\\\\\\/Q91W6yLS\\\\\\\/B23\\\\\\\/lOD1fW8btt7XlHeCFjILevqEomXZxDOtp9QZddF3VgELC98M7OWWj1Ba8dZW4QiBQhFcgNRHUvf6keZJtLGTeh2CLiNYc0gR0AvPP3nxaQhlMaOTdCM0O1bI9YGenqNwmFtJI2FkI9RcSv\\\\\\\/9MeIgCkj9ibyj7WeoBPQ5shEUw\\\\\\\/e0i7fbCl3pKrPfk1t7roQUspKrn1cKIWOAP8FGIeTMoYxpAusg6Ln5oR\\\\\\\/mkPDGvHZ0cMetCvPzOGP\\\\\\\/z7S0W8nC0HNzz49iexkCboVuxwlaPm\\\\\\\/J0IUetvsefKnU1J9HTvG4QRC6dscXL3XNF8nbZs5Lbk0yKW46Bdbu\\\\\\\/8kU7CwLgExBaXM4L0rLDe414gOb1h8sB81jhnenA3\\\\\\\/44cJ604EnG0sQc6hrBrcBZRc6uHZLjvQnFhM8VKmZVQFKxqF0+auPag9HSqv5z3upA9PbCGGO\\\\\\\/VCZaX41p6Hm7POuAXcA\\\\\\\/i16g4c\\\\\\\/gZ\\\\\\\/vXk0QNGbT0q7pa6OdXKzo9Hr4u3BM4IQFnkuLj+\\\\\\\/Ucm13UTgRjthN8bY+yPJjb1IL8PaWTTZ+WhRnhbCCc\\\\\\\/PLHbgxPym3MYXFmO4yRwHao1J03eU1PWqmxDFaKhyL94jBo8JdTFrZUSmajw9G4Lny\\\\\\\/ELuV0tk9oNMd2gac2FoqWVuCQ3657D+rRLh5iLG+rrPiYZhL1Gg+BCX9KLjuGswKqq7nvMlY4JX4\\\\\\\/juskEHBnyMGFO2R3hMyB2x08LqOSvDL7QAqVvB9jytFO9q9mDS45Cx0BMQhCAfJhmOynniAUrS885IyLnQfRDs6ifdt5ZDr2MUPYMH5LF\\\\\\\/Cz55ih+MRBVCG6JQn7WLPFM1sx19eifxrpBlbThKFuuRuhm2U0oDiF5yIyItljZCCBXxYEa5ICp4Hr0BeMlo4YOx312kLO\\\\\\\/sr9ARPJeP1D1p1tinOo\\\\\\\/ys+Eh85YZWkDnigTDnIBtw3+rrsivC1mfrfEKezRFyq0qY7ZOIxRkvzx8k9MAJ1UalhzLdSeHqBvkEKGhrvSV9KDaIiQcRniMpvC0M6Mr5fdjDWEyeYEd+wH1fxrONe4A\\\\\\\/KK2nhIveDmSOwK1wTrOus3zF6T90idWFF\\\\\\\/5xxcetmrN\\\\\\\/btLByvSdYUkoOZtESFAJ6WbX4c0uCtWlJPnNWN5kVz8789Ijhikummc7K83LneaEsba6rJ6\\\\\\\/e7I5H3LZMZIhP537\\\\\\\/LKILR8In9t1LVq1Ei2qDdafkiW465GpeCMlzxoq0YoRUoXdcFbkoxHZfe57juLO+N2dyueUH0bXKEdjPgdCmMxWB\\\\\\\/Qnds9BgyOLNeX0SsWlHrBo29KSosLZb7LzksB+XAh0WIr\\\\\\\/Im+ctbS+Uc+vCpPD5j9hZpXGnQ1pRBZOUW4aViymPuKjgH9TeuFYHbaSbSCBqReJ9eGXqoSIyM2JyYGp5SrHqSIQY68NQ7HmWlrwnc5aC7v9vaHpjijsdIrJg1A8o6ke\\\\\\\/ZuaNg6j1wNd8ZEiMLDbEhKxB\\\\\\\/eCqg3BgxdRYXuGZu8qmzJQ\\\\\\\/ALxcDKSnSxBSkPjQM7DZiDvOHGRLU7O2cG44UXbuYTWLVxSQPPkc1iQMWzFcEW47paVri0Rqyk0E7NNzxnk\\\\\\\/Ya76Tho3Z12PZ+neRUo16yvlsiMGFO84tpIfcEqp7eFqUJg9LYAzAxGGMKpkb7RkR4aVwAMQyFB67AGX1nMhQnoXzn1gjCC5Jdx62mRyv46yF+iSVAYLVTbGXBGxVeW\\\\\\\/KV53Q0orCE38aibsCcyj5qsBIvK0Rbm1yCHNdDvJgtDFWuoBWJGny6bbRhPIVJKDtG+3\\\\\\\/mKbLQDeMH5V0pgnzYi1WHcFS6RzG8fjwmEpZuHrL1AaD0C3m0Sx0u4cW1roVTPmiHOTQzCdYATGY0LAKJOwMK9nv6IeuQA1E8F2+WHvrRlgynajUDPStrOZM\\\\\\\/AQsDwFsMCp8R9CvYoHxwdRB9Fltf+WYSgJf3lfdcmgomREszqrwI0kDpXBVQ+DaTDyxaqvzEaEoH0QmBs4EkE8WXXzFnE5CKndQdgQAahjrp4Mt7gKRYx+A8ywdN4M9FmA0m0KVYdT1+pd3PjcZm29NGsLnAyLS4XGeA4fPfbYSwVVDjTZVkJuCJY7+jtfNbSQVMJ8uCG3bJg60J3d01cgNu2HWUJBDKYG8TlDikt3wRe+IQml638Ps9DPjqFQwN+4p8lvBhnn10UG9QyrcSSnGuHGF9wDpSI6jOt4yt+TgReVRmLjUwLXEX82mnIUPfnzapbhcnFjyuC3uh16JLKKtvIxFK\\\\\\\/ex6Q0Xs9ytnUoelZq8koAPZrPiEiMaYm8oYmmHDtrX8fbigrTRrxjLV5d44Z8tJse2o3nsT3qqT7uKdz2Sne+Zu6n7qNPUu95zUJVwLZt62uSKkJZKMXpsEFv7ELR43zeefiurrL0hBnF9\\\\\\\/1O8Co78f0Ot6RFcLTjDisYfb0RFVryeOhoWI+9jW7hpNjiD+kBylpVKIrDC8a7grGh72UzPmMZ+buY12yCYnb+86ionOEBf87c4T2ENCagMi0PZARAHeOy+efcI8FoC3Wxh+iCJbeVI7GZxp9SW3oAzn72ThO5z1v+SrpXdH1MxFo\\\\\\\/1SfNR6bNJZjIiSvKL0LejCpZ4lTcCT2LpB8y+Zrv4lefoK3dw06tCLrNj5ZaCXO0UESIxMmaOsXgh4mnLuMEtxKeZixpLhWIOP8VCEjbxSB1VEGf4ZK3Vk6OhDGWYEmRWRShwLwn0N8rxgsl7+7SfGU6Wb7CNKtNR76hW+iWJes9yHHhSJ1oFRo7+rHmP+YYnP4ST0dKE9ujNnX87xu6NZHO+NnjEhRPvn99CFhk8vtmJ6lWCuxUAlfM+UhuHysOGqA+iUsAjEuVJMKkHTlJ4xldDGypDKpZ91XCJlOaibyi5p78I0imZsiuH2JYp0r7LbX\\\\\\\/mviWlUfNSTBguurqDcaNh9K2AoFsotTyF\\\\\\\/5LlAjSTYUBMqquNZHIIk7rTaYRMdqiRunjJjdq4dJg5gsBODnrjSeQ8Hhqowf7HmOhwUBfvbLLAfANYfq+jytnbcI5efuEozJVT4RPyKArp2saTft8qRG6gLe+CtkEHAKOhNAMKcTv5q9OjmZjelf9rGZ8Nm+n\\\\\\\/Ae4uw1TjpzJJQ+PNV7ENGEJjRKRBjf90iMyOLCfd0lFqc+JzCxyr2hYtpLbuXh5ANxCZ4PvWftutvY\\\\\\\/iO05xQke4Ppi5O\\\\\\\/3d36g78kkIQ+6p+cx0X8rgPilv6YfOGtZ+yBz8PFdy1ALlohis5pcz5a1pJl3RMV2Dz+o437fkK7kjRcTBHygqcV3HH1CiBsmSCS\\\\\\\/nZajdAVjI2GNqdpBGYSobb4+jNYanxPxiK+5CaL6IjKAJ66EO34ZbA1f9HW\\\\\\\/0ddH3L0vtiYZlpb0XzmoZStbUm5BlsW\\\\\\\/MMtrc0gKQWdI40CG6gno9RL1HsrztC+N+XPPqIYunVSV\\\\\\\/YBB9ZffudRshnvoMWYLG\\\\\\\/CiHW5YYpHKTzFfZQWhGyaCDPTaGGrmCVpYlaSJqxRuHqLu5feaSnk7ul+ney7c9BpvVl6EHZcgqUhHzX1ZrZaHXx3RumFRI\\\\\\\/\\\\\\\/FGnTklDHghtCXeHt+SlSkgZZG4UjKiqXhtfuclzrjFCNrZ8ly+pqi53Z+VylBVPMVZl+graK4ukAh3Aa9FupG+ZxVbP\\\\\\\/dxsVH1NY2nS89xGTeI68VMj6lULM6CmvQc8gA1jBG3rcLj1kBeW1ZO\\\\\\\/k50WvBacvoGRffSNvUEMq6TXGPYaPaZvGNRzctDK37HkOjIxgDjYU\\\\\\\/KVcoZYfQoAb0et8IFIelzPIWBkD2dzz1MyYx+rGQev30CPBGBMIDWyBkmyMv1LH7vZj+pFhgttVsXkju6WVp\\\\\\\/STUb1F+BIiJ8zZByqDY5YxsXo9zKrUYrETF5iKUpAU4VQGbKzcXJHkGIFmtbuudxRNGK+01UjujLadQHQiOgNXJDChDqaUCTK6PVu7L2homFFOuzMtFV0x\\\\\\\/htCWeezOJVJwBU7M1X3uJY3PSQAZdNTYGc8sk8TNxKOfkh1YB9cq6ymYN6kkWE5jCTKL936K3kwgz+Y\\\\\\\/eL44cGlpeiWbJefHje\\\\\\\/PXEhCMwugM4HHkAirvPFU6VbUpzDeLAzEPe+A7whPGPs33kS9mfbc1JY8Z7x0snmcoT3V9LMovc6t8GBiPsfi\\\\\\\/7LtnfuSeuaOYP\\\\\\\/aFnyA+xKwYf8HxXRZ0VTut7Zt7LyK+OeXw6Pf+qPHi08+9w8YFqiUu9tdx9+HMOUs05FcBgKfc+1XNhrue4V+GZh4V2+\\\\\\\/UBBhmyMvJd+oorj0eeTsLnULvr2Z5ToRz\\\\\\\/xZSRSpucb0dSP3wEqMEW9KeKKDLdCetOGWogG6NcLfQ5gmi6cFp9TyFOccQxDc0fUJ+Cj55D8ikq1Q+0SuWTqg6EEV7rtRdmloTNgnDoEvgYNCsy\\\\\\\/S2jPdNwJezmGWYotdwwAhau9v6zEYkAblwgaF9bFlGcK4gKAQuipZfsYnznRCWrau9zA55QPk6JM\\\\\\\/V+ojjt6ruPkNcDbkn1B4qaSFfY+FcMNmd02k+YGbUx6vg+C+PfSSteVaM0ZJGBa+h5lgXYZ9pXNLdJQModNaUp9hzd7GAS+nybkUvauhjkFQ\\\\\\\/kIJJDVkxr86165EfnHOswOD6DTIa9UGegYjnO28ykzp5UJ743vCuc58uRL9nEP57hn31wAizGstpqCc1741cZ+Zq2tUQGZNL7SIS8EPTybJsLviqwV+aIsKIOJKyjAHqilswOMN+XgVbQcp7SKnHNQ1FygDL0Ryp\\\\\\\/JfIzRWGV2sohUWO0nxCHbKXy48pvf+tgJfYFVeRMtbPSu2YsRfG35Y9hPwbUq\\\\\\\/AVEd3m\\\\\\\/5EnaPzJsDgP+MJnTqMFoFdlEa91uBbbUMQZhDhRwIkd+7vACOJaxn5ZWBGccItVGlcp++U95Db32Y\\\\\\\/d\\\\\\\/WeIbCsFz6W2VRop6LDDO5Z2qNwfTKnW0rcPFZj8+JOx32JlETrl\\\\\\\/MIRUCFjbnx43fuPR4QXdlHaY+lgNypiLlgevSKVmI8odS0e+ENiprPfhOIRFQD90BAryOuIS0nItadjOzW7rdRUf0cy7gGTY7CmVFDKGI1hdQB72\\\\\\\/W8nlaPb44yfy\\\\\\\/GSXjp6mt2erNoaxcfWt3lkgwxA7xcwLfKsx7sBTAlYW4zx6W5tohs4GzvMQmtngTIyZBpIi5lB+BP\\\\\\\/b96VbLEc2Dl2XAhJFSbqnYS1rJ1UJEeVQ34R4zlFiNH7QDX7i4jPp2G\\\\\\\/8WJomDq6+9gk22hxVR1fqeLYb1IaDkN6tH3dKc5h1ppiVA5rJ5QQhBcCQv7cWlM8nwrXxCXzikZos6RXy82R0FRcFpBWQYmNwPpGlPt8mIxhX5sIv3+vLbgqF\\\\\\\/97dX4bbIzqcWoMfeS10fEcetaIQd7SEJKjPTkkjb7nD3NmfgPBnaGySM3dV51O08cM7jFrbHptVDqdcnuj\\\\\\\/0\\\\\\\/Q0p7pG3rHmJPi4tk1zLfMavRQ0Fz71BFAIPFYoB0IrP1gv3V9IezAojh68h7q4T1Eu+IrwqQGnaeuMcj1YvnYkkh+C5wBXfy357r3vGf+xxm3khlZTlt3QhKx9YJ3GRCRnKhEQkqruBqNyJWTMMj+qVIAz6M9hufJR25hhh+Xn2py1U6YrZlaoxmKT6vseiNBZSyxEtNPlNtAmeJPU\\\\\\\/u\\\\\\\/4WEpihNBZ5\\\\\\\/giYgnw7HRpV7Gm2gHA9sEJ9iEAt1UjSTFqH9ztTJ9xtxGmGljaT2Hb+js8DhWvDEoxOwI0ilBgxu4+fY8FckrhzLpRhvLc0Bvyzp2omsl5LkioMm6MTFGzxKe+9SFIZGGeAeR7xeBcPqzILdvbqIybNzyheW2bjGvx90vwLgszEUj6Z6mOmJEzCa+vk1oqPJ8f++gdiZLAY9tNd+9E1cSlxGsyJb8rgNRaOTKFQ\\\\\\\/ZxCezZ4Pzf\\\\\\\/uEARjIpknQ9y9LUTDsOYhJrXibQbBBFAcmS8taT5z2QWSvFesYdM95aZ\\\\\\\/JoJEZPLARCLEUptTVHGW\\\\\\\/r9GEyi7tlfJoeLk6gyIv9G1qjWRRg6JbcsK4W2qG9ijaghZfOAjZKRmixkmcIEqYTvfx4jDzjZttKWbFylptVcVuk1PmmiCRt4AEzuH2paOLDPsonuxTT5FTlqKHA6vOxDnjU1KLqyprUXxsL\\\\\\\/vm1Bn97YcFao8qDgTD+EhQOXz1WvZHkqH4u7yzLPQcisQMYxyt92VudqRdxp\\\\\\\/9dcmPhxFHLuOdzjEFG2lek\\\\\\\/JE1wSzBjzfQyPkIS3IwM9jOjxRS6ASWDf8Zewi5EFTpKQ51QVZ+Hl\\\\\\\/ottQhhQ+maLaCLTLDW0skdakEYeEU082MeWAo4ZGAX3kAgcl8UYXm3d4k97BBBd3jyVcSM6518zdEMea9e0c3o9FM0TwnhjviBoy3pMBw8STNgvZ76IDIf4pY0Sx8DLMa7DRvAKlCPNsck+pFB16R5NW\\\\\\\/C\\\\\\\/2gFOPrA2PJ\\\\\\\/i2bIaUc7Ib5i6k0hk2JtajHB54I8MEBD272w8\\\\\\\/AGNseQx1ypkfJeq\\\\\\\/UElPWzrJxnlmr2NU1bXaq73+zPqQBuCl2MeNKzDZOibX064o80CNi1lKWrqZZPzttSKLElRNDq1CmWPP\\\\\\\/Bc2d9JPoIUB9nbmBRNNvatGXXqCqQJgOFb9Wrcn0w7d4AEFua4UBDyvCvsnwCpteTOH7e1tngR9w2rgeMKsHJXi2jv1tuVI1TJIn+PtPqvgJntSaeJPuqCUYeaWqp1GXOSVDlfYj5DO0WfZtpAOBJ2Ocf2PC9bGeNxfS+Ol9a6PrfJyh8Y0OQHxDJ6nu021tNOYkLtc2ndMsR4uqjwaDAstqVTR2VEN3ZgeoGLcUIX46YHMXBm5blj8hcrId4m7L+ERrLMRZy2q8e1YE1aW3+FxwHixqpVVu0eyUY0pKGeD9wjCHTrepm\\\\\\\/0EFsXPjxux3YE2cKYyHjpYS7Ouajf68zQhlQfGuIkUWruagtTrhBoTcB08i\\\\\\\/ceitqe+zYWj1AbUr0kDMz1VKC8DtcK+MudPbtwWa1fjH55ufMEjrTcShGNBe4km9nV007Ye+ugnSNYQZV6oTvHz8UEZ+qnJtKwzATU3CBnxu9Nc5zysRQZSDCP29uC5hs7Z\\\\\\\/9gVGmpB8JGM\\\\\\\/JVTgOvDg4KhizbD77C60EwsjPyBjL1sFVCHD89TH9CahDsBEkc4xOQCgKpmDHDqznIffrSV276r+rQmGtOthujWBWIsviWFMg7\\\\\\\/91kmnoPaXeVD\\\\\\\/lA24kzIw2CeSzzyysAGQMir2XMH1OGgTrsHGuzTW0fZGfhoODD0yd4eMDZimWcg8ShxiIaKDP+1EfooC5IRSjF23wybhCrysK\\\\\\\/3x3JVX0zHpnRy6KSHHzacErIUJhdIfBUP3Yl\\\\\\\/GVXgMrQMqn\\\\\\\/94lGm\\\\\\\/KZ0FmlJWi3f8qtJqiNUFh+EfuibemIOzW+eIQp9r7EnknzjdWv8+oTe\\\\\\\/sKjP8Vqn0fkfMeRZw8z2dNokJzGanhdRittwWGr\\\\\\\/TFnVp1cdw2i19zCUGf5FRx1hHLRJxe8Y9EaJDH5DLQS\\\\\\\/Awou5KsRre9YylV2b85f7lZaKbYuA02IO6arMnjVKn5ZdyxUmlPrZB4Ypg0xyRUZf8xMqfB5DtqFp3lSVC3ZM8IaLwtPgjekM66vuhXmPZOcKKpVn\\\\\\\/OGEBx6emMUN8AKK2wTqQccaql1ltSYTFJWpJupt+wiWK89QAstV5MHHntCWFn5vvVywfYj+gltDOZlaa8vhoHC6K1pym81LCS\\\\\\\/6szPk9LospSi\\\\\\\/tk5pUGXEiWiLwF\\\\\\\/LOPSTw\\\\\\\/Xmgj8sKVVU8yaCbYnLL7pcwFmeNnznZKPpz6Ca6L6d7YrXjTeUe05m\\\\\\\/KjEMyURoVHQICPgoSIjkKvJ5m2z3wWfNSVaC+AOkiXlUeDjwjuYUMgnQNmII02Rd6mlAfkI8DkzWKbFMtrupCGh+fKUReiCQfgnk8+a9CfujR9IMGuwXTqk5bGO3uvgrnBWECPVenTjI73GjPZWqUzd0DfLnL9WExtQlcZvUm\\\\\\\/ieDWxx76sOiknSJ1g5HDFdBf9Fnui9gYeW5QhCKqrBwurqkQ6y6aUr1Snumd0nePGjEy40Uu22pcr6eHVq3WcDscvMd9AxMfMaiiEZ\\\\\\\/orpjK65qcdZ8B9wfPo2lEg+a2P9796hTfsk565OEXhf+YLnPr\\\\\\\/b5mE36NEHCE6jWuGZD9Q4uIOtaSn+FaKh6uU1F6d816QJ4zFjdupnEctNNfKu702Uifz2JacLJ9ZO5qXaW\\\\\\\/QMXYRzKhxX7a3XY1OYcri6fq9rCOlQkgZEBrnh2dDmhT3+++5ulb\\\\\\\/repyXPVgfARGBwHIDYcHMModS6GvrYjQvAj8DidP5VVBvC3uKcmXcp3gGKPVgbZsWfiWF8LXRRUyXgsa9hvrbhK7Rm1facnbDJZ1nTTFXgbhjHr++ubh0ljfvU+7Uzf1zK4ysCNQrNYOUjhFqxZWHpYVBXqyIxmdPpOrSCbD\\\\\\\/l7WTwOfTTdnsRW\\\\\\\/q\\\\\\\/xukvDAaRcKsgY6rVZRvA3YhENwyRfqoZ5ZW3WikpQ+jrlr9GlwYRVoinZlvKXTROW96x4\\\\\\\/SmcjUPk6mZ1ALa+cwOX583gall\\\\\\\/\\\\\\\/XCDOHONmsAyxMHWVkGbaszJvEgY5\\\\\\\/PAtkNMSElE4NyP6mbTXKQIcYs+T979j9JZGu\\\\\\\/5VWiIYVhyks6+qRpJm5fYjgSkWBU3Wj4MJPZiQJNmpW++8p\\\\\\\/xq+GqzdKkQ7+tYlhRoySSYt742ijG0mpVdOItXx7yWbBmUHFjcE7lw1pGg2yaVg3tIPilAQJnMxx5nTLMKd3YFagRnsDlrfIi8hUprPZSzLuqXAmGQOnTsQeOPEtnkrWiPq0U9cLfmgnyBgjILfph\\\\\\\/gxg7wtXGKWXMBYyevz43KcLrCfXlXblZAgtAA\\\\\\\/LI264hHnIUwG\\\\\\\/K4K5d2tnrt2qbGHYEfwBlkGgc8qWTf4IAhlwo4rA0WS0ko2qYldBj\\\\\\\/FNnfcHBVDTrrH5fqK\\\\\\\/wbPvgrNjLi\\\\\\\/LuK6lVT\\\\\\\/L0LsvtnKHmWoMDpnnmwHODfOJwOW4JWBWpAauvdmSVOpX8qbL1wvsUFJBiFnLoPbRSdnxpywR53K41r2hwhLrIn5P7as34djuJTzfkY3w+\\\\\\\/bsGCWgePuvatrz44UzloQk29UhjlBBDSlV775uux4lmjATbiyZDXIP7OeGerxNOJ0RRtRQtbtcT0oNfUB8yyYY1rVQO6opYsKYYUXWbBlngHzbGVKbDGBmfNGC2H51SSPAOp20BO4Ojxers5hme+c9adi0YWWwZ1gOV\\\\\\\/mbLSG1gKn47Hwd4Qf\\\\\\\/6vGs0zkhAXvo1ddSUpSb54iuI1iLyN4ke+aD0t+ilPMamLoQQYv\\\\\\\/OvYb6ydO1b2dm299sMiU54MzkQEyJEwcMPMz7VnZw2D48eeo2cvnXX0QYD2W\\\\\\\/\\\\\\\/x+aUdS4FhB33wT7FwhQUmVuMTAhsoC6yJatZ3SSe8s4sW2s3Cg+NttoOxmNpChQ2y0a5dWfjms15L8tBVD60fJ5zd0rR5f89wo3q7i8SKQJp2oYCN8i2D0\\\\\\\/sAbBLVJjBbR\\\\\\\/lKg\\\\\\\/fsmXTkDXtpVmm60idgQEfv10+ZVhZHNFHjO5w6OimB04w0Wf2L1Py2eZ14+hqEMiAa8IueTPAyJj\\\\\\\/JnmUN1pUNNuCwaCDwNxwTHc1ZO\\\\\\\/qIWC31MLy+s\\\\\\\/aQks\\\\\\\/2a8CXnIV6+S4wHkLv2xSAuToA03+MUeIqZMk41PhCdF\\\\\\\/5ZgAAey+e27ob8MgBSarzLcGKEsNnFaa9kgLHOJHjea\\\\\\\/Slx6Sqomy9iDABN2ACZBukqgRd4tpXpV01PUTjORzN1GYl1QG9xdOvbOYD3N2gQN0BRedvqIRjd5J0++7wkgEpq3AzZ0accMl9c20v\\\\\\\/\\\\\\\/BWqE5PlUjOvTA+hITWevOKcNazWUzKc1R0F9cTpFvwilvsmZLX7CzRXs9Rmxqk0gUHKQqY3hAUAww60G1MQwgNdJ4zoSbMrwtfxfgqTCvVRep\\\\\\\/afmKx9yTeBjRycH9HHHhi28+1N27NwfzQViLaoegg2ek6h5ZVNg0CGWS5CzW7KCqyn++zdtxLCrRq6vqesGwREL6LmnKVJUvESyub5o+oPLuhdkTO4YpNxziNK9F3cMWZvIfwenzyn6pvx\\\\\\\/a8Dvq6OJqbf3uq93zvGyEmlylywYtxSbCm6KJcCVDGx5X+uGaHYjG8IqMzUEGmQpziemmlo5TA8u1sPCY3SXtIOarRqU9VeJH3S+ZIvwv5TuNpL45TDV+J610A2iuT51qTMCqc56XjDv4mqFy2xvTdgvxS57y+OhmkADVIirWVQP8a5b7N5+5aspEekZ\\\\\\\/d9S\\\\\\\/DDa0VVYZofYIkoF83AiO68q6owkIVtIPpHruPYoauBl41Lacwu2Vs8eR3Tz2Hvpa7yy6\\\\\\\/brwVGL3U6YG\\\\\\\/hbaaT6c44Obwsjc79tLSgZLqEMXtD58uL\\\\\\\/atDqGYuchcdJzS49nAvRuV0+mWE8r5rxGIC+xADR0X8mcysQehhWi7tirCq4OVHq2uPMwqX86zaPcVBvt3Q0EI5Aq6wwVPZI8S5beYk+YZbXA1d7aZFVoypDlyEnyrGHs\\\\\\\/6a3cQkIYOl6THAyy+nzk58JRYxmrkrgg6FnNPPRVDKzcsA8MssvMHsYA5d1mQ3AzPSvrH0Uitth9nkmHTDkZAGJeiQ8IaR6xqVZdp6mTGzyiryvCb55shoXDgV+wawJ+sda8\\\\\\\/rLb5T8J4QYF8un\\\\\\\/srrr+vTsz\\\\\\\/Ri2wg5uv6Nn+6CmGzUv2abC03h29OlwMttUsDx9zHfKQDXdJMT2HKI1x2ZL0u8sE5hOGUuTX9QOfhFYIK+5eAsRVq\\\\\\\/qxrbZlbHaqnkXd1cKGJUAlDNxI0pL9qtxKLukR0+dhFsWBaIPzd2NlFx+YyJc0Y+sKV8JBP2lHCSbZ8Y8XVzljDavQf3RE0xy2ACWfbgxtdDrhz+j2qK9K2iVmKvt\\\\\\\/eEE7P\\\\\\\/NaG9N83D7GVwVECUI6EZbJpTRr7VDS1H3kokWfEVdwZqHLkTCDj7Q7GZGnRJo5fivxsFHyiLuo2MO5sbhFG2fMF0cvFhv4714Ym41BcTI1LiYIc1XJY\\\\\\\/\\\\\\\/pXlRdpSjYJewtvxI\\\\\\\/7SdUN9wZ\\\\\\\/oj7QXN6tCkUBqwvDkG6cBlBsgfyknRrFttTXfT9bUYED7Df3UKCiaTwtHQUaEZ2ZyLlkK\\\\\\\/9QXqzy1rFDTgJL09vIdAByiSCQbSZhmBqKFDQ9Emp09jctP1r6dATIiPxyX6LTlDVyasvtXFm9ml8Xje0gV0Wampq89ZR7wNKxg1QEROiuvdvX2y+fGdT7dl8l78kbY5FZJKklq7TKub5DxEaraIRV6o7LUMv4Nhv90BgcVkzabfS7EUUT9U8uJd3FaQc0SuwITYdNE7PGt9xjdBGIBrRnlOQ81XIS5BsNG04S1nTu+wdCJgcf3\\\\\\\/lMmZLADKrdo56BtxMxQWI0UuQsd3yPioakSjudZr\\\\\\\/2mr+k4\\\\\\\/nsSqfVuERGqGbLUvaq+j0IeAH5H+FIb6Dc8KngGu7JoRzIMHKBZOe+7yhDAmxF8p\\\\\\\/Qr5P\\\\\\\/xLIifvBZJMVoI9oVbbtCzaIImEEyem6\\\\\\\/XcMba4wScUcXzC+rcWpR9LbknBIAeZMAHsuCldV0Uil1xT5a0JrlhXYnnc86zfx8qF6AWkAzBhk7BEjXteS\\\\\\\/HVufBsUJVRklk3bh0vkb2ji3XYzTUTpziVl00fPSiWKxx\\\\\\\/QqpVEKeGYZfUfSE8kTSwjlDLMSIvY9MNs\\\\\\\/sdaRgvmJUZTB0aNUQAzZw4WUvm79pCwMDgHgBpazoieBtyV\\\\\\\/mS0jh3lcdy8j7vNBdpxXn9P5Czy\\\\\\\/Aofec2ILhSmVUhNDMJonCqn8jLAcbGerSnuf68v0ZgJUYv7kLY\\\\\\\/6KrWgTD+78QINCRGH5VPw0p9A5iuRgg+e+WpVePuUr3vpL2xhTOnG3wdCVhjuW79O4H6tAXdYBNThSlM2dgdvuJnPBRKlNVuDJAa\\\\\\\/xkmCjtZraESdXnbmJyTuInvTA3yI3uDgc5jF+d1PrFrEiDhoSNzLpcuHpSO3V6vxiSDKZDL\\\\\\\/x3FunpZOYB4nvl2b105Pw472FETw5S2qOassMOJfBDMBJZGX5aM6DU3MRWhxC5zXiyARB0uD4h0pCpd1c5hKGVss0QeeK\\\\\\\/2nZgxzxuzOOk2LuN7BWEDfLnBCJIdDFKulHDjz8STUYVGMktU3mYFZh+Q2Z7cfTLikfJB\\\\\\\/7fR+MAIi5Yt2eQW\\\\\\\/IPZdg9d5T1MLn3fyw88ipYgQ3z9jn80cvqhS9ot0MAjMAInbk1ytrYa0FEmCbiTt3KQ0qyXejCTRuTzWbwLK5hI6g3JeMNLwDTLiCBAt4n5TXGvD0X+tLZN7WxWk7AgnapFoCAZzwQXTtqSTgmEVTzV3upF+lb22R\\\\\\\/z17dTgaMvAXXc4UgK2fkxToBKbVGiKCJjRGtZd2U+ZiCPVN4L+noduhWGuT1mvtD4YJcwVWKEiQCUmTuUgxnJhRiDzGZMlGgygawWjEhuO3bM+RyaBsILLWiY7wPqW8R+KYCPmqP3VeW9NVPKRvScy+mPbXD1axSWGRsey6MV13LRywnpbN9QqgwtLrQWNv9psZPCm3i66DiZk+m03agwDGW8znt9TlWEstS2UVTdhgSszK1vGOWcA7IbkgQiG41k1dyI4qLaQXaY0fSJbThdgJkvtv0F+htcIqgXTNazWaCggo6K6pMCeTTeZjjulh1r+ebfwaU5xBQ22YHzva5C1JkNmYyX6DDGn2kI5mQgwOgn2tWEbfcmnxfAcyRjV4jm848\\\\\\\/PS5rLqaTFAInBSWQJY5pMQzh2Wkdm\\\\\\\/UGsvAsqb83FoR2NI8J5hUwHz083jTjt880cyAVn6r\\\\\\\/jqo8XBdcWiTeDSXNVr8Ee9fr\\\\\\\/26k8YEbFY7XMaxyGvGmXVxrs18gGaixgjgxAo2999htvQDTC4iEuSVzI6PkAzOTYFYPRPfJtWgb74oh5FeGo8mrxvmxZSuR\\\\\\\/ulyPdGppOA6i+4ikcCl2fh9y5tcxyhq1oyXC+MrJsKKWMcinmd+73i70HpTtNs6yUMljiVQtmmAqeD1YVcaHcK4+q4pBtt4\\\\\\\/O5IWebuSk5yWu+bz6Vf9VKY744DjkrjnhpaUvbSLg0pF5mllRhSzoyI17RxRELKL72Xvq\\\\\\\/lcG2XXkifqwUGS421DlRHOiODZuujhUeAjsR1jbVDGRaBNJ4am971HC0yw+ylkf\\\\\\\/bjP6s1FjLInAR\\\\\\\/2Cl38CseU8XZs5nq1mJmA1Fwt4FKd2soBNxlBB0R8kl4papiCAh0I5JBoFP1xmpXQOh+FDcF1+v5r+xVSty51bL3KNRcp0Nf8tmfdoCSKjD9oepJdnn673EFY\\\\\\\/\\\\\\\/eBik1EC03XxtGT\\\\\\\/6sBCc6trthP3f4pm5HwJ85yE5bfyZN1WzRJyBdAMWpv6TDamJCDnohPGa7j0N9bRwjtQm62w5z4\\\\\\\/T+jt6AuIFBxPXB4RqZtnFoq56HOGFhx3Lx9TMCpS382EA8KctSAFDAtxaR7DMII8E+Flqq6cqii3am+QtXNuDclcCxDSrCfbSX2XAsfeuSETbeEamFbLApQMXa8R1CmWcTFjcB57wwEKioeerlqaBAOODi2XWGW1b77vYs3uR6JzcvQjgfFClZzWTfIJuzHQxwKUEtM5Pyolvpc3s9sc+eQC7y2hmWrLaRpWSOlo0WvhrfCdUSLsKGik+1P4s+g9bgKT8sA0MzE\\\\\\\/v3\\\\\\\/psRnlef9Pbv\\\\\\\/sdbg\\\\\\\/UxbzLBPXfoq3Uu2wbCcvUuqzs0LDXre025yvehAckAuiHV5ide\\\\\\\/PpcDT\\\\\\\/DcxaEykkhrAmMF65pNOEW0JKcFF2dr7WTbpGL8x0ADpbeH010EXkG5CSp6M2TAyIFpUK3wj2w1dEWXT3zMX2fHXTlXLhX0QV1\\\\\\\/81hY9Adg5KROj7PseWI\\\\\\\/9eh567yROTrgG6HLzJlW\\\\\\\/5BfnQ21bzjE\\\\\\\/8p3qzuecX8JL78S9hpciQ\\\\\\\/dB24oM3ZcxnPLvgUxWy9IAuIKjhYdz47Tb8WPBqpMxQuar1iwgxrR4GX+X5NQndVtoLB012LqjKbYZZWFshfbUCc9svlx4EJTJvxb9HFv7R3uWxf3ntrTCG2xCSHEXHHIsrVZ5MzyrGYkWeA2j1q\\\\\\\/cIA5F7\\\\\\\/CZN\\\\\\\/Tw7ouU7hYCrlF1TKJWDk+Zvfd10sBLa7sqvXdSQHkDhX9zzl+29uu7wXRbJCP0SZh5q7W2+aQ82unpSEox54QZbVAS2tJxcnWsorXJb4bzcC8Jh9Darn1zmWqfBit8zXb08MbHRpBYAKoZyuYFIz+tnPwkgSYg+ewdBncl5\\\\\\\/ZZ0h9N2lrcPZm5H+LSwrziYTTTZT\\\\\\\/CJacHIxGTlcr0Gu1ievgW3Lj9cLIY9A+pQXrGyEgAVt7O9y7FkBbiByOygxPhH7KloTMueP4pcmg1pRpdBk5g+gZquvZnnmBxsRAlLIQHNIH9H8NUMmXc2vnm40uu8+ZiUmjL72l3j4NpkA\\\\\\\/b4ky82NIAWgE6guW7U+R1EVp39GDZhn7KxMMXupuoHZpT\\\\\\\/sJOkbbymB8XtXX8ySE7wLO6VZsVJ7OKET4fArLdJ3\\\\\\\/jJJen\\\\\\\/q+fRN1z1TeEm70XAWvyyd6Neg2B5zTk5dys+1+SPR8mczuqpM2+5ati9fdDH3q35E94IlL\\\\\\\/C9UR++rCnjYyOGXcVR\\\\\\\/dHK1lcW0FgPSznj9AKoCvHHa142Qwn6UqyYdQMoa+fSoEQPIykiZzITosS+l7QFOe8mgqG2jMjGMtWCq6D7Crop6TTbWnVNcPVuaM2Atu+yopuuB2p3n2HNhYt72e\\\\\\\/5RTljGASor3OmPDs3MhQMN\\\\\\\/FFm4ZeUxsWV+NhJZ7SqLelRY6Bd0KAoe\\\\\\\/FAWXsj5QpBcEp7spAQ5z+kVF+kzv1x212YTiF86v\\\\\\\/vsLmB65oSkIg\\\\\\\/uvg8pTyzToJGyvJWvfk5dWu\\\\\\\/7q3DwrBhREvnWQbos5kszGKNIcqJdiPtfD+87k77BlkQ50E1Sjbo225quzFHo6HWYvDnLEb7Sr87j0xhjXLbd4Drm+fkLwDqzjb0P7Z6Uaj\\\\\\\/JXjY7R7nERiGnCwo30eMbSph6eHmCZYDkz5zrMuG9Rh0usAeCcM6bQN2NuWIVn2dqe\\\\\\\/UxZCRAry4zDSp6T1DweZzAmXqzVOEjPobvI8XNCI+FPARe\\\\\\\/ThoRCyOggL3RlmT9EiiZ9jdVhTKdTIpZpHpkSEmzBmhrWzyRZOI7\\\\\\\/9Q80xj9\\\\\\\/nZfHcDcYg6N70tuOWMQsa\\\\\\\/lfN0ScT8PRtUnvdWxRv47YKA\\\\\\\/gM+UP2h5Wqbh4D2SqcPeONgbXmlEKoSJ9ubyV5RkHUP1Li69M0QxAiikbzMtl01zi8zMIg1XpWVXVnQHdKDaOqo0+RhXSmUJPe6cPc8+9lLSy\\\\\\\/K4r9zueSkn\\\\\\\/zvgsNftsC5Yhz9Voy7bS1ddijgw2fbids7YOA42BfIdFIPwIY6CVJKPTTl13f\\\\\\\/yR\\\\\\\/dvseaMVWVKNEQ98TX8od4EScJLPfgJqHjlvuz8svzdGctALXY6Wf\\\\\\\/xjwvvvq3WDPNO4UyAcOQ6irDejyBI8B4OEqcZVkA+s\\\\\\\/1YXfx8IasdmYzd2ow6ZPuG3ra3zWpudNN+IPIa1hImLKNJu9L80Qy2U3xGfdCaG+gDfSdraHjRnDn\\\\\\\/d1MkWDkhNL87UaiOpBKl0Ui\\\\\\\/cOHr4U92dyOlzQmnLVVxvKbTZCpBHdGlso+pIjcIPbNLQ5hEPz8yAaUo\\\\\\\/gkiFaOZE02bfwr5Ydr5rTRrSjSaHOh4vClwD4SjK+DxBtPMUZovrpxN30w2WGolZpTIhHkaRL20TQUTxhu\\\\\\\/6etnI3FDvootKyAE6vt5fgaMWUR+AL06FCP4qcnU60Jx8eGsJXi5VTcw+Uq\\\\\\\/cHTUPgsNoyvU9YHeDgiKBonJNxZZmDanetAqVUkddYkn\\\\\\\/qjA5H+UHljSM5748LG4AayF2AZQafZFRNVE1w2PNANCZ1bqv4OK1UlKDg3VrmBwt9GbcYaRdw1mrHI3c4AQAYufiIpPI70q1ZOqyo8oucUIdlst4dgERKqaBNrCQJFmRAqUJuOBgsBjz3oqU+en0sMSj9\\\\\\\/okeDXtRukBskBj3wc2sw1mleapRF276Lv\\\\\\\/YHe7p\\\\\\\/cyMl0iOfagcjibJ+xZi3OAoEnMRVFnsXnVXpN+W1kTP18ds9r1VjVVzHN9D1CtK6Fq5fn0aNebTUxoDtO6iZe44rQPRi5jM\\\\\\\/zQnKzmL+YxGVCnUcW3mJxwe8IpjGpjuq5G0kJkF4obHeJTUU+m\\\\\\\/R3LxSesuWWyz8n25\\\\\\\/hrgbu9+Zaht28k55qsr1GdAKBvYlTHDwavSLHgUMknGNFR4PzdFNUTIQcUlsfbLTzBAy2wCYjM9oRv0vQPVBfXB7xHXQMNHN2DEho8W\\\\\\\/8S\\\\\\\/td+K1NPAfkHXe84HtfYfG\\\\\\\/KdFos6Nq4Z60dw1nPs75pfkcYecOTGzYO6w8W5Ydpw3ATH\\\\\\\/uVaSC3Xf6s3Vgr6Cgbo9Y3qT\\\\\\\/5tr4vSpb3OZCjvdbqDHhpIlyIqZn9aWnGDk24ofq8kulQCfQyGkEud0K8YaR\\\\\\\/DZeB6v94fYpv76L84veRKFQKWOmFjUAj\\\\\\\/lPZpIefyvAmXLVLkkgK6y4z6NLoR25E78FLAthB9t9LhvxukOfb3pQGwDUoO1YqZSPHxEgkTvvbdyCifwbPNf95XbtBJyiGrtow\\\\\\\/7xe9wNn0bW6GG0NdyFGFN5GkWHPSjE\\\\\\\/O1hoJJb81o48UMHk4b8ePuYBdgvH0\\\\\\\/U2OB3i\\\\\\\/AfxOnuRmZcY3qNxuCSZAueXPkrNOFQU0dUv7Pi81GBnORo6HH7XqVXwFV+3GEwzg9YiWNCInbmZs8BUDuYwtemxw1jriSJEXoHAD1m3N5mEy1yuxMStAucoeg0LTTQOkQHbVwUrF8bznm7+KN6d7+H5xMkM2wTCRDFGrJ8y8WoAg2TmqHfHYlHWFgkdXLT3SW2UV+Q2zotIVw7F3r\\\\\\\/EPniuiSQBNej3BQVITaYMKKxRD0g3WAaVzj18yG\\\\\\\/pllQF3EB8cMb0wwrcaY55Z6EWKJPQrthIdWNyf1Z4VMcxymZcDnuBM4t2fUYAugzpM81uV\\\\\\\/2ojrpba5DVpSNbEWxiJe12WfRzTEVR4lO2YLkP1RoMGvJPLwOes0UJdREFDM50xH8Oh0ktPkpkkwAfgJnEcmQfhqAcihfekHvOh1\\\\\\\/svcBNhepyxQBb5YL7SKydv5vFV0Q+x9RNg6fgj+nyNFfKKgAM7zF\\\\\\\/D1MRkrxRxqDi4H4hvdB1JGGQbMLod3GAX25vQj9t3P\\\\\\\/\\\\\\\/EO+sQ8rkp7Hb2DpRkjGKqLJ1YS+g\\\\\\\/AvWmxHdG5+8vypS8hzsHdl+FXkSOJg7NTEt9xgYVpRWEVlu3X\\\\\\\/uX2JchayMHVez3CiAnq+1hn9DX+BBMYAABk67rXi08amxfGpehNmkwds0IadjrCWjZrlHVDmVbjPgzTOoXmITiFpWq1VHXQisfTCsz3v1uxcMN9fKfb+O0waiSVcYVKPpW\\\\\\\/SMkOlQoWugZlSJDFMuMRr3frOQPjweeprWa0cAfuvRQOcRAFffcdiZNBYM\\\\\\\/Bu8IRB0za3Uzi2fUVRtimQISQi8dIF6iYhX3NKZgOoNQiH4pe5JWZGx5hwANA5gHku7TgQL5nw8LZMYBsM9v2KZITfmWbns2p8M1iFQ7EbP+uWNtGFnDOUYJw28NlZp7k\\\\\\\/+xtzofV0i2gq\\\\\\\/sGmUae3URCIhqee7sQQlZfpqGZU3B3PDf0Uhc5bChHgnjGSHlDenHX4OjByRYu+a160PLExX11XRcRm8\\\\\\\/osh5N35omOgRj+NwKrom+5\\\\\\\/JILHP\\\\\\\/Arjz8dzdlfiDkmy7g\\\\\\\/tRQ8A9po88Wi1Sq3AQKPedjjcmX\\\\\\\/2lBl0cVEFgXb1M99q\\\\\\\/yW\\\\\\\/lx5WPTs1nX80gZWK0mfTE62YzRGeSD\\\\\\\/AioDxWHrgsVY6QzIEx66Ws7h+MNcMII4jAeYFZmDfjPjNZ2u6Zzh3ObshBKH+KGttpYqfDL3yn62+XQ6HZfYgsTvmWh\\\\\\\/RnRocTVCDGGJaqYDjtTtc6W9UIuyZgKsherBXE3wxlOQH1paGtcga5hg61yZqC2qJHFTuVnS0zIjezsnPsA34+0Tzyv48chMyoUxKG4PxiC4YA+dnz+AoukM8cSB+xbzapfUjDFJT7RvLJRKmleLU1qTOCmdL15EJq+8aT6HJ1s3H76BR92BMdvJaJYplpT5g9IDMTXgNSNIhnRONNaNKtpyHT0vXgXiFQM4Z4N\\\\\\\/F7JvHjPpEV\\\\\\\/FWRSVndl0I0SKqyVFTaquzZG2TEuN2AIc4qn8ZeKNIlMFaoDJ7WFjzUp8I4oBNBPo8Nl9Z+FEx2UMnEg5YseTAEMWrLi70EfRln80VOhDQJklg\\\\\\\/lQEZ7StK1Cm58h8gTBuwX4xu8mR96vcpUMgcZU2grgZgSR57EcZRmcWaEyF+k2stJJcH+SgCUadRPKGSx4mHgIRNCSPS3ZhgC2gbUTObRIYLTmCThA007487eljIl5CThvqZtcyPySVkNdn+\\\\\\\/FI5PyJCXYWJLSUfvHpgG8g8s0cIOkl8xKD57xInhXxhav6AIJm\\\\\\\/SeKe3ABCVU84sTkGrTIRq\\\\\\\/VD90brcpehAfPCgY\\\\\\\/VMhbAiLWJJQAuZat57RLO6D692LMfDT0jXTaGagexBOCH6cQM\\\\\\\/qolv0uJxZUoLQc88OY6X5HfDeS0goJKo1XR6uHMC8i9JJfYIHc28Es7QNicFrtVKyTNQEVR3Yj6rei1psQeyIiVeVKPRkFbxHo3JHtyWOtu0nKtazUoO1ct7Pom+8Ul54YVCHAU8YxbWIyN8sN\\\\\\\/6JnHXkuSBcWzINsQJ+nJgieu+idD\\\\\\\/u3PSmO2gwYMcRS9ns8zPtH\\\\\\\/PHhdLL87SMThI2iQeFrJND5H5hdd2ynd42b8Lr\\\\\\\/2wdUBvTmnkCtH4vRnZe0VzcZBv27VFVAFFSSJ6Dk5doUacnUgoq1NfNgBV\\\\\\\/K+uITqQ8t7bThKVGmnBG+zivBFSFY6IX5FG1B82b7ZR7wzt+oOmLNK8a4j7py0xXVNVbcc7ZdjX62Duy8bxtGjq\\\\\\\/epafNUPSbg6Dild48qjsUdxbJnwA77DDXmEwoFcsrLNVLaIGYQZUXxo+z\\\\\\\/VeGUrti4ID10n8AZ6rqS+k+ffJAMlytWNTWxpJYIsYl6\\\\\\\/pnWSW3R6JfrULyAv3G3f4QrzocE66I2n8HIzz1F+Nan30qK3RKo82jL3DxgG4hafmub+RXVD6bYNLSKZ8W1IY1WbPpUDuAeDsDk7Pp30NDb4Ttrh6go4JIJu5MoSqWzx5G3XSNvm7txSOIaPQITCRlxPZ514KAJUqp6k7\\\\\\\/HfZ85cnKWAfEeCxdsZtTpfJkj0+st0wSnk1UCTiF71qzQfYvY59LWmnN+09KxcwKLBnO5saEQydlM+jqTCmoSGl\\\\\\\/6V8PO53yej1CFs0v1HpnQX49zfy47TKyidWbE5ss9YBWi3U+ARGkU4n8yEB\\\\\\\/j3lan6zI9lk1gmalmY7MwmAtRhuwtbrJaEE9pgL1ZThehEbOKs+Um7uiTRX+gaVioWuQXdlguU1UtNID6VdWwgTSBeNTd14OBrPMK8bL5WFvUCsJ1PEerN4Sr9C\\\\\\\/GhIeFP4p0q74QifYNCDuDddZQADsbGy1vV+HPEAPV6\\\\\\\/dfLcJuw1SjecrjfUfK3wD9m+lcOObhDFw08rhIj6OpRk77QTYnYsq5IkSl125Ebbm4opIfVSLhlz7D4GZ1tVZn14b7AnfWffMLWiEmldbWeioqxZb09N9uTLV88SViUBTRMMVAFVZiG6bgYgwTWWJ8MFfafUUgoOWvn9J453orUn9D0Igi4OKWNtfs\\\\\\\/XjNzHQJ7mwDmeS1vJ7GOHnKyk7yKlBm7YI0YAXjsjAQjRGQAmFLengX30Gil8WCvXtpxI7AJ6U0ydBWCfoxoNzlj4vj8PKaNtarU\\\\\\\/Pj0u8uOYpzckZBhehadlQNqO+Lg2TOXWK9ZvgJvdd0DBOP\\\\\\\/1NP5Mm8NaFxqy6Htdz0qmoJoUo6OPQ9guASr3c9r6RquVBxc4pCokoTxLWPMeWukxc5A1W0\\\\\\\/ECmguHMyQKmNfCNGK6dPnxCK8Wl+vyhnJpKE6v3x4WV\\\\\\\/a5xfUetJZAG\\\\\\\/6P3HqH8wSk2pCSxD0+I3GDPeAJGJOkIvPSZYXRUwj9p\\\\\\\/7x6pfaK033A\\\\\\\/bHxHvSgeZAwmxBL4lb+wcRPwiIixQL+rafUVjcBpjOFOhQI3Eem\\\\\\\/9BBpEqh+cleW9kRxkmn9Zy28\\\\\\\/5rpoaV3ESdZyWxne5EMbD+5tjkDwXnhGjdMZwzOvKpwQiR41kBeof5KYeyGwaY\\\\\\\/ifcWjbrZFa+JaaEGYD10GnWyCLElFezRd\\\\\\\/R3T7M1rojxjEcsrn6kBkxRYxeRFZ+4FjifCVe9TgtfiVCwg5UGpyEcVmyP6wIwzeOnuWgWMEPPop82U17nF50VURcXEpIfb2i6CkTiiHyauZZPQHAF4xC4ytio0z4fPGZ2A7UYKWME1Cll3oVYoRWLl\\\\\\\/ir2Gy\\\\\\\/baTm3DNNfSw7hQGdQljy\\\\\\\/J3Ik2nSa7VgaLJWCKD6V3MgvopIi0S3lMo+nbaw8VgF1PDcob2eEb2CSMpCYpZLSZJBRtYdHV11Dlb\\\\\\\/HS6OqUjvydCUmCU\\\\\\\/E\\\\\\\/O0\\\\\\\/334dlqQNfYRJOV5YIxRI5G\\\\\\\/fML6qNZtTvDBPO1dBNOQ4Fb8Tlojh9ovyXL8onNOB7P9lzxChclvjtFzK3WhIVsIPFDN4zAvn3Nn36jSJkKzR+0M0Lh607mG8khF0mjNjBE4imvuQdrRWR3ZSzXa1YLxhoAsFBM5tu1RXk46RZW+umin6png0NcledhepXT1p5OLChnTn8dWC05UAtfmC1qd88mu40OLyH07UQW+YL5jqa49ttBYayNxwaDvXe2yLc6fzwiiZbIFcKe0pDFHuUm60Z+RGdD9SVVc\\\\\\\/zknu3WTbn2lKZHVDEyFJDD1vK6VitLtxmcnbfUZu+5cRF2YvvML4dJpB\\\\\\\/fLmE1OwocPP\\\\\\\/w+sorhMK7ow\\\\\\\/KBx208AR3b7Dv5cqnQvYSAqxUa\\\\\\\/ephHArivDzlihJEZlzkmvP\\\\\\\/K2FUWXUuEJIGR3bUdHmiNsKwCDu3NUZjAp8CQQZ9IRcrQaS7VHGgZ5krVdsNLI3X44ymWrmu76KdrKukP5U4dSiYw6b2RB12TLXLKQeiRi5m4gnI20Tj2RPxj9nP8KhSVWT4qGgjvxJ\\\\\\\/bT8MmXralP2Eu4o13xFQe7Se1FBmCE0k+dRS2J5vCYqXmq9NWFVf7NTPrCmaJ4SA6lSYQRdHG3eNRfJnrt56O8mUSaIGOV6BXJpDPbYuRVD0h5V1YOHpYzTkjN8BNt5Eouao5+hlt5CP0xgGZtcJjNrR2tGY6e+fV42BviBltw1VQOQ15JAmqBeUtRv0ubBvoy9IJp7XXFEY1nPfFisvH1FEvRvN7iHC5R0Mti\\\\\\\/Hy3ErmX46mKK7U3CfzLWtlkt\\\\\\\/rOoFoO0sWR6duAlfUAjgimmqiWTauPvNmwv7KGjDIjCEDo8RyoJF2IjSeinxNi87pYetbE0+L\\\\\\\/Zw76tQzUAwj7R4nL3ixcQ6D9ZSEFwulz3QwtVOQ2FujjY9RMq3kIvm7uTDNPOaCUa+HObvbKkpxdlZyVuHKZIsBGn+T83bVoS6EU7SvWA0\\\\\\\/KLkhSyRilJ9N8FDGOfEL076Ux3ud4XVHiDau1rABmXyJn9MuaZ7TPNbX2RBn5\\\\\\\/UIYpIY0310O5gmD7VBS+x6Zzup9U5nAHEJAk0Ku62fmGOm6NwC23MHeICmnth0yzZbR+WrEXxMXBJt3mpvdwGozBm5Sbdb5rRdoS74st3IKgdOFJsujll5z6X9L5PU3FdUFrTzdcxIxF06ZkWtdeLG6fULPZfpfUIXMtr2Dc6Ovo\\\\\\\/JyOfBnByiMqufxUvDLGzsPeSLB5ZMPhXoV20dV66vzhNfwKzfEw7e\\\\\\\/oY0\\\\\\\/NMyDifmEZehQ6JcGtYvSW3yM7W+iRfLqs7CvIL5+CuPnqUv4ok9Owfms3bAfaBFBFh10rnliaohYn+\\\\\\\/wgwrB11gq7a68rqkEHcZ7CN\\\\\\\/jiWJN0K9tNYMp8zGNMRwDb2U6U8UtldHQOIBBb9+o4rQn0jC+OdFZbU4Ny\\\\\\\/SXZhvcJxHEpY9YbQ90aiynUnSFTCdnmWl2YFr8nwf5Wi\\\\\\\/TYOhPu2hGrAJY2MC4CnEYDaMPsq1O1O8hU8BxT1ITfG1+avkg19YlgOpIt\\\\\\\/+q7ONAAUzhJpPLeVAAGTCdaWp\\\\\\\/kDBXEMsuCLwN71h0SsLwntmrEipHL+2FazgLpbR8gVRidzymew0D2UtHcknWRrocAgODVKY2lWgRZxAneNqKfMvhbJJEvIUPe9JSSjkdAQO\\\\\\\/8zpaeuFkhhLVQFe3ZphagBQI48OfWchu\\\\\\\/g4GR0HgzxoO+tq7wXNXfKQWLErS0Q4aHByKlagzmxjIWjHXMwnX8WNh04IsWfM1dE\\\\\\\/FRDOkGdXuSu3EjStTYSC3g4oMpf55JAlgWRh7uQY+wzVxFQB6VW6wS65ImIB5PVsdYpeyn+F5RhyCCFn6PocsFFgCwra8pSUK1eV\\\\\\\/qFwTu1ruDp6J9tcF\\\\\\\/O1\\\\\\\/qbnl4F+GgXLtKH\\\\\\\/R6F38zfj7KIvIbKzFyTqv4vA1LTJGCMdyzrZ\\\\\\\/VxDgtREjKDGCTyqEguJi3LGIVogZ1+\\\\\\\/ydNBbs7Leq8geiX5L5l280a7kjN5ciZ71tfLhyfYNy0PmeYLjbycgxrfDAjuf7up+seuTG2NTRMmG0XaP9W2q8kpy\\\\\\\/Ytwuo4gbITImCwHpy6h32PQeCS0QVi5Vj3SuG76eicjFmoEDwNkPmPZ1fldLuLHVYn0xcmqjCh5qk7HmC2suQhAjmCids9fp6ty0U292qwHlt9XdWJ5kLSIsUhy4gVFloChTtMioCZE1+57JqjJan6zl9I5Zxy6R0uUaL56xx9yaO1aEyOutW3AKWWXaJOT5+2htZPZQqhHJery62HPi5eeQw0yZuv\\\\\\\/es8N+cXbrimJ1SXmZrC7Pire22\\\\\\\/B3zNVrL3OBl9Tomo+EaUeayzXxIbSZXzJ+UCMBqgAg6EaTGv0j0eQnBSqOKwtF92UCk2punUBbYeQ\\\\\\\/RGQAFtm3nDbcENnQEbE05yVBAjfoCjG1j7D7DFtBfYsR4+tEyhdRqrH23Bpi+2XzsZAlKY1xR\\\\\\\/ALQc8H49jcV2CpOVLezIuMk68WhQ0aJQgvycQ0f6rdWvG1xM1bA+Bu\\\\\\\/ZFSLt3HxlUBIxkqzkzGmEuYOsjUc8qfZyJa+I4nnZdB9niZwSey6imuVhwMwoaaK9XNQopoOKEqia7U8\\\\\\\/F0N6gY91uEGhNwMTk4LL40Wt0lTNWcRJkN0n5+LHRMvkH8R9Mw1WSbqPDQRId5upyvqAhB1vlEEmxxgHAX2\\\\\\\/JflHl5C6s5+Kn\\\\\\\/9gaMiJStI0q\\\\\\\/vaXA825G16+Rsi36UYM5pcoce+6nsPO5jQwIdO6ZP1KvqlJdrDwCf1Nxb3gbwKIKI0lKdTPTBALA5dtK3ToVX+w2NIPp6galwz9J12prBV3SLbYIrPshJeCnXui04EogP+GbXblvjWGtZxKN016IvCDbgdtSdEjo1o62eGwwFoCPjeuUEI\\\\\\\/dlUHgieseHwEIZ6bL4R9uGnn6cEhaj33hd4+E9sejyOOQaJW5NEI5ls\\\\\\\/TSOQU2UaRJUsxwGpET6SUOsWSyIS\\\\\\\/amFQV8finzoXbd10gI8WXGC9ReQZolFCy1IFqokUb0m3j\\\\\\\/llGIECwsODIEQcI+p3xbDH+k7274s7gFg0xKK4UlOLTocKleEq8vT5K6SPe6jJEiQrz7zSaXn7yompaEQQ+AZIfr72Mla\\\\\\\/TxfbcUL8d9OnZTjBnvyGln0Xk+IQRese6E015JhGXXuregbFAVuVVEZqjEdyqz83TqULSx95VBNUlvg==\\\u0022,\\\u0022iv\\\u0022:\\\u00229v8WNTWvahuwItw3+K+PZQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022abbdce14d2eec7a88d9227393ec5565c6807a4557397ddb158fc3e1ac044b231\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RL26S+6AkYJVX5HIM5l47O+BLaReNxa86czNy10J5rqM3Nh\\\\\\\/fs3D\\\\\\\/tIG7zjPTniNOzsUwlS\\\\\\\/gFpzYUrNv4LjUDm82BJ3cD2KMFVsaEcukbxBvMC1gWfbw6UZW80CA98Ojvau0b9y5kbu4zc3CwaqaJKdELhYJN4OQaLaeQZPuSZkD\\\\\\\/63L96D09eHSrt6LIC7BBXWu6qOjD8v1jM92rt+jpPp6KaZDJ3nZ4YpigEYQDke3UHl6G\\\\\\\/TPKWc6g+MhFZTdjN20clZaYPVwetiU+6raQPKxVO6PB7Nrb\\\\\\\/1Byo6mtTiGkk0+hFeNl+HZ8hONPBicdktNhpGgjfXLV56Dj3qWWYEZGoKvZAD9Aj2f\\\\\\\/toKJD1RckoacmWxLKmVtZGmY1OVgKvNYZ3b6cKBMQMney7dbBAwbXlia\\\\\\\/aNHG9sQ1PkEuTTHBrV8Qi\\\\\\\/+Hq6sb1dfNoFLiZU+vUtNAO0kdn\\\\\\\/io3ioI2sZ27R08QrYU1GZuuV9BL4KQCQ23WE7UjH6TtNfvmj3lHmLfOp5pzZKfwkL745u02jro\\\\\\\/SGef9k1GTlaKzN04l1r2V3bDOxMu4FyAXEJJQSGl\\\\\\\/q2IPCCTRB30nn9hEz1FuKP0SzuoYABXJ86DM+OvopFCJ6yOHwjIMpzuQsyyoKSd1NZca75BIxyhyDRDhF8XTNUWsrrjc8JbZpLZX7DJCKV3U+JPbiT1NyprtrEt7jrxNd3p5L3y71yrz8NqjvJq6HoZhk7D4rt2FZlqgw1sGilGo9wELSDruZjOZbLeuasjaiMcgqG9eL+Jvdctla2LsZigUph0UKCknEh4PWdoau05QlnMHZjjKL4TCcSLs58pWcDE3niOwlKJD0sJkzaJFFwqJsFM0Qb35ZNVLDUbEiIHpCV1s5EB1fX0jlG6QZ0mRXXJylO1EO8h+x+Dun\\\\\\\/wbw4q\\\\\\\/CgI6lc+oHrXnX8YxUtWlsH35oBOgqmiSGaOvKZoFOncCd3y4KnMpjoOO6ZLrrnypdtwNN+mMv4xJLFJHrmW82aUOrqt3a3f1DFf628+gZRcq1SgBLRAk1EqlEqWaPevlHZTLsYkfdha4Snv7GbrX8AWztyT19sP+7xuefWLMDgjENX5Dh6jX73pbsLhrGcS++fPXBIhy\\\\\\\/d8F2XYy2NxGfZECd2+WRz3Bh\\\\\\\/6iiqE1khjwPI8g4hHVwNp+N9rHdvfhF7rVnkytb8jm1H5nTwWQJXjDLYziZrwLiPhDHXzokejkaXe+rzy3+sG\\\\\\\/vU9vdmKOwhGlx99XM2MUMWuYXOYn0pF8OyPbLYPNh1xcsHr4wftMAtyUSnTRWZCtpuAZlED6J9Fw5n0GY8kioz+z6VXD9tmxpSrTWd8O0c45TRmirkBlce28\\\\\\\/D0ln0c\\\\\\\/BzNJYDZDMG46CI6l+XQfJ\\\\\\\/r9AbwtEsvryqFf70nq3YHG171L2pyquwT\\\\\\\/l4ju+oTxpROR044nW5f7EX8pvbOYdsytwvgZlU0gozNmX64+l52bmmpof4HNnS72PaDz50NkXq7Efcpi2flsYmuRstZfj8RbnnWhPPvZuy2Ro58wRqPMEKtGIM8nGKUJkGWdLEm0DASUGukKBLoiSYLO8Ph2ohBcWsC0o49sWppxmFtzenLIN6SZTBE6jHA22xPOPiE8CjaIHMiEF+GfN1Oe+eVHKc6vqGKXqh2njovWZEPxYnaXZUtT7cfkTwM47z4pjeMcXeI5+5AIUBTzwuf3kzx+JSj9HdzxgsIw96oz24wZbqNdobE\\\\\\\/a8HOWKsXX5RjnmpILYeDNmSFbiuhSGk\\\\\\\/EfqUimc0IZ91GFQxY169TmVAB31yP4SSkNV5yK6fuS01vG7Uya0Gl8kCnnvOAx7\\\\\\\/7s2fIwK81oKMDXKeg51VvjkkKs5Gs1RQHYaMOYmkTJwChDBMV3ZfBqFl4sk5A58Zz2lsJWJ0HiJdzD+7pTnOzlkImd\\\\\\\/vMXXyG85hN8tO54N5LEeYc2uwyEBTcrp\\\\\\\/pW2CmDNnARGPNfi5sq7MgtEyGI8COWwvw9F0+TfOcSy0npaCGW8ig==\\\u0022,\\\u0022iv\\\u0022:\\\u0022X8UVEvlk7iBOeTKouLGADg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022be3fea6abd583be638e3e57b9b9009ac6865f39c2788c5c0acaae998a3bdd186\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493110 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1557" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QMoScCb1kzD1e3n8IZyOviI_wnWSF5dPKa5Pz-Kd6uo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1557" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gPaDtP7k3HDsmnMH-uKLmUcjSYRv9i0jg8p7b3rwhyM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1545" title="New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1571" title="Cleaning Genotype Data from Diversity Outbred Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493110" id="node493110" data-pisa="ggg;9/5/1557" data-pisa-master="ggg;g3.118.200994" data-apath="/ggg/9/5/1557.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2497-3136" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frank Technow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1557-1569;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200994 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493110" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Maize Product Development/Systems and Innovation for Breeding and Seed Products, DuPont Pioneer, Tavistock/Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2497-3136" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frank Technow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frank.technow@pioneer.com" class="" data-icon-position="" data-hide-link-title="0">frank.technow@pioneer.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1557" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493110" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493110/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1557.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493110" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493110/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1557.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493110" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493110/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1557.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493110" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493110/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Developing training sets for genomic prediction in hybrid crops requires producing hybrid seed for a large number of entries. In autogamous crop species (<em>e.g.</em>, wheat, rice, rapeseed, cotton) this requires elaborate hybridization systems to prevent self-pollination and presents a significant impediment to the implementation of hybrid breeding in general and genomic selection in particular. An alternative to F1 hybrids are bulks of F2 seed from selfed F1 plants (F1:2). Seed production for F1:2 bulks requires no hybridization system because the number of F1 plants needed for producing enough F1:2 seed for multi-environment testing can be generated by hand-pollination. This study evaluated the suitability of F1:2 bulks for use in training sets for genomic prediction of F1 level general combining ability and hybrid performance, under different degrees of divergence between heterotic groups and modes of gene action, using quantitative genetic theory and simulation of a genomic prediction experiment. The simulation, backed by theory, showed that F1:2 training sets are expected to have a lower prediction accuracy relative to F1 training sets, particularly when heterotic groups have strongly diverged. The accuracy penalty, however, was only modest and mostly because of a lower heritability, rather than because of a difference in F1 and F1:2 genetic values. It is concluded that resorting to F1:2 bulks is, in theory at least, a promising approach to remove the significant complication of a hybridization system from the breeding process.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/f2-bulks" class="hw-term hw-article-keyword hw-article-keyword-f2-bulks" rel="nofollow">F2 bulks</a></li><li class="kwd"><a href="/keyword/hybridization-systems" class="hw-term hw-article-keyword hw-article-keyword-hybridization-systems" rel="nofollow">hybridization systems</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/hybrid-breeding" class="hw-term hw-article-keyword hw-article-keyword-hybrid-breeding" rel="nofollow">hybrid breeding</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Since the pioneering work of <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Shull (1908)</a>, hybrid breeding made a significant contribution to increased productivity of globally important field (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Duvick 1999</a>) and horticultural (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">da Silva Dias 2010</a>) crops. The development of hybrid varieties rests on the ability to evaluate general combining ability (GCA) of parental inbred lines in earlier stages of the breeding cycle, as well as specific combining ability (SCA), or hybrid performance in general, of particular combinations in stages leading up to release and commercialization (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Sprague and Tatum 1942</a>).</p><p id="p-4">Genomic prediction methodology (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Meuwissen <em>et al.</em> 2001</a>) has successfully been applied to prediction of GCA (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Albrecht <em>et al.</em> 2011</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Würschum <em>et al.</em> 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Jan <em>et al.</em> 2016</a>) and hybrid performance (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Technow <em>et al.</em> 2014a</a>). This has greatly increased the scale, speed and accuracy of breeding operations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cooper <em>et al.</em> 2014</a>) and promises an increased rate of genetic gain (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Heffner <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gaynor <em>et al.</em> 2017</a>). Building accurate genomic prediction models requires training data sets of hundreds or even thousands of phenotyped and genotyped individuals (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Jannink <em>et al.</em> 2010</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lorenz 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hickey <em>et al.</em> 2014</a>) which can put enormous strain on resources. Promising approaches for increasing the efficiency and throughput of phenotyping (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Araus and Cairns 2014</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Tardieu <em>et al.</em> 2017</a>) and genotyping (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Poland and Rife 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gorjanc <em>et al.</em> 2017</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Technow and Gerke 2017</a>) are currently being developed.</p><p id="p-5">However, producing high-quality F1 testcross seed in sufficient quantities for multi-environment field trials can be an enormous challenge when done for hundreds or even thousands of training individuals. This is the case particularly for autogamous species with hermaphrodite flowers, a group that includes important field crops such as wheat (<em>Triticum aestivum</em>), rapeseed (<em>Brassica napus</em>), cotton (<em>Gossypium hirsutum</em>), sugar beet (<em>Beta vulgaris</em>), rice (<em>Oryza sativa</em>), and sunflower (<em>Helianthus annuus</em>), as well as many horticultural species. Here, hybrid seed production requires elaborate hybridization systems (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kempe and Gils 2011</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Whitford <em>et al.</em> 2013</a>) that add considerable complexity and cost to the breeding process (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">da Silva Dias 2010</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Longin <em>et al.</em> 2015</a>). For example, cytoplasmic male sterility (CMS), one of the most widely used hybridization systems (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Janick 1998</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Kempe and Gils 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bohra <em>et al.</em> 2016</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kim and Zhang 2018</a>), requires (1) development of sterile versions of inbreds (“A-lines”) from the female heterotic pool by backcrossing them into a sterile cytoplasm source, (2) retaining of fertile versions of the same inbreds for seed multiplication (“B-lines”) and (3) introgression of effective fertility restoration genes into the male inbred lines. The complexity and costs arising from applying this and similar systems to hundreds or thousands of individuals for training set development might be prohibitive, particularly when those individuals were chosen with regard to maximum informativeness for model building (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Rincent <em>et al.</em> 2012</a>) but have no breeding purpose as selection candidates otherwise.</p><p id="p-6">One possible solution for making large scale production of testcross hybrid seed more feasible is to field test bulks of selfed F1 seed (F1:2), instead of the F1 hybrids directly. There are examples of commercial use of F2 hybrid seed in cases where F1 hybrid seed production is not economically feasible (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Janick 1998</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Wu <em>et al.</em> 2004</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bosland 2005</a>). This option has also been considered for application of classical mating designs such as Designs I and II (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Comstock and Robinson 1948</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">1952</a>) to autogamous species (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Stuber 1970</a>). The idea is to test bulked F2 seed from selfed F1 plants obtained by hand pollination (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In this system, the seed is produced from vigorous F1 plants instead of inbred lines and according to the natural mode of fertilization (at least for autogamous species). Only very few F1 plants would therefore be required for producing sufficient seed quantities for multi-environment testing and those could easily be produced by hand-pollination, without requiring any form of pollination control such as CMS. Using F1:2 bulks instead of F1 hybrids could therefore be a cost efficient option for producing testcross seed for the hundreds or thousands of individuals comprising training sets in genomic prediction. Under non-additive gene action, however, F1 and F1:2 performance are expected to differ. Using predictions obtained from F1:2 based training sets could then negatively impact genetic gain at the F1 level, which remains the selection target. The objective of this study is therefore to evaluate the prospects of using F1:2 bulks for genomic prediction of F1 hybrid and GCA performance with quantitative genetic theory and stochastic simulation of a genomic prediction experiment.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic visualization of creation of F1:2 bulks of F1 interpopulation hybrids. The individual steps are (1) making a F1 cross between a male and a female inbred line by hand pollination (2) selfing the resulting F1 plant (3) creating F1:2 bulks by harvesting the seed from each selfed F1 plant and bulking it." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491559244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic visualization of creation of F1:2 bulks of F1 interpopulation hybrids. The individual steps are (1) making a F1 cross between a male and a female inbred line by hand pollination (2) selfing the resulting F1 plant (3) creating F1:2 bulks by harvesting the seed from each selfed F1 plant and bulking it.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1557/F1.medium.gif" width="440" height="374"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1557/F1.medium.gif" width="440" height="374"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492978" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Schematic visualization of creation of F1:2 bulks of F1 interpopulation hybrids. The individual steps are (1) making a F1 cross between a male and a female inbred line by hand pollination (2) selfing the resulting F1 plant (3) creating F1:2 bulks by harvesting the seed from each selfed F1 plant and bulking it.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Theory</h3><p id="p-8">Assume a heterotic pattern formed by two populations (“heterotic groups”), arbitrarily labeled “male” (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-1.gif"/></noscript></span></span>) and “female” (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-2.gif"/></noscript></span></span>). The members of both populations are fully homozygous inbred lines, either produced as doubled haploids (DH), the method of choice in many crop species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dwivedi <em>et al.</em> 2015</a>), or by repeated selfing to a degree that residual heterozygosity is negligible. Consider further two biallelic, independent loci in linkage equilibrium, with alleles <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-5.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-6.gif"/></noscript></span></span>, respectively. A superscript <em>m</em> or <em>f</em> will be used to indicate the origin of the allele (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-7.gif"/></noscript></span></span> when the allele <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-8.gif"/></noscript></span></span> originates from <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-9.gif"/></noscript></span></span>). The alleles are assumed identical in state (<em>i.e.</em>, biological function) in both populations (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-10.gif"/></noscript></span></span> is identical in state with <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-11.gif"/></noscript></span></span>). Let <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-14.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-15.gif"/></noscript></span></span> denote the frequencies of the <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-17.gif"/></noscript></span></span> alleles in <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-18.gif"/></noscript></span></span> and of the <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-19.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-20.gif"/></noscript></span></span> alleles in <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-21.gif"/></noscript></span></span>, respectively. The frequencies of the alternate alleles are, for example, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-22.gif"/></noscript></span></span>. Allele frequencies might differ in <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-24.gif"/></noscript></span></span>.</p><p id="p-9">When the populations <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-25.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-26.gif"/></noscript></span></span> are intermated strictly at random, the resulting set of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-27.gif"/></noscript></span></span> F1 hybrids forms a “gene-orthogonal population” (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Schnell 1965</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Melchinger <em>et al.</em> 2005</a>). With two biallelic loci and alleles defined according to their origin, there are 16 distinct genotypes, indexed as <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-28.gif"/></noscript></span></span>. For notational simplicity this will be shortened to <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-29.gif"/></noscript></span></span>. In a gene-orthogonal population the genotype frequencies of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-30.gif"/></noscript></span></span> follow from the products of the allele frequencies in <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-32.gif"/></noscript></span></span>, <em>e.g.</em>, the frequency of <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-33.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-34.gif"/></noscript></span></span>. These frequencies will be denoted by <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-35.gif"/></noscript></span></span>.</p><p id="p-10">Selfing each member of <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-36.gif"/></noscript></span></span> and “bulking” the progeny results in a set of F1:2 bulks denoted by <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-37.gif"/></noscript></span></span> (<em>i.e.</em>, the F2 seed from each F1 is bulked separately for each member of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-38.gif"/></noscript></span></span>). With alleles defined according to origin, each individual <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-39.gif"/></noscript></span></span> bulk comprises 16 distinct genotypes, each with genotype frequency of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-40.gif"/></noscript></span></span> when assuming absence of segregation distortion. The frequencies of the different <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-41.gif"/></noscript></span></span> bulks themselves are also <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-42.gif"/></noscript></span></span>.</p><p id="p-11">Consider a three by three matrix U with elements <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-43.gif"/></noscript></span></span> equal to the genotypic value of the two-locus genotypes formed by the xth genotype of the ’B’ locus and the yth at the ’C’ locus. The homozygous <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-44.gif"/></noscript></span></span> genotype thereby corresponds to row <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-45.gif"/></noscript></span></span>, the heterozygous <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-46.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-47.gif"/></noscript></span></span> genotype to <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-48.gif"/></noscript></span></span> and the alternate homozygous to <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-49.gif"/></noscript></span></span>, correspondingly for the <em>C</em> locus (see below for examples). Because alleles are assumed to be identical in state in both populations, it is not necessary to distinguish them by origin for the purpose of describing possible genotypic values. Let <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-50.gif"/></noscript></span></span> denote the genotypic values of the members of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-51.gif"/></noscript></span></span>. The rows <em>x</em> and columns <em>y</em> of U corresponding to elements in <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-52.gif"/></noscript></span></span> are <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-53.gif"/></noscript></span></span> and column <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-54.gif"/></noscript></span></span>.</p><p id="p-12">Similarly, let <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-55.gif"/></noscript></span></span> denote the average genotypic values of the F1:2 bulks <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-56.gif"/></noscript></span></span>. Those can also be obtained from U as the average genotypic value of the possible genotypes in the bulk, weighted by their frequencies, when assuming absence of segregation distortion. For this purpose, the origin of the allele is again not distinguished. For example, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-57.gif"/></noscript></span></span> would equal <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-58.gif"/></noscript></span></span>.</p><p id="p-13">The correlation between <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-59.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-60.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-61.gif"/></noscript></span></span>), a measure for the similarity between both, was calculated according to the standard statistical definition (<em>e.g.</em>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Mood <em>et al.</em> 1973</a>), with the mean of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-62.gif"/></noscript></span></span> being<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-63.gif"/></noscript></span></span>and the variance<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-64.gif"/></noscript></span></span>and with <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-65.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-66.gif"/></noscript></span></span> obtained analogously. Thus,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-67.gif"/></noscript></span></span>These and all other equations developed here are implemented in a worksheet that is available as supplemental material (File S1).</p><p id="p-14">The GCA of a male inbred with genotype <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-68.gif"/></noscript></span></span> was evaluated as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-69.gif"/></noscript></span></span>with the corresponding values for the other population and those for the F1:2 bulks (<span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-70.gif"/></noscript></span></span>) being defined accordingly. The correlation between <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-71.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-72.gif"/></noscript></span></span> was used to assess the similarity of GCA effects evaluated in F1 hybrids and F1:2 bulks and will be denoted as <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-73.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-74.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-76.gif"/></noscript></span></span>, respectively. These correlations were obtained analogously to <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-77.gif"/></noscript></span></span> and according to their standard statistical definition (<em>e.g.</em>, <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Mood <em>et al.</em> 1973</a>). The variances of <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-78.gif"/></noscript></span></span> etc., which are required for computing the correlations, will be denoted as <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-79.gif"/></noscript></span></span> etc. Because populations <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-81.gif"/></noscript></span></span> are intermated at random, the male and female effects are uncorrelated with each other. The variance of SCA effects can then be calculated as <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-82.gif"/></noscript></span></span>, similarly for <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-83.gif"/></noscript></span></span>.</p><p id="p-15">The quantities <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-84.gif"/></noscript></span></span>, etc., are statistical effects that depend on the allele frequencies in both populations. Thus, two members of <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-85.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-86.gif"/></noscript></span></span> with identical genotypes will have different GCA values (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-87.gif"/></noscript></span></span>), unless both populations have identical allele frequencies, even though the alleles are assumed to be identical in state in both populations (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Schnell 1965</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Stuber and Cockerham 1966</a>). Therefore also <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-88.gif"/></noscript></span></span> in general. For simplicity, however, the average across both, denoted as <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-89.gif"/></noscript></span></span> will be used, where appropriate.</p><div id="sec-3" class="subsection"><h4>Models of genetic architecture and population structure:</h4><p id="p-16">Several models of gene action will be considered, all encoded through U. The first involves only additive and dominant gene action (“dominance model”), here U is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-90.gif"/></noscript></span></span>where homozygous effects for the B and C locus, <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-91.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-92.gif"/></noscript></span></span>, and heterozygous effects <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-93.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-94.gif"/></noscript></span></span> are defined according to <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Falconer and Mackay (1996)</a>. A purely “additive model” follows by setting <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-95.gif"/></noscript></span></span>. Here, however, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-96.gif"/></noscript></span></span> in the absence of segregation distortion and so <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-97.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-98.gif"/></noscript></span></span> are equal to one. The same is the case for the “additive by additive” epistatic model without dominance or epistatic interactions involving dominance (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hill <em>et al.</em> 2008</a>):<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-99.gif"/></noscript></span></span>where <em>z</em> is an arbitrary constant. The “duplicate factor model” involving additive and dominant gene action as well as all forms of epistatic interactions (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Hill <em>et al.</em> 2008</a>) is<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-100.gif"/></noscript></span></span>and the “complementary model”, also involving all forms of gene action (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Hill <em>et al.</em> 2008</a>), is<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-101.gif"/></noscript></span></span>Biological interpretations and examples for the latter two models are given by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Holland (2001)</a>.</p><p id="p-17">For the duplicate and complementary epistasis models, the quantities <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-102.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-103.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-104.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-105.gif"/></noscript></span></span> as well as the proportion of SCA to total genetic variance (<span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-106.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-107.gif"/></noscript></span></span>) were evaluated across a dense grid of degree of allele frequency differences between <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-108.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-109.gif"/></noscript></span></span> for the <em>B</em> and <em>C</em> locus. This difference will henceforth be referred to as “allele divergence” and defined as the difference between, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-110.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-111.gif"/></noscript></span></span>, with the midpoint being 0.5. Thus, at a divergence of 0.20, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-112.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-113.gif"/></noscript></span></span>, for example. For both models, <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-114.gif"/></noscript></span></span> was used. Only one locus was considered for the dominance model (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-115.gif"/></noscript></span></span>) and in addition to the allele divergence, the degree of dominance was varied from 0.0 to 3.0 (the homozygous effect was kept constant at <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-116.gif"/></noscript></span></span>).</p></div></div><div id="sec-4" class="subsection"><h3>Simulation of genomic prediction experiments</h3><p id="p-18">A comprehensive simulation of genomic prediction experiments was carried out to evaluate the accuracy of genomic models fitted from training data sets of F1 hybrids and F1:2 bulks for the purpose of predicting hybrid and GCA performance at the F1 level.</p><div id="sec-5" class="subsection"><h4>Parental inbred line genomes:</h4><p id="p-19">The observed genotypes at 35,478 single nucleotide polymorphism (SNP) markers of 209 maize inbred lines from the Dent (123) and Flint (86) heterotic groups of the maize breeding program of the University of Hohenheim in Germany formed the starting point of the simulation. This data set is available publicly as supplemental material to a publication by <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Technow <em>et al.</em> (2014a)</a>. This data were chosen to ensure that the simulated experiments are reflective of the genome properties (allele frequency distribution, linkage pattern and population structure) of an applied hybrid breeding program. The genome properties as well as the history of these populations and the heterotic pattern they form were described in previous studies (<a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Technow <em>et al.</em> 2014a</a>,<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">b</a>). For consistency sake, the Dent and Flint populations will arbitrarily be referred to as “male” and “female”, respectively.</p></div><div id="sec-6" class="subsection"><h4>In-silico biparental populations:</h4><p id="p-20">Biparental families of DH lines derived from elite inbred parents are the predominant population type encountered in early stages of breeding programs (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Mikel and Dudley 2006</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hickey <em>et al.</em> 2015</a>). In the simulation, each heterotic group was represented by 40 biparental families. The process of how those were created will be described for the male group but was followed analogously for the female group. Five of the 123 male inbred lines were assigned to the “high importance” group, twenty to the “medium importance” group and the remaining 98 to the “low importance” group. This assignment was done at random. The “high importance” inbreds were given a weight of 0.1, the “medium importance” inbreds a weight of 0.0125 and the remaining inbreds a weight of 0.00255 (the weights of all inbreds sum to 1.0). Then all <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-117.gif"/></noscript></span></span> possible biparental crosses among those 123 inbreds were assigned weights equal to the product of the weights of the corresponding parents. The 40 biparental families were then drawn at random from all possible families with probabilities proportional to the weights previously assigned. This results in an unbalanced contribution of inbred lines to the breeding populations and reflects that often just a few, very successful inbred lines are used disproportionately by breeders (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Mikel and Dudley 2006</a>; <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Technow <em>et al.</em> 2014b</a>).</p><p id="p-21">Then, 25 recombinant DH lines were generated in-silico from each family by simulating meiosis between the gametes of the parents followed by a chromosome doubling step. Meiosis was simulated according to the Haldane mapping function with the software package “hypred” (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Technow 2013</a>). Thus, each heterotic group comprised 1,000 DH lines from 40 biparental families. A random sample of 15,000 of the SNP markers was used for these simulations to facilitate computations. Because simulation of meiosis requires a genetic linkage map, the physical map positions of the SNP loci where rescaled linearly to the chromosome lengths of the genetic map reported by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fu <em>et al.</em> (2006)</a>.</p></div><div id="sec-7" class="subsection"><h4>Simulation of genetic architecture:</h4><p id="p-22">200 loci were defined to be QTL with direct influence on a generic complex trait. Those loci were chosen from the set of 15,000 SNP included in the simulation according to two population structure scenarios: “convergent” and “divergent”. In the convergent scenario, the QTL had similar allele frequencies in both heterotic groups with a maximum absolute difference of 0.05 (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-118.gif"/></noscript></span></span>). This corresponds to a newly formed hybrid breeding program before the establishment of distinct heterotic groups (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Melchinger 1999</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fischer <em>et al.</em> 2009</a>). In the divergent scenario, both populations had very different allele frequencies with a minimum absolute difference of 0.60 (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-119.gif"/></noscript></span></span>), which corresponds to a well established hybrid breeding program in which heterotic groups have diverged as a result of many cycles of reciprocal recurrent selection (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Labate <em>et al.</em> 1999</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Reif <em>et al.</em> 2007</a>; <a id="xref-ref-83-4" class="xref-bibr" href="#ref-83">Technow <em>et al.</em> 2014a</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Larièpe <em>et al.</em> 2017</a>). The definitions of the convergent and divergent heterotic group scenarios correspond to those in <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Technow <em>et al.</em> (2012)</a>. An additional requirement was that loci used as QTL had to have a minimum minor allele frequency of 0.025 in each heterotic group to ensure that they were contributing to genetic variation. Within those constraints, the 200 QTL were drawn at random.</p><p id="p-23">They were then randomly separated into 100 two-loci pairs. Each pair was assigned a matrix U describing the genotypic values. On average, 5% of the loci were assigned to the additive and another 5% to the additive by additive gene action models. The dominance model was assigned to 10% of the pairs. The remaining 80% of pairs were assigned to the complementary and the duplicate factor gene action models in equal proportion. Thus, on average 90% of the QTL gave rise to non-additive gene action effects that affect F1 and F1:2 genetic values differently. Note, however, that the latter three gene action models contain all types of gene action effects, including additive and additive by additive.</p><p id="p-24">The homozygous effects <em>a</em> (used for the additive and dominance models) were drawn from a Normal distribution with mean zero and standard deviation of <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-120.gif"/></noscript></span></span> (throughout, the Normal distribution will be parametrized by its mean and standard deviation). Their absolute values then have an expectation of 0.25. The heterozygosity gene effects <em>d</em> were drawn from <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-121.gif"/></noscript></span></span>, resulting in an average degree of dominance of one, which is consistent with experimental results in hybrid crops (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gardner 1963</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Radoev <em>et al.</em> 2008</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Schön <em>et al.</em> 2010</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Larièpe <em>et al.</em> 2012</a>). Using <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-122.gif"/></noscript></span></span> as standard deviation ensures that 90% of the sampled heterozygosity effects are above zero. This was done because dominance effects for traits showing hybrid vigor tend to be positive (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Semel <em>et al.</em> 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bennewitz and Meuwissen 2010</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Huang <em>et al.</em> 2015</a>). For the additive by additive model, the value of <em>z</em> was drawn from <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-123.gif"/></noscript></span></span> and for the duplicate and complementary gene action models from <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-124.gif"/></noscript></span></span>. The absolute values of <em>z</em> thus have expectations of 0.25 and 1.00, respectively. Those settings for the distributions of <em>a</em>, <em>d</em>, and <em>z</em> were chosen to ensure that the various gene action effects have the same magnitude in all gene action models. For example, with <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-125.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-126.gif"/></noscript></span></span> in the duplicate and complementary gene action models and hence equal to their expected values in the additive and dominance gene action models. Gene action effects for arbitrary U can be calculated with File S1 according to definitions by <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Holland (2001)</a>. Because interaction systems among loci can rarely be cleanly assigned to a particular model (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Holland 2001</a>), a small amount of “genetic noise” was added to the elements of each U matrix. Those values were drawn from a Normal distribution with mean zero and standard deviation equal to <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-127.gif"/></noscript></span></span>. The mean absolute value of those deviations was thus expected to be equal to 1/10<sup>th</sup> of the mean absolute value of the elements of a particular instance of U. All U matrices thus slightly deviated from their assigned models of gene action and as a result, even pairs assigned the simple additive model will give rise to small amounts of variation due to dominance and epistatic gene action.</p></div><div id="sec-8" class="subsection"><h4>In-silico population of hybrids:</h4><p id="p-25">The genotypic values of all F1 hybrids from the full factorial of <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-128.gif"/></noscript></span></span> interpopulation hybrids were calculated by summing the genetic effects across all 100 two-loci pairs according to the QTL genotypes of the hybrids. Because the parents were fully homozygous DH lines, the QTL genotypes of the hybrids follow directly from the genotypes of their parents. This full factorial was defined as the reference population of hybrids. The true GCA values of all 1,000 male and 1,000 female DH lines were calculated from the row and column means of the full factorial table (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Sprague and Tatum 1942</a>). Accordingly, the true SCA effects of all hybrids were obtained as the difference between the performance of the individual hybrids and the parental GCA effects.</p></div><div id="sec-9" class="subsection"><h4>Genomic prediction training set:</h4><p id="p-26">The DH from 20 male and 20 female families, chosen at random from the 40 male and female families created, were used for building the training set. Each of the 500 DH from the 20 families from one heterotic group was paired at random with one DH from the 20 families of the opposite heterotic group, resulting in a set of 500 hybrid combinations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). This design is similar to the reciprocal testcrossing design used by <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kadam <em>et al.</em> (2016)</a>. The 500 male and 500 female DH lines as well as the 500 hybrid combinations among them will henceforth also be referred to as “tested”. The 500 hybrid combinations comprising the training set are a subset of the total population of hybrids and the genotypic performance values of the F1 versions were calculated as described before. Phenotypic values were obtained by adding a Gaussian noise term with standard deviation chosen in such a way that the broad sense heritability was 0.5. This training set will also be refered to as F1 training set. The alternative F1:2 training set was made up of F1:2 bulks created from the same hybrids that were used for the F1 training set. For this, each F1 was selfed in-silico and 100 F2 individuals generated by simulating meiosis as described before. The true genetic values of each of the 100 F2s was calculated similarly as for the F1s and then averaged to arrive at the genetic performance of the F1:2 bulk. Phenotypic values were generated according two scenarios. In the first, the residual variation used for the F1 training set was assumed constant (“constant residual variation” scenario), in the second a constant heritability of 0.5 was assumed also for the F1:2 estimation set by adjusting the residual variation accordingly (“constant heritability” scenario).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic visualization of reciprocal crossing design used to build the training set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491559244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic visualization of reciprocal crossing design used to build the training set.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1557/F2.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1557/F2.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493131" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Schematic visualization of reciprocal crossing design used to build the training set.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Prediction model:</h4><p id="p-28">The following mixed model was fitted to the data<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-129.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-130.gif"/></noscript></span></span> is the scaled and centered phenotypic value of the <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-131.gif"/></noscript></span></span> training set entry (either a F1 hybrid or a F1:2 bulk, depending on the training set). The intercept is denoted by <em>μ</em>. Male and female additive main and additive by additive epistatic interaction effects are <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-132.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-133.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-134.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-135.gif"/></noscript></span></span>, respectively, which together constitute the GCA of a fully homozygous male and female parents of the <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-136.gif"/></noscript></span></span> hybrid combination (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Falconer and Mackay 1996</a>, p. 276). Dominance effects are denoted by <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-137.gif"/></noscript></span></span>, interaction effects between male and female additive and dominance effects are <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-138.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-139.gif"/></noscript></span></span>, respectively, and <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-140.gif"/></noscript></span></span> denote the dominance by dominance interaction effects. Together, those effects constitute the SCA effect of the <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-141.gif"/></noscript></span></span> hybrid combination. The residual associated with <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-142.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-143.gif"/></noscript></span></span> and was modeled as iid <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-144.gif"/></noscript></span></span>. The other effects were modeled by Multivariate-Normal distributions with mean vectors of zero and covariance matrices defined as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-145.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-146.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-147.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-148.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-149.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-150.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-151.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-152.gif"/></noscript></span></span>following <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Stuber and Cockerham (1966)</a>, where <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-153.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-154.gif"/></noscript></span></span> are the additive genomic relationship matrices of the tested male and female parents, respectively, of the training set hybrids and ’∘’ indicates element-wise multiplication. These matrices were calculated from the marker data following <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">VanRaden (2008)</a> as <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-155.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-156.gif"/></noscript></span></span> is the number of markers and <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-157.gif"/></noscript></span></span> (with <em>u</em> indexing the parent and <em>v</em> the marker), <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-158.gif"/></noscript></span></span> coding the number of reference alleles (taking values of 0 or 2), and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-159.gif"/></noscript></span></span> being the allele frequency of the reference allele in the male population. The genomic relationship matrix of the female parents was obtained analogously. Both matrices were calculated from the same set of 5,000 marker loci, which were obtained by randomly sampling from the initial set of 15,000 loci but excluding the 200 loci defined as QTL. The terms <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-160.gif"/></noscript></span></span> etc. are normalization factors equal to <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-161.gif"/></noscript></span></span>, etc. and help to bring the estimated variance components onto a comparable scale with the residual variance (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Xu 2013</a>). This model is similar to the one used by <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Massman <em>et al.</em> (2013)</a> and <a id="xref-ref-83-5" class="xref-bibr" href="#ref-83">Technow <em>et al.</em> (2014a)</a> except that these authors did not consider epistatic effects. The model was fitted using the R package ‘BGLR’ (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pérez and Campos 2014</a>) and its default settings for prior distributions and hyperparameters. A total of 1,000 samples were obtained from a chain of length 100,000, with a burn-in of 50,000 and a thinning interval of 50. The posterior means of the variance components and of the intercept were used as point estimates.</p></div><div id="sec-11" class="subsection"><h4>Genomic prediction accuracy:</h4><p id="p-29">The performance of all F1 hybrids from the full factorial was predicted using BLUP as <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-162.gif"/></noscript></span></span> following <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Henderson (1973)</a>, where <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-163.gif"/></noscript></span></span> is the genetic covariance matrix of predicted and tested hybrid combinations and <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-164.gif"/></noscript></span></span> the phenotypic covariance matrix of the data. The elements of <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-165.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-166.gif"/></noscript></span></span> were computed according to <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo (1996)</a>, with the addition of the covariance matrices of epistatic effects not considered there. Specifically, <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-167.gif"/></noscript></span></span>. Further, <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-168.gif"/></noscript></span></span>with <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-169.gif"/></noscript></span></span> etc., calculated analogously to the corresponding <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-170.gif"/></noscript></span></span> etc. but using additive relationship matrices <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-171.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-172.gif"/></noscript></span></span>, which represent the additive relationships of the parents of the 1,000,000 hybrid combination to be predicted and the parents of the training set hybrids. The normalization constants calculated previously were used to normalize these matrices, too. Predictions for SCA effects were obtained by limiting <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-173.gif"/></noscript></span></span> to the covariance matrices of the effects contributing to SCA. Following <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Technow <em>et al.</em> (2012)</a>, hybrids for which both the male and the female parent were tested were assigned to the “T2” prediction set, hybrids for which either the male or the female parent were tested (but not both) were assigned to the “T1” set and hybrids without any tested parent were assigned to the “T0” set. The tested hybrids themselves were assigned to the “T3” set. Prediction accuracy of total hybrid performance and SCA effects was defined as the Pearson correlation coefficient between predicted values and the corresponding true values of the F1 hybrids. To emphasize, the true performance and SCA effects of the F1 hybrids were used as reference values also when assessing the accuracy of the model fitted from the F1:2 training set.</p><p id="p-30">When predicting the GCA of DH lines, <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-174.gif"/></noscript></span></span> for males and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-175.gif"/></noscript></span></span> for females. The matrices <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-176.gif"/></noscript></span></span> etc., were computed as before but from <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-177.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-178.gif"/></noscript></span></span> which for this purpose contained the relationships between all 1,000 male or female parents and the male or female parents of the estimation set hybrids. Prediction accuracy of GCA effects was defined as the Pearson correlation coefficient between predicted and true GCA effects obtained from the full factorial of F1 hybrids. Thus, the prediction accuracy evaluates the ability to predict F1 based GCA, even when the F1:2 training set is used. The prediction accuracy was calculated separately for tested and untested DH lines and within and across families. The within family accuracies were averaged across families. To simplify the results further, all accuracies were averaged across male and female heterotic groups.</p><p id="p-31">The whole simulation was repeated independently 2,500 times for each heritability and population structure scenario and results averaged. All computations were conducted within the R statistical computing environment (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">R Core Team 2018</a>).</p></div></div><div id="sec-12" class="subsection"><h3>Data Availability</h3><p id="p-32">The SNP genotypes of the Dent and Flint inbred lines used as the initial population and the physical map of the SNP are available from the online supplement of <a id="xref-ref-83-6" class="xref-bibr" href="#ref-83">Technow <em>et al.</em> (2014a)</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7493996">https://doi.org/10.25387/g3.7493996</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Theoretical models</h3><p id="p-33">The correlation between F1 hybrids and F1:2 bulks (<span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-179.gif"/></noscript></span></span>) was mostly high (above 0.9) in the dominance (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) and both epistatic models (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4A</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">4G</a>). The widest range of values was thereby observed for the duplicate model, where <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-180.gif"/></noscript></span></span> reached below 0.5 when allele divergence was extreme (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). In the dominance model <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-181.gif"/></noscript></span></span> decreased with increasing degree of dominance, but remained high throughout. In the complementary model it was lowest when divergence was high at both loci, but again only slightly below the highest values, which were achieved when none or only one of the loci were strongly diverged (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4G</a>). The correlation between GCA effects obtained from F1 or F1:2 (<span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-182.gif"/></noscript></span></span>, average across male and female correlations) in the duplicate model followed a similar trend as <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-183.gif"/></noscript></span></span> (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>) but remained considerably higher also under extreme divergence. In the complementary model <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-184.gif"/></noscript></span></span> was high and confined to a narrow range, similar to <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-185.gif"/></noscript></span></span> (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4H</a>). The highest values were observed when divergence was either similar at both loci or very different. However, minimum and maximum values were only marginally different. With just a single locus, <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-186.gif"/></noscript></span></span> in the dominance model is either 1.0 or 0.0, depending on the combination of allele divergence and degree of dominance (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). More specifically, <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-187.gif"/></noscript></span></span> is a result of the correlation being 1.0 in the population with higher allele frequency and <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-188.gif"/></noscript></span></span> in the other. Values of <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-189.gif"/></noscript></span></span> are thereby observed within a narrow band defined by particular combinations of allele divergence and degree of dominance. The genetic variance of F1 hybrids (<span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-190.gif"/></noscript></span></span>) in the dominance model increased with increasing degree of dominance and decreasing divergence (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). In the epistatic models, it decreased with increasing divergence at one or both loci (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figures 4C</a> and <a id="xref-fig-4-8" class="xref-fig" href="#F4">4I</a>). Under the dominance and complementary models, the ratio <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-191.gif"/></noscript></span></span> was below one throughout, indicating that F1 hybrids are expected to have a greater genetic variance than F1:2 bulks (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figures 3D</a> and <a id="xref-fig-4-9" class="xref-fig" href="#F4">4J</a>). In the duplicate model the opposite was the case, and <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-192.gif"/></noscript></span></span> could be considerably larger than <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-193.gif"/></noscript></span></span> when allele divergence became extreme (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4D</a>). The proportion of SCA to total genetic variance in the F1 hybrids (<span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-194.gif"/></noscript></span></span>) followed similar trends as the total genetic variance under all models, <em>i.e.</em>, it decreased with increasing allele divergence and decreasing degree of dominance (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figures 3E</a>, <a id="xref-fig-4-11" class="xref-fig" href="#F4">4E</a> and <a id="xref-fig-4-12" class="xref-fig" href="#F4">4K</a>). Finally was the ratio <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-195.gif"/></noscript></span></span> below one throughout for all three models, indicating that relatively less genetic variation can be attributed to SCA effects in F1:2 bulks than in F1 hybrids (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figures 3G</a>, <a id="xref-fig-4-13" class="xref-fig" href="#F4">4G</a> and <a id="xref-fig-4-14" class="xref-fig" href="#F4">4L</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expected values of  (A),  (B),  (C),  (D),  (E) and  (F) under the &#x201C;dominance&#x201D; model as a function of allele frequency divergence and degree of dominance. Note that the color scale is not constant across sub-figures." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491559244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Expected values of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-196&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (A), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-197&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (B), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-198&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (C), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-199&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (D), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-200&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (E) and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-201&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (F) under the &#x201C;dominance&#x201D; model as a function of allele frequency divergence and degree of dominance. Note that the color scale is not constant across sub-figures.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1557/F3.medium.gif" width="440" height="67"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1557/F3.medium.gif" width="440" height="67"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493022" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Expected values of <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-196.gif"/></noscript></span></span> (A), <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-197.gif"/></noscript></span></span> (B), <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-198.gif"/></noscript></span></span> (C), <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-199.gif"/></noscript></span></span> (D), <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-200.gif"/></noscript></span></span> (E) and <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F3/embed/mml-math-201.gif"/></noscript></span></span> (F) under the “dominance” model as a function of allele frequency divergence and degree of dominance. Note that the color scale is not constant across sub-figures.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expected values of  (A,G),  (B,H),  (C,I),  (D,J), and  (E,K) and  (F,L) under the duplicate model (top row) and complementary model (bottom row) as a function of allele frequency divergence at both loci. Note that the color scale is not constant across sub-figures. The values in sub-figure D were transformed to  to aid visibility." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491559244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Expected values of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-202&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (A,G), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-203&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (B,H), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-204&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (C,I), &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-205&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (D,J), and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-206&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (E,K) and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-207&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (F,L) under the duplicate model (top row) and complementary model (bottom row) as a function of allele frequency divergence at both loci. Note that the color scale is not constant across sub-figures. The values in sub-figure D were transformed to &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-208&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; to aid visibility.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1557/F4.medium.gif" width="440" height="125"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1557/F4.medium.gif" width="440" height="125"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1557/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493015" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Expected values of <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-202.gif"/></noscript></span></span> (A,G), <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-203.gif"/></noscript></span></span> (B,H), <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-204.gif"/></noscript></span></span> (C,I), <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-205.gif"/></noscript></span></span> (D,J), and <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-206.gif"/></noscript></span></span> (E,K) and <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-207.gif"/></noscript></span></span> (F,L) under the duplicate model (top row) and complementary model (bottom row) as a function of allele frequency divergence at both loci. Note that the color scale is not constant across sub-figures. The values in sub-figure D were transformed to <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/F4/embed/mml-math-208.gif"/></noscript></span></span> to aid visibility.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Simulation results</h3><div id="sec-16" class="subsection"><h4>Quantitative genetic parameters:</h4><p id="p-36">Because these parameters are not affected by the heritability scenario, only results for the “constant residual variation” scenario will be shown. The F1 hybrids had a larger genetic variance than F1:2 bulks (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The difference between the two was relatively larger under the divergent heterotic group structure, where <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-209.gif"/></noscript></span></span> was less than half of <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-210.gif"/></noscript></span></span>. Overall, genetic variance was larger in the convergent structure than in the divergent structure, for both F1 and F1:2. The proportion of SCA to total genetic variation was larger in the convergent than the divergent structure by almost 10 percentage points (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). This quantity could only be assessed for the F1 hybrids for which the full factorial was created. Finally, the correlation between F1 hybrids and F1:2 bulks was largest in the convergent heterotic group structure and very close to 1.00 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). However, this correlation was with 0.91 very high in the divergent scenario, too.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493168/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493168/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average simulation results of total genetic variance, proportion of SCA variance and correlation between F1 hybrids and F1:2 bulks</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h4>GCA prediction:</h4><p id="p-40">Under constant residual variation, the prediction accuracy obtained from the F1 training set was higher than that for the F1:2 training set throughout (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Under a convergent heterotic group structure, the differences were small and did not exceed 0.04 points. Differences under the divergent structure were in the magnitude of 0.10 points. When the heritability of the F1:2 bulk phenotypes was made equal to that of the F1 hybrids (constant heritability scenario), the GCA prediction accuracy from the F1:2 training set increased considerably and now was close to (divergent structure) or even slightly higher (convergent structure) than that of the F1 training set. (The heritability of the latter was 0.5 in both cases and so the accuracy values were not expected to change.) Overall, prediction accuracy was higher for DH lines that contributed to the training set (“tested”) than for DH lines that did not (“untested”) and lower within families than across.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493075/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493075/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Average simulation results for genomic prediction accuracy of GCA effects</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>Hybrid prediction:</h4><p id="p-43">Under constant residual variation, the prediction of hybrid performance was more accurate when using the F1 training set than for the F1:2 training set (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The difference between the two thereby was largest for the divergent heterotic group structure, where it reached from 0.15 points for T3 hybrids to 0.06 points for T0 hybrids. Under the convergent heterotic group structure, the differences reached from 0.09 points (T3) to 0.01 points (T0). With constant heritability the accuracy of the predictions from the F1:2 training set increased markedly and now were similar (divergent structure) or even slightly higher (convergent structure) than for the F1 training set. In general, the performance of T3 hybrids was predicted with highest accuracy, followed by T2, T1 and T0 hybrids. The prediction accuracy of SCA effects was considerably lower than that of total hybrid performance throughout (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Similar trends held, however, with the exception that even under constant heritability the F1:2 training set was considerably less accurate than the F1 training set.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493024/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493024/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Average simulation results for genomic prediction accuracy of hybrid performance and SCA effects</span><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-47">Using F1:2 bulks for genomic prediction training sets and more generally testcross and hybrid evaluation is an alternative when production of large quantities of seed of F1 hybrids is expensive, significantly increases the complexity of a breeding program or is impossible altogether. The objective of this study was to assess the promise of this approach from a theoretical point of view with the help of quantitative genetics theory and stochastic simulation.</p><div id="sec-20" class="subsection"><h3>Quantitative genetic properties</h3><p id="p-48">Both the simulation as well as the theoretical results indicate that the correlation between F1 hybrids and F1:2 bulks can be expected to be high across a wide range of scenarios (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-15" class="xref-fig" href="#F4">4</a>). A few exceptions should be pointed out, however. Lower values, particularly for <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-215.gif"/></noscript></span></span>, were observed for increasing divergence between the allele frequencies in the two heterotic groups, particularly for the duplicate epistatic model (<a id="xref-fig-4-16" class="xref-fig" href="#F4">Figure 4A</a>). Here, most of the F1 hybrids are expected to have genotypic value of <em>z</em>, and only a small fraction a value of zero (those homozygous for the ’2’ allele at both loci). F1:2 bulks derived from F1 hybrids heterozygous at one or both loci, however, will exhibit an average performance that is intermediate between <em>z</em> and 0.0. With increasing allele frequency divergence, relatively more F1 hybrids are heterozygous at one or both loci, leading to a greater frequency of cases with differing F1 and F1:2 performance and hence a lower correlation.</p><p id="p-49">Two factors will lessen the impact of strongly diverged pairs of loci with duplicate gene action on the overall correlation for complex traits. First, is the genetic variance generated by loci pairs with duplicate gene action considerable lower than for pairs of loci with complementary gene action but similar divergence and magnitude of gene action effects (compare <a id="xref-fig-4-17" class="xref-fig" href="#F4">figures 4C</a> and <a id="xref-fig-4-18" class="xref-fig" href="#F4">4I</a>). Second, does the generated variance decrease with increasing divergence (<a id="xref-fig-4-19" class="xref-fig" href="#F4">Figure 4C</a>). Thus, when a trait is influenced by both duplicate and complementary epistasis and by loci with different degrees of divergence, the impact of strongly diverged loci exhibiting duplicate epistasis on the overall correlation between F1 and F1:2 genetic effects will be relatively minor.</p><p id="p-50">Under dominant gene action, the correlation between F1 and F1:2 GCA effects in the heterotic group with lower frequency of the allele that increase genetic value can take values of -1 within a narrow band defined by particular combinations of allele divergence and degree of dominance (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3B</a>). An explanation for this phenomenon is provided in the supplemental file S2. For brevity it should suffice here to state that for degrees of dominance below one (<em>i.e.</em>, partial dominance), the correlation is always +1 and that very high degrees of dominance are required to reach this “band” for low or moderate divergence of allele frequencies. As can be seen from the example in File S2 (and from File S1) as well, loci that exhibit this behavior will contribute relatively little to the variance of GCA effects in the affected heterotic group. Thus, to have an impact on the overall GCA correlation, the vast majority of loci controlling a complex trait must be within the specific combinations of allele divergence and degree of dominance defining this “band” and the alleles increasing genotypic value must consistently have a lower frequency in the same heterotic group. This seems unlikely for a complex trait controlled by hundreds or thousands of genes. The low contribution to GCA variance in the heterotic group in which <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-216.gif"/></noscript></span></span> is -1 also explains why <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-217.gif"/></noscript></span></span> remains high even then.</p><p id="p-51">The theoretical results show that to the degree that <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-218.gif"/></noscript></span></span> (and <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-219.gif"/></noscript></span></span>) did decrease, it largely did so as a function of increasing allele divergence. This explains that <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1557/embed/mml-math-220.gif"/></noscript></span></span> was lower in the divergent heterotic group structure, characterized by strong differences in male and female frequencies of QTL alleles, than in the convergent structure, where QTL alleles were constrained to have similar frequency in both heterotic groups.</p><p id="p-52">The observation that the genetic variance is lower among F1:2 bulks than among F1 hybrids (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) is also in line with the theory, which further predicts that this difference increases as allele frequencies diverge. This again is observed in the simulation results, in which the F1:2 had approximately 61% of the variation of the F1 in the convergent heterotic group structure but only about 48% in the divergent structure. Duplicate epistasis again presents somewhat of an exception, because here the F1:2 actually had a larger variance than the F1 (<a id="xref-fig-4-20" class="xref-fig" href="#F4">Figure 4D</a>) and the more so the more the alleles diverged. As mentioned before, however, the weight of loci with duplicate epistatic effects on determining overall genetic trends for complex traits is considerably lower than that of loci with other types of gene action.</p><p id="p-53">Finally, the theoretical results show that the proportion of SCA to total genetic variation in the F1 hybrids decreases with increasing interpopulation divergence for the dominance (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3E</a>), the duplicate (<a id="xref-fig-4-21" class="xref-fig" href="#F4">Figure 4E</a>) and the complementary model (<a id="xref-fig-4-22" class="xref-fig" href="#F4">Figure 4K</a>). This was also observed in the simulation, where the relative amount of SCA variation was almost 10 percentage points higher in the convergent than in the divergent population structure scenario. <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Reif <em>et al.</em> (2007)</a> previously showed that increasing interpopulation divergence reduces the proportion of SCA variance when considering only dominance. For random mating populations in Hardy-Weinberg equilibrium it was further shown that genetic variance generated by the epistatic and dominance models considered here is largely additive when allele frequencies are at extreme values (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Hill <em>et al.</em> 2008</a>). Theoretical results also imply that the contribution of SCA to total variance is expected to be relatively lower for F1:2 bulks than F1 hybrids. This is easiest to see under dominant gene action, where <em>d</em> is in essence halved because only 50% of the segregants within a F1:2 bulk derived from a heterozygous F1 will have a genetic value of <em>d</em> and the genetic values of the homozygous segregants will cancel in the determination of the mean value of the bulk.</p><p id="p-54">The theory developed here in principle applies also to QTL with large effects on the trait of interest. An exception, however, are major QTL that affect plant stature and morphology. A prime example of such QTL are “dwarfing genes” (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hedden 2003</a>). Their segregation in the F1:2 bulks would result in uneven plant stands as well as increased plant to plant competition, which would add to the differential performance between F1 hybrids and F1:2 bulks in a way not accounted for by the theory or simulations developed here. It could also result in an increased residual variance for the F1:2 bulks and hence a lower heritability, the adverse effects of which will be discussed below.</p></div><div id="sec-21" class="subsection"><h3>Genomic prediction of GCA effects</h3><p id="p-55">Based on the developed theory, F1:2 bulks are expected to have a lower genetic variance than F1 hybrids. With constant residual variation, <em>i.e.</em>, F1 and F1:2 training sets phenotyped with same number of locations and replications, the latter is thus expected to have a lower heritability. Prediction accuracy is expected to decrease with decreasing heritability of the training set phenotypes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Daetwyler <em>et al.</em> 2010</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Lorenz 2013</a>). The reduced prediction accuracy coming from the F1:2 training set in the “constant residual variation” scenario (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) thus was in part due to the lower heritability of the F1:2 phenotypes. Indeed, when both training sets had the same heritability, accuracy was similar, too. In the convergent heterotic group scenario the accuracy of the F1:2 training set was then even slightly higher than that of the F1 training set. SCA effects essentially act as noise and even modeling them is unlikely to completely remove their confounding effects on the estimation of GCA effects. This surprising result can therefore be explained by the relatively lower amount of SCA variation in the F1:2 training set, as indicated by the theoretical results (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figures 3F</a>, <a id="xref-fig-4-23" class="xref-fig" href="#F4">4F</a>, and <a id="xref-fig-4-24" class="xref-fig" href="#F4">4L</a>). SCA variation is expected to be relatively less important under a divergent heterotic group structure, as shown by theory and simulation results obtained here and elsewhere (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Reif <em>et al.</em> 2007</a>) as well as in practice (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Fischer <em>et al.</em> 2009</a>). The relatively lower amount of SCA variation in the F1:2 training set will therefore be less of a benefit in this case.</p><p id="p-56">Even if the genomic model fitted could explain 100% of the genetic variation in the F1:2 bulks, the prediction accuracy would have an upper bound proportional to the correlation between F1 and F1:2 genetic effects. It was shown here that this correlation is expected to decrease with increasing allele divergence between heterotic groups and observed to be lower lower in the divergent heterotic groups structure than in the convergent structure. Finally, both theory and the simulation show that the relative reduction in genetic variance is greater, the greater the degree of divergence. With a constant residual variation, the reduction in heritability was therefore larger in the divergent scenario. These three factors combined, <em>i.e.</em>, the lower benefit from the reduced proportion of SCA variance, the lower correlation between F1 and F1:2 genetic effects, and the lower heritability can explain why a larger accuracy penalty from using F1:2 training sets was observed for the divergent heterotic group structure.</p><p id="p-57">The described trends held in general for prediction of tested and untested DHs. The differences between the accuracy from F1 and F1:2 training sets, however, were larger when predicting tested DH lines than for untested ones (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), at least for the “constant residual variation” scenario. Genomic predictions of tested individuals (<em>i.e.</em>, individuals contributing to the training set), are strongly influenced by the phenotypic observations available for them (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Endelman and Jannink 2012</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Müller <em>et al.</em> 2015</a>). It can be speculated that they are therefore more sensitive to differences in F1 and F1:2 genetic values and the lower heritability of the latter. Nonetheless, accuracy for tested DH was always considerably higher than for untested DH. Each tested DH was not only represented in the training set directly in the form of a hybrid progeny, but also had 24 full-sib DH that directly contributed to the training set as well. Previous studies demonstrated the positive effect on prediction accuracy of direct observations (<em>e.g.</em>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Endelman <em>et al.</em> 2014</a>) and presence of full-sibs in the training set (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Habier <em>et al.</em> 2013</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Schopp <em>et al.</em> 2017</a>). The trends concerning accuracy of F1 and F1:2 training sets also held for within and across family prediction. That the latter was considerably more accurate was expected because it is largely driven by separation of family means (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Windhausen <em>et al.</em> 2013</a>; <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Riedelsheimer <em>et al.</em> 2013</a>).</p></div><div id="sec-22" class="subsection"><h3>Genomic prediction of hybrid performance</h3><p id="p-58">Because GCA is an important component of total hybrid performance, similar trends were observed here. Namely that the prediction accuracy from the F1:2 training set was trailing that of the F1 training set under constant residual variation but caught up to or even exceeded it under constant heritability (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Similarly, the accuracy difference was larger for the divergent population structure and for hybrids that were part of the training set (T3). The explanations given above for GCA apply also here. That prediction for T3 hybrids was most accurate, followed by that of T2, T1 and T0 hybrids, was observed in previous studies (<em>e.g.</em>, <a id="xref-ref-83-7" class="xref-bibr" href="#ref-83">Technow <em>et al.</em> 2014a</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Zhao <em>et al.</em> 2015</a>) and is a result of the decreasing degree of relatedness between the hybrids in the four classes and the training set. SCA represents higher order statistical effects and is therefore estimated with greater error than the GCA effects contributing to hybrid performance (<a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">Technow <em>et al.</em> 2012</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Kadam <em>et al.</em> 2016</a>). Their accuracy was therefore considerably lower than that of GCA effects (compare the “across population” section of <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> with <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Because it is relatively less important with divergent heterotic groups, the lower accuracy of predicted SCA effect will matter less for determining total hybrid performance, which consequently was predicted with greater accuracy than in the convergent scenario.</p><p id="p-59">There has recently been interest in revisiting reciprocal full-sib mating designs (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hallauer and Eberhart 1970</a>) in light of advances in genomic prediction of hybrid (<a id="xref-ref-82-4" class="xref-bibr" href="#ref-82">Technow <em>et al.</em> 2012</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Kadam <em>et al.</em> 2016</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fritsche-Neto <em>et al.</em> 2018</a>) and GCA (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Giraud <em>et al.</em> 2017</a>) performance. As noted by <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Giraud <em>et al.</em> (2017)</a>, reciprocal testing has several advantages over the still predominant topcross design, in which individuals under evaluation are crossed with a common partner (“tester”) from the opposite heterotic group (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Jenkins and Brunson 1932</a>). A major advantage is that field testing is twice as efficient because testcross hybrids are informative for both heterotic groups, meaning that the same number of individuals can be tested with half the resources (<em>e.g.</em>, 1,000 instead of 500 total hybrids would have been required to represent 500 male and 500 female DH if a topcross design would have been used). The other advantage is that when the crosses are made at random, they comply with the assumptions of a gene-orthogonal population (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Schnell 1965</a>) thus facilitating unbiased estimation of random GCA and SCA effects and their variances. The training set used in this study was constructed in this reciprocal fashion by directly pairing male and female DH at random, without the use of a topcross tester. The results are thus additional confirmation that hybrid performance as well as GCA and SCA effects can be evaluated and predicted accurately with such a design, even when each parent is represented in only a single hybrid combination.</p></div><div id="sec-23" class="subsection"><h3>Practical considerations</h3><p id="p-60">This study focused on assessing the theoretical potential of using F1:2 bulks for evaluating and predicting F1 hybrid and GCA performance, based on measures such as the expected correlations between F1 and F1:2 derived performance values and effects on genomic prediction accuracy. However, biological and practical considerations will also factor into the relative merit of the proposed approach. Key considerations hereby are the seed multiplication rate (= number of seeds harvested per plant) and the seeding rate (= number of seeds planted per area unit) of the crop in question. Both will determine how many F1 plants, which have to be generated by hand pollination, are required to produce the amount of F1:2 seed needed for conducting multi-environment field trials. All else equal, the proposed approach will be more promising for crops with high seed multiplication rate and low seeding rate, like rapeseed and sugar beet, than for crops with a low multiplication rate and high seeding rate, which tends to be the case for many small grain cereals. Other factors influencing the economical viability of F1:2 production for research purposes include flower morphology, determining the ease with which hand pollination can be carried out, and ultimately also the availability and cost of labor. Those biological and practical factors, however, are highly crop specific and, as far as labor costs are concerned, also depend on the geography of the breeding program. A detailed economic cost-benefit analysis of the F1:2 approach is therefore not within the scope of this study.</p><p id="p-61">In cases where neither the implementation of a hybridization system nor the production of F1:2 seed is feasible on a large scale, inbred per-se evaluation might be considered. The correlation between inbred per-se and testcross performance is, however, typically expected to be low for complex traits like grain yield (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Smith 1986</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Schopp <em>et al.</em> 2015</a>). This therefore seems promising only for scenarios in which selection for simply inherited traits with predominantly additive gene action is important and can be carried out as a first step in a multi-stage (genomic or phenotypic) selection scheme. Production of either F1 (through a hybridization system) or F1:2 seed for testcross evaluation might then be possible for the much reduced set of selected inbred lines.</p><p id="p-62">A reduction in heritability was identified as the main factor reducing prediction accuracy of F1:2 training sets relative to that of F1 training sets, rather than differences in F1 and F1:2 genetic values. Options for increasing the heritability of F1:2 phenotypes include testing in more environments and/or replications and possibly in larger field plots (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rebetzke <em>et al.</em> 2014</a>). Alternatively, the lower heritability could partly be compensated by increasing training set size (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Lorenz 2013</a>). The increase in resource requirements associated with each option might be justifiable with the expected reduction in complexity and cost of seed production.</p><p id="p-63">This study showed that the ability of F1:2 training sets to predict GCA and hybrid performance on the F1 level is expected to be greater, the lower the degree of divergence between heterotic groups. Most crops for which hybrid breeding is considered promising currently lack clearly defined heterotic patterns (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Melchinger and Gumber 1998</a>; <a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Zhao <em>et al.</em> 2015</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beukert <em>et al.</em> 2017</a>), certainly not to the extent present in maize, where many decades of reciprocal-recurrent selection led to a system of divergent, co-evolving heterotic groups (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Duvick <em>et al.</em> 2004</a>; <a id="xref-ref-83-8" class="xref-bibr" href="#ref-83">Technow <em>et al.</em> 2014a</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gerke <em>et al.</em> 2015</a>). Thus, for many crop species, for which the lack of an efficient and reliable hybridization system is a major impediment (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Longin <em>et al.</em> 2012</a>), resorting to F1:2 bulks is particularly promising.</p><p id="p-64">It should be noted that for all self-pollinating crop species, the baseline for comparison are not the “true” F1 hybrids considered in this study, but those produced with the help of hybridization systems. Chemical hybridization agents (CHA), for example, often result in only partial sterility of the female parent, meaning that a certain proportion of the harvested seed is in fact the result of a self pollination. The degree with which this happens is genotype dependent (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adugna <em>et al.</em> 2004</a>), leading to a potential confounding of genetic effects and F1 seed purity in field trials. In a CMS system, several cycles of backcrossing are required to develop sterile “A-line” versions of the female inbred lines. In early backcrossing generations, the A-lines still contain significant amounts of donor genome and do not accurately reflect the B-line genotype (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ahmadikhah <em>et al.</em> 2015</a>). Neither is necessarily a major concern for production of commercial seed, where CHA applications can be optimized for each commercial hybrid and there is enough time for as many cycles of backcrossing as necessary to remove nearly all residual donor genome. Optimization of CHA applications for hundreds or thousands of experimental hybrids used in a training set or for testcross evaluation, however, is unfeasible, and carrying out three or more cycles of backcrossing before testcrossing would increase the total length of the breeding cycle significantly. Thus, the F1 hybrids used in practice for training set development and testcross evaluation are also not expected to reflect “true” F1 performance without bias.</p><p id="p-65">Even in the “worst-case” scenario however, <em>i.e.</em>, without the ability of compensating for the reduced heritability of F1:2 bulks and with strong divergence between heterotic groups, should the prediction accuracy observed here for GCA and hybrid performance of tested and untested individuals be high enough to facilitate genetic gain and identification of superior hybrids. The modest accuracy penalty when using F1:2 bulks might therefore be a reasonable price to pay for the prospect of removing the significant complication and resource requirement of a hybridization system from the breeding process.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-66">This article is dedicated to Albrecht E. Melchinger on the occasion of his retirement, to honour his lasting contribution to plant breeding and genetics and his teaching and guidance that shaped generations of students.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-67">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7493996">https://doi.org/10.25387/g3.7493996</a>.</p></li><li class="fn" id="fn-10"><p id="p-68"><em>Communicating editor: J.-L. Jannink</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-20"><span class="received-label">Received </span>December 20, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-09"><span class="accepted-label">Accepted </span>March 9, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Technow</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.1" data-doi="10.1023/B:EUPH.0000013320.28114.c6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adugna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nanda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bains</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A comparison of cytoplasmic and chemically-induced male sterility systems for hybrid seed production in wheat (Triticum aestivum L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/B:EUPH.0000013320.28114.c6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D135%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253AEUPH.0000013320.28114.c6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:EUPH.0000013320.28114.c6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000188422900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahmadikhah</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirarab</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pahlevani</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nayyeripasand</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Marker-assisted backcrossing to develop an elite cytoplasmic male sterility line in rice.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>. <a href="https://doi.org/10.3835/plantgenome2014.07.0031">https://doi.org/10.3835/plantgenome2014.07.0031</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.3" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome based prediction of testcross values in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1587-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAlbrecht%26rft.auinit1%253DT.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btestcross%2Bvalues%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.4" data-doi="10.1016/j.tplants.2013.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Field high-throughput phenotyping: the new crop breeding frontier.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2013.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139902&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330013700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.5" data-doi="10.1111/j.1439-0388.2009.00847.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The distribution of QTL additive and dominance effects in porcine F2 crosses.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2009.00847.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DBennewitz%26rft.auinit1%253DJ.%26rft.volume%253D127%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D179%26rft.atitle%253DThe%2Bdistribution%2Bof%2BQTL%2Badditive%2Band%2Bdominance%2Beffects%2Bin%2Bporcine%2BF2%2Bcrosses.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2009.00847.x%26rft_id%253Dinfo%253Apmid%252F20536634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2009.00847.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20536634&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.6" data-doi="10.2135/cropsci1996.0011183X003600040009x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Best linear unbiased prediction of the performance of crosses between untested maize inbreds.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">876</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1996.0011183X003600040009x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D872%26rft.epage%253D876%26rft.atitle%253DBest%2BLinear%2BUnbiased%2BPrediction%2Bof%2Bthe%2BPerformance%2Bof%2BCrosses%2Bbetween%2BUntested%2BMaize%2BInbreds%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1996.0011183X003600040009x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1996.0011183X003600040009x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.7" data-doi="10.1186/s12284-017-0163-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beukert</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandra</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-based identification of heterotic patterns in rice.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12284-017-0163-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D10%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12284-017-0163-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12284-017-0163-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.8" data-doi="10.1007/s00299-016-1949-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bohra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">U. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adhimoolam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bisht</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Cytoplasmic male sterility (CMS) in hybrid breeding in field crops.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">993</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00299-016-1949-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BRep.%26rft.volume%253D35%26rft.spage%253D967%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00299-016-1949-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00299-016-1949-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.9" data-doi="10.21273/HORTSCI.40.6.1679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bosland</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Second generation (F2) hybrid cultivars for Jalapeño production.</span> <abbr class="cit-jnl-abbrev">HortScience</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1679</span>–<span class="cit-lpage">1681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.21273/HORTSCI.40.6.1679</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortScience%26rft.stitle%253Dhortsci%26rft.aulast%253DBosland%26rft.auinit1%253DP.%2BW.%26rft.volume%253D40%26rft.issue%253D6%26rft.spage%253D1679%26rft.epage%253D1681%26rft.atitle%253DSecond%2BGeneration%2B%2528F2%2529%2BHybrid%2BCultivars%2Bfor%2BJalapeno%2BProduction%26rft_id%253Dinfo%253Adoi%252F10.21273%252FHORTSCI.40.6.1679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiaG9ydHNjaSI7czo1OiJyZXNpZCI7czo5OiI0MC82LzE2NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.10" data-doi="10.2307/3001412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comstock</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">H. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">The components of genetic variance in populations of biparental progenies and their use in estimating the average degree of dominance.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/3001412</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DComstock%26rft.auinit1%253DR.%2BE.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D254%26rft.epage%253D266%26rft.atitle%253DThe%2Bcomponents%2Bof%2Bgenetic%2Bvariance%2Bin%2Bpopulations%2Bof%2Bbiparental%2Bprogenies%2Band%2Btheir%2Buse%2Bin%2Bestimating%2Bthe%2Baverage%2Bdegree%2Bof%2Bdominance.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3001412%26rft_id%253Dinfo%253Apmid%252F18108899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3001412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18108899&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1948UG24700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comstock</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">H. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1952</span> <span class="cit-article-title">Estimation of average dominance of genes.</span> In <span class="cit-source">Heterosis</span>, pp. <span class="cit-fpage">494</span>–<span class="cit-lpage">516</span>, <span class="cit-publ-name">Iowa State College</span>, <span class="cit-publ-loc">Ames, IA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.12" data-doi="10.1071/CP14007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messina</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Totir</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumgarten</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Predicting the future of plant breeding: complementing empirical evaluation with genetic prediction.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">311</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/CP14007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D65%26rft.spage%253D311%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP14007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP14007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000337726700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.13" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duvick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Heterosis: feeding people and protecting natural resources</span>, pp. <span class="cit-fpage">19</span>–<span class="cit-lpage">29</span> in <span class="cit-source">The genetics and exploitation of heterosis in crops</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-publ-name">CSSA</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duvick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Long-term selection in a commercial hybrid maize breeding program</span>, pp. <span class="cit-fpage">109</span>–<span class="cit-lpage">152</span> in <span class="cit-source">Plant Breeding Reviews</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Janick</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.16" data-doi="10.1016/j.biotechadv.2015.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dwivedi</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Britt</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tripathi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Haploids: constraints and opportunities in plant breeding.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Adv.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">812</span>–<span class="cit-lpage">829</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.biotechadv.2015.07.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnol.%2BAdv.%26rft.volume%253D33%26rft.spage%253D812%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biotechadv.2015.07.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biotechadv.2015.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.17" data-doi="10.2135/cropsci2013.03.0154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Optimal design of preliminary yield trials with genome-wide markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.03.0154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.03.0154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.03.0154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.18" data-doi="10.1534/g3.112.004259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shrinkage estimation of the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.004259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1405%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004259%26rft_id%253Dinfo%253Apmid%252F23173092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.004259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23173092&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to quantitative genetics</span>, <span class="cit-edition">Ed. 4th</span>. <span class="cit-publ-name">Pearson</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.20" data-doi="10.2135/cropsci2008.12.0689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiemt</span>  <span class="cit-name-given-names">E.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Impact of genetic divergence on the ratio of variance due to specific <em>vs.</em> general combining ability in winter triticale.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">2119</span>–<span class="cit-lpage">2122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.12.0689</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DFischer%26rft.auinit1%253DS.%26rft.volume%253D49%26rft.issue%253D6%26rft.spage%253D2119%26rft.epage%253D2122%26rft.atitle%253DImpact%2Bof%2BGenetic%2BDivergence%2Bon%2Bthe%2BRatio%2Bof%2BVariance%2BDue%2Bto%2BSpecific%2Bvs.%2BGeneral%2BCombining%2BAbility%2Bin%2BWinter%2BTriticale%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.12.0689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.12.0689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000271607300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.21" data-doi="10.1007/s00122-018-3068-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fritsche-Neto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Accuracy of genomic selection to predict maize single-crosses obtained through different mating designs.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3068-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D1153%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3068-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3068-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.5.1557.22" data-doi="10.1007/s00122-018-3118-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fritsche-Neto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Correction to: accuracy of genomic selection to predict maize single-crosses obtained through different mating designs.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1603</span>. (erratum)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3118-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D1603%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3118-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3118-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.23" data-doi="10.1534/genetics.106.060376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">T.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic dissection of intermated recombinant inbred lines using a new genetic map of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1671</span>–<span class="cit-lpage">1683</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.060376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.060376%26rft_id%253Dinfo%253Apmid%252F16951074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzE2NzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Estimates of genetic parameters in cross-fertilizing plants and their implications in plant breeding</span>, pp. <span class="cit-fpage">225</span>–<span class="cit-lpage">251</span> in <span class="cit-source">Statistical Genetics and Plant Breeding</span>, <span class="cit-vol">Vol. 982</span>. <span class="cit-publ-name">Comittee on Plant Breeding and Genetics of the Agricultural Board at the North Carolina State College</span>, <span class="cit-publ-loc">Raleigh, NC</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistical%2BGenetics%2Band%2BPlant%2BBreeding%26rft.volume%253D982%26rft.spage%253D225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.25" data-doi="10.2135/cropsci2016.09.0742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A two-part strategy for using genomic selection to develop inbred lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">2372</span>–<span class="cit-lpage">2386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.09.0742</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D2372%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.09.0742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.09.0742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.26" data-doi="10.1534/genetics.115.182410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guill</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomic impacts of drift and selection for hybrid performance in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.182410</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182410%26rft_id%253Dinfo%253Apmid%252F26385980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS8zLzEyMDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.27" data-doi="10.1534/genetics.117.300305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraud</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madur</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Combes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Reciprocal genetics: identifying QTLs for general and specific combining abilities in hybrids between multiparental populations from two maize (<em>Zea mays</em> L.) heterotic groups.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">1167</span>–<span class="cit-lpage">1180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300305%26rft_id%253Dinfo%253Apmid%252F28971957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy8zLzExNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.28" data-doi="10.2135/cropsci2016.06.0526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Battagin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumasy</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Prospects for cost-effective genomic selection via accurate within-family imputation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.06.0526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D216%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.06.0526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.06.0526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.29" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.30" data-doi="10.2135/cropsci1970.0011183X001000030033x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhart</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Reciprocal full-sib selection.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1970.0011183X001000030033x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHallauer%26rft.auinit1%253DA.%2BR.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D315%26rft.epage%253D316%26rft.atitle%253DReciprocal%2BFull-Sib%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1970.0011183X001000030033x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1970.0011183X001000030033x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970G657200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.31" data-doi="10.1016/S0168-9525(02)00009-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The genes of the green revolution.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(02)00009-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHedden%26rft.auinit1%253DP.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D9%26rft.atitle%253DThe%2Bgenes%2Bof%2Bthe%2BGreen%2BRevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252900009-4%26rft_id%253Dinfo%253Apmid%252F12493241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)00009-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12493241&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180294900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.32" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.11.0662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.33" data-doi="10.1093/ansci/1973.Symposium.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Sire evaluation and genetic trends.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">1973</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/ansci/1973.Symposium.10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D1973%26rft.issue%253DSymposium%26rft.spage%253D10%26rft.epage%253D41%26rft.atitle%253DSIRE%2BEVALUATION%2BAND%2BGENETIC%2BTRENDS%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fansci%252F1973.Symposium.10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ansci/1973.Symposium.10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.34" data-doi="10.2135/cropsci2013.03.0195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1476</span>–<span class="cit-lpage">1488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.03.0195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1476%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.03.0195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.03.0195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.35" data-doi="10.2135/cropsci2014.09.0648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nettelblad</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Imputation of single nucleotide polymorphism genotypes in biparental, backcross, and topcross populations with a hidden Markov model.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1934</span>–<span class="cit-lpage">1946</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.09.0648</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1934%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.09.0648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.09.0648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.36" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Epistasis and plant breeding</span>, pp. <span class="cit-fpage">27</span>–<span class="cit-lpage">92</span> in <span class="cit-source">Plant Breeding Reviews</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Janick</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.38" data-doi="10.1038/ncomms7258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic analysis of hybrid rice varieties reveals numerous superior alleles that contribute to heterosis.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms7258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6258%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7258%26rft_id%253Dinfo%253Apmid%252F25651972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25651972&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.39" data-doi="10.1371/journal.pone.0147769"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jan</span>  <span class="cit-name-given-names">H. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbadi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lücke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of testcross performance in canola (Brassica napus).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0147769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0147769</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0147769%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0147769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0147769&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janick</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Hybrids in horticultural crops.</span> <abbr class="cit-jnl-abbrev">CSSA Special Publication</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCSSA%2BSpecial%2BPublication%26rft.volume%253D25%26rft.spage%253D45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.41" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics Proteomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.42" data-doi="10.2134/agronj1932.00021962002400070004x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunson</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1932</span> <span class="cit-article-title">Methods of testing inbred lines of maize in crossbred combinations.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">530</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj1932.00021962002400070004x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DJenkins%26rft.auinit1%253DM.%2BT.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D523%26rft.epage%253D530%26rft.atitle%253DMethods%2Bof%2BTesting%2BInbred%2BLines%2Bof%2BMaize%2Bin%2BCrossbred%2BCombinations%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1932.00021962002400070004x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1932.00021962002400070004x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadam</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohn</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Genomic prediction of single crosses in the early stages of a maize hybrid breeding pipeline. G3 (Bethesda) 6: 3442–3453. https://10.1534/g3.116.031286</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.44" data-doi="10.1007/s11032-011-9555-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gils</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Pollination control technologies for hybrid breeding.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">437</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-011-9555-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D27%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-011-9555-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-011-9555-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000288714400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.45" data-doi="10.1016/j.tplants.2017.10.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Molecular control of male fertility for crop hybrid breeding.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.10.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D23%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.10.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.46" data-doi="10.1007/s001220051321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Temporal changes in allele frequencies in two reciprocally selected maize populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220051321</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D99%26rft.spage%253D1166%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.47" data-doi="10.1534/genetics.111.133447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larièpe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Combes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genetic basis of heterosis: multiparental quantitative trait loci mapping reveals contrasted levels of apparent overdominance among traits of agronomical interest in maize (Zea mays L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.133447</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.133447%26rft_id%253Dinfo%253Apmid%252F22135356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNzk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.48" data-doi="10.1007/s00122-016-2822-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larièpe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laborde</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mezmouk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">General and specific combining abilities in a maize (Zea mays L.) test-cross hybrid panel: relative importance of population structure and genetic divergence between parents.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">417</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2822-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D403%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2822-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2822-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.49" data-doi="10.1007/s00122-015-2505-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mi</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection in wheat: optimum allocation of test resources and comparison of breeding strategies for line and hybrid breeding.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1306</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2505-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D1297%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2505-1%26rft_id%253Dinfo%253Apmid%252F25877519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2505-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25877519&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.50" data-doi="10.1007/s00122-012-1967-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mühleisen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hybrid breeding in autogamous cereals.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1087</span>–<span class="cit-lpage">1096</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1967-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLongin%26rft.auinit1%253DC.%2BF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1087%26rft.epage%253D1096%26rft.atitle%253DHybrid%2Bbreeding%2Bin%2Bautogamous%2Bcereals.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1967-7%26rft_id%253Dinfo%253Apmid%252F22918662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1967-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22918662&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.51" data-doi="10.1534/g3.112.004911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.004911</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.004911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.52" data-doi="10.1007/s00122-012-1955-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordillo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide predictions from maize single-cross data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1955-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1955-y%26rft_id%253Dinfo%253Apmid%252F22886355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1955-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22886355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic diversity and heterosis</span>, pp. <span class="cit-fpage">99</span>–<span class="cit-lpage">118</span> in <span class="cit-source">The genetics and exploitation of heterosis in crops</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-publ-name">ASA, CSSA, and SSSA</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gumber</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Overview of heterosis and heterotic groups in agronomic crops</span>, pp. <span class="cit-fpage">29</span>–<span class="cit-lpage">44</span> in <span class="cit-source">Concepts and Breeding of Heterosis in Crop Plant</span>, <span class="cit-publ-name">CSSA</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.5.1557.55"><div class="cit-metadata unstructured">Melchinger, A. E., C. F. H. Longin, H. F. Utz, and J. C. Reif, 2005 Hybrid maize breeding with doubled haploid lines: quantitative genetic and selection theory for optimum allocation of resources. In <em>Proceedings of the forty first annual Illinois corn breeders’ School 2005</em>, pp. 8–21, Urbana-Champaign, Illinois, USA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.56" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.57" data-doi="10.2135/cropsci2005.10-0371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mikel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolution of North American dent corn from public to proprietary germplasm.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2005.10-0371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMikel%26rft.auinit1%253DM.%2BA.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D1193%26rft.epage%253D1205%26rft.atitle%253DEvolution%2Bof%2BNorth%2BAmerican%2BDent%2BCorn%2Bfrom%2BPublic%2Bto%2BProprietary%2BGermplasm%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.10-0371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.10-0371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237375900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mood</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graybill</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duane</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-source">Introduction to the theory of statistics</span>, <span class="cit-edition">Ed. 3rd</span>. <span class="cit-publ-name">McGraw Hill</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.59" data-doi="10.1007/s00122-015-2464-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Shrinkage estimation of the genomic relationship matrix can improve genomic estimated breeding values in the training set.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2464-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D693%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2464-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2464-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.60" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression &amp; prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.61" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.05.0005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1557.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R: a language and environment for statistical computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.63" data-doi="10.1534/genetics.108.089680"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Radoev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecke</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic analysis of heterosis for yield and yield components in rapeseed (Brassica napus L.) by quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1547</span>–<span class="cit-lpage">1558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.089680</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089680%26rft_id%253Dinfo%253Apmid%252F18562665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1NDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.64" data-doi="10.1071/FP13177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebetzke</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">R. T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herwaarden</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Plot size matters: interference from intergenotypic competition in plant phenotyping studies.</span> <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/FP13177</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D41%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.1071%252FFP13177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/FP13177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.65" data-doi="10.1534/genetics.107.074146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gumpert</span>  <span class="cit-name-given-names">F.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Impact of interpopulation divergence on additive and dominance variance in hybrid populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1931</span>–<span class="cit-lpage">1934</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.074146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074146%26rft_id%253Dinfo%253Apmid%252F17507673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE5MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.66" data-doi="10.1534/genetics.113.150227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stange</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic predictability of interconnected biparental maize populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.150227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150227%26rft_id%253Dinfo%253Apmid%252F23535384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.67" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">728</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.141473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.68" data-doi="10.1002/bimj.19650070102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnell</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Die Covarianz zwischen Verwandten in einer gen-orthogonalen Population.</span> <abbr class="cit-jnl-abbrev">Biometrische Zeitung</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bimj.19650070102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrische%2BZeitung%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbimj.19650070102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bimj.19650070102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.69" data-doi="10.1007/s00122-009-1209-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High congruency of QTL positions for heterosis of grain yield in three crosses of maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1209-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchon%26rft.auinit1%253DC.%2BC.%26rft.volume%253D120%26rft.issue%253D2%26rft.spage%253D321%26rft.epage%253D332%26rft.atitle%253DHigh%2Bcongruency%2Bof%2BQTL%2Bpositions%2Bfor%2Bheterosis%2Bof%2Bgrain%2Byield%2Bin%2Bthree%2Bcrosses%2Bof%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1209-9%26rft_id%253Dinfo%253Apmid%252F19911156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1209-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19911156&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272803700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.70" data-doi="10.1534/genetics.116.193243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accuracy of genomic prediction in synthetic populations depending on the number of parents, relatedness, and ancestral linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.193243</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.193243%26rft_id%253Dinfo%253Apmid%252F28049710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA1LzEvNDQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.71" data-doi="10.1007/s00122-015-2577-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Forecasting the accuracy of genomic prediction with different selection targets in the training and prediction set as well as truncation selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">2189</span>–<span class="cit-lpage">2201</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2577-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D2189%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2577-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2577-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.72" data-doi="10.1073/pnas.0604635103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Semel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nissenbaum</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menda</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krieger</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Overdominant quantitative trait loci for yield and fitness in tomato.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">12981</span>–<span class="cit-lpage">12986</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0604635103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0604635103%26rft_id%253Dinfo%253Apmid%252F16938842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM1LzEyOTgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1557.73"><div class="cit-metadata unstructured">Shull, G. H., 1908 The composition of a field of maize. J Hered os-4: 296–301.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.74" data-doi="10.1007/s10681-010-0237-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">da Silva Dias</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Impact of improved vegetable cultivars in overcoming food insecurity.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-010-0237-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D176%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-010-0237-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-010-0237-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.75" data-doi="10.2135/cropsci1986.0011183X002600030023x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">O. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Covariance between line per se and testcross performance.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">543</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1986.0011183X002600030023x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSmith%26rft.auinit1%253DO.%2BS.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D540%26rft.epage%253D543%26rft.atitle%253DCovariance%2Bbetween%2BLine%2Bper%2Bse%2Band%2BTestcross%2BPerformance%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1986.0011183X002600030023x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1986.0011183X002600030023x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986C310500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.76" data-doi="10.2134/agronj1942.00021962003400100008x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sprague</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatum</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1942</span> <span class="cit-article-title">General <em>vs.</em> specific combining ability in single crosses of corn.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">932</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj1942.00021962003400100008x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DSprague%26rft.auinit1%253DG.%2BF.%26rft.volume%253D34%26rft.issue%253D10%26rft.spage%253D923%26rft.epage%253D932%26rft.atitle%253DGeneral%2Bvs.%2BSpecific%2BCombining%2BAbility%2Bin%2BSingle%2BCrosses%2Bof%2BCorn%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1942.00021962003400100008x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1942.00021962003400100008x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.77" data-doi="10.2135/cropsci1970.0011183X001000020002x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Estimation of genetic variances using inbred relatives.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1970.0011183X001000020002x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DStuber%26rft.auinit1%253DC.%2BW.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D129%26rft.epage%253D135%26rft.atitle%253DEstimation%2Bof%2BGenetic%2BVariances%2BUsing%2BInbred%2BRelatives%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1970.0011183X001000020002x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1970.0011183X001000020002x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.78" data-doi="10.3168/jds.2014-8210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Gene effects and variances in hybrid populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1279</span>–<span class="cit-lpage">1286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStuber%26rft.auinit1%253DC.%2BW.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D1279%26rft.epage%253D1286%26rft.atitle%253DGENE%2BEFFECTS%2BAND%2BVARIANCES%2BIN%2BHYBRID%2BPOPULATIONS%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-8210%26rft_id%253Dinfo%253Apmid%252F17248353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiI1NC82LzEyNzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.79" data-doi="10.1016/j.cub.2017.05.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tardieu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrera-Bosquet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pridmore</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Plant phenomics, from sensors to knowledge.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">R770</span>–<span class="cit-lpage">R783</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2017.05.055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D27%26rft.spage%253DR770%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2017.05.055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2017.05.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1557.80"><div class="cit-metadata unstructured">Technow, F., 2013 <em>hypred: Simulation of genomic data in applied genetics</em>. R package version 0.4.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.81" data-doi="10.1371/journal.pone.0190271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Parent-progeny imputation from pooled samples for cost-efficient genotyping in plant breeding.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0190271</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0190271</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0190271%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0190271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0190271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.82" data-doi="10.1007/s00122-012-1905-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1905-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1181%26rft.epage%253D1194%26rft.atitle%253DGenomic%2Bprediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Bwith%2Bmodels%2Bincorporating%2Bdominance%2Band%2Bpopulation%2Bspecific%2Bmarker%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1905-8%26rft_id%253Dinfo%253Apmid%252F22733443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1905-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.83" data-doi="10.1534/genetics.114.165860"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipprack</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1343</span>–<span class="cit-lpage">1355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.165860</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165860%26rft_id%253Dinfo%253Apmid%252F24850820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEzNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.84" data-doi="10.1007/s00122-014-2396-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipprack</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Identification of key ancestors of modern germplasm in a breeding program of maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">2545</span>–<span class="cit-lpage">2553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2396-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D2545%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2396-6%26rft_id%253Dinfo%253Apmid%252F25208647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2396-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25208647&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.85" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.86" data-doi="10.1093/jxb/ert333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitford</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleury</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hybrid breeding in wheat: technologies to improve hybrid wheat seed production.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">5411</span>–<span class="cit-lpage">5428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/ert333</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fert333%26rft_id%253Dinfo%253Apmid%252F24179097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/ert333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24179097&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329135600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.87" data-doi="10.1534/g3.112.003699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.003699</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1427%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.003699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.88" data-doi="10.1111/j.1439-0523.2004.00990.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Heterosis performance of yield and fibre quality in F1 and F2 hybrids in upland cotton.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">289</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0523.2004.00990.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D123%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2004.00990.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2004.00990.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.89" data-doi="10.1186/1471-2156-14-85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraft</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection in sugar beet breeding populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-14-85</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-14-85%26rft_id%253Dinfo%253Apmid%252F24047500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-14-85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24047500&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.90" data-doi="10.1534/genetics.113.157032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping quantitative trait loci by controlling polygenic background effects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1209</span>–<span class="cit-lpage">1222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.157032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.157032%26rft_id%253Dinfo%253Apmid%252F24077303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1557.91" data-doi="10.1073/pnas.1514547112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-based establishment of a high-yielding heterotic pattern for hybrid wheat breeding.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">15624</span>–<span class="cit-lpage">15629</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1514547112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1514547112%26rft_id%253Dinfo%253Apmid%252F26663911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUxLzE1NjI0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1557.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1545" title="New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1571" title="Cleaning Genotype Data from Diversity Outbred Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1557" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GNpuxNn_XbY0mWZShN0seuVsr_Yf3s_4z4I2JJwKGQw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1557" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1557" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1557" class="active" data-icon-position="" data-hide-link-title="0">Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493110" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-QJU3ZfUMsjGTl2ZXglwiwxV73RMpQ46v5OUkc063SHA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1557.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1557" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1Os0PnWzGxT1pnj_5HHbM9CvxHwCijeyvV2mG35-xwM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1557" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1557.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493110" id="node493110--2" data-pisa="ggg;9/5/1557" data-pisa-master="ggg;g3.118.200994" data-apath="/ggg/9/5/1557.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2497-3136" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frank Technow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1557-1569;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200994 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493110--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Maize Product Development/Systems and Innovation for Breeding and Seed Products, DuPont Pioneer, Tavistock/Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2497-3136" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frank Technow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frank.technow@pioneer.com" class="" data-icon-position="" data-hide-link-title="0">frank.technow@pioneer.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1557" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1557&amp;title=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1557&amp;title=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1557&amp;title=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1557&amp;text=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1557&amp;title=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1557&amp;t=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1557&amp;title=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1557&amp;title=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493110/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493110" id="node493110--3" data-pisa="ggg;9/5/1557" data-pisa-master="ggg;g3.118.200994" data-apath="/ggg/9/5/1557.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2497-3136" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frank Technow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1557-1569;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200994 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493110--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Maize Product Development/Systems and Innovation for Breeding and Seed Products, DuPont Pioneer, Tavistock/Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2497-3136" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frank Technow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frank.technow@pioneer.com" class="" data-icon-position="" data-hide-link-title="0">frank.technow@pioneer.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1557&amp;rft.epage=1569&amp;rft.atitle=Use%20of%20F2%20Bulks%20in%20Training%20Sets%20for%20Genomic%20Prediction%20of%20Combining%20Ability%20and%20Hybrid%20Performance&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Technow%2C+Frank"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493110/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493110/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493110/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493110/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493110/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493110/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493110/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493110/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493110/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493110/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493110/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493110/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1557" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1557.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1557.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1557.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
