<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1693" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1693.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1693.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1693.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1693/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1693.atom" />
<meta name="HW.pisa" content="ggg;9/5/1693" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction" />
<meta name="DC.Identifier" content="10.1534/g3.119.400132" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Huang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Polyploids are organisms whose genomes consist of more than two complete sets of chromosomes. Both autopolyploids and allopolyploids may display polysomic inheritance. A peculiarity of polysomic inheritance is multivalent formation during meiosis resulting in double-reduction, which occurs when sister chromatid fragments segregate into the same gamete. Double-reduction can result in gametes carrying identical-by-descent alleles and slightly increasing homozygosity. This will cause the genotypic frequencies to deviate from expected values and will thus bias the results of standard population genetic analytical methods used in molecular ecology and selective breeding. In this study, we extend existing double-reduction models to account for any even level of ploidy, and derive the symbolic expressions for genotypic frequencies via two methods. Inbreeding coefficients and heterozygosity under double-reduction and inbreeding are also calculated. Numerical solutions obtained by computer simulations are compared with analytical solutions predicted by the model to validate the model." />
<meta name="DC.Contributor" content="Kang Huang" />
<meta name="DC.Contributor" content="Tongcheng Wang" />
<meta name="DC.Contributor" content="Derek W. Dunn" />
<meta name="DC.Contributor" content="Pei Zhang" />
<meta name="DC.Contributor" content="Xiaoxiao Cao" />
<meta name="DC.Contributor" content="Rucong Liu" />
<meta name="DC.Contributor" content="Baoguo Li" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Polyploids are organisms whose genomes consist of more than two complete sets of chromosomes. Both autopolyploids and allopolyploids may display polysomic inheritance. A peculiarity of polysomic inheritance is multivalent formation during meiosis resulting in double-reduction, which occurs when sister chromatid fragments segregate into the same gamete. Double-reduction can result in gametes carrying identical-by-descent alleles and slightly increasing homozygosity. This will cause the genotypic frequencies to deviate from expected values and will thus bias the results of standard population genetic analytical methods used in molecular ecology and selective breeding. In this study, we extend existing double-reduction models to account for any even level of ploidy, and derive the symbolic expressions for genotypic frequencies via two methods. Inbreeding coefficients and heterozygosity under double-reduction and inbreeding are also calculated. Numerical solutions obtained by computer simulations are compared with analytical solutions predicted by the model to validate the model.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1693" />
<meta name="citation_id" content="9/5/1693" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1693" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1693.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1693.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1693.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400132" />
<meta name="citation_pmid" content="30910817" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1693" />
<meta name="citation_lastpage" content="1706" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kang Huang" />
<meta name="citation_author_institution" content="Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8357-117X" />
<meta name="citation_author" content="Tongcheng Wang" />
<meta name="citation_author_institution" content="Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069" />
<meta name="citation_author" content="Derek W. Dunn" />
<meta name="citation_author_institution" content="Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5909-1224" />
<meta name="citation_author" content="Pei Zhang" />
<meta name="citation_author_institution" content="Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4433-382X" />
<meta name="citation_author" content="Xiaoxiao Cao" />
<meta name="citation_author_institution" content="Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069" />
<meta name="citation_author" content="Rucong Liu" />
<meta name="citation_author_institution" content="Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069" />
<meta name="citation_author" content="Baoguo Li" />
<meta name="citation_author_institution" content="Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069" />
<meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming, China, 650223" />
<meta name="citation_author_email" content="baoguoli@nwu.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7430-3889" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Automat. Contr.;citation_author=H. Akaike;citation_title=A new look at the statistical model identification.;citation_pages=716-723;citation_volume=19;citation_year=1974;citation_doi=10.1109/TAC.1974.1100705" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Barone;citation_author=C. Gebhardt;citation_author=L. Frusciante;citation_title=Heterozygosity in 2n gametes of potato evaluated by RFLP markers.;citation_pages=98-104;citation_volume=91;citation_year=1995;citation_doi=10.1007/BF00220864" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. D. Burow;citation_author=C. E. Simpson;citation_author=J. L. Starr;citation_author=A. H. Paterson;citation_title=Transmission Genetics of Chromatin From a Synthetic Amphidiploid to Cultivated Peanut (Arachis hypogaea L.): Broadening the Gene Pool of a Monophyletic Polyploid Species;citation_pages=823-837;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606556" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. V. Butruille;citation_author=L. S. Boiteux;citation_title=Selection-mutation balance in polysomic tetraploids: impact of double reduction and gametophytic selection on the frequency and subchromosomal localization of deleterious mutations.;citation_volume=97;citation_year=2000;citation_pmid=10823890;citation_doi=10.1073/pnas.100101097" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=L. V. Clark;citation_author=M. Jasieniuk;citation_title=polysat: an R package for polyploid microsatellite analysis.;citation_pages=562-566;citation_volume=11;citation_year=2011;citation_pmid=21481215;citation_doi=10.1111/j.1755-0998.2011.02985.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Cockerham;citation_title=ANALYSES OF GENE FREQUENCIES;citation_pages=679-700;citation_volume=74;citation_year=1973;citation_issue=4;citation_pmid=17248636" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=J. G. Crowley;citation_author=H. Rees;citation_title=Fertility and selection in tetraploid Lolium.;citation_pages=300-308;citation_volume=24;citation_year=1968;citation_doi=10.1007/BF00336197" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=C. D. Darlington;citation_title=Chromosome behaviour and structural hybridity in the Tradescantiae.;citation_pages=207-286;citation_volume=21;citation_year=1929;citation_doi=10.1007/BF02984208" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. David;citation_author=S. Blum;citation_author=M. W. Feldman;citation_author=U. Lavi;citation_author=J. Hillel;citation_title=Recent duplication of the common carp (Cyprinus carpio L.) genome as revealed by analyses of microsatellite loci.;citation_volume=20;citation_year=2003;citation_pmid=12832638;citation_doi=10.1093/molbev/msg173" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=H. N. De Silva;citation_author=A. J. Hall;citation_author=E. Rikkerink;citation_author=M. A. McNeilage;citation_author=L. G. Fraser;citation_title=Estimation of allele frequencies in polyploids under certain patterns of inheritance.;citation_pages=327-334;citation_volume=95;citation_year=2005;citation_issue=4;citation_pmid=16094298;citation_doi=10.1038/sj.hdy.6800728" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Resour;citation_author=L. Excoffier;citation_author=H. E. L. Lischer;citation_title=Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows.;citation_pages=564-567;citation_volume=10;citation_year=2010;citation_issue=3;citation_pmid=21565059;citation_doi=10.1111/j.1755-0998.2010.02847.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Hum. Genet.;citation_author=R. A. Fisher;citation_title=Allowance for double reduction in the calculation of genotype frequencies with polysomic inheritance.;citation_pages=169-171;citation_volume=12;citation_year=1943" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=R. A. Fisher;citation_author=K. Mather;citation_title=The inheritance of style length in Lythrum salicaria.;citation_pages=1-23;citation_volume=12;citation_year=1943;citation_doi=10.1111/j.1469-1809.1943.tb02307.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Geiringer;citation_title=CHROMATID SEGREGATION OF TETRAPLOIDS AND HEXAPLOIDS;citation_pages=665-684;citation_volume=34;citation_year=1949;citation_issue=6;citation_pmid=15393914" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. B. S. Haldane;citation_title=Theoretical genetics of autopolyploids.;citation_pages=359-372;citation_volume=22;citation_year=1930;citation_doi=10.1007/BF02984197" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=O. J. Hardy;citation_title=Population genetics of autopolyploids under a mixed mating model and the estimation of selfing rate.;citation_pages=103-117;citation_volume=16;citation_year=2016;citation_doi=10.1111/1755-0998.12431" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=O. J. Hardy;citation_author=X. Vekemans;citation_title=spagedi: a versatile computer program to analyse spatial genetic structure at the individual or population levels.;citation_pages=618-620;citation_volume=2;citation_year=2002;citation_doi=10.1046/j.1471-8286.2002.00305.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=K. Huang;citation_author=S. T. Guo;citation_author=X. G. Qi;citation_author=P. Zhang;citation_author=B. G. Li;citation_title=A maximum-likelihood estimation of pairwise relatedness for autopolyploids.;citation_pages=133-142;citation_volume=114;citation_year=2015;citation_pmid=25370210;citation_doi=10.1038/hdy.2014.88" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=K. Huang;citation_author=K. Ritland;citation_author=S. T. Guo;citation_author=M. Shattuck;citation_author=B. G. Li;citation_title=A pairwise relatedness estimator for polyploids.;citation_pages=734-744;citation_volume=14;citation_year=2014;citation_pmid=24460904;citation_doi=10.1111/1755-0998.12217" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=A. Jacquard;citation_title=Genetic information given by a relative.;citation_pages=1101-1114;citation_volume=28;citation_year=1972;citation_issue=4;citation_pmid=4648793;citation_doi=10.2307/2528643" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=S. T. Kalinowski;citation_author=M. L. Taper;citation_title=Maximum likelihood estimation of the frequency of null alleles at microsatellite loci.;citation_pages=991-995;citation_volume=7;citation_year=2006;citation_doi=10.1007/s10592-006-9134-9" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. T. Kalinowski;citation_author=M. L. Taper;citation_author=T. C. Marshall;citation_title=Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment.;citation_pages=1099-1106;citation_volume=16;citation_year=2007;citation_issue=5;citation_pmid=17305863;citation_doi=10.1111/j.1365-294X.2007.03089.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. T. Limborg;citation_author=W. A. Larson;citation_author=L. W. Seeb;citation_author=J. E. Seeb;citation_title=Screening of duplicated loci reveals hidden divergence patterns in a complex salmonid genome.;citation_pages=4509-4522;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14201" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=A. Lloyd;citation_author=K. Bomblies;citation_title=Meiosis in autopolyploid and allopolyploid Arabidopsis.;citation_pages=116-122;citation_volume=30;citation_year=2016;citation_pmid=26950252;citation_doi=10.1016/j.pbi.2016.02.004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. W. Luo;citation_author=Z. E. Zhang;citation_author=R. M. Zhang;citation_author=M. Pandey;citation_author=O. Gailing;citation_title=Modeling population genetic data in autotetraploid species.;citation_volume=172;citation_year=2006;citation_pmid=16172506;citation_doi=10.1534/genetics.105.044974" />
<meta name="citation_reference" content="citation_title=Genetics and analysis of quantitative traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=A. Madlung;citation_title=Polyploidy and its effect on evolutionary success: old questions revisited with new tools.;citation_pages=99-104;citation_volume=110;citation_year=2013;citation_pmid=23149459;citation_doi=10.1038/hdy.2012.79" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=T. C. Marshall;citation_author=J. Slate;citation_author=L. E. B. Kruuk;citation_author=J. M. Pemberton;citation_title=Statistical confidence for likelihood-based paternity inference in natural populations.;citation_pages=639-655;citation_volume=7;citation_year=1998;citation_issue=5;citation_pmid=9633105;citation_doi=10.1046/j.1365-294x.1998.00374.x" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=K. Mather;citation_title=Reductional and equational separation of the chromosomes in bivalents and multivalents.;citation_pages=53-78;citation_volume=30;citation_year=1935;citation_doi=10.1007/BF02982205" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=P. G. Meirmans;citation_author=P. H. van Tienderen;citation_title=genotype and genodive: two programs for the analysis of genetic diversity of asexual organisms.;citation_pages=792-794;citation_volume=4;citation_year=2004;citation_doi=10.1111/j.1471-8286.2004.00770.x" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=H. J. Muller;citation_title=A new mode of segregation in Gregory’s tetraploid primulas.;citation_pages=508-512;citation_volume=48;citation_year=1914;citation_doi=10.1086/279426" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. Nei;citation_title=Genetic distance between populations.;citation_pages=283-292;citation_volume=106;citation_year=1972;citation_doi=10.1086/282771" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=M. Nei;citation_title=F-statistics and analysis of gene diversity in subdivided populations.;citation_pages=225-233;citation_volume=41;citation_year=1977;citation_issue=2;citation_pmid=596830;citation_doi=10.1111/j.1469-1809.1977.tb01918.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. L. Odong;citation_author=J. van Heerwaarden;citation_author=J. Jansen;citation_author=T. J. van Hintum;citation_author=F. A. van Eeuwijk;citation_title=Determination of genetic structure of germplasm collections: are traditional hierarchical clustering methods appropriate for molecular marker data?;citation_pages=195-205;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21472410;citation_doi=10.1007/s00122-011-1576-x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. P. Otto;citation_title=The evolutionary consequences of polyploidy.;citation_pages=452-462;citation_volume=131;citation_year=2007;citation_issue=3;citation_pmid=17981114;citation_doi=10.1016/j.cell.2007.10.022" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=C. Parisod;citation_author=R. Holderegger;citation_author=C. Brochmann;citation_title=Evolutionary consequences of autopolyploidy.;citation_pages=5-17;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=20070540;citation_doi=10.1111/j.1469-8137.2009.03142.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Peakall;citation_author=P. E. Smouse;citation_title=genalex 6.5: genetic analysis in Excel. Population genetic software for teaching and researchan update.;citation_volume=28;citation_year=2012;citation_pmid=22820204;citation_doi=10.1093/bioinformatics/bts460" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=L. Qu;citation_author=J. Hancock;citation_author=J. Whallon;citation_title=Evolution in an autopolyploid group displaying predominantly bivalent pairing at meiosis: genomic similarity of diploid Vaccinium darrowi and autotetraploid V. corymbosum (Ericaceae);citation_pages=698-698;citation_volume=85;citation_year=1998;citation_issue=5;citation_pmid=21684952;citation_doi=10.2307/2446540" />
<meta name="citation_reference" content="citation_title=A glossary of genetics and cytogenetics: Classical and molecular;citation_author=R. Rieger;citation_author=A. Michaelis;citation_author=M. M. Green;citation_year=1968;citation_doi=10.1007/978-3-662-01012-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Robertson;citation_author=W. G. Hill;citation_title=DEVIATIONS FROM HARDY-WEINBERG PROPORTIONS: SAMPLING VARIANCES AND USE IN ESTIMATION OF INBREEDING COEFFICIENTS;citation_pages=703-718;citation_volume=107;citation_year=1984;citation_issue=4;citation_pmid=6745643" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Resour;citation_author=F. Rousset;citation_title=genepop'007: a complete re-implementation of the genepop software for Windows and Linux.;citation_pages=103-106;citation_volume=8;citation_year=2008;citation_issue=1;citation_pmid=21585727;citation_doi=10.1111/j.1471-8286.2007.01931.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Schwarz;citation_title=Estimating the dimension of a model.;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. M. Session;citation_author=Y. Uno;citation_author=T. Kwon;citation_author=J. A. Chapman;citation_author=A. Toyoda;citation_title=Genome evolution in the allotetraploid frog Xenopus laevis.;citation_pages=336-343;citation_volume=538;citation_year=2016;citation_pmid=27762356;citation_doi=10.1038/nature19840" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Biol.;citation_author=V. G. Solís Neffa;citation_author=A. Fernández;citation_title=Chromosome studies in Turnera (Turneraceae).;citation_pages=925-930;citation_volume=23;citation_year=2000;citation_doi=10.1590/S1415-47572000000400037" />
<meta name="citation_reference" content="citation_title=Variation and Evolution in Plants;citation_author=G. L. Stebbins;citation_year=1950" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Stift;citation_author=C. Berenos;citation_author=P. Kuperus;citation_author=P. H. van Tienderen;citation_title=Segregation models for disomic, tetrasomic and intermediate inheritance in tetraploids: a general procedure applied to rorippa (yellow cress) microsatellite data.;citation_volume=179;citation_year=2008;citation_pmid=18689891;citation_doi=10.1534/genetics.107.085027" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=M. S. Swaminathan;citation_author=K. Sulbha;citation_title=Multivalent frequency and seed fertility in raw and evolved tetraploids of Brassica campestris var. toria.;citation_pages=385-392;citation_volume=90;citation_year=1959;citation_doi=10.1007/BF00888813" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.1111/j.1558-5646.1984.tb05657.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. L. Wu;citation_author=M. Gallo-Meagher;citation_author=R. C. Littell;citation_author=Z. B. Zeng;citation_title=A General Polyploid Model for Analyzing Gene Segregation in Outcrossing Tetraploid Species;citation_pages=869-882;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606559" />
<meta name="citation_reference" content="citation_title=Principles of Statistical Genomics;citation_author=S. Z. Xu;citation_year=2013;citation_doi=10.1007/978-0-387-70807-2" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mol. Sci.;citation_author=H. Zhou;citation_author=T. Y. Ma;citation_author=R. Zhang;citation_author=Q. Z. Xu;citation_author=F. Shen;citation_title=Analysis of different ploidy and parent-offspring genomic DNA methylation in the loach Misgurnus anguillicaudatus.;citation_pages=1299;citation_volume=17;citation_year=2016;citation_doi=10.3390/ijms17081299" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Polyploids are organisms whose genomes consist of more than two complete sets of chromosomes. Both autopolyploids and allopolyploids may display polysomic inheritance. A peculiarity of polysomic inheritance is multivalent formation during meiosis resulting in double-reduction, which occurs when sister chromatid fragments segregate into the same gamete. Double-reduction can result in gametes carrying identical-by-descent alleles and slightly increasing homozygosity. This will cause the genotypic frequencies to deviate from expected values and will thus bias the results of standard population genetic analytical methods used in molecular ecology and selective breeding. In this study, we extend existing double-reduction models to account for any even level of ploidy, and derive the symbolic expressions for genotypic frequencies via two methods. Inbreeding coefficients and heterozygosity under double-reduction and inbreeding are also calculated. Numerical solutions obtained by computer simulations are compared with analytical solutions predicted by the model to validate the model." />
<meta name="og-title" property="og:title" content="Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1693" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Polyploids are organisms whose genomes consist of more than two complete sets of chromosomes. Both autopolyploids and allopolyploids may display polysomic inheritance. A peculiarity of polysomic inheritance is multivalent formation during meiosis resulting in double-reduction, which occurs when sister chromatid fragments segregate into the same gamete. Double-reduction can result in gametes carrying identical-by-descent alleles and slightly increasing homozygosity. This will cause the genotypic frequencies to deviate from expected values and will thus bias the results of standard population genetic analytical methods used in molecular ecology and selective breeding. In this study, we extend existing double-reduction models to account for any even level of ploidy, and derive the symbolic expressions for genotypic frequencies via two methods. Inbreeding coefficients and heterozygosity under double-reduction and inbreeding are also calculated. Numerical solutions obtained by computer simulations are compared with analytical solutions predicted by the model to validate the model." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__HLkFhTtFYKr9UM35ZkeLEJf0AA-RJsk2jDIDlTr7P6E__db-Yi0lrH1NHADEOMB-20Dvr8Qs6F9pwOIMQF9BHr1w__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SmWd3hoqW7zKLjIlnYxUggfumoezgT-f-wnKcFiAUh8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492863","apath":"\/ggg\/9\/5\/1693.atom","pisa":"ggg;9\/5\/1693","ac":{"\/ggg\/9\/5\/1693.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1693.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1693"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1693"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1693":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bvxXkUuO4FQoNnT0hy7GpAaw0y1f+6ooQz5hEZsxW8aPcRrJ0SsCXMtfnm0nR\\\\\\\/YPTFjYoGjr\\\\\\\/T6bmk5dnoTU5Xsgbm7vOlNanUB0Mr6cFUbSO9XvhZDFa8tcXc8vaZOJuoPeyPWXconC6WdZfcLos2z7aveIlLf\\\\\\\/eZKM3bwX3zuIsjC\\\\\\\/C0raJjeQGv5K\\\\\\\/lGsdpOuHnWM+grihEaN\\\\\\\/HJ90vshJzqJKJwy3Nu25\\\\\\\/q5rhNDZjMLMwR+kyUdwpX0Thjf7NAEcadhKgA+eU2NHl4TjZiPOttVvgKB4bNN5PLAbcoBpgbb3gI9KY97WEFZqw4AqAhCdmzRyueUxNqCIi+dUuhA3J5nJt6vyFy8o54ZXslCAg74sGpOKFWMQ4wyF6B9BdNcq1kAGzwNhe2Z0vmzQbjkfi6NkuaTET+N45BCD2QDN3MewulD+Eji9+hvfQ7bPSqK+iPAoY7CDHhTCgj69ecIsk1kgIsW6Bp7Hu6PGyAhEWGwEcp3BRtbcl6Xa3cpCCstp4t4i688id96XdaNDWjsG6H0PRPOjyg0R7hJJzxIG+ulOnO4LHrQnpSONheIZsftPP2V2ozl5WC0c6K1h4yHIoZ8L0mPtQWBlLb0KaVOddTmjrk4oVTgbKFBdURKdjqRe7Loo5fARyDv3M7CKMlsAUDdKSD4fOsA16bWlEjEGJVF6+WTLWGj88GxfcqHvhlV+9RUaMQ5aWVFihNFWlUsB0nTX0HfBWGaKfNBTOG7bVIgKPSlK\\\\\\\/LKz0tfiinF9r\\\\\\\/qJKHBiZJH3cUZxNniAO9WRfQa5leDqGL1gQxJNVabp7a6C0G3yRZ+dN0V0SZU9p\\\\\\\/4WSKUtpSJGNYdyAwZnQ1QQ9wYsr1z0LeYQoSB+5o\\\\\\\/7\\\\\\\/j1zyF1hj3Oo7VoJxteSTHVWrkn\\\\\\\/qPTj\\\\\\\/71W8YxOqtXtgjl0E0ZPHuz802RUyJU8RcUKMYiL\\\\\\\/MDSefWjGyGE1Fze0tBn2JUcIv70mFIV2H1ZTP7unmi6XuIOdBPaLCuSaHRNoNHzgSCsKeVrhAFmvVEEf1b9QgJS1pYUvo3FaRtzdAAw6GPsR4f6qD\\\\\\\/S9Y39i7d\\\\\\\/uF1nBmEvxEKVdjKT94mPDmYT6vYWQGfaVcbMuHPgiq5CS0OL6e0vtaPpshRbWn86tnE\\\\\\\/+465HUg3y\\\\\\\/TuY8yKz2QdLmA2z6P\\\\\\\/Efa8t9llgCn8wxxYg1p3hTEVru6XeAt4R2\\\\\\\/onTY9wCtXUvX85x7bB0R+4fDwpLY0phFBM9mpeFxbNnUg48oO+zFGQA8TXs+R8iTwhsrJ+Xd3zwZ\\\u0022,\\\u0022iv\\\u0022:\\\u0022V48lijeSxHlQpKPnJ2FPbg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220cbd8cf3d66be585160c45423a66e84b8a36e086332a7447b1f49a4b612b8759\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tqHgVuzCy5t5eKdiBAeX3gm7\\\\\\\/Dbj6nbjWk6Hckmgjc0t1n3GJVzhyJK3OuZu\\\\\\\/U6u\\\\\\\/gkYOmc7G4ysZgJOF8n6iD4Epy2IBr5Pi8lm3h6Lh2FWUJ2564cwjLJxuSY7nVUTN20yCdJVVpDbxx2n8uPzHvVG\\\\\\\/+hpSl8Q9vEJEq4qXPyl07mI8obxLuOFx4iSC7gRhinZzMgrRvFvRvg6gYDmAsZJm0+bcm6wsxZd\\\\\\\/vf4etc4xeDllj6oKzmKbXFYbK5SGwdgJXkohvCqo6Ha+\\\\\\\/enINC3bae6NsQNMYRGfwQbQjEEDmPvMc+yBvpxWStZazdEzZGi\\\\\\\/CYRwZfzg6GVBLUmCWoe9JwRg6\\\\\\\/owCI330\\\\\\\/bn+lP6daGDJ2NSjd9jbLygs+n5NLyodPPNmWlCMOIJW5neu5M1EivuJYdmoYwq2pH3K0evWxk+BertFkz2GcPw+JsiD2tdS8D4KsXuRZE2aGPgnOLs8q1gB4EoRLS0j33oKGOIYXOmstc\\\\\\\/SOKVXkgyyyIEMy8Ggj5it5TKlx9Ee0d4GdsWYNvrL37GRZH1papBrLG6\\\\\\\/HpIL2E\\\\\\\/Gupt4esKfRB9n70CRIiSzGCgTC5QJXfWDffyWtW2GB6bi5wh8zET6vA1UV8caXI\\\\\\\/AI62iyV88j7J+QQxOJxzNfFHO76Pru3JLYJQClenVZbcs+wHq0yiDh+ojwWLalz6zEJYYmJTlXlojMNMgGNX9Wvrkycv0PtqCjk7\\\\\\\/KKvDjtKCUFh7xNMr8FR426jFHh+My329JyibEdMedmpRgBbRmvjxmrBjzaKWCUW6XVzamTAHfFVGZBctfenPR1z0O4A\\\\\\\/i+McqNL+aKm9yTlCxKtc7KrotLLZaiHoXP\\\\\\\/pGVPFzZgCE38pSevrC\\\\\\\/vk4EZoOaVCFXi9FCKjY38z3oGRbbNOGgeL+4txo\\\\\\\/LM2DSDV680Rs5KEKYhxy96ZTv+EQ9Lov+YGzSDLQg2EulFtfKREGJFiWrmppXPLTLIsizvpmH4MaKip4UoVByOR1UY+Tm6lokUDx\\\\\\\/sNHFOCnC6gwoxx9qVeXBCDMkPehPf19w+c9fZEFHn5YUprmJFeubkvtApMF9+AG+wPHikTVfPswwy8WOLbQm8XCFu\\\\\\\/bY7mRpr67mJOLJHUDfG\\\\\\\/ODkfGqJi1rM2BprHtn57O\\\\\\\/ILlbuOBn1M8eTtaaflxyruUn36x\\\\\\\/uLY6oysnHLvun\\\\\\\/CCqIXA4io4Hs+wtG65OoUeouzO1WHgmmh5Tgtx1JSUz\\\\\\\/BGEgH1Uby51KfEGpQP5G37xo23ST\\\\\\\/KbinEYep2DkEDYonNPlIlAiGYbreueSs3ICF3lsy5PWXNQ21AlBXZHLyRDhQkAvUF2BE83CB9N\\\\\\\/dl0XvWHIYIEG3ve4shEikDsf\\\\\\\/VE1aWKSyiqT2MbiOx\\\\\\\/9njor3+JEvENZAbjuzQPtisbf1YXlrWJsUMcwBOyhNobxFfBDqLsamtEJKoMjJw4Ar\\\\\\\/8W6fDXULhqJORn6GjR7ZaIHRh7dWSuR4SNdz67JPuDYcHx+gXS8HiCYXxpYDiaCd8JLrCsL+Jy9V9YAro8+aMFoWkILi+4E6pcqir+FrIf61JsWLZ63UAd9t2zd0uUdV72XeA412KA0iLPBUPv2ARQ6HuysHnzANEiuaAMIgvvpnHXziS0Mfysh31J+m+kxIX4d\\\\\\\/LhriLUgj\\\\\\\/lbSoxRZjdRqHj9r76DKqec3CxjP+ZDhfcQuGy72uRLwY54ZRfi5kjDEvCc3Sfr\\\\\\\/utTyRg2mq7P8q76edJWX68ycHX6lqfUAzjtM+w3UAGDPGlesJfdfngRKGfA7Z4q79A0aIWiLalVtEG8kWDn6Vn0Dq46QHYXKhFvwrIOwU5Z5IDeuNEYQQUsV0jy1Qf6T+ZFlxFqxIDEpglIvBnK3lsEX+chJ6H3fusvMrbra5VQ8ca3VSP7Ay3R17r9DwzbsX7RNilNijL6ZlJPjEFVtGl9rBxV161U1VivVzeGqkeo3DLvm8lS2APTzAf\\\\\\\/6sl3vt0\\\\\\\/GkWyIpo05XCi6WV9OG859Zy7zEYXH+O3Lxj0\\\\\\\/9j+JS7bxU+8oAP3Q170ofHisjERbjTwjxs4Z5m4ehmvtUhdhw89JMfWgjZI8mtZdcj9UtbPWYhgtXZU1o31TJaNJhmG7xWlww7IMbzI7xGBfNG64HxWzO38RqIuImm+DNnGdbfhlRgunwXpnIiFUtaBApHK8D3qvqPOWJqFduGkYXHXdq5nhD2CBH7nPsftowBTbi5KKo0GUIKOOHJwGF9bG0mWhG6IfLRabSFuIz+9uj3TdMYa6s\\\\\\\/btgjVogeRdSeXisum7\\\\\\\/BEce1QnisivJtcrJtdhftARUnQqXpIvHvuLOJ6LggpfDdK05VQCflCg84l8zWzpIQAkpzcfmdpfGlqBVnM2fuDLB1EbESorbenkk21Kglv8T6g4j\\\\\\\/McG5sowsm7qlcG221gVBwFBqawpdQ6hxXVw4rC8+Phl6luCSGlcU+k84IGgOMC15WMUKzI\\\\\\\/5SG0RGf0wFvoMI8K0gA4HSoGYluTYzPEfOnVu4IhcoRl92w67RWXBNbeIuWtkbFj2ODKWvv1uQXjaQxe\\\\\\\/dX9p2czAEAKqmXTTYq\\\\\\\/Ify7v8ZbpvPCwiPUPVwgPJqnZjFJ\\\\\\\/1qeTisrX4enJXBpmZ9fURk12oLfxl8vNeFd1ZYOE592qgS\\\\\\\/tEGZvvcpty7cEkIKrfuNKd6HBT\\\\\\\/UYmYh4cEyeKhSUg++fDObisJq6D8GqbVbyrtw4F6wYBjS61IigdT2Lh3ENbi+ElGbBMS3\\\\\\\/xZtZmeoGsgIqIw5IpGrNOPaFC27ipr7OtGYQYAFm5mSFmYOpZXz\\\\\\\/Cr0wTZxKQ8tIP7ksVtdZXpEI2Og4zqD6AR2Wvnb9NOtsTIB7hSo6Ckk5b1HNoJKwHln\\\\\\\/8nnhcZPOX1lX549S\\\\\\\/FCJg7C2RBjgnxqII7eefwyagghjlj6FToVo+w2QhXf8FZlbRU5IOY4VvkcBwRGkOl3KrKtE\\\\\\\/LitJdKo\\\\\\\/kSGF1EPlXGGVdHDmHdTxzFqgS4SPMWFxwidIJEsGIOZYubeXEyXHdY8x7AlzTdvSMzZdNpHpLOOnaYp4FC3WeS9XcJgWFwY153xREhllfd7OwK7u6VuA641cAXhdBn7u4bHwg30nRwveZA4TUjqC3ih8\\\\\\\/49rSTPpdZAmNyrSdXsxr1Mux3M1bhJ5KrfTy8LNHlACiSYDw33tel6\\\\\\\/nOC\\\\\\\/jtSt\\\\\\\/dsf6SCrEO\\\\\\\/3zh3TAj48tIjZme8KmzzPpVxRZNafNvLRkAWtzBymwvM2FpegEKRKsodaiU6JacYakXzmwNSVTSTBgSbq0BwbUVLlpb36kzCoyD1+xuphJnQ04nqGdOoEhbHaoYN4xfDCyRv7JPW37pdntPcbT6Xgp\\\\\\\/tp0J\\\\\\\/CfTSHV4brVvr7or76k+3HhK5OyMUPGsyI0QR1N80zpqU3fPViWegpplOA2TQg6voZb87OxPKPsrmJu8LxqGSl9meH29cz06uP+\\\\\\\/sFqfLhCXAvM+sP6tcY6iR0wkVK2\\\\\\\/7LKiQMo5lWklLMVFuavy289ZBWG09O3wnPCEApjz77SFHPLdxiV0dnWPaXUx4lABiXr764aB05fgtDXT0Mjmw3RCg6hwCkoJY0jslowNIKdXAMC\\\\\\\/PPmmUi7p3NCEsqaTkSifAlhsu2SpgzPjagpKkVg8Iwr+z6oX2pXkiJh5RQnfiOt\\\\\\\/yn\\\\\\\/utEsJdqjbml4v8bu\\\\\\\/3ZpXk2fSyIh\\\\\\\/URnrwA3V0SJokZpu0Q3VUO\\\\\\\/MgMQFzQ\\\\\\\/B1x1UVhlz4RKSPdKpWvBKsZA+3shGTmxEqJGP40LLuNNJzTM4h9np1em2qyzxC74EvBKfyGITOzicX1HCmbXY1ANlKsdP1gqaJ9ZFdvRo9SaafnPf1Hg7vRAyDYwEu5wKmPzfd55XGpW7gsmbaNwJOdh8WM118dxdRnagLMZL5+gTXKDLGe1EFw25teN5sywvHFx928sb8ljZ7moJdt8mzkJ+vOThP09N98ijt3OBKnTCCsKH\\\\\\\/Rw6FzN+H08kH12w9DDyZZfD\\\\\\\/f4tV19gdPWa3P3m5OuW86XhNzCl5pjBp9uQM9jJst8IJ0EKdG+tXRASWwHCrv7cmXO++BWg72IhPDIAD1gQLCgxTMyCAZ3v54lk8PLc2hrH1LXXGgd1VXteBPgJBkwzn+ss1J2S2gwz8jlNYUJGQjYc5U2nUpBdPNg1yG\\\\\\\/KbKoq3W5yH5mKJfY6DNyCLPrhdm+9shlqoYXROGVMJBXCkXe8cW68UvE7lidEyo21hZW4vp1S4vQeNTcNKLHIiQ3Wem5hLGTNYRHouv\\\\\\\/z+l+yTVaRm\\\\\\\/UWQe1S8Qn21s1zEW+dNUSDLtn\\\\\\\/+ok136\\\\\\\/v3iItyvr\\\\\\\/\\\\\\\/UMCeJisaR7A1pLSdnsVaP9lXFDc+MpEGw5JhU10U8+o+Can2y+k3qsyCVo6rMJCi5GaTfO1wWZlSKT5XjaCWfLIOM9bRwW61sndA8B5XT\\\\\\\/ZZFHsecQH+CeYeJQ97ECEB306jhi\\\\\\\/oaiYCUb9hDIR5bsHjW+8Q\\\\\\\/CNNJb9GGESQ+YWcRMetwxwz1q1Pf2+hSJgnR7WqJrSJ1lfRyt8DK6OsSq1VhgghO2mZZquZprm1RcYZnEwV4MU6xTcJrDZOQCzzk5vcWxGWoquJZPH+pR95xGE7lvxb8d7RRUusyu2i2Q93IE0FOkzH6Hr8MzFV4OIS9Fjz2zCuap0dTl9gOpphrnlULfAoqNdutqJeV5g3oKlK41KzwWO8AIvqMkDhqtTvAAwvUC5gnWHiKaC7\\\\\\\/mQ1+KmrUBhS9djZpbtMFEvosNKrebWmBwkT8P6t39QCPeFMRS9vq3a\\\\\\\/+C4a6M3kx7EB+2dIUFlBx8VJSgAAXTjovV3b3d7wN+OsWPuuk3FwhiofSkEkXwVIB8TcianKlCl0ToAZNY+PvqM13ycKOxtGB\\\\\\\/2yioYQMWYjLI7B8LRNYlb5vVMeJ01wArSYdBhBKQssQpE9Iv4wAN9\\\\\\\/c3aQTiAvzrO3DCvaSbaAePhLnGfbgamCKndjzAkpYgiGb32oGFqqyT3TdkdXyqraLetkClWxCBUMjEbmHqBmmBwI2l5qgW5aj2OXp1m0tVWjW5ait910jLJufokdcuue4lyCYZeYustXero75FlnmhECdnNa2SZ4pOdAjgC1CgcKE8fkQ1wB8dqMFOfpug08c5IOVMP1bqo5kasKEvtt0uSo9EIgFubBHZkgttYz0Wvmseuj7K9I55hk4BUTBGVss4zsPkbYtb7iog234BQiaC+S\\\\\\\/+0Td6gav7MzAck0n2C3QgwvrBnqQSk54sEtd3bWh1hZOSdSxxhiJw7OUw0KUC\\\\\\\/LvYg1xZOpBVuFo0XSjXYla5AEZtf1VCp3olpepJ1VekNp\\\\\\\/nq9lmwOGMOUU45RDRlu\\\\\\\/NAAHCF7KrkGdTmkQTmbXaruUgcOsDA8kkoOoBw80km\\\\\\\/+7Ft8ztKKJRwvzGHIbvtCFYsWW1hZS1Lv4QSmnV+N9TODSAgMcaF7CBzFRxeO3DVp+bqsuYLGm82\\\\\\\/lahpiXrJFXPrgRkQk86YZEvJ4BZbawroEBelYhChAoNMaAu03hCJeGF0o80CjpWProyP1JE0rFmSQ5DDRo4F7zPgMrYcBeqIGEg8Xs\\\\\\\/L7XNVL5lxocGNkPYePVbZIi\\\\\\\/9eUMmvCGFeqmicMZWKk51dMFClXDLifUcs+1xD5wUKADhHrC5kPNByj3xBUz0nG+etxJKZXcp5UZTetysRlDhC\\\\\\\/z4AFF6Adl01h\\\\\\\/hMKNB61GnT9kMMm2ly42f+SvG4QTkKn\\\\\\\/GZnVAzV2ip9HBHBConr9Rm0c8erRPChWcuTqU6yks+zTwRDnwh1AaHE2hP4f3xwMG9jGAhEpplIRwW72WWQbaASL4tLL5yzwFNgv9ZoQ61DXc3XNXBnWh1sbtspJNSwL1\\\\\\\/kmuRBwJ\\\\\\\/Qb6nxAxWbjkrhAkF1mUk0rJLgDNU+dbhSfF20P70I\\\\\\\/fnO9oEquk6mN8CwHWtsfg81rKf4WxY+aY8uUAKsRf7AEp3tKAk8FzEc9Chvr1bA0NbzrlMR3e0zv\\\\\\\/hm\\\\\\\/cwfIvUup3+N2hHe0TWAmB8GOLrvo+FT6Zh9jrjNWFfKdyq7ZrlAkbhTyGGICPVOSiRLGWusOU7aMJjMNc4j9LMBZs6jLPRz2Mm7ZwhkazoRISw+0UXGiAAOQDPKXfZPx1Oojghv8Sxi+fZW3+H\\\\\\\/VUIPKen6gywFvhEtFZOR4047MV3IXOa170Tfu+SrAm3VLZsimhdLIPZW37bfhnQFgd8Acat8IVDPxC5DX4TnwOhP624aFg\\\\\\\/3Etwh2a6uN4d8nGkHdzJoHofYC\\\\\\\/dEgbpb0Xm5GNC8GWst7BizBGCqUi0Xc483pdGUlgoklsb\\\\\\\/z87+m0EPKasuQHgGuH0M7ViEmm6X6US0KcaoIWZUSIm1VS8Ix2Oy6cEe55FUon\\\\\\\/mfSbv6R8UFOtC8LdfQmKGQDj2tFRT+v5vIeGlFYzBTkcyGPBwg3AleP\\\\\\\/bzW7DDE2KnlrsrPo55D639ekNy8a7Aex0T2raV+cdorRQvDiI5d6Nxaj29ALBLVJLPNu472ZuuORKtp8yYGhV2T3\\\\\\\/o4d38ijPlvsx+dk34ZSY76WN9sqBOGSLSDZA2BLiQXVXR8SyU0bVETZo490x+yoVqtC\\\\\\\/IZujnXQNPtZNOZCgakp3eX2oeurxv\\\\\\\/JuqT8k8nlFPt2fxl+qMDD+IqtotfepCRjHasxAVYkMnM1rBfBeqhjIiTWzThmDr44q\\\\\\\/zIFbVK\\\\\\\/b1G3slXQGk+Ob11LHDkrhK9vTZbe7sDcGHZa9bvlyX8echPyX7S516ibqq1HIAOQNFlbe3YR9aV3DO3CCoZctV44AXF\\\\\\\/vlASPgM3OWgHrn7cGnOOEHfLAEDxddi50VA1ILJMpRdNnYltXRjE2maioI6mmdffYV7GxaaSEejsFHHa1hQYr\\\\\\\/O+pGqa3uAUg5zHrZTQbI2qUOUUncmFNWcrHsKZf6g2yEu+J8YLLUgcqN6RjggknNUSt\\\\\\\/0KjsIo96yz02YrfCuWNDOmqI39\\\\\\\/sU3w2jqOvLjEIn4gcLBH3rLOVJwB\\\\\\\/ncopoCxbb4uKE8zsQbA6uKK\\\\\\\/PQjL58\\\\\\\/dN6AjL\\\\\\\/8qMIAwYEfd\\\\\\\/88QidBUK+zXrPG4FRuIvti6xoG9SX8cbpY5xoWc3VmnKlC2G7EgkHuRzgsgYvXLasFiDqtm5Y4gY9LgXnwc8Il\\\\\\\/DLy3Syj+CbRX\\\\\\\/Bcu7F2KAVGtRkJjg\\\\\\\/lXav04I1QFCvyCt13\\\\\\\/Nl+EMOoG4ZqZkKwcWsQjgZBziC+8knXyPlytMzgqkBkjkKTw2kG0jYcPW7ZCenWs+Zqj45hmDI8xo3wGvRLx9IU2VnASwDzagzWhJbNJrpjbhwnjecIo175aEmYvGF\\\\\\\/S7TiuEcVgGccRaOyi4Jk7cOzYPC1nUWlPc7OROMN70UgkvEUFsZM5VCJnGx\\\\\\\/LnHQruwkHwbSLizwHDQyRjW3S5WNHZ5\\\\\\\/ZTIuYgKb\\\\\\\/7xDCdRczr+FrxV8GqjDhlG95KZV0+C5L3myjKxktOQLSEQIPYInuICDzOT2dRY6YV+jb1PlUPTu4WQfLJdbmfKV9csNKNs0HpjsYKtDHoo+CchzCMwjWc6spF9vcNmSicQG499Yh1WGdcmASC94f\\\\\\\/XlZgnM0VJkEBYFk2TnMeZRTFZ\\\\\\\/Ug9m3Hv57xhEPpTXgTFZCBD++fxhlrQhWZ2P5Qt1lHE92mIHB4OY6XxI7BX9rIXKMy7Tuqt+UYVbY7coP3rBvt+GWgwkUetDc7cRv0mqqMcMmyNy3gz9UNlnDLoAwnNdjN+6MMqXjt9XRvCMRvlR0cCHcGtu1eBBw2HAnrJTFUginXuhZScMXaD\\\\\\\/uqhFPQBV3Zn1wzZtiWwPM\\\\\\\/ahmXs3jX+VOQ4A5NSNWa5SF4JMhuyBR11lDXOPgGDDR2khAAOo5ybA9BONl7L\\\\\\\/5PsIkrFcI\\\\\\\/QnRTAW7n3s+M+v09ojpf28x4o26wo6wEdRUrydfpS9KvB6MKxgeap0qzsExtHgohdMzAAmqUA7USAyHlE+jWWRAPIjggZ4usJi5koRoC4nA0Hm+T0uX0ngvElvgPAc\\\\\\\/qCwATPeQzWBhhFfAv0XYG91+5\\\\\\\/BpwL3q8fum53iG\\\\\\\/tfQk4rI41mKvq2slMsmoTfcv5JzWAYBirX4qJhV5kF2Ca2EiD+DQxUWHZfnCnVIK8DaxRVekQVAPyA9Yn7NQ7crz0+yYZp+8al4hYJQL1stEG9e4GrgTsVGGIZNRqnwa5IK4XO\\\\\\\/IqohVi4Clkoe43ry12ZKjLLl6I1Fqm4lIOjaPhjCwK\\\\\\\/ZUGIANOn0KJjv59Mjfy1B\\\\\\\/o3JccJH6xUOVn13PI\\\\\\\/HaoBXigpH5adAIACujKKQwhg5SpCstgS47q9eZ4eS4DPi4EqDSwlivs3aJUzuB23BBwpV9bxf8NNfTiWBNi\\\\\\\/S1mz3qIV6UrGCG8ANq8ntj24vV8CUONpUlqj2ycsv24lLY7UTAxIffIRfE66kCDQP7gA3Aj5eZgnDoEr2FD2tOKf5exvC9vmHAfnzKz8cPyAEyFYfJqzsXF372A2OWcjuLUEDf6KgNmqf+n66JDnuiYwlYWR406d+TRipFK1LhM0ugB1B2YmGHoyG8p2sj9x4j9+PmAbLoWV9OQXe1nn7j69ZUN4LGMotsEMyg+wpzaMeBImcsslbAcStXWyDrTc\\\\\\\/LkSa6HNtjDJLXSb99EtXeWJ\\\\\\\/mT0rbpI0gM9421fA3AIKJskHsonweU7EOP0ZckPt7WAyxAJ02\\\\\\\/r0gY2GEc0Ac3\\\\\\\/Cp1IX61JunEhNGXDumVdcxxyoAqXSCNdHJgs9alNQs2gQnJfCFE2Vm1QtbqGx+00Y0hN4uAzsYy1rlKRZGGfoAz7aRMZOaLO0xbGxyMGxT9FNhCZ\\\\\\\/vM3Cv0GuCl5nEOGgvxdS3M\\\\\\\/Z\\\\\\\/7NMYhC1Z4Z0fT59Qt3cU4AFPgY6d2j\\\\\\\/YbxXL\\\\\\\/Zlp2Ef4872jAnM19ON9E076+M+JkkTvaAH\\\\\\\/QrxiFZ0\\\\\\\/eDZVGN67faaiAQDso3vnXyjjMOBJhzvPaT6KHv08+SjufYjLobvA\\\\\\\/4hoSY2cBxKS9ug5uATtJIHsTWKM2ngg2Z5AtXnEjnCZ0ADEPNbSb5XbFPJ6b9qOynBK0E\\\\\\\/+faPAxiQ99AyHgslT12+yULTtN9aQ2\\\\\\\/DV3LE88dSJpaj9xRJb9INCsanaSgWhZ2ewya08emPX6E3z9I2sWviCIRMasqYw5DFxNrvhPCo8b8ghQSQjikFtvB1hHDXRoVtGvkCEgHXp9iJdfbDfji423lDX0dIhTaPklDNPY1IuHbIVQxhpuHEbyU9My\\\\\\\/xgztztE\\\\\\\/XQp9yNVAqGA5wHjmgys47apHH9iviD2ql\\\\\\\/p0wBWb5yBOKnfXp3ExFNC8N\\\\\\\/QfTlgm95b9wUaH6+4GQFzwz8Lnc+cO6S1U6LRYNsocbadb9xin8azEeG1GMVEE9sy7GQjMNQRTT5x1BjC8O0LM1k0oW+dKuFCafVhh2SCW2uvyAs\\\\\\\/tzceHbAlkMZUvYGYLyxH5xGOVVwOwXMGlf1WHrtRfDu0EI\\\\\\\/Sz9t1z9bB5Ln8YISDwsq6zrEJgCcd3o4D9sCqAvBUSbOSg+8jpqHBQ0mmTEBav36S7FTRRU0LBgQcokadRCg85HS0HJD\\\\\\\/G3WWr5jYmByhUbQ9\\\\\\\/\\\\\\\/Kbyvi+gOSVfpfTcx29Q4awEnHNcC5Av6l1uxYCibwfpna6WWXFogZgemGKa6+MkUVFUV0hdShUsLpsYxmRBifZmwE9vyox3QkrcaErENnOso1hmVhFYqkIIJCYEFGJID5Yy6HchMe\\\\\\\/8Poy3KPRjAHRhs57s97fEtQgKL8Zp88K7AlGYy0f9BCHzunxCuPAy\\\\\\\/AE1iQIbEOziq8rafs2EjrG3ue9X6nS2N+CAi+IkankTSvlneCwcE6568WdEA3U3GdCjMjewen5wpL+Uc2Jw8hma3ziE8DdNfEnOGTb7wdLkY0O6C7A39GPT6Kx7K7TnwbUgyUxA5qnK3v9l742sFtb9p\\\\\\\/ryYkfN2iqj6crHQxriL5lnYjtpUVO9xDdpxm3VO6npPKNLtO74HfLcV7FENztwfp0Osa8OgffqyAfYLmRo1tplFVi7aCCGXThvS9Grpypccs3R8vDjCUJ27b1NBWMI6oqHoVO8DuG0WfN92\\\\\\\/TZk8tzHD7+wsB50UC1fX5wFh\\\\\\\/7nHHN50tIQ99ksctrrrZOU202iMdP+s7JctP9nDO0hnpkbtV8ehmGOEld98f9JibycPS8PzJkCRLDBaPNU1dixZlZbxoWBkrof\\\\\\\/vMxHgYYcf1G94Qb3KDY7D4I2T1jG6lp20DnV+NpWJtXhrktazgc5hxvzHD53\\\\\\\/YKbO+m7o2k8v3RA9EcEyknQlOA2razSyBPlMMWGxhSkeUuOMu3TuZH9jOuFa8OtfOAiOhSc\\\\\\\/NBmYa1LRAQUW7Kwod3ewlUVBcxHo7pmCmVZ9P0\\\\\\\/hoj5XB4XgcMRtUQJLfT4de+sTlqfBNKJvpfldAWYMHH16PR2DVFtvcLim0JoArncoZ+uDBpslIamYxKJxaYSWbBiK8fIcwO1QB+Cgy+JRRgqzB9krDHu2aASXRb8EXwM4sietMywCSoRxkVvf+PoS7mHdHUPAB1GWRKFPstHhv3R0TYgjHbaHJBeojHE50PEq1M3yW1rmLonP\\\\\\\/wWvYlGLIvOIbIY5og03FYiMVtEa3XlqhScs4gY6GskJCvrHfSUtgSgRIxJnnJZqVVkw8y327lMmc6noKWgwc+3nHils7Oon7EDO9jDnVAIFhwE7Gc9KGXJCgpbc0\\\\\\\/yI8wgvf7ojTWAxKQ23Z2MI6+b6QKSaKIe5aMI7f9lYjA1lOznlvZITnICOMsrBpJMOPtFrf\\\\\\\/ZoFVHTGMDx6XyAAX1lA99PlXX\\\\\\\/fK+BuUFhSdWBTeQC47rcYIuJgbtyM1lTHRL4WoDCPiab7\\\\\\\/sG1YJ5ssDmCD5RdszDr4jEsmTih\\\\\\\/CsPaNioMAkppcL83fhZRMrgN9keM\\\\\\\/dgShxvgxiSXQ5MvVY8XB0xRmbalJWKCd7wVhPoXLYJz4QBlCEDbDy2VgZtBjOeAghI3VexwG4rXz3pTOPHQQjY8JwK+Jjmat1\\\\\\\/9slHLLHKmul7vfl07aZa6QFa28EImR\\\\\\\/CgGy9X8go3qEZayxkBm1Z2pLSQMDePCNcO5rBAmwb0iW5hfAj+oZmzCVeE5v\\\\\\\/y57x7cXVPUi3\\\\\\\/43qFuLAfsXWeFSIvYJInHYtMT2M3iIyTkio8n7EFzorcftbdCP4fHSKnlUOEdhNdBPFcwpoNvsnqtWeWjBwkhvMB5oJU4KYVRsG1UvA4PSTHrXoxETI9UiLCad\\\\\\\/Eud4Idm0Au3SHfNjgQMK+l1bnmKsAkAMeXObo3BnKh9RovD8WU81AfNJoNrvNigphAjtTxaPFzAGTCjtmo7XXHYJuEa5wgaLhjmR9Duy8kwgyEIXIcNqK8LD5aD5p3JeFOfwSmlhIHEnuWM7kedoWSjrL1hUkflAGtDHz3RuUXAnS2DmIC4v2aBDNUhiWIpLkdcmr+suwlji9B7hfzjbfLgfpus+N1ShC3lX+VVgGZE0jb3Eb7M2EmzF0QgStenYbNrIqqgXyrhsCPtZjVPHOhtdqItMSjCtE6kBRFSTTsX95Puhenb3URX\\\\\\\/I8wPpHZ4isz52A8S7bbJ+RkhR9Abx8S2QtOGuzzhlsrzuy12AKuhkYaqk4R9ozI8bR8MJQ3IMURIJ+2lKE8HfPal2G1bSJ6ymDOMAU1o7pXFpKVEW6n6it6Nn5LY5lScNUXRcAGVtJTwq5DavY\\\\\\\/8vCKdbMemUfyQ8WAOY+t54got64IxAyLOkC8dF+GrHbmw5+XlSGDGYpLgHkA4zaKiH5nNlRwUJlJS0UG5T0yWr+q\\\\\\\/TRKEK1V7y4cMJ3YT+oYU8RV75sBLfiPx3xdzMN48gBo1O9ok0yFbw5Abbss4vYcqImyYDk1FAlbikSwjc7xmcXdwqFnvwbbWVK+QEyK\\\\\\\/p0M1kX+Hh84G3KxhSgBF9RJ\\\\\\\/4+7PCTSX4ImmNxl\\\\\\\/f0mUeLwXI6IyRny8gHUSc3oZ4PE51q2wztO6WF6SWKgvamT3CGE7pGpplqYZgSOdsGm\\\\\\\/cGNhvu2I8DLVdZbdGYNG7qKf7ujFvZ8o7DqqoBatVwGkn6zUDPdHSRH420VophLxRJvtiTmycVHTjhErkh02oRrEu5BpMCrfUNQTRVgfQ95ngpb50g+YnvWx1iu\\\\\\\/Y07459hHOGi4YtMDukJ7Ut5DAVrYZbv4aVNsva4cVUCg2Me7Dy4F5XOyrSbbwbHryzYlAisDdjga49ntYUiYQPGfpPsMMKf4v7Eh0+k+iFfYpWiSze87k6kHu7fXq6sky9r8Zs4aXruF0Cjv+TVyG7zCnMiBg+BphRTqzU5KqyG7dKsITjAW0i740V0SVhdugeUOy8e2tBsVtT0Faw1xHXQvhjkPRwLWvymE9A0iBRF3EwJM8RSVe+a6CNQfJNAjE1rdDy0c+1gVQmj48VKKyI8fL0BbjvdBof3VDT558YJFG0disw6DPfT5i61pBM5LjxcdfmpsD\\\\\\\/IUw009jcuqgttcTIXTCz5Qz0iejKJM61jO1mlqwfkSPimowhjUk\\\\\\\/u9q2P2xR8oH45HLooZBANEmeUTIRUGQhTbk51O6JmRquxpdcDiHM0BarFa8rUr\\\\\\\/Id2bONeOIiXvUlQNBUgkNgRc9h\\\\\\\/+3jAIcp8fOJXRbsIOPlquWmPkTBgu9Qj1IU3PfJ1ZROsulS4MXHw8ceKlMmNCaWwIr9WTZzITU\\\\\\\/YMwZwU0onpeZiCFeGiiI51xfno7uf9JHzHvnGV+1wTmJjAc5hgfN\\\\\\\/nCPHkAjYYz8VTt\\\\\\\/TWo4JOrbsGxLbCaDIn+aE4E8irnUt0N1BPdawD6YmH6QiJBBsUfrowObb+oYVNyPE\\\\\\\/eMoDInEx9zrZIvJGg2jflORcVr18tZJxgtXn9aMWDwMUxMETVJJ9ixuI8BoZHVGramM1wlnPUX4rDJOm1BRC2bhKL0DRSru+YOrT4yCdH\\\\\\\/05ZZKYlamhuHOpRQUc03+fqtQY357z7ZjED6ptMnv8zWHdUk6b8HX58Jlt8eASKVQ+Wnq5bZilb74kaCLNuSaQNSy5lWWYf7vqEPv+W4cIdgEnwb59Q1Bnx0hGOJoEewMl9vwka4xgEU4AGX2bt6GQmD0jtkoGUpvDt0S5YIQax9XPjz4puHtpd6qegyeZRfkFdozjio+86CJw50HcZzVd0N4LwLR5TQBpIC7GbBshd25P6THcrzAfid8hxTEPpI8UGDU6HDhmPMd+2y9DO74IveClKegqW5tvNWEkwHPrFC7aqfiM1wUDXZEojdUaWs51VDf7QHXpRx3raRXDBaY2n0MXLroE5pW\\\\\\\/k9BUkn3oaClOPsZ7JEhLcQuiMoY26b+MY9MJcIy11cEE7\\\\\\\/7Zsn2anoYhMYSBsV1NXLemiXhGkKpJ6U30NLLgXkvqrV29vNlxokduFjd8zBoHK9\\\\\\\/Dlix\\\\\\\/AI\\\\\\\/ZhT6bC1noZWKSZOqRPJ2kdx5zd0neeCqtBpfUqCCzgBpuCmtFSlcHV4sj\\\\\\\/OAei2OKSacklzaVmdcQ5QdWvGPlyn+95jETFcvvqpevRV3F\\\\\\\/C\\\\\\\/ejTh1l7\\\\\\\/oePlLLGezyJVpLJZC9M+ni6vh3lAPsRJTURDZrsqISySwfOIdTFBqUqZnd37NVLperfmWT80qIQYsYam177olrHc7BD2syNCKVqKbLXuBYH+bzZuidOYLLhM2wTZQyn1L6uVsw+CUQzpNWHYpRiQW8LhzTLWaWlqfyC4npJ++Q7Xe3GIorkqmpwLMxD1F4E6HkEeE4BDN\\\\\\\/xqBnoLUGokvCmN2m+QdakjthUQBQySKfEZKgHFoRzwc1HZ49o8+GB5bu50NjMBGLFbPNz6QuijgwMivvSwGwYKJuoJYsCzLU0APLNQpr3gz+AkOLQH5MrteV85LdbJDmkz0nZxHSCc4N39NbYLKhpAoIk8AQLeArwxGU+UFMxrFG1uw9VNn1jDy5lI0iBGjWV2Z4NyKwooC86pH5KwM+V5nAbds3QKN1GYQGZObJRuBxT+Tq7ugJewk5ju\\\\\\\/nw0jxDK4CqoQ\\\\\\\/hNhZfjkaQxUQ6ZRgjt\\\\\\\/4NHZHexXRYFggdfX9iQFfAz5KMz+xS0P3ihhlDRBU8uvtr1I9B0l77LLJRrSY\\\\\\\/1pqeoxxFXvZfn27JOD3e\\\\\\\/PbNEOukBH1J+hc9S+pJZEfWvgGVxXByAqkp7rS7NmK+Jty\\\\\\\/ew7CllyAwFVcw+YGTfNNhCrm2xNeKyi45opqdxNbtaB1\\\\\\\/3yJGW8I3\\\\\\\/T35F8BZjtRdp4MgNF9kX2jhR8A9nSOhSKkiV70LzSRMFlX12dOX2k8JqIqEMSu9YmVmMHSDgH6Q2H4nnB8c35jVy4W\\\\\\\/0bV\\\\\\\/nAUDarFfKr0bUIzSxgOMg2t6km3BmITsn6fsHoV2OxV8iTFc5qpjNzk5x+mDbAcoFlPt5UkWQoecP\\\\\\\/P6esqDOZyWRVn7eMNvf+1WSjAwW\\\\\\\/qkf35kqQs4J7Ay5vBg3RNnYoicJNyPCuWU9CLMME7JjQKGZ1AhK+ICRSITb3dh28PvzVGK5DqciileyLa4zwa279yqmyrd3AU+9TshOPnoSAc4H6HMA\\\\\\\/h9ZCPxZJRzwW8Fec0B3Kccrkx08g24hBXfci99zuzqjtVyUCG\\\\\\\/af9lctpMcCejYFI5IE7\\\\\\\/xuLgrw+9eqjSSidS0vGRFD1i\\\\\\\/t2jhsgxSO0hUHrzpIxmYDIWPD7dVNm3HJtx8iX1Isj1kwiKXTp5YtGL\\\\\\\/8nha\\\\\\\/p2WZsveuHv0sbMT0PFqjNCQ+fqglPIQgsCmO\\\\\\\/fQaXjs7a60AT\\\\\\\/J1mab8gTegO3RfHOlm6LzR794Lyg4mBIuTv94zlkrXjEGI\\\\\\\/G7N0y9pUjvexWQS7rXXZgHGyRLSNkuNpX8krqx67mrpjLUHhpe9WOoqVslT5ov4x8aiHnapohbdUGbxXR4DMa2+SF7nSOjX1vzXJC3TN2JoV0EzlS0m7+aMaTANoLb\\\\\\\/E8HoFwiGgusKhx4E1w+LWkpgRRK6Fhyiu4nJpA15pWzRvN1gRqcF81f5pOUcpmeBYt6y5Q\\\\\\\/JpdxVvR6kSNghr89tMhaPIKFLoMw9r0E3JSmsYQsvAzemC81fTrOXJsV6gusHLDrf3XvHclWcccU9vfVXgdWDJqbqVsg52FQ3531rICKtJHd7uuDsZ4sFBVwMKWfwpM1KCJOjZRyZVqTR5pnAnSPpFyb3CTmfFN9uyDqiJQeaLI+rJ2mPoMrdiQTLEc11YmFIVFoyGyPQyvVvD+QkfTJ78wNJ9ymfrJhgjbzcKMGhjunvKsgMvtsAWq6LstjOV9Tnz4sCzACdt9Rex7sly4qZ0M88DwXOZB0\\\\\\\/frJvGp1FNwDRId3wuy0OKLUZAQm9o50A+5MDbdfKq\\\\\\\/FWXjH\\\\\\\/Ap8A5PC9eoGRa+8H8ggeZGBMfrulgw+ByNC27mhwh1TCBDqrDuAOZUuFFxNtNPkuBXA3e0pBAeYqZHuNLFcT5f7rrPxPN1\\\\\\\/QIz1\\\\\\\/W3lcBbyiypFHcPt+myqtORyQe3+0TC3ubOzLegQCigaG66AJqw7yui4pjx9Um0tecl0osPb7P2zv5Q62bKPenfrgZp9GdH61svEXQiTOe+C9VZoTDR4VU+BZSKqjXGtIQIY+4Hctvt6n0iRcjceR\\\\\\\/GIfYnlkT0gPpKXEBgjHRZ+n5mBUUmJfYupysgB\\\\\\\/RiHQ8+rC+p74HFOhDqBYLY8k16d6X36J4DVI8w6UImpvzMtnBvSncoaslBsgdZO7AboD5sz8zchAl2Y0nNmdq64y+xZeGo6z6bwwQqxS\\\\\\\/Pt7D2emptEw\\\\\\\/j+T7NjKh1xUEqkAwBUepjzaMCrk9yoQp0Y3sWZFMvxWuKmpckkF6cmdz\\\\\\\/LmNiB6KUHsfbrm9KjnekeIR8tpIyAOt4dik59THWLQfsiWGHJTtc5Am3JenGZcKUTzaxTJU5w6XUJdUaSC4ztakGTwmEP\\\\\\\/+j7lwdYQhByGcgIqazOYoIilMT8MVILTjzS\\\\\\\/Z6UfreCtkWo6v1+IF32Xo33HUdcntfZ92GDi3oHmWWTfFKAPODJbmCCvYNQvbdXJiv+qhfNBFF46LLlecOmd97uSJ6RkqvTomernne3axdgp3zLCZP1\\\\\\\/RYmrQsvgpz\\\\\\\/LPKFXga07Jf8BpybqSh2YeWW301BmNMSbVuNoBASyt7wTbLPKV1pQQdSEN\\\\\\\/L8+nGyPsRJKmgBkdr+r+s7sigPVkriLdT0ULUkNvoGMxxGRHseaxZfseFvAk2vLgaLqWznDdoEgtEtZMnK1KwsM4HRzKymyqByuayh\\\\\\\/ZfReIjBGEQkLzpBxF+r8SZBoQw3IGt1Yk25fBLrOdTGfkq2gl5neEB65sah3mEMvF5ia83nLP2TZmq\\\\\\\/dG9q0\\\\\\\/REdWb4JW7clDeQ1mvwx1dGIRtZbnQNI4rjQ+iZHR4qizH8+zwjgn9wZMrZWFJWVT+nIAygQ6mj9NDGTN\\\\\\\/LqyurVNWMYhkNRAXOhDSsj7llj\\\\\\\/5lYbHTaeU9udFS+safkQH9eOte+TbDlximjj\\\\\\\/NI0rfllYB6pV+hiA+77PElYirxaEzPVXKgxWONG8wuxfxfixt++SR2sgm4lT6TPuwMEx+lWodMeMzz71JK5FEr6IMl64ozzmOJIq+R+Pcx0vAW8L78MkJF\\\\\\\/b+ghOq0Tamq2GgQa\\\\\\\/m+OWnbSFzutOqo2i5K1W2s1\\\\\\\/Ge+s2PK9L1WH9qLymy7vpINRvj3h\\\\\\\/YnX0SpWf\\\\\\\/2Fx1bBRCcPGMxmR7dd\\\\\\\/Uw68Nb1W0OyczCONR\\\\\\\/wuhLtLMtdD\\\\\\\/9w6ja+oDatS4M9fW5PZGR8fmkU+CfcjCk4AOo+MlxrJMslsjbeYk31CTUxu2BCNLeS4LvL7XcjEzE+OGnL4jsOrnkJCDsMwqh7w7sMpHEK7xCjNpNMeA4tLanKlmjOTG6RdoCzf8ZDOjSyUPLnQItuhwhs1SVUd60zGqwLex0bBz3tldvLNen7LJ\\\\\\\/ImctS0bM\\\\\\\/GrqWAmMV5g3cyrqDN2cnsWvYY0oQI0Z8FGy6ob1GyCXK\\\\\\\/4+o84sv0npPVs78SA0CLCDBZBm96JI0VXwr2nkkPMCuaC92\\\\\\\/Ca7IlwupIxUbSnutyA\\\\\\\/y1zb3nH9d5JkWyzTcTJEru+IGOQsD5J2iVU0ABqfoZBCqB8q+MbLPkzQ+ePd45r+mh5yFQ92Es5qg91LfFuswoy0L7rbm2zWK9X7WlzYFxjhC2DWNlhrrN98O+nPGubhr9F4W\\\\\\\/ElLIQPu8fIdExpVYZCzUIR6mv+ZvPWxT1VUxGsDERuWY4MAHrGobbNGTf52kQL66Kw0aHuWZgiNbdTa4nX9X7rKs31K7XkKUEk95q2d65O2Xkk7D7Kgh9sDPotVV\\\\\\\/x6PSNznYC52JtQh2A9heVl3w8mepdxoVoJbQZNzvjvDSkg3aeeAtCErVVnOpnyyKycIyZI\\\\\\\/te4JvUJyucNMOS2P9oDmZ5JINS\\\\\\\/KrK\\\\\\\/jYmts3lyzfcDin9xEG2PVs\\\\\\\/bTLKek1Bqkw3bY3GeL5FLHuIP8n9WE4Pimu2AJzkB7UDxbjqjN9OUJTuHj\\\\\\\/KA5LyJBoltda\\\\\\\/87Q\\\\\\\/HZMfjZYW\\\\\\\/\\\\\\\/scfOIeKBozee22lEOJRlvECJn7TliS0B34XFwaW29dWW8im9HNkY2ypfVc0jXmTUXQe3e7wqKIpdATxp9TrVe4UItQ3ayu4OpoGFUA+OTMEkdiCahVQdu5TSAISGAr4MPfg4WMER2WY9pbjAciapm+Zyl1+U57emPj7QKm5EFhwOGZ1ONW6vYdnIQ8eeTxV0kdFQ7s8YaQjtkmEoV93ZcWiZUh+210xifaWgERLJ5ao4Qrx1pq9\\\\\\\/yrfBZMFehApRFvgun\\\\\\\/1mw6jzZGUuqn3Se4ifVUMfim84VrBQCqzn+IAuO\\\\\\\/VN5vbaNMd4O1TKiuMRUbHDCGgFclnIJemyiVjbyH6QheclQPKpH4wlvmcHEsZf\\\\\\\/19sfTp8VsLOvZM17b2vqnMpJV1GnpEG+qg3ppdy3bpuP5hiPLR6I57nJ3Wv1HnbJV3BpR3SGIe3l84u+xLfm5faL+f4Nu+EYEVypM6F73gBriqCl6yuRptHYs14BnzDCKVT+kUkE295t81i8EK9kX2fpyvLnzKjlgStOtFK15VWHIjcz0vBRGYywkLR\\\\\\\/HCoeBo+n2zPg7hG+VsIox0x+ftzSLHHoLhAg9R\\\\\\\/S6gXuvDBUpuJkgBdolSgbuDgr+BWaAJtXggijk73k7bdwZFWe\\\\\\\/Ea9muMmDft8OgE6lcJM7oYYjitl0dwcAxRmJuDwUSLKHsGVoVXgN3mig6z5KC\\\\\\\/2zwJzfPkc\\\\\\\/SDN0qA0vt6GcxM7OJSegZ5mnuopOh0OVs2rVM6G2M0JwinNIRvVUOxvB10jwlgjslediAwQDyOyK0K33v5QmGkgUq+icBWBW6kdBbgUVYSZSr7\\\\\\\/4vZj8RPcZpUAknhsqOPgTgObF7xBcVKGO03PcgnQl\\\\\\\/XPCSWu6Ps7nLgbPSP3lSLJGzlX\\\\\\\/Qlvy+m2PDALkr49Q5t9MYvfPgPsXkmVkkAeatijixMiuPOt4ebT8bLrRA4DwPO7aRg2AJtmURcWeY5Gdk5HlbrYqoJU\\\\\\\/tYZ1rD4BN18x4ghU6yXtLMI8Gmdo6A0QlKDQtV6oG1ssRV+W7+iiwFxrf4ZsCzR7L02YVWWfQgfZZ7xczSwr36dKPyYUyU7XOQEHux\\\\\\\/oD19QGXZnWiAhOBiplk73XTmGKrWsrWwQamQCKue1sTCNBHXck8ZTHYKIU3r9bVqyUZ1QgBEQzve9Um9pII+gXZ14u8v5qOOxtxEBiZwQBY3hrvJIV4ifI59CSRHLzP6xd4iUOhkd783DhS7EiFG5htUqC+KYCQtqpxiqBj1iuqAnEsoWmYsI8b48+WbaY7wjJYRS3D6TuiUqvUTKK+KwRqd7I+aNHFgm\\\\\\\/FbicPgfv8f80mzqNFcaqbzm0AuZmuDNP3gCeavkzazKFewSNYyoPQNhyi4NwLpTv2UffNLD\\\\\\\/qrcbYsHpZ2rnQkqaC5LcSDFRCrLD5d8shMAj4feCJ7jL4iXwsi+FUWQ80aTY20E7BexCFc9uEiN27bZE9ZZNsWFhZc85Ev1xEZoFclXUFc17TVYyUJv\\\\\\\/WgHTbYidLej8yzT\\\\\\\/jZb+S5AlSXuFsNSz6JgTANbkZTFkaqzsfZzGZCKx6Ewg4g6g7eRYHV9btetCHuWpm1v6gguPdeasULZkXgUfBASUi9C9ewBpfR7JmokN1E2+ckR5sV\\\\\\\/\\\\\\\/+RRyHXaNrWnD3a4p4542Op5drtLyfGSJIcQGEhkpZMJu7dL86MSoYMoCj6b4cJIgIUI0bbrvbu77E3z0cSw58iIwI\\\\\\\/gcpX986rzE+jQCq0mUBhqkDH+3J0EZQZmRd4Qk0Bzji3Z6B9AvF+cw3wpq09OXdd4yWjs4tgD3nWV90iG2o50\\\\\\\/whQlQuS2ihZz2rkd\\\\\\\/\\\\\\\/I0sRP1U0GbvGPA3CnJHIR+32D+hKRbAAAYdsSZQSZPk11RxUz8yUwAz1F21E6\\\\\\\/jPcqANOfZbiUByDgWUTdrLk8hxEntg3Nvc7tpI1CGQohhD12YHcUygZbdo1pGGV5PDxpIcfNq5R4ncjlTApgq1h1OACog6ghJXdVW8nOx9klM3bFgKDv8Zn2H0HO9FYl7D6mOJzVkt0O87V6KyibWOWXrU4+04OSTiLPzgjtJSMvrI1vnup3InX7k9VKIlD2HPw52JP0t9cefxPACPEIQcsgszS+tQSpAYRZ9gnhqWhDoGPABacKBneR8H3xYLLPKcsQClcvLh7pBLTJyXK41mCk\\\\\\\/RDpENo6zzE+8T3GtA7dVZxbzv+3wilbqJoFydsbexKIpOKELtiXk15L1WMROP4cWS5joJZgetMmehapmusSdWA4JWkVFQ5ts8wBwYHK6Mie6nDDeiU5yoqe8409A6uhO444OaG50P3\\\\\\\/KT4d5a1+0dcM4FU4jq5UrPflFPRmZHPykBDbSBQx+\\\\\\\/ScDGNDf+jxEpr6Nt6AOWZDIRdsMAxhH8mwmgclrVPQBwYB5SFOS\\\\\\\/318aQDsp+Q8uIsN6ghqazxBY028+ijbeuZAEjgo+25QES2FIWkVP6C8QrxSnWD7eNFeL8MCCXwauvRdD5+Mr5zA91dgDuZr3QhbPiPYF4\\\\\\\/ML6FwmIKomyEOFuVSvfj9upivd4ydCK2CU\\\\\\\/jpS0m5sIbEb3KM1KY\\\\\\\/NsQ3lKwYG4reOShtXlmZwOx0iCJPYm7lBgZQBMmK6mGKE8eF6W\\\\\\\/BpSquXkAZxcHui01AtpI0KhX55DLuupnsggh6buA+Q4ABj6q701z7t7gM+78jXkzTAUiq3vgth70f3ul8YnLKavO\\\\\\\/FLiJ1Fjk\\\\\\\/TikV8WcvWtgqN7zIK31GWwby75URujW6aDSQDby1FolM\\\\\\\/KPaxq99\\\\\\\/Jo6Fz77L5AqWBndycRKhCZwr8+bJQf\\\\\\\/ygNL6fKhgzZBf2XeeC0KQax67H1+7qIOvnsr71SourTtevRJ6lYOGLltpnw2lEZUjgDjB+651rzdxuioRItORCepGAxDwDdz9XwlMMT22UEuayL+rB37qjvQuNn8SIF6JRR7rj0e2kbt37yZgO2in0LXzQLpH3JZw20vxy63OlDTweXjIbK9Xl3MobI083WxuZalKpNx3x+t1lY5oUNEs6TYMasVYSrch8RgP2FEeA0FJhoi8TQtvhAcjOmeR5wSNwEAOT74Xyn\\\\\\\/Q4kBLFdDbjJVgLFiJiW9zPAXJSMgLPOBh5XobvHBOwxQdzjoKDd\\\\\\\/bR8+xBFDxHhGO03wd5TRhZsjs2w2xdxPJ4I\\\\\\\/E6MA2CBseeC0idcE0WW5yIN2HyATGnm9b5\\\\\\\/nZ2zybGK+p0iZuZie1coi\\\\\\\/I9HjpJIGA17yesBpWtfkrruoTdi4X2DuSAgccuB1lA0b1TbRx9v1jrdlcPUYZZM1vxQ1cYEwJxV3q7tTXzQkbT1bubxuMZgK+4WN7CMSqnyyLqxN6GFl7dMB2NEwTH3+bWFcNVXHAtW2RKKAOgmHSyXkJ8yKRNC1qWrVwprDnPHkzogj53JR2xBh1GkxedMsml3oiTzFYR7IC0ZwE37CEe1Rxw6nu\\\\\\\/lVDczW3ASZT4qq+l\\\\\\\/GcbYSKm7KsXZF+i9qsqyN9OMF3yD4o6yRkolLCHVIzumluizeACPf88qRoY0f4+TP6qoLHaKc0Td1lfM05gaI7xvQKEFfPaMSd2YCkN+Lt4hTw8uAFmJRCLcsKFXESnEQK0SYZBOoIgwlC2T6KksJyWBKAr\\\\\\\/p0PGDDUdoBXa9UNOiR6ABTpc0xBdphbmurJ31NQCbT1+OskpI3BF9o4IuEHdt6rmZGdaElRdZA+Auh3aWctzMjRSqhaSu7soG36H3+GYbMkYfp8YZzFHdP5G5BgkB9IgoRi5jHb3A9koI0vSCcjfaiu3AGX7FkCZRTbAh7QHYK3\\\\\\\/60pZGhztpFfoP5zTqVbKQPj6YJu4bgg\\\\\\\/NEYpNQtDgKn1jAu95gzJI1B1aTW3koGo+Io\\\\\\\/WTNkEc3+S5WR2fuLPzGXutpkAv7kMoibck+NU2ixOqKc49u0eHSX+dbAA1mdtptyeNMmE+ohpQCwJ0MfqClYpAyB2pTRjW6sBmfuhcu+3LMdU6WCANYCoM90+TagXO7uvg30uJ8y\\\\\\\/XEbCJ3dFv6gtgItyA+NQ0a+J398oIRAatwMYGJ\\\\\\\/B0q95bI4wjRg0+ZSWzjI9\\\\\\\/2EVsRm5ImlmhQQJYkAm1suSDDbpffE81qlofOC6CN1CS8kUFZQ0+m4OlwwXcAq6+DrUaQFDeXjjzXsT2ZwVqrQ4fdl9L1NJom0tUsPFeGSNIzLk5ZRymcIy9wbZsWyX2PyjMshJBH444eRIyXJ6yENo7B21438b9cJMU7RX+qOOESVxEzpCuDtmOMTchMSxegiWxPa7bRDg2vnMZXjQP8NnWXM6SLWUp2mv1ELQId\\\\\\\/mF7Ha4QxB19rIZ0FQLTkfVAoWu+zCTuhtO37pGe+9edLoRAyjgfOvRf4PSvTnQCCyrZRyuvXgBdTvpdEvMPQosVgBnz6Y0AJ\\\\\\\/C3j\\\\\\\/qdz32pKhLDOQtFSJnBSk+u5FYymxZAI8HqSI77Bo3Zvv5DrSbOS4\\\\\\\/eXIXALsLUXBlJkzTnAW+mHsXMMPXrueNxycV9SpLVlsH\\\\\\\/TnTC63IKTqlbijSCc4ofLJyd7EsP4\\\\\\\/n56qx+f1YBEdhMivirm+S7tjzZHMSTl3OS\\\\\\\/WQO+ZEEYriaczKdeVvwjfKoPst5WxbBI\\\\\\\/hVEbJVBU\\\\\\\/z9RLBM4WhBk4pLhTNUXEuC+vEzTQbB9SmMjx2XQKx\\\\\\\/lVldZPCYQSkoeyUJd8+ZHVsygboblKx9I\\\\\\\/Tvzdoj9w55TtGfIsNa9AnZos6HI33uCOKytpTwrpypQ11Nd0Ktjrms5R0G9dH5XV0uZpQyaZ\\\\\\\/OFs8rXnoZCQ8Q2Vr\\\\\\\/4CTrvgAZUMOP9aZcb9P\\\\\\\/MZqjlQkUHQNvB22FCd+HD22MWAJqd1Qrq0gRgs0ypxnmjmklh+gnrf1fF4pIvzM\\\\\\\/ydUQxTRn5yd8S+eJ9LrPfG0MYadz0DDOT1LmNf4gTwvgC5QK3rPxuE60OoKXF+LPaCFJlyPatOseZ3GezYzNcnOzEHxwpEYI7YGpYVjcc2sIPxT6fIQxGOFZHcbLpOrpS6ww4dpnKSWvDF6ucrtWjSe16m7fccyv74CuvgNhEfT+YxbuC9JET3IPTqExlJgIbBDid\\\\\\\/19Azn92NYdPo2jQhAQpdM0E07UwVnkOw5TA7yN5nzHM+USlqWiFrAldT7npgH1W5gIcGroTUlI8OMcUN8\\\\\\\/GYFzqDfWE12+k96Zz09OPvHDZWYk13C5aFfTwhzujOI7VhvNuy\\\\\\\/2etud\\\\\\\/8h0Q2Oh7Z6nwCDQf1HCuMkk2ya5PfnIIq4eYaDg9PtTMLVhYVsf9YY8nyF0g9h8JmW441y++S1Irz3KbiVX7kQ5YeHSwlfGRF2GU8HHCVRcPeNuXByrPBUf6D94XJ7V+R\\\\\\\/TUdGstjInNrRrw+YVFFY2vBBU75csRhFxy6iRMuXqnsDEcxVuMIBJymHD78jeIyKBZyK6xqQHEqTOFNMbUsf7A\\\\\\\/CM+t6Nf6uy\\\\\\\/HZFGqBxCG+Cus\\\\\\\/M\\\\\\\/uFFKwF4BXZyXSuw6h\\\\\\\/7lcRYB7mDIYK9YUDQUTfkfKFmgENX42qewdtaokiltsQjT+DoJyM0XWfSq1A7fubf2cABrgTjXVg2Sop6a4DgTXCQYinouVWWIB5H2+KLXgyg4wDyVXaxXYCp19jjwWwdApFHW6SJDnRarESHHiB62bPjo3SlE2VtQ54IUcMRuE7I8yzKmw3e0JR+7YfdTiV\\\\\\\/ejiJ+Ff528LbHhf4StSRmtOyU2oZ6rIpOaRFz0kqwQ+ALzaxafRBtTVUPU4ZOnqXJsXN0NW58sDJ6wXKyB\\\\\\\/sA7S3OG2KOTeeHAD\\\\\\\/3WW0Cu9eYRE7b1B323EE4GdYa7qx0jpeQWKxtpjZJuhUsTTJsT9c314inih46Jijk4wNuUM\\\\\\\/00ox9LTF0Q9coASvI7VxUzCJeg6y8h8cPQvEdlr7BYD1OGfNKqIkt33T9p673Gx7qF9BCOoIW2Tl5v664cfqcmirEgkPMxCAwNwTEBBLsfnSJDEKb8dI14PGQaldQFqZxrqzcES+jIPPYFzduAhnQlDn3CTrREfvDYR20ch8TMMwNbJNNSZbm6AaRAVS6R+eFS\\\\\\\/fD+s5HVpbLYZSvb6OgQZNDqGHAGHtGWSaiMTW1ZY69GWJQUffbdMnLjMA88\\\\\\\/6SNwmWZ4Rku8d2+OfAyUGTjey8TmZ37olcLejURYxPmlgVOD7v5+zDejA5G6ex4+U8jaxg3VngRNQSGm1kLvFRyG9jytdh\\\\\\\/oitee7YWVFymgXLObAxMOMVM+IKqIWAozFNLeF30Vfu2KO87pfIoIygFps2iqGBhS90M8nLmrUHEAT2oZNPvN77i9iWQMrIJXd\\\\\\\/9vSp7+\\\\\\\/6XM57OI+Sq3bVZ7bQu1Jn8uyUQQxMOCXfLL\\\\\\\/TRcjEKOpFEstWPO7e6GTVhbmEq81bZkt6fDQKpLr4pSy22xgBiER2ZW1pkWpaNma3SYxIvxkn8bOfw0dxi4DshExsImP6xu46euuBtRsTn7l15aFJ61o9g5QhFadnZGXSMainq\\\\\\\/l9C70z\\\\\\\/qFSPtM\\\\\\\/B35SeAuMXf\\\\\\\/t\\\\\\\/K1x2G5VscdV6luSkC+grEPeEPosLD+cWS68NZvl6ZoIOiqu24IrMFFz99tQJJNqnFCCuK1eNkx\\\\\\\/QHDThswLy8OIVoEyw84EFaBJimuodvluAzRT0ap7jcinPTh5Lq\\\\\\\/5LA17vzXtemxFhVBjeSihcJ7JWcywC5jhOwhcGkuMbqQ+N2EIVMGNdwcad1IlkFBg\\\\\\\/Cmw+XEhQyeOleVm4N7FoAoj5o21RataaWyLzLUf3vNHexmd25IHECQ9kiP\\\\\\\/uu+S2f6w\\\\\\\/uqXm9xYhkYgOAZSsLY1UhDKQtCBX9TZko2MmPwfGl9tyKsyRJg6ELlYOSk6r+W4fNqoa8AsbjY0T83QETCV53GPKUL04kdt3DYhkuxqPp3hhV6srbplBlafV8514QqKTZYMM\\\\\\\/3e9hc+JOOx3MqH\\\\\\\/Ubf3N5WpPhYBjd9nIVR+Op7pKc4YG9Nh3Ulql1\\\\\\\/Ft0vqEpAIHylLSe7JafncoGvirko1s3hXS2WLjnAJcocpRoU\\\\\\\/053MMHLp9jBbg6H4tJREhqYsA3AkphB3zGI8h3hac5NW4oQVg2HIBR4c2Rj4wOIn3viIi9eXw9y5SslGzMkqn7byhsIOvKLa5Q2UIfD09E69ftmAJYhfOw\\\\\\\/jwwO7m5kH8JjvGwfOnZ2DNDIqtMKkgDMZ+XE4Q\\\\\\\/ivLhu0hfl7VeTvBwmeWRL3s0UdXqBGzeVcWs6PbhGm67S8zGVQ8d6drNEf9\\\\\\\/Nh6gjZNTzU6LAbJFcwD4o0EVhsYeuORyMYhhbdfq2nDBzIGhSbgBmA7G1+ESqQwD5POqF0JzNTD8y4tyM8be+s6xB4zeOyRB96icSovbuR9dXH3u91IhailObQ1lTy87vSAHpcHfeckyXMWMVYAeQ8BDQkrDqToz6zT3f+Yr20cLo1ArnNopqMLs30fzRv+EZA3nGFa+g0\\\\\\\/60dGRjLI5\\\\\\\/5N2koe+JKnyfEm8q5hRhGyMl725SmvyDIdiYxlRs7ZY9IND5zwyjqGvCpb1Wh+b4PEiuC9nCOdFFenGTdPCwf6l\\\\\\\/WzHsdah6AwmzGWc8YXcii75ST0ZydPRbc4PjtS4HMgp+Pai2MRcf5\\\\\\\/RyafmZBmPpFwE6A3obTZO6O04wAd3T\\\\\\\/zEUf5XsneFtM937Fb5kCrXS3mLhExbtS6qnDhc5xBBpSZXlhi19YXc2p5hyBEjYtC\\\\\\\/Nz4imn45HHZ9LOKl8Z2VZ7PkI1pWCy0iYB9kRUdioddTNHixRTrbi8hw0KKXodS8QNU2rsMpYvr2hT4Zl+43jS66cQvINoyZswesja4gC7ErpdsQ+Gt8jeiH6UdtYKWItxrkCApIe2Xlr6iRYIbTMD33TnzNV1VgM5kC5cbKHZa4eEs55PSs8ftk2Re0QJxgvmXhT7EQz5BRL3t4Hd72YrzHwIg6oFj\\\\\\\/e5hPe9suLi4O\\\\\\\/FzJ75yYydMPcL50F3InFMdE+PCmML4hYVk3V1eSg0z1+ZWXahRrq177YFsOfm8HYfKrZsLQGr4txa9A\\\\\\\/pRL2pTo2xIdOTNbHVGlq3xWkyJMDOBTvpDYUZY9lLIS0BBbMZqChgNW7T8jNEpNvLmsl4ZpYpn4fc\\\\\\\/mMaUBFIqQQWv4FQtkStIjLjDstRl37byZj\\\\\\\/eEaXQN\\\\\\\/L7s6eGZzTdspOOO\\\\\\\/V1G5+jTiCklgCnVradrYn8e404vnZJ2BYaei5xox7XMafHnwvcP3SGoBuZzlNVciWxA4BqpoZ+SpdoRDgqLZ1+CtDWA9EMMXnWNYTTd6fIxI64b4KNIncWNVMn79MFijJl6pvNQOjKRL3wOgAIbTHWRX2ai22zDTPGaX\\\\\\\/3q5gcskoPeD62ph+nS26+JX+C+SRoSmXdxQkmp4UtpticWj7hDuT06+1Ii3QxTFq9iT92VvTZRCw48zvWGFGpi3pa5OIqGg+KZYi3Xxauj8+4rYZpJrQ\\\\\\\/uLFeWVHY4gtAavdqYqLWZnCLFdWRHByQHbVdsBGOyiLS+DH5q76bB87Znmwmf0gwJYdxbsFpDSpaVSW6Opox4L9YmdGzHK4yEVnWH99gj0vWugBBQ7ySDygN\\\\\\\/B74wXJnOAWtNmIyWXw0w+vK4Ti9jkW+2Z6acFNI6Cqm6\\\\\\\/4MOw5QAj9X0K9XQUmlvQW8E0yeez6Ji7zYviqrFXTffyKE4frIsZ\\\\\\\/+XO\\\\\\\/DgP+BD1e+bHWQhx4xsUAcs0Gdd\\\\\\\/Z3S4z4MP1mZbEVzrpKtyL4Pkm1K81Rg0V4+9RJzyzoiduNDbwJBA6QbRJqpW6tLJlpwiTofrJht4ZyRzVWoEID+OlPYTNF0lx8nCPDvcVBcl2ldib9uAhpP4oVrWlOt84IenV1vn49OJ0mNq0E9dyphchk3rusPPcBNdwA0UrM4elRbfbO1lEnSAOsKoWZchQmfg4I0QEaVT6lTNUnA+StXmoB3VLNTsBLDbz0GU0Vb+8t9GMiBCIGJyXRZHTLx+Fi49H+H4rQaJT\\\\\\\/StMg2s0mCtBkHTCibAHPREXyzdJ53\\\\\\\/RAFpfBRXDJ4W1XZH1nsLOf\\\\\\\/6E8P2\\\\\\\/68GBNyD8OD5z2rhJT5Y9lY5n0JyI3Ud+Cds6PUXZpWVdg2Bn9VXagksKYnn7PxpXsgyERYMA13JK4PeJ6msmPO7uEH2SrCwcR6+kS4KpZx4DYhSiEjCguQqSiGmWO3dttIPlMcbwzs4H2Wn8A8K67FWRyJqYzXcrd1cYQ6ORfmjcWnOFcD7CpV9u6zjV\\\\\\\/pjbGxqfLISTRvn45z8QNQk5rtG97sD1D9ovMsLWVpqbKVKa0eqcbL1TWAfaqSJ2JQcrPnpxu\\\\\\\/1XjF2axHK5pEC1mgeCLXuXBRt92YzSw7eniQTsja5z2q16UA9IhZ\\\\\\\/Rs5YRmcoYk2nWaTsZM+zUG2vXOCQ60zYSZPRJZ8Wngt8Nxv4Bf5PS2eqkCmfvgissdZos7bJQVz1wC7F5UvD2Kn7tq\\\\\\\/+u79XicALofT5z4yav1gTHF\\\\\\\/ZvxGnOhGrF5yGmIVn4UusXix\\\\\\\/QNAmljNVV6cWLnr2pzUn5+7f53Rg3Pbx7OqRg\\\\\\\/5BDW5\\\\\\\/9qqRcLN3htJlTjtSFyHl8aT83NRX8C07hAW47j4IJI0NSJOYGKpFA36IvqWHEjC8i7M1triRB\\\\\\\/vS8tMg4RPEXQL+bgPVCRCcLIubdwaf1Iic9SPAtsZu86AjQ57Y4E2u\\\\\\\/uF79Vg1iP2Br+npufVVHKXrkUFwrl\\\\\\\/esfATDJ4Wv4qklz\\\\\\\/lLywLOoXETz8mXh5DCaNZ1rRWUbE3HzAJ5\\\\\\\/IifDhinBqxY434\\\\\\\/\\\\\\\/tmhwhU9wjMjyL+CpFB5Y1QViELS\\\\\\\/kdSZWDWEEWZ3NurQ64JS0rJhwblouwX3SmVZHHr1ygh3HCUxc+IyZrDdRYBl4pZfBGPpkTetrrrcd5YmqZf6Kpzi3q31FK9bS0DW7hTDYahEUhXy3Zk0JkiKryRrpQ6Jpj1iAfEWwQ6FfZHTLCC7KzqUA6ec+XJw4CzFQJLI6tg5iRrg8iCN9MWFsFXqrTFS1wBMhqNWOEjqfrtP5vBL\\\\\\\/RQPmJOB2IKOi3gtT4UHm5I3+PNdmcWWhbFImDG\\\\\\\/PlYj3np4510DTGhDVdNI0M+S2S+5SXPnTLS+y3QXCn36y9yDMadjbBjAJL5kbcUt6Ri9SitjEUjLeGBpanQFr0CLs4gwzfNp+dcCAzL3FovHXvUcklQVjR1RyXnSqv0dDJ8TfQyF6ca7q+0U1Ps1LnrZWH4wYBg2X1av\\\\\\\/ZctdYZURHfbEKh3vxb8h40OzFR3OprkN6OqrEdhWNsu64kYgLDSTQD6I\\\\\\\/gt21m+Ns8StbuN2EBKeQ8yZDTxRouYOWebB+IDb+\\\\\\\/EygGhFGT6TacWkbytMczGV7vcNB2d2No34ochXuYaQp8Uc0YtrsZfYj14R8GGggzO6T5LJlhTnbkArKAvrNS1T9Vzp+gSj4CyocOm8oIx5yJIuHha3DAOJ6eQwgTwEjLQlOEUZY\\\\\\\/Gs68PnAXEDUoqOQfFXckjpf2IaUSGItXLj3yC6EBSg2fWvr62BujwDi+7Sns4F9ko9hNjODTlYDYjZTBYnW8hZZNgWRyfhVrUNANse3ff9TRUMnKoOTLg71bFEqdq8yoK7YXtCHPWsJwaK62gm3LdWGkmTCXl8ZYXj3XZSLHCqSiMBFFEdd6xnKsRDJZ5sWbgN2VPBRM\\\\\\\/RTuHYQBgpkmlM5f2W9WhzBeGDwWA==\\\u0022,\\\u0022iv\\\u0022:\\\u00227nP34zhqu7EPmk+cEgVoaA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022780c864ae5ec4a7e61c7302b82d3e69806893cebf5077083c2fe2e2a73b142f1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ojaB3\\\\\\\/gg58alV+0Pg28oiotrkrBC3wLMpVhsz27FNOQqbN9lsuSfH5FiWB3KDsP6Y1zOBrlhaS6eRZfPFep2XAuxxMI0kFFLB3WkPeoAmnpwPwExxb0rdCy6WfpsrfCl1XOmqoAoVD6tjNb3o4d7JpmT0OkPpQGqh2dtWFT5ULLYmSsIbJ6kvK3Ong+qq57FUaTSgEWdXVEtc4TzTdwHQ5Q0kOmG65wFbUerMaw0aLtYXlUzMns97v0gbxHNiZeKKfa8oeEhUHZFAoU825Wxb+o\\\\\\\/26v7EjfE\\\\\\\/JPqisUwuaF+c8S0cvFCUI17T7fCPhEg1GmS2OqboJ7E8AwceB4OBlLPIjeQ+7SfOlPH292F9IWrmW439zdBExkAs9rBy9RAqWHT5cimibJad408+E6IvS6er8bJrXtmplmmUQFuQVFNu4+5joqcwwja+YbDVm5cosZnG0Jr7PGgB2i\\\\\\\/NrzSJf+EG+h1aR6PkMt74CCVLwaU6u7khu8pz++kd+Re2aCHkXuPeh5oR1iCc65oIdbtIkRChHoJNA52fKUeWwtCqCcWPbeAiAZA3oqSN1hJGf2WN6YNYYAze2nXmab7QEHU4O94NsVmwUZ\\\\\\\/hKyFtySRn\\\\\\\/tKgF3dSENGtwYGPC+6mPk4BDSH+S15t5LSSWH3Gkh834AwaAYkTnii9ktj1qfAs6141z\\\\\\\/fdvYYdUwdXXsoHxHiE9PfPHCwtEyVfyyj7QmFa47IsRthi1Hlb2A7fQOl1vEezQZcesqdwdRsn52Rwr+u\\\\\\\/tXY44TMmLuqDQq0S\\\\\\\/7ekcXlhHU6elLKAODpwq1h+riI7WceozxejjQXdWSiK1px7Z5VTjnQ6D0pOuHPVs2i+nnGXlzYfOOg4YvdSPi02LxcwGS6XHGgcV5TxehrgDE0eRM8iKB9aydqRPTVAlOFamhK4M7bXX+eLcoBUX2p3ylDbmrquwxex6UPDrMK1oIRj7CqX08x5RzdBHoLI7QDkb0dycRTbCN7Tq+11SMIiojQCWfjVEqFMs\\\\\\\/F685Mi8i7IYQpHAE1AqjiuXvGUO2JYl7Enz47K96T+McvyO5QIzimWdqgAjkhpHECCSAEmsX26NAUPw\\\\\\\/M6jWi2x+xqdqsuLqz+v62GIa78dzRiVRFEgAu+62FZ7ikcsqqoyDSaokCEWecJp4dr3S7npE3ZY98ZoPsKUYrbNnpKNb+dvdWfkdpAAVj9TmheJt\\\\\\\/MOP47EfBdLo74\\\\\\\/OAYOiL2ANxyZltajtSDk63kaaMuOTKpE3MOSuAtlEf0QSWYELI2YFcpJFpKmLk2kYIxkmI75xbfMCG99GmZdwB3OPFOV78\\\\\\\/LAs1My+AlQjtUepE7yC74oeYNDr8nJVSOSTcvePJLo\\\\\\\/RerxsNT2APVuXgDPJmhCQuocDod8mnaEgO+egXq6LUjz4A9AVrQ7XB558Z4tgMmsZ+MJXhlViiAz48KEYYO8zQ8uiMGaCVl63dpA8wYwxBqLxeC21sDzLGmeWfkmQb5MdrDeEx6bq9PLuUIVzp8QRINc54UlBaSMPy\\\\\\\/HKo1H+Br9ujNlWLEBWr69OktN6dhlgx+lWYnpl3m4ki+IFu8VaktY3pj1NftlxFqH2wiss8XLYN7E4Um0nDQjLfkrOD3WgJdiSAOGUz1zDtGeL\\\\\\\/2nJEEdkCB0SjYCVX6E8d2NP45Sjsr1Qf2+xkT4obyih7VIlZja2d4EBgGguUzmZQduRSE0j\\\\\\\/clmT2xw0huSgNPyPmdGr1WaubE5ar3nsX09NHUIUQekKD4kRtq9pUp6vsHWQ7l5\\\\\\\/GB2C040Ec9GEv5Unu7pdIwIiq7retebCpJqyqWmRT3d2tz6PMmGyzuB87IN+z7Ho3CibcR4Y\\\\\\\/ULCbM8PpE9yyKZxx1H5AJH9DZZ2uLwN8SUyHvMdr3cAW48UypbwcdAm6ggDcfzLHayCD0sQ1UDfqc7CsDocpUFDLf8n0o5JIyP6sR29IXHWCzye4KEQlqLNreAMSCoWRDhi79AB\\\\\\\/7AzCscHgiVY9NMbzDaZWBLpdlHg==\\\u0022,\\\u0022iv\\\u0022:\\\u00227Fi\\\\\\\/wVaLvHVdmxwjG3UkTg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fa199dfbd3dfadeb8bf2398c72a6a557edad488cdf8babca90cf3f79870844d3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492863 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1693" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SJsCb0Nt3ftrakf7ZDNtj3hmrZclMYf8euDKvAR8lEs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1693" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iWRCE_HWSo6vm7lvsrtZ244eIAp_f-47SR65OjHZL6Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1679" title="HSB-1 Inhibition and HSF-1 Overexpression Trigger Overlapping Transcriptional Changes To Promote Longevity in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1707" title="Determinants of QTL Mapping Power in the Realized Collaborative Cross" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492863" id="node492863" data-pisa="ggg;9/5/1693" data-pisa-master="ggg;g3.119.400132" data-apath="/ggg/9/5/1693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8357-117X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kang Huang</span>, <span class="highwire-citation-author" data-delta="1">Tongcheng Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5909-1224" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Derek W. Dunn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4433-382X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pei Zhang</span>, <span class="highwire-citation-author" data-delta="4">Xiaoxiao Cao</span>, <span class="highwire-citation-author" data-delta="5">Rucong Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-7430-3889" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Baoguo Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1693-1706;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400132 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492863" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kang Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kang%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKang%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8357-117X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kang Huang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tongcheng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tongcheng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATongcheng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Derek W. Dunn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BW.%2BDunn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunn%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADerek%2BW.%2BDunn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5909-1224" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Derek W. Dunn</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Pei Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pei%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APei%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4433-382X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pei Zhang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xiaoxiao Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoxiao%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoxiao%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rucong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rucong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARucong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Baoguo Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming, China, 650223</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baoguo%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABaoguo%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7430-3889" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Baoguo Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:baoguoli@nwu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">baoguoli@nwu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1693" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492863" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1693.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492863" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1693.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492863" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1693.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492863" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Polyploids are organisms whose genomes consist of more than two complete sets of chromosomes. Both autopolyploids and allopolyploids may display polysomic inheritance. A peculiarity of polysomic inheritance is multivalent formation during meiosis resulting in double-reduction, which occurs when sister chromatid fragments segregate into the same gamete. Double-reduction can result in gametes carrying identical-by-descent alleles and slightly increasing homozygosity. This will cause the genotypic frequencies to deviate from expected values and will thus bias the results of standard population genetic analytical methods used in molecular ecology and selective breeding. In this study, we extend existing double-reduction models to account for any even level of ploidy, and derive the symbolic expressions for genotypic frequencies via two methods. Inbreeding coefficients and heterozygosity under double-reduction and inbreeding are also calculated. Numerical solutions obtained by computer simulations are compared with analytical solutions predicted by the model to validate the model.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polysomic-inheritance" class="hw-term hw-article-keyword hw-article-keyword-polysomic-inheritance" rel="nofollow">Polysomic inheritance</a></li><li class="kwd"><a href="/keyword/double-reduction" class="hw-term hw-article-keyword hw-article-keyword-double-reduction" rel="nofollow">double-reduction</a></li><li class="kwd"><a href="/keyword/genotypic-frequency" class="hw-term hw-article-keyword hw-article-keyword-genotypic-frequency" rel="nofollow">genotypic frequency</a></li><li class="kwd"><a href="/keyword/inbreeding-coefficient" class="hw-term hw-article-keyword hw-article-keyword-inbreeding-coefficient" rel="nofollow">inbreeding coefficient</a></li><li class="kwd"><a href="/keyword/heterozygosity" class="hw-term hw-article-keyword hw-article-keyword-heterozygosity" rel="nofollow">heterozygosity</a></li></ul><p id="p-3">Polyploids are organisms whose genomes consist of more than two complete sets of chromosomes (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Madlung 2013</a>). They represent a significant proportion of plant species, with 30–80% of angiosperm species showing at least some polyploidy (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burow <em>et al.</em> 2001</a>) and most lineages showing evidence of paleoploidy (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Otto 2007</a>). Polyploid plants can arise spontaneously in nature by several mechanisms, including meiotic or mitotic failures, and fusion of unreduced gametes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Comai 2005</a>).</p><p id="p-4">There are two distinct mechanisms of genome duplication that result in polyploidy: allopolyploidy and autopolyploidy. Autopolyploids are thought to usually arise within a species by the doubling of structurally similar homologous genomes, whereas allopolyploids arise via interspecific hybridization and subsequent doubling of non-homologous genomes (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Parisod <em>et al.</em> 2010</a>). Both autopolyploids and allopolyploids can be found among both wild and domesticated plant species. Although rare, polyploidy is also found in a few species of vertebrates such as some salmonid fish (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Limborg <em>et al.</em> 2017</a>), the weather loach (<em>Misgurnus anguillicaudatus</em>) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhou <em>et al.</em> 2016</a>), the common carp (<em>Cyprinus carpio</em>) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">David <em>et al.</em> 2003</a>), and the African clawed frog (<em>Xenopus laevis</em>) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Session <em>et al.</em> 2016</a>).</p><p id="p-5">In autopolyploids, more than two homologous chromosomes can pair at meiosis, resulting in the formation of multivalents and polysomic inheritance (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rieger <em>et al.</em> 1968</a>). A peculiarity of polysomic inheritance is the possibility that a gamete inherits a single gene copy twice, termed double-reduction (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Butruille and Boiteux 2000</a>). For example, an autotetraploid individual <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-1.gif"/></noscript></span></span> produces a gamete <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-2.gif"/></noscript></span></span>. In prophase I, crossovers can happen between the locus and the centromere, resulting in an exchange of chromatid fragments between pairing chromosomes. In a multivalent configuration, the separation of chromosomes can be either disjunctional or nondisjunctional (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">De Silva <em>et al.</em> 2005</a>). For nondisjunctional separation, chromosomes that were previously paired in Prophase I segregate into a single secondary oocyte. Double-reduction occurs when fragments of sister chromatids further segregate into a single gamete.</p><p id="p-6">Double-reduction arises from a combination of three major events during meiosis: crossing-over between non-sister chromatids, an appropriated pattern of disjunction, and the subsequent migration of the chromosomal segments carrying a pair of sister chromatids to the same gamete (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Darlington 1929</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Haldane 1930</a>). Multivalent chromosome pairing during meiosis can also occur in allopolyploids, resulting in a mixed inheritance pattern across loci in the genome, termed segmental allopolyploidy (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stebbins 1950</a>).</p><p id="p-7">Geneticists have developed several mathematical models to simulate double-reduction. For instance, for tetrasomic inheritance, the rate of double-reduction <em>α</em> is assumed to have a minimum value of 0 under <em>random chromosome segregation</em> (RCS, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Muller 1914</a>). This increases to <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-3.gif"/></noscript></span></span> with <em>pure random chromatid segregation</em> (PRCS, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Haldane 1930</a>), and to <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-4.gif"/></noscript></span></span> with <em>complete equational segregation</em> (CES, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mather 1935</a>). A diagram of these models can be found in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1693/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagrams of double-reduction models under tetrasomic inheritance. The leftmost column shows the primary oocytes, the middle column shows the secondary oocytes (A and C) or tetrad (B) and the rightmost column shows the gametes. The gametes with a gray background carry identical-by-double-reduction alleles. Dashed lines denote cellular fission, solid lines denote the arms of the chromosome, and the circles connecting solid lines denote the centromere. The target locus is located in the long arm of the chromosome and the identical-by-descent alleles are denoted by the same letter. (A) Random chromosome segregation (RCS) ignores the crossover between the target locus and the centromere (Muller 1914). In the absence of crossing over, gametes may originate from any combination of homologous chromosomes, and two sister chromatids never sort into the same gamete (Parisod et al. 2010); (B) Pure random chromatid segregation (PRCS) accounts for the crossing over between the target locus and the centromere, and assumes the chromatids behave independently and randomly segregate into gametes (Haldane 1930). When sister chromatids segregate into the same gamete, double-reduction occurs. The probability that the two chromatids in a gamete are sister chromatids is 4 (the number of sister chromatids pairs) divided by  (the number of ways to sample two chromatids from eight chromatids), which is equal to . (C) Complete equational segregation (CES), homologous chromosomes pair and chromatids exchange via recombination (Mather 1935). The whole arms of sister chromatids are exchanged into different chromosomes. The probability that two homologous chromosomes within a single secondary oocyte were previously paired at a target locus in prophase I is . In this case, these sister chromatid fragments will segregate in a single gamete at a ratio of , so the rate of double-reduction is thus  for tetrasomic inheritance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2103631657" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Diagrams of double-reduction models under tetrasomic inheritance. The leftmost column shows the primary oocytes, the middle column shows the secondary oocytes (A and C) or tetrad (B) and the rightmost column shows the gametes. The gametes with a gray background carry identical-by-double-reduction alleles. Dashed lines denote cellular fission, solid lines denote the arms of the chromosome, and the circles connecting solid lines denote the centromere. The target locus is located in the long arm of the chromosome and the identical-by-descent alleles are denoted by the same letter. (A) Random chromosome segregation (RCS) ignores the crossover between the target locus and the centromere (Muller 1914). In the absence of crossing over, gametes may originate from any combination of homologous chromosomes, and two sister chromatids never sort into the same gamete (Parisod &lt;em&gt;et al.&lt;/em&gt; 2010); (B) Pure random chromatid segregation (PRCS) accounts for the crossing over between the target locus and the centromere, and assumes the chromatids behave independently and randomly segregate into gametes (Haldane 1930). When sister chromatids segregate into the same gamete, double-reduction occurs. The probability that the two chromatids in a gamete are sister chromatids is 4 (the number of sister chromatids pairs) divided by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-5&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (the number of ways to sample two chromatids from eight chromatids), which is equal to &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-6&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. (C) Complete equational segregation (CES), homologous chromosomes pair and chromatids exchange via recombination (Mather 1935). The whole arms of sister chromatids are exchanged into different chromosomes. The probability that two homologous chromosomes within a single secondary oocyte were previously paired at a target locus in prophase I is &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-7&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. In this case, these sister chromatid fragments will segregate in a single gamete at a ratio of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-8&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, so the rate of double-reduction is thus &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for tetrasomic inheritance.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1693/F1.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1693/F1.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1693/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1693/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493003" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Diagrams of double-reduction models under tetrasomic inheritance. The leftmost column shows the primary oocytes, the middle column shows the secondary oocytes (A and C) or tetrad (B) and the rightmost column shows the gametes. The gametes with a gray background carry identical-by-double-reduction alleles. Dashed lines denote cellular fission, solid lines denote the arms of the chromosome, and the circles connecting solid lines denote the centromere. The target locus is located in the long arm of the chromosome and the identical-by-descent alleles are denoted by the same letter. (A) Random chromosome segregation (RCS) ignores the crossover between the target locus and the centromere (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Muller 1914</a>). In the absence of crossing over, gametes may originate from any combination of homologous chromosomes, and two sister chromatids never sort into the same gamete (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Parisod <em>et al.</em> 2010</a>); (B) Pure random chromatid segregation (PRCS) accounts for the crossing over between the target locus and the centromere, and assumes the chromatids behave independently and randomly segregate into gametes (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Haldane 1930</a>). When sister chromatids segregate into the same gamete, double-reduction occurs. The probability that the two chromatids in a gamete are sister chromatids is 4 (the number of sister chromatids pairs) divided by <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-5.gif"/></noscript></span></span> (the number of ways to sample two chromatids from eight chromatids), which is equal to <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-6.gif"/></noscript></span></span>. (C) Complete equational segregation (CES), homologous chromosomes pair and chromatids exchange via recombination (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Mather 1935</a>). The whole arms of sister chromatids are exchanged into different chromosomes. The probability that two homologous chromosomes within a single secondary oocyte were previously paired at a target locus in prophase I is <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-7.gif"/></noscript></span></span>. In this case, these sister chromatid fragments will segregate in a single gamete at a ratio of <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-8.gif"/></noscript></span></span>, so the rate of double-reduction is thus <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/F1/embed/mml-math-9.gif"/></noscript></span></span> for tetrasomic inheritance.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Double-reduction results in gametes carrying <em>identical-by-descent</em> (IBD) alleles and slightly increased homozygosity (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hardy 2016</a>). Experiments aimed at estimating the frequency of double-reduction in autotetraploids have yielded values ranging from 0 to almost 0.30 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2001</a>). It has a similar effect as inbreeding, and can cause a deviation in genotypic frequencies from the Hardy-Weinberg equilibrium (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Luo <em>et al.</em> 2006</a>). <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fisher (1943)</a> first studied the Mid gene of <em>Lythrum salicaria</em>, and produced an analytical method to estimate genotypic frequencies for tetrasomic inheritance for a biallelic locus. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Geiringer (1949)</a> extended this work to a multiallelic locus for tetrasomic inheritance. However, the genotypic frequencies for polysomic inheritance for higher levels of ploidy and for a multiallelic locus have been neglected.</p><p id="p-10">Genotypic frequencies are important in population genetics, molecular ecology and molecular breeding, with the <em>Hardy-Weinberg equilibrium</em> (HWE) being the basal standard for analytical methods in disomic inheritance. Several methods assume that loci conform to the HWE for the calculations of certain model parameters or statistics, for example, the likelihood of a genotype (population assignment, Bayesian clustering, parentage analysis), the statistics measuring the deviation from expectations (linkage disequilibrium test, differentiation test, heterozygote deficiency/excess test, bottle-neck effect test), and other parameters (<em>e.g.</em>, individual inbreeding coefficient, relatedness coefficient, kinship coefficient, QTL mapping). Although some analytical methods have been developed for organisms with polysomic inheritance (<em>e.g.</em>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Clark and Jasieniuk 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hardy and Vekemans 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2015</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meirmans and van Tienderen 2004</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pritchard <em>et al.</em> 2000</a>), none of these methods accounts for double-reduction, and thus in the presence of double-reduction will produce misleading results. For example, in tetrasomic inheritance, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-10.gif"/></noscript></span></span> may produce an offspring <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-11.gif"/></noscript></span></span>, and both parents are excluded as the true parents in parentage analysis.</p><p id="p-11">Due to the genotypic ambiguity of polyploid species, the correct genotype cannot be obtained because some genotypes share a common electrophoretic band type in PCR-based co-dominant markers (<em>e.g.</em>, microsatellites) and the allelic dosage is unknown (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hardy 2016</a>). For example, in autotetraploids, the three genotypes <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-12.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-14.gif"/></noscript></span></span> have a same electrophoretic band type of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-15.gif"/></noscript></span></span>. We define the set of alleles within an individual as its phenotype.</p><p id="p-12">Here, we extend an existing double-reduction model to incorporate the recombination rate into CES, and show two methods to derive the symbolic expressions for genotypic frequencies at equilibrium. To enable other researchers make similar calculations, we make available a C++ source code that enables the calculation of genotypic/phenotypic frequencies of gametes/zygotes at equilibrium to a maximum ploidy level of ten.</p><div class="section other1" id="sec-1"><h2 class="">Double-reduction models</h2><p id="p-13">In this paper, we assume that the population size is sufficiently large and there are no differences in selection between both zygotes and gametes. All zygotes will thus have an equal opportunity to mature and reproduce, and all gametes will have an equal opportunity to become fertilized. Therefore, the zygote frequencies are equal to the genotypic frequencies of reproducing individuals, and as a result, we use the zygote frequencies to indicate the genotypic frequencies of reproducing individuals. The double-reduction rates (<em>i.e.</em>, alpha) are assumed to be equal among reproducing individuals (<em>e.g.</em>, in females and males). Mutation and migration are not included in our model, whereas inbreeding is included for analyzing the influence of inbreeding and double-reduction on the inbreeding coefficient and for estimating heterozygosity. Organisms with odd levels of ploidy are not considered because of their inherent inability to produce euploid gametes — these organisms are sterile.</p><div id="sec-2" class="subsection"><h3>Transitional probability from zygotes to gametes</h3><p id="p-14">The transitional probability from a zygote to a gamete is the probability that a zygote produces a gamete of a certain genotype, which can be used to find the double-reduction rates of PRCS and to derive the analytical expression of gamete frequencies in the first step of the non-linear method (see Non-linear method section). Such a probability can also be applied to some population genetics studies such as parentage analysis, and the calculation of segregation ratios in mapping populations (see Discussion).</p><p id="p-15">If the rate of double-reduction is greater than zero, the gametes will carry <em>identical-by-double-reduction</em> (IBDR) alleles. For tetrasomic and hexasomic inheritance, there are only two and three allele copies within a gamete, respectively. Hence, it can be found at most one pair of IBDR alleles within a gamete, and a single parameter can be used to measure the rate of double-reduction.</p><p id="p-16">In polysomic inheritance with a higher level of ploidy, there may be more than one pair of IBDR alleles in a gamete. Therefore, it is necessary to add some additional parameters in the segregation ratios. The transitional probability from zygotes to gametes for tetrasomic and hexasomic inheritances at a biallelic locus (the Mid gene in <em>Lythrum salicaria</em>) has been studied previously by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fisher and Mather (1943)</a>. Here, we extend the work of Fisher and Mather and use a general mathematical expression of this transitional probability. There are at most <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-16.gif"/></noscript></span></span> pairs of IBDR alleles within a gamete, where v is the ploidy level of the resulting zygote. Let <em>G</em> and g be the genotypes of a zygote and a gamete, respectively, and let <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-17.gif"/></noscript></span></span> be the probability that the gamete carries <em>i</em> pairs of IBDR alleles, then <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-18.gif"/></noscript></span></span>. Suppose that <em>h</em> is the number of alleles at target locus, then the probability that <em>G</em> produces g is the following weighted sum:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-21.gif"/></noscript></span></span> are the numbers of copies of the <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-22.gif"/></noscript></span></span> allele (say <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-23.gif"/></noscript></span></span>) in <em>G</em> and g, respectively, and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-24.gif"/></noscript></span></span> is the number of IBDR allele pairs <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-25.gif"/></noscript></span></span> in g, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-26.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-27.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-29.gif"/></noscript></span></span>.</p><p id="p-17">The symbol <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-30.gif"/></noscript></span></span> in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> is a binary variable, which is used to limit the range of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-31.gif"/></noscript></span></span>, so as to avoid some invalid situations. Note that the combination <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-32.gif"/></noscript></span></span> implies <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-33.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-34.gif"/></noscript></span></span> implies <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-35.gif"/></noscript></span></span>, or equivalently it implies <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-36.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-37.gif"/></noscript></span></span>. Hence <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-38.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-39.gif"/></noscript></span></span>, otherwise <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-40.gif"/></noscript></span></span>.</p><p id="p-18">In <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>, for every <em>i</em> (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-41.gif"/></noscript></span></span>), two events need consideration. First, is the event that g carries <em>i</em> pairs of IBDR alleles. Second, is the event that those <em>i</em> pairs of IBDR alleles are distributed into <em>h</em> kinds of alleles. Because both types of event are mutually exclusive, the products of the probabilities of the two events stated above are summed to obtain <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-42.gif"/></noscript></span></span>.</p><p id="p-19">Each chromosome is duplicated during meiosis, so each allele in <em>G</em> becomes a pair of duplicated alleles. For the <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-43.gif"/></noscript></span></span> kind of allele, first we sample <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-44.gif"/></noscript></span></span> pairs from the <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-45.gif"/></noscript></span></span> pairs of duplicated alleles <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-46.gif"/></noscript></span></span>, then there are <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-47.gif"/></noscript></span></span> ways (where <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-48.gif"/></noscript></span></span>), and so the first combination in the numerator of <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> is obtained. We then sample the non-IBDR alleles to form g. The remaining is <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-49.gif"/></noscript></span></span> pairs of duplicated alleles <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-50.gif"/></noscript></span></span>, which still requires <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-51.gif"/></noscript></span></span> copies of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-52.gif"/></noscript></span></span>. Then we sample <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-53.gif"/></noscript></span></span> pairs of duplicated alleles <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-54.gif"/></noscript></span></span>, and so there are <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-55.gif"/></noscript></span></span> ways (where <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-56.gif"/></noscript></span></span>), which obtains the second combination in the numerator of <a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>. Moreover, if we sample one copy of <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-57.gif"/></noscript></span></span> in each pair of the <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-58.gif"/></noscript></span></span> pairs of duplicated alleles, then there are altogether <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-59.gif"/></noscript></span></span> possible combinations. Because each sampling combination is independent for different kinds of alleles, the total number of combinations is the product <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-60.gif"/></noscript></span></span>.</p><p id="p-20">In the following example, we consider the total number of combinations to produce a gamete consisting of <em>i</em> pairs of IBDR alleles without accounting for the specific genotypes <em>G</em> and g. Similar to the above process, we first sample <em>i</em> pairs from the v pairs of duplicated alleles, and these alleles will become the IBD alleles in g. We then sample <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-61.gif"/></noscript></span></span> pairs from the remaining <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-62.gif"/></noscript></span></span> pairs, and then one allele in each pair of the <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-63.gif"/></noscript></span></span> pairs is further sampled to form g. The total number of allele combinations is <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-64.gif"/></noscript></span></span>. Here the coefficient <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-65.gif"/></noscript></span></span> is eliminated in the fraction in <a id="xref-disp-formula-1-5" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>.</p><p id="p-21"><a id="xref-disp-formula-1-6" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> is a general expression of the transitional probability, and can be applied at multiallelic loci and at any even level of ploidy. As an example, the transitional probability from zygotes to gametes in octosomic inheritance at a biallelic locus is shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Another example is shown in Appendix A, which is the derivation of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-66.gif"/></noscript></span></span>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493064/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493064/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Transitional probability from zygotes to gametes in octosomic inheritance at a biallelic locus</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Random chromosome segregation</h3><p id="p-22">The random chromosome segregation model ignores the crossover between the target locus and the centromere, and the rate of double-reduction (α) is equal to zero (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Muller 1914</a>). In the absence of crossing over, gametes may originate from any combination of homologous chromosomes, and two sister chromatids never sort into the same gamete (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Parisod <em>et al.</em> 2010</a>, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Therefore, genotypic frequencies concur with the HWE. Here, the HWE is expanded to account for polysomic inheritance, in which the alleles in a genotype are independent and randomly appear according to their frequencies. Thus, the genotypic frequency under the RCS can be expressed as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-112.gif"/></noscript></span></span>Where <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-113.gif"/></noscript></span></span> denotes the frequency of <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-114.gif"/></noscript></span></span>. For example, the frequency of a tetraploid genotype <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-115.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-116.gif"/></noscript></span></span>.</p><p id="p-23">When the conditions of the RCS are met, <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-117.gif"/></noscript></span></span> whenever <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-118.gif"/></noscript></span></span>, then <a id="xref-disp-formula-1-7" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> can be simplified to</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-119.gif.backup.1557862297.8459"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-119.gif.backup.1557862297.8459"/></noscript></span><span class="disp-formula-label">(2)</span></span></div><div id="sec-4" class="subsection"><h3>Pure random chromatid segregation</h3><p id="p-24">Pure random chromatid segregation accounts for crossing over and assumes the chromatids behave independently in meiotic anaphases, and randomly segregate into gametes (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Haldane, 1930</a>, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Then the probability that a zygote <em>G</em> produces a gamete <em>g</em> at a multiallelic locus can be derived by sampling <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-120.gif.backup.1557862297.8844"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-120.gif.backup.1557862297.8844"/></noscript></span></span> chromatids (or allele copies) in a total number of <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-121.gif.backup.1557862297.9114"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-121.gif.backup.1557862297.9114"/></noscript></span></span> chromatids (or allele copies). Because the number of chromatids is twice the number of chromosomes, the transitional probability <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-122.gif.backup.1557862297.9328"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-122.gif.backup.1557862297.9328"/></noscript></span></span> can be modified from <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation (2)</a> by duplicating the terms <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-123.gif.backup.1557862297.9666"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-123.gif.backup.1557862297.9666"/></noscript></span></span> and v in those combinations as follows:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-124.gif.backup.1557862297.994"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-124.gif.backup.1557862297.994"/></noscript></span><span class="disp-formula-label">(3)</span></span>The value of alpha of PRCS can be derived by simulating the meiosis, first by sampling <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-125.gif.backup.1557862298.0244"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-125.gif.backup.1557862298.0244"/></noscript></span></span> IBDR allele pairs out of v IBDR allele pairs, and then by segregating <em>i</em> IBDR allele pairs and <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-126.gif.backup.1557862298.0613"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-126.gif.backup.1557862298.0613"/></noscript></span></span> non-IBDR alleles into the gamete. Therefore, the value of <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-127.gif.backup.1557862298.0946"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-127.gif.backup.1557862298.0946"/></noscript></span></span> is given by<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-128.gif.backup.1557862298.1234"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-128.gif.backup.1557862298.1234"/></noscript></span></span>The values of alpha under PRCS, ranging from tetrasomic to dodecasomic inheritance, are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The expected number of IBDR allele pairs in a gamete <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-129.gif.backup.1557862298.1763"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-129.gif.backup.1557862298.1763"/></noscript></span></span>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493205/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493205/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><strong>The probabilities of a gamete carrying <em>i</em> pairs of identical-by-double-reduction alleles (</strong><span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/T2/embed/mml-math-130.gif.backup.1557862298.2158"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/T2/embed/mml-math-130.gif.backup.1557862298.2158"/></noscript></span></span><strong>) and the inbreeding coefficient in an outcrossed population with pure random chromatid segregation (PRCS), complete equational segregation (CES) or partial equational segregation (PES)</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Complete equational segregation</h3><p id="p-26">The <em>complete equational segregation</em> (CES) assumes that the whole arms of the two pairing chromatids are exchanged between pairing chromosomes (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Mather 1935</a>). In <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>, the leftmost two chromosomes in the primary oocyte are paired in the long arms, as well as the rightmost two chromosomes. In Metaphase I, the chromosomes randomly segregate into the secondary oocytes. If the pairing chromosomes segregate in the same secondary oocyte, then the duplicated alleles may further segregate into a single gamete.</p><p id="p-27">The production of double-reduction gametes requires the fulfillment of two conditions: (i) the chromosomes paired at target locus segregate into the same secondary oocyte, and (ii) the duplicated alleles segregate into the same gamete. In order to construct a mathematical model of CES, we simulate this meiosis, and the values of alpha will be derived by using a two-step process.</p><p id="p-28">First, we model the probability, <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-191.gif.backup.1557862298.2477"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-191.gif.backup.1557862298.2477"/></noscript></span></span>, that <em>j pairs of chromosomes paired at the target locus</em> (PCPs) segregate into a secondary oocyte. There are <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-192.gif.backup.1557862298.2837"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-192.gif.backup.1557862298.2837"/></noscript></span></span> PCPs in the primary oocyte, so we have <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-193.gif.backup.1557862298.3114"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-193.gif.backup.1557862298.3114"/></noscript></span></span> combinations to sample <em>j</em> pairs. Next, for the remaining <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-194.gif.backup.1557862298.3558"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-194.gif.backup.1557862298.3558"/></noscript></span></span> PCPs, it still requires <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-195.gif.backup.1557862298.3819"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-195.gif.backup.1557862298.3819"/></noscript></span></span> chromosomes that are not paired with each other at target locus during prophase I to form the secondary oocyte. We therefore (i) sample <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-196.gif.backup.1557862298.4112"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-196.gif.backup.1557862298.4112"/></noscript></span></span> PCPs from the remaining <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-197.gif.backup.1557862298.4407"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-197.gif.backup.1557862298.4407"/></noscript></span></span> PCPs (there are <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-198.gif.backup.1557862298.4747"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-198.gif.backup.1557862298.4747"/></noscript></span></span> combinations), and (ii) sample one chromosome from each PCP in the previously sampled <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-199.gif.backup.1557862298.4999"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-199.gif.backup.1557862298.4999"/></noscript></span></span> PCPs (there are <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-200.gif.backup.1557862298.522"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-200.gif.backup.1557862298.522"/></noscript></span></span> chromosome combinations). The product of the above three numbers of combinations is divided by the total number of segregation modes in Metaphase I, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-201.gif.backup.1557862298.5429"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-201.gif.backup.1557862298.5429"/></noscript></span></span>, to derive <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-202.gif.backup.1557862298.5724"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-202.gif.backup.1557862298.5724"/></noscript></span></span>. The process described above can be summarized by the following expression:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-203.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Second, we model the probability, <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-204.gif.backup.1557862298.6009"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-204.gif.backup.1557862298.6009"/></noscript></span></span>, that <em>i</em> pairs of duplicated alleles segregate into a single gamete. There are <em>j</em> PCPs in the second oocyte, and each has four alleles (<em>e.g.</em>, the alleles in both chromosomes are <em>A</em> and <em>B</em> in the top second oocyte of <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). There are four segregation modes for each PCP in Anaphase II, where two of them (<span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-205.gif.backup.1557862298.654"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-205.gif.backup.1557862298.654"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-206.gif.backup.1557862298.6849"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-206.gif.backup.1557862298.6849"/></noscript></span></span>) produce gametes with IBDR alleles and the other two (<span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-207.gif.backup.1557862298.7092"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-207.gif.backup.1557862298.7092"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-208.gif.backup.1557862298.7386"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-208.gif.backup.1557862298.7386"/></noscript></span></span>) pass non-IBDR alleles into the gamete. In order to produce a gamete consisting of <em>i</em> pairs of IBDR alleles, <em>i</em> PCPs are sampled and segregate IBDR alleles. Following the description above, we have <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-209.gif.backup.1557862298.7492"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-209.gif.backup.1557862298.7492"/></noscript></span></span> combinations of sampling <em>i</em> PCPs that pass IBDR alleles into a gamete from <em>j</em> PCPs. Next, the remaining <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-210.gif.backup.1557862298.7874"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-210.gif.backup.1557862298.7874"/></noscript></span></span> PCPs pass non-IBDR alleles, and so there are <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-211.gif.backup.1557862298.811"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-211.gif.backup.1557862298.811"/></noscript></span></span> combinations. In each of the remaining <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-212.gif.backup.1557862298.8332"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-212.gif.backup.1557862298.8332"/></noscript></span></span> chromosomes that are not paired with each other during prophase I, one of the two alleles segregates into the gamete, and hence there are <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-213.gif.backup.1557862298.8595"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-213.gif.backup.1557862298.8595"/></noscript></span></span> allele combinations. The product of these numbers of combinations is divided by the total number of segregation modes in Metaphase II, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-214.gif.backup.1557862298.8828"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-214.gif.backup.1557862298.8828"/></noscript></span></span>, to obtain the expression <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-215.gif.backup.1557862298.9071"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-215.gif.backup.1557862298.9071"/></noscript></span></span>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-216.gif.backup.1557862298.9282"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-216.gif.backup.1557862298.9282"/></noscript></span></span>, which is the value of <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-217.gif.backup.1557862298.9449"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-217.gif.backup.1557862298.9449"/></noscript></span></span> conditional on <em>j</em>, where <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-218.gif.backup.1557862298.9672"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-218.gif.backup.1557862298.9672"/></noscript></span></span>. Because the events that the secondary oocyte contains different numbers of PCPs are mutually exclusive, their probabilities are weighted by <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-219.gif.backup.1557862298.9866"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-219.gif.backup.1557862298.9866"/></noscript></span></span> to obtain the weighted sum, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-220.gif.backup.1557862299.0084"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-220.gif.backup.1557862299.0084"/></noscript></span></span>, whose expression is as follows:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-221.gif.backup.1557862299.0411"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-221.gif.backup.1557862299.0411"/></noscript></span><span class="disp-formula-label">(5)</span></span>Using <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">Equation (5)</a>, the values of alpha of the CES can be derived for different inheritance modes, and these are presented in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. The example deriving the values of alpha under octosomic inheritance is given in Appendix B. The expected number of IBDR allele pairs in a gamete <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-222.gif.backup.1557862299.0719"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-222.gif.backup.1557862299.0719"/></noscript></span></span>.</p></div><div id="sec-6" class="subsection"><h3>Partial equational segregation</h3><p id="p-29">The <em>recombination fraction</em> (denoted by <em>r</em>) between two loci is defined as the ratio of the number of recombined gametes to the total number of gametes produced. The maximum recombination fraction for diploids is <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-223.gif.backup.1557862299.0964"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-223.gif.backup.1557862299.0964"/></noscript></span></span> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xu 2013</a>). However, in autopolyploids, because there are <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-224.gif.backup.1557862299.1121"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-224.gif.backup.1557862299.1121"/></noscript></span></span> PCPs, the maximum recombination fraction for an autopolyploid organism is <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-225.gif.backup.1557862299.1319"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-225.gif.backup.1557862299.1319"/></noscript></span></span>, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-226.gif.backup.1557862299.1509"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-226.gif.backup.1557862299.1509"/></noscript></span></span> for autotetraploids.</p><p id="p-30">We used the <em>single chromatid recombination rate</em> (denoted by <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-227.gif.backup.1557862299.1702"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-227.gif.backup.1557862299.1702"/></noscript></span></span>) between the target locus and the centromere (<span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-228.gif.backup.1557862299.1895"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-228.gif.backup.1557862299.1895"/></noscript></span></span>) to evaluate the degree of recombination in polysomic inheritance, which is the probability that an allele in a chromatid is exchanged with the pairing chromatid. Hence, <em>r</em> and <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-229.gif.backup.1557862299.2137"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-229.gif.backup.1557862299.2137"/></noscript></span></span> can be converted to each other by the formula <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-230.gif.backup.1557862299.2391"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-230.gif.backup.1557862299.2391"/></noscript></span></span>. Moreover, <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-231.gif.backup.1557862299.2744"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-231.gif.backup.1557862299.2744"/></noscript></span></span> can be calculated by Haldane’s mapping function <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-232.gif.backup.1557862299.3013"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-232.gif.backup.1557862299.3013"/></noscript></span></span> where <em>d</em> is the distance between the target locus and the centromere (the unit of length is the centimorgan).</p><p id="p-31">In the CES model, the two alleles in a pair of pairing chromatids are assumed to be exchanged at a probability of 100%. This ‘ideal’ condition is unlikely to occur in nature. We thus usually consider that the maximum single chromatid recombination rate is <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-233.gif.backup.1557862299.3206"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-233.gif.backup.1557862299.3206"/></noscript></span></span>%, and that the distance between the target locus and the centromere is extremely long. Although <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Mather (1935)</a> incorporated the recombination rate into his model, assuming 100% recombination has been widely used in similar studies (<em>e.g.</em>, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Butruille and Boiteux 2000</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2001</a>).</p><p id="p-32">We incorporated the single chromatid recombination rate <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-234.gif.backup.1557862299.3443"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-234.gif.backup.1557862299.3443"/></noscript></span></span> into the CES to allow a greater generalization, which allows for the situation that only some fragments of the two pairing chromatids are exchanged between pairing chromosomes. Such a model is called the <em>partial equational segregation</em> (PES) model.</p><p id="p-33">In the first step for CES, we model the probability <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-235.gif.backup.1557862299.3684"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-235.gif.backup.1557862299.3684"/></noscript></span></span> that <em>j</em> PCPs segregate into a secondary oocyte, which remains unchanged in the PES.</p><p id="p-34">Assuming that the alleles in each PCP are exchanged at a probability of <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-236.gif.backup.1557862299.3923"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-236.gif.backup.1557862299.3923"/></noscript></span></span> between pairing chromatids and those <em>j</em> PCPs are exchanged independently, then the number of <em>exchanged PCPs</em> (EPCPs) is drawn from the binomial distribution, and so the probability <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-237.gif.backup.1557862299.4109"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-237.gif.backup.1557862299.4109"/></noscript></span></span> that the second oocyte contains <em>k</em> EPCPs is<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-238.gif.backup.1557862299.4302"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-238.gif.backup.1557862299.4302"/></noscript></span><span class="disp-formula-label">(6)</span></span>It is noteworthy that only the EPCPs are able to produce IBDR alleles under a PES model. Following the process of the second step for CES, we derive the alpha for PES by: (i) sampling <em>i</em> EPCPs from <em>k</em> EPCPs (there are <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-239.gif.backup.1557862299.4482"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-239.gif.backup.1557862299.4482"/></noscript></span></span> ways to do this); (ii) sampling IBDR alleles from those <em>i</em> EPCPs (there are <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-240.gif.backup.1557862299.4662"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-240.gif.backup.1557862299.4662"/></noscript></span></span> ways to do this); (iii) sampling non-IBDR alleles from the remaining <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-241.gif.backup.1557862299.4871"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-241.gif.backup.1557862299.4871"/></noscript></span></span> EPCPs (there are <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-242.gif.backup.1557862299.5093"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-242.gif.backup.1557862299.5093"/></noscript></span></span> ways to do this); (iv) sampling the remaining non-IBDR alleles from the remaining <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-243.gif.backup.1557862299.5288"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-243.gif.backup.1557862299.5288"/></noscript></span></span> chromosomes (there are <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-244.gif.backup.1557862299.5478"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-244.gif.backup.1557862299.5478"/></noscript></span></span> ways to do this). The product of the numbers of ways to do the above four steps is now divided by the total number of segregation modes in Metaphase II, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-245.gif.backup.1557862299.5694"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-245.gif.backup.1557862299.5694"/></noscript></span></span>, to obtain the expression <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-246.gif.backup.1557862299.6328"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-246.gif.backup.1557862299.6328"/></noscript></span></span>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-247.gif.backup.1557862299.7545"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-247.gif.backup.1557862299.7545"/></noscript></span></span>, which is the value of <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-248.gif.backup.1557862299.7769"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-248.gif.backup.1557862299.7769"/></noscript></span></span> conditional on <em>k</em>, where <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-249.gif.backup.1557862299.7942"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-249.gif.backup.1557862299.7942"/></noscript></span></span>. Moreover, the events that the secondary oocyte contains different numbers of EPCPs are mutually exclusive. Therefore, the weighted sum of their probabilities with <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-250.gif.backup.1557862299.8133"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-250.gif.backup.1557862299.8133"/></noscript></span></span> as their weights are calculated to obtain <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-251.gif.backup.1557862299.8396"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-251.gif.backup.1557862299.8396"/></noscript></span></span> under PES, whose expression is as follows:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-252.gif.backup.1557862299.8892"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-252.gif.backup.1557862299.8892"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-253.gif.backup.1557862299.9085"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-253.gif.backup.1557862299.9085"/></noscript></span></span> is given in <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">Equation (6)</a>.</p><p id="p-35">Using <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation (7)</a>, the values of alpha in the PES model for tetrasomic to dodecasomic inheritance can be derived, and the results are presented in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>. An example deriving the values of alpha under octosomic inheritance is also given in Appendix B. The expected number of IBDR allele pairs in a gamete is <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-254.gif.backup.1557862299.9316"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-254.gif.backup.1557862299.9316"/></noscript></span></span>.</p></div></div><div class="section other2" id="sec-7"><h2 class="">The genotypic frequencies at equilibrium</h2><p id="p-36">Here we present two methods to derive the genotypic frequencies at a multiallelic locus for higher levels of ploidy. The first method, the non-linear method, is modified from <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fisher’s (1943)</a> method and uses the transitional probability from zygotes to gametes to account for the multiallelic loci. Because this method is computationally difficult for a ploidy level higher than six under current conditions, we develop an alternative method to derive the genotypic frequency up to and including decasomic inheritance. The second method, the linear method, uses the solution of the non-linear method at a biallelic locus as the initial solution, and at each step a novel allele is segregated from an existing allele and the genotypic frequencies are updated.</p><div id="sec-8" class="subsection"><h3>Non-linear method</h3><p>For this method, the total probability formula together with <a id="xref-disp-formula-1-8" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> are used to derive the gamete frequencies and the multiplication theorem of probability is used to derive the zygote frequencies. If the genotypic frequencies reach equilibrium, they will not change subsequently, and the system of equations that we will construct can be solved by adding some constraints. In the following text, we denote GFG for <em>genotypic frequencies of gametes</em>, and GFZ for <em>genotypic frequencies of zygotes</em>. Our system of non-linear equations will be presented by the following two steps.</p><ol class="list-romanlower " id="list-1"><li id="list-item-1"><p id="p-38">Simulating meiosis. Given a gamete g, if the genotypic frequencies reach equilibrium, then the GFG is determined by the total probability formula, <em>i.e.</em>, the sum of the transition probabilities <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-255.gif.backup.1557862299.9484"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-255.gif.backup.1557862299.9484"/></noscript></span></span> weighted by GFZ, symbolically<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-256.gif.backup.1557862299.9674"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-256.gif.backup.1557862299.9674"/></noscript></span><span class="disp-formula-label">(8)</span></span>Here <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-257.gif.backup.1557862299.9888"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-257.gif.backup.1557862299.9888"/></noscript></span></span> is taken from all zygote genotypes and <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-258.gif.backup.1557862300.015"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-258.gif.backup.1557862300.015"/></noscript></span></span> can be obtained by <a id="xref-disp-formula-1-9" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>. Because there are <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-259.gif.backup.1557862300.04"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-259.gif.backup.1557862300.04"/></noscript></span></span> gamete genotypes, <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">Equation (8)</a> represents <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-260.gif.backup.1557862300.0609"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-260.gif.backup.1557862300.0609"/></noscript></span></span> equations.</p></li><li id="list-item-2"><p id="p-39">Simulating fertilization. The GFZ in the next generation is determined by the multiplication theorem of probability with GFG as the weight. Under equilibrium, the GFG and GFZ are constant across generations, then for a zygote <em>G</em>, we can establish an equation as follows:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-261.gif.backup.1557862300.0833"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-261.gif.backup.1557862300.0833"/></noscript></span><span class="disp-formula-label">(9)</span></span></p></li></ol><p>where <em>G</em> is regarded as a multiset, <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-262.gif"/></noscript></span></span> is taken from all possible different subsets of <em>G</em> with <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-263.gif.backup.1557862300.1163"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-263.gif.backup.1557862300.1163"/></noscript></span></span> elements and the symbol ∖ denotes the operation of set difference. Here, a <em>multiset</em> is a generalized set, whose elements can be repeated (<em>e.g.</em>, if <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-264.gif.backup.1557862300.1384"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-264.gif.backup.1557862300.1384"/></noscript></span></span>), it can be regarded as a multiset consisting of four elements, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-265.gif.backup.1557862300.1545"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-265.gif.backup.1557862300.1545"/></noscript></span></span>, and its different subsets containing two elements are <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-266.gif.backup.1557862300.1736"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-266.gif.backup.1557862300.1736"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-267.gif.backup.1557862300.1927"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-267.gif.backup.1557862300.1927"/></noscript></span></span>. Moreover, if <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-268.gif.backup.1557862300.21"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-268.gif.backup.1557862300.21"/></noscript></span></span> is regarded as <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-269.gif.backup.1557862300.2313"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-269.gif.backup.1557862300.2313"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-270.gif.backup.1557862300.2503"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-270.gif.backup.1557862300.2503"/></noscript></span></span>. Because there are <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-271.gif.backup.1557862300.2742"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-271.gif.backup.1557862300.2742"/></noscript></span></span> zygote genotypes, <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">Equation (9)</a> represents <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-272.gif.backup.1557862300.3008"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-272.gif.backup.1557862300.3008"/></noscript></span></span> equations. If the double-reduction ratios in two parents are different, then the genotypic frequencies of egg and sperm should be treated separately in <a id="xref-disp-formula-11-2" class="xref-disp-formula" href="#disp-formula-11">Equation (9)</a>, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-273.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-274.gif.backup.1557862300.3239"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-274.gif.backup.1557862300.3239"/></noscript></span></span> are replaced by the frequencies of an egg <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-275.gif.backup.1557862300.3513"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-275.gif.backup.1557862300.3513"/></noscript></span></span> and a sperm <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-276.gif.backup.1557862300.3704"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-276.gif.backup.1557862300.3704"/></noscript></span></span>, respectively, and their frequencies are calculated by substituting different double-reduction parameters into <a id="xref-disp-formula-10-2" class="xref-disp-formula" href="#disp-formula-10">Equation (8)</a>.</p><p id="p-40">The process that transforms the allele frequencies into the GFG can be described by a linear substitution in the sense that every allele frequency can be expressed as a linear combination of the GFG. By substituting the equations determined by <a id="xref-disp-formula-11-3" class="xref-disp-formula" href="#disp-formula-11">Equation (9)</a> into the equations determined by <a id="xref-disp-formula-10-3" class="xref-disp-formula" href="#disp-formula-10">Equation (8)</a>, we obtain a system of non-linear equations with the GFG as unknowns. This system of equations together with those expressions of the linear combinations mentioned above still forms a system of non-linear equations with the GFG as unknowns and the allele frequencies as parameters. For a lower level of ploidy, the latter system of non-linear equations can be solved, whose solution is with parameters. An example is given in Appendix C, which shows how to find the solution of the latter system of non-linear equations (<em>i.e.</em>, how to derive the expressions of GFG with parameters) for tetrasomic inheritance at a triallelic locus.</p><p id="p-41">In Appendix C, for the case of GFG, the whole gamete genotypes are <span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-277.gif.backup.1557862300.3955"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-277.gif.backup.1557862300.3955"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-278.gif"/></noscript></span></span>. We choose <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-279.gif.backup.1557862300.4432"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-279.gif.backup.1557862300.4432"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-280.gif.backup.1557862300.4624"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-280.gif.backup.1557862300.4624"/></noscript></span></span> as representatives of the genotype pattern, then each genotype can be classified into one of these patterns. The partial solution determined by the representatives is called the <em>generalized form</em> of the solution mentioned above, whose expression is given in Equation (A3). For the case of GFZ, the notion of generalized form can be similarly defined, whose expression is given in Equation (A4).</p><p id="p-42">For each gamete with <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-281.gif.backup.1557862300.4821"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-281.gif.backup.1557862300.4821"/></noscript></span></span> allele copies, the number of alleles <em>h</em> at the target locus should at least be <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-282.gif.backup.1557862300.499"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-282.gif.backup.1557862300.499"/></noscript></span></span> to allow for all genotypic patterns to be displayed. For example, the full gamete heterozygote <span class="inline-formula" id="inline-formula-272"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-283.gif.backup.1557862300.5187"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-283.gif.backup.1557862300.5187"/></noscript></span></span> under octosomic inheritance can be observed at a tetra-allelic locus. Furthermore, if <span class="inline-formula" id="inline-formula-273"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-284.gif.backup.1557862300.5411"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-284.gif.backup.1557862300.5411"/></noscript></span></span>, the generalized form for all genotypic patterns of GFG can be obtained. Because the allele frequencies at each locus sum to unity, the number of degrees of freedom is equal to <span class="inline-formula" id="inline-formula-274"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-285.gif.backup.1557862300.5615"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-285.gif.backup.1557862300.5615"/></noscript></span></span>. The full gamete heterozygotes can therefore be expressed by <span class="inline-formula" id="inline-formula-275"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-286.gif.backup.1557862300.5876"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-286.gif.backup.1557862300.5876"/></noscript></span></span> allele frequencies.</p><p id="p-43">Using the tetrasomic inheritance under RCS as an example, the frequency of a gamete <span class="inline-formula" id="inline-formula-276"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-287.gif.backup.1557862300.6214"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-287.gif.backup.1557862300.6214"/></noscript></span></span> can be solved at a biallelic locus, and the solution is <span class="inline-formula" id="inline-formula-277"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-288.gif.backup.1557862300.6412"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-288.gif.backup.1557862300.6412"/></noscript></span></span>. Moreover, its solution with the generalized form at a triallelic locus can also be obtained, with the answer being <span class="inline-formula" id="inline-formula-278"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-289.gif.backup.1557862300.6775"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-289.gif.backup.1557862300.6775"/></noscript></span></span>.</p><p id="p-44">The number of equations determined by <a id="xref-disp-formula-10-4" class="xref-disp-formula" href="#disp-formula-10">Equation (8)</a> or <a id="xref-disp-formula-11-4" class="xref-disp-formula" href="#disp-formula-11">Equation (9)</a> increases exponentially with increasing levels of ploidy. For example, for the case of <a id="xref-disp-formula-10-5" class="xref-disp-formula" href="#disp-formula-10">Equation (8)</a>, the number equals 6, 20, 70, 252 and 924 for tetrasomic inheritance to dodecasomic inheritance, while the corresponding sequence is 15, 84, 495, 3003 and 18564 for the case of <a id="xref-disp-formula-11-5" class="xref-disp-formula" href="#disp-formula-11">Equation (9)</a>. The two numerical sequences increase by about 3.5 and 6.0 times for each level, respectively. Moreover, each expression also becomes more complex as the ploidy level increases. For the case of dodecasomic inheritance, it takes multiple gigabytes of space to store these expressions.</p><p id="p-45">Therefore, the system of non-linear equations becomes computationally difficult for higher levels of ploidy. Using this method, we solved the GFG and GFZ for a maximum ploidy level of six on a workstation with two Intel Xeon E5 2696 V3 CPUs, which have 36 cores in total.</p></div><div id="sec-9" class="subsection"><h3>Linear method</h3><p id="p-46">An alternative method to solve the genotypic frequencies at equilibrium is via the linear method. For a working example we focus on a protein-coding gene in a panmictic population at equilibrium with an infinite number of individuals. Suppose that there are three alleles with a unique DNA sequence, denoted by <em>A</em>, <em>B</em> and <em>C</em>, and let <span class="inline-formula" id="inline-formula-279"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-290.gif.backup.1557862300.6976"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-290.gif.backup.1557862300.6976"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-280"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-291.gif.backup.1557862300.7185"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-291.gif.backup.1557862300.7185"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-281"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-292.gif.backup.1557862300.7512"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-292.gif.backup.1557862300.7512"/></noscript></span></span> be their frequencies. Due to the degeneration of codons, two alleles (<em>e.g.</em>, <em>B</em> and <em>C</em>) may code for the same protein. If isozyme electrophoresis is used for genotyping, then these two alleles cannot be distinguished, so both of them will, for example, be typed as <em>B</em>. However, if DNA sequencing is used, both alleles can be distinguished.</p><p id="p-47">Therefore, it can be inferred that the number of alleles of this gene depends on how it is typed, with both biallelic and triallelic states conforming to equilibrium assumptions (<em>i.e.</em>, large population size, random mating, no mutation, no selection and no migration) and genotypic frequencies should concur with the equilibrium.</p><p id="p-48">This scenario can also be expressed as follows: (i) originally, the locus is biallelic and the distribution of genotypes is at equilibrium; (ii) at each step, one novel allele mutates from an existing allele, the mutation rate is constant among different copies of this allele in all genotypes, and the IBD alleles in a genotype mutate simultaneously; (iii) therefore, after the mutation at each step, the genotypic frequency will still concur with the equilibrium; (iv) the mutations are repeated until there are <span class="inline-formula" id="inline-formula-282"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-293.gif.backup.1557862300.7798"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-293.gif.backup.1557862300.7798"/></noscript></span></span> alleles to obtain the generalized form of GFG. The following deductions are therefore derived.</p><div id="sec-10" class="subsection"><h4>Deduction (i):</h4><p id="p-49">For both GFZ and GFG, the frequency of a genotype that contains only the unchanged alleles remains unchanged. In fact, the analytical expression of genotypic frequencies can be regarded as a function of the allele frequencies (refer to Equations (A3) and (A4) in Appendix C). Therefore, when a genotype contains only the unchanged alleles, the corresponding expression of the function is invariable in the process of mutation.</p></div><div id="sec-11" class="subsection"><h4>Deduction (ii):</h4><p id="p-50">For both GFZ and GFG, the summations of the frequencies before and after the mutation for each genotype that has the same composition of unchanged alleles are equal (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-283"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-294.gif.backup.1557862300.8027"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-294.gif.backup.1557862300.8027"/></noscript></span></span> where the last digit in every subscript denotes the number of alleles observed). This fact can be interpreted as several genotypes being merged into one when the locus switches from a triallelic state to a biallelic state.</p></div><div id="sec-12" class="subsection"><h4>Deduction (iii):</h4><p id="p-51">For both GFZ and GFG, the ratio of frequencies of changed alleles in the genotypes that have the same composition of unchanged alleles is equal to the ratio of frequencies of changed alleles in the population. For example, if the changed alleles are <em>B</em> and <em>C</em>, we have <span class="inline-formula" id="inline-formula-284"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-295.gif.backup.1557862300.8299"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-295.gif.backup.1557862300.8299"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-285"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-296.gif.backup.1557862300.8511"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-296.gif.backup.1557862300.8511"/></noscript></span></span>. Because the mutation rate is constant in different copies of <em>B</em> in all genotypes, a fixed proportion <span class="inline-formula" id="inline-formula-286"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-297.gif.backup.1557862300.8764"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-297.gif.backup.1557862300.8764"/></noscript></span></span> of <em>B</em> are mutated to <em>C</em>.</p></div><div id="sec-13" class="subsection"><h4>Deduction (iv):</h4><p id="p-52">For two genotypes with the same pattern, we can use the expression of one genotype frequency to derive that of the other. For example, if the analytical expression <span class="inline-formula" id="inline-formula-287"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-298.gif.backup.1557862300.8976"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-298.gif.backup.1557862300.8976"/></noscript></span></span> of the function is given, by replacing <span class="inline-formula" id="inline-formula-288"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-299.gif.backup.1557862300.9197"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-299.gif.backup.1557862300.9197"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-289"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-300.gif.backup.1557862300.9555"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-300.gif.backup.1557862300.9555"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-290"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-301.gif.backup.1557862300.9855"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-301.gif.backup.1557862300.9855"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-291"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-302.gif.backup.1557862301.0239"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-302.gif.backup.1557862301.0239"/></noscript></span></span>, it follows the expression of <span class="inline-formula" id="inline-formula-292"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-303.gif.backup.1557862301.0578"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-303.gif.backup.1557862301.0578"/></noscript></span></span>, that is <span class="inline-formula" id="inline-formula-293"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-304.gif.backup.1557862301.0894"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-304.gif.backup.1557862301.0894"/></noscript></span></span>.</p><p id="p-53">According to Deduction (iv), if a generalized form for the GFG or GFZ is given, we can write down the whole expressions of the GFG or GFZ.</p><p id="p-54">Using these deductions as stated above, the GFG can be solved step-by-step. At each step, one novel allele is separated from an existing allele. The procedure begins with a biallelic locus, which can be solved by the non-linear method. After the number of alleles reaches <span class="inline-formula" id="inline-formula-294"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-305.gif.backup.1557862301.122"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-305.gif.backup.1557862301.122"/></noscript></span></span>, the generalized form of GFG is obtained, and then the GFZ can be derived by using <a id="xref-disp-formula-11-6" class="xref-disp-formula" href="#disp-formula-11">Equation (9)</a>.</p><p id="p-55">Using hexasomic inheritance as an example, we assume that the GFG at a biallelic locus is solved by the non-linear method, <em>i.e.</em>, the expressions of <span class="inline-formula" id="inline-formula-295"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-306.gif.backup.1557862301.1436"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-306.gif.backup.1557862301.1436"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-296"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-307.gif.backup.1557862301.164"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-307.gif.backup.1557862301.164"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-297"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-308.gif.backup.1557862301.1955"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-308.gif.backup.1557862301.1955"/></noscript></span></span> are derived. If a new allele (<em>e.g.</em>, <em>C</em>) is segregated from <em>B</em>, we obtain <span class="inline-formula" id="inline-formula-298"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-309.gif.backup.1557862301.2305"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-309.gif.backup.1557862301.2305"/></noscript></span></span> via Deduction (i), and <span class="inline-formula" id="inline-formula-299"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-310.gif.backup.1557862301.2701"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-310.gif.backup.1557862301.2701"/></noscript></span></span> via Deduction (iii). Additionally, <span class="inline-formula" id="inline-formula-300"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-311.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-311.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-301"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-312.gif.backup.1557862301.3328"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-312.gif.backup.1557862301.3328"/></noscript></span></span> can be derived from <span class="inline-formula" id="inline-formula-302"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-313.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-313.gif"/></noscript></span></span> via Deduction (iv). Similarly, <span class="inline-formula" id="inline-formula-303"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-314.gif.backup.1557862301.38"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-314.gif.backup.1557862301.38"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-304"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-315.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-315.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-305"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-316.gif.backup.1557862301.4179"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-316.gif.backup.1557862301.4179"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-306"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-317.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-317.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-307"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-318.gif.backup.1557862301.4539"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-318.gif.backup.1557862301.4539"/></noscript></span></span> can be derived from <span class="inline-formula" id="inline-formula-308"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-319.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-319.gif"/></noscript></span></span>. Furthermore, <span class="inline-formula" id="inline-formula-309"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-320.gif.backup.1557862301.4931"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-320.gif.backup.1557862301.4931"/></noscript></span></span> will be obtained from <span class="inline-formula" id="inline-formula-310"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-321.gif.backup.1557862301.5149"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-321.gif.backup.1557862301.5149"/></noscript></span></span> via Deduction (ii). Because the degrees of freedom are currently two, one more step is required. In this step, <span class="inline-formula" id="inline-formula-311"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-322.gif.backup.1557862301.5323"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-322.gif.backup.1557862301.5323"/></noscript></span></span> can be updated by Deduction (iii), and expressed by <span class="inline-formula" id="inline-formula-312"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-323.gif.backup.1557862301.5599"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-323.gif.backup.1557862301.5599"/></noscript></span></span>. Finally, <span class="inline-formula" id="inline-formula-313"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-324.gif.backup.1557862301.5825"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-324.gif.backup.1557862301.5825"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-314"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-325.gif.backup.1557862301.6156"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-325.gif.backup.1557862301.6156"/></noscript></span></span> remain unchanged because of Deduction (i), and the remaining GFG will be obtained via Deduction (iv). The results of GFG and GFZ for hexasomic inheritance are presented in Appendix D.</p><p id="p-56">The linear method can also be characterized by a system <span class="inline-formula" id="inline-formula-315"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-326.gif.backup.1557862301.6552"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-326.gif.backup.1557862301.6552"/></noscript></span></span> of linear equations. Using the first mutation in hexasomic inheritance as an example, the coefficient matrix A is established (see <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> for details, in which <span class="inline-formula" id="inline-formula-316"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-327.gif.backup.1557862301.6787"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-327.gif.backup.1557862301.6787"/></noscript></span></span> and <em>r</em> denote <span class="inline-formula" id="inline-formula-317"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-328.gif.backup.1557862301.7007"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-328.gif.backup.1557862301.7007"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-318"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-329.gif.backup.1557862301.7243"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-329.gif.backup.1557862301.7243"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-319"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-330.gif.backup.1557862301.7515"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-330.gif.backup.1557862301.7515"/></noscript></span></span>, respectively). The elements in b can be derived from the above four deductions. For example, the first seven equations of <span class="inline-formula" id="inline-formula-320"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-331.gif.backup.1557862301.7715"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-331.gif.backup.1557862301.7715"/></noscript></span></span> are<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-366.gif.backup.1557862301.8008"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-366.gif.backup.1557862301.8008"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-367.gif.backup.1557862301.8344"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-367.gif.backup.1557862301.8344"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-368.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-368.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-369.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-369.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-370.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-370.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-371.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-371.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-372.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-372.gif"/></noscript></span></span>Then the first seven elements in b are <span class="inline-formula" id="inline-formula-355"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-373.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-373.gif"/></noscript></span></span>. For these seven equations, one is derived by Deduction (i), where one is number of unchanged genotypes; three are derived by Deductions (ii) and (iii), respectively, where 3 is the number of changed genotypes. Therefore, with the first three deductions, we are able to establish the core equations. Furthermore, by Deduction (iv) we can write down the remaining equations and the total number of equations is <em>h</em> times of the number of core equations.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493018/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493018/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><strong>Matrix</strong> A <strong>of the linear method of hexasomic inheritance at triallelic locus</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-58">The numbers of rows and columns of A are 21 and 10, respectively, which means that there are 21 equations and 10 unknowns (<em>i.e.</em>, the whole GFG after mutation, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-356"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-374.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-374.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-357"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-375.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-375.gif"/></noscript></span></span> and so on.) in this system <span class="inline-formula" id="inline-formula-358"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-376.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-376.gif"/></noscript></span></span>. There are thus many non-independent equations. The rank of A can be derived by symbolic calculation, whose value is 10, equal to the number of unknowns. Therefore, our system of linear equations has a unique solution: <span class="inline-formula" id="inline-formula-359"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-377.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-377.gif"/></noscript></span></span>.</p><p id="p-59">The numbers of rows and columns of A are respectively 39 and 28 for dodecasomic inheritance at a triallelic locus, and the rank of A is 27 by symbolic calculation. Unfortunately, because there are 28 unknowns, our system <span class="inline-formula" id="inline-formula-360"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-378.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-378.gif"/></noscript></span></span> is underdetermined and has an infinite number of solutions.</p><p id="p-60">Using this method, we are able to derive the GFZ and GFG from tetrasomic to decasomic inheritance. For the cases of tetrasomic and hexasomic inheritance, these results are identical to the solutions obtained from the non-linear method. Subsequent expressions become more complex with increasing levels of ploidy, so we only present the results for hexasomic inheritance in Appendix D.</p><p id="p-61">As an alternative, these expressions are placed in a 660 KB C++ source code for a library, and the source code is available in our supplementary material. The phenotypic frequencies can then be calculated by taking the sum of the frequencies of all possible genotypes that yield this phenotype, which are also given in the source code.</p></div></div><div id="sec-14" class="subsection"><h3>Inbreeding coefficient and heterozygosity</h3><p id="p-62">Double-reduction has a similar effect as inbreeding, both not only resulting in IBD alleles in the zygote but also slightly increasing homozygosity (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Hardy 2016</a>). Here, we assess the impact of inbreeding and double-reduction on the inbreeding coefficient. The concept of inbreeding is relative to a reference population, which by definition is without inbreeding or relatedness, with all alleles in the reference population defined as not being IBD (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lynch and Walsh 1998</a>).</p><p id="p-63">The <em>inbreeding coefficient</em> in polysomic inheritance is defined as the probability of sampling two IBD alleles from a genotype without replacement (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2015</a>). In polyploids, gametes also have multiple alleles at a target locus, and the inbreeding coefficient and heterozygosity can also be applied to gametes to measure the degree of IBD or <em>identical-by-state</em> (IBS) allele pairs (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barone <em>et al.</em> 1995</a>).</p><p id="p-64">After fertilization, there are <span class="inline-formula" id="inline-formula-361"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-379.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-379.gif"/></noscript></span></span> pairs of alleles in a zygote, where <span class="inline-formula" id="inline-formula-362"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-380.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-380.gif"/></noscript></span></span> pairs of alleles are from one gamete, and <span class="inline-formula" id="inline-formula-363"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-381.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-381.gif"/></noscript></span></span> pairs of alleles are from different gametes. The numbers of IBD allele pairs in both types of allele pairs are derived as follows.</p><p id="p-65">The number of IBDR allele pairs in the <span class="inline-formula" id="inline-formula-364"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-382.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-382.gif"/></noscript></span></span> allele pairs from the same gamete is the expected value λ of allele pairs, where <span class="inline-formula" id="inline-formula-365"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-383.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-383.gif"/></noscript></span></span>. In the remaining allele pairs, the number of IBD allele pairs inherited from the parent is a proportion <em>F</em> of those remaining, where <em>F</em> is the inbreeding coefficient in zygotes of the previous generation. Therefore, there are <span class="inline-formula" id="inline-formula-366"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-384.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-384.gif"/></noscript></span></span> pairs of IBD alleles in each gamete. Hence the inbreeding coefficient in the gamete is:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-385.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-385.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>For the <span class="inline-formula" id="inline-formula-367"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-386.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-386.gif"/></noscript></span></span> allele pairs from two different fertilizing gametes, the mating individuals in the presence of inbreeding may also share IBD alleles. The co-ancestry coefficient is used to measure the degree of kinship in mating individuals, where the <em>co-ancestry coefficient</em>, denoted by <em>θ</em>, is defined as the probability that two alleles, one randomly sampled from each individual, are IBD (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jacquard 1972</a>). Then there is a value <span class="inline-formula" id="inline-formula-368"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-387.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-387.gif"/></noscript></span></span> expected IBD allele pairs between fertilizing gametes.</p><p id="p-66">From the above derivation, the number of pairs of IBD alleles in the zygote is in total <span class="inline-formula" id="inline-formula-369"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-388.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-388.gif"/></noscript></span></span>. Therefore, the inbreeding coefficient for zygotes of the current generation <span class="inline-formula" id="inline-formula-370"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-389.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-389.gif"/></noscript></span></span> is as follows:<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-390.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-390.gif"/></noscript></span></span>The formula is the transition function of inbreeding coefficients. If the genotypic frequencies reach equilibrium, these inbreeding coefficients will be constant across generations. By replacing <span class="inline-formula" id="inline-formula-371"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-391.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-391.gif"/></noscript></span></span> with <em>F</em> in the transition function, the inbreeding coefficient <em>F</em> at equilibrium can be solved, whose expression is as follows:<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-392.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-392.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>For an outcrossed population, <span class="inline-formula" id="inline-formula-372"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-393.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-393.gif"/></noscript></span></span>. Then, by substituting <a id="xref-disp-formula-21-1" class="xref-disp-formula" href="#disp-formula-21">Equation (11)</a> for <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">Equation (10)</a>, the inbreeding coefficients <em>F</em> and <em>f</em> can be obtained, whose values at equilibrium for different double-reduction models are shown in the rightmost two columns of <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>.</p><p id="p-67">If the relationships between mating individuals can be classified into several types (<em>e.g.</em>, self: selfing, parent-offspring: backcross), then the co-ancestry coefficient <em>θ</em> can be derived by a weighted average of the co-ancestry coefficients of those types of relationships. An example to derive the expressions of these co-ancestry coefficients as well as the inbreeding coefficient <em>F</em> is given in Appendix E.</p><p id="p-68">Following the modification of the definition of inbreeding coefficients, the <em>heterozygosity</em> in polysomic inheritance is defined as the probability of sampling two non-IBS alleles from a genotype without replacement (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Hardy 2016</a>). For the two alleles sampled, if they are IBD, then these alleles cannot be IBS; otherwise they are independent and are non-IBS alleles at a probability of <span class="inline-formula" id="inline-formula-373"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-394.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-394.gif"/></noscript></span></span>. Let <em>H</em> be the heterozygosity of zygotes and <em>h</em> be that of gametes, then:<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-395.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-395.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-396.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-396.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>Where <em>F</em> is the inbreeding coefficient of zygotes and <em>f</em> is that of gametes. These expressions are identical to <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nei’s (1977)</a> estimator of inbreeding coefficients, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-374"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-397.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-397.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-375"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-398.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-398.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-376"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-399.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-399.gif"/></noscript></span></span> are the observed and expected heterozygosities, respectively.</p></div><div id="sec-15" class="subsection"><h3>Simulation</h3><p id="p-69">The numerical solutions of genotypic and phenotypic frequencies of gametes and zygotes calculated by computer simulations will be used to compare with the analytical solutions so as to validate our models.</p><p id="p-70">A population is generated with its initial genotypic frequency drawn from the HWE. Because we do not store the specific genotype for each individual, but only the frequency of each genotype, the population size can be considered as infinite. To display all kinds of genotype patterns, the number of alleles <em>h</em> should be greater than or equal to the ploidy level. For example, the complete heterozygote in tetrasomic inheritance <span class="inline-formula" id="inline-formula-377"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-400.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-400.gif"/></noscript></span></span> cannot be observed at a triallelic locus. For simplicity, suppose that the allele frequencies are uniformly distributed, <em>i.e.</em>, the frequency of any allele is equal to <span class="inline-formula" id="inline-formula-378"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-401.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-401.gif"/></noscript></span></span>, so that the frequencies of genotypes or phenotypes with a same pattern are equal (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-379"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-402.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-402.gif"/></noscript></span></span>).</p><p id="p-71">The double-reduction models are simulated to calculate the gamete frequencies. First, for PRCS, the alleles in the zygotes are duplicated and then randomly segregate into four gametes. Second, for CES, the chromosomes are randomly paired and the alleles are exchanged between pairing chromosomes, and then the chromosomes randomly segregate into two secondary oocytes, finally the alleles within the same chromosomes randomly segregate into two gametes. Third, for PES, the chromosomes are randomly paired and the alleles are exchanged between paired chromosomes at a probability of <span class="inline-formula" id="inline-formula-380"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-403.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-403.gif"/></noscript></span></span>, and the remaining steps are the same as CES.</p><p id="p-72">For a zygote, there are many possible ways to generate gametes. We enumerate each possibility and calculate the weighted average of gamete frequencies according to the probability of each possibility. The frequencies of identical gametes produced from different zygotes are weighed again by the zygote frequencies so as to obtain the gamete frequencies in the population.</p><p id="p-73">The gametes are randomly merged to simulate fertilization, then the zygote frequency in the next generation can be calculated. The population is reproduced for 100 generations to converge the genotypic frequencies. The simulation program can be found in the electronic supplementary materials.</p><p id="p-74">There are many potential genotypes and phenotypes in a population, and it is thus impractical to show all of these frequencies in the manuscript. As an alternative, the phenotypic frequencies of zygote patterns are shown in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, and the genotypic frequencies of gamete patterns are shown in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. As a comparison, the corresponding analytical solutions of model predictions are also shown. It is clear from the data shown in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">5</a> that both of numerical and analytical solutions of phenotypic frequencies of zygotes are approximately equal and their differences are due to the rounding-off error, as are the genotypic frequencies of gametes obtained via different approaches.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493056/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493056/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title"><strong>The numerical and analytical solutions of phenotypic frequencies obtained by numerical simulation and model predication. The number of alleles at this locus is equal to the ploidy level and their frequencies are uniformly distributed.</strong></span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492997/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492997/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title"><strong>The numerical and analytical solutions of gamete frequencies obtained by numerical simulation and model predication. The number of alleles at this locus is equal to the ploidy level and their frequencies are uniformly distributed.</strong></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section other3" id="sec-16"><h2 class="">Data Availability</h2><div id="sec-17" class="subsection"><h3>Material S1</h3><p id="p-75">The appendices.</p></div><div id="sec-18" class="subsection"><h3>Material S2</h3><p id="p-76">The simulation computer program to obtain the numerical solutions of genotypic and phenotypic frequencies of zygotes and the genotypic frequencies of gametes. The program can be run on Windows platforms and requires the .Net Framework V3.5 runtime library.</p></div><div id="sec-19" class="subsection"><h3>Material S3</h3><p id="p-77">The C++ source code of the genotypic and phenotypic frequencies of polysomic inheritance under double-reduction. The prefix of functions is GFZ (genotypic frequencies of zygotes), GFG (genotypic frequencies of gametes), PFZ (phenotypic frequencies of zygotes), or PFG (phenotypic frequencies of gametes). An example that is the function in the second case (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-694"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-717.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-717.gif"/></noscript></span></span>) in Equation (A4) is given by</p><p id="p-78"><kbd>double GFZ4_iiij(double a1, double pi, double pj)</kbd></p><p id="p-79"><kbd>{</kbd></p><p id="p-80"><kbd>double pi2 = pi*pi;</kbd></p><p id="p-81"><kbd>return (8*(-1+a1)*pi2*(-3*a1+2*(-1+a1)*pi)*pj)/pow(2+a1,2);</kbd></p><p id="p-82"><kbd>}</kbd></p><p id="p-83">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7883150">https://doi.org/10.25387/g3.7883150</a>.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><div id="sec-21" class="subsection"><h3>Double-reduction models</h3><p id="p-84">Here, we describe existing double-reductions models and extend them to the systems that account for any even level of ploidy. However, some of these models assume ideal situations that are unrealistic in nature. The CES model assumes that the single chromatid recombination rate <span class="inline-formula" id="inline-formula-695"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-718.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-718.gif"/></noscript></span></span> is one at any locus, but the maximum value of <span class="inline-formula" id="inline-formula-696"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-719.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-719.gif"/></noscript></span></span> is 50% if the locus is located extremely far from the centromere. We incorporate the single chromatid recombination rate into CES, and define this as the partial equational segregation (PES).</p><p id="p-85">Although PES probably reflects more closely to natural patterns of occurrence and may thus more accurately explain real genotypic frequencies, other models (RCS, PRCS and CES) are still useful. Researchers are able to apply them to calculate the genotypic frequencies in their applications without estimating alpha. In contrast, the PES needs an additional parameter <span class="inline-formula" id="inline-formula-697"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-720.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-720.gif"/></noscript></span></span> to calculate the genotypic frequencies, and thus has one more degree of freedom if <span class="inline-formula" id="inline-formula-698"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-721.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-721.gif"/></noscript></span></span> is estimated from the same data. Although PES may perform better in calculating the likelihood from genotypic/phenotypic data, it is also penalized more via the AIC (Akaike information criterion, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akaike 1974</a>) or the BIC (Bayesian information criterion, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schwarz 1978</a>).</p><p id="p-86">In addition, all of these models (RCS, PRCS, CES and PES) assume that the chromosomes must form multivalents during prophase I. However, meiosis of polyploids can be much more complex. Some allopolyploid species display a mode of inheritance intermediate between disomic and polysomic at some loci (segmental allopolyploids, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stift <em>et al.</em> 2008</a>), and some autopolyploid species also form bivalent, univalent and other types of valent during meiosis (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Qu <em>et al.</em> 1998</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lloyd and Bomblies 2016</a>). The formation of different types of valent may influence the sterility of the gametes or seeds (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Swaminathan and Sulbha 1959</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crowley and Rees 1968</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Solís Neffa and Fernández 2000</a>). Such a model can hardly be established if we consider so many factors, <em>e.g.</em>, fertility, proportion and double-reduction ratio for each kind of valent.</p><p id="p-87">Regardless of how complex the nature of meiosis, different models will yield the same result: the IBDR alleles are in the fertile gamete. Therefore, all of these models can be incorporated into a generalized framework, <em>i.e.</em>, using the values of alpha to express a model. Meanwhile, comparing with RCS, PRCS and CES models, the number of degrees of freedom of such a model increases only by <span class="inline-formula" id="inline-formula-699"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-722.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-722.gif"/></noscript></span></span>.</p></div><div id="sec-22" class="subsection"><h3>Genotypic frequencies, inbreeding coefficient and heterozygosity</h3><p id="p-88">Two methods are proposed to solve the genotypic equilibrium under double-reduction at a locus with an arbitrary number of alleles. The first method is the non-linear method, which can theoretically be applied to any even level of ploidy. However, it becomes computationally difficult for organisms with high levels of ploidy because these organisms have too many potential genotypes (<em>e.g.</em>, there are 19494 non-linear equations in dodecasomic inheritance). Although the system of non-linear equations for a higher level of ploidy is computationally difficult and we can currently only solve the genotypic frequencies for hexasomic inheritance, it will be possible with more powerful computers and more advanced algorithms.</p><p id="p-89">The second method is the linear method, which requires more steps but less computational power. This uses the solution obtained by the non-linear method at a biallelic locus as the initial solution, and at each step it is assumed that one novel allele mutates from an existing allele. Some novel genotypes will be separated from existing genotypes, while the genotypic frequencies will still concur with the equilibrium. Using the four deductions, the equations of the genotypic frequencies of gametes are initially established by the total probability formula, and then the equations of the genotypic frequencies of zygotes are established by the multiplication theorem of probability. Unfortunately, the constraints are insufficient to obtain a unique solution for ploidy levels greater than or equal to 12. Although this is not perfect, our method is more than adequate to answer most current research questions on this subject.</p><p id="p-90">Double-reduction results in gametes carrying IBD alleles and slightly increased homozygosity (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Hardy 2016</a>). It has similar effects as inbreeding, and can cause a deviation in genotypic frequencies from the Hardy-Weinberg equilibrium (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Luo <em>et al.</em> 2006</a>). By modifying the definition of the inbreeding coefficient and heterozygosity in polysomic inheritance, the symbolic expressions measuring how double-reduction and inbreeding influence the inbreeding coefficient and heterozygosity are derived. We found that double-reduction can cause a maximum inbreeding coefficient being 0.0769 (in CES) under tetrasomic inheritance in an outcrossed population. For the same double-reduction model, the double-reduction ratio is increased while the inbreeding coefficient is reduced with increasing levels of ploidy (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-91">In the presence of double-reduction and/or inbreeding, the theoretical observed heterozygosity can be calculated from the inbreeding coefficient, and is identical to <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nei’s (1977)</a> estimator of the inbreeding coefficient. This means that the inbreeding coefficient in polysomic inheritance can still be estimated by previous estimators, the same as for other <em>F</em>-statistics, although some modifications should be made to account for the finite sample size (<em>e.g.</em>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Robertson and Hill 1984</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Weir and Cockerham 1984</a>).</p></div><div id="sec-23" class="subsection"><h3>Applications of zygote frequencies</h3><p id="p-92">Genotypic frequencies can be used for many applications, such as for population genetics, molecular ecology, molecular breeding, and so on. Some of the applications of zygote frequencies are listed as follows.</p><ol class="list-romanlower " id="list-2"><li id="list-item-3"><p id="p-93">Estimating the allele frequency from phenotypes: the probability that observing a genotype conditional on a phenotype can be derived from the GFZ and Bayes formula. By counting the number of copies of each allele in each possible genotype and using the conditional probability as a weight, the allele frequencies can be updated from an initial solution with an Expectation Maximization algorithm (<em>e.g.</em>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">De Silva <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kalinowski and Taper 2006</a>).</p></li><li id="list-item-4"><p id="p-94">Based on the allele frequency, many subsequent analyses can be performed. For example, genetic diversity analysis (<em>e.g.</em>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rousset 2008</a>), genetic distance analysis (<em>e.g.</em>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nei 1972</a>), population differentiation analysis (<em>e.g.</em>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cockerham 1973</a>), analysis of molecular variances (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Excoffier and Lischer 2010</a>), principal coordination analysis (<em>e.g.</em>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Peakall and Smouse 2012</a>), and hierarchy clustering (<em>e.g.</em>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Odong <em>et al.</em> 2011</a>).</p></li><li id="list-item-5"><p id="p-95">Population assignment and Bayesian clustering: by calculating the product of multilocus genotypic frequencies, the probability of randomly sampling an individual with certain multilocus genotypes from a population can be obtained (<em>i.e.</em>, likelihood). The individual is assigned to the population with maximum likelihood (<em>e.g.</em>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Peakall and Smouse 2012</a>). In the non-admixture model of Bayesian clustering, the population origin of each individual is randomly drawn from the populations according to the posterior probability that the individual originated from each population, and is updated iteratively in a Markov Chain Monte Carlo (MCMC) algorithm (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Pritchard <em>et al.</em> 2000</a>). The posterior probability is calculated from the multilocus genotypic frequencies of the individual from each population by the Bayes formula (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Pritchard <em>et al.</em> 2000</a>).</p></li><li id="list-item-6"><p id="p-96">Equilibrium test: as a Hardy-Weinberg equilibrium test, this test is able to evaluate whether the distribution of genotypes/phenotypes concurs with the equilibrium state for a particular double-reduction model. The observed number of each genotype/phenotype can be obtained by counts from the data, and the expected number can be obtained by the GFZ or PFZ. A Chi-square goodness-of-fit test or a Fisher’s <em>G</em> test can be used for this application (<em>e.g.</em>, <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Rousset 2008</a>).</p></li><li id="list-item-7"><p id="p-97">Linkage disequilibrium test: the contingency table of a pair of loci is established, with each row denoting a genotype of the first locus with each column denoting that of the second locus. The expected number in each cell under the hypothesis that the two loci are under linkage equilibrium is the product of the GFZ at these two loci (<em>e.g.</em>, <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Rousset 2008</a>).</p></li><li id="list-item-8"><p id="p-98">Estimating individual inbreeding coefficient, relatedness coefficient or kinship coefficient from phenotypes: method-of-moment estimators (<em>e.g.</em>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hardy and Vekemans 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang <em>et al.</em> 2014</a>) or the maximum-likelihood estimator (<em>e.g.</em>, <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2015</a>) can be used to estimate each of these coefficients between possible genotypes, with the estimates being weighted according to the conditional probability in application (i) above.</p></li></ol></div><div id="sec-24" class="subsection"><h3>Applications of gamete frequencies</h3><p id="p-99">There are also some potential applications of the transitional probability from zygotes to gametes and the gamete frequencies, in additional to solve the values of alpha for PRCS (<a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation (3)</a>) and to derive the genotypic frequencies at equilibrium (<a id="xref-disp-formula-10-6" class="xref-disp-formula" href="#disp-formula-10">Equation (8)</a>).</p><ol class="list-romanlower " id="list-3"><li id="list-item-9"><p id="p-100">Parentage analysis: the transitional probability that a parent or a pair of parents produce an offspring is used in the calculation of likelihoods of the following two hypotheses: (a) the alleged father is the true father of the offspring and (b) the alleged father is unrelated to the offspring and is randomly sampled from the population (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Marshall <em>et al.</em> 1998</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kalinowski <em>et al.</em> 2007</a>). This transitional probability is a summation of the products of two gamete frequencies that form the offspring’s genotype, where the frequency is equal to <span class="inline-formula" id="inline-formula-700"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-723.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-723.gif"/></noscript></span></span> if the mother’s genotype is known, or equal to <span class="inline-formula" id="inline-formula-701"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-724.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-724.gif"/></noscript></span></span> if the mother’s genotype is unknown, in which <span class="inline-formula" id="inline-formula-702"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-725.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-725.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-703"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-726.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-726.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-704"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-727.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-727.gif"/></noscript></span></span> denote the genotypes of the offspring, alleged father and true mother, respectively.</p></li><li id="list-item-10"><p id="p-101">Deriving the segregation ratio in mapping populations, or deriving the genotypic frequencies for populations which are not at equilibrium: we use an F2 population in tetrasomic inheritance as an example. Assuming the genotypes of F1 individuals are <span class="inline-formula" id="inline-formula-705"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-728.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-728.gif"/></noscript></span></span>, using <a id="xref-disp-formula-1-10" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>, the gamete ratio is <span class="inline-formula" id="inline-formula-706"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-729.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-729.gif"/></noscript></span></span>. From the multiplication theorem, the segregation ratio of the F2 population is <span class="inline-formula" id="inline-formula-707"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-730.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-730.gif"/></noscript></span></span></p></li><li id="list-item-11"><p id="p-102">Estimating the values of alpha from mapping populations: by equating the observed segregation ratio with the expected value, a system of non-linear equations can be established. The values of alpha can be solved by the least-squares method.</p></li><li id="list-item-12"><p id="p-103">Perform an equilibrium test or a linkage disequilibrium test: the deviation of the observed gamete frequencies from expected can be measured by either a <span class="inline-formula" id="inline-formula-708"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-731.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1693/embed/mml-math-731.gif"/></noscript></span></span> or a <em>G</em> statistic by using either a Chi-square test or a Fisher’s <em>G</em>-test, respectively. With the cumulative distribution function of a Chi-squared distribution, the significance can be calculated.</p></li></ol></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-104">We would like to thank the two anonymous reviewers for their suggestions and comments. This study was funded by the Strategic Priority Research Program of the Chinese Academy of Sciences (XDB31020302), the Natural Science Foundation of China (31770411, 31730104, 31572278 and 31770425), and the Young Elite Scientists Sponsorship Program by CAST (2017QNRC001), the National Key Program of Research and Development, Ministry of Science and Technology (2016YFC0503200), and the Natural Science Basic Research Plan in Shaanxi Province of China (2018JM3024). DWD is supported by a Shaanxi Province Talents 100 Fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-105">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7883150">https://doi.org/10.25387/g3.7883150</a>.</p></li><li class="fn" id="fn-4"><p id="p-106"><em>Communicating editor: Y. Kim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-26"><span class="received-label">Received </span>January 26, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-20"><span class="accepted-label">Accepted </span>March 20, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Huang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.1" data-doi="10.1109/TAC.1974.1100705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akaike</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">A new look at the statistical model identification.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Automat. Contr.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/TAC.1974.1100705</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BAutomat.%2BContr.%26rft.volume%253D19%26rft.spage%253D716%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTAC.1974.1100705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TAC.1974.1100705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.2" data-doi="10.1007/BF00220864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebhardt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frusciante</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Heterozygosity in 2<em>n</em> gametes of potato evaluated by RFLP markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00220864</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D91%26rft.spage%253D98%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RJ83400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Transmission genetics of chromatin from a synthetic amphidiploid to cultivated peanut (<em>Arachis hypogaea</em> L.): broadening the gene pool of a monophyletic polyploid species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBurow%26rft.auinit1%253DM.%2BD.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D823%26rft.epage%253D837%26rft.atitle%253DTransmission%2BGenetics%2Bof%2BChromatin%2BFrom%2Ba%2BSynthetic%2BAmphidiploid%2Bto%2BCultivated%2BPeanut%2B%2528Arachis%2Bhypogaea%2BL.%2529%253A%2BBroadening%2Bthe%2BGene%2BPool%2Bof%2Ba%2BMonophyletic%2BPolyploid%2BSpecies%26rft_id%253Dinfo%253Apmid%252F11606556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvODIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.4" data-doi="10.1073/pnas.100101097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butruille</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boiteux</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Selection-mutation balance in polysomic tetraploids: impact of double reduction and gametophytic selection on the frequency and subchromosomal localization of deleterious mutations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">6608</span>–<span class="cit-lpage">6613</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.100101097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.100101097%26rft_id%253Dinfo%253Apmid%252F10823890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTIvNjYwOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.5" data-doi="10.1111/j.1755-0998.2011.02985.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasieniuk</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">polysat: an R package for polyploid microsatellite analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">566</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2011.02985.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D562%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.02985.x%26rft_id%253Dinfo%253Apmid%252F21481215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.02985.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21481215&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Analyses of gene frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D74%26rft.issue%253D4%26rft.spage%253D679%26rft.epage%253D700%26rft.atitle%253DANALYSES%2BOF%2BGENE%2BFREQUENCIES%26rft_id%253Dinfo%253Apmid%252F17248636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvNC82NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.7" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.8" data-doi="10.1007/BF00336197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crowley</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rees</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Fertility and selection in tetraploid <em>Lolium</em>.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00336197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D24%26rft.spage%253D300%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00336197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00336197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.9" data-doi="10.1007/BF02984208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darlington</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1929</span> <span class="cit-article-title">Chromosome behaviour and structural hybridity in the Tradescantiae.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02984208</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D21%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02984208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02984208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200609700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.10" data-doi="10.1093/molbev/msg173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Recent duplication of the common carp (<em>Cyprinus carpio</em> L.) genome as revealed by analyses of microsatellite loci.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1425</span>–<span class="cit-lpage">1434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msg173</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg173%26rft_id%253Dinfo%253Apmid%252F12832638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12832638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185175500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.11" data-doi="10.1038/sj.hdy.6800728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Silva</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rikkerink</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeilage</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">L. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of allele frequencies in polyploids under certain patterns of inheritance.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DDe%2BSilva%26rft.auinit1%253DH.%2BN.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D327%26rft.epage%253D334%26rft.atitle%253DEstimation%2Bof%2Ballele%2Bfrequencies%2Bin%2Bpolyploids%2Bunder%2Bcertain%2Bpatterns%2Bof%2Binheritance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800728%26rft_id%253Dinfo%253Apmid%252F16094298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16094298&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232184900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.12" data-doi="10.1111/j.1755-0998.2010.02847.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lischer</span>  <span class="cit-name-given-names">H. E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">567</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2010.02847.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DExcoffier%26rft.auinit1%253DL.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D564%26rft.epage%253D567%26rft.atitle%253DArlequin%2Bsuite%2Bver%2B3.5%253A%2Ba%2Bnew%2Bseries%2Bof%2Bprograms%2Bto%2Bperform%2Bpopulation%2Bgenetics%2Banalyses%2Bunder%2BLinux%2Band%2BWindows.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02847.x%26rft_id%253Dinfo%253Apmid%252F21565059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02847.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21565059&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">Allowance for double reduction in the calculation of genotype frequencies with polysomic inheritance.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BHum.%2BGenet.%26rft.volume%253D12%26rft.spage%253D169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.14" data-doi="10.1111/j.1469-1809.1943.tb02307.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">The inheritance of style length in <em>Lythrum salicaria</em>.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1943.tb02307.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02307.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02307.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geiringer</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">Chromatid segregation of tetraploids and hexaploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGeiringer%26rft.auinit1%253DH.%26rft.volume%253D34%26rft.issue%253D6%26rft.spage%253D665%26rft.epage%253D684%26rft.atitle%253DCHROMATID%2BSEGREGATION%2BOF%2BTETRAPLOIDS%2BAND%2BHEXAPLOIDS%26rft_id%253Dinfo%253Apmid%252F15393914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzNC82LzY2NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.16" data-doi="10.1007/BF02984197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-article-title">Theoretical genetics of autopolyploids.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02984197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D22%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02984197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02984197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200610100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.17" data-doi="10.1111/1755-0998.12431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">O. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population genetics of autopolyploids under a mixed mating model and the estimation of selfing rate.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12431</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12431&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.18" data-doi="10.1046/j.1471-8286.2002.00305.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vekemans</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">spagedi: a versatile computer program to analyse spatial genetic structure at the individual or population levels.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1471-8286.2002.00305.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D2%26rft.spage%253D618%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8286.2002.00305.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8286.2002.00305.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000179678400078&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.19" data-doi="10.1038/hdy.2014.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">X. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A maximum-likelihood estimation of pairwise relatedness for autopolyploids.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2014.88</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.88%26rft_id%253Dinfo%253Apmid%252F25370210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25370210&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.20" data-doi="10.1111/1755-0998.12217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shattuck</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A pairwise relatedness estimator for polyploids.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">734</span>–<span class="cit-lpage">744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12217</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D734%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12217%26rft_id%253Dinfo%253Apmid%252F24460904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24460904&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.21" data-doi="10.2307/2528643"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacquard</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic information given by a relative.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2528643</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DJacquard%26rft.auinit1%253DA.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D1101%26rft.epage%253D1114%26rft.atitle%253DGenetic%2Binformation%2Bgiven%2Bby%2Ba%2Brelative.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528643%26rft_id%253Dinfo%253Apmid%252F4648793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2528643&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4648793&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1972O313600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.22" data-doi="10.1007/s10592-006-9134-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalinowski</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taper</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Maximum likelihood estimation of the frequency of null alleles at microsatellite loci.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">995</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10592-006-9134-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D7%26rft.spage%253D991%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10592-006-9134-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10592-006-9134-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000241947900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.23" data-doi="10.1111/j.1365-294X.2007.03089.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalinowski</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taper</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Revising how the computer program cervus accommodates genotyping error increases success in paternity assignment.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2007.03089.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DKalinowski%26rft.auinit1%253DS.%2BT.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D1099%26rft.epage%253D1106%26rft.atitle%253DRevising%2Bhow%2Bthe%2Bcomputer%2Bprogram%2BCERVUS%2Baccommodates%2Bgenotyping%2Berror%2Bincreases%2Bsuccess%2Bin%2Bpaternity%2Bassignment.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03089.x%26rft_id%253Dinfo%253Apmid%252F17305863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03089.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17305863&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244245300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.24" data-doi="10.1111/mec.14201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Screening of duplicated loci reveals hidden divergence patterns in a complex salmonid genome.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4509</span>–<span class="cit-lpage">4522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14201</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D4509%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.25" data-doi="10.1016/j.pbi.2016.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomblies</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Meiosis in autopolyploid and allopolyploid <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2016.02.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D30%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2016.02.004%26rft_id%253Dinfo%253Apmid%252F26950252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2016.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26950252&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.26" data-doi="10.1534/genetics.105.044974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gailing</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Modeling population genetic data in autotetraploid species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.044974</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044974%26rft_id%253Dinfo%253Apmid%252F16172506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNjM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1693.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and analysis of quantitative traits</span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.28" data-doi="10.1038/hdy.2012.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madlung</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Polyploidy and its effect on evolutionary success: old questions revisited with new tools.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2012.79</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.79%26rft_id%253Dinfo%253Apmid%252F23149459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.79&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23149459&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.29" data-doi="10.1046/j.1365-294x.1998.00374.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Statistical confidence for likelihood-based paternity inference in natural populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">655</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-294x.1998.00374.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMarshall%26rft.auinit1%253DT.%2BC.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D639%26rft.epage%253D655%26rft.atitle%253DStatistical%2Bconfidence%2Bfor%2Blikelihood-based%2Bpaternity%2Binference%2Bin%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.1998.00374.x%26rft_id%253Dinfo%253Apmid%252F9633105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.1998.00374.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9633105&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073810200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.30" data-doi="10.1007/BF02982205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1935</span> <span class="cit-article-title">Reductional and equational separation of the chromosomes in bivalents and multivalents.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02982205</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D30%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02982205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02982205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.31" data-doi="10.1111/j.1471-8286.2004.00770.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meirmans</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Tienderen</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">genotype and genodive: two programs for the analysis of genetic diversity of asexual organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">792</span>–<span class="cit-lpage">794</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1471-8286.2004.00770.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D4%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2004.00770.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2004.00770.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225496600081&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.32" data-doi="10.1086/279426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1914</span> <span class="cit-article-title">A new mode of segregation in Gregory’s tetraploid primulas.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/279426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D48%26rft.spage%253D508%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200856200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.33" data-doi="10.1086/282771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic distance between populations.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/282771</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D106%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972M475000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.34" data-doi="10.1111/j.1469-1809.1977.tb01918.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title"><em>F</em>-statistics and analysis of gene diversity in subdivided populations.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1977.tb01918.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D225%26rft.epage%253D233%26rft.atitle%253DF-statistics%2Band%2Banalysis%2Bof%2Bgene%2Bdiversity%2Bin%2Bsubdivided%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1977.tb01918.x%26rft_id%253Dinfo%253Apmid%252F596830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1977.tb01918.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=596830&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977DZ99600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.35" data-doi="10.1007/s00122-011-1576-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Odong</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Hintum</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Determination of genetic structure of germplasm collections: are traditional hierarchical clustering methods appropriate for molecular marker data?</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1576-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DOdong%26rft.auinit1%253DT.%2BL.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D195%26rft.epage%253D205%26rft.atitle%253DDetermination%2Bof%2Bgenetic%2Bstructure%2Bof%2Bgermplasm%2Bcollections%253A%2Bare%2Btraditional%2Bhierarchical%2Bclustering%2Bmethods%2Bappropriate%2Bfor%2Bmolecular%2Bmarker%2Bdata%253F%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1576-x%26rft_id%253Dinfo%253Apmid%252F21472410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1576-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21472410&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.36" data-doi="10.1016/j.cell.2007.10.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The evolutionary consequences of polyploidy.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">462</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2007.10.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP.%26rft.volume%253D131%26rft.issue%253D3%26rft.spage%253D452%26rft.epage%253D462%26rft.atitle%253DThe%2Bevolutionary%2Bconsequences%2Bof%2Bpolyploidy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.10.022%26rft_id%253Dinfo%253Apmid%252F17981114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.10.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17981114&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250618500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.37" data-doi="10.1111/j.1469-8137.2009.03142.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parisod</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holderegger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochmann</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolutionary consequences of autopolyploidy.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2009.03142.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DParisod%26rft.auinit1%253DC.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D17%26rft.atitle%253DEvolutionary%2Bconsequences%2Bof%2Bautopolyploidy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03142.x%26rft_id%253Dinfo%253Apmid%252F20070540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03142.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20070540&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.38" data-doi="10.1093/bioinformatics/bts460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peakall</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smouse</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">genalex 6.5: genetic analysis in Excel. Population genetic software for teaching and researchan update.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2537</span>–<span class="cit-lpage">2539</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts460</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts460%26rft_id%253Dinfo%253Apmid%252F22820204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820204&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309687500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.39" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.40" data-doi="10.2307/2446540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whallon</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Evolution in an autopolyploid group displaying predominantly bivalent pairing at meiosis: genomic similarity of diploid <em>Vaccinium darrowi</em> and autotetraploid <em>V. corymbosum</em> (Ericaceae).</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">698</span>–<span class="cit-lpage">703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2446540</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DQu%26rft.auinit1%253DL.%26rft.volume%253D85%26rft.issue%253D5%26rft.spage%253D698%26rft.epage%253D698%26rft.atitle%253DEvolution%2Bin%2Ban%2Bautopolyploid%2Bgroup%2Bdisplaying%2Bpredominantly%2Bbivalent%2Bpairing%2Bat%2Bmeiosis%253A%2Bgenomic%2Bsimilarity%2Bof%2Bdiploid%2BVaccinium%2Bdarrowi%2Band%2Bautotetraploid%2BV.%2Bcorymbosum%2B%2528Ericaceae%2529%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2446540%26rft_id%253Dinfo%253Apmid%252F21684952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijg1LzUvNjk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1693.41" data-doi="10.1007/978-3-662-01012-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rieger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-source">A glossary of genetics and cytogenetics: Classical and molecular</span>, <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-662-01012-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2Bglossary%2Bof%2Bgenetics%2Band%2Bcytogenetics%253A%2BClassical%2Band%2Bmolecular%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-662-01012-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-662-01012-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Deviations from Hardy-Weinberg proportions: sampling variances and use in estimation of inbreeding coefficients.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRobertson%26rft.auinit1%253DA.%26rft.volume%253D107%26rft.issue%253D4%26rft.spage%253D703%26rft.epage%253D718%26rft.atitle%253DDEVIATIONS%2BFROM%2BHARDY-WEINBERG%2BPROPORTIONS%253A%2BSAMPLING%2BVARIANCES%2BAND%2BUSE%2BIN%2BESTIMATION%2BOF%2BINBREEDING%2BCOEFFICIENTS%26rft_id%253Dinfo%253Apmid%252F6745643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA3LzQvNzAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.43" data-doi="10.1111/j.1471-8286.2007.01931.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rousset</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">genepop007: a complete re-implementation of the genepop software for Windows and Linux.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1471-8286.2007.01931.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DRousset%26rft.auinit1%253DF.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D106%26rft.atitle%253Dgenepop%2527007%253A%2Ba%2Bcomplete%2Bre-implementation%2Bof%2Bthe%2Bgenepop%2Bsoftware%2Bfor%2BWindows%2Band%2BLinux.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01931.x%26rft_id%253Dinfo%253Apmid%252F21585727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01931.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21585727&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253827100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.44" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Estimating the dimension of a model.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176344136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.45" data-doi="10.1038/nature19840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Session</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uno</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome evolution in the allotetraploid frog <em>Xenopus laevis</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature19840</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D336%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19840%26rft_id%253Dinfo%253Apmid%252F27762356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27762356&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.46" data-doi="10.1590/S1415-47572000000400037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solís Neffa</span>  <span class="cit-name-given-names">V. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Chromosome studies in <em>Turnera</em> (Turneraceae).</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">930</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/S1415-47572000000400037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BBiol.%26rft.volume%253D23%26rft.spage%253D925%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS1415-47572000000400037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S1415-47572000000400037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1693.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stebbins</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-source">Variation and Evolution in Plants</span>, <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.48" data-doi="10.1534/genetics.107.085027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stift</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berenos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuperus</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Tienderen</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Segregation models for disomic, tetrasomic and intermediate inheritance in tetraploids: a general procedure applied to rorippa (yellow cress) microsatellite data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">2113</span>–<span class="cit-lpage">2123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.085027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085027%26rft_id%253Dinfo%253Apmid%252F18689891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS80LzIxMTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.49" data-doi="10.1007/BF00888813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swaminathan</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulbha</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Multivalent frequency and seed fertility in raw and evolved tetraploids of <em>Brassica campestris</em> var. <em>toria</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00888813</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D90%26rft.spage%253D385%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00888813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00888813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.50" data-doi="10.1111/j.1558-5646.1984.tb05657.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating <em>F</em>-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1984.tb05657.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.1984.tb05657.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.1984.tb05657.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallo-Meagher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littell</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A general polyploid model for analyzing gene segregation in outcrossing tetraploid species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D869%26rft.epage%253D882%26rft.atitle%253DA%2BGeneral%2BPolyploid%2BModel%2Bfor%2BAnalyzing%2BGene%2BSegregation%2Bin%2BOutcrossing%2BTetraploid%2BSpecies%26rft_id%253Dinfo%253Apmid%252F11606559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvODY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1693.52" data-doi="10.1007/978-0-387-70807-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Principles of Statistical Genomics</span>, <span class="cit-publ-name">Springer New York</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-70807-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPrinciples%2Bof%2BStatistical%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-70807-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-70807-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1693.53" data-doi="10.3390/ijms17081299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Analysis of different ploidy and parent-offspring genomic DNA methylation in the loach <em>Misgurnus anguillicaudatus</em>.</span> <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/ijms17081299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMol.%2BSci.%26rft.volume%253D17%26rft.spage%253D1299%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms17081299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms17081299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1693.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1679" title="HSB-1 Inhibition and HSF-1 Overexpression Trigger Overlapping Transcriptional Changes To Promote Longevity in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1707" title="Determinants of QTL Mapping Power in the Realized Collaborative Cross" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1693" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iDr8SlIhUn6aTkU7CLUuK0qaZYJgeE1IqT9bBImGhsY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1693" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1693" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1693" class="active" data-icon-position="" data-hide-link-title="0">Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492863" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JxNOabcOTHpX3tSwoEzZ50XaDZ4X6c2RE981SMK5Kfs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1693.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1693" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-9NTw-tOsXwMbk1sUHUXcBp2JvEDecJHmRnYDUNECADQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1693" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1693.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492863" id="node492863--2" data-pisa="ggg;9/5/1693" data-pisa-master="ggg;g3.119.400132" data-apath="/ggg/9/5/1693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8357-117X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kang Huang</span>, <span class="highwire-citation-author" data-delta="1">Tongcheng Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5909-1224" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Derek W. Dunn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4433-382X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pei Zhang</span>, <span class="highwire-citation-author" data-delta="4">Xiaoxiao Cao</span>, <span class="highwire-citation-author" data-delta="5">Rucong Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-7430-3889" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Baoguo Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1693-1706;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400132 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492863--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kang Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kang%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKang%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8357-117X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kang Huang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tongcheng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tongcheng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATongcheng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Derek W. Dunn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BW.%2BDunn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunn%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADerek%2BW.%2BDunn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5909-1224" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Derek W. Dunn</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Pei Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pei%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APei%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4433-382X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pei Zhang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xiaoxiao Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoxiao%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoxiao%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rucong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rucong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARucong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Baoguo Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming, China, 650223</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baoguo%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABaoguo%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7430-3889" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Baoguo Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:baoguoli@nwu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">baoguoli@nwu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1693" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1693&amp;title=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1693&amp;title=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1693&amp;title=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1693&amp;text=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1693&amp;title=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1693&amp;t=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1693&amp;title=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1693&amp;title=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492863/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492863" id="node492863--3" data-pisa="ggg;9/5/1693" data-pisa-master="ggg;g3.119.400132" data-apath="/ggg/9/5/1693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8357-117X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kang Huang</span>, <span class="highwire-citation-author" data-delta="1">Tongcheng Wang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5909-1224" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Derek W. Dunn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4433-382X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pei Zhang</span>, <span class="highwire-citation-author" data-delta="4">Xiaoxiao Cao</span>, <span class="highwire-citation-author" data-delta="5">Rucong Liu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-7430-3889" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Baoguo Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1693-1706;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400132 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492863--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kang Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kang%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKang%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8357-117X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kang Huang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tongcheng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tongcheng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATongcheng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Derek W. Dunn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BW.%2BDunn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunn%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADerek%2BW.%2BDunn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5909-1224" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Derek W. Dunn</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Pei Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pei%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APei%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4433-382X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pei Zhang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Xiaoxiao Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoxiao%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoxiao%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rucong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rucong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARucong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Baoguo Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Shaanxi Key Laboratory for Animal Conservation, College of Life Sciences, Northwest University, Xi’an, China, 710069</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming, China, 650223</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baoguo%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABaoguo%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7430-3889" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Baoguo Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:baoguoli@nwu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">baoguoli@nwu.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1693&amp;rft.epage=1706&amp;rft.atitle=Genotypic%20Frequencies%20at%20Equilibrium%20for%20Polysomic%20Inheritance%20Under%20Double-Reduction&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Huang%2C+Kang&amp;rft.au=Wang%2C+Tongcheng&amp;rft.au=Dunn%2C+Derek+W.&amp;rft.au=Zhang%2C+Pei&amp;rft.au=Cao%2C+Xiaoxiao&amp;rft.au=Liu%2C+Rucong&amp;rft.au=Li%2C+Baoguo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492863/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492863/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492863/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492863/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492863/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492863/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492863/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492863/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492863/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492863/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492863/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492863/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1693" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Double-reduction models</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">The genotypic frequencies at equilibrium</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Data Availability</a></li><li class="odd"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1693.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1693.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1693.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
