<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1303" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1303.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1303.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1303.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1303.atom" />
<meta name="HW.pisa" content="ggg;9/5/1303" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population" />
<meta name="DC.Identifier" content="10.1534/g3.119.400039" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Shorter et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Two key features of recombinant inbred panels are well-characterized genomes and reproducibility. Here we report on the sequenced genomes of six additional Collaborative Cross (CC) strains and on inbreeding progress of 72 CC strains. We have previously reported on the sequences of 69 CC strains that were publicly available, bringing the total of CC strains with whole genome sequence up to 75. The sequencing of these six CC strains updates the efforts toward inbreeding undertaken by the UNC Systems Genetics Core. The timing reflects our competing mandates to release to the public as many CC strains as possible while achieving an acceptable level of inbreeding. The new six strains have a higher than average founder contribution from non- domesticus strains than the previously released CC strains. Five of the six strains also have high residual heterozygosity (&gt;14%), which may be related to non- domesticus founder contributions. Finally, we report on updated estimates on residual heterozygosity across the entire CC population using a novel, simple and cost effective genotyping platform on three mice from each strain. We observe a reduction in residual heterozygosity across all previously released CC strains. We discuss the optimal use of different genetic resources available for the CC population." />
<meta name="DC.Contributor" content="John R. Shorter" />
<meta name="DC.Contributor" content="Maya L. Najarian" />
<meta name="DC.Contributor" content="Timothy A. Bell" />
<meta name="DC.Contributor" content="Matthew Blanchard" />
<meta name="DC.Contributor" content="Martin T. Ferris" />
<meta name="DC.Contributor" content="Pablo Hock" />
<meta name="DC.Contributor" content="Anwica Kashfeen" />
<meta name="DC.Contributor" content="Kathryn E. Kirchoff" />
<meta name="DC.Contributor" content="Colton L. Linnertz" />
<meta name="DC.Contributor" content="J. Sebastian Sigmon" />
<meta name="DC.Contributor" content="Darla R. Miller" />
<meta name="DC.Contributor" content="Leonard McMillan" />
<meta name="DC.Contributor" content="Fernando Pardo-Manuel de Villena" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Two key features of recombinant inbred panels are well-characterized genomes and reproducibility. Here we report on the sequenced genomes of six additional Collaborative Cross (CC) strains and on inbreeding progress of 72 CC strains. We have previously reported on the sequences of 69 CC strains that were publicly available, bringing the total of CC strains with whole genome sequence up to 75. The sequencing of these six CC strains updates the efforts toward inbreeding undertaken by the UNC Systems Genetics Core. The timing reflects our competing mandates to release to the public as many CC strains as possible while achieving an acceptable level of inbreeding. The new six strains have a higher than average founder contribution from non-&lt;i&gt;domesticus&lt;/i&gt; strains than the previously released CC strains. Five of the six strains also have high residual heterozygosity (&amp;gt;14%), which may be related to non-&lt;i&gt;domesticus&lt;/i&gt; founder contributions. Finally, we report on updated estimates on residual heterozygosity across the entire CC population using a novel, simple and cost effective genotyping platform on three mice from each strain. We observe a reduction in residual heterozygosity across all previously released CC strains. We discuss the optimal use of different genetic resources available for the CC population.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1303" />
<meta name="citation_id" content="9/5/1303" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1303" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1303.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1303.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1303.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400039" />
<meta name="citation_pmid" content="30858237" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="1303" />
<meta name="citation_lastpage" content="1311" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="John R. Shorter" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4732-5526" />
<meta name="citation_author" content="Maya L. Najarian" />
<meta name="citation_author_institution" content="Department of Computer Science, and" />
<meta name="citation_author" content="Timothy A. Bell" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9546-6334" />
<meta name="citation_author" content="Matthew Blanchard" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author" content="Martin T. Ferris" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1241-6268" />
<meta name="citation_author" content="Pablo Hock" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author" content="Anwica Kashfeen" />
<meta name="citation_author_institution" content="Department of Computer Science, and" />
<meta name="citation_author" content="Kathryn E. Kirchoff" />
<meta name="citation_author_institution" content="Department of Computer Science, and" />
<meta name="citation_author" content="Colton L. Linnertz" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author" content="J. Sebastian Sigmon" />
<meta name="citation_author_institution" content="Department of Computer Science, and" />
<meta name="citation_author" content="Darla R. Miller" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author" content="Leonard McMillan" />
<meta name="citation_author_institution" content="Department of Computer Science, and" />
<meta name="citation_author" content="Fernando Pardo-Manuel de Villena" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_email" content="fernando@med.unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5738-5795" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=H. J. Abu-Toamih Atamni;citation_author=Y. Ziner;citation_author=R. Mott;citation_author=L. Wolf;citation_author=F. A. Iraqi;citation_title=Glucose tolerance female- specific QTL mapped in collaborative cross mice.;citation_pages=20-30;citation_volume=28;citation_year=2017;citation_doi=10.1007/s00335-016-9667-2" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=H. J. Abu Toamih Atamni;citation_author=A. Nashef;citation_author=F. A. Iraqi;citation_title=The Collaborative Cross mouse model for dissecting genetic susceptibility to infectious diseases.;citation_pages=471-487;citation_volume=29;citation_year=2018;citation_doi=10.1007/s00335-018-9768-1" />
<meta name="citation_reference" content="citation_journal_title=J. Immunol.;citation_author=R. Collin;citation_author=L. Balmer;citation_author=G. Morahan;citation_author=S. Lesage;citation_title=Common Heritable Immunological Variations Revealed in Genetically Diverse Inbred Mouse Strains of the Collaborative Cross.;citation_volume=202;citation_year=2019;citation_pmid=30587532;citation_doi=10.4049/jimmunol.1801247" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=E. J. Chesler;citation_title=Out of the bottleneck: the diversity outcross and collaborative cross mouse populations in behavioral genetics research.;citation_pages=3-11;citation_volume=25;citation_year=2014;citation_pmid=24272351;citation_doi=10.1007/s00335-013-9492-9" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=E. J. Chesler;citation_author=D. R. Miller;citation_author=L. R. Branstetter;citation_author=L. D. Galloway;citation_author=B. L. Jackson;citation_title=The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.;citation_pages=382-389;citation_volume=19;citation_year=2008;citation_issue=6;citation_pmid=18716833;citation_doi=10.1007/s00335-008-9135-8" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=G. A. Churchill;citation_author=D. M. Gatti;citation_author=S. C. Munger;citation_author=K. L. Svenson;citation_title=The diversity outbred mouse population.;citation_pages=713-718;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892839;citation_doi=10.1007/s00335-012-9414-2" />
<meta name="citation_reference" content="citation_journal_title=Environ. Health Perspect.;citation_author=J. A. Cichocki;citation_author=S. Furuya;citation_author=A. Venkatratnam;citation_author=T. J. McDonald;citation_author=A. H. Knap;citation_title=Characterization of Variability in Toxicokinetics and Toxicodynamics of Tetrachloroethylene Using the Collaborative Cross Mouse Population.;citation_pages=057006;citation_volume=125;citation_year=2017;citation_doi=10.1289/EHP788" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the collaborative cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. J. Donoghue;citation_author=A. Livraghi-Butrico;citation_author=K. M. McFadden;citation_author=J. M. Thomas;citation_author=G. Chen;citation_title=Identification of trans protein QTL for secreted airway mucins in mice and a causal role for Bpifb1.;citation_volume=207;citation_year=2017;citation_pmid=28851744;citation_doi=10.1534/genetics.117.300211" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=A. Dorman;citation_author=D. Baer;citation_author=I. Tomlinson;citation_author=R. Mott;citation_author=F. A. Iraqi;citation_title=Genetic analysis of intestinal polyp development in Collaborative Cross mice carrying the Apc (Min/+) mutation.;citation_pages=46;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12863-016-0349-6" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. G. Doran;citation_author=K. Wong;citation_author=J. Flint;citation_author=D. J. Adams;citation_author=K. W. Hunter;citation_title=Deep genome sequencing and variation analysis of 13 inbred mouse strains defines candidate phenotypic alleles, private variation and homozygous truncating mutations.;citation_pages=167;citation_volume=17;citation_year=2016;citation_pmid=27480531;citation_doi=10.1186/s13059-016-1024-y" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Durrant;citation_author=H. Tayem;citation_author=B. Yalcin;citation_author=J. Cleak;citation_author=L. Goodstadt;citation_title=Collaborative Cross mice and their power to map host susceptibility to Aspergillus fumigatus infection.;citation_volume=21;citation_year=2011;citation_pmid=21493779;citation_doi=10.1101/gr.118786.110" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=M. T. Ferris;citation_author=D. L. Aylor;citation_author=D. Bottomly;citation_author=A. C. Whitmore;citation_author=L. D. Aicher;citation_title=Modeling host genetic regulation of influenza pathogenesis in the collaborative cross.;citation_pages=e1003196;citation_volume=9;citation_year=2013;citation_pmid=23468633;citation_doi=10.1371/journal.ppat.1003196" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=J. B. Graham;citation_author=J. L. Swarts;citation_author=C. Wilkins;citation_author=S. Thomas;citation_author=R. Green;citation_title=A Mouse Model of Chronic West Nile Virus Disease.;citation_pages=e1005996;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.ppat.1005996" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=J. B. Graham;citation_author=S. Thomas;citation_author=J. Swarts;citation_author=A. A. McMillan;citation_author=M. T. Ferris;citation_title=Genetic diversity in the collaborative cross model recapitulates human West Nile virus disease outcomes.;citation_pages=e00493-e00515;citation_volume=6;citation_year=2015;citation_doi=10.1128/mBio.00493-15" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=J. B. Graham;citation_author=J. L. Swarts;citation_author=M. Mooney;citation_author=G. Choonoo;citation_author=S. Jeng;citation_title=Extensive Homeostatic T Cell Phenotypic Variation within the Collaborative Cross.;citation_pages=2313-2325;citation_volume=21;citation_year=2017;citation_doi=10.1016/j.celrep.2017.10.093" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. E. Gralinski;citation_author=M. T. Ferris;citation_author=D. L. Aylor;citation_author=A. C. Whitmore;citation_author=R. Green;citation_title=Genome wide identification of SARS-CoV susceptibility loci using the collaborative cross.;citation_pages=e1005504;citation_volume=11;citation_year=2015;citation_pmid=26452100;citation_doi=10.1371/journal.pgen.1005504" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=L. E. Gralinski;citation_author=V. D. Menachery;citation_author=A. P. Morgan;citation_author=A. Totura;citation_author=A. Beall;citation_title=Allelic variation in the toll-like receptor adaptor protein ticam2 contributes to SARS-coronavirus pathogenesis in mice.;citation_volume=7;citation_year=2017;citation_pmid=28592648;citation_doi=10.1534/g3.117.041434" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R. Green;citation_author=C. Wilkins;citation_author=S. Thomas;citation_author=A. Sekine;citation_author=D. M. Hendrick;citation_title=Oas1b-dependent immune transcriptional profiles of west nile virus infection in the collaborative cross.;citation_pages=1665-1682;citation_volume=7;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Toxicology;citation_author=J. H. Hartman;citation_author=G. P. Miller;citation_author=A. A. Caro;citation_author=S. D. Byrum;citation_author=L. M. Orr;citation_title=1,3-Butadiene- induced mitochondrial dysfunction is correlated with mitochondrial CYP2E1 activity in Collaborative Cross mice.;citation_pages=114-124;citation_volume=378;citation_year=2017;citation_doi=10.1016/j.tox.2017.01.005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Holt;citation_author=L. McMillan;citation_title=Merging of multi-string BWTs with applications.;citation_volume=30;citation_year=2014;citation_pmid=25172922;citation_doi=10.1093/bioinformatics/btu584" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=F. A. Iraqi;citation_author=G. Churchill;citation_author=R. Mott;citation_title=The Collaborative Cross, developing a resource for mammalian systems genetics: a status report of the Wellcome Trust cohort.;citation_pages=379-381;citation_volume=19;citation_year=2008;citation_issue=6;citation_pmid=18521666;citation_doi=10.1007/s00335-008-9113-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. Kumar;citation_author=K. Kim;citation_author=C. Joseph;citation_author=S. Kourrich;citation_author=S.-H. Yoo;citation_title=C57BL/6N Mutation in Cytoplasmic FMRP interacting protein 2 Regulates Cocaine Response;citation_pages=1508-1512;citation_volume=342;citation_year=2013;citation_issue=6165;citation_pmid=24357318;citation_doi=10.1126/science.1245503" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Levy;citation_author=R. F. Mott;citation_author=F. A. Iraqi;citation_author=Y. Gabet;citation_title=Collaborative cross mice in a genetic association study reveal new candidate genes for bone microarchitecture.;citation_pages=1013;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2213-x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=Fast construction of FM-index for long sequence reads.;citation_volume=30;citation_year=2014;citation_pmid=25107872;citation_doi=10.1093/bioinformatics/btu541" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Lilue;citation_author=A. G. Doran;citation_author=I. T. Fiddes;citation_author=M. Abrudan;citation_author=J. Armstrong;citation_title=Sixteen diverse laboratory mouse reference genomes define strain-specific haplotypes and novel functional loci.;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-018-0223-8" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. H. Mao;citation_author=S. A. Langley;citation_author=Y. Huang;citation_author=M. Hang;citation_author=K. E. Bouchard;citation_title=Identification of genetic factors that modify motor performance and body weight using Collaborative Cross mice.;citation_pages=16247;citation_volume=5;citation_year=2015;citation_doi=10.1038/srep16247" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Rep.;citation_author=R. C. McMullan;citation_author=M. T. Ferris;citation_author=T. A. Bell;citation_author=V. D. Menachery;citation_author=R. S. Baric;citation_title=CC002/Unc females are mouse models of exercise-induced paradoxical fat response.;citation_pages=e13716;citation_volume=6;citation_year=2018;citation_doi=10.14814/phy2.13716" />
<meta name="citation_reference" content="citation_journal_title=Mol. Autism;citation_author=R. T. Molenhuis;citation_author=H. Bruining;citation_author=M. Brandt;citation_author=P. E. van Soldt;citation_author=H. J. Abu-Toamih Atamni;citation_title=Modeling the quantitative nature of neurodevelopmental disorders using Collaborative Cross mice.;citation_pages=63;citation_volume=9;citation_year=2018;citation_doi=10.1186/s13229-018-0252-2" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=G. Morahan;citation_author=L. Balmer;citation_author=D. . Monley;citation_title=Establishment of &quot;The Gene Mine&quot;: a resource for rapid identification of complex trait genes.;citation_pages=390-393;citation_volume=19;citation_year=2008;citation_issue=6;citation_pmid=18716834;citation_doi=10.1007/s00335-008-9134-9" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. P. Morgan;citation_author=C.-P. Fu;citation_author=C.-Y. Kao;citation_author=C. E. Welsh;citation_author=J. P. Didion;citation_title=The mouse universal genotyping array: from substrains to subspecies.;citation_volume=6;citation_year=2015;citation_pmid=26684931;citation_doi=10.1534/g3.115.022087" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Sci.;citation_author=M. Mosedale;citation_author=Y. Kim;citation_author=W. J. Brock;citation_author=S. E. Roth;citation_author=T. Wiltshire;citation_title=Editor’s Highlight: Candidate Risk Factors and Mechanisms for Tolvaptan-Induced Liver Injury Are Identified Using a Collaborative Cross Approach.;citation_pages=438-454;citation_volume=156;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=A. Nachshon;citation_author=H. J. Abu-Toamih Atamni;citation_author=Y. Steuerman;citation_author=R. Sheikh-Hamed;citation_author=A. Dorman;citation_title=Dissecting the Effect of Genetic Variation on the Hepatic Expression of Drug Disposition Genes across the Collaborative Cross Mouse Strains.;citation_pages=172;citation_volume=7;citation_year=2016;citation_doi=10.3389/fgene.2016.00172" />
<meta name="citation_reference" content="citation_journal_title=J. Periodontol.;citation_author=A. Nashef;citation_author=H. J. Abu‐Toamih Atamni;citation_author=Y. Buchnik;citation_author=H. Hasturk;citation_author=A. Kantarci;citation_title=Collaborative Cross Mouse Population for Studying Alveolar Bone Changes and Impaired Glucose Tolerance Comorbidity After High‐Fat Diet Consumption.;citation_pages=e150-e158;citation_volume=88;citation_year=2017;citation_doi=10.1902/jop.2017.170075" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. G. Oreper;citation_author=Y. Cai;citation_author=L. M. Tarantino;citation_author=F. Pardo-Manuel de Villena;citation_author=W. Valdar;citation_title=Inbred Strain Variant Database (ISVDB): a repository for probabilistically informed sequence differences among the collaborative cross strains and their founders.;citation_volume=7;citation_year=2017;citation_pmid=28592645;citation_doi=10.1534/g3.117.041491" />
<meta name="citation_reference" content="citation_author=K. Orgel;citation_author=J. M. Smeekens;citation_author=P. Ye;citation_author=L. Fotsch;citation_author=R. Guo;citation_pages=1027;citation_volume=143;citation_year=2018;citation_doi=10.1016/j.jaci.2018.10.009" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. L. Rasmussen;citation_author=A. Okumura;citation_author=M. T. Ferris;citation_author=R. Green;citation_author=F. Feldmann;citation_title=Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance.;citation_volume=346;citation_year=2014;citation_pmid=25359852;citation_doi=10.1126/science.1259595" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Protocols;citation_journal_abbrev=Cold Spring Harb Protoc;citation_author=K. M. Reilly;citation_title=Using the Collaborative Cross to Study the Role of Genetic Diversity in Cancer-Related Phenotypes;citation_pages=pdb.prot079178-pdb.prot079178;citation_volume=2016;citation_year=2016;citation_issue=3;citation_pmid=26933242;citation_doi=10.1101/pdb.prot079178" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=A. R. Rogala;citation_author=A. P. Morgan;citation_author=A. M. Christensen;citation_author=T. J. Gooch;citation_author=T. A. Bell;citation_title=The collaborative cross as a resource for modeling human disease: CC011/Unc, a new mouse model for spontaneous colitis.;citation_pages=95-108;citation_volume=25;citation_year=2014;citation_pmid=24487921;citation_doi=10.1007/s00335-013-9499-2" />
<meta name="citation_reference" content="citation_journal_title=Genes Brain Behav.;citation_author=S. Schoenrock;citation_author=D. Oreper;citation_author=J. Farrington;citation_author=R. McMullan;citation_author=R. Ervin;citation_title=Perinatal nutrition interacts with genetic background to alter behavior in a parent-of-origin-dependent manner in adult Collaborative Cross mice.;citation_pages=e12438;citation_volume=17;citation_year=2018;citation_doi=10.1111/gbb.12438" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. R. Shorter;citation_author=F. Odet;citation_author=D. L. Aylor;citation_author=W. Pan;citation_author=C.-Y. Kao;citation_title=Male infertility is responsible for nearly half of the extinction observed in the mouse Collaborative Cross.;citation_volume=206;citation_year=2017;citation_pmid=28592496;citation_doi=10.1534/genetics.116.199596" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Srivastava;citation_author=A. P. Morgan;citation_author=M. L. Najarian;citation_author=V. K. Sarsani;citation_author=J. S. Sigmon;citation_title=Genomes of the mouse Collaborative Cross.;citation_volume=206;citation_year=2017;citation_pmid=28592495;citation_doi=10.1534/genetics.116.198838" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-resolution genetic mapping using the Mouse Diversity outbred population.;citation_volume=190;citation_year=2012;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_title=BioRxiv;citation_title=Low rate of index hopping on the Illumina HiSeq X platform.;citation_author=T. van der Valk;citation_author=F. Vezzi;citation_author=M. Ormestad;citation_author=L. Dalén;citation_author=K. Guschanski;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Sci.;citation_author=A. Venkatratnam;citation_author=S. Furuya;citation_author=O. Kosyk;citation_author=A. Gold;citation_author=W. Bodnar;citation_title=Editor’s Highlight: Collaborative Cross Mouse Population Enables Refinements to Characterization of the Variability in Toxicokinetics of Trichloroethylene and Provides Genetic Evidence for the Role of PPAR Pathway in Its Oxidative Metabolism.;citation_volume=158;citation_year=2017;citation_doi=10.1093/toxsci/kfx065" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=C. E. Welsh;citation_author=D. R. Miller;citation_author=K. F. Manly;citation_author=J. Wang;citation_author=L. McMillan;citation_title=Status and access to the Collaborative Cross population.;citation_pages=706-712;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22847377;citation_doi=10.1007/s00335-012-9410-6" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Zhang;citation_author=D. Malo;citation_author=R. Mott;citation_author=J. J. Panthier;citation_author=X. Montagutelli;citation_title=Identification of new loci involved in the host susceptibility to Salmonella typhimurium in collaborative cross mice.;citation_pages=303;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4667-0" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Two key features of recombinant inbred panels are well-characterized genomes and reproducibility. Here we report on the sequenced genomes of six additional Collaborative Cross (CC) strains and on inbreeding progress of 72 CC strains. We have previously reported on the sequences of 69 CC strains that were publicly available, bringing the total of CC strains with whole genome sequence up to 75. The sequencing of these six CC strains updates the efforts toward inbreeding undertaken by the UNC Systems Genetics Core. The timing reflects our competing mandates to release to the public as many CC strains as possible while achieving an acceptable level of inbreeding. The new six strains have a higher than average founder contribution from non- domesticus strains than the previously released CC strains. Five of the six strains also have high residual heterozygosity (&gt;14%), which may be related to non- domesticus founder contributions. Finally, we report on updated estimates on residual heterozygosity across the entire CC population using a novel, simple and cost effective genotyping platform on three mice from each strain. We observe a reduction in residual heterozygosity across all previously released CC strains. We discuss the optimal use of different genetic resources available for the CC population." />
<meta name="og-title" property="og:title" content="Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1303" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Two key features of recombinant inbred panels are well-characterized genomes and reproducibility. Here we report on the sequenced genomes of six additional Collaborative Cross (CC) strains and on inbreeding progress of 72 CC strains. We have previously reported on the sequences of 69 CC strains that were publicly available, bringing the total of CC strains with whole genome sequence up to 75. The sequencing of these six CC strains updates the efforts toward inbreeding undertaken by the UNC Systems Genetics Core. The timing reflects our competing mandates to release to the public as many CC strains as possible while achieving an acceptable level of inbreeding. The new six strains have a higher than average founder contribution from non- domesticus strains than the previously released CC strains. Five of the six strains also have high residual heterozygosity (&gt;14%), which may be related to non- domesticus founder contributions. Finally, we report on updated estimates on residual heterozygosity across the entire CC population using a novel, simple and cost effective genotyping platform on three mice from each strain. We observe a reduction in residual heterozygosity across all previously released CC strains. We discuss the optimal use of different genetic resources available for the CC population." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"cveHVF3l4PE5cSO-8SYWvzwheaYNZaWMpFhNKtR00jo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492861","apath":"\/ggg\/9\/5\/1303.atom","pisa":"ggg;9\/5\/1303","ac":{"\/ggg\/9\/5\/1303.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1303.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1303"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1303"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1303":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uR1q7qYjTzjldYSnV5Chhj1x12Nxf0BtcypBIqmepNZwh1M2o3gmKUFwGLz72bn1LfPqhLbK6r6Sbv61XeBL8N+OQ7y7fIAhOwQmLXz8\\\\\\\/0FelfimBqAweQqVD2dRcIPnK3PAG6d+bQyla4z56hHi9pqf0yZQy\\\\\\\/NR+94NX6014G7ZJZ\\\\\\\/vpfUVaHx9xFawhUU5Hs0bm9cYR1uuOQuBV59H0I1PlvQVQlnA3epR9pEKHsP1Th9aQITvqff3Nq9gLUHq3WW54I1sSr+sfbpy3WrmhPJqf44jiIcEQDO9xQ2rLpTinvTRkCtbtwENoJY7NBaI1YkTggt9RcwCUbGgWxKyV2+5+7dR8lbmBv6jG05aDW2vsyTksla1hiHOdp\\\\\\\/53Y3cF9q95eDcHnhfxF5ZyT4GW0XU5e6grGo5fBtaZj6n+fLqB3TelvihWcZAYP0hXZnbTqzhIQ3nrkjFgOcxsa9TUa0OUsyNAhz4iaNcw4zH379XQSBSMMR\\\\\\\/ZfsF3Mb5P7BKY5VSZeba60SME28lw6mv6+E4DOZYAbJWx+42bjs0VBSF\\\\\\\/N5ZEq5ByA8b7hlwlQyTKoFxHoNCQX4DUNeA9DrbAogTbyvOYxmClefHSiLEjhh0OCHOaB1Isf9Zdc9t491sTpPfrJBXXsG3gExEvzJZrcd9F1r5EP\\\\\\\/BHc8bhWPLSlSNlXUjXDFs\\\\\\\/Mxnfzh+cvzzaWL5kyzJBqlb3UHmTtOe7PdKkhpQdLVK\\\\\\\/vlWQnxxJjHxIcfOuOOk7A770bRQFYSNdhZTj1aWF0AG4qgCEYSYPoLLRXhDvlCx5ks0JOgXKBk4wLYlmoBFbLZnC1XCYufulrMui4q4wZGThv1VQgoIEjZ4fi1gOq2U9sGjjhZOfkWJ2lnMXsV+JoCR53P8R70eSh0wyil\\\\\\\/VOHCKHmgW8HEB1cft0HFGOAiQkmtU9nHIynZ1IGHS0bo7ovaUj6FH4SG4UpEXb2uuUU85Y49L64V0hAudDTwxzY29NtN7\\\\\\\/cO4F7rmwc3JdEN4g7gBwR02z3NveihBi3cUuwsDVD9LPcrFDJB8YjRzFe\\\\\\\/3rzfm\\\\\\\/5gFpFGSQ6ZFdC+fRrpHAubBKbUq9H1ATXElmc5a3e8mU4rnAjPWmt8SdETPPcJiNTaA+B6MnpQty9w\\\\\\\/DrnfzIdATgoaMjLe266\\\\\\\/V7YYFavTYzt9BcD7eZ71JA\\\\\\\/cSbTPWgZru5N9qrvH1I4PGpZ5fY2i2j8RQXPeQ4oJtUp+6nqHMjrggLXpyM3PP05efTh\\\\\\\/j5sUj9gt9TuWfkLWR+cWPViYdx\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022H5ijJEw0i3Jh05rCXD8+aA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e6377bc6ddf60b8cfb79a06897c6a81cd7fe93386801da60fc4fae0d03562152\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022StzCzSJ7omkQvFK0WOe4hwIWBSn\\\\\\\/M3GtS2RjMwPUg0Oe+1GQnvhEgskK0JPAIhnOtesT7eS\\\\\\\/y3fMRMlaONtgLIbkrg1kHoXJnYHNJQXDcgJsnjTLqp4fryP23LHTTjtron9slP5UsqmP39Gwjldp978nxdgyvOxS1uQKllC5D0QKGK8YMBug7d3pOXP5NmqpL9UZr1wxiIRoatpyxXrN01qEWwJVRPklow56CBaoGrjVGwqQGWhfhLfeCk\\\\\\\/\\\\\\\/8y\\\\\\\/62MdHkd7A+K6gK6Ds5VbTsc1bmDIzd4y+KBq9MC9IPyKsj9RvG8ybw9BID3lcOGD4xSRqN9hx0QcCANzEI9NK8SVlUxL+3g\\\\\\\/FAtzPQVcCb\\\\\\\/h0TFsLARgP6v4RVrTHR1Ku8s1pOIZ4JWjrZBa8brqXuhXrO843FGCpZf0veZYWVgWj1MOVgipu7D4i9ilMAsp8DXREsYHRbJNLVKwoRTSniTIJ\\\\\\\/hMshXspru3j6Jz85D64qWUbWFCUPN4bTpPC5MEXWmC2FZuEJwtlFCA0EiD\\\\\\\/ZQw67\\\\\\\/NwWQtPzLVdNjZhnN5gTr5JyprAragkV0qhwBzaFqzMbfiINlWqLXvC1tbXrXJ4JppqvrIoIcwc5mYS+AudpARfW5DtVx8OBpHBHhyveIhi\\\\\\\/ug35RaRipRjeZtlJaB+yFs6HIJrtFxN0SE9MG6vGCBvhRbkW+NoTyz6RCPkAfqbSTgFY7LZspU6uNCLOubK\\\\\\\/SK7Ytpftf1\\\\\\\/SRrj90eEYOxaLgr57sAvAJ4m\\\\\\\/Qatra\\\\\\\/lkPHFY7kw+cV1+vb1QMeMFsM7cLJEvHWTeoF5D+OtVsPGI0bo\\\\\\\/y7ezeD+e3rUEN4sbk++6pEJHE6ObXqEB9a73eOjQCuQMzIdm\\\\\\\/aAD6hiD5zTZj9iBA\\\\\\\/+Q2jCOKcONxODNuXJDVCA7akiOwrlTBsoncRumspRSDNwBn3FxbmxHgXjc0mt3RL\\\\\\\/EVKh+zkERcTm6Cr9DKVWYmgjFT8L8nU5tRHWmRrGWEAHZ8dOuzjf8rMnU19Dsxp6mGpt2OJryQriHBlVcv8uZtjr43HwKwStamL\\\\\\\/8KovQjeWoSnBTh2eC\\\\\\\/RZjL0dDanwCdLIJxoIsAg3Uh5tmGPz7gKKfvugwbsQkj0ZQPN2cGb4c+y82jZ541EVfsLJEK2ONTgjGVlYVGhqMoLMKRKJRK3ZT8FtGnYuFjAZ2ssPGD1E06EIVRFEDgJJIofj1Xi6l5Z8jn3ZhY653d\\\\\\\/LKiI19ZJd8ETn8kN4yMvBRF7ajlk0IbCJEpvO2I35DnikXJjVnTqFJBwK84KBV6Fpj3E\\\\\\\/vTTn4SRiTslc4Q5HLUbdiZ1nGpkhgadWe9r59BQ8HoQEyQW\\\\\\\/5kt9tObBmwHikTc9P49jYMlQL9PxtNShZEwiOfgfxj8\\\\\\\/4uqL0GKtA4zfvLHsFTm4fBp+5oruH2VBtqwBgwpnLKj\\\\\\\/vPWjulNixHHPFliugkkIFhArUhJ2dEyihzwrjBlJaiIUEmNxZNNX6UcxDRXh30mA1pH69OWuEmpiYDYeRBbuJR9HTYfhJrj4z+HNR4W6gAQhRsELy5+099Pf5GIH3iTUCkguGZO5BrcH1q5Lmn2+0AWUEIypq33ZzpCoHVDQAooH7r34vxNTWj2f1gQrzgJfoPX4Iho8y9xJgj9K1nI4JSJnK00aizyrbvnGsghFJ9wARyr4Itt1jeAh999+hV4qHuygGSR0gD5NHO5kbsxX91T5K198kv6prKYmcvZkjfqX+iyg7R1Af\\\\\\\/l+66dA3ZGvg6Wj2InS3Hr\\\\\\\/m0EYP1ZDzoL8j1bW8zvcHbbLw\\\\\\\/iXzx1L3YCli2cJpJznWl8M2DxxmAkpJ36LnqHTEJzyJ\\\\\\\/S4DIkXGj9nY\\\\\\\/uENuSW81GBtmgiu67sobnzTH+XnrufIpi9y5qoJltHSThZM8MngQw40Kchg2g8+JvPUm0AkUcMY6PvD0fSjBgf5yeX\\\\\\\/16mFSVNbFzYVI3FwKu4fF9iGzrHcQow\\\\\\\/1G6xNDv0EKxcGRk\\\\\\\/wyj2sfREiybWKHKtASCWrV3vbqE7L5eRGUTSYhWTcqQ1YqvOQ6l8FpbMPmhJMEhzYBfWt4uBbopnAG3ay35mW0auYOID2HlJyMOwPeb4qd73bS1NuRNSowGbcNaWUC6RUOdytRmUxIUnFksH8RZYwqPUf8NK0fIb5E\\\\\\\/rQmlC0LYEH00l5+6hdjlI27o7ozVQIyNfxAh9heh9fRf54l8DIvc1B0auwdUYpDb3BY3J0QN+qEvUw\\\\\\\/SS77Yyq1Aar+mdwKf67hV2eTwDVu7v2cgWdglmSiepUm55MwA9yrXQUtP1dLBi3o564xvaQnwMEdeRRnShzOBeItwEbw502WmBxYv\\\\\\\/JfY76ofocmJVlo3gRG4f5kX8DHEIo9ChjpxCWLGMql65OYIlT6FnOCSwQwMNJtqMRuaumrKbaXzibcx6k8A9sJ+g9W2aboSZQHNYNSFbeBo9kvraKnXkkmyYuGqTafYot4aFxJQoUwEiGOH1AK6fY\\\\\\\/7IjjIJphnmocjZlRJ8WKaZ6TMl8C7x8r20wXVgUxKlOYz5fp43WMkEMzHF9b\\\\\\\/3ZrZnYIGdUOeR8GAdr97RGpy5paxN6LileGqk0ua0wf5kP9u7MJqjc97mfoUEYsPi\\\\\\\/76IBa7ZT1Ae0oD+MCztAO4VGwDF2sPDR0LtxCc5kBQ9JoANQgmt7Te1g2VOApS1WlnSyQQOOOgtVq4UdDAndqDEdUQsQ4DdVtZbTw1esmw3qLW0\\\\\\\/JjVz9IClCmhPKkTJhSUONrGr207RXYixYCVCuy+8Py3ejsgZb8LAA5ldAzRq92AeBaScVEaJJmYgU6RccFf9Zbrmo5NM07m5VzJSbX0vm5njlz+d8rFjqrq8GekOtKbF+qCxuBNc+rUVIXewNvbPiXxQ7Q9+Grd1MfzQY7vQf7i4ONsqelWyOxPbCj7PV9r69OGkD0mu4YDWR6YIq9IKRPRlHfSzvlryHL0GvL3iVb3j+yRnN5WyV+oK3uDtds1tnIUF4zZcbP9AbfWPrii07tFYpW9BX30QUoNygfl4z+zuWqX\\\\\\\/JVBKRwQ837sZXpedy+kBhRAA0iMY\\\\\\\/CfbqxmWJb2aSLnfH9PDk2RFoRIhQwT4sgXMyRPF7iL8xKWuLgORPVY\\\\\\\/JVBKsJxjlH21uHlcrPXQC1YcM0nPE65yeM6nDCkMxHJqsB3OldD1Xo3Tc8dieIihUwE1bmPYurcoYYYc\\\\\\\/Yf5KBdzWVq4E2hOLYIrK5YpSq6biJq4KKrBU2aDm4jtmJiC9XNYuNiNHyboeUZbN82NYAnR2FpGpmbKXdFW7Q\\\\\\\/R6VsZgWszBuB3chhYnQqnYmGKf0vTwnkpH1nVmNmp6AQRZhkP7o3xjKVpG2jVd\\\\\\\/6u2rWRIsgbubxIjq4iFgiVI9x86xpvJkT311XEmsRrKUMB9e1axENY043Z9FOo7VFPWY7N+V9Y39UoKW7c4SxPq91Q0sKmKWBPUe3eOGNzFdDbEii7h\\\\\\\/Ynkz5N2Mm4Xl4Db6pLFzABo7iqToaPuQUR3x2FDDyjej6\\\\\\\/h4z6H2KtmRd0ixB7iVMHxEVxJ6shuXd3jfWlFKVA\\\\\\\/3AJlpKTNXRLiGiTSdmsxD5qHGMfD2OBojMdW0y6yZadFfX7kwzwoOhipw48MkCo57NPzAGonoc\\\\\\\/eFxf9+\\\\\\\/LOKjHDxb9VaYvxUVraaJ4oVCaMx\\\\\\\/OuLFhuLu81PXA6ZYy3I2vgBYENjy3rDXvT4YsvR3mufzQtdY6cYCmweqJhTkswLpV8Usip7Oea\\\\\\\/pC7k0pEWEtYEPwLSmU0GaYFABJjQVqWz2XRdVpahoYgE6X+sgIX0I0EFdI+k+BF6dIG798yS3AcmXSUfoZQrV5qN7i+3l7tmbwVHhCYNJnopZPXX0vVG7WYucnWz2SphQBKyapFnyX3m+xF53h8OrtvFHAQTiCbDoVho2dIYpiM8+Jzy+tB6WSj466UgYcPYrtn5+\\\\\\\/3WkSN24uoI55Tmm+6FM2WTa59O+pNXxy2zomYPBVpg7f0xJb3OZ1nA+3aZxQJxSbqoXUvydgnIF3mO2QSO6lUd8eA8i2V8\\\\\\\/0GoNICT\\\\\\\/eCNDVuDclczTJPIt9Z+eoRwVa50Uuhi2ejRAl\\\\\\\/Scgx9dGEFAFvZd5\\\\\\\/sb7e4ZhubLzhUbFGINBRDO7Se0HsN01rvUqowHUMctPLwsonQR\\\\\\\/FKcHen76I3aW65lM9Ldw16kqeXFAOgfA9nBG2\\\\\\\/7kLuKTtiWcIRyQ21TpbOClvlyPuGiuM6hPHvCRxpIy9+dePZgPGKJ470i+9s65Q6XIDRYXnRGlXp32p4RX\\\\\\\/kmB5r7KRKUrg+Uo4kmFMTAdQAID2kvqomXER0c\\\\\\\/qRDRPy9WABHuVhA+ndp+HyVb+2ACN2bzeS\\\\\\\/WUEx9gSIYhlzVISPQjM22oOcH3XstwCYB8kjINKJXJg3z+osGiYQILTH9XWNkgsLv9bIF3rls8zByuMIkhe6zYPBVDuEukalm5Sx\\\\\\\/xKaKY63tDdH3nB6zEm6Pzihi+HYsYST0\\\\\\\/W9S329VSOfy0rBQdkpiQaLxDt0nIAt70IVzH84iUkNNv6imRk1T9K0Prs2RdJrGr5B6OaYl1oXI53LjeazdrZWx6Iw0YAmRsdomufLrM67MtiOlli6YUpr0+0VS46N5i8HKlxSd1pZoRp98EytoD34SjkpLjLSI8pJIbZUZKHzi+Zxsk0yHcqkURZhp0dC3EGTOvbTgn+\\\\\\\/qn6NX\\\\\\\/OzV05nOBFYwqXLzjNerBi+AgvmBWujKPnFYD8o3JnsJrJqK9EH\\\\\\\/awRNyf1H5\\\\\\\/ID9f5\\\\\\\/gJM+b4dHOa9ywzWncdlxqw8V9t9jyqbWE7JtnKiBO0LvnRGdHq+wRpOPlHvy6hXeEUF4RnrDHy2wrlwU1i2SkSJ8JOjfv2zesQGXTr7Z6qfS1nfJiD9yDviNjXw7J7e31IAQkVkP69PUQDo37xVPHkfnGHiiGvs2M48vwaU9ugZWHZ\\\\\\\/VLYaaXrN7CGA85ci4y\\\\\\\/CoDq\\\\\\\/48hNQKDHYRxgzp+kp04AVypXLFNnRUI2ggIV07pxoau1oiiilQmKeGVhvIgRmzd98t1Myk1G8rEylPJBKH5q7PIneUhVDusXUuuQ03w1dhXYBa\\\\\\\/NhLr9OggWHhHYg5r5wsg+z5uxTSTUeAr6GDdFUG+fn1dkagjErjoWhVWzarzhBfUjPOmcGLRF7GsFa8Otobef4d+0nDhtd6ZN4Xqc2qP+Mqk\\\\\\\/ZOSUDylpe3W7Ekpn53ZBhp+rZ5cVDkT5snhgEFRPxMVxduY1wAKYxy5SSYQvweXOPiz8W3RyZdfvhcS6SkNUrgINwEBP5ZH09LCP7nkOYNfLapI65zClz4+USC7SQdXAbOBqn+ABt8NdpYp7wUi8uKnDOzCE67TgB+Q\\\\\\\/9cGxprqrODd98MLux1ffPMejExucdFUu+1wKYeKPT6l2K05Aq2f9J9P6\\\\\\\/uwDyDo9ARpnCp60Eg3kHFUxz7qVD+4SWIf\\\\\\\/70lq+9A02es7wxVXQYoX2nHx2D9SPZNWncBq+Wz4BuVf8lb5i8uA2pYjq4suoo5mf0AJUUOGt3+GfaAT\\\\\\\/M3ckz3rAo\\\\\\\/UPrDch6TkQMS6sxb\\\\\\\/BINWQBCNW2+X0tQn1s5yPq84SYV081EOhp6u1xmXCZ9hyIM5vG2C2hTXgfvBiFxMSSW2r9M+J\\\\\\\/UKoxtUDxh4ToQBTD6IyrUSLCd4vWLwkwooDP9Ms3QXRbec4mbXVRevR6DS4Uw88DecGtlmvr7\\\\\\\/7vKoKMccDpnfbX+CWtZRODrbPQywi\\\\\\\/WR93juk+Vj0bpElV8KEAsdIMfN+tJeHE9IBa\\\\\\\/ecYww\\\\\\\/8WnVfYjE7ze7NOWyZftYN\\\\\\\/j38Chws1DBlouv7a71EuGN3IBCXWixbs\\\\\\\/\\\\\\\/9Zq1ytzZLH+ApaB8QfrgYPm24BUKDx5rbzQvPq4\\\\\\\/SGO3nUhsvbjkZnmgUiJ290j+cmdcl22xlMp3d4fR4yD0vVGcz3H\\\\\\\/4AxKCN0oczC59avJZSwXXS\\\\\\\/LCOWjYzV8jNF2bRh5WlRYzVht8QE34gvia+W1HgezK+0mSbXzl5J9ZZ4Ifi8SEmtoOwp07dOJl+1F\\\\\\\/xyC4vFswVblXT+xi8Xi\\\\\\\/0Rsuj+5Y3GGL50AVVc1fbsXB7E59FP7Y7F5DHLUVW7BPW+InhQoRxfROnnJ6G51YoMPTWF+Xv+R1RzPHoHSFBb9Rrl7EwEx0Fy6Re1DRg3Apk2RB6vLWTlNT3zpFaj\\\\\\\/d7vt2FJRDgmkLbam8e8gVsekBh4arl9xy0BcatT1RmXpu4JUvN+mqwTMfqAjumPoG1qVBdrgStyc0B+ON7vyRurtkpl2qSU7cb9Cj0Cf4uUJHhhvSbRI8jn2cmTHXbgeZFkpyMV7G+jhTUiR\\\\\\\/x9bKQoRCKJcuf+vL4gsLsjiJ0gnMAh1PSw14ZNtSWoAxhTCWMOcdANamQKBrY+FCL+MwSZgF9FczTxq5bulJJLaEIYW8vkvWdim8yw56UC0TfGNDTkYKTHJkv0DPwlUMI0P4YrzNoPT6qxTpW\\\\\\\/ErCj9rHY7MgnTO8cB\\\\\\\/lmFWYg1FsNFY1G78k8RSmrFHRG9FbuDIxK+XKi+DcI+Oy5JadmLLlXqUBAwu9Tg82k44OjkxiVZ0uuWuLAEYcJozwNGmgCK7uwdGXucOuopB5jjRJaIyGq73Q2IqYFc79mRK6QwOFpef+pYI5azvqRi505ngHH3vEJ75+mQN1cN9\\\\\\\/MIRfDBlcgiNxGK9zw6vWT9iX0y25sEMHxFxVLSVvmbuVAyP1OF7HM0uJHextShjcCk4mA\\\\\\\/wtRcdSGOJVi99N04hNMeLO2fn9SLkOG3otHfOT6CgVZbEimekQ03ZEzm2nSGuH7LZr+Opogj13DfA16SRgmKr0waPhnC0eOIfgxTRs3LiSqlEInmgA5bJTN7AIarsRohpKORKpbE31GFE2DtopJogmjHyd75SxE5elx3AlDZCuj+gKDtqWEa\\\\\\\/GQLj8JkkQTkKlDw3jZ9zHOQW\\\\\\\/5yFXhOAbHaNYV1rbJDRkwBEIse6EL\\\\\\\/oXDz+CMWRBrmap6ClgtJi7RQnq9u2TOc5hDypQawsPEomRmFglRr2nONZ1wADcgsURBn5avFy0SlOPidDcdd7Kcm3uh1rKnAmJ\\\\\\\/KIcZzDZAGL6jZjQ15qqPl0SuEzukd8Tu31hYcdlkTbtEQKCo8hl+A3dcR89KmffU7UNGYFBFatO9meIQgbgAJqjWH6DaGCeuk4m+Ty7q+d68pu1AIjgqR14WT3HskRI0sk0T6NAqhD0A4SLrS3pclH2\\\\\\\/2IJEEym3g35wb9N\\\\\\\/zXyGm1Mrwx2wxN0kkyBnJ\\\\\\\/VQ0HLAHTi\\\\\\\/CxN5VBIfn1SwbYanO9rKOPT9G0KP3uwFmPBRsn5fwnHHmo4wu74fyM7lyw0XRXXjK4215o0idwxjnN3H7JABB+VfhnPHDwsimK7pEYtwdY57Xm8zCvPTP4slFHSAZJHMC4CYp\\\\\\\/5seKTWnyH0zaKiioUV3o7pW0VEqQeO3qew1oRVE0Z3YRU6rcMR+3O8xLi4R3VP\\\\\\\/jOZHliZ\\\\\\\/GIBmMkiIcCjtpwBFOASivZbZPbvoTNFMBbQ6qJpfm+0q7o+9OceKhoyvHEs7cckm7Xpnk7UhH\\\\\\\/B2ANKcIzWXaFODk9HDwI7re1f9njNL09erL3Dq3bfqTX5232kTeYSVfL5pVe45KfPrLM\\\\\\\/L\\\\\\\/q0fHag3DNsWsYgBv32gov4Wc0QLN4suu69qMURB3ElLEJncpF8Gw1Vkl+fCy1kvfaPd8u1y66M9Kqj98Lp+rMCLGq9nlqBoT110JX1A+BwrcbMkoWLq6\\\\\\\/7czE3ouY\\\\\\\/oxQiaxtkWnaFj4+IolCrX71TgjmmgEQ16UEFDXgVkmapLWLXUJVFPf+\\\\\\\/dpfr\\\\\\\/ovEdafyRFCAQUoTDOKzP1GsMB1b\\\\\\\/BOvYe58AHmwOc3Tz\\\\\\\/wqzjf\\\\\\\/X\\\\\\\/WQ1vsktMtXSvqFZSIewOHUOtk07o+EF0IR8C0Kl5Q3uOFMi88OjvjG1J\\\\\\\/+kKjJ3T6dm0vO88UeQlkYEKV1oWivLy0Y9C8p3h70XXovh++Pmt0E57wRYqPlvqZH4cMkLWkn+nhgAFcu3QjjKZjgUAsMSWnB1AJOMjQ5rs9HSs1DCG+7mz6JSlrVouShTcTbygWBaBd+wrjRU\\\\\\\/g6a8dGfBqqJuXqrWrTz3AGVQ04fnkEuqHZ5rP\\\\\\\/r2RF8aS7Tc2Thv6Ct5t\\\\\\\/MrP0G3mUCOWFM9fK31EniOgLDO2zjavp1+FCtxr2FbztVrD9XYIj\\\\\\\/DpulSe7Uhr7qrxkcMb5xVwXrlesJlPAy3j79z5NiIb5xQScv+4B2\\\\\\\/0NPYycKpGBovmA6H07WGQ4wXqCb753LICaPfnMQocfyxPtx3H36IRQGuLaOus5\\\\\\\/l2d4r\\\\\\\/dOZ7YkzzW2e1nSEoyUVlI16m0+VoRfjgEPznapeqRzMmHrKvJTSCM\\\\\\\/rMZ\\\\\\\/xxhhLPKEIP5ZQYNNyFjPP+97SSs+p2d9S9beJ1mNNJ14kJtjxzcpxqwDwFvOhTmSv5\\\\\\\/K\\\\\\\/cl5PdEdnVSmnyT4ARFoAF8Ri6JkdzFc99yit981PQzQ9QYyMj\\\\\\\/d4N6G8LbW6OGsg5RCn1WE3iZanWM2RZPNs+cOpX6HHAeeIGs1097SyOlwTkvac51amSa0L8GtYA3pCMZA4RhRPpxYlq+PlTaeH+7AjYU+MUqtoYtgMvGZXp8GUKjKJOLSuL4uPoHLhYWd3+o\\\\\\\/CF2ZIHswoEYrXUEMuGVuvo1gBIUJ9eNbGQM1WdQySxGJQfSXEE7faHdilI3nxqmD6NTwjuB3WJvyVZmRhbHv25U7y4u9JYPM8wU6KT\\\\\\\/f\\\\\\\/1Ohuti\\\\\\\/aWeZRCmJYhg5Rkrkf2aTAwG4INNPc5Kbc3LG9Z1XOjFKm\\\\\\\/vQmXptxY+V4Xd1roXTV2j3ZjwwbFas\\\\\\\/9hhrcnWhZUy6SJHbw9WpxDfgCznFRmiTmGtKRClUTSu+mG6R\\\\\\\/tf84xAXkNqzlKxooKzhTyit+6HxfxaMLaZOxYJlYS8uSRjQESR4eznkVHC6uF5nNd6KeuZA04q3G+NhhmijWIi31GldiIUgG466wvByAstogMBFLcbN5UGr2IqrqTtGYuCylXaAOhewT2Y4RUvhJyBNubqWuGquzuAe0Z13rF+RiuCCo\\\\\\\/2XfImXSEkRNaK9tCIzyU8WC1aGDu30dJ2mfnfqmHsndXEckqrgOV+hC4HcHgI2yN2OI\\\\\\\/XZ5s14\\\\\\\/qwLhl2xyhX\\\\\\\/L6dTGFASxGiwz6sffxRIObbz2deDHTSbAc2k6Mqrn\\\\\\\/eK2N7UmgOPRgez2OIfVUHt\\\\\\\/hU60\\\\\\\/FYVXa3ZumheyA2qfXgBQhO5jnDpjD2DZk\\\\\\\/VQCJ49oryS8iaIUyxwzgLoPBELH1RPne6vtNiLRcGTFTLVoWf7B2z0X8Qlz7uNljCEUUs00\\\\\\\/FLbBYbckM3Lum3kNCK4SumYrihI9ryKe\\\\\\\/MMeImRBipnTF5PcHpMp1xh2hClYc+6DrThMxk4UrT2+kUVfhh9P8zu3CEAJ62cSonauwDLObSxrxj66bWvCjdn0hvjrWDbpWuINCzEZgIuTOsQ9jBOcWCNXUiWGnaVhBXhUu7i\\\\\\\/ta7nylMPDDAJBIjB5EMipx2bkKGbX+Jgd+O\\\\\\\/hxPbqHBJRlzabB1aL9YYlh0urgfBRLVuzINkXXwllCkUvJIJf8fAPWylS\\\\\\\/rudt5tS6YVJbFjlpACj0Hat8VpzAUHFfH8m7unMZ3lmhEDfaFig8wOYIvvLTpMeMQWe0nyWfUpJ3badLh6CHQ4QQV7YVn9F+Ed\\\\\\\/h1D9rkySsrZzs3\\\\\\\/n36oXExO4wshIc34v95UauagoSczWMpXbOfVwwUUR8l8jwp\\\\\\\/HriKc1im6PGuzfwA6sSZ+c3h73DvTtCMp5TdPrX3Cc5nV3yO+GF+c76JbCjulCnXk\\\\\\\/wYHQCxl+c7sd+Kdn8yPg1z8SupXI1q1ERxxtlAfPZ0L0lBrSzuYXoeJ+wlHe6kB3UYydr4DGq1zqbb0JsnU4HNCQbv5iI4oEqe72aGjYDyCXNkfiaA2UhN1CE3aR28v0pz7xmIAfMWd3QbDZfGP4kLormvGLE22tQACEdvjBn1rx0EXqi3rTymqx3G2Ubc1d504gOyrqqrNOztASt18GCeDHzIES3D+0d0bVSX2q\\\\\\\/BnHsQYCwC3mo5XhFg1IzSrAjdrr+S++3HB7ZVE1iESwDqamVLPd\\\\\\\/waKwH\\\\\\\/7Y1+sHxt+3Xpvcgti1yaRAX2R5AyCj0hJDZlfHo01utMWKOZjIz+OC+pZOmvylo5PZIyz8aFxflULIguN\\\\\\\/apL3ZfLsKE3RiNlGwm5qdAvjMd37Yrbyr5tc95mS+QC9xTKgXX7E0Z0rL+vgjdJIak9O1d4RPvPSz1fFQLCLBEUnFpyVVQPTeuP2K1fSxSo3CHscoK9RErUnMjqAwn0vCas7Z4mYSDpkonSwU+5\\\\\\\/v1nWBOxccabl3iRt\\\\\\\/lXbI37YhMxVu7bj9K1oP6k96bCEgA4K6MiIkfu3ehsGJRQcZIVGkwgriPP1WJJvgDMqbRglH7e5LvfcQMfehFTdKtcOyHB0h0HQBdwBUgwufKGkGn2sO7Li023q6Ca0jJdRpMXZr4jwA7WrOTGq6FJOWFBmCFP4lNfW2eb+Ju4uB6okCmRuZDyhSibjVSJrZU4xdf9QCCJoVSaMw1cvkp7hr\\\\\\\/jcmA5461uL+gfIwF16RCRS5g8JFVAGd0Wlkkei11GDae1xI\\\\\\\/13nIOoBeJZygU8dP2upPZHFm1kVDypebIgDZ4\\\\\\\/ZLAGplpR8CHgei\\\\\\\/3VQ8H6t\\\\\\\/mw4wLwgOZgsELfMjlgA0dKLyU6VQZocQqPQVKc25JkBFf1dT53w0cGWuwESGOdZh9EZx1bL9lN0KNbFbpprRfuFMflKa9gHN0ozqSFjkrBYpRf1auJr2+Fnf1TS8d4lhtIVVItT2V\\\\\\\/chYXFbcHtnzLIK2o8ZqcdV6hIY6w725RneloI9a2C4fn+iopEJQY4XUeUDb5qo71Mq0aodDLggNm9JUvTJnkeitvSoW7EPDecCcSr8eMUo2qIGi2VaQvXMjlqJ8kdy27oOAVs7ARyoWKalxx70g67j3CKu421cWWVNqAS6QlbDeD5fyAR\\\\\\\/or7ZSzITJYsP6TxDfS+0IJgrxLzSRFF0ETqRJyO1ubVkiKyobKXbeEi\\\\\\\/25+5q5xsl7N\\\\\\\/djQzXIfGl8tDbxIp0HFVKUZrrjTbwK3Ehyu9Yyq5dcCvq7zYmmPpWopfcOYpPN1k+fQVPPE0XEkBYdZfavuakLtl1HDezlmfMZbd5IQDk80+hBX\\\\\\\/4dWDhUmP\\\\\\\/zZBTT4RI1O2hpMq4+bGfGT2y2\\\\\\\/kYok8Lt53vWzht9r\\\\\\\/42ppKJERhzJfwkEK8D7xuwwhnV9b\\\\\\\/SyknG4AY0ZVjmMJT2KY\\\\\\\/KXG+x14RQxE8w9mdnk5fx\\\\\\\/c04lhNDya0CklS5m3DBHeaBw9dRwVhad6xz4BufJOvX28\\\\\\\/MGsa3TyUcv\\\\\\\/+N3MpRR6o1HteStB9V\\\\\\\/GG9ABIh5dtVFoQVYEgsIiVvtUA91W0hkAvzoK8oohG9AEbkieFhhtXFlz\\\\\\\/rKgoixjb438\\\\\\\/huh7WJaXcTUqw8z5moSvKF5apgpU4rgdzElAAdt2SRu75s5dJFZK9I+dpkfU8B7I4ICoURCp21jJDlLmR6x\\\\\\\/2P0ALheJmORm83ZOItMcSEBSMAT3bd3njWZvWkeeP92ybV+oVAANv7WOv1Y5RA\\\\\\\/RNGSV0TSPJrHfB0h9dZhvnNlMTWbe3q+62uQ0okK6m0s36DihAifVNvS00dCdxozh1pXJnLgVUtdefTtQ7KtCdPTCKOh3CGNYE4aNQvtOji\\\\\\\/ETqjTWbGTbgMjCdlSm4zSMMSdwh8XeYWZSxgyZU169kMEUWr9AnKzUSLWc0Vn+\\\\\\\/lD+B1N6LGJIFqCv+qW10wU6fRcgSfpn7\\\\\\\/UPLf2rHbUGH1CNsd4wFMUsFCmxaEdR2whtgecIUeIjbSkBv1eAQQmxdM8mKvRDWukwqX\\\\\\\/yVxWHMgRhojb7QfLmGyXa1hqYcq4c1R5lmhMBC+oo4ramzRtQRGLsB3keEq7ZKBOcR+SjsFHV9YEe2\\\\\\\/Ef1L+\\\\\\\/Ao1UngVPaXK9JhrDR1+k0T9vSucRRbwd1CZHgaCcNN7YDJ9ZQuLWJX9bYSGre8zXTEwhKZCntOEwvSUhyCNvCPZJLb3tJGhNhfLqcnms\\\\\\\/TFIjLWQIqQ1CEjo1XZSMIfB\\\\\\\/zAMmHwzekqHwBVd5cPfKKnUIrg7XtAGU7fwiWi1GfT9zTtUA1HIl91NFitWjBe1RDwahvI2y4o6RcJ6iwqD\\\\\\\/bUDhJDfvuXNP6jgcgx2V4lffQ+D+960H0VQbJ6EoMF+Q67fw5JwAwy+cakYvEQ0ct8796sMpfOFMKh6K17dfsp0eBssZtdo7oeG\\\\\\\/GYo6RWJZ+u\\\\\\\/viWM0SBfrHKm7cr9aUVo4OXszAOtmmryvbNIhYGDkTA8GUr9cwDC2Xq9GYlMOXRtzZ6tF\\\\\\\/dNckpMx\\\\\\\/ae+ErYYO6PTQV\\\\\\\/Q\\\\\\\/g0ybjgl1RVSG\\\\\\\/EzXn4wWBQiG7oOD0uJMdrF8PN5nDGZ9Uxb1OZOK0MhAhzOZLQGUFXdbEpHnAFNYC9lf+YB6o0i99mXNgrIzPyzs3yiBcO\\\\\\\/KcHO2zMiBj23e+l93QTmT6tJMWN5qqLOm79XM8K045vd7K8GfzdzxvpQ0CvaUIb7M3OFm0L\\\\\\\/t4qoXSG3NviwjE+78wyQ4dOiNytPD2NtsvwfI0VN56OeYR\\\\\\\/hDXvn7ctjnYFREketFDSDbqFEp6nnrdmVISUggX\\\\\\\/5eDqzoUvfF\\\\\\\/YVTDtuZcPxl9VdDoH3421ZqjcdOhUf9FMJIZCzye9dQp+JeliO0aWdXjnO7nzX8C0\\\\\\\/A4GqEcUuLKZwjiycQVbZsdMS6XWw+JyS3dxfzV2QBDWmhBPP3WkCDr1UO3F1lFbzEuEvKCbNGHCf5kBa+hJft\\\\\\\/MjH+5yJaqlpsanebkMUhblfOIUVZaEq0xHDtCg7IcpG\\\\\\\/yZE0piYt6meQzDjsfMwP\\\\\\\/U21rggSgvnRgY1mmYJJJSgsvQGpLIaP+2yHbrIl8+2FUxXU\\\\\\\/zHdeV+L5IsZz99RQtEWaUUwWf42IwOi9\\\\\\\/LeIz6zLNP30x0\\\\\\\/n\\\\\\\/Mzyb1eMzqpAeDztq9Tiw5JkawMazAUltg2uniJQPCYivoF4jTHZgi1OGdi2k7wge\\\\\\\/NoCcCFPdmq6sKsCy8CHWwfomEla6B2QPWhqBNhsxUiKLOkdnkd2p92Qu18zNCoFSJh1WVcFJdiNxeZ7WuxOypNSm9W8DM+wYr1RDmBbrtG8s4qoF2+mFvbb4S5DZO8exdfAzw2rP3\\\\\\\/3Vgz\\\\\\\/zR2nFyOKza9VgN9vnYjljzY4aG85g8smHx2XmvJAC1FT\\\\\\\/auKBz+OCcsCWbiayV8nwXZYWRW1jT9gRdqxHxtjsPywkusizVBFfdCkBGoeZcFPD2ndw4PQPnu2BDBuWG7N4rDYurq5719jbxuUutnB465zhCHv3Dq68CimExM3pnnAq6w5m2wKuD8wgS0wwcbiJIWeF6DgLYZpUM0O\\\\\\\/9N5TuPCafQbRb+1F+tS9lVesM2Fq\\\\\\\/Vno+htlTorXPJrC\\\\\\\/Aykkdx0B5YtqtiZMjbYqQFbeW7P1hnGLfJmAts2qYNPEpZmkY86wDGNIiYQqRXrDAWeD6\\\\\\\/tbjPIirZH+lhGKk\\\\\\\/6UNLLXci4USCLf0iNYm1seyBR5GhRD6Q7MXrJ7\\\\\\\/qOA0PMfUA5iwcd0r42TvIasK2KTltAMYQt\\\\\\\/7+dDmLmHq2Nshouaevd86jjAs6F6RZ9of7Djdd6nK5fn1Gy7SOod1ZNiEEXDxgkriWCE9mFCfZVECoZqELkGl8Qu8Gx1LX6F6BSY0YIuPatmHjDlFFAt0ybXI\\\\\\\/\\\\\\\/8wQ5bo7u2xtLykCYmLIURJdVgL+cRNU\\\\\\\/mkjhcvvhRbE+VZIQ14Ap+YYGsL4uYThowa7FqmFdr4fWLub4VY22LiMmoWGEj43+A7WCG4aedUkrETNLoM0Ucl75oFIFuSpx4eRS\\\\\\\/1Uiw74f1q77V8Y4lpfWAIMyBSq+RKEd20Z+Bi6i7IhKklARlF+lh9QcibR8fikMEcQjyuLSQlkOjMKAx0zSofuBpm\\\\\\\/LuHSXAwOexOMjRh7BjWzzi0hlycA\\\\\\\/JHWXbnu84oebZPqK6P\\\\\\\/LOEeIH1BA+oR+gh8kCu2S4rsKDKze\\\\\\\/dJEaQ\\\\\\\/vhwXYjVXTUU8bNeNCI31OAUvd4\\\\\\\/6loIMxAp4IBbEDXF9cjIpGMt6ZF5qQuKBzJYPw7qKhYqKbuO3e743XNpHHPlArXMK5kCvK2v0zbgdT6jkz6bfTqb7jxrZoMyHaDPi1iFoC0Hkvc8FHYg7Ml1vm2jrcF4KWZ9kkLeAa9jHbP+176WcdLT6nIDeEvAyeHxp3gZOMyGebZWELn8OXfiXDOSYhgT3HNXkJVbH4cIEeR2EgNP8Rihs90s3IxhGroj\\\\\\\/78VQ5tIykICnTR1VJzvkkC66gNTgx\\\\\\\/gU7e9ZSbaix2W14rCuf6U8S55P8wJCHYpA+HJJOQDZJUZNY1DeHBHgk9Mp36brY53631WUjWp8UGYWCAYe3TLIMuxAtOpm4Fiq1Tpr+bH4Ir5eyA6eSXpbQ\\\\\\\/y2qEIB2SHg1E+kbvcQJjrYm2HL9Pt4qgYCVWc1OGDYGShHs2mFF\\\\\\\/WAyQqM5PT7cPXkg6SeXQOQ8sfTVo7vGcGuIc2KPs2kHh8J6Unl7WsMiaoqGnBguKcCjs0FywW12C9QZuCt7diyZfOaHG0XevBlgWrZMY3rMCpTin5AlXGSdBGYvr\\\\\\\/+1J7dIYheXhm0yNCTkQbU\\\\\\\/n72OKYOMBeCQLnVI5u\\\\\\\/AX82bxRPQfYQHEKZNrlnQ7gOqIjzFYxlCQKHWOzmwe1mtHD3qXDqCMp2Az8jgWWugv7osu\\\\\\\/CiZmlPI5TVXA3bMGT8Iv+xxMninE70C+494mw6DYJkjYYEGTeOTstD1lzFGKbMgys3zpn8f2DYF3Q6mlF\\\\\\\/V0j6Ymvkmc\\\\\\\/O0WkhnEzWqAinYP9WhkTwYkh+5V0ipVFIYiMj5ExrfrisQ1a3fyJ1qCfLGEgkVvXWZWHfgK314o2WLxmVBZi+xEvWNCI+WkyLKh7lMeYcn\\\\\\\/UNQI1qLs0jop1MT7AiZv6ua5RVefq77WEJ1vEANc+1MPrlcOpD4EnzrqijsB\\\\\\\/aQHVouH\\\\\\\/8rpk6G2mYlnxvmjZHfwEdCH1KBl15cn+bWnbcXNu\\\\\\\/76nvxnrGx5FIRnWSJ2x\\\\\\\/6uh9XFQj907IANTg8UvEQFae2r9Q87bwE9I0YHKlaqui70Fd8+QrzrCj4D2OwJFuHrzrvDIrQA69Fj5yVpNLZzVViX2BGvuHkIg0NHFGV3DQhoDmHXUxJqUsiOh\\\\\\\/SVdUfcQZiquAfgYZGykKwqFZxpnbdINXWcKRh0AXlOYkUVniQtCW5TvlG6o51Gr093rqyCYaT1ygCA1fDp6cekO7y+yiReOmn2xTINrBdZXppcGRBurID+GdrykHWZP4bbX0gIeLr\\\\\\\/CqVKNQM2ZtbYJmHsTnGiGdZVHzy\\\\\\\/tWnT\\\\\\\/1hBVXbPh1iD5SI2r2Js0G60tFbUvbEak5vyIyEmla8ktiIf4CRXAozkHA9mZ0NpoVjJzIA4OYKzMTnZDb8Uh39ju9\\\\\\\/us4geQwmP3RGlIlPhTq7cJJ\\\\\\\/s19ih014zS3DuuEplR0k4nTNtvoFNnF4YB5g51ur2REU9iS8+TxJuG7wWdBYKMLIDJnjojY\\\\\\\/r9eONTDOWbtXrsYjCzP4ItG9f1hHLc\\\\\\\/APIFLW6LRIphzzKhqBJDAqX7F\\\\\\\/rMw\\\\\\\/QnCOhudCv+CPvGaQrFsCFxpOXTHXDhdHPZg4hy9CUAoW8twiSxa7B+zbqjM7DDti0fdW43cXufqozlQWwHERUYTi2d5tkKAaopd6A30i8ZloSmq9f28QFVMinAwssumMVCvpg+0ZXSaxdPXsNOTux5rgV\\\\\\\/QLwHL9GqbQZQexh+tWYc1FW2bi88rgjhV0KUyYJerG0yPxKMBG+gbaPh6jIpM9SfqTmD18FkhLsCpAvfkK73tY4wBqNNbdB9x\\\\\\\/aq6A9mkisVr5ihJkdTK3ZRt1d3+B0xatJGE8xtsmvi8ps17qS2ntXAVQref+qTlZFpS\\\\\\\/DW9thYsFa5omS77pve0+FJXvnGiPhk7eDRG96XbmiePquc4GcCNn9Mg4xvUsXb3u9NoMuAOnMMGjQC3HCAD8Q6tCiX8k3w4RtdQEo6LKt1piqRPpIsEv3el1SosYFdPNWaTxSqPRgaCy0gtMZMsoupTZjlmAc84Uowz+pG4qu8TtQfT1IQAz7CNdXw6TxemO02UwV\\\\\\\/wS0qStjU+0RgeE1gfLhs4oSlE8f\\\\\\\/krh+rqbAPdAVdxkBIoYA9zyrcJPCZE19iHKHBW\\\\\\\/S4Puklt3DFdTvhroo8wcPauNALykoEqhijVLHi0cAUfFkJfijOYLf6aVrVWHHbbSOkqq2AMOK9Rq+8pmfq6c8O4WkdnNLYif7Y60X6CiiMG4VaR5jaalSa\\\\\\\/LbhdgxK72PQQyAaHczsdfxMeANLSrdFRK8TApyzJP\\\\\\\/gSX5q+26d\\\\\\\/HAVX2nVD1fnkSHEiHlUWKzWEm8SmoGYmP4PxoF5s7mYfCO1hSbtdsXheX0UzRvQ4tS\\\\\\\/FUwRUwPpqWQLsxO7OqrlMgOUrdG+1DF6\\\\\\\/pmLADSLJwyy4dhVeJkmpoDn7DwHD0tpSwgYC9HFk9ET+0EhnT7bfYpTJqTqj9IJ5jymQ8uS6MtFGHLw8yGpRitxeBfzCN7MpkSRlv2mjxgjOcAnu7WZmRpXJ8hcJ5HHneUIUb3BWoXYP\\\\\\\/hA\\\\\\\/0qZNy\\\\\\\/TOye+pSVb8CpwiJcBs\\\\\\\/Cy7edwGTuppwQPeDYr7fcQpQjru7vsWtdSH\\\\\\\/A8lRTFx\\\\\\\/nlfc0AGhbSirvMEA\\\\\\\/ePGzJSm0PjTNrI3QN1rNc7ioI+hMDe2DT\\\\\\\/RM6oALYzHBUz0xuXV2j7dRoUe20qeIDsOSp5SK+MMticXjQN+KJ\\\\\\\/1KU\\\\\\\/1mcXfdH+lkEiVrUbfK3sEv8TB6tFpj+vnDMi4Nxd1ZtEpsy1Hm83q4jv7AnsD2Bt2ZZq5SBl4ST5+3l7k9QWTNAWQfGjeYcCZ+m0loexyMkLnQFYhciC6weGk4amYx82j+qjk+fYSdGDxDC9Hmov8h3PLtXMcYdqC0jLiyXP7Hin9zDQMIgPUsHTG5dkniuNA46rwBHh3rbwoWw0A1ynYcVAMWKtIiWlEsC1b5p+b8Os+wgZgfvNPzG3JdOFi2e00jBcIcsd26FqwhrrFxnFvEle16cRS1axHg\\\\\\\/nO1W49d7l6zmPJ\\\\\\\/q\\\\\\\/P\\\\\\\/LBnxup\\\\\\\/roNz7uiAlFWuDhznDEEdySFRPPDApoNPGlDFVKN4130mSGeCExSt3348YYz10Wv\\\\\\\/mp4NqEKxbf\\\\\\\/NLTmyVCvCnq4yOcRa\\\\\\\/\\\\\\\/3Y32DOfOvIaK8iip4Y3P8j2yMehqf2g33R8O5b2MEi9\\\\\\\/v2rn2\\\\\\\/HM\\\\\\\/E8bAccMdQ8e0TwsRKEHliewo5nDxF2r9GuS+MP4zlRo5qdXcU3zFKwsDKadvFpUqkSSqG63MRUKwQjRKokKJsXNC8fCjv5xo2O\\\\\\\/Yo+DGqS3xEH8zrwYEIm7YVUfOFYqjMiN1OUi59gW\\\\\\\/L5HlRfzZrc7rX+VY+VPiqeK4abpwZML+IAQDz9YHSXRpaYV+CHuKxiEq9W5ZjPQL9WTtwQwE8MXWFXvbzkxYUPKt\\\\\\\/kMofdJ+w2hVe7MeaVvWBKe5C7hOSpuyYA2voxajZv4hdPLhXhyZZVhNZemyU50J6DSwk7NldbwWIy7G\\\\\\\/LYR6PuZU6nQ42wsKYhpB7hVwnZVNtiwpsoU6jMH9ovD622T1mpzYUhmodrjlEBGI1NJoyZMYQHaXIiapih7CUK\\\\\\\/f+bD7U\\\\\\\/sm9OhlrF\\\\\\\/7W0Eq+BvNU5AwcSet4qqRWkdX2vSeEbY26hzqM1qAk7w81q3Fp3rpDeOJwYtnwTR3Yl0kFxHW6PmITmJmSh0QvIwTw2lWMl5R6TW33nzPTQA5sBeXlRhWBzVig5k1dgj1MojgG0N9zYsmXNVSqZw0JZrU71HtQcc+H4z\\\\\\\/iXZeMxKxB70maXxDz+9P3S2dBgpmBQFEd\\\\\\\/OrLksxFc8R47isdIFVg7ogRnlfyZeBVyduFcIQ+H\\\\\\\/jBNufDTAJU+JxKvGkbwZgNdWX\\\\\\\/2xhcXHRv\\\\\\\/yihZiYPDX7q7Alok3loohVpPalBUiwDTs5d2OR0Lr2gTWja14Xg8fhdpBDnINqob3CRLtxqbTjeFhXQ3iiv06INheFbn3d7JEDGyZ50IZ9vQ3tt2CHmcP7Bdr\\\\\\\/5gOUxble3apj+b0rzgwc8FVrCdWsIyRfFJS5wh5vSN7Kb7kpReWcF7SiWo5DUngQy6Em7GNQ1Sc9qS4vVfFQclP1Ke1jeIsbu5UiFVneTENGeLG+h\\\\\\\/T++QcKZleWsZqG3P2rHWUdPDp1mVWP9ygHYACYWFIPXn8XwOI8tODTlKSe8EnenbE51\\\\\\\/8vXvNXMyCN4bBV1\\\\\\\/DM2XM++9yH8LIR7wWsV8Ou5WfBlBl3FoDZvM9aYKJSU15CAaDeakC7+gNT0tHzav7jmU5HQnom8wyRWk9qKzXWSAT452ptSLJVdzu+r7ogVt+SgMBKv0EQlXX+KSKT2JAlBoRE5ABw+kyK0ayabIDHy1X\\\\\\\/6WUTzplFHE\\\\\\\/d5DJfM+8PI6y8BxnOv7cy1A6ebfxL+q3ZSvQBzJvOsoEGfMOHwyfnjNzvpD8GtdJcEjU4PelHHuQY62NtGabPFp0YVWs1S922vwhAQaDn2lSvQzYutNeGdF0X1rzwZNljOnZP4R7IRKy1IqJ9zeSHz4QLKqAIBg+Pm6lFkZFvb5jQXln3WYVZe\\\\\\\/EBzlvyLg0H1TFa4X9I8QOjhUiSxCr0Ulfg\\\\\\\/KAUctUR4Io+NgtA\\\\\\\/8bXsWZIESYk7R0EOALs0Z0\\\\\\\/HFlI5ex\\\\\\\/lbknLnfUXaAlGpfRYf1dwqI2VYm8HLT2Fc8HZ1sLwLQfqLU9hvR4\\\\\\\/J31ibjHqOCb7oCm8b2I0\\\\\\\/cBiiYmxwOEapuy+ID4RXVE2BR9y7+R6Z9xCaOCgNORKIMRFgMSjpMVyo\\\\\\\/jbREc6afzyLBxiWisI1QrhbNteiujiHw9EZy5zDxrMCv0laZ5gnxH7Apj5S7\\\\\\\/zCF1MaS4wATDI19iV5yVSkSssXPQwl8+9JIf4YNuKR3JhzT\\\\\\\/HJWJEUkKJrr8CSc+7vTWfxh87a+u53ZRnA79xunlnAFrVIdC+OcHrEqoPbFxBXE1lw1MSoyBqoAYGMXlWvaZsvQ\\\\\\\/Q2BvNIF2MOAsfwVIOjLRH\\\\\\\/t5b8JhG7jXjQp9GY9M3xoYMahWI36cwTJwpR4LOPVLm\\\\\\\/GscA\\\\\\\/pMWdO5wb05XpzAHKT0D2LFvFS2\\\\\\\/gQ5RyoOQlRLLsBDnaduWkhOgGHCFvLD4Zorjz0Z3hVIZjt49tHsNC8oqMjLBtJexi9k0Ji\\\\\\\/rm9HOmqlpk\\\\\\\/wO73jvcozaQZr8vB0vjBBL5B4Sg53WBhj+mXObCmAsLEIJo5d8jn+ovJTWqAR33A9FS1\\\\\\\/beMeB7eQUloDPNYCi8Nj1I+DzOQO8PwKWXBM29ij6wHvkA1syPeo2WK1\\\\\\\/pRRkkIqENchpmy\\\\\\\/Jh0JwjcbtPnhD1alzKYWbJ\\\\\\\/x8Th6wNd2BIOp+cjqG7kVYwrt4h0v9FMMzgxGqCfn+l\\\\\\\/R6lux9kY6N\\\\\\\/zh213wp3Vo6Z5TQwqydl8soL8ZWLgRz2qwrGc0B5wqSJbIbxBLkARWSPjBiLm0uzkDncL57I5c8J1x+TGD4oaTWn0q+wPZPEZF5AiKlys\\\\\\\/wSRctd3mJ3YBGcPK4fsFF2DT7tEh5da8hhQOo3waV0HKyNIYDji6Uvf+BdMh4LdkpLijzY8\\\\\\\/2lCOGjAVVtxDegCckaFyXXWKEUSL+ksmc9JmDEXIkyhCgRbn6HwmVptfmWjkNsU0iAGxcoKxGWelNZzR+amjIkYvOP+dWKcMyNZNhN8gozp\\\\\\\/HXPX8WYZDceN+14mYkYqGpymymbm0RBdY9B7iv8JIWTqN\\\\\\\/y9jjVppFHcLSFoqxRlA+721mvo9026gOaEDZE9atuvKKGdKbDaAEp4v\\\\\\\/FSXyOTy4cnDmK23nNDsyK45PZjMgxbOzPmkFjDZExrvks8Sop7Q2xgEWMqhSVC1S0sdit748ZbDLAz1vpdO3dZIBjan9MksUAtnGUhCP3kcuOvqqmRWfUpw3oVLRukxGKs+8AQP9a3ZpeZuo2qP+aDY2zMAXHIbSGZb8HNpD9b\\\\\\\/5kbuOOjtt4WjusrWO7bF\\\\\\\/7RlPMXS1owIwbZQYJE+3wOr7Lsxcb+spbeoHyHjvb1Oppc1P+epxUyJIuiO9AIhhr3p4HTXnM1ihkdo8EaFWSD\\\\\\\/JS2LfhmqiMVFBzwv\\\\\\\/eYfXGqM9l30RoGqOMny7j7A\\\\\\\/RK+Bh33t\\\\\\\/JEzLKBRSjQkQCJj3FpM69IVymRyVfPFlGG8LQ8Vy2JZccdo7LShYl67Bod3x3dk6eKL4clpEy++LIM\\\\\\\/lA32gg21glOaBAWEqNwnkNejgpkh4EjaxuWya2Q6AjLnajVUQkkg4HRBKsNegC\\\\\\\/6sGCvzSX2l1ruarhceAL1LLDueTv+MnjOfv\\\\\\\/DElwpyJiQH\\\\\\\/YvIkWqxY0IHp1IBkFook6ohTm1M7TEGf4i8OUDaN10O+wdsaQuLyt0xOiB3iSej5NdUR5cLzKZU9sFYuCt8wzAqQlg7OrF80qpX\\\\\\\/9xRNW\\\\\\\/SzL+o9NTbahb1M8kQpQKLNLsSYFSSPBRh5vnOgUruHTNGlfc\\\\\\\/SfBL7Ai3bOzrtLFTsK6TcCOv7aNaLZ5QAhca3G0Z0ZPWiJxtWK8t50nPH0eKAiMpWw1H+gMUoRnfoGM5Q36z5+5zUy\\\\\\\/624vJw0F0u11mvLaBBMkfLDIPhTRfwND9GFRYIKNv8fEwu\\\\\\\/ODle4TRWZs7dKE4Bzhm9hd7xPu63OzB7Fhe7oXBBhZiNt9HMRh\\\\\\\/66M\\\\\\\/YJOiUS9MZQ21+ujYiWDElOX9RVYc3Veg69ZuEX1Iz\\\\\\\/Cduw7tWRqZ6P5HfNFEl\\\\\\\/8cs3BgqORHspe1tS0KcHBvPJWEbWuuMxKppDuUCkbdB6CA6wz9Ca17dJEeTLbeF0Nhgq1\\\\\\\/QH8UqQ8CYzei4e1qOsqYpqWPxuE7xIthcEGBcIr4sPJXGmzLduqSHbuCJ7uc50YlnRLmy9q+fgtPs0EGJJU64BvKK6phu2IEXT3JU3afFQqaaFwDLiJPDppLsKrB1Fbhhee76Do+VV0diI9xghAprV5dsnjoTskMTlDlW6YPgyBYBzUshkucCChVjfsVgvLBqJUP3XztCNv+HMqQiad+W91\\\\\\\/hq7habUW0dUCX+fgS\\\\\\\/6tKJ2450OQ6WZnNUMW40lYArs7Pbdz1MeHw6yMEn3\\\\\\\/CiJmX5aJmZu54y64RlZOUDTviGnf1oTFMxxBFy9HRih1KoQoAWj8xXXdA1TE1FV48s2Fj6aJFwwlHUdnoJaEhUkhiQI\\\\\\\/rm96QCdYBUTZ1yAGpeo8xjK\\\\\\\/Iq9iugUHWO6Iqo\\\\\\\/159h2ZkE5TXl4AYX9X+9I\\\\\\\/MRCejKs13Em19H7ADJWdXMULR7l2yEYFepsvB5YhMlLtoWWMlJ1ujCIz\\\\\\\/xgQ+og7NFp1efL\\\\\\\/4brTSpNQiuXfCmF4SLs\\\\\\\/3yfG+VYFz\\\\\\\/H+ual7j7M1P1AO54a7k35Ep1voE7SxcMCKXbyuKjWLVklIDxGgx+FTEb7TKP12OnE3fgIKMzLLsAyZT+qDeM3meUVB0fI\\\\\\\/Sk5v\\\\\\\/H00nTtNegmNwpWKeB2aGxEfqfR42xHusIn7Wiv++WQ077EJ3ZaKd\\\\\\\/1NRH0H9qaoBKwtUhlpvLn7zANhAYZ29J\\\\\\\/0TKFrD8MUtggPlq+w6+YeuYK94IvEGEDkTiLq6q9xPrBdPnFm\\\\\\\/AVi6KXHlGkJmUNFDjo5hod3TcMG+weoqPVxRqKYBQ8uTGrC+MrnjS5mfh4DAHL5MIkAdwsAfVHk+CbYDGB1hD8as+Q6n4e1Q5tULXVOtNOTSWCkba5F9l6wOqC921vPB2PbRKd4Jcl0YS0boIMH5zeIZdu0Bp5I5UBDQj4VfCh+DTkbchwNLc+qQW5ha9BrMq00elHE0hV8VOdY75PjVpcvYGUIKzgUZ9FiNndyOjwInP5Ks+fq2LNfXshbB7FDVpmlfBo8mZUdS47kfirnhDeNnmErPpOrxwuWpO5cTDmPhUZZ\\\\\\\/UhcvDwpJuMJmxYMJA7JeK8ZI9zwuGctkfO51wHjQV2bM7W6Y90Sqz2UYq7VgyT3YLFGhZ\\\\\\\/aYj+mJTCesNyZgZkfN+Cqys\\\\\\\/Fezz0HqekuiE0k1jBY2pQi6omNIwfRD9e3h2JE0COwO05as4qcuf\\\\\\\/wqi0AD2tKY87z19dY3d9AA++yCtJkpet7gTeH9hVe1yWdiNg7YW5bXovnt1U3belPhEm5vSdAhQC7sWh6OlzvZEw\\\\\\\/VCCd5JQEhWpnsj4Dg6o2s4P3lkUpP8t33t1A+9HwieB3M\\\\\\\/PQQSSNKk0fQfV5ghiog7yu1whuxDAqME6uj8jbyKqu2QKhODPKibsmZs1N8Jfx92JoMP+y29gNX7kR+X9ze8UEl8hZ1VkhQf+s1t7z5INfFb0cZmDYYpVQh1UuqGLFyMnfiVFVYDdhD86iYAvzxvFgiCaaPF1DnnXeATeYMM0pXEW3MH9TQseC7GKrEbuOrOPLIdpoSz6rTJqF59amtZx1UyIjxj7zp\\\\\\\/ZiJBYNFESbmBxsNiinjyLsRqdDlCYKqcc4PXvbEKVget7MnkLyu6dx4sG23MVBLr9u8VjvZ33wnHXt9c64uU5vT0FiNcn13TY9ysEZgCMV8p28+Xct5DwimpADHyqgw6Unhp\\\\\\\/DQz++75ZlHv71QZch3jhPRMxQFdG9FSqR8DIaudXngRaFEdCpNVIP1qvHsZowfEvEoZrB61SH6gwV\\\\\\\/YETA3YtRKRk61ng7KeDteK3Mq1c2uWYYvvN2AuMnCNiCj6UzVMCgelOZ\\\\\\\/neAtkEhOfQfVImey2xLxmp5wHs8f+4cHh89cxmd3b4Giwr3hzyUJyVdgi3LSGIF+5P93uQ4jWlADgpQAsZSaNQLcv1ppZ5PdFI8bPcU7bTSxIsU4g3pYIHTYm9Dl1DTYwvvMpWbV\\\\\\\/V0IGQeYjle55pLdF8QPZHiusErT0RF8kkphz9pIbz3Bg2hLJlsmP09lNajgU7Y3Q1G4svMTqV61QXMNfbjxlKSVMrm+NaTGfTtZtaZs7v3wuY7uJqPaoK6lReUca5MJoINwtZIULZ6PZAbyrqJneOUMSo8JgtrJwykFOIkozuLkSokUQk5K3YBceNndl9q\\\\\\\/t6EaP3yvF+eNhvti\\\\\\\/bykUMoZUwxGdFCBvSBRFdzwsy61OaM8kupUsR5FsaiikJ4xAB0+sU82fc0TFSaQsjDSH+jVss91Vi8N+ZhlTsZVjol69tApp0lTkmMxSxGYJQILPpxUkHnIbEtzCAUzfjFIJm\\\\\\\/w0vp22qzGZIrwcKixp61lm4MOyOxyflCJJyAUDF7Yw+oep2vpNDORauTY6j4i6LRuHayvi4vLRQMwhJaSxZtkEz4644byu1bjn8U75phzU7XA42cHmI85W1WOA8FTPReNQcxZbxF31uTTvG\\\\\\\/ToNJaCK4AmB5jJbxwXVCU1hXvSRmftAkdIDGns8x1+i0pjtJwUd\\\\\\\/2Js2u\\\\\\\/PTFu7QPqB12dMhHiQyxtmJQFR5r+xHgRlOT2Z7XDXcp7YQP115IBdJOa8TKVBvWdVfYDSRUT6+DdNKcXTcTR+rJoKIAYvqQn445osT4Nc3l1I+WPrNkJMJSELgMhLuM0g7NsTl\\\\\\\/2wt8bUXpuNKsRqe+KdrjZrxc3a8at9wJtMg2jTsvKIGvtM3Tpdertu0hcgPZ6AFAI3T\\\\\\\/hTCBoBw7rWJVJh4+qU897Sb\\\\\\\/lBw6gS7JO14YzPKhVq6drzJNx8UMBUxnpBeGMSBU8Vp7e8GaC+evZ+2Rs7OUXose3LkzeNIzhN710sKPOHgZ2ImIXXYBNOx6vgVt8GtbFs\\\\\\\/YKKahq3Y3p7Qdx5U11dzdZnm1ENwx\\\\\\\/5EpiM65wiA\\\\\\\/trj9MtibrFQ+R65O0Wt983\\\\\\\/LgUW\\\\\\\/9C5se7xa\\\\\\\/\\\\\\\/1fiA\\\\\\\/WhF6mMx3rLbdiXV1BUWNnOXK9pfYobDQK0ozCymh2v99judukY5aod0VjpzWQ77mZhW7BD19Z+bu8DXu2T5JucK7iqBJNeAataqyPuGAgfsKNR9+VD5m3y1cP7s4utIdtUt9y3FXK+1TOUnlilh1tdcCVIuiHbYgrgJBi4zTklzKY54ogNOcE7g3Ns\\\\\\\/OLNJgaUmvjR+NNqSITK9IlTLVvPPAVGbMLhV5n3Dcm+vIzovl4nMEAFfZ4SjkzAwv6mfSBr+6rVIcTrpJtP8UAPJob1pk79eii19S+1XloJ95YzujR4B\\\\\\\/EYCpS3kE3JUdQ+dBMQ777TUtHvj0uu04q+EDGTHMtvy9TOe9U++kM4yo6L0fEej44ar9w2ST7f+66IZ+ZkVNH5HftYCI7L1ZdCH\\\\\\\/UxQbiZ1mhiQM5yRZnfkP44+pVurfbKecQsCNwgkVSvLtbqX4heIrNwC0ZLGVMuawxDrhmNbaBUkWGaSvicvYY0bvXn+d+Is5UbfGWjHIwukOnJmDnPlAOGdZit6DPgz5TrOEqrZrk4jZLx1y4AhuiY8VhgnYENLki6ryic\\\\\\\/SIKUJyYzBaL9Pt\\\\\\\/EZGnGWmKlSAWjbsjPibIF6eLUCXFJHYOwZHxwCHNzu0AebP1Dab4eseHf8ZwEgv0H+me9DQYAu1j3RiLmfKX87RXb6jF+2orpvIC4Zn\\\\\\\/7RCfAJg+N+bbRrgOTGrP9qMTuFKiUWLpqvFoFrqSK8fSEPBVpXfqZ0fATZmweF\\\\\\\/sLP4h+OSmfaP1\\\\\\\/pk4Z\\\\\\\/eAEsKpfh1PUFM7jhlCHksW66nD8nteSoCwzAALLNS5ny08+ActXdvc4UCEyqMCi\\\\\\\/kTeseLVtQQrVjLb+GX5dhYXD2rTkWx3pdZuC7ePJyDhffCWtLr\\\\\\\/KfaTbb5JukXSJIjUvpFhhKu6yCSMIeY1d6FbCITiL8G6t28AR5V2TKhpeboFQqOP0Xip\\\\\\\/Dk+eiebT2NMoQwwwyz4pWb2CnL5y\\\\\\\/thtAhby1kwMT1hUCOSQn8LfDeiGhLR1RQQwW1uwod9puuk6y3F+hnGj0f8noSNmsgtvka1NkNc94mvTirolUUuSabXVezA+GBJgyvNDwHo1JgFjeVpsqMyHfefU0LO+9G34Q6xmbu6X\\\\\\\/D\\\\\\\/weFb8kfFr9iriAODw\\\\\\\/OmRfsCg6M1YAsJbS+UplMxZwzVjuv79RMUeVKqV\\\\\\\/7Tba+1gczhlrU+oaYU5+59fYD63TSyXBsUGlI6yMCzPmc7ujlVnHmNGLLsjt0eu\\\\\\\/2094XhbvCLEqfIbMzyKddbor0xNqzkJZV5Jm3i\\\\\\\/XWXsj6GEDDVg\\\\\\\/Et1Oqo6OIQD9VnA9wpRUPFPuxcblqPEYYFlzhTmc1DXVfKkOG9gKgMCTQsjrMxgZhefBKicy1LPrcynt2VANF2JCXBRMX\\\\\\\/BNb1lK1T8iWXcYj8UgTLmTq6yAxEqaFNgH5yk5cWoQVd33rwMOLHdrf49\\\\\\\/V9Ry88kkkZz2QyRNUCUjgWSXk2Ogg4SI7G5R5VFNqqko87XBbOHYsPU6d0trwLwjVuX8PBonTuLCiYXJZLy102Tlxcw\\\\\\\/kaL5r9qQttrXtW977nodgP3r8cUa3HVWwOJCS2I1WjqEC60UWIBfxrEGWDIxHo2HyrnjM9gq2GsT0hAzaQZVP5HbvZYTBXNfEVcMuVvptzJDuw0XRA5rLbVLo2DRQCVMrVq+Y+bdsm0DL7DmP5KO6\\\\\\\/p4oLV1g5VSNE9ApG6FoJW1z2I\\\\\\\/73WCFS1dn5pZ7m5JocA9KG5TAUltLagjDMU6rqQr9BTAKlD38K\\\\\\\/g0xKHc9SR4aSxPg7IdVZ6OZW1LMjRQe0ETZIMkOvgRmHQQam31G9CmOo1Bnzl0QEAk8bNV51jF2EHcWDnnAD+h5UNaZFoDVyD42c34DN+W582byO14rqlITMdf5q8ZpjB5sk7dpG\\\\\\\/BJMwXcsB63g\\\\\\\/6uEZNzoV5Erlhk3IsxwPuLXX1h4NO9j5N1wg5Mx\\\\\\\/7u8kC2BNXcAxFMc\\\\\\\/kMMuEFhSGSI\\\\\\\/s0OK7PEDcnCK7jPnIu1ZfZ28LRYKT9n+BgzEeYHB4q1itbC5CF3ouLP0T4agFJXjW24XXLny5kROUOOxsuqtYbGl+62Pqm8QJ9D\\\\\\\/gLuE08ixdd2b1+30MJITQiRhebLciXV98uLT9dQQrtqiytkZNtdDFyvZ1v3JwVs5gYvgwWD++0AVfROaMvgWQ9HXq3dQsixOfxL2Cnll2wo5lLQjXsxwTcn2Q\\\\\\\/J0UI74fVByiaHtKc1aDdi8dD1\\\\\\\/anDDt+hF5K5T9i4JaxIXzLZ4Mt8cXV8mGUxRsbiskfFYSt3sETbT2ZsxirQ\\\\\\\/PjPRET3OxfqxczF\\\\\\\/LLasmG7KcvGSmh4KUpaMm9tUQpHda4m7IUTSjTBYnOMRk39XKfCNQgZ8uaoJTQnUKtikBRtwzZ5uqmmCeJjTqc2oVkQFgxBVxd8x4XiBYzoNvlO+F+2hJivNhHTbVVIPvbpHohb926JO\\\\\\\/kD8II0SSf6Si998qZ6fHvCAZ\\\\\\\/eI\\\\\\\/IMr8a+sR52jCJHs5IcTXbvYBt7a6\\\\\\\/WbH8FHYY30eBUc2WFysUX2Vi1F47FCKUuyEw9dMO9aqSP+vUxiV5vVfbqNsWhVopAYNuu7kPDvqvAJg8IrWcNeOUhdhGhRL1THYNcdeNwsUte5XEFD9W2JhASXeV7UEPmXYmuGtJWLMrofsMeSz4epwafDVCB2WBW5apgJbGYpESF28FW3rKIdnX3vfw4fCuVfdqyeitmoISXNMkJ1HW91AvHIdUuj0cH1SN8pml2emzBRgxp6qgUpxCifpPcieYs\\\\\\\/rZ16ssmEtUbEENduh0QSU8SHnZSbBJv1A0NaNDc\\\\\\\/uJ+SjJoPH4gbX+sEGJPTJ35nYVQ9fhpQ09LnWDfNb56rzFplhRobG1qi72of79Gw86iPDEMFUgSkt\\\\\\\/a9P\\\\\\\/aYwRfzRTBuedug\\\\\\\/qL9DVdrwLvMC+zdDaiq4nEvqxE9kuGBeJDSpf+qjZktfNClIMCGgBykd9Hhwc5cL0kTP6Rlw+I2jYUU3TKuBsPNi1lBVXbJgNprq+mYHeElFRNJco7SV\\\\\\\/uDf03TZscQBk+TPXC7DuCoi8R6O037c0TL5679rTwIcwVz6jfhVX2mSn8HD\\\\\\\/O3sRwwrHiGytL4q6CX\\\\\\\/r4TIGb1N5wTOnw7X52JOK6ENes0EHcr\\\\\\\/yq1p3XYB6N9ZgIBuYsdP4lTCG40rhmMF9EGW1YFpZQ8+pZCoxSzCQKV3j451fg6JLD4sdwJvi3kRRUxPvsQFcesyDJYtDPfVWdAm68Sskr+sQkricBvKVucDPFJW5NkjFlXWa\\\\\\\/+wyDYOwKVN5xWP167lzwMoVZufG50X8n\\\\\\\/jEig3p2kWREoVgIcctCmbmjL3u4myI1+dPgy++ZxYSrCCutbmWuZKn0D\\\\\\\/3hECEZJN\\\\\\\/Pr03s2Iqz0BOngM9wPCDMQwbyHylj8l750UGAHqLsripH7JNkeX6PYwyktVPfud0fbuePUQd4u5dAj\\\\\\\/M5ki4HZJpyAz9KcaBgsB7Tk5wTaJTm6W6NfgtlKXX7gAciNzecLwcxJGPFGIs1mYH5U0+kk8XwtLN5fMiQ8MSFxPGWow90L1Gi42MCpBx1mfd9pggwNDleKFLj5xd87ruHCCwcBPIaGUfD8s5UJ5quYdVYbdMCBEIDDDs9Znl9C66SLPXI4z8oI\\\\\\\/SaQIV3jWCnokWDuPsf0NRIhbybnkD8F3Y7LQKx6KnjQo8tzebV9T9Nuacjy86QFg6a\\\\\\\/zKKeJWNWW6iwEX6vOnXVr1fFoEYEMSoxEA5eB903T3p13trNIpzxPUserUy1W6\\\\\\\/bWpGWBQhJOYoR+Kr\\\\\\\/BCLdQU0JKv7Ceav2gxPfuCEgdAFnugThuZFblsQy\\\\\\\/RlGUBpBIhgMZAuzJzZT3rRQIzu887D1O1m09iRx2bx57hFxHLehwrVqJiVeeuiCrsvJ1nBsPoBYUlsOOs+Y+7Gfi5TlTD0Ae6WifY07\\\\\\\/1lVpv320MD1Qj2xsKieJBSS4nX9jzyloReTb6i+9t7722UDnMuFKwdFLjvXDBrtVElPL4bpIG7G8k86zObbGDOODvrFCVjUoSRXoQZK12ihegMZNMbGC03y+c\\\\\\\/6BkBrXduuXM2fcNlOM34TOIKFE6JO\\\\\\\/cLX1raukAlZVejnN6LKjBwAyrvlFhn313uIjzaarFXu1sHvk1\\\\\\\/3MkCcEG5Pr6QxiVH8S9ZvxCWSMnmZ82ZcQk+9YD3gedXkoaO0Rj\\\\\\\/Tar0w5B0gTgEo60BgR5EfqOE\\\\\\\/IH3bcISSpcVOu0njVqhT5GpGHRa1BX1wg==\\\u0022,\\\u0022iv\\\u0022:\\\u00223lSUIrvIaRH\\\\\\\/ifKWXp1cIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c9aaa6d6cc50bcdecc7f3f7e2aa53dbd75f8cae0c5f26a94bbd057e14cf9c155\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002226\\\\\\\/SkAGDmEjb4E0I42NbpKSyacQiZC5UWkMp+omTqiBeFxhADZKjogYFxgJNqJpXU2aGjUtzA56zCjxCQzJsJ6AYDSfQR3bFcDd+yMM03wL1RdBa9M4kKYX3KuAPYtN82Y7sRJ071Zw+CqUzWjYx1sgTz4fRxKGjJeTPyVoB0nXfmEDAUDpuw19CHsBtJO+i96l1r0\\\\\\\/LGaOef+WkE+q2c42V5qEDcppM28Ipd2MTQP2uKQiOW0rwsLfvB6\\\\\\\/gYxpEwXqnOF7SU3hXHrzXw9JmYLX1oLzdCr\\\\\\\/Vinh6Mez1mPWXDMT5rAUouQnPtloyHUKO+tK6CiP0LZZ67P2AMcu62TnlEbY2xhtFJn+oEbdK7Klw8QXqsMe9CQ3+HZr7kbCezMZYPYNN+D0ws3lwfeAoOeCcpjgbLFA0lWXw81vJD84EENCLXwviv4cbVbMxbKRQyP3yZFMKORg8vJJnnrbn9cLIoGGingI3ZLQbXfCbnFemF4A3CbjcJUh\\\\\\\/0zux2ROf0SCxC1blxb2kHWaTWBb6xYFB4PQrwC441mPulS9jOfN2dPlVcUoH1Kxn5zBtvvYa0lffmC9qcRpBja54ezWiEGqCn\\\\\\\/2BCc1B\\\\\\\/3uCasrFPI6GzR\\\\\\\/KJcZBU\\\\\\\/\\\\\\\/kay1xflCBMwZ6a71L9QCUjeojCIKq27u\\\\\\\/MNtB1DiItip0U3yPGv9MXsulEj06s1Orv5HCzqiB7GchFaP6YXRtoFSbFXQg\\\\\\\/z0x9BD+F92BtxHOt\\\\\\\/0qNOeNbqja0iXBDn9UHySyFGN1Nlux9WVtvlGus6z5iVPNEODX1GZYdTY\\\\\\\/W0MlKgZxG1nDlRQY8zEO2t1mLwB9QJEKxKaG+BueWf1YJxEe\\\\\\\/Cp9yDZAU7cxxSWYHSYiAlj2d0akzN1STcBl7C1C5Q29ziUUGB1EWIc7tQabzL3WyBQ+hC4ErPQN0G9ocz6LgYPpX3cd+rlFma+kPHVL3brszRLoHIgd6GB\\\\\\\/cqPzxcmw456bYviY1brltAa+n7Om6c2lmnf5As5eDAdki1Q\\\\\\\/7rwYgU+eNy8Dzq7bwT\\\\\\\/+nbAOCagIHv9d+P9IBSYT8cIdGxnVceWfneF3y1t3enXqU2TRMtYM7KW0Ac9vXL4kuclTS9yGRCB7SxeMKFcGC5\\\\\\\/fWAcEo\\\\\\\/M\\\\\\\/XjoehUJ3bpL5\\\\\\\/DpxO2C\\\\\\\/SCAdduNH5Uiarr4w\\\\\\\/czw5uAPgYeRWFVAlZnLbP\\\\\\\/k0UBo78\\\\\\\/514moTas5+S0C0pOS2SrXSkk8eqesLkVO6BDAvgSyiTHIJ0gBoK4jEHBiHnG+bWf41nRaCqv2ysKUkJVi0RavFKLW\\\\\\\/7Bk2n+GnPlm2BtYZWpUvFp+nkx9LRx4G1MHkuox9T\\\\\\\/nAQnEuH79RIlp+Qk7LfPC4LYbdXiUvuekT0igieRsHkYGUe+jTuXyFvMYpigEaG7LHYjE6O\\\\\\\/flVbIknj250DGNbvy6AmNiLDlt5KgMtjzxC3Nk1fMCgv3fllXqE0Y2JlS+o02Sc7C72iSxAaZuBMN6YFX2bOaPbS670tdj1XJq\\\\\\\/I7jy3j9ymRzTvyEVp8akQ1XONeldxUWlq0VxQzgvZBQhR1Q0hGfeNOtBNG6Jdube4+v6V+jFdn0IWTYVZBFc7Hgrw4ogDQD7HSe6M8noHyrWci3ue9OuLkyhlMof08uOcPh2v2nkgH\\\\\\\/TTPeJhdZakeVxkuO4Y8marlEoc9fd9CJhoFkfPvimrXICIZgsUG9oZWGg8vZ8OXbIDBnzW2HjoBU5zULP4CG5w0GlkrgpirWMZyPEADOf73c9DnYcSvoWuh1XaM34WNLtJN99ZmZRjJqYBu07Ib80nx2mpGwMQt+h5f9NvNp0qPrkHV0OVWvz6TQV+wz1TKVhPmV7dqmjc\\\\\\\/4oZZyUsXXs27VIj58pkATceSZg0M2QKnutHZ6Lva7abTG9mUqzT213MFDFFzRq7wGcupUPhJF2gy0pJMRfdhr5APGt7I+st1m9\\\\\\\/aJWjTMX8Xio1SZs0Lyf81vvUF1xmX4K0KB4lN0Yu\\\\\\\/ol\\\\\\\/Avmu\\\\\\\/13GPBAMYR1tgPw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022d6nR0GWsGZIC4\\\\\\\/ysRk0Kzg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223dc68a48fdbda546ae788aac842cca6333204558d2837fe7f3f24e36463f3576\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492861 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1303" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-E_y7sQK0iLBKS__lMxqVcNOjP4hXzrCkiEPbYwENVuk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1303" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-L0dXYuiq8PLVlHQZH9PdmieitOok8b5V8CdihFY7Guk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1295" title="The Whole Genome Sequence and mRNA Transcriptome of the Tropical Cyclopoid Copepod Apocyclops royi" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1313" title="Genome and Ontogenetic-Based Transcriptomic Analyses of the Flesh Fly, Sarcophaga bullata" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492861" id="node492861" data-pisa="ggg;9/5/1303" data-pisa-master="ggg;g3.119.400039" data-apath="/ggg/9/5/1303.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4732-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John R. Shorter</span>, <span class="highwire-citation-author" data-delta="1">Maya L. Najarian</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9546-6334" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="3">Matthew Blanchard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="5">Pablo Hock</span>, <span class="highwire-citation-author" data-delta="6">Anwica Kashfeen</span>, <span class="highwire-citation-author" data-delta="7">Kathryn E. Kirchoff</span>, <span class="highwire-citation-author" data-delta="8">Colton L. Linnertz</span>, <span class="highwire-citation-author" data-delta="9">J. Sebastian Sigmon</span>, <span class="highwire-citation-author" data-delta="10">Darla R. Miller</span>, <span class="highwire-citation-author" data-delta="11">Leonard McMillan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1303-1311;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400039 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492861" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John R. Shorter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BR.%2BShorter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shorter%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BR.%2BShorter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4732-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John R. Shorter</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maya L. Najarian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maya%2BL.%2BNajarian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Najarian%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaya%2BL.%2BNajarian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9546-6334" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy A. Bell</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin T. Ferris</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Pablo Hock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BHock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hock%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BHock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anwica Kashfeen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anwica%2BKashfeen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kashfeen%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnwica%2BKashfeen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kathryn E. Kirchoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathryn%2BE.%2BKirchoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirchoff%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathryn%2BE.%2BKirchoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Colton L. Linnertz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Colton%2BL.%2BLinnertz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linnertz%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AColton%2BL.%2BLinnertz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">J. Sebastian Sigmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BSebastian%2BSigmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sigmon%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BSebastian%2BSigmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1303" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492861" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492861/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1303.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492861" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492861/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1303.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492861" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492861/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1303.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492861" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492861/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Two key features of recombinant inbred panels are well-characterized genomes and reproducibility. Here we report on the sequenced genomes of six additional Collaborative Cross (CC) strains and on inbreeding progress of 72 CC strains. We have previously reported on the sequences of 69 CC strains that were publicly available, bringing the total of CC strains with whole genome sequence up to 75. The sequencing of these six CC strains updates the efforts toward inbreeding undertaken by the UNC Systems Genetics Core. The timing reflects our competing mandates to release to the public as many CC strains as possible while achieving an acceptable level of inbreeding. The new six strains have a higher than average founder contribution from non-<em>domesticus</em> strains than the previously released CC strains. Five of the six strains also have high residual heterozygosity (&gt;14%), which may be related to non-<em>domesticus</em> founder contributions. Finally, we report on updated estimates on residual heterozygosity across the entire CC population using a novel, simple and cost effective genotyping platform on three mice from each strain. We observe a reduction in residual heterozygosity across all previously released CC strains. We discuss the optimal use of different genetic resources available for the CC population.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/whole-genome-sequence" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequence" rel="nofollow">whole genome sequence</a></li><li class="kwd"><a href="/keyword/heterozygosity" class="hw-term hw-article-keyword hw-article-keyword-heterozygosity" rel="nofollow">heterozygosity</a></li><li class="kwd"><a href="/keyword/multiparent-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparent-populations" rel="nofollow">Multiparent Populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li><li class="kwd"><a href="/keyword/inbreeding" class="hw-term hw-article-keyword hw-article-keyword-inbreeding" rel="nofollow">inbreeding</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li></ul><p id="p-3">The Collaborative Cross (CC) is a mouse multiparent population (MPP) designed for studying complex traits, modeling human diseases, and is the original stock of the Diversity Outbred population (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Iraqi <em>et al</em>. 2008</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chesler <em>et al</em>. 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Morahan <em>et al</em>. 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Churchill <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Svenson <em>et al.</em> 2012</a>). Over the past several years, the CC has been used to model a wide range of biomedically relevant traits including: allergy (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Orgel <em>et al.</em> 2018</a>), asthma (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Donoghue <em>et al.</em> 2017</a>), behavior (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chesler 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schoenrock <em>et al.</em> 2018</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Molenhuis <em>et al.</em> 2018</a>), bone development (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Levy <em>et al.</em> 2015</a>), cancer (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dorman <em>et al.</em> 2016</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Reilly 2016</a>), cellular immune phenotypes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Graham <em>et al.</em> 2017</a>), drug disposition (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nachshon <em>et al.</em> 2016</a>), exercise (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McMullan <em>et al.</em> 2018</a>), fertility (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shorter <em>et al.</em> 2017</a>), glucose tolerance (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abu-Toamih Atamni <em>et al.</em> 2017</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nashef <em>et al.</em> 2017</a>), infectious disease susceptibility (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Durrant <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rasmussen <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gralinski <em>et al.</em> 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Graham <em>et al.</em> 2015</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Graham <em>et al.</em> 2016</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gralinski <em>et al.</em> 2017</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Green <em>et al.</em> 2017</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Abu Toamih Atamni <em>et al</em>. 2018</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhang <em>et al.</em> 2018</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Collin <em>et al.</em> 2019</a>), motor performance and body weight (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mao <em>et al.</em> 2015</a>), spontaneous colitis (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rogala <em>et al.</em> 2014</a>), and toxicology (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cichocki <em>et al.</em> 2017</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hartman <em>et al.</em> 2017</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mosedale <em>et al.</em> 2017</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Venkatratnam <em>et al.</em> 2017</a>).</p><p id="p-4">The CC is derived from eight inbred strains: five classical strains (A/J, C57BL/6J, 129S1/ SvImJ, NOD/ShiLtJ, and NZO/HILtJ) and three wild-derived strains that represent three <em>Mus musculus</em> subspecies (WSB/EiJ from <em>Mus m. domesticus</em>, PWK/PhJ from <em>M. m. musculus</em>, and CAST/EiJ from <em>M. m. castaneus</em>) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Collaborative Cross Consortium 2012</a>). We previously reported on the genome sequence of 69 CC strains (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a>). We identified a significant reduction in overall genomic contribution from the two strains with non-<em>domesticus</em> subspecific origin, CAST/ EiJ and PWK/PhJ. These observations are consistent with the expectation that genetic incompatibilities between the three mouse subspecies contribute to the large-scale extinction of the CC lines during inbreeding (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Shorter <em>et al.</em> 2017</a>). We also reported on the overrepresentation of the wild-derived strains in regions of residual heterozygosity present within individual CC strains. Residual (<em>i.e.</em>, segregating) heterozygosity in the CC refers to regions where more than one founder haplotype at a locus can be found within a single strain.</p><p id="p-5">Publicly available whole genome sequence (WGS) is the new standard for widely used mouse inbred strains (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Doran <em>et al.</em> 2016</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lilue <em>et al.</em> 2018</a>). In MPPs, sequencing helps to resolve recombination breakpoints between consecutive haplotypes, assign the most likely founder haplotype in identical by descent (IBD) regions, and identify strain specific variants (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a>). However, the sequence of a single individual may provide an incomplete and biased view of an inbred strain, especially when residual heterozygosity remains at relatively high levels. Multiple individuals from a strain need to be analyzed in order to accurately determine the level of residual heterozygosity. The Most Recent Common Ancestors (MRCA/MRCAs) are the best description of residual heterozygosity at the time when a CC strain is released to the public (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Welsh <em>et al.</em> 2012</a>). In most cases, the MRCAs are the complete set of obligate ancestors. In a few cases, the ancestors were not available and additional individuals were added to better capture patterns of residual heterozygosity. As inbreeding has continued during the maintenance and distribution of the CC, residual heterozygosity is expected to decrease. The time elapsed between the MRCAs and the present should be proportional to the reduction in residual heterozygosity.</p><p id="p-6">Here, we publicly release genetic information (MRCA and whole genome sequence) on six previously unreported CC strains completed by the Systems Genetics Core Facility (SGCF) at the University of North Carolina (UNC). These six genomes have an increased frequency of haplotypes from the two non-<em>domesticus</em> inbred wild-derived strains compared to the previously reported 69 strains. The analysis of the inbreeding progress using genotypes from three recent mice from 72 CC strains reveals a decrease in residual heterozygosity across all strains. We discuss the strengths of the available CC genomic resources and plans from the SGCF to track inbreeding progress and residual heterozygosity.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mouse Strains</h3><p id="p-7">All CC mice were obtained from the SGCF at UNC (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Welsh <em>et al.</em> 2012</a>). CC mice are publicly available and can be obtained from the SGCF at UNC (<a href="http://csbio.unc.edu/CCstatus/index.py">http://csbio.unc.edu/CCstatus/index.py</a>) or from The Jackson Laboratory. The MRCAs for the six new strains were born between May 2014 and August 2015. Mice used for sequencing were all males and born between April 2016 and July 2016. Mice used for MiniMUGA genotyping were mostly born between 2017 and 2018. For strains that were extinct or not available at UNC at the time of this study, mice genotyped for MiniMUGA were born between 2015 to 2016 (Table S1).</p></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-8">All sequenced samples and MRCAs were genotyped using GigaMUGA (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Morgan <em>et al.</em> 2015</a>). Three additional mice (two females and one male) were selected from 72 CC strains and were genotyped using the MiniMUGA platform (Neogen, Lincoln NE). To ensure the correct mice were sequenced and genotyped on MiniMUGA, each sample’s genotype was compared to the sequenced mouse’s genotype for concordance (Table S2).</p></div><div id="sec-4" class="subsection"><h3>Estimation of residual heterozygosity</h3><p id="p-9">To estimate residual heterozygosity in the autosomes and the <em>X</em> chromosome in the CC, we used a subset of MiniMUGA markers. These markers are biallelic and informative in the CC founders. The genotypes in the CC founders are consistent among biological replicates and do not contain H or N calls. Finally, genotypes in the founders perfectly predict the genotypes in F1 hybrids. There are 6,293 such markers distributed across the genome. Markers where the three samples per strain had one or more H calls or do not share the same genotype call were treated as evidence of residual heterozygosity. Within each strain, clusters of markers with evidence of residual heterozygosity were combined to conservatively estimate the fraction of the genome with residual heterozygosity. To estimate relatedness between samples genotyped in MiniMUGA and their corresponding CC strain, we used a subset of 3,295 markers that overlapped between GigaMUGA and MiniMUGA genotyping platforms.</p></div><div id="sec-5" class="subsection"><h3>Haplotype reconstruction</h3><p id="p-10">Haplotype reconstruction for the MRCAs and each sequenced sample was performed as previously described in <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a> to identify the founder haplotype contribution across the genome. Founder haplotypes are reported as a probability vector for each of the 36 possible founder states (8 inbred, and 28 founder-pairs) at each marker position.</p><p id="p-11">The obligate ancestors for each line were genotyped using a combination of MegaMUGA and GigaMUGA platforms as described previously (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Morgan <em>et al.</em> 2015</a>). In the case where an ancestor was genotyped using the more dense GigaMUGA platform, MegaMUGA founder probabilities were imputed from a forward-backward Hidden Markov Model (HMM) that was used to estimate the 36-state probability at each GigaMUGA marker based on its genotype (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a>). MegaMUGA probabilities were imputed via linear interpolation at MegaMUGA marker positions. For samples genotyped on MegaMUGA a forward-backward HMM was used to directly estimate the founder-state probabilities.</p><p id="p-12">The founder-state probabilities for each strain are estimated by combining the probabilities of the strain’s obligate ancestors using the following rules. At each marker, a maximum probable founder-state was determined for each of the obligate ancestors. If the maximum likelihood was inbred, but from different founder states, then those probabilities were redistributed (added) to all combinations of heterozygous states involved (ex. <em>p</em> for AA, was added to the AB probability if BB was the highest probability for the marker in any other obligate ancestor for the strain and the probability of AA was set to 0. Likewise the sample or samples with BB as the maximum probability with value <em>r</em>, <em>r</em> was added to AB and BB set to 0). Once the discordant inbred probabilities were redistributed within a sample’s marker, the maximum probability for each founder-state was then selected from the set of obligate ancestors and the resulting vector of 36 maximum probabilities were normalized. These normalized probabilities are reported as the strain’s haplotype reconstruction.</p><p id="p-13">There are small differences in how sex chromosomes are treated by the pipeline. For males, the forward-backward HMM has only 8-states for the <em>X</em>-chromosome, but when merged with the female obligate ancestors they are extended to 36 with the heterozygous states set to 0 probabilities, and they are then combined into strain probabilities as before.</p></div><div id="sec-6" class="subsection"><h3>Sequencing</h3><p id="p-14">Whole genomic DNA was isolated as described in <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> (2017)</a> from tail tissue of a single male from the following six CC strains: CC078/TauUnc, CC079/TauUnc, CC080/TauUnc, CC081/Unc, CC082/Unc and CC083/Unc. Samples were sequenced at the UNC High Throughput Sequencing Facility (HTSF). DNA was sheared by ultrasonication, and size selection was targeted at 350 bp using a PippinPrep system. The HTSF generated sequencing libraries using Kapa (Kapa Biosystems) DNA Library Preparation Kits for Illumina sequencing. Each CC sample was run in two lanes of HiSeq4000 (Illumina). Paired end 150-bp sequencing was performed on these samples.</p></div><div id="sec-7" class="subsection"><h3>Burrows-Wheeler transforms</h3><p id="p-15">FastQC (v0.11.5) was used to confirm the quality of the raw sequencing reads. Multi- string Burrows-Wheeler transforms (msBWTs) were then constructed using the raw fastq sequencing reads of each CC sample. The msBWTs were constructed using ropeBWT (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li 2014</a>) and the msBWT merge algorithm (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Holt and McMillan 2014</a>). The msBWT data structure is a lossless compressed representation of the raw sequenced reads that can be efficiently queried for any specific subsequence. The msBWTs were employed in two pipelines. The first was to query each genome against the set of annotated variants from the Sanger Institute (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Keane <em>et al.</em> 2011</a>) that are informative among the CC founders. This set of queries was described in <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a>. The msBWTs were then queried against 45 base pair subsequences (both forward and reverse complement) from the <em>Mus musculus</em> reference genome (GRCm38.68) overlapping by 15 base pairs. The resulting read counts were archived as a data matrix and used for deletion discovery.</p></div><div id="sec-8" class="subsection"><h3>Large deletions</h3><p id="p-16">We identified homozygous genomic deletions greater than 500 base pairs in non- repetitive genomic regions that were unique to each strain. This was accomplished using msBWT queries of overlapping 45-mers shifted by 15 base pairs from the GRCm38v68 reference genome. The msBWT reports the number of reads containing each 45-base substring in both forward and reverse-complement orientations. In order to mitigate for random sequencing errors and contamination due to “index-hopping” (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">van der Valk <em>et al.</em> 2018</a>), we selected a minimum threshold of at least 3 reads as sufficient evidence of a 45-mer’s presence in each sample. Once missing 45-mer intervals were identified the msBWT was used to extract the flanking non-zero reads at the interval boundaries, and a consensus sequence was constructed for 45 to 60 bases inside of the missing interval. If this consensus was consistent with a simple polymorphism the new consensus sequence was used to extend the flanking sequence, and the msBWT was used to further extend into the near-zero interval until either the gap closed, or the new consensus’ edit distance from the reference was inconsistent with a simple polymorphism.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-17">Genotypes for MiniMUGA and GigaMUGA are available at the MMRRC-UNC <a href="https://www.med.unc.edu/mmrrc/genotypes">https://www.med.unc.edu/mmrrc/genotypes</a>. The 36 state probabilities and haplotype reconstruction of the MRCAs for the six additional CC strains are publicly available at <a href="http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs">http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs</a>. Sequenced genomes can be queried using the msBWT tools available at <a href="http://www.csbio.unc.edu/CEGSseq/index.py?run=MsbwtTools">http://www.csbio.unc.edu/CEGSseq/index.py?run=MsbwtTools</a>. The sequenced samples are available at ENA accession PRJEB31495 as FASTQ files. Zenodo accession no. 2586963 provides access to the genotypes from MiniMUGA and GigaMUGA, ideogram and haplotype files based on GigaMUGA genotypes, whole genome sequence, and MRCAs. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7814399">https://doi.org/10.25387/g3.7814399</a>.</p></div></div><div class="section results-discussion" id="sec-10"><h2 class="">Results and Discussion</h2><p id="p-18">The six strains reported here were not included in the initial report of the CC genomes (<a id="xref-ref-43-7" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a>) because of differences in health status and high levels of residual heterozygosity. Three strains (CC078/TauUnc, CC079/TauUnc and CC080/TauUnc) were quarantined to remove potential pathogens. Five strains (CC078/TauUnc, CC080/TauUnc, CC081/Unc, CC082/Unc and CC083/Unc) had levels of residual heterozygosity over 10%. Typically, &lt;10% residual heterozygosity was recommended before distribution of a CC strain independent of residual heterozygosity levels. While not fully inbred, many CC users will find that these new strains are valuable for genetic mapping, validation experiments, and disease modeling because they add haplotypes from underrepresented CC founder strains.</p><p id="p-19">Based on the MRCAs, the average residual heterozygosity of the six new strains is 19.9%, compared to 8.0% in the previously released strains. Among all 75 CC strains sequenced, five of the new strains are among the seven least inbred strains. In four of these new strains, over 20% of the genome is segregating in the MRCAs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). CC083/Unc has the unique distinction among CC strains, where three haplotypes are segregating at a locus. Specifically, 129S1/SvImJ, NOD/ShiLtJ and CAST/EiJ haplotypes are present in a 17 Mb region on chromosome <em>15</em> (31,869,802-58,735,934). All six new strains have haplotype contributions from all eight founders while almost 20% of the previously released strains have only six or seven founders (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Overall, three founder haplotypes (A/J, CAST/EiJ, PWK/PhJ) are overrepresented and two (NOD/ShiLtJ, NZO/HlLtJ) are underrepresented in the six new strains (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Importantly, two of the overrepresented founders (CAST/EiJ, PWK/PhJ) are of non-<em>domesticus</em> origin (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> &amp; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">2</a>). These two founders were underrepresented in the previous report (<a id="xref-ref-43-8" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a>). We also previously reported that wild derived founder haplotypes were significantly overrepresented in regions of residual heterozygosity in the CC (<a id="xref-ref-43-9" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a>). We observe the same pattern in the new CC strains where wild-derived haplotypes are enriched in regions of residual heterozygosity (<em>P</em> &lt; 0.0001).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493046/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493046/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">MRCA information for the six new sequenced strains</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493072/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493072/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Founder haplotype frequency in sequenced samples</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">The six new CC strains contribute at least one CAST/EiJ or PWK/PhJ haplotype to 62% and 54% of the genome, respectively. This partially relieves the deficit of these underrepresented haplotypes in the CC population. For example, chromosome <em>5</em> contains a locus between 127Mb and 137Mb where currently only one CC strain has a CAST/EiJ founder haplotype (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The new CC strains add an extra homozygous CAST/EiJ haplotype into the CC population on chromosome <em>5</em>. Another example is chromosome <em>X</em> where the previously released CC strains have a low frequency (&lt; 5%) of haplotypes from CAST/EiJ or PWK/PhJ. The six new CC strains add at least one haplotype from CAST/EiJ or PWK/PhJ, spanning 70% and 41% of the entire <em>X</em> chromosome, respectively. The addition of strains with underrepresented haplotypes is important for different reasons. First, it increases power for genetic mapping in the CC simply by increasing the number of tested haplotypes. Additional strains with the relevant haplotypes also enable independent validation experiments in the CC (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Shorter <em>et al.</em> 2017</a>). Interestingly, there are regions in the CC population where non-<em>domesticus</em> haplotypes are not represented at all (chromosome <em>5</em>: 61Mb to 64Mb; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). These areas have a decreased non-<em>domesticus</em> haplotype frequency on the periphery of the highlighted area, suggesting selection is acting against those particular haplotypes in the CC.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic contribution of new strains to the CC resource on chromosome 5. (A) Haplotype structure for the 6 new sequenced strains. (B) Current haplotype frequency among the previously sequenced 69 strains. Dotted lines represent areas with low PWK or CAST representation. (C) The substrain contribution across the eight CC founders. The CAST founder has genomic regions of domesticus origin within the highlighted area. We use the following colors and letter codes to represent the eight founder strains of the CC: A/J, yellow; C57BL/6J, gray; 129S1/SvImJ, pink; NOD/ShiLtJ, dark blue; NZO/HlLtJ, light blue; CAST/EiJ, green; PWK/ PhJ, red; and WSB/EiJ, purple." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-319661196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic contribution of new strains to the CC resource on chromosome 5. (A) Haplotype structure for the 6 new sequenced strains. (B) Current haplotype frequency among the previously sequenced 69 strains. Dotted lines represent areas with low PWK or CAST representation. (C) The substrain contribution across the eight CC founders. The CAST founder has genomic regions of &lt;em&gt;domesticus&lt;/em&gt; origin within the highlighted area. We use the following colors and letter codes to represent the eight founder strains of the CC: A/J, yellow; C57BL/6J, gray; 129S1/SvImJ, pink; NOD/ShiLtJ, dark blue; NZO/HlLtJ, light blue; CAST/EiJ, green; PWK/ PhJ, red; and WSB/EiJ, purple.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1303/F1.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1303/F1.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492930" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Genomic contribution of new strains to the CC resource on chromosome 5. (A) Haplotype structure for the 6 new sequenced strains. (B) Current haplotype frequency among the previously sequenced 69 strains. Dotted lines represent areas with low PWK or CAST representation. (C) The substrain contribution across the eight CC founders. The CAST founder has genomic regions of <em>domesticus</em> origin within the highlighted area. We use the following colors and letter codes to represent the eight founder strains of the CC: A/J, yellow; C57BL/6J, gray; 129S1/SvImJ, pink; NOD/ShiLtJ, dark blue; NZO/HlLtJ, light blue; CAST/EiJ, green; PWK/ PhJ, red; and WSB/EiJ, purple.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The eight founder strains of the CC have been sequenced at high depth, their variants annotated, and their genome recently <em>de novo</em> assembled (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Keane <em>et al.</em> 2011</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Doran <em>et al.</em> 2016</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lilue <em>et al.</em> 2018</a>). Previous genetic variation analyses have used sequence from these founders to infer haplotype specific genetic variation in the CC (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Durrant <em>et al.</em> 2011</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gralinski <em>et al.</em> 2015</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Oreper <em>et al.</em> 2017</a>). By directly sequencing CC strains, we can define founder specific variants across multiple CC strains sharing a founder haplotype at each locus, and identify strain specific variation unique to a CC strain. We first compared the sequenced samples’ GigaMUGA genotypes and confirmed that they are consistent with the corresponding MRCA. Next, we queried variation within identical founder haplotype regions at any given locus across the CC to identify <em>de novo</em> mutations. We identified four unique <em>de novo</em> deletions larger than 500bp in the newly sequenced mice that are not present in the assembled reference genome of the CC founders or in the previous cohort of 69 sequenced CC mice (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows a 19.8kb deletion in CC079/TauUnc that is predicted to delete the <em>Gm14515</em> gene. The breakpoints were resolved using msBWTs to locally assemble sequence flanking the deletion. A PCR assay was designed to discriminate between the presence or absence of this deletion, and the deletion was fixed in both the MRCA and the three CC079/TauUnc mice tested. We conclude that this mutation arose before or during the early CC inbreeding stages. Mutation and genetic drift may cause distinct variation to arise and become fixed in individual CC strains leading to genetic differences between founder strains and their haplotypes in the CC population.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492953/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492953/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Analysis of <em>de novo</em> deletions</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="De novo private deletions in the new CC sequenced strains. (A) A deletion on the X chromosome in CC079/TauUnc not shared with CC058/Unc, which also has a PWK haplotype. (B) The normalized whole-genome coverage of sequence in 1-kb bins for CC079/TauUnc. The deletion spans the pseudogene Gm14515. (C) Assembled sequencing from msBWTs shows the breakpoint of the deletion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-319661196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;De novo&lt;/em&gt; private deletions in the new CC sequenced strains. (A) A deletion on the &lt;em&gt;X&lt;/em&gt; chromosome in CC079/TauUnc not shared with CC058/Unc, which also has a PWK haplotype. (B) The normalized whole-genome coverage of sequence in 1-kb bins for CC079/TauUnc. The deletion spans the pseudogene &lt;em&gt;Gm14515&lt;/em&gt;. (C) Assembled sequencing from msBWTs shows the breakpoint of the deletion.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1303/F2.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1303/F2.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493151" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child"><em>De novo</em> private deletions in the new CC sequenced strains. (A) A deletion on the <em>X</em> chromosome in CC079/TauUnc not shared with CC058/Unc, which also has a PWK haplotype. (B) The normalized whole-genome coverage of sequence in 1-kb bins for CC079/TauUnc. The deletion spans the pseudogene <em>Gm14515</em>. (C) Assembled sequencing from msBWTs shows the breakpoint of the deletion.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To better characterize inbreeding progress in the CC, we performed a colony survey using the newly developed MiniMUGA array to genotype three mice (two females and one male) for 72 CC strains, including the six newly sequenced strains. We selected this sex ratio to sample the <em>Y</em> chromosome in each colony and to increase the <em>X</em> chromosome representation in the analysis. First, we confirmed that all sampled mice had consistent <em>Y</em> and mitochondria with their respective MRCA and sequenced sample. Next, we confirmed that all sampled mice had autosomal genotypes that matched with their respective sequenced sample using 3,295 markers common to MiniMUGA and GigaMUGA (Figure S1, Table S2, Table S3). We observe that each of the 216 samples most closely match their respective sequenced sample. This set of markers is sufficient to distinguish two closely related CC strains (CC051/TauUnc and CC059/TauUnc) (Figure S1). The colony survey also allows us to estimate current levels of corresponding MRCA (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The four clear outliers that have both a high level of current heterozygosity and a small decrease in residual heterozygosity are four of the six new CC strains. We hypothesize that the overrepresentation of non-<em>domesticus</em> haplotypes combined with high initial residual heterozygosity and small number of generations between MRCAs and genotyped samples contributes to the low change in residual heterozygosity.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Residual heterozygosity levels in CC populations. (A) Change in CC strain heterozygosity compared to total percent heterozygosity. Red dots represent the 6 new strains. Dark blue line represents the regression for change in heterozygosity compared to heterozygosity levels in the MRCAs. (B) Heterozygosity levels in sequenced samples compared to heterozygosity levels in MiniMUGA samples. Dotted black line represents the equal level of heterozygosity between both populations. Dots to the right of the dotted line are samples where the sequenced sample underestimates heterozygosity and dots to the left are where the sequenced sample over represents heterozygosity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-319661196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Residual heterozygosity levels in CC populations. (A) Change in CC strain heterozygosity compared to total percent heterozygosity. Red dots represent the 6 new strains. Dark blue line represents the regression for change in heterozygosity compared to heterozygosity levels in the MRCAs. (B) Heterozygosity levels in sequenced samples compared to heterozygosity levels in MiniMUGA samples. Dotted black line represents the equal level of heterozygosity between both populations. Dots to the right of the dotted line are samples where the sequenced sample underestimates heterozygosity and dots to the left are where the sequenced sample over represents heterozygosity.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1303/F3.medium.gif" width="378" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1303/F3.medium.gif" width="378" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1303/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492922" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Residual heterozygosity levels in CC populations. (A) Change in CC strain heterozygosity compared to total percent heterozygosity. Red dots represent the 6 new strains. Dark blue line represents the regression for change in heterozygosity compared to heterozygosity levels in the MRCAs. (B) Heterozygosity levels in sequenced samples compared to heterozygosity levels in MiniMUGA samples. Dotted black line represents the equal level of heterozygosity between both populations. Dots to the right of the dotted line are samples where the sequenced sample underestimates heterozygosity and dots to the left are where the sequenced sample over represents heterozygosity.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The CC population now has three distinct resources that describe their genomes. The first resource is genotype and haplotype reconstructions for the MRCAs of each CC strain. The second resource is whole genome sequence from a single male for 75 CC strains. The third resource is a colony survey of 72 CC strains using MiniMUGA. On average, 2.4 years separate MRCAs from sequenced samples and three years separate sequenced samples from the colony survey. However, there is considerable variation for the lengths of these periods among CC strains (Table S1). These three resources have complementary strengths that can help users of the CC.</p><p id="p-27">The MRCAs, being the most conservative estimate of haplotype composition from the CC colony, are the most robust for genetic mapping, especially when mice were acquired from a wide time range, or sampled from colonies maintained by individual investigators. Users of the CC should be aware that genetic drift has likely fixed regions of residual heterozygosity in the MRCAs, as shown by the MiniMUGA residual heterozygosity analysis (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). At regions with residual heterozygosity in the MRCAs, experimental mice could have any of the three combinations of the two segregating haplotypes. However, experimental mice will not contain any additional haplotypes not already in the MRCAs. Mapping power will be reduced in analyses where the MRCAs shows residual heterozygosity that is actually fixed in an experimental population, but one of the two haplotypes will be correctly classified. The MRCAs can be accessed through <a href="http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs">http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs</a> as genotype probability files for genetic mapping. The probability files are downloadable for each individual MRCA as well as a strain consensus probability file in the GRC mouse build 38.</p><p id="p-28">The whole genome sequence from a single male for 75 CC strains is an excellent resource to define consistent founder haplotype specific variants, identify <em>de novo</em> mutations not present in the genomes of the eight CC founder strains, and improve resolution in the transition regions between founder haplotypes. However, attempting to conduct genetic mapping based on the sequenced sample would lead to systemic under-sampling of haplotypes segregating in the SGCF colony. For 51 strains (72% of the sampled CC), the sequenced sample underestimates residual heterozygosity compared to the three more recent samples genotyped in MiniMUGA (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). On the other hand, identification of potential causative variants within candidate intervals following genetic mapping should rely on the sequenced CC mice relative to the sequenced founder strains. This alleviates potential differences between the sequences of the founders (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Keane <em>et al.</em> 2011</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Doran <em>et al.</em> 2016</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lilue <em>et al.</em> 2018</a>) and the CC (<a id="xref-ref-43-10" class="xref-bibr" href="#ref-43">Srivastava <em>et al.</em> 2017</a>), and includes strain specific variants. We created a search tool for comparing sequence across the CC to provide CC users with easy access to sequence data. The msBWT k-mer search is a fast and user-friendly tool to query any k-mer shorter than read length and returns matching sequence <a href="http://www.csbio.unc.edu/CEGSseq/index.py?run=MsbwtTools">http://www.csbio.unc.edu/CEGSseq/index.py?run=MsbwtTools</a>.</p><p id="p-29">The colony survey of 72 CC strains using the set of 6,293 markers defined in the Materials and Methods is a resource that will allow CC users to confirm the strain ID of their CC mice through genotyping. The genotypes of these strains are available at <a href="https://www.med.unc.edu/mmrrc/genotypes">https://www.med.unc.edu/mmrrc/genotypes</a>. Using a subset of 3,295 markers that overlap between MiniMUGA and GigaMUGA, we verified that genotypes of these mice were consistent with the sequenced samples and can be used as a comparison resource (Figure S1, Table S2, Table S3). We recommend that users of the CC preserve tissue samples from all CC mice (and derivatives) used in an experiment. Genetic mapping using genotypes from parental mice (<em>i.e.</em>, breeders) would provide the most accurate estimation of current segregating haplotypes for that population. This will also facilitate the correction of potential errors in mouse identity, and serve as a validation of key resources. Our colony sampling of the CC at the SGCF highlights that residual heterozygosity is decreasing, but still present. For experiments where residual heterozygosity could disrupt analysis (<em>i.e.</em> vQTL mapping studies, F<sub>2</sub> mapping studies, recombinant inbred crosses), accurate analyses and interpretation of experimental results will likely benefit from genotyping individual experimental animals to ensure accurate haplotype assignment. Residual heterozygosity may also influence genetic mapping and the presence of QTL, if only one of the two haplotypes at a locus influences a trait (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rogala <em>et al.</em> 2014</a>). QTL that have residual heterozygosity may be a double-edged sword. Fixation of a haplotype (or population subsampling) may impede experimental replication in later studies. Alternately, residual heterozygosity can allow for QTL validation studies utilizing a segregating locus within a strain.</p><p id="p-30">Given the level and potential effects of residual heterozygosity in many of the CC strains, regular and comprehensive updates on the heterozygosity still remaining within the SGCF’s colony of CC mice would improve the value of the CC resource. Therefore, the SGCF plans on heterozygosity segregating in the colony over time. Finally, it would be helpful to expand whole genome sequencing beyond a single mouse per strain to ensure that mutations that arise in the colony are taken in to account and potentially exploited in genetic analyses (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kumar <em>et al.</em> 2013</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">This work was supported in part by the following grants from the National Institutes of Health: 4U19AI100625 (FPMV, MTF), 1P01AI132130 (FPMV, MTF), 5U42OD010924 (FPMV, LM), U24HG10100 (LM, FPMV), and T32HD040127 (JRS). The Collaborative Cross project is also supported by the University Cancer Research Fund granted to Lineberger Comprehensive Cancer Center (MCR012CCRI) and a contract to FPMV by Neogen Corporation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-32">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7814399">https://doi.org/10.25387/g3.7814399</a>.</p></li><li class="fn" id="fn-2"><p id="p-33"><em>Communicating editor: L. McIntyre</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-28"><span class="received-label">Received </span>January 28, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-08"><span class="accepted-label">Accepted </span>March 8, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Shorter <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.1" data-doi="10.1007/s00335-016-9667-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abu-Toamih Atamni</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziner</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Glucose tolerance female- specific QTL mapped in collaborative cross mice.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-016-9667-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D28%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-016-9667-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-016-9667-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.2" data-doi="10.1007/s00335-018-9768-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abu Toamih Atamni</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nashef</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The Collaborative Cross mouse model for dissecting genetic susceptibility to infectious diseases.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">487</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-018-9768-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D29%26rft.spage%253D471%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-018-9768-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-018-9768-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.3" data-doi="10.4049/jimmunol.1801247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balmer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morahan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesage</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Common Heritable Immunological Variations Revealed in Genetically Diverse Inbred Mouse Strains of the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4049/jimmunol.1801247</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1801247%26rft_id%253Dinfo%253Apmid%252F30587532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6OToiMjAyLzMvNzc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.4" data-doi="10.1007/s00335-013-9492-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Out of the bottleneck: the diversity outcross and collaborative cross mouse populations in behavioral genetics research.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-013-9492-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D25%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9492-9%26rft_id%253Dinfo%253Apmid%252F24272351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9492-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24272351&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.5" data-doi="10.1007/s00335-008-9135-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branstetter</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galloway</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The collaborative cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">389</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-008-9135-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChesler%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D382%26rft.epage%253D389%26rft.atitle%253DThe%2BCollaborative%2BCross%2Bat%2BOak%2BRidge%2BNational%2BLaboratory%253A%2Bdeveloping%2Ba%2Bpowerful%2Bresource%2Bfor%2Bsystems%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9135-8%26rft_id%253Dinfo%253Apmid%252F18716833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9135-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18716833&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.6" data-doi="10.1007/s00335-012-9414-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Diversity Outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-012-9414-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D713%26rft.epage%253D718%26rft.atitle%253DThe%2Bdiversity%2Boutbred%2Bmouse%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9414-2%26rft_id%253Dinfo%253Apmid%252F22892839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9414-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892839&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.7" data-doi="10.1289/EHP788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cichocki</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furuya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatratnam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knap</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Characterization of Variability in Toxicokinetics and Toxicodynamics of Tetrachloroethylene Using the Collaborative Cross Mouse Population.</span> <abbr class="cit-jnl-abbrev">Environ. Health Perspect.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">057006</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1289/EHP788</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BHealth%2BPerspect.%26rft.volume%253D125%26rft.spage%253D057006%26rft_id%253Dinfo%253Adoi%252F10.1289%252FEHP788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1289/EHP788&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.8" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the collaborative cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132639</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.9" data-doi="10.1534/genetics.117.300211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donoghue</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Livraghi-Butrico</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McFadden</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Identification of <em>trans</em> protein QTL for secreted airway mucins in mice and a causal role for <em>Bpifb1</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">801</span>–<span class="cit-lpage">812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300211%26rft_id%253Dinfo%253Apmid%252F28851744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA3LzIvODAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.10" data-doi="10.1186/s12863-016-0349-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomlinson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic analysis of intestinal polyp development in Collaborative Cross mice carrying the Apc (Min/+) mutation.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">46</span> (erratum: BMC Genet 17: 147).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-016-0349-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D17%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-016-0349-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-016-0349-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.11" data-doi="10.1186/s13059-016-1024-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doran</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep genome sequencing and variation analysis of 13 inbred mouse strains defines candidate phenotypic alleles, private variation and homozygous truncating mutations.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-1024-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1024-y%26rft_id%253Dinfo%253Apmid%252F27480531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-1024-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27480531&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.12" data-doi="10.1101/gr.118786.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tayem</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Collaborative Cross mice and their power to map host susceptibility to <em>Aspergillus fumigatus</em> infection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.118786.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118786.110%26rft_id%253Dinfo%253Apmid%252F21493779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xMzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.13" data-doi="10.1371/journal.ppat.1003196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottomly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aicher</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Modeling host genetic regulation of influenza pathogenesis in the collaborative cross.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1003196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253De1003196%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003196%26rft_id%253Dinfo%253Apmid%252F23468633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23468633&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.14" data-doi="10.1371/journal.ppat.1005996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Mouse Model of Chronic West Nile Virus Disease.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005996</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1005996</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D12%26rft.spage%253De1005996%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005996&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.15" data-doi="10.1128/mBio.00493-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic diversity in the collaborative cross model recapitulates human West Nile virus disease outcomes.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e00493</span>–<span class="cit-lpage">e00515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mBio.00493-15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D6%26rft.spage%253De00493%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00493-15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00493-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.16" data-doi="10.1016/j.celrep.2017.10.093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mooney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choonoo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Extensive Homeostatic T Cell Phenotypic Variation within the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2313</span>–<span class="cit-lpage">2325</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2017.10.093</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D21%26rft.spage%253D2313%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2017.10.093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2017.10.093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.17" data-doi="10.1371/journal.pgen.1005504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome wide identification of SARS-CoV susceptibility loci using the collaborative cross.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005504%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005504%26rft_id%253Dinfo%253Apmid%252F26452100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26452100&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.18" data-doi="10.1534/g3.117.041434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menachery</span>  <span class="cit-name-given-names">V. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Totura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Allelic variation in the toll-like receptor adaptor protein ticam2 contributes to SARS-coronavirus pathogenesis in mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1653</span>–<span class="cit-lpage">1663</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.041434</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041434%26rft_id%253Dinfo%253Apmid%252F28592648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xNjUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekine</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrick</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Oas1b-dependent immune transcriptional profiles of west nile virus infection in the collaborative cross.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1665</span>–<span class="cit-lpage">1682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D7%26rft.spage%253D1665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.20" data-doi="10.1016/j.tox.2017.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caro</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrum</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">1,3-Butadiene- induced mitochondrial dysfunction is correlated with mitochondrial CYP2E1 activity in Collaborative Cross mice.</span> <abbr class="cit-jnl-abbrev">Toxicology</abbr> <span class="cit-vol">378</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tox.2017.01.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicology%26rft.volume%253D378%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tox.2017.01.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tox.2017.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.21" data-doi="10.1093/bioinformatics/btu584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Merging of multi-string BWTs with applications.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3524</span>–<span class="cit-lpage">3531</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu584%26rft_id%253Dinfo%253Apmid%252F25172922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25172922&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.22" data-doi="10.1007/s00335-008-9113-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The collaborative cross, developing a resource for mammalian systems genetics: a status report of the Wellcome Trust cohort.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-008-9113-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DIraqi%26rft.auinit1%253DF.%2BA.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D379%26rft.epage%253D381%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Bdeveloping%2Ba%2Bresource%2Bfor%2Bmammalian%2Bsystems%2Bgenetics%253A%2Ba%2Bstatus%2Breport%2Bof%2Bthe%2BWellcome%2BTrust%2Bcohort.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9113-1%26rft_id%253Dinfo%253Apmid%252F18521666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9113-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18521666&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.23" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10413</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.24" data-doi="10.1126/science.1245503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kourrich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">C57BL/6N mutation in cytoplasmic FMRP interacting protein 2 regulates cocaine response.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">1508</span>–<span class="cit-lpage">1512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1245503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKumar%26rft.auinit1%253DV.%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1508%26rft.epage%253D1512%26rft.atitle%253DC57BL%252F6N%2BMutation%2Bin%2BCytoplasmic%2BFMRP%2Binteracting%2Bprotein%2B2%2BRegulates%2BCocaine%2BResponse%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1245503%26rft_id%253Dinfo%253Apmid%252F24357318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDIvNjE2NS8xNTA4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.25" data-doi="10.1186/s12864-015-2213-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabet</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Collaborative cross mice in a genetic association study reveal new candidate genes for bone microarchitecture.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1013</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2213-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1013%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2213-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2213-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.26" data-doi="10.1093/bioinformatics/btu541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast construction of FM-index for long sequence reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3274</span>–<span class="cit-lpage">3275</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu541</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu541%26rft_id%253Dinfo%253Apmid%252F25107872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu541&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25107872&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.27" data-doi="10.1038/s41588-018-0223-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lilue</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doran</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiddes</span>  <span class="cit-name-given-names">I. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrudan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Sixteen diverse laboratory mouse reference genomes define strain-specific haplotypes and novel functional loci.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1574</span>–<span class="cit-lpage">1583</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0223-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0223-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0223-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.28" data-doi="10.1038/srep16247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchard</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of genetic factors that modify motor performance and body weight using Collaborative Cross mice.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">16247</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep16247</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D16247%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep16247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep16247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.29" data-doi="10.14814/phy2.13716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menachery</span>  <span class="cit-name-given-names">V. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">CC002/Unc females are mouse models of exercise-induced paradoxical fat response.</span> <abbr class="cit-jnl-abbrev">Physiol. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e13716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14814/phy2.13716</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BRep.%26rft.volume%253D6%26rft.spage%253De13716%26rft_id%253Dinfo%253Adoi%252F10.14814%252Fphy2.13716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14814/phy2.13716&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.30" data-doi="10.1186/s13229-018-0252-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molenhuis</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruining</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Soldt</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abu-Toamih Atamni</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Modeling the quantitative nature of neurodevelopmental disorders using Collaborative Cross mice.</span> <abbr class="cit-jnl-abbrev">Mol. Autism</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13229-018-0252-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BAutism%26rft.volume%253D9%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13229-018-0252-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13229-018-0252-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.31" data-doi="10.1007/s00335-008-9134-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morahan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balmer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monley</span>  <span class="cit-name-given-names">D. </span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Establishment of “The Gene Mine”: a resource for rapid identification of complex trait genes.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DMorahan%26rft.auinit1%253DG.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D390%26rft.epage%253D393%26rft.atitle%253DEstablishment%2Bof%2B%2522The%2BGene%2BMine%2522%253A%2Ba%2Bresource%2Bfor%2Brapid%2Bidentification%2Bof%2Bcomplex%2Btrait%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9134-9%26rft_id%253Dinfo%253Apmid%252F18716834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9134-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18716834&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.32" data-doi="10.1534/g3.115.022087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The mouse universal genotyping array: from substrains to subspecies.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.022087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.022087%26rft_id%253Dinfo%253Apmid%252F26684931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMi8yNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mosedale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Editor’s Highlight: Candidate Risk Factors and Mechanisms for Tolvaptan-Induced Liver Injury Are Identified Using a Collaborative Cross Approach.</span> <abbr class="cit-jnl-abbrev">Toxicol. Sci.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BSci.%26rft.volume%253D156%26rft.spage%253D438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.34" data-doi="10.3389/fgene.2016.00172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachshon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abu-Toamih Atamni</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steuerman</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheikh-Hamed</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Dissecting the Effect of Genetic Variation on the Hepatic Expression of Drug Disposition Genes across the Collaborative Cross Mouse Strains.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2016.00172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D7%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2016.00172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2016.00172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.35" data-doi="10.1902/jop.2017.170075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nashef</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abu‐Toamih Atamni</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchnik</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasturk</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kantarci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Collaborative Cross Mouse Population for Studying Alveolar Bone Changes and Impaired Glucose Tolerance Comorbidity After High‐Fat Diet Consumption.</span> <abbr class="cit-jnl-abbrev">J. Periodontol.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">e150</span>–<span class="cit-lpage">e158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1902/jop.2017.170075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPeriodontol.%26rft.volume%253D88%26rft.spage%253De150%26rft_id%253Dinfo%253Adoi%252F10.1902%252Fjop.2017.170075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1902/jop.2017.170075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.36" data-doi="10.1534/g3.117.041491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oreper</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarantino</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Inbred Strain Variant Database (ISVDB): a repository for probabilistically informed sequence differences among the collaborative cross strains and their founders.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.041491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041491%26rft_id%253Dinfo%253Apmid%252F28592645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xNjIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.37" data-doi="10.1016/j.jaci.2018.10.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orgel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeekens</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fotsch</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> Genetic diversity between mouse strains allows identification of the CC027/GeniUnc strain as an orally reactive model of peanut allergy. J. Allergy Clin. Immunol. pii: S0091–6749(18)31448–9.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jaci.2018.10.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D143%26rft.spage%253D1027%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jaci.2018.10.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jaci.2018.10.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.38" data-doi="10.1126/science.1259595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">991</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1259595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259595%26rft_id%253Dinfo%253Apmid%252F25359852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDYvNjIxMi85ODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.39" data-doi="10.1101/pdb.prot079178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reilly</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, 2016 Using the Collaborative Cross to Study the Role of Genetic Diversity in Cancer-Related Phenotypes. Cold Spring Harb Protoc. 2016(3):pdb.prot079178.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/pdb.prot079178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BProtocols%26rft.stitle%253DCold%2BSpring%2BHarb%2BProtoc%26rft.aulast%253DReilly%26rft.auinit1%253DK.%2BM.%26rft.volume%253D2016%26rft.issue%253D3%26rft.spage%253Dpdb.prot079178%26rft.epage%253Dpdb.prot079178%26rft.atitle%253DUsing%2Bthe%2BCollaborative%2BCross%2Bto%2BStudy%2Bthe%2BRole%2Bof%2BGenetic%2BDiversity%2Bin%2BCancer-Related%2BPhenotypes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fpdb.prot079178%26rft_id%253Dinfo%253Apmid%252F26933242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicHJvdG9jb2xzIjtzOjU6InJlc2lkIjtzOjIxOiIyMDE2LzMvcGRiLnByb3QwNzkxNzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.40" data-doi="10.1007/s00335-013-9499-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogala</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gooch</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The collaborative cross as a resource for modeling human disease: CC011/Unc, a new mouse model for spontaneous colitis.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-013-9499-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D25%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9499-2%26rft_id%253Dinfo%253Apmid%252F24487921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9499-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487921&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.41" data-doi="10.1111/gbb.12438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoenrock</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oreper</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrington</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ervin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Perinatal nutrition interacts with genetic background to alter behavior in a parent-of-origin-dependent manner in adult Collaborative Cross mice.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">e12438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/gbb.12438</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BBrain%2BBehav.%26rft.volume%253D17%26rft.spage%253De12438%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgbb.12438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gbb.12438&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.42" data-doi="10.1534/genetics.116.199596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shorter</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Odet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Male infertility is responsible for nearly half of the extinction observed in the mouse Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.199596</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199596%26rft_id%253Dinfo%253Apmid%252F28592496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.43" data-doi="10.1534/genetics.116.198838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarsani</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigmon</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomes of the mouse Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198838%26rft_id%253Dinfo%253Apmid%252F28592495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.44" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the Mouse Diversity outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132597</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzEzMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1303.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Valk</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezzi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ormestad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalén</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guschanski</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Low rate of index hopping on the Illumina HiSeq X platform.</span> <span class="cit-source">BioRxiv</span>. <a href="https://doi.org/10.1101/179028">https://doi.org/10.1101/179028</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.46" data-doi="10.1093/toxsci/kfx065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venkatratnam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furuya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosyk</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodnar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Editor’s Highlight: Collaborative Cross Mouse Population Enables Refinements to Characterization of the Variability in Toxicokinetics of Trichloroethylene and Provides Genetic Evidence for the Role of PPAR Pathway in Its Oxidative Metabolism.</span> <abbr class="cit-jnl-abbrev">Toxicol. Sci.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/toxsci/kfx065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftoxsci%252Fkfx065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/toxsci/kfx065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.47" data-doi="10.1007/s00335-012-9410-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manly</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Status and access to the Collaborative Cross population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">712</span> (erratum: Mamm. Genome 25: 192).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-012-9410-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DWelsh%26rft.auinit1%253DC.%2BE.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D706%26rft.epage%253D712%26rft.atitle%253DStatus%2Band%2Baccess%2Bto%2Bthe%2BCollaborative%2BCross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9410-6%26rft_id%253Dinfo%253Apmid%252F22847377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9410-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847377&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1303.48" data-doi="10.1186/s12864-018-4667-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panthier</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montagutelli</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Identification of new loci involved in the host susceptibility to <em>Salmonella typhimurium</em> in collaborative cross mice.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4667-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4667-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4667-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1303.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1295" title="The Whole Genome Sequence and mRNA Transcriptome of the Tropical Cyclopoid Copepod Apocyclops royi" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1313" title="Genome and Ontogenetic-Based Transcriptomic Analyses of the Flesh Fly, Sarcophaga bullata" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1303" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tt1IS9cfOVyfNthhoAVlw8J1iw7n7qtAO0v6Cd-GEW4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1303" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1303" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1303" class="active" data-icon-position="" data-hide-link-title="0">Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492861" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-vvF09KBmkzf5fx8a6MvyqoB-sL9E67zVQIJSunvKS8I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1303.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1303" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Pk6TUvWyNi538hLq0bpvfShG09vhfgxfYbfrySPkvKo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1303" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1303.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492861" id="node492861--2" data-pisa="ggg;9/5/1303" data-pisa-master="ggg;g3.119.400039" data-apath="/ggg/9/5/1303.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4732-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John R. Shorter</span>, <span class="highwire-citation-author" data-delta="1">Maya L. Najarian</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9546-6334" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="3">Matthew Blanchard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="5">Pablo Hock</span>, <span class="highwire-citation-author" data-delta="6">Anwica Kashfeen</span>, <span class="highwire-citation-author" data-delta="7">Kathryn E. Kirchoff</span>, <span class="highwire-citation-author" data-delta="8">Colton L. Linnertz</span>, <span class="highwire-citation-author" data-delta="9">J. Sebastian Sigmon</span>, <span class="highwire-citation-author" data-delta="10">Darla R. Miller</span>, <span class="highwire-citation-author" data-delta="11">Leonard McMillan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1303-1311;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400039 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492861--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John R. Shorter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BR.%2BShorter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shorter%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BR.%2BShorter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4732-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John R. Shorter</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maya L. Najarian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maya%2BL.%2BNajarian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Najarian%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaya%2BL.%2BNajarian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9546-6334" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy A. Bell</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin T. Ferris</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Pablo Hock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BHock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hock%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BHock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anwica Kashfeen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anwica%2BKashfeen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kashfeen%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnwica%2BKashfeen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kathryn E. Kirchoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathryn%2BE.%2BKirchoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirchoff%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathryn%2BE.%2BKirchoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Colton L. Linnertz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Colton%2BL.%2BLinnertz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linnertz%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AColton%2BL.%2BLinnertz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">J. Sebastian Sigmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BSebastian%2BSigmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sigmon%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BSebastian%2BSigmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1303" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1303&amp;title=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1303&amp;title=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1303&amp;title=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1303&amp;text=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1303&amp;title=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1303&amp;t=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1303&amp;title=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1303&amp;title=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492861/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492861" id="node492861--3" data-pisa="ggg;9/5/1303" data-pisa-master="ggg;g3.119.400039" data-apath="/ggg/9/5/1303.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequencing and Progress Toward Full Inbreeding of the Mouse Collaborative Cross Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4732-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John R. Shorter</span>, <span class="highwire-citation-author" data-delta="1">Maya L. Najarian</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9546-6334" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="3">Matthew Blanchard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="5">Pablo Hock</span>, <span class="highwire-citation-author" data-delta="6">Anwica Kashfeen</span>, <span class="highwire-citation-author" data-delta="7">Kathryn E. Kirchoff</span>, <span class="highwire-citation-author" data-delta="8">Colton L. Linnertz</span>, <span class="highwire-citation-author" data-delta="9">J. Sebastian Sigmon</span>, <span class="highwire-citation-author" data-delta="10">Darla R. Miller</span>, <span class="highwire-citation-author" data-delta="11">Leonard McMillan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1303-1311;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400039 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492861--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John R. Shorter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BR.%2BShorter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shorter%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BR.%2BShorter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4732-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John R. Shorter</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maya L. Najarian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maya%2BL.%2BNajarian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Najarian%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaya%2BL.%2BNajarian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9546-6334" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy A. Bell</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin T. Ferris</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Pablo Hock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BHock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hock%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BHock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anwica Kashfeen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anwica%2BKashfeen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kashfeen%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnwica%2BKashfeen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kathryn E. Kirchoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathryn%2BE.%2BKirchoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirchoff%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathryn%2BE.%2BKirchoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Colton L. Linnertz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Colton%2BL.%2BLinnertz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linnertz%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AColton%2BL.%2BLinnertz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">J. Sebastian Sigmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BSebastian%2BSigmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sigmon%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BSebastian%2BSigmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1303&amp;rft.epage=1311&amp;rft.atitle=Whole%20Genome%20Sequencing%20and%20Progress%20Toward%20Full%20Inbreeding%20of%20the%20Mouse%20Collaborative%20Cross%20Population&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shorter%2C+John+R.&amp;rft.au=Najarian%2C+Maya+L.&amp;rft.au=Bell%2C+Timothy+A.&amp;rft.au=Blanchard%2C+Matthew&amp;rft.au=Ferris%2C+Martin+T.&amp;rft.au=Hock%2C+Pablo&amp;rft.au=Kashfeen%2C+Anwica&amp;rft.au=Kirchoff%2C+Kathryn+E.&amp;rft.au=Linnertz%2C+Colton+L.&amp;rft.au=Sigmon%2C+J.+Sebastian&amp;rft.au=Miller%2C+Darla+R.&amp;rft.au=McMillan%2C+Leonard&amp;rft.au=Pardo-Manuel+de+Villena%2C+Fernando"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492861/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492861/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492861/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492861/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492861/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492861/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492861/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492861/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492861/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492861/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492861/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492861/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1303" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1303.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1303.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1303.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
