<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1707" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1707.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1707.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1707.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1707/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1707.atom" />
<meta name="HW.pisa" content="ggg;9/5/1707" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Determinants of QTL Mapping Power in the Realized Collaborative Cross" />
<meta name="DC.Identifier" content="10.1534/g3.119.400194" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Keele et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Collaborative Cross (CC) is a mouse genetic reference population whose range of applications includes quantitative trait loci (QTL) mapping. The design of a CC QTL mapping study involves multiple decisions, including which and how many strains to use, and how many replicates per strain to phenotype, all viewed within the context of hypothesized QTL architecture. Until now, these decisions have been informed largely by early power analyses that were based on simulated, hypothetical CC genomes. Now that more than 50 CC strains are available and more than 70 CC genomes have been observed, it is possible to characterize power based on realized CC genomes. We report power analyses from extensive simulations and examine several key considerations: 1) the number of strains and biological replicates, 2) the QTL effect size, 3) the presence of population structure, and 4) the distribution of functionally distinct alleles among the founder strains at the QTL. We also provide general power estimates to aide in the design of future experiments. All analyses were conducted with our R package, SPARCC (Simulated Power Analysis in the Realized Collaborative Cross), developed for performing either large scale power analyses or those tailored to particular CC experiments." />
<meta name="DC.Contributor" content="Gregory R. Keele" />
<meta name="DC.Contributor" content="Wesley L. Crouse" />
<meta name="DC.Contributor" content="Samir N. P. Kelada" />
<meta name="DC.Contributor" content="William Valdar" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Determinants of QTL Mapping Power in the Realized Collaborative Cross" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Collaborative Cross (CC) is a mouse genetic reference population whose range of applications includes quantitative trait loci (QTL) mapping. The design of a CC QTL mapping study involves multiple decisions, including which and how many strains to use, and how many replicates per strain to phenotype, all viewed within the context of hypothesized QTL architecture. Until now, these decisions have been informed largely by early power analyses that were based on simulated, hypothetical CC genomes. Now that more than 50 CC strains are available and more than 70 CC genomes have been observed, it is possible to characterize power based on realized CC genomes. We report power analyses from extensive simulations and examine several key considerations: 1) the number of strains and biological replicates, 2) the QTL effect size, 3) the presence of population structure, and 4) the distribution of functionally distinct alleles among the founder strains at the QTL. We also provide general power estimates to aide in the design of future experiments. All analyses were conducted with our R package, SPARCC (Simulated Power Analysis in the Realized Collaborative Cross), developed for performing either large scale power analyses or those tailored to particular CC experiments.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1707" />
<meta name="citation_id" content="9/5/1707" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1707" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1707.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1707.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1707.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400194" />
<meta name="citation_pmid" content="30914424" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="21" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="1707" />
<meta name="citation_lastpage" content="1727" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gregory R. Keele" />
<meta name="citation_author_institution" content="Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1843-7900" />
<meta name="citation_author" content="Wesley L. Crouse" />
<meta name="citation_author_institution" content="Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5745-4490" />
<meta name="citation_author" content="Samir N. P. Kelada" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Marsico Lung Institute, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2676-9232" />
<meta name="citation_author" content="William Valdar" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_email" content="william.valdar@unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2419-0430" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging Collaborative Cross.;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Ayroles;citation_author=S. M. Buchanan;citation_author=C. O’Leary;citation_author=K. Skutt-Kakaria;citation_author=J. K. Grenier;citation_title=Behavioral idiosyncrasy reveals genetic control of phenotypic variability.;citation_volume=112;citation_year=2015;citation_pmid=25953335;citation_doi=10.1073/pnas.1503830112" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Baud;citation_author=R. Hermsen;citation_author=V. Guryev;citation_author=P. Stridh;citation_author=D. Graham;citation_title=Combined sequence-based and genetic mapping analysis of complex traits in outbred rats.;citation_pages=767-775;citation_volume=45;citation_year=2013;citation_pmid=23708188;citation_doi=10.1038/ng.2644" />
<meta name="citation_reference" content="citation_author=W. Beavis;citation_pages=250-266;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=J. K. Belknap;citation_title=Effect of within-strain sample size on QTL detection and mapping using recombinant inbred mouse strains.;citation_pages=29-38;citation_volume=28;citation_year=1998;citation_issue=1;citation_pmid=9573644;citation_doi=10.1023/A:1021404714631" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=J. K. Belknap;citation_author=S. R. Mitchell;citation_author=L. A. O’Toole;citation_author=M. L. Helms;citation_author=J. C. Crabbe;citation_title=Type I and type II error rates for quantitative trait loci (QTL) mapping studies using recombinant inbred mouse strains.;citation_pages=149-160;citation_volume=26;citation_year=1996;citation_issue=2;citation_pmid=8639150;citation_doi=10.1007/BF02359892" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Bouchet;citation_author=M. O. Olatoye;citation_author=S. R. Marla;citation_author=R. Perumal;citation_author=T. Tesso;citation_title=Increased Power To Dissect Adaptive Traits in Global Sorghum Diversity Using a Nested Association Mapping Population.;citation_volume=206;citation_year=2017;citation_pmid=28592497;citation_doi=10.1534/genetics.116.198499" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_author=D. M. Gatti;citation_author=P. Simecek;citation_author=N. A. Furlotte;citation_author=P. Prins;citation_title=R/qtl2: Software for Mapping Quantitative Trait Loci with High-Dimensional Data and Multiparent Populations.;citation_volume=211;citation_year=2019;citation_pmid=30591514;citation_doi=10.1534/genetics.118.301595" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=L. D. Brown;citation_author=T. T. Cai;citation_author=A. DasGupta;citation_title=Interval Estimation for a Binomial Proportion.;citation_pages=101-133;citation_volume=16;citation_year=2001;citation_doi=10.1214/ss/1009213286" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=E. J. Chesler;citation_author=D. R. Miller;citation_author=L. R. Branstetter;citation_author=L. D. Galloway;citation_author=B. L. Jackson;citation_title=The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.;citation_pages=382-389;citation_volume=19;citation_year=2008;citation_issue=6;citation_pmid=18716833;citation_doi=10.1007/s00335-008-9135-8" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the Collaborative Cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. M. Cowen;citation_title=The use of replicated progenies in marker-based mapping of QTL’s.;citation_pages=857-862;citation_volume=75;citation_year=1988;citation_doi=10.1007/BF00258045" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. J. Crowley;citation_author=Y. Kim;citation_author=A. B. Lenarcic;citation_author=C. R. Quackenbush;citation_author=C. J. Barrick;citation_title=Genetics of adverse reactions to haloperidol in a mouse diallel: a drug-placebo experiment and Bayesian causal analysis.;citation_volume=196;citation_year=2014;citation_pmid=24240528;citation_doi=10.1534/genetics.113.156901" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. J. Crowley;citation_author=V. Zhabotynsky;citation_author=W. Sun;citation_author=S. Huang;citation_author=I. K. Pakatci;citation_title=Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance.;citation_pages=353-360;citation_volume=47;citation_year=2015;citation_pmid=25730764;citation_doi=10.1038/ng.3222" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Dell’Acqua;citation_author=D. M. Gatti;citation_author=G. Pea;citation_author=F. Cattonaro;citation_author=F. Coppens;citation_title=Genetic properties of the MAGIC maize population: a new platform for high definition QTL mapping in Zea mays.;citation_pages=167;citation_volume=16;citation_year=2015;citation_pmid=26357913;citation_doi=10.1186/s13059-015-0716-z" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Doerge;citation_author=G. Churchill;citation_title=Permutation Tests for Multiple Loci Affecting a Quantitative Character;citation_pages=285-294;citation_volume=142;citation_year=1996;citation_issue=1;citation_pmid=8770605" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. J. Donoghue;citation_author=A. Livraghi-Butrico;citation_author=K. M. McFadden;citation_author=J. M. Thomas;citation_author=G. Chen;citation_title=Identification of trans Protein QTL for Secreted Airway Mucins in Mice and a Causal Role for Bpifb1.;citation_volume=207;citation_year=2017;citation_pmid=28851744;citation_doi=10.1534/genetics.117.300211" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=F. Dudbridge;citation_author=B. P. Koeleman;citation_title=Efficient computation of significance levels for multiple associations in large studies of correlated data, including genomewide association studies.;citation_pages=424-435;citation_volume=75;citation_year=2004;citation_issue=3;citation_pmid=15266393;citation_doi=10.1086/423738" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Durrant;citation_author=H. Tayem;citation_author=B. Yalcin;citation_author=J. Cleak;citation_author=L. Goodstadt;citation_title=Collaborative Cross mice and their power to map host susceptibility to Aspergillus fumigatus infection.;citation_volume=21;citation_year=2011;citation_pmid=21493779;citation_doi=10.1101/gr.118786.110" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=K. C. Falke;citation_author=M. Frisch;citation_title=Power and false-positive rate in QTL detection with near-isogenic line libraries.;citation_pages=576-584;citation_volume=106;citation_year=2011;citation_issue=4;citation_pmid=20683482;citation_doi=10.1038/hdy.2010.87" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=M. T. Ferris;citation_author=D. L. Aylor;citation_author=D. Bottomly;citation_author=A. C. Whitmore;citation_author=L. D. Aicher;citation_title=Modeling Host Genetic Regulation of Influenza Pathogenesis in the Collaborative Cross.;citation_pages=e1003196;citation_volume=9;citation_year=2013;citation_pmid=23468633;citation_doi=10.1371/journal.ppat.1003196" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Pathol.;citation_author=M. F. W. Festing;citation_title=Inbred strains should replace outbred stocks in toxicology, safety testing, and drug development.;citation_volume=38;citation_year=2010;citation_pmid=20562325;citation_doi=10.1177/0192623310373776" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L.-Y. Wu;citation_author=W. Valdar;citation_title=Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice.;citation_pages=1623-1633;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.114.013748" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=J. B. Graham;citation_author=J. L. Swarts;citation_author=M. Mooney;citation_author=G. Choonoo;citation_author=S. Jeng;citation_title=Extensive Homeostatic T Cell Phenotypic Variation within the Collaborative Cross.;citation_pages=2313-2325;citation_volume=21;citation_year=2017;citation_doi=10.1016/j.celrep.2017.10.093" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. E. Gralinski;citation_author=M. T. Ferris;citation_author=D. L. Aylor;citation_author=A. C. Whitmore;citation_author=R. Green;citation_title=Genome Wide Identification of SARS-CoV Susceptibility Loci Using the Collaborative Cross.;citation_pages=e1005504;citation_volume=11;citation_year=2015;citation_pmid=26452100;citation_doi=10.1371/journal.pgen.1005504" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. M. Kaeppler;citation_title=Quantitative trait locus mapping using sets of near-isogenic lines: Relative power comparisons and technical considerations.;citation_pages=384-392;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050574" />
<meta name="citation_reference" content="citation_journal_title=Neurosci. Biobehav. Rev.;citation_author=N. Kafkafi;citation_author=J. Agassi;citation_author=E. J. Chesler;citation_author=J. C. Crabbe;citation_author=W. E. Crusio;citation_title=Reproducibility and replicability of rodent phenotyping in preclinical studies.;citation_pages=218-232;citation_volume=87;citation_year=2018;citation_doi=10.1016/j.neubiorev.2018.01.003" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Kafkafi;citation_author=Y. Benjamini;citation_author=A. Sakov;citation_author=G. I. Elmer;citation_author=I. Golani;citation_title=Genotype-environment interactions in mouse behavior: a way out of the problem.;citation_volume=102;citation_year=2005;citation_pmid=15764701;citation_doi=10.1073/pnas.0409554102" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S.-Y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Obesity (Silver Spring);citation_author=G. R. Keele;citation_author=J. W. Prokop;citation_author=H. He;citation_author=K. Holl;citation_author=J. Littrell;citation_title=Genetic Fine-Mapping and Identification of Candidate Genes and Variants for Adiposity Traits in Outbred Rats.;citation_pages=213-222;citation_volume=26;citation_year=2018;citation_doi=10.1002/oby.22075" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. N. P. Kelada;citation_title=Plethysmography Phenotype QTL in Mice Before and After Allergen Sensitization and Challenge.;citation_volume=6;citation_year=2016;citation_pmid=27449512;citation_doi=10.1534/g3.116.032912" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. N. P. Kelada;citation_author=D. L. Aylor;citation_author=B. C. E. Peck;citation_author=J. F. Ryan;citation_author=U. Tavarez;citation_title=Genetic Analysis of Hematological Parameters in Incipient Lines of the Collaborative Cross.;citation_pages=157-165;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.111.001776" />
<meta name="citation_reference" content="citation_author=E. G. King;citation_author=A. D. Long;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. G. King;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=Properties and power of the Drosophila synthetic population resource for the routine dissection of complex traits.;citation_volume=191;citation_year=2012;citation_pmid=22505626;citation_doi=10.1534/genetics.112.138537" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Kislukhin;citation_author=E. G. King;citation_author=K. N. Walters;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=The Genetic Architecture of Methotrexate Toxicity Is Similar in Drosophila melanogaster and Humans. G3: Genes, Genomes;citation_pages=1301-1310;citation_volume=3;citation_year=2013;citation_doi=10.1534/g3.113.006619" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. R. Klasen;citation_author=H. P. Piepho;citation_author=B. Stich;citation_title=QTL detection power of multi-parental RIL populations in Arabidopsis thaliana.;citation_pages=626-632;citation_volume=108;citation_year=2012;citation_issue=6;citation_pmid=22334115;citation_doi=10.1038/hdy.2011.133" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. J. Knapp;citation_author=W. C. Bridges;citation_title=Using molecular markers to estimate quantitative trait locus parameters: power and genetic variances for unreplicated and replicated progeny.;citation_pages=769-777;citation_volume=126;citation_year=1990;citation_issue=3;citation_pmid=2249768" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Levy;citation_author=R. F. Mott;citation_author=F. A. Iraqi;citation_author=Y. Gabet;citation_title=Collaborative cross mice in a genetic association study reveal new candidate genes for bone microarchitecture.;citation_pages=1013;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2213-x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Li;citation_author=P. Bradbury;citation_author=E. Ersoz;citation_author=E. S. Buckler;citation_author=J. Wang;citation_title=Joint QTL linkage mapping for multiple-cross mating design sharing one common parent.;citation_pages=e17573-e17573;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21423655;citation_doi=10.1371/journal.pone.0017573" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Lippert;citation_author=J. Listgarten;citation_author=Y. Liu;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_title=FaST linear mixed models for genome-wide association studies.;citation_pages=833-835;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21892150;citation_doi=10.1038/nmeth.1681" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Y. Liu;citation_author=Q. Zhang;citation_author=L. McMillan;citation_author=F. P.-M. de Villena;citation_author=W. Wang;citation_title=Efficient genome ancestry inference in complex pedigrees with inbreeding.;citation_volume=26;citation_year=2010;citation_pmid=20529906;citation_doi=10.1093/bioinformatics/btq187" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=N. I. Lorè;citation_author=F. A. Iraqi;citation_author=A. Bragonzi;citation_title=Host genetic diversity influences the severity of Pseudomonas aeruginosa pneumonia in the Collaborative Cross mice.;citation_pages=106;citation_volume=16;citation_year=2015;citation_pmid=26310945;citation_doi=10.1186/s12863-015-0260-6" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. C. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=O. Martínez;citation_author=R. N. Curnow;citation_title=Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers.;citation_pages=480-488;citation_volume=85;citation_year=1992;citation_pmid=24197463;citation_doi=10.1007/BF00222330" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Physiol. Endocrinol. Metab.;citation_author=W. F. Mathes;citation_author=D. L. Aylor;citation_author=D. R. Miller;citation_author=G. A. Churchill;citation_author=E. J. Chesler;citation_title=Architecture of energy balance traits in emerging lines of the Collaborative Cross.;citation_volume=300;citation_year=2011;citation_pmid=21427413;citation_doi=10.1152/ajpendo.00707.2010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Autism;citation_author=R. T. Molenhuis;citation_author=H. Bruining;citation_author=M. J. V. Brandt;citation_author=P. E. van Soldt;citation_author=H. J. Abu-Toamih Atamni;citation_title=Modeling the quantitative nature of neurodevelopmental disorders using Collaborative Cross mice.;citation_pages=63;citation_volume=9;citation_year=2018;citation_doi=10.1186/s13229-018-0252-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. Morgan;citation_author=C.-P. Fu;citation_author=C.-Y. Kao;citation_author=C. E. Welsh;citation_author=J. P. Didion;citation_title=The Mouse Universal Genotyping Array: From Substrains to Subspecies. G3: Genes, Genomes;citation_pages=263-279;citation_volume=6;citation_year=2016;citation_doi=10.1534/g3.115.022087" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Sci.;citation_author=M. Mosedale;citation_author=Y. Kim;citation_author=W. J. Brock;citation_author=S. E. Roth;citation_author=T. Wiltshire;citation_title=Candidate Risk Factors and Mechanisms for Tolvaptan-Induced Liver Injury Are Identified Using a Collaborative Cross Approach.;citation_volume=156;citation_year=2017;citation_pmid=28115652;citation_doi=10.1093/toxsci/kfw269" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mott;citation_author=C. J. Talbot;citation_author=M. G. Turri;citation_author=A. C. Collins;citation_author=J. Flint;citation_title=A method for fine mapping quantitative trait loci in outbred animal stocks.;citation_volume=97;citation_year=2000;citation_pmid=11050180;citation_doi=10.1073/pnas.230304397" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. A. Najarro;citation_author=J. L. Hackett;citation_author=B. R. Smith;citation_author=C. A. Highfill;citation_author=E. G. King;citation_title=Identifying Loci Contributing to Natural Variation in Xenobiotic Resistance in Drosophila.;citation_pages=e1005663;citation_volume=11;citation_year=2015;citation_doi=10.1371/journal.pgen.1005663" />
<meta name="citation_reference" content="citation_author=L. M. Noble;citation_author=I. Chelo;citation_author=T. Guzella;citation_author=B. Afonso;citation_author=D. D. Riccardi;citation_year=2017;citation_pmid=29066469;citation_doi=10.1534/genetics.117.300406" />
<meta name="citation_reference" content="citation_journal_title=J. Allergy Clin. Immunol.;citation_author=K. Orgel;citation_author=J. M. Smeekens;citation_author=P. Ye;citation_author=L. Fotsch;citation_author=R. Guo;citation_title=Genetic diversity between mouse strains allows identification of the CC027/GeniUnc strain as an orally reactive model of peanut allergy.;citation_pages=1027-1037.e7;citation_volume=143;citation_year=2019;citation_doi=10.1016/j.jaci.2018.10.009" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=J. L. Peirce;citation_author=L. Lu;citation_author=J. Gu;citation_author=L. M. Silver;citation_author=R. W. Williams;citation_title=A new set of BXD recombinant inbred lines from advanced intercross populations in mice.;citation_pages=7-7;citation_volume=5;citation_year=2004;citation_pmid=15117419;citation_doi=10.1186/1471-2156-5-7" />
<meta name="citation_reference" content="Pfaff, B. and A. McNeil, 2018 evir: Extreme Values in R. R package version 1.7–4." />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. M. Philip;citation_author=G. Sokoloff;citation_author=C. L. Ackert-Bicknell;citation_author=M. Striz;citation_author=L. Branstetter;citation_title=Genetic analysis in the Collaborative Cross breeding population.;citation_volume=21;citation_year=2011;citation_pmid=21734011;citation_doi=10.1101/gr.113886.110" />
<meta name="citation_reference" content="citation_journal_title=Genes Immun.;citation_author=J. Phillippi;citation_author=Y. Xie;citation_author=D. R. Miller;citation_author=T. A. Bell;citation_author=Z. Zhang;citation_title=Using the emerging Collaborative Cross to probe the immune system.;citation_pages=38-46;citation_volume=15;citation_year=2014;citation_pmid=24195963;citation_doi=10.1038/gene.2013.59" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Ram;citation_author=M. Mehta;citation_author=L. Balmer;citation_author=D. M. Gatti;citation_author=G. Morahan;citation_title=Rapid Identification of Major-Effect Genes Using the Collaborative Cross;citation_pages=75-86;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236450;citation_doi=10.1534/genetics.114.163014" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. L. Rasmussen;citation_author=A. Okumura;citation_author=M. T. Ferris;citation_author=R. Green;citation_author=F. Feldmann;citation_title=Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance.;citation_volume=346;citation_year=2014;citation_pmid=25359852;citation_doi=10.1126/science.1259595" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=A. R. Rogala;citation_author=A. P. Morgan;citation_author=A. M. Christensen;citation_author=T. J. Gooch;citation_author=T. A. Bell;citation_title=The Collaborative Cross as a resource for modeling human disease: CC011/Unc, a new mouse model for spontaneous colitis.;citation_pages=95-108;citation_volume=25;citation_year=2014;citation_pmid=24487921;citation_doi=10.1007/s00335-013-9499-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Detecting major genetic loci controlling phenotypic variability in experimental crosses.;citation_volume=188;citation_year=2011;citation_pmid=21467569;citation_doi=10.1534/genetics.111.127068" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Rutledge;citation_author=D. L. Aylor;citation_author=D. E. Carpenter;citation_author=B. C. Peck;citation_author=P. Chines;citation_title=Genetic regulation of Zfp30, CXCL1, and neutrophilic inflammation in murine lung.;citation_volume=198;citation_year=2014;citation_pmid=25114278;citation_doi=10.1534/genetics.114.168138" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. R. Shorter;citation_author=F. Odet;citation_author=D. L. Aylor;citation_author=W. Pan;citation_author=C.-Y. Kao;citation_title=Male Infertility Is Responsible for Nearly Half of the Extinction Observed in the Mouse Collaborative Cross.;citation_volume=206;citation_year=2017;citation_pmid=28592496;citation_doi=10.1534/genetics.116.199596" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=A. Shusterman;citation_author=Y. Salyma;citation_author=A. Nashef;citation_author=M. Soller;citation_author=A. Wilensky;citation_title=Genotype is an important determinant factor of host susceptibility to periodontitis in the Collaborative Cross and inbred mouse populations.;citation_pages=68;citation_volume=14;citation_year=2013;citation_doi=10.1186/1471-2156-14-68" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Soller;citation_author=J. S. Beckmann;citation_title=Marker-based mapping of quantitative trait loci using replicated progenies.;citation_pages=205-208;citation_volume=80;citation_year=1990;citation_pmid=24220897;citation_doi=10.1007/BF00224388" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Srivastava;citation_author=A. P. Morgan;citation_author=M. L. Najarian;citation_author=V. K. Sarsani;citation_author=J. S. Sigmon;citation_title=Genomes of the mouse Collaborative Cross.;citation_volume=206;citation_year=2017;citation_pmid=28592495;citation_doi=10.1534/genetics.116.198838" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-resolution genetic mapping using the Mouse Diversity outbred population.;citation_volume=190;citation_year=2012;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Takuno;citation_author=R. Terauchi;citation_author=H. Innan;citation_title=The Power of QTL Mapping with RILs.;citation_pages=e46545-e46545;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23056339;citation_doi=10.1371/journal.pone.0046545" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. W. Threadgill;citation_author=G. A. Churchill;citation_title=Ten Years of the Collaborative Cross.;citation_volume=190;citation_year=2012;citation_pmid=22345604;citation_doi=10.1534/genetics.111.138032" />
<meta name="citation_reference" content="Threadgill, D. W., K. W. Hunter, and R. W. Williams, 2002 Genetic dissection of complex and quantitative traits: from fantasy to reality via a community effort. Mamm. Genome 13: 175–8. 10.1007/s00335-001-4001-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Valdar;citation_author=J. Flint;citation_author=R. Mott;citation_title=Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice.;citation_volume=172;citation_year=2006;citation_pmid=16361245;citation_doi=10.1534/genetics.104.039313" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_title=Modern Applied Statistics with S;citation_author=W. N. Venables;citation_author=B. D. Ripley;citation_year=2002;citation_doi=10.1007/978-0-387-21706-2" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Sci.;citation_author=A. Venkatratnam;citation_author=S. Furuya;citation_author=O. Kosyk;citation_author=A. Gold;citation_author=W. Bodnar;citation_title=Collaborative Cross Mouse Population Enables Refinements to Characterization of the Variability in Toxicokinetics of Trichloroethylene and Provides Genetic Evidence for the Role of PPAR Pathway in Its Oxidative Metabolism.;citation_volume=158;citation_year=2017;citation_doi=10.1093/toxsci/kfx065" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. Vered;citation_author=C. Durrant;citation_author=R. Mott;citation_author=F. A. Iraqi;citation_title=Susceptibility to Klebsiella pneumonaie infection in collaborative cross mice is a complex trait controlled by at least three loci acting at different time points.;citation_pages=865;citation_volume=15;citation_year=2014;citation_pmid=25283706;citation_doi=10.1186/1471-2164-15-865" />
<meta name="citation_reference" content="Welsh, C. E., D. R. Miller, K. F. Manly, J. Wang, L. McMillan, et al., 2012 Status and access to the Collaborative Cross population. Mamm. Genome 23: 706–12. (erratum: Mamm. Genome 25:192). 10.1007/s00335-012-9410-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Theoretical Basis of the Beavis Effect;citation_pages=2259-2268;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704201;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Yalcin;citation_author=J. Flint;citation_author=R. Mott;citation_title=Using progenitor strain information to identify quantitative trait nucleotides in outbred mice.;citation_volume=171;citation_year=2005;citation_pmid=16085706;citation_doi=10.1534/genetics.104.028902" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=E. Yamamoto;citation_author=H. Iwata;citation_author=T. Tanabata;citation_author=R. Mizobuchi;citation_author=J.-i. Yonemaru;citation_title=Effect of advanced intercrossing on genome structure and on the power to detect linked quantitative trait loci in a multi-parent population: a simulation study in rice.;citation_pages=50;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2156-15-50" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic design and statistical power of nested association mapping in maize.;citation_volume=178;citation_year=2008;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Zhang;citation_author=W. Wang;citation_author=W. Valdar;citation_title=Bayesian modeling of haplotype effects in multiparent populations.;citation_volume=198;citation_year=2014;citation_pmid=25236455;citation_doi=10.1534/genetics.114.166249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Zheng;citation_author=M. P. Boer;citation_author=F. A. van Eeuwijk;citation_title=Reconstruction of Genome Ancestry Blocks in Multiparental Populations.;citation_volume=200;citation_year=2015;citation_pmid=26048018;citation_doi=10.1534/genetics.115.177873" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Genome-wide efficient mixed-model analysis for association studies.;citation_pages=821-824;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706312;citation_doi=10.1038/ng.2310" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Zöllner;citation_author=J. K. Pritchard;citation_title=Overcoming the winner's curse: estimating penetrance parameters from case-control data.;citation_pages=605-615;citation_volume=80;citation_year=2007;citation_issue=4;citation_pmid=17357068;citation_doi=10.1086/512821" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Zou;citation_author=Z. Xu;citation_author=T. Vision;citation_title=Assessing the significance of quantitative trait loci in replicable mapping populations.;citation_volume=174;citation_year=2006;citation_pmid=16888335;citation_doi=10.1534/genetics.106.059469" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Determinants of QTL Mapping Power in the Realized Collaborative Cross" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="The Collaborative Cross (CC) is a mouse genetic reference population whose range of applications includes quantitative trait loci (QTL) mapping. The design of a CC QTL mapping study involves multiple decisions, including which and how many strains to use, and how many replicates per strain to phenotype, all viewed within the context of hypothesized QTL architecture. Until now, these decisions have been informed largely by early power analyses that were based on simulated, hypothetical CC genomes. Now that more than 50 CC strains are available and more than 70 CC genomes have been observed, it is possible to characterize power based on realized CC genomes. We report power analyses from extensive simulations and examine several key considerations: 1) the number of strains and biological replicates, 2) the QTL effect size, 3) the presence of population structure, and 4) the distribution of functionally distinct alleles among the founder strains at the QTL. We also provide general power estimates to aide in the design of future experiments. All analyses were conducted with our R package, SPARCC (Simulated Power Analysis in the Realized Collaborative Cross), developed for performing either large scale power analyses or those tailored to particular CC experiments." />
<meta name="og-title" property="og:title" content="Determinants of QTL Mapping Power in the Realized Collaborative Cross" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1707" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Collaborative Cross (CC) is a mouse genetic reference population whose range of applications includes quantitative trait loci (QTL) mapping. The design of a CC QTL mapping study involves multiple decisions, including which and how many strains to use, and how many replicates per strain to phenotype, all viewed within the context of hypothesized QTL architecture. Until now, these decisions have been informed largely by early power analyses that were based on simulated, hypothetical CC genomes. Now that more than 50 CC strains are available and more than 70 CC genomes have been observed, it is possible to characterize power based on realized CC genomes. We report power analyses from extensive simulations and examine several key considerations: 1) the number of strains and biological replicates, 2) the QTL effect size, 3) the presence of population structure, and 4) the distribution of functionally distinct alleles among the founder strains at the QTL. We also provide general power estimates to aide in the design of future experiments. All analyses were conducted with our R package, SPARCC (Simulated Power Analysis in the Realized Collaborative Cross), developed for performing either large scale power analyses or those tailored to particular CC experiments." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Determinants of QTL Mapping Power in the Realized Collaborative Cross | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7ENk3ZlrJ747UwJ6btov_364vCwx2EBTHckXC2vwpuc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493200","apath":"\/ggg\/9\/5\/1707.atom","pisa":"ggg;9\/5\/1707","ac":{"\/ggg\/9\/5\/1707.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1707.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1707"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1707"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1707.figures-only":true,"\/content\/9\/5\/1707":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BFZP5uRe17EXmDtly9ec27JOP4xf7IflBq4dD+CQPbLQuukdB2VfHSSTCeseH\\\\\\\/IIKa3GdOum2PKsu08i7tI5ZmDWOS0IWaC7MPkOAFG+nrdZBymHPtR3nml0\\\\\\\/8DuMsGVamMXb5MWyYLNft0YbXBcUhy\\\\\\\/elVStKE5wIg1D654iVrlBAosUimNInQqFotcz\\\\\\\/gZkEjCNRpo82Yoff3yOirdfT\\\\\\\/MeQ1YQwVu33S4EZpOi3rXwScxOsaocfpW\\\\\\\/J1oqu4sRxlV5lL7Y2AlmOLaVn6OLxBwnE00KI5V5peme3FNUZVEhkUBOZURj\\\\\\\/skgEuHEjmv6QRbMwlV1oV0V\\\\\\\/oh5fvjEQz90yQSktlKcnnXcs\\\\\\\/OOLj\\\\\\\/JnlF4DGdoUnQLA8P56D36eBpOevciw0JXwG6390tutRuG4B\\\\\\\/0V3YTYWlNHzQ99rBFb\\\\\\\/Fwz\\\\\\\/9mfPQYJSvwqZjcgewVuJQKBzjawTvLbiq47F2dLknhvDKtRKv+O\\\\\\\/rWOZHnaaMGo7aFQhDbdjR6MBUHC5XH+YNmELHUodX88v9nJP5Z1TZCv2p\\\\\\\/TGE0j\\\\\\\/bnM8+5V49QLpOEE5iiMuQXQVaLA+s6pJ6Lphn+N\\\\\\\/ElrfLHX9lbt\\\\\\\/2sjlWN5f5xFc4fEL4sx0twyV4qm6ASZDW2OEfswUNZcHTD7bqZjJx1o\\\\\\\/fSEix4MATqAbkZS+sAO5FkZY8Q0iKYj3uqhyxPx47j9aHAlNNADSGDvH4EdaWvGjMwl4kDwiotG+LiCZ2c8OaeMs2OhRZIOUsoGmJPa\\\\\\\/rD+tnbxpU0eOXCX2duyKKMjhA8HHzFq88gmt84ePl5+++VYLipRMhL4la20QF15PK88WGFR+i7v8RHsNqUlGQfA9BJmcrrimJXNhuT4rOnoyPFS9i5a7n+mmSp4sv\\\\\\\/ZCGcLEImGNTO4b\\\\\\\/+t0TBr\\\\\\\/kjax9rskIfvYRPfTTyhtaYL54npjoSnC+vHdmBRTBrNslw6egSFTqcg7SnVc\\\\\\\/XBa3hb8JbFLUOkk6E\\\\\\\/2\\\\\\\/GJ5Ey2\\\\\\\/qHu1mc5VWfyRCpqyYuJxIb2iXbFprEZhBWYk\\\\\\\/05Uvh6HyPWzI8\\\\\\\/8yuPgXWNzbsdkmJ5L0eRW8D1N\\\\\\\/xGeVR2vBYwnAD1146sK0F7LmhJNZav8YJnnek2ncMv47MxhZKRiC7z330qObDqLER+Sjy4oHM3gCqg4FiqCkZ0+oE+O+htloPbwshoxrofAK5NYI4as2W0ilcXAAUNM5rtM\\\\\\\/\\\\\\\/Hx5TYO4bGo73VcF\\\\\\\/iAszQMtGU5iBPaIyyt\\\\\\\/3ubtnIGmgolKZ\\\\\\\/wyPH0NUHNBlyIpHT7ffIKL\\\u0022,\\\u0022iv\\\u0022:\\\u00228MVY\\\\\\\/IoJLAMPH3TR00eMtg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228bbbc6b1973f74ed95995052260d0580fc798459e606d18c705b7624a5938a06\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xSMiFiBDRZMsu9K4bAvKGCbL7\\\\\\\/UV67e16DSNGDgCwJha7udtQWK0WNE\\\\\\\/MDjX09BzFzVbVsfvo8oV1wkiAbUFKRPYM+jYBTAsq57+QTtiGX0\\\\\\\/cqsUGjcAV0DHoVLT27gCCDip3NdUmHfYHQ0xJmuHspJ\\\\\\\/Zd3CzX2P3fWKDRsPqfyiGFeqb\\\\\\\/yDHV+LgSCXUir4smiOArq4iqWwHAfwt8dq6LR5KdqQM5k06kUOFQcK+vXocFTNOutf1\\\\\\\/wArWpNLpgUFe1Pj0gVu7C6Jh1nSASDjbl\\\\\\\/wQkuAloH5lnfVt+edJZLikXL8CSSIHlwM\\\\\\\/Mq233S+Qwz1wdjEhyL0ySGjfMYIcBsr2KZc0btXy7ctxb4Hw4S0IciApvihfu7y38qJ79VeUJ\\\\\\\/oZyI657HPJAZ+VBwPUHjQtWrCwNuKJvZaO6DAlCjyioVAnQT4wCs0RNOz2W73AxjQEQG+LpK8Sx+X1yCAwa0ekkHeM1wHUz+4ss1ZKgjExFuXnK43zFTrcjt7ScPt\\\\\\\/oH4bQxiMzSUUocZNXJFJVv8xRE9ahLV+Enxcnp7Xn9mzWOPgnQd5zfH59UHkfDKIRw0QvngBtDo9sDrfzF1K8QkAR8AFZEL11nd+QQ3oB6XZzZRkviYDl9KpMuyOKQSTlU6o5\\\\\\\/IEoqwYx8rfn7GKF52oc81+\\\\\\\/F25heeWQheRcpbo6lE2d566qQITcSsJwjawFCp6R5\\\\\\\/m7ZXiF7MmyKc+HmWUFRlfmohnmeO8dFYH3Vty\\\\\\\/\\\\\\\/ywmxudPzrbgoBKvrDw8hfWaTcgPnZAdtazPyfPdtP9rvUUo9t1PGzynjjFk0j+7aLETqM9oMsbXvMKx038gsbOhzbYonroHMbbonYMGqGrNpRvsQ3j6EEoUaAOq9rALqVY9wkKS+raSrMTv9MK1IxFjwY9cVj5C5BAomZUDO89qKz6AkkqbL+h4HZZR64bM8zs1uUc1cib6lQOYLK22rbVuSgcaxrxy90obgCjGYOEZwX1GvnicJyTkeKOwgZY7o1VMEIUcri3ToWoXQx2xP2QHFQJf8Pq1yRv74FbD1ApDRDQxaT7Rxw1BVn4UIdZn4RIt0Q0OWRqaISlwkTVqhOl0GFg4QAYN0KmMCZxS9pOUn7nfxieg4wawZIxCo41\\\\\\\/9E0Zam4wmm5FZyXhS2JLuWDV40rkhWkEOAtHHeYRa8aauNfFhBtEIsllTwKRc4FyDqNo0GRRHL3CVPBvOqOUFg2qOnJpvauUivOGGvEuXz54W\\\\\\\/b+upzY8VOSs27bJMC7BGJDyanFG9gWoG4WJ5HF4McdcCIyHSsxE8mMxW9hG88j27i1wN3dKX+2yynanYqaRlUOqw6kh+JcYX\\\\\\\/zkerAwSak\\\\\\\/oQHz+cseIiI7uUnS5oPQVax4\\\\\\\/7QPbAT0oHxaObwbXq8lfrovrTGiUxpSzk79RWxLE+7KYfcnbxJHNVPAp2Ez4GjZnBasQabk3DhBPMx44eh1nxtv0RmtB4kFTy+E1SyyZc94NwP7qT55Js2lP4WDrCfIueafXJoz08o+MZacAaQxa51911yOFwiPoSuQ05HMyV9vfUHJVFIh4CYW4WfNJ49uQLX+AC5SUQhGMwKLOgs2\\\\\\\/G9X4dHAOk2I00U7uDywEZD8HbPVz13T5X3wU9dNcItGhMGA4+AFYeV3YaYQ+dVIXWbQSaPV6PHokon3n4IMh4XodBoFvloeh3nNhhQn\\\\\\\/tx1M+4yxP5Gpn1kz4IGRmTuNKy7LW5BDdID9cMe0dQPsrw5f2081doZl8KHEAFvnVkxEECFmNP593JSP56zezdTzYPbThZ4uoKuy6x7wNaPB4kTOOoZuBEZNpSrVfXBpKLB6b3oP01RHFiitviIDED3IaLNwGfALNEobyR+e\\\\\\\/1v0OiFODaGX0RS1qq+F8C71EOHknLX4Ld7RxK4QWSSju96vSgrhNBL2AH0fb0v7A9vhTYkGLjegSMN6jb5IchGs08vLwpUdHDrnL3yGsJI0iDEqRdFmbCSUIfF5aUcc0F2iqJneX2tWeOtRX4T3sU9Qp1qZng\\\\\\\/KMzyVF9jzd\\\\\\\/6hWPZrDsVYLJmt8SQcFfdCMQBIA5HNNYsKdemM777z5BXfGYCDeYK\\\\\\\/aumHJTL\\\\\\\/RsY0G5D17HIZuG6559tO9x7HZF7fAgynqnWE\\\\\\\/hQqkJ2V7n47cKsk14k3mYkte\\\\\\\/R4S12DALfxb+wjoepaIUv\\\\\\\/SDVASdwkLeUPXI7BitGE\\\\\\\/kUptJLD4V+Ym8juE4pMHk1wyGqhXgTVqRfyHBwCV4NEcqaci2XgNNd4YuUDR3zh6PMdXp9nklqoRTIQ4vsBt9O5m6mYmPOF69Sx1Z8VcqxL0blbbNbeCqkqxGxH6EaiwAlzCa2fLRMzKyJu5fDXaUQATbNVapF7fEUn5QHJQhpYxaIyiayS5Yo3\\\\\\\/Ijfn\\\\\\\/c3+d0sRUkXwdS4UZlUFEkT7XjVaTFL16+FUdTOKy4mExUdwt+d\\\\\\\/FnXk0\\\\\\\/genYJGWmf\\\\\\\/KSufb8QTn39m8wdXBM\\\\\\\/BrXYijlgLkxhIECbkEei+K7KLKSRPGOxCl76Q1Jz9b2fvncPp\\\\\\\/6DrR0BBGpiezZLKSNnlRYmdOHlfoe16g6Ho1i0k+1AY985zTsBldc8U3BXXutj58id4MGB1ym3nbmeWF0ERrO+Vus0zCPxONAvUGzFQoSVTFGqassOSq7Avf98vM1IXhqmtBMsEZpFeQDC0H+NA77CZQRWqLeGufscdJKJTAGsabg6PcLEicx03HNDeC5G6Nr+T9FET5IaF8sX4GsIw81j6R0drf1AvDjRvYq0k81Zquhr5DGc9JZAdXgB7pSD+r05qB1LmV2UJ0KrnFqijVLIlgOck+W8whi\\\\\\\/Ln6KCv\\\\\\\/sHq0Yf1TPgnGHMvysIqiZT\\\\\\\/mjLLXl\\\\\\\/BJkOmj41MPTbQaHDZNvLtuCw\\\\\\\/KtluDvFkFVweg++fA4yva\\\\\\\/cajE1Qe3ZR7AObRLgx+FVokB+p\\\\\\\/321kBidTD7W9WgVvQaVu3KvswCJYOT2Bz0KJF13fN0XlVtKjirn6vgra8OLuqrIhtt3bQDpPFjlc6b5jzeW4y7oCRk5HYAhw42iwwhy6oyvM9elOgtNq3wBfFkkvtWlGMSrJv1VBXYzCQQfuoow+5VnQ4jKzOB0Mof3kDgJJPyNlqm7yjLaa72hd4sqPYdzzQ+8JUP+EgQa03Cfi1kSNJpkKzyDPZPThRBybbfesPbsrweTuZ68rUd\\\\\\\/Pk+1elrZ3A34VStv00CAy2YLCrAK\\\\\\\/gmVRZLqZA6FXwg7obQqHm0NnVZUbeLp\\\\\\\/TKJXYab5oGNR2\\\\\\\/P+3FuB7+X2yshUX0O\\\\\\\/JYSoOT14m87SmYORpcMDTXyz12aKHDWeqqPjbdbpzIGp7wVXUO5Pz3vL4mAecKbz5PS8EqL4wUYkbW01cvOoJbsZbUST9Q32Tuei3sPkKb08nnYI1wfXP1izwuPRgOtUHROwte3ns6CvMv5wj1y\\\\\\\/MXe+S6r7A6jNjtf57\\\\\\\/mQ49cQyRwKgmJR8bhuLJyLBf2oqHwyxiVIHrziE9X5yAD9QtY\\\\\\\/hUiDUNd2TaWVV+PR7lzkhcNZIeAHKZUpBhIhS+8O26oU70YdazhT+3RhM4\\\\\\\/jVXvlS2EvXwMaScRM7mflpWck4fLmJ7Y+bLyGDpEVDQku+18BslBh669TVx9Zc6fonAaC1+OTWjlBT97mSdPmyZOm9aYO1C3uq5wmEyndhmPgG8Ooo6O9+elA0P+UTlw6xH505JxSGWBrn4M2xC0ciqxt17fDiYLmXdG3HVDh+gns2eeZHI43YX+1AI51kU5Y26eBU+WGfgIiAbkZtfjYahYKyUBI3ofWIi6Zu4lzNkYjnzu6co\\\\\\\/IWDu\\\\\\\/5+bL4q6499QC8Q8uN6MU2FeZKFA93pNG02gDVEmWSToT2ew2qi9v0B0wpyDtFI8czih6zAtVZqRZT5OoUHoTPoUdQH8zf+0DyvGpme7TIg31zEqXExcCBg2m39o9wHBlJ6y0rsxdiA0ooXRamrZqh17S3JYB4D9PJFL7Z1ITC0l69mtQel0MuL9RX21Yelrc9\\\\\\\/FRtJplSHd3e9F+ZKrCwIqdbZV0Jjv5Z0fpKk\\\\\\\/T6TVlck+objhK\\\\\\\/D\\\\\\\/\\\\\\\/9NtykKJ0uHCzywfRoljb1iVOhGFSL2FZT7CUJjHbJI4PVEFfsvQih1O1uuGbCGeXMv3QN8J8hUtZqxWDls1gLRDcbU+g9YS1gI+GA67ONX3CrCgX5K4dOYYsqY931ZlxLlygoyKJm0uR9SCxRMHxAnKbz63gZRU6CoWTdRdB0RK+hrsvdWR2FiKLDEuk8YKuhbK4muCIvi0ws+tBEe3FQDW8ZlcXjiU2xA6WkT1Xr7vk8Cb8fGFtkak5cRZws5NtXrRD3FGcgifwyPAp5JHXQL5Bs75gN5y0vNRQ0UVkZptvqQf\\\\\\\/p33zrScP365E5W4tbuL0kGuXor\\\\\\\/nef3n2r9RhLFNYXR4XOfxVvsbF\\\\\\\/rtZs7Vs0Ll3Fv8k0zsU8U4Ok5+vKxZuJpKCTjcAfBLy3NHT7Ac9wKDW\\\\\\\/nCjHKaHglKnk+V4ueERhWXjOS4Puo4RXpMB+BeTJ8ZEuPCjyfmrXyRFRp6MWbynrPiMZAbVI7MSCmiPf2LsfJ5GmSkZl0e91vo3GNyMspjDdU2Y7mUvsD\\\\\\\/bOfrzPEf3TYpvZ+qZQVrHbFx+KKdfdpceJdngbX6S3jOAdEHQiQC9x4dw7oAFLIpLJS2lSZhozfuxzOoCjocyf5VV\\\\\\\/deZSKc1IQs5fjEGjiQJr52lRi0a1B6v1QR\\\\\\\/7ej73\\\\\\\/8frOKskuBy\\\\\\\/4gFp6t4znYhGZK4siVni59Z0QtFtuOd4B8OIUIcom3Ij0dw3aoXWYSfM1DPlIR\\\\\\\/JkuDijQfF1p7+X1yY+PYZ6Uv93724Fy2\\\\\\\/B9n7dXMwRhJAxRjuxJOOCJ9ByL9Qrva1QJMaYXcXk5Lyh5k88kqnh606hYnGRiBozbKG2tu9pSqhwt+QjBdfdTMS0HSs4m5LYUyF7BcygaRHIewvOpcvMfLI3QMB8fUxQ6CxOHHtsl5uPmOWZTCfttufpf3CzFqlYzsXDmvNmr7Si+9+Q7MiZBuoxpv4jdCnBlMbjcjsEDTHYTLp9hrb07eOhCJKlWMf9EtgMRg3VJMAbpemvhcRbdSyXOqpiRLvch5uoKRyIpxQ73xk+838w8Pgt7xizpVCSdSrXLP8QQfakgOWLc9fTY6RJN+L3idAx3kzxwsUtHB1ytih1qbu1\\\\\\\/FCYJ6Eqs4mULG8QvmFNhXxZ5Dr3g81ZY4qKl0budWygMdGKPzFms3ug1tCfyIH8Kc+oGeyOt1BFS5SUxMXosRpeZV5vLf0q5D8g55imo9pw8ORnmSV+cfZzzpO8X3tS0pRCIZtnusjmVIcEuRKRnPI2erV8SFlN9KNS7uFX8DWgN1kPj3TX6Oh\\\\\\\/Ow47nDwd0nbqsRQ9VTw63S27LocZliVGonv6Gl98udC+E9GbdNrHffo1QPQWHXMt4W44gSoUNvodR+E2spC0lLSUIcTafo5\\\\\\\/Yg\\\\\\\/mxAlRoY+QHadSrO1iU5goCZeoFO7Uoyle0fIrgrB0iUVi2sE+iiCBqT2\\\\\\\/FwrMfZaJSwCLWgqdvnkqOhl8QSC1sTZNhwl1DgYuk1\\\\\\\/bCyJ0SiKY7J3LqvSLvNA4PNee8Rq1+x24zFeS3L92mHTrOl5aSy4ILTGtKg5CmEZNGwGDt0PiWajbfvg1P088xZ5v9Pg5oSGIm8DMl4Lw7ZemxA8NnIPgE4GDcjUwhx52vMLvF7oZ77A8LMjAAVhSVmxa2CcwnLgVzeFGBzXq9kcKxksncbvmL4j8X6S1SnRT5FLZsjinoFmRCZHz\\\\\\\/iKGSsnuHZtTbiGcFYFv0R6eke0h2ZYXme4+NfbwJKHCEiG2o5Bg7SAFYln\\\\\\\/gCMkBvwnoKxV61hgspbPhrOHwJ7r5E8dXnuo6dOqUiQQmTbkAC2dzg7WuB++BwTy7fFs6IOu5UlQ95T6ixC\\\\\\\/gM\\\\\\\/j+C\\\\\\\/3rpFlq\\\\\\\/\\\\\\\/jhDsu+yEoD68EcS8rrkPuckU9FBryHwUuNJ8Vdu6+pTRTEqoiTYwQd9oitdsIpVytlrmmmkQrwz\\\\\\\/7Vg3Koq20\\\\\\\/F3B+dt4iFdke1qsCGSN+HY2dSqA7KK0iqErC8e2MIBIQim0k5KH6HXyxRwW0eeiw9Ch1ncvVenKPrZv78j9UtOcfcXjGULkSglcjQg6xhi0ciOmK\\\\\\\/Z1fm5bh8C7lfGPSLQ0MqS9hYiyhIx5FMi0bYWKdZOrwUWFB40IbNPJyLbTEmrMddgxYUdHiSLjjq227yX8ryn4MBzrVkDYEyy1moUXiSRFsJCXwh5Tt8HOh3MuuIk+BH9v9XV18mIEvMxzaZf8z1ezEqmmqJoQoleFBunRuinXttdv2kM0ei\\\\\\\/kW5mmumiBsP0G6yTqJW30h37BGzXTJT8yIUQcEWEDysVWU6I2MIRgeeQFi0mVBjcnA3z\\\\\\\/yWW4R6NtIriC\\\\\\\/ZT8wmju7DG6YQQIHSWJPo3BIunGZAOttbjdIaWESoi5UchSApWMZkYHT8MMBdtQ\\\\\\\/0Gep2mmV+vJFdfd26VbQcf5YaniiW+GhbfGzGzXmYWe04Ot+hnq4Txeh3Ng0yx\\\\\\\/FEIWDWxWxQIMB2z39iHlMJfLc8\\\\\\\/wLm\\\\\\\/L7otbxn\\\\\\\/PRxkKkdWC2vQq1X9ZesF+Q5JggMXSA79rPH4c0AjpAWoBE+OtDNDq7QNE30+XEyhd2RKd2ddpyPBLFHWiJgjKIEYYu+yLeF4Rb\\\\\\\/0nACl\\\\\\\/fXfFiJfnoKNRyAPGCBoovRIvllpeOh7GEN8dXhh22di2HAkabKZSKv1\\\\\\\/UohbChpDXAkpLKJJZ5LYb0e6gkbI6KpQfMCXLXEdx0l1HfkK83Rh7YRO7N1CUFe+j7ZHOknHXETYmtHDfEuzYudyDm944+0fJJvyAdUP60csIFN8zAn1sOul78\\\\\\\/IdPvvQ5hcEk9wiOLorWBs8jGVopYEbdFJ0sX1SQNy0yHtHZzdA8GT0DVTIKbbOqPXQtkh3hjD5fwcS\\\\\\\/eFtX+eHCQP4PjnCUXXum6tNSoLuU09Disfz99tBrhBRlwwrmlqhbHjBdiShJfyrRfb9FpupJbJOJel1vWh\\\\\\\/Teca8L9NYYhTUgTipoJHwBNATI3u1nGxb6GXi5vktwL7laLsif+l\\\\\\\/\\\\\\\/smerXTTb0YEWajN8W8S5JV6\\\\\\\/7grfng43G\\\\\\\/LABZn+GL9dGDHUFZsJ2kzmeoaj4uL4ZUG+2j8ohywQQAcA1jtlsUSsN3JCdXGWGac6c4cvrAkawaDFHbNjuX0SIMjHjVN8Und+0rLz2Y9l3lADTwpZrq59fhkA5Ydb5mO1y8tVsc4nhp1y7Uw1+jT5vfniiL0Mv3c+LysnD6F+7wjZl+jfImgb9UUqgE55RIsAXm43rNa+\\\\\\\/+tLLbIhXcNgBThUHxbWVxEVHp+dboT7xMnq8qNPQMYe6qjd5A8h+uSjtHTzN8s65QGtXM5W6DhrOlJi9Eep8\\\\\\\/k0qce8jMA\\\\\\\/ZFS0PxfWzi9aDWcBlcx0hB5eGS8fiOig5RicGMbdcub0ok0qoh2U5jqSARPKbJj\\\\\\\/1UpyfuAexWw5QkIv9LaAo7A2gJKEQgPK3zfRf7aXd89u3ju\\\\\\\/CFDZ4iD8I5RhmHtlUbSxbB1Akn9P+zm57q8KyVv01CLiMmYUbVEheNto3VjXEN8QWqhqH8sD15RVrteRnK\\\\\\\/TULw74cfFvajw+p3I79X8RJegqylWwb0uKzK13rEi\\\\\\\/zBX86+H8e4nEtZXkxQW1f3i1PGXgijwgpD8UQy\\\\\\\/5bC+UaJt3vfdCc0XaAkitZGjLsjTpaGxKheQ8hvmNk0No+Q8xAWu21RUpnqDUlQT4Dznfm+888qLZo9pISed2kQnoSvaVpx1OxV+qKktzeALCESHjL3YFQJVjN0btVdjDV1pKBNi3h0LXOY4ztWpHbxgBQHqbthrNLfF6uXhp2BSGK22pkxZ5onax1btVWIqAY3Qa+e\\\\\\\/tFNea5cZrGdpNrcKLQH3\\\\\\\/HtJNXCvrryOeCI4iDkYfYu3y6Q9KntFHXXnhpDfPZ2qPXiE0nxmuKApUOfu2wgbeH6a27N5pRzFsrcttgjLPe\\\\\\\/pkJkaXckLzq48+yK0n4eV\\\\\\\/jXAYXm7pTAlroA8PE6sBhz0aTAdkiTWdrHeZNOFqf9xvWNx0tmMancWC1aZK0IcM+9luxuNKpw+uhxPRf0rsD5YWCkflNIqiH1Y\\\\\\\/zkIOcf1Lqul2wGqXGBuy8QAEAkojjRGQq45ddGMRQSLj81SgEkEOngZxs9+w08LXilHDpKVDhPk76lqjlIfd8Tg284BKAqRUfjDPwZQciiJJE0LKQT4IL\\\\\\\/PWqWuMitarRJZCa3sOdj4ylzFnwtqCRCszfUMKYpPaXtImkRIY9r8eBg9sG3LuYC7YIF4wxx2wbuc24cUySUiKzsYVYNn40LlM7JlRKszdCuI1Bd5aEWv3u+Qs1CbKZHubsvFhCPRyC1y4UBbI2AWgdiDlYxBrZIV+sWTg7JPMJrijPFJ3q5a6+gm7zx3Kya0AAocZl7h6raOBO0S\\\\\\\/QnSSO9o2AlmjEVKAMIxZpOb4rh0jAwGBSGD9yO2ywJ3+abtY\\\\\\\/V7KrRUQ\\\\\\\/o\\\\\\\/4RtTMLU4Xp644ZODON8T6YsfFYjbHhVEaNkdCxJn95qO9KjN2Vy0nR\\\\\\\/Pcd4pAUYr1X0JYk5J9xYFWXc\\\\\\\/16+VczegTxP\\\\\\\/e6L\\\\\\\/t5MDVHzsvzsUdhO9GYaLfolefSQSTgmGx\\\\\\\/AoSLvFXA8Cq6HdSk\\\\\\\/cNB\\\\\\\/+VfN8dzVvSAH0L0pBKRJPQ0379Kof+aQqSMD0CUB9Xu9\\\\\\\/X0K2FCSZdgCm9cMB0H6fRbKEI6H6YSQ9wV\\\\\\\/41r+h\\\\\\\/9DOvUb\\\\\\\/WABsyFh8PxUMif7iAZQB4MCbknf3RP56GLro1tjASvSkA1sEwHdpY7o2W3V4iTxFr3DDZqoflnIhluOvAKrfC07SdOuZwQtEnaHkVfFs2H3e2u5fuT1GQdapz6NEq7zQ5eWiltydcLc9KYoi84nwP1PZXt0P0BmP5pOK5XM5IMuA9wke0FizSWm\\\\\\\/nGxw8jnb1EaeyVCQPDoMZ30MyjUXX4iICrCzz+HM+GDlk3abT1\\\\\\\/+TKWYFVU0cTl\\\\\\\/84PuadD6\\\\\\\/zU3kEyXenz\\\\\\\/TE5AD8t5PDnSZJ8Z6hpXU4iDboUw78SvxECbtLAxHAQOvyw3CWXPwtjKenAHJuVi87udaXgMvcoOr9CKwM9cwtjAWqkiENMw1V1AZP1GdipyzGS0R0byL6aIZzCPUhoeue4nGHXuRYDJybVl5rFAAtO25spDNvCRcKbVtAEVsYap59OMr\\\\\\\/SPA0QcxnSGK7KLRh5hAa6JTrYS8X2j1hD3fJIBPcE4Po9aRHOLxui\\\\\\\/fwPw0y07tzOzXjQjKlfmyeq8hQv4Q0IsUJv8tiYu06LnA64nl8gm2mbjtRChDwP1HRbA5e1dJg40MKOEo\\\\\\\/+\\\\\\\/kaVWHDSintivDZMXiR7AzErtZGKK0QOzCVbioXZwiFZq1oONN6XBYOmshLtsChtyjwqtoey\\\\\\\/mbn\\\\\\\/c39s4lSMX5smCbD+xJsvLakXim7TzySZAPdWdco9G5NcBDeGyNan\\\\\\\/Lu\\\\\\\/xGnisWk+XyUetLNJ3u0tjZ33yASUEVfIIE4g2rQMfMpmin5IlGSxWWnJAlgLc2z+MRhc2ju2DO\\\\\\\/jcbC5YTMvC5b+zkx3YO9kEKTpIjQ+DG2VPnzu+mXvUIAjYthCz9vwQ0JsxcnzfhJJ8XCsYGRTo7LWwCq907shV5oOuuaPtnS5hPO\\\\\\\/FN1ugbIy9zqDu6Jf4BeerQLQuy5xgddA2SwTQnhzI2mZnAsfb2BFTxV99o4KJGnCNLUPBEdJun3qUDZfFmEnZTEJdeLAaZZJ39EenWsKzPTc6PxfMx7beh5Usb5VlcD3oI3pOflD1nSnIdYMXJTEkoSb4LHHpENP8Epu7T7obu+dn\\\\\\\/NHRcfNibp8kkGf68Rsh6vRofTdR0wICK3KDZNu7gErtvv0wCjAY\\\\\\\/EqQSkKEW+NxveK12s1nkbIcrQ\\\\\\\/gdLeuGYNNUzWhuFjXtZVaN7nuFJ267vOAoMkSpH8KG5ec87ou0We2vKHQerRImqHl4yzyw4OxtjM7\\\\\\\/W5x8OqHgb+11fRX4FJBKnRjoSmZyNx2iLZGEYEe0A1591GfAijQrkL+OMTT5kLvjtANnjuCXyegjfNgo+dmY5KpqCaMjnj+JNS5r8sLt8L+ekUTfz9Wbae68vT2EmA9yYvjTcesLJNRSeey8HCKbW9vxJxhBJeusxEQMLGuAxOiQ8EjtyptYgO2DfyJtDhaCQzJcETUQweSbpTSVxCSXZxZ8hbz\\\\\\\/CdmonYAgXoFzfmXeMqLscZFxE0iv+0SSztKbOzS5dYbo0fTvlMya2MBa1uEG17JfJrCls\\\\\\\/ft2q0VOXLZP56sNMEE3nYAWYS4HJj9aKE\\\\\\\/MjRiNFodKEHLI5cg2VQfU37gt5wAtIBSjOHG8cpn0tSBhu0WagWk\\\\\\\/H2gf1wh4ztZZPPYQHh4vk2Ni\\\\\\\/ZCnM6Dz2iX6LScn64bcTyjR\\\\\\\/ZNKEIpgtpSwkb6AUiXnOwi61HPJWAOtyzq9BZZPMbTa+CE4qTF6pZd+czVPFBeSFE2NSmmY7M\\\\\\\/F9QH4J9p3gdiMGSLWnHGXZ6T8Yi8PeTPfrLPyp6XojqhqkoeSFQtbrD4ZKPrbWEyqC6bgVUdKadZTJWluT\\\\\\\/i5RH\\\\\\\/4amZb9IH3JM5noZe5+ohz+iMNgEPU1s9onog5\\\\\\\/Vuzrw9nSMXSgKZAaJQwzHebW+r0HXGumvSE\\\\\\\/VCxgPNX7yIgIoaB+mEG9GAFs3rJPxcq0JzR+XYIZXW1V4p4rQiYJWVShTsKzGzToSYseKIM6J0Rxo3py4LrYmUFrhB6jlPDsvP3+KC7B2RgYj6f0tQwMyog\\\\\\\/98\\\\\\\/bG6uZNx4Bzu5W790MVZDsPb5CWfn5\\\\\\\/r1ZbJdaQMEcHxRG2AILUw+vKe01vZ9+HGZjXLPJFgQ9BY8twxp1iA\\\\\\\/UUlmmINYtyB\\\\\\\/ytDrvanbEZEzwSYrrQo172yu3V4nX4y7z7g2mz3SeBrRGPvDfJiunjNEzhNn3fCtO20PE9IdpxZ\\\\\\\/\\\\\\\/Pz\\\\\\\/O8MWhkq+YuVqfV\\\\\\\/fUTFCBZbJkUVjDpGK\\\\\\\/hVrlqmP3jTLG0GSkPqbhblWFjeLSZsZmZSM9BR8x5tWPRsoG8MtP4xfw0OX6oFAy355\\\\\\\/NDAeEXqs3IfmtyF\\\\\\\/ecczkJv\\\\\\\/t7BXpUt51V+5wSA4mB19cf6w6yeav3jj9vhar2e4Ex8QTjJHlkeLRthm+mAtsUgZLlVAm9DibHNa+orKHDb40owGxiXU9VFFLr97ZUae9dLVd8fVc5k2p1EPZedSIXAzvqkQgeScjA2BqNwq2YsIOTyaMsJPQoRiQh2zjPQH3ROrLpaFGQQfG4exwXwg1w60UzhnZY80U2sEb88tRmImwRyt0vL63zlQASSdMKnQa9GGxw4MfxuIkrHjUeeQ7A4p7mWjhWO\\\\\\\/3qkOO9ok8GJvlBrFVW84wffVd8Yh8+VnNvwsH+Mq1cSrDLEk73tBscOJs\\\\\\\/5+3NDoqPuJKEADTCt38Vg3Z0UFnGf\\\\\\\/qkCts3xoSq+ETo5tO9T6B+TCTS4AMOqx3KwKMTrnlsb11pUfAg7l29uHnLRqFyu7AYkQ73rh7ZS0yaWf7r+xd87ntBgxy1FlLzfph3reIlRZBhwCgGx7D+eUBC3s0ZdOCclZq8xAayrYeqGPLR2RamOCxigYpr+k+gNemLkxVw7NjFs9MTgWFuI+Hs\\\\\\\/0Gsrz7lRcZ50oY\\\\\\\/PfvPwJIJ9hkGKXufOVLUz+i43OgCMWHC1h\\\\\\\/sjRx2IK902\\\\\\\/gaRlSQ0SUlWZkOyCtKrZmUSfpNszUQpHTp4YUOTASJQTzf3w\\\\\\\/laDItv2cgQ4HOJ2STcvYLys\\\\\\\/KEVklRX9Bht2jvk4xNSvPFYUWWVbVS6bisP83PBiecgfym48kKRb4x17SJyxTk7MQegoAGr20y6b4WD7X12IuxURiFA30hS765lcInR98sOWaA0VPNq0wx51VQZwXGc9EO8DC4mV\\\\\\\/sqzmpfNlQOwiiqh9A0r1QEZTiFp5CUZ8VSPK\\\\\\\/MHDcCsCBHVEXS++KRZtsrpLVzSHxkBUGTxbvCN4yjW8WqaMthtgCcraPuvXLWRBCU0Z4vaJGM2wB4H+AhmZYtrB3K5AkerdU70hicVDc2EpcqCe1SHYn7o6Fx8JVyfyiobBhqBmcvV64uC84lK6lhG0vOBnhpJXevc6jqg3x0+Y67TtzrMTW0VJlbyXTn8zR1f0XWm7bzGJv2G2GG\\\\\\\/Dak4fTwCoDqGty88NN7lNQ\\\\\\\/kP3AstSPWe25u0dAoOkj2MeDToC267vlxb78xEIPKHuieIonUuzxX+svoZeNIfNrtVzo5ZqlJodOstDN8Qz3plCBtVPCn7kTcWjAKnE2yu8dMY2bKO6N1lz+hcGzG6trI\\\\\\\/G75ygHBWWDJRgZSEKPJnn9oXXLbGowcSwTDlJ78WBKwZQLvmFLqJbYhAz78Nq19bH9YSjmbkANXMLb0NCLgOGCbxWLMIObW5JsrDERi0m\\\\\\\/6OCiRJOIGMxVwogSnoItw1FE4bYJXIm+jnro\\\\\\\/u\\\\\\\/P0hmnkenBbJf9OFXC8g5b7cogsoFXUlASfTG7SQlpbv0QQc+6NvQFXfWyi\\\\\\\/l5csFOLB4TyOemBA49SXC7x9io3p2l7xLUaDIjoKQnvF2jwmw7ZYa5Qn9w+BJlF1tQHdmFA+u5\\\\\\\/C4tH9Eb7Yo0yPxyTL0Rm93JbojWoQWC28JbFceNS53vRBhqaOOFiTSqNfzO2gU2lNxCad5bA5+K+9VE4Z4IK8xVbdCMl1B1FvnJKyPJ4uFQp\\\\\\\/dMM2T+qb4r6WhoWH5jXXCxuNxNz2iC6RoJy57pwy35TGccV7ZlUUZ9lyne1\\\\\\\/gE\\\\\\\/Y9aBH2Va5kL44jQ99JxzlVMOb4qjFK7hsKO\\\\\\\/GGMxDV2r5PqjCKwEtO52nB+7Pop\\\\\\\/YWgp04IetiiYi+WORZOM6IgZTCsBsLJpeoh3hPWRFgQ9e+SfCGxUvbVBASjqwDkw0oEBWOGxCol\\\\\\\/xAp\\\\\\\/eDLBa9nz4TC4UMGfzE75V3pc1H3xhsw76rblvtpmld+ddLc6qZFxtZZqqDjNIx3pVnrxAvWUANSbh6f10Oj3U76XERBXl9hB0eRv1WdYXGXozGiwD8EswFqknUIhj0jC\\\\\\\/kQMpWo8NNgoCP8TyJs\\\\\\\/IlYNUVTk1FCafij2y\\\\\\\/O6VX+FjhTPm8WibdWwbgdSjjweu7eB9tl0G10f+I0\\\\\\\/Ji7V9\\\\\\\/5cIE2vFRI9l8mSXi5Av3CymCsY1+XASnDkr7AAc\\\\\\\/mjRcoiUsYx9bLO4uDZ47nq06Xu4lZBl6WO\\\\\\\/gv1iVIXxaig2Ay1xk1zRQ7jgKI5vokbRGXtBVPi6khtrhMD4gZAssX2CKH1OGQZf6vBUZNmSVpHq7L5VTVL4YsVDTIg1YIzKSWkf2sRCBthPZSnSav5JLOOqxkAePTzlJ023+65KtbyP+Yd0KShWmefOnjgNEhFmKV8U7zsTvmXaVL\\\\\\\/MMsvynrJWLhROV\\\\\\\/ByYIuWT5KQEa+DohC0vDW6ah3jEQ7uURopL\\\\\\\/j6wA\\\\\\\/3SidHxQnLuIN6YsH46jEDlcSQb91pXdOVOicY+x7rSFWcBK7GZvTi+4WQ5qnQGhCaVDSeg6zUJu\\\\\\\/1Cpbbocshphm+Hy3Pk+J8Xt7t3hdMyRACY6f28Mah06JCQyTYwCmy22h4Qg321fuQuSFyf1AAoJSDmBwZude+zO9Fx8k+99KNsgtBZhcpu+G7ZioJR0PU9QNV+QprAlt3\\\\\\\/nxpuqrgQE7DjmXNx54Li3m9tppCN352VB8RXDpAt8jD8aNY1jRvuqGvCGMMSZVF+QalKwFXMFxh+l360R0MN74\\\\\\\/890U8JS4KZNzy3dzIFYpA\\\\\\\/\\\\\\\/xemOqlt5TU4XI+DJR8a8Tqx1hQBD2wtSeRGlb16n6KWKb11lvfKIVdbEO6iRVdNtZ4mcMWClDA4ref4zUk85izmKuJP+IlU5W9AXKrSmcWtNZ4xg\\\\\\\/LUG5ztp9uA6Hfe8lNTwvEU1Gl0QcfqOJxVmYeWlSnbFQR5h3xGDzsebPUabIt3LMvKaVbWPIKNadQpzlw1VB+jtfgN7zRO6+1m+k49f+Ryy5i7CZWPgs+yieKM9A0qXHvou4wl+WDOVEFZ4Qea\\\\\\\/6Nfj3H5Okt5R7dd7pRfJZ1WRV7p8DA\\\\\\\/UwxQSu61m63S+3XjrE1vqs9+rF+zeVqOeFlCKZ+4fMbHQDdN8baHpQ7k\\\\\\\/Jbwuc1Je9o5eV3xSS1jQDKecmRE8+dGASQ2TQUyK56bW2uUpviUHGnVH2vT1XZUjv2A4ntY3Y11yoTrIEoz4G733FOo4Ten3z7k4jwYIKjCqKQ8QFItRr\\\\\\\/qqa\\\\\\\/BkRyndUxRhkWzHPribDXY74VrJ3dDAJzaqwLkKN\\\\\\\/K\\\\\\\/lY1mmj3515MQNMT8nkyE59z9ibyFr9SzcNzXvsRjGcYd\\\\\\\/C7Z2cCoe0iwh+nt1nn1gdvbC97qfgtdazwS8aaBmde8NfpnHhmvEVsIpxsZa73Eax4iGI5XLuz4VeOumMeBhcwruhoEhn8dL66m0El8a3f4Sc4OFwHPP2X5\\\\\\\/+XcDQhI\\\\\\\/KZbfCoKScjMtz5uqR102MKz8UWSHCjZlvsfL6xXv2l6FiMkPx7iZITGhFmM4SLxKET2xT4q10w258LE1xCrbhDjg2WL+6PA3G9d9Y4x+CZeunraznqd8ZAXUv0631oa2q3b1m\\\\\\\/ma56StuPP2uzvZQb907SlncBPdpfTJuYKuScx4FU3Sr0qgSIiwlqw00ZiUSGHlszALqG7YFLRTb7K4+Szod1TZXeoH\\\\\\\/U0kejokjEJ+S5lO3QvAUHDA4lvpOTuZFf5pKTOd9Orv5M7Gy\\\\\\\/V+GxBG3OHlqsyWBUo586adsVIjaKtYMadtp7HwXYSSt+7qqjnPXMJpuiISD\\\\\\\/pC1jTmVsxzgbNAU05T6b6A1FS1vd2MZS+XQTL\\\\\\\/+8AUz9QjafpFqTO\\\\\\\/RLhT2qtXeORugHXf+JSD0dFw\\\\\\\/ipsozw3QIuCA8\\\\\\\/eP43M2ZSC8jjteay4qzhd\\\\\\\/YvowyxA4nS6B1qWry3WD4jW\\\\\\\/Obzg8u\\\\\\\/t\\\\\\\/ZAoPXRcqQhLuSzZgvVLSlq7yauvGorFmZruEbtDw3XcxHFNt2htNneutiimQBYrIpkmtl9X8jqd7AiBgCOYN2VfSa+OQdF5yDfPYYbIqAhtm\\\\\\\/yKzdnC+XITaotf7\\\\\\\/tpX8avnZnB3MgBZGc9\\\\\\\/8fmabK+O7jvcbnyo2gSwgDlxW2c9EjttTnHkYBKSM9A4n34MC9l1MpDiA6g5npVUj2wZ00hFImHLDiC24yjfMn1AEwAZPvdf4qW0jFyF\\\\\\\/Z2P1+Dz8BNtuI45LaGiZExecvUIgde+sWmblfZt\\\\\\\/Pic7GFqLvR+sFqZeMLsfVIVYsxyCODvPYUmiWw+bbDawC0sgDIw5evoIAJ2H9W8NGyRn+DxBBwgEhfDWvFZr48emLtbbNIOaBzn0LPnv1blVVjQkGfdYK1LDqRv0hoReuYn2twc5N5DvnOP0d7VRvrZ9zBZZSS9PM\\\\\\\/3loP0P\\\\\\\/TucrqF7A0jzQZUwrdYNoE3dba8MjUvw26ToC5EK9W3IKVjfGDuGnd1+adCnLf8nMAIz+Xe9syNiqAz\\\\\\\/a1zpaIpuYbBCUUoccYC0ccElXkv2bLfPXwiuFkSjfBvObUBq66\\\\\\\/COSZOeMHIsWiROQ4PH9xWdLFN776UHRp4hOUeGBGAGcSZANQxmIGgeMYkC0C8vx\\\\\\\/hubgqMn2TVkNTF5+Q3Ng1f8h74v4nTEWem5Rehk90+9ymLgoncpJAb0VKJySTqPX3g\\\\\\\/adeMGm\\\\\\\/x+B0aJMDqiTKr9pem8Pm5Si9AX2Pklt7bismh4Yv0mxYRkTvqr5JBU7OwpJiODMXR1\\\\\\\/q2re8bBH2woRipHffizHL2wspnN5IZ\\\\\\\/s3oGWrPAVk4ZzhgAdG6exIcLO6tVjf0DA9vRzVOm3HNVkDVZCej5x0q1+yBRMMY1TH6aC1fgOVf9Jn8GG7SmqUo59NBWVEf3TFUrrb3AGlC9FN59uoN0OmIlDa7BLJLJpeBrTU\\\\\\\/86GKeyY43mYtP3lV8IEoPg0yShaygQrM3FCb2Sr0P9duWV022SLusjPoU4QqhUgfIu+HiRABj\\\\\\\/Ll5C1FKmAB8OKWhVQfDAOK248z293dedYPvUyEWqxARVJY9JlOuOFwANuTTfnJ2kLIT0fCtFgKkjiKOkfs4I8ygQgzGRooGF89W2CAyKc\\\\\\\/u9vpkBlOfCQLGaIuAy9Lk1vKYFIdLayY2DhOonL5Yh80UCjEBwzjhsuWRA3Iy3JXyc6ldodD3+dpW4+RMoovuPErJv64hnH6iuEalqAwuUPSXP2mB5otP4WKwhGe+RqVPiL+u3rWhH4DVn3SPJOBZ7Uq6t8pInZ2shGlL9N+3NutWf5lbLJjRPprPzke7Ml3R4TbafyL8etV7GKVJtU+uTbTZynhQ53C4aDo\\\\\\\/TNXp0NK1C54DkDWejtdCrI8KcTFKCJ+1Derv761l7boQh7h6ng5nOBiAr58+QvEEdGm4m3GGchM8ZhBWmu6OFKWe1nOAYao+7+jsUJPEUG25Y2n\\\\\\\/CTEeN+ZZrehP46cE8Kc+zE2xEROoDzdo1z3DpggF8R6rRqLi0TSpFIHOJG8qzgWTxCJT9YZe9I9QnA0orNgvKDy1Ci0TSUqfa2FmLGMEx6oRJMlk8fkilYbGImWchF6ZOacRs1GJd3R4aHDsSdJM3nTdi224tSK6oPuSVNQnb+XK1kVFszOeQU4CaaPvPk+Rb1XVPiK5Qh9zvl6sPsVD579Q++4SOO1a\\\\\\\/3lcTtTQKc076d81RDpalHyy46K4tOegcmeaMR2ElicS+yA2sLPgYyoqBLlGOqnf+b+15vngTKN+7NDADLbgYdUhU+Ma5YuqvSn2l7\\\\\\\/tuJXrYbpxKJxcIqoXfkgF4r23bLvMFWCkalsITOo2xwNFRBiADCywGVa3y4fkUsf7298iBcCR+772T15FeGw4kvgvYVb5erZC2eHwutA1omRhVNba1FuSLJzcG+H9HvIhvbuPkhV64uzuBsaT1xmlK0n\\\\\\\/iXvmwN8\\\\\\\/xmTwhgFcCkGTgYaoVbVe161AVfFBiXDl2hyTA6vSAykzoDG8EmC3R3gzqRcUHxeIPNouEQJFuIgDPeQ0AW87\\\\\\\/hUH3Y2QBTUSwPa5UiM\\\\\\\/TdBccT4wpOzoznAujEYuvDqlZo5lXvShGLcYXXISLsCh\\\\\\\/HfCZJj5rG7MM1rR3xZWxZ22u9WYa1iPnhWhDiimhJH4htTGgmqJe9hfhOzm7a8MKg6U4f2u43Np5QW2X\\\\\\\/canjQbYs\\\\\\\/yRh6uvR82tTBnGKUc\\\\\\\/gBaGRanmnbspdgYAWq3yzy1N1NwguGJ9M499uQ9VRefz1SOuy1duo1xT7xZVEs6wcfksjtCA6yk1S7i5Xj\\\\\\\/tj+ZMaXF4OwfPYxiooxvW2gxleIzTp9NBotBQD535m22omW\\\\\\\/iPLVmyxtl\\\\\\\/m7qiGIBE1NNecabu5IecpfnSxlbn1+AjRM6kWKEtE4sToITT17r+cgUNFa2Lb\\\\\\\/mPd53y454HZuCI\\\\\\\/6x1KzULwRhy7XlmEGknkMq050TEw9AoPY\\\\\\\/G6b5wDdMXwfDyq51qj8wJL6Y2MZ3eAU7eiIAA3Y3Y8OzIa85i\\\\\\\/YbR3X3nEkbFTAzrNNPac14sSaLIi2+e+1F7b5WX62t3R0px3hESua8Xy1fvoo8mH1hr\\\\\\\/nK4U1Y0SfPmLN9b+QLIWjWkb80+NUCSOMsmxpQPJIhU69hIThVUrAyPbyArsx+JEeECvXy4zzL7bSRMclXMyAspAyQi+LQp8ZF0mXpZh++JL4rjjTw\\\\\\\/yW2Lnqjbr\\\\\\\/Fz2HjxE0lvzRCeGSoyU9wsxj04w778F72EeJXKP6NkMdMAHmWEFzNJ\\\\\\\/DLvjjLHK1SgE9nLVhPKcfBmMRsuLCTPCwUB1XTBt3kz3gePo+UmqSfP4atWbvNMX56Xy7EtJT\\\\\\\/G0Oop+8dexhYGsCYz2owZmH8LPh0RMr7fLXkcjYivMFs\\\\\\\/qQ6O6MQZdSJAQv+yRF\\\\\\\/xMSauMq3MGinAsxO9WCwPUy0d1As0u46kR+82T174oAL4ajfAlsiApINjsPzZg+0umj7wj3uvT2wPXAUqpIqcKi+w0BJGhCFOga5s1rOV0g184JGIGprN4WsuTovGw++8Sy+4usAVQ8ciwXIvwZDb288PN7U+PNojTtF2AFbvfJnE+zc4QtRgOKWzOkPHFMvpIcsHK+2Zr2KK3Am\\\\\\\/g0uNmhqgJxrKA+ZuSO\\\\\\\/gMHvHbliJdX1hBo12hZIcxtlh+VS\\\\\\\/hgeXSdELraAhuKMwqkNpbfMtHD0ty95Lqlpk6W9Qk+rMceoSoH4ekmfSTCa7rypj+hcjvoEaW9b69RRzQ8sme8FVbQIRxe3BaSYHXV886bHY11b2DcAEvgajxctmr5mkxvT9VWOeIHA8iCH17uPZ\\\\\\\/5hqk7WrnfqAmr9AiQCktUbNt+NzejSDHcdW0gLibZOkF+zjX8XQVayNO\\\\\\\/difQFnAPvkzPe0QQEAEDE\\\\\\\/ucZkpzIM5CR9UuGjuWNgR+9TlrvwnakXnRtXjkqXzfgwEtGbeEN2EJeDjp4btxY5iAtYUg9VJCwKcs2ukMaF7bJ\\\\\\\/vaWPj1bsBK2Tmtd5b+Mch9aens2OHe1VlFryqEH10q\\\\\\\/RACPPZ9KFjHBgxJ0bpW3xumTLqoikGV2HO57fNIVWjpYYDfoZXdfau3oxQZGGlXQLPSKFCuZnM\\\\\\\/jx4EKaPCIJjFVQ+VJkW3RhgTWyU6wG34KdO+tgsRUpQuMMjWCuD\\\\\\\/w4up2+mK2wP+XAuLbyLeFtvqm5XjifnrbelO\\\\\\\/VxxnUzTIX6Vi43ltBvxP56ZOVlHWkZ3nNA\\\\\\\/iIhHFly8oCeeWfKIIgyyHb5KVY9GEKBEyoJ6sPGnrpMcE1FXLWp78N+JXTpLVel4yxve+pZPFx\\\\\\\/8d5sWZWN5NC+D2O5O4Wj\\\\\\\/t9fMIQ1gtbt2F72n5+bDGPTX7wy35me7Io3qtkqGg\\\\\\\/oMA5IQhAl2YKcXOaVOUB+4bJGCrDawIhPVExvVCV2tAbXR2eJ942Ss3Dp+sAJ83vMrsgIOqx\\\\\\\/R8TX\\\\\\\/qag\\\\\\\/A34R00RwAgi\\\\\\\/\\\\\\\/6jytyZUVQveSL+Uqa9qSaRW+z4fHm4CIGy364\\\\\\\/xkK0igPy3UT\\\\\\\/HXrq0PmuYRbBVmVE7VBoKxAUkDnwlf0zwXdeHCUxI+WamI4YTJ03MlDZB\\\\\\\/7cEL5YRUZTU4hAGUsbASSIO0XHYjTQQb0OgUHrqfIvJiwGiD5rjz4m2HBSBKqHmJhBSlcZITCgtM5BLZzSolOdd4oUig+d+TTaEmGu9O2QYUMyzz6GAi1c0NyFHy6s7k89IE2fKpoaxb9yqZUJvEeyoWziucadONtC7YIFjqsPSR3Euqe5JK9zCgNXqEEtSytB3eiuRluUsdlbxmPBfJGXOjW+1S0+rP74QhmV8JjJ9zBBkjuUcBdakwEMFoScIO\\\\\\\/N9ebmzMWCksEYps9lwgLZgK49kmFp2WcISSDr3SLKMS7GUk7faSYXY+kgTCvvD5Fegl1T0XCJhWiHoQLGb\\\\\\\/rKaf22g\\\\\\\/ogcCaoYCDqMScwTLGMkcEvK\\\\\\\/tbpc5SGUExqqcjHzfURtaFqtRBLx8yb56OBnDsTJs4asNWrXvFojqyXW8+F1\\\\\\\/dhZ1fPlYB8a1T1xl9HwhVpwZdLLG5HcN9runihZZ\\\\\\\/8uwTMQLNlsAYVDWdoZq2hu3qQLLPAseiQmSz7djK21XrIooX5TX7ZFVf3D02XZOG6g7sdtIkMCHPj910lOX66\\\\\\\/W7u7xVFXHuXMm7flWbQB6bJdz7XwQLSBkwKKtB4vED7yTUORT6QYN\\\\\\\/pUCIwLxNqxqRiha7f1MD8d23e6jFeWRfWUPSIFJd6wixSMXif1BWvZQ16ItqrhNOOds4sFrCXK8LoE0zJMneH4m8mhVuh7UuLaMp89HBWDmqavB2zLjkjNqubyNKYlgPeGESHqVNU4L3hO3aPTsUZwQ6tW7GP4mDqH\\\\\\\/P1dga61TLwjnE\\\\\\\/7KIBMo4HJWcg99B3MBQ3dhky2KqPeOFTHfCrrKZEyQu4AVZa5yKFI146zJ4lTSGG2OXqqxRONnLwwDquHLSaAF18jQUDH58iKAcPaoGAZO+MunrC\\\\\\\/W9OkKzNBihdnXCT0vfwZ\\\\\\\/C3C2Rw3PY1Z\\\\\\\/b7QurieyZFXnPASd8C+BduuPUnT2PNCXqji51ulbtqmYj5ImNeWhxgq7VxSo+ephX+lem4+I9p4Kyl1wOgO+sfr6gnwWQWj5Kh69WGFeDqToEDcjDlINW3DBqIl7k5WStWS9Jhwl0tLUuk8ePhWpBXwIq7LSyxUGBCM6l1CIzFt6+FUVghw3oyA1Sigiv7oK94C48Rq8JxJCWnXCgYIvfLe9ZVCG4veYJ3bsNf\\\\\\\/8sINOmDVWNcBYr8DiVFC4UE3ifnnifmuVRSvf7tKdOlNoi2H2oXpo7aEf5Tw+UvxJSFvMjHWiCUm9iFKJAM0oDXOy+sVVDohYBwCh2QdzxewqP9Lp\\\\\\\/fLe+s69rLClgC1ZysLQIg9eIB4aev3sLWCPt6XP+RbZ5THO09HxorQpcMmS2BYc4GBpd7FxxMZ1RiwTCQG7W+4FuOPG4RAgeRNFshMPKMi6Fj0FJnJHvZ+Nvt2zDlovSTpij+TMRElXnzRk02KwXvWQwZCVLu\\\\\\\/AbBYcIiaYe4NBNU2raCcSXwGW3uBB\\\\\\\/oJX1tXSOJBwq92v12ffD4E9K8P9FACBAcMM\\\\\\\/AkRJQaNkPLL7IB8fzNqi88rNhhoUyjb5cu8am+xyMsDvS5PH\\\\\\\/XTG0NPQJG7gZ4A\\\\\\\/idkCE+hm\\\\\\\/0chpmJ8jwQd019l3kthydyVMVvvVmV1xcWqitUMbm\\\\\\\/cplG+h47hTiovn7z4RbMbOW39XIVsZ6aoHQWBSpr+RpDPejjVv0uGmi9lLegMO1bJcWawMAbHotAKgqWmKtdhZg7jDzxZFw4te4LFPMPBp1l3zwZE8k5ezDQvQ2xWgttOkYNdhiA7C+wUDJv\\\\\\\/mx2gIdYhMXbE46CLl+e2+2Hit3IaJm3Htv4OK8fQfw06IkRNAkhxe7x2sq3CFOXy7qXHvh7R+1u+G9g7Uhn3kqVrtvb7t7DD099E0pkCSVzxcCbEKBm5SqL18\\\\\\\/+P+pXnc0pKff0whmZXo1Cls0zM+8\\\\\\\/6NJ4sajItf0DAuIPKWhNIthkp1MbZhuzwCAdWg5mnDQ6lx7tM4lF8njQWuGpmLBEoUYWr+pjeD\\\\\\\/dvm\\\\\\\/kAxuq+I1nbw9nlBIJp4oNyWV1KvQNtqIhq8\\\\\\\/yddvBjf2AYD95lPHa3f8yCWclvBZq6O8b06mgJ3839yeaNwQjmhRsNW8+VbpZfgbdCkflAI+0jVx+SQAmiCYkpuPcBpoF5U1ACyxsU+ejS5Qa3yGU6z9n4hTqe+h35\\\\\\\/80jxKZQVt3b2temS23F5Ef1vuq1wCkmxw3RKysVwButkXrMfQh36Un0Y0AhhywLr+O5UEdq4jVJGeBKu9oNe0gDx8SFhhzDZ4gtxocG4qH0MQy7IYpdTJfsQPfXgZK7z0Q4G7i4TfHXeJKZXXiKrRbaOxGLq0lKNrp8ABaU6Y3xWpoBZCUDr9m1O4X05TRR2aTz6orbkhqmxZIuEAF04IKgawR2s4mrJnJ0xCPHh2flAM\\\\\\\/IRsXI+1JLUv63qRvWrOB25cBb2a\\\\\\\/3rONFuGupbz7oKdxeMf4v7LIM3k+USaDdBHBMcAZIPK2Pb\\\\\\\/rxk8kEe1gKTpbOITiM2g6BWJbD6wFxREbghbHcEr+DGhOJs1ewZ3Q7lOFePnYJUDs8peskCdNIXfpxU+mgtxL8nknENtgj9NyVi4D4fFmLr9fp7N99j5ozTeY7DOpH3nsZeBs93iE+so4r5fHl86eSGwafcbOwmMybUUB5aOoOCOpEd3mhRAVqv172awo8p6KW+77DfJRXBCQLga5Tc7iprdwxPsv7Sjw6R+hcgkkxD47RTA7BjdyU5cESZfoc9QW2QJLevjmZUrvBUo0aNzYP\\\\\\\/401ragdsgPlYoL6d3+rnuOJkgwnCOuqrUQBLLI11LZiFRLzKn2ot3qsWB85FUWBkNcu2xgy1D+7QRx8T0EajMR9hD9HmXy2m\\\\\\\/v3z7MFmhXLR+Ytub4KF3RvFX\\\\\\\/hpoJdzqkehwSB137HAcEhRgdVgcyRWql2AamYLu1cde386QfNp7zR4c85Ut+AiP7PSC0yvAxatXcmTu42\\\\\\\/RrTHq+hi4179vArYlOHnAtu5FqOvunBqhGDrfYySYVOO6nE\\\\\\\/sW53K+UYOGEGdrNE8JzS\\\\\\\/\\\\\\\/wEfrl8nIEz5HX4EE9zD88dZTxQ3UGtVdpdsZwSD0K1WvvMlVdPPcU7GkrJM3VPzGlq84ABjHe\\\\\\\/MkK9c46F68lhclAxo0UO\\\\\\\/s+kqBXJ3c\\\\\\\/YB\\\\\\\/5A46XVgtN8zUE22ijf\\\\\\\/mUr\\\\\\\/tM5S24Ypap7Tc3swIWYVKclOTilXjQS9skp98zJdnwVl1b5BsbO9w77aSQFfyNN7dD7btZr\\\\\\\/2uNPm5FAi2wBI83lsSAaCpXgr6+apDEywwIhdx038CglQfXId+QKDUUdJsl2snMWJwNmR0nqq1i7vP7H46\\\\\\\/z8nAp7bjUcbv2yFdB3mApLlWf9aBY6Qo2ZUrNAajHq6e01uoFyhiLjR\\\\\\\/BkjygbQnJHHeiCPvwe5RrSdRtN+7EP4hl7qfQDOBwn\\\\\\\/xgxypUT9nm0hchZMV5T5cZJ2W24PvUERJY6RSE9ayI4z09tgSMiuW1JZt5musXy9+cZ7JEkYWtYatJqNt6Xy1lJPtb3dUbgTJ9azx1\\\\\\\/LJXvfMOAb717lSKCNQVbLgVx2b3i1FTaqgZYjQx5rzfqMqeyG5A2esm2ww\\\\\\\/X3qjJRAR3dLVNWqF8qEAB6kGxtgR8fNbvy\\\\\\\/\\\\\\\/S+07tDiq+nepFlNNo+rXQtzdrLB4EAfuHBd+tc9dWGj7uXXrINjOa+ctn2g3QcENRiQMw6\\\\\\\/a4dGFWGk3akvdX+weLgWKkB0O3UzxGnt59BgBBhLMdHbo4zualWL5wvOi3xDK7y+lHqH1Xp0FJQVZ3SF\\\\\\\/oC3WWF8oZ\\\\\\\/PRtGmBGM1VZGh27zrZyTsIlj42ZN+rutADJxsIIse2Tcglmiy8WxazVh9PMytzkbIE8bRr7NXD8X70pac2aMfkZ+0ctgW+DtFrllAzwv3y5VTxmaaBV2JA9nDiMdjAEfvLhq3ZzZkWSoZsQ+jwnm\\\\\\\/tB2AWrzLwUOKAzvaxUIwuT7xQCKRmVvwbJbYJDkIusSU6xbyQNlOK70l6GrmIjjpg+QkwJRMiWYO5IBK6scaOyHdGbEvvs4N9nnCftppBJHTcadMXI9Hgluqlofk\\\\\\\/emIwO+h3dl2mPX9aEOnBdKG\\\\\\\/ETyarqv3iyJyMDilpajL+HVRifS9vOQW3AYS9vTimOO5v2\\\\\\\/xxyk\\\\\\\/394Ex+6ufpPIv6SMqlrH1j0Z8rhnzMT1qbi5i9g3Ba45eZ0451PqG3zm\\\\\\\/1PiYQ\\\\\\\/x8qLtimB5ycZOBQmEs4vdxwdm72DSJK5hZuhWq7r2agPszfpZjuaHOla3sgO9GUYF3PCjkObLnlNpDPqWOn9D4e1lptJqhbh5DGHrn7pAPXMa4zdpKZcuJafi4Aj3x7JPCEZJdXGHUeY0YuT3UkPbtvZY09cZ0LwEWZyAa6dTpVteJBsuGioOui7QUl9IXAU3M0ACo2sWbw4apkSUpDCrlL1\\\\\\\/3rw2QzNsyjrS900LyEPlw5XlgTNAsiyelVxdJLjGagI06xaJEtf6+ksHLsE8HiGheTtxOuEnEB1yDwh4PFhzx4GO3Iw2s1yHou3pfQq2Ln2rGNjRAB5AIm8m9T+boNdNWkIFrcEcuH5iU8A2KYDoE\\\\\\\/o4x73qRlHwUsUpL2jk1r0qYw0SdnoezztBRzCPPImCbzNt27uQxefzA1BSRiVlkCI4z0brIt4zBUBgZTLCnSQBuMohDgEEoZesPdtTL0e0zgsDbmOlUCVTOVpeOCKcg5Aqhzlh0llMeHiDJOmAU1LEmCkat4DOsseRIvF\\\\\\\/SUCGDRPvuffpS9qj086yOUAR4z7nBydT6VOsqGGPDyusi7V7NuWuSeADiOdcH3zyGvPIhadi38p1lqvLQMk32YSrF1LCWqju7wKFqLcYxNSEaueK\\\\\\\/Jb9Eyh8mb2PcfosIRoJ9RNwK623OHLszZAamLWNzb2zY2f5LkEvCPPH6PMXhYF0g9Xav6uK0qO+IFrJpBVT74LF8JKwxmdErY0peiRiGGLw1r7Ck+B5VhtNVXveZZHHiRZm6UdfDtAvWAhbJHxxDe7weXCMGh1nYzUPafMmBRAjpaNDdNBLIsvUJ3UIwE3FulDMQe9idP2qQOlBtjonImldVR\\\\\\\/Sp4j1IjAZ5yuml4wTJuC3Kw0ZiAmGZ2fD7\\\\\\\/0Sj+PmuEiYFquhfNtuqPPXrmmiNQlQqlQZ8usFBRD472yOaNjQui9f7z+rhWCI9rN2j1JE478Tajq9v4JLz+fx7NCKpV727xXHnrGiSFVbWHjKnVczXKlYQ7Wz2LBTWKhBtzMaltJhexR7Gb6vIphb0eyy\\\\\\\/7uWd2ZGXcA2i8n0mN5HkHANUybs67GcxEcSIPxpQmbl5TLS1k48QAzgQoFtwzmgXTm2RA3PVcJNRRzmDWylCxoOazBhfT+isKbE\\\\\\\/vqIfWuX8dpEGlM\\\\\\\/FRRwuNe0YhDOAx9nmDXEjhbO\\\\\\\/rUc46f8IxVsyCj4o63c9pAbVna8Hzt8YDIBxPNnQvUcrTYmeQ3+Ua8BEGbaeYKL2lutchKAD23WGpAT+QTYs45xMn8zR7G7zb9oYGxGZit31Qx5pNoJarTV6KEIvY0JKGFz3D7bb1SaDGsCS+Y1+WHaxI4dHgsQly61UDYy8xogfHiGchCYxpESDfO8dOUHu7mPejiQSG7DyNtZvgb+L1atNleIV8deOvO23FJM3hrDwUMjisme7QcXLxm0oQ\\\\\\\/Kno7fkfRlDHou8ubWQski9ZQf3L3zY0HdKrIrdv5dUOpksTmGXr5k7twt3uRGUbfSHczK0xc08j0sblprs0x\\\\\\\/ccpCa+UQHptk5BA7Lu7j+66h9Z1TF3ZIbyAfTS80qLVZHkdb0caI7IOkCzW4BlF3dbRp9CaYCteJyjCp73iAFA8VGDjg7j1uQaIx\\\\\\\/vuHvdAT6PJGbYRny1CvrClddY7LN1SfHlbdr531LzKnotpCXznTXJx5ZTEQeSOnYh+QIcIBGiTbo0GHDlTPm2hw0B4HDJuKAZ0pEzxJKVT1ICtgg9Pudhs2O6Fyxma85VDjLBbXrMIUG\\\\\\\/tsz5KZEPofKHmWY2jYxHhh6CfQjnLLLzZyXee7kxthD\\\\\\\/mxgePETSZbYUWbvvf8skQEMD1Lv3oyYuvyLv74Ugbp3I7fGjGkKWInrs7weWg435mUKeW4Dk5j78tmXt9sYBtmEx956JJQBNINkBo9FjE0ecIQGcCqwqJzW5ZnMmni1Ly5wjg5XQ5kW4JZcs\\\\\\\/dE5BikhEbK699QLAmjlCofLzHYRTU+evC6W1kQfPmpECut40AVeAYKaLW+MoUDt1HmBGwV221nJrT7JQhpa0wn5oZ9mfWXRKa1AKCfMBtW3Fz\\\\\\\/ZwKb9xOcvb4BuMTehaKrL2WBxWbUBoNkvLYvuG4U2uiWvh\\\\\\\/0IbVbYKErDxX4R0bJbFAyk0Z1V28e\\\\\\\/4B61SgFfaH20r4ddGsYtDLcOYdCKeec+2aO55G9DNYVoJqTO2xijNwMu9hseDKaq2zIxuOlCcBeEqopBuJNlACDM7nG2oZkxCUb2SbJRAqxy+HJl\\\\\\\/XxhXs7J0ckgC65WVGmeHjW58vWXedHO+ITjWsy8e0nDJwyOYS1Gy8Rx8I+tAV3m3AEcEpoAif0D4ebiQdIb\\\\\\\/Y\\\\\\\/Ub1gyZdEI3Ykump7h8g1pnl5onkhwd6ukqihFTvN88RteW1fBXvPfFTFd5U2zHDXhSwu9d1+pcs+Dffqu6NGjwt4NXUt1NFAKKntJDt0AV0yDbsFOAw3x\\\\\\\/fK3XIZl5fDrpW\\\\\\\/\\\\\\\/3y8LbfRVj+4KeVPVv6TXyVzP7fZQR8xKWbcTbUgaCn0wFEqG4Jw2jIX4jh7oxePZHOQdueVQSzyhkf3Lk06DXb255sMyNy8web4Zd7StHx\\\\\\\/1VNZoGmpMueKGHVu\\\\\\\/l1YsIlvjnmPiKsBcbExDJfNa+FUpStx8BJlDSDs1t+QQjnatb83l4vo70i82UoNvaAwJ0pXy1CPX90iL017dfdxvvuyHwV5tNGPRwA6yCWd43onPrg5dE5aVbx3S4qqahTEjJhd+l9fO80QE3b\\\\\\\/FJ+ISE1efZRDBgTSo8kb2jAqlEaXt2gvmc8kvQUikfD81Si0YjSCDRwVoh1I9\\\\\\\/feLIw7a4tir9spdJ29W0nGUH89v2JpFBksrGrKp3NXuFwetqRIizE+CpvrlK9jsxSmxBSdT1Ub2k9GCMtR0h\\\\\\\/ETRhLUrKUjBRlBEu3gyqInfrIMMFXyEYMYcKwQK1M0HokKS2M\\\\\\\/qqLZwUuW8HqD93cPTgcrb7BTtWht+7VMvHwM0kMBX4TECDzszMJjX7Pnjfun4C3TROMs9\\\\\\\/ivon2e1dUnCqHlLjmNUtUNXsOYmuI+9YfIQDYPJEuUOTwvUmRoOXlgw1BzRNdwOH6oyI4suM+\\\\\\\/\\\\\\\/kudjiSvK25RuxKNLvJPIoX3s6QS818SUKyQ9OhSB0ujYXNh4qoipFt5Q8Ba35RQUaQ6s46hfF+wkVcbGGdOAEhwFP+TjYLzkkRE6ug2gA7v6k8wzhqM0G8nM6IGPlPLJPg84oyVZcRpZmPyiOePscQmB8CgrCDEtCESAg36EiNBgtEO8l+tqW5ppFvHGrBFWmzSSRIsfe7VFY018t2NpP5qQWC0jF0rtXGPVkeQPA8hoIZA++e5dorGIvspCwaFz5gB7alKH+9m5GsjPFFkeGof5JA7lpNIVubiKIXSnuV9v77RS1qewokl\\\\\\\/7j5SSirejVRF1dmVMBg000SnJepHnbL55\\\\\\\/cw0nJZagwl7PHkmU0BF\\\\\\\/57jUtjBN+r5LlVmevIXcwDcLJq1OPDChM6jZUAn4QcWYnxjSEr\\\\\\\/rtuXkQUkOMqlplMiUfN6cZ88QmqWc\\\\\\\/fDbRWfb1+JHpPHKUmT2CtHiutMe0vBl66kTNK9tq2OeKKKZ9fsxidF60T7EF5AHdV2FXLPK3M7otCw+BzWgvwR60do\\\\\\\/SRNyzkY9rtBACjEGdWFtPcHWpV\\\\\\\/CFkoP3hgYZFMI0PoSJQYuzMXoDiJjcExb9AdltYTIK7qswZFTQQ5AudfuH9BeM91lG1PshdRD3QI635I6A6FW1oL4thsfYJKHOh+YaoW3eYTdeePvpvO+d\\\\\\\/j5GeNGzay5elJm1SK+o2RN0yR4FQf9vmar7QDz9TyWPstvAr5P\\\\\\\/IRgURM1Z7Ets5aD1k2vY3qyuJNeXknpbwZnZyxAkHKPbFufVQd8ya2TzpPz+AES9GtomNeo7gDqC1TavyeLOYr3Y2vK6MePY0KBIRus\\\\\\\/ly7NnezNGNNyS5C8FsnEwLH6SXLDhg\\\\\\\/tJmXV1df8qS\\\\\\\/qGyhMYmO\\\\\\\/ks\\\\\\\/rqtO3hKkfEhjecvLwtJaAsR+waBYjVrynnn0ltr4UQ+vcwMJmxygMEiW2fwDU+6mzNq5wwPqd+Tg9GyrkEwniiLQO5LlvtTJXmJ1PML4AoG1DRpsiBMePPrXMJacdfZu7DIvJDpjQlJzqR3gOoBHFTpedhDBg3elgPyoLwm4G0+QJNNk6ceI0yE9+QVS0zH4zxyj\\\\\\\/rbwTTqLw1tCiLisu3mAuSzXjAekPOvX5WVDEZsj7mUTYCZiZzJ\\\\\\\/vrzBvt+Uc6A+xSiuDNCWhGF2tAx2LxEobwhN+9CyXuLqXY6WCbL6FnMUm4\\\\\\\/nMRJWUc4S0BID8muuxix8MQlc9afsZ20Seb29Tvubs9HyJ6UL0LpFaVBdx3uVEs1jzOKoji4RxCm6Dm4oo7Mx6OXV4V5l9vnvLoGTXwv3pw8rGojPIxM8TtfdKm8vhVRTdFIwHd4H\\\\\\\/y9AImTrco8MzKnxBrqnHqol7V7nfbPJ54iqsMrekRbau30\\\\\\\/DJ0cwPyDj0fUNiT\\\\\\\/H9FFwN9l\\\\\\\/XFYJmv7jsy9MZNDceFqOs2znJyOe4\\\\\\\/ts0e5Pw1TGy5DDrlzcOM5RybgIbNo0Ww2isAze7aTvf5Wx7mG9gWspeceO54ZKBW32Z6GnHHWkpFwdNAlcGoP8YzpWpgn470oP01K+XXx6030ZUz3CTWhbI0tFpHwQ3GmxvDMBm9Iu3FYaCthFr8xTn7q+gCIlkke9NcLpWGqHuHL\\\\\\\/GL8TlujXP4wI9kKr+u5W2QwZN5J8BmWf5CtlvOYeFGNEZAkkpOyMPha1\\\\\\\/ZaM2QuSnilagSO9rHl9YrD\\\\\\\/hKLa6+hIQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022URdQn5YJCm4ECKlAGHgSMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225eb7ec4b1d85349c0e5f3074d0a06ead8c3b1d9ad23fcfc04a84aca9f344ec6b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eS6RTtrCpRXhoXyU5Ck+cbqOL\\\\\\\/Sy0sFWE8Y5M+gk1D7sJvvbGluiT\\\\\\\/TNxjho4DhwSVDMpu19PsG+B9h7SkVuA\\\\\\\/se1CXIFsw1CLhuH8sfrPWOQsJesEj6NXT2UiseeSLCgYN+08HOj1KAJConQlFhP2EqPQb6OXbn+Hg5NxIFt1F2+oZdUGLTeHeH6caiKXtJy6M8SM7denJ0bQY+zrLvE8DjlVBR1bIJRXfMDXRXdsMj3H4kdaVTpaMuXD8eucynPWcm1J6PTliDKA3M86EdOeMHgv3o8t7ylzQkCfWuu3OpJO+ULqbEQR3yCLArXDhojSeGc5P7Bnze+knnZb8cHjar7H6ZAEsDB4lDnp\\\\\\\/\\\\\\\/uLlP\\\\\\\/Mi4UZiJ\\\\\\\/rMi9gEv6i6UKJyKzUjW8oKaTmAuIV0b1B0SvzVsW5C+loH29cjEtdvJt6TY5aK6AJ688hiz\\\\\\\/RsqOJFN3PQN1K01taCnbygrfuCNESFSqFa3VR+lnJUd0UQDlL3vTXsySW9GM89BK\\\\\\\/aWgU3A6lIjttWPUEg4eUXVwv5RScl9RRheDJP258EAYwuAVZy+BLex8cUQk6GrdX+Uk0406wzXZ1VTUFQPDAUA9+nIPq58fKpOuqmvaCKVZXqDjfZvHCwBoDLcGQBch37su02t0\\\\\\\/9SvJbqGYq5AaZv+gO6iwlxmkw7e8VMHEm7QasYWFMa4JIrI0DETtUrAIR9Zi+TADT4iRpYKMSFxLXNLxRciGuJX8JMY9A4a\\\\\\\/M9C6PMDpFHh41mam2TqVtlceJkeQdX8mWTR+0+IdG\\\\\\\/inMlEK2YARwjpxe54ObqZulhjdvRKKAGTYDvLdKWFQ+tQ0bW1s5ubjshH+Ywl+J1br4jPxBtwfCSFKdk5WvbtXzs1oZDHTGd8QYaxZftyv04Wnipr3AOxaWmuI\\\\\\\/9Qh6DFdLSQuuR4ZuFq6JgVzkWSavw0vDBHpNdQcck5FFW9eMNAdf5q8yaHMVTKe3xuNHIJhkT2C9NA1nEPHH1v1aLs0uLwTDSKVyl5IFwgnjCeugyAx5zGOBLbFzw6UINqTFqh1RgDZHAu\\\\\\\/bKLwo5qpkuGPgKvRjMBU+VxoNF0b7stB0DeaQoWkl+7HXvWB0Y8\\\\\\\/d\\\\\\\/4E09k4sv\\\\\\\/sHW4iZYkhG\\\\\\\/7EQHKRQBAM\\\\\\\/592kiNCVKA7i5a3cRrG668lHW+3xcF\\\\\\\/pWbsCLM31I\\\\\\\/mKCwzeR97MjxYEYJS98AqMXX8G0ItEnIVu9t5UHF1oevtzo9LBw7Qz\\\\\\\/8MT2yfuimqmgRdeD0654CFrjG++8eaZeirZY12kaY0Y2mXHwoGiJRW\\\\\\\/fXgIR1plTjncNu2YsGapmgsy5xOuUHim\\\\\\\/9TSESXGADRSgmrIW1V8CcXEJ6O3Si7cR2\\\\\\\/kl2sBeIeHvXtqolrwuu1ckorvnv4aCjgG9\\\\\\\/5U0KA5iWGy\\\\\\\/K2W25nR6U5KOkSfS5Lec8nsYM2xirV4kPDT3E6PhkJB5rnYj651tIU1\\\\\\\/eZlW3gYtrKpJx+O\\\\\\\/PUvlnk\\\\\\\/vB0blhZcED1QwMIririDCvDitFB3m\\\\\\\/egDSLg4K1j7+ah09US4dHWakRadhEpfSDwk96H6r2B8DtmkZVIPReQfEguqH\\\\\\\/SOYv4Y+2fPTFIiBmI29GZNZDA6NfTYh1XlgihG1Yfjy6Hd+oBXtaBFaCJzvGhuLPSzt7\\\\\\\/9RDOMNCz8OS5kR9ipMVl2LWNsYchk1Kufx6hbTM5tl6qW8rBoC\\\\\\\/14MF2oDWRMe13YRCyabHgL9PBOzIMVDCgQYXFE5r2ji5bKWGXTbMk5\\\\\\\/ZHIOQ5KwAsanihK0TrpN40tMs4M+M2O0bbFpcwfLN8MxZGiUrgaXtNqoZxFKzTW7NoEjW4nCrgqyYvA2yl+y1gzC2DQH+39o\\\\\\\/OJT\\\\\\\/u0IoYG8+vfP2\\\\\\\/jy7IyZBS7anli8CDaeD\\\\\\\/dzbbf3y\\\\\\\/uiMxqeaLzmwWjVhpFjPdOptQyirZfXNQdiDS36A+QGfmUl0HVbKVx921m4\\\\\\\/ajfG6Ht7HZcbKqoAP2gDOJfj0+8CmYBT6SxED30Q5FRJBDPxpvSWooeBatag==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/2WDS\\\\\\\/6InG1cES0e4FPULg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f751d3748cceb7efab7996fd4daeb8f1071bbbb592ccd81a5b97898e5ddf80ba\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493200 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1707" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sWU0Enh2DZ_3Zg248d3u2huQUKzvGBEhymgmlFl_DTM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1707" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MFehfP5WUEcvNVjpUtJRD4Vm0m607tM-1dvzssvWSAc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1693" title="Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1729" title="Long-Term Combinatorial Exposure to Trichloroethylene and Inorganic Arsenic in Genetically Heterogeneous Mice Results in Renal Tubular Damage and Cancer-Associated Molecular Changes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493200" id="node493200" data-pisa="ggg;9/5/1707" data-pisa-master="ggg;g3.119.400194" data-apath="/ggg/9/5/1707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Determinants of QTL Mapping Power in the Realized Collaborative Cross</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory R. Keele</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5745-4490" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wesley L. Crouse</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2676-9232" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samir N. P. Kelada</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1707-1727;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493200" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory R. Keele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BR.%2BKeele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keele%20GR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BR.%2BKeele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory R. Keele</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wesley L. Crouse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wesley%2BL.%2BCrouse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crouse%20WL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWesley%2BL.%2BCrouse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5745-4490" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wesley L. Crouse</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Samir N. P. Kelada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Marsico Lung Institute, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samir%2BN.%2BP.%2BKelada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelada%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamir%2BN.%2BP.%2BKelada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2676-9232" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samir N. P. Kelada</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1707" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493200" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1707.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493200" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1707.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493200" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1707.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493200" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The Collaborative Cross (CC) is a mouse genetic reference population whose range of applications includes quantitative trait loci (QTL) mapping. The design of a CC QTL mapping study involves multiple decisions, including which and how many strains to use, and how many replicates per strain to phenotype, all viewed within the context of hypothesized QTL architecture. Until now, these decisions have been informed largely by early power analyses that were based on simulated, hypothetical CC genomes. Now that more than 50 CC strains are available and more than 70 CC genomes have been observed, it is possible to characterize power based on realized CC genomes. We report power analyses from extensive simulations and examine several key considerations: 1) the number of strains and biological replicates, 2) the QTL effect size, 3) the presence of population structure, and 4) the distribution of functionally distinct alleles among the founder strains at the QTL. We also provide general power estimates to aide in the design of future experiments. All analyses were conducted with our R package, SPARCC (Simulated Power Analysis in the Realized Collaborative Cross), developed for performing either large scale power analyses or those tailored to particular CC experiments.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/recombinant-inbred-lines" class="hw-term hw-article-keyword hw-article-keyword-recombinant-inbred-lines" rel="nofollow">recombinant inbred lines</a></li><li class="kwd"><a href="/keyword/haplotype-association" class="hw-term hw-article-keyword hw-article-keyword-haplotype-association" rel="nofollow">haplotype association</a></li><li class="kwd"><a href="/keyword/allelic-series" class="hw-term hw-article-keyword hw-article-keyword-allelic-series" rel="nofollow">allelic series</a></li><li class="kwd"><a href="/keyword/multiparental-population" class="hw-term hw-article-keyword hw-article-keyword-multiparental-population" rel="nofollow">multiparental population</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li><li class="kwd"><a href="/keyword/quantitative-trait" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait" rel="nofollow">quantitative trait</a></li><li class="kwd"><a href="/keyword/complex-trait" class="hw-term hw-article-keyword hw-article-keyword-complex-trait" rel="nofollow">complex trait</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-inter-cross" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-inter-cross" rel="nofollow">multiparent advanced generation inter-cross</a></li><li class="kwd"><a href="/keyword/magic" class="hw-term hw-article-keyword hw-article-keyword-magic" rel="nofollow">MAGIC</a></li></ul><p id="p-4">The Collaborative Cross (CC) is a multiparental population (MPP) recombinant inbred (RI) strain panel of laboratory mice derived from eight inbred founder strains (letter abbreviation in parentheses): A/J (A), C57BL/6J (B), 129S1/SvImJ (C), NOD/ShiLtJ (D), NZO/H1LtJ (E), CAST/EiJ (F), PWK/PhJ (G), and WSB/EiJ (H) (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Threadgill <em>et al.</em> 2002</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Churchill <em>et al.</em> 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chesler <em>et al.</em> 2008</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Threadgill and Churchill 2012</a>). This set of founder strains represents three subspecies of the house mouse <em>Mus musculus</em> (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Yang <em>et al.</em> 2011</a>) and, in large part due to the inclusion of three wild-derived founders (F-H), endows the CC panel with far greater genetic variation than previous RI panels derived solely from pairs of classical inbred strains. As an RI panel, the CC thus provides a diverse set of reproducible genomes and represents a powerful tool for genetic analysis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Srivastava <em>et al.</em> 2017</a>). Indeed, although the CC RI panel has only become available in the last six years (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Welsh <em>et al.</em> 2012</a>), it has already yielded new insights into human disease and basic mouse biology (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Shusterman <em>et al.</em> 2013</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Rogala <em>et al.</em> 2014</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Rasmussen <em>et al.</em> 2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lorè <em>et al.</em> 2015</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Levy <em>et al.</em> 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gralinski <em>et al.</em> 2015</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Venkatratnam <em>et al.</em> 2017</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Orgel <em>et al.</em> 2019</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Molenhuis <em>et al.</em> 2018</a>).</p><p id="p-5">As originally envisaged, a key use of the CC was as a resource for QTL mapping (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Threadgill <em>et al.</em> 2002</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Churchill <em>et al.</em> 2004</a>). In theory, its broad genetic diversity makes it ideal for this purpose, and its replicability permits the mapping of phenotypes such as drug-response that are otherwise hard to measure in organisms with non-reproducible genomes (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Mosedale <em>et al.</em> 2017</a>). Its utility for QTL mapping in practice was also predicted by studies in the incipient CC lines (pre-CC) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Durrant <em>et al.</em> 2011</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Philip <em>et al.</em> 2011</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Mathes <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kelada <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Ram <em>et al.</em> 2014</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Rutledge <em>et al.</em> 2014</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kelada 2016</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Donoghue <em>et al.</em> 2017</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Phillippi <em>et al.</em> 2014</a>).</p><p id="p-6">Nonetheless, QTL mapping power depends in part on the number of strains available, and in the CC this number is, and will remain, far less than the 1,000 proposed in <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Churchill <em>et al.</em> (2004)</a>: At the time of this work, mice were available for 59 CC strains from the UNC Systems Genetics Core, with a subset from these 59 and an additional 11 expected to be offered through The Jackson Laboratory (JAX), representing a total of 70 CC strains potentially.</p><p id="p-7">A reduction in strain numbers as a function of allelic incompatibilities between subspecies (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Shorter <em>et al.</em> 2017</a>) was expected, and this winnowed the number of resulting CC strains down to 50-70. This population size, although smaller than originally intended, reflects the biological and financial realities of maintaining a sustainable mammalian genome reference population. [Whereas cost grows proportional to the the number of strains, demand does not, and a much larger number of strains would threaten the economic viability of the operation (F. Pardo-Manuel de Villena, <em>pers. comm</em>.).] Nonetheless, subsets of the available CC strains have already been used to map QTL, as evidenced by a growing list of studies (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Vered <em>et al.</em> 2014</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Mosedale <em>et al.</em> 2017</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Graham <em>et al.</em> 2017</a>). Beyond these successes, however, it is unclear how much the reduction in the number of strains has affected the ability to map QTL in the CC in general.</p><p id="p-8">The initially proposed figure of 1,000 CC strains in <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Churchill <em>et al.</em> (2004)</a> was more formally justified in <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a> as being necessary to provide enough power both to map single QTL and for robust, genome-wide detection of epistasis. That estimate was based on simulations involving larger numbers (500-1,000) of hypothetical CC genomes. Those simulations, performed before any CC strains existed and with the goal of guiding the CC’s design, had a broad scope, exploring the effect of varying strain numbers, alternative mapping approaches [association of single nucleotide polymorphisms (SNPs) <em>vs.</em> association of inferred haplotypes], and alternative breeding strategies. As such, the power estimates that were reported do not reflect the number of CC strains now available, nor their actual, realized founder mosaic genomes. An updated, more focused power analysis that both exploits and works within the constraints of the realized genomes is therefore timely.</p><p id="p-9">Power analyses have been performed previously for a number of RI panels. For biparental RIs, they have been performed analytically in plants (<em>e.g.</em>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kaeppler 1997</a>), animals [<em>e.g.</em>, the BXD lines in mice (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Belknap <em>et al.</em> 1996</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Peirce <em>et al.</em> 2004</a>)], and in general (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cowen 1988</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Soller and Beckmann 1990</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Knapp and Bridges 1990</a>), as well as through simulation (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Falke and Frisch 2011</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Takuno <em>et al.</em> 2012</a>). For MPP RIs, they have most often been reported as those resources were introduced to the community. This includes, in plants: <em>Arabidopsis</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Klasen <em>et al.</em> 2012</a>), nested association mapping (NAM) populations (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Li <em>et al.</em> 2011</a>) in maize (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Yu <em>et al.</em> 2008</a>) and sorghum (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bouchet <em>et al.</em> 2017</a>), and multigenerational advanced intercross (MAGIC) populations of rice (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Yamamoto <em>et al.</em> 2014</a>) and maize (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dell’Acqua <em>et al.</em> 2015</a>). In animals, other than aforementioned prospective study of <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a>: <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Noble <em>et al.</em> (2017)</a> assessed mapping power of SNP association while introducing a 507-strain nematode resource, the <em>Caenorhabditis elegans</em> Multiparental Experimental Evolution (CeMEE) panel; and <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">King <em>et al.</em> (2012)</a> estimated haplotype-based association power while introducing the <em>Drosophila</em> Synthetic Population Resource (DSPR), a fly panel with more than 1,600 lines. In a follow-up DSPR power analysis, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">King and Long (2017)</a> compared the DSPR with the related <em>Drosophila</em> Genetic Reference Panel (DGRP) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mackay <em>et al.</em> 2012</a>). They illustrated how QTL effect size differs between a population whose allele frequencies are more balanced (DSPR) <em>vs.</em> one whose allele frequencies are less balanced (DGRP). They also explored implications for cross-population and compared mapping power for bi-allelic QTL, based on single SNPs, and multi-allelic QTL constructed from actual adjacent SNPs within genes.</p><p id="p-10">Here we examine related topics on QTL mapping power in the realized CC, including: 1) how power is affected by the number of strains and replicates; 2) how it is affected by the number of functional alleles and their distributions among the founders; and 3) how the QTL effect size is specific to a particular population or sample and how that influences a power estimate’s interpretation.</p><p id="p-11">To allow researchers to repeat our power analysis framework, but tailored to their own specific requirements or with updated CC genome lists, we provide an R package SPARCC (Simulated Power Analysis of the Realized Collaborative Cross), a tool that evaluates the power to map QTL by performing efficient haplotype regression-based association analysis of simulated QTL using the currently available CC genomes. SPARCC is highly flexible, allowing QTL to be simulated with any possible allele-to-founder pattern and scaled with respect to different reference populations. As a re-usable resource, researchers could estimate power calculations based on the CC strains available to them and potentially incorporate prior knowledge about the genetic architecture of the likely QTL or the phenotype as whole.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-12">Our power calculations are based on three main processes:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-13">Simulation of CC data, including selection of CC strains from a fixed set of realized CC genomes, and QTL location, and simulation of phenotypes.</p></li><li id="list-item-2"><p id="p-14">QTL mapping, including determination of significance thresholds.</p></li><li id="list-item-3"><p id="p-15">Evaluation of QTL detection accuracy, power, and false positive rate (FPR).</p></li></ol><p id="p-16">These are described in detail below, after a description of the genomic data that serves as the basis for the simulations.</p><div id="sec-2" class="subsection"><h3>Data on realized CC genomes</h3><div id="sec-3" class="subsection"><h4>CC strains:</h4><p id="p-17">Genome data on all 19 autosomes and the X chromosome were obtained for a set of 72 CC strains (listed in <strong>Appendix C</strong>) available at the time of writing from <a href="http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs">http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs</a>. Genome data were in the form of founder haplotype mosaics (see below) for each strain, estimated with genotype data from the MegaMUGA genotyping platform (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Morgan <em>et al.</em> 2016</a>) applied to composites of multiple mice per strain. Since genotyping, some of the 72 strains have become extinct, and more may do so in the future (Darla Miller <em>pers. comm</em>.), although it is also possible that more may be added. At the time of writing, however, these were all genomes that had been observed at UNC.</p><p id="p-18">Of the 72 CC strains used in the simulations, it is planned that 54 will be maintained and distributed by The UNC Systems Genetics Core, along with another 5 whose genome data were not available in time for this study (see <strong>Discussion</strong>) to give a UNC total of 59 strains (listed in <strong>Appendix C</strong>). A subset of the UNC 59 will also eventually be maintained by The Jackson Laboratory, which will also potentially maintain 11 of the 72 not among the UNC 59.</p><p id="p-19">The 72 strains used in the simulations included two that were more closely related than others: CC051 and CC059. These strains, which are among the UNC 59, were derived from the same breeding funnel (making the number of independent strains available from UNC arguably 58). Their relatedness, though not explicitly modeled in the simulations, is nonetheless marked in the figures, which include an indicator denoting 58 as a currently realistic maximum for strain number in CC studies.</p></div><div id="sec-4" class="subsection"><h4>Reduced dataset of haplotype mosaics:</h4><p id="p-20">The genomes of the CC, as with other MPPs, can be represented by inferred mosaics of the original founder haplotypes (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Mott <em>et al.</em> 2000</a>). Founder haplotype mosaics were inferred previously by the UNC Systems Genetics Core (<a href="http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs">http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs</a>) using the hidden Markov model (HMM) of <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fu <em>et al.</em> (2012)</a> applied to genotype calls from MegaMUGA, a genotyping platform providing data on 77,800 SNP markers (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Morgan <em>et al.</em> 2016</a>). The HMM inference provides a vector of 36 probabilities for each CC strain for each of 77,551 loci (each defined as the interval between adjacent SNP markers) across the genome. Rather than using all of the available data for our simulations, we used a reduced version: since adjacent loci often have almost identical descent, mapping using all loci is both computationally expensive and—at least for the purposes of the power analysis—largely redundant. Thus, prior to analysis, the original dataset was reduced by averaging adjacent genomic intervals whose diplotype probabilities were highly similar. Specifically, adjacent genomic intervals were averaged if the maximum L2 norm between the probability vectors of all individuals is less than 10% of the maximum possible L2 norm (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-1.gif"/></noscript></span></span>); this reduced the file storage from 610 MB to 288 MB, and the genome from 77,551 to 17,900 intervals (76.9% reduction in positions to be evaluated in a scan).</p></div></div><div id="sec-5" class="subsection"><h3>Phenotype simulation</h3><p id="p-21">Phenotypes for CC strains were simulated based on effects from a single QTL, plus effects of polygenic background (“strain effects”), and noise. Within our simulation framework, we specified: 1) the QTL location, which was randomly sampled from the genome; 2) the sample size in terms of both strains and replicates; 3) how the eight possible haplotypes at that location are grouped into eight or fewer functional alleles (the “allelic series”; see below); and 4) how those alleles, along with strain information, are used to generate phenotype values (see below).</p><div id="sec-6" class="subsection"><h4>Underlying phenotype model:</h4><p id="p-22">Simulated phenotypes were generated according to the following linear model. For given QTL with <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-2.gif"/></noscript></span></span> functional alleles, phenotype values <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-3.gif"/></noscript></span></span> for <em>N</em> individuals in <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-4.gif"/></noscript></span></span> strains were generated so that<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <strong>1</strong> is an <em>N</em>-vector of 1’s, <em>μ</em> is an intercept, Z is an <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-6.gif"/></noscript></span></span> incidence matrix mapping individuals to strains, X is an <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-7.gif"/></noscript></span></span> allele dosage matrix mapping strains to their estimated dosage of each of the <em>m</em> alleles, β is an <em>m</em>-vector of allele effects, u is an <em>n</em>-vector of strain effects (representing polygenic background variation), and ε is an <em>N</em>-vector of unstructured, residual error. The parameter vectors β, u, and ε were each generated as being equivalent to independent normal variates rescaled to have specific variances: the strain effects u and residual ε were rescaled to have population (rather than sample) variances <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-9.gif"/></noscript></span></span> respectively; the allele effects β were rescaled so that the QTL contributes a variance <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-10.gif"/></noscript></span></span>, with this latter rescaling performed in one of three distinct ways (described later).</p><p id="p-23">The relative contributions of the QTL, polygenic background, and noise were thus controlled through three parameters: the QTL effect size <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-11.gif"/></noscript></span></span>, the strain effect size <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-12.gif"/></noscript></span></span>, and the residual variance <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-13.gif"/></noscript></span></span>. By convention, these were specified as fractions summing to exactly 1.</p><p id="p-24">The allele dosage matrix X was generated by collapsing functionally equivalent haplotypes according to a specified allelic series. Let D be an <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-14.gif"/></noscript></span></span> incidence matrix describing the diplotype state of each CC strain at the designated QTL, with columns corresponding to AA,..., HH, AB,..., GH, such that, for example, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-15.gif"/></noscript></span></span> implies CC strain 3 has diplotype AA. (Note that throughout, the X-chromosome was treated identically to an autosome, most closely reflecting a study using female mice.) Then<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where A is an <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-17.gif"/></noscript></span></span> additive model matrix that maps diplotype state to haplotype dosage (<em>e.g.</em>, diplotype AA equals 2 doses of A), and M is an <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-18.gif"/></noscript></span></span> “merge matrix” [after <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Yalcin <em>et al.</em> (2005)</a>] that encodes the allelic series, mapping the 8 haplotypes to <em>m</em> alleles, such that if haplotypes A and B were both in the functional group “allele 1”, then diplotype AB in D would correspond to 2 doses of allele 1 in X (see examples in <strong>Appendix D</strong>).</p></div><div id="sec-7" class="subsection"><h4>QTL allelic series:</h4><p id="p-25">The specification of an allelic series, rather than assuming all haplotype effects are distinct, acknowledges that for many QTL we would expect the same functional allele to be carried by multiple founder haplotypes. For our main set of simulations, the allelic series was randomly sampled from all possible configurations (examples in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>); in a smaller, more focused investigation of the effects of allele frequency imbalance, we sampled from all possible configurations of bi-alleles.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example allelic series with differing numbers of functional alleles. Each row is an allelic series, each column of the grid is a CC founder, and colors correspond to functional allele. Two examples of allelic series are provided for each number of functional alleles: a balanced series and an imbalanced series. The entire space of allelic series are not shown here; however, the full space of series with two alleles is shown in Figure 9A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example allelic series with differing numbers of functional alleles. Each row is an allelic series, each column of the grid is a CC founder, and colors correspond to functional allele. Two examples of allelic series are provided for each number of functional alleles: a balanced series and an imbalanced series. The entire space of allelic series are not shown here; however, the full space of series with two alleles is shown in Figure 9A.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F1.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1707/F1.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492944" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Example allelic series with differing numbers of functional alleles. Each row is an allelic series, each column of the grid is a CC founder, and colors correspond to functional allele. Two examples of allelic series are provided for each number of functional alleles: a balanced series and an imbalanced series. The entire space of allelic series are not shown here; however, the full space of series with two alleles is shown in <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h4>Alternative definitions of QTL effect size: B and DAMB:</h4><p id="p-27">The QTL effect size (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-19.gif"/></noscript></span></span>) is a critical determinant of mapping power; yet its precise definition and corresponding interpretation often varies between studies and according to what question is being asked. We used two alternative definitions, “B” and “DAMB”, described below. These alternatives acknowledge that the proportion of variance explained by a particular QTL, and thus the power to detect that QTL, is not determined solely by <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-20.gif"/></noscript></span></span>, but rather depends on several additional factors, namely: the variance of the finite sample of allele effects β; the allelic series configuration M; and the particular set of CC strains and their locus diplotypes D.</p><p id="p-28">Definition B scales the allele effects so that <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-21.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-22.gif"/></noscript></span></span> denotes the population variance (rather than the sample variance). The QTL effect size is interpretable as the variance that would be explained by the QTL in a theoretical population that is balanced with respect to the functional alleles. As such, the proportion of variance explained by the QTL in the mapping population will deviate from <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-23.gif"/></noscript></span></span> due to imbalance in both M and D. Conversely, for a given <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-24.gif"/></noscript></span></span>, the allelic values at a QTL will be constant across populations. (Note: the 2 multiplier ensures proper scaling since X from <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> includes dosages of founder haplotypes at the QTL, ranging from 0 to 2.)</p><p id="p-29">Definition DAMB scales the QTL effect so that <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-25.gif"/></noscript></span></span>. The QTL effect size is exactly the variance explained by the QTL in the mapping population, essentially the <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-26.gif"/></noscript></span></span>. As such, it depends on both M and D. Correspondingly, for a given <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-27.gif"/></noscript></span></span>, the allelic values will adjust depending on which population they are in. [In the <strong>Supplement</strong>, for completeness, we also describe a further, intermediate option, Definition MB, where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-28.gif"/></noscript></span></span>, corresponding to balanced founder contributions.]</p><p id="p-30">The earlier power study of <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a>, which considered only bi-allelic QTL, defined effect size in a manner comparable to Definition B.</p></div><div id="sec-9" class="subsection"><h4>Averaging over strains and causal loci:</h4><p id="p-31">The previous subsections described simulation of a single phenotype conditional on a set of strains and a causal genomic locus. For each of <em>S</em> simulations, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-29.gif"/></noscript></span></span>, we averaged over these variables by uniformly sampling 1) the set of strains included in the experiment (for a specified number of strains), 2) the causal locus underlying the QTL, and 3) the allelic series (for a specified number of functional alleles). This was intended to produce power estimates that take into account many sources of uncertainty and are thus broadly applicable.</p></div></div><div id="sec-10" class="subsection"><h3>QTL detection and power estimation</h3><div id="sec-11" class="subsection"><h4>QTL mapping model:</h4><p id="p-32">QTL mapping of the simulated data were performed using a variant of Haley-Knott (HK) regression (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Haley and Knott 1992</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Martínez and Curnow 1992</a>) that is commonly used in MPP studies (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Mott <em>et al.</em> 2000</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Liu <em>et al.</em> 2010</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gatti <em>et al.</em> 2014</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Zheng <em>et al.</em> 2015</a>) whereby association is tested between the phenotype and the local haplotype state, the latter having been inferred probabilistically from genotype (or sequence data) and represented as a set of diplotype probabilities or, in the case of an additive model, a set of haplotype dosages then used as predictors in a linear regression. Specifically, we used HK regression on the strain means (<a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> 2006a</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Zou <em>et al.</em> 2006</a>) via the linear model<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-30.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-31.gif"/></noscript></span></span> is the s<sup>th</sup> simulated <em>n</em>-vector of strain means, P is an <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-32.gif"/></noscript></span></span> matrix of inferred diplotype probabilities for the sampled CC genomes at the QTL [<em>i.e.</em>, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-33.gif"/></noscript></span></span>; see <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Zhang <em>et al.</em> (2014)</a>], and ϵ is the <em>n</em>-vector of residual error on the means, distributed as <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-34.gif"/></noscript></span></span>. The above implies an eight-allele model (cf <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> with <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-35.gif"/></noscript></span></span>). Although this assumption could lead to reduced power when there are fewer functional alleles, particularly at loci in which the functional alleles are not well represented, it is commonly used in practice, in accordance with the fact that the allelic series of an unmapped QTL would typically be unknown in advance [<em>e.g.</em>, <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Mott <em>et al.</em> (2000)</a>; <a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a</a>,<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">b</a>); <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Svenson <em>et al.</em> (2012)</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gatti <em>et al.</em> (2014)</a>]. Additional factors that might contribute to variation in an experiment, such as covariates or batch effects, are neither simulated nor modeled; it is assumed that such factors would be adequately accounted for by, for instance, addition of suitable covariates, pre-processing (<em>e.g.</em>, residualizing) of phenotype values, and ultimately lead to a more-or-less equivalent analysis to that described here. The fit of <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a> was compared with that of an intercept-only null model via an F-test, and produced a p-value, reported as its negative base 10 logarithm, the logP. This procedure was performed for all loci across the genome, resulting in a genome scan for <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-36.gif"/></noscript></span></span>.</p></div><div id="sec-12" class="subsection"><h4>Genome-wide significance thresholds and QTL detection:</h4><p id="p-33">Genome-wide significance thresholds were determined empirically by permutation. The CC panel is a balanced population with respect to founder genomic contributions and, by design, has minimal population structure. These features support the assumption of exchangeability among strain genomes: that under a null model in which the genetic contribution to the phenotype is entirely driven by infinitesimal (polygenic) effects, all permutations of the strain labels (or equivalently, of the strain means vector <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-37.gif"/></noscript></span></span>) are equally likely to produce a given configuration of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-38.gif"/></noscript></span></span>. Permutation of the strain means, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-39.gif"/></noscript></span></span>, was therefore used to find the logP critical value controlling genome-wide type I error rate (GWER) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Doerge and Churchill 1996</a>). Briefly, we sampled 100 permutations and perform genome scans for each; this was done efficiently using a standard matrix decomposition approach (<strong>Appendix A</strong>). The maximum logPs per genome scan and simulation <em>s</em> were then recorded, and these are fitted to a generalized extreme value distribution (GEV) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dudbridge and Koeleman 2004</a>; <a id="xref-ref-78-6" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> 2006a</a>) using the R package evir (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Pfaff and McNeil 2018</a>). The upper <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-40.gif"/></noscript></span></span> quantile of this fitted GEV was then taken as the α-level significance threshold, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-41.gif"/></noscript></span></span>. If the maximum observed logP for <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-42.gif"/></noscript></span></span> in the region of the simulated QTL exceeded <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-43.gif"/></noscript></span></span>, then the corresponding locus was considered to be a (positively) detected QTL (see immediately below).</p></div><div id="sec-13" class="subsection"><h4>Performance evaluation:</h4><p id="p-34">For a given simulation, we declared a true positive if the detected QTL was within ±5Mb of the true (simulated) QTL. The 5Mb window size was used to approximate a QTL support interval, which is partly a function of linkage disequilibrium (LD) in the CC. (LD has been characterized in the CC previously but not summarized with a single point estimate (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Collaborative Cross Consortium 2012</a>); our choice of 5Mb is therefore an approximation, but we find that it only marginally increased mapping power relative to using smaller windows.) A false positive was declared if one or more QTL were detected on chromosomes other than the chromosome harboring the simulated QTL. Simulations in which a QTL was detected on the correct chromosome but outside the 5Mb window were disregarded; although this was potentially wasteful of data and biased FPR slightly downward due to loss of false positives on the chromosome with the simulated QTL, it avoided the need for arbitrary rules to handle edge cases in which it was ambiguous whether the simulated signal had been detected or not. Power for a given simulation setting was then defined as the proportion of true positives among all simulations at that setting, and the FPR was defined as the proportion of false positives.</p><p id="p-35">As a measurement of mapping resolution, for true positive detection, we recorded the mean and the 95% quantile of the genomic distance from the true QTL. Given our criterion for calling true positives, the maximum distance was necessarily 5Mb, and experimental settings that correspond to low power would be expected to have fewer data points, yielding estimates that are unstable. In order to obtain more stable estimates, we used a regularization procedure, estimating the mean distance and 95% quantiles as weighted averages of the observed values and prior pseudo-observations. Specifically, for an arbitrarily small but detected true positive QTL, it is reasonable to expect the peak signal to be distributed uniformly within the ± 5Mb window. This implies a mean location error of 2.5Mb and a 95% quantile of 4.75Mb. Thus, when calculating the regularized mean location error we assumed 10 prior pseudo-observations of 2.5Mb, and when calculating the regularized 95% quantile we assume 10 prior pseudo-observations of 4.75Mb. This number of pseudo-observations represents 1% of the maximum number of possible data points.</p></div></div><div id="sec-14" class="subsection"><h3>Overview of the simulations</h3><div id="sec-15" class="subsection"><h4>Simulation settings:</h4><p id="p-36">Simulations for all combinations of the following parameter settings:</p><ul class="list-unord " id="list-2"><li id="list-item-4"><p id="p-37">Number of strains: [(10-70 by 5), 72]</p></li><li id="list-item-5"><p id="p-38">QTL effect size (%): [1, (5-95 by 5)]</p></li><li id="list-item-6"><p id="p-39">Number of functional alleles: [2, 3, 8]</p></li></ul><p id="p-40">The number of observations per strain were fixed at <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-44.gif"/></noscript></span></span> and the background strain effect size was fixed at <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-45.gif"/></noscript></span></span> with the understanding that results from these simulations provide information on other numbers of replicates and strain effect sizes implicitly. Specifically, a simulated mapping experiment on strain means that assumes <em>r</em> replicates, strain effect <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-46.gif"/></noscript></span></span>, and QTL effect size <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-47.gif"/></noscript></span></span> is equivalent to a single-observation mapping experiment with no strain effect and QTL effect size <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-48.gif"/></noscript></span></span>, where<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-49.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>[<a id="xref-ref-78-7" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a>, after <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Soller and Beckmann (1990)</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Knapp and Bridges (1990)</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Belknap (1998)</a>]. For example, a mapping experiment on strain means with QTL effect size <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-50.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-51.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-52.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-53.gif"/></noscript></span></span>, is equivalent to our simulation of a single-observation with no strain effect but QTL effect size <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-54.gif"/></noscript></span></span> (<strong>Supplement</strong>).</p><p id="p-41">We conducted <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-55.gif"/></noscript></span></span> simulation trials per setting. CC strains and the position of the QTL were sampled for each simulation, providing estimates of power that are effectively averaged over the CC population. We ran these settings for QTL effect sizes specified with respect to the observed mapping population (Definition DAMB) and a theoretical population that is balanced in terms of the functional alleles (Definition B). Confidence intervals for power were calculated based on Jeffreys interval (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2001</a>) for a binomial proportion. A description of the computing environment and run-times are provided in <strong>Appendix B</strong>.</p></div></div><div id="sec-16" class="subsection"><h3>Examining FPR when accounting for non-exchangeability of CC strain genomes</h3><p id="p-42">In the simulations and mapping procedures described above, strain effects are modeled under the assumption that all CC strains are (at least approximately) equally related. That is, the effects <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-56.gif"/></noscript></span></span> in <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> are simulated as <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-57.gif"/></noscript></span></span> such that any permutation of the values is equally likely (the effects are exchangeable), and this same assumption is made in both the mapping model of <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a> and the permutation-based estimation of significance thresholds.</p><p id="p-43">An assumption of equal relatedness among CC strains is commonplace: it is suggested by the exchangeable random funnel design used in the CC, is supported by the results of <a id="xref-ref-78-8" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a>, and has been made in every CC or pre-CC mapping analysis to our knowledge. Making this assumption simplifies QTL mapping analysis by obviating the need for an explicit modeling of genomic similarity [as in, <em>e.g.</em>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kang <em>et al.</em> (2008)</a>], since, when those similarities are approximately equal and the analysis is performed on strain means, the strain effects are absorbed into the residual error.</p><p id="p-44">Nonetheless, CC strains are equally related only in expectation. Much like the “equal” relatedness of siblings, realized relatedness will depart from expectation due to chance at the point of mixing, and, in the case of the CC, due to selection [<em>e.g.</em>, arising from male sterility (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Shorter <em>et al.</em> 2017</a>)] and genetic drift during inbreeding [as reflected in unequal founder contributions by <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Srivastava <em>et al.</em> (2017)</a>]. This combination of stochastic forces can produce unequal relatedness, correlated effects among strains, and population structure, at least at some level.</p><p id="p-45">To quantify population structure in the realized CC, we compared the eigenvalues of the realized genetic relationship matrix K, calculated from the founder mosaic probabilities [after <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Gatti <em>et al.</em> (2014)</a>], with those from an idealized K that reflects equal relatedness of the CC strains, whose off-diagonal elements were set to the mean value observed for the off-diagonal elements in the realized K. We observed that slightly fewer principal components are required to explain 95% of the variation in the realized K than are required for the balanced K (64 <em>vs.</em> 68 components, respectively; Figure S5A). This reduction was attenuated with the omission of CC059, one of the two cousin strains, but not completely (64 <em>vs.</em> 67 components; Figure S5B). This suggested that the realized CC strains have mild population structure.</p><p id="p-46">To evaluate to what degree the population structure in the realized CC genomes could inflate FPR when mapping using an analytic model and threshold procedure that ignores it (<em>i.e.</em>, that assumes exchangeability), we performed an additional set of null simulations in which strain effects were generated according to additive infinitesimal model (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lynch and Walsh 1998</a>) based on the actual genomic similarities. Specifically, we set <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-58.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-59.gif"/></noscript></span></span> but left our mapping protocol unchanged. We conducted 10,000 such null simulations with r = 1 for each setting of strain effect size (%): [0-100 by 20]. These simulations were performed using either all 72 founder strains or 71 strains with the omission of CC059, one of the two highly-related cousin strains. A false positive was declared if any QTL were detected based on the permutation-based significance threshold.</p></div><div id="sec-17" class="subsection"><h3>Measuring the Beavis effect</h3><p id="p-47">The “Beavis effect” (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beavis 1994</a>) refers to an upward bias in estimated effect sizes for detected QTL. This phenomenon, also known as the “winner’s curse” (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Zöllner and Pritchard 2007</a>), arises because the data used for effect estimation are substantially selected during QTL discovery; the resulting (post-selection) estimates are thus inflated due to ascertainment bias. The Beavis effect was evaluated theoretically in <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Xu (2003)</a> and found to be most pronounced in studies of smaller sample size (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-60.gif"/></noscript></span></span>), suggesting that it could be a significant feature of CC mapping studies.</p><p id="p-48">To assess the extent of the Beavis effect in CC mapping experiments, we performed simulations (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-61.gif"/></noscript></span></span>) mapping a bi-allelic QTL, with one replicate (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-62.gif"/></noscript></span></span>) and zero background strain effect (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-63.gif"/></noscript></span></span>) for all combinations of simulated QTL effect size under Definition DAMB <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-64.gif"/></noscript></span></span> and numbers of strains <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-65.gif"/></noscript></span></span>. If an association was detected within the 10Mb window (using permutation-based thresholds as above), then we recorded the QTL effect size as the <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-66.gif"/></noscript></span></span> of the model fit at the peak locus (which may or may not be the locus at which the QTL was simulated).</p></div><div id="sec-18" class="subsection"><h3>Availability of data and software</h3><div id="sec-19" class="subsection"><h4>R package:</h4><p id="p-49">All analyses were conducted in the statistical programming language R (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">R Core Team 2018</a>). SPARCC is available as an R package on GitHub at <a href="https://github.com/gkeele/sparcc">https://github.com/gkeele/sparcc</a>. Specific arguments that control the phenotype simulations, the strains used, genomic position of simulated QTL, and allelic series, are listed in the <strong>Supplement</strong>. A static version of SPARCC is also provided there (File S2).</p><p id="p-50">Also included within the SPARCC R package are several results datasets. These include data tables of power summaries from our simulations, as well as table summaries from simulations of a bi-allelic QTL that is balanced in the founders, maximally unbalanced in the founders, and the distance between detected and simulated QTL. Further details are provided in File S1 of the <strong>Supplement</strong>, an account of all the supplemental files. These files are available at figshare, including data, and scripts to run the analysis and produce the figures. File S3 contains the founder haplotype mosaics required for the SPARCC package. Files S4, S5, and S6 can be used to perform the large-scale power analysis. File S7 describes options in the SPARCC package, and also provides two simple tutorials, the first of which generates <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. File S8 produces the figures in this paper and <strong>Supplement</strong>. File S9 is the supplemental tables and figures. Supplemental material is available at <a href="https://doi.org/10.25387/g3.7409405">https://doi.org/10.25387/g3.7409405</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated CC data and resulting genome scans. Five simulated genome scans are generated by the code provided in a simple example using the R package SPARCC. Red dashed lines represent 95% significance thresholds based on 100 permutation scans. A blue tick represents the simulated position for a QTL that was successfully detected, whereas a red tick marks a QTL that was missed. These simulations were based on a specified set of 65 CC strains, five replicates of each strain, two functional alleles, 10% QTL effect size, and no background strain effect. The QTL is not mapped in the fourth simulation, ranked top to bottom, resulting in a power of 80%. Actual power calculations are based on a greater number of simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulated CC data and resulting genome scans. Five simulated genome scans are generated by the code provided in a simple example using the R package SPARCC. Red dashed lines represent 95% significance thresholds based on 100 permutation scans. A blue tick represents the simulated position for a QTL that was successfully detected, whereas a red tick marks a QTL that was missed. These simulations were based on a specified set of 65 CC strains, five replicates of each strain, two functional alleles, 10% QTL effect size, and no background strain effect. The QTL is not mapped in the fourth simulation, ranked top to bottom, resulting in a power of 80%. Actual power calculations are based on a greater number of simulations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F2.medium.gif" width="291" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1707/F2.medium.gif" width="291" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492901" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-51" class="first-child">Simulated CC data and resulting genome scans. Five simulated genome scans are generated by the code provided in a simple example using the R package SPARCC. Red dashed lines represent 95% significance thresholds based on 100 permutation scans. A blue tick represents the simulated position for a QTL that was successfully detected, whereas a red tick marks a QTL that was missed. These simulations were based on a specified set of 65 CC strains, five replicates of each strain, two functional alleles, 10% QTL effect size, and no background strain effect. The QTL is not mapped in the fourth simulation, ranked top to bottom, resulting in a power of 80%. Actual power calculations are based on a greater number of simulations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h4>CC strains:</h4><p id="p-52">The 72 CC strains with available data that were included in the simulations are described in <strong>Appendix C</strong>. Founder diplotype probabilities for each CC strain are available on the CC resource website (<a href="http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs">http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs</a>). We used probabilities corresponding to build 37 (mm9) of the mouse genome, though build 38 (mm10) is also available at the same website.</p><p id="p-53">We store the founder haplotype data in a directory structure that SPARCC is designed to use, and was initially established by the HAPPY software package (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Mott <em>et al.</em> 2000</a>). The reduced data are available on GitHub at <a href="https://github.com/gkeele/sparcc_cache">https://github.com/gkeele/sparcc_cache</a>.</p></div></div></div><div class="section results" id="sec-21"><h2 class="">Results</h2><p id="p-54">Power simulations were performed for varying numbers of strains, replicates, functional alleles, and for a ladder of QTL effect sizes. QTL effect size was defined in two ways: as the variance explained in a hypothetical population that is balanced with respect to the alleles (Definition B; see <strong>Methods</strong>), or as the variance explained in the realized population (Definition DAMB). In this section we focus on results using the first of these, Definition B, owing to its more consistent theoretical interpretation. Under that definition, plots of power against numbers of strains are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, and power across a representative selection of conditions is shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. For comparison, these numbers are also provided for simulations under Definition DAMB in Table S1. Throughout these simulations the false positive rate was controlled at the target 0.05 level (Figure S2).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Power curves by number of CC strains. Results are stratified by a number of replicates, background strain effect size, and the number of functional alleles. The [top] row is based on a single observation per strain and no background strain effect. The [middle] row corresponds to five replicates per strain and no background strain effect. For the [bottom row], five replicates are observed and the QTL effect size and background strain effect size sum to 50%, thus penalizing smaller QTL more harshly. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC. The columns, left to right, correspond to two, three, and eight functional alleles. Closed circles represent power estimates that were directly assessed, whereas open circles were interpolated. Simulations define allele effect size according to Definition B (see Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Power curves by number of CC strains. Results are stratified by a number of replicates, background strain effect size, and the number of functional alleles. The &lt;strong&gt;[top]&lt;/strong&gt; row is based on a single observation per strain and no background strain effect. The &lt;strong&gt;[middle]&lt;/strong&gt; row corresponds to five replicates per strain and no background strain effect. For the &lt;strong&gt;[bottom row]&lt;/strong&gt;, five replicates are observed and the QTL effect size and background strain effect size sum to 50%, thus penalizing smaller QTL more harshly. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC. The columns, left to right, correspond to two, three, and eight functional alleles. Closed circles represent power estimates that were directly assessed, whereas open circles were interpolated. Simulations define allele effect size according to Definition B (see Methods).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F3.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1707/F3.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493021" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-55" class="first-child">Power curves by number of CC strains. Results are stratified by a number of replicates, background strain effect size, and the number of functional alleles. The <strong>[top]</strong> row is based on a single observation per strain and no background strain effect. The <strong>[middle]</strong> row corresponds to five replicates per strain and no background strain effect. For the <strong>[bottom row]</strong>, five replicates are observed and the QTL effect size and background strain effect size sum to 50%, thus penalizing smaller QTL more harshly. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC. The columns, left to right, correspond to two, three, and eight functional alleles. Closed circles represent power estimates that were directly assessed, whereas open circles were interpolated. Simulations define allele effect size according to Definition B (see Methods).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493160/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493160/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">QTL mapping power in the Collaborative Cross based on QTL effect sizes in a balanced population (Definition B)</span><div class="sb-div caption-clear"></div></div></div><div id="sec-22" class="subsection"><h3>Large effect QTL usually detected by 50 or more strains</h3><p id="p-58">As a baseline for describing mapping power in the CC, an experiment using one replicate (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-67.gif"/></noscript></span></span>) of all 72 strains is well-powered to detect QTL explaining &gt;40% of phenotypic variance but moderately or low powered for QTL explaining 30% or less (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Specifically, assuming eight functional alleles, there is 96.4% power to detect a 50% QTL, 79.2% for a 40% QTL, 44.1% for a 30% QTL, and 12.4% for a 20% QTL.</p><p id="p-59">More broadly, simulations across different allele effect types and numbers of strains showed that studies without replicates and with large numbers of strains (&gt;50) were found to be well-powered to detect large effect QTL (&gt;40%) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> <strong>[top]</strong>).</p><p id="p-60">Identifying smaller effect QTL should be feasible, however, using replicates. Replicates improve power by reducing the individual noise variance; as such the extent of the power improvement diminishes as more variance is attributable to background strain effects than noise. Assuming no background strain effect, and using 50 strains, we determined the power to detect a 20% effect-size QTL with a single replicate was near zero; with 5 replicates it approached 80%. Detecting QTL with effect sizes <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-68.gif"/></noscript></span></span>, however, was challenging. For example, achieving 80% power to detect an effect size of 10% when all 72 CC strains were used required more than 5 replicates per strain (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> <strong>[middle right]</strong>). Moreover, assuming a background strain effect, as would be expected with a complex trait, can reduce the QTL mapping power of small effect QTL substantially (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> <strong>[bottom]</strong>).</p></div><div id="sec-23" class="subsection"><h3>Additional strains improve power more than additional replicates</h3><p id="p-61">We investigated the relationship between power and the total number of mice, evaluating whether power gains were greater with additional CC strains or additional replicate observations. Power was interpolated over a grid of values for number of replicates and total number of mice from simulations based on a single observation per strain (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This showed that additional CC strains improved mapping power more than additional replicates; this is indicated by higher power values for lower numbers of replicates while holding number of mice constant (see Figure 5, bordered vertical section at 250 mice).</p></div><div id="sec-24" class="subsection"><h3>Location error of detected QTL</h3><p id="p-62">To obtain an approximation of mapping resolution, for all true positive detections we recorded the location error, or the genomic distance between simulated and detected QTL. The mean and the 95% quantile of the location error are reported as stabilized estimates for different numbers of strains and QTL effect sizes, but averaged over all other conditions, in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. (The stabilization procedure is described in <strong>Methods</strong>; raw, unstabilized estimates provided Figure S3.) The location error statistics require careful interpretation: for a detection to be classed as a true positive it had to be within 5Mb of the simulated QTL; therefore, location error was artificially capped at 5Mb. Poor performance thus corresponds to when that location seems uniformly (and therefore arbitrarily) distributed over the ±5Mb interval, that is, having a mean of 2.5Mb and a 95% quantile of 4.8Mb.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The mean (A) and 95% quantile (B) of location error, the distance in Mb between the detected and simulated QTL, by effect size and number of strains for 1,000 simulations of each setting. The simulations define QTL effect size based on Definition B with an eight allele QTL, and only a single observation per strain. Cells are colored red to white with decreasing mean and blue to white with decreasing 95% quantile. Regularization of the means and 95% quantile was accomplished through averaging the observed results with pseudo-counts; see Figure S3 for the raw measurements. Increasing the number of strains reduces the location error, both in terms of the mean and 95% quantile, more so than QTL effect size, also shown in Figure S6. The maximum possible location error was 5Mb due to the 10Mb window centered around the true QTL position used for detecting QTL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The mean (A) and 95% quantile (B) of location error, the distance in Mb between the detected and simulated QTL, by effect size and number of strains for 1,000 simulations of each setting. The simulations define QTL effect size based on Definition B with an eight allele QTL, and only a single observation per strain. Cells are colored red to white with decreasing mean and blue to white with decreasing 95% quantile. Regularization of the means and 95% quantile was accomplished through averaging the observed results with pseudo-counts; see Figure S3 for the raw measurements. Increasing the number of strains reduces the location error, both in terms of the mean and 95% quantile, more so than QTL effect size, also shown in Figure S6. The maximum possible location error was 5Mb due to the 10Mb window centered around the true QTL position used for detecting QTL.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F4.medium.gif" width="422" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1707/F4.medium.gif" width="422" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493089" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-63" class="first-child">The mean (A) and 95% quantile (B) of location error, the distance in Mb between the detected and simulated QTL, by effect size and number of strains for 1,000 simulations of each setting. The simulations define QTL effect size based on Definition B with an eight allele QTL, and only a single observation per strain. Cells are colored red to white with decreasing mean and blue to white with decreasing 95% quantile. Regularization of the means and 95% quantile was accomplished through averaging the observed results with pseudo-counts; see Figure S3 for the raw measurements. Increasing the number of strains reduces the location error, both in terms of the mean and 95% quantile, more so than QTL effect size, also shown in Figure S6. The maximum possible location error was 5Mb due to the 10Mb window centered around the true QTL position used for detecting QTL.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap of QTL mapping power by number of replicates and total number of mice in the experiment. Power is based on a QTL effect size of 20%, no background strain effect, and two functional alleles, though varying these parameters does not affect the dynamic between number of strains and replicates. The gray diagonal lines represent fixed values of the number of CC strains, ranging from 10 to 70 in intervals of five. Holding the total number of mice fixed, power is reduced as the percentage of the sample that are replicates is increased. This is illustrated with a cutout band centered on 250 mice, where power is lower at the top of the band when replicate mice are a relatively higher proportion of the total number of mice." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heatmap of QTL mapping power by number of replicates and total number of mice in the experiment. Power is based on a QTL effect size of 20%, no background strain effect, and two functional alleles, though varying these parameters does not affect the dynamic between number of strains and replicates. The gray diagonal lines represent fixed values of the number of CC strains, ranging from 10 to 70 in intervals of five. Holding the total number of mice fixed, power is reduced as the percentage of the sample that are replicates is increased. This is illustrated with a cutout band centered on 250 mice, where power is lower at the top of the band when replicate mice are a relatively higher proportion of the total number of mice.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F5.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/5/1707/F5.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493141" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-64" class="first-child">Heatmap of QTL mapping power by number of replicates and total number of mice in the experiment. Power is based on a QTL effect size of 20%, no background strain effect, and two functional alleles, though varying these parameters does not affect the dynamic between number of strains and replicates. The gray diagonal lines represent fixed values of the number of CC strains, ranging from 10 to 70 in intervals of five. Holding the total number of mice fixed, power is reduced as the percentage of the sample that are replicates is increased. This is illustrated with a cutout band centered on 250 mice, where power is lower at the top of the band when replicate mice are a relatively higher proportion of the total number of mice.</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">Location error was improved (reduced) by increasing the number of strains, increasing the QTL effect size, or both. In particular, as with power, location error was improved by increasing the number of strains even when while holding the total number of mice constant (Figure S4), consistent with mapping resolution being improved by an increased number of recombination events in the QTL region. Distributions of raw location error, stratified by levels of the number of strains, the number of functional alleles, and the QTL effect size can be found in Figure S6.</p></div><div id="sec-25" class="subsection"><h3>False positive rate</h3><p id="p-66">The FPR for the QTL power simulations was estimated as the percentage of scans (per setting) that produced a statistically significant signal on a chromosome without a QTL, shown in Figure S2. As expected, FPR was not elevated from 5% when the strain effects were simulated independently, as the effects were exchangeable by construction. The FPR did not vary with the number of strains or the number of alleles.</p><p id="p-67">In additional null simulations that included strain effects that were correlated due to realized genomic similarity, QTL scans assuming independent strain effects (and thus, exchangeability) had elevated FPR (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and Table S2). Using all 72 CC strains, the FPR varied from a maximum of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-69.gif"/></noscript></span></span> when strain effects explain all variability to the well controlled FPR of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-70.gif"/></noscript></span></span> when the strain effects were relatively small. Omitting CC059, one of the highly-related cousin strains (CC053 and CC059), because of its obvious violation of equal relatedness, reduced the FPR, although it was still elevated (<span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-71.gif"/></noscript></span></span> for maximum strain effect). This demonstrates that, when strain effects are large relative to individual error (<em>i.e.</em>, highly heritable trait, or the use of many replicates), failure to account for population structure due to realized imbalance in founder contributions can increase the risk of false positives.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The FPR increases due to population structure among the realized genomes of the CC strains in the presence of a background strain effect and no QTL. Curves are based on 10,000 simulations for each setting of strain effect and strain sample, based on a single observation per strain. The inflation in FPR is greater for all 72 CC strains, which includes two closely related cousin strains (CC051 and CC059). Removing CC059 reduces the inflation in FPR (gray line). The dashed red line marks the specified type I error rate of 0.05, which is approximately met as expected when no strain effect is simulated, as in Figure S2. Table S2 reports the specific FPR values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The FPR increases due to population structure among the realized genomes of the CC strains in the presence of a background strain effect and no QTL. Curves are based on 10,000 simulations for each setting of strain effect and strain sample, based on a single observation per strain. The inflation in FPR is greater for all 72 CC strains, which includes two closely related cousin strains (CC051 and CC059). Removing CC059 reduces the inflation in FPR (gray line). The dashed red line marks the specified type I error rate of 0.05, which is approximately met as expected when no strain effect is simulated, as in Figure S2. Table S2 reports the specific FPR values.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F6.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/5/1707/F6.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492966" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-68" class="first-child">The FPR increases due to population structure among the realized genomes of the CC strains in the presence of a background strain effect and no QTL. Curves are based on 10,000 simulations for each setting of strain effect and strain sample, based on a single observation per strain. The inflation in FPR is greater for all 72 CC strains, which includes two closely related cousin strains (CC051 and CC059). Removing CC059 reduces the inflation in FPR (gray line). The dashed red line marks the specified type I error rate of 0.05, which is approximately met as expected when no strain effect is simulated, as in Figure S2. Table S2 reports the specific FPR values.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Beavis effect</h3><p id="p-69">It is an expected feature of QTL mapping studies that estimates of QTL effect size, when calculated for detected QTL only, will be biased upwards. This phenomenon, known as the Beavis effect, is a form of selection bias and as such is expected to be most extreme when the selection involved is most severe, that is, under low power conditions, <em>e.g.</em>, when detection rates are low and/or estimates have high variance.</p><p id="p-70">We explored the Beavis effect in our simulations. Assuming a one-replicate (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-72.gif"/></noscript></span></span>) experiment, we found that, for example, the estimated effect size of a simulated 20% QTL was inflated by threefold when mapping in 40 CC strains, and by twofold when mapped in 72 CC strains. More generally, and as expected, the Beavis effect was reduced with larger numbers of strains and larger QTL effect sizes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Beavis effect (inflation of QTL effect size estimates) is more pronounced with smaller simulated QTL effect sizes and reduced numbers of strains. For different settings of numbers of strains (40, 50, 60, 72) and simulated QTL effect sizes (20%, 30%, 40%, 50%, 60%, 70%), black dots plot the ratio of the estimated effect size at a detected QTL peak to the effect size that was simulated at the true QTL locus. Out of 1,000 simulations under each setting, only successful detections are shown. Black diamonds represents the mean ratio for a category; horizontal red dashed line marks a ratio of 1, when QTL effect size estimates are unbiased (i.e., no Beavis effect)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The Beavis effect (inflation of QTL effect size estimates) is more pronounced with smaller simulated QTL effect sizes and reduced numbers of strains. For different settings of numbers of strains (40, 50, 60, 72) and simulated QTL effect sizes (20%, 30%, 40%, 50%, 60%, 70%), black dots plot the ratio of the estimated effect size at a detected QTL peak to the effect size that was simulated at the true QTL locus. Out of 1,000 simulations under each setting, only successful detections are shown. Black diamonds represents the mean ratio for a category; horizontal red dashed line marks a ratio of 1, when QTL effect size estimates are unbiased (&lt;em&gt;i.e.&lt;/em&gt;, no Beavis effect).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F7.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/5/1707/F7.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492975" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-71" class="first-child">The Beavis effect (inflation of QTL effect size estimates) is more pronounced with smaller simulated QTL effect sizes and reduced numbers of strains. For different settings of numbers of strains (40, 50, 60, 72) and simulated QTL effect sizes (20%, 30%, 40%, 50%, 60%, 70%), black dots plot the ratio of the estimated effect size at a detected QTL peak to the effect size that was simulated at the true QTL locus. Out of 1,000 simulations under each setting, only successful detections are shown. Black diamonds represents the mean ratio for a category; horizontal red dashed line marks a ratio of 1, when QTL effect size estimates are unbiased (<em>i.e.</em>, no Beavis effect).</p><div class="sb-div caption-clear"></div></div></div><p id="p-72">These results also imply that the Beavis effect would be reduced by use of replicates, at least to the extent that replicates boost effective QTL effect size. For example, consider again the mapping of a 20% QTL effect in 40 strains, which with <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-73.gif"/></noscript></span></span> replicates implies threefold effect size inflation. Although this inflation could be reduced to twofold by increasing the number of strains to 72, the same reduction could be achieved using replicates: assuming no background strain effect, increasing replicates to a theoretical <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-74.gif"/></noscript></span></span> (so as to give a total sample size of <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-75.gif"/></noscript></span></span>) would boost the QTL effect size to an effective ≈31% (according to <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a>) and, as shown in <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, have approximately the same result. The ability of replicates to reduce the Beavis effect, however, will diminish to the extent that there is a significant background strain effect, following the general relationship of replicates to QTL effect size described in <a id="xref-disp-formula-4-3" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a>.</p></div><div id="sec-27" class="subsection"><h3>Allele frequency imbalance reduces power</h3><p id="p-73">For a fixed set of QTL allele effects, it is expected that power will always be greatest when allele frequencies are balanced. Accordingly, when QTL effect size was defined in terms of the variance that would be explained in a theoretical population with balanced allele frequencies (Definition B), deviations from balance in the mapping population—either from imbalance in functional alleles among the founders or imbalance of the founders among the CC strains—inevitably reduce power (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). This reduction in power under Definition B is most evident for bi-allelic QTL (pink), in which the potential imbalance in allelic series is most extreme, namely when a single founder carries one functional allele and the other seven possess the alternative allele (7v1).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL effect sizes are in reference to a population, though effect size in the specific mapping population will determine the mapping power. Consider two populations as examples: the mapping population (definition DAMB) and a population balanced in the functional alleles (definition B). (A) QTL effect size distributions based on 10,000 simulations of the QTL for 72 strains. Using definition B, the effect sizes for the mapping population for two alleles is pink and eight alleles is red. Using definition DAMB, the effect sizes in the balanced population for two alleles is light blue and eight alleles is dark blue. Horizontal lines within the violin plots represent the 25th, 50th, and 75th quantiles from the estimated densities. Gray dots represent actual data points. (B) Power curves corresponding to the previously described settings of alleles and QTL effect size definitions. Power curves are estimated from 1,000 simulations per number of strains for a 50% QTL, no background strain effect, and a single observation per strain. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL effect sizes are in reference to a population, though effect size in the specific mapping population will determine the mapping power. Consider two populations as examples: the mapping population (definition DAMB) and a population balanced in the functional alleles (definition B). (A) QTL effect size distributions based on 10,000 simulations of the QTL for 72 strains. Using definition B, the effect sizes for the mapping population for two alleles is pink and eight alleles is red. Using definition DAMB, the effect sizes in the balanced population for two alleles is light blue and eight alleles is dark blue. Horizontal lines within the violin plots represent the 25&lt;sup&gt;th&lt;/sup&gt;, 50&lt;sup&gt;th&lt;/sup&gt;, and 75&lt;sup&gt;th&lt;/sup&gt; quantiles from the estimated densities. Gray dots represent actual data points. (B) Power curves corresponding to the previously described settings of alleles and QTL effect size definitions. Power curves are estimated from 1,000 simulations per number of strains for a 50% QTL, no background strain effect, and a single observation per strain. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F8.medium.gif" width="440" height="189"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/5/1707/F8.medium.gif" width="440" height="189"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493195" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-74" class="first-child">QTL effect sizes are in reference to a population, though effect size in the specific mapping population will determine the mapping power. Consider two populations as examples: the mapping population (definition DAMB) and a population balanced in the functional alleles (definition B). (A) QTL effect size distributions based on 10,000 simulations of the QTL for 72 strains. Using definition B, the effect sizes for the mapping population for two alleles is pink and eight alleles is red. Using definition DAMB, the effect sizes in the balanced population for two alleles is light blue and eight alleles is dark blue. Horizontal lines within the violin plots represent the 25<sup>th</sup>, 50<sup>th</sup>, and 75<sup>th</sup> quantiles from the estimated densities. Gray dots represent actual data points. (B) Power curves corresponding to the previously described settings of alleles and QTL effect size definitions. Power curves are estimated from 1,000 simulations per number of strains for a 50% QTL, no background strain effect, and a single observation per strain. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC.</p><div class="sb-div caption-clear"></div></div></div><p id="p-75">Conversely, when the QTL effect size is defined in terms of variance explained in the mapping population (Definition DAMB, which is similar to an <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-76.gif"/></noscript></span></span> measure), power remains constant across different allelic series and degrees of balance. Although note that this definition carries with it the (possibly unrealistic) implication that allele effects vary depending what population they are in.</p><p id="p-76">When averaged over many allelic series, QTL mapping power based on Definition B is reduced relative to Definition DAMB, with the greatest reduction occurring for bi-allelic QTL (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8 B</a>). Though this modest reduction in power may seem to suggest that simulating with respect to a balanced population (Definition B) <em>vs.</em> the mapping population (Definition DAMB) is unimportant in terms of designing a robust mapping experiment in the CC, we reiterate the value of using Definition B. Specifically, simulating with respect to Definiton DAMB is overly optimistic regarding mapping power for QTL with imbalanced allelic series.</p><p id="p-77">We performed additional simulations to evaluate bi-allelic QTL in more detail, these being more prone to drastic imbalance under Definition B. All 127 possible bi-allelic series are visualized as a grid in <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9A</a>, ordered from balance and high power to imbalance and low power. The corresponding power estimates are shown in <a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9B</a>. Power was maximized when the bi-allelic series is balanced (4v4; 35/127 possible allelic series) and minimized when imbalanced (7v1; 8/127 possible allelic series). Uniform sampling of bi-allelic series, the approach in the more general simulations described earlier, slightly reduced power relative to balanced 4v4 allelic series due to averaging over many cases of balance and some cases of extreme imbalance. These latter, more focused simulations highlight the extent that the reduction in QTL effect size, and thus mapping power, when simulating based on Definition B, is highly dependent on the allelic series. This could be of particular importance when considering QTL that result from a causal variant inherited from a wild-derived founder, such as CAST, which will present as both imbalanced and bi-allelic.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The balance of the allelic series for QTL with two functional alleles, and its effect on QTL mapping power. (A) The 127 possible allelic series for a bi-allelic QTL, categorized by the balance in the distribution of alleles among the CC founder strains, and ordered with balanced allelic series at the top and imbalanced at the bottom. (B) Power curves comparing three different sampling approaches for the allelic series with two functional alleles, for populations simulated to have a QTL effect size of 50% in a balanced theoretical population, with a single observation per CC strain. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-637570751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The balance of the allelic series for QTL with two functional alleles, and its effect on QTL mapping power. (A) The 127 possible allelic series for a bi-allelic QTL, categorized by the balance in the distribution of alleles among the CC founder strains, and ordered with balanced allelic series at the top and imbalanced at the bottom. (B) Power curves comparing three different sampling approaches for the allelic series with two functional alleles, for populations simulated to have a QTL effect size of 50% in a balanced theoretical population, with a single observation per CC strain. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1707/F9.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/9/5/1707/F9.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1707/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493006" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-78" class="first-child">The balance of the allelic series for QTL with two functional alleles, and its effect on QTL mapping power. (A) The 127 possible allelic series for a bi-allelic QTL, categorized by the balance in the distribution of alleles among the CC founder strains, and ordered with balanced allelic series at the top and imbalanced at the bottom. (B) Power curves comparing three different sampling approaches for the allelic series with two functional alleles, for populations simulated to have a QTL effect size of 50% in a balanced theoretical population, with a single observation per CC strain. The horizontal red dotted line marks 80% power. The vertical black dashed line marks 58 strains, which is currently the number of unrelated strains available from UNC.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-28"><h2 class="">Discussion</h2><p id="p-79">Now that the CC strains have been largely finalized, it is possible to investigate more deeply how, in potential mapping experiments, power is affected by factors such as the number of strains, the number of replicates, and the allelic series at the QTL. We find that the CC can powerfully map large effect QTL (≥ 50%) with single observations of 60 or more strains. Through the use of replicates, the power to map QTL can be greatly improved, potentially mapping QTL ≥ 20% in 60 strains with 5 replicates per strain with no background strain effect. To guide the design of new CC experiments, we provide broad power curves and tables in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a> and S1.</p><p id="p-80">The power calculations described here take advantage of realized CC genomes, allowing the power estimates to be highly specific to the available strains but also necessarily restricting the number that can be used. This differs from the simulations of <a id="xref-ref-78-9" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a>, which primarily focused on comparing potential breeding designs with numbers of strains that far exceed (500-1,000) the population now realized (50-70). As such, directly comparing these studies is challenging. The closest comparison case is for a 5% QTL with 45% background strain effect in 100 simulated strains with 10 replicates, for which <a id="xref-ref-78-10" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a> estimates 4% power. Matching those settings with the exception of 72 strains instead of 100, and using the DAMB definition of QTL effect size, we find 0.4% power. The relatively lower power with the realized data likely reflects both reduction in the number of strains by 28% (72 to 100) and the deviations from an ideally-randomized population, such as the observed reduction in contributions from the CAST and PWK founders (<a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Srivastava <em>et al.</em> 2017</a>). This emphasizes the challenge in projecting the results from <a id="xref-ref-78-11" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a> into the realized population for the purpose of designing an experiment.</p><p id="p-81">We did not attempt power simulations with epistatic QTL or phenotypes with large background strain effect. From the results of <a id="xref-ref-78-12" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a>, it was clear that mapping studies in the realized CC, even with replicates, would not be well-powered in those contexts. Nonetheless, despite the reduced number of strains, we found that successful mapping experiments can be designed in the realized CC, particularly by harnessing the ability of genetic replicates to reduce random noise, as well as within the context of molecular phenotypes such as gene expression for which the genetic architecture is relatively simple.</p><div id="sec-29" class="subsection"><h3>Interpreting QTL effect sizes</h3><p id="p-82">Our simulations suggest that QTL mapping experiments in the CC are well-powered for large-effect QTL, in the neighborhood of 20–40%, depending on the number of strains and replicates, and the presence of a background strain effect. As such, it is useful to provide some context for what traits might plausibly yield QTL of this size. That said, we note that comparisons of reported estimates of QTL effect size should be interpreted with caution since they vary across different traits and model systems, are calculated under different experimental protocols that may vary in levels of noise, numbers of strains and/or replicates, and may be estimated by different analysis conditions (statistical methods, data transformations, etc.). And ultimately, these estimates are subject to overestimation due to both the aforementioned Beavis effect and reporting bias.</p><p id="p-83">Multiple studies in the pre-CC, which had more strains than the realized CC population, have reported QTL effect sizes for a variety of traits. <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Philip <em>et al.</em> (2011)</a> report effect sizes for 17 QTL for 102 morphological and behavioral traits in 235 incipient CC strains, ranging from 5.3% (tail-clip latency) to 26% (red cell distribution width). <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Durrant <em>et al.</em> (2011)</a> mapped seven QTL for susceptibility to <em>Aspergillus fumigatus</em> infection in 371 mice from 66 strains, with effects ranging from 12.2–16.2%. <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gralinski <em>et al.</em> (2015)</a> identified four SARS susceptibility QTL in 140 strains with effect sizes between 21–26% (vascular cuffing, 21% and 26%; viral titer, 22%; eosinophilia, 26%).</p><p id="p-84">More closely mirroring the number of strains considered here, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Levy <em>et al.</em> (2015)</a> detected six strong QTL for traits related to trabecular bone microstructure using 160 mice from 31 strains, which ranged from 61–86%. In an ongoing project involving the mapping of expression QTL (eQTL) from RNA-seq data collected from three tissues of single individuals from 47 strains, 478-739 eQTL were detected at genome-wide significance, ranging in effect size from 60–90%. These results reiterate that QTL mapping studies in the CC are best suited for detection of large effect QTL, as are more common in molecular traits.</p><p id="p-85">In considering the above, it is useful to understand how this relates to effect sizes seen in humans, for which the CC is often used as a model system [eg, <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Rogala <em>et al.</em> (2014)</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Orgel <em>et al.</em> (2019)</a>]. In particular, human genome-wide association studies (GWASs), which often use much larger sample sizes, routinely report QTL with estimated effect sizes far smaller than is detectable in the CC. Nonetheless, there are reasons to expect effect sizes in the CC to be larger than in humans. Human GWASs are observational, and as such include many additional sources of noise, reducing QTL effect sizes relative to what would be possible in more tightly-controlled experimental designs. Experimental populations will also have larger QTL effect sizes because: 1) they typically have more balanced allele frequencies; 2) in the case of panels of RILs, such as the CC, they are homozygous across the genome, which increases the contrast in additive allele effects and thus boosts additive QTL effect size; and 3), again for RILs, they furnish biological replicates, which, as illustrated in <a id="xref-disp-formula-4-4" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a>, can increase effect size by reducing individual error.</p></div><div id="sec-30" class="subsection"><h3>Strains <em>vs.</em> replicates</h3><p id="p-86">When holding the total number of mice fixed, we found that adding more strains improves power and reduces location error to a greater degree than adding more replicates. Moreover, this inference was made in the absence of a background strain effect—given that replicates reduce individual-level variance but not strain-level variance, the presence of background effects would reduce the relative value of replicates yet further. These observations are consistent with the results of <a id="xref-ref-78-13" class="xref-bibr" href="#ref-78">Valdar <em>et al.</em> (2006a)</a> and established theoretical arguments (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Soller and Beckmann 1990</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Knapp and Bridges 1990</a>).</p><p id="p-87">Nonetheless, for many CC mapping experiments we predict that adding replicates will provide considerable value. First, for all but the most highly polygenic traits, mapping on the means of replicates, a strategy originally termed “replicated progeny” (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cowen 1988</a>) or “progeny testing” (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lander and Botstein 1989</a>), will always provide additional power. Indeed, with a limited number of strains available, and the possibility that all available strains are used, replicates may sometimes be the only way power can be further increased (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Belknap 1998</a>).</p><p id="p-88">Second, replicates provide not only an insurance policy against phenotyping errors, but also a way to average over batches and similar nuisance parameters (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Cowen 1988</a>), thus protecting against the negative consequences of gene by environment interactions while providing the opportunity for such interactions to be detected [<em>e.g.</em>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kafkafi <em>et al.</em> (2005</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2018</a>)].</p><p id="p-89">Third, replicates enable deeper phenotypic characterization and in particular measurement of strain-level phenotypes that are necessarily a function of multiple individuals. For example, treatment response phenotypes (<em>e.g.</em>, response to drug) are ideally defined in terms of counterfactual-like observations of drug-treated and vehicle-treated strain replicates [<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Festing (2010)</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> (2014)</a>] and recombinant inbred lines such as the CC are uniquely able to combine such definitions with QTL mapping [<em>e.g.</em>, <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Mosedale <em>et al.</em> (2017)</a> and also, in flies, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kislukhin <em>et al.</em> (2013)</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Najarro <em>et al.</em> (2015)</a>]. Similarly, strain-specific phenotypic variance ideally requires replicates (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Rönnegård and Valdar 2011</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ayroles <em>et al.</em> 2015</a>). We did not consider such elaborations here, but we expect the trade-off between number of strains <em>vs.</em> replicates will be more nuanced in such cases.</p></div><div id="sec-31" class="subsection"><h3>Population structure in the CC</h3><p id="p-90">Our simulations indicate that deviations from equal relatedness in the realized CC strains have introduced a degree of population structure that potentially increases the risk of false positives if not addressed, albeit to a far lesser extent than has been observed in traditional inbred strain association (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kang <em>et al.</em> 2008</a>). In particular, null simulations that assumed correlated strain effects due to genetic relatedness increased FPR for our mapping approach when the strain effect was large relative to individual error, as would be the case for a highly heritable polygenic trait or when using many replicates. This elevated FPR supports the use of QTL mapping approaches that account for the effect of genetic similarity on phenotypes, such as a linear mixed effect model (LMM) (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Kang <em>et al.</em> 2008</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lippert <em>et al.</em> 2011</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Zhou and Stephens 2012</a>), especially in the context of marginally significant QTL, which may not remain significant given a higher threshold that controls FPR more appropriately. Software packages that can fit the LMM specifically with CC data include our miQTL package (available on GitHub at <a href="https://github.com/gkeele/miqtl">https://github.com/gkeele/miqtl</a>) and R/qtl2 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Broman <em>et al.</em> 2019</a>).</p><p id="p-91">For the analyses reported here, a mixed effect model approach was not feasible owing to its increased computational burden (and in particular, its incompatibility with the computational shortcut in <strong>Appendix A</strong>). Instead, we simulated independent strain effects and employed a fixed effect mapping procedure due to its computational efficiency, especially when computing permutation-based significance thresholds. Nonetheless, the conclusions drawn in this study should be largely consistent with the use of a mixed effect model that correctly controls for correlated strain effects due to genetic relatedness.</p></div><div id="sec-32" class="subsection"><h3>Allelic series, and use of an eight allele mapping model</h3><p id="p-92">We found that the allelic series can strongly affect power through its influence on observed allele frequencies. Specifically, imbalanced bi-allelic QTL have significantly reduced mapping power when the sole alternative allele is rarely observed, whereas highly multi-allelic QTL are more easily detected because they will have multiple alleles observed within a given sample of strains.</p><p id="p-93">Regardless of the true allelic series at a QTL, which is unknown in practice, our statistical procedure assumed an eight allele model. For QTL with fewer functional alleles than founder strains, this assumption could reduce power due to the estimation of redundant allele effect parameters. Indeed, QTL consistent with a bi-allelic series have been more powerfully detected in some MPP studies using SNP association (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baud <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Keele <em>et al.</em> 2018</a>).</p><p id="p-94">Nonetheless, multi-allelic QTL (with more than two alleles) do occur. This has been seen, for example, <em>in cis</em>-regulation of gene expression that largely corresponds to the three subspecies lineages of <em>Mus musculus</em>, present in the CC (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Crowley <em>et al.</em> 2015</a>). Moreover, multi-allelic QTL will not be as powerfully detected through SNP association, as seen, for example, in <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aylor <em>et al.</em> (2011)</a>. SNP (or more generally, variant) association also poses additional challenges, such as how to handle regions of the genome (and variants) that are difficult to genotype, as well as the requirement of extensive quality control filtering to remove markers with low minor allele frequencies. These challenges are implicitly reduced in haplotype analysis.</p><p id="p-95">An ideal statistical procedure would formally model the unknown allelic series and their corresponding uncertainty. Though challenging, the development of alternative mapping strategies that specifically account for the allelic series is clearly an imperative methodological advance that would greatly benefit QTL analyses in MPPs with diverse founder alleles. That said, allelic series-aware approaches would likely be computationally expensive and poorly suited to simulation-based power analyses. Meanwhile, in the absence of more sophisticated approaches, the eight allele model, though potentially redundant, has several advantages over SNP association that suggest it will remain a useful (and maybe the default) tool for CC mapping, namely: it encompasses all possible simpler allelic series, implicitly models local epistasis, and, in reflecting the LD decay around detected QTL, more clearly delineates the limits of mapping resolution.</p></div><div id="sec-33" class="subsection"><h3>Inclusion of extinct CC strains in simulations</h3><p id="p-96">Our simulations included genomes from CC strains that are now extinct, and also did not include all the CC strains that are currently available. This discrepancy reflects the inherent challenge of maintaining a stable genetic population resource. RI panels such as the CC are an approximation to an ideal: they attempt to provide reproducible genomes that can be observed multiple times as well as across multiple studies; yet, as a biological population, those genomes are mutable, and through time will accumulate mutations and drift, and even potentially go extinct.</p><p id="p-97">Although the inclusion of genomes of extinct strains, or those that have drifted since the strains were genotyped, result in power calculations that do not perfectly correspond to the current CC population, they are preferable to simulated genomes, since they represent genomes that were viable at some point.</p></div><div id="sec-34" class="subsection"><h3>Future use and directions</h3><p id="p-98">Any analysis of power is subject to the assumptions underlying that analysis. One of the advantages of simulation is the ability to evaluate the impact of many of these assumptions, as well as the consideration of new scenarios by re-running the simulation under different settings, or by elaborating the simulation itself. We have attempted to make re-running the simulations under different settings straightforward for other researchers by developing a software package for this purpose. This package could be used to investigate highly-specialized questions, such as the power for specific combinations of CC strains or assessing how the power to detect QTL varies depending on genomic position. In future work, the simulation code itself could be expanded to investigate additional topics of interest, such as how power is influenced by variance heterogeneity or model mis-specification.</p></div><div id="sec-35" class="subsection"><h3>Conclusion</h3><p id="p-99">We used a focused simulation approach that incorporates realized CC genomes to provide more accurate estimates of QTL mapping power than were previously possible. As such, the results of our simulations provide tailored power calculations to aide the design of future QTL mapping experiments in the CC. Additionally, we evaluate how the balance of alleles at the QTL can strongly influence power to map QTL in the CC. We make available the R package SPARCC that we developed for running these simulations and analyses. It leverages an efficient model fitting approach in order to explore power in a level of detail that has previously been impractical, it is replicable, and it can be extended to user-specified questions of interest.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-123">This work was primarily supported by the National Institute of General Medical Sciences under awards R01-GM104125 and R35-GM127000 (to W.V) and the National Institute of Environmental Health Sciences under award R01-ES024965 (to S.N.P.K). Computing resources were generously provided by the University of North Carolina Information Technology Services.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-36"><h3>QR decomposition for fast regression</h3><p id="p-100">To maximize power to detect QTL while controlling the FPR, permutations to determine significance thresholds are needed, which is computationally expensive and thus the underlying regression functionality must be highly optimized. We accomplish this through the QR matrix decomposition, which we will describe briefly (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Venables and Ripley 2002</a>).</p><p id="p-101">Let <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-77.gif"/></noscript></span></span> be the <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-78.gif"/></noscript></span></span> design matrix included in <a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">Eq 3</a>, with <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-79.gif"/></noscript></span></span>. The solution for β from the least squares normal equations is <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-80.gif"/></noscript></span></span>. Through the QR decomposition, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-81.gif"/></noscript></span></span>, for which Q is an <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-82.gif"/></noscript></span></span> orthonormal matrix <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-83.gif"/></noscript></span></span> and R is a <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-84.gif"/></noscript></span></span> upper triangular matrix. With matrix algebra, it is fairly straightforward to show that <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-85.gif"/></noscript></span></span>, which is also more numerically stable than calculating <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-86.gif"/></noscript></span></span> through <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-87.gif"/></noscript></span></span>. After solving for <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-88.gif"/></noscript></span></span>, the residual sums of squares, and ultimately logP, can be rapidly calculated. Because our simulation approach involves regressing many permuted outcomes <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-89.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-90.gif"/></noscript></span></span> is a permutation matrix that re-orders <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-91.gif"/></noscript></span></span> randomly, on the same design matrices, computational efficiency can be vastly increased by pre-computing and saving the QR decompositions for all X.</p><p>Once the QR decomposition has been stored for a design matrix <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-92.gif"/></noscript></span></span>, where <em>j</em> indexes locus, it is highly computationally efficient to conduct additional tests for any y, which encompasses all permuted outcomes <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-93.gif"/></noscript></span></span>. If <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-94.gif"/></noscript></span></span> is the same across <em>S</em> simulations, the boost in computation can extend beyond permutations to samples of <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-95.gif"/></noscript></span></span>, as is the case when the set of CC strains is fixed. Thus, our R package SPARCC handles two cases: when the set of CC strains is fixed, and when the set varies.</p><ul class="list-unord " id="list-3"><li id="list-item-7"><p id="p-103">Fixed set of CC strains</p><ol class="list-ord " id="list-4"><li id="list-item-8"><p id="p-104">Store QR decompositions of <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-96.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-97.gif"/></noscript></span></span></p></li><li id="list-item-9"><p id="p-105">Run genome scans for <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-98.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-99.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-100.gif"/></noscript></span></span></p></li></ol></li><li id="list-item-10"><p id="p-106">Varied set of CC strains</p><ol class="list-ord " id="list-5"><li id="list-item-11"><p id="p-107">Store QR decompositions of <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-101.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-102.gif"/></noscript></span></span></p></li><li id="list-item-12"><p id="p-108">Run genome scans for <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-103.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-104.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-105.gif"/></noscript></span></span></p></li><li id="list-item-13"><p id="p-109">Repeat steps 1 and 2 for <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-106.gif"/></noscript></span></span></p></li></ol></li></ul><p id="p-110">Varying the set of CC strains increases computation time linearly with respect to <em>S</em>. If the investigators do not have a predefined set of strains, it is appropriate that this source of variability be incorporated into the power calculation.</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="section" id="sec-37"><h3>Computing environment and performance</h3><p id="p-111">We performed 1,000 simulations (in batches of 100) for each combination of the parameters, resulting in 8,400 individual jobs. These jobs were submitted in parallel to a distributed computing cluster (<a href="http://its.unc.edu/rc-services/killdevil-cluster/">http://its.unc.edu/rc-services/killdevil-cluster/</a>). Runtime varied depending on parameter settings and the hardware used, with the longest jobs taking approximately seven hours to complete.</p></div></div><div class="section app" id="app-3"><h2 class="">Appendix C</h2><div class="section" id="sec-38"><h3>CC strains</h3><p id="p-112">This study used haplotype mosiac data available from <a href="http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs">http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs</a> for the following 72 CC strains: CC001, CC002, CC003, CC004, CC005, CC006, CC007, CC008, CC009, CC010, CC011, CC012, CC013, CC014, CC015, CC016, CC017, CC018, CC019, CC020, CC021, CC022, CC023, CC024, CC025, CC026, CC027, CC028, CC029, CC030, CC031, CC032, CC033, CC034, CC035, CC036, CC037, CC038, CC039, CC040, CC041, CC042, CC043, CC044, CC045, CC046, CC047, CC048, CC049, CC050, CC051, CC052, CC053, CC054, CC055, CC056, CC057, CC058, CC059, CC060, CC061, CC062, CC063, CC065, CC068, CC070, CC071, CC072, CC073, CC074, CC075, CC076. This includes two strains CC051 and CC059 that are derived from the same breeding funnel and thus more closely related than typical pairs of CC strains.</p><p id="p-113">Of the the 72 CC strains used here, 54 are among a larger set of 59 that are currently maintained and distributed by UNC (personal correspondence with Darla Miller, UNC). These 54/59 strains are CC001, CC002, CC003, CC004, CC005, CC006, CC007, CC008, CC009, CC010, CC011, CC012, CC013, CC015, CC016, CC017, CC019, CC021, CC023, CC024, CC025, CC026, CC027, CC029, CC030, CC031, CC032, CC033, CC035, CC036, CC037, CC038, CC039, CC040, CC041, CC042, CC043, CC044, CC045, CC046, CC049, CC051, CC053, CC055, CC057, CC058, CC059, CC060, CC061, CC062, CC065, CC068, CC071, CC072. The remaining 5/59 strains (CC078, CC079, CC080, CC081, CC083) lacked haplotype mosaic data at the time of simulation and so were not included (although note that their mosaics have since been added to the website).</p></div></div><div class="section app" id="app-4"><h2 class="">Appendix D</h2><div class="section" id="sec-39"><h3>Additive model and allelic series matrices</h3><div id="sec-40" class="subsection"><h4>Additive matrix</h4><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-107.gif"/></noscript></span></span><p id="p-114">We can use matrices to specify simplifying linear combinations of the 36 diplotypes. The additive model matrix A is commonly used, and we use it here. Post-multiplication of the diplotype design matrix D with the A rotates the diplotypes at the locus to dosages of the founder haplotypes. If there is no uncertainty on the diplotype identities, <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-108.gif"/></noscript></span></span> will be the matrix of founder haplotype counts at the locus.</p></div><div id="sec-41" class="subsection"><h4>Allelic series matrices</h4><p id="p-115">We explore the influence of the allelic series on QTL mapping power via the simulation procedure. The QTL mapping procedure estimates separate parameters for each founder, though in reality there are likely fewer functional alleles. We denote the <em>q</em><sup>th</sup> functional allele as <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-109.gif"/></noscript></span></span>. The allelic series can be sampled and encoded in the M.ID argument within the sim.CC.data() function of SPARCC. Below are examples of allelic series, including balanced (4v4) and unbalanced (7v1) bi-allelic series, as well as allelic series with multiple alleles.</p></div><div id="sec-42" class="subsection"><h4>Allelic series with eight alleles (maximum)</h4><p id="p-116"><kbd>M.ID = “0,1,2,3,4,5,6,7”</kbd></p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-110.gif"/></noscript></span></span></div><div id="sec-43" class="subsection"><h4>Example balanced (4v4) bi-allelic series</h4><p id="p-117"><kbd>M.ID = “0,1,0,0,1,0,1,1”</kbd></p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-111.gif"/></noscript></span></span><p id="p-118"><kbd>M.ID = “0,1,1,1,0,0,1,0”</kbd></p><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-112.gif"/></noscript></span></span></div><div id="sec-44" class="subsection"><h4>Example unbalanced (7v1) bi-allelic series</h4><p id="p-119"><kbd>M.ID = “0,0,0,0,0,1,0,0”</kbd></p><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-113.gif"/></noscript></span></span><p id="p-120"><kbd>M.ID = “0,1,0,0,0,0,0,0”</kbd></p><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-114.gif"/></noscript></span></span></div><div id="sec-45" class="subsection"><h4>Example tri-allelic series</h4><p id="p-121"><kbd>M.ID = “0,0,1,2,2,0,2,0”</kbd></p><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-115.gif"/></noscript></span></span><p id="p-122"><kbd>M.ID = “0,1,0,0,0,0,2,2”</kbd></p><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1707/embed/mml-math-116.gif"/></noscript></span></span></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-124">Supplemental material available at <a href="https://doi.org/10.25387/g3.7409405">https://doi.org/10.25387/g3.7409405</a>.</p></li><li class="fn" id="fn-5"><p id="p-125"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-04"><span class="received-label">Received </span>December 4, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-20"><span class="accepted-label">Accepted </span>March 20, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Keele <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.1" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of complex traits in the emerging Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>–<span class="cit-lpage">1222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.111310.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.2" data-doi="10.1073/pnas.1503830112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skutt-Kakaria</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Behavioral idiosyncrasy reveals genetic control of phenotypic variability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">6706</span>–<span class="cit-lpage">6711</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1503830112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503830112%26rft_id%253Dinfo%253Apmid%252F25953335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzIxLzY3MDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.3" data-doi="10.1038/ng.2644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baud</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guryev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stridh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Combined sequence-based and genetic mapping analysis of complex traits in outbred rats.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">775</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2644</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D767%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2644%26rft_id%253Dinfo%253Apmid%252F23708188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23708188&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.5.1707.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> The power and deceit of qtl experiments: lessons from comparative qtl studies. In <em>Proceedings of the forty-ninth annual corn and sorghum industry research conference</em>, pp. <span class="cit-fpage">250</span>–<span class="cit-lpage">266</span>, Washington, DC.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.5" data-doi="10.1023/A:1021404714631"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belknap</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Effect of within-strain sample size on GTL detection and mapping using recombinant inbred mouse strains.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1021404714631</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DBelknap%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D38%26rft.atitle%253DEffect%2Bof%2Bwithin-strain%2Bsample%2Bsize%2Bon%2BQTL%2Bdetection%2Band%2Bmapping%2Busing%2Brecombinant%2Binbred%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1021404714631%26rft_id%253Dinfo%253Apmid%252F9573644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1021404714631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9573644&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072994900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.6" data-doi="10.1007/BF02359892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belknap</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Toole</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helms</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crabbe</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Type I and type II error rates for quantitative trait loci (QTL) mapping studies using recombinant inbred mouse strains.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02359892</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DBelknap%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D26%26rft.issue%253D2%26rft.spage%253D149%26rft.epage%253D160%26rft.atitle%253DType%2BI%2Band%2Btype%2BII%2Berror%2Brates%2Bfor%2Bquantitative%2Btrait%2Bloci%2B%2528QTL%2529%2Bmapping%2Bstudies%2Busing%2Brecombinant%2Binbred%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02359892%26rft_id%253Dinfo%253Apmid%252F8639150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02359892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8639150&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UE22600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.7" data-doi="10.1534/genetics.116.198499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouchet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olatoye</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marla</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perumal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesso</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increased Power To Dissect Adaptive Traits in Global Sorghum Diversity Using a Nested Association Mapping Population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">585</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198499</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198499%26rft_id%253Dinfo%253Apmid%252F28592497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.8" data-doi="10.1534/genetics.118.301595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furlotte</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">R/qtl2: Software for Mapping Quantitative Trait Loci with High-Dimensional Data and Multiparent Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301595%26rft_id%253Dinfo%253Apmid%252F30591514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjExLzIvNDk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.9" data-doi="10.1214/ss/1009213286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DasGupta</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Interval Estimation for a Binomial Proportion.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/ss/1009213286</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D16%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1009213286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1009213286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000171504900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.10" data-doi="10.1007/s00335-008-9135-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branstetter</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galloway</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">389</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-008-9135-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChesler%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D382%26rft.epage%253D389%26rft.atitle%253DThe%2BCollaborative%2BCross%2Bat%2BOak%2BRidge%2BNational%2BLaboratory%253A%2Bdeveloping%2Ba%2Bpowerful%2Bresource%2Bfor%2Bsystems%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9135-8%26rft_id%253Dinfo%253Apmid%252F18716833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9135-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18716833&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.11" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Collaborative Cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1104-1133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.12" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132639</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.13" data-doi="10.1007/BF00258045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cowen</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The use of replicated progenies in marker-based mapping of QTL’s.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">862</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00258045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D75%26rft.spage%253D857%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00258045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00258045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.14" data-doi="10.1534/genetics.113.156901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crowley</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenarcic</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of adverse reactions to haloperidol in a mouse diallel: a drug-placebo experiment and Bayesian causal analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">347</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.156901</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156901%26rft_id%253Dinfo%253Apmid%252F24240528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMzIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.15" data-doi="10.1038/ng.3222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crowley</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhabotynsky</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pakatci</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">360</span>. (erratum: Nat. Genet. 47: 690).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3222</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D353%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3222%26rft_id%253Dinfo%253Apmid%252F25730764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25730764&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.16" data-doi="10.1186/s13059-015-0716-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dell’Acqua</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattonaro</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coppens</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic properties of the MAGIC maize population: a new platform for high definition QTL mapping in Zea mays.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0716-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0716-z%26rft_id%253Dinfo%253Apmid%252F26357913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0716-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26357913&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Permutation tests for multiple loci affecting a quantitative character.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDoerge%26rft.auinit1%253DR.%2BW.%26rft.volume%253D142%26rft.issue%253D1%26rft.spage%253D285%26rft.epage%253D294%26rft.atitle%253DPermutation%2BTests%2Bfor%2BMultiple%2BLoci%2BAffecting%2Ba%2BQuantitative%2BCharacter%26rft_id%253Dinfo%253Apmid%252F8770605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQyLzEvMjg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.18" data-doi="10.1534/genetics.117.300211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donoghue</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Livraghi-Butrico</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McFadden</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Identification of trans Protein QTL for Secreted Airway Mucins in Mice and a Causal Role for Bpifb1.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">801</span>–<span class="cit-lpage">812</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300211%26rft_id%253Dinfo%253Apmid%252F28851744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA3LzIvODAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.19" data-doi="10.1086/423738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudbridge</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koeleman</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Efficient Computation of Significance Levels for Multiple Associations in Large Studies of Correlated Data, Including Genomewide Association Studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/423738</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DDudbridge%26rft.auinit1%253DF.%26rft.volume%253D75%26rft.issue%253D3%26rft.spage%253D424%26rft.epage%253D435%26rft.atitle%253DEfficient%2Bcomputation%2Bof%2Bsignificance%2Blevels%2Bfor%2Bmultiple%2Bassociations%2Bin%2Blarge%2Bstudies%2Bof%2Bcorrelated%2Bdata%252C%2Bincluding%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F423738%26rft_id%253Dinfo%253Apmid%252F15266393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/423738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266393&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223272200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.20" data-doi="10.1101/gr.118786.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tayem</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Collaborative Cross mice and their power to map host susceptibility to Aspergillus fumigatus infection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.118786.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118786.110%26rft_id%253Dinfo%253Apmid%252F21493779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.21" data-doi="10.1038/hdy.2010.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falke</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Power and false-positive rate in QTL detection with near-isogenic line libraries.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2010.87</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DFalke%26rft.auinit1%253DK.%2BC.%26rft.volume%253D106%26rft.issue%253D4%26rft.spage%253D576%26rft.epage%253D584%26rft.atitle%253DPower%2Band%2Bfalse-positive%2Brate%2Bin%2BQTL%2Bdetection%2Bwith%2Bnear-isogenic%2Bline%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.87%26rft_id%253Dinfo%253Apmid%252F20683482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20683482&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.22" data-doi="10.1371/journal.ppat.1003196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottomly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aicher</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Modeling Host Genetic Regulation of Influenza Pathogenesis in the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1003196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253De1003196%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003196%26rft_id%253Dinfo%253Apmid%252F23468633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23468633&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.23" data-doi="10.1177/0192623310373776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Festing</span>  <span class="cit-name-given-names">M. F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Inbred strains should replace outbred stocks in toxicology, safety testing, and drug development.</span> <abbr class="cit-jnl-abbrev">Toxicol. Pathol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1177/0192623310373776</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BPathol.%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0192623310373776%26rft_id%253Dinfo%253Apmid%252F20562325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0192623310373776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562325&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.5.1707.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Villena</span>  <span class="cit-name-given-names">F. P.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Inferring ancestry in admixed populations using microarray probe intensities. In <em>Proceedings of the ACM Conference on Bioinformatics</em>, <em>Computational Biology and Biomedicine - BCB ’12</em>, pp. <span class="cit-fpage">105</span>–<span class="cit-lpage">112</span>, <span class="cit-publ-loc">New York, New York, USA</span>, <span class="cit-publ-name">ACM Press</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/2382936.2382950</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.25" data-doi="10.1534/g3.114.013748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.013748</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1623%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.013748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.26" data-doi="10.1016/j.celrep.2017.10.093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mooney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choonoo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Extensive Homeostatic T Cell Phenotypic Variation within the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2313</span>–<span class="cit-lpage">2325</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2017.10.093</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D21%26rft.spage%253D2313%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2017.10.093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2017.10.093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.27" data-doi="10.1371/journal.pgen.1005504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome Wide Identification of SARS-CoV Susceptibility Loci Using the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005504%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005504%26rft_id%253Dinfo%253Apmid%252F26452100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26452100&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.28" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.1992.131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.29" data-doi="10.1007/s001220050574"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Quantitative trait locus mapping using sets of near-isogenic lines: Relative power comparisons and technical considerations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">384</span>–<span class="cit-lpage">392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220050574</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D384%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050574&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XV26800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.30" data-doi="10.1016/j.neubiorev.2018.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kafkafi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agassi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crabbe</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crusio</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Reproducibility and replicability of rodent phenotyping in preclinical studies.</span> <abbr class="cit-jnl-abbrev">Neurosci. Biobehav. Rev.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neubiorev.2018.01.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurosci.%2BBiobehav.%2BRev.%26rft.volume%253D87%26rft.spage%253D218%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neubiorev.2018.01.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neubiorev.2018.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.31" data-doi="10.1073/pnas.0409554102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kafkafi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elmer</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golani</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genotype-environment interactions in mouse behavior: a way out of the problem.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">4619</span>–<span class="cit-lpage">4624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0409554102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409554102%26rft_id%253Dinfo%253Apmid%252F15764701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzEyLzQ2MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.32" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.33" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.080101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.34" data-doi="10.1002/oby.22075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keele</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prokop</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holl</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littrell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic Fine-Mapping and Identification of Candidate Genes and Variants for Adiposity Traits in Outbred Rats.</span> <abbr class="cit-jnl-abbrev">Obesity (Silver Spring)</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/oby.22075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DObesity%2B%2528Silver%2BSpring%2529%26rft.volume%253D26%26rft.spage%253D213%26rft_id%253Dinfo%253Adoi%252F10.1002%252Foby.22075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/oby.22075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.35" data-doi="10.1534/g3.116.032912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelada</span>  <span class="cit-name-given-names">S. N. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Plethysmography Phenotype QTL in Mice Before and After Allergen Sensitization and Challenge.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2857</span>–<span class="cit-lpage">2865</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032912</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032912%26rft_id%253Dinfo%253Apmid%252F27449512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yODU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.36" data-doi="10.1534/g3.111.001776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelada</span>  <span class="cit-name-given-names">S. N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peck</span>  <span class="cit-name-given-names">B. C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavarez</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic Analysis of Hematological Parameters in Incipient Lines of the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.001776</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.001776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster. G3 (Bethesda) 7: 1643 LP – 1652.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.38" data-doi="10.1534/genetics.112.138537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Properties and power of the Drosophila synthetic population resource for the routine dissection of complex traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.138537</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.138537%26rft_id%253Dinfo%253Apmid%252F22505626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvOTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.39" data-doi="10.1534/g3.113.006619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kislukhin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Genetic Architecture of Methotrexate Toxicity Is Similar in Drosophila melanogaster and Humans. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1301</span>–<span class="cit-lpage">1310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.006619</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D3%26rft.spage%253D1301%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.006619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.113.006619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.40" data-doi="10.1038/hdy.2011.133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klasen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">QTL detection power of multi-parental RIL populations in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">626</span>–<span class="cit-lpage">632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2011.133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DKlasen%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D108%26rft.issue%253D6%26rft.spage%253D626%26rft.epage%253D632%26rft.atitle%253DQTL%2Bdetection%2Bpower%2Bof%2Bmulti-parental%2BRIL%2Bpopulations%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.133%26rft_id%253Dinfo%253Apmid%252F22334115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22334115&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bridges</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Using molecular markers to estimate quantitative trait locus parameters: power and genetic variances for unreplicated and replicated progeny.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnapp%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D126%26rft.issue%253D3%26rft.spage%253D769%26rft.epage%253D777%26rft.atitle%253DUsing%2Bmolecular%2Bmarkers%2Bto%2Bestimate%2Bquantitative%2Btrait%2Blocus%2Bparameters%253A%2Bpower%2Band%2Bgenetic%2Bvariances%2Bfor%2Bunreplicated%2Band%2Breplicated%2Bprogeny.%26rft_id%253Dinfo%253Apmid%252F2249768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI2LzMvNzY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.42" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000551</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.44" data-doi="10.1186/s12864-015-2213-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabet</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Collaborative cross mice in a genetic association study reveal new candidate genes for bone microarchitecture.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1013</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2213-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1013%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2213-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2213-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.45" data-doi="10.1371/journal.pone.0017573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Joint QTL linkage mapping for multiple-cross mating design sharing one common parent.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e17573</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0017573</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De17573%26rft.epage%253De17573%26rft.atitle%253DJoint%2BQTL%2Blinkage%2Bmapping%2Bfor%2Bmultiple-cross%2Bmating%2Bdesign%2Bsharing%2Bone%2Bcommon%2Bparent.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017573%26rft_id%253Dinfo%253Apmid%252F21423655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21423655&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.46" data-doi="10.1038/nmeth.1681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FaST linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">835</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLippert%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D835%26rft.atitle%253DFaST%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1681%26rft_id%253Dinfo%253Apmid%252F21892150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21892150&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.47" data-doi="10.1093/bioinformatics/btq187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Villena</span>  <span class="cit-name-given-names">F. P.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Efficient genome ancestry inference in complex pedigrees with inbreeding.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">i199</span>–<span class="cit-lpage">i207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq187%26rft_id%253Dinfo%253Apmid%252F20529906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20529906&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.48" data-doi="10.1186/s12863-015-0260-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorè</span>  <span class="cit-name-given-names">N. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bragonzi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Host genetic diversity influences the severity of Pseudomonas aeruginosa pneumonia in the Collaborative Cross mice.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-015-0260-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-015-0260-6%26rft_id%253Dinfo%253Apmid%252F26310945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-015-0260-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26310945&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1707.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.50" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Drosophila melanogaster Genetic Reference Panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10811</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.51" data-doi="10.1007/BF00222330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curnow</span>  <span class="cit-name-given-names">R. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00222330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00222330%26rft_id%253Dinfo%253Apmid%252F24197463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00222330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24197463&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KE52200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.52" data-doi="10.1152/ajpendo.00707.2010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathes</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Architecture of energy balance traits in emerging lines of the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Am. J. Physiol. Endocrinol. Metab.</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">E1124</span>–<span class="cit-lpage">E1134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/ajpendo.00707.2010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPhysiol.%2BEndocrinol.%2BMetab.%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fajpendo.00707.2010%26rft_id%253Dinfo%253Apmid%252F21427413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/ajpendo.00707.2010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21427413&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290959000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.53" data-doi="10.1186/s13229-018-0252-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molenhuis</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruining</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">M. J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Soldt</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abu-Toamih Atamni</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Modeling the quantitative nature of neurodevelopmental disorders using Collaborative Cross mice.</span> <abbr class="cit-jnl-abbrev">Mol. Autism</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13229-018-0252-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BAutism%26rft.volume%253D9%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13229-018-0252-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13229-018-0252-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.54" data-doi="10.1534/g3.115.022087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Mouse Universal Genotyping Array: From Substrains to Subspecies. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.022087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.022087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.115.022087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.55" data-doi="10.1093/toxsci/kfw269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mosedale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Candidate Risk Factors and Mechanisms for Tolvaptan-Induced Liver Injury Are Identified Using a Collaborative Cross Approach.</span> <abbr class="cit-jnl-abbrev">Toxicol. Sci.</abbr> <span class="cit-vol">156</span>: <span class="cit-elocation-id">kfw269</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/toxsci/kfw269</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftoxsci%252Fkfw269%26rft_id%253Dinfo%253Apmid%252F28115652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/toxsci/kfw269&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28115652&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.56" data-doi="10.1073/pnas.230304397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turri</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A method for fine mapping quantitative trait loci in outbred animal stocks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12649</span>–<span class="cit-lpage">12654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.230304397</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.230304397%26rft_id%253Dinfo%253Apmid%252F11050180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjMvMTI2NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.57" data-doi="10.1371/journal.pgen.1005663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Najarro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Highfill</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identifying Loci Contributing to Natural Variation in Xenobiotic Resistance in Drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005663</span>. (erratum: PLoS Genetics 12(1): e1005830).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005663</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005663%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.58" data-doi="10.1534/genetics.117.300406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chelo</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guzella</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Afonso</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riccardi</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> Polygenicity and Epistasis Underlie Fitness-Proximal Traits in the Caenorhabditis elegans Multiparental Experimental Evolution (CeMEE) Panel. Genetics 207<span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300406%26rft_id%253Dinfo%253Apmid%252F29066469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy80LzE2NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.59" data-doi="10.1016/j.jaci.2018.10.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orgel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeekens</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fotsch</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genetic diversity between mouse strains allows identification of the CC027/GeniUnc strain as an orally reactive model of peanut allergy.</span> <abbr class="cit-jnl-abbrev">J. Allergy Clin. Immunol.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1037.e7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jaci.2018.10.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAllergy%2BClin.%2BImmunol.%26rft.volume%253D143%26rft.spage%253D1027%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jaci.2018.10.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jaci.2018.10.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.60" data-doi="10.1186/1471-2156-5-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peirce</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A new set of BXD recombinant inbred lines from advanced intercross populations in mice.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-5-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DPeirce%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D5%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DA%2Bnew%2Bset%2Bof%2BBXD%2Brecombinant%2Binbred%2Blines%2Bfrom%2Badvanced%2Bintercross%2Bpopulations%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-5-7%26rft_id%253Dinfo%253Apmid%252F15117419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-5-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15117419&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1707.61"><div class="cit-metadata unstructured">Pfaff, B. and A. McNeil, 2018 <em>evir: Extreme Values in R</em>. R package version 1.7–4.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.62" data-doi="10.1101/gr.113886.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Philip</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokoloff</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackert-Bicknell</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Striz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branstetter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis in the Collaborative Cross breeding population.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.113886.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.113886.110%26rft_id%253Dinfo%253Apmid%252F21734011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIyMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.63" data-doi="10.1038/gene.2013.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillippi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using the emerging Collaborative Cross to probe the immune system.</span> <abbr class="cit-jnl-abbrev">Genes Immun.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/gene.2013.59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BImmun.%26rft.volume%253D15%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fgene.2013.59%26rft_id%253Dinfo%253Apmid%252F24195963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/gene.2013.59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24195963&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1707.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.65" data-doi="10.1534/genetics.114.163014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ram</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balmer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morahan</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rapid identification of major-effect genes using the collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.163014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRam%26rft.auinit1%253DR.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D86%26rft.atitle%253DRapid%2BIdentification%2Bof%2BMajor-Effect%2BGenes%2BUsing%2Bthe%2BCollaborative%2BCross%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.163014%26rft_id%253Dinfo%253Apmid%252F25236450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk4LzEvNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.66" data-doi="10.1126/science.1259595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">991</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1259595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259595%26rft_id%253Dinfo%253Apmid%252F25359852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDYvNjIxMi85ODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.67" data-doi="10.1007/s00335-013-9499-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogala</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gooch</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Collaborative Cross as a resource for modeling human disease: CC011/Unc, a new mouse model for spontaneous colitis.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-013-9499-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D25%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9499-2%26rft_id%253Dinfo%253Apmid%252F24487921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9499-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487921&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.68" data-doi="10.1534/genetics.111.127068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detecting major genetic loci controlling phenotypic variability in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.127068</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127068%26rft_id%253Dinfo%253Apmid%252F21467569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvNDM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.69" data-doi="10.1534/genetics.114.168138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutledge</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peck</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chines</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic regulation of Zfp30, CXCL1, and neutrophilic inflammation in murine lung.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.168138</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.168138%26rft_id%253Dinfo%253Apmid%252F25114278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNzM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.70" data-doi="10.1534/genetics.116.199596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shorter</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Odet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Male Infertility Is Responsible for Nearly Half of the Extinction Observed in the Mouse Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.199596</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199596%26rft_id%253Dinfo%253Apmid%252F28592496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.71" data-doi="10.1186/1471-2156-14-68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shusterman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salyma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nashef</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilensky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotype is an important determinant factor of host susceptibility to periodontitis in the Collaborative Cross and inbred mouse populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">68</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-14-68</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-14-68%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-14-68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.72" data-doi="10.1007/BF00224388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beckmann</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Marker-based mapping of quantitative trait loci using replicated progenies.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00224388</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D80%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00224388%26rft_id%253Dinfo%253Apmid%252F24220897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00224388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24220897&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DV10200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.73" data-doi="10.1534/genetics.116.198838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarsani</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigmon</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomes of the mouse Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198838%26rft_id%253Dinfo%253Apmid%252F28592495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.74" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the Mouse Diversity outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132597</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.75" data-doi="10.1371/journal.pone.0046545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takuno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terauchi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The power of QTL mapping with RILs.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e46545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTakuno%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46545%26rft.epage%253De46545%26rft.atitle%253DThe%2BPower%2Bof%2BQTL%2BMapping%2Bwith%2BRILs.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046545%26rft_id%253Dinfo%253Apmid%252F23056339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23056339&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.76" data-doi="10.1534/genetics.111.138032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ten Years of the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.138032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.138032%26rft_id%253Dinfo%253Apmid%252F22345604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMjkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1707.77" data-doi="10.1007/s00335-001-4001-y"><div class="cit-metadata"><cite>Threadgill, D. W., K. W. Hunter, and R. W. Williams, 2002 Genetic dissection of complex and quantitative traits: from fantasy to reality via a community effort. Mamm. Genome 13: 175–8.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-001-4001-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DThreadgill%26rft.auinit1%253DD.%2BW.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D175%26rft.epage%253D178%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bcomplex%2Band%2Bquantitative%2Btraits%253A%2Bfrom%2Bfantasy%2Bto%2Breality%2Bvia%2Ba%2Bcommunity%2Beffort.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-001-4001-y%26rft_id%253Dinfo%253Apmid%252F11956758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-001-4001-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11956758&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175046300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.78" data-doi="10.1534/genetics.104.039313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1783</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.039313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039313%26rft_id%253Dinfo%253Apmid%252F16361245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE3ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.79" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1840</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1707.80" data-doi="10.1007/978-0-387-21706-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venables</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripley</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Modern Applied Statistics with S</span>, <span class="cit-edition">Ed. 4th</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-21706-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DModern%2BApplied%2BStatistics%2Bwith%2BS%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-21706-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-21706-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.81" data-doi="10.1093/toxsci/kfx065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venkatratnam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furuya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosyk</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodnar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Collaborative Cross Mouse Population Enables Refinements to Characterization of the Variability in Toxicokinetics of Trichloroethylene and Provides Genetic Evidence for the Role of PPAR Pathway in Its Oxidative Metabolism.</span> <abbr class="cit-jnl-abbrev">Toxicol. Sci.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/toxsci/kfx065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftoxsci%252Fkfx065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/toxsci/kfx065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.82" data-doi="10.1186/1471-2164-15-865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vered</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Susceptibility to Klebsiella pneumonaie infection in collaborative cross mice is a complex trait controlled by at least three loci acting at different time points.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">865</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-865</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D865%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-865%26rft_id%253Dinfo%253Apmid%252F25283706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25283706&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1707.83" data-doi="10.1007/s00335-012-9410-6"><div class="cit-metadata"><cite>Welsh, C. E., D. R. Miller, K. F. Manly, J. Wang, L. McMillan, <em>et al.</em>, 2012 Status and access to the Collaborative Cross population. Mamm. Genome 23: 706–12. (erratum: Mamm. Genome 25:192).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-012-9410-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DWelsh%26rft.auinit1%253DC.%2BE.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D706%26rft.epage%253D712%26rft.atitle%253DStatus%2Band%2Baccess%2Bto%2Bthe%2BCollaborative%2BCross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9410-6%26rft_id%253Dinfo%253Apmid%252F22847377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9410-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847377&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.84" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Theoretical basis of the Beavis effect.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2259</span>–<span class="cit-lpage">2268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2259%26rft.epage%253D2268%26rft.atitle%253DTheoretical%2BBasis%2Bof%2Bthe%2BBeavis%2BEffect%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F14704201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIyNTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.85" data-doi="10.1534/genetics.104.028902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Using progenitor strain information to identify quantitative trait nucleotides in outbred mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.028902</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.028902%26rft_id%253Dinfo%253Apmid%252F16085706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcxLzIvNjczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.86" data-doi="10.1186/1471-2156-15-50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanabata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizobuchi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yonemaru</span>  <span class="cit-name-given-names">J.-i.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Effect of advanced intercrossing on genome structure and on the power to detect linked quantitative trait loci in a multi-parent population: a simulation study in rice.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-15-50</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-15-50%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-15-50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.87" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">655</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.88" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.074245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.89" data-doi="10.1534/genetics.114.166249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bayesian modeling of haplotype effects in multiparent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.166249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166249%26rft_id%253Dinfo%253Apmid%252F25236455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE3MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.90" data-doi="10.1534/genetics.115.177873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reconstruction of Genome Ancestry Blocks in Multiparental Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1087</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177873</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177873%26rft_id%253Dinfo%253Apmid%252F26048018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEwNzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.91" data-doi="10.1038/ng.2310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706312&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.92" data-doi="10.1086/512821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zöllner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Overcoming the winner’s curse: estimating penetrance parameters from case-control data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/512821</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DZollner%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D605%26rft.epage%253D615%26rft.atitle%253DOvercoming%2Bthe%2Bwinner%2527s%2Bcurse%253A%2Bestimating%2Bpenetrance%2Bparameters%2Bfrom%2Bcase-control%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F512821%26rft_id%253Dinfo%253Apmid%252F17357068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/512821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17357068&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1707.93" data-doi="10.1534/genetics.106.059469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vision</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assessing the significance of quantitative trait loci in replicable mapping populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1063</span>–<span class="cit-lpage">1068</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.059469</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.059469%26rft_id%253Dinfo%253Apmid%252F16888335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8yLzEwNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1707.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1693" title="Genotypic Frequencies at Equilibrium for Polysomic Inheritance Under Double-Reduction" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1729" title="Long-Term Combinatorial Exposure to Trichloroethylene and Inorganic Arsenic in Genetically Heterogeneous Mice Results in Renal Tubular Damage and Cancer-Associated Molecular Changes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1707.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-R9QT5Ze5qaiOd6gOrbgj4A383e6flRv3WqdF0ppU0Ew" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1707" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1707" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1707" class="active" data-icon-position="" data-hide-link-title="0">Determinants of QTL Mapping Power in the Realized Collaborative Cross</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493200" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0L7vP--ZR-hljm-2YswgSlijrzHM3lvpu0KPYhLFYtw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1707.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1707" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Gh4XA9_0T3OtaDx_z03wvXYvw6sjFhol07K-V5wZ5Cc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1707" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1707.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493200" id="node493200--2" data-pisa="ggg;9/5/1707" data-pisa-master="ggg;g3.119.400194" data-apath="/ggg/9/5/1707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Determinants of QTL Mapping Power in the Realized Collaborative Cross</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory R. Keele</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5745-4490" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wesley L. Crouse</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2676-9232" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samir N. P. Kelada</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1707-1727;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493200--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory R. Keele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BR.%2BKeele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keele%20GR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BR.%2BKeele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory R. Keele</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wesley L. Crouse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wesley%2BL.%2BCrouse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crouse%20WL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWesley%2BL.%2BCrouse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5745-4490" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wesley L. Crouse</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Samir N. P. Kelada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Marsico Lung Institute, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samir%2BN.%2BP.%2BKelada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelada%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamir%2BN.%2BP.%2BKelada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2676-9232" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samir N. P. Kelada</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1707" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1707&amp;title=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1707&amp;title=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1707&amp;title=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1707&amp;text=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1707&amp;title=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1707&amp;t=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1707&amp;title=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1707&amp;title=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493200/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493200" id="node493200--3" data-pisa="ggg;9/5/1707" data-pisa-master="ggg;g3.119.400194" data-apath="/ggg/9/5/1707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Determinants of QTL Mapping Power in the Realized Collaborative Cross</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory R. Keele</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5745-4490" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wesley L. Crouse</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2676-9232" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samir N. P. Kelada</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1707-1727;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493200--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory R. Keele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BR.%2BKeele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keele%20GR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BR.%2BKeele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory R. Keele</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wesley L. Crouse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wesley%2BL.%2BCrouse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crouse%20WL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWesley%2BL.%2BCrouse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5745-4490" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wesley L. Crouse</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Samir N. P. Kelada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Marsico Lung Institute, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samir%2BN.%2BP.%2BKelada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelada%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamir%2BN.%2BP.%2BKelada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2676-9232" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samir N. P. Kelada</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1707&amp;rft.epage=1727&amp;rft.atitle=Determinants%20of%20QTL%20Mapping%20Power%20in%20the%20Realized%20Collaborative%20Cross&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Keele%2C+Gregory+R.&amp;rft.au=Crouse%2C+Wesley+L.&amp;rft.au=Kelada%2C+Samir+N.+P.&amp;rft.au=Valdar%2C+William"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493200/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493200/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493200/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493200/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493200/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493200/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493200/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493200/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493200/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493200/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493200/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493200/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496049" id="node496049" data-pisa="ggg;g3.119.400658v1" data-pisa-master="ggg;g3.119.400658" data-apath="/ggg/early/2020/03/26/g3.119.400658.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.119.400658" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Dissecting Adaptive Traits with Nested Association Mapping: Genetic Architecture of Inflorescence Morphology in Sorghum</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1707" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-28" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="even"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix C</a></li><li class="odd"><a href="#app-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix D</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1707.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1707.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1707.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
