<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1545" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1545.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1545.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1545.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1545/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1545.atom" />
<meta name="HW.pisa" content="ggg;9/5/1545" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes" />
<meta name="DC.Identifier" content="10.1534/g3.119.300585" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Montesinos-López et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multiple-trait experiments with mixed phenotypes (binary, ordinal and continuous) are not rare in animal and plant breeding programs. However, there is a lack of statistical models that can exploit the correlation between traits with mixed phenotypes in order to improve prediction accuracy in the context of genomic selection (GS). For this reason, when breeders have mixed phenotypes, they usually analyze them using univariate models, and thus are not able to exploit the correlation between traits, which many times helps improve prediction accuracy. In this paper we propose applying deep learning for analyzing multiple traits with mixed phenotype data in terms of prediction accuracy. The prediction performance of multiple-trait deep learning with mixed phenotypes (MTDLMP) models was compared to the performance of univariate deep learning (UDL) models. Both models were evaluated using predictors with and without the genotype × environment (G×E) interaction term (I and WI, respectively). The metric used for evaluating prediction accuracy was Pearson’s correlation for continuous traits and the percentage of cases correctly classified (PCCC) for binary and ordinal traits. We found that a modest gain in prediction accuracy was obtained only in the continuous trait under the MTDLMP model compared to the UDL model, whereas for the other traits (1 binary and 2 ordinal) we did not find any difference between the two models. In both models we observed that the prediction performance was better for WI than for I. The MTDLMP model is a good alternative for performing simultaneous predictions of mixed phenotypes (binary, ordinal and continuous) in the context of GS." />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Javier Martín-Vallejo" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Daniel Gianola" />
<meta name="DC.Contributor" content="Carlos M. Hernández-Suárez" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="Philomin Juliana" />
<meta name="DC.Contributor" content="Ravi Singh" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multiple-trait experiments with mixed phenotypes (binary, ordinal and continuous) are not rare in animal and plant breeding programs. However, there is a lack of statistical models that can exploit the correlation between traits with mixed phenotypes in order to improve prediction accuracy in the context of genomic selection (GS). For this reason, when breeders have mixed phenotypes, they usually analyze them using univariate models, and thus are not able to exploit the correlation between traits, which many times helps improve prediction accuracy. In this paper we propose applying deep learning for analyzing multiple traits with mixed phenotype data in terms of prediction accuracy. The prediction performance of multiple-trait deep learning with mixed phenotypes (MTDLMP) models was compared to the performance of univariate deep learning (UDL) models. Both models were evaluated using predictors with and without the genotype × environment (G×E) interaction term (I and WI, respectively). The metric used for evaluating prediction accuracy was Pearson’s correlation for continuous traits and the percentage of cases correctly classified (PCCC) for binary and ordinal traits. We found that a modest gain in prediction accuracy was obtained only in the continuous trait under the MTDLMP model compared to the UDL model, whereas for the other traits (1 binary and 2 ordinal) we did not find any difference between the two models. In both models we observed that the prediction performance was better for WI than for I. The MTDLMP model is a good alternative for performing simultaneous predictions of mixed phenotypes (binary, ordinal and continuous) in the context of GS.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1545" />
<meta name="citation_id" content="9/5/1545" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1545" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1545.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1545.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1545.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.300585" />
<meta name="citation_pmid" content="30858235" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="1545" />
<meta name="citation_lastpage" content="1556" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática" />
<meta name="citation_author" content="Javier Martín-Vallejo" />
<meta name="citation_author_institution" content="Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="aml_uach2004@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Daniel Gianola" />
<meta name="citation_author_institution" content="Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Wisconsin 53706" />
<meta name="citation_author" content="Carlos M. Hernández-Suárez" />
<meta name="citation_author_institution" content="Facultad de Ciencias, Universidad de Colima, Colima, 28040, México" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Jalisco, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="aml_uach2004@hotmail.com" />
<meta name="citation_author" content="Philomin Juliana" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México" />
<meta name="citation_author" content="Ravi Singh" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=B. Alipanahi;citation_author=A. Delong;citation_author=M. T. Weirauch;citation_author=B. J. Frey;citation_title=Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning.;citation_pages=831-838;citation_volume=33;citation_year=2015;citation_pmid=26213851;citation_doi=10.1038/nbt.3300" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=C. Angermueller;citation_author=T. Pärnamaa;citation_author=L. Parts;citation_author=O. Stegle;citation_title=Deep learning for computational biology;citation_pages=878-878;citation_volume=12;citation_year=2016;citation_issue=7;citation_pmid=27474269;citation_doi=10.15252/msb.20156651" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: Software for association mapping of comple traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_title=Deep Learning with R. Manning Publications, Manning Early Access Program;citation_author=F. Chollet;citation_author=J. J. Allaire;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Neuroimage;citation_author=J. H. Cole;citation_author=P. K. Rudra;citation_author=D. T. Poudel;citation_author=W. A. Matthan;citation_author=C. S. Caan;citation_title=Predicting brain age with deep learning from raw imaging data results in a reliable and heritable biomarker.;citation_pages=115-124;citation_volume=163;citation_year=2017;citation_doi=10.1016/j.neuroimage.2017.07.059" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mach. Learn. Comput.;citation_author=A. Dingli;citation_author=K. S. Fournier;citation_title=Financial Time Series Forecasting – A Deep Learning Approach;citation_pages=118-122;citation_volume=7;citation_year=2017;citation_doi=10.18178/ijmlc.2017.7.5.632" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in Whole-Genome Regression: The Bayesian Alphabet Returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_title=Deep Learning. A Practitioner’s Approach.;citation_author=A. Gibson;citation_author=J. Patterson;citation_pages=532;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS : A High Capacity Genotyping by Sequencing Analysis Pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=J. Artif. Intell. Res.;citation_author=Y. Goldberg;citation_title=A primer on neural network models for natural language processing.;citation_pages=420;citation_volume=57;citation_year=2016" />
<meta name="citation_reference" content="citation_author=I. Goodfellow;citation_author=Y. Bengio;citation_author=A. Courville;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Deep Learning with Keras: Implementing deep learning models and neural networks with the power of Python;citation_author=A. Gulli;citation_author=P. Sujit;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. He;citation_author=D. Kuhn;citation_author=L. Parida;citation_title=Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction.;citation_volume=32;citation_year=2016;citation_pmid=27307640;citation_doi=10.1093/bioinformatics/btw249" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=D. Jarquín;citation_author=C. Lemes da Silva;citation_author=R. C. Gaynor;citation_author=J. Poland;citation_author=A. R. Fritz;citation_title=Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.12.0130" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Jiang;citation_author=Q. Zhang;citation_author=L. Ma;citation_author=J. Li;citation_author=Z. Wang;citation_title=Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.;citation_pages=29-36;citation_volume=115;citation_year=2015;citation_pmid=25873147;citation_doi=10.1038/hdy.2015.9" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Juliana;citation_author=R. P. Singh;citation_author=J. Poland;citation_author=S. Mondal;citation_author=J. Crossa;citation_title=Prospects and challenges of applied genomic selection-A new paradigm in breeding for grain yield in bread wheat.;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2018.03.0017" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=W. J. Krzanowski;citation_title=Distance between populations using mixed continuous and categorical variables;citation_pages=235-243;citation_volume=70;citation_year=1983;citation_issue=1;citation_doi=10.1093/biomet/70.1.235" />
<meta name="citation_reference" content="citation_title=Machine Learning with R.;citation_author=B. Lantz;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. LeCun;citation_author=Y. Bengio;citation_author=G. Hinton;citation_title=Deep learning.;citation_pages=436-444;citation_volume=521;citation_year=2015;citation_pmid=26017442;citation_doi=10.1038/nature14539" />
<meta name="citation_reference" content="citation_title=Deep learning made easy with R. A gentle introduction for data science;citation_author=N. D. Lewis;citation_year=2016" />
<meta name="citation_reference" content="Liu, S., Tang, J., Zhang, Z., Gaudiot, J.L. 2017. CAAD: Computer Architecture for Autonomous Driving. ariv preprint ariv:1702.01894." />
<meta name="citation_reference" content="citation_journal_title=Bull. Math. Biophys.;citation_author=W. S. McCulloch;citation_author=W. Pitts;citation_title=A logical calculus of the ideas immanent in nervous activity.;citation_pages=115-133;citation_volume=5;citation_year=1943;citation_doi=10.1007/BF02478259" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. P. Menden;citation_author=F. Iorio;citation_author=M. Garnett;citation_author=U. McDermott;citation_author=C. H. Benes;citation_title=Machine learning prediction of cancer cell sensitivity to drugs based on genomic and chemical properties.;citation_pages=e61318;citation_volume=8;citation_year=2013;citation_pmid=23646105;citation_doi=10.1371/journal.pone.0061318" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_author=D. Money;citation_author=K. Gardner;citation_author=Z. Migicovsky;citation_author=H. Schwaninger;citation_author=G. Zhong;citation_author=S. Myles;citation_year=2015;citation_pmid=26377960;citation_doi=10.1534/g3.115.021667" />
<meta name="citation_reference" content="citation_author=A. Montesinos-López;citation_author=O.A. Montesinos-López;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=C.M. Hernández-Suárez;citation_year=2018;citation_pmid=30291107;citation_doi=10.1534/g3.118.200740" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="R Core Team, 2018. R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna. Austria. ISBN 3–900051–07–0. URL http://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Geophys. Res. Lett.;citation_author=B. Rouet-Leduc;citation_author=C. Hulbert;citation_author=N. Lubbers;citation_author=K. Barros;citation_author=C. J. Humphreys;citation_title=Machine Learning Predicts Laboratory Earthquakes.;citation_pages=9276-9282;citation_volume=44;citation_year=2017;citation_doi=10.1002/2017GL074677" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. W. Schulthess;citation_author=Y. Zhao;citation_author=C. F. H. Longin;citation_author=J. C. Reif;citation_title=Advantages and limitations of multiple-trait genomic prediction for Fusarium head blight severity in hybrid wheat (Triticum aestivum L.).;citation_year=2017;citation_doi=10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=N. Srivastava;citation_author=G. Hinton;citation_author=A. Krizhevsky;citation_author=I. Sutskever;citation_author=R. Salakhutdinov;citation_title=Dropout: A Simple Way to Prevent Neural Networks from Overfitting.;citation_pages=1929-1958;citation_volume=15;citation_year=2014;citation_doi=10.1214/12-AOS1000. 1102.4807" />
<meta name="citation_reference" content="Tavanaei, A., Anandanadarajah, N., Maida, A.S., Loganantharaj, R. 2017. A Deep Learning Model for Predicting Tumor Suppressor Genes and Oncogenes from PDB Structure. bioRiv 177378; 10.1101/177378" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Forecast.;citation_author=R. Weron;citation_title=Electricity price forecasting: A review of the state-of-the-art with a look into the future.;citation_pages=1030-1081;citation_volume=30;citation_year=2014;citation_doi=10.1016/j.ijforecast.2014.08.008" />
<meta name="citation_reference" content="Wiley, J.F. 2016. R Deep Learning Essentials. Packt. ISBN (Print) 9781785280580." />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Multiple-trait experiments with mixed phenotypes (binary, ordinal and continuous) are not rare in animal and plant breeding programs. However, there is a lack of statistical models that can exploit the correlation between traits with mixed phenotypes in order to improve prediction accuracy in the context of genomic selection (GS). For this reason, when breeders have mixed phenotypes, they usually analyze them using univariate models, and thus are not able to exploit the correlation between traits, which many times helps improve prediction accuracy. In this paper we propose applying deep learning for analyzing multiple traits with mixed phenotype data in terms of prediction accuracy. The prediction performance of multiple-trait deep learning with mixed phenotypes (MTDLMP) models was compared to the performance of univariate deep learning (UDL) models. Both models were evaluated using predictors with and without the genotype × environment (G×E) interaction term (I and WI, respectively). The metric used for evaluating prediction accuracy was Pearson’s correlation for continuous traits and the percentage of cases correctly classified (PCCC) for binary and ordinal traits. We found that a modest gain in prediction accuracy was obtained only in the continuous trait under the MTDLMP model compared to the UDL model, whereas for the other traits (1 binary and 2 ordinal) we did not find any difference between the two models. In both models we observed that the prediction performance was better for WI than for I. The MTDLMP model is a good alternative for performing simultaneous predictions of mixed phenotypes (binary, ordinal and continuous) in the context of GS." />
<meta name="og-title" property="og:title" content="New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1545" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multiple-trait experiments with mixed phenotypes (binary, ordinal and continuous) are not rare in animal and plant breeding programs. However, there is a lack of statistical models that can exploit the correlation between traits with mixed phenotypes in order to improve prediction accuracy in the context of genomic selection (GS). For this reason, when breeders have mixed phenotypes, they usually analyze them using univariate models, and thus are not able to exploit the correlation between traits, which many times helps improve prediction accuracy. In this paper we propose applying deep learning for analyzing multiple traits with mixed phenotype data in terms of prediction accuracy. The prediction performance of multiple-trait deep learning with mixed phenotypes (MTDLMP) models was compared to the performance of univariate deep learning (UDL) models. Both models were evaluated using predictors with and without the genotype × environment (G×E) interaction term (I and WI, respectively). The metric used for evaluating prediction accuracy was Pearson’s correlation for continuous traits and the percentage of cases correctly classified (PCCC) for binary and ordinal traits. We found that a modest gain in prediction accuracy was obtained only in the continuous trait under the MTDLMP model compared to the UDL model, whereas for the other traits (1 binary and 2 ordinal) we did not find any difference between the two models. In both models we observed that the prediction performance was better for WI than for I. The MTDLMP model is a good alternative for performing simultaneous predictions of mixed phenotypes (binary, ordinal and continuous) in the context of GS." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__HLkFhTtFYKr9UM35ZkeLEJf0AA-RJsk2jDIDlTr7P6E__db-Yi0lrH1NHADEOMB-20Dvr8Qs6F9pwOIMQF9BHr1w__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__qoHMxai33VxgVvIHOVQupamFpg1jn-ylrf2gIvUolBs__Fp-C6VyxAewt6D0gYbv6aOnifLuGF1NoN4YO_dqmh1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UhtS53oBiLrIJKk9161AR_c3wbxSbqVYqTlTLDL3dW0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492857","apath":"\/ggg\/9\/5\/1545.atom","pisa":"ggg;9\/5\/1545","ac":{"\/ggg\/9\/5\/1545.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1545.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1545"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1545"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1545":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U7kEVTxSm5IW\\\\\\\/yxcBMfyUGF89rT16FXgH\\\\\\\/LZfRaqvw3p5nhr\\\\\\\/8grAHGhAG+7LvwI2n2ROdZ1qJuz31BkjWsXwsxFd21sfvvTDbFOZdnU0xcwDTt3i7li+vLA\\\\\\\/rFmnqG9PbVFQoZxeWrlbFQOPL3UO2pO68WbDxq9lnr\\\\\\\/FtFUHSGhX0gOmKaA+1GBroMZ+KZRGs4oowFULqaUyyTwgKdnX8Pzf0Gl6E9b\\\\\\\/R7X9t9x6X0XCnhr9l+n2bd6YsvmoThiy6yzmGjmBvjQiWHuGNCrgEVuPsWwViQNBqCG68Dxnlr5zSu+j+thkN+lMs4pqVGf+oiOx+c186iVyL\\\\\\\/ba5XigwKdu1IBwOAm8NdoG\\\\\\\/D1WF0gN8j9xL0NKgJwPaRb8VtgGZfZOKCMs+3Ks8dUnRuAwH5mxYWKVc9xmPd3zc4GcqMU6laKXRrjwlDS53HTxm26wSqScuaI3g98ROvHKLuSBWgAaGG7VIHWz+VFfBiYLmqzr8vglMdduhLqKcfSUDBkSjpc97xrUFQHFQgt6JST7ClRonCXdO+tsGWmyZGM4Hn3obqPbvJ0AJ5LQVVCRJ2xQ8DazeVurC0z4iZGNLUyfnxM6P1SgsO3R+EjHT5bbCBXO8RnPYAHizMHeAlkph8+HdymlSqcc4YPaTCzHxu862SY3TIL9x\\\\\\\/uTwRJOWZJAwgffdMtv4SBAf0uUrqWT+3\\\\\\\/LJonYl+HH7BaTYvShZAgztRZRepOLLX+nsGVHgGi0eb3VK4H91mVQYQwOClkBtoBhNsz1xezjMa4DnKT9J\\\\\\\/cCNRi\\\\\\\/M4PpU9y8SH4FXK1erKKQMSVz9EjcReosiV\\\\\\\/dZLpAH7XBv6LTQtEJWxfKsMyFAybPoh3Ipf107bV+assNP52W9n1LXjvxmmrVsolDUnpVQIy6atxKS\\\\\\\/xOCYRRytm4M1WDz4c8Q66Tgso4tfWtb8Bhm75VbVATQOOpHDCqFfWwpvGhI6kAGONU7g0kS7oPzNmfr5AwQ1WuuaNO9xfWDmh6A9bcNWgMHrDpPSKfOOZcX4u8tvGcuhK4TtWZPzqyVCdWODEL7Y4IQYzEet7MRKFxPUMumtrIrf7RxL15fXqWtOt4907PELNGykhAD6rReattJ\\\\\\\/QF78qZR3J6QaIXfVI1ukS6V\\\\\\\/h10T\\\\\\\/AptmAjvDYbGHlAUHj1MQ29ORfw9J+ZKcsgaCfIecBPApheI81JrC11J7ItICn1\\\\\\\/DRtN6dK905JyhPH3fovYb1nSk2s7In1t0ofYrUoZtGvP2dD9\\\\\\\/DUvLQXYDMxVTmScEz5fboA47\\\u0022,\\\u0022iv\\\u0022:\\\u0022PIWF9keOmjjgQU8Wvzeudw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221e90022834ee840f353b5b3d04183576fd234ded5b1b06014161cdeebbb6ae3a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eU3D2rA1HAnvonTnyBMvPmqYg\\\\\\\/n1Qcy\\\\\\\/6IWK4URD8om1b4GRvHk3XoFyEQ1Wou\\\\\\\/N+XoHOLOLwoKGS5rvzr3ibRtiT\\\\\\\/wCZvos+zeIh8R90xTU\\\\\\\/\\\\\\\/MQiKLMd+xdKM0M5Wd0JeiG2gd2Ad\\\\\\\/1V0oQFcOJoE0WIO4t4+EFxkAQi6xc+VX6m2fpyrBKODT\\\\\\\/0ESEGTGJi7yd\\\\\\\/kw4Y4LTD3XhMtu4URSQGYJToGE1rMYrnfTiuAUeaC5P68X7nDRnmri0tzceQA7FiZ393EpcRiRCoZXN0cxxpOiFD9KbpVuDsWsDztn+Q36lRx3jqb4yvATDUQ6nIfdiATBJqPieBw1SqV298lyZ0VoJKLE2RXFdjz2V4NCtz\\\\\\\/oPArnt6e16xhfethSlnz62vVbqDJGRIR+3atYTIzwVqqizT1yqHtql2sQTF3TmCtiHn\\\\\\\/26Bz+urixxvndVtUP0RRJzOXOGUkRNojmNGVkRCkJqEq22UP5dAwIubjrVv6x6Ohhb6yH1XdrbZWhAA7uRopA\\\\\\\/GnK9cbFVcAo5c4R5Y1HLV625kNTtYgB9TQ9pS0GbxQvNSOqxrXktkWXhE+xBIh7FxWqq7c\\\\\\\/QwH12CPqiWv9bKcM0FY1wd1vetdrIflfVatn541NAqregHQxlGgwESgdwY7zEaznG99P5Mmv8opl0s3DQnNzttSYcU9lKPCNDJl+r8gQTP8wGQ8jcujpONyFWXYyfN4COkf7FedR+XlrBCSLG0+HBwee5lwl0O6KLzTd2uuRncQry18ZxX619HKcLeaZPpS6pF8K0RSloYtDP9TBd\\\\\\\/CiHpsXGiaOcj13opO3JjM2Mbp\\\\\\\/oD1+HCHFN9Qb1gy\\\\\\\/NgxjdnA7oWR\\\\\\\/hUzj\\\\\\\/0L3dZj0B851BpxKrJsllzq1Bbcs+qxV\\\\\\\/BEzxSuuQ3TjapiF8qoNJX4Zm2rCe5Y9DMnvZ6vKdG1vxXrwwLY75ssJA98pcr56Dolv+y5k0k0z+mWyj\\\\\\\/6AFk9PtggdFgkavetE\\\\\\\/EOXP+Nq3GPZ5AQe7Z9x8+V37VGrqAr4zL21yfI8CBytGE0A0x4JKP1Md6RL84pb65BtozBnx9\\\\\\\/+3BDVo0ywShHPFN1DB5lai0FOeHoqvb5qXB6LNg6hdb3TJL\\\\\\\/gVQreM3DY2WC0U6ZiLqgAp79FZUl9OvmIjcoKDpig90g6bh1wJpHN87hPS7KojUlC7S+0PDQf\\\\\\\/Pqf04gxF1E5JY52CT47o8n0kIYOoLtGvPr3ko\\\\\\\/u+CKKVPJHPejyIJEL8Mjcrsa\\\\\\\/vnqUOlU\\\\\\\/cVYTmMr8ILYBN93FJqs\\\\\\\/cYvdi6n6roMklavPUUAN3wRXTXavrEDx4KX4pNfIdmSU9utiCSMTnImXkUpzCH0yJ5QdFVZMJOoa45UJQp55gfT2sLRhXDiIJwF2Sg3GHR\\\\\\\/PqKbezCHhRpcIMthteV\\\\\\\/\\\\\\\/gg9OsQrImCCybE\\\\\\\/Na7ZQp61g0tTROS02dLVI4xZUPDI53aYG6oFoSIGzXgAEn6Z4iIQZgKKaTcBY78bPSN3biKc5JdK0KqmfWIN1k7QrHPQBH7iF0Ng1\\\\\\\/GmH86AXHFj5VmRkGm3kx69QfZuWtzo776y6qv80fXmYT7PpApxxbLL6UCDeyC+SvoYiPH5Eg06DC+aNhYQJlipt6MAkLBttQFkLTIQ93cj7Y3ptsa0v8pBCKQjY+gTANnPRbXJ4HXAHmcYgtggA3TpSh2Mk3akWDiO2Ylv7qg\\\\\\\/UyDFaXaltQBqNBBtSA2vHmqHiWsxGTUXx0SMX6cQ01ZnlwpML20q+aWU+WX71pduZavgplBCK6kmZYEeJEPOvRc5SDaqmn1G5OjHj5279evWou1D39lELM3h09MENPuUCvjwJkiLAJKfgK7rBJyjRtbZ2F\\\\\\\/uDudxfpgpwEs9JTlHTEQwwxdLR979Bedcay1U+eLo9us5XWHaWJaIzBrjuFrlMEq9rHLfl9jRiFoRt9Tn+Hi5IuU\\\\\\\/9RX\\\\\\\/CBo7E37q59lcMxNb3Lt3Q9TWwKX9q3x3pg5csJTufV9XxFPVJMqFRZ43JRxh2tlOk0ejJlOHCOjLkfbyqb73D1Sp6L86jxX9eaL+jgpVy7YpoofhCB3uUN3orNA2FemIzwj1g088mkGmMVLD17Cy+1jKkWzvlMX3hT9FCne3p5IP5WYmnJd5OM\\\\\\\/Y3xWSK3MGJm9SspjnC8Fln57qjTOaAF8VEKaWLhpLrV4s\\\\\\\/lzUpV+7fA3kg\\\\\\\/uHYtxuxiqL5tKR92Ga+KFu1c1tqsyzozxUKY8R7TPz9axypMDOorgT1tMavCYOcTpxsNU9GOmOU\\\\\\\/tvRDFdEi9miOGxUWjh7ZAzR0N3FmmPptXy29+Yr3ZqldLA\\\\\\\/\\\\\\\/G\\\\\\\/p+hQvSSFnR73TNNvOqgVeRJ1mS7NeVVfm8yGeZ98Q9iR\\\\\\\/uGpx7Zi5Q1KJJ8rwtG0Vm1VbVm6sRop+dS6dVsRpROpiadlqe++3W7+RA3DamHoM9ClCRC\\\\\\\/02l3F6Omt06jCUSEB0hHMxn39X\\\\\\\/MEnuc2UfXlrqJIYdhh0886bVKm3wJg8\\\\\\\/ZIlXrZfnvHOGkvH0QENTQL+H4htO2XHHuHo45TPOi1Hg0e8BfwkVSv5JBvS\\\\\\\/SfUT1RputLzYrnLXTiBNibDRSg+YzExu5TKJJKIoOT\\\\\\\/QXfAOYzVM6C3+L9iTE70rHmWgB64PBqw7mgrompLJcfN2h6T8o5cuhPSWnBs\\\\\\\/OEVdvz1BmvrroV2c1GZiVI+cv30OpZnrDXe5+1lk7cXhT6ypaczTdmOu2oqrfW0beFtVDhCuppemf5CWJTQHDJaHhdH2FSeQXN2ujFVUfbxKKn5zfmN2zK7U62XPz4+n+YxcOmCCBHvO\\\\\\\/SD34Bs3fXTl2xtacWzdTd4JIsz\\\\\\\/fYr62t\\\\\\\/ZfSo8qpiNiz\\\\\\\/ZgbhEX1nmY65VWg4hAkONIe24inszX5ndkTVKREtyxz+F+kfU5ipKt7HNFvFP9clIDuWqXQ0t3TlOHxQtkbfe7\\\\\\\/QYvO7Gt6L\\\\\\\/dtJKFFkJ38DSi\\\\\\\/ygUweI7CCMVYVhU6gSl6yOVBtup6ZDBhCxhNXSDtU61ug+34iYM6kVTN5O\\\\\\\/05zI8hscPirOLzfHNrLgTE\\\\\\\/sDiWI4DmyOi7nTR+fuj3TX2gxIh+V\\\\\\\/tuwBz8jB1c5tGuXKRt3mF7ICEqzxBvIteb+0ua0gzMPb3cVCVGUnj3GL\\\\\\\/p6uHXO\\\\\\\/cQCOybJ+dtp3PmG6m6+9ZUdmm1n9w87qf7Gn958wSJYf6EWG7zTpqxv8SP2\\\\\\\/\\\\\\\/+sRLHkpb3dO3UpygbNs2JjA65+EI2x1CIvW+ddUea3I+QEhODGhSca1GeMwAaw0zKCR\\\\\\\/EoitUF7\\\\\\\/4NZ2H05t+slLLCpe\\\\\\\/NEmOT7FXq1I1TtsfDYUKonvkiYsshLUbIIkvJcjBr\\\\\\\/d80k4WnWsFDDZF2FD\\\\\\\/SgouIhGrBp1ZYRhDB0\\\\\\\/zXVMK5ab8thWTYAgxQIYx+ZXjmPJYPH+jL35WhFTSADcGttuxRLiI8lM\\\\\\\/NniK1q9IHMJwgf7AzbWqu0W1gI\\\\\\\/jgTov4A7d+jA028CJwT6OePmqzM2HqKsGArx9QdVTcKU1G3cYPrkDTbvOKKmBJP8gMX11m9E4Zs79Vbc1kVgLYEr6tWmnHhQPDkIxzbMf3eb0bYBe\\\\\\\/Np1UyitTmsUPIq9CBp1gQw5kcRyPUgl9JcYAcI4y3xdiqC\\\\\\\/XqOUmiSmDWV86uDClfR56XkIeNBUA3CvIwrnSIK4WRx0vZdj8bmfuUGPF6Q5pCb3WsnJLMgVvWsT1mVYaUOOn8rEjNVHktQykJFfiALNgw9tDzE7UKlpNI8P45S35f\\\\\\\/SNTyl4YaB7fWXLVzr41aeyxRIsZBIyI6xKLvvFRwGpXbTAVt83UPyBwPBcLqRTTySCAaVs2Vfj83\\\\\\\/TZBwGgqc5LsXODaxe8KiXeJFmVBG+i\\\\\\\/NFEUa0mz1foW3JdfyO2EJlst0tVoNJtodpD\\\\\\\/0AX9srclCuOm2iG8qL1CKkL+CyMFShsEwLRijLJWJjKhqYKKKrlFH3hMhqJ2B\\\\\\\/6hvPE1pXrsPTGEBsLCdeHrLFbyg1LVZKkE4rA0Ts9jTbZowTdZ5NIh7Z1UxRXtuvnrcGEuasTby7Hp0tsRcOyypQcBxNHITP3ZpU6VccnZNOvHGY7wjxaDrJgqh9yh+rSDqgFc\\\\\\\/tkuNf9G+I2IIJD1Q5T53NbE2atWjB9FBwGPkpYV4GLB10FDiTFwq5VjEVySYYj5AIQ+jRQrb6f5HsVCdxsFwDm66jYtz4lpcximAHGnlsHcG\\\\\\\/dysmIzS0gLaA8d5TJqVfz2I4JUBSDHmfnZFuh\\\\\\\/ySLVKC9J2slNS\\\\\\\/4WH9YDM97vKfibdmTTljg7B8fdDOnSYStZkaa8LpcKiepq2Udu0X\\\\\\\/gE2goVEpkvtR2N9Kz2MRwtj+G+RBg9RRkgoKfGwLgdi4QwI7PFDI1IBb9QpCshbuCQhK08n1KRJadcId1JJQkHTxYKzYXY3eWAslKAZWZ501LwOX7hqKxmjh\\\\\\\/DJyNZaVN9bbEhfrm2LNHqt82qRlnTDXE266W1ty4V1eZTsLXvv41+jzsqWIO41ekoXsxVjjB1pVD9VX7pV2nauzr3zPKYo\\\\\\\/YZqla4WkXIIfAdL5NSZvXzd3S+iZXDKOIqiocXjNeqJFKGx3Ng88XKoHlQs1TtZvwA7N28MeL3RGGxZQJkhT3a1w04Gf\\\\\\\/JJHQa5Vv+Et9YUsmtfrVpUNk2rfHCyoCK\\\\\\\/cmx3bsGisN9gLaIJFMuLmbZcRTyUBob7J5hsTeWT+dMEHqzKI4lR04fOsZ9f9nMIplKi\\\\\\\/yACKp1GEFw1j1vFy98LoWn0DXyJ+gH8QusJ6Z7sVd6WDWRTmQcyXzcj4t484j498aJrV2XesRijBYBNISON\\\\\\\/K1SiMIHu8fShZdZKpggfUCCmgdVHOBuVYRT0HplF+LG6B5b5sNvkatNjaJRrTfYMfKHI4JYuXF\\\\\\\/cjUzva7GnIUQniPWOQGNbDCWF5jcuoXZ1Pgv1va4zmUHjJydqmckbVvFVnn3ZGVOiygKbb6Ef+ti7S26TwFtwvQdTA5K+EXAfaoZitP\\\\\\\/8uzMqSNASmVmvexGwMlAoRz+Qj55wNK5YLZDPCPXLGUYe0QR9\\\\\\\/v7aqBPCas+SkrxUPOwi27HG7uu49ubkAEGkl5awrW+PEAuAfOeL+U7UkwFYxZ9u+bRXrL6w\\\\\\\/ROAWuoGU5\\\\\\\/7lGSL+FN0vXlqijSs8RMNCzA1IhfJUlc84KrGIgakzQCqF3fXTT4VSqDkbQ32VGOa2VDfQOJv7HEZHPrZgP5LvpFUwV7litISR24idOPF3dqxOXk8lY5FI0S1GxYDmPk5p\\\\\\\/EmhaDsZ8WOhhKycQqMX6iwPRUx9NukTaaTneT5OlwYb+kcGzydQPmOUz1kPDLFhkZpyIKXSVlNT9tNgNqN\\\\\\\/aTGs3FqPzp6iCPrCXVU0HsTCxJqiBglqHcJtC5Zh1N432GD38N7Zf96x3cE39B8kAkpLtekcK\\\\\\\/qheAJZTf9pZGWrgpxyHS8tDLAkne0fFGkZibn4ufm2v9j6gY4Qdi8+70OSrmtYQp5Qx2ECJKsew6nHKdiR3qqUilZbZbNizw6DshGhzEmjQvm+VthrEnAPjdITmfLqTRL3YqrraoCgvuxwcSCDXbsCDMdtk8+FbnNT8Jy83SPOzfquKFtZNn\\\\\\\/l0xDCDI3MDTT+Zb+XJIWUGTZaJqYW\\\\\\\/ITr8NKqtYtmNDP1+AGZmfavKZ11pxoJpKxw7ynI4d48v4SDdSobHbqj3qsVkIq6iuQCddD\\\\\\\/v6nkVqdMGGr+DvoCdNS2IEI3fk4SvP0NAl2MHSKzkNZlss3FIqq2DnsrAcKq4oNLFX7fZg7u6RdigfKjzwRZ+iClyC1czGEzP2Di1ZLug8qFox5+iZNUZIXm3AITYbbv3oylQz+SoX1yhI27YtROWWZuy4zizFqpzdYOaWCslvcqOKLqVQjtHYBpWUkk2C50KvUyQ3HasEpBXgn3yWzBWpaF7OhTE6rGaxMQlDdAjAQtjiXX1tXCB8rPYR0XaBn+HCAiO7JtnSPVdvg0Bpg1GF3h49ul+6xHxWUcQQFHcAAgnTGeLIStDWenpdFjsqxffaYzOKJ8lIh6okEgbzTHlYfJ+yTpXwJ2aN7iMQp9xhjXs\\\\\\\/mZxOt8iufEH6oPQ4aCOPEaCUrWuacHANb4tUDl6KT3WUAPAt0FIWm3sM5L4qdvbdiuse\\\\\\\/rdwwd\\\\\\\/XOwgHGaF0oxrRefDi4fDyA95zdeuR1n0qO2Fs6M4Inc2oqeaOuvQGxfhwUJATuiMvZ2I8nokhx95xwEmM2dnMoPNLTxI5xy0mZiuiidu3SpMulyBZ04nXDGBKhioXxntCsJDyz79GDGUzSRAaw60WiJzhJP5oCvd+xfLMi2gTEK6TPiN6071GMMr7ds+7cY6eODfgYjDm9l7AnppplqDas+pqmU7zBpl1852SvQAxzPkcCfT+i+f3Q194aui3RRKucclJO+ZMoU55iZ\\\\\\\/7Yy0tZsgoXYkj0nPbOdADeMxUkg07\\\\\\\/OeHxmhtjztbhVdwibTLCNybXtN6BvbMMEcpvArWf1NAP9j6qtQBUVQZ3p1wQN8NNV10tQQfgWPgtJd5Z3uerPdr3ABEfhWLsLI8QGPrZoRaYCeWV3nJJiZk2yUPOoqi4kYMX\\\\\\\/t76b4NERTkEmP\\\\\\\/a8fMGjvuwp1nwKSRAYrEhOhkfW77u7FkxDQGxjsFKg7wS51TEjis26BYuK9WqyDd3lZ+NiuuqNUZIc8PFbSIG9uN8HRzefT8ALl2pRB1ouCgLhh17BBI\\\\\\\/huhGnS3GImCoxe8SmGcI\\\\\\\/RjF9NgmPnV6XgYvB+mmkjAK4HZswD5SPY+0XbfAkYvxC1U3qVzfHXk68PdozF\\\\\\\/c8vtjNsW6Ie\\\\\\\/4ar6dwA6f6Fw+m13uDQVDczupCwqXO4hbe71HTorA1phQX5OruQsEZOMkWzDySj5qUkwZdICpn610RfJRWnA3fgqfljNlxEvGfTxcQY3\\\\\\\/V5C5K+uXYwxQE\\\\\\\/CVfsktgmbj7btOYvKJCE+PFTjf+PhbraZKnkNqmk6JrSCSWZNOiOs3WkH9Rmla8LHu5taFoK4BHgSZ8y0keTem\\\\\\\/\\\\\\\/dAMzrCblZWdWVdzdpl+b7L1NV2\\\\\\\/yNse23u1+chbifoR5u0xkc8M2\\\\\\\/mLnW4QxoLi7\\\\\\\/AaYwhhgFbafFPcqIgZlZa47Z5jVXCShrGjY8O2l8Ph5hRW1FySFQjVBHxp4rMu4azB\\\\\\\/pwbSHpHt4QTMmPlBq3l6vS9LWK5qrX1SLbvS7c\\\\\\\/T+fj4kbTfA1nk\\\\\\\/Ik+AE74t8wcMebW1eeXK\\\\\\\/hRTIEZeypiN+XWdObuBkoZM4QxfSlhfQiHh4Qb2gackCYlAtzLJHL7J+pVWDXUD6Nl0bUmNw1dhD1HlRlnien0szUvo5fsAt0ohSCnEzP2j+8usXvDbAJ4lVnJV29oRI8UNlY7wwcvsnUu2f8t2oaeJRF87APqpfiUimAp2jZ70nM9dF4+XxONvlpoNaPnpRuYVkQriW2rSPLYMhSy449Z0STxv1aVFZoHMVxIyBWJ1N40sRCPV2EzVqzMY6cXEoj7b6I3tpBF2xf8ngLV\\\\\\\/zxAs3Y2YC5tC6jf7A614TclRTVghksjh\\\\\\\/6TgLsDR7bA02FbiyrRqkm0xMi5R3vFfv4ovn2sdfCp9j3uS8Zdzb7dNetEwFeBatO5rxppcnguMeld1fDUxtkWqshjMCDtPvzFz05+smdUyWz9zvNpomYB4EXFumwX87wsfTG91FMk+m8+GuoX6z4Lx2m9SbZwwgeXu75A7lvpLZjaadP5hVtp+\\\\\\\/ryNBJyUJ\\\\\\\/fr\\\\\\\/4rrjumCkv\\\\\\\/WMxU7kYVM53uFenH48GQ6uDlJN7H9mN4F6Nchd392r8eXTKEMiZi0+T+kBoQNZdXiGB4r2H6PRQ3D17angbSnp6PxHvdCmx2Sbqk4UHiejpAYitU32kSv0Od1xh\\\\\\\/+SphcbrCyJijG3vwdNLbe6IVO1SKsz0uFBrck\\\\\\\/pkDFFbnqvbfwQhwFBKecWjO0lp\\\\\\\/+QPkmIuOQQGSy9NR96l6Qm0s\\\\\\\/Ekek+rHq\\\\\\\/97ISQU12QwPX06t8KzGQNtWlj8K4LpHjIPepkTx90hwwcIak3iDp+rQfwMADlm5tofX1Ym\\\\\\\/2SqsdzI4XTw5RBMBM9gpV8jlsH\\\\\\\/rqFvX2j86yHx7QVU7c+BI1pn5aKiABtktxkJVVkJ5PHfa4uovpbGEf63xCRZxsv+6a9x0243v3SfEt3r7M+VY+UqSlCgS\\\\\\\/dLOUmHL+zTho9mPWR\\\\\\\/6GD1KsZNX0TOwHeZbHeGPBZ3H9TtLVXHnKcBPOvuESAh\\\\\\\/R3d9Ph1Xy6hnrdoonCW4+y78T8dK+76Dzhoj4mZHN797gvPWMfsDzKA0k\\\\\\\/en\\\\\\\/M7G3jTxwsp7ZYSgpJeV2u\\\\\\\/ehguAGEX2azOPL7g6U\\\\\\\/oL0tJ5I+xVJtHKlgpoXVSk4djIcXR4aUb0Ne6yG7+J7aUrHY8ZosxVsr6jIMUumwIf6QnnWbpPtjkQuCnt2\\\\\\\/j8MbfTBnsdKE52eIf+sMUoFryoRoeB9o498aMitTb\\\\\\\/iDVlPMCgWuvUxVnDD5IN51jWWDMmISrwYzKTGH03jwYfNT+kq8n6radc4NFvkEvICdnA9e4sQLmkmO4mQsbOw40+YU1Iag2QSPa6bB1jVv30ZAvwQvvVIQEBBxkYR9yigJdpO0Ho2UOVqr32Kat9wG+hlHR0Kl3sk97B\\\\\\\/\\\\\\\/31Br4IQrSYvfpMZdeInzFiDsTDYt03w21gMcy4FQbVvOySgh\\\\\\\/U1kCEoTTzSvJas6LwGyrvopcJhf1w23EFziQjXoHRj5nnfBkXZG1qnZXSG1SS2HZE3YM3J16cT0TINTF7tsT2bggyEmW1dnT4vv+DgRVvt7lhu0QdzqHNPqsuhwESJi86HKvKXZLBGx2e5V75EVZRmkAfdnaVsTH+x1REoDnGOAUv29fBJsi22wQwohwt+Dvg7KoM5rvv\\\\\\\/Hl2OhxWypI6I50aoah7B3RGtO2vRKumftNkB9HarBtdywipFbiDJN0K7OvD6bqymq5rwB8CsrIxVYBihN4uvtULPOcUErHZazDIvdpNEuW+vMtEYHxX+lPfRCGmiHaoqDoLRi1UI4NwO0qFkeSENYwqnqXFSwCKii04oSgacmcQcsHBBUkXqpIH2bqb3pxfAMKSgu9E\\\\\\\/t2v\\\\\\\/M4j2ZjBc8JcVQPhsYSas0p1vFO9D3HWdqMUGkUmbijLu1nIyDQ6Dqeasm+mDQIPOhJT32ly+q\\\\\\\/qHbxBis+CdDyB2cMjMPIyLqS0npfXatBn+vLPdc43l2OIrXKkKLsYS74gJQ\\\\\\\/hfl8MvscIl0HVCdALINKSCcp3OIxCHTlq1Hm0\\\\\\\/qkAGQisUEUE0BlVtyNaXQKq9N5IgNafs1YXuGHeDsmA6O8QZEBYIk\\\\\\\/xO6ZGS7a3f66EMhywyVXaGIW2lz14mJVNkwBWNvT0XfV96fNdojOoRtAGlokYJ5omyB5YRXmYIwfaMhpAmVBqAYca9xLdWf4lciwUlgpZY9SLqzbgGE3yIDMMgQ0RbTk27DqKnSG4RwOyEaHED\\\\\\\/fquNmiiVbT8Szqr7QGe6qVdzFLdGGbfYM9zGsDYLH1m8mU2bBinN\\\\\\\/HFffwW+OW5zp0pAHYmRT6tBfTfAu\\\\\\\/6vTskcIMUcpCqCi5l1FB\\\\\\\/f\\\\\\\/5EDwvypyz9egX6tWeiHgPsKESbvdkM3Q\\\\\\\/LTRO1eAeCqpPuHvEKaGKDESc3x0kWDv3iNxI9\\\\\\\/nB2hOtX9svU7AZerIM7wZztLUN7C4pkjmEChBSDa+R0f5Z2Yn7LHoVp2UraC1Vj+kVFqEN\\\\\\\/fWgeaYOAycbZ9abvbMcd40sMdUxxCk2MumpT0zQALR8iXUtpv8MMp48vKEn1Lo+8pHfVBDCLTh06GRoR9qO\\\\\\\/x77HqZIWMcDwXz0wXH4ZN5C7f1dwkJ3ShSOW0XwoDdAvjVNu5MYmLZN6JW0XzVPALkYF0aK\\\\\\\/MOlNhyhUBtsnBQN1BaSxlJ6Nm4USjlzDIkHOQBF2QoA6U7BiYPEAwtLGHHjwoUFjZ\\\\\\\/GpxLSSpixRQyxGnR1bDoQJL5iemWKZK2UwOg38pU+mFmeq1RS\\\\\\\/yn6gF0WHwuy34PG+MrkBW62efy11TmGEzgK5SeRvQ2b3P2njiQnUyewBvCEV29Kq9OKuYlj4Ak1igYyYvGCMppzDAUGSHFpV01bSv29lvA87AKo+zI2DXrBNTcoE1vZ6w9cND5qNOHcw7nfSuD8IojMlSAcxXtaamoSjz6+WXFd+X4cW0NknBBBlLEdDLD9jkZ5w8VSIhJMVghrccg9g4zox4seCNG5Ov7QCVsNqFCaaNGETlbBaehAxsUOPa3xQNuAKPrRbrhwZNNkuMu0ycTomAqyZKkjuSGMwwNqYPkoaJc4NJ4sQt0oWXLD4DVKLYbXmw2Wz7dv0AHmb12VivWg+qjZzyFHASeMmD7Dx+OIIQ3vkOwM\\\\\\\/lNLm0Mq8O9SNMMmpiHVlweoNtw4Z+3EcIlZrhMuCQ1e7JbPgx\\\\\\\/4h2srmEbxruScQ8NBa1\\\\\\\/cAcO4WkMYEn8oufab6nQc+zM3eKI8QXm493YuPT0nthMTDW\\\\\\\/\\\\\\\/lERPckqWcSJ8C1+O\\\\\\\/JvDxKy3KYpLFv4eVgKnCcYwiW1kZmwi8SYxd3wJH5BZgpXbL9COc8OAJUNo4T6c5lzW0OBNIuTAi6sZ09xtrE+l0OGWvrq4Omm+C0+79INowKpjw7Auoep4scvnmTkqn29UGjM7wxYTeZFbD5+7g4vOiXFb5CM7jxdOWASJoqE+OQ8Z\\\\\\\/47fVrTUV2CYyGSFLJxOQz5PW22T8eUCZCuVrcmWVW7sBBHpdYJN3odFYOAXkeBawFOMHpZr6AA2aULWN8lNgMC\\\\\\\/gzyBUl7eV309kVvEV3u\\\\\\\/3qg2e1Vk+teaIcSnuoMXj7UThm9keVf8yaUkB6OTWacyotl2\\\\\\\/ZI+5Gu6XrjJdbY4w49iu5wr5O8JxGJ6fPHXTjB6vbFzCuzFwG8PuRC8jLYdTvgzCpAHnL1EWClDf2fYLZKPTg7IAJg9I\\\\\\\/IN9qEeat+Ss5VR7yIqR9iGdh0if40rEy8M1B5JrN\\\\\\\/1qAGEXqduR0j+Qi41ictHzL+v0zL6a5fkm4CC+mJZbEoIKxqapNIfPoIbkxNm4cCGZagjhNgx+TI0MOxkESu\\\\\\\/+sFtzI8nx3muR0BzLt0EWYaJBeN0lYlZE9TQ354Z5LSDDls4iXNKs4IJOuBPPJaA\\\\\\\/fz9AOKPTgAGl4vun6NSdQsNeCIvofCph\\\\\\\/J3KU7Sq3fuiW8dPfhuY3a9T73QT95YuiNVCtz8LTpvND+r0agIWFhnz918wqoLkyHnUhfRFKAiP1LR2kSjqDq9bumNe2uposOZZywOvWuzYt\\\\\\\/lVeF8\\\\\\\/\\\\\\\/Of66WyJKomDo8KVYGfOJjJAt8JDtnNJuocKHfe+RvxvMQwlgbJEJmstFvnUzs1PiB6RtQP0sbcNJDH2IJ+vExUOJYX8cDhdfKMdi3aMIKmfiO0W\\\\\\\/DbF8EwA3UyH+TD41OXLNuU\\\\\\\/zlWXihYHqbrL8orBxRBJU7rAx68bvE4K0bqsjHcFbAi3dfhYTQ2saMysMEVtABMyWZSZ3GQ4M4dguAUHgD0qMGhw+vnX5\\\\\\\/rboi+K+VCKtl+0FYQuuriibclrK495u0D0cn76hC79YCZx1xADw3cOHLe2w7Esb2CTA1y2\\\\\\\/RSstYD1qa8pXOebCMoShWbpAajhgqSszurPHW+wLo\\\\\\\/tJ7KjaRcwtOe9S3\\\\\\\/rf\\\\\\\/Rn8b9ZRFR796xouLyg+M8qeEkBS1EgwGYvyvFeBseIUsNCRMI1wQaA02S+5FVQp8iawW5ux8\\\\\\\/0f1tCmx3eJNXE8HDtySMQ\\\\\\\/0DbxB7f7IhYO7pnqxoKc8D14GpoVyE1+BvihQyu9yo0Y+tAMbjuug582mwQNHMYwblsSPtI0dWQuVApRsZqw8kyoD4+ZSwzdfa1k8Ty7+H40e9QTy9hhUHtp2W7jWU3Ym01caB3rIzrV1nCFBPPEFVx5\\\\\\\/tZhZ8DiMpetIVhEwQqPeDLYTk0FD6RrQjaKLDC6SkJFd5o1QAMAQDOUnFTs1WecsBlb6mR8o8PkOEkn2YZMT4T\\\\\\\/Y+R3hGfpAz\\\\\\\/ZkSlH6whFg+T9KXqmxA2BLtBif2nl4ue2TJ2ILgkM0pfq\\\\\\\/bjSAArHWEjAkr2I1cH1xcofj3C6hMZfNyJSV5up7zaUakZWS+IKzkpXLn7uv3kkvFJC4\\\\\\\/sAjJVutX8Zgn4fY0Njw56cO5pxkvW77bsctdm+k0GMp54ckxlR054ZbSXYydIFFlFETq0n8jGgmZW2zZqEQPZCJLRmuoMYb3jRLFdr9vfzGX2soPm5tV3nwujr0aHH6fzUUAlWyClD6mQUEKT77IdQGgowAtBrxoPvh0gP96sMzpyYArSKHYAe4piZ9MGj88cvM9vZLf5PSEXfz84XD9YTBXLG2qnK+DLHroHXo7TapndBWUykuvbRdNkyzRdudxOGfPKsi8gE23u1++vrI++6XXay+gGu6EsqHNG9yLSgbc1m2knbQEK\\\\\\\/hUn8mTc7Te\\\\\\\/F4jWj561q6jAnXFm1NJwYlSCcybAAxqrm0X6Q20Nauu+mEHVA3ByLHVjkDzF1a+rUve2gDlH1\\\\\\\/VJz1t6vTrXsQpUNSMOIf0V3pb6yBWlsFGMjva9y4Dq4fxwBjWF5HD6FpQYUKvdHZOUNYCuJu8cHJNyc1qOLNHWOTtWzmvVsgeuUOX5WVtoE9UZa6yllAHl1vVPNqh0KmYJxHScDpr3sRUgXMMrGHGPbX\\\\\\\/dpDLe62nAEWXkjTEFcBWVfK29XzyVaiB2JhLxuXqFHZounkp2OB3f\\\\\\\/ymPUf0Qt\\\\\\\/0I6jKUpM+8RItsAi6MijxMh8Cmrao9VhVC4TuyH2eCKXXMpzMDTS0MnrRDICffxiMoSE0nYL85dE\\\\\\\/IR20pXzpjeS+yZNsMx\\\\\\\/S8ol\\\\\\\/cmOmWlF0n1wg9TT+3t7fYCS+96icIO9n\\\\\\\/YomQit0zEal7f9rjC7nOvUq3LL23TNjnrH0015DoEtiNQ74nmrHxPnqFMrtEjcxM1SaOxLyvFa9JNrpnO16meQeobsJKhwYm8ibj+1lKt6GVQ5tyz7HOEP5I+SVJIB\\\\\\\/P+v5E5a6RXHeTzkGx862RhrKKYg0+78JQNFLy7aXgmGmtMBDSZ5RFIhBmvcD\\\\\\\/82GuSBjcZRO9LU8LguxRCFL0pn+r+8K+vBT8xkdcPNURHKdORUmRqaTTMXQWMJ+CFlTbSTSYC1eYLdVi5THYKB7tXH5Rd2WzRaqmju3HBNu79B6FfAjkc\\\\\\\/rUulpDVnbhrHLDVd4nWQKzcMDJlwFBalcnPtPTdDf8YOOQIUlVUmhX6v27HofmdCNdifS0EQ+zwr+VSzM2EWBKrTBrrjHOayDlxnI56r+2yWwVVsvjcq6wMvmrJV1bOmGvnxYun0sw5GQBTrTysOEADDqU2g\\\\\\\/lzOTqF+Yn+npKFd+2cGx3o58FevN1gQr2RyL9ZoqttH4ahsgWwqzBbeWEOlXfQk3ZU7G4iU5QaFdElYcTPUCX6Be3Fdk1JdfZdmRlVj2HtfKZHjOrNdeQCmsQNn01zcJMiTw7UqsEvUo8t7hBmkuxfSZNla+3hdSzeEiJEQRrNVJiT9jQW3Ig7G5YA3OlKSOgODx+GLaq96cxgVQMAJXOt8B24i\\\\\\\/3sDQ2PUUp+t0I57xP5hhTQDiIytYM7yD0kUpA558aqsmvrQoghXzChaMFk+4YFTs5qtAMsanmzsFarxxtDWYDxUhakKn93K0C6HvvRSeAvYnjdGGWqL4oyXJ+qfI2qj0XnEhdL4g5ET+wNNVACLuG5Ahbd5lnzp7TwxfS5xsvMZVz\\\\\\\/7yEwPHYcM1isC\\\\\\\/393yBmE5jcnmFYqdeOApvGhjJUPIGn8lJxS2GnckLI6IUSOIHXmWxpbIvfCPR8enAiEKkfifCC5SfDJTPF\\\\\\\/qoixHaVxDSJYK54DPjWu7aJWzsXW8R7e1T89GywSjcbfDdBpm11lRQ7nSutNnMFPo94lu6MgBU40l\\\\\\\/B2PaPtXRB+kHTl3Uyn8\\\\\\\/Kdo+oNieh0ldBTc6L0\\\\\\\/DeBbZW6F\\\\\\\/cN5VGnMf7zETyc\\\\\\\/xN\\\\\\\/9kvb5gc448yDqqrN3dWm9z7TNRDlBfoHeD0gWViOuQKPd7WnymGwypmNlLZEgpoKevpoTDuzUzPpqfknD1KsJ0TGd9dAJ8RurXZUFxsq3hdxruAPLHP92bF1U02uGnkjS5KIioRFaYILrgRwjHuM2RHtyavfSkoO3QY3vIaOjkZfdJFJjeWElT3yTTVv6zkGIizrjZBAvOT5m5EMlSjYPEUSsQCuNc+TpgERJIaq55CeIlvldBMsEfRMgWlO8qH0RVxwwHldKTJQR76in30MV1E3Sz2g8kHztT7\\\\\\\/hOjQtbvYj0Njd3E5jh+fzKSc4pfLimKnBLXS+VQxe2U5zziO1ANIMIKOXJUemgNvqUx6zoKFhc\\\\\\\/fME9r7RT81hm9o93uibOyIaelWWgcg2enCOy0tDzD6+G9hSb\\\\\\\/WBBX+LPnBn6fZD3SdpPJzWiM1x7ysCHYqArw3mY\\\\\\\/OxOlQsFiJJUZeaH12Q2mSmT\\\\\\\/8xmQgMnjGm6OYP35C0dDnIkrPVzVNHHek06cRp5JhvoX3idT8fkTDxGcQ1o6RxZcDvUe15CWgh70kWyIpEughGw\\\\\\\/l+Dx83oFnYV3ZYuuIH\\\\\\\/m+HZ2oSr7MdCJoLjcBE2U+ioCkLRKeK07kOD61F38xFce8y3G7SCFx5WEGR7LiTBcXl9cMzMdNWz0Mzlm1MrzYt\\\\\\\/Rb3qvdCAJXnZFfVDsVCg3ZX6z8y4xqRv5\\\\\\\/lle25aYfkpMU8Hj1PIxXDPDQifs1skRiHlaixZErR1Y\\\\\\\/umG4Jr\\\\\\\/NSOR4Zkak2RUTRyCDO8QS0jIAd+z77FpJYIbrnYOawauMX2YC+tIYtS2YdXMHbqodGllpiYNvkbr4s5OH7KQFwRAaNGFAXBNI233zkGXQy9ALuaMVh6Taj30z1ddHskJrIcVpptKSQ7JpXOnde\\\\\\\/nySE+RU2GIKWDRZQj5152F1QS91rrgFM6syoz8NfBI\\\\\\\/2H44pbGSLHbgdLzSwNOZg5qkNJyB0LtbCgVHriRFbhghhI7ArSEclleq+bcx1KQ++YQc3CgDG7VK\\\\\\\/QmMc4\\\\\\\/8slsjqyTFlsfq8wQ892vGWeBORvlQLw9vQE9sErCrUtTyAFcVdcz1l5iRccGBmDvMoHY2SaKgURB\\\\\\\/oWiFUD5ntf47\\\\\\\/s\\\\\\\/3+jiyU9150Bb38lOzqymAVOmINuzHiujvvLm3rEkugljKVz\\\\\\\/0iA44+iBtlQABBnBI3pd1PgBohpa43z\\\\\\\/9KRb1ER0NUrCl80pPw8pwRBd74D4G0DpuVDw3MRlbcZ\\\\\\\/c8yuVvWE9qTcdshcgSLCZix\\\\\\\/zEosSXh52z4gn1C86EMZ5DmsIy7hhhy1Yju3U6hn+HCkDOD2h9TaXQ5NYwsH7Xl+4LuWpsfjJApvo9mDAWw0VM22Fq4dyYoJc3ERqpmqbxWJYsSoU5NLOU37aqVyei8vI3aFQ0JZL79\\\\\\\/Ea6CkNAicPD\\\\\\\/IW4y80ud38Qn9J8JIkvL5w4vxHuLYcmg5FD8zXWyLBAfl59SennCYcm5kwuVhLYPC4utcJpY+6S7E7wnZNIRmBAa4+Rx4r10funt7uBr\\\\\\\/dXe7q6YA\\\\\\\/0Iut+ONqy2hI07rhgYubw\\\\\\\/5OcxFTuV8zC2zaNinbuILJaabrhFcVBTny6oJUq2SB53s8yVOhBImQQ4ot1yFlhwZMV5jkOCGWuSUR3hfi3hPo\\\\\\\/YdGMe6IJhNS6HcD0EE3H1j6h3VbZl34m\\\\\\\/TlnbZBZH0Z0dW+lWd3LvkDy+GE2vrAPbOST+W9W20DQ30kmPzwSY6hg3t6eM8T9kO\\\\\\\/k1AlDoLt7MPHRG0bu+\\\\\\\/RvMC5pvvr4oAQfj+2W5chT1IG4OQq7itIiFdRO5T3WFAf0C1izFtztIPaWqD2+ziFuUmFVdBYKlbTDAoTAqxESm82Zsity+LC+ehglbZx6SMrwscdjg+OJEgyOgE2sRV9t7cx\\\\\\\/pvhukkPKonZU\\\\\\\/oJa7zOoxC5PrO4pgWhlGYQouKI8YkeOh4ICB787JgcU+Vz4uk8lykgqggHAFi5JSrEkkJHhAJ2CZ1QplLMbqKJW5NJ4+QPwxtX1Xy8HRs89Yc4kmUplVM4emJLEPkRuWuNgGM5k9ufwN9GdmiqchG9PYY2r1QHNnTWuJjPZn8X0ntUHKxELaH0sx4ZIfKn8ShR40WGT4QZt7lZGTLtuPOoWXWxFNjMwua6J8pyJwsLyPjIsjI2Cd9cy9wKV3B4icR0lITNssk6mRyFpQyHvTyVAU7XXWeichREuaH1HiD\\\\\\\/cv\\\\\\\/nOKoDJhao175tk7COrKkdsAAnqJ5LMnqAl2A9euwdW3uvxdSLwLNHKbrT432iw49MkketcZxoPyysEftPdHO731PMdmVahclyj4sFx4ZaQMPLkBZHhV7rvmwYfOF3PYeArKwY37EgyJiA7OAVIwanxvT9dCU9W1lAARfkmxRnr5PdqMDVlGiXJnjlAZIBAmXKwKzXmQ+XoFsfPBstJCYjQpv4Pes5ICWPe5pVa4sQV40V+3R45IvwJ5PC9U\\\\\\\/d6RSTDpXFnxoL2\\\\\\\/pOW6tMm4qQh7NZd4HG9vcDr+qhSSPWUH04NkpanGKMzH7KJm3BP0GIgcOZcdBlTgKjETgcCZB2OHwfs3n3y+PKc50ZO88c7CJ8\\\\\\\/mxL4IXqVyN+LvPYI8n9nBTvMXzhESutk4DVek\\\\\\\/2LtrdaOZSPD8X+zEuOYY9TxOg2FqiwRgSioYFBDgeXSewykhQRyveQVkAJ\\\\\\\/o35giZ5tLoeT\\\\\\\/nvoW\\\\\\\/iYlWgVu9uxQNBNkI2bA0qc1\\\\\\\/Z66y\\\\\\\/AvYN6hB4QGOIetVQnOboD2B0xrtAXS8PZYOIWCbY2marVENHNU\\\\\\\/HmV9S7VYTKlRGz2K04X3xGZYB+mahzMJ4HMnhyb3ggRiuDS358lhOi2PblYMJ9gVRQnz3Sscha2ra3HKjkh\\\\\\\/ISxrEn9FzPjV1h3DKZRloIsBAnOv\\\\\\\/nQT2s6oSYGHRbcmdkga7l\\\\\\\/4G2FBqGxzcud78SCgCfWrgKG90mlrMeDEgekPLNY39M3XPYs0MYxxFWIRMPoJFJYwxjgwar3\\\\\\\/H8zLWD3HZzdo0X4R0PkBjjz8um1Pk9ZY7RdQD1NnQ6adJVf+B3KQNXFbji1X1ENnv2IwrwqYqk5d8N9rKZER0TjgQQcPKCB1HMVnHEGuuoqSC1PsTi4QCbzJfwuBGD\\\\\\\/YdEkX2abR3c0QqNQ21Fqn2C8Aq028My4WJKinFRoMrZP0++hdqmOUs0ZkYPPI\\\\\\\/Lr+G2UdqIMK5L6r3LHx5rhK2L7Oj3PIrgpto9X+Wm8NiOc6mQgNXGEKA9bRaCpUd4xNXDeUUxnwakfv\\\\\\\/Sk9ptncCxiWg0i+F31q\\\\\\\/IVouajfZCmEtHp4pljVdSrCf327Y87jXq9hJoF78ipGfu+jhR02z3kqLR+9AnX5d3yhXCTAnX9JC3gdpOs10RL9mNCoxNkhkajjzhhTx3xqyu5PvmjsGEUOuMoBRL\\\\\\\/2qdDGTxmvAvHTE+3WFofeWQQYCpluD8ynkTML0pFUs2znpWHMx2NpMGnMD\\\\\\\/qk52e5x8OYkxouNcu32uY4sHVxxrakDR3LdaK\\\\\\\/jKaC0I8\\\\\\\/R4LIZGuCjJ0+YEtXEdrn1wFslPbl6TB7NvdtlQR1HCsjX6swnDUPz\\\\\\\/GSauOvaoAtpVhcYDNKqdymS1SxV92ZHGUXXrYNuvoUgkD4XZsEj\\\\\\\/vilPuGxIoPtwHQIbJ\\\\\\\/97SvuMmawTnbl5fqw0DlN2r1bXR\\\\\\\/7Zt4sX6LpbylACNg2MI3ealRrNF8H+nrM0+8vML2tAAkrytTEfIOdHnUHHTWgjrZ6IVZcztPj2eZYC6CyjBGvWOUSMTq08dXUXfGX1kyaUjB10yXLvsiWZHUcFgFJZ72qSJEePMLqtZsryhrcJmMSYFroY1EiyIHpOgPZpR8uwqr9pY6if+PdSZCPc6U1R+kMQYHyhchELf6sPhJt1xk5eaJWlE2b2vsi+i9d45TP+Bbjw2pC9FtwYKY23bBAr0CPJS5EMGNgxPJJBKl+zQ3GXYVEBQS5o0ThEISjkARG9A4dr0AB3mAgI1aNbrI8wERzUWbEldTIVQrW5s7GTCtry6Ebr0DL7Dq29jpscUA5ZulV3EOz8uXIm9UFFssZIoJ8mVqpDpkSAlo01X+pvMatv8OkYLNtGugJ90Ke5neFflP0elbteB5oB6+25M+mzVX8U3pJhrCpRjj0N612VdazHqeAsYR91mg\\\\\\\/zZK7FsqWuHv3Sz+eY7bkMCcwkB6JSP4aQC519oA1SaA+4VXms7CXUb5SKCHAqcwyXdZTE4zx8TKt9DhsqMjcSLljEuRPocWV8GTdPpDP8JCbxRlkouIKwhf9wNf+3vw4OJgxhTaOFyAWlYN\\\\\\\/gZaNZChPhDM8bStdHB+xZipgWMJDQkaefLy57a7bM6nfJ0zDFHVDO6Zlef2wvlOAKx\\\\\\\/sGeiF2CXc+KMDh6uodySJ49hlVkqs1Gdgiuvm4u6b4xtx5uB3LV2lleC9acSZiLR2GJPe37geid51Y9lBCL\\\\\\\/biHUAe3VBmFm6984f9pzLjoSHsI1cjizjx6F9mDHmtsT3tkL3iTVtBviy05fhIYc8vxY3aLc4ZmhPAQW0ts9Io+LIpw7Hy\\\\\\\/AlL7xHkLcXzNvJPQPtuDz4OFd5qDQjNsbneUmGnuL5Kc5LJvppdK3INpTFQ1mHjPXO2KTKoqN2vYp0Zalshf0omD7d9F2T588o5feYydO1qwyzVWM0gkxaKZWsWDAmpP2IJVLaperv4lCpsfQ4jA\\\\\\\/c5ugmT5K6pOB8W6q3UXF+h\\\\\\\/C0ulQi8uTNCgV4ZmghHH4HyeZc2aZ65HMt+aoKrsSxyMHKeKIoVmRR6DM9nNgY++y2Fh\\\\\\\/DJxuqYE\\\\\\\/WV2OJV5ji84CrzXUdJhzRuhoZnvLATqcsKl9+3iGImP8WNUh4xBulML3f9QJyEzkVUDGWxwMtgNjoO1VckAduxYAO6zBIEXMgz2V0pU29N12L4lQdJHXokYY9tz0MJPnjZ36ZQKHHdmyc085plldEoiVCkZQ6PV3KyGd+Dtvenfo6pU4C1EbkHKXSN9XVRJ6SqGAyVJFV5L8ibuW5bE3EyBV0rX0KbcbhAhDdTErOB4vWBIoEr1SznIWiB5l5\\\\\\\/KtIVOWHeyEXwIuclJ7wAtqze40SUOyvGgA2bKGSBWOIBBWMsSWkwnscE2KbW974HqSV+vKsA9vuoXstscHLMZOAhvIjFHR+E+zVkPWDpZIRkrNXEExH5agrODsCNdkLAu9H2BFLnNtPBeVg3yocE5ViQ1Y5hc0LJlS+l+OHeRqIi8XIuPmE6O6pu6zWnvDwsvQ1wKeaaVvTxEp2RDmdhNzUh7fG7vpepB+AWgjzsgoOoSI08YDUxFxKMJ8sbnMPpenZkorjp6GseiA6c5LUqtzpRMQru\\\\\\\/rEbTs7PIHfMmS8X\\\\\\\/VV7gCMJrMNrmqnA7O\\\\\\\/mdTTPKTDA04oMKCcdwVHWm470FIl3SLG0TPbwqxSzAjtCrLUcmIFH2lAo7PNtNzDi2kXebBX7eMoRylrCgv7FJyHyxPil3+Omf0OPJgjxyZxV\\\\\\\/uGiHy3aLAu2yYyNaoZo3QqQRPlHRQfVsDWduZhrFmdRyFfLBiyZMu67vV7UZX69N6SviOTNk+MYNANJFbHtTC3XC16cpZuZoXCsmC40eFQ5XsCi8iKvELGB4EsPuDn6dtNp+kwKZifHSlMDfcvndWlheamHu5EyAwDa8QogRBIlbj76p1VKpgVA5T2G9p26oMhV7mUNc1yw2uzhBd5lJ3tOxPlv46dmBWRjjibS8kzUfzYpc3jmxZDYGt2C0E1uPIGhsI49mYXxl0VmmUy1j29V9tEEM6AkwswTGQc5MANkuWSqQkZMd3RJNmOnYwLK+PyazfswnZqByFU6cWqJ2b7bnOHwYwLNgAyHEoaQqXbkg8UUiUwMK\\\\\\\/K00BZDaCUJF4RJJ\\\\\\\/TC1\\\\\\\/FPMscHyZS1+FlQhP1qPxXeza1YS+VDqUC3H9L7oS54PFpgo62J9aztsJ1ONj9UzpndUkY96UbTaBHQ78wi9LmGyhSo0B\\\\\\\/KwjGEAxlUoVz8tedq3KUR8Fr8EUWEy9r3DzVOCxMVDbDCkemNAHltjB0vumQRoLqybwJ60BcawB2jjQCXQCUgV9b8PC4TLLbcCQOft36I4JDv+sKRzQgPKWqGl\\\\\\\/Yu1iHqYdPRwb7nFlptOlIuVpEVl9OkmyWRBWzoSO\\\\\\\/xgYVwF3su5zpL1RJvPnRmJsRZ8YHwtDvak6e9Ma26e1b2NoF80iIDSXj7jbLQdSMHrM2oInUplVzeagdVoVG0IEENGX2KlPEzExPZ4IzbksdkmaH0TdGSuOVVXZGfJ1FREjEYQ+U9L4ulzZnjD8\\\\\\\/J0DeVtf1MHV4sbwfZcrUJ5kzaX1LqxQI\\\\\\\/OHwM2DPiIbg9taDFFen2UB37Y9B2S9qN0NuJ+m69pv2e3+iYEUoe54CAcbkQVsEBMbccgbyd7td3wPqVIkoLO91DcQm\\\\\\\/XhMQhCalt33C1dX+HXJ5u8Wy31sCIi+G2z8ic\\\\\\\/dvqPdHTFI\\\\\\\/ZTZ0\\\\\\\/6l0jcBHbBU9Uhu246NmFD7Tntug3dEmKpAF4vDXE9Nw+AhAE3wRoZ86YmAYvk5NmVU5aEAF8COvD7iRlTkVMTK+zfKJrBeOlz5Zu6fn4MF5AFMe3gVh\\\\\\\/DpPmmBmGmjjKirFXyaVtlTWxbsp9N47TNe7TbZs1OPPBPhTj7\\\\\\\/xkg9PSFuGeCI1VSntcGAnwUmjo42nfAEjDRViAPv4cXJz+qKWmM6NUVsAgCs8ABbhoH64nXOF\\\\\\\/UvzGiZha3onANkzzO4uL2hdGgJX8zWAO9yXdevKHAfYFY1BNa0XRrTZB0NQ52XIdAmhzl9zyWywu+IkcnxZmWS\\\\\\\/dLJY0tYyKb\\\\\\\/NuA0e6hzVXSdBn8XIMIquTSlQ03uZs3hEJqKNdG2lt1GSk4aAgoHOj8PJ7CiixVz6iK4frThc5bTCVOycsz3aXW16y5NsR1VkPzz8zi+LTsIZL7p8d\\\\\\\/y4qzEDw2Azq0U+DYWn94aipFUibNhwpjPXeIkKUHCAIhgTf4yNp6yM31PfPaAJmLqkoTkg1mv9xXzD8cew6qmUHDNRxncGCY\\\\\\\/ZnNwlBLD+yn6Pq\\\\\\\/XRTmtXLH0LOCZJTRx9ELfQQ3eYmCexcr4j8Iu8AzTmBpjBDZ+dWiK8Xd725pC9inJewaq4\\\\\\\/ryUxJPkOBaxh8DCy7RE7M921tnYAii0aq\\\\\\\/x6yckHDyEzo5yETa\\\\\\\/e6gTAufpRmX8p4KClZ82djtl3dilsMn6UE8dmuxmJEAxbJh5fV67R1xQr8dkNg92zcDYqQFd1OM+rM2Vr2ORn+g2rKcq\\\\\\\/GSyby9l2tXQzgayFhOCw4uBpTyjapuJeu8lInGs+G2aYSOQyveo90zRolgIKk9h9sSS+01y\\\\\\\/hOMWFN8\\\\\\\/l4nsF6GzYrGmmlhJSkpXafe1ValMh1Te0EGZOF+jc+NYTrPWsF+rvQNm7PD2+w8dXMYDaLw4Z8qd7BbXO38iTCsDFMoen5BUS2Z2La1OtWJMeGxvC6nhI8JzKmuO+\\\\\\\/Fwh\\\\\\\/rPN0MDaUqIf+eQ0blGVG2igiYjniTUE1qmNQPM0Iq4WOBET83rRSOBtvtJwaPWOSAk4+WWmpHJdJdskmib4DnVD7nXO56+9Ml9i4BWgszuMlLZTA4skCtUAePbaJYUrPCKGAh+esZ1OW9y\\\\\\\/h2791pzpMs9NU8tcXq46PfVcE+NPRFSpgDbcMKbgSn1P+6kHBrpLMRixm95zlB7Vhs1mLa0VEiabRTNeBfeOgoZRkmeTQNknR52MrUiLbacwDRywDP5lulpEaseeWq0Xa0UTly\\\\\\\/bU4tvQ\\\\\\\/l+a\\\\\\\/TAz6eIKswjZWIZJxj5jt8ADV2zKQMll4euktQKBBQxDngjd0t3agEWHohnORBly++ISDf4s0R0c9T1W1KmW2GOiBeu9DFJCJZhzESBgHx5BAR9C8Zgq9zeaqewZG2hKimKDdoNr2EBTo0tcm0MuhTCFYLxUyTrP475VbFS5UubMZaSu2UPr8s1+8XA9VDbFIWChrpBDquxdT+rG7HowfkFq+4+CI45oeUdRrKONSIDCkSGNWQUI5BDsgJdq6+XaD7fT4yQeupjAGJ+el8t9y\\\\\\\/GUxCYvvXwcafhbeg8DgXXi\\\\\\\/WVHpvrfpNZGyzXdXs6W\\\\\\\/sOOINndupVPx9xLrA8MVh+KAoXBffYEczTFEBJzEceFqai26GzYirsbjSgm92IbKVah+FiKzRXzPt1PaymG7w1LK+bXtUVICGTQAfoi1njQ96zbVeM1\\\\\\\/P+gjEJxpyllRUQ\\\\\\\/rQfsjTOJ1PDm3cvos5\\\\\\\/ZgxKZKe\\\\\\\/Brr2oNgRc+jWLZNpgh2qGcJPJ9kqwSd72jHPrjZJrz7f7QfZA6+KOl81vDslGEIcDYXotYF\\\\\\\/WNVo7coM3ZueVyTyD+9an58WRNUc\\\\\\\/\\\\\\\/M9KW5urdQsjVZyG58BtALEfmvW7nN5WHpbUvbaDJ8TH12kzb8HstOhlbsuk0ZJo0YL720ELPSMDgvVfHZherEJg5x92oaMiMWCaqYoERvNM0bLqDmxC4yu11cbpE5irtdScJHaAPzac\\\\\\\/nYSq2UmS7M7AD833YtrUvMT0W3lidzRCODG40i6OnFexAgyzqMXdjIVKjRe6O6wlL5IqesRBHxy2h9P\\\\\\\/Ztznu13MXbezfBziXVCfnuEq21MFA2hKOPkaMZ\\\\\\\/AbrdQXQP5+PZ2Z4Jrg2KCPzRe145NTkwFId6jPM8EsRQVYGCj4j1SmH25KFYFZTScOlfE1M55GOf9MtsQwAUWILOXMRxBToBar2BaE37VKAyMfhxtNA6BclKK95nn8YiHvb3QYmFF6I2cNbzY4WqOIh5vlVWDA9FYZ5g61l++pjr2SuvelwoJo7hfy3oxyPAwdEOZzxRA6b3Bo6ShduaBAV7kAlz1Vmf47NYBjRe73duXkn2IDDPv\\\\\\\/\\\\\\\/dZnAUvRqzsxmM18x0JIxhXus4YqHugJNPEH1g66OD+D9eCJaMiFfYQnvGPaCmscO8f986QFZieshBpNGtkGR+jc52+94W9A7jGod1AM5kefVh5mjuuzHIhLh8XkbmJ8f6hfpls\\\\\\\/Fd5E7s69B1zWQric4jpcyeGUPSB7cOfN035DTSqLEeGOe1nHEqgaDfkdbgdVlmlZcTdTWXE3PhkVkV1Z89NdrBpkaLsMY1QrGVCSMhI9iOsqpYVI5Q1Jh8lpoR+1chxNAQLE129HEaiYCt8Qxh8MBqR5JNWmIYQArYYEW\\\\\\\/CKiu5ro+zpESCS1ctaHY7bSs5\\\\\\\/hcklLJf9PKHic0BxZrF8n1wCF18e6NaHaYOmTEjaEEtPVm4mnQtvGsbezPZFVBDAjJrdlllQLBmSwThKoAVVgG7Tz1baI9xZIwpsvUMblBvl+fxaQJzNRMY97nb7QUsLYvnGfAj18lQ3CoufrzU2HmHu476MQEnGUzmQ1xMksl70K0MJyaFaejE59nxkXhvoGYCO5QpIxxbD2IL0Gwv4wypyeigwFW7cNTvZfx2v9t6RUzEDQoGw5rUsSaDofWwU2EVzzSBENgaKbAb4srL+GmleLYYB\\\\\\\/3TNgoDOOQyeA2g1y2kEkuFAyJYfJXLHRfHdsK2XBycvL3fYxnqBmxNXR8Lb\\\\\\\/WLwyMuSbNKS6Ze0Ci3szDhVHK4UyEZb5mGvwYgwRV\\\\\\\/rQRf27ICGt+zPObVVGpDWmLtv+HFgldzPS\\\\\\\/bSIYvUMF1FGJshvWQHwG3OHCeplIwpDyeoKE5XAuMk8TfH10w9XPGBMn1FPU6bAVkgcdHM17x8VNfaGnvwR2+hn588CXcbwrIFCj9yWjcRTZDAiO48pehKwj9nx9y5DpDkC7hUpgh5jsEIvfHAM9uSRfjZD5geJd0pViTKr4Z608JdLSqbKF6PusNHMGEA403iIehb5o5qaK16TkOMFfBrQW6ZqMK5BznEYKkjWEV9K\\\\\\\/skDljICL3bdliUUZOqS\\\\\\\/IJM9xEQSGV2ZkFDbwqIM0WczOG8LqpE02RS+8ITcXYK1VdW6acoJB10pWql+pzDEcPBiAn+O23RzWJaNUKdj3ICP+d2uqVi6crXJDKwChviil\\\\\\\/gwCq3Em+U9qJRigEgq2Gb3z02nl01ERa6MgHW8DvaLXJYMeVXz1coEQEUwQcmOtRoD7Yf9RKdOvN6qBGnk1BF6Vs4\\\\\\\/KmsCrm9kgRnDYVfVtNxZbMJ8I09TeABTnfUdicJjxB9jZeeI5CYS2S3aN7AjygsWQjW3GzHW7xChFnXU3rqmJvssr2U7DglV2MkXkNwnxrWsXS4EbdnlAwuAV7Ts85\\\\\\\/axae7P9yCEy5U8gKjwktfcy5DhOayqgzxt0dHGyBrH9ruAAfMlPgR5umwFL\\\\\\\/MEP38Sc98XO0GXRcV+4k3gw\\\\\\\/mmbqSS30kR14f2DgwvqOUf4o3xQBMVg6QSDvmiKOTs2wRKUh0z3LXpEgr\\\\\\\/YnmtdPzFBjXGUlnwIPHM4rhp0VkLK1IxZHdiADFxlxRZ9ZIGBW5fP+QsSti4aqZAKC+EfnppzBtIWI+3F7EhCeBB\\\\\\\/Ffjks+6Bw0OJbB4SwiG0VbPjU2fPTdBzTcV6hyLlEvxs5Qk0KJj9llTvCYDi3cvWg0a6XIoDBEy+hVJqiPvhPpLnavDFc8afPN4Vy5d4TrEHiX1z98bzWM0aM5gGI\\\\\\\/qaQOHTYa2KcazXDfY4fY8CFSzW73NOO6YB7VBiTUuU7kKMfW0wdibmV1B451ldAPuwgiL4rVLFwqgHcYB5HtOPay72HisIe\\\\\\\/L9pPUCmA1B++4+gqe1B0dti1ZdDyTo7U784R6mdPtpHmBWZTWYC6NYam4XxMm+jW3aZmVLkcYV8V7oibORY2hzCmNCWmWvW5exug\\\\\\\/PUHPrsQLr7EnvivgS5W3gUEcqJ0eMV4dZyp4J2ELuGtU6FnPUOnF\\\\\\\/NsZxd4oAnSfb5M\\\\\\\/E11zrgCYkB2za0RcqHcpXK3WPBSrZ4VMJ9lfOa8ocPFliGfo0YSvo\\\\\\\/+D5KcnW2wz4HEorXVGQMjs4DPsNty0Y2NjcesZjBTjYpYhahVjkwkVv6qi991YTcKLp3R8o6ZaAtKqgwy3wSaNHTRyjsCvPx3zhrNtRgdq3uhsTHImgDFYYoDiNyQLXvTMyXjUEut2B+EH3USMqQGmPsAfNyKpdSn2Cs1Ys9poyMn5GadjcWlp6zG57lxXxcKR9DtjMjoTb8LjHNhJbeHnayJcYuwePzhOA5pH8hkzMQZcFniD3XX6lNYGsjAC2XFdhQFdWt23hVtheQ2dPDPJyj7MR18vC2D0Lm\\\\\\\/NQiZjZlysIURRHI2o6Z9tMqcFkrdRuL1epcw35dloFijoCS2BEaGI9xlCE5KOdjstNb0R08u69sahoI5O4bK9GPMmttPOY5UOU+\\\\\\\/\\\\\\\/sHAQHDJOnUuJVsFaOOl1YhXaR\\\\\\\/iek8x674pOz6Y3d4NQdLtp98RuJXwIMzb0kd3VfIvrkjmJK2bpDIDy4CvIHjZrglh1ZCuv3Xelvnq4AI0i\\\\\\\/0ZYpCgHgRzcYZhYyCpXXHU3pYJ4sZ6TYHLnWUfLZMYM86Z8lESc8cyuUpP0xUmd3GIFVhMix1Es+BIuNlzjgUhzXrAHIWJd4kb7zVXufz3Hz0kTglO+hiVI5JzaTLpMYWcYQbdEm6qLDqb82J8RlzPs6N5PrPVqIHpAOwOBQ47pi9+uIDSWzxEiEPNqYaYOGyE4Zo0iwyGpYqnyPk44cTef90c9o75b66msiPe2uHSdDuDads1mTLN3p+MbGXrNmLr0XmWKzN2uE4WYMI2+UkyEjk7vPIruD3QiRpaNoQ1DXCk\\\\\\\/6iUBlBfRNL7Qvq8UUbYk1NkhL3+GfKheyCacACUMN+7lQnXy8J6LkX\\\\\\\/ZhJpf+0WZEWmSs5AN8\\\\\\\/qYFhUdz7oCOWCon9g4uh5Kzoca6jXAursqPqO8Vkpn7Zz1tK7o4wzucN\\\\\\\/FjmGBVQEdfwft7KO5YcQn3MaJfHby51aMzMLvqrVrMfOPu6Wv0WmEEz0O95WA7\\\\\\\/3Tg2SDajuht\\\\\\\/9wy+SO+cMgfp8bAuVU4tFCPR7WgxNPvcyHHej7W7TBURncii+BwCIPRJ6awZzkKTjVjmH5Od+22fn+n2cJlLeIzn+bpN33xFK00cOBYkPmCBIOl6RswBiRCWj1A7NI+4QD\\\\\\\/Pub4Z5uFRSKAd7O74mI8nrOCu83MmWOdWydVwGkK3xY2ARJf6pjzNTj34ErZ1AHz1L\\\\\\\/ze+c0gw4SYBqV\\\\\\\/NfJ\\\\\\\/lnygeasaEj0yXPcIHkLs\\\\\\\/Ndv+XQnh6oONOumQkulf0FxKGkXahlFr1cPomU\\\\\\\/UKwT3OMquytFhVu+GICgB+tKMctmG8z\\\\\\\/OpHjvXhJNZfQIK6D7u0CriBydrnrR6OprbDq+S7AT7lc2UIrC9mlcPTP315S2XM9Z+5aQkojxPhDYSdVdW7DEkqB79P3558+W4ew8+euKj2XLXVlUrduIL9WSEzHtPEw9z8jhBB+NrDfq+42ONzebsQawg6waEAriTWt0Zd6avexIdf0dgzLthgQc\\\\\\\/0T9o\\\\\\\/RAPFV6Sa0sN0CeKNorQliaa4mi70Z0p4\\\\\\\/ZSmHtsHV2eFr\\\\\\\/Q4nDjKtrsDh0R1BELQ5iCHR9MGUKzqLVHht3dGMrVQJzpS8E1rmBI8fIQ9dHDhKyaVhbdbD4amaZrAQM\\\\\\\/Zl2iMHrvL6fKw1T1gJtTfyJHy3qjCzsZq9u8iHbz8Jz8PyypzR0x1RpB5gi\\\\\\\/DHZ0WjzDH2XCLIzXvuEW+MXTQXJnfwh7CkQYtGWh2SxEPw3BkW5+U763H1IWXSc3LP3gu\\\\\\\/GQ7o+chs\\\\\\\/D9hbRo7aSiYPd\\\\\\\/uiD30R1amyFhZGy5J8+hRI3lTq0OixB5sclQgeNJbCFBiMSZpQApmr9BMQyFD0rJx7078rAIeyRq8G42kr70QkBxwRVYDPzlZdok0woqEg+qR28khPfJRMB5+UMB6uM0dUSt\\\\\\\/7u9OhxsvBqLEzQ3xWEOkcQXj5+1iO7K1SvVj6B4pYWLV7UFhKLs6sc9vBUvS5vwTSZenqP2lc+UzZOOYMVR8d80VN3LZ7iRLsy4siLXrmiCeNbARlnuCZcpBF9HLoiXkataLvoG0+\\\\\\\/vVVpl0lZUdCG5ZUgd+1hdiuOF7bhrG+MZwBX\\\\\\\/32f9VPbHTp5mAREylRzwTinl1nuKTeiM7+98wOAev4f42kcVXgXLlXESxetN7t8FQws91sMpFbDyn3YR\\\\\\\/UKwmSkmGd57ImjnqbXR7s46fA74cdUPFVoyjKYBXqbvA1x\\\\\\\/HcG\\\\\\\/UrJIFgKcLVC1uCWfahwz\\\\\\\/mI2V66R\\\\\\\/8orlpt\\\\\\\/L0gvCOnum5MuCcJXzubSaqLA6wrrHSm0qGFXd7L0gvcChnC66HPi+OsaQjFVQr3UDSxbKGGcYsA67n7Tl7DjzAcYDLC9PC5GI5RFhUGA5UgF\\\\\\\/z+DML7CYkG09gZqscoqiRaWpm0AxNTfAidUPBDadw5pcZelpsMy7\\\\\\\/F9VOB+EPsuvEWMTc23LNWMyVyaWATEih0gWbZ3EsHYal5ldaV7bcEwrW2Dndi1e2uJGhPi9uzZarkv76IrJtgr3hkpFuhIOHsfPy+lwEdR8TDV7W1XyYIDmblWq6Zg03cguLfhXM8lHZMqWB4DgrHrAIXmv3IO61bsZ1LoLQEt4yhqYHIux+5kUF5FAhiuQHUvhHGKvPSHOmT28mtm6dgOX87VW8tqt+hj11DfY2SMxkIHGfXaDYQCkOLH7pbzZWZdHK1rS1N4ej2hbzwXpFkW3pOGcHupU3gXaX9KuL6x8eEfVtAudLIVr5tRpJAJmVupaJrvT7rXHXeZOLfo64Oq6jIYjGJduXUnhiSHP9\\\\\\\/iR2OablC1P6XcTUu5FL6D\\\\\\\/JZ1KaQ+Hc+vfaTlA\\\\\\\/re8ee7vSIajUxeRt9hCzfO5e9befYcHA4giVFlQ4dcyVOX4uwLUfSVK8nNeBj5jd7EHetSnn+2Ou4GMl16lILTdCZSvsLTEWo92f2ufueAVdoq7PkEC8xwqTwFhklOWfvmoPnZNQ\\\\\\\/TkOsSfTq4sbosDJr7XXY1i78VAXsBkp3mGiwy5lz+jdRMwmgvM7xDDlyRK9P+ijYCz2\\\\\\\/yDKkE0UVusqQzTRHV+6Wd9+PFdUV5Kp1lJU0VABksspJYgYb2KoLnwUvV+1g23YKeDmvIspPiezBZrupxayHLtMJY\\\\\\\/9n+ibZE6TX7CcwiGKif5y5BQEGC9mKUDHWd4DjawN4XLxPqZAjHjk+8eGNQvZBOCltGOrJjvSO1BKQa29pjzw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022h4zjHFWtJGrYjCQ+IVDdug==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022317fc276afb9f5a8db056328aa738b68065ac971264be3f91441efa8adbccea2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221LcjPI3H3CK4trJdLNvjEd+b6Vhlf972SSib+Clg\\\\\\\/7HQZmzoBGk4Z35ofE5ue67AxbR6pgbWt2azhpcgJ8HDtkkVvM4HNB2uBBikt\\\\\\\/vsc9fWsJZAD7NSYcrfDo+fFw5GoZcQtSR6OWhBLHXAH4vPTyjnZ+a8rE6Sl2vdMW176H0mlEzqT4OmJv5N27e55YEsTwkPU1X3KOdwYDYNzxtQp6p2pOnCFJwXv6YE\\\\\\\/Ua+s8tQOJ9cYkyS2ZAxyXaSvQCiDZo\\\\\\\/osc\\\\\\\/1SATL9mRLPYQ29kmrALUQjAZ8l\\\\\\\/1iMUbieLJMjuG\\\\\\\/K+yPuYrMleb46egrVS6YVqppMW+wxHdG21WI4IwAbBrvXE8GB6yRk7nW\\\\\\\/Wih5hXYN31jVcM7ICn0jNAeSKMqWKEtPbMR2Sp5cuuz8\\\\\\\/aQu4\\\\\\\/4RycAwZlFvB\\\\\\\/yXxZLk0sgOSCWIL8vm7eTj\\\\\\\/6c8120izZ5A2Nr0x0H0v5u+xIPRaiXrXTsdcwaIYQZzmdHNfc+u8GBIMT3kg9aRwJQGZ4WnlAN28YIpxPcfDtfODObLUZaVvW9w4uGTf6MSM4YcmYN39qDhdZPh3JFYojXUeZGL08mKDQ2c12gKnNtuLZnFKLePksBtnPnqCDiycrMyCtPKcZI82F4L8x9cA9ctvSObsSfiBlKdYgNpQzcpppHSzZGXy2po9eQc7WCL3laZ7R8ZpkRrRjcWj\\\\\\\/0Ze21MPsvZyIkkP\\\\\\\/hv0vvENH\\\\\\\/pg\\\\\\\/tJTRu1NosaXUJdICxCfwXOItnTU6nfrQ6ChzS56I28LKOT8R\\\\\\\/oeN87GAtGAqagnA5di4heJkXD3eaQuZ0vvY31GuqijyN99uev+Jr5\\\\\\\/qxLv+VVkVUhyGdKZsLTFsEABDwu+q4qBzYKfRKPBWrkh4TDNHkrPR\\\\\\\/waQjgzPrB7ssW1nCsRB5nFYqN+xUcWHtUWX1JcAMfKs0570q9y4eZaiMVQDzEnxM53Y54CD1glBdYMxCLJ8XnoIElTNsJQHNAt1YCjJC+XD5h4Kaa7r9gJM8TtqSVp8bQPnCTSoCufB8erd\\\\\\\/pKYtj34qnKLETqNdmUgI\\\\\\\/Y2fo9cTyqPfnqFE82YRZhlERDwsUK7Im55MxgJrsfOM6W2GPO4L5XNODi5Pc37ZX3IX2JC7Cyb+IpQsXvL4J7B0W16EfKTpvGx+uDmYA6\\\\\\\/HBJQq+nInoUJdz0e2Y4PzrLPY1WDraJAXYdghLoFn3bXwqgY+\\\\\\\/mYg5nMNkbPsumCt9ZnTLUcU2PiOtRqjBUMC5Q+zy1woW65jT9d7zaoRTZsYhqzzkDk6xhL\\\\\\\/zl8cxi8JQUZ8zpdhgXqsek0AXtnDMW2EiDKhEtbhDiDjB0IkxOvmZ8NKi5j\\\\\\\/3Dgi5PPoyl04yNjrzdSJhevHD4u5wpH3jJueolPtGLHL2j4utgcadqxPRsQXwgU3E+hZ7EQdDBIRK78mLiGmnNvL9oXjfgwQlAuFQzHCyyQ13E7685IIqsoVbFTHJdWafijJmEdLTZ8DmtuqrESn\\\\\\\/RabXdL\\\\\\\/mBLv78n+Jql2w24rgJ9DhSDTiHmK3P+qXVURGwV3e+vynUIpNdzQn2fwzD1WqMlXqFhIu0OWfOMYO77xLawoMUaf45a2g8i434\\\\\\\/kXESBU8su\\\\\\\/UyTLyeJI0TaEmpbN4ae1wudeMTpxQBDfErru9tANH3VZ89WI8SlC2RmuKtB+3Qc+Qyav4ENZzP12wejjnJZnm9fLR18FiDjlu4qfT4VVYlw0W0bZ325fY48pJCL6BOM+ZDdAPWBwdOxkaktOimgBOF3vu9YEBWxl+VHTUGGzge9vhhT9K38ZZ0QrEOdJW47eNUek8yACCGNCvdDDxo9TMlfXYtwrhJbw5bGmXDwqxDVFja8A3OvqnJIViO861rxGzTsE0uqjrbtwAifESEXD1Jtl3cKbPzHObQUQ1HTxe5IHHPxaUM++ilL\\\\\\\/fCFQLoLclosUtsCq7+WiNueepdO0vCDSH0HoBjDecOk6LIcrSqrUasuuMc3TX17unA61PlBI5o\\\\\\\/FOwgMMap4l956ysyg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GkWw6KaiuGD852d7inMJaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226985c506abcfeb2c1e4c6e6c3c62c8c665a4476c3cdc8b15836de80623a117c0\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492857 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1545" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1BaLnqTbehxOwZpQXSx3dLKXVWn7vjg9VhLvnwNW26c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1545" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nQiEDRHCvMi87O2FK-Vjiyrh15TWFWmgEP1cfJyX_9A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1533" title="Variation in Filamentous Growth and Response to Quorum-Sensing Compounds in Environmental Isolates of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1557" title="Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492857" id="node492857" data-pisa="ggg;9/5/1545" data-pisa-master="ggg;g3.119.300585" data-apath="/ggg/9/5/1545.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Javier Martín-Vallejo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span>, <span class="highwire-citation-author" data-delta="5">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="6">Philomin Juliana</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1545-1556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.300585 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492857" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1545" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492857" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1545.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492857" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1545.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492857" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1545.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492857" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multiple-trait experiments with mixed phenotypes (binary, ordinal and continuous) are not rare in animal and plant breeding programs. However, there is a lack of statistical models that can exploit the correlation between traits with mixed phenotypes in order to improve prediction accuracy in the context of genomic selection (GS). For this reason, when breeders have mixed phenotypes, they usually analyze them using univariate models, and thus are not able to exploit the correlation between traits, which many times helps improve prediction accuracy. In this paper we propose applying deep learning for analyzing multiple traits with mixed phenotype data in terms of prediction accuracy. The prediction performance of multiple-trait deep learning with mixed phenotypes (MTDLMP) models was compared to the performance of univariate deep learning (UDL) models. Both models were evaluated using predictors with and without the genotype × environment (G×E) interaction term (I and WI, respectively). The metric used for evaluating prediction accuracy was Pearson’s correlation for continuous traits and the percentage of cases correctly classified (PCCC) for binary and ordinal traits. We found that a modest gain in prediction accuracy was obtained only in the continuous trait under the MTDLMP model compared to the UDL model, whereas for the other traits (1 binary and 2 ordinal) we did not find any difference between the two models. In both models we observed that the prediction performance was better for WI than for I. The MTDLMP model is a good alternative for performing simultaneous predictions of mixed phenotypes (binary, ordinal and continuous) in the context of GS.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/deep-learning" class="hw-term hw-article-keyword hw-article-keyword-deep-learning" rel="nofollow">deep learning</a></li><li class="kwd"><a href="/keyword/multiple-trait" class="hw-term hw-article-keyword hw-article-keyword-multiple-trait" rel="nofollow">multiple-trait</a></li><li class="kwd"><a href="/keyword/mixed-phenotypes-binary-ordinal-and-continuous-0" class="hw-term hw-article-keyword hw-article-keyword-mixed-phenotypes-binary-ordinal-and-continuous" rel="nofollow">mixed phenotypes (binary ordinal and continuous)</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/plant-breeding" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding" rel="nofollow">plant breeding</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Many times a breeder wishes to genetically improve more than one trait but with mixed phenotypes. For example, grain yield (GY) measured on a continuous scale, presence or absence of a certain disease or level of infection of a certain disease (non-infected, low level of infection, middle level of infection, high level of infection and totally infected). However, when breeders wish to analyze the data with mixed phenotypes, (a) they usually analyze the data using univariate statistical models; (b) they subject the discrete variables (binary and ordinal) to some numerical score (using transformations), so that all variables can be treated as continuous, and then they apply conventional multivariate analysis; and (c) the continuous variables are discretized through some grouping criteria, which allows all variables to be treated as discrete and then analyzed under a multivariate model for discrete responses. However, these three approaches involve some level of subjectivity since the first approach does not take into account the correlation between traits, the second method introduces a high level of subjectivity in the numerical scoring scheme and the third solution produces considerable loss of information due to the discretization process of the continuous variables (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Krzanowski 1983</a>). Although there are multivariate techniques for regression analysis for association studies and prediction modeling (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Montesinos-López <em>et al.</em>, 2016</a>), versatile regression models for mixed outcomes are lacking, since conventional multivariate statistical tools generally rely on the assumption that the data, or suitable transformations of them, follow a normal distribution.</p><p id="p-4">Plant breeders have a long tradition using multivariate models for association studies and prediction modeling; however, with the new paradigm called genomic selection (GS) proposed by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Meuwissen <em>et al.</em> (2001)</a>, there is renewed interest in multivariate modeling to exploit the correlation between traits to improve parameter estimates or prediction accuracy. <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jia and Jannink (2012)</a> provided evidence that multivariate analyses outperform univariate analysis when there is at least moderate correlation between traits. <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jiang <em>et al.</em> (2015)</a> and <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Montesinos-López <em>et al.</em> (2016)</a> came to the same conclusion in favor of multivariate analysis. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">He <em>et al.</em> (2016)</a> and <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schulthess <em>et al.</em> (2017)</a> also found that, compared to single-trait analysis, multivariate analysis could improve prediction accuracy for correlated traits. However, the application of traditional multivariate models in the context of GS is not straightforward due to the problem of the large amount of independent variables (marker information and environmental information) and few observations (lines), a problem commonly known as “large p and small n”. For this reason, in recent years, intensive research has been conducted to develop statistical models (or adapt conventional models) for the context of genome-wide association analysis and genome-enabled prediction (GP). In the context of univariate and multivariate models with continuous phenotypes for GP, Bayesian models have proved to be more efficient than models based on maximum likelihood or restricted maximum likelihood because they are better suited for dealing with data sets with large p and small n. As evidence of this, the term “Bayesian alphabet” was coined; it refers to the growing number of letters of the alphabet used to denote various Bayesian linear regressions used in GP that differ in the priors adopted, while sharing the same sampling model (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gianola 2013</a>). <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gianola (2013)</a> also pointed out that the Bayesian alphabet is of paramount importance in whole-genome prediction of phenotypes, but has somewhat doubtful inferential value, at least when the sample size is such that n ≪ p.</p><p id="p-5">However, to date there are no Bayesian regression models for GP for mixed phenotypes due to the difficulty of developing efficient analytic Gibbs samplers to perform a Markov Chain Monte Carlo (MCMC) algorithm for approximating a specific multivariate probability distribution, when direct sampling is difficult. But nowadays there are computationally efficient methodologies for predicting multiple-trait (called multiple-output) response variables (phenotypes) in deep learning (DL), which is a part of a broader family of machine learning methods based on learning data representations, as opposed to task-specific algorithms. In general terms, machine learning is devoted to developing and using algorithms that learn from raw data in order to make predictions (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wiley 2016</a>).</p><p id="p-6">Inspiration for DL models is rooted in the functioning of biological nervous systems. These models are not new because their roots trace back to the introduction of the McCulloch-Pitts (MCP) model, which is considered the ancestor of the artificial neural model (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">McCulloch and Pitts 1943</a>) that has now gone mainstream thanks to its practical applications and availability in terms of consumable technology and affordable hardware. An artificial neural network (ANN) models the relationship between a set of input signals and an output signal using a model derived from our understanding of how a biological brain responds to stimuli from sensory inputs. Similar to how the brain uses a network of neurons (interconnected cells or units) to create a massive parallel processor, ANN uses a network of artificial neurons to solve learning problems (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lantz 2015</a>). For this reason, an ANN is described as a directed graph whose nodes correspond to neurons and whose edges correspond to links between them. Each neuron receives as input a weighted sum of the outputs of the neurons connected to its incoming edges (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Feedforward networks (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) are those in which the underlying graph does not contain cycles (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Shalev-Shwartz and Ben-David 2014</a>). Thanks to major innovations in the field of neural networks, a technique that is known as deep learning has emerged. The term deep refers to the fact that we can now train different ANN configurations with more than a single hidden layer, such as the conventional multilayer perceptron, which has shown to have better generalization capabilities (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goodfellow <em>et al.</em>, 2016</a>). The adjective “deep” applies not in itself to the knowledge acquired, but to the way in which knowledge is acquired (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lewis 2016</a>). In other words, DL is a subfield of machine learning that generalizes conventional neural networks to work with more than two hidden layers and more neurons; it is devoted to building prediction algorithms that explain and learn a high and low level of abstraction (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gibson and Patterson 2017</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of a feedforward neural network with 8 input variables (x1,&#x2026;,x8), four output variables (y1, y2, y3, y4), and two hidden layers with three neurons each." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example of a feedforward neural network with 8 input variables (x1,&#x2026;,x8), four output variables (y1, y2, y3, y4), and two hidden layers with three neurons each.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F1.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1545/F1.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493086" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Example of a feedforward neural network with 8 input variables (x1,…,x8), four output variables (y1, y2, y3, y4), and two hidden layers with three neurons each.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">For this reason, DL models have been implemented in many areas of knowledge: prediction of time series (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dingli and Fournier 2017</a>); language processing (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Goldberg 2016</a>); self-driving cars (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Liu <em>et al.</em>, 2017</a>); predicting breast, brain (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cole <em>et al.</em>, 2017</a>) or skin cancer using personalized medicine based on Biobank-data, voice search and voice-activated intelligent assistants (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">LeCun <em>et al.</em>, 2015</a>); automatically adding sound to silent movies; automatic translation of text and images (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">LeCun <em>et al.</em>, 2015</a>); automatic text generation; automatic handwriting generation (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">LeCun <em>et al.</em>, 2015</a>); image recognition (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">LeCun <em>et al.</em>, 2015</a>); automatic image captioning (that is, given an image, the system must generate a caption that describes the content of the image); automatic colorization; advertising; predicting earthquakes (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rouet-Leduc <em>et al.</em>, 2017</a>); energy market price forecasting (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Weron 2014</a>); and genomic prediction (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em>, 2018a</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">b</a>).</p><p id="p-9">There are also many applications of DL and machine learning for the biological sciences. For example, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fox <em>et al.</em> (2018)</a> applied DL for predicting blood glucose trajectories, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Menden <em>et al.</em> (2013)</a> predicted cancer cell sensitivity to drugs with machine learning, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alipanahi <em>et al.</em> (2015)</a> used DL for predicting the sequence specificities of DNA- and RNA-binding proteins, and <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Tavanaei <em>et al.</em> (2017)</a> applied DL for predicting tumor suppressor genes and oncogenes. Recently, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em> (2018a</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">b</a>) have shown how to apply DL methods with densely connected network architecture and several hyperparameter combinations to extensive plant breeding data involving different traits and environments. Compared with the standard genomic model and ignoring genotype × environment interaction (G×E), the DL method was better than the conventional genomic models in terms of prediction accuracy. The previously mentioned authors also developed DL models and methods for multiple traits (all continuous traits) and compared their prediction accuracy with a Bayesian multiple-trait multiple-environment model. Among models without genotype <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-1.gif"/></noscript></span></span>environment interaction, the multiple-trait DL model was the best, while among models with genotype <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-2.gif"/></noscript></span></span>environment interaction, the Bayesian multiple-trait multiple-environment model was superior. However, no models or methods have been developed for genomic selection and prediction in animal and plant breeding that can incorporate together binary, ordinal and continuous traits and their G×E when collected in different environments.</p><p id="p-10">For the reasons outlined above, we applied DL methods for predicting multiple traits with mixed phenotypes (binary, ordinal and continuous traits). We also compared their performance with that of univariate deep learning models where a model was individually trained for each trait. To evaluate the performance of both models (multiple-trait and univariate-trait deep learning), we used seven wheat data sets from the Global Wheat Breeding Program of the International Maize and Wheat Improvement Center (CIMMYT). The performance of both models in terms of prediction accuracy was evaluated using cross-validation in terms of average Pearson’s correlation (APC) for continuous traits and in terms of the percentage of cases correctly classified (PCCC) for binary and ordinal traits.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Multiple-trait deep learning with mixed phenotypes (MTDLMP) model</h3><p id="p-11">While there are different DL architectures (densely connected networks, convolutional networks, recurrent networks, etc.), in this paper we focus on a standard densely connected network. Details of each type of network, its assumptions and input characteristics can be found in <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gulli and Sujit (2017)</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Angermueller <em>et al.</em> (2016)</a> and <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chollet and Allaire (2017)</a>. A densely connected network does not assume a specific structure in the input features. In general, the basic structure of a densely connected network consists of an input layer, L output layers (for multiple-trait modeling) and multiple hidden layers between the input and output layers. This type of neural network is also known as a feedforward neural network (see <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-12">In this paper we implemented the architecture shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> for the seven data sets used in this study with 1, 2 and 3 hidden layers and number of neurons (from 10 to 500 with increases of 20). As input variables (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-3.gif"/></noscript></span></span> i = 1,2,..,n; <em>P</em> = 1,2,..,<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-4.gif"/></noscript></span></span>) for the proposed deep learning model, we included predictor variables as illustrated in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>, since we used as input information for each line the resulting covariates of concatenating information on environments, information on markers through the Cholesky decomposition of the genomic relationship matrix and information on the genotype × environment (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-5.gif"/></noscript></span></span> interaction. This meant that first we built the design matrices of environments (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-6.gif"/></noscript></span></span>, genotypes (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-7.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-8.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-9.gif"/></noscript></span></span> then we obtained the Cholesky decomposition of the genomic relationship matrix (G). Then we post-multiplied the design matrix of genotypes by the transpose of the upper triangular factor of the Cholesky decomposition (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-10.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-11.gif"/></noscript></span></span>, and finally the <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-12.gif"/></noscript></span></span> term was obtained as the product of the design matrix of the <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-13.gif"/></noscript></span></span> term post-multiplied by the Kronecker product of the identity matrix of order equal to the number of environments and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-14.gif"/></noscript></span></span>, that is, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-15.gif"/></noscript></span></span>). Finally, the matrix with input covariates used for implementing both deep learning models was equal to <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-16.gif"/></noscript></span></span>. The input variables (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-17.gif"/></noscript></span></span>) are connected to the neurons in the first hidden layer via weights. The input variables simply pass the information to the units in the first hidden layer. The net input into the jth hidden unit in the first hidden layer is <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-18.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-19.gif"/></noscript></span></span> is the total number of input variables, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-20.gif"/></noscript></span></span> is the weight of input unit p to hidden unit j in the first hidden layer, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-21.gif"/></noscript></span></span> is the value of the pth input variable and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-22.gif"/></noscript></span></span> is a bias specific to neuron (unit) j in layer 1. Then the jth hidden unit in the first hidden layer applies an activation function to its net input and outputs <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-23.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-24.gif"/></noscript></span></span>. Similarly, neuron <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-25.gif"/></noscript></span></span>in the second hidden layer receives a net input <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-26.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-27.gif"/></noscript></span></span> is the total number of input neurons that come from hidden layer 1 to neuron <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-28.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-29.gif"/></noscript></span></span> is the weight from unit j of layer 1 that goes to unit k in hidden layer 2, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-30.gif"/></noscript></span></span> is the value of the output of unit j in layer 1 and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-31.gif"/></noscript></span></span> is a bias specific term to neuron k in layer 2. Then the kth hidden unit in the second hidden layer applies an activation function to its net input and outputs <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-32.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-33.gif"/></noscript></span></span>. Similarly, output unit t with <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-34.gif"/></noscript></span></span> receives a net input of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-35.gif"/></noscript></span></span>, where M is the number of hidden units from hidden layer 2, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-36.gif"/></noscript></span></span> represents the weight from hidden unit k in layer 2 to output t. Finally, the prediction of an individual in trait t is obtained as: <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-37.gif"/></noscript></span></span>. It is important to point out that in the output layer (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-38.gif"/></noscript></span></span>, the sigmoid, softmax and rectified linear units (ReLU) activation functions were used for binary, ordinal and continuous traits, respectively.</p><p id="p-13">Successful implementation of most DL models requires an appropriate hyperparameter tuning process. However, implementing a feedforward neural network is challenging because it requires a tuning process of the following hyperparameters: number of units (U), number of layers, number of epochs (E), type of regularization method and type of activation function. Based on the literature review, we decided to use the ReLU, sigmoid and softmax activation functions for the continuous, binary and ordinal response (output) variables, respectively, while for the hidden layers, we used the ReLU activation function. As for the type of regularization, we chose dropout regularization for training the models (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gulli and Sujit 2017</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chollet and Allaire 2017</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Srivastava <em>et al.</em>, 2014</a>), and for the hidden layers we used 1, 2 and 3 hidden layers.</p><p id="p-14">Concerning the number of epochs and number of units in the hidden layers, we performed a grid search. The grid search was done with number of epochs from 1 to 100 and number of units between 10 to 500 with increases of 20. For more details on model selection in DL models, we suggest reading the papers of <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em> (2018a</a>, <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">b</a>), where the authors evaluate the prediction performance of univariate and multivariate DL models for continuous response variables. It is important to point out that we also implemented the univariate counterpart of the MTDLMP model described at the beginning of this section, where each trait was implemented using a univariate deep learning (UDL) model with exactly the same architecture given in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>, except that it only had one output variable. When the output variable was continuous, ReLU activation was used, whereas the sigmoid and softmax activation functions were used for the binary and ordinal response (output) variables, respectively. Both the MTDLMP and UDL models were implemented in the keras package (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Chollet and Allaire 2017</a>) in the open-source software R (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">R Core Team 2018</a>).</p></div><div id="sec-3" class="subsection"><h3>Evaluating prediction performance with cross-validation</h3><p id="p-15">The prediction accuracy of both models (MTDLMP and UDL) was evaluated with an outer CV while an inner CV was used for tuning the hyperparameters. The outer CV consisted of a fivefold CV, where the original data sets were partitioned into five subsamples of equal size and each time four of them were used for training (TRN) and the remaining one for testing (TST). In our outer CV, one observation cannot appear in more than one fold. In the design, some lines can be evaluated in some, but not all, target environments, which mimics a prediction problem faced by breeders in incomplete field trials. For this reason, our cross-validation strategy is exactly the same as the strategy denoted as CV2 that was proposed and implemented by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jarquín <em>et al.</em> (2017)</a>, where a certain portion of test lines (genotypes) in a certain portion of test environments are predicted since some test lines that were evaluated in some test environments are assumed to be missing in others.</p><p id="p-16">The metric used to measure the prediction accuracy of both models was Pearson’s correlation for continuous traits and the percentage of cases correctly classified (PCCC) for the binary and ordinal variables. They were calculated from each trait-environment combination for each of the testing sets and the average of all folds was reported as a measure of prediction performance. It is important to point out that, to avoid biased results, the tuning step was done in each fold using only the training set.</p><p id="p-17">For the tuning process we implemented the inner CV with the proposed grid for the number of epochs and units, and 20% of each training set was used as a validation set (validation-inner). Due to the amount of data and the complexity of the MTDLMP and UDL models, the training process requires a lot of time for the tuning process; for this reason, the training was performed using the internal capabilities of keras, where we set the validation_split argument on the fit() function to 20% of the size of each of our training data sets. This automatic validation procedure of keras implemented the inner CV and evaluated the performance of the model on the validation data set for each epoch and avoided implementing manual k-fold cross-validation for the inner CV, which requires more computational resources (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Chollet and Allaire 2017</a>).</p><div id="sec-4" class="subsection"><h4>Experimental data sets:</h4><p id="p-18">In this study we used the data set used by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Juliana <em>et al.</em> (2018)</a>. The data used belong to four elite yield trial (EYT) nurseries from the Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT). The EYT nurseries were planted in mid-November because that is the best time to plant CIMMYT’s yield trials. Bed and flat planting systems in optimally irrigated environments received 500 mm of water at the Norman E. Borlaug Research Station, Ciudad Obregon, Sonora, Mexico. The nurseries were sown in 39 trials, each comprising 28 lines and two high-yielding checks (Kachu and Borlaug) that were arranged in an alpha lattice design with three replications and six blocks. The nurseries were evaluated for the following traits: number of days from germination to 50% spike emergence (days to heading, DTHD), number of days from germination to 50% physiological maturity (days to maturity, DTMT), grain yield (GY, tons per hectare) and plant height (Height, centimeters). All these nurseries were evaluated during four seasons 2013-2014 (EYT 13-14; here called <strong>data set 1</strong>), 2014-2015 (EYT 14-15; called <strong>data set 2</strong>), 2015-2016 (EYT 15-16; called <strong>data set 3)</strong> and 2016-2017 (EYT 16-17; called <strong>data set 4</strong>).</p><p id="p-19">Data set 1 included 767 lines, data set 2 had 775 lines, data set 3 comprised 964, and data set 4 had 980 lines (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Juliana <em>et al.</em>, 2018</a>). In addition, in each season we studied six environments resulting from the level of irrigation (IR) and planting system (bed or flat) which we called: Bed2IR, Bed5IR, Flat5IR, FlatDrip, EHT and LHT. However, all these environments were not evaluated in all seasons (data sets). In <strong>data set 1</strong>, only environments Bed5IR, EHT, Flat5IR and LHT were evaluated. In <strong>data set 2</strong>, the evaluated environments were: Bed2IR, Bed5IR, EHT, Flat5IR and LHT. In <strong>data set 3</strong>, the evaluated environments were: Bed2IR, Bed5IR, Flat5IR and FlatDrip, where 5IR and 2IR refer to 5 and 2 irrigation levels, EHT denotes early heat, LHT is late heat, and bed and flat are two different planting systems. In <strong>data set 4</strong>, the evaluated environments were: Bed5IR, EHT, Flat5IR and FlatDrip.</p><p id="p-20">It is important to point out that here we used the BLUEs of each of the lines obtained (as suggested by <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Juliana <em>et al.</em>, 2018</a>) adjusted for trials, blocks and replications in each data set. Three of the four traits were discretized because the original data sets are continuous, only to illustrate the MTDLMP model. Traits DTHD and DTMT were discretized at quantiles 33.33% and 66.67% (in <strong>data sets 1</strong> and <strong>2</strong>) to obtain three categories, while trait Height was discretized at quantile 50% to obtain 2 categories (in <strong>data sets 1, 2, 3</strong> and <strong>4</strong>) the discretization process was done in each environment of each data set. For <strong>data sets 3</strong> and <strong>4</strong>, traits DTHD and DTMT were discretized at quantiles 20%, 45%, 70% and 90%.</p><p id="p-21"><strong>Data set 5</strong> is part of <strong>data set 3</strong>; for this reason, the phenotypic information and genomic information were obtained in the same way as in <strong>data set 3</strong>; however, only 964 lines had complete data of the total 980 lines under study in <strong>data set 3.</strong> But now the traits measured in <strong>data set 5</strong> were grain color (GC) (1 = yes, 2 = no), leaf rust (ordinal scale with 5 points), stripe rust (ordinal scale with 3 points) and GY, which is a continuous trait. <strong>Data set 6</strong> and <strong>data set 7</strong> are part of the wheat yield trial (YT) nurseries from CIMMYT’s Global Wheat Breeding Program. For <strong>data set 6,</strong> the number of lines used was 945, and for <strong>data set 7</strong>, 1145 wheat lines were used. A continuous trait (grain yield, GY) and an ordinal trait (lodging; ordinal scale of 5 points) were measured on both data sets.</p><div id="sec-5" class="subsection"><h5>Genotypic data:</h5><p id="p-22">All 4,368 lines evaluated in the four seasons (nurseries) comprising the EYT of <strong>data sets 1</strong>, <strong>2</strong>, <strong>3</strong>, and <strong>4</strong> were genotyped using genotyping-by-sequencing (GBS) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Elshire <em>et al.</em>, 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Poland <em>et al.</em>, 2012</a>) at Kansas State University, using an Illumina HiSeq2500 for obtaining genome-wide markers. Marker polymorphisms were called across all lines using the TASSEL (Trait Analysis by Association Evolution and Linkage) GBS pipeline (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Glaubitz <em>et al.</em>, 2014</a>) and anchored to the International Wheat Genome Sequencing Consortium’s (IWGSC) first version of the reference sequence (RefSeq v1.0) assembly of the bread wheat variety Chinese Spring. Markers with more than 60% missing data, less than 5% minor allele frequency and percent heterozygosity greater than 10% were removed and we obtained 2,038 markers. Missing marker data were imputed using LinkImpute (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Money <em>et al.</em>, 2015</a>) implemented in TASSEL (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bradbury <em>et al.</em>, 2007</a>), version 5. The lines were also filtered for more than 50% missing data, found in 3,485 lines (767 lines from <strong>data set 1</strong>, 775 lines from <strong>data set 2</strong>, 964 lines from <strong>data set 3</strong> and 980 lines from <strong>data set 4</strong>) (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Juliana <em>et al.</em>, 2018</a>). The lines used in <strong>data sets 5</strong>, <strong>6</strong>, and <strong>7</strong> were genotyped with the same marker system that was used for the other data sets.</p></div></div></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-23">All seven data sets (Data Sets 1-7) including phenotypic and genotypic data plus the Supplementary Material with the 14 figures (Figure D1_SA, D1_SB-D7_Sa, D7_SB) can be downloaded from the following link: <a href="http://hdl.handle.net/11529/10548140">http://hdl.handle.net/11529/10548140</a>. </p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-24">The results are given in seven main sections. Each section describes the results of one data set. Each section is divided into two subsections, one that includes a descriptive analysis and another that reports the genomic-enabled prediction accuracy of the proposed models.</p><div id="sec-8" class="subsection"><h3>Data set 1</h3><p id="p-25">This data set had four traits (one binary, two ordinal and one continuous). Across environments, the binary trait (Height) had 53.6% of cases for category 2 and 46.4% for category 1 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Traits DTHD and DTMT had similar distributions of individuals between categories. The average GY was around 6 ton/ha for three environments (Bed5IR, EHT and Flat5IR) and around 3 ton/ha in environment LHT (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of each ordinal response for data set 1 across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of each ordinal response for &lt;em&gt;data set 1&lt;/em&gt; across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F2.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1545/F2.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493034" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Percentage of each ordinal response for <em>data set 1</em> across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">There were no major prediction accuracy differences in this wheat data set between the MTDLMP and UDL models using 1 layer when <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-39.gif"/></noscript></span></span> interaction was taken into account (I) in any of the traits under study (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a>). However, when <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-40.gif"/></noscript></span></span> interaction was ignored (WI), we found statistical differences in trait GY, with better performance under the MTDLMP model (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3a</a>), and in average Pearson’s correlation (APC), the MTDLMP was superior to the UDL in prediction accuracy by 22.44%. With two layers, we only found statistical differences between the MTDLMP and UDL models in trait GY, where the MTDLMP was again superior (Figure D1_SA; Supplementary material, hdl:11529/10548140); however, with three layers, we did not find any statistical differences between the two models (Figure D1_SB; Supplementary material, hdl:11529/10548140). The PCCC using 1 layer for the binary and ordinal traits (DTHD, DTMT and Height) ranged from 0.5697 to 0.6815, while the APC for the GY trait ranged from 0.3593 to 0.4633 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3a</a>). Also, when comparing the prediction accuracy using different numbers of layers (1, 2, and 3) under the MTDLMP model (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3b</a>) and under the UDL model (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3c</a>), we did not find statistical differences in terms of prediction performance between using 1, 2, or 3 layers with the <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-41.gif"/></noscript></span></span> interaction term (I) and only found statistical differences between using 1, 2 and 3 layers in trait GY without the <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-42.gif"/></noscript></span></span> interaction term (WI).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of data set 1 in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson&#x2019;s correlation for trait grain yield (GY). (A) Prediction accuracy of MTDLMP and UDL models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait with 1 layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI); and (C) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy of &lt;em&gt;data set 1&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson&#x2019;s correlation for trait grain yield (GY). (&lt;strong&gt;A&lt;/strong&gt;) Prediction accuracy of MTDLMP and UDL models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait with 1 layer; (&lt;strong&gt;B&lt;/strong&gt;) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI); and (&lt;strong&gt;C&lt;/strong&gt;) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F3.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/5/1545/F3.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493079" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Prediction accuracy of <em>data set 1</em> in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson’s correlation for trait grain yield (GY). (<strong>A</strong>) Prediction accuracy of MTDLMP and UDL models with the G×E term (I) and without the G×E term (WI) for each trait with 1 layer; (<strong>B</strong>) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G×E term (I) and without the G×E term (WI); and (<strong>C</strong>) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G×E term (I) and without the G×E term (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Data set 2</h3><p id="p-29">Trait Height (binary trait) across environments had 52.8% of cases in category 2 and 47.2% in category 1. Traits DTHD and DTMT had similar distribution of individuals in each of the 3 categories. In trait GY, the average GY was 6 ton/ha for three environments (Bed5IR, EHT and Flat5IR), around 3 ton/ha in environment LHT and around 4.3 tons/ha in environment Bed2IR (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of each ordinal response for data set 2 across environment for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percentage of each ordinal response for data set 2 across environment for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F4.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/5/1545/F4.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492920" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Percentage of each ordinal response for data set 2 across environment for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The pattern of genomic-enabled prediction accuracy is similar to that found in <strong>data set 1</strong>. For GY with (I) and without (WI) <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-43.gif"/></noscript></span></span> interaction, the best prediction was provided by the MTDLMP model, which outperformed the UDL model by 19.21% with interaction (I) and by 40.02% without interaction (WI) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5a</a>). However, for trait GY with 2 layers, there were significant differences in favor of the MTDLMP model with the interaction term (I) (Figure D2_SA; Supplementary material, hdl:11529/10548140), while with 3 layers, no significant differences were observed between the MTDLMP and the UDL models for GY (Figure D2_SB; Supplementary material, hdl:11529/10548140). The PCCC ranged from 0.6071 to 0.7124 using 1 layer for the binary and ordinal traits (DTHD, DTMT and Height), and from 0.3301 to 0.5504 for trait GY in terms of APC (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5a</a>). Also, when comparing the prediction accuracy using different numbers of layers (1, 2, and 3) under the MTDLMP model (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5b</a>), we only found statistical differences in terms of prediction performance when using 1, 2 or 3 layers in both models with (I) and without (WI) interaction term in trait GY; the worst prediction was observed with 3 layers, and the best when using 1 layer. Also under the UDL model (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5c</a>), we only found statistical differences when using 1, 2 or 3 layers in trait GY with (I) and without (WI) the interaction term, and the best predictions were observed using 3 layers with the interaction term (I) and with 2 and 3 layers without the interaction term (WI).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of data set 2 in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson&#x2019;s correlation for trait grain yield (GY). (A) Prediction accuracy of MTDLMP and UDL models with the G&#xD7;E term (I) and without (WI) for each trait with 1 layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI); and (C) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy of &lt;em&gt;data set 2&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson&#x2019;s correlation for trait grain yield (GY). (&lt;strong&gt;A&lt;/strong&gt;) Prediction accuracy of MTDLMP and UDL models with the G&#xD7;E term (I) and without (WI) for each trait with 1 layer; (&lt;strong&gt;B&lt;/strong&gt;) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI); and (&lt;strong&gt;C&lt;/strong&gt;) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F5.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/5/1545/F5.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493144" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Prediction accuracy of <em>data set 2</em> in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson’s correlation for trait grain yield (GY). (<strong>A</strong>) Prediction accuracy of MTDLMP and UDL models with the G×E term (I) and without (WI) for each trait with 1 layer; (<strong>B</strong>) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G×E term (I) and without the G×E term (WI); and (<strong>C</strong>) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G×E term (I) and without the G×E term (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Data set 3</h3><p id="p-33">This data set also had four traits; the trait Height (binary trait) had 46.2% and 53.8.2% of observations in categories 1 and 2, respectively, across environments. The distribution pattern across the individuals in the five categories of ordinal variables DTHD and DTMT was similar. The average GY was above 6 ton/ha in environments Bed5IR and Flat5IR, around 4 ton/ha in environment Bed2IR and less than 3 tons/ha in environments FlatDrip and LHT (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of each ordinal response for data set 3 across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of each ordinal response for &lt;em&gt;data set 3&lt;/em&gt; across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F6.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/5/1545/F6.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492992" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Percentage of each ordinal response for <em>data set 3</em> across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">The genomic-enabled prediction accuracy for GY with 1 and 2 layers under MTDLMP and UDL was significantly different with (I) and without (WI) the <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-44.gif"/></noscript></span></span> interaction term; the best predictions were observed under the MTDLMP model, which outperformed the UDL model by 52.92% (with 1 layer), by 1.97% (with 2 layers) and by 22.21% (with 3 layers) with the interaction term (I), and by 67.27% (with 1 layer), by 13.34% (with 2 layers) and by 15.16% (with 3 layers) without the interaction term (WI) (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7a</a>; Figures D3_SA and D3_SB, Supplementary material, hdl:11529/10548140). The PCCC ranged from 0.4259 to 0.6746 using 1 layer for the binary and ordinal traits (DTHD, DTMT and Height), while it ranged from 0.1484 to 0.4535 for GY in terms of APC (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7a</a>). Also, when comparing the prediction accuracy using different numbers of layers (1, 2, and 3) under the MTDLMP model (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7b</a>), we found no statistical differences in terms of prediction performance between using 1, 2 or 3 layers in the four traits with (I) and without (WI) the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-45.gif"/></noscript></span></span> interaction term. Under the UDL model (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7c</a>) we only found statistical differences between using 1, 2 or 3 layers in trait GY with (I) and without the interaction term (WI).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of data set 3 in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson&#x2019;s correlation for trait grain yield (GY). (A) Prediction accuracy of MTDLMP and UDL models with the G&#xD7;E term (I) and without (WI) for each trait with 1 layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI); and (C) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy of &lt;em&gt;data set 3&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson&#x2019;s correlation for trait grain yield (GY). (&lt;strong&gt;A&lt;/strong&gt;) Prediction accuracy of MTDLMP and UDL models with the G&#xD7;E term (I) and without (WI) for each trait with 1 layer; (&lt;strong&gt;B&lt;/strong&gt;) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI); and (&lt;strong&gt;C&lt;/strong&gt;) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F7.medium.gif" width="440" height="262"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/5/1545/F7.medium.gif" width="440" height="262"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493145" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-36" class="first-child">Prediction accuracy of <em>data set 3</em> in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson’s correlation for trait grain yield (GY). (<strong>A</strong>) Prediction accuracy of MTDLMP and UDL models with the G×E term (I) and without (WI) for each trait with 1 layer; (<strong>B</strong>) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G×E term (I) and without the G×E term (WI); and (<strong>C</strong>) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G×E term (I) and without the G×E term (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Data set 4</h3><p id="p-37">In trait Height (binary trait), the first category across environments had 47.6% of cases, while category 2 had 52.4% of cases. As in the previous data sets, the distribution of individuals for both ordinal traits DTHD and DTMT in each category was similar. The average GY (continuous trait) was 6 ton/ha in three environments (Bed5IR, EHT and Flat5IR) and less than 3 ton/ha in environment FlatDrip (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of each ordinal response for data set 4 across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of each ordinal response for &lt;em&gt;data set 4&lt;/em&gt; across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F8.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/5/1545/F8.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492887" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-38" class="first-child">Percentage of each ordinal response for <em>data set 4</em> across environments for traits days to heading (DTHD), days to maturity (DTMT) and Height. Boxplot of trait grain yield (GY) for each environment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">In this data set, in terms of prediction performance using 1 layer when the <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-46.gif"/></noscript></span></span> interaction was taken into account (I), there were only significant differences between the MTDLMP and UDL models in trait GY, and the UDL model outperformed the MTDLMP model by 13.51% (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9a</a>). However, when the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-47.gif"/></noscript></span></span> interaction was ignored (WI), statistical differences were found in two (DTHD and GY) out of four traits, with a better performance under the UDL model (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9a</a>). In terms of prediction accuracy, the UDL model outperformed the MTDLMP model by 6.24% and 14.39% for traits DTHD and GY, respectively. With 2 layers, we found statistical differences between the MTDLMP and UDL models in trait GY with the interaction term (I), while the UDL model was 18.81% better than the MTDLMP; without the interaction term, we only found differences in trait Height where the MTDLMP model outperformed the UDL model by 15.71% (Figure D4_SA; Supplementary material, hdl:11529/10548140). With 3 layers and the interaction term (I), we found statistical differences between the two models in trait GY, for the UDL model outperformed the MTDLMP model by 17.71%, (Figure D4_SA; Supplementary material, hdl:11529/10548140). Without the interaction term (WI) in two traits (DTMT and Height), we found statistical differences between the two models and in both traits: the MTDLMP model outperformed the UDL model by 15.93% in trait DTMT and by 19.18% in trait Height (Figure D4_SB; Supplementary material, hdl:11529/10548140). The PCCC using 1 layer for the binary and ordinal traits (DTHD, DTMT and Height) ranged from 0.3974 to 0.6197, while the APC for the GY trait ranged from 0.4346 to 0.5105 (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9a</a>). When comparing the prediction accuracy using different number of layers (1, 2, and 3) under the MTDLMP model (<a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9b</a>), no statistical differences in terms of prediction performance were found between using one, two or three layers with the <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-48.gif"/></noscript></span></span> interaction term (I), but without the interaction term (WI) statistical differences were found between using 1, 2, and 3 layers in traits DTHD, DTMT and GY with lower predictions with 1 layer. However, under the UDL model (<a id="xref-fig-9-5" class="xref-fig" href="#F9">Figure 9c</a>), there were no statistical differences between using one, two and three layers.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of data set 4 in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson&#x2019;s correlation for grain yield (GY) trait. (A) Prediction accuracy of MTDLMP and UDL models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait with 1 layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI); and (C) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy of &lt;em&gt;data set 4&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson&#x2019;s correlation for grain yield (GY) trait. (&lt;strong&gt;A&lt;/strong&gt;) Prediction accuracy of MTDLMP and UDL models with the G&#xD7;E term (I) and without the G&#xD7;E term (WI) for each trait with 1 layer; (&lt;strong&gt;B&lt;/strong&gt;) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI); and (&lt;strong&gt;C&lt;/strong&gt;) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G&#xD7;E term (I) and without the G&#xD7;E term (WI).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F9.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/9/5/1545/F9.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493059" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-40" class="first-child">Prediction accuracy of <em>data set 4</em> in terms of percentage of cases correctly classified (PCCC) for traits days to heading (DTHD), days to maturity (DTMT) and Height and in terms of average Pearson’s correlation for grain yield (GY) trait. (<strong>A</strong>) Prediction accuracy of MTDLMP and UDL models with the G×E term (I) and without the G×E term (WI) for each trait with 1 layer; (<strong>B</strong>) prediction accuracy with different numbers of layers (1, 2 and 3) across environments with the MTDLMP model with the G×E term (I) and without the G×E term (WI); and (<strong>C</strong>) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL model with the G×E term (I) and without the G×E term (WI).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Data set 5</h3><p id="p-41">This data set had four traits with mixed phenotypes. In binary trait GC, the first category had 79.7% of the cases, while category 2 had 20.3% of cases. For ordinal trait Leaf Rust, the first category had 9.2% of the cases, category 2 had 4.6% of the cases, category 3, 11.7% of the cases, and category 4, 10.0% of the cases. The most numerous category was category 5, with 64.5% of the cases. On the other hand, the first category of trait Stripe Rust had 90.4% of the cases, whereas category 2 and category 3 had 0.90% and 8.7% of individuals, respectively. The average of the continuous trait GY was around 4.4 ton/ha (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of each ordinal response for data set 5 for traits grain color (GC), Leaf Rust and Stripe Rust. Boxplot of grain yield (GY)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of each ordinal response for &lt;em&gt;data set 5&lt;/em&gt; for traits grain color (GC), Leaf Rust and Stripe Rust. Boxplot of grain yield (GY).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F10.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/9/5/1545/F10.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493048" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-42" class="first-child">Percentage of each ordinal response for <em>data set 5</em> for traits grain color (GC), Leaf Rust and Stripe Rust. Boxplot of grain yield (GY).</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">In this data set, in terms of prediction performance using 1 and 2 layers, no significant differences were found in any trait between the MTDLMP and UDL models (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 11a</a>; Figure D5_SA; Supplementary material, hdl:11529/10548140). With 3 layers, there were statistical differences between the two models only for trait GY, where the UDL model outperformed the MTDLMP model by 23.40%, (Figure D5_SB; Supplementary material, hdl:11529/10548140). The PCCC using 1 layer for the binary and ordinal traits (GC, Leaf Rust, Stripe Rust) ranged from 0.6442 to 0.9035, while the APC for the GY trait ranged from 0.4574 to 0.4920 (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10a</a>). When comparing the prediction accuracy using different numbers of layers (1, 2 and 3) under the MTDLMP model (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Figure 11b</a>) and the UDL model (<a id="xref-fig-11-3" class="xref-fig" href="#F11">Figure 11C</a>), there were no statistical differences in terms of prediction performance between using one, two or three layers.</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of data set 5 in terms of percentage of cases correctly classified (PCCC) for traits grain color (GC), Leaf Rust and Stripe Rust and in terms of average Pearson&#x2019;s correlation for trait grain yield (GY). (A) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (C) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy of &lt;em&gt;data set 5&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for traits grain color (GC), Leaf Rust and Stripe Rust and in terms of average Pearson&#x2019;s correlation for trait grain yield (GY). (&lt;strong&gt;A&lt;/strong&gt;) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (&lt;strong&gt;B&lt;/strong&gt;) prediction accuracy with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (&lt;strong&gt;C&lt;/strong&gt;) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F11.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11" src="https://www.g3journal.org/content/ggg/9/5/1545/F11.medium.gif" width="440" height="264"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493017" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11</span> <p id="p-44" class="first-child">Prediction accuracy of <em>data set 5</em> in terms of percentage of cases correctly classified (PCCC) for traits grain color (GC), Leaf Rust and Stripe Rust and in terms of average Pearson’s correlation for trait grain yield (GY). (<strong>A</strong>) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (<strong>B</strong>) prediction accuracy with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (<strong>C</strong>) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Data set 6</h3><p id="p-45">This data set had two traits (one ordinal and one continuous). The ordinal trait (Lodging) had 12.2% of cases in category 1, 7.8% in category 2, 9.4% in category 3, 38.2% in category 3 and 32.4% in category 5. For continuous trait GY, the average was around 6.7 ton/ha (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure 12</a>).</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of the ordinal response (Lodging) for data set 6 and boxplot of grain yield (GY)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of the ordinal response (Lodging) for &lt;em&gt;data set 6&lt;/em&gt; and boxplot of grain yield (GY).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F12.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 12" src="https://www.g3journal.org/content/ggg/9/5/1545/F12.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 12</span> <p id="p-46" class="first-child">Percentage of the ordinal response (Lodging) for <em>data set 6</em> and boxplot of grain yield (GY).</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">In terms of prediction performance, in <strong>data set 6</strong> when using 1 and 2 layers, there were no significant differences in any trait between the MTDLMP and UDL models (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure 13a</a>; Figure D6_SA Supplementary material, hdl:11529/10548140). With 3 layers, there were statistical differences between the two models only in GY, where the MTDLMP model outperformed the UDL model by 29.78% (Figure D6_SB, Supplementary material, hdl:11529/10548140). The PCCC using 1 layer for the ordinal trait (Lodging) ranged from 0.4392 to 0.4603, while the APC for the GY trait ranged from 0.3491 to 0.3896 (<a id="xref-fig-13-2" class="xref-fig" href="#F13">Figure 13a</a>). Also, comparing the prediction accuracy using different numbers of layers (1, 2, and 3) under the MTDLMP (<a id="xref-fig-13-3" class="xref-fig" href="#F13">Figure 13b</a>) and UDL (<a id="xref-fig-13-4" class="xref-fig" href="#F13">Figure 13C</a>) models, there were no statistical differences in terms of prediction performance using 1, 2 or 3 layers.</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of data set 6 in terms of percentage of cases correctly classified (PCCC) for trait Lodging and in terms of average Pearson&#x2019;s correlation for grain yield GY. (A) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (B) prediction accuracy with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (C) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy of &lt;em&gt;data set 6&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for trait Lodging and in terms of average Pearson&#x2019;s correlation for grain yield GY. (&lt;strong&gt;A&lt;/strong&gt;) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (&lt;strong&gt;B&lt;/strong&gt;) prediction accuracy with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (&lt;strong&gt;C&lt;/strong&gt;) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 13" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F13.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 13" src="https://www.g3journal.org/content/ggg/9/5/1545/F13.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492928" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 13</span> <p id="p-48" class="first-child">Prediction accuracy of <em>data set 6</em> in terms of percentage of cases correctly classified (PCCC) for trait Lodging and in terms of average Pearson’s correlation for grain yield GY. (<strong>A</strong>) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (<strong>B</strong>) prediction accuracy with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (<strong>C</strong>) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Data set 7</h3><p id="p-49">This real data set had two traits (one ordinal and one continuous). The ordinal trait (Lodging) had 50.6%, 14.1%, 16.4%, 10.9% and 8.0% of individuals in categories 1, 2, 3, 4 and 5, respectively. The continuous trait GY had an average of around 5.75 ton/ha (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Figure 14</a>).</p><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of each ordinal response for data set 7 for traits grain color (GC), Leaf Rust and Stripe Rust and boxplot of grain yield (GY)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Percentage of each ordinal response for &lt;em&gt;data set 7&lt;/em&gt; for traits grain color (GC), Leaf Rust and Stripe Rust and boxplot of grain yield (GY).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 14" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F14.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 14" src="https://www.g3journal.org/content/ggg/9/5/1545/F14.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 14" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493090" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 14</span> <p id="p-50" class="first-child">Percentage of each ordinal response for <em>data set 7</em> for traits grain color (GC), Leaf Rust and Stripe Rust and boxplot of grain yield (GY).</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">There were statistical differences in terms of genome-enabled prediction accuracy between models MTDLMP and UDL only for trait GY (<a id="xref-fig-15-1" class="xref-fig" href="#F15">Figure 15a</a>; Figures D7_SA, and Figure D7_SB, Supplementary material, hdl:11529/10548140). Model MTDLMP outperformed model UDL by 25.14%, 30.46% and 32.83% when using 1, 2 and 3 layers, respectively. The PCCC using 1 layer for the ordinal trait (Lodging) ranged from 0.5082 to 0.5117, while the APC for the GY trait ranged from 0.2914 to 0.3893 (<a id="xref-fig-15-2" class="xref-fig" href="#F15">Figure 15a</a>). When comparing the prediction accuracy using different numbers of layers (1, 2 and 3) under the MTDLMP (<a id="xref-fig-15-3" class="xref-fig" href="#F15">Figure 15b</a>) and UDL (<a id="xref-fig-15-4" class="xref-fig" href="#F15">Figure 15c</a>) models, no statistical differences were found in genomic-enabled prediction performance accuracy using 1, 2 and 3 layers.</p><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of data set 7 in terms of percentage of cases correctly classified (PCCC) for trait Lodging and in terms of average Pearson&#x2019;s correlation for grain yield (GY). (A) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (B) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (C) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1630592716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction accuracy of &lt;em&gt;data set 7&lt;/em&gt; in terms of percentage of cases correctly classified (PCCC) for trait Lodging and in terms of average Pearson&#x2019;s correlation for grain yield (GY). (&lt;strong&gt;A&lt;/strong&gt;) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (&lt;strong&gt;B&lt;/strong&gt;) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (&lt;strong&gt;C&lt;/strong&gt;) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 15" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1545/F15.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 15" src="https://www.g3journal.org/content/ggg/9/5/1545/F15.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 15" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1545/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492913" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 15</span> <p id="p-52" class="first-child">Prediction accuracy of <em>data set 7</em> in terms of percentage of cases correctly classified (PCCC) for trait Lodging and in terms of average Pearson’s correlation for grain yield (GY). (<strong>A</strong>) Prediction accuracy between the MTDLMP and UDL models with 1 layer; (<strong>B</strong>) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the MTDLMP model; and (<strong>C</strong>) prediction accuracy obtained with different numbers of layers (1, 2 and 3) with the UDL.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-53">In genomic prediction, various approaches have been developed for increasing prediction accuracy mainly of continuous traits (for example, all models under the Bayesian alphabet). However, few approaches have been developed for non-normal phenotypes and multivariate prediction of mixed phenotypes: binary, ordinal and continuous. In the context of mixed phenotypes, the most common approach used is to perform a separate univariate analysis for each trait; this ignores the correlations among multiple traits. For this reason, in this paper we propose using multiple-trait deep learning for the prediction of mixed phenotypes: binary, ordinal and continuous. We compare the prediction performance of the MTDLMP model with those of the UDL model using fivefold cross-validation in seven data sets using the PCCC (for binary and ordinal phenotypes) and Pearson’s correlation (for continuous phenotypes) as metrics for measuring the prediction performance. In four data sets, we discretized some continuous traits to make them ordinal and binary, while in the other three data sets, discretization was not necessary since these three data sets naturally contain binary, ordinal and continuous traits.</p><p id="p-54">Our results using seven data sets showed that using multiple-trait deep learning is a practical approach for simultaneously predicting multiple traits with mixed phenotypes (binary, ordinal and continuous), given that the predictions obtained under the MTDLMP model are not low. The gain in terms of prediction performance of the MTDLMP over the UDL model was intermediate, given that when the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-49.gif"/></noscript></span></span> interaction term was taken into account, the MTDLMP was better than the UDL model in 4 out of the 7 data sets, while when the <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1545/embed/mml-math-50.gif"/></noscript></span></span> interaction term was ignored, the MTDLMP was better in 5 out of the 7 data sets.</p><p id="p-55">It is also important to point out that the observed gain in terms of prediction performance of the MTDLMP over the UDL model was observed only in the continuous trait GY, while in the remaining traits no statistical differences were observed between the two deep learning models. Part of these results can be attributed in part to the fact that the phenotypic correlations between traits are not high since for data set 1 the minimum, average and maximum values were 0.103, 0.237 and 0.794, respectively, for data set 2 the minimum, average, and maximum values were -0.044, 0.166 and 0.782, respectively, for data set 3 the minimum, average, and maximum values were -0.219, 0.058 and 0.719, respectively, for data set 4 the minimum, average, and maximum values were -0.079, 0.179 and 0.803, respectively, for data set 5 the minimum, average, and maximum values were -0.245, -0.041 and 0.051, and for data set 6 and 7 the correlations between the two traits under study were -0.517 and -0.4056, respectively. Also, two of the weakness of our study is that the number of markers used in the 7 data sets is substantially low which may be affecting the prediction accuracy. For this reason, we are aware that more empirical evaluations are needed to have a better picture of the predictive power of the MTDLMP model. We also found that, in general, in these data sets increasing the number of hidden layers did not help to significantly increase the prediction accuracy, since in most situations that we evaluated, the best predictions were obtained with only one hidden layer.</p><p id="p-56">To successfully implement the MTDLMP, the following issues need to be taken into account. The continuous traits need to be standardized (subtracting the mean and dividing by the standard deviation) in each training set when the response variable is not close to mean zero and the variance is equal to 1. Since it is necessary to specify different activation functions for continuous, binary and ordinal data, we used the ReLU activation function for continuous traits and the sigmoid and softmax activation functions for binary and ordinal traits. Different metrics must be used to measure the prediction performance for continuous and ordinal (and binary) traits, <em>i.e.</em>, Pearson’s correlation for continuous traits and percentage of cases correctly classified (PCCC) for ordinal and binary traits. The process for choosing the optimal (or near optimal) hyperparameters must be tuned in order to increase the chances of prediction.</p><p id="p-57">It is not possible to do a formal comparison of the results of this study with those of <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em> (2018a</a>, <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">b</a>). However, it should be noted that the prediction accuracies of the DL method with genotype × environment interaction for single traits such as grain yield, plant height, days to heading and days to maturity obtained by <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Montesinos-López <em>et al.</em> (2018a)</a> are higher than the prediction accuracies found in this study. The prediction accuracies reported by <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Montesinos-López <em>et al.</em> (2018b)</a> for the multiple-trait DL (MTDL) are similar to those reported in this study for ordinal data. Furthermore, the results where the MTDL without including genotype × environments is slightly superior in prediction accuracy to the Bayesian multiple-trait multiple-environment and vice versa (when the genotype × environments is ignored) agree with the results obtained in this study.</p><p id="p-58">It should be noted that the proposed method for the simultaneous prediction of mixed phenotypes (binary, ordinal and continuous) under the deep learning (a type of machine learning method) framework is novel, since nowadays in plant breeding there are no statistical models available that are able to simultaneously predict mixed phenotypes, given that multiple-trait (multivariate) models have only been developed for continuous traits. It is important to point out that multivariate models for mixed response variables (traits) are available in the statistical literature under classic (maximum likelihood) and Bayesian methods. However, these available models are not appropriate for dealing with large data sets nor with the problem of large p and small n in genomic-enabled prediction, since those Bayesian models were not built with an analytical Gibbs sampler, due to the complexity of the likelihood function, which is a mixture of normal and other types of distributions.</p><p id="p-59">Finally, this application of deep learning for the simultaneous prediction of mixed phenotypes (binary, ordinal and continuous) is important due to the fact that there is a lack of multivariate models for simultaneously predicting mixed phenotypes (binary, ordinal and continuous) in plant breeding and because we found that this model can be implemented using the open-source R statistical software with the keras package. Also, this package is easy to implement and does not require that its users have a strong computational and mathematical background. It is efficient in terms of the computing resources required and allows the implementation of other types of DL architectures, such as convolutional networks, recurrent networks, etc., that can help improve prediction accuracy in some circumstances.</p></div><div class="section conclusions" id="sec-16"><h2 class="">Conclusions</h2><p id="p-60">In this paper we propose applying deep learning for simultaneously predicting multiple traits with mixed phenotypes (binary, ordinal and continuous). This application is novel in GS since, to the best of our knowledge, nowadays there are no multiple-trait models available for the simultaneous prediction of mixed phenotypes. When comparing the prediction performance of the MTDLMP model with that of the UDL model, gains in terms of prediction accuracy were only obtained in trait grain yield, and no differences were detected in binary and ordinal traits. In general, the deep learning model for simultaneously predicting mixed phenotypes is an attractive alternative for breeders due to the lack of models for the simultaneous prediction of mixed phenotypes; the existence of friendly open-source software for its implementation is also an important advantage. For this reason, we believe deep learning models should be included in the toolkit of scientific breeders, since there is empirical evidence that there is no universally best prediction model for genomic-enabled prediction.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-61">We thank all scientists, field workers, and lab assistants from National Programs and CIMMYT who collected the data used in this study. We acknowledge the financial support provided by the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) in Norway through NFR grant 267806. We are also thankful for the financial support provided by CIMMYT CRP (maize and wheat), the Bill &amp; Melinda Gates Foundation, as well the USAID projects (Cornell University and Kansas State University) that generated the CIMMYT wheat data analyzed in this study. We are thankful to two anonymous reviewers who invested long hours reviewing this article and making suggestions that significantly improved the quality of the manuscript. Also, the first author of this research thanks the Coimbra Group for the Scholarship provided to spend three months at the Departamento de Estadística of Universidad de Salamanca, Spain, where all the analyses of this research were done.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-62">Supplemental material available at <a href="http://hdl.handle.net/11529/10548140">http://hdl.handle.net/11529/10548140</a>.</p></li><li class="fn" id="fn-2"><p id="p-63"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-09"><span class="received-label">Received </span>January 9, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-08"><span class="accepted-label">Accepted </span>March 8, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Montesinos-López <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.1" data-doi="10.1038/nbt.3300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alipanahi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D831%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3300%26rft_id%253Dinfo%253Apmid%252F26213851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26213851&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.2" data-doi="10.15252/msb.20156651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angermueller</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pärnamaa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep learning for computational biology.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">878</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.15252/msb.20156651</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DAngermueller%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D878%26rft.epage%253D878%26rft.atitle%253DDeep%2Blearning%2Bfor%2Bcomputational%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20156651%26rft_id%253Dinfo%253Apmid%252F27474269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjEyLzcvODc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.3" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: Software for association mapping of comple traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1545.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chollet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allaire</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source"><em>Deep Learning with R. Manning Publications</em>, <em>Manning Early Access Program</em></span>, <span class="cit-edition">Ed. 1st</span>. <span class="cit-publ-name">MEA</span>. Shelter Island, NY.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.5" data-doi="10.1016/j.neuroimage.2017.07.059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudra</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poudel</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthan</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caan</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Predicting brain age with deep learning from raw imaging data results in a reliable and heritable biomarker.</span> <abbr class="cit-jnl-abbrev">Neuroimage</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuroimage.2017.07.059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuroimage%26rft.volume%253D163%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2017.07.059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2017.07.059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.6" data-doi="10.18178/ijmlc.2017.7.5.632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dingli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fournier</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Financial Time Series Forecasting – A Deep Learning Approach</span>. <abbr class="cit-jnl-abbrev">Int. J. Mach. Learn. Comput.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18178/ijmlc.2017.7.5.632</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMach.%2BLearn.%2BComput.%26rft.volume%253D7%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.18178%252Fijmlc.2017.7.5.632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18178/ijmlc.2017.7.5.632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.7" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.5.1545.8"><div class="cit-metadata"><cite>Fox, I., A. Lynn, J. Mamta, P.-B. Rodica, and W. Jenna, 2018. Deep Multi-Output Forecasting: Learning to Accurately Predict Blood Glucose Trajectories. In KDD ’18: The 24th ACM SIGKDD International Conference on Knowledge Discovery &amp; Data Mining, August 19–23, 2018, London, UK. ACM, New York, NY, 9 pages.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/3219819</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.9" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in Whole-Genome Regression: The Bayesian Alphabet Returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS81LzE1NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1545.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source"><em>Deep Learning. A Practitioner’s Approach</em>.</span> <span class="cit-publ-name">O’Reilly Media</span>, <span class="cit-publ-loc">Newton, MA</span>. <span class="cit-fpage">532</span> pages.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.11" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS : A High Capacity Genotyping by Sequencing Analysis Pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A primer on neural network models for natural language processing.</span> <abbr class="cit-jnl-abbrev">J. Artif. Intell. Res.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BArtif.%2BIntell.%2BRes.%26rft.volume%253D57%26rft.spage%253D420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1545.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bengio</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courville</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>. Deep learning. MIT Press; &lt;<a href="http://www.deeplearningbook.org">http://www.deeplearningbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1545.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gulli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sujit</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">Deep Learning with Keras: Implementing deep learning models and neural networks with the power of Python</span>, <span class="cit-publ-name">Packt Publishing Ltd.</span>, <span class="cit-publ-loc">Birmingham, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.15" data-doi="10.1093/bioinformatics/btw249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">i37</span>–<span class="cit-lpage">i43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw249%26rft_id%253Dinfo%253Apmid%252F27307640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27307640&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.16" data-doi="10.3835/plantgenome2016.12.0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemes da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.12.0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.12.0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.12.0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.17" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.18" data-doi="10.1038/hdy.2015.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.9%26rft_id%253Dinfo%253Apmid%252F25873147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25873147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.19" data-doi="10.3835/plantgenome2018.03.0017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juliana</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Prospects and challenges of applied genomic selection-A new paradigm in breeding for grain yield in bread wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2018.03.0017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2018.03.0017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2018.03.0017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.20" data-doi="10.1093/biomet/70.1.235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzanowski</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Distance between populations using mid continuous and categorical variables.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">243</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biomet/70.1.235</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DKRZANOWSKI%26rft.auinit1%253DW.%2BJ.%26rft.volume%253D70%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D243%26rft.atitle%253DDistance%2Bbetween%2Bpopulations%2Busing%2Bmixed%2Bcontinuous%2Band%2Bcategorical%2Bvariables%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F70.1.235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/70.1.235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983QH66900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1545.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lantz</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source"><em>Machine Learning with R</em>.</span> <span class="cit-edition">Second Edition</span>. <span class="cit-publ-name">Packt Publishing</span>, <span class="cit-publ-loc">Briminghan, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.22" data-doi="10.1038/nature14539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LeCun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bengio</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Deep learning.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">521</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">444</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature14539</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D521%26rft.spage%253D436%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14539%26rft_id%253Dinfo%253Apmid%252F26017442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14539&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26017442&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1545.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">N. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source">Deep learning made easy with R. A gentle introduction for data science</span>, <span class="cit-publ-name">CreateSpace Independent Publishing Platform</span>, <span class="cit-publ-loc">Scotts Valley, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1545.24"><div class="cit-metadata unstructured">Liu, S., Tang, J., Zhang, Z., Gaudiot, J.L. 2017. CAAD: Computer Architecture for Autonomous Driving. ariv preprint ariv:1702.01894.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.25" data-doi="10.1007/BF02478259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pitts</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">A logical calculus of the ideas immanent in nervous activity.</span> <abbr class="cit-jnl-abbrev">Bull. Math. Biophys.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02478259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull.%2BMath.%2BBiophys.%26rft.volume%253D5%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02478259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02478259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.26" data-doi="10.1371/journal.pone.0061318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menden</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iorio</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garnett</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDermott</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benes</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Machine learning prediction of cancer cell sensitivity to drugs based on genomic and chemical properties.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e61318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0061318</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De61318%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0061318%26rft_id%253Dinfo%253Apmid%252F23646105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0061318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23646105&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.27" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTU0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.28" data-doi="10.1534/g3.115.021667"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Money</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Migicovsky</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwaninger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite> <span class="cit-pub-date">2015</span>. LinkImpute : Fast and Accurate Genotype Imputation for Nonmodel Organisms. G3 (Bethesda) 5: 2383–2390.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021667</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021667%26rft_id%253Dinfo%253Apmid%252F26377960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTEvMjM4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNTQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toledo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Hernández</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite> <span class="cit-pub-date">2016</span>. A Genomic Bayesian Multi-trait and Multi-environment Model. G3 (Bethesda) 6: 2725–2744.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032359</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.30" data-doi="10.1534/g3.118.200740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Suárez</span>  <span class="cit-name-given-names">C.M.</span></span></li></ol><cite> <span class="cit-pub-date">2018</span><span class="cit-comment">a</span>. Multi-environment genomic prediction of plant traits using deep learners with a dense architecture. G3 (Bethesda).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200740</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200740%26rft_id%253Dinfo%253Apmid%252F30291107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjgvMTIvMzgxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNTQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Suárez</span>  <span class="cit-name-given-names">C.M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2018</span><span class="cit-comment">b</span>. Multi-trait, multi-environment deep learning modeling for genomic-enabled prediction of plant traits. G3(Bethesda).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200728</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.32" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1545.33"><div class="cit-metadata unstructured">R Core Team, 2018. R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna. Austria. ISBN 3–900051–07–0. URL <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.34" data-doi="10.1002/2017GL074677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouet-Leduc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulbert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubbers</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barros</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphreys</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Machine Learning Predicts Laboratory Earthquakes.</span> <abbr class="cit-jnl-abbrev">Geophys. Res. Lett.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">9276</span>–<span class="cit-lpage">9282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/2017GL074677</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeophys.%2BRes.%2BLett.%26rft.volume%253D44%26rft.spage%253D9276%26rft_id%253Dinfo%253Adoi%252F10.1002%252F2017GL074677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/2017GL074677&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1545.35"><div class="cit-metadata"><cite>Shalev-Shwartz, S., and S. Ben-David, 2014 <em>Understanding Machine Learning. From Theory to Algorithms</em>. Cambridge University Press, New York, NY 10013–2473.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/CBO9781107298019</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.36" data-doi="10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Advantages and limitations of multiple-trait genomic prediction for Fusarium head blight severity in hybrid wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122–017–3029–7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B017%2526ndash%253B3029%2526ndash%253B7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;017&amp;ndash;3029&amp;ndash;7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.37" data-doi="10.1214/12-AOS1000. 1102.4807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krizhevsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutskever</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salakhutdinov</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dropout: A Simple Way to Prevent Neural Networks from Overfitting.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1929</span>–<span class="cit-lpage">1958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D15%26rft.spage%253D1929%26rft_id%253Dinfo%253Adoi%252F10.1214%252F12-AOS1000.%2B1102.4807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/12-AOS1000. 1102.4807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1545.38" data-doi="10.1101/177378"><div class="cit-metadata"><cite>Tavanaei, A., Anandanadarajah, N., Maida, A.S., Loganantharaj, R. 2017. A Deep Learning Model for Predicting Tumor Suppressor Genes and Oncogenes from PDB Structure. bioRiv 177378;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/177378</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1101%252F177378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIxNzczNzh2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNS8xNTQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1545.39" data-doi="10.1016/j.ijforecast.2014.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weron</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Electricity price forecasting: A review of the state-of-the-art with a look into the future.</span> <abbr class="cit-jnl-abbrev">Int. J. Forecast.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1030</span>–<span class="cit-lpage">1081</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ijforecast.2014.08.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BForecast.%26rft.volume%253D30%26rft.spage%253D1030%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijforecast.2014.08.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijforecast.2014.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-other" id="cit-9.5.1545.40"><div class="cit-metadata unstructured">Wiley, J.F. 2016. <em>R Deep Learning Essentials</em>. Packt. ISBN (Print) 9781785280580.</div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1545.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1533" title="Variation in Filamentous Growth and Response to Quorum-Sensing Compounds in Environmental Isolates of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1557" title="Use of F2 Bulks in Training Sets for Genomic Prediction of Combining Ability and Hybrid Performance" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1545" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xr6awoW0fYX8UDz2SQapxcMt0Jren181ZSAiuTOwiU8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1545" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1545" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1545" class="active" data-icon-position="" data-hide-link-title="0">New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492857" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NL5oSxFacqHxyQem31b58rQj91ykHXFsD86lSVXKO-Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1545.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1545" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-moWPSPDM4zuJNKZNy-xhn1cdZ74SlGE-vt8v_tI-7eQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1545" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1545.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492857" id="node492857--2" data-pisa="ggg;9/5/1545" data-pisa-master="ggg;g3.119.300585" data-apath="/ggg/9/5/1545.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Javier Martín-Vallejo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span>, <span class="highwire-citation-author" data-delta="5">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="6">Philomin Juliana</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1545-1556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.300585 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492857--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1545" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1545&amp;title=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1545&amp;title=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1545&amp;title=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1545&amp;text=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1545&amp;title=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1545&amp;t=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1545&amp;title=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1545&amp;title=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492857/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492857" id="node492857--3" data-pisa="ggg;9/5/1545" data-pisa-master="ggg;g3.119.300585" data-apath="/ggg/9/5/1545.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">New Deep Learning Genomic-Based Prediction Model for Multiple Traits with Binary, Ordinal, and Continuous Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Javier Martín-Vallejo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="4">Carlos M. Hernández-Suárez</span>, <span class="highwire-citation-author" data-delta="5">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="6">Philomin Juliana</span> and <span class="highwire-citation-author" data-delta="7">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1545-1556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.300585 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492857--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Javier Martín-Vallejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Universidad de Salamanca, c/Espejo 2, Salamanca, 37007, España</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Javier%2BMart%C3%ADn-Vallejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADn-Vallejo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJavier%2BMart%25C3%25ADn-Vallejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos M. Hernández-Suárez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Facultad de Ciencias, Universidad de Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BM.%2BHern%C3%A1ndez-Su%C3%A1rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hern%C3%A1ndez-Su%C3%A1rez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BM.%2BHern%25C3%25A1ndez-Su%25C3%25A1rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach2004@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach2004@hotmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Ciudad de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1545&amp;rft.epage=1556&amp;rft.atitle=New%20Deep%20Learning%20Genomic-Based%20Prediction%20Model%20for%20Multiple%20Traits%20with%20Binary%2C%20Ordinal%2C%20and%20Continuous%20Phenotypes&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Mart%C3%ADn-Vallejo%2C+Javier&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Gianola%2C+Daniel&amp;rft.au=Hern%C3%A1ndez-Su%C3%A1rez%2C+Carlos+M.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Juliana%2C+Philomin&amp;rft.au=Singh%2C+Ravi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492857/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492857/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492857/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492857/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492857/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492857/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492857/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492857/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492857/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492857/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492857/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492857/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1545" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1545.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1545.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1545.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
