<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/5/1519" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/5/1519.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/5/1519.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/5/1519.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/5/1519/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/5/1519.atom" />
<meta name="HW.pisa" content="ggg;9/5/1519" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (Oryza sativa L.) Grown in Subtropical Areas" />
<meta name="DC.Identifier" content="10.1534/g3.119.400064" />
<meta name="DC.Date" content="2019-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Monteverde et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Understanding the genetic and environmental basis of genotype × environment interaction (G×E) is of fundamental importance in plant breeding. If we consider G×E in the context of genotype × year interactions (G×Y), predicting which lines will have stable and superior performance across years is an important challenge for breeders. A better understanding of the factors that contribute to the overall grain yield and quality of rice ( Oryza sativa L.) will lay the foundation for developing new breeding and selection strategies for combining high quality, with high yield. In this study, we used molecular marker data and environmental covariates (EC) simultaneously to predict rice yield, milling quality traits and plant height in untested environments (years), using both reaction norm models and partial least squares (PLS), in two rice breeding populations ( indica and tropical japonica ). We also sought to explain G×E by differential quantitative trait loci (QTL) expression in relation to EC. Our results showed that PLS models trained with both molecular markers and EC gave better prediction accuracies than reaction norm models when predicting future years. We also detected milling quality QTL that showed a differential expression conditional on humidity and solar radiation, providing insight for the main environmental factors affecting milling quality in subtropical and temperate rice growing areas." />
<meta name="DC.Contributor" content="Eliana Monteverde" />
<meta name="DC.Contributor" content="Lucía Gutierrez" />
<meta name="DC.Contributor" content="Pedro Blanco" />
<meta name="DC.Contributor" content="Fernando Pérez de Vida" />
<meta name="DC.Contributor" content="Juan E. Rosas" />
<meta name="DC.Contributor" content="Victoria Bonnecarrère" />
<meta name="DC.Contributor" content="Gastón Quero" />
<meta name="DC.Contributor" content="Susan McCouch" />
<meta name="article:published_time" content="2019-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (Oryza sativa L.) Grown in Subtropical Areas" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Understanding the genetic and environmental basis of genotype × environment interaction (G×E) is of fundamental importance in plant breeding. If we consider G×E in the context of genotype × year interactions (G×Y), predicting which lines will have stable and superior performance across years is an important challenge for breeders. A better understanding of the factors that contribute to the overall grain yield and quality of rice (&lt;i&gt;Oryza sativa&lt;/i&gt; L.) will lay the foundation for developing new breeding and selection strategies for combining high quality, with high yield. In this study, we used molecular marker data and environmental covariates (EC) simultaneously to predict rice yield, milling quality traits and plant height in untested environments (years), using both reaction norm models and partial least squares (PLS), in two rice breeding populations (&lt;i&gt;indica&lt;/i&gt; and &lt;i&gt;tropical japonica&lt;/i&gt;). We also sought to explain G×E by differential quantitative trait loci (QTL) expression in relation to EC. Our results showed that PLS models trained with both molecular markers and EC gave better prediction accuracies than reaction norm models when predicting future years. We also detected milling quality QTL that showed a differential expression conditional on humidity and solar radiation, providing insight for the main environmental factors affecting milling quality in subtropical and temperate rice growing areas.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/05/01" />
<meta name="citation_mjid" content="ggg;9/5/1519" />
<meta name="citation_id" content="9/5/1519" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/5/1519" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/5/1519.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/5/1519.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/5/1519.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400064" />
<meta name="citation_pmid" content="30877079" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1519" />
<meta name="citation_lastpage" content="1531" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Eliana Monteverde" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca NY 14853" />
<meta name="citation_author" content="Lucía Gutierrez" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin - Madison WI 53706" />
<meta name="citation_author" content="Pedro Blanco" />
<meta name="citation_author_institution" content="Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay" />
<meta name="citation_author" content="Fernando Pérez de Vida" />
<meta name="citation_author_institution" content="Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay" />
<meta name="citation_author" content="Juan E. Rosas" />
<meta name="citation_author_institution" content="Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay" />
<meta name="citation_author_institution" content="Unidad de Biotecnología, Instituto Nacional de Investigación Agropecuaria (INIA), Estación Experimental Wilson Ferreira Aldunate 90200, Uruguay" />
<meta name="citation_author" content="Victoria Bonnecarrère" />
<meta name="citation_author_institution" content="Unidad de Biotecnología, Instituto Nacional de Investigación Agropecuaria (INIA), Estación Experimental Wilson Ferreira Aldunate 90200, Uruguay" />
<meta name="citation_author" content="Gastón Quero" />
<meta name="citation_author_institution" content="Department of Plant Biology, College of Agriculture, Universidad de la República, Montevideo, Uruguay" />
<meta name="citation_author" content="Susan McCouch" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca NY 14853" />
<meta name="citation_author_email" content="srm4@cornell.edu" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. H. Aastveit;citation_author=H. Martens;citation_title=ANOVA interactions interpreted by partial least squares regression.;citation_pages=829-844;citation_volume=42;citation_year=1986;citation_doi=10.2307/2530697" />
<meta name="citation_reference" content="Alvarado, J. R., 2002 Influence of air temperature on rice population, length of period from sowing to flowering and spikelet sterility. In Proceedings of the second temperate rice conference, IRRI, Philippines." />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=A. A. Ambardekar;citation_author=T. J. Siebenmorgen;citation_author=P. A. Counce;citation_author=S. B. Lanning;citation_author=A. Mauromoustakos;citation_title=Impact of field-scale nighttime air temperatures during kernel development on rice milling quality.;citation_pages=179-185;citation_volume=122;citation_year=2011;citation_doi=10.1016/j.fcr.2011.03.012" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995;citation_pmid=16632515;citation_doi=10.1093/biostatistics/kxj037" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Boer;citation_author=D. Wright;citation_author=L. Feng;citation_author=D. W. Podlich;citation_author=L. Luo;citation_title=A mixed-model quantitative trait loci (QTL) analysis for multiple-environment trial data using environmental covariables for QTL-by-environment interactions, with an example in maize.;citation_volume=177;citation_year=2007;citation_pmid=17947443;citation_doi=10.1534/genetics.107.071068" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=A. L. Boulesteix;citation_author=K. Strimmer;citation_title=Partial least squares: a versatile tool for the analysis of high-dimensional genomic data.;citation_volume=8;citation_year=2006;citation_pmid=16772269;citation_doi=10.1093/bib/bbl016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=K. W. Broman;citation_author=T. P. Speed;citation_title=A model selection approach for the identification of quantitative trait loci in experimental crosses.;citation_pages=641-656;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00354" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=P. L. Cornelius;citation_author=R. Trethowan;citation_author=G. McLaren;citation_title=Modeling Additive x Environment and Additive x Additive x Environment Using Genetic Covariances of Relatives of Wheat Genotypes;citation_pages=311-320;citation_volume=47;citation_year=2007;citation_issue=1;citation_doi=10.2135/cropsci2006.09.0564" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic Prediction of Breeding Values when Modeling Genotype × Environment Interaction using Pedigree and Dense Molecular Markers.;citation_pages=707;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=B. T. Campbell;citation_author=P. S. Baezinger;citation_author=K. M. Eskridge;citation_author=H. Budak;citation_author=N. A. Streck;citation_title=Using Environmental Covariates to Explain Genotype x Environment and QTL x Environment Interactions for Agronomic Traits on Chromosome 3A of Wheat;citation_pages=620-627;citation_volume=44;citation_year=2004;citation_issue=2;citation_doi=10.2135/cropsci2004.6200" />
<meta name="citation_reference" content="citation_journal_title=Protoplasma;citation_author=J. Chen;citation_author=L. Tang;citation_author=P. Shi;citation_author=B. Yang;citation_author=T. Sun;citation_title=Effects of short term high temperature on grain quality and starch granules of rice (Oryza sativa L.) at post-anthesis stage.;citation_pages=935-943;citation_volume=254;citation_year=2016;citation_doi=10.1007/s00709-016-1002-y" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=C. Colombani;citation_author=P. Croiseau;citation_author=S. Fritz;citation_author=F. Guillaume;citation_author=A. Legarra;citation_title=A comparison of partial least squares (PLS) and sparse PLS regressions in genomic selection in French dairy cattle.;citation_pages=2120-2131;citation_volume=95;citation_year=2012;citation_issue=4;citation_pmid=22459857;citation_doi=10.3168/jds.2011-4647" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Cooper;citation_author=I. H. DeLacy;citation_title=Relationships among analytical methods used to study genotypic variation and genotype-by-environment interaction in plant breeding multi-environment experiments.;citation_pages=561-572;citation_volume=88;citation_year=1994;citation_doi=10.1007/BF01240919" />
<meta name="citation_reference" content="citation_journal_title=Cereal Chem.;citation_author=N. T. W. Cooper;citation_author=T. J. Siebenmorgen;citation_author=P. A. Counce;citation_title=Effects of night- time temperature during kernel development on rice physico-chemical properties.;citation_pages=276-282;citation_volume=85;citation_year=2008;citation_doi=10.1094/CCHEM-85-3-0276" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Covarrubias-Pazaran;citation_title=Genome-assisted prediction of quantitative traits using the R package sommer.;citation_pages=e0156744;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0156744" />
<meta name="citation_reference" content="citation_journal_title=TAG Theor Appl Genet;citation_author=J. Crossa;citation_author=M. Vargas;citation_author=F. A. van Eeuwijk;citation_author=C. Jiang;citation_author=G. O. Edmeades;citation_title=Interpreting genotype × environment interaction in tropical maize using linked molecular markers and environmental covariates.;citation_pages=611-625;citation_volume=99;citation_year=1999;citation_doi=10.1007/s001220051276" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=J. D. Edwards;citation_author=A. K. Jackson;citation_author=A. M. McClung;citation_title=Genetic architecture of grain chalk in rice and interactions with a low phytic acid locus.;citation_pages=116-123;citation_volume=205;citation_year=2017;citation_doi=10.1016/j.fcr.2017.01.015" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=K. Finlay;citation_author=G. Wilkinson;citation_title=The analysis of adaptation in a plant-breeding programme.;citation_pages=742-754;citation_volume=14;citation_year=1963;citation_doi=10.1071/AR9630742" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=M. A. Fitzgerald;citation_author=S. R. McCouch;citation_author=R. D. Hall;citation_title=Not just a grain of rice: the quest for quality.;citation_pages=133-139;citation_volume=14;citation_year=2009;citation_issue=3;citation_pmid=19230745;citation_doi=10.1016/j.tplants.2008.12.004" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=H. G. Gauch;citation_title=Model selection and validation for yield trials with interaction.;citation_pages=705-715;citation_volume=44;citation_year=1988;citation_doi=10.2307/2531585" />
<meta name="citation_reference" content="citation_title=Statistical analysis of regional yield trials: AMMI analysis of factorial designs;citation_author=H. G. Gauch;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. Gutiérrez;citation_author=S. Germán;citation_author=S. Pereyra;citation_author=P. M. Hayes;citation_author=C. A. Pérez;citation_title=Multi-environment multi-QTL association mapping identifies disease resistance QTL in barley germplasm from Latin America.;citation_pages=501-516;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-014-2448-y" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Iwata;citation_author=K. Ebana;citation_author=Y. Uga;citation_author=T. Hayashi;citation_title=Genomic prediction of biological shape: elliptic fourier analysis and kernel partial least squares (PLS) regression applied to grain shape prediction in rice (Oryza sativa L.);citation_pages=e0120610;citation_volume=10;citation_year=2015;citation_pmid=25825876;citation_doi=10.1371/journal.pone.0120610" />
<meta name="citation_reference" content="citation_journal_title=TAG Theor Appl Genet;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high - dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=D. Jarquín;citation_author=C. L. da Silva;citation_author=R. C. Gaynor;citation_author=J. Poland;citation_author=A. Fritz;citation_title=Increasing genomic-enabled prediction accuracy by modeling genotype × environment interactions in Kansas wheat.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.12.0130" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="Lê Cao, K. A., F. Rohart, I. Gonzalez, S. Dejean, B. Gautier et al., 2016 mixomics: Omics data integration project. R package version 6.1.1. https://CRAN.R-project.org/package=mixOmics" />
<meta name="citation_reference" content="citation_journal_title=Genet;citation_author=M. Lopez-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker x Environment Interaction Genomic Selection Model. G3 Genes, Genomes;citation_pages=569-582;citation_volume=5;citation_year=2015;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=N. Long;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_title=Dimension reduction and variable selection for genomic selection: application to predicting milk yield in Holsteins.;citation_pages=247-257;citation_volume=128;citation_year=2011;citation_issue=4;citation_pmid=21749471;citation_doi=10.1111/j.1439-0388.2011.00917.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. B. Lyman;citation_author=K. S. V. Jagadish;citation_author=L. L. Nalley;citation_author=B. L. Dixon;citation_author=T. Siebenmorgen;citation_title=Neglecting rice milling yield and quality underestimates economic losses from high-temperature stress.;citation_pages=e72157;citation_volume=8;citation_year=2013;citation_pmid=23991056;citation_doi=10.1371/journal.pone.0072157" />
<meta name="citation_reference" content="citation_journal_title=Front. Physiol.;citation_author=M. Malosetti;citation_author=J. M. Ribaut;citation_author=F. A. van Eeuwijk;citation_title=The statistical analysis of multi-environment data: modeling genotype-by-environment interaction and its genetic basis.;citation_pages=44;citation_volume=4;citation_year=2013;citation_pmid=23487515;citation_doi=10.3389/fphys.2013.00044" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. Malosetti;citation_author=D. V. Bustos-Korts;citation_author=M. P. Boer;citation_author=F. A. van Eeuwijk;citation_title=Predicting responses in multiple environments: Issues in relation to genotype × environment interactions.;citation_pages=2210-2222;citation_volume=56;citation_year=2016;citation_doi=10.2135/cropsci2015.05.0311" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Malosetti;citation_author=J. Voltas;citation_author=I. Romagosa;citation_author=S. E. Ullrich;citation_author=F. A. van Eeuwijk;citation_title=Mixed models including environmental covariables for studying QTL by environment interaction.;citation_pages=139-145;citation_volume=137;citation_year=2004;citation_doi=10.1023/B:EUPH.0000040511.46388.ef" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. A. Malvar;citation_author=P. Revilla;citation_author=A. Butrón;citation_author=B. Gouesnard;citation_author=A. Boyat;citation_title=Performance of Crosses among French and Spanish Maize Populations across Environments;citation_pages=1052-1057;citation_volume=45;citation_year=2005;citation_issue=3;citation_doi=10.2135/cropsci2004.0301" />
<meta name="citation_reference" content="citation_journal_title=Cienc. Rural;citation_author=E. Marchezan;citation_author=M. I. D. S. Aude;citation_title=Adverse effect of the wind in irrigated rice.;citation_pages=379-381;citation_volume=23;citation_year=1993;citation_doi=10.1590/S0103-84781993000300025" />
<meta name="citation_reference" content="citation_journal_title=Chemom. Intell. Lab. Syst.;citation_author=T. Mehmood;citation_author=K. H. Liland;citation_author=L. Snipen;citation_author=S. Sæbø;citation_title=A review of variable selection methods in partial least squares regression.;citation_pages=62-69;citation_volume=118;citation_year=2012;citation_doi=10.1016/j.chemolab.2012.07.010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=J. Chemometr.;citation_author=B. H. Mevik;citation_author=H. R. Cederkvist;citation_title=Mean squared error of prediction (MSEP) estimates for principal component regression (PCR) and partial least squares regression (PLSR).;citation_pages=422-429;citation_volume=18;citation_year=2004;citation_doi=10.1002/cem.887" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. Monteverde;citation_author=J. E. Rosas;citation_author=P. Blanco;citation_author=F. Pérez de Vida;citation_author=V. Bonnecarrère;citation_title=Multienvironment models increase prediction accuracy of complex traits in advanced breeding lines of rice.;citation_pages=1519-1530;citation_volume=58;citation_year=2018;citation_doi=10.2135/cropsci2017.09.0564" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=G. Quero;citation_author=L. Gutiérrez;citation_author=E. Monteverde;citation_author=P. Blanco;citation_author=F. Pérez de Vida;citation_title=Genome-wide association study using historical breeding populations discovers genomic regions involved in high-quality rice.;citation_pages=170076;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.08.0076" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Agric. Res.;citation_author=N. S. Raju;citation_author=P. Senguttuvel;citation_author=S. R. Voleti;citation_author=A. H. Prasad;citation_author=V. P. Bhadana;citation_title=Stability analysis of flowering and yield traits to high temperature stress adopting different planting dates in rice (O. sativa L.).;citation_pages=137-148;citation_volume=8;citation_year=2013;citation_doi=10.3923/ijar.2013.137.148" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. E. Rosas;citation_author=S. Martínez;citation_author=P. Blanco;citation_author=F. Pérez de Vida;citation_author=V. Bonnecarrere;citation_title=Resistance to multiple temperate and tropical stem and sheath diseases of rice.;citation_pages=170029;citation_volume=11;citation_year=2017;citation_doi=10.3835/plantgenome2017.03.0029" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=U. R. Rosyara;citation_author=W. S. De Jong;citation_author=D. S. Douches;citation_author=J. B. Endelman;citation_title=Software for genome-wide association studies in autopolyploids and its application to potato.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2015.08.0073" />
<meta name="citation_reference" content="citation_title=Factors affecting rice milling quality;citation_author=T. J. Siebenmorgen;citation_author=P. A. Counce;citation_author=C. E. Wilson;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=A. B. Smith;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_title=The analysis of crop cultivar breeding and evaluation trials: An overview of current mixed model approaches.;citation_pages=449-462;citation_volume=143;citation_year=2005;citation_doi=10.1017/S0021859605005587" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. R. Solberg;citation_author=A. K. Sonesson;citation_author=J. A. Woolliams;citation_author=T. H. Meuwissen;citation_title=Reducing dimensionality for prediction of genome-wide breeding values.;citation_pages=29-29;citation_volume=41;citation_year=2009;citation_pmid=19296851;citation_doi=10.1186/1297-9686-41-29" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=N. Sreenivasulu;citation_author=V. M. Butardo;citation_author=G. Misra;citation_author=R. P. Cuevas;citation_author=R. Anacleto;citation_title=Designing climate-resilient rice with ideal grain quality suited for high-temperature stress.;citation_volume=66;citation_year=2015;citation_pmid=25662847;citation_doi=10.1093/jxb/eru544" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=K. Swarts;citation_author=H. Li;citation_author=J. A. Romero Navarro;citation_author=D. An;citation_author=M. C. Romay;citation_title=Novel methods to optimize genotypic imputation for low-coverage, next generation sequence data in crop plants.;citation_pages=1;citation_volume=7;citation_year=2014;citation_doi=10.3835/plantgenome2014.05.0023" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=T. W. I. F. Tashiro;citation_author=I. F. Wardlaw;citation_title=The effect of high temperature on kernel dimensions and the type and occurrence of kernel damage in rice;citation_pages=485-496;citation_volume=42;citation_year=1991;citation_doi=10.1071/AR9910485" />
<meta name="citation_reference" content="citation_title=Gentoype by environment interaction;citation_author=F. A. van Eeuwijk;citation_author=J.-B. Denis;citation_author=M. S. Kang;citation_title=Incorporating additional information on genotypes and environments in models for two-way genotype by environment tables;citation_year=1996;citation_doi=10.1201/9781420049374.ch2" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Vargas;citation_author=J. Crossa;citation_author=K. Sayre;citation_author=M. Reynolds;citation_author=M. E. Ramírez;citation_title=Interpreting Genotype x Environment Interaction in Wheat by Partial Least Squares Regression;citation_pages=679-689;citation_volume=38;citation_year=1998;citation_issue=3;citation_doi=10.2135/cropsci1998.0011183X003800030010x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Vargas;citation_author=J. Crossa;citation_author=F. A. van Eeuwijk;citation_author=M. Ramírez;citation_author=K. Sayre;citation_title=Using Partial Least Squares Regression, Factorial Regression, and AMMI Models for Interpreting Genotype x Environment Interaction;citation_pages=955-967;citation_volume=39;citation_year=1999;citation_issue=4;citation_doi=10.2135/cropsci1999.0011183X003900040002x" />
<meta name="citation_reference" content="citation_journal_title=Jpn. J. Crop. Sci.;citation_author=K. Wakamatsu;citation_author=O. Sasaki;citation_author=A. Tanaka;citation_title=Effects of the amount of insolation and humidity during the ripening period on the grain quality of brown rice in warm regions of Japan.;citation_pages=476-482;citation_volume=78;citation_year=2009;citation_doi=10.1626/jcs.78.476" />
<meta name="citation_reference" content="citation_title=Multivariate Analysis;citation_title=Estimation of principal components and related models by iterative least sqares;citation_author=H. Wold;citation_pages=114-142;citation_year=1966" />
<meta name="citation_reference" content="citation_journal_title=Chemom. Intell. Lab. Syst.;citation_author=S. Wold;citation_author=M. Sjostrom;citation_author=L. Eriksson;citation_title=PLS-regression: A basic tool for chemometrics.;citation_pages=109-130;citation_volume=58;citation_year=2001;citation_doi=10.1016/S0169-7439(01)00155-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=H. Yamakawa;citation_author=T. Hirose;citation_author=M. Kuroda;citation_author=T. Yamaguchi;citation_title=Comprehensive expression profiling of rice grain filling-related genes under high temperature using DNA microarray.;citation_volume=144;citation_year=2007;citation_pmid=17384160;citation_doi=10.1104/pp.107.098665" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=F. Yates;citation_author=W. G. Cochran;citation_title=The analysis of groups of experiments.;citation_pages=556-580;citation_volume=28;citation_year=1938;citation_doi=10.1017/S0021859600050978" />
<meta name="citation_reference" content="citation_title=Fundamentals of rice crop science;citation_author=S. Yoshida;citation_year=1981" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. Zhao;citation_author=M. Fitzgerald;citation_title=Climate change: implications for the yield of edible rice.;citation_pages=e66218;citation_volume=8;citation_year=2013;citation_pmid=23776635;citation_doi=10.1371/journal.pone.0066218" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=X. Zhao;citation_author=D. D. Daygon;citation_author=K. L. McNally;citation_author=R. Sackville;citation_author=F. Xie;citation_title=Identification of stable QTLs causing chalk in rice grains in nine environments.;citation_pages=141-153;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-015-2616-8" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (Oryza sativa L.) Grown in Subtropical Areas" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
<meta name="twitter:description" content="Understanding the genetic and environmental basis of genotype × environment interaction (G×E) is of fundamental importance in plant breeding. If we consider G×E in the context of genotype × year interactions (G×Y), predicting which lines will have stable and superior performance across years is an important challenge for breeders. A better understanding of the factors that contribute to the overall grain yield and quality of rice ( Oryza sativa L.) will lay the foundation for developing new breeding and selection strategies for combining high quality, with high yield. In this study, we used molecular marker data and environmental covariates (EC) simultaneously to predict rice yield, milling quality traits and plant height in untested environments (years), using both reaction norm models and partial least squares (PLS), in two rice breeding populations ( indica and tropical japonica ). We also sought to explain G×E by differential quantitative trait loci (QTL) expression in relation to EC. Our results showed that PLS models trained with both molecular markers and EC gave better prediction accuracies than reaction norm models when predicting future years. We also detected milling quality QTL that showed a differential expression conditional on humidity and solar radiation, providing insight for the main environmental factors affecting milling quality in subtropical and temperate rice growing areas." />
<meta name="og-title" property="og:title" content="Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (Oryza sativa L.) Grown in Subtropical Areas" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/5/1519" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Understanding the genetic and environmental basis of genotype × environment interaction (G×E) is of fundamental importance in plant breeding. If we consider G×E in the context of genotype × year interactions (G×Y), predicting which lines will have stable and superior performance across years is an important challenge for breeders. A better understanding of the factors that contribute to the overall grain yield and quality of rice ( Oryza sativa L.) will lay the foundation for developing new breeding and selection strategies for combining high quality, with high yield. In this study, we used molecular marker data and environmental covariates (EC) simultaneously to predict rice yield, milling quality traits and plant height in untested environments (years), using both reaction norm models and partial least squares (PLS), in two rice breeding populations ( indica and tropical japonica ). We also sought to explain G×E by differential quantitative trait loci (QTL) expression in relation to EC. Our results showed that PLS models trained with both molecular markers and EC gave better prediction accuracies than reaction norm models when predicting future years. We also detected milling quality QTL that showed a differential expression conditional on humidity and solar radiation, providing insight for the main environmental factors affecting milling quality in subtropical and temperate rice growing areas." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5.cover-source.jpg" />
    <title>Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (Oryza sativa L.) Grown in Subtropical Areas | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"H4vpNT_puwDQZ_l_E9mrS7Gd9mY7mGHe8anFEdv0qQw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493104","apath":"\/ggg\/9\/5\/1519.atom","pisa":"ggg;9\/5\/1519","ac":{"\/ggg\/9\/5\/1519.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/5\/1519.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1519"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/5\/1519"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/5\/1519":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pvkPiqOs1xN5stgxcBVn0KmIaNWnXVBtowR73CLZbkpV6Tvu5Xj3lrzpKKHZwN9c1Xp1YLUfVssCNA1lbjOurbANtBkEU8XR5FSEqc8XvaRlXhwpIq+T0Z06BEBQWuBwz9InOdUb02SJgqh1XfUlfv47V1g9Cbyc1kuEPYfzyPauynsPLa2srJGjYzQLmzepjdjmwdmyqytex0b7ehz9GP7x15ZpkaUhhMfwzVt+PxJdRru+6\\\\\\\/KTkfdQ9C5COMp4iX5L+sDsQzJ529WD6zKMwpjtd9r2+PpJB39gSgVz3\\\\\\\/BczZhwPc0wE7i\\\\\\\/Z79WNcK5G0sjq24nP4+QVxQBSjWa0Xonkx15HHxZCqmuhd2QLu9aTqDIz0swDN\\\\\\\/K2e1Fp7vq2sa0PMCb0s8qHKKQDRgVpmBJGUtjgUlz9kMjs3i\\\\\\\/TiF\\\\\\\/6cvXHKyvcTGG38wuts\\\\\\\/0\\\\\\\/KaU8gwXOzJY4yu60gF4PIOoLL7kInd9os+zk\\\\\\\/y1i5iy3XeOQpsn+5y3PHXaNwfXakisOVJQq\\\\\\\/Alzf8gzapGDz549H1ArkET37nWyYIaPGNyp7Tho8HnX++U9biv9VwGDKAFK+Dn7eMEwILZ\\\\\\\/qKLk1xq7DGWacAU2Pn+4ye\\\\\\\/BP3hgrzJFHenhRPMqgWI1UEcxPaIu5HqaS621C4FOLDj1ULvQIyZYElHcT3BxDYtW3sLxUmLFH4ic3NZNDrwkYF7w1cT2b1sP+Apc\\\\\\\/qNX3xO5c6bD03BRebdY6nFWXKnSqLYvyg3Qop0MtnD0l1BjUl4jHsb4KmrvHwpvWxVkeSL0Nmxwja4NbVLDjU5IrI\\\\\\\/pYfIwFhfhNig9B9QPgvD5uVjDevVNBlWfZdhQzlYsvIgfTpvpdFhYBJxgidHArqugFP0XTEChbluQ7TVaKzi2MRZeFtkHwBc51ZhkzUjmOIAtiaNsPww1rVVL\\\\\\\/6G2v6cNJgPANR25AiZhv70vuf3YB5g+9BRnfg3Yvqf0YL9zjLeeBiepyRHQoagy6p\\\\\\\/SsEPFUwdDz4Iii+ekYLFazCxmlUAxpItk4R9pXDB7tfGuwaR4tPo+tLf7YBIrI5XM7rDh\\\\\\\/hbafU8Z6TwSFFvkVWSxM9w1UumKygXFVpqbbsJGMSYCRdna4FX3UJ0egoQAnqtT2dRxBrnAL1A6Bu2HRrQFZPUfwiHuHdLAUOiyr3DLqLpt5dDIwgwgh4II7Bo0RZ32XaMwL2CR0dwZ7\\\\\\\/W+O6JYT7xYIsWxwyMAACCoqxwkMojTQJVQKXpRsHSZaFkCkS8sWyO5xdximipBYnIOYzQfRos\\\u0022,\\\u0022iv\\\u0022:\\\u0022ec88wjwC\\\\\\\/RNxvNgx1SgpAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ef3b62cbd6207b0e970292c1ce77ddaff504f9e371ab127afa9880e47df11d22\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+UIBPkwXfWyjjj0WPq8UCqNuxPit\\\\\\\/1l4lo9nx5D\\\\\\\/af55TzuwsyDwBbEqSmMVDdYr\\\\\\\/13hHlauUy2S7WDJqYOBtvLkolfQSfp\\\\\\\/xHHk9bInaz5e9mU0OU\\\\\\\/2Ul0324ET7gl2NJKs3u8H42nTOVO7\\\\\\\/ezhmxwptkbxdUyU3bMpKXIcjKZBvO+5jxZD4Tkb1HsB0\\\\\\\/gZtxzY2Fz2rBwibCnnp066W75cIflbAe7UxT3WGpUESwgMj2DpzNvvzf52imw1pF0uxFrLmKO+BHWWX6yLK83TIDt67ZFsKuLg9e7GLUBJsOHPGaXpAjOnLQzYEySHr4hyJlgKvingle6ttwsBeDdPeIPINFyiGstKnSladEt7aZid7kzN44ayv3ObzPtDrYDDRSYYSw\\\\\\\/6l1xhovm3fvu+Q2wjBPa2qdwawPT+rrA7kzZ+HGuzafZa2wuwnp2dvO5BWK\\\\\\\/REsO9dI3kPVnx0eE+CMb8DKFPAU71OltXveuY6Ieq98RNwTps1N8q+Yx8Vf+GeHpuZOAx51NIL3I1VwK4j2X1mlHVwPj5Gu6nnmsQRn0hSxYUoqmHhn+8PyI1AWoKxwUORi3x5Ud4u5+NlW6GmwN+4R64NNOgmguOvMB4sG85RR7mS8Uwu9uYJej3zpVhcd0LUECmSTkhnnYxTZ8hcieMWdtlZqPPoHbVeyJbFFDKwldvjGp\\\\\\\/\\\\\\\/0d+0HGgwmdk71Z2ss2xyJbpPBxmF+y8NDXtLY7++2j+XJSim7XzTx5tXg9mAIIxZdUTnchGUs5VCsDxtUMF67h7XJxSXqSBHLBKJAq9k0cw1Hjr7nXlvSZ+ndmFe0iLsgTSL1Zw48YiNtw5uWGla3A85n8sHxtVdySLPCp1Prr1OqDWtie5Kl31fQ7+UXPL\\\\\\\/YntBGJ+n3y1bFuhXMdZ6eZTG6HgTNRx\\\\\\\/6CkwUWacE+qc8\\\\\\\/bTAerby\\\\\\\/uRlR8If0GTS5mrl+ByvIQxAhPZ\\\\\\\/nRzqIm2sOayDfyBq0HvGT2grk+AEMMbcGDRiig2EWeZYeRLsg8it8Zc2WO0B5y2yS61jJ1HoctkkANhmfD\\\\\\\/3x7f4S9Zxv\\\\\\\/zlTYPZ\\\\\\\/IPkaqKJswrsHkDjevc4y3kfBxHQ79uoAxUusWPe7FbYGtT2l+PfxCPMYueeiHGOEebO8Oj1c702Czb+VYLUeO\\\\\\\/hy8w7XEx7vdQb9xPGTEBUI45GI4Sqbpo18XPMKkpN+EGRvQCuZ9zUy0C1fRAJBdKMMThcfQvbyHJ41dsojhVRNIy1yFkagIj0hv0LUC9IVsAAwfOd\\\\\\\/rhtnmIDsy8vtTSnlNahZT0ME0SqcGus3YWnpy2P1fBjRWwpMhj\\\\\\\/dSNHJAHqzEIfNSYtJoyrvcJJ3J1r+jprdnOOa9nuvXsnmRWe8YJp34rYPy7Q8zwSeIK2yyLzRMXYWnJZFSwaJIqD4UqdzcU4WSSmUQ\\\\\\\/XIdICOpjC0L5TSogWr3wkZZM71\\\\\\\/GXUlVkaIOrzAFrZUvd8FnjUS0tpPiSK0Fm4pgwauhG+20wMpvjPMNs9Az\\\\\\\/AKneUgbC+GWWEFPs8qyTjz9mvodUfxLXRPpgm0BMsE4ReBD0cUofD4T7ln5EXWt5AayVL+2ARgEj0U5EREsPMe05\\\\\\\/Lt9H20gJSmXhf9GL88TZUwR8bqkCbuENXnoNPPjHnpTUtUuVvS7RBPoQ0cePSQ9uC\\\\\\\/Kff5H6EL6gXbusswxuzs7fy8gkLecF7QhrHOw\\\\\\\/Mnt9PD9U8jAQNTWPZ+v8VOKDjaY2JwkoXpVBs+pLjWo7MvRLV1kdlToEtQhDOe2MKF+SHBtlyHQvYY3rf5gw4Pusw4DwdhxiIXj\\\\\\\/1jaqMgfpfgFY65NRP7fWcGLM\\\\\\\/w6A7\\\\\\\/pom9SuGRAl+k+Hed91LTOg\\\\\\\/+eQjUAa7EFeDLrFBd0+yi\\\\\\\/MTYIUQUkzNIxHnIXHUm28+V9sRH+AhOBNf59kJNaixKnSxGesBE1bHHkFPkYeo2NYCf\\\\\\\/s3MSpblKbnRee9GEcvwHkJcFKLuSv0K3gmf701uZXHjLMTl5\\\\\\\/0FwiVBKJJfKAfTvNUtVUV864pyx6WV8vR6dtMpX2CWLYnOPjpPb8zE66D\\\\\\\/vFEdWuyVi3xk628J4YEtel\\\\\\\/cSfI8hzVephu42n0oYFLRUh\\\\\\\/s6WLxP1OdOw2AYUpU9XJ4aiknb5D\\\\\\\/xg60TdzPVzOqvnu6y4lc9yIEnQvPLojhyu4KWTw2Wgj2RaAFqprAF8d2vavlYoabO20by4gSFTCkNPUCr8ymPAaFUrA\\\\\\\/vTKd9fTo2n7zXZFnAB0qZkomDI1j8Afvn3XEAWpUuWCJjqox72IEQ57eqThY2YYE8lU+ANEFqPoCALWFfKHQ1gb\\\\\\\/QtXksTsofOFwXe+aVgyBqrkt3Q6YWxz\\\\\\\/\\\\\\\/tI6DAvWP\\\\\\\/mN013egqz6tvKebRtngWSyrsHCNqSCu7Rp1aBBoP9O05QXTCDIZqdV4OBvanJN5z8yhRdY1RIZ98dTAEyAhpH0WvnkqqCOiLMx3kpBACFAMOMLs7H0BW2AXq0z9Zd\\\\\\\/kdktLL8R3PetOQdY10JpEpDOQfPyDdgQLgtRQlHdeVcTNGS2feZ8cTLS1kDu9RslMXZyZkKos3RdY1WemHC4WfcM2AAru683+DskRlS9KXmD+byO+U+6MLcAWtghNkmpM3YoDyh0nziva5tKF\\\\\\\/9gIA9Lh5nFCgsyoYSgzL3+k665wH+58\\\\\\\/LHvjOYyxmfpSaEfR2rCUx8kf4Nnl9QTJSNerOqDxjAM9MYIkpXLc+jIRAMXB30ItADpzoeu0Ue463BKQLR+Y55IV2zqbJelEoEQDZ9B49Gw7j5kZdtTdoiMKcf1C\\\\\\\/6GFzddtTg9Mj+6AnDBrwUEjq0RI38HxCQ7LyQrZSH260HaVCSjZS1iGPh9Gkvwb7Fv3SP6W4SVCf+TlSJrQAIbiJEvG5pqDs+tZBmFYkHRFcLF9iRNA1Z2railhUWYIuQ3xqRtom9ME7FpMGFxBPb3I5QDZ8OUYptkSXDABfGQYo9p5YkiN7iD\\\\\\\/Y+iPiFUH6IH54lvqE7IMcS3tZouAeigBAGMlZ9AYlMipzJvXUw+tzkK6VA4klE5Gxb7jKsqda7vO8IVEgoxGmrbqnTjWZJWOZxJ9A0CiLEBU1sWGohQ40fbzMJIrV6oU1V4Uk+l7Jly\\\\\\\/5sn9MB7EkETo03YgzYpJyi66M78l7+tLzIzqIzVxdlQLvXfaWz4IYtT8Xn2gK8lkPNXcA01kJyP2396WtO60y8gmzfti9wOdZN7rxQlw8MLU3b2isVPtGcYZmvu1IbUYnpXdOTG8pbEJffkAU9kn0YvlsNMID8nUJZ\\\\\\\/BeLgr2KmGx84H8hWEMDmfAuIkoHrCuumHMOnsNzJJdjYmPYDtsSDtNbNHkiBd7+zY2cAE3PdfnTJI\\\\\\\/ONCj2hQ0cmJoTdiXojdU1J2K77HknqHPBki3ny8oLe7wRyxTeJ0QLIIhJup1yC0yT3XiCEmuBm\\\\\\\/QSV1dRd7kwt6H3kT8+kXz5iHK+pSEjNfmIYgozKhIx+lH3+KparCo94q8uUGopi41sq7sxx1ld4eGs2G0AxMp4x5kGyZKV3gM+HJZlhalrdNhA6SWtvWhrfeUHeF7IpNJyA76JuHEmAhiD9h1ynEvuRBEtuiPkY6ww2RhHlbhcBn+vo8VDerlxcB8p8YfM8Ufn83rZDYN6W6NzhE90a0kXYKoI\\\\\\\/3LfRmweKIN1YzIR2VfD+MlS7qdoemD9nQe2OP9GU6nwDpMiR\\\\\\\/VTqR1gaC7trX9jk\\\\\\\/EgZ2Mdebp5WA2crEIAdtirlNIU1vMauGGANHFObGa3Kv8NAD\\\\\\\/iVq+XYa4MhWLRfcHSMmaaDxW8Of5\\\\\\\/Y\\\\\\\/i04msZ\\\\\\\/3UMrf1HvhRPhVRf7fvg4OyIjNjEHoM5lnzPEw75mUWrLCDT7tMLUnu799o37AUdUkYmdRDzzP0\\\\\\\/BmIGtqpcyRVVAkdHqTDPn\\\\\\\/sLqbF6xlJrtsQJ80UxF4hTHF+u\\\\\\\/nVC0Fc3IX4sUnnbzIyO8Isbbv1cMQetxp3wNnQ8NdP5LfAF3LjODNd+OgAxzfEHgV31wtTJmdblQmlEdOW4PD6mYHg5SqtB7+Q+zloeP3JSljC1HjjqLsBQZx67ghp8sYRtYZNN04RWuYpO1QGieAii4vXus0CBtV2iE5pYWAQpu1qtR8hkpvQ\\\\\\\/QIL4GmYHLxE4gtnzRr0yNWYMJa15TO5+Jufg7qQ6StLg02eCpP24UCA0xrNIxaHc5wqZAoNwC23NN\\\\\\\/g\\\\\\\/FeEZySS\\\\\\\/0YTQ2AFofAfVLJ6UeUTnXtZhtiD2AGCTqzsDKs6xZiQX5ysu5cvWp2L5F0SS76qFzD4fHEskE19xXBziRjd14rvu4dZOmvnwMagvFkQr26RMAC5I\\\\\\\/5\\\\\\\/SCuRoyuV+IKQK3bKj59\\\\\\\/1Dwxh94b6e4CNYVudqzPGG\\\\\\\/R2TvlTtyIj5NOdGKNT96pcBTDYCiBklVs2AoeyVCMB2vn7kNDJFTyD3QO6hIlfThdNshA8zdQDFfrdJgj6dSuSp6BYwhW4lVSuW\\\\\\\/Oq54gEFoE415eJnIDewmir921FfjEXE\\\\\\\/vnxkCij4PXPLbrjW+pS\\\\\\\/7XuzZiUmYPwHZ02RQZm4miHVe+wgislkhdoc2XX9aL2eZeoKFbloMmBhlD8md5KJnaE3idMCWAoSxosV8tJrjUSk1zEHnUunlxc17\\\\\\\/q6tvtFvVlOphgxzmrIabfWRH1KuK7b2nv49cSKyq3n3gooe4Kucx04b\\\\\\\/un90wyN6YB4BVi5iMmKDy1\\\\\\\/bsiRxch48Nw6dt5VknjykUKXe6tNnS8muF6ZlbOLDWhUwGxFmIzlIz8tQ+wv+oYRqGJv5p\\\\\\\/NYz99iqNhwUJKT7GgyoXTLNXfbYSOjEe\\\\\\\/pmnYtsM1g1FmkVzs4WWYnuN8Wrq4W4oDQ1S\\\\\\\/RUfQib8ZOzL5SnjPSPqPKGPeruXC5bIxlyla5aR1pf7ZRx5llmcCMrrgstyoOwS89qogYxSAT7Eb61GWcbLQpYOmt+wYzdyquMSdP7tWrX29X9K7UOYiKxNfGvBmLmwU3C7WlImjK6J1IGJST4PuOUJX6a+Yl5v+BqMbfjlnbutqbqGWIKbjXDsDQRunxKcJsWTs7KFKL02VYWzDjIFst5BHgEhosHUF\\\\\\\/uYpH9rd\\\\\\\/879OSSjjFW0AL3Jfa9rQMHbGu5\\\\\\\/InxJp39si1YnO0\\\\\\\/g3Fhlq0D0fCFWb\\\\\\\/MaXBONo3OzwEZsD49OYdXb54AXYsSjHmx\\\\\\\/t9DhE4jLXxO2dXNxnlekziswHTQtAU83y6MpUOLVSZypxMJatQQzlza89RzrVmBBDCv56PQRqSS8leWPpUK8d0pjysr6icx9JJYP1WkkDtYgL3jI6IN4PwMqRRwnvtKK40ZUnMA26l123vTpfF5i94gIFP97ECkGOm1ft\\\\\\\/dhknS6kWTO43SxHB6d1q3AspRFEXb+k5g+SKDrnczlWyAIsh8iwg+BSR0ggkYMfdj04d1xoUr5hvUXsglPfEgeqVlPelc7rPHvqLP2v7WIrFYcJLVA3\\\\\\\/Ex8dSMwhHx2Ad6xHVS6tbEU+8YQgS+qnMn6Nqv9rdoilRtMN6qzujze+mE5Pt\\\\\\\/jPtXbKvYThGBOFmYpPtzG\\\\\\\/D+AVxo4nmmXvreBKSiZp46Two1V5oikqLPXjkaWNoZANaQHehGMOxnMO01JCWU12PxE3wtX9i+z+FbK7Y0czi7Y\\\\\\\/aIfEfMeGXDly5Vs0kwv5D+vm2\\\\\\\/8gkt7segjvY6Uas4IWXtDTHT5HyZd7mk7NY3r2o9oGLnvr4PxZVajJLe6xjO2MX3ebTBtvzXssFlr68uLemVqnXqUQtJM+L+vIWxsUQ7qoDsD4VTSh38OJ8A5JLc+VAwYBwpSJg\\\\\\\/9+r+QT5kAaVxCq5JXCl7LBSEIRnWcpbFedluCWTrDJfIiF4XMCigMRHjSXsU6RSTKuXwNwZ+h4BhY5lruWdvzBR2SdQcKwkpDjH2wpdAWow5F+Li4Ky5F91RLahKFlzIjFHiL28ob539To6qMs1z+YK1gN3GJ\\\\\\\/X1wHxHeDMryYu1XglqGwxv2sNTln+aJJN9GYZ\\\\\\\/vy6JYN0JWDoA4IUwpz1r55\\\\\\\/9l09gDcl9CA2WFTl3siSsiOIrizqSvOq+DHAXpxJSjahm\\\\\\\/wpkYZ140bK7\\\\\\\/pnzo5BXN+S3ErP93FicnORIpnDO6Ml8j7l0IAyJ\\\\\\\/uDd7SJEU8qKVvr9qs2JNA8CEHMy7dCnr0AtPlcbZKwDt2sNZ2G+t4dvJjOEV42C6J27S+2GlFDjYvhEtnu62w+yNY5sAMSBmzuKJrXJMvY4uNxROlf\\\\\\\/Xw4NZZ\\\\\\\/tqd\\\\\\\/pSxDFYUcKJMgHgWYt4YPCpLbChxp2HuY93uny0r1KHbBB6XDDf5TpBQgNrEOTsBbn+F80ZS+Hc\\\\\\\/4fh\\\\\\\/11UgDBwaPhiVVMdVXqf79PGz1mv0Wk650GxOn4SmEs9G4VgeByyCXf2F0RndeNNgFJ0vDv2tF68Dk31Uw9C61HzfGhs01Mv3eOHemYPkRlnWdm5UXOQVu1H7qw2E1eSe5Lu6GGx2PVF\\\\\\\/pUB4c5HJMciNm8+uTVnWopsPV1CH+uCXb1l7WOsDqB59wSrjQ3wgtjuEx8dISdlh0H8yqbqO0oFzXVEV3TxZZU9s5t8liZF+usT6TNphMeesALA7cyIlYFgNCwei8pNXs3qwBVikAAGnqMPx\\\\\\\/b4Wu57aXg5bEKSihZbSiWVd5AWeWHLlrQ\\\\\\\/GCLFsw67waKhlub7311vFszG4G2NgOKESjkbadq\\\\\\\/qcxyXizl+wWdjJidznXumueNetU2ZeS5uO3guD4IXLpA+iLJj4fW2nLDj9z+3p3mIhAyEP\\\\\\\/eAmM6KdV7HeMaCqcXCo1Tqi6ZShYCkZqT9yyehvXGedHfKPiEnWzbLm\\\\\\\/f6SIZCTzlxashNH45127LUs4f5UYiXUayVcdQjWX2\\\\\\\/MBeDVmojlezG61mqJtEDI81Gl9n8reUIbqagUgX\\\\\\\/FL7pVOoRN2YpIpwa6E3ZM\\\\\\\/T+a4JD7duyGxbbQ237vYFaczwgvff4B27ApShwITeJO0Brp7SnmuSWAhiLQlTy1o3cas8LZL+z+QCrAJoxpyAz\\\\\\\/2Jlpp2L0ypgDrisG64DqrnpzkHplOb+qp+16OWYqC5Ha4\\\\\\\/0ArcREh0oXHS6iFByNGIj\\\\\\\/kY71EItPDvTtC9umGHKOuB+roqYf0xu1N20y0jsQQ0YmDW2P5ez5Wynh8mc9vctGV5vq1zUTtYZQ4iy0AGykXlix6XwQDVtKsuhqZHZfN7565VGPqwSFNeApvZYPgizwtOJF3RpfkHgWIcGe0+ADrjWIhVv8mUd4Ph\\\\\\\/YDpvl6nQ\\\\\\\/+WHmu1umaORCYZvPzA4s239rMOtAmvyuQ0GrG7lTqY8rOE0KwzIv7U4QpN8SmlSKDa96LEBa7hVogBy6qCJo1\\\\\\\/0Y0oZpNyJZ8hLbx7GHDX1SRsRoTGKTpBXaEXHhkd8sJqwmiIg6ed1ifJjJJ3tg2\\\\\\\/rsIruudBLooenK9FweAELieSEgoBpeQ71dOcdxzfHkjFCUvx1ZDtS2zUspXRb2KiRSfQ7H70YxBkt961bfWksesPoQ\\\\\\\/d1Yvyl2aNXaa+BSlTCrXuuYqoZK4eLCslHTqEZZYx2H8gnPmPFxfEflBNoLxnf8fKEsBGTsB99gqld7hDjmU791oerg\\\\\\\/y3ZRUXJHhtIZ\\\\\\\/ErtAZK2inw6Y1CmxpnTBfK3xtibwOlHQpe29R0tz0TfdTH2+\\\\\\\/1oUYKzE57yd1A8Pt4OpllqgLYxh7166NTtCP3EQSmff5Jd8+qFRYJwi9jqlbCbY6s3FoomWr2Vey\\\\\\\/FvFfPAK1dBw+YxyoZ3Vf8liDBuzCTcH6epqdD6n+GFTuwdA7+1mOuyiG3PqdjkMdOFToJF9IV6\\\\\\\/N1bFjgxuPIOcf9jH+JwL2cJs4gR0D91Q3IuAlYRHwbHL2OnzqEGNX4lJu5TDTiWw5qMFon+XzAjNlEcPqUMhC2ena2U+9x++OAucddEYr4+tr0dqCTs3GXb3Qjk+oJreTr4GEfMj+U6rDIMc\\\\\\\/yS6NakNf5Np5++u46NQ3lroqEFxCgfCLfdtn\\\\\\\/dP16A2geFH37MI\\\\\\\/aqobHABykALDQiaDyeRT+C6eAsXguIfHLRdYaZM9pc7gaHy0fOGMhCHN28szKsFQYHv+alHS0jrcEDSHQKSd3+b5jdcKz\\\\\\\/V3WKlJS41v9Mxl5uzM5SV9jhULrKN1rT8vKiRbo2vT\\\\\\\/fYMLpHw+BQ5U6dxLZyuYH5cGGdKM8S0v4zdBpO59jfd77z0wu3PX3+wdj23XbmGmbS8tt+DrLies\\\\\\\/ChmSH2NJ\\\\\\\/Ih+umMKfvp8dA0E\\\\\\\/RXNhoKdijVJ5WN3QFW3XRJi4bADS2B0FSReWUYP5TSbUOyLWPjsj1sYnEeVcMTZx9d\\\\\\\/4yvNTR21pGOyuPVrf6nlA0VvbyYTiys2n1EzFyxaI9D1fg9yqVJQawK5ytwxfRahOnp7PPtRY+79gS0iZb16WpMv6c4Wgj+7wyGHZ8vkap8Gf\\\\\\\/hJTDAra6KnAvVwhNrO96PKN282pJ6v4kPb1xgMxYN6xPMJgE2a8HPjvvgv4nPv85yp3SDzv2lZY3SKbmpY\\\\\\\/f9fHHNVulqIcmKM6URRPhorIpF1rWz\\\\\\\/nkT9\\\\\\\/QzJ2swnu7+xP5EBk6+0b6zdwvGhq2rzN\\\\\\\/hxRa7kC3grH6AftL\\\\\\\/gIzINQnlT2P0Cj0QQaqovXEPwh4tvdrLyhB5c4olXGKlnOmBuvqfgZH4BUW83qnOiiKdebkhLlKwD214hCt5H+AKpOSGFNy3MUozu+AuLgVcXBQ7L3yBBXcpg\\\\\\\/6lOhg\\\\\\\/1ePcmBw5TIpzkhPMdtBJbFARa8Go6DfactUWDIbdt\\\\\\\/v3NVV1EEjCr2EGhlIxgvjZSDbwyMX23DnxfNeNbYlpwO+SGNlTAqzkhC1bW+2hCiWrcEyTTnyJhMCrGGCiTac4SOrITF9PulLCH6MvJDn2emHC6EO3d2cjfL0ApVS1B4iB9tB06xiY\\\\\\\/Qs7rjL5EyRkp1DTx8ZlOaiYVpjwdg7hxtNzXTUrIHUjtwI0jfexV\\\\\\\/8mo\\\\\\\/bImGWUXJXJsJrnYG4KpgS1LzTJS6OIObPVyUl7g9xjU1838l7F7sU7OUjnDuUGX0S+VXj0ZwNMfnXfY70sZ7n2NR2qt+\\\\\\\/iVXXHqqeBsgbivU9GZpL7tQGlCrtp14rubodJr7exs\\\\\\\/ejuWGAw5qkB8HdP5v1dHPcjWcLfi0yqmjXmbGrzCNGbaySRWbvPNO3R4znC+M4OgSZbr0I9+jstlltLkTFf9m7m6M6zIYdiNU+rvMjm6I\\\\\\\/Wo9pcL2DRhTD3UpFBIDCejbSc4bzNdHZOvj7ReyZMp4A51BRLTlbDgQ0p6PYvzihjwWtzp3PKhwX7cQOKVSCOFbk1ACtB+w9NssSj+NrDUIa4pKfNMjjor4IuK9Akha2cAVENbZBbEA9HydajFpS3pgWs6ljVeqXIF3SALMzzyyG\\\\\\\/cx1lu9JndX8HYaXWs\\\\\\\/upKzjyhYVq8+uawzo8mIESyhi88Fc5C9vHuzL7V2cQvs7tkiVe2GClf+S8yUcFVdekGX6YHosLobjyb1gmLs1d4TCccMWdst2NLwRPMbr11KXApxLdtKgqbJ2CZTGPZUez4fIDipyAPPQ8psmBTjDPLXn0W5GitKyKAKwzpMpSwlElDqe1nf+qDAq0HqiteYtMiJ5tYpF88xifTL6GOLAOkTYjFZYKPEssvaB3mN1SfmEMHzjyviUrJbHWnniEkmGibM3+A2laS8aRe7MsxE149sbLif0Lx10CVBBXT9cU6wFdN6+vfUDK0UBY7EdgdtY931M7MpCWc0q94DMZDh7B2gJIhOVD2RqlxRxq1Epfdk3iF7XJPMnzF\\\\\\\/yqvSxx5z2NlfpUz6k\\\\\\\/jBukYfr3HdvE79OAqEGB5TjE4ELyA8l8DDBCxFqUo5R7mSDF3aMPk4v9NIZhd2pZGPUJ5MXIhYdgRx5HRXJWhuPVqqEmAXSl65xRsmaVgSWuD1UzE6Wky+pa8anWf6FiPsSZRNP1LijfKfbTA3o0w00dicZ6n9\\\\\\\/oKDlX7ZISZycHG\\\\\\\/w3vq\\\\\\\/gNSIqhcOXSLXGibv7Z6qbizQ4AFJjzsn44xlv2xZHIcsPwNNGnf74AOrFYdW6lI2fQBdFqewmLuH+NslUlEV6w7ZO96RWympG0q1fJKC+ICKlV1IpQ841J5XWH7DmdEZHCAQFbJtsX9i7PKYLXwL69xeR2W2klHnZ86wFxY0GjlhqpU4+KyrzXpjM8Rgbu8oUiU7\\\\\\\/ZJGUJTebRvvAySFtSDaH02XYi4yddXtt5az\\\\\\\/T0\\\\\\\/g\\\\\\\/BPkY1\\\\\\\/AfeM5nBLHfbIkt0z6lqNECnGhURLDzF+09qmB1V4B4JRi\\\\\\\/i92pSYWffKLdvn3oqj0AQrAqCk3BrXGy\\\\\\\/aWWyk65hr\\\\\\\/ZlTGjgEG+H8aVTOcc3INOKdPs9YU+xPDORTEU362\\\\\\\/WEcgqVmiEGbyQE+NlNug4xHM250ZsiAbxrD9u82S1\\\\\\\/8R+7eWGZYlxQBcPLljID2KnsDCg0d8naDmf5L77DoR6deUEHP6ynFQwzVAPz2SIj7XMauCwmHRLEraPaXrGtov4m3Mw3MsXyOGnvOyV6h5DxgCuhSLBqOj1FoBG7EmAcMUE4XO\\\\\\\/bfLma9K1yA+OnCOfOJ5wcI33cAUCpqYbWomAAQLB9ctvXpNdDaQ2dWpyAMtlkEaFFjvc3cAYb0kinxiEN+9\\\\\\\/9OM2iMgWdYWh0OW4n0o7JfkiaFuLLb\\\\\\\/RENh4Fy2yrvr9orXNZ6wYNblNqoQDAVFLdpUfwBoubQ9e08DT4lW6q6xQCCFYMnZeO\\\\\\\/i0Bqutzk8MQHDWn5p5DhpZvJ3iAFZwmy0L3J34iHWBQZX7bU1zV5w5bo+pF2J06Fu04SuugSV1Um6RVn3\\\\\\\/OWEwJI2Vsy8wHKRCuxZsiQCkCyU7U1PJtjlNoGY4P+yhmvlBcuHB3imxSz7q\\\\\\\/S0V6P6Xu8Nuu5tTpCMOo8qXU8+spZV7X+Fm9ZkCbSr9H9HhlNcZ\\\\\\\/FL2DuaDzfxPCLfO5C+gquALLLr82KmcB5mD4HN\\\\\\\/1W6htYjtvpbAi9QrDlHyoahMGrVbprHfV2Z+KFgnL1vV0R+ErJg\\\\\\\/\\\\\\\/R+2zdOUsm\\\\\\\/yASh3wDZCFGvUTnSwfSE7dSGu7wcek47Pg0wCVoLC5YjfQRQ47\\\\\\\/AvQR+zUgzqaust5bD+5J5OF1TVzoIcruIAVdDyI3p\\\\\\\/unrp5XW8n\\\\\\\/9tzVU3qq3a5ijpQM3OBFJSPlTSeYGX00NK+6qPRPpv2zwh0g+yedWnGAd9ZRsgFeiuTCzyID1QINyoxr6ORc6KXVOnWZsjnAzbfBmE8LTAkWbUThuXKCWsl1YjVxVUT9nh6d9TEtBdIh0ZwIGMeZFV9yPJh7NUqahqxT2Yjje\\\\\\\/1mky+oxiU+ZQiybJ7yNEjdPjrwQWCB\\\\\\\/C2MUSgG8cbURpYTRR0L1+HIiHlKW\\\\\\\/xC\\\\\\\/XdJNf5ScdWuGZhLlKO5MrHsG5RRPx3Px3A1dxITRtNBJpIZOT\\\\\\\/hQ\\\\\\\/9ebVR1iMweLc89ibiS9wx9Crf+9jxoyZwylIcMF8cFEzCImn7gQk\\\\\\\/4iD5rjH95vfUTPxBnNpAaj\\\\\\\/O3nyLqK3VOjh2iwX9EYwS2T5kAikA+WlH23BhfcbiFg0ZuhjeT0fohFIb6nhNwZeGKsureuN6BFjHGOLFIO5QGAD1jnvv4LTZBpDYDYe78blhgqOIRrhS2bcCivFeBUSweyeQzL1y0eOW+tTIUB+y0+LoAXsfhXmCYs8UJR+fmVf2bozZzAXDHEfLGslWrl2d5QJMugl21\\\\\\\/hdxgg+kB6i9d5mVxz79ODtXIrwC+x0Arz3tfRaYu6An+KjoHtsyd3OqCOxiYxHRkDw2Qz5eFX18IuKz95W6Qn7A\\\\\\\/SdpzgRfd3t52w6lFzVq1jZtkNj2DLvRlEWbERns0BhR38SLxZb7Xkm6iFxMjwIzJg6tmZ+YcP0kxu\\\\\\\/Y6XyLg9UPrE3gsGbK\\\\\\\/IBEoeOkBczsxO7HO8hz61nBthDyrEgnulKd\\\\\\\/UFOrR+DFY4fL7EmE7IFdxhX60QTrbB9YAlwhMQhmVaeg6sWID3JwwS4vXL4Mv5Zl3k6cFNKfWkd7Nu+G\\\\\\\/y\\\\\\\/TonZUnvVS85KeCjwYAU0mFtCHOXxf4PzU4A14wfWvBQfbZ0D8leZG7m9GLAj8HfVxjfv0auwIfx7OX51sJbDJDkpEocZSTCpFCHwau4AfJ3LXVBNSLMzSj7RIjCvzOq6eA\\\\\\\/vAk6oFOvzB2945mQW0vhWf17T2+Ldwm0zQSh6qL4lW506EUQ7\\\\\\\/puLTBVhOZYBeIewhcrh3xQOhKp77u\\\\\\\/WWw6F3z6l1hdVoiUEyczU\\\\\\\/Pv\\\\\\\/IBcHp0ZtK0PjPTCfogCet6yJ7n+ZK8XWKW7KOGxSPJ0HVXwWe9ArIHb9a2v\\\\\\\/L4x4r7b00eff0P7M5kIfnLMRXK6JsKfzomMF\\\\\\\/hYwUAyzxxbtPwz6NQHYWyVFgCfHyrOuhIliADUf4oe9QNnJduZrWGJgcrE20+vdIFrOMdPTddIkdBHrLu0loClVtvmACnLC4fx+2YzwEb\\\\\\\/qhOzjYeNLpUuTYusG6wklGb7yhXsk9cl2WRvdPH1IJL2IKAwRdpiaBz48UDLGigYNauPVVxA3nJ8HXisOtEeSe0TnDcXdYSUe+Hwu0\\\\\\\/Lq6Ms5DKBY+H0ONEh2qNRnmE2LmgbBdmcSHR6uzTDzqNuicq3il29PWJ6Zwvbp0n8B\\\\\\\/FlmC1nrrpTgEtfQXjOYQT8QhviemNNMVWoFjrnapx2eu27oT2lyiN8eHJGhryEwoDYnTxq3p8xtCQAuKeuRP\\\\\\\/TFBUy4PJ3xpAxEr4Q5yNy+ogpGmCSJgKY9MTpNzzjmxBEXZi0zBVlquGz\\\\\\\/AFqJFSemfdWcrelrkAZMA3tOJUS7x1rZ7f8iwqR3Y1H4\\\\\\\/if3DW1xTTqk0XV0Qqdi02dF6ST\\\\\\\/zaGJujlgu9ooNypqOUKQfrFVWh1ySm8Kx91bbX3NuWqAQkwpWDuwPFQ61eKGbdiN8k+\\\\\\\/pZgvbLgDXYeNpnvIrnpqP6mtOCxILyd0WkVC9oTrEArxgG1W+mMgUtBwXdTcI3GqY+YiieOykHG5Dk8J+epqLew+7F\\\\\\\/iROmBFrD71BFfqNdcBroRTqTMSsOYkIY1FRP40RjUJ5Yt9HKrfgL4YdZGAJeFxvt\\\\\\\/ySpoZ1CuyNdboK1B0Z5ODDgHpWv1yW77IiodGHeueLwMbH8P1ofW7FmvRBuN75JbWZQwOPSqo3YnLlpP72ptnAoKWqwQCPgUg1JqV1PayBX\\\\\\\/NtkDgDr\\\\\\\/m2KwrAV\\\\\\\/WcaCfNQ9CA+DObALGTAHbxZPicLa77hN8UBl1rc\\\\\\\/st5iURyxtLruKQB6nSfCNryLmFaeMarfGkUc9clYnGjQrUPck\\\\\\\/HfDA3xnx4octncqCPQ5XMepekSLo7aTUgozpc6nAmPAPaJD3q6wyY3ENLDjvpOV1UlqGRmxK+vYNlW14p5QKjXIlkTXMjPLK6gz+GPlioZu2+7A\\\\\\\/yH1Ye8vyqDrti180bGcZZA2uKkTlgQMBhrcTJ0p\\\\\\\/ai4rDn8aCXXRvVgU\\\\\\\/oB96a60yjXcaOn5mGtPZW119eNipXfX8cvWU5wA4qTCMmRyYa6LE+Hh5\\\\\\\/0LPoOQNS9OVTQtZaFxkra9f0UrS01H8nwBnWceP6BN7Nqq6D6slUz6+yAKeLXTWh\\\\\\\/Urq6UM7MBaqsKaZPspa2ZcuFPcU1lL+UWY0UzM\\\\\\\/DNGOaRPDxarv4WRL+G4FodmFT0F0hsRAeoJAfMmrqyQxHT4cWHdxG8G5solmzuG1KUMqdPuS6eBRjNbhYLYFg97U1pUwBNC2Sl0hZBcaFSvRb7bmDfebUJC\\\\\\\/AWjW2IOdCj4s0k0+axjh1Y3RP59uxVk1PzS1eoNgOlcJb4QQri1QbEgfXHYZ+rRVlzmbWmya3XEw0keMlpKwYO44TkACgqEjR\\\\\\\/f3V9bl4Oz\\\\\\\/fY+HT0dmh7OCIFVT+lDPbJwu3n5ixAJNCgcZjOd0sSbPPoX9wfHFwdFD2GYZ6\\\\\\\/Kw2gBDjJG7rIrBVMmAl+tDv7xF6OZRocnZFoxFr1dfHDmI0l73UcbnArHbU1tVnw0Q4Arbl+RwgP2wJwUGL\\\\\\\/xgtHxB2dfSDK3h+GjxDOV767FV8dywNUM+ftjZjc9Y+\\\\\\\/ql6rheqP82D97vUntTzdt3MHoqLR+KANbDHmlVG+XnkBOmjzLzLXRplY2TzMiRlaNsGASFsfQfOMAUdEQq4B5RPGFVasEIYoXoTFdnasqJW8FOmyGJwrpfunWpr2xOAuK6fvn5+wXYk8HwkrxaLOZXj6660JZGXihJA+BRwofte4x8DwNEixFzGnFn7uEwyzjOgNEV2wkgR+n6SSICZZ5d9KqaCjaGnd0z0qCbtIrXqxWiZSy4I6RC0+swagmuMVKPxEOQq4Vk4T2xadLXXe\\\\\\\/vPQP2tLJkAEXGNjskMZW3g5Y+UOEHnLvMKiKIOB7mfgyyRUEoT\\\\\\\/GPLPU5jnTo0aNygCz\\\\\\\/+TmfSiNZWM9Vmn0dCVp1Oa9GKxnflVxIvzZ7tNxBmGi623EX3QOpNZ57Ns3gIPmg+8XXF+m+9dK41ljvN3sYbJ0+2k3j9B9YeBegKvPjuhfDZvf0NJML6CyZJg6Soxo1Rr05zhAEZUr1N\\\\\\\/NjuiUELBkYNFHWOz+2mGRvbqHeOT9kKwB5vyl8R6NXso01rI8HpaU+oXjpUBtHTpupCWhnYrirHc+Hcac5LJ8BWfIXoayf8Hq3+wWyRsKqkZh0xj8KVuCSoivRojgF6qiqNWj+2uDk9nMlocV7LZlOy6PaJKhVyC73SU48tGhzph1yFVCY2EtF1VF9D6EB2OXBL23r4Ayplh0UngC4sb6w6dR032OPw\\\\\\\/GA7rGkT7XnDzOx9W9atLuWUfsLidJDUFvexT7+KMgmNc6NRY48IehpWpBr5JJMidB9dADTzZOGBgVYAYT9O1GY4CTk5OkfOJhE5iQ71AA+ABeZ1EZpqm3rUE3eHCWn\\\\\\\/ZPiZ6A9f6l3L2RDJ4\\\\\\\/DAqi\\\\\\\/XpjyUiOEC01v\\\\\\\/Ie\\\\\\\/aSaUImFvQ3L0QNTUNlQqRfXsAwFVo2Rje8Nk0n3I8+CoRZMrjUAyr+2S+LApujK\\\\\\\/rbyNbq3+PXH1p8lHXpJA5bxlkneqUwUVUdQ3F5Qw5OzOZw3T1E0ZdALFax7JKIQfL\\\\\\\/wUTiXKn9tmQIR01wht2j8tosJr\\\\\\\/YPEHVC5q4k7jlQ0FsByQXxAm93nODFitaKMcisEVG\\\\\\\/9fC0AmOS+OsP\\\\\\\/UODlgxHpTcEKuTaXjyozK9JDNaSshd0sToZJBPqUL2eJxkRiorWjOVH\\\\\\\/W+qzdFRzlex7rfI10pihAF5QB6wMohCkRQtWlfiyJMFAIffpKvxKREM6ykEYNtx36iI7FGjnk3eV0AtqoGkr4mSLVxCp6ZgXj7GJmAQRWS3HTZiCXLq5md42eITVf9tgR8OQzpW683rTOSYam89Iu1wtUNonBsgc1egjEKaGQJyGgfTp\\\\\\\/S5D18n8VU\\\\\\\/v+VMvHqr+\\\\\\\/upSyH70NzztSwxhkwZfcbhOZ1u6XpIr\\\\\\\/qaqp5O00YWHF0E5R0KrEzIuJrg46wcOA+s5lwExqAUtvjHiEHbubzZtVvbWdXwdu4377uUj2nV4vmNmtywsbhMGeHaXhIqBFceQhj73Qkw0l\\\\\\\/7YDx9SBkajyb\\\\\\\/hR9QF1rW5nwBW6Us48WrkhGsT3XhgQp61GG8HLJts1xh2flL\\\\\\\/6tAF9DI\\\\\\\/Q3ycu\\\\\\\/wabRUiQSBU\\\\\\\/FUBkiu4ib3qPF7KsMVWskSKvhU8Wfjo9JKhJQgCvrhRpA975LoWVDmLb\\\\\\\/AHoQZrOKfIFCxPkwsuvFZZ\\\\\\\/og\\\\\\\/NGcHftqn5qGzLNbpi4RqGd8f\\\\\\\/BjRlOgT9\\\\\\\/Pu6t3R9zPX8r5vggElKttiKBlXRhGEg3BsP8WDfb7HoXTTEcUgwvExo53j9i3fb4LCr6XEcd6kCp7NJsQnSKQ1f6pDRyhCiX1HUZmy76iD8N2c+6fU+Q+\\\\\\\/xqwmDfXEd+T6D36QkGro0clLNEJ5XGcxe12myYR6M\\\\\\\/HP2zbZgSAOZarqLFfD4G9lb7X8S6XWDph3Toea71a6a6LdtO5Dc1MKZ9JVkQxNm2TpZPPTxtyS5dnXBFtuDTCQ\\\\\\\/mpqrAkd\\\\\\\/nNcSJaBzHE\\\\\\\/w0y6KooHmGzVwNorYdiYbB07corthYruVmfINIFddT+7x9V2ShpUW5nqFeT2r9T+2ZpgJoVtn6p\\\\\\\/+iiIBNSWSYcvsO\\\\\\\/wv6tqIInX4WLpdt00DUJWOFDATOh9SDDsXlu7NYla6fb4eLY4VmBazMSmZVC8TYOEomosNkBVE5r8AjM6NE8mhnt6QgcOKl7vGNf2mpsPoDa98OnpFSCtVkzM0609OD0fgRGid9+5UhP0dLdZeFUA5WOsUNMMcWdmtYhhVkAwJdD+pXyouTwRIhVXKwPn1psk2cmhOWT0gBLfW\\\\\\\/7AQIrakiZW9\\\\\\\/POXOYxT+Logo6yWZchlCLx4I2VZlimmMnItfFONBapLuGSc0+ntdt+oE81fr4k9OjO\\\\\\\/lDbMqKL2VKhFa9CgPQU6+OMVAwvd96jgEUqjelnc8K5Il5FLqw51oNrZTGvOy70NJMfF8vVpCNTR53762+CBdW2cqAlXzSIGlpLlz1fQI62MigNpLPsnXLMQSAoloTyMoYT\\\\\\\/7PLF196njUV33Ckr7OeHiU1CWV57TGlsZX745XZ2kFEluEBE8ndLA62xBk4DGCEjrv24XieTzm9iUaLODH4PE059ra09bnP6fj8JkgNenHKdqzbKwTETfFdhJ9DNnY3OKXuYbTooR0XpJVVri7cZvJk3ILoNd+7EkEMO1fls7B7GJGeg9GBZaVVWlq01JiodvRZKFCaOpbCPRVvvFNE2HzPnK2WQXdGdMPkbgLWCtrYbntqJaGpYU+bElAPpky2JlzoLPsZ1lr2xcS0rHSA9jbm6MWlSmJiQwIIPbi\\\\\\\/Y7K0Myfo\\\\\\\/XvckYmfQPpjjJuqzIFa3P63ICPUxdF3jpsEvQKvQ+PO+78+qkGJ\\\\\\\/AXq2y\\\\\\\/Hvk1QmR7kh1zZzIkd8yKhWsZ8rwBCXTDlwTaH70cUHXOtlx\\\\\\\/C+O8nfYjqwIHqjZKm3CaRqhK+x+AQJEiUXTsCy1IYxqLpRjPsKWoHhMDXrlHQrkHhfQrMeAvVQOA6HNCXQyoDF+uFGZBySRHrxf0KyfeEgvAjCemzxAKkURAG+7UTx0Q9FlnuIIV3SyrYiSAviqkx4aUWCWDSdOxuvwbiVqwMAVfMSMuHbuUKVHrWNBklV0VMxMcWRF+JcBDVpi2SdPTseKwATT40OWjDaZNYvayBpUvkroBU4o\\\\\\\/xkE0vS3qcKab84SPS24m\\\\\\\/dn88U6zTUk0CBtqAZmfGeoCcq7xoXR9keVUZKq7Py9xcz\\\\\\\/i3SHLqAbkB\\\\\\\/w+DKyFm9IEIsyJsFuBRpzvlwQ\\\\\\\/rpidbyJfncLdRVR8wCLdWEEnu5DiRvsNIm646KSAFUOK1zPV7QmoVbWajHbJcydCCwXjldgzwj1nbThHUhxZ1MnZX0EgxeUgb3h33hpz+FI7Lw0av3GzeK0jr35Skgg8C5QGmBbevEU6fRTmUFh7kbanYoO2iutdVxu5Me8TfK3i9WOwWSsjftD2wap0GFOSzjCo7PFkP1a39QIEWoSMLboUTpD5MtLWUCRALasbYNkUrU3Us6CAdABGODtDNxmN1fd4xjEx3b6BOnftD5mb3MyY8nhRAVOR11AYoz06yrW82\\\\\\\/zzXAvC9wBPs\\\\\\\/zaO3P08ZzzZV5oNoL+xWAfFscacgSknpgf0u94jwDRBiNMKW342RWArSSjGvuBvCygcp81ZEBIw\\\\\\\/qH+QsPRkijEhRGPPe8HAGh0krdYXKYPwJGMwd5U+DKjnxglSgwmLwPMGILKb4n2gJN6yB47FFG\\\\\\\/p2Gb1OcOlEGNc3xuzcpalTJwqZd2afHvcUkNQm8FR0PgyYlMprPrm8EIb5+1xgyv0Np5R57DUQOdeORUZyvbB6G\\\\\\\/yJ3LX9rSCdEXh5QkIJ4Q+qBjOOWXCEu1RnerX5vD+P6oycyNnS\\\\\\\/zi59b3LbLHFfvBSnYrK2U3IzM9G+C18kSM+izLs+RBW1UuXKpljgGEIxz9TYMdHdvL\\\\\\\/cOwLa04jULiGDenSgcqJYGzsDHo5NReJL2\\\\\\\/PMPv1iDuoxLencQ61KIInFYa8sLOhMCzYH7SIboHKlMT2T5\\\\\\\/QOkcQiH9jY3r9JDABOkoF1uK5rPm69CG+rj+OwZfb3oaQBBcDvcy4LXwXa\\\\\\\/gDwscJhehGKoaWT\\\\\\\/6jGr78oBJ8UxYQCC31jSjPCmBKqG0\\\\\\\/hkkv9oJCBM8HmPcyGyCsguJ62jBNdeWqrq5SWrCq8M5HJjqOx\\\\\\\/n2f9THtVz3uvrxGKh7\\\\\\\/Y0LmGFnjvGoUW0N9l1wqrFP0ESPrW1aKfJ08w7J8qgiL5nAG1V\\\\\\\/DleeFbTBVvokAzhz3abvaVwv9\\\\\\\/MSHKVxHz12oOjMXwrDgTtGs0iD7NGCJvOTj3dHLfkPEnsZcSv0bXeIrStcyLCdY2RXT8s5GzF7ELEJwNW3lASz6QfkzfiTfraNfmCxOL1Xm7EYeKKNxueREY0LRE4J3uX1yu5XVRBUMQkBkFT2jCJqvfT1CFVvctlT9gIi9SddlqbKRiz4wetUYc3znoCj7PBqJG42rWt\\\\\\\/krPeSoeYirNXa1jvzkq1Ds+9pqmvQ8u1kzeGoBxp8jg3tzQlXP6RMI5LD+jOvuv+XsO7Q1J7+rgDh0V9tyL7jz7Vzlsi5utv+pSuqIn6zHZjbCSOJgXSYi7hhN\\\\\\\/wyvL7SrBBz6s0MUQ0F0d+51S7Wi8K20Tkv3VXPwf+MpzfN4KVGIUs+N8SZFr8\\\\\\\/Fo+q\\\\\\\/RAEkOQsdWAAHMbYSTPxlme0TqC0Gik1CCj+c\\\\\\\/qKmO2D7ibahlpfIyYJ7jyUqrn02EEEkn44zUE1vMrWFDFG0+7qMII5UQRE9WUO3ErlCa9oBF1QzSE3SSL3b85AG6x79awtpYrJt11alw7NoIlvy4VmO5AGHPwi0iJDM28uYzXgx5SYF9bAzFy7kCDLm+lRo5SaDPnXPjVCFW5Qwn1RRVaQmDXHlyy+SgqRVNuWUsScQhEqdEEkomjEADIBDWGMnZ1zAQoN+9NN8BwmlpDjfyABvs9JdkROwnka8p4jKmVr+5rNBNK1r9SKI36EPUjozK1nam9tzGEBWqUWOJdx7R\\\\\\\/MpyzBwwEWNlda3LTDcs\\\\\\\/fmr5a+IQyV\\\\\\\/anpdGlcYdxg4h1oH2tLwKNxDJBPzWvmyM2kFCPHdX1q82NnJY0gMlH81PenbDPob5dSn8GWhqLlchAvvFrLqJ0OxokphvBIUtc7GVP5A5TgSWQWmyu40g1i+yzZ+Rxha6XowdHo\\\\\\\/\\\\\\\/ew\\\\\\\/HVotTSA2kBx6ZFV27AdpEIWp10MNjANujxaHEL1QsIgE1Eov4Gr5KVlb2KPYqu5nhPudEcjhfx\\\\\\\/yLqdwTU5Ha\\\\\\\/OiNGFRhvBb47+3wbNYvfgfg+4d6fzJkiV3N6KfIwVvnPuoqp6wCC8YA+j+agAvUk2k1Rs2nLkiompW0bGJUC\\\\\\\/QTzyD7mZeNlEZpMHHxN+VWV51\\\\\\\/tlg\\\\\\\/hDwPPpIzkoqIqIELzld+zm4FsIC\\\\\\\/g0xQVz1LxArhzWr8Qa+WXiiamwTCOuKuh5ppJfFW4LCs6PZwDIDCKBWuPduocLe4gSvLR3Y\\\\\\\/mpU0VFmaeb4l87jfZN9gFUtGtCAEP9Rt2nuuZn9JmzUywEXtLanSFbvFVJYbiN9WMxPabBf8xrf43MDbTWLc5MvSsGZNdO\\\\\\\/9Y8KUrYAL3ONar100xthhpLpEV1FR7AAjn4jDjTDmImbsFpfp8TV6rELcqXYTw4nG2SqfAb6dw6w1kGKmluZKDz31jq8KmFHn9I8Ou539O3BooNI7R\\\\\\\/diKdVbKWD5MzzL34wuCZnffKlDmZpQb1+OooAWsLQkEOnmw2y5q3cEZs4VyHe3Z27Pn\\\\\\\/VfJddANEvrj\\\\\\\/9KC+GnbyF3q1aJUfU5cDPumOnA9v+FG\\\\\\\/tmXN6FbceZxx+8tvM5d7HVFqqQeHNg6KufSgMIdIoEDGnBTAcw6K\\\\\\\/eiCElUUmhF7skzJKrDGF7iu0ce1hiJZusw2qZ86QPRi31nzngnNHoPXaKADHMjJ1UJF9Q43i2lK2iW+VEM6Dg+p0sLSI3yz91RO2B8iB\\\\\\\/uEMQdCJHWKtk8nIx0eSb6qQi5PfTFxt2UAIqcewcMkrlKMLLcC3dXZwebOsPnk3erVDdWmaK9xPWy6wqSM0iCHVm4eftfp3rAlMRjjV3x+\\\\\\\/YFuz+B2zvDJ92fLZEu4BW\\\\\\\/oFizrr00YRhTUeHdsHPAG4zGG7VFopFcdY6b+flPyx854ocb4bKiBQ7FuZVQjz53Hu9+RsTPeeuHgdnlRc9NSFhpMGamuQkj6azK7lqptf85HNdfadwSIgRUdhlUNO5LAtMInlesqHyXOBXF6ZTWNSuYw\\\\\\\/JNfp8Xt5q\\\\\\\/m25\\\\\\\/KdeybBOpFMUl\\\\\\\/dw6jHowQoIP47+Fu9qIeNTuM+q2BGC2u2pvEU\\\\\\\/VBU26TWhh1vgOC54JPoQb+7u95zYcFmgqd1Kx9hU1S2EWxjPN4mIl1tbQIjFrh06NL0XJHWM3xDLpAQfiWN7cTInV5gBnctKIPEQJqQkitRpYY4fIKn83hUKgR+2ADgHxYavMVdiZau4UAP09BERbBVCxHsAhsLl8y47vQDM2EF9byTn273ezKBRcfbTLOD6vawN6vMxgsirSHbYrmuZXzAih6vcJQ69+WJ8BXjS0jP5\\\\\\\/1YM33JcOMkoYQjR3ISq8tAZSubHeh2dqndGqZSPB7KdgyTfMLSAu3tyrBujlgxSfKFlzoJsR7NsttcZDRxSn9oG7BOwWDswNbTphYVU9uO7dBWSVmvR5uN1vwfpd+swvRfSxa7RR2\\\\\\\/ckk0jQMCN4ou2txzvwY5J78AXiyEhuOoMbsaOC2AQphllWW4wZcdueZ\\\\\\\/rLa+SDzMmoHJPnd8YM9A583YS0moo8IF5WkTDc93KtbQ+uQadrLoSBFx4ytZeZogLkLw8UgvHZwApGmHhUKY75A9xokOEJqPIE368fXpORcOF9ZWtpLo\\\\\\\/S6c7H7oppXBAreCxDPSRbxtw65x+BapGicgd\\\\\\\/7scfap+Fh6baBoPPdRxyhTk1xN+LGVvQ5UKRW6HMSR2btqE8x5GQRkdC3ugFqXsTPuasPtGw1HkpHLKmEMWyDH82WXcTtB4NieNapMWH\\\\\\\/YbJLzwbwo3t\\\\\\\/bVht4S\\\\\\\/oANcIDvxPtjdSNkHS\\\\\\\/MZUeNhKxkylZ51lUeqHZdUhehIyPMgxo7OgDi1\\\\\\\/sQfBJDNxcct0fa9w908abBEwaUcplWOaLsG30ZxNOOi5Pxtx80\\\\\\\/lJ6Va1Sz8BILpZ03Apg0\\\\\\\/Z8vAi9+uMOUk3UiWAzuH0UljZwk1bdHP8oBvMtapujPoOUFuu\\\\\\\/9jTPRxBxBl5EDETjpqTKbxYoLqX+W8FIA\\\\\\\/CkWKFVyk8E7bBPG4XoDgmhr4NSzvZE2t6vKhmBHeXGYXkwza4l3wOcEvwbAaEyQKO6Z3q5CmAM29Tn7PDdooUg0nrrUje47OfPmd+HowJfNsU78pNc2gSgBWdV5c2aAlBDS4rnGWWATOxhYVSu1dZCj6qlqT4ObgxBYsmyzzhI4W+hybTedYOxVJ9NxJtIA9qES5jP3bCrzTx\\\\\\\/+eSInHahGiAlR1lVyfd\\\\\\\/eNsr+oUCZX5ohz+vW8pmp5SDuRCStBkX5sPvT4wSpcy1IXBXLdN9xY5GCve7BYdjjYQ6vkVbefThqIAWtgdnkvHR+7KEN5diTvDHnb0CaWUd7GUHGPCpf6JxG3AkQDDcmha7XnsLMB\\\\\\\/elPq2IJkSIGl7QwAUL8tiySaqEJ6Aq5FlpHDJAB31APeiGkk5DaO3+nriq4zXpCUXLnI5AEYXuJnc+jDI185G\\\\\\\/uknrk1MONwk+9vAUpDYwOit35syKqGz0ZFDyuglVJ01+wVLHmChLxC4kgFosgKIyvg6Bfr7DqXitYwHUhrYDKPdTS4RTCd26tdFLtZtAdff0Ik417y6NIjtOOvdg1gR\\\\\\\/r7jTw\\\\\\\/Sb0HgVnQ8BoLL0OAWKtYvnKXOK7g7i1xGwaBZiutnlZL+Dg3KbIVvdRdXf2XcctIM6t9yPN+tTigdwojrmRHlWmImA8zpUBVMlNR+rCuyzAAqMH6+hd7Yp7vT0Zx9bAXmW\\\\\\\/MomVqWIiZjONPwfom6dP5mpS6qCGjxisnGJmh8XVmb1EA7G4h77IJ85mP5\\\\\\\/\\\\\\\/1NtoFIoOpJgaTyqGBcFP9J8fFHFME4Tt4LjXSVzOShaD\\\\\\\/4nsZGPwRVtMd9ldBgRB5J3q76xdqDiDa7ADf8Ds984fjIRc3fJ9KrsNk113hev40S4k+J8p2OHyaL4pd7roPDqoK0Pg35vktUzlEuVSlg0ErYBJ5Qi3+YsMh8kNTBX71Ip7Zse5gkpyvWDUjsI7PUoFKBXLfRvs5SpZG1n2d3umA3E\\\\\\\/HaaFgmjkwPU2IAvFnhP8c0U\\\\\\\/Xzd98XkooA4GCTtoeQXioNrxFEIiLgWkui+xmbPCaqIY1Q9scfUCBp5aH+h8pBhwAe\\\\\\\/JWRr\\\\\\\/DyYb4v4lUjwVXd0nIoT7v7\\\\\\\/a9T6QjRsHlM9ueDL\\\\\\\/MSNzzYiuMuhSR7GjKmEOhCtr4Q4KQc7cqGukxFAGE4X04juaYRInQxnlvRG7gjSLhmqHyRWnf4praFUIx4Qs\\\\\\\/SM\\\\\\\/wvRI+QAupLE4S6dupROjB6mfDMlrIyVGYGfKuwp9TefvGE\\\\\\\/jvXsOluo7GCijrec9vs3mHl2p3hOO0NmGODOnYTx\\\\\\\/sk0bFI7ruNtCzDXjqHKpfww4QpxI\\\\\\\/ZTowVZ4nxQbaB+WAkt8tB6Cb6J7Jxj29c\\\\\\\/iMyzWtMmQFmVJEn+31rjrIFj\\\\\\\/3OFkXu3Rh56VpHS6HEyoFPH2HJP\\\\\\\/peXg4K+5YGtvL\\\\\\\/4MICrnDcsAQWOOWFFed6q59QXEs1HkCIq+jai6L3IjG+VoDCJL4scECdtv\\\\\\\/4hW2eG3tpFl0IVnIBVE1lgBYrDYohyfX0sOMlCyd7UB1imV5J7pP4F60s02DviIbsrgFAexWjWDx8SyTDKM2YAntIMmHqSwdfempRWsckK1LknOp\\\\\\\/f4nzvtALXdERZxiob7MljYZEtP5KZUyqRWPs08NbKmFfrf5fYNcSxoyKQPcNv3O837N3L\\\\\\\/BWk2h0PJKD9EOlR\\\\\\\/WcwE09l2MlFZ1EFPac+sIZG04oO1jHRxRuPtuotavU6LfWoq5ya9ich2J8OKbZPLbiGnU\\\\\\\/+mouHZYkU4FXx6bXiT88Yo6IwYrjsEmpYS0S+5+DULY\\\\\\\/lBZh1+y6hTRz5yEEjhGPUs9Al1qynwoHohOhBLK2e1lSnvnh5TEiMXa8rcPzTgtKCXY5Am4dzOz+mi1R41gyWN\\\\\\\/qmHMcTRq+izNLmspeSmIkKsgEu4ZM6PP0XKQI+bBHCmiMEe0Ehlq5cs4gFEyoA4EQFpK\\\\\\\/Y9GPsrTNTF913YcMuS497m9smFJsPujfntrZI3PmajR6IMEsNVqZu84p0xVOuJCooTXqrfLUuSQyb6VXSsMbtZyb4ZHK+dcyI2CJdb56qjWHbhoL8TUo0dxlo1IGfeD0YbA3xC0hrbaxYpv3nGcbgbXVREHQYcs9k1+3MAQImaiTKJd5kHYRPvACz5UgMST8XqnnKGiZJ4d3XGFbZFUERzHDByUbvYUGGo8FnE10aByF4yFL4h3JpbbN80\\\\\\\/6yDeeRSXSV+ZpeH5SP27BzEyX1dSCtKrNw5b2JD0l1lXgqXgYKeXLRMejIQnBN54HjAGPn+47QaatL7fPKwCoKkRK6JYVxilZ4XSM06VyNfX+UowScf0HFOtndp15O+WXK5wvAXnTTRrZVl9novJgZt0heYlI3fOOHa4AkdsBsLvlRU7tXGZgLc6WXv4nrHsZ7+O2\\\\\\\/pAvrtNM0oEnv8SgF45g8TKPSidvZfvbvXrh\\\\\\\/QbtiWpsiiiw1aLO5p8oA\\\\\\\/p00dtA5l7sT71ke1MUnYuf72ZjS085Ba9+AASfFFMBh4PpCpF4FRBlMws3YNT4kvaj2IvJN1C1uDir3fdNqOal9LOPl0IrzZzKFu4lJEXE+\\\\\\\/RKRGNEmQqacLGs2LI1jE2AaVwf105CHCx7M+6ZJo0wXJeSR5bH0Ge0UmYWcU3h9kswIbhUV6XQTcS2JEm6zX0+0WXzmTKxUCHb4LmVHDMhzJhe5mWW7WGa6WC19p8ExMu7vJT5eRlqYvIfWEfpqj9LyX35g58qPf\\\\\\\/g9+kdLIDEatPXVauorEocN3G90thDTLZY5aQ68Ng9n8yKvOBwWl15K6qqvHBO4AIjFQ18c\\\\\\\/FrHNGaf8yH8G8lAoCG5P97WIpC7wh+YtAnqGc0Pb0ZTo5OBgRmG5VtQvXJ5jY81\\\\\\\/JojNAV5XFJ2J3A7vxfeCY1goO9rWw6w6rLvajEaKNGy01A1m7srCwWWMy1FXYJGqsgdQCUZUbPR3XsdZ8U1zS1aQShcCZ6Vtk4q5Ah\\\\\\\/EV3TfyeO6wy2nJGYZ0PgcJhhmsxuW8aMincdwt83ooKPvxNwJGq9db1AQKuHBLFUAwzINtHZFzVLxDFLjKxHJwuXPp2gkMf2KVBMtSZLDlhBDqg1vdJlWThFUKJlMU8FohJxaVDjRJyXVCqu9UXqeUC0RN5eivG3zBvAxJpQIUsiADaqO\\\\\\\/GHbtUdlwkZyK5OMYl0jekeDPBe+SdaYMxw1sSf5eijEvK2zS3IDB4ZwWEH3d4QgdyzmaYHje8PRdb4DthOEV90l64FxPMim2sS4\\\\\\\/co0OL21zPMXoaxM2rcwyQng\\\\\\\/uu3Spke5PJwSKIHLUDNxhXgy6ehuX\\\\\\\/qxelPJ3DQ1DlEV2P+Ln8CgPvsgRJortV9mh08CiJI6nUx7LDT6vn6mYix73dYyHCxmLkKDqXuJZaijsFBqcWC0sQlRjftH1slX7bAGPiPlwEG41VVLiAaO8JcEDcCvS7RaUVuBN6\\\\\\\/fJvyw3WupQU2ecN5mp9fziQkUBhqkamKuWtrIF4TUHNX\\\\\\\/heW1FKGj1dG91VXV79+HZ2zcTSxmh1p0J4I2ICoN1\\\\\\\/7j1c84w3zD77ZfM3gNP\\\\\\\/XES4DVcOhXsiyc+9oduR1WkdSxUSuPPjyc1gbD7bmzY3K92QyR4uv2pWJ+wIfGGiJb1G53Rh5RflyvHjtT5KRLQTyHuW9fWq5cL0XEE4z2S6YMJvN+X0Xq6KKjlwR0U55pfaFLnWYyOqZO\\\\\\\/\\\\\\\/Xx5r4e5+zr91DSCKuIs0zihRdRXqwnc1a9A28FbSFsf3L6C45J3ESVK\\\\\\\/AyxmU8I1aL9fx+CZiFYXAiE5e1WtnE9ZFmEomNfdedsr6QbPMuZCVOmPBwOn5yM4jKIFX6SVZrIvLcCk27IlLyVgZGoCvIQYPeCXAwpnIY7+kpLf\\\\\\\/pQLdT2L69HT+iPTCuQ5bcoYwvAToLXaKPu21FziSBcX0ORo5Am01gVrLVrDUuCqa3Bw7PfcZ+5SZjzs1NCd1md5Q8pTwDa7AoG5DZEO4D6mOVNb3jVFrr37mbdFyNVqPeziYO\\\\\\\/kzFmPcY+io43hzxixnezkvWZvN8Z+bl27c9WM0FMmjngn4rZoZnUUBKLQJx8r2YLESeOcszR20G3zPZbYIp7mZMIjPx1NnlhWYJmCJWKUFOAk39jGfhPd3E9\\\\\\\/5CkC\\\\\\\/DcoRMZjdVG6KPg1ZIsAKF3pscPYJCcA\\\\\\\/HCW4ZlMJYY427SNXbzlIGnwkze60QJoGrbHEGyhDra6ywZfmPUjrs2LZhp3q4sRFmz8xcp9bIPxSWBEC3l2wKLtT48op6q8jcg+kWK2JYxwagiN0HOlF\\\\\\\/GUoizivI4asFvuhC8nKXfh5ZEKRCMpA6VtavvbIOfpssefirJwVcE5zm0lC5nnICZNNxx8Ch1PtHBgXYfxEmi5XSRhpvdkZ47qFP8b\\\\\\\/tGA9Qxd7qDh0f8m5fl\\\\\\\/OKugddWPGpX79tc38h7kZ1iaWgI0Nol8Umsi78\\\\\\\/fmqNhw8wSZwOYnRjKc2fl2+XfHuPlm7AUE+S8Z0o+4ICOYArCabKakckxRzrw5RomMlUX1tutn6MQfAI+MIGg6puornXloXHQr93vpkPIv5YK529493NI35xdabKNz1kkPUcYD9dMAiO36KepyvhWKC8Xtp1EyxZZrzWba48JGxPxU\\\\\\\/a2pltjfzVcPL7laY0riKMZBxRd\\\\\\\/oluGfnFsDXcpSDXLuDHwoBRFBXEENIOrEI0rtn1VJ3ujwuuvFTFipOfI8Ll2KHlKoDEuOchvdBWpX2SZ0\\\\\\\/Kvv+W26\\\\\\\/I2I\\\\\\\/egmaKEUvSU0Qpey8YewoQXaZA3vhaQtn4D69wW38qqNQudKmIk+Q\\\\\\\/J+3cwqZ7q66xMX5tpXQZaboz+nCyrGtLfvD3RKl2PYkpLpqPEEYEyANa9fTV3p\\\\\\\/sfsbBgvnaRQ4lUaqxL4vQivxVE+nAbANp54cJEhF42Ll7fAecdjgzD4eKR5oSbsy2DBDujgzyP0NwszUrSTz790gzoBEUh+8t2k+oUkrC2\\\\\\\/+R8LPZNx8iLVuKsyvoqGJufQsHfHgeyCNudzWGfggsDlqlqVbPcLHW3CJavAVZM6GkQl4BD2U4MWstDpalp1L9fkAkScV8TqN9UqIAir2aj\\\\\\\/Kup6DEZ7FktArStFFdAyGY0w+ny74v1eAEFNHUVD9ROjmmR65F0szo4wf7HKcZ9HYKE6\\\\\\\/Kqw0lfyALaaFTBvGLzTBU43GfNTN\\\\\\\/0Dg+KiSAwJHht1cNW3vnQKb3VqrdTObFPJtpITWa8YV1QznEsMGZ2\\\\\\\/wCcqVpiEAXcorKNOZYQqTAYmQWwRhshpksWY6JnQHPVpuFj57Mf4J6qoJvLGXsmAoydFN+YnUKb3OgW22xVbPSaDy8gohWDd2K0Ckgl\\\\\\\/bgboYwv4gwIkoqKxiAP1oe8opVTKIf3lnSzsP8XtCmCvXYiJkorw9YXFGtI4k9lAqCPU51tufYwx6+AK1EHl9LMvYuoCMBIrPlRm+3cXcNw9RC3PuNXpj2XMH3ozbRB\\\\\\\/892z9pXzUFznSUfHPFAmDCoEn6xYB0l1Qg5YeaDrGjXtzZU0YiLVxZf9BiRtDcwN\\\\\\\/\\\\\\\/IQ7OFhIrce\\\\\\\/uEa0Y\\\\\\\/xWVImsd++t6txXHRxNmiftQa0\\\\\\\/Q5NWj2zZORyrlMABV8nwB1y4iw9izkt+SvNQzqJoAgZF87FsvDycsib0yJ1jQviM\\\\\\\/gDhHwBjW7YzrP3s1i68x2QixthDXKUF9QfS7Q+EEudxJrP3\\\\\\\/GlbtjaoCK2uQBFQxLzB7\\\\\\\/Qr4vVZeheeCh\\\\\\\/OBGpmQ6a9BBzVMKa7ft7\\\\\\\/dEwacqb2gHx6mLnNy5U3YQG+6cfYBZJQWsTUqaPy9AudiekGX+K3tckj6C6zLa\\\\\\\/enjhlgYEF6VDCLTpMPZHWOtEddfQEq+Ig\\\\\\\/yaC9ESa5kN7I40wb\\\\\\\/sy+T1tOfYp\\\\\\\/V0tnHQ\\\\\\\/\\\\\\\/E3O5hfx3CXJiufdkrS6Xit7xxcqKt1dXeOb1J7\\\\\\\/w0Gbf6Lqb8U2tIjyw\\\\\\\/VPk1vIFHZeu7DCP3ADLGbg5mLViKb1mHhTYiz\\\\\\\/KE4jz\\\\\\\/c\\\\\\\/9yKJuhfT6sX7h6taHLSv\\\\\\\/44QcGO8nxW5pkPFw2I+0CqtvyC0WmP8Xrvdau85b+vyNpxBSY10vt9VgIlM3VbEwwVYHnO3ABh2pj21mbfbASrg8K+qcYSn95Q1qcuTghnPPOovddvqdou2jUrg9e3E8FNHC\\\\\\\/pow4f5pc5eRLTCQz\\\\\\\/gvYHd\\\\\\\/GJNJ9lrVg\\\\\\\/tWKs2iKN4ODAg3DNPsC4nDlHZepJrXL987Q3m3ZqbcX2bStaEPJIrKD9en7pLbPS32kaXaw6N4kiOgzaAjpjS1wA\\\\\\\/zts0Pz1Jo\\\\\\\/3VFs4SMl9U7u6eJFyc90huH+OAlZahA5sYGFf1yN1mluWbNZyU97IBG6kMD4k1W6FAeZjCb3diUQ+kWGVkOrBOuZg8ISp3UoDXvaSuV4+WQM3dL\\\\\\\/v9deRUwFQmYGccQuozPBwnUiK3rZAZEPRm7NPVPU9Zq7LjuHuhILdWaMbqyxW5KTISW1H2kQEiBzjl6LzytsaSfvv2B6\\\\\\\/Qeo0PNPhbtRrhkZGtnM2472aD4xDNuDdxzHBn2iqXiCklVIRPWupE1O6v\\\\\\\/vyEsbYVPjvAsGaCKGUUG1Ppm9XMoD+7izJoBeSGvIoIWEPsp3hyKk1OJyw4DlGKnGZJC+lR0g71BwyrNAETty6P07EqUXXYGwnxxZD0oEx12BVKKbtuQBwCOxKiUNe0quTJsZQ6JxGmAi7Y1mTJwCO3J5We3xrqe+ejS2Pd7TZlbARR9QticOYX+eG\\\\\\\/GRf+MBkO53S557GJu3fooF0+qbGvH3VWWUBzyxTyjoervaav5zTWGHgOkkdbfHvcLxyT14UddXsw6aw==\\\u0022,\\\u0022iv\\\u0022:\\\u00223\\\\\\\/lPw\\\\\\\/g6OQT9mkzNMhUKSQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221e144d667a4871f473ab1af31690da04d24b7b17c47fafd513c81eca9d3ba4e3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022T1NKYBMjVJ\\\\\\\/9QbiHMvDNV7tvYAqUzRYZcR8T\\\\\\\/4i3hOZmjCT85Xc5ReSvJgfMqOxTi1GCor97WbglkqqpYZyNQcXvqIXxEabFgHo7IsxRaeIpc0WYwbgpuS8iK8n2R9tcUbU3pZv0NRrvMvolY2\\\\\\\/JpNwAYcA63wvsaKSHe2OVYoNmr\\\\\\\/vMs3ePOOb30gmHR93A3\\\\\\\/pIZpvEII+mppyXtMCDghOF05kwCVt530qHk5sK\\\\\\\/gbUTBZXAcxg3V+\\\\\\\/MskLHkLH0roHOO3JprbZRW\\\\\\\/oQ7qRNOR1drFXjzqvCZaL7DFZvmpC3r4hpyR3cpBfqUjqO5YYU7Gsl\\\\\\\/4B7kzo\\\\\\\/fu8Tb1V0+vRLoM2ZK7XznPefwLwqmdVk4asC\\\\\\\/8Hz2RPm\\\\\\\/n40rzlmf4hjJr7urtp\\\\\\\/y6i00bNbSasAUVG\\\\\\\/q44wPx0RHDlcVemaYHlN5hMKkDBq1x83BSSLiQvqbWueA4h5HHVwiGNsRZjOxdqifDMLlVPWmy9W1BtOCvcG\\\\\\\/dmbWLuyPXUYXCAVcrGTdv425oJddlOMYMns3OzBc7vMK7Bf7xiVWMkavfJ\\\\\\\/lMXuNcd8uH4d+5Lrgyl7YBkITCUW6ERcbubzc4oQBQELOIvinSTAin7ImAjNlGuNCrNWWO+h8f5TRRcVJyBT\\\\\\\/t1pqVxreZkfI9pgwgE+u4jCviNn7ElSBGZoH0\\\\\\\/dsJMzjdciv1gx3YLdlvp2HxmzTV5jGWRsgZ6pDUlskn3VA9\\\\\\\/ovSFK69LMO9bt7DIUdT5UaUKpA7O1GwLDMaI2fiC20d7v8DUPwwt1S4Ie0c\\\\\\\/5x3EoZH+7N3rbK7BFU+PLnEwknxCHu+zjcYlD+O0Ach2Nw\\\\\\\/7WzdEh6pZNUWfUq5py0vYjzWfIbxtuo4LeSEbdFJ9dJ0HdTF4I3yDh4Uvc3yfIh2FMVBFJRAfk\\\\\\\/NPy5Kfr0Ne2EypxH5pTp3w1\\\\\\\/rvM1p4hzTNJ3z8LSMQNr8SU9VVp2wKX0fELg80VX2pn1W2MTRKeoITID1XQI\\\\\\\/jdjz0CpGOTEhde0CTAgB5NS6R2PYglQGkuQUrpdxzlanN+f90J1LMzM0jNurv3y5gK2CDTZ3aObi0CGW+CI2bGw5XCikPE+QDDXkEEYqGm9ZOwj5g+C\\\\\\\/0kfv9CtbVCzBCzJXABWHHmHvcZWZey6Km6Vu0pYJqSxvSsNdTCrbGv12btMxHHZWQ7bHKCiBUIAakN+XFh4mSGsI0ix111BqEOJFELIjCLETskZsXFsJGktG7Bo4VgQIKHcEAqAv0ASie44KdZ1NF9LUBGslcuw08DZMSnquUOu+Vg0p50lDEmohltQWbiC9pp2VupHSZtII7uy9RuhEZB\\\\\\\/aKOF385XDIpzVB\\\\\\\/vLntafdKCTVk+cYWzpWr\\\\\\\/IEUrXAPXO2QUcdjSOilaaTMd7Wf8DVDQ7AlSFTBuGiCIciBQ3a4m9xpBHy8ByWxBrR6ks1g4IufNQPEXUaBwocjKS+i5l5+ZJgsb3qCs78O\\\\\\\/QCm\\\\\\\/KF7I0oOwuzHBMc3u5S6IUJd95WKwz+5Kn2BFRANJSWM7S9cZGWU9XyR2j6Ury0h0k2QEOswWsNrcL4sI5gIOWVsdM\\\\\\\/LDRLj+V5u1Z1Q5UsjXNm\\\\\\\/MxHX5EsT3MTESX69aYbrTr2p\\\\\\\/HKUHIOEdr9lbYDqg6y32Ik7ZCtEb5wMUPZcXdSzAEyg2Itdp48DVYFWYgfp\\\\\\\/kBMOmjAlEkglZamgX1ui2ZDzBwydVFrXF3HNDXWHCcWFTNSRmP+nTijPr74acGdAxnOxOJdF+TlFOhCCUA\\\\\\\/rQM6ps3Ee2hTkGvshATGNgQVkpl6Jr1QzycoY\\\\\\\/+cGjRsa4fnlDTtPZ7U7zc5W\\\\\\\/cq3voqaow5NTTXPFnbwoUiWnGNDsaaEMPFUfeTI1GC1dmbFF5l3QR7xb3xSoJfgQSov7ExrXIGPKqwJI+9as6ZLCQxQMholezCLDEFHabEpizx2YkIvaCXEHgECi2yB4IIN+1GF+sxbZLbrLx9thUSxIHJg2Z7wp\\\\\\\/ReO\\\\\\\/sQ89SphNdnwnGbqW+zVQbNaAsA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022NqQLbqmsv8yhu4a3sVLGtg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a97a8f17c8ff3c16feadd3f2ecfd98766df5cb49029c1c971260efdd7ce864c5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493104 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1519" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wTvbVixNv2GkMlkGFPsUEoDueyEh7CNxtnQNSTUjD7E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1519" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YMCpVuzlFShseM0iupLTmtes9Y3sDf_LTEXZu5GBqbI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1507" title="Small RNA-Seq Analysis Reveals miRNA Expression Dynamics Across Tissues in the Malaria Vector, Anopheles gambiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1533" title="Variation in Filamentous Growth and Response to Quorum-Sensing Compounds in Environmental Isolates of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493104" id="node493104" data-pisa="ggg;9/5/1519" data-pisa-master="ggg;g3.119.400064" data-apath="/ggg/9/5/1519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (<em>Oryza sativa</em> L.) Grown in Subtropical Areas</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Eliana Monteverde</span>, <span class="highwire-citation-author" data-delta="1">Lucía Gutierrez</span>, <span class="highwire-citation-author" data-delta="2">Pedro Blanco</span>, <span class="highwire-citation-author" data-delta="3">Fernando Pérez de Vida</span>, <span class="highwire-citation-author" data-delta="4">Juan E. Rosas</span>, <span class="highwire-citation-author" data-delta="5">Victoria Bonnecarrère</span>, <span class="highwire-citation-author" data-delta="6">Gastón Quero</span> and <span class="highwire-citation-author" data-delta="7">Susan McCouch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1519-1531;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400064 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493104" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Eliana Monteverde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eliana%2BMonteverde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monteverde%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEliana%2BMonteverde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lucía Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, University of Wisconsin - Madison WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%C3%ADa%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuc%25C3%25ADa%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pedro Blanco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BBlanco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanco%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BBlanco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando Pérez de Vida </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BP%C3%A9rez%2Bde%2BVida%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20de%20Vida%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BP%25C3%25A9rez%2Bde%2BVida%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan E. Rosas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Unidad de Biotecnología, Instituto Nacional de Investigación Agropecuaria (INIA), Estación Experimental Wilson Ferreira Aldunate 90200, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BE.%2BRosas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosas%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BE.%2BRosas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Victoria Bonnecarrère </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Unidad de Biotecnología, Instituto Nacional de Investigación Agropecuaria (INIA), Estación Experimental Wilson Ferreira Aldunate 90200, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BBonnecarr%C3%A8re%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonnecarr%C3%A8re%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictoria%2BBonnecarr%25C3%25A8re%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gastón Quero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Biology, College of Agriculture, Universidad de la República, Montevideo, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gast%C3%B3n%2BQuero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quero%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGast%25C3%25B3n%2BQuero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Susan McCouch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BMcCouch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCouch%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BMcCouch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srm4@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">srm4@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/5/1519" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493104" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1519.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493104" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/5/1519.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493104" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/5/1519.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493104" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493104/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Understanding the genetic and environmental basis of genotype × environment interaction (G×E) is of fundamental importance in plant breeding. If we consider G×E in the context of genotype × year interactions (G×Y), predicting which lines will have stable and superior performance across years is an important challenge for breeders. A better understanding of the factors that contribute to the overall grain yield and quality of rice (<em>Oryza sativa</em> L.) will lay the foundation for developing new breeding and selection strategies for combining high quality, with high yield. In this study, we used molecular marker data and environmental covariates (EC) simultaneously to predict rice yield, milling quality traits and plant height in untested environments (years), using both reaction norm models and partial least squares (PLS), in two rice breeding populations (<em>indica</em> and <em>tropical japonica</em>). We also sought to explain G×E by differential quantitative trait loci (QTL) expression in relation to EC. Our results showed that PLS models trained with both molecular markers and EC gave better prediction accuracies than reaction norm models when predicting future years. We also detected milling quality QTL that showed a differential expression conditional on humidity and solar radiation, providing insight for the main environmental factors affecting milling quality in subtropical and temperate rice growing areas.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rice" class="hw-term hw-article-keyword hw-article-keyword-rice" rel="nofollow">rice</a></li><li class="kwd"><a href="/keyword/genotype-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-genotype-by-environment-interaction" rel="nofollow">genotype-by-environment interaction</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/qtl-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-qtl-by-environment-interaction" rel="nofollow">QTL by environment interaction</a></li><li class="kwd"><a href="/keyword/environmental-covariates" class="hw-term hw-article-keyword hw-article-keyword-environmental-covariates" rel="nofollow">environmental covariates</a></li></ul><p id="p-3">Genetic by environment interaction (G×E) could be expressed as a difference in the relative response of genotypes across diverse environments. When we consider a set of genotypes exposed to different environments, their performance will differ depending on the interaction of genetic properties with the different environmental conditions, leading to differences in variances and rank changes among genotypes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cooper and DeLacy 1994</a>). These rank changes represent a very important challenge for breeders due to the difficulties of selecting genotypes with stable performance over diverse environments.</p><p id="p-4">Environments can be different both in time and space. For this reason, the concept of G×E embraces both interactions that take place between genotypes and a particular location (genotype by location interaction), and between genotypes and particular years (genotype by year interaction). Genotype by location interactions are usually determined by soil and climate conditions, while genotype by year interactions are characterized by plot-to-plot variability and weather conditions (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Malosetti <em>et al.</em> 2016</a>).</p><p id="p-5">Several statistical approaches have been proposed to describe G×E in the context of classical plant breeding. The classic parametric approaches used to evaluate G×E are based on linear regression and ANOVA techniques. Linear regression analysis (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Yates and Cochran 1938</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Finlay and Wilkinson 1963</a>) measures individual genotype performance over environmental means. Multiplicative models combine univariate and multivariate approaches for reducing data dimensionality and facilitate the interpretation of results (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gauch 1988</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Smith <em>et al.</em> 2005</a>). The most commonly used multiplicative model is the additive main effect and multiplicative interaction model (AMMI; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gauch 1992</a>). AMMI combines univariate (ANOVA) and multivariate (singular value decomposition; SVD) techniques for estimating genotype and environment main effects, and G×E effects, respectively. Factorial regression models are another type of models that allow the modeling of genotype sensitivity to specific environmental covariates (EC) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">van Eeuwijk <em>et al.</em> 1996</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Vargas <em>et al.</em> 1998</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Malosetti <em>et al.</em> 2004</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Malvar <em>et al.</em> 2005</a>). Linear mixed-models became very popular for the analysis of G×E since they allow different correlation structures among environments (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Burgueño <em>et al.</em> 2007</a>). These covariance structures may range from a compound symmetry form, where homogeneous variance and homogeneous covariance between environments are assumed, to an unstructured form where a covariance parameter is assumed between each pair of environments and environments are assumed to have heterogeneous variances.</p><p id="p-6">Recent developments in sequencing technologies and statistical modeling have made it possible to use dense genotypic information to predict phenotypic responses through genomic prediction (GP). This idea was introduced by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Meuwissen <em>et al.</em> (2001)</a>, and provides an alternative approach to indirect selection in crop breeding. GP models were originally developed for traits evaluated in single environments, but more recently standard GP models have been extended to account for G×E. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Burgueño <em>et al.</em> (2012)</a> were the first to extend genomic best linear unbiased prediction (GBLUP) to a multi-environment context, by combining genetic and environmental covariance matrices and using different covariance structures to model the environmental component. Lopez Cruz <em>et al.</em> (2015) proposed a marker by environment approach where marker effects and genotypic values are partitioned into main effects across environments (stability) effects that are specific to each environment (interactions).</p><p id="p-7">Standard GP models can be modified to accommodate climate information in the form of EC. However, including EC in the analysis can pose some similar constraints encountered when predicting breeding values with multiple markers. As climatic and agronomic systems develop, a very high number of covariates can potentially be obtained increasing the dimensionality of the data and also the possibility of being correlated with each other. Several studies have proposed different ways to deal with highly dimensional data, showing that the incorporation of explicit environmental and genetic information can improve prediction accuracies and predict performance in untested environments (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Heslot <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Malosetti <em>et al.</em> 2016</a>). <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jarquín <em>et al.</em> (2014)</a> proposed a Bayesian reaction norm model where the main genetic and environmental effects were modeled using covariate structures as functions of molecular markers and EC respectively, and the interaction effects between markers and EC were modeled using a multiplicative operator. <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Heslot <em>et al.</em> (2014)</a> proposed a factorial regression model, where instead of using all the available EC and molecular markers, they chose the EC that most significantly influenced the growth and development of the crop by using crop growth models (CGM). These variables were introduced in the factorial regression model along with those markers that showed the most variable effects across environments and reducing thus, dimensionality of both markers and EC.</p><p id="p-8">The partial least square regression (PLS) (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wold <em>et al.</em> 2001</a>) is a generalization of multiple linear regression (MLR). PLS is a dimension reduction approach that can accommodate a large number of correlated genetic and environmental variables simultaneously, by finding one or few factors named latent variables (LV) that explain both the variance of the X matrix (containing predictor variables) and the covariance between matrices X and Y (containing response variables). PLS can be used for variable selection, in order to improve estimation/prediction performance, but also to improve model interpretation and understanding of the system studied. Another advantage of PLS is that it can be more robust against multicollinearity (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aastveit and Martens 1986</a>). PLS models have previously been used for GP both in plant and animal breeding (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Solberg <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Long <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Colombani <em>et al.</em> 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Iwata <em>et al.</em> 2015</a>), to detect highly influential environmental and marker covariates that explain a significant proportion of the total G×E (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Vargas <em>et al.</em> 1998</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Crossa <em>et al.</em> 1999</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Vargas <em>et al.</em> 1999</a>).</p><p id="p-9">Understanding the genetic basis of G×E is also necessary to gain predictive capability, and one way to do this is detecting QTL with varying effects across different environmental conditions, or QTL by environment interaction (QTL×E). Methods usually employed to detect QTL×E have been very useful to detect QTL with differential expression across environments, but provide no explanation of the underlying environmental factors involved. When weather data are available, factorial regression models can be used to determine the extent of influence of these factors on QTL×E (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Crossa <em>et al.</em> 1999</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Campbell <em>et al.</em> 2004</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Malosetti <em>et al.</em> 2004</a>).</p><p id="p-10">Rice is one of the world’s most important staple food crops, constituting over 21% of the caloric intake of the world’s population and up to 76% of the caloric needs in many Asian countries (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fitzgerald <em>et al.</em> 2009</a>). World markets dictate the value of rice mainly based on milling quality traits, so breeding for both high yield and quality is a major breeding objective for rice exporting countries like Uruguay.</p><p id="p-11">In a previous study we showed that accounting for heterogeneous covariance parameters between pairs of environments can be beneficial for predicting yield and milling quality performance in Uruguayan rice for untested environments (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Monteverde <em>et al.</em> 2018</a>). In another study, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Quero <em>et al.</em> (2018)</a> found a set of QTL for milling yield traits in the same Uruguayan <em>indica</em> and <em>tropical japonica</em> populations. However, none of these studies tested the use of EC to both predict yield and milling quality traits in untested environments, and investigate QTL responses in specific environments. The main objectives of this study were to: 1) use molecular marker data and environmental covariates simultaneously to predict rice yield and milling quality traits in untested environments (years), and 2) Detect marker by environment covariate interactions that provide explanations of variable QTL effects across environments. Two rice breeding populations (<em>indica</em> and <em>tropical japonica</em>) were used in this study and were evaluated for grain yield, plant height and grain quality traits (head rice percentage and chalky grain percentage) across 3-5 years in Eastern Uruguay. Results from these two analyses provided clues about the main environmental variables that could be driving G×E in temperate rice-growing regions such as Uruguay.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Germplasm</h3><p id="p-12">The germplasm consists of two rice-breeding populations, an <em>indica</em> and a <em>tropical japonica</em> population belonging to the National Institute of Agricultural Research (INIA-Uruguay). Both populations were evaluated in a single location, Paso de la Laguna Experimental Station (UEPL), Treinta y Tres, Uruguay (33°15’S, 54°25’W) between 2009-2013.</p><p id="p-13">The <em>indica</em> population consisted of 327 elite breeding lines evaluated over three years (2010-2012), and the field design consisted in a randomized complete block design with two or three replications. Trait correlations, heritabilities, and genomic prediction accuracies for this dataset were computed in previous studies (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rosas <em>et al.</em> 2017</a>, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Monteverde <em>et al.</em> 2018</a>, <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Quero <em>et al.</em> 2018</a>). The <em>tropical japonica</em> population consisted of 320 elite breeding lines evaluated over five years (2009-2013). The number of accessions observed each year ranged from 93 to 319, as detailed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. This dataset was unbalanced with non-random missing data, since ∼50% genotypes were dropped from testing every year based on performance, and new genotypes were added over time. Each year, the genotypes were planted independently in replicated trials in six-row plots using an augmented randomized complete block design with two or three replications. Both <em>indica</em> and <em>tropical japonica</em> trials were conducted under irrigated conditions using appropriate pest and weed control.</p><p id="p-14">The agronomic traits of interest used in this study were Grain Yield (GY of paddy rice in kilograms per hectare) and Plant Height (PH measured in cm from the soil surface to the tip of the flag leaf). The grain quality traits measured were Percentage of Head Rice Recovery (PHR measured in grams, as the weight of whole milled kernels, using a 100g sample of rough rice), and the percentage of Chalky Grain (GC measured as % of chalky kernels in a subsample of 50 g of total milled rice, where the area of chalk -core, white back or white belly- was larger than half the kernel area based on visual inspection). More details about how grain quality traits were measured can be found in <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Quero <em>et al.</em> (2018)</a> and <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Monteverde <em>et al.</em> (2018)</a>.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic analysis</h3><p id="p-15">Phenotypic data for each trait were analyzed separately each year. The model used to calculate the best linear unbiased estimators (BLUEs) for each year was:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-2.gif"/></noscript></span></span> is the trait score, μ is the overall mean, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-3.gif"/></noscript></span></span> is the random effect of the i<sup>th</sup> block with <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-5.gif"/></noscript></span></span> is the block variance, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-6.gif"/></noscript></span></span> is the genotypic effect of the j<sup>th</sup> genotype, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-7.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-8.gif"/></noscript></span></span> are the random k<sup>th</sup> row and l<sup>th</sup> column effects nested within the i<sup>th</sup> block with <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-10.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-12.gif"/></noscript></span></span> are the row and column variances respectively, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-13.gif"/></noscript></span></span>is the model residual vector with <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-14.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-15.gif"/></noscript></span></span> is the error variance.</p><p id="p-16">Trait heritabilities in <em>tropical japonica</em> for years 2009 and 2013 (data not yet published) was calculated on a per line basis as <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-16.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-17.gif"/></noscript></span></span> is the variance among genotypes, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-18.gif"/></noscript></span></span> is the error variance, and <em>r</em> is the number of replicates.</p></div><div id="sec-4" class="subsection"><h3>Genotypic characterization</h3><p id="p-17">The lines were genotyped using genotyping-by-sequencing (GBS). SNP calling was performed using the TASSEL 3.0 GBS pipeline (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bradbury <em>et al.</em> 2007</a>), and SNPs were aligned to the Nipponbare reference genome MSU version 7.0 (<a href="http://rice.plantbiology.msu.edu/">http://rice.plantbiology.msu.edu/</a>) using Bowtie 2 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Langmead and Salzberg 2012</a>). Imputation of missing data were performed with the FILLIN algorithm implemented in TASSEL 5.0 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Swarts <em>et al.</em> 2014</a>) for both datasets separately. The GBS datasets were filtered to retain markers with &lt;50% missing data after imputation, and a minor allele frequency MAF &gt; 0.05, as reported by <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Quero <em>et al.</em> (2018)</a>, and <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Monteverde <em>et al.</em> (2018)</a>. The final <em>indica</em> and <em>tropical japonica</em> marker dataset consisted of 92,430 and 44,598 SNP markers respectively.</p></div><div id="sec-5" class="subsection"><h3>Derivation of EC From weather data</h3><p id="p-18">Daily weather data were obtained from GRAS unit from INIA (<a href="http://www.inia.uy/gras/Clima/Banco-datos-agroclimatico">http://www.inia.uy/gras/Clima/Banco-datos-agroclimatico</a>). The database contains weather data from 1965 to the last calendar month completed, for all 5 INIA experimental stations in Uruguay. The variables available were related to temperature, precipitation, solar radiation, humidity, wind, and evaporation.</p><p id="p-19">To compute the EC from daily weather data for each rice genotype, the plant development stage has to be determined in order to account for the differential effect that weather variables may have in different stages of crop development. This information is usually hard to obtain directly or, as in our case, not available. For this reason, the phenology of the crop was defined according to flowering time (days to 50% flowering, FT), which was measured for each line every year. With this measure, and sowing and harvest date, three main phenology stages were determined for each year, according to <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yoshida (1981)</a>: vegetative stage (of variable length, starting on sowing date), reproductive stage (starting 35 days before FT), and maturation stage (ending 30 days after FT).</p><p id="p-20">Once these phenological stages are defined for each year, EC can be computed from daily weather data. Covariates with zero variance were removed from the analysis. For prediction, both markers and EC were centered by subtracting the mean, and standardized to unit variance by dividing the centered values by the standard deviation of the marker or EC. A total of 54 EC were used in both populations (18 for each developmental stage: vegetative, reproductive, and maturation), and are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493153/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493153/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Environmental covariates used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>PLS regression</h3><p id="p-21">PLS regression was first introduced by <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wold (1966)</a>, and was originally developed for econometrics and chemometrics. It is a multivariate statistical technique that was designed to deal with the <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-19.gif"/></noscript></span></span> problem; <em>i.e.</em>, when the number of explanatory variables (p) is much larger (and more highly correlated) than the number of observations (n). A brief explanation of PLS relating one response variable <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-20.gif"/></noscript></span></span> to a set of explanatory variables <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-21.gif"/></noscript></span></span> is given below, but it can be extended to more than one response variable (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boulesteix and Strimmer 2006</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Wold <em>et al.</em> 2001</a>).</p><p id="p-22">In PLS, the data for p explanatory variables are given by the matrix <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-22.gif"/></noscript></span></span>, and data for the dependent variables are given by the response vector y. Each <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-23.gif"/></noscript></span></span> and y vectors have <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-24.gif"/></noscript></span></span> dimensions corresponding to the number of observations. In this work, the y vector contains all the observations for a given trait in different environments (years), and the columns of the X matrix are the variables corresponding to either markers only, or markers and EC. All variables in PLS must be centered and scaled.</p><p id="p-23">PLS is based on the latent variable (LV) decomposition:</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><p id="p-24">where T is a <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-27.gif"/></noscript></span></span> matrix giving the LV (also called scores) for the n observations, and P <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-28.gif"/></noscript></span></span> is a matrix of p-dimensional orthogonal vectors called X-loadings, q (<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-29.gif"/></noscript></span></span>) is a vector of scalars and, also named Y-loadings, E <sub>(<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-30.gif"/></noscript></span></span>)</sub> and f <sub>(<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-31.gif"/></noscript></span></span>)</sub> are a residual matrix and vector respectively.</p><p id="p-25">The LV matrix T that relates the X matrix to the vector y is calculated as:</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-32.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><p id="p-26">where W is a (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-33.gif"/></noscript></span></span>) matrix of weights. For a given matrix W, the LV obtained by forming corresponding linear transformations of the variables in X, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-34.gif"/></noscript></span></span> are denoted as <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-35.gif"/></noscript></span></span>:</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-36.gif"/></noscript></span></span><p id="p-27">These LV are then used for prediction in place of the original variables. After computing the T matrix, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-37.gif"/></noscript></span></span> is obtained as the least squares solution of <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation (2)</a>:</p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-38.gif"/></noscript></span></span><p id="p-28">The vector b of regression coefficients for the model <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-39.gif"/></noscript></span></span>, to predict new responses, is calculated as:</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-40.gif"/></noscript></span></span><p id="p-29">Since regression and dimension reduction are performed simultaneously, all b, T, W, P and q are part of the output. Both X and y are taken into account when calculating the LV in T. Moreover, they are defined so that the covariance between the LV and the response is maximized.</p><p id="p-30">In PLS, the optimal number of LV (c) must be determined. In this work, we used the root means squared error of prediction (RMSEP),</p><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-41.gif"/></noscript></span></span><p id="p-31">which was minimized with 10-fold cross-validation in the training data set and for each value of LV (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mevik and Cederkvist 2004</a>). In this study, two PLS models were fitted: the PLS-G model used marker covariates as predictors, and the PLS-GW model, which used both marker covariates and EC as predictors. PLS models calculations were performed with the R package “mixOmics” (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lê Cao <em>et al.</em> 2016</a>).</p></div><div id="sec-7" class="subsection"><h3>Genomic Best Linear Prediction (GBLUP) and reaction norm models</h3><p id="p-32">Mixed linear models were used as a baseline comparison of prediction accuracies with PLS models. The models used considered the random main effects of markers (G model), the random main effects of markers and EC (G+W model), and the random main effects of markers, EC, and the interactions between them (G+W+GW model).</p><p id="p-33">The G model constituted of a standard GBLUP model for the mean performance of genotypes within each set of environments, using the following model:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-42.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where μ is the overall mean, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-43.gif"/></noscript></span></span> is the genotypic random effect of the i<sup>th</sup> line expressed as a regression on marker covariates of the form: <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-44.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-45.gif"/></noscript></span></span> is the genotype of the i<sup>th</sup> line at the m<sup>th</sup> marker, and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-46.gif"/></noscript></span></span> is the effect of the m<sup>th</sup> marker. Marker effects are considered as IID draws from normal distributions of the form <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-47.gif"/></noscript></span></span>.</p><p id="p-34">The vector <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-48.gif"/></noscript></span></span> contains the genomic values of all the lines, and follows a multivariate normal density with null mean and covariance matrix <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-49.gif"/></noscript></span></span>, where G is a genomic relationship matrix whose entries are given by <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-50.gif"/></noscript></span></span><sub>.</sub></p><p id="p-35">As previously reported by <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Jarquín <em>et al.</em> (2014)</a>, it is possible to model the environmental effects with a random regression on the EC that describes the environmental conditions faced by each genotype, that is: <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-51.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-52.gif"/></noscript></span></span> is the value of the q<sup>th</sup> EC evaluated in the ij<sup>th</sup> environment × genotype combination, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-53.gif"/></noscript></span></span> is the main effect of the corresponding EC, and Q is the total number of EC. Again, we consider the effects of the EC as IID draws from normal densities, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-54.gif"/></noscript></span></span>. The vector <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-55.gif"/></noscript></span></span> follows a multivariate normal density with null mean and a covariance matrix proportional to Ω whose entries are computed the same way as those of the G matrix but using EC instead of markers. This covariance structure describes the similarity among environmental conditions. Then, the model becomes:</p><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-56.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span><p id="p-36">This model also includes a marker × EC interaction term, where the covariance of the interaction is modeled by the Hadamard product of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-57.gif"/></noscript></span></span> and Ω, denoted as <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-58.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-59.gif"/></noscript></span></span> is an incidence matrix for the vector of additive genetic effects. This model extends <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation (4)</a> as follows:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-60.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>with <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-61.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-62.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-63.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-64.gif"/></noscript></span></span>.</p></div><div id="sec-8" class="subsection"><h3>Assessing prediction accuracy for new environments</h3><p id="p-37">The prediction problem studied here was that of predicting future seasons, also denoted as “leave one environment out” prediction scenario. This prediction was performed by including phenotypic records and parameter information of either two (<em>indica</em> dataset) or four (<em>tropical japonica</em> dataset) years in the training population to predict a third (<em>indica</em> dataset) or fifth (<em>tropical japonica</em> dataset) year, where no phenotypic data were collected. Prediction accuracies obtained from both PLS and Reaction norm models were assessed by calculating the Pearson correlation between the predicted values from each model for a particular testing year, and the observed phenotypic values for that same year.</p></div><div id="sec-9" class="subsection"><h3>QTL by EC interactions</h3><p id="p-38">For the detection of QTL by environment interaction we used a two-step strategy as described in <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gutiérrez <em>et al.</em> (2015)</a>. In the first step, we scanned the genome of both <em>indica</em> and <em>tropical japonica</em> subspecies to detect QTL in individual environments (single environment QTL mapping). In the second step, QTL expression across environments was regressed on environmental covariates in order to explain QTL effects in terms of sensitivities to environmental covariates (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Malosetti <em>et al.</em> 2004</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boer <em>et al.</em> 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Malosetti <em>et al.</em> 2013</a>).</p><p id="p-39">For the first step, we fitted a mixed model for single environment QTL detection. The model used was the kinship model with:</p><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-65.gif"/></noscript></span></span><p id="p-40">where y is the vector of phenotypic means for that environment, X is the molecular marker score matrix, β is the vector of marker effects, Z is an incidence matrix, u is the vector of random background polygenic effects with variance <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-66.gif"/></noscript></span></span> (where K is the kinship matrix, and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-67.gif"/></noscript></span></span> is the genetic variance), and e is the vector of residuals. A GWAS analysis for each dataset, trait and environment was performed using the R statistical software (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">R Core Team 2017</a>) with the package GWASpoly (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rosyara <em>et al.</em> 2016</a>) fitting the additive model. For QTL determination in each environment, we used the Benjamini-Hochberg FDR (α=0.05) to control the type I error (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>).</p><p id="p-41">In the second step, all marker-trait associations detected in the first step were fitted in a second mixed model testing for interaction with all available EC. This model assumes a linear relationship between the effect of the QTL and a given environmental covariate, using the model presented in <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Malosetti <em>et al.</em> (2013)</a> given by:</p><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-68.gif"/></noscript></span></span><p id="p-42">where <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-69.gif"/></noscript></span></span> is the phenotype of individual i at environment j, μ is the general mean, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-70.gif"/></noscript></span></span> effect of the j<sup>th</sup> environment, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-71.gif"/></noscript></span></span> is the value of the i<sup>th</sup> marker predictor, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-72.gif"/></noscript></span></span> is the effect of the q<sup>th</sup> QTL in the average environment, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-73.gif"/></noscript></span></span> corresponds to the change of the QTL effect per unit of change of the covariable’s value, and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-74.gif"/></noscript></span></span> is the random effect corresponding to the residual (unexplained) QTL effect, with <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-75.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-76.gif"/></noscript></span></span> is the random remaining (not due to the QTL) genotype effect with <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-77.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-78.gif"/></noscript></span></span> is the remaining (random) G×E effect, with <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-79.gif"/></noscript></span></span>. All EC were tested for interaction using three different models for the variance-covariance matrix <strong>Σ</strong> were compared: compound symmetry (CS) where the genetic variances are homogeneous across environments <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-80.gif"/></noscript></span></span> and the genetic covariances between environments are modeled by <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-81.gif"/></noscript></span></span>; heterogeneous compound symmetry (HCS), which allows for heterogeneous genetic variances across environments (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-82.gif"/></noscript></span></span>) and a common genetic covariance parameter <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/5/1519/embed/mml-math-83.gif"/></noscript></span></span>; and the unstructured (UN) model with a specific genetic variance parameter per environment and a specific genetic covariance between environment. The different models were compared using the Bayesian information criterion (BIC) to select the optimal model (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Broman and Speed 2002</a>). We tested for the significance of the fixed terms in mixed models using Wald test at a <em>p</em> value of 0.05, following <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Malosetti <em>et al.</em> (2004)</a>. For QTLxEC interaction testing we used the Benjamini-Hochberg FDR (α=0.05) to control the type I error (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>).</p><p id="p-43">Mixed models for QTL×EC interaction were computed with the R package <em>sommer</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Covarrubias-Pazaran 2016</a>).</p></div><div id="sec-10" class="subsection"><h3>Data Availability</h3><p id="p-44">All the data used in this study, as well as Supplemental Table 1 are provided in Supplemental Material available at Figshare. Genotype data can be found as RDS files (“geno_indica.rds” and “geno_japonica.rds”), phenotypic data can be found as “.csv “ files (“pheno_indica.csv” and “pheno_japonica”), marker positions can be found in files “map_indica.csv” and “map_japonica.csv”, and EC data are available in files “EC_indica.csv” and “EC_japonica.csv”. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7685636">https://doi.org/10.25387/g3.7685636</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-45">The <em>indica</em> dataset was balanced with a total of 327 lines per environment, while the <em>tropical japonica</em> dataset was unbalanced, with a total of 23 lines common to all environments (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Estimations of broad-sense heritability estimated on a line-mean basis per trait by year for both datasets were medium to high, with PHR having the highest values of heritability in both datasets.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493139/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493139/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Description of the rice breeding lines evaluated each year and broad-sense heritabilities for each trait calculated in a line-basis. GY: grain yield, PHR: percentage of head rice, GC: percentage of chalky grains, PH: plant height</span><div class="sb-div caption-clear"></div></div></div><p id="p-47"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows the partitioning of the observed phenotypic variance into different sources of variation for both rice datasets. In the <em>indica</em> population, PHR and GC showed the highest proportion of variance explained by G×Y, at 20.04% and 13.22%, respectively. On the other hand, the year component was the highest variance component for GY and PH (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). In the <em>tropical japonica</em> population, the year component was the highest; it was above all components for the four traits, and much higher than for the <em>indica</em> population. In contrast, the G×Y component was lower in <em>tropical japonica</em> compared to <em>indica</em> (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492954/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492954/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Trait variance component estimation and proportion of the total variance explained for the four traits evaluated in Uruguayan <em>indica</em> and <em>tropical japonica</em> populations. GY: grain yield, PHR: percentage of head rice, GC: percentage of chalky grains, PH: plant height</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genomic prediction of untested years</h3><p id="p-48">Bar plots showing prediction accuracy for the four traits in the <em>indica</em> population are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. PLS-based methods showed higher prediction accuracies than reaction norm-based models for all traits except GC, where prediction accuracies for the PLS method using both markers and EC (PLS-GW) were the same as the reaction norm models. For PLS models, the use of EC in addition to molecular markers resulted in higher prediction accuracies in all cases, though PHR in 2011 and GC in 2012 had identical prediction accuracies for both methods. For reaction norm models, fitting the main effect of genotypes, environments and interaction (G+W+GW model) resulted in either lower or equal prediction accuracies than fitting the simpler model without the interaction term (G+W model) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1519/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between predicted vs. observed values for the &#x201C;leave one year out&#x201D; prediction scenario for Grain Yield (GY), Head Rice Percentage (PHR), Grain Chalkiness percentage (GC) and Plant Height (PH) for predicting untested years with the G, G+W, G+W+GW, PLS-G and PLS-GW for the indica rice breeding population. G = genotypic main effect modeled with marker covariates, W = Environmental main effect modeled with EC, GW = interaction between genotypic and environmental effects, PLS-G = Partial least squares using marker covariates as predictors, PLS-GW = Partial least squares using marker covariates and EC as predictors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2298457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlations between predicted &lt;em&gt;vs.&lt;/em&gt; observed values for the &#x201C;leave one year out&#x201D; prediction scenario for Grain Yield (GY), Head Rice Percentage (PHR), Grain Chalkiness percentage (GC) and Plant Height (PH) for predicting untested years with the G, G+W, G+W+GW, PLS-G and PLS-GW for the &lt;em&gt;indica&lt;/em&gt; rice breeding population. G = genotypic main effect modeled with marker covariates, W = Environmental main effect modeled with EC, GW = interaction between genotypic and environmental effects, PLS-G = Partial least squares using marker covariates as predictors, PLS-GW = Partial least squares using marker covariates and EC as predictors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1519/F1.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/5/1519/F1.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1519/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1519/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493154" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-49" class="first-child">Correlations between predicted <em>vs.</em> observed values for the “leave one year out” prediction scenario for Grain Yield (GY), Head Rice Percentage (PHR), Grain Chalkiness percentage (GC) and Plant Height (PH) for predicting untested years with the G, G+W, G+W+GW, PLS-G and PLS-GW for the <em>indica</em> rice breeding population. G = genotypic main effect modeled with marker covariates, W = Environmental main effect modeled with EC, GW = interaction between genotypic and environmental effects, PLS-G = Partial least squares using marker covariates as predictors, PLS-GW = Partial least squares using marker covariates and EC as predictors.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">In the <em>tropical japonica</em> population, the use of PLS-based models was always better than reaction norm models, with the single exception of GY in 2010 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In all cases, including both markers and EC (PLS-GW) was better than using markers only (PLS-G). Within the reaction norm models, the G+W method was the best, with the exception of GC in 2013. Fitting a G×E component in these models resulted in lower prediction accuracies than fitting the G+W model (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/5/1519/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between predicted vs. observed values for the &#x201C;leave one year out&#x201D; prediction scenario for Grain Yield (GY), Head Rice Percentage (PHR), Grain Chalkiness percentage (GC) and Plant Height (PH) for predicting untested years with the G, G+W, G+W+GW, PLS-G and PLS-GW for the tropical japonica rice breeding population. G = genotypic main effect modeled with marker covariates, W = Environmental main effect modeled with EC, GW = interaction between genotypic and environmental effects, PLS-G = Partial least squares using marker covariates as predictors, PLS-GW = Partial least squares using marker covariates and EC as predictors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2298457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlations between predicted &lt;em&gt;vs.&lt;/em&gt; observed values for the &#x201C;leave one year out&#x201D; prediction scenario for Grain Yield (GY), Head Rice Percentage (PHR), Grain Chalkiness percentage (GC) and Plant Height (PH) for predicting untested years with the G, G+W, G+W+GW, PLS-G and PLS-GW for the &lt;em&gt;tropical japonica&lt;/em&gt; rice breeding population. G = genotypic main effect modeled with marker covariates, W = Environmental main effect modeled with EC, GW = interaction between genotypic and environmental effects, PLS-G = Partial least squares using marker covariates as predictors, PLS-GW = Partial least squares using marker covariates and EC as predictors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/5/1519/F2.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/5/1519/F2.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/5/1519/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/5/1519/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493142" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-51" class="first-child">Correlations between predicted <em>vs.</em> observed values for the “leave one year out” prediction scenario for Grain Yield (GY), Head Rice Percentage (PHR), Grain Chalkiness percentage (GC) and Plant Height (PH) for predicting untested years with the G, G+W, G+W+GW, PLS-G and PLS-GW for the <em>tropical japonica</em> rice breeding population. G = genotypic main effect modeled with marker covariates, W = Environmental main effect modeled with EC, GW = interaction between genotypic and environmental effects, PLS-G = Partial least squares using marker covariates as predictors, PLS-GW = Partial least squares using marker covariates and EC as predictors.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Many of the EC used in this study were correlated. This may result in lower prediction accuracies in the reaction norm model, since it would weight the environmental covariances toward the highly correlated variables. Reaction norm models with a subset of less correlated variables were tried, resulting in very similar or even lower prediction accuracies than when using the entire set of EC (Table S2).</p><p id="p-53">When running PLS with all the environments within each dataset, we can detect which variables best explain each trait by looking at the coefficients. <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the ranking of coefficients for the EC variables for each trait in both datasets. For GY, variables related to temperature and humidity during flowering stage were among the most important. For PHR and GC, the 5 variables with the highest coefficients were related to temperature, humidity and solar radiation during maturation. In the <em>tropical japonica</em> dataset, variables related to humidity, solar radiation and rainfall during maturation showed the highest coefficients for PHR and GC. For GY, two variables at flowering time showed higher coefficient values than the rest: maximum temperature and wind speed (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492967/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492967/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Top 5 PLS-GW coefficients for the environmental covariates for Grain Yield (GY), Head Rice Percentage (PHR), Grain Chalkiness percentage (GC) and Plant Height (PH) for the <em>indica</em> and <em>tropical japonica</em> rice breeding populations. Suffixes V, R and M mean Vegetative stage, Reproductive stage, and Maturation stage respectively</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Detecting QTL in single environments</h3><p id="p-54">We searched for significant trait-marker associations in single years to find QTL to test for interactions with EC in the next step. In this first analysis, we could not find any QTL that passed the FDR threshold for GY in any environment in any population. In the case of PH, we did not find any QTL for the <em>indica</em> population, but we found one major effect QTL on chromosome 1 (position: 37,755,448 - 38,755,448 bp) that was significant in all environments in the <em>japonica</em> dataset; it corresponds to the <em>sd-1</em> gene (position: 38,363,881 bp).</p><p id="p-55">We detected QTL for grain quality traits in both datasets (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). In the <em>indica</em> population, a total of 13 QTL (chromosomes 1, 2, 3, 4, 6, 7, 10 and 11) were found for PHR, and a total of 4 QTL (chromosomes 1, 3 and 4) for GC. QTL were found only in years 2010 and 2012 for PHR, and in years 2011 and 2012 for GC. Three of the QTL were reported in a previous GWAS analysis using this same dataset (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Quero <em>et al.</em> 2018</a>). These QTL were: qPHR.i.2.2 (S2_24210614), qPHR.i.3.1 (S3_10247958), qGC.i.1.1 (S1_1066894). Two additional QTL were in LD with two previously reported QTL in the same study. These were qPHR.i.3.2 and qPHR.i.6.1, which were in LD with S3_15365726 and S6_829223 in our study, respectively. In the <em>tropical japonica</em> population, a total of 5 QTL were found for PHR (chromosomes 1, 2, 3, 6, and 8), and one for GC (chromosome 6) (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). Two of these QTL were in LD with previously reported QTL: qPHR.j.3.1 with S3_1395165, and qGC.j.6.2 with S6_27402260 (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Quero <em>et al.</em> 2018</a>). No significant QTL were found for GY or PH for any year in either of the populations.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493069/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493069/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Marker-trait associations for percentage of head rice (PHR) and percentage of chalky grain (GC) traits in <em>indica</em> and <em>tropical japonica</em> rice breeding populations. Chromosome position (bp), year, effect of the alternative allele, and score (-log<sub>10</sub>(p-value)) are shown in the table</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>QTL × environmental covariate interactions</h3><p id="p-56">A decomposition of the QTL with significant QTL × environment interaction was obtained by introducing environmental covariates as explanatory variables. We first tested different covariance structures for the modeling of the G×E component and compared them using BIC (see Methods). For all QTL and traits, BIC values decreased when using the HCS matrix compared to the CS matrix. However, the HCS model already behaves quite similar to the maximally complex UN model, so the HCS was the model of choice (Table S1). The QTL responses for the <em>indica</em> dataset are shown in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>. One QTL showed significant interaction with environmental covariates related to precipitation and humidity during the maturation stage. Marker S2_24210614 showed a negative relationship with PpitDay_M and RelH_M. The high correlation between these two variables (<em>r</em> = 0.99) explains why they show the same coefficients for the main QTL effect (α), and the interaction (β). No significant main effect was detected for this QTL (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492936/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492936/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">QTL responses to EC for percentage of head rice (PHR) in the <em>indica</em> rice population. Suffixes R and M mean Reproductive stage, and Maturation stage respectively</span><div class="sb-div caption-clear"></div></div></div><p id="p-60">Results for regression of marker covariates on environmental covariates for the <em>tropical japonica</em> dataset are shown in <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>. For GC, marker S6_27402260, located in chromosome 6, showed a significant positive response to weather covariates related to precipitation and minimum temperature, and a negative response to sunshine duration and solar radiation. This marker also showed a significant main effect (<a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493045/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493045/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">QTL responses to EC for head rice percentage (PHR) and percentage of chalky grain (GC) in the <em>tropical japonica</em> rice population. Suffixes R and M mean Reproductive stage, and Maturation stage respectively</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-64">In this work we proposed to characterize and interpret G×E interaction for four traits (GY, PHR, GC and PH) in two different breeding populations of rice (<em>indica</em> and <em>tropical japonica</em>) grown in a subtropical/temperate climate. In the first part of our paper, we compare the performance of different genomic prediction models that account for genotype, environment and G×E components, to predict untested years, and we identify the most influential weather covariates for our two datasets. In the second part, we map environment-specific QTL and study the environmental variables that affect their expression, in order to interpret the QTL×E effects that account for the total G×E.</p><div id="sec-17" class="subsection"><h3>Prediction accuracies for untested environments</h3><p id="p-65">Usually genomic prediction models are tested and compared using cross-validation strategies. In a multiple environment context, most studies include two basic random cross-validation schemes (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Burgueño <em>et al.</em> 2012</a>): CV1, which tests the performance of lines that have not been evaluated in any of the observed environments, and CV2, which tests the performance of lines that have been evaluated in some environments but not in others. These two scenarios have the disadvantage of training and validating the models with the same data, which could lead to an overestimation of the prediction accuracy the model would attain if it had been applied in an independent test dataset. Predicting new environments is a more difficult task but could represent a good validation strategy because the performance of prediction models is assessed in an independent dataset. In this work we used a cross-validation scheme for prediction in untested environments, represented by years, a component of G×E that is not easy to reproduce. This is a very relevant type of prediction for a small plant breeding program, where data from multiple locations is either limited or absent, and the need is to predict which lines are more likely to perform better in future environments. The use of EC to model the environment component explicitly has been previously shown to increase prediction accuracies for untested environments (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Malosetti <em>et al.</em> 2016</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jarquín <em>et al.</em> 2017</a>), and this situation also applies to our work.</p><p id="p-66">For prediction, we compared two modeling approaches that differ in the way that multiple and correlated variables are handled: 1) a variance components approach that allows modeling the main and interaction effects of markers and EC using covariance structures, and 2) a PLS approach that models genotype and environment effects by identifying a linear combination of all the explanatory variables, providing latent vectors that optimally predict the response variable. We found that the PLS-GW model was in all cases superior to or not different from PLS-G and reaction norm models in both datasets. Although the variance explained by the G×E component in the <em>indica</em> population, shown in <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>, was comparable in some cases to the variance explained by the genotype and/or the year main components, the proportion of variance explained jointly by the genotype, environment and G×E components, was never superior to 50% of the total variance. This could explain the lower prediction accuracies obtained in this population compared to the <em>japonica</em> population. It is possible that the EC used in this study explained only a limited proportion of the across environment interaction in the <em>indica</em> dataset, and for this reason reaction norm models, when fitting covariance matrices for the environment and marker by environment interaction, did not improve prediction accuracies in comparison to the simpler GBLUP model. In the <em>japonica</em> population, the proportion of the total variance explained by G×E was very low compared to the main genotype and environment components, which also explains why modeling a specific interaction covariance matrix did not give better results than modeling the main genotype and environment covariance matrices alone. In this population, the main environment effect was better represented by the EC, and thus, prediction accuracies, when including an EC covariance matrix (W) or the EC in the PLS model, were higher than when using a G matrix or molecular markers alone.</p><p id="p-67">Besides the ability of handling numerous and correlated predictors, an additional advantage of using PLS models is that we can detect which covariates are the most explanatory in our model by looking at the model coefficients (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Wold <em>et al.</em> 2001</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mehmood <em>et al.</em> 2012</a>). Previous studies have shown the benefits of PLS for identifying the set of EC that best explain G×E (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Vargas <em>et al.</em> 1998</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Vargas <em>et al.</em> 1999</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Crossa <em>et al.</em> 1999</a>). In these studies, the G×E component of the trait was used as a response and regressed to EC only. In our case, we decided to report the results of the regression of the trait means to both EC and markers, since regressing the G×E component to EC resulted in increased MSEP with an increasing number of components, and thus a poor model fit. For GY, minimum and average temperature, and effective precipitation during flowering time showed the highest positive coefficients for <em>indica</em> rice, as shown in <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>. In regions with a temperate climate, low temperatures during flowering can affect grain yield by inducing spikelet sterility (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Yoshida 1981</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alvarado 2002</a>). The probability of occurrence of temperatures under 15° during January (when rice usually enters the flowering stage) in Eastern Uruguay is about 20%, and would be most detrimental for <em>indica</em> varieties, which are best adapted to tropical climates. In the <em>tropical japonica</em> population, the two EC that showed the highest (negative) coefficients for GY were wind speed during flowering, and maximum temperature during grain filling. Both wind speed and high temperatures during reproduction have been proven to negatively affect GY due to pollen dehydration and consequent spikelet sterility (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Marchezan and Aude 1993</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Raju <em>et al.</em> 2013</a>).</p><p id="p-68">For the grain quality traits, EC related to humidity, solar radiation and sunshine duration during grain ripening were among the most important in both datasets. The positive coefficients for solar radiation, and the negative coefficients for humidity reflect the relative effects of these variables on milling quality, as previously reported (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Siebenmorgen <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Edwards <em>et al.</em> 2017</a>). Many studies have reported negative effects of high temperatures on grain chalk and percent head rice (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tashiro and Wardlaw, 1991</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lyman <em>et al.</em> 2013</a>). For example, for <em>japonica</em> cultivars, temperatures higher than 26° Can cause chalky grain appearance (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2016</a>), but maximum daytime temperatures higher than 33° Cause dramatic changes in the distribution of head and broken rice, and increase the proportion of chalky grain (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ambardekar <em>et al.</em> 2011</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lyman <em>et al.</em> 2013</a>). In Eastern Uruguay, maximum temperatures during February-March, the period in which rice kernels usually develop, rarely reach 32°. In our own dataset, the average maximum temperatures were never higher than 30°, so it is probable that in the absence of high stress-inducing temperatures in sub-tropical rice growing areas, other variables such as humidity and solar radiation are more important, as is reflected in our results.</p></div><div id="sec-18" class="subsection"><h3>QTL detection and interaction With environmental covariates</h3><p id="p-69">For this part of the analysis we used mixed-models to analyze QTL by EC interactions because of their flexibility, and the possibility of modeling genetic correlations between environments. We first performed an association mapping analysis for each of the four traits in each environment in both populations. In the case of PH, <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Rosas <em>et al.</em> (2017)</a> performed a GWAS analysis on these same populations using the mean across environments and found a major effect QTL corresponding to the <em>sd-1</em> gene which was segregating in the <em>japonica</em> population, but fixed in the semi-dwarf <em>indica</em> population (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Rosas <em>et al.</em> 2017</a>). When we performed a single environment scan we could not find any other QTL in either population, other than a major-effect QTL corresponding to the <em>sd-1</em> gene in <em>japonica</em>.</p><p id="p-70">Of the 23 QTL we found for PHR and GC in both populations in <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>, 8 were coincident with QTL reported by <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Quero <em>et al.</em> (2018)</a> in the same populations using the mean across environments. For PHR in <em>indica</em>, we found evidence of one genomic region, located in chromosome 2 that is affected by humidity, one of the main environmental factors that affect milling quality in rice (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cooper <em>et al.</em> 2008</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zhao and Fitzgerald 2013</a>).</p><p id="p-71">Two putative QTL in <em>tropical japonica</em> were co-located on chromosome 6: S6_27834772 for PHR and S6_27402260 for GC. These two QTL are in LD with qPHR.j.6.1 and qGC.j.6.2 previously found by <a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Quero <em>et al.</em> (2018)</a>, and contain genes related to starch metabolism, such as <em>OsBEI</em> (LOC_Os06g51084). It is known that the expression of starch branching enzymes, like <em>OsBEI</em>, can be affected by temperature (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yamakawa <em>et al.</em> 2007</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sreenivasulu <em>et al.</em> 2015</a>). According to our results, QTL S6_27402260 showed interaction with low temperature, precipitation, sunshine duration and solar radiation for GC, as shown in <a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a>. Other researchers have shown that periods of intense solar radiation and high humidity during the ripening stage can increase the incidence of chalky grains (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wakamatsu <em>et al.</em> 2009</a>, <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> 2016</a>). But these reports do not constitute proof that there is a causal relationship between the expression of these QTL and the EC, because many EC are correlated in a complex way and not all EC were observed. In temperate climates, where day and night temperatures are never as high as in the tropics, other environmental factors such as humidity and solar radiation can affect milling quality in a negative way. These findings should be confirmed by analyzing more lines in more environments to properly quantify QTL main and environment-specific effects.</p><p id="p-72">The approach of mapping QTL by environment interaction used in this study requires a QTL to have a strong effect in a specific environment. This poses the limitation that QTL with smaller effects in individual environments but capable to explain larger proportions of the observed phenotypic G×E may be overlooked. In our datasets, the proportion of phenotypic variance explained by the G×Y component is low in the <em>tropical japonica</em> dataset (1.8–7.96%, <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>), but higher in the <em>indica</em> dataset (4.12–20%). However, approaches for testing for genotype by year interaction at each SNP were performed and no significant QTL were found.</p><p id="p-73">In this work we used PLS, multiplicative reaction norm and mixed models to analyze our data, predict genotypic performance for yield, height and milling quality traits, and detect QTL by EC interactions. In all these analyses we assumed that the relationships between molecular markers and EC were linear, which constitutes a major limitation since interactions between genes and environmental conditions may take many different forms. A next step would be to fit statistical models with more biological realism, using models that could accommodate non-linear and more complex responses over a more extensive number of environments. Crop growth models also hold promise as a way to integrate more complex biological knowledge into the prediction process of G×E (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Malosetti <em>et al.</em> 2016</a>). Although rather small, our two datasets allowed us to extract some broad conclusions about the nature of G×E in the Uruguayan mega-environment. Additional research, including more environments and modeling non-linear relationships between genes and EC, will be of particular value to better understand and predict the nature of G×E for commercially relevant traits of rice grown in temperate regions. Results from PLS and QTL by EC interactions suggest that in temperate and subtropical regions, humidity and solar radiation may have a stronger influence on milling quality traits than temperature, due to the fact that temperatures in these regions are never as high as in the tropics.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-74">The authors wish to thank the technical staff at INIA-Treinta y Tres and INIA-Las Brujas in Uruguay, and at Cornell University for their valuable support in laboratory and fieldwork. We are especially grateful to Jean-Luc Jannink and Deniz Akdemir for their valuable input and comments regarding data analysis, and to the three anonymous reviewers for their constructive criticism and comments that improved this manuscript. This study was funded by INIA’s Rice Association Mapping Project, a Ph.D. fellowship to support E. Monteverde from Monsanto’s Beachell-Borlaug International Scholarship Program, and research support from the Fulbright Commission and the Uruguayan Research and Innovation Agency (ANII).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-75">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7685636">https://doi.org/10.25387/g3.7685636</a>.</p></li><li class="fn" id="fn-9"><p id="p-76"><em>Communicating editor: G. Morris</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-06"><span class="received-label">Received </span>February 6, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-03-05"><span class="accepted-label">Accepted </span>March 5, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Monteverde <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.1" data-doi="10.2307/2530697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aastveit</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">ANOVA interactions interpreted by partial least squares regression.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">844</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2530697</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D42%26rft.spage%253D829%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2530697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2530697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.5.1519.2"><div class="cit-metadata unstructured">Alvarado, J. R., 2002 Influence of air temperature on rice population, length of period from sowing to flowering and spikelet sterility. In Proceedings of the second temperate rice conference, IRRI, Philippines.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.3" data-doi="10.1016/j.fcr.2011.03.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ambardekar</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebenmorgen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Counce</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanning</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauromoustakos</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Impact of field-scale nighttime air temperatures during kernel development on rice milling quality.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2011.03.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D122%26rft.spage%253D179%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2011.03.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2011.03.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.4" data-doi="10.1093/biostatistics/kxj037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxj037%26rft_id%253Dinfo%253Apmid%252F16632515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxj037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16632515&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.5" data-doi="10.1534/genetics.107.071068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A mixed-model quantitative trait loci (QTL) analysis for multiple-environment trial data using environmental covariables for QTL-by-environment interactions, with an example in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1813</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.071068</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071068%26rft_id%253Dinfo%253Apmid%252F17947443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE4MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.6" data-doi="10.1093/bib/bbl016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boulesteix</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Partial least squares: a versatile tool for the analysis of high-dimensional genomic data.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbl016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbl016%26rft_id%253Dinfo%253Apmid%252F16772269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbl016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16772269&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243593400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.7" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.8" data-doi="10.1111/1467-9868.00354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A model selection approach for the identification of quantitative trait loci in experimental crosses.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">656</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1467-9868.00354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D64%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.9" data-doi="10.2135/cropsci2006.09.0564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modeling additive × environment and additive × additive × environment using genetic covariances of relatives of wheat genotypes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2006.09.0564</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBurgueno%26rft.auinit1%253DJ.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D311%26rft.epage%253D320%26rft.atitle%253DModeling%2BAdditive%2Bx%2BEnvironment%2Band%2BAdditive%2Bx%2BAdditive%2Bx%2BEnvironment%2BUsing%2BGenetic%2BCovariances%2Bof%2BRelatives%2Bof%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.09.0564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.09.0564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244430100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.10" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic Prediction of Breeding Values when Modeling Genotype × Environment Interaction using Pedigree and Dense Molecular Markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.11" data-doi="10.2135/cropsci2004.6200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baezinger</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Budak</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streck</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Using environmental covariates to explain Genotype × Environment and QTL × Environment interactions for agronomic traits on chromosome 3A of wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">627</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2004.6200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCampbell%26rft.auinit1%253DB.%2BT.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D620%26rft.epage%253D627%26rft.atitle%253DUsing%2BEnvironmental%2BCovariates%2Bto%2BExplain%2BGenotype%2Bx%2BEnvironment%2Band%2BQTL%2Bx%2BEnvironment%2BInteractions%2Bfor%2BAgronomic%2BTraits%2Bon%2BChromosome%2B3A%2Bof%2BWheat%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.6200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.6200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000189384600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.12" data-doi="10.1007/s00709-016-1002-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effects of short term high temperature on grain quality and starch granules of rice (<em>Oryza sativa</em> L.) at post-anthesis stage.</span> <abbr class="cit-jnl-abbrev">Protoplasma</abbr> <span class="cit-vol">254</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">943</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00709-016-1002-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtoplasma%26rft.volume%253D254%26rft.spage%253D935%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00709-016-1002-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00709-016-1002-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.13" data-doi="10.3168/jds.2011-4647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colombani</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croiseau</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comparison of partial least squares (PLS) and sparse PLS regressions in genomic selection in French dairy cattle</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">2120</span>–<span class="cit-lpage">2131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2011-4647</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DColombani%26rft.auinit1%253DC.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D2120%26rft.epage%253D2131%26rft.atitle%253DA%2Bcomparison%2Bof%2Bpartial%2Bleast%2Bsquares%2B%2528PLS%2529%2Band%2Bsparse%2BPLS%2Bregressions%2Bin%2Bgenomic%2Bselection%2Bin%2BFrench%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4647%26rft_id%253Dinfo%253Apmid%252F22459857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22459857&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.14" data-doi="10.1007/BF01240919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeLacy</span>  <span class="cit-name-given-names">I. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Relationships among analytical methods used to study genotypic variation and genotype-by-environment interaction in plant breeding multi-environment experiments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF01240919</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D88%26rft.spage%253D561%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01240919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01240919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PA75000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.15" data-doi="10.1094/CCHEM-85-3-0276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">N. T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebenmorgen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Counce</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Effects of night- time temperature during kernel development on rice physico-chemical properties.</span> <abbr class="cit-jnl-abbrev">Cereal Chem.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/CCHEM-85-3-0276</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCereal%2BChem.%26rft.volume%253D85%26rft.spage%253D276%26rft_id%253Dinfo%253Adoi%252F10.1094%252FCCHEM-85-3-0276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/CCHEM-85-3-0276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.16" data-doi="10.1371/journal.pone.0156744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Covarrubias-Pazaran</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-assisted prediction of quantitative traits using the R package sommer.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0156744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0156744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0156744%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.17" data-doi="10.1007/s001220051276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edmeades</span>  <span class="cit-name-given-names">G. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Interpreting genotype × environment interaction in tropical maize using linked molecular markers and environmental covariates.</span> <abbr class="cit-jnl-abbrev">TAG Theor Appl Genet</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">625</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220051276</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%2BTheor%2BAppl%2BGenet%26rft.volume%253D99%26rft.spage%253D611%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.18" data-doi="10.1016/j.fcr.2017.01.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClung</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic architecture of grain chalk in rice and interactions with a low phytic acid locus.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2017.01.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D205%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2017.01.015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2017.01.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.19" data-doi="10.1071/AR9630742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finlay</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">The analysis of adaptation in a plant-breeding programme.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">742</span>–<span class="cit-lpage">754</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/AR9630742</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D14%26rft.spage%253D742%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR9630742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR9630742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19636069A00005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.20" data-doi="10.1016/j.tplants.2008.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitzgerald</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Not just a grain of rice: the quest for quality.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2008.12.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DFitzgerald%26rft.auinit1%253DM.%2BA.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D133%26rft.epage%253D139%26rft.atitle%253DNot%2Bjust%2Ba%2Bgrain%2Bof%2Brice%253A%2Bthe%2Bquest%2Bfor%2Bquality.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2008.12.004%26rft_id%253Dinfo%253Apmid%252F19230745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2008.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19230745&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264681000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.21" data-doi="10.2307/2531585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gauch</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Model selection and validation for yield trials with interaction.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">715</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2531585</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D44%26rft.spage%253D705%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2531585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2531585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988Q069100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1519.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gauch</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Statistical analysis of regional yield trials: AMMI analysis of factorial designs</span>, <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">Amsterdam, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.23" data-doi="10.1007/s00122-014-2448-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Germán</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereyra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multi-environment multi-QTL association mapping identifies disease resistance QTL in barley germplasm from Latin America.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">516</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2448-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D501%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2448-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2448-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.24" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2231-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.25" data-doi="10.1371/journal.pone.0120610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebana</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uga</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic prediction of biological shape: elliptic fourier analysis and kernel partial least squares (PLS) regression applied to grain shape prediction in rice (Oryza sativa L.)</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0120610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0120610</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0120610%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0120610%26rft_id%253Dinfo%253Apmid%252F25825876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0120610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25825876&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.26" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high - dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">TAG Theor Appl Genet</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%2BTheor%2BAppl%2BGenet%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.27" data-doi="10.3835/plantgenome2016.12.0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">da Silva</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increasing genomic-enabled prediction accuracy by modeling genotype × environment interactions in Kansas wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-fpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.12.0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.12.0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.12.0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.28" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-web" id="cit-9.5.1519.29"><div class="cit-metadata unstructured">Lê Cao, K. A., F. Rohart, I. Gonzalez, S. Dejean, B. Gautier <em>et al.</em>, 2016 mixomics: Omics data integration project. R package version 6.1.1. <a href="https://CRAN.R-project.org/package=mixOmics">https://CRAN.R-project.org/package=mixOmics</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.5.1519.30" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker x Environment Interaction Genomic Selection Model. G3 Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genet</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.016097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet%26rft.volume%253D5%26rft.spage%253D569%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.016097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.31" data-doi="10.1111/j.1439-0388.2011.00917.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dimension reduction and variable selection for genomic selection: application to predicting milk yield in Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2011.00917.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DLong%26rft.auinit1%253DN.%26rft.volume%253D128%26rft.issue%253D4%26rft.spage%253D247%26rft.epage%253D257%26rft.atitle%253DDimension%2Breduction%2Band%2Bvariable%2Bselection%2Bfor%2Bgenomic%2Bselection%253A%2Bapplication%2Bto%2Bpredicting%2Bmilk%2Byield%2Bin%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00917.x%26rft_id%253Dinfo%253Apmid%252F21749471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00917.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21749471&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.32" data-doi="10.1371/journal.pone.0072157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyman</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jagadish</span>  <span class="cit-name-given-names">K. S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nalley</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebenmorgen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Neglecting rice milling yield and quality underestimates economic losses from high-temperature stress.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e72157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0072157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De72157%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0072157%26rft_id%253Dinfo%253Apmid%252F23991056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0072157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23991056&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.33" data-doi="10.3389/fphys.2013.00044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The statistical analysis of multi-environment data: modeling genotype-by-environment interaction and its genetic basis.</span> <abbr class="cit-jnl-abbrev">Front. Physiol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fphys.2013.00044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPhysiol.%26rft.volume%253D4%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffphys.2013.00044%26rft_id%253Dinfo%253Apmid%252F23487515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fphys.2013.00044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23487515&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.34" data-doi="10.2135/cropsci2015.05.0311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustos-Korts</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Predicting responses in multiple environments: Issues in relation to genotype × environment interactions.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2210</span>–<span class="cit-lpage">2222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.05.0311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2210%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.05.0311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.05.0311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.35" data-doi="10.1023/B:EUPH.0000040511.46388.ef"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voltas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romagosa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ullrich</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mixed models including environmental covariables for studying QTL by environment interaction.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/B:EUPH.0000040511.46388.ef</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D137%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253AEUPH.0000040511.46388.ef%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:EUPH.0000040511.46388.ef&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.36" data-doi="10.2135/cropsci2004.0301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malvar</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Revilla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butrón</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouesnard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Performance of crosses among French and Spanish maize populations across environments.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1052</span>–<span class="cit-lpage">1057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2004.0301</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMalvar%26rft.auinit1%253DR.%2BA.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D1052%26rft.epage%253D1057%26rft.atitle%253DPerformance%2Bof%2BCrosses%2Bamong%2BFrench%2Band%2BSpanish%2BMaize%2BPopulations%2Bacross%2BEnvironments%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000229105700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.37" data-doi="10.1590/S0103-84781993000300025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchezan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aude</span>  <span class="cit-name-given-names">M. I. D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Adverse effect of the wind in irrigated rice.</span> <abbr class="cit-jnl-abbrev">Cienc. Rural</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/S0103-84781993000300025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCienc.%2BRural%26rft.volume%253D23%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS0103-84781993000300025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S0103-84781993000300025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.38" data-doi="10.1016/j.chemolab.2012.07.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehmood</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liland</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snipen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sæbø</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A review of variable selection methods in partial least squares regression.</span> <abbr class="cit-jnl-abbrev">Chemom. Intell. Lab. Syst.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">69</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.chemolab.2012.07.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemom.%2BIntell.%2BLab.%2BSyst.%26rft.volume%253D118%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chemolab.2012.07.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chemolab.2012.07.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.39" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.40" data-doi="10.1002/cem.887"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mevik</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cederkvist</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mean squared error of prediction (MSEP) estimates for principal component regression (PCR) and partial least squares regression (PLSR).</span> <abbr class="cit-jnl-abbrev">J. Chemometr.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">429</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/cem.887</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChemometr.%26rft.volume%253D18%26rft.spage%253D422%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcem.887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cem.887&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227960300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.41" data-doi="10.2135/cropsci2017.09.0564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monteverde</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosas</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez de Vida</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnecarrère</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Multienvironment models increase prediction accuracy of complex traits in advanced breeding lines of rice.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1530</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.09.0564</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D58%26rft.spage%253D1519%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.09.0564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.09.0564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.42" data-doi="10.3835/plantgenome2017.08.0076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quero</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monteverde</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez de Vida</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-wide association study using historical breeding populations discovers genomic regions involved in high-quality rice.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: 170076.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.08.0076</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D11%26rft.spage%253D170076%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.08.0076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.08.0076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1519.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A language and environment for statistical computing</span>, <span class="cit-publ-name">R Found. Stat. Comput</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.44" data-doi="10.3923/ijar.2013.137.148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senguttuvel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voleti</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhadana</span>  <span class="cit-name-given-names">V. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stability analysis of flowering and yield traits to high temperature stress adopting different planting dates in rice (<em>O. sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Int. J. Agric. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3923/ijar.2013.137.148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D8%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.3923%252Fijar.2013.137.148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3923/ijar.2013.137.148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.45" data-doi="10.3835/plantgenome2017.03.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosas</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez de Vida</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnecarrere</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Resistance to multiple temperate and tropical stem and sheath diseases of rice.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">170029</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.03.0029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D11%26rft.spage%253D170029%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.03.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.03.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.46" data-doi="10.3835/plantgenome2015.08.0073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosyara</span>  <span class="cit-name-given-names">U. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Software for genome-wide association studies in autopolyploids and its application to potato.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.08.0073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.08.0073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.08.0073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1519.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siebenmorgen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Counce</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Factors affecting rice milling quality</span>, <span class="cit-publ-name">Univ.</span>, <span class="cit-publ-loc">Fayetteville, Arkansas</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.48" data-doi="10.1017/S0021859605005587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The analysis of crop cultivar breeding and evaluation trials: An overview of current mixed model approaches.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">462</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0021859605005587</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D143%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859605005587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859605005587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.49" data-doi="10.1186/1297-9686-41-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reducing dimensionality for prediction of genome-wide breeding values.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-29</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSolberg%26rft.auinit1%253DT.%2BR.%26rft.volume%253D41%26rft.spage%253D29%26rft.epage%253D29%26rft.atitle%253DReducing%2Bdimensionality%2Bfor%2Bprediction%2Bof%2Bgenome-wide%2Bbreeding%2Bvalues.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-29%26rft_id%253Dinfo%253Apmid%252F19296851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19296851&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.50" data-doi="10.1093/jxb/eru544"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sreenivasulu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butardo</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anacleto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Designing climate-resilient rice with ideal grain quality suited for high-temperature stress.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">1737</span>–<span class="cit-lpage">1748</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/eru544</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru544%26rft_id%253Dinfo%253Apmid%252F25662847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/eru544&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25662847&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.51" data-doi="10.3835/plantgenome2014.05.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero Navarro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel methods to optimize genotypic imputation for low-coverage, next generation sequence data in crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.52" data-doi="10.1071/AR9910485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tashiro</span>  <span class="cit-name-given-names">T. W. I. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wardlaw</span>  <span class="cit-name-given-names">I. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The effect of high temperature on kernel dimensions and the type and occurrence of kernel damage in rice</span>. <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/AR9910485</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D42%26rft.spage%253D485%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR9910485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR9910485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FH91900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1519.53" data-doi="10.1201/9781420049374.ch2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Incorporating additional information on genotypes and environments in models for two-way genotype by environment tables</span>, pp. <span class="cit-fpage">15</span>–<span class="cit-lpage">49</span> in <span class="cit-source">Gentoype by environment interaction</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Gauch</span>  <span class="cit-name-given-names">H. G.</span>  <span class="cit-name-suffix">Jr</span>.</span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1201/9781420049374.ch2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGentoype%2Bby%2Benvironment%2Binteraction%26rft_id%253Dinfo%253Adoi%252F10.1201%252F9781420049374.ch2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1201/9781420049374.ch2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.54" data-doi="10.2135/cropsci1998.0011183X003800030010x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayre</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramírez</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Interpreting genotype × environment interaction in wheat by partial least squares regression.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1998.0011183X003800030010x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DVargas%26rft.auinit1%253DM.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D679%26rft.epage%253D689%26rft.atitle%253DInterpreting%2BGenotype%2Bx%2BEnvironment%2BInteraction%2Bin%2BWheat%2Bby%2BPartial%2BLeast%2BSquares%2BRegression%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1998.0011183X003800030010x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1998.0011183X003800030010x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000074754600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.55" data-doi="10.2135/cropsci1999.0011183X003900040002x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramírez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayre</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Using partial least squares regression, factorial regression, and AMMI models for interpreting genotype × environment interaction.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">967</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1999.0011183X003900040002x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DVargas%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D955%26rft.epage%253D967%26rft.atitle%253DUsing%2BPartial%2BLeast%2BSquares%2BRegression%252C%2BFactorial%2BRegression%252C%2Band%2BAMMI%2BModels%2Bfor%2BInterpreting%2BGenotype%2Bx%2BEnvironment%2BInteraction%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.0011183X003900040002x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.0011183X003900040002x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000081515700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.56" data-doi="10.1626/jcs.78.476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakamatsu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effects of the amount of insolation and humidity during the ripening period on the grain quality of brown rice in warm regions of Japan.</span> <abbr class="cit-jnl-abbrev">Jpn. J. Crop. Sci.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">476</span>–<span class="cit-lpage">482</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1626/jcs.78.476</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJpn.%2BJ.%2BCrop.%2BSci.%26rft.volume%253D78%26rft.spage%253D476%26rft_id%253Dinfo%253Adoi%252F10.1626%252Fjcs.78.476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1626/jcs.78.476&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1519.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Estimation of principal components and related models by iterative least sqares</span>, pp. <span class="cit-fpage">114</span>–<span class="cit-lpage">142</span> in <span class="cit-source">Multivariate Analysis</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Krishnaiah</span>  <span class="cit-name-given-names">P. R.</span></span> <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.58" data-doi="10.1016/S0169-7439(01)00155-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sjostrom</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">PLS-regression: A basic tool for chemometrics.</span> <abbr class="cit-jnl-abbrev">Chemom. Intell. Lab. Syst.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">130</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0169-7439(01)00155-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemom.%2BIntell.%2BLab.%2BSyst.%26rft.volume%253D58%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-7439%252801%252900155-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-7439(01)00155-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000172360800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.59" data-doi="10.1104/pp.107.098665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamakawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirose</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuroda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comprehensive expression profiling of rice grain filling-related genes under high temperature using DNA microarray.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.107.098665</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.107.098665%26rft_id%253Dinfo%253Apmid%252F17384160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDQvMS8yNTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzUvMTUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.60" data-doi="10.1017/S0021859600050978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochran</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">The analysis of groups of experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0021859600050978</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D28%26rft.spage%253D556%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859600050978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859600050978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-book" id="cit-9.5.1519.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-source">Fundamentals of rice crop science</span>, <span class="cit-publ-name">International Rice Research Institute</span>, <span class="cit-publ-loc">Los Banos, Philippines</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.62" data-doi="10.1371/journal.pone.0066218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzgerald</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Climate change: implications for the yield of edible rice.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e66218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0066218</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De66218%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0066218%26rft_id%253Dinfo%253Apmid%252F23776635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0066218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23776635&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F5%2F1519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.5.1519.63" data-doi="10.1007/s00122-015-2616-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daygon</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sackville</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of stable QTLs causing chalk in rice grains in nine environments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2616-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2616-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2616-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/5/1519.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/5/1507" title="Small RNA-Seq Analysis Reveals miRNA Expression Dynamics Across Tissues in the Malaria Vector, Anopheles gambiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/5/1533" title="Variation in Filamentous Growth and Response to Quorum-Sensing Compounds in Environmental Isolates of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/5"><span>Volume 9 Issue 5, May 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/5.cover-source.jpg?itok=5ji6xjqZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/5/1519" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vvYQVmFufeDoe3_KMQKQ7rsrNBm5xw9KKV-OtqcbGFU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/5/1519" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/5/1519" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/5/1519" class="active" data-icon-position="" data-hide-link-title="0">Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (Oryza sativa L.) Grown in Subtropical Areas</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493104" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LLpIPdc_SLu-Adt3YU7RXooVsIKO4R3SnDicpfK-tL0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1519.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/5/1519" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZlHCxFPBXceIl9P5u8CB3hZlrDeSi-y_0lm1SmNOGpE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/5/1519" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/5/1519.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493104" id="node493104--2" data-pisa="ggg;9/5/1519" data-pisa-master="ggg;g3.119.400064" data-apath="/ggg/9/5/1519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (<em>Oryza sativa</em> L.) Grown in Subtropical Areas</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Eliana Monteverde</span>, <span class="highwire-citation-author" data-delta="1">Lucía Gutierrez</span>, <span class="highwire-citation-author" data-delta="2">Pedro Blanco</span>, <span class="highwire-citation-author" data-delta="3">Fernando Pérez de Vida</span>, <span class="highwire-citation-author" data-delta="4">Juan E. Rosas</span>, <span class="highwire-citation-author" data-delta="5">Victoria Bonnecarrère</span>, <span class="highwire-citation-author" data-delta="6">Gastón Quero</span> and <span class="highwire-citation-author" data-delta="7">Susan McCouch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1519-1531;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400064 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493104--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Eliana Monteverde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eliana%2BMonteverde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monteverde%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEliana%2BMonteverde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lucía Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, University of Wisconsin - Madison WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%C3%ADa%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuc%25C3%25ADa%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pedro Blanco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BBlanco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanco%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BBlanco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando Pérez de Vida </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BP%C3%A9rez%2Bde%2BVida%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20de%20Vida%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BP%25C3%25A9rez%2Bde%2BVida%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan E. Rosas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Unidad de Biotecnología, Instituto Nacional de Investigación Agropecuaria (INIA), Estación Experimental Wilson Ferreira Aldunate 90200, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BE.%2BRosas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosas%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BE.%2BRosas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Victoria Bonnecarrère </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Unidad de Biotecnología, Instituto Nacional de Investigación Agropecuaria (INIA), Estación Experimental Wilson Ferreira Aldunate 90200, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BBonnecarr%C3%A8re%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonnecarr%C3%A8re%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictoria%2BBonnecarr%25C3%25A8re%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gastón Quero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Biology, College of Agriculture, Universidad de la República, Montevideo, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gast%C3%B3n%2BQuero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quero%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGast%25C3%25B3n%2BQuero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Susan McCouch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BMcCouch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCouch%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BMcCouch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srm4@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">srm4@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/5/1519" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/5/1519&amp;title=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/5/1519&amp;title=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/5/1519&amp;title=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/5/1519&amp;text=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/5/1519&amp;title=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/5/1519&amp;t=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/5/1519&amp;title=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/5/1519&amp;title=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493104/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493104" id="node493104--3" data-pisa="ggg;9/5/1519" data-pisa-master="ggg;g3.119.400064" data-apath="/ggg/9/5/1519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Integrating Molecular Markers and Environmental Covariates To Interpret Genotype by Environment Interaction in Rice (<em>Oryza sativa</em> L.) Grown in Subtropical Areas</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Eliana Monteverde</span>, <span class="highwire-citation-author" data-delta="1">Lucía Gutierrez</span>, <span class="highwire-citation-author" data-delta="2">Pedro Blanco</span>, <span class="highwire-citation-author" data-delta="3">Fernando Pérez de Vida</span>, <span class="highwire-citation-author" data-delta="4">Juan E. Rosas</span>, <span class="highwire-citation-author" data-delta="5">Victoria Bonnecarrère</span>, <span class="highwire-citation-author" data-delta="6">Gastón Quero</span> and <span class="highwire-citation-author" data-delta="7">Susan McCouch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1519-1531;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400064 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493104--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Eliana Monteverde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eliana%2BMonteverde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monteverde%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEliana%2BMonteverde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lucía Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, University of Wisconsin - Madison WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%C3%ADa%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuc%25C3%25ADa%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pedro Blanco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BBlanco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanco%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BBlanco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando Pérez de Vida </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BP%C3%A9rez%2Bde%2BVida%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20de%20Vida%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BP%25C3%25A9rez%2Bde%2BVida%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan E. Rosas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Programa Nacional de Investigación en arroz, Instituto Nacional de Investigación Agropecuaria (INIA), INIA Treinta y Tres 33000, Uruguay</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Unidad de Biotecnología, Instituto Nacional de Investigación Agropecuaria (INIA), Estación Experimental Wilson Ferreira Aldunate 90200, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BE.%2BRosas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosas%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BE.%2BRosas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Victoria Bonnecarrère </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Unidad de Biotecnología, Instituto Nacional de Investigación Agropecuaria (INIA), Estación Experimental Wilson Ferreira Aldunate 90200, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BBonnecarr%C3%A8re%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonnecarr%C3%A8re%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictoria%2BBonnecarr%25C3%25A8re%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gastón Quero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Biology, College of Agriculture, Universidad de la República, Montevideo, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gast%C3%B3n%2BQuero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quero%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGast%25C3%25B3n%2BQuero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Susan McCouch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BMcCouch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCouch%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BMcCouch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srm4@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">srm4@cornell.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1519&amp;rft.epage=1531&amp;rft.atitle=Integrating%20Molecular%20Markers%20and%20Environmental%20Covariates%20To%20Interpret%20Genotype%20by%20Environment%20Interaction%20in%20Rice%20%28Oryza%20sativa%20L.%29%20Grown%20in%20Subtropical%20Areas&amp;rft.volume=9&amp;rft.issue=5&amp;rft.date=2019-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Monteverde%2C+Eliana&amp;rft.au=Gutierrez%2C+Luc%C3%ADa&amp;rft.au=Blanco%2C+Pedro&amp;rft.au=P%C3%A9rez+de+Vida%2C+Fernando&amp;rft.au=Rosas%2C+Juan+E.&amp;rft.au=Bonnecarr%C3%A8re%2C+Victoria&amp;rft.au=Quero%2C+Gast%C3%B3n&amp;rft.au=McCouch%2C+Susan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493104/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493104/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493104/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493104/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493104/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493104/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493104/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493104/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493104/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493104/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493104/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493104/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/5/1519" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/5/1519.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/5/1519.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/5/1519.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
