<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/7/2301" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/7/2301.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/7/2301.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/7/2301.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/7/2301/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/7/2301.atom" />
<meta name="HW.pisa" content="ggg;8/7/2301" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait" />
<meta name="DC.Identifier" content="10.1534/g3.118.200336" />
<meta name="DC.Date" content="2018-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Legarra et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Bayesian models for genomic prediction and association mapping are being increasingly used in genetics analysis of quantitative traits. Given a point estimate of variance components, the popular methods SNP-BLUP and GBLUP result in joint estimates of the effect of all markers on the analyzed trait; single and multiple marker frequentist tests (EMMAX) can be constructed from these estimates. Indeed, BLUP methods can be seen simultaneously as Bayesian or frequentist methods. So far there is no formal method to produce Bayesian statistics from GBLUP. Here we show that the Bayes Factor, a commonly admitted statistical procedure, can be computed as the ratio of two normal densities: the first, of the estimate of the marker effect over its posterior standard deviation; the second of the null hypothesis (a value of 0 over the prior standard deviation). We extend the BF to pool evidence from several markers and of several traits. A real data set that we analyze, with ours and existing methods, analyzes 630 horses genotyped for 41711 polymorphic SNPs for the trait “outcome of the qualification test” (which addresses gait, or ambling, of horses) for which a known major gene exists. In the horse data, single marker EMMAX shows a significant effect at the right place at Bonferroni level. The BF points to the same location although with low numerical values. The strength of evidence combining information from several consecutive markers increases using the BF and decreases using EMMAX, which comes from a fundamental difference in the Bayesian and frequentist schools of hypothesis testing. We conclude that our BF method complements frequentist EMMAX analyses because it provides a better pooling of evidence across markers, although its use for primary detection is unclear due to the lack of defined rejection thresholds." />
<meta name="DC.Contributor" content="Andres Legarra" />
<meta name="DC.Contributor" content="Anne Ricard" />
<meta name="DC.Contributor" content="Luis Varona" />
<meta name="article:published_time" content="2018-07-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Bayesian models for genomic prediction and association mapping are being increasingly used in genetics analysis of quantitative traits. Given a point estimate of variance components, the popular methods SNP-BLUP and GBLUP result in joint estimates of the effect of all markers on the analyzed trait; single and multiple marker frequentist tests (EMMAX) can be constructed from these estimates. Indeed, BLUP methods can be seen simultaneously as Bayesian or frequentist methods. So far there is no formal method to produce Bayesian statistics from GBLUP. Here we show that the Bayes Factor, a commonly admitted statistical procedure, can be computed as the ratio of two normal densities: the first, of the estimate of the marker effect over its posterior standard deviation; the second of the null hypothesis (a value of 0 over the prior standard deviation). We extend the BF to pool evidence from several markers and of several traits. A real data set that we analyze, with ours and existing methods, analyzes 630 horses genotyped for 41711 polymorphic SNPs for the trait “outcome of the qualification test” (which addresses gait, or ambling, of horses) for which a known major gene exists. In the horse data, single marker EMMAX shows a significant effect at the right place at Bonferroni level. The BF points to the same location although with low numerical values. The strength of evidence combining information from several consecutive markers increases using the BF and decreases using EMMAX, which comes from a fundamental difference in the Bayesian and frequentist schools of hypothesis testing. We conclude that our BF method complements frequentist EMMAX analyses because it provides a better pooling of evidence across markers, although its use for primary detection is unclear due to the lack of defined rejection thresholds.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/07/01" />
<meta name="citation_mjid" content="ggg;8/7/2301" />
<meta name="citation_id" content="8/7/2301" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/7/2301" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/7/2301.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/7/2301.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/7/2301.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200336" />
<meta name="citation_pmid" content="29748199" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2301" />
<meta name="citation_lastpage" content="2308" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andres Legarra" />
<meta name="citation_author_institution" content="INRA (Institut National de la Recherche Agronomique), UMR 1388 GenPhySE, F-31326 Castanet-Tolosan, France" />
<meta name="citation_author_email" content="andres.legarra@inra.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8893-7620" />
<meta name="citation_author" content="Anne Ricard" />
<meta name="citation_author_institution" content="INRA (Institut National de la Recherche Agronomique), UMR 1313 GABI, 78352 Jouy-en-Josas, France" />
<meta name="citation_author_institution" content="IFCE (Institut Francais du Cheval et de l’Equitation), Recherche et Innovation, 61310 Exmes, France" />
<meta name="citation_author" content="Luis Varona" />
<meta name="citation_author_institution" content="Departamento de Anatomía, Embriología y Genética, Universidad de Zaragoza, 50013 Zaragoza, Spain" />
<meta name="citation_author_institution" content="Instituto Agroalimentario de Aragón (IA2), 50013 Zaragoza, Spain" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. S. Andersson;citation_author=M. Larhammar;citation_author=F. Memic;citation_author=H. Wootz;citation_author=D. Schwochow;citation_title=Mutations in DMRT3 affect locomotion in horses and spinal circuit function in mice.;citation_pages=642-646;citation_volume=488;citation_year=2012;citation_issue=7413;citation_pmid=22932389;citation_doi=10.1038/nature11399" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=Y. L. Bernal Rubio;citation_author=J. L. Gualdrón Duarte;citation_author=R. O. Bates;citation_author=C. W. Ernst;citation_author=D. Nonneman;citation_title=Meta-analysis of genome-wide association from genomic prediction models.;citation_pages=36-48;citation_volume=47;citation_year=2016;citation_doi=10.1111/age.12378" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=S. Brard;citation_author=A. Ricard;citation_title=Should we use the single nucleotide polymorphism linked to in genomic evaluation of French trotter?;citation_pages=4651-4659;citation_volume=93;citation_year=2015;citation_doi=10.2527/jas.2015-9224" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=S. Casiró;citation_author=D. Velez-Irizarry;citation_author=C. W. Ernst;citation_author=N. E. Raney;citation_author=R. O. Bates;citation_title=Genome-wide association study in an F2 Duroc x Pietrain resource population for economically important meat quality and carcass traits.;citation_pages=545-558;citation_volume=95;citation_year=2017;citation_doi=10.2527/jas.2016.1003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Chen;citation_author=J. P. Steibel;citation_author=R. J. Tempelman;citation_title=Genome-Wide Association Analyses Based on Broadly Different Specifications for Prior Distributions, Genomic Windows, and Estimation Methods.;citation_volume=206;citation_year=2017;citation_pmid=28637709;citation_doi=10.1534/genetics.117.202259" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. F. Christensen;citation_title=Compatibility of pedigree-based and marker-based relationship matrices for single-step genetic evaluation.;citation_pages=37-37;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=23206367;citation_doi=10.1186/1297-9686-44-37" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Dikmen;citation_author=J. B. Cole;citation_author=D. J. Null;citation_author=P. J. Hansen;citation_title=Genome-wide association mapping for identification of quantitative trait loci for rectal temperature during heat stress in Holstein cattle.;citation_pages=e69202;citation_volume=8;citation_year=2013;citation_pmid=23935954;citation_doi=10.1371/journal.pone.0069202" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Acta Agric. Scand. A;citation_author=R. L. Fernando;citation_author=D. Habier;citation_author=C. Stricker;citation_author=J. C. M. Dekkers;citation_author=L. R. Totir;citation_title=Genomic selection.;citation_pages=192-195;citation_volume=57;citation_year=2007" />
<meta name="citation_reference" content="citation_author=R. L. Fernando;citation_author=D. Garrick;citation_year=2013;citation_doi=10.1007/978-1-62703-447-0_10" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=J. L. Gualdrón Duarte;citation_author=R. J. Cantet;citation_author=R. O. Bates;citation_author=C. W. Ernst;citation_author=N. E. Raney;citation_title=Rapid screening for phenotype-genotype associations by linear transformations of genomic evaluations.;citation_pages=246;citation_volume=15;citation_year=2014;citation_pmid=25038782;citation_doi=10.1186/1471-2105-15-246" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. J. Hayes;citation_author=J. Pryce;citation_author=A. J. Chamberlain;citation_author=P. J. Bowman;citation_author=M. E. Goddard;citation_title=Genetic architecture of complex traits and accuracy of genomic prediction: coat colour, milk-fat percentage, and type in Holstein cattle as contrasting model traits.;citation_pages=e1001139-e1001139;citation_volume=6;citation_year=2010;citation_issue=9;citation_pmid=20927186;citation_doi=10.1371/journal.pgen.1001139" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. C. Heath;citation_title=Markov chain Monte Carlo segregation and linkage analysis for oligogenic models.;citation_pages=748-760;citation_volume=61;citation_year=1997;citation_issue=3;citation_pmid=9326339;citation_doi=10.1086/515506" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. D. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=N. A. Zaitlen;citation_author=S. Kong;citation_author=N. B. Freimer;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=R. E. Kass;citation_author=A. E. Raftery;citation_title=Bayes factors.;citation_pages=773-795;citation_volume=90;citation_year=1995;citation_doi=10.1080/01621459.1995.10476572" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=B. Kennedy;citation_author=M. Quinton;citation_author=J. Van Arendonk;citation_title=Estimation of effects of single genes on quantitative traits;citation_pages=2000-2012;citation_volume=70;citation_year=1992;citation_issue=7;citation_pmid=1644672;citation_doi=10.2527/1992.7072000x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. Kruglyak;citation_title=Prospects for whole-genome linkage disequilibrium mapping of common disease genes.;citation_pages=139-144;citation_volume=22;citation_year=1999;citation_issue=2;citation_pmid=10369254;citation_doi=10.1038/9642" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Misztal;citation_title=Technical Note: Computing Strategies in Genome-Wide Selection;citation_pages=360-366;citation_volume=91;citation_year=2008;citation_issue=1;citation_pmid=18096959;citation_doi=10.3168/jds.2007-0403" />
<meta name="citation_reference" content="Legarra A., Ricardi A., Filangi O., 2011 GS3: Genomic Selection, Gibbs Sampling, Gauss-Seidel (and BayesCp). Available at: http://genoweb.toulouse.inra.fr/∼alegarra." />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=A. Legarra;citation_author=P. Croiseau;citation_author=M. P. Sanchez;citation_author=S. Teyssèdre;citation_author=G. Sallé;citation_title=A comparison of methods for whole-genome QTL mapping using dense markers in four livestock species.;citation_pages=6;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0087-7" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=A. Legarra;citation_author=Z. G. Vitezica;citation_title=Genetic evaluation with major genes and polygenic inheritance when some animals are not genotyped using gene content multiple-trait BLUP.;citation_pages=89;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0165-x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=D. L. Lourenco;citation_author=S. Tsuruta;citation_author=B. O. Fragomeni;citation_author=Y. Masuda;citation_author=I. Aguilar;citation_title=Genetic evaluation using single-step genomic best linear unbiased predictor in American Angus.;citation_pages=2653-2662;citation_volume=93;citation_year=2015;citation_pmid=26115253;citation_doi=10.2527/jas.2014-8836" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=R. Maier;citation_author=G. Moser;citation_author=G.-B. Chen;citation_author=S. Ripke;citation_author=D. Absher;citation_title=Joint Analysis of Psychiatric Disorders Increases Accuracy of Risk Prediction for Schizophrenia, Bipolar Disorder, and Major Depressive Disorder.;citation_pages=283-294;citation_volume=96;citation_year=2015;citation_pmid=25640677;citation_doi=10.1016/j.ajhg.2014.12.006" />
<meta name="citation_reference" content="citation_author=I. Misztal;citation_author=S. Tsuruta;citation_author=T. Strabel;citation_author=B. Auvray;citation_author=T. Druet;citation_pages=28-07;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. Moser;citation_author=S. H. Lee;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_author=N. R. Wray;citation_title=Simultaneous Discovery, Estimation and Prediction Analysis of Complex Traits Using a Bayesian Mixture Model.;citation_pages=e1004969;citation_volume=11;citation_year=2015;citation_pmid=25849665;citation_doi=10.1371/journal.pgen.1004969" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Nagamine;citation_author=R. Pong-Wong;citation_author=P. Navarro;citation_author=V. Vitart;citation_author=C. Hayward;citation_title=Localising Loci underlying Complex Trait Variation Using Regional Genomic Relationship Mapping.;citation_pages=e46501-e46501;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23077511;citation_doi=10.1371/journal.pone.0046501" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=K. L. Parker Gaddis;citation_author=J. B. Cole;citation_author=J. S. Clay;citation_author=C. Maltecca;citation_title=Genomic selection for producer-recorded health event data in US dairy cattle.;citation_pages=3190-3199;citation_volume=97;citation_year=2014;citation_doi=10.3168/jds.2013-7543" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Pérez-Enciso;citation_author=L. Varona;citation_title=Quantitative Trait Loci Mapping in F2 Crosses Between Outbred Lines;citation_pages=391-405;citation_volume=155;citation_year=2000;citation_issue=1;citation_pmid=10790412" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=A. Ricard;citation_title=Does heterozygosity at the DMRT3 gene make French trotters better racers?;citation_pages=10;citation_volume=47;citation_year=2015;citation_pmid=25886871;citation_doi=10.1186/s12711-015-0095-7" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Strandén;citation_author=D. J. Garrick;citation_title=Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit;citation_pages=2971-2975;citation_volume=92;citation_year=2009;citation_issue=6;citation_pmid=19448030;citation_doi=10.3168/jds.2008-1929" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. Teyssèdre;citation_author=J.-M. Elsen;citation_author=A. Ricard;citation_title=Statistical distributions of test statistics used for quantitative trait association mapping in structured populations.;citation_pages=32-32;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=23146127;citation_doi=10.1186/1297-9686-44-32" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=S. Tsuruta;citation_author=I. Misztal;citation_author=I. Aguilar;citation_author=T. Lawlor;citation_title=Multiple-trait genomic evaluation of linear type traits using genomic and phenotypic data in US Holsteins.;citation_pages=4198-4204;citation_volume=94;citation_year=2011;citation_issue=8;citation_pmid=21787955;citation_doi=10.3168/jds.2011-4256" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=L. Varona;citation_author=L. A. García-Cortés;citation_author=M. Pérez-Enciso;citation_title=Bayes factors for detection of quantitative trait loci.;citation_pages=133-152;citation_volume=33;citation_year=2001;citation_issue=2;citation_pmid=11333831;citation_doi=10.1051/gse:2001113" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=L. Varona;citation_title=Understanding the use of Bayes factor for testing candidate genes.;citation_pages=16-25;citation_volume=127;citation_year=2010;citation_issue=1;citation_pmid=20074183;citation_doi=10.1111/j.1439-0388.2009.00826.x" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=J. Wakefield;citation_title=Bayes factors for genome-wide association studies: comparison with P-values.;citation_pages=79-86;citation_volume=33;citation_year=2009;citation_issue=1;citation_pmid=18642345;citation_doi=10.1002/gepi.20359" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Epidemiol.;citation_author=J. Wakefield;citation_title=Commentary: Genome-wide significance thresholds via Bayes factors.;citation_volume=41;citation_year=2012;citation_pmid=22345299;citation_doi=10.1093/ije/dyr241" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=H. Wang;citation_author=I. Misztal;citation_author=I. Aguilar;citation_author=A. Legarra;citation_author=W. Muir;citation_title=Genome-wide association mapping including phenotypes from relatives without genotypes.;citation_pages=73-83;citation_volume=94;citation_year=2012;citation_issue=2;citation_pmid=22624567;citation_doi=10.1017/S0016672312000274" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=T. Wang;citation_author=Y.-P. P. Chen;citation_author=P. J. Bowman;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=A hybrid expectation maximisation and MCMC sampling algorithm to implement Bayesian mixture model based genomic prediction and QTL mapping.;citation_pages=744;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-3082-7" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
<meta name="twitter:description" content="Bayesian models for genomic prediction and association mapping are being increasingly used in genetics analysis of quantitative traits. Given a point estimate of variance components, the popular methods SNP-BLUP and GBLUP result in joint estimates of the effect of all markers on the analyzed trait; single and multiple marker frequentist tests (EMMAX) can be constructed from these estimates. Indeed, BLUP methods can be seen simultaneously as Bayesian or frequentist methods. So far there is no formal method to produce Bayesian statistics from GBLUP. Here we show that the Bayes Factor, a commonly admitted statistical procedure, can be computed as the ratio of two normal densities: the first, of the estimate of the marker effect over its posterior standard deviation; the second of the null hypothesis (a value of 0 over the prior standard deviation). We extend the BF to pool evidence from several markers and of several traits. A real data set that we analyze, with ours and existing methods, analyzes 630 horses genotyped for 41711 polymorphic SNPs for the trait “outcome of the qualification test” (which addresses gait, or ambling, of horses) for which a known major gene exists. In the horse data, single marker EMMAX shows a significant effect at the right place at Bonferroni level. The BF points to the same location although with low numerical values. The strength of evidence combining information from several consecutive markers increases using the BF and decreases using EMMAX, which comes from a fundamental difference in the Bayesian and frequentist schools of hypothesis testing. We conclude that our BF method complements frequentist EMMAX analyses because it provides a better pooling of evidence across markers, although its use for primary detection is unclear due to the lack of defined rejection thresholds." />
<meta name="og-title" property="og:title" content="GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/7/2301" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Bayesian models for genomic prediction and association mapping are being increasingly used in genetics analysis of quantitative traits. Given a point estimate of variance components, the popular methods SNP-BLUP and GBLUP result in joint estimates of the effect of all markers on the analyzed trait; single and multiple marker frequentist tests (EMMAX) can be constructed from these estimates. Indeed, BLUP methods can be seen simultaneously as Bayesian or frequentist methods. So far there is no formal method to produce Bayesian statistics from GBLUP. Here we show that the Bayes Factor, a commonly admitted statistical procedure, can be computed as the ratio of two normal densities: the first, of the estimate of the marker effect over its posterior standard deviation; the second of the null hypothesis (a value of 0 over the prior standard deviation). We extend the BF to pool evidence from several markers and of several traits. A real data set that we analyze, with ours and existing methods, analyzes 630 horses genotyped for 41711 polymorphic SNPs for the trait “outcome of the qualification test” (which addresses gait, or ambling, of horses) for which a known major gene exists. In the horse data, single marker EMMAX shows a significant effect at the right place at Bonferroni level. The BF points to the same location although with low numerical values. The strength of evidence combining information from several consecutive markers increases using the BF and decreases using EMMAX, which comes from a fundamental difference in the Bayesian and frequentist schools of hypothesis testing. We conclude that our BF method complements frequentist EMMAX analyses because it provides a better pooling of evidence across markers, although its use for primary detection is unclear due to the lack of defined rejection thresholds." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
    <title>GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"g4Uqyv-mD5j--UQBtyKeQ4JmKnzmg1hKrRHp_64ojiY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490146","apath":"\/ggg\/8\/7\/2301.atom","pisa":"ggg;8\/7\/2301","ac":{"\/ggg\/8\/7\/2301.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/7\/2301.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/7\/2301"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/7\/2301"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/7\/2301":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cslyyB\\\\\\\/Y7VrGumtuYw769s3XBC9LnKmJ0WV5Pe+qt0y19tf7ryw0fGWkFQAna8LT5zqSNDGEeeXD7oGHLfk6eEjBJTgZe0923tmQfqm5AdhxV7GFzhIj81+AEClH7pnR3oicACk1dJsVZolGS14CGcS++NANghV7IjIF6fj40buyjFJ6BErreAM2kxEE7b0f7T1eqO3QggmjXrVBf4jj\\\\\\\/3vMyY3TR3w\\\\\\\/c0k+CaCYIz9I7MgcettErcNHaaF7QJ7XIsi7zbtyrHH+u6+M5Db7XBzTPa9YS1rwP7opJ7uKeL93Y\\\\\\\/yFKYA4O\\\\\\\/u10ZCdza6\\\\\\\/vNer+ISH8vUwRlnHhPbA2mhtKADC4TWmyv16as8zb1KeHkNwT78JKKFcBYDKJgnLmySdQDwsmJYp+nON3Nv\\\\\\\/7R3tZwGbIStaQG\\\\\\\/eiAuwIiZWeMXdIq7eIFTj5bKJCXGraZYCd6DdXRY5EmIjwv2Y3SE0wj04S09PNgPjj9iKxAjcIlEDJL9O0pE7a25QMJETjY851plcZehIFM9Ms8zGbK46ahQaBudVtzx86K\\\\\\\/yNWjkd7CK3umU+CiENE+EN8XoPkG5IjrNB7IF6Az9HgQtqkmoJ9kMaKeV4U\\\\\\\/X6z3vKQHL2TuFAyRp+grQeyZtysWdHw09PVSJyu7hYMKLpAOx9JISqn+JXJksV4nr5c8DFa5cYxagX0XHdWVM3AVKh2QZQWEaZKJcow+7kF1K7ttcYDtSQViwB\\\\\\\/u6OvzC15JBJmiV3WWiBdO1D7xHH24HHS+QAOJCd9ILax69yShawfTRah2qQ5z1Ux9Km8wqyOv35HO6MAmVLXvQoFk70cpveXSSem7zuir9FxzTZnciFihPhb5melAd18K15ejlxqG2PTSzkxPWaydHbFxnCmHkpzWFAsbyzY+71Vn1oguvGAyTp\\\\\\\/\\\\\\\/gum50DkfMnTUegoG4py6FOtqbpATsyf5Bimh4JLO+qSY2fgU\\\\\\\/FUv8cxY6X9WqswqcGDAd8ZIFh6D9hBHEU7fUwjSaILiGsSk5dsvntLXgCZBLzxjsXXUYkkOg3t5BAwdrQnJzx+GmOeXV1oJhFYCN3spYI2h4mAgCft+VsmIcnRlGGW\\\\\\\/g6LvpaPUaZazMaAiMzNBHTOa2gwJmhxyHaNpqe0c2Okfw0UMTw3X00Q8bR6F+eJyi8H7Q2KslB5Ifisnp08asg67HlaAbEv3MxpoACYFZcnEPOlErSDhWH+SKt82cB+a0PFJODNIB21vqE\\\\\\\/9360AMpCCuB9Po4Yt7mgNv+zGJqGh+hSrY+mM4GwPY\\\u0022,\\\u0022iv\\\u0022:\\\u0022dgfAbwCWWWmUxL2Likhaew==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b404c5531f5df771008e22178b18b6149b4bae48b07500e9915b28a943d576e6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022maRerALcXqeKVB\\\\\\\/iynge8LMETuaV+qloF5Dmxi9I9tpzTJeG4hQDfGMi4+SufSEEPGQwNV20SCQSGQ3UV2DAzdoBdk1\\\\\\\/r1Nmxm6Kgjk0wrPayhhnLze2e+BrAm2PmYyTKKI2T27Dys9BvZXCb5AZKC3V6sRm4tL+LVSWHq8sNJR\\\\\\\/i2EQsH6gbfB1DsSDZ6P6Ih\\\\\\\/pfxupzHRnim5bnh0iQlAqyhykY0wghBrutfAA0UxpKR4qp\\\\\\\/Ygs2Z2QE6czDuIgaNAgfS0maH5TehehoU5sdXEqFOoNdW0Wv+MUQkJsDajoON\\\\\\\/ubIipJk8S1EIeYcqTGIYvJRuf8JS4m+brBtt\\\\\\\/fJJgY54yVeSWjZrhb1ndl\\\\\\\/pPdqupa3SzZX+UcwevMugyA4rGAW6QCV1b98peGFoB0XdJBxWbefIvPuStGGFTHw7TFt+gdyI0Yo1r20JXd4rWUKVVyXVJf0Tb7v3gpzqEN6FjXnBl\\\\\\\/xdxhlvUWM7cWMLvx\\\\\\\/lCCumh88tmcpiegUyOhJzi7hJFuzyolEE\\\\\\\/kVWjLYm6bPWmv4UyEkNI6v\\\\\\\/r9PP6N7mECQpk+x8gWffD1lucTfCOQPvVarSq2HmKwKe+z\\\\\\\/K2TCXEzf71hYhogUGIyWPwIhfOoV8LzIzvV8RhjKumbLZCMfXKpzUKMmr1oGL4HvAchQTNSvEZVotMpSt4y2OVuE20vo47Vi1Aq6fJAwgvNX4lZczxDUc3GDdKpJd3ZTMmvs1N1\\\\\\\/SdJ5eqzjDGhOPVhznS2CJVn1MeRUOwoccENkSwllFyuwoFbKsSu3w9cH9vga57sm1D1KNOFzVz5ONht25JInlXPDsx4CYKeeUujUs3jbt\\\\\\\/bU6j5ngbSpOEdPOpztn9UjzT\\\\\\\/OWAco0C7UMtY+80L1WoWkvFHVdvcjUI9DgKdnnqqWkn5xRk5z36vAWFyT2fE0WXwKnxZUldmNjcjpi7bykQltw+GL62S+jwVihx\\\\\\\/mlwrp1hr\\\\\\\/HjciLlAss5Gl\\\\\\\/QLLbmZbJAQVDZd5V\\\\\\\/RdxdrorETxTRsJISOLd4nrCyvMUXFAgVzPiQvFwEGMhrGGz\\\\\\\/6AsyIaUBpn+74h1TdxH2llnZUr+ab2fum5AiHmQs8ow+vJpat9PncuQsOPyMgTTBoevWUo3gJy7xegR6bQtO243f\\\\\\\/J+ggElvVRszh4DVqxFGHqWyfNXPUxpJpWOhFOoebIa2SCL+4sywNlkwZighsVctTFgVYJ9mP3i2t5CPrkUyGselSebQg5adOwakTDCmfV+tsm8osSAZQwHZB7MUKFTzMPxOppa8E09WBYu+9tBNR4XtypNqpeNTsLEWspS3c4pHxcWgyeMKQ\\\\\\\/awE0vs8w79i6SGn1BUtzfeSwtSzN4mNN+ujsEVmUTAFbs0vkPnPY3FXsxgGGlNd1rmO1VQBaxcBY67BqQCoLZgiDnUPFiP4p9kl6GnhW+Hk1gEPnWiro95mEh7sBqBHMiiSWnmEM56y1lGix8FK0ohMMpnrjohvn8weFq4NTW02bKSyeuscHDe7e9nFL9XojRpeFJL7t1tNOTLs3helK4qBolaisDgvl+z85AJ84A8RbhPuXZ681Yj+8E3MU6FbeSGj+ggTktgit+vy5lBWmES4Jg5AO1K3IyApXcb4cpXtHNRdT9g6gNHMxpJc8hoYc9xKDhe+1w8t7p6zyzAld0Dbg4tN81Ysrg9MC+Xf9m7H5vJlJDzpBazcDEHuha6Uy6KwyG8weRNMpWsmY40tnOq28+0NcRYPVFNBsem2Q5AvQa6fhkzlD1fLFWOLgos21aKDLQHphIwlfGqXHxfODAfl7efRvsQ4+UqIm9KA2HQ1re+76WHnBWmfjHG7jkTWQ0Agunc+aRyCHdvkEFEWM652sBi6ZkEj6RdKGL4PsTBYvkeOWTmolBtgSgkXRzAheEsAWK9mSQ2KZcxxwobV4Q8bvO8etWlS6ooru0G6tEP3HIgp0AhUe2ljd4NyR0aNGHscJ\\\\\\\/2A7vAVYaJa1CxhAGS7Oxkrsw4LIKdZckCI5jCLn7IAKnl4\\\\\\\/Wyvaj7393f5iodGEbP2j9KU8McXof1DpQG2d4FUtN1jNFj4InVwccvpemHstFALQKzikxiuuYP3e79qqO3E8rYO9J3MTI50Y3v9+Hqa5ji0GXLn40GdND+rpaKtteVDiS6qPiRpPm3M3Opkn4BUCp7QOciSNOP9V9Uchta58ZaSLjx6M\\\\\\\/ogJa2TqMwRU00dIJut3UMFzsaNhiH1zZSEKlEf5AyksuXrp9PTni9WYDX9XxsGu9CqQOG0t2sAEewSFFRoXEUgAyFYqPD4FNOZwMN6G+EaBahEm7V27yaNvgEs0n73TWIAdTk+hErMyRtwxDQaPuPqIKXjCCM\\\\\\\/+al\\\\\\\/o8ur7aRakfB7IPFooBSq7MjmRiqxtHm64x7o+aqya+erGBqRecMMN\\\\\\\/NUMOzmgf6DgSNHSaD7+oXnJZL+023DdGNeCtvGyMCr00ZxeBhGx8LQ4e15zUmz7ZzzB6J+xfW1Ynn18OXUF+ae\\\\\\\/zpNofFgAsw+XllSGz6TKIuRxp3rUU6lnsPqVBIfhXAc7w8UxFB4uyzrPCCmE1K2Mpg6Hr31BNRCTuVABfb9EbPWxAMzFjePP8tysySdYqeEs5uePiAYXrSm\\\\\\\/eRKwHaihHYISa75+FhpJJ4gkebhp6+3B2Mb4P\\\\\\\/\\\\\\\/TrbZDTxTcq+0pW2DSWWk2rwDj3tSEmoKYa\\\\\\\/yZbfuUtvol9njo8mPoAaAWGI\\\\\\\/cugCUExIUO5G1lWpZlO603EnBFh5UBd2kUz3cBi4ul9o5W2hik421XzOga28lg1MHAwRReYWbSmCyve+u0OMLveCp4HzTKh6wlCs+XFVXTbRTYQekbCmin+s1f7EszpByIA1gbNTgbnbe2ZPboWyPfNvgQPQjOKIP9eX2fMCL7+tHypnbH4lEp22BU1R6qCoQtVtwM\\\\\\\/eLsAFw6ahOOVjmFL3R1SWyByR2TdLObpSXghKFX8xXkBH\\\\\\\/x04AoFIYq\\\\\\\/SaPJwU+dBtgql0ytGdfkd5wOZ25L4L0XS1lQAMpJxaRFeJQyhVWd6dXv45lWNsPAlTlhCSp9SkxO9YDjuHNfjo\\\\\\\/YR9C6Vohi9vDHTW09onP4AtbahDycGEmEui2kcrthGBdD3OAqrikL+17AoFs66v\\\\\\\/3CmW93bTyNHn5RHgJHw5JNCZyU6Qunac1KbRSKejdbLskrrMVdoDRTp+9Zo0PMxCtuJ+iE8Rbf1RaXcuVk3W4q6\\\\\\\/r0impogC42xfcPAwUzMOsLbGQ5q0m1YzGKoyXIroaDiNI5OGLDnwdUezBJvjGRajGUIKPqTOA7tLpebIb6P4APQTOdH0QcZu3mo5+KBZLbwi4s6afy69zQklcXFaGu2Gz58aRAY50rStcKFHntDrP7++DaVBhw6\\\\\\\/F+RI54AHpHluA+E8yPHgZ5\\\\\\\/Mmhmlg\\\\\\\/l7Z3C1VcMQnzOxUT7\\\\\\\/LCL\\\\\\\/9n6HpIuwUiQG71X8oseoDKi6W+yl4hmjik4JdYxZGZM6VY8uA1mbVZOZUIpUW6T+sNeRIW9cavyD1QOarr1OwvWHsjjQg33zmTsehOxPQGJ152amnutwU2raRNn3EvdnGzWSWIuXnEEtDunjMo+EginGPIdEt1YgNND9l8ZVlnzgnrFuqt6iF95dU2wz\\\\\\\/6WuXdsZqvrjmER\\\\\\\/1T\\\\\\\/VtnUWUtZeb21lhDkgdhiO8KNd+laray\\\\\\\/nrk6EVn\\\\\\\/PXTu+108HogdVbJdJPjw\\\\\\\/x4gbBZQvV9J49carW2P8cT9q+Fajq5E15fdiFW1chouybYA6RGq9rGbi2Sww\\\\\\\/2RnqrLoSF29zWmLZ3nDGVwV69YA9iY\\\\\\\/ti2WPP7q33kHX3wSduSSeT1HF8Mla1JZ4qTRKWCnexSRMA5wYV+rSrVcljCevwZ+FwL5AkBmHFXn9i934O+q83CFMYrKfGgOBcwy9SZzLvRGm9lH661aSPM5C\\\\\\\/EIT+TDTnxbioNq+CP0VJPMmT0Z092SNXbQRY\\\\\\\/YejtWBd+0usjnMCsrER4ZaDj76xPHy\\\\\\\/AOuakY7j8NkgrwjsI\\\\\\\/\\\\\\\/72EfUTRY\\\\\\\/YBMh65IWPmvxO+UR4kTizOdrO00WJiPyp3IgcNq9UZ0dRHZ1AqKmWntzHP12wYRKJvlrz9bp5JLm53QWbyFt1VPKvLjTkdbjF1Odx8xsfglzV7q7YzqAGZk3zDyV\\\\\\\/phO8XD4cLVok2pOIzbvKBoLw4Rk3yPP\\\\\\\/NsfTVkLcBOgXjy+NaFzwuOEm4voOlIxUvHcAM0i6WcwftEpIm9+8z5n7F+xOPq3vM8GiRh+vQqYc1\\\\\\\/iSJWyc6X51gUrpjF48wcs5B0ndcb9WIYEf\\\\\\\/iPDA9B6VeAr7BVF6clrosoJs9I5FdTDGZNZcV7UKLYmAMO2MAIO49KmZJbVrzXsLK6kiBBvpZD+tpkeumlt8ePnSLxtggjLgImBR\\\\\\\/v2oLjRfCK4eXhSA26AoI\\\\\\\/0wBZzMnBZ9XUY7krU\\\\\\\/VSkpsLCdBeE\\\\\\\/5Kfw1nRU6M8U1Lm\\\\\\\/u6MgEEP4oSYgHSq+NcPCOLe1jbfZs21RihMQf5zk5AedBk6lwK6t6oFgLOjnhln7imz3UY8bapwCm4Ya8LDuV\\\\\\\/gFLSaTgi4KFVdraPCfTiq4mxuXEcigoYAAaHIebO3lpQ7o\\\\\\\/S8IdHUf1k\\\\\\\/UOALdqRLmqpU5PwsDf00bsp1Cvbj6gTbXhu+yb9vAesMqY6jLdxYbJmNNEFQhZ1jCmNPzCqIwbJI6Qv2D5t4u+qnNRPCHZ6pTgks46UEOmdmB2LZ0\\\\\\\/p+NWkO80yzCPq2GGQlHkPFl4NYa49i2G9Bg2chOXaWgvBIPgYOok8flTpq9gnwKgLXH\\\\\\\/EcN5mLXyHoVPtcEx31Tczr1o2eWIPHnax1cZbp3\\\\\\\/ToSOlIgXh1FiVBkkjW1seL9AdAGo9gaFTUxTGvkXoLhgSW47fax1i\\\\\\\/IizfpPbioDiQ+ac+cqvCh6OVkL3wmHv2wkCy4yTqyhpYiIGrU\\\\\\\/O+\\\\\\\/rucToKnBR1ig+P07PjBaaq1DhR2C1TdJ7XDWup9M6sSGhnprrIKkSJvopLLxOdsk7hms0xNx\\\\\\\/EdcqJLk1uCz6qLFp3jgHLsFWyAbAljiY3hqWaAa1x0P0GK\\\\\\\/xBG6JY35lV1MCZX\\\\\\\/vMrTicQ+lTfC1xeQKWArXTZg1GAOqRybeg1hapolX0posqChYKdyjd62YoybBFcnEXa6ozopDTTtvxSMYjjCw9GBLd+G3Ev9lUTWbuc7Icw0zavzUPmsCnvrzuCDzXRCLaFnoTV+vUND6iqeBdtsG3IXyQ6EizjHPbZUu+0ODBUVWSEjOgRA2TMzUuDVhn40msULeMcBQlYUv85RsaoLv+5jXytZ2z8n1AQxM5q3W4SLM3yn36f36BYfnrIQuUT42YCQZAUbAEdyXUN66f35a3RuNaqSUAnW0Eo\\\\\\\/G3kK\\\\\\\/mF+6h\\\\\\\/Y565zmLV4+WE4MXGq9I3f6M7ogf\\\\\\\/0DZ4LPySLu4twN3WKfHTOI7VViiYo2bRl+pVk5aa3XJ3nU8U3evn3Sz0xYByOj3nGddVNH1YZWoc0wtuKgvOK5NgLpc0CTeLA5M8e9nuCw8baX9C28ND7P\\\\\\\/9VHuDvHMAx0BFeXiBPgW7SlTNBYWVmy8n7+zc6PSZGKTotd5z\\\\\\\/tJqWyASsK9AG5heklXEnVx8oYQ+SF0f6T901irq9ACjNg5+kI99cYGulHPdfclu5Rxc9OphAU0hRA6ElLnOgq5hWL0\\\\\\\/elhSNvAfLYPRO7CmHLpLosaUsgMwbjMMRzyDFlUcgAWoA6FTGxbJBWM1ZjkVWd96xxhUz3Z64xCuxnHEAGM8Gv6ptnZNfmDDu\\\\\\\/VAFVwR1m0dEUlmIdSTfb0r6JyBzMLDChEvpdk9yaXcRxDLAjQOD4TMma6+7IB4Pl3QQ69dJUgBgzB5zjdEi7g0MGlU4C7gHPCVU731kH12h8plae2nuqH6ISxJckbEnpSSdNieova9aDkD16qMDTG75TH3ONT2ifJiKOL5k0WGv+CltI+Xl0SS5Tc+l+4UJOqVWNe4cie9DtIP788VU5HF3ha2qRIxncqANRl3QvRlVNqp3jR6k0K8se6vkoKDGmx0Z6CG82PL5ylJBUo47jYQ1lKPfqhO4IpsvHSPZrDG+cjTTDnWEl5p9C\\\\\\\/RrloRO1a9jm8xy5boCjYgxRHy6NS+Huct6ic5AxTHmt3Zc7Iq\\\\\\\/liHVSUiNHmWi0UiHAAoLr13VfpK9LdTpavLKuxQg1drYN9MrnPhdkSMuI3Wcv1D8gDMqU6HF1E0V7Ly6zuLQn4BYIb7igkR3LHTcELDHly\\\\\\\/8v7C3lgD83KgKwPu9AOwaoG+yCbZqZgx87LS7aq\\\\\\\/TVTiwtjUzYE15xMl\\\\\\\/iUNWntAinlB1aKgH8NetdvkOUaFwQjdaoJYdFrV3ojFIkxCP7V61dKGrVOAQxYK5rMo\\\\\\\/LFD5FVD9P9FwA+dOIU9ddrkFHb9CJnOOTe1fekh3EGm46PezQCeVEP+eQf+hbuzuhQtw83cPvtQkTLXNW8SX7s9LqrLtiRBXxk80hUXaWRupwN9k4tLyq7ErUbDB8Yca5iOOF7kHMIswSP+xQhG8ryy97v2ahQw48LJlrpam1SNKUAKO12l+rIuB0UthPTQFDd9F3ZCLNL2PFZbBFk0lBmEhiOT9yyKNoD5qiN3LMJ0o6Ly85EZPwwQzSq3ByJAFtwNZRuYv6ZzH\\\\\\\/zzuWSYV6qt7lU524wOoOAFaaJnNwzGlQBz7H7aRv0czuiz+GEEcc6n6RsyIGhAc0JTT7AFiEwvH9d4HbJyN1jAlDhOLsxGZhYPYSBP5Mckq27njHKga8uA7MYjZ4UrjYhxETVF9Ll21cuyoRVhBeVo\\\\\\\/Nh8D0oH81O8AIuVOjV91XefXg1twbeNnuuKWcjE5DLdBkdnP28qRthB9KRWGUg+nW\\\\\\\/AjW76WlW2pxic2Z4cALb1+0VVcCz7a31LuixBvGU7zFz\\\\\\\/q1DW7w7YAASLJVOWJYnP27ad\\\\\\\/kEIITrMoxdL+Dtv2cvl7Y8HIsiXRSw5sIxmfNT4QI\\\\\\\/5ELotesGP9FSjdM55b8yp33UO7QweAvoJdH\\\\\\\/Vw4rmfSWka3jHYtesBPnxKxjkfyfrBE2PQSKLEpv0R87myeEN4XRSEzcyI3PV+Xqvnenbr3ufcpB7fUCR3Qn9D29uRmUKExdWGmtGX9DAQIPVv\\\\\\\/CuGYr6TdbE8\\\\\\\/D+uZ8XXF07wixBJ1wJ7WH\\\\\\\/qWJAXR42Q6c\\\\\\\/\\\\\\\/dlO8Ukg5JMqtMmiSzJQGRkL0IZHJ0Kg7\\\\\\\/d2dgM\\\\\\\/PPWf1ZLShZU0dpT9Mfh7mXV3LObfa0ZQKYAD\\\\\\\/dDLxn9hVq\\\\\\\/QDMdGLDy8gUZurE2pdQPyjWZ+\\\\\\\/zoAcOK6iwjhnKRuaRqd0boy8ozvw\\\\\\\/hajvgkv1igOFmfo88e7qsarDmvcAaDmsEb2YQkk5bh4lYbhXC\\\\\\\/PCkhE9Ltbc34FY8y98nOzRWwFuQznazxVlKejFGYmHulX9io1jmdRXRDB3Z3p3tj9b1g6GqdkBLBSS\\\\\\\/WSvTDQG2EaUW6raPkiRS7CEjixxUe+dH3D\\\\\\\/fURm2dsSEP8L1wK+yb8gK4Li1LEneqsEACaBcCvxkMIprpW18DutSe\\\\\\\/yRXCGmDP0Qfea8VxOGVb8hXPB\\\\\\\/+vl8ZW\\\\\\\/h2tVrBftmkfsnEluKfVjaP945hj4XI3qBu88i2mQtkhen2RKX6JQHOfVVn2f8VU1Q3z+2DSi9EqAJt2ZUrnzmnYDPN20U\\\\\\\/to51tYOFT2nSIjouJcJiznPoikM8DYKyM3PYt\\\\\\\/GFaBQeVOndWshMQLLmUZXyOtx\\\\\\\/7qsxR6GEprs+PUa\\\\\\\/Z\\\\\\\/n5B\\\\\\\/b\\\\\\\/Nj8ZafchqGOwkqYFzq4PhSaUinlFbXFIbUueCdhvlptLUvR46vO8gV8nLONEqTbquXGOK86OPLjY4rOxBddY4av97VsC0Havn\\\\\\\/AHf65CZeP+0g7OexNZCfUay4xFedCG9bmkRoHhVnztzsfYz44+EWG8utkRseQv\\\\\\\/DyFNUBdBfCi35Wr+hEer8FgRBlfejwrJtq5P8pkUG\\\\\\\/QGbH+qw6HkQMFbY+BE1JuJuOde8U\\\\\\\/+vQd41yzJ97lASPO5wKA6YTITyyfADrGangm3OdUiKX9lrdQbXwRNDrGx\\\\\\\/Sz7x1uIAW4GmzANEplRZaFJlYaM2iw0JKyop3Ale2\\\\\\\/eCg1antLxy\\\\\\\/eECoNq4hzor1tVPo86DvTWjhB+rYzKeKQji2DR7oCHezF1VPqPBOi2XfBfvd7aBwomI881+Y8f7AFaWYE3hNcCeYeUh0XGzRIyuPkMW6pttjzhCt+CRxrpl6fC5c1WN13\\\\\\\/MpfLeZrJr8yMWJGPaF09xcK24x+KcrMs9kEUE4pDWmD23nwzVhtxnSAiDkiT8BCPYU3NxtRi6aAC1xeZ8BocU2dDjQqiy6ZEaj7TU4GZYa08Q46tIRWYew6ubwk+xnGdMC\\\\\\\/boDUVt+yb05FMuD2myCpC8ladXznnFVpJHo0uodzpBu8Xe\\\\\\\/OOX6XFj9uy4WNKcaDeke5sivvJfi9deu4M4WX8BhtVh15gt1vmALmjQbDpPM6Lqo37RA4uUf\\\\\\\/zNpzPqXyyvNpr1Ckn32QDn3iVNRZhuB5WnhwaaYlstCrgV9SvEeo8c3URAmE69aRYwICkvm85dCrdGYbEuBAwlozh7hpacOv2JsguvTjspbEItuWHJbAvFgh7juWgrItrZG96nUdpQjyGEXbqBEgbwSPa1KvAcRWLtIJYrzQKUmhv+T\\\\\\\/Tfrxs3biFiTutlsJTjEtHIWIwtxBfVr1tyo8xlCKGFkFnyksuj9XhRa1BCmWHiWm7hYDRdk2DwAU58ULBQgcEn7+Bq4JFeVtPpxlwk\\\\\\\/ktS2wDIYt7YfRHHzbQm7E36H49WsOfWdxaLrIsE+8hK5fAxOhprcs1jbW9RABvpZVwOqX4qu2X2St5CoQD4XT\\\\\\\/kXwvtPoQ\\\\\\\/TaDkG7JXchqTSLjTooTM+hFjNfKtFnWn2rsVMM7eP\\\\\\\/PYkm4vv5a5ZD8Sk+If5lFALi8ph\\\\\\\/SkQxBuxukFlYpADSCByfvnb5Rq0thQIjc4rPQN37wPasH8Wfi\\\\\\\/4HIKXnYfLUEb1iEPpCS5o8aMxr3Vm9e0ItKBl+1isOztkCY47Sy\\\\\\\/Mua+e9aNjoHyNoyoAKi9wckH28Oo8JDXGAtGJ1nCF\\\\\\\/ZQUns9W4w7nlu2srCAMicW1lRFln4RLvzkQKNwiJka3G2D8AGOcMf15iRe7tzU1vtv4sF6WjmkWzIeGOhA4k\\\\\\\/dQqDMnBzswhBOvbnvg9SpZW0IGLtBB8Hhth+EB0LvGLYM81jbnKE+NrMot1NxNFZJfphq0YCWV2kVgRY7ZRf+cvGgz\\\\\\\/13ukn6m2RFDSkrJt9Zpgr+QTfuNlgUJTJjEW0dm87zL44ZAz7oVtdXql8eoEU8jLrz+4T+h2PxP\\\\\\\/eA8eOKkjEHOK9Wz5sP7bU7U1ixyKLLM+gbbZZxtx4bhEZhqzdbw2na16BFauR4SWNjAfM4Sv7qr74R4cVpTX7o1wGtYFl5pCKtpba2qq1Ze0cJeMjsNnkCnvt5kDwsS9jz3g7x\\\\\\\/1FGQqkZhD9+FTztDEG8POOxuzuM97eUagmN2t1A2Z+1MPUW4UdgIhWaI34n\\\\\\\/tNkBqrObh3O1CVoYwcgNVGuN+Dp7Zc9VBeCt3YEgzoqX7P0XxvORbtQsPahWIT4NGc0UYWryBxBKfDPc\\\\\\\/i9DuPn0Tq+Qxq2dcOTrgbm0k8E5P3jtAbV4PqsynnO+NDezi4KQh6uyRh9VaQPcSd4Cy6bvVukKGJYnYHqdFr2DkQJF2CGB\\\\\\\/jQh1MaiRgKY4fF35vr4skT4s8mE1NCaEnbNs9cBh6jqlklEKNzDyDLSPasGXhQpL7EjbZcMZU42Llj+qtayvx81w1D4Itq705r95wSQhoA2HDbEx1\\\\\\\/ZV4Qzz9i0CmXTmExddaxje8zTFv2NbW9DA5fY7NcM631YuGIlcWhLeWxRCTSyo\\\\\\\/JbcBLsO3H6T0WN\\\\\\\/13d3ZeEK9SlreXMm8fksVY50paw+UdEL5mPvzD+lg0FdnLJUCOXycuLPWpjdVERtgipUP+3rZ3kvov1FV0TJgllaCWfnauG0p6yJRXDyFqTFosWfRGnF2sb2sJJiQjpUyUiBeRXdAckzohLU8S8yShTgvW6Mi+qpWvxFMiBysvp9RDB7k4GCxRGnJfIC0Lif4b0Wjj5b02ecNuZajYZ71UoDyv6ay5eqFOy\\\\\\\/kGiYBWAtwEpxqDG47CxpXKgX9ID1HSdp1eXPxs7CQHZrDTf6HW9JRxQUPWXuWJYPl4xkf014ODjoH6VuYY7pulbrq3A1GAKd6h8YVZpKSv\\\\\\\/SZV9jJyiH0TitEbngBw1V0IqgUE2ZNSSSFb4zF2tJmxh8kCD35rG2JAwEdkScrkGWZOYb1e3Lyyh0NVDJWLNbdJ0YZ+B4dt2N+a\\\\\\\/rpFDkvy+YS71\\\\\\\/aJhkYQU37toGHJ06GNwdGn\\\\\\\/zn6y112VFqqAi1VEjRZhEXlw2\\\\\\\/HjOGVXaxqY1GpuM3ku4x5\\\\\\\/4nMZSKqTW0d5w0mdELXYo+85EzbqPQVpOotertrZYkbe75gPWXqD9sJn935qj8WprKWQidSqimwWmvgiBNngbnOkuGgb1FiH50BUgjwB8D57U8lBjI993XBt5mdehel93MaR035cvlcjFWkbWngiXk8nLf3UjNL99IKQVva9kvfTbwHi8HcexWc\\\\\\\/ylbD5kjlYcinOlRp2QnUwcQXvPNJlZHpqzgXEXZRfIEg3nmZqKSrr4WgNed0Wpk10zOJm86c098DnqvXKcQxQWqSPBfXKTZWj1+OpuoM3lyzohX\\\\\\\/6zHVtPeogd2W0nX2lJk6Go5SOUZ1oE7hwQ0M3+9M5KwVepAmoU2bralGCPOTkLwNNrJAbNFW9kMCY1tQTi6amMRXu\\\\\\\/NRnAyTR0bp3NQGjPcvqlFpjiFksw2hTBCAqTGT5hTZK45EdrICsDAUI2awrGdbme4\\\\\\\/KtRrqPUfngT2YT8aS6lPZeoYavZhZWL9lWv5MUl+Oc4qkFH99tzZ2SIWMKF3FtLWU80SJ9BsGQaAQ+Fyh8fvpyYNGqdYhiwH93bFiv1NGBires0NMcE4lTck4j7yBVdY9fx4BxWcdlQgT3wZ2LtLBR\\\\\\\/7YoK7CxPzCmRNfDBTOAs7cCU5v2s5p926WczghkZrBKTm9DeaPdcoUqZ9VJRgMfI140WocQsFMokcEa5nvYJjuampdAv\\\\\\\/QyRHneXR5G7WX\\\\\\\/mMmKXh9xK5MgHWGXNZSKiyggzp5yzUqV80N\\\\\\\/jMJ3TtgoL5K4HnfQai0LFnNYMe3uVpowZuSrSPZVFPa2CxlocG6dCUUOZNp6ckJg8Gsh4tUX9ZAQNgcDNopuOV+RFoBq+hQqo3n0\\\\\\\/etDr9hswzqQPahqtvMEtd\\\\\\\/v6WWWMRdULYlIvHGwmK3f1TxE3yznrwIozd2XuZSHW18BQ0LKLFHSTxRFPYp30OB18yilq2qMUC9dqpb5syplC\\\\\\\/B+U02c4HihEUxtKHwYkUuOKkMicksPtDlUJs12rkzPQkWioisS9D1hx1LTrUu24dmwg4KEhJalFJkKawqIKDe7UYniJuEcxnc1oeM4qCqhg7+Hkmbl31g4Y2phdQaPX15hgWezWMHInGpoOOKob1lnNrj1H\\\\\\\/g\\\\\\\/RXwwts+gZIpfqIN\\\\\\\/OTYsz7AdAixdg2JqYDJNMNS65OgivdtCjQWzoyT8nK+6jvnoIX\\\\\\\/y3gQ2ZCTfqhYJ\\\\\\\/MmPRC62AZU85mLzbqEM+gCUnHe1dtcCUtrwZclVCRkRQLREMUQDe2+pAicLftHkrtULs8IKP42rWOyVKKPynJ6mJ82iVIENwUKGz2Sfvc+RZgkaw2SBs0wNzOtfeAOmTWjq9ECD9YbfmrXxOWjbeat5VufEi3HqAOOu+CxWE+mAztn6K5Q6RXCOreqSzA1uYp3VV9zGD0VYmGAxgWotIT+IL+Qk97dLOUabrd1RSKsTsb6JoYyrqVHnJ0JjB6\\\\\\\/BfQAPLU2DLkQ+f+uGwXITpJXey7K5vshTLNpqBErUte86fLxdH7OJWJrO3OVDgsO+dQxyKpOlY1mCoWIbPKHbtvngzCc1bcJAqMd82KVgEVE6yUeBUMQcGW6XBcDRH1b37cbtAoa49ekCboEmclRvQqAutLw5DI2eUaVSDkX+9c6T3Et2Zf\\\\\\\/oIHNpA45p3MDgdLke2D6WpRMddl4XTVboF1Vt9NpUAqHBPHR7t94p1ozy7DmcpQS03eEYKr5RTYV8SzjtPWzcVg\\\\\\\/9SumQIwA1z5syY\\\\\\\/AgG+j0FsDyN0CGtURyuMB+JLWe2MAPxWPOmEoZP0waUgWhkEcWu8QLoDCjOM\\\\\\\/iZPQKQyq79QDTiN0yGIFM\\\\\\\/ZCS+moj1CsM\\\\\\\/zs02mvvpVQVxqSYMeoFlcGT+rtLo2NXSItAIovFSMYP7\\\\\\\/1Eqv6Wr5tWntberix5SheRV7AdSpwwntEisZDwuRWT9nKtoC9pBelVphxR7Pd0hrXYzHg2JVWO1kmTqIaf+1mI5IRnNRzX2wvmUnYRornXtb2R8eyTswyeZLwMZHdwgoaWZP34UKflOwPfs7qTco9a1E0G0bVcZe9Cb0m764AO6+Xr3+XgbI9iXOr0YrXhAGiO1o4kQFBs1NoIcXEDhsQKBKXOAvX+56aCHRwHAxUT3DI9Hsh5ewJBoLrcW5xK1XZbIuul4rMU+Hso0sxTo8dwyPZHONdYDZikNkNJUjbY98zQk0umeXZXCTiIdgd14Gyx7CYqxP59wSIHsJgW9FsSKsDUSeCceCErMMoESRL+ZGElXl8IG0sb7HT37YsF7KnjIXgoH3qReeCbcF1BikjOUWyh1d1tzVbQw5AhMeqVJmV1o9DNIhegywWjVYR\\\\\\\/ziE\\\\\\\/dpTM7Igogwvpmd4XZ6kToMOHOBWumPLFqrbWjAn019mmAKTxJw7tVgQaYLDSaGqytXH+MX0L+KERCsDESpuHysops\\\\\\\/4eQkWZjw2kHNDMrkloh\\\\\\\/cZHFrOkviJTx7KvELpstO3hdJou79Rx5OYnRedmGYVp0uVpFYTg9DDgcBsw3eyUh+qFRr68HXeQMwd8+QsM\\\\\\\/LtANmTfIH\\\\\\\/mmBs2blJrDa+gqPP7QFi2yaqwaxHyBAthjLMevdL1VQHt2TbbC6POoOOWDEzp2IYVJ\\\\\\\/pNyHgFjuNfFIXeFGYOms5lBLjnPpReL0psKaPc6VXD+e8AL0dIK\\\\\\\/pcmn\\\\\\\/UfqilNFYST91MG7ZNXN5ffjNCYkNTS8gq47im5DrCcX2+agrmgAAET5U+5DmtmpyVOEijc8Qss9A\\\\\\\/t\\\\\\\/iTiVQVougbZixYWYZ2zQkfpGFEX4489JufgHOV2oyDh9od1D+hZEVh+lq8k3Mg5jIaYgpNOVzk4iSWv+84mxwTWsuDKDzpL4VWhSovOV1LlCl4YON1LACgg+JXByuk2UwM\\\\\\\/Jxd7Vxkrw+p4TqELK2TTYc1TVFHGcQJSj1BwHaLwBs52dkzBxJEP+RZxouqSdIeCkFygMfA9hTgzRbVod0yooVmj5MaYLnxQwQD7ldOezb6qpBkNHT17aoNHOYg0XL1DMFpIDQBEhAR1whWLIDWOSY2DGyLzEichLsBnarQmw6x85Jgjyccu0BnCGy1oPoz4ST8VY252KvWN76r2V3cGx+rfCT7U8vg+dNXlBcyl+gsBThLaIn6hLOKPyaRv+\\\\\\\/ZQk7lcpAZB4mfWKsY0ZbEAUfkrGbbiiZyp\\\\\\\/Imr32ZrHM\\\\\\\/yGpuYv6kE3GFj+yqYQT19F1OAe+gvYfon3zKdR9hYnIXzhS8AAVuoH1r\\\\\\\/\\\\\\\/AyJmLxf7WkRASgJN7Una3zIgB99aHSO8Mh8Mfzrt1GqPwGgc4Bk32FwIcvfKpBl3eMH2j0nr7jCMRATxHQ+5QTljITPP\\\\\\\/J5kusI5fmeI6znDFtli1kalBJEHmBD0jWTS8tvbXpvJH8reEJGnZ0cgZaKHqJZbQ3opWqhHWKsCL1EMTl7D\\\\\\\/0geRKiuJ1rw5OILT2bqUbYGPmZXtki8dBtO2t6TuVO0hd1fzoR8MbGYbm7JJN7RXOV\\\\\\\/Tser3nCXR21YjTIXqAXWRzX4+8TFB9lPxKO4384PIapgEb65oXPvOPA0\\\\\\\/AKKrnrRaRe8w\\\\\\\/GqzxW23V8uiIgsLBSUbqCzOqrrQ+3o3tnuX73hOjamfCCGZFierbWHR1RbqeCIH4npS5o1AXes8MoqL3jghD+tTinT\\\\\\\/Of4\\\\\\\/9dc8Vs8TG3XRvhHyjO2utvuri6u8QGSdQMFxEHhXg+ZHLbnBTVSjYRxH0ZGuCregQxNfMnWIK7eYPB6w51LRMH+R7W86wl7xnvbZLnNl9rEOQ4oXOGltyybOOuniNj3HFq2s4WyJCV7wc999CBqWsEnFj8kvmd8yOKyRkBYRE8SdnjwEH5v2JcetJzQdp6Auo4iGicNJYGw+WPYHTNlP9H7xYen2LPozX6DC17a6V4n1DFv6Zavz4aJvhK2JoGGUusrht\\\\\\\/NaRuVQuCoczEt+sMuNZA6tz4wVtlVblBJAkL3c1j3VamQhi5Y9Me8vYygeIS4jvLDwmLoFx1XoWFwCKQ1pyZ02IQjc\\\\\\\/bzrD7mdaRK\\\\\\\/MhpJg+59V9V0yTtnjgJ4f12uadfGl3kmLl1BPMQl0alUeP5g4kJ886tqRKU6jJdgp7EQC\\\\\\\/C8P\\\\\\\/aZo0cR0wIdDCizPPooUHMfTqkBGP8RgzpC6NaPc7rue94djDsDvGnwkKLHmaFqn3mVxKepGUsb7XhbmYLFHiEn17wTpZCbybOJ3y2x29BiGhSQCDO748nuK\\\\\\\/SdQdn8xp+W4DnZOcL98v7CgTw3yBjpWHnPfwmuLCbJFoc7HFYowJmUUqt3OfYRNJl8i9xBk0+jfANn\\\\\\\/lhDDb8NKiqnEK3q6lZQYS7KyljhpV2MQq0IFmp8QM07t6LbbGlgTGtUx9c1U9nMZ4yCC9gRCn+CaQHzdXfgQ3KRwGKGFuqq8Pt6NsDx0Ecn2v6Xd2gecyEIfk+2w\\\\\\\/ib61iUKmFTmNKdIzMO5+H6DpTSIe6CBbo2M3oVJTjm68voEeu\\\\\\\/NbJ942FIBGakO7FxqPH8pMZhsm9l2ukzKcz3GbdyniQ2fyPu82JuJi+9MUeX02bcHtS68SaLiBFRvqyg48xRTtfMDYkTJF+jxLCQzbWFo21kmebi6Yejh1ivVH4fhZacibcZT3vyig0mMwn1TbZuSqRwBXqqyLcq2XhItdxLRAQRyiXnCRVuZCnikn2cXNlCCUh6YnGvuwf8N7FEj9awmdjmem4PgGPhx2TYizjOTTabRHACblyM0dSzcQEeitbWyDq6OI5c3YpNVlyT3ATAL70bL7+y5zTiX+5M420Rho4bbE71VnT3Ko4euAytgM3hMX1NYLw+p0Sl1LzzCbzmZDWJKmEdM\\\\\\\/YYHpfMOgod6ft2GRXbeMeQzse2bf15LKT6pCZhj69cSQFBMzTDjRy0cvsn66KPcYeKnO\\\\\\\/8hx4QrhHje4UKFddf2zNlcuKKbiBg75WEuBgvRvH4Vy3SZQtxb7gWr1\\\\\\\/8mP8oTaafZrz95jKw+pBe012e5sd\\\\\\\/Z\\\\\\\/i7EQBV94qyu6YSqGZ3AqNRMnvQS8lZGlOy8COVQeuifRf3t09juAN4BX7ifABHo4X7Bm4eQV3gsNvxMejl+TPTtGhct89MxEo27nanWMrX9i1GWbAEgqXVcDQUt3ZtUXE8gFCiwK9Mf\\\\\\\/1kgNtzk5QjbSweaBafF4Xjm8hOHKk7NZD1oZ7BGkpjwymi5Mm\\\\\\\/\\\\\\\/TXTUnIXJ\\\\\\\/Q5TSBcHzPRjhQjzdHyFQ4FhLJOsfmsDLpHSe7UUYyHLzTsjwNY8GT78fdNmoQANqOD17i17EYp+mdmojtno1+mT6S17MtBef+MRdRyeiEJk\\\\\\\/mFdlAnBUywRZVsHTv\\\\\\\/DvpewKSRl26Bya86FukfvoImOt+v+txU21dMdNmQmFBNyJWrGHPzsggBYtgHXmfjfApWOBxC\\\\\\\/KxMYVqIQWgxEWvNN6Qir3KujKUPDqvSPzU8O4C6ID4WwBqCvdrxTxA3K5IYQcGPyQcQ6ODCe7IVayPzRuYL1I5pD3u48zQzELehNL32\\\\\\\/\\\\\\\/0nD3lemtGpGpNhI1O9a7XLxnZN7+niQwuCb0yMSXB+Os2DEkv6vwXPPlMMhaFQTGYiOXxrWVr+ZQ8xrHAI1ijVk72jZNi1CH8H2FtA6z9RJjH4OIFTRShG3qUrAZpKvEQ5EKkmGqFm1TwgEEYuGvrdjkzgHjZWGAr\\\\\\\/5XRaTAWYZRg16l3OZRQ3OrfthrX\\\\\\\/RiDPc0iQhof2oPdmwRt8qEpDKcINZ4a+M2FsP7M7sJ04lFZvQLFjHoU3V6UIMjOETzrGCna8XMSW7TbrmeVFNnKe+fYxXXKzxSl7DLf0YZm4H8rtyqA6rwMhIgyI\\\\\\\/0xwsp6rxhjTQLGFCwnXtMyRdFaAY7wff4OSZRCRbg\\\\\\\/WeJkKl7zJwI3sceRBTIcsivUT\\\\\\\/5LMkSpDS4WwiSufeodx1HxlCrRIFgteoBjw97TMzfLqBO1PvFf\\\\\\\/JdcfbxQnE1tC+1vAORRIgES+w5tSzcahUq8BXQmWBeVuIgQ5Xyakj8qMJSjgLBBVJbXX0cWnONWStCWzOYtZP91qf2wsFr1kd2lgQrMYQgxjDoniCFzTmrhvV7\\\\\\\/g\\\\\\\/sTgYjz70e6fzb41nQ009bC6IyLUSv\\\\\\\/\\\\\\\/X5sqK6Fb6RkJCSjK9piu6KCOpw3rfi1gIXNj5JuwX7Si0hyRjim1KVUgd48gnJheY1o9lER0Q7EhSAbJykoGMlUOBG5hi74f2m4nVvyb\\\\\\\/z0qbY52WfGle0CBh57PO5rfURTNuDn9cZfZw+0TTn2jiaK7ShvKWD8abxB4r3JdKGwsoMFXEv+pd6ZdYR19T4SJAzCUF3Z6Dd8qL5aCKcHFxkOxop5BFvzbOwhhFNpu\\\\\\\/91zAn1g8\\\\\\\/z5FAxfCwy2fHg0clXDEWFhkGDqukknVqIkytm\\\\\\\/Dvi9r\\\\\\\/\\\\\\\/TFw5P6ajZuUkPMczkboI4\\\\\\\/p3as3AOttaI7NCck3Z\\\\\\\/Nng9Ohvw7fRgvA9R3bIDqAQ4aT1C9SEycM3H\\\\\\\/EVBOAGNLYvd3qzkhzeTMcZgcwyzaGGia+PQ4S2uMHMUb8PawYxGN1vneAGm\\\\\\\/ulagboeuCPt77ldqLTkYcroe52eVmSakxs3BelLMFVoVr1SpmH\\\\\\\/n45PrxWPAPPankXtyGhi42gwPVcv2bWlNjzEn0gYVSG+rg9cUvQIdtBLd1kvO4PRzOdOqTJO4rHH3Z6+Rdr27UkASJHNzCqO5Blj8fjP8cMKBO\\\\\\\/rDf9le3scEu15iQUtXIaTo42xaFfpb5GhtixvEwnLk7LuC4iU99tRbgbHLrOdjMysBNayDEbxfnej++pA9Qo1IaAuUIABEhVv72cD80M0KtNNzSfPsbN0vniLAUBoZaylGUeK1tTu\\\\\\\/9mB7+ENEJv7w7jOgnIIQVqL9M7p2bkZvDletBGNXlRIUu9Xxngysx8ZVDj9af1XGHs5KLPt8wn54RiJkwH7TdgZhxKvnGGsqsUdylXrRlDzvSJqx3DOBSJitFDNuNYpQX9BQ9bWUIExTmjNqXjAYTdbJvjDppH4PSKa0LTQFAyk0vZHxuisNP\\\\\\\/3Dh7mYXWbZZ3ALa6AC\\\\\\\/ByMkjHIPA596xjxBQOT3Ep+GyM7NUvDPHCiULlnBwMXVucku2cJMcslNL2ec6Yf4VeoKP49SCh2TSNQi\\\\\\\/p5UwIn5jx6FeNCcPdlC6x9tn\\\\\\\/9wxXm30tgNocUCxipp520MZL3DSZh\\\\\\\/N7f9YTpxkLptMzToR9Ic92Tad6JoFz38q1t49ZqGrzVuiKMg+LX7wbXA4EqvWlvVfAtcMNhChYt4gEodbhKwSFVW1mjQ8DMZ+x7o2ObwvtXSFCAlKPo5CFD2sPhD1wAVphUD9z7uKe\\\\\\\/63+oste\\\\\\\/6yykX\\\\\\\/dyLdkSuRWb63VwQ6loS3rrbNnKwMgEjLJXUFFB\\\\\\\/Amn2V7kQxbSuXiIZFGmlvZTyYAyHs16e815w5cifTKmf3bl1tiHI+sIo4lFcUfFF\\\\\\\/jIKvczJ7ltE5Y9kucY4OacugvN43dBi\\\\\\\/b\\\\\\\/B4vEeYCgZoivq6uTg2OeM005W5oWeR2N2RnZi\\\\\\\/HF+dH+1ftvUnfL2U5bToyPNmu09rrpcwVCxqwG7z0t7zCphIemk7tmxJvvUrtU7vKzfGUW+5TlOtRS+f4rB15Qhv41yD4FQLCMweLEO5bbL2tCwHThvjQK0t9Daln+9Q8VmYVe56+2k10c4IKGequSRir1WkbODpNmRaTwdSyCoHnoCS1rBNbY+KmwYKNdlNc6L4KwMZ80yvWRRFsx6COQX7giB2uFQc\\\\\\\/ybrVHMJhjWG5Hwl\\\\\\\/eyd+7ZWlKRLWQ6Wv6ouVR7dymDZxP9flbHM3B38+bsWZrfN5RhwOWUkLOa5u84JnIW+v6W6GfW2RIbLIaEncaTb3C5MXwOpPevpWCz5wWxkc0RBsg7q3igUSCeYeqOWTgfs8wzOREKGEJZ0srOZYfzzPVWKT0BWzA1VUnQ7xIsHPjSguFPbzlz1xeOqFHQsozdCw9YDvKNXzmYn42puSSyQgLwaxxJbcEm3LA+39GQF7KrGEfsbnRmaRmJN4U87BZYtWECZTNjQ70Xnzv\\\\\\\/REudLbbAhO4KuzTxTtjQ0P19CTz8mML9eIYhlXzg8YJUh8qdFxJa7KGx6kNeTZ8Oa8VPuIdHXXtQlbqpp6ZW3pc4xD3K\\\\\\\/jZ2Xal+sVk3JCf2pymsNdXRPRtJpjVmu\\\\\\\/3ZimFwV9pulCMdNS7Zbo2CxJNpXwxd5DTpVgiuOAeUNE4vCtxlDZiaU6yVn4qpXUvJ7\\\\\\\/iXo\\\\\\\/5awG9cI+eNnl7HvfBNyypcuo1jnN7uJykG\\\\\\\/\\\\\\\/WbqvVGZyw6PZVGn2dC\\\\\\\/YsLDJDgdBfcat0IjUi6MNw2AQWWo5pDZX2OqfTD1h\\\\\\\/lf8HXalPhJj9+Tq4ZuOyPsRUU6hHabE4STZi3ie9xEeB+GoJ3lboxflPtq4wxDW95DFSLTP\\\\\\\/nwynSDfYssakPZpZOMi4VDs9wnZHfGj1VCRYxL80nySoZcTmnbfsa6mjHukTBzQZe2+aob4fFO23T3sJKqQ+LnqEoVDrcuLl7ObYP32v3QrAY+SVqB0fqESmBboic1DxvBwNejNi38IvAC8RXImU4UK\\\\\\\/wDpYarUFpYZ+JGW0gZW5RwycEMEcDEb2IgGnXkR\\\\\\\/fuQBJkzPWhAusnSWbtaFyAddBy8sb5uOWCI5zPggwcS3ywcyzYUD3Dt042jg0vF1QPAalhJU3UEOaY99bynmMDaAuGyBlWzm+le2LReG9c69rIArMejL2qyF3k45gEOGbi+fdmMM1xOMmMc9foVgsgIaUMTfsevDiVJXhW\\\\\\\/z+LYeOmkZYhmkIK2DR1HFEiJI0WE26G\\\\\\\/xqXIAYfgb+tRtwM6LUJl7LHybOp7KD5tT9uY7wwvqDbzmJK+BbbjfSYne1BOR5MUloU0NwcYtKQ4yt9HKeWJ5U2PMJ1ZCsSmRiscpk9SLF5mnttgx2LZbiscbsv448Jegc8au7gzRkn8P0j7LLFwib\\\\\\\/aE3QYmf7jEU2VceJKcSDrWxdQKE2hD\\\\\\\/OGGnurSUgZ5tarW6fmdn2hw0SDiCs7ShAOFMyURjO3cCrSaSMDpvCYs0CK6p\\\\\\\/r8trIeIt1\\\\\\\/L\\\\\\\/zccNblrAv7E+\\\\\\\/5uElZe4iwU7hanVeiUXlIXUq4dPif5s40ybR56yHsohTXUYMX4yDiotGY6UgxmzDPLMOJlOwQdQy2RMlV1+AwLkYpoTTh7vhuIaFxzeHiKe0Wc0Rer3Y4WI2g2n0eh1XZkhZ+3SEJcbP6eg9sFOmc\\\\\\\/BXfnwZVPsag7Fq59qK9ggUOFWyrlCLjf2LTT8GcBLdRXJ6HHBSnQASMD8V6ed+qbSibxAazW2ENOPZ8bSoHyIP4Ul4rsz9JEzo9Mq54Nu20BQ\\\\\\\/aTaFRPyjcH6oHyFlHperWVVLB3uwHVnp66mv+xco6liaumCdEcDLl0P1iCMjTKeZfU4TsXy9DCcl4q4Nc9eo\\\\\\\/22dNCaXr9pheMi+Q0Y6K1XeJapHZYlsGvQ\\\\\\\/3ckktpxkulmZMm3D\\\\\\\/ng6vNnnNeDvPXBUsBecQw\\\\\\\/GgR2fl1EsK2ZwRTIGQgphljbjtU1scrzHpe7uUx8\\\\\\\/Jc3s3Xf8DE1tyNRYFYPaGGa8TL6yv9SgyyAhAUM9\\\\\\\/KoNbmnFR79ev\\\\\\\/\\\\\\\/gIpRClYElGhq0qt30y23HHNeafMPNt8qxiyD\\\\\\\/tBBZtZhTF+NnZP3jREoqODuuosH9LejCXU4H14ok8S92pZXovpwYhYMM1SwtmW5FrxarBvuVaAPolTLNxmftwsCurDK0rL9+xWvKi4+YiiLzykwVcC4be64m0906J8MEWGHxqFq7CzW\\\\\\\/h8V1jB+m3ZlpxGV78p44jTL5Lganf9drdJ9YLKcG2CWmbmoVxFJVxtdsK03\\\\\\\/OhoZ16jHM5XLT9mRVO8HSURxogFEcqTNIad+0BuSu3iA1l+qISmB8OQLVXve7uh8Rx8WzbFIs9g\\\\\\\/xEdSWK11KjvT8g7sDqLV9uNWjccdI+K+Uz7OoaWffLqh5RbwwqMMj6RX9dFUuV6QLV4uLv7E3DO8XlySFZnhNtvSOxyUKzdklAR1a\\\\\\\/3ybS3vCx+v1lX\\\\\\\/XxASYXONT\\\\\\\/5utAd7JnOGs9CUDOPn707d7f3Q739A3NAlObKPkKO\\\\\\\/WQaaD0RW+rjb4y7tplwkEEDB0vnrwXUdFvyKxlBZxEYo7BgkzBf9Nc3G5yqQZ1souaP5THw3Rwo3zeSW2W3HewVl\\\\\\\/aZhwLaCOsnG6NmqaLl4ywhOff+FAsBV3YhdQjNX\\\\\\\/6aMgsi6mBIVOlOx0AMJIKn26O+PW6nif1sdFzFEEbCC4qhby27WHH89DSzZX6uuEjeycivQpiVIattk4zMQQ\\\\\\\/wr5f96RkmBmIMG4AHefp83bOnamqId27SqdRMcnUQJUEccqAsDvaY0dUpL0owpCAJ5MNuH90jmYop\\\\\\\/GDDVyVwJHPhHHUz9bHcECJl1oYR95zKlbwpspbWbn\\\\\\\/TWCQuRFaUIW\\\\\\\/x82tKOrZXa74m+pLVC2iyyD\\\\\\\/pkZRzauGm70FLWErdjK3JlG0x9\\\\\\\/5ZRUb3CVFWtMqCpAfGVVYTCpfSnzQlPYhxhtXZHkl7QICaXGT0\\\\\\\/jwyoA91Qb\\\\\\\/qibg8u0M0hL4Zj\\\\\\\/G+Eakmz1UQHmso30tAC\\\\\\\/Y\\\\\\\/p\\\\\\\/cLHiU+Als8V9gHQ+02MKJIHfJNyEbHe\\\\\\\/oSD46Siqqr42oze2N8Icq009Aflqft\\\\\\\/XKi6oRXnLafoKwa56tz5jlHELkGrAkBwMd\\\\\\\/+YA22+D6ojeBZ4q0+3+pLC5iSLlio1ViCanGbJBhwvFll12WQdw76rdymkl3azCt7MRVTt7+obV1vEGZPcughpTST+ka6Mh9AlA7fNnlmY7bQhOjp9Yn+tUVI9eo2+lhddrtfymczBB03jkLXYRRDzQsE6dqKx0xEtop4FNyqEkIKUb7JELB+eGt8596V4+gz4NewmKe0GrI3MNpVGsPcL45Xo6+HBPD6lQZ6MCo08bp8OKfZFj6MniGUA73ewFx0\\\\\\\/fyk2yI6eC3\\\\\\\/2FNwePIancTJSkoIbm05Byy5tUHNfoq5C6b1cZwfwjtjGkfbU0NMV4mJjWeGh5Yxl\\\\\\\/7iA76G5O9oekCssmk+lFpUfiutk+5pr5LuG+5hTedriClGcN4jLRCIwYR04gxuOBKRzUcbfaEwo+KLkYkpznWk24XsmRwDwsPARVo0KmlP0ogdLeTtpTzP4MyUAMECEh22Y3bWnRquHoR7YzI7WDs9r1T8L7\\\\\\\/BoQe627KrpdgsGbbdz79jHo7XC6V6Woxmx3tC2wLHtmBLyDCafn1xTfLrhSlSWIL8IgjAdV1TLQW15Nno\\\\\\\/gumb6pRS3o2e6xrBphh1vi4eTKBCvdTfnoLX3iVHy9Ge7QahZ1HNLG+SHlPLH1lMwgmqpz7yiqHkmsOYL2cZDIQ1q0+wCnbYW+ZmTo6jUXuBVs525O5EL4YOMPPB7bj+8bZb\\\\\\\/u0szFKa2y5vOER4qbZgyAsNjoOXLCNkQLN1Rthn+gQC4RculcaWU34KLb37E5HJP7XdoDrdtbho8rk3zjnijV4eB394v1HxhI2EPZcZ7dE55Cx94\\\\\\\/HbFIRhIKhiPs\\\\\\\/oO+jdzJukKk+sErWyTJ4QngvASoPAAozmfgHsPe\\\\\\\/qHjz+o\\\\\\\/UWtyEyQaOLRVcREu9VAEjunie8Onv5F+N5GgLC4A84MSgHpv5v9mMCNX0iy017mG8by6Hr7AB4Lapk3bx3ljtPmiTEaGb8PNO0KmcQSn+sCD9NXq6q\\\\\\\/atPXAqNrzGaghMFBKmyKhNHWe0T7XSzGgIjf7aopE7e4lVhyLto77E3TBtOj6OrTw5goBqHOjsZURDtNI84gh1CWvn71GILfAQAoIF3ZDTKdYO22i1q0A502bMaDMLR2rP9n2PVL8rbKqDyOwaAiWOQClMu2lWlS31nA62Oswsdr2Vyg19uWFAXasTd3KkbOF5PH6c8Ko8XFbi1g1ofRc1D3UtmLiddPkr6Oh7j+jpJwqPuM3huw6WamsjzTdPZO6zSYbbNeDagj3HVZ98l7DNmQESiiUcBy\\\\\\\/tm2kJUOat4MonkXeyvq\\\\\\\/S4O0YzrCscIa0UHc5sOv\\\\\\\/PxW30vFJfRkO5dIJEqDk1m3X0DuL+52Edo5Zok0oOWN\\\\\\\/v7+TF1HdkwlYylSrTSvgdzJFqNncXvnFTC8ElgU62UWdcVVGrHm8sYnY5QZqUBfGfn4tj5aIbBJgvfsjevz3jwglmdUB6AFjnABKzzzArrOTIVajmUn3zDEkOKC2bbOLfaujUZi\\\\\\\/yRRfrxVKfmpaJ+5O4e3ICVu95mMsZjy5m9CbcznGt9HNu+LrvqEyb791cQ0XjrtJwb3Gr0pGTVk4PF9P\\\\\\\/bEcCJ5YBFjctmC0KQQdjo3WtFiTw6\\\\\\\/fiKxKDw4H4J3WmdOFwCODhoPpBvEghnShNRS8zw5BCBJZMgCxyqiJyVe2WUwAiZ3UytDRYXYq7Jx8jAfEj4X79c+o1lqa5SJ59Gt2qqW2RIs1wZck180i82Ltlcj4B9V7ihdwMoEVwmptp+XqieXv9OXkebhkTeXy6OJrMRID4WqLo1psRAw0PtXGxE+yX\\\\\\\/oBWycDBX6XY4FlL7qqbFu8ssPtSlL0kk+5f+NQpOw4F9L0neXN90w+k8C3o0s4UrEj1WglwUta0id3yXkCSbG+eClPgcqtrTfddTRBjruzvuWqM9KSKDxeWNw+XC7jARcxmRk2OTTj5R2NCem5t8t0bG4D0+u8eRkiNRyW9NLkrP9WUKwgsQ9gzkKf9IX5N4eDXeEWZ2kW34drCy\\\\\\\/A9ACNmusimqZeq6eaiWDFPV09oakY8oB5bSaDEWONA7NUW32WCLAj7VEpYbsQe\\\\\\\/atXgEV17zsZDNVL9U9OmmBUcJlIJyWJHUDyESxWaWpEeMIBWAogK9WMG6+Yl3cMZWJbCOhtJ7jMCCmd4VwviRci7qIbBygMBChkiEQBGokio0d0x4Ik5MuCp6EPEb12GOUmrQWcAXOAyu6FPZv3Zr8CP3F4KK7ea1N7VY9886sZP+1jzEbcu\\\\\\\/vmmRflKkMCUV5eXYS7is9RYsHrPsfscb1PoMk60sLywQ3wpyQgtxJgOHHXEIH8lLowYi8nj9hqmfRXh3eK8NYBR3+f6r9+nGcdYCODHukXhQo4du0Iv3J4QWJnRRK2LaJmrQhwYW+ZJUIJQYa37ckWVCOWGVhEeYDe7DChRHSzziw+wgpWVP2hpgAbM3kHKVV7av1v\\\\\\\/TG7k\\\\\\\/Phddxg3Poj8X97ttrVAuo9qHapIDiQzVkxGmV+gQr8VHYqZFHWfxzCgX3NzLhNKSW5tQZpaW1lw2NXCuRgHkh7yLiiE\\\\\\\/8IASFKm3jHSK6c0s\\\\\\\/gDS44zk8jWFCwUSfKakfjPb1xYqKUIKLCGmMu2QoEPEQNd4O5lWhJOThvT2MrAHWqzmx+fnDVog6F9Y17IFvIVv6mdF60YsbJ6AibHAHblTwdw4mPNmBAs1yie+8aUdNN5tFZWF5B05E1r8R\\\\\\\/3WgxOwjASyuWosNE7vbb0phjuDAMO3z4OQMm0ln6qEkxiRJsjTOywvyIQDeF9ewl5TV0U\\\\\\\/aLqXaz+8KDkM4fxmkhrQL0XlF6krPA8FXrKaDh9xAOjY9E0jTGtlfAxNaSBHdCok0cQkXk1qpLHhnAMSDwYoJdbiX8jMJoyrzUtJ0025zhM0rbK+FWMPWUwPAr3p3AM9aIoJktOnmTk5FBk4fCk73k3HMwAVJLKgaZck9gIlhN79ZsvDWkpe2U8yoGNLRv8K1X2gOpKDYfKUOPWEhtOcEMJp+K3F6G05jd8N2J2TGPL9VAMHy9CzNZcjSlYbRNIGoxLanJzUNYqRgXes+sBOHn705phMOfOfW1mX+K5oRy8LPjPcZzZKNdIA7S6l8uxAhw8JOfVOg4Y35A6ydF5Ceo3aS\\\\\\\/A7JPAhoS\\\\\\\/81zf2Vg638KNPvZ79w9O+l\\\\\\\/dtHESy1wP2gE04RoQrLwdLKFd41uVyvYhLONwVr7HcjWOs8K1Xkhat6KpKAhQomIJ\\\\\\\/FF8j0hbMWhSPccnu9OxzDSX3iP\\\\\\\/2b0w5NrUSTFHO3q0jDOE51nfHcnb5M8PUqO+0SX+y7mcsDmuagc\\\\\\\/7IXa93McV0y7hZtyjEZUw8O7U+FsnFjB\\\\\\\/X4vszb\\\\\\\/h6OoTgkL+OkP6ccTHqFjF9yk4TYJmKHJa4+5wkn+KYxAcx8m\\\\\\\/SnfTjn8\\\\\\\/DMq1O3lehpj964jOaRtrZAE8n2h5MYNGb2N177kdzO2f8+\\\\\\\/N5dAeg863RUZb1Xq1MMmGaGMj6AAXT5UcIz8VzbOa\\\\\\\/sF\\\\\\\/Kxp1hWaDeqNmhtmordFqZ1Pf51mwk\\\\\\\/TVHjIpiGoMXwp09NDb6ZtwNB6UqRUaFDWCQ2WrMeVkseJTcvMj\\\\\\\/vl8CbGZbslJF8\\\\\\\/laQk6Hcs4WG2VfJxgR70LQlIFittOycYhyGaA6YgiO7vRdJCMvnMBqvfN9jjcMBRoEpYZHRzQdDJjApj7mD8HV4rno86TW4IRhnyZJW4psbKoXoB2bSSUwtU\\\\\\\/yLoOZAofMa50EIuJ4dH4fRPtWN8+lX6WUKuAvgxnTGEYB0+g\\\\\\\/UBhfLgZPsfobwKlwK52ILJi5HdFVkob\\\\\\\/L9Q+aZfrGeg3BwYPSZ\\\\\\\/w8Yx+zLB\\\\\\\/nIt7h0WE5UG24exIpti3OSPj4YSF4KXA4oJ2brWn31VlM+ijdFV3H5Q+lgmYtcnZHiT3QSFMsjhrrGuSWv1Pj+1r8dwMJhL0u0MbcnIhQKnEIg4cAP8e05OLPF6yq\\\\\\\/TRVPwYfOt4D8bUTLMwGNleYIEtTRzcuxPiiobytxol\\\\\\\/kQRnCIUAUMyb0f6NRyh8Bten4j+lhNcS0vRC75nvidrV2yMieMPMuQ7tN4z1+gaB6dcrzO+rjOHiWlRaHDy60iknZPrqZuzLKzpZZ\\\\\\\/T6w7NMgnMaVpiWaYbda+DrfhD1pQESgg1+w\\\\\\\/vMh7AeMvGcjHB0S5QWF95RKjrMZYBTVcnhHoPF9gHCL3lBdyrUrJozsgOK8TvwkHIqAfLXD5zlqx49rbeigucOpfGr2GrUnQFL\\\\\\\/aP91Hl8\\\\\\\/L64t3sIYMM9IXLD5UskrDdk+hF4z8EHrCXT\\\\\\\/9lCPCNQ5ifEyDTzLpyoJzFwsf5rzUKoK5kOEV3G+wVfWhGACCEg\\\\\\\/Qs3PxuIYCRx5cjD5YeYkx91X3BHDMa\\\\\\\/9I40m5iBlA0E5Ng1iOQRMDhkAEQoC6XzJWT2ezcHCVqILwo07sxzqssgDkgsXJ1trg01l75DI+kmk8aJ0EnItBksaPUpYxos0Z8C3avQ+M0oOeLCStF2sz0p\\\\\\\/t1PiJoBeJ5fUWXTwgo8xotxcEFGcjdqiy7R8IJ6q3z\\\\\\\/u7UDcOZLYci4uABs\\\\\\\/jseB2XmqXkvjvCTIMzWgkbZrur6vxOwRo9wM30uUajFOle56phKvyrrknGfF5vxKFCR51jHtpGjGXiA7HG7fUk8MgE8E2r5Gcw1URABcth45TDDKKfjtzw3gDEI3npxCmyEfD1BffYFFDF+XbdwhDzGE5twXII6daIFzK+\\\\\\\/2WE97IJz2Q3g3uvrsrk7MeRODqhpaOdJNFJbogPHVpmiZjfFXyVutD5WE3yF46LANNkXO\\\\\\\/dQ\\\\\\\/eKs4bqz03diwXev5nF+iLl0uE5xgTwtdCKO\\\\\\\/F3IgvEeQgo1VU5s3vZ+7yaLraEHtTRf4OtDpyNYZNtQl3BSFVKVpq6ZrjTvNsKoEHBHOQ0AikLMsgNOER4qOZLjNlJs4vG9FFYOJQpLH4K2S\\\\\\\/0DQkhkBhijwCmvsSRxTzf\\\\\\\/z+JJcaR25X68shJ\\\\\\\/1N0BQ\\\\\\\/QyuUcrOtAhCz4B2vYuxhHrcU+JMep5RnMXdEDCOOBM6qKdfON1LH+uF3DbsHsdbunauc7J\\\\\\\/ncZtDOL8psZpkjc26dbEXUSWpLF\\\\\\\/3X1FsMoO2wTjk3n5UusSfGlXNvfyKL09r\\\\\\\/8rntlxZTvcbklv\\\\\\\/go3t+g6X1BT76A1rOSLWLT4I6bBdpiV7Otc6WlA5hwMRmKsPoLwdOwRyYc4OkRxlQj1lZYdNXLDXQSN\\\\\\\/35nXzGREsRTYlfkmY6vqp9p2dCEyaBjUKm9EoDW4910DeVJh2EohYViKi\\\\\\\/\\\\\\\/kqF1rQDN0mkKrL5htqQxON7yAy1EGnleiQxNs16JnN\\\\\\\/ojr26qrQB9aK\\\\\\\/r5ftgMVB0xT3WKL7IKT02BmQ+sb60mMIb3PnBC21BjA8bhrBG5mIMX+JElJLIE2xTQTLOE27aq3fU4UQOv\\\\\\\/wpqIQmZS\\\\\\\/ADwKuZMczNnTAccJqkqgjK1mxEQGGu8qKxVJ4hM5sdQvQLRacpeXS0HM7GT4vZ0pOfei9YcjFpAu\\\\\\\/TtrGluBjDBzOn0l8gVeoeZ3dAYvYBwCEdBzNE\\\\\\\/MjrnvweQ091kVsYFYm+84Bh60rgUh8ivaI9tVNo97nfyEOZRS+ZCB8qwxVR\\\\\\\/U7mKlrpohP8A31nPTM85S6NmxssKfBDN5ee6anle9+++JUv\\\\\\\/mpBqif2mm2ZSQ7iuSKJcL\\\\\\\/8DGpqMXGFbviYPHF3ZzaPEllmOAQSpYTFx5RRjuMizhlrZjOIYELR7IIyPdmO4xrXYLz86\\\\\\\/yhsZRU6RNg0l8uRHa9fJZE49oVktR5VtQrQFueKK7eSBi5s7mqzQFDAlZyM\\\\\\\/F\\\\\\\/QDnL05GOAXHDN5tbKiDvjCZhf+FpldIub2XV5n3xfLAJ7wEPIJ9hy+8nJuayiUAomtYIxV7tZxvBB82Dit5gMFtXXJ9aKtKWP5xLEZGA9jkUR6HQJpGqu4WxvK825KSakMXdv7hddb3iWs6WQIs3J+XFGIXtCAoP+jYwIQu7C3LGXhMphYgwAl7QsGtKZtrOew\\\\\\\/UwPLN8HhU43ZgrFzbbbMk4CJp7JIt2bsvYaHZGtrfX6ndnFqxsFns5zEOtjciyjTuwC+5kzC+MEnKI\\\\\\\/uxMjoND9Re7y3a0vnLi1HJnUvLRIfA2R0j3iKjAhZMJVogrR6qU8YxzgWyVrB9tNDuLlobaQ0iUtSPDwIpdCCBUWdgPbpn6SM2j3jsNaBzpCS4I7tlOh7FyQaSTxFUx+PWMBUhyNu478ykG2CCo80C+r+KMkt12znXx0ypWF3Oe\\\\\\\/kO46FvBoGl0LRxMiFflUuqzk3i6Xtuv+m+5HlUMFK+ybaMq5ZFzOvOuwXCzioRl2e1RaP50zrOjBTBdXqC98TRYS+W9eIVJXxQ3Ux7fbFgo5W9zw4nlYW\\\\\\\/YmtP6kAxb+wegCotEK5G+4rqYs75vpum0c8JupEuqbzE1yfpvtYaTrEzEde9iUSRFk+n82wDizT+2cqxmzyvJ75Jzio7ehuGY0QduwAg6CUwY9n9gls0dJ3GzMLfWteTE8wvJEBhScQnZwOzwN6shqdHjGjjQ01rBFQ8BjduuWSp6tuKAjjlLYdxBqHjrWaTI4PpnAe7AGjJ7piRSBxynXKMSirk5AyzoLOMLy0lrQfvAjgl5A3V5Rr7HoG5wImpHt6otp4IhDY06Qa0qeA5b4ySLCtupWnFr8FMGJlAi3CX2kwjsi5isxYY0TuvGyRh9q81Kp05JWzhM0OsabRd2O5soXonYakx7Fd87exFP9bZogah7pbS8Cp25qmd3Fa3miae33ZrQv0HerIpLRv4oaL34iFnTRPdJ\\\\\\\/VdpLQkYvoRHmJItw4BVm3cuvO3JwSUw0bUoPC2pL\\\\\\\/d\\\\\\\/Set5xq1Lv47s7rsIm41nDJmVG8LqsHGZKXTKOZJDZZzeSCjoae2erEmLA+hAfDS+6b7TdRET+MKFKpGXL6pcN5PWQXoJfEqclukTAuI44aPWOnDWqUbkhbcz8JKQ00i\\\\\\\/PwLrdJkiJVMytuHJ8lro4NQhw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PypOqq0pXUpRWnhkAVCogQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e416637a54dab87d4f8bf836812d60a00c05eacc01162a106f3cc561a30babc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wVb4ykR3d2SHx\\\\\\\/C8PmdgOtCcuOEcfuvCQ6ivuqK6EbP9JrcpJ6js\\\\\\\/zOHdtJXDwjMX3cYabADbY9lXF7SiCg2LtXdNMiSRkwIc0pcAOVWr68DwFyp9AinJPDvx\\\\\\\/Csp5GW3pt\\\\\\\/Yxy\\\\\\\/PR9oXXZ+vpVTUv7tH2VXX7rkDGDJoI\\\\\\\/lmh542siZgUkwbiQHmGt9TFyH+9ON+9s86f33XgQ4wDOtC4goZg0KvCInzjBtDM6QRJdC1R+rx\\\\\\\/FjmJWjiwhGoug6dw\\\\\\\/2EENeZuuwvPAZBW9Y4igt165zPV4+NMAEjFm1tTDaBaz9ZzFIlb66W76R8VPebZPH\\\\\\\/J3tJyiuH0oNxolVAgVfCqrCNXpdeLSyvF1k5ExA6o94UTWLDfWr1bfWr6qkXTnlD00Z81jIUsh6Yx1cPUV7vKX1YUSKlWpkFbr3htIFp0\\\\\\\/huSylRreDrs5tibzhXJPGEJviTiMxcZdpgpMk8OC1IsnEd0EEXbukDw8LHLkj2bRhvTlv7H\\\\\\\/iPuXBSOozFC18ZKc\\\\\\\/AHq0Lp5hWhoBlkQ36dWLsrlspEns3kULiw733NxC4K2FHpqWi3L1ylJJVv8+D7+CqyzjF7mMnvmvwV+axo6N892ID1KfMD4HzoJ8\\\\\\\/XqU2rg+mDyVeczGLX8s0SfEiTU6XQSan9McH+0ITZDxhZVvCc4GSvYmauAE\\\\\\\/aiOLgIN1rg66Rykwzkl5h7DOc3Af4orrXb8y+NPsi1jPQYdW\\\\\\\/FPLDfRJRu6EpBvSQAyiqoT2+EfkisqGH09Cxuiu2FmlBFDHqZCRWwhH686NM31VsXnP3gXfAHQxnTO6ZQhf8gUfQV1TB1mQa\\\\\\\/CbDu9s7xZqun4yc+qu0Ez\\\\\\\/Y7Z2D2l0\\\\\\\/PqhtSkWFZ0viK1QJafY9L2xKgvwRsDzIeVNOwKXaitmdeST3iQ2YnrP27xvzCd+7KXquT+j\\\\\\\/FFt+yD+VGJNYiCEK1OtxZ7qPzhYa54UKtxMhfj40u\\\\\\\/ODEgvhd\\\\\\\/VfO6smj5GV645xddiWqXLChOvvQNxcYwSO4eUGhzjL5OyvjEUd3HJ7WJxh0gr5f3vvwH2qCCJiDAZ2JCgaAAe9M8Q3TKHJof6OqABufX\\\\\\\/xLhflStiuN8xulaV9xSs9hFlBMCBXEEpfna3SnZl6kTpN6z5WtuhRh0H0eBBwfCwyHZe7JqFUwHUwrG57mhsf3ARj8zWkKSN8KNW0+p2Cnn7mLg\\\\\\\/SXnfoZXrRrQ9lCb28VoLUX03bqvetsOz48akYHGGp9iTdQCzFhzlxADg8JAzATVUeuyB9B95ohsFk1455jIl2wNfdWJ5257o\\\\\\\/JJDDybsf4nTFCUTf8DPWWSTv7L5j6QvfIpVvGvPfQ2ZmMEM8f9d8wryuIxkWdePBkMemgW8x179WpRN\\\\\\\/zdmBIaywwEQBhNdc7WtYv51OV+2T47HrjPNqN2yN0huub5kkr9CHfqCeDiJ2OpWRWZDwQjzQdiUJ6owzOU8hQb5EWuRXJiLV9zaMFG3zOXu53kZlew\\\\\\\/d+5JbDx0rdjUBiE\\\\\\\/XTGNnVgghdcnQnKmblMj2Lkz5hGUwMQIlRLUY3jyVAAgyBmrfbrI\\\\\\\/ULVGeKrzs0QyodkQpmiYtg5Cj8XJFIak8XWQe8EcNv+L1d3VJr4NdSH9zSoP7W3qhl\\\\\\\/eFk8Bqcu7k+wAOnmIKA1kEXuioOBRERu8zRrGkmILbivH5sfF0dDF3iFSLetdNVQXP79mE2WldrJqvBaxemorWlxBiLIip45dI4Gy4l72gZ7yLkAtQf8bdlhcvsOdL9oVePuaZ3roTMH8uiXAfOpcn6v1QX1VKW5V\\\\\\\/7Dfinn39WEFna4fHhEofOXpHhZ8TM6NFv6WPnIfpBuoqzPS3evDJCn6c89eFKjHgKJkBJAEbeJx2cRJgUh7hjMel7VddoXm+uc7a5jWz6vHJfRtzCPbFpIIJArVmDqnCxuzjDpaPigzS7ZrnBU5uVO5x1kTkpmvrTX4w\\\\\\\/QvLcMhNaLb1iXv+PqlFLYZgMJocc3kDl4Qee3k393tqGhGc1NNyeyw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HaGmV9aUqgYvATI0NBMJTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022467c485b1ab3ebd473bcb259dbf894ea7278cf6c3d79b695f8ffb1804506b5b4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490146 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2301" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mNsu0IlEz9rNg8a8PPTCI2pc_brXgj_7ndQ_qMnXWAA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2301" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0xxjkVD2Ot5qtdMqh8ub7Au4Pdbi50munFZmgfkMP60" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/7/2291" title="Predicted Glycerol 3-Phosphate Dehydrogenase Homologs and the Glycerol Kinase GlcA Coordinately Adapt to Various Carbon Sources and Osmotic Stress in Aspergillus fumigatus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/7/2309" title="Genome-Wide Search for Quantitative Trait Loci Controlling Important Plant and Flower Traits in Petunia Using an Interspecific Recombinant Inbred Population of Petunia axillaris and Petunia exserta" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490146" id="node490146" data-pisa="ggg;8/7/2301" data-pisa-master="ggg;g3.118.200336" data-apath="/ggg/8/7/2301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8893-7620" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andres Legarra</span>, <span class="highwire-citation-author" data-delta="1">Anne Ricard</span> and <span class="highwire-citation-author" data-delta="2">Luis Varona</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2301-2308;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200336 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490146" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andres Legarra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA (Institut National de la Recherche Agronomique), UMR 1388 GenPhySE, F-31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andres%2BLegarra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Legarra%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndres%2BLegarra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8893-7620" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andres Legarra</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andres.legarra@inra.fr" class="" data-icon-position="" data-hide-link-title="0">andres.legarra@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Ricard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA (Institut National de la Recherche Agronomique), UMR 1313 GABI, 78352 Jouy-en-Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>IFCE (Institut Francais du Cheval et de l’Equitation), Recherche et Innovation, 61310 Exmes, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BRicard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ricard%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BRicard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luis Varona </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Anatomía, Embriología y Genética, Universidad de Zaragoza, 50013 Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Instituto Agroalimentario de Aragón (IA2), 50013 Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVarona%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varona%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuis%2BVarona%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/7/2301" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490146" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490146/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2301.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490146" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490146/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2301.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490146" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490146/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2301.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490146" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490146/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/7/2301.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490146" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490146/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Bayesian models for genomic prediction and association mapping are being increasingly used in genetics analysis of quantitative traits. Given a point estimate of variance components, the popular methods SNP-BLUP and GBLUP result in joint estimates of the effect of all markers on the analyzed trait; single and multiple marker frequentist tests (EMMAX) can be constructed from these estimates. Indeed, BLUP methods can be seen simultaneously as Bayesian or frequentist methods. So far there is no formal method to produce Bayesian statistics from GBLUP. Here we show that the Bayes Factor, a commonly admitted statistical procedure, can be computed as the ratio of two normal densities: the first, of the estimate of the marker effect over its posterior standard deviation; the second of the null hypothesis (a value of 0 over the prior standard deviation). We extend the BF to pool evidence from several markers and of several traits. A real data set that we analyze, with ours and existing methods, analyzes 630 horses genotyped for 41711 polymorphic SNPs for the trait “outcome of the qualification test” (which addresses gait, or ambling, of horses) for which a known major gene exists. In the horse data, single marker EMMAX shows a significant effect at the right place at Bonferroni level. The BF points to the same location although with low numerical values. The strength of evidence combining information from several consecutive markers increases using the BF and decreases using EMMAX, which comes from a fundamental difference in the Bayesian and frequentist schools of hypothesis testing. We conclude that our BF method complements frequentist EMMAX analyses because it provides a better pooling of evidence across markers, although its use for primary detection is unclear due to the lack of defined rejection thresholds.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/association-analysis" class="hw-term hw-article-keyword hw-article-keyword-association-analysis" rel="nofollow">association analysis</a></li><li class="kwd"><a href="/keyword/single-marker-regression" class="hw-term hw-article-keyword hw-article-keyword-single-marker-regression" rel="nofollow">single marker regression</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/bayesian-regression" class="hw-term hw-article-keyword hw-article-keyword-bayesian-regression" rel="nofollow">Bayesian regression</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">Bayesian models including simultaneously all marker effects are becoming very popular for GWAS analysis (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2012</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">2016</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Moser <em>et al.</em> 2015</a>). The most frequently used prior for marker effects is the normal distribution, known as RRBLUP or SNP-BLUP (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">VanRaden 2008</a>), which is equivalent to GBLUP (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">VanRaden 2008</a>), also known in the human literature as GCTA analysis (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yang <em>et al.</em> 2010</a>). GBLUP is simple and can be generalized to marker data missing in a large fraction of individuals in the so-called Single Step methods (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Christensen 2012</a>), and also to multiple traits, or complex models (random regression, genotype by environment, etc.). Because of the equivalence of GBLUP and SNP-BLUP, it is straightforward to obtain from Single Step methods estimates of marker effects for complex traits like, <em>e.g.</em>, multiple trait maternal effects (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lourenco <em>et al.</em> 2015</a>) or genotype-environment models (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jarquín <em>et al.</em> 2014</a>).</p><p id="p-4">Therefore, GWAS can be done exploiting results of GBLUP (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dikmen <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Casiró <em>et al.</em> 2017</a>). Most of these works (<em>e.g.</em>, (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dikmen <em>et al.</em> 2013</a>) do not report classical statistics neither p-values, whereas standard GWAS by fixed regression “one marker at a time” (<em>e.g.</em>, EMMAX (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kennedy <em>et al.</em> 1992</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kang <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Teyssèdre <em>et al.</em> 2012</a>)) yields a normal test, <em>i.e.</em>, dividing the estimate of the marker effect by its standard error of the estimate, with associated p-values. Remarkably, Gualdrón-Duarte <em>et al.</em> (2014) and <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernal-Rubio <em>et al.</em> (2016)</a> proved that in (SS)GBLUP or SNP-BLUP, dividing the estimate of the marker effect by its standard error is mathematically equivalent to fixed regression EMMAX, even if markers are estimated as random effects in GBLUP and as fixed effects in EMMAX. In addition, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> (2017)</a> generalized the single marker EMMAX test to a multiple marker test that considers simultaneous sets of markers. In this test, signals from neighboring markers are pooled to create a single p-value measuring strength of association.</p><p id="p-5">This paper has two objectives. The first one is to show that, in addition to previous frequentist tests (single marker and multiple marker EMMAX) it is possible to obtain from GBLUP analysis single marker and multiple marker Bayes Factors (BF) as strength of evidence for the presence or absence of a QTL. In short, the BF is the ratio of probabilities of the data given two competing models (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kass and Raftery 1995</a>) and has been often used in QTL mapping (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Heath 1997</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Varona <em>et al.</em> 2001</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wakefield 2009</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Varona 2010</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2015</a>). The BF empirically seems to provide a consistent procedure across traits and species (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2015</a>). In current Markov Chain MonteCarlo implementations, computation of the BF require indicator variables for “null” or “non null” effects of markers (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2011</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2015</a>) and does not include the extensively used GBLUP and SNP-BLUP. In this work, we show how the BF can be easily computed from results of SNP-BLUP or GBLUP, for evidence of a single loci or a set of loci (possibly contiguous). The resulting BF considers correctly both the estimated effect and its incertitude, at one or several loci.</p><p id="p-6">The second objective is to illustrate properties of these two procedures (single and multiple marker EMMAX and BF), plus a Bayesian multiple marker regression (BayesCPi), by analysis of a challenging small horse real data set with presence of a known, yet barely significant, major gene (DMRT3) for gait.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Distributions of marker effect estimates</h3><p id="p-7">The methods use the prior (before observing the data) and posterior (estimates and associated errors) distributions of marker effects assuming <em>a priori</em> multivariate normality (<em>i.e.</em>, SNP-BLUP or GBLUP). Most theory can be found in (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gualdrón Duarte <em>et al.</em> 2014</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bernal Rubio <em>et al.</em> 2016</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2017</a>) and we include it in the Appendix for completion. We will assume throughout that variance components are known; this is a frequent assumption that allows obtaining of closed-form estimators. In particular, variance components can be estimated beforehand (<em>e.g.</em>, by REML), or (making strong assumptions) they can be borrowed from pedigree analysis. In either case, a point estimate is used “as if” it was exact, which results in optimistic results. The main notation that we need is a vector of marker effects <em><strong>a</strong></em> normally distributed with <em>a priori</em> mean <strong>0</strong> and variance <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-1.gif"/></noscript></span></span>, and their prediction error variance <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-2.gif"/></noscript></span></span>.</p></div><div id="sec-3" class="subsection"><h3>EMMAX tests of association from GBLUP</h3><p id="p-8">This section is a reminder of (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gualdrón Duarte <em>et al.</em> 2014</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bernal Rubio <em>et al.</em> 2016</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2017</a>) and we include it here for completeness.</p><div id="sec-4" class="subsection"><h4>Single marker:</h4><p id="p-9">The single marker EMMAX procedure is a normal test obtained, in our notation, with the statistic<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-3.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-4.gif"/></noscript></span></span> is the marker estimate of the locus under consideration, obtained from a single SNP-BLUP evaluation (or an equivalent model), and where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-5.gif"/></noscript></span></span> is the frequentist distribution (over conceptual repeated sampling of y) of the SNP-BLUP estimator of the effect a. Somewhat surprisingly, the numerical value of <em>t</em> is the same as if <em>a</em> was fit as a “fixed regression” GWAS and therefore the distribution of <em>t</em> under the null is <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-6.gif"/></noscript></span></span> (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Bernal Rubio <em>et al.</em> 2016</a>). For instance, assume that <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-7.gif"/></noscript></span></span> is the <em>a priori</em> variance of marker effects. Output of the SNP-BLUP gives an estimate of the marker effect <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-8.gif"/></noscript></span></span> with a standard deviation of the posterior distribution <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-9.gif"/></noscript></span></span>. With these numbers, the frequentist <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-10.gif"/></noscript></span></span>. Thus, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-11.gif"/></noscript></span></span> which yields a p-value of 0.006.</p></div><div id="sec-5" class="subsection"><h4>Multiple marker:</h4><p id="p-10">Consider a subset of <em>n</em> markers (possibly consecutive), starting at marker <em>i</em>. The statistic is a quadratic form <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-12.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-13.gif"/></noscript></span></span> is the frequentist covariance of these marker effects. <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> (2017)</a> proved that under multivariate normality the quadratic form <em>x</em> follows a chi-square distribution of <em>n</em> degrees of freedom, which yields p-values for the multiple marker EMMAX. Alternatively, derivation of the Hotelling-t squared test, that tests whether a set of correlated sample means are simultaneously different from zero, yields the same result. The previous normal test for the single marker EMMAX is also equivalent to the chi-squared test. Matrix <em><strong>Σ</strong></em> takes into account uncertainty and collinearity of marker estimates.</p><p id="p-11">For instance, consider two markers with effects <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-14.gif"/></noscript></span></span> (similar effects) with <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-15.gif"/></noscript></span></span> (estimates of effects are negatively correlated because of linkage disequilibrium) and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-16.gif"/></noscript></span></span>. The quadratic form has value <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-17.gif"/></noscript></span></span> with p-value 0.27. The evidence given by the p-value lowers because the two effects are correlated.</p></div></div><div id="sec-6" class="subsection"><h3>Bayes Factors from GBLUP</h3><p id="p-12">In this section we include our original derivations.</p><div id="sec-7" class="subsection"><h4>Single marker:</h4><p id="p-13">There are two competing models in the BF: that the marker <em>i</em> with effect <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-18.gif"/></noscript></span></span> “has some effect” (<sup>1</sup>H: <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-19.gif"/></noscript></span></span>) or «has 0 effect» (H0: <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-20.gif"/></noscript></span></span>), and the BF can be written as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-21.gif"/></noscript></span></span>The BF measures whether the data y is more probable under either of the hypothesis. This can be written, alternatively, as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-22.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-23.gif"/></noscript></span></span> is the effect of the marker. Typically, this involves a complex MCMC integration. In the particular case of multivariate normality with known variances, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Varona <em>et al.</em> (2001)</a>, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Varona (2010)</a> showed that the expression (1) is equal to<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-25.gif"/></noscript></span></span> is the density of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-26.gif"/></noscript></span></span> <em>a priori</em> evaluated at <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-27.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-28.gif"/></noscript></span></span> is the density of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-29.gif"/></noscript></span></span> <em>a posteriori</em> evaluated at <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-30.gif"/></noscript></span></span>. Computation of BF using (2) is straightforward because <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-31.gif"/></noscript></span></span> is a normal density. In particular, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-32.gif"/></noscript></span></span> is the density of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-33.gif"/></noscript></span></span> knowing that there is an estimate <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-34.gif"/></noscript></span></span> with a certain <em>a posteriori</em> variance <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-35.gif"/></noscript></span></span> (<em>e.g.</em>, different for each data set). In algebraic form this is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-37.gif"/></noscript></span></span> is the density of x in the normal distribution with mean y and variance z. Consider the same example as before: <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-38.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-39.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-40.gif"/></noscript></span></span>. The BF is thus, in R code:</p><p id="p-14">dnorm(0,0,sqrt(0.2))/dnorm(0,0.5,0.05)</p><p id="p-15">which is 20.76 in the log10 scale. According to <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kass &amp; Raftery (1995)</a> this is « Very Strong » evidence.</p></div><div id="sec-8" class="subsection"><h4>Multiple marker:</h4><p id="p-16">Evidence from several consecutive markers in a segment can be pooled together using the BF. Expression (3) is generalized to several SNP markers (markers from <em>i</em> to <em>n</em>) as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-41.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where MVN is the density of a multivariate normal distribution and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-42.gif"/></noscript></span></span> is the posterior (co)variance matrix between the marker estimates. Posterior covariance matrix <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-43.gif"/></noscript></span></span>, which is a submatrix of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-44.gif"/></noscript></span></span>, takes into account colinearity between markers caused by LD. In this case, the BF tests whether a set of markers are all simultaneously 0, against the alternative that some of them (if not all) are different from zero.</p><p id="p-17">Consider the same example as before: two markers with effects <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-45.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-46.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-47.gif"/></noscript></span></span>. The BF can be computed in R as</p><p id="p-18">dmvnorm(c(0,0),mean = c(0,0),sigma = diag(0.2,2))/dmvnorm(ahat,mean = c(0,0),sigma = Caa) yielding a BF of 1.65 in the log10 scale, lower than the single marker analysis. In a way, this reflects that there is a confusion of marker effects.</p></div><div id="sec-9" class="subsection"><h4>Multiple trait:</h4><p id="p-19">Above methods can be easily extended to the multiple trait case. Multiple trait genomic predictions can be done from Bayesian regressions, SNP-BLUP or (Single Step) GBLUP (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tsuruta <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jia and Jannink 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Maier <em>et al.</em> 2015</a>). Then, the EMMAX tests or the BF for several traits (and possibly markers) simultaneously is very similar to the “Several markers” case considering joint estimates of marker effects <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-48.gif"/></noscript></span></span> for the <em>n</em> traits, the <em>a priori</em> covariance among marker effects for the <em>n</em> traits <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-49.gif"/></noscript></span></span>, and the <em>a posteriori</em> covariance matrix of marker effect estimates <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-50.gif"/></noscript></span></span>. Vector <em><strong>a</strong></em> can include either one or several markers. Typically <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-51.gif"/></noscript></span></span> is a function of <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-52.gif"/></noscript></span></span>, the genetic covariance among traits.</p></div><div id="sec-10" class="subsection"><h4>Data:</h4><p id="p-20">We used a horse real data set to explore and illustrate the properties of the procedures. We also did a limited number of simulations but we chose not to present them as this was extensively done in (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2017</a>).</p><p id="p-21">A single base polymorphism at the gene DMRT3 in chromosome 23 has a strong effect on horse ambling gaits (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andersson <em>et al.</em> 2012</a>). In French trotters, a SNP marker (marker BIEC2-620109 on chromosome 23 at position 22967656 bp) in strong disequilibrium with this polymorphism has a strong effect in qualification at the race (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ricard 2015</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brard and Ricard 2015</a>). In this work we reanalyzed the same data set, which contains 630 horses and 41711 polymorphic SNP markers. The trait was “outcome of the qualification test”, with a heritability of 0.56. The major gene was not discovered in this data set, and therefore there is no bias due to discovery. We tried the following methods for GWAS:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-22">Bayes factors with the mixture model BayesCPi: (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2011</a>) fixing <em>a priori</em> that only 0.1% of the markers have an effect (see (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2015</a>) for a full description). This method provides BFs, although our implementation only considers single markers.</p></li><li id="list-item-2"><p id="p-23">Single marker and multiple marker EMMAX tests: as presented in this work, computed via MCMC, up to segments of 100 consecutive markers.</p></li><li id="list-item-3"><p id="p-24">Bayes factors from SNP-BLUP: as presented in this work, computed via MCMC, up to segments of 100 consecutive markers.</p></li></ul><p id="p-25">EMMAX was fitted using blupf90 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Misztal <em>et al.</em> 2002</a>) and homemade scripts, whereas the other used our software GS3 (available at <a href="https://github.com/alegarra/gs3">https://github.com/alegarra/gs3</a>), using “OPTION Bayes Factor”. After completion of the analysis, we produced Manhattan plots based on BF and the other statistics; for EMMAX we used Bonferroni corrections to claim genowide significance; for Bayesian procedures, we did not address thresholds for declaring detection; this point will be addressed in the discussion.</p></div></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-26">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6241928">https://doi.org/10.25387/g3.6241928</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><p id="p-27"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows results from the single marker association test (EMMAX), the Bayesian multi-marker mixture model GWAS (BayesCPi) and the single marker BF. All three methods point to the SNP (BIEC2-620109 at position 22967656 bp) closest and most associated to the causal gene, and the four significant markers in the EMMAX single marker regression are in LD with each other. This reproduces the results in <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Ricard (2015)</a>. The EMMAX yields significant p-values at the Bonferroni level. Concerning BF, a threshold of 150 (2.17 in the log10 scale) has been suggested (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2015</a>), and the BayesCPi analysis in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> <em>does</em> reach this threshold, but this is not the case in the single marker BF using GBLUP.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2301/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results (from top to bottom) of single marker regression EMMAX, Bayes Factor for BayesCPi, and Bayes Factor for SNP-BLUP. Bonferroni rejection threshold in EMMAX is 5.9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1120190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results (from top to bottom) of single marker regression EMMAX, Bayes Factor for BayesCPi, and Bayes Factor for SNP-BLUP. Bonferroni rejection threshold in EMMAX is 5.9.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2301/F1.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/7/2301/F1.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2301/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2301/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490231" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child">Results (from top to bottom) of single marker regression EMMAX, Bayes Factor for BayesCPi, and Bayes Factor for SNP-BLUP. Bonferroni rejection threshold in EMMAX is 5.9.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2301/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bayes factor profiles (top) and p-values (bottom) for qualification test in French trotters, chromosome 23. The location of the causal mutation marked with a red vertical bar. Bonferroni rejection threshold is 5.9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1120190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bayes factor profiles (top) and p-values (bottom) for qualification test in French trotters, chromosome 23. The location of the causal mutation marked with a red vertical bar. Bonferroni rejection threshold is 5.9.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2301/F2.medium.gif" width="216" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/7/2301/F2.medium.gif" width="216" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2301/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2301/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490301" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Bayes factor profiles (top) and p-values (bottom) for qualification test in French trotters, chromosome 23. The location of the causal mutation marked with a red vertical bar. Bonferroni rejection threshold is 5.9.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">In both analyses (BayesCPi and single marker BF), a large number of markers fall below the threshold of 0 in log10(BF), in other words, BF &lt; 1. This means that for those regions the hypothesis that these markers have an effect is <em>less</em> likely that the hypothesis that they do <em>not</em> have an effect.</p><p id="p-31"><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> shows that evidence of the causal gene increases when using BF across consecutive markers. Systematically, the same location (BIEC2-620109) is spotted. It can be seen that the strength of evidence increase dramatically with increasing consecutive numbers, reaching the suggested “suggestive” threshold of BF &gt; 3 (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Kass and Raftery 1995</a>), but not the much higher threshold of 150 suggested (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2015</a>). On the other hand, evidence from EMMAX does actually decrease, becomes non-significant, and, moreover, the highest peak deviates from the true location. This is at first sight a rather surprising result that will be discussed later.</p></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-32">The standard test for GWAS by association analysis is the single marker association analysis (<em>e.g.</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kruglyak 1999</a>). Association analysis can account for genetic relationships (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kennedy <em>et al.</em> 1992</a>), population structure (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kang <em>et al.</em> 2010</a>) and also to a part of individuals not genotyped (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Legarra and Vitezica 2015</a>). An alternative is to fit multiple marker simultaneously in the form of Bayesian regression (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fernando and Garrick 2013</a>). <a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> (2015)</a> did not see qualitative differences of Bayesian regressions and association analysis over five data sets and species, and concluded that the interest of Bayesian procedures is to complement regular association analysis. Anyway, Bayesian regression is of interest for three reasons: first, the Bayesian analysis has interesting properties of automatically accounting for multiple test, structure, unbiasedness, false discovery rate and power (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wakefield 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fernando and Garrick 2013</a>); second, genomic evaluation routinely generates marker estimates and these may be used for GWAS; third, complex models used in genetic evaluation can be considered, for instance multiple trait disease all-or-none traits (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Parker Gaddis <em>et al.</em> 2014</a>).</p><p id="p-33">The analysis that we propose can be seen as an approximation to a mixture analysis such as BayesCPi. For a given marker, we ask the question: “is this marker worth being included in the model?” whereas we pretend that all the other markers are included in the model. Implicitly, the prior is of a normal distribution with a known variance for loci not being tested and a mixture of a point mass at zero and a normal distribution for the locus being tested. In a mixture model (BayesCPi and similar ones), all markers are scrutinized simultaneously, and the strength of evidence compared against the probability value that a marker should be included in the model (usually labeled as <em>π</em>). This is probably why the actual numbers for the BF are so different across both methods.</p><p id="p-34">We stress that the SNP-BLUP or GBLUP estimation is run only once, and its results are used to construct BFs for different groups of markers (consecutive or not), if desired. This BF combining information from several markers is quite different from estimating the effect of segments of alleles forming haplotypes, where a haplotype can be seen as a multiallelic marker, and where a different complete estimation must be run for each segment length.</p><p id="p-35">In Bayesian regression models there is a lack of unique criterion to define “relevance” of the association and of corresponding well-defined thresholds; see (<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2015</a>) for a description.The numerical values depend strongly on the assumed prior for marker effects (as can be seen in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Thus, two researchers fitting, say, BayesCPi and BayesA may obtain different results. The most popular procedure for genomic evaluation and Bayesian regression is SNP-BLUP or its equivalent GBLUP, both of which assume multivariate normality of marker effects. Most often, a reasonable assumption (point estimate) on the variance of marker effects exists, by a transformation of previous estimates of genetic variance (obtained by pedigree analysis or, using the same data set, by genomic REML or similar methods). Using this point estimate underestimates noise linked to estimation of variance components. Here, we present for the first time a closed-form method to estimate BFs for association analysis based on GBLUP results, and we advocate its use. The statistical properties of the BF have been extensively discussed in the statistics literature, but for mapping causal variants it has two very few relevant properties: the BF can show evidence <em>against</em> and <em>for</em> the null hypothesis, and as data cumulates, the Bayes Factor favors the true hypothesis.</p><p id="p-36">Our results from real data sets show that all methods point to the right marker (the one in stronger LD with the unobserved, but known, QTL). Classical regression analysis is significant and BayesCPi yields a “strong” BF signal. However, the BF observed from SNP-BLUP is 1.07 for the truly associated marker, which is very small support.</p><p id="p-37">Evidence from BF increases when we extend the BF to gather evidence from several markers. A multi – SNP test captures the divergence of the posterior distribution from the 0 vector, and takes into account the posterior dependencies, due to LD, between marker estimates. This is similar to the idea of using the amount of variance explained by each genomic segment (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pérez-Enciso and Varona 2000</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hayes <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nagamine <em>et al.</em> 2012</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Fernando and Garrick 2013</a>). The inconvenience of these methods is mostly computational: they require to do either Restricted Maximum Likelihood (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Nagamine <em>et al.</em> 2012</a>) or MCMC (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hayes <em>et al.</em> 2010</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Fernando and Garrick 2013</a>) to estimate variance components, and that only the Restricted Maximum Likelihood estimation has an associated statistical test (Likelihood ratio test), for which consensual threshold exist (such as 0.05 genome-wide corrected by Bonferroni) whereas the MCMC methods use <em>ad hoc</em> thresholds that are less consensual. Our proposal does not require MCMC or Restricted Maximum Likelihood, but establishing a threshold for the BF is still ambiguous. An approximate method pools information from estimates of marker effects (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2012</a>), but this does not consider not the error in the estimation of marker effects, neither their <em>a posteriori</em> correlation in presence of LD. Our proposal is exact, given a point estimate of variance components but does not necessarily require Restricted Maximum Likelihood or MCMC.</p><p id="p-38">The interpretation of the BF in this study is as follows. There are two models, in the first (null) model all markers have 0 effect, whereas in the second (alternative) model at least one of the markers has an effect. In other words, the BF is a contrast between the region contributing, or not, to the genetic variance. When markers’ evidence is pooled across contiguous markers, the evidence for either of the two competing models increases.</p><p id="p-39">Strangely, in our study including more markers in multiple marker EMMAX does not reinforce evidence, contrary to the BF. This is contrary to results of <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> (2017)</a>. The reason is possibly due to the not-too-strong linkage disequilibrium in our data set, for which p-values do not cumulate information across multiple markers. It would seem that, in our data set, it is more difficult to <em>disprove</em> several null hypotheses (null hypothesis in EMMAX: <em>all</em> markers are zero) than to <em>prove</em> an alternative hypothesis (alternative hypothesis in BF: <em>some</em> marker is different from zero).</p></div><div class="section" id="sec-14"><h2 class="">Conclusions</h2><p id="p-40">We present a Bayesian method (the BF) that complements existing EMMAX methods for QTL detection using marker estimates from SNP-BLUP or (SS)GBLUP from a commonly accepted prior (multivariate normality combined with prior estimates of the genetic variance) and commonly accepted, and used, methods (SNP-BLUP and SSGBLUP). Computations are reasonable and pooling information from several markers is straightforward. Based on our real data set, single marker EMMAX is better to claim significance, whereas multiple marker BF gives a better perspective of influence of LD on the result. This is likely to be data dependent.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">INRA SelGen metaprogram is acknowledged for financing in its project EpiSel. Project partly supported by the Toulouse Midi-Pyrenees Bioinformatics platform. Luis Varona acknowledges the CGL2016-80155 project of Ministerio de Economía y Competitividad of Spain. Editor and reviewers of preliminary and current submissions made very useful comments and pointed out the Bernal Rubio <em>et al.</em> study.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-15"><h3>EMMAX tests from GBLUP and SNP-BLUP results</h3><p id="p-41">Most of this development is in (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Gualdrón Duarte <em>et al.</em> 2014</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Bernal Rubio <em>et al.</em> 2016</a>; <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2017</a>).</p><div id="sec-16" class="subsection"><h4>SNP-BLUP:</h4><p id="p-42">The procedure is easier to be presented from the SNP-BLUP method point of view. In this method, the multivariate prior distribution of marker effect is, for a random locus,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-53.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-54.gif"/></noscript></span></span> is a variance component that usually (but not necessarily) is assumed related to genetic variance in the form <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-55.gif"/></noscript></span></span> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fernando <em>et al.</em> 2007</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">VanRaden 2008</a>). For several loci, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-56.gif"/></noscript></span></span> <em>i.e.</em>, loci effects are assumed uncorrelated <em>a priori</em>. A linear model for SNP-BLUP is <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-57.gif"/></noscript></span></span> where <em><strong>Z</strong></em> is a matrix of coded genotypes. In SNP-BLUP, the posterior distribution of <em>a</em> can be obtained by Markov Chain MonteCarlo (MCMC) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Legarra and Misztal 2008</a>) or from the inverse of the left hand side of Henderson’s Mixed Model Equations:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-58.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span>In both cases, it is possible to obtain (a) the estimate of the marker effects is <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-59.gif"/></noscript></span></span>, and (b) two measures of incertitude of <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-60.gif"/></noscript></span></span>, the (frequentist) sampling variance, <em>i.e.</em> <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-61.gif"/></noscript></span></span> and the (Bayesian) posterior variance, <em>i.e.</em> <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-62.gif"/></noscript></span></span>. For instance, if the inverse of the right hand side of (A1) is computed:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-63.gif"/></noscript></span></span>then <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-64.gif"/></noscript></span></span> (Bayesian, conditional on data) and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-65.gif"/></noscript></span></span> (frequentist, over repeated sampling of <em><strong>y</strong></em>). Matrix <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-66.gif"/></noscript></span></span> contains <em>a posteriori</em> covariances of marker effects, which reflect allelic frequencies (<em>i.e.</em>, a rare SNP is more difficult to estimate) and linkage disequilibrium across markers (two markers in strong LD will have correlated estimates <em>a posteriori</em>, and any of them will be less accurate than a marker not in LD with any other). If estimates are obtained by MCMC, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-67.gif"/></noscript></span></span> can be estimated as the covariance matrix of the samples of the posterior distribution, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-68.gif"/></noscript></span></span>. The R package RRBLUP (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Endelman 2011</a>) produces <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-69.gif"/></noscript></span></span>, and our software GS3 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Legarra <em>et al.</em> 2011</a>) can produce parts of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-70.gif"/></noscript></span></span>.</p></div><div id="sec-17" class="subsection"><h4>GBLUP:</h4><p id="p-43">the equivalence between GBLUP and SNP-BLUP implies that marker solutions (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-71.gif"/></noscript></span></span>) can be backsolved for individual solutions (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-72.gif"/></noscript></span></span> (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">VanRaden 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Strandén and Garrick 2009</a>). Proof is as follows. If individual effects are the sum of marker effects <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-73.gif"/></noscript></span></span> then <em><strong>u</strong></em> and <em><strong>a</strong></em> follow a joint degenerate multivariate normal distribution such that <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-74.gif"/></noscript></span></span>. Under the usual assumptions <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-76.gif"/></noscript></span></span> (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">VanRaden 2008</a>), <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-77.gif"/></noscript></span></span> and it can be shown (if <em><strong>G</strong></em> is invertible) that<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-78.gif"/></noscript></span></span>The estimation error of <em><strong>a</strong></em> from GBLUP estimation is more cumbersome to obtain. Assume that the model for GBLUP is <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-79.gif"/></noscript></span></span>, and mixed model equations are:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-80.gif"/></noscript></span><span class="disp-formula-label">(A2)</span></span>With the inverse of the left hand side equations <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-81.gif"/></noscript></span></span>. Gualdrón-Duarte <em>et al.</em> (2014) showed that:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-82.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-83.gif"/></noscript></span></span> is the element of the inverse of the left hand side matrix corresponding to <em><strong>u</strong></em>; this inverse can be computed by inversion or, again, by MCMC. Finally, the posterior variance of <em><strong>a</strong></em> is<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2301/embed/mml-math-84.gif"/></noscript></span></span>This allows computing estimates of marker effects and their errors for very complex models, something difficult to do with standard GWAS or Bayesian Regressions.</p></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-45">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6241928">https://doi.org/10.25387/g3.6241928</a>.</p></li><li class="fn" id="fn-2"><p id="p-46"><em>Communicating editor: G. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-20"><span class="received-label">Received </span>February 20, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-05-06"><span class="accepted-label">Accepted </span>May 6, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Legarra <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.1" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hot topic: A unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score1.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.2" data-doi="10.1038/nature11399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larhammar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Memic</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wootz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwochow</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mutations in DMRT3 affect locomotion in horses and spinal circuit function in mice.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11399</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAndersson%26rft.auinit1%253DL.%2BS.%26rft.volume%253D488%26rft.issue%253D7413%26rft.spage%253D642%26rft.epage%253D646%26rft.atitle%253DMutations%2Bin%2BDMRT3%2Baffect%2Blocomotion%2Bin%2Bhorses%2Band%2Bspinal%2Bcircuit%2Bfunction%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11399%26rft_id%253Dinfo%253Apmid%252F22932389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22932389&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308095100053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.3" data-doi="10.1111/age.12378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernal Rubio</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gualdrón Duarte</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nonneman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Meta-analysis of genome-wide association from genomic prediction models.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/age.12378</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D47%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.4" data-doi="10.2527/jas.2015-9224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricard</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Should we use the single nucleotide polymorphism linked to in genomic evaluation of French trotter?</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">4651</span>–<span class="cit-lpage">4659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2015-9224</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D93%26rft.spage%253D4651%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2015-9224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2015-9224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.5" data-doi="10.2527/jas.2016.1003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casiró</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velez-Irizarry</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association study in an F2 Duroc x Pietrain resource population for economically important meat quality and carcass traits.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2016.1003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D95%26rft.spage%253D545%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2016.1003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2016.1003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.6" data-doi="10.1534/genetics.117.202259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steibel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-Wide Association Analyses Based on Broadly Different Specifications for Prior Distributions, Genomic Windows, and Estimation Methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1791</span>–<span class="cit-lpage">1806</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.202259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.202259%26rft_id%253Dinfo%253Apmid%252F28637709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi80LzE3OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.7" data-doi="10.1186/1297-9686-44-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Compatibility of pedigree-based and marker-based relationship matrices for single-step genetic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DCompatibility%2Bof%2Bpedigree-based%2Band%2Bmarker-based%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Bgenetic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-37%26rft_id%253Dinfo%253Apmid%252F23206367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23206367&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.8" data-doi="10.1371/journal.pone.0069202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dikmen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Null</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association mapping for identification of quantitative trait loci for rectal temperature during heat stress in Holstein cattle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e69202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0069202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De69202%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0069202%26rft_id%253Dinfo%253Apmid%252F23935954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0069202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23935954&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.9" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stricker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Totir</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">Acta Agric. Scand. A</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgric.%2BScand.%2BA%26rft.volume%253D57%26rft.spage%253D192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.11" data-doi="10.1007/978-1-62703-447-0_10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Bayesian methods applied to GWAS. Genome-Wide Assoc. Stud. Genomic Predict.: <span class="cit-fpage">237</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-62703-447-0_10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-447-0_10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-447-0_10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.12" data-doi="10.1186/1471-2105-15-246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gualdrón Duarte</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantet</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rapid screening for phenotype-genotype associations by linear transformations of genomic evaluations.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-15-246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-246%26rft_id%253Dinfo%253Apmid%252F25038782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25038782&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.13" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.14" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.15" data-doi="10.1371/journal.pgen.1001139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic Architecture of Complex Traits and Accuracy of Genomic Prediction: Coat Colour, Milk-Fat Percentage, and Type in Holstein Cattle as Contrasting Model Traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1001139</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De1001139%26rft.epage%253De1001139%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Band%2Baccuracy%2Bof%2Bgenomic%2Bprediction%253A%2Bcoat%2Bcolour%252C%2Bmilk-fat%2Bpercentage%252C%2Band%2Btype%2Bin%2BHolstein%2Bcattle%2Bas%2Bcontrasting%2Bmodel%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001139%26rft_id%253Dinfo%253Apmid%252F20927186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927186&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.16" data-doi="10.1086/515506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heath</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Markov chain Monte Carlo segregation and linkage analysis for oligogenic models.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">748</span>–<span class="cit-lpage">760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/515506</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHeath%26rft.auinit1%253DS.%2BC.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D748%26rft.epage%253D760%26rft.atitle%253DMarkov%2Bchain%2BMonte%2BCarlo%2Bsegregation%2Band%2Blinkage%2Banalysis%2Bfor%2Boligogenic%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F515506%26rft_id%253Dinfo%253Apmid%252F9326339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/515506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9326339&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XY52300031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.17" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheyron</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.18" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjMwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.19" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freimer</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.20" data-doi="10.1080/01621459.1995.10476572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayes factors.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">795</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1995.10476572</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D90%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1995.10476572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1995.10476572&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.21" data-doi="10.2527/1992.7072000x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Arendonk</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimation of effects of single genes on quantitative traits.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">2000</span>–<span class="cit-lpage">2012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/1992.7072000x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DKennedy%26rft.auinit1%253DB.%2BW.%26rft.volume%253D70%26rft.issue%253D7%26rft.spage%253D2000%26rft.epage%253D2012%26rft.atitle%253DEstimation%2Bof%2Beffects%2Bof%2Bsingle%2Bgenes%2Bon%2Bquantitative%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.2527%252F1992.7072000x%26rft_id%253Dinfo%253Apmid%252F1644672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/1992.7072000x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1644672&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JC17400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.22" data-doi="10.1038/9642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Prospects for whole-genome linkage disequilibrium mapping of common disease genes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/9642</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D139%26rft.epage%253D144%26rft.atitle%253DProspects%2Bfor%2Bwhole-genome%2Blinkage%2Bdisequilibrium%2Bmapping%2Bof%2Bcommon%2Bdisease%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F9642%26rft_id%253Dinfo%253Apmid%252F10369254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/9642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10369254&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080680900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.23" data-doi="10.3168/jds.2007-0403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Technical note: Computing strategies in genome-wide selection.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">360</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0403</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D360%26rft.epage%253D366%26rft.atitle%253DTechnical%2BNote%253A%2BComputing%2BStrategies%2Bin%2BGenome-Wide%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0403%26rft_id%253Dinfo%253Apmid%252F18096959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18096959&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251800200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-8.7.2301.24"><div class="cit-metadata unstructured">Legarra A., Ricardi A., Filangi O., 2011 GS3: Genomic Selection, Gibbs Sampling, Gauss-Seidel (and BayesCp). Available at: <a href="http://genoweb.toulouse.inra.fr/%E2%88%BCalegarra">http://genoweb.toulouse.inra.fr/∼alegarra</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.25" data-doi="10.1186/s12711-015-0087-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croiseau</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teyssèdre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sallé</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A comparison of methods for whole-genome QTL mapping using dense markers in four livestock species.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0087-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0087-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0087-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.26" data-doi="10.1186/s12711-015-0165-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic evaluation with major genes and polygenic inheritance when some animals are not genotyped using gene content multiple-trait BLUP.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0165-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0165-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0165-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.27" data-doi="10.2527/jas.2014-8836"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lourenco</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragomeni</span>  <span class="cit-name-given-names">B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masuda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic evaluation using single-step genomic best linear unbiased predictor in American Angus.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">2653</span>–<span class="cit-lpage">2662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2014-8836</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D93%26rft.spage%253D2653%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2014-8836%26rft_id%253Dinfo%253Apmid%252F26115253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2014-8836&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26115253&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.28" data-doi="10.1016/j.ajhg.2014.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint Analysis of Psychiatric Disorders Increases Accuracy of Risk Prediction for Schizophrenia, Bipolar Disorder, and Major Depressive Disorder.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2014.12.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D96%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.12.006%26rft_id%253Dinfo%253Apmid%252F25640677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25640677&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.7.2301.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strabel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auvray</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> BLUPF90 and related programs (BGF90), pp. <span class="cit-fpage">28</span>–<span class="cit-lpage">07</span> in: <em>7th World Congress on Genetics Applied to Livestock Production</em>, <span class="cit-publ-name">CD-ROM Communication</span>, Montpellier, France.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.30" data-doi="10.1371/journal.pgen.1004969"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Simultaneous Discovery, Estimation and Prediction Analysis of Complex Traits Using a Bayesian Mixture Model.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004969</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004969</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004969%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004969%26rft_id%253Dinfo%253Apmid%252F25849665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004969&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25849665&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.31" data-doi="10.1371/journal.pone.0046501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagamine</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitart</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Localising Loci underlying Complex Trait Variation Using Regional Genomic Relationship Mapping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNagamine%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46501%26rft.epage%253De46501%26rft.atitle%253DLocalising%2BLoci%2Bunderlying%2BComplex%2BTrait%2BVariation%2BUsing%2BRegional%2BGenomic%2BRelationship%2BMapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046501%26rft_id%253Dinfo%253Apmid%252F23077511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23077511&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.32" data-doi="10.3168/jds.2013-7543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker Gaddis</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clay</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maltecca</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection for producer-recorded health event data in US dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">3190</span>–<span class="cit-lpage">3199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2013-7543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D97%26rft.spage%253D3190%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2013-7543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2013-7543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Quantitative trait loci mapping in F2 crosses between outbred lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">391</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPerez-Enciso%26rft.auinit1%253DM.%26rft.volume%253D155%26rft.issue%253D1%26rft.spage%253D391%26rft.epage%253D405%26rft.atitle%253DQuantitative%2BTrait%2BLoci%2BMapping%2Bin%2BF2%2BCrosses%2BBetween%2BOutbred%2BLines%26rft_id%253Dinfo%253Apmid%252F10790412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzEvMzkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzIzMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.34" data-doi="10.1186/s12711-015-0095-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ricard</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Does heterozygosity at the DMRT3 gene make French trotters better racers?</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0095-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0095-7%26rft_id%253Dinfo%253Apmid%252F25886871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0095-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25886871&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.35" data-doi="10.3168/jds.2008-1929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1929</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2971%26rft.epage%253D2975%26rft.atitle%253DTechnical%2Bnote%253A%2BDerivation%2Bof%2Bequivalent%2Bcomputing%2Balgorithms%2Bfor%2Bgenomic%2Bpredictions%2Band%2Breliabilities%2Bof%2Banimal%2Bmerit%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1929%26rft_id%253Dinfo%253Apmid%252F19448030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448030&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.36" data-doi="10.1186/1297-9686-44-32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teyssèdre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricard</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Statistical distributions of test statistics used for quantitative trait association mapping in structured populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-32</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DTeyssedre%26rft.auinit1%253DS.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D32%26rft.atitle%253DStatistical%2Bdistributions%2Bof%2Btest%2Bstatistics%2Bused%2Bfor%2Bquantitative%2Btrait%2Bassociation%2Bmapping%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-32%26rft_id%253Dinfo%253Apmid%252F23146127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-32&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23146127&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.37" data-doi="10.3168/jds.2011-4256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawlor</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiple-trait genomic evaluation of linear type traits using genomic and phenotypic data in US Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">4198</span>–<span class="cit-lpage">4204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2011-4256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DTsuruta%26rft.auinit1%253DS.%26rft.volume%253D94%26rft.issue%253D8%26rft.spage%253D4198%26rft.epage%253D4204%26rft.atitle%253DMultiple-trait%2Bgenomic%2Bevaluation%2Bof%2Blinear%2Btype%2Btraits%2Busing%2Bgenomic%2Band%2Bphenotypic%2Bdata%2Bin%2BUS%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4256%26rft_id%253Dinfo%253Apmid%252F21787955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21787955&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293011700047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.38" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient Methods to Compute Genomic Predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.39" data-doi="10.1051/gse:2001113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Cortés</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Bayes factors for detection of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-33-2-133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DVarona%26rft.auinit1%253DL.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D133%26rft.epage%253D152%26rft.atitle%253DBayes%2Bfactors%2Bfor%2Bdetection%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2001113%26rft_id%253Dinfo%253Apmid%252F11333831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2001113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11333831&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168433600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.40" data-doi="10.1111/j.1439-0388.2009.00826.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Understanding the use of Bayes factor for testing candidate genes.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2009.00826.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DVarona%26rft.auinit1%253DL.%26rft.volume%253D127%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D25%26rft.atitle%253DUnderstanding%2Bthe%2Buse%2Bof%2BBayes%2Bfactor%2Bfor%2Btesting%2Bcandidate%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2009.00826.x%26rft_id%253Dinfo%253Apmid%252F20074183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2009.00826.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20074183&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.41" data-doi="10.1002/gepi.20359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakefield</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bayes factors for genome‐wide association studies: comparison with P‐values.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DWakefield%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D86%26rft.atitle%253DBayes%2Bfactors%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%253A%2Bcomparison%2Bwith%2BP-values.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20359%26rft_id%253Dinfo%253Apmid%252F18642345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18642345&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262244900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.42" data-doi="10.1093/ije/dyr241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakefield</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Commentary: Genome-wide significance thresholds via Bayes factors.</span> <abbr class="cit-jnl-abbrev">Int. J. Epidemiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/ije/dyr241</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BEpidemiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fije%252Fdyr241%26rft_id%253Dinfo%253Apmid%252F22345299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ije/dyr241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22345299&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.43" data-doi="10.1017/S0016672312000274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping including phenotypes from relatives without genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672312000274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D94%26rft.issue%253D2%26rft.spage%253D73%26rft.epage%253D83%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bincluding%2Bphenotypes%2Bfrom%2Brelatives%2Bwithout%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672312000274%26rft_id%253Dinfo%253Apmid%252F22624567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672312000274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22624567&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.44" data-doi="10.1186/s12864-016-3082-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.-P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A hybrid expectation maximisation and MCMC sampling algorithm to implement Bayesian mixture model based genomic prediction and QTL mapping.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-3082-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D744%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-3082-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-3082-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2301.45" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2301.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/7/2301.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/7/2291" title="Predicted Glycerol 3-Phosphate Dehydrogenase Homologs and the Glycerol Kinase GlcA Coordinately Adapt to Various Carbon Sources and Osmotic Stress in Aspergillus fumigatus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/7/2309" title="Genome-Wide Search for Quantitative Trait Loci Controlling Important Plant and Flower Traits in Petunia Using an Interspecific Recombinant Inbred Population of Petunia axillaris and Petunia exserta" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/7"><span>Volume 8 Issue 7, July 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/7.cover-source.jpg?itok=dg_JR4dD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2301" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j3Wj9FN_1HkBzX_lcfgcpMXta91oZ9_w11SjC93HFUs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/7/2301" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/7/2301" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/7/2301" class="active" data-icon-position="" data-hide-link-title="0">GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490146" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hn635n5sT0HO3fn_uLBdVV7CnPrIn0wG5V8nry4h6tA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/7/2301.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/7/2301" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nOmzFRBTFVrnq40aorh05ybaqo2YFeutx5Msrqk5r9E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/7/2301" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/7/2301.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490146" id="node490146--2" data-pisa="ggg;8/7/2301" data-pisa-master="ggg;g3.118.200336" data-apath="/ggg/8/7/2301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8893-7620" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andres Legarra</span>, <span class="highwire-citation-author" data-delta="1">Anne Ricard</span> and <span class="highwire-citation-author" data-delta="2">Luis Varona</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2301-2308;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200336 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490146--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andres Legarra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA (Institut National de la Recherche Agronomique), UMR 1388 GenPhySE, F-31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andres%2BLegarra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Legarra%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndres%2BLegarra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8893-7620" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andres Legarra</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andres.legarra@inra.fr" class="" data-icon-position="" data-hide-link-title="0">andres.legarra@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Ricard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA (Institut National de la Recherche Agronomique), UMR 1313 GABI, 78352 Jouy-en-Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>IFCE (Institut Francais du Cheval et de l’Equitation), Recherche et Innovation, 61310 Exmes, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BRicard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ricard%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BRicard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luis Varona </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Anatomía, Embriología y Genética, Universidad de Zaragoza, 50013 Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Instituto Agroalimentario de Aragón (IA2), 50013 Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVarona%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varona%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuis%2BVarona%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/7/2301" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/7/2301&amp;title=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/7/2301&amp;title=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/7/2301&amp;title=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/7/2301&amp;text=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/7/2301&amp;title=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/7/2301&amp;t=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/7/2301&amp;title=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/7/2301&amp;title=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490146/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490146" id="node490146--3" data-pisa="ggg;8/7/2301" data-pisa-master="ggg;g3.118.200336" data-apath="/ggg/8/7/2301.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GWAS by GBLUP: Single and Multimarker EMMAX and Bayes Factors, with an Example in Detection of a Major Gene for Horse Gait</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8893-7620" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andres Legarra</span>, <span class="highwire-citation-author" data-delta="1">Anne Ricard</span> and <span class="highwire-citation-author" data-delta="2">Luis Varona</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2301-2308;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200336 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490146--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andres Legarra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>INRA (Institut National de la Recherche Agronomique), UMR 1388 GenPhySE, F-31326 Castanet-Tolosan, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andres%2BLegarra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Legarra%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndres%2BLegarra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8893-7620" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andres Legarra</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andres.legarra@inra.fr" class="" data-icon-position="" data-hide-link-title="0">andres.legarra@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Ricard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA (Institut National de la Recherche Agronomique), UMR 1313 GABI, 78352 Jouy-en-Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>IFCE (Institut Francais du Cheval et de l’Equitation), Recherche et Innovation, 61310 Exmes, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BRicard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ricard%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BRicard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luis Varona </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Anatomía, Embriología y Genética, Universidad de Zaragoza, 50013 Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Instituto Agroalimentario de Aragón (IA2), 50013 Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVarona%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varona%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuis%2BVarona%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2301&amp;rft.epage=2308&amp;rft.atitle=GWAS%20by%20GBLUP%3A%20Single%20and%20Multimarker%20EMMAX%20and%20Bayes%20Factors%2C%20with%20an%20Example%20in%20Detection%20of%20a%20Major%20Gene%20for%20Horse%20Gait&amp;rft.volume=8&amp;rft.issue=7&amp;rft.date=2018-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Legarra%2C+Andres&amp;rft.au=Ricard%2C+Anne&amp;rft.au=Varona%2C+Luis"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490146/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490146/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490146/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490146/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490146/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490146/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490146/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490146/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490146/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490146/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490146/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490146/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/7/2301" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/7/2301.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/7/2301.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/7/2301.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/7/2301.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
