<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/7/2501" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/7/2501.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/7/2501.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/7/2501.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/7/2501/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/7/2501.atom" />
<meta name="HW.pisa" content="ggg;8/7/2501" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Bayesian Networks Predict Neuronal Transdifferentiation" />
<meta name="DC.Identifier" content="10.1534/g3.118.200401" />
<meta name="DC.Date" content="2018-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2018 Ainsworth et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6349553" />
<meta name="DC.Description" content="We employ the language of Bayesian networks to systematically construct gene-regulation topologies from deep-sequencing single-nucleus RNA-Seq data for human neurons. From the perspective of the cell-state potential landscape, we identify attractors that correspond closely to different neuron subtypes. Attractors are also recovered for cell states from an independent data set confirming our models accurate description of global genetic regulations across differing cell types of the neocortex (not included in the training data). Our model recovers experimentally confirmed genetic regulations and community analysis reveals genetic associations in common pathways. Via a comprehensive scan of all theoretical three-gene perturbations of gene knockout and overexpression, we discover novel neuronal trans -differrentiation recipes (including perturbations of SATB2, GAD1, POU6F2 and ADARB2) for excitatory projection neuron and inhibitory interneuron subtypes." />
<meta name="DC.Contributor" content="Richard I. Ainsworth" />
<meta name="DC.Contributor" content="Rizi Ai" />
<meta name="DC.Contributor" content="Bo Ding" />
<meta name="DC.Contributor" content="Nan Li" />
<meta name="DC.Contributor" content="Kai Zhang" />
<meta name="DC.Contributor" content="Wei Wang" />
<meta name="article:published_time" content="2018-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Bayesian Networks Predict Neuronal Transdifferentiation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We employ the language of Bayesian networks to systematically construct gene-regulation topologies from deep-sequencing single-nucleus RNA-Seq data for human neurons. From the perspective of the cell-state potential landscape, we identify attractors that correspond closely to different neuron subtypes. Attractors are also recovered for cell states from an independent data set confirming our models accurate description of global genetic regulations across differing cell types of the neocortex (not included in the training data). Our model recovers experimentally confirmed genetic regulations and community analysis reveals genetic associations in common pathways. Via a comprehensive scan of all theoretical three-gene perturbations of gene knockout and overexpression, we discover novel neuronal &lt;i&gt;trans&lt;/i&gt;-differrentiation recipes (including perturbations of SATB2, GAD1, POU6F2 and ADARB2) for excitatory projection neuron and inhibitory interneuron subtypes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/07/01" />
<meta name="citation_mjid" content="ggg;8/7/2501" />
<meta name="citation_id" content="8/7/2501" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/7/2501" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/7/2501.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/7/2501.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/7/2501.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200401" />
<meta name="citation_pmid" content="29848620" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2501" />
<meta name="citation_lastpage" content="2511" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Richard I. Ainsworth" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3350-5692" />
<meta name="citation_author" content="Rizi Ai" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry" />
<meta name="citation_author" content="Bo Ding" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry" />
<meta name="citation_author" content="Nan Li" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry" />
<meta name="citation_author" content="Kai Zhang" />
<meta name="citation_author_institution" content="Graduate Program in Bioinformatics and Systems Biology, University of California, San Diego, La Jolla, California 92093-0359" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3454-7357" />
<meta name="citation_author" content="Wei Wang" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry" />
<meta name="citation_author_email" content="wei-wang@ucsd.edu" />
<meta name="citation_reference" content="citation_journal_title=Endocrinology;citation_author=B. D. Adams;citation_author=K. P. Claffey;citation_author=B. A. White;citation_title=Argonaute-2 expression is regulated by epidermal growth factor receptor and mitogen-activated protein kinase signaling and correlates with a transformed phenotype in breast cancer cells.;citation_volume=150;citation_year=2009;citation_pmid=18787018;citation_doi=10.1210/en.2008-0984" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=E. A. Alcamo;citation_author=L. Chirivella;citation_author=M. Dautzenberg;citation_author=G. Dobreva;citation_author=I. Farias;citation_title=Satb2 regulates callosal projection neuron identity in the developing cerebral cortex.;citation_pages=364-377;citation_volume=57;citation_year=2008;citation_issue=3;citation_pmid=18255030;citation_doi=10.1016/j.neuron.2007.12.012" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Amamoto;citation_author=P. Arlotta;citation_title=Development-Inspired Reprogramming of the Mammalian Central Nervous System;citation_pages=1239882-1239882;citation_volume=343;citation_year=2014;citation_issue=6170;citation_pmid=24482482;citation_doi=10.1126/science.1239882" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Mech.;citation_author=V. D. Blondel;citation_author=J.-L. Guillaume;citation_author=R. Lambiotte;citation_author=E. Lefebvre;citation_title=Fast unfolding of communities in large networks.;citation_pages=P10008;citation_volume=2008;citation_year=2008;citation_doi=10.1088/1742-5468/2008/10/P10008" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=O. Britanova;citation_author=C. de Juan Romero;citation_author=A. Cheung;citation_author=K. Y. Kwan;citation_author=M. Schwark;citation_title=Satb2 is a postmitotic determinant for upper-layer neuron specification in the neocortex.;citation_pages=378-392;citation_volume=57;citation_year=2008;citation_issue=3;citation_pmid=18255031;citation_doi=10.1016/j.neuron.2007.12.028" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Y. Buganim;citation_author=D. A. Faddah;citation_author=A. W. Cheng;citation_author=E. Itskovich;citation_author=S. Markoulaki;citation_title=Single-cell expression analyses during cellular reprogramming reveal an early stochastic and a late hierarchic phase.;citation_pages=1209-1222;citation_volume=150;citation_year=2012;citation_issue=6;citation_pmid=22980981;citation_doi=10.1016/j.cell.2012.08.023" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=R. Chang;citation_author=R. Shoemaker;citation_author=W. Wang;citation_title=Systematic search for recipes to generate induced pluripotent stem cells.;citation_pages=e1002300-e1002300;citation_volume=7;citation_year=2011;citation_issue=12;citation_pmid=22215993;citation_doi=10.1371/journal.pcbi.1002300" />
<meta name="citation_reference" content="Chickering, D., D. Geiger, and D. Heckerman, 1995 Learning Bayesian Networks: Search Methods and Experimental Results. Number 112â128 in Proceedings of the Fifth International Workshop on Artificial Intelligence and Statistics." />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=N. Friedman;citation_author=M. Linial;citation_author=I. Nachman;citation_author=D. Peâer;citation_title=Using Bayesian networks to analyze expression data.;citation_pages=601-620;citation_volume=7;citation_year=2000;citation_issue=3-4;citation_pmid=11108481;citation_doi=10.1089/106652700750050961" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.;citation_author=L. C. Greig;citation_author=M. B. Woodworth;citation_author=M. J. Galazo;citation_author=H. Padmanabhan;citation_author=J. D. Macklis;citation_title=Molecular logic of neocortical projection neuron specification, development and diversity.;citation_pages=755-769;citation_volume=14;citation_year=2013;citation_pmid=24105342;citation_doi=10.1038/nrn3586" />
<meta name="citation_reference" content="Hartemink, A. J., D. K. Gifford, T. S. Jaakkola, and R. A. Young, 2001 Using graphical models and genomic expression data to statistically validate models of genetic regulatory networks. Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing." />
<meta name="citation_reference" content="citation_journal_title=EMBO Reports;citation_journal_abbrev=EMBO Reports;citation_author=J. HÃ¶ck;citation_author=L. Weinmann;citation_author=C. Ender;citation_author=S. Rdel;citation_author=E. Kremmer;citation_title=Proteomic and functional analysis of Argonaute-containing mRNA-protein complexes in human cells.;citation_pages=1052-1060;citation_volume=8;citation_year=2007;citation_issue=11;citation_pmid=17932509;citation_doi=10.1038/sj.embor.7401088" />
<meta name="citation_reference" content="citation_journal_title=Advances in genetics;citation_journal_abbrev=Adv Genet;citation_author=M. Hogg;citation_author=S. Paro;citation_author=L. P. Keegan;citation_author=M. A. OâConnell;citation_title=RNA editing by mammalian ADARs.;citation_pages=87-120;citation_volume=73;citation_year=2011;citation_pmid=21310295;citation_doi=10.1016/B978-0-12-380860-8.00003-3" />
<meta name="citation_reference" content="citation_journal_title=EMBO Reports;citation_journal_abbrev=EMBO Reports;citation_author=Y. Kawahara;citation_author=B. Zinshteyn;citation_author=T. P. Chendrimada;citation_author=R. Shiekhattar;citation_author=K. Nishikura;citation_title=RNA editing of the microRNA-151 precursor blocks cleavage by the Dicer-TRBP complex.;citation_pages=763-769;citation_volume=8;citation_year=2007;citation_issue=8;citation_pmid=17599088;citation_doi=10.1038/sj.embor.7401011" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Kepecs;citation_author=G. Fishell;citation_title=Interneuron cell types are fit to function.;citation_pages=318-326;citation_volume=505;citation_year=2014;citation_pmid=24429630;citation_doi=10.1038/nature12983" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. B. Lake;citation_author=R. Ai;citation_author=G. E. Kaeser;citation_author=N. S. Salathia;citation_author=Y. C. Yung;citation_title=Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain;citation_pages=1586-1590;citation_volume=352;citation_year=2016;citation_issue=6293;citation_pmid=27339989;citation_doi=10.1126/science.aaf1204" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=Y. Liu;citation_author=H. U. Bernard;citation_author=D. Apt;citation_title=NFI-B3, a Novel Transcriptional Repressor of the Nuclear Factor I Family, Is Generated by Alternative RNA Processing;citation_pages=10739-10745;citation_volume=272;citation_year=1997;citation_issue=16;citation_pmid=9099724;citation_doi=10.1074/jbc.272.16.10739" />
<meta name="citation_reference" content="citation_journal_title=Trends Neurosci.;citation_author=S. Lodato;citation_author=A. S. Shetty;citation_author=P. Arlotta;citation_title=Cerebral cortex assembly: generating and reprogramming projection neuron diversity.;citation_pages=117-125;citation_volume=38;citation_year=2015;citation_pmid=25529141;citation_doi=10.1016/j.tins.2014.11.003" />
<meta name="citation_reference" content="citation_author=K. P. Murphy;citation_year=2002" />
<meta name="citation_reference" content="citation_author=R. E. Neapolitan;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=The European journal of neuroscience;citation_journal_abbrev=Eur J Neurosci;citation_author=N. Schaeren-Wiemers;citation_author=E. Andr;citation_author=J. P. Kapfhammer;citation_author=M. Becker-Andr;citation_title=The expression pattern of the orphan nuclear receptor RORbeta in the developing and adult rat nervous system suggests a role in the processing of sensory information and in circadian rhythm.;citation_pages=2687-2701;citation_volume=9;citation_year=1997;citation_issue=12;citation_pmid=9517474;citation_doi=10.1111/j.1460-9568.1997.tb01698.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=G. Steele-Perkins;citation_author=C. Plachez;citation_author=K. G. Butz;citation_author=G. Yang;citation_author=C. J. Bachurski;citation_title=The Transcription Factor Gene Nfib Is Essential for both Lung Maturation and Brain Development;citation_pages=685-698;citation_volume=25;citation_year=2005;citation_issue=2;citation_pmid=15632069;citation_doi=10.1128/MCB.25.2.685-698.2005" />
<meta name="citation_reference" content="citation_journal_title=Journal of Neuroscience;citation_journal_abbrev=J. Neurosci.;citation_author=T. J. Van de Ven;citation_title=The Nonkinase Phorbol Ester Receptor {alpha}1-Chimerin Binds the NMDA Receptor NR2A Subunit and Regulates Dendritic Spine Density;citation_pages=9488-9496;citation_volume=25;citation_year=2005;citation_issue=41;citation_pmid=16221859;citation_doi=10.1523/JNEUROSCI.2450-05.2005" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=W. Yang;citation_author=T. P. Chendrimada;citation_author=Q. Wang;citation_author=M. Higuchi;citation_author=P. H. Seeburg;citation_title=Modulation of microRNA processing and expression through RNA editing by ADAR deaminases.;citation_pages=13-21;citation_volume=13;citation_year=2006;citation_issue=1;citation_pmid=16369484;citation_doi=10.1038/nsmb1041" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Bayesian Networks Predict Neuronal Transdifferentiation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
<meta name="twitter:description" content="We employ the language of Bayesian networks to systematically construct gene-regulation topologies from deep-sequencing single-nucleus RNA-Seq data for human neurons. From the perspective of the cell-state potential landscape, we identify attractors that correspond closely to different neuron subtypes. Attractors are also recovered for cell states from an independent data set confirming our models accurate description of global genetic regulations across differing cell types of the neocortex (not included in the training data). Our model recovers experimentally confirmed genetic regulations and community analysis reveals genetic associations in common pathways. Via a comprehensive scan of all theoretical three-gene perturbations of gene knockout and overexpression, we discover novel neuronal trans -differrentiation recipes (including perturbations of SATB2, GAD1, POU6F2 and ADARB2) for excitatory projection neuron and inhibitory interneuron subtypes." />
<meta name="og-title" property="og:title" content="Bayesian Networks Predict Neuronal Transdifferentiation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/7/2501" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We employ the language of Bayesian networks to systematically construct gene-regulation topologies from deep-sequencing single-nucleus RNA-Seq data for human neurons. From the perspective of the cell-state potential landscape, we identify attractors that correspond closely to different neuron subtypes. Attractors are also recovered for cell states from an independent data set confirming our models accurate description of global genetic regulations across differing cell types of the neocortex (not included in the training data). Our model recovers experimentally confirmed genetic regulations and community analysis reveals genetic associations in common pathways. Via a comprehensive scan of all theoretical three-gene perturbations of gene knockout and overexpression, we discover novel neuronal trans -differrentiation recipes (including perturbations of SATB2, GAD1, POU6F2 and ADARB2) for excitatory projection neuron and inhibitory interneuron subtypes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
    <title>Bayesian Networks Predict Neuronal Transdifferentiation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__Ta-UXINC0YONAj3hv7M-Srfc-_B4Wn0KATTOQGieTps__agTSMicuoC50kRvu-VcTNoPubLJgflHIUuLq_Jt5dMY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"LmipSpSvNKujpxktbzzoYWag_0PJ6894zAYlNRuowcg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490132","apath":"\/ggg\/8\/7\/2501.atom","pisa":"ggg;8\/7\/2501","ac":{"\/ggg\/8\/7\/2501.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/7\/2501.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/7\/2501"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/7\/2501"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/7\/2501.figures-only":true,"\/content\/8\/7\/2501":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YCTLQW\\\\\\\/wC18AQm40FmsA1AXcF6T3mK+4IvDFDbRYbHkFthRwmVeD0R+m4VYGj6NrGrsFW5ZfxCZA63yEGicMblHErQP2BEYp46Psbu3YtE7fRVLnAngcJ1pJ7eIlZUi\\\\\\\/oUJWI1y2gkk0fAZZVELUB15ODRuUKuaxQmz9fKlY9U\\\\\\\/oz6491Ywlf7Qv3LRxNgN7Y2ae6b1WrlOjHCu3S0hLNxYdF75o0+CbG5jYT5neJlp1QnWd51NV0H3nZIfiauj+Xr5C\\\\\\\/7NvMIxQRwCHO8+zLkiVewvkNVV9I1d3imYW0qA8wBN1jXKyiwIDlqGr94N0pjd4V3FqdtHL9q0ME4SQG4ZW69mGDbxGHTPDkCE4vxY7nqynFdZsxp8p8Zn4P07ejHh3VWvI4BsKuy7rTRUF39QOF0AyTzfm9hQlKnWbESJBkdLSzArqN7f4m1wTizeT+6RAiChLwzLdEb90t\\\\\\\/v2jOZUUXJcZ3chuV8273ptwIn1QsPyLaK6w55NsPTAxW3rnAN1O5511LNNtg0wyCoH6NVBW6GRRZ9FpXpER3ocm+KTpXAvAg67GwZqW7+i0OwKVCKtSSBEHVoa5xlIgNt6rGRrMg91Kaf2ohxVmweqTZq7akI3etEKFunQ3jFGBn8b\\\\\\\/0LonEIiJUe1L0KOAE3jX5m6T8Yo2oEvdgxvieYK0D6fflej4gYSszCDiacP56ZrdXuH++Rz01RXn45VhpdyFY7i5JdYcOycCA45po7pejap\\\\\\\/43kZzZP7cZTJm61HQK7VUM9V5uCaGiHNRXXbLe3Mns7xzIutM\\\\\\\/XgwO+kdZqHIi4LugIWEwb72mSeJ6iYun\\\\\\\/gaQ6uyjowh0rL61mUfodut\\\\\\\/\\\\\\\/gh9LfmGNTQYkb82bxRC5lnXNm44g7pUSFT464gV5t6dQ2Ii\\\\\\\/SMvY7wEAObx+JuaRoQHTIakmMPFrqqoLLG2OoCFE\\\\\\\/iJy1dwHQ5l5K6sHydtNdQjG6eIpTVYh8GKNYbQCC2cVMRdPdkuha5Z4I96vCRuNVY\\\\\\\/4CDyx25AHIdqMwo\\\\\\\/zNZ5osJgZfiSJXaGOhuvtVOQFszIAHTeIOeBIkQzFTlpL+XfTuQiXcObbWMpRcn6sIUEsN4YXIa54LxhmjkJxz8d1UQTs6FREsIOYZD6ey7vl3kIOWS3YHM\\\\\\\/4mrFLfSiQazCO4PezFaM3MD0BPExQCiv8ccrp7SGViqP9aI6hK\\\\\\\/NWhXsUiVIugCLWGk\\\\\\\/+qOPHbxk0mMorcMoBOCsC9h3fc34WQfvgCnvAwwH1VxSpJXeAjE\\\\\\\/3PIeFZE3Az01e\\\u0022,\\\u0022iv\\\u0022:\\\u0022wtyyalkESEyhd\\\\\\\/3w6qGLKQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b21ee8ad724ed5067d2e8fa4f9bfc31091a472e02acd8261013893ad30fff6be\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PplSl4U72dPPQJ4UJQnmrLozJiLDpA97Yd1STGiokbTtrq1HAslI0501dFeRu32p780PjLKVbKx82r80C2eIgX4HzfNcpOVhzLgZkZS8y\\\\\\\/uIHA4fNWmEZfF626LbHtE+TtEsKJz1umAmFK6F9IPWFBwGWCIQX5GmO50gyjwOd3a4sYhlNNL3M+SSuNBJrmSJHdZtLbNIpn6NpM8McLaxn0M7I1jhvkx3Gh0LlvuVMf1Y+NBl30jTmoTQGSwwcTT5dLWhVNJOfAzYNbDT1fIMCKrhLnoOppOO\\\\\\\/eSayYa52qF531f8flSwnwsSjUr+FTRrwQ90a+0gkbboDqDKf0x9pyPSoMLTQElnoXdWpn90ftAed\\\\\\\/yYxgeTQ3uUYjnlCUciaDy4e9C70aQNFjcURbwjH0qT\\\\\\\/M0ztiTZ37zv9bkYtMEou5XZZW7tCMIPk4\\\\\\\/1wH8vQE4dRPo4c5kN5gIyWhVfGBt5qzxuM4T3SaeYlUST7KZu5ylhcI0glTuipPMK7a92wkbWux+pZkQTLb+EtAtZonrQ\\\\\\\/Nwb4SNq6CnfE838nHPw\\\\\\\/Hqd\\\\\\\/ukQ76KOr8p+QsmgSZK8dIH\\\\\\\/91ridE1I3bLkDIxyZS3DEMLgb9ViY8\\\\\\\/LVdiUl+\\\\\\\/BzpM4coyHyOnOIhND8P9BInnuVTdwj3oX+UD4Hajp3Tah9\\\\\\\/We69mGnFQLFTz5ts6r5f3jNxqyavPRIZngkwh+gT7b7tlTaNTp75kuHyYmg1PHIh7YOQQrezko3S5Kqt6dSzB7gDjtWTNdSJ0g91FD8A4Qa4Jrka4ZVVkwfxMxwHH3UCTHLz6fA0whaNyvg9FHCF8fxDnFaex3j4YQoOQYHV8Odb+3d394u6YR4ZPDDzIDBB5Hs72LlXU7nYPAlQceG7osgomXHRXabKnKraSaS0jMzKieRuosaTNMj+98B2hQ7eHnXz+\\\\\\\/przpAdcb8hQYzLrzeW63CNxiSDN3L\\\\\\\/681T1i94KGSls53Ysf9deVuFQvvnFG9Ty50oEClcpMaVpVSovUm+5yM2KjAolyviOCqYr7Fmys2ApPg3JjzmdwE\\\\\\\/p5lx7o79Z562VQ6\\\\\\\/hkHAHbS6kGBqSbQl66eP6j5lELoOXwf7x7HZINyzeDPa7S4LOIRplJny5xnSmFY6DYzxN22Cdf7zFGlkYgX\\\\\\\/CleZKpB8MLL5x+zcU9Ip0DKB5bJEYP4Xa4UHLYzNDFphk6oLHgzeRvf+4xLD2CJMTsioJDLGarZ7LQ1Y5knUcSkrhDuocZItLZA2xjUebsIO4+wz+SOSJ+dHTLU7VBzjfEK6oHAgv9FmVcz741eT9+ZkTPVZsd4LPoS+nf5DfsBobHBGTDhz3W3po1lARRxK03WUdIVH8GSzW19htxVyb9S+naVGF6fsHujFcOrSv0qpVOHhPpPCB\\\\\\\/mzIBCMSDxLi1ImztakP7IS3Gciew9fn5Bxt\\\\\\\/nPraD2+wX8gXum3Ldn81YhCODGf3Z0KvNq5+McJ8EPbFCfZBTHwcpxP2QErVKrhb3gVpFE0GPKZRoF7HdQrwF69lv1YoShveZ7uJtKvMjd71KFcDKZu6Wj6Zbz33jasOm8nW7mcAMpcHhR2qP9okcTYzLcwE6jubCeiJ4eTPDyP0IFzZpmpp5ryceI6BwVumhuV963WoYHjRvvRvbItq3S4Ga\\\\\\\/yXGcj1UONKU9WPgNx9EWe7cwKDWOcihcZUWbvQrvCDxhsbSqN28wydvrgKpOCZtoOrJAbgwueTeeEbHuJJd2DbdMEr4FL7MOSRCb8PqxdkxsUt03BUbr3aLpWV3I4W5LHyP0ijZVNc6AfVLa48Ti3M8mYVqpmvhexkHvxHzu6iGcboWOjExg3RmhW37mv2cIH1i5gMlc6F5R6rlFdPGXoOli4ScyZnieAdXXJ4pe0kZSR5rJP6zGoWkD2msSC8sf2is9Iq21OQg6zkHtIkzjCqDY6jusfa3Jqtmyp26PU+SjWQ+ikRE7VMuDRoQp4RXj5oaq9L57YE\\\\\\\/bn5APNOrMiaaVY0Zhz4V\\\\\\\/A\\\\\\\/Wj2hR7gi9otPhW1It0cZFaSi1aVFIq+BlS80WgGxj2CZvMvbZoUD2czKbwZbHkY4bZUfJDia4nLNz16xKS32GEe26U4MY3LhksBPqnZuM97iuIud2+F+ziwlXHbx0U2Q8v1FWrheinG2TCKuxLqGfTGGZvS0sr3sfPf0KMwDU7H1yNS0lz2Tahe7EvxKP5SuxTcJOXVX9apIud9a9g5HSIPKOEoIK9EycGo8+95Khvgf8JIxdSsEYWuQW15NbzWDX8ZNeQf3aKbF1atSDFEC1Fu5mDsTFZs8L6YS0STRribiv50DUXNDU+JGuieaFCQcKtUcGqLM4Cv1dImZF61hbr1s45WItvSS1e2Xka8PVxsDhH7IzWV2yk5BjzKQRL1F7t+Lkbhn9WFmDV0vzuptmq\\\\\\\/01jf0sJDgGYISPv8pbH0knC2In\\\\\\\/wc+4dSn2J+sjth807fMq++QNEq8OAnqjH\\\\\\\/wiJiw5IsJ3BjQY8YGBy7WfzfqR4fRXv4TF\\\\\\\/TgjzKUie\\\\\\\/8Rbjh+qI+0nbJt4\\\\\\\/ZJ\\\\\\\/j1KztBmGpvHptnnP+E40SN0oDs0dNWOnjENnYDhMM3ZuR6EKq6UZI0oKBfOAdCFSufMO0AgJRLjD2Hk9qUFeALngmHoVTVXWXUEORibqtdLo4mzrWA1s0HonjRuA2oMOt1Lg0XS1rJaL8H9Xyt2VMk8r+yum8yEKTNR+0k+IJbcfH7E2PCRtw7k\\\\\\\/gOz2wLmNOoDeuhKj9FCMsBRnioVZQ6FqTyB0wbaSfM3kLieh1YaIDGDqbekJqLAgvl+5uC0mkbdcN7SpCq20itUfYihiFFKEZ743zqQwWm\\\\\\\/PN2lIl2kk3gRllg3jHiUkO+o4HJQ3fSXABbk59d2arnpKaJAlPIGUInQMVC3fnr3W3haHewnZDjOziAVsWdi6dpn6kiaP5T6GYd7jIiSrBIGmteVkO0WSXuc7nfuLbZ8v4ibEOAeTWH+58S3x4I5HihLZxaaak+REbK2I9IGGOtAc8f+NWBR1PkG4Yu4a31H4WTQHeCILszrhN+wmuMaz5I7x63gKJBH0zbY4Oc+6y9B+tUWBscXnhKD9G1LDMNASnpok1PZi0jYG5y3nqlRA9F8jVILvIinHOT66aYVI1qqrHOSnSz\\\\\\\/SBb9PHbxjL9X\\\\\\\/Br95heIFPouN+Es4xpOBwKTLUFqCAuo4J7beU4JT+l1hUZv7Zz0x+M+zJeHD5gxcIZmb0WqkUyyjRA6wN4\\\\\\\/yuMHz6VNapzGRYQ6zTl46eufW28sfNVffgBAYj0kWBFjLitjBUPp9Fkdpym5oUaiKlHt7gUjmItp\\\\\\\/0bJEsZ\\\\\\\/ea6Q5eR67ZT7QE0OzyJnMQawFroP4lkM80mLeU6nZVy7TDkSe9sDqFUQcd5rBEdIpmz17bRQtfIi8oWxJ2kPxTSyOqsTN0FQ8ZZioWJG9PNLFl2SPjr8T0uX87G9qxwgTsgc1H78TuF4uOeN0bFuNrdVlXZ7teLFUJnCam4I6myctfoJdYvmjcV4VUxX146tnAPM3rkqf1OsM2yDTCUgvSQmYfpKPpl5mb1N3qCYzJ1iA1KX3uMr28WTvS2FN+265RRc7ovZ1KT0j6kSvUAExVc6K31GtD80CDV\\\\\\\/Nku2WnkFHmPEwXKdywtLLpshxZaPPqU0xnXMJDdbsvoP9Igfjh2z7AYH26ZTrDM2j4tqyF44vz\\\\\\\/zkmfARtJsRKQYoWd4n4MzIgySP5Tntx9CA25wtx79bPhjzzQwepeL5g4P8YxoapWHG\\\\\\\/YzA2vGprTaa3IfX72kEqTSBX6f1MKsEH2new10qULSTFvviJQZ9wu3J5MMDwKY04YHsaSTyezxwh15HLcWX+rFStM0ZvTUKLkNgtUZ9DSR155kcrIwKnIKZ0amWgv7AWVIYkIHkCPSXNG6SuHjSQ2wrfX02kmhVaZ1vXvP3jZy8z5Q91jSMSsRXuDCL\\\\\\\/PO8qs6A3QVvZfJ1u7Ui26t6Vozq5wnm3cUV98AuFrlZqJ3MAAdVAqQwUqOPTv8Kw3yENZSOBxgw0iW8uI0C4VmUZdpoBt5pdzSdyVRliskFwCempFyOTxedR7OKWSlHH5Hq7l1qhnvJvslCkbJUS\\\\\\\/OGlD0azJpuGAcKz\\\\\\\/T7hsv9yK3DUWoN0yNfpro4ZqQ9xinGZ2wI0XanKr\\\\\\\/Yp0LD2UQuMWpzZCuB10IQ8e6hGcX99xHeXIrZuOMc5KhDMTiGzi\\\\\\\/thzEYsR55aHkumJjPfNh0IK9s\\\\\\\/RmFwdq\\\\\\\/RVXONTfzQyuzbcU\\\\\\\/23Opx\\\\\\\/7Y2Yfdvura0CXgwsNTQeW5hRtweNHZQaMWDd+C+mPhppLOrM5DKoYTP6MZij62v\\\\\\\/bKXy6U7x\\\\\\\/5BNew1UxXaZzk\\\\\\\/BlH5x+Kfv8Et9B20Wy9HXWkQ87ToBk+oi6vy8An5u70avOWjLE6NUNRu55bj3ZNuxfdTimjQBszYmHpfA5jsXZRbwPQf4Qup\\\\\\\/CbsriGHR0Xq7+NrHWpbRZ\\\\\\\/eNq2b\\\\\\\/0yvaAb0\\\\\\\/7bWSUnrz9DIiK8lNUjdE8IQw26vNe2Kl+Kh3zSWKylTCJedIMiuEHbQXVjwfKnBNElHTLuk5LgcYdsYMNS\\\\\\\/\\\\\\\/BOThvtMaa2EOSnWd6+xmOypjoUXQVW\\\\\\\/R6dbSQfXVPlq9hHPnY6oFLbAiF6s7rX2UdWnZIQibp5aNRis5hChsGOFcLrHuGqYSRcNrRtPhxLnFADWfVMS2e3uJ9ezIzi1jWJ9utT1NndFmJp23b904+MVX+tHJW6UUI8ATsm6KgenJgnYTsOpJYJ57DDaLYO08KG6VEyBqYlxq1U21EnoIwPe+hKmQvNafaUq2rZafuScqjoE\\\\\\\/I6GYJ9jvnfXyBtN9ia86Cr2+QfQfV1EQDpG85l+bkNWPpNrlR2HYVxGfkoTy9Catigaab84Zq57jMM+z2gw0gvqeKo0ma1WdyyiKPAbLwv+KGQMgn6XE+PNy1MOewmj6+kNl1CZYNhJdT78xch6tJmaBwlSU\\\\\\\/\\\\\\\/BZD4q89x3QEjBvq2SHVw6dlJ6ie1OsRl7ZSAifSx5WCOktMlAfD83J+weIKT0RA2pSzEJQ+LGdvGYlXn9mNkjuMvDAl7iJLknwvmPkyw2tafV2jMqx5MlZMz8Eqc+IpEiC8l0sY95hRGvz0ZUJd4pXViJS1H+ejbNZN0u\\\\\\\/o\\\\\\\/6vmtH7c1PLyfjkYIOzFTcPWrQHN09tlP6TN0k+RbLZ\\\\\\\/AMwdwD9t2VOxJVKEoyax42ov7dAyFDCkV8CP9ZlxVkp3tgXa7qedic6OtmeAY\\\\\\\/Hp3u4NAC4wEDEpv+9jcKml5eKDN3r37PDQN7ncX2Le5+oRNIQkMo2ca+Pm37IQDjbuOGNrtV+USv11GJHrzh2iDo9p3K6OcWsEq5Gq0DFjklTnttg946F5OVvg05K+Sgbsze+mOarSiIWXRnoLOHTU0y+QASYAwGWWtHQfRkjwuWcE33aLy+AlwVGW8+9CdXS2XTJ4jFLM2FeQXLzQ0DkJGQBFyrG\\\\\\\/c2\\\\\\\/RZHtMqROcyuLPEtiuXHoNdAW+3KnjzAeykZovpv4hIap\\\\\\\/alHylQG61cfr8mzuz3FHIXqoxuzkFsfv+4XOV7syNz8JIns0t7jEfztan6Vicepv4b310Wo4RT3CIdu6tqoR0vDwhhKLR4F6nuXCuzeS3ZYYZPzl0kaqZo\\\\\\\/tRdg6F2M82haPciyEt86Q\\\\\\\/8oin6JA4LTPip1YjETjXmRXXuMjy57CTDuSrvVoCU6505DKZgAcHwuGGvHXi4yOxLD5fGutA+boIRCVbELddQhjaQFj\\\\\\\/Np70hw8KqVFu5NduKAcc46XRCb89YDLj\\\\\\\/Q10uIGPugMNWRZLNW2Y0V8uoqcWiTBgI8Hve4RKr5TY67IAxNFPVF17a0NB1jvW3nRcYGvowUJFmtkebajYjQ1\\\\\\\/jcGPrPoPRtXOOX+n1wXbaTsVjSOh1JzU9OeZ2U2Ki5Y3VEV7SZr4\\\\\\\/0n7URDOhxlJFc1H4KPluEXsat7da6IDSN8zfD8apwbOLGESjC28lplTqmmRZ7zI7r2VHQIsywDdXXkd1\\\\\\\/Ljxnrq4q8uNrxcFlMBD03fLSCQQeGj3EJA96rbkIhB7vVNVxQuh2qTUeLrazbyqOK48MBFV1wnNSyoRitar\\\\\\\/14eObvZjP1nHTLVAfWoJC9OKKpokJwIvfjKhTR15Rv63XiRr\\\\\\\/MKNtaNm1awGV6PeJMfXwr7yyu9BXEwsqcGjo0oJQq9IpArlT7gZXi9dIs5SprGqifDA4JXC8tATaPnb6TiIdIo+wIxXVlFTquAbWbOPp7ur3v4X6UqfripkaHJMr2xyXwtdeAk5GSdLcc1Pd1+ZPfqP9jHRcW1h\\\\\\\/sgmkaiRApFtsQgTfyUMpa\\\\\\\/kHS\\\\\\\/brcWBsAdxoqQBu4WTEiLy18DGs1dpnqGruNB2HfqnkLx2iMc20V\\\\\\\/qETcwuENM3bxB706FXkhcdftE3d5Bn4gd+riFYUknQ2iS+8xNVS313qhTVuGtzU\\\\\\\/la502Aii2tOHiVQA569JqqJsChU0AxX+jkcMIp1hllWZjAR6Yv3T6ueYT6SF+uLhzIzoybcwUp7S+w\\\\\\\/n6qlpyto3QDy9aia0Aq6255BdTElpMgtUKc1wl9C1ja73QCz91etQr+byxIVHQB4A6yy83hJpbZEkW2vVyolRDwZAfLYZUC\\\\\\\/c6jy9R3FepXnsp46gBTKqmi5ZpSmgntp+nBgsiye91r6T5CDj9+yOqMYZ3zVtwTOS+eDfOP0GZ3hEe\\\\\\\/VV8fyqfT2ES\\\\\\\/Hj5l5TsMIFqizygL900e3fqFbc7TWZkBXK\\\\\\\/WC\\\\\\\/wFjrGHyq+b1PQCSdMPFV04KglJA2+0hadHg9HaWNS2hnWcnD\\\\\\\/MmMAwMcdDp\\\\\\\/5pwyKuvD2JVEX4cojxUJpJdFwrZ0iCle8LPkILK80w7KZfuN2ERzo8CQcrnkFxlCPpWBZmo1qfsVJBFjCizPtsSVjhuhwnUFEp\\\\\\\/wzEsB4vUc9l4Df\\\\\\\/uBUiYZlI\\\\\\\/4qTF4tTQPaWES8cqbw3w9HSIy0sxG2Bagqcx0K0ZmDDaSQd\\\\\\\/PY6BFd9Z52dtfDV65iKf0jiwvMGTwLH7Xam6Dc2PmaoactbgEms\\\\\\\/QkYjr3ehPBUKpbg8OgF8yMCgqA\\\\\\\/kiv9hsxNth+Vczce++Ctfs5D\\\\\\\/AXr3KRhUPV3xvOsuNqxLZiwVDnGVYOltV36w1pqqmr5xzMN2fwsDLi9iGtLkOUWh2NssD42Xknb9nHx87gHvFr8\\\\\\\/37PEFIXebfYxamFdZCwmdRp7gstzNrdhLvtykxt03bNJD2HpzrXQOCx8ZsAOgLkHKyT1skGFcoI\\\\\\\/Ni1SCtAfJr+9U5z18U6hQsHRWOx7Nl\\\\\\\/X2Itpbs9zSmpEeecz81XPiNyv0b3OJkoxWKmLFXqjtAjcilZym+n8omI1KRdHhMW3ShkEIgjXVY\\\\\\\/ZJWOnOC2UQCfFMpPcWze86Qwra+H9j7W4Cr73A94Pdut\\\\\\\/1\\\\\\\/CaneUty2EROOZeKXIjmaDPuR9\\\\\\\/HLKfg5s0x9pHsDOep8d2ljs1r6PZHHkbSO30rYDo8Rs2gx7aNI2sCaBA1IuvjEQ2SZDpo62tXhw7RRiZOyYSk1Ai71QVhYhi8Yj+UNrXAzSiyLIAnWQblOQhQBunQp+HydAOib+vD65i+eV9a3ZI+UU0rV+HPV6XxS5MR7DMWbnQqzVu+XNdIC2igYjd0ctgil9L+v7ZvsQ7KCCulsY1amvbzZXqcjshBAYFFMGLFB5ItiUQRMN8K8iavAMOOzqfX4awHakSUqTnpRbBa9HsRUFCYbTVsOLu3FkBAivoPX0AWSepP84hg74wE992TTQWovXXnT+g9pVkZlDaKQalb6vxPtbfxEnEGBWYiRGSfHu+Ai7KhwhkvUwAYjs5kKf0LHI45jlGP\\\\\\\/HfsyI+wezU71zoqObb8ZYwcXleI+J6+mP4G3ES4u8BoBnUNH7vjXitqV94N8qghk0G5SMZN4fJuWMvEZClk7otSu0W\\\\\\\/ao9C+Mv7Hxay8lo1iQJjfgPy4TZFTF8x8T4mZ9CGe2zoTJ\\\\\\\/wQYx4OqjNE2\\\\\\\/mfOuQc9duSZV6YVfQ4ft7dKc5LZNDcVOY\\\\\\\/ilcCasqvnjQ4CJTePL7tL2dsaj4YN6EKoYxSWttHs8ysP9qiq9Qrlay4h6MsEV2wgVafRYYiWiDSjmIlOJcgqgsbbLaQFeXlaQFjgjkmMCIxGiJ9WT9D6ljuCC7hNF\\\\\\\/CbQW8XAtpIDtj7FIMyeiHv18zlyuKpq4lbLiK094OMp0FpCRnpJthsZguyFnUSEW308hWFpS7ySRb7P12ZkzhOO9\\\\\\\/f4QgxcUmVu9+8CVZJfMDdqx\\\\\\\/sl2wXjdB1KFjfMzJwtCjRzwsIwhgCuIOqnKtJ3L4I5z+GHv4mr5XRV\\\\\\\/89nDPDky6z+NHU1obGVCFGS5vhDS8SjYAOHr1nUI8Hoj+k4L9x0c1RSY73MvR0pkvdrzqhTnjvJwkVxzA9k4fW4Z8UCEJfZpphb8n+bgCZvlZ7dnvKLZ+ytEuv5vj3awFcnF+Te\\\\\\\/WRveq0aQoLignZTfzsf0SiaQ49WU01grbYFAKT7Mvcf2Hv8BEfJu23UNmsQfws4V1mlqfRt1zr5eKAVzDZ\\\\\\\/HbjVjZcUkCqdVMoYxpiN+fFUI3SeF8xfUibAWpgNTgl6chPhetMEYHG2dzwO44wvpQ56oK8snHsv2J3NCYG6\\\\\\\/yr0a+kAGMZSuWy1Z6Yx\\\\\\\/V6EeWduUEcvDnjmQh5ewEOg4AXKIZAw7t4w8eH4FWr3CsQ6fQruMG6VGloAAkC7HerxOUexoI0p22GZF6+RLEJgprk8bvdWwDStHMDfD5mwatcotUQeBZCOB3yD9vtmaA5C4EDfD7tZxq5xuW0d5I7UlwGeh2S1pL+wUh3htBz9613gHKFiRRiKX1NtDzZbPHAuB25xlZyqWBp7P+0nBpgHUbAyJ5qVHODhcEPHfbq27o1yo9Sm7a\\\\\\\/7pSTSAutd7cwrB3KPNAcD5OVuXI8Ogb2w26+IVQyhPlHDQzeCXiGX2K63nxl4eZ8M8\\\\\\\/ZY4Wf8DVTap5oBwNF7hYMQKFstv7RyFTNkbMAo\\\\\\\/fTns5CwGSyEckWOOxzD9wvc7lmpoJncD6NiGsH6G\\\\\\\/10Xp4Q4ZKBgl2f2wG8KEBV1M8LIedCetxE8QiDJYRB8qu3SZHHsVBPoV3PKrQ8UKDzqUo2XdI\\\\\\\/\\\\\\\/J5MR0QAGtnNWqlfQWVVZEXGQ\\\\\\\/86NmUFfUSc+vLyIROkNHy055l9nzb\\\\\\\/CdlN36CX49nfojRX4lW29\\\\\\\/urp7vy5IAoRwEmXyAoDJyG8q8Id6re5i2Z\\\\\\\/aRGqi6S++xOXM6pIog42zJMW3YxCcEmbYHONrJwgPPrHadg\\\\\\\/Hmh+ZHT9u426TLq7iLsuxh\\\\\\\/b7x9mq4\\\\\\\/CwqPd5ztRDeb1sRarETnwWsFg4HfJg1YELmJk7ml7sY5aLrmAIAm8o7LdynzxsHlMjkvjvIcxnAhz3Ikr1mysNSLTSkS1jbNnlQB8Gnnbgubx\\\\\\\/pp6ziq2SduGOsrYD2EZWlvbOFQBCo8xOJKZPIf0xYxSBRbd+M9bo4Alcr3MaC1gXjyA755DDko\\\\\\\/tK\\\\\\\/8d\\\\\\\/lvWWWCrlVkK9fJC6vL\\\\\\\/tVBizcGU\\\\\\\/wxF3KDW9XmUCMRHc0eqa6+MRiyZArh638TGwOtewaa7Rjqi59mlhtZXAfB8pYTWyijWuGyFhXVCXS\\\\\\\/i\\\\\\\/uuieAm7XIgpjSsLAWVG3QEVPIT0Rl5VQ0S9RefzRwyuNBclGY9vGX3D89+nI5MFvcB5U26c8+o8xp1yfkEJZlhFOABGB2QB7vXRB\\\\\\\/Zs+6J5kvDqXs\\\\\\\/myJEJlhDFDfCWnC0lWmQKBJLqoSKOR7UcecAHzwJB1RxoR5rKIo1+ei5M+ZOhjiy4aFRo63sMT+N9Z2ythDRGnjFqPIU3Gsq9tZwsvrR7Pl7xX1b+VRgM9ZrmQOj+MrFiD\\\\\\\/XSbb0\\\\\\\/vddYaa2pS9sl1QxxazQBPVacb9YeGGL3lMg6FzTMJqOjqjJ8YJOCbgmDKmNdZXl6b3Ro0XMLvMfhXcR4GJf+jlI64PY3c2JEGqEFYwwkb8ZFu8z4eM89gbOifAkrT5pAKKb72VyFOsDKp+\\\\\\\/ry2fzlRQRyV7VQK\\\\\\\/zfN7+3\\\\\\\/bcPeDSbjYNKqug3Glkpf2xs1xitMpXrHKqD++QlTNIw2C5yixVq8Tr8hY8a4JhsDNYDuxxG\\\\\\\/WpY2NT6h0\\\\\\\/7WbBrHRyF1vvJg6lNgBBxeP70TQqmXN30qqazV8dFnENLVV+GT4BNddI1LN3WDhywD+mrTJaBmQJAjkJcvzFmxMKcGRQBgUBbn0S1MuDYwsTl6reWwShhsVbauLVqZfWeLKqJ8ck\\\\\\\/9YqN+0zy\\\\\\\/f7YoFWi0dQNOT2xPktDwsCA+LIkSm2NA4RHPg5gnEbK6t9bsZvj89r2wtwx\\\\\\\/73DjYRAV5Xg2Ioc\\\\\\\/sPFzVPuIj8nwU+64D4N390kxt7fDRepefg5Q6xDtyBkzqn3tLNj\\\\\\\/nk20nOKY3qq7hZeSczjb6HKALfjJqo2XgIeu3BZa8fdSHalSlK5PnAgN3\\\\\\\/ItGMnSsk1awh3kQlkXMfAVfpmNJsiw\\\\\\\/nU2AiKbC0JRx03DIZaf6HOD6E3DK+Vf1W\\\\\\\/iBdg+a3gVDHumh31aI354Uln6ZM6Lh7GUQTasxyRe3JAXKcKoz0J\\\\\\\/DcEMJvKqrvysaeZwHUGFCw5PMBiDRpacsHbrOZ\\\\\\\/YkRCAjcnItyBYh1szBOy1W2sX1jw7FU\\\\\\\/usbDViXA\\\\\\\/KUbocMlZDmKq2\\\\\\\/5oOgHhytjoSvy9yvvX7y4i8wop\\\\\\\/acX0Oca+1iX+AsLD1IT2m+mOCkqTwaxHwFLVOeLt6F5uSBw\\\\\\\/qzordPOOMsQxp6KbtOQMyO1bGSMkiEp3t0FfdhBmzZh7eJoDUQNZizWNM37RpQoaOUs2QUrzsfJ33KlXFNnmFSfBposCWP5AICpAi+MbYZCRWTJVY4ZfaXP4t2UfwiCwyO79MCkmi30bfw8YbytcMIASfUwwXXmDqA09H3ypwi0b\\\\\\\/vHDgS8zDw35mQallbez\\\\\\\/8rLb6V14YwmPXAoLt4c\\\\\\\/JVk3Bp8L5V5mWM3NlvrtGA70IIzcSSLXEZNyzqiJPTasrID53AyN8BOLPK6QASryrlqwPH6xxNdmnCjJg6wOWpE\\\\\\\/dZN1LGrot2D9kTH04FLCbEPW5QAIhRaW23aPjrrqzf5SdAnfYv6ab74oP52Lb2G\\\\\\\/pN4aPa0VMo61bc2pErmNzYE+2QOIrhODoK9byIS+LvmXIU1C86Rrg+gy25jS3CWfg5cfr\\\\\\\/gCyAad96gdG4ggmrpqB6mUMquQpBHLQprL6d\\\\\\\/WVlUcV6LHIf6c\\\\\\\/4tbIsK9hdFBGmozGanNH7EowfCWsxAqXGcgcW8E3ixndyLWBz9UKI69H8YnG1406pC3rHJqC5lTUZprqXL45BVAoQ9Ir6tHVFyLyxD\\\\\\\/pAEX3OHt\\\\\\\/RZ2knJrIqrPiyCoXw1b7eEs5f5PPNhqE5OLGd8fPltbwTKPmx\\\\\\\/nb189VhwOE3iFKV9ApczwPhwm8ju\\\\\\\/tMBhqUObwzzJwCwhgImECHICoRYBAzwU324W1mDooCfhykhzyP0XJoftkIKtr89\\\\\\\/1oL7YLu\\\\\\\/QfusJ8cGDB5W14nI9gidrgzHzKuqRx3pUsAGYzW43QBIZZq\\\\\\\/yFjjI\\\\\\\/1CIZRW6d0jpUhcCPFrC1vw4sHLaBbKpZQ4qgkhqxIxN2dbOQx7np1JYExz8tYsfUztnEi2j\\\\\\\/bW86bfhfZ1\\\\\\\/s3fulWC3kNX4Lf5ClF71Xl3kYA03HzLKe2XNac2A+OJ5daml2W4P3bdat72vfZ4AqsilXVn\\\\\\\/kSGVPwPiLqg9OIgeAeSbc8JAuZQrlRtMwfu+xYgwLcQxFx1Y59AvJWZ+bYMMnMf+i3AXdlZv3sjoyWRVlvV1hBBlJAhY38wBNQ\\\\\\\/Q1olG7BQPDvq6Abvj+VfcaFDe5D8EJ3fWg1+DyW0Qdcse4NXitvgQGqF3DDrcGHPQdtPflqZ0T3j5+W7vkOgWZpg3cD+re5i7brl981PMN6Wsty2dI1+6VRjXr3AJ9dMbTxjZqWw3qbireF5LwAan\\\\\\\/A\\\\\\\/xNzw6TLqwAuJvsO0f\\\\\\\/1eh+DHlhkhuqGgnbvEW2kDPrktnHNBAr\\\\\\\/clFJI+Zv14HWYzNTzOROSk+e4\\\\\\\/IgaIFis3gbT2qPBEfVpRykUxkFjbp8YrSCYT4BR5zWNJFtVAhwxtGZRGBek7v9CIOKejsN8cu5rrzLE\\\\\\\/DcaSIvd3ATDerXY9O+8RkozEZs5quos7KhbgE5gXIQLu+xk\\\\\\\/WO6QDnlHc8qYQHzuTp3Ev4mNdL8XmHLQ9oRqLDmMxN9KJ+D\\\\\\\/gGmE4b7L1cWq97QdqZXXCG6vCamRbOXSLcmuHIob18zPSSr8uG1KhrlhT2b\\\\\\\/oeda4joeWLgqDBcUmoegBD5X4itxGqOxL1FivEMAv5sV30GRjjEhNpA0fZgSff9b9H9EzoqaPwRsXRCb0FYL31S9W5v\\\\\\\/s6w31k60wqT+\\\\\\\/uT7GnLXzMwz2hOJD4wHS6Gs7wHC401GS3zUjsUwuPbQUn\\\\\\\/V3CtjXRyCk2AKvrha5pCGfHo4vpyUtj9G60Y1RUm5U7MA4R1gFVDImc7w7UtG6cQyXbj\\\\\\\/IdgNS+VrW5mPpItnHDz1jWiWMe5MGoGkTcosxJxyvPWRi+KpFpfDmDWWSWZBtTk9WmTXcA8536zBwgPWq3Rw7ueNTrRUAxju0yTRof7KJFs09D4nGterddo5AYtWarDj\\\\\\\/f1eSeDShhmlSqLsh\\\\\\\/thRJMWl\\\\\\\/0NpxvHcDtZpv0sM\\\\\\\/0WDUSz7WYq6df0oeOdAv9eCooAu58lvkuQ3UoJTsVelZlaYE9UXOK6FNkISc1eQRnSviZ9GYRqiW9zrQ2FVwi96ByN++j5RV0c13t8tli9AOoZ1IIkMXBFNYeXij4KMFLIhwBucKp9VAIEHMFin1k29f0TxOXdXdjMKohS\\\\\\\/wVx7zQnp6bV5afSKiaIf0EauUl7hmyWZTEj\\\\\\\/11VpUTDVIuS8Het7FkUmNFwN4DAnjYkte8P0szf6QF409\\\\\\\/CQlRiz1AJQlaiFMEjVGgNxTRm73Gqr5I8mfKE1ghYj60jNr\\\\\\\/aNFz+feodcEeO7Azs2smNBKf9UyEHpX+ObSbWxC5lWyZrUt1hF\\\\\\\/f037Jm00sVucNIlKZllubpaMerePxzSV6cbAaFpsEX1C0ZrntVmM8KEv5\\\\\\\/E4JxDDveQu+yIfP8xSwO6NyufoEIDo8RjZjhQAPv6hIJS0+8F+lAxECgo2lcr+xnQZpBHQDohjr4o\\\\\\\/AHHHj7Uz945WQuaIfAIlxFTGwRbPM2UahiBzXlgOkf5zz3ETu3yk+Oi0JZyojWYKXANkyQboJhYdjfbtZy1hNBbEOrtHTy5RlRxwyHQn2BFhgIP6d2h6ok5MG7gDemZddgZ5URt5fojupf4XTqgLKTifXADXIRk4fLsg6UrZpi4Ag\\\\\\\/SnokUi\\\\\\\/qD3DinEwa9De9kYPL1c+EMwfm34tX\\\\\\\/l6XwHf14jH5oeYaOh3j6C9P0E3MK9E\\\\\\\/tEM1+HlL039o2ed3q\\\\\\\/X1\\\\\\\/VH13zVFxMkd9heRdu\\\\\\\/sD5lFxvDHW4qjXAs0qyU3yL7Ks+TaLt2zH9znmz+BL4Da+hxIHd6xzKQZGidl2VqAgMGsbXS74Dcqnl92pSFVls+oaCqGLgqdJ1Z8mCtt3hgp74zhBdFJEY\\\\\\\/gfzG+Xu6mxacFuYWlKKsnHJ24sbQeRk7RHQx2LTXWbEvqFmkxIoZbd3nkIMB3djiV36x6v10PM52e2xS6VjjjM8\\\\\\\/UClOoLKAg8RiR6AOqsK73Qarm2BJ9ojlwPSOW4CTlon97lvciaOhPfctvc34Bp7BC0dYc6iUS4Ys9+4ZkKEpv2np5ELqWBN9G35qneTgUe\\\\\\\/EFEPlBfITReOmEA1L8MRsPJ8b4kCVaum3kuQ+QFYDirS23lB0Eoqr2NewAKRim46v01GZ6Sc+jwA7IwiJhbyzp9WXB0Y849KXeKTnavJlJ6dEl44puK27mbnHe4o1lvpecEbOAyX603PY5DcyiWN24RcdcXkZmzgDTKtmyyu7T+8igqgziD+lxxSkl7MlA2bgBL+vwCQT9gHF88gkFMcsGrHGDVWXLjXKwJ2MS1UF9jGXH34BMLHP+PJGWBXbELYFaESOvCVCX1D9+EDQr\\\\\\\/5QcUv\\\\\\\/+NUVyWge8NwGLIxLI1F9PtpnK6VtkVkbUJBuZyLkBwez\\\\\\\/7WMaab74Wib5lIjJDyflySWEdXWj2twkY8Li5UIJTjJy5OR9SxfDViu2S9BNdPvu8oWjZo0I4U5vqQjCCmbHL6OShXeKKRt3xwlLC+t3QzAyiNWUgR7OXwdMRtyCHFh\\\\\\\/BCtjhA7601j50zZYoer+Fipq7d4UEvKlozrQv5Jam9OnKMtHDFD9VLOMkzWY+Rhu0Yx2QkFIRO2g802GaW8b+ZG+swzowaUFR3KpXvgoYaxFXQh6r0KjJKo7VetXJw02vmQqIzT1LmvCkxeAjcy06nAubnXqs0xdqXBadFpiGCQHucNnRVDfeZqcajDHVAoeNdSwOPCSlMH8Ofj7cQ3XZnBgeltoJCgB9QO+erjkhrWQZj7ekMSYCFug8uHm8SJ5uaMhF1cNvcfqCtgcL4iwXx7uot+N6rwSfm\\\\\\\/UYkg6KwOdmE\\\\\\\/VT6oJ5J\\\\\\\/Iv8Lfd1CgJkMP+0EUaPfA9+rHX7w96\\\\\\\/1GTNQ7mrdge1RgJigT\\\\\\\/Fw3\\\\\\\/2qy6aHJXeIvXxHmbUFyUpmfCBh1A5nHQndyBUPLsTKZGXJcZrzb3mvEeRxrYt+Z3DZpRz432xhHh1weB7PW94E7kC3UbLeqi1n1nNjjPtcptuZ2uq65WqwCKb8s5LIzT1cKZ6EF6syo\\\\\\\/yG\\\\\\\/Bl\\\\\\\/3XkZferibqF0HMdd6cTGz\\\\\\\/0vVFSPcTh+6BHALXqCfiHP2M3hU7MXlZQ8\\\\\\\/imGV3c+bfVtkJNlx+Oh8L3OBkLtcHQaooKNfTCOou9082cglYgWEoG4VxSVcSUBuI8FrpYYv7lq4N4A1i+tstL+Er9J\\\\\\\/HbB3tnN2uNPqdSnX7IGvtdfBzMi2O88DecSos\\\\\\\/kYWTyS1+9K0rF3Qs8TRN2CTda34wfApHmPj+iCpIOmag0lj52hqqp1ygJDlnkUJAqSwCVeWiJWbdASbpd\\\\\\\/qapAHOg2admO3\\\\\\\/yEooyPm0WbTDePf1E2xrwzVAE8vM9hpuPFvF2O3kowrZ3aYOqHKxDcrnkBwtXF\\\\\\\/nUSf3mBVi\\\\\\\/9QQFJL4WuNvhZEei+SAiceMT6MyOWVDa6RNqK8qqcPDc8QnvOkfQBacPW4R9JQ6RxkwvRt0wdxzLlzm9qoaQqAu1SLCHPzY0mNG7Gk7PFHxqTHmjgDDWUxMG2CHgw8hmaCRxZ9bCNe2NxvEfl7eAJSvYfXQmWfBFb82WaJ\\\\\\\/t5aeQbZQyLTFa\\\\\\\/gaORyYIod56VmsPIIssUZVQOOfMFy1XARZL\\\\\\\/KKqO45QhakI1b2OxlGBN6uhr8IKy7y6vUqiXywZd3Su7TLTfxowKuHLaEhln8RmW5JxdHmmsYtyv2Pph\\\\\\\/szzs0a27KvLTpxlH5hgJEkbQ0z2DbkjvKYCcTOqPZfbo3oqr80V+1o62LgZBQ\\\\\\\/COHJ+eBxXyiEC9OUeP6Bwfmu0HIRXX38\\\\\\\/aQEaLWievtYxXbyadHAN\\\\\\\/AgOK\\\\\\\/iWq5rwiBpyAx1LTsqsLE7KS15Obh0kWF5m7TNhrGv6hbgjDSKjfQblAjpoyKEspoGEPI5wuvmq9OPqS3t61+S2snbCclOQ2fL7Qse31U3MnlFMhXYS2hnJU0\\\\\\\/llr8k\\\\\\\/SkoVeW7VyZIiLTVrc\\\\\\\/bCW0kFucyalryWJkjUS6K6h04BmvNQwEqLuu4SzHjwsazfBK1q1N6c+hTW8HN2RW1AlSEekwC0+1V8LooKVI4r64SgQiyp72g6lsHwaTOp22tWZg5Gc1qNPUx1c3kyZRls65fRWjIr3zh7TNt2pWWsACfBltLee2cSn9D\\\\\\\/QpYElvx98zqL2r5df0F5JqqHhZiYKt7H\\\\\\\/UEpF0nUrCCS0y4BlO7vh6CeM8o\\\\\\\/PM3Vto3+zGkLk4o\\\\\\\/YoNj6\\\\\\\/v67qOOuek6lf9wlVkcrCG+nwpG6eKeloofSFzwRo6Iyo5eEuvOO+4eYwcT1Y+aAii6MHxfPIv2f6p318wbBtOhvqqR1o9MglhlxLi8Ww3XM1OpFPwRzd8Rgs\\\\\\\/a7TdMBe8hNJuXNKNLxh2qnlFJ4y0XyO3zCQchEhP4bR+MhyZ3Gk34jmIpz+kgKgvRZ\\\\\\\/b1hgXISN6W1RauJgM9C6aizwm18375L9dVnv8rZCCkwGNPFT48GCg8OmyFlaXrV3axv43fQYQn4vZ7hZ81PU8o3RrvucLGCO+TxUbdorVylT0tHnKbLQH6hBdCi5PpAg0ijE1hjk6pGH7pODNUVgXMq74NJJz\\\\\\\/oCqi+gjU8JD2U4I6SJxk7g1oCd5V6SaMLqdnvbjjCNyaHI6d6exco7dklFA9QVW8WymtwZRhN9cd0HmkfJQXV2QMv8KD8O6WCtBGplce0WCFl2l77hWwHO2uUelep3oZBh18Qr\\\\\\\/QFeqVC22JirzJ8Y7h\\\\\\\/ybDnEf2iEm2AzB8rPFr1qTIR39x1Le3kSWu5JdHoX5w3Xjvzj1oMyBvAIxJLD87g8dmxo+CBVYH3uSdCj80Hlo5obhaljUVW5gvunKe1D5D9wClVMYBpLFSO8JDW0G5qKgju+syhe0aYckjRLI84TtxOHu8VwrgyP9ENWAuWdesTSqT8nSIm8Oh3KA2E8yq54uIAWPYyZwMQXPIpNmwVa5MxGkPApKoyBYdAACcdmnowt5chRNzdCVNur0q65JNM5qiEVrdshRuN+WoJa54rRJDD9+Gy3im9Un8PkzAnwM21zHsczQroFcWHfCJAybqAQBfvdoKdJdwh77XDqKujKhTufFiaTCQYrLCH0OdrJaYbw\\\\\\\/EoHuZkH51kw3J3+9fv4FJVSUm7LSaO\\\\\\\/fi9E2AAZTKCKu8ROE+JOhAEEyRnSWCte0XAph1y6QNSt41RjUvqwzwmVoh4n9IyiLTw+hckNyA90HauCXiZllFA+NzcuCGYzvyiG7z+qkug79XKoCpRTCdF2z+XB0xiM60qlDh+HSGdRqpsFgeEGGERpQCVBnlqQ+YqBMoECIpv\\\\\\\/HE\\\\\\\/1RBtSLEFZwSkhIjHX2y0S4wuA2idRIMDTGqHXdpqN7R93wP0r1Y0bqluAYHqNYpj4xTM3wxXLrux8hpZMLa6jIJIvoQjJxXbBbkdYkjETsCZM9qOhBxzdJOGfHVWi3wpuf\\\\\\\/knDl5C1tsQRRrylswRdnAaq+uZkoSnZ2jgSeLtTZkiOkBSyTT0+dAkPTDlNZbbd+hPxWkPdmKmtUMYAaqfy7a\\\\\\\/R4dk2XazH1Sm\\\\\\\/SLaKIFza4gwYLTpjzJ\\\\\\\/+JxBd8yEEHf5VsqfOwsCDUr7FM1lvxwt6rJXtmJTDPAoTlbAotcoccjGRu1GRuVouSbGwMDJmUiKeigji2vACKO6dzRiBkIZDAGIyAMNcPANr6byyO4B\\\\\\\/jrwMUw2r3FOIqe8Iop1R6zo0NxytWzJue4xif822+XfeEzA9FswdZurRnmjtJrKMmuSP1SAHtA+39gDJpl9436jxZ7v0iVpXSLPkPjKorCzd5+tDheR\\\\\\\/BRrci0UrrHk1\\\\\\\/t3wSC2cYhF2avoZCQllJGMwgsbRX702rKLJ2gxjCnViaPSlRFNTaOJuNGC3y8+5tnf+YfqruaLdwTXlxAqsCM\\\\\\\/ns5D5UY1QvTyMvIdgRh+O+YPD\\\\\\\/OCgPEVZEtPteJeBq8gHAs4cuHu4n+HfJTRLViMc12j4G5TqfFhA\\\\\\\/DXnxkbxa9pOIzm9me8UBZo71bXAsUbYpcTSfJ9+gvuDWhmbTLq20IjPuarb+lJjW3YZ01QSL+bGFjUwLljekWC8jIwq\\\\\\\/BDWX8Zadhqdf2tPNZtb5+OGdSupvJhXIgRe4I5cFExFfNCCUkW7h7yUGmBaPQ9L4TiOwfk9UVgUp\\\\\\\/eBgsVOv7mnu6zCAxeU7GwaMgJ+sfJABszWC8RGPlq08wuTDkLFHNtaIx8uPTf\\\\\\\/VITHDIChVC1VWrP4OTwXOXbIlid9EoZzrDSo6QWIA6RINEtkNZZmahns60t37ad5PgTNIyk0n8nYTihPkXaty4jf0L59XOGFBxJqYe2ANZsbikNqC2669Ad4I4yqaDelf\\\\\\\/i0mIxcMciK0tsWUT9nPLwPi1Cv6NAQuVNcTMvokw2cNiqSuhgZTV23k33mx5e\\\\\\\/wt7GgKbjZSXQ3LQtyBYSsezkDB4+jVWgQ05scKIZG7SCaKZp6rtfoBsTtoGh8bRgMtahfd2px+R60YV+0F9wnZai3dA2\\\\\\\/94eQneiZKWCF4cJ49VskDbUpaeOoa+pnWLfcdZCz8+0VngjiywdI9Xm8kiS2kEAnfZpgT01RTx9AnIPps1J4Dl3B3zwQxQEvI1XgexVIOhrDsvOVppK6tDlLo\\\\\\\/bIkyv4s6r4lz39q8iLke7sKQvzZUzetWfWxab\\\\\\\/Mhk65Z2LukaIPlkDh84q6hNl1LUHHlCqxkrIjN8XsrqHmc6QvTpSo4vbrXH6un0j82O0NGhu5OwnZbdWI3U5ZqVDND\\\\\\\/M+ioAVkXxVp09tIXQhAEwsdzDk8VCCLw2gi\\\\\\\/BXMp5gUixLhJl6sd+90i64dvELrSBqa+cFdjDz+R32CE7Z16WCN93\\\\\\\/Qq6am6ByE\\\\\\\/vQcYXTksRJik53QvX94AXGxwMO7jw47jpdRLRDOe3GY1ihZn7zJu+eFOdwG0GnYR4aCR8otkwr3BsL0Qu3xYL1VjDhG5R0RjAwu88rsq3X0XeYS2zBTFGLCy\\\\\\\/4zPfB6u5MkxtMNnnw8d0CD5ygZM6a2+vjVgDgzo4yFywLBBqaBOh6BcS6XfGaKjHj3pxb1k3kwOfxf6RTzc0EqJVqVFBbecQIPsVs+o8pO0HQC\\\\\\\/kt2kPVxpJ7zoEaXBbix7ayzuNvQHAdYHTyPyNTChx8hQtyfWPIYx+hnFwLDKTIHn6KGelDcmmCFYschXHpPTKl3FXA+Uy5dVOC804dAKXaAa7eQS18PcCKEBsnKq+V6Wi\\\\\\\/e4AQbv8fnt4Ip19CgQK6p0pVnC4\\\\\\\/WrYFq1YIDtnbkbeebkMcDOv5+b+feodaV1wqKnzHSkvyxEvyfR+JQkeFJGpry507xpcs0YK4jpudPa+OKkSaqcaLZ7wT5+DANilI7dgrdilbppnLbrOlLGphWA89q4q4A0ZVmi1r0L+ArpQB\\\\\\\/dVQGQkROGlVh+P0r+jCj8EelTcaH515YRAUKvIN629E+lr1n7fccYqHRU9fXF7ZP+mXtWcysefeE0LxOXgb3moI8opq2xWZm6O8kw4uxY6AkUJ+aq90RPm2xJG5fGURR36NOPYkLGnsfdXu4KsbgaK5zc6zql0Cs+GForf7Zv8keYXmDsxwC13ggKt4vvQUx0yYvaLSqnttdIk1G18PwMlVd9Qf+rNQnb+L\\\\\\\/ninumjlhotGDRJ\\\\\\\/NAmZeUq9QU\\\\\\\/nWwAc1lKbnRRb6M1o0edDnmupjgwouG2l04ClPG1I7vBy\\\\\\\/1Qe0MDOz9IrfuaGORaNBnfi3nNfGZlTDr1j6ImaHfCtjWKAgw32Yz4XjX81ddf7VUw4pyODJnoDHXKjb6OWmgN8LSIrdablD1GG0vkzc3keXs3OYsGg6pf21NBR6OqRPwltz0cGBCaK2d4NNhkKMAUWr8L7s+zwO5GAmlukrSoZyW0TDE2VoV1YFnxqImxq7ORpvPs4e9L56IlURY83EqjWPZrGDJaQiuvYRuseQUZ1I0SPQl2\\\\\\\/y8mmkSWDNx1yRrdyL7fZnDtMPHkAeSiG3Wrh2vAER5P51EnJ+eqnH1r9+nn5J0Jkw90uSr7u8UWpx44\\\\\\\/WWvtLF2yEuohR45gxeaxLmiPY5TGMwwKPmMIkesQoRTLCGLz5YtJJlsXCdKYFKrx8JHQLMcwbvBJvnU0LuW6oVYJr0+cOz9GklFCSrfzTj\\\\\\\/XnK8MSpjxL6uoR4es0ELTTxOeLsXkoJ8\\\\\\\/SyBVjPEHPnwzxF6F\\\\\\\/xYaRkF0Un01ZVgEiPRNECCEddejfqL0GjkMf8D4i244sKR8fsG\\\\\\\/ofHaFgGKzh6D1GRFzBtZDra3YBnafDfNK0tmd7Re\\\\\\\/O5jveFaFnxSYXQeuCOIk+8XdWCWVOVv62OfBvOKOZJbY3PBGSV6hBhs69WeYJ25tOYyYAA97SmV\\\\\\\/3ZKN2IGX1vUHo+apn1B84l\\\\\\\/pOeI1pQZ1tjLEJ3PTRba3ZDEkRYYCf0\\\\\\\/0i1OAKOVtBatm6346kG5qKqMU34i9E\\\\\\\/0SyCipPBgpno+uq+Hlv7SFMYZiWODZdSTHvASh3ylVRp63ESYTY8GSZzIDkFx58hF8OOTF6l\\\\\\\/PtOcQrMUolWASkVY7udcUL\\\\\\\/8GBszpuikJAANZh+rPZ1+J0z+aKL+Cm6gRFjO6NdPJWX80Q4CpaIYFz7MT3us8N5IatBUA3TqfHu4abuZrNDxE4yRbQ+jT5O0K8I9kFuFZbhEmxK\\\\\\\/JtkmPamG678W2J9wBKgFn8YQBnWUVCu5xMiZyBVmQwVrFFTGajRmgHDIcyHu0lFoa5abpQiWXp+Qen+At2KyfsZeph+BP\\\\\\\/Ivnj6b5V6z5ErzlWcu9SLW7+wYmjlvUzXdHpAcjr0T+3p\\\\\\\/EEZ\\\\\\\/yIuJrk0fdvOAww5TLpE1PP3c\\\\\\\/E8FTWf12RRPftps4j16Uq1CrzkV25Xc0\\\\\\\/8IRan5ESevlh1CL93BX4\\\\\\\/XnwwjAfXqpKZ+\\\\\\\/bdLZtsX0V6jHsq9tO\\\\\\\/dPLnGk+FACqiOz4v3ZMRmDDneagZ+PK3B3xxpSCJqGf5VSfLewSgMrjFYZIqsrWAd4P+52ctRaR5PKABXQbV6DHbRGLRI1oaPt59tpSs7W5A8FtgyNZNtWNXa5IbtCKImgkynvwXekotRboD6G5dCetbMaMONlpnBVKk307woH0SZ5tn2H+tqbeP7je7ZpwbhdLz2guI+Maf6GP8rdi5ElDAKi58dBGlifJso8WP1gyOXnMfrgyVOhTgMjcCUEgrz4mzmQlsRHiexm5ivqTNypKq5QAdYkJqNBZqxLHxekxXbCiTSthCNyV0IZWeFBboKfF4XZ06qU4Ovmgm0NzkEnmp7Tb04GwsfhAJ7zDBDdGwXwv0EWeVxYJXZ\\\\\\\/1LfxAxPj\\\\\\\/W3hoOWTOQk\\\\\\\/H4qMZuW7VfDRGb2wIgyeezohkdJb6JXwSAPV\\\\\\\/DdiLuyCt6v+KcZVoriX+rkdz8yedhlPRUBb\\\\\\\/lNvH\\\\\\\/X2uB5YjoExKHoLw+l5UFE9dsE51gYlSWWSL6mmB7BeG+YRGgOIOBq7zD\\\\\\\/fDP9MmEhDRob7A+rPz2ENOQi3tU4wA9t5RNOA18GG+duFrmp4ANel291BRoNsZcc43+zHv55iIC3q787b3t25YEJj9mYhcQQUGvQuSCRCaR9IQW2cey8SFHh6mTWG\\\\\\\/aFJXNizddJibQ3mTnq8M1k\\\\\\\/y5tl\\\\\\\/MolCmJ\\\\\\\/dE4ctVAJKcFmFDoJOHfr3TsrAnv1UxksHHUQ61c77\\\\\\\/TOl4O5hKRt6ovp2X6O5bPdFTLfLq9o+QuyD\\\\\\\/LIbmbQtGOTIqB9dj\\\\\\\/3OntwSKdYon1PRX\\\\\\\/io\\\\\\\/ODtxc3CqE2bJe4SD+uKbo3JRo9UShBRbwLH5dnYJXf\\\\\\\/cnvVbna5ZrMOB+lA+hRhJGakKvUL688mOnF929Xq5OUB6TbhG5g9VVGrOqSz52JnwZstbPEhDe0+UczfYta2WBnN5doxZ\\\\\\\/CaIM+xrMdbOjv4lsjsKI367VZAsjqTN3+RFQRqrorBsad6IV27GV5nyxmt6tTS4spthJm58wFkhEHZ6ujUEbBzVL4cCaMRnB8q86sTMhG2jYXQC7nMBDNWHKzuWTwVNChJoyEi+GKjWBL8d2qMo0CyiaDUjmu5Y526Ta\\\\\\\/A3JLRBNpG3AINck0WmEIAuSTCyIvoj0MYDV\\\\\\\/nLaYPjbZU2iezsZsQGNGMKzA\\\\\\\/jlmN8qaEaFW6IJMiNmYg\\\\\\\/3aZpUeuT5a2M4TOQ84kpwIa5So\\\\\\\/lZ+pFc1dA7h8nDo9SzastGbOkKeRG60aU9DN80sABMKupyQj1cnLcVMLfbN3ti9RjWAXuIY68HfiqtfT1fIkGeWO4XKircmt0Dgz0acIfWx99RPsvc9WwKKPWa8z\\\\\\\/HjImHxwk3kFNSOaU4dOBCaTL5ODlIUoNXO+eRSNwk2RZcwb91mtQvVB7jZXFYjppppXItv6D4c\\\\\\\/8vGxcDL5mvSZvsBAaPHnd+y2UQwtlwHcoP3SVFVN1eWBHmQezbGFgsyAMP3cQGBJtSG5rVZrTdzgLLeoKL6C0uQFCTk2Z\\\\\\\/bUmojEIvpDYteHYF+OLtqeLxOxw9ulClvLIYcWAR996tPS6QONSwNpz7AlICiAf8B\\\\\\\/eYajGt6qBF34AgRDgTbHqOfEp8I1R\\\\\\\/e2woPR3Fud23fbJNisxPCyKTXPh\\\\\\\/HmCAhbk3z3HadGl8TVetUBSR1t9N844AlUqz6yjScgUx54\\\\\\\/g2V\\\\\\\/doTT7Q1RjkcnlPYoaGBvqRgXf\\\\\\\/gcnjUapaF9dV6LBYNSGo8j6Tj6VY1reojTB+g2Ii+a7T8rBcTDFlXYN8ppfBF7\\\\\\\/3ox5nneFsPTobjYSkehMNjh43xMNN69Ol5IzrAk5DOF78OoYqQ7cYNyp+VUlkDHi2y9aEYnvS7VHyL3OxWvgm\\\\\\\/YnTuE8EaXIg0wlKknGWU26+1+GDBlBnfEBFDjtOD3DykQRa+AITZv2ajmDuc8IQ7ybnCJC5Vucn+N\\\\\\\/5BqFb9Hs3Kixt8uDRH9qL4yAPucEmGfRvT1RuyN5vY9SxcUdiiVkERRKcUUY7GmYK842U9cs\\\\\\\/\\\\\\\/OGMMQRWpLiOGrdSd6ophTwOshA9EmklGqfNJXRuxmRjD6tyPD+JHQ3X4ohmV+OLAwQ4+wfDKCrGHEYRDlbVJeSQMj9xrvEqHkG5rh5vpLqTyXb81IEBnSDp+1rOZa6riNi7rWtEPn\\\\\\\/0weJzpSF8BTQX27Sw9B3lVujH3hUnNccR2E2tmOHEmeXajnMy\\\\\\\/oa\\\\\\\/vgUeQGoptlUuseFabc\\\\\\\/mPP8UsPGWErwsvbfWWRqcZnZL448sp5UK+Ln5FbeZYMyCJP6mgnlf7S60+zJlT8mkUPlga69snysmvdOgrjATtRNChQUP5bl+9gE5rwWaINRAI6bwMb9hoqQAl2X6o09HGI7mLgI2nRjho1n1gWlVTASfOQJqjlxrKcUV3MuZE9LIBWxOXt+db8iP0W8DS30FQ7uYekEVslsx4At+l0C0dx97Nxe5k+gWdppAKKlRbqQSnammbMwQIETPwpbP6OmgQkQE5QxdhFm5c82rKNKEOFTOEx\\\\\\\/UWI2eEv81zqaAtScAb6erLwaMiSuuB1qXDgUe1e0IIpetcj\\\\\\\/Ty0Ob1UwJF\\\\\\\/n9WaqxC29aBaamUcPEDL8xCjDQpLzY6qg5a79SonNsBygzmqRUb2a8qt4RpYap6NNN4neGaFqaxtZ0ctRPCzqgBlP\\\\\\\/0cP6pUNPLAg3JazRaaZibRPJ1bJEeZspafonoI+hQ02kGJGmyOq+CmgdWm9P\\\\\\\/GoKTVSF\\\\\\\/PjtECcZcZRVQVIbhXVGHK4LrIc9JUFM7o4\\\\\\\/9JfOpu9BK+5mJTK6ff1sBijD\\\\\\\/MokrOK6FlHM9qiEIw43zHNvpuAZfUq18tDdkzvtcuWz\\\\\\\/x6MrU3Ij+7aAGFJbJyE43KOpHuJl+DcKWt\\\\\\\/7R0fJuhGIdf7EuPx0cMkKJwBoZnBXMzrwQOAU8hyMsOniaKULjKu6EKWXQwEGFYVekmcao0tD\\\\\\\/5xhvUkKoP+UeLW2GelprI2cF9PQw3j+OJS1xImyUEt17V9b15p7pe0fCdlR7G2GpkMb8Pk6adPD8oFQQ5zajVStc4oauvPYBTo5tYiJAVQuxtBLJAA1ZMl1SW2rWTc1QG4f+0\\\\\\\/Oqzw3OCCZQuBzBhf3k54fQ77eJmLAw\\\\\\\/H8Zh+0ujtqFC4wxf3NgWb8\\\\\\\/wCHgw0MEWUoJ2nlEQxJUmPrlMmn65JxUV3kKQouxtaefSDb7D0OCIGyIVqvM1060z0GIom4btpHYxlvWKyUbC6GJzucGP+UssDo75aoT2BnjJ1BuS1J1mqX+NA51ApKlKt4onknxDNw8d\\\\\\\/csZ5xT1iliqA4n+kLvlFv3l1mdFGw+U9zkdms4KGmEbwPLGCZ7RKsin04b4QBqfcfxZv\\\\\\\/kJBgiFYxAYbGntssQfpPzV0THKbRaGyu0QwRxfBtzKXUE6O7jYt7VaLUWc3Onuxylidh+OQSmoKP2edq2Kth\\\\\\\/Y+BDgzZjsuMjsTLR17RkqiRHUOycdwyoRmNI4\\\\\\\/ruB\\\\\\\/qLzJtosTYKWW4j8G31UdqY88gg1HV+rXSd3hp\\\\\\\/RzKAr1FQlRLrQzcz396XWQAkrUbb+1XvtanyWKhq0gwg\\\\\\\/MTG0nOOocbxTRquUPdqTcJm525rg25nmoXoX4Tgl8fMYtAqU+2q6Go\\\\\\\/\\\\\\\/CsHweNluIheiIRcFf1UoyB3rvLFl\\\\\\\/z+EVEUR\\\\\\\/2UuuVRti90tp9SFbOK5weQ9+j2qpA1LLB6VL+oBnE9P6Hn+wUQ7liciiTukqQOncpYBdrcqvrbxJoPwhZ+TjPD3XFWLPBDaZHDyHiDq7GIxgCDK3LWM5HKYYgwwj1RRaHVcn6gP3BK9dvm++ofNOISyz6fSgp\\\\\\\/vmdfWpThlMG3QDwzKfRPkNNL6NXFMYbbkQLyfe0b\\\\\\\/2UTK+MFpCVBXyyLNjsz2uB85NIYjEhl\\\\\\\/bD3HH\\\\\\\/cIHNjcK1g1YJdbIuQKEsBJFGNhIADiygcmM1pm\\\\\\\/I29Jw1GIV9Y0s01SevLhNEfNR3vI+4PLz8gA7nRGKGXP\\\\\\\/QaK5tiVRqzsEAl0XGBG6NN4gxKWxBEjclnyGMDP0CvA4y2NrwPvOoi3qbcW6k8A8oGf07oyQMJUr6R51rTGGememwNzTZVjEpBWp8unWwWCFZoS\\\\\\\/+i8KGn5BE8kj7Av83iB7mAOAlqFmx9G\\\\\\\/noa9x7NdOuo6OI2ftz31L2iT9zs34L9UYhO20uLWDlOSrPuOn3lQneoUoN8y3FrBSLHHNCjAUU+BCxa0igKTrJYo1JIVHM7DTQBs\\\\\\\/9zgn6kjz3UALpjyCUOhaZ3YjPmYBf0DNS3mZdVoEkOqbSCtkGfT5RrCoYRaOWgc6OHo1vi0nvdzoznuaKlrDxIFEFkeaYBCxf0cLDm7rR1JJ6BeEe3qJHeXE2iuFA\\\\\\\/GF+SkPsH5XOeZf07DprvUd8vuf+XSXaQAumbCyqwX+cKcIuBFc7knNqG8mb42fLOLhnlx2vfrpBrhiSwsW0dawdkmBhXGDIKb0v2PVRuoHJQ2NIjiTjlAufeOmh3\\\\\\\/q6iyr0MFaFzrfsWsLujATYBXY+zpwdtouRreUrdQVHO3l08Vh98fzZB+upAZ2fGdt1IrAEcq315AB258gIoxqAtR66yRbFYcwPvK+MNq7jEFjo1IqN08tNnpF7wgqjTFOr4uKkIHbEYpo5ek4VDelvLuiuN9jVyf5GtGNE9Gvkdru6mC0UQo\\\\\\\/uiTVnIMeZaEFdrFuo+9TxdodxTtTsG80CFJzAJ7yjaJm6EAxzzGfNR\\\\\\\/+qf+7GZ9qmKe6jmN88\\\\\\\/QUlFMpk2HJTna6q8sAgWyp\\\\\\\/+HuqV2sdBo32oaWauXo8yXwtSOj7GY0Uawn19XEALFQa3\\\\\\\/1+Www22cJgohnjVbeaEpXyBt2GcM+JBfhpbxM2FdVc4BepP\\\\\\\/Eusum92g8NPtqEvjWalE\\\\\\\/QnQ4dG3+ivEiziH51YsXiTcM\\\\\\\/zhSxl+m2zbcA7panEdVsp9wGNXBlB09bzICgdgh6iOvKveMFMiOvq32dIq75vPjpN0JSmVlMwfyH152W+nciqpDzi7jN18L8qLAFlvYlDEGRiBZHfIzvbDf2XEbimbwpwnuCQSNcUIH0Ff8+M\\\\\\\/WGsuWVtJK3f\\\\\\\/w1q1ohL\\\\\\\/bqS2Hc7ckzEXk5cVWnU9NhnO6foBxy2eoMDKMrb+cCbAPa5rtL6h2p6p4bXCRFCNRlPS73hYFekE\\\\\\\/3XFv53luBA65v0nEM\\\\\\\/qEqszsvkbBi7bvRfgzLI0G5mavJacEl\\\\\\\/HuaxYEzLmlVrWDCq51oZeo8wrgCxHYIl5JGXkp2OYI9keqHNSxmkwr4lPIcjFfjK88\\\\\\\/x5RhOPi2Wdev6w9on150pzfvIGHGEF0WqXrRXK\\\\\\\/BWoKAd9U562\\\\\\\/FV317g8\\\\\\\/uIh74bwkaJmRGCQy1OIsFoAkID1Gpgmtop\\\\\\\/gHNuCC3X5wDE\\\\\\\/cMCzDTz7TdySA8vm+D6L2m9nhtVM5+MKGNuUb4dpKQ5IOw8w8yWRsjvOTWq35f68Ia0w5yIi5ZLRkecKPD3Pc49M9sMZNQPEYoS7yVogLwlWkkWOk0bNfWUgqUfIShq6Gs3GeDnbedAJ3qKJ2DXzkOxH4shP2fxp6\\\\\\\/wSJj0AfNuTNH9SSR1zUokqSmEnK4D+4GCY+8oviu6Zw3itY5wgqcLWtE0vCwSdOVR7rBrdte+y78Djq+QvbvR5QrGYxYMG6KOWCbZWGTp3ZtGmKIj6Y3z+trouSwDmGrM6S4kf9QXUVVFvJnOmynatBnO9txs+iwqF89e6JEPzfQoYG\\\\\\\/nME75KM37YlmMKwGtOpRDUqG3Eex8M0Ck\\\\\\\/tBOhTIDOT9tArQ1LT62fxW5WJ6aZOwwMfF0sf7ZrGLbLsCixdaesJBvG0E4i3inyVsaiD29NqHvvBqTaRb+403Tu9olD\\\\\\\/ZdaoZULh9hBTFP4wjZB7wERkz80g6zMGfm2cYxRNzPk7Gh1Mq1u6\\\\\\\/9w+nDqATzXuen4MF6DliEpD\\\\\\\/\\\\\\\/ZkN9CNPtxHkL8zwIggd6MZq+Qikjc3uHo2W8Pc9jiE8PU\\\\\\\/majdRedpfBQhwyqMoNmLD1dSOfpyxfgZdlFEeagw2VQGFeO+vmmSwyWOUBTVgc1UHEI6OT1gL1pEgvnOmGjmJ3EeFGIL+0aXviWDid3M1DTc0vMcmnyQB\\\\\\\/AQHmBHlOjS0UWPm+9JPUt8VaiVJN0DFoF3cGt3sguv530eY+tMP5iS21gtv9fEbC52gE97bY8vki3m7gzMTDYbrnSdbPjTNRftECBkDqqdhVifAyFEq47ashRoYuXVEMJw4NWwS0OJX2QTzUYJdMpqbt\\\\\\\/6NlrvTov3GNqTszRZBP+rtnjts2syfnAsLQQTPX0bWEfTBZ65tZx1gHihJlcDWs5l3\\\\\\\/UzP6SDqfjQQwv11\\\\\\\/sSV5lOrtq8zo5s3SW5Y1A+ui+U0ulHGhCmllL70U6sJAoVEEgqz+LmoJn1OgbmpyvZAwxnwYmgh9HYiEqq6QauvLVvLU08z09wsw4b5oQ68fHA9R2klWNcD51a3+0BqkWlOSENdng7I8VpJWMPOIqRlyICIvo5TCO9do\\\\\\\/kL0tClR3p3oGWSPyJGphZllcW0tnmfithU7N\\\\\\\/3A3qC5FXXiRToEG2jZRHeYKIA2l4cGKSi3y3Ft1JrKciSxu6dDd28QnxEtZ9AfTXhPBPYgWNHbRjgf\\\\\\\/6XVI1h23Ynv0l8HCQgZgxZZOV60fm8cb5Qi5s1ZgFXxNZEIu+v5kehCwg0tcdCanRyauxcXUDnXBgplTifRx1aNc6iEAIMS5AnBUdm4Z\\\\\\\/Ee4dId6aJAdz9GEO7fXnT4jJwRhGC+Dt8CuRJ\\\\\\\/QR2zaBiGhKhxsxpjIaN8I\\\\\\\/j8X+IH1mNCDSk26xuoTpXymqo3EnfpaVIz2m2HkZ8kSiFD6QAWIGg4RbscTqqp09Yb5xK\\\\\\\/GmDJTPso0Qesy0AGAVRLAz2x\\\\\\\/GKb6aW1BSt8rtI2cmoVhaqZjwhysRNcsmzmP9j3\\\\\\\/pxXIwYCk1VxRLCw4z5ggKHA4awZ5cuFHZ\\\\\\\/Bqzvv0o62qlgyLuxRb\\\\\\\/BQY3juhHAMtE0a2GfQceTo4siVycIJuEfa9l4KZBlcpQDDibsSzwj1lBeN50KHrCR5xwkcYdmmuE584wK7NFdE+6chVccp0DNEYbF6KRi0HinLYL2ECC6P3NMevDsab6IJHWA1FByUkQ07mAtrK1Hrs8qSjtTEv34LwT+7EfpEh3xQ0Wdw9GnSLRN0fy5F8OR9u5dKI7ivp3IyPFzo7kdYVAmUk6N9xqwjotFbBge16hRBaI7TW4HTzTeoEl0r\\\\\\\/LXzaMI2wycsJU6+EntVphUY+8gaPvVv68IWbyfJl+6CbSHPtm7iUKnVht3JhTgcwtLIJl1DviE\\\\\\\/OfXFHIVj97ZbZI0a43Zl\\\\\\\/lwgr1+vL1KfVTB0tSx8k7vEeA7nu6gg5uvJafRDzaYQC2UpvJMqqdiARvyltPZfH2Af1Idct4TMBb6NTU0OG1Ac8mTdPca8PWTKw7g1V5PSDIQoK6Mo1C+N9svCtnWDWmB2Z6pRddLsQI8cwScEYVSKc2t9Ams+6JPt8JV7hJaKI5mjdeGeo44Y5rhxWOZgUy0gfAqRyEGWlQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022es\\\\\\\/CV18dudCUP5RSWtU+Ew==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002211cfebcf92952f7f060a273e9124b43e3f49868c9e52c2ea04c302800f6a2fac\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Sc\\\\\\\/oTcZ9Q+rJDue73IWuffEXGY7xkYGWXKEPzrbdRU1LxtMUXeKpqfRoBdySMg2WB4lMOmlvxinym7+J3pf7vcbex5yPjqO8FntA9gFBK8yTfuadJwH40LUya5gWyg1kVmYrDyLtIOfDqJOjThv7N6UVgGeae3tPD4hSAVzA9kRXQdJ6sX99KyieBjOxke1Q3yXEX2QX0DDERmdoZdZDAXb9hrRUfUEG7Ob1KvMO+078z99C+jnou58MmQjNaPiCkcYq+vWARD+sjqysXTFwmCcQNtSbDvoI5DNv7iKjWJG\\\\\\\/Lx9O52J7fyI5iiyWUYbB08YSYx3zxRYoKrm\\\\\\\/BnqiSy\\\\\\\/COv5gl2MlWPGpXpdcClhcXHVEUQkcMhQggai3d6YHOZOe6bY+hqvk2KR\\\\\\\/+kuKHy0N13cAcIaCynu5Q42OA1ddPT3EnlDoUjdE8NZZ7UOi1ZSuxio4kdYOIXVMIlD9CeVxnBizeui9rqj6SOh60V21PykWq2IKTC2fqh9obCBe3h1e2aSEvEPOt9+d9Ik9ofi+iK7KEQs2oGGpeolDMaTLWjso8B98D6tRUeqgnra9RN8eaykqHSv33zB1eowCz7baHlyZyl54yywlvBV5YlPYWdYrH1dFNIolXnoJskW1ykz8qtuC+ORbQ\\\\\\\/0LzLQ06t8mJLR6\\\\\\\/4gRE4do2nCBIu590XwqANpKFHXiByJadbH7lraOqrwmOCnUhzuRN3Tm8CawuhXRlP5\\\\\\\/dLCnuWWrUSptOpTDuGWtVdzSyChT93JxQ1aYhOQjvHdYgjY4IQO3Q7K3gUR5lTyPgK5gNFSQQzHCSf\\\\\\\/\\\\\\\/VJPMen7JDONljz91\\\\\\\/QRsO\\\\\\\/rteUaprky9xQ+VvFgya55L4eORUmgbnxaAkirCbEmzKTOuWEKrtyIGzvDfhQXjSu7BKv6vsWrldFuKdQLBHQ5IKDZRwsCO1BBj0JHkE9ZA9DGLYtj\\\\\\\/0AoITGZA9Pz0RrSkXcSsmWAvpyeuCpmjVv1kwDbAhUAzBvdK+BBFUBT5Vj8QcrkwAtf\\\\\\\/jSOIYu8yWhhGcwQi9cX2ZwZv0kEF2baW2KA5pBcrnPk5lCpFmTaz9Svo8UHbTaDZO0QRDEtjAEN0GC6RQWwuwGptS+RZrl14hfpVjaLjHr\\\\\\\/3McPlCaDMJAiOTOwIVg+jWujTEtaZryodORbRXxDkG8vr5\\\\\\\/Z1FmPJ2YxLTFA\\\\\\\/lirfqq5GT1eoVOurtA2tb5Jmdr51CYsmnug15ImnISaGOGd4dHV7aSkPkJPLTWq09sgRp+oGTlSTkajXB5aRgirKglEOoHKMrosjertsljN3wEfOc6Sajtfxy5roqIt9EqFpR1xFTpGwL82IgykaE2PVoXyMgeTqQkLaws4KyxyceMztieFHpl\\\\\\\/L1H2XdEFvKPn9L8xWSPS7ttZwJ3+Wu4m+RsXHpuQtXhGVL96NX2jvjyJErnt\\\\\\\/weJzwAoZgo0BiWZV0Pb3nlXA9SbZwbInWXhvL1SU4jGWYUZHf6awBSmZRNckALDIBfv7BBqnUdKawu\\\\\\\/tQUYnI7BfBkTeo1yNqWPXI9IiEvlU5zj7roWhCN4AMuNt4Yk1XF0sOLP3FWQFVzvEHkgaUlzhilfaevxbQAnAUJ+uizYCWfHKR4qddKXYKa2CHLmFNgCzTKPQvshVy0kojnKJqrJVI6zRRbrfR9Ddq+vT+rQFERmSVC\\\\\\\/VhyMeW63YAVqfss7opnlO7P44xNJKD6RZDlegzD6T9QHkSGkzvs5N3iCmlmzxUDxKG6wn9r7h6OPiKhZAX9lzxdCOAtrj7gkTD\\\\\\\/qGs1UwMAalbVvv6Te9yW5Y6xcMqyR9wgZCNeKCug9XG1NX2Hu7+rTtXLKac5X3WHHethb51Ojt2WTCu94fWj+DJBJ+vfp3SUzf2+hvZbRItkAOF9t2goSD3uXTY95I0o1kt6IGA7pjpL+8sFuJWa6WpwTAStAOkiDjHMgy2aLujcvtfZHC9cPeFNaTjPVntk33waAgLQft+yCmudx0QmzDW5hKb7UEJX0r77XfXkIwn8U4GABvSh4rmiqTIzcHP5ZKMzfkLcpZHvKBl9Io\\\u0022,\\\u0022iv\\\u0022:\\\u0022Le1rVd1QRAJjTfpQr+Cw1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227bd3f9ffc9900a00a091b64561f7dddb317205f958efe34136cf2b306e6f301e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490132 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2501" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-z4TxNcoMdFPLlqk9rZLi7KAnVrykeruowDEo8leO3Nk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2501" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vj5n5ZUMs4dL5b8ue-3RiT3ZzKeASYVkYQ4rmIoze5E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/7/2483" title="Genome-Wide Analysis of Multidrug and Toxic Compound Extrusion (MATE) Family in Gossypium raimondii and Gossypium arboreum and Its Expression Analysis Under Salt, Cadmium, and Drought Stress" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/7/2513" title="Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490132" id="node490132" data-pisa="ggg;8/7/2501" data-pisa-master="ggg;g3.118.200401" data-apath="/ggg/8/7/2501.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Networks Predict Neuronal Transdifferentiation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3350-5692" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Richard I. Ainsworth</span>, <span class="highwire-citation-author" data-delta="1">Rizi Ai</span>, <span class="highwire-citation-author" data-delta="2">Bo Ding</span>, <span class="highwire-citation-author" data-delta="3">Nan Li</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3454-7357" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kai Zhang</span> and <span class="highwire-citation-author" data-delta="5">Wei Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2501-2511;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200401 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490132" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Richard I. Ainsworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BI.%2BAinsworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainsworth%20RI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BI.%2BAinsworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3350-5692" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Richard I. Ainsworth</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rizi Ai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rizi%2BAi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ai%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARizi%2BAi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bo Ding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BDing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ding%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BDing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nan Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kai Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Graduate Program in Bioinformatics and Systems Biology, University of California, San Diego, La Jolla, California 92093-0359</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKai%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3454-7357" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kai Zhang</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wei-wang@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">wei-wang@ucsd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/7/2501" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490132" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2501.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490132" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2501.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490132" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2501.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490132" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/7/2501.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490132" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We employ the language of Bayesian networks to systematically construct gene-regulation topologies from deep-sequencing single-nucleus RNA-Seq data for human neurons. From the perspective of the cell-state potential landscape, we identify attractors that correspond closely to different neuron subtypes. Attractors are also recovered for cell states from an independent data set confirming our models accurate description of global genetic regulations across differing cell types of the neocortex (not included in the training data). Our model recovers experimentally confirmed genetic regulations and community analysis reveals genetic associations in common pathways. Via a comprehensive scan of all theoretical three-gene perturbations of gene knockout and overexpression, we discover novel neuronal <em>trans</em>-differrentiation recipes (including perturbations of SATB2, GAD1, POU6F2 and ADARB2) for excitatory projection neuron and inhibitory interneuron subtypes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/systems-biology" class="hw-term hw-article-keyword hw-article-keyword-systems-biology" rel="nofollow">systems biology</a></li><li class="kwd"><a href="/keyword/gene-regulation-network" class="hw-term hw-article-keyword hw-article-keyword-gene-regulation-network" rel="nofollow">gene regulation network</a></li><li class="kwd"><a href="/keyword/neuroscience" class="hw-term hw-article-keyword hw-article-keyword-neuroscience" rel="nofollow">neuroscience</a></li></ul><p id="p-3">The classification of cortical neurons is a debated topic with differing schemes using anatomical, molecular and physiological characteristics in order to make distinctions. It is generally accepted that there exist two major groups of neurons namely, excitatory Projection Neurons (PNs) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Greig <em>et al.</em> 2013</a>) and inhibitory Interneurons (INs) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kepecs and Fishell 2014</a>). All neurons are generated only during embryonic development (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lodato <em>et al.</em> 2015</a>) after which class-specific traits remain unchanged for the life of the organism. It is classically thought this precludes any change in identity postnatally. Intriguingly neurons may exhibit more plasticity than previously thought. As far back as 2002, astrocytes were directly reprogrammed into neurons and more recently post-mitotic neurons have been converted from one subtype to another in young animals as reviewed by Amamoto <em>et al.</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amamoto and Arlotta 2014</a>).</p><p id="p-4">A Bayesian network (BN) is a graph-based model of joint multivariate probability distributions that captures properties of conditional independence between variables. Bayesian networks can be used for representing statistical dependencies in a set of data and were applied to the problem of reconstructing gene regulation networks (GRN) from expression data by Friedmann <em>et al.</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Friedman <em>et al.</em> 2000</a>) and Hartemink <em>et al.</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hartemink <em>et al.</em> 2001</a>). It is known that the protein transcription factor produced by one gene can have a causal effect on the expression of another gene. BN can be used to represent the conditional dependencies between genes and thus interpret these as causal patterns of gene regulations.</p><p id="p-5">We challenge the paradigm that neurons of the mammalian cortex are a permanently post-mitotic and differentiated cell type via modeling genetic perturbations that facilitate direct transdifferentiation. In this theoretical study we present the application of Bayesian network techniques to high quality deep-sequencing data in order to reverse engineer the genetic regulations in human neurons. We identify those attractors that correspond to different neuron subtypes and validate our model with an independent data set. Using dynamic bayesian inference we derive interconversion recipes between differing neuron subtypes and from the perspective of the cell-state potential energy landscape, describe those interconversion pathways.</p><div class="section methods" id="sec-1"><h2 class="">Method</h2><div id="sec-2" class="subsection"><h3>Data processing, clustering and discretization</h3><p id="p-6">Lake <em>et al.</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lake <em>et al.</em> 2016</a>) previously conducted single-nucleus RNA sequencing on post-mortem adult human cerebral cortex and generated 3,227 quality-filtered single neuron data sets. These nuclei were subsequently resolved into 17 clusters, based on the differential regulation of 16,242 protein-coding genes, through repeated rounds of unsupervised hierarchical clustering and supervised classification (technical details can be found in ref. (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Lake <em>et al.</em> 2016</a>)). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> A.i. is a reproduction of the authors hierarchical tree down to Level 2 including the clusters considered in this work. At each of the 3 splits, we consider the ten-fold differentially expressed genes (DEGs) giving a total of 74 unique genes. Transcription levels were previously analyzed as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-1.gif"/></noscript></span></span> of transcript per million mapped reads (TPM). Thus, for the complete data set at Level 2 (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-2.gif"/></noscript></span></span> samples) we calculate the weighted arithmetic mean for each gene <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-3.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-4.gif"/></noscript></span></span> where <em>n</em> is the number of samples in each cluster. Each data point was subsequently discretized according to:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>thus transforming the data:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-6.gif"/></noscript></span></span>where gene <em>x</em> runs from 1 to <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-7.gif"/></noscript></span></span> and sample <em>j</em> runs from 1 to <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-8.gif"/></noscript></span></span> The experimental barcodes for each cluster, post-discretization, are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Workflow pipeline. A.ii. Hierarchical clustering of 3,227 quality-filtered single neuron data sets from previous single-nucleus RNA sequencing study. Number of 10-fold up-(&#x2191;) and down-regulated (&#x2193;) DEGs given at each junction. These are defined as up-regulated in the subsequent left hand branch and down-regulated in the left hand branch each relative to the right hand branch. Terminal clusters I - IV at Level 2 used in this work.(Lake et al. 2016). A.i. 77 ten-fold DEGs (of which 74 are unique) used to train networks. Splits refer to junctions in the hierachical tree in Figure 1 A.ii. Genes in bold occur in multiple splits. B. Data discretization on the weighted arithmetic mean of the log2(TPM) for each gene across 3,203 Level 2 single neurons. Data downsampled to 1,920 samples. C. Structure learning directed acyclic gene reulation networks using the discretized downsampled data with local and global optimization routines. 20 random seeds used to generate 20 different structures. 5 structures randomly chosen for further calculations. D.i. 450,000 random initialisations of the nodes in the continuous interval [0,1] for the 5 network structures. 2 TBN DBN inference performed for each network in each initial state. Converged attractor states subsequently clustered. D.ii. Nodes in 5 network structures initialised in the continuous interval [0,1] corresponding to the four neuronal subtype cell states. For each network structure in each initial state, 2 TBN DBN inference carried out for all 3-gene perturbation combinations (clamping nodes as overexpressed or knocked out for duration of inference). Subsequent node probabilities averaged over the 5 structures for each 3-gene perturbation in each initial state." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108535840" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Workflow pipeline. A.ii. Hierarchical clustering of 3,227 quality-filtered single neuron data sets from previous single-nucleus RNA sequencing study. Number of 10-fold up-(&#x2191;) and down-regulated (&#x2193;) DEGs given at each junction. These are defined as up-regulated in the subsequent left hand branch and down-regulated in the left hand branch each relative to the right hand branch. Terminal clusters I - IV at Level 2 used in this work.(Lake &lt;em&gt;et al.&lt;/em&gt; 2016). A.i. 77 ten-fold DEGs (of which 74 are unique) used to train networks. Splits refer to junctions in the hierachical tree in Figure 1 A.ii. Genes in bold occur in multiple splits. B. Data discretization on the weighted arithmetic mean of the log&lt;sub&gt;2&lt;/sub&gt;(TPM) for each gene across 3,203 Level 2 single neurons. Data downsampled to 1,920 samples. C. Structure learning directed acyclic gene reulation networks using the discretized downsampled data with local and global optimization routines. 20 random seeds used to generate 20 different structures. 5 structures randomly chosen for further calculations. D.i. 450,000 random initialisations of the nodes in the continuous interval [0,1] for the 5 network structures. 2 TBN DBN inference performed for each network in each initial state. Converged attractor states subsequently clustered. D.ii. Nodes in 5 network structures initialised in the continuous interval [0,1] corresponding to the four neuronal subtype cell states. For each network structure in each initial state, 2 TBN DBN inference carried out for all 3-gene perturbation combinations (clamping nodes as overexpressed or knocked out for duration of inference). Subsequent node probabilities averaged over the 5 structures for each 3-gene perturbation in each initial state.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2501/F1.medium.gif" width="308" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/7/2501/F1.medium.gif" width="308" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490371" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Workflow pipeline. A.ii. Hierarchical clustering of 3,227 quality-filtered single neuron data sets from previous single-nucleus RNA sequencing study. Number of 10-fold up-(â) and down-regulated (â) DEGs given at each junction. These are defined as up-regulated in the subsequent left hand branch and down-regulated in the left hand branch each relative to the right hand branch. Terminal clusters I - IV at Level 2 used in this work.(<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Lake <em>et al.</em> 2016</a>). A.i. 77 ten-fold DEGs (of which 74 are unique) used to train networks. Splits refer to junctions in the hierachical tree in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> A.ii. Genes in bold occur in multiple splits. B. Data discretization on the weighted arithmetic mean of the log<sub>2</sub>(TPM) for each gene across 3,203 Level 2 single neurons. Data downsampled to 1,920 samples. C. Structure learning directed acyclic gene reulation networks using the discretized downsampled data with local and global optimization routines. 20 random seeds used to generate 20 different structures. 5 structures randomly chosen for further calculations. D.i. 450,000 random initialisations of the nodes in the continuous interval [0,1] for the 5 network structures. 2 TBN DBN inference performed for each network in each initial state. Converged attractor states subsequently clustered. D.ii. Nodes in 5 network structures initialised in the continuous interval [0,1] corresponding to the four neuronal subtype cell states. For each network structure in each initial state, 2 TBN DBN inference carried out for all 3-gene perturbation combinations (clamping nodes as overexpressed or knocked out for duration of inference). Subsequent node probabilities averaged over the 5 structures for each 3-gene perturbation in each initial state.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Experimental barcodes for clusters I - IV post data processing. For gene expression probabilities  code displayed as green and for gene expression probabilities  code displayed as red. Genes that display the same pattern across all four clusters are displayed on the same row. Genes are colored red, purple, green and blue for those expressed in clusters I - IV respectively. DEG that are expressed in both PN subtypes I and II are colored magenta and DEG expressed in both IN subtypes III and IV are colored cyan." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108535840" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Experimental barcodes for clusters I - IV post data processing. For gene expression probabilities &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; code displayed as green and for gene expression probabilities &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; code displayed as red. Genes that display the same pattern across all four clusters are displayed on the same row. Genes are colored red, purple, green and blue for those expressed in clusters I - IV respectively. DEG that are expressed in both PN subtypes I and II are colored magenta and DEG expressed in both IN subtypes III and IV are colored cyan.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2501/F2.medium.gif" width="440" height="118"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/7/2501/F2.medium.gif" width="440" height="118"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490345" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-8" class="first-child">Experimental barcodes for clusters I - IV post data processing. For gene expression probabilities <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F2/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F2/embed/mml-math-9.gif"/></noscript></span></span> code displayed as green and for gene expression probabilities <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F2/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F2/embed/mml-math-10.gif"/></noscript></span></span> code displayed as red. Genes that display the same pattern across all four clusters are displayed on the same row. Genes are colored red, purple, green and blue for those expressed in clusters I - IV respectively. DEG that are expressed in both PN subtypes I and II are colored magenta and DEG expressed in both IN subtypes III and IV are colored cyan.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">In order to prevent the network topologies being biased toward any one neuronal sub-type, the discretized data were down-sampled via the random removal of samples from clusters I, II and III until each cluster contained 480 samples, thus matching the lowest cluster size, that of cluster IV. For each of the three downsampled clusters, the appropriate number of samples were randomly selected for removal and the probability of expression (assigned according to the fraction of the 480 samples that were in a state 1 post-discretization) was calculated for each gene. Comparison was then made to the probability of expression for each gene from the complete, non-downsampled cluster via the root mean square deviation (RMSD) and Pearson R correlation summed over all 74 genes. This process was repeated <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-11.gif"/></noscript></span></span> times for each cluster and the set of samples that gave the lowest resultant RMSD were removed. The <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-12.gif"/></noscript></span></span> values for each downsampled cluster compared to the relevant complete cluster were <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-13.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-14.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-15.gif"/></noscript></span></span> This gave a data matrix of size <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-16.gif"/></noscript></span></span> for 20 separate structure learning runs.</p></div><div id="sec-3" class="subsection"><h3>Structure learning</h3><p id="p-10">For a given directed acyclic graph (DAG) model G based on data <em>D</em> with <em>n</em> binomial variables, it can be shown that<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>n</em> is the number of variables, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-18.gif"/></noscript></span></span> is the number of instantiations of the parents <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-19.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-20.gif"/></noscript></span></span> is the ascertained prior belief of the number of times <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-21.gif"/></noscript></span></span> takes its first value when parents <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-22.gif"/></noscript></span></span> are in their <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-23.gif"/></noscript></span></span> instantiation, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-24.gif"/></noscript></span></span> is equivalent to <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-25.gif"/></noscript></span></span> but with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-26.gif"/></noscript></span></span> taking its second value, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-27.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-28.gif"/></noscript></span></span> is the number of times in the data <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-29.gif"/></noscript></span></span> takes its first value when parents <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-30.gif"/></noscript></span></span> are in their <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-31.gif"/></noscript></span></span> instantiation, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-32.gif"/></noscript></span></span> is equivalent to <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-33.gif"/></noscript></span></span> but with <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-34.gif"/></noscript></span></span> taking its second value, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-35.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-36.gif"/></noscript></span></span> is the gamma function (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Neapolitan 2009</a>). Equation 2 is defined as the Bayesian score assuming Dirichlet priors. In order to punish overly complex DAGs and reduce the possibility of overfitting, we use the Bayesian information criterion (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-37.gif"/></noscript></span></span>) to score structures:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-38.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>which includes an error term, where <em>m</em> is the number of samples and <em>d</em> is the dimension of the DAG <em>i.e.</em>, the number of parameters.</p><p id="p-11">Our procedure follows a three-stage score-based approach common to the method of Chang <em>et al.</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chang <em>et al.</em> 2011</a>) but omitting prior knowledge incorporation and is thus purely data driven. We give a brief overview here for the readers convenience. Since the problem of learning optimal structure is NP-hard (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chickering <em>et al.</em> 1995</a>) we use heuristics in the form of the greedy algorithm to maximimise the <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-39.gif"/></noscript></span></span> score during the first stage. Starting with an empty network, two random nodes (genes) A and B are selected. If no edge exists between them, either the directed edge A â B or the opposite regulation B â A is generated, each with a probability of 0.5. If an edge already exists between the two nodes (a possibility from the second step onwards), it is either reversed or deleted each with a 0.5 probability. For each of the four outcomes, the change to the network is accepted if the <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-40.gif"/></noscript></span></span> score increases, else it is rejected. In our case, this started with an empty network and was iterated <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-41.gif"/></noscript></span></span> times (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-42.gif"/></noscript></span></span> NSteps).</p><p id="p-12">During the second stage, we employ the metaheuristic approach of simulated annealing in order to approximate the global minimum. The network is instantaneously heated to a temperature T and uniformly cooled over the course of the stage. According to the logic set out in stage 1, an edge is generated. However, if this leads to an unfavorable decrease in the <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-43.gif"/></noscript></span></span> score, the edge is accepted with a probability of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-44.gif"/></noscript></span></span> <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows this probability as a function of the <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-45.gif"/></noscript></span></span> for a given edge introduction, at different temperatures. Stage 2 was iterated <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-46.gif"/></noscript></span></span> times (NSteps) from a starting <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-47.gif"/></noscript></span></span> down to <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-48.gif"/></noscript></span></span></p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Acceptance criterion for edges that are unfavorable to  score during simulated annealing stage of structure learning. Representative temperatures in the range  plotted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108535840" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Acceptance criterion for edges that are unfavorable to &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; score during simulated annealing stage of structure learning. Representative temperatures in the range &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-50&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; plotted.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2501/F3.medium.gif" width="440" height="374"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/7/2501/F3.medium.gif" width="440" height="374"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490172" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-13" class="first-child">Acceptance criterion for edges that are unfavorable to <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F3/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F3/embed/mml-math-49.gif"/></noscript></span></span> score during simulated annealing stage of structure learning. Representative temperatures in the range <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F3/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F3/embed/mml-math-50.gif"/></noscript></span></span> plotted.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Stage 3 consists of a final stage <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-51.gif"/></noscript></span></span> maximization via the greedy algorithm, identical to the protocol of stage 1. This was iterated <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-52.gif"/></noscript></span></span> times (<span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-53.gif"/></noscript></span></span> NSteps). The change in the average <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-54.gif"/></noscript></span></span> score was converged with the value of NSteps as shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Convergence of  (referenced from structure with 1.1M Nsteps) with NSteps (defined as number of steps during simulated annealing phase). Each value is the arithmetic mean over 5 independent structure runs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108535840" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Convergence of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-55&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (referenced from structure with 1.1M Nsteps) with NSteps (defined as number of steps during simulated annealing phase). Each value is the arithmetic mean over 5 independent structure runs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2501/F4.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/7/2501/F4.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490298" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-15" class="first-child">Convergence of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F4/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F4/embed/mml-math-55.gif"/></noscript></span></span> (referenced from structure with 1.1M Nsteps) with NSteps (defined as number of steps during simulated annealing phase). Each value is the arithmetic mean over 5 independent structure runs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Since the data used was non-temporal, only edges that did not introduce a loop into the structure were accepted. We employed this structure learning procedure using 20 random seeds and learnt an ensemble of models.</p></div><div id="sec-4" class="subsection"><h3>Inference</h3><p id="p-17">During inference we apply Bayesâ rules to obtain the posterior probabilities. Since we are interested in developing a gene regulation model that can be initialised into neuronal subtype cell states; for each source cluster I - IV (see <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> A.i.), initial probabilities for each node <em>x</em> were defined as the probability that the node was in a state 1, <em>i.e.</em>, expressed, for the given cluster and were input as values in the continuous interval <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-56.gif"/></noscript></span></span> These probabilities were assigned according to the fraction of the total samples, in the given cluster, that where in a state 1 post-discretization.</p></div><div id="sec-5" class="subsection"><h3>Model averaging</h3><p id="p-18">We use the full Bayesian framework and do not attempt to approximate one true underlying distribution with a single structure. The <em>a posteriori</em> distribution of models is:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-57.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-58.gif"/></noscript></span></span> is the likelihood of the model and the prior, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-59.gif"/></noscript></span></span> is not assumed to be uniformly distributed and thus not constant. We therefore perform probabilistic inference by model averaging where the averaged conditional distribution of variable <em>X</em> is obtained by integrating over models:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-60.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>Since the model space is written as <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-61.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-62.gif"/></noscript></span></span> is the continuous ensemble of the conditional probability table (CPT) configurations for each structure <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-63.gif"/></noscript></span></span> For every <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-64.gif"/></noscript></span></span> each possible parameterization in the CPT configuration ensemble <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-65.gif"/></noscript></span></span> defines a member, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-66.gif"/></noscript></span></span> and the distribution is normalized against all models:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-67.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where the normalization factor <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-68.gif"/></noscript></span></span> Equation 5 is thus extended to a double-integral over structure space and structure-dependent parameter space:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>The Bayesian network models <em>k</em> sum over discrete structure space and the parameter vector configuration <em>Î¸</em> integrates over continuous parameter space and can thus become intractable by analytical methods. Within the Markov Chain Monte Carlo (MCMC) approach we use the order statistics of a uniform distribution [0,1] to simulate a sample from posterior distribution <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-70.gif"/></noscript></span></span> (a beta-density function).</p></div><div id="sec-6" class="subsection"><h3>Learned parameter retrieval</h3><p id="p-19">For a given structure, the probability of gene <em>X</em> with no parents remains as the initial probability for the course of the inference run. For gene <em>X</em> with one parent gene <em>Y</em>, the probability of expression of gene <em>X</em> (<em>i.e.</em> <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-71.gif"/></noscript></span></span>) is dependent on the two parameters <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-72.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-73.gif"/></noscript></span></span> which represent the two respective conditional probabilities. We calculate these conditional probabilities according to the associated beta-density function as:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-74.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>and<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-76.gif"/></noscript></span></span> are the number of data samples where <em>Y</em> and <em>X</em> occur with the values <em>h</em> and <em>k</em> in the discretization <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-77.gif"/></noscript></span></span> The additive value of 1 is for the case that there are no data samples in the given instantiation. A gene X with multiple parents <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-78.gif"/></noscript></span></span> has <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-79.gif"/></noscript></span></span> associated with it, where <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-80.gif"/></noscript></span></span> is the number of parents of gene <em>X</em>. Thus the more general formulation for the parameters is:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-81.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-82.gif"/></noscript></span></span> is the number of samples in the data where the parents of <em>X</em> occur collectively in state <em>j</em> and <em>X</em> occurs in state <em>k</em> <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-83.gif"/></noscript></span></span></p></div><div id="sec-7" class="subsection"><h3>Dynamic Bayesian network</h3><p id="p-20">To simulate the evolution of cell state from initial state to the equilibrium or steady state solution we use the dynamic Bayesian network (DBN) model. Here the probabilistic inference is performed by using a 2-time slice Bayesian network (2TBN) and the interface algorithm, (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Murphy 2002</a>) which uses static junction trees as a subroutine to compute exact inference in the 2TBN which is then repeated sequentially over time. Accordingly node (gene) probabilities (expressions) evolve over time according to:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>where <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-85.gif"/></noscript></span></span> is the product of the probabilities of each parent of <em>X</em> being in the binary state <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-86.gif"/></noscript></span></span> corresponding to the collective state <em>j</em>. Thus the probability of <em>X</em> being expressed at time <em>t</em> depends on its parameters, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-87.gif"/></noscript></span></span> as well as the probabilities of expression of its parents <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-88.gif"/></noscript></span></span> at time <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-89.gif"/></noscript></span></span> Each set of parameter samples <em>Î¸</em> forms an instance of the DBN model which were averaged for each inference run. The inference runs for each structure <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-90.gif"/></noscript></span></span> were then averaged for the final result. During perturbations, node combinations were either clamped at <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-91.gif"/></noscript></span></span> (overexpression) or <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-92.gif"/></noscript></span></span> (knockout) for the duration of the inference run.</p></div><div id="sec-8" class="subsection"><h3>Landscape analysis</h3><p id="p-21">In order to identify those attractors in the unperturbed state, we randomly initialise the genes in the continuous interval <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-93.gif"/></noscript></span></span> and perform 2TBN DBN. The resulting method corresponds to converging on the minima accessible to the region of cell state space in which the initialisation took place. As per the method of inference under perturbation, node probabilities evolve over time according to Equation 11. In the case of attractor identification we do not apply any clamps and all nodes are free to evolve to a steady state solution. For all inference calculations a set of parameter samples <em>Î¸</em> forms an instance which are averaged over for each unique random initialisation. This process was repeated 450,000 times until the number of unique attractors had converged (234 attractors defined in binary cell state space) thus the sample is large enough to accurately describe the entire space. The basin size of the potential energy landscape corresponding to each attractor can then be calculated as a percentage of the total number of samples that converge to each state.</p></div><div id="sec-9" class="subsection"><h3>Transition states</h3><p id="p-22">Transition states during inference were calculated as per Chang <em>et al.</em> (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chang <em>et al.</em> 2011</a>) by applying a maximum-a-posterior (MAP) estimation to predict the state-transition pathways. That is, at each time step t, the state which maximizes the cell state posterior at the current time step is selected as the current cell state:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-94.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>where the probability propagation in DBN cell states is defined as:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-95.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>For each unique binarised cell state, the state probability for the <em>i</em>th state (<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-96.gif"/></noscript></span></span>) and thus the potential energy, <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-97.gif"/></noscript></span></span> of the state <em>i</em> can be calculated. These potential energy values for all <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-98.gif"/></noscript></span></span> binary states can be represented as <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-99.gif"/></noscript></span></span> Cell potentials differ according to cell state conditions. In this work we investigate the cell state potential changes for specific transition states under a specific 3-gene perturbation <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-100.gif"/></noscript></span></span></p></div><div id="sec-10" class="subsection"><h3>Method validation</h3><p id="p-23">Buganim <em>et al.</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Buganim <em>et al.</em> 2012</a>) have previously conducted a single-cell gene-expression analysis of mouse embryonic fibroblasts (MEFs) during cellular reprogramming. They profiled 48 genes from early time points, intermediate cells, and fully re-programmed iPSCs. These data were used to train a simplified Bayes model of hierarchical gene regulation in iPSCs. Using their regulation topology they chose five transcription factor combinations predicted to induce activation of the pluripotency circuitry and generate fully reprogrammed iPSCs. These were experimentally verified via flow cytomeric analysis using OCT4-GFP with <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-101.gif"/></noscript></span></span> reprogramming efficiency. Using this independent dataset we learnt 20 independent structures and performed inference calculations to predict the effect of their experimentally verified three and four gene overexpression combinations using our methods. The reference Pearson R correlation between the MEF initial cell-state and the fully reprogrammed iPSc final state was calculated to be 0.155. The average Pearson R correlation of the experimentally verified reprogramming recipes and the final iPSc cell-state was predicted to be 0.838 using our methods. This was compared to 20 random 3-gene overexpression combinations with an average Pearson R correlation of 0.270.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-24">Data used has been previously deposited with dbGaP (accession phs000833.v3.p1), curated by the NIH Single Cell Analysis Program Transcriptome (SCAP-T) Project (<a href="http://www.scap-t.org">http://www.scap-t.org</a>) as stated in Lake <em>et al.</em>(<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Lake <em>et al.</em> 2016</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6349553">https://doi.org/10.25387/g3.6349553</a>.</p></div></div><div class="section results-discussion" id="sec-12"><h2 class="">Results And Discussion</h2><div id="sec-13" class="subsection"><h3>Experimental expression profiles</h3><p id="p-25">Lake <em>et al.</em> (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Lake <em>et al.</em> 2016</a>) categorized excitatory PN by layer position and as such, cluster I neurons (which were further split in their hierachical tree) were labeled as a combination of granular neurons (GN) from layer 4, sub-cortical projection neurons (SCPNs) from layer 5 and cortico-thalamic projection neurons (CThPNs) from layer 6. Cluster II were classified as cortical projection neurons (CPNs) residing in layers 2/3. The CPNs were shown to express CPN-associated CUX2 and the layer 2/3 marker gene LAMP5 both of which were 10-fold DEGs between clusters I and II and thus included as nodes in our network. Functionally, layers 1-3, termed the supragranular layers, are unique in the neocortex and are the primary origin and termination of intracortical connections. These can be functionally contrasted with internal granular layer 4 and infragranular layers 5 and 6. RORB a marker for layer 4 neurons (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Schaeren-Wiemers <em>et al.</em> 1997</a>) is also shown to be up-regulated in cluster I compared with cluster II consistent with this analysis. Interneuron subcategories were found to be distributed across across the neocortex and were classified based on developmental origin. (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Lake <em>et al.</em> 2016</a>) Cluster IV IN were found to originate from lateral (LGE), or caudal ganglionic eminences (CGE) and were VIP+ and RELN+ with positive expression of P8 and NR2F2. Whereas cluster III IN showed MGE marker expression such as LHX6 and SATB1.</p><p id="p-26">The 74 unique genes used to train the GRNs in this work are given in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> A.ii. They are those that are 10-fold DEGs between the clusters at Levels 1 and 2 of the hierarchical tree (see <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> A.i.). The retrieval of expression profiles for clusters I - IV post data processing and discretization was conducted via the generation of experimental barcodes (see <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). For each cluster, the gene probabilities were calculated as outlined in the Methods section and subsequently binarised based on a cutoff of 0.5. In this way each cluster is represented as one of the <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-102.gif"/></noscript></span></span> possible states. As a further assessment of the clustering and cluster uniqueness, we compared the RMSD and Pearson R correlation, summed over all node probabilities in <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-103.gif"/></noscript></span></span> between all four clusters (see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table I</a>). From this we retrieve the fact that inter-neuron expression differences for the excitatory PN clusters I and II are less distinct (<span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-104.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-105.gif"/></noscript></span></span>) than those between the IN clusters III and IV (<span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-106.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-107.gif"/></noscript></span></span>) in agreement with Lake <em>et al.</em> (<a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Lake <em>et al.</em> 2016</a>). For all other intra-neuron subtype comparisons (<em>i.e.</em>, PN sub-categories <em>vs.</em> IN sub-categories) we find negative correlations exist in the range <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-108.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-109.gif"/></noscript></span></span> thus find, with the possible exception of comparison between clusters I and II, the cluster expression profiles to be adequately unique (within our state space defined by the 74 DEGs) after data processing and discretization.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490290/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490290/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE I:</span> <span class="caption-title">Experimental RMSD/Pearson R correlation, summed over all 74 node probabilities, between clusters I - IV</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Landscape analysis</h3><p id="p-27">Due to the computational intensity of 2TBN DBN inference over a sufficiently large sample of state space, five structures were randomly chosen, from the ensemble of 20 BN structures that were learnt, for landscape analysis and inference calculations. For attractor analysis node probabilities for the 74 genes (1 single cell state) were randomly initialised in the continuous interval <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-110.gif"/></noscript></span></span> and the node probabilities were converged to a steady state. Post ensemble averaging the node probabilities were discretized. The process was repeated for more randomly selected initial cell states until the number of unique attractors (local minima) was converged. We found that the random sampling of 450,000 initial states was sufficient to identify all the major attractors in the network. 1082 unique attractors were found and hierarchically clustered using the heatmap.2 function with the Euclidean distance metric in R as shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. Arbitrarily cutting the tree at a distance of 1.3, groups the attractors into 3 representative cell state clusters. The basin size of a given attractor on the cell state potential landscape can be defined as the percentage of random initial cell states that converge to the given attractor. The basin size for each unique attractor within each of the three representative cell state clusters A, B or C was summed to give the total basin size for those representative state clusters.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Attractor heat map for all 1082 unique attractors of the unperturbed network model. Cell states defined using quaternary intervals  and hierarchically clustered using the heatmap.2 function in R. Basin sizes for the 3 representative cell state clusters A, B and C given in legend." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108535840" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Attractor heat map for all 1082 unique attractors of the unperturbed network model. Cell states defined using quaternary intervals &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-111&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and hierarchically clustered using the heatmap.2 function in R. Basin sizes for the 3 representative cell state clusters A, B and C given in legend.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2501/F5.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/7/2501/F5.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490385" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Attractor heat map for all 1082 unique attractors of the unperturbed network model. Cell states defined using quaternary intervals <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F5/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F5/embed/mml-math-111.gif"/></noscript></span></span> and hierarchically clustered using the heatmap.2 function in R. Basin sizes for the 3 representative cell state clusters A, B and C given in legend.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">An important test of the four models was to assess the correlation between the initial experimental gene expression values (calculated as described in the Method Section) and those expression values (node probabilities) to which the structure relaxes during 2TBN DBN inference in the unperturbed state. In terms of the cell-state potential landscape, this represents the proximity of the nearest (defined in terms of cell state similarity) âlocalâ minimum or attractor to which the inference results converge. All Pearson R rank correlation coefficients between initial experimental values and relaxed unperturbed inference results are given in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table II</a>. All values are in the range <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-112.gif"/></noscript></span></span> showing that the model adequately describes the neuronal cell states corresponding to subcategories IâIV.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490299/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490299/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE II:</span> <span class="caption-title">Pearson R rank correlation coefficients between initial cluster gene expression values and relaxed state node (gene) probabilities in the unperturbed state when initialised in experimental state</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">The two unique attractors corresponding to the two PN subcategories I and II cluster together in representative cell state cluster C (see <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) with a basin size covering 27% of cell state space. Further, the two unique attractors corresponding to the two IN subcategories III and IV cluster together in representative cell state cluster B with a basin size covering 28% of cell state space.</p><p id="p-31">In order to further validate our network model, additional single-cell RNA-Seq samples, taken from further Brodmann areas in the adult human cerebral cortex and processed in identical fashion to Lake <em>et al.</em> (<a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Lake <em>et al.</em> 2016</a>), were used. These samples included neuronal and non-neuronal single cells, such as glia, astrocytes, oligodendrocytes and microglia. Subsequent sample filtering resulted in 546 quality samples which were discretized in the binary interval [0,1] based on the arithmetic mean for each gene summed over all samples. This resulted in all 546 samples displaying a unique cell state (using the same 74 gene space as the model). To make comparison to the attractors predicted by the network model, the 1082 quaternary cell states (clustered in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>) were binarised resulting in 234 unique attractors. Comparison between the additional samples and network attractors was made with the Hamming distance metric. It was found that 451 of the 546 additional samples (83%) had a hamming distance of 20 or less with one or more of the 234 network attractors, <em>i.e.</em>, 54 or more genes (73%+) were in the same state of expression. This is suggestive of the fact that the network model (trained only on neuronal subtypes) captures global genetic regulations for other non-neuronal cell types of the mammalian cerebral cortex in addition to accurately describing neuronal cell states.</p></div><div id="sec-15" class="subsection"><h3>Topological analysis</h3><p id="p-32">As previously described, 20 BN structures were trained from the combined down-sampled data of clusters I - IV (see <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). The mean number of edges learnt across all 20 structures was <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-113.gif"/></noscript></span></span> and the standard deviation <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-114.gif"/></noscript></span></span> The relative sparsity of these networks (<span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-115.gif"/></noscript></span></span>) owes to the inclusion of the error term in the <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-116.gif"/></noscript></span></span> scoring function (see Equation 3) leading to the penalisation of overly complex structures. (We use, in the standard definition of network density (D), the number of possible edges in a complete graph (Emax = n(n â 1)) in a directed network where we do not allow self-regulation/loops but hypothetically cycles would be included (a formulation forbidden in our BN learning approach).) A consequence of this and learning multiple structures is that despite the highly stochastic nature of the learning protocol, edges that do occur in higher frequencies across structure learning runs should represent true biological regulations that are coded for in the data.</p></div><div id="sec-16" class="subsection"><h3>Community Detection &amp; Pathway Analysis</h3><p id="p-33"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the merged GRN for all 20 structures with only those edges that occur in <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-117.gif"/></noscript></span></span> of structures displayed. Node sizes are scaled by out-degree. Community detection analysis using the fast unfolding heuristic algorithm of Blondel <em>et al.</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blondel <em>et al.</em> 2008</a>) shows there to be 4 communities at a resolution of 1.55. Interestingly of the 20 nodes depicted as being members of the orange community 13 (65%) were also up-regulated IN relative to PN and as such were DEGs from the root split, furthermore 6 (30%) of genes in this community were further up-regulated in cluster III relative to cluster IV from the IN split. This suggests that regulatory mechanisms in this community likely lead to strong co-expression and activating regulation between the nodes in IN and more specifically cluster III IN (MGE derived). Furthermore 13 of the 16 genes (81%) in the purple community are up-regulated in cluster IV IN (LGE/CGE derived) relative to cluster III IN. The same analysis on the green community reveals that 12/23 (52%) genes are broadly up-regulated in PN at the root split and 6/23 (26%) are further up-regulated in cluster II CPN. Finally, the blue community is almost exclusively, with the exception of two nodes, from the PN split with 5 and 7 genes up- and down-regulated respectively in the I GN/SCPN/CThPN cluster. Based on the fact that the network gene list derives from IN and PN specific DEGs, the neuronal subtype DEG specificity, as related to the communities, is partially expected.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Merged GRN for 20 independent BN structures. 228 unique edges with frequency  included (from a total of 870 unique edges summed over all 20 BN). Displayed using Yifan Hu algorithm as implemented in Gephi version 0.9.1. Node size proportional to out-degree and edge width proportional to frequency. Community detection algorithm (Blondel et al. 2008) run with resolution of 1.55. Modularity = 0.421. Number of communities = 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108535840" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Merged GRN for 20 independent BN structures. 228 unique edges with frequency &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-118&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; included (from a total of 870 unique edges summed over all 20 BN). Displayed using Yifan Hu algorithm as implemented in Gephi version 0.9.1. Node size proportional to out-degree and edge width proportional to frequency. Community detection algorithm (Blondel &lt;em&gt;et al.&lt;/em&gt; 2008) run with resolution of 1.55. Modularity = 0.421. Number of communities = 4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2501/F6.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/7/2501/F6.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490286" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Merged GRN for 20 independent BN structures. 228 unique edges with frequency <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F6/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/F6/embed/mml-math-118.gif"/></noscript></span></span> included (from a total of 870 unique edges summed over all 20 BN). Displayed using Yifan Hu algorithm as implemented in Gephi version 0.9.1. Node size proportional to out-degree and edge width proportional to frequency. Community detection algorithm (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Blondel <em>et al.</em> 2008</a>) run with resolution of 1.55. Modularity = 0.421. Number of communities = 4.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Pathway analysis of all 74 genes reveals a significant number of enriched pathways from âSignal Transductionââ, âImmune Systemâ, âTransmembrane transport of small moleculesâ and âNeuronal Systemâ among others. In particular within the âNeuronal Systemâ, the âNeurotransmitter release cycleâ (<em>P</em> value <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-119.gif"/></noscript></span></span>) includes the genes GAD1, GAD2, SLC6A1 and SLC17A7, the first 3 of which are part of the orange community and form a clique (complete subnetwork with edges between all 3 members in both directions) each forming an edge with each other at a frequency of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-120.gif"/></noscript></span></span> across all 20 structures. The pathway âSignaling by Type 1 Insulin-like Growth Factor 1 Receptorâ (<em>P</em> value <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-121.gif"/></noscript></span></span>) containing the genes IGF1, ERBB4, KIT and EGFR is also enriched. All of these genes are identified by community analysis to be part of the same community and further, the two edges <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-122.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-123.gif"/></noscript></span></span> occur in <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-124.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-125.gif"/></noscript></span></span> of all structures respectively.</p></div><div id="sec-17" class="subsection"><h3>Edge distribution</h3><p id="p-36">The top 27 edges that occur in all 20 structures learnt are given in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table III</a>. One of these regulations is <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-126.gif"/></noscript></span></span> SLC17A7, a known regulator of brain physiology, is a brain-specific solute carrier and is found to regulate the neuronal signal transducer, chimerin-1 (CHN1). SLC17A7 specifically functions as as glutamate transporter and it has been found that <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-127.gif"/></noscript></span></span>-chimerin regulates dendritic spine density. (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Van de Ven 2005</a>) Spine morphological changes, associated with long-term depression, can be induced in hippocampal neurons by metabotropic glutamate receptor activity suggesting possible support for this learnt regulation.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490289/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490289/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE III:</span> <span class="caption-title">Top 27 edges that are learnt in all 20 structures</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">The POU family members are transcriptional regulators, many of which are known to control cell type-specific differentiation pathways. STXBP6 codes for the syntaxin binding protein 6 (amisyn) and an edge that occurs in all structures is <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-155.gif"/></noscript></span></span> STXBP6 is known to regulate SNARE complex assembly, a protein complex involved in membrane fusion, that play an important role in neurotransmitter release.</p><p id="p-38">ADARB2 forms two edges that occur in all structures and is the rank 3 node as ranked by total degree (see <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table IV</a>) suggesting it plays an important role in neuronal gene regulation and identity. It is a member of the ADAR family which contains 3 members, two of which, ADAR and ADAR1, are catalytically active. The ADARB2 gene encodes a catalytically inactive protein, expressed in brain, amygdala and thalamus. (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hogg <em>et al.</em> 2011</a>) It is known to prevent the binding of other ADAR enzymes to targets <em>in vitro</em>, and decreases the efficiency of these enzymes. These enzymes are responsible for RNA editing via the conversion of adenosine to idenosine which has been observed in some pri-miRNAs(<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kawahara <em>et al.</em> 2007</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Yang <em>et al.</em> 2006</a>); that can in turn affect the function of miRNAs which are thought to have a functional roles in gene regulation. The edge between <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-156.gif"/></noscript></span></span> occurs in all structures. Interestingly EGFR/MAPK has been shown to regulate AGO2, (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2009</a>) which itself is a member of the AGO protein family that play a central role in the function of the RNA-induced silencing complex (RISC) and therefore potentially miRNA function. (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">HÃ¶ck <em>et al.</em> 2007</a>)</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490300/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490300/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE IV:</span> <span class="caption-title">Top 20 nodes as ranked by degree summed over all 20 structures. Average degree per structure given in parentheses</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">NFIB, a gene essential for brain development in mice (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Steele-Perkins <em>et al.</em> 2005</a>) and NFIX form another high frequency edge <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-157.gif"/></noscript></span></span> (see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table III</a>). Both genes belong to the NFI family encoding site-specific transcription factors whose functional diversity is generated in part through protein heterodimerization, (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 1997</a>) thus providing strong evidence for a protein-protein interaction and a mechanism of co-regulation.</p><p id="p-40">The edge <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-158.gif"/></noscript></span></span> occurs in <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-159.gif"/></noscript></span></span> of structures and is part of the purple community found to be up-regulated in cluster IV LGE/CGE derived IN. Putative homologs of these genes were found interacting in other organisms such as the protein-protein binding interaction in <em>Drosophila melanogaster</em> of CG31692 and ics and in saccharomyces cerevisiae the protein-protein interaction between RAS2 and CYR1.</p></div><div id="sec-18" class="subsection"><h3>Transdifferentiation gene recipes</h3><p id="p-41">Throughout this section we refer to the âsource stateâ as node probabilities that are initialised to the given probability of expression for the source cluster IâIV and the âtarget stateâ as the node probabilities of the final or target cluster IâIV.</p><p id="p-42">Perturbations were applied as either overexpression (clamping the node probability to 1 for the course of inference) or knockout (clamping the node probability to 0 for the course of inference). A full scan of the three-gene recipe combinatorial space was conducted. The calculations were performed using five randomly selected structures from the 20 trained and final node probabilities were averaged over these structures under each perturbation. Three-gene recipes for the 12 interconversions were ranked based on the RMSD between all node probabilities for non-perturbed genes (71) in the perturbed source state post relaxation and the corresponding node probabilities of the target state. The five best recipes for each of the 12 interconversions are given in Supplementary Tables I and II.</p><p id="p-43"><a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table V</a> shows the best 12 interconversion recipes. We find symmetries exist between the recipes, for example with the exception of S-IIâT-I, for conversion to PN subtypes I and II the overexpression (â) of SATB2 is in all recipes (irrespective of source cluster type). Contrastingly, all of the best 6 conversion recipes to IN subtypes III and IV include the knockout (â) of SATB2. SATB2 is a DNA-binding protein that regulates chromatin organization and gene expression and is important in the development of corticocortical connectivity in the developing cerebral cortex in mice. (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alcamo <em>et al.</em> 2008</a>) Broadly defined as an excitatory marker, SATB2 was found to be regulated between the PN and IN and is a DEG at the root split (see <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> A.ii.). Topologically, SATB2 is the rank-two gene by degree (see <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table IV</a>) forming 18.15 edges on average per structure and further occupies a central position in the network making connections to all 4 communities (see <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). The SATB2 protein through its interactions with both the CTIP2 promoter upstream region and histone deacetylase complex, controls chromatin remodeling. Upper layer pyramidal neurons lose their identity in the absence of SATB2 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Britanova <em>et al.</em> 2008</a>) perhaps consistent with our prediction of SATB2 knockout being important in inter-neuron PN â IN transdifferentiation and also a regulation that is required to be âheld in placeâ for inter-neuron IN subcategory â IN subcategory transdifferentiation. Similarly, but in reverse regulation logic to SATB2, the knockout of GAD1 is in all but one of the 6 recipes for conversion to PN subtypes and its overexpression is in 50% of the recipes for conversion to IN subtypes namely, S-IâT-III, S-IIâT-III and S-IIâT-IV. GAD1 is an inhibitory marker and is up-regulated in IN. It is the rank one node by degree forming 18.20 edges on average per structure and connects to nodes from three of the four communities identified (see <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). GAD1 is involved in pathways including âNeurotransmitter release cycleâ and âTransmission across Chemical Synapsesâ and is an integral enzyme in âGaba Synthesisâ. The overexpression of GAD1 is consistent with transdifferentiation to IN targets since the majority of IN are GABAergic.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490308/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490308/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table V.</span> <span class="caption-title">The 12 best three-gene recipes between source (S-) and target (T-) clusters as ranked by RMSD. Perturbations defined as overexpressed â (node clamped to probability of 1 during inference) and knockout â (node clamped to probability of 0 during inference)</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">In addition to these inter-neuron transdifferentiation recipe symmetries there also exist symmetries in targeting specific PN or IN subtypes. For example, the best 3 recipes targeting T-I include the overexpression of POU6F2 and by contrast all the best 3 recipes to T-II include the knockout of POU6F2. POU6F2 is a transcription factor involved in DNA binding and is only expressed in the CNS. Moreover the gene is enriched in GO terms for âcentral nervous system developmentâ and âregulation of transcription, DNA-templatedâ consistent with its high rank by degree and the fact that it forms edges as a parent node to nodes in three communities. Finally of note in the best transdifferentiation recipes (see <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table V</a>) is ADARB2, of which the knockout and overexpression targets cluster III and cluster IV IN respectively. This is the rank three node by degree and is the most connected gene in the purple community in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> and its functions were discussed in the Edge Distribution subsection.</p><p id="p-45"><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows representative plots for the three best 3-gene recipes from source cluster S-I. Each scatter plot contains the target state node probabilities <em>vs.</em> first the source state node probabilities in the unperturbed (experimental <em>vs.</em> experimental) states in cyan and second the perturbed (target experimental <em>vs.</em> source theoretical perturbed) states in dark blue. We can see remarkable reprogramming success as reflected by the improvement in Pearson R correlation coefficient which in the case of S-IâT-IV changes from a strong negative correlation in the experimental <em>vs.</em> experimental plot of <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-160.gif"/></noscript></span></span> to 0.9589 under the perturbation ERBB4â / ADARB2â / SATB2â. All the 12 best transdifferentiation recipes achieve final correlations in the range <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-161.gif"/></noscript></span></span> (see Supplementary Tables I and II).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The 3 best transdifferentiaion recipes as ranked by RMSD between source cluster I perturbed states and experimental target states. Subplots are target state node probabilities vs. source state node probabilities in the unperturbed (experimental vs. experimental) and perturbed (target experimental vs. source theoretical perturbed) states, represented by cyan and dark blue data points respectively. Best recipes are highlighted in green (overexpression clamp) and red (knockout clamp) and arrows are drawn to show the direction and magnitude of node probability for the given clamp." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108535840" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The 3 best transdifferentiaion recipes as ranked by RMSD between source cluster I perturbed states and experimental target states. Subplots are target state node probabilities &lt;em&gt;vs.&lt;/em&gt; source state node probabilities in the unperturbed (experimental &lt;em&gt;vs.&lt;/em&gt; experimental) and perturbed (target experimental &lt;em&gt;vs.&lt;/em&gt; source theoretical perturbed) states, represented by cyan and dark blue data points respectively. Best recipes are highlighted in green (overexpression clamp) and red (knockout clamp) and arrows are drawn to show the direction and magnitude of node probability for the given clamp.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2501/F7.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/7/2501/F7.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490330" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-46" class="first-child">The 3 best transdifferentiaion recipes as ranked by RMSD between source cluster I perturbed states and experimental target states. Subplots are target state node probabilities <em>vs.</em> source state node probabilities in the unperturbed (experimental <em>vs.</em> experimental) and perturbed (target experimental <em>vs.</em> source theoretical perturbed) states, represented by cyan and dark blue data points respectively. Best recipes are highlighted in green (overexpression clamp) and red (knockout clamp) and arrows are drawn to show the direction and magnitude of node probability for the given clamp.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Transition state analysis</h3><p id="p-47">It is instructive to monitor cell state probabilities during the transdifferentiation procedure via node clamping. In terms of the cell state potential energy landscape, each of the <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-162.gif"/></noscript></span></span> cell states (as represented by a unique combination of 74 binarised node states) in the unperturbed landscape has a potential energy associated with it that is calculated as <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-163.gif"/></noscript></span></span> where (<span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-164.gif"/></noscript></span></span>) is the state probability for the <em>i</em>th state. Under a 3-gene perturbation the available cell states with a finite probability is reduced by a factor 1/8 to <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-165.gif"/></noscript></span></span> and the probability of remaining states also changes. This adjustment to the landscape results in the re-positioning of minima and of energy barriers on the landscape which effectively makes states which were previously inaccessible, open to sampling. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> shows the probability of the states along the transition paths for 10 independent runs for the transdifferentiation S-IâT-IV under the perturbation ERBB4â / ADARB2â / SATB2â for one of the five structures used in inference. The initial unperturbed state (labeled transition state 1) has a potential energy of 33, the perturbation is then applied which raises the energy of the system to <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-166.gif"/></noscript></span></span> âtransportingâ the cell to a new, previously inaccessible area on the potential energy landscape. This new energy allows the cell to now relax to the new minima which coincides with the target cluster T-IV. We can see that the probability is converged in <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-167.gif"/></noscript></span></span> transition states which is common for most recipes.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cell state potential changes along a representative path for a single structure under perturbed conditions. Each line represents a different sampling of the conditional probabilities. The best 3-gene recipe for the interconversion between S-I&#x2192; T-IV, ERBB4&#x2191; / ADARB2&#x2191; / SATB2&#x2193;, is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108535840" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cell state potential changes along a representative path for a single structure under perturbed conditions. Each line represents a different sampling of the conditional probabilities. The best 3-gene recipe for the interconversion between S-I&#x2192; T-IV, ERBB4&#x2191; / ADARB2&#x2191; / SATB2&#x2193;, is shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2501/F8.medium.gif" width="417" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/7/2501/F8.medium.gif" width="417" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2501/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490197" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-48" class="first-child">Cell state potential changes along a representative path for a single structure under perturbed conditions. Each line represents a different sampling of the conditional probabilities. The best 3-gene recipe for the interconversion between S-Iâ T-IV, ERBB4â / ADARB2â / SATB2â, is shown.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Conclusions</h3><p id="p-49">In this work, we applied Bayesian network methods to make <em>de novo</em> predictions for neuronal transdifferentiation recipes between Projection neuron and Interneuron subtypes. Our network, trained on high quality single-cell RNA-Seq data, accurately describes the four cell subtypes in the unperturbed state and is well validated against an additional data set of single-cells from more varied areas of the human cerebral cortex, using attractor analysis. Many of the regulatory edges learnt between the genes are validated from the wider literature and community analysis reveals significant enrichment in neuron specific pathways among others. We conducted a systematic search for transdifferentiation recipes that could achieve reprogramming. The three-gene recipes identified achieved remarkable success the best of which achieve final correlations, with the target state, in the range <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2501/embed/mml-math-168.gif"/></noscript></span></span> Master inter-neuron regulators are identified as SATB2 and GAD1 as well the identification of POU6F2 and ADARB2 as important intra-neuron regulators.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">The authors would like to thank Dr Richard Stein for the development of the structure learning and DBN inference code which was modified in this work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-51">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6349553">https://doi.org/10.25387/g3.6349553</a>.</p></li><li class="fn" id="fn-2"><p id="p-52"><em>Communicating editor: C. Myers</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-03-12"><span class="received-label">Received </span>March 12, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-05-21"><span class="accepted-label">Accepted </span>May 21, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2018 Ainsworth <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.1" data-doi="10.1210/en.2008-0984"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claffey</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Argonaute-2 expression is regulated by epidermal growth factor receptor and mitogen-activated protein kinase signaling and correlates with a transformed phenotype in breast cancer cells.</span> <abbr class="cit-jnl-abbrev">Endocrinology</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">14</span>â<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1210/en.2008-0984</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEndocrinology%26rft_id%253Dinfo%253Adoi%252F10.1210%252Fen.2008-0984%26rft_id%253Dinfo%253Apmid%252F18787018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1210/en.2008-0984&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18787018&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262052300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.2" data-doi="10.1016/j.neuron.2007.12.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alcamo</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chirivella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dautzenberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobreva</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farias</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Satb2 regulates callosal projection neuron identity in the developing cerebral cortex.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">364</span>â<span class="cit-lpage">377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2007.12.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DAlcamo%26rft.auinit1%253DE.%2BA.%26rft.volume%253D57%26rft.issue%253D3%26rft.spage%253D364%26rft.epage%253D377%26rft.atitle%253DSatb2%2Bregulates%2Bcallosal%2Bprojection%2Bneuron%2Bidentity%2Bin%2Bthe%2Bdeveloping%2Bcerebral%2Bcortex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2007.12.012%26rft_id%253Dinfo%253Apmid%252F18255030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2007.12.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18255030&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253075300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.3" data-doi="10.1126/science.1239882"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amamoto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arlotta</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Development-inspired reprogramming of the mammalian central nervous system.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">1239882</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1239882</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAmamoto%26rft.auinit1%253DR.%26rft.volume%253D343%26rft.issue%253D6170%26rft.spage%253D1239882%26rft.epage%253D1239882%26rft.atitle%253DDevelopment-Inspired%2BReprogramming%2Bof%2Bthe%2BMammalian%2BCentral%2BNervous%2BSystem%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1239882%26rft_id%253Dinfo%253Apmid%252F24482482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDMvNjE3MC8xMjM5ODgyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzI1MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.4" data-doi="10.1088/1742-5468/2008/10/P10008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blondel</span>  <span class="cit-name-given-names">V. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambiotte</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefebvre</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Fast unfolding of communities in large networks.</span> <abbr class="cit-jnl-abbrev">J. Stat. Mech.</abbr> <span class="cit-vol">2008</span>: <span class="cit-fpage">P10008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1088/1742-5468/2008/10/P10008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BMech.%26rft.volume%253D2008%26rft.spage%253DP10008%26rft_id%253Dinfo%253Adoi%252F10.1088%252F1742-5468%252F2008%252F10%252FP10008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1088/1742-5468/2008/10/P10008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.5" data-doi="10.1016/j.neuron.2007.12.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Britanova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Juan Romero</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwark</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Satb2 is a postmitotic determinant for upper-layer neuron specification in the neocortex.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">378</span>â<span class="cit-lpage">392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2007.12.028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DBritanova%26rft.auinit1%253DO.%26rft.volume%253D57%26rft.issue%253D3%26rft.spage%253D378%26rft.epage%253D392%26rft.atitle%253DSatb2%2Bis%2Ba%2Bpostmitotic%2Bdeterminant%2Bfor%2Bupper-layer%2Bneuron%2Bspecification%2Bin%2Bthe%2Bneocortex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2007.12.028%26rft_id%253Dinfo%253Apmid%252F18255031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2007.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18255031&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253075300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.6" data-doi="10.1016/j.cell.2012.08.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buganim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faddah</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itskovich</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markoulaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Single-cell expression analyses during cellular reprogramming reveal an early stochastic and a late hierarchic phase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1209</span>â<span class="cit-lpage">1222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2012.08.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBuganim%26rft.auinit1%253DY.%26rft.volume%253D150%26rft.issue%253D6%26rft.spage%253D1209%26rft.epage%253D1222%26rft.atitle%253DSingle-cell%2Bexpression%2Banalyses%2Bduring%2Bcellular%2Breprogramming%2Breveal%2Ban%2Bearly%2Bstochastic%2Band%2Ba%2Blate%2Bhierarchic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.08.023%26rft_id%253Dinfo%253Apmid%252F22980981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.08.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22980981&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308928100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.7" data-doi="10.1371/journal.pcbi.1002300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Systematic search for recipes to generate induced pluripotent stem cells.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1002300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DChang%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De1002300%26rft.epage%253De1002300%26rft.atitle%253DSystematic%2Bsearch%2Bfor%2Brecipes%2Bto%2Bgenerate%2Binduced%2Bpluripotent%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002300%26rft_id%253Dinfo%253Apmid%252F22215993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22215993&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-confproc" id="cit-8.7.2501.8"><div class="cit-metadata unstructured">Chickering, D., D. Geiger, and D. Heckerman, 1995 <em>Learning Bayesian Networks: Search Methods and Experimental Results</em>. Number 112â128 in Proceedings of the Fifth International Workshop on Artificial Intelligence and Statistics.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.9" data-doi="10.1089/106652700750050961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linial</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peâer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Using bayesian networks to analyze expression data.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">601</span>â<span class="cit-lpage">620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/106652700750050961</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DFriedman%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.issue%253D3-4%26rft.spage%253D601%26rft.epage%253D620%26rft.atitle%253DUsing%2BBayesian%2Bnetworks%2Bto%2Banalyze%2Bexpression%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F106652700750050961%26rft_id%253Dinfo%253Apmid%252F11108481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/106652700750050961&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11108481&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165512600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.10" data-doi="10.1038/nrn3586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodworth</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galazo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padmanabhan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macklis</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Molecular logic of neocortical projection neuron specification, development and diversity.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Neurosci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">755</span>â<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrn3586</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BNeurosci.%26rft.volume%253D14%26rft.spage%253D755%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrn3586%26rft_id%253Dinfo%253Apmid%252F24105342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrn3586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24105342&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-confproc" id="cit-8.7.2501.11"><div class="cit-metadata unstructured">Hartemink, A. J., D. K. Gifford, T. S. Jaakkola, and R. A. Young, 2001 Using graphical models and genomic expression data to statistically validate models of genetic regulatory networks. Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.12" data-doi="10.1038/sj.embor.7401088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">HÃ¶ck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinmann</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ender</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rdel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremmer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Proteomic and functional analysis of argonaute-containing mRNAprotein complexes in human cells.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1052</span>â<span class="cit-lpage">1060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.embor.7401088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BReports%26rft.stitle%253DEMBO%2BReports%26rft.aulast%253DHock%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D1052%26rft.epage%253D1060%26rft.atitle%253DProteomic%2Band%2Bfunctional%2Banalysis%2Bof%2BArgonaute-containing%2BmRNA-protein%2Bcomplexes%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.embor.7401088%26rft_id%253Dinfo%253Apmid%252F17932509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.embor.7401088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17932509&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250619100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.13" data-doi="10.1016/B978-0-12-380860-8.00003-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hogg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paro</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keegan</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">OâConnell</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">RNA editing by mammalian ADARs.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">87</span>â<span class="cit-lpage">120</span> <span class="cit-comment">Elsevier</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-12-380860-8.00003-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bgenetics%26rft.stitle%253DAdv%2BGenet%26rft.aulast%253DHogg%26rft.auinit1%253DM.%26rft.volume%253D73%26rft.spage%253D87%26rft.epage%253D120%26rft.atitle%253DRNA%2Bediting%2Bby%2Bmammalian%2BADARs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-380860-8.00003-3%26rft_id%253Dinfo%253Apmid%252F21310295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-380860-8.00003-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21310295&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.14" data-doi="10.1038/sj.embor.7401011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawahara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinshteyn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chendrimada</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiekhattar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishikura</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">RNA editing of the microRNA-151 precursor blocks cleavage by the dicerTRBP complex.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">763</span>â<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.embor.7401011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BReports%26rft.stitle%253DEMBO%2BReports%26rft.aulast%253DGomez%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D763%26rft.epage%253D769%26rft.atitle%253DRNA%2Bediting%2Bof%2Bthe%2BmicroRNA-151%2Bprecursor%2Bblocks%2Bcleavage%2Bby%2Bthe%2BDicer-TRBP%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.embor.7401011%26rft_id%253Dinfo%253Apmid%252F17599088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.embor.7401011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17599088&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248447300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.15" data-doi="10.1038/nature12983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kepecs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishell</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Interneuron cell types are fit to function.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>: <span class="cit-fpage">318</span>â<span class="cit-lpage">326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D505%26rft.spage%253D318%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12983%26rft_id%253Dinfo%253Apmid%252F24429630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12983&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24429630&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329621800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.16" data-doi="10.1126/science.aaf1204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lake</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeser</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salathia</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yung</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">352</span>: <span class="cit-fpage">1586</span>â<span class="cit-lpage">1590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aaf1204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLake%26rft.auinit1%253DB.%2BB.%26rft.volume%253D352%26rft.issue%253D6293%26rft.spage%253D1586%26rft.epage%253D1590%26rft.atitle%253DNeuronal%2Bsubtypes%2Band%2Bdiversity%2Brevealed%2Bby%2Bsingle-nucleus%2BRNA%2Bsequencing%2Bof%2Bthe%2Bhuman%2Bbrain%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf1204%26rft_id%253Dinfo%253Apmid%252F27339989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTIvNjI5My8xNTg2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzI1MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.17" data-doi="10.1074/jbc.272.16.10739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">H. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apt</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">NFI-b3, a novel transcriptional repressor of the nuclear factor i family, is generated by alternative RNA processing.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">10739</span>â<span class="cit-lpage">10745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.272.16.10739</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DLiu%26rft.auinit1%253DY.%26rft.volume%253D272%26rft.issue%253D16%26rft.spage%253D10739%26rft.epage%253D10745%26rft.atitle%253DNFI-B3%252C%2Ba%2BNovel%2BTranscriptional%2BRepressor%2Bof%2Bthe%2BNuclear%2BFactor%2BI%2BFamily%252C%2BIs%2BGenerated%2Bby%2BAlternative%2BRNA%2BProcessing%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.272.16.10739%26rft_id%253Dinfo%253Apmid%252F9099724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzIvMTYvMTA3MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjUwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.18" data-doi="10.1016/j.tins.2014.11.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lodato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shetty</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arlotta</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Cerebral cortex assembly: generating and reprogramming projection neuron diversity.</span> <abbr class="cit-jnl-abbrev">Trends Neurosci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">117</span>â<span class="cit-lpage">125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tins.2014.11.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BNeurosci.%26rft.volume%253D38%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tins.2014.11.003%26rft_id%253Dinfo%253Apmid%252F25529141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tins.2014.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25529141&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-other" id="cit-8.7.2501.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>âDynamic bayesian networks: Representation, inference and learning. <a href="https://ibug.doc.ic.ac.uk/media/uploads/documents/courses/DBN-PhDthesis-LongTutorail-Murphy.pdf">https://ibug.doc.ic.ac.uk/media/uploads/documents/courses/DBN-PhDthesis-LongTutorail-Murphy.pdf</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-book" id="cit-8.7.2501.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neapolitan</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>âProbabilistic methods for bioinformatics: with an introduction to Bayesian networks. Morgan Kaufmann/Elsevier, OCLC: ocn316097968.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.21" data-doi="10.1111/j.1460-9568.1997.tb01698.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeren-Wiemers</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andr</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapfhammer</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker-Andr</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">The expression pattern of the orphan nuclear receptor RORbeta in the developing and adult rat nervous system suggests a role in the processing of sensory information and in circadian rhythm.</span> <abbr class="cit-jnl-abbrev">Eur. J. Neurosci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2687</span>â<span class="cit-lpage">2701</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1460-9568.1997.tb01698.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEuropean%2Bjournal%2Bof%2Bneuroscience%26rft.stitle%253DEur%2BJ%2BNeurosci%26rft.aulast%253DSchaeren-Wiemers%26rft.auinit1%253DN.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D2687%26rft.epage%253D2701%26rft.atitle%253DThe%2Bexpression%2Bpattern%2Bof%2Bthe%2Borphan%2Bnuclear%2Breceptor%2BRORbeta%2Bin%2Bthe%2Bdeveloping%2Band%2Badult%2Brat%2Bnervous%2Bsystem%2Bsuggests%2Ba%2Brole%2Bin%2Bthe%2Bprocessing%2Bof%2Bsensory%2Binformation%2Band%2Bin%2Bcircadian%2Brhythm.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1460-9568.1997.tb01698.x%26rft_id%253Dinfo%253Apmid%252F9517474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1460-9568.1997.tb01698.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9517474&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072461400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.22" data-doi="10.1128/MCB.25.2.685-698.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steele-Perkins</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plachez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butz</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachurski</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The transcription factor gene nfib is essential for both lung maturation and brain development.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">685</span>â<span class="cit-lpage">698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.25.2.685-698.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSteele-Perkins%26rft.auinit1%253DG.%26rft.volume%253D25%26rft.issue%253D2%26rft.spage%253D685%26rft.epage%253D698%26rft.atitle%253DThe%2BTranscription%2BFactor%2BGene%2BNfib%2BIs%2BEssential%2Bfor%2Bboth%2BLung%2BMaturation%2Band%2BBrain%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.2.685-698.2005%26rft_id%253Dinfo%253Apmid%252F15632069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI1LzIvNjg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzI1MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.23" data-doi="10.1523/JNEUROSCI.2450-05.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van de Ven</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The nonkinase phorbol ester receptor 1-chimerin binds the NMDA receptor NR2a subunit and regulates dendritic spine density.</span> <abbr class="cit-jnl-abbrev">J. Neurosci.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">9488</span>â<span class="cit-lpage">9496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1523/JNEUROSCI.2450-05.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DVan%2Bde%2BVen%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D25%26rft.issue%253D41%26rft.spage%253D9488%26rft.epage%253D9496%26rft.atitle%253DThe%2BNonkinase%2BPhorbol%2BEster%2BReceptor%2B%257Balpha%257D1-Chimerin%2BBinds%2Bthe%2BNMDA%2BReceptor%2BNR2A%2BSubunit%2Band%2BRegulates%2BDendritic%2BSpine%2BDensity%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.2450-05.2005%26rft_id%253Dinfo%253Apmid%252F16221859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIyNS80MS85NDg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzI1MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.7.2501.24" data-doi="10.1038/nsmb1041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chendrimada</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higuchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeburg</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Modulation of microRNA processing and expression through RNA editing by ADAR deaminases.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">13</span>â<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nsmb1041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DYang%26rft.auinit1%253DW.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D21%26rft.atitle%253DModulation%2Bof%2BmicroRNA%2Bprocessing%2Band%2Bexpression%2Bthrough%2BRNA%2Bediting%2Bby%2BADAR%2Bdeaminases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1041%26rft_id%253Dinfo%253Apmid%252F16369484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16369484&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2501.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234561900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/7/2501.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/7/2483" title="Genome-Wide Analysis of Multidrug and Toxic Compound Extrusion (MATE) Family in Gossypium raimondii and Gossypium arboreum and Its Expression Analysis Under Salt, Cadmium, and Drought Stress" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/7/2513" title="Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/7"><span>Volume 8 Issue 7, July 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/7.cover-source.jpg?itok=dg_JR4dD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2501.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BAFI1e0JbKHHHriHJ-rcJXnc3Hx2wpBsi5zP5cGyty0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/7/2501" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/7/2501" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/7/2501" class="active" data-icon-position="" data-hide-link-title="0">Bayesian Networks Predict Neuronal Transdifferentiation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490132" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rQcqWITqzghK9XnjoZGhNhaPaUfBASoMwngLfALlIv4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/7/2501.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/7/2501" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xS0_RBaWr21EMWPLVYAcPO7d_gWsma62FNzcyeqwbws" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/7/2501" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/7/2501.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490132" id="node490132--2" data-pisa="ggg;8/7/2501" data-pisa-master="ggg;g3.118.200401" data-apath="/ggg/8/7/2501.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Networks Predict Neuronal Transdifferentiation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3350-5692" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Richard I. Ainsworth</span>, <span class="highwire-citation-author" data-delta="1">Rizi Ai</span>, <span class="highwire-citation-author" data-delta="2">Bo Ding</span>, <span class="highwire-citation-author" data-delta="3">Nan Li</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3454-7357" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kai Zhang</span> and <span class="highwire-citation-author" data-delta="5">Wei Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2501-2511;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200401 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490132--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Richard I. Ainsworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BI.%2BAinsworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainsworth%20RI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BI.%2BAinsworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3350-5692" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Richard I. Ainsworth</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rizi Ai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rizi%2BAi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ai%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARizi%2BAi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bo Ding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BDing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ding%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BDing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nan Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kai Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Graduate Program in Bioinformatics and Systems Biology, University of California, San Diego, La Jolla, California 92093-0359</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKai%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3454-7357" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kai Zhang</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wei-wang@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">wei-wang@ucsd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/7/2501" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/7/2501&amp;title=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/7/2501&amp;title=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/7/2501&amp;title=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/7/2501&amp;text=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/7/2501&amp;title=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/7/2501&amp;t=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/7/2501&amp;title=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/7/2501&amp;title=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490132/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490132" id="node490132--3" data-pisa="ggg;8/7/2501" data-pisa-master="ggg;g3.118.200401" data-apath="/ggg/8/7/2501.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Networks Predict Neuronal Transdifferentiation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3350-5692" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Richard I. Ainsworth</span>, <span class="highwire-citation-author" data-delta="1">Rizi Ai</span>, <span class="highwire-citation-author" data-delta="2">Bo Ding</span>, <span class="highwire-citation-author" data-delta="3">Nan Li</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3454-7357" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kai Zhang</span> and <span class="highwire-citation-author" data-delta="5">Wei Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2501-2511;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200401 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490132--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Richard I. Ainsworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BI.%2BAinsworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainsworth%20RI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BI.%2BAinsworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3350-5692" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Richard I. Ainsworth</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rizi Ai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rizi%2BAi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ai%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARizi%2BAi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bo Ding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BDing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ding%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BDing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nan Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kai Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Graduate Program in Bioinformatics and Systems Biology, University of California, San Diego, La Jolla, California 92093-0359</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKai%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3454-7357" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kai Zhang</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wei Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemistry and Biochemistry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wei-wang@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">wei-wang@ucsd.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2501&amp;rft.epage=2511&amp;rft.atitle=Bayesian%20Networks%20Predict%20Neuronal%20Transdifferentiation&amp;rft.volume=8&amp;rft.issue=7&amp;rft.date=2018-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ainsworth%2C+Richard+I.&amp;rft.au=Ai%2C+Rizi&amp;rft.au=Ding%2C+Bo&amp;rft.au=Li%2C+Nan&amp;rft.au=Zhang%2C+Kai&amp;rft.au=Wang%2C+Wei"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490132/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490132/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490132/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490132/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490132/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490132/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490132/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490132/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490132/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490132/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490132/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490132/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/7/2501" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Method</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results And Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/7/2501.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/7/2501.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/7/2501.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/7/2501.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
