<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/7/2513" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/7/2513.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/7/2513.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/7/2513.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/7/2513/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/7/2513.atom" />
<meta name="HW.pisa" content="ggg;8/7/2513" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids" />
<meta name="DC.Identifier" content="10.1534/g3.118.200242" />
<meta name="DC.Date" content="2018-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Liang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Pearl millet is a non-model grain and fodder crop adapted to extremely hot and dry environments globally. In India, a great deal of public and private sectors’ investment has focused on developing pearl millet single cross hybrids based on the cytoplasmic-genetic male sterility (CMS) system, while in Africa most pearl millet production relies on open pollinated varieties. Pearl millet lines were phenotyped for both the inbred parents and hybrids stage. Many breeding efforts focus on phenotypic selection of inbred parents to generate improved parental lines and hybrids. This study evaluated two genotyping techniques and four genomic selection schemes in pearl millet. Despite the fact that 6× more sequencing data were generated per sample for RAD-seq than for tGBS, tGBS yielded more than 2× as many informative SNPs (defined as those having MAF &gt; 0.05) than RAD-seq. A genomic prediction scheme utilizing only data from hybrids generated prediction accuracies (median) ranging from 0.73-0.74 (1000-grain weight), 0.87-0.89 (days to flowering time), 0.48-0.51 (grain yield) and 0.72-0.73 (plant height). For traits with little to no heterosis, hybrid only and hybrid/inbred prediction schemes performed almost equivalently. For traits with significant mid-parent heterosis, the direct inclusion of phenotypic data from inbred lines significantly ( P &lt; 0.05) reduced prediction accuracy when all lines were analyzed together. However, when inbreds and hybrid trait values were both scored relative to the mean trait values for the respective populations, the inclusion of inbred phenotypic datasets moderately improved genomic predictions of the hybrid genomic estimated breeding values. Here we show that modern approaches to genotyping by sequencing can enable genomic selection in pearl millet. While historical pearl millet breeding records include a wealth of phenotypic data from inbred lines, we demonstrate that the naive incorporation of this data into a hybrid breeding program can reduce prediction accuracy, while controlling for the effects of heterosis per se allowed inbred genotype and trait data to improve the accuracy of genomic estimated breeding values for pearl millet hybrids." />
<meta name="DC.Contributor" content="Zhikai Liang" />
<meta name="DC.Contributor" content="Shashi K. Gupta" />
<meta name="DC.Contributor" content="Cheng-Ting Yeh" />
<meta name="DC.Contributor" content="Yang Zhang" />
<meta name="DC.Contributor" content="Daniel W. Ngu" />
<meta name="DC.Contributor" content="Ramesh Kumar" />
<meta name="DC.Contributor" content="Hemant T. Patil" />
<meta name="DC.Contributor" content="Kanulal D. Mungra" />
<meta name="DC.Contributor" content="Dev Vart Yadav" />
<meta name="DC.Contributor" content="Abhishek Rathore" />
<meta name="DC.Contributor" content="Rakesh K. Srivastava" />
<meta name="DC.Contributor" content="Rajeev Gupta" />
<meta name="DC.Contributor" content="Jinliang Yang" />
<meta name="DC.Contributor" content="Rajeev K. Varshney" />
<meta name="DC.Contributor" content="Patrick S. Schnable" />
<meta name="DC.Contributor" content="James C. Schnable" />
<meta name="article:published_time" content="2018-07-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Pearl millet is a non-model grain and fodder crop adapted to extremely hot and dry environments globally. In India, a great deal of public and private sectors’ investment has focused on developing pearl millet single cross hybrids based on the cytoplasmic-genetic male sterility (CMS) system, while in Africa most pearl millet production relies on open pollinated varieties. Pearl millet lines were phenotyped for both the inbred parents and hybrids stage. Many breeding efforts focus on phenotypic selection of inbred parents to generate improved parental lines and hybrids. This study evaluated two genotyping techniques and four genomic selection schemes in pearl millet. Despite the fact that 6× more sequencing data were generated per sample for RAD-seq than for tGBS, tGBS yielded more than 2× as many informative SNPs (defined as those having MAF &amp;gt; 0.05) than RAD-seq. A genomic prediction scheme utilizing only data from hybrids generated prediction accuracies (median) ranging from 0.73-0.74 (1000-grain weight), 0.87-0.89 (days to flowering time), 0.48-0.51 (grain yield) and 0.72-0.73 (plant height). For traits with little to no heterosis, hybrid only and hybrid/inbred prediction schemes performed almost equivalently. For traits with significant mid-parent heterosis, the direct inclusion of phenotypic data from inbred lines significantly (&lt;i&gt;P&lt;/i&gt; &amp;lt; 0.05) reduced prediction accuracy when all lines were analyzed together. However, when inbreds and hybrid trait values were both scored relative to the mean trait values for the respective populations, the inclusion of inbred phenotypic datasets moderately improved genomic predictions of the hybrid genomic estimated breeding values. Here we show that modern approaches to genotyping by sequencing can enable genomic selection in pearl millet. While historical pearl millet breeding records include a wealth of phenotypic data from inbred lines, we demonstrate that the naive incorporation of this data into a hybrid breeding program can reduce prediction accuracy, while controlling for the effects of heterosis &lt;i&gt;per se&lt;/i&gt; allowed inbred genotype and trait data to improve the accuracy of genomic estimated breeding values for pearl millet hybrids.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/07/01" />
<meta name="citation_mjid" content="ggg;8/7/2513" />
<meta name="citation_id" content="8/7/2513" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/7/2513" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/7/2513.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/7/2513.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/7/2513.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200242" />
<meta name="citation_pmid" content="29794163" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2513" />
<meta name="citation_lastpage" content="2522" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhikai Liang" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9963-8631" />
<meta name="citation_author" content="Shashi K. Gupta" />
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India" />
<meta name="citation_author" content="Cheng-Ting Yeh" />
<meta name="citation_author_institution" content="Iowa State University, Ames, IA" />
<meta name="citation_author" content="Yang Zhang" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1712-7211" />
<meta name="citation_author" content="Daniel W. Ngu" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE" />
<meta name="citation_author" content="Ramesh Kumar" />
<meta name="citation_author_institution" content="Chaudhary Charan Singh Haryana Agricultural University, Hisar, Haryana, India" />
<meta name="citation_author" content="Hemant T. Patil" />
<meta name="citation_author_institution" content="Mahatma Phule Krishi Vidyapeeth, Dhule, Maharashtra, India" />
<meta name="citation_author" content="Kanulal D. Mungra" />
<meta name="citation_author_institution" content="Junagadh Agricultural University, Jamnagar, Gujarat, India" />
<meta name="citation_author" content="Dev Vart Yadav" />
<meta name="citation_author_institution" content="Chaudhary Charan Singh Haryana Agricultural University, Hisar, Haryana, India" />
<meta name="citation_author" content="Abhishek Rathore" />
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6887-4095" />
<meta name="citation_author" content="Rakesh K. Srivastava" />
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India" />
<meta name="citation_author" content="Rajeev Gupta" />
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India" />
<meta name="citation_author" content="Jinliang Yang" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0999-3518" />
<meta name="citation_author" content="Rajeev K. Varshney" />
<meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4562-9131" />
<meta name="citation_author" content="Patrick S. Schnable" />
<meta name="citation_author_institution" content="Iowa State University, Ames, IA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9169-5204" />
<meta name="citation_author" content="James C. Schnable" />
<meta name="citation_author_institution" content="University of Nebraska-Lincoln, Lincoln, NE" />
<meta name="citation_author_email" content="schnable@unl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6739-5527" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=V. Wimmer;citation_author=J. O. Ogutu;citation_author=C. Knaak;citation_title=Genome-based prediction of maize hybrid performance across genetic groups, testers, locations, and years.;citation_pages=1375-1386;citation_volume=127;citation_year=2014;citation_pmid=24723140;citation_doi=10.1007/s00122-014-2305-z" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=T. Dickert;citation_author=W. Tracy;citation_title=Heterosis for flowering time and agronomic traits among early open-pollinated sweet corn cultivars.;citation_pages=793-797;citation_volume=127;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=F. G. Dohleman;citation_author=S. P. Long;citation_title=More productive than maize in the midwest: how does miscanthus do it?;citation_volume=150;citation_year=2009;citation_pmid=19535474;citation_doi=10.1104/pp.109.139162" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. E. G. e Gama;citation_author=A. R. Hallauer;citation_title=Relation Between Inbred and Hybrid Traits in Maize;citation_pages=703-706;citation_volume=17;citation_year=1977;citation_issue=5;citation_doi=10.2135/cropsci1977.0011183X001700050007x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with r package rrblup.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Gowda;citation_author=Y. Zhao;citation_author=H. P. Maurer;citation_author=E. A. Weissmann;citation_author=T. Würschum;citation_title=Best linear unbiased prediction of triticale hybrid performance.;citation_pages=223-230;citation_volume=191;citation_year=2013;citation_doi=10.1007/s10681-012-0784-z" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=S. Gupta;citation_author=K. Rai;citation_author=P. Singh;citation_author=V. Ameta;citation_author=S. K. Gupta;citation_title=Seed set variability under high temperatures during flowering period in pearl millet (Pennisetum glaucum L. (R.) Br.).;citation_pages=41-53;citation_volume=171;citation_year=2015;citation_doi=10.1016/j.fcr.2014.11.005" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=S. Gupta;citation_author=A. Rathore;citation_author=O. Yadav;citation_author=K. Rai;citation_author=I. Khairwal;citation_title=Identifying mega-environments and essential test locations for pearl millet cultivar selection in india.;citation_pages=2444-2453;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2013.01.0053" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=W. Hanna;citation_title=Characteristics and Stability of a New Cytoplasmic-Nuclear Male-Sterile Source in Pearl Millet;citation_pages=1457-1459;citation_volume=29;citation_year=1989;citation_issue=6;citation_doi=10.2135/cropsci1989.0011183X002900060026x" />
<meta name="citation_reference" content="citation_title=Plant Breeding Reviews;citation_title=Estimating and interpreting heritability for plant b: an update;citation_author=J. Holland;citation_author=W. Nyquist;citation_author=C. Cervantes-Martínez;citation_pages=9-112;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J.-L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145-158;citation_volume=128;citation_year=2015;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. C. Kadam;citation_author=S. M. Potts;citation_author=M. O. Bohn;citation_author=A. E. Lipka;citation_author=A. J. Lorenz;citation_title=Genomic prediction of single crosses in the early stages of a maize hybrid breeding pipeline. G3: Genes, Genomes;citation_pages=3443-3453;citation_volume=6;citation_year=2016;citation_doi=10.1534/g3.116.031286" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=E. S. Mace;citation_author=K. K. Buhariwalla;citation_author=H. K. Buhariwalla;citation_author=J. H. Crouch;citation_title=A high-throughput dna extraction protocol for tropical molecular breeding programs.;citation_pages=459-460;citation_volume=21;citation_year=2003;citation_doi=10.1007/BF02772596" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. R. Miller;citation_author=J. P. Dunham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated dna (rad) markers.;citation_volume=17;citation_year=2007;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Mühleisen;citation_author=H.-P. Piepho;citation_author=H. P. Maurer;citation_author=J. C. Reif;citation_title=Yield performance and stability of cms-based triticale hybrids.;citation_pages=291-301;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-014-2429-1" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Ott;citation_author=S. Liu;citation_author=J. C. Schnable;citation_author=C.-T. E. Yeh;citation_author=K.-S. Wang;citation_title=tgbs genotyping-by-sequencing enables reliable genotyping of heterozygous loci.;citation_pages=e178;citation_volume=45;citation_year=2017;citation_doi=10.1093/nar/gkx853" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=R. Ramya;citation_author=L. Ahamed;citation_author=R. Yadav;citation_author=P. Katiyar;citation_author=B. Raj;citation_title=Towards defining heterotic gene pools in pearl millet [Pennisetum glaucum L. (R.) Br.].;citation_pages=1934;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01934" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. A. Schrag;citation_author=H. P. Maurer;citation_author=A. E. Melchinger;citation_author=H.-P. Piepho;citation_author=J. Peleman;citation_title=Prediction of single-cross hybrid performance in maize using haplotype blocks associated with QTL for grain yield.;citation_pages=1345-1355;citation_volume=114;citation_year=2007;citation_issue=8;citation_pmid=17323040;citation_doi=10.1007/s00122-007-0521-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=O. Smith;citation_title=Covariance between Line per se and Testcross Performance;citation_pages=540-543;citation_volume=26;citation_year=1986;citation_issue=3;citation_doi=10.2135/cropsci1986.0011183X002600030023x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=P. Virk;citation_author=B. Collard;citation_title=Genomic selection and association mapping in rice (Oryza sativa): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.;citation_pages=e1004982;citation_volume=11;citation_year=2015;citation_pmid=25689273;citation_doi=10.1371/journal.pgen.1004982" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc.;citation_author=M. E. Tabangin;citation_author=J. G. Woo;citation_author=L. J. Martin;citation_title=The effect of minor allele frequency on the likelihood of obtaining false positives.;citation_pages=S41;citation_volume=3;citation_year=2009;citation_pmid=20018033;citation_doi=10.1186/1753-6561-3-S7-S41" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Technow;citation_author=T. A. Schrag;citation_author=W. Schipprack;citation_author=E. Bauer;citation_author=H. Simianer;citation_title=Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize.;citation_volume=197;citation_year=2014;citation_pmid=24850820;citation_doi=10.1534/genetics.114.165860" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=R. K. Varshney;citation_author=C. Shi;citation_author=M. Thudi;citation_author=C. Mariac;citation_author=J. Wallace;citation_title=Pearl millet genome sequence provides a resource to improve agronomic traits in arid environments.;citation_pages=969-976;citation_volume=35;citation_year=2017;citation_doi=10.1038/nbt.3943" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments. G3: Genes, Genomes;citation_pages=1427-1436;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.112.003699" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=S. Nacu;citation_title=Fast and snp-tolerant detection of complex variants and splicing in short reads.;citation_volume=26;citation_year=2010;citation_pmid=20147302;citation_doi=10.1093/bioinformatics/btq057" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. Xing;citation_author=C. Fan;citation_author=Y. Zhou;citation_title=Parental selection of hybrid breeding based on maternal and paternal inheritance of traits in rapeseed (brassica napus l.).;citation_pages=e103165;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0103165" />
<meta name="citation_reference" content="citation_journal_title=Indian J. Plant. Genet. Resour.;citation_author=O. P. Yadav;citation_author=K. Rai;citation_author=H. Yadav;citation_author=B. Rajpurohit;citation_author=S. Gupta;citation_title=Assessment of diversity in commercial hybrids of pearl millet in india.;citation_pages=130-136;citation_volume=29;citation_year=2016;citation_doi=10.5958/0976-1926.2016.00018.8" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Yu;citation_author=X. Li;citation_author=C. Zhu;citation_author=C.-T. Yeh;citation_author=W. Wu;citation_title=Genic and non-genic contributions to natural variation of quantitative traits in maize.;citation_volume=22;citation_year=2012;citation_pmid=22701078;citation_doi=10.1101/gr.140277.112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J. C. Dekkers;citation_author=R. L. Fernando;citation_author=J.-L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study.;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
<meta name="twitter:description" content="Pearl millet is a non-model grain and fodder crop adapted to extremely hot and dry environments globally. In India, a great deal of public and private sectors’ investment has focused on developing pearl millet single cross hybrids based on the cytoplasmic-genetic male sterility (CMS) system, while in Africa most pearl millet production relies on open pollinated varieties. Pearl millet lines were phenotyped for both the inbred parents and hybrids stage. Many breeding efforts focus on phenotypic selection of inbred parents to generate improved parental lines and hybrids. This study evaluated two genotyping techniques and four genomic selection schemes in pearl millet. Despite the fact that 6× more sequencing data were generated per sample for RAD-seq than for tGBS, tGBS yielded more than 2× as many informative SNPs (defined as those having MAF &gt; 0.05) than RAD-seq. A genomic prediction scheme utilizing only data from hybrids generated prediction accuracies (median) ranging from 0.73-0.74 (1000-grain weight), 0.87-0.89 (days to flowering time), 0.48-0.51 (grain yield) and 0.72-0.73 (plant height). For traits with little to no heterosis, hybrid only and hybrid/inbred prediction schemes performed almost equivalently. For traits with significant mid-parent heterosis, the direct inclusion of phenotypic data from inbred lines significantly ( P &lt; 0.05) reduced prediction accuracy when all lines were analyzed together. However, when inbreds and hybrid trait values were both scored relative to the mean trait values for the respective populations, the inclusion of inbred phenotypic datasets moderately improved genomic predictions of the hybrid genomic estimated breeding values. Here we show that modern approaches to genotyping by sequencing can enable genomic selection in pearl millet. While historical pearl millet breeding records include a wealth of phenotypic data from inbred lines, we demonstrate that the naive incorporation of this data into a hybrid breeding program can reduce prediction accuracy, while controlling for the effects of heterosis per se allowed inbred genotype and trait data to improve the accuracy of genomic estimated breeding values for pearl millet hybrids." />
<meta name="og-title" property="og:title" content="Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/7/2513" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Pearl millet is a non-model grain and fodder crop adapted to extremely hot and dry environments globally. In India, a great deal of public and private sectors’ investment has focused on developing pearl millet single cross hybrids based on the cytoplasmic-genetic male sterility (CMS) system, while in Africa most pearl millet production relies on open pollinated varieties. Pearl millet lines were phenotyped for both the inbred parents and hybrids stage. Many breeding efforts focus on phenotypic selection of inbred parents to generate improved parental lines and hybrids. This study evaluated two genotyping techniques and four genomic selection schemes in pearl millet. Despite the fact that 6× more sequencing data were generated per sample for RAD-seq than for tGBS, tGBS yielded more than 2× as many informative SNPs (defined as those having MAF &gt; 0.05) than RAD-seq. A genomic prediction scheme utilizing only data from hybrids generated prediction accuracies (median) ranging from 0.73-0.74 (1000-grain weight), 0.87-0.89 (days to flowering time), 0.48-0.51 (grain yield) and 0.72-0.73 (plant height). For traits with little to no heterosis, hybrid only and hybrid/inbred prediction schemes performed almost equivalently. For traits with significant mid-parent heterosis, the direct inclusion of phenotypic data from inbred lines significantly ( P &lt; 0.05) reduced prediction accuracy when all lines were analyzed together. However, when inbreds and hybrid trait values were both scored relative to the mean trait values for the respective populations, the inclusion of inbred phenotypic datasets moderately improved genomic predictions of the hybrid genomic estimated breeding values. Here we show that modern approaches to genotyping by sequencing can enable genomic selection in pearl millet. While historical pearl millet breeding records include a wealth of phenotypic data from inbred lines, we demonstrate that the naive incorporation of this data into a hybrid breeding program can reduce prediction accuracy, while controlling for the effects of heterosis per se allowed inbred genotype and trait data to improve the accuracy of genomic estimated breeding values for pearl millet hybrids." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
    <title>Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"R30IQ-S_kIzJHZk2BEVVoT-xEf1TzrHfh9bZGoe8EQ0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490111","apath":"\/ggg\/8\/7\/2513.atom","pisa":"ggg;8\/7\/2513","ac":{"\/ggg\/8\/7\/2513.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/7\/2513.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/7\/2513"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/7\/2513"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/7\/2513":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eZ+j9X8NPK2joYDGc\\\\\\\/SY9ykq9u7iokhoVtzjwjN5Df8AifYpelpH9GuBiFuk8iv+bRu8syhCUhvDpN8GBVkPIF1J7Uy+bNN2J0tTkNzs60vL+ykO3mwPLRbExX+qe\\\\\\\/3CCVYFFyOs8XGul9zk5IK80ZFgBTvKxMkZPS4zHgpUpMIOdFVPVaq6Ys5L9YcSCb8TdrJzyEu1Ill6zygPNrTo7nZtqgYcfSkKiskZIPXj0o2\\\\\\\/GItXnt3jlAROGRAaA9bSgjsAMA58nNjLstUWonSKtRq+YnlY15LwevXJTepCcVPLw7wqDli6389A4MWe+gCUm+48FABExF6t4V6Py5si2c4cbUjBpTvjs7QmnI5ilK5uKIEKaJ6p1TftY44Cva\\\\\\\/mHuCorp5p1VPpGNMZzjFu+lZPKhTb3215XXXwrc12vWwfhQ0ZjSDqxoFuuneL2IPoGESKYP+TpS7naMzGdtQAom09f+\\\\\\\/VUcVgvXCOkZnP1Kxo5kTXI+XOGTlui6QjNsipC92nUmdDSMVUEQVS1AqAs4uY4pzfLomsFHDrLanZ4ZjnC\\\\\\\/HCIgREuqAnJlqLQJVwI0MPXIUCxPq\\\\\\\/MaXCjsGxk8qqRsJNQOfaOpoduScRWmPNVzLB3gPNF4oMCG6HxELFotAws5he4Ia5qNbenZdxOB\\\\\\\/uyDwel9tuuuAVVzpz6L57l65WWO1Vx5QcLFNyHOyOw3ZaUfHf8KbYsrOLXpqj6qByFmdg+YHx0+JEe61hOaPcSUgFqwD+3y45RKwgtrYQrGhXNoUcuIvrS9GwZQkAtE\\\\\\\/ZjPPHracozmdpPZyudf66qN85TTCHs81gQ3+ksr6zr1+psrNBCcxidnALtvUMDINiUoHC2ahGX5OhIaWsAXfQZs75XJWQ3ARrOF5wLG2Qa8JDgwv17vf6jOJeWyPUYZYp4Uk9O4vD3WQiVVXxdN1e8QNlk2yPvZnITpEwRlzp4RwJIdXF6GkxwJ9pIY53GhAAKA9hWOeYooEA5w2FvnaS2eoU3DwoC2AW12jtDkiOIO9xe3O75L8LpH+7EJRkJ9pm2iUAbVkFKY2pyZ5DPoem9vMHKZ9Ok0E5OOrD+ZPE7\\\\\\\/SzOOCN8NkqMKR\\\\\\\/SC8oU3\\\\\\\/CEpLZ8uSFABQKY58d6GqzTWe\\\\\\\/ji76l+WiZoyWLo2KfApyW9W8up5lOhqi63rlR1KLrZSCIG6+GAHNh0+WRS\\\\\\\/odEkN0ZoG8+Z0FuUVW7ehSoTLSZH3OFCN2f+IaOoXVMtnVPc8h\\\\\\\/bMaVApcKgB+JXZP3ZTIoht63SHYxD\\\\\\\/D+vi\\\u0022,\\\u0022iv\\\u0022:\\\u0022lgPA2PkQk94Z6VydmCLaGw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002269224289e5467d4ebdb07e66d10cd5b5c55992afa2a1297a372d236ffd6f6878\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qcUFnVuhn7N5C806\\\\\\\/aQ3FTdAK8jKUiYs3k4wGga\\\\\\\/vAo3p7zJQlO8vlSJgBPgUCA0nSnuOnLikR9r9tXU0R7Emm47RzSrV6R5hDrPGlJEta0YmrAw+l+UWy6sea\\\\\\\/2lxYa8LY3RZxqAlmkzE903RPHsd12RUfBwwqfrdMd\\\\\\\/XgqzP07yPg2x1pyjndADRs3v7C8wAZk3bBrv2hH2JrJqTmemgy+LyOyjEE\\\\\\\/nIWwZL2VJ9uG9YhmP6o5lvceuzG1TRspAETxBk+dttgma+HuuSbO+vaD+Q1EsLKgPsNA27h0N\\\\\\\/j2cC4PYQw22j6WAs\\\\\\\/bxj+jzzOcZMc\\\\\\\/MKYzR\\\\\\\/OGor5nI8Y7nn8Ni4e4aPOVQihCUecgN7PDku6+BWzIvnG+WG7nBv4d5+kssd\\\\\\\/OjejR8o7ujz+l60C1x3su6whpZ9hy7qlSQXLPfQXlgWIzf2mUhfEdMG94ivNDce2YjTcASKKoolWyViiZ0qQTP\\\\\\\/g9aDaQpoQvproZGCmkg3aesmyXCqoPjSVXiNyMPcigOQYgHGEhBD1oOMiPWXK4AZXjY3RRgFqwEGpeJRcsNxePeE\\\\\\\/BLhP2fP2j9ir+FLbiioBFRKKvN4iSHWusDfneD1Jb9LsZ0GMIHXa+fr1tkIr47igGYA9FADYS4tzvNib9AcSYJEyolpmhSv1D6NzVDNekDl7ltNbxI8yg2Ozqml4as9RemLLKf2LGBXUtcCAI3gjheKeiy30oTaZideeyRulPEaX4hSDXd99QrgKOAqscpGx8xDSsQZqSD+bvgjH7dfzFlogiSVvMQ+ULZSNpMUOxBBXeCAjdHFOXYLexAa\\\\\\\/C+W85\\\\\\\/rCC6ZHLGjPJIp6v5gvBoBC4uSCfW8O8k\\\\\\\/osZ1gFtF5CbZRt7GYzwo54ArNwqQMuwF7qgjmA\\\\\\\/KCmgm469DYaRmAT91aLf3fMECJX\\\\\\\/Bya270bTiayO2KHNDGEGr6rpnO6RUjzkhwlqm\\\\\\\/Cyw\\\\\\\/i6JJFFWPg0PIHZ9HVaanRb7kmSWAV9n\\\\\\\/VmBu9pZL1x2O9N0VvcGZZlTaxc5ZCpvf0qyISADGiDa0qT2JzMiPESOAQC0NNT4tcXtJ1lM91PyfhT2ELdeIgcWBDP81cf9qkqhOqEWsGZ2KfMHrYTnbsWrbfGUMvUMiQcoACYTBoGriNljpnNHc4Dy2zyAY996eVDROEzt6O7\\\\\\\/pNA4C76+wOFYQFRUjOTfi56LjYgSe+iIt1fxAkM\\\\\\\/84sylyJZObcwS\\\\\\\/gsZADJi\\\\\\\/nkgrfWZztNWagAipmlxdSim76ebpH5BNP2aKJ+W0fRhkIQHhf8uDXATz8RYWq8X3Kr+H51qbkyg2g9tlPSO\\\\\\\/BiCGOClIBgqILhcyjKybgbUogycfypvZPAaeEipqOV9HFSs8sXsHyKYUxkTFCKT45Uin8UFM\\\\\\\/EeUZWGbJhRHoonjLhzc8xbrIraz7mIarcBt1ilQoI1bjCPCRVSQRy7V9KItCPjzUV2hdB+gbCNuPNJJ8vD\\\\\\\/C6Y1ywCFAoTQUdKjiZqpHtXmkl7K1I7Z+sgEndLUuZMet8wt6mX99idQN6Q2PXl3zXoBvfnLHJIq0nmPHDOZLTU3muPaFqKckG+JgzxyxnFJLxSxJkbL7Oj9ghKOj0maDEMSkgA9yAKVmPXLhbf\\\\\\\/uHiF7pNjeS3W8BuTMsvmr+3CQ6KyqZDHJxHVLP8gCx1AoAaP5uhOwgGZO386giGInGVS6U2cEXet6AJgDdaVoh5Ta6cxxr3R2p9Iblw3Vhb0DQ7wgFJ6RqxSUj0z1hObwAxPUp+tj\\\\\\\/oi\\\\\\\/u4xomIS4XQ8dJyEDEzAMfa9AUvYonpqlJKp4G6ScnEWGG\\\\\\\/iuusDkVi3Qz9Oj7ORy2PyRDln+Rpb+to8JmihvhYrIHaGJzNFfK0OVBk1VC0RvJ3ly5t\\\\\\\/VdAkAUUclo2y3cAc26xiSCOPqqBi5iJJa\\\\\\\/susHD7PHoxYNDo8hXchbV37tFvNvggBHUwBr0q3ekxPrFyh4PBm0UBSH2ofz5bKzh6guhEoVcyTkyH\\\\\\\/s4uQxBXUIf5WyeVyW4lwlgsGIIDj7noD8Ue53vIaBNYk+q53tksJfHZi+j5ghtuMz0Z9248UoTCkoPCJlFyoFXIGPtLf\\\\\\\/hbF2YjTfxAJ3nRQSOhnpOnbKc3RVgQVSf4pAJ7QybF+3sIWnKn09c77EaKHr6sxWY5yv+Bq3yRae7kVnLn+l22UySVt3MzdNJa8plYaojol8xCPz7eAIgDTbasOR2s\\\\\\\/6cEwO3ASz2EAmYc5rp594O7dCxt7Sh0RPMH9WdMIVjpZ5kkFw14lG7xhwhGxqAzTRBFMxBp1YLd1uYHJQswoz38lKzgdVLopwZrWXBuJ9cUcWQwcjM13xszR08wISvpUPYHcbKGFGCvLmbB9EVaWWRW9AnZXQwjOVzlC4OeuLJYC2M2S6R3toV6NTvdFJPs\\\\\\\/yYIxRCXew3KrJtFor\\\\\\\/b\\\\\\\/Ujo6pNqcZzfPvUwEwy3MJXOfeSIFfeiI74545oaKO6Q0Qu1Uh\\\\\\\/\\\\\\\/oFbyvnADfIwmxxzv0OU\\\\\\\/RGGivtf2v7DtlqmM8JH6TQul8WtRI2dZyhLxn3BuRv2ysJQQxCBoa74yIzzM7CoHVkVNvycrTOvwJsBUI7B7Y85a+btPiZEnTwB7SFgBs9LjZrgRHnE7Jrd9dQoOE56tS3T7v3LWGVIDTdpzlF6rg6LMz518pgDSa3xsHVf6Wip3ZqRXrOpVC+eahQ4639yXEtFkbZMZmxwumb4KVne2Sfk0vtUbzScTw2Cuu1vxGUCcaoxtLwn54x9uirk7u\\\\\\\/Dc4BJ7Nxk2KTHNXR7cPZL3jAcXwd+IK79kpX52PIZbtA1gI6DtcCBhtWIbMYaBaW41Wtpl0hb7N3dZ561\\\\\\\/uWSStZGUJiGv\\\\\\\/Ok68Ublvwq59sbtCH7SOhIITNq0jj54eQ0DvwocqOebrwY+2R4cOR2\\\\\\\/rtHSKSpdLNQkx4bJXDnOpKeFGPUx3C9tnO5V9RPJieBc3yrclsBw0WV7AoZSUUCvzrfEc98OXui4cVAubS4CNpOWTa+g23rkD6gLDfYNQKPYQZ8GtKHEDHURN9aFCW3DjlfP\\\\\\\/KdEtVvVP6BaE6vT8ceL77H+V4YNbGCunFfbypb4QwbMtpzt37gTeMbf1YMDNxtcKNmhOmN4yIi22QeZm5A0hd0pakTBIPwqNtax9RtQp6+rR7vYxGstLVg2HM0pnHmkBehAuXmd79ZtrsbxU1Rw5l+0dQOJl7IkjoWwOTtBVIMV9Zie7u7Ao+kXRpsH3wkB48Rgu1ZLdfAXZNBJenYvFpSzSi8uyHaYSJlWtOe1R\\\\\\\/RrmOU84dLBLMJwOMSYVgQd2KHBrB52ewIVWpo12\\\\\\\/xKmaDvtAYbz64WiH3Y5xiHt5zlQnPHs5+MrMDu\\\\\\\/dLu18iTLxvFWXcgWmTMPVPpzank1NaiAstqF+XcaZwiO0j5l7piaY5b\\\\\\\/edune+TJlkwcD+PwTn\\\\\\\/hbJ1yWot1PFsWUX9D7DCMByn\\\\\\\/bP8pHzvfnXJZC7tx80Z\\\\\\\/m1QoevfvdY3IKzgMpiGKCzv3QDoWpLQu7iF2b42B9G5vvfl+7kR0dwLRBBEoM9PYp1lqLSRDKkcO+GL0c42QQjDvWKqOgcL7GJ9EE0Sl+g+oJmu1Umiuo6gLWZAh0t9xvuAlOMqbxC0dZMRUXH2NKGaD\\\\\\\/l7YkzuTkqbj2HYWXJ3thHhN128TackDSiNjLMLgAdz1X3q2qI1VcCPItEIqzh+PzrAA8cytO3fSj8sHINDw2VZQFtijEQWW8I+nyst3grGH76jPEMmWfMw4OTXP2xKNEwqsOuVyokpq7jf0+UQQHMP72P8wPZo58JS71x\\\\\\\/ULm9vBXdEyQKs8gL5OMpmui21eCmsJYhhM0xpWKVVRu7gfaxAxvXC2JhIfJIdci5nw\\\\\\\/eajYfyrbOlvR05fjIM5WvnnUk7TbqllQVRZbtZvrklUGAT4Nd5kH1p4UPtXr9Qvki2ejjgbuuUu6eCA82GE8dhSOGp8+9hkRfNbg6WPbmFw2JTN6ZC+ScX7Jjxrf3Mtz8U9rC\\\\\\\/zaYpR1VrGOrZmZTX9\\\\\\\/CfY93hPSdU\\\\\\\/WNONneaTxvev5ywvCNUFpqV76vKeaXHWUWkvt5K4lFMI7d2WhdZKgp\\\\\\\/CJ58++KOglyI\\\\\\\/97Bjx4tneJPoNzw2QpNSXHt2l5XJ1ts5C2Kw+7cP25NIn4yeLwyie7dintBlk7zbGoP9FbMQ0qV9gaYkdc0OakfCpP2ZGPPOlr7cEZ9JrF9DTBjqhMcGBt467s5GhY+wWWQLW8oCu5DwbK4TImrrzBb5V+RFYx6t1gSe6L2kL4ploheFD7rvhdLNcvzOXZKJa2D4mvj28IKOvJ+OWOtnfflaSSd703stGmBnqheS8LvgLrnZqycJ1PX9dS3Y+aoGiXcGZGttjn4ymqc18F8FA0i4xqGKcLOJTw\\\\\\\/fUT3igXF6nqYvUjPOjXYQ8M+mLNZoQ6T\\\\\\\/vd70eh4Y7gMgjtnvNq5HW5AJ3rQACpbNv8HRcu+EBY1PtYlTtEK908Kf63i\\\\\\\/YgrbryMrcix3JMCyaR+EaWiB2bj4jHZGkNoKfhQL1rlV\\\\\\\/uW0er9gdncfFnKCvZZHJoGp9uXgeC+3t1DwkCnhe4yY\\\\\\\/6ReLrpT5s9GsmQiqsyUGEuIrUBtiOr4e5AipGRrnAs3sMi4Al+92BzArSrGmmka6GtlIR5bsmAM9rcFboVoORDMyCkwz7VUr20od7bi1BiBPyXbxEV3p+piAzd0J+wboctzA8rAdteUc8j5ofkvGX+lE5943Rng2iCWXybO0aT76Ye7W0nFdMXqkcUy4fqHr0TKExg12sCE0YPEjbPk+ZqMMklZhoL9tUhmWK1EKBRrahjf9SDOpZuZw9j3oJfxWrGokB5sBJXqTI9IocHDzgdxTn9n4KjO9Q8u6i1YHkd4HcVlXzrhRcnhRhdyJsa2K6KolmOnk2BRA\\\\\\\/xIopoe2gotR7thCkmEYlMNbBABeKYrZVa87DsnvT6jbqknM\\\\\\\/lXMr6+jnfe\\\\\\\/b9h8s7OGh08C7f+nxAwkmuxVUEvG0eu4v0opgt7iokJngLCzNVAHxeXOuyvrdmepwHeZxaUhr3xkPR1Q6\\\\\\\/VPPDZkU70fVTC1eTc7WIVOdvIp7uwZsFYm6mACB5iWLAOuqnKNag9yNhKH\\\\\\\/8ssyGFq\\\\\\\/XSVIgVCJ4dTM+l6hw3R\\\\\\\/6Fun\\\\\\\/JuU87bP\\\\\\\/Crk+IRUfRVN35PfC8h2ZgUf2v4RW5PykOdTWdZWBvDGA3VndYdXr7xTCHNwarY3tBRB3M8suN5do4eoz+E4KyfJKSm0t0sZBOOrRPK0uuzeMrnS8iEx1MvE72lMHdZi+qn8rSsyWbw0H3ua45hoMkeaJqMoAFMoj2mN68wkFthmLGCvdTvDx6u9sBeNtuqHShQqSHjD9Cyf1xRk8eOim8DLSHTz140\\\\\\\/usaukqcOaZ8Q\\\\\\\/c4wyeL8PizGh64UD5fuE5Vx+jc9Gsi\\\\\\\/CZiIFbpBO6G5mk1q6vV7fMrTgYnql9zZ9mLLYpCUEq+R+nicx7MlLIZPXrQJEuPuGIqIJNcKJOhfHlirRw5q3bG+6pskMSDav9YdkByXcaO8LaY5G7NsAIBTDTMPQ67tPMcMlwiMPT4\\\\\\\/hFrlLv8Ihhi9qxu1u4pSOkCdx8w2JRT3m6ZyOa\\\\\\\/dzUj9VGQm8OUsds2f8pCid1L1szggtQhJzmEllCIysG4vlUU\\\\\\\/VfjFM4f3JX78YSsOh96jytvgf8l7W7y1NMiKf1W\\\\\\\/7U49hmJgL8q65J4qOD6Ar3\\\\\\\/IyaL3vfBNoE\\\\\\\/qa1027JWEHook3opAXTfR6ktBjJAd+NcGmai60bJv8jcigGvpNrs8Trh4JAyoOZHeUEi4RE0KwNeTopxiWgpzL+5XPelGirWbDTC8vUGkhToutsvgUtRW1fUmsfLS99Sf7EC2wNBIKN0wURaV4wo2KR\\\\\\\/nyEQzymfBY8W2rPhfR1HF8rgPNjSOJLUJQnrWh7YcQIs06cv5ObI6VYI0GtI\\\\\\\/gtnxTlsFHRBiNXBWCqmwOP4paq6gU6jf2TuBwIHjfh9DIwAUsMwvKWl\\\\\\\/kld\\\\\\\/pS+QbGpdl96nRwGXa859b3fHM\\\\\\\/rH+2itVFWeoy9QddaP1e6lteo+nZMUKiyVs19ysxWjQ7tLMClyj8MAvu2Ote+xBAQGWKpDMQm+TD3kVb6dPPL7QBt4FyzphrHa\\\\\\\/9a3WWkZ6+qNIANDBjhDmjBtbHMyJrVJ+IK7VrZWX6wJSi3sbs2hhlmUWaQzSU\\\\\\\/Kt38XN8bXp3NgycPvKuXAdddJ3Zlofc\\\\\\\/6yX6svnbGFv5eu+MZPkxUSqO6n4+ApSfZzCcc\\\\\\\/kIpCUPjnkp8w3bOO7U\\\\\\\/phBa4unG3vwe8Oat05SF++485nYaMlWFrrzWmcmUVUGUSAu6PKij+gRaaG9KyP2f5otU8bKZKtiDgSWSox+6c3tReVPt4R\\\\\\\/WW2y267AYBvER+f5tLmR\\\\\\\/F8IiGoX0w91sSyj82Asns5S48L9lht5mvATrMGcsfjMVKVDCrGhpxME2LbwHn47V90R\\\\\\\/h95LEDxeY9\\\\\\\/3z5HDSpytNHIZvWO9EcAXQ74QPqR8u6dTsgSrtqb\\\\\\\/j9c5kjgsuIon2BkjNU\\\\\\\/XVzZM0YgstGVmTbW\\\\\\\/6ZSPEN9ulJ02kVX3aoraOUuunfiQy2N4H2MUdXa4YyqIl9Zv9i6HiDP3opL18DdtCg\\\\\\\/qjMQI9FhDXrXOvQf8sVmJB5pGMZpipz\\\\\\\/ayr45MxaG\\\\\\\/BsStwG4CKdHmJKUxI9dg3WzingRPmGgPE4qvov1hXWJU3mt4gF6w2qihuH03eAbRBEnC0eV0KcvUElzyrBgXUa9rBIsg8eF0XpuPWJzG9FBm8nNkz2OwKn5fL3AVLExGMwfTXFpLksUweUPZ8QcZEvakHhtzg7FF8M0jEOR7C19KT66abMLVF1qqmvJGWXNluwlOZtwHoxJvqCG2UrHfae6eLjDuf8lGmTyabnzWxSV4DDbsoLu73U2DDTq3vcdbsSBIdJ1+dyDUw4lxnNsamWNHIwq8uOFI7nuf6FzlSSJiq0fxZyH2r9AkiEqEwkdZ7R9LT8ZOpkpoN5S\\\\\\\/uvvNic6SxY6mLRSRvxJl6bg8Xd01qkqL92Rq\\\\\\\/nNVf521iZZLsbmsjwfiZNEuyBlo\\\\\\\/6wFow\\\\\\\/K1SFKI5VPzKMLv4asCfS665sOJMqUz7RnxA9V9cuw3uNSM7AE4XRM8xreHopnr01eiWbZWlbQoxAQySJp5iuj4Zw+q\\\\\\\/kS7JS6qkqAEL0ZZxG8\\\\\\\/soqVzUZgj+lZIChJPyOdt+SD09SQ9ROeG7prVWsH3oGvLepcn59Uh6hlADg8m5IKC5qE7iJsRGFja4d6qeg\\\\\\\/IRF3FUQKNtEpjakE4iEpqKTT5KT+MB4p6ND0F\\\\\\\/FQG6Ed+M53+DCgjE1CtMm8cIYVPRKEBkNWI169T\\\\\\\/bIb3jSjO0onLyei5RH22g4WrTueIwxbkByJR\\\\\\\/zA4tniH51tXS2KkmxRSECTZ0sJ3jKnZEEoT8B+3fWOVLbA3sPScuD\\\\\\\/8f8lMSBZXEE6ct0uxo++1pqt2Ag0X8VMjDbw4\\\\\\\/hkoEiQs5nBfrkKu6zJldQNN+CMSIQJqoH9oSWpodGIlft1BX8SVTABtSU9AWK8Cp4mEK3\\\\\\\/HzO0gmFAmwppJnI3sd\\\\\\\/kCt49sly8jTkmzrN63F0CoEPtF3Tdj\\\\\\\/a+LFhBvBT7qgoWCGsIH67XqD81ezwf8KGUZU79CSZ0MHd+8dqVs\\\\\\\/2Iq6EWF\\\\\\\/Y+fDZXLBCNexbtwwbWWKh5hVBJgcv8DEq6K1EUNJAwQGmjVT\\\\\\\/wSDJjW9j8J2sL\\\\\\\/I2x0ZA9dJGFGM7XfNQuPYypj\\\\\\\/9+6xerlWS\\\\\\\/aGMnA8WVhh7fDhE8s8Ml0EwX+qN3zqwDxJHEjfG1q6jOyvBwG8qQFnvow28cCZNol7siED3k5F98At2BFYpLo3+P4YNd\\\\\\\/0B2AcWyKvE7iy7hALTh4sMYfsR5Xgni4TuxV6ceKcZzCPcRkPd4wB4FhTf4wo32Iep+iIB6mAmk7lf15s4JOx6blyY7G7zBDLP8aGN8bqbQFysyIzFCKZm11EJit31Vz3j7+ODvPjdn2iy02roLRlw2\\\\\\\/Q1gUVX3dsOREEUc2E3kMJFrVbsKelQG+BxSvuAtT661gmjRGpg\\\\\\\/JAE+jpxUkC6E36GDU2Dpgfw0q0jW1qD62WSdPyzrpLenaOqpe\\\\\\\/eHJ9ZuFyoVJI6k5dA8aC7PuYc5rApp7p4o7yddYLaGqgimMAraDqXz8i3hXsXN8gAPmqZnw2vIyugNL0oHMypEBFTboHJDbL8\\\\\\\/6yoBA54v2ApNTP0uQ64ba61C3mSf\\\\\\\/14g3t5SZyLWU58wes5teCqPtEVZMkB\\\\\\\/UJEnucrIWvhMZU2SWB0dDXlIMqR0kdlZiT9ung6DzZU23rKmyZzdg1NuQAFjbVz7nwFUDBT46bjtvRSlgiCVWmrPmd2qQTqxtbSKEqSKt3kMONVvQEkeBKhHjevVGonudEE0kf6Wi7xTNn5Sa3HoOTMTGb3nwaS\\\\\\\/UIeUZHJkFfvHcHwMbZK7XyJvqS1AuVnDDQnHO\\\\\\\/8IuAhxA756tXsK7wMVvBy8G5KEoTGEMJkrUhP\\\\\\\/Bk7haEBRRyq4GbNtUHkYhCprePjjST8A19OYy4dKw\\\\\\\/6Mj4pkZGXv5P2g+LkQWYIDLkqD6fS1WV+XaO+XM5aHT+U3UieXEQznPJfwKYZoiCoWtlcr6PLVuJ+W\\\\\\\/nj3nIJirReYK0HNnQYcd19wXPrAH9xZ2bPWQ\\\\\\\/O\\\\\\\/xf+oVyHu+iDEQHrZHaX7NnV5knmnjyN36wzCupF2aHxpjWaHyELxwFV3lzYEWDzhmbEIuA\\\\\\\/CaopMjwekSCaijKrctwWzRmQ9I+PgBQBFVy9zfVt9cQzR274XmUAlUUty0V7uIvTvPNk4h+I\\\\\\\/NngcdESva6v0kaaSgEd2YGjBsDddHhUdaCf0iJ1YbvaK95lbQ5vZslXUBK1Tdiioez2A4FYn9S3o6HOrYCx8QRawKIbdsIMknhqYQSfB4ELv2pb9gV5U+VLmh4EXSyQyvOs2GXnkruKjZrG6JR29Z1V77uE0W6ru5+HWmpvB80gKen5wm7GwYd1GVhkJvTfaxL6EGazB2xDiazhnLz62B3kJdCRxnmCz7r\\\\\\\/UtdXJxyJmjrtVz9+XkIfUQGZcJAKtJMmn1mZMq0HagTJznkyP41H+C6SO3UXacpnB\\\\\\\/9eUW\\\\\\\/LLvjeJT3hzr2kCrKgcaH\\\\\\\/QmCXQm+y0bMFVgYqImROQIZ+A\\\\\\\/oRmkplDf7iqcdWbj3EQuDgpf7J1fgXCJ9SYDCR9Dh4PFOsYFh2O4U\\\\\\\/+M2wopQf6TX+haJY\\\\\\\/OT4D26wKVubXEk5NBxeFHUSajc15JJfuY7iDcknaMFUBkHJGDExs\\\\\\\/fgdWA857FOzq+zXWhpj8tqH\\\\\\\/fzxbcPmN\\\\\\\/aHCVHdr39wO3mZ67xnnZJkiL\\\\\\\/5oLI46NbGbKA2GSIG\\\\\\\/J+RoJ9MXQv9cufgtr9ZGO\\\\\\\/mBp3BiH64e6xzrdvr\\\\\\\/uLT\\\\\\\/dVebKMDsJqERv8s7oe8Br8GQXk62qY9LmTBUplnqy8fqs3uqzvXf7mED5\\\\\\\/dnhyD1W50yXqLlARY7yDIs55\\\\\\\/TqrzeUX5cXggNDWUAiDgdbuGoX8M5IJCOLyU03qCR0iKWn7\\\\\\\/P6oHdgGFQiTdGSvhskIv1WUiLCXRUlgMxEhQ6Uf2iysVQP42PLpBZ5nUdXz607hN79gcb4fnLrRWMOHs\\\\\\\/ducLOjg5mBWs0cQW3rZiS3SQgGnnwAWalPqljBh4Oa3\\\\\\\/kbO6Cq+qxTGT5xubwgsyiwyr5pn7IT\\\\\\\/cOb63ZZ+IHrsSyg6nnLZSbTNdkVCOmT2Sfq3HaeBp87fgBjVcojp\\\\\\\/qsa14qA6jKO+RvpfwBl6M1rhf3MXCtsJYjkVpvDHj1q2K8KeLd0C2O7tbZRpuocDlchf7HdHawQqzq2yqeJiaxSlL7ZNaXk2Rje6KbvPv+ozsBaqnPcSgZ13IlV4KxszNZfzhk+\\\\\\\/cjhbROxf2ButwJtAQlHHeuSYAV8GDShHMLz0p+GNBUY\\\\\\\/p+XG7MFHK\\\\\\\/wfAuNJc2aQ6tI1ACNQ7Xqoiw5ELcCTDrVTl7YNA6PmvC2a\\\\\\\/5hNCAS04QUpvKWBaHNwZ43E\\\\\\\/4voh8MFU6zYY4o9G1oSggiVQlQASfsMim7FdISYVoLjSuZaRdrNsnuzad1ncCMLqvABvNh92G1QZaacxGYZCn5sqoJB\\\\\\\/F02LAGf00q96vf5mbc\\\\\\\/E0327efiTTxZWbk7JmEAMZ0J1U+vHN5AkQFubjLV2XvzicjmAY\\\\\\\/nBBeGCr5ZNrW54R\\\\\\\/+jbgrm5cUWb5HOYKpaOHiZWNIZTsfcT\\\\\\\/lqSSdVH1v2Pf\\\\\\\/QaJg8\\\\\\\/7ZtRMX\\\\\\\/cI4fJ8hyzDrtyRYM6K+0aH9hejh\\\\\\\/hmqmYZJ2SNntRWWq6hb04kimVJI8O+1PighcqWxcDOMV18IE9151hk8uZ1w0zwzEjJeeEVzgWh66\\\\\\\/FJB+nTJAFQ9cPmp729W8W80\\\\\\\/HfEQZAgDKX+fYMW5WP91mRsDWbJtjXXsxlOTeP3dLeshlA+Kj+SyJezuZ6Wt9RfZNLZ1LQifVizoS5Y1+B+RdCfFoF2uDJmc\\\\\\\/DoCOWYw3diIlt\\\\\\\/4hHBpYg2+eQy+Cs\\\\\\\/sWf9t2C7GrfXRwkfP9uQiXQp1HiWGVZZhHwiY3Rc\\\\\\\/9i\\\\\\\/k\\\\\\\/OIy9H1RwFDnAhI2mNJuhZ8ACtO633fztpY6KqeZi3RAeCIJvru4mRc9XXSeuURt0AY1lIiXLgM9gy5McOhB9od37FU04pqwSUnqdvIYeu4\\\\\\\/xnH70SBKSYysrjLrTV82PgIhgu65QfX6\\\\\\\/D6fH8r7TvHrAsQgRo77fpXFAPwg1xXJwlS4kJR5mg1WONS87wHzwchM6BMAvsrVkvssKne25v\\\\\\\/2oP9SJTj+PWlCFrKcGYwRNlfNTM7atif3yl4j0dRGDdpXAe2MivnZYB+AHU0AlxzqKmWLGNcyBYfdQInnwkxiPdUOmMTTiWZBxZVgg4Jb7rz4b\\\\\\\/79YjUm0mWElszxTHA2LhzEE7zsGHVhwlPO4dds9\\\\\\\/x40svHEUtkQpVvLBcjAnzO6WZpojI5LciFtI2Ps+92mXxvhR8lEsTapL37i74pUbHHun9ElkN8cXTXLVU0xyZCvRgJBu9DmJy7BtzqyJWlL9uhKPm06JVgu\\\\\\\/\\\\\\\/ALpugZHzO8w8YEqXgJ8RMzp5F0bN3dbv7dobkricJKPwQX44ny9W7h6Y5o4ppCCghu+TN0qSmSNq20g1EbLxGu7K5Ppykz\\\\\\\/lEAUDmxyIdOT2EBpRZa0EIf5i4c4XHRlmyfeYSc7Rtdvi+rB4BF7gn7G6smvdmy6UifQNhnPgaonOwamoKQLA4uQCvkoUCsBfS\\\\\\\/7f9G0Wo8dOAsHB32SKm7\\\\\\\/EhCEDVkLsv14BmMF6dEwRbLCpsqi0XmtmYs6aKsxMfEV3M2GVGiwtNsOio+wRzcTESPtOTKTO60Wol0cn4nRYG5GFHVUgvb2IHAVeAyYjNJgUGGyQrkZjt38rJs0St+w1PY2zTf+M+bc\\\\\\\/i75BoLjRDr3LSqTlrORXaU5PXUGWdtUPMTMgXWxC06b5PO6+V8O5lsZaf5U5tyu5Sxd5tGk23YlG+PiUhyKRHK7W2u1eIBlF4Gll8nZ0SOmPNZ\\\\\\\/0rYJxPwLB6rRYI+8Z0LYrxfUGYvPGeE9ayJQS8Et4qxkYDoH7flI8Lo8Z99bUYCi6\\\\\\\/v\\\\\\\/pkFUgm1\\\\\\\/XbPn\\\\\\\/VsmfydxHYw0s9jD26a59zCnu4icwoocV++x+IbT8yfZAIawEUDoVHN1rC0NitWSMWNDbLCYe3c3jF\\\\\\\/c2yzqck1ZaY1jRgA3arrvEi8E2RwP4dqPnvlbNflVixCN7rcpgkPLqL1A+5Mgz36xLbWQy3HPxsKHm32II7PanLHq4oD9Rq3W3MrYj3I\\\\\\\/gcWbOm6sYmT6S\\\\\\\/SU\\\\\\\/OzMWeoVUYM4MOSFX0hr0WjJWuikHUyMy6X2r8LMi8Y14XOrgylNN+6C4iypAXR+aT2z5sbBRCg1eFJjYJ6S2wJYHx7rtR2ppym5QZzKlyjI9K8hZqxif3uOUwqvxkMZE0EqmHWgVcQVDsbH+pbZxHX4aTe1HMFZWkknM1bJvuqu7xLEEKlTT2f9wUHy+T1t6O2+Wf6LDEZyUXiylYinx17HhfU8Bx7y9q5dwgkfwW\\\\\\\/s3JzzqkllVufo3HM58vZs3yHk3OYe2coT6YF6OcReK6z71ROm+JNf06pTfUJa1QHhbYCu7\\\\\\\/Eh6fbSerlvEhTCcSBsjOU3uUbTSJ8YWlNZY\\\\\\\/pyTsRpvuhuIHHf4MV7gX8edGUAvsGbO21sv3mwGFz1yBQwm+18epqdG0wcOl7q+\\\\\\\/a7heOqg6F13VG2o1vqeMKw4AGU30gFRUB2NOg5sWV\\\\\\\/mzg2lcM1xWNuydmPzWBX6XQCxnRGrmBmUr6qaMixJg\\\\\\\/U5wHaZ8\\\\\\\/20hK0oko1gy5DeAA\\\\\\\/ZkP82NWsaFiFpd0AQ92AgzIo4T9xoMXacWid2m3cyJBIFu22GBVVV0SPNYx+sdf+ZM716AEyncEx1pGmlW226jei1R\\\\\\\/GEHXjiQPVY5WWPmLILlPJkKEb+yVIDhTRL3DxlUcIncqb0xCMJCoM98\\\\\\\/ILlF+ZiikfeeY8pfl5T9CiJ1jqHeXJbpYN2c9A5KSmd2J2rHNoHU532cwkwTdMnxmiW2yJjQhc\\\\\\\/DcSuLkOiKixuP0OOmZgHuC\\\\\\\/M68kYEJXsVOiOuEg0KL\\\\\\\/bor8IRGMtAMLbwUBmukgsEyfUihnMsMcu4kkrnvnfT31YknEc2hNpCVcQjk2o0\\\\\\\/VpDr2eKXxXkDGlK7y7jGYBV+yE69C4U4IhqKDoQ+k++xIESq6adWBkFaIN9IXhm5+ItnNjt\\\\\\\/JtenLPmWSQyVOv\\\\\\\/yd770o4LXX1t5VydQP\\\\\\\/pxyiP8LrhLhCysdwjyWasn6EnO5HV0CCZva73pM7A8NBkNAYVUUNx9zxrLSimBQ5XZhMukOjSJ0CfFhgCo3tX2v0CspzD6JjPjs6FaT9K\\\\\\\/hb8ZQlI9ux+Ad7sYHbgxqYVlCKd06s6rIHP7Nnjk52nGB+Yddjp9L0I2XA54A+cQJek3JbdH3A8F3qpLiGrgTOAG+XVlkLod387zUkAULY4TflKPbVL8Hg4ZFnVN9aaNZ69H+x9\\\\\\\/ldArSFCGttp2Up0Cq3SX2uJG5MTWsRi2xgcyTopvFyYn\\\\\\\/w0Z14HmTYU1+JfPqlq0oZwRFW3Ec+ScYZ4OPtm2Q3KrRO33PRp7eTp8IV\\\\\\\/gSS64ZPKirmUq+q4oAZTJ3xm+0rSM53QecrQZbQHIq7Iluh1O1HM0lF9fnovqskQpBEYdaQwGk1mufAm6r2t1kAGWPU4e4Bp8mFgSWvsTxTaYV7q0jqxzBIUHjuVbmO+CHaCIJ4BRD+TUu8bnqs6CbfWOHe+7dN1CVZgrzn2aHP8g6N6SPIconU33sj5mYZgEB9HEbZv94r2W8BffMQ1nZpTnzrq\\\\\\\/Sn7X5Y2dP6MUipkAuBvkwBIpCXCGkjN2172sj8TkMZWmyZYyAdLt9HYuayi23\\\\\\\/MnvpwhlDsRA9kHm0Gl4L8aIU\\\\\\\/I09JgUC+LnOjuU0\\\\\\\/eZ45N+dusGeM4tEwqHTY5LWimjagLEKmGvz\\\\\\\/kFnW6U4ZbvVQateJ2m+utWoRunYnSNlGCKAIw0MBukzNMLIhCKVExUQhTWRBFVnK1AEBF6uhs8pS7mUqOB7uCrccQJhDr\\\\\\\/307bi453cq2gIAfNoUIlFuDknXF3VgRJbZsj0gCz1Gcow28utlssJ1Vdfb1CD5K1Wp+n7A4i4Ma08KrEYN2qvw5FoJWt38VDoQXlps2AxbdofC22a7Voik+M+jf7agGM\\\\\\\/HCZFBNK8xsrRU8NeAXykpzQnw5oonjCG697p+E6LPy7M4P9b2pUy1pRcOqwJ9F+ZYHSKeDjdIlCa6wzt2Y2W0JQCOWNhS60FaX2SdKfQPFcjz8\\\\\\\/4TJY2yk49Ou0OvSQXUWeAFmkTyCWYf+OwZ+xWwT5MB7tHI2J9dzRsAUZ\\\\\\\/\\\\\\\/c3vegaxLeusIcL+Cg4aRy6XxPA3qpDcLnzOr67d1pYbGA2s0NZtoxML9G8skumsyAL8nmn1hcQYCbEQPeC4R1EVKYNS9\\\\\\\/WV4Ya0loSp6SwnpamneMmTH6OZhr2u\\\\\\\/dKukpBu1GPM4cvGU\\\\\\\/AxxHMFairrogj0eVGxRh4hZ0VSka\\\\\\\/M8FnqajsxqzjXnPVhelcxZ5IO1GBDN4zqe\\\\\\\/VMtahX\\\\\\\/rw\\\\\\\/M2A0umGuCb7jOcW2zbtq5AxbyURWBhznVJ6dbgNfDWLRuOgBcSNsKEVQZQg01pGA8rhyWjQ\\\\\\\/\\\\\\\/vJ0zZJFdtEKyLc0lHi8JMHV0z34CKq+9m6qU9e9C4xjreqvQACHkyW5OpMDmV79vl7EDCB9\\\\\\\/osrP\\\\\\\/kXVciRT6c\\\\\\\/iB+C5zMQ1UVFG1tDj4lWtu\\\\\\\/M5nujBF4OkvDcqObyTqk2WHltrPxzvIbHiRlwTaSvwEZyx10kQZ2oCeB+r2ctF5vbDYQiD3eLpnBdn\\\\\\\/uvnc7YA6DiM1cmcXuY33TGSuDmCCT1HRXfnv+VouVRlo8WNmD5ACJC0+7RlM50H2\\\\\\\/GpFmuiFNVUwqUA7BKsp5goAOIgsjdJoXBxRW9ItZK51i+0lyo\\\\\\\/SjPxmP1k6wD8ygpqj9kxB33XjdERUy6jKkcG8KidPOWYeZ5wTXv360Ph6Twziito+6Tb7vn8KhBxN+bwCwg8WsmPmBhAFD79D2fqq+MAArWUJ2\\\\\\\/mfwDwAjNqyQP6MRmXPIr\\\\\\\/zB3JVtU0ATfOyIGvhUpcwYXxFEDTNBh16li4uL\\\\\\\/h8B3S23y4z11sXgPXmJ5QU\\\\\\\/Xqyh+uNMDfAHEk0I3JkUoigWJ1cYWhEbsC7PNZtxm5U9q25gypta+6Uo0AtzrdDXOXkXksXXOKEDY6rUzgPkmgoWe20rcaHKVuoa8KIUACmXoTiqHTyHz\\\\\\\/kzQx3\\\\\\\/XpI6KXu\\\\\\\/8oZTI2HPU4V++dDNb7R+hdVN0cBuU8XA3r4BQbAi3SObidebOvvhK5Mnp8ON40EsK+J3MZZ605Do1OI3UHFKpVpL3ZpKNYv3Y0HOOVCLIaPRyxWUgrT9pY4m9IrsUVKrveuMgjEB+lLdESHx6ITmjoKSvxR+W3\\\\\\\/OkCCQ0Mi1IFSRB9aiomu+qgCAVrqgvcJ5eVYXuHoxklAavW7Z6xMbNYGhCpyTVbDa4Nbe40YilZQOFPTUDRTeSgSld+dtJ4Qty0iyUIiGlf2v17D\\\\\\\/p3QXdlTfphW4t0U+BKpMu9VJ0syaqrzMqPNhaw40pinboPDgJ8XUUkjYHD0\\\\\\\/SRL5kfRyjxIygvyXoEYPoEy+wWMF3muLZaPqAPwKEKkv+OyM2asNdXG1VQ6EV51\\\\\\\/VZJlWDGGBbWh4rIxZmwNHUrkZa6ZAelFez473cGCMeFol8pe\\\\\\\/NAvBlQgh2DDSnwcY7aM23rS9oup19Ktf5TXIiBIA0bnjd7sx7xOzrI6rt61M6+6\\\\\\\/Q4kBBC1dpIlKhocOgp9oN9PUx4rIMQ5VxWL0fpHEu7Wnmky1BbckIAuNYv\\\\\\\/YIRN1Va4XdYkVqWy4Tb5hBdNDo11LITHnq4Xz88pJPrC6g0hz9ou+qJ91BkxZNhvKnHd4iO4uop6IqBDKohMTp\\\\\\\/mYGN8INXkQcflvEHg3pKmNP3ye7+6Nv4cjhybCgEr\\\\\\\/r+wAabe0cFABA30Ew4YUvXQLkDRhhuqNtCwZEWeai33eSJ7UR8EIscU1tiAL3C2DR3pfuL2gAAx32wYhKubxJhKuO2E0IhyY2zro7vM8Bq8SNrRecUSb6G+CcLasLvR9GHWZBtUBVGyPKWGXThrhjHzt4dp9LrSA\\\\\\\/4b6uucqL\\\\\\\/bsXFTSdoDU\\\\\\\/cl4gGeDg2Vwrx7\\\\\\\/YvOpEYF7HRWOj0Ph4WM58iBjavWUnGCAUL2V3ZDXOI+rYAdcu1UMnhkm0Lo8PC\\\\\\\/taXIU7aa7A1KKGgBO86DR5Op32TWCrqUCJ0nljZ0rWlpmaVA+HLa7ZXsoJR+z\\\\\\\/gRlpjIn0YAT5kwA3lHcgTYB2urRN8UlaHm7j3J7YnBOyelUldiwmui2Rg16JH10OSG9mlk\\\\\\\/kLxZrlc\\\\\\\/YHGDNLDP0bmK96ESpJwxBZD0ctCimlnQC9sgrT9RYHN33zgQ7aX+yuAvkZP9thVw5Jimm6w3haT3Jw3fNbC\\\\\\\/UVx5Nyxw3LDAEbE41zB4DJ2y81jP6Q8RUf3A2eoxDayHAh85Z5OVLG6S1PHVjO2QFBfpJ0BN\\\\\\\/lRFbHkyHY9I2y4\\\\\\\/31230pb20MJN0KRH7zwKQ9dHGP5CKn5uBc27tjO1vxaXdveu4Kj8\\\\\\\/3UwBiAFZz9rr32URCfeq9CztI3uldaVVe6F+y\\\\\\\/YIJ+wPKxaGev76yTi+M7cE69OKxLWQTVxLUXis4sr9Vt8DLoFuQ78XwDPCxtyVua\\\\\\\/Fo95fjhlFW86OmIZMOjp1sp0Jz5lJs9aNXbCWwzIsrYTSgO4RB4pgr9cHXpx9H9YHJ5f4usBUt9364hcXjMsWkSKGflmtOQLkpGbpH4QITY5Z2EIzYhXYvBEItMGkj5ap6awcpc3bVHII5pOcgVojhTr3A\\\\\\\/xkIJ2DBChGpw10IeRCO8ssWTQrQ7Ohuq46xVWcUJFu7aRH3MEmxPDVhNLkyc1Q5j886fVbM1k0Ip7fpqMvOoxVlLgGTFvKrWA47nhOuv9TwJG3sfHVNwMSte04fscBd7RnJSWvi2A0Kx127g9YfW09iANjCoJl0JJllKJqLZY7EZT+vcyYdiG8S2+iJ31BpoObJkZ9iWC3WebWMxkMwUiP1gNTcMfeqDPKW1Sxd8WE6g99wm4N8SlFALJ1d3RN4pDF3ARiCGciLN49KZn4lDflkfbZVwhcURm++3kxzxYCyKIdkTClssbru3eXHEzZHbewhHpJJFssxM0a9qLuUzFDNV1NzxXznjHxl91o2XClv+sjv6Gl+ucmYB2StqAdXpjgruJsfdMfn80tKCXa3y\\\\\\\/kCs1dWBghpV28SULfU+pIw9wdBTXuQUu06TpHGwf5ST8JFQkmpPZ0SrrCfsRi8HFMpMnGO83lmkGPYGl4LsWaSnyQtrvTZ+yYNWjLRVuXujPdAowzx20GXePEylv8Nrb6Mggv7ly8pXMDYtaRIltesN92KwnNvK29DXrfulhmnycbJzLcPIDJznXWjeGu8pRRrwV50rEw\\\\\\\/FVS7ivswgl+Eonu0jH1qY9phMlvFpl2SY64DZbp4AvxyhQQJZhKt0WGSVialqlNVxdInLgXdbIOC527\\\\\\\/67TNKQgwK6aPX5WyxODPpgI3hi5K+Ttwf3fcpwgaxU75DDd\\\\\\\/tSr29OGys4Sm4bIZxnd2BMmIetJbWLXrThce8U4TQ4jpwDQTuxcX9aCVXDD037oCGCc0gdDsg43vCFgRiQiciOyA19dF3VUc+bnltYsKnx5jrrT8nX+5EGzCIDef8I6PGj8DtgC4mTu9nZK5wrZUU0FKj4HRFZXogC8Oy+7lTuapwaakRw5OvaUG2Tw63zSYa+a8+uFs2eoBQKvb98h4CkvzH\\\\\\\/VZE8d9eGZWR7MFs2OYJigibpvKJug9YHYOIGvPn\\\\\\\/CGyuutJl2m9AjWsa9YdWE+QA9duW8m9xKOSjRMsESxTP61EcFVRv3eznNNVzaEgP\\\\\\\/ZV8XvevcDyHU4znbiOFlaxlnHfLAskeJYpl6Pa6+jSKQy9uHZFsWwmNgP8aa4FeF+KE+CiAhM5KJpJJLJEqFPI+9u4J8h4qd9zZdqWl9rPwAsB4vXr1dZUQbMZTOlgWTk6MzTkzo1m7wfmmlf1+TFISfAhUJUZYNvpT9PYJg3PIU1AdLd1dgwq9LkYkhpeJqtMY1hNJAyQND6MYJieYZ44NRkffWcM6\\\\\\\/EilkS5hXBF\\\\\\\/n1f37SWW8Q7ivVhY3IEfB5AKTIw+2Gxii7evY7aoLmge9U0ogsbDlDO7P7smlAQbp5FnFCTtuO9RX5ChTZlOM17FEJf3wGnVNPGJ3jr5APKKW0RDgaOaoEclMnjg95iuPsH\\\\\\\/+oa9bghT4kKHDS\\\\\\\/TCfaSMc7e0XPF5oWFIFthq6YcGvNtGDWlC2PDj65QjqIM\\\\\\\/ZvCiZTS7WHH2pJkaVxQrFPe3sm4hvjAb9lW15o3dJlMrArcaoSY1pKjSlEzbwrGFEN2rN1Qp+3x8i4uuhZL5FQDlfEq2L2zy2X23FGnTy3eH+8mKbLpLQbAekf0DfW4QUXiX\\\\\\\/IxFFMjJJDLoJ6DPRymmw8dUyYuDyRfoailpAYlKLgYfF7MTlFlXafviFAXOx1eGTLG4eNO6BaJyrBP3G8j6T\\\\\\\/gxiCGoJ71bwCFqjRwZomyd4qfBCJMpKApvV68\\\\\\\/OSx8LNSA+NDk4xexjxusU+lJ5WPzy8Omp5wzd4YtcWXoAw4hL5GgrdvnpJ+B6SbM\\\\\\\/fB7t\\\\\\\/PWPfI3ICNxMw3vohn7hdY+V+ms3Qbz1QCWKPOfF98LikJrg1fakjerXdMCMcTc+N3rKNnr1DvLKtyAzJI2gl7kh9PSwzGQwqAAJIb+cICi5HD466PdgFdnsOrT\\\\\\\/EPYLYsfGeVG8ilf\\\\\\\/oiDTp08eprmc7cI3Kx8Fuy+ziy2XmW9r0sNlyoAE2XlVAViNwD022pMBf2XFu4HfEWkLpTPkMTINQObRnJTMr+KC2aswU2b\\\\\\\/nLnZs7K+IUA\\\\\\\/1B0NHMEm1e9XORX\\\\\\\/EbPR4SRwo\\\\\\\/JrZjjEejKwnunoqNEq5xfGRQ4E1XOMwGe3rxQhEFVHiRM+Y6Mk8RBZy+kVqqq6HsOpCeXxqnCVPrZBnJY7IgyaaI9MkqShQxNSdvioDT83qcdkj5wtl\\\\\\\/a+2TsNrwOHJn4eOjZUJV14skSRYx3ayEw\\\\\\\/M4Gsdd9At9o0b5alONInmnN8ZzUf01Jq0qLPSrwFMuETyGaI5+MZqowJUtfB5mrAE9rlKvS5ayTRl0kSJ4T5c3IWWodf5U4\\\\\\\/eclGwzdAS9uYWrayGBudw0wMKmJeSGaHEtNYH4VsY41I1ZwdbSV\\\\\\\/5WsYz\\\\\\\/Ys85f4VkaZewpkjYp3Ig4jj\\\\\\\/s3e596RpXMoYhUdu\\\\\\\/GCM7m3m6uFaTmoV7UH3It03zapCsuy9Z9zlnCuprKMk5Na16\\\\\\\/KwcLAZWTXowJg7WeE7XWUCscfUmPVWv1Cs7fQcm8aFJLfrlNqQTmCRehc9rSYDJIVagDKSs8GZVcdJPCEwHE1+lyFW\\\\\\\/v2\\\\\\\/XQ8AYicjP2JOZFcAIsGXbtI4lxhBHtb2Ta4ETLYmAaCLdEArZ+8sNgwpjE9bJaCF3pbheF0JVnXXplZFgOUSlJLWUkIM5gdH2BjgYD\\\\\\\/HZIQY0ZSD4RnSonh\\\\\\\/Esjaz1LGRwIRgdt8kZLgdJb9Zifk0OakQUCg80tMj6dG\\\\\\\/4Ieiovt3lcCVsJH3WJem+8ZeaIXBllqSrViVvhwfQbkljuyvkd4f+IO\\\\\\\/7\\\\\\\/JyI2yR9EV2+fiUqJ4AF\\\\\\\/2AP+LospuCUKrenCgQBeQPX68KaLBGi8S8\\\\\\\/UySGA3MJt9bbIrtxr9tz7ohGlF+CiS4JRBWVT\\\\\\\/qWTCvexT39v5xNGgYlmirq011RnWyEv6RGo4VInuJuvxEaHsxJWZUWg2XUUvuEixDXdigSR92jYYkrPsGIbgV4\\\\\\\/wvsC6tQL6rdb\\\\\\\/YZ9vEmunb8QCtY83RaTS9s0yQjBtsr999gqXAiWqZlMYfoRjXldZ4huHEEO2pZKR70Ww+7V2ei+KfxarvexXyEaE0dO47T7b88zOgQ7\\\\\\\/333jVPQXpdE5hzumpLJmNWsk8k7DLfcR2rFC\\\\\\\/WVBizsXnDbhGP9yJkc2VYliDjOeAjlbjFGZ52Qo0BPuYQu5XHlsiJHPuogdGpT3tsooJWOfmdkz6xlZwU1nIQI7gfilAIytjLG1Gc69XkpsGYcJnS8SCDXnOffl4acyRcXOGxh4nej9O7K7DCropBQSAqCxGIEDZmeEgj5xSPQ\\\\\\\/+a7ia338odprP5Ojp9TEl5TU7pfEpdGmENHaSpOXrB4mcdnsnOLblEGLAhKxXIE0ToOm+cHsHykxk9EgtrUvAvj9JHLCVHiKO1aLdPqAX5Bu154DWw0mvaCvzusj3dZX1HWxyIESaXhNGP6DevIn3FFFvVgFhbWgtJIP76qn9BX3P5YqEhIeJPLWlgOMgTBSaJcNhzDsaM+CxZea5FZaIO90mWVcXNtW9BTs6rMx5xWnHt5v\\\\\\\/PS8unfLYtjdQpGcIV6XPcgt658kSs1xUMzKr6CkpUdBCYWeo9BfDjB\\\\\\\/1AMEk8E2KDff+XAlpX\\\\\\\/MT5WEGtR+n\\\\\\\/T8MlNCl825D93S68N8XL6QOL4RQtasBLQYUgh9wNZv7YAzrh5Jbd\\\\\\\/nb3okymmRqo5E6Gplmlyu0moqkm4Jx2VScZZlfoiCyhbMDJhWASp3nAsCWOcUJ6odB3uB4ObcARyg0hrMf5EB+PKLKODTUkivkjWaoDGeO5sTluyryLYoZnl7Esyyf01dNQrylsqRdJhKkkCSnyWzsZs5y9hmx5ZVS6FyEpJjuuEakHGSgceYhmPiQYPGCaTMyo3fr7RDhMMSqbfGVEDDpFqigpR\\\\\\\/xsYU0d2u41noId6GyuxzGWwNxMajcmUMA7LP38frtc+OxbrbCgSarbNBoOpPV\\\\\\\/FOno3SyeiEgk2OfBCFw\\\\\\\/7FiTn6\\\\\\\/Z3Ai+JHBZGdkmg2dEJ\\\\\\\/j1V5WUFiu4+6Aa4\\\\\\\/nKAwwoKWDQW7HgGT1JANf4aw8q7VGD6Bxi3ypLaxox1DpiYfAiDIAZkl0P+n2xsfZagCn0\\\\\\\/10LCHKRun2O6ZrU6Pz9cVURWqsmIsaJ7EQcJouDOtDTrcqF3Q0zAg4anfF4tT+LVQ53uGEP5JuY0gChvRkzY\\\\\\\/\\\\\\\/FovSQHL\\\\\\\/cnRvzSmDhK7kSPkocYMlkIFZ86trb9E1zI30+IHO16IexoC+nNqcOh1JwZuRKkP8UIHigZ+4ZYQ9fm8HNTrkvwZgiWmq81Fkd2sgEmzKnUu7WypH9UysjnGn45oC1u4NtWPL3tqBv2zuGJjT7+tLFZH7uamEu3KOPPHUTcoVWvimE+lF2o7oNehyXquo8InSdtkDIWcy+rIggk6jVeZ491\\\\\\\/FMtBMSAiMdni2GpcgouNBfXCjjJ4mi9ZE8c0IMT+p3D2vPYZ1NN96sCYzT6K7elBasT5Iwat4UQo0xjc4KF2PjY7pi9neBWCPIhWrM36OBkXaWFo1KFE4kGa7sZaJfrE0uWV7FygqMoeAmabTJZJ+snderzBiSSyuGHclYfTGz45mg1yKLQNZ63wJl6ixEJqmr9t4Zc7oQ6E3+0OJj+vrkZrRViXbfUInGMzB5eEBu1lJviTjv4fkykwiaFtbRV37cFxduuQe83pcNyourRq9E5xsIvFrYAPzxuvzp2qRXH78DX9f1dDu\\\\\\\/NS+5FJN1kcJPe5kGDS6zI+d7+JZJceG3PLyrzzBlVqeIKbYfQGPuWP+M8vYI6sNKomQg3dJodlPHg6G5Tte1TekglKODFPnRDsa7nv\\\\\\\/ueg\\\\\\\/pBa5i0tJz8v6o9wvLzgHNxEW4eqjxFKM3SMoIZHbO9CFznw84rnY8INDiAB7nwrKoJXGWg8SMuQIauGY6MNT48cYe4qkyXKKSb3ynUit8Ebhmwc4fRTtPs0ozz+8SA0qqIrhM1la79qZzH0T1hW5MKwjrMg13z\\\\\\\/hPedr6yoz9djzUYs6quab4OhkG1fj\\\\\\\/Z4uWyt5DP4T6FExOjqczGo\\\\\\\/578NKKh\\\\\\\/E32iKAqVtWlMEubEHoH9P5kDxKYrpdc8Ask3\\\\\\\/mXIYANrCinIQCQBOI1tzV2T2Cjqv5cneyNcQu57cAmxawet5CP5MRV7YYYI9NNZvutr61ylLH21VL3NaQ7+lTLV5x58qFWW6BsEXtPDSfNo7lFipYzm+iKpmqMO9dOE7s0h9HAgGuDK4pYS+8dCyiIUZ3X4loMQlpiFUw5dNaCYpHsAGYeXYYI8zGCqdL77TjiQEuxZf3KnSl\\\\\\\/ZHx1FHSXClC0fFKUJ0t8EPC0VfUa8Mnnjh7YVlDuWkoYOyfujuCrMhIaaYt4iN0tjPO1CS2\\\\\\\/5qnT4BBzPbGo5Z21l9GrdB+Li4eOloPdBG4w5YCXavOOygJuP9drrUrGZi1ZNV9iWcAX4JsfISPWKs\\\\\\\/\\\\\\\/6By97UjvNU0p3sTN7\\\\\\\/yUkX6KZnXfFP7WVHZEnmiN3K0SHYhRY\\\\\\\/0McKBIGDsxMOPIskEdIexDQZfOcbbfBlfOsnpUxbhvfmoEd2LNKYqgEl75pwJOqTz+GssMyMsuCtFdkVSC2S8Pqfat2dqaVZiqTbnVqvl+M6k\\\\\\\/nV\\\\\\\/KDW5buDbAUI9SkvMEJyI1reMrgeC1aITBCaKSRgRy8ib\\\\\\\/zjxXBNV\\\\\\\/qBuOg7a7B4HVH+buRgXqwQB0IdI0WrBf4man\\\\\\\/Q2zToTKlDguPDwBLkbv2DmYWRVCEMcMAnZ\\\\\\\/i0+hmHTX5\\\\\\\/Q6c3sIYsqCFL55Ss\\\\\\\/1yC4NZSD2Uf++CdL181pE9nzAv2L16KVDxjHpYCbZwPx5MXApZI3PMeEXKf5foKTHou5GINGdzheuEBswgC7xw8s0jwu1IWI44Q\\\\\\\/YWppPhlCIgC4sRn6SzUTw8ciAt4dteE1eJbMK66O4F2xYTMg4bPNw0+Xa3NxC0qFdlF35LYsIublQNbJA80j0IM0iMf2xCzIPEBGjxI5M2npkRhD2ZVxvEnoGMXgRDaq7aoRT9ziRbA0EOANnHmjj+hbem8pEAxDgEkv0Qw1qptiFmAxMmt6W+sUttdjLuVCVvPwcQkc3bvtNSl7sa365x2Ec3e3bAXBBVrw6Sd7bdRdZPEv1dOeX\\\\\\\/B990eOMY\\\\\\\/t155kS3jL7YmMS2O05RWK3SbKSh9GGsWr7f+XByANC12gNnp\\\\\\\/Jm9WYXChG8mfrIz3cuAfZ5W+tNAhlPu0vetQ6338WIdmfPn6U3VArWNOHg0FL36gttshxMU\\\\\\\/3wvFclsMUo6QBo\\\\\\\/+1pTBjmQJBr7IVXnlIgQQJ2BI691\\\\\\\/\\\\\\\/WJWtW9XKh3dZ94HmR3TdTQBRxyfzOF3RHJPSjM+uup5PdU33q76gFcIzAVRXQ9AEgogClLXUJqYKkD+0XfrTvrPpRuK62NNIq7orciQAWPz4qWZh5A3k7QqxMOWdwHySyhJfs5M4DJVUUQHgrdPjp4uy8ftshK90372BRWBsBMpNSr4tsjIm136kLOQf7YOWQjCf45hoayb7G\\\\\\\/KuwSNomWJ3oI9ff68xVgm16mTIHalhox4Jo+YACq2Zart1QY\\\\\\\/psKeJpk7TZvy+Cvc1nw80\\\\\\\/+\\\\\\\/W0Bja3eslz9gKbOTWWnXlsl\\\\\\\/PgXb966mfn\\\\\\\/GvNph7j8fnOaTFwbH\\\\\\\/gcKmgXfITxVI73eJGhx+BX5S0MCO10QTuvZBIerfoG1HwmtFBb7qLr7YbR2Fc5w\\\\\\\/c1AbgrFV61BPJXnhzUR6FfRddZq\\\\\\\/pAuUyXJ+EReoEBR+VltiR9dejfFn7VjUMw88BAzce+6pVN8n5CfjWDnr1Cq0pB9xCtQ0T9RUbpik4bdMAKY4amwLTqsj2UY0svapBUuqbju1xh3HImgCHbylXruG5NJLnV0ozDx\\\\\\\/9y\\\\\\\/aUCwYszKx9S+IjjRi\\\\\\\/iMR7oy0nmX9alGL1y1Hl1eo0sF1lcOGBXNVL4Id9aU9X+U9xdb8vZioPjuDvMjqc0WjMECnhiqo\\\\\\\/PZmGOq5E7hH4ZioZkbtNbVvOcuLMwcJOZkKJx8hQM8b2zgiNLytHENgFrMgeKmjDOR3odoF5sAsKQjwhqMnu2wPV1rNIL07L3J64Fciw3WqOW1GbQRIkgBe85kChTvuS+lrnfdl2Dg9Ouf6fRVY2x6\\\\\\\/h2e4bD8tuFxLpouf+3nzMntJW6zSHyB4oidNFSUR6373\\\\\\\/xYKbM0ia8lFvo7hWb8GXTTt7oOM3tv7\\\\\\\/Gc6EXWFVjbmgQT3sey5NblMfT7P5Yxe32UP5T4LXZXprbydWS0f0O7LzQYgwZNQtxiYwU\\\\\\\/B990OdKvIBVxixxQ7ppfhzZoHy+PlLmF\\\\\\\/o1R7JE7VqdRTGqJX435X5BAZRMedXIf3Hrd4xc+YHG\\\\\\\/2JmTv2WsGDxByHm7eZHpkLYgPheKU1esYtb6enwhzMJ6TVR\\\\\\\/r0eOB700VI4lunYuIMOJj\\\\\\\/Ys\\\\\\\/kzXs+HeyTXgNpEBrd2Ix9in5PUmM2pt4vUwMLAwdkv7vcOXEO2dKwoPFokN6lTt6aOEMwg6NOFDy08hg+SqZB2U5ZtsR1E4ZrC85UYMulrB3HNACszMAY71GFwuGw+6sX7T2IVH8uuJpgi64BXK7VZGd1Lsyv0IBh2zNGYgZFIrl8xh9xxNZTo18KMO5gqFhsd6k7ZKRC66aUah7f\\\\\\\/H6w\\\\\\\/Hn1i8Sqh7RDuXjbJyRBOe4hBYder0CFKq\\\\\\\/WH+VgeFTrtk97W6cYY1lLkjiDC9jVQDk9aWZ+EQ\\\\\\\/cM6RJPBnZUJ4jkcrfVjqbxnYrBWZmfGrZOt38pyRWUxDxVMhBZjlBv48F65V0YgKmPqMjY8RoZsroqwnuxikuepghfUg+d9H84nZZTUlHFrSBBwP\\\\\\\/iEOu35y9OSoWyHvNh+OvOPpYapTLLNEONBmeEEJ6BLKL92FFsQdOA8qTsKhOxvTeESTd7gnnkMwjFYM2yQ7xhlrFOFCjqGBDmGxrfmKu7zONTpgOFsL3+uo4T5v2iAo9f2MUcoafPMf1\\\\\\\/0TiDcR7NquRf0lWyi\\\\\\\/mI6nj7mzpFMhyydTumuxzcSXrfzkF3TBgKVXMfT4kgVJXjLnb10kOQ2GUULfpmn\\\\\\\/NtUkQtoLsTaW+8uFx4AFJ2k+iCmShQT4tA0lrt981sJ9AK9mQnkOi3+Pmu0isxJ6T0Uv1Y2g0bwFvWTePgv1sXRBWYXDToCp4rruSny12FBjU7w6f93Ft+6gl6ODeG1U251JAglgmDBP1PVJy1YY8p1GBN8To77DFXX\\\\\\\/9\\\\\\\/X2d\\\\\\\/gipyN6kiJV3xH4aBpxro6VB+6BRiwaYiTacIa6SDU4nARrQQtqzr3sFHbyTS1EJm84+FkeTEQZk0XODjFlpmsDudzXerNAdaPOhgp8\\\\\\\/2maQAH5RnnrwLfs\\\\\\\/Y+L\\\\\\\/WlJ0JYwWZhR9dn9K6gQ\\\\\\\/MgNARBrX4qTXDGbXV7qX8EsSHpCLyb8yVVja6sJCxsjWUD0Cw83jim4oh4LSPQSwS6ujL8hVOtSzc9v1Hij5BdcS9gpLsF4x3kCVTepmXWsVvaPg7oPw1rGj3uRHSJJ\\\\\\\/bHnVqsqxXb57R00k4ufhPM2Ln6Ft7mc1WmroQMY\\\\\\\/Lox1B+AZKFi6Xu6MvktmPwxWTCeCSKre36vxHQsqmrQxfceEC5fNal+yFcp8FnfYkA7wtxHFGeME3cCtQmj2QJ\\\\\\\/0Gtz3bwdB8nNvLigMf9p+ChYI6YS6mPl\\\\\\\/7ExXMEjWw4z0l+\\\\\\\/9eidbLRjghd4Nycq2p1jAk3OUZ+dUItlEKhxBHHj2JgizKXNCqyRScVubPDgMX3ifBZ+zn9PgPZtsBSjwEhqbkNWnhbFN0UyC2nQGbf8ztO1R\\\\\\\/noYaQXiBIH9jfJo3ZzmWI+A+X6Zdb\\\\\\\/Q9NNMpEcEcvDlPDt4vbNfBOcPGyoFxaj4sLPMk495wKaby6J6\\\\\\\/OosonNiXQ63EyqgYiCYWfZ75dHiUKV6Vk2RMiHJKKmB\\\\\\\/s\\\\\\\/0O3YlcSCAralmwMZp+Uv9Vpd0Mt3APp4HefEBXDbLHy9bnhmw2nHLYq8wMihfO3sXwstDyYAdCguGhGoWvpn2pdc3PrWOWm\\\\\\\/u4X4nGRM44qMn3Dx5ZH8ZjlTh+jw46prw\\\\\\\/Re+ABMcEM5nWeAxUb2EAYgogTIS3uA4vv+ARGmv0JlKrkKlJsl0WQcdqgJocZNBMNl5t4JVrKLMqJXkNrAR\\\\\\\/HUAaraodlN8bVlDy1dIR1ovvaV+vaX4Ms4JrZolDLpLTgPsdGmeUDXFtXu7GGen+jE3fe5sHHwfkC8Ch2EuesQVglEFZl\\\\\\\/rTXkhiF2vxZza2CAjN9xZ3ef4zyxFVY9WWMOcM0YZ2qf1W1vUKvoJ4KYNqPs+XZ1a8mXpnMS6f2hE6roiv3V9MjXNT84xCJxA0CMtKG+X775CGMFRoEtKhBSOJmx79YI6Gj+BDxM\\\\\\\/9s87F1KBS5fMZ+r4Uozmey2Q04UQW7c1ZG0b2Y60h7kyxWXeWW2JsayIt3wlaTyb\\\\\\\/U+tEOzo8jBmjJwtdSW4FxhbYpaKREQ8KXv98nhfTOnAkhx5ab7jWeXjyuJ737x3qmkLgLbH1Fxrrnzs2wyTQUKgbLa\\\\\\\/eFa8SzWQuVV7BdY7zbhmCIoYFvw4s6A+KyUajfhh2LOQdcHAmPiNOqIr5Ja0HalIBsn06sz615rvFqDP51r0FszBKXcE1FeAd16Bj0akVrAh+D6iqspERD4FM\\\\\\\/MT1cxVgNA25n4oQdjmy1exlUtFdq0E9Uf2ig+ZacY\\\\\\\/vwJmA8PgxKHb1aAjuTsNDLbqwHhCQ2ZY1q+b0rEg60xWGG1nUaWnqMaB+hJxPN32pB8uJyoZWcNqrVcB0GMWkqdprAp\\\\\\\/HVkEaHaBgMotLgtRlKae8mThPUOZ7w1op6n\\\\\\\/anum8i\\\\\\\/Tbz6CBE7lsgMn8LAb\\\\\\\/yg0yAXBOIiWBTXz0tyyExIFBKeBCBm6h9HCHaj5bxJzTut\\\\\\\/e6WYeInKY72oxAPU17STHrWPpJE5IxZ5oQgcS9xMTe067z5EMmG0iFxXtJJxbOU2ryZijMa1VR\\\\\\\/DTKH+7713ptpaBkDnCkY+q89MPjRvsZyg3+4xRHdBg+P8h+kHBjuq5i9m0CTznd9KsiVqfYt\\\\\\\/cGHqysYJ\\\\\\\/UOD+gfVwNnKg7xoVDVy+f8qqEKk7k+cjN7x7EdIoTlx\\\\\\\/tAWoEK3ct2lRrpQEmD2vY6Dlp3INFoXkFNbJ0FrtFF+7hVVFHAH1EnB7xKhZei1Ms4vTyTSeo0JKEpFHkIRW2sRIzdAkgasO8LHDCxI57OgnbmoD8I0B\\\\\\\/GYB9VJNtSMKHfo6+21goanej5MU6SWw+eVInPkxvGOiv5y7gpMLjY0JnXmoq6Zr9ND7OMpTjByRithnlPelZaahuXQ6IOitz5rp3qFfzx3LrbVWjX9LXeaUjulQsHSy+0QAWNxsr8Dl4t8P2TeN83zTZq24I9jmEbDjHa+nF\\\\\\\/KAMDhpUCYy4tBu7Boz1aSLpe91dMBdH\\\\\\\/2WemfH8JPDg80U4WvNx6tUQ04HOuTTfQ5U2veZK9wt79hHyQTGEtzY5jVZI8u0Eq2qBU47jr\\\\\\\/8100O6cqKrtGdXZF7KcLMwbkINs7ZDwVH2JczBJp80cs1YNnKhMhZjLTXEgimhATsxYoOLeRHWA5wvnjaxkcwV3WM2C9JgYTA3+m7uRZz49xlarZrKTb1DzSO\\\\\\\/mlKfskXh9Cx2igCmQWbqiQxKtgWimAd+XEPSjaFuY+EG\\\\\\\/NFp3ZG5mae6YjWPVRF2tU0Ub6M6jP3CDWrm4+ktn4lSrc6a8WRd+ak2HXp63vOYD6vrAdSvqmPMe+WdGk7D9ShsDick1HxsoB5+icJO+ITVXqHrGrmGUShqnqE52ot3aIy7t5QI3WsjWizFfLYahp5T0GimAn2b3YNtQJNsvwJJkqE\\\\\\\/HV7FXIaEICqYRSNLHce9mRJ0P3\\\\\\\/sQlwf\\\\\\\/VASXyq2uM7exNAg2EHDpyybIRbBMS\\\\\\\/VguMlFl+UHyF5tChzrvqOou0Q\\\\\\\/90Hi9kzrRTQvYb9L7HlyEfI1kn80Ewabk7t8MVCjBOTYxQlmdoiAv2bKP9fjoGSG1s+jdE99ReloKSRL+Z0KuwKQWKAsNipk7jmZcehYuiRsxruwkBzoOFpuPVXxXE+X\\\\\\\/RmEdL4jAkft+jISp1ZG3\\\\\\\/M4vo6LHohHhfBYjSHH6kbSaFiNfG1FTFeOBDYMYB5RT7oHb9943pGR1wCOFxk8tfnbt\\\\\\\/B2NtcNIBqGBhWjodFwCDCc0B1Px7optDtoxF7TBHdDQrH8jgprtNk6ApdDR2F99oMKvSiAc8ZqckxeO+AT+MDPbJNhv+0yizP67Db9\\\\\\\/sIgjBCoR\\\\\\\/E4xSSRU0coTml5mprR93NSLmimD1zixHHHfvcC\\\\\\\/Qsi3bdJRnthiTSjic\\\\\\\/UNl3INEC7S618CBgK3yWlULhPWUnKCFT5\\\\\\\/4c3usMKaR2mU8nZdoj1cglogrsc\\\\\\\/a9jkngATEkW6E9UTwXwXXcc0X3+wkn1stM1KFNufX8wOV9LGpMeUK9arLhaVbFOdhboWeoU1E1W9CjPELKqDVkLAlhiqD\\\\\\\/MxXfj4TXwGbubcl0hGxzPc9iiDnoHbo342V6\\\\\\\/0er1hmbfabpnd1ZrWFfS0IXnUIFmO4IRprjX4YNZb8KJNyMCXBgLINP9796rIrq8hTk3LeSC+pwb8W+CPtoWtpHrV5MVGH3ohchLxiGBjRgNclhiQUIeDx8+IEp6VmCds\\\\\\\/RZEUZb7ic3vkFExBm4FAtwfVfkklF+KvZw+oYIzZjnY3YA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022A6sYbEpC5bT6ESzfFBb\\\\\\\/aw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bf49f7f68e4fa2f77fc7e3cdc2d73bd90e4461c728809d12796c748566265b0d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OgJG0TR6eiwDUHdmlJVBo\\\\\\\/V+RIEcD5YLMTOJqmHCTwm6UcaiXpbY6Xv92bQK54Cbfjt\\\\\\\/tOlCOC\\\\\\\/TVrJjMdX8+nBevb97sx4Zg9It+0l\\\\\\\/UzX9MruD9tYCR8lsDriXdcXGoHs8zdHKMUCQPE\\\\\\\/+qxPbw0Ud4M2\\\\\\\/C4fFV4CbH3R51zPIEd9E6TSMTnUp8lMLEkvgukXweBOnqI\\\\\\\/xN5Hm6jWoD3U3LowDSiy+2arl2UG71N\\\\\\\/gSeoK6FI+iZV43flm6\\\\\\\/kjpzQZ0U007Mifpt94dbVkpsmefFbkQZUzegoD59QVGH3Gsio1iLRcdF6jVsNiat1EcRwb+R6IjKSKfioD5Q6nEBJxOld2fecHSA\\\\\\\/AU\\\\\\\/C0FqUtHp1lMBPF3krIIDK\\\\\\\/QkVhPOuV\\\\\\\/kA6DseGRBTaxRhSWwN4iYmfHKHI7Nc16UcRt0O5sYmAdXTjLnlK47vIG5o+NL+m02DA4phcPDCZhiL2TDpA7nRc+V5czJ2XN9KBlqs6fkI\\\\\\\/Brp9JpZbYmDEKXjmTyaaF9DTsej9Ifjx5Sk76i2Ts7KOPOONsnKk\\\\\\\/c+V8nPdbbrAPXmyJmvz7rBTOu8G\\\\\\\/nqFt1dpFaExbFv5Y2psZDvkCiMatEDGntK2UGJrzXYzVPRcvHD+oHXv9kzd1\\\\\\\/otJWgHTgfl1WrBP5zxKQxvzlvC6rzfxeg2xfe0OTZG+Tip7MJxfoUOgIV+dQc\\\\\\\/ujHNtyfFFO+HZBaMNzYEGQTR0CuYo6Lj\\\\\\\/w4u76kxnmh5vls6i9x2dHJJc4mkojaWVOO7\\\\\\\/SjXTp8mpkUuvb2XhgoA0l+H2R60cVLb4xTAG9X+nz\\\\\\\/MVygY19XmzeBzF4VQGGZLYEofswu9EQioTWY69ZyZeRiuboxXnQjY6GxXqLrCvecn6Xe551u1pYwKeAOSWS6KfnOUqn+9w\\\\\\\/MB3td0JKIpoS\\\\\\\/wm78yV78JI2Q38xivUOl\\\\\\\/1LDyIzs7qtzOWKF7iTLHOQrWFDcj8vetZ\\\\\\\/NJKpjmh3Aea+EpTu0EnBe3DOPSng1MMG8TdElwmxfQbn1unAqNxA4yMr5tIPBjpGxphKqbiab6\\\\\\\/iT1Tx99rXMifA6siHOm3F0qt8HHG\\\\\\\/Q5\\\\\\\/GGD40UYfFULZ6iO0ZOKMP1SVGFHfCAxdZcs\\\\\\\/HXWl198hhfh8iMpN2Sz4+2XdGGD80EJjzYlV2dWNhUwdz4kKKcb3bjV0xlfZJuUdCsmeikbdx47VFLTz1oNRbk4LUbmpDGTS985h3V1oKRgxItwPIDS0AFrQbRMcdtmeJRhvAO+vKnsjIqz6wwPMsYsrpJbnYRsiJTvS2ueDZ2MAaKUU6PniRr0dtWlN+riN9tSdpPkIZldtFWQ3+pe+YKTisqJ+LytJyn0ADGmwWde+s3KtFdas6cB07oHVoMYxVYXFpP2\\\\\\\/5kfhZ3vXKXe+hqjCXjgwrA+mGe2fMRwdehcdwes4qFvnAgwVqLTzbQCP0aKddW+CT4GCeeL0O5mo0A5VQY\\\\\\\/eE\\\\\\\/N0pd2gqm9926KDUv54O39tQ9ZEutYxyeTdhlMfS2GjcuLJn8tsxAhxvBCPcJpm5nWzgEbtdTP5xchsvNb8lPhdmO31fGV3Zwf39MZMAZAsjNhaNtJQ9GGk237cDscySIqDVyqHjqWUFBpCHH0j2QM1pbnbLhEU3I2LwSKKR4joXJ\\\\\\\/te6dYKDyi4\\\\\\\/Gq1YyKNCNzECai0fKMmK1husvRJcH1bcczsYwjmZ6KvSbGK8ZZjX6o4obCWxMfqe4fLu5QuTxmjUWc7t+k0R8lAJsK8eGRfgiQ7lFYUhxi6XzeovPCWOmFJfAOrsVMnKoqlYG4aGyOSPIhM9MOyawxv6FpoidZwwH0fRUqp3gJvYptTjQkhad7x2s+zkUjD3jgbuvJA5HVYvY9218hH+KjhFhynNTNHYAK1tLj2vyZ4JxGWQiJFO+JS4FtTKBJfSdKuJdz8R9s9AjbCvhIKIlpOLbU\\\\\\\/BR3AfUzl9Ehml0Ljp0IO3bek65z6YQwefeUYLWYZ\\\\\\\/AJPtEQH5SyIaEh6\\\\\\\/lybDCA4HXZaTYgQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022z+NllR4wzBgjc5MMrUVFUQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e889f9e3d0770d01073a2028b979355cf439b83a4c9d853bb816906be0410858\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490111 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2513" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pvLws_Xnw2h1vzKuHbZ_3TxIbWfWBouFtFQO3YQ3AFE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2513" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aNIsIcmQKNq7tTymdJRfrBQMAYCqPYfYUPPCW6d5Nis" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/7/2501" title="Bayesian Networks Predict Neuronal Transdifferentiation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/7/2523" title="Limited Mutation-Rate Variation Within the Paramecium aurelia Species Complex" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490111" id="node490111" data-pisa="ggg;8/7/2513" data-pisa-master="ggg;g3.118.200242" data-apath="/ggg/8/7/2513.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9963-8631" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhikai Liang</span>, <span class="highwire-citation-author" data-delta="1">Shashi K. Gupta</span>, <span class="highwire-citation-author" data-delta="2">Cheng-Ting Yeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1712-7211" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yang Zhang</span>, <span class="highwire-citation-author" data-delta="4">Daniel W. Ngu</span>, <span class="highwire-citation-author" data-delta="5">Ramesh Kumar</span>, <span class="highwire-citation-author" data-delta="6">Hemant T. Patil</span>, <span class="highwire-citation-author" data-delta="7">Kanulal D. Mungra</span>, <span class="highwire-citation-author" data-delta="8">Dev Vart Yadav</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-6887-4095" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Abhishek Rathore</span>, <span class="highwire-citation-author" data-delta="10">Rakesh K. Srivastava</span>, <span class="highwire-citation-author" data-delta="11">Rajeev Gupta</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jinliang Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-4562-9131" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajeev K. Varshney</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick S. Schnable</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0001-6739-5527" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James C. Schnable</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2513-2522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200242 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490111" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhikai Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhikai%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhikai%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9963-8631" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhikai Liang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shashi K. Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shashi%2BK.%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShashi%2BK.%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cheng-Ting Yeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Iowa State University, Ames, IA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cheng-Ting%2BYeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeh%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACheng-Ting%2BYeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1712-7211" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yang Zhang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel W. Ngu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BW.%2BNgu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ngu%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BW.%2BNgu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ramesh Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Chaudhary Charan Singh Haryana Agricultural University, Hisar, Haryana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamesh%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hemant T. Patil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Mahatma Phule Krishi Vidyapeeth, Dhule, Maharashtra, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hemant%2BT.%2BPatil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patil%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHemant%2BT.%2BPatil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kanulal D. Mungra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Junagadh Agricultural University, Jamnagar, Gujarat, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanulal%2BD.%2BMungra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mungra%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKanulal%2BD.%2BMungra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dev Vart Yadav </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Chaudhary Charan Singh Haryana Agricultural University, Hisar, Haryana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BVart%2BYadav%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yadav%20DV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BVart%2BYadav%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Abhishek Rathore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BRathore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rathore%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbhishek%2BRathore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6887-4095" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Abhishek Rathore</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Rakesh K. Srivastava </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rakesh%2BK.%2BSrivastava%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srivastava%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARakesh%2BK.%2BSrivastava%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Rajeev Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajeev%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jinliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jinliang Yang</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Rajeev K. Varshney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BK.%2BVarshney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varshney%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajeev%2BK.%2BVarshney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4562-9131" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajeev K. Varshney</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Patrick S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Iowa State University, Ames, IA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick S. Schnable</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">James C. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BC.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BC.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6739-5527" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James C. Schnable</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schnable@unl.edu" class="" data-icon-position="" data-hide-link-title="0">schnable@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/7/2513" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490111" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490111/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2513.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490111" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490111/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2513.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490111" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490111/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/7/2513.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490111" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490111/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Pearl millet is a non-model grain and fodder crop adapted to extremely hot and dry environments globally. In India, a great deal of public and private sectors’ investment has focused on developing pearl millet single cross hybrids based on the cytoplasmic-genetic male sterility (CMS) system, while in Africa most pearl millet production relies on open pollinated varieties. Pearl millet lines were phenotyped for both the inbred parents and hybrids stage. Many breeding efforts focus on phenotypic selection of inbred parents to generate improved parental lines and hybrids. This study evaluated two genotyping techniques and four genomic selection schemes in pearl millet. Despite the fact that 6× more sequencing data were generated per sample for RAD-seq than for tGBS, tGBS yielded more than 2× as many informative SNPs (defined as those having MAF &gt; 0.05) than RAD-seq. A genomic prediction scheme utilizing only data from hybrids generated prediction accuracies (median) ranging from 0.73-0.74 (1000-grain weight), 0.87-0.89 (days to flowering time), 0.48-0.51 (grain yield) and 0.72-0.73 (plant height). For traits with little to no heterosis, hybrid only and hybrid/inbred prediction schemes performed almost equivalently. For traits with significant mid-parent heterosis, the direct inclusion of phenotypic data from inbred lines significantly (<em>P</em> &lt; 0.05) reduced prediction accuracy when all lines were analyzed together. However, when inbreds and hybrid trait values were both scored relative to the mean trait values for the respective populations, the inclusion of inbred phenotypic datasets moderately improved genomic predictions of the hybrid genomic estimated breeding values. Here we show that modern approaches to genotyping by sequencing can enable genomic selection in pearl millet. While historical pearl millet breeding records include a wealth of phenotypic data from inbred lines, we demonstrate that the naive incorporation of this data into a hybrid breeding program can reduce prediction accuracy, while controlling for the effects of heterosis <em>per se</em> allowed inbred genotype and trait data to improve the accuracy of genomic estimated breeding values for pearl millet hybrids.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pearl-millet" class="hw-term hw-article-keyword hw-article-keyword-pearl-millet" rel="nofollow">pearl millet</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li><li class="kwd"><a href="/keyword/hybrid-breeding" class="hw-term hw-article-keyword hw-article-keyword-hybrid-breeding" rel="nofollow">hybrid breeding</a></li><li class="kwd"><a href="/keyword/genotyping" class="hw-term hw-article-keyword hw-article-keyword-genotyping" rel="nofollow">genotyping</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-4">Pearl millet [<em>Cenchrus americanus</em> (L.) Morrone; Syn. <em>Pennisetum glaucum</em> (L.) R. Br.] is able to grow on infertile and marginal soils under limiting soil moisture conditions and high soil temperatures. It is a climate resilient species, and is one of the most widely grown millets globally <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ramya <em>et al.</em> (2017)</a>. Pearl millet can thrive in arid environments, and successfully set seed at temperatures above 40°, which would kill the pollen/stigmas of many other grain crops <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gupta <em>et al.</em> (2015)</a>. Pearl millet can also tolerate infertile and marginal soils, limited soil moisture, and high soil temperatures. While most pearl millet production in Africa utilizes open pollinated varieties, Indian pearl millet production now makes extensive use of hybrid seed generated using three line cytoplasmic-genetic male sterility systems (CMS) <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hanna (1989)</a>. Three line CMS systems employ female lines which carry male sterile cytoplasm and non-restoring nuclear gene(s) (A-lines), maintainer lines carry an identical nuclear genome to each A-line in a compatible fertile cytoplasm, resulting in male fertile plants (B-lines) and are able to maintain the male sterility of A-line, and pollinator/male lines which carry dominant nuclear restorer of fertility gene(s) (R-lines).</p><p id="p-5">Plant breeding for hybrid crops requires generating and testing large numbers of hybrids under different field conditions. Performing crosses to generate F1 hybrids is a labor intensive process. Top-crossing between B- and R-lines can reduce the amount of labor required per cross, but only in crossing schemes where many female lines are being crossed to one or a few male lines. Evaluating each new hybrid across field trials with several environments also requires significant time and resources. As a result, methods for selecting parental inbred lines and determining which crosses are likely to yield the best hybrids is a critical part of crop improvement. In pearl millet, a widespread approach has been used to evaluate the phenotypes of new potential inbred parents as a first pass screen of their potential in hybrid breeding programs.</p><p id="p-6">Traditionally, mid-parental values have been a common way to predict performance of hybrids on the basis of inbred values, combined with estimates of General Combining Ability (GCA) in cases where phenotypes cannot be scored in parental lines or individuals directly <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gowda <em>et al.</em> (2013)</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Xing <em>et al.</em> (2014)</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mühleisen <em>et al.</em> (2015)</a>. However, for traits where significant heterosis exists, the phenotypes of hybrids can vary significantly from what would be predicted through the use of mid-parent values and estimated GCA. In these cases, it can be necessary to estimate Specific Combining Ability (SCA) values for each potential cross. The incorporation of genetic markers can improve the accuracy with which both GCA and SCA can be predicted by enabling the sharing of data across multiple tested lines carrying common haplotypes <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Schrag <em>et al.</em> (2007)</a>. When applied to sets of genetic markers across the whole genome, this process is referred to as genomic prediction (GP), which can be used to implement breeding programs based on estimated breeding values from genome wide sets of markers, a process know as genomic selection (GS).</p><p id="p-7">Approximately 90-100 pearl millet hybrids are currently cultivated on about 5 million hectares in India <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Yadav <em>et al.</em> (2016)</a>. Both public and private sector organizations, including 30-40 seed companies, perform thousands of test-crosses each year. Resulting hybrids are then evaluated over multiple years and multiple locations to identify small numbers of new hybrids with superior performance which can be marketed/released for cultivation. The high investment of time and resources into initial hybrid evaluation would benefit significantly from the use of GP/GS to exclude many potential test crosses which can be discarded as unlikely to outperform existing hybrids prior to field evaluation, reducing the vast number of crosses which must be performed and evaluated.</p><p id="p-8">The use of GP/GS to obtain estimated breeding values have been widely evaluated and employed in inbreeding crops such as wheat <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Poland <em>et al.</em> (2012)</a>, barley <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Zhong <em>et al.</em> (2009)</a>, rice <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Spindel <em>et al.</em> (2015)</a>. In crops where production is based upon hybrids, genomic prediction for single-cross hybrid performance are only starting to appear in the public sector literature <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Technow <em>et al.</em> (2014)</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kadam <em>et al.</em> (2016)</a>, although genomic predictions for hybrid performance across populations all crossed to a single common tester are more common <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Windhausen <em>et al.</em> (2012)</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> (2014)</a>. Pearl millet presents an intriguing opportunity in that both hybrid and open pollinated production systems are widely employed, and phenotypic data are thus available from both hybrids and inbred R- and B- lines. A-lines, being male sterile, do not produce grain when grown in isolation.</p><p id="p-9">Here we evaluated two potential genotyping strategies – RAD-seq <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Miller <em>et al.</em> (2007)</a> and tGBS <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ott <em>et al.</em> (2017)</a> to characterize a set of inbred pearl millet lines developed by ICRISAT in Hyderabad, India, and then evaluated the utility of GP/GS to predict optimal hybrid combinations among possible combinations of these inbreds using a scheme trained using phenotypic data collected from hybrid trials alone, inbred trials alone, or both.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Field Traits</h3><p id="p-10">Field trials were conducted at four locations in India, spanning two agro-ecological zones (A- and B- zone, having rainfall of &gt;400mm/annum) of pearl millet cultivation. The Hisar and Jamnagar sites fall within the A zone of pearl millet cultivation in northwest India, while Dhule and Patancheru are located in the B zone of pearl millet cultivation in southern (peninsular) India <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gupta <em>et al.</em> (2013)</a>. While pearl millet is also grown in the A1 zone - highly drought prone areas with less than 400 mm of rainfall per year - the majority of hybrid pearl millet is confined to the A and B agroecological zones. Data were collected from 320 hybrids and 37 inbreds at field trials in four locations in 2015 in India (Dhule: N20.90°,E74.77°; Patancheru: N17.53°,E78.27°; Jamnagar: N22.47°,E70.06° and Hisar: N29.10°,E75.46°). In CMS system, A-lines are sterile and hence will not produce grain when grown in isolation. Therefore, genotyping and phenotyping were conducted on non-sterile B-lines carrying the same nuclear genome as A-lines in a compatible cytoplasm, rendering them male fertile. Lines in plots were grown in an alpha lattice design with two replicates and 28 15-plot blocks in each location. Each block included two common control lines/hybrids (ICMH 356 and 9444) and 13 experimental lines. Hybrid plots were randomly assigned to the first 25 blocks of each replicate, and inbred plots were randomly assigned to the last three blocks of each replicate (Experimental design, plot distribution and recorded phenotypes was provided in FigShare <a href="https://figshare.com/articles/pearl_millet_genomic_selection_field_layout/5969230">https://figshare.com/articles/pearl_millet_genomic_selection_field_layout/5969230</a>).</p></div><div id="sec-3" class="subsection"><h3>Phenotype measurement</h3><p id="p-11">Phenotypic traits scored include days to 50% flowering (days), plant height (centimeters), grain yield (kilograms/hectare), and 1000-grain weight (grams). The criteria used to measure each of these four traits were as follows. 1) Plant height (centimeters): Plant height for a given plant was measured from where the main stem meets to soil to the tip of the panicle of the primary tiller at the time of harvest. For each plot, five random plants were randomly selected for height measurements and the mean value of these five measurements was reported; 2) Days to 50% flowering (days): Days to flowering was measured as the time between the planting date and the date at which at least 50% of plants within a given plot exhibited the initiation of stigma emergence on the panicle of their primary tiller; 3) 1000 seed weight (grams): 200 seeds were counted out from the pooled grain collected from a given research plot, weighed, and multiplied by a factor of 5 to determine 1000 seed weight (grams); 4) Grain yield (kilograms/hectare): For each entry, all panicles within a given a plot were harvested at physiological maturity, and these panicles were sun dried for 10 to 15 days and then threshed for grain yield. Planting density and plot size varied across locations, but for each location the total yield was multiplied by the number of plots per hectare to estimate the final yield per hectare.</p></div><div id="sec-4" class="subsection"><h3>DNA extraction and library construction</h3><p id="p-12">Thirty to thirty five seeds from each inbred line were sown in a four inch pot in a darkroom at ICRISAT°s Patancheru. The pots were maintained at a temperature between 18° and 25°. Etiolated leaf tissues were harvested eight days after planting. Pooled leaf tissue from 20 to 25 seedlings per line was collected for DNA extraction. DNA was extracted using a modified DNA extraction method described by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Mace <em>et al.</em> (2003)</a>. The DNA was stained by 5 ng/µl of ethidium bromide and checked using 0.8% (w/v) agarose gel electrophoresis in Tris-acetate-EDTA (TAE) buffer for 1 h at 90 V with visualization under ultraviolet (UV) light. tGBS sequencing libraries for 192 B-lines and 192 R-lines were prepared following the protocol outlined in <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Ott <em>et al.</em> (2017)</a>. RAD-seq libraries for a set of inbreds including all but 12 of the lines genotyped using tGBS were constructed as described in <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Varshney <em>et al.</em> (2017)</a>. RAD-seq libraries were sequenced using an HiSeq 2000 and tGBS libraries were sequenced using an Ion Proton (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490279/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490279/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Comparison between RAD-seq and tGBS genotyping technologies</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>SNP calling and filtering</h3><p id="p-13">Raw sequence data obtained from both genotyping strategies was analyzed using the same analytical pipeline to enable accurate comparisons between the two. Raw reads were aligned to the pearl millet reference genome (v1.1 <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Varshney <em>et al.</em> (2017)</a>) using default settings of GSNAP <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Wu and Nacu (2010)</a>.</p><p id="p-14">After alignment, SNPs were called using the software package 123SNP <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Yu <em>et al.</em> (2012)</a> ignoring the first and end 3bp of aligned reads. After ignoring the first and last 3 bp of each read, polymorphic sites were determined using the following criteria: 1) 5 aligned reads covering the position in the genome; 2) PHRED quality greater than 20. Genotype calls for individual samples were determined in the following fashion. The genotype for a given SNP marker in a given sample was determined to be homozygous if the site was covered by 5 aligned reads from that individual samples and one allele had a frequency &gt;0.9. The genotype for a given SNP marker in a given sample was determined to be heterozygous if the site was covered by 5 reads, at least 90% of the reads support the two most frequent alleles, at least two reads supported the two most frequent alleles, and both alleles had a frequency &gt;0.2. Any cases which did not satisfy the conditions for either a homozygous or heterozygous SNP call were treated as missing data.</p><p id="p-15">The initial set of SNPs was filtered to exclude any SNP site more than two alleles were identified, sites where only one genotype call was present, sites where more than 10% of samples with genotype calls heterozygous, sites where the minor allele was not identified in at least 5 samples, and sites where &lt;20% of individuals had a genotype call for the site. These sets of filtered SNPs were used to calculate missing data rate (# of samples with missing data / total sample), heterozygosity (# of samples with heterozygous genotype calls / (# samples with homozygous genotype calls + # of samples with heterozygous genotype calls)) and minor allele frequency ((2×# of samples with homozygous minor allele genotype calls) + # of samples with heterozygous genotype calls) / 2 × total sample without missing sample). Then they were imputed using Beagle (Version: 16-06-2016). The filtered but unimputed and imputed SNP sets used in this paper have been uploaded to FigShare (<a href="https://doi.org/10.6084/m9.figshare.5566843.v1">https://doi.org/10.6084/m9.figshare.5566843.v1</a>). Genetic markers with low MAFs (Minor Allele Frequency) are frequently removed prior to quantitative genetic analysis <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Tabangin <em>et al.</em> (2009)</a>. In downstream analysis, only SNPs with MAF larger than 0.05 were employed.</p></div><div id="sec-6" class="subsection"><h3>Projecting Hybrid Genotypes</h3><p id="p-16">Hybrid genotypes for each possible combination of an A/B-line and an R-line were derived from genotypes of the corresponding parental inbred lines. If both parental lines were homozygous for the same allele at a given marker, the F1 progeny received the same genotype call at that marker. If the parental lines were homozygous for opposite alleles at a given marker, the F1 progeny received a heterozygous genotype call at that marker. If either parent was genotyped as heterozygous at a given marker was treated as having a genotype of (parent 1 genotype + parent 2 genotype)/2 on a scale of 0 to 2, where 0 is a genotype call of homozygous reference allele, and 2 is a genotype call of homozygous non-reference allele.</p></div><div id="sec-7" class="subsection"><h3>Phenotype calculation</h3><p id="p-17">A linear mixed model was used to estimate the best linear unbiased prediction (BLUP) for the phenotypic traits of inbred and hybrid lines. In the model, genotype (G), location (L), genotype by location interaction (G×L), replication (R) and block (B) were treated as random effects.</p><p id="p-18">In addition, the calculated variance of factors were used to estimate broad-sense heritability <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-1.gif"/></noscript></span></span> using the following formula (from <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Holland <em>et al.</em> (2003)</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-3.gif"/></noscript></span></span> is the number of locations and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-4.gif"/></noscript></span></span> is the number of replicates. <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-5.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-7.gif"/></noscript></span></span> represent the variance of the studied phenotypes controlled by genotype (G), interaction between genotype and environment (G×E) and residual factors.</p><p id="p-19">In the analysis presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, variance attributed to residual includes both <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-8.gif"/></noscript></span></span> as well as block and replicate variance. A customized R package (available at: <a href="https://jyanglab.github.io/g3tools/">https://jyanglab.github.io/g3tools/</a>) was used to perform the above analyses.</p><p id="p-20">The mid-parent heterosis for each trait was calculated from the BLUP values using the following formula:</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span></div><div id="sec-8" class="subsection"><h3>Genomic selection and cross-validation</h3><p id="p-21">BLUP values for lines with genotypic information – either direct genotyping for inbred lines or projected genotypes for hybrids – were used as training data for genomic prediction. For each approach described in results, predictions were conducted using the implementation of RR-BLUP (Ridge Regression Best Linear Unbiased Prediction) in the R-package rrBLUP <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Endelman (2011)</a>. The genomic prediction model was represented as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <em>y</em> is the matrix of BLUPs of all individuals, <em>μ</em> is the overall mean, k is the total number of SNPs, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-11.gif"/></noscript></span></span> is the <em>i</em>th SNP genotype, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2513/embed/mml-math-12.gif"/></noscript></span></span> is the effect for <em>i</em>th SNP and <em>e</em> is the residual.</p><p id="p-22">For each trait, randomly selected subsets of SNPs ranging from 64 (2<sup>6</sup>) to 16,384 (2<sup>14</sup>) plus total projected hybrid SNPs were tested. For each subsampled set of SNPs, the individuals were divided into 5 groups, and five separate genomic prediction analyses were conducted, using four of the five groups as training data and the remaining group as testing data. The mean correlation coefficient across these five sub-predictions was treated as a single estimate of the accuracy of the prediction model for estimates of accuracy and standard deviation.</p><p id="p-23">The creation of the five individual sub-predictions varied somewhat across the four different prediction schemes described below (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For each set of parameters with each scheme, a total of 20 sets of fivefold of cross-validation were performed. Thus, for each number of SNPs for each trait, a total of 20 × 5 = 100 sets of predictions were made. In scheme 1 (M1), the total set of genotyped and phenotyped inbreds was divided into five equal groups. Each sub-prediction used four of these five groups to predict phenotypic values for all genotyped and phenotyped hybrids. Scheme 2 (M2) utilized conventional five fold cross validation where the total set of genotyped and phenotyped hybrids was divided into five equal groups, and each sub-prediction used training data from four of the five groups to predict the remaining 20% of the data. The other two schemes utilized the same system as scheme 2, with the addition of all genotyped and phenotyped inbreds to the training dataset for all five subpredictions which either used BLUPs calculated across all individuals (M3A) or BLUPs calculated separately for inbred and hybrid populations (M3B).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Four approaches taken to training and testing genomic prediction schemes. Scheme 1 (M1) uses different sets of 4/5s of the inbred phenotypic data to build a model which is tested by comparing predicted and measured traits for all hybrids. Scheme 2 (M2) is conventional fivefold cross validation, where the hybrids tested are divided into five equal parts, and the genomic estimated breeding values for hybrids in each are predicted using a model trained with the other four parts of the dataset. Scheme 3A (M3A), follows the same strategy outlined for M2, with the the training set extended to include the phenotypic and genotypic data for the inbred lines from M1. Scheme 3B (M3B) follows the same strategy as M3A but normalizes for the separate mean trait values of the inbred and hybrid populations prior to combining them into the training dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1579289755" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Four approaches taken to training and testing genomic prediction schemes. Scheme 1 (M1) uses different sets of 4/5s of the inbred phenotypic data to build a model which is tested by comparing predicted and measured traits for all hybrids. Scheme 2 (M2) is conventional fivefold cross validation, where the hybrids tested are divided into five equal parts, and the genomic estimated breeding values for hybrids in each are predicted using a model trained with the other four parts of the dataset. Scheme 3A (M3A), follows the same strategy outlined for M2, with the the training set extended to include the phenotypic and genotypic data for the inbred lines from M1. Scheme 3B (M3B) follows the same strategy as M3A but normalizes for the separate mean trait values of the inbred and hybrid populations prior to combining them into the training dataset.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2513/F1.medium.gif" width="440" height="303"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/7/2513/F1.medium.gif" width="440" height="303"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490340" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Four approaches taken to training and testing genomic prediction schemes. Scheme 1 (M1) uses different sets of 4/5s of the inbred phenotypic data to build a model which is tested by comparing predicted and measured traits for all hybrids. Scheme 2 (M2) is conventional fivefold cross validation, where the hybrids tested are divided into five equal parts, and the genomic estimated breeding values for hybrids in each are predicted using a model trained with the other four parts of the dataset. Scheme 3A (M3A), follows the same strategy outlined for M2, with the the training set extended to include the phenotypic and genotypic data for the inbred lines from M1. Scheme 3B (M3B) follows the same strategy as M3A but normalizes for the separate mean trait values of the inbred and hybrid populations prior to combining them into the training dataset.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">To assess the accuracy of predictions for hybrids where one or more parents are completely unobserved, one B-line and one R-line were selected as “hold out” parents, and all hybrids which had either of these lines as a parent were excluded prior to the division of the remaining data into five groups. Each sub-prediction consisted of training the model using the hybrids four of these the five groups, and then predicting the genomic estimated breeding values of the hybrids with a “hold out” parent. Relative to the analysis without hold-out parents, prediction accuracy in this scenario decreased modestly and variance in prediction accuracy increased dramatically (Figure S2). Finally, all hybrids with genotype and genotype data were used to train a model that then produced genomic estimated breeding values for all 36,864 possible hybrid (Figure S3).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-26">The authors affirm that all data necessary for confirming the conclusions of this article are represented fully within the article and its tables and figures. Supplemental material available at Figshare: doi: <a href="https://doi.org/10.6084/m9.figshare.5969230">https://doi.org/10.6084/m9.figshare.5969230</a>; doi: <a href="https://doi.org/10.6084/m9.figshare.5566843">https://doi.org/10.6084/m9.figshare.5566843</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Phenotype analysis</h3><p id="p-27">Phenotypic variance was partitioned into four components: genotype (G), environment (E), interaction between genotype and environment (G×E), residual (R). For each trait, the pattern of relative contribution of each of these factors was roughly similar between inbred and hybrid pearl millet populations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Plant height was the trait with the greatest proportion of variance explained by purely genetic factors, while flowering time had the great proportion of variance explained by environments. As expected, grain yield had the highest residual value, making this critical trait the most difficult to predict accurately using quantitative genetic models. Broad sense heritability – <em>i.e.</em>, the proportion of total variance in trait values explained by genetic factors – for grain yield, plant height, flowering time and 1000-grain weight were estimated to be 0.60, 0.86, 0.88 and 0.74 respectively for pearl millet hybrids and 0.72, 0.92, 0.88 and 0.74 for inbreds. However, caution should be used in interpreting differences between the inbred and hybrid heritability values, given the large difference in the number of individuals between the two populations. BLUP values were calculated for these four traits (see methods). Each trait exhibited an approximately normal distribution (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). Heterosis can be defined in different ways but the definition employed here is mid-parent heterosis which is the degree to which the measured trait values of hybrids tend to exceed the average of measured values for the same trait in both parents. Mid-parent heterosis was observed for all four traits with median values of 8% (flowering time), 15% (1000-grain weight), 37% (plant height) and 84% (grain yield) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). Note that the direct of effect for heterosis was reversed for flowering time. This is consistent with studies in maize which indicate that hybrid tend to flower earlier than their parents <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Dickert and Tracy (2002)</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Proportion of phenotypic variance explained by genotype, location (considered as a environmental factor), genotype by location (GxE) interaction for either inbred pearl millet lines or hybrid pearl millet lines. (B) Phenotype investigation of four studied traits in pearl millet population. *** p value of the significance of this correlation is &#x2264; 0.001, ** p value of the significance of this correlation is &#x2264; 0.01 and * p value of the significance of this correlation is &#x2264; 0.05; (C) Distribution of observed mid-parent heterosis for each of the four traits scored in this study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1579289755" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Proportion of phenotypic variance explained by genotype, location (considered as a environmental factor), genotype by location (GxE) interaction for either inbred pearl millet lines or hybrid pearl millet lines. (B) Phenotype investigation of four studied traits in pearl millet population. *** p value of the significance of this correlation is &#x2264; 0.001, ** p value of the significance of this correlation is &#x2264; 0.01 and * p value of the significance of this correlation is &#x2264; 0.05; (C) Distribution of observed mid-parent heterosis for each of the four traits scored in this study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2513/F2.medium.gif" width="220" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/7/2513/F2.medium.gif" width="220" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490229" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">(A) Proportion of phenotypic variance explained by genotype, location (considered as a environmental factor), genotype by location (GxE) interaction for either inbred pearl millet lines or hybrid pearl millet lines. (B) Phenotype investigation of four studied traits in pearl millet population. *** p value of the significance of this correlation is ≤ 0.001, ** p value of the significance of this correlation is ≤ 0.01 and * p value of the significance of this correlation is ≤ 0.05; (C) Distribution of observed mid-parent heterosis for each of the four traits scored in this study.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Characteristics of the tGBS and RAD-seq datasets</h3><p id="p-29">A total of 4,550 million barcoded RAD-seq reads were generated on an Illumina HiSeq 2000, for an average of 12.2 million reads per sample. tGBS libraries were sequenced using seven Ion Proton runs, generating a total of 810 million raw reads included 584 million of barcoded reads, for an average of 2.1 million reads per sample.</p><p id="p-30">After aligning to the pearl millet reference genome and quality filtering (See Methods), 649,067 polymorphic SNPs were identified using RAD-seq data and 73,291 SNPs were identified using tGBS data. As expected given the different subsets of the genome targeted by these two technologies <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Miller <em>et al.</em> (2007)</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Ott <em>et al.</em> (2017)</a>, there was only minimal overlap between the two methods with only 439 SNPs identified and scored by both technologies. The missing data rates for RAD-seq genotypes exhibited a bimodal distribution while tGBS genotypes exhibited a unimodal distribution skewed toward high missing data rates. RAD-seq genotyping was much less likely to genotype sites as heterozygous, which may reflect a difference in the technologies, or may be explained by the observation that many SNPs identified by RAD-seq had low minor allele frequencies, while tGBS SNPs, tended to have higher minor allele frequencies (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). A more detailed comparison of the outcomes of RAD-seq and tGBS genotyping is provided in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Downstream analyses utilized only those SNPs with MAF &gt;0.05 from each dataset (15,306 RAD-seq SNPs and 32,463 tGBS SNPs).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of missing data rates (A, D), heterozygosity (B, E), and minor allele frequency (C, F) for SNPs identified and scored in either the RAD-seq or tGBS dataset. A-C summarize raw SNP data prior to imputation. D-F show densities for the same characteristics subsequent to imputation. However, no missing sites were left after imputation, hence panel D is blank. Dashed line in C &amp; F indicates the cut off of MAF = 0.05 for SNPs which were utilized in downstream genomic prediction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1579289755" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of missing data rates (A, D), heterozygosity (B, E), and minor allele frequency (C, F) for SNPs identified and scored in either the RAD-seq or tGBS dataset. A-C summarize raw SNP data prior to imputation. D-F show densities for the same characteristics subsequent to imputation. However, no missing sites were left after imputation, hence panel D is blank. Dashed line in C &amp; F indicates the cut off of MAF = 0.05 for SNPs which were utilized in downstream genomic prediction.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2513/F3.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/7/2513/F3.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490186" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Distribution of missing data rates (A, D), heterozygosity (B, E), and minor allele frequency (C, F) for SNPs identified and scored in either the RAD-seq or tGBS dataset. A-C summarize raw SNP data prior to imputation. D-F show densities for the same characteristics subsequent to imputation. However, no missing sites were left after imputation, hence panel D is blank. Dashed line in C &amp; F indicates the cut off of MAF = 0.05 for SNPs which were utilized in downstream genomic prediction.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Evaluating the accuracy of genomic prediction</h3><p id="p-32">The ability of genomic prediction using projected hybrid genotypes from both genotyping methods was then assessed for each phenotype using cross validation. For each tested set of SNPs, 20 random rounds of fivefold cross validation were performed. The median correlation coefficient of 1000-grain weight, days to flowering time, grain yield and plant height using all available SNPs was 0.73, 0.89, 0.51 and 0.72 for RAD-seq and 0.74, 0.87, 0.48 and 0.73 for tGBS (Figure S1). The differences in prediction accuracies observed for the two methods, either utilizing random sub-sampling of equal numbers of SNPs for each dataset or all SNPs obtained using each genotyping method were not statistically significant (student’s <em>t</em>-test).</p></div><div id="sec-14" class="subsection"><h3>Comparison of prediction models</h3><p id="p-33">Four different approaches (see Methods) to genomic prediction were evaluated to test whether inbred phenotypic data can add value to genomic prediction as part of a hybrid breeding program. Scheme M1, which used trait trait from inbreds to predict genomic estimated breeding values of hybrids performed the worst of all four approaches for all four phenotypic traits tested. Notably, the rank of traits by mid-parent heterosis had a perfect negative correlation with the rank of the traits by phenotypic prediction accuracy using inbred parent training data. Scheme 2 (M2) produced a significant increase (<em>P</em> &lt; 0.05) in accuracy relative to the scheme 1 (M1) for all four phenotypes, although, consistent with its high residual values when fitting the original BLUP, the accuracy of prediction for yield was the lowest of the four phenotypes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Scheme 3A (M3A), which merged phenotypes from both inbred parents and hybrid trials to predict hybrid trials performed equivalently to scheme 2 (M2) for 1000-grain weight (1kwt) and flowering time (D2F), the two traits with the lowest degree of mid-parent heterosis. However, for traits with significant amounts of mid-parent heterosis (grain yield and plant height), adding inbred parent phenotype data to the prediction model either provided no benefit (plant height, <em>P</em> = 0.17) or significant <em>decreases</em> in prediction accuracy (grain yield, <em>P</em> = 0.09e-2) compared to a purely hybrid phenotype data training set. Scheme 3B (M3B), which, instead of employing absolute trait values for inbreds and hybrids, summarized phenotypes as the differences between the predicted trait value for a given inbred or hybrid line and the mean trait value for either all hybrids or all inbreds (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5D</a>), performed approximately equal to, or sometimes marginally better than M2 (hybrid only scheme). An additional 1,000 permutations of fivefold cross validation were conducted for scheme 2 (M2) and scheme 3B (M3B) using the “All SNPs” dataset. The increase in prediction accuracy in M3B relative to M2 was statistically significant for two out of four traits tested: flowering time (<em>P</em> = 6.00e-4), and grain yield (<em>P</em> = 5.03e-9).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy for each of four phenotypes scored in this pearl millet population employing the four schemes outlined in Figure 1 using tGBS SNP calls. Scheme 3A (M3A) employed absolute predicted trait values for inbreds and hybrids to train a genomic prediction model, while scheme 3B (M3B) employed predicted trait data for inbreds and hybrids calculated relative to the separate mean trait values for inbred and hybrid lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1579289755" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy for each of four phenotypes scored in this pearl millet population employing the four schemes outlined in Figure 1 using tGBS SNP calls. Scheme 3A (M3A) employed absolute predicted trait values for inbreds and hybrids to train a genomic prediction model, while scheme 3B (M3B) employed predicted trait data for inbreds and hybrids calculated relative to the separate mean trait values for inbred and hybrid lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2513/F4.medium.gif" width="440" height="298"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/7/2513/F4.medium.gif" width="440" height="298"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490257" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Prediction accuracy for each of four phenotypes scored in this pearl millet population employing the four schemes outlined in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> using tGBS SNP calls. Scheme 3A (M3A) employed absolute predicted trait values for inbreds and hybrids to train a genomic prediction model, while scheme 3B (M3B) employed predicted trait data for inbreds and hybrids calculated relative to the separate mean trait values for inbred and hybrid lines.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A proposed model for the decrease in genomic prediction accurary for high heterosis traits when inbred individuals are introduced into training populations. A) Distribution of BLUP scores for yield for populations of hybrid and inbred individuals based on a combined BLUP analysis. B) Distribution of scores for a hypothetical marker having an equally large effect size in inbred and hybrid individuals. When allele frequencies differ between these populations, and the ratio of hybrid to inbred individuals may vary between the groups of individuals with genotype AA or with genotype BB. C) Distribution of scores for a hypothetical marker with no effect on trait value. D) Distribution of BLUP scores for yield for populations of hybrid and inbred individuals based on a separate BLUP analysis for hybrid and inbred individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1579289755" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A proposed model for the decrease in genomic prediction accurary for high heterosis traits when inbred individuals are introduced into training populations. A) Distribution of BLUP scores for yield for populations of hybrid and inbred individuals based on a combined BLUP analysis. B) Distribution of scores for a hypothetical marker having an equally large effect size in inbred and hybrid individuals. When allele frequencies differ between these populations, and the ratio of hybrid to inbred individuals may vary between the groups of individuals with genotype AA or with genotype BB. C) Distribution of scores for a hypothetical marker with no effect on trait value. D) Distribution of BLUP scores for yield for populations of hybrid and inbred individuals based on a separate BLUP analysis for hybrid and inbred individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2513/F5.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/7/2513/F5.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2513/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490249" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">A proposed model for the decrease in genomic prediction accurary for high heterosis traits when inbred individuals are introduced into training populations. A) Distribution of BLUP scores for yield for populations of hybrid and inbred individuals based on a combined BLUP analysis. B) Distribution of scores for a hypothetical marker having an equally large effect size in inbred and hybrid individuals. When allele frequencies differ between these populations, and the ratio of hybrid to inbred individuals may vary between the groups of individuals with genotype AA or with genotype BB. C) Distribution of scores for a hypothetical marker with no effect on trait value. D) Distribution of BLUP scores for yield for populations of hybrid and inbred individuals based on a separate BLUP analysis for hybrid and inbred individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Increases in prediction accuracy coming from increasing numbers of markers tended to plateau at smaller total marker numbers for scheme 1 (M1 which is inbred only) than for scheme 2 (M2 which is hybrid only), with scheme 3 (M3 which is inbreds plus hybrids) was intermediate between the two. However, even 64 (2<sup>6</sup>) random SNPs provided significant (<em>P</em> &lt; 0.05) predictive ability for all traits and all schemes tested. The relatively small set of inbred parents used to create the set of hybrids tested as part of this analysis may have resulted in inflated apparent prediction accuracies for each trait. When using a hold-two-parents out approach to segregating hybrids with common parentage between the training and testing datasets (see Methods), accuracy was lower and standard deviations of prediction accuracy were higher (Figure S2), indicating that our estimates of prediction accuracy are likely to be optimistic relative to the ability to predictions for hybrids where one or both parents have not previously served as parents for tested hybrids.</p><p id="p-37">Finally, grain yield and time to flowering were predicted for every possible F1 hybrid between a genotyped A/B-line and a genotyped R-line in the dataset. Within this prediction space, the highest yielding potential hybrids tend to be associated with somewhat longer flowering times defining a production possibility frontier for the trade off between growing season length and yield among the pearl millet hybrids which could be generated using the inbred germplasm genotyped as part of this study (Figure S3).</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-38">Here we found that the naive integration of trait data collected from inbred lines into genomic prediction for a hybrid breeding program can actually reduce prediction accuracy, particularly for traits exhibiting significant heterosis. However, controlling for the effect of heterosis <em>per se</em> by calculating BLUPs separated for inbred and hybrid lines eliminated this negative impact on prediction accuracy and could in fact increase prediction accuracy for some traits relative to predicting using data from hybrid lines alone. In addition we found prediction accuracy was equivalent for SNPs generated through either RAD-seq or tGBS. Given the greater number of high MAF (&gt;0.05) SNPs generated per million reads with tGBS, this methodology is likely to be more cost effective in the context of many genomic selection based breeding programs. In addition, the more rapid turn-around time enabled by Ion Proton sequencing (∼4 hr) relative to Illumina HiSeq 2000 (∼8 days for 2x100 sequencing) increases the feasibility the utilizing genomic prediction to guide real time breeding decisions. However, it should also be noted that there is nothing inherent about either the RAD-seq or tGBS protocol which prevents the adaptation of either protocol to sequencing using either instrument.</p><p id="p-39">Longer growing seasons will generally – in the absence of constraints from temperature, or resources abundance – result in more total fixed carbon <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Dohleman and Long (2009)</a>. Whether this increase in carbon fixation results in an increase in yield depends, among other factors on harvest index, the partitioning of carbon between vegetative and reproductive development. In our data, grain yield displayed only weak correlations with flowering time (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). Grain yield and plant height show a strong positive correlation with each other which is not what would be expected based on models of carbon partitioning. One potential explanation is that, in small plot trials with substantial height variation among accessions, tall plots can shade shorter plots if the experimental design is not blocked by height. This shading effect produces an apparent yield penalty for short accessions which does not translate to larger scale yield trials or commercial production. However, there may also be significant room to improve the yield and resource use efficiency of pearl millet through selection for improved harvest index. A second explanation is loci responsible for tolerance of heat and or drought stress are segregating in the population, sensitive genotypes are likely to exhibit both stunted growth and low grain yields, as pearl millet is grown in marginal soils and high levels of abiotic stress (high temperatures and water constraint).</p><p id="p-40">As described above, we found that the naive incorporation of inbred genotype and trait information into training datasets decreased prediction accuacy for high heterosis traits. In <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, we propose a model to explain this finding. When BLUP values for a trait with a high degree of heterosis are calculated in a population containing a mix of inbred and hybrid individuals, most inbreds will be assigned negative BLUP scores and most hybrids postive BLUP scores (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). Distributions of allele frequencies will vary between inbred and hybrid populations. As a result, inbred individuals may be relatively more common among the population of individuals with AA or BB genotypes. Alleles more common in inbred individuals relative to hybrid individuals will tend to be assigned a more negative or less positive effect value by a genomic prediction model trained with a mixed hybrid/inbred population than by a genomic prediction model trained on a purely hybrid or purely inbred population (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a> <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). Calculating BLUPs separately for inbred and hybrid individuals removes this source of bias in the training data by centering the distributions of both inbred and hybrid individuals (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a>).</p><p id="p-41">Consistent with earlier studies in maize, we found that inbred trait values alone were poor predictors of hybrid performance, particularly for yield e Gama and Hallauer (1977); <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Smith (1986)</a>, although the prediction values in scheme 1 (M1), trained only on inbred data were at least statistically significantly greater than zero (<em>P</em> &lt; 0.05). Here we found that when using a conventional additive genomic prediction model (RR-BLUP), traits with higher median heterosis (grain yield and plant height) experienced a decrease in prediction accuracy when inbred data were naively incorporated into the training dataset (M3A). Segregating BLUP means for inbreds and hybrids (M3B) statistically significantly moderately enhanced prediction accuracy for grain yield (<em>P</em> = 5.03e-9) and flowering time (<em>P</em> = 6.00e-4), compared to a scheme which excluded phenotypic data from inbred parents (M2). While statistically significant, the absolute values of the increases in prediction accuracy are moderate: flowering time (M2: 0.87, M3B: 0.87) and grain yield (M2: 0.50, M3B: 0.52). Yield was both the most difficult trait to predict, and the trait where the inclusion of inbred trait data provided the largest increase in prediction accuracy. In this study the number of inbred lines for which genotypic and phenotypic data were available was quite small. Even in scheme 3B, inbred lines made up less than 15% of the training dataset. Given that extensive inbred trait datasets exist for pearl millet, it may be that the incorporation of genotypic and phenotypic data from larger numbers of inbred lines would produce a larger absolute increase in prediction accuracy. As inbred lines must be grown prior to the production of hybrid seed, the collection of trait data from these lines comes at relatively low cost, and may have additional value when integrated into training datasets which also include genotypic and phenotypic data from a sample of hybrid lines.</p><p id="p-42">Even small numbers of selected SNPs can achieve relatively high prediction accuracy in this pearl millet population. The implementation of a hybrid GS/GP guided pearl millet breeding program has the potential to significantly improve the efficiency of breeding efforts (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). However, it must be noted that in our training set the high representation of haplotypes drawn from 33 common parental lines produces close relationships between sub-sampled training and testing populations, and this could also be a reason to explain why a smaller number set of SNPs can reach plateaus in accuracy for genomic prediction of some studied traits. As a result, our estimates of model prediction accuracy are likely inflated relative to predictions on unrelated populations <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Isidro <em>et al.</em> (2015)</a>. To expand the applicability of this genomic prediction model to a wider pearl millet genomic selection assisted breeding program, it will be necessary to incorporate data from a hybrids derived from a broader genetic basis.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">This work was supported by an CGIAR-US Universities Linkage Program on Dryland Cereals (404-40-89) to PSS, JCS, SKG and RKS.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Present Address: St. Jude Children’s Research Hospital, Memphis, TN.</p></li><li class="fn" id="fn-2"><p id="p-44">Supplemental material available at Figshare: doi: <a href="https://doi.org/10.6084/m9.figshare.5969230">https://doi.org/10.6084/m9.figshare.5969230</a>; doi: <a href="https://doi.org/10.6084/m9.figshare.5566843">https://doi.org/10.6084/m9.figshare.5566843</a>.</p></li><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: A. Doust</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-03-14"><span class="received-label">Received </span>March 14, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-05-21"><span class="accepted-label">Accepted </span>May 21, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Liang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.1" data-doi="10.1007/s00122-014-2305-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-based prediction of maize hybrid performance across genetic groups, testers, locations, and years.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1375</span>–<span class="cit-lpage">1386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2305-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1375%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2305-z%26rft_id%253Dinfo%253Apmid%252F24723140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2305-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24723140&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickert</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tracy</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Heterosis for flowering time and agronomic traits among early open-pollinated sweet corn cultivars.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D127%26rft.spage%253D793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.3" data-doi="10.1104/pp.109.139162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dohleman</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">More productive than maize in the midwest: how does miscanthus do it?</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">2104</span>–<span class="cit-lpage">2115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.109.139162</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.109.139162%26rft_id%253Dinfo%253Apmid%252F19535474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTUwLzQvMjEwNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNy8yNTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.7.2513.4" data-doi="10.2135/cropsci1977.0011183X001700050007x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">e Gama</span>  <span class="cit-name-given-names">E. E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Relation between inbred and hybrid traits in maize 1.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">706</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1977.0011183X001700050007x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253De%2BGama%26rft.auinit1%253DE.%2BE.%2BG.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D703%26rft.epage%253D706%26rft.atitle%253DRelation%2BBetween%2BInbred%2Band%2BHybrid%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1977.0011183X001700050007x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1977.0011183X001700050007x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.5" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with r package rrblup.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.6" data-doi="10.1007/s10681-012-0784-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissmann</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Best linear unbiased prediction of triticale hybrid performance.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-012-0784-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D191%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-012-0784-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-012-0784-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.7" data-doi="10.1016/j.fcr.2014.11.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ameta</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Seed set variability under high temperatures during flowering period in pearl millet (<em>Pennisetum glaucum</em> L. (R.) Br.).</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2014.11.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D171%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2014.11.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2014.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.8" data-doi="10.2135/cropsci2013.01.0053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rathore</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadav</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khairwal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identifying mega-environments and essential test locations for pearl millet cultivar selection in india.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">2444</span>–<span class="cit-lpage">2453</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.01.0053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D2444%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.01.0053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.01.0053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000330371000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.9" data-doi="10.2135/cropsci1989.0011183X002900060026x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Characteristics and stability of a new cytoplasmic-nuclear male-sterile source in pearl millet.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1459</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1989.0011183X002900060026x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHanna%26rft.auinit1%253DW.%2BW.%26rft.volume%253D29%26rft.issue%253D6%26rft.spage%253D1457%26rft.epage%253D1459%26rft.atitle%253DCharacteristics%2Band%2BStability%2Bof%2Ba%2BNew%2BCytoplasmic-Nuclear%2BMale-Sterile%2BSource%2Bin%2BPearl%2BMillet%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1989.0011183X002900060026x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1989.0011183X002900060026x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-8.7.2513.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyquist</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cervantes-Martínez</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating and interpreting heritability for plant b: an update</span>, pp. <span class="cit-fpage">9</span>–<span class="cit-lpage">112</span> in <span class="cit-source">Plant Breeding Reviews</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Janick</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.11" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2418-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.12" data-doi="10.1534/g3.116.031286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadam</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohn</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of single crosses in the early stages of a maize hybrid breeding pipeline. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3443</span>–<span class="cit-lpage">3453</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.031286</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D3443%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.031286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.116.031286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.13" data-doi="10.1007/BF02772596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buhariwalla</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buhariwalla</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crouch</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A high-throughput dna extraction protocol for tropical molecular breeding programs.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">460</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02772596</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D21%26rft.spage%253D459%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02772596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02772596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.14" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated dna (rad) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.5681207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzI1MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.15" data-doi="10.1007/s00122-014-2429-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mühleisen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Yield performance and stability of cms-based triticale hybrids.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">301</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2429-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2429-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2429-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.16" data-doi="10.1093/nar/gkx853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ott</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.-S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">tgbs genotyping-by-sequencing enables reliable genotyping of heterozygous loci.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">e178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx853</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253De178%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.17" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.06.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.18" data-doi="10.3389/fpls.2017.01934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahamed</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadav</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katiyar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Towards defining heterotic gene pools in pearl millet [<em>Pennisetum glaucum</em> L. (R.) Br.].</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1934</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01934</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1934%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.19" data-doi="10.1007/s00122-007-0521-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peleman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prediction of single-cross hybrid performance in maize using haplotype blocks associated with qtl for grain yield.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1345</span>–<span class="cit-lpage">1355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-007-0521-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchrag%26rft.auinit1%253DT.%2BA.%26rft.volume%253D114%26rft.issue%253D8%26rft.spage%253D1345%26rft.epage%253D1355%26rft.atitle%253DPrediction%2Bof%2Bsingle-cross%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Busing%2Bhaplotype%2Bblocks%2Bassociated%2Bwith%2BQTL%2Bfor%2Bgrain%2Byield.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0521-5%26rft_id%253Dinfo%253Apmid%252F17323040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0521-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17323040&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.20" data-doi="10.2135/cropsci1986.0011183X002600030023x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Covariance between line per se and testcross performance 1.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">543</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1986.0011183X002600030023x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSmith%26rft.auinit1%253DO.%2BS.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D540%26rft.epage%253D543%26rft.atitle%253DCovariance%2Bbetween%2BLine%2Bper%2Bse%2Band%2BTestcross%2BPerformance%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1986.0011183X002600030023x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1986.0011183X002600030023x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986C310500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.21" data-doi="10.1371/journal.pgen.1004982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection and association mapping in rice (<em>Oryza sativa</em>): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004982</span> <span class="cit-comment">(correction: PLoS Genet. 11: e1005350)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004982%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004982%26rft_id%253Dinfo%253Apmid%252F25689273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004982&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25689273&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.22" data-doi="10.1186/1753-6561-3-S7-S41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabangin</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The effect of minor allele frequency on the likelihood of obtaining false positives.</span> <abbr class="cit-jnl-abbrev">BMC Proc.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">S41</span> <span class="cit-comment">BioMed Central</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1753-6561-3-S7-S41</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D3%26rft.spage%253DS41%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-3-S7-S41%26rft_id%253Dinfo%253Apmid%252F20018033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-3-S7-S41&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20018033&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.23" data-doi="10.1534/genetics.114.165860"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipprack</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1343</span>–<span class="cit-lpage">1355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.165860</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165860%26rft_id%253Dinfo%253Apmid%252F24850820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEzNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjUxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.24" data-doi="10.1038/nbt.3943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thudi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariac</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Pearl millet genome sequence provides a resource to improve agronomic traits in arid environments.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3943</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D35%26rft.spage%253D969%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3943&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.25" data-doi="10.1534/g3.112.003699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.003699</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D1427%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.003699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.26" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and snp-tolerant detection of complex variants and splicing in short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20147302&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276045800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.27" data-doi="10.1371/journal.pone.0103165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Parental selection of hybrid breeding based on maternal and paternal inheritance of traits in rapeseed (brassica napus l.).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e103165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0103165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De103165%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0103165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0103165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.28" data-doi="10.5958/0976-1926.2016.00018.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yadav</span>  <span class="cit-name-given-names">O. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadav</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajpurohit</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assessment of diversity in commercial hybrids of pearl millet in india.</span> <abbr class="cit-jnl-abbrev">Indian J. Plant. Genet. Resour.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5958/0976-1926.2016.00018.8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIndian%2BJ.%2BPlant.%2BGenet.%2BResour.%26rft.volume%253D29%26rft.spage%253D130%26rft_id%253Dinfo%253Adoi%252F10.5958%252F0976-1926.2016.00018.8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5958/0976-1926.2016.00018.8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.29" data-doi="10.1101/gr.140277.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genic and non-genic contributions to natural variation of quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2436</span>–<span class="cit-lpage">2444</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.140277.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.140277.112%26rft_id%253Dinfo%253Apmid%252F22701078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMi8yNDM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzI1MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2513.30" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.098277</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzI1MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/7/2513.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/7/2501" title="Bayesian Networks Predict Neuronal Transdifferentiation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/7/2523" title="Limited Mutation-Rate Variation Within the Paramecium aurelia Species Complex" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/7"><span>Volume 8 Issue 7, July 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/7.cover-source.jpg?itok=dg_JR4dD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2513" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZYgkAOOC6OalIElhobAi1O2tWsfzznTroJbP8ekv-9c" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/7/2513" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/7/2513" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/7/2513" class="active" data-icon-position="" data-hide-link-title="0">Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490111" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-H2GK6vVnuzTxAnaaD1j8WpqhiHedW0wZHPhpdNnyNqU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/7/2513.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/7/2513" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-hsskQRRr1gUiYJxqMo8PonPKdFuj-fOUeC5pmyMr3_M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/7/2513" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/7/2513.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490111" id="node490111--2" data-pisa="ggg;8/7/2513" data-pisa-master="ggg;g3.118.200242" data-apath="/ggg/8/7/2513.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9963-8631" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhikai Liang</span>, <span class="highwire-citation-author" data-delta="1">Shashi K. Gupta</span>, <span class="highwire-citation-author" data-delta="2">Cheng-Ting Yeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1712-7211" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yang Zhang</span>, <span class="highwire-citation-author" data-delta="4">Daniel W. Ngu</span>, <span class="highwire-citation-author" data-delta="5">Ramesh Kumar</span>, <span class="highwire-citation-author" data-delta="6">Hemant T. Patil</span>, <span class="highwire-citation-author" data-delta="7">Kanulal D. Mungra</span>, <span class="highwire-citation-author" data-delta="8">Dev Vart Yadav</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-6887-4095" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Abhishek Rathore</span>, <span class="highwire-citation-author" data-delta="10">Rakesh K. Srivastava</span>, <span class="highwire-citation-author" data-delta="11">Rajeev Gupta</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jinliang Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-4562-9131" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajeev K. Varshney</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick S. Schnable</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0001-6739-5527" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James C. Schnable</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2513-2522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200242 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490111--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhikai Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhikai%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhikai%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9963-8631" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhikai Liang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shashi K. Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shashi%2BK.%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShashi%2BK.%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cheng-Ting Yeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Iowa State University, Ames, IA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cheng-Ting%2BYeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeh%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACheng-Ting%2BYeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1712-7211" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yang Zhang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel W. Ngu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BW.%2BNgu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ngu%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BW.%2BNgu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ramesh Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Chaudhary Charan Singh Haryana Agricultural University, Hisar, Haryana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamesh%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hemant T. Patil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Mahatma Phule Krishi Vidyapeeth, Dhule, Maharashtra, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hemant%2BT.%2BPatil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patil%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHemant%2BT.%2BPatil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kanulal D. Mungra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Junagadh Agricultural University, Jamnagar, Gujarat, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanulal%2BD.%2BMungra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mungra%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKanulal%2BD.%2BMungra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dev Vart Yadav </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Chaudhary Charan Singh Haryana Agricultural University, Hisar, Haryana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BVart%2BYadav%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yadav%20DV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BVart%2BYadav%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Abhishek Rathore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BRathore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rathore%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbhishek%2BRathore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6887-4095" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Abhishek Rathore</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Rakesh K. Srivastava </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rakesh%2BK.%2BSrivastava%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srivastava%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARakesh%2BK.%2BSrivastava%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Rajeev Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajeev%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jinliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jinliang Yang</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Rajeev K. Varshney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BK.%2BVarshney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varshney%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajeev%2BK.%2BVarshney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4562-9131" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajeev K. Varshney</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Patrick S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Iowa State University, Ames, IA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick S. Schnable</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">James C. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BC.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BC.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6739-5527" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James C. Schnable</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schnable@unl.edu" class="" data-icon-position="" data-hide-link-title="0">schnable@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/7/2513" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/7/2513&amp;title=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/7/2513&amp;title=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/7/2513&amp;title=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/7/2513&amp;text=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/7/2513&amp;title=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/7/2513&amp;t=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/7/2513&amp;title=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/7/2513&amp;title=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490111/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490111" id="node490111--3" data-pisa="ggg;8/7/2513" data-pisa-master="ggg;g3.118.200242" data-apath="/ggg/8/7/2513.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phenotypic Data from Inbred Parents Can Improve Genomic Prediction in Pearl Millet Hybrids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9963-8631" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhikai Liang</span>, <span class="highwire-citation-author" data-delta="1">Shashi K. Gupta</span>, <span class="highwire-citation-author" data-delta="2">Cheng-Ting Yeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1712-7211" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yang Zhang</span>, <span class="highwire-citation-author" data-delta="4">Daniel W. Ngu</span>, <span class="highwire-citation-author" data-delta="5">Ramesh Kumar</span>, <span class="highwire-citation-author" data-delta="6">Hemant T. Patil</span>, <span class="highwire-citation-author" data-delta="7">Kanulal D. Mungra</span>, <span class="highwire-citation-author" data-delta="8">Dev Vart Yadav</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-6887-4095" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Abhishek Rathore</span>, <span class="highwire-citation-author" data-delta="10">Rakesh K. Srivastava</span>, <span class="highwire-citation-author" data-delta="11">Rajeev Gupta</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jinliang Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-4562-9131" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajeev K. Varshney</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick S. Schnable</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0001-6739-5527" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>James C. Schnable</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2513-2522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200242 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490111--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhikai Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhikai%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhikai%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9963-8631" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhikai Liang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shashi K. Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shashi%2BK.%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShashi%2BK.%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cheng-Ting Yeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Iowa State University, Ames, IA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cheng-Ting%2BYeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeh%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACheng-Ting%2BYeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1712-7211" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yang Zhang</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel W. Ngu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BW.%2BNgu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ngu%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BW.%2BNgu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ramesh Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Chaudhary Charan Singh Haryana Agricultural University, Hisar, Haryana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamesh%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hemant T. Patil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Mahatma Phule Krishi Vidyapeeth, Dhule, Maharashtra, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hemant%2BT.%2BPatil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patil%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHemant%2BT.%2BPatil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kanulal D. Mungra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Junagadh Agricultural University, Jamnagar, Gujarat, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanulal%2BD.%2BMungra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mungra%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKanulal%2BD.%2BMungra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dev Vart Yadav </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Chaudhary Charan Singh Haryana Agricultural University, Hisar, Haryana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dev%2BVart%2BYadav%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yadav%20DV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADev%2BVart%2BYadav%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Abhishek Rathore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abhishek%2BRathore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rathore%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbhishek%2BRathore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6887-4095" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Abhishek Rathore</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Rakesh K. Srivastava </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rakesh%2BK.%2BSrivastava%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srivastava%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARakesh%2BK.%2BSrivastava%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Rajeev Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajeev%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jinliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jinliang Yang</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Rajeev K. Varshney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Hyderabad, Telangana State, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BK.%2BVarshney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varshney%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajeev%2BK.%2BVarshney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4562-9131" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajeev K. Varshney</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Patrick S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Iowa State University, Ames, IA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick S. Schnable</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">James C. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska-Lincoln, Lincoln, NE</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BC.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BC.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6739-5527" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James C. Schnable</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schnable@unl.edu" class="" data-icon-position="" data-hide-link-title="0">schnable@unl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2513&amp;rft.epage=2522&amp;rft.atitle=Phenotypic%20Data%20from%20Inbred%20Parents%20Can%20Improve%20Genomic%20Prediction%20in%20Pearl%20Millet%20Hybrids&amp;rft.volume=8&amp;rft.issue=7&amp;rft.date=2018-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Liang%2C+Zhikai&amp;rft.au=Gupta%2C+Shashi+K.&amp;rft.au=Yeh%2C+Cheng-Ting&amp;rft.au=Zhang%2C+Yang&amp;rft.au=Ngu%2C+Daniel+W.&amp;rft.au=Kumar%2C+Ramesh&amp;rft.au=Patil%2C+Hemant+T.&amp;rft.au=Mungra%2C+Kanulal+D.&amp;rft.au=Yadav%2C+Dev+Vart&amp;rft.au=Rathore%2C+Abhishek&amp;rft.au=Srivastava%2C+Rakesh+K.&amp;rft.au=Gupta%2C+Rajeev&amp;rft.au=Yang%2C+Jinliang&amp;rft.au=Varshney%2C+Rajeev+K.&amp;rft.au=Schnable%2C+Patrick+S.&amp;rft.au=Schnable%2C+James+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490111/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490111/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490111/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490111/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490111/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490111/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490111/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490111/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490111/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490111/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490111/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490111/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/7/2513" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/7/2513.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/7/2513.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/7/2513.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
