<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/7/2265" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/7/2265.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/7/2265.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/7/2265.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/7/2265/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/7/2265.atom" />
<meta name="HW.pisa" content="ggg;8/7/2265" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Genomic Region Containing REC8 and RNF212B Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (Cervus elaphus)" />
<meta name="DC.Identifier" content="10.1534/g3.118.200063" />
<meta name="DC.Date" content="2018-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Johnston et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Recombination is a fundamental feature of sexual reproduction, ensuring proper disjunction, preventing mutation accumulation and generating new allelic combinations upon which selection can act. However it is also mutagenic, and breaks up favorable allelic combinations previously built up by selection. Identifying the genetic drivers of recombination rate variation is a key step in understanding the causes and consequences of this variation, how loci associated with recombination are evolving and how they affect the potential of a population to respond to selection. However, to date, few studies have examined the genetic architecture of recombination rate variation in natural populations. Here, we use pedigree data from ∼ 2,600 individuals genotyped at ∼ 38,000 SNPs to investigate the genetic architecture of individual autosomal recombination rate in a wild population of red deer ( Cervus elaphus ). Female red deer exhibited a higher mean and phenotypic variance in autosomal crossover counts (ACC). Animal models fitting genomic relatedness matrices showed that ACC was heritable in females (![Formula][1]&lt;/img&gt; = 0.12) but not in males. A regional heritability mapping approach showed that almost all heritable variation in female ACC was explained by a genomic region on deer linkage group 12 containing the candidate loci REC8 and RNF212B , with an additional region on linkage group 32 containing TOP2B approaching genome-wide significance. The REC8/RNF212B region and its paralogue RNF212 have been associated with recombination in cattle, mice, humans and sheep. Our findings suggest that mammalian recombination rates have a relatively conserved genetic architecture in both domesticated and wild systems, and provide a foundation for understanding the association between recombination loci and individual fitness within this population.

 [1]: /embed/mml-math-1.gif" />
<meta name="DC.Contributor" content="Susan E. Johnston" />
<meta name="DC.Contributor" content="Jisca Huisman" />
<meta name="DC.Contributor" content="Josephine M. Pemberton" />
<meta name="article:published_time" content="2018-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Genomic Region Containing REC8 and RNF212B Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (Cervus elaphus)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recombination is a fundamental feature of sexual reproduction, ensuring proper disjunction, preventing mutation accumulation and generating new allelic combinations upon which selection can act. However it is also mutagenic, and breaks up favorable allelic combinations previously built up by selection. Identifying the genetic drivers of recombination rate variation is a key step in understanding the causes and consequences of this variation, how loci associated with recombination are evolving and how they affect the potential of a population to respond to selection. However, to date, few studies have examined the genetic architecture of recombination rate variation in natural populations. Here, we use pedigree data from ∼ 2,600 individuals genotyped at ∼ 38,000 SNPs to investigate the genetic architecture of individual autosomal recombination rate in a wild population of red deer (&lt;i&gt;Cervus elaphus&lt;/i&gt;). Female red deer exhibited a higher mean and phenotypic variance in autosomal crossover counts (ACC). Animal models fitting genomic relatedness matrices showed that ACC was heritable in females (&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me3&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msup&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/math&gt; = 0.12) but not in males. A regional heritability mapping approach showed that almost all heritable variation in female ACC was explained by a genomic region on deer linkage group 12 containing the candidate loci &lt;i&gt;REC8&lt;/i&gt; and &lt;i&gt;RNF212B&lt;/i&gt;, with an additional region on linkage group 32 containing &lt;i&gt;TOP2B&lt;/i&gt; approaching genome-wide significance. The &lt;i&gt;REC8/RNF212B&lt;/i&gt; region and its paralogue &lt;i&gt;RNF212&lt;/i&gt; have been associated with recombination in cattle, mice, humans and sheep. Our findings suggest that mammalian recombination rates have a relatively conserved genetic architecture in both domesticated and wild systems, and provide a foundation for understanding the association between recombination loci and individual fitness within this population.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/07/01" />
<meta name="citation_mjid" content="ggg;8/7/2265" />
<meta name="citation_id" content="8/7/2265" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/7/2265" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/7/2265.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/7/2265.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/7/2265.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200063" />
<meta name="citation_pmid" content="29764960" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2265" />
<meta name="citation_lastpage" content="2276" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Susan E. Johnston" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author_email" content="Susan.Johnston@ed.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5623-8902" />
<meta name="citation_author" content="Jisca Huisman" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9744-7196" />
<meta name="citation_author" content="Josephine M. Pemberton" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0075-1504" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Arbeithuber;citation_author=A. J. Betancourt;citation_author=T. Ebner;citation_author=I. Tiemann-Boege;citation_title=Crossovers are associated with mutation and biased gene conversion at recombination hotspots.;citation_volume=112;citation_year=2015;citation_pmid=25646453;citation_doi=10.1073/pnas.1416622112" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. S. Aulchenko;citation_author=S. Ripke;citation_author=A. Isaacs;citation_author=C. M. van Duijn;citation_title=GenABEL: an R library for genome-wide association analysis.;citation_volume=23;citation_year=2007;citation_pmid=17384015;citation_doi=10.1093/bioinformatics/btm108" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=Z. Baker;citation_author=M. Schumer;citation_author=Y. Haba;citation_author=L. Bashkirova;citation_author=C. Holland;citation_title=Repeated losses of PRDM9-directed recombination despite the conservation of PRDM9 across vertebrates.;citation_volume=6;citation_year=2017;citation_pmid=28590247;citation_doi=10.7554/eLife.24133" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. H. Barton;citation_author=B. Charlesworth;citation_title=Why sex and recombination?;citation_pages=187-195;citation_volume=74;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Battagin;citation_author=G. Gorjanc;citation_author=A.-M. Faux;citation_author=S. E. Johnston;citation_author=J. M. Hickey;citation_title=Effect of manipulating recombination rates on response to selection in livestock breeding programs.;citation_pages=44;citation_volume=48;citation_year=2016;citation_pmid=27335010;citation_doi=10.1186/s12711-016-0221-1" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=F. Baudat;citation_author=J. Buard;citation_author=C. Grey;citation_author=A. Fledel-Alon;citation_author=C. Ober;citation_title=PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice.;citation_volume=327;citation_year=2010;citation_pmid=20044539;citation_doi=10.1126/science.1183439" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. Bérénos;citation_author=P. A. Ellis;citation_author=J. G. Pilkington;citation_author=S. H. Lee;citation_author=J. Gratten;citation_title=Heterogeneity of genetic architecture of body size traits in a free-living population.;citation_pages=1810-1830;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13146" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Brandvain;citation_author=G. Coop;citation_title=Scrambling eggs: meiotic drive and the evolution of female recombination rates.;citation_volume=190;citation_year=2012;citation_pmid=22143919;citation_doi=10.1534/genetics.111.136721" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=R. Brauning;citation_author=P. J. Fisher;citation_author=A. F. McCulloch;citation_author=R. J. Smithies;citation_author=J. F. Ward;citation_title=Utilization of high throughput genome sequencing technology for large scale single nucleotide polymorphism discovery in red deer and canadian elk.;citation_year=2015;citation_doi=10.1101/027318" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=A. Burt;citation_title=Perspective: sex, recombination, and the efficacy of selection--was Weismann right?;citation_pages=337-351;citation_volume=54;citation_year=2000;citation_issue=2;citation_pmid=10937212;citation_doi=10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2" />
<meta name="citation_reference" content="Butler, D. G., B. R. Cullis, A. R. Gilmour, and B. J. Gogel, 2009 Mixed Models for S language Environments: ASReml-R reference manual. https://www.vsni.co.uk/software/asreml-r/" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Chelysheva;citation_author=D. Vezon;citation_author=A. Chambon;citation_author=G. Gendrot;citation_author=L. Pereira;citation_title=The Arabidopsis HEI10 is a new ZMM protein related to Zip3.;citation_pages=e1002799-e1002799;citation_volume=8;citation_year=2012;citation_issue=7;citation_pmid=22844245;citation_doi=10.1371/journal.pgen.1002799" />
<meta name="citation_reference" content="citation_title=Red Deer. Behavior and Ecology of Two Sexes.;citation_author=T. Clutton-Brock;citation_author=F. Guinness;citation_author=S. Albon;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=O. Delaneau;citation_author=J.-F. Zagury;citation_author=J. Marchini;citation_title=Improved whole-chromosome phasing for disease and population genetic studies.;citation_pages=5-6;citation_volume=10;citation_year=2013;citation_pmid=23269371;citation_doi=10.1038/nchembio.1414" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=A. B. Devlin;citation_author=K. Roeder;citation_author=B. Devlin;citation_title=Genomic control for association studies.;citation_pages=997-1004;citation_volume=55;citation_year=1999;citation_issue=4;citation_pmid=11315092;citation_doi=10.1111/j.0006-341X.1999.00997.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. L. Dumont;citation_author=K. W. Broman;citation_author=B. A. Payseur;citation_title=Variation in genomic recombination rates among heterogeneous stock mice.;citation_volume=182;citation_year=2009;citation_pmid=19535547;citation_doi=10.1534/genetics.109.105114" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. L. Dumont;citation_author=M. A. White;citation_author=B. Steffy;citation_author=T. Wiltshire;citation_author=B. A. Payseur;citation_title=Extensive recombination rate variation in the house mouse species complex inferred from genetic linkage maps.;citation_volume=21;citation_year=2011;citation_pmid=20978138;citation_doi=10.1101/gr.111252.110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=R. Gómez;citation_author=A. Viera;citation_author=I. Berenguer;citation_author=E. Llano;citation_author=A. M. Pendás;citation_title=Cohesin removal precedes topoisomerase IIα-dependent decatenation at centromeres in male mammalian meiosis II.;citation_pages=129-146;citation_volume=123;citation_year=2014;citation_doi=10.1007/s00412-013-0434-9" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Gonen;citation_author=M. Battagin;citation_author=S. Johnston;citation_author=G. Gorjanc;citation_author=J. Hickey;citation_title=The potential of shifting recombination hotspots to increase genetic gain in livestock breeding.;citation_pages=55;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0330-5" />
<meta name="citation_reference" content="citation_title=Documentation for CRIMAP, Version 2.4.;citation_author=P. Green;citation_author=K. Falls;citation_author=S. Crooks;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Hassold;citation_author=P. Hunt;citation_title=To err (meiotically) is human: the genesis of human aneuploidy.;citation_pages=280-291;citation_volume=2;citation_year=2001;citation_issue=4;citation_pmid=11283700;citation_doi=10.1038/35066065" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. E. Hughes;citation_author=R. S. Hawley;citation_title=Topoisomerase II Is Required for the Proper Separation of Heterochromatic Regions during Drosophila melanogaster Female Meiosis.;citation_pages=e1004650;citation_volume=10;citation_year=2014;citation_pmid=25340780;citation_doi=10.1371/journal.pgen.1004650" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. Huisman;citation_title=Pedigree reconstruction from SNP data: parentage assignment, sibship clustering and beyond.;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12665" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Huisman;citation_author=L. E. B. Kruuk;citation_author=P. A. Ellis;citation_author=T. Clutton-Brock;citation_author=J. M. Pemberton;citation_title=Inbreeding depression across the lifespan in a wild mammal population.;citation_volume=113;citation_year=2016;citation_pmid=26979959;citation_doi=10.1073/pnas.1518046113" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genomics Hum. Genet.;citation_author=K. Inoue;citation_author=J. R. Lupski;citation_title=Molecular mechanisms for genomic disorders.;citation_volume=3;citation_year=2002;citation_pmid=12142364;citation_doi=10.1146/annurev.genom.3.032802.120023" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Jaramillo-Lambert;citation_author=A. S. Fabritius;citation_author=T. J. Hansen;citation_author=H. E. Smith;citation_author=A. Golden;citation_title=The Identification of a Novel Mutant Allele of topoisomerase II in Caenorhabditis elegans Reveals a Unique Role in Chromosome Segregation During Spermatogenesis.;citation_volume=204;citation_year=2016;citation_pmid=27707787;citation_doi=10.1534/genetics.116.195099" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. E. Johnston;citation_author=C. Bérénos;citation_author=J. Slate;citation_author=J. M. Pemberton;citation_title=Conserved genetic architecture underlying individual recombination rate variation in a wild population of soay sheep (ovis aries).;citation_volume=203;citation_year=2016;citation_pmid=27029733;citation_doi=10.1534/genetics.115.185553" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. E. Johnston;citation_author=J. Huisman;citation_author=P. A. Ellis;citation_author=J. M. Pemberton;citation_title=A High Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (Cervus elaphus).;citation_volume=7;citation_year=2017;citation_pmid=28667018;citation_doi=10.1534/g3.117.044198" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. Kawakami;citation_author=C. F. Mugal;citation_author=A. Suh;citation_author=A. Nater;citation_author=R. Burri;citation_title=Whole-genome patterns of linkage disequilibrium across flycatcher populations clarify the causes and consequences of fine-scale recombination rate variation in birds.;citation_pages=4158-4172;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14197" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Kong;citation_author=J. Barnard;citation_author=D. F. Gudbjartsson;citation_author=G. Thorleifsson;citation_author=G. Jonsdottir;citation_title=Recombination rate and reproductive success in humans.;citation_pages=1203-1206;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15467721;citation_doi=10.1038/ng1445" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Kong;citation_author=G. Thorleifsson;citation_author=M. L. Frigge;citation_author=G. Masson;citation_author=D. F. Gudbjartsson;citation_title=Common and low-frequency variants associated with genome-wide recombination rate.;citation_pages=11-16;citation_volume=46;citation_year=2014;citation_pmid=24270358;citation_doi=10.1038/ng.2833" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Kong;citation_author=G. Thorleifsson;citation_author=H. Stefansson;citation_author=G. Masson;citation_author=A. Helgason;citation_title=Sequence variants in the RNF212 gene associate with genome-wide recombination rate.;citation_volume=319;citation_year=2008;citation_pmid=18239089;citation_doi=10.1126/science.1152422" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. Lenormand;citation_author=J. Dutheil;citation_title=Recombination difference between sexes: a role for haploid selection.;citation_pages=e63-e63;citation_volume=3;citation_year=2005;citation_issue=3;citation_pmid=15736976;citation_doi=10.1371/journal.pbio.0030063" />
<meta name="citation_reference" content="citation_journal_title=Biology of Reproduction;citation_journal_abbrev=Biol. Reprod.;citation_author=X.-M. Li;citation_author=C. Yu;citation_author=Z.-W. Wang;citation_author=Y.-L. Zhang;citation_author=X.-M. Liu;citation_title=WORLD OF REPRODUCTIVE BIOLOGY;citation_pages=1-1;citation_volume=89;citation_year=2013;citation_issue=1;citation_doi=10.1095/biolreprod.113.110478" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Ma;citation_author=J. R. O’Connell;citation_author=P. M. VanRaden;citation_author=B. Shen;citation_author=A. Padhi;citation_title=Cattle Sex-Specific Recombination and Genetic Control from a Large Pedigree Analysis.;citation_pages=e1005387;citation_volume=11;citation_year=2015;citation_pmid=26540184;citation_doi=10.1371/journal.pgen.1005387" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=V. Moskvina;citation_author=K. M. Schmidt;citation_title=On multiple-testing correction in genome-wide association studies.;citation_pages=567-573;citation_volume=32;citation_year=2008;citation_issue=6;citation_pmid=18425821;citation_doi=10.1002/gepi.20331" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res.;citation_author=H. Muller;citation_title=The relation of recombination to mutational advance.;citation_pages=2-9;citation_volume=1;citation_year=1964;citation_pmid=14195748;citation_doi=10.1016/0027-5107(64)90047-8" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=V. Muñoz-Fuentes;citation_author=M. Marcet-Ortega;citation_author=G. Alkorta-Aranburu;citation_author=C. Linde Forsberg;citation_author=J. M. Morrell;citation_title=Strong Artificial Selection in Domestic Mammals Did Not Result in an Increased Recombination Rate.;citation_volume=32;citation_year=2015;citation_pmid=25414125;citation_doi=10.1093/molbev/msu322" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Myers;citation_author=L. Bottolo;citation_author=C. Freeman;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A Fine-Scale Map of Recombination Rates and Hotspots Across the Human Genome;citation_pages=321-324;citation_volume=310;citation_year=2005;citation_issue=5746;citation_pmid=16224025;citation_doi=10.1126/science.1117196" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Nagamine;citation_author=R. Pong-Wong;citation_author=P. Navarro;citation_author=V. Vitart;citation_author=C. Hayward;citation_title=Localising Loci underlying Complex Trait Variation Using Regional Genomic Relationship Mapping.;citation_pages=e46501-e46501;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23077511;citation_doi=10.1371/journal.pone.0046501" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. O’Connell;citation_author=D. Gurdasani;citation_author=O. Delaneau;citation_author=N. Pirastu;citation_author=S. Ulivi;citation_title=A General Approach for Haplotype Phasing across the Full Spectrum of Relatedness.;citation_pages=e1004234;citation_volume=10;citation_year=2014;citation_pmid=24743097;citation_doi=10.1371/journal.pgen.1004234" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. P. Otto;citation_author=N. H. Barton;citation_title=Selection for recombination in small populations.;citation_pages=1921-1931;citation_volume=55;citation_year=2001;citation_issue=10;citation_pmid=11761054;citation_doi=10.1554/0014-3820(2001)055[1921:SFRISP]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. P. Otto;citation_author=T. Lenormand;citation_title=Resolving the paradox of sex and recombination.;citation_pages=252-261;citation_volume=3;citation_year=2002;citation_issue=4;citation_pmid=11967550;citation_doi=10.1038/nrg761" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Parisi;citation_author=M. J. McKay;citation_author=M. Molnar;citation_author=M. A. Thompson;citation_author=P. J. van der Spek;citation_title=Rec8p, a Meiotic Recombination and Sister Chromatid Cohesion Phosphoprotein of the Rad21p Family Conserved from Fission Yeast to Humans;citation_pages=3515-3528;citation_volume=19;citation_year=1999;citation_issue=5;citation_pmid=10207075;citation_doi=10.1128/MCB.19.5.3515" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Petit;citation_author=J.-M. Astruc;citation_author=J. Sarry;citation_author=L. Drouilhet;citation_author=S. Fabre;citation_title=Variation in Recombination Rate and Its Genetic Determinism in Sheep Populations.;citation_volume=207;citation_year=2017;citation_pmid=28978774;citation_doi=10.1534/genetics.117.300123" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. Qiao;citation_author=H. B. D. P. Rao;citation_author=Y. Yang;citation_author=J. H. Fong;citation_author=J. M. Cloutier;citation_title=Antagonistic roles of ubiquitin ligase HEI10 and SUMO ligase RNF212 regulate meiotic recombination.;citation_pages=194-199;citation_volume=46;citation_year=2014;citation_pmid=24390283;citation_doi=10.1038/ng.2858" />
<meta name="citation_reference" content="citation_journal_title=Immunogenetics;citation_author=P. Reinink;citation_author=I. Van Rhijn;citation_title=The bovine T cell receptor alpha/delta locus contains over 400 V genes and encodes V genes without CDR2.;citation_pages=541-549;citation_volume=61;citation_year=2009;citation_doi=10.1007/s00251-009-0384-9" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Reynolds;citation_author=H. Qiao;citation_author=Y. Yang;citation_author=J. K. Chen;citation_author=N. Jackson;citation_title=RNF212 is a dosage-sensitive regulator of crossing-over during mammalian meiosis.;citation_pages=269-278;citation_volume=45;citation_year=2013;citation_pmid=23396135;citation_doi=10.1038/ng.2541" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=L. Rönnegård;citation_author=S. E. McFarlane;citation_author=A. Husby;citation_author=T. Kawakami;citation_author=H. Ellegren;citation_title=Increasing the power of genome wide association studies in natural populations using repeated measures: evaluation and implementation.;citation_pages=792-799;citation_volume=7;citation_year=2016;citation_doi=10.1111/2041-210X.12535" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Sandor;citation_author=W. Li;citation_author=W. Coppieters;citation_author=T. Druet;citation_author=C. Charlier;citation_title=Genetic variants in REC8, RNF212, and PRDM9 influence male recombination in cattle.;citation_pages=e1002854-e1002854;citation_volume=8;citation_year=2012;citation_issue=7;citation_pmid=22844258;citation_doi=10.1371/journal.pgen.1002854" />
<meta name="citation_reference" content="citation_author=J.-H. Shin;citation_author=S. Blay;citation_author=B. McNeney;citation_author=J. Graham;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=J. Stapley;citation_author=P. G. D. Feulner;citation_author=S. E. Johnston;citation_author=A. W. Santure;citation_author=C. M. Smadja;citation_title=Variation in recombination frequency and distribution across Eukaryotes: patterns and processes.;citation_volume=372;citation_year=2017;citation_pmid=29109219;citation_doi=10.1098/rstb.2016.0455" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=L. Theodosiou;citation_author=W. O. McMillan;citation_author=O. Puebla;citation_title=Recombination in the eggs and sperm in a simultaneously hermaphroditic vertebrate.;citation_pages=20161821;citation_volume=283;citation_year=2016;citation_doi=10.1098/rspb.2016.1821" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Z. Wang;citation_author=B. Shen;citation_author=J. Jiang;citation_author=J. Li;citation_author=L. Ma;citation_title=Effect of sex, age and genetics on crossover interference in cattle.;citation_pages=37698;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep37698" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Yang;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=GCTA: a tool for genome-wide complex trait analysis.;citation_pages=76-82;citation_volume=88;citation_year=2011;citation_issue=1;citation_pmid=21167468;citation_doi=10.1016/j.ajhg.2010.11.011" />
<meta name="citation_reference" content="citation_journal_title=Cell Cycle;citation_author=S.-W. Yang;citation_author=C. Gao;citation_author=L. Chen;citation_author=Y.-L. Song;citation_author=J.-L. Zhu;citation_title=NEK9 regulates spindle organization and cell cycle progression during mouse oocyte meiosis and its location in early embryo mitosis.;citation_pages=4366-4377;citation_volume=11;citation_year=2012;citation_pmid=23159858;citation_doi=10.4161/cc.22690" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. R. Zerbino;citation_author=P. Achuthan;citation_author=W. Akanni;citation_author=M. Amode;citation_author=D. Barrell;citation_title=Ensembl 2018.;citation_pages=D754-D761;citation_volume=46;citation_year=2018;citation_pmid=29155950;citation_doi=10.1093/nar/gkx1098" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Genomic Region Containing REC8 and RNF212B Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (Cervus elaphus)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
<meta name="twitter:description" content="Recombination is a fundamental feature of sexual reproduction, ensuring proper disjunction, preventing mutation accumulation and generating new allelic combinations upon which selection can act. However it is also mutagenic, and breaks up favorable allelic combinations previously built up by selection. Identifying the genetic drivers of recombination rate variation is a key step in understanding the causes and consequences of this variation, how loci associated with recombination are evolving and how they affect the potential of a population to respond to selection. However, to date, few studies have examined the genetic architecture of recombination rate variation in natural populations. Here, we use pedigree data from ∼ 2,600 individuals genotyped at ∼ 38,000 SNPs to investigate the genetic architecture of individual autosomal recombination rate in a wild population of red deer ( Cervus elaphus ). Female red deer exhibited a higher mean and phenotypic variance in autosomal crossover counts (ACC). Animal models fitting genomic relatedness matrices showed that ACC was heritable in females (![Formula][1]&lt;/img&gt; = 0.12) but not in males. A regional heritability mapping approach showed that almost all heritable variation in female ACC was explained by a genomic region on deer linkage group 12 containing the candidate loci REC8 and RNF212B , with an additional region on linkage group 32 containing TOP2B approaching genome-wide significance. The REC8/RNF212B region and its paralogue RNF212 have been associated with recombination in cattle, mice, humans and sheep. Our findings suggest that mammalian recombination rates have a relatively conserved genetic architecture in both domesticated and wild systems, and provide a foundation for understanding the association between recombination loci and individual fitness within this population.

 [1]: /embed/mml-math-1.gif" />
<meta name="og-title" property="og:title" content="A Genomic Region Containing REC8 and RNF212B Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (Cervus elaphus)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/7/2265" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recombination is a fundamental feature of sexual reproduction, ensuring proper disjunction, preventing mutation accumulation and generating new allelic combinations upon which selection can act. However it is also mutagenic, and breaks up favorable allelic combinations previously built up by selection. Identifying the genetic drivers of recombination rate variation is a key step in understanding the causes and consequences of this variation, how loci associated with recombination are evolving and how they affect the potential of a population to respond to selection. However, to date, few studies have examined the genetic architecture of recombination rate variation in natural populations. Here, we use pedigree data from ∼ 2,600 individuals genotyped at ∼ 38,000 SNPs to investigate the genetic architecture of individual autosomal recombination rate in a wild population of red deer ( Cervus elaphus ). Female red deer exhibited a higher mean and phenotypic variance in autosomal crossover counts (ACC). Animal models fitting genomic relatedness matrices showed that ACC was heritable in females (![Formula][1]&lt;/img&gt; = 0.12) but not in males. A regional heritability mapping approach showed that almost all heritable variation in female ACC was explained by a genomic region on deer linkage group 12 containing the candidate loci REC8 and RNF212B , with an additional region on linkage group 32 containing TOP2B approaching genome-wide significance. The REC8/RNF212B region and its paralogue RNF212 have been associated with recombination in cattle, mice, humans and sheep. Our findings suggest that mammalian recombination rates have a relatively conserved genetic architecture in both domesticated and wild systems, and provide a foundation for understanding the association between recombination loci and individual fitness within this population.

 [1]: /embed/mml-math-1.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7.cover-source.jpg" />
    <title>A Genomic Region Containing REC8 and RNF212B Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (Cervus elaphus) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"quNrh3PTQDQRmhQLJs962wBUPMQFPYfb3IL1mdeAzU4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490130","apath":"\/ggg\/8\/7\/2265.atom","pisa":"ggg;8\/7\/2265","ac":{"\/ggg\/8\/7\/2265.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/7\/2265.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/7\/2265"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/7\/2265"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/7\/2265":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ipe+F5Yd6UPeC+Fr0fxlc6kR4aq\\\\\\\/s4YcUTwAn\\\\\\\/oTdlqZ4YV79l2abiFt7dphNnNISykjg8V7G7RYTk7uJaLkD5DNoDbCbsLyA3M1cxAwLHZUPK+wNW48SzE24++bH1FiRaXahOqzzmDcgnd2J4m82ACMS5R\\\\\\\/cil8t\\\\\\\/37nmrgljYscyuqKegFDz3IPFi++hAPlnNbFC0EKvnlcDVz4ZCXip8GtCSVIyrSOQsgRuyPK5aCDyz18fz9EK7taMVljRMfg+X4+Ulde835KgjHGArZy6M+E1uwB3zjXMVnWn8mRfUPiBDvSn4vq\\\\\\\/ttfYKsJjUqmd5KFueNy3tQgpHpqTcas8xTs+4UW1d3Jd0KRizEoSRh9PlFKIbtl6j7kfNRCITX71jsX23VmkN9\\\\\\\/6C1F+xNYCVyNlY9ckJibTAK6LUo8jMpOltCScKODNz+6idTlZ4JHn96N0gpL3WSBhVK3VTb0bCtKJG7\\\\\\\/xztQrPy9cUZMxyI2Fqu7AexaGHs\\\\\\\/IMNxB60TIi0Jm+Re6EIN70qeCxRyiQRrd7ShQbMwgiin0b9eeCdnefUAVMnjKpFxPpbj9T8gz419E8CNbMqpxW7BCodB2emZQXeEqMxjwmEondL4iE44SRZnNY1m36444M5lrmtzixropFYc3mG3vhqgjPv0eGTY8s0rpy6SnQUxrHeqLmmnzgsXGLB5eA3bB5unNz5bxeSt29WMB5VoEIzKyGLIRKLynNytIl2c\\\\\\\/y\\\\\\\/0BpNNMAVZOc8j3GMxegdWZY9GtuIIDN05uyj2rFAS1ufHF+sGw63npckzID39t+kWH9Rapg2hYy4Ziq9YRKR4ujK40VKchL9cYlQ+4ObMtd7l+J99aLfiHE8nC\\\\\\\/XOkbgfqTwhVML539EiqHiylQTtYbv0Im1gkX695uKyy\\\\\\\/t36gizEDjpJ9K8lKC\\\\\\\/H1qCOijhdwFWFMASjJd0O8rMh+IAUiqLLXtALf0v3Kjo9wA3GrLuYMsE1J3vrbC\\\\\\\/EpD2Csw82u\\\\\\\/dhHtfx2I8l3SQwBZ8z9n5Sm+5ryyYArdyIOP+K5u7e1TaHykwZuR4v7JmeX886qMePS2ZzvgvwSJivzAE97gYH+FKnLk+E3yuZHnsJ09YcGAc0BqzbATv7mX9oCfBERwuhMHxwWjJnDcB8Sir0c1ePAVYJavxZQs4CgiaWQ0MKMZMdvs9MT0jZnoYsdodWBrfScFkiJZVbnIqdik\\\\\\\/iX7fjmOtJYOf3tCWNHtawfJ5rLwejTH8fEYTGOzwtoSkS4A3KjnALWVDl2faw\\\\\\\/s4q5I5Uta\\\u0022,\\\u0022iv\\\u0022:\\\u0022AYoI2vcTCHKBrWAgeX6G5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ed7d399ad83ca396335d8d03f0ce768d9b1fe384df2d08ddd0f0b08a61e0f6bc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022s7BbXBrjpzH5adTIS2dwDM0mxmcm7LKj3LJr0blQzCtdT3830cMM7FLWHmpG7EHfgkTuSph\\\\\\\/5w9+XCdWGeZVoIWbHsSddc9kQY+kej7M7J9TOczbxKhAVJULpbJjltwIKun88cOX6izWCVF3wyIJTsYKpOgrys2OLlDoqW7fc78f\\\\\\\/zhjHa+SR3uovRkiDFHVQzpUQhiq2i0xG\\\\\\\/G0mtMmlAbciYID4pUa5fb4NhXrbW7gidALTP92F0Dj1FUQJwgFKsYF2tTFt1dF0Ch\\\\\\\/IcUdcDfbHyrMzoWBwQO6ZoG5jamTmEhEe76n1FzqC3Dy7IY\\\\\\\/w36cTvfgD1GHwluTVP5AZ4XAtDrGnXIeM2t4HbK\\\\\\\/u\\\\\\\/dXFz19Exigrsf3HVYahEZy\\\\\\\/k0iqXHtGUXaY1O6IkSFsCyqYMY774lkHEm77etRe4SIEPO37RDaE5C68Up6iA4YtqEkFenw6JX4mBI\\\\\\\/Us+eacFK\\\\\\\/WFpTVaMqMXYJGycIW19\\\\\\\/fHZSyF\\\\\\\/k5UzTT4a2JfNCrJthVJKr6EUAeVpcK+WOSTLS+tk63j8+yFyBWesTXMyViIx6JQtU8xW7vSWBC1kuMzq+6DvRqCHb2LtjcecI+pYnsvFar+TooMqXQihTjgXN8KPNA+bbZ8VZYFfGLufNbRMbnpm18OZ4OSlZt3Bg\\\\\\\/\\\\\\\/ivIOBgYMVTHz\\\\\\\/tN4Mj0eVNZGzyMkMfy8pV\\\\\\\/NS3S+LB\\\\\\\/\\\\\\\/+4WjRzSiioijMwHFJw7aCbYpTYcH9wvJaNTI7Czw4\\\\\\\/G2d9TA+Xbg+YQQL+KNjgayZiPeGhzs5ijs8XaoDit2Wc5Zvqi0\\\\\\\/NzM6U56tBGZ54p1gT6Kf1F9LsTut2WmFj+tk26QfLjvqET8ARJppmTSDEKqA8cenyIMEz71lhevxcHADPN3STtDaLQjHPidRFYg1XBTyVgWe+mlcRZ+3KgHsXBHw4I826QbuhXmjgDIr5Wi8wdPr8GQ4qG6GpvF8okwxP6TfEHVWwpg1VnJ7yCYTer11cW++ehnt+isfrOOUr\\\\\\\/atYT+0V7ygfbi\\\\\\\/rGlG5ryjEOyrfNYBD6wAaSwIkfDmOVcYDrTZdbYngfOVlHc\\\\\\\/3KvoJpzv36iCz6qzJc9T8HLlvwjG4ZxOBOhiwTEZnf9W5bvB0rzfTxXCIT8UC\\\\\\\/p9j9aH3sTtYWSXtTHHMG+3HQVq0ovLa6tVpRpsWmKs6jDAK1v3+yCMBTU6ryMt0pFWkIYJrnWAN55m27AdyppuPGzitQMQGw87XGrCaIbAgqg+OP5fcp+yiHnfUf5j5uboMicfeORx6ULuMWIaJbwQDmej1gRuKhDIlhB7PqXl1541QGOhhPdx\\\\\\\/n3JMV0RKhtyww44X7CAw8PKReCQgNC9Z1ahrBBoJO1W+2Y7lusTz+o8ev2P0rQe+Maaa6WZjfsHgZ48myUOgCATwgG50gPtKUoUOSsh8lsvc9\\\\\\\/ouOyDmnaS7Mqz+L+YCwxleTzileg\\\\\\\/hfuoww7P7iDM2+7eJYq7I3JmZbgLj+59oHeCWtkiLOJdYROpLP4uFehew9rI4pIW07yyvvNOMdeL6UhaTr74c5zGf3zmBBWVGnQZDb7WJ\\\\\\\/R4M7n1qQxfX3K61ukHNzIzvJbDGrzW3gf3iLocX2Zd1M3bkMG2Z2mvVkNhncn5SKneuKwQUerTxRo\\\\\\\/eqKfkaM6t+Kzthj\\\\\\\/5jQc5CuM+uWTV7G1t5AqVZaXyB85FbJEQPbAw5Sj8svlhJacaUSjf9utGVDx3k7RC+pW6ZTKr3zxUA7QcaqJdEglmKqOz9RERViDxF\\\\\\\/\\\\\\\/Z5uIh3\\\\\\\/SYcdkt9hPP4IMxCqDRoXKs3DoO\\\\\\\/NrPb5acZ5Cc030UsGwl7CFPDAiUq1ScmVywKz\\\\\\\/erAlDJUGD5eoygC7X0hDNXQli80SzliLSk8H+Fvsf+9UlV8s8K4FyZVlnFLNYcFrYpQ6XpbM\\\\\\\/qtUkL4jU2jBcrjkO3FGJcV6Q84lGDsKv2EbpO7WPejF9Jne3830I56T9w3BzUliAVRfZGa7drl9CM1ZbTwtVIiicuVNFQ9ExcX3QCtl489EJ0ztBdf6y9kCNuXV\\\\\\\/NNgu4NwWidu7mMF6lq6SS99oa4DqdR6vpZpzCj3GqbJDllERKebUWiSC9cdSB\\\\\\\/02PqwweG9lEfUxDeiytVdz9P\\\\\\\/3lvdnAw1ppC0wj8Wg2lTEfjZQvUUw9XA5eGZnmH74DUQbrZgxdHerY2n4du44CrPb10bvYGZxdjLDZUjTabxcCgO2+s4RgSSgukHDApFS1bRnUlZUhYJ\\\\\\\/dEuyZeH17OFu2St1tXIMI1HOyl7I6fQrS\\\\\\\/f8Ipp6fzfzm568xQnNpibrnXZZOh8V4lCKosSBFa\\\\\\\/5odhSvHcIoozz8f7sr8LWsgkSuovJZfAPmEaNUJG98uw2rUotVHvGQ7EenkzeC9asSL24cL\\\\\\\/yQPLeeU0ij5G8P+zLXuBNUGIJ9bz3sobgk+XOsY93iMo89Slm9gf7wt5MEwg6dV+PKGMA1pyMHK3yuuwHvk8mhsMxrJEHhuF30WvOnM9+dqnf1apYaA7L5\\\\\\\/pU5i\\\\\\\/AS4CnHaVhIivTPNmt6CU85nXpzAMD4JkAd9UJtdTXINvddRO8wpb5AmKYHIDku1ND0fFG9Gk\\\\\\\/6wWDEbEt19Dlry9iSQc8ThGn3b1G1XZ1cpj5pGYIJEJ8+SloR1glbV7sd8Ze9brY1PBxFdlYKDGsacp0+nPGDECaANEu9gyKNjG\\\\\\\/Lra8tyAFvfBrJbLQGTfR\\\\\\\/7aV5FXLLMXAu9Jitc85KyqxVdrPnXRKhcO730ImfCwus\\\\\\\/BpodvL\\\\\\\/vclvI4sBobckJHTFwwZyGp5CA\\\\\\\/zVysM43+tozCiGtmuvLpSkw\\\\\\\/r6oanS\\\\\\\/VrR9PA4am6h0MuUC8C2QZnBET4lCg26j2BgtnR6Gon2jcZJmOtXb\\\\\\\/xAKJ1db324Lt2j71lI7HvUfw4tbbaRLb2HdowpcgOrO4AzQrkU0xtbiCLbN87dkJvICMIxuUQP7cjdGZbHngTOiXoIPvO2ztT97cJw2EY\\\\\\\/nRzWj7gFPovh\\\\\\\/w0IdL8Hpo8xRS8gG4o2aebNnUUhJGr\\\\\\\/ZJhiCSawyhGiknqskkOoX00ZaUfPoX4EvkAX7xGg7b0ogR7uCAQbqGEwNEPQ8birgTXFVeMJZXErFAcMZfS\\\\\\\/3+yzzJ\\\\\\\/BxMaltcIfIk5+KdTp9cfWObjPZHYiwhDA6UJLsq6JUg+JzrcMrc7c57pr0WKULmY5N02N2DaQ8T\\\\\\\/QYpaisZUW9QyqLl2Z6Nb4TenytZrJv6zCPzjMzbvyitmB76l0MRWq96GB4NreUZhu8nFfyjshnHgfTN+L4U8gQ3Nn4fr8n\\\\\\\/B9djXmCcvgEMMUNj1mikI8nU2Km7du7ZolOASYpLvsNtxaUtvYFPmb6G2hYFOdh98wUybxmT9I5Ra3NGXmgPWdNkZ7vmiv2gxmKuBUaCnDALaxpoYBPFJjk6zP+Ro4nZ8HPpCYkMCspg9XbjF0o+uEqGeQDoxMdYWOfmg1wP2Qzqj3\\\\\\\/D8FTJ4PaSaFS9fErViRdaqDfBvnCjhMZqFpw+08KbBQp09R1thpxZ1RrfaV\\\\\\\/f2GM1PYTiEvYs3loYaWxlWNV7JxQBk0iQ9LCrJRNstVwZ6KKhqLzC7F3fvkf5ti1p035BQ85J\\\\\\\/Wib+zMOobhUsuOEiyGtCidSa9PPQ3lny7qwGNZViipBuN7ImnRTV7e+6VGTtontFshnxCRngC9JGrP84BECzx6ZhBNRQ2eBTPTw3L9xgbhvE2987TM7ZuJcdsMXOhlszrdEolodmN+w8Q+p8n\\\\\\\/PivP0XmxMGgNCKR4B7EgPU4Oksw31jDXVVIaGq0L4PJd2esL8crVuWz1b4mx1904T3fXRmlojyxn2f0fi9uS\\\\\\\/+6M4z\\\\\\\/irH82JdSwJr\\\\\\\/sKEa67VVWXvOtFlaw1LKB8oxB9lXQNluURr6x+A3dHPJR\\\\\\\/2BqF5f3jnnHfauefQmnkGqfLyQldHCvaRUBJruTgDkgHJzv9HE9b0NzRzX4OOsNJwIHNRMoA6UrcrO9vKqlJmWg8C+kXpuU62jh5OvAd9FO2FlASBVh+AbembsZ+bwsJdCknKiJoEWnXEN1YRJAhnG2c+W5nmgGcCg6wlzz5s0pPDdMsO7LYLQfqkVeXcrPxMhnyTl8ljtyb9VORK+rI5Ik4iSdTaAid2Uj3oQsFAxMMepBZ+xZwpbz1pX6QDdBZ9Qbucz0qddOJGHvqoVRLG9PJzvsfqleb9yn4oBIPlaMgnYWSWnDe0OALs34SMaZ\\\\\\\/yLk5xsJsk2XBheh64zreKuqz\\\\\\\/yIDQVvlQ+sZdNy9LhmG\\\\\\\/TkE9nPTs4osAvchEzrNfNuUBiWGqykCFkA5vYqzX67AU1IS7j\\\\\\\/cHOc6NU8xHhKu2TyZ57aLbqgrC\\\\\\\/PIdrppEsAf\\\\\\\/b3Sil+5Grzv7z\\\\\\\/XQ0ju+cQODv5BS3+kJeIc92YmrbiSl8jOhYriRUPCjGg47Brl+IwcyiM0jT+cnHaTOX6SsqgHTyEOgIOAKBZXSn7cKEKo7d2SNEYsFYewg6aUKoceWNQrqKFsH\\\\\\\/klIdb501FMSmmvoNeQGKtedVrpnzg5xTyWMtDYoP9yp9NXU4DY2qEa4Uh+9jUsxDsmSCg9\\\\\\\/JDmlA8kbQRzoXJiuRKuS3ic6GVHQ96sBXEhbuB69To+dqAO0M2hjdESkcgYCL\\\\\\\/WIMeW2guGGVKi\\\\\\\/N+8YvFufBudavcrBBnWrYtgqwlsFItb0EX\\\\\\\/LC+\\\\\\\/8P3qUxBw7gF8gkQLyZZXW6wby6qGcMJuXKh5Ly0PmsH9DKt6gA9Bkn3CfVwJoWQoi8WE1HJPVM7rkZbVIoizzG3UJZuM0YzFL35Bdsyq2\\\\\\\/duPjvc+KuviRlPyfKUrIORnndCfce46p0y4o8VWzAwAZRpzFvN4j4MtacSpmtdgkHLPuCNgqcFZr\\\\\\\/Nhue2ch5cw+OjTV8PfqnKpcmSsu1HYauJA7\\\\\\\/DemWSNp6945vl+G+1cIKzED6\\\\\\\/W81\\\\\\\/Q9d2R4wfUc1PnOcyGs\\\\\\\/\\\\\\\/3DhEXIa\\\\\\\/JDTniPPQ73Kc0fn8u+AK2JJriCTGtsUz\\\\\\\/gjr0AXXEHlxnfuEbTUp29yj8Yv6DmvxW52Fq\\\\\\\/IMGIg14daNrg7hONtDkzmH7u\\\\\\\/1tAB21LYdlol1ZdZ2vbLyxK4DyevQY4A4+zXEvTh34S4NUPKZvEOtHpMebQcND52ip\\\\\\\/qFza+N5KSSbTxvc7bEn811CPfvqFefLPKWaAP1cBKA5iQxIgNB0jhBjUAXLtWXLVOI73h3ioFLnPVeJ0GZHgGX3Rwo1h+h225iI5+ua+oabyA+48bsh5FesvrICTVbogwXvDpjSidwp0abYwMkfuY3r60nndbhv1QQY5UoPc+2H7M8ROTP6WOjaRaYh9V+Qkjw5Fa5VHRuxL6e0Ig3sdImLT5E5IzPKCIbWao0mHqxzeKRclQnq0SKFjdl5S3CxpLL9u+kOqS6V6VZjesTPuQ4PSHiV9fIKS0vxoBfJzzMee\\\\\\\/ThdsIkwjznFWZkuatPQk8RHJq2QsL4Vu8HkArIeS5qB4XO3wOoWe4i97w1vdWJBVy5khud38hiRowC1Vsjf5YvgEofRe1fHa8BttctfhLwSgZ5aFNRBLgrfIRi5KiKI3gc71pkUK256fOVLNdJ6voz9+AMRYaKKGcovluDsd5nxWVWoLBCeNq438zPXJA21K2IAvODQWgVyO9y7uoJHN4P3lQPIiO0lj7GYVTpA21mVgoFU3CWugqg0rQChkdDUECD9eQeqO8h1CRyhwo3DYrDvErQaZKpcVlDbXjd5IGxJyJOLGIzrZQgphRciz\\\\\\\/D80jKzEfswPhkLtqCTKvR1nbBP\\\\\\\/sSVogwjxSA8Q0\\\\\\\/tbrUeDzqN2WkUs6xf49AZ6NyvEu\\\\\\\/inHaFCNofIp5pPVut6F9nMK47UDKakiBJOv0DLuqqn2suMwhciy5bPkiuR0yBA+O7iIcon1xq5x7Fxc2WBSCF80ewAshLa16gsT+w5+dwlFIMZaeWRR\\\\\\\/EnKMWoDfY3ZmTs4YnATsjSb7ZWHFW5YLd1K9vMNloyNCZfn9xkn9dHKQtkI4UChmUxS3901rTzzdNvwiQ4I5ahQwSpkutoisMrzOTlx0oOw8NPvaOMs96MIYFvPkFdpsuCWOhfhE8\\\\\\\/jNWa\\\\\\\/6mUeYro0sBV9CfJpe1gMlQcbEkFfMYaHirUPS8BgWyDbWjxEU6ov5B03x0CjDhGbx+vFR+3XmZbOBj4oRWQr\\\\\\\/s6JxyQYzNzWOkWZ1ki4h9pBCQD0qlI4uCQsOpuhy\\\\\\\/qnRvIBwPsoLp1F6kqBsaG9ekPppHZgTud1bKEWgyNxbJHf7AlzL7RhW1GcCegvvlA\\\\\\\/DlrhI8ZrZmSlpLXm2uQqt+qZrVNqp\\\\\\\/Jyy2jhj40eCefnGTC7fGgyb9B9OHF6JAoNJNtGeODGu4\\\\\\\/NrKLRKQ1qNY1cTjGYdQ2d6TZ+HfTqKVgZivDWBXZkWN\\\\\\\/ZUeqW1GA5hscLOaPD\\\\\\\/EHu0ZLGhmZt5jrhcC6HQaq1YVl5JIiWEKzft2f9ES81qcmDP6dJSrj8yVQscaQjbgE8sbuGhYhpC8dwWbzItznqvN2gVS5ZBC9rt0h3dPTlotVnVg1O8n58QnHmfKjpIOfkF0jf+hjKt9wBd4vHeG8fDGXSWVyghGHhAkO\\\\\\\/ebCd1DZZ+ZLhe6tT\\\\\\\/y1smj73cr1aiIvM8pkp6D2hJ4OZbL1B+skGEwqf8N9M2JBWQBZ\\\\\\\/z+ZQcrZ9gfH5LWlOu36twXoKHDcKAPChySdvRBbA4gFBizmdoNGbDAPbxfM4MLrNBstuR\\\\\\\/P9lypQWFIA1r7+yaa\\\\\\\/DRzlB1docPKXrBBCLDt75EAQPXdtrz3PUfxoiJbuPmr9YdEKDXfoFNGeyrRuUwlEOowL4phE7kRktdd36hD2GxnszA7QnkeyWXxbMss1kADN3JBkIgY7IyrjwKxDNxorM+zbH\\\\\\\/NhSOvEFJTCiX0X\\\\\\\/m9Br6WgFKIyouQkyD6gUC4HPz\\\\\\\/Qf97GFHlPl3mUOKcPV06ovTDQJUa+rSIBtOXMnulOPDL5T4Ua3AXe2AACuA7pD+c+iN13SRpLrUbwNFjOs4GgQmKj+5ca\\\\\\\/VkjIg0+t0fy6gW6poInizR32HWaDg8jgUH+gNWa5Ck8uMxiGtR1QcXw9bRMOs2CGFHLo+tPJOj0wpHlUqYMOEzndA55ZDwdm0SkEf155Uj1UiryQobs47nr1HaFkUBknFJwNHiPOMgHZ1+9K3J+ABLT0Y6NlnLqkrFWxn7JKnT+TR9kXvnwG5uyxtw3JPb\\\\\\\/6HSQtF8OmuZT0hKtJ8HHiLxDZfYsQp4KFFKw9zH4\\\\\\\/Jkb9VmyETMWlrTTfemmJBb7ZHpaLyQDaK9NGvrVe9nek8bn4rAIa8kNKGkrcaNeWZ2MFnFO3+PLqm52nr50fHsNVCOVbrGyXhrNwiHgGJgvd9E4BWnLbbKQknMqYdlv7dZh0qOyVmWLPC1kE56ADZD8mfonRZmU\\\\\\\/iei8mOTGe7T7LIAmzZb+LzyUFkDWMy6pdWT6X3PMV2Zvqz8+sDQD+INqFCP8mI47dWqWgah6QuKcjSAaZSveOSNbUVNqTVpOnIcrKoatMj2ltg5idaNgSJWi+WWZytjHVoy3dwtloJZvykhEF7fQGWECAtxX7EQ+TkU8BvG4XRrgOzh+b6j6zuNvtohto9KaxXTMvEZDw2+tC1dYp412WqobqTIqYH5jr\\\\\\\/5GsGR6iWuXbwwCWP8i\\\\\\\/IHOnCERtqof0sbTVH6fJQr4WoHt6Xl75fgTyJFB5LyK9xp61MUqHXpkYvA6UfMd7b6HQRALc5SpeOoFmNFzFg06Uej4tc90W1En7btf9PriGuNVYp\\\\\\\/mT36pEWezaDzLWWzTO3Ts284W1+MBoV\\\\\\\/Je8jcpfpQd4tcyEtDwwqWgbDSy85g4CkNQquyZiNwKuOak+Go+RQ7DznZmGpZXqRuKHCQdQl07+1OGTvy4qRYbW1d2jDptQqXRpQB7rJFfJyVWXto+xfBgiJO4SdVvEVR5FhNF50TlPsXDxuhFYeHDSAGwtd6+0KHDt7KY44XP+uNMiXmtEhXoHB628e9Nm\\\\\\\/vUDK\\\\\\\/DyKKR3AwlgoGW7qzso1596O5WDsDaYsviA1AeG\\\\\\\/H8\\\\\\\/f58+amrlUjZMlCYO2tB5pv36pe03ojrSC6BdkIxJniSuL8r67htm9ChqrSEz\\\\\\\/7kchx5mz2DAngKolNG6CrKrvjjQySYwU76mTOXfW\\\\\\\/tsV\\\\\\\/9gtT61NVT9fgcjBMfRLK536SzcorEBZy4uOhM6MhUy8tpcFrjPIXpkrP9bsJ5DkH9BCK1AueJ7V6mZa4oJ5GMxr6ILOnKLCI60tDtzWNbmmovFtJ4KCEjenqSqjpS8GHyjaDzoe3BZip2j7qqZFTfF9zPXHl1dnsjoqZl8qikXQ7AI+yxlM\\\\\\\/FXCsgc8UkQdTKBX8p0xSikqQW+rreUT+qj7WwZcMEXDOiC+SU+mdqWzIHwWg7878gYGRLL+xcw0pn4Nifs1wdcjkp7MmighBB7SARzYBumV9hVrO0Nbl9P6vjx7fN26BKEUGNujPA18jqe2k5mmfIQ64H2xpdG7R5txrXUwUT\\\\\\\/3QRb0LiAhspF7c5xt7lpKSbqXVNbgteZc6te6jn40Uabib7\\\\\\\/7a7qNAcYKCMSCsW+rXn0MND+EXRrPLvYTewp0B4EQgKkZEsb5QcpKnvbLv3JgPR4qeiOk5g7QHfdRZIzQOheZvD1yL210QrxDgCF1Z0yHjKfvGsDWtymVLUrTeZfjePD3O7NyoB6SEaJMIf3T1r067j7xKFlABrI+MKnWDNlaWf6xpmSQIisahdQang\\\\\\\/sP9iBTLSLit2sSTztOKTq5+f9jT9Qou6rwSWsL1dn4E+j45DiXUEwQJqnFMnuq07\\\\\\\/Ny5rwEAoHNn8OZBEvPaj2MGhyxbNWdjSkVL7F3szpwLZEF0quNHVKzUOSPiCRTC2t3RE9hSS5WTiobWOGAKdilGuquLCp6iyu7WgXkdMr06KtMHxMPOCG89wF935tAZ+e\\\\\\\/aPMMPmazPIVmb6HNtbIjgkn9wKpkUVsYVwWiyw54zP3b59n\\\\\\\/NF4AfvUcWYxhO3888efPaUSjBHlic4m4HrmXQNxN\\\\\\\/znpMZWBjWC24bPX16G+SXZ+09hWR0reixR8B7+6fLY2S4tJCM1jA\\\\\\\/ToIjJdBTRAIQJ1KCzwfqQKbMffvjAt\\\\\\\/eiGjhNIrPEQWgZaqAvMBvd71Bopig+B7j0BUdFe0Dqtl6nkmznBQbayrUuFYquTRsFUnk2tUZvnMdxfXdnMbqLRSHNx4v3hky0izONalh5zftP06L72HIxUOgk45SFEdonzHmS+BTNQIry56CSpQl43tHFq+GSpQBY+9WyAxXex8oGNlc7otqk96VJcN9tciG1vRpL3HFfOj8dYEjclqOh6KKB3nFDjgygI7CsrnySyMc2QRA9aMwDBScqPiM6IN0z3Yjbg3NDBak4k0NQArhSoxOD7UiSUOcUb31uByUSHzr+\\\\\\\/jHIvujywNx2h6pxvdtFGSV\\\\\\\/rT9MwVn2Zfx0lFYFYjX6NzCERsROwuhwslm0O1NrLpGPoGsp3Et3H4cdED98BxhGJAfMlY44BVnwNBQJLHuji4CcFdzyGAP8l4xeOaOIGyLRcbmcfXVlgf84XRsBbunR1YgmtQozGHKRxiUvwyQPFe3Z+pszUd7TWepPlMG0SqQH\\\\\\\/amfQLhM63fGR3YHgdGGU9J5gI82JRjRa9Z3nFVecy7NXEawfp5nUCWmDTM8uu0kVaEMwK5KqwuyggrzU9ol7VifgGmza29v0kDOIEbRQzZlYXiFsd4DUUf\\\\\\\/Xvshr1CB8l3Gz0YLAClsKpytzgu89VhFfGVJHdn+PP2WW6tG6YsgEIVSoFVbHJC\\\\\\\/+X4sckSlzFE3S+NAuJKEab3qIpXEWeG9gYES9p1t5lZv0xyiG1QD\\\\\\\/WflK16+rZB3gl9xnNidXfu8YOuOE4WKehKrF1yZIwl9TUGjUdSV8kTwxml8a4niswdrHXF3fB5DFHyuRO0gQNGulqDuVIYEfxJUoE4B+eOhTA3anmPoVTjm0GLGkx3CLuqLn4HyzN8lOPsOVAWKtzvnhz+vW0\\\\\\\/YUu1mT\\\\\\\/r3Zq9w\\\\\\\/La5iqg+aI1eIfixHBUyAx1pDkkQzVPUqPAThfDpHdyrt+N9TqdJv8oDowVJ4IV+VXhtexTyYlRJnY65FVfWxgos5su3jsFFCi56YoBIbuAalfaRvylagOa+fZNjLQbwfYmOxOueDCPRKaro76pz5PNL89qTVmocqrnXe5VMxAIt\\\\\\\/3OMjc\\\\\\\/R2GHe4kKIh2ehpp7FSvwCBjo2j79erGA6xBB3ayV5mDXcJgLvUv+T+lqq9nxFxA+H7BduU68AwkTmu4ICw4wMy4KWq7aXz2Ntx69Fx+G1Wk853qjubTV0CIFL3vq9xpVXYfIWNLIA58ol9CQGToPeaOSYJUdEvFeZ6qr82NJ6DwzGKpoKz8x+CQLnOv0x\\\\\\\/0fnXpa\\\\\\\/HIPE0R\\\\\\\/qw9haEZzuNrQ7nFz+f7YgrYcKAB8mX9AHUFA54wqV2RTn1r7y2PdtY5qQR5KplGKqwLA4v9J9vNmi3kke6HFsrLR0535PP4G9mOdPvEix69\\\\\\\/0nCXH\\\\\\\/X9f06SqCmc1vKumWAcWe05\\\\\\\/pA84NyZQ+Ygs7z2WuKHUAN\\\\\\\/4xMTTBj4nOr\\\\\\\/R8bXZLYFas8o8NvL4DTIQbL6n+Hv0TibmzIRWpicOmQStbS3ELgASDgviY\\\\\\\/bGNiSoFXz3tQiAdTm8KPRS8\\\\\\\/7PfddK+UX71iWmog\\\\\\\/py10TLnYrG4YIXcKB\\\\\\\/3aDNwW5iUWLiR4rzzIfgnbobMILyJVYavzaZh+yk6ghhgBiRI9tGI8lRIe\\\\\\\/Ah7svLsEcE1tvSDr2jqKfHNRp5jMihvftgpKFXEYFDSf3U0UO6dZyPV3LpYCLrCZGntclRGlY80NL\\\\\\\/Q7N1FdZlM9FwudLGl5C0Okv+OWfbAUJEO+ihDRjhFp\\\\\\\/SQB+sUMkOB3TLYfB9wpvxcuSTRlNcHkycAm1NQ90TpA7MdJ+O7Zsy5eIx\\\\\\\/brCobwukU6Kp1GGijucQYMdDFe0F\\\\\\\/s3J2E8QGuwBzaO4C+MvJ1qx\\\\\\\/e4SfHXPK5i1BJu\\\\\\\/Et9turAdk953sJ9xcmgQ41MdBFeoPEdtrI97SktEBAvgREsZEDlXIzD9xcbfVYLJ6mRXBEfKhhp8S8SwQgN9v9C8waxRA70ySLoxJOoq2EUULzEQSj1hzMED+jTR2TYZDkJD+ouApqQSOPbBgAnQAvQoEmfEWUi5RdfxhnKnNefNYWvhZNwhlvQDLH6oNyaFqJA0LXlRe1\\\\\\\/eXsT3ok040AvuwF6LgFkgH5AOvnYesTg4FJ16eposf7aI7Gl747ypiaNqoB5rbKf9WYUpLiYNJlrILav\\\\\\\/4PvKbxOZ7C9WDrbGVN31jaThdVz859CFfRX5zYrrg6IlzfilaCqEkBHrwzhpgG+8WyxMCFHuPWZykkOKR8gFz3Bw48odhMJLD+ReHwkBGLVrneFKNXxy4U5P+Q57C8weOgp6\\\\\\\/FeQwPCcnFDkqTv9zUoleTk34A6GypSa51nrQOaLeRzaxibJKfgVQqk5AYlq6Tv5NnuMQZJrOsddTAjf9dSBbpdBclfYMit+pCzt2sYSJDEHQRixHrLdnoSIqsUKu8eTOdKrBrMxW8E0j9LtO9riKDxJmpKftOTP+yyKKhr+9r+8OSfhGJMWI+pVf0z6OpNbgmlbXES33a+5HJaBL3ATMGtE77AKZqe6sR+PfIYAgJiR1DcIuRuc8+RlRan6NTogfxdsa2zYgTdR2ugutcd4sD4dB6oEmO3l7DL82BHx\\\\\\\/q9hwIKLw8yO306sUR3DfSUfU65aCyMnWzGAXMWba8o7VBJ9ScSAiNBDvbIDtuCdEA9J77QOckTVpVK5qEk6uPy2FoEDzG+YGiYOiT5FMLadxMi9vq+GlA7zUdmGVEUJVal3YtohPiFJgmupF7Y6s3ii3sbQkfdPuqQCxD0XnhKPSJtACuePC5kYeOWihXPmH7FoXbxpmTDTZyPVDWC0XHOBAzTK8r0kO8yJVL6AyIin+B28uChONXkhb1Uli7KusN\\\\\\\/Xzy94zd8blBE\\\\\\\/CWfXCiBd09rrlcUK8QuRan21MWKkrsP4jlY592hOBXoQ3F5spMZhx2qmxcrY0fKc1DZSN5rfGKh1x5zOmdhiI8xsU1FpBx3TqreANPaALVsNP0Z49LFgagmH84gCTJmtJEW95FK3nacFsbqHFrjH8JO6IPXxd85jOeg7Ta9+f33erJvqQlv1iKG0mPRPj86lFVhIOYBN+Oc4MAHnRVWn3rjsdAReXisvrGSbMtbQQpZKEGJsZb2rVMIZADYI6oaFe5Vn6yucH3tVsNDQYo9r5xErs9jIgfbX8MufUw5R+3xCVkFUZn2N3u5ppU7daIcwQPWAkJUKIPkrpsNnkyL5SBEdisd\\\\\\\/zF+e4\\\\\\\/le7atHe1SEpF6egs44f0j\\\\\\\/kctD1VWAh6PkYEkcBs3U1fZ+\\\\\\\/MnJZkuaXf44FYy4p17JuowgloMo10Nprd2IMZhMsb4+QiUrsYRZ5FgPDHeyinTkUzWVBqMtbvJQ+EuY62TOBPz1hhAVytYRLijZdQ6PealAZCh6H0HD9YQ2UQBTfgToeJxIMaKcZRWKMqJzLT54EbmicgKS0vjirWFwgUb9W7yelV0WPD8s08ljJjk8yKmSNohs6ckBPlT+wVrM5JO7iU55tPF0owSOKEZzD2c4D3I627YNB6UMYdWMTrYXBk3hg8H2qtIzqDAsIND4PJ9LxnoFUTJm8nc07vKDjyLP+RJkZZxmGg2A5dmG7z7IsKE9fQBCyKxEU9laKKEtTkr\\\\\\\/KOCXsFOuLmiLg0gHXNvMFTWbpUz8YFgA8XitH0MB2QGptsYe00p\\\\\\\/uT5KIOi+cGRfO97muC\\\\\\\/UoArIffILIomthCden2dCZXukoRJSdaqix633BI3LOHGgOuXvnqO3+jIFtONWTaxj2COm1tSFHkQvCI9JnyZodkCiFoP8dCK8hqSJ\\\\\\\/SvxSaRev+GlL0t00WU\\\\\\\/oU0+vGDgDkbe6CRLu73uU5sCpDvjwAMxQkWFZBLz313SWMNQUHg+hK040tYKbKZ7YDm2ndslJA79nFvUsmNL7Ar70GuO2yobHWCTCvjNda64ZLycewDpYdKYMmhGWBZAphY\\\\\\\/6b3Wmjfq2oyPVnPJm1nvJoKCFj\\\\\\\/eKt\\\\\\\/nwVCjzJqY0iUzVwsQKQVQIc1QSub0QNnSC51ak6TsRDbEmrIlU71hBxY5Ngr8iFg6mQWl\\\\\\\/b022Z9zbACwU5jdMtdz5KBE\\\\\\\/SKgayOc1PkeqtW5XcFGlGYOIHEnMl8x8hJAC+5l\\\\\\\/BYNaXhOzeY5QhhGQAF2fHUe1i+1Q+ShCcg491vct\\\\\\\/GtMnwWbJYZKXkgbksSFO6R3zmHwA6ygVLYBXNscYkZ\\\\\\\/oxzze9mCjt9yTjVrpyDQjffeRkvpDtVrH844IwN73DrKfvDr6H8q2BYcU66DfhVBO5t3NG8fDmQzQ6j19\\\\\\\/FGGlKKZvt7kolHRSiIV6FVIBhGldt6pIsakge1Gi+O0oxh2wsinmeuSTJW8qsc0cmShRufBr7HyiXI1ppl+BjQsOPL6QQtQj24ClH6pLRxLAco4PytHxV9AXh3ayaUD+3n2xJN9nAppcYmnnOQ30f1V40qIkroO7tKtZ+jvXS2kl4H9uQB3HXfA36bmMEudK6hSDkbxceWdm5l06Q\\\\\\\/DfLuvardyvgp5+zz+FNeCE2I7Diw+FM4ik+ti+LZeyguq\\\\\\\/IzBte\\\\\\\/\\\\\\\/tDd6FtKyBVGWB1aVYDuRUq3aL6gFA2ivwwadMeNolDaDPqm4Zd7rpO1Awo8ZoLsGa54RVmTx2JYOuVD5doxuBtV\\\\\\\/MJ5iR+yb09hMKHfwOmb8c8y4Df\\\\\\\/OQFQmXAWvtg4e++ooEWo1W5JcLaXqAaENkhCBL07oP\\\\\\\/Wjug8PYQPKmjleXsKgOVuNueyr0p4pee7HozIv\\\\\\\/7sQcWfB\\\\\\\/QDt4cE3Q8JdV18j2E503B\\\\\\\/XC+xnAg3Cvy+BCYxtjR0ehK3CEORaI7EwDvppge7LS+UuuSuhh+zIpicL5CHoPlKOUkwz7YV56k2sbT7M1mtUlly07y2pkWzGHOPY0kxUEvu4h32g5kF9VmyQd5Us5HGr5tM8bH0ofMsnupBfQt+kQg6JPS9FdqleaDsLteA\\\\\\\/GOY2LGLMsHrwAXESYv8DrlHppmlyUfwWn+Cyi2QGHi5Omc5\\\\\\\/V3GGi0n6ip4ECZrIZKmPm2SQpm9esx1gq\\\\\\\/HKBcLvfCy6+RU14G1v4M+365SMO8NNGzKatZm4hPUdgj6aFEA2ZyUEpbJHxXRbW4EsvD0riPTIZC3kZDvEFHLpviOXCIa3yXuFP+Q2nANsutKO5LZM4kOYSBcKl8lBESqI9GQneNElaEim7IgfMXIvYo4xoVv6v0NYFZBqvSyyR8BYqkhUfqtwgxDmpu\\\\\\\/sw5Bew4YzpV3QXnoWaC3kbfS4HURCKFVil+EZsFQZ20Z\\\\\\\/eZKlqBg\\\\\\\/w5+j6OaPMbziB1ABBCYtS\\\\\\\/McWTGgiPOu6M+6jKlfGoK8a\\\\\\\/bx1lPgAGYZUjL6YkKaQE7CsuWV8FRTTixCD7UOUKH0xWXKWqIc9cnQ0Tz\\\\\\\/9juZWFvcCJt\\\\\\\/SQPZJ2HTxFnmyL1Mo6\\\\\\\/OeNq3DM6COFnweOHP5pVQooTZ7Ysu9NKNclbF26MMoopzidQuGtCRlLx8Y8gxQscCuM+IK5A+uCyNQUVWlU698VPN5FWcU3TOlQ6Rdlc1mWmYRy1ZA37pO9f2UciJdhnS+CmufQnMDGPvkVSffBBDtzCnB9oGlbqyzek7sOWqXYQb16Q81+e8oX3vn7gCQieMdHABV2A\\\\\\\/mSJhXRsT6BABWS7JlrelH2OShzwRsyn9QC\\\\\\\/5p17RQxWoRkC8QdIBejrsHcJ1TUBf7qLRl4sEIh\\\\\\\/SQD43H+OEQXKRUu1k+0rLD4iWZvnTSKg\\\\\\\/JhQ1vXeztxmZ2BQCwlmvGZBAKtxqvrF5gMOasmJrJemrXT4LwWRrzdR+7Gs02efR8WM\\\\\\\/N7Zu6FGaK5qCejSLunmcSJy0owMgsF5ZmGomNX\\\\\\\/np8+t8QIg9bxyicuIEMz3tTpFZ65Stt2\\\\\\\/A17JoDji5QYVtvEayXiUa+xwfoT\\\\\\\/r4+pA4H3snc\\\\\\\/dpJ020gU2aflJYPtcL\\\\\\\/\\\\\\\/jq0KZstlqtcqVSwYmefP\\\\\\\/BxgjvpqdYHw6jI1LGgOQu6U7MzI3iVEFj+J5LZrPmnFQdFOwAZbZaad962i2cz7IojxrBYTdxfZ9bI7Umyx5s8uJNILrUnX9fGIVS9e\\\\\\\/HvmV+sGZQNKUHBBKzWbfzgvWUDbrbAHe5W+Tr5rCwic85xVnCiyClOQo3F3Gj4SE87klQcBYclEUtASiF3kLiUtGh0ls7N8syrdpR8wiCGsnO7n6fjysJK1GKlVlnajrzjVj7J7oZh4jpWstEYhyfeLRnJXTfeF+vo89lSFP+qLGzJxXEe3JbVgEKvkx\\\\\\\/D9FK4WPyx9vUmERW0yZySsOnBfam54m1CFEtWgAj4hYtbTdFoxP0VT\\\\\\\/FaG8rkENg9Wxxd9+c8Ila5dA2uU9+w1KXbjw3zcPHcmgPtPbHr9VboMnA3DNyWVy7YEwwMmQ6u2TDzHxfJUDEhFtTrjJS68ZP+5GT8ydywcnya06KltBIa1rYizVCe6oInOgEEvqtW5kX8OKJdIcPJ9IZrhbU3ejUsFg\\\\\\\/GIGL\\\\\\\/HG5BIgU3i5hxwuhNxdu0+EB\\\\\\\/Z3UwBbj1XejTQ+XnoNAHEg7IZwzQCYJzXBXI8JdFAoRiWoS1BQAos62UkvjQhjkg+apaKwhRRAm\\\\\\\/XdvqHjpDGAK9FpobGOe9ZEWr17vB+qPUXFd+Wn1dWnLLTBIENtfuQhmB75BgL4eAzJjztfk0Th4PhriycHi303GHqe\\\\\\\/sVlKq12sJM\\\\\\\/xllTXbug+4nUu4you2yamFWy2iRMQLCxfQ6WCetFVJTy2pPbIq1VgG9WSVQX\\\\\\\/N2r4c2djIWKg4b4MmT++CQrKfXx1HbsHjEUnxet4zLcLHUeCTMJg1puU+dlDKGCOetzLR4oL0KsRVIrobOH8ntmflPnWH1IjUNdy4Wm5K8wQX7bpHVZltk2rRT7svfpwupdORzAsNod9+FAZ5l4aFD\\\\\\\/eiWoidUwdXVp+P5QG\\\\\\\/c9JTWAqKrAlQKbrjbBe0tG6ZEPjz4LpVdLAx\\\\\\\/j5mSInVgphW6h51gsNWN2RypLOPjTnM\\\\\\\/WAqb5ppbpmC503H6k8xPJeEqgbdlBjj21SRLN8A2xb\\\\\\\/drT59caVQkAO5cdn1t64Lli6yRMsq89K7c7VIXWhmO3FVOZVFUjsQyYpeewriBw46XRNtVAb0Z\\\\\\\/uP24R3YVihjpiL4Z3KEe2S0utF+Mf0q4NimYNIrIt\\\\\\\/JYPGXEAEnohCW+EV1ohbMA3y4uXHkY57UlqRpj4N8VQZdx6mqs7xl9ZhVcod8C5H1zBeZV7TLfwPP\\\\\\\/mvp0DxLOUQTobHHZt99lMxsqUJ7DNRg2KU2CXrJEAr7miajHEz+Br774jSM3srdQT7PHr+qxQLJulPsmmlr\\\\\\\/S02PHUI0x8n7Lq1zWIenNHo2mcgSDFjdIndr1BVYdvzbH1VFHMHqcogDjpGWuu9gjJ2wAkun4qz3QwxO4eGyHRaQP2YRPo+Y8CG4JvJlnUxYu2rSnjG4ooBE9nPScCdtmYUVStRcXXeGR05oT71AfxBTLFvT7vBt45QUdZryED0hEZGkqsRfGT41QfkBKSA6udJMpL3p\\\\\\\/6rk1fPqnmTIUUcH3nlRscREiAzbUxdvbaycb0TXl70EqZNTC7A6Xbrk0b4lDYb2yL92I92BT6BJOGETDnqlN2WvNtCFhPEYDXDVux5u37gMGsnrZUeezLyQ2QHolUpasdHGjcsectXSP9+o4hMrEQqWV5uz0UqPZeHDLOePDAr2gRwenV8m+4TpgpBynAwp9UxnKXOpcyukEcuTwLWF8KaWqRw9OtPl8y7RjnOtvs0HQ7tLYVftxGPgVNqvFHlZ1HTPGxKhqeXmF+8KySmgfe98wtM4RkBOFrI4i+k9jYWcwJvl1IlQtqwYSOmMkK2QI3\\\\\\\/uAYoYJ91zoUXezY5Nv2\\\\\\\/GsSZl4nO+RjbU2h+5LKUDmfGdAfjoVlnAmN7gymvhcOMJcjhVOS0XoerXGoHa6zf+zxfFatduL+Yav0XUTJ4\\\\\\\/faG+hQUpOBc0vJ1SlP5j8SF4e63q9IjVvRijBQO\\\\\\\/iGcne9F6ndvvkiJgJcezeGTOSl2X3F61Dul0hRUJdAwrEkm6y+8mHFmatSZRRYzyWWFFpjbO+Gq7NTQvqfQibbdS74tvbxTRT2sfNbVCYLVyMDjFglZfQKR3NS350hjpx4vbgSBUH20l2OCXz6em\\\\\\\/swJmjZCpVk0cCCFDWGc3qqqAxt\\\\\\\/b+h6Z\\\\\\\/U4N3T0J5l1rI0PEzIId\\\\\\\/Zm65DpDvujC1OD6UwGYX1bs5jtJZe383UJ1LxoeZlMYuEGW3ukilYhFmwdTLfeX+JS7AsLzRonEUrucdKWZaCIFfIx7XdSopTRxGsPsgWSITSXPDOVaxqMV2WzQ6gcNpXT8Gm\\\\\\\/sYQDAK9QqyFx0tJEUHdyF+ccm45AvmhxsPZVaf7h8PYTUmLbEImo1o2oHOC180FV4uAOIVeXDGrJ9L67dmr3gCc\\\\\\\/tCh11xSXntJ+Qk+OI6ID4zzkHio5aRmCcXLq9ZxGwq8pDCQP+7oUDHK5u1mqBQq01ASmppLaT2Dk5RHhz+1NF\\\\\\\/CRVcaKHnuZMn5eSmC157gSKacTLLWte5gc8E8gtz1PbeOHkSA2atsmqDmTHplC8GxpEDX0gweb5seT0aEPjfF7oTu0rMaBcZwF1c6c5ztf60xVK5giGM5HIxV+KhGzC0gwlDdju6dg4RJST+D0O6hDYE17inSxgDGQmm6E8b3MzE7YqkIgrZrL0iMRb\\\\\\\/65lY3Ki4gkQhGSRI7JiSaO2R38MBdeXpF3b\\\\\\\/A\\\\\\\/Z3Niwwt8yuYtwf+Zz0M4gKg8rGsNw3f7cDSS2PpXhwqdJxNnuZa3g2upSzEev4+Vxftmw1G8paJG5WnZxCSDPcrPA1vKyJfDgtHN2uLCaoEj2aH2egsRBZX2x\\\\\\\/RcKbYWvl\\\\\\\/y7nbqug8ejLFf0BUcvKPhvWeaATaLG65wKu9eJWvmgVQwvKWgmlzZffJvtzNUEjehQ7P3aFLrxpi9mjMJKJ+94IjYHj8KA\\\\\\\/EmWmJMUahNQ2p\\\\\\\/vLdApLpUS2E45rqwCeBO6dZTUr4boIFMdmVnvPbm1VxUESWTr7+ov+4Swdcig\\\\\\\/E8nw5SXUXsYSOhkrgER+jnqeMGuCwqOnRqJH0KBm3oThHLDE7hJX6mG1OVlJqLcfTOSepocTBy0uMunmyDlL26nm1F1B2mwlobJZBzakFpjpv0qOQNYl6WoB+ifCLsV855RxIXxZ5PxnEfFnVhu6Vr\\\\\\\/70AYV9F0NtzDtZ0o8X\\\\\\\/O257PsDYyKl5xreq3rm6wHUoTqRahen3XwL1642r7F74ArtHWfVZw\\\\\\\/Pd+Ev+1yTxk1Wcone2PNQ9kUAAUvqswYK028x3FUw5z0ZaE6M7U5kr9QKHY6nZ4vp7X8Ryl57D3sNwO2TJMbvosZuso1cqYdnLs1wqkkOVok0rHMJId1St\\\\\\\/mWZGlSBWmhsCp1dSoortwB38fqFiOerikKc+h+JdHbrGAHDjYmqnNgRFjFgCysldLPvYU3qbTmsUllZlkXzRmQDa6GAOZkxwc2MI1SGGQkeqfcneUFcQOCyTl1yn0YSm5h+ls2ZbZXWsdG714cKElggN2n19BXlU\\\\\\\/hB6dHnxjHFiCXHa1FspAtPzXnajBKe83339l3lYznCFywIc9hBmXiaWtCGJ20zFMge4wKv6on0d1INp2Idcl4ukPugXj7TDYj\\\\\\\/JChwxG6wI32LW6jWXrcVNFAmHC+LlrdiTRre9SYC\\\\\\\/89mZNFw2vD+Lavn7qn2DsDtzW6e0IvK4tlmQhRG5BJ9IgGTN3CKZAooGmnwkzeusqRrxI39MexdM18dRuw+nWRfdfSCpxd26g\\\\\\\/eJaNXKNebg\\\\\\\/7vsqEGVOZotnT4fVOFHG4RG4rPKyj9kQlSPfeC+gbJ2BhNIks\\\\\\\/rISfukJt5JMMLe5Ru3Ky5BhNvhgxMDLYaLsQnjCusWBFstl5HRqMvc3h+\\\\\\\/uk51HBZkUpwZ38eWuZ4RRtZwObdxLUH6ts4qhSwAmHkjazkeOaKpg0P8+ypithcPqK+9T5szaP3f9QiNLozY\\\\\\\/FOlBlMYApQqIXx1FCeXL3xt185\\\\\\\/JU2CPJIMCic0pDXj2XZp1hgEMemyCnR6GzMBRy0P\\\\\\\/Mgqbc1xr0GypQHjCBvSwR3ybrrP5uPtzGt6sBudziaB8f7gbU5\\\\\\\/T9dHiYEWIAW68DwOEcblZ2ZvDAQ5vLgdV2nLMJ4uEbrrQ3pHcctBq8y89sFSNMH4q77Ey7WYpCpW8coZIKoHnc\\\\\\\/lqGlsYWHjd8OWKyPjZu9hBJm3nN++Y1ItvB4qhx0QkDh+6B9SSJKbsLuTsWy9jqhhaO2VurB4VJMOsXt4sk5VFE+2sCHqBOFZdzZXYn6Kw0FcnlCqAnVoTyCdA+Y4k31LXq13yt3pCeeq68yoz7AwflfIE7xxPo\\\\\\\/UjdSNYP5fnfdPDKkOQaXGBhvxC3lzxm474dLlhj00pYGyzanwPR3YAuoXe8AnbPzjqkvOkeOMCKrKFPdaE6xHwfArukFq8vv7vJyZLgigsHhOHhBPCrlFmetC5QUWbFrQNGSduWwst4FVspQQ3emU8uLETZeeDVyiGumFR\\\\\\\/21VLItCzBJWVLpUel+XyEYJIfe40B8SxKhpTx5Pgvm8XjXRbMZCy5nep8u0ayEsfL1AhUR9Uf7xgXICrS9+BDDG1oxB7QXWU\\\\\\\/gR+JIUXU4QCqh1UwpbIt4Gf5VuDyq+VrMkcpxRcLRSR1FJgr8Y2dfbrik49JMeXawhIlIvCK\\\\\\\/iL\\\\\\\/sr7DQSeBneFGip5SMdjBLRobgqXlDDz4BZMcEzM2OPN0bb3xlTeVD16J3V3hR9WSzn2vw9txofeLk\\\\\\\/aYyb3vxWNwSB4sZ8m\\\\\\\/TgMPchsqpTAzPW9a2wQmztfGEJVmhtlcrMBmpk24JpCr38sYjaOXdb9AbWF+rrf\\\\\\\/XczVSRl3nFZThThyAeHB756X2p0mMpW0dBdHWHZkx\\\\\\\/XJCK1fDIXEE9NPIiR4JpNNXOcn8RkbAdKYj34Rk50RkqgSJY6MP8MZv9LUitpW4oQghyKhOlDEr2dGGVvt4yOgKokjq+aTRP+RoUNBfLIXyQT1mlS0G2vpt5rSkA0ORDclEZUb0zI6kG2rRFE9Ic\\\\\\\/dw7GL0VqFqIaCP\\\\\\\/JfkboN+D19EhvWS2oIMG7GW5PzVjYI8pqvBsXl1fOBoI3kMoLEozWM3E9H4xNW8Hbkj\\\\\\\/\\\\\\\/jhFa4YIBxtSNaXy+nQFtvtJmUxTYOsRwq5fMGBrJptzDCnVpxoKYGvRnPXjg+vH+h1yLnrJHf42FEVcRiCmWQwMWEsSaFsDrHqe7uXqQHwOEMJpT0soFWGQ1ee2Gz1pkapOEGE9tDOOez8jyLL+LcMaL1ABPBT0EnbrXwDSd33FQ+olUfCiTlfi5a19fCgv3y1CqLeqlK8mSmRuu5glxAaGNQLRpk5pWOKNEcOPyqyzL+P3MYezEYjNlGZFePRV9BGHlHvqEHZzTu57QMKaBjHfnsevNcgYzrvk2ApYpKG+0MRJ4M\\\\\\\/feggXO\\\\\\\/JKFitCJaxntVDWziQWTYqUR43aaesT16osojdqD1T386G92F2avEaOoPR1HcQt969PpTVd0IIX7a5mULynxqTeyj9cS2jrL51k1eLbQQFa4A7k4t7MdhRMRhvSQ+KYbqUmXIbxNdJBR89qMdYeBoV13LH+QaQX6O5fhJMjpTIKNHwx0XtGDcPhplBFpmJZXZBNVVLPEACMbQPMgZAKZlD7VyHyHFUQXhwvAS4RoP5VFVzopO4g6rZeD4HqEMMDKgCZ+ZbhiAnabjQgRjijny0OmD2knnyOyWzMyeZZqSXpcDcQ1xgguu+24+pXZFU2KBBYT1bMsrKqe9qcw8AoAj1fBGFAxcOoPBo12m3etRh5dl04cmfOpSjxSBWDZZk\\\\\\\/0pUScYGT6\\\\\\\/gW968hrk7xiIBlZG64b\\\\\\\/B3+SAqir4DMjV0jm0zSqlGTcsPH7mH258Ap0E4rUCmJi9YJ3wawkWGLSnNUsIKQI9EtgFWoUlvbPn0\\\\\\\/6ojdCRxBAucFchrJBECNIPgIJ+9t\\\\\\\/+pb18mFX4d2C984pQlpPGeN00XNaY9P+m9O4MMKDQQgkuLcCHTydxDJTmOaLYiZSnSg56mD3Ha\\\\\\\/Y8oiCd2tdtGbQ6WCG9RYsYipadpUyA3vw4xMtHZzucdPZ+\\\\\\\/f3vf4rdqVgbhUMS7DGhJTieJ99tuScRCluccn+RKnDHK7EXpnIcZKmjjwNofL0I74rj2zGOjJtzp1725Ac0bLFGR0eydAlFhzugL\\\\\\\/FMN8j0ZzQTS40mHr2wq4efPAgJBDOFTYMCm7z6h+Wf1IeZhKxELrbafpT8tIgyxHaJQiPZb9LGlTTgOR4g0qKt9LWPsmzTH8019fkNB+Zd9U7HwxSj7iW2tx6zT74X1Gf+nfprx46R1TBXWDLRggs8IMcH8Za6\\\\\\\/HemRiUzgnYyvcJr6mmrAByTmEeDtFISuOvp4YkSaKxcsH54b1dEJLhvsL5zOJZMGQQrVfOu4qRU\\\\\\\/a3C8wKtfGlTKJ2OwBNWaU0U\\\\\\\/QLcJIgm5R\\\\\\\/0fivpekg68kSNmzoa1L67fp\\\\\\\/Ls5TCr\\\\\\\/LdRH2FaHCBRhrL5GP2meXzDI8YeLLyOKCteqSb7n3JTevlQRd5mvTuZrNsgTKxbV0h9f11bvBjpDeJbYq+SBN+hM\\\\\\\/q+1hWwHh5oNhNEVr8ZlpOrmf\\\\\\\/R5t5oudfGd7HUD6QLAmkU38TsXJnMe6cWetYZUAw2NdcmuJh013s3QMqvyBqt9MX4o04DtIHsP+JdRKxHrMF4XjkjFYo3EKsQxwYqfWAErLrtgRw0V0N9v2EmdCsrHTXvwSkLdgdnJE5Q\\\\\\\/KB2qvrCLJKHkfW9IxUFhB6ELfvDBM6HTKy4sksCghssI5+dK1HU\\\\\\\/zy7C44NGYgN4ZUiARESD7gtJiAtXxoFnI5TJSZwXL8q4q4jFH7BdvWj+UfhWhvyTLewfxv3JQg5UDLNGOEcdSqqezRQYFyr4HE3zuNk9EkeXsjm8g2jW+sXzHqDJJqAfMFjTZJrErp\\\\\\\/bYN4DWAgXo4oUNozl3jdKHtiIS+xGMKr910IBhXS3z4XaHmzOzEIe0auyhriOm5HSFyapkN953COwvvyAlQwvQPLPpgmxHpMjrfBoyMXJUokcwVA\\\\\\\/WF3QkHSCrjn2HkXlzLmaWwAlzXVyZkHOWcqEnSPpk\\\\\\\/1nwITs1Bp5TtT43qY6qcA8GVI3tNasUjrg4U2wVuF4rvgHDFRZeCVqWjWWxsMxyXgqWlJgBwdQEYh+B7WfFu2IUVpbIFd7ZxofoIRpk7\\\\\\\/pPQpJGD\\\\\\\/mAuHQdgkzBfX9gOANwUIky\\\\\\\/78g8pf+vCNGjb2Wsprq9qzLj7H433OUQtS\\\\\\\/KvBWi\\\\\\\/vbgS60BI6cgifgJ997nf8AnaUzSo38xZjARVnioe6T44lApqvVu96yTMlmBucTyrcapCn2QKSsC5XJ1vOyPAqXMjtvhFeFc+VNnwaFpCc0cuB82C2yKzoeWqJNP\\\\\\\/zMlnDrPdWQZpryqcxSrbmQRCGqZSPIwhgP3QDp7Mj3OnQc9gU813WHuW4D6cF1g7qjdWAaUtfnZ\\\\\\\/nfMy+tfNWo\\\\\\\/DLhxH2pZHR7wX35i5DwTYk7hx5rjGSq0Bg2iRvPm2xauoqEhxoH+mf1lk+D3Bb6EN9PPCqchHn6p\\\\\\\/v6XGXBsoMJNzTMVr+oQNmUdBnAPVX208D5TRshVYvBEti9ruZv0eCBBMvDD94I8CFCC83nVXagzFCmimkDHXvppBoaARRSlHbPmp5\\\\\\\/aTSgkw+te3jFOgxdGiBG4B1C22iDnSYQcaPr+M+XVFR9gNlkq4Q47KXOdITPdvh6KRgFsPf+aZieqGnl+jhbQ2q98Qjjo7tG6O2Mb5oR3mxjRlnN07FdqOeLb\\\\\\\/cnaemEXprV2U8FnIIg1vbSd28nuBWRlmIzfj1SkmrJaLc5M4HK3ibHTy4X+w7EUoOr7yiBG1odRvwTvBQiSfrDYO86m\\\\\\\/hOk7r4lLafh+XkQPRFJ+qtZI0fXBzyqKtRnO1r9On4lZhlDC6617zvKBprmrrffsSHpOybTWFlacRJFlyhMoMT\\\\\\\/4vxG0cde9n2LNdfjhkBfCn9Y6d\\\\\\\/WbY3pX3X93rzGikwKEYRpWn2O\\\\\\\/IymcD4J\\\\\\\/nN91FMvRi1fzw5wJuTNm+DKsP8gUJL+\\\\\\\/U9F7hj8kJJ18dgeznE2KOhIxyeS+7k\\\\\\\/KEiBO9D3+xDeE2iocJrvLwWcQCPMpEkFJm+1o3WNLOncGA\\\\\\\/5WZAZuu0TlxLU\\\\\\\/K+yDfWiIdQiPtaDyyghpaTv9Tc1uyCEGoM97hHDtxBmJOXnRbHuRUjQanMCIJQ7qiAlou1WpZp3G2GOfhncX\\\\\\\/Tc\\\\\\\/c4IQzfUZeuwkpmOKkxfxCUH1Ne7VlghHFQzjVHmm2jucYr4HffOQhzp1pwc4o4SCJm5A5M65oQtcB9X7tAXAXJdKTn+QbH51\\\\\\\/9n9\\\\\\\/QFYjlNdMTuzY7BpugcyjK0iAiYrCBKtt1UK2eQFCiQl1FKFkynuBRhVg3N7EARl0wZ9p2zJUyTSRnhpNjinihXtgt6brEtRCZrf7HFGvczQ+MDty+uKg7siB4QeZUOegmSVqIdeObgnQTkSTRIvD3h2\\\\\\\/lt8z6WXXOTuWLsm5JZyFrhsBel5Ko2oqwEcmt5bek8q\\\\\\\/x+5uB4z3WzXS9j\\\\\\\/i\\\\\\\/egDTD9vbbbHrt5Wso6\\\\\\\/Eiyv6WtZdYtLrfaQwt\\\\\\\/+tUogI7xHq18okFJBxOW4T3d3rgTh+Fm6kvbz2qjyqLMsA9ENUAYuR5l8ieOz\\\\\\\/xHh9v8mqq+PQkLPQKirQ4ywbKPulkPBPbiStOpCmOuf1qHB5ZIyC9DD4tq3kBia+LCpnLwy+baAjt2E4TDSBBQXcMozhXyl33X7QBe6ldNktKUM66g\\\\\\\/uI1z3iUjvkiSHfxlzKb4txJjjvU+hoBkYjtKiisWOrhArK6o9n\\\\\\\/GAuJkpRxOPxaj+Cu5XKwG+YaxroQuf4N3NBKByC\\\\\\\/Vwf6dV9loy53JV0OioZzhOFGdqhrjBBL+7liUQm+TQJQaqcecubipv40BmJChpe5YxMu8buP74o8Tfsu2E0BMWl2e4JKUQ73fuD45jn7ZrbPJkkHdaTZM7hfnEOGHzIaBv37ea4dkA0asY\\\\\\\/xneZzezWPGyxA9g+lI7PwoTbCLl3OAWvLn+kCKPHbMJHWWGchmbN0Q8TV5+RqV1INsSzU\\\\\\\/JiyPpi7YRBnnN3QTKFbvGPKTaTZEUhUTl1vr9o7hgA4KhLtyN2QogqPHU4YTzsX8xnZTi67NbAHVSPVWo4BDwg6qRqqo1UE9sf6rhtISyoy944HCO+401bsowRf6nG0IxWKarE\\\\\\\/puJOyMQtPizqH+FvDrpZbBOY7eE\\\\\\\/GfGbtLY7IOUtqG0kJV5Ckg5LusC+QfZrgrpURkBf+SkAOxuZUVF+tEaKN9vcr3TnMy1UzbxGEeTySyJutcoCLfk\\\\\\\/WTg+4\\\\\\\/r7X5PpNRo+npDIjUsZDS1UVwPbwRKU4L2vc93qESRSLthCPAdIpj+W+OkvBS2hypmoq4KMWtW+t7HgtnifnOr7KYXGrWds6Ua7lw55ItB8gpn\\\\\\\/kZtalTKgUOqwI2ucQV7K1ojnUU2uATtjkxAsynxAaadWYoPPlhXBY2pju9slN7TQIkbXmIKX2nOpzkFk8peyMVccuvE15TySBSh+AKNvfEaOLfxPxINIpTGo9KirXWSe0v5cEc7X6kWXngYNGkFo67LgkNYY490fZDNSHBlhWHZnRlkCl8AXVq9DmlKNH8WemUvEsXbATEPdeILWbpSr5W7JWbk7UnpzGj2umcazCYmVXvBgqqtU6lOpmuKt\\\\\\\/nX+PKk2T\\\\\\\/UQ+NL79koClxv5MgdvCf9Jf7lcJIc+xUoXFDfqATwyKshNXsOtZpc2or0lUnAEe4wcOmIDWcq401Qbka8uNZnOj5JBmDqAiJuOCpSz0SCNKf1dc78T1118E2UWQURxA2bBYiGjxXpu1Kesg\\\\\\\/koTBqtseFYCRDXLMt7wnxikcGnE3qMYDILPWoqcQqdNnmWK5zSbsm5y3lAey0Clqo\\\\\\\/z1PrdQ4STihOVXyMS6X666+ubLaclfnG9MOR2M1DQNSjzjlGzUKxY\\\\\\\/i03fKHKrtmyDrUBrKzz9b4bbHJerktmzZ10aQWfk1EKmfY42xXa70FF5q\\\\\\\/zxOVrf1fGVA82\\\\\\\/F\\\\\\\/vYCUTiQao9z0HywNs0ogdYNLw9oVv0tkzbT\\\\\\\/Q5kgi7ZE9jrw36Gs5aMRJvgC+atYcH9qFIpcEa5CRUPGWWTHFd637FrO1E2yndEzgmNgyTpgyweH4SPAMrZRNC1BMPbV99ekAN\\\\\\\/YiZVjY3o\\\\\\\/cXDzYzASfY+iNexALSPCo2wtozy0xxMZhPMoDZLw4b5B7oqZYAFcjBD95tHbqZCZOmuMSZM\\\\\\\/8JeKb7zhlSrgNa44RomQJOmQEaGtSmTHYuK9dwu6mPdP1P2bfXY\\\\\\\/jOaFJdFGjSvPqPPdKH6p3IVFh8cebze0ZszRsi8aRsBhVizofsUQr6O9kq50gFZxi1tPtSWbwl+m\\\\\\\/MLnxV33Wa83DSxu\\\\\\\/teKTjjSrZjG6FBe9AP0zQF5ro6B2geU0Htr6WAWJuyD\\\\\\\/PZG6SYvAdn+VoLdL34KNOotnxHh67W8Nj8q3xfFO3JUlnmdsSdbVTU6hHJ0bi\\\\\\\/FUS7qXGSWqyzn72Qui0Bo\\\\\\\/2OiD+bvd2Y1Fpw93Js2B5gZGzeuB95n8TiQbRoNwUaCzzHHiYsR4E5uGeVGo0JAaBFL9YfyX+DU+36tBzEMV3g7hMeX4CjUgawrNoEIdRjJAx4okZoF65FfUguW6\\\\\\\/aNHDaiyA3ojOGK\\\\\\\/EWIq6uJnUwQxpVN9PaXHtvQefMg24ZEz516oEias4WfwXWWQLSkWwVZ8P0lQvH2Onq9V4PP65GMnNw6mhCJVlT92xLgCcKFIkgcaqCbOL2ArJjhr3wLSnDSj3fcBnXJALk2dk9P9l4\\\\\\\/+AgTsB4uG8EzIv0pkhHT7+Zu9rupldldt22JHihQSg4X0X3EuEegQ1Iittu7laXmela2R3xGB+f424xulqw7gCz9qVSBZqVZL3bZNggx4iNwhID+\\\\\\\/eUY+pm1xsCdx8dtmvIJpsCghJKbI1cWf\\\\\\\/kDPRCrsB5GXPuCdeiihkIdtV455QcNYYu2s91lXvrylVFoGEXFaa4vRp1wkbozEoCq6sRw0NE0lGvJ6G\\\\\\\/Amqn+LT5H2MwksfkZdXKGbTOvhlakFqe43AdXuqr+G4XVRT63gdsvoEwXTRGH44s\\\\\\\/lSTEWz5cBd\\\\\\\/SlaqEEccJ+GPv\\\\\\\/FhfCoR4QjKDAA6Aoq0G3LFBjBm80pk7LSxXSfJwEVuUVvKZVq58DrMBrBSksNfI8ZDD36j+5m3H+24XnhzuSoI3tnkBYWTuWA\\\\\\\/4+fKijCA4MEuJvbWVpQMp4J9fYPifwef0DfNT5TVs2+N4uRZxgtZFd7UYRWRQEzhSDzZT\\\\\\\/Gnj0ZpcwWaxlOoPbDlW8LQi3RS7ICKi9UEB1kYN31Uv7imDmVdT96YtLsB4M6fNDlgRrjGhvEf39ZZ9a4xhja1svpoL4T8hOgBEGbMem1Y6331naHj9JKzVdyCv7qIXI0QeM6XkUGl4iX3hvFeuH\\\\\\\/u7kLUdBD9FjcH3yqfhERTyNp4fMVFaPLbj5L2\\\\\\\/C4DQeV2XkeGSTSCttSL9tzh46XoSw1dHQgknZkLhxLEhO\\\\\\\/FUpHK4twnUcWcfoGJWsKKdEfdErqoOoWErfNw7esgxkMVxaRj5h+0IcqPHfXeltGoCg4bIfgqAq4XbHbLQSMnsAVlcHFiQrVe1b0hEylrF+wsJ+q6mmKXQ2YBxvxGmzE7ToEPBKAwJClvmrotX3Wx\\\\\\\/dgsfU5anPKCBKkwMy0LN7unHnQa0\\\\\\\/ZkcZGCeARxdG1Z\\\\\\\/dl9ce18cxIFjn6vOVOpv9GO6cOjlfolPOA8K2pvtV5WBUbqXMCLtMnCXk6Fcstj+XbWoTI20vwVRa44fHMDxFo9kw\\\\\\\/Y6D5zycMlOyKa3ViADQcAHzKpxvFRkXSg1O5rOYowoV+Ag7+tNZ204A7qPlSeR0OF9zIJAtwGb97ZDd1rWOkqFGLqA3j25GAneX\\\\\\\/bLKKUVaDbQyKbAmL4tOa8dCKuNkavu\\\\\\\/d35gw1jTLHfJ6pQupealqXH8FtEALPfTcX93CEnJu94nvaM3HQJ5HzrnZ0D9a9Vx4uNpMhv618wjKGZ50siDU5mcIN1i2xgCDYvpLKq2PYk4pXrurOHN9gjUMmbSBLdVOl1Ks7Z2vtR4tS\\\\\\\/fuDw6C60\\\\\\\/WgjFtxg\\\\\\\/cikjYGu+wkQ5b5s8lU8XOstmZAR78kcQOIJ+9lkFLVsotoYHXk2O4YQPhSWHDKVgcOkEhKsE+VYelJtW7EusDv\\\\\\\/P1eal\\\\\\\/0ZZ\\\\\\\/JD0+FEjRd5mf+2\\\\\\\/xpHRcu4sek+6J3Hj1cVR8eyu1TnS2e9gFCHtIDnqAI1EHKkgvkr1H4v7LFAPoOMUIpUPjLW38TyFQu9tminIC6N+RqU8I13qt8aXhvfsW6l379MzL\\\\\\\/1c2CW2Z0IdNN49CY5MDVfB9DxtM+s0bT0OceQ+RwgixTMqYos8yg0IaV61qfR\\\\\\\/cOE4yRFj\\\\\\\/2a8tbfC4SwV744t\\\\\\\/DbLP+uAutCyhtV+bZ09RCZIDIMPKFYjF4gtSic3a3aeluK2hZJqjyHfA8HsCQaiy6FWFq+Bu9+X4fi3Pb2LtKEc4xHlTEx+QbLwkHjGL2hyGV2jsmaN4EUTJsxgsJybOAkdnhN4ngw6QqPkdCGfn0152JIl7YI0z55SyFdpjujVPMwq\\\\\\\/lxiF1g5DoOE7qpdpimVUpbkQz7AyEOO+6HJHWPUQyH799dIDAjHhmgurmNZDUTZXv+WVXwEvPiQ+JILTDFz0V6aRZnC\\\\\\\/RQiDdJ9eWG7Hqy1hUgmxzO42S0647eSB57xRE\\\\\\\/hBdoaRHiX5XhTV+pwdb2TUe9BhF6nbUEjyXCXIQKfFZtgTKnvt2VsQxT0G8DtZuZm7J+HYFH\\\\\\\/gy770a0enHPU28AR9+7WMkW6ign969iU14rTl7SluH8SZEQru49Hec8vOEU\\\\\\\/vbQP48YDPD5NsJecYqiw79ha02ykbhrBgoFMjubCPb2ESZuH0AcgUCqv1bvBEDSq35i0EA==\\\u0022,\\\u0022iv\\\u0022:\\\u00228\\\\\\\/ZhNas6pUMbU6vZqVR5QA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222bb4620ff912a0c6f158565e867843d0c3e1b6a00b05d9bd3df70473c35a009e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022m67DcID06Ps6xXSIYWFTcuFdP6SHKiIYjArtv4eJi4Q4TIH\\\\\\\/MzJimcO8+PqpaMgZe1fgHYdq7LDd5ptK1nOD40bxvERQxzQsf65dcopIHr4CfMj8Tug24a3NPKEQmq55cKACr0PldAQ+m6IFBbB1c2FegSYcUGyBKHWKvnzYG9LN9+C4xiOU99\\\\\\\/smk3pBLUTOBWMNxSSV4HIzWmMhcXjNsndLEM\\\\\\\/5gQvdXW5HbdbxLDhN+zgu3H0EwJmn+MIMvbLF7K3NMlrzft8x4subKuA\\\\\\\/ktnForIYPWakKixSJcJDc4Cu36GPcK1ZyAw4Cj3YzxTrsIVJ08q2hEuDWqustbNwAuH9QBg0jvNnAGkMlXMHnoyswG9xItr2megiru6wwXxQs9SWemHxcFxO\\\\\\\/KDE8JEZ4CysFbkK\\\\\\\/suVWXzSXionMyNofqvqliQp8GQexxYdza7+P9LHCWk464CvvxauPH6ugvQZaUVSlWF1l6xXfEp8v26Bp6gmNyAMeq5qO3EhzL7q35tuu2S40xJ94HobO7MWwbZOjfET8M4avRfMGNRLpDaCLxFAfE7U9SV+eMdDlLztHXBcMxk\\\\\\\/WMkwSddnBgrohgUJ9n9g35ZuJZOAUxDYILGCa0HTZH3YdL4AUkdJVawXSt\\\\\\\/wPoSXwiUntEeBG\\\\\\\/wDaw\\\\\\\/ILpeEzktiobP3WTL7QztSxSeMhdFkDQWJczqpqxBf6dLnVvFbUp4GUTBUjGMPYGWqAmuKTzADDPOYZREQoHnbEcEhupDzxekcHSNfv9M+xefZcqzzbTPhObks5uc0OPGf2cpIqbbEIQiJsLD1qUifFrCivU3jaFheXGfkjs8H3Y8TvFvABKtaKlQdnYhwXp5eIVyIK9i87+XBuqXVqK3sYKvs5LkZ6hTC40OkLFFaSZr392Tef3xFRccR8PMKZcFyEnE7TSUFMf7ZQTCBtVZ9dQsZVVVQR62F6tOqnFMLAvx7jbDQbFgtnsUszCNiWxhrCmSPSz7+NykU1ecMtO7EPvLTbZUbDNG6DN4eqhmUr0pfV1E0AaKS3et8xyB34QhmRk6aFAHfbDr9VrBj1KQozxJjscxW1mxY0wjIGSmdOwjY71TX2jCtZ4Xk5agonuPJl1Pk0Zv6yskF1KIEbZXqKfmu6oC2\\\\\\\/ZGQbtvpqHP52MvqjI9QKbgVFDYNR6QDwWpWFJVpAL1u2ksvyerSeLOE\\\\\\\/g3ngw96YcmxB+4QJeV3xV++ESC+Zvlz2yHfGJ5iHVjoC0ovkU0Wabkewbie33dx3jJU88qeTmA4q8CB4mSQocqSrFRt8av2UMNjIzbyS7e1kAvnBGf6oAMCd7h5iK+Ga1RNINKt1yH\\\\\\\/\\\\\\\/WxxX1cHmrpkQuF3mw+FbPq3Ixa5Rk0qFGKV7GxVTjiZ\\\\\\\/3GB2fSyj4AfwiPNXMc7ev\\\\\\\/pDNBwfnYMnNGjuNPRHQ2E0hpsE6qcu275O5PhGw42JgzVPd+5loRZKOygHeE3f0tOOqKS4vOdmrhs8l5Y1owQZ+lTtAF7fmvT94OOwzsm0Pu1IKG2awmdb1L8dxetGhV4J+paG7auW8Vx+f7hE8LGKJEiemI1zvFAa79iaz+zPVpTSvx+rQjDR6IFa2q3EFF+A5waV2fhIrDyMriCFyW+8bPgDkgsVJFUIJ86JwIKAAyh\\\\\\\/FSvfgS4ebOI18pJW83T6WK8Yi1BCA1f\\\\\\\/nzQoO3V6nzTKxpwqrvaOWkqN2KC4nR9XhQc2cDMMzQPg4yT\\\\\\\/4MrPhZLQdaq4nudnzYTHSUse2pCiZSmbsR4ZURjf50McUXkqg9YXHkIPJ365qCTIFm3RF8qHv4BW\\\\\\\/C48eG1gJr5snmgH0wXI5U\\\\\\\/5waUZvyy+IWEoN77RVV6tGMr2lNR+78mpmAbifBuI1qAG8hyfaxdedXkF2xyi5+fMaU5lUc6+V1DwMTSLVpq5o8OlNRDRaNzNT1GrPsUQzQ5UUDo7oi7C5VTwIrDOC8Ar009L5HIziZ7CucnbzsgPTEPexXlKzDd5RtMz9bETkBeo8sQ21dpVhjKA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BqJ67\\\\\\\/BA+cq0skbLiOBg9A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f13bd3feee1e03c7eac3e547a37803c530b88284e04d0a36e210b780c37408f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490130 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2265" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-svXUAOZC2MbIM_u5viqLA7tO8HzgE7RoTgvJa3YTr-I" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2265" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-skD1QwdXa4DaOipbjbMe5ULmu7hw6ipTSnPQdU4vosI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/7/2257" title="A Genome-Wide Assay Specifies Only GreA as a Transcription Fidelity Factor in Escherichia coli" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/7/2277" title="Tissue-Specific Functions of fem-2/PP2c Phosphatase and fhod-1/formin During Caenorhabditis elegans Embryonic Morphogenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490130" id="node490130" data-pisa="ggg;8/7/2265" data-pisa-master="ggg;g3.118.200063" data-apath="/ggg/8/7/2265.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Region Containing <em>REC8</em> and <em>RNF212B</em> Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (<em>Cervus elaphus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan E. Johnston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jisca Huisman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josephine M. Pemberton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2265-2276;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200063 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490130" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susan E. Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BE.%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan E. Johnston</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Susan.Johnston@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Susan.Johnston@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jisca Huisman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jisca%2BHuisman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huisman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJisca%2BHuisman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jisca Huisman</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Josephine M. Pemberton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BM.%2BPemberton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pemberton%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosephine%2BM.%2BPemberton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josephine M. Pemberton</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/7/2265" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490130" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2265.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490130" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2265.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490130" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/7/2265.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490130" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/7/2265.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490130" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Recombination is a fundamental feature of sexual reproduction, ensuring proper disjunction, preventing mutation accumulation and generating new allelic combinations upon which selection can act. However it is also mutagenic, and breaks up favorable allelic combinations previously built up by selection. Identifying the genetic drivers of recombination rate variation is a key step in understanding the causes and consequences of this variation, how loci associated with recombination are evolving and how they affect the potential of a population to respond to selection. However, to date, few studies have examined the genetic architecture of recombination rate variation in natural populations. Here, we use pedigree data from ∼ 2,600 individuals genotyped at ∼ 38,000 SNPs to investigate the genetic architecture of individual autosomal recombination rate in a wild population of red deer (<em>Cervus elaphus</em>). Female red deer exhibited a higher mean and phenotypic variance in autosomal crossover counts (ACC). Animal models fitting genomic relatedness matrices showed that ACC was heritable in females (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-1.gif"/></noscript></span></span> = 0.12) but not in males. A regional heritability mapping approach showed that almost all heritable variation in female ACC was explained by a genomic region on deer linkage group 12 containing the candidate loci <em>REC8</em> and <em>RNF212B</em>, with an additional region on linkage group 32 containing <em>TOP2B</em> approaching genome-wide significance. The <em>REC8/RNF212B</em> region and its paralogue <em>RNF212</em> have been associated with recombination in cattle, mice, humans and sheep. Our findings suggest that mammalian recombination rates have a relatively conserved genetic architecture in both domesticated and wild systems, and provide a foundation for understanding the association between recombination loci and individual fitness within this population.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/crossover" class="hw-term hw-article-keyword hw-article-keyword-crossover" rel="nofollow">crossover</a></li><li class="kwd"><a href="/keyword/genome-wide-association-study" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study" rel="nofollow">genome-wide association study</a></li><li class="kwd"><a href="/keyword/genomic-relatedness" class="hw-term hw-article-keyword hw-article-keyword-genomic-relatedness" rel="nofollow">genomic relatedness</a></li><li class="kwd"><a href="/keyword/heritability" class="hw-term hw-article-keyword hw-article-keyword-heritability" rel="nofollow">heritability</a></li><li class="kwd"><a href="/keyword/meiotic-recombination" class="hw-term hw-article-keyword hw-article-keyword-meiotic-recombination" rel="nofollow">meiotic recombination</a></li><li class="kwd"><a href="/keyword/red-deer" class="hw-term hw-article-keyword hw-article-keyword-red-deer" rel="nofollow">red deer</a></li></ul><p id="p-3">Meiotic recombination (or crossing-over) is a fundamental feature of sexual reproduction and an important driver of diversity in eukaryotic genomes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Felsenstein, 1974</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barton and Charlesworth, 1998</a>). It has several benefits: it ensures the proper disjunction of homologous chromosomes during meiosis (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hassold and Hunt, 2001</a>), prevents mutation accumulation (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Muller, 1964</a>) and generates novel haplotypes, increasing the genetic variance for fitness and increasing the speed and degree to which populations respond to selection (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hill and Robertson, 1966</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Battagin <em>et al.</em> 2016</a>). However, recombination can also come at a cost: it requires the formation of DNA double strand breaks which increase the risk of local mutation and chromosomal rearrangements (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Inoue and Lupski, 2002</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arbeithuber <em>et al.</em> 2015</a>); it can also break up favorable allele combinations previously built up by selection, reducing the mean fitness of successive generations (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Barton and Charlesworth, 1998</a>). Therefore, as the relative costs and benefits of recombination vary within different selective contexts, it is expected that recombination rates should vary within and between populations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Burt, 2000</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Otto and Lenormand, 2002</a>). Indeed, recent studies have shown that recombination rates can vary within and between chromosomes (<em>i.e.</em>, recombination “hotspots”; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2005</a>), individuals (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kong <em>et al.</em> 2004</a>), populations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dumont <em>et al.</em> 2011</a>) and species (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stapley <em>et al.</em> 2017</a>).</p><p id="p-4">Genomic studies in humans, cattle, sheep and mice have shown that variation in recombination rate is often heritable, and may have a conserved genetic architecture (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kong <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ma <em>et al.</em> 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Johnston <em>et al.</em> 2016</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Petit <em>et al.</em> 2017</a>). The loci <em>RNF212</em>, <em>REC8</em> and <em>HEI10</em>, among others, have been identified as candidates driving variation in rate, with <em>PRDM9</em> driving recombination hotspot positioning in mammals (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baudat <em>et al.</em> 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baker <em>et al.</em> 2017</a>). This oligogenic architecture suggests that recombination rates and landscapes have the potential to evolve rapidly under different selective scenarios, in turn affecting the rate at which populations respond to selection (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Barton and Charlesworth, 1998</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Burt, 2000</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Otto and Barton, 2001</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gonen <em>et al.</em> 2017</a>). However, it remains unclear how representative the above studies are of recombination rate variation and its genetic architecture in natural populations. For example, experimental and domesticated populations tend to be subject to strong selection and have small effective population sizes, both of which have been shown theoretically to indirectly select for increased recombination rates to escape Hill-Robertson interference (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Otto and Barton 2001</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Otto and Lenormand 2002</a>; but see <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Muñoz-Fuentes <em>et al.</em> 2015</a>). Therefore, it may be that prolonged artificial selection results in different recombination dynamics and underlying genetic architectures. As broad recombination patterns are characterized in greater numbers of natural systems (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Johnston <em>et al.</em> 2016</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">2017</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Theodosiou <em>et al.</em> 2016</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kawakami <em>et al.</em> 2017</a>), it is clear that broad and fine-scale recombination rates and landscapes can vary to a large degree even within closely related taxa (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Stapley <em>et al.</em> 2017</a>). Therefore, determining the genetic architecture of recombination rate in non-model, natural systems are key to elucidating the broad evolutionary drivers of recombination rate variation and quantifying its costs and benefits at the level of the individual.</p><p id="p-5">In this study, we investigate the genetic basis of recombination rate variation in a wild population of red deer (<em>Cervus elaphus</em>) on the island of Rùm, Scotland (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Clutton-Brock <em>et al.</em> 1982</a>). This population has been subject to a long term study since the early 1970s, with extensive pedigree and genotype information for ∼2,600 individuals at &gt;38,000 SNPs (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Huisman <em>et al.</em> 2016</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a>). We use this dataset to identify autosomal crossover rates and their genetic architecture in &gt;1,300 individuals. The aims of the study are to: (a) determine which common environmental and individual effects, such as age, sex and birth year affect individual recombination rates; (b) determine if recombination rate is heritable; and (c) identify genomic regions that are associated with recombination rate variation. Addressing these objectives will provide a foundation for future studies investigating the association between the genetic architecture of recombination rate and individual fitness, to determine how this trait evolves within contemporary natural populations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials And Methods</h2><div id="sec-2" class="subsection"><h3>Study population and genomic dataset</h3><p id="p-6">The study population of red deer is situated in the North Block of the Isle of Rùm, Scotland (57°02‘N, 6°20‘W) and has been subject to individual monitoring since 1971 (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Clutton-Brock <em>et al.</em> 1982</a>). Research was conducted following approval of the University of Edinburgh’s Animal Welfare and Ethical Review Body and under appropriate UK Home Office licenses. DNA was extracted from neonatal ear punches, cast antlers and post-mortem tissue (see <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Huisman <em>et al.</em> 2016</a> for full details). DNA samples from 2880 individuals were genotyped at 50,541 SNP loci on the Cervine Illumina BeadChip (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brauning <em>et al.</em> 2015</a>) using an Illumina genotyping platform (Illumina Inc., San Diego, CA, USA). SNP genotypes were scored using Illumina GenomeStudio software, and quality control was carried out using the <em>check.marker</em> function in GenABEL v1.8-0 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aulchenko <em>et al.</em> 2007</a>) in R v3.3.2, with the following thresholds: SNP genotyping success &gt;0.99, SNP minor allele frequency &gt;0.01, and ID genotyping success &gt;0.99, with 38,541 SNPs and 2,631 IDs were retained. There were 126 pseudoautosomal SNPs identified on the X chromosome (<em>i.e.</em>, markers showing autosomal inheritance patterns; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a>). Heterozygous genotypes within males at non-pseudoautosomal X-linked SNPs were scored as missing. A pedigree of 4,515 individuals has been constructed using microsatellite and SNP data using the software Sequoia (see <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huisman, 2017</a>). The genomic inbreeding coefficient (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-2.gif"/></noscript></span></span>), was calculated for each deer in the software GCTA v1.24.3 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yang <em>et al.</em> 2011</a>), using information for all autosomal SNP loci passing quality control. A linkage map of 38,083 SNPs has previously been constructed, with marker orders and estimated base-pair positions known for all 33 autosomes (CEL1 to CEL33) and the X chromosome (CEL34) (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a> and data archive doi: 10.6084/m9.figshare.5002562). All chromosomes are acrocentric with the exception of one metacentric autosome (CEL5).</p></div><div id="sec-3" class="subsection"><h3>Quantification of meiotic crossovers</h3><p id="p-7">A standardized sub-pedigree approach was used to identify the positions of meiotic crossovers (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Johnston <em>et al.</em> 2016</a>). The full pedigree was split as follows: for each focal individual (FID) and offspring pair, a sub-pedigree was constructed that included the FID, its mate, parents and offspring (Figure S1), where all five individuals were genotyped on the SNP chip. This pedigree structure allows phasing of SNPs within the FID, characterizing the crossovers occurring in the gamete transferred from the FID to the offspring. All remaining analyses outlined in this section were conducted in the software CRI-MAP v2.504a (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Green <em>et al.</em> 1990</a>) within the R package crimaptools v0.1 (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a>) implemented in R v3.3.2. Mendelian incompatibilities within sub-pedigrees were identified using the <em>prepare</em> function and removed from all affected individuals; sub-pedigrees containing more than 0.1% mismatching loci between parents and offspring were discarded. The <em>chrompic</em> function was used to identify the grand-parental phase of SNP alleles on chromosomes transmitted from the FID to the offspring, and to provide a sex-averaged linkage map. Switches in phase indicated the position of a crossover (Figure S1). Individuals with high numbers of crossovers per gamete (&gt;60) were assumed to have widespread phasing errors and were removed from the analysis; the maximum number of crossovers for an individual was 45 in the remaining dataset.</p><p id="p-8">Errors in determining allelic phase can lead to incorrect calling of double crossovers (<em>i.e.</em>, ≥ 2 crossovers occurring on the same chromosome) over short map distances. To reduce the likelihood of calling false double crossover events, phased runs consisting of a single SNP were recoded as missing (390 out of 7652 double crossovers; Figure S2) and <em>chrompic</em> was rerun. Of the remaining double crossovers, those occurring over distances of ≤ 10cM (as measured by the distance between markers immediately flanking the double crossover) were recoded as missing (170 out of 6959 double crossovers; Figure S2). After this process, 1341 sub-pedigrees passed quality control, characterizing crossovers in gametes transmitted to 482 offspring from 81 unique males and 859 offspring from 256 unique females.</p></div><div id="sec-4" class="subsection"><h3>Genetic architecture of recombination rate variation</h3><div id="sec-5" class="subsection"><h4>Heritability and cross-sex genetic correlation:</h4><p id="p-9">Autosomal crossover count (ACC) was modeled as a trait of the FID. A restricted maximum-likelihood (REML) “animal model” approach (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Henderson, 1975</a>) was used to partition phenotypic variance and examine the effect of fixed effects on ACC; these were implemented in ASReml-R (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Butler <em>et al.</em> 2009</a>) in R v3.3.2. The additive genetic variance was calculated by fitting a genomic relatedness matrix (GRM) constructed for all autosomal markers in GCTA v1.24.3 (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Yang <em>et al.</em> 2011</a>); the GRM was adjusted assuming similar frequency spectra of genotyped and causal loci using the argument <em>–grm-adj 0</em>. There was no pruning of related individuals from the GRM (<em>i.e.</em>, we did not use the <em>–grm-cutoff</em> argument) as there is substantial relatedness within the population, and initial models included parental effects and common environment which controls for effects of shared environments between relatives. ACC was modeled first using a univariate model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-3.gif"/></noscript></span></span>where <em>y</em> is a vector of ACC; <em>X</em> is an incidence matrix relating individual measures to a vector of fixed effects, β; <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-5.gif"/></noscript></span></span> are incidence matrices relating individual measures with additive genetic and random effects, respectively; <em>a</em> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-6.gif"/></noscript></span></span> are vectors of GRM additive genetic and additional random effects, respectively; and <em>e</em> is a vector of residual effects. The narrow-sense heritability <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-7.gif"/></noscript></span></span> was calculated as the ratio of the additive genetic variance to the sum of variance components estimated for all random effects. Model structures were tested with several fixed effects, including sex, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-8.gif"/></noscript></span></span> and FID age; random effects included individual identity (<em>i.e.</em>, permanent environment) to account for repeated measures in the same FID, maternal and paternal identity, and common environment effects of FID birth year and offspring birth year. A dominance GRM constructed in GCTA was also fitted but was not significant and explained &lt;1% of the phenotypic variance, and was therefore not included in the final models. The significance of fixed effects was tested with a Wald test, and the significance of random effects was calculated using likelihood-ratio tests (LRT, defined as 2 × the difference in log-likelihoods between the two models, distributed as <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-9.gif"/></noscript></span></span> with 1 degree of freedom) between models with and without the focal random effect. <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-10.gif"/></noscript></span></span> and individual identity were retained in all models, irrespective of statistical significance, to account for possible underestimation of ACC and pseudoreplication, respectively. As the variance in recombination rates differed between the sexes, models were also run for each sex separately.</p><p id="p-10">Bivariate models of male and female ACC were run to determine whether additive genetic variation was associated with sex-specific variation and the degree to which this was correlated between the sexes. The additive genetic correlation <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-11.gif"/></noscript></span></span> was determined using the CORGH error-structure function in ASReml-R (correlation with heterogeneous variances) with <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-12.gif"/></noscript></span></span> set to be unconstrained. Model structure was otherwise the same as for univariate models. To determine whether genetic correlations were significantly different from 0 and 1, the unconstrained model was compared with models where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-13.gif"/></noscript></span></span> was fixed at values of 0 or 0.999. Differences in additive genetic variance in males and females were tested by constraining both to be equal values using the CORGV error-structure function in ASReml-R. Models then were compared using LRTs with 1 degree of freedom.</p></div><div id="sec-6" class="subsection"><h4>Genome-wide association study:</h4><p id="p-11">Genome-wide association studies (GWAS) of ACC were conducted using the function <em>rGLS</em> in the R library RepeatABEL v1.1 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rönnegård <em>et al.</em> 2016</a>) implemented in R v3.3.2. This function accounts for population structure by fitting the GRM as a random effect, and allows fitting of repeated phenotypic measures per individual. Models were run including sex and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-14.gif"/></noscript></span></span> as fixed effects; sex-specific models were also run. Association statistics were corrected for inflation due to population stratification that was not captured by the GRM, by dividing them by the genomic control parameter λ, which was calculated as the observed median <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-15.gif"/></noscript></span></span> statistic divided by the null expectation median <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-16.gif"/></noscript></span></span> statistic (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Devlin <em>et al.</em> 1999</a>). The significance threshold after multiple testing was calculated using a linkage disequilibrium (LD) based approach in the software <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-17.gif"/></noscript></span></span> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Moskvina and Schmidt, 2008</a>) specifying a sliding window of 50 SNPs. The effective number of tests was calculated as 35,264, corresponding to a P value of <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-18.gif"/></noscript></span></span> at <em>α</em> = 0.05. GWAS of ACC included the X chromosome and 458 SNP markers of unknown position. It is possible that some SNPs may show an association with ACC if they are in LD with polymorphic recombination hotspots (<em>i.e.</em>, associations in <em>cis</em>), rather than SNPs associated with recombination rate globally across the genome (<em>i.e.</em>, associations in <em>trans</em>). Therefore, we repeated the GWAS modeling <em>trans</em> variation only, by examining associations between each SNP and ACC, minus the crossovers that occurred on the same chromosome as the SNP. For example, if the focal SNP occurred on linkage group 1, association was tested with ACC summed over linkage groups 2-33. Marker positions are known relative to the cattle genome vBTA_vUMD_3.1; in cases of significant associations with recombination rate, gene annotations and positions were obtained from Ensembl (Cattle gene build ID BTA_vUMD_3.1.89). LD was calculated between loci in significantly associated regions using the allelic correlation <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-19.gif"/></noscript></span></span> in the R package LDheatmap v0.99-2 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Shin <em>et al.</em> 2006</a>) in R v3.3.2.</p></div><div id="sec-7" class="subsection"><h4>Regional heritability analysis:</h4><p id="p-12">As a single locus approach, GWAS has reduced power to detect variants with small effect sizes and/or low linkage disequilibrium with causal mutations (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Yang <em>et al.</em> 2011</a>). Partitioning additive genetic variance within specific genomic regions (<em>i.e.</em>, a regional heritability approach) incorporates haplotype effects and determines the proportion of phenotypic variance explained by defined regions. The additive genetic variance was partitioned across all autosomes in sliding windows of 20 SNPs (with an overlap of 10 SNPs) as follows (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nagamine <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bérénos <em>et al.</em> 2015</a>):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-20.gif"/></noscript></span></span>where <em>y</em> is a vector of ACC; <em>X</em> is an incidence matrix relating individual measures to a vector of fixed effects, β; <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-21.gif"/></noscript></span></span> is a vector of additive genetic effects explained by autosomal genomic region in window <em>i</em>; <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-22.gif"/></noscript></span></span> is the vector of the additive genetic effects explained by all autosomal markers not in window <em>i</em>; <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-24.gif"/></noscript></span></span> are incidence matrices relating individual measures with additive genetic effects for the focal window and the rest of the genome, respectively; <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-25.gif"/></noscript></span></span> is an incidence matrix relating individual measures with additional random effects, where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-26.gif"/></noscript></span></span> is a vector of additional random effects; and <em>e</em> is a vector of residual effects. The mean window size was 1.29 ± 0.32 Mb. Models were implemented in ASReml-R (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Butler <em>et al.</em> 2009</a>) in R v3.3.2. GRMs were constructed in the software GCTA v1.24.3 with the argument <em>–grm-adj 0</em> (<a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Yang <em>et al.</em> 2011</a>). The significance of additive genetic variance for window <em>i</em> was tested by comparing models with and without the <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-27.gif"/></noscript></span></span> term with LRT (<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-28.gif"/></noscript></span></span>). We attempted to model the X-chromosome GRMs as calculated in GCTA omitting the <em>–grm-adj 0</em> argument. However, X-chromosome models had negative pivots in the GRMs, possibly a result of small sample and window sizes, and/or due to the fact that we could not modify the assumed frequency spectra of causal and typed loci as we could for the autosomal markers. To correct for multiple testing, a Bonferroni approach was used, taking the number of windows and dividing by 2 to account for window overlap; the threshold P-value was calculated as <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-29.gif"/></noscript></span></span> at <em>α</em> = 0.05. In the most highly associated region, this analysis was repeated for windows of 20, 10 and 6 SNPs in sliding windows overlapping by <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-30.gif"/></noscript></span></span> SNPs in order to fine map the associated regions. This was carried out from approximately 5MB before and after the significant region.</p></div><div id="sec-8" class="subsection"><h4>Accounting for sample size difference between males and females:</h4><p id="p-13">Sample sizes within this dataset are markedly different between males and females (see above and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). A consequence of this may be that there is lower power to detect associations with male recombination rate. We repeated the heritability and GWAS analyses in sampled datasets of the same size within each sex. Briefly, 482 recombination rate measures (representing the total number in males) were sampled with replacement within the male and female datasets, and the animal model and GWAS analyses were repeated in the sampled dataset. This process was repeated 100 times, with sampling carried out in R v3.3.2. The observed and simulated heritabilities compared to see how often a similar results would be obtained. This was repeated for association at the most highly associated GWAS SNPs and regional heritability regions. The differences between the mean simulated values in each sex were investigated using a Welch two-sample <em>t</em>-test assuming unequal variances.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490343/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490343/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Data set information and animal model results for autosomal crossover count (ACC). Numbers in parentheses are the standard error, except for <em>Mean</em>, which is the standard deviation. <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-31.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-32.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-33.gif"/></noscript></span></span> are the number of ACC measures, the number of focal individuals (FIDS) and the total number of crossovers in the dataset. The mean ACC was calculated from the raw data. <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-34.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-35.gif"/></noscript></span></span> are the phenotypic variance and additive genetic variance, respectively. <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-36.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-37.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-38.gif"/></noscript></span></span> are the narrow-sense heritability, the permanent environment effect, and the residual effect, respectively; all are calculated as the proportion of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-39.gif"/></noscript></span></span> that they explain. The additive genetic components were modeled using genomic relatedness matrices. <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-40.gif"/></noscript></span></span> is the significance of the <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T1/embed/mml-math-41.gif"/></noscript></span></span> term in the model as determined using a likelihood ratio test</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h4>Haplotyping and effect size estimation:</h4><p id="p-14">Haplotype construction was carried out to examine haplotype variation within regions significantly associated with recombination rate variation in the regional heritability analysis. SNP data from deer linkage group 12 was phased using SHAPEIT v2.r837 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Delaneau <em>et al.</em> 2013</a>), specifying the linkage map positions and recombination rates for each locus. This analysis used pedigree information with the <em>–duohmm</em> flag to allow the use of pedigree information in the phasing process (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">O'Connell <em>et al.</em> 2014</a>). Haplotypes were then extracted for the most significant window from the regional heritability analysis.</p><p id="p-15">Effect sizes on ACC for the top GWAS SNPs were estimated using animal models in ASReml-R; SNP genotype was fit as a fixed factor, with pedigree relatedness fit as a random effect to account for the remaining additive genetic variance. To determine the effect sizes on ACC for the regional heritability analysis, animal models were run as follows: for a given haplotype, A, its effect was estimated relative to all other haplotypes combined, <em>i.e.</em>, treating them as a single allele, B, by fitting genotypes A/A, A/B and B/B as a fixed factor. This was repeated for each haplotype allele where more than 10 copies were present in the full dataset.</p></div></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-16">Raw data are publicly archived at doi: <a href="https://doi.org/10.6084/m9.figshare.5002562">https://doi.org/10.6084/m9.figshare.5002562</a> (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a>). Code for the analysis is archived at <a href="https://github.com/susjoh/Deer_Recombination_GWAS">https://github.com/susjoh/Deer_Recombination_GWAS</a>. Supplementary data files are archived in the GSA figshare portal. Table S2 contains raw and quality controlled ACCs per individual/chromosome/meiosis. Table S3 contains the ACC GWAS results for both sexes and in males and females only. Table S4 contains the ACC regional heritability results in both sexes and in males and females only. Table S5 contains the ACC regional heritability results in the most highly associated region on deer linkage group CEL12. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6166364">https://doi.org/10.25387/g3.6166364</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Variation and heritability in autosomal crossover count</h3><p id="p-17">Autosomal crossover count (ACC) was significantly higher in females than in males, where females had 4.32 ± 0.41 more crossovers per gamete (<em>Z</em> = 10.57, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-51.gif"/></noscript></span></span> &lt;0.001; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>); there was no effect of FID age or inbreeding on ACC (<em>P</em> &gt; 0.05, Table S1). Females had significantly higher phenotypic variance in ACC than males (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-52.gif"/></noscript></span></span> = 32.02 and 15.33, respectively; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). ACC was heritable in both sexes combined (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-53.gif"/></noscript></span></span> = 0.13, SE = 0.05, <em>P</em> = 0.002) and within females only (<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-54.gif"/></noscript></span></span> = 0.11, SE = 0.06, <em>P</em> = 0.033), but was not heritable in males alone (<em>P</em> &gt; 0.05; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The remaining phenotypic variance was explained by the residual error term, and there was no variance explained by the permanent environment effect, birth year, year of gamete transmission, or parental identities of the FID in any model (<em>P</em> &gt; 0.05). Heritability estimates from the GRM were similar to those estimated using pedigree relatedness, which were 0.14 (SE = 0.052), 0.13 (SE = 0.064) and 0.10 (SE = 0.110) for both sexes, females and males, respectively. Bivariate models of ACC between the sexes indicated that the genetic correlation (<span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-55.gif"/></noscript></span></span>) between males and females was 0.346, but was not significantly different from zero or one (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-56.gif"/></noscript></span></span> &gt;0.05). This may be due to the relatively small sample size of this dataset resulting in a large standard error around the <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-57.gif"/></noscript></span></span> estimate, or the fact that ACC was not heritable in males. Sampling of 482 measures from each sex showed no difference in heritability estimates between the sexes, indicating reduced power to quantify heritable variation in the smaller male dataset (t = 0.242, <em>P</em> = 0.810, Figure S3). Raw and quality controlled ACC counts and positions for each FID and offspring combination per chromosome are provided in Table S2.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of ACCs in the raw data for females and males." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1236748836" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of ACCs in the raw data for females and males.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2265/F1.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/7/2265/F1.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490346" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Distribution of ACCs in the raw data for females and males.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genetic architecture of autosomal crossover count</h3><div id="sec-14" class="subsection"><h4>Genome-wide association study:</h4><p id="p-19">No SNPs were significantly associated with ACC at the genome-wide level (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and Table S3). The most highly associated SNP in both sexes was <em>cela1_red_10_26005249</em> on deer linkage group 12 (CEL12), corresponding to position 26,005,249 on cattle chromosome 10 (BTA10). This marker was also the most highly associated SNP when considering recombination in <em>trans</em>, indicating that this region affects ACC across the genome (Table S3). The observed association was primarily driven by female ACC (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). In females, the most highly associated SNP was <em>cela1_red_10_25661750</em> on CEL12, corresponding to position 25,661,750 on BTA10. For both SNPs, sampling of 482 measures from each sex showed that the observed associations were significantly higher in females than in males when considering the same sample size (<em>cela1_red_10_25661750</em>: t = 18.60, <em>P</em> &lt; 0.001; <em>cela1_red_10_26005249</em>: t = 4.89, <em>P</em> &lt; 0.001; Figure S4). Based on its position relative to the cattle genome, <em>cela1_red_10_26005249</em> was ∼600bp upstream of an olfactory receptor <em>OR5AU1</em> and ∼24kb downstream from a gene of unknown function (ENSBTAG00000011396). There were four candidate genes within 1Mb of both loci, including <em>TOX4</em>, <em>CHD8</em>, <em>SUPT16H</em> and <em>CCNB1IP1</em> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>; see Discussion). Similar results were obtained when considering recombination rate on all chromosomes excluding that on which the SNP occurred, indicating that all associations affect recombination rate variation in <em>trans</em> across the genome (Table S3).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plot of genome-wide association of autosomal crossover count (ACC) for (A) all deer, (B) females only and (C) males only. The dashed line is the genome-wide significance threshold equivalent to P &lt; 0.05. The left-hand plots show association relative to the estimated genomic positions on deer linkage groups from Johnston et al. (2017). Points have been color coded by chromosome. The right-hand plots show the distribution of observed  values against those under the null expectation. Association statistics have been corrected for the genomic control inflation factor &#x3BB; . Underlying data are provided in Table S3 and sample sizes are given in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1236748836" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Manhattan plot of genome-wide association of autosomal crossover count (ACC) for (A) all deer, (B) females only and (C) males only. The dashed line is the genome-wide significance threshold equivalent to &lt;em&gt;P&lt;/em&gt; &lt; 0.05. The left-hand plots show association relative to the estimated genomic positions on deer linkage groups from Johnston &lt;em&gt;et al.&lt;/em&gt; (2017). Points have been color coded by chromosome. The right-hand plots show the distribution of observed &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-58&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values against those under the null expectation. Association statistics have been corrected for the genomic control inflation factor &lt;em&gt;&#x3BB;&lt;/em&gt; . Underlying data are provided in Table S3 and sample sizes are given in Table 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2265/F2.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/7/2265/F2.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490183" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Manhattan plot of genome-wide association of autosomal crossover count (ACC) for (A) all deer, (B) females only and (C) males only. The dashed line is the genome-wide significance threshold equivalent to <em>P</em> &lt; 0.05. The left-hand plots show association relative to the estimated genomic positions on deer linkage groups from <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> (2017)</a>. Points have been color coded by chromosome. The right-hand plots show the distribution of observed <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/F2/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/F2/embed/mml-math-58.gif"/></noscript></span></span> values against those under the null expectation. Association statistics have been corrected for the genomic control inflation factor <em>λ</em> . Underlying data are provided in Table S3 and sample sizes are given in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490386/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490386/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The top five most significant hits from a genome-wide association study of ACC in (A) Both sexes, (B) Females only and (C) Males only. No SNPs reached the genome-wide significance of <em>P</em> = <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T2/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T2/embed/mml-math-59.gif"/></noscript></span></span> The SNP locus names indicate the position of the SNPs relative to the cattle genome assembly vBTA_vUMD_3.1 (indicated by <em>Chromosome_Position</em>). Linkage groups and map positions (in centiMorgans, cM) are from <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> (2017)</a>. A and B are the reference alleles. Effect B is the estimated effect and standard error of the B allele as estimated in RepeatABEL (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Rönnegård <em>et al.</em> 2016</a>). P-values have been corrected for the genomic inflation parameter λ. Full results are available in Table S3</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>Regional heritability analysis:</h4><p id="p-21">The genome-wide regional heritability analysis of ACC showed a significant association in both sexes and in females only with a ∼2.94Mb region on CEL12 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The most highly associated window (∼1.36 Mb) within this region contained 42 genes, including REC8 meiotic recombination protein (<em>REC8</em>; 20,810,610 - 20,817,662 bp on BTA10). Detailed examination of this region in sliding windows of 6, 10 and 20 SNPs found the highest association at a 10 SNP window of ∼463kb containing 36 genes, including <em>REC8</em> (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). This region explained all heritable variation in recombination rate, with regional heritability estimates of 0.143 (SE = 0.053) and 0.146 (SE = 0.045) for all deer and females only, respectively. The sex-specific effect was supported by sampling of 482 measures, where females had consistently higher associations than in males (t = 19.03, <em>P</em> &lt; 0.001, Figure S5). The total significant region after detailed examination was ∼3.01Mb wide, flanked by SNPs <em>cela1_red_10_18871213</em> and <em>cela1_red_10_21878407</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> &amp; Table S5) and containing ∼87 genes. This wider region contained the protein coding region for ring finger protein 212B (<em>RNF212B</em>; 21,466,337 - 21,494,696 bp on BTA10), a homolog of <em>RNF212</em>, which has been directly implicated in synapsis and crossing-over during meiosis in mice (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Reynolds <em>et al.</em> 2013</a>). Genetic variants at both <em>RNF212B</em> and <em>RNF212</em> have been associated with recombination rate variation in humans, cattle and sheep (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kong <em>et al.</em> 2008</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Ma <em>et al.</em> 2015</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Johnston <em>et al.</em> 2016</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Petit <em>et al.</em> 2017</a>). While this region was close to the most highly associated SNPs from the genome-wide association study, there was no overlap between the two analyses, with the mostly highly associated regions separated by an estimated ∼5.5Mb (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The mean <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-61.gif"/></noscript></span></span> LD between the top regional heritability window and the top GWAS SNPs was 0.258 for <em>cela1_red_10_25661750</em> and 0.276 for <em>cela1_red_10_26005249</em>, with the top <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-62.gif"/></noscript></span></span> of 0.665 observed between the SNPs <em>cela1_red_10_21807996</em> and <em>cela1_red_10_26005249</em> (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regional heritability plot of association of autosomal crossover count for (A) all deer, (B) females only and (C) males only. Each point represents a sliding window of 20 SNPs with an overlap of 10 SNPs. The dashed line is the genome-wide significance threshold equivalent to P &lt; 0.05 as calculated using Bonferroni. Lines have been color coded by chromosome. Underlying data are provided in Table S4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1236748836" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Regional heritability plot of association of autosomal crossover count for (A) all deer, (B) females only and (C) males only. Each point represents a sliding window of 20 SNPs with an overlap of 10 SNPs. The dashed line is the genome-wide significance threshold equivalent to &lt;em&gt;P&lt;/em&gt; &lt; 0.05 as calculated using Bonferroni. Lines have been color coded by chromosome. Underlying data are provided in Table S4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2265/F3.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/7/2265/F3.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490217" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Regional heritability plot of association of autosomal crossover count for (A) all deer, (B) females only and (C) males only. Each point represents a sliding window of 20 SNPs with an overlap of 10 SNPs. The dashed line is the genome-wide significance threshold equivalent to <em>P</em> &lt; 0.05 as calculated using Bonferroni. Lines have been color coded by chromosome. Underlying data are provided in Table S4.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490243/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490243/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The most significant hits from a regional heritability analysis of ACC in (A) Both sexes, (B) Females only and (C) Males only. Sliding windows were 20 SNPs wide with an overlap of 10 SNPs. Lines in italics are the most highly associated regions from detailed examination of significant regions - in each case these are for 10 SNP windows. The <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T3/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/T3/embed/mml-math-63.gif"/></noscript></span></span> and P values are for likelihood ratio test comparisons between models with and without a genomic relatedness matrix for that window; values in bold type are significant the the genome-wide level. The SNP locus names indicate the position of the SNPs relative to the cattle genome assembly vBTA_vUMD_3.1 (indicated by <em>Chromosome_Position</em>). Full results are available in Tables S4 &amp; S5</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detailed figure of genes, association statistics and linkage disequilibrium patterns at the most highly associated region on on CEL12 (homologous to BTA10) for all deer of both sexes. All X-axis positions are given relative to the cattle genome vBTA_vUMD_3.1. The top panel shows protein coding regions, with annotation for candidate loci. The central panel shows the results for the regional heritability analysis (where lines represents a sliding windows of 6, 10 and 20 SNPs with an overlap of n-1 SNPs) and the genome-wide association study (where points indicate single SNP associations). The dashed lines are the genome-wide significance thresholds (green = regional heritability, black = genome-wide association). The checked shaded area shows the position of the T cell receptor alpha/delta locus (see Discussion). Underlying data are provided in Tables S3 &amp; S5. The lower panel shows linkage disequilibrium between each loci using allelic correlations ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1236748836" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Detailed figure of genes, association statistics and linkage disequilibrium patterns at the most highly associated region on on CEL12 (homologous to BTA10) for all deer of both sexes. All X-axis positions are given relative to the cattle genome vBTA_vUMD_3.1. The top panel shows protein coding regions, with annotation for candidate loci. The central panel shows the results for the regional heritability analysis (where lines represents a sliding windows of 6, 10 and 20 SNPs with an overlap of n-1 SNPs) and the genome-wide association study (where points indicate single SNP associations). The dashed lines are the genome-wide significance thresholds (green = regional heritability, black = genome-wide association). The checked shaded area shows the position of the T cell receptor alpha/delta locus (see Discussion). Underlying data are provided in Tables S3 &amp; S5. The lower panel shows linkage disequilibrium between each loci using allelic correlations (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-66&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/7/2265/F4.medium.gif" width="392" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/7/2265/F4.medium.gif" width="392" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/7/2265/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490191" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-23" class="first-child">Detailed figure of genes, association statistics and linkage disequilibrium patterns at the most highly associated region on on CEL12 (homologous to BTA10) for all deer of both sexes. All X-axis positions are given relative to the cattle genome vBTA_vUMD_3.1. The top panel shows protein coding regions, with annotation for candidate loci. The central panel shows the results for the regional heritability analysis (where lines represents a sliding windows of 6, 10 and 20 SNPs with an overlap of n-1 SNPs) and the genome-wide association study (where points indicate single SNP associations). The dashed lines are the genome-wide significance thresholds (green = regional heritability, black = genome-wide association). The checked shaded area shows the position of the T cell receptor alpha/delta locus (see Discussion). Underlying data are provided in Tables S3 &amp; S5. The lower panel shows linkage disequilibrium between each loci using allelic correlations (<span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/F4/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/F4/embed/mml-math-66.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">A second region on linkage group 32 almost reached genome-wide significance in the regional heritability analysis, corresponding to the region ∼38.7 - 41.3Mb on cattle chromosome 27. This region contained the locus topoisomerase (DNA) II beta (<em>TOP2B</em>); inhibitors of this gene lead to defects in chromosome segregation and heterochromatin condensation during meiosis I in mice, <em>Drosophila melanogaster</em> and <em>Caenorhabditis elegans</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gómez <em>et al.</em> 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hughes and Hawley 2014</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jaramillo-Lambert <em>et al.</em> 2016</a>; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> and Table S4). Full results for the regional heritability analyses are provided in Tables S4 and S5.</p></div><div id="sec-16" class="subsection"><h4>Effect size estimation:</h4><p id="p-25">At the most highly associated GWAS SNP, <em>cela1_red_10_26005249</em>, carrying one or two copies of the G allele conferred 3.3 to 3.9 fewer crossovers per gamete in females (Wald <em>P</em> &lt; 0.001) and 1.8 - 2.8 fewer crossovers per gamete in males (<em>P</em> = 0.009; <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The most highly associated SNP in females, <em>cela1_red_10_25661750</em>, had a significant effect on ACC in females (<em>P</em> &lt; 0.001) but not in males (<em>P</em> &gt; 0.05; <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). This locus conferred 2.03 more crossovers in A/G females and 13.68 more in G/G females; however, the latter category contained 7 unique measures in only two individuals, and so this estimate is likely to be subject to large sampling error.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490329/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490329/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Effect sizes for the most highly associated GWAS SNPs and for the AGGAGAGAAG haplotype at the most highly associated regional heritability region. Models were run for each sex separately and included a pedigree relatedness as a random effect. Count and ID Count indicate the number of ACC measures and the number of unique individuals for each genotype, respectively. Wald.P indicates the P-value for a Wald test of genotype as a fixed effect</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">A total of 17 haplotypes in the 10 SNP region spanning cela1_red_10_20476277 and cela1_red_10_20939342 had more than ten copies in unique individual females (Table S6). Of these, two haplotypes, AGGAGAGAAG and AGAGAAGAGA, had a significant effect on ACC relative to all other haplotypes (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> and Table S6, Figure S6). Haplotype AGGAGAGAAG increased female ACC by 2.4 crossovers per gamete in heterozygotes (<em>P</em> &lt; 0.001); homozygotes for the haplotype were rare (13 measures in 4 individuals) and so the large effect size estimate was again likely to be subject to large sampling effects (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). The haplotype AGAGAAGAGA reduced female ACC by 2.2 crossovers per gamete in heterozygous individuals (<em>P</em> &lt; 0.05; <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>). The <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-67.gif"/></noscript></span></span> LD between haplotype AGGAGAGAAG and the two most highly associated GWAS SNPs was 0.464 and 0.885 for cela1_red_10_26005249 and cela1_red_10_25661750, respectively; for haplotype AGAGAAGAGA, it was 0.229 and 0.036 for cela1_red_10_26005249 and cela1_red_10_25661750, respectively.</p></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-27">In this study, we have shown that autosomal crossover count (ACC) is 1.2× higher in red deer females than in males, with females exhibiting higher phenotypic and additive genetic variance for this trait; ACC was not significantly heritable in males. Almost all genetic variation in females was explained by a ∼7Mb region on deer linkage group 12. This region contained several candidate genes, including <em>RNF212B</em> and <em>REC8</em>, which have previously been implicated in recombination rate variation in other mammal species, including humans, mice, cattle and sheep (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kong <em>et al.</em> 2008</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Reynolds <em>et al.</em> 2013</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Ma <em>et al.</em> 2015</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Johnston <em>et al.</em> 2016</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Petit <em>et al.</em> 2017</a>). Here, we discuss in detail the genetic architecture of individual recombination rate, candidate genes underlying heritable variation, sexual-dimorphism in this trait and its architecture, and the conclusions and implications of our findings for other studies of recombination in the wild.</p><div id="sec-18" class="subsection"><h3>The genetic architecture of individual recombination rate</h3><p id="p-28">Using complementary trait mapping approaches, we identified a ∼7Mb region on deer linkage group 12 (homologous to cattle chromosome 10) associated with ACC. The most highly associated GWAS region occurred between ∼25.6 and 26Mb (relative to the cattle genome position), although this association was not significant at the genome-wide level. The most highly associated regional heritability region occurred between ∼20.5 and 20.9MB, around 5Mb away from the top GWAS hits (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>); association at this region was significant at the genome-wide level and explained almost all of the heritable variation in ACC in both sexes and in females only. Most variation in mean ACC was attributed to two haplotypes within this region (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a> and Table S6; Figure S6).</p><p id="p-29">At present, it is not clear why the results of the two analyses occur in close vicinity, yet do not overlap. Assuming homology with humans, cattle, sheep and mice (Ensembl release 91, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zerbino <em>et al.</em> 2018</a>), the two regions are separated by the highly repetitive T-cell receptor alpha/delta variable (TRAV/DV) locus, which may contain up to 400 TRAV/DV genes in cattle (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Reinink and Van Rhijn 2009</a>; <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>). This region is of an unknown size in deer; relative to the cattle genome, these regions are separated by 4.72Mb, but the deer linkage map distance is estimated as 1.86 centiMorgans (cM). The sex-averaged genome-wide recombination rate in deer is ∼1.04cM/Mb, suggesting this genomic region may be shorter in deer (<a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a>) and that these two regions are in closer vicinity This is supported by both the linkage map distance and patterns of linkage disequilibrium between the associated loci, particularly at the associated haplotypes (see Results &amp; <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>). In addition, the small sample size used in the current study may result in increased sensitivity to sampling effects and bias in the estimation of the relative contribution of SNPs to the trait mean (GWAS) or variance (Regional heritability). Further investigation with higher samples sizes, whole genome sequencing approaches and improved genome assembly may allow more accurate determination of the most likely candidate genes and potential causal mutations (coding or regulatory) within this species.</p></div><div id="sec-19" class="subsection"><h3>Candidate genes for recombination rate variation</h3><div id="sec-20" class="subsection"><h4>Regional heritability analysis:</h4><p id="p-30">The most highly associated region in the regional heritability analysis contained the gene <em>REC8</em>, the protein of which is required for the separation of sister chromatids during meiosis (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Parisi <em>et al.</em> 1999</a>). It also contained <em>RNF212B</em>, a paralogue of <em>RNF212</em>. <em>RNF212</em> has been associated with recombination rate variation in humans, cattle and sheep (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Kong <em>et al.</em> 2008</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sandor <em>et al.</em> 2012</a>; <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Johnston <em>et al.</em> 2016</a>; <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Petit <em>et al.</em> 2017</a>); the <em>REC8/RNF212B</em> region is associated with recombination rate in cattle, and has a large effect size on ACC phenotype than <em>RNF212</em> in this species (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Sandor <em>et al.</em> 2012</a>; <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Ma <em>et al.</em> 2015</a>). A second region on deer linkage group 32 almost reached genome-wide significance (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a> and Table S4). This region was relatively gene-poor, but contained ∼6 genes, including the candidate topoisomerase (DNA) II beta (<em>TOP2B</em>): inhibitors of this gene lead to defects in chromosome segregation and heterochromatin condensation during meiosis I in mice, <em>Drosophila melanogaster</em> and <em>Caenorhabditis elegans</em> (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2013</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gómez <em>et al.</em> 2014</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hughes and Hawley, 2014</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jaramillo-Lambert <em>et al.</em> 2016</a>). No association was observed at the region homologous to <em>RNF212</em> (predicted to be at position ∼109.2Mb on cattle chromosome 6, corresponding to ∼57.576cM on deer linkage group 6) for the GWAS or regional heritability analysis.</p></div><div id="sec-21" class="subsection"><h4>Genome-wide association study (GWAS):</h4><p id="p-31">Examination of annotated regions within 500kb of either side of the most significant GWAS SNPs identified three genes, TOX High Mobility Group Box Family Member 4 (<em>TOX4</em>), Chromodomain Helicase DNA Binding Protein 8 (<em>CHD8</em>) and SPT16 Homolog Facilitates Chromatin Remodelling Subunit (<em>SUPT16H</em>). These genes are involved in with chromatin binding and structure (<em>SP16H</em>, <em>TOX4</em>), histone binding (<em>CHD8</em>, <em>SUPT16H</em>), nucleosome organization (<em>SP16H</em>) and cell cycle transition (<em>TOX4</em>). One of these genes, <em>SUPT16H</em>, interacts with NIMA related kinase 9 (<em>NEK9</em>), which is involved with meiotic spindle organization, chromosome alignment and cell cycle progression in mice (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yang <em>et al.</em> 2012</a>) and is a strong candidate locus for crossover interference in cattle (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wang <em>et al.</em> 2016</a>). The SNP <em>cela1_red_10_26005249</em> was ∼825kb from Cyclin B1 Interacting Protein 1 (<em>CCNB1IP1</em>), also known as Human Enhancer Of Invasion 10 (<em>HEI10</em>), which interacts with <em>RNF212</em> to allow recombination to progress into crossing-over in mice (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Qiao <em>et al.</em> 2014</a>) and <em>Arabidopsis</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chelysheva <em>et al.</em> 2012</a>); this locus is also associated with recombination rate variation in humans (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Kong <em>et al.</em> 2014</a>).</p></div><div id="sec-22" class="subsection"><h4>Sexual dimorphism in genetic architecture of recombination rate:</h4><p id="p-32">The results of this analysis suggest that there is sexual dimorphism in the genetic architecture of recombination rate variation in deer. Higher female recombination rates are typical in mammals (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brandvain and Coop, 2012</a>) and in this system is driven by higher female recombination rates near centromeric regions (<a id="xref-ref-31-10" class="xref-bibr" href="#ref-31">Johnston <em>et al.</em> 2017</a>). At present, the mechanisms driving differences in mean ACC within and between species are unknown (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lenormand and Dutheil, 2005</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Stapley <em>et al.</em> 2017</a>). Male ACC was not significantly heritable, although we could not rule out that this was a consequence of the smaller sample size relative to females (Figure S3). No regions of the genome were significantly associated with male ACC in the regional heritability and GWAS analyses, but sampling did indicate that differences observed between male and female genomic associations were statistically significant (Figures S4 &amp; S5). Investigation of genetic correlations between males and females was inconclusive, as the <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/7/2265/embed/mml-math-68.gif"/></noscript></span></span> of ACC was not significantly different from 0 or 1. The observed sex differences are consistent with previous studies of the genetic architecture of ACC in mammals, where a sexually-dimorphic architecture has been observed at the paralogous <em>RNF212</em> region in humans and sheep (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Kong <em>et al.</em> 2014</a>; <a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Johnston <em>et al.</em> 2016</a>). Nevertheless, some observed associations were stronger when considering both male and female deer in the same analysis, for example at the most highly associated GWAS SNP, and the amplified signal for the regional heritability analysis on linkage group 33 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2</a> &amp; <a id="xref-fig-3-4" class="xref-fig" href="#F3">3</a>), suggesting that there may be some degree of shared architecture within these regions.</p></div></div><div id="sec-23" class="subsection"><h3>Conclusions and implications for studies of recombination in the wild</h3><p id="p-33">We have shown that recombination rate is heritable in female red deer, and that it has a sexually dimorphic genetic architecture. Genomic regions associated with recombination rate in red deer are associated with this trait in other mammal species, supporting the idea that recombination rate variation has a conserved genetic architecture across distantly related taxa. A key motivation for this study is to compare how recombination rate and its genetic architecture is similar or different to that of model species that have experienced strong selection in their recent history, such as humans, cattle, mice and sheep. The heritability of recombination rate in deer was lower than that observed in other mammal systems (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dumont <em>et al.</em> 2009</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Kong <em>et al.</em> 2014</a>; <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Ma <em>et al.</em> 2015</a>; <a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Johnston <em>et al.</em> 2016</a>), with no observed heritable variation present in male deer. While we were able to test their effects, we found no contribution of contribution of individual and common environmental effects on recombination rate (<em>i.e.</em>, age, year of birth, year of gamete transmission); indeed, most phenotypic variance in recombination was attributed to residual effects. This suggests that despite some underlying genetic variation, recombination rate is mostly driven by stochastic effects, or otherwise unmeasured effects.</p><p id="p-34">This represents one of the smallest datasets in which recombination rate has been investigated, and so it may be that the observed effects are underestimated due to the small sample size, sampling effects, or perhaps that other genetic variants present in this species do not segregate in the Rùm deer population. Nevertheless, identification of clear candidate genes and their effects on phenotype represents a valuable contribution to understanding the genetic architecture of recombination more broadly. Ultimately, our findings allow future investigation of the fitness consequences of variation in recombination rate and the relationship between identified variants and individual life-history variation, to address questions on the maintenance of genetic variation for recombination rates, and the relative roles of selection, sexually antagonistic effects and stochastic processes in contemporary natural populations.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">We thank T. Clutton-Brock and L. Kruuk for maintaining the long-term field project, and A. Morris, S. Morris, M. Baker, F. Guinness and many others for collecting field data and DNA samples. We thank Scottish Natural Heritage for permission to work on the Isle of Rùm National Nature Reserve. We thank P. Ellis for sample preparation and DNA extraction, and the Wellcome Trust Clinical Research Facility Genetics Core in Edinburgh for performing the genotyping. This work has made extensive use of the resources provided by the University of Edinburgh Compute and Data Facility (<a href="http://www.ecdf.ed.ac.uk/">http://www.ecdf.ed.ac.uk/</a>). The long-term project on Rùm red deer is funded by the UK Natural Environment Research Council.</p></div><div class="section ack" id="ack-2"><h2 class="">Author Contributions</h2><p id="p-36">The work was financially supported by a European Research Council Advanced Grant and a Natural Environment Research Council Grant to J.M.P (EC 250098 WEG &amp; NE/L00688X/1), and a Royal Society University Research Fellowship to S.E.J. (UF150448). S.E.J and J.M.P. conceived the study. J.M.P. organized the collection of samples. J.H. conducted DNA sample extraction and genotyping, curated the genomic data and constructed the pedigree. S.E.J. developed the methodology, conducted the data analysis and wrote the paper. All authors contributed to revisions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-37">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6166364">https://doi.org/10.25387/g3.6166364</a>.</p></li><li class="fn" id="fn-2"><p id="p-38"><em>Communicating editor: F. Pardo-Manuel de Villena</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-01-22"><span class="received-label">Received </span>January 22, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-04-30"><span class="accepted-label">Accepted </span>April 30, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Johnston <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.1" data-doi="10.1073/pnas.1416622112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arbeithuber</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiemann-Boege</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Crossovers are associated with mutation and biased gene conversion at recombination hotspots.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">2109</span>–<span class="cit-lpage">2114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1416622112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1416622112%26rft_id%253Dinfo%253Apmid%252F25646453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEyLzcvMjEwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNy8yMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.2" data-doi="10.1093/bioinformatics/btm108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaacs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">GenABEL: an R library for genome-wide association analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1294</span>–<span class="cit-lpage">1296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm108%26rft_id%253Dinfo%253Apmid%252F17384015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17384015&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247348300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.3" data-doi="10.7554/eLife.24133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schumer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haba</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bashkirova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Repeated losses of PRDM9-directed recombination despite the conservation of PRDM9 across vertebrates.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e24133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.24133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.24133%26rft_id%253Dinfo%253Apmid%252F28590247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.24133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28590247&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Why sex and recombination?</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D74%26rft.spage%253D187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.5" data-doi="10.1186/s12711-016-0221-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Battagin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faux</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effect of manipulating recombination rates on response to selection in livestock breeding programs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0221-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0221-1%26rft_id%253Dinfo%253Apmid%252F27335010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0221-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27335010&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.6" data-doi="10.1126/science.1183439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">840</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1183439</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183439%26rft_id%253Dinfo%253Apmid%252F20044539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.7" data-doi="10.1111/mec.13146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bérénos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilkington</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gratten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Heterogeneity of genetic architecture of body size traits in a free-living population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1810</span>–<span class="cit-lpage">1830</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D1810%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.8" data-doi="10.1534/genetics.111.136721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Scrambling eggs: meiotic drive and the evolution of female recombination rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.136721</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136721%26rft_id%253Dinfo%253Apmid%252F22143919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNzA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzIyNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.9" data-doi="10.1101/027318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brauning</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smithies</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Utilization of high throughput genome sequencing technology for large scale single nucleotide polymorphism discovery in red deer and canadian elk.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/027318</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F027318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwMjczMTh2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNy8yMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.10" data-doi="10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Sex, Recombination, and the Efficacy of Selection - was Weismann Right?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBurt%26rft.auinit1%253DA.%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D351%26rft.atitle%253DPerspective%253A%2Bsex%252C%2Brecombination%252C%2Band%2Bthe%2Befficacy%2Bof%2Bselection--was%2BWeismann%2Bright%253F%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282000%2529054%255B0337%253APSRATE%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F10937212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10937212&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086886500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-other" id="cit-8.7.2265.11"><div class="cit-metadata unstructured">Butler, D. G., B. R. Cullis, A. R. Gilmour, and B. J. Gogel, 2009 Mixed Models for S language Environments: ASReml-R reference manual. <a href="https://www.vsni.co.uk/software/asreml-r/">https://www.vsni.co.uk/software/asreml-r/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.12" data-doi="10.1371/journal.pgen.1002799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chelysheva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gendrot</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Arabidopsis HEI10 Is a New ZMM Protein Related to Zip3.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002799</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002799</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DChelysheva%26rft.auinit1%253DL.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002799%26rft.epage%253De1002799%26rft.atitle%253DThe%2BArabidopsis%2BHEI10%2Bis%2Ba%2Bnew%2BZMM%2Bprotein%2Brelated%2Bto%2BZip3.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002799%26rft_id%253Dinfo%253Apmid%252F22844245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002799&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22844245&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-8.7.2265.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clutton-Brock</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guinness</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albon</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-source"><em>Red Deer. Behavior and Ecology of Two Sexes</em>.</span> <span class="cit-publ-name">University of Chicago Press</span>, <span class="cit-publ-loc">Chicago</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.14" data-doi="10.1038/nchembio.1414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delaneau</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zagury</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improved whole-chromosome phasing for disease and population genetic studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2307</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnchembio.1414%26rft_id%253Dinfo%253Apmid%252F23269371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nchembio.1414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23269371&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312810100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.15" data-doi="10.1111/j.0006-341X.1999.00997.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genomic Control for Association.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1004</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0006-341X.1999.00997.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DDevlin%26rft.auinit1%253DB.%26rft.volume%253D55%26rft.issue%253D4%26rft.spage%253D997%26rft.epage%253D1004%26rft.atitle%253DGenomic%2Bcontrol%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0006-341X.1999.00997.x%26rft_id%253Dinfo%253Apmid%252F11315092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0006-341X.1999.00997.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11315092&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084218000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.16" data-doi="10.1534/genetics.109.105114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Variation in genomic recombination rates among heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1345</span>–<span class="cit-lpage">1349</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.105114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105114%26rft_id%253Dinfo%253Apmid%252F19535547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEzNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.17" data-doi="10.1101/gr.111252.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extensive recombination rate variation in the house mouse species complex inferred from genetic linkage maps.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.111252.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111252.110%26rft_id%253Dinfo%253Apmid%252F20978138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzEvMTE0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzIyNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.19" data-doi="10.1007/s00412-013-0434-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gómez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berenguer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llano</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pendás</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cohesin removal precedes topoisomerase II<em>α</em>-dependent decatenation at centromeres in male mammalian meiosis II.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00412-013-0434-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D123%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-013-0434-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-013-0434-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.20" data-doi="10.1186/s12711-017-0330-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Battagin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The potential of shifting recombination hotspots to increase genetic gain in livestock breeding.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0330-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0330-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0330-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-8.7.2265.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falls</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source"><em>Documentation for CRIMAP</em>, <em>Version 2.4</em>.</span> <span class="cit-publ-name">Washington University School of Medicine</span>, <span class="cit-publ-loc">St. Louis</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.22" data-doi="10.1038/35066065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hassold</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">To err (meiotically) is human: the genesis of human aneuploidy.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35066065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHassold%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D280%26rft.epage%253D291%26rft.atitle%253DTo%2Berr%2B%2528meiotically%2529%2Bis%2Bhuman%253A%2Bthe%2Bgenesis%2Bof%2Bhuman%2Baneuploidy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35066065%26rft_id%253Dinfo%253Apmid%252F11283700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35066065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11283700&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167837900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.23" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2529430</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.24" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300010156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.25" data-doi="10.1371/journal.pgen.1004650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Topoisomerase II Is Required for the Proper Separation of Heterochromatic Regions during Drosophila melanogaster Female Meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004650</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004650</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004650%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004650%26rft_id%253Dinfo%253Apmid%252F25340780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25340780&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.26" data-doi="10.1111/1755-0998.12665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Pedigree reconstruction from SNP data: parentage assignment, sibship clustering and beyond.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1024</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12665</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.27" data-doi="10.1073/pnas.1518046113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clutton-Brock</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inbreeding depression across the lifespan in a wild mammal population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">3585</span>–<span class="cit-lpage">3590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1518046113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518046113%26rft_id%253Dinfo%253Apmid%252F26979959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzEzLzM1ODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.28" data-doi="10.1146/annurev.genom.3.032802.120023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular mechanisms for genomic disorders.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genom.3.032802.120023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenomics%2BHum.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.3.032802.120023%26rft_id%253Dinfo%253Apmid%252F12142364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.3.032802.120023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12142364&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178678800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.29" data-doi="10.1534/genetics.116.195099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo-Lambert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabritius</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golden</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Identification of a Novel Mutant Allele of topoisomerase II in Caenorhabditis elegans Reveals a Unique Role in Chromosome Segregation During Spermatogenesis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">1407</span>–<span class="cit-lpage">1422</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.195099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.195099%26rft_id%253Dinfo%253Apmid%252F27707787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC80LzE0MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.30" data-doi="10.1534/genetics.115.185553"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérénos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Conserved genetic architecture underlying individual recombination rate variation in a wild population of soay sheep (ovis aries).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">598</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.185553</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185553%26rft_id%253Dinfo%253Apmid%252F27029733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvNTgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzIyNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.31" data-doi="10.1534/g3.117.044198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A High Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (<em>Cervus elaphus</em>).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2859</span>–<span class="cit-lpage">2870</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.044198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.044198%26rft_id%253Dinfo%253Apmid%252F28667018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOC8yODU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzIyNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.32" data-doi="10.1111/mec.14197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugal</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nater</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Whole-genome patterns of linkage disequilibrium across flycatcher populations clarify the causes and consequences of fine-scale recombination rate variation in birds.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4158</span>–<span class="cit-lpage">4172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D4158%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.33" data-doi="10.1038/ng1445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonsdottir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Recombination rate and reproductive success in humans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1203</span>–<span class="cit-lpage">1206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1445</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1203%26rft.epage%253D1206%26rft.atitle%253DRecombination%2Brate%2Band%2Breproductive%2Bsuccess%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1445%26rft_id%253Dinfo%253Apmid%252F15467721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15467721&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.34" data-doi="10.1038/ng.2833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frigge</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Common and low-frequency variants associated with genome-wide recombination rate.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2833</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2833%26rft_id%253Dinfo%253Apmid%252F24270358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2833&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24270358&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.35" data-doi="10.1126/science.1152422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefansson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helgason</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequence variants in the RNF212 gene associate with genome-wide recombination rate.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">1398</span>–<span class="cit-lpage">1401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1152422</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1152422%26rft_id%253Dinfo%253Apmid%252F18239089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2OC8xMzk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzIyNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.36" data-doi="10.1371/journal.pbio.0030063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutheil</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination difference between sexes: a role for haploid selection.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0030063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLenormand%26rft.auinit1%253DT.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253De63%26rft.epage%253De63%26rft.atitle%253DRecombination%2Bdifference%2Bbetween%2Bsexes%253A%2Ba%2Brole%2Bfor%2Bhaploid%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030063%26rft_id%253Dinfo%253Apmid%252F15736976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15736976&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.37" data-doi="10.1095/biolreprod.113.110478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DNA Topoisomerase II Is Dispensable for Oocyte Meiotic Resumption but Is Essential for Meiotic Chromosome Condensation and Separation in Mice.</span> <abbr class="cit-jnl-abbrev">Biol. Reprod.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1095/biolreprod.113.110692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2Bof%2BReproduction%26rft.stitle%253DBiol.%2BReprod.%26rft.aulast%253DSchubert%26rft.auinit1%253DC.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DWORLD%2BOF%2BREPRODUCTIVE%2BBIOLOGY%26rft_id%253Dinfo%253Adoi%252F10.1095%252Fbiolreprod.113.110478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImJpb2xyZXByb2QiO3M6NToicmVzaWQiO3M6NjoiODkvMS8xIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzIyNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.38" data-doi="10.1371/journal.pgen.1005387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padhi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cattle Sex-Specific Recombination and Genetic Control from a Large Pedigree Analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005387</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005387</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005387%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005387%26rft_id%253Dinfo%253Apmid%252F26540184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26540184&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.39" data-doi="10.1002/gepi.20331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moskvina</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">On multiple-testing correction in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">573</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20331</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DMoskvina%26rft.auinit1%253DV.%26rft.volume%253D32%26rft.issue%253D6%26rft.spage%253D567%26rft.epage%253D573%26rft.atitle%253DOn%2Bmultiple-testing%2Bcorrection%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20331%26rft_id%253Dinfo%253Apmid%252F18425821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18425821&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258871100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.40" data-doi="10.1016/0027-5107(64)90047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The relation of recombination to mutational advance.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0027-5107(64)90047-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%26rft.volume%253D1%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252864%252990047-8%26rft_id%253Dinfo%253Apmid%252F14195748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(64)90047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14195748&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1964WT37300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.41" data-doi="10.1093/molbev/msu322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Fuentes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Ortega</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkorta-Aranburu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linde Forsberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Strong Artificial Selection in Domestic Mammals Did Not Result in an Increased Recombination Rate.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">510</span>–<span class="cit-lpage">523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu322%26rft_id%253Dinfo%253Apmid%252F25414125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25414125&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.42" data-doi="10.1126/science.1117196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottolo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A fine-scale map of recombination rates and hotspots across the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1117196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DS.%26rft.volume%253D310%26rft.issue%253D5746%26rft.spage%253D321%26rft.epage%253D324%26rft.atitle%253DA%2BFine-Scale%2BMap%2Bof%2BRecombination%2BRates%2Band%2BHotspots%2BAcross%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1117196%26rft_id%253Dinfo%253Apmid%252F16224025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0Ni8zMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzcvMjI2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.43" data-doi="10.1371/journal.pone.0046501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagamine</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitart</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Localising Loci underlying Complex Trait Variation Using Regional Genomic Relationship Mapping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNagamine%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46501%26rft.epage%253De46501%26rft.atitle%253DLocalising%2BLoci%2Bunderlying%2BComplex%2BTrait%2BVariation%2BUsing%2BRegional%2BGenomic%2BRelationship%2BMapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046501%26rft_id%253Dinfo%253Apmid%252F23077511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23077511&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.44" data-doi="10.1371/journal.pgen.1004234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurdasani</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaneau</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirastu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulivi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A General Approach for Haplotype Phasing across the Full Spectrum of Relatedness.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004234%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004234%26rft_id%253Dinfo%253Apmid%252F24743097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24743097&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.45" data-doi="10.1554/0014-3820(2001)055[1921:SFRISP]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Selection for recombination in small populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1921</span>–<span class="cit-lpage">1931</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2001.tb01310.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP.%26rft.volume%253D55%26rft.issue%253D10%26rft.spage%253D1921%26rft.epage%253D1931%26rft.atitle%253DSelection%2Bfor%2Brecombination%2Bin%2Bsmall%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B1921%253ASFRISP%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11761054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[1921:SFRISP]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11761054&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172308600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.46" data-doi="10.1038/nrg761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Resolving the paradox of sex and recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">261</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg761</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D252%26rft.epage%253D261%26rft.atitle%253DResolving%2Bthe%2Bparadox%2Bof%2Bsex%2Band%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg761%26rft_id%253Dinfo%253Apmid%252F11967550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11967550&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174739800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.47" data-doi="10.1128/MCB.19.5.3515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parisi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molnar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Spek</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Rec8p, a Meiotic Recombination and Sister Chromatid Cohesion Phosphoprotein of the Rad21p Family Conserved from Fission Yeast to Humans.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3515</span>–<span class="cit-lpage">3528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.19.5.3515</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DParisi%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D3515%26rft.epage%253D3528%26rft.atitle%253DRec8p%252C%2Ba%2BMeiotic%2BRecombination%2Band%2BSister%2BChromatid%2BCohesion%2BPhosphoprotein%2Bof%2Bthe%2BRad21p%2BFamily%2BConserved%2Bfrom%2BFission%2BYeast%2Bto%2BHumans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.5.3515%26rft_id%253Dinfo%253Apmid%252F10207075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE5LzUvMzUxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNy8yMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.48" data-doi="10.1534/genetics.117.300123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astruc</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drouilhet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in Recombination Rate and Its Genetic Determinism in Sheep Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300123%26rft_id%253Dinfo%253Apmid%252F28978774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA3LzIvNzY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC83LzIyNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.49" data-doi="10.1038/ng.2858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">H. B. D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fong</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cloutier</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Antagonistic roles of ubiquitin ligase HEI10 and SUMO ligase RNF212 regulate meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">194</span>–<span class="cit-lpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2858</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D194%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2858%26rft_id%253Dinfo%253Apmid%252F24390283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24390283&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.50" data-doi="10.1007/s00251-009-0384-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinink</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Rhijn</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The bovine T cell receptor alpha/delta locus contains over 400 V genes and encodes V genes without CDR2.</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">549</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00251-009-0384-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.volume%253D61%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00251-009-0384-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00251-009-0384-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.51" data-doi="10.1038/ng.2541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNF212 is a dosage-sensitive regulator of crossing-over during mammalian meiosis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">278</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2541</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2541%26rft_id%253Dinfo%253Apmid%252F23396135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2541&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23396135&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.52" data-doi="10.1111/2041-210X.12535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McFarlane</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Increasing the power of genome wide association studies in natural populations using repeated measures: evaluation and implementation.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">792</span>–<span class="cit-lpage">799</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/2041-210X.12535</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D7%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.53" data-doi="10.1371/journal.pgen.1002854"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandor</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coppieters</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic variants in <em>REC8</em>, <em>RNF212</em>, and <em>PRDM9</em> influence male recombination in cattle.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002854</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002854</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSandor%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002854%26rft.epage%253De1002854%26rft.atitle%253DGenetic%2Bvariants%2Bin%2BREC8%252C%2BRNF212%252C%2Band%2BPRDM9%2Binfluence%2Bmale%2Brecombination%2Bin%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002854%26rft_id%253Dinfo%253Apmid%252F22844258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002854&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22844258&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeney</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> Ldheatmap: An r function for graphical display of pairwise linkage disequilibria between single nucleotide polymorphisms. J. Stat. Soft. 16: Code Snippet 3.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.55" data-doi="10.1098/rstb.2016.0455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feulner</span>  <span class="cit-name-given-names">P. G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santure</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smadja</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in recombination frequency and distribution across Eukaryotes: patterns and processes.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">372</span>: <span class="cit-fpage">20160455</span> <span class="cit-comment">[corrigenda: Philos. Trans. R. Soc. Lond. B Biol. Sci. 373 (2018)]</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2016.0455</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0455%26rft_id%253Dinfo%253Apmid%252F29109219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNzIvMTczNi8yMDE2MDQ1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNy8yMjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.56" data-doi="10.1098/rspb.2016.1821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theodosiou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">W. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puebla</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Recombination in the eggs and sperm in a simultaneously hermaphroditic vertebrate.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">20161821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2016.1821</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft.volume%253D283%26rft.spage%253D20161821%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2016.1821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2016.1821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.57" data-doi="10.1038/srep37698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effect of sex, age and genetics on crossover interference in cattle.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">37698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep37698</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D37698%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep37698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep37698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.58" data-doi="10.1016/j.ajhg.2010.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GCTA: a tool for genome-wide complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2010.11.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D82%26rft.atitle%253DGCTA%253A%2Ba%2Btool%2Bfor%2Bgenome-wide%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.11.011%26rft_id%253Dinfo%253Apmid%252F21167468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21167468&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.59" data-doi="10.4161/cc.22690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NEK9 regulates spindle organization and cell cycle progression during mouse oocyte meiosis and its location in early embryo mitosis.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">4366</span>–<span class="cit-lpage">4377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/cc.22690</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D11%26rft.spage%253D4366%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.22690%26rft_id%253Dinfo%253Apmid%252F23159858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.22690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23159858&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311951500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.7.2265.60" data-doi="10.1093/nar/gkx1098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achuthan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akanni</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amode</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Ensembl 2018.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D754</span>–<span class="cit-lpage">D761</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD754%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1098%26rft_id%253Dinfo%253Apmid%252F29155950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29155950&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F7%2F2265.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/7/2265.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/7/2257" title="A Genome-Wide Assay Specifies Only GreA as a Transcription Fidelity Factor in Escherichia coli" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/7/2277" title="Tissue-Specific Functions of fem-2/PP2c Phosphatase and fhod-1/formin During Caenorhabditis elegans Embryonic Morphogenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/7"><span>Volume 8 Issue 7, July 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/7.cover-source.jpg?itok=dg_JR4dD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/7/2265" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ivg8DPw5jRLlKvhBF7bT3xW09hPy9hXq9RMNX7kzLw4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/7/2265" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/7/2265" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/7/2265" class="active" data-icon-position="" data-hide-link-title="0">A Genomic Region Containing REC8 and RNF212B Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (Cervus elaphus)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490130" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xQm-kw6bq4VuVpQTBKmRY05ndIGeltzahHRPzs-3uP0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/7/2265.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/7/2265" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nStN9LdFnWqx31EShJygYlNwioR_Mci6ddW_3Esl--Y" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/7/2265" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/7/2265.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490130" id="node490130--2" data-pisa="ggg;8/7/2265" data-pisa-master="ggg;g3.118.200063" data-apath="/ggg/8/7/2265.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Region Containing <em>REC8</em> and <em>RNF212B</em> Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (<em>Cervus elaphus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan E. Johnston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jisca Huisman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josephine M. Pemberton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2265-2276;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200063 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490130--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susan E. Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BE.%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan E. Johnston</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Susan.Johnston@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Susan.Johnston@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jisca Huisman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jisca%2BHuisman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huisman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJisca%2BHuisman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jisca Huisman</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Josephine M. Pemberton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BM.%2BPemberton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pemberton%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosephine%2BM.%2BPemberton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josephine M. Pemberton</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/7/2265" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/7/2265&amp;title=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/7/2265&amp;title=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/7/2265&amp;title=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/7/2265&amp;text=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/7/2265&amp;title=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/7/2265&amp;t=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/7/2265&amp;title=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/7/2265&amp;title=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490130/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490130" id="node490130--3" data-pisa="ggg;8/7/2265" data-pisa-master="ggg;g3.118.200063" data-apath="/ggg/8/7/2265.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Region Containing <em>REC8</em> and <em>RNF212B</em> Is Associated with Individual Recombination Rate Variation in a Wild Population of Red Deer (<em>Cervus elaphus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan E. Johnston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jisca Huisman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josephine M. Pemberton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2265-2276;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200063 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490130--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susan E. Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BE.%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan E. Johnston</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Susan.Johnston@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Susan.Johnston@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jisca Huisman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jisca%2BHuisman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huisman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJisca%2BHuisman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jisca Huisman</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Josephine M. Pemberton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BM.%2BPemberton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pemberton%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosephine%2BM.%2BPemberton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josephine M. Pemberton</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2265&amp;rft.epage=2276&amp;rft.atitle=A%20Genomic%20Region%20Containing%20REC8%20and%20RNF212B%20Is%20Associated%20with%20Individual%20Recombination%20Rate%20Variation%20in%20a%20Wild%20Population%20of%20Red%20Deer%20%28Cervus%20elaphus%29&amp;rft.volume=8&amp;rft.issue=7&amp;rft.date=2018-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Johnston%2C+Susan+E.&amp;rft.au=Huisman%2C+Jisca&amp;rft.au=Pemberton%2C+Josephine+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490130/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490130/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490130/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490130/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490130/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490130/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490130/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490130/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490130/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490130/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490130/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490130/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/7/2265" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials And Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#ack-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Author Contributions</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/7/2265.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/7/2265.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/7/2265.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/7/2265.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
